

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Gene Expression Analysis of In Vivo Fluorescent Cells</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001151"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001151" />

    <meta name="citation_title" content="Gene Expression Analysis of In Vivo Fluorescent Cells"/>
    <meta itemprop="name" content="Gene Expression Analysis of In Vivo Fluorescent Cells"/>

      <meta name="citation_author" content="Konstantin Khodosevich"/>
            <meta name="citation_author_institution" content="Department of Clinical Neurobiology, Interdisciplinary Center for Neuroscience, University of Heidelberg, Heidelberg, Germany"/>
      <meta name="citation_author" content="Dragos Inta"/>
            <meta name="citation_author_institution" content="Department of Clinical Neurobiology, Interdisciplinary Center for Neuroscience, University of Heidelberg, Heidelberg, Germany"/>
      <meta name="citation_author" content="Peter H. Seeburg"/>
            <meta name="citation_author_institution" content="Department of Molecular Neuroscience, Max-Planck-Institute for Medical Research, Heidelberg, Germany"/>
      <meta name="citation_author" content="Hannah Monyer"/>
            <meta name="citation_author_institution" content="Department of Clinical Neurobiology, Interdisciplinary Center for Neuroscience, University of Heidelberg, Heidelberg, Germany"/>

    <meta name="citation_date" content="2007/11/7"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001151.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1151"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Laser capture microdissection.; citation_author=MR Emmert-Buck; citation_author=RF Bonner; citation_author=PD Smith; citation_author=RF Chuaqui; citation_author=Z Zhuang; citation_journal_title=Science; citation_volume=274; citation_number=1; citation_pages=998-1001; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Analysis of anterior pituitary hormone mRNA expression in immunophenotypically characterized single cells after laser capture microdissection.; citation_author=L Jin; citation_author=CA Thompson; citation_author=X Qian; citation_author=SJ Kuecker; citation_author=E Kulig; citation_journal_title=Lab Invest; citation_volume=79; citation_number=2; citation_pages=511-512; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Influence of histochemical stains on quantitative gene expression analysis after laser-assisted microdissection.; citation_author=AF Okuducu; citation_author=V Janzen; citation_author=JC Hahne; citation_author=Y Ko; citation_author=N Wernert; citation_journal_title=Int J Mol Med; citation_volume=11; citation_number=3; citation_pages=449-453; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Immunostaining and laser-assisted cell picking for mRNA analysis.; citation_author=L Fink; citation_author=T Kinfe; citation_author=MM Stein; citation_author=L Ermert; citation_author=J Hanze; citation_journal_title=Lab Invest; citation_volume=80; citation_number=4; citation_pages=327-333; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=IF-LCM: laser capture microdissection of immunofluorescently defined cells for mRNA analysis rapid communication.; citation_author=H Murakami; citation_author=L Liotta; citation_author=RA Star; citation_journal_title=Kidney Int; citation_volume=58; citation_number=5; citation_pages=1346-1353; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The influence of immunohistochemistry on mRNA recovery from microdissected frozen and formalin-fixed, paraffin-embedded sections.; citation_author=LM Gjerdrum; citation_author=HN Abrahamsen; citation_author=B Villegas; citation_author=BS Sorensen; citation_author=H Schmidt; citation_journal_title=Diagn Mol Pathol; citation_volume=13; citation_number=6; citation_pages=224-233; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Histological staining methods preparatory to laser capture microdissection significantly affect the integrity of the cellular RNA.; citation_author=H Wang; citation_author=JD Owens; citation_author=JH Shih; citation_author=MC Li; citation_author=RF Bonner; citation_journal_title=BMC Genomics; citation_volume=7; citation_number=7; citation_pages=97; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gene expression nervous system atlas (GENSAT).; citation_author=N Heintz; citation_journal_title=Nat Neurosci; citation_volume=7; citation_number=8; citation_pages=483; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Imaging neuronal subsets in transgenic mice expressing multiple spectral variants of GFP.; citation_author=G Feng; citation_author=RH Mellor; citation_author=M Bernstein; citation_author=C Keller-Peck; citation_author=QT Nguyen; citation_journal_title=Neuron; citation_volume=28; citation_number=9; citation_pages=41-51; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=In vivo somatic delivery of plasmid DNA and retrograde transport to obtain cell-specific gene expression in the central nervous system.; citation_author=R Morris; citation_author=BS Morgan; citation_author=TM Lewis; citation_author=KD Pierce; citation_author=A Pisano; citation_journal_title=J Neurochem; citation_volume=90; citation_number=10; citation_pages=1445-1452; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Germline transmission and tissue-specific expression of transgenes delivered by lentiviral vectors.; citation_author=C Lois; citation_author=EJ Hong; citation_author=S Pease; citation_author=EJ Brown; citation_author=D Baltimore; citation_journal_title=Science; citation_volume=295; citation_number=11; citation_pages=868-872; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Formaldehyde cross-linking for studying nucleosomal dynamics.; citation_author=V Jackson; citation_journal_title=Methods; citation_volume=17; citation_number=12; citation_pages=125-139; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Gene expression profiling of primary tumor cell populations using laser capture microdissection, RNA transcript amplification, and GeneChip microarrays.; citation_author=VI Luzzi; citation_author=V Holtschlag; citation_author=MA Watson; citation_journal_title=Methods Mol Biol; citation_volume=293; citation_number=13; citation_pages=187-207; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Adult neurogenesis: from precursors to network and physiology.; citation_author=DN Abrous; citation_author=M Koehl; citation_author=M Le Moal; citation_journal_title=Physiol Rev; citation_volume=85; citation_number=14; citation_pages=523-569; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparative localization of two forms of glutamic acid decarboxylase and their mRNAs in rat brain supports the concept of functional differences between the forms.; citation_author=M Esclapez; citation_author=NJ Tillakaratne; citation_author=DL Kaufman; citation_author=AJ Tobin; citation_author=CR Houser; citation_journal_title=J Neurosci; citation_volume=14; citation_number=15; citation_pages=1834-1855; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Detection of oligodendrocytes in tissue sections using PCR synthesis of digoxigenin-labeled probes.; citation_author=W Jalabi; citation_author=M Cerghet; citation_author=RP Skoff; citation_author=MS Ghandour; citation_journal_title=J Histochem Cytochem; citation_volume=51; citation_number=16; citation_pages=913-919; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Differentially expressed transcripts from phenotypically identified olfactory sensory neurons.; citation_author=TT Yu; citation_author=JC McIntyre; citation_author=SC Bose; citation_author=D Hardin; citation_author=MC Owen; citation_journal_title=J Comp Neurol; citation_volume=483; citation_number=17; citation_pages=251-262; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Neuronal expression of macrophage colony stimulating factor in Purkinje cells and olfactory mitral cells of wild-type and cerebellar-mutant mice.; citation_author=S Murase; citation_author=Y Hayashi; citation_journal_title=Histochem J; citation_volume=34; citation_number=18; citation_pages=85-95; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A family of AMPA-selective glutamate receptors.; citation_author=K Keinanen; citation_author=W Wisden; citation_author=B Sommer; citation_author=P Werner; citation_author=A Herb; citation_journal_title=Science; citation_volume=249; citation_number=19; citation_pages=556-560; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Axon development in mouse cerebellum: embryonic axon forms and expression of synapsin I.; citation_author=CA Mason; citation_journal_title=Neuroscience; citation_volume=19; citation_number=20; citation_pages=1319-1333; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Global transcriptome analysis of genetically identified neurons in the adult cortex.; citation_author=MJ Rossner; citation_author=J Hirrlinger; citation_author=SP Wichert; citation_author=C Boehm; citation_author=D Newrzella; citation_journal_title=J Neurosci; citation_volume=26; citation_number=21; citation_pages=9956-9966; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Laser capture microdissection of fluorescently labeled embryonic cranial neural crest cells.; citation_author=V Bhattacherjee; citation_author=P Mukhopadhyay; citation_author=S Singh; citation_author=EA Roberts; citation_author=RC Hackmiller; citation_journal_title=Genesis; citation_volume=39; citation_number=22; citation_pages=58-64; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Effects of fixation on RNA extraction and amplification from laser capture microdissected tissue.; citation_author=SM Goldsworthy; citation_author=PS Stockton; citation_author=CS Trempus; citation_author=JF Foley; citation_author=RR Maronpot; citation_journal_title=Mol Carcinog; citation_volume=25; citation_number=23; citation_pages=86-91; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Assessment of fixatives, fixation, and tissue processing on morphology and RNA integrity.; citation_author=ML Cox; citation_author=CL Schray; citation_author=CN Luster; citation_author=ZS Stewart; citation_author=PJ Korytko; citation_journal_title=Exp Mol Pathol; citation_volume=80; citation_number=24; citation_pages=183-191; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Protecting RNA in fixed tissue: an alternative method for LCM users.; citation_author=AH Kihara; citation_author=AS Moriscot; citation_author=PJ Ferreira; citation_author=DE Hamassaki; citation_journal_title=J Neurosci Methods; citation_volume=148; citation_number=25; citation_pages=103-107; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene expression patterns in rat dentate granule cells: comparison between fresh and fixed tissue.; citation_author=Y Qin; citation_author=VM Heine; citation_author=H Karst; citation_author=PJ Lucassen; citation_author=M Joels; citation_journal_title=J Neurosci Methods; citation_volume=131; citation_number=26; citation_pages=205-211; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Laser microdissection and RNA analysis.; citation_author=L Fink; citation_author=RM Bohle; citation_journal_title=Methods Mol Biol; citation_volume=293; citation_number=27; citation_pages=167-185; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of chromatin structure by in vivo formaldehyde cross-linking.; citation_author=V Orlando; citation_author=H Strutt; citation_author=R Paro; citation_journal_title=Methods; citation_volume=11; citation_number=28; citation_pages=205-214; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The history and future of the fluorescence activated cell sorter and flow cytometry: a view from Stanford.; citation_author=LA Herzenberg; citation_author=D Parks; citation_author=B Sahaf; citation_author=O Perez; citation_author=M Roederer; citation_journal_title=Clin Chem; citation_volume=48; citation_number=29; citation_pages=1819-1827; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=FACS-array profiling of striatal projection neuron subtypes in juvenile and adult mouse brains.; citation_author=MK Lobo; citation_author=SL Karsten; citation_author=M Gray; citation_author=DH Geschwind; citation_author=XW Yang; citation_journal_title=Nat Neurosci; citation_volume=9; citation_number=30; citation_pages=443-452; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Chemical labeling strategies for cell biology.; citation_author=KM Marks; citation_author=GP Nolan; citation_journal_title=Nat Methods; citation_volume=3; citation_number=31; citation_pages=591-596; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Bioconductor: open software development for computational biology and bioinformatics.; citation_author=RC Gentleman; citation_author=VJ Carey; citation_author=DM Bates; citation_author=B Bolstad; citation_author=M Dettling; citation_journal_title=Genome Biol; citation_volume=5; citation_number=32; citation_pages=R80; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001151" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Gene Expression Analysis of In Vivo Fluorescent Cells"/>
    <meta name="twitter:description" content="BackgroundThe analysis of gene expression for tissue homogenates is of limited value because of the considerable cell heterogeneity in tissues. However, several methods are available to isolate a cell type of interest from a complex tissue, the most reliable one being Laser Microdissection (LMD). Cells may be distinguished by their morphology or by specific antigens, but the obligatory staining often results in RNA degradation. Alternatively, particular cell types can be detected in vivo by expression of fluorescent proteins from cell type-specific promoters.Methodology/Principal FindingsWe developed a technique for fixing in vivo fluorescence in brain cells and isolating them by LMD followed by an optimized RNA isolation procedure. RNA isolated from these cells was of equal quality as from unfixed frozen tissue, with clear 28S and 18S rRNA bands of a mass ratio of &sim;2&#8758;1. We confirmed the specificity of the amplified RNA from the microdissected fluorescent cells as well as its usefulness and reproducibility for microarray hybridization and quantitative real-time PCR (qRT-PCR).Conclusions/SignificanceOur technique guarantees the isolation of sufficient high quality RNA obtained from specific cell populations of the brain expressing soluble fluorescent marker, which is a critical prerequisite for subsequent gene expression studies by microarray analysis or qRT-PCR."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001151.g004"/>

  <meta property="og:title" content="Gene Expression Analysis of In Vivo Fluorescent Cells" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001151" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7077'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5872'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4336&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001151">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001151" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001151&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Gene Expression Analysis of <i>In Vivo</i> Fluorescent Cells
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Konstantin Khodosevich, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Clinical Neurobiology, Interdisciplinary Center for Neuroscience, University of Heidelberg, Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dragos Inta, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Clinical Neurobiology, Interdisciplinary Center for Neuroscience, University of Heidelberg, Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter H. Seeburg, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Neuroscience, Max-Planck-Institute for Medical Research, Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hannah Monyer
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:monyer@urz.uni-hd.de">monyer@urz.uni-hd.de</a></p>

                <p>Affiliation:
                  Department of Clinical Neurobiology, Interdisciplinary Center for Neuroscience, University of Heidelberg, Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 07, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001151</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001151" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001151" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001151" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001151" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001151-g001" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001151-g002" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001151-g003" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001151-t001" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001151-g004" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001151-t002" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001151">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001151" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The analysis of gene expression for tissue homogenates is of limited value because of the considerable cell heterogeneity in tissues. However, several methods are available to isolate a cell type of interest from a complex tissue, the most reliable one being Laser Microdissection (LMD). Cells may be distinguished by their morphology or by specific antigens, but the obligatory staining often results in RNA degradation. Alternatively, particular cell types can be detected <em>in vivo</em> by expression of fluorescent proteins from cell type-specific promoters.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We developed a technique for fixing <em>in vivo</em> fluorescence in brain cells and isolating them by LMD followed by an optimized RNA isolation procedure. RNA isolated from these cells was of equal quality as from unfixed frozen tissue, with clear 28S and 18S rRNA bands of a mass ratio of ~2:1. We confirmed the specificity of the amplified RNA from the microdissected fluorescent cells as well as its usefulness and reproducibility for microarray hybridization and quantitative real-time PCR (qRT-PCR).</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our technique guarantees the isolation of sufficient high quality RNA obtained from specific cell populations of the brain expressing soluble fluorescent marker, which is a critical prerequisite for subsequent gene expression studies by microarray analysis or qRT-PCR.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Khodosevich K, Inta D, Seeburg PH, Monyer H (2007) Gene Expression Analysis of <em>In Vivo</em> Fluorescent Cells. PLoS ONE 2(11):
          e1151.
            doi:10.1371/journal.pone.0001151</p><p><strong>Academic Editor: </strong>Peter Sommer, Institut Pasteur Korea, Republic of Korea</p><p><strong>Received:</strong> September 3, 2007; <strong>Accepted:</strong> October 22, 2007; <strong>Published:</strong> November 7, 2007</p><p><strong>Copyright:</strong> © 2007 Khodosevich et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported in part by the Schilling Foundation, the SFB484 and the BMBF grants 01GS0117 and 01GS0498.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>There is increasing interest in tissue or cell type-specific gene expression analysis to identify genes involved in diseases, cell fate determination, or response to external stimuli. Researchers have attempted to develop methods for the isolation of homogeneous cell populations, such as flow cytometry and mechanical dissection, but these methods have practical limitations. Laser Microdissection (LMD), developed a decade ago <a href="#pone.0001151-EmmertBuck1">[1]</a> and currently used routinely in clinical and basic research applications, has permitted the isolation of distinct cell populations from complex tissues. It allows for precise analysis of DNA, RNA or proteins from cells of interest. Cells may be conveniently distinguished on the basis of morphology or specific antigens, which is achieved by staining with different dyes or specific antibodies. Incubation with these agents does not interfere with the integrity of DNA or protein isolated from microdissected cells, but compromises RNA quality due to RNase activity <a href="#pone.0001151-Jin1">[2]</a>, <a href="#pone.0001151-Okuducu1">[3]</a>. In spite of the existence of staining protocols optimized for RNA quality<a href="#pone.0001151-Fink1">[4]</a>, <a href="#pone.0001151-Murakami1">[5]</a>, RNA degradation that occurs during staining procedures is still an unresolved problem <a href="#pone.0001151-Gjerdrum1">[6]</a>, <a href="#pone.0001151-Wang1">[7]</a>. Furthermore, only few cell types can be identified by cell morphology and specific antibodies for immunohistochemistry are available only for a limited range of proteins.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>More recently, the identification of distinct cell types has been facilitated by the transgenic expression of fluorescent proteins. A major advance has been the generation of numerous transgenic mouse lines that express EGFP (enhanced green fluorescent protein) in defined cell populations <a href="#pone.0001151-Heintz1">[8]</a>, <a href="#pone.0001151-Feng1">[9]</a>. Also, to date, several techniques for <em>in vivo</em> fluorescent labeling of particular cell types/populations have been developed including viral or naked DNA delivery <a href="#pone.0001151-Morris1">[10]</a>, <a href="#pone.0001151-Lois1">[11]</a>. However, there is a clear need for a reliable technique to separate specific <em>in vivo</em> fluorescent neural cell populations for subsequent gene expression analysis. We have now developed such a technique for the harvesting by LMD of fluorescent cells from brain tissue with subsequent RNA isolation and gene expression analysis by real-time PCR and microarrays. We demonstrate high quality of the isolated RNA from a defined cell population, the periglomerular cells of the olfactory bulb, and show the usefulness and specificity of the technique for subsequent gene expression studies by quantitative real-time PCR and microarrays studies.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We employed in our study transgenic 5HT3<sub>A</sub>-EGFP mice, in which the enhanced green fluorescent protein, EGFP, is expressed from the promoter of the serotonin receptor gene <em>5HT3<sub>A</sub></em>. The unique and faithful expression pattern of the transgene has been reported elsewhere (Inta et al., manuscript submitted). During early postnatal life, EGFP is specifically expressed in neuroblasts migrating from the subventricular zone (SVZ) prominently to the olfactory bulb where neuroblasts mature into granule and periglomerular cells, subpopulations of GABAergic interneurons. Green fluorescent cells are clearly distinguishable in the olfactory bulb as well as on their route to the olfactory bulb from the SVZ.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>To fix the fluorescence in the cells of interest, we perfused the mice with paraformaldehyde (PFA, 0.5% or 2%) and then cryoprotected the fixed tissue by sucrose to preserve tissue morphology. Perfusion is a fast and effective way for complete fixation of tissues, which is very important when working with RNA, as it prevents RNase activity.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>Fixed brains were frozen, and sliced into 6–8 µm sagittal sections. Although PFA fixation alone was sufficient to preserve fluorescence for a short time, it was not enough to keep intensity of fluorescence signal longer then 30–60 min (data not shown). Therefore, sections were additionally fixed and dehydrated by alcohol, following clearing by xylene. Incubation in a precipitative fixative, such as alcohol, constitutes an additional fixation step, necessary not only to remove the water to prevent RNA degradation, but also to render the aldehyde-crosslinks more stable, thus preserving the fluorescence. Alcohol fixation alone also was not sufficient to preserve fluorescence of the soluble EGFP and prevent it from leaching out and diffusing to neighboring tissue making it impossible to specifically identify green fluorescent cells (data not shown). To identify the conditions that best preserve tissue morphology and cause minimal tissue autofluorescence, we tested different alcohols. Thus, 50% ethanol followed by 100% n-butanol was found to be optimal. Different brain areas containing green fluorescent cells are shown in <a href="#pone-0001151-g001">Fig. 1A–C</a> (for comparison with standard fixation procedure by 4% PFA in fluorescence preservation, see the same brain areas in <a href="#pone-0001151-g001">Fig. 1F–H</a>). The concentration of PFA was not found to influence specific fluorescence-there was no notable difference when 0.5% or 2% PFA was used-but autofluorescence was reduced at lower PFA concentrations.</p>
<div class="figure" id="pone-0001151-g001"><div class="img"><a name="pone-0001151-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001151.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001151.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Green fluorescent cells in the 5HT3<sub>A</sub>-EGFP mouse brain.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Cortex (<em>A</em>) and (<em>F</em>), hippocampus (<em>B</em>) and (<em>G</em>), and olfactory bulb (<em>C</em>) and (<em>H</em>) fixed by LMD-adapted and standard fixation protocol, respectively. (<em>D</em>) and (<em>E</em>) Periglomerular cell layer of olfactory bulb before and after microdissection, respectively. (<em>A</em>)–(<em>E</em>) images were made on a LMD microscope with brain sections mounted on membrane polyester slides. (<em>F</em>)–(<em>H</em>) images were made on an upright fluorescent microscope. For images (<em>A</em>)–(<em>E</em>) LMD microscope-generated scale bars are shown in the left down corner, (<em>A</em>)-50 µm, (<em>B</em>) and (<em>C</em>)-100 µm, (<em>D</em>) and (<em>E</em>)-25 µm. For images (<em>F</em>)–(<em>H</em>) scale bar is 25 µm.</p>
<span>doi:10.1371/journal.pone.0001151.g001</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>Sections containing the periglomerular cell layer of olfactory bulb were mounted directly on membrane slides for LMD and immediately processed by alcohol dehydration/fixation. Leaving mounted sections even at −70°C was found to impair tissue morphology. Since membrane slides are not as solid as glass slides, and mounted tissue might lead to membrane shrinkage. Green fluorescent periglomerular cells were microdissected into dry 0.2 ml tube caps coated with silicon (<a href="#pone-0001151-g001">Fig. 1D, E</a>). To minimize RNA degradation we did not use any collecting liquid. After dissecting 3,000–5,000 cells, we isolated their RNA by a modified proteinase K/acid phenol method.</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>As methylene bridges formed by aldehyde fixation are reversible <a href="#pone.0001151-Jackson1">[12]</a>, we were able to reverse the extensive network of aldehyde crosslinking by an optimized proteinase K/SDS lysis solution and a subsequent treatment with phenol pH 4.2. Using this RNA isolation procedure, we obtained from the fixed tissue RNA of the same quality as that of RNA isolated from unfixed frozen tissue (<a href="#pone-0001151-g002">Fig. 2A, B</a>, respectively). The 28S:18S rRNA ratio in the samples obtained was around 2:1, which is excellent for RNA isolated from tissues by LMD <a href="#pone.0001151-Luzzi1">[13]</a>. Furthermore, the yield of RNA from the fixed tissue (6–9 ng from 3,000–5,000 cells) exceeded that from frozen tissue (4–7 ng from 3,000–5,000 cells), probably due to inactivation of RNases by aldehyde fixation.</p>
<div class="figure" id="pone-0001151-g002"><div class="img"><a name="pone-0001151-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001151.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001151.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>RNA analysis by the Bioanalyzer 2100.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>(<em>A</em>) RNA isolated from fixed tissue by the optimized proteinase K/acid phenol method described here. (<em>B</em>) RNA isolated from frozen tissue by the optimized proteinase K/acid phenol method. (<em>C</em>) RNA isolated from fixed tissue by TRIzol method. (<em>D</em>) RNA isolated from fixed tissue by RNeasy Micro Kit. <em>(E)</em> One round of amplification of RNA isolated from fixed tissue by the optimized proteinase K/acid phenol method. (<em>F</em>) One round of amplification of Ambion Control RNA. (<em>G</em>) Two rounds of amplification of RNA isolated from fixed tissue by the optimized proteinase K/acid phenol method. (<em>H</em>) RNA ladder: first peak is RNA marker, next mark 200, 500, 1000, 2000, 4000 and 6000 nt.</p>
<span>doi:10.1371/journal.pone.0001151.g002</span></div><a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>We compared our procedure for RNA isolation with the standard TRIzol (Invitrogen, Germany) method and the RNeasy Micro Kit (QIAGEN, Germany). Yield and quality of RNA obtained by these methods were inferior to those obtained by our procedure (<a href="#pone-0001151-g002">Fig. 2C, D</a>), most likely due to the lack of reversing the network of aldehyde crosslinks.</p>
<a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>We next amplified 2–3 ng of periglomerular cell RNA isolated by the method described above using a MessageAmp II aRNA Amplification Kit (Ambion, USA). The yield and size range of the amplified RNA were comparable to those obtained for the Ambion Control RNA (Ambion, USA) (<a href="#pone-0001151-g002">Fig. 2E, F</a>, respectively). The average length of the amplified mRNA pool (typical yield ~50 ng) was around 1,000 nucleotides and was comparable to the Ambion RNA control. The length of the mRNA was judged by comparing it with the Ambion RNA Marker (<a href="#pone-0001151-g002">Fig. 2H</a>). After a second round of RNA amplification we typically obtained 50–100 µg of RNA of a 200–2,000 nucleotide range (<a href="#pone-0001151-g002">Fig. 2G</a>). Thus, we were able to obtain from 3,000–5,000 microdissected fluorescent cells high quality RNA that suffices for several microarray hybridizations.</p>
<a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p>We performed microarray hybridization for three samples of amplified periglomerular cell RNA and analyzed gene expression based on the signal detection value (<a href="#pone-0001151-t001">Table 1</a>). Full analysis of microarray data as well as raw data can be found on <a href="http://www.ebi.ac.uk/miamexpress">http://www.ebi.ac.uk/miamexpress</a> (accession number is E-MEXP-1234, public access is available from the 10.11.2007). Since the background signal detection value is about 50–70, genes with a signal detection value higher than 100 were further analyzed using GeneOntology (GO) and KEGG Pathway databases. The reproducibility and quality of the data were demonstrated by scatter plot analysis (<a href="#pone-0001151-g003">Fig. 3</a>).</p>
<div class="figure" id="pone-0001151-g003"><div class="img"><a name="pone-0001151-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001151.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001151.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Scatter plots of signal detection values obtained in three independent microarray hybridizations.</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>(<em>A</em>) Scatter plot between array N1 and array N2. (<em>B</em>) Scatter plot between array N1 and array N3. (<em>C</em>) Scatter plot between array N2 and array N3. Diagonal lines represent 2x, 3x, 5x and 10x difference between signal detection values. Red dots = P-P, present in both experiments. Black dots = P-A, present in the first experiment and absent in the second. Green dots = A-P, absent in the first experiment and present in the second. Yellow dots = A-A, A-M, M-A, M-M, absent or marginal in the first and/or second experiment. Blue dots = M-P, P-M, present or marginal in the first or second experiment.</p>
<span>doi:10.1371/journal.pone.0001151.g003</span></div><div class="figure" id="pone-0001151-t001"><div class="img"><a name="pone-0001151-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001151.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001151.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Microarray and qRT-PCR analysis for expressed and non-expressed genes in periglomerular cells<sup>1</sup></span></strong></p><span>doi:10.1371/journal.pone.0001151.t001</span></div><a id="article1.body1.sec2.p9" name="article1.body1.sec2.p9"></a><p>Subsequently, we confirmed microarray expression data by quantitative real-time PCR (qRT-PCR) (<a href="#pone-0001151-t001">Table 1</a>). This analysis included both genes whose expression in periglomerular cells had been shown as well as genes whose lack of expression had been documented. As expected, among prominently expressed mRNAs were those encoding EGFP and the 5HT3<sub>A</sub> receptor, in accordance with the green fluorescent cells expressing EGFP from the 5HT3<sub>A</sub> gene (<em>Htr3a</em>) promoter. The expression of 5HT3<sub>A</sub> gene was somewhat under-estimated by the microarray data. This is most likely due to the fact that the probe is located further 5′ compared to the qRT-PCR primers. Since material from two rounds of amplification was used, differential detectability by the two approaches can be expected. The migratory neuroblast and GABAergic phenotype of periglomerular cells was reflected in the expression of <em>Dcx</em> <a href="#pone.0001151-Abrous1">[14]</a> (immature neurons) and <em>Gad1</em> <a href="#pone.0001151-Esclapez1">[15]</a> (GAD67 protein in GABAergic interneurons), respectively. Conversely, transcription levels of genes specific for neighboring regions or cell types such as oligodendrocytes (<em>Mog</em> <a href="#pone.0001151-Jalabi1">[16]</a>), olfactory sensory cells (<em>Omp</em> <a href="#pone.0001151-Yu1">[17]</a>), mitral cells (<em>Csf1</em> <a href="#pone.0001151-Murase1">[18]</a>), external plexiform layer (<em>Gria3</em>, GluR3 protein <a href="#pone.0001151-Keinanen1">[19]</a>) and mature neurons (<em>Syn1</em> <a href="#pone.0001151-Mason1">[20]</a>) were low.</p>
<a id="article1.body1.sec2.p10" name="article1.body1.sec2.p10"></a><p>Estimating that β-actin mRNA occurs in 500–2,000 copies per cell, we suggest that these negative markers are represented in our samples by at most 1 copy for every 10 cells, because their amplification required 11–18 cycles more than β-actin mRNA. Hence, it can be inferred that there was only a small percentage of non-periglomerular cells among the 3,000–5,000 dissected cells, which demonstrates the remarkable sensitivity of our technique.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We developed a technique of high quality RNA isolation from <em>in vivo</em> fluorescent cells that is useful for gene expression pattern analysis of defined cell populations (<a href="#pone-0001151-g004">Fig. 4</a>). We applied this technique for a specific interneuronal cell type–periglomerular neurons–of mouse brain, the most complex mouse tissue, employing a transgenic mouse line expressing EGFP from the promoter for the serotonergic receptor 5HT3<sub>A</sub>. Using this technique we obtained periglomerular cell RNA of excellent quality and confirmed its cell type specificity. We amplified isolated RNA and showed that it could be used in gene expression studies with qRT-PCR and/or microarrays.</p>
<div class="figure" id="pone-0001151-g004"><div class="img"><a name="pone-0001151-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001151.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001151.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Flow diagram of the whole technique described.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Asterisk–perfusion conditions for P15 mouse.</p>
<span>doi:10.1371/journal.pone.0001151.g004</span></div><a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>To date there have been only few attempts to separate <em>in vivo</em> fluorescent cells from complex tissue using LMD. One reported approach is based on alcohol fixed slices that enabled the identification and microdissection of fluorescent cells <a href="#pone.0001151-Rossner1">[21]</a>. However, the success of these experiments required the additional generation of transgenic mice expressing the nuclear version of the fluorescent protein. This was necessary in order to preserve the fluorescence in the cells and to avoid leaching and diffusion of the fluorescent marker. Thus, although it was possible in the study to obtain specific and high quality mRNA from fluorescent cells, this approach only rarely will be applicable since in most transgenic mouse lines generated so far the cytosolic form of the fluorescent protein has been used. In other report authors used 4% PFA to preserve fluorescence <a href="#pone.0001151-Bhattacherjee1">[22]</a>. Although they were able to fix fluorescence in the cells, they did not determine whether the quality and yield of RNA isolated from the fixed tissue was appropriate for gene expression analysis. As reported by other labs <a href="#pone.0001151-Goldsworthy1">[23]</a>–<a href="#pone.0001151-Fink2">[27]</a>, and in our hands too, it was not possible to obtain RNA of good quality from fixed tissue using the conventional guanidine isothiocyanate method or the standard proteinase K method. The main problems concerning RNA isolation from aldehyde-fixed tissues are RNA modifications and RNA trapping <a href="#pone.0001151-Jackson1">[12]</a>. RNA modifications could be resolved by SDS and acid phenol treatment <a href="#pone.0001151-Jackson1">[12]</a>, <a href="#pone.0001151-Orlando1">[28]</a>, which revert methylene crosslinks formed by paraformaldehyde. Furthermore, any hydration of methylene bridges in RNA-protein complexes will set RNA free due to the preference of SDS to bind to the protein part of methylene crosslinks and the repulsive character of the bulky SDS-protein compound <a href="#pone.0001151-Jackson1">[12]</a>. It is very important to use acid phenol, as in addition to reversing crosslinks, it also effectively deprotonates methylene links due to low pH. We also overcame the problem of RNA trapping. Aldehyde fixatives induce the formation of extensive inter- and intra-molecular networks that result in RNA trapping and low RNA yield, especially for high molecular weight RNAs. As a result, RNA of shorter size would be isolated preferentially over longer RNA, which would compromise the subsequent gene expression analysis. This problem was resolved using excessive concentrations of proteinase K and SDS in the lysis solution and an optimized buffer for proteinase K digestion. We also optimized the lysis solution for RNase inactivation.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Using an optimized proteinase K/acid phenol procedure for RNA isolation, we obtained RNA from fixed tissue with the quality of unfixed frozen tissue RNA and having a 28S:18S rRNA ratio of ~2:1. We showed that the RNA amplification profile of RNA isolated by our method resembled that of Ambion Control RNA, and hence, the representation of different sequence species in the mRNA population appears not to change during amplification. We were able to obtain sufficient RNA for several microarray hybridizations. Comparing three independent experiments, we provide evidence for excellent reproducibility and hence the reliability of the microarray data (<a href="#pone-0001151-g003">Fig. 3</a>, full analysis data can be found on <a href="http://www.ebi.ac.uk/miamexpress">http://www.ebi.ac.uk/miamexpress</a>, accession number is E-MEXP-1234, public access is available from the 10.11.2007). Furthermore, we demonstrated the specificity of the amplified RNA pool by analyzing expression levels of different positive and negative gene markers using microarray hybridization and qRT-PCR. Positive markers were genes known to be expressed at high levels in periglomerular cells <a href="#pone.0001151-Abrous1">[14]</a>, whereas negative markers were genes that were shown to be expressed in surrounding cell populations but not in the cell type analyzed here. Thus, we validated both usefulness and specificity of our technique for global gene expression analysis of <em>in vivo</em> fluorescent neural cells</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Fluorescent Activated Cell Sorting (FACS) appears to be a viable alternative, as it allows the isolation of small subpopulations of <em>in vivo</em> fluorescent cells with excellent specificity <a href="#pone.0001151-Herzenberg1">[29]</a>. However, during the pre-sorting and sorting steps, cells are incubated in a non-natural microenvironment for several hours. Moreover, sorting requires tissue homogenization by proteases for a prolonged time (e. g. 45 min incubation at 37°C to homogenize striata <a href="#pone.0001151-Lobo1">[30]</a>). All these steps could dramatically change gene expression patterns, especially during development, when cells undergo rapid changes in gene expression. The technique described here employs rapid tissue fixation by PFA and hence, gene expression remain unchanged. Furthermore, this approach guarantees a better cell resolution than that obtained by FACS, which is an important consideration, especially, when separating fluorescent cell populations that are located in close proximity but may have different functions and hence different gene expression.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Indeed, our technique should prove of considerable advantage for the analysis of gene expression of specific cell populations in the many existing transgenic mouse lines having genes tagged by fluorescent protein expression. For example, about 450 transgenic mouse lines expressing EGFP from specific promoters have been generated (80 new lines are under investigation <a href="#pone.0001151-Heintz1">[8]</a>) by the Gene Expression Nervous System ATlas Program (GENSAT) to date and by several other research labs (e.g. <a href="#pone.0001151-Feng1">[9]</a>). In addition to transgenic animals in which a fluorescent protein is expressed from a specific promoter, other methods exist for fluorescence delivery to specific cell types/populations. The most prominent one is viral delivery of a specific construct, which is expressed only in particular cells <a href="#pone.0001151-Lois1">[11]</a>. Others include naked DNA delivery <a href="#pone.0001151-Morris1">[10]</a> as well as specific protein labeling <a href="#pone.0001151-Marks1">[31]</a>. Such a diversity of <em>in vivo</em> fluorescent labeling of specific cells makes the technique described here both useful and promising for the analysis of gene expression pattern of distinct cell types/populations.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Brain fixation for Laser Microdissection (LMD)</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Transgenic 5HT3<sub>A</sub>-EGFP mice (P15) received an intraperitoneal injection of anesthesia (ketanest 18 mg/ml, xylasin 0.24%, final concentration) and were transcardially perfused by 1xPBS for 20 sec (8 ml/min), 0.5% or 2% paraformaldehyde (PFA) for 10 min (8 ml/min) and then by 20% sucrose for 7 min (8 ml/min). After perfusion, the brain was rapidly removed from the skull and frozen on dry ice. Brains were stored at −80°C.</p>


<h4>Brain fixation-standard protocol</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Transgenic 5HT3<sub>A</sub>-EGFP mice (P15) received an intraperitoneal injection of anesthesia (ketanest 18 mg/ml, xylasin 0.24%, final concentration) and were transcardially perfused by 1xPBS for 20 sec (8 ml/min) and 4% PFA for 20 min (8 ml/min). After perfusion, the brains were fixed in 4% PFA overnight and then washed by 1xPBS. Brains were stored at 4°C.</p>


<h4>Preparation of sections to visualize <em>in vivo</em> fluorescence using the standard protocol</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>75 µm-thick sagittal brain sections were made from transgenic 5HT3<sub>A</sub>-EGFP mice (P15) using a vibratome (Leica VT1000S). Sections were mounted onto slides with Moviol (Roth, Germany) and were subsequently analyzed using an upright fluorescent microscope (Zeiss Axioplan 2).</p>


<h4>Preparation of sections for LMD</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Frozen brains were embedded in Tissue Freezing Medium (Leica Instruments, Germany) at −20°C, and 6–8 µm-thick sagittal brain sections were made using the vibratome Microm HM500 (MICROM International, Germany). Sections were mounted on membrane polyester slides (Leica Microsystems, Germany), briefly thawed and dehydrated by sequential incubation in 50% ethanol for 20 sec and n-butanol:ethanol (25:1) for 90 sec, followed by 60 sec of xylene substitution (Sigma-Aldrich, Germany) clearing, to which 1/25 volume of n-butanol was added. Sections were dried for 5 min and used for LMD.</p>


<h4>Laser Microdissection</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>LMD was performed on a Leica LMD6000 B microscope (Leica Microsystems, Germany). Approximately 3,000–5,000 cells were dissected from 50–70 sagittal brain sections of 6–8 µm from one transgenic 5HT3<sub>A</sub>-EGFP mouse within 4–5 hr. Cells were dissected into dry 0.2 ml tube caps coated with silicon (Leica Microsystems, Germany) at the power 55–57 and speed 3 using a 40x objective or at power 53–54 and speed 3 using a 63x objective. To facilitate collection of such a high cell number, ensembles of adjacent fluorescent cells were often co-dissected during the same laser movement. Three mice were used to assess reproducibility.</p>


<h4>RNA isolation</h4>
<h5>Optimized proteinase K/acid phenol method.</h5><a id="article1.body1.sec4.sec6.sec1.p1" name="article1.body1.sec4.sec6.sec1.p1"></a><p>Directly following microdissection, the collected 3,000–5,000 cells were lysed in 100 µl of lysis solution [10 mM Tris-HCl (pH 7.9), 50 mM EDTA (pH 7.9), 0.2 M NaCl, 2.2% SDS, 0.5 µg/µl AntiRNase (Ambion, USA) and 1000 µg/ml proteinase K (Ambion, USA)] at 55°C for 3h with vigorous shaking. The volume was adjusted to 600 µl by water followed by adding an equal volume of phenol, pH 4.2. The solution was vigorously mixed during 2 min, left for 5 min on ice and centrifuged at 14,000 g for 10 min at 4°C. The aqueous phase was aspirated into a fresh tube and subjected to equal volume of phenol:chloroform (1:1) treatment. The mixture was vortexed during 2 min, left for 5 min and centrifuged at 14,000 g for 10 min at 4°C. Again, the aqueous phase was transferred into a fresh tube and mixed with an equal volume of isopropanol and 20 µg of glycogen (Ambion, USA). The mixture was incubated at –20°C for 30 min and centrifuged at 14,000 g for 10 min at 4°C. The pellet was washed with 600 µl of cold 70% ethanol, air-dried and dissolved in 26 µl of water and 3 µl of 10x DNase buffer (Ambion, USA). To eliminate any remaining DNA, the mixture was treated by 1 U of DNase I (Ambion, USA) for 15 min at 37°C and purified by use of RNeasy MinElute Cleanup Kit (QIAGEN, Germany) according to manufacturer's recommendations, except for using RPE solution (included in the Cleanup Kit) instead of 80% ethanol in the second washing step and performing the elution step twice, each with 15 µl of water. Obtained RNA (typically 6–9 ng) was concentrated by Eppendorf Concentrator 5301 (Eppendorf, Germany) and analyzed by Bioanalyzer 2100 (Agilent, USA).</p>

<h5>Conventional methods.</h5><a id="article1.body1.sec4.sec6.sec2.p1" name="article1.body1.sec4.sec6.sec2.p1"></a><p>RNA was isolated either by the TRIzol (Invitrogen, Germany) method or by use of an RNeasy Micro Kit (QIAGEN, Germany) according to manufacturers' recommendations.</p>



<h4>RNA amplification</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Total RNA (2–3 ng) was amplified using the MessageAmp II aRNA Amplification Kit (Ambion, USA) according to manufacturer's recommendations. During the T7 <em>in vitro</em> transcription step, the mixture was incubated at 37°C for 14–16h. After each amplification round, the RNA was analyzed by Bioanalyzer 2100 (Agilent, USA). We typically obtained 100–200 ng and 50–100 µg of amplified RNA after the first and second amplification round, respectively. Amplifications were done from three periglomerular cell RNA samples obtained from three 5HT3<sub>A</sub>-EGFP mice. For microarray hybridization, a second round RNA amplification was performed using biotinylated nucleotides.</p>


<h4>Microarray hybridization</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Microarray hybridization was performed according to the Affymetrix GeneChip Expression Analysis Technical Manual (<a href="http://www.affymetrix.com">www.affymetrix.com</a>). For 3 samples of periglomerular cell RNA, 10 µg of biotinylated cRNA was hybridized onto an Affymetrix Gene Chip Mouse Genome 430A 2.0 arrays. Arrays were scanned with the Affymetrix GeneArray 2500 scanner. Gene expression data were obtained using the Affymetrix software.</p>


<h4>Microarray data analysis</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Microarray data analysis was carried out using R packages gcrma, affy and genefilter of Bioconducter project <a href="#pone.0001151-Gentleman1">[32]</a>. The genes that had 100% P-calls and signal detection value of more than 100 in all three arrays were filtered out from the raw hybridization data.</p>


<h4>cDNA synthesis and quantitative real-time PCR (qRT-PCR)</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>cDNA was synthesized from 2 µg of amplified RNA by High Capacity cDNA Reverse Transcription Kit (Applied Biosystems, Germany). QRT-PCR was performed on a TaqMan ABI Prism 7000 Sequence detection system (Applied Biosystems, Germany) using Power SYBR Green PCR Master Mix (Applied Biosystems, Germany) with the following parameters: 50°C–2 min, 95°C–10 min followed by 45 cycles of 95°C–15 sec and 60°C–1 min. Sequences of primers for real-time PCR are listed in the <a href="#pone-0001151-t002">Table 2</a>. MRNA levels detected by qRT-PCR were normalized to mRNA levels for β-actin, a well-known house-keeping gene. Quantifications were made using the relative standard curve method with comparison to Stratagene QPCR Mouse total RNA (Stratagene Europe, Netherlands). To create the standard curves, a dilution series across 5 orders of magnitude of cDNA concentration, generated from Stratagene QPCR Mouse total RNA, was prepared for each experiment. Each sample was amplified in triplicate.</p>
<div class="figure" id="pone-0001151-t002"><div class="img"><a name="pone-0001151-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001151" data-uri="info:doi/10.1371/journal.pone.0001151.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001151.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001151.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001151.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001151.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001151.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Primers sequences used for qRT-PCR analysis</span></strong></p><span>doi:10.1371/journal.pone.0001151.t002</span></div>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Prof. H.-J. Grone and Dr. M. Kenzelmann for letting us to use the Bioanalyzer and Prof. R. Bartenschlager for the use of the TaqMan ABI Prism 7000 Sequence detection system. We also thank Dr. Stefan Bonn for microarray data analysis and Drs. M. Eliava and V. Grinevich for advices and help to optimize tissue processing.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: HM KK. Performed the experiments: KK. Analyzed the data: KK. Contributed reagents/materials/analysis tools: HM PS KK DI. Wrote the paper: HM PS KK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001151-EmmertBuck1" id="pone.0001151-EmmertBuck1"></a>Emmert-Buck MR, Bonner RF, Smith PD, Chuaqui RF, Zhuang Z, et al.  (1996) Laser capture microdissection. Science  274: 998–1001.  <ul class="find" data-citedArticleID="1015260" data-doi="10.1126/science.274.5289.998"><li><a href="http://dx.doi.org/10.1126/science.274.5289.998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Laser+capture+microdissection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Laser+capture+microdissection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001151-Jin1" id="pone.0001151-Jin1"></a>Jin L, Thompson CA, Qian X, Kuecker SJ, Kulig E, et al.  (1999) Analysis of anterior pituitary hormone mRNA expression in immunophenotypically characterized single cells after laser capture microdissection. Lab Invest  79: 511–512.  <ul class="find" data-citedArticleID="1015284"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+anterior+pituitary+hormone+mRNA+expression+in+immunophenotypically+characterized+single+cells+after+laser+capture+microdissection.&amp;auth=&amp;atitle=Analysis+of+anterior+pituitary+hormone+mRNA+expression+in+immunophenotypically+characterized+single+cells+after+laser+capture+microdissection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+anterior+pituitary+hormone+mRNA+expression+in+immunophenotypically+characterized+single+cells+after+laser+capture+microdissection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+anterior+pituitary+hormone+mRNA+expression+in+immunophenotypically+characterized+single+cells+after+laser+capture+microdissection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001151-Okuducu1" id="pone.0001151-Okuducu1"></a>Okuducu AF, Janzen V, Hahne JC, Ko Y, Wernert N (2003) Influence of histochemical stains on quantitative gene expression analysis after laser-assisted microdissection. Int J Mol Med  11: 449–453.  <ul class="find" data-citedArticleID="1015306" data-doi="10.3892/ijmm.11.4.449"><li><a href="http://dx.doi.org/10.3892/ijmm.11.4.449" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+histochemical+stains+on+quantitative+gene+expression+analysis+after+laser-assisted+microdissection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+histochemical+stains+on+quantitative+gene+expression+analysis+after+laser-assisted+microdissection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001151-Fink1" id="pone.0001151-Fink1"></a>Fink L, Kinfe T, Stein MM, Ermert L, Hanze J, et al.  (2000) Immunostaining and laser-assisted cell picking for mRNA analysis. Lab Invest  80: 327–333.  <ul class="find" data-citedArticleID="1015266" data-doi="10.1038/labinvest.3780037"><li><a href="http://dx.doi.org/10.1038/labinvest.3780037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunostaining+and+laser-assisted+cell+picking+for+mRNA+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunostaining+and+laser-assisted+cell+picking+for+mRNA+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001151-Murakami1" id="pone.0001151-Murakami1"></a>Murakami H, Liotta L, Star RA (2000) IF-LCM: laser capture microdissection of immunofluorescently defined cells for mRNA analysis rapid communication. Kidney Int  58: 1346–1353.  <ul class="find" data-citedArticleID="1015302" data-doi="10.1046/j.1523-1755.2000.00295.x"><li><a href="http://dx.doi.org/10.1046/j.1523-1755.2000.00295.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=IF-LCM%3A+laser+capture+microdissection+of+immunofluorescently+defined+cells+for+mRNA+analysis+rapid+communication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22IF-LCM%3A+laser+capture+microdissection+of+immunofluorescently+defined+cells+for+mRNA+analysis+rapid+communication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001151-Gjerdrum1" id="pone.0001151-Gjerdrum1"></a>Gjerdrum LM, Abrahamsen HN, Villegas B, Sorensen BS, Schmidt H, et al.  (2004) The influence of immunohistochemistry on mRNA recovery from microdissected frozen and formalin-fixed, paraffin-embedded sections. Diagn Mol Pathol  13: 224–233.  <ul class="find" data-citedArticleID="1015272" data-doi="10.1097/01.pdm.0000134779.45353.d6"><li><a href="http://dx.doi.org/10.1097/01.pdm.0000134779.45353.d6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+influence+of+immunohistochemistry+on+mRNA+recovery+from+microdissected+frozen+and+formalin-fixed%2C+paraffin-embedded+sections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+influence+of+immunohistochemistry+on+mRNA+recovery+from+microdissected+frozen+and+formalin-fixed%2C+paraffin-embedded+sections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001151-Wang1" id="pone.0001151-Wang1"></a>Wang H, Owens JD, Shih JH, Li MC, Bonner RF, et al.  (2006) Histological staining methods preparatory to laser capture microdissection significantly affect the integrity of the cellular RNA. BMC Genomics  7: 97.  <ul class="find" data-citedArticleID="1015314"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Histological+staining+methods+preparatory+to+laser+capture+microdissection+significantly+affect+the+integrity+of+the+cellular+RNA.&amp;auth=&amp;atitle=Histological+staining+methods+preparatory+to+laser+capture+microdissection+significantly+affect+the+integrity+of+the+cellular+RNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histological+staining+methods+preparatory+to+laser+capture+microdissection+significantly+affect+the+integrity+of+the+cellular+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histological+staining+methods+preparatory+to+laser+capture+microdissection+significantly+affect+the+integrity+of+the+cellular+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001151-Heintz1" id="pone.0001151-Heintz1"></a>Heintz N (2004) Gene expression nervous system atlas (GENSAT). Nat Neurosci  7: 483.  <ul class="find" data-citedArticleID="1015276" data-doi="10.1038/nn0504-483"><li><a href="http://dx.doi.org/10.1038/nn0504-483" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+nervous+system+atlas+%28GENSAT%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+nervous+system+atlas+%28GENSAT%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001151-Feng1" id="pone.0001151-Feng1"></a>Feng G, Mellor RH, Bernstein M, Keller-Peck C, Nguyen QT, et al.  (2000) Imaging neuronal subsets in transgenic mice expressing multiple spectral variants of GFP. Neuron  28: 41–51.  <ul class="find" data-citedArticleID="1015264" data-doi="10.1016/s0896-6273(00)00084-2"><li><a href="http://dx.doi.org/10.1016/s0896-6273(00)00084-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Imaging+neuronal+subsets+in+transgenic+mice+expressing+multiple+spectral+variants+of+GFP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Imaging+neuronal+subsets+in+transgenic+mice+expressing+multiple+spectral+variants+of+GFP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001151-Morris1" id="pone.0001151-Morris1"></a>Morris R, Morgan BS, Lewis TM, Pierce KD, Pisano A, et al.  (2004) In vivo somatic delivery of plasmid DNA and retrograde transport to obtain cell-specific gene expression in the central nervous system. J Neurochem  90: 1445–1452.  <ul class="find" data-citedArticleID="1015300" data-doi="10.1111/j.1471-4159.2004.02612.x"><li><a href="http://dx.doi.org/10.1111/j.1471-4159.2004.02612.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+somatic+delivery+of+plasmid+DNA+and+retrograde+transport+to+obtain+cell-specific+gene+expression+in+the+central+nervous+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+somatic+delivery+of+plasmid+DNA+and+retrograde+transport+to+obtain+cell-specific+gene+expression+in+the+central+nervous+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001151-Lois1" id="pone.0001151-Lois1"></a>Lois C, Hong EJ, Pease S, Brown EJ, Baltimore D (2002) Germline transmission and tissue-specific expression of transgenes delivered by lentiviral vectors. Science  295: 868–872.  <ul class="find" data-citedArticleID="1015292" data-doi="10.1126/science.1067081"><li><a href="http://dx.doi.org/10.1126/science.1067081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Germline+transmission+and+tissue-specific+expression+of+transgenes+delivered+by+lentiviral+vectors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Germline+transmission+and+tissue-specific+expression+of+transgenes+delivered+by+lentiviral+vectors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001151-Jackson1" id="pone.0001151-Jackson1"></a>Jackson V (1999) Formaldehyde cross-linking for studying nucleosomal dynamics. Methods  17: 125–139.  <ul class="find" data-citedArticleID="1015280" data-doi="10.1006/meth.1998.0724"><li><a href="http://dx.doi.org/10.1006/meth.1998.0724" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Formaldehyde+cross-linking+for+studying+nucleosomal+dynamics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Formaldehyde+cross-linking+for+studying+nucleosomal+dynamics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001151-Luzzi1" id="pone.0001151-Luzzi1"></a>Luzzi VI, Holtschlag V, Watson MA (2005) Gene expression profiling of primary tumor cell populations using laser capture microdissection, RNA transcript amplification, and GeneChip microarrays. Methods Mol Biol  293: 187–207.  <ul class="find" data-citedArticleID="1015294" data-doi="10.1385/1-59259-853-6:187"><li><a href="http://dx.doi.org/10.1385/1-59259-853-6:187" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+profiling+of+primary+tumor+cell+populations+using+laser+capture+microdissection%2C+RNA+transcript+amplification%2C+and+GeneChip+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+profiling+of+primary+tumor+cell+populations+using+laser+capture+microdissection%2C+RNA+transcript+amplification%2C+and+GeneChip+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001151-Abrous1" id="pone.0001151-Abrous1"></a>Abrous DN, Koehl M, Le Moal M (2005) Adult neurogenesis: from precursors to network and physiology. Physiol Rev  85: 523–569.  <ul class="find" data-citedArticleID="1015254" data-doi="10.1152/physrev.00055.2003"><li><a href="http://dx.doi.org/10.1152/physrev.00055.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adult+neurogenesis%3A+from+precursors+to+network+and+physiology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adult+neurogenesis%3A+from+precursors+to+network+and+physiology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001151-Esclapez1" id="pone.0001151-Esclapez1"></a>Esclapez M, Tillakaratne NJ, Kaufman DL, Tobin AJ, Houser CR (1994) Comparative localization of two forms of glutamic acid decarboxylase and their mRNAs in rat brain supports the concept of functional differences between the forms. J Neurosci  14: 1834–1855.  <ul class="find" data-citedArticleID="1015262"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+localization+of+two+forms+of+glutamic+acid+decarboxylase+and+their+mRNAs+in+rat+brain+supports+the+concept+of+functional+differences+between+the+forms.&amp;auth=&amp;atitle=Comparative+localization+of+two+forms+of+glutamic+acid+decarboxylase+and+their+mRNAs+in+rat+brain+supports+the+concept+of+functional+differences+between+the+forms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+localization+of+two+forms+of+glutamic+acid+decarboxylase+and+their+mRNAs+in+rat+brain+supports+the+concept+of+functional+differences+between+the+forms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+localization+of+two+forms+of+glutamic+acid+decarboxylase+and+their+mRNAs+in+rat+brain+supports+the+concept+of+functional+differences+between+the+forms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001151-Jalabi1" id="pone.0001151-Jalabi1"></a>Jalabi W, Cerghet M, Skoff RP, Ghandour MS (2003) Detection of oligodendrocytes in tissue sections using PCR synthesis of digoxigenin-labeled probes. J Histochem Cytochem  51: 913–919.  <ul class="find" data-citedArticleID="1015282" data-doi="10.1177/002215540305100706"><li><a href="http://dx.doi.org/10.1177/002215540305100706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+oligodendrocytes+in+tissue+sections+using+PCR+synthesis+of+digoxigenin-labeled+probes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+oligodendrocytes+in+tissue+sections+using+PCR+synthesis+of+digoxigenin-labeled+probes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001151-Yu1" id="pone.0001151-Yu1"></a>Yu TT, McIntyre JC, Bose SC, Hardin D, Owen MC, et al.  (2005) Differentially expressed transcripts from phenotypically identified olfactory sensory neurons. J Comp Neurol  483: 251–262.  <ul class="find" data-citedArticleID="1015316" data-doi="10.1002/cne.20429"><li><a href="http://dx.doi.org/10.1002/cne.20429" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differentially+expressed+transcripts+from+phenotypically+identified+olfactory+sensory+neurons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differentially+expressed+transcripts+from+phenotypically+identified+olfactory+sensory+neurons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001151-Murase1" id="pone.0001151-Murase1"></a>Murase S, Hayashi Y (2002) Neuronal expression of macrophage colony stimulating factor in Purkinje cells and olfactory mitral cells of wild-type and cerebellar-mutant mice. Histochem J  34: 85–95.  <ul class="find" data-citedArticleID="1015304"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Neuronal+expression+of+macrophage+colony+stimulating+factor+in+Purkinje+cells+and+olfactory+mitral+cells+of+wild-type+and+cerebellar-mutant+mice.&amp;auth=&amp;atitle=Neuronal+expression+of+macrophage+colony+stimulating+factor+in+Purkinje+cells+and+olfactory+mitral+cells+of+wild-type+and+cerebellar-mutant+mice." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neuronal+expression+of+macrophage+colony+stimulating+factor+in+Purkinje+cells+and+olfactory+mitral+cells+of+wild-type+and+cerebellar-mutant+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neuronal+expression+of+macrophage+colony+stimulating+factor+in+Purkinje+cells+and+olfactory+mitral+cells+of+wild-type+and+cerebellar-mutant+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001151-Keinanen1" id="pone.0001151-Keinanen1"></a>Keinanen K, Wisden W, Sommer B, Werner P, Herb A, et al.  (1990) A family of AMPA-selective glutamate receptors. Science  249: 556–560.  <ul class="find" data-citedArticleID="1015286" data-doi="10.1126/science.2166337"><li><a href="http://dx.doi.org/10.1126/science.2166337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+family+of+AMPA-selective+glutamate+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+family+of+AMPA-selective+glutamate+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001151-Mason1" id="pone.0001151-Mason1"></a>Mason CA (1986) Axon development in mouse cerebellum: embryonic axon forms and expression of synapsin I. Neuroscience  19: 1319–1333.  <ul class="find" data-citedArticleID="1015298" data-doi="10.1016/0306-4522(86)90146-6"><li><a href="http://dx.doi.org/10.1016/0306-4522(86)90146-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Axon+development+in+mouse+cerebellum%3A+embryonic+axon+forms+and+expression+of+synapsin+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Axon+development+in+mouse+cerebellum%3A+embryonic+axon+forms+and+expression+of+synapsin+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001151-Rossner1" id="pone.0001151-Rossner1"></a>Rossner MJ, Hirrlinger J, Wichert SP, Boehm C, Newrzella D, et al.  (2006) Global transcriptome analysis of genetically identified neurons in the adult cortex. J Neurosci  26: 9956–9966.  <ul class="find" data-citedArticleID="1015312" data-doi="10.1523/jneurosci.0468-06.2006"><li><a href="http://dx.doi.org/10.1523/jneurosci.0468-06.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+transcriptome+analysis+of+genetically+identified+neurons+in+the+adult+cortex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+transcriptome+analysis+of+genetically+identified+neurons+in+the+adult+cortex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001151-Bhattacherjee1" id="pone.0001151-Bhattacherjee1"></a>Bhattacherjee V, Mukhopadhyay P, Singh S, Roberts EA, Hackmiller RC, et al.  (2004) Laser capture microdissection of fluorescently labeled embryonic cranial neural crest cells. Genesis  39: 58–64.  <ul class="find" data-citedArticleID="1015256" data-doi="10.1002/gene.20026"><li><a href="http://dx.doi.org/10.1002/gene.20026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Laser+capture+microdissection+of+fluorescently+labeled+embryonic+cranial+neural+crest+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Laser+capture+microdissection+of+fluorescently+labeled+embryonic+cranial+neural+crest+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001151-Goldsworthy1" id="pone.0001151-Goldsworthy1"></a>Goldsworthy SM, Stockton PS, Trempus CS, Foley JF, Maronpot RR (1999) Effects of fixation on RNA extraction and amplification from laser capture microdissected tissue. Mol Carcinog  25: 86–91.  <ul class="find" data-citedArticleID="1015274" data-doi="10.1002/(sici)1098-2744(199906)25:2<86::aid-mc2&gt;3.0.co;2-4"><li><a href="http://dx.doi.org/10.1002/(sici)1098-2744(199906)25:2<86::aid-mc2&gt;3.0.co;2-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+fixation+on+RNA+extraction+and+amplification+from+laser+capture+microdissected+tissue." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+fixation+on+RNA+extraction+and+amplification+from+laser+capture+microdissected+tissue.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001151-Cox1" id="pone.0001151-Cox1"></a>Cox ML, Schray CL, Luster CN, Stewart ZS, Korytko PJ, et al.  (2006) Assessment of fixatives, fixation, and tissue processing on morphology and RNA integrity. Exp Mol Pathol  80: 183–191.  <ul class="find" data-citedArticleID="1015258" data-doi="10.1016/j.yexmp.2005.10.002"><li><a href="http://dx.doi.org/10.1016/j.yexmp.2005.10.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessment+of+fixatives%2C+fixation%2C+and+tissue+processing+on+morphology+and+RNA+integrity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessment+of+fixatives%2C+fixation%2C+and+tissue+processing+on+morphology+and+RNA+integrity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001151-Kihara1" id="pone.0001151-Kihara1"></a>Kihara AH, Moriscot AS, Ferreira PJ, Hamassaki DE (2005) Protecting RNA in fixed tissue: an alternative method for LCM users. J Neurosci Methods  148: 103–107.  <ul class="find" data-citedArticleID="1015288" data-doi="10.1016/j.jneumeth.2005.04.019"><li><a href="http://dx.doi.org/10.1016/j.jneumeth.2005.04.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protecting+RNA+in+fixed+tissue%3A+an+alternative+method+for+LCM+users." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protecting+RNA+in+fixed+tissue%3A+an+alternative+method+for+LCM+users.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001151-Qin1" id="pone.0001151-Qin1"></a>Qin Y, Heine VM, Karst H, Lucassen PJ, Joels M (2003) Gene expression patterns in rat dentate granule cells: comparison between fresh and fixed tissue. J Neurosci Methods  131: 205–211.  <ul class="find" data-citedArticleID="1015310" data-doi="10.1016/j.jneumeth.2003.08.010"><li><a href="http://dx.doi.org/10.1016/j.jneumeth.2003.08.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+patterns+in+rat+dentate+granule+cells%3A+comparison+between+fresh+and+fixed+tissue." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+patterns+in+rat+dentate+granule+cells%3A+comparison+between+fresh+and+fixed+tissue.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001151-Fink2" id="pone.0001151-Fink2"></a>Fink L, Bohle RM (2005) Laser microdissection and RNA analysis. Methods Mol Biol  293: 167–185.  <ul class="find" data-citedArticleID="1015268" data-doi="10.1385/1-59259-853-6:167"><li><a href="http://dx.doi.org/10.1385/1-59259-853-6:167" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Laser+microdissection+and+RNA+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Laser+microdissection+and+RNA+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001151-Orlando1" id="pone.0001151-Orlando1"></a>Orlando V, Strutt H, Paro R (1997) Analysis of chromatin structure by in vivo formaldehyde cross-linking. Methods  11: 205–214.  <ul class="find" data-citedArticleID="1015308" data-doi="10.1006/meth.1996.0407"><li><a href="http://dx.doi.org/10.1006/meth.1996.0407" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+chromatin+structure+by+in+vivo+formaldehyde+cross-linking." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+chromatin+structure+by+in+vivo+formaldehyde+cross-linking.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001151-Herzenberg1" id="pone.0001151-Herzenberg1"></a>Herzenberg LA, Parks D, Sahaf B, Perez O, Roederer M (2002) The history and future of the fluorescence activated cell sorter and flow cytometry: a view from Stanford. Clin Chem  48: 1819–1827.  <ul class="find" data-citedArticleID="1015278"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+history+and+future+of+the+fluorescence+activated+cell+sorter+and+flow+cytometry%3A+a+view+from+Stanford.&amp;auth=&amp;atitle=The+history+and+future+of+the+fluorescence+activated+cell+sorter+and+flow+cytometry%3A+a+view+from+Stanford." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+history+and+future+of+the+fluorescence+activated+cell+sorter+and+flow+cytometry%3A+a+view+from+Stanford." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+history+and+future+of+the+fluorescence+activated+cell+sorter+and+flow+cytometry%3A+a+view+from+Stanford.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001151-Lobo1" id="pone.0001151-Lobo1"></a>Lobo MK, Karsten SL, Gray M, Geschwind DH, Yang XW (2006) FACS-array profiling of striatal projection neuron subtypes in juvenile and adult mouse brains. Nat Neurosci  9: 443–452.  <ul class="find" data-citedArticleID="1015290" data-doi="10.1038/nn1654"><li><a href="http://dx.doi.org/10.1038/nn1654" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FACS-array+profiling+of+striatal+projection+neuron+subtypes+in+juvenile+and+adult+mouse+brains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FACS-array+profiling+of+striatal+projection+neuron+subtypes+in+juvenile+and+adult+mouse+brains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001151-Marks1" id="pone.0001151-Marks1"></a>Marks KM, Nolan GP (2006) Chemical labeling strategies for cell biology. Nat Methods  3: 591–596.  <ul class="find" data-citedArticleID="1015296" data-doi="10.1038/nmeth906"><li><a href="http://dx.doi.org/10.1038/nmeth906" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemical+labeling+strategies+for+cell+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemical+labeling+strategies+for+cell+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001151-Gentleman1" id="pone.0001151-Gentleman1"></a>Gentleman RC, Carey VJ, Bates DM, Bolstad B, Dettling M, et al.  (2004) Bioconductor: open software development for computational biology and bioinformatics. Genome Biol  5: R80.  <ul class="find" data-citedArticleID="1015270" data-doi="10.1186/gb-2004-5-10-r80"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-10-r80" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bioconductor%3A+open+software+development+for+computational+biology+and+bioinformatics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bioconductor%3A+open+software+development+for+computational+biology+and+bioinformatics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.XML" value="74277"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.PDF" value="282896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g001.PNG_L" value="4215325"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g001.PNG_M" value="260293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g001.PNG_S" value="14900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g001.TIF" value="5940444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g001.PNG_I" value="88836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g002.PNG_L" value="725169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g002.PNG_M" value="67726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g002.PNG_S" value="9205"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g002.TIF" value="980770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g002.PNG_I" value="23947"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g003.PNG_L" value="1440023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g003.PNG_M" value="78660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g003.PNG_S" value="8744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g003.TIF" value="2253802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g003.PNG_I" value="25694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.t001.PNG_L" value="75292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.t001.PNG_M" value="97257"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.t001.PNG_S" value="14424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.t001.TIF" value="371668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.t001.PNG_I" value="33794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g004.PNG_L" value="134153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g004.PNG_M" value="47590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g004.PNG_S" value="9039"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g004.TIF" value="215728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.g004.PNG_I" value="15085"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.t002.PNG_L" value="111878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.t002.PNG_M" value="149690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.t002.PNG_S" value="18146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.t002.TIF" value="613446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001151.t002.PNG_I" value="68749"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001151&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001151" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001151&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001151&volume=&issue=&title=Gene Expression Analysis of In Vivo Fluorescent Cells&author_name=Konstantin%20Khodosevich%2C%20Dragos%20Inta%2C%20Peter%20H.%20Seeburg%2C%20Hannah%20Monyer&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001151" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001151" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001151" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001151&amp;t=Gene%20Expression%20Analysis%20of%20In%20Vivo%20Fluorescent%20Cells" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001151&title=Gene%20Expression%20Analysis%20of%20In%20Vivo%20Fluorescent%20Cells&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001151&amp;title=Gene%20Expression%20Analysis%20of%20In%20Vivo%20Fluorescent%20Cells" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001151" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001151&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Gene Expression Analysis of In Vivo Fluorescent Cells';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001151';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Gene%20Expression%20Analysis%20of%20In%20Vivo%20Fluorescent%20Cells http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001151" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001151" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25134"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25134"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenols%22" title="Search for articles in the subject area:'Phenols'"><div class="flagText">Phenols</div></a>
              <div data-categoryid="21767" data-articleid="25134"
                   data-categoryname="Phenols"
                   class="flagImage" title="Flag 'Phenols' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="47289" data-articleid="25134"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteases%22" title="Search for articles in the subject area:'Proteases'"><div class="flagText">Proteases</div></a>
              <div data-categoryid="19447" data-articleid="25134"
                   data-categoryname="Proteases"
                   class="flagImage" title="Flag 'Proteases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+amplification%22" title="Search for articles in the subject area:'RNA amplification'"><div class="flagText">RNA amplification</div></a>
              <div data-categoryid="21647" data-articleid="25134"
                   data-categoryname="RNA amplification"
                   class="flagImage" title="Flag 'RNA amplification' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+hybridization%22" title="Search for articles in the subject area:'RNA hybridization'"><div class="flagText">RNA hybridization</div></a>
              <div data-categoryid="19365" data-articleid="25134"
                   data-categoryname="RNA hybridization"
                   class="flagImage" title="Flag 'RNA hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+isolation%22" title="Search for articles in the subject area:'RNA isolation'"><div class="flagText">RNA isolation</div></a>
              <div data-categoryid="47423" data-articleid="25134"
                   data-categoryname="RNA isolation"
                   class="flagImage" title="Flag 'RNA isolation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9676'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1924'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1080&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=525">Additional details and some hints of the protocol</a><br>Posted by hobot</p>
      <p><a href="/annotation/listThread.action?root=13565">Referee Comments: Referee 1 (Robert MacKenzie)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
