

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Extensive Polycistronism and Antisense Transcription in the Mammalian Hox Clusters</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000356"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000356" />

    <meta name="citation_title" content="Extensive Polycistronism and Antisense Transcription in the Mammalian Hox Clusters"/>
    <meta itemprop="name" content="Extensive Polycistronism and Antisense Transcription in the Mammalian Hox Clusters"/>

      <meta name="citation_author" content="Gaëll Mainguy"/>
            <meta name="citation_author_institution" content="Hubrecht Laboratory, Center for Biomedical Genetics, Utrecht, The Netherlands"/>
      <meta name="citation_author" content="Jan Koster"/>
            <meta name="citation_author_institution" content="Department of Human Genetics, Academic Medical Center, University of Amsterdam, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Joost Woltering"/>
            <meta name="citation_author_institution" content="Hubrecht Laboratory, Center for Biomedical Genetics, Utrecht, The Netherlands"/>
      <meta name="citation_author" content="Hans Jansen"/>
            <meta name="citation_author_institution" content="Hubrecht Laboratory, Center for Biomedical Genetics, Utrecht, The Netherlands"/>
      <meta name="citation_author" content="Antony Durston"/>
            <meta name="citation_author_institution" content="Hubrecht Laboratory, Center for Biomedical Genetics, Utrecht, The Netherlands"/>

    <meta name="citation_date" content="2007/4/4"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000356.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e356"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The pathophysiology of HOX genes and their role in cancer.; citation_author=DG Grier; citation_author=A Thompson; citation_author=A Kwasniewska; citation_author=GJ McGonigle; citation_author=HL Halliday; citation_journal_title=J. Pathol; citation_volume=205(2); citation_number=1; citation_pages=154-71; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Vertebrate hox gene regulation: clustering and/or colinearity?; citation_author=D Duboule; citation_journal_title=Curr Opin Genet Dev; citation_volume=8(5); citation_number=2; citation_pages=514-8; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Multiplex detection of RNA expression in Drosophila embryos.; citation_author=D Kosman; citation_author=CM Mizutani; citation_author=D Lemons; citation_author=WG Cox; citation_author=W McGinnis; citation_journal_title=Science; citation_volume=305(5685); citation_number=3; citation_pages=846; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MicroRNA-directed cleavage of HOXB8 mRNA.; citation_author=S Yekta; citation_author=IH Shih; citation_author=DP Bartel; citation_journal_title=Science; citation_volume=304(5670); citation_number=4; citation_pages=594-6; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The regulatory content of intergenic DNA shapes genome architecture.; citation_author=CE Nelson; citation_author=BM Hersh; citation_author=SB Carroll; citation_journal_title=Genome Biol; citation_volume=5(4); citation_number=5; citation_pages=R25; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Why are Hox genes clustered?; citation_author=RS Mann; citation_journal_title=Bioessays; citation_volume=19(8); citation_number=6; citation_pages=661-4; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Positive cross-regulation and enhancer sharing: two mechanisms for specifying overlapping Hox expression patterns.; citation_author=A Gould; citation_author=A Morrison; citation_author=G Sproat; citation_author=RA White; citation_author=R Krumlauf; citation_journal_title=Genes Dev; citation_volume=11(7); citation_number=7; citation_pages=900-13; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=At least three human homeoboxes on chromosome 12 belong to the same transcription unit.; citation_author=A Simeone; citation_author=M Pannese; citation_author=D Acampora; citation_author=M D'Esposito; citation_author=E Boncinelli; citation_journal_title=Nucleic Acids Res; citation_volume=16(12); citation_number=8; citation_pages=5379-90; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Transcriptional readthrough of Hox genes Ubx and Antp and their divergent post-transcriptional control during crustacean evolution. Evol Dev.; citation_author=Y Shiga; citation_author=K Sagawa; citation_author=R Takai; citation_author=H Sakaguchi; citation_author=H Yamagata; citation_journal_title=Sep–Oct;; citation_volume=8(5); citation_number=9; citation_pages=407-14; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Hoxa 11 structure, extensive antisense transcription, and function in male and female fertility.; citation_author=AP Hsieh-Li HM Davis; citation_author=DP Witte; citation_author=SS Potter; citation_author=MR Capecchi; citation_journal_title=Development; citation_volume=121(5); citation_number=10; citation_pages=1373-85; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Complex Loci in Human and Mouse Genomes.; citation_author=PG Engstrom; citation_author=H Suzuki; citation_author=N Ninomiya; citation_author=A Akalin; citation_author=L Sessa; citation_journal_title=PLoS Genetics.; citation_volume=2(4); citation_number=11; citation_pages=e47; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Analysis of the mouse transcriptome based on functional annotation of 60,770 full-length cDNAs.; citation_author=Y Okazaki; citation_author=M Furuno; citation_author=T Kasukawa; citation_author=J Adachi; citation_author=H Bono; citation_journal_title=Nature; citation_volume=420(6915); citation_number=12; citation_pages=563-73; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Using the transcriptome to annotate the genome.; citation_author=S Saha; citation_author=AB Sparks; citation_author=C Rago; citation_author=V Akmaev; citation_author=CJ Wang; citation_journal_title=Nat Biotechnol; citation_volume=20(5); citation_number=13; citation_pages=508-12; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The transcriptional activity of human Chromosome 22.; citation_author=JL Rinn; citation_author=G Euskirchen; citation_author=P Bertone; citation_author=R Martone; citation_author=NM Luscombe; citation_journal_title=Genes Dev; citation_volume=17(4); citation_number=14; citation_pages=529-40; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=An overview of the kallikrein gene families in humans and other species: emerging candidate tumour markers.; citation_author=GM Yousef; citation_author=EP Diamandis; citation_journal_title=Clin Biochem; citation_volume=36(6); citation_number=15; citation_pages=443-52; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Nongenic transcription, gene regulation and action at a distance.; citation_author=PR Cook; citation_journal_title=J Cell Sci; citation_volume=116(Pt 22); citation_number=16; citation_pages=4483-91; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Conservation of human alternative splice events in mouse.; citation_author=TA Thanaraj; citation_author=F Clark; citation_author=J Muilu; citation_journal_title=Nucleic Acids Res; citation_volume=31(10); citation_number=17; citation_pages=2544-52; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene clusters and polycistronic transcription in eukaryotes.; citation_author=T Blumenthal; citation_journal_title=Bioessays; citation_volume=20(6); citation_number=18; citation_pages=480-7; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=MicroRNA-responsive ‘sensor’ transgenes uncover Hox-like and other developmentally regulated patterns of vertebrate microRNA expression.; citation_author=JH Mansfield; citation_author=BD Harfe; citation_author=R Nissen; citation_author=J Obenauer; citation_author=J Srineel; citation_journal_title=Nat Genet; citation_volume=36(10); citation_number=19; citation_pages=1079-83; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mammalian polycomb-mediated repression of Hox genes requires the essential spliceosomal protein Sf3b1.; citation_author=K Isono; citation_author=Y Mizutani-Koseki; citation_author=T Komori; citation_author=MS Schmidt-Zachmann; citation_author=H Koseki; citation_journal_title=Genes Dev; citation_volume=19(5); citation_number=20; citation_pages=536-41; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Antisense transcripts in the human genome.; citation_author=B Lehner; citation_author=G Williams; citation_author=RD Campbell; citation_author=CM Sanderson; citation_journal_title=Trends Genet; citation_volume=18(2); citation_number=21; citation_pages=63-5; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Widespread occurrence of antisense transcription in the human genome.; citation_author=R Yelin; citation_author=D Dahary; citation_author=R Sorek; citation_author=EY Levanon; citation_author=O Goldstein; citation_journal_title=Nat Biotechnol; citation_volume=21(4); citation_number=22; citation_pages=379-86; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Do natural antisense transcripts make sense in eukaryotes?; citation_author=C Vanhee-Brossollet; citation_author=C Vaquero; citation_journal_title=Gene; citation_volume=211(1); citation_number=23; citation_pages=1-9; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Initial sequencing and comparative analy sis of the mouse genome.; citation_author=RH Waterston; citation_author=K Lindblad-Toh; citation_author=E Birney; citation_author=J Rogers; citation_author=JF Abril; citation_journal_title=Nature; citation_volume=420(6915); citation_number=24; citation_pages=520-62; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=BLAT–the BLAST-like alignment tool.; citation_author=WJ Kent; citation_journal_title=Genome Res; citation_volume=12(4); citation_number=25; citation_pages=656-64; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=An improved sequence assembly program.; citation_author=X Huang; citation_journal_title=Genomics; citation_volume=33(1); citation_number=26; citation_pages=21–; citation_date=1996; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000356" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Extensive Polycistronism and Antisense Transcription in the Mammalian Hox Clusters"/>
    <meta name="twitter:description" content="The Hox clusters play a crucial role in body patterning during animal development. They encode both Hox transcription factor and micro-RNA genes that are activated in a precise temporal and spatial sequence that follows their chromosomal order. These remarkable collinear properties confer functional unit status for Hox clusters. We developed the TranscriptView platform to establish high resolution transcriptional profiling and report here that transcription in the Hox clusters is far more complex than previously described in both human and mouse. Unannotated transcripts can represent up to 60% of the total transcriptional output of a cluster. In particular, we identified 14 non-coding Transcriptional Units antisense to Hox genes, 10 of which (70%) have a detectable mouse homolog. Most of these Transcriptional Units in both human and mouse present conserved sizeable sequences (&gt;40 bp) overlapping Hox transcripts, suggesting that these Hox antisense transcripts are functional. Hox clusters also display at least seven polycistronic clusters, i.e., different genes being co-transcribed on long isoforms (up to 30 kb). This work provides a reevaluated framework for understanding Hox gene function and dys-function. Such extensive transcriptions may provide a structural explanation for Hox clustering."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000356.g003"/>

  <meta property="og:title" content="Extensive Polycistronism and Antisense Transcription in the Mammalian Hox Clusters" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000356" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6153'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=94'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4455&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000356">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000356" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000356&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Extensive Polycistronism and Antisense Transcription in the Mammalian <i>Hox</i> Clusters
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gaëll Mainguy
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:gaell.mainguy@institut.veolia.org">gaell.mainguy@institut.veolia.org</a> (GM); <a href="mailto:A.J.Durston@biology.leidenuniv.nl">A.J.Durston@biology.leidenuniv.nl</a> (AD)</p>

                <p>Affiliation:
                  Hubrecht Laboratory, Center for Biomedical Genetics, Utrecht, The Netherlands
                </p>

                <p>
                    Current address: Institut Veolia Environnement, Paris, France
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jan Koster, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, Academic Medical Center, University of Amsterdam, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joost Woltering, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Hubrecht Laboratory, Center for Biomedical Genetics, Utrecht, The Netherlands
                </p>

                <p>
                    Current address: Institute of Biology, IBL, Faculty of Science and Mathematics, University of Leiden, Leiden, The Netherlands
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hans Jansen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Hubrecht Laboratory, Center for Biomedical Genetics, Utrecht, The Netherlands
                </p>

                <p>
                    Current address: Institute of Biology, IBL, Faculty of Science and Mathematics, University of Leiden, Leiden, The Netherlands
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Antony Durston
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:gaell.mainguy@institut.veolia.org">gaell.mainguy@institut.veolia.org</a> (GM); <a href="mailto:A.J.Durston@biology.leidenuniv.nl">A.J.Durston@biology.leidenuniv.nl</a> (AD)</p>

                <p>Affiliation:
                  Hubrecht Laboratory, Center for Biomedical Genetics, Utrecht, The Netherlands
                </p>

                <p>
                    Current address: Institute of Biology, IBL, Faculty of Science and Mathematics, University of Leiden, Leiden, The Netherlands
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 04, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000356</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000356" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000356" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000356" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000356" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000356-g001" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000356-g002" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000356-t001" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000356-t002" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000356-t003" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000356-g003" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000356">Reader Comments (5)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000356" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The <em>Hox</em> clusters play a crucial role in body patterning during animal development. They encode both <em>Hox</em> transcription factor and micro-RNA genes that are activated in a precise temporal and spatial sequence that follows their chromosomal order. These remarkable collinear properties confer functional unit status for <em>Hox</em> clusters. We developed the TranscriptView platform to establish high resolution transcriptional profiling and report here that transcription in the <em>Hox</em> clusters is far more complex than previously described in both human and mouse. Unannotated transcripts can represent up to 60% of the total transcriptional output of a cluster. In particular, we identified 14 non-coding Transcriptional Units antisense to <em>Hox</em> genes, 10 of which (70%) have a detectable mouse homolog. Most of these Transcriptional Units in both human and mouse present conserved sizeable sequences (&gt;40 bp) overlapping <em>Hox</em> transcripts, suggesting that these <em>Hox</em> antisense transcripts are functional. <em>Hox</em> clusters also display at least seven polycistronic clusters, i.e., different genes being co-transcribed on long isoforms (up to 30 kb). This work provides a reevaluated framework for understanding <em>Hox</em> gene function and dys-function. Such extensive transcriptions may provide a structural explanation for <em>Hox</em> clustering.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Mainguy G, Koster J, Woltering J, Jansen H, Durston A (2007) Extensive Polycistronism and Antisense Transcription in the Mammalian <em>Hox</em> Clusters. PLoS ONE 2(4):
          e356.
            doi:10.1371/journal.pone.0000356</p><p><strong>Academic Editor: </strong>Thomas Zwaka, Baylor College of Medicine, United States of America</p><p><strong>Received:</strong> January 31, 2007; <strong>Accepted:</strong> February 26, 2007; <strong>Published:</strong> April 4, 2007</p><p><strong>Copyright:</strong> © 2007 Mainguy et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the European Union FP5 Marie Curie and Biotech programmes, and also by an EMBO long term fellowship tp GM and a Marie Curie individual fellowship to G.M. These sponsors supported all phases of this work up until the final writing phase.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Hox clusters are amongst the most remarkable genomic objects, the structure and function of which are crucial to understand, as Hox clusters are implicated in a growing number of diseases from cancers to congenital malformations <a href="#pone.0000356-Grier1">[1]</a>. Mammals possess four similar Hox clusters, HoxA, HoxB, HoxC and HoxD, located on different chromosomes, consisting of 9 to 11 <em>Hox</em> genes arranged in tandem. The order of <em>Hox</em> genes along the chromosome corresponds to the order in which they act along the body axes and this collinear property links clustering to function emphasizing that Hox clusters are functional units <a href="#pone.0000356-Duboule1">[2]</a>. The Hox clusters also contain 5 micro RNA (miRNA) genes intercalated at two homologous positions <a href="#pone.0000356-Kosman1">[3]</a>, <a href="#pone.0000356-Yekta1">[4]</a>. The organization of Hox complexes is highly conserved in vertebrates and <em>Hox</em> and <em>mir</em> genes not only stay clustered but also in close proximity to each other despite their very complex and dynamic expression patterns, a property in apparent contradiction with the observation that the more complex the expression pattern of a gene is, the larger its flanking non coding DNA <a href="#pone.0000356-Nelson1">[5]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>This apparent paradox raises the question of the selective pressure(s) at work for maintaining <em>Hox</em> and <em>mir</em> genes clustered. Current models propose that clustering is maintained <em>via</em> the sharing of <em>cis</em>-regulatory elements that control several <em>Hox</em> genes either locally or globally <a href="#pone.0000356-Mann1">[6]</a>, <a href="#pone.0000356-Gould1">[7]</a>, <a href="#pone.0000356-Duboule1">[2]</a>. Other aspects of transcriptional structure could also be important. First, a case of polycistronism has been reported where <em>Hoxc6</em>, <em>Hoxc5</em> and <em>Hoxc4</em> are co-transcribed and gene-specific transcripts result from alternative splicing <a href="#pone.0000356-Simeone1">[8]</a>. Notably, polycistronic Hox transcripts have also been reported in a number of crustaceans <a href="#pone.0000356-Shiga1">[9]</a>, indicating their importance in diverse metazoa. Second, a <em>Hoxa11</em> antisense RNA is transcribed immediately 5′ to <em>HoxA11</em> and is involved in its regulation <a href="#pone.0000356-HsiehLiHMDavis1">[10]</a>. Thus, Hox clusters present unusual transcriptional characteristics that may play an important role for Hox gene expression.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The transcriptional complexity of mammalian genomes is increasingly recognized <a href="#pone.0000356-Engstrom1">[11]</a> and data mining provides a suitable way to establish transcriptional structure of poorly expressed genes.Here we present a thorough analysis of the best described vertebrate (human and murine) Hox clusters.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>The majority of the transcriptional activity of the Hox Clusters is not annotated</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>As the gene is a misleading concept we follow the unambiguous definitions proposed by the FANTOM consortium: A transcriptional unit (TU) is a segment of the genome flanked by the most distal exons from which transcripts are generated <a href="#pone.0000356-Okazaki1">[12]</a>. The transcripts sharing any exon are merged into a single TU. If two transcripts do not share any single exon, they constitute two different TUs, even if they overlap or if one is localized in the intron of the other. In particular, two transcripts on opposite strands always constitute two different TUs. Aligning the genome with all of the ESTs and mRNA provides a reliable method to delineate exons and deduce TU structures in the entire organism, independently of time and space and throughout its life cycle <a href="#pone.0000356-Saha1">[13]</a>. We computationally mined public mouse and human databases using a dedicated software platform, TranscriptView (see <a href="#s3">material and methods</a>) and found that Hox cluster profiles are far more complex than annotated (<a href="#pone-0000356-g001">fig 1a</a>) but nonetheless very similar between human and mouse (see supporting online material). The importance of transcription beyond annotation has been established and 12.2% of the unannotated human chromosome 22 is transcribed <a href="#pone.0000356-Rinn1">[14]</a>. In the Hox clusters, we found that this proportion ranged from 67% (HoxC) to 92% (HoxD) (<a href="#pone-0000356-g001">fig 1c</a>). Moreover, these unannotated transcripts can represent up to 60% of the total transcriptional output of a cluster (<a href="#pone-0000356-g001">fig 1d</a>), while it is a marginal phenomenon in two other clustered gene families, Globin and Kallikrein (&lt;5%) (<a href="#pone-0000356-g001">fig 1b,d</a>). Kallikrein genes present a loosely clustered organization with 15 and 25 genes in human and mouse respectively, the function if any of the clustering being not known <a href="#pone.0000356-Yousef1">[15]</a>. On the other hand, the β-Globin cluster is another example of functional clustering since β-Globin gene expression displays temporal collinearity. Even for the β-Globin cluster that presents extensive intergenic transcription (<a href="#pone.0000356-Cook1">[16]</a> and <a href="#pone-0000356-g001">figure 1b,c</a>), more than 95% of the transcribed sequences match annotated genes (see <a href="#pone-0000356-g001">fig 1b,d</a>). In general, the distribution of ESTs to genes is highly skewed as a large number of genes are represented by only one or a few transcripts <a href="#pone.0000356-Thanaraj1">[17]</a>, our results are therefore likely to be an underestimation.</p>
<div class="figure" id="pone-0000356-g001"><div class="img"><a name="pone-0000356-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000356.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000356.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Transcription profiles of the Human Hox clusters.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(a) Human <em>Hox</em> Cluster transcriptographs. (b) Transcriptograph of the human β-globin cluster. Note that despite an extensive transcription, the vast majority of sequences correspond to annotated genes. In 1a and 1b, annotations of genes in Refseq are depicted in red. (c) Proportion of the clusters that are primarily transcribed (d) Amount of transcription not currently annotated.</p>
<span>doi:10.1371/journal.pone.0000356.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In an effort to re-annotate the Hox clusters, we used the following strategy to establish TUs and discriminate functional RNAs. First, we restricted our analysis to spliced transcripts as splicing is evidence against genomic contamination and splice site asymmetry allows transcript orientation. As most of these transcripts are non-coding (see below), protein conservation was a useless criterion. To categorize the TUs along a scale of degree of confidence, we therefore focused on the exon-intron structure and nucleotide sequence conservation. In our analysis, <strong>Transcript existence</strong> (1) is defined by presence of multiple spliced transcripts in human databases, <strong>Sequence conservation</strong> (2) is observed when transcripts from two different species share a conserved sequence and <strong>Exon-Exon structure conservation</strong> (3) characterizes transcripts from two different species displaying the same intron boundaries. Our findings are summarized in <a href="#pone-0000356-t001">tables 1</a> and <a href="#pone-0000356-t002">2</a> and are depicted in <a href="#pone-0000356-g002">figure 2</a>. This delineation of TU in the Hox clusters reveals the occurrence of two major phenomena, polycistronism and antisense transcription.</p>
<div class="figure" id="pone-0000356-g002"><div class="img"><a name="pone-0000356-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000356.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000356.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Synopsis of transcriptional activity in the Human Hox Clusters.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Sense and antisense transcriptions are in red and green respectively. Dark and light shaded boxes represent exon and intron. Mir genes are in blue. The three long transcripts presenting a murine homolog with a conserved exon-exon boundary are headed by **. The 13 TUs (12 antisense and one sense) located at similar position in human and mouse are denoted by an asterisk (*) while the 10 TUs showing conservation are depicted with</p>
<span>doi:10.1371/journal.pone.0000356.g002</span></div><div class="figure" id="pone-0000356-t001"><div class="img"><a name="pone-0000356-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000356.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000356.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Polycistronism in the Human Hox cluster</span></strong></p><span>doi:10.1371/journal.pone.0000356.t001</span></div><div class="figure" id="pone-0000356-t002"><div class="img"><a name="pone-0000356-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000356.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000356.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Sense AntiSense overlaps</span></strong></p><span>doi:10.1371/journal.pone.0000356.t002</span></div>

<h4>Polycistronic clusters</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A polycistronic cluster designates two or more genes co-transcribed from a single promoter, sharing a non-coding exon, and whose products are generated by alternative splicing <a href="#pone.0000356-Blumenthal1">[18]</a>. An operon is a particular case where the mRNA retains the different products after splicing. In Mammals, both operons and polycistronic clusters are scarcely documented <a href="#pone.0000356-Blumenthal1">[18]</a>. One clear example nonetheless, is the case of the <em>Hoxc4</em>, <em>Hoxc5</em> and <em>Hoxc6</em> genes that can be co-transcribed from a common promoter <a href="#pone.0000356-Simeone1">[8]</a>. We found 22 Hox transcripts for which introns seem to encompass other genes. In three cases we could identify a homolog in rodent that presented a conserved exon-exon boundary (&gt;85% identity over at least 60 nucleotides, see supporting online material). In total, multiple alignment and identification of orthologs provided support for the existence of seven polycistronic clusters which concern 38% (15/39) of the <em>Hox</em> genes (<a href="#pone-0000356-t001">table 1</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Remarkably, the five miRNAs are located within introns of atypical transcripts and are therefore co-transcribed with <em>Hox</em> genes (<a href="#pone-0000356-g002">figure 2</a>). Evidence for <em>Hoxb4</em> and <em>mir-10a</em> was missing in the databases and we confirmed their co-transcription by RT-PCR, providing hereby an explanation for the observation that these two genes have markedly similar expression patterns <a href="#pone.0000356-Mansfield1">[19]</a>. More generally, co-transcription of mir and <em>Hox</em> genes gives a seductive framework to interpret the stability of <em>Hox</em> and <em>mir</em> gene positions relative to each other. Our results also shed light on the importance of splicing regulation within the Hox clusters, a conclusion in accordance with the recent finding that the knock out of the gene encoding the spliceosomal protein Sf3b1 leads to deregulation of <em>Hox</em> gene expressions and severe skeletal transformations <a href="#pone.0000356-Isono1">[20]</a>.</p>


<h4>Widespread antisense transcription</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Our analysis also revealed the existence of 15 TUs distinct from the <em>Hox</em> and <em>mir</em> genes that are poly-adenylated and alternatively spliced like genuine products of RNA Polymerase II. Most of these TUs (14/15) are transcribed antisense (AS) to <em>Hox</em> genes (see <a href="#pone-0000356-g002">fig 2</a>), and AS transcription can represent up to 38% of the spliced transcripts (38.46% for HoxA, 33.11% for HoxB, 13.16% for HoxC and 34.84% for HoxD). Cis-encoded antisenses and bidirectional promoters are now known to be abundant in the human genome <a href="#pone.0000356-Engstrom1">[11]</a>. Whereas most of the previously identified vertebrate AS transcripts encode proteins <a href="#pone.0000356-Lehner1">[21]</a>, <a href="#pone.0000356-Yelin1">[22]</a>, we did not detect any conserved open reading frames suggesting that all of the Hox AS TUs are non-coding (see <a href="#s3">methods</a>). However, 12 AS TUs can also be assigned to mouse Hox clusters at similar positions and 10 human AS TUs (71%) have a detectable homolog in the mouse transcriptome (<a href="#pone-0000356-g002">figure 2</a> and <a href="#pone-0000356-t003">table 3, Sm</a>) and they are therefore likely to be functional.</p>
<div class="figure" id="pone-0000356-t003"><div class="img"><a name="pone-0000356-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000356.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000356.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Conservation of TU AS between human and mouse</span></strong></p><span>doi:10.1371/journal.pone.0000356.t003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To date, AS RNAs have been implicated in various aspects of eukaryotic gene expression as diverse as genomic imprinting, RNA interference, translational regulation, alternative splicing, or RNA editing <a href="#pone.0000356-Lehner1">[21]</a>, <a href="#pone.0000356-Yelin1">[22]</a>, <a href="#pone.0000356-VanheeBrossollet1">[23]</a>. AS transcripts frequently originate from the same locus as sense transcripts and are called <em>cis</em>-encoded antisenses. They are thought to exert a control on RNA sense expression by sense-antisense (SAS) pairing <a href="#pone.0000356-Lehner1">[21]</a>, <a href="#pone.0000356-Yelin1">[22]</a>. We searched for potential SAS contacts (&gt;40 bp) and found that nine AS TUs (65%) have sequences reverse-complementary to twelve <em>Hox</em> mRNAs (<a href="#pone-0000356-t002">table 2</a>). This proportion is rather high as, on a genomic scale, natural AS transcription has been evaluated to target from 2 to 8% of the human genes <a href="#pone.0000356-Lehner1">[21]</a>, <a href="#pone.0000356-Yelin1">[22]</a>. Similarly, in mouse sequences eight <em>Hox</em> genes are subjected to <em>cis</em>-antisense interactions, three of which, <em>Hoxa3</em>, <em>Hoxb3</em> and <em>Hoxb5</em>, present the same SAS in both human and mouse (<a href="#pone-0000356-t002">table 2</a>). The conservation of SAS sequences between human and mouse strongly supports the hypothesis that these AS TUs are functional. Moreover, all of the SAS overlap sequences are remarkably conserved in the other species suggesting that <em>cis</em>-encoded antisenses could target as many as 22 <em>Hox</em> genes (<a href="#pone-0000356-t002">table 2</a>). Besides these interactions, <em>trans</em>-encoded AS RNAs have also been reported where the AS transcript originates from a different locus and displays only partial complementarity with the sense transcript <a href="#pone.0000356-VanheeBrossollet1">[23]</a>. We identified 6 and 5 potential <em>trans</em>-interactions in human and mouse respectively (SAS contact; &gt;40 nucleotides, &gt;85% identity) (<a href="#pone-0000356-t002">table 2</a>). These SAS interactions usually occur within a paralog group (A1/B1, A3/B3 or A11/C11/D11) but there are three noteworthy exceptions (B4/B5, B2/A9 and B2/D3). Remarkably, antisense transcripts with the potential to recognize <em>Hoxb4</em> and <em>Hoxd3</em> in <em>trans</em> are present in both human and mouse.</p>


<h4>Functional clustering and extensive transcription correlate with absence of transposons</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Our analysis suggests that, in addition to the sharing of <em>cis</em>-regulatory elements, the existence of operons, polycistronism and antisense-sense pairing provide additional constraints for maintaining Hox clusters as functional units. If this were the case, exogenous start and stop transcription signals would be highly counter-selected. Indeed, the four Hox clusters are by far the most repeat-poor regions of the genome in both human and mouse, and the current explanation is that insertions would interfere with the dense network of <em>cis</em>-interactions <a href="#pone.0000356-Waterston1">[24]</a>. We analyzed the repeat distribution and found that transposons are virtually absent from transcribed regions but that they can accumulate within the clusters at untranscribed regions. The HoxB cluster provides a threefold example of this mutual exclusiveness between transposons and transcription (see <a href="#pone-0000356-g003">figure 3</a>, and see supporting online material for the other clusters). In both human and mouse, the intergenic region between <em>Hoxb1</em> and <em>Hoxb2</em> is notably not transcribed (see <a href="#pone-0000356-g001">fig 1</a>) and has been independently colonized by SINEs (13 in human, 17 in mouse). (2) The sequence upstream of <em>Hoxb9</em> is massively filled with repeats as <em>Hoxb13</em> is drifting away (Hs:107 SINEs, 31 LTRs, 61 LINEs; Mm: 113 SINEs, 18 LTRs, 14 LINEs). (3) But reciprocally, the posterior limit of repeat accumulation does not coincide with <em>Hoxb9</em> but with the non-coding TU that is upstream of it (<a href="#pone-0000356-g003">figure 3</a>). Moreover, whereas transposons are indeed very rare, on the other hand simple repeats of di- or tri-nucleotides are found throughout the Hox clusters (<a href="#pone-0000356-g003">figure 3</a>) arguing against the preeminence of sequence disruption <em>per se</em>. An alternative explanation could be that transposons are counter selected for their potential to interfere with transcription. Incidentally, this inverted correlation supports the hypothesis that these non-coding transcription products are functional.</p>
<div class="figure" id="pone-0000356-g003"><div class="img"><a name="pone-0000356-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000356" data-uri="info:doi/10.1371/journal.pone.0000356.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000356.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000356.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000356.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000356.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000356.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Transposons are excluded from transcribed regions.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Human HoxB cluster has been repeat-masked. Deduced exons retrieved from TranscriptView are in blue. Human repeats are depicted by colored squares. Note that simple repeats, represented by yellow squares are distributed throughout the HoxB cluster independently of the gene content.</p>
<span>doi:10.1371/journal.pone.0000356.g003</span></div>

<h4>Concluding Remarks</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Our analysis confers on Hox clusters the status of the most complex objects reported to date in mammals in terms of both polycistronism and antisense and suggests that, in addition to enhancer sharing, these mechanisms provide additional constraints for maintaining Hox clusters as functional units. There is increasing recognition that the production of RNA transcripts from both orientations can produce coordinate regulation and since mammalian mRNAs that form sense-antisense pairs frequently exhibit reciprocal expression patterns <a href="#pone.0000356-Lehner1">[21]</a> it is tempting to speculate that antisense transcription in the Hox Clusters is instrumental in establishing limits of gene expression. In conclusion, by unraveling the complex transcriptional organisation of the Hox clusters, our analysis blurs the traditional view of <em>Hox</em> genes and provides a reevaluated framework for understanding <em>Hox</em> gene function and dys-function.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Methods"></a><h3>Methods</h3>
<h4>The TranscriptView software platform</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We used the TranscriptView software platform to obtain and manipulate clusters of human expressed sequences aligned to genomic DNA. TranscriptView makes use of public genome alignment data for EST and mRNA sequences generated with BLAT by the UCSC genome consortium (<a href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>). The BLAT program is specifically designed for transcript to genome alignments making it possible to align large collections of sequences to the genome <a href="#pone.0000356-Kent1">[25]</a>. Expressed sequences are compared to the human genome to find high quality hits, and are then aligned to it using a spliced alignment model that allows long gaps, for modeling introns. The maximum intron length allowed by BLAT is 500,000 bases. When a single EST aligned in multiple places, the alignment having the highest base identity is identified. Low-quality sequence ends that disagree with the DNA are trimmed. Only alignments having a base identity level within 0.5% of the best and at least 96% base identity with the genomic sequence are kept (<a href="http://genome.ucsc.edu/cgi-bin/hgTrackUi?g%E2%80%8A=%E2%80%8Aest">http://genome.ucsc.edu/cgi-bin/hgTrackUi​?g= est</a>). Further, expressed sequences aligning to two or more chromosomes are discarded as suspected chimeras. Overlapping expressed sequences and corresponding genomic sequences are multiply aligned. Positions on the genomic sequence in which there is at least one expressed sequence that opens or closes a long gap, are considered splice sites. The exact position of the splice sites is determined taking the GT...AG rule into consideration as described in <a href="#pone.0000356-Yelin1">[22]</a>. The list of all of the alignment boundaries is generated allowing a quantitative determination of the transcriptional status of any genomic segment at the base pair level. The deduced Exon-intron organization and the orientation when available are also accessible through the TranscriptView graphical interface.</p>


<h4>Datasets</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Using BLAT we retrieved 2630 Ests and mRNA sequences that aligned with the human Hox clusters (HoxA 837, HoxB 807, HoxC 441, HoxD 545). The distribution of this primary set is described in <a href="#pone-0000356-g001">figure 1</a>. As similarities of sequence within a cluster of tandemly repeated genes can be a source of misalignment we compared our results with two other clustered family of genes, Kallikrein and β-Globin clusters. Subsequent analysis of polycistronism and antisense was restricted to spliced sequences that account for ca. 25% of the primary set (HoxA 202, HoxB 241, HoxC 127, HoxD 133).</p>


<h4>TU annotations and transcript analysis</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Among this secondary set, 96 sequences displayed at least one intron longer than 7 kb (see the list in supporting online material for references and characteristics). These sequences were then merged with ‘classical’ <em>Hox</em> transcripts, grouped according to cluster and orientation and TUs were constructed using the Contig Assembly Program (<a href="http://www.infobiogen.fr/services/analyseq/cgi-bin/cap_in.pl">http://www.infobiogen.fr/services/analys​eq/cgi-bin/cap_in.pl</a>) <a href="#pone.0000356-Huang1">[26]</a>. CAP generated contigs were then checked for misalignments. To identify putative homologuous TUs, non-redundant representative sequences for each TU were selected on the basis of the CAP contigs and blasted against vertebrate transcription databases.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In the case of the 14 antisense TUs, we collected a representative set of 52 sequences to identify putative homologous and to evaluate the coding potential. Using the Diogenes ORF prediction program (<a href="http://web.ahc.umn.edu/cgi-bin/diogenes/diogenes.cgi">http://web.ahc.umn.edu/cgi-bin/diogenes/​diogenes.cgi</a>), eight different sequences presented a score compatible with an ORF (p&gt;10-3) but subsequent BLAST analysis failed to detect any conserved pattern outside human.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>These 52 sequences were systematically blasted against human database and alignment with sense <em>Hox</em> transcripts were reported as an indication of putative SAS contacts. Imperfect alignment and inconsistency in the genomic locations were the signs of putative <em>trans</em>-SAS contacts. Conservation of the SAS sequences was assessed by species cross-blasting. We undertook a similar procedure for the mouse <em>Hox</em> antisense TUs. The results are summarized in <a href="#pone-0000356-t002">table 2</a>.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p><a href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a> UCSC genome consortium: home page</p>
<a id="article1.back1.ack1.p2" name="article1.back1.ack1.p2"></a><p><a href="http://genome.ucsc.edu/cgi-bin/hgTrackUi?g%E2%80%8A=%E2%80%8Aest">http://genome.ucsc.edu/cgi-bin/hgTrackUi​?g= est</a> UCSC genome consortium: BLAT database.</p>
<a id="article1.back1.ack1.p3" name="article1.back1.ack1.p3"></a><p><a href="http://www.infobiogen.fr/services/analyseq/cgi-bin/cap_in.pl">http://www.infobiogen.fr/services/analys​eq/cgi-bin/cap_in.pl</a> Contig Assembly Program.</p>
<a id="article1.back1.ack1.p4" name="article1.back1.ack1.p4"></a><p><a href="http://web.ahc.umn.edu/cgi-bin/diogenes/diogenes.cgi">http://web.ahc.umn.edu/cgi-bin/diogenes/​diogenes.cgi</a> Diogenes ORF prediction program</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: GM. Performed the experiments: GM JW HJ. Analyzed the data: JK AD GM JW HJ. Contributed reagents/materials/analysis tools: JK AD. Wrote the paper: AD GM. Other: Co-initiator and planner of the project (but biggest contribution by Mainguy), Head of Lab, Contributed to analysis, Provided all facilities, Co-writer: AD. Main author: GM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000356-Grier1" id="pone.0000356-Grier1"></a>Grier DG, Thompson A, Kwasniewska A, McGonigle GJ, Halliday HL, et al.  (2005) The pathophysiology of HOX genes and their role in cancer. J. Pathol  205(2): 154–71.  <ul class="find" data-citedArticleID="942796" data-doi="10.1002/path.1710"><li><a href="http://dx.doi.org/10.1002/path.1710" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+pathophysiology+of+HOX+genes+and+their+role+in+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+pathophysiology+of+HOX+genes+and+their+role+in+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000356-Duboule1" id="pone.0000356-Duboule1"></a>Duboule D (1998) Vertebrate hox gene regulation: clustering and/or colinearity? Curr Opin Genet Dev  8(5): 514–8.  <ul class="find" data-citedArticleID="942790" data-doi="10.1016/s0959-437x(98)80004-x"><li><a href="http://dx.doi.org/10.1016/s0959-437x(98)80004-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vertebrate+hox+gene+regulation%3A+clustering+and%2For+colinearity%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vertebrate+hox+gene+regulation%3A+clustering+and%2For+colinearity%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000356-Kosman1" id="pone.0000356-Kosman1"></a>Kosman D, Mizutani CM, Lemons D, Cox WG, McGinnis W, et al.  (2004) Multiplex detection of RNA expression in Drosophila embryos. Science  305(5685): 846.  <ul class="find" data-citedArticleID="942806" data-doi="10.1126/science.1099247"><li><a href="http://dx.doi.org/10.1126/science.1099247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplex+detection+of+RNA+expression+in+Drosophila+embryos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplex+detection+of+RNA+expression+in+Drosophila+embryos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000356-Yekta1" id="pone.0000356-Yekta1"></a>Yekta S, Shih IH, Bartel DP (2004) MicroRNA-directed cleavage of HOXB8 mRNA. Science  304(5670): 594–6.  <ul class="find" data-citedArticleID="942832" data-doi="10.1126/science.1097434"><li><a href="http://dx.doi.org/10.1126/science.1097434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA-directed+cleavage+of+HOXB8+mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA-directed+cleavage+of+HOXB8+mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000356-Nelson1" id="pone.0000356-Nelson1"></a>Nelson CE, Hersh BM, Carroll SB (2004) The regulatory content of intergenic DNA shapes genome architecture. Genome Biol  5(4): R25.  <ul class="find" data-citedArticleID="942814" data-doi="10.1186/gb-2004-5-4-r25"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-4-r25" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+regulatory+content+of+intergenic+DNA+shapes+genome+architecture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+regulatory+content+of+intergenic+DNA+shapes+genome+architecture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000356-Mann1" id="pone.0000356-Mann1"></a>Mann RS (1997) Why are Hox genes clustered? Bioessays  19(8): 661–4.  <ul class="find" data-citedArticleID="942810" data-doi="10.1002/bies.950190804"><li><a href="http://dx.doi.org/10.1002/bies.950190804" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+are+Hox+genes+clustered%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+are+Hox+genes+clustered%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000356-Gould1" id="pone.0000356-Gould1"></a>Gould A, Morrison A, Sproat G, White RA, Krumlauf R (1997) Positive cross-regulation and enhancer sharing: two mechanisms for specifying overlapping Hox expression patterns. Genes Dev  11(7): 900–13.  <ul class="find" data-citedArticleID="942794" data-doi="10.1101/gad.11.7.900"><li><a href="http://dx.doi.org/10.1101/gad.11.7.900" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+cross-regulation+and+enhancer+sharing%3A+two+mechanisms+for+specifying+overlapping+Hox+expression+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+cross-regulation+and+enhancer+sharing%3A+two+mechanisms+for+specifying+overlapping+Hox+expression+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000356-Simeone1" id="pone.0000356-Simeone1"></a>Simeone A, Pannese M, Acampora D, D'Esposito M, Boncinelli E (1988) At least three human homeoboxes on chromosome 12 belong to the same transcription unit. Nucleic Acids Res  16(12): 5379–90.  <ul class="find" data-citedArticleID="942824" data-doi="10.1093/nar/16.12.5379"><li><a href="http://dx.doi.org/10.1093/nar/16.12.5379" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=At+least+three+human+homeoboxes+on+chromosome+12+belong+to+the+same+transcription+unit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22At+least+three+human+homeoboxes+on+chromosome+12+belong+to+the+same+transcription+unit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000356-Shiga1" id="pone.0000356-Shiga1"></a>Shiga Y, Sagawa K, Takai R, Sakaguchi H, Yamagata H, et al.  (2006) Transcriptional readthrough of Hox genes Ubx and Antp and their divergent post-transcriptional control during crustacean evolution. Evol Dev. Sep–Oct;  8(5): 407–14.  <ul class="find" data-citedArticleID="942822" data-doi="10.1111/j.1525-142x.2006.00114.x"><li><a href="http://dx.doi.org/10.1111/j.1525-142x.2006.00114.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+readthrough+of+Hox+genes+Ubx+and+Antp+and+their+divergent+post-transcriptional+control+during+crustacean+evolution.+Evol+Dev." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+readthrough+of+Hox+genes+Ubx+and+Antp+and+their+divergent+post-transcriptional+control+during+crustacean+evolution.+Evol+Dev.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000356-HsiehLiHMDavis1" id="pone.0000356-HsiehLiHMDavis1"></a>Hsieh-Li HM Davis AP, Witte DP, Potter SS, Capecchi MR, et al.  (1995) Hoxa 11 structure, extensive antisense transcription, and function in male and female fertility. Development  121(5): 1373–85.  <ul class="find" data-citedArticleID="942798"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hoxa+11+structure%2C+extensive+antisense+transcription%2C+and+function+in+male+and+female+fertility.&amp;auth=&amp;atitle=Hoxa+11+structure%2C+extensive+antisense+transcription%2C+and+function+in+male+and+female+fertility." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hoxa+11+structure%2C+extensive+antisense+transcription%2C+and+function+in+male+and+female+fertility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hoxa+11+structure%2C+extensive+antisense+transcription%2C+and+function+in+male+and+female+fertility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000356-Engstrom1" id="pone.0000356-Engstrom1"></a>Engstrom PG, Suzuki H, Ninomiya N, Akalin A, Sessa L, et al.  (2006) Complex Loci in Human and Mouse Genomes. PLoS Genetics.  2(4): e47.  <ul class="find" data-citedArticleID="942792" data-doi="10.1371/journal.pgen.0020047"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+Loci+in+Human+and+Mouse+Genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+Loci+in+Human+and+Mouse+Genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000356-Okazaki1" id="pone.0000356-Okazaki1"></a>Okazaki Y, Furuno M, Kasukawa T, Adachi J, Bono H, et al.  (2002) Analysis of the mouse transcriptome based on functional annotation of 60,770 full-length cDNAs. Nature  420(6915): 563–73.  <ul class="find" data-citedArticleID="942816" data-doi="10.1038/nature01266"><li><a href="http://dx.doi.org/10.1038/nature01266" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+mouse+transcriptome+based+on+functional+annotation+of+60%2C770+full-length+cDNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+mouse+transcriptome+based+on+functional+annotation+of+60%2C770+full-length+cDNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000356-Saha1" id="pone.0000356-Saha1"></a>Saha S, Sparks AB, Rago C, Akmaev V, Wang CJ, et al.  (2002) Using the transcriptome to annotate the genome. Nat Biotechnol  20(5): 508–12.  <ul class="find" data-citedArticleID="942820" data-doi="10.1038/nbt0502-508"><li><a href="http://dx.doi.org/10.1038/nbt0502-508" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+the+transcriptome+to+annotate+the+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+the+transcriptome+to+annotate+the+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000356-Rinn1" id="pone.0000356-Rinn1"></a>Rinn JL, Euskirchen G, Bertone P, Martone R, Luscombe NM, et al.  (2003) The transcriptional activity of human Chromosome 22. Genes Dev  17(4): 529–40.  <ul class="find" data-citedArticleID="942818" data-doi="10.1101/gad.1055203"><li><a href="http://dx.doi.org/10.1101/gad.1055203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptional+activity+of+human+Chromosome+22." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptional+activity+of+human+Chromosome+22.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000356-Yousef1" id="pone.0000356-Yousef1"></a>Yousef GM, Diamandis EP (2003) An overview of the kallikrein gene families in humans and other species: emerging candidate tumour markers. Clin Biochem  36(6): 443–52.  <ul class="find" data-citedArticleID="942836" data-doi="10.1016/s0009-9120(03)00055-9"><li><a href="http://dx.doi.org/10.1016/s0009-9120(03)00055-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+overview+of+the+kallikrein+gene+families+in+humans+and+other+species%3A+emerging+candidate+tumour+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+overview+of+the+kallikrein+gene+families+in+humans+and+other+species%3A+emerging+candidate+tumour+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000356-Cook1" id="pone.0000356-Cook1"></a>Cook PR (2003) Nongenic transcription, gene regulation and action at a distance. J Cell Sci  116(Pt 22): 4483–91.  <ul class="find" data-citedArticleID="942788" data-doi="10.1242/jcs.00819"><li><a href="http://dx.doi.org/10.1242/jcs.00819" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nongenic+transcription%2C+gene+regulation+and+action+at+a+distance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nongenic+transcription%2C+gene+regulation+and+action+at+a+distance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000356-Thanaraj1" id="pone.0000356-Thanaraj1"></a>Thanaraj TA, Clark F, Muilu J (2003) Conservation of human alternative splice events in mouse. Nucleic Acids Res  31(10): 2544–52.  <ul class="find" data-citedArticleID="942826" data-doi="10.1093/nar/gkg355"><li><a href="http://dx.doi.org/10.1093/nar/gkg355" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservation+of+human+alternative+splice+events+in+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservation+of+human+alternative+splice+events+in+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000356-Blumenthal1" id="pone.0000356-Blumenthal1"></a>Blumenthal T (1998) Gene clusters and polycistronic transcription in eukaryotes. Bioessays  20(6): 480–7.  <ul class="find" data-citedArticleID="942786" data-doi="10.1002/(sici)1521-1878(199806)20:6<480::aid-bies6&gt;3.3.co;2-k"><li><a href="http://dx.doi.org/10.1002/(sici)1521-1878(199806)20:6<480::aid-bies6&gt;3.3.co;2-k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+clusters+and+polycistronic+transcription+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+clusters+and+polycistronic+transcription+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000356-Mansfield1" id="pone.0000356-Mansfield1"></a>Mansfield JH, Harfe BD, Nissen R, Obenauer J, Srineel J, et al.  (2004) MicroRNA-responsive ‘sensor’ transgenes uncover Hox-like and other developmentally regulated patterns of vertebrate microRNA expression. Nat Genet  36(10): 1079–83.  <ul class="find" data-citedArticleID="942812" data-doi="10.1038/ng1421"><li><a href="http://dx.doi.org/10.1038/ng1421" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA-responsive+%26%23x2018%3Bsensor%26%23x2019%3B+transgenes+uncover+Hox-like+and+other+developmentally+regulated+patterns+of+vertebrate+microRNA+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA-responsive+%26%23x2018%3Bsensor%26%23x2019%3B+transgenes+uncover+Hox-like+and+other+developmentally+regulated+patterns+of+vertebrate+microRNA+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000356-Isono1" id="pone.0000356-Isono1"></a>Isono K, Mizutani-Koseki Y, Komori T, Schmidt-Zachmann MS, Koseki H, et al.  (2005) Mammalian polycomb-mediated repression of Hox genes requires the essential spliceosomal protein Sf3b1. Genes Dev  19(5): 536–41.  <ul class="find" data-citedArticleID="942802" data-doi="10.1101/gad.1284605"><li><a href="http://dx.doi.org/10.1101/gad.1284605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+polycomb-mediated+repression+of+Hox+genes+requires+the+essential+spliceosomal+protein+Sf3b1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+polycomb-mediated+repression+of+Hox+genes+requires+the+essential+spliceosomal+protein+Sf3b1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000356-Lehner1" id="pone.0000356-Lehner1"></a>Lehner B, Williams G, Campbell RD, Sanderson CM (2002) Antisense transcripts in the human genome. Trends Genet  18(2): 63–5.  <ul class="find" data-citedArticleID="942808" data-doi="10.1016/s0168-9525(02)02598-2"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02598-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antisense+transcripts+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antisense+transcripts+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000356-Yelin1" id="pone.0000356-Yelin1"></a>Yelin R, Dahary D, Sorek R, Levanon EY, Goldstein O, et al.  (2003) Widespread occurrence of antisense transcription in the human genome. Nat Biotechnol  21(4): 379–86.  <ul class="find" data-citedArticleID="942834" data-doi="10.1038/nbt808"><li><a href="http://dx.doi.org/10.1038/nbt808" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+occurrence+of+antisense+transcription+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+occurrence+of+antisense+transcription+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000356-VanheeBrossollet1" id="pone.0000356-VanheeBrossollet1"></a>Vanhee-Brossollet C, Vaquero C (1998) Do natural antisense transcripts make sense in eukaryotes? Gene  211(1): 1–9.  <ul class="find" data-citedArticleID="942828" data-doi="10.1016/s0378-1119(98)00093-6"><li><a href="http://dx.doi.org/10.1016/s0378-1119(98)00093-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Do+natural+antisense+transcripts+make+sense+in+eukaryotes%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Do+natural+antisense+transcripts+make+sense+in+eukaryotes%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000356-Waterston1" id="pone.0000356-Waterston1"></a>Waterston RH, Lindblad-Toh K, Birney E, Rogers J, Abril JF, et al.  (2002) Initial sequencing and comparative analy sis of the mouse genome. Nature  420(6915): 520–62.  <ul class="find" data-citedArticleID="942830"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Initial+sequencing+and+comparative+analy+sis+of+the+mouse+genome.&amp;auth=&amp;atitle=Initial+sequencing+and+comparative+analy+sis+of+the+mouse+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequencing+and+comparative+analy+sis+of+the+mouse+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequencing+and+comparative+analy+sis+of+the+mouse+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000356-Kent1" id="pone.0000356-Kent1"></a>Kent WJ (2002) BLAT–the BLAST-like alignment tool. Genome Res  12(4): 656–64.  <ul class="find" data-citedArticleID="942804" data-doi="10.1101/gr.229202."><li><a href="http://dx.doi.org/10.1101/gr.229202." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BLAT%26%23x2013%3Bthe+BLAST-like+alignment+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BLAT%26%23x2013%3Bthe+BLAST-like+alignment+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000356-Huang1" id="pone.0000356-Huang1"></a>Huang X (1996) An improved sequence assembly program. Genomics  33(1): 21–.  <ul class="find" data-citedArticleID="942800" data-doi="10.1006/geno.1996.0155"><li><a href="http://dx.doi.org/10.1006/geno.1996.0155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+improved+sequence+assembly+program." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+improved+sequence+assembly+program.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.XML" value="75940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.PDF" value="233809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g001.PNG_L" value="47583"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g001.PNG_M" value="60001"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g001.PNG_S" value="5603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g001.TIF" value="176852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g001.PNG_I" value="48995"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g002.PNG_L" value="43102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g002.PNG_M" value="59143"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g002.PNG_S" value="2935"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g002.TIF" value="94594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g002.PNG_I" value="24507"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t001.PNG_L" value="82550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t001.PNG_M" value="29743"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t001.PNG_S" value="2274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t001.TIF" value="449892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t001.PNG_I" value="12814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t002.PNG_L" value="167476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t002.PNG_M" value="58066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t002.PNG_S" value="3283"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t002.TIF" value="1101946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t002.PNG_I" value="24540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t003.PNG_L" value="83427"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t003.PNG_M" value="88097"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t003.PNG_S" value="4196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t003.TIF" value="413794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.t003.PNG_I" value="37940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g003.PNG_L" value="71507"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g003.PNG_M" value="52215"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g003.PNG_S" value="2728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g003.TIF" value="115268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000356.g003.PNG_I" value="20912"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000356&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000356" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000356&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000356&volume=&issue=&title=Extensive Polycistronism and Antisense Transcription in the Mammalian Hox Clusters&author_name=Ga%C3%ABll%20Mainguy%2C%20Jan%20Koster%2C%20Joost%20Woltering%2C%20Hans%20Jansen%2C%20Antony%20Durston&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000356" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000356" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000356" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000356&amp;t=Extensive%20Polycistronism%20and%20Antisense%20Transcription%20in%20the%20Mammalian%20Hox%20Clusters" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000356&title=Extensive%20Polycistronism%20and%20Antisense%20Transcription%20in%20the%20Mammalian%20Hox%20Clusters&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000356&amp;title=Extensive%20Polycistronism%20and%20Antisense%20Transcription%20in%20the%20Mammalian%20Hox%20Clusters" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000356" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000356&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Extensive Polycistronism and Antisense Transcription in the Mammalian Hox Clusters';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000356';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Extensive%20Polycistronism%20and%20Antisense%20Transcription%20in%20the%20Mammalian%20Hox%20Clusters http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000356" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000356" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Conservation+science%22" title="Search for articles in the subject area:'Conservation science'"><div class="flagText">Conservation science</div></a>
              <div data-categoryid="40107" data-articleid="23550"
                   data-categoryname="Conservation science"
                   class="flagImage" title="Flag 'Conservation science' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="23550"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23550"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="20277" data-articleid="23550"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="23550"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Multiple+alignment+calculation%22" title="Search for articles in the subject area:'Multiple alignment calculation'"><div class="flagText">Multiple alignment calculation</div></a>
              <div data-categoryid="20427" data-articleid="23550"
                   data-categoryname="Multiple alignment calculation"
                   class="flagImage" title="Flag 'Multiple alignment calculation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="23550"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="23550"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5810'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1019'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3732&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=10195">reference author error</a><br>Posted by stevepot3333</p>
      <p><a href="/annotation/listThread.action?root=8995">Supporting Online Material 6</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=8989">Supporting Online Material 3, 4, and 5</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
