

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Counting Mutagenized Genomes and Optimizing Genetic Screens in Caenorhabditis elegans</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001117"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001117" />

    <meta name="citation_title" content="Counting Mutagenized Genomes and Optimizing Genetic Screens in Caenorhabditis elegans"/>
    <meta itemprop="name" content="Counting Mutagenized Genomes and Optimizing Genetic Screens in Caenorhabditis elegans"/>

      <meta name="citation_author" content="Shai Shaham"/>
            <meta name="citation_author_institution" content="Laboratory of Developmental Genetics, The Rockefeller University, New York, New York, United States of America"/>

    <meta name="citation_date" content="2007/11/7"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001117.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1117"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Mutagenesis.; citation_author=P Anderson; citation_journal_title=Meth Cell Biol; citation_volume=48; citation_number=1; citation_pages=31-58; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The Genetics of Caenorhabditis elegans.; citation_author=S Brenner; citation_journal_title=Genetics; citation_volume=77; citation_number=2; citation_pages=71-94; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Lethals, Steriles and Deficiencies in a Region of the X Chromosome of Caenorhabditis elegans.; citation_author=P Meneely; citation_author=R Herman; citation_journal_title=Genetics; citation_volume=92; citation_number=3; citation_pages=99-115; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=unc-93(e1500): A Behavioral Mutant of Caenorhabditis elegans That Defines a Gene with a Wild-Type Null Phenotype.; citation_author=I Greenwald; citation_author=HR Horvitz; citation_journal_title=Genetics; citation_volume=96; citation_number=4; citation_pages=147-164; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Essential Genes in the hDf6 Region of chromosome I in Caenorhabditis elegans.; citation_author=AM Howell; citation_author=AM Rose; citation_journal_title=Genetics; citation_volume=126; citation_number=5; citation_pages=583-592; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Genetic Analysis of a Major Segment [LGV(left)] of the Genome of Caenorhabditis elegans.; citation_author=RC Johnsen; citation_author=DL Baillie; citation_journal_title=Genetics; citation_volume=129; citation_number=6; citation_pages=735-752; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Mutation.; citation_author=RC Johnsen; citation_author=DL Baillie; citation_number=7; citation_pages=79-96; citation_date=1997; citation_publisher=Cold Spring Harbor Press; " />
      <meta name="citation_reference" content="citation_title=In vivo mutagenesis by O-methylguanine built into a unique site in a viral genome.; citation_author=EL Loechler; citation_author=CL Green; citation_author=JM Essigmann; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=81; citation_number=8; citation_pages=6271-6275; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Base-pairing Properties of O-Methylguanine in Template DNA during in Vitro DNA Replication.; citation_author=ET Snow; citation_author=RS Foote; citation_author=S Mitra; citation_journal_title=J Biol Chem; citation_volume=259; citation_number=9; citation_pages=8095-8100; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Mutational specificity of ethyl methanesulfonate in excision-repair-proficient and -deficient strains of Drosophila melanogaster.; citation_author=A Pastink; citation_author=E Heemskerk; citation_author=MJM Nivard; citation_author=CJ van Vliet; citation_author=EW Vogel; citation_journal_title=Mol Gen Genet; citation_volume=229; citation_number=10; citation_pages=213-218; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Spectrum of Chemically Induced Mutations from a Large-Scale Reverse-Genetic Screen in Arabidopsis.; citation_author=EA Greene; citation_author=CA Codomo; citation_author=NE Taylor; citation_author=JG Henikoff; citation_author=BJ Till; citation_author=SH Reynolds; citation_author=LC Enns; citation_author=C Burtner; citation_author=JE Johnson; citation_author=AR Odden; citation_author=L Comai; citation_author=S Henikoff; citation_journal_title=Genetics; citation_volume=164; citation_number=11; citation_pages=731-740; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Sequence of an unusually large protein implicated in regulation of myosin activity in C. elegans.; citation_author=GM Benian; citation_author=JE Kiff; citation_author=N Neckelmann; citation_author=DG Moerman; citation_author=RH Waterston; citation_journal_title=Nature; citation_volume=342; citation_number=12; citation_pages=45-50; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The C. elegans Heterochronic Gene lin-4 Encodes Small RNAs with Antisense Complementarity to lin-14.; citation_author=RC Lee; citation_author=RL Feinbaum; citation_author=V Ambros; citation_journal_title=Cell; citation_volume=75; citation_number=13; citation_pages=843-854; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Genes Required for the Engulfment of Cell Corpses during Programmed Cell Death in Caenorhabditis elegans.; citation_author=RE Ellis; citation_author=DM Jacobson; citation_author=HR Horvitz; citation_journal_title=Genetics; citation_volume=129; citation_number=14; citation_pages=79-94; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The Art and Design of Genetic Screens: Caenorhabditis elegans.; citation_author=EM Jorgensen; citation_author=SE Mango; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=15; citation_pages=356-369; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001117" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Counting Mutagenized Genomes and Optimizing Genetic Screens in Caenorhabditis elegans"/>
    <meta name="twitter:description" content="In genetic screens, the number of mutagenized gametes examined is an important parameter for evaluating screen progress, the number of genes of a given mutable phenotype, gene size, cost, and labor. Since genetic screens often entail examination of thousands or tens of thousands of animals, strategies for optimizing genetics screens are important for minimizing effort while maximizing the number of mutagenized gametes examined. To date, such strategies have not been described for genetic screens in the nematode Caenorhabditis elegans. Here we review general principles of genetic screens in C. elegans, and use a modified binomial strategy to obtain a general expression for the number of mutagenized gametes examined in a genetic screen. We use this expression to calculate optimal screening parameters for a large range of genetic screen types. In addition, we developed a simple online genetic-screen-optimization tool that can be used independently of this paper. Our results demonstrate that choosing the optimal F2-to-F1 screening ratio can significantly improve screen efficiency."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001117.g005"/>

  <meta property="og:title" content="Counting Mutagenized Genomes and Optimizing Genetic Screens in Caenorhabditis elegans" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001117" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1325'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2561'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7409&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001117">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001117" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001117&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Counting Mutagenized Genomes and Optimizing Genetic Screens in <i>Caenorhabditis elegans</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shai Shaham
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:shaham@rockefeller.edu">shaham@rockefeller.edu</a></p>

                <p>Affiliation:
                  Laboratory of Developmental Genetics, The Rockefeller University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 07, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001117</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001117" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001117" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001117" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001117" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001117-g001" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001117-t001" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001117-g002" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001117-g003" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001117-g004" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001117-g005" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001117">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001117" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>In genetic screens, the number of mutagenized gametes examined is an important parameter for evaluating screen progress, the number of genes of a given mutable phenotype, gene size, cost, and labor. Since genetic screens often entail examination of thousands or tens of thousands of animals, strategies for optimizing genetics screens are important for minimizing effort while maximizing the number of mutagenized gametes examined. To date, such strategies have not been described for genetic screens in the nematode <em>Caenorhabditis elegans</em>. Here we review general principles of genetic screens in <em>C. elegans</em>, and use a modified binomial strategy to obtain a general expression for the number of mutagenized gametes examined in a genetic screen. We use this expression to calculate optimal screening parameters for a large range of genetic screen types. In addition, we developed a simple online genetic-screen-optimization tool that can be used independently of this paper. Our results demonstrate that choosing the optimal <em>F2</em>-to-<em>F1</em> screening ratio can significantly improve screen efficiency.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Shaham S (2007) Counting Mutagenized Genomes and Optimizing Genetic Screens in <em>Caenorhabditis elegans</em>. PLoS ONE 2(11):
          e1117.
            doi:10.1371/journal.pone.0001117</p><p><strong>Academic Editor: </strong>Rodolfo Aramayo, Texas A&amp;M University, United States of America</p><p><strong>Received:</strong> August 6, 2007; <strong>Accepted:</strong> October 15, 2007; <strong>Published:</strong> November 7, 2007</p><p><strong>Copyright:</strong> © 2007 Shai Shaham. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a grant from the NIH to SS. SS is a Klingenstein fellow in the neurosciences and a Monique Weill-Caulier Scholar.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The identification of mutant animals is usually the first step in the genetic analysis of a biological process. In animals amenable to genetic study, such mutants are commonly identified by random mutagenesis, followed by screening for the trait under study. The mutation frequencies conferred by different mutagens in different organisms have been extensively documented <a href="#pone.0001117-Anderson1">[1]</a>. Furthermore, for a given mutagen dose, the probability of identifying a mutant in a specific gene is a function of the number of mutagenized gametes (or haploid genomes) examined. Typically, mutagens are employed at doses inducing 10–100 mutations per haploid genome. At such mutagenesis frequencies the number of induced mutations is orders of magnitude above the accumulation of spontaneous mutations, yet is not too large as to preclude organismal viability <a href="#pone.0001117-Anderson1">[1]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>For screens in the nematode <em>Caenorhabditis elegans</em>, typical mutagenesis regimens require isolation and examination of thousands of animals to approach saturation. Depending on the screening approach, examination of such numbers of animals can be quite labor intensive. In these situations, therefore, it is advantageous to identify optimal screening strategies that maximize the number of genomes screened, while minimizing the work involved. Indeed, as we show here, choosing a reasonable, yet suboptimal ratio of <em>F</em>2 to <em>F</em>1 animals, can double or triple the work involved in screening a given number of mutagenized <em>F</em>1 animals, as compared to screening using optimal parameters. Thus, suboptimal screen strategies may unnecessarily prolong screens, and use up excess reagents.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Currently, a description of how to optimize genetic screens in <em>C. elegans</em> is not available. We therefore set out to develop a general algorithm for optimizing genetic screens in this organism. Here we examine such an optimization approach. We begin by reviewing the variables affecting the number of mutagenized haploid genomes needed to achieve saturation screening. We then derive a general expression, valid for most genetic screening approaches used in <em>C. elegans</em>, for counting the number of mutagenized <em>F</em>1 animals examined in a genetic screen. The expression we derive is, as we show, essentially independent of the total number of animals scored during the course of the screen, and is independent of variations in locus mutability. Although the number of mutagenized <em>F</em>1 animals is often approximated by the Poisson distribution, we demonstrate that for at least one major screen class, the Poisson approximation leads to large errors.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Using the generalized expression described above, we solve optimization equations to maximize the efficiencies of two common genetic screen types. Our solutions reveal that an optimal <em>F</em>2-to-<em>F</em>1 screening ratio always exists for these screens, and that this ratio is dependent neither on the total number of animals scored, nor on the number of mutagenized gametes examined. Rather, the optimal screening ratio depends only on the type of mutation sought (<em>e.g.</em> recessive, dominant) and on the relative work involved in picking and scoring <em>F</em>1 and <em>F</em>2 animals. We use our results to delineate a simple algorithm for setting up and following the progress of genetic screens in <em>C. elegans</em>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Although valid for <em>C. elegans</em>, our results can be simply extended to genetic screens in other organisms.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>(Readers not interested in the mathematical exposition that follows can skip to the last section of the Discussion, which describes a simple algorithm for optimizing genetic screens in <em>C. elegans</em>).</p>

<h4>The Probability of Identifying a Mutant <em>F</em>1 Animal</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The nematode <em>C. elegans</em> is a self-fertilizing hermaphrodite that, at least under laboratory settings, rarely uses males for reproduction <a href="#pone.0001117-Brenner1">[2]</a>. Genetic screens in <em>C. elegans</em> generally follow a scheme similar to the one outlined in <a href="#pone-0001117-g001">Figure 1A</a>. First, animals (the <em>P</em>0 generation) are exposed to a mutagen inducing mutations in sperm and oocytes <a href="#pone.0001117-Anderson1">[1]</a>. The mutagen ethyl methanesulfonate (EMS), for example, when used at a concentration of 50 mM, induces loss-of-function mutations in a given <em>C. elegans</em> gene at an average frequency of one every 2,500 mutagenized <em>P</em>0 gametes <a href="#pone.0001117-Anderson1">[1]</a>–<a href="#pone.0001117-Johnsen1">[6]</a>. Generally, fourth-larval-stage (L4) or young-adult animals are used as mutagenesis targets to maximize the probability that the mutations generated are derived from independent events <a href="#pone.0001117-Anderson1">[1]</a>.</p>
<div class="figure" id="pone-0001117-g001"><div class="img"><a name="pone-0001117-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001117.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001117.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>General genetic screening scheme in <em>C. elegans</em>.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) <em>P</em>0 animals are mutagenized, and allowed to self-fertilize to produce <em>F</em>1 animals. To identify recessive mutations, <em>F</em>1 animals are allowed to self-fertilize to produce the <em>F</em>2 generation. In this paper we consider the case of <em>n F</em>1 animals giving rise to <em>m F</em>2 animals. (B) Plots describing the probability, π(<em>n</em>), that among <em>n F</em>1 animals screened, following mutagenesis by EMS (<em>r</em> = 1,250), will be found at least one <em>F</em>1 animal heterozygous for a loss-of-function mutation in a gene of interest. The parameter <em>a</em> is as defined in the text. The plots were generated by the program Mathematica 5.0 (Wolfram Research), using <em>n</em> as a continuous variable.</p>
<span>doi:10.1371/journal.pone.0001117.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Next, <em>P</em>0 animals are allowed to self-fertilize, to produce <em>F</em>1 progeny. If mutations in either copy of a gene under consideration can be revealed in subsequent analysis, and if <em>n F</em>1 animals are examined, then the number of haploid genomes screened, defined as the number of <em>P</em>0 gametes examined, is given by 2<em>n</em>. More generally, however, the number of haploid genomes examined is given by <em>a</em> times <em>n</em> (<em>an</em>) where <em>a</em> takes on the value of either 1 or 2. For example, if a mutation in a gene of interest is suspected to be lethal or sterile, attempts to induce it on a marked chromosome, opposite a balancer chromosome, are often undertaken <a href="#pone.0001117-Johnsen2">[7]</a>. For such a screen, only mutations induced on the marked, non-balancer chromosome are sought, and <em>a</em> = 1.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Following mutagenesis, the probability of finding at least one <em>F</em>1 animal heterozygous for a mutation in a specific gene of interest is influenced by a number of parameters, and can often vary greatly from gene to gene. Thus, for example, smaller genes may be less likely to be hit by mutagen. Furthermore, some mutagens, such as EMS, preferentially alter certain nucleotides <a href="#pone.0001117-Loechler1">[8]</a>–<a href="#pone.0001117-Greene1">[11]</a>, thus nucleotide content of a gene may also affect its mutation frequency. It is also possible that chromatin structure and packing of DNA in the environs of a gene may play a role in its mutagenesis frequency. Thus, obtaining an exact estimate for the number of <em>F</em>1 animals to be screened is difficult, and screens are generally considered near saturation when multiple alleles of a given gene have been identified.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Nonetheless, in many instances, assuming an average mutagenesis frequency can lead to useful estimates regarding screen progress, and deviations from these estimates can often hint at unique features of a gene, such as unusually large or small size <a href="#pone.0001117-Anderson1">[1]</a>, <a href="#pone.0001117-Benian1">[12]</a>, <a href="#pone.0001117-Lee1">[13]</a>. To calculate such average probabilities we can proceed as follows.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>The probability of finding at least one <em>F</em>1 animal heterozygous for a mutation in a gene of interest among <em>n F</em>1 progeny of mutagenized <em>P</em>0 animals, assuming <em>a</em> = 2, is 1−(probability no <em>F</em>1 animals carry a mutation)−(probability an <em>F</em>1 animal carries independent mutations in each copy of a gene of interest). For common mutagenesis frequencies, the last term is exceedingly small and can be ignored without significant loss of accuracy. Thus, the probability of finding a heterozygous <em>F</em>1 animal, π(<em>n</em>), is given by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where 1/r is the fraction of <em>F</em>1 animals expected, on average, to carry a mutation in a gene of interest. As described above, <em>r</em> = 1,250 for loss-of-function mutations obtained by EMS mutagenesis.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>Using equation (1), we can calculate how many <em>F</em>1 animals should be examined, on average, to obtain at least one animal carrying a loss-of-function mutation in a gene of interest. For example, to achieve a 95% probability (π(<em>n</em>) = 0.95) of obtaining an <em>F</em>1 carrier, we subtract 1 from both sides of equation (1), divide both sides by −1, take the logarithm of both sides of the equation, and rearrange to obtain n = ln0.05/ln(1−1/r). For large values of <em>r</em>, as is the case for most mutagenesis regimes, ln(1−1/<em>r</em>)≈−1/<em>r</em>. We thus obtain the expression <em>n</em>/<em>r</em> = −ln0.05, or <em>n</em>/<em>r</em>≈3, a commonly used result (see also <a href="#s4">Materials and Methods</a> section of ref. 14).</p>
<a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>A similar calculation for <em>a</em> = 1 (see example above) is more complex if the <em>F</em>1 animals, in which the mutation of interest is induced on the balancer chromosome, cannot be readily distinguished from those in which the mutation is induced on the marked chromosome, and we proceed as follows. Of a collection of <em>n F</em>1 animals, assume that <em>q</em> are heterozygous for mutations in a given gene. The odds that this is the case are given by the probability of obtaining <em>q</em> heterozygotes, (1/<em>r</em>)<em><sup>q</sup></em>, multiplied by the probability of the remaining <em>n</em>−<em>q F</em>1 animals being non-carriers, (1−1/<em>r</em>)<em><sup>n−q</sup></em>, multiplied by the number of ways such an arrangement can occur, given by the binomial coefficient <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e002&amp;representation=PNG" class="inline-graphic"></span>. The probability that at least one of the <em>q</em> animals carries the mutation on the marked chromosome is given by 1−(1/2)<em><sup>q</sup></em>, where (1/2)<em><sup>q</sup></em> is the probability that all <em>q</em> heterozygotes carry the mutation on the balancer chromosome. Thus, the probability that of <em>n F</em>1 animals, <em>q</em> are heterozygous for mutations in a given gene, and at least one of these <em>q</em> animals carries the mutation on the marked chromosome is <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e003&amp;representation=PNG" class="inline-graphic"></span>. Therefore, to obtain the probability that at least one informative <em>F</em>1 animal is present among the <em>n F</em>1 animals picked, we sum the individual probabilities for each value of <em>q</em> to obtain<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br></p>
<a id="article1.body1.sec2.sec1.p8" name="article1.body1.sec2.sec1.p8"></a><p>For large <em>n</em> and <em>r</em>, equation (2) can be accurately approximated using the Poisson distribution (see <a href="#s4">Materials and Methods</a>), yielding:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br></p>
<a id="article1.body1.sec2.sec1.p9" name="article1.body1.sec2.sec1.p9"></a><p>For a given <em>n</em> it is expected that π<em><sub>a</sub></em><sub> = 2</sub>(<em>n</em>)&gt; π<em><sub>a</sub></em><sub> = 1</sub>(<em>n</em>), a result illustrated in <a href="#pone-0001117-g001">Figure 1B</a> where the probabilities π(<em>n</em>) are plotted for values of <em>n</em> between 1 and 20,000, assuming <em>r</em> = 1,250.</p>


<h4>Counting F1 Animals Screened- an Example</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The discussion of the previous section suggests that gene-to-gene variations in mutability make it difficult to predict precisely how many mutagenized <em>F</em>1 animals must be sifted through to identify a mutant of choice. However, the question arises as to whether an optimal screening strategy, <em>independent</em> of locus mutability, might exist, and if so, what are its properties.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To begin to address this issue, we must first determine how many mutagenized <em>F</em>1 animals have been examined during a genetic screen. In this section and the following sections we make a distinction between the number of <em>F</em>1 animals <em>picked</em> for analysis, <em>n</em>, and the number of <em>F</em>1 animals whose mutation content has actually been <em>examined</em>, <em>n<sub>act</sub></em>. If the mutation being sought is predicted to behave in a simple dominant fashion, then counting how many <em>F</em>1 animals were examined is trivial, and is precisely equal to the number of <em>F</em>1 animals picked, that is, <em>n<sub>act</sub></em> = <em>n</em> (<em>e.g.</em> <a href="#pone-0001117-t001">Table 1</a>). However, in most schemes, as illustrated in <a href="#pone-0001117-g001">Figure 1A</a>, recessive mutations are sought, whose presence cannot be discerned in the <em>F</em>1 generation. Thus, <em>F</em>1 animals are allowed to self-fertilize, to produce <em>F</em>2 animals, among which may be identified homozygous mutants in the gene of interest, displaying a scorable phenotype. In this case it is generally the case that <em>n<sub>act</sub></em>≠<em>n</em>, and <em>n<sub>act</sub></em> must be used instead of <em>n</em> in equations (1), (2), and (3) to obtain π(<em>n</em>).</p>
<div class="figure" id="pone-0001117-t001"><div class="img"><a name="pone-0001117-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001117.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001117.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Optimal Numbers of F1 and F2 Animals Required to Screen 5,000 Mutagenized F1 Animals for Different Screen Parameters.</span></strong></p><span>doi:10.1371/journal.pone.0001117.t001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>To obtain a general expression for <em>n<sub>act</sub></em>, we begin by considering, as an example, a genetic screen in which all <em>n F</em>1 animals are picked to a single plate, from which <em>m F</em>2 animals are then scored for the mutant phenotype. We assume that all <em>F</em>1 animals produce approximately the same number of <em>F</em>2 animals. Indeed, for standard EMS screens in <em>C. elegans</em>, only about 5% of <em>F</em>1 animals have greatly reduced fertility, and fewer than 1% of <em>F</em>1 animals have reduced fertility using one common protocol for trimethylpsoralen and ultraviolet light mutagenesis (S.S., unpublished results). Furthermore, for many mutagenesis schemes, only healthy <em>F</em>1 animals are picked for subsequent screening, further reducing the number of animals producing low brood counts.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>For such a screen design, <em>n<sub>act</sub></em> can be approximated fairly accurately in two limiting cases. First, we consider a screen in which <em>m</em>≫<em>n</em>. In this case, many <em>F</em>2 progeny have been scored for each <em>F</em>1 animal, making it very likely that the mutation content of all <em>F</em>1 animals has been established. Therefore, <em>n<sub>act</sub></em>≈<em>n</em>.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Second, we consider the case for which <em>n</em>≫<em>m</em>. Here, the likelihood that two or more of the <em>F</em>2 animals scored derive from the same <em>F</em>1 animal is small. Thus, <em>n<sub>act</sub></em>≈<em>m</em>(1−<em>p</em>), where <em>p</em>, is the probability that a scored <em>F</em>2 has failed to reveal whether its <em>F</em>1 parent had the mutation of interest. In the case of simple screening schemes involving recessive mutations, <em>p</em> = 3/4.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>Although these limiting cases have important uses, genetic screens that involve manually picking <em>F</em>1 animals, <em>F</em>2 animals, or both, may fail to satisfy the limiting conditions considered above. To calculate <em>n<sub>act</sub></em> for the general case of this example, we proceed as follows.</p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>The probability that among the <em>m</em> scored <em>F</em>2 animals are represented <em>q</em> progeny of a particular <em>F</em>1 animal is given by the binomial term<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e006&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>where (1/<em>n</em>)<em><sup>q</sup></em> is the probability of scoring <em>q</em> progeny of a particular <em>F</em>1 animal, (1−1/<em>n</em>)<em><sup>m−q</sup></em> is the probability of the remaining scored <em>F</em>2 animals not being progeny of the particular <em>F</em>1 animal under consideration, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e007&amp;representation=PNG" class="inline-graphic"></span> representing the number of ways such a combination can be picked.</p>
<a id="article1.body1.sec2.sec2.p8" name="article1.body1.sec2.sec2.p8"></a><p>The probability that at least one of the <em>q</em> scored <em>F</em>2 animals is informative about the presence of a mutation of interest in the particular <em>F</em>1 animal under consideration, is given by 1−<em>p<sup>q</sup></em>, where <em>p</em> is the probability of the <em>F</em>2 animal not being informative. Therefore, by analogy to the calculation in the previous section, the probability that at least one informative <em>F</em>2 progeny of a particular <em>F</em>1 animal is found among the <em>m F</em>2 animals scored is given by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e008&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br></p>
<a id="article1.body1.sec2.sec2.p9" name="article1.body1.sec2.sec2.p9"></a><p>In this paper we aim to describe optimal genetic screening strategies, and are, in general, interested in the optimal screening ratio of <em>F</em>2 to <em>F</em>1 animals, <em>y</em> = <em>m</em>/<em>n</em>. It will become useful, therefore, to represent the functions <em>B</em>(<em>q</em>,<em>m</em>,<em>n</em>) and <em>P</em>(<em>m</em>,<em>n</em>) as functions of <em>y</em> and <em>N</em> = <em>n</em>+<em>m</em>, the total number of animals picked in the screen. By making the appropriate substitutions we can rewrite these functions as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e009&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br></p>
<a id="article1.body1.sec2.sec2.p10" name="article1.body1.sec2.sec2.p10"></a><p>and<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e010&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>The number of <em>F</em>1 animals examined for possession of a mutation in a specific gene, then, is given by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br></p>
<a id="article1.body1.sec2.sec2.p11" name="article1.body1.sec2.sec2.p11"></a><p>In <a href="#pone-0001117-g002">Figure 2A</a> we plot the quantity <em>n<sub>act</sub></em>/<em>N</em> as a function of log<sub>10</sub><em>y</em> for a simple recessive mutation with <em>p</em> = 0.75 and <em>N</em> = 1000. As expected, for <em>n</em>≫<em>m</em>, <em>n<sub>act</sub></em>/<em>N</em> is asymptotic to the curve <em>m</em>/(4<em>N</em>) = <em>y</em>/(4(1+<em>y</em>)) (red line). For <em>m</em>≫<em>n</em>, <em>n<sub>act</sub></em>/<em>N</em> is asymptotic to the curve <em>n/N</em> = 1/(1+<em>y</em>) (blue line). As is evident from the figure, for <em>p</em> = 0.75, the asymptotic curves overestimate <em>n<sub>act</sub></em> by only 5% or less for <em>n</em> and <em>m</em> such that <em>y</em> = <em>m</em>/<em>n</em>≤0.4 or <em>m/n</em>≥12.2. However, in between these values, a very large error, that may exceed 100% of the true value of <em>n<sub>act</sub></em>, can occur (not shown), justifying the more detailed analysis presented in this section. For large <em>m</em> and <em>n</em>, equations (7) and (8) are very closely approximated using the Poisson distribution (see <a href="#s4">Materials and Methods</a>) to obtain<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br></p>
<a id="article1.body1.sec2.sec2.p12" name="article1.body1.sec2.sec2.p12"></a><p>and<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br></p>
<div class="figure" id="pone-0001117-g002"><div class="img"><a name="pone-0001117-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001117.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001117.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The effects of different parameters on the number of haploid genomes screened.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A–C) Plots assuming all <em>F</em>1 animals are placed together on a single plate. <em>p</em> = 0.75, unless otherwise indicated. All plots were generated using the program Mathematica 5.0 (Wolfram Research). (A) Black line, graph of <em>n<sub>act</sub></em>/<em>N vs.</em> log<sub>10</sub><em>y</em> for <em>N</em> = 1,000. Blue and red lines, graphs of asymptotes and their equations. (B) Graphs of <em>n<sub>act</sub></em>/<em>N vs.</em> log<sub>10</sub><em>y</em>. Different colors indicate <em>n<sub>act</sub></em>/<em>N</em> for different specified values of <em>N</em>. Black line, Poisson approximation, colored lines, exact solutions. Inset, magnification of the graph for the region of log<sub>10</sub><em>y</em> between 0.3 and 0.5. For each value of <em>N</em>, <em>y</em> can only take on values such that 1/(<em>N</em>−1)≤<em>y</em>≤<em>N</em>−1. Furthermore, although for illustration purposes we have drawn the curves as continuous, <em>y</em> is <em>not</em> a continuous variable, and treating it as such only works for large <em>N</em>. This is most obvious for <em>N</em> = 10 where <em>y</em> can only take on the values 1/9, 1/4, 3/7, 2/3, 1, 3/2, 7/3, 4, and 9. (C) Graphs of <em>n<sub>act</sub></em>/<em>N vs.</em> log<sub>10</sub><em>y</em> for varying values of <em>p</em>, as defined in the text. Graphed using the Poisson approximation. (D) Graphs depicting the fractional error incurred when using the Poisson approximation to estimate <em>n<sub>act</sub></em>/<em>N</em> for screens in which one <em>F</em>1 animal is plated per plate. Although graphs are continuous, only integer values of <em>y</em> are relevant. Also note that the smallest allowable value of <em>y</em> is chosen so that at least 1 <em>F</em>2 animal is chosen per plate.</p>
<span>doi:10.1371/journal.pone.0001117.g002</span></div><a id="article1.body1.sec2.sec2.p13" name="article1.body1.sec2.sec2.p13"></a><p><a href="#pone-0001117-g002">Figure 2B</a> demonstrates this graphically, showing the tight agreement between equation (8) and the Poisson approximated equation (10) (black line), for values of <em>N</em> as small as 10.</p>
<a id="article1.body1.sec2.sec2.p14" name="article1.body1.sec2.sec2.p14"></a><p><a href="#pone-0001117-g002">Figure 2B</a> also demonstrates the advantage of using the scaled parameter <em>n<sub>act</sub></em>/<em>N</em> in our analysis. Although equation (8) shows that <em>n<sub>act</sub></em>/<em>N</em> is dependent on the value of <em>N</em>, <a href="#pone-0001117-g002">Figure 2B</a> reveals that this dependence is very weak. Indeed, equation (10) shows that for large <em>m</em> and <em>n</em>, <em>n<sub>act</sub></em>/<em>N</em> becomes entirely independent of <em>N</em>, making <em>n<sub>act</sub></em>/<em>N</em> useful for the analysis of most screens.</p>


<h4>Single-Plate Screens for Mutants That Are Not Strictly Recessive</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Although fully penetrant recessive mutations for which <em>p</em> = 0.75 are the most commonly sought mutations <a href="#pone.0001117-Jorgensen1">[15]</a>, other screening modes are frequently used for which <em>p</em>≠0.75. For example, if the <em>F</em>1 animals are heterozygous for a balancer chromosome and an <em>un</em>marked homologous chromosome, such that animals homozygous for the balancer are dead, or easily identifiable, then <em>p</em> becomes 2/3 = 0.67 for a fully penetrant recessive mutation. As another example, dominant mutations that are rescued by a wild-type maternal genotype cannot be isolated in the <em>F</em>1 generation, but can be sought among <em>F</em>2 animals. For strictly dominant mutations of this type, <em>p</em> = 0.25.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The case <em>p</em> = 0 occurs when <em>F</em>1 animals are heterozygous for a balancer and a marked chromosome, with the mutation of interest induced on the latter. In this case, only marked <em>F</em>2 animals are scored, and these should all be homozygous for the mutation of interest. The condition <em>p</em> = 0 also holds for rare screens where dominant maternal-effect mutants are sought. In such screens, heterozygous <em>F</em>1 animals do not have the phenotype of interest, but all of their progeny do.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The condition <em>p</em> = 0 arises frequently in genetic screens of haploid organisms. In this case, <em>n<sub>act</sub></em>/<em>N</em> is a measure of the representation of an initial pool of mutagenized organisms in the progeny that have been examined.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>In <a href="#pone-0001117-g002">Figure 2C</a> we plot curves of <em>n<sub>act</sub></em>/<em>N v</em>s. log<sub>10</sub><em>y</em> for a number of possible values of <em>p</em>. The curves agree with the expectation that the more informative an <em>F</em>2 animal is about the mutation state of its <em>F</em>1 parent (<em>i.e.</em>, the smaller <em>p</em> is), the fewer <em>F</em>2 animals must be examined to achieve a specific value of <em>n<sub>act</sub></em>/<em>N</em>.</p>


<h4>The General Expression for the Number of <em>F</em>1 Animals Screened</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The analysis described in the preceding two sections holds for the specific case in which all <em>F</em>1 animals are placed on a single plate, from which <em>F</em>2 progeny are sampled. However, usually, it is necessary for <em>F</em>1 animals to be placed individually, or in groups, on multiple plates. <em>F</em>2 animals are then drawn from each plate. Individual plating of <em>F</em>1 animals is a particularly common scheme that is utilized if the expected mutation is lethal, and heterozygous <em>F</em>2 siblings need to be recovered, or in a situation where males are present in the population and mating between <em>F</em>1 animals must be avoided. Should <em>F</em>1 plating strategy affect <em>n<sub>act</sub></em>? Consider the case where single <em>F</em>1 animals are placed on individual plates, and equal numbers of <em>F</em>2 progeny are drawn from each plate. Because we know for certain that in such a scheme <em>F</em>2 animals scored are derived from every <em>F</em>1 animal picked, it is predicted that for a given <em>y</em>, <em>n<sub>act</sub></em> should be greater than in a scheme in which all <em>F</em>1 animals were plated on a single plate, where some <em>F</em>1 animals may not be sampled in the <em>F</em>2 generation. Thus, for the same number of <em>F</em>1 and <em>F</em>2 animals, <em>n<sub>act</sub></em> will indeed be dependent on plating strategy.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>To calculate the general expression for <em>n<sub>act</sub></em> explicitly, we let ν equal the number of <em>F</em>1 animals picked to a plate, and μ equal the number of <em>F</em>2 animals scored per plate. In general, ν and μ can be different for every plate. <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e014&amp;representation=PNG" class="inline-graphic"></span>, the actual number of <em>F</em>1 animals examined on the <em>i</em>th plate, is given, as in equation (8), by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e015&amp;representation=PNG" class="inline-graphic"><span class="note">(11)</span></span><br>where<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e016&amp;representation=PNG" class="inline-graphic"><span class="note">(12)</span></span><br>The general expression for <em>n<sub>act</sub></em>/<em>N</em>, is, therefore, obtained by summing the actual number of <em>F</em>1 animals examined over all plates, or<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e017&amp;representation=PNG" class="inline-graphic"><span class="note">(13)</span></span><br></p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Equation (13) is valid for essentially every type of genetic screen, involving any plating strategy. The equation can be significantly simplified if we assume that the same number of <em>F</em>1 and <em>F</em>2 animals are plated and scored per plate. In this case, ν<em><sup>i</sup></em>, μ<em><sup>i</sup></em>, and β(<em>q</em>, ν<em><sup>i</sup></em>, μ<em><sup>i</sup></em>) are identical for each plate, and equation (13) can be written as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e018&amp;representation=PNG" class="inline-graphic"><span class="note">(14)</span></span><br>where <em>k</em> is the number of plates examined.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>Equation (14) can be expressed in terms of <em>y</em> as follows. To express μ as a function of <em>y</em> we note that <em>y</em> = <em>m</em>/<em>n</em> = <em>k</em>μ/(<em>k</em>ν) = μ/ν. Rearranging terms yields μ = ν<em>y</em>. Also, <em>k</em>ν/<em>N</em> = <em>n</em>/(<em>n</em>+<em>m</em>) = 1/(1+<em>m</em>/<em>n</em>) = 1/(1+<em>y</em>), yielding<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e019&amp;representation=PNG" class="inline-graphic"><span class="note">(15)</span></span><br></p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>Four points regarding equation (15) are of note. First, for the common plating strategy of one <em>F</em>1 animal/plate, equation (15) can be reduced to<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e020&amp;representation=PNG" class="inline-graphic"><span class="note">(16)</span></span><br>an equation of considerable practical value. Furthermore, as expected, for a single plate with all <em>F</em>1 animals on the same plate, ν becomes <em>n</em>, and β(<em>q</em>,ν,<em>y</em>) becomes <em>B</em>(<em>q</em>,<em>y</em>,<em>N</em>), as in equation (8).</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>Second, unlike equations (5) and (8), equation (15), even for large <em>N</em>, is <em>not</em> well-approximated by the Poisson distribution if a small number of <em>F</em>1 animals is plated per plate, which is generally the case for clonal screens. In <a href="#pone-0001117-g002">Figure 2D</a> we plot the error introduced in <em>n<sub>act</sub></em>/<em>N</em> by using the Poisson approximation for different values of <em>p</em> for the case of one <em>F</em>1 animal per plate. Note that for <em>p</em> = 0.25, the error can be nearly 30% off the exact value. Indeed, it can be shown that the fractional error, [(<em>n<sub>act</sub></em>/<em>N</em>)<em><sub>exact</sub></em>−(<em>n<sub>act</sub></em>/<em>N</em>)<em><sub>Poisson</sub></em>]/(<em>n<sub>act</sub></em>/<em>N</em>)<em><sub>exact</sub></em>, plotted in <a href="#pone-0001117-g002">Figure 2D</a> approaches a maximal value of <em>p</em>−<em>e</em><sup>(<em>p</em>−1)</sup>/(<em>p</em>−1) as <em>y</em>→1 (see <a href="#s4">Materials and Methods</a>).</p>
<a id="article1.body1.sec2.sec4.p7" name="article1.body1.sec2.sec4.p7"></a><p>Third, equation (15) reveals that <em>n<sub>act</sub></em>/<em>N</em> is independent of <em>N</em>, although for a given <em>N</em> only values of μ+ν that are divisors of <em>N</em> are possible.</p>
<a id="article1.body1.sec2.sec4.p8" name="article1.body1.sec2.sec4.p8"></a><p>Fourth, for a given <em>N</em>, the larger the number of <em>F</em>1 animals per plate (<em>k</em>Ø1), the better is equation (15) approximated by equation (10). Simulations for different ratios of <em>n</em> to <em>k</em> reveal that for <em>n/k</em>≥30 equation 10 gives an excellent estimate of <em>n<sub>act</sub></em> (error&lt;5%; data not shown).</p>


<h4>Optimizing Genetic Screens- Preliminaries</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The results described in the previous sections provide us with the appropriate tools to consider how to optimize genetic screens in <em>C. elegans</em>. In general, a measure of screen efficiency should take into account the amount of work performed in a screen as well as the total number of mutagenized <em>F</em>1 animals examined, <em>n<sub>act</sub></em>. Work can be defined in a number of ways. Here, we will generally define work as the amount of time spent picking and scoring animals. Alternatively, work can be a measure of the total amount of reagents needed for the screen, <em>etc.</em>, and much of the analysis that follows would still be valid using this definition. Regardless of the precise definition of work used, the work expended in a screen must be of the form γ<em>n</em>+α<em>m</em>, where γ and α represent work per animal and have values between 0 and ∞. If we measure work in such a way that γ = α = 1 defines a unit of work, we can write the total work expended as <em>W</em> = γ<em>n</em>+α<em>m</em>. Given this definition, we now propose to define the efficiency of a genetic screen as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e021&amp;representation=PNG" class="inline-graphic"><span class="note">(17)</span></span><br>That is, the efficiency is a direct measure of the number of mutagenized <em>F</em>1 animals examined per unit of work. As we will demonstrate below, this definition allows us to make quantitative estimates of the optimal screening ratio, <em>y</em>, for use in a broad range of genetic screen types.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We note that for screens in which a unit of work is defined as the time spent picking a single animal to a plate, and in which equal work is expended to pick <em>F</em>1 and pick and score <em>F</em>2 animals, we define γ = α = 1; then, <em>W</em> = <em>N</em>, and ε = <em>n<sub>act</sub></em>/<em>N</em>, which is the parameter we have used throughout this paper (<em>e.g</em>. <a href="#pone-0001117-g002">Figure 2A</a>).</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>We can now pose the optimization problem for genetic screens as follows: what is the <em>F</em>2-to-<em>F</em>1 screening ratio that minimizes the amount of work needed to screen a required number of mutagenized <em>F</em>1 animals, <em>n<sub>req</sub></em>. Before we address the problem, it is worth considering whether the problem is itself a reasonable one. That is, is it reasonable to assume that a minimal value of <em>W</em> exists for every genetic screen? An examination of <a href="#pone-0001117-g002">Figure 2A</a> shows that <em>n<sub>act</sub></em>/<em>N</em> has a maximum with respect to <em>y</em>, suggesting that at least for the specific case of γ = α = 1, an optimal <em>F</em>2-to-<em>F</em>1 screening ratio, <em>y<sub>max</sub></em>, exists. Extending this result, it can be demonstrated (see next section) that a maximum indeed exists for every value of γ and α&gt;0.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Consider <a href="#pone-0001117-g003">Figure 3</a>, where we have plotted <em>n<sub>act</sub></em> as a function of log<sub>10</sub><em>y</em> for three different values of <em>W</em> where γ = α = 1. The minimal amount of work, <em>W<sub>min</sub></em>, required to screen 5,000 mutagenized <em>F</em>1 animals, <em>n<sub>req</sub></em> = 5,000, occurs at <em>W</em> = 37,642. Values of <em>W</em> smaller than this will never achieve <em>n<sub>req</sub></em>, while values of <em>W</em> greater than 37,642 do not minimize <em>W</em>, by definition. We can therefore formulate the following general criteria: <em>W<sub>min</sub></em> is the value of <em>W</em> satisfying the two conditions, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e022&amp;representation=PNG" class="inline-graphic"></span>, and <em>n<sub>act</sub></em>(<em>W<sub>min</sub></em>,<em>y<sub>max</sub></em>) = <em>n<sub>req</sub></em>. Although these equations can be numerically solved for the general case represented by equation (13), we restrict our analysis below to the two most common screen cases for which either equations (10) or (16) are valid.</p>
<div class="figure" id="pone-0001117-g003"><div class="img"><a name="pone-0001117-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001117.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001117.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Determining parameters for maximally efficient screens.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Graphs of <em>n<sub>act</sub> vs.</em> log<sub>10</sub><em>y</em> are plotted for three different values of <em>W</em>. For <em>n<sub>req</sub></em> = 5,000 (horizontal line), the minimal value of <em>W</em> is 37,642. log<sub>10</sub><em>y<sub>max</sub></em> is indicated. Plots generated by the program Mathematica 5.0 (Wolfram Research).</p>
<span>doi:10.1371/journal.pone.0001117.g003</span></div>

<h4>Optimizing Genetic Screens- All <em>F</em>1 Animals on a Single Plate</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>For genetic screens involving plating all <em>F</em>1 animals on a single plate or on a small number of plates relative to the total number of <em>F</em>1 animals examined, we can use the Poisson approximation to write the efficiency of a screen as:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e023&amp;representation=PNG" class="inline-graphic"><span class="note">(18)</span></span><br></p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>We differentiate this equation with respect to <em>y</em> and set the result equal to 0 to obtain the following transcendental equation for <em>y<sub>max</sub></em>, the optimal <em>F</em>2-to-<em>F</em>1 screening ratio:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e024&amp;representation=PNG" class="inline-graphic"><span class="note">(19)</span></span><br></p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>Two features of this equation are of interest. First, the equation cannot be solved analytically, and must be evaluated numerically. Second, and more importantly, the value of <em>y<sub>max</sub></em> is only dependent on the relative values of γ and α. Thus, the optimal screening ratio, <em>y<sub>max</sub></em>, is independent of the absolute work expended to pick and pick and score <em>F</em>1 and <em>F</em>2 animals, respectively.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>Does <em>y<sub>max</sub></em> always exist? Inspection of equation (19) reveals that it is of the general form <em>x</em> = ln(<em>a</em>+<em>x</em>), where <em>x</em> = <em>y</em>(1−<em>p</em>), and <em>a</em> is always greater than 1. Consider the range of possible values of <em>x</em>, from 0 to ∞. At <em>x</em> = 0, the left hand side of the preceding equation is always smaller than the right hand side (ln<em>a</em>). As <em>x</em>→∞, ln(<em>a</em>+<em>x</em>)≈ln(<em>x</em>), and thus the right hand side of the equation is always smaller than the left hand side. Since both <em>x</em> and ln(<em>a</em>+<em>x</em>) are continuous functions, these observations mean that there always exists an intersection point of the functions <em>x</em> and ln(<em>a</em>+<em>x</em>), defining <em>y<sub>max</sub></em>. Therefore, <em>y<sub>max</sub></em> exists for all values of α and γ. Differentiation of ε twice with respect to <em>y</em> shows that <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e025&amp;representation=PNG" class="inline-graphic"></span> for all values of <em>y<sub>max</sub></em>, guaranteeing that <em>y<sub>max</sub></em> indeed represent a global maximum of ε.</p>
<a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p><a href="#pone-0001117-g004">Figure 4D</a> depicts values of the optimal screening ratio for a range of ratios of α and γ. Insertion of <em>y<sub>max</sub></em> into equation (18) yields ε<em><sub>max</sub></em> for given values of α and γ. <a href="#pone-0001117-g004">Figure 4A</a> depicts ε<em><sub>max</sub></em> for a range of values of α and γ and for different values of <em>p</em>. As expected, the smaller the values of α and γ, the larger is ε, and the more efficient the screen.</p>
<div class="figure" id="pone-0001117-g004"><div class="img"><a name="pone-0001117-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001117.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001117.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Optimal <em>F</em>2-to-<em>F</em>1 screening ratios and screen efficiency calculations.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) Contour plots examining maximal screen efficiency, ε<em><sub>max</sub></em>, as a function of α and γ, for different values of <em>p</em>, for screens where all <em>F</em>1 animals are plated on one or a small number of plates. Plots generated using the program MatLab (MathWorks). (B) Graphs examining fold increase in work performed as screening ratio (<em>m</em>/<em>n</em>) deviates from its optimal value, for screens where all <em>F</em>1 animals are plated on one or a small number of plates and α/γ = 10. (C) Graphs depicting maximal screen efficiencies as a function of α for screens in which <em>F</em>1 animals are plated individually. In these graphs γ = 1, which is the most common value for this screening mode. (D) Graphs of the optimal <em>F</em>2-to-<em>F</em>1 screening ratios (<em>m</em>/<em>n</em>) for different values of <em>p</em> as functions of α/γ, for screens where all <em>F</em>1 animals are plated on one or a small number of plates. Note that the vertical axis is the natural log of <em>m</em>/<em>n</em> and not the base 10 log. (E) Graphs of the optimal <em>F</em>2-to-<em>F</em>1 screening ratios (<em>m</em>/<em>n</em>) for different values of <em>p</em> as functions of α, for screens in which <em>F</em>1 animals are plated individually. In these graphs γ = 1, which is the most common value for this screening mode.</p>
<span>doi:10.1371/journal.pone.0001117.g004</span></div>

<h4>Optimizing Genetic Screens- One <em>F</em>1 Animal per Plate</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>For genetic screens involving plating one <em>F</em>1 animal per plate, the Poisson approximation cannot be used (see above), and instead we use equation (16) to write the efficiency of a screen as:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e026&amp;representation=PNG" class="inline-graphic"><span class="note">(20)</span></span><br></p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>As above, we differentiate equation (20) to obtain the following transcendental equation for <em>y<sub>max</sub></em>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e027&amp;representation=PNG" class="inline-graphic"><span class="note">(21)</span></span><br></p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>Similar reasoning to that of the previous section guarantees the existence of a solution for <em>y</em>. However, the value of <em>y</em> obtained here cannot be used directly to compute <em>y<sub>max</sub></em> or to calculate ε<em><sub>max</sub></em>, for two reasons. First, unlike the previous section, <em>y</em> cannot be treated as a continuous variable here, and the maximum calculated in equation (21) makes this assumption. Second, <em>y</em> can only take on integer values ≥1. Thus, to identify <em>y<sub>max</sub></em>, we numerically calculate the solution to equation (21). We then check whether the obtained value of <em>y</em> is smaller than 1. If so, then <em>y<sub>max</sub></em> = 1. If not, we calculate the efficiency of screening, using equation (20) for the two nearest integer values of <em>y</em>, and choose <em>y<sub>max</sub></em> as the value giving the highest value of ε. The results of these calculations for ε<em><sub>max</sub></em> and <em>y<sub>max</sub></em> are presented in <a href="#pone-0001117-g004">Figures 4C and 4E</a>, respectively. Note that in these figures we have assumed γ = 1, since this is most often the case when performing a screen of the type considered here. However, other values of γ may be possible, in which case α in this figure should be replaced by α/γ.</p>


<h4>Deviations from Maximal Screening Efficiency</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>In <a href="#pone-0001117-g004">Figure 4B</a> we plot the ratio of the work expended in a genetic screen, <em>W</em>, to the minimal work, <em>W<sub>min</sub></em>, calculated using equations (17), (18), and (19), as a function of <em>m</em>/<em>n</em>, for a screen in which scoring and picking <em>F</em>2 animals requires ten times more work than picking <em>F</em>1 animals (α/γ = 10). Such parameters are often encountered, and may serve as a model for screens in which <em>F</em>2 animals need to be examined individually on a compound microscope, for example. As the figure demonstrates, relatively small deviations in the screening ratio can have a large impact on the amount of work carried out to screen the same number of mutagenized <em>F</em>1 animals. For example, screening five <em>F</em>2 animals for every <em>F</em>1 parent for recessive mutations nearly doubles the work required to screen the same number of mutagneized <em>F</em>1 animals compared to the optimal screening ratio of 0.86. The differences are even more dramatic when dominant mutations are sought.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>These results clearly show that using optimized screen parameters can have a significant impact on the progress and output of genetic screens in <em>C. elegans</em>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Summary of Key Points</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>In this paper, we derive a strategy for optimizing genetic screens in the nematode <em>C. elegans</em>. We demonstrate two key points. First, an optimal screening strategy always exists for every genetic screen of the types considered here. Second, calculation of this optimal strategy is possible. <a href="#pone-0001117-g004">Figures 4D and 4E</a> depict the results of such calculations, displaying the optimal <em>F</em>2-to-<em>F</em>1 screening ratios for a large range of screen parameters. As shown in <a href="#pone-0001117-g004">Figure 4B</a>, using optimal screening parameters for screens of any type can make a significant difference in the amount of time, labor, and/or reagents used to identify mutants of interest. This difference in efficiency between optimal and suboptimal screening strategies is most accentuated under two condition: when there is a significant difference in the work done picking <em>F</em>1 animals and picking and scoring <em>F</em>2 animals; and when the mutations sought manifest themselves in the <em>F</em>2 population in increased proportion (as might occur with a dominant mutation; <a href="#pone-0001117-g004">Figure 4B</a>).</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Our results suggest a general rule of thumb: in pursuing a genetic screen, optimal efficiency is achieved by minimizing as much as possible the more difficult task between picking <em>F</em>1 animals or picking and scoring <em>F</em>2 animals.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>In addition to the key results discussed above, we have also derived a number of other useful results. First, we have shown that the optimal screening strategy does not depend on the total amount of effort expended in a screen, but only depends on the ratio of the work involved in picking <em>F</em>1 animals to the work expended in picking and scoring <em>F</em>2 animals, and on the type of mutation being sought. Second, our studies reveal that use of the Poisson approximation to count the number of mutagenized <em>F</em>1 animals examined in a screen is not appropriate for all situations. Third, instead, we derive an equation (equation (13)), that is valid for a large number of genetic screens. Fourth, we demonstrate that two limiting cases of this equation, in which a large number of <em>F</em>1 animals are placed on a small number of plates, or in which <em>F</em>1 animals are plated individually, yield simplified equations (equation (10), based on the Poisson approximation, and equation (16)), that are well known and of considerable practical use. Finally, analysis of these limiting cases also reveals that the number of mutagenized <em>F</em>1 animals examined is dependent on the mode in which <em>F</em>1 animals are plated. In general, we show that plating <em>F</em>1 animals individually, followed by scoring their <em>F</em>2 progeny, allows more mutagenized <em>F</em>1 animals to be examined than plating the same number of <em>F</em>1 animals on a single or small number of plates. However, it should be noted that because plating <em>F</em>1 animals individually can be more time consuming and may require more reagents, the overall screen efficiency may or may not be higher using this strategy (see below).</p>


<h4>A Classification of Genetic Screens</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Our analysis suggests that genetic screens in <em>C. elegans</em> can be divided into three general categories, based on the difficulties involved in picking <em>F</em>1 animals (the value of γ, see <a href="#s2">Results</a> and <a href="#pone-0001117-g005">Figure 5</a>) and picking and scoring <em>F</em>2 animals (the value of α, see <a href="#s2">Results</a> and <a href="#pone-0001117-g005">Figure 5</a>).</p>
<div class="figure" id="pone-0001117-g005"><div class="img"><a name="pone-0001117-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001117" data-uri="info:doi/10.1371/journal.pone.0001117.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001117.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001117.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001117.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001117.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Algorithm for performing an optimal genetic screen.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Flowchart begins on the top left corner at “START”. All parameters and equations are described and derived in the text. Parameters of relevance are also described in the Glossary portion of the figure. Diamond shapes indicate steps where a choice must be made.</p>
<span>doi:10.1371/journal.pone.0001117.g005</span></div><h5>Type I.</h5><a id="article1.body1.sec3.sec2.sec1.p1" name="article1.body1.sec3.sec2.sec1.p1"></a><p>Picking <em>F</em>1 animals and picking and scoring <em>F</em>2 animals is of similar magnitude of difficulty. A remarkable consequence of our analysis is that regardless of the type of screen, or the work involved, all screens for which α/γ is fixed, have the same optimal screening ratio. Thus, for screens of this type the ratio of α and γ will determine the precise <em>F</em>2-to-<em>F</em>1 screening ratio. For screens of this type, equations (10) and (16), can be used to determine <em>n<sub>act</sub></em> and follow screen progress.</p>

<h5>Type II.</h5><a id="article1.body1.sec3.sec2.sec2.p1" name="article1.body1.sec3.sec2.sec2.p1"></a><p>Picking <em>F</em>1 animals is much harder than picking <em>F</em>2 animals (α/γ→0). As shown in <a href="#pone-0001117-g004">Figure 4D</a>, for very small values of α/γ, the optimal screening ratio becomes large, and thus, many <em>F</em>2 animals should be scored for each <em>F</em>1. As we showed in the beginning section of the paper, under such conditions <em>n<sub>act</sub></em>≈<em>n</em>, and screen progress is limited by the number of <em>F</em>1 animals that can be examined.</p>

<h5>Type III.</h5><a id="article1.body1.sec3.sec2.sec3.p1" name="article1.body1.sec3.sec2.sec3.p1"></a><p>Picking/scoring <em>F</em>2 animals is much more difficult than picking <em>F</em>1 animals (α/γ→∞). <a href="#pone-0001117-g004">Figure 4D</a> reveals that as α/γ becomes large, the optimal screening ratio becomes small, so that the number of <em>F</em>1 animals picked should be much greater than the number of <em>F</em>2 animals scored. Under these conditions, <em>n<sub>act</sub></em>≈<em>m</em>(1−<em>p</em>), and progress is determined by the type of mutation being sought (<em>e.g.</em> recessive, dominant), and by the number of <em>F</em>2 animals screened.</p>
<a id="article1.body1.sec3.sec2.sec3.p2" name="article1.body1.sec3.sec2.sec3.p2"></a><p>The merits of the classification system described above are that it allows a quick determination of whether an in-depth analysis of screen parameters is required to follow screen progress. Specifically, for screens of types II and III, calculating screen progress (<em>n<sub>act</sub></em>) is very simple and does not depend on plating strategy. Screens of type I, however, require a more detailed study of the screen and plating parameters.</p>
<a id="article1.body1.sec3.sec2.sec3.p3" name="article1.body1.sec3.sec2.sec3.p3"></a><p>In <a href="#pone-0001117-t001">Table 1</a> we compare the optimal screening ratios to screen 5,000 mutagenized <em>F</em>1 animals for recessive mutations for different estimated values of α and γ (see table legend for estimation procedure; also see next section). The table illustrates a number of points. First, it provides estimates of the amount of work expended for each screen. Although the table provides exact numbers, it is important to note that these numbers only approximate the actual work involved because some of the parameters used in calculating the work, such as α and γ, may not be exact. Second, it provides specific applications of the different screen strategies to common screens undertaken in <em>C. elegans</em>. Third, the table demonstrates that strategies that minimize both α and γ are, as expected, most efficient (see also <a href="#pone-0001117-g004">Figure 4A</a>).</p>
<a id="article1.body1.sec3.sec2.sec3.p4" name="article1.body1.sec3.sec2.sec3.p4"></a><p>Interestingly, examination of <a href="#pone-0001117-g004">Figure 4A</a> shows that screen efficiency is not perfectly symmetric with respect to α and γ. This lack of symmetry is partially a function of <em>p</em> and, it is easily shown that the maximal efficiency, (ε<em><sub>max</sub></em>), of screens of type II approaches 1/γ, whereas the efficiency of screens of type III approaches (1−<em>p</em>)/α. Thus, if one is debating between screening strategies of type II and III where γ(type II)≈α(type III), screens of type II will always be more efficient.</p>



<h4>Estimating α and γ</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>As we describe in the Results section, precise values of α and γ are not important for calculating the optimal <em>F</em>2-to-<em>F</em>1 screening ratio. Indeed, as equations (19) and (21) show, only the ratio of α to γ is relevant. It turns out, however, that even an exact measurement of this ratio is not needed in practice. The reason for this is shown in <a href="#pone-0001117-g004">Figure 4D</a>. As shown in this figure, varying α/γ over six orders of magnitude, only changes the <em>F</em>2-to-<em>F</em>1 screening ratio by two orders of magnitude (note that the vertical axis is the natural log, and not base 10 log of the screening ratio). Thus, the screening ratio is not very sensitive to variations in α and γ. This observation is of clear practical importance, since it is not always easy to precisely compare the work involved in picking <em>F</em>1 animals and picking and scoring <em>F</em>2 animals. These results suggest that order-of-magnitude estimates of the relative work involved will give good estimates of the appropriate screening ratio.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In <a href="#pone-0001117-t001">Table 1</a> we provide estimates for α and γ for picking and scoring animals in the course of different genetic screens. The estimates are based on personal experience of the author (see legend to <a href="#pone-0001117-t001">Table 1</a>), and may vary with individual expertise and protocol. However, since the values of α and γ are essentially user defined, individual variations in estimating screen parameters do not affect the results presented here. More accurate values of α and γ can be obtained from pilot screens, which are often carried out anyway, where these parameters can be measured directly by keeping track of the amount of work done (time spent) to pick a fixed number of <em>F</em>1 animals and pick and score a fixed number of <em>F</em>2 animals. Each work segment is then divided, respectively, by either the number of <em>F</em>1 or <em>F</em>2 animals to obtain γ and α, respectively. Values for these parameters can also be adjusted as a screen proceeds, based on estimates derived from earlier stages of screening.</p>


<h4>An Algorithm for Optimal Screening</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Our results suggest that the design of many types of genetic screens in <em>C. elegans</em> can follow a simple set of rules. In <a href="#pone-0001117-g005">Figure 5</a> we present a simple algorithm for designing a genetic screen and for following screen progress. Initially, a choice must be made as to whether <em>F</em>1 animals will be plated onto individual plates (individually) or onto a small number of plates (bulk). This choice is usually not driven by efficiency, but by constraints of the screen. For example, screens that require a clonal strategy, because the identity of the <em>F</em>1 parent is important (as might occur in screens for recessive lethals, where heterozygous siblings are to be isolated), demand plating <em>F</em>1 animals individually.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>If both plating strategies are applicable, the choice of which plating strategy to use will be determined by comparing the efficiencies of each strategy. For example, consider a screen for recessive mutations in which it is not necessary to keep tabs on the <em>F</em>1 parents, and for which <em>F</em>2 animals must be picked to individual plates for scoring (α≈1; for definitions of α and γ see Glossary of <a href="#pone-0001117-g005">Figure 5</a>, or Results section). Two possible screen options are: <em>F</em>1 animals will be plated individually and therefore α≈1 and γ = 1; or, <em>F</em>1 animals will be plated in bulk where instead of picking <em>F</em>1 animals with a pick, they are loaded from a synchronized liquid culture, and thus γ≈0.001, but α is still about 1. Examination of <a href="#pone-0001117-g004">Figures 4C and 4A</a>, respectively (or using the webcalculator, see below), reveals that the efficiency of the first screen is 0.145, whereas the efficiency of the second screen is 0.245. Therefore, if the screens are carried out at maximal efficiency, the second screen is about 1.7 times more efficient. On the other hand, if in the second screen, animals are plated on a small number of plates, but picked individually to those plates, instead of being loaded from a liquid culture, then γ≈1, and the efficiencies of the first and second screens are 0.145 and 0.133, respectively. Thus, in this case, even though there is no need to preserve the identity of each <em>F</em>1 parent, an individual plating strategy is more efficient.</p>
<a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>Once plating strategy is established, a determination of whether a fixed number of mutagenized <em>F</em>1 animals is to be screened, or whether the screen will be open ended is made. Estimates of α and γ are obtained (if these have not been obtained already), and these are used to identify the optimal <em>F</em>2-to-<em>F</em>1 screening ratio ( = <em>m</em>/<em>n</em>) (<a href="#pone-0001117-g004">Figures 4D, and 4E</a>). From this ratio, the indicated equations in <a href="#pone-0001117-g005">Figure 5</a> may be used to obtain the numbers of <em>F</em>1 and <em>F</em>2 animals to be screened. If the screen is open ended, progress can be followed using the indicated equations.</p>
<a id="article1.body1.sec3.sec4.p4" name="article1.body1.sec3.sec4.p4"></a><p>To aid with the calculations described here, in <a href="#pone-0001117-g005">Figure 5</a>, and throughout the paper, we have developed a website with a simple interface that can be used independently of this paper. The site yields precise numerical results, and is thus more accurate than <a href="#pone-0001117-g004">Figure 4</a>, where, by necessity, precise values are difficult to read from the graphs. The site can be accessed at “<a href="http://b5.rockefeller.edu/cgi-bin/labheads/shaham/genetic_screens/screenfrontpage.cgi">http://b5.rockefeller.edu/cgi-bin/labhea​ds/shaham/genetic_screens/screenfrontpag​e.cgi</a>”.</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>We have presented a systematic approach for optimizing genetic screens in <em>C. elegans</em>, and calculated optimal <em>F</em>2-to-<em>F</em>1 screening ratios for a large range of screen parameters. Calculation of these parameters was aided by obtaining a general expression for counting the number of mutagenized gametes examined in the course of a genetic screen. The strategies described here can, in principle, be applied, with relevant modifications, to the evaluation of equivalent parameters in genetic screens in other organisms.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Derivation of Poisson approximations</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>In this paper, we employ expressions of the following form<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e028&amp;representation=PNG" class="inline-graphic"></span><br>Such expressions can be simplified using Poisson terms to approximate the respective binomial terms. Specifically, it is well known that for large <em>m</em> and <em>n</em>,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e029&amp;representation=PNG" class="inline-graphic"></span><br>Thus, we can write<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e030&amp;representation=PNG" class="inline-graphic"></span><br>Expanding parentheses, this term can be rewritten as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e031&amp;representation=PNG" class="inline-graphic"></span><br>Using the series expansion <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e032&amp;representation=PNG" class="inline-graphic"></span> for large <em>m</em> and <em>n</em>, we simplify the expression above to<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e033&amp;representation=PNG" class="inline-graphic"></span><br>leading to the approximation<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e034&amp;representation=PNG" class="inline-graphic"></span><br></p>


<h4>Derivation of maximal error in using the Poisson approximation when plating <em>F</em>1 animals singly</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The fractional error in using the Poisson approximation is defined as <em>f</em> = [(<em>n<sub>act</sub></em>/<em>N</em>)<em><sub>exact</sub></em>−(<em>n<sub>act</sub></em>/<em>N</em>)<em><sub>Poisson</sub></em>]/(<em>n<sub>act</sub></em>/<em>N</em>)<em><sub>exact</sub></em>. Using equations 10 and 16, the fractional error is written as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e035&amp;representation=PNG" class="inline-graphic"></span><br>which upon rearrangement yields<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e036&amp;representation=PNG" class="inline-graphic"></span><br>As <em>y</em> approaches 1, this expression, therefore becomes<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001117.e037&amp;representation=PNG" class="inline-graphic"></span><br></p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>I would like to thank Taulant Bacaj, Margherita Peliti, Max Heiman, and anonymous reviewers of this and a previous version of the manuscript for comments and suggestions, and Matthieu Louis for help in generating <a href="#pone-0001117-g004">Figure 4A</a>.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SS. Performed the experiments: SS. Analyzed the data: SS. Wrote the paper: SS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001117-Anderson1" id="pone.0001117-Anderson1"></a>Anderson P (1995) Mutagenesis. Meth Cell Biol  48: 31–58.  <ul class="find" data-citedArticleID="1012650"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mutagenesis.&amp;auth=&amp;atitle=Mutagenesis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutagenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutagenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001117-Brenner1" id="pone.0001117-Brenner1"></a>Brenner S (1974) The Genetics of <em>Caenorhabditis elegans</em>. Genetics  77: 71–94.  <ul class="find" data-citedArticleID="1012654"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Genetics+of+Caenorhabditis+elegans.&amp;auth=&amp;atitle=The+Genetics+of+Caenorhabditis+elegans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Genetics+of+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Genetics+of+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001117-Meneely1" id="pone.0001117-Meneely1"></a>Meneely P, Herman R (1979) Lethals, Steriles and Deficiencies in a Region of the <em>X</em> Chromosome of <em>Caenorhabditis elegans</em>. Genetics  92: 99–115.  <ul class="find" data-citedArticleID="1012674"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Lethals%2C+Steriles+and+Deficiencies+in+a+Region+of+the+X+Chromosome+of+Caenorhabditis+elegans.&amp;auth=&amp;atitle=Lethals%2C+Steriles+and+Deficiencies+in+a+Region+of+the+X+Chromosome+of+Caenorhabditis+elegans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lethals%2C+Steriles+and+Deficiencies+in+a+Region+of+the+X+Chromosome+of+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lethals%2C+Steriles+and+Deficiencies+in+a+Region+of+the+X+Chromosome+of+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001117-Greenwald1" id="pone.0001117-Greenwald1"></a>Greenwald I, Horvitz HR (1980) <em>unc-93(e1500)</em>: A Behavioral Mutant of <em>Caenorhabditis elegans</em> That Defines a Gene with a Wild-Type Null Phenotype. Genetics  96: 147–164.  <ul class="find" data-citedArticleID="1012660"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=unc-93%28e1500%29%3A+A+Behavioral+Mutant+of+Caenorhabditis+elegans+That+Defines+a+Gene+with+a+Wild-Type+Null+Phenotype.&amp;auth=&amp;atitle=unc-93%28e1500%29%3A+A+Behavioral+Mutant+of+Caenorhabditis+elegans+That+Defines+a+Gene+with+a+Wild-Type+Null+Phenotype." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=unc-93%28e1500%29%3A+A+Behavioral+Mutant+of+Caenorhabditis+elegans+That+Defines+a+Gene+with+a+Wild-Type+Null+Phenotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22unc-93%28e1500%29%3A+A+Behavioral+Mutant+of+Caenorhabditis+elegans+That+Defines+a+Gene+with+a+Wild-Type+Null+Phenotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001117-Howell1" id="pone.0001117-Howell1"></a>Howell AM, Rose AM (1990) Essential Genes in the <em>hDf6</em> Region of chromosome <em>I</em> in <em>Caenorhabditis elegans</em>. Genetics  126: 583–592.  <ul class="find" data-citedArticleID="1012662"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Essential+Genes+in+the+hDf6+Region+of+chromosome+I+in+Caenorhabditis+elegans.&amp;auth=&amp;atitle=Essential+Genes+in+the+hDf6+Region+of+chromosome+I+in+Caenorhabditis+elegans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Essential+Genes+in+the+hDf6+Region+of+chromosome+I+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Essential+Genes+in+the+hDf6+Region+of+chromosome+I+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001117-Johnsen1" id="pone.0001117-Johnsen1"></a>Johnsen RC, Baillie DL (1991) Genetic Analysis of a Major Segment [LGV(left)] of the Genome of <em>Caenorhabditis elegans</em>. Genetics  129: 735–752.  <ul class="find" data-citedArticleID="1012664"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+Analysis+of+a+Major+Segment+%5BLGV%28left%29%5D+of+the+Genome+of+Caenorhabditis+elegans.&amp;auth=&amp;atitle=Genetic+Analysis+of+a+Major+Segment+%5BLGV%28left%29%5D+of+the+Genome+of+Caenorhabditis+elegans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+Analysis+of+a+Major+Segment+%5BLGV%28left%29%5D+of+the+Genome+of+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+Analysis+of+a+Major+Segment+%5BLGV%28left%29%5D+of+the+Genome+of+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001117-Johnsen2" id="pone.0001117-Johnsen2"></a>Johnsen RC, Baillie DL (1997)  Mutation. In: Riddle DL, Blumenthal T, Meyer BJ, Priess JR, editors. <em>C. ELEGANS</em> II. Cold Spring Harbor: Cold Spring Harbor Press. pp. 79–96.  <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0001117-Loechler1" id="pone.0001117-Loechler1"></a>Loechler EL, Green CL, Essigmann JM (1984) <em>In vivo</em> mutagenesis by <em>O</em>-methylguanine built into a unique site in a viral genome. Proc Natl Acad Sci USA  81: 6271–6275.  <ul class="find" data-citedArticleID="1012672" data-doi="10.1073/pnas.81.20.6271"><li><a href="http://dx.doi.org/10.1073/pnas.81.20.6271" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+mutagenesis+by+O-methylguanine+built+into+a+unique+site+in+a+viral+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+mutagenesis+by+O-methylguanine+built+into+a+unique+site+in+a+viral+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001117-Snow1" id="pone.0001117-Snow1"></a>Snow ET, Foote RS, Mitra S (1984) Base-pairing Properties of <em>O</em>-Methylguanine in Template DNA during <em>in Vitro</em> DNA Replication. J Biol Chem  259: 8095–8100.  <ul class="find" data-citedArticleID="1012678"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Base-pairing+Properties+of+O-Methylguanine+in+Template+DNA+during+in+Vitro+DNA+Replication.&amp;auth=&amp;atitle=Base-pairing+Properties+of+O-Methylguanine+in+Template+DNA+during+in+Vitro+DNA+Replication." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-pairing+Properties+of+O-Methylguanine+in+Template+DNA+during+in+Vitro+DNA+Replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-pairing+Properties+of+O-Methylguanine+in+Template+DNA+during+in+Vitro+DNA+Replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001117-Pastink1" id="pone.0001117-Pastink1"></a>Pastink A, Heemskerk E, Nivard MJM, van Vliet CJ, Vogel EW (1991) Mutational specificity of ethyl methanesulfonate in excision-repair-proficient and -deficient strains of <em>Drosophila melanogaster</em>. Mol Gen Genet  229: 213–218.  <ul class="find" data-citedArticleID="1012676" data-doi="10.1007/bf00272158"><li><a href="http://dx.doi.org/10.1007/bf00272158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutational+specificity+of+ethyl+methanesulfonate+in+excision-repair-proficient+and+-deficient+strains+of+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutational+specificity+of+ethyl+methanesulfonate+in+excision-repair-proficient+and+-deficient+strains+of+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001117-Greene1" id="pone.0001117-Greene1"></a>Greene EA, Codomo CA, Taylor NE, Henikoff JG, Till BJ, Reynolds SH, Enns LC, Burtner C, Johnson JE, Odden AR, Comai L, Henikoff S (2003) Spectrum of Chemically Induced Mutations from a Large-Scale Reverse-Genetic Screen in Arabidopsis. Genetics  164: 731–740.  <ul class="find" data-citedArticleID="1012658"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Spectrum+of+Chemically+Induced+Mutations+from+a+Large-Scale+Reverse-Genetic+Screen+in+Arabidopsis.&amp;auth=&amp;atitle=Spectrum+of+Chemically+Induced+Mutations+from+a+Large-Scale+Reverse-Genetic+Screen+in+Arabidopsis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spectrum+of+Chemically+Induced+Mutations+from+a+Large-Scale+Reverse-Genetic+Screen+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spectrum+of+Chemically+Induced+Mutations+from+a+Large-Scale+Reverse-Genetic+Screen+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001117-Benian1" id="pone.0001117-Benian1"></a>Benian GM, Kiff JE, Neckelmann N, Moerman DG, Waterston RH (1989) Sequence of an unusually large protein implicated in regulation of myosin activity in <em>C. elegans</em>. Nature  342: 45–50.  <ul class="find" data-citedArticleID="1012652" data-doi="10.1038/342045a0"><li><a href="http://dx.doi.org/10.1038/342045a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+of+an+unusually+large+protein+implicated+in+regulation+of+myosin+activity+in+C.+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+of+an+unusually+large+protein+implicated+in+regulation+of+myosin+activity+in+C.+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001117-Lee1" id="pone.0001117-Lee1"></a>Lee RC, Feinbaum RL, Ambros V (1993) The C. elegans Heterochronic Gene <em>lin-4</em> Encodes Small RNAs with Antisense Complementarity to <em>lin-14</em>. Cell  75: 843–854.  <ul class="find" data-citedArticleID="1012670" data-doi="10.1016/0092-8674(93)90529-y"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)90529-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+C.+elegans+Heterochronic+Gene+lin-4+Encodes+Small+RNAs+with+Antisense+Complementarity+to+lin-14." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+C.+elegans+Heterochronic+Gene+lin-4+Encodes+Small+RNAs+with+Antisense+Complementarity+to+lin-14.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001117-Ellis1" id="pone.0001117-Ellis1"></a>Ellis RE, Jacobson DM, Horvitz HR (1991) Genes Required for the Engulfment of Cell Corpses during Programmed Cell Death in <em>Caenorhabditis elegans</em>. Genetics  129: 79–94.  <ul class="find" data-citedArticleID="1012656" data-doi="10.1016/0168-9525(92)90009-s"><li><a href="http://dx.doi.org/10.1016/0168-9525(92)90009-s" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+Required+for+the+Engulfment+of+Cell+Corpses+during+Programmed+Cell+Death+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+Required+for+the+Engulfment+of+Cell+Corpses+during+Programmed+Cell+Death+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001117-Jorgensen1" id="pone.0001117-Jorgensen1"></a>Jorgensen EM, Mango SE (2002) The Art and Design of Genetic Screens: <em>Caenorhabditis elegans</em>. Nat Rev Genet  3: 356–369.  <ul class="find" data-citedArticleID="1012668" data-doi="10.1038/nrg794"><li><a href="http://dx.doi.org/10.1038/nrg794" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Art+and+Design+of+Genetic+Screens%3A+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Art+and+Design+of+Genetic+Screens%3A+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.XML" value="95215"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.PDF" value="482192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g001.PNG_L" value="199796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g001.PNG_M" value="19840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g001.PNG_S" value="5518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g001.TIF" value="319176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g001.PNG_I" value="13664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e001.PNG" value="8240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e001.TIF" value="24104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e002.PNG" value="8676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e002.TIF" value="24448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e003.PNG" value="11905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e003.TIF" value="27596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e004.PNG" value="13148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e004.TIF" value="29032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e005.PNG" value="6571"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e005.TIF" value="22540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.t001.PNG_L" value="243628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.t001.PNG_M" value="138741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.t001.PNG_S" value="13757"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.t001.TIF" value="1062240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.t001.PNG_I" value="43187"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e006.PNG" value="11281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e006.TIF" value="26876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e007.PNG" value="6433"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e007.TIF" value="22352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e008.PNG" value="10260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e008.TIF" value="26084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e009.PNG" value="12737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e009.TIF" value="28716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e010.PNG" value="10464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e010.TIF" value="26320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e011.PNG" value="16158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e011.TIF" value="31836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e012.PNG" value="7213"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e012.TIF" value="23200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e013.PNG" value="8830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e013.TIF" value="24704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g002.PNG_L" value="1367284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g002.PNG_M" value="102721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g002.PNG_S" value="9838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g002.TIF" value="2312734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g002.PNG_I" value="36439"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e014.PNG" value="5302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e014.TIF" value="21216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e015.PNG" value="10159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e015.TIF" value="25948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e016.PNG" value="11170"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e016.TIF" value="26836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e017.PNG" value="12212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e017.TIF" value="27744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e018.PNG" value="10800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e018.TIF" value="26504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e019.PNG" value="10683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e019.TIF" value="26408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e020.PNG" value="9178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e020.TIF" value="25000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e021.PNG" value="6288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e021.TIF" value="22204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e022.PNG" value="9907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e022.TIF" value="25628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g003.PNG_L" value="350708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g003.PNG_M" value="31469"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g003.PNG_S" value="6038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g003.TIF" value="488394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g003.PNG_I" value="10738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e023.PNG" value="11285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e023.TIF" value="26868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e024.PNG" value="10008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e024.TIF" value="25928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e025.PNG" value="7529"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e025.TIF" value="23480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g004.PNG_L" value="1452893"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g004.PNG_M" value="121864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g004.PNG_S" value="13985"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g004.TIF" value="3510956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g004.PNG_I" value="75871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e026.PNG" value="10406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e026.TIF" value="26112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e027.PNG" value="9272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e027.TIF" value="25220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g005.PNG_L" value="3260295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g005.PNG_M" value="221529"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g005.PNG_S" value="12163"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g005.TIF" value="5658226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.g005.PNG_I" value="72924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e028.PNG" value="11788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e028.TIF" value="27352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e029.PNG" value="12205"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e029.TIF" value="27772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e030.PNG" value="19390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e030.TIF" value="35080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e031.PNG" value="13176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e031.TIF" value="28880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e032.PNG" value="7936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e032.TIF" value="23740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e033.PNG" value="10049"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e033.TIF" value="25824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e034.PNG" value="12467"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e034.TIF" value="28216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e035.PNG" value="10390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e035.TIF" value="26164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e036.PNG" value="8192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e036.TIF" value="24036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e037.PNG" value="7245"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001117.e037.TIF" value="23200"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001117&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001117" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001117&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001117&volume=&issue=&title=Counting Mutagenized Genomes and Optimizing Genetic Screens in Caenorhabditis elegans&author_name=Shai%20Shaham&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001117" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001117" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001117" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001117&amp;t=Counting%20Mutagenized%20Genomes%20and%20Optimizing%20Genetic%20Screens%20in%20Caenorhabditis%20elegans" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001117&title=Counting%20Mutagenized%20Genomes%20and%20Optimizing%20Genetic%20Screens%20in%20Caenorhabditis%20elegans&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001117&amp;title=Counting%20Mutagenized%20Genomes%20and%20Optimizing%20Genetic%20Screens%20in%20Caenorhabditis%20elegans" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001117" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001117&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Counting Mutagenized Genomes and Optimizing Genetic Screens in Caenorhabditis elegans';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001117';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Counting%20Mutagenized%20Genomes%20and%20Optimizing%20Genetic%20Screens%20in%20Caenorhabditis%20elegans http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001117" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001117" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">









          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+genomics%22" title="Search for articles in the subject area:'Animal genomics'"><div class="flagText">Animal genomics</div></a>
              <div data-categoryid="17455" data-articleid="25068"
                   data-categoryname="Animal genomics"
                   class="flagImage" title="Flag 'Animal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Caenorhabditis+elegans%22" title="Search for articles in the subject area:'Caenorhabditis elegans'"><div class="flagText">Caenorhabditis elegans</div></a>
              <div data-categoryid="42653" data-articleid="25068"
                   data-categoryname="Caenorhabditis elegans"
                   class="flagImage" title="Flag 'Caenorhabditis elegans' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="25068"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+screens%22" title="Search for articles in the subject area:'Genetic screens'"><div class="flagText">Genetic screens</div></a>
              <div data-categoryid="47743" data-articleid="25068"
                   data-categoryname="Genetic screens"
                   class="flagImage" title="Flag 'Genetic screens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Graphs%22" title="Search for articles in the subject area:'Graphs'"><div class="flagText">Graphs</div></a>
              <div data-categoryid="19267" data-articleid="25068"
                   data-categoryname="Graphs"
                   class="flagImage" title="Flag 'Graphs' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutagenesis%22" title="Search for articles in the subject area:'Mutagenesis'"><div class="flagText">Mutagenesis</div></a>
              <div data-categoryid="21421" data-articleid="25068"
                   data-categoryname="Mutagenesis"
                   class="flagImage" title="Flag 'Mutagenesis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="25068"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Poisson+approximation%22" title="Search for articles in the subject area:'Poisson approximation'"><div class="flagText">Poisson approximation</div></a>
              <div data-categoryid="27559" data-articleid="25068"
                   data-categoryname="Poisson approximation"
                   class="flagImage" title="Flag 'Poisson approximation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2399'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=627'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6481&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=11389">Figure 2C legend</a><br>Posted by shai</p>
      <p><a href="/annotation/listThread.action?root=6347">Copy editing mistake</a><br>Posted by shai</p>
      <p><a href="/annotation/listThread.action?root=22093">A couple of errors as of 11/13/08</a><br>Posted by shai</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
