

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Global Identification of Myc Target Genes Reveals Its Direct Role in Mitochondrial Biogenesis and Its E-Box Usage In Vivo</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001798"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001798" />

    <meta name="citation_title" content="Global Identification of Myc Target Genes Reveals Its Direct Role in Mitochondrial Biogenesis and Its E-Box Usage In Vivo"/>
    <meta itemprop="name" content="Global Identification of Myc Target Genes Reveals Its Direct Role in Mitochondrial Biogenesis and Its E-Box Usage In Vivo"/>

      <meta name="citation_author" content="Jonghwan Kim"/>
            <meta name="citation_author_institution" content="Section of Molecular Genetics and Microbiology, Center for Systems and Synthetic Biology, Institute for Cellular and Molecular Biology, University of Texas at Austin, Austin, Texas, United States of America"/>
      <meta name="citation_author" content="Ji-hoon Lee"/>
            <meta name="citation_author_institution" content="Section of Molecular Genetics and Microbiology, Center for Systems and Synthetic Biology, Institute for Cellular and Molecular Biology, University of Texas at Austin, Austin, Texas, United States of America"/>
      <meta name="citation_author" content="Vishwanath R. Iyer"/>
            <meta name="citation_author_institution" content="Section of Molecular Genetics and Microbiology, Center for Systems and Synthetic Biology, Institute for Cellular and Molecular Biology, University of Texas at Austin, Austin, Texas, United States of America"/>

    <meta name="citation_date" content="2008/3/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001798.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1798"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Translocations involving c-myc and c-myc function.; citation_author=LM Boxer; citation_author=CV Dang; citation_journal_title=Oncogene; citation_volume=20; citation_number=1; citation_pages=5595-5610; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Myc/Max/Mad network and the transcriptional control of cell behavior.; citation_author=C Grandori; citation_author=SM Cowley; citation_author=LP James; citation_author=RN Eisenman; citation_journal_title=Annu Rev Cell Dev Biol; citation_volume=16; citation_number=2; citation_pages=653-699; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=c-Myc-regulated microRNAs modulate E2F1 expression.; citation_author=KA O'Donnell; citation_author=EA Wentzel; citation_author=KI Zeller; citation_author=CV Dang; citation_author=JT Mendell; citation_journal_title=Nature; citation_volume=435; citation_number=3; citation_pages=839-843; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=c-MYC: more than just a matter of life and death.; citation_author=S Pelengaris; citation_author=M Khan; citation_author=G Evan; citation_journal_title=Nat Rev Cancer; citation_volume=2; citation_number=4; citation_pages=764-776; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Human c-myc onc gene is located on the region of chromosome 8 that is translocated in Burkitt lymphoma cells.; citation_author=R Dalla-Favera; citation_author=M Bregni; citation_author=J Erikson; citation_author=D Patterson; citation_author=RC Gallo; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=79; citation_number=5; citation_pages=7824-7827; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Translocation of the c-myc gene into the immunoglobulin heavy chain locus in human Burkitt lymphoma and murine plasmacytoma cells.; citation_author=R Taub; citation_author=I Kirsch; citation_author=C Morton; citation_author=G Lenoir; citation_author=D Swan; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=79; citation_number=6; citation_pages=7837-7841; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Genetic dissection of c-myc apoptotic pathways.; citation_author=CE Nesbit; citation_author=JM Tersak; citation_author=LE Grove; citation_author=A Drzal; citation_author=H Choi; citation_journal_title=Oncogene; citation_volume=19; citation_number=7; citation_pages=3200-3212; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Deconstructing myc.; citation_author=RN Eisenman; citation_journal_title=Genes Dev; citation_volume=15; citation_number=8; citation_pages=2023-2030; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulation and transformation by Myc proteins.; citation_author=S Adhikary; citation_author=M Eilers; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=6; citation_number=9; citation_pages=635-645; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Max: a helix-loop-helix zipper protein that forms a sequence-specific DNA-binding complex with Myc.; citation_author=EM Blackwood; citation_author=RN Eisenman; citation_journal_title=Science; citation_volume=251; citation_number=10; citation_pages=1211-1217; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The Myc:Max protein complex and cell growth regulation.; citation_author=EM Blackwood; citation_author=B Luscher; citation_author=L Kretzner; citation_author=RN Eisenman; citation_journal_title=Cold Spring Harb Symp Quant Biol; citation_volume=56; citation_number=11; citation_pages=109-117; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Mad: a heterodimeric partner for Max that antagonizes Myc transcriptional activity.; citation_author=DE Ayer; citation_author=L Kretzner; citation_author=RN Eisenman; citation_journal_title=Cell; citation_volume=72; citation_number=12; citation_pages=211-222; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Myc-Max heterodimers activate a DEAD box gene and interact with multiple E box-related sites in vivo.; citation_author=C Grandori; citation_author=J Mac; citation_author=F Siebelt; citation_author=DE Ayer; citation_author=RN Eisenman; citation_journal_title=EMBO J; citation_volume=15; citation_number=13; citation_pages=4344-4357; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Analysis of Myc bound loci identified by CpG island arrays shows that Max is essential for Myc-dependent repression.; citation_author=DY Mao; citation_author=JD Watson; citation_author=PS Yan; citation_author=D Barsyte-Lovejoy; citation_author=F Khosravi; citation_journal_title=Curr Biol; citation_volume=13; citation_number=14; citation_pages=882-886; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Negative regulation of the mammalian UV response by Myc through association with Miz-1.; citation_author=S Herold; citation_author=M Wanzel; citation_author=V Beuger; citation_author=C Frohme; citation_author=D Beul; citation_journal_title=Mol Cell; citation_volume=10; citation_number=15; citation_pages=509-521; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mechanism for the transcriptional repression by c-Myc on PDGF beta-receptor.; citation_author=H Izumi; citation_author=C Molander; citation_author=LZ Penn; citation_author=A Ishisaki; citation_author=K Kohno; citation_journal_title=J Cell Sci; citation_volume=114; citation_number=16; citation_pages=1533-1544; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Myc represses the p21(WAF1/CIP1) promoter and interacts with Sp1/Sp3.; citation_author=AL Gartel; citation_author=X Ye; citation_author=E Goufman; citation_author=P Shianov; citation_author=N Hay; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=17; citation_pages=4510-4515; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Expression analysis with oligonucleotide microarrays reveals that MYC regulates genes involved in growth, cell cycle, signaling, and adhesion.; citation_author=HA Coller; citation_author=C Grandori; citation_author=P Tamayo; citation_author=T Colbert; citation_author=ES Lander; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=97; citation_number=18; citation_pages=3260-3265; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Characterization of the c-MYC-regulated transcriptome by SAGE: identification and analysis of c-MYC target genes.; citation_author=A Menssen; citation_author=H Hermeking; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=19; citation_pages=6274-6279; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=An integrated database of genes responsive to the Myc oncogenic transcription factor: identification of direct genomic targets.; citation_author=KI Zeller; citation_author=AG Jegga; citation_author=BJ Aronow; citation_author=KA O'Donnell; citation_author=CV Dang; citation_journal_title=Genome Biol; citation_volume=4; citation_number=20; citation_pages=R69; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Myc stimulates nuclearly encoded mitochondrial genes and mitochondrial biogenesis.; citation_author=F Li; citation_author=Y Wang; citation_author=KI Zeller; citation_author=JJ Potter; citation_author=DR Wonsey; citation_journal_title=Mol Cell Biol; citation_volume=25; citation_number=21; citation_pages=6225-6234; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Global mapping of c-Myc binding sites and target gene networks in human B cells.; citation_author=KI Zeller; citation_author=X Zhao; citation_author=CW Lee; citation_author=KP Chiu; citation_author=F Yao; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=22; citation_pages=17834-17839; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Binding of myc proteins to canonical and noncanonical DNA sequences.; citation_author=TK Blackwell; citation_author=J Huang; citation_author=A Ma; citation_author=L Kretzner; citation_author=FW Alt; citation_journal_title=Mol Cell Biol; citation_volume=13; citation_number=23; citation_pages=5216-5224; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Control of pancreas and liver gene expression by HNF transcription factors.; citation_author=DT Odom; citation_author=N Zizlsperger; citation_author=DB Gordon; citation_author=GW Bell; citation_author=NJ Rinaldi; citation_journal_title=Science; citation_volume=303; citation_number=24; citation_pages=1378-1381; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic reconstruction of a functional transcriptional regulatory network.; citation_author=Z Hu; citation_author=PJ Killion; citation_author=VR Iyer; citation_journal_title=Nat Genet; citation_volume=39; citation_number=25; citation_pages=683-687; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome-wide location and function of DNA binding proteins.; citation_author=B Ren; citation_author=F Robert; citation_author=JJ Wyrick; citation_author=O Aparicio; citation_author=EG Jennings; citation_journal_title=Science; citation_volume=290; citation_number=26; citation_pages=2306-2309; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification of serum-inducible genes: different patterns of gene regulation during G0–>S and G1–>S progression.; citation_author=M Wick; citation_author=C Burger; citation_author=S Brusselbach; citation_author=FC Lucibello; citation_author=R Muller; citation_journal_title=J Cell Sci; citation_volume=107; citation_number=27; citation_pages=227-239; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Serum- and polypeptide growth factor-inducible gene expression in mouse fibroblasts.; citation_author=JA Winkles; citation_journal_title=Prog Nucleic Acid Res Mol Biol; citation_volume=58; citation_number=28; citation_pages=41-78; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The transcriptional program in the response of human fibroblasts to serum.; citation_author=VR Iyer; citation_author=MB Eisen; citation_author=DT Ross; citation_author=G Schuler; citation_author=T Moore; citation_journal_title=Science; citation_volume=283; citation_number=29; citation_pages=83-87; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The myc oncogene: MarvelouslY Complex.; citation_author=SK Oster; citation_author=CS Ho; citation_author=EL Soucie; citation_author=LZ Penn; citation_journal_title=Adv Cancer Res; citation_volume=84; citation_number=30; citation_pages=81-154; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=PI3K signaling and miRNA expression during the response of quiescent human fibroblasts to distinct proliferative stimuli.; citation_author=J Gu; citation_author=VR Iyer; citation_journal_title=Genome Biol; citation_volume=7; citation_number=31; citation_pages=R42; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomic targets of the human c-Myc protein.; citation_author=PC Fernandez; citation_author=SR Frank; citation_author=L Wang; citation_author=M Schroeder; citation_author=S Liu; citation_journal_title=Genes Dev; citation_volume=17; citation_number=32; citation_pages=1115-1129; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A global transcriptional regulatory role for c-Myc in Burkitt's lymphoma cells.; citation_author=Z Li; citation_author=S Van Calcar; citation_author=C Qu; citation_author=WK Cavenee; citation_author=MQ Zhang; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=100; citation_number=33; citation_pages=8164-8169; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transcriptional interactions between yeast tRNA genes, flanking genes and Ty elements: a genomic point of view.; citation_author=EC Bolton; citation_author=JD Boeke; citation_journal_title=Genome Res; citation_volume=13; citation_number=34; citation_pages=254-263; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=DAVID: Database for Annotation, Visualization, and Integrated Discovery.; citation_author=G Dennis; citation_author=BT Sherman; citation_author=DA Hosack; citation_author=J Yang; citation_author=W Gao; citation_journal_title=Genome Biol; citation_volume=4; citation_number=35; citation_pages=P3; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=FatiGO: a web tool for finding significant associations of Gene Ontology terms with groups of genes.; citation_author=F Al-Shahrour; citation_author=R Diaz-Uriarte; citation_author=J Dopazo; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=36; citation_pages=578-580; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=HIF-1 inhibits mitochondrial biogenesis and cellular respiration in VHL-deficient renal cell carcinoma by repression of C-MYC activity.; citation_author=H Zhang; citation_author=P Gao; citation_author=R Fukuda; citation_author=G Kumar; citation_author=B Krishnamachary; citation_journal_title=Cancer Cell; citation_volume=11; citation_number=37; citation_pages=407-420; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Myc's mastery of mitochondrial mischief.; citation_author=F Morrish; citation_author=D Hockenbery; citation_journal_title=Cell Cycle; citation_volume=2; citation_number=38; citation_pages=11-13; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=c-MYC apoptotic function is mediated by NRF-1 target genes.; citation_author=F Morrish; citation_author=C Giedt; citation_author=D Hockenbery; citation_journal_title=Genes Dev; citation_volume=17; citation_number=39; citation_pages=240-255; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Activation of the human mitochondrial transcription factor A gene by nuclear respiratory factors: a potential regulatory link between nuclear and mitochondrial gene expression in organelle biogenesis.; citation_author=JV Virbasius; citation_author=RC Scarpulla; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=91; citation_number=40; citation_pages=1309-1313; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=MitoP2, an integrated database on mitochondrial proteins in yeast and man.; citation_author=C Andreoli; citation_author=H Prokisch; citation_author=K Hortnagel; citation_author=JC Mueller; citation_author=M Munsterkotter; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=41; citation_pages=D459-462; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analysis of genomic targets reveals complex functions of MYC.; citation_author=JH Patel; citation_author=AP Loboda; citation_author=MK Showe; citation_author=LC Showe; citation_author=SB McMahon; citation_journal_title=Nat Rev Cancer; citation_volume=4; citation_number=42; citation_pages=562-568; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Myc and Mad bHLHZ domains possess identical DNA-binding specificities but only partially overlapping functions in vivo.; citation_author=L James; citation_author=RN Eisenman; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=43; citation_pages=10429-10434; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mapping DNA-protein interactions in large genomes by sequence tag analysis of genomic enrichment.; citation_author=J Kim; citation_author=AA Bhinge; citation_author=XC Morgan; citation_author=VR Iyer; citation_journal_title=Nat Methods; citation_volume=2; citation_number=44; citation_pages=47-53; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Signaling and circuitry of multiple MAPK pathways revealed by a matrix of global gene expression profiles.; citation_author=CJ Roberts; citation_author=B Nelson; citation_author=MJ Marton; citation_author=R Stoughton; citation_author=MR Meyer; citation_journal_title=Science; citation_volume=287; citation_number=45; citation_pages=873-880; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001798" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Global Identification of Myc Target Genes Reveals Its Direct Role in Mitochondrial Biogenesis and Its E-Box Usage In Vivo"/>
    <meta name="twitter:description" content="The Myc oncoprotein is a transcription factor involved in a variety of human cancers. Overexpression of Myc is associated with malignant transformation. In normal cells, Myc is induced by mitotic signals, and in turn, it regulates the expression of downstream target genes. Although diverse roles of Myc have been predicted from many previous studies, detailed functions of Myc targets are still unclear. By combining chromatin immunoprecipitation (ChIP) and promoter microarrays, we identified a total of 1469 Myc direct target genes, the majority of which are novel, in HeLa cells and human primary fibroblasts. We observed dramatic changes of Myc occupancy at its target promoters in foreskin fibroblasts in response to serum stimulation. Among the targets of Myc, 107 were nuclear encoded genes involved in mitochondrial biogenesis. Genes with important roles in mitochondrial replication and biogenesis, such as POLG, POLG2, and NRF1 were identified as direct targets of Myc, confirming a direct role for Myc in regulating mitochondrial biogenesis. Analysis of target promoter sequences revealed a strong preference for Myc occupancy at promoters containing one of several described consensus sequences, CACGTG, in vivo. This study thus sheds light on the transcriptional regulatory networks mediated by Myc in vivo."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001798.g006"/>

  <meta property="og:title" content="Global Identification of Myc Target Genes Reveals Its Direct Role in Mitochondrial Biogenesis and Its E-Box Usage In Vivo" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001798" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5249'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3137'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3069&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001798">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001798" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001798&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Global Identification of Myc Target Genes Reveals Its Direct Role in Mitochondrial Biogenesis and Its E-Box Usage In Vivo
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jonghwan Kim, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Section of Molecular Genetics and Microbiology, Center for Systems and Synthetic Biology, Institute for Cellular and Molecular Biology, University of Texas at Austin, Austin, Texas, United States of America
                </p>

                <p>
                    Current address: Hematology and Oncology, Children's Hospital Boston and Harvard Medical School, Boston, Massachusetts, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ji-hoon Lee, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Section of Molecular Genetics and Microbiology, Center for Systems and Synthetic Biology, Institute for Cellular and Molecular Biology, University of Texas at Austin, Austin, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vishwanath R. Iyer
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:vishy@mail.utexas.edu">vishy@mail.utexas.edu</a></p>

                <p>Affiliation:
                  Section of Molecular Genetics and Microbiology, Center for Systems and Synthetic Biology, Institute for Cellular and Molecular Biology, University of Texas at Austin, Austin, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 12, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001798</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001798" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001798" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001798" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001798" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001798-g001" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001798-g002" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001798-t001" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001798-g003" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001798-g004" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001798-g005" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001798-g006" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001798">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001798" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The Myc oncoprotein is a transcription factor involved in a variety of human cancers. Overexpression of Myc is associated with malignant transformation. In normal cells, Myc is induced by mitotic signals, and in turn, it regulates the expression of downstream target genes. Although diverse roles of Myc have been predicted from many previous studies, detailed functions of Myc targets are still unclear. By combining chromatin immunoprecipitation (ChIP) and promoter microarrays, we identified a total of 1469 Myc direct target genes, the majority of which are novel, in HeLa cells and human primary fibroblasts. We observed dramatic changes of Myc occupancy at its target promoters in foreskin fibroblasts in response to serum stimulation. Among the targets of Myc, 107 were nuclear encoded genes involved in mitochondrial biogenesis. Genes with important roles in mitochondrial replication and biogenesis, such as POLG, POLG2, and NRF1 were identified as direct targets of Myc, confirming a direct role for Myc in regulating mitochondrial biogenesis. Analysis of target promoter sequences revealed a strong preference for Myc occupancy at promoters containing one of several described consensus sequences, <span class="gene">CACGTG</span>, in vivo. This study thus sheds light on the transcriptional regulatory networks mediated by Myc in vivo.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kim J, Lee J-h, Iyer VR (2008) Global Identification of Myc Target Genes Reveals Its Direct Role in Mitochondrial Biogenesis and Its E-Box Usage In Vivo. PLoS ONE 3(3):
          e1798.
            doi:10.1371/journal.pone.0001798</p><p><strong>Academic Editor: </strong>Peter Fraser, The Babraham Institute, United Kingdom</p><p><strong>Received:</strong> November 2, 2007; <strong>Accepted:</strong> February 12, 2008; <strong>Published:</strong> March 12, 2008</p><p><strong>Copyright:</strong> © 2008 Kim et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported in part by grants from the NIH, NSF, and the US Army.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The Myc oncogene is a transcription factor that has important roles in regulation of the cell cycle, proliferation, differentiation, apoptosis, and miRNA activation <a href="#pone.0001798-Boxer1">[1]</a>, <a href="#pone.0001798-Grandori1">[2]</a>, <a href="#pone.0001798-ODonnell1">[3]</a>, <a href="#pone.0001798-Pelengaris1">[4]</a>. Deregulation of Myc expression by translocation causes Burkitt's lymphoma <a href="#pone.0001798-DallaFavera1">[5]</a>, <a href="#pone.0001798-Taub1">[6]</a>. Several studies have shown that Myc expression is increased in many different tumors, indicating that a high level of Myc favors uncontrolled cell proliferation, inhibits cell differentiation and contributes to tumorigenesis <a href="#pone.0001798-Nesbit1">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Several Myc-related genes, such as c-Myc, N-Myc, L-Myc, and S-Myc are known in mammals <a href="#pone.0001798-Eisenman1">[8]</a>, and among them, c-Myc, N-Myc, and L-Myc have been implicated in the genesis of human tumors <a href="#pone.0001798-Adhikary1">[9]</a>. A basic helix-loop-helix leucine zipper (bHLH/LZ) protein, Myc dimerizes with another bHLH/LZ protein, Max, to function as a sequence specific DNA-binding transcription factor <a href="#pone.0001798-Blackwood1">10</a>, <a href="#pone.0001798-Blackwood2">11</a>. Max forms Max-Max homodimers or heterodimerizes with Mad family proteins <a href="#pone.0001798-Adhikary1">[9]</a>. Myc-Max heterodimers are believed to generally function as an activator complex and Mad-Max heterodimers function as a repressor complex <a href="#pone.0001798-Ayer1">[12]</a>, <a href="#pone.0001798-Grandori2">[13]</a>. However, some studies have shown that Max also mediates Myc-dependent transcriptional repression <a href="#pone.0001798-Mao1">[14]</a>, and that Myc represses the transcription of downstream target genes by interfering with the function of other transcriptional activators such as Miz-1 <a href="#pone.0001798-Herold1">[15]</a>, NF-Y <a href="#pone.0001798-Izumi1">[16]</a>, or Sp1 <a href="#pone.0001798-Gartel1">[17]</a>. Given the functions of Myc in various biological processes, identifying the downstream targets of Myc has been an area of active research. Genome-wide analysis using microarrays and serial analysis of gene expression (SAGE) suggest that Myc could regulate several hundred genes in mammalian cells <a href="#pone.0001798-Coller1">[18]</a>, <a href="#pone.0001798-Menssen1">[19]</a>. More than a thousand targets of Myc in mammalian cells have been identified and compiled in public databases, which however include direct as well as secondary targets <a href="#pone.0001798-Zeller1">[20]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>More recently, genome-wide chromatin immunoprecipitation studies <a href="#pone.0001798-Mao1">[14]</a>, <a href="#pone.0001798-Li1">[21]</a>, <a href="#pone.0001798-Zeller2">[22]</a> have been performed in different cellular contexts to identify approximately 1300 direct binding target sites of Myc. These studies suggest that about 15% of all annotated genes are direct targets of Myc. Although some of these studies were comprehensive, the overlap between Myc targets identified in different large scale studies is small. It is thus likely that our knowledge of the targets and therefore, the functions, of this oncogene is still limited, given the possibility that it regulates distinct sets of targets in different cell types and physiological growth conditions. Changes in the binding of Myc to its target sites in different physiological conditions in the same cell type have not been systematically examined. Although many previous studies have implicated Myc regulation of many cellular processes, in general, it is not clear whether regulation by Myc occurs by virtue of its position at the head of a transcriptional regulatory cascade, or whether these processes are regulated directly by Myc binding to the promoters of the relevant targets. Myc is known to bind to a canonical consensus DNA sequence <span class="gene">CACGTG</span>, termed the E-box, but has also been reported to bind in vitro to several other non-canonical DNA motifs, such as <span class="gene">CATGTG</span>, <span class="gene">CATGCG</span>, <span class="gene">CACGCG</span>, <span class="gene">CACGAG</span>, and <span class="gene">CAACGTG</span> <a href="#pone.0001798-Blackwell1">[23]</a>. However, it is not clear whether Myc exhibits significant selectivity with regard to these sequence motifs in binding to its target sites in the genome.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here, we identified 1469 Myc direct binding target genes in HeLa cells and human foreskin fibroblasts using human core promoter microarrays. A majority of these targets have not been identified as Myc targets before and hence are novel. We observed a dramatic increase in Myc binding to its target promoters in response to serum stimulation in normal fibroblasts. A strikingly significant fraction of Myc direct target genes were those involved in mitochondrial biogenesis. Additionally, we also analyzed the usage of Myc consensus sequences in vivo and show that the canonical E-box <span class="gene">CACGTG</span> and non-canonical <span class="gene">CACGCG</span> sequences are the major motifs that are utilized by Myc in binding its target promoters in vivo.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Myc binds to many target gene promoters in HeLa cells</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We used human promoter microarrays that contained 9,303 proximal promoters to identify the direct targets of Myc in two different human cell lines, HeLa cells and primary foreskin fibroblasts. The genomic regions between –750 bp and +250 bp relative to transcriptional start sites (TSS) of well annotated genes were amplified and printed on the microarrays <a href="#pone.0001798-Odom1">[24]</a>. We carried out independent ChIP reactions from four independently grown HeLa cell cultures and co-hybridized the ChIP samples together with a mock immunoprecipitated sample as the reference. We determined the median ratio of each spot on the microarray from the four independent hybridizations, and also calculated a <em>p</em>-value for significant ratios using a previously described error model <a href="#pone.0001798-Hu1">[25]</a>, <a href="#pone.0001798-Ren1">[26]</a>. We considered spots with a ratio ≥2.0 and P value ≤0.005 to represent direct Myc target loci. Among the 9,303 promoter spots analyzed, 1351 spots were in this category (<a href="#pone-0001798-g001">Figure 1A</a>). Control hybridizations with differentially labeled mock immunoprecipitated samples did not yield any significantly enriched loci at this threshold (data not shown).</p>
<div class="figure" id="pone-0001798-g001"><div class="img"><a name="pone-0001798-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Direct binding targets of Myc in HeLa cells, quiescent foreskin fibroblasts, and serum stimulated foreskin fibroblasts.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Fibroblasts are designated “2091”. Target binding loci of Myc were detected by ChIP-chip analysis in (A) HeLa cells, (B) quiescent (serum starved for 3 days) foreskin fibroblasts, and (C) serum stimulated foreskin fibroblasts (for 4 hours). Each scatter plot shows the red/green ratio (log<sub>2</sub> transformed) on the Y-axis, and the <em>p</em>-value in X-axis. Red data points indicate Myc target loci with red/green log<sub>2</sub> ratio &gt;1 (2-fold), and <em>p</em>&lt;0.005. 1351, 196, and 788 promoters were occupied by Myc respectively in the three conditions. (D) The Venn diagram compares target genes in HeLa cells, quiescent foreskin fibroblasts, and serum stimulated foreskin fibroblasts. A total of 1469 target promoters were identified as targets of Myc. Microarray data are also available through NCBI's Gene Expression Omnibus (GEO, <a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) under GEO Series accession number GSE5197.</p>
<span>doi:10.1371/journal.pone.0001798.g001</span></div>

<h4>Occupancy of target promoters by Myc changes rapidly in response to serum activation of quiescent fibroblasts</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The response to serum stimulation has been used as a model for studying mammalian cell cycle activation and proliferation <a href="#pone.0001798-Wick1">[27]</a>, <a href="#pone.0001798-Winkles1">[28]</a>. These studies have led to the identification of Myc as a critical downstream element of proliferation signaling cascades. As an immediate early response transcription factor, the expression levels of Myc mRNA and protein are rapidly induced by a variety of growth factors and serum stimulation <a href="#pone.0001798-Iyer1">[29]</a>, <a href="#pone.0001798-Oster1">[30]</a>. Based on the fact that expression of Myc is strongly and rapidly induced by serum stimulation, and the fact that Myc is known to be an important regulator of cell proliferation, we hypothesized that Myc might show changes in binding to its target promoters in response to serum stimulation. To determine if this was the case, we measured the occupancy of Myc at target promoters in quiescent foreskin fibroblasts as well as in fibroblasts after serum stimulation. Two independent cultures of quiescent foreskin fibroblasts grown in low serum (0.1% FBS) media for 72 hours, as well as fibroblasts stimulated by serum (10% FBS) for 4 hours were used for ChIP, and hybridizations were carried out as described for HeLa cells above, using a mock immunoprecipitated sample as reference. We averaged the ratios from duplicate microarray hybridizations and used the same threshold as for HeLa cells to define the target promoters occupied by Myc in foreskin fibroblasts. We detected 196 promoters occupied by Myc in quiescent fibroblasts (<a href="#pone-0001798-g001">Figure 1B</a>), whereas 788 promoters were occupied by Myc in response to serum stimulation (<a href="#pone-0001798-g001">Figure 1C</a>). Our result thus reveals differential binding of Myc to its target promoters under different growth conditions in the same cell type. Interestingly, Myc occupied a much larger number of target promoters in HeLa cells although the HeLa cells used in our study were not synchronized. Altogether, we identified 1469 direct downstream targets of Myc from at least one of the conditions we tested.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We compared the Myc targets identified in HeLa cells with the Myc targets in fibroblasts to see if there were any differences in target profiles among cell lines or different growth conditions. There were many common Myc targets among the different conditions we examined. Approximately 86% (676 out of 788) of the Myc targets in serum-stimulated fibroblasts were also targets in HeLa cells (<a href="#pone-0001798-g001">Figure 1D</a>). 86% (169 out of 196) of Myc targets in quiescent fibroblasts were also targets of Myc in serum stimulated fibroblasts, and 95% (185 out of 196) of them were also Myc targets in HeLa cells (<a href="#pone-0001798-g001">Figure 1D</a>). Interestingly, a more continuous analysis of target overlaps based on pair-wise global comparisons of each data set (<a href="#pone-0001798-g002">Figure 2A–C</a>) suggested that the occupancy of target promoters by Myc was largely consistent across cell lines and growth conditions. Thus, the differences between its binding targets in HeLa cells and fibroblasts are likely to reflect lower levels of Myc in fibroblasts, rather than its occupancy of a markedly different set of targets in the different cells or conditions.</p>
<div class="figure" id="pone-0001798-g002"><div class="img"><a name="pone-0001798-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Myc binds largely to the same target loci among different cell types or physiological conditions.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Myc binding to promoters is represented by spots in each scatter plot on a log<sub>2</sub> transformed scale. Comparisons of Myc occupancy at promoters (A) between HeLa cells and quiescent foreskin fibroblasts, (B) between serum stimulated and quiescent foreskin fibroblasts, and (C) between HeLa cells and in serum stimulated foreskin fibroblasts, show a relatively high degree of similarity of Myc binding targets in different cell lines or conditions.(D) As a negative control, a comparison between E2F4 binding to promoters in quiescent fibroblasts and Myc binding to promoters in the same condition was performed. There is little correlation between Myc occupancy and E2F4 occupancy at the promoters (R<sup>2</sup> = 0.19).</p>
<span>doi:10.1371/journal.pone.0001798.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Since Myc expression is induced by serum stimulation, the small set of genes (27) whose promoters were occupied by Myc in serum starved fibroblasts and not in serum stimulated fibroblasts is noteworthy. We examined expression changes of this subset of genes upon serum stimulation, using an expression data set published from our lab <a href="#pone.0001798-Gu1">[31]</a>. We found that this subset of Myc targets included genes that are activated and repressed by serum stimulation, as well as genes that show no significant expression change (Supplementary <a href="#pone.0001798.s001">Figure S1</a>). This behavior is consistent with the previously reported role for Myc as a repressor <a href="#pone.0001798-Mao1">[14]</a>, <a href="#pone.0001798-Herold1">[15]</a>, <a href="#pone.0001798-Izumi1">[16]</a>, <a href="#pone.0001798-Gartel1">[17]</a>, as well as the finding that Myc binding can occur without a corresponding change in mRNA expression levels <a href="#pone.0001798-Zeller2">[22]</a>.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>A comparison with Myc target genes identified in previously published large-scale experiments <a href="#pone.0001798-Mao1">[14]</a>, <a href="#pone.0001798-Zeller2">[22]</a>, <a href="#pone.0001798-Fernandez1">[32]</a>, <a href="#pone.0001798-Li2">[33]</a> also confirmed that there were many targets in common between our study and previous studies, even though each of these studies used different cell lines and different experimental platforms (see Supplementary <a href="#pone.0001798.s003">Table S1</a>). These overlaps do not however reflect some kind of broad ChIP-chip artifact. As a negative control for our pair-wise comparisons, we used data from an E2F4 ChIP-chip experiment. As expected, the Myc occupancy and E2F4 occupancy datasets showed much weaker correlations than the different Myc datasets (<a href="#pone-0001798-g002">Figure 2D</a>).</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>The 1469 target genes of Myc in HeLa cells and foreskin fibroblasts we report here form one of the largest sets of direct Myc targets identified in a single experiment. 1104 of our 1469 targets were not identified in other recent large-scale experiments <a href="#pone.0001798-Mao1">[14]</a>, <a href="#pone.0001798-Zeller2">[22]</a>, <a href="#pone.0001798-Fernandez1"></a>,<a href="#pone.0001798-Bolton1">[34]</a> (also listed in Supplementary <a href="#pone.0001798.s003">Table S1</a>). This is likely in part because many of these studies used different cell types. For example, Li <em>et al</em> used a smaller promoter microarray to identify direct Myc targets in Burkitt lymphoma cells <a href="#pone.0001798-Li2">[33]</a>. However, even though the overlap between the two target sets was modest (46%), it was statistically highly significant (<em>p</em>&lt;10<sup>−50</sup>). Based on our results, and extrapolating from the coverage of promoters on this array, we estimate that the number of direct target genes of Myc in human cells totals approximately 16% of all human genes.</p>


<h4>Functional categories of Myc targets</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The direct Myc target genes we identified fell into a variety of functional categories that suggested a role for Myc in diverse cellular processes. We used DAVID <a href="#pone.0001798-Dennis1">[35]</a> and FatiGO <a href="#pone.0001798-AlShahrour1">[36]</a> to quantitate the enrichment of different functional groups of genes among its targets and confirm the involvement of Myc in many processes identified in previous large scale experiments, such as DNA, RNA, and protein metabolism, transport, apoptosis, cell cycle, signal transduction, and cellular proliferation (Supplementary <a href="#pone.0001798.s004">Table S2</a>). KEGG pathway analysis <a href="#pone.0001798-Dennis1">[35]</a> revealed that the Myc direct target genes we identified by ChIP-chip are additionally involved in many previously known and unknown pathways (<a href="#pone-0001798-t001">Table 1</a>). Ribosomal genes comprised one of the most prominent previously known sets of genes that Myc regulates directly, but our analysis identified that it also directly regulates genes involved in oxidative phosphorylation, aminoacyl-tRNA biosynthesis, and glutamate metabolism.</p>
<div class="figure" id="pone-0001798-t001"><div class="img"><a name="pone-0001798-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001798.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001798.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Functional categories of Myc target genes as indicated by KEGG pathways.</span></strong></p><span>doi:10.1371/journal.pone.0001798.t001</span></div>

<h4>Direct roles of Myc in mitochondrial biogenesis</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>It is known that Myc activates nuclear-encoded mitochondrial gene expression <a href="#pone.0001798-Coller1">[18]</a>, and that many direct targets of Myc are involved in mitochondrial function <a href="#pone.0001798-Mao1">[14]</a>, <a href="#pone.0001798-Fernandez1">[32]</a>, <a href="#pone.0001798-Li2">[33]</a>. Myc directly regulates TFAM, a transcription factor important for mitochondrial transcription and mitochondrial DNA replication <a href="#pone.0001798-Li1">[21]</a>, as well as PGC-1β, a regulator of mitochondrial biogenesis <a href="#pone.0001798-Zhang1">[37]</a>. However, the full extent of the relationship between the direct role of Myc acting at its target promoters and mitochondrial function has not been systematically addressed. In addition to these previously reported Myc targets that are involved in mitochondrial biogenesis, it is possible that additional targets in this process are also directly regulated by Myc by its binding to their promoters, indicative of a concerted and multi-pronged regulation of mitochondrial biogenesis by Myc. Indeed, our data and analysis revealed that many functions and pathways that Myc target genes are involved in are closely related to general mitochondrial functions, such as metabolism, apoptosis, oxidative phosphorylation, and glutamate metabolism. It has been suggested that the proliferation and cell cycle progression caused by Myc is in part due to its activation of mitochondrial biogenesis and oxidative phosphorylation <a href="#pone.0001798-Li1">[21]</a>, <a href="#pone.0001798-Morrish1">[38]</a>, <a href="#pone.0001798-Morrish2">[39]</a>, which are critical for cell proliferation. However, only a few genes in these pathways have been identified as direct targets of Myc <a href="#pone.0001798-Morrish2">[39]</a>. We found that Myc occupies the promoters of genes such as POLG, POLG2, and NRF1, which have important roles in mitochondrial biogenesis. The promoter of TFBM1, a mitochondrial transcription specificity factor, showed a positive red/green ratio on the promoter microarray, indicative of occupancy by Myc, although it did not meet the <em>p</em>-value threshold for defining targets (R/G &gt;2, <em>p</em> = 0.0283). NRF1, one of the direct Myc targets we identified, is known to regulate nuclear encoded mitochondrial genes including cytochrome c <a href="#pone.0001798-Morrish2">[39]</a>, <a href="#pone.0001798-Virbasius1">[40]</a>. Our result thus suggests possible co-regulation of mitochondrial biogenesis by a feed-forward network loop involving Myc and NRF1. It has been previously shown that the apoptotic function of Myc is mediated by NRF1 target genes, and that the balance of mitochondrial genes regulated by Myc and NRF1 is important in cell viability <a href="#pone.0001798-Morrish2">[39]</a>. Our finding suggests that Myc could be an upstream regulator of NRF1 and both of these transcription factors are thus important regulators of mitochondrial biogenesis and diverse mitochondrial functions.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In order to systematically quantitate the extent of the involvement of Myc in mitochondrial biogenesis, we searched MitoP2, an integrated database of mitochondrial proteins <a href="#pone.0001798-Andreoli1">[41]</a> which lists 785 RefSeq annotated genes, of which promoters for 337 were represented on our promoter arrays. 107 of these 337 mitochondrial genes showed a red/green ratio greater than 2 and are listed in Supplementary <a href="#pone.0001798.s005">Table S3</a>. The red/green ratios of the set of all mitochondrial gene promoters tended to be higher than the overall red/green ratio of all promoters on the array, reflecting higher occupancy of the former set of promoters by Myc (<a href="#pone-0001798-g003">Figure 3A</a>).</p>
<div class="figure" id="pone-0001798-g003"><div class="img"><a name="pone-0001798-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Myc directly regulates genes involved in mitochondrial biogenesis.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A) The occupancy of Myc at all promoters in HeLa cells (gray bars) and at the subset of promoters of genes involved in mitochondrial biogenesis (black bars) is shown. (B) The histogram shows expression profiles of all genes in HeLa cells detected by cDNA microarray (gray bars) and expression of genes involved in mitochondrial biogenesis (black bars). (C) Myc occupancy at the promoters of selected target genes were independently verified by quantitative real-time PCR. The promoter of the actin gene (ACTB) served as a negative control. The graph shows fold enrichment over background, as an average of triplicate PCR reactions from two independent biological replicates. 20 of 24 tested promoters showed significant enrichment compared to the negative control. (D) A gene-by-gene visualization of the correlation between Myc occupancy at the promoters of genes in mitochondrial biogenesis and gene expression in HeLa cells.</p>
<span>doi:10.1371/journal.pone.0001798.g003</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>The relationship between Myc binding and target gene expression is not a simple one <a href="#pone.0001798-Patel1">[42]</a>. However, ectopic expression of Myc is known to transcriptionally activate many nuclear genes involved in mitochondrial biogenesis <a href="#pone.0001798-Li1">[21]</a>. To determine if this was also the case for the binding targets we identified by ChIP-chip, we examined the level of mitochondrial gene expression in microarray data. In HeLa cells containing high levels of Myc, the expression levels of mitochondrial genes tended to be higher than the background (<a href="#pone-0001798-g003">Figure 3B</a>), which is similar to the pattern of Myc occupancy of its the target promoters (<a href="#pone-0001798-g003">Figure 3A</a>). We independently verified the occupancy by Myc of the promoters of many of these mitochondrial function genes, as well as several other targets identified by our core promoters using quantitative real-time PCR (<a href="#pone-0001798-g003">Figure 3C</a>). A gene-by-gene comparison of Myc binding and target gene expression levels also showed that most of the nuclear mitochondrial genes whose promoters were occupied by Myc were transcriptionally active (<a href="#pone-0001798-g003">Figure 3D</a>). Taken together, our data strongly support the notion that Myc directly binds to the promoters of many nuclear encoded mitochondrial genes and positively regulates the transcription of these genes.</p>


<h4>In vivo usage of E-box elements by Myc</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Myc regulates its downstream target genes by binding to cis-acting DNA motifs called E-boxes. The canonical E-box sequences <span class="gene">CACGTG</span> and <span class="gene">CATGTG</span> <a href="#pone.0001798-Blackwell1">[23]</a> as well as non-canonical E-boxes, such as <span class="gene">CATGCG</span>, <span class="gene">CACGCG</span>, <span class="gene">CACGAG</span>, and <span class="gene">CAACGTG</span> have been identified as cis-acting elements bound by Myc <a href="#pone.0001798-James1">[43]</a>. Although several E-boxes and E-box derivatives have been identified, the E-box sequence preference of Myc for binding to its chromosomal sites in vivo remains largely unknown. To elucidate the relationship between Myc binding and each of the E-box derivatives occurring in proximal promoters in vivo, we analyzed our ChIP-chip data set.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We first examined the distribution of the previously defined E-box sequences in all human core promoters, considering the regions from 3,000 bp upstream to 1,000 bp downstream of the transcription start sites of 18,193 well-annotated human genes in RefSeq. Three of the six E-box sequences we analyzed–<span class="gene">CACGTG</span>, <span class="gene">CATGCG</span> and <span class="gene">CACGCG</span>–showed a relative enrichment in the promoter region between 800 bp upstream and 200 bp downstream of transcription start site (TSS), whereas in contrast, the sequence <span class="gene">CATGTG</span> showed a relative depletion in the same region (<a href="#pone-0001798-g004">Figure 4A</a>). The sequences <span class="gene">CACGAG</span> and <span class="gene">CAACGTG</span> did not show any significant bias within the 4 kb core promoter region that we analyzed. When we considered only the promoter region between 800 bp upstream and 200 bp downstream, about 55% of all promoters of the RefSeq genes contained at least one of the six E-box elements (<a href="#pone-0001798-g004">Figure 4B</a>). However, each E-box sequence was found in less than 20% of all human promoters.</p>
<div class="figure" id="pone-0001798-g004"><div class="img"><a name="pone-0001798-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Distribution of E-box elements in human promoters and in vivo usage of E-box sequences in HeLa cells.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) Distribution (number of occurrences per bin as a function of position relative to the transcription start site (+1, TSS)) of each E-box sequence is shown. Of the six previously defined E-box sequences, <span class="gene">CACGTG</span>, <span class="gene">CATGCG</span>, and <span class="gene">CACGCG</span> were well clustered near the TSS region. A bin size of 50, and a total of 18,193 promoter sequences spanning 3 kb upstream and 1 kb downstream of the TSS were used in this analysis. (B) Number of promoters that contained each E-box element in the region between −800 bp, and +200 bp of the TSS. (C) The relationship between the presence of each E-box sequence in the promoter region (between −800 bp, and +200 bp of the TSS), and Myc binding represented by the log<sub>2</sub> red/green ratio on the X axis. The Y-axis represents the frequency of each E-box element. A moving window average (size 100) was applied to the frequency of the E-box sequence. The canonical E-box sequence <span class="gene">CACGTG</span>, and the non-canonical E-box sequence <span class="gene">CACGCG</span> in promoters showed the strongest correlation with Myc binding to its target promoters in HeLa cells.</p>
<span>doi:10.1371/journal.pone.0001798.g004</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Approximately 64% of promoters that were occupied by Myc in our ChIP-chip analysis contained at least one of the six E-box sequences. Thus, 36% of the promoters occupied by Myc in vivo lacked any known E-box sequence. We then tested the relationship between the presence of each E-box sequence and Myc occupancy at the promoters. We found that there was a strong correlation between the presence of the canonical E-box sequence <span class="gene">CACGTG</span> and Myc occupancy of a promoter. The E-box sequence <span class="gene">CACGCG</span> showed lower correlation, while <span class="gene">CACGAG</span>, <span class="gene">CATGCG</span>, <span class="gene">CATGTG</span>, and <span class="gene">CAACGTG</span> did not show any significant correlation with Myc binding (<a href="#pone-0001798-g004">Figure 4C</a>). Another postulated binding sequence, <span class="gene">CACATG</span> (the reverse complement of <span class="gene">CATGTG</span>) also showed no enrichment in Myc target promoters in our data (Supplementary <a href="#pone.0001798.s002">Figure S2</a>), although <span class="gene">CACATG</span>/<span class="gene">CATGTG</span> have been previously shown to be enriched among Myc binding sites <a href="#pone.0001798-Zeller2">[22]</a>. However, the previous study also examined intronic and intergenic sites which were not included in our promoter arrays, and this could potentially reflect differences in the occurrence of Myc motifs in different types of target sites. Overall, our data show that Myc has a preference to bind only some of the previously defined E-box sequences in vivo. Interestingly although the E-box <span class="gene">CATGCG</span> is enriched at proximal promoter regions (<a href="#pone-0001798-g004">Figure 4A</a>) it did not show a significant correlation with Myc occupancy in vivo.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>To obtain a more detailed understanding of in vivo Myc usage of its binding sites, we examined the relationship between Myc occupancy and the presence of each E-box sequence within the subset of promoters that were most strongly or most weakly occupied by Myc in vivo. More than half the promoters that were most strongly occupied by Myc (the top 2%) contained the <span class="gene">CACGTG</span> sequence (<a href="#pone-0001798-g005">Figure 5A</a>). The promoters most strongly occupied by Myc showed a increased frequency of all the previously defined E-box sequences except, interestingly, <span class="gene">CATGTG</span>. We observed the same trend when we examined the top or bottom 10% of promoters occupied by Myc (<a href="#pone-0001798-g005">Figure 5B</a>). Our data thus suggest that Myc has a marked preference to utilize promoters in vivo that contain any of the known E-box sequences except <span class="gene">CATGTG</span>. However, E-box sequences <span class="gene">CACGTG</span> and <span class="gene">CACGCG</span> showed the strongest correlations with Myc occupancy (<a href="#pone-0001798-g004">Figure 4C</a>).</p>
<div class="figure" id="pone-0001798-g005"><div class="img"><a name="pone-0001798-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The canonical E-box sequence <span class="gene">CATGTG</span> is not preferred by Myc.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>The relationship between the presence of six E-box sequences in the promoter (between −800 bp and +200 bp of TSS) and Myc binding was analyzed. (A) Top and bottom 2% Myc occupancy data and (B) top and bottom 10% of Myc occupancy data were used. Black bars in both panels represent target promoters that were occupied by Myc and gray bars represent promoters that were not occupied by Myc. Promoters that contained the E-box sequence CATGTG did not show a positive correlation with Myc binding.</p>
<span>doi:10.1371/journal.pone.0001798.g005</span></div><a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>To determine whether the most strongly preferred Myc sites, <span class="gene">CACGTG</span> and <span class="gene">CACGCG</span>, were both also used under distinct physiological conditions that might affect Myc binding, we separately analyzed ChIP-chip data from quiescent and serum-stimulated foreskin fibroblasts. Promoters containing the highly preferred <span class="gene">CACGTG</span> sequence were preferentially occupied by Myc in both conditions in foreskin fibroblasts, but promoters with the <span class="gene">CACGCG</span> sequence did not show significant correlation with Myc binding (<a href="#pone-0001798-g006">Figure 6A and B</a>). We also did not observe a strong correlation between any of the other E-box sequences and Myc binding in serum-starved fibroblasts, or in serum-stimulated fibroblasts. As a control, we examined the occupancy of E2F4 at these same promoters in quiescent fibroblasts. Despite the fact that there are several common targets for both Myc and E2F4, we did not observe a significant correlation between the presence of the E-box sequences and E2F4 binding, which implies that the E-box sequence is specific to Myc, but not to E2F4 (<a href="#pone-0001798-g006">Figure 6C</a>).</p>
<div class="figure" id="pone-0001798-g006"><div class="img"><a name="pone-0001798-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001798" data-uri="info:doi/10.1371/journal.pone.0001798.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001798.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001798.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001798.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001798.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>E-box usage in foreskin fibroblasts.</span></strong></p><a id="article1.body1.sec2.sec5.fig3.caption1.p1" name="article1.body1.sec2.sec5.fig3.caption1.p1"></a><p>The relationship between the presence of the two E-box sequences (<span class="gene">CACGTG</span>, and <span class="gene">CACGCG</span>) and Myc binding represented by the log<sub>2</sub> red/green ratio in (A) quiescent foreskin fibroblasts, and (B) in serum stimulated fibroblasts is shown. Only the canonical E-box sequence <span class="gene">CACGTG</span> showed good correlation with Myc binding in both conditions. Average moving window (size 100) was applied to the frequency of E-box sequence. (D) E2F4 ChIP-chip data was used as a negative control, and showed no correlation with E-box elements.</p>
<span>doi:10.1371/journal.pone.0001798.g006</span></div><a id="article1.body1.sec2.sec5.p6" name="article1.body1.sec2.sec5.p6"></a><p>Taken together, our data indicate that the oncoprotein Myc strongly prefers to use promoters containing the E-box sequence <span class="gene">CACGTG</span> in vivo, although most of the other E-box sequences except <span class="gene">CATGTG</span> appeared to be utilized in vivo to varying extents. The discrepancy between our observations and previous findings might be due to the fact that most of the previous experiments were performed in vitro, and also did not consider the number of actual E-box sequences present in the promoter. We observed an increase in Myc binding to promoters that contain each of the previously defined E-box sequences, except <span class="gene">CATGTG</span>. However, the contribution of most E-box sequences to Myc binding in vivo is low except for the E-box sequence <span class="gene">CACGTG</span>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cell lines and antibody</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Human HeLa cells and human fibroblasts derived from foreskin (ATCC CRL 2091) were grown to about 60% confluency in 15 cm plates in DMEM containing glucose (1 g/liter), antibiotics, and 10% fetal bovine serum (FBS, Hyclone). Exponentially growing HeLa cells were used for ChIP. For the serum starvation experiment, fibroblasts were washed once with phosphate-buffered saline (PBS) and twice with DMEM medium lacking FBS. Low-serum DMEM medium (containing 0.1% FBS) was added to the plates and incubated for 72 hours. For serum activation, cells in low-serum medium were washed twice with PBS, and DMEM medium containing 10% FBS was added to the cells and incubated for 4 hours. For crosslinking, 37% formaldehyde was directly added to the media, and ChIP procedures were subsequently performed using an anti-Myc antibody (sc-764x, Santa Cruz).</p>


<h4>Chromatin immunoprecipitation (ChIP)</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>HeLa cells and fibroblasts were cross-linked by the addition of formaldehyde (1% final concentration) directly to tissue culture plates and incubated for 7 min at room temperature. Cross-linking was terminated by adding glycine to a final concentration of 125 mM. Cells were washed with cold PBS containing PMSF, scraped off the plates, collected by centrifugation, and washed again with PBS. After centrifugation, the pellet was resuspended in SDS lysis buffer (1% SDS, 10 mM EDTA, 50 mM Tris-Cl pH 8.1, plus protease inhibitors) and incubated at room temperature for 20 min. Cells were sonicated on ice, and fragmented DNA was visualized on an agarose gel (average size 1 kb). The sample was centrifuged at 12000 rpm at 4°C for 10 min, and ChIP dilution buffer (1% Triton X-100, 2 mM EDTA, 20 mM Tris-Cl pH 8.1, 150 mM NaCl, plus protease inhibitors) was added to the collected supernatant. The sample was precleared with protein A-agarose beads (previously washed with ChIP dilution buffer) at 4°C for 1 hour. Precleared chromatin was incubated with an antibody against Myc (sc-764x, Santa Cruz) at 4°C overnight. For the mock IP controls, the antibody was not added. Prewashed protein A-agarose beads were added and protein-DNA complexes were recovered after a 2 hour incubation at 4°C. Immunoprecipitated complexes were successively washed twice with low salt wash buffer (0.1% sodium deoxycholate, 1% Triton X-100, 1 mM EDTA, 50 mM HEPES pH 7.5, 150 mM NaCl), once with high-salt wash buffer (0.1% sodium deoxycholate, 1% Triton X-100, 1 mM EDTA, 50 mM HEPES pH 7.5, 500 mM NaCl), once with LiCl wash buffer (250 mM LiCl, 0.5% NP-40, 0.5% sodium deoxycholate, 1 mM EDTA, 10 mM Tris-Cl pH 8.1), and twice with TE buffer (10 mM Tris-Cl pH 7.5, 1 mM EDTA). SDS elution buffer was added and incubated at 65°C for 30 min to recover protein-DNA complexes. Crosslinks were reversed by incubating at 65 °C overnight. The sample was treated with RNase A and Proteinase K, extracted with phenol/chloroform, and precipitated. The pellet was resuspended in 25 µl of water.</p>


<h4>Ligation mediated PCR (LM-PCR)</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>LM-PCR was performed as described <a href="#pone.0001798-Ren1">[26]</a> with minor modifications. LM-PCR consisted of three steps: blunting of ChIP DNA, ligation of linkers to ChIP DNA, and PCR. Approximately 200 ng of immunoprecipitated DNA was diluted to a final volume of 100 µl with water. A mixture of 11 µl 10X T4 DNA polymerase buffer, 0.5 µl BSA (10 mg/ml), 0.5 µl dNTP mix (25 mM each), and 0.2 µl T4 DNA polymerase (3 U/µl) was added, mixed by pipetting, and incubated at 12°C for 20 min. After the incubation, 11.5 µl 3M sodium acetate, and 0.5 µl glycogen (20 mg/ml) were added. The sample was extracted with 120 µl of phenol/chloroform/isoamyl alcohol (25:24:1) in a 0.5 ml Phase-Lock Gel tube (Eppendorf) and centrifuged for 5 min at maximum speed in a microcentrifuge. The upper phase was transferred to a new 1.5-ml Eppendorf tube and precipitated with 230 µl ice-cold ethanol (100%). After centrifugation, the pellet was washed with 70% ethanol and air dried briefly. The pellet was resuspended in 25 µl water and placed on ice. Ligase mix containing 8 µl water, 10 µl 5X ligase buffer, 6.7 µl annealed linkers, and 0.5 µl T4 DNA ligase was added and incubated overnight at 16°C. Linkers were prepared as follows: 300 µl each of oligo Long (40 µM stock, sequence: <span class="gene">5′-GCGGTGACCCGGGAGATCTGAATTC</span>), and oligo Short (40 µM stock, sequence: <span class="gene">5′-GAATTCAGATC</span>) were mixed. 50-µl aliquots of oligo mixture were placed at 95°C for 5 min, 70°C for 5 min and placed at room temperature for 20 min. Aliquots were then transferred to 4°C and allowed to stand overnight. The annealed linkers were stored at −20°C.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>To precipitate the linker-ligated DNA, 6 µl of 3 M sodium acetate (pH 5.2) and 130 µl of cold ethanol were added to the tube and centrifuged for 15 min at 4°C. The pellet was washed with 70% ethanol and resuspended in 25 µl water. A mixture of 4 µl 10X PCR buffer, 6.75 µl water, 2 µl low TTP mix (5 mM each dATP, dCTP, dGTP, and 2 mM dTTP), 1 µl Cy3-dUTP or Cy5-dUTP, and 1.25 µl of oligo Long (40 µM), was added to the sample DNA and transferred to PCR tubes. PCR was performed as follows: first, 2 min at 55°C, followed by addition of a mixture of 8 µl water,1 µl 10X PCR buffer, and 1 µl Taq polymerase (5 U/µl ), 5 min at 72°C, 2 min at 95°C, 35 cycles of 30 sec at 95°C, 30 sec at 55°C, and 1 min at 72°C. After PCR amplification, 5 µl of sample was loaded on a 1.5% agarose gel to verify the size of amplified ChIP DNA. The average size of amplified ChIP DNA was approximately 400 bp. The remaining PCR product was purified using a Qiaquick PCR purification kit (Qiagen), and eluted in 13 µl of water.</p>


<h4>Microarrays and data analysis</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Human core promoter microarrays were manufactured as described <a href="#pone.0001798-Kim1">[44]</a>. Amplified and fluorescently labeled ChIP and reference DNA samples were simultaneously hybridized to the microarrays and ChIP enrichment of target loci was calculated based on significant ratios of ChIP signal to reference signal at each spot. Independent biological replicate experiments were combined by averaging the red/green ratio of each spot on microarrays. The confidence of the ratio measurement was calculated as a <em>p</em>-value using a previously described error model <a href="#pone.0001798-Hu1">[25]</a>, <a href="#pone.0001798-Ren1">[26]</a>, <a href="#pone.0001798-Roberts1">[45]</a>. Microarray elements whose red/green ratio was ≥2 and <em>p</em>-value was ≤0.005 were considered to be direct targets of Myc.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001798"></div><a name="pone.0001798.s001" id="pone.0001798.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001798.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Gene expression changes for the subset of genes whose promoters were occupied by Myc in serum starved fibroblasts (designated “2091”) but not in serum stimulated fibroblasts. Expression data is taken from Gu &amp; Iyer (2006) Genome Biol 7: R42, and is shown for the genes in this category for which data was available in the previous dataset.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001798.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.52 MB TIF)</p>
<a name="pone.0001798.s002" id="pone.0001798.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001798.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Occurrence of the CACATG motif among target promoters, compared to the canonical CACGTG and CACGCG motifs. No enrichment is evident.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001798.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.62 MB TIF)</p>
<a name="pone.0001798.s003" id="pone.0001798.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001798.s003">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">List of 1469 Myc direct target genes identified in this work, and overlap with previously identified targets from global ChIP studies. The occurrence of the consensus motif CACGTG among target promoters is also indicated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001798.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.19 MB XLS)</p>
<a name="pone.0001798.s004" id="pone.0001798.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001798.s004">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Functional categories of Myc target genes identified in this study.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001798.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.37 MB XLS)</p>
<a name="pone.0001798.s005" id="pone.0001798.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001798.s005">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Nuclear encoded genes involved in mitochondrial biogenesis whose promoters were occupied by Myc.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001798.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: VI JK. Performed the experiments: JK JL. Analyzed the data: JK. Wrote the paper: VI JK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001798-Boxer1" id="pone.0001798-Boxer1"></a>Boxer LM, Dang CV (2001) Translocations involving c-myc and c-myc function. Oncogene  20: 5595–5610.  <ul class="find" data-citedArticleID="1074758" data-doi="10.1038/sj.onc.1204595"><li><a href="http://dx.doi.org/10.1038/sj.onc.1204595" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translocations+involving+c-myc+and+c-myc+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translocations+involving+c-myc+and+c-myc+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001798-Grandori1" id="pone.0001798-Grandori1"></a>Grandori C, Cowley SM, James LP, Eisenman RN (2000) The Myc/Max/Mad network and the transcriptional control of cell behavior. Annu Rev Cell Dev Biol  16: 653–699.  <ul class="find" data-citedArticleID="1074772" data-doi="10.1146/annurev.cellbio.16.1.653"><li><a href="http://dx.doi.org/10.1146/annurev.cellbio.16.1.653" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Myc%2FMax%2FMad+network+and+the+transcriptional+control+of+cell+behavior." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Myc%2FMax%2FMad+network+and+the+transcriptional+control+of+cell+behavior.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001798-ODonnell1" id="pone.0001798-ODonnell1"></a>O'Donnell KA, Wentzel EA, Zeller KI, Dang CV, Mendell JT (2005) c-Myc-regulated microRNAs modulate E2F1 expression. Nature  435: 839–843.  <ul class="find" data-citedArticleID="1074804" data-doi="10.1038/nature03677"><li><a href="http://dx.doi.org/10.1038/nature03677" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=c-Myc-regulated+microRNAs+modulate+E2F1+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22c-Myc-regulated+microRNAs+modulate+E2F1+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001798-Pelengaris1" id="pone.0001798-Pelengaris1"></a>Pelengaris S, Khan M, Evan G (2002) c-MYC: more than just a matter of life and death. Nat Rev Cancer  2: 764–776.  <ul class="find" data-citedArticleID="1074812" data-doi="10.1038/nrc904"><li><a href="http://dx.doi.org/10.1038/nrc904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=c-MYC%3A+more+than+just+a+matter+of+life+and+death." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22c-MYC%3A+more+than+just+a+matter+of+life+and+death.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001798-DallaFavera1" id="pone.0001798-DallaFavera1"></a>Dalla-Favera R, Bregni M, Erikson J, Patterson D, Gallo RC, et al.  (1982) Human c-myc onc gene is located on the region of chromosome 8 that is translocated in Burkitt lymphoma cells. Proc Natl Acad Sci USA  79: 7824–7827.  <ul class="find" data-citedArticleID="1074762" data-doi="10.1073/pnas.79.24.7824"><li><a href="http://dx.doi.org/10.1073/pnas.79.24.7824" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+c-myc+onc+gene+is+located+on+the+region+of+chromosome+8+that+is+translocated+in+Burkitt+lymphoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+c-myc+onc+gene+is+located+on+the+region+of+chromosome+8+that+is+translocated+in+Burkitt+lymphoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001798-Taub1" id="pone.0001798-Taub1"></a>Taub R, Kirsch I, Morton C, Lenoir G, Swan D, et al.  (1982) Translocation of the c-myc gene into the immunoglobulin heavy chain locus in human Burkitt lymphoma and murine plasmacytoma cells. Proc Natl Acad Sci USA  79: 7837–7841.  <ul class="find" data-citedArticleID="1074818" data-doi="10.1073/pnas.79.24.7837"><li><a href="http://dx.doi.org/10.1073/pnas.79.24.7837" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translocation+of+the+c-myc+gene+into+the+immunoglobulin+heavy+chain+locus+in+human+Burkitt+lymphoma+and+murine+plasmacytoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translocation+of+the+c-myc+gene+into+the+immunoglobulin+heavy+chain+locus+in+human+Burkitt+lymphoma+and+murine+plasmacytoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001798-Nesbit1" id="pone.0001798-Nesbit1"></a>Nesbit CE, Tersak JM, Grove LE, Drzal A, Choi H, et al.  (2000) Genetic dissection of c-myc apoptotic pathways. Oncogene  19: 3200–3212.  <ul class="find" data-citedArticleID="1074802" data-doi="10.1038/sj.onc.1203636"><li><a href="http://dx.doi.org/10.1038/sj.onc.1203636" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+dissection+of+c-myc+apoptotic+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+dissection+of+c-myc+apoptotic+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001798-Eisenman1" id="pone.0001798-Eisenman1"></a>Eisenman RN (2001) Deconstructing myc. Genes Dev  15: 2023–2030.  <ul class="find" data-citedArticleID="1074766" data-doi="10.1101/gad928101"><li><a href="http://dx.doi.org/10.1101/gad928101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deconstructing+myc." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deconstructing+myc.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001798-Adhikary1" id="pone.0001798-Adhikary1"></a>Adhikary S, Eilers M (2005) Transcriptional regulation and transformation by Myc proteins. Nat Rev Mol Cell Biol  6: 635–645.  <ul class="find" data-citedArticleID="1074742" data-doi="10.1038/nrm1703"><li><a href="http://dx.doi.org/10.1038/nrm1703" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulation+and+transformation+by+Myc+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulation+and+transformation+by+Myc+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001798-Blackwood1" id="pone.0001798-Blackwood1"></a>Blackwood EM, Eisenman RN (1991) Max: a helix-loop-helix zipper protein that forms a sequence-specific DNA-binding complex with Myc. Science  251: 1211–1217.  <ul class="find" data-citedArticleID="1074752" data-doi="10.1126/science.2006410"><li><a href="http://dx.doi.org/10.1126/science.2006410" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Max%3A+a+helix-loop-helix+zipper+protein+that+forms+a+sequence-specific+DNA-binding+complex+with+Myc." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Max%3A+a+helix-loop-helix+zipper+protein+that+forms+a+sequence-specific+DNA-binding+complex+with+Myc.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001798-Blackwood2" id="pone.0001798-Blackwood2"></a>Blackwood EM, Luscher B, Kretzner L, Eisenman RN (1991) The Myc:Max protein complex and cell growth regulation. Cold Spring Harb Symp Quant Biol  56: 109–117.  <ul class="find" data-citedArticleID="1074754" data-doi="10.1101/sqb.1991.056.01.015"><li><a href="http://dx.doi.org/10.1101/sqb.1991.056.01.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Myc%3AMax+protein+complex+and+cell+growth+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Myc%3AMax+protein+complex+and+cell+growth+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001798-Ayer1" id="pone.0001798-Ayer1"></a>Ayer DE, Kretzner L, Eisenman RN (1993) Mad: a heterodimeric partner for Max that antagonizes Myc transcriptional activity. Cell  72: 211–222.  <ul class="find" data-citedArticleID="1074748" data-doi="10.1016/0092-8674(93)90661-9"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)90661-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mad%3A+a+heterodimeric+partner+for+Max+that+antagonizes+Myc+transcriptional+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mad%3A+a+heterodimeric+partner+for+Max+that+antagonizes+Myc+transcriptional+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001798-Grandori2" id="pone.0001798-Grandori2"></a>Grandori C, Mac J, Siebelt F, Ayer DE, Eisenman RN (1996) Myc-Max heterodimers activate a DEAD box gene and interact with multiple E box-related sites in vivo. EMBO J  15: 4344–4357.  <ul class="find" data-citedArticleID="1074774"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Myc-Max+heterodimers+activate+a+DEAD+box+gene+and+interact+with+multiple+E+box-related+sites+in+vivo.&amp;auth=&amp;atitle=Myc-Max+heterodimers+activate+a+DEAD+box+gene+and+interact+with+multiple+E+box-related+sites+in+vivo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Myc-Max+heterodimers+activate+a+DEAD+box+gene+and+interact+with+multiple+E+box-related+sites+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Myc-Max+heterodimers+activate+a+DEAD+box+gene+and+interact+with+multiple+E+box-related+sites+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001798-Mao1" id="pone.0001798-Mao1"></a>Mao DY, Watson JD, Yan PS, Barsyte-Lovejoy D, Khosravi F, et al.  (2003) Analysis of Myc bound loci identified by CpG island arrays shows that Max is essential for Myc-dependent repression. Curr Biol  13: 882–886.  <ul class="find" data-citedArticleID="1074794" data-doi="10.1016/s0960-9822(03)00297-5"><li><a href="http://dx.doi.org/10.1016/s0960-9822(03)00297-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+Myc+bound+loci+identified+by+CpG+island+arrays+shows+that+Max+is+essential+for+Myc-dependent+repression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+Myc+bound+loci+identified+by+CpG+island+arrays+shows+that+Max+is+essential+for+Myc-dependent+repression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001798-Herold1" id="pone.0001798-Herold1"></a>Herold S, Wanzel M, Beuger V, Frohme C, Beul D, et al.  (2002) Negative regulation of the mammalian UV response by Myc through association with Miz-1. Mol Cell  10: 509–521.  <ul class="find" data-citedArticleID="1074778" data-doi="10.1016/s1097-2765(02)00633-0"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00633-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Negative+regulation+of+the+mammalian+UV+response+by+Myc+through+association+with+Miz-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Negative+regulation+of+the+mammalian+UV+response+by+Myc+through+association+with+Miz-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001798-Izumi1" id="pone.0001798-Izumi1"></a>Izumi H, Molander C, Penn LZ, Ishisaki A, Kohno K, et al.  (2001) Mechanism for the transcriptional repression by c-Myc on PDGF beta-receptor. J Cell Sci  114: 1533–1544.  <ul class="find" data-citedArticleID="1074784"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mechanism+for+the+transcriptional+repression+by+c-Myc+on+PDGF+beta-receptor.&amp;auth=&amp;atitle=Mechanism+for+the+transcriptional+repression+by+c-Myc+on+PDGF+beta-receptor." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+for+the+transcriptional+repression+by+c-Myc+on+PDGF+beta-receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+for+the+transcriptional+repression+by+c-Myc+on+PDGF+beta-receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001798-Gartel1" id="pone.0001798-Gartel1"></a>Gartel AL, Ye X, Goufman E, Shianov P, Hay N, et al.  (2001) Myc represses the p21(WAF1/CIP1) promoter and interacts with Sp1/Sp3. Proc Natl Acad Sci USA  98: 4510–4515.  <ul class="find" data-citedArticleID="1074770" data-doi="10.1073/pnas.081074898"><li><a href="http://dx.doi.org/10.1073/pnas.081074898" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Myc+represses+the+p21%28WAF1%2FCIP1%29+promoter+and+interacts+with+Sp1%2FSp3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Myc+represses+the+p21%28WAF1%2FCIP1%29+promoter+and+interacts+with+Sp1%2FSp3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001798-Coller1" id="pone.0001798-Coller1"></a>Coller HA, Grandori C, Tamayo P, Colbert T, Lander ES, et al.  (2000) Expression analysis with oligonucleotide microarrays reveals that MYC regulates genes involved in growth, cell cycle, signaling, and adhesion. Proc Natl Acad Sci USA  97: 3260–3265.  <ul class="find" data-citedArticleID="1074760" data-doi="10.1073/pnas.97.7.3260"><li><a href="http://dx.doi.org/10.1073/pnas.97.7.3260" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+analysis+with+oligonucleotide+microarrays+reveals+that+MYC+regulates+genes+involved+in+growth%2C+cell+cycle%2C+signaling%2C+and+adhesion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+analysis+with+oligonucleotide+microarrays+reveals+that+MYC+regulates+genes+involved+in+growth%2C+cell+cycle%2C+signaling%2C+and+adhesion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001798-Menssen1" id="pone.0001798-Menssen1"></a>Menssen A, Hermeking H (2002) Characterization of the c-MYC-regulated transcriptome by SAGE: identification and analysis of c-MYC target genes. Proc Natl Acad Sci USA  99: 6274–6279.  <ul class="find" data-citedArticleID="1074796" data-doi="10.1073/pnas.082005599"><li><a href="http://dx.doi.org/10.1073/pnas.082005599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+c-MYC-regulated+transcriptome+by+SAGE%3A+identification+and+analysis+of+c-MYC+target+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+c-MYC-regulated+transcriptome+by+SAGE%3A+identification+and+analysis+of+c-MYC+target+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001798-Zeller1" id="pone.0001798-Zeller1"></a>Zeller KI, Jegga AG, Aronow BJ, O'Donnell KA, Dang CV (2003) An integrated database of genes responsive to the Myc oncogenic transcription factor: identification of direct genomic targets. Genome Biol  4: R69.  <ul class="find" data-citedArticleID="1074826" data-doi="10.1186/gb-2003-4-10-r69"><li><a href="http://dx.doi.org/10.1186/gb-2003-4-10-r69" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+integrated+database+of+genes+responsive+to+the+Myc+oncogenic+transcription+factor%3A+identification+of+direct+genomic+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+integrated+database+of+genes+responsive+to+the+Myc+oncogenic+transcription+factor%3A+identification+of+direct+genomic+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001798-Li1" id="pone.0001798-Li1"></a>Li F, Wang Y, Zeller KI, Potter JJ, Wonsey DR, et al.  (2005) Myc stimulates nuclearly encoded mitochondrial genes and mitochondrial biogenesis. Mol Cell Biol  25: 6225–6234.  <ul class="find" data-citedArticleID="1074790" data-doi="10.1128/mcb.25.14.6225-6234.2005"><li><a href="http://dx.doi.org/10.1128/mcb.25.14.6225-6234.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Myc+stimulates+nuclearly+encoded+mitochondrial+genes+and+mitochondrial+biogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Myc+stimulates+nuclearly+encoded+mitochondrial+genes+and+mitochondrial+biogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001798-Zeller2" id="pone.0001798-Zeller2"></a>Zeller KI, Zhao X, Lee CW, Chiu KP, Yao F, et al.  (2006) Global mapping of c-Myc binding sites and target gene networks in human B cells. Proc Natl Acad Sci USA  103: 17834–17839.  <ul class="find" data-citedArticleID="1074828" data-doi="10.1073/pnas.0604129103"><li><a href="http://dx.doi.org/10.1073/pnas.0604129103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+mapping+of+c-Myc+binding+sites+and+target+gene+networks+in+human+B+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+mapping+of+c-Myc+binding+sites+and+target+gene+networks+in+human+B+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001798-Blackwell1" id="pone.0001798-Blackwell1"></a>Blackwell TK, Huang J, Ma A, Kretzner L, Alt FW, et al.  (1993) Binding of myc proteins to canonical and noncanonical DNA sequences. Mol Cell Biol  13: 5216–5224.  <ul class="find" data-citedArticleID="1074750"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Binding+of+myc+proteins+to+canonical+and+noncanonical+DNA+sequences.&amp;auth=&amp;atitle=Binding+of+myc+proteins+to+canonical+and+noncanonical+DNA+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Binding+of+myc+proteins+to+canonical+and+noncanonical+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Binding+of+myc+proteins+to+canonical+and+noncanonical+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001798-Odom1" id="pone.0001798-Odom1"></a>Odom DT, Zizlsperger N, Gordon DB, Bell GW, Rinaldi NJ, et al.  (2004) Control of pancreas and liver gene expression by HNF transcription factors. Science  303: 1378–1381.  <ul class="find" data-citedArticleID="1074806" data-doi="10.1126/science.1089769"><li><a href="http://dx.doi.org/10.1126/science.1089769" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+pancreas+and+liver+gene+expression+by+HNF+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+pancreas+and+liver+gene+expression+by+HNF+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001798-Hu1" id="pone.0001798-Hu1"></a>Hu Z, Killion PJ, Iyer VR (2007) Genetic reconstruction of a functional transcriptional regulatory network. Nat Genet  39: 683–687.  <ul class="find" data-citedArticleID="1074780" data-doi="10.1038/ng2012"><li><a href="http://dx.doi.org/10.1038/ng2012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+reconstruction+of+a+functional+transcriptional+regulatory+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+reconstruction+of+a+functional+transcriptional+regulatory+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001798-Ren1" id="pone.0001798-Ren1"></a>Ren B, Robert F, Wyrick JJ, Aparicio O, Jennings EG, et al.  (2000) Genome-wide location and function of DNA binding proteins. Science  290: 2306–2309.  <ul class="find" data-citedArticleID="1074814" data-doi="10.1126/science.290.5500.2306"><li><a href="http://dx.doi.org/10.1126/science.290.5500.2306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+location+and+function+of+DNA+binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+location+and+function+of+DNA+binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001798-Wick1" id="pone.0001798-Wick1"></a>Wick M, Burger C, Brusselbach S, Lucibello FC, Muller R (1994) Identification of serum-inducible genes: different patterns of gene regulation during G0–&gt;S and G1–&gt;S progression. J Cell Sci  107: 227–239.  <ul class="find" data-citedArticleID="1074822"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+serum-inducible+genes%3A+different+patterns+of+gene+regulation+during+G0%3F%26gt%3BS+and+G1%3F%26gt%3BS+progression.&amp;auth=&amp;atitle=Identification+of+serum-inducible+genes%3A+different+patterns+of+gene+regulation+during+G0%3F%26gt%3BS+and+G1%3F%26gt%3BS+progression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+serum-inducible+genes%3A+different+patterns+of+gene+regulation+during+G0%E2%80%93%26gt%3BS+and+G1%E2%80%93%26gt%3BS+progression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+serum-inducible+genes%3A+different+patterns+of+gene+regulation+during+G0%E2%80%93%26gt%3BS+and+G1%E2%80%93%26gt%3BS+progression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001798-Winkles1" id="pone.0001798-Winkles1"></a>Winkles JA (1998) Serum- and polypeptide growth factor-inducible gene expression in mouse fibroblasts. Prog Nucleic Acid Res Mol Biol  58: 41–78.  <ul class="find" data-citedArticleID="1074824" data-doi="10.1016/s0079-6603(08)60033-1"><li><a href="http://dx.doi.org/10.1016/s0079-6603(08)60033-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Serum-+and+polypeptide+growth+factor-inducible+gene+expression+in+mouse+fibroblasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Serum-+and+polypeptide+growth+factor-inducible+gene+expression+in+mouse+fibroblasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001798-Iyer1" id="pone.0001798-Iyer1"></a>Iyer VR, Eisen MB, Ross DT, Schuler G, Moore T, et al.  (1999) The transcriptional program in the response of human fibroblasts to serum. Science  283: 83–87.  <ul class="find" data-citedArticleID="1074782" data-doi="10.1126/science.283.5398.83"><li><a href="http://dx.doi.org/10.1126/science.283.5398.83" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptional+program+in+the+response+of+human+fibroblasts+to+serum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptional+program+in+the+response+of+human+fibroblasts+to+serum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001798-Oster1" id="pone.0001798-Oster1"></a>Oster SK, Ho CS, Soucie EL, Penn LZ (2002) The myc oncogene: MarvelouslY Complex. Adv Cancer Res  84: 81–154.  <ul class="find" data-citedArticleID="1074808" data-doi="10.1016/s0065-230x(02)84004-0"><li><a href="http://dx.doi.org/10.1016/s0065-230x(02)84004-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+myc+oncogene%3A+MarvelouslY+Complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+myc+oncogene%3A+MarvelouslY+Complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001798-Gu1" id="pone.0001798-Gu1"></a>Gu J, Iyer VR (2006) PI3K signaling and miRNA expression during the response of quiescent human fibroblasts to distinct proliferative stimuli. Genome Biol  7: R42.  <ul class="find" data-citedArticleID="1074776"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=PI3K+signaling+and+miRNA+expression+during+the+response+of+quiescent+human+fibroblasts+to+distinct+proliferative+stimuli.&amp;auth=&amp;atitle=PI3K+signaling+and+miRNA+expression+during+the+response+of+quiescent+human+fibroblasts+to+distinct+proliferative+stimuli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PI3K+signaling+and+miRNA+expression+during+the+response+of+quiescent+human+fibroblasts+to+distinct+proliferative+stimuli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PI3K+signaling+and+miRNA+expression+during+the+response+of+quiescent+human+fibroblasts+to+distinct+proliferative+stimuli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001798-Fernandez1" id="pone.0001798-Fernandez1"></a>Fernandez PC, Frank SR, Wang L, Schroeder M, Liu S, et al.  (2003) Genomic targets of the human c-Myc protein. Genes Dev  17: 1115–1129.  <ul class="find" data-citedArticleID="1074768" data-doi="10.1101/gad.1067003"><li><a href="http://dx.doi.org/10.1101/gad.1067003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+targets+of+the+human+c-Myc+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+targets+of+the+human+c-Myc+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001798-Li2" id="pone.0001798-Li2"></a>Li Z, Van Calcar S, Qu C, Cavenee WK, Zhang MQ, et al.  (2003) A global transcriptional regulatory role for c-Myc in Burkitt's lymphoma cells. Proc Natl Acad Sci USA  100: 8164–8169.  <ul class="find" data-citedArticleID="1074792" data-doi="10.1073/pnas.1332764100"><li><a href="http://dx.doi.org/10.1073/pnas.1332764100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+global+transcriptional+regulatory+role+for+c-Myc+in+Burkitt%27s+lymphoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+global+transcriptional+regulatory+role+for+c-Myc+in+Burkitt%27s+lymphoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001798-Bolton1" id="pone.0001798-Bolton1"></a>Bolton EC, Boeke JD (2003) Transcriptional interactions between yeast tRNA genes, flanking genes and Ty elements: a genomic point of view. Genome Res  13: 254–263.  <ul class="find" data-citedArticleID="1074756" data-doi="10.1101/gr.612203"><li><a href="http://dx.doi.org/10.1101/gr.612203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+interactions+between+yeast+tRNA+genes%2C+flanking+genes+and+Ty+elements%3A+a+genomic+point+of+view." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+interactions+between+yeast+tRNA+genes%2C+flanking+genes+and+Ty+elements%3A+a+genomic+point+of+view.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001798-Dennis1" id="pone.0001798-Dennis1"></a>Dennis G Jr, Sherman BT, Hosack DA, Yang J, Gao W, et al.  (2003) DAVID: Database for Annotation, Visualization, and Integrated Discovery. Genome Biol  4: P3.  <ul class="find" data-citedArticleID="1074764" data-doi="10.1186/gb-2003-4-5-p3"><li><a href="http://dx.doi.org/10.1186/gb-2003-4-5-p3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DAVID%3A+Database+for+Annotation%2C+Visualization%2C+and+Integrated+Discovery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DAVID%3A+Database+for+Annotation%2C+Visualization%2C+and+Integrated+Discovery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001798-AlShahrour1" id="pone.0001798-AlShahrour1"></a>Al-Shahrour F, Diaz-Uriarte R, Dopazo J (2004) FatiGO: a web tool for finding significant associations of Gene Ontology terms with groups of genes. Bioinformatics  20: 578–580.  <ul class="find" data-citedArticleID="1074744" data-doi="10.1093/bioinformatics/btg455"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg455" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FatiGO%3A+a+web+tool+for+finding+significant+associations+of+Gene+Ontology+terms+with+groups+of+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FatiGO%3A+a+web+tool+for+finding+significant+associations+of+Gene+Ontology+terms+with+groups+of+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001798-Zhang1" id="pone.0001798-Zhang1"></a>Zhang H, Gao P, Fukuda R, Kumar G, Krishnamachary B, et al.  (2007) HIF-1 inhibits mitochondrial biogenesis and cellular respiration in VHL-deficient renal cell carcinoma by repression of C-MYC activity. Cancer Cell  11: 407–420.  <ul class="find" data-citedArticleID="1074830" data-doi="10.1016/j.ccr.2007.04.001"><li><a href="http://dx.doi.org/10.1016/j.ccr.2007.04.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIF-1+inhibits+mitochondrial+biogenesis+and+cellular+respiration+in+VHL-deficient+renal+cell+carcinoma+by+repression+of+C-MYC+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIF-1+inhibits+mitochondrial+biogenesis+and+cellular+respiration+in+VHL-deficient+renal+cell+carcinoma+by+repression+of+C-MYC+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001798-Morrish1" id="pone.0001798-Morrish1"></a>Morrish F, Hockenbery D (2003) Myc's mastery of mitochondrial mischief. Cell Cycle  2: 11–13.  <ul class="find" data-citedArticleID="1074798" data-doi="10.4161/cc.2.1.275"><li><a href="http://dx.doi.org/10.4161/cc.2.1.275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Myc%27s+mastery+of+mitochondrial+mischief." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Myc%27s+mastery+of+mitochondrial+mischief.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001798-Morrish2" id="pone.0001798-Morrish2"></a>Morrish F, Giedt C, Hockenbery D (2003) c-MYC apoptotic function is mediated by NRF-1 target genes. Genes Dev  17: 240–255.  <ul class="find" data-citedArticleID="1074800" data-doi="10.1101/gad.1032503"><li><a href="http://dx.doi.org/10.1101/gad.1032503" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=c-MYC+apoptotic+function+is+mediated+by+NRF-1+target+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22c-MYC+apoptotic+function+is+mediated+by+NRF-1+target+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001798-Virbasius1" id="pone.0001798-Virbasius1"></a>Virbasius JV, Scarpulla RC (1994) Activation of the human mitochondrial transcription factor A gene by nuclear respiratory factors: a potential regulatory link between nuclear and mitochondrial gene expression in organelle biogenesis. Proc Natl Acad Sci USA  91: 1309–1313.  <ul class="find" data-citedArticleID="1074820" data-doi="10.1073/pnas.91.4.1309"><li><a href="http://dx.doi.org/10.1073/pnas.91.4.1309" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+of+the+human+mitochondrial+transcription+factor+A+gene+by+nuclear+respiratory+factors%3A+a+potential+regulatory+link+between+nuclear+and+mitochondrial+gene+expression+in+organelle+biogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+of+the+human+mitochondrial+transcription+factor+A+gene+by+nuclear+respiratory+factors%3A+a+potential+regulatory+link+between+nuclear+and+mitochondrial+gene+expression+in+organelle+biogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001798-Andreoli1" id="pone.0001798-Andreoli1"></a>Andreoli C, Prokisch H, Hortnagel K, Mueller JC, Munsterkotter M, et al.  (2004) MitoP2, an integrated database on mitochondrial proteins in yeast and man. Nucleic Acids Res  32: D459–462.  <ul class="find" data-citedArticleID="1074746" data-doi="10.1093/nar/gkh137"><li><a href="http://dx.doi.org/10.1093/nar/gkh137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MitoP2%2C+an+integrated+database+on+mitochondrial+proteins+in+yeast+and+man." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MitoP2%2C+an+integrated+database+on+mitochondrial+proteins+in+yeast+and+man.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001798-Patel1" id="pone.0001798-Patel1"></a>Patel JH, Loboda AP, Showe MK, Showe LC, McMahon SB (2004) Analysis of genomic targets reveals complex functions of MYC. Nat Rev Cancer  4: 562–568.  <ul class="find" data-citedArticleID="1074810" data-doi="10.1038/nrc1393"><li><a href="http://dx.doi.org/10.1038/nrc1393" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+genomic+targets+reveals+complex+functions+of+MYC." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+genomic+targets+reveals+complex+functions+of+MYC.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001798-James1" id="pone.0001798-James1"></a>James L, Eisenman RN (2002) Myc and Mad bHLHZ domains possess identical DNA-binding specificities but only partially overlapping functions in vivo. Proc Natl Acad Sci USA  99: 10429–10434.  <ul class="find" data-citedArticleID="1074786" data-doi="10.1073/pnas.162369299"><li><a href="http://dx.doi.org/10.1073/pnas.162369299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Myc+and+Mad+bHLHZ+domains+possess+identical+DNA-binding+specificities+but+only+partially+overlapping+functions+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Myc+and+Mad+bHLHZ+domains+possess+identical+DNA-binding+specificities+but+only+partially+overlapping+functions+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001798-Kim1" id="pone.0001798-Kim1"></a>Kim J, Bhinge AA, Morgan XC, Iyer VR (2005) Mapping DNA-protein interactions in large genomes by sequence tag analysis of genomic enrichment. Nat Methods  2: 47–53.  <ul class="find" data-citedArticleID="1074788" data-doi="10.1038/nmeth726"><li><a href="http://dx.doi.org/10.1038/nmeth726" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+DNA-protein+interactions+in+large+genomes+by+sequence+tag+analysis+of+genomic+enrichment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+DNA-protein+interactions+in+large+genomes+by+sequence+tag+analysis+of+genomic+enrichment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001798-Roberts1" id="pone.0001798-Roberts1"></a>Roberts CJ, Nelson B, Marton MJ, Stoughton R, Meyer MR, et al.  (2000) Signaling and circuitry of multiple MAPK pathways revealed by a matrix of global gene expression profiles. Science  287: 873–880.  <ul class="find" data-citedArticleID="1074816" data-doi="10.1126/science.287.5454.873"><li><a href="http://dx.doi.org/10.1126/science.287.5454.873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signaling+and+circuitry+of+multiple+MAPK+pathways+revealed+by+a+matrix+of+global+gene+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signaling+and+circuitry+of+multiple+MAPK+pathways+revealed+by+a+matrix+of+global+gene+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.XML" value="97811"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.PDF" value="560784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g001.PNG_L" value="946306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g001.PNG_M" value="118084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g001.PNG_S" value="10922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g001.TIF" value="1731438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g001.PNG_I" value="42667"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g002.PNG_L" value="493340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g002.PNG_M" value="53019"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g002.PNG_S" value="9034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g002.TIF" value="1009034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g002.PNG_I" value="18441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.t001.PNG_L" value="181846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.t001.PNG_M" value="100116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.t001.PNG_S" value="11246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.t001.TIF" value="836276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.t001.PNG_I" value="31088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g003.PNG_L" value="539290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g003.PNG_M" value="91653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g003.PNG_S" value="13062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g003.TIF" value="2027898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g003.PNG_I" value="55105"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g004.PNG_L" value="1332784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g004.PNG_M" value="135088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g004.PNG_S" value="14935"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g004.TIF" value="2990688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g004.PNG_I" value="109274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g005.PNG_L" value="315021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g005.PNG_M" value="31612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g005.PNG_S" value="8469"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g005.TIF" value="506118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g005.PNG_I" value="36086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g006.PNG_L" value="247334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g006.PNG_M" value="23110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g006.PNG_S" value="5779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g006.TIF" value="515144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.g006.PNG_I" value="8503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.s001.TIF" value="379483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.s002.TIF" value="598290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.s003.XLS" value="186880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.s004.XLS" value="374272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001798.s005.XLS" value="33792"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001798&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001798" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001798&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001798&volume=&issue=&title=Global Identification of Myc Target Genes Reveals Its Direct Role in Mitochondrial Biogenesis and Its E-Box Usage In Vivo&author_name=Jonghwan%20Kim%2C%20Ji-hoon%20Lee%2C%20Vishwanath%20R.%20Iyer&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001798" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001798" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001798" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001798&amp;t=Global%20Identification%20of%20Myc%20Target%20Genes%20Reveals%20Its%20Direct%20Role%20in%20Mitochondrial%20Biogenesis%20and%20Its%20E-Box%20Usage%20In%20Vivo" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001798&title=Global%20Identification%20of%20Myc%20Target%20Genes%20Reveals%20Its%20Direct%20Role%20in%20Mitochondrial%20Biogenesis%20and%20Its%20E-Box%20Usage%20In%20Vivo&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001798&amp;title=Global%20Identification%20of%20Myc%20Target%20Genes%20Reveals%20Its%20Direct%20Role%20in%20Mitochondrial%20Biogenesis%20and%20Its%20E-Box%20Usage%20In%20Vivo" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001798" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001798&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Global Identification of Myc Target Genes Reveals Its Direct Role in Mitochondrial Biogenesis and Its E-Box Usage In Vivo';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001798';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Global%20Identification%20of%20Myc%20Target%20Genes%20Reveals%20Its%20Direct%20Role%20in%20Mitochondrial%20Biogenesis%20and%20Its%20E-Box%20Usage%20In%20Vivo http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001798" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001798" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biosynthesis%22" title="Search for articles in the subject area:'Biosynthesis'"><div class="flagText">Biosynthesis</div></a>
              <div data-categoryid="40369" data-articleid="26420"
                   data-categoryname="Biosynthesis"
                   class="flagImage" title="Flag 'Biosynthesis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin+immunoprecipitation%22" title="Search for articles in the subject area:'Chromatin immunoprecipitation'"><div class="flagText">Chromatin immunoprecipitation</div></a>
              <div data-categoryid="40167" data-articleid="26420"
                   data-categoryname="Chromatin immunoprecipitation"
                   class="flagImage" title="Flag 'Chromatin immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fibroblasts%22" title="Search for articles in the subject area:'Fibroblasts'"><div class="flagText">Fibroblasts</div></a>
              <div data-categoryid="47975" data-articleid="26420"
                   data-categoryname="Fibroblasts"
                   class="flagImage" title="Flag 'Fibroblasts' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="17701" data-articleid="26420"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HeLa+cells%22" title="Search for articles in the subject area:'HeLa cells'"><div class="flagText">HeLa cells</div></a>
              <div data-categoryid="19567" data-articleid="26420"
                   data-categoryname="HeLa cells"
                   class="flagImage" title="Flag 'HeLa cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="26420"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="26420"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26420"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3671'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9145'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5654&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
