

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Comparative Bacterial Proteomics: Analysis of the Core Genome Concept</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001542"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001542" />

    <meta name="citation_title" content="Comparative Bacterial Proteomics: Analysis of the Core Genome Concept"/>
    <meta itemprop="name" content="Comparative Bacterial Proteomics: Analysis of the Core Genome Concept"/>

      <meta name="citation_author" content="Stephen J. Callister"/>
            <meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America"/>
      <meta name="citation_author" content="Lee Ann McCue"/>
            <meta name="citation_author_institution" content="Computational Sciences and Mathematics Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America"/>
      <meta name="citation_author" content="Joshua E. Turse"/>
            <meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America"/>
      <meta name="citation_author" content="Matthew E. Monroe"/>
            <meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America"/>
      <meta name="citation_author" content="Kenneth J. Auberry"/>
            <meta name="citation_author_institution" content="Environmental Molecular Sciences Laboratory, Pacific Northwest National Laboratory, Richland, Washington, United States of America"/>
      <meta name="citation_author" content="Richard D. Smith"/>
            <meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America"/>
      <meta name="citation_author" content="Joshua N. Adkins"/>
            <meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America"/>
      <meta name="citation_author" content="Mary S. Lipton"/>
            <meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America"/>

    <meta name="citation_date" content="2008/2/6"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001542.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1542"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The cyanobacterial genome core and the origin of photosynthesis.; citation_author=AY Mulkidjanian; citation_author=EV Koonin; citation_author=KS Makarova; citation_author=SL Mekhedov; citation_author=A Sorokin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=1; citation_pages=13126-13131; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparative genome analysis of four magnetotactic bacteria reveals a complex set of group-specific genes implicated in magnetosome biomineralization and function.; citation_author=M Richter; citation_author=M Kube; citation_author=DA Bazylinski; citation_author=T Lombardot; citation_author=FO Glockner; citation_journal_title=J Bacteriol; citation_volume=189; citation_number=2; citation_pages=4899-4910; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Degenerative minimalism in the genome of a psyllid endosymbiont.; citation_author=MA Clark; citation_author=L Baumann; citation_author=MLL Thao; citation_author=NA Moran; citation_author=P Baumann; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=3; citation_pages=1853-1861; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The genetic core of the universal ancestor.; citation_author=JK Harris; citation_author=ST Kelley; citation_author=GB Spiegelman; citation_author=NR Pace; citation_journal_title=Genome Res; citation_volume=13; citation_number=4; citation_pages=407-412; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolution of the core and pan-genome of Strepococcus: positive selection, recombination, and genome composition.; citation_author=T Lefebure; citation_author=MJ Stanhope; citation_journal_title=Genome Biol; citation_volume=8; citation_number=5; citation_pages=Art. No. R71 2007; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A minimal gene set for cellular life derived by comparison of complete bacterial genomes.; citation_author=AR Mushegian; citation_author=EV Koonin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=6; citation_pages=10268-10273; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Evolution of the core genome of Pseudomonas syringae, a highly clonal, endemic plant pathogen.; citation_author=SF Sarkar; citation_author=DS Guttman; citation_journal_title=Appl Environ Microbiol; citation_volume=70; citation_number=7; citation_pages=1999-2012; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The silencing of pseudogenes.; citation_author=A Mira; citation_author=R Pushker; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=8; citation_pages=2135-2138; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Advances in proteomics data analysis and display using an accurate mass and time tag approach.; citation_author=JSD Zimmer; citation_author=ME Monroe; citation_author=WJ Qian; citation_author=RD Smith; citation_journal_title=Mass Spectrom Rev; citation_volume=25; citation_number=9; citation_pages=450-482; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Numerical-Analysis of sodium dodecyl sulfate-polyacrylamide gel-electrophoretic protein-patterns for the classification, identification and typing of medically important bacteria.; citation_author=M Costas; citation_journal_title=Electrophoresis; citation_volume=11; citation_number=10; citation_pages=382-391; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Differentiation between Xanthomonas-campestris pv oryzae, Xanthomonas-campestris pv oryzicola and the bacterial brown blotch pathogen on rice by numerical-analysis of phenotypic features and protein gel electrophoregrams.; citation_author=CMV Cruz; citation_author=F Gossele; citation_author=K Kersters; citation_author=P Segers; citation_author=M Vandenmooter; citation_journal_title=J Gen Microbiol; citation_volume=130; citation_number=11; citation_pages=2983-2999; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Universal protein families and the functional content of the last universal common ancestor.; citation_author=N Kyrpides; citation_author=R Overbeek; citation_author=C Ouzounis; citation_journal_title=J Mol Evol; citation_volume=49; citation_number=12; citation_pages=413-423; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Automatic clustering of orthologs and in-paralogs from pairwise species comparisons.; citation_author=M Remm; citation_author=CEV Storm; citation_author=ELL Sonnhammer; citation_journal_title=J Mol Biol; citation_volume=314; citation_number=13; citation_pages=1041-1052; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=JH Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Research; citation_volume=25; citation_number=14; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=An approach to correlate tandem mass-spectral data of peptides with amino-acid-sequences in a protein database.; citation_author=JK Eng; citation_author=AL McCormack; citation_author=JR Yates; citation_journal_title=J Am Soc Mass Spectrom; citation_volume=5; citation_number=15; citation_pages=976-989; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Distinguishing homologous from analogous proteins.; citation_author=WM Fitch; citation_journal_title=Systematic Zoology; citation_volume=19; citation_number=16; citation_pages=99-&; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Comparison of aerobic and photosynthetic Rhodobacter sphaeroides 2.4.1 proteomes.; citation_author=SJ Callister; citation_author=CD Nicora; citation_author=XH Zeng; citation_author=JH Roh; citation_author=MA Dominguez; citation_journal_title=J Microbiol Methods; citation_volume=67; citation_number=17; citation_pages=424-436; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A dedicated translation factor controls the synthesis of the global regulator Fis.; citation_author=RM Owens; citation_author=G Pritchard; citation_author=P Skipp; citation_author=M Hodey; citation_author=SR Connell; citation_journal_title=Embo J; citation_volume=23; citation_number=18; citation_pages=3375-3385; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=‘Conserved hypothetical’ proteins: prioritization of targets for experimental study.; citation_author=MY Galperin; citation_author=EV Koonin; citation_journal_title=Nucleic Acids Research; citation_volume=32; citation_number=19; citation_pages=5452-5463; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mapping of promoters for the nucleus encoded plastid RNA polymerase (NEP) in the iojap maize mutant.; citation_author=D Silhavy; citation_author=P Maliga; citation_journal_title=Curr Genet; citation_volume=33; citation_number=20; citation_pages=340-344; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Identification of novel Escherichia coli ribosome-associated proteins using isobaric tags and multidimensional protein identification techniques.; citation_author=M Jiang; citation_author=SM Sullivan; citation_author=AK Walker; citation_author=JR Strahler; citation_author=PC Andrews; citation_journal_title=J Bacteriol; citation_volume=189; citation_number=21; citation_pages=3434-3444; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Screening for synthetic lethal mutants in Escherichia coli and identification of EnvC (YibP) as a periplasmic septal ring factor with murein hydrolase activity.; citation_author=TG Bernhardt; citation_author=PAJ de Boer; citation_journal_title=Mol Microbiol; citation_volume=52; citation_number=22; citation_pages=1255-1269; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=SCRATCH: a protein structure and structural feature prediction server.; citation_author=J Cheng; citation_author=AZ Randall; citation_author=MJ Sweredoski; citation_author=P Baldi; citation_journal_title=Nucleic Acids Research; citation_volume=33; citation_number=23; citation_pages=W72-W76; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MEGA4: Molecular evolutionary genetics analysis (MEGA) software version 4.0.; citation_author=K Tamura; citation_author=J Dudley; citation_author=M Nei; citation_author=S Kumar; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=24; citation_pages=1596-1599; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome of Geobacter sulfurreducens: Metal reduction in subsurface environments.; citation_author=BA Methe; citation_author=KE Nelson; citation_author=JA Eisen; citation_author=IT Paulsen; citation_author=W Nelson; citation_journal_title=Science; citation_volume=302; citation_number=25; citation_pages=1967-1969; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Geobacter-Metallireducens gen-nov sp-vov, a microorganism capable of coupling the complete oxidation of organic-compounds to the reduction of iron and other metals.; citation_author=DR Lovley; citation_author=SJ Giovannoni; citation_author=DC White; citation_author=JE Champine; citation_author=EJP Phillips; citation_journal_title=Arch Microbiol; citation_volume=159; citation_number=26; citation_pages=336-344; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=ZipA is a MAP-Tau homolog and is essential for structural integrity of the cytokinetic FtsZ ring during bacterial cell division.; citation_author=D RayChaudhuri; citation_journal_title=Embo J; citation_volume=18; citation_number=27; citation_pages=2372-2383; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic relationships of Salmonella typhi and Salmonella typhimurium based on 16S rRNA sequence analysis.; citation_author=HR Chang; citation_author=LH Loo; citation_author=K Jeyaseelan; citation_author=L Earnest; citation_author=E Stackebrandt; citation_journal_title=Int J Syst Bacteriol; citation_volume=47; citation_number=28; citation_pages=1253-1254; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genomic comparisons of Salmonella enterica serovar Dublin, Agona, and typhimurium strains recently isolated from milk filters and bovine samples from ireland, using a Salmonella microarray.; citation_author=FJ Reen; citation_author=EF Boyd; citation_author=S Porwollik; citation_author=BP Murphy; citation_author=D Gilroy; citation_journal_title=Appl Environ Microbiol; citation_volume=71; citation_number=29; citation_pages=1616-1625; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolution of minimal-gene-sets in host-dependent bacteria.; citation_author=L Klasson; citation_author=SGE Andersson; citation_journal_title=Trends Microbiol; citation_volume=12; citation_number=30; citation_pages=37-43; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=How essential are nonessential genes?; citation_author=G Fang; citation_author=E Rocha; citation_author=A Danchin; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=31; citation_pages=2147-2156; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Deletional bias and the evolution of bacterial genomes.; citation_author=A Mira; citation_author=H Ochman; citation_author=NA Moran; citation_journal_title=Trends Genet; citation_volume=17; citation_number=32; citation_pages=589-596; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Global transposon mutagenesis and a minimal mycoplasma genome.; citation_author=CA Hutchison; citation_author=SN Peterson; citation_author=SR Gill; citation_author=RT Cline; citation_author=O White; citation_journal_title=Science; citation_volume=286; citation_number=33; citation_pages=2165-2169; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Comparative genomics, minimal gene-sets and the last universal common ancestor.; citation_author=EV Koonin; citation_journal_title=Nat Rev Microbiol; citation_volume=1; citation_number=34; citation_pages=127-136; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Essential genes are more evolutionarily conserved than are nonessential genes in bacteria.; citation_author=IK Jordan; citation_author=IB Rogozin; citation_author=YI Wolf; citation_author=EV Koonin; citation_journal_title=Genome Res; citation_volume=12; citation_number=35; citation_pages=962-968; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome streamlining in a cosmopolitan oceanic bacterium.; citation_author=SJ Giovannoni; citation_author=HJ Tripp; citation_author=S Givan; citation_author=M Podar; citation_author=KL Vergin; citation_journal_title=Science; citation_volume=309; citation_number=36; citation_pages=1242-1245; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Essential genes of a minimal bacterium.; citation_author=JI Glass; citation_author=N Assad-Garcia; citation_author=N Alperovich; citation_author=S Yooseph; citation_author=MR Lewis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=37; citation_pages=425-430; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Experimental determination and system level analysis of essential genes in Escherichia coli MG1655.; citation_author=SY Gerdes; citation_author=MD Scholle; citation_author=JW Campbell; citation_author=G Balazsi; citation_author=E Ravasz; citation_journal_title=J Bacteriol; citation_volume=185; citation_number=38; citation_pages=5673-5684; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Large-scale identification of essential Salmonella genes by trapping lethal insertions.; citation_author=K Knuth; citation_author=H Niesalla; citation_author=CJ Hueck; citation_author=TM Fuchs; citation_journal_title=Mol Microbiol; citation_volume=51; citation_number=39; citation_pages=1729-1744; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Developmental Plasticity and Evolution.; citation_author=MJ West-Eberhard; citation_number=40; citation_date=2003; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Whole genome transplantation becomes a reality.; citation_author=M Stone; citation_journal_title=Microbe; citation_volume=2; citation_number=41; citation_pages=474-475; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Analysis of the Salmonella typhimurium proteome through environmental response toward infectious conditions.; citation_author=JN Adkins; citation_author=HM Mottaz; citation_author=AD Norbeck; citation_author=JK Gustin; citation_author=J Rue; citation_journal_title=Mol Cell Proteomics; citation_volume=5; citation_number=42; citation_pages=1450-1461; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The proteome of dissimilatory metal-reducing microorganism Geobacter sulfurreducens under various growth conditions.; citation_author=YHR Ding; citation_author=KK Hixson; citation_author=CS Giometti; citation_author=A Stanley; citation_author=A Esteve-Nunez; citation_journal_title=BBA-Proteins Proteomics; citation_volume=1764; citation_number=43; citation_pages=1198-1206; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Confirmation of the expression of a large set of conserved hypothetical proteins in Shewanella oneidensis MR-1.; citation_author=DA Elias; citation_author=ME Monroe; citation_author=RD Smith; citation_author=JK Fredrickson; citation_author=MS Lipton; citation_journal_title=J Microbiol Methods; citation_volume=66; citation_number=44; citation_pages=223-233; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Enrichment of functional redox reactive proteins and identification by mass spectrometry results in several terminal Fe(III)-reducing candidate proteins in Shewanella oneidensis MR-1.; citation_author=DA Elias; citation_author=F Yang; citation_author=HM Mottaz; citation_author=AS Beliaev; citation_author=MS Lipton; citation_journal_title=J Microbiol Methods; citation_volume=68; citation_number=45; citation_pages=367-375; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Differential label-free quantitative proteomic analysis of Shewanella oneidensis cultured under aerobic and suboxic conditions by accurate mass and time tag approach.; citation_author=RH Fang; citation_author=DA Elias; citation_author=ME Monroe; citation_author=YF Shen; citation_author=M McIntosh; citation_journal_title=Mol Cell Proteomics; citation_volume=5; citation_number=46; citation_pages=714-725; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Proteome analysis of Desulfovibrio desulfuricans G20 mutants using the accurate mass and time (AMT) tag Approach.; citation_author=Q Luo; citation_author=KK Hixson; citation_author=SJ Callister; citation_author=MS Lipton; citation_author=BE Morris; citation_journal_title=J Proteome Res; citation_volume=6; citation_number=47; citation_pages=3042-3053; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Targeted protein degradation by Salmonella under phagosome-mimicking culture conditions investigated using comparative peptidomics.; citation_author=NP Manes; citation_author=JK Gustin; citation_author=J Rue; citation_author=HM Mottaz; citation_author=SO Purvine; citation_journal_title=Mol Cell Proteomics; citation_volume=6; citation_number=48; citation_pages=717-727; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Validation of Shewanella oneidensis MR-1 small proteins by AMT tag-based proteome analysis.; citation_author=MF Romine; citation_author=DA Elias; citation_author=ME Monroe; citation_author=K Auberry; citation_author=RH Fang; citation_journal_title=Omics; citation_volume=8; citation_number=49; citation_pages=239-254; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Global whole-cell FTICR mass spectrometric proteomics analysis of the heat shock response in the radioresistant bacterium Deinococcus radiodurans.; citation_author=AK Schmid; citation_author=MS Lipton; citation_author=H Mottaz; citation_author=ME Monroe; citation_author=RD Smith; citation_journal_title=J Proteome Res; citation_volume=4; citation_number=50; citation_pages=709-718; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Proteomic analysis of Salmonella enterica serovar Typhimurium isolated from RAW 264.7 macrophages-Identification of a novel protein that contributes to the replication of serovar Typhimurium inside macrophages.; citation_author=L Shi; citation_author=JN Adkins; citation_author=JR Coleman; citation_author=AA Schepmoes; citation_author=A Dohnkova; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=51; citation_pages=29131-29140; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An accurate mass tag strategy for quantitative and high-throughput proteome measurements.; citation_author=RD Smith; citation_author=GA Anderson; citation_author=MS Lipton; citation_author=L Pasa-Tolic; citation_author=YF Shen; citation_journal_title=Proteomics; citation_volume=2; citation_number=52; citation_pages=513-523; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Characterization of purified c-type heme-containing peptides and identification of c-type heme-attachment sites in Shewanella oneidenis cytochromes using mass spectrometry.; citation_author=F Yang; citation_author=B Bogdanov; citation_author=EF Strittmatter; citation_author=AN Vilkov; citation_author=M Gritsenko; citation_journal_title=J Proteome Res; citation_volume=4; citation_number=53; citation_pages=846-854; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=LC-MS/MS based proteomic analysis and functional inference of hypothetical proteins in Desulfovibrio vulgaris.; citation_author=WW Zhang; citation_author=DE Culley; citation_author=MA Gritsenko; citation_author=RJ Moore; citation_author=L Nie; citation_journal_title=Biochem Biophys Res Commun; citation_volume=349; citation_number=54; citation_pages=1412-1419; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A proteomic view of Desulfovibrio vulgaris metabolism as determined by liquid chromatography coupled with tandem mass spectrometry.; citation_author=WW Zhang; citation_author=MA Gritsenko; citation_author=RJ Moore; citation_author=DE Culley; citation_author=L Nie; citation_journal_title=Proteomics; citation_volume=6; citation_number=55; citation_pages=4286-4299; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Proteomics analysis of the causative agents of typhoid fever.; citation_author=C Ansong; citation_author=H Yoon; citation_author=AD Norbeck; citation_author=J Gustin; citation_author=JE McDermott; citation_journal_title=J Proteome Res: In Press; citation_number=56; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=PRISM: A data management system for high-throughput proteomics.; citation_author=GR Kiebel; citation_author=KJ Auberry; citation_author=N Jaitly; citation_author=DA Clark; citation_author=ME Monroe; citation_journal_title=Proteomics; citation_volume=6; citation_number=57; citation_pages=1783-1790; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=VIPER: an advanced software package to support high-throughput LC-MS peptide identification.; citation_author=ME Monroe; citation_author=N Tolic; citation_author=N Jaitly; citation_author=JL Shaw; citation_author=JN Adkins; citation_journal_title=Bioinformatics; citation_volume=23; citation_number=58; citation_pages=2021-2023; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search.; citation_author=A Keller; citation_author=AI Nesvizhskii; citation_author=E Kolker; citation_author=R Aebersold; citation_journal_title=Anal Chem; citation_volume=74; citation_number=59; citation_pages=5383-5392; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Probability-based evaluation of peptide and protein identifications from tandem mass spectrometry and SEQUEST analysis: The human proteome.; citation_author=WJ Qian; citation_author=T Liu; citation_author=ME Monroe; citation_author=EF Strittmatter; citation_author=JM Jacobs; citation_journal_title=J Proteome Res; citation_volume=4; citation_number=60; citation_pages=53-62; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001542" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Comparative Bacterial Proteomics: Analysis of the Core Genome Concept"/>
    <meta name="twitter:description" content="While comparative bacterial genomic studies commonly predict a set of genes indicative of common ancestry, experimental validation of the existence of this core genome requires extensive measurement and is typically not undertaken. Enabled by an extensive proteome database developed over six years, we have experimentally verified the expression of proteins predicted from genomic ortholog comparisons among 17 environmental and pathogenic bacteria. More exclusive relationships were observed among the expressed protein content of phenotypically related bacteria, which is indicative of the specific lifestyles associated with these organisms. Although genomic studies can establish relative orthologous relationships among a set of bacteria and propose a set of ancestral genes, our proteomics study establishes expressed lifestyle differences among conserved genes and proposes a set of expressed ancestral traits."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001542.g005"/>

  <meta property="og:title" content="Comparative Bacterial Proteomics: Analysis of the Core Genome Concept" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001542" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7813'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1429'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=400&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001542">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001542" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001542&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Comparative Bacterial Proteomics: Analysis of the Core Genome Concept
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen J. Callister, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lee Ann McCue, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational Sciences and Mathematics Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joshua E. Turse, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew E. Monroe, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kenneth J. Auberry, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Environmental Molecular Sciences Laboratory, Pacific Northwest National Laboratory, Richland, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Richard D. Smith, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joshua N. Adkins
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Joshua.Adkins@pnl.gov">Joshua.Adkins@pnl.gov</a> (JA); <a href="mailto:mary.lipton@pnl.gov">mary.lipton@pnl.gov</a> (ML)</p>

                <p>Affiliation:
                  Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mary S. Lipton
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Joshua.Adkins@pnl.gov">Joshua.Adkins@pnl.gov</a> (JA); <a href="mailto:mary.lipton@pnl.gov">mary.lipton@pnl.gov</a> (ML)</p>

                <p>Affiliation:
                  Biological Sciences Division, Pacific Northwest National Laboratory, Richland, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 06, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001542</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001542" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001542" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001542" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001542" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001542-g001" data-doi="info:doi/10.1371/journal.pone.0001542" data-uri="info:doi/10.1371/journal.pone.0001542.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001542-g002" data-doi="info:doi/10.1371/journal.pone.0001542" data-uri="info:doi/10.1371/journal.pone.0001542.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001542-g003" data-doi="info:doi/10.1371/journal.pone.0001542" data-uri="info:doi/10.1371/journal.pone.0001542.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001542-g004" data-doi="info:doi/10.1371/journal.pone.0001542" data-uri="info:doi/10.1371/journal.pone.0001542.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001542-g005" data-doi="info:doi/10.1371/journal.pone.0001542" data-uri="info:doi/10.1371/journal.pone.0001542.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001542">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001542" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>While comparative bacterial genomic studies commonly predict a set of genes indicative of common ancestry, experimental validation of the existence of this core genome requires extensive measurement and is typically not undertaken. Enabled by an extensive proteome database developed over six years, we have experimentally verified the expression of proteins predicted from genomic ortholog comparisons among 17 environmental and pathogenic bacteria. More exclusive relationships were observed among the expressed protein content of phenotypically related bacteria, which is indicative of the specific lifestyles associated with these organisms. Although genomic studies can establish relative orthologous relationships among a set of bacteria and propose a set of ancestral genes, our proteomics study establishes expressed lifestyle differences among conserved genes and proposes a set of expressed ancestral traits.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Callister SJ, McCue LA, Turse JE, Monroe ME, Auberry KJ, et al.  (2008) Comparative Bacterial Proteomics: Analysis of the Core Genome Concept. PLoS ONE 3(2):
          e1542.
            doi:10.1371/journal.pone.0001542</p><p><strong>Academic Editor: </strong>James Fraser, University of Queensland, Australia</p><p><strong>Received:</strong> October 19, 2007; <strong>Accepted:</strong> January 9, 2008; <strong>Published:</strong> February 6, 2008</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>The research described in this paper was performed in the Environmental Molecular Sciences Laboratory, a national scientific user facility sponsored by the Department of Energy's Office of Biological and Environmental Research and located at Pacific Northwest National Laboratory. Portions of this work were supported by the Department of Energy Office of Biological and Environmental Research at PNNL grant (ER63232-1018220-0007203), the National Institute of Allergy and Infectious Diseases (NIH/DHHS through interagency agreement Y1-AI-4894-01) and the NIH National Center for Research Resources (RR18522). PNNL is a multi-program national laboratory operated by Battelle for the DOE under Contract DE-AC05-76RLO 1830.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>As a result of the numerous bacterial genome sequences currently available, the concept of a core genome–a set of orthologous genes commonly derived in bacterial genomic studies–is being used increasingly to explore genomic relationships among bacteria. For example, important insights into the origin of photosynthesis were recently obtained from the analysis of 892 core genes identified among 15 cyanobacteria genomes <a href="#pone.0001542-Mulkidjanian1">[1]</a>. In another comparative genomic study of 4 magnetotactic bacteria, several unique genes from a core of 891 genes were identified as potentially important to the magnetic field sensing and taxis abilities of this group of prokaryotes <a href="#pone.0001542-Richter1">[2]</a>. A general observation from these studies is that the number of genes that make up the core genome depends on the number and diversity of organisms being compared <a href="#pone.0001542-Mulkidjanian1">[1]</a>–<a href="#pone.0001542-Sarkar1">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>While the use of the core genome concept has led to important insights into the evolution of bacterial species and identification of potentially important novel genes, there has been little discussion regarding actual expression of the core genome genes as proteins and the extent of this expression across the set of bacteria under study. The assumption that a gene will always produce a gene product, i.e., protein, is debatable as evidence suggests that genes are silenced by evolutionary mechanisms and as such, will not be expressed <a href="#pone.0001542-Mira1">[8]</a>. Thus, the expression of a core gene in one organism, but not in another can provide insight into the effects of both evolution and environmental pressures on the expressed phenotype. Yet the expression of identified genes within core genomes is rarely verified by experimental observation due to the extensive resources and rigorous experimental design required to do so.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We hypothesized that a core genome could be supported by a set of conserved proteins or core proteome, where the proteome is defined as the collection of structural and functional proteins actually present in the cell <a href="#pone.0001542-Zimmer1">[9]</a> and is thus a direct expression of cell phenotype <a href="#pone.0001542-Costas1">[10]</a>, <a href="#pone.0001542-Cruz1">[11]</a>. Herein, we show that examination of this hypothesis has important implications for a broad range of microbiological applications, such as determining the essentiality of genes derived from the core genome, deriving traits that correspond to a common ancestor (orthology) <a href="#pone.0001542-Harris1">[4]</a>, <a href="#pone.0001542-Kyrpides1">[12]</a>, and on a more practical note, the direct identification of therapeutic and environmental targets or markers for additional characterization.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Enabled by a database of ~967,000 experimentally determined unique peptides linked to specific protein information and publicly available genome sequences, we examined protein expression in a core genome of 17 bacteria. The peptide database is the result of high-throughput liquid chromatography mass spectrometry-based proteomics measurements obtained over six-years. Among the selected bacterial genotypes are the phyla Actinobacteria, Deinococcus-Thermus, Proteobacteria, and Cyanobacteria representing large evolutionary distances (based on 16S rDNA sequence alignment), as well as the species <em>Geobacter metallireducens</em> and <em>Geobacter sulfurreducens</em> that represent relatively short evolutionary distance. Notable bacteria include both pathogens, e.g., the <em>Yersinia</em> species and environmental bacteria, e.g., the metabolically diverse <em>Rhodobacter sphaeroides</em> and the ocean-dwelling <em>Peligibacter ubique</em>. We first identified a core genome by predicting orthologs among consecutively larger numbers of the bacteria (from 2 to 17), using the INPARANOID algorithm <a href="#pone.0001542-Remm1">[13]</a> in conjunction with BLAST <a href="#pone.0001542-Altschul1">[14]</a>. Next, we searched our peptide databases for proteins that corresponded to the predicted orthologs. We required a minimum of two unique peptides identified using tandem mass spectrometry in conjunction with the SEQUEST algorithm <a href="#pone.0001542-Eng1">[15]</a> to confirm the presence of a protein in each organism.</p>

<h4>Identified Orthologs Supported by Protein Observation</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>On the basis of our experimental design, we surmised that the likelihood of observing a large percentage of proteins from our core genome would be small because of phylogenetic distance and the difference in environments required for growth. However, we were surprised that 105 (74%) of the 144 predicted orthologs that comprised our core genome had corresponding proteins expressed across all 17 bacteria (<a href="#pone-0001542-g001">Fig 1</a>). The percentage of observed proteins initially decreased as the number of selected bacteria increased from 2 to 5, but then increased as the number increased from 5 to 17 organisms. The former trend highlights the bias that results from selecting several pairs and a triplicate of bacteria that were related by the same genus, had similar growth environments, and had a proportionately large number of genomic orthologs identified among them (<a href="#pone-0001542-g002">Fig. 2A</a>). The latter trend suggests that the likelihood of proteins being observed and expressed in nature increases when they represent orthologs among multiple organisms (in this study, &gt;5 bacteria).</p>
<div class="figure" id="pone-0001542-g001"><div class="img"><a name="pone-0001542-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001542" data-uri="info:doi/10.1371/journal.pone.0001542.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001542.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001542.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The relationship between the number of bacteria and the percent of observed proteins from predicted orthologs.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The number of predicted orthologs represents the sum of orthologs identified <em>in-silico</em> from any combination of bacteria within the set. As the number of organisms increased from 5 to 17, proteomes among the individual bacteria converged to a set of conserved proteins, or core proteome.</p>
<span>doi:10.1371/journal.pone.0001542.g001</span></div><div class="figure" id="pone-0001542-g002"><div class="img"><a name="pone-0001542-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001542" data-uri="info:doi/10.1371/journal.pone.0001542.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001542.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001542.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Predicted orthologs verified by proteomic observations.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>A) Orthologs were predicted between consecutively greater numbers of bacteria, beginning with all pairwise combinations and ending with all of the 17 bacteria. Clustered results reveal a core genome of 144 genes and more exclusive orthologs between bacteria of the same species. B) Observed protein orthologs measured using liquid chromatography tandem mass spectrometry were included and given greater weight than predicted orthologs only. Clustering resulted in improved agreement with phylogenetic predictions. 105 of the 144 core genes were verified by protein observations, which represent the core proteome for the set of bacteria.</p>
<span>doi:10.1371/journal.pone.0001542.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>As the number of organisms increased to 17, the proteomes of the individual organisms converged upon a set of conserved proteins; that is, the core proteome. Overall, our genomic comparison established the relative orthologous relationships among the 17 bacteria and proposed a set of possible ancestral genes assumed to be orthologous <a href="#pone.0001542-Fitch1">[16]</a>. Comparative proteomic measurements were then used to establish expressed lifestyle differences among these relationships (<a href="#pone-0001542-g002">Fig. 2B</a>), as well as proposed a set of expressed traits associated with an ancestral bacteria.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Further evaluation of organism-specific proteomes revealed that a significant percentage of each proteome is composed of peptides representative of core proteome proteins (<a href="#pone-0001542-g003">Fig. 3A</a>). This observation was independent of the size of an organism's proteome. For example, ~68% of the <em>Y. pestis</em> proteome, which had the second smallest set of observed peptides, and ~62% of the <em>Salmonella enterica</em> subsp. <em>enterica</em> serotype Typhimurium LT2 <em>(S. typhimurium)</em> proteome, which had the second largest set of observed peptides, were composed of unique peptides from the core proteome. The set of peptides observed for each of the 17 organisms ranged in number from 11,870 to 103,873.</p>
<div class="figure" id="pone-0001542-g003"><div class="img"><a name="pone-0001542-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001542" data-uri="info:doi/10.1371/journal.pone.0001542.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001542.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001542.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>The analysis of peptides and their corresponding proteins identified within each bacterium's database of observed peptides.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Organisms on the x-axis are sorted by proteome size (increasing). A) The percent of each proteome composed of peptides identifying core proteins predicted by the core genome. A significant percentage of each proteome was composed of these peptides, which suggests that they are regularly observed. B) The percentage of core proteins observed out of the total number of proteins identified by peptides within each proteome. As the number of peptides required to identify a protein increased (from 2 to 6 peptides), the percentage of core proteins out of the total number of observed proteins also increased.</p>
<span>doi:10.1371/journal.pone.0001542.g003</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>At the protein level, the percentage of observed proteins within each proteome that corresponded to core proteins increased as the number of peptides required to identify a protein was increased from 2 to 6 peptides (<a href="#pone-0001542-g003">Fig. 3B</a>). For example, 2547 proteins from the <em>R. sphaeroides</em> proteome database, including 141 proteins expressed from the core genome, were identified by 2 or more unique peptides (5.5%). Increasing the stringency from 2 to 6 peptides resulted in 1504 identified proteins composed of 129 core proteins (8.6%). For specific organisms such as <em>R. sphaeroides</em>, the observed proteome was constructed from as few as two culture conditions <a href="#pone.0001542-Callister1">[17]</a>; whereas, the observed proteome for <em>S. oneidensis</em> was generated from many (~10) culture conditions. Based on the large percentage of observed proteins representative of the core proteome among a number of different culture conditions, we conclude that the core proteome is largely ubiquitous, in great abundance, and likely independent of culture condition.</p>


<h4>Functional Characterization of the Core Proteome</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In terms of functional assignments (<a href="http://www.tigr.org">www.tigr.org</a>), a little over half (55%) of the proteins observed from the core genome are devoted to protein synthesis (<a href="#pone-0001542-g004">Fig.4</a>) and composed of ribosomal proteins and functional proteins associated with tRNA-aminoacylation, including methionyl-tRNA formyltransferase and methionyl-tRNA synthetase. Strikingly, ~7% of the observed proteins have not been completely characterized with regard to functionality (<a href="#pone.0001542.s002">Table S1</a>). For example, a single protein in the core proteome was assigned a general regulatory function (<a href="#pone-0001542-g004">Fig. 4</a>). Designated as BipA /TypA, this protein belongs to the elongation factor GTPase superfamily and affects cellular function under multiple growth conditions <a href="#pone.0001542-Owens1">[18]</a>. Although BipA interacts with the ribosome and its GTPase activity is directly connected to the 70S ribosome charged with mRNA and aminoacylated tRNAs in <em>Escherichia coli</em>, its regulatory role remains unknown <a href="#pone.0001542-Owens1">[18]</a>.</p>
<div class="figure" id="pone-0001542-g004"><div class="img"><a name="pone-0001542-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001542" data-uri="info:doi/10.1371/journal.pone.0001542.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001542.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001542.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Functional categories (http://www.tigr.org/) assigned to the core genome and core proteome.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The largest portion of the core proteome is involved in protein synthesis, which suggests the essentiality of these protein synthesis functions to freeliving bacteria. However, several proteins that were not well characterized according to functional category were also observed as part of the core proteome, which highlights the need for better characterization of these proteins.</p>
<span>doi:10.1371/journal.pone.0001542.g004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>As another example, a recent review of previously identified core genes placed the <em>ybeB</em> gene near the top of a list that prioritized targets for experimentation <a href="#pone.0001542-Galperin1">[19]</a>. Observed as a core protein within our bacterial set, this small protein (~11 to ~13 kDa) is a homolog of the Iojap plant protein. Mutations of this gene (e.g., in maize) lack expression of a plastid encoded RNA polymerase that exhibits some sequence similarity to bacterial RNA polymerases <a href="#pone.0001542-Silhavy1">[20]</a>. Recent evidence suggests this protein is associated with the 50S ribosomal subunit <a href="#pone.0001542-Jiang1">[21]</a> and/or is involved in cell division <a href="#pone.0001542-Bernhardt1">[22]</a>. Alignment of secondary structure predictions based on amino acid sequence <a href="#pone.0001542-Cheng1">[23]</a>, <a href="#pone.0001542-Tamura1">[24]</a> for this protein indicate a high degree of symmetry around an alpha-helix structure (<a href="#pone.0001542.s001">Fig. S1</a>), similar to the secondary structure of the protein Calmodulin, which binds many protein targets and is involved in multiple cell functions. The presence of this protein and other such poorly characterized proteins across a broad spectrum of bacteria suggests a need for a greater understanding of basal functions associated with the free-living bacterial domain.</p>


<h4>Relative Comparison of Expressed Lifestyles</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>While certain basal functions across the set of 17 organisms are represented by conserved proteins within the core proteome, lifestyle differences become distinguishable outside the core (<a href="#pone-0001542-g002">Fig. 2B</a>). At the most exclusive regions where the search for ortholog assignments was between combinations of two organisms, large otholog clusters were identified for close phylogenetically related organisms. For example, a large cluster of 425 orthologs was identified as unique to the two <em>Geobacter</em> species in our bacterial set. Both organisms were cultured in the presence of Fe(III) citrate and 288 of the 425 identified orthologs had proteins expressed in both organisms, which demonstrates similar lifestyle responses to this electron acceptor. When <em>G. sulfurreducens</em> was cultured in the presence of fumarate and <em>G. metallireducens</em> in the presence of nitrate, differences in lifestyles associated with these two electron acceptors were also observed as a result of the different culture environments. Against the backdrop of unique orthologs, the lifestyle similarities and differences of these environmentally important metal reducing bacteria <a href="#pone.0001542-Methe1">[25]</a>, <a href="#pone.0001542-Lovley1">[26]</a> may serve as important environmental indicators for heavy metal reduction and as markers for monitoring the redox state required to maintain the immobilization of toxic metals.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Our comparative bacterial proteomic analysis lends itself to a unique reductionist approach for comparing lifestyles relative to a selected bacterium. <a href="#pone-0001542-g005">Figure 5</a> shows individual proteomes of 16 of the organisms normalized relative to <em>S. typhimurium</em> (<a href="#pone-0001542-g005">Fig. 5</a>). In this comparison, the core proteome across all bacteria gives way to smaller subsets of common proteins among consecutively smaller numbers of bacteria. Forty proteins (<a href="#pone.0001542.s003">Table S2</a>), which included a unique RNase (mRNA degradation) and asparagine synthetase (multiple isozymes reported), were observed as common solely to the <em>Yersinia</em> species and the <em>Salmonella</em> serovars. With the addition of <em>S. oneidensis</em> MR1, the number of observed proteins common to the set dropped to 26 (<a href="#pone.0001542.s003">Table S2</a>). Among the 26 was the cell division protein ZipA, which is not highly conserved and present in only a limited number of gram-negative bacteria <a href="#pone.0001542-RayChaudhuri1">[27]</a>. Ultimately, this type of comparison presents an opportunity for identifying proteins as unique environmental markers and potential broad-based or specific therapeutic targets.</p>
<div class="figure" id="pone-0001542-g005"><div class="img"><a name="pone-0001542-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001542" data-uri="info:doi/10.1371/journal.pone.0001542.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001542.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001542.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001542.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001542.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Predicted and observed orthologs shown relative to predicted and observed proteins in <em>S. typhimurium</em>.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Approximately 50% of predicted proteins in <em>S. typhimurium</em> exhibited orthology (blue area) to at least one other bacterium in the set of bacteria. As expected, <em>S. typhi</em> had the greatest degree of predicted and observed orthology to <em>S. typhimurium</em>.. Certain regions of orthology are unique to the two serovars and include A) orthologs predicted only, B) orthologs predicted in both serovars, but observed in only one, and C) orthologs predicted and observed in both serovars. Categories A and B highlight proteins for future investigation as potential therapeutic targets.</p>
<span>doi:10.1371/journal.pone.0001542.g005</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In an initial demonstration, we applied this approach to the proteomes <em>S. typhimurium</em> and <em>Salmonella enterica</em> subsp. <em>enterica</em> serotype Typhi TY2 (<em>S. typhi</em>) to generate a set of potential therapeutic protein targets. The proteomes consisted of proteins extracted from several different cultures relevant to the pathogenicity of each organism, and a majority of the predicted proteins were observed for each organism. Although the <em>Salmonella</em> serovars exhibited 84% genome hybridization similarity <a href="#pone.0001542-Chang1">[28]</a>, differences in their relative proteomic content were revealed by (<a href="#pone-0001542-g005">Fig. 5</a>): 1) unique orthologs predicted for both organisms, but not observed; 2) predicted orthologs uniquely observed, but in only one of the two serovars; and 3) predicted orthologs uniquely observed in both organisms. Proteins in the first category are of less interest as there are no experimentally observed gene products, i.e., proteins. Proteins in the second category represent an important group of potential therapeutic targets because proteome measurements delineated one organism from the other even though genomic comparisons predicted ortholog similarity. For example, 11 proteins were observed in <em>S. typhi</em>, but not in <em>S. typhimurium</em> and have putative annotations with predicted localizations in the inner membranes and cytoplasm. One of these 11 proteins is designated as a chaperone for the stabilization of fimbriae, important virulent proteins involved in the attachment of a pathogen to a host cell <a href="#pone.0001542-Reen1">[29]</a>. While the predicted orthologs from genomic comparison separate the <em>Salmonella</em> species from the rest of the bacteria in the third category, observation of gene products for each of these orthologs makes them particularly attractive as potential therapeutic targets for both serovars. A number of putative cytoplasmic, inner membrane and periplasmic proteins, as well as proteins from several characterized operons (e.g., <em>ssa</em>, <em>sse</em>, and <em>inv</em>) that contain known virulence genes make up this third category. We expect that future addition of organisms to our proteomic comparison will narrow this list to a subset of potential targets that have an even greater potential of therapeutic value.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our peptide-centric proteomic measurements experimentally demonstrate the existence of a core set of genes that define bacterial life for a diverse set of bacteria. We suspect that the number of protein encoding genes within the core genome is dependent on the number of bacteria compared, but the expression of these genes as proteins is relatively inflexible to culture condition. As such, the core proteome represents an important set of expressed conserved proteins that have survived repeated speciation events.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>An important implication of the core proteome is its essentiality to the set of bacteria studied and to the bacterial domain as a whole. In comparative genomic studies, gene essentiality is a common theme <a href="#pone.0001542-Mushegian1">[6]</a>, <a href="#pone.0001542-Klasson1">[30]</a> and is often discussed in the context of environment <a href="#pone.0001542-Fang1">[31]</a> where genes in a single species are essential for one environment, but nonessential for another. This essentiality is especially pertinent to free-living microbes, where a species must be able to subsist within a range of environmental fluctuations. For host-dependent microbes a relatively stable environment reduces the genome size compared to free-living bacteria; thereby, reducing the need for a large array of biological functions <a href="#pone.0001542-Mira2">[32]</a>. Numerous characterization studies of these minimal genomes in terms of essentiality have been performed <a href="#pone.0001542-Clark1">[3]</a>, <a href="#pone.0001542-Mushegian1">[6]</a>, <a href="#pone.0001542-Klasson1">[30]</a>, <a href="#pone.0001542-Fang1">[31]</a>, <a href="#pone.0001542-Hutchison1">[33]</a>, <a href="#pone.0001542-Koonin1">[34]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>In evaluating gene essentiality in the broader context, we conclude that essentiality of a gene for bacterial life depends in part on gene conservation among organisms <a href="#pone.0001542-Jordan1">[35]</a>, as well as on the expression of these genes as proteins regardless of environment. In generating our core genome, we emphasized gene essentiality by including <em>P. ubique</em>, which has the fewest number of predicted protein encoding genes relative to its genome size of any free-living organism <a href="#pone.0001542-Giovannoni1">[36]</a> and by requiring gene conservation across a phylogenetically diverse bacterial set. Our observed core proteome also suggests the need to perform random mutagenesis on individual species within our selected set of bacteria to further evaluate gene essentiality <a href="#pone.0001542-Hutchison1">[33]</a>, <a href="#pone.0001542-Glass1">[37]</a> in the broader context. Nevertheless, essentiality of many of the translated genes within our core proteome has been empirically shown in other model organisms, such as <em>Escherichia coli</em> K12 MG1655 (<a href="#pone.0001542.s004">Table S3</a>) <a href="#pone.0001542-Gerdes1">[38]</a> and <em>S. typhimurium</em> <a href="#pone.0001542-Knuth1">[39]</a> (<a href="#pone.0001542.s004">Table S3</a>).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In this study, observation of an ortholog in one organism and not another is likely a result of phenotypic plasticity, i.e., the ability of an organism to change its phenotype based on environment <a href="#pone.0001542-WestEberhard1">[40]</a>. (Admittedly, the lack of protein observations in some organisms, for example <em>Y. pestis</em>, could also result from the more modest set of experimental results as compared to the other organisms.) The effect of plasticity on the hierarchical clustering of orthologs is illustrated in <a href="#pone-0001542-g002">Figure 2</a>. <a href="#pone-0001542-g002">Figure 2A</a> predicts a common set of genes at each internal node (possibly ancestral genes), and the order of clustered organisms is in reasonable agreement with established phylogeny. Conversely, the clustering of expressed proteins from predicted orthologs (<a href="#pone-0001542-g002">Fig. 2B</a>) represents the union of phenotypic traits (possibly common ancestral traits) at internal nodes, with the core proteome representing the root node.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>As one progresses from the internal nodes toward the root node within the hierarchical structure presented in <a href="#pone-0001542-g002">Figure 2B</a>, one might speculate that the reduced degree of phenotypic plasticity suggests these common phenotypic traits are increasingly independent of the current niches of our bacterial set and rather are representative of a primordial niche. Ultimately, understanding phenotypic plasticity will be important to researchers interested in designing synthetic organisms for the purpose of biofuel production, pollution clean-up etc. <a href="#pone.0001542-Stone1">[41]</a>, as a baseline of phenotypic traits will be required. To accomplish these designs, a set of relatively non-plastic phenotypic traits needs to be identified, which cannot be determined by genomics alone.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Organisms and Culture Conditions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The bacteria used for this study were previously cultured by several laboratories interested in the proteomic characterization of a given organism. Samples were kindly generated (Acknowledgements) for the purpose of developing an observed reference peptide database for each organism, utilizing the high-throughput proteomic capabilities present at Pacific Northwest National Laboratory, Richland WA. Many of the laboratory culture conditions have previously been published in connection with the primary proteomics work being conducted at Pacific Northwest National Laboratory <a href="#pone.0001542-Callister1">[17]</a>, <a href="#pone.0001542-Adkins1">[42]</a>–<a href="#pone.0001542-Zhang2">[55]</a>.</p>


<h4>Sample Preparation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Either an established <a href="#pone.0001542-Smith1">[52]</a> or optimized <a href="#pone.0001542-Callister1">[17]</a>, <a href="#pone.0001542-Adkins1">[42]</a>–<a href="#pone.0001542-Ansong1">[56]</a> protein extraction protocol was applied to each cell culture. In brief, global (total), insoluble, and soluble protein digests were extracted from lysed cultures that were washed and suspended in 100 mM NH<sub>4</sub>HCO<sub>3</sub>, pH 8.4 buffer. For global extracts, proteins were denatured and reduced by adding urea, thiourea, and dithiothreitol (DTT) followed by incubation at ~60°C for ~30 min. Following incubation, the global protein samples were diluted to reduce salt concentration then proteolytic digested, at 37°C for ~4 h, using sequencing grade trypsin (Roche, Indianapolis, IN) at a ratio of 1 unit per 50 units of protein (1 unit = ~1 µg of protein). Following incubation, digested samples were desalted using an appropriately sized C-18 SPE column (Supelco, St. Louis, MO) and a vacuum manifold. The collected peptides were concentrated to a final volume ranging from 50 µl to 100 µl and measured using the BCA assay (Pierce Chemical Co., Rockfort, IL) according to the manufacturer's instructions.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>For the insoluble protein digest, the cell lysate was ultracentrifuged at 4°C and 100,000 rpm for 10 min. The resulting supernatant that contained soluble proteins was separated from the pellet and retained for digestion as previously described for the global extraction. The pellet was washed by suspending it in 100 mM NH<sub>4</sub>HCO<sub>3</sub>, pH 7.8, using mild sonication and then ultracentrifuged at 100,000 rpm for 5 min, again at 4°C. Following centrifugation, the pellet was resuspended in a solubilizing solution that contained urea, thiourea, 1% CHAPS in 50 mM NH<sub>4</sub>HCO<sub>3</sub>, pH 7.8. An aliquot of 50 mM DTT solution was also added to final concentration of 5 mM. The insoluble protein sample was then incubated and digested as described above with the exception that a 50 mM NH<sub>4</sub>HCO<sub>3</sub>, pH 7.8 buffer was used for the dilution step. Following proteolytic digestion, the pH of the sample was slowly lowered to &lt;4.0 by adding small volumes (1 µl to 2 µl) of 20% formic acid. Removal of salts and detergent was performed using either an appropriately sized strong cation exchange (SCX) or solid phase extraction column (Supelco, St. Louis, MO) and vacuum manifold. Peptides were then concentrated and their concentration measured as described above.</p>


<h4>Database Generation and Filtering</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Databases of observed peptides were generated according to an established protocol <a href="#pone.0001542-Smith1">[52]</a>, <a href="#pone.0001542-Kiebel1">[57]</a>, <a href="#pone.0001542-Monroe1">[58]</a>. In brief, peptides from the global, insoluble, and soluble digests were fractionated (25 to 100 fractions each) using high resolution reversed-phase SCX high pressure liquid chromatography (HPLC). The HPLC system was operated in an exponential gradient mode with mobile phase B (0.1% TFA in 90% ACN and 10% water) replacing mobile phase A (0.2% acetic acid, 0.05% TFA in water) 10 min after sample injection, which was accomplished by using an in-house mixer, capillary column selector, and sample loop.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>From each collected fraction, a consistent mass of peptides were analyzed by reversed phase HPLC coupled on line to an ion trap mass spectrometer (LCQ and/or LTQ ThermoFischer, San Jose, CA) operated in a data-dependent MS/MS mode. MS/MS spectra were analyzed using the SEQUEST algorithm <a href="#pone.0001542-Eng1">[15]</a> in conjunction with publicly available predicted protein sequences from the appropriate genome sequence. Preliminary filtering of identified peptides was performed using a minimum cross-correlation cut-off (<em>Xcorr</em>) of either 1.9, 2.2, or 3.75 for 1+, 2+, or 3+ charge states, respectively, for fully tryptic (peptides that contained either an arginine or lysine at the site of cleavage), partially tryptic, and non-tryptic peptides. All peptides were a minimum of 6 amino acids long. For this specific study, peptides in the databases were further filtered using a PeptideProphet <a href="#pone.0001542-Keller1">[59]</a> score of at least 0.90. Note that PeptideProphet calculates the probability that a peptide sequence has been correctly assigned <a href="#pone.0001542-Keller1">[59]</a>. Although database dependent, filtering on a PeptideProphet score of 0.95 roughly corresponds to a ~5% false discovery rate based on reverse database searching techniques <a href="#pone.0001542-Luo1">[47]</a>, <a href="#pone.0001542-Qian1">[60]</a>.</p>


<h4>Ortholog Identification</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Orthologs were idendified using INPARANOID v.1.35 <a href="#pone.0001542-Remm1">[13]</a>. This program uses BLAST <a href="#pone.0001542-Altschul1">[14]</a> to compare the complete set of protein sequences from one genome with that of another, and identifies the reciprocal best hits. We set the parameters to utilize the BLOSUM62 matrix and a minimum bit score of 30, and we required that the BLAST alignment cover at least 50% of both proteins. The resulting ortholog tables were analyzed by Perl scripts to identify complete ortholog graphs (<a href="http://mathworld.wolfram.com/CompleteGraph.html">http://mathworld.wolfram.com/CompleteGra​ph.html</a>) where the nodes of the graphs are the proteins and the edges are the INPARANOID ortholog connections. Complete ortholog graphs have <em>n</em> nodes and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001542.e001&amp;representation=PNG" class="inline-graphic"></span> edges, where <em>n</em> is the number of input genomes.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001542"></div><a name="pone.0001542.s001" id="pone.0001542.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001542.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Aligned secondary structure predictions based on amino acid sequence for YbeB. A conserved and symmetrical secondary structure was predicted for this protein indicative of a possible binding protein. H-helix; E-extended strand</p>
<p class="siDoi">doi:10.1371/journal.pone.0001542.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.23 MB PDF)</p>
<a name="pone.0001542.s002" id="pone.0001542.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001542.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Core proteins described as having a general functional characterization or no functional characterization.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001542.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.14 MB PDF)</p>
<a name="pone.0001542.s003" id="pone.0001542.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001542.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Conserved proteins from different sub-sets of bacteria relative to <em>S. typhimurium</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001542.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.12 MB PDF)</p>
<a name="pone.0001542.s004" id="pone.0001542.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001542.s004">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Core proteome proteins observed in <em>E. coli</em> K12 MG1655 and <em>S. typhimurium</em> and their published genes noted as essential. (Source: Gerdes, et al. 2003. <em>J. Bacteriol.</em> 185(19):5673–5684; Knuth, et al. 2004. <em>Mol Microbiol.</em> 51(6):1729–1744)</p>
<p class="siDoi">doi:10.1371/journal.pone.0001542.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.25 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors acknowledge Penny Colton for technical editing and to express gratitude to the <em>Shewanella</em> Federation, the Biological Separations and Mass Spectrometry Group at Pacific Northwest National Laboratory, and the following individuals and institutions that have graciously supplied cell cultures over the years: M. J. Daly (Uniformed Services University of the Health Sciences, Bethesda MD); T. J. Donohue (University of Wisconsin, Madison WI); S. J. Giovannoni (Oregon State University, Corvallis OR); F. Heffron (Oregon Health and Science University, Portland OR); S. Kaplan (University of Texas Health Science Center, Houston TX); A. E. Konopka (Pacific Northwest National Laboratory, Richland WA); L. R. Krumholz (University of Oklahoma, Norman OK); S. B. Levy (Tufts University, Boston MA); D. R. Lovely (University of Massachusetts, Amherst MA); S. L. McCutchen-Maloney (Lawrence Livermore National Laboratory, Livermore CA); H. B. Pakrasi (Washington University in St. Louis MO); L. Shapiro (Stanford University, Stanford CA); D. J. Wall (University of Missouri, Columbia MO). A request for available mass spectrometry data can be made at <a href="http://omics.pnl.gov/">http://omics.pnl.gov/</a>; <a href="http://www.proteomicsresource.org/">http://www.proteomicsresource.org/</a>; <a href="http://ober-proteomics.pnl.gov/">http://ober-proteomics.pnl.gov/</a></p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SC ML JA. Analyzed the data: SC ML JT JA. Contributed reagents/materials/analysis tools: MM LM KA. Wrote the paper: SC. Other: PI for grants that provided partial support: RS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001542-Mulkidjanian1" id="pone.0001542-Mulkidjanian1"></a>Mulkidjanian AY, Koonin EV, Makarova KS, Mekhedov SL, Sorokin A, et al.  (2006) The cyanobacterial genome core and the origin of photosynthesis. Proc Natl Acad Sci U S A  103: 13126–13131.  <ul class="find" data-citedArticleID="1050594" data-doi="10.1073/pnas.0605709103"><li><a href="http://dx.doi.org/10.1073/pnas.0605709103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+cyanobacterial+genome+core+and+the+origin+of+photosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+cyanobacterial+genome+core+and+the+origin+of+photosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001542-Richter1" id="pone.0001542-Richter1"></a>Richter M, Kube M, Bazylinski DA, Lombardot T, Glockner FO, et al.  (2007) Comparative genome analysis of four magnetotactic bacteria reveals a complex set of group-specific genes implicated in magnetosome biomineralization and function. J Bacteriol  189: 4899–4910.  <ul class="find" data-citedArticleID="1050608" data-doi="10.1128/jb.00119-07"><li><a href="http://dx.doi.org/10.1128/jb.00119-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genome+analysis+of+four+magnetotactic+bacteria+reveals+a+complex+set+of+group-specific+genes+implicated+in+magnetosome+biomineralization+and+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genome+analysis+of+four+magnetotactic+bacteria+reveals+a+complex+set+of+group-specific+genes+implicated+in+magnetosome+biomineralization+and+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001542-Clark1" id="pone.0001542-Clark1"></a>Clark MA, Baumann L, Thao MLL, Moran NA, Baumann P (2001) Degenerative minimalism in the genome of a psyllid endosymbiont. J Bacteriol  183: 1853–1861.  <ul class="find" data-citedArticleID="1050530" data-doi="10.1128/jb.183.6.1853-1861.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.6.1853-1861.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Degenerative+minimalism+in+the+genome+of+a+psyllid+endosymbiont." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Degenerative+minimalism+in+the+genome+of+a+psyllid+endosymbiont.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001542-Harris1" id="pone.0001542-Harris1"></a>Harris JK, Kelley ST, Spiegelman GB, Pace NR (2003) The genetic core of the universal ancestor. Genome Res  13: 407–412.  <ul class="find" data-citedArticleID="1050558" data-doi="10.1101/gr.652803"><li><a href="http://dx.doi.org/10.1101/gr.652803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+core+of+the+universal+ancestor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+core+of+the+universal+ancestor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001542-Lefebure1" id="pone.0001542-Lefebure1"></a>Lefebure T, Stanhope MJ (2007) Evolution of the core and pan-genome of Strepococcus: positive selection, recombination, and genome composition. Genome Biol  8: Art. No. R71 2007.  <ul class="find" data-citedArticleID="1050578" data-doi="10.1186/gb-2007-8-5-r71"><li><a href="http://dx.doi.org/10.1186/gb-2007-8-5-r71" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+core+and+pan-genome+of+Strepococcus%3A+positive+selection%2C+recombination%2C+and+genome+composition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+core+and+pan-genome+of+Strepococcus%3A+positive+selection%2C+recombination%2C+and+genome+composition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001542-Mushegian1" id="pone.0001542-Mushegian1"></a>Mushegian AR, Koonin EV (1996) A minimal gene set for cellular life derived by comparison of complete bacterial genomes. Proc Natl Acad Sci U S A  93: 10268–10273.  <ul class="find" data-citedArticleID="1050596" data-doi="10.1073/pnas.93.19.10268"><li><a href="http://dx.doi.org/10.1073/pnas.93.19.10268" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+minimal+gene+set+for+cellular+life+derived+by+comparison+of+complete+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+minimal+gene+set+for+cellular+life+derived+by+comparison+of+complete+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001542-Sarkar1" id="pone.0001542-Sarkar1"></a>Sarkar SF, Guttman DS (2004) Evolution of the core genome of <em>Pseudomonas syringae</em>, a highly clonal, endemic plant pathogen. Appl Environ Microbiol  70: 1999–2012.  <ul class="find" data-citedArticleID="1050612" data-doi="10.1128/aem.70.4.1999-2012.2004"><li><a href="http://dx.doi.org/10.1128/aem.70.4.1999-2012.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+core+genome+of+Pseudomonas+syringae%2C+a+highly+clonal%2C+endemic+plant+pathogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+core+genome+of+Pseudomonas+syringae%2C+a+highly+clonal%2C+endemic+plant+pathogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001542-Mira1" id="pone.0001542-Mira1"></a>Mira A, Pushker R (2005) The silencing of pseudogenes. Mol Biol Evol  22: 2135–2138.  <ul class="find" data-citedArticleID="1050588" data-doi="10.1093/molbev/msi209"><li><a href="http://dx.doi.org/10.1093/molbev/msi209" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+silencing+of+pseudogenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+silencing+of+pseudogenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001542-Zimmer1" id="pone.0001542-Zimmer1"></a>Zimmer JSD, Monroe ME, Qian WJ, Smith RD (2006) Advances in proteomics data analysis and display using an accurate mass and time tag approach. Mass Spectrom Rev  25: 450–482.  <ul class="find" data-citedArticleID="1050634" data-doi="10.1002/mas.20071"><li><a href="http://dx.doi.org/10.1002/mas.20071" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Advances+in+proteomics+data+analysis+and+display+using+an+accurate+mass+and+time+tag+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Advances+in+proteomics+data+analysis+and+display+using+an+accurate+mass+and+time+tag+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001542-Costas1" id="pone.0001542-Costas1"></a>Costas M (1990) Numerical-Analysis of sodium dodecyl sulfate-polyacrylamide gel-electrophoretic protein-patterns for the classification, identification and typing of medically important bacteria. Electrophoresis  11: 382–391.  <ul class="find" data-citedArticleID="1050532" data-doi="10.1002/elps.1150110507"><li><a href="http://dx.doi.org/10.1002/elps.1150110507" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Numerical-Analysis+of+sodium+dodecyl+sulfate-polyacrylamide+gel-electrophoretic+protein-patterns+for+the+classification%2C+identification+and+typing+of+medically+important+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Numerical-Analysis+of+sodium+dodecyl+sulfate-polyacrylamide+gel-electrophoretic+protein-patterns+for+the+classification%2C+identification+and+typing+of+medically+important+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001542-Cruz1" id="pone.0001542-Cruz1"></a>Cruz CMV, Gossele F, Kersters K, Segers P, Vandenmooter M, et al.  (1984) Differentiation between <em>Xanthomonas-campestris</em> pv oryzae, <em>Xanthomonas-campestris</em> pv oryzicola and the bacterial brown blotch pathogen on rice by numerical-analysis of phenotypic features and protein gel electrophoregrams. J Gen Microbiol  130: 2983–2999.  <ul class="find" data-citedArticleID="1050534" data-doi="10.1099/00221287-130-11-2983"><li><a href="http://dx.doi.org/10.1099/00221287-130-11-2983" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differentiation+between+Xanthomonas-campestris+pv+oryzae%2C+Xanthomonas-campestris+pv+oryzicola+and+the+bacterial+brown+blotch+pathogen+on+rice+by+numerical-analysis+of+phenotypic+features+and+protein+gel+electrophoregrams." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differentiation+between+Xanthomonas-campestris+pv+oryzae%2C+Xanthomonas-campestris+pv+oryzicola+and+the+bacterial+brown+blotch+pathogen+on+rice+by+numerical-analysis+of+phenotypic+features+and+protein+gel+electrophoregrams.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001542-Kyrpides1" id="pone.0001542-Kyrpides1"></a>Kyrpides N, Overbeek R, Ouzounis C (1999) Universal protein families and the functional content of the last universal common ancestor. J Mol Evol  49: 413–423.  <ul class="find" data-citedArticleID="1050576" data-doi="10.1007/pl00006564"><li><a href="http://dx.doi.org/10.1007/pl00006564" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Universal+protein+families+and+the+functional+content+of+the+last+universal+common+ancestor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Universal+protein+families+and+the+functional+content+of+the+last+universal+common+ancestor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001542-Remm1" id="pone.0001542-Remm1"></a>Remm M, Storm CEV, Sonnhammer ELL (2001) Automatic clustering of orthologs and in-paralogs from pairwise species comparisons. J Mol Biol  314: 1041–1052.  <ul class="find" data-citedArticleID="1050606" data-doi="10.1006/jmbi.2000.5197"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.5197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automatic+clustering+of+orthologs+and+in-paralogs+from+pairwise+species+comparisons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automatic+clustering+of+orthologs+and+in-paralogs+from+pairwise+species+comparisons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001542-Altschul1" id="pone.0001542-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang JH, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Research  25: 3389–3402.  <ul class="find" data-citedArticleID="1050518" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001542-Eng1" id="pone.0001542-Eng1"></a>Eng JK, McCormack AL, Yates JR (1994) An approach to correlate tandem mass-spectral data of peptides with amino-acid-sequences in a protein database. J Am Soc Mass Spectrom  5: 976–989.  <ul class="find" data-citedArticleID="1050542" data-doi="10.1016/1044-0305(94)80016-2"><li><a href="http://dx.doi.org/10.1016/1044-0305(94)80016-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+approach+to+correlate+tandem+mass-spectral+data+of+peptides+with+amino-acid-sequences+in+a+protein+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+approach+to+correlate+tandem+mass-spectral+data+of+peptides+with+amino-acid-sequences+in+a+protein+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001542-Fitch1" id="pone.0001542-Fitch1"></a>Fitch WM (1970) Distinguishing homologous from analogous proteins. Systematic Zoology  19: 99–&amp;.  <ul class="find" data-citedArticleID="1050548" data-doi="10.2307/2412448"><li><a href="http://dx.doi.org/10.2307/2412448" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinguishing+homologous+from+analogous+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinguishing+homologous+from+analogous+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001542-Callister1" id="pone.0001542-Callister1"></a>Callister SJ, Nicora CD, Zeng XH, Roh JH, Dominguez MA, et al.  (2006) Comparison of aerobic and photosynthetic <em>Rhodobacter sphaeroides</em> 2.4.1 proteomes. J Microbiol Methods  67: 424–436.  <ul class="find" data-citedArticleID="1050524" data-doi="10.1016/j.mimet.2006.04.021"><li><a href="http://dx.doi.org/10.1016/j.mimet.2006.04.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+aerobic+and+photosynthetic+Rhodobacter+sphaeroides+2.4.1+proteomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+aerobic+and+photosynthetic+Rhodobacter+sphaeroides+2.4.1+proteomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001542-Owens1" id="pone.0001542-Owens1"></a>Owens RM, Pritchard G, Skipp P, Hodey M, Connell SR, et al.  (2004) A dedicated translation factor controls the synthesis of the global regulator Fis. Embo J  23: 3375–3385.  <ul class="find" data-citedArticleID="1050598" data-doi="10.1038/sj.emboj.7600343"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600343" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+dedicated+translation+factor+controls+the+synthesis+of+the+global+regulator+Fis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+dedicated+translation+factor+controls+the+synthesis+of+the+global+regulator+Fis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001542-Galperin1" id="pone.0001542-Galperin1"></a>Galperin MY, Koonin EV (2004) ‘Conserved hypothetical’ proteins: prioritization of targets for experimental study. Nucleic Acids Research  32: 5452–5463.  <ul class="find" data-citedArticleID="1050550" data-doi="10.1093/nar/gkh885"><li><a href="http://dx.doi.org/10.1093/nar/gkh885" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%E2%80%98Conserved+hypothetical%E2%80%99+proteins%3A+prioritization+of+targets+for+experimental+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%E2%80%98Conserved+hypothetical%E2%80%99+proteins%3A+prioritization+of+targets+for+experimental+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001542-Silhavy1" id="pone.0001542-Silhavy1"></a>Silhavy D, Maliga P (1998) Mapping of promoters for the nucleus encoded plastid RNA polymerase (NEP) in the iojap maize mutant. Curr Genet  33: 340–344.  <ul class="find" data-citedArticleID="1050618" data-doi="10.1007/s002940050345"><li><a href="http://dx.doi.org/10.1007/s002940050345" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+of+promoters+for+the+nucleus+encoded+plastid+RNA+polymerase+%28NEP%29+in+the+iojap+maize+mutant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+of+promoters+for+the+nucleus+encoded+plastid+RNA+polymerase+%28NEP%29+in+the+iojap+maize+mutant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001542-Jiang1" id="pone.0001542-Jiang1"></a>Jiang M, Sullivan SM, Walker AK, Strahler JR, Andrews PC, et al.  (2007) Identification of novel <em>Escherichia coli</em> ribosome-associated proteins using isobaric tags and multidimensional protein identification techniques. J Bacteriol  189: 3434–3444.  <ul class="find" data-citedArticleID="1050562" data-doi="10.1128/jb.00090-07"><li><a href="http://dx.doi.org/10.1128/jb.00090-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+novel+Escherichia+coli+ribosome-associated+proteins+using+isobaric+tags+and+multidimensional+protein+identification+techniques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+novel+Escherichia+coli+ribosome-associated+proteins+using+isobaric+tags+and+multidimensional+protein+identification+techniques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001542-Bernhardt1" id="pone.0001542-Bernhardt1"></a>Bernhardt TG, de Boer PAJ (2004) Screening for synthetic lethal mutants in <em>Escherichia coli</em> and identification of EnvC (YibP) as a periplasmic septal ring factor with murein hydrolase activity. Mol Microbiol  52: 1255–1269.  <ul class="find" data-citedArticleID="1050522" data-doi="10.1111/j.1365-2958.2004.04063.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04063.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Screening+for+synthetic+lethal+mutants+in+Escherichia+coli+and+identification+of+EnvC+%28YibP%29+as+a+periplasmic+septal+ring+factor+with+murein+hydrolase+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Screening+for+synthetic+lethal+mutants+in+Escherichia+coli+and+identification+of+EnvC+%28YibP%29+as+a+periplasmic+septal+ring+factor+with+murein+hydrolase+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001542-Cheng1" id="pone.0001542-Cheng1"></a>Cheng J, Randall AZ, Sweredoski MJ, Baldi P (2005) SCRATCH: a protein structure and structural feature prediction server. Nucleic Acids Research  33: W72–W76.  <ul class="find" data-citedArticleID="1050528" data-doi="10.1093/nar/gki396"><li><a href="http://dx.doi.org/10.1093/nar/gki396" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SCRATCH%3A+a+protein+structure+and+structural+feature+prediction+server." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SCRATCH%3A+a+protein+structure+and+structural+feature+prediction+server.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001542-Tamura1" id="pone.0001542-Tamura1"></a>Tamura K, Dudley J, Nei M, Kumar S (2007) MEGA4: Molecular evolutionary genetics analysis (MEGA) software version 4.0. Mol Biol Evol  24: 1596–1599.  <ul class="find" data-citedArticleID="1050624" data-doi="10.1093/molbev/msm092"><li><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA4%3A+Molecular+evolutionary+genetics+analysis+%28MEGA%29+software+version+4.0." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA4%3A+Molecular+evolutionary+genetics+analysis+%28MEGA%29+software+version+4.0.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001542-Methe1" id="pone.0001542-Methe1"></a>Methe BA, Nelson KE, Eisen JA, Paulsen IT, Nelson W, et al.  (2003) Genome of <em>Geobacter sulfurreducens</em>: Metal reduction in subsurface environments. Science  302: 1967–1969.  <ul class="find" data-citedArticleID="1050586" data-doi="10.1126/science.1088727"><li><a href="http://dx.doi.org/10.1126/science.1088727" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+of+Geobacter+sulfurreducens%3A+Metal+reduction+in+subsurface+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+of+Geobacter+sulfurreducens%3A+Metal+reduction+in+subsurface+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001542-Lovley1" id="pone.0001542-Lovley1"></a>Lovley DR, Giovannoni SJ, White DC, Champine JE, Phillips EJP, et al.  (1993) <em>Geobacter-Metallireducens</em> gen-nov sp-vov, a microorganism capable of coupling the complete oxidation of organic-compounds to the reduction of iron and other metals. Arch Microbiol  159: 336–344.  <ul class="find" data-citedArticleID="1050580" data-doi="10.1007/bf00290916"><li><a href="http://dx.doi.org/10.1007/bf00290916" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Geobacter-Metallireducens+gen-nov+sp-vov%2C+a+microorganism+capable+of+coupling+the+complete+oxidation+of+organic-compounds+to+the+reduction+of+iron+and+other+metals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Geobacter-Metallireducens+gen-nov+sp-vov%2C+a+microorganism+capable+of+coupling+the+complete+oxidation+of+organic-compounds+to+the+reduction+of+iron+and+other+metals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001542-RayChaudhuri1" id="pone.0001542-RayChaudhuri1"></a>RayChaudhuri D (1999) ZipA is a MAP-Tau homolog and is essential for structural integrity of the cytokinetic FtsZ ring during bacterial cell division. Embo J  18: 2372–2383.  <ul class="find" data-citedArticleID="1050602" data-doi="10.1093/emboj/18.9.2372"><li><a href="http://dx.doi.org/10.1093/emboj/18.9.2372" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ZipA+is+a+MAP-Tau+homolog+and+is+essential+for+structural+integrity+of+the+cytokinetic+FtsZ+ring+during+bacterial+cell+division." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ZipA+is+a+MAP-Tau+homolog+and+is+essential+for+structural+integrity+of+the+cytokinetic+FtsZ+ring+during+bacterial+cell+division.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001542-Chang1" id="pone.0001542-Chang1"></a>Chang HR, Loo LH, Jeyaseelan K, Earnest L, Stackebrandt E (1997) Phylogenetic relationships of <em>Salmonella typhi</em> and <em>Salmonella typhimurium</em> based on 16S rRNA sequence analysis. Int J Syst Bacteriol  47: 1253–1254.  <ul class="find" data-citedArticleID="1050526" data-doi="10.1099/00207713-47-4-1253"><li><a href="http://dx.doi.org/10.1099/00207713-47-4-1253" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+relationships+of+Salmonella+typhi+and+Salmonella+typhimurium+based+on+16S+rRNA+sequence+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+relationships+of+Salmonella+typhi+and+Salmonella+typhimurium+based+on+16S+rRNA+sequence+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001542-Reen1" id="pone.0001542-Reen1"></a>Reen FJ, Boyd EF, Porwollik S, Murphy BP, Gilroy D, et al.  (2005) Genomic comparisons of <em>Salmonella enterica</em> serovar Dublin, Agona, and typhimurium strains recently isolated from milk filters and bovine samples from ireland, using a <em>Salmonella</em> microarray. Appl Environ Microbiol  71: 1616–1625.  <ul class="find" data-citedArticleID="1050604" data-doi="10.1128/aem.71.3.1616-1625.2005"><li><a href="http://dx.doi.org/10.1128/aem.71.3.1616-1625.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+comparisons+of+Salmonella+enterica+serovar+Dublin%2C+Agona%2C+and+typhimurium+strains+recently+isolated+from+milk+filters+and+bovine+samples+from+ireland%2C+using+a+Salmonella+microarray." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+comparisons+of+Salmonella+enterica+serovar+Dublin%2C+Agona%2C+and+typhimurium+strains+recently+isolated+from+milk+filters+and+bovine+samples+from+ireland%2C+using+a+Salmonella+microarray.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001542-Klasson1" id="pone.0001542-Klasson1"></a>Klasson L, Andersson SGE (2004) Evolution of minimal-gene-sets in host-dependent bacteria. Trends Microbiol  12: 37–43.  <ul class="find" data-citedArticleID="1050570" data-doi="10.1016/j.tim.2003.11.006"><li><a href="http://dx.doi.org/10.1016/j.tim.2003.11.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+minimal-gene-sets+in+host-dependent+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+minimal-gene-sets+in+host-dependent+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001542-Fang1" id="pone.0001542-Fang1"></a>Fang G, Rocha E, Danchin A (2005) How essential are nonessential genes? Mol Biol Evol  22: 2147–2156.  <ul class="find" data-citedArticleID="1050544" data-doi="10.1093/molbev/msi211"><li><a href="http://dx.doi.org/10.1093/molbev/msi211" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+essential+are+nonessential+genes%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+essential+are+nonessential+genes%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001542-Mira2" id="pone.0001542-Mira2"></a>Mira A, Ochman H, Moran NA (2001) Deletional bias and the evolution of bacterial genomes. Trends Genet  17: 589–596.  <ul class="find" data-citedArticleID="1050590" data-doi="10.1016/s0168-9525(01)02447-7"><li><a href="http://dx.doi.org/10.1016/s0168-9525(01)02447-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deletional+bias+and+the+evolution+of+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deletional+bias+and+the+evolution+of+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001542-Hutchison1" id="pone.0001542-Hutchison1"></a>Hutchison CA, Peterson SN, Gill SR, Cline RT, White O, et al.  (1999) Global transposon mutagenesis and a minimal mycoplasma genome. Science  286: 2165–2169.  <ul class="find" data-citedArticleID="1050560" data-doi="10.1126/science.286.5447.2165"><li><a href="http://dx.doi.org/10.1126/science.286.5447.2165" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+transposon+mutagenesis+and+a+minimal+mycoplasma+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+transposon+mutagenesis+and+a+minimal+mycoplasma+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001542-Koonin1" id="pone.0001542-Koonin1"></a>Koonin EV (2003) Comparative genomics, minimal gene-sets and the last universal common ancestor. Nat Rev Microbiol  1: 127–136.  <ul class="find" data-citedArticleID="1050574" data-doi="10.1038/nrmicro751"><li><a href="http://dx.doi.org/10.1038/nrmicro751" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomics%2C+minimal+gene-sets+and+the+last+universal+common+ancestor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomics%2C+minimal+gene-sets+and+the+last+universal+common+ancestor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001542-Jordan1" id="pone.0001542-Jordan1"></a>Jordan IK, Rogozin IB, Wolf YI, Koonin EV (2002) Essential genes are more evolutionarily conserved than are nonessential genes in bacteria. Genome Res  12: 962–968.  <ul class="find" data-citedArticleID="1050564" data-doi="10.1101/gr.87702"><li><a href="http://dx.doi.org/10.1101/gr.87702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Essential+genes+are+more+evolutionarily+conserved+than+are+nonessential+genes+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Essential+genes+are+more+evolutionarily+conserved+than+are+nonessential+genes+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001542-Giovannoni1" id="pone.0001542-Giovannoni1"></a>Giovannoni SJ, Tripp HJ, Givan S, Podar M, Vergin KL, et al.  (2005) Genome streamlining in a cosmopolitan oceanic bacterium. Science  309: 1242–1245.  <ul class="find" data-citedArticleID="1050554" data-doi="10.1126/science.1114057"><li><a href="http://dx.doi.org/10.1126/science.1114057" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+streamlining+in+a+cosmopolitan+oceanic+bacterium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+streamlining+in+a+cosmopolitan+oceanic+bacterium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001542-Glass1" id="pone.0001542-Glass1"></a>Glass JI, Assad-Garcia N, Alperovich N, Yooseph S, Lewis MR, et al.  (2006) Essential genes of a minimal bacterium. Proc Natl Acad Sci U S A  103: 425–430.  <ul class="find" data-citedArticleID="1050556" data-doi="10.1073/pnas.0510013103"><li><a href="http://dx.doi.org/10.1073/pnas.0510013103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Essential+genes+of+a+minimal+bacterium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Essential+genes+of+a+minimal+bacterium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001542-Gerdes1" id="pone.0001542-Gerdes1"></a>Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, et al.  (2003) Experimental determination and system level analysis of essential genes in <em>Escherichia coli</em> MG1655. J Bacteriol  185: 5673–5684.  <ul class="find" data-citedArticleID="1050552" data-doi="10.1128/jb.185.19.5673-5684.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.19.5673-5684.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+determination+and+system+level+analysis+of+essential+genes+in+Escherichia+coli+MG1655." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+determination+and+system+level+analysis+of+essential+genes+in+Escherichia+coli+MG1655.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001542-Knuth1" id="pone.0001542-Knuth1"></a>Knuth K, Niesalla H, Hueck CJ, Fuchs TM (2004) Large-scale identification of essential <em>Salmonella</em> genes by trapping lethal insertions. Mol Microbiol  51: 1729–1744.  <ul class="find" data-citedArticleID="1050572" data-doi="10.1046/j.1365-2958.2003.03944.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03944.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+identification+of+essential+Salmonella+genes+by+trapping+lethal+insertions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+identification+of+essential+Salmonella+genes+by+trapping+lethal+insertions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001542-WestEberhard1" id="pone.0001542-WestEberhard1"></a>West-Eberhard MJ (2003) Developmental Plasticity and Evolution. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0001542-Stone1" id="pone.0001542-Stone1"></a>Stone M (2007) Whole genome transplantation becomes a reality. Microbe  2: 474–475.  <ul class="find" data-citedArticleID="1050622"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Whole+genome+transplantation+becomes+a+reality.&amp;auth=&amp;atitle=Whole+genome+transplantation+becomes+a+reality." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole+genome+transplantation+becomes+a+reality." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole+genome+transplantation+becomes+a+reality.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001542-Adkins1" id="pone.0001542-Adkins1"></a>Adkins JN, Mottaz HM, Norbeck AD, Gustin JK, Rue J, et al.  (2006) Analysis of the <em>Salmonella typhimurium</em> proteome through environmental response toward infectious conditions. Mol Cell Proteomics  5: 1450–1461.  <ul class="find" data-citedArticleID="1050516" data-doi="10.1074/mcp.m600139-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m600139-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+Salmonella+typhimurium+proteome+through+environmental+response+toward+infectious+conditions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+Salmonella+typhimurium+proteome+through+environmental+response+toward+infectious+conditions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001542-Ding1" id="pone.0001542-Ding1"></a>Ding YHR, Hixson KK, Giometti CS, Stanley A, Esteve-Nunez A, et al.  (2006) The proteome of dissimilatory metal-reducing microorganism <em>Geobacter sulfurreducens</em> under various growth conditions. BBA-Proteins Proteomics  1764: 1198–1206.  <ul class="find" data-citedArticleID="1050536" data-doi="10.1016/j.bbapap.2006.04.017"><li><a href="http://dx.doi.org/10.1016/j.bbapap.2006.04.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+proteome+of+dissimilatory+metal-reducing+microorganism+Geobacter+sulfurreducens+under+various+growth+conditions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+proteome+of+dissimilatory+metal-reducing+microorganism+Geobacter+sulfurreducens+under+various+growth+conditions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001542-Elias1" id="pone.0001542-Elias1"></a>Elias DA, Monroe ME, Smith RD, Fredrickson JK, Lipton MS (2006) Confirmation of the expression of a large set of conserved hypothetical proteins in <em>Shewanella oneidensis</em> MR-1. J Microbiol Methods  66: 223–233.  <ul class="find" data-citedArticleID="1050538" data-doi="10.1016/j.mimet.2005.11.009"><li><a href="http://dx.doi.org/10.1016/j.mimet.2005.11.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Confirmation+of+the+expression+of+a+large+set+of+conserved+hypothetical+proteins+in+Shewanella+oneidensis+MR-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Confirmation+of+the+expression+of+a+large+set+of+conserved+hypothetical+proteins+in+Shewanella+oneidensis+MR-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001542-Elias2" id="pone.0001542-Elias2"></a>Elias DA, Yang F, Mottaz HM, Beliaev AS, Lipton MS (2007) Enrichment of functional redox reactive proteins and identification by mass spectrometry results in several terminal Fe(III)-reducing candidate proteins in <em>Shewanella oneidensis</em> MR-1. J Microbiol Methods  68: 367–375.  <ul class="find" data-citedArticleID="1050540" data-doi="10.1016/j.mimet.2006.09.023"><li><a href="http://dx.doi.org/10.1016/j.mimet.2006.09.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enrichment+of+functional+redox+reactive+proteins+and+identification+by+mass+spectrometry+results+in+several+terminal+Fe%28III%29-reducing+candidate+proteins+in+Shewanella+oneidensis+MR-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enrichment+of+functional+redox+reactive+proteins+and+identification+by+mass+spectrometry+results+in+several+terminal+Fe%28III%29-reducing+candidate+proteins+in+Shewanella+oneidensis+MR-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001542-Fang2" id="pone.0001542-Fang2"></a>Fang RH, Elias DA, Monroe ME, Shen YF, McIntosh M, et al.  (2006) Differential label-free quantitative proteomic analysis of <em>Shewanella oneidensis</em> cultured under aerobic and suboxic conditions by accurate mass and time tag approach. Mol Cell Proteomics  5: 714–725.  <ul class="find" data-citedArticleID="1050546" data-doi="10.1074/mcp.m500301-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m500301-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+label-free+quantitative+proteomic+analysis+of+Shewanella+oneidensis+cultured+under+aerobic+and+suboxic+conditions+by+accurate+mass+and+time+tag+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+label-free+quantitative+proteomic+analysis+of+Shewanella+oneidensis+cultured+under+aerobic+and+suboxic+conditions+by+accurate+mass+and+time+tag+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001542-Luo1" id="pone.0001542-Luo1"></a>Luo Q, Hixson KK, Callister SJ, Lipton MS, Morris BE, et al.  (2007) Proteome analysis of <em>Desulfovibrio desulfuricans</em> G20 mutants using the accurate mass and time (AMT) tag Approach. J Proteome Res  6: 3042–3053.  <ul class="find" data-citedArticleID="1050582" data-doi="10.1021/pr070127o"><li><a href="http://dx.doi.org/10.1021/pr070127o" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteome+analysis+of+Desulfovibrio+desulfuricans+G20+mutants+using+the+accurate+mass+and+time+%28AMT%29+tag+Approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteome+analysis+of+Desulfovibrio+desulfuricans+G20+mutants+using+the+accurate+mass+and+time+%28AMT%29+tag+Approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001542-Manes1" id="pone.0001542-Manes1"></a>Manes NP, Gustin JK, Rue J, Mottaz HM, Purvine SO, et al.  (2007) Targeted protein degradation by <em>Salmonella</em> under phagosome-mimicking culture conditions investigated using comparative peptidomics. Mol Cell Proteomics  6: 717–727.  <ul class="find" data-citedArticleID="1050584" data-doi="10.1074/mcp.m600282-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m600282-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeted+protein+degradation+by+Salmonella+under+phagosome-mimicking+culture+conditions+investigated+using+comparative+peptidomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeted+protein+degradation+by+Salmonella+under+phagosome-mimicking+culture+conditions+investigated+using+comparative+peptidomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001542-Romine1" id="pone.0001542-Romine1"></a>Romine MF, Elias DA, Monroe ME, Auberry K, Fang RH, et al.  (2004) Validation of <em>Shewanella oneidensis</em> MR-1 small proteins by AMT tag-based proteome analysis. Omics  8: 239–254.  <ul class="find" data-citedArticleID="1050610" data-doi="10.1089/omi.2004.8.239"><li><a href="http://dx.doi.org/10.1089/omi.2004.8.239" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Validation+of+Shewanella+oneidensis+MR-1+small+proteins+by+AMT+tag-based+proteome+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Validation+of+Shewanella+oneidensis+MR-1+small+proteins+by+AMT+tag-based+proteome+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001542-Schmid1" id="pone.0001542-Schmid1"></a>Schmid AK, Lipton MS, Mottaz H, Monroe ME, Smith RD, et al.  (2005) Global whole-cell FTICR mass spectrometric proteomics analysis of the heat shock response in the radioresistant bacterium <em>Deinococcus radiodurans</em>. J Proteome Res  4: 709–718.  <ul class="find" data-citedArticleID="1050614" data-doi="10.1021/pr049815n"><li><a href="http://dx.doi.org/10.1021/pr049815n" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+whole-cell+FTICR+mass+spectrometric+proteomics+analysis+of+the+heat+shock+response+in+the+radioresistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+whole-cell+FTICR+mass+spectrometric+proteomics+analysis+of+the+heat+shock+response+in+the+radioresistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001542-Shi1" id="pone.0001542-Shi1"></a>Shi L, Adkins JN, Coleman JR, Schepmoes AA, Dohnkova A, et al.  (2006) Proteomic analysis of <em>Salmonella enterica</em> serovar Typhimurium isolated from RAW 264.7 macrophages-Identification of a novel protein that contributes to the replication of serovar Typhimurium inside macrophages. J Biol Chem  281: 29131–29140.  <ul class="find" data-citedArticleID="1050616" data-doi="10.1074/jbc.m604640200"><li><a href="http://dx.doi.org/10.1074/jbc.m604640200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteomic+analysis+of+Salmonella+enterica+serovar+Typhimurium+isolated+from+RAW+264.7+macrophages-Identification+of+a+novel+protein+that+contributes+to+the+replication+of+serovar+Typhimurium+inside+macrophages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteomic+analysis+of+Salmonella+enterica+serovar+Typhimurium+isolated+from+RAW+264.7+macrophages-Identification+of+a+novel+protein+that+contributes+to+the+replication+of+serovar+Typhimurium+inside+macrophages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001542-Smith1" id="pone.0001542-Smith1"></a>Smith RD, Anderson GA, Lipton MS, Pasa-Tolic L, Shen YF, et al.  (2002) An accurate mass tag strategy for quantitative and high-throughput proteome measurements. Proteomics  2: 513–523.  <ul class="find" data-citedArticleID="1050620" data-doi="10.1002/1615-9861(200205)2:5<513::aid-prot513&gt;3.0.co;2-w"><li><a href="http://dx.doi.org/10.1002/1615-9861(200205)2:5<513::aid-prot513&gt;3.0.co;2-w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+accurate+mass+tag+strategy+for+quantitative+and+high-throughput+proteome+measurements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+accurate+mass+tag+strategy+for+quantitative+and+high-throughput+proteome+measurements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001542-Yang1" id="pone.0001542-Yang1"></a>Yang F, Bogdanov B, Strittmatter EF, Vilkov AN, Gritsenko M, et al.  (2005) Characterization of purified c-type heme-containing peptides and identification of <em>c</em>-type heme-attachment sites in <em>Shewanella oneidenis</em> cytochromes using mass spectrometry. J Proteome Res  4: 846–854.  <ul class="find" data-citedArticleID="1050628" data-doi="10.1021/pr0497475"><li><a href="http://dx.doi.org/10.1021/pr0497475" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+purified+c-type+heme-containing+peptides+and+identification+of+c-type+heme-attachment+sites+in+Shewanella+oneidenis+cytochromes+using+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+purified+c-type+heme-containing+peptides+and+identification+of+c-type+heme-attachment+sites+in+Shewanella+oneidenis+cytochromes+using+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001542-Zhang1" id="pone.0001542-Zhang1"></a>Zhang WW, Culley DE, Gritsenko MA, Moore RJ, Nie L, et al.  (2006) LC-MS/MS based proteomic analysis and functional inference of hypothetical proteins in <em>Desulfovibrio vulgaris</em>. Biochem Biophys Res Commun  349: 1412–1419.  <ul class="find" data-citedArticleID="1050630" data-doi="10.1016/j.bbrc.2006.09.019"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2006.09.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LC-MS%2FMS+based+proteomic+analysis+and+functional+inference+of+hypothetical+proteins+in+Desulfovibrio+vulgaris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LC-MS%2FMS+based+proteomic+analysis+and+functional+inference+of+hypothetical+proteins+in+Desulfovibrio+vulgaris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001542-Zhang2" id="pone.0001542-Zhang2"></a>Zhang WW, Gritsenko MA, Moore RJ, Culley DE, Nie L, et al.  (2006) A proteomic view of <em>Desulfovibrio vulgaris</em> metabolism as determined by liquid chromatography coupled with tandem mass spectrometry. Proteomics  6: 4286–4299.  <ul class="find" data-citedArticleID="1050632" data-doi="10.1002/pmic.200500930"><li><a href="http://dx.doi.org/10.1002/pmic.200500930" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+proteomic+view+of+Desulfovibrio+vulgaris+metabolism+as+determined+by+liquid+chromatography+coupled+with+tandem+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+proteomic+view+of+Desulfovibrio+vulgaris+metabolism+as+determined+by+liquid+chromatography+coupled+with+tandem+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001542-Ansong1" id="pone.0001542-Ansong1"></a>Ansong C, Yoon H, Norbeck AD, Gustin J, McDermott JE, et al.  (2007) Proteomics analysis of the causative agents of typhoid fever. J Proteome Res: In Press.   <ul class="find" data-citedArticleID="1050520" data-doi="10.1021/pr070434u"><li><a href="http://dx.doi.org/10.1021/pr070434u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteomics+analysis+of+the+causative+agents+of+typhoid+fever." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteomics+analysis+of+the+causative+agents+of+typhoid+fever.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001542-Kiebel1" id="pone.0001542-Kiebel1"></a>Kiebel GR, Auberry KJ, Jaitly N, Clark DA, Monroe ME, et al.  (2006) PRISM: A data management system for high-throughput proteomics. Proteomics  6: 1783–1790.  <ul class="find" data-citedArticleID="1050568" data-doi="10.1002/pmic.200500500"><li><a href="http://dx.doi.org/10.1002/pmic.200500500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PRISM%3A+A+data+management+system+for+high-throughput+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PRISM%3A+A+data+management+system+for+high-throughput+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001542-Monroe1" id="pone.0001542-Monroe1"></a>Monroe ME, Tolic N, Jaitly N, Shaw JL, Adkins JN, et al.  (2007) VIPER: an advanced software package to support high-throughput LC-MS peptide identification. Bioinformatics  23: 2021–2023.  <ul class="find" data-citedArticleID="1050592" data-doi="10.1093/bioinformatics/btm281"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btm281" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VIPER%3A+an+advanced+software+package+to+support+high-throughput+LC-MS+peptide+identification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VIPER%3A+an+advanced+software+package+to+support+high-throughput+LC-MS+peptide+identification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001542-Keller1" id="pone.0001542-Keller1"></a>Keller A, Nesvizhskii AI, Kolker E, Aebersold R (2002) Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search. Anal Chem  74: 5383–5392.  <ul class="find" data-citedArticleID="1050566" data-doi="10.1021/ac025747h"><li><a href="http://dx.doi.org/10.1021/ac025747h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+statistical+model+to+estimate+the+accuracy+of+peptide+identifications+made+by+MS%2FMS+and+database+search." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+statistical+model+to+estimate+the+accuracy+of+peptide+identifications+made+by+MS%2FMS+and+database+search.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001542-Qian1" id="pone.0001542-Qian1"></a>Qian WJ, Liu T, Monroe ME, Strittmatter EF, Jacobs JM, et al.  (2005) Probability-based evaluation of peptide and protein identifications from tandem mass spectrometry and SEQUEST analysis: The human proteome. J Proteome Res  4: 53–62.  <ul class="find" data-citedArticleID="1050600" data-doi="10.1021/pr0498638"><li><a href="http://dx.doi.org/10.1021/pr0498638" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probability-based+evaluation+of+peptide+and+protein+identifications+from+tandem+mass+spectrometry+and+SEQUEST+analysis%3A+The+human+proteome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probability-based+evaluation+of+peptide+and+protein+identifications+from+tandem+mass+spectrometry+and+SEQUEST+analysis%3A+The+human+proteome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.XML" value="98738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.PDF" value="420689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g001.PNG_L" value="67496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g001.PNG_M" value="26590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g001.PNG_S" value="8538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g001.TIF" value="129020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g001.PNG_I" value="8819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g002.PNG_L" value="972449"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g002.PNG_M" value="214747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g002.PNG_S" value="15978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g002.TIF" value="2402208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g002.PNG_I" value="74090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g003.PNG_L" value="831985"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g003.PNG_M" value="95805"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g003.PNG_S" value="17832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g003.TIF" value="1889028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g003.PNG_I" value="108726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g004.PNG_L" value="404067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g004.PNG_M" value="81697"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g004.PNG_S" value="12532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g004.TIF" value="769438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g004.PNG_I" value="48176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g005.PNG_L" value="331422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g005.PNG_M" value="83238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g005.PNG_S" value="10922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g005.TIF" value="1057532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.g005.PNG_I" value="29322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.e001.PNG" value="8208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.e001.TIF" value="24052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.s001.PDF" value="227189"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.s002.PDF" value="138345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.s003.PDF" value="115474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001542.s004.PDF" value="21204"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001542&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001542" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001542&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001542&volume=&issue=&title=Comparative Bacterial Proteomics: Analysis of the Core Genome Concept&author_name=Stephen%20J.%20Callister%2C%20Lee%20Ann%20McCue%2C%20Joshua%20E.%20Turse%2C%20Matthew%20E.%20Monroe%2C%20Kenneth%20J.%20Auberry%2C%20Richard%20D.%20Smith%2C%20Joshua%20N.%20Adkins%2C%20Mary%20S.%20Lipton&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001542" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001542" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001542" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001542&amp;t=Comparative%20Bacterial%20Proteomics%3A%20Analysis%20of%20the%20Core%20Genome%20Concept" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001542&title=Comparative%20Bacterial%20Proteomics%3A%20Analysis%20of%20the%20Core%20Genome%20Concept&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001542&amp;title=Comparative%20Bacterial%20Proteomics%3A%20Analysis%20of%20the%20Core%20Genome%20Concept" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001542" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001542&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Comparative Bacterial Proteomics: Analysis of the Core Genome Concept';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001542';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Comparative%20Bacterial%20Proteomics%3A%20Analysis%20of%20the%20Core%20Genome%20Concept http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001542" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001542" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="25910"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Comparative+genomics%22" title="Search for articles in the subject area:'Comparative genomics'"><div class="flagText">Comparative genomics</div></a>
              <div data-categoryid="32937" data-articleid="25910"
                   data-categoryname="Comparative genomics"
                   class="flagImage" title="Flag 'Comparative genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25910"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="25910"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomics%22" title="Search for articles in the subject area:'Genomics'"><div class="flagText">Genomics</div></a>
              <div data-categoryid="18221" data-articleid="25910"
                   data-categoryname="Genomics"
                   class="flagImage" title="Flag 'Genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Peptides%22" title="Search for articles in the subject area:'Peptides'"><div class="flagText">Peptides</div></a>
              <div data-categoryid="17847" data-articleid="25910"
                   data-categoryname="Peptides"
                   class="flagImage" title="Flag 'Peptides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomes%22" title="Search for articles in the subject area:'Proteomes'"><div class="flagText">Proteomes</div></a>
              <div data-categoryid="18649" data-articleid="25910"
                   data-categoryname="Proteomes"
                   class="flagImage" title="Flag 'Proteomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomic+databases%22" title="Search for articles in the subject area:'Proteomic databases'"><div class="flagText">Proteomic databases</div></a>
              <div data-categoryid="43483" data-articleid="25910"
                   data-categoryname="Proteomic databases"
                   class="flagImage" title="Flag 'Proteomic databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=595'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6347'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6451&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
