

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Comparative Analysis of Acinetobacters: Three Genomes for Three Lifestyles</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001805"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001805" />

    <meta name="citation_title" content="Comparative Analysis of Acinetobacters: Three Genomes for Three Lifestyles"/>
    <meta itemprop="name" content="Comparative Analysis of Acinetobacters: Three Genomes for Three Lifestyles"/>

      <meta name="citation_author" content="David Vallenet"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Patrice Nordmann"/>
            <meta name="citation_author_institution" content="Service de Bactériologie-Virologie, Hôpital de Bicêtre, Assistance Publique/Hôpitaux de Paris, Faculté de Médecine Paris-Sud, Université Paris XI, Kremlin-Bicêtre, France"/>
      <meta name="citation_author" content="Valérie Barbe"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Laurent Poirel"/>
            <meta name="citation_author_institution" content="Service de Bactériologie-Virologie, Hôpital de Bicêtre, Assistance Publique/Hôpitaux de Paris, Faculté de Médecine Paris-Sud, Université Paris XI, Kremlin-Bicêtre, France"/>
      <meta name="citation_author" content="Sophie Mangenot"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Elodie Bataille"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Carole Dossat"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Shahinaz Gas"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Annett Kreimeyer"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Patricia Lenoble"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Sophie Oztas"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Julie Poulain"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Béatrice Segurens"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Catherine Robert"/>
            <meta name="citation_author_institution" content="Unité des Rickettsies, CNRS UMR6020, Faculté de Médecine, Université de la Méditerranée, Marseille, France"/>
      <meta name="citation_author" content="Chantal Abergel"/>
            <meta name="citation_author_institution" content="Information Génomique et Structurale, CNRS UPR2589, IBSM, Marseille, France"/>
      <meta name="citation_author" content="Jean-Michel Claverie"/>
            <meta name="citation_author_institution" content="Information Génomique et Structurale, CNRS UPR2589, IBSM, Marseille, France"/>
      <meta name="citation_author" content="Didier Raoult"/>
            <meta name="citation_author_institution" content="Unité des Rickettsies, CNRS UMR6020, Faculté de Médecine, Université de la Méditerranée, Marseille, France"/>
      <meta name="citation_author" content="Claudine Médigue"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Jean Weissenbach"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>
      <meta name="citation_author" content="Stéphane Cruveiller"/>
            <meta name="citation_author_institution" content="Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France"/>

    <meta name="citation_date" content="2008/3/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001805.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1805"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The epidemiology and control of Acinetobacter baumannii in health care facilities.; citation_author=PE Fournier; citation_author=H Richet; citation_journal_title=Clin Infect Dis; citation_volume=42; citation_number=1; citation_pages=692-699; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Acinetobacter baumannii in human body louse.; citation_author=B La Scola; citation_author=D Raoult; citation_journal_title=Emerg Infect Dis; citation_volume=10; citation_number=2; citation_pages=1671-1673; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Taxonomy of the genus Acinetobacter with the recognition of Acinetobacter baumannii sp. nov., Acinetobacter haemolyticus sp. nov., Acinetobacter johnsonii sp. nov., and Acinetobacter junii sp. nov. and emended descriptions of Acinetobacter calcoaceticus and Acinetobacter lwoffi.; citation_author=PJM Bouvet; citation_author=PA Grimont; citation_journal_title=Int J Syst Bacteriol; citation_volume=36; citation_number=3; citation_pages=228-240; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Identification and biotyping of clinical isolates of Acinetobacter.; citation_author=PJ Bouvet; citation_author=PA Grimont; citation_journal_title=Ann Inst Pasteur Microbiol; citation_volume=138; citation_number=4; citation_pages=569-578; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Identification of Acinetobacter genomic species by amplified ribosomal DNA restriction analysis.; citation_author=M Vaneechoutte; citation_author=L Dijkshoorn; citation_author=I Tjernberg; citation_author=A Elaichouni; citation_author=P de Vos; citation_journal_title=J Clin Microbiol; citation_volume=33; citation_number=5; citation_pages=11-15; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Discrimination of Acinetobacter genomic species by AFLP fingerprinting.; citation_author=P Janssen; citation_author=K Maquelin; citation_author=R Coopman; citation_author=I Tjernberg; citation_author=P Bouvet; citation_journal_title=Int J Syst Bacteriol; citation_volume=47; citation_number=6; citation_pages=1179-1187; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Reliability of phenotypic tests for identification of Acinetobacter species.; citation_author=P Gerner-Smidt; citation_author=I Tjernberg; citation_author=J Ursing; citation_journal_title=J Clin Microbiol; citation_volume=29; citation_number=7; citation_pages=277-282; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Acinetobacter: environmental and biotechnological applications.; citation_author=D Abdel-El-Haleem; citation_journal_title=African Journal Of Biotechnology; citation_volume=2; citation_number=8; citation_pages=71-74; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Unique features revealed by the genome sequence of Acinetobacter sp. ADP1, a versatile and naturally transformation competent bacterium.; citation_author=V Barbe; citation_author=D Vallenet; citation_author=N Fonknechten; citation_author=A Kreimeyer; citation_author=S Oztas; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=9; citation_pages=5766-5779; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Comparative genomics of multidrug resistance in Acinetobacter baumannii.; citation_author=PE Fournier; citation_author=D Vallenet; citation_author=V Barbe; citation_author=S Audic; citation_author=H Ogata; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=10; citation_pages=e7; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of Acinetobacter isolates in the A. calcoaceticus-A. baumannii complex by restriction analysis of the 16S-23S rRNA intergenic-spacer sequences.; citation_author=L Dolzani; citation_author=E Tonin; citation_author=C Lagatolla; citation_author=L Prandin; citation_author=C Monti-Bragadin; citation_journal_title=J Clin Microbiol; citation_volume=33; citation_number=11; citation_pages=1108-1113; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Estimation of ribosomal RNA operon (rrn) copy number in Acinetobacter isolates and potential of patterns of rrn operon-containing fragments for typing strains of members of this genus.; citation_author=C Maslunka; citation_author=E Carr; citation_author=V Gurtler; citation_author=P Kampfer; citation_author=R Seviour; citation_journal_title=Syst Appl Microbiol; citation_volume=29; citation_number=12; citation_pages=216-228; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cephalosporinase over-expression resulting from insertion of ISAba1 in Acinetobacter baumannii.; citation_author=C Heritier; citation_author=L Poirel; citation_author=P Nordmann; citation_journal_title=Clin Microbiol Infect; citation_volume=12; citation_number=13; citation_pages=123-130; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Is IS(ABA-1) customized for Acinetobacter?; citation_author=H Segal; citation_author=S Garny; citation_author=BG Elisha; citation_journal_title=FEMS Microbiol Lett; citation_volume=243; citation_number=14; citation_pages=425-429; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The natural transformation of the soil bacteria Pseudomonas stutzeri and Acinetobacter sp. by transgenic plant DNA strictly depends on homologous sequences in the recipient cells.; citation_author=J de Vries; citation_author=P Meier; citation_author=W Wackernagel; citation_journal_title=FEMS Microbiol Lett; citation_volume=195; citation_number=15; citation_pages=211-215; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Type IV pili-related natural transformation systems: DNA transport in mesophilic and thermophilic bacteria.; citation_author=B Averhoff; citation_author=A Friedrich; citation_journal_title=Arch Microbiol; citation_volume=180; citation_number=16; citation_pages=385-393; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Extracellular DNA required for bacterial biofilm formation.; citation_author=CB Whitchurch; citation_author=T Tolker-Nielsen; citation_author=PC Ragas; citation_author=JS Mattick; citation_journal_title=Science; citation_volume=295; citation_number=17; citation_pages=1487; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Survival of Acinetobacter baumannii on dry surfaces.; citation_author=C Wendt; citation_author=B Dietze; citation_author=E Dietz; citation_author=H Ruden; citation_journal_title=J Clin Microbiol; citation_volume=35; citation_number=18; citation_pages=1394-1397; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Biofilm formation by Acinetobacter baumannii.; citation_author=R Vidal; citation_author=M Dominguez; citation_author=H Urrutia; citation_author=H Bello; citation_author=G Gonzalez; citation_journal_title=Microbios; citation_volume=86; citation_number=19; citation_pages=49-58; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Bacterial biofilms: a common cause of persistent infections.; citation_author=JW Costerton; citation_author=PS Stewart; citation_author=EP Greenberg; citation_journal_title=Science; citation_volume=284; citation_number=20; citation_pages=1318-1322; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Attachment to and biofilm formation on abiotic surfaces by Acinetobacter baumannii: involvement of a novel chaperone-usher pili assembly system.; citation_author=AP Tomaras; citation_author=CW Dorsey; citation_author=RE Edelmann; citation_author=LA Actis; citation_journal_title=Microbiology; citation_volume=149; citation_number=21; citation_pages=3473-3484; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The thin pili of Acinetobacter sp. strain BD413 mediate adhesion to biotic and abiotic surfaces.; citation_author=O Gohl; citation_author=A Friedrich; citation_author=M Hoppert; citation_author=B Averhoff; citation_journal_title=Appl Environ Microbiol; citation_volume=72; citation_number=22; citation_pages=1394-1401; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The global carbon metabolism regulator Crc is a component of a signal transduction pathway required for biofilm development by Pseudomonas aeruginosa.; citation_author=GA O'Toole; citation_author=KA Gibbs; citation_author=PW Hager; citation_author=PV Phibbs; citation_author=R Kolter; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=23; citation_pages=425-431; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A novel two-component system controls the expression of Pseudomonas aeruginosa fimbrial cup genes.; citation_author=HD Kulasekara; citation_author=I Ventre; citation_author=BR Kulasekara; citation_author=A Lazdunski; citation_author=A Filloux; citation_journal_title=Mol Microbiol; citation_volume=55; citation_number=24; citation_pages=368-380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The involvement of cell-to-cell signals in the development of a bacterial biofilm.; citation_author=DG Davies; citation_author=MR Parsek; citation_author=JP Pearson; citation_author=BH Iglewski; citation_author=JW Costerton; citation_journal_title=Science; citation_volume=280; citation_number=25; citation_pages=295-298; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Phenazine antibiotic biosynthesis in Pseudomonas aureofaciens 30-84 is regulated by PhzR in response to cell density.; citation_author=LS Pierson; citation_author=VD Keppenne; citation_author=DW Wood; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=26; citation_pages=3966-3974; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Agrobacterium conjugation and gene regulation by N-acyl-L-homoserine lactones.; citation_author=L Zhang; citation_author=PJ Murphy; citation_author=A Kerr; citation_author=ME Tate; citation_journal_title=Nature; citation_volume=362; citation_number=27; citation_pages=446-448; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Convergence of hormones and autoinducers at the host/pathogen interface.; citation_author=KP Rumbaugh; citation_journal_title=Anal Bioanal Chem; citation_number=28; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Silencing the majority.; citation_author=NA Whitehead; citation_author=M Welch; citation_author=GP Salmond; citation_journal_title=Nat Biotechnol; citation_volume=19; citation_number=29; citation_pages=735-736; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=AiiA, an enzyme that inactivates the acylhomoserine lactone quorum-sensing signal and attenuates the virulence of Erwinia carotovora.; citation_author=YH Dong; citation_author=JL Xu; citation_author=XZ Li; citation_author=LH Zhang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=30; citation_pages=3526-3531; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Degradation of acyl-homoserine lactone molecules by Acinetobacter sp. strain C1010.; citation_author=BR Kang; citation_author=JH Lee; citation_author=SJ Ko; citation_author=YH Lee; citation_author=JS Cha; citation_journal_title=Can J Microbiol; citation_volume=50; citation_number=31; citation_pages=935-941; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A study of the Moraxella group. II. Oxidative-negative species (genus Acinetobacter).; citation_author=P Baumann; citation_author=M Doudoroff; citation_author=RY Stanier; citation_journal_title=J Bacteriol; citation_volume=95; citation_number=32; citation_pages=1520-1541; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Pathways for biosynthesis of a bacterial capsular polysaccharide. I. Carbohydrate metabolism and terminal oxidation mechanisms of a capsuleproducing coccus.; citation_author=WH Taylor; citation_author=E Juni; citation_journal_title=J Bacteriol; citation_volume=81; citation_number=33; citation_pages=694-703; citation_date=1961; " />
      <meta name="citation_reference" content="citation_title=Nucleotide sequence of the gene coding for quinoprotein glucose dehydrogenase from Acinetobacter calcoaceticus.; citation_author=AM Cleton-Jansen; citation_author=N Goosen; citation_author=G Odle; citation_author=P van de Putte; citation_journal_title=Nucleic Acids Res; citation_volume=16; citation_number=34; citation_pages=6228; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Cloning, characterization and DNA sequencing of the gene encoding the Mr 50,000 quinoprotein glucose dehydrogenase from Acinetobacter calcoaceticus.; citation_author=AM Cleton-Jansen; citation_author=N Goosen; citation_author=K Vink; citation_author=P van de Putte; citation_journal_title=Mol Gen Genet; citation_volume=217; citation_number=35; citation_pages=430-436; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=A complete collection of single-gene deletion mutants of Acinetobacter baylyi ADP1.; citation_author=V de Berardinis; citation_author=D Vallenet; citation_author=V Castelli; citation_author=M Besnard; citation_author=A Pinet; citation_journal_title=Mol Syst Biol; citation_volume=0; citation_number=36; citation_pages=0-0; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=The cobalamin (coenzyme B12) biosynthetic genes of Escherichia coli.; citation_author=JG Lawrence; citation_author=JR Roth; citation_journal_title=J Bacteriol; citation_volume=177; citation_number=37; citation_pages=6371-6380; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Opportunities for genetic investigation afforded by Acinetobacter baylyi, a nutritionally versatile bacterial species that is highly competent for natural transformation.; citation_author=DM Young; citation_author=D Parke; citation_author=LN Ornston; citation_journal_title=Annu Rev Microbiol; citation_volume=59; citation_number=38; citation_pages=519-551; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genes for chlorogenate and hydroxycinnamate catabolism (hca) are linked to functionally related genes in the dca-pca-qui-pob-hca chromosomal cluster of Acinetobacter sp. strain ADP1.; citation_author=MA Smith; citation_author=VB Weaver; citation_author=DM Young; citation_author=LN Ornston; citation_journal_title=Appl Environ Microbiol; citation_volume=69; citation_number=39; citation_pages=524-532; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transcriptional organization of genes for protocatechuate and quinate degradation from Acinetobacter sp. strain ADP1.; citation_author=S Dal; citation_author=G Trautwein; citation_author=U Gerischer; citation_journal_title=Appl Environ Microbiol; citation_volume=71; citation_number=40; citation_pages=1025-1034; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic analysis of a chromosomal region containing vanA and vanB, genes required for conversion of either ferulate or vanillate to protocatechuate in Acinetobacter.; citation_author=A Segura; citation_author=PV Bunz; citation_author=DA D'Argenio; citation_author=LN Ornston; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=41; citation_pages=3494-3504; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Osmotic control of glycine betaine biosynthesis and degradation in Rhizobium meliloti.; citation_author=LT Smith; citation_author=JA Pocard; citation_author=T Bernard; citation_author=D Le Rudulier; citation_journal_title=J Bacteriol; citation_volume=170; citation_number=42; citation_pages=3142-3149; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Identification of sulfate starvation-regulated genes in Escherichia coli: a gene cluster involved in the utilization of taurine as a sulfur source.; citation_author=JR van der Ploeg; citation_author=MA Weiss; citation_author=E Saller; citation_author=H Nashimoto; citation_author=N Saito; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=43; citation_pages=5438-5446; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Taurine: biological update.; citation_author=CE Wright; citation_author=HH Tallan; citation_author=YY Lin; citation_author=GE Gaull; citation_journal_title=Annu Rev Biochem; citation_volume=55; citation_number=44; citation_pages=427-453; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=The PaaX repressor, a link between penicillin G acylase and the phenylacetyl-coenzyme A catabolon of Escherichia coli W.; citation_author=B Galan; citation_author=JL Garcia; citation_author=MA Prieto; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=45; citation_pages=2215-2220; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Carbon regulation and the role in nature of the Escherichia coli penicillin acylase (pac) gene.; citation_author=E Merino; citation_author=P Balbas; citation_author=F Recillas; citation_author=B Becerril; citation_author=F Valle; citation_journal_title=Mol Microbiol; citation_volume=6; citation_number=46; citation_pages=2175-2182; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Role of iron in bacterial infection.; citation_author=JJ Bullen; citation_author=HJ Rogers; citation_author=E Griffiths; citation_journal_title=Curr Top Microbiol Immunol; citation_volume=80; citation_number=47; citation_pages=1-35; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=The siderophore-mediated iron acquisition systems of Acinetobacter baumannii ATCC 19606 and Vibrio anguillarum 775 are structurally and functionally related.; citation_author=CW Dorsey; citation_author=AP Tomaras; citation_author=PL Connerly; citation_author=ME Tolmasky; citation_author=JH Crosa; citation_journal_title=Microbiology; citation_volume=150; citation_number=48; citation_pages=3657-3667; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complete sequence of virulence plasmid pJM1 from the marine fish pathogen Vibrio anguillarum strain 775.; citation_author=M Di Lorenzo; citation_author=M Stork; citation_author=ME Tolmasky; citation_author=LA Actis; citation_author=D Farrell; citation_journal_title=J Bacteriol; citation_volume=185; citation_number=49; citation_pages=5822-5830; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A histidine decarboxylase gene encoded by the Vibrio anguillarum plasmid pJM1 is essential for virulence: histamine is a precursor in the biosynthesis of anguibactin.; citation_author=ME Tolmasky; citation_author=LA Actis; citation_author=JH Crosa; citation_journal_title=Mol Microbiol; citation_volume=15; citation_number=50; citation_pages=87-95; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Comparison of differential plating media and two chromatography techniques for the detection of histamine production in bacteria.; citation_author=LA Actis; citation_author=JC Smoot; citation_author=CE Barancin; citation_author=RH Findlay; citation_journal_title=J Microbiol Methods; citation_volume=39; citation_number=51; citation_pages=79-90; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Ferric reductases or flavin reductases?; citation_author=M Fontecave; citation_author=J Coves; citation_author=JL Pierre; citation_journal_title=Biometals; citation_volume=7; citation_number=52; citation_pages=3-8; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Experimental infection of human body lice with Acinetobacter baumannii.; citation_author=L Houhamdi; citation_author=D Raoult; citation_journal_title=Am J Trop Med Hyg; citation_volume=74; citation_number=53; citation_pages=526-531; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=BACTERIAL IRON SOURCES: From Siderophores to Hemophores.; citation_author=C Wandersman; citation_author=P Delepelaire; citation_journal_title=Annual Review of Microbiology; citation_volume=58; citation_number=54; citation_pages=611-647; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bacterial iron homeostasis.; citation_author=SC Andrews; citation_author=AK Robinson; citation_author=F Rodriguez-Quinones; citation_journal_title=FEMS Microbiol Rev; citation_volume=27; citation_number=55; citation_pages=215-237; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The overlapping angB and angG genes are encoded within the trans-acting factor region of the virulence plasmid in Vibrio anguillarum: essential role in siderophore biosynthesis.; citation_author=TJ Welch; citation_author=S Chai; citation_author=JH Crosa; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=56; citation_pages=6762-6773; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Plasmid-mediated iron uptake and virulence in Vibrio anguillarum.; citation_author=M Stork; citation_author=M Di Lorenzo; citation_author=TJ Welch; citation_author=LM Crosa; citation_author=JH Crosa; citation_journal_title=Plasmid; citation_volume=48; citation_number=57; citation_pages=222-228; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Outer membrane proteins: key players for bacterial adaptation in host niches.; citation_author=J Lin; citation_author=S Huang; citation_author=Q Zhang; citation_journal_title=Microbes Infect; citation_volume=4; citation_number=58; citation_pages=325-331; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Acinetobacter outer membrane protein A (OmpA) is a secreted emulsifier.; citation_author=G Walzer; citation_author=E Rosenberg; citation_author=EZ Ron; citation_journal_title=Environ Microbiol; citation_volume=8; citation_number=59; citation_pages=1026-1032; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Outer membrane protein 38 of Acinetobacter baumannii localizes to the mitochondria and induces apoptosis of epithelial cells.; citation_author=CH Choi; citation_author=EY Lee; citation_author=YC Lee; citation_author=TI Park; citation_author=HJ Kim; citation_journal_title=Cell Microbiol; citation_volume=7; citation_number=60; citation_pages=1127-1138; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An OmpA-like protein from Acinetobacter spp. stimulates gastrin and interleukin-8 promoters.; citation_author=E Ofori-Darko; citation_author=Y Zavros; citation_author=G Rieder; citation_author=SA Tarle; citation_author=M Van Antwerp; citation_journal_title=Infect Immun; citation_volume=68; citation_number=61; citation_pages=3657-3666; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Bacterial phospholipases C.; citation_author=RW Titball; citation_journal_title=Microbiol Rev; citation_volume=57; citation_number=62; citation_pages=347-366; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Expression of phospholipase D, the major virulence factor of Corynebacterium pseudotuberculosis, is regulated by multiple environmental factors and plays a role in macrophage death.; citation_author=SC McKean; citation_author=JK Davies; citation_author=RJ Moore; citation_journal_title=Microbiology; citation_volume=153; citation_number=63; citation_pages=2203-2211; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=New insights into Acinetobacter baumannii pathogenesis revealed by high-density pyrosequencing and transposon mutagenesis.; citation_author=MG Smith; citation_author=TA Gianoulis; citation_author=S Pukatzki; citation_author=JJ Mekalanos; citation_author=LN Ornston; citation_journal_title=Genes Dev; citation_volume=21; citation_number=64; citation_pages=601-614; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Microbial synergy via an ethanol-triggered pathway.; citation_author=MG Smith; citation_author=SG Des Etages; citation_author=M Snyder; citation_journal_title=Mol Cell Biol; citation_volume=24; citation_number=65; citation_pages=3874-3884; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=ISfinder: the reference centre for bacterial insertion sequences.; citation_author=P Siguier; citation_author=J Perochon; citation_author=L Lestrade; citation_author=J Mahillon; citation_author=M Chandler; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=66; citation_pages=D32-36; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Repseek, a tool to retrieve approximate repeats from large DNA sequences.; citation_author=G Achaz; citation_author=F Boyer; citation_author=EP Rocha; citation_author=A Viari; citation_author=E Coissac; citation_journal_title=Bioinformatics; citation_volume=23; citation_number=67; citation_pages=119-121; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=AMIGene: Annotation of MIcrobial Genes.; citation_author=S Bocs; citation_author=S Cruveiller; citation_author=D Vallenet; citation_author=G Nuel; citation_author=C Medigue; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=68; citation_pages=3723-3726; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The use of gene clusters to infer functional coupling.; citation_author=R Overbeek; citation_author=M Fonstein; citation_author=M D'Souza; citation_author=GD Pusch; citation_author=N Maltsev; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=69; citation_pages=2896-2901; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Syntons, metabolons and interactons: an exact graph-theoretical approach for exploring neighbourhood between genomic and functional data.; citation_author=F Boyer; citation_author=A Morgat; citation_author=L Labarre; citation_author=J Pothier; citation_author=A Viari; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=70; citation_pages=4209-4215; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MaGe: a microbial genome annotation system supported by synteny results.; citation_author=D Vallenet; citation_author=L Labarre; citation_author=Z Rouy; citation_author=V Barbe; citation_author=S Bocs; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=71; citation_pages=53-65; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput.; citation_author=RC Edgar; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=72; citation_pages=1792-1797; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Syst Biol; citation_volume=52; citation_number=73; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Jalview Java alignment editor.; citation_author=M Clamp; citation_author=J Cuff; citation_author=SM Searle; citation_author=GJ Barton; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=74; citation_pages=426-427; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001805" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Comparative Analysis of Acinetobacters: Three Genomes for Three Lifestyles"/>
    <meta name="twitter:description" content="Acinetobacter baumannii is the source of numerous nosocomial infections in humans and therefore deserves close attention as multidrug or even pandrug resistant strains are increasingly being identified worldwide. Here we report the comparison of two newly sequenced genomes of A. baumannii. The human isolate A. baumannii AYE is multidrug resistant whereas strain SDF, which was isolated from body lice, is antibiotic susceptible. As reference for comparison in this analysis, the genome of the soil-living bacterium A. baylyi strain ADP1 was used. The most interesting dissimilarities we observed were that i) whereas strain AYE and A. baylyi genomes harbored very few Insertion Sequence elements which could promote expression of downstream genes, strain SDF sequence contains several hundred of them that have played a crucial role in its genome reduction (gene disruptions and simple DNA loss); ii) strain SDF has low catabolic capacities compared to strain AYE. Interestingly, the latter has even higher catabolic capacities than A. baylyi which has already been reported as a very nutritionally versatile organism. This metabolic performance could explain the persistence of A. baumannii nosocomial strains in environments where nutrients are scarce; iii) several processes known to play a key role during host infection (biofilm formation, iron uptake, quorum sensing, virulence factors) were either different or absent, the best example of which is iron uptake. Indeed, strain AYE and A. baylyi use siderophore-based systems to scavenge iron from the environment whereas strain SDF uses an alternate system similar to the Haem Acquisition System (HAS). Taken together, all these observations suggest that the genome contents of the 3 Acinetobacters compared are partly shaped by life in distinct ecological niches: human (and more largely hospital environment), louse, soil."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001805.g003"/>

  <meta property="og:title" content="Comparative Analysis of Acinetobacters: Three Genomes for Three Lifestyles" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001805" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=55'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9451'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4421&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001805">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001805" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001805&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Comparative Analysis of Acinetobacters: Three Genomes for Three Lifestyles
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Vallenet
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:vallenet@genoscope.cns.fr">vallenet@genoscope.cns.fr</a> (DV); <a href="mailto:scruveil@genoscope.cns.fr">scruveil@genoscope.cns.fr</a> (SC)</p>

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patrice Nordmann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Service de Bactériologie-Virologie, Hôpital de Bicêtre, Assistance Publique/Hôpitaux de Paris, Faculté de Médecine Paris-Sud, Université Paris XI, Kremlin-Bicêtre, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Valérie Barbe, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laurent Poirel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Service de Bactériologie-Virologie, Hôpital de Bicêtre, Assistance Publique/Hôpitaux de Paris, Faculté de Médecine Paris-Sud, Université Paris XI, Kremlin-Bicêtre, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sophie Mangenot, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elodie Bataille, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carole Dossat, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shahinaz Gas, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Annett Kreimeyer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patricia Lenoble, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sophie Oztas, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Julie Poulain, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Béatrice Segurens, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Catherine Robert, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité des Rickettsies, CNRS UMR6020, Faculté de Médecine, Université de la Méditerranée, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chantal Abergel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Information Génomique et Structurale, CNRS UPR2589, IBSM, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean-Michel Claverie, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Information Génomique et Structurale, CNRS UPR2589, IBSM, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Didier Raoult, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité des Rickettsies, CNRS UMR6020, Faculté de Médecine, Université de la Méditerranée, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claudine Médigue, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean Weissenbach, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stéphane Cruveiller
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:vallenet@genoscope.cns.fr">vallenet@genoscope.cns.fr</a> (DV); <a href="mailto:scruveil@genoscope.cns.fr">scruveil@genoscope.cns.fr</a> (SC)</p>

                <p>Affiliation:
                  Génomique Métabolique, CNRS UMR8030, CEA–Institut de Génomique-Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 19, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001805</li>
      <li>Featured in <a href="http://www.ploscollections.org">PLOS Collections</a></li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001805" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001805" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001805" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001805" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001805-t001" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001805-g001" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001805-g002" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001805-g003" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001805-t002" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001805-t003" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001805">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001805" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p><em>Acinetobacter baumannii</em> is the source of numerous nosocomial infections in humans and therefore deserves close attention as multidrug or even pandrug resistant strains are increasingly being identified worldwide. Here we report the comparison of two newly sequenced genomes of <em>A. baumannii</em>. The human isolate <em>A</em>. <em>baumannii</em> AYE is multidrug resistant whereas strain SDF, which was isolated from body lice, is antibiotic susceptible. As reference for comparison in this analysis, the genome of the soil-living bacterium <em>A. baylyi</em> strain ADP1 was used. The most interesting dissimilarities we observed were that i) whereas strain AYE and <em>A. baylyi</em> genomes harbored very few Insertion Sequence elements which could promote expression of downstream genes, strain SDF sequence contains several hundred of them that have played a crucial role in its genome reduction (gene disruptions and simple DNA loss); ii) strain SDF has low catabolic capacities compared to strain AYE. Interestingly, the latter has even higher catabolic capacities than <em>A. baylyi</em> which has already been reported as a very nutritionally versatile organism. This metabolic performance could explain the persistence of <em>A. baumannii</em> nosocomial strains in environments where nutrients are scarce; <em>iii</em>) several processes known to play a key role during host infection (biofilm formation, iron uptake, quorum sensing, virulence factors) were either different or absent, the best example of which is iron uptake. Indeed, strain AYE and <em>A. baylyi</em> use siderophore-based systems to scavenge iron from the environment whereas strain SDF uses an alternate system similar to the Haem Acquisition System (HAS). Taken together, all these observations suggest that the genome contents of the 3 Acinetobacters compared are partly shaped by life in distinct ecological niches: human (and more largely hospital environment), louse, soil.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Vallenet D, Nordmann P, Barbe V, Poirel L, Mangenot S, et al.  (2008) Comparative Analysis of Acinetobacters: Three Genomes for Three Lifestyles. PLoS ONE 3(3):
          e1805.
            doi:10.1371/journal.pone.0001805</p><p><strong>Academic Editor: </strong>Neil Hall, University of Liverpool, United Kingdom</p><p><strong>Received:</strong> September 20, 2007; <strong>Accepted:</strong> February 9, 2008; <strong>Published:</strong> March 19, 2008</p><p><strong>Copyright:</strong> © 2008 Vallenet et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no support or funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Although low-grade pathogens in humans, human-adapted <em>Acinetobacter</em> species mainly belong to the <em>A. baumannii-A. calcoaceticus complex</em> are of growing interest due to increased incidence of multidrug resistance (MDR) <a href="#pone.0001805-Fournier1">[1]</a>. <em>A. baumannii</em> strains are isolated in up to 1% of nosocomial infections mostly from immunocompromised patients hospitalized in intensive care units. Although <em>A. baumannii</em> isolates are commonly found in clinical environment, Acinetobacters are mostly free-living saprophytes ubiquitously found in nature (soil, water, sewage). <em>Acinetobacter</em> species have been also identified in small-size living organisms (body lice, fleas and ticks) that are potential vectors for infection transmission <a href="#pone.0001805-LaScola1">[2]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The genus <em>Acinetobacter</em> consists of strictly aerobic, Gram-negative coccobacillary rods, non-motile, catalase positive, oxidase negative and growing at 20°–30°C on usual laboratory culture media. g-proteobacteria classified as members of the genus <em>Acinetobacter</em> have a long history of taxonomic changes moving from the family <em>Neisseriaceae</em> to the family <em>Moraxellaceae</em>. Various methods (DNA-DNA hybridization <a href="#pone.0001805-Bouvet1">[3]</a>, biotyping <a href="#pone.0001805-Bouvet2">[4]</a>, Amplified Ribosomal DNA Restriction Analysis <a href="#pone.0001805-Vaneechoutte1">[5]</a>, AFLP <a href="#pone.0001805-Janssen1">[6]</a>) now make it possible to distinguish 32 different species within the genus <em>Acinetobacter</em>, 17 of which have officially been given names. Nevertheless, unambiguous species identification may be difficult in particular for some genetically closely related species. Standard tests indeed fail to distinguish <em>A. baumannii</em>, <em>A. calcoaceticus</em> and the unnamed genomic species 3 and 13TU (the <em>A. calcoaceticus-A. baumannii</em> complex <a href="#pone.0001805-GernerSmidt1">[7]</a>).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Natural competence as well as high metabolic capacities have been reported in several <em>Acinetobacter</em> species making those species very attractive for environmental and biotechnological use <a href="#pone.0001805-AbdelElHaleem1">[8]</a>. For example, <em>Acinetobacter baylyi</em> ADP1 is highly competent and may grow on a large variety of compounds <a href="#pone.0001805-Barbe1">[9]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Two <em>Acinetobacter baumannii</em> strains (AYE and SDF) were initially sequenced in a clinical/medical context. Since the primary goal of the project was to identify the complete repertoire of genes involved in resistance to various antibiotics <a href="#pone.0001805-Fournier2">[10]</a>, much of our effort was concentrated on a resistance island of <em>ca.</em> 86-kb long which has been uncovered in the genome of the multi-drug resistant human clinical isolate strain AYE responsible for a nationwide outbreak in France in 2001. In contrast, this resistance island was not present in <em>A. baumannii</em> strain SDF associated with human body louse, partly explaining its susceptibility to antibiotics.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>A few years ago, we published the complete genome of another Acinetobacter (<em>A. baylyi</em> ADP1 <a href="#pone.0001805-Barbe1">[9]</a>) for which human experts have performed the annotation process. This complete annotation set therefore appeared to be a strong basis for fast and accurate annotation of the two <em>A. baumannii</em> strains.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>The availability of 3 complete genome sequences from members of the same genus (<em>i.e.</em> AYE, SDF and <em>A. baylyi</em> ADP1 hereafter referred as <em>A. baylyi</em>) prompted us to compare them in a more general context. Here we provide a detailed comparison of the three genomes that highlights differences that might reflect adaptation to a specific environment. The in-depth analysis of gene contents reveals that though the three organisms share a large fraction of genes, large differences exist. For instance, we show that strain AYE has high catabolic and drug/metal resistance capacities whereas strain SDF is riddled with numerous prophage regions and Insertion Sequences (ISs). Moreover, the example of iron uptake illustrates the diverse strategies used by <em>Acinetobacter</em> to thrive in various environmental conditions. These features emphasize the diversity of the <em>Acinetobacter</em> genus. We conclude this report with an analysis of potential virulence factors.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Three <em>Acinetobacter</em> genomes: general features and taxonomic considerations</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The genome of the <em>A. baumannii</em> isolates is made of a single chromosome and several plasmids (4 and 3 for AYE and SDF, respectively). In <em>A. baylyi</em>, no plasmid was detected <a href="#pone.0001805-Barbe1">[9]</a>. Strain AYE has the largest chromosome (ca. 3.9 Mb versus 3.4 Mb for strain SDF and 3.6 Mb for <em>A. baylyi</em>; <a href="#pone-0001805-t001">Table 1</a>). The GC-content of the three sequences is around 40%, a value corresponding to that reported for other members of the <em>Acinetobacter</em> genus.</p>
<div class="figure" id="pone-0001805-t001"><div class="img"><a name="pone-0001805-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001805.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001805.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Comparison of the general features of the 3 <em>Acinetobacter</em> chromosomes.</span></strong></p><span>doi:10.1371/journal.pone.0001805.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Although tRNAs for each of the 20 amino acids were found in both <em>A. baumannii</em> strains AYE and SDF, the latter has a lower copy number of tRNAs for eight amino acids namely Ala, Arg, Asp, Gln, Ile, Ser, Tyr and Val. The largest differences were found for tRNA-Asp (a single copy instead of three) and for tRNA-Gln (2 copies instead of 4 in AYE and 5 in <em>A. baylyi</em>). Dolzani and colleagues <a href="#pone.0001805-Dolzani1">[11]</a> showed that the number of rRNA operons may vary between 5 and 7 for the <em>Acinetobacter</em> species. Confirming these data, strain AYE possessed 6 rRNA clusters whereas strain SDF contained only 5. Like in the case of tRNAs, the missing rRNA cluster may have been lost during one of the numerous recombination events probably mediated by Insertion Sequences (ISs; see below). As reported in the literature, the 16S–23S Intergenic Spacer Region (16S–23S ISR) of <em>A. baylyi</em> strain ADP1 rRNA clusters is of variable length and probably resulted from recombination between rRNA operons <a href="#pone.0001805-Maslunka1">[12]</a>. This does not seem to be a feature shared by species belonging to the <em>Acinetobacter</em> genus since rRNA clusters 16S–23S ISRs in both <em>A. baumannii</em> strains are of constant length (663 bp). Strain SDF's species status was uncertain because this strain has no b-lactamase activity and is unable to grow at 44°C, two features characterizing the <em>A. baumannii</em>-<em>A.calcoaceticus</em> complex. However, phylogenetic reconstruction based on <em>Acinetobacter spp.</em> 16S–23S ISRs currently available in Genbank, clearly placed strain SDF as sister group of other <em>A</em>. <em>baumannii</em> strains (<a href="#pone.0001805.s001">Figure S1</a>). Furthermore, we found in strain SDF a tiny fragment (24 amino acids) corresponding to the C-terminus of <em>ampC</em> gene that shared the same chromosomal context as strain AYE <em>ampC</em>. In addition, this fragment is located next to one IS<em>Aba6</em> copy implying that <em>ampC</em> gene has been disrupted during evolution of this strain. This latter finding is further evidence that SDF is a member of <em>A. baumannii</em> species.</p>


<h4>Diversity of <em>Acinetobacter species</em>: core genome vs. flexible gene pool</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Comparative analyses of the gene content of the three strains define two distinct gene pools: a core genome, the set of orthologous genes shared by the three strains and a flexible gene pool, the sets of genes that either are common to two species out of three or that do not have any significant homolog in any of the two other species (<a href="#pone-0001805-g001">Figure 1</a>). The candidate core genome contains 2,052 genes corresponding to 57.2%, 70.7% and 62.6% of the DNA sequences of <em>A</em>. <em>baumannii</em> strain AYE, strain SDF and <em>A. baylyi</em>, respectively. The sizes of the flexible gene pools may be correlated with different life styles in distinct ecological niches. Even though 361 genes are shared only by AYE and <em>A. baylyi</em>, each species possesses almost 800 specific genes. These figures may reveal a high capacity of adaptation in fluctuating environmental conditions. Consistent with life in a less variable environment (hematophage gut), the amount of specific genes in SDF is significantly lower (n = 398; see <a href="#pone-0001805-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0001805-g001"><div class="img"><a name="pone-0001805-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001805.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001805.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Comparison of gene content of <em>A. baylyi</em> ADP1, <em>A. baumannii</em> AYE and <em>A. baumannii</em> SDF chromosomes.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Putative orthologs are defined as genes showing a minimum of 40% identity and a ratio of 0.8 of the length of the smallest protein, or as two genes included in a synteny group. The intersections between the three circles give the number of genes found in the 2 or 3 compared species. Genes outside these areas are specific to the corresponding organism. The total number of annotated genes is also given under each species name. Figures in brackets indicate the number of pseudogenes.</p>
<span>doi:10.1371/journal.pone.0001805.g001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>For the three species under study, genes were assigned to various functional categories (<a href="#pone-0001805-g002">Figure 2</a>). At first sight, some of the categories display peculiar patterns. Considering <em>A. baumannii</em> AYE, 42% of the genes dedicated to catabolic functions and two third of genes involved in drug/metal resistance are specific. In turn in strain SDF, 80% of prophagic genes and all ISs are specific to this organism. The specificity of strain SDF mainly resides in its extrachromosomal elements content whereas strain AYE has pronounced capacities in catabolism and drug/metal resistance which may explain its successful adaptation to the hospital environment (see below).</p>
<div class="figure" id="pone-0001805-g002"><div class="img"><a name="pone-0001805-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001805.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001805.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Gene content comparison of the three <em>Acinetobacter</em> chromosomes according to nine functional categories.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>The proportions of specific genes which have no ortholog in the two other strains are indicated by light colors. Genes having an ortholog in at least one of the other 2 species were regarded as non specific (dark colors). Absolute gene count for each subclass is reported in boxes at bars bottom. Two genes were considered as orthologs when their respective product shares more than 40% identity over more than 80% of their length.</p>
<span>doi:10.1371/journal.pone.0001805.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>IS elements in the three strains were strikingly different (<a href="#pone-0001805-t001">Table 1</a>). Strain SDF has by far the largest number of IS elements made of two novel elements (IS<em>Aba6</em> and IS<em>Aba7</em>) in about equal proportions and evenly distributed along the chromosome (<a href="#pone-0001805-g003">Figure 3</a>). Those elements belong to IS<em>982</em> and IS<em>5</em> families, respectively. It is very likely that these IS elements have significantly contributed to the genome size reduction, especially via recombination events and gene disruptions. Twenty-one copies of IS<em>Aba1</em> (IS<em>4</em> family) were detected in the genome of the human strain <em>A. baumannii</em> AYE. The IS element IS<em>Aba1</em> has been recently identified from human isolates of <em>A. baumannii</em> <a href="#pone.0001805-Heritier1">[13]</a>. Although its role is unclear, it has been demonstrated to contribute at least to overexpression of the naturally-occurring ß-lactamase <em>ampC</em> gene leading to high-level resistance to ß-lactams and to overexpression of acquired ß-lactamases genes <a href="#pone.0001805-Heritier1">[13]</a>. Whereas no <em>ISAba1</em> copy was identified on any of the plasmids of isolate AYE, chromosomal copies showed no site integration preference (<a href="#pone-0001805-g003">Figure 3</a>). It is important to note that although IS<em>Aba1</em> is supposed to be “customized” for <em>A. baumannii</em> <a href="#pone.0001805-Segal1">[14]</a>, this element is not present in strain SDF genome.</p>
<div class="figure" id="pone-0001805-g003"><div class="img"><a name="pone-0001805-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001805.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001805.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Circular representation of the three <em>Acinetobacter</em> genomes (ADP1, left; AYE, middle; SDF, right).</span></strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>Circles display from the outside in: (1) GC deviation (mean GC content in a 1kb window–overall mean GC). Red areas indicate that deviation is higher than 1.5 SD. (2) Location of various genomic regions. (3) Gene specificity (grey, <em>Acinetobacter spp.</em> core genome; green, <em>A. baumannii</em> core genome; red, strain specific genes). (4) GC skew (G+C/G−C using a 1kb sliding window). (5) Pseudogenes (yellow), ISs (pink), tRNA (green) and rDNA (blue).</p>
<span>doi:10.1371/journal.pone.0001805.g003</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>DNA acquisition/exchanges in bacteria can occur between organisms of the same or different species via one of three mechanisms: conjugative transfer, phage transduction, or natural transformation. The last mechanism relies on the ability of cells to take up and stably maintain large pieces of “alien” DNA. This phenomenon called “natural competence” has been extensively studied in two Gram-negative soil living bacteria, namely <em>A. baylyi</em> and <em>Pseudomonas stutzeri</em> (see for instance <a href="#pone.0001805-deVries1">[15]</a>). Besides their involvement in biofilm formation (see below), some components related to type IV pili biogenesis play an important role in natural transformation process <a href="#pone.0001805-Averhoff1">[16]</a>. This is consistent with the fact that free extracellular DNA itself may trigger biofilm formation <a href="#pone.0001805-Whitchurch1">[17]</a>. Almost all genes previously identified in the two models (<em>com</em> and <em>pil</em> genes) have been found in <em>A. baumannii</em> strain AYE whereas their counterparts are either absent or disrupted in strain SDF, suggesting that the latter strain is not competent for natural transformation (<a href="#pone-0001805-t002">Table 2</a>). Several putative prophage regions were roughly delimited in both <em>A. baumannii</em> genomes (six for AYE and eight for SDF; <a href="#pone-0001805-t002">Table 2</a>). Though not exceptional, these regions constitute a sizable part of both genomes accounting for 5.1% and 6.7% of whole chromosomes of strain AYE and SDF, respectively. Notably, all prophage regions detected were not similar (within and between genomes; <a href="#pone-0001805-t002">Table 2</a>) as suggested by very low similarities between well-known phage markers and by the obvious absence of synteny between the three genomes. Even though plasmids can be gained via natural transformation, a more specific process of plasmid acquisition is conjugative transfer. In contrast to <em>A. baylyi</em>, strains AYE and SDF possess several plasmids (4 and 3 respectively) on which no gene involved in resistance and/or virulence has been characterized. Nevertheless, putative mobilization systems MobS/MobL for DNA exchange and putative toxin/antitoxin systems for plasmid maintenance are present.</p>
<div class="figure" id="pone-0001805-t002"><div class="img"><a name="pone-0001805-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001805.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001805.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Genomic regions found in the 3 <em>Acinetobacter</em> species.</span></strong></p><span>doi:10.1371/journal.pone.0001805.t002</span></div>

<h4>Cell-cell signaling and related processes: biofilm formation and quorum sensing</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In a hospital environment, <em>Acinetobacter</em> strains may survive several days under severely dry conditions <a href="#pone.0001805-Wendt1">[18]</a>. Vidal and colleagues <a href="#pone.0001805-Vidal1">[19]</a> reported the ability of the nosocomial pathogen <em>A. baumannii</em> biotype 9 ACAB715 to form a biofilm on a glass surface, suggesting that this phenomenon plays a key role for the survival of this bacterium under unfavorable environmental conditions. Since biofilm formation may be associated with resistance to immune host response, many clinically significant bacterial species have been studied for their capacity to form biofilm <a href="#pone.0001805-Costerton1">[20]</a>. The different appendages that have been proved to be involved in biofilm formation are type IV pili, flagella, curli and fimbriae. In <em>A. baumannii</em> strain AYE, four putative chaperone usher secretion (<em>csu</em>) systems have been identified (<a href="#pone-0001805-t002">Table 2</a>). The first system (ABAYE1319-24) has an organization similar to the locus described by Tomaras and colleagues <a href="#pone.0001805-Tomaras1">[21]</a> in <em>A. baumannii</em> strain ATCC19606 including the Fur box also detected upstream the gene cluster. Furthermore, three putative surface adhesion protein regions have been detected only in strain AYE (<a href="#pone-0001805-t002">Table 2</a>). Apart from the two <em>csu</em> systems previously identified in <em>A. baylyi</em> <a href="#pone.0001805-Gohl1">[22]</a>, the present comparative genomic analysis led to the conclusion that the cluster ACIAD3332-37 might be a third system. Diverse regulation networks such as Crc (Catabolite repression pathway) <a href="#pone.0001805-OToole1">[23]</a> or two-component systems <a href="#pone.0001805-Kulasekara1">[24]</a> regulate biofilm formation. Further experiments are therefore required to unravel the respective roles of ABAYE0143 (Crc), ABAYE0258-0259 (EnvZ/OmpR) and the regions ABAYE0667-0671/ABAYE3534-3535 (type IV pili biogenesis regulators/twitching motility) in biofilm production by <em>A. baumannii</em> strain AYE.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In many Gram-negative bacteria, quorum sensing (QS) relies on small diffusible molecules, the N-acylhomoserine lactones (AHL). In response to population density, AHL-mediated QS systems modulate expression of genes involved in various biological functions including biofilm formation <a href="#pone.0001805-Davies1">[25]</a>, antibiotics production <a href="#pone.0001805-Pierson1">[26]</a> or Ti plasmid conjugal transfer <a href="#pone.0001805-Zhang1">[27]</a>. AHLs are also known to alter gene expression in host cells <a href="#pone.0001805-Rumbaugh1">[28]</a>. A genomic region (ABAYE3750-ABAYE3761) of strain AYE contains an AHL synthase (ABAYE3761) neighboring a LuxR family transcriptional regulator autoinducer (ABAYE3758). Interestingly, this region has been disrupted by an IS<em>Aba7</em> copy in strain SDF and completely lacks in <em>A. baylyi</em>. Although strain AYE certainly uses QS modulation, further experiments are required to determine in which broader biological process this system takes part. Nevertheless, additional genes found in the QS genomic region provide some clues. They correspond to genes encoding a Non Ribosomal Peptide Synthase (NRPS) coupled with an efflux pump. NRPS systems are known to produce a large family of natural products like toxins, siderophores, antibiotics or pigments.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Degradation of AHLs may be of medical relevance since AHLs-mediated QS systems <em>i)</em> are widespread and highly conserved in many pathogenic bacteria and <em>ii)</em> control the expression of genes involved in pathogenicity <em>sensu lato</em>. For instance, Whitehead and colleagues <a href="#pone.0001805-Whitehead1">[29]</a> hypothesized that the disruption of AHLs-mediated QS systems could possibly help to control diseases. Although the ultimate goal is not clear (competition with other species, neutralization of antibiotics produced by other species…), some bacterial species are able to disrupt a QS signal by the mean of a natural AHL lactonase (see for example <a href="#pone.0001805-Dong1">[30]</a>). Since other <em>Acinetobacter s</em>trains produce AHLase (see for instance <a href="#pone.0001805-Kang1">[31]</a>), genes coding for similar proteins were searched for. This analysis was difficult since <em>i)</em> the similarity between known AHLases is low (ca. 25% identity); <em>ii)</em> a part of the motif that characterizes AHLase (HXHXDH) is shared by several unrelated families of proteins (metallo-b-lactamase, glyoxalase II and arylsulfatase). Nevertheless, we were able to find a candidate in those three species (ABAYE0825, ABSDF0820 and ACIAD0766; <a href="#pone.0001805.s002">Figure S2</a>).</p>


<h4>Metabolism: different gene content for different lifestyles</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Extensive metabolic pathway reconstruction was undertaken for the three <em>Acinetobacter</em> strains. As reported in <em>A. baylyi</em> str. ADP1 <a href="#pone.0001805-Barbe1">[9]</a>, the central metabolism study of both <em>A. baumannii</em> strains confirms the lack of several key enzymes: a glucokinase, proteins of a phosphotransferase transport system, a 6-phosphofructokinase and a pyruvate kinase. The latter activity which exists in <em>Pseudomonas</em> spp. seems to be a good marker to clearly delineate both genera. Few <em>Acinetobacter</em> isolates are able to grow on glucose as sole carbon source and this property was early used as a main criterion to classify <em>Acinetobacter</em> species <a href="#pone.0001805-Baumann1">[32]</a>. The Embden–Meyerhof–Parnas glycolysis can certainly not be performed in <em>Acinetobacter</em> spp. given the lack of the previously mentioned enzymes. An alternative glycolytic route passing through the Entner-Doudoroff pathway was elucidated several decades ago <a href="#pone.0001805-Taylor1">[33]</a>. It starts with glucose periplasmic oxidation and leads to pyruvate formation. All the genes required for this pathway were identified in the three Acinetobacter genomes. Interestingly, <em>A. baylyi</em> and <em>A. baumannii</em> strain SDF only possess the <em>gcd</em> gene which encodes a membrane-bound quinoprotein glucose dehydrogenase <a href="#pone.0001805-CletonJansen1">[34]</a> whereas strain AYE has two additional genes (ABAYE1605, ABAYE1751) which encode soluble forms of quinoprotein dehydrogenases (s-GDH) <a href="#pone.0001805-CletonJansen2">[35]</a> thus possibly allowing the utilization of alternative substrates by the cells.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The three studied strains are prototrophic and are able to grow using succinate as sole carbon source. Among the 499 genes of <em>A. baylyi</em> which are essential for life on minimal medium <a href="#pone.0001805-deBerardinis1">[36]</a>, more than 90 percent have homolog in both <em>A. baumannii</em> genomes. Consitent with this high level of conversation, the predicted amino acid biosynthetic pathways of the AYE and SDF strains are similar to those previously described in <em>A. baylyi</em> <a href="#pone.0001805-Barbe1">[9]</a>. In strain SDF, almost all genes involved in the biosynthesis of main cell components were kept intact except the <em>trpE</em> pseudogene (ABSDF3251) which normally encodes the first subunit of the anthranilate synthase, an essential enzyme for tryptophan biosynthesis. However, this deficiency may be compensated by two genes (ABSDF3335, ABSDF3336) which encode the two anthranilate synthase subunits and are located in the AbaG1 island <a href="#pone.0001805-Fournier2">[10]</a>. This genomic region is likely to be under strong selective pressure to maintain a tryptophan prototrophy. Despite its genome reduction (pseudogenization and gene loss), strain SDF successfully maintained a prototrophic lifestyle.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>In contrast to <em>A. baylyi</em>, both <em>A. baumannii</em> strains genomes harbor the <em>cobUTCS</em> genes (<a href="#pone-0001805-t002">Table 2</a>). This is a major difference in terms of biosynthesis capabilities since these genes encode the necessary enzymes for the last steps of adenosyl-cobalamin (coenzyme B12) synthesis starting from dimethylbenzimidazole and cobinamide as precursors <a href="#pone.0001805-Lawrence1">[37]</a>. However, this pathway is probably not functional anymore in strain SDF because <em>cobS</em> and <em>cobT</em> genes are truncated. Only two reactions requiring coenzyme B12 as cofactor have been detected in the <em>A. baumannii</em> str. AYE genome. The first is a methionine synthase encoded by the gene <em>metH</em> (ABAYE2822) which also exists in <em>A. baylyi</em>. This B12-dependent reaction allows <em>Acinetobacter</em> to synthesize methionine via a non-essential alternative route. The second is the ethanolamine ammonia-lyase which is essential for ethanolamine catabolism. The two subunits of the corresponding protein are encoded by the <em>eutB</em> (ABAYE1458) and <em>eutC</em> (ABAYE1457) genes in strain AYE. These genes are missing in <em>A. baylyi</em> and <em>eutC</em> is disrupted in strain SDF. The catabolism of ethanolamine by strain AYE may be an important carbon and nitrogen sources during the infection process since ethanolamine is a major constituent of cell membrane phospholipids. Furthermore, this compound is a common excipient used in drug solutions.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p><em>A. baylyi</em> is a nutritionally versatile soil bacterium. Its genome analysis revealed that a large amount of genes (10%) are dedicated to the catabolism of a large variety of compounds <a href="#pone.0001805-Barbe1">[9]</a>, <a href="#pone.0001805-Young1">38</a>. Most of these genes are clustered in several genomic regions called catabolic islands which are themselves located side by side in an archipelago accounting for one-quarter of the overall genome (<a href="#pone-0001805-g003">Figure 3</a>). The comparison between strain AYE and <em>A. baylyi</em> revealed a notable conservation of catabolic capacity (<a href="#pone-0001805-t002">Table 2</a>). Furthermore, strain AYE islands are also clustered on the chromosome (i.e. around the terminus of replication and within the third quarter of the genome, <a href="#pone-0001805-g003">Figure 3</a>). The <em>dca-pca-qui-pob-hca</em> chromosomal cluster of <em>A. baylyi</em> encodes enzymes that degrade dicarboxylic acids and aromatic compounds <a href="#pone.0001805-Smith1">[39]</a>. Interestingly, many of these compounds are components of suberin, a protective polymer which is produced by plants in response to stress. In strain AYE, all these genes are also present supporting the hypothesis that soil is also the primary niche of <em>A. baumannii</em> species. Nevertheless, the local organization of these islands is different between both strains. Only the <em>pca-qui</em> genes have been maintained together in strain AYE. This observation is in agreement with experimental results performed in <em>A. baylyi</em> reporting a possible <em>pca-qui</em> operon structure of 14-kb <a href="#pone.0001805-Dal1">[40]</a>. As an alternative in strain AYE, <em>hca</em> and <em>van</em> genes are gathered and may improve ferulic acid degradation leading to vanillate production which is subsequently metabolized by the <em>van</em> genes <a href="#pone.0001805-Segura1">[41]</a>. Different selective pressures may have shaped the genomic island organization in accordance to available nutrients in their natural environment. In addition, <em>A. baumannii</em> str. AYE harbors nine new gene clusters that are absent in <em>A. baylyi</em> (<a href="#pone-0001805-t002">Table 2</a>). Three of them, called CAI1-R, SEU and CAI2-R, form large-size genomic islands of 30.4-kb, 16-kb and 19.8-kb respectively. These regions contain a high proportion of genes encoding enzymes like oxygenases and need to be deeply investigated in order to determine their exact catabolic role. Other AYE specific regions have been assigned a more precise function.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>The CBE cluster of strain AYE may encode genes for choline degradation to produce betaine. In contrast to <em>A. baylyi</em>, <em>sox</em> genes are missing in strain AYE and are necessary for betaine catabolism. Presumably, betaine is not a carbon source for strain AYE and is accumulated in the cytoplasm. In response to osmotic stress, high concentrations of glycine betaine would act as an osmoprotectant <a href="#pone.0001805-Smith2">[42]</a>. In addition, this molecule is also a potent protector against mutagenic compounds and radiation-induced damages. As ethanolamine, choline is a main component of cell membrane phospholipids.</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>The TAU cluster of strain AYE contains genes encoding a taurine transporter and the dioxygenase TauD. Like <em>Escherichia coli</em> K12 <a href="#pone.0001805-vanderPloeg1">[43]</a>, strain AYE could utilize taurine as a sulfur source when facing sulfate starvation conditions. Taurine plays several important roles in mammals: neuromodulation, bile acid conjugation, detoxification, osmoregulation, membrane stabilization, and regulation of intracellular Ca<sup>2+</sup> homeostasis <a href="#pone.0001805-Wright1">[44]</a>. Competition for taurine use might take place during <em>A. baumannii</em> infection process.</p>
<a id="article1.body1.sec2.sec4.p7" name="article1.body1.sec2.sec4.p7"></a><p>Finally, the PEN genomic region of strain AYE chromosome contains fifteen genes which are likely to be involved in penicillin catabolism providing carbon and nitrogen to the bacteria. One of them (ABAYE1713) encodes a putative penicillin acylase (Pac). The substrate specificity of this acylase is still unknown in strain AYE. As reported in <em>E. coli</em> str. W <a href="#pone.0001805-Galan1">[45]</a>, the <em>pac</em> gene is positively regulated by the PaaX repressor of phenylacetate catabolism. Pac could be a scavenger enzyme for natural compounds containing a phenylacetate residue or derivative <a href="#pone.0001805-Merino1">[46]</a>.</p>


<h4>Iron uptake systems: toward a high degree of specialization</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>One possible defense mechanism of human host against bacterial infection is the reduction of free extracellular iron concentration via iron-binding proteins (lactoferrin, transferrin). Indeed, normal concentration of free iron in the body oscillates around 10<sup>−18</sup> M, whereas the minimal concentration required for the bacterial survival is ca. 10<sup>−6</sup> M <a href="#pone.0001805-Bullen1">[47]</a>. Iron/heme acquisition can be achieved thanks to direct contact between the bacterium and the exogenous iron/heme sources. Bacteria may also take out iron from their environment using high affinity iron molecules called siderophores/hemophores that are released outside cells.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p><em>A. baumannii</em> strain AYE may chelate iron using a siderophore (ABAYE1085-ABAYE1104) that can compete with host iron-binding proteins to overcome the iron starvation imposed by the host. The same siderophore (acinetobactin) has been described in another <em>Acinetobacter</em> strain (ATCC19606; <a href="#pone.0001805-Dorsey1">[48]</a>), which was structurally related to the anguibactin of the fish pathogen <em>Vibrio anguillarum</em> <a href="#pone.0001805-DiLorenzo1">[49]</a>. One of the key compounds of these siderophores is the histamine which results from histidine decarboxylation. Tolmasky and colleagues <a href="#pone.0001805-Tolmasky1">[50]</a> showed that when <em>V. anguillarum</em> cells are cultured on 1% histidine, a histidine decarboxylase activity is detected, leading to the production of histamine. Likewise, <em>A. baumannii</em> strain ATCC19606 produces histamine when grown with histidine <a href="#pone.0001805-Actis1">[51]</a>. As expected, we found a histidine decarboxylase gene (<em>hdc</em>) located within the iron uptake region in strain AYE (ABAYE1098). Interestingly, <em>A. baylyi</em> genome has a siderophore gene which composition is not based on histamine because this species lacks the <em>hdc</em> gene <a href="#pone.0001805-Barbe1">[9]</a>.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The transport of the ferri-siderophore (iron-siderophore complex) into the cell is performed by the Iron Regulated Outer Membrane Proteins system (IROMPs). This IROMPs system is typically made of one specific outer-membrane receptor, one periplasmic protein and several inner-membrane-associated proteins acting either as receptors or as energy transducers (<em>i.e.</em> the ExbB:ExbD:TonB complex). The strain AYE potentially possesses two similar uptake systems (ABAYE2046-2051 and ABAYE3318-3320). Once internalised, the ferri-siderophore is reduced to release iron by an enzyme having a ferric reductase activity. We did not identify any structurally related protein. However, Fontecave et al. <a href="#pone.0001805-Fontecave1">[52]</a>, have reported that several enzymes (nitrite reductase, sulfite reductase…) which are not necessarily tightly linked to iron metabolism could present a ferric reductase activity. Thus, a possible candidate is the gene tandem ABAYE1544-ABAYE1545 which encodes the two subunits (large and small) of the nitrite reductase.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Surprisingly, siderophores were not identified in strain SDF. Since iron is necessarily required for life, strain SDF must use a different strategy to scavenge iron from its environment. Several evidences suggest that A. <em>baumannii</em> strain SDF is able to use haem or haemoglobin as source of iron. <em>A</em>. <em>baumannii</em> SDF is in close contact with the gut of haematophage organisms <a href="#pone.0001805-Houhamdi1">[53]</a> and therefore is readily in contact with blood cells. In addition, we found in the genome of this strain several haemagglutinin/haemolysin-related genes encoding proteins that can aggregate erythrocytes and destroy their membrane releasing haemoglobin. We located also a gene cluster (ABSDF2280-88) sharing similarities with genes associated with haem acquisition system (HAS) already described in several species (reviewed in <a href="#pone.0001805-Wandersman1">[54]</a>). According to the literature the transport system of haem or haemoglobin into the cell could also be ExbB:ExbD:TonB complex dependant but in this case the periplasmic intermediary is not required <a href="#pone.0001805-Andrews1">[55]</a>. Several clusters (N = 3) of ExbB, ExbD, and TonB genes have been detected on strain SDF genome as well as several associated ABC transporters which are probably involved in the translocation of haem/haemoglobin from the periplasm to the cytosol. Finally, the key enzyme (Haem Oxygenase HemO = ABSDF2281) required for the release of iron has also been found in this strain.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>Bacteria generally regulate iron import into the cell according to iron availability. In both AYE and SDF strains, the regulation of this process is likely to be mediated by the ferric regulator uptake protein (Fur: ABAYE2920 and ABSDF2538) which acts as a transcription repressor of genes involved either in the siderophore synthesis, in transport or even in degradation of ferri-siderophore/haemo(globin). Furthermore, we identified <em>fur</em> boxes in the upstream region of most of the gene clusters involved in iron capture. Besides the central role of Fur protein, secondary positive regulators may also interfere in the iron uptake process. In <em>Vibrio anguillarum</em>, the assembly of the siderophore anguibactin and the enhancement of iron transport are promoted by products encoded in a transacting factor region of the virulence plasmid pJM1 <a href="#pone.0001805-Welch1">[56]</a>, <a href="#pone.0001805-Stork1">[57]</a>.</p>
<a id="article1.body1.sec2.sec5.p6" name="article1.body1.sec2.sec5.p6"></a><p>The present comparison of iron uptake provides a good example of specialization of <em>A. baumannii</em> strains facing different environmental conditions. In the case of <em>A. baumannii</em> strain AYE, the competition against human host for iron using siderophores may explain part of the pathogenicity of this strain. In contrast, strain SDF may not have to compete for iron and hence use an alternate system.</p>


<h4>Acinetobacter pathogenicity: insights into virulence factors</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Gene products which give a microbe the ability to persist and invade a host of particular species are often referred as virulence factors. These factors generally comprise toxins, cell surface components (adhesins, glycoproteins), and hydrolytic enzymes that may contribute to the pathogenicity of the bacterium. Since no toxin or toxin-like encoding gene has been detected in both <em>A. baumannii</em> strains, their pathogenicity relies essentially on genes which belong to the two last categories.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Outer membrane proteins (OMPs) of Gram-negative bacteria are known to have essential roles in antibiotic resistance as well as in adaptation and pathogenesis in host cells <a href="#pone.0001805-Lin1">[58]</a>. Several OMPs of the OmpA family were characterized in various <em>Acinetobacter</em> strains and represent one of the major OMP in this genus. The <em>A. baylyi</em> OmpA is a secreted protein having an emulsifying activity <a href="#pone.0001805-Walzer1">[59]</a> whereas other OmpA proteins are associated to virulence properties which induce apoptosis of epithelial cells <a href="#pone.0001805-Choi1">[60]</a> or stimulate gastrin and interleukin-8 gene expression <a href="#pone.0001805-OforiDarko1">[61]</a>. The present genome analysis reveals that only one gene in each <em>Acinetobacter</em> strains encodes an OmpA protein (ACIAD0697, ABAYE0640, and ABSDF0605). These proteins share more than 89% of identical amino acids and have the same chromosomal context. These findings suggest that OmpA proteins are ubiquitous in the <em>Acinetobacter</em> genus but also raise questions on their exact role.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>Among hydrolytic enzymes, phopholipases have been the subject of extensive investigation, especially the two types of phospholipases C produced by <em>Pseudomonas aeruginosa</em>: PLC-H encoded by the <em>plcS</em> gene is acidic and has a strong hemolytic activity whereas its counterpart PLC-N encoded by the <em>plcN</em> gene is rather basic and does not have any hemolytic activity (for a review see <a href="#pone.0001805-Titball1">[62]</a>). PLC-H is actually posttranslationnally altered by one of the <em>plcR</em> gene product thus enhancing its hydrolytic capacity <a href="#pone.0001805-Titball1">[62]</a>. We found in each <em>Acinetobacter baumannii</em> genome two copies of phospholipase C (<em>plc</em>) genes sharing about 50 percent identity (<a href="#pone.0001805.s003">Table S1</a>). This suggests that the proteins arose from an ancient duplication. Interestingly, though each copy has its best BlastP match with PLC-N, each protein has a pI lower than 7 which is rather a feature of PLC-H. We therefore cannot conclude on the hemolytic activity of <em>A. baumannii</em> phopholipases C. Furthermore, no <em>plcR</em> gene homolog has been found in both <em>A. baumannii</em> strains. Whether <em>plcR</em> is required for PLC-H activity is not clear, the hemolytic property of <em>plc</em> genes products will therefore remain an open question.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>As other phospholipases (D, A) are thought to be major virulence factors in other species (see for example <a href="#pone.0001805-McKean1">[63]</a>), we have checked for their presence in the genomes under study. We effectively found one or several copies of the two kinds of phospholipases. The most interesting case is certainly the <em>pld</em> gene copies harbored by the AYE genome. Among the 4 copies, 3 displayed peculiar features. They <em>i</em>) were bordered at both sides by an insertion sequence (IS<em>Aba1</em>); <em>ii</em>) were identical indicating that the amplification of the <em>pld</em> copies is rather recent; <em>iii</em>) presented a slippery sequence (<span class="gene">GGGGG AAC CUU</span>) resulting in a +1 frameshift analogous to the one found in the <em>prfB</em> gene of <em>E. coli</em>. Given these evidences, we speculate that the intact copy of <em>pld</em> maintains a basal activity that is necessary for the cell to exploit phosphorous sources whereas the 3 remaining ones are likely to be active but solicited only when facing exceptional environmental conditions (infection process, etc.). Indeed, the strong promoter provided by IS<em>Aba1</em> and the programmed frameshift would allow the regulation of the production of fully functional phospholipase D.</p>
<a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p>While this work was in progress, the sequence of another <em>A. baumannii</em> (strain ATCC17978) was reported <a href="#pone.0001805-Smith3">[64]</a>. In this human-adapted <em>A. baumannii</em> isolate, the virulence factors previously described in strain SDF and AYE were also present. Moreover, in this study, candidate virulence genes were identified by insertional mutagenesis followed by two screening assays (reduced brood size of <em>C. elegans</em> and inhibition of <em>D. discoideum</em>) in which <em>A. baumannii</em> ATCC17978 virulence was primarily induced by 1% ethanol <a href="#pone.0001805-Smith4">[65]</a>. Among the 35 Ethanol-stimulated virulence (Esv) mutants retained by Smith and colleagues, 14 (16 Esv genes are actually reported in the Genbank file) apparently did not have any homolog in <em>A. baylyi</em>. However, after confronting their data and ours, we noticed that actually only 5 Esv genes did not have any correspondence in <em>A. baylyi</em> including 2 from phage origin and 3 putative transcriptional regulators (<a href="#pone-0001805-t003">Table 3</a>). Still according to our annotation, many of the candidate virulence genes were either involved in main cell functions (DNA repair, general metabolic functions, etc.) or proved to be essential in other species thus raising doubts on their effective inactivation by the transposon insertion.</p>
<div class="figure" id="pone-0001805-t003"><div class="img"><a name="pone-0001805-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001805" data-uri="info:doi/10.1371/journal.pone.0001805.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001805.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001805.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001805.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001805.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001805.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Functional annotation proposed for 16 Esv-mutants published by smith and colleagues <a href="#pone.0001805-Smith3">[64]</a>.</span></strong></p><span>doi:10.1371/journal.pone.0001805.t003</span></div><a id="article1.body1.sec2.sec6.p6" name="article1.body1.sec2.sec6.p6"></a><p>Although additional investigations are needed to demonstrate the presence of true virulence genes in <em>A. baumannii</em>, the present work provides keys for understanding the adaptability of Acinetobacters to various ecological niches.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sequencing</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The complete genome sequences of <em>A. baumannii</em> strains AYE and SDF were determined using the whole-genome shotgun method. Two libraries, obtained by chemical shearing of genomic DNA and cloning of generated 2–5 kb and 6–17 kb inserts into the plasmids pNAV (A) and pCNS (B) (pcDNA2.1 and pSU18 derived, respectively), were constructed per genome. DNA Plasmids were purified and end-sequenced (61440 (A) and 20928 (B) for AYE and 62784 (A) and 22272 (B) for SDF) by dye-terminator chemistry on ABI3730 sequencers (Applied Biosystems, Foster City, United States). All reads were first compared to <em>A. baylyi</em> ADP1 rRNA (16S and 23S sequences) and ISfinder database <a href="#pone.0001805-Siguier1">[66]</a>, and those sharing at least 80% identity on a length of 100 nt were tagged as rRNA sequences (C) or ISs (D). A first assembly for each genome, using Phred/Phrap/Consed software package (<a href="http://www.phrap.com">www.phrap.com</a>), was performed without 713 (C) and 2691 (D) reads for AYE and 514 (C) and 18156 (D) reads for SDF. Repeated sequences longer than 600 nt were detected with the RepSeek software <a href="#pone.0001805-Achaz1">[67]</a> and subsequently recorded to an internal database. Similarity searches across the newly created database using same criteria as mentioned above, allowed the identification of 1699 and 2434 reads as part of repeated sequences in AYE and SDF, respectively. A new assembly was constructed without rRNA, IS and internal repeated sequences and rRNA. IS and internal repeated reads were then reinserted in the assembly on the clone-links basis. Additional reactions were necessary to complete the genomic sequences of <em>A. baumannii</em> strains AYE and SDF (9648 and 19240, respectively).</p>


<h4>Annotation and Comparative Genome Analyses</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>CoDing Sequences were first predicted using the AMIGene (Annotation of Microbial Genomes) software <a href="#pone.0001805-Bocs1">[68]</a>. Each CDS was then submitted to automatic functional annotation. Putative orthologous relations between two genomes are defined as gene pairs that satisfy either the Bidirectional Best Hit criterion <a href="#pone.0001805-Overbeek1">[69]</a> or an alignment threshold (at least 40% sequence identity on at least 80% of the length of the smallest protein). These relations are subsequently used to search for conserved gene clusters (synteny group) among several bacterial genomes. The algorithm based on an exact graph-theoretical approach is detailed in <a href="#pone.0001805-Boyer1">[70]</a>. This method allows for <em>i)</em> multiple correspondences between genes and, thus, paralogy relations and/or gene fusions are easily detected; <em>ii)</em> Chromosomal rearrangements (inversion, insertion/deletion). A ‘gap’ parameter, representing the maximum number of consecutive genes which are not involved in a synteny group, was set to five.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>As a final step, manual validation of automatic annotations was performed using the MaGe web interface <a href="#pone.0001805-Vallenet1">[71]</a>.</p>


<h4>Data Availability</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Annotation sets of all <em>A. baumannii</em> replicons have been deposited in EMBL database under the following references:</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p><em>Acinetobacter baumannii</em> str. SDF: Chromosome, CU468230; Plasmid 1, CU468231; Plasmid 2, CU468232; Plasmid 3, CU468233.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p><em>Acinetobacter baumannii</em> str. AYE: Chromosome, CU459141; Plasmid 1, CU459137; Plasmid 2, CU459138; Plasmid 3, CU459140; Plasmid 4, CU459139.</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>These annotations as well as comparisons results are publicly available for consultation at <a href="http://www.genoscope.cns.fr/agc/mage">http://www.genoscope.cns.fr/agc/mage</a>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001805"></div><a name="pone.0001805.s001" id="pone.0001805.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001805.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree reconstruction for all ISRs from <em>Acinetobacter spp.</em> available in Genbank (release 160), using as outgroup <em>Escherichia coli</em> BCRC 15486 ISR. This analysis shows that <em>A. baumannii</em> strain SDF is effectively a baumannii species although it is devoid of <em>ampC</em> gene and is unable to grow at 44°C (two phenotypic traits that characterize species belonging to the <em>A. calcoaceticus</em>-<em>A. baumannii</em> complex). Moreover, it confirms the closeness of the relationship between <em>A. calcoaceticus</em>, <em>A. baumannii</em> and genomic species 3 and 13TU. Sequences were first aligned with MUSCLE <a href="#pone.0001805-Edgar1">[72]</a> and the PhyML software <a href="#pone.0001805-Guindon1">[73]</a> was used to build the tree (substitution model: HKY, 100 bootstrap replicates).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001805.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.52 MB TIF)</p>
<a name="pone.0001805.s002" id="pone.0001805.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001805.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Alignment of presumed AHL lactonases of the 3 Acinetobacters under study together with other known AHL lactonases from <em>Bacillus subtilis</em>, <em>Bacillus thurengiensis</em>, <em>Arthrobacter sp.</em> and <em>Agrobacterium tumefaciens</em>. The red boxes indicate active residues of AHLases. Alignment was performed with MUSCLE <a href="#pone.0001805-Edgar1">[72]</a> embedded in JalView <a href="#pone.0001805-Clamp1">[74]</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001805.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(2.43 MB TIF)</p>
<a name="pone.0001805.s003" id="pone.0001805.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001805.s003">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p>Labels of genes annotated as (putative) phospholipase in four <em>Acinetobacter</em> genomes. Frameshifted copies are indicated in bold type (part1–part2).</p>
<a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="preSiDOI"><sup>a</sup>Data from <a href="#pone.0001805-Smith3">[64]</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001805.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p3" name="article1.body1.sec4.supplementary-material3.caption1.p3"></a><p class="postSiDOI">(0.02 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors would like to thank V. de Berardinis (Genoscope), G.N. Cohen (Institut Pasteur, Paris, France) for critically reading the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Performed the experiments: CR VB SM CA BS LP EB PL SO JP. Analyzed the data: DV VB CM SC LP AK. Contributed reagents/materials/analysis tools: JW JC CR DR PN LP. Wrote the paper: DV VB SC PN. Other: Provided A. baumannii strain AYE: PN. Did the manual expert annotation of the two strains: VB. Managed informatics resources of the project: CD SG. Designed the initial project: DR PN JC. Provided A. baumannii strain SDF: DR. Contributed to the annotation: DV SC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001805-Fournier1" id="pone.0001805-Fournier1"></a>Fournier PE, Richet H (2006) The epidemiology and control of Acinetobacter baumannii in health care facilities. Clin Infect Dis  42: 692–699.  <ul class="find" data-citedArticleID="1075414" data-doi="10.1086/500202"><li><a href="http://dx.doi.org/10.1086/500202" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+epidemiology+and+control+of+Acinetobacter+baumannii+in+health+care+facilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+epidemiology+and+control+of+Acinetobacter+baumannii+in+health+care+facilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001805-LaScola1" id="pone.0001805-LaScola1"></a>La Scola B, Raoult D (2004) Acinetobacter baumannii in human body louse. Emerg Infect Dis  10: 1671–1673.  <ul class="find" data-citedArticleID="1075436" data-doi="10.3201/eid1009.040242"><li><a href="http://dx.doi.org/10.3201/eid1009.040242" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acinetobacter+baumannii+in+human+body+louse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acinetobacter+baumannii+in+human+body+louse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001805-Bouvet1" id="pone.0001805-Bouvet1"></a>Bouvet PJM, Grimont PA (1986) Taxonomy of the genus Acinetobacter with the recognition of <em>Acinetobacter baumannii</em> sp. nov., <em>Acinetobacter haemolyticus</em> sp. nov., <em>Acinetobacter johnsonii</em> sp. nov., and <em>Acinetobacter junii</em> sp. nov. and emended descriptions of <em>Acinetobacter calcoaceticus</em> and <em>Acinetobacter lwoffi</em>. Int J Syst Bacteriol  36: 228–240.  <ul class="find" data-citedArticleID="1075380" data-doi="10.1099/00207713-36-2-228"><li><a href="http://dx.doi.org/10.1099/00207713-36-2-228" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Taxonomy+of+the+genus+Acinetobacter+with+the+recognition+of+Acinetobacter+baumannii+sp.+nov.%2C+Acinetobacter+haemolyticus+sp.+nov.%2C+Acinetobacter+johnsonii+sp.+nov.%2C+and+Acinetobacter+junii+sp.+nov.+and+emended+descriptions+of+Acinetobacter+calcoaceticus+and+Acinetobacter+lwoffi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Taxonomy+of+the+genus+Acinetobacter+with+the+recognition+of+Acinetobacter+baumannii+sp.+nov.%2C+Acinetobacter+haemolyticus+sp.+nov.%2C+Acinetobacter+johnsonii+sp.+nov.%2C+and+Acinetobacter+junii+sp.+nov.+and+emended+descriptions+of+Acinetobacter+calcoaceticus+and+Acinetobacter+lwoffi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001805-Bouvet2" id="pone.0001805-Bouvet2"></a>Bouvet PJ, Grimont PA (1987) Identification and biotyping of clinical isolates of Acinetobacter. Ann Inst Pasteur Microbiol  138: 569–578.  <ul class="find" data-citedArticleID="1075382" data-doi="10.1016/0769-2609(87)90042-1"><li><a href="http://dx.doi.org/10.1016/0769-2609(87)90042-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+biotyping+of+clinical+isolates+of+Acinetobacter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+biotyping+of+clinical+isolates+of+Acinetobacter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001805-Vaneechoutte1" id="pone.0001805-Vaneechoutte1"></a>Vaneechoutte M, Dijkshoorn L, Tjernberg I, Elaichouni A, de Vos P, et al.  (1995) Identification of Acinetobacter genomic species by amplified ribosomal DNA restriction analysis. J Clin Microbiol  33: 11–15.  <ul class="find" data-citedArticleID="1075484" data-doi="10.1016/s0723-2020(98)80006-4"><li><a href="http://dx.doi.org/10.1016/s0723-2020(98)80006-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Acinetobacter+genomic+species+by+amplified+ribosomal+DNA+restriction+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Acinetobacter+genomic+species+by+amplified+ribosomal+DNA+restriction+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001805-Janssen1" id="pone.0001805-Janssen1"></a>Janssen P, Maquelin K, Coopman R, Tjernberg I, Bouvet P, et al.  (1997) Discrimination of Acinetobacter genomic species by AFLP fingerprinting. Int J Syst Bacteriol  47: 1179–1187.  <ul class="find" data-citedArticleID="1075430" data-doi="10.1099/00207713-47-4-1179"><li><a href="http://dx.doi.org/10.1099/00207713-47-4-1179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discrimination+of+Acinetobacter+genomic+species+by+AFLP+fingerprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discrimination+of+Acinetobacter+genomic+species+by+AFLP+fingerprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001805-GernerSmidt1" id="pone.0001805-GernerSmidt1"></a>Gerner-Smidt P, Tjernberg I, Ursing J (1991) Reliability of phenotypic tests for identification of Acinetobacter species. J Clin Microbiol  29: 277–282.  <ul class="find" data-citedArticleID="1075420"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Reliability+of+phenotypic+tests+for+identification+of+Acinetobacter+species.&amp;auth=&amp;atitle=Reliability+of+phenotypic+tests+for+identification+of+Acinetobacter+species." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reliability+of+phenotypic+tests+for+identification+of+Acinetobacter+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reliability+of+phenotypic+tests+for+identification+of+Acinetobacter+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001805-AbdelElHaleem1" id="pone.0001805-AbdelElHaleem1"></a>Abdel-El-Haleem D (2003) Acinetobacter: environmental and biotechnological applications. African Journal Of Biotechnology  2: 71–74.  <ul class="find" data-citedArticleID="1075364"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Acinetobacter%3A+environmental+and+biotechnological+applications.&amp;auth=&amp;atitle=Acinetobacter%3A+environmental+and+biotechnological+applications." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acinetobacter%3A+environmental+and+biotechnological+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acinetobacter%3A+environmental+and+biotechnological+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001805-Barbe1" id="pone.0001805-Barbe1"></a>Barbe V, Vallenet D, Fonknechten N, Kreimeyer A, Oztas S, et al.  (2004) Unique features revealed by the genome sequence of Acinetobacter sp. ADP1, a versatile and naturally transformation competent bacterium. Nucleic Acids Res  32: 5766–5779.  <ul class="find" data-citedArticleID="1075374" data-doi="10.1093/nar/gkh910"><li><a href="http://dx.doi.org/10.1093/nar/gkh910" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unique+features+revealed+by+the+genome+sequence+of+Acinetobacter+sp.+ADP1%2C+a+versatile+and+naturally+transformation+competent+bacterium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unique+features+revealed+by+the+genome+sequence+of+Acinetobacter+sp.+ADP1%2C+a+versatile+and+naturally+transformation+competent+bacterium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001805-Fournier2" id="pone.0001805-Fournier2"></a>Fournier PE, Vallenet D, Barbe V, Audic S, Ogata H, et al.  (2006) Comparative genomics of multidrug resistance in Acinetobacter baumannii. PLoS Genet  2: e7.  <ul class="find" data-citedArticleID="1075416" data-doi="10.1371/journal.pgen.0020007"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomics+of+multidrug+resistance+in+Acinetobacter+baumannii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomics+of+multidrug+resistance+in+Acinetobacter+baumannii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001805-Dolzani1" id="pone.0001805-Dolzani1"></a>Dolzani L, Tonin E, Lagatolla C, Prandin L, Monti-Bragadin C (1995) Identification of Acinetobacter isolates in the A. calcoaceticus-A. baumannii complex by restriction analysis of the 16S-23S rRNA intergenic-spacer sequences. J Clin Microbiol  33: 1108–1113.  <ul class="find" data-citedArticleID="1075404"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+Acinetobacter+isolates+in+the+A.+calcoaceticus-A.+baumannii+complex+by+restriction+analysis+of+the+16S-23S+rRNA+intergenic-spacer+sequences.&amp;auth=&amp;atitle=Identification+of+Acinetobacter+isolates+in+the+A.+calcoaceticus-A.+baumannii+complex+by+restriction+analysis+of+the+16S-23S+rRNA+intergenic-spacer+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Acinetobacter+isolates+in+the+A.+calcoaceticus-A.+baumannii+complex+by+restriction+analysis+of+the+16S-23S+rRNA+intergenic-spacer+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Acinetobacter+isolates+in+the+A.+calcoaceticus-A.+baumannii+complex+by+restriction+analysis+of+the+16S-23S+rRNA+intergenic-spacer+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001805-Maslunka1" id="pone.0001805-Maslunka1"></a>Maslunka C, Carr E, Gurtler V, Kampfer P, Seviour R (2006) Estimation of ribosomal RNA operon (rrn) copy number in Acinetobacter isolates and potential of patterns of rrn operon-containing fragments for typing strains of members of this genus. Syst Appl Microbiol  29: 216–228.  <ul class="find" data-citedArticleID="1075442" data-doi="10.1016/j.syapm.2005.07.015"><li><a href="http://dx.doi.org/10.1016/j.syapm.2005.07.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+ribosomal+RNA+operon+%28rrn%29+copy+number+in+Acinetobacter+isolates+and+potential+of+patterns+of+rrn+operon-containing+fragments+for+typing+strains+of+members+of+this+genus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+ribosomal+RNA+operon+%28rrn%29+copy+number+in+Acinetobacter+isolates+and+potential+of+patterns+of+rrn+operon-containing+fragments+for+typing+strains+of+members+of+this+genus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001805-Heritier1" id="pone.0001805-Heritier1"></a>Heritier C, Poirel L, Nordmann P (2006) Cephalosporinase over-expression resulting from insertion of ISAba1 in Acinetobacter baumannii. Clin Microbiol Infect  12: 123–130.  <ul class="find" data-citedArticleID="1075426" data-doi="10.1111/j.1469-0691.2005.01320.x"><li><a href="http://dx.doi.org/10.1111/j.1469-0691.2005.01320.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cephalosporinase+over-expression+resulting+from+insertion+of+ISAba1+in+Acinetobacter+baumannii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cephalosporinase+over-expression+resulting+from+insertion+of+ISAba1+in+Acinetobacter+baumannii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001805-Segal1" id="pone.0001805-Segal1"></a>Segal H, Garny S, Elisha BG (2005) Is IS(ABA-1) customized for Acinetobacter? FEMS Microbiol Lett  243: 425–429.  <ul class="find" data-citedArticleID="1075458" data-doi="10.1016/j.femsle.2005.01.005"><li><a href="http://dx.doi.org/10.1016/j.femsle.2005.01.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+IS%28ABA-1%29+customized+for+Acinetobacter%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+IS%28ABA-1%29+customized+for+Acinetobacter%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001805-deVries1" id="pone.0001805-deVries1"></a>de Vries J, Meier P, Wackernagel W (2001) The natural transformation of the soil bacteria Pseudomonas stutzeri and Acinetobacter sp. by transgenic plant DNA strictly depends on homologous sequences in the recipient cells. FEMS Microbiol Lett  195: 211–215.  <ul class="find" data-citedArticleID="1075508" data-doi="10.1111/j.1574-6968.2001.tb10523.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.2001.tb10523.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+natural+transformation+of+the+soil+bacteria+Pseudomonas+stutzeri+and+Acinetobacter+sp.+by+transgenic+plant+DNA+strictly+depends+on+homologous+sequences+in+the+recipient+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+natural+transformation+of+the+soil+bacteria+Pseudomonas+stutzeri+and+Acinetobacter+sp.+by+transgenic+plant+DNA+strictly+depends+on+homologous+sequences+in+the+recipient+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001805-Averhoff1" id="pone.0001805-Averhoff1"></a>Averhoff B, Friedrich A (2003) Type IV pili-related natural transformation systems: DNA transport in mesophilic and thermophilic bacteria. Arch Microbiol  180: 385–393.  <ul class="find" data-citedArticleID="1075372" data-doi="10.1007/s00203-003-0616-6"><li><a href="http://dx.doi.org/10.1007/s00203-003-0616-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Type+IV+pili-related+natural+transformation+systems%3A+DNA+transport+in+mesophilic+and+thermophilic+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Type+IV+pili-related+natural+transformation+systems%3A+DNA+transport+in+mesophilic+and+thermophilic+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001805-Whitchurch1" id="pone.0001805-Whitchurch1"></a>Whitchurch CB, Tolker-Nielsen T, Ragas PC, Mattick JS (2002) Extracellular DNA required for bacterial biofilm formation. Science  295: 1487.  <ul class="find" data-citedArticleID="1075496" data-doi="10.1126/science.295.5559.1487"><li><a href="http://dx.doi.org/10.1126/science.295.5559.1487" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extracellular+DNA+required+for+bacterial+biofilm+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extracellular+DNA+required+for+bacterial+biofilm+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001805-Wendt1" id="pone.0001805-Wendt1"></a>Wendt C, Dietze B, Dietz E, Ruden H (1997) Survival of Acinetobacter baumannii on dry surfaces. J Clin Microbiol  35: 1394–1397.  <ul class="find" data-citedArticleID="1075494"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Survival+of+Acinetobacter+baumannii+on+dry+surfaces.&amp;auth=&amp;atitle=Survival+of+Acinetobacter+baumannii+on+dry+surfaces." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Survival+of+Acinetobacter+baumannii+on+dry+surfaces." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Survival+of+Acinetobacter+baumannii+on+dry+surfaces.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001805-Vidal1" id="pone.0001805-Vidal1"></a>Vidal R, Dominguez M, Urrutia H, Bello H, Gonzalez G, et al.  (1996) Biofilm formation by Acinetobacter baumannii. Microbios  86: 49–58.  <ul class="find" data-citedArticleID="1075486"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Biofilm+formation+by+Acinetobacter+baumannii.&amp;auth=&amp;atitle=Biofilm+formation+by+Acinetobacter+baumannii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biofilm+formation+by+Acinetobacter+baumannii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biofilm+formation+by+Acinetobacter+baumannii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001805-Costerton1" id="pone.0001805-Costerton1"></a>Costerton JW, Stewart PS, Greenberg EP (1999) Bacterial biofilms: a common cause of persistent infections. Science  284: 1318–1322.  <ul class="find" data-citedArticleID="1075396" data-doi="10.1126/science.284.5418.1318"><li><a href="http://dx.doi.org/10.1126/science.284.5418.1318" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+biofilms%3A+a+common+cause+of+persistent+infections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+biofilms%3A+a+common+cause+of+persistent+infections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001805-Tomaras1" id="pone.0001805-Tomaras1"></a>Tomaras AP, Dorsey CW, Edelmann RE, Actis LA (2003) Attachment to and biofilm formation on abiotic surfaces by Acinetobacter baumannii: involvement of a novel chaperone-usher pili assembly system. Microbiology  149: 3473–3484.  <ul class="find" data-citedArticleID="1075480" data-doi="10.1099/mic.0.26541-0"><li><a href="http://dx.doi.org/10.1099/mic.0.26541-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Attachment+to+and+biofilm+formation+on+abiotic+surfaces+by+Acinetobacter+baumannii%3A+involvement+of+a+novel+chaperone-usher+pili+assembly+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Attachment+to+and+biofilm+formation+on+abiotic+surfaces+by+Acinetobacter+baumannii%3A+involvement+of+a+novel+chaperone-usher+pili+assembly+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001805-Gohl1" id="pone.0001805-Gohl1"></a>Gohl O, Friedrich A, Hoppert M, Averhoff B (2006) The thin pili of Acinetobacter sp. strain BD413 mediate adhesion to biotic and abiotic surfaces. Appl Environ Microbiol  72: 1394–1401.  <ul class="find" data-citedArticleID="1075422" data-doi="10.1128/aem.72.2.1394-1401.2006"><li><a href="http://dx.doi.org/10.1128/aem.72.2.1394-1401.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+thin+pili+of+Acinetobacter+sp.+strain+BD413+mediate+adhesion+to+biotic+and+abiotic+surfaces." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+thin+pili+of+Acinetobacter+sp.+strain+BD413+mediate+adhesion+to+biotic+and+abiotic+surfaces.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001805-OToole1" id="pone.0001805-OToole1"></a>O'Toole GA, Gibbs KA, Hager PW, Phibbs PV Jr, Kolter R (2000) The global carbon metabolism regulator Crc is a component of a signal transduction pathway required for biofilm development by Pseudomonas aeruginosa. J Bacteriol  182: 425–431.  <ul class="find" data-citedArticleID="1075448" data-doi="10.1128/jb.182.2.425-431.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.2.425-431.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+global+carbon+metabolism+regulator+Crc+is+a+component+of+a+signal+transduction+pathway+required+for+biofilm+development+by+Pseudomonas+aeruginosa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+global+carbon+metabolism+regulator+Crc+is+a+component+of+a+signal+transduction+pathway+required+for+biofilm+development+by+Pseudomonas+aeruginosa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001805-Kulasekara1" id="pone.0001805-Kulasekara1"></a>Kulasekara HD, Ventre I, Kulasekara BR, Lazdunski A, Filloux A, et al.  (2005) A novel two-component system controls the expression of Pseudomonas aeruginosa fimbrial cup genes. Mol Microbiol  55: 368–380.  <ul class="find" data-citedArticleID="1075434" data-doi="10.1111/j.1365-2958.2004.04402.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04402.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+two-component+system+controls+the+expression+of+Pseudomonas+aeruginosa+fimbrial+cup+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+two-component+system+controls+the+expression+of+Pseudomonas+aeruginosa+fimbrial+cup+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001805-Davies1" id="pone.0001805-Davies1"></a>Davies DG, Parsek MR, Pearson JP, Iglewski BH, Costerton JW, et al.  (1998) The involvement of cell-to-cell signals in the development of a bacterial biofilm. Science  280: 295–298.  <ul class="find" data-citedArticleID="1075400" data-doi="10.1126/science.280.5361.295"><li><a href="http://dx.doi.org/10.1126/science.280.5361.295" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+involvement+of+cell-to-cell+signals+in+the+development+of+a+bacterial+biofilm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+involvement+of+cell-to-cell+signals+in+the+development+of+a+bacterial+biofilm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001805-Pierson1" id="pone.0001805-Pierson1"></a>Pierson LS 3rd, Keppenne VD, Wood DW (1994) Phenazine antibiotic biosynthesis in Pseudomonas aureofaciens 30-84 is regulated by PhzR in response to cell density. J Bacteriol  176: 3966–3974.  <ul class="find" data-citedArticleID="1075454"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phenazine+antibiotic+biosynthesis+in+Pseudomonas+aureofaciens+30-84+is+regulated+by+PhzR+in+response+to+cell+density.&amp;auth=&amp;atitle=Phenazine+antibiotic+biosynthesis+in+Pseudomonas+aureofaciens+30-84+is+regulated+by+PhzR+in+response+to+cell+density." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenazine+antibiotic+biosynthesis+in+Pseudomonas+aureofaciens+30-84+is+regulated+by+PhzR+in+response+to+cell+density." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenazine+antibiotic+biosynthesis+in+Pseudomonas+aureofaciens+30-84+is+regulated+by+PhzR+in+response+to+cell+density.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001805-Zhang1" id="pone.0001805-Zhang1"></a>Zhang L, Murphy PJ, Kerr A, Tate ME (1993) Agrobacterium conjugation and gene regulation by N-acyl-L-homoserine lactones. Nature  362: 446–448.  <ul class="find" data-citedArticleID="1075504" data-doi="10.1038/362446a0"><li><a href="http://dx.doi.org/10.1038/362446a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Agrobacterium+conjugation+and+gene+regulation+by+N-acyl-L-homoserine+lactones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Agrobacterium+conjugation+and+gene+regulation+by+N-acyl-L-homoserine+lactones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001805-Rumbaugh1" id="pone.0001805-Rumbaugh1"></a>Rumbaugh KP (2006) Convergence of hormones and autoinducers at the host/pathogen interface. Anal Bioanal Chem.   <ul class="find" data-citedArticleID="1075456" data-doi="10.1007/s00216-006-0694-9"><li><a href="http://dx.doi.org/10.1007/s00216-006-0694-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Convergence+of+hormones+and+autoinducers+at+the+host%2Fpathogen+interface." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Convergence+of+hormones+and+autoinducers+at+the+host%2Fpathogen+interface.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001805-Whitehead1" id="pone.0001805-Whitehead1"></a>Whitehead NA, Welch M, Salmond GP (2001) Silencing the majority. Nat Biotechnol  19: 735–736.  <ul class="find" data-citedArticleID="1075498" data-doi="10.1038/90780"><li><a href="http://dx.doi.org/10.1038/90780" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Silencing+the+majority." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Silencing+the+majority.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001805-Dong1" id="pone.0001805-Dong1"></a>Dong YH, Xu JL, Li XZ, Zhang LH (2000) AiiA, an enzyme that inactivates the acylhomoserine lactone quorum-sensing signal and attenuates the virulence of Erwinia carotovora. Proc Natl Acad Sci U S A  97: 3526–3531.  <ul class="find" data-citedArticleID="1075406" data-doi="10.1073/pnas.97.7.3526"><li><a href="http://dx.doi.org/10.1073/pnas.97.7.3526" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AiiA%2C+an+enzyme+that+inactivates+the+acylhomoserine+lactone+quorum-sensing+signal+and+attenuates+the+virulence+of+Erwinia+carotovora." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AiiA%2C+an+enzyme+that+inactivates+the+acylhomoserine+lactone+quorum-sensing+signal+and+attenuates+the+virulence+of+Erwinia+carotovora.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001805-Kang1" id="pone.0001805-Kang1"></a>Kang BR, Lee JH, Ko SJ, Lee YH, Cha JS, et al.  (2004) Degradation of acyl-homoserine lactone molecules by Acinetobacter sp. strain C1010. Can J Microbiol  50: 935–941.  <ul class="find" data-citedArticleID="1075432" data-doi="10.1139/w04-083"><li><a href="http://dx.doi.org/10.1139/w04-083" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Degradation+of+acyl-homoserine+lactone+molecules+by+Acinetobacter+sp.+strain+C1010." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Degradation+of+acyl-homoserine+lactone+molecules+by+Acinetobacter+sp.+strain+C1010.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001805-Baumann1" id="pone.0001805-Baumann1"></a>Baumann P, Doudoroff M, Stanier RY (1968) A study of the Moraxella group. II. Oxidative-negative species (genus Acinetobacter). J Bacteriol  95: 1520–1541.  <ul class="find" data-citedArticleID="1075376"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+study+of+the+Moraxella+group.+II.+Oxidative-negative+species+%28genus+Acinetobacter%29.&amp;auth=&amp;atitle=A+study+of+the+Moraxella+group.+II.+Oxidative-negative+species+%28genus+Acinetobacter%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+study+of+the+Moraxella+group.+II.+Oxidative-negative+species+%28genus+Acinetobacter%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+study+of+the+Moraxella+group.+II.+Oxidative-negative+species+%28genus+Acinetobacter%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001805-Taylor1" id="pone.0001805-Taylor1"></a>Taylor WH, Juni E (1961) Pathways for biosynthesis of a bacterial capsular polysaccharide. I. Carbohydrate metabolism and terminal oxidation mechanisms of a capsuleproducing coccus. J Bacteriol  81: 694–703.  <ul class="find" data-citedArticleID="1075474"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Pathways+for+biosynthesis+of+a+bacterial+capsular+polysaccharide.+I.+Carbohydrate+metabolism+and+terminal+oxidation+mechanisms+of+a+capsuleproducing+coccus.&amp;auth=&amp;atitle=Pathways+for+biosynthesis+of+a+bacterial+capsular+polysaccharide.+I.+Carbohydrate+metabolism+and+terminal+oxidation+mechanisms+of+a+capsuleproducing+coccus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathways+for+biosynthesis+of+a+bacterial+capsular+polysaccharide.+I.+Carbohydrate+metabolism+and+terminal+oxidation+mechanisms+of+a+capsuleproducing+coccus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathways+for+biosynthesis+of+a+bacterial+capsular+polysaccharide.+I.+Carbohydrate+metabolism+and+terminal+oxidation+mechanisms+of+a+capsuleproducing+coccus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001805-CletonJansen1" id="pone.0001805-CletonJansen1"></a>Cleton-Jansen AM, Goosen N, Odle G, van de Putte P (1988) Nucleotide sequence of the gene coding for quinoprotein glucose dehydrogenase from Acinetobacter calcoaceticus. Nucleic Acids Res  16: 6228.  <ul class="find" data-citedArticleID="1075392" data-doi="10.1093/nar/16.13.6228"><li><a href="http://dx.doi.org/10.1093/nar/16.13.6228" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleotide+sequence+of+the+gene+coding+for+quinoprotein+glucose+dehydrogenase+from+Acinetobacter+calcoaceticus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleotide+sequence+of+the+gene+coding+for+quinoprotein+glucose+dehydrogenase+from+Acinetobacter+calcoaceticus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001805-CletonJansen2" id="pone.0001805-CletonJansen2"></a>Cleton-Jansen AM, Goosen N, Vink K, van de Putte P (1989) Cloning, characterization and DNA sequencing of the gene encoding the Mr 50,000 quinoprotein glucose dehydrogenase from Acinetobacter calcoaceticus. Mol Gen Genet  217: 430–436.  <ul class="find" data-citedArticleID="1075394" data-doi="10.1007/bf02464914"><li><a href="http://dx.doi.org/10.1007/bf02464914" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning%2C+characterization+and+DNA+sequencing+of+the+gene+encoding+the+Mr+50%2C000+quinoprotein+glucose+dehydrogenase+from+Acinetobacter+calcoaceticus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning%2C+characterization+and+DNA+sequencing+of+the+gene+encoding+the+Mr+50%2C000+quinoprotein+glucose+dehydrogenase+from+Acinetobacter+calcoaceticus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001805-deBerardinis1" id="pone.0001805-deBerardinis1"></a>de Berardinis V, Vallenet D, Castelli V, Besnard M, Pinet A, et al.  (2008) A complete collection of single-gene deletion mutants of Acinetobacter baylyi ADP1. Mol Syst Biol  0: 0–0.  <ul class="find" data-citedArticleID="1075506" data-doi="10.1038/msb.2008.10"><li><a href="http://dx.doi.org/10.1038/msb.2008.10" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+complete+collection+of+single-gene+deletion+mutants+of+Acinetobacter+baylyi+ADP1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+complete+collection+of+single-gene+deletion+mutants+of+Acinetobacter+baylyi+ADP1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001805-Lawrence1" id="pone.0001805-Lawrence1"></a>Lawrence JG, Roth JR (1995) The cobalamin (coenzyme B12) biosynthetic genes of Escherichia coli. J Bacteriol  177: 6371–6380.  <ul class="find" data-citedArticleID="1075438"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+cobalamin+%28coenzyme+B12%29+biosynthetic+genes+of+Escherichia+coli.&amp;auth=&amp;atitle=The+cobalamin+%28coenzyme+B12%29+biosynthetic+genes+of+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+cobalamin+%28coenzyme+B12%29+biosynthetic+genes+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+cobalamin+%28coenzyme+B12%29+biosynthetic+genes+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001805-Young1" id="pone.0001805-Young1"></a>Young DM, Parke D, Ornston LN (2005) Opportunities for genetic investigation afforded by Acinetobacter baylyi, a nutritionally versatile bacterial species that is highly competent for natural transformation. Annu Rev Microbiol  59: 519–551.  <ul class="find" data-citedArticleID="1075502" data-doi="10.1146/annurev.micro.59.051905.105823"><li><a href="http://dx.doi.org/10.1146/annurev.micro.59.051905.105823" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Opportunities+for+genetic+investigation+afforded+by+Acinetobacter+baylyi%2C+a+nutritionally+versatile+bacterial+species+that+is+highly+competent+for+natural+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Opportunities+for+genetic+investigation+afforded+by+Acinetobacter+baylyi%2C+a+nutritionally+versatile+bacterial+species+that+is+highly+competent+for+natural+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001805-Smith1" id="pone.0001805-Smith1"></a>Smith MA, Weaver VB, Young DM, Ornston LN (2003) Genes for chlorogenate and hydroxycinnamate catabolism (hca) are linked to functionally related genes in the dca-pca-qui-pob-hca chromosomal cluster of Acinetobacter sp. strain ADP1. Appl Environ Microbiol  69: 524–532.  <ul class="find" data-citedArticleID="1075464" data-doi="10.1128/aem.69.1.524-532.2003"><li><a href="http://dx.doi.org/10.1128/aem.69.1.524-532.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+for+chlorogenate+and+hydroxycinnamate+catabolism+%28hca%29+are+linked+to+functionally+related+genes+in+the+dca-pca-qui-pob-hca+chromosomal+cluster+of+Acinetobacter+sp.+strain+ADP1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+for+chlorogenate+and+hydroxycinnamate+catabolism+%28hca%29+are+linked+to+functionally+related+genes+in+the+dca-pca-qui-pob-hca+chromosomal+cluster+of+Acinetobacter+sp.+strain+ADP1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001805-Dal1" id="pone.0001805-Dal1"></a>Dal S, Trautwein G, Gerischer U (2005) Transcriptional organization of genes for protocatechuate and quinate degradation from Acinetobacter sp. strain ADP1. Appl Environ Microbiol  71: 1025–1034.  <ul class="find" data-citedArticleID="1075398" data-doi="10.1128/aem.71.2.1025-1034.2005"><li><a href="http://dx.doi.org/10.1128/aem.71.2.1025-1034.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+organization+of+genes+for+protocatechuate+and+quinate+degradation+from+Acinetobacter+sp.+strain+ADP1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+organization+of+genes+for+protocatechuate+and+quinate+degradation+from+Acinetobacter+sp.+strain+ADP1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001805-Segura1" id="pone.0001805-Segura1"></a>Segura A, Bunz PV, D'Argenio DA, Ornston LN (1999) Genetic analysis of a chromosomal region containing vanA and vanB, genes required for conversion of either ferulate or vanillate to protocatechuate in Acinetobacter. J Bacteriol  181: 3494–3504.  <ul class="find" data-citedArticleID="1075460"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+analysis+of+a+chromosomal+region+containing+vanA+and+vanB%2C+genes+required+for+conversion+of+either+ferulate+or+vanillate+to+protocatechuate+in+Acinetobacter.&amp;auth=&amp;atitle=Genetic+analysis+of+a+chromosomal+region+containing+vanA+and+vanB%2C+genes+required+for+conversion+of+either+ferulate+or+vanillate+to+protocatechuate+in+Acinetobacter." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+analysis+of+a+chromosomal+region+containing+vanA+and+vanB%2C+genes+required+for+conversion+of+either+ferulate+or+vanillate+to+protocatechuate+in+Acinetobacter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+analysis+of+a+chromosomal+region+containing+vanA+and+vanB%2C+genes+required+for+conversion+of+either+ferulate+or+vanillate+to+protocatechuate+in+Acinetobacter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001805-Smith2" id="pone.0001805-Smith2"></a>Smith LT, Pocard JA, Bernard T, Le Rudulier D (1988) Osmotic control of glycine betaine biosynthesis and degradation in Rhizobium meliloti. J Bacteriol  170: 3142–3149.  <ul class="find" data-citedArticleID="1075466"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Osmotic+control+of+glycine+betaine+biosynthesis+and+degradation+in+Rhizobium+meliloti.&amp;auth=&amp;atitle=Osmotic+control+of+glycine+betaine+biosynthesis+and+degradation+in+Rhizobium+meliloti." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Osmotic+control+of+glycine+betaine+biosynthesis+and+degradation+in+Rhizobium+meliloti." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Osmotic+control+of+glycine+betaine+biosynthesis+and+degradation+in+Rhizobium+meliloti.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001805-vanderPloeg1" id="pone.0001805-vanderPloeg1"></a>van der Ploeg JR, Weiss MA, Saller E, Nashimoto H, Saito N, et al.  (1996) Identification of sulfate starvation-regulated genes in Escherichia coli: a gene cluster involved in the utilization of taurine as a sulfur source. J Bacteriol  178: 5438–5446.  <ul class="find" data-citedArticleID="1075510"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+sulfate+starvation-regulated+genes+in+Escherichia+coli%3A+a+gene+cluster+involved+in+the+utilization+of+taurine+as+a+sulfur+source.&amp;auth=&amp;atitle=Identification+of+sulfate+starvation-regulated+genes+in+Escherichia+coli%3A+a+gene+cluster+involved+in+the+utilization+of+taurine+as+a+sulfur+source." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+sulfate+starvation-regulated+genes+in+Escherichia+coli%3A+a+gene+cluster+involved+in+the+utilization+of+taurine+as+a+sulfur+source." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+sulfate+starvation-regulated+genes+in+Escherichia+coli%3A+a+gene+cluster+involved+in+the+utilization+of+taurine+as+a+sulfur+source.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001805-Wright1" id="pone.0001805-Wright1"></a>Wright CE, Tallan HH, Lin YY, Gaull GE (1986) Taurine: biological update. Annu Rev Biochem  55: 427–453.  <ul class="find" data-citedArticleID="1075500" data-doi="10.1146/annurev.bi.55.070186.002235"><li><a href="http://dx.doi.org/10.1146/annurev.bi.55.070186.002235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Taurine%3A+biological+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Taurine%3A+biological+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001805-Galan1" id="pone.0001805-Galan1"></a>Galan B, Garcia JL, Prieto MA (2004) The PaaX repressor, a link between penicillin G acylase and the phenylacetyl-coenzyme A catabolon of Escherichia coli W. J Bacteriol  186: 2215–2220.  <ul class="find" data-citedArticleID="1075418" data-doi="10.1128/jb.186.7.2215-2220.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.7.2215-2220.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PaaX+repressor%2C+a+link+between+penicillin+G+acylase+and+the+phenylacetyl-coenzyme+A+catabolon+of+Escherichia+coli+W." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PaaX+repressor%2C+a+link+between+penicillin+G+acylase+and+the+phenylacetyl-coenzyme+A+catabolon+of+Escherichia+coli+W.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001805-Merino1" id="pone.0001805-Merino1"></a>Merino E, Balbas P, Recillas F, Becerril B, Valle F, et al.  (1992) Carbon regulation and the role in nature of the Escherichia coli penicillin acylase (pac) gene. Mol Microbiol  6: 2175–2182.  <ul class="find" data-citedArticleID="1075446" data-doi="10.1111/j.1365-2958.1992.tb01391.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1992.tb01391.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carbon+regulation+and+the+role+in+nature+of+the+Escherichia+coli+penicillin+acylase+%28pac%29+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carbon+regulation+and+the+role+in+nature+of+the+Escherichia+coli+penicillin+acylase+%28pac%29+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001805-Bullen1" id="pone.0001805-Bullen1"></a>Bullen JJ, Rogers HJ, Griffiths E (1978) Role of iron in bacterial infection. Curr Top Microbiol Immunol  80: 1–35.  <ul class="find" data-citedArticleID="1075386"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Role+of+iron+in+bacterial+infection.&amp;auth=&amp;atitle=Role+of+iron+in+bacterial+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+iron+in+bacterial+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+iron+in+bacterial+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001805-Dorsey1" id="pone.0001805-Dorsey1"></a>Dorsey CW, Tomaras AP, Connerly PL, Tolmasky ME, Crosa JH, et al.  (2004) The siderophore-mediated iron acquisition systems of Acinetobacter baumannii ATCC 19606 and Vibrio anguillarum 775 are structurally and functionally related. Microbiology  150: 3657–3667.  <ul class="find" data-citedArticleID="1075408" data-doi="10.1099/mic.0.27371-0"><li><a href="http://dx.doi.org/10.1099/mic.0.27371-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+siderophore-mediated+iron+acquisition+systems+of+Acinetobacter+baumannii+ATCC+19606+and+Vibrio+anguillarum+775+are+structurally+and+functionally+related." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+siderophore-mediated+iron+acquisition+systems+of+Acinetobacter+baumannii+ATCC+19606+and+Vibrio+anguillarum+775+are+structurally+and+functionally+related.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001805-DiLorenzo1" id="pone.0001805-DiLorenzo1"></a>Di Lorenzo M, Stork M, Tolmasky ME, Actis LA, Farrell D, et al.  (2003) Complete sequence of virulence plasmid pJM1 from the marine fish pathogen Vibrio anguillarum strain 775. J Bacteriol  185: 5822–5830.  <ul class="find" data-citedArticleID="1075402" data-doi="10.1128/jb.185.19.5822-5830.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.19.5822-5830.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+sequence+of+virulence+plasmid+pJM1+from+the+marine+fish+pathogen+Vibrio+anguillarum+strain+775." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+sequence+of+virulence+plasmid+pJM1+from+the+marine+fish+pathogen+Vibrio+anguillarum+strain+775.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001805-Tolmasky1" id="pone.0001805-Tolmasky1"></a>Tolmasky ME, Actis LA, Crosa JH (1995) A histidine decarboxylase gene encoded by the Vibrio anguillarum plasmid pJM1 is essential for virulence: histamine is a precursor in the biosynthesis of anguibactin. Mol Microbiol  15: 87–95.  <ul class="find" data-citedArticleID="1075478" data-doi="10.1111/j.1365-2958.1995.tb02223.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1995.tb02223.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+histidine+decarboxylase+gene+encoded+by+the+Vibrio+anguillarum+plasmid+pJM1+is+essential+for+virulence%3A+histamine+is+a+precursor+in+the+biosynthesis+of+anguibactin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+histidine+decarboxylase+gene+encoded+by+the+Vibrio+anguillarum+plasmid+pJM1+is+essential+for+virulence%3A+histamine+is+a+precursor+in+the+biosynthesis+of+anguibactin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001805-Actis1" id="pone.0001805-Actis1"></a>Actis LA, Smoot JC, Barancin CE, Findlay RH (1999) Comparison of differential plating media and two chromatography techniques for the detection of histamine production in bacteria. J Microbiol Methods  39: 79–90.  <ul class="find" data-citedArticleID="1075368" data-doi="10.1016/s0167-7012(99)00099-8"><li><a href="http://dx.doi.org/10.1016/s0167-7012(99)00099-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+differential+plating+media+and+two+chromatography+techniques+for+the+detection+of+histamine+production+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+differential+plating+media+and+two+chromatography+techniques+for+the+detection+of+histamine+production+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001805-Fontecave1" id="pone.0001805-Fontecave1"></a>Fontecave M, Coves J, Pierre JL (1994) Ferric reductases or flavin reductases? Biometals  7: 3–8.  <ul class="find" data-citedArticleID="1075412" data-doi="10.1007/bf00205187"><li><a href="http://dx.doi.org/10.1007/bf00205187" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ferric+reductases+or+flavin+reductases%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ferric+reductases+or+flavin+reductases%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001805-Houhamdi1" id="pone.0001805-Houhamdi1"></a>Houhamdi L, Raoult D (2006) Experimental infection of human body lice with Acinetobacter baumannii. Am J Trop Med Hyg  74: 526–531.  <ul class="find" data-citedArticleID="1075428"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Experimental+infection+of+human+body+lice+with+Acinetobacter+baumannii.&amp;auth=&amp;atitle=Experimental+infection+of+human+body+lice+with+Acinetobacter+baumannii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+infection+of+human+body+lice+with+Acinetobacter+baumannii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+infection+of+human+body+lice+with+Acinetobacter+baumannii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001805-Wandersman1" id="pone.0001805-Wandersman1"></a>Wandersman C, Delepelaire P (2004) BACTERIAL IRON SOURCES: From Siderophores to Hemophores. Annual Review of Microbiology  58: 611–647.  <ul class="find" data-citedArticleID="1075490" data-doi="10.1146/annurev.micro.58.030603.123811"><li><a href="http://dx.doi.org/10.1146/annurev.micro.58.030603.123811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BACTERIAL+IRON+SOURCES%3A+From+Siderophores+to+Hemophores." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BACTERIAL+IRON+SOURCES%3A+From+Siderophores+to+Hemophores.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001805-Andrews1" id="pone.0001805-Andrews1"></a>Andrews SC, Robinson AK, Rodriguez-Quinones F (2003) Bacterial iron homeostasis. FEMS Microbiol Rev  27: 215–237.  <ul class="find" data-citedArticleID="1075370" data-doi="10.1016/s0168-6445(03)00055-x"><li><a href="http://dx.doi.org/10.1016/s0168-6445(03)00055-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+iron+homeostasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+iron+homeostasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001805-Welch1" id="pone.0001805-Welch1"></a>Welch TJ, Chai S, Crosa JH (2000) The overlapping angB and angG genes are encoded within the trans-acting factor region of the virulence plasmid in Vibrio anguillarum: essential role in siderophore biosynthesis. J Bacteriol  182: 6762–6773.  <ul class="find" data-citedArticleID="1075492" data-doi="10.1128/jb.182.23.6762-6773.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.23.6762-6773.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+overlapping+angB+and+angG+genes+are+encoded+within+the+trans-acting+factor+region+of+the+virulence+plasmid+in+Vibrio+anguillarum%3A+essential+role+in+siderophore+biosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+overlapping+angB+and+angG+genes+are+encoded+within+the+trans-acting+factor+region+of+the+virulence+plasmid+in+Vibrio+anguillarum%3A+essential+role+in+siderophore+biosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001805-Stork1" id="pone.0001805-Stork1"></a>Stork M, Di Lorenzo M, Welch TJ, Crosa LM, Crosa JH (2002) Plasmid-mediated iron uptake and virulence in Vibrio anguillarum. Plasmid  48: 222–228.  <ul class="find" data-citedArticleID="1075472" data-doi="10.1016/s0147-619x(02)00111-7"><li><a href="http://dx.doi.org/10.1016/s0147-619x(02)00111-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmid-mediated+iron+uptake+and+virulence+in+Vibrio+anguillarum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmid-mediated+iron+uptake+and+virulence+in+Vibrio+anguillarum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001805-Lin1" id="pone.0001805-Lin1"></a>Lin J, Huang S, Zhang Q (2002) Outer membrane proteins: key players for bacterial adaptation in host niches. Microbes Infect  4: 325–331.  <ul class="find" data-citedArticleID="1075440" data-doi="10.1016/s1286-4579(02)01545-9"><li><a href="http://dx.doi.org/10.1016/s1286-4579(02)01545-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Outer+membrane+proteins%3A+key+players+for+bacterial+adaptation+in+host+niches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Outer+membrane+proteins%3A+key+players+for+bacterial+adaptation+in+host+niches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001805-Walzer1" id="pone.0001805-Walzer1"></a>Walzer G, Rosenberg E, Ron EZ (2006) The Acinetobacter outer membrane protein A (OmpA) is a secreted emulsifier. Environ Microbiol  8: 1026–1032.  <ul class="find" data-citedArticleID="1075488" data-doi="10.1111/j.1462-2920.2006.00994.x"><li><a href="http://dx.doi.org/10.1111/j.1462-2920.2006.00994.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Acinetobacter+outer+membrane+protein+A+%28OmpA%29+is+a+secreted+emulsifier." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Acinetobacter+outer+membrane+protein+A+%28OmpA%29+is+a+secreted+emulsifier.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001805-Choi1" id="pone.0001805-Choi1"></a>Choi CH, Lee EY, Lee YC, Park TI, Kim HJ, et al.  (2005) Outer membrane protein 38 of Acinetobacter baumannii localizes to the mitochondria and induces apoptosis of epithelial cells. Cell Microbiol  7: 1127–1138.  <ul class="find" data-citedArticleID="1075388" data-doi="10.1111/j.1462-5822.2005.00538.x"><li><a href="http://dx.doi.org/10.1111/j.1462-5822.2005.00538.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Outer+membrane+protein+38+of+Acinetobacter+baumannii+localizes+to+the+mitochondria+and+induces+apoptosis+of+epithelial+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Outer+membrane+protein+38+of+Acinetobacter+baumannii+localizes+to+the+mitochondria+and+induces+apoptosis+of+epithelial+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001805-OforiDarko1" id="pone.0001805-OforiDarko1"></a>Ofori-Darko E, Zavros Y, Rieder G, Tarle SA, Van Antwerp M, et al.  (2000) An OmpA-like protein from Acinetobacter spp. stimulates gastrin and interleukin-8 promoters. Infect Immun  68: 3657–3666.  <ul class="find" data-citedArticleID="1075450" data-doi="10.1128/iai.68.6.3657-3666.2000"><li><a href="http://dx.doi.org/10.1128/iai.68.6.3657-3666.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+OmpA-like+protein+from+Acinetobacter+spp.+stimulates+gastrin+and+interleukin-8+promoters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+OmpA-like+protein+from+Acinetobacter+spp.+stimulates+gastrin+and+interleukin-8+promoters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001805-Titball1" id="pone.0001805-Titball1"></a>Titball RW (1993) Bacterial phospholipases C. Microbiol Rev  57: 347–366.  <ul class="find" data-citedArticleID="1075476"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bacterial+phospholipases+C.&amp;auth=&amp;atitle=Bacterial+phospholipases+C." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+phospholipases+C." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+phospholipases+C.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001805-McKean1" id="pone.0001805-McKean1"></a>McKean SC, Davies JK, Moore RJ (2007) Expression of phospholipase D, the major virulence factor of Corynebacterium pseudotuberculosis, is regulated by multiple environmental factors and plays a role in macrophage death. Microbiology  153: 2203–2211.  <ul class="find" data-citedArticleID="1075444" data-doi="10.1099/mic.0.2007/005926-0"><li><a href="http://dx.doi.org/10.1099/mic.0.2007/005926-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+phospholipase+D%2C+the+major+virulence+factor+of+Corynebacterium+pseudotuberculosis%2C+is+regulated+by+multiple+environmental+factors+and+plays+a+role+in+macrophage+death." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+phospholipase+D%2C+the+major+virulence+factor+of+Corynebacterium+pseudotuberculosis%2C+is+regulated+by+multiple+environmental+factors+and+plays+a+role+in+macrophage+death.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001805-Smith3" id="pone.0001805-Smith3"></a>Smith MG, Gianoulis TA, Pukatzki S, Mekalanos JJ, Ornston LN, et al.  (2007) New insights into Acinetobacter baumannii pathogenesis revealed by high-density pyrosequencing and transposon mutagenesis. Genes Dev  21: 601–614.  <ul class="find" data-citedArticleID="1075468" data-doi="10.1101/gad.1510307"><li><a href="http://dx.doi.org/10.1101/gad.1510307" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+insights+into+Acinetobacter+baumannii+pathogenesis+revealed+by+high-density+pyrosequencing+and+transposon+mutagenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+insights+into+Acinetobacter+baumannii+pathogenesis+revealed+by+high-density+pyrosequencing+and+transposon+mutagenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001805-Smith4" id="pone.0001805-Smith4"></a>Smith MG, Des Etages SG, Snyder M (2004) Microbial synergy via an ethanol-triggered pathway. Mol Cell Biol  24: 3874–3884.  <ul class="find" data-citedArticleID="1075470" data-doi="10.1128/mcb.24.9.3874-3884.2004"><li><a href="http://dx.doi.org/10.1128/mcb.24.9.3874-3884.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+synergy+via+an+ethanol-triggered+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+synergy+via+an+ethanol-triggered+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001805-Siguier1" id="pone.0001805-Siguier1"></a>Siguier P, Perochon J, Lestrade L, Mahillon J, Chandler M (2006) ISfinder: the reference centre for bacterial insertion sequences. Nucleic Acids Res  34: D32–36.  <ul class="find" data-citedArticleID="1075462" data-doi="10.1093/nar/gkj014"><li><a href="http://dx.doi.org/10.1093/nar/gkj014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ISfinder%3A+the+reference+centre+for+bacterial+insertion+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ISfinder%3A+the+reference+centre+for+bacterial+insertion+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001805-Achaz1" id="pone.0001805-Achaz1"></a>Achaz G, Boyer F, Rocha EP, Viari A, Coissac E (2007) Repseek, a tool to retrieve approximate repeats from large DNA sequences. Bioinformatics  23: 119–121.  <ul class="find" data-citedArticleID="1075366" data-doi="10.1093/bioinformatics/btl519"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl519" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repseek%2C+a+tool+to+retrieve+approximate+repeats+from+large+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repseek%2C+a+tool+to+retrieve+approximate+repeats+from+large+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001805-Bocs1" id="pone.0001805-Bocs1"></a>Bocs S, Cruveiller S, Vallenet D, Nuel G, Medigue C (2003) AMIGene: Annotation of MIcrobial Genes. Nucleic Acids Res  31: 3723–3726.  <ul class="find" data-citedArticleID="1075378" data-doi="10.1093/nar/gkg590"><li><a href="http://dx.doi.org/10.1093/nar/gkg590" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AMIGene%3A+Annotation+of+MIcrobial+Genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AMIGene%3A+Annotation+of+MIcrobial+Genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001805-Overbeek1" id="pone.0001805-Overbeek1"></a>Overbeek R, Fonstein M, D'Souza M, Pusch GD, Maltsev N (1999) The use of gene clusters to infer functional coupling. Proc Natl Acad Sci U S A  96: 2896–2901.  <ul class="find" data-citedArticleID="1075452" data-doi="10.1073/pnas.96.6.2896"><li><a href="http://dx.doi.org/10.1073/pnas.96.6.2896" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+gene+clusters+to+infer+functional+coupling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+gene+clusters+to+infer+functional+coupling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001805-Boyer1" id="pone.0001805-Boyer1"></a>Boyer F, Morgat A, Labarre L, Pothier J, Viari A (2005) Syntons, metabolons and interactons: an exact graph-theoretical approach for exploring neighbourhood between genomic and functional data. Bioinformatics  21: 4209–4215.  <ul class="find" data-citedArticleID="1075384" data-doi="10.1093/bioinformatics/bti711"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti711" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Syntons%2C+metabolons+and+interactons%3A+an+exact+graph-theoretical+approach+for+exploring+neighbourhood+between+genomic+and+functional+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Syntons%2C+metabolons+and+interactons%3A+an+exact+graph-theoretical+approach+for+exploring+neighbourhood+between+genomic+and+functional+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001805-Vallenet1" id="pone.0001805-Vallenet1"></a>Vallenet D, Labarre L, Rouy Z, Barbe V, Bocs S, et al.  (2006) MaGe: a microbial genome annotation system supported by synteny results. Nucleic Acids Res  34: 53–65.  <ul class="find" data-citedArticleID="1075482" data-doi="10.1093/nar/gkj406"><li><a href="http://dx.doi.org/10.1093/nar/gkj406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MaGe%3A+a+microbial+genome+annotation+system+supported+by+synteny+results." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MaGe%3A+a+microbial+genome+annotation+system+supported+by+synteny+results.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001805-Edgar1" id="pone.0001805-Edgar1"></a>Edgar RC (2004) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res  32: 1792–1797.  <ul class="find" data-citedArticleID="1075410" data-doi="10.1093/nar/gkh340"><li><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001805-Guindon1" id="pone.0001805-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol  52: 696–704.  <ul class="find" data-citedArticleID="1075424"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001805-Clamp1" id="pone.0001805-Clamp1"></a>Clamp M, Cuff J, Searle SM, Barton GJ (2004) The Jalview Java alignment editor. Bioinformatics  20: 426–427.  <ul class="find" data-citedArticleID="1075390" data-doi="10.1093/bioinformatics/btg430"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg430" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Jalview+Java+alignment+editor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Jalview+Java+alignment+editor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.XML" value="161773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.PDF" value="276862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t001.PNG_L" value="126810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t001.PNG_M" value="128601"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t001.PNG_S" value="18042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t001.TIF" value="659118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t001.PNG_I" value="63674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g001.PNG_L" value="451203"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g001.PNG_M" value="69626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g001.PNG_S" value="6527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g001.TIF" value="1004524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g001.PNG_I" value="24899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g002.PNG_L" value="612783"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g002.PNG_M" value="88493"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g002.PNG_S" value="11355"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g002.TIF" value="1850070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g002.PNG_I" value="48540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g003.PNG_L" value="2047563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g003.PNG_M" value="139267"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g003.PNG_S" value="7179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g003.TIF" value="2185598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.g003.PNG_I" value="46494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t002.PNG_L" value="751834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t002.PNG_M" value="130958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t002.PNG_S" value="20516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t002.TIF" value="3659690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t002.PNG_I" value="165836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t003.PNG_L" value="278510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t003.PNG_M" value="162373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t003.PNG_S" value="15164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t003.TIF" value="1183074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.t003.PNG_I" value="49871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.s001.TIF" value="2562579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.s002.TIF" value="3020273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001805.s003.XLS" value="18944"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001805&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001805" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001805&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001805&volume=&issue=&title=Comparative Analysis of Acinetobacters: Three Genomes for Three Lifestyles&author_name=David%20Vallenet%2C%20Patrice%20Nordmann%2C%20Val%C3%A9rie%20Barbe%2C%20Laurent%20Poirel%2C%20Sophie%20Mangenot%2C%20Elodie%20Bataille%2C%20Carole%20Dossat%2C%20Shahinaz%20Gas%2C%20Annett%20Kreimeyer%2C%20Patricia%20Lenoble%2C%20Sophie%20Oztas%2C%20Julie%20Poulain%2C%20B%C3%A9atrice%20Segurens%2C%20Catherine%20Robert%2C%20Chantal%20Abergel%2C%20Jean-Michel%20Claverie%2C%20Didier%20Raoult%2C%20Claudine%20M%C3%A9digue%2C%20Jean%20Weissenbach%2C%20St%C3%A9phane%20Cruveiller&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001805" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001805" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001805" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001805&amp;t=Comparative%20Analysis%20of%20Acinetobacters%3A%20Three%20Genomes%20for%20Three%20Lifestyles" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001805&title=Comparative%20Analysis%20of%20Acinetobacters%3A%20Three%20Genomes%20for%20Three%20Lifestyles&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001805&amp;title=Comparative%20Analysis%20of%20Acinetobacters%3A%20Three%20Genomes%20for%20Three%20Lifestyles" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001805" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001805&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Comparative Analysis of Acinetobacters: Three Genomes for Three Lifestyles';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001805';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Comparative%20Analysis%20of%20Acinetobacters%3A%20Three%20Genomes%20for%20Three%20Lifestyles http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001805" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001805" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->

<div class="block">
    <div class="header">
        <h3>Included in the Following Collection</h3>
    </div>
    <p><dd><a href="http://www.ploscollections.org/article/browse/issue/info%3Adoi%2F10.1371%2Fissue.pcol.v02.i03" title="Browse the Open-Access Collection"><em>PLoS ONE</em>: Prokaryotic Genome Collection</a></dd></p>
</div>

<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Acinetobacter%22" title="Search for articles in the subject area:'Acinetobacter'"><div class="flagText">Acinetobacter</div></a>
              <div data-categoryid="44221" data-articleid="26434"
                   data-categoryname="Acinetobacter"
                   class="flagImage" title="Flag 'Acinetobacter' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Acinetobacter+baumannii%22" title="Search for articles in the subject area:'Acinetobacter baumannii'"><div class="flagText">Acinetobacter baumannii</div></a>
              <div data-categoryid="44843" data-articleid="26434"
                   data-categoryname="Acinetobacter baumannii"
                   class="flagImage" title="Flag 'Acinetobacter baumannii' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Acinetobacter+infections%22" title="Search for articles in the subject area:'Acinetobacter infections'"><div class="flagText">Acinetobacter infections</div></a>
              <div data-categoryid="50727" data-articleid="26434"
                   data-categoryname="Acinetobacter infections"
                   class="flagImage" title="Flag 'Acinetobacter infections' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="26434"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+pathogens%22" title="Search for articles in the subject area:'Bacterial pathogens'"><div class="flagText">Bacterial pathogens</div></a>
              <div data-categoryid="42717" data-articleid="26434"
                   data-categoryname="Bacterial pathogens"
                   class="flagImage" title="Flag 'Bacterial pathogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="26434"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Comparative+genomics%22" title="Search for articles in the subject area:'Comparative genomics'"><div class="flagText">Comparative genomics</div></a>
              <div data-categoryid="32937" data-articleid="26434"
                   data-categoryname="Comparative genomics"
                   class="flagImage" title="Flag 'Comparative genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="26434"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=261'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=430'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8543&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
