

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Role for MRE11, NBS1, and Recombination Junctions in Replication and Stable Maintenance of EBV Episomes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001257"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001257" />

    <meta name="citation_title" content="A Role for MRE11, NBS1, and Recombination Junctions in Replication and Stable Maintenance of EBV Episomes"/>
    <meta itemprop="name" content="A Role for MRE11, NBS1, and Recombination Junctions in Replication and Stable Maintenance of EBV Episomes"/>

      <meta name="citation_author" content="Jayaraju Dheekollu"/>
            <meta name="citation_author_institution" content="The Wistar Institute, Philadelphia, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Zhong Deng"/>
            <meta name="citation_author_institution" content="The Wistar Institute, Philadelphia, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Andreas Wiedmer"/>
            <meta name="citation_author_institution" content="The Wistar Institute, Philadelphia, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Matthew D. Weitzman"/>
            <meta name="citation_author_institution" content="The Salk Institute, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Paul M. Lieberman"/>
            <meta name="citation_author_institution" content="The Wistar Institute, Philadelphia, Pennsylvania, United States of America"/>

    <meta name="citation_date" content="2007/12/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001257.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1257"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Maintenance of genome stability in Saccharomyces cerevisiae.; citation_author=RD Kolodner; citation_author=CD Putnam; citation_author=K Myung; citation_journal_title=Science; citation_volume=297; citation_number=1; citation_pages=552-557; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Multiple interlinked mechanisms to circumvent DNA replication roadblocks.; citation_author=B Eppink; citation_author=C Wyman; citation_author=R Kanaar; citation_journal_title=Exp Cell Res; citation_volume=312; citation_number=2; citation_pages=2660-2665; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Surviving the Breakup: The DNA Damage Checkpoint.; citation_author=JC Harrison; citation_author=JE Haber; citation_journal_title=Annu Rev Genet; citation_number=3; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sensing and repairing DNA double-strand breaks.; citation_author=SP Jackson; citation_journal_title=Carcinogenesis; citation_volume=23; citation_number=4; citation_pages=687-696; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The DNA damage response during DNA replication.; citation_author=D Branzei; citation_author=M Foiani; citation_journal_title=Curr Opin Cell Biol; citation_volume=17; citation_number=5; citation_pages=568-575; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Mre11 complex and the metabolism of chromosome breaks: the importance of communicating and holding things together.; citation_author=TH Stracker; citation_author=JW Theunissen; citation_author=M Morales; citation_author=JH Petrini; citation_journal_title=DNA Repair (Amst); citation_volume=3; citation_number=6; citation_pages=845-854; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=BRCA1 and BRCA2: 1994 and beyond.; citation_author=SA Narod; citation_author=WD Foulkes; citation_journal_title=Nat Rev Cancer; citation_volume=4; citation_number=7; citation_pages=665-676; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Epstein-Barr virus: 40 years on.; citation_author=LS Young; citation_author=AB Rickinson; citation_journal_title=Nat Rev Cancer; citation_volume=4; citation_number=8; citation_pages=757-768; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Epstein-Barr virus and its replication.; citation_author=E Kieff; citation_number=9; citation_pages=2343-2396; citation_date=1996; citation_publisher=Lippincott-Raven Publishers; " />
      <meta name="citation_reference" content="citation_title=Epstein-Barr Virus.; citation_author=AB Rickinson; citation_author=E Kieff; citation_number=10; citation_pages=2397-2446; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Epstein-Barr virus-derived plasmids replicate only once per cell cycle and are not amplified after entry into cells.; citation_author=JL Yates; citation_author=N Guan; citation_journal_title=J Virol; citation_volume=65; citation_number=11; citation_pages=483-488; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Molecular mechanisms of maintenance and disruption of virus latency.; citation_author=B Sugden; citation_author=ER Leight; citation_number=12; citation_pages=3-11; citation_date=2001; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Origins of bidirectional replication of Epstein-Barr virus: Models for understanding mammalian origins of DNA synthesis.; citation_author=J Wang; citation_author=B Sugden; citation_journal_title=J Cell Biochem; citation_volume=94; citation_number=13; citation_pages=247-256; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Human DNA replication initiation factors, ORC and MCM, associate with oriP of Epstein-Barr virus.; citation_author=B Chaudhuri; citation_author=H Xu; citation_author=I Todorov; citation_author=A Dutta; citation_author=JL Yates; citation_journal_title=Proc Natl Acad Sci, U S A; citation_volume=98; citation_number=14; citation_pages=10085-10089; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Replication from oriP of Epstein-Barr virus requires human ORC and is inhibited by geminin.; citation_author=SK Dhar; citation_author=K Yoshida; citation_author=Y Machida; citation_author=P Khaira; citation_author=B Chaudhuri; citation_journal_title=Cell; citation_volume=106; citation_number=15; citation_pages=287-296; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Human origin recognition complex binds to the region of the latent origin of DNA replication of Epstein-Barr virus.; citation_author=A Schepers; citation_author=M Ritzi; citation_author=K Bousset; citation_author=E Kremmer; citation_author=JL Yates; citation_journal_title=EMBO J; citation_volume=20; citation_number=16; citation_pages=4588-4602; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sequence-specific DNA binding of the Epstein-Barr virus nuclear antigen (EBNA-1) to clustered sites in the plasmid maintenance region.; citation_author=DR Rawlins; citation_author=G Milman; citation_author=SD Hayward; citation_author=GS Hayward; citation_journal_title=Cell; citation_volume=42; citation_number=17; citation_pages=859-868; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=A cis-acting element from Epstein-Barr viral genome that permits stable replication of recombinant plasmids in latently infected cells.; citation_author=JL Yates; citation_author=N Warren; citation_author=P Reisman; citation_author=B Sugden; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=81; citation_number=18; citation_pages=3806-3810; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Genetic evidence that EBNA-1 is needed for efficient, stable latent infection by Epstein-Barr virus.; citation_author=MA Lee; citation_author=ME Diamond; citation_author=JL Yates; citation_journal_title=J Virol; citation_volume=73; citation_number=19; citation_pages=2974-2982; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The EBV nuclear antigen 1 (EBNA1) enhances B cell immortalization several thousandfold.; citation_author=S Humme; citation_author=G Reisbach; citation_author=R Feederle; citation_author=HJ Delecluse; citation_author=K Bousset; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=20; citation_pages=10989-10994; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Telomeric proteins regulate episomal maintenance of Epstein-Barr virus origin of plasmid replication.; citation_author=Z Deng; citation_author=L Lezina; citation_author=CJ Chen; citation_author=S Shtivelband; citation_author=W So; citation_journal_title=Mol Cell; citation_volume=9; citation_number=21; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Telomere repeat binding factors TRF1, TRF2, and hRAP1 modulate replication of Epstein-Barr virus OriP.; citation_author=Z Deng; citation_author=C Atanasiu; citation_author=JS Burg; citation_author=D Broccoli; citation_author=PM Lieberman; citation_journal_title=J Virol; citation_volume=77; citation_number=22; citation_pages=11992-12001; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The minimal replicator of Epstein-Barr virus oriP.; citation_author=JL Yates; citation_author=SM Camiolo; citation_author=JM Bashaw; citation_journal_title=J Virol; citation_volume=74; citation_number=23; citation_pages=4512-4522; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification of new human origins of DNA replication by an origin-trapping assay.; citation_author=J Gerhardt; citation_author=S Jafar; citation_author=MP Spindler; citation_author=E Ott; citation_author=A Schepers; citation_journal_title=Mol Cell Biol; citation_volume=26; citation_number=24; citation_pages=7731-7746; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Plasmid maintenance assay based on green fluorescent protein and FACS of mammalian cells.; citation_author=M Vogel; citation_author=K Wittmann; citation_author=E Endl; citation_author=G Glaser; citation_author=R Knuchel; citation_journal_title=BioTechniques; citation_volume=24; citation_number=25; citation_pages=540-544; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Nucleoprotein complexes and DNA 5′-ends at oriP of Epstein-Barr virus.; citation_author=HH Niller; citation_author=G Glaser; citation_author=R Knuchel; citation_author=H Wolf; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=26; citation_pages=12864-12868; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Protein and sequence requirements for the recruitment of the human origin recognition complex to the latent cycle origin of DNA replication of Epstein-Barr virus oriP.; citation_author=MD Julien; citation_author=Z Polonskaya; citation_author=J Hearing; citation_journal_title=Virology; citation_volume=326; citation_number=27; citation_pages=317-328; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=ORC binding to TRF2 stimulates OriP replication.; citation_author=C Atanasiu; citation_author=Z Deng; citation_author=A Wiedmer; citation_author=J Norseen; citation_author=PM Lieberman; citation_journal_title=EMBO Rep; citation_volume=7; citation_number=28; citation_pages=716-721; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Protection of mammalian telomeres.; citation_author=T de Lange; citation_journal_title=Oncogene; citation_volume=21; citation_number=29; citation_pages=532-540; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=T-loops and the origin of telomeres.; citation_author=T de Lange; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=5; citation_number=30; citation_pages=323-329; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mammalian telomeres end in a large duplex loop.; citation_author=JD Griffith; citation_author=L Comeau; citation_author=S Rosenfield; citation_author=RM Stansel; citation_author=A Bianchi; citation_journal_title=Cell; citation_volume=97; citation_number=31; citation_pages=503-514; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=T-loop assembly in vitro involves binding of TRF2 near the 3′ telomeric overhang.; citation_author=RM Stansel; citation_author=R de Lange; citation_author=JD Griffith; citation_journal_title=EMBO J; citation_volume=20; citation_number=32; citation_pages=5532-5540; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Homologous recombination generates T-loop-sized deletions at human telomeres.; citation_author=RC Wang; citation_author=A Smogorzewska; citation_author=T de Lange; citation_journal_title=Cell; citation_volume=119; citation_number=33; citation_pages=355-368; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The basic domain of TRF2 directs binding to DNA junctions irrespective of the presence of TTAGGG repeats.; citation_author=N Fouche; citation_author=AJ Cesare; citation_author=S Willcox; citation_author=S Ozgur; citation_author=SA Compton; citation_journal_title=J Biol Chem; citation_number=34; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Shelterin: the protein complex that shapes and safeguards human telomeres.; citation_author=T de Lange; citation_journal_title=Genes Dev; citation_volume=19; citation_number=35; citation_pages=2100-2110; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cell-cycle-regulated association of RAD50/MRE11/NBS1 with TRF2 and human telomeres.; citation_author=X-D Zhu; citation_author=B Kuster; citation_author=M Mann; citation_author=JHJ Petrini; citation_author=T de Lange; citation_journal_title=Nat Genet; citation_volume=25; citation_number=36; citation_pages=347-352; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Functional links between telomeres and proteins of the DNA-damage response.; citation_author=F d'Adda di Fagagna; citation_author=SH Teo; citation_author=SP Jackson; citation_journal_title=Genes Dev; citation_volume=18; citation_number=37; citation_pages=1781-1799; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Telomeres and the DNA damage response: why the fox is guarding the henhouse.; citation_author=RS Maser; citation_author=RA DePinho; citation_journal_title=DNA Repair (Amst); citation_volume=3; citation_number=38; citation_pages=979-988; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mre11 complex and DNA replication: linkage to E2F and sites of DNA synthesis.; citation_author=RS Maser; citation_author=OK Mirzoeva; citation_author=J Wells; citation_author=H Olivares; citation_author=BR Williams; citation_journal_title=Mol Cell Biol; citation_volume=21; citation_number=39; citation_pages=6006-6016; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Branch migrating sister chromatid junctions form at replication origins through Rad51/Rad52-independent mechanisms.; citation_author=M Lopes; citation_author=C Cotta-Ramusino; citation_author=G Liberi; citation_author=M Foiani; citation_journal_title=Mol Cell; citation_volume=12; citation_number=40; citation_pages=1499-1510; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Increased recombination intermediates and homologous integration hot spots at DNA replication origins.; citation_author=M Segurado; citation_author=M Gomez; citation_author=F Antequera; citation_journal_title=Mol Cell; citation_volume=10; citation_number=41; citation_pages=907-916; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=DNA replication-dependent formation of joint DNA molecules in Physarum polycephalum.; citation_author=M Benard; citation_author=C Maric; citation_author=G Pierron; citation_journal_title=Mol Cell; citation_volume=7; citation_number=42; citation_pages=971-980; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Adenovirus oncoproteins inactivate the Mre11-Rad50-NBS1 DNA repair complex.; citation_author=TH Stracker; citation_author=CT Carson; citation_author=MD Weitzman; citation_journal_title=Nature; citation_volume=418; citation_number=43; citation_pages=348-352; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Epstein-Barr virus integration in human lymphomas and lymphoid cell lines.; citation_author=ML Gulley; citation_author=M Raphael; citation_author=CT Lutz; citation_author=DW Ross; citation_author=N Raab-Traub; citation_journal_title=Cancer; citation_volume=70; citation_number=44; citation_pages=185-191; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Visualization of DNA replication on individual Epstein-Barr virus episomes.; citation_author=P Norio; citation_author=CL Schildkraut; citation_journal_title=Science; citation_volume=294; citation_number=45; citation_pages=2361-2364; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A method for preparing genomic DNA that restrains branch migration of Holliday junctions.; citation_author=T Allers; citation_author=M Lichten; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=46; citation_pages=e6; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=T7 endonuclease I resolves Holliday junctions formed in vitro by RecA protein.; citation_author=B Muller; citation_author=C Jones; citation_author=SC West; citation_journal_title=Nucleic Acids Res; citation_volume=18; citation_number=47; citation_pages=5633-5636; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Checking on the fork: the DNA-replication stress-response pathway.; citation_author=AJ Osborn; citation_author=SJ Elledge; citation_author=L Zou; citation_journal_title=Trends Cell Biol; citation_volume=12; citation_number=48; citation_pages=509-516; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Break-induced replication: a review and an example in budding yeast.; citation_author=E Kraus; citation_author=WY Leung; citation_author=JE Haber; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=49; citation_pages=8255-8262; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=ATM and ATR promote Mre11 dependent restart of collapsed replication forks and prevent accumulation of DNA breaks.; citation_author=K Trenz; citation_author=E Smith; citation_author=S Smith; citation_author=V Costanzo; citation_journal_title=Embo J; citation_volume=25; citation_number=50; citation_pages=1764-1774; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA repair proteins affect the lifecycle of herpes simplex virus 1.; citation_author=CE Lilley; citation_author=CT Carson; citation_author=AR Muotri; citation_author=FH Gage; citation_author=MD Weitzman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=51; citation_pages=5844-5849; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A new connection at human telomeres: association of the Mre11 complex with TRF2.; citation_author=T de Lange; citation_author=JH Petrini; citation_journal_title=Cold Spring Harb Symp Quant Biol; citation_volume=65; citation_number=52; citation_pages=265-273; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The Epstein-Barr virus origin of plasmid replication, oriP, contains both the initiation and termination sites of DNA replication.; citation_author=TA Gahn; citation_author=CL Schildkraut; citation_journal_title=Cell; citation_volume=58; citation_number=53; citation_pages=527-535; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Initiation of latent DNA replication in the Epstein-Barr virus genome can occur at sites other than the genetically defined origin.; citation_author=RD Little; citation_author=CL Schildkraut; citation_journal_title=Mol Cell Biol; citation_volume=15; citation_number=54; citation_pages=2893-2903; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Role of the EBNA-1 protein in pausing of replication forks in the Epstein-Barr virus genome.; citation_author=OV Ermakova; citation_author=L Frappier; citation_author=CL Schildkraut; citation_journal_title=J Biol Chem; citation_volume=271; citation_number=55; citation_pages=33009-33017; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Role of EBNA-1 in arresting replication forks at the Epstein-Barr virus oriP family of tandem repeats.; citation_author=V Dhar; citation_author=CL Schildkraut; citation_journal_title=Mol Cell Biol; citation_volume=11; citation_number=56; citation_pages=6268-6278; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Plasticity of DNA replication initiation in Epstein-Barr virus episomes.; citation_author=P Norio; citation_author=CL Schildkraut; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=57; citation_pages=e152; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Methods to study replication fork collapse in budding yeast.; citation_author=G Liberi; citation_author=C Cotta-Ramusino; citation_author=M Lopes; citation_author=J Sogo; citation_author=C Conti; citation_journal_title=Methods Enzymol; citation_volume=409; citation_number=58; citation_pages=442-462; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the Holliday junction resolving enzyme T7 endonuclease I.; citation_author=JM Hadden; citation_author=MA Convery; citation_author=AC Declais; citation_author=DM Lilley; citation_author=SE Phillips; citation_journal_title=Nat Struct Biol; citation_volume=8; citation_number=59; citation_pages=62-67; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The amino terminus of Epstein-Barr Virus (EBV) nuclear antigen 1 contains AT hooks that facilitate the replication and partitioning of latent EBV genomes by tethering them to cellular chromosomes.; citation_author=J Sears; citation_author=M Ujihara; citation_author=S Wong; citation_author=C Ott; citation_author=J Middeldorp; citation_journal_title=J Virol; citation_volume=78; citation_number=60; citation_pages=11487-11505; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Efficient persistence of extrachromosomal KSHV DNA mediated by latency-associated nuclear antigen.; citation_author=ME Ballestas; citation_author=PA Chatis; citation_author=KM Kaye; citation_journal_title=Science; citation_volume=284; citation_number=61; citation_pages=641-644; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=EBV's plasmid replicon: an enigma in cis and trans.; citation_author=B Sugden; citation_author=ER Leight; citation_journal_title=Curr Top Microbiol Immunol; citation_volume=258; citation_number=62; citation_pages=3-11; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=EBP2 plays a key role in Epstein-Barr virus mitotic segregation and is regulated by aurora family kinases.; citation_author=P Kapoor; citation_author=BD Lavoie; citation_author=L Frappier; citation_journal_title=Mol Cell Biol; citation_volume=25; citation_number=63; citation_pages=4934-4945; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Symmetrical localization of extrachromosomally replicating viral genomes on sister chromatids.; citation_author=T Kanda; citation_author=M Kamiya; citation_author=S Maruo; citation_author=D Iwakiri; citation_author=K Takada; citation_journal_title=J Cell Sci; citation_volume=120; citation_number=64; citation_pages=1529-1539; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Nbs1 is essential for DNA repair by homologous recombination in higher vertebrate cells.; citation_author=H Tauchi; citation_author=J Kobayashi; citation_author=K Morishima; citation_author=DC van Gent; citation_author=T Shiraishi; citation_journal_title=Nature; citation_volume=420; citation_number=65; citation_pages=93-98; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=From a single double helix to paired double helices and back.; citation_author=K Nasmyth; citation_author=A Schleiffer; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=359; citation_number=66; citation_pages=99-108; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The origin recognition complex functions in sister-chromatid cohesion in Saccharomyces cerevisiae.; citation_author=K Shimada; citation_author=SM Gasser; citation_journal_title=Cell; citation_volume=128; citation_number=67; citation_pages=85-99; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Complex protein-DNA dynamics at the latent origin of DNA replication of Epstein-Barr virus.; citation_author=M Ritzi; citation_author=K Tillack; citation_author=J Gerhardt; citation_author=E Ott; citation_author=S Humme; citation_journal_title=J Cell Sci; citation_volume=116; citation_number=68; citation_pages=3971-3984; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cell cycle regulation of chromatin at an origin of DNA replication.; citation_author=J Zhou; citation_author=CM Chau; citation_author=Z Deng; citation_author=R Shiekhattar; citation_author=MP Spindler; citation_journal_title=Embo J; citation_volume=24; citation_number=69; citation_pages=1406-1417; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Biological methods for cell-cycle synchronization of mammalian cells.; citation_author=PK Davis; citation_author=A Ho; citation_author=SF Dowdy; citation_journal_title=Biotechniques; citation_volume=30; citation_number=70; citation_pages=1322-1326, 1328, 1330–1321; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The localization of replication origins on ARS plasmids in S. cerevisiae.; citation_author=BJ Brewer; citation_author=WL Fangman; citation_journal_title=Cell; citation_volume=51; citation_number=71; citation_pages=463-471; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=BLM and the FANC proteins collaborate in a common pathway in response to stalled replication forks.; citation_author=P Pichierri; citation_author=A Franchitto; citation_author=F Rosselli; citation_journal_title=Embo J; citation_volume=23; citation_number=72; citation_pages=3154-3163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A redox-sensitive cysteine in Zta is required for Epstein-Barr virus lytic cycle DNA replication.; citation_author=P Wang; citation_author=L Day; citation_author=J Dheekollu; citation_author=PM Lieberman; citation_journal_title=J Virol; citation_volume=79; citation_number=73; citation_pages=13298-13309; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001257" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Role for MRE11, NBS1, and Recombination Junctions in Replication and Stable Maintenance of EBV Episomes"/>
    <meta name="twitter:description" content="Recombination-like structures formed at origins of DNA replication may contribute to replication fidelity, sister chromatid cohesion, chromosome segregation, and overall genome stability. The Epstein-Barr Virus (EBV) origin of plasmid replication (OriP) provides episomal genome stability through a poorly understood mechanism. We show here that recombinational repair proteins MRE11 and NBS1 are recruited to the Dyad Symmetry (DS) region of OriP in a TRF2- and cell cycle-dependent manner. Depletion of MRE11 or NBS1 by siRNA inhibits OriP replication and destabilized viral episomes. OriP plasmid maintenance was defective in MRE11 and NBS1 hypomorphic fibroblast cell lines and only integrated, non-episomal forms of EBV were detected in a lympoblastoid cell line derived from an NBS1-mutated individual. Two-dimensional agarose gel analysis of OriP DNA revealed that recombination-like structures resembling Holliday-junctions form at OriP in mid S phase. MRE11 and NBS1 association with DS coincided with replication fork pausing and origin activation, which preceded the formation of recombination structures. We propose that NBS1 and MRE11 promote replication-associated recombination junctions essential for EBV episomal maintenance and genome stability."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001257.g007"/>

  <meta property="og:title" content="A Role for MRE11, NBS1, and Recombination Junctions in Replication and Stable Maintenance of EBV Episomes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001257" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=330'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7748'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9682&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001257">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001257" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001257&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Role for MRE11, NBS1, and Recombination Junctions in Replication and Stable Maintenance of EBV Episomes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jayaraju Dheekollu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Wistar Institute, Philadelphia, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhong Deng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Wistar Institute, Philadelphia, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andreas Wiedmer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Wistar Institute, Philadelphia, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew D. Weitzman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Salk Institute, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul M. Lieberman
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:lieberman@wistar.org">lieberman@wistar.org</a></p>

                <p>Affiliation:
                  The Wistar Institute, Philadelphia, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 05, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001257</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001257" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001257" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001257" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001257" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001257-g001" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001257-g002" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001257-g003" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001257-g004" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001257-g005" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001257-g006" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001257-g007" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001257">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001257" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Recombination-like structures formed at origins of DNA replication may contribute to replication fidelity, sister chromatid cohesion, chromosome segregation, and overall genome stability. The Epstein-Barr Virus (EBV) origin of plasmid replication (OriP) provides episomal genome stability through a poorly understood mechanism. We show here that recombinational repair proteins MRE11 and NBS1 are recruited to the Dyad Symmetry (DS) region of OriP in a TRF2- and cell cycle-dependent manner. Depletion of MRE11 or NBS1 by siRNA inhibits OriP replication and destabilized viral episomes. OriP plasmid maintenance was defective in MRE11 and NBS1 hypomorphic fibroblast cell lines and only integrated, non-episomal forms of EBV were detected in a lympoblastoid cell line derived from an NBS1-mutated individual. Two-dimensional agarose gel analysis of OriP DNA revealed that recombination-like structures resembling Holliday-junctions form at OriP in mid S phase. MRE11 and NBS1 association with DS coincided with replication fork pausing and origin activation, which preceded the formation of recombination structures. We propose that NBS1 and MRE11 promote replication-associated recombination junctions essential for EBV episomal maintenance and genome stability.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Dheekollu J, Deng Z, Wiedmer A, Weitzman MD, Lieberman PM (2007) A Role for MRE11, NBS1, and Recombination Junctions in Replication and Stable Maintenance of EBV Episomes. PLoS ONE 2(12):
          e1257.
            doi:10.1371/journal.pone.0001257</p><p><strong>Academic Editor: </strong>Beth Sullivan, Duke University, United States of America</p><p><strong>Received:</strong> September 18, 2007; <strong>Accepted:</strong> November 7, 2007; <strong>Published:</strong> December 5, 2007</p><p><strong>Copyright:</strong> © 2007 Dheekollu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>DAMD17-03-1-0313, NIH CA93606, PA Department of Health</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Genome stability depends on the successful completion of chromosome duplication prior to cell division <a href="#pone.0001257-Kolodner1">[1]</a>. Damaged DNA, nucleotide depletion, and endogenous nucleoprotein structures may present significant challenges to replication fork progression and prevent the completion of DNA synthesis. Several mechanisms are known to overcome these challenges, including the utilization of multiple origins of DNA replication and an elaborate post-replication repair system <a href="#pone.0001257-Eppink1">[2]</a>. One important source of post-replication repair involves the use of homologous recombination between sister chromatids <a href="#pone.0001257-Harrison1">[3]</a>. Homologous recombination-based mechanisms have been implicated in replication re-initiation at sites of collapsed replication forks and double strand breaks. Proteins implicated in homologous recombination are known to play a role in replication fork formation, stability, and processing <a href="#pone.0001257-Kolodner1">[1]</a>, <a href="#pone.0001257-Jackson1">[4]</a>–<a href="#pone.0001257-Stracker1">[6]</a>. Many of these homologous recombination repair proteins, including MRE11, NBS1, RAD50, BRCA1, and BRCA2 for example, are commonly mutated in human cancers and heritable genome instability disorders <a href="#pone.0001257-Branzei1">[5]</a>–<a href="#pone.0001257-Narod1">[7]</a>. However, the precise function of these homologous recombination repair proteins during normal DNA replication are only beginning to be understood in molecular detail.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Epstein-Barr Virus (EBV) provides a unique, and clinically significant model system to study genome stability in human cells. EBV is a gamma herpesvirus that establishes a long-term stable episomal infection in B-lymphocytes <a href="#pone.0001257-Young1">[8]</a>. The latent infection is tightly linked to several human cancers, including forms of Burkitt's lymphoma, Hodgkin's disease, and nasopharyngeal carcinomas <a href="#pone.0001257-Kieff1">[9]</a>, <a href="#pone.0001257-Rickinson1">[10]</a>. The viral episome replicates synchronously with the cellular genome and utilizes many of the same replication and cell cycle licensing factors as chromosomal DNA <a href="#pone.0001257-Yates1">[11]</a>–<a href="#pone.0001257-Schepers1">[16]</a>. The plasmid stability of EBV is conferred by a small genetic element, referred to as OriP, which consists of a family of repeats (FR) and a dyad symmetry (DS) element <a href="#pone.0001257-Rawlins1">[17]</a>, <a href="#pone.0001257-Yates2">[18]</a>. The repeats in the FR bind to the viral encoded nuclear antigen, EBNA1, which is essential for viral maintenance and immortalization of primary lymphocytes <a href="#pone.0001257-Lee1">[19]</a>, <a href="#pone.0001257-Humme1">[20]</a>. The DS region consists of phased EBNA1 sites juxtaposed with telomere repeat factor (TRF) binding sites, which together function as an efficient origin of DNA replication initiation <a href="#pone.0001257-Deng1">[21]</a>–<a href="#pone.0001257-Yates3">[23]</a> . However, the DS must function coordinately with the FR for stable episomal maintenance <a href="#pone.0001257-Gerhardt1">[24]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>TRF2 binding sites within DS contribute to the DNA replication activity and plasmid stability of OriP-containing plasmids <a href="#pone.0001257-Deng1">[21]</a>, <a href="#pone.0001257-Deng2">[22]</a>, <a href="#pone.0001257-Vogel1">[25]</a>–<a href="#pone.0001257-Julien1">[27]</a>. One function of TRF2 at DS is to facilitate the recruitment of cellular origin recognition complex (ORC) <a href="#pone.0001257-Atanasiu1">[28]</a>. At cellular telomeres, TRF2 is essential for the protection of chromosome ends from double-strand break repair and end-to-end fusions <a href="#pone.0001257-deLange1">[29]</a>, <a href="#pone.0001257-deLange2">[30]</a>. TRF2 stabilizes the lariat-like structure at the end of telomeres, referred to as the T-loop, which is formed by the single strand 5′-end invasion of the duplex DNA within the telomere repeats <a href="#pone.0001257-Griffith1">[31]</a>, <a href="#pone.0001257-Stansel1">[32]</a>. The precise role of TRF2 in telomere repeat DNA strand invasion and DNA replication is not completely understood, but it has been proposed that TRF2 regulates branch migration and may bind to recombination-like structures resembling Holliday junctions <a href="#pone.0001257-Wang2">[33]</a>, <a href="#pone.0001257-Fouche1">[34]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Homologous recombination proteins interact with telomere repeat binding factors and contribute to telomere DNA replication and maintenance <a href="#pone.0001257-deLange3">[35]</a>. TRF2 can be isolated in a complex with components of the MRE11/RAD50/NBS1 (MRN) complex and the association with NBS1 was found to be cell cycle regulated <a href="#pone.0001257-Zhu1">[36]</a>. The MRN complex is a highly conserved heterotrimer that functions in several different aspects of cellular DNA repair, recombination, and checkpoint surveillance <a href="#pone.0001257-Stracker1">[6]</a>. MRE11 has both endo- and exonuclease activity, while RAD50 forms a zinc-hook capable of linking neighboring molecules <a href="#pone.0001257-dAddadiFagagna1">[37]</a>. The MRN complex accumulates at double strand break-induced foci, along with several other cellular proteins, including 53BP1, CHK2, and BRCA1. In addition to their role at sites of damaged DNA, MRN and other members of the intra-S phase checkpoint response have been implicated in normal replication initiation events at origins and during re-initiation at sites of stalled or collapsed replication forks <a href="#pone.0001257-Maser1">[38]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In earlier studies, MRE11 and NBS1 have been implicated in the replication activity of OriP through an association with the E2F binding sites located over 1 kb from the DS region <a href="#pone.0001257-Maser2">[39]</a>. We now provide evidence that MRE11 and NBS1 can be recruited to the DS region of OriP through additional interactions, including those with TRF2, and that deficiencies in MRE11 and NBS1 inhibit Orip replication and maintenance function. We present evidence that that recombination-like structures form in the later stages of S phase at OriP. We suggest that these recombination structures form as a consequence of the pausing and initiation, as has been proposed for origins of replication in lower eukaryotes <a href="#pone.0001257-Lopes1">[40]</a>–<a href="#pone.0001257-Benard1">[42]</a>. We also suggest that recombination structures formed at OriP provide a junction between sister chromatids that are important for cohesion and segregation of EBV episomes.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Telomere repeats enhance MRE11 binding to OriP</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Previous work has shown that TRF2 binds to the three telomere repeats within the DS region of OriP <a href="#pone.0001257-Deng1">[21]</a>, <a href="#pone.0001257-Deng2">[22]</a>. Since TRF2 can interact with the MRN components, we tested whether MRE11 could be recruited to DS in vitro using a DNA affinity purification assay (<a href="#pone-0001257-g001">Fig. 1A</a>). We compared DNA templates containing wild type DS (DSwt), DS with substitution mutations in the telomere repeats (DSnm-), or OriP DNA with a 120 bp deletion of the DS (ΔDS) (<a href="#pone-0001257-g001">Fig. 1B</a>). As expected, TRF2 bound efficiently to DS wt, but not to DS nm- or ΔDS. Noteably, MRE11 was detected in the DSwt, but not in any of the control templates lacking telomere repeats. Control protein EBNA1 bound to DS wt and DS nm-, but not to ΔDS lacking EBNA1 binding sites. Replication protein PCNA did not bind to any of the templates, and the DNA binding protein PARP1 bound non-specifically to all of the templates. These findings indicate that MRE11 can be recruited to DS template DNA in vitro, and that binding depends on the telomere repeat sequence in DS.</p>
<div class="figure" id="pone-0001257-g001"><div class="img"><a name="pone-0001257-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>MRE11 associates with the DS region of OriP.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A) DNA affinity purification assays were performed with Raji nuclear extracts and DNA templates derived from DS wt, DS nm-, or ΔDS. Input and template bound proteins were analyzed by Western blot with antibodies to TRF2, MRE11, EBNA1, PCNA, and PARP1, as indicated. B) Schematic of template DNA used for DNA affinity purification in A, and their respective protein binding sites. C) ChIP assays to monitor EBNA1, TRF2, and MRE11 binding at OriP wt or OriP nm- in transfected 293 cells. Immunoprecipitated DNA was quantified by real time PCR relative to total input DNA recovered for each transfected cell.</p>
<span>doi:10.1371/journal.pone.0001257.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To determine if MRE11 bound to DS in vivo, we used the chromatin immunoprecipitation (ChIP) assay followed by real time PCR (<a href="#pone-0001257-g001">Fig. 1C</a>). We tested the effect of the telomere repeats on protein interactions by comparing OriP wt and OriP nm- containing plasmid in transfected cells. We found that EBNA1 bound to OriP wt and nm- to a similar extent. TRF2 bound OriP wt (~22% input), but its binding was reduced over 10 fold (~1.5% input) for OriP nm-. We also found that MRE11 bound OriP wt (~26% input), but was reduced by 4 fold (~6.5% input) for OriP nm-. These findings indicate that MRE11 binds to OriP in vivo, and that telomere repeats contribute to this binding.</p>


<h4>MRE11 and NBS1 contribute to OriP replication and episome maintenance function</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To determine if MRE11 and NBS1 contribute to OriP replication functions, we assayed the effects of siRNA depletion on OriP plasmid replication (<a href="#pone-0001257-g002">Fig. 2</a>). For these experiments, we used the EBV positive adherent cell line D98/HR1 which carry EBV episomes and have relatively high transfection efficiency. siRNA depletion of MRE11 and NBS1 was estimated at greater than 80% reduction as monitored by Western blotting (<a href="#pone-0001257-g002">Fig. 2A</a>). OriP-dependent plasmid replication was assayed by Southern blot analysis of Dpn I resistant plasmid (top panel) relative to total input DNA linearized by BamHI (lower panel) (<a href="#pone-0001257-g002">Fig. 2B</a>). The DpnI resistant form measures the DNA replicated in mammalian cells since its methylation pattern differs from the E. coli generated plasmid DNA initially transfected. PhosphorImager quantification of at least four independent transfections revealed that siRNA depletion of MRE11 and NBS1 reduced OriP replication by ~3.7 and ~2.0 fold, respectively (<a href="#pone-0001257-g002">Fig. 2B</a>, and data not shown). MRE11 and NBS1 depletion had no appreciable effect on D98/HR1 cell proliferation nor on cell cycle distribution as measured by propidium iodide staining and FACS analysis of transfected cells (<a href="#pone.0001257.s002">Fig. S1</a>). This suggests that the MRE11and NBS1 depletion has a modest, but consistent inhibitory effect on OriP replication.</p>
<div class="figure" id="pone-0001257-g002"><div class="img"><a name="pone-0001257-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>MRE11 and NBS1 contribute to OriP replication and EBV episome stability.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A) D98/HR1 cells were transfected with siRNA for MRE11 or NBS1, or Luciferase control and assayed by immuno-blot (IB) with antibodies specific for MRE11 (left pane) or NBS1 (right panel), or PCNA (lower panel) as a loading control. B) Transient DNA replication assays of OriP-containing plasmids were analyzed by Southern blot. D98/HR1 cells were transfected with OriP plasmid and siRNA for either MRE11, NBS1, or luciferase control (as indicated above each lane). Hirt extracted plasmid DNA was digested with DpnI plus BamHI (top panel) or BamHI only (lower panel) and probed with OriP specific probes. DNA was quantified by PhosphorImager analysis and the replication value was determined as the ratio of DpnI/BamHI to BamHI recovered products. Quantification shown below is a summary of at least four independent replication assays (data not shown). C)) EBV episomes from latently infected D98/HR1 cells were analyzed by pulse field electrophoresis and Southern blotting after transfection of siRNA for control (siLuc), MRE11, or NBS1. Raji cells were used as a control for EBV episome size and abundance. The percentage of EBV episomes relative to the EBV DNA retained in the well is calculated below.</p>
<span>doi:10.1371/journal.pone.0001257.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>OriP is also essential for episome maintenance of EBV. To determine if MRE11 or NBS1 contribute to EBV episome maintenance, we examined the stability of EBV episomes in D98/HR1 cells after siRNA depletion of MRE11 and NBS1. The episomal form can be detected by pulse field gel (PFG) electrophoresis and Southern blotting of D98/HR1 cell DNA purified in agarose plugs (<a href="#pone-0001257-g002">Fig. 2C</a>). We found ~22% of the EBV DNA could be isolated as intact episomes in D98/HR1 cells transfected with control siRNA (siControl). The DNA retained in the well is a combination of integrated and trapped episomal DNA. This was similar to that observed for latently infected Raji cells, which are known to have mostly episomal, but some integrated forms of EBV DNA. Transfection of D98/HR1 cells with siRNA specific for MRE11 or NBS1 decreased EBV episomes to ~10 and 6.5% of total EBV DNA, corresponding to a 2.2 and 3.1 fold loss of episomes in five days (<a href="#pone-0001257-g002">Fig. 2C</a>). MRE11 and NBS1 depletion also caused aberrant migration of EBV episomes in PFG electrophoresis, suggesting that these proteins contribute to the structural stability of latent EBV genomes. These effects suggest that MRE11 and NBS1 function in EBV episome maintenance, as well as in genome stability.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>To further assess the role of MRE11 and NBS1 in OriP function, we assayed OriP plasmid maintenance in MRE11 and NBS1 hypomorphic cell lines. Isogenic cell lines derived from mutant and wt replacements of MRE11 and NBS1 have been described previously <a href="#pone.0001257-Stracker2">[43]</a>. Cell lines mutated in MRE11 (ATLD3) or NBS1 (GM7166) and there corresponding wild-type corrected cell lines ATLD3/wtMRE11 or GM7166/wtNBS1, respectively, were tested for their ability to support OriP plasmid maintenance (<a href="#pone-0001257-g003">Fig. 3A</a>). Wt and mt cell lines were transfected with OriP plasmid expressing the EBNA1 protein, and then assayed for plasmid maintenance at 1 day or 7 days post-transfection. We found that MRE11 and NBS1 mt cell lines were impaired for maintaining OriP plasmids after 7 days post-transfection, with no detectable difference observed at 1 day post-transfection (<a href="#pone-0001257-g003">Fig. 3A</a>). Wt and mt cell lines proliferated at similar rates, with similar cell cycle profiles (data not shown), and identical cell numbers were analyzed for each cell type. Quantification of the average of four independent experiments revealed that MRE11 and NBS1 mutant cell lines were reduced by ~2 and ~2.5 fold for plasmid maintenance relative to wt cell lines.</p>
<div class="figure" id="pone-0001257-g003"><div class="img"><a name="pone-0001257-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Evidence that MRE11 and NBS1 are required for EBV episome maintenance.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>A) OriP plasmid maintenance assays were performed in MRE11 mutant (ATLD3) and reconstituted (ATLD3/wtMRE11) cell lines (left panel) or NBS1 mutant (GM7166) and reconstituted (GM7166/wtNBS1) cell lines (right panel). Plasmids containing OriP and EBNA1 were monitored by Southern blotting of Hirt lysates at 1 (lower panel) and 7 (upper panel) days post-transfection. Phosphorimager quantification of four independent experiments as shown in panal A where maintenance is measured as the ratio of day 7 to day 1 for OriP plasmid detection is shown below. B) EBV transformed B-lymphocytes were analyzed by PFG and Southern blotting for the presence of episomal forms of the EBV genome. Raji, Namalwa, NBS1 (GM15808), NBS1 (GM07078), LCL3472, and LCLAW7 DNA was loaded at equal concentrations and analyzed by ethidium staining of PFG (lower panel) or by Southern blot (upper panel) with EBV specific probe. C) Western blot of the same cell lines used for PFG analysis shown in panel B was probed with antibodies specific for NBS1 (antibody 143 and antibody 277), MRE11, or loading control Actin.</p>
<span>doi:10.1371/journal.pone.0001257.g003</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>One prediction of these studies is that MRE11 and NBS1 are required for stable episome maintenance of EBV during lymphoblastoid cell transformation. To test this prediction, we analyzed the structure of EBV genomes in two lymphoblastoid cells derived from NBS1 mutated patients (<a href="#pone-0001257-g003">Fig. 3B</a>). EBV episomes were monitored by Southern blotting of agarose gels after PFG electrophoresis. Episomes were readily detected in Raji cells, which are known to carry multiple copies of circularized EBV genomes. In contrast, no episomes were detected for Namalwa Burkitt lymphoma cellls, which are known to have integrated EBV genomes and lack episomal forms <a href="#pone.0001257-Gulley1">[44]</a>. Interestingly, we found that NBS1 GM15808 cells had no detectable episomes (<a href="#pone-0001257-g003">Fig. 3B</a>). More extensive analysis of the EBV termini indicated that GM15808 cells had integrated viral genomes, similar to Namalwa (<a href="#pone.0001257.s003">Fig. S2</a>). In contrast, a second NBS1 mutated cell line GM0708 had EBV episomes at levels comparable to that of other lymphoblastoid cell lines, LCL3472 and LCLAW7. Simlar preparations of total cellular DNA were analyzed for each sample as demonstated by ethidium staining of the PFG (<a href="#pone-0001257-g003">Fig. 3B</a>, lower panel). To determine if the lack of episomes in NBS1 GM15808 correlates with NBS1 and MRE11 protein expression or structure, we analyzed these cell lines by Western blot analysis (<a href="#pone-0001257-g003">Fig. 3C</a>). We found that NBS1 protein was absent from both NBS1 cell lines when assayed with antibody 143. In contrast, we found that NBS1 protein was detectable, but with a slightly faster mobility when probed with antibody 277, which recognizes a different NBS1 epitope. Interestingly, we found that MRE11 protein had a markedly faster mobility in NBS1 GM15808 cells relative to all other cell types analyzed, including NBS1 GM07078 (<a href="#pone-0001257-g003">Fig. 3C</a>). Since lymphoblastoid cells are non-clonal expansions of EBV immortalized B-lymphocytes they are expected to have episomal EBV. The lack of episomes in GM15808 suggest that their more severe defect in both NBS1 and MRE11contributes to their failure to support episomal maintenance of EBV.</p>


<h4>Cell cycle-dependent recruitment of MRE11 and NBS1 to episomal OriP</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>MRN is known to have cell cycle-specific activities, and interact with TRF2 in a cell cycle-dependent manner. We therefore examined whether MRN components, MRE11 and NBS1 associate with OriP in a cell cycle-dependent manner (<a href="#pone-0001257-g004">Fig. 4</a>). MutuI Burkitt lymphoma cell lines maintain multiple copies of the latent EBV episome. MutuI cells were fractionated by centrifugal elutriation to isolated cell cycle synchronized populations. Cell cycle synchronizations were verified by FACS analysis after propidium iodide staining (<a href="#pone-0001257-g004">Fig. 4A</a>). Cells isolated at different stages of the cell cycle were assayed by ChIP for binding of MRE11 and NBS1. We compared binding to OriP with a control region within the alternative replication initiation zone used in Raji cells, referred to as OriR <a href="#pone.0001257-Norio1">[45]</a>. ChIP values were quantified as percentage of input DNA for different stages of the cell cycle and for each region of the genome. We found that MRE11 and NBS1 were enriched at OriP, but not at OriR. We found a significant cell cycle-dependent association of MRE11 and NBS1 at OriP, with ~10 fold increase in binding in mid-S phase (fraction 24) relative to early G1 (<a href="#pone-0001257-g004">Fig. 4B</a>). MRE11 and NBS1 were also recruited to OriP in mid S phase when cells were synchronized by double thymidine block and release (<a href="#pone.0001257.s004">Fig. S3</a>), indicating that the method of synchronization was not responsible for the interaction or timing. Furthermore, high levels of DNA damaging agents, including gamma-irradiation (IR) or 5 mM hydroxyurea (HU), caused a depletion of MRE11 and NBS1 from OriP (<a href="#pone.0001257.s004">Fig. S3B</a>). These findings suggest that cell cycle association of MRE11 and NBS1 with OriP is distinct from the general cellular response to DNA damage induced by HU and IR.</p>
<div class="figure" id="pone-0001257-g004"><div class="img"><a name="pone-0001257-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Early S phase recruitment of MRE11 and NBS1 to DS.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A) Mutu I cells were fractionated by centrifugal elutriation and analyzed for cell cycle synchronization by FACS. Cell cycle fraction numbers correspond to rates of counter flow used for centrifugal elutriation. B) MutuI cells were fractionated by centrifugal elutriation as shown in panel B, and then analyzed by chromatin immunoprecipitation with antibodies to MRE11, NBS1, and control IgG. Immunoprecipitated DNA was quantified by real-time PCR with primers specific for OriP or control OriR regions of the EBV genome. ChIP values were presented as percentage of input DNA. Cell cycle fractions correspond to the FACS analysis in panel A.</p>
<span>doi:10.1371/journal.pone.0001257.g004</span></div>

<h4>Recombination-like structures form at OriP</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>MRE11 and NBS1 have been implicated in the homologous recombination pathway and intra-S phase checkpoint response. Since MRE11 and NBS1 localize to OriP in mid-S phase, we explored the possibility that recombination-like structures may form at OriP. The DNA structures formed during replication and recombination can be analyzed using two-dimensional neutral agarose gel (2D gel) electrophoresis. The migration properties of prominent DNA structures have been well established, including the Y-arcs formed by replication fork progression, replication pause sites that appear as bulges in the Y-arc (<a href="#pone-0001257-g005">Fig. 5A</a>, red arrows), origin bubbles arcs (<a href="#pone-0001257-g005">Fig. 5A</a>, green arrow), and Holliday junctions or X-structures which appear as vertical spikes emerging from the 2n spot (<a href="#pone-0001257-g005">Fig. 5A</a>, magenta arrows). Holliday junctions are notoriously unstable due to their transient nature and branch migration on linearized fragments. To enrich for recombination-like structures, DNA was isolated with the cationic-detergent CTAB, which inhibits branch migration during DNA isolation and manipulation <a href="#pone.0001257-Allers1">[46]</a>. To further enrich for transient recombination structures, and to determine if recombination structures formed at OriP at times when MRE11 and NBS1 associated with OriP, we analyzed cells that were isolated from different stages across the cell cycle using centrifugal elutriation, as shown in <a href="#pone-0001257-g004">Fig. 4A</a>. Cell cycle fractions from EBV positive Mutu I cells were analyzed by 2-D gel electrophoresis (<a href="#pone-0001257-g005">Fig. 5B</a>). At early stages of S phase, we found accumulation of Y-structures and the formation of a pronounced fork pausing site (<a href="#pone-0001257-g005">Fig. 5C</a>, red arrows, fractions 18, 22, 25). This pause was most likely localized to the DS region based on the appearance in the large Y-arc of the PvuII digested OriP DNA. Further progression through S phase revealed that the formation of a bubble arc indicative of replication initiation (green arrow, fraction 25). As cells further progress through mid S phase, the replication pause sight disappeared, and a vertical spike emerging from the 2n position appeared. This vertical spike has been interpreted as recombinational intermediates, which may include Holliday junctions, X-structures, and hemicatenanes (magenta arrows, fractions 27, 29, 30). By G2/M, the major replication and recombination structures were less apparent (fraction 36). Fork pausing and recombination-like structures were also observed at OriP in Raji cells (<a href="#pone.0001257.s005">Fig. S4</a>). A schematic representation of these structures is presented in <a href="#pone-0001257-g005">Figure 5C</a>.</p>
<div class="figure" id="pone-0001257-g005"><div class="img"><a name="pone-0001257-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Cell cycle analysis of replication and recombination structures formed at OriP DNA.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>A) Schematic of replication and recombination-associated DNA structures resolved by 2-dimensional neutral agarose gels. Holliday junction and X-like structures migrate as a vertical spike emerging from the 2n spot (shown in magenta). The bubble arc (shown in green) and replication fork pause sites (shown in red) are also indicated. B) Two dimensional neutral agarose gel analysis of OriP isolated using CTAB method from Mutu I cells after cell cycle fractionation. Cell cycle fractions from centrifugal elutration are indicated in the upper right corner of each image. C) Cartoon interpretation of the major OriP DNA structures observed for each cell cycle fraction in panel C.</p>
<span>doi:10.1371/journal.pone.0001257.g005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Two-dimensional gel electrophoresis of CTAB isolated DNA provided evidence that recombination-like structures form at OriP (<a href="#pone-0001257-g005">Fig. 5</a>). These structures were further analyzed using one dimensional gel electrophoresis (<a href="#pone-0001257-g006">Fig. 6</a>). DNA was extracted from MutuI cells with or without CTAB, digested with EarI restriction, and then analyzed by Southern blotting (<a href="#pone-0001257-g006">Fig. 6A</a>). Analysis with OriP specific probe revealed the predicted ~3.4 kb (1n) species, as well as several slower migrating forms. When DNA was extracted with CTAB, we observed a novel OriP containing species migrating at ~7.5 kb. Reprobing the same blot for a ~3.2 kb fragment adjacent to EBV OriLyt showed no structures dependent on CTAB extraction. This indicates that a CTAB-stabilized structure migrating as ~7.5 kb occurs at OriP, but not at OriLyt-containing DNA.</p>
<div class="figure" id="pone-0001257-g006"><div class="img"><a name="pone-0001257-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Evidence for recombination junctions formed at OriP.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>A) MutuI cell DNA was extracted with (+) or without (−) CTAB to stabilize recombinational structures. DNA was isolated and cleaved by EarI and then analyzed by Southern blot from a one-dimensional agarose gel with probes to OriP (left panel) or OriLyt (right panel). The position of the expected 1n products are indicated. B) MutuI cells were fractionated by centrifugal elutriation as shown for <a href="#pone-0001257-g004">Fig. 4B</a>. DNA was extracted using CTAB method and cleaved with EarI, and analyzed by Southern blot with probe to OriP. C) DNA was extracted from MutuI cells with CTAB, digested with EarI, and then mock treated (lanes 1 and 2), or treated with T7 endonuclease I (lanes 3 and 4) or with mung bean nuclease (lanes 5 and 6). DNA fragments were visualized by Southern blot with probe to OriP. D) NBS1 mt (GM7166) or reconstituted (GM7166/wtNBS1) cell lines were transfected with plasmid OriP and selected for 48 hrs using hygromycin. Total cell DNA was isolated using CTAB method, linearized with EcoRI, and then detected by Southern blot with probe specific for plasmid OriP. The 5.5 kB linear DNA fragment (1n) and slower mobility form (arrow) are indicated. E) Pulse field electrophoresis and Southern blot of EBV episomes isolated from MutuI cells at different stages of the cell cycle, as indicated above each pair of lanes. The 1n position corresponds to a monomeric circular genome (~900 Kb of linear DNA), while the 2n corresponds to a dimeric circular genome (~1.8 Mb of linear DNA).</p>
<span>doi:10.1371/journal.pone.0001257.g006</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Our 2D gel analysis (<a href="#pone-0001257-g005">Fig. 5</a>) suggested that recombination-like structures form in mid to late S phase (fractions 26–30). We therefore assayed the formation of 2n species across the cell cycle using MutuI cells that were fractionated by centrifugal elutriation (<a href="#pone-0001257-g006">Fig. 6B</a>). We found that the 7.5 kb CTAB-stabilized species (indicated by the arrow) formed predominantly in mid S phase (fraction 26). This species was not detected in cells derived from early S phase (fraction 25), or G2/M (fraction 45), and was detectable in small amounts during late S phase (fractions 32 and 36). The appearance of the 7.5 kb structure coincides with the formation of the spike structure in 2D gels (<a href="#pone-0001257-g005">Fig. 5</a>).</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>The DNA structure formed at OriP was further investigated with the use of a structure-specific endonuclease. Holliday junctions (HJ) and related recombinational structures are known to be cleaved by T7 endonuclease I (T7endoI) <a href="#pone.0001257-Muller1">[47]</a>. MutuI cell DNA was extracted with CTAB and linearized with a EarI as above. The restricted DNA was then subject to digestion with or without T7endoI and assayed by one-dimensional gel and Southern blotting with OriP-specific probe (<a href="#pone-0001257-g006">Fig. 6C</a>). We found that T7endoI eliminated the 7.5 kb CTAB-sensitive species. T7endoI also partially digested a faster migrating 6.0 kb (small arrow) species, with a corresponding increase in the faster migrating (1n) species. We also found that T7endoI eliminates the vertical spikes, along with replication pause sites, observed in 2D neutral agarose gels (<a href="#pone.0001257.s006">Figs. S5</a> and <a href="#pone.0001257.s007">S6</a>). Additionally, incubation at 65°C which facilitates branch migration and the resolution of HJ structures on linear DNA, reduced the appearance of the vertical spikes in 2D gels (<a href="#pone.0001257.s007">Fig. S6</a>). We conclude that the vertical spike observed in 2D gels, and the CTAB-stabilized structures observed in 1D gels, are consistent with HJ and related recombination structures.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>We next asked whether the MRN complex contributed to the formation of the slower migrating forms of OriP DNA. NBS1 hypomorphic cell line GM7166, or NBS1-wt reconstituted cells GM7166/NBS1wt were transfected with OriP plasmid and selected for 48 hrs prior to extraction with CTAB and linearized with EcoRI restriction endonuclease (<a href="#pone-0001257-g006">Fig. 6D</a>). In GM7166/NBSwt cells, we observed a slower migrating species (~11 kb, indicated by the arrow) that formed 6 fold greater relative to the 1n species (~5.5 kb EcoRI fragment). In GM7166 cells, the slower migrating species formed only ~1.6 fold over the 1n species (<a href="#pone-0001257-g006">Fig. 6D</a>). These results indicate that NBS1 contributes to the formation of the slower migrating form of OriP DNA, and suggests that NBS1 is required for the formation of recombinational structures at OriP.</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>The recombination junctions that form at OriP are predicted to form linkages between newly replicated sister EBV genomes. To test this possibility, we analyzed EBV genome migration by PFG electrophoresis at different stages of the cell cycle (<a href="#pone-0001257-g006">Fig. 6E</a>). Cell cycle fractions were collected by centrifugal elutriation as described in <a href="#pone-0001257-g004">Fig. 4A</a> and EBV episomes were analyzed by PFG electrophoresis and Southern blotting as described in <a href="#pone-0001257-g002">Fig. 2C</a> and <a href="#pone-0001257-g003">3B</a>. We found that monomeric episomes were the predominant form in G1 and early S, migrating at a position equivalent to ~900 kb of linear DNA . A slower migrating species with the predicted mobility of a 2n dimeric circle (~1.8 MB of linear DNA) appeared in mid S and accumulated to almost 50% of the total molecules by G2/M (<a href="#pone-0001257-g006">Fig. 6E</a>). This finding is consistent with the model that EBV genomes form topologically linked 2n species after DNA replication.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Intra-S phase checkpoint proteins are thought to monitor and process replication intermediates formed during DNA damage conditions, as well as during normal cellular division <a href="#pone.0001257-Osborn1">[48]</a>, <a href="#pone.0001257-Kraus1">[49]</a>. The MRN complex, in particular, is known to colocalize with sites of DNA replication and is thought to stabilize and repair stalled or collapsed replication forks <a href="#pone.0001257-Stracker1">[6]</a>, <a href="#pone.0001257-Maser2">[39]</a>, <a href="#pone.0001257-Trenz1">[50]</a>. MRN components have also been implicated in the replication and genome stability of several DNA viruses, including herpes simplex <a href="#pone.0001257-Lilley1">[51]</a>. However, the precise role of the MRN in cellular DNA replication processing remains elusive, partially as a consequence of having limited knowledge of the genetic loci where MRN functions. Here, we provide evidence that components of the MRN complex function at the replication origin and plasmid maintenance element of Epstein-Barr virus. We provide evidence that recombination-like X structures form at OriP following replication pausing and origin activation. We suggest that MRN proteins regulate the formation and/or processing of origin-associated recombinational structures, and that these structures contribute to the plasmid maintenance function of OriP.</p>

<h4>Cell cycle-dependent association and function of MRN at DS</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Telomere repeats within DS contribute to the DNA replication and plasmid maintenance function of OriP <a href="#pone.0001257-Deng1">[21]</a>. It has been previously shown that TRF2 associates with components of the MRN complex <a href="#pone.0001257-Zhu1">[36]</a>, <a href="#pone.0001257-deLange4">[52]</a>. We therefore explored the potential role of MRE11 and NBS1 in the functions of OriP. We found that MRE11 associates with the DS region of OriP and that the telomere repeats enhanced this association (<a href="#pone-0001257-g001">Fig. 1</a>). We also found that MRE11 and NBS1 bound to DS in a cell cycle-dependent manner with maximum binding occuring in middle S phase (<a href="#pone-0001257-g004">Fig. 4</a>). A cell cycle-dependent interaction of NBS1 with TRF2 has been previously reported, thus potentially explaining the cell cycle association of MRN with the DS region of OriP <a href="#pone.0001257-Zhu1">[36]</a>. Deletion of telomere repeat sites in DS only partially eliminated MRN binding in vivo, suggesting that other mechanisms also contribute to MRN recruitment to OriP. An E2F site located ~1 kb from DS can also contribute to the localization of MRN at OriP <a href="#pone.0001257-Maser2">[39]</a>. However, it is also possible that MRN is recruited to DS through the recognition of complex DNA structures associated with replication initiation, pausing, and termination. We speculate that TRF2 sites evolved at DS to reinforce MRN recruitment and enhance the processing of DNA structures important for episome maintenance.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The function of MRN proteins at DS was investigated using the DNA replication and plasmid maintenance assays. We found that OriP replication activity and EBV genome stability was reduced in cells where MRE11 and NBS1 were depleted by siRNA (<a href="#pone-0001257-g002">Fig. 2</a>). We also found fibroblasts with hypomorphic mutations in MRE11 or NBS1 were defective in maintaining OriP-dependent plasmids (<a href="#pone-0001257-g003">Fig. 3</a>). Furthermore, an EBV immortalized lymphoblastoid cell line derived from an NBS1 mutated individual lacked any detectable episomal forms of EBV (<a href="#pone-0001257-g003">Fig. 3B</a>). Although a second NBS1 mutated lymphoblastoid cell line retained EBV episomes, this second cell line had normal MRE11 protein and less severe alterations in NBS1 (<a href="#pone-0001257-g003">Fig. 3C</a>). Since the EBV transformed NBS1 cells are not a clonal isolate, we consider the exclusively integrated form to be highly significant, since it represents the statistical average of many transformation events. Almost all non-cloned LCLs have episomal and linear forms of EBV. The NBS1 mutated cells had no linear forms, and in the case of the more severely mutated NBS1 and MRE11 defective cell line GM15808, there were no detectable episomes. These findings argue strongly that NBS1 and MRE11 are important for maintaining stable episomes of EBV during latent infection in lymphoblastoid cells. Since MRE11 and NBS1 localize to DS by ChIP, and not to other regions of the OriP plasmid, we consider it likely that MRN acts directly at DS to process DNA structures formed during DNA replication that are necessary for stable episome maintenance.</p>


<h4>Replication pausing, initiation, and termination at OriP</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Two dimensional agarose gel analysis of OriP has revealed that replication can both initiate, pause and terminate near the DS and FR regions of OriP <a href="#pone.0001257-Gahn1">[53]</a>–<a href="#pone.0001257-Dhar2">[56]</a>. Our data shows that a replication pause site forms within or adjacent to DS early in S phase (<a href="#pone-0001257-g005">Fig. 5B</a>, factions 18), prior to origin bubble arc formation (<a href="#pone-0001257-g005">Fig. 5B</a>, fraction 25), and persists until mid S phase (factions 29). Weaker pause sites could also be detected at FR. This suggests that EBNA1 sites at DS and FR obstruct replication fork progression. The accumulation of Y-structures at OriP is most probably a consequence of replication initiating outside of OriP. Single molecule analysis of DNA replication revealed that replication can initiate at other sites in EBV, and that OriP is a major site of replication fork pausing and termination <a href="#pone.0001257-Norio2">[57]</a>. In this scenario, the origin function of DS may be required only under conditions where initiation fails at other locations in the viral genome, or if replication of the viral genome is not complete. Replication initiation at DS may be required to complete DNA synthesis of the regions between the DS and FR, especially if these EBNA1 binding sites function as replication fork barriers. This arrangement of OriP elements may promote recombination structures associated with replication fork stalling, initiation, and temination events.</p>


<h4>Recombination structures form at OriP</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Evidence that recombination-like structures formed at OriP were provided by one and two-dimensional agarose gel electrophoresis studies (<a href="#pone-0001257-g005">Figs. 5</a> and <a href="#pone-0001257-g006">6</a>). The formation of a vertical spike emerging from the 2n species in two-dimensional agarose gels has been interpreted as recombination-like structures, which include Holliday junctions and hemicatenanes <a href="#pone.0001257-Segurado1">[41]</a>, <a href="#pone.0001257-Benard1">[42]</a>. The recombination-like structures at OriP were detected after the appearance of the bubble arc in mid S phase and persisted through the remainder of late S and G2 (<a href="#pone-0001257-g005">Fig. 5B</a>, fraction 26–30). A similar cell cycle pattern of recombination structures forming after origin initiation has been observed in lower eukaryotes <a href="#pone.0001257-Lopes1">[40]</a>, <a href="#pone.0001257-Liberi1">[58]</a>. Further evidence for recombination junctions at OriP was provided by 1-D agarose gel analysis (<a href="#pone-0001257-g006">Fig. 6</a>). Recombination junctions can be stabilized by CTAB extraction which inhibits branch migration <a href="#pone.0001257-Allers1">[46]</a>. We were able to show that CTAB stabilized a form of OriP DNA that migrated at higher molecular mass (~2n) (<a href="#pone-0001257-g006">Fig. 6A</a>). The CTAB-stabilized species was enriched in S phase (<a href="#pone-0001257-g006">Fig. 6B</a>) and senstive to T7endoI (<a href="#pone-0001257-g006">Fig. 6C</a>). Furthermore, T7endoI eliminated the vertical spike in 2D gel analysis (<a href="#pone.0001257.s006">Figs. S5</a> and <a href="#pone.0001257.s007">S6</a>), indicating that the CTAB stabilized structure in 1D gels corresponds to the vertical spike in the 2D gel analysis. The vertical spike in 2D gels was also sensitive to heat treatment at 65°C, suggesting that this structure is sensitive to branch migration (<a href="#pone.0001257.s007">Fig. S6</a>). Since T7endoI has known specificity for HJ structures <a href="#pone.0001257-Hadden1">[59]</a>, we consider it likely that HJ and related recombinational structures form at OriP.</p>


<h4>Implications for EBV episomal maintenance</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The prevailing model for episomal maintenance of gammaherpesviruses argues that replicated viral genomes will distribute stochastically to daughter cells through a non-specific tethering of newly replicated genomes to cellular chromosomes <a href="#pone.0001257-Sears1">[60]</a>–<a href="#pone.0001257-Sugden2">[62]</a>. More recent studies have revealed that the viral episomes maintain a stable copy number through a mechanism involving sister chromatid cohesion and anaphase segregation, similar to cellular chromosomes <a href="#pone.0001257-Kapoor1">[63]</a>, <a href="#pone.0001257-Kanda1">[64]</a>. Our findings suggest that recombination-like junctions, which form at OriP, may contribute to the sister-chromatid cohesion required for genome segregation (<a href="#pone-0001257-g007">Fig. 7A</a>). Consistent with this model, we found that topologically linked dimers of EBV accumulate in late S and G2/M phase of the cell cycle (<a href="#pone-0001257-g006">Fig. 6D</a>). We propose that MRN components, which localize at OriP and contribute to OriP replication and maintenance function, are important for the establishment of these linked structures (<a href="#pone-0001257-g007">Fig. 7B</a>). Others have shown that MRN components contribute to HJ formation in mammalian cells <a href="#pone.0001257-Tauchi1">[65]</a>. MRN may work in conjunction with TRF2 at DS to stimulate strand invasion, in a manner similar to that proposed at telomere T-loops <a href="#pone.0001257-Fouche1">[34]</a>. While the precise structure of the recombination-like junctions are not yet known, we suggest that these junctions contibute to the sister chromatid cohesion necessary for homologous DNA repair and for chromosome segregation <a href="#pone.0001257-Nasmyth1">[66]</a>. Interestingly, the origin recognition complex (ORC) promotes sister chromatid cohesion in yeast (ORC) <a href="#pone.0001257-Shimada1">[67]</a>, and it is possible that OriP recruits ORC to generate cohesion between sister viral genomes. This suggests that the origin function of OriP is mechanistically linked to the plasmid maintenance function. These findings provide a new mechanistic paradigm for OriP plasmid maintenance, and have implications for the episome maintenance mechanisms of related viruses, including KSHV and HPV. These findings may also have implications for the mechanisms of generating sister chromatid cohesion at cellular chromosome sites, like telomere repeats and origins of replication, where recombinational junctions are likely to form.</p>
<div class="figure" id="pone-0001257-g007"><div class="img"><a name="pone-0001257-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001257" data-uri="info:doi/10.1371/journal.pone.0001257.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001257.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001257.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001257.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001257.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Model of recombination junctions formed at OriP and their function in sister-chromatid cohesion.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>A) Schematic of EBV episome DNA replication with fork pausing and terminating at OriP. Recombination structures are formed at OriP and provide sister chromatid cohesion through topological linkages. Resolution of these linkages in anaphase promotes genome segregation and stable plasmid maintenance. B) Speculative model showing MRN and recombination structures that may form at DS. These include strand invasion, double Holliday junctions, and hemicatenane of nascent strand DNA.</p>
<span>doi:10.1371/journal.pone.0001257.g007</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Cell lines and antibodies</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>EBV positive lymphoblastoid cell lines (LCLs) and Burkitt lymphoma cell lines (Raji, MutuI, and Namalwa) were maintained in RPMI medium supplemented with 10% fetal bovine serum, glutamine, penicillin, and streptomycin sulfate (Cellgro). D98/HR1 (EBV<sup>+</sup> adherent cells), Hela, and 293 (EBV-adherent cells) cells were maintained in DMEM medium supplemented with 10% fetal bovine serum, glutamine, penicillin, and streptomycin sulfate (Cellgro). Mre11, Nbs1 mutant, and reconstituted cells were provided by M.D. Weitzman <a href="#pone.0001257-Stracker2">[43]</a>. Lymphoblastoid cell lines from NBS1 mutated individuals, GM15808 and GM07078 were purchased from Coriel Institute (Camden, NJ). The following rabbit polyclonal antibodies were used: anti-Mre11 (NB 100-142; Novus), anti-Nbs1 (Nb 100-277; Novus), anti-Brca1 (Abcam), and control rabbit immunoglobulin G (IgG; Jackson Laboratories). Rabbit polyclonal anti-EBNA1 and Trf2 were raised against a recombinant full-length EBNA1 and Trf2.</p>


<h4>Cell Cycle Synchronization</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Cell cycle fractionation using centrifugal elutriation was performed with a modified Beckman JE 5.0 using counter flow rates for EBV positive B-lymphocytes as described previously<a href="#pone.0001257-Ritzi1">[68]</a>, <a href="#pone.0001257-Zhou1">[69]</a>. Alternatively, asynchronous Raji cells were arrested in G1/S using double thymidine method and released into S phase, then collected at different time points at early S (2 h), middle S (4 h), and late S (6 h), or at G2/M by treatment with 150 ng Nocodozole (Sigma) for 18 h <a href="#pone.0001257-Davis1">[70]</a>.</p>


<h4>CTAB Isolation of DNA</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>DNA extraction with cetyltrimethylammonium bromide (CTAB) was essentially as described <a href="#pone.0001257-Allers1">[46]</a>. See supplemental methods for additional details.</p>


<h4>Two Dimensional Gel Electrophoresis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Two dimensional gel electrophoresis was performed essentially as described previously, except that DNA was prepared using the CTAB method <a href="#pone.0001257-Allers1">[46]</a>, <a href="#pone.0001257-Brewer1">[71]</a>.</p>


<h4>Additional Methods</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Chromatin Immunoprecipitation, FACS analysis of cell cycle, and DNA replication assays have been described previously<a href="#pone.0001257-Deng2">[22]</a>. Primers used for real time PCR analysis of ChIP assays were: OriP (DS) (atgtaaataaaaccgtgacaggctcat; ttacccaacgggaagcatatg), OriR (ggccacgctgataaagttgt ; ctagaaaccctggcgaccat) and OriLyt (cgtcttactgcccagcctact; agtgggagggcaggaaat). siRNA were synthesized as duplex RNA (Dharmacon Inc.) with the following target sequences for Luciferase control (cgtacgcggaatacttcga) and a pool of three target sequences for Mre11 (cctgcctcgagttattaag; ctgcgagtggactatagtg; gatgccattgaggaattag,) as described previously <a href="#pone.0001257-Pichierri1">[72]</a>. siRNA for NBS1 is commercially available as Smartpool products (Dharmacon Inc.). Plasmid or siRNA controls were used in parallel for each siRNA experiment. Pulse field electrophoresis of EBV episomes purified from agarose plugs were described previously <a href="#pone.0001257-Wang3">[73]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001257"></div><a name="pone.0001257.s001" id="pone.0001257.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001257.s001">Text S1. </a>Supplementary Methods</strong></p><p class="siDoi">doi:10.1371/journal.pone.0001257.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001257.s002" id="pone.0001257.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001257.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">MRE11 and BRCA1 siRNA depletion does not cause cell cycle arrest. FACS analysis of propidium iodide stained cells after control or siRNA depletion of MRE11 (A), or NBS1 (B) used for replication assays shown in <a href="#pone-0001257-g002">Figure 2</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001257.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(4.04 MB TIF)</p>
<a name="pone.0001257.s003" id="pone.0001257.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001257.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Evidence for integrated EBV genomes in NBS1 GM15808. Total genomic DNA was isolated from Raji, Namalwa, NBS1 GM15808, NBS1 GM07078, LCL 3472, or LCLAW7, and linearized with BamHI. DNA was then analyzed by Southern blot and hybridized to probes specific for the left junction of the terminal repeats (TR_L) (panel A), the right junction of the terminal repeat (TR_R) (panel B), or OriP region (panel C). Different fragments size of the terminal repeats is indicative of integrated forms.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001257.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.88 MB TIF)</p>
<a name="pone.0001257.s004" id="pone.0001257.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001257.s004">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Early S phase recruitment of MRE11, NBS1, and BRCA1 to DS. A) Chromatin immunoprecipitation of EBV positive Raji cells was analyzed at different times in the cell cycle with antibodies to MRE11, NBS1, BRCA1 and control IgG. Immunoprecipitated DNA was quantified by real-time PCR with primers specific for DS or control OriLyt (OL) regions of the EBV genome. ChIP values were presented as percentage of input DNA. B) ChIP assays were used to measure MRE11, NBS1, and BRCA1 binding to OriP in asynchronous Raji cells (Asy) or Raji cells treated with 5 mM HU for 1 hr (HU), or 9 Gy of gamma irradiation (Gy). C) FACS profile of propidium iodide treated Raji cells synchronized by double thymidine block and used for ChIP assays shown in panel A.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001257.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(3.98 MB TIF)</p>
<a name="pone.0001257.s005" id="pone.0001257.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001257.s005">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Formation of X-structures at OriP in Raji cells. Two-dimensional neutral agarose gels were used to analyze DNA extracted from Raji cells after centrifugal elutriation. The cell cycle fractions are indicated in the above right, and correspond to nearly identical cell cycle distribution as shown for MutuI cells (<a href="#pone-0001257-g004">Figure 4A</a>). Raji cell DNA was extracted by CTAB method, digested with PvuII and visualized by Southern blot hybridization to an OriP specific probe.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001257.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(7.13 MB TIF)</p>
<a name="pone.0001257.s006" id="pone.0001257.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001257.s006">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">T7EndoI Sensitivity of Recombinational Structures at OriP. MutuI DNA was extracted by CTAB, linearized with PvuII and then further treated with a) mock (Control), b) mung bean, or c) T7EndoI nuclease. DNA was then analyzed by 2-D neutral agarose gel electrophoresis and Southern blot probed for OriP. PhosphorImager gels of two independent experiments are above (Expt I and Expt II), and a schematic interpretation is shown below.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001257.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(15.91 MB TIF)</p>
<a name="pone.0001257.s007" id="pone.0001257.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001257.s007">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Thermal Sensitivity of Recombinational Structures at OriP. Mutu I DNA was extracted with CTAB, linearized with PvuII, and then treated with either T7 EndoI nuclease, 65°C for 30 min, or mock treatment (control). DNA was analyzed by 2D gel and Southern blotting with OriP-specific probe. B) A schematic interpretation of the salient DNA structures is shown.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001257.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(8.90 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We also thank Latasha Day for technical support and the Wistar Institute Flow Cytometry Core.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PL JD. Performed the experiments: JD ZD AW. Analyzed the data: PL. Contributed reagents/materials/analysis tools: MW. Wrote the paper: PL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001257-Kolodner1" id="pone.0001257-Kolodner1"></a>Kolodner RD, Putnam CD, Myung K (2002) Maintenance of genome stability in Saccharomyces cerevisiae. Science  297: 552–557.  <ul class="find" data-citedArticleID="1025390" data-doi="10.1126/science.1075277"><li><a href="http://dx.doi.org/10.1126/science.1075277" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maintenance+of+genome+stability+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maintenance+of+genome+stability+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001257-Eppink1" id="pone.0001257-Eppink1"></a>Eppink B, Wyman C, Kanaar R (2006) Multiple interlinked mechanisms to circumvent DNA replication roadblocks. Exp Cell Res  312: 2660–2665.  <ul class="find" data-citedArticleID="1025360" data-doi="10.1016/j.yexcr.2006.06.011"><li><a href="http://dx.doi.org/10.1016/j.yexcr.2006.06.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+interlinked+mechanisms+to+circumvent+DNA+replication+roadblocks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+interlinked+mechanisms+to+circumvent+DNA+replication+roadblocks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001257-Harrison1" id="pone.0001257-Harrison1"></a>Harrison JC, Haber JE (2006) Surviving the Breakup: The DNA Damage Checkpoint. Annu Rev Genet.   <ul class="find" data-citedArticleID="1025376" data-doi="10.1146/annurev.genet.40.051206.105231"><li><a href="http://dx.doi.org/10.1146/annurev.genet.40.051206.105231" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Surviving+the+Breakup%3A+The+DNA+Damage+Checkpoint." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Surviving+the+Breakup%3A+The+DNA+Damage+Checkpoint.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001257-Jackson1" id="pone.0001257-Jackson1"></a>Jackson SP (2002) Sensing and repairing DNA double-strand breaks. Carcinogenesis  23: 687–696.  <ul class="find" data-citedArticleID="1025380" data-doi="10.1093/carcin/23.5.687"><li><a href="http://dx.doi.org/10.1093/carcin/23.5.687" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sensing+and+repairing+DNA+double-strand+breaks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sensing+and+repairing+DNA+double-strand+breaks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001257-Branzei1" id="pone.0001257-Branzei1"></a>Branzei D, Foiani M (2005) The DNA damage response during DNA replication. Curr Opin Cell Biol  17: 568–575.  <ul class="find" data-citedArticleID="1025344" data-doi="10.1016/j.ceb.2005.09.003"><li><a href="http://dx.doi.org/10.1016/j.ceb.2005.09.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+DNA+damage+response+during+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+DNA+damage+response+during+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001257-Stracker1" id="pone.0001257-Stracker1"></a>Stracker TH, Theunissen JW, Morales M, Petrini JH (2004) The Mre11 complex and the metabolism of chromosome breaks: the importance of communicating and holding things together. DNA Repair (Amst)  3: 845–854.  <ul class="find" data-citedArticleID="1025440" data-doi="10.1016/j.dnarep.2004.03.014"><li><a href="http://dx.doi.org/10.1016/j.dnarep.2004.03.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Mre11+complex+and+the+metabolism+of+chromosome+breaks%3A+the+importance+of+communicating+and+holding+things+together." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Mre11+complex+and+the+metabolism+of+chromosome+breaks%3A+the+importance+of+communicating+and+holding+things+together.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001257-Narod1" id="pone.0001257-Narod1"></a>Narod SA, Foulkes WD (2004) BRCA1 and BRCA2: 1994 and beyond. Nat Rev Cancer  4: 665–676.  <ul class="find" data-citedArticleID="1025410" data-doi="10.1038/nrc1431"><li><a href="http://dx.doi.org/10.1038/nrc1431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BRCA1+and+BRCA2%3A+1994+and+beyond." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BRCA1+and+BRCA2%3A+1994+and+beyond.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001257-Young1" id="pone.0001257-Young1"></a>Young LS, Rickinson AB (2004) Epstein-Barr virus: 40 years on. Nat Rev Cancer  4: 757–768.  <ul class="find" data-citedArticleID="1025466" data-doi="10.1038/nrc1452"><li><a href="http://dx.doi.org/10.1038/nrc1452" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epstein-Barr+virus%3A+40+years+on." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epstein-Barr+virus%3A+40+years+on.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001257-Kieff1" id="pone.0001257-Kieff1"></a>Kieff E (1996)  Epstein-Barr virus and its replication. In: Knipe D, Howley PM, editors. Field's Virology. Third Edition ed. Philadelphia: Lippincott-Raven Publishers. pp. 2343–2396.  <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0001257-Rickinson1" id="pone.0001257-Rickinson1"></a>Rickinson AB, Kieff E (1996) Epstein-Barr Virus. Fields Virology, Third Edition: Lippincott-Raven Publishers. pp. 2397–2446.  <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0001257-Yates1" id="pone.0001257-Yates1"></a>Yates JL, Guan N (1991) Epstein-Barr virus-derived plasmids replicate only once per cell cycle and are not amplified after entry into cells. J Virol  65: 483–488.  <ul class="find" data-citedArticleID="1025460"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Epstein-Barr+virus-derived+plasmids+replicate+only+once+per+cell+cycle+and+are+not+amplified+after+entry+into+cells.&amp;auth=&amp;atitle=Epstein-Barr+virus-derived+plasmids+replicate+only+once+per+cell+cycle+and+are+not+amplified+after+entry+into+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epstein-Barr+virus-derived+plasmids+replicate+only+once+per+cell+cycle+and+are+not+amplified+after+entry+into+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epstein-Barr+virus-derived+plasmids+replicate+only+once+per+cell+cycle+and+are+not+amplified+after+entry+into+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001257-Sugden1" id="pone.0001257-Sugden1"></a>Sugden B, Leight ER (2001)  Molecular mechanisms of maintenance and disruption of virus latency. In: Takada K, editor. Epstein-Barr virus and human cancer. Heidelberg: Springer. pp. 3–11.  <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0001257-Wang1" id="pone.0001257-Wang1"></a>Wang J, Sugden B (2005) Origins of bidirectional replication of Epstein-Barr virus: Models for understanding mammalian origins of DNA synthesis. J Cell Biochem  94: 247–256.  <ul class="find" data-citedArticleID="1025454" data-doi="10.1002/jcb.20324"><li><a href="http://dx.doi.org/10.1002/jcb.20324" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origins+of+bidirectional+replication+of+Epstein-Barr+virus%3A+Models+for+understanding+mammalian+origins+of+DNA+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origins+of+bidirectional+replication+of+Epstein-Barr+virus%3A+Models+for+understanding+mammalian+origins+of+DNA+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001257-Chaudhuri1" id="pone.0001257-Chaudhuri1"></a>Chaudhuri B, Xu H, Todorov I, Dutta A, Yates JL (2001) Human DNA replication initiation factors, ORC and MCM, associate with oriP of Epstein-Barr virus. Proc Natl Acad Sci, U S A  98: 10085–10089.  <ul class="find" data-citedArticleID="1025348" data-doi="10.1073/pnas.181347998"><li><a href="http://dx.doi.org/10.1073/pnas.181347998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+DNA+replication+initiation+factors%2C+ORC+and+MCM%2C+associate+with+oriP+of+Epstein-Barr+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+DNA+replication+initiation+factors%2C+ORC+and+MCM%2C+associate+with+oriP+of+Epstein-Barr+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001257-Dhar1" id="pone.0001257-Dhar1"></a>Dhar SK, Yoshida K, Machida Y, Khaira P, Chaudhuri B, et al.  (2001) Replication from oriP of Epstein-Barr virus requires human ORC and is inhibited by geminin. Cell  106: 287–296.  <ul class="find" data-citedArticleID="1025356" data-doi="10.1016/s0092-8674(01)00458-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(01)00458-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replication+from+oriP+of+Epstein-Barr+virus+requires+human+ORC+and+is+inhibited+by+geminin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replication+from+oriP+of+Epstein-Barr+virus+requires+human+ORC+and+is+inhibited+by+geminin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001257-Schepers1" id="pone.0001257-Schepers1"></a>Schepers A, Ritzi M, Bousset K, Kremmer E, Yates JL, et al.  (2001) Human origin recognition complex binds to the region of the latent origin of DNA replication of Epstein-Barr virus. EMBO J  20: 4588–4602.  <ul class="find" data-citedArticleID="1025430" data-doi="10.1093/emboj/20.16.4588"><li><a href="http://dx.doi.org/10.1093/emboj/20.16.4588" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+origin+recognition+complex+binds+to+the+region+of+the+latent+origin+of+DNA+replication+of+Epstein-Barr+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+origin+recognition+complex+binds+to+the+region+of+the+latent+origin+of+DNA+replication+of+Epstein-Barr+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001257-Rawlins1" id="pone.0001257-Rawlins1"></a>Rawlins DR, Milman G, Hayward SD, Hayward GS (1985) Sequence-specific DNA binding of the Epstein-Barr virus nuclear antigen (EBNA-1) to clustered sites in the plasmid maintenance region. Cell  42: 859–868.  <ul class="find" data-citedArticleID="1025424" data-doi="10.1016/0092-8674(85)90282-x"><li><a href="http://dx.doi.org/10.1016/0092-8674(85)90282-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence-specific+DNA+binding+of+the+Epstein-Barr+virus+nuclear+antigen+%28EBNA-1%29+to+clustered+sites+in+the+plasmid+maintenance+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence-specific+DNA+binding+of+the+Epstein-Barr+virus+nuclear+antigen+%28EBNA-1%29+to+clustered+sites+in+the+plasmid+maintenance+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001257-Yates2" id="pone.0001257-Yates2"></a>Yates JL, Warren N, Reisman P, Sugden B (1984) A <em>cis</em>-acting element from Epstein-Barr viral genome that permits stable replication of recombinant plasmids in latently infected cells. Proc Natl Acad Sci USA  81: 3806–3810.  <ul class="find" data-citedArticleID="1025462" data-doi="10.1073/pnas.81.12.3806"><li><a href="http://dx.doi.org/10.1073/pnas.81.12.3806" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cis-acting+element+from+Epstein-Barr+viral+genome+that+permits+stable+replication+of+recombinant+plasmids+in+latently+infected+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cis-acting+element+from+Epstein-Barr+viral+genome+that+permits+stable+replication+of+recombinant+plasmids+in+latently+infected+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001257-Lee1" id="pone.0001257-Lee1"></a>Lee MA, Diamond ME, Yates JL (1999) Genetic evidence that EBNA-1 is needed for efficient, stable latent infection by Epstein-Barr virus. J Virol  73: 2974–2982.  <ul class="find" data-citedArticleID="1025394"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+evidence+that+EBNA-1+is+needed+for+efficient%2C+stable+latent+infection+by+Epstein-Barr+virus.&amp;auth=&amp;atitle=Genetic+evidence+that+EBNA-1+is+needed+for+efficient%2C+stable+latent+infection+by+Epstein-Barr+virus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+evidence+that+EBNA-1+is+needed+for+efficient%2C+stable+latent+infection+by+Epstein-Barr+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+evidence+that+EBNA-1+is+needed+for+efficient%2C+stable+latent+infection+by+Epstein-Barr+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001257-Humme1" id="pone.0001257-Humme1"></a>Humme S, Reisbach G, Feederle R, Delecluse HJ, Bousset K, et al.  (2003) The EBV nuclear antigen 1 (EBNA1) enhances B cell immortalization several thousandfold. Proc Natl Acad Sci U S A  100: 10989–10994.  <ul class="find" data-citedArticleID="1025378" data-doi="10.1073/pnas.1832776100"><li><a href="http://dx.doi.org/10.1073/pnas.1832776100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+EBV+nuclear+antigen+1+%28EBNA1%29+enhances+B+cell+immortalization+several+thousandfold." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+EBV+nuclear+antigen+1+%28EBNA1%29+enhances+B+cell+immortalization+several+thousandfold.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001257-Deng1" id="pone.0001257-Deng1"></a>Deng Z, Lezina L, Chen CJ, Shtivelband S, So W, et al.  (2002) Telomeric proteins regulate episomal maintenance of Epstein-Barr virus origin of plasmid replication. Mol Cell  9:   <ul class="find" data-citedArticleID="1025352" data-doi="10.1016/s1097-2765(02)00476-8"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00476-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Telomeric+proteins+regulate+episomal+maintenance+of+Epstein-Barr+virus+origin+of+plasmid+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Telomeric+proteins+regulate+episomal+maintenance+of+Epstein-Barr+virus+origin+of+plasmid+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001257-Deng2" id="pone.0001257-Deng2"></a>Deng Z, Atanasiu C, Burg JS, Broccoli D, Lieberman PM (2003) Telomere repeat binding factors TRF1, TRF2, and hRAP1 modulate replication of Epstein-Barr virus OriP. J Virol  77: 11992–12001.  <ul class="find" data-citedArticleID="1025354" data-doi="10.1128/jvi.77.22.11992-12001.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.22.11992-12001.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Telomere+repeat+binding+factors+TRF1%2C+TRF2%2C+and+hRAP1+modulate+replication+of+Epstein-Barr+virus+OriP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Telomere+repeat+binding+factors+TRF1%2C+TRF2%2C+and+hRAP1+modulate+replication+of+Epstein-Barr+virus+OriP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001257-Yates3" id="pone.0001257-Yates3"></a>Yates JL, Camiolo SM, Bashaw JM (2000) The minimal replicator of Epstein-Barr virus oriP. J Virol  74: 4512–4522.  <ul class="find" data-citedArticleID="1025464" data-doi="10.1128/jvi.74.10.4512-4522.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.10.4512-4522.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+minimal+replicator+of+Epstein-Barr+virus+oriP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+minimal+replicator+of+Epstein-Barr+virus+oriP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001257-Gerhardt1" id="pone.0001257-Gerhardt1"></a>Gerhardt J, Jafar S, Spindler MP, Ott E, Schepers A (2006) Identification of new human origins of DNA replication by an origin-trapping assay. Mol Cell Biol  26: 7731–7746.  <ul class="find" data-citedArticleID="1025368" data-doi="10.1128/mcb.01392-06"><li><a href="http://dx.doi.org/10.1128/mcb.01392-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+new+human+origins+of+DNA+replication+by+an+origin-trapping+assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+new+human+origins+of+DNA+replication+by+an+origin-trapping+assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001257-Vogel1" id="pone.0001257-Vogel1"></a>Vogel M, Wittmann K, Endl E, Glaser G, Knuchel R, et al.  (1998) Plasmid maintenance assay based on green fluorescent protein and FACS of mammalian cells. BioTechniques  24: 540–544.  <ul class="find" data-citedArticleID="1025452"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Plasmid+maintenance+assay+based+on+green+fluorescent+protein+and+FACS+of+mammalian+cells.&amp;auth=&amp;atitle=Plasmid+maintenance+assay+based+on+green+fluorescent+protein+and+FACS+of+mammalian+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmid+maintenance+assay+based+on+green+fluorescent+protein+and+FACS+of+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmid+maintenance+assay+based+on+green+fluorescent+protein+and+FACS+of+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001257-Niller1" id="pone.0001257-Niller1"></a>Niller HH, Glaser G, Knuchel R, Wolf H (1995) Nucleoprotein complexes and DNA 5′-ends at oriP of Epstein-Barr virus. J Biol Chem  270: 12864–12868.  <ul class="find" data-citedArticleID="1025414" data-doi="10.1074/jbc.270.21.12864"><li><a href="http://dx.doi.org/10.1074/jbc.270.21.12864" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleoprotein+complexes+and+DNA+5%E2%80%B2-ends+at+oriP+of+Epstein-Barr+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleoprotein+complexes+and+DNA+5%E2%80%B2-ends+at+oriP+of+Epstein-Barr+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001257-Julien1" id="pone.0001257-Julien1"></a>Julien MD, Polonskaya Z, Hearing J (2004) Protein and sequence requirements for the recruitment of the human origin recognition complex to the latent cycle origin of DNA replication of Epstein-Barr virus oriP. Virology  326: 317–328.  <ul class="find" data-citedArticleID="1025382" data-doi="10.1016/j.virol.2004.05.026"><li><a href="http://dx.doi.org/10.1016/j.virol.2004.05.026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+and+sequence+requirements+for+the+recruitment+of+the+human+origin+recognition+complex+to+the+latent+cycle+origin+of+DNA+replication+of+Epstein-Barr+virus+oriP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+and+sequence+requirements+for+the+recruitment+of+the+human+origin+recognition+complex+to+the+latent+cycle+origin+of+DNA+replication+of+Epstein-Barr+virus+oriP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001257-Atanasiu1" id="pone.0001257-Atanasiu1"></a>Atanasiu C, Deng Z, Wiedmer A, Norseen J, Lieberman PM (2006) ORC binding to TRF2 stimulates OriP replication. EMBO Rep  7: 716–721.  <ul class="find" data-citedArticleID="1025338" data-doi="10.1038/sj.embor.7400730"><li><a href="http://dx.doi.org/10.1038/sj.embor.7400730" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ORC+binding+to+TRF2+stimulates+OriP+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ORC+binding+to+TRF2+stimulates+OriP+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001257-deLange1" id="pone.0001257-deLange1"></a>de Lange T (2002) Protection of mammalian telomeres. Oncogene  21: 532–540.  <ul class="find" data-citedArticleID="1025474" data-doi="10.1038/sj.onc.1205080"><li><a href="http://dx.doi.org/10.1038/sj.onc.1205080" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protection+of+mammalian+telomeres." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protection+of+mammalian+telomeres.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001257-deLange2" id="pone.0001257-deLange2"></a>de Lange T (2004) T-loops and the origin of telomeres. Nat Rev Mol Cell Biol  5: 323–329.  <ul class="find" data-citedArticleID="1025476" data-doi="10.1038/nrm1359"><li><a href="http://dx.doi.org/10.1038/nrm1359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T-loops+and+the+origin+of+telomeres." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T-loops+and+the+origin+of+telomeres.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001257-Griffith1" id="pone.0001257-Griffith1"></a>Griffith JD, Comeau L, Rosenfield S, Stansel RM, Bianchi A, et al.  (1999) Mammalian telomeres end in a large duplex loop. Cell  97: 503–514.  <ul class="find" data-citedArticleID="1025370" data-doi="10.1016/s0092-8674(00)80760-6"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80760-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+telomeres+end+in+a+large+duplex+loop." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+telomeres+end+in+a+large+duplex+loop.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001257-Stansel1" id="pone.0001257-Stansel1"></a>Stansel RM, de Lange R, Griffith JD (2001) T-loop assembly in vitro involves binding of TRF2 near the 3′ telomeric overhang. EMBO J  20: 5532–5540.  <ul class="find" data-citedArticleID="1025438" data-doi="10.1093/emboj/20.19.5532"><li><a href="http://dx.doi.org/10.1093/emboj/20.19.5532" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T-loop+assembly+in+vitro+involves+binding+of+TRF2+near+the+3%E2%80%B2+telomeric+overhang." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T-loop+assembly+in+vitro+involves+binding+of+TRF2+near+the+3%E2%80%B2+telomeric+overhang.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001257-Wang2" id="pone.0001257-Wang2"></a>Wang RC, Smogorzewska A, de Lange T (2004) Homologous recombination generates T-loop-sized deletions at human telomeres. Cell  119: 355–368.  <ul class="find" data-citedArticleID="1025456" data-doi="10.1016/j.cell.2004.10.011"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.10.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homologous+recombination+generates+T-loop-sized+deletions+at+human+telomeres." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homologous+recombination+generates+T-loop-sized+deletions+at+human+telomeres.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001257-Fouche1" id="pone.0001257-Fouche1"></a>Fouche N, Cesare AJ, Willcox S, Ozgur S, Compton SA, et al.  (2006) The basic domain of TRF2 directs binding to DNA junctions irrespective of the presence of TTAGGG repeats. J Biol Chem.   <ul class="find" data-citedArticleID="1025364" data-doi="10.1074/jbc.m608778200"><li><a href="http://dx.doi.org/10.1074/jbc.m608778200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+basic+domain+of+TRF2+directs+binding+to+DNA+junctions+irrespective+of+the+presence+of+TTAGGG+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+basic+domain+of+TRF2+directs+binding+to+DNA+junctions+irrespective+of+the+presence+of+TTAGGG+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001257-deLange3" id="pone.0001257-deLange3"></a>de Lange T (2005) Shelterin: the protein complex that shapes and safeguards human telomeres. Genes Dev  19: 2100–2110.  <ul class="find" data-citedArticleID="1025478" data-doi="10.1101/gad.1346005"><li><a href="http://dx.doi.org/10.1101/gad.1346005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shelterin%3A+the+protein+complex+that+shapes+and+safeguards+human+telomeres." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Shelterin%3A+the+protein+complex+that+shapes+and+safeguards+human+telomeres.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001257-Zhu1" id="pone.0001257-Zhu1"></a>Zhu X-D, Kuster B, Mann M, Petrini JHJ, de Lange T (2000) Cell-cycle-regulated association of RAD50/MRE11/NBS1 with TRF2 and human telomeres. Nat Genet  25: 347–352.  <ul class="find" data-citedArticleID="1025470"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cell-cycle-regulated+association+of+RAD50%2FMRE11%2FNBS1+with+TRF2+and+human+telomeres.&amp;auth=&amp;atitle=Cell-cycle-regulated+association+of+RAD50%2FMRE11%2FNBS1+with+TRF2+and+human+telomeres." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell-cycle-regulated+association+of+RAD50%2FMRE11%2FNBS1+with+TRF2+and+human+telomeres." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell-cycle-regulated+association+of+RAD50%2FMRE11%2FNBS1+with+TRF2+and+human+telomeres.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001257-dAddadiFagagna1" id="pone.0001257-dAddadiFagagna1"></a>d'Adda di Fagagna F, Teo SH, Jackson SP (2004) Functional links between telomeres and proteins of the DNA-damage response. Genes Dev  18: 1781–1799.  <ul class="find" data-citedArticleID="1025472" data-doi="10.1101/gad.1214504"><li><a href="http://dx.doi.org/10.1101/gad.1214504" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+links+between+telomeres+and+proteins+of+the+DNA-damage+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+links+between+telomeres+and+proteins+of+the+DNA-damage+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001257-Maser1" id="pone.0001257-Maser1"></a>Maser RS, DePinho RA (2004) Telomeres and the DNA damage response: why the fox is guarding the henhouse. DNA Repair (Amst)  3: 979–988.  <ul class="find" data-citedArticleID="1025404" data-doi="10.1016/j.dnarep.2004.05.009"><li><a href="http://dx.doi.org/10.1016/j.dnarep.2004.05.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Telomeres+and+the+DNA+damage+response%3A+why+the+fox+is+guarding+the+henhouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Telomeres+and+the+DNA+damage+response%3A+why+the+fox+is+guarding+the+henhouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001257-Maser2" id="pone.0001257-Maser2"></a>Maser RS, Mirzoeva OK, Wells J, Olivares H, Williams BR, et al.  (2001) Mre11 complex and DNA replication: linkage to E2F and sites of DNA synthesis. Mol Cell Biol  21: 6006–6016.  <ul class="find" data-citedArticleID="1025406" data-doi="10.1128/mcb.21.17.6006-6016.2001"><li><a href="http://dx.doi.org/10.1128/mcb.21.17.6006-6016.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mre11+complex+and+DNA+replication%3A+linkage+to+E2F+and+sites+of+DNA+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mre11+complex+and+DNA+replication%3A+linkage+to+E2F+and+sites+of+DNA+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001257-Lopes1" id="pone.0001257-Lopes1"></a>Lopes M, Cotta-Ramusino C, Liberi G, Foiani M (2003) Branch migrating sister chromatid junctions form at replication origins through Rad51/Rad52-independent mechanisms. Mol Cell  12: 1499–1510.  <ul class="find" data-citedArticleID="1025402" data-doi="10.1016/s1097-2765(03)00473-8"><li><a href="http://dx.doi.org/10.1016/s1097-2765(03)00473-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Branch+migrating+sister+chromatid+junctions+form+at+replication+origins+through+Rad51%2FRad52-independent+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Branch+migrating+sister+chromatid+junctions+form+at+replication+origins+through+Rad51%2FRad52-independent+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001257-Segurado1" id="pone.0001257-Segurado1"></a>Segurado M, Gomez M, Antequera F (2002) Increased recombination intermediates and homologous integration hot spots at DNA replication origins. Mol Cell  10: 907–916.  <ul class="find" data-citedArticleID="1025434" data-doi="10.1016/s1097-2765(02)00684-6"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00684-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Increased+recombination+intermediates+and+homologous+integration+hot+spots+at+DNA+replication+origins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Increased+recombination+intermediates+and+homologous+integration+hot+spots+at+DNA+replication+origins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001257-Benard1" id="pone.0001257-Benard1"></a>Benard M, Maric C, Pierron G (2001) DNA replication-dependent formation of joint DNA molecules in Physarum polycephalum. Mol Cell  7: 971–980.  <ul class="find" data-citedArticleID="1025342" data-doi="10.1016/s1097-2765(01)00237-4"><li><a href="http://dx.doi.org/10.1016/s1097-2765(01)00237-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+replication-dependent+formation+of+joint+DNA+molecules+in+Physarum+polycephalum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+replication-dependent+formation+of+joint+DNA+molecules+in+Physarum+polycephalum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001257-Stracker2" id="pone.0001257-Stracker2"></a>Stracker TH, Carson CT, Weitzman MD (2002) Adenovirus oncoproteins inactivate the Mre11-Rad50-NBS1 DNA repair complex. Nature  418: 348–352.  <ul class="find" data-citedArticleID="1025442" data-doi="10.1038/nature00863"><li><a href="http://dx.doi.org/10.1038/nature00863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adenovirus+oncoproteins+inactivate+the+Mre11-Rad50-NBS1+DNA+repair+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adenovirus+oncoproteins+inactivate+the+Mre11-Rad50-NBS1+DNA+repair+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001257-Gulley1" id="pone.0001257-Gulley1"></a>Gulley ML, Raphael M, Lutz CT, Ross DW, Raab-Traub N (1992) Epstein-Barr virus integration in human lymphomas and lymphoid cell lines. Cancer  70: 185–191.  <ul class="find" data-citedArticleID="1025372" data-doi="10.1002/1097-0142(19920701)70:1<185::aid-cncr2820700129&gt;3.0.co;2-j"><li><a href="http://dx.doi.org/10.1002/1097-0142(19920701)70:1<185::aid-cncr2820700129&gt;3.0.co;2-j" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epstein-Barr+virus+integration+in+human+lymphomas+and+lymphoid+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epstein-Barr+virus+integration+in+human+lymphomas+and+lymphoid+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001257-Norio1" id="pone.0001257-Norio1"></a>Norio P, Schildkraut CL (2001) Visualization of DNA replication on individual Epstein-Barr virus episomes. Science  294: 2361–2364.  <ul class="find" data-citedArticleID="1025416" data-doi="10.1126/science.1064603"><li><a href="http://dx.doi.org/10.1126/science.1064603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Visualization+of+DNA+replication+on+individual+Epstein-Barr+virus+episomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Visualization+of+DNA+replication+on+individual+Epstein-Barr+virus+episomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001257-Allers1" id="pone.0001257-Allers1"></a>Allers T, Lichten M (2000) A method for preparing genomic DNA that restrains branch migration of Holliday junctions. Nucleic Acids Res  28: e6.  <ul class="find" data-citedArticleID="1025336" data-doi="10.1093/nar/28.2.e6"><li><a href="http://dx.doi.org/10.1093/nar/28.2.e6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+method+for+preparing+genomic+DNA+that+restrains+branch+migration+of+Holliday+junctions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+method+for+preparing+genomic+DNA+that+restrains+branch+migration+of+Holliday+junctions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001257-Muller1" id="pone.0001257-Muller1"></a>Muller B, Jones C, West SC (1990) T7 endonuclease I resolves Holliday junctions formed in vitro by RecA protein. Nucleic Acids Res  18: 5633–5636.  <ul class="find" data-citedArticleID="1025408" data-doi="10.1093/nar/18.19.5633"><li><a href="http://dx.doi.org/10.1093/nar/18.19.5633" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T7+endonuclease+I+resolves+Holliday+junctions+formed+in+vitro+by+RecA+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T7+endonuclease+I+resolves+Holliday+junctions+formed+in+vitro+by+RecA+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001257-Osborn1" id="pone.0001257-Osborn1"></a>Osborn AJ, Elledge SJ, Zou L (2002) Checking on the fork: the DNA-replication stress-response pathway. Trends Cell Biol  12: 509–516.  <ul class="find" data-citedArticleID="1025420" data-doi="10.1016/s0962-8924(02)02380-2"><li><a href="http://dx.doi.org/10.1016/s0962-8924(02)02380-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Checking+on+the+fork%3A+the+DNA-replication+stress-response+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Checking+on+the+fork%3A+the+DNA-replication+stress-response+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001257-Kraus1" id="pone.0001257-Kraus1"></a>Kraus E, Leung WY, Haber JE (2001) Break-induced replication: a review and an example in budding yeast. Proc Natl Acad Sci U S A  98: 8255–8262.  <ul class="find" data-citedArticleID="1025392" data-doi="10.1073/pnas.151008198"><li><a href="http://dx.doi.org/10.1073/pnas.151008198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Break-induced+replication%3A+a+review+and+an+example+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Break-induced+replication%3A+a+review+and+an+example+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001257-Trenz1" id="pone.0001257-Trenz1"></a>Trenz K, Smith E, Smith S, Costanzo V (2006) ATM and ATR promote Mre11 dependent restart of collapsed replication forks and prevent accumulation of DNA breaks. Embo J  25: 1764–1774.  <ul class="find" data-citedArticleID="1025450" data-doi="10.1038/sj.emboj.7601045"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ATM+and+ATR+promote+Mre11+dependent+restart+of+collapsed+replication+forks+and+prevent+accumulation+of+DNA+breaks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ATM+and+ATR+promote+Mre11+dependent+restart+of+collapsed+replication+forks+and+prevent+accumulation+of+DNA+breaks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001257-Lilley1" id="pone.0001257-Lilley1"></a>Lilley CE, Carson CT, Muotri AR, Gage FH, Weitzman MD (2005) DNA repair proteins affect the lifecycle of herpes simplex virus 1. Proc Natl Acad Sci U S A  102: 5844–5849.  <ul class="find" data-citedArticleID="1025398" data-doi="10.1073/pnas.0501916102"><li><a href="http://dx.doi.org/10.1073/pnas.0501916102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+repair+proteins+affect+the+lifecycle+of+herpes+simplex+virus+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+repair+proteins+affect+the+lifecycle+of+herpes+simplex+virus+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001257-deLange4" id="pone.0001257-deLange4"></a>de Lange T, Petrini JH (2000) A new connection at human telomeres: association of the Mre11 complex with TRF2. Cold Spring Harb Symp Quant Biol  65: 265–273.  <ul class="find" data-citedArticleID="1025480" data-doi="10.1101/sqb.2000.65.265"><li><a href="http://dx.doi.org/10.1101/sqb.2000.65.265" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+connection+at+human+telomeres%3A+association+of+the+Mre11+complex+with+TRF2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+connection+at+human+telomeres%3A+association+of+the+Mre11+complex+with+TRF2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001257-Gahn1" id="pone.0001257-Gahn1"></a>Gahn TA, Schildkraut CL (1989) The Epstein-Barr virus origin of plasmid replication, oriP, contains both the initiation and termination sites of DNA replication. Cell  58: 527–535.  <ul class="find" data-citedArticleID="1025366" data-doi="10.1016/0092-8674(89)90433-9"><li><a href="http://dx.doi.org/10.1016/0092-8674(89)90433-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Epstein-Barr+virus+origin+of+plasmid+replication%2C+oriP%2C+contains+both+the+initiation+and+termination+sites+of+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Epstein-Barr+virus+origin+of+plasmid+replication%2C+oriP%2C+contains+both+the+initiation+and+termination+sites+of+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001257-Little1" id="pone.0001257-Little1"></a>Little RD, Schildkraut CL (1995) Initiation of latent DNA replication in the Epstein-Barr virus genome can occur at sites other than the genetically defined origin. Mol Cell Biol  15: 2893–2903.  <ul class="find" data-citedArticleID="1025400"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Initiation+of+latent+DNA+replication+in+the+Epstein-Barr+virus+genome+can+occur+at+sites+other+than+the+genetically+defined+origin.&amp;auth=&amp;atitle=Initiation+of+latent+DNA+replication+in+the+Epstein-Barr+virus+genome+can+occur+at+sites+other+than+the+genetically+defined+origin." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initiation+of+latent+DNA+replication+in+the+Epstein-Barr+virus+genome+can+occur+at+sites+other+than+the+genetically+defined+origin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initiation+of+latent+DNA+replication+in+the+Epstein-Barr+virus+genome+can+occur+at+sites+other+than+the+genetically+defined+origin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001257-Ermakova1" id="pone.0001257-Ermakova1"></a>Ermakova OV, Frappier L, Schildkraut CL (1996) Role of the EBNA-1 protein in pausing of replication forks in the Epstein-Barr virus genome. J Biol Chem  271: 33009–33017.  <ul class="find" data-citedArticleID="1025362" data-doi="10.1074/jbc.271.51.33009"><li><a href="http://dx.doi.org/10.1074/jbc.271.51.33009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+EBNA-1+protein+in+pausing+of+replication+forks+in+the+Epstein-Barr+virus+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+EBNA-1+protein+in+pausing+of+replication+forks+in+the+Epstein-Barr+virus+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001257-Dhar2" id="pone.0001257-Dhar2"></a>Dhar V, Schildkraut CL (1991) Role of EBNA-1 in arresting replication forks at the Epstein-Barr virus oriP family of tandem repeats. Mol Cell Biol  11: 6268–6278.  <ul class="find" data-citedArticleID="1025358"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Role+of+EBNA-1+in+arresting+replication+forks+at+the+Epstein-Barr+virus+oriP+family+of+tandem+repeats.&amp;auth=&amp;atitle=Role+of+EBNA-1+in+arresting+replication+forks+at+the+Epstein-Barr+virus+oriP+family+of+tandem+repeats." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+EBNA-1+in+arresting+replication+forks+at+the+Epstein-Barr+virus+oriP+family+of+tandem+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+EBNA-1+in+arresting+replication+forks+at+the+Epstein-Barr+virus+oriP+family+of+tandem+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001257-Norio2" id="pone.0001257-Norio2"></a>Norio P, Schildkraut CL (2004) Plasticity of DNA replication initiation in Epstein-Barr virus episomes. PLoS Biol  2: e152.  <ul class="find" data-citedArticleID="1025418" data-doi="10.1371/journal.pbio.0020152"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020152" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasticity+of+DNA+replication+initiation+in+Epstein-Barr+virus+episomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasticity+of+DNA+replication+initiation+in+Epstein-Barr+virus+episomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001257-Liberi1" id="pone.0001257-Liberi1"></a>Liberi G, Cotta-Ramusino C, Lopes M, Sogo J, Conti C, et al.  (2006) Methods to study replication fork collapse in budding yeast. Methods Enzymol  409: 442–462.  <ul class="find" data-citedArticleID="1025396" data-doi="10.1016/s0076-6879(05)09026-9"><li><a href="http://dx.doi.org/10.1016/s0076-6879(05)09026-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methods+to+study+replication+fork+collapse+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methods+to+study+replication+fork+collapse+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001257-Hadden1" id="pone.0001257-Hadden1"></a>Hadden JM, Convery MA, Declais AC, Lilley DM, Phillips SE (2001) Crystal structure of the Holliday junction resolving enzyme T7 endonuclease I. Nat Struct Biol  8: 62–67.  <ul class="find" data-citedArticleID="1025374" data-doi="10.1038/83067"><li><a href="http://dx.doi.org/10.1038/83067" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+Holliday+junction+resolving+enzyme+T7+endonuclease+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+Holliday+junction+resolving+enzyme+T7+endonuclease+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001257-Sears1" id="pone.0001257-Sears1"></a>Sears J, Ujihara M, Wong S, Ott C, Middeldorp J, et al.  (2004) The amino terminus of Epstein-Barr Virus (EBV) nuclear antigen 1 contains AT hooks that facilitate the replication and partitioning of latent EBV genomes by tethering them to cellular chromosomes. J Virol  78: 11487–11505.  <ul class="find" data-citedArticleID="1025432" data-doi="10.1128/jvi.78.21.11487-11505.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.21.11487-11505.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+amino+terminus+of+Epstein-Barr+Virus+%28EBV%29+nuclear+antigen+1+contains+AT+hooks+that+facilitate+the+replication+and+partitioning+of+latent+EBV+genomes+by+tethering+them+to+cellular+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+amino+terminus+of+Epstein-Barr+Virus+%28EBV%29+nuclear+antigen+1+contains+AT+hooks+that+facilitate+the+replication+and+partitioning+of+latent+EBV+genomes+by+tethering+them+to+cellular+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001257-Ballestas1" id="pone.0001257-Ballestas1"></a>Ballestas ME, Chatis PA, Kaye KM (1999) Efficient persistence of extrachromosomal KSHV DNA mediated by latency-associated nuclear antigen. Science  284: 641–644.  <ul class="find" data-citedArticleID="1025340" data-doi="10.1126/science.284.5414.641"><li><a href="http://dx.doi.org/10.1126/science.284.5414.641" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+persistence+of+extrachromosomal+KSHV+DNA+mediated+by+latency-associated+nuclear+antigen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+persistence+of+extrachromosomal+KSHV+DNA+mediated+by+latency-associated+nuclear+antigen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001257-Sugden2" id="pone.0001257-Sugden2"></a>Sugden B, Leight ER (2001) EBV's plasmid replicon: an enigma in cis and trans. Curr Top Microbiol Immunol  258: 3–11.  <ul class="find" data-citedArticleID="1025446" data-doi="10.1007/978-3-642-56515-1_1"><li><a href="http://dx.doi.org/10.1007/978-3-642-56515-1_1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EBV%27s+plasmid+replicon%3A+an+enigma+in+cis+and+trans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EBV%27s+plasmid+replicon%3A+an+enigma+in+cis+and+trans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001257-Kapoor1" id="pone.0001257-Kapoor1"></a>Kapoor P, Lavoie BD, Frappier L (2005) EBP2 plays a key role in Epstein-Barr virus mitotic segregation and is regulated by aurora family kinases. Mol Cell Biol  25: 4934–4945.  <ul class="find" data-citedArticleID="1025386" data-doi="10.1128/mcb.25.12.4934-4945.2005"><li><a href="http://dx.doi.org/10.1128/mcb.25.12.4934-4945.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EBP2+plays+a+key+role+in+Epstein-Barr+virus+mitotic+segregation+and+is+regulated+by+aurora+family+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EBP2+plays+a+key+role+in+Epstein-Barr+virus+mitotic+segregation+and+is+regulated+by+aurora+family+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001257-Kanda1" id="pone.0001257-Kanda1"></a>Kanda T, Kamiya M, Maruo S, Iwakiri D, Takada K (2007) Symmetrical localization of extrachromosomally replicating viral genomes on sister chromatids. J Cell Sci  120: 1529–1539.  <ul class="find" data-citedArticleID="1025384" data-doi="10.1242/jcs.03434"><li><a href="http://dx.doi.org/10.1242/jcs.03434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Symmetrical+localization+of+extrachromosomally+replicating+viral+genomes+on+sister+chromatids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Symmetrical+localization+of+extrachromosomally+replicating+viral+genomes+on+sister+chromatids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001257-Tauchi1" id="pone.0001257-Tauchi1"></a>Tauchi H, Kobayashi J, Morishima K, van Gent DC, Shiraishi T, et al.  (2002) Nbs1 is essential for DNA repair by homologous recombination in higher vertebrate cells. Nature  420: 93–98.  <ul class="find" data-citedArticleID="1025448" data-doi="10.1038/nature01125"><li><a href="http://dx.doi.org/10.1038/nature01125" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nbs1+is+essential+for+DNA+repair+by+homologous+recombination+in+higher+vertebrate+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nbs1+is+essential+for+DNA+repair+by+homologous+recombination+in+higher+vertebrate+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001257-Nasmyth1" id="pone.0001257-Nasmyth1"></a>Nasmyth K, Schleiffer A (2004) From a single double helix to paired double helices and back. Philos Trans R Soc Lond B Biol Sci  359: 99–108.  <ul class="find" data-citedArticleID="1025412" data-doi="10.1098/rstb.2003.1417"><li><a href="http://dx.doi.org/10.1098/rstb.2003.1417" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+a+single+double+helix+to+paired+double+helices+and+back." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+a+single+double+helix+to+paired+double+helices+and+back.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001257-Shimada1" id="pone.0001257-Shimada1"></a>Shimada K, Gasser SM (2007) The origin recognition complex functions in sister-chromatid cohesion in Saccharomyces cerevisiae. Cell  128: 85–99.  <ul class="find" data-citedArticleID="1025436" data-doi="10.1016/j.cell.2006.11.045"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.11.045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+origin+recognition+complex+functions+in+sister-chromatid+cohesion+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+origin+recognition+complex+functions+in+sister-chromatid+cohesion+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001257-Ritzi1" id="pone.0001257-Ritzi1"></a>Ritzi M, Tillack K, Gerhardt J, Ott E, Humme S, et al.  (2003) Complex protein-DNA dynamics at the latent origin of DNA replication of Epstein-Barr virus. J Cell Sci  116: 3971–3984.  <ul class="find" data-citedArticleID="1025428" data-doi="10.1242/jcs.00708"><li><a href="http://dx.doi.org/10.1242/jcs.00708" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+protein-DNA+dynamics+at+the+latent+origin+of+DNA+replication+of+Epstein-Barr+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+protein-DNA+dynamics+at+the+latent+origin+of+DNA+replication+of+Epstein-Barr+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001257-Zhou1" id="pone.0001257-Zhou1"></a>Zhou J, Chau CM, Deng Z, Shiekhattar R, Spindler MP, et al.  (2005) Cell cycle regulation of chromatin at an origin of DNA replication. Embo J  24: 1406–1417.  <ul class="find" data-citedArticleID="1025468" data-doi="10.1038/sj.emboj.7600609"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600609" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+cycle+regulation+of+chromatin+at+an+origin+of+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+cycle+regulation+of+chromatin+at+an+origin+of+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001257-Davis1" id="pone.0001257-Davis1"></a>Davis PK, Ho A, Dowdy SF (2001) Biological methods for cell-cycle synchronization of mammalian cells. Biotechniques  30: 1322–1326, 1328, 1330–1321.  <ul class="find" data-citedArticleID="1025350"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Biological+methods+for+cell-cycle+synchronization+of+mammalian+cells.&amp;auth=&amp;atitle=Biological+methods+for+cell-cycle+synchronization+of+mammalian+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biological+methods+for+cell-cycle+synchronization+of+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biological+methods+for+cell-cycle+synchronization+of+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001257-Brewer1" id="pone.0001257-Brewer1"></a>Brewer BJ, Fangman WL (1987) The localization of replication origins on ARS plasmids in S. cerevisiae. Cell  51: 463–471.  <ul class="find" data-citedArticleID="1025346" data-doi="10.1016/0092-8674(87)90642-8"><li><a href="http://dx.doi.org/10.1016/0092-8674(87)90642-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+localization+of+replication+origins+on+ARS+plasmids+in+S.+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+localization+of+replication+origins+on+ARS+plasmids+in+S.+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001257-Pichierri1" id="pone.0001257-Pichierri1"></a>Pichierri P, Franchitto A, Rosselli F (2004) BLM and the FANC proteins collaborate in a common pathway in response to stalled replication forks. Embo J  23: 3154–3163.  <ul class="find" data-citedArticleID="1025422" data-doi="10.1038/sj.emboj.7600277"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600277" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BLM+and+the+FANC+proteins+collaborate+in+a+common+pathway+in+response+to+stalled+replication+forks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BLM+and+the+FANC+proteins+collaborate+in+a+common+pathway+in+response+to+stalled+replication+forks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001257-Wang3" id="pone.0001257-Wang3"></a>Wang P, Day L, Dheekollu J, Lieberman PM (2005) A redox-sensitive cysteine in Zta is required for Epstein-Barr virus lytic cycle DNA replication. J Virol  79: 13298–13309.  <ul class="find" data-citedArticleID="1025458" data-doi="10.1128/jvi.79.21.13298-13309.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.21.13298-13309.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+redox-sensitive+cysteine+in+Zta+is+required+for+Epstein-Barr+virus+lytic+cycle+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+redox-sensitive+cysteine+in+Zta+is+required+for+Epstein-Barr+virus+lytic+cycle+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.XML" value="121969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.PDF" value="596739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g001.PNG_L" value="249795"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g001.PNG_M" value="58203"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g001.PNG_S" value="12905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g001.TIF" value="470918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g001.PNG_I" value="75730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g002.PNG_L" value="176055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g002.PNG_M" value="43206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g002.PNG_S" value="10125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g002.TIF" value="195628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g002.PNG_I" value="50887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g003.PNG_L" value="415950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g003.PNG_M" value="49952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g003.PNG_S" value="12162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g003.TIF" value="412122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g003.PNG_I" value="95411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g004.PNG_L" value="308046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g004.PNG_M" value="120583"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g004.PNG_S" value="13435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g004.TIF" value="512920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g004.PNG_I" value="66567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g005.PNG_L" value="1348410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g005.PNG_M" value="106775"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g005.PNG_S" value="17135"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g005.TIF" value="2225740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g005.PNG_I" value="105861"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g006.PNG_L" value="881721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g006.PNG_M" value="195440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g006.PNG_S" value="13099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g006.TIF" value="1134638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g006.PNG_I" value="67557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g007.PNG_L" value="1013794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g007.PNG_M" value="172938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g007.PNG_S" value="13134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g007.TIF" value="1558482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.g007.PNG_I" value="58006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.s001.DOC" value="44544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.s002.TIF" value="4045106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.s003.TIF" value="1878924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.s004.TIF" value="3981304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.s005.TIF" value="7138686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.s006.TIF" value="15931936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001257.s007.TIF" value="8909910"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001257&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001257" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001257&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001257&volume=&issue=&title=A Role for MRE11, NBS1, and Recombination Junctions in Replication and Stable Maintenance of EBV Episomes&author_name=Jayaraju%20Dheekollu%2C%20Zhong%20Deng%2C%20Andreas%20Wiedmer%2C%20Matthew%20D.%20Weitzman%2C%20Paul%20M.%20Lieberman&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001257" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001257" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001257" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001257&amp;t=A%20Role%20for%20MRE11%2C%20NBS1%2C%20and%20Recombination%20Junctions%20in%20Replication%20and%20Stable%20Maintenance%20of%20EBV%20Episomes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001257&title=A%20Role%20for%20MRE11%2C%20NBS1%2C%20and%20Recombination%20Junctions%20in%20Replication%20and%20Stable%20Maintenance%20of%20EBV%20Episomes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001257&amp;title=A%20Role%20for%20MRE11%2C%20NBS1%2C%20and%20Recombination%20Junctions%20in%20Replication%20and%20Stable%20Maintenance%20of%20EBV%20Episomes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001257" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001257&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Role for MRE11, NBS1, and Recombination Junctions in Replication and Stable Maintenance of EBV Episomes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001257';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Role%20for%20MRE11%2C%20NBS1%2C%20and%20Recombination%20Junctions%20in%20Replication%20and%20Stable%20Maintenance%20of%20EBV%20Episomes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001257" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001257" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+cycle+and+cell+division%22" title="Search for articles in the subject area:'Cell cycle and cell division'"><div class="flagText">Cell cycle and cell division</div></a>
              <div data-categoryid="17353" data-articleid="25346"
                   data-categoryname="Cell cycle and cell division"
                   class="flagImage" title="Flag 'Cell cycle and cell division' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+recombination%22" title="Search for articles in the subject area:'DNA recombination'"><div class="flagText">DNA recombination</div></a>
              <div data-categoryid="33483" data-articleid="25346"
                   data-categoryname="DNA recombination"
                   class="flagImage" title="Flag 'DNA recombination' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+replication%22" title="Search for articles in the subject area:'DNA replication'"><div class="flagText">DNA replication</div></a>
              <div data-categoryid="19773" data-articleid="25346"
                   data-categoryname="DNA replication"
                   class="flagImage" title="Flag 'DNA replication' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+structure%22" title="Search for articles in the subject area:'DNA structure'"><div class="flagText">DNA structure</div></a>
              <div data-categoryid="34399" data-articleid="25346"
                   data-categoryname="DNA structure"
                   class="flagImage" title="Flag 'DNA structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Small+interfering+RNA%22" title="Search for articles in the subject area:'Small interfering RNA'"><div class="flagText">Small interfering RNA</div></a>
              <div data-categoryid="19855" data-articleid="25346"
                   data-categoryname="Small interfering RNA"
                   class="flagImage" title="Flag 'Small interfering RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Southern+blot%22" title="Search for articles in the subject area:'Southern blot'"><div class="flagText">Southern blot</div></a>
              <div data-categoryid="49093" data-articleid="25346"
                   data-categoryname="Southern blot"
                   class="flagImage" title="Flag 'Southern blot' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Synthesis+phase%22" title="Search for articles in the subject area:'Synthesis phase'"><div class="flagText">Synthesis phase</div></a>
              <div data-categoryid="19095" data-articleid="25346"
                   data-categoryname="Synthesis phase"
                   class="flagImage" title="Flag 'Synthesis phase' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+replication%22" title="Search for articles in the subject area:'Viral replication'"><div class="flagText">Viral replication</div></a>
              <div data-categoryid="20015" data-articleid="25346"
                   data-categoryname="Viral replication"
                   class="flagImage" title="Flag 'Viral replication' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3155'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4384'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1676&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
