

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Experimental Comparison and Evaluation of the Affymetrix Exon and U133Plus2 GeneChip Arrays</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000913"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000913" />

    <meta name="citation_title" content="Experimental Comparison and Evaluation of the Affymetrix Exon and U133Plus2 GeneChip Arrays"/>
    <meta itemprop="name" content="Experimental Comparison and Evaluation of the Affymetrix Exon and U133Plus2 GeneChip Arrays"/>

      <meta name="citation_author" content="Diana Abdueva"/>
            <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Children's Hospital Los Angeles, Keck School of Medicine, University of Southern California, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Michele R. Wing"/>
            <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Children's Hospital Los Angeles, Keck School of Medicine, University of Southern California, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Betty Schaub"/>
            <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Children's Hospital Los Angeles, Keck School of Medicine, University of Southern California, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Timothy J. Triche"/>
            <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Children's Hospital Los Angeles, Keck School of Medicine, University of Southern California, Los Angeles, California, United States of America"/>

    <meta name="citation_date" content="2007/9/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000913.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e913"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_number=1; " />
      <meta name="citation_reference" content="citation_title=High correspondence between Affymetrix exon and standard expression arrays.; citation_author=MJ Okoniewski; citation_author=Y Hey; citation_author=SD Pepper; citation_author=CJ Miller; citation_journal_title=Biotechniques; citation_volume=42(2); citation_number=2; citation_pages=181-185; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Absolute mRNA concentrations from sequence-specific calibration of oligonucleotide arrays.; citation_author=D Hekstra; citation_author=AR Taussig; citation_author=M Magnasco; citation_author=F Naef; citation_journal_title=Nucleic Acids Res; citation_volume=31(7); citation_number=3; citation_pages=1962-1968; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Modeling oligonucleotide microarray signals.; citation_author=Y Zhang; citation_author=A Ferreira; citation_author=C Cheng; citation_author=Y Wu; citation_author=J Zhang; citation_journal_title=Appl Bioinformatics; citation_volume=5(3); citation_number=4; citation_pages=151-160; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Free energy of DNA duplex formation on short oligonucleotide microarrays.; citation_author=L Zhang; citation_author=C Wu; citation_author=R Carta; citation_author=H Zhao; citation_journal_title=Nucleic Acids Res; citation_volume=35(3); citation_number=5; citation_pages=e18; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Sensitivity, specificity, and the hybridization isotherms of DNA chips.; citation_author=A Halperin; citation_author=A Buhot; citation_author=EB Zhulina; citation_journal_title=Biophys J; citation_volume=86(2); citation_number=6; citation_pages=718-730; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Statistical analysis of adsorption models for oligonucleotide microarrays.; citation_author=CJ Burden; citation_author=YE Pittelkow; citation_author=SR Wilson; citation_journal_title=Stat Appl Genet Mol Biol; citation_volume=3; citation_number=7; citation_pages=Article35; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Robust locally weighted regression and smoothing scatterplots.; citation_author=WS Cleveland; citation_journal_title=J. Amer. Statist. Assoc.; citation_volume=74; citation_number=8; citation_pages=829-836; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Non-linear analysis of GeneChip arrays.; citation_author=D Abdueva; citation_author=D Skvortsov; citation_author=S Tavare; citation_journal_title=Nucleic Acids Res; citation_volume=34(15); citation_number=9; citation_pages=e105; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000913" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Experimental Comparison and Evaluation of the Affymetrix Exon and U133Plus2 GeneChip Arrays"/>
    <meta name="twitter:description" content="BackgroundAffymetrix exon arrays offer scientists the only solution for exon-level expression profiling at the whole-genome scale on a single array. These arrays feature a new chip design with no mismatch probes and a radically new random primed protocol to generate sense DNA targets along the entire length of the transcript. In addition to these changes, a limited number of validating experiments and virtually no experimental data to rigorously address the comparability of all-exon arrays with conventional 3&prime;-arrays result in a natural reluctance to replace conventional expression arrays with the new all-exon platform.MethodologyUsing commercially available Affymetrix arrays, we assess the performance of the Human Exon 1.0 ST (HuEx) and U133 Plus 2.0 (U133Plus2) platforms directly through a series of &lsquo;spike-in&rsquo; hybridizations containing 25 transcripts in the presence of a fixed eukaryotic background. Specifically, we compare the measures of expression for HuEx and U133Plus2 arrays to evaluate the precision of these measures as well as the specificity and sensitivity of the measures' ability to detect differential expression.SignificanceThis study presents an experimental comparison and systematic cross-validation of Affymetrix exon arrays and establishes high comparability of expression changes and probe performance characteristics between Affymetrix conventional and exon arrays. In addition, this study offers a reliable benchmark data set for the comparison of competing exon expression measures, the selection of methods suitable for mapping exon array measures to the wealth of previously generated microarray data, as well as the development of more advanced methods for exon- and transcript-level expression summarization."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000913.g003"/>

  <meta property="og:title" content="Experimental Comparison and Evaluation of the Affymetrix Exon and U133Plus2 GeneChip Arrays" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000913" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7273'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1869'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4444&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000913">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000913" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000913&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Experimental Comparison and Evaluation of the Affymetrix Exon and U133Plus2 GeneChip Arrays
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Diana Abdueva
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:abdueva@usc.edu">abdueva@usc.edu</a></p>

                <p>Affiliation:
                  Department of Pathology and Laboratory Medicine, Children's Hospital Los Angeles, Keck School of Medicine, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michele R. Wing, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology and Laboratory Medicine, Children's Hospital Los Angeles, Keck School of Medicine, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Betty Schaub, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology and Laboratory Medicine, Children's Hospital Los Angeles, Keck School of Medicine, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Timothy J. Triche
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology and Laboratory Medicine, Children's Hospital Los Angeles, Keck School of Medicine, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000913</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000913" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000913" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000913" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000913" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000913-t001" data-doi="info:doi/10.1371/journal.pone.0000913" data-uri="info:doi/10.1371/journal.pone.0000913.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000913-g001" data-doi="info:doi/10.1371/journal.pone.0000913" data-uri="info:doi/10.1371/journal.pone.0000913.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000913-g002" data-doi="info:doi/10.1371/journal.pone.0000913" data-uri="info:doi/10.1371/journal.pone.0000913.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000913-g003" data-doi="info:doi/10.1371/journal.pone.0000913" data-uri="info:doi/10.1371/journal.pone.0000913.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000913">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000913" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Affymetrix exon arrays offer scientists the only solution for exon-level expression profiling at the whole-genome scale on a single array. These arrays feature a new chip design with no mismatch probes and a radically new random primed protocol to generate sense DNA targets along the entire length of the transcript. In addition to these changes, a limited number of validating experiments and virtually no experimental data to rigorously address the comparability of all-exon arrays with conventional 3′-arrays result in a natural reluctance to replace conventional expression arrays with the new all-exon platform.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Using commercially available Affymetrix arrays, we assess the performance of the Human Exon 1.0 ST (HuEx) and U133 Plus 2.0 (U133Plus2) platforms directly through a series of ‘spike-in’ hybridizations containing 25 transcripts in the presence of a fixed eukaryotic background. Specifically, we compare the measures of expression for HuEx and U133Plus2 arrays to evaluate the precision of these measures as well as the specificity and sensitivity of the measures' ability to detect differential expression.</p>


<h3>Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This study presents an experimental comparison and systematic cross-validation of Affymetrix exon arrays and establishes high comparability of expression changes and probe performance characteristics between Affymetrix conventional and exon arrays. In addition, this study offers a reliable benchmark data set for the comparison of competing exon expression measures, the selection of methods suitable for mapping exon array measures to the wealth of previously generated microarray data, as well as the development of more advanced methods for exon- and transcript-level expression summarization.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Abdueva D, Wing MR, Schaub B, Triche TJ (2007) Experimental Comparison and Evaluation of the Affymetrix Exon and U133Plus2 GeneChip Arrays. PLoS ONE 2(9):
          e913.
            doi:10.1371/journal.pone.0000913</p><p><strong>Academic Editor: </strong>Rodolfo Aramayo, Texas A&amp;M University, United States of America</p><p><strong>Received:</strong> June 18, 2007; <strong>Accepted:</strong> August 28, 2007; <strong>Published:</strong> September 19, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>This study was funded by a U01CA114757 grant from the NCI.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>We are witnessing a rapid evolution of microarray technology with the potential for clinical application in diagnostics, therapeutic target identification, patient risk stratification, and pre-clinical toxicology and drug development. Today, Affymetrix GeneChip Exon Array system provides the opportunity to interrogate over one million RNA transcripts on a single array <a href="#pone.0000913-GeneChip1">[1]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The oligonucleotide probes of exon arrays radically differ from those on conventional 3′ expression arrays in their design, density, and coverage. While the conventional Affymetrix GeneChips (e.g. U133Plus2) feature a probeset consisting of 11–20 probes selected from the 3′ end of the mRNA sequence, the new all-exon arrays (e.g. Human Exon 1.0 ST), in contrast, have a mere 4 probes selected from each putative exonic region (see Affymetrix technical documentation for Exon Array probe annotations). To generate the target, HuEx arrays use T7 linked random hexamers for cDNA synthesis as opposed to all previous Affymetrix expression arrays, which employ an oligo-dT linked T7 and thus require an intact poly-A tail. Importantly, this new WT Sense Target Labeling Assay generates DNA targets and therefore results in DNA/DNA duplex formation during hybridization, as opposed to DNA/RNA heteroduplexes in conventional arrays. While first attempts to describe the performance of HuEx in comparison to conventional arrays have been made <a href="#pone.0000913-Okoniewski1">[2]</a>, systematic studies of signal behavior on HuEx arrays have not been described, and comparative analysis of this behavior with 3′ targeted arrays like the U133Plus2 series is unknown. A number of physical models have been proposed to describe DNA/RNA hybridization on expression arrays in the past <a href="#pone.0000913-Hekstra1">[3]</a>, <a href="#pone.0000913-Zhang1">[4]</a>, but none for DNA/DNA interactions as seen on the HuEx arrays. Affymetrix Latin-square spike-in datasets were pivotal in the development of these realistic physical models, reliable analysis algorithms, and the establishment of benchmarks for conventional GeneChip expression measures. However, no similar studies have been published for the new exon arrays to study the DNA/DNA hybridization mechanism. By offering a missing experimental component, our study was designed to evaluate the sensitivity of HuEx signal and the platform's ability to detect differential expression in a wide range of target concentrations as well as to compare all-exon array performance to conventional U133Plus2 arrays.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>This controlled experiment was designed to follow a Latin Square with 25 Human clones arranged in 5 spike-in gene groups at 5 concentrations in the presence of a complex background, repeated in triplicate. Additionally, we used the same experimental design and same samples to generate measures of expression for Affymetrix U133Plus2, thereby allowing an unbiased comparison of the two platforms. This study offers a reliable benchmark data set that may be used for the comparison of 1) competing exon expression measures, 2) the variety of methods suitable for mapping exon array measures to the wealth of previously generated microarray data as well as 3) the development of more advanced exon- and transcript-level expression summarization methods.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Platform Representation of the Selected Clones</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To establish platform performance and cross-platform comparisons, a set of 25 sequence-verified spike-in mRNAs was hybridized to both U133Plus2 and HuEx arrays according to a Latin Square experimental design as described in <a href="#s4">Materials and Methods</a>. Affymetrix probeset annotations were used to establish that these 25 spike-in clones are represented by a total of 112 probesets on U133Plus2 and 1075 probesets on HuEx. It should be noted that while U133Plus2 arrays have an average of one probeset per transcript, our clone selection process was designed to provide an over-representation of probesets on U133Plus2 arrays that would allow addressing possible alternative polyadenylation sites and possible alternative 3′ transcript ends. Due to an oversight in clone selection, one of the clones, I.M.A.G.E. 4843460, was not presented in either U133Plus2 or HuEx arrays, i.e. none of the probes were found to be complementary to the clone sequence. Thus, we exclude this clone from further analysis of probe signal. See <a href="#pone-0000913-t001">Table 1</a> for a detailed summary of clone representation.</p>
<div class="figure" id="pone-0000913-t001"><div class="img"><a name="pone-0000913-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000913" data-uri="info:doi/10.1371/journal.pone.0000913.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000913.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000913.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of Spike-in Clones Representation in Affymetrix Platforms</span></strong></p><span>doi:10.1371/journal.pone.0000913.t001</span></div>

<h4>Signal Response to Concentration and Isotherm Fit</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The majority of both U133Plus2 and HuEx probes that are complementary to spike-in targets respond to concentration changes in a non-linear Langmuir fashion, as previously reported for U95 and U133A arrays <a href="#pone.0000913-Hekstra1">[3]</a>. 78 (or 7.3%) of HuEx 1.0 and 11 (or 3.6%) U133Plus2 probes with sequence complementary to target did not respond to changes in concentrations; see Supporting <a href="#pone.0000913.s001">Tables S1</a> and <a href="#pone.0000913.s002">S2</a> for probe summaries. These probes were excluded from further analysis.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Next we examined the overall probe signal behavior on both platforms. <a href="#pone-0000913-g001">Figure 1</a>, Panel A shows a side-by-side distribution of log2 raw signal intensities with medians of 5.4 and 6.3 for HuEx and U133Plus2 arrays, respectively. The observed decrease in median fluorescent intensity of HuEx arrays can be explained by differences in information content, i.e. number of probes and probesets represented by each platform. HuEx 1.0 arrays were designed to represent annotated transcripts from primary sequence databases and gene prediction sets as well as many unannotated transcripts of unknown function, antisense and intronic probesets. Therefore HuEx contains a higher proportion of targets that are expected to be less frequently or not expressed. Additionally, changes in probe nucleotides composition as well as differences in DNA/DNA vs. DNA/RNA binding properties <a href="#pone.0000913-Zhang2">[5]</a>, <a href="#pone.0000913-Halperin1">[6]</a> may alter mismatch discrimination abilities of the platforms and contribute to differences in dynamic range. In addition to reduced signal intensity, we also observe reduced dynamic range of HuEx signal, see <a href="#pone-0000913-g001">Figure 1</a> Panel.</p>
<div class="figure" id="pone-0000913-g001"><div class="img"><a name="pone-0000913-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000913" data-uri="info:doi/10.1371/journal.pone.0000913.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000913.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000913.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>A) Distribution of array intensities for all probesets in HuEx 1.0 [red] and U133Plus2 [blue].</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>B) Signal response to spike-in concentrations in HuEx 1.0 [red] and U133Plus2 [blue].</p>
<span>doi:10.1371/journal.pone.0000913.g001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Examining individual probe signal response curves, we observe that HuEx 1.0 raw intensity signal follows a hyperbolic function, similar to the one observed in conventional Affymetrix arrays. We fit a non-linear Langmuir-inspired model, introduced in the <a href="#s4">Methods</a> section, to raw signal for both platforms and extract the probe parameters. As shown in <a href="#pone-0000913-g002">Figure 2</a>, Panel A a transformation of raw signal, using parameters, obtained in the fit, collapses the data tightly onto theoretical prediction, suggesting that the Langmuir model thoroughly captures the physical chemistry of GeneChip hybridization on both U133Plus2 and HuEx 1.0 platforms.</p>
<div class="figure" id="pone-0000913-g002"><div class="img"><a name="pone-0000913-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000913" data-uri="info:doi/10.1371/journal.pone.0000913.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000913.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000913.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>A) Model fit for HuEx [red] and U133Plus2 [blue].</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>After each probe has been fit to the model, described in <a href="#s4">Material and Methods</a>, the rescaled variables <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.e001&amp;representation=PNG" class="inline-graphic"></span> overlay the classical Langmuir adsorption isotherm, shown in dotted black. B) Distribution of Initial Slopes for HuEx [red boxplot] and U133Plus2 [blue boxplot] spiked probes. C) Local slopes as a function of concentration for HuEx [median red line and inter-quartile range shaded red] vs U133Plus2 [median blue line and inter-quartile range shaded blue]. Local slopes represent the fitted log2 fold-change for probes with true log2 fold-change of 1 as a function of the nominal concentration. D) Distribution of detection threshold for HuEx 1.0 [red boxplot] and U133Plus2 [blue boxplot] spiked probes. Detection threshold is referred to probe concentration where signal-to-noise ratio exceeds 3.</p>
<span>doi:10.1371/journal.pone.0000913.g002</span></div>

<h4>HuEx 1.0 vs U133Plus2 Cross-platforms Comparison</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Due to saturation, the slope of non-linear signal response to concentration decreases with the increase in spike-in concentration. The maximum sensitivity should then theoretically be observed at small concentrations, providing initial slope as a measure of absolute probe responsiveness. Based on fitted probe parameters, we calculate initial slope for all probes across both platforms at 1 pM and observe that median log2 initial slope in HuEx is 7.0 and is 1.7 times smaller on natural scale than in U133Plus2 with median log2 initial slope equal to 7.7, see <a href="#pone-0000913-g002">Figure 2</a>, Panel B.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Even though we found that the average responsiveness of HuEx 1.0 probes is lesser than U133Plus2 probes i.e. the same change in transcript concentration leads to smaller absolute intensity increase, this characteristic is less important for traditional relative comparison of two or more conditions where the ratio of intensities is calculated. A more informative measure of performance in such comparisons is a log-log slope metric-a slope of signal response to concentration on a log scale that indicates the rate of change, i.e. how much log signal increases in response to a unit of concentration change in different concentration ranges.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Comparison of log-log slopes showed that U133Plus2 is more sensitive, with median slope attaining a maximum of 0.86 in U133Plus2 vs 0.78 in HuEx 1.0, though the difference between platforms in regards to local slopes is significantly less than in absolute probes affinity comparisons; see <a href="#pone-0000913-g002">Figure 2</a>, Panel C.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>The detection sensitivity of both platforms was assessed by measuring the smallest probe concentration at which signal-to-background ratio exceeds 3. We found that median probe detection threshold is 1.84 pM for U133Plus2 and 2.92 pM for HuEx 1.0; see <a href="#pone-0000913-g002">Figure 2</a>, Panel D for detection threshold distributions for both platforms.</p>


<h4>Transcript-Level Summary and Cross-Platform Comparison</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Because the number of competing summarization methods is large and growing, a thorough and systematic assessment of their performance deserves an independent effort and is outside of the scope of this study. While avoiding comparisons of summarization methods, we focus on probeset-level summaries for both platforms, obtained using Affymetrix Expression Console tools, described in <a href="#s4">Material and Methods</a>. Summarized probeset signal response to nominal concentration is shown on <a href="#pone-0000913-g003">Figure 3</a> where Panel A plots log2 summarized expression measures vs. log2 of nominal concentration and Panel B represents the relative bias in concentration detection, i.e. the difference between summarized expression measures and nominal concentration shifted arbitrarily so that the zero value is at 2 pM spike. While an apparent difference in probeset expression measures in different array platforms is observed on Panel A, the ability to detect difference in fold changes for both platforms is comparable as seen in Panel B.</p>
<div class="figure" id="pone-0000913-g003"><div class="img"><a name="pone-0000913-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000913" data-uri="info:doi/10.1371/journal.pone.0000913.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000913.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000913.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000913.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000913.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>A) Summarized expression measures response to log2 concentration in HuEx 1.0 [red] and U133Plus2 [blue].</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>B) Expression bias as a function of log2 concentration in HuEx 1.0 [red] and U133Plus2 [blue].</p>
<span>doi:10.1371/journal.pone.0000913.g003</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>This study presents a comparison of two Affymetrix platforms–market leading U133Plus2 and relatively new all-exon HuEx arrays, which provide for the first time exon-level expression profiling at the whole-genome scale. Despite several major technological changes, we observe a high concordance between these platforms; i.e. individual HuEx probes are capable to reliably detect concentration changes and thus provide unbiased expression measures at exon level. We showed that Langmuir isotherms, devised earlier for previous generations of Affymetrix GeneChips, accurately capture the non-linear relation between measured intensity and concentration for both U133Plus2 and HuEx 1.0 arrays. Using a non-linear physical model, we estimated individual probe parameters and proceeded to compare platform performance in terms of signal sensitivity and specificity. We found that probe intensity changes in response to concentration in HuEx arrays exhibit a smaller dynamic range, higher detection threshold, and decreased sensitivity compared to U133Plus2.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>A critical issue for Affymetrix GeneChip™ users is HuEx compatibility to previous generations of Affymetrix arrays. Vast amounts of U133plus2 and earlier data are publicly available, but their comparability with new GeneChip arrays like the HuEx is unknown. Evidence that data from both sources can be reliably compared is of critical importance. The results reported here are thus reassuring. As we showed in log-log slope analysis, median relative sensitivity is similar in both platforms, suggesting that an accurate cross-platform mapping can be established. However, differences in detection thresholds demand special attention to this problem. While offering a basic summarized signal comparison, we recognize the need for further improvements in summarization routines for all-exon arrays. By releasing the results of this study to the academic community, we hope to facilitate the comparison of competing exon- and transcript-level expression measures, alternative splicing detection, and selection of methods suitable for mapping exon array measures to the wealth of previously generated microarray data.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Clone Selection</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Using publicly available microarray data, deposited in NCBIs Gene Expression Omnibus (GEO, <a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) and accessible through GEO Series accession number GSE5823, DNA clones for this experiment were selected based on their lack of expression in total RNA isolated from HeLa cells. The only other restrictions on clone selection were 1) the presence of both a polyA tail and a XhoI restriction site at the 3′ end of the coding sequence to simplify linearization of cDNA and 2) a size of 1400 to 1800 bases. Due to alternative polyadenylation sites and variant transcripts that may be differently regulated, some genes are represented by multiple probesets in the design of U133Plus2 arrays. In order to address this redundancy and to compare the sensitivity of these probesets, we specifically selected clones that were represented by more than one probeset in U133Plus2 arrays. Clone information is summarized in supplementary materials; see Supporting <a href="#pone.0000913.s003">Table S3</a>.</p>


<h4>Target Preparation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>cDNA clones with ORFs ranging from 1,400 to 1,800 nucleotides were purchased from Open Biosystems (Open Biosystems, Huntsville, AL, USA) as glycerol stocks in the pOTB7 vector. Each cDNA was bacterially amplified, isolated, and sequenced with vector specific primers (M13F and M13R). Following linearization with Xho1, each cDNA was column purified (Qiagen) and <em>in vitro</em> transcribed into mRNA (AmpliScribe(tm) SP6-Flash Transcription Kit). The resulting mRNA was then purified (RNeasy columns-Qiagen), quantified (Nanodrop), and analyzed (Nanochips and Bioanalyzer 2100–Agilent) to ensure full length mRNA products. The 25 full-length mRNAs that passed our requirements were serially diluted and then pooled into 5 ‘spike-in’ RNA mixes. These mixes (&lt;3 ng total) were added to HeLa cell total RNA (10 ug) and aliquots of the same RNA mix were used for carrying out both the WT Sense Target Labeling Assay for HuEx arrays and One-Cycle Target Labeling Assay for U133Plus 2 arrays.</p>


<h4>Experimental Design</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Human cRNA transcripts were spiked into labeled complex human backgrounds at known concentrations, and hybridization intensities were obtained for HuEx and U133Plus2 arrays. Target groups were arranged in a classic Latin square design so that each hybridization mixture contained five targets at each chosen target concentration. 25 cRNA targets were spiked at 5 concentrations that included 0, 2, 32, 128, and 512 pM; see Supporting <a href="#pone.0000913.s003">Table S3</a> for a list of clones. Thus, this data set consists of 3 technical replicates of 5 separate hybridizations of 25 spiked transcripts in a complex human background at concentrations ranging from 0 pM to 512 pM. Hybridization and wash conditions were the same as indicated in the Affymetrix gene expression manuals. Hybridization intensities were generated for the experiments according to the standard procedures for GeneChip expression probe arrays.</p>


<h4>Target Hybridization and Data Collection</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Arrays were washed and stained according to standard Affymetrix protocols using an Affymetrix Fluidics Workstation. Arrays were scanned using an Affymetrix GeneArray scanner and the cell intensities (.CEL files) were captured by the Affymetrix GeneChip Operating Software (GCOS). To allow for direct comparison of probe intensities across multiple arrays within the same platform we applied a locally weighted linear regression, implemented as R routine lowess() <a href="#pone.0000913-Burden1">[7]</a>, independently to each platform in order to normalize raw probe intensities.</p>


<h4>Model-Based Analysis of Probe Response</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The main operational principal of microarrays is based on specific binding of DNA or RNA target to oligonucleotide probes synthesized on the array surface. Previous generations of conventional Affymetrix microarrays (e.g. U133Plus2) relied on the formation of RNA/DNA hetero-duplexes where attached DNA probes featured exact complementary sequences to the target RNA. In contrast, new HuEx arrays rely on more selective DNA/DNA binding with significantly different physical properties. Additionally, the HuEx platform pioneers a new data structure where individual probesets represent distinct exon or RNA transcript expression that can be assembled into global transcript expression measures. These changes in array design and sample preparation protocols present significant challenges in the application of existing analysis methods. One of the main goals of this study is to access and quantify the differences and similarities between conventional and all-exon Affymetrix platforms.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>In recent years, DNA/RNA duplex formation in microarrays has been studied extensively. In particular, it was shown <a href="#pone.0000913-Hekstra1">[3]</a>, <a href="#pone.0000913-Halperin1">[6]</a>, <a href="#pone.0000913-Cleveland1">[8]</a> that the nonlinear response of microarray hybridization intensity to the transcript concentration is best described by the hyperbolic function, known as the Langmuir adsorption isotherm:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.e002&amp;representation=PNG" class="inline-graphic"></span><br>where <em>I<sub>sat</sub></em> is a maximum attainable probe intensity, <em>K</em> is an equilibrium constant, <em>c</em> is target concentration and <em>BG</em> is the background component of the signal, i.e. probe signal in the absence of target transcript that represents probe response to complex genomic background. Nonlinearity, caused by saturation effects, varies significantly for different probes depending on the values of the equilibrium constant <em>K</em>. To assess platform performance, we study and compare signal dynamic range, minimal detectable concentration and responsiveness of probes to concentration changes.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>We assess individual probe performance by fitting the Langmuir isotherm to the hybridization intensity of spiked probes vs known spike concentration. We adapt a non-linear fitting routine <a href="#pone.0000913-Abdueva1">[9]</a> that uses a least square nlm() optimization routine in R. Briefly, multimodel inference, incorporated within the NLFIT fitting routine <a href="#pone.0000913-Abdueva1">[9]</a>, involves a full model above, where <em>c</em> varies across arrays as well as reduced model, where <em>c</em> is constant across all conditions. Evaluating model goodness-of-fit, i.e., measuring quantitatively the agreement between a proposed model and the data, allows to observe that responding probes result in significantly reduced residuals sum square for the full nonlinear model compared to the null model, while probes that do not respond to concentration changes do not benefit from extending the model. Hence, for each probe in U133Plus2 and HuEx we calculate <em>R<sup>2</sup></em> coefficients <em>(RSS<sub>0</sub>-RSS<sub>fit</sub>)/RSS<sub>o</sub></em>, where <em>RSS<sub>0</sub></em> denotes the residual sum of squares from fitting the reduced model and <em>RSS<sub>fit</sub></em> denotes the residual sum of squares from fitting the full model. Probes that demonstrate <em>R<sup>2</sup></em> values above a 0.95 cutoff are considered to be responsive and selected for further performance analysis.</p>


<h4>Log-Log Slope Calculation</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Log-log slopes represent the observed log fold-change for probes with true fold-change of 1 as a function of the total nominal concentration. In the case of Affymetrix Latin Square experiments, where each consecutive experimental concentration doubles in the [1..512 pM] range, local slope is simply the difference between log signal in consecutive experiment. In this study, a reduced Latin Square design was used, resulting in irregular concentration intervals with observation over only 5 concentrations. Thus, a numeric estimation will not be accurate and we resort to estimating log-log slope based on the signal model introduced above, where<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.e003&amp;representation=PNG" class="inline-graphic"></span><br></p>


<h4>Detection Threshold Calculation</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Clearly, the absolute level of background noise will affect the detection threshold of the array platform. The spatial variation of the signal due to random non-specific hybridization reduces sensitivity by adding independent noise and compromises our ability to detect differential expression changes in low concentration ranges. Signal-to-noise (SNR) ratio is used in many signal-detection disciplines as a quantitative measure of the ability to resolve true signal from background noise. In microarrays, SNR quantifies how well specific signal is resolved from the non-specific noise. SNR is commonly calculated as: <em>(Signal-Background)/SD(background)</em>. A higher SNR indicates higher signal over background noise; a signal-to-noise ratio of 3 is commonly considered the lower limit for accurate detection. Signal may be detected below this value, but the accuracy of quantitative measurements decreases significantly.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>In low concentration area, where saturation effects are negligible, the signal can be represented by <em>I = I<sub>sat</sub>·K·c</em> and SNR can be then defined as: <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000913.e004&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec4.sec7.p3" name="article1.body1.sec4.sec7.p3"></a><p>The derivation of the detection threshold requires a priori knowledge of the mean and variance of background. Latin square design, used in this study, allows us to estimate the non-specific background from the zero-spike concentration, where target is not present in the mix. We obtain SNR for each probe in both platforms and define detection threshold as concentration where SNR exceeds 3.</p>


<h4>Transcript Level Summarization</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>We summarized transcript level for U133Plus2 and HuEx 1.0 using standard analysis tools, provided by Affymetrix. For U133Plus2 arrays, we ran Probe Logarithmic Intensity Error Estimation (PLIER) within Affymetrix Expression Console software. For HuEx arrays, Affymetrix Exon Array Computational Tool (ExACT) software package was used to generate PLIER-summarized probeset expressions.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000913"></div><a name="pone.0000913.s001" id="pone.0000913.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000913.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Human Exon Array Spike-In Probes Summary. We assess individual probe performance by fitting the Langmuir isotherm to the hybridization intensity of spiked probes vs known spike concentration. Human Exon Array probes that demonstrate R2 values above a 0.95 cutoff are considered to be responsive and summarized in this table.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000913.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.46 MB XLS)</p>
<a name="pone.0000913.s002" id="pone.0000913.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000913.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">U133Plus2 Array Spike-In Probes Summary. We assess individual probe performance by fitting the Langmuir isotherm to the hybridization intensity of spiked probes vs known spike concentration. U133Plus2 Array probes that demonstrate R2 values above a 0.95 cutoff are considered to be responsive and summarized in this table.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000913.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.09 MB XLS)</p>
<a name="pone.0000913.s003" id="pone.0000913.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000913.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Spike-In Clone Summary.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000913.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to acknowledge the many useful suggestions and direct assistance from Daniel Wai, Elai Davicioni and Dr. Beth Lawlor of the Saban Research Institute and Department of Pathology and Laboratory Medicine at Childrens Hospital Los Angeles, part of the USC Keck School of Medicine.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DA TT MW. Performed the experiments: MW BS. Analyzed the data: DA. Wrote the paper: DA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000913-GeneChip1" id="pone.0000913-GeneChip1"></a>  GeneChip Exon Array Design, <a href="http://www.affymetrix.com/products/arrays/specific/exon.affx">http://www.affymetrix.com/products/array​s/specific/exon.affx</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000913-Okoniewski1" id="pone.0000913-Okoniewski1"></a>Okoniewski MJ, Hey Y, Pepper SD, Miller CJ (2007) High correspondence between Affymetrix exon and standard expression arrays. Biotechniques  42(2): 181–185.  <ul class="find" data-citedArticleID="994008" data-doi="10.2144/000112315"><li><a href="http://dx.doi.org/10.2144/000112315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+correspondence+between+Affymetrix+exon+and+standard+expression+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+correspondence+between+Affymetrix+exon+and+standard+expression+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000913-Hekstra1" id="pone.0000913-Hekstra1"></a>Hekstra D, Taussig AR, Magnasco M, Naef F (2003) Absolute mRNA concentrations from sequence-specific calibration of oligonucleotide arrays. Nucleic Acids Res  31(7): 1962–1968.  <ul class="find" data-citedArticleID="994006" data-doi="10.1093/nar/gkg283"><li><a href="http://dx.doi.org/10.1093/nar/gkg283" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Absolute+mRNA+concentrations+from+sequence-specific+calibration+of+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Absolute+mRNA+concentrations+from+sequence-specific+calibration+of+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000913-Zhang1" id="pone.0000913-Zhang1"></a>Zhang Y, Ferreira A, Cheng C, Wu Y, Zhang J (2006) Modeling oligonucleotide microarray signals. Appl Bioinformatics  5(3): 151–160.  <ul class="find" data-citedArticleID="994010" data-doi="10.2165/00822942-200605030-00003"><li><a href="http://dx.doi.org/10.2165/00822942-200605030-00003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+oligonucleotide+microarray+signals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+oligonucleotide+microarray+signals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000913-Zhang2" id="pone.0000913-Zhang2"></a>Zhang L, Wu C, Carta R, Zhao H (2007) Free energy of DNA duplex formation on short oligonucleotide microarrays. Nucleic Acids Res  35(3): e18.  <ul class="find" data-citedArticleID="994012" data-doi="10.1093/nar/gkl1064"><li><a href="http://dx.doi.org/10.1093/nar/gkl1064" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Free+energy+of+DNA+duplex+formation+on+short+oligonucleotide+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Free+energy+of+DNA+duplex+formation+on+short+oligonucleotide+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000913-Halperin1" id="pone.0000913-Halperin1"></a>Halperin A, Buhot A, Zhulina EB (2004) Sensitivity, specificity, and the hybridization isotherms of DNA chips. Biophys J  86(2): 718–730.  <ul class="find" data-citedArticleID="994004" data-doi="10.1016/s0006-3495(04)74150-8"><li><a href="http://dx.doi.org/10.1016/s0006-3495(04)74150-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sensitivity%2C+specificity%2C+and+the+hybridization+isotherms+of+DNA+chips." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sensitivity%2C+specificity%2C+and+the+hybridization+isotherms+of+DNA+chips.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000913-Burden1" id="pone.0000913-Burden1"></a>Burden CJ, Pittelkow YE, Wilson SR (2004) Statistical analysis of adsorption models for oligonucleotide microarrays. Stat Appl Genet Mol Biol  3: Article35.  <ul class="find" data-citedArticleID="993998" data-doi="10.2202/1544-6115.1095"><li><a href="http://dx.doi.org/10.2202/1544-6115.1095" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+analysis+of+adsorption+models+for+oligonucleotide+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+analysis+of+adsorption+models+for+oligonucleotide+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000913-Cleveland1" id="pone.0000913-Cleveland1"></a>Cleveland WS (1979) Robust locally weighted regression and smoothing scatterplots. J. Amer. Statist. Assoc.  74: 829–836.  <ul class="find" data-citedArticleID="994000" data-doi="10.1080/01621459.1979.10481038"><li><a href="http://dx.doi.org/10.1080/01621459.1979.10481038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robust+locally+weighted+regression+and+smoothing+scatterplots." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robust+locally+weighted+regression+and+smoothing+scatterplots.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000913-Abdueva1" id="pone.0000913-Abdueva1"></a>Abdueva D, Skvortsov D, Tavare S (2006) Non-linear analysis of GeneChip arrays. Nucleic Acids Res  34(15): e105.  <ul class="find" data-citedArticleID="993996" data-doi="10.1093/nar/gkl435"><li><a href="http://dx.doi.org/10.1093/nar/gkl435" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-linear+analysis+of+GeneChip+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-linear+analysis+of+GeneChip+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.XML" value="54021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.PDF" value="269942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.t001.PNG_L" value="155660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.t001.PNG_M" value="94094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.t001.PNG_S" value="14597"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.t001.TIF" value="915908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.t001.PNG_I" value="28244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g001.PNG_L" value="442484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g001.PNG_M" value="57540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g001.PNG_S" value="8873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g001.TIF" value="1300864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g001.PNG_I" value="19768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.e001.PNG" value="8840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.e001.TIF" value="25004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g002.PNG_L" value="1909835"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g002.PNG_M" value="106150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g002.PNG_S" value="12303"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g002.TIF" value="3656188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g002.PNG_I" value="34597"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g003.PNG_L" value="557756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g003.PNG_M" value="72260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g003.PNG_S" value="9400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g003.TIF" value="1831328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.g003.PNG_I" value="24727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.e002.PNG" value="9905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.e002.TIF" value="25720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.e003.PNG" value="11523"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.e003.TIF" value="27272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.e004.PNG" value="9081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.e004.TIF" value="24872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.s001.XLS" value="464384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.s002.XLS" value="89088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000913.s003.XLS" value="16896"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000913&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000913" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000913&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000913&volume=&issue=&title=Experimental Comparison and Evaluation of the Affymetrix Exon and U133Plus2 GeneChip Arrays&author_name=Diana%20Abdueva%2C%20Michele%20R.%20Wing%2C%20Betty%20Schaub%2C%20Timothy%20J.%20Triche&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000913" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000913" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000913" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000913&amp;t=Experimental%20Comparison%20and%20Evaluation%20of%20the%20Affymetrix%20Exon%20and%20U133Plus2%20GeneChip%20Arrays" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000913&title=Experimental%20Comparison%20and%20Evaluation%20of%20the%20Affymetrix%20Exon%20and%20U133Plus2%20GeneChip%20Arrays&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000913&amp;title=Experimental%20Comparison%20and%20Evaluation%20of%20the%20Affymetrix%20Exon%20and%20U133Plus2%20GeneChip%20Arrays" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000913" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000913&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Experimental Comparison and Evaluation of the Affymetrix Exon and U133Plus2 GeneChip Arrays';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000913';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Experimental%20Comparison%20and%20Evaluation%20of%20the%20Affymetrix%20Exon%20and%20U133Plus2%20GeneChip%20Arrays http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000913" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000913" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Background+signal+noise%22" title="Search for articles in the subject area:'Background signal noise'"><div class="flagText">Background signal noise</div></a>
              <div data-categoryid="21733" data-articleid="24664"
                   data-categoryname="Background signal noise"
                   class="flagImage" title="Flag 'Background signal noise' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-RNA+hybridization%22" title="Search for articles in the subject area:'DNA-RNA hybridization'"><div class="flagText">DNA-RNA hybridization</div></a>
              <div data-categoryid="22997" data-articleid="24664"
                   data-categoryname="DNA-RNA hybridization"
                   class="flagImage" title="Flag 'DNA-RNA hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Exon+mapping%22" title="Search for articles in the subject area:'Exon mapping'"><div class="flagText">Exon mapping</div></a>
              <div data-categoryid="47727" data-articleid="24664"
                   data-categoryname="Exon mapping"
                   class="flagImage" title="Flag 'Exon mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Isotherms%22" title="Search for articles in the subject area:'Isotherms'"><div class="flagText">Isotherms</div></a>
              <div data-categoryid="22717" data-articleid="24664"
                   data-categoryname="Isotherms"
                   class="flagImage" title="Flag 'Isotherms' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="24664"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="24664"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Probe+hybridization%22" title="Search for articles in the subject area:'Probe hybridization'"><div class="flagText">Probe hybridization</div></a>
              <div data-categoryid="19653" data-articleid="24664"
                   data-categoryname="Probe hybridization"
                   class="flagImage" title="Flag 'Probe hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Signal+to+noise+ratio%22" title="Search for articles in the subject area:'Signal to noise ratio'"><div class="flagText">Signal to noise ratio</div></a>
              <div data-categoryid="22147" data-articleid="24664"
                   data-categoryname="Signal to noise ratio"
                   class="flagImage" title="Flag 'Signal to noise ratio' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3783'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1277'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7002&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
