

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Origin and Evolution of GALA-LRR, a New Member of the CC-LRR Subfamily: From Plants to Bacteria?</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001694"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001694" />

    <meta name="citation_title" content="Origin and Evolution of GALA-LRR, a New Member of the CC-LRR Subfamily: From Plants to Bacteria?"/>
    <meta itemprop="name" content="Origin and Evolution of GALA-LRR, a New Member of the CC-LRR Subfamily: From Plants to Bacteria?"/>

      <meta name="citation_author" content="Andrey V. Kajava"/>
            <meta name="citation_author_institution" content="Centre de Recherches de Biochimie MacromolÃ©culaire, CNRS, University of Montpellier 1 and 2, Montpellier, France"/>
      <meta name="citation_author" content="Maria Anisimova"/>
            <meta name="citation_author_institution" content="Institute of Computational Science, ETH Zurich, Zurich, Switzerland"/>
            <meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland"/>
      <meta name="citation_author" content="Nemo Peeters"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes Micro-organismes (LIPM), UMR 441-2594 (INRA-CNRS), Castanet-Tolosan, France"/>

    <meta name="citation_date" content="2008/2/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001694.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1694"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The leucine-rich repeat as a protein recognition motif.; citation_author=B Kobe; citation_author=AV Kajava; citation_journal_title=Curr Opin Struct Biol; citation_volume=11; citation_number=1; citation_pages=725-732; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The leucine-rich repeat: a versatile binding motif.; citation_author=B Kobe; citation_author=J Deisenhofer; citation_journal_title=Trends Biochem Sci; citation_volume=19; citation_number=2; citation_pages=415-421; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Structural diversity of leucine-rich repeat proteins.; citation_author=AV Kajava; citation_journal_title=J Mol Biol; citation_volume=277; citation_number=3; citation_pages=519-527; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Ralstonia solanacearum requires F-box-like domain-containing type III effectors to promote disease on several host plants.; citation_author=A Angot; citation_author=N Peeters; citation_author=E Lechner; citation_author=F Vailleau; citation_author=C Baud; citation_journal_title=Proc Natl Acad Sci U S A; citation_number=4; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Inventory and functional analysis of the large Hrp regulon in Ralstonia solanacearum: identification of novel effector proteins translocated to plant host cells through the type III secretion system.; citation_author=S Cunnac; citation_author=A Occhialini; citation_author=P Barberis; citation_author=C Boucher; citation_author=S Genin; citation_journal_title=Mol Microbiol; citation_volume=53; citation_number=5; citation_pages=115-128; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The SCF ubiquitin ligase: insights into a molecular machine.; citation_author=T Cardozo; citation_author=M Pagano; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=5; citation_number=6; citation_pages=739-751; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A flexible motif search technique based on generalized profiles.; citation_author=P Bucher; citation_author=K Karplus; citation_author=N Moeri; citation_author=K Hofmann; citation_journal_title=Comput Chem; citation_volume=20; citation_number=7; citation_pages=3-23; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Evidence in the Legionella pneumophila genome for exploitation of host cell functions and high genome plasticity.; citation_author=C Cazalet; citation_author=C Rusniok; citation_author=H Bruggemann; citation_author=N Zidane; citation_author=A Magnier; citation_journal_title=Nat Genet; citation_volume=36; citation_number=8; citation_pages=1165-1173; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Insights into SCF ubiquitin ligases from the structure of the Skp1-Skp2 complex.; citation_author=BA Schulman; citation_author=AC Carrano; citation_author=PD Jeffrey; citation_author=Z Bowen; citation_author=ER Kinnucan; citation_journal_title=Nature; citation_volume=408; citation_number=9; citation_pages=381-386; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Structural basis of the Cks1-dependent recognition of p27(Kip1) by the SCF(Skp2) ubiquitin ligase.; citation_author=B Hao; citation_author=N Zheng; citation_author=BA Schulman; citation_author=G Wu; citation_author=JJ Miller; citation_journal_title=Mol Cell; citation_volume=20; citation_number=10; citation_pages=9-19; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Conformation of polypeptides and proteins.; citation_author=GN Ramachandran; citation_author=V Sasisekharan; citation_journal_title=Adv Protein Chem; citation_volume=23; citation_number=11; citation_pages=283-438; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Eubacterial proteasomes.; citation_author=A Lupas; citation_author=F Zuhl; citation_author=T Tamura; citation_author=S Wolf; citation_author=I Nagy; citation_journal_title=Mol Biol Rep; citation_volume=24; citation_number=12; citation_pages=125-131; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Exploitation of eukaryotic ubiquitin signaling pathways by effectors translocated by bacterial type III and type IV secretion systems.; citation_author=A Angot; citation_author=A Vergunst; citation_author=S Genin; citation_author=N Peeters; citation_journal_title=PLoS Pathog; citation_volume=3; citation_number=13; citation_pages=e3; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Interaction of the virulence protein VirF of Agrobacterium tumefaciens with plant homologs of the yeast Skp1 protein.; citation_author=B Schrammeijer; citation_author=E Risseeuw; citation_author=W Pansegrau; citation_author=TJ Regensburg-Tuink; citation_author=WL Crosby; citation_journal_title=Curr Biol; citation_volume=11; citation_number=14; citation_pages=258-262; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Protein interaction analysis of SCF ubiquitin E3 ligase subunits from Arabidopsis.; citation_author=EP Risseeuw; citation_author=TE Daskalchuk; citation_author=TW Banks; citation_author=E Liu; citation_author=J Cotelesage; citation_journal_title=Plant J; citation_volume=34; citation_number=15; citation_pages=753-767; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Symbiotic phenotypes and translocated effector proteins of the Mesorhizobium loti strain R7A VirB/D4 type IV secretion system.; citation_author=A Hubber; citation_author=AC Vergunst; citation_author=JT Sullivan; citation_author=PJ Hooykaas; citation_author=CW Ronson; citation_journal_title=Mol Microbiol; citation_volume=54; citation_number=16; citation_pages=561-574; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ralstonia solanacearum requires F-box-like domain-containing type III effectors to promote disease on several host plants.; citation_author=A Angot; citation_author=N Peeters; citation_author=E Lechner; citation_author=F Vailleau; citation_author=C Baud; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=17; citation_pages=14620-14625; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=VirB/D4-dependent protein translocation from Agrobacterium into plant cells.; citation_author=AC Vergunst; citation_author=B Schrammeijer; citation_author=A den Dulk-Ras; citation_author=CM de Vlaam; citation_author=TJ Regensburg-Tuink; citation_journal_title=Science; citation_volume=290; citation_number=18; citation_pages=979-982; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genomic Structure and Phylogeny of the Plant Pathogen Ralstonia solanacearum Inferred from Gene Distribution Analysis.; citation_author=A Guidot; citation_author=P Prior; citation_author=J Schoenfeld; citation_author=S Carrere; citation_author=S Genin; citation_journal_title=J Bacteriol; citation_volume=189; citation_number=19; citation_pages=377-387; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of the plant pathogen Ralstonia solanacearum.; citation_author=M Salanoubat; citation_author=S Genin; citation_author=F Artiguenave; citation_author=J Gouzy; citation_author=S Mangenot; citation_journal_title=Nature; citation_volume=415; citation_number=20; citation_pages=497-502; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Natural transformation-based foreign DNA acquisition in a Ralstonia solanacearum mutS mutant.; citation_author=A Mercier; citation_author=F Bertolla; citation_author=E Passelegue-Robe; citation_author=P Simonet; citation_journal_title=Res Microbiol; citation_volume=158; citation_number=21; citation_pages=537-544; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=ATV: display and manipulation of annotated phylogenetic trees.; citation_author=CM Zmasek; citation_author=SR Eddy; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=22; citation_pages=383-384; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A novel potential surface protein in Trichomonas vaginalis contains a leucine-rich repeat shared by micro-organisms from all three domains of life.; citation_author=RP Hirt; citation_author=N Harriman; citation_author=AV Kajava; citation_author=TM Embley; citation_journal_title=Mol Biochem Parasitol; citation_volume=125; citation_number=23; citation_pages=195-199; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Complex recombination events at the hypermutable minisatellite CEB1 (D2S90).; citation_author=J Buard; citation_author=G Vergnaud; citation_journal_title=Embo J; citation_volume=13; citation_number=24; citation_pages=3203-3210; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A census of protein repeats.; citation_author=EM Marcotte; citation_author=M Pellegrini; citation_author=TO Yeates; citation_author=D Eisenberg; citation_journal_title=J Mol Biol; citation_volume=293; citation_number=25; citation_pages=151-160; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The ubiquitin/26S proteasome pathway, the complex last chapter in the life of many plant proteins.; citation_author=RD Vierstra; citation_journal_title=Trends Plant Sci; citation_volume=8; citation_number=26; citation_pages=135-142; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Codon-substitution models for heterogeneous selection pressure at amino acid sites.; citation_author=Z Yang; citation_author=R Nielsen; citation_author=N Goldman; citation_author=AM Pedersen; citation_journal_title=Genetics; citation_volume=155; citation_number=27; citation_pages=431-449; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Bayes empirical bayes inference of amino acid sites under positive selection.; citation_author=Z Yang; citation_author=WS Wong; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=28; citation_pages=1107-1118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A maximum likelihood method for detecting functional divergence at individual codon sites, with application to gene family evolution.; citation_author=JP Bielawski; citation_author=Z Yang; citation_journal_title=J Mol Evol; citation_volume=59; citation_number=29; citation_pages=121-132; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic screening of Hrp type III-related pathogenicity genes controlled by the HrpB transcriptional activator in Ralstonia solanacearum.; citation_author=T Mukaihara; citation_author=N Tamura; citation_author=Y Murata; citation_author=M Iwabuchi; citation_journal_title=Mol Microbiol; citation_volume=54; citation_number=30; citation_pages=863-875; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of open reading frames unique to a select agent: Ralstonia solanacearum race 3 biovar 2.; citation_author=DW Gabriel; citation_author=C Allen; citation_author=M Schell; citation_author=TP Denny; citation_author=JT Greenberg; citation_journal_title=Mol Plant Microbe Interact; citation_volume=19; citation_number=31; citation_pages=69-79; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Hox complex is the Ralstonia solanacearum species complexâ? in: Bacterial Wilt Disease and the Ralstonia solanacearum species complex: Eds. Allen C, Prior P and Hayward A. C.; citation_author=M Fegan; citation_author=P Prior; citation_number=32; citation_pages=449-461; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mechanism of auxin perception by the TIR1 ubiquitin ligase.; citation_author=X Tan; citation_author=LI Calderon-Villalobos; citation_author=M Sharon; citation_author=C Zheng; citation_author=CV Robinson; citation_journal_title=Nature; citation_volume=446; citation_number=33; citation_pages=640-645; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The FHA domain: a putative nuclear signalling domain found in protein kinases and transcription factors.; citation_author=K Hofmann; citation_author=P Bucher; citation_journal_title=Trends Biochem Sci; citation_volume=20; citation_number=34; citation_pages=347-349; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Interactive program for visualization and modelling of proteins, nucleic acids and small molecules.; citation_author=HE Dayringer; citation_author=A Tramontano; citation_author=SR Sprang; citation_author=RJ Fletterick; citation_journal_title=J Mol Graphics; citation_volume=4; citation_number=35; citation_pages=82-87; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=PROCHECK: a program to check the stereochemical quality of protein structures.; citation_author=RA Laskowski; citation_author=MW McArthur; citation_author=DS Moss; citation_author=JM Thornton; citation_journal_title=J Appl Crystallog; citation_volume=26; citation_number=36; citation_pages=282-291; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Statistical methods for detecting molecular adaptation.; citation_author=Z Yang; citation_author=JP Bielawski; citation_journal_title=Trends In Ecology And Evolution; citation_volume=15; citation_number=37; citation_pages=496-503; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Accuracy and power of the likelihood ratio test in detecting adaptive molecular evolution.; citation_author=M Anisimova; citation_author=JP Bielawski; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=38; citation_pages=1585-1592; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Accuracy and power of bayes prediction of amino acid sites under positive selection.; citation_author=M Anisimova; citation_author=JP Bielawski; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=39; citation_pages=950-958; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detecting amino acid sites under positive selection and purifying selection.; citation_author=T Massingham; citation_author=N Goldman; citation_journal_title=Genetics; citation_volume=169; citation_number=40; citation_pages=1753-1762; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Not so different after all: a comparison of methods for detecting amino acid sites under selection.; citation_author=SL Kosakovsky Pond; citation_author=SD Frost; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=41; citation_pages=1208-1222; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Syst Biol; citation_volume=52; citation_number=42; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach.; citation_author=S Whelan; citation_author=N Goldman; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=43; citation_pages=691-699; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data.; citation_author=O Gascuel; citation_journal_title=Mol Biol Evol; citation_volume=14; citation_number=44; citation_pages=685-695; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Approximate likelihood-ratio test for branches: A fast, accurate, and powerful alternative.; citation_author=M Anisimova; citation_author=O Gascuel; citation_journal_title=Syst Biol; citation_volume=55; citation_number=45; citation_pages=539-552; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of porcine ribonuclease inhibitor, a protein with leucine-rich repeats.; citation_author=B Kobe; citation_author=J Deisenhofer; citation_journal_title=Nature; citation_volume=366; citation_number=46; citation_pages=751-756; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The PyMol Molecular Graphics System.; citation_author=WL DeLano; citation_number=47; citation_date=2002; citation_publisher=DeLano Scientific; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001694" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Origin and Evolution of GALA-LRR, a New Member of the CC-LRR Subfamily: From Plants to Bacteria?"/>
    <meta name="twitter:description" content="The phytopathogenic bacterium Ralstonia solanacearum encodes type III effectors, called GALA proteins, which contain F-box and LRR domains. The GALA LRRs do not perfectly fit any of the previously described LRR subfamilies. By applying protein sequence analysis and structural prediction, we clarify this ambiguous case of LRR classification and assign GALA-LRRs to CC-LRR subfamily. We demonstrate that side-by-side packing of LRRs in the 3D structures may control the limits of repeat variability within the LRR subfamilies during evolution. The LRR packing can be used as a criterion, complementing the repeat sequences, to classify newly identified LRR domains. Our phylogenetic analysis of F-box domains proposes the lateral gene transfer of bacterial GALA proteins from host plants. We also present an evolutionary scenario which can explain the transformation of the original plant LRRs into slightly different bacterial LRRs. The examination of the selective evolutionary pressure acting on GALA proteins suggests that the convex side of their horse-shoe shaped LRR domains is more prone to positive selection than the concave side, and we therefore hypothesize that the convex surface might be the site of protein binding relevant to the adaptor function of the F-box GALA proteins. This conclusion provides a strong background for further functional studies aimed at determining the role of these type III effectors in the virulence of R. solanacearum."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001694.g003"/>

  <meta property="og:title" content="Origin and Evolution of GALA-LRR, a New Member of the CC-LRR Subfamily: From Plants to Bacteria?" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001694" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2920'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9754'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7482&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001694">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001694" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001694&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Origin and Evolution of GALA-LRR, a New Member of the CC-LRR Subfamily: From Plants to Bacteria?
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrey V. Kajava
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:andrey.kajava@crbm.cnrs.fr">andrey.kajava@crbm.cnrs.fr</a></p>

                <p>Affiliation:
                  Centre de Recherches de Biochimie MacromolÃ©culaire, CNRS, University of Montpellier 1 and 2, Montpellier, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maria Anisimova, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Institute of Computational Science, ETH Zurich, Zurich, Switzerland, 
                  Swiss Institute of Bioinformatics, Lausanne, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nemo Peeters
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire des Interactions Plantes Micro-organismes (LIPM), UMR 441-2594 (INRA-CNRS), Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 27, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001694</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001694" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001694" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001694" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001694" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001694-g001" data-doi="info:doi/10.1371/journal.pone.0001694" data-uri="info:doi/10.1371/journal.pone.0001694.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001694.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001694-g002" data-doi="info:doi/10.1371/journal.pone.0001694" data-uri="info:doi/10.1371/journal.pone.0001694.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001694.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001694-g003" data-doi="info:doi/10.1371/journal.pone.0001694" data-uri="info:doi/10.1371/journal.pone.0001694.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001694.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001694">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001694" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The phytopathogenic bacterium <em>Ralstonia solanacearum</em> encodes type III effectors, called GALA proteins, which contain F-box and LRR domains. The GALA LRRs do not perfectly fit any of the previously described LRR subfamilies. By applying protein sequence analysis and structural prediction, we clarify this ambiguous case of LRR classification and assign GALA-LRRs to CC-LRR subfamily. We demonstrate that side-by-side packing of LRRs in the 3D structures may control the limits of repeat variability within the LRR subfamilies during evolution. The LRR packing can be used as a criterion, complementing the repeat sequences, to classify newly identified LRR domains. Our phylogenetic analysis of F-box domains proposes the lateral gene transfer of bacterial GALA proteins from host plants. We also present an evolutionary scenario which can explain the transformation of the original plant LRRs into slightly different bacterial LRRs. The examination of the selective evolutionary pressure acting on GALA proteins suggests that the convex side of their horse-shoe shaped LRR domains is more prone to positive selection than the concave side, and we therefore hypothesize that the convex surface might be the site of protein binding relevant to the adaptor function of the F-box GALA proteins. This conclusion provides a strong background for further functional studies aimed at determining the role of these type III effectors in the virulence of <em>R. solanacearum.</em></p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kajava AV, Anisimova M, Peeters N (2008) Origin and Evolution of GALA-LRR, a New Member of the CC-LRR Subfamily: From Plants to Bacteria? PLoS ONE 3(2):
          e1694.
            doi:10.1371/journal.pone.0001694</p><p><strong>Academic Editor: </strong>Cathal Seoighe, University of Cape Town, South Africa</p><p><strong>Received:</strong> July 5, 2007; <strong>Accepted:</strong> January 24, 2008; <strong>Published:</strong> February 27, 2008</p><p><strong>Copyright:</strong> Â© 2008 Kajava et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>MA was supported by the BBSRC (UK) grant to Prof Ziheng Yang and, at finalizing stage of this study, by the ETH, Zurich. This work is partly supported by a 2007 ANR jeune-chercheur (FR) grant to NP.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Leucine-rich repeats (LRRs) are 20â29-residue sequence motifs present in a number of proteins with diverse functions <a href="#pone.0001694-Kobe1">[1]</a>, <a href="#pone.0001694-Kobe2">[2]</a>. In the 3D structures, each LRR corresponds to one coil of the solenoidal fold. The coils consist of a Î²-strand and mostly Î±-helical elements (can also be 3<sub>10</sub> helix or polyproline helix) connected by loops. The coils are arranged so that all the strands and helices are parallel to a common axis, resulting in a non-globular, horseshoe-shaped molecule with a curved parallel Î²-sheet lining the inner circumference of the horseshoe and the helices flanking the outer circumference. In LRR proteins, a six-residue motif LxxLxL is conserved (x can be any amino acid and L-positions can be occupied by Leu, Val, Ile, and Phe), and in the known structures corresponds to a turn and a consecutive Î²-strand; whereas the remaining parts of repeats may be very different. While the invariant motif of the Î²-region is a characteristic feature of the entire LRR superfamily, the consensus sequences of the variable part suggest several specific subfamilies. LRR proteins can be subdivided into at least seven subfamilies <a href="#pone.0001694-Kobe1">[1]</a>, <a href="#pone.0001694-Kajava1">[3]</a>. The repeats from different subfamilies retain a similar solenoidal fold and non-globular horseshoe shape but differ by 3D structures of individual repeats. Based on sequence analysis, it was concluded that LRRs from different subfamilies never occur concomitantly within one LRR protein <a href="#pone.0001694-Kajava1">[3]</a>. This observation is explained by mutually exclusive inter-coil packing arrangement of LRRs from different subfamilies <a href="#pone.0001694-Kajava1">[3]</a>. Such a relationship for LRRs suggests that LRR proteins of different subfamilies most probably have emerged independently during evolution rather than descended from a common ancestor. In line with this conclusion, the described LRR subfamilies could be assigned to a specific subgroup of eukaryotes or prokaryotes, and share similar functions and cellular locations <a href="#pone.0001694-Kajava1">[3]</a>. For example, the bacterial LRR subfamily with the shortest known LRRs contains only extracellular proteins of Gram-negative bacteria. The Plant-Specific LRR (PS-LRR) subfamily has exclusively extracellular proteins from plants. Proteins of ribonuclease inhibitor-like LRR (RI-LRR) subfamily are intracellular and all belong to the Metazoa kingdom.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Since 1998, when these conclusions were formulated, a large number of new LRR proteins have been identified and several new 3D structures of LRR proteins have been determined <a href="#pone.0001694-Kobe1">[1]</a>. After a lapse of nine years, the classification of the LRRs and most of the previously made conclusions, including the mutual exclusive rule, withstand the test of time. At the same time, the analysis of some newly identified LRRs shows that their assignment within the existing classification of the LRR subfamilies may lead to confusion.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Recently, it was shown that the phytopathogenic bacterium <em>Ralstonia solanacearum</em> encodes several type III effectors, called GALA proteins, that contain F-box and LRR domains <a href="#pone.0001694-Angot1">[4]</a>, <a href="#pone.0001694-Cunnac1">[5]</a>. The F-box domain enables the interaction with SKP1 in the SCF-type E3 ubiquitin ligase protein complex <a href="#pone.0001694-Cardozo1">[6]</a>. Their LRRs (hereafter GALA-LRR) have a specific consensus pattern with characteristic differences from the previously described consensus sequences of LRR subfamilies, especially from the known bacterial LRR subfamilies. On the other hand, among the LRR subfamilies that are closest to <em>R. solanacearum</em> GALA-LRRs there is the Cysteine-Containing LRR (CC-LRR) subfamily of plant, animal and fungi proteins which can also contain the F-box domains and, therefore, may have a similar function. Thus, it was not clear, whether the GALA-LRR proteins are members of the CC-LRR subfamily or they should be assigned to a new LRR subfamily. Here we clarify this ambiguous case by using sequence analysis and molecular modeling. We also focus our analysis on the origin and evolution of GALA proteins from <em>R. solanacearum</em>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Sequence analysis of GALA LRRs</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Analysis of F-box containing GALA proteins from <em>Ralstonia solanacearum</em> shows that their 24-residue long LRRs have a specific consensus pattern that has characteristic differences (<a href="#pone-0001694-g001">Figure 1</a>) from the previously described LRRs <a href="#pone.0001694-Kobe1">[1]</a>, <a href="#pone.0001694-Kajava1">[3]</a>. Comparison of GALA-LRRs with the other known 24-residue LRRs such as typical LRRs, PS-LRRs shows that GALA-LRRs frequently have Ile instead of Leu in position 5, Gly or Ala instead of Leu in position 9, Ala instead of Pro in position 10, and do not have a conserved Leu in position 16. The GALA-LRR consensus motif also has some differences with the 26-residue CC-LRR motif. For example, positions 3 and 16 of the GALA-LRR motif do not have a conserved Cys and position 6 is frequently occupied by Gly instead of Thr.</p>
<div class="figure" id="pone-0001694-g001"><div class="img"><a name="pone-0001694-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001694.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001694" data-uri="info:doi/10.1371/journal.pone.0001694.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001694.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001694.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001694.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001694.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001694.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001694.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001694.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001694.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001694.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Specific consensus patterns of GALA-LRRs and GL-LRRs.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) An arrangement of LRRs (rhombs), F-boxes (rectangles) and BTB domain (ellipse) within new representative proteins of CC-LRR subfamily. The following proteins are shown: GALA4 from <em>R. Solanacearum,</em> strain GMI1000 GenBank accession number <span class="underline">CAD15502</span>; GALA protein from <em>Legionella pneumophila</em> subsp.pneumophila str. Philadelphia 1, <span class="underline">AAU27032</span>; hypothetical protein from <em>Arabidopsis thaliana,</em> <span class="underline">AAF82144</span>; putative regulatory subunit from <em>Gemmata sp. Wa1-1</em>, <span class="underline">AAX07517</span>; hypothetical protein from <em>Parachlamydia sp. UWE25</em>, <span class="underline">CAF23996</span>; hypothetical protein from <em>Parachlamydia sp. UWE25,</em> <span class="underline">CAF24006</span>. (B) Alignment of some known (indicated by *) and newly identified LRR consensus sequences. (C) A consensus sequences of an updated CC-LRR. The boxes over the alignment outline known or putative Î±-helical and Î²-structural regions. Bold uppercase and lowercase letters indicate more than 60% and 20% identity, correspondingly. âOâ denotes an apolar residue, âxâ denotes any residue, â-â is a position of a gap. Numbers below the GALA-LRR consensus sequence show the positions of conserved residues (see also <a href="#pone-0001694-g002">Figure 2A</a>).</p>
<span>doi:10.1371/journal.pone.0001694.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Using the generalized profile technique <a href="#pone.0001694-Bucher1">[7]</a>, a sensitive method for sequence database searches, we found the GALA-LRR type of repeats in about 40 proteins including proteins of the cucurbit crops pathogenic Î²-proteobacterium <em>Acidovorax avenae subsp. Citrulli</em>, the human pathogen, and Î³-proteobacterium <em>Legionella pneumophila</em> <a href="#pone.0001694-Cazalet1">[8]</a>, as well as in the aquatic planctomycete bacterium <em>Gemmata sp.</em> Wal 1. These proteins, unlike <em>R. solanacearum's</em> GALA proteins, don't contain an F-box domain. Sometimes their entire sequence corresponds to the LRR domain (<a href="#pone-0001694-g001">Figure 1A</a>). Some proteins have LRRs that are similar to GALA-LRR (GALA-like or GL-LRR hereafter), however, their consensus sequence has several characteristic differences from GALA-LRRs such as Val instead of Ile in position 5, Leu instead of Ala in position 10, presence of conserved Leu in position 16 (<a href="#pone-0001694-g001">Figure 1B</a>). Remarkably, isolated examples of GALA-LRR are found in GL-LRR domains of two F-box containing proteins from plants (<a href="#pone-0001694-g001">Figure 1A</a>). Sequence database searches with generalized profiles revealed GL-LRRs in more than a hundred LRR proteins. Among them are plant proteins, and also proteins from bacteria <em>(Gemmata sp. Wa1-1</em>, <em>Parachlamydia sp.</em>, <em>Legionella pneumophila</em>, <em>Rhodopirellula baltica)</em>, protists (<em>Entamoeba histolytica</em>, <em>Leishmania</em>, <em>Trypanosoma, Dictyostelium</em>) and animals (<em>Danio, Tetraodon, Drosophila, Anopheles gambiae, Xenopus, Strongylo, and Homo sapiens</em>). Interestingly, some of the GL-LRR proteins from plant, animal and protista also contain F-box domains.</p>


<h4>Place of GALA-LRRs and GL-LRRs in the classification of LRR proteins</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Although, the newly identified GALA-LRRs and GL-LRRs do not perfectly fit any of the previously described consensus sequences of seven LRR subfamilies <a href="#pone.0001694-Kobe1">[1]</a>, <a href="#pone.0001694-Kajava1">[3]</a>, they have some similarities in the consensus sequences with CC-LRRs (<a href="#pone-0001694-g001">Figure 1B</a>). In particular, a characteristic ITD-motif of CC-LRRs (positions 5 to 7) is aligned with similar Igd- and Vtd-motifs of GALA- and GL-LRRs respectively. Furthermore, conserved apolar residues in positions 5, 10, 13, 19, 22 and 24 of the 24-residue-long GALA- and GL-LRRs can be aligned to the 26-residue-long CC-LRRs by deleting a residue in each of the two connecting loop regions of the CC-LRRs (<a href="#pone-0001694-g001">Figure 1B</a>). These loop regions are known to be the most accommodative for such length differences. Interestingly, many of the CC-LRR proteins, similarly to GALA-LRR and GL-LRR proteins, contain F-box domains (<a href="#pone-0001694-g001">Figure 1A</a>). Hence they can share functional similarity in that they recruit proteins, <em>via</em> their LRRs, to the SCF-type E3-ubiquitin ligase complex <a href="#pone.0001694-Cardozo1">[6]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>On the assumption of the membership of GALA and GL-LRRs in the CC-LRR subfamily, the previously proposed CC-LRR consensus <a href="#pone.0001694-Kajava1">[3]</a> requires modifications. The updated CC-LRR consensus sequence is shown on <a href="#pone-0001694-g001">Figure 1C</a>. In this motif, Cys is not the only residue that occurs in positions 3 and 16: the other frequently occurring residues are Thr and Asn in position 3 and Leu, Asn and Ser in position 16. The updated CC-LRR often has Gly in addition to Thr in position 6. Finally, position 9 is frequently occupied by Gly. Database searches with the updated CC-LRR generalized profiles were able to detect such domains in heterogeneous group of about thousand proteins. Among these newly defined CC-LRR proteins there are not only proteins of animal or plant origin, but also proteins from pathogenic or non-pathogenic microorganisms such as bacteria (<em>Parachlamydia sp.),</em> protista (<em>Dictyostelium)</em> and fungi (<em>Cryptococcus, Candida albicans, Candida glabrata, Neurospora</em>). The sequence profiles and search results can be viewed on a dedicated web-page <a href="http://bioinfo.montp.cnrs.fr">http://bioinfo.montp.cnrs.fr</a> (Tools&gt;Profiles&gt;Show Profiles). Interestingly, about 600 of them have an F-box domain. This strongly supports a similar role for these specific LRRs in binding the protein substrate that is then recruited by the SCF-type E3-ubiquitin ligase.</p>


<h4>Structural implications</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The knowledge of even one 3D protein structure in a given sequence alignment provides a powerful means to test the correctness of the whole alignment. In our case, the 3D structure of one CC-LRR protein, the human F-box protein Skp2 is known <a href="#pone.0001694-Schulman1">[9]</a>, <a href="#pone.0001694-Hao1">[10]</a> and was used to verify the alignment of GALA- and GL-LRRs with CC-LRRs. The analysis shows that all conserved and apolar residues of GALA- and GL-LRRs in the suggested alignment (positions 5, 10, 13, 19, 22 and 24) correspond to the residues of Skp2 CC-LRRs that form the hydrophobic core inside of the structure (<a href="#pone-0001694-g001">Figures 1B</a> and <a href="#pone-0001694-g002">2A</a>). In the conserved position 3, GALA-LRRs have an Asn residue and GL-LRRs have a Thr residue instead of a Cys in CC-LRRs. This is an additional support for the alignment, because, in general, position 3 of LRRs tolerates a few amino acid residues including mentioned Asn, Thr and Cys. These residues being in position 3 can form specific hydrogen bonds with the peptide groups of the backbone. The two extra residues in the typical 26-residue-long CC-LRRs compared to the 24-residue-long GALA- and GL-LRRs, in the alignment are located in the loop regions of CC-LRRs connecting Î±-helices and Î²-strands (<a href="#pone-0001694-g001">Figure 1B</a>). The LRRs of Skp2 are variable in length and some of them are 1â2 residues shorter than typical 26-residue CC-LRR. The superposition of the 3D structures of these LRRs revealed that the loops are the most variable regions. In particular, missing residues of the short 24- and 25-residue LRRs of Skp2 are located in the loops. These structures represent good examples of how each of two loops of the CC-LRR can accommodate the loss of one residue. One of these short LRRs from Skp2 crystal structure (residue 2211 to 2235) was used as a template for construction of the GALA-LRR model (see <a href="#s3">Methods</a> for details). <a href="#pone-0001694-g002">Figure 2</a> shows structural models of a single GALA-LRR and a complete set of LRRs (12 repeats) from GALA4 of <em>Ralstonia solanacearum</em> (strain MolK2, personal communication C. Boucher and S. Genin). In <a href="#pone-0001694-g002">Figure 2</a>, the superposition of the GALA-LRR model and the crystal structure of Skp2 demonstrates that the difference between them is in the loops connecting Î±-helices and Î²-strands. The conserved asparagine of GALA-LRR (position 3 in <a href="#pone-0001694-g002">Figure 2</a>) similarly to the majority of the other LRR structures, located right after the Î²-strand so that it is able to form a network of specific hydrogen bonds with these NH and CO groups, thus satisfying their hydrogen-bonding potential in the hydrophobic core of the structure. The conserved bulky aliphatic residues form the hydrophobic core of the structure. The conserved small Ala and Gly residues allow a tighter side-by-side packing of Î±-helices.</p>
<div class="figure" id="pone-0001694-g002"><div class="img"><a name="pone-0001694-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001694.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001694" data-uri="info:doi/10.1371/journal.pone.0001694.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001694.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001694.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001694.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001694.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001694.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001694.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001694.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001694.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001694.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Structural model of GALA-LRR.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) CÎ±-trace superposition of a modeled GALA-LRR and the known CC-LRR from human Skp2 protein <a href="#pone.0001694-Hao1">[10]</a> and RI-LRR from porcine ribonuclease inhibitor <a href="#pone.0001694-Kobe3">[46]</a>. GALA-LRR model is shown in a ball-and-stick representation, CC-LRR is shown by a blue trace and RI-LRR by a magenta trace. Numbering of the conserved GALA-LRR residues is taken from <a href="#pone-0001694-g001">Figure 1</a>. Numbers in red point to positions inferred to be under positive selection. The carbon atoms are in green, oxygen in red, nitrogen in blue. (B) A ribbon diagram of a structural model of the C-terminal LRR domain of GALA4 type III effector protein from <em>R. solanacearum</em> (strain MolK2, region 170 to 460, accession code ZP_00946474). The figure was generated with Pymol <a href="#pone.0001694-DeLano1">[47]</a>. The atomic coordinates of the model are available on request.</p>
<span>doi:10.1371/journal.pone.0001694.g002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The modeling also shows that GALA-, GL-, and CC-LRRs can be packed well together and therefore, in contrast to the other LRR subfamilies, do not have mutually exclusive relationships with CC-LRRs in terms of inter-LRR packing (<a href="#pone-0001694-g003">Figure 3</a>). This conclusion is based on the following analysis. The conserved Î²-structural parts of the known crystal structures of LRR domains from different subfamilies and the model of GALA-LRRs were superimposed with the CC-LRR domain and the side-by-side packing of variable LRR fragments was analyzed (see <a href="#s3">Methods</a> for details). The analysis shows that only the Î±-helices of GALA-LRR and CC-LRR have an energetically favorable âknobs-into-holesâ interface while the superposition of LRRs from the other analyzed subfamilies with the CC-LRR results in âknobs-into-knobsâ packing with steric tensions and voids (<a href="#pone-0001694-g003">Figure 3</a>). For example, the RI-LRR and CC-LRR, PS-LRR and CC-LRR, and SDS22-LRR and CC-LRR interfaces have distances between CÎ± and (or) CÎ² atoms of 2.1-2.7 Ã that are 0.5â1.1 Ã closer than normally allowed limits for such distances <a href="#pone.0001694-Ramachandran1">[11]</a>. This steric tension could be alleviated by a deformation of the LRR Î²-structure, but the distortion of the Î²-structural H-bonds would eventually also lead to the loss in the structure stability. The superposition of the typical LRR and CC-LRR domains does not lead to such close contacts, however, it results in an energetically unfavorable âknobs-into-knobsâ packing with voids (<a href="#pone-0001694-g003">Figure 3</a>). Thus, our analysis suggests that some LRRs with different sequence motifs have an energetically favorable (âpermissiveâ) packing, while simultaneous occurrence of the other ones in the same structure results in unfavorable (âmutually exclusiveâ) packing. The permissive packing of repeats with different consensus sequences may serve as a criterion for their membership in the same subfamily, at the same time as the mutually exclusive packing defines the boundaries between the LRR subfamilies.</p>
<div class="figure" id="pone-0001694-g003"><div class="img"><a name="pone-0001694-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001694.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001694" data-uri="info:doi/10.1371/journal.pone.0001694.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001694.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001694.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001694.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001694.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001694.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001694.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001694.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001694.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001694.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Side-by-side packing of LRRs having different consensus sequence motifs.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>âKnobs-into-holesâ packing of GALA-LRRs against CC LRRs (contoured by a thin line box) and âknobs-into-knobsâ packing of PS-LRRs, RI-LRRs, SDS22-LRRs, typical LRRs, bacterial LRRs against CC-LRRs. The CC-LRR-domain is in blue color. The side-chains that are involved in the inter-domain packing are shown by ball-and-stick representation. Arrows point at the clash sites where CÎ± and (or) CÎ² atoms have distances between of 2.1â2.7 Ã.</p>
<span>doi:10.1371/journal.pone.0001694.g003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>It is worth mentioning that GALA-LRRs are erroneously assigned to the RI-LRR subfamily in the annotation of protein databases on the NCBI Web site (<a href="http://www.ncbi.nlm.nih.gov/entrez">http://www.ncbi.nlm.nih.gov/entrez</a>). In order to dissipate any doubt, <a href="#pone-0001694-g002">Figure 2B</a> displays the apparent difference of GALA-LRR and RI-LRR through the backbone superposition.</p>


<h4>Inferring origin of <em>R. solanacearum</em> GALA proteins</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>GALA F-box domains are functionally related to plant F-box domains <a href="#pone.0001694-Angot1">[4]</a>. Although some bacteria have a proteasome-like compartmentalized protease system <a href="#pone.0001694-Lupas1">[12]</a>, they do not have an ubiquitin-dependent protein degradation system like in eukaryotes. Still several bacteria have in their genome typical eukaryotic E3 ubiquitin ligase-like proteins among which F-box proteins, like the GALA proteins from <em>R. solanacearum</em> <a href="#pone.0001694-Angot2">[13]</a>. These bacterial F-box proteins also often contain eukaryote-like protein-protein interaction domains like LRR, ankyrin and WD40.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We systematically searched all the sequenced eubacterial genomes available (353 genomes available through TIGR Comprehensive Microbial Resource, release 24.0) for the presence of the F-box domain (automatic search with Pfam Hiden Markov Model for F-box (PF00646) service available at TIGR CMR). We only found F-box domains present in one chlamydiae species out of 11 complete sequence available (<em>Candidatus Protochlamydia amoebophila</em> strain UWE25) and in 9 proteobacteria out of 184 complete sequences available (alphaproteobacteria: <em>Mesorhizobium loti, Agrobacterium tumefaciens</em>; betaproteobacterium: <em>Ralstonia solanacearum</em>, gammaproteobacteria: <em>Pseudomonas syringae, Sodalis glossinidius, Coxiella burnetii, legionella pneumophila, Xanthomonas campestris</em> and <em>X. axonopodis</em>). All these positive hits correspond indeed to the presence of a canonical F-box domain. The evidence for functional F-box domains is available for both <em>A. tumefaciens</em> and <em>R. solanacearum</em> F-box containing proteins <a href="#pone.0001694-Angot2">[13]</a>, <a href="#pone.0001694-Schrammeijer1">[14]</a>. A few low scoring hits (in proteins from <em>Borrelia burgdorferi, B. garinii, Chlamydophila caviae, Rhizobium etli, Salmonella tiphimurium</em> and <em>Streptococcus pneumoniae</em>) were inspected and clearly ruled out as being F-box domains (by constraints in primary sequence, see e.g. <a href="#pone.0001694-Risseeuw1">[15]</a>).</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Within the proteobacteria phylum, 9 out of 184 completely sequenced bacteria clearly contain at least one F-box-containing predicted protein. Among the 175 negatively scoring bacteria, we believe we can rule out the presence of âremnantsâ of F-box domain, which could have been indicative of gene loss. Considering such sporadic presence of this F-box domain, the scenario of systematic gene loss appears very unlikely</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>The F-box domain has its only described function in eukaryotic cells and is overrepresented in this kingdom (interpro F-box domain (IPR001810) hits: 735 in <em>A. thaliana,</em> 428 in <em>Caenorhabditis elegans,</em> 120 in humans, and only 46 hits among all bacteria sequence available, mostly in proteobacteria, see above). It is interesting to mention that all the bacteria containing F-box domains in their genome intimately interact with eukaryotes. For example, <em>P. amoebophila, S. glossinidius</em> and <em>M. loti</em> are symbionts of amoeba, insects and plants; <em>A. tumefaciens, R. Solanacearum, P. syringae, X. campestris</em> and <em>X. axonopodis</em> are plant pathogens and <em>C. burnetii</em> and <em>L. pneumophila</em> are human pathogens. Finally, for several of these F-box-containing bacterial proteins injection into their host cells (via specialised bacterial secretion systems) has been proven (<em>M. loti, A. tumefaciens, R. solanacearum</em>) <a href="#pone.0001694-Hubber1">[16]</a>â<a href="#pone.0001694-Vergunst1">[18]</a> or predicted (<em>L. pneumophilae</em>) <a href="#pone.0001694-Cazalet1">[8]</a>.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>Among the seven <em>GALA</em> genes from the <em>R. solanacearum</em> genome (strain GMI1000, (<a href="http://bioinfo.genopole-toulouse.prd.fr/annotation/iANT/bacteria/ralsto/">http://bioinfo.genopole-toulouse.prd.fr/âannotation/iANT/bacteria/ralsto/</a>), <em>GALA1(</em>RSp0914) is located in an alternative codon usage region, <em>GALA2</em>(RSp0672) is flanked by a region duplicated elsewhere in the genome and <em>GALA3</em>(RSp0028) is flanked at either side by an alternative codon usage region. These genomic characteristic have been previously identified as potential signatures of LGT <a href="#pone.0001694-Guidot1">[19]</a>, <a href="#pone.0001694-Salanoubat1">[20]</a>. Furthermore, considering the capacity of <em>R. solanacearum</em> to uptake DNA <a href="#pone.0001694-Mercier1">[21]</a>, it is natural to suggest a lateral gene transfer (LGT) from host plant DNA that gave rise to the F-box domain (and possibly the LRRs) of the GALA proteins.</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>One way of testing such a hypothesis is through phylogenetic analysis of the protein origins to identify putative donor for a potential LGT. Apart from the F-box domain, the F-box-containing proteins are constituted of repeat domains (mostly LRRs). Thus phylogenetic inference on the whole protein for large numbers of divergent taxa appears highly problematic. Instead we focused the analysis on the F-box domain, using 50 aa on their own, as well as together with 150 aa from the downsteam F-box-adjacent region or a shorter region containing 2 or 3 LRRs. Fifteen large datasets were assembled to include homologues from the broadest possible species range. The datasets varied by the similarity thresholds used in the profile searches <a href="#pone.0001694-Bucher1">[7]</a>, criteria used to align the sequences (profile, penalties), amount of gaps, and included from 217 to 2853 taxa (and from 64 to 217 aa in the alignment). Trees were inferred with neighbour-joining and maximum likelihood methods (see <a href="#s3">methods</a> section). The resulted (approx.) 30 inferred phylogenies had low average branch supports, which demonstrated that phylogenetic inference for our data (with many short sequences of deep divergences and large percentage of gaps) is indeed problematic, even when considering the conserved F-box domain. The choice of the analysis model did not influence the inference much but considerably more variation in branching order was observed when analyzing different sets of sequences and alignments. In particular, varying the amount of gaps in the alignment had an impact. However, most phylogenies favoured the scenario where all <em>R. solanacearum</em> GALA genes clustered together and with <em>Arabidopsis thaliana</em> or with <em>Oryza sativa</em> as their closest basal lineages (for a representative example of an inferred Maximum Likelihood (ML) tree see <a href="#pone.0001694.s002">Figure S1</a> that is a supplemental file phymlGALA.tre, which can be viewed with ATV-forester <a href="#pone.0001694-Zmasek1">[22]</a> from <a href="http://www.phylosoft.org/atv/">www.phylosoft.org/atv/</a>). Only in some rare cases we also observed that one or two GALA genes were grouped with a non-plant lineage (but with low clade supports). Overall, the underlying phylogenetic signal appears to be in favour of the postulated LGT from a plant lineage. At the same time, the limited accuracy of inference does not enable us to confidently suggest a putative donor.</p>
<a id="article1.body1.sec2.sec4.p7" name="article1.body1.sec2.sec4.p7"></a><p>Our conclusion that the GALA-LRRs belong to the CC-LRR subfamily is consistent with the hypothesis of the lateral transfer. The updated CC-LRR subfamily has proteins from a very heterogeneous group of organisms including animals, plants, fungi, protista, and bacteria. By its wide taxonomic distribution it resembles the TpLRR subfamily <a href="#pone.0001694-Kobe1">[1]</a> that includes proteins found in all three domains of life <a href="#pone.0001694-Hirt1">[23]</a>. The broad distribution of TpLRR proteins also has been explained by LGT. Furthermore, the analysis of TpLRRs suggested that genes linked to pathogenicity can be shared between parasitic bacteria and parasitic eukaryotes <a href="#pone.0001694-Hirt1">[23]</a>. The results of our present analysis of CC-LRRs agree with this hypothesis. The updated CC-LRR subfamily includes many proteins from bacteria (<em>Gemmata sp., Parachlamydia sp., Legionella pneumophila, Rhodopirellula baltica, Ralstonia solanacearum),</em> protista <em>(Entamoeba histolytica, Leishmania, Trypanosoma, Dictyostelium)</em> and fungi (<em>Cryptococcus, Candida albicans, Candida glabrata, Neurospora</em>), among which many are parasitic organisms colonizing plants and animals.</p>
<a id="article1.body1.sec2.sec4.p8" name="article1.body1.sec2.sec4.p8"></a><p>Despite the similarity of GALA-LRRs and CC-LRRs, they have some systematic differences (<a href="#pone-0001694-g001">Figure 1B</a>) and it needs to be explained. Usually, only about half of residue positions of LRRs remain conserved over a long evolutionary period <a href="#pone.0001694-Kajava1">[3]</a>. The conservation usually reflects the importance of these residues for the preservation of the structure. However, GALA-LRRs are nearly perfectly repeated and this suggests that they emerged relatively recently. On the other hand, our molecular modeling indicates that GALA- and CC-LRRs fold in very similar structures that can be compatible and well-packed together, if a CC-LRR is inserted between GALA-LRR or <em>visa versa</em> (<a href="#pone-0001694-g003">Figure 3</a>). This conclusion is supported by the fact that two plant F-box-LRR proteins have a couple of GALA-LRRs inserted in GL-LRR tandem arrays (<a href="#pone-0001694-g001">Figure 1A</a>). Considering that CC-LRRs are much more abundant in plants than GALA-LRRs, and based on the above-mentioned facts, we propose the following sequence of evolutionary events that could âtransformâ the CC-LRR into GALA-LRR tandem arrays. First, the accumulation of point mutations may lead to the spontaneous occurrence of the first GALA-LRR and due to the structural complementarities between this new LRR and the CC-LRRs (see previous section) the occurrence of GALA-LRR does not significantly affect the overall structure and stability of the CC-LRR domain. Second, it is known that repetitive sequences can evolve more rapidly than non-repetitive ones <a href="#pone.0001694-Buard1">[24]</a>, <a href="#pone.0001694-Marcotte1">[25]</a>. This applies both to the repeat multiplication and to the repeat deletion. Therefore, once appeared, GALA-LRRs can multiply and CC-LRRs disappear. As a result the plant CC-LRR genes, being acquired by a bacterium, may shed their CC-LRRs replacing them by GALA-LRRs seeded in their original sequences.</p>
<a id="article1.body1.sec2.sec4.p9" name="article1.body1.sec2.sec4.p9"></a><p>Currently it is not clear why <em>R. solanacearum</em> may prefer to generate arrays of GALA-LRRs instead of CC-LRRs. It has been shown that GALA are type III effectors required for virulence of <em>R. solanacearum</em> on three different plants, namely Arabidopsis, Tomato and <em>Medicago truncatula.</em> <a href="#pone.0001694-Angot2">[13]</a>. Furthermore it is very likely that the GALA type III effectors participate in virulence through their action in plant cells as the adaptors in SCF-type E3-ubiquitine ligases <a href="#pone.0001694-Angot1">[4]</a>, <a href="#pone.0001694-Vierstra1">[26]</a>. In the SCF-type E3 ubiquitin ligase complex F-box containing proteins interact through their LRR (or other protein-protein interaction domains) with the protein targets to be ubiquitinated. We propose that a possible conversion from an original F-box and CC-LRR protein to an F-box and GALA-LRR protein was dictated by functional constrains. It is possible that the new GALA-LRRs have better plant-protein target recognition and are more versatile adaptors suitable to detect protein targets from diverse host plants.</p>


<h4>Testing GALA-LRRs for positive selection in an attempt to establish their functional binding sites</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To gain insight into the function of the GALA proteins we examined whether adaptation could have acted on a proportion of protein residues during the evolution of GALA LRRs and identified positions of such sites, using likelihood ratio tests (LRTs) and the Bayesian prediction <a href="#pone.0001694-Yang1">[27]</a>, <a href="#pone.0001694-Yang2">[28]</a>. In the agreement with the evolutionary scenario suggested by us for GALA-LRRs, data sets containing aligned LRRs were used to analyze the strength of selective pressure across its residues since their common ancestor sequence was acquired by the bacterium. To some extent, the evolution of LRRs in one particular GALA protein may be likened to the evolution of the gene family members after a duplication event whereby paralogous genes originate from the common single ancestral sequence <a href="#pone.0001694-Bielawski1">[29]</a>. Using this analogy we studied the process of the accumulation of substitutions at each site in a single LRR by comparing the codon substitutions at the homologous sites in other repeat sequences from the group of orthologous GALA proteins in four different <em>R. solanacearum</em> strains: GMI1000 <a href="#pone.0001694-Angot1">[4]</a>, <a href="#pone.0001694-Salanoubat1">[20]</a>, RS1000 <a href="#pone.0001694-Mukaihara1">[30]</a>, UW551 <a href="#pone.0001694-Gabriel1">[31]</a> and MolK2 (C. Boucher and S. Genin, personal communication). The first two strains belong to the phylotype I and the others to the phylotype II, among the four phylotypes defined previously for this species complex <a href="#pone.0001694-Fegan1">[32]</a>.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The full coding DNA alignment of all LRRs from all the available GALA proteins (&gt;400 sequences) was analyzed and none of the tests gave a significant evidence for positive selection, although parameter estimates hinted that this possibility could not be ruled out. This could mean that only for some GALAs the LRRs accumulated changes due to positive selection, as by averaging over all domains we loose power to detect positive selection affecting only certain GALA proteins. To test this we subdivided the aligned LRR sequences, so that only sequences from the orthologous GALAs were analyzed together. Parameter estimates from these alignments showed that for all GALA proteins 50â70% of the LRR positions are rather conserved while substitutions at remaining sites generally have neutral effect on the fitness of the protein. However for GALA2 both LRTs for positive selection were highly significant (with <em>P</em>-values &lt;0.01). Estimates suggested that 8% of sites evolved under positive selection. For GALA2 LRRs, the Bayesian approach detected positions 8 and 15 (numbering as in <a href="#pone-0001694-g001">Figures 1</a> and <a href="#pone-0001694-g002">2</a>) with high probability (e.g., using model M8, the corresponding probabilities were 0.97 and 0.99, see <a href="#s3">Methods</a> and <a href="#pone.0001694.s001">Table S1</a> of Supporting Information for details). In accordance with the modeled GALA-LRR structure these residue positions are located in the Î±-helical region and exposed to the solution (<a href="#pone-0001694-g002">Figure 2A</a>). In the LRR domains these positions are located on the convex surface of the horseshoe shaped structure (<a href="#pone-0001694-g002">Figure 2B</a>).</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>For GALA7 LRRs, only the LRT comparing M7 vs. M8 supported positive selection (<em>P</em>-value &lt;0.05), but the estimate of the Ï ratio (describing selective pressure) was only slightly higher than 1 (Ïâ1.15), indicating the lack of clear support for positive selection signal. Model M1a that does not allow positive selection described data equally as well as model M2a that allows positive selection. The Bayesian inference suggested that positions 4, 8, 11, 15 and 17 had a slightly elevated ratio of nonsynonymous to synonymous changes. Changes at these sites at the very least should be neutral to the fitness of the protein but may have a mild advantageous effect, possibly indicating a recent increase of adaptive pressure. The same can be concluded about position 4 in GALA1 and GALA3 LRRs and position 11 in GALA5 LRRs. If mapped on the structural model of the GALA-LRR, most of them (positions 8, 11, 15 and 17) are located on the external side of the Î±-helix and on the convex surface of the LRR solenoid. The side-chain in position 4 belongs to the loop connecting Î²-strand with the Î±-helix and also is exposed to the solvent (<a href="#pone-0001694-g002">Figure 2A</a>).</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>To see if signature of positive selection on GALA 2 and 7 is detectable on the level of the entire LRR domain of these proteins, we analyzed separately the groups of four GALA2 and GALA7 orthologous sequences from the different strains of <em>R. solanacearum</em>. Analysis of both GALA2 and GALA7 LRRs returned highly significant results for both tests (with <em>P</em>-values from 0.0014 to 0.025), providing the evidence of positive selection on both genes. The lack of the strong evidence for positive selection in GALA7 LRRs in the previous analysis suggests that positive selection may affect only certain (orthologous) repeats of GALA7 while the homologous sites in other repeats of this protein evolve neutrally. In this last analysis the number of sequences is too low for the Bayesian prediction to be accurate, and so the results of such inference are used only in an explorative manner, to see if the predicted positive selection sites correspond to any particular repeats and where such sites could be located. Mapping of the predicted sites of GALA2 onto the repeats of the LRR domain (with probability &gt;0.85 from BEB) shows that they are located in position 15 in four LRRs and in position 21 of one LRR and dispersed over the LRR domain mostly on the convex surface (see <a href="#pone.0001694.s003">Figures S2</a> and <a href="#pone.0001694.s004">S3</a> of Supporting Information). Interestingly, the analysis of individual LRRs of GALA2 also pointed at position 15, that is the most represented in the analysis of the entire LRR domain. In the GALA7 LRR domain, these sites are also dispersed over the convex surface of the protein and are found in exactly the same positions as predicted in individual LRRs of GALA7 (positions 4, 11, 15 and 17).</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>Thus, it is encouraging to observe that most inferred positions throughout the LRR domain of orthologous GALAs coincide with those inferred in individual LRR repeats analysis on groups of GALA orthologues. It is important to mention that all positions, that are inferred to be under positive selection, are found on the surface of the structural model of GALA LRR domain. This grants additional support for the GALA-LRR structure prediction described above. Furthermore, our analysis suggests that the convex surface of the horse-shoe shaped GALA-LRR domain is more prone to positive selection than its concave one. It is tempting to propose that the selective pressure leading to an increase of variability on such residues could be the site of protein binding, relevant to the adaptor function of the F-box GALA proteins in the SCF-type E3 ubiquitin ligase. This study provides a strong background for further functional studies.</p>


<h4>Conclusions</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The GALA-LRR example shows that the differences in LRR consensus motifs can be not only mutually exclusive in terms of inter-LRR packing as it was observed in LRRs from different subfamilies <a href="#pone.0001694-Kajava1">[3]</a>, but also permissive as we found in the case of GALA-LRRs and CC-LRRs. The permissive packing may serve as a criterion for the affiliation of LRRs having different consensus sequences with the same LRR subfamily. Therefore, one may expect to find a subfamily of evolutionary related proteins that share similar functions, cellular location, globular domains and, at the same time, having quite different repetitive consensus patterns. The relationships between GALA-, GL- and the other CC-LRRs suggest that structural constraints, namely, permissive packing of repeats may control the limits of the LRR variability within a subfamily. This result provides new insight into the fascinating interplay between the structural constraints and unusual evolutionary dynamics of LRRs and can be used to classify other newly identified LRR domains.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>The <em>R. solanacearum</em> GALA proteins are bacterial F-box proteins containing a new kind of LRR, which can be found in other bacteria, plants and unicellular eukaryotes. These GALA-LRRs and related GL-LRRs are part of the CC-LRR subfamily, which is generally associated with an F-box domain. The presence of this F-box domain in GALA proteins is indicative of the probable ancient lateral transfer from eukaryotic (possibly plant) genes into a <em>R. solanacearum</em> ancestral recipient strain. We further looked into the GALA-LRR in all the GALA sequences available and found that for some GALA proteins there is a strong signature of positively selected residues and only on the convex side of the GALA-LRR structure. This suggests that the GALA proteins, probable E3-ubiquitin ligase adaptors, necessary for the virulence of <em>R. solanacearum</em> on its host plants, could bind their potential protein ligand on the convex side of the LRR domains, similarly to the <em>A. thaliana</em> TIR1 F-box type E3 ubiquitin ligase <a href="#pone.0001694-Tan1">[33]</a>. As we have an experimental system that enables us to test for the functionality of GALA proteins in virulence <a href="#pone.0001694-Angot1">[4]</a>, this current study provides a strong theoretical background for testing the relevance of specific GALA-LRR residues to pathogenesis.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Methods"></a><h3>Methods</h3>
<h4>Sequence profile search</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The generalized sequence profile method and the <em>pftools</em> package <a href="#pone.0001694-Bucher1">[7]</a> were used. Since a single LRR would be unlikely to form a stable structure on its own, we limited the search to proteins containing at least three tandem repeats, thus increasing the selectivity of the search. Several profiles corresponding to different types of LRRs were constructed (GALA-LRR, GL-LRR and updated CC-LRR). The probability that a match is a product of chance alone was calculated by analyzing the score distribution obtained from a profile search against a regionally randomized version of the protein database, assuming an extreme value distribution <a href="#pone.0001694-Hofmann1">[34]</a>. All database searches were performed with a nonredundant data set constructed from 2006 releases of non-redundant protein sequence database including GenPept, Swissprot, PIR, PDF, PDB and NCBI RefSeq and available on the NCBI FTP site (<a href="http://ftp://ftp.ncbi.nih.gov/blast/db">ftp://ftp.ncbi.nih.gov/blast/db</a>). The sequence profiles and results of the search can be viewed on a dedicated web-page <a href="http://bioinfo.montp.cnrs.fr/profiles">http://bioinfo.montp.cnrs.fr/profiles</a> (Tools/Profiles/Show Profiles).</p>


<h4>Molecular Modelling</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The initial template for GALA-LRRs was taken from a 24-residue LRR (residues 2211 to 2235) of the known crystal structure of human Skp2 protein <a href="#pone.0001694-Schulman1">[9]</a>, <a href="#pone.0001694-Hao1">[10]</a> (see Results and Discussion) using the Insight II program <a href="#pone.0001694-Dayringer1">[35]</a>. The amino acid sequence of the template was edited in accordance with the GALA sequences using the homology modeling option of Insight II program. The structure was further refined by the energy minimization procedure based on the steepest descent algorithm implemented in the Discovery subroutine of Insight II, and tethering heavy backbone atoms to their starting conformations with force constant K = 100. The 300 steps of minimization led to a maximum RMS derivative of 0.4 kcal/(mol*Ã). The next stage of minimization was 500 steps of the conjugate gradients algorithm, tethering the backbone atoms with lower force (K = 50), and then 300 steps with K = 25. The tethering was accompanied by setting the distance constraints at K = 50, in order to improve the geometry of H-bonds. To allay the concern that these constraints generated significant tensions in the minimized structure, the last calculation was performed without any restrictions, to an RMS derivative of 0.3 kcal/(mol*Ã). The CVFF force field and the distance dependent dielectric constant were used for the energy calculations. The program PROCHECK <a href="#pone.0001694-Laskowski1">[36]</a> was used to check the quality of the modeled structure. In accordance with the PROCHECK results all residues of the LRR domain of the GALA4 model have backbone conformations from allowed regions of the Ramachandran plot; and G-factors of the polypeptide stereochemistry (a log-odds score based on the observed distribution of the covalent geometry) equal to â0.15. The overall average G-factors for the model is â0.49, values that would be expected for good-quality model.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>To examine the side-by-side packing of LRRs from different subfamilies the following procedure was used. First, fragments of the LRR domains corresponding to different LRR subfamilies were extracted from the known crystal structures (CC-LRR, pdb code 2AST; Typical LRR, pdb code 2O6Q; Bacterial LRR, pdb code 1G9U; SDS22-like LRR, pdb code 1D0B; RI-LRR, pdb code 2BNH; PS-LRR, pdb code 1OGQ). Second, each of these structures and the structural model of GALA-LRRs were superimposed with CC-LRR domain. For the superposition, the conserved Î²-structural parts of the LRRs were used. Two adjacent Î²-strands of CC-LRR and the analyzed structures were superimposed and the side-by-side packing of the variable LRR fragments was analyzed.</p>


<h4>Tests for positive selection</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The selective pressure at the protein level was measured by the ratio of nonsynonymous to synonymous rates <em>Ï = d</em><sub>N</sub>/<em>d</em><sub>S</sub>, with <em>Ï</em>&lt;1, = 1, or &gt;1 indicating conserved, neutral or adaptive evolution respectively <a href="#pone.0001694-Yang3">[37]</a>. Selective pressure was evaluated using the probabilistic Markov models of codon substitution <a href="#pone.0001694-Yang1">[27]</a>, <a href="#pone.0001694-Yang2">[28]</a>. Such models describe the substitution process based on a multiple alignment tree. The transitions from one codon state to another are described by the transition probability matrix over time <em>t</em> as <em>P</em>(<em>t</em>) = exp(<em>Qt</em>). The generator matrix <em>Q</em> = {<em>q<sub>ij</sub></em>} defines the instantaneous substitution rates at site <em>s</em> from codon <em>i</em> to codon <em>j</em> :<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001694.e001&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Here <em>Ï<sub>j</sub></em> is the frequency of codon <em>j</em>, parameter <em>Îº</em> is the transition/transversion ratio, and <em>Ï</em><sup>(<em>s</em>)</sup> is the <em>Ï</em> ratio for site <em>s</em>. The codon substitution process is assumed independent among sites, and model parameters are estimated by maximizing the log-likelihood function of sequence data <em>X</em> = {<em>x</em><sub>s</sub>} given a phylogeny with branch lengths Ï and a model Î:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001694.e002&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>The models used in the analysis differed by statistical distributions of the <em>Ï</em> ratio used to describe the variation of selective pressure along a sequence. Likelihood ratio test (LRT) for positive selection compares maximum log-likelihoods of two nested models, one of which allows sites under positive selection while another does not. To test that a model allowing positive selection describes data significantly better, twice the log-likelihood difference is compared to the Ï<sup>2</sup>-distribution with degrees of freedom equal to the difference in the number of free parameters between the two models. We performed two LRTs for positive selection, comparing models M2a and M8 that allow sites with <em>Ï</em>&gt;1 (alternative hypotheses) with simpler models M1a and M7 respectively that do not allow sites with <em>Ï</em>&gt;1 (null hypotheses). Model M1a (nearly-neutral) assumes two site classes in proportions <em>p</em><sub>0</sub> and <em>p</em><sub>1</sub> = 1â<em>p</em><sub>0</sub>: one with <em>Ï</em><sub>0</sub> ratio estimated between 0 and 1, and the other with <em>Ï</em><sub>1</sub> fixed at 1. The alternative model M2a (positive selection) extends the null model M1a by adding a proportion <em>p</em><sub>2</sub> of positively selected sites with <em>Ï</em><sub>2</sub>&gt;1, estimated from data. The second LRT uses the null model M7 (beta) that assumes the <em>Ï</em> ratio is drawn from a beta distribution defined between 0 and 1. The alternative model M8 has an extra class of sites under positive selection with <em>Ï</em>&gt;1.</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>We also considered two other codon models: the most simple one-ratio model M0, where <em>Ï</em> is assumed to be constant over all sites in the sequence, and the discrete model M3 that allows three discrete classes of sites with ratios <em>Ï</em><sub>0</sub>, <em>Ï</em><sub>1</sub>, and <em>Ï</em><sub>2</sub> occurring in proportions <em>p</em><sub>0</sub>, <em>p</em><sub>1</sub> and <em>p</em><sub>2</sub> = 1â<em>p</em><sub>0</sub>â<em>p</em><sub>1</sub>. Models M0 and M3 are also nested, and can be used to perform the LRT for heterogeneity of selective pressure along the sequence <a href="#pone.0001694-Anisimova1">[38]</a>. This test is often significant, as most coding data has significantly heterogeneous selective pressures acting on different sites of the sequence, according to their functional importance and the role in the protein folding and stability. In comparison with models M8 and M2a, model M3 better combines the algorithmic simplicity with sufficient complexity necessary to reflect heterogeneity of selection pressure in nature. This model is often used to evaluate the underlying distribution of the selective pressure across sites in a sequence. Inconsistencies in estimates under different models may be a sign that the algorithm has not converged to a global optimum. To insure proper convergence, we performed repeated runs for each model (with different starting values) and confirmed that the distribution of selective pressure described by estimates under models M2a and M8 were compatible with the distribution estimated under M3 for all datasets analyzed.</p>
<a id="article1.body1.sec3.sec3.p5" name="article1.body1.sec3.sec3.p5"></a><p>Where a LRT for positive selective pressure was significant, we used the Bayesian inference to calculate posterior probabilities that a site belongs to a particular site class. The posterior distribution of the parameter of interest (in our case Ï) is proportional to the product of its assumed prior distribution and the likelihood of the observed data given this prior. In this study we used the Bayesian Empirical Bayesian approach <a href="#pone.0001694-Yang2">[28]</a>, where the posteriors are obtained by integrating over the prior distribution of selection-related parameters, while setting other model parameters to their maximum likelihood estimates. Sites with high posteriors probabilities (&gt;0.95) of coming from a class with Ï&gt;1 are likely to have evolved under positive selection. Anisimova <em>et al</em>. <a href="#pone.0001694-Anisimova2">[39]</a> showed that the major factor affecting the accuracy Bayesian site prediction is the diversity of the data set and the number of sequences used. The sequence length was shown to have little effect on the accuracy of this method. We therefore believe that our LRRs analysis should have good accuracy as short sequence length may be compensated by the diversity and large numbers of sequences used in this study. Several site-by-site studies support this notion <a href="#pone.0001694-Massingham1">[40]</a>, <a href="#pone.0001694-KosakovskyPond1">[41]</a>. While extreme levels of sequence divergence do not seem to compromise the accuracy of the LRTs, the Bayesian prediction becomes unreliable <a href="#pone.0001694-Yang2">[28]</a>, <a href="#pone.0001694-Mukaihara1">[30]</a>. In this study the divergence levels ranged from 0.21 to 0.34 nucleotide changes per codon per branch (calculated as the tree length divided by the number of branches in the unrooted tree; see <a href="#pone.0001694.s001">Table S1</a>). This corresponds to the optimal divergence levels and so insures good accuracy of Bayesian prediction results reported here.</p>
<a id="article1.body1.sec3.sec3.p6" name="article1.body1.sec3.sec3.p6"></a><p>All ML phylogenies were inferred using PHYML program <a href="#pone.0001694-Guindon1">[42]</a>. The phylogenies used to perform the selection tests was inferred using coding sequences under HKY+Î, and all phylogenies used for testing LGT hypothesis were inferred under WAG model <a href="#pone.0001694-Whelan1">[43]</a>. Phylogenies for testing LGT hypothesis were also inferred using fast neighbor-joining method implemented in BIONJ <a href="#pone.0001694-Gascuel1">[44]</a>, <a href="#pone.0001694-Anisimova3">[45]</a>. Branch supports were inferred using approximate LRT <a href="#pone.0001694-Anisimova3">[45]</a> and 100 bootstrap replicates where computation permitted.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001694"></div><a name="pone.0001694.s001" id="pone.0001694.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001694.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Maximum likelihood estimates of selection parameters for codon models</p>
<p class="siDoi">doi:10.1371/journal.pone.0001694.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001694.s002" id="pone.0001694.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001694.s002">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree supplemental file phymlGALA.tre, which can be viewed with ATV-forester from <a href="http://www.phylosoft.org/atv/">www.phylosoft.org/atv/</a></p>
<p class="siDoi">doi:10.1371/journal.pone.0001694.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB TXT)</p>
<a name="pone.0001694.s003" id="pone.0001694.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001694.s003">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Positions of positive selection in GALA2</p>
<p class="siDoi">doi:10.1371/journal.pone.0001694.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001694.s004" id="pone.0001694.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001694.s004">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Positions of positive selection in GALA2</p>
<p class="siDoi">doi:10.1371/journal.pone.0001694.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Analyzed the data: NP MA. Wrote the paper: NP AK MA. Other: Bioinformatics analysis of amino acid sequences, molecular modeling: AK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001694-Kobe1" id="pone.0001694-Kobe1"></a>Kobe B, Kajava AV (2001) The leucine-rich repeat as a protein recognition motif. Curr Opin Struct Biol  11: 725â732.  <ul class="find" data-citedArticleID="1064518" data-doi="10.1016/s0959-440x(01)00266-4"><li><a href="http://dx.doi.org/10.1016/s0959-440x(01)00266-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+leucine-rich+repeat+as+a+protein+recognition+motif." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+leucine-rich+repeat+as+a+protein+recognition+motif.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001694-Kobe2" id="pone.0001694-Kobe2"></a>Kobe B, Deisenhofer J (1994) The leucine-rich repeat: a versatile binding motif. Trends Biochem Sci  19: 415â421.  <ul class="find" data-citedArticleID="1064520" data-doi="10.1016/0968-0004(94)90090-6"><li><a href="http://dx.doi.org/10.1016/0968-0004(94)90090-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+leucine-rich+repeat%3A+a+versatile+binding+motif." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+leucine-rich+repeat%3A+a+versatile+binding+motif.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001694-Kajava1" id="pone.0001694-Kajava1"></a>Kajava AV (1998) Structural diversity of leucine-rich repeat proteins. J Mol Biol  277: 519â527.  <ul class="find" data-citedArticleID="1064516" data-doi="10.1006/jmbi.1998.1643"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.1643" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+diversity+of+leucine-rich+repeat+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+diversity+of+leucine-rich+repeat+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001694-Angot1" id="pone.0001694-Angot1"></a>Angot A, Peeters N, Lechner E, Vailleau F, Baud C, et al.  (2006) Ralstonia solanacearum requires F-box-like domain-containing type III effectors to promote disease on several host plants. Proc Natl Acad Sci U S A.   <ul class="find" data-citedArticleID="1064470" data-doi="10.1073/pnas.0509393103"><li><a href="http://dx.doi.org/10.1073/pnas.0509393103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ralstonia+solanacearum+requires+F-box-like+domain-containing+type+III+effectors+to+promote+disease+on+several+host+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ralstonia+solanacearum+requires+F-box-like+domain-containing+type+III+effectors+to+promote+disease+on+several+host+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001694-Cunnac1" id="pone.0001694-Cunnac1"></a>Cunnac S, Occhialini A, Barberis P, Boucher C, Genin S (2004) Inventory and functional analysis of the large Hrp regulon in Ralstonia solanacearum: identification of novel effector proteins translocated to plant host cells through the type III secretion system. Mol Microbiol  53: 115â128.  <ul class="find" data-citedArticleID="1064492" data-doi="10.1111/j.1365-2958.2004.04118.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04118.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inventory+and+functional+analysis+of+the+large+Hrp+regulon+in+Ralstonia+solanacearum%3A+identification+of+novel+effector+proteins+translocated+to+plant+host+cells+through+the+type+III+secretion+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inventory+and+functional+analysis+of+the+large+Hrp+regulon+in+Ralstonia+solanacearum%3A+identification+of+novel+effector+proteins+translocated+to+plant+host+cells+through+the+type+III+secretion+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001694-Cardozo1" id="pone.0001694-Cardozo1"></a>Cardozo T, Pagano M (2004) The SCF ubiquitin ligase: insights into a molecular machine. Nat Rev Mol Cell Biol  5: 739â751.  <ul class="find" data-citedArticleID="1064488" data-doi="10.1038/nrm1471"><li><a href="http://dx.doi.org/10.1038/nrm1471" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+SCF+ubiquitin+ligase%3A+insights+into+a+molecular+machine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+SCF+ubiquitin+ligase%3A+insights+into+a+molecular+machine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001694-Bucher1" id="pone.0001694-Bucher1"></a>Bucher P, Karplus K, Moeri N, Hofmann K (1996) A flexible motif search technique based on generalized profiles. Comput Chem  20: 3â23.  <ul class="find" data-citedArticleID="1064486" data-doi="10.1016/s0097-8485(96)80003-9"><li><a href="http://dx.doi.org/10.1016/s0097-8485(96)80003-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+flexible+motif+search+technique+based+on+generalized+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+flexible+motif+search+technique+based+on+generalized+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001694-Cazalet1" id="pone.0001694-Cazalet1"></a>Cazalet C, Rusniok C, Bruggemann H, Zidane N, Magnier A, et al.  (2004) Evidence in the Legionella pneumophila genome for exploitation of host cell functions and high genome plasticity. Nat Genet  36: 1165â1173.  <ul class="find" data-citedArticleID="1064490" data-doi="10.1038/ng1447"><li><a href="http://dx.doi.org/10.1038/ng1447" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+in+the+Legionella+pneumophila+genome+for+exploitation+of+host+cell+functions+and+high+genome+plasticity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+in+the+Legionella+pneumophila+genome+for+exploitation+of+host+cell+functions+and+high+genome+plasticity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001694-Schulman1" id="pone.0001694-Schulman1"></a>Schulman BA, Carrano AC, Jeffrey PD, Bowen Z, Kinnucan ER, et al.  (2000) Insights into SCF ubiquitin ligases from the structure of the Skp1-Skp2 complex. Nature  408: 381â386.  <ul class="find" data-citedArticleID="1064546"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Insights+into+SCF+ubiquitin+ligases+from+the+structure+of+the+Skp1-Skp2+complex.&amp;auth=&amp;atitle=Insights+into+SCF+ubiquitin+ligases+from+the+structure+of+the+Skp1-Skp2+complex." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+SCF+ubiquitin+ligases+from+the+structure+of+the+Skp1-Skp2+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+SCF+ubiquitin+ligases+from+the+structure+of+the+Skp1-Skp2+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001694-Hao1" id="pone.0001694-Hao1"></a>Hao B, Zheng N, Schulman BA, Wu G, Miller JJ, et al.  (2005) Structural basis of the Cks1-dependent recognition of p27(Kip1) by the SCF(Skp2) ubiquitin ligase. Mol Cell  20: 9â19.  <ul class="find" data-citedArticleID="1064508" data-doi="10.1016/j.molcel.2005.09.003"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.09.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+of+the+Cks1-dependent+recognition+of+p27%28Kip1%29+by+the+SCF%28Skp2%29+ubiquitin+ligase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+of+the+Cks1-dependent+recognition+of+p27%28Kip1%29+by+the+SCF%28Skp2%29+ubiquitin+ligase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001694-Ramachandran1" id="pone.0001694-Ramachandran1"></a>Ramachandran GN, Sasisekharan V (1968) Conformation of polypeptides and proteins. Adv Protein Chem  23: 283â438.  <ul class="find" data-citedArticleID="1064538" data-doi="10.1016/s0065-3233(08)60402-7"><li><a href="http://dx.doi.org/10.1016/s0065-3233(08)60402-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformation+of+polypeptides+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformation+of+polypeptides+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001694-Lupas1" id="pone.0001694-Lupas1"></a>Lupas A, Zuhl F, Tamura T, Wolf S, Nagy I, et al.  (1997) Eubacterial proteasomes. Mol Biol Rep  24: 125â131.  <ul class="find" data-citedArticleID="1064528" data-doi="10.1023/a:1006803512761"><li><a href="http://dx.doi.org/10.1023/a:1006803512761" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eubacterial+proteasomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eubacterial+proteasomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001694-Angot2" id="pone.0001694-Angot2"></a>Angot A, Vergunst A, Genin S, Peeters N (2007) Exploitation of eukaryotic ubiquitin signaling pathways by effectors translocated by bacterial type III and type IV secretion systems. PLoS Pathog  3: e3.  <ul class="find" data-citedArticleID="1064472" data-doi="10.1371/journal.ppat.0030003"><li><a href="http://dx.doi.org/10.1371/journal.ppat.0030003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploitation+of+eukaryotic+ubiquitin+signaling+pathways+by+effectors+translocated+by+bacterial+type+III+and+type+IV+secretion+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploitation+of+eukaryotic+ubiquitin+signaling+pathways+by+effectors+translocated+by+bacterial+type+III+and+type+IV+secretion+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001694-Schrammeijer1" id="pone.0001694-Schrammeijer1"></a>Schrammeijer B, Risseeuw E, Pansegrau W, Regensburg-Tuink TJ, Crosby WL, et al.  (2001) Interaction of the virulence protein VirF of Agrobacterium tumefaciens with plant homologs of the yeast Skp1 protein. Curr Biol  11: 258â262.  <ul class="find" data-citedArticleID="1064544" data-doi="10.1016/s0960-9822(01)00069-0"><li><a href="http://dx.doi.org/10.1016/s0960-9822(01)00069-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+of+the+virulence+protein+VirF+of+Agrobacterium+tumefaciens+with+plant+homologs+of+the+yeast+Skp1+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+of+the+virulence+protein+VirF+of+Agrobacterium+tumefaciens+with+plant+homologs+of+the+yeast+Skp1+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001694-Risseeuw1" id="pone.0001694-Risseeuw1"></a>Risseeuw EP, Daskalchuk TE, Banks TW, Liu E, Cotelesage J, et al.  (2003) Protein interaction analysis of SCF ubiquitin E3 ligase subunits from Arabidopsis. Plant J  34: 753â767.  <ul class="find" data-citedArticleID="1064540" data-doi="10.1046/j.1365-313x.2003.01768.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.2003.01768.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+interaction+analysis+of+SCF+ubiquitin+E3+ligase+subunits+from+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+interaction+analysis+of+SCF+ubiquitin+E3+ligase+subunits+from+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001694-Hubber1" id="pone.0001694-Hubber1"></a>Hubber A, Vergunst AC, Sullivan JT, Hooykaas PJ, Ronson CW (2004) Symbiotic phenotypes and translocated effector proteins of the Mesorhizobium loti strain R7A VirB/D4 type IV secretion system. Mol Microbiol  54: 561â574.  <ul class="find" data-citedArticleID="1064514" data-doi="10.1111/j.1365-2958.2004.04292.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04292.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Symbiotic+phenotypes+and+translocated+effector+proteins+of+the+Mesorhizobium+loti+strain+R7A+VirB%2FD4+type+IV+secretion+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Symbiotic+phenotypes+and+translocated+effector+proteins+of+the+Mesorhizobium+loti+strain+R7A+VirB%2FD4+type+IV+secretion+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001694-Angot3" id="pone.0001694-Angot3"></a>Angot A, Peeters N, Lechner E, Vailleau F, Baud C, et al.  (2006) Ralstonia solanacearum requires F-box-like domain-containing type III effectors to promote disease on several host plants. Proc Natl Acad Sci U S A  103: 14620â14625.  <ul class="find" data-citedArticleID="1064474" data-doi="10.1073/pnas.0509393103"><li><a href="http://dx.doi.org/10.1073/pnas.0509393103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ralstonia+solanacearum+requires+F-box-like+domain-containing+type+III+effectors+to+promote+disease+on+several+host+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ralstonia+solanacearum+requires+F-box-like+domain-containing+type+III+effectors+to+promote+disease+on+several+host+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001694-Vergunst1" id="pone.0001694-Vergunst1"></a>Vergunst AC, Schrammeijer B, den Dulk-Ras A, de Vlaam CM, Regensburg-Tuink TJ, et al.  (2000) VirB/D4-dependent protein translocation from Agrobacterium into plant cells. Science  290: 979â982.  <ul class="find" data-citedArticleID="1064550" data-doi="10.1126/science.290.5493.979"><li><a href="http://dx.doi.org/10.1126/science.290.5493.979" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VirB%2FD4-dependent+protein+translocation+from+Agrobacterium+into+plant+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VirB%2FD4-dependent+protein+translocation+from+Agrobacterium+into+plant+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001694-Guidot1" id="pone.0001694-Guidot1"></a>Guidot A, Prior P, Schoenfeld J, Carrere S, Genin S, et al.  (2007) Genomic Structure and Phylogeny of the Plant Pathogen Ralstonia solanacearum Inferred from Gene Distribution Analysis. J Bacteriol  189: 377â387.  <ul class="find" data-citedArticleID="1064504" data-doi="10.1128/jb.00999-06"><li><a href="http://dx.doi.org/10.1128/jb.00999-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+Structure+and+Phylogeny+of+the+Plant+Pathogen+Ralstonia+solanacearum+Inferred+from+Gene+Distribution+Analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+Structure+and+Phylogeny+of+the+Plant+Pathogen+Ralstonia+solanacearum+Inferred+from+Gene+Distribution+Analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001694-Salanoubat1" id="pone.0001694-Salanoubat1"></a>Salanoubat M, Genin S, Artiguenave F, Gouzy J, Mangenot S, et al.  (2002) Genome sequence of the plant pathogen Ralstonia solanacearum. Nature  415: 497â502.  <ul class="find" data-citedArticleID="1064542" data-doi="10.1038/415497a"><li><a href="http://dx.doi.org/10.1038/415497a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+the+plant+pathogen+Ralstonia+solanacearum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+the+plant+pathogen+Ralstonia+solanacearum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001694-Mercier1" id="pone.0001694-Mercier1"></a>Mercier A, Bertolla F, Passelegue-Robe E, Simonet P (2007) Natural transformation-based foreign DNA acquisition in a Ralstonia solanacearum mutS mutant. Res Microbiol  158: 537â544.  <ul class="find" data-citedArticleID="1064534" data-doi="10.1016/j.resmic.2007.05.003"><li><a href="http://dx.doi.org/10.1016/j.resmic.2007.05.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+transformation-based+foreign+DNA+acquisition+in+a+Ralstonia+solanacearum+mutS+mutant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+transformation-based+foreign+DNA+acquisition+in+a+Ralstonia+solanacearum+mutS+mutant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001694-Zmasek1" id="pone.0001694-Zmasek1"></a>Zmasek CM, Eddy SR (2001) ATV: display and manipulation of annotated phylogenetic trees. Bioinformatics  17: 383â384.  <ul class="find" data-citedArticleID="1064562" data-doi="10.1093/bioinformatics/17.4.383"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.4.383" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ATV%3A+display+and+manipulation+of+annotated+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ATV%3A+display+and+manipulation+of+annotated+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001694-Hirt1" id="pone.0001694-Hirt1"></a>Hirt RP, Harriman N, Kajava AV, Embley TM (2002) A novel potential surface protein in Trichomonas vaginalis contains a leucine-rich repeat shared by micro-organisms from all three domains of life. Mol Biochem Parasitol  125: 195â199.  <ul class="find" data-citedArticleID="1064510" data-doi="10.1016/s0166-6851(02)00211-6"><li><a href="http://dx.doi.org/10.1016/s0166-6851(02)00211-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+potential+surface+protein+in+Trichomonas+vaginalis+contains+a+leucine-rich+repeat+shared+by+micro-organisms+from+all+three+domains+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+potential+surface+protein+in+Trichomonas+vaginalis+contains+a+leucine-rich+repeat+shared+by+micro-organisms+from+all+three+domains+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001694-Buard1" id="pone.0001694-Buard1"></a>Buard J, Vergnaud G (1994) Complex recombination events at the hypermutable minisatellite CEB1 (D2S90). Embo J  13: 3203â3210.  <ul class="find" data-citedArticleID="1064484"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Complex+recombination+events+at+the+hypermutable+minisatellite+CEB1+%28D2S90%29.&amp;auth=&amp;atitle=Complex+recombination+events+at+the+hypermutable+minisatellite+CEB1+%28D2S90%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+recombination+events+at+the+hypermutable+minisatellite+CEB1+%28D2S90%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+recombination+events+at+the+hypermutable+minisatellite+CEB1+%28D2S90%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001694-Marcotte1" id="pone.0001694-Marcotte1"></a>Marcotte EM, Pellegrini M, Yeates TO, Eisenberg D (1999) A census of protein repeats. J Mol Biol  293: 151â160.  <ul class="find" data-citedArticleID="1064530" data-doi="10.1006/jmbi.1999.3136"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.3136" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+census+of+protein+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+census+of+protein+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001694-Vierstra1" id="pone.0001694-Vierstra1"></a>Vierstra RD (2003) The ubiquitin/26S proteasome pathway, the complex last chapter in the life of many plant proteins. Trends Plant Sci  8: 135â142.  <ul class="find" data-citedArticleID="1064552" data-doi="10.1016/s1360-1385(03)00014-1"><li><a href="http://dx.doi.org/10.1016/s1360-1385(03)00014-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ubiquitin%2F26S+proteasome+pathway%2C+the+complex+last+chapter+in+the+life+of+many+plant+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ubiquitin%2F26S+proteasome+pathway%2C+the+complex+last+chapter+in+the+life+of+many+plant+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001694-Yang1" id="pone.0001694-Yang1"></a>Yang Z, Nielsen R, Goldman N, Pedersen AM (2000) Codon-substitution models for heterogeneous selection pressure at amino acid sites. Genetics  155: 431â449.  <ul class="find" data-citedArticleID="1064556"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites.&amp;auth=&amp;atitle=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001694-Yang2" id="pone.0001694-Yang2"></a>Yang Z, Wong WS, Nielsen R (2005) Bayes empirical bayes inference of amino acid sites under positive selection. Mol Biol Evol  22: 1107â1118.  <ul class="find" data-citedArticleID="1064558" data-doi="10.1093/molbev/msi097"><li><a href="http://dx.doi.org/10.1093/molbev/msi097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayes+empirical+bayes+inference+of+amino+acid+sites+under+positive+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayes+empirical+bayes+inference+of+amino+acid+sites+under+positive+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001694-Bielawski1" id="pone.0001694-Bielawski1"></a>Bielawski JP, Yang Z (2004) A maximum likelihood method for detecting functional divergence at individual codon sites, with application to gene family evolution. J Mol Evol  59: 121â132.  <ul class="find" data-citedArticleID="1064482" data-doi="10.1007/s00239-004-2597-8"><li><a href="http://dx.doi.org/10.1007/s00239-004-2597-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+maximum+likelihood+method+for+detecting+functional+divergence+at+individual+codon+sites%2C+with+application+to+gene+family+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+maximum+likelihood+method+for+detecting+functional+divergence+at+individual+codon+sites%2C+with+application+to+gene+family+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001694-Mukaihara1" id="pone.0001694-Mukaihara1"></a>Mukaihara T, Tamura N, Murata Y, Iwabuchi M (2004) Genetic screening of Hrp type III-related pathogenicity genes controlled by the HrpB transcriptional activator in Ralstonia solanacearum. Mol Microbiol  54: 863â875.  <ul class="find" data-citedArticleID="1064536" data-doi="10.1111/j.1365-2958.2004.04328.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04328.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+screening+of+Hrp+type+III-related+pathogenicity+genes+controlled+by+the+HrpB+transcriptional+activator+in+Ralstonia+solanacearum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+screening+of+Hrp+type+III-related+pathogenicity+genes+controlled+by+the+HrpB+transcriptional+activator+in+Ralstonia+solanacearum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001694-Gabriel1" id="pone.0001694-Gabriel1"></a>Gabriel DW, Allen C, Schell M, Denny TP, Greenberg JT, et al.  (2006) Identification of open reading frames unique to a select agent: Ralstonia solanacearum race 3 biovar 2. Mol Plant Microbe Interact  19: 69â79.  <ul class="find" data-citedArticleID="1064500" data-doi="10.1094/mpmi-19-0069"><li><a href="http://dx.doi.org/10.1094/mpmi-19-0069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+open+reading+frames+unique+to+a+select+agent%3A+Ralstonia+solanacearum+race+3+biovar+2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+open+reading+frames+unique+to+a+select+agent%3A+Ralstonia+solanacearum+race+3+biovar+2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001694-Fegan1" id="pone.0001694-Fegan1"></a>Fegan M, Prior P (2005) Hox complex is the Ralstonia solanacearum species complexâ? in: Bacterial Wilt Disease and the Ralstonia solanacearum species complex: Eds. Allen C, Prior P and Hayward A. C. pp. 449â461.  <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0001694-Tan1" id="pone.0001694-Tan1"></a>Tan X, Calderon-Villalobos LI, Sharon M, Zheng C, Robinson CV, et al.  (2007) Mechanism of auxin perception by the TIR1 ubiquitin ligase. Nature  446: 640â645.  <ul class="find" data-citedArticleID="1064548" data-doi="10.1038/nature05731"><li><a href="http://dx.doi.org/10.1038/nature05731" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+of+auxin+perception+by+the+TIR1+ubiquitin+ligase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+of+auxin+perception+by+the+TIR1+ubiquitin+ligase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001694-Hofmann1" id="pone.0001694-Hofmann1"></a>Hofmann K, Bucher P (1995) The FHA domain: a putative nuclear signalling domain found in protein kinases and transcription factors. Trends Biochem Sci  20: 347â349.  <ul class="find" data-citedArticleID="1064512" data-doi="10.1016/s0968-0004(00)89072-6"><li><a href="http://dx.doi.org/10.1016/s0968-0004(00)89072-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+FHA+domain%3A+a+putative+nuclear+signalling+domain+found+in+protein+kinases+and+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+FHA+domain%3A+a+putative+nuclear+signalling+domain+found+in+protein+kinases+and+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001694-Dayringer1" id="pone.0001694-Dayringer1"></a>Dayringer HE, Tramontano A, Sprang SR, Fletterick RJ (1986) Interactive program for visualization and modelling of proteins, nucleic acids and small molecules. J Mol Graphics  4: 82â87.  <ul class="find" data-citedArticleID="1064494" data-doi="10.1016/0263-7855(86)80002-9"><li><a href="http://dx.doi.org/10.1016/0263-7855(86)80002-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interactive+program+for+visualization+and+modelling+of+proteins%2C+nucleic+acids+and+small+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interactive+program+for+visualization+and+modelling+of+proteins%2C+nucleic+acids+and+small+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001694-Laskowski1" id="pone.0001694-Laskowski1"></a>Laskowski RA, McArthur MW, Moss DS, Thornton JM (1993) PROCHECK: a program to check the stereochemical quality of protein structures. J Appl Crystallog  26: 282â291.  <ul class="find" data-citedArticleID="1064526" data-doi="10.1107/s0021889892009944"><li><a href="http://dx.doi.org/10.1107/s0021889892009944" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001694-Yang3" id="pone.0001694-Yang3"></a>Yang Z, Bielawski JP (2000) Statistical methods for detecting molecular adaptation. Trends In Ecology And Evolution  15: 496â503.  <ul class="find" data-citedArticleID="1064560" data-doi="10.1016/s0169-5347(00)01994-7"><li><a href="http://dx.doi.org/10.1016/s0169-5347(00)01994-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+methods+for+detecting+molecular+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+methods+for+detecting+molecular+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001694-Anisimova1" id="pone.0001694-Anisimova1"></a>Anisimova M, Bielawski JP, Yang Z (2001) Accuracy and power of the likelihood ratio test in detecting adaptive molecular evolution. Mol Biol Evol  18: 1585â1592.  <ul class="find" data-citedArticleID="1064476" data-doi="10.1093/oxfordjournals.molbev.a003945"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003945" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accuracy+and+power+of+the+likelihood+ratio+test+in+detecting+adaptive+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accuracy+and+power+of+the+likelihood+ratio+test+in+detecting+adaptive+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001694-Anisimova2" id="pone.0001694-Anisimova2"></a>Anisimova M, Bielawski JP, Yang Z (2002) Accuracy and power of bayes prediction of amino acid sites under positive selection. Mol Biol Evol  19: 950â958.  <ul class="find" data-citedArticleID="1064478" data-doi="10.1093/oxfordjournals.molbev.a004152"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004152" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accuracy+and+power+of+bayes+prediction+of+amino+acid+sites+under+positive+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accuracy+and+power+of+bayes+prediction+of+amino+acid+sites+under+positive+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001694-Massingham1" id="pone.0001694-Massingham1"></a>Massingham T, Goldman N (2005) Detecting amino acid sites under positive selection and purifying selection. Genetics  169: 1753â1762.  <ul class="find" data-citedArticleID="1064532" data-doi="10.1534/genetics.104.032144"><li><a href="http://dx.doi.org/10.1534/genetics.104.032144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+amino+acid+sites+under+positive+selection+and+purifying+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+amino+acid+sites+under+positive+selection+and+purifying+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001694-KosakovskyPond1" id="pone.0001694-KosakovskyPond1"></a>Kosakovsky Pond SL, Frost SD (2005) Not so different after all: a comparison of methods for detecting amino acid sites under selection. Mol Biol Evol  22: 1208â1222.  <ul class="find" data-citedArticleID="1064524" data-doi="10.1093/molbev/msi105"><li><a href="http://dx.doi.org/10.1093/molbev/msi105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Not+so+different+after+all%3A+a+comparison+of+methods+for+detecting+amino+acid+sites+under+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Not+so+different+after+all%3A+a+comparison+of+methods+for+detecting+amino+acid+sites+under+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001694-Guindon1" id="pone.0001694-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol  52: 696â704.  <ul class="find" data-citedArticleID="1064506"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001694-Whelan1" id="pone.0001694-Whelan1"></a>Whelan S, Goldman N (2001) A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach. Mol Biol Evol  18: 691â699.  <ul class="find" data-citedArticleID="1064554" data-doi="10.1093/oxfordjournals.molbev.a003851"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003851" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+empirical+model+of+protein+evolution+derived+from+multiple+protein+families+using+a+maximum-likelihood+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+empirical+model+of+protein+evolution+derived+from+multiple+protein+families+using+a+maximum-likelihood+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001694-Gascuel1" id="pone.0001694-Gascuel1"></a>Gascuel O (1997) BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data. Mol Biol Evol  14: 685â695.  <ul class="find" data-citedArticleID="1064502" data-doi="10.1093/oxfordjournals.molbev.a025808"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025808" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BIONJ%3A+an+improved+version+of+the+NJ+algorithm+based+on+a+simple+model+of+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BIONJ%3A+an+improved+version+of+the+NJ+algorithm+based+on+a+simple+model+of+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001694-Anisimova3" id="pone.0001694-Anisimova3"></a>Anisimova M, Gascuel O (2006) Approximate likelihood-ratio test for branches: A fast, accurate, and powerful alternative. Syst Biol  55: 539â552.  <ul class="find" data-citedArticleID="1064480"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Approximate+likelihood-ratio+test+for+branches%3A+A+fast%2C+accurate%2C+and+powerful+alternative.&amp;auth=&amp;atitle=Approximate+likelihood-ratio+test+for+branches%3A+A+fast%2C+accurate%2C+and+powerful+alternative." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Approximate+likelihood-ratio+test+for+branches%3A+A+fast%2C+accurate%2C+and+powerful+alternative." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Approximate+likelihood-ratio+test+for+branches%3A+A+fast%2C+accurate%2C+and+powerful+alternative.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001694-Kobe3" id="pone.0001694-Kobe3"></a>Kobe B, Deisenhofer J (1993) Crystal structure of porcine ribonuclease inhibitor, a protein with leucine-rich repeats. Nature  366: 751â756.  <ul class="find" data-citedArticleID="1064522" data-doi="10.1038/366751a0"><li><a href="http://dx.doi.org/10.1038/366751a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+porcine+ribonuclease+inhibitor%2C+a+protein+with+leucine-rich+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+porcine+ribonuclease+inhibitor%2C+a+protein+with+leucine-rich+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001694-DeLano1" id="pone.0001694-DeLano1"></a>DeLano WL (2002) The PyMol Molecular Graphics System. San Carlos, CA, USA: DeLano Scientific.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.XML" value="102733"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.PDF" value="316997"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g001.PNG_L" value="684235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g001.PNG_M" value="101981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g001.PNG_S" value="15435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g001.TIF" value="1082308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g001.PNG_I" value="134730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g002.PNG_L" value="2498658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g002.PNG_M" value="256769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g002.PNG_S" value="16519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g002.TIF" value="4413518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g002.PNG_I" value="154454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g003.PNG_L" value="3391827"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g003.PNG_M" value="484380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g003.PNG_S" value="14675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g003.TIF" value="4937920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.g003.PNG_I" value="157466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.e001.PNG" value="34832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.e001.TIF" value="53900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.e002.PNG" value="10225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.e002.TIF" value="26360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.s001.DOC" value="55808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.s002.TXT" value="44292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.s003.DOC" value="30720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001694.s004.DOC" value="23040"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001694&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001694" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001694&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001694&volume=&issue=&title=Origin and Evolution of GALA-LRR, a New Member of the CC-LRR Subfamily: From Plants to Bacteria?&author_name=Andrey%20V.%20Kajava%2C%20Maria%20Anisimova%2C%20Nemo%20Peeters&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001694" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001694" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001694" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001694&amp;t=Origin%20and%20Evolution%20of%20GALA-LRR%2C%20a%20New%20Member%20of%20the%20CC-LRR%20Subfamily%3A%20From%20Plants%20to%20Bacteria%3F" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001694&title=Origin%20and%20Evolution%20of%20GALA-LRR%2C%20a%20New%20Member%20of%20the%20CC-LRR%20Subfamily%3A%20From%20Plants%20to%20Bacteria%3F&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001694&amp;title=Origin%20and%20Evolution%20of%20GALA-LRR%2C%20a%20New%20Member%20of%20the%20CC-LRR%20Subfamily%3A%20From%20Plants%20to%20Bacteria%3F" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001694" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001694&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Origin and Evolution of GALA-LRR, a New Member of the CC-LRR Subfamily: From Plants to Bacteria?';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001694';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Origin%20and%20Evolution%20of%20GALA-LRR%2C%20a%20New%20Member%20of%20the%20CC-LRR%20Subfamily%3A%20From%20Plants%20to%20Bacteria%3F http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001694" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001694" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="26214"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Database+searching%22" title="Search for articles in the subject area:'Database searching'"><div class="flagText">Database searching</div></a>
              <div data-categoryid="32021" data-articleid="26214"
                   data-categoryname="Database searching"
                   class="flagImage" title="Flag 'Database searching' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="26214"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+bacterial+pathogens%22" title="Search for articles in the subject area:'Plant bacterial pathogens'"><div class="flagText">Plant bacterial pathogens</div></a>
              <div data-categoryid="23115" data-articleid="26214"
                   data-categoryname="Plant bacterial pathogens"
                   class="flagImage" title="Flag 'Plant bacterial pathogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+evolution%22" title="Search for articles in the subject area:'Plant evolution'"><div class="flagText">Plant evolution</div></a>
              <div data-categoryid="18531" data-articleid="26214"
                   data-categoryname="Plant evolution"
                   class="flagImage" title="Flag 'Plant evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="26214"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="26214"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26214"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6209'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5330'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2476&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=4501">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=16289">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
