

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Multiplexed SNP Typing of Ancient DNA Clarifies the Origin of Andaman mtDNA Haplogroups amongst South Asian Tribal Populations</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000081"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000081" />

    <meta name="citation_title" content="Multiplexed SNP Typing of Ancient DNA Clarifies the Origin of Andaman mtDNA Haplogroups amongst South Asian Tribal Populations"/>
    <meta itemprop="name" content="Multiplexed SNP Typing of Ancient DNA Clarifies the Origin of Andaman mtDNA Haplogroups amongst South Asian Tribal Populations"/>

      <meta name="citation_author" content="Phillip Endicott"/>
            <meta name="citation_author_institution" content="Department of Zoology, University of Oxford, Oxford, United Kingdom"/>
      <meta name="citation_author" content="Mait Metspalu"/>
            <meta name="citation_author_institution" content="Department of Evolutionary Biology, Estonian Biocenter, University of Tartu, Tartu, Estonia"/>
      <meta name="citation_author" content="Chris Stringer"/>
            <meta name="citation_author_institution" content="Natural History Museum London, London, United Kingdom"/>
      <meta name="citation_author" content="Vincent Macaulay"/>
            <meta name="citation_author_institution" content="Department of Statistics, University of Glasgow, Glasgow, United Kingdom"/>
      <meta name="citation_author" content="Alan Cooper"/>
            <meta name="citation_author_institution" content="Australian Centre for Ancient DNA, University of Adelaide, Adelaide, Australia"/>
      <meta name="citation_author" content="Juan J. Sanchez"/>
            <meta name="citation_author_institution" content="Instituto Nacional de Toxicologia, Departamento de Canarias, Santa Cruz de Tenerife, Spain"/>

    <meta name="citation_date" content="2006/12/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000081.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e81"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="1"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Harvesting the fruit of the human mtDNA tree.; citation_author=A Torroni; citation_author=A Achilli; citation_author=V Macaulay; citation_author=M Richards; citation_author=HJ Bandelt; citation_journal_title=Trends in Genetics; citation_number=1; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Problems in FBI mtDNA database.; citation_author=HJ Bandelt; citation_author=A Salas; citation_author=C Bravi; citation_journal_title=Science; citation_volume=305; citation_number=2; citation_pages=1402-1404; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Artificial recombination in forensic mtDNA population databases.; citation_author=HJ Bandelt; citation_author=A Salas; citation_author=S Lutz-Bonengel; citation_journal_title=Int J Legal Med; citation_volume=118; citation_number=3; citation_pages=267-273; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=More evidence for non-maternal inheritance of mitochondrial DNA?; citation_author=HJ Bandelt; citation_author=QP Kong; citation_author=W Parson; citation_author=A Salas; citation_journal_title=J Med Genet; citation_volume=42; citation_number=4; citation_pages=957-960; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of Native American founder mtDNAs through the analysis of complete mtDNA sequences: some caveats.; citation_author=HJ Bandelt; citation_author=C Herrnstadt; citation_author=YG Yao; citation_author=QP Kong; citation_author=T Kivisild; citation_journal_title=Ann Hum Genet; citation_volume=67; citation_number=5; citation_pages=512-524; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Etruscan artifacts.; citation_author=HJ Bandelt; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=6; citation_pages=919-920; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA heteroplasmy or artefacts–a matter of the amplification strategy?; citation_author=A Brandsättter; citation_author=W Parson; citation_journal_title=Int J Legal Med; citation_volume=117; citation_number=7; citation_pages=180-184; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=To trust or not to trust an idiosyncratic mitochondrial data set.; citation_author=YG Yao; citation_author=V Macauley; citation_author=T Kivisild; citation_author=YP Zhang; citation_author=HJ Bandelt; citation_journal_title=Am J Hum Genet; citation_volume=72; citation_number=8; citation_pages=1341-1346; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genotyping errors: causes, consequences and solutions.; citation_author=F Pompanon; citation_author=A Bonin; citation_author=E Bellemain; citation_author=P Taberlet; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=9; citation_pages=847-859; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Quality assessment of DNA sequence data: autopsy of a mis-sequenced mtDNA population sample.; citation_author=HJ Bandelt; citation_author=T Kivisild; citation_journal_title=Ann Hum Genet; citation_volume=70; citation_number=10; citation_pages=314-326; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA and human evolution.; citation_author=B Pakendorf; citation_author=M Stoneking; citation_journal_title=Annu Rev Genomics Hum Genet; citation_volume=6; citation_number=11; citation_pages=165-183; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The matrilineal ancestry of Ashkenazi Jewry: portrait of a recent founder event.; citation_author=DM Behar; citation_author=E Metspalu; citation_author=T Kivisild; citation_author=A Achilli; citation_author=Y Hadid; citation_journal_title=Am J Hum Genet; citation_volume=78; citation_number=12; citation_pages=487-497; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ancient DNA from the first European farmers in 7500-year-old Neolithic sites.; citation_author=W Haak; citation_author=P Forster; citation_author=B Bramanti; citation_author=S Matsumura; citation_author=G Brandt; citation_journal_title=Science; citation_volume=310; citation_number=13; citation_pages=1016-1018; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=South Asia, the Andamanese, and the genetic evidence for an “early” human dispersal out of Africa.; citation_author=R Cordaux; citation_author=M Stoneking; citation_journal_title=Am J Hum Genet; citation_volume=72; citation_number=14; citation_pages=1586-1590; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Assessment of the southern dispersal: GIS-based analyses of potential routes at oxygen isotopic stage.; citation_author=JS Field; citation_author=MM Lahr; citation_journal_title=Journal of World Prehistory; citation_volume=19; citation_number=15; citation_pages=1-45; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolution. Did early humans go north or south?; citation_author=P Forster; citation_author=S Matsumura; citation_journal_title=Science; citation_volume=308; citation_number=16; citation_pages=965-966; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Why did modern human populations disperse from Africa ca. 60,000 years ago? A new model.; citation_author=P Mellars; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=17; citation_pages=9381-9386; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The genetic origins of the Andaman Islanders.; citation_author=P Endicott; citation_author=MT Gilbert; citation_author=C Stringer; citation_author=C Lalueza-Fox; citation_author=E Willerslev; citation_journal_title=Am J Hum Genet; citation_volume=72; citation_number=18; citation_pages=178-184; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic affinities of the Andaman Islanders, a vanishing human population.; citation_author=K Thangaraj; citation_author=L Singh; citation_author=AG Reddy; citation_author=VR Rao; citation_author=SC Sehgal; citation_journal_title=Curr Biol; citation_volume=13; citation_number=19; citation_pages=86-93; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Autosomal STR data on the enigmatic Andaman Islanders.; citation_author=K Thangaraj; citation_author=G Chaubey; citation_author=AG Reddy; citation_author=VK Singh; citation_author=L Singh; citation_journal_title=Forensic Sci Int; citation_number=20; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Most of the extant mtDNA boundaries in south and southwest Asia were likely shaped during the initial settlement of Eurasia by anatomically modern humans.; citation_author=M Metspalu; citation_author=T Kivisild; citation_author=E Metspalu; citation_author=J Parik; citation_author=G Hudjashov; citation_journal_title=BMC Genet; citation_volume=5; citation_number=21; citation_pages=26; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The dazzling array of basal branches in the mtDNA macrohaplogroup M from India as inferred from complete genomes.; citation_author=C Sun; citation_author=QP Kong; citation_author=MG Palanichamy; citation_author=S Agrawal; citation_author=HJ Bandelt; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=22; citation_pages=683-690; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Reconstructing the origin of Andaman Islanders.; citation_author=K Thangaraj; citation_author=G Chaubey; citation_author=T Kivisild; citation_author=AG Reddy; citation_author=VK Singh; citation_journal_title=Science; citation_volume=308; citation_number=23; citation_pages=996; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comment on “Reconstructing the origin of Andaman islanders”.; citation_author=MG Palanichamy; citation_author=S Agrawal; citation_author=YG Yao; citation_author=QP Kong; citation_author=C Sun; citation_journal_title=Science; citation_volume=311; citation_number=24; citation_pages=470; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A practical guide to mitochondrial DNA error prevention in clinical, forensic, and population genetics.; citation_author=A Salas; citation_author=A Carracedo; citation_author=V Macaulay; citation_author=M Richards; citation_author=HJ Bandelt; citation_journal_title=Biochem Biophys Res Commun; citation_volume=335; citation_number=25; citation_pages=891-899; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The fingerprint of phantom mutations in mitochondrial DNA data.; citation_author=HJ Bandelt; citation_author=L Quintana-Murci; citation_author=A Salas; citation_author=V Macaulay; citation_journal_title=Am J Hum Genet; citation_volume=71; citation_number=26; citation_pages=1150-1160; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detecting errors in mtDNA data by phylogenetic analysis.; citation_author=HJ Bandelt; citation_author=P Lahermo; citation_author=M Richards; citation_author=V Macaulay; citation_journal_title=Int J Legal Med; citation_volume=115; citation_number=27; citation_pages=64-69; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Low “penetrance” of phylogenetic knowledge in mitochondrial disease studies.; citation_author=HJ Bandelt; citation_author=A Achilli; citation_author=QP Kong; citation_author=A Salas; citation_author=S Lutz-Bonengel; citation_journal_title=Biochem Biophys Res Commun; citation_volume=333; citation_number=28; citation_pages=122-130; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A reappraisal of complete mtDNA variation in East Asian families with hearing impairment.; citation_author=YG Yao; citation_author=A Salas; citation_author=CM Bravi; citation_author=HJ Bandelt; citation_journal_title=Hum Genet; citation_volume=119; citation_number=29; citation_pages=505-515; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ancient DNA: do it right or not at all.; citation_author=A Cooper; citation_author=HN Poinar; citation_journal_title=Science; citation_volume=289; citation_number=30; citation_pages=1139; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mosaics of ancient mitochondrial DNA: positive indicators of nonauthenticity.; citation_author=HJ Bandelt; citation_journal_title=Eur J Hum Genet; citation_volume=13; citation_number=31; citation_pages=1106-1112; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic analyses from ancient DNA.; citation_author=S Pääbo; citation_author=H Poinar; citation_author=D Serre; citation_author=V Jaenicke-Despres; citation_author=J Hebler; citation_journal_title=Annu Rev Genet; citation_volume=38; citation_number=32; citation_pages=645-679; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Typing of Y chromosome SNPs with multiplex PCR methods.; citation_author=JJ Sanchez; citation_author=C Borsting; citation_author=N Morling; citation_journal_title=Methods Mol Biol; citation_volume=297; citation_number=33; citation_pages=209-228; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=From gels to chips: “minisequencing” primer extension for analysis of point mutations and single nucleotide polymorphisms.; citation_author=AC Syävnen; citation_journal_title=Hum Mutat; citation_volume=13; citation_number=34; citation_pages=1-10; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Y-SNP typing of U.S. African American and Caucasian samples using allele-specific hybridization and primer extension.; citation_author=PM Vallone; citation_author=JM Butler; citation_journal_title=J Forensic Sci; citation_volume=49; citation_number=35; citation_pages=723-732; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Forensic DNA typing by the solid-phase minisequencing method.; citation_author=AC Syvänen; citation_author=A Sajantila; citation_author=M Lukka; citation_journal_title=Exs; citation_volume=67; citation_number=36; citation_pages=275-282; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Single nucleotide polymorphisms over the entire mtDNA genome that increase the power of forensic testing in Caucasians.; citation_author=MD Coble; citation_author=RS Just; citation_author=JE O'Callaghan; citation_author=IH Letmanyi; citation_author=CT Peterson; citation_journal_title=Int J Legal Med; citation_volume=118; citation_number=37; citation_pages=137-146; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Multiplex amplified product-length polymorphism analysis of 36 mitochondrial single-nucleotide polymorphisms for haplogrouping of East Asian populations.; citation_author=K Umetsu; citation_author=M Tanaka; citation_author=I Yuasa; citation_author=N Adachi; citation_author=A Miyoshi; citation_journal_title=Electrophoresis; citation_volume=26; citation_number=38; citation_pages=91-98; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dissection of mitochondrial superhaplogroup H using coding region SNPs.; citation_author=A Brandstätter; citation_author=A Salas; citation_author=H Niederstatter; citation_author=C Gassner; citation_author=A Carracedo; citation_journal_title=Electrophoresis; citation_number=39; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Typing of mitochondrial DNA coding region SNPs of forensic and anthropological interest using SNaPshot minisequencing.; citation_author=B Quintans; citation_author=V Alvarez-Iglesias; citation_author=A Salas; citation_author=C Phillips; citation_author=MV Lareu; citation_journal_title=Forensic Sci Int; citation_volume=140; citation_number=40; citation_pages=251-257; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Multiplex amplification of the mammoth mitochondrial genome and the evolution of Elephantidae.; citation_author=J Krause; citation_author=PH Dear; citation_author=JL Pollack; citation_author=M Slatkin; citation_author=H Spriggs; citation_journal_title=Nature; citation_volume=439; citation_number=41; citation_pages=724-727; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Developing high-definition multiplexed SNP assays wth special reference to degraded DNA templates.; citation_author=JJ Sanchez; citation_author=P Endicott; citation_journal_title=Nature Protocols; citation_number=42; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Single, rapid coastal settlement of Asia revealed by analysis of complete mitochondrial genomes.; citation_author=V Macaulay; citation_author=C Hill; citation_author=A Achilli; citation_author=C Rengo; citation_author=D Clarke; citation_journal_title=Science; citation_volume=308; citation_number=43; citation_pages=1034-1036; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=How many clones need to be sequenced from a single forensic or ancient DNA sample in order to determine a reliable consensus sequence?; citation_author=MA Bower; citation_author=M Spencer; citation_author=S Matsumura; citation_author=RE Nisbet; citation_author=CJ Howe; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=44; citation_pages=2549-2556; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogeny of east Asian mitochondrial DNA lineages inferred from complete sequences.; citation_author=QP Kong; citation_author=YG Yao; citation_author=C Sun; citation_author=HJ Bandelt; citation_author=CL Zhu; citation_journal_title=Am J Hum Genet; citation_volume=73; citation_number=45; citation_pages=671-676; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=MV Portman; citation_number=46; citation_date=1990 [1899]; citation_publisher=Asian Educational Services; " />
      <meta name="citation_reference" content="citation_author=EH Man; citation_number=47; citation_date=1932 [1884]; citation_publisher=Royal Anthropological Insitutue of Great Britain and Ireland; " />
      <meta name="citation_reference" content="citation_title=Natural selection shaped regional mtDNA variation in humans.; citation_author=D Mishmar; citation_author=E Ruiz-Pesini; citation_author=P Golik; citation_author=V Macaulay; citation_author=AG Clark; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=48; citation_pages=171-176; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Tracking down human contamination in ancient human teeth.; citation_author=ML Sampietro; citation_author=MT Gilbert; citation_author=O Lao; citation_author=D Caramelli; citation_author=M Lari; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=49; citation_pages=1801-1807; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=J Sambrook; citation_author=EF Fritsch; citation_author=T Maniatis; citation_number=50; citation_date=1989; citation_publisher=Cold Spring Harbor Laboratory; " />
      <meta name="citation_reference" content="citation_title=The genetic heritage of the earliest settlers persists both in Indian tribal and caste populations.; citation_author=T Kivisild; citation_author=S Rootsi; citation_author=M Metspalu; citation_author=S Mastana; citation_author=K Kaldma; citation_journal_title=Am J Hum Genet; citation_volume=72; citation_number=51; citation_pages=313-332; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=PCR bias toward the wild-type k-ras and p53 sequences: implications for PCR detection of mutations and cancer diagnosis.; citation_author=R Barnard; citation_author=V Futo; citation_author=N Pecheniuk; citation_author=M Slattery; citation_author=T Walsh; citation_journal_title=Biotechniques; citation_volume=25; citation_number=52; citation_pages=684-691; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Multiplex genotyping of PCR products with MassTag-labeled primers.; citation_author=LA Haff; citation_author=IP Smirnov; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=53; citation_pages=3749-3750; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Universal, robust, highly quantitative SNP allele frequency measurement in DNA pools.; citation_author=N Norton; citation_author=NM Williams; citation_author=HJ Williams; citation_author=G Spurlock; citation_author=G Kirov; citation_journal_title=Hum Genet; citation_volume=110; citation_number=54; citation_pages=471-478; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Assessing ancient DNA studies.; citation_author=MT Gilbert; citation_author=HJ Bandelt; citation_author=M Hofreiter; citation_author=I Barnes; citation_journal_title=Trends Ecol Evol; citation_volume=20; citation_number=55; citation_pages=541-544; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000081" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Multiplexed SNP Typing of Ancient DNA Clarifies the Origin of Andaman mtDNA Haplogroups amongst South Asian Tribal Populations"/>
    <meta name="twitter:description" content="The issue of errors in genetic data sets is of growing concern, particularly in population genetics where whole genome mtDNA sequence data is coming under increased scrutiny. Multiplexed PCR reactions, combined with SNP typing, are currently under-exploited in this context, but have the potential to genotype whole populations rapidly and accurately, significantly reducing the amount of errors appearing in published data sets. To show the sensitivity of this technique for screening mtDNA genomic sequence data, 20 historic samples of the enigmatic Andaman Islanders and 12 modern samples from three Indian tribal populations (Chenchu, Lambadi and Lodha) were genotyped for 20 coding region sites after provisional haplogroup assignment with control region sequences. The genotype data from the historic samples significantly revise the topologies for the Andaman M31 and M32 mtDNA lineages by rectifying conflicts in published data sets. The new Indian data extend the distribution of the M31a lineage to South Asia, challenging previous interpretations of mtDNA phylogeography. This genetic connection between the ancestors of the Andamanese and South Asian tribal groups &sim;30 kya has important implications for the debate concerning migration routes and settlement patterns of humans leaving Africa during the late Pleistocene, and indicates the need for more detailed genotyping strategies. The methodology serves as a low-cost, high-throughput model for the production and authentication of data from modern or ancient DNA, and demonstrates the value of museum collections as important records of human genetic diversity."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000081.g006"/>

  <meta property="og:title" content="Multiplexed SNP Typing of Ancient DNA Clarifies the Origin of Andaman mtDNA Haplogroups amongst South Asian Tribal Populations" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000081" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9862'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6446'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2370&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000081">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000081" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000081&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Multiplexed SNP Typing of Ancient DNA Clarifies the Origin of Andaman mtDNA Haplogroups amongst South Asian Tribal Populations
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Phillip Endicott
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:phillip.endicott@zoo.ox.ac.uk">phillip.endicott@zoo.ox.ac.uk</a>
            </p>

                <p>Affiliation:
                  Department of Zoology, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mait Metspalu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Evolutionary Biology, Estonian Biocenter, University of Tartu, Tartu, Estonia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chris Stringer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Natural History Museum London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vincent Macaulay, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistics, University of Glasgow, Glasgow, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alan Cooper, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Australian Centre for Ancient DNA, University of Adelaide, Adelaide, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Juan J. Sanchez
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Instituto Nacional de Toxicologia, Departamento de Canarias, Santa Cruz de Tenerife, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 20, 2006</li>
    <li>DOI: 10.1371/journal.pone.0000081</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000081" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000081" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000081" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000081" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000081-g001" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000081-g002" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000081-g003" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000081-g004" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000081-g005" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000081-g006" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g006" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000081">Reader Comments (5)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000081" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The issue of errors in genetic data sets is of growing concern, particularly in population genetics where whole genome mtDNA sequence data is coming under increased scrutiny. Multiplexed PCR reactions, combined with SNP typing, are currently under-exploited in this context, but have the potential to genotype whole populations rapidly and accurately, significantly reducing the amount of errors appearing in published data sets. To show the sensitivity of this technique for screening mtDNA genomic sequence data, 20 historic samples of the enigmatic Andaman Islanders and 12 modern samples from three Indian tribal populations (Chenchu, Lambadi and Lodha) were genotyped for 20 coding region sites after provisional haplogroup assignment with control region sequences. The genotype data from the historic samples significantly revise the topologies for the Andaman M31 and M32 mtDNA lineages by rectifying conflicts in published data sets. The new Indian data extend the distribution of the M31a lineage to South Asia, challenging previous interpretations of mtDNA phylogeography. This genetic connection between the ancestors of the Andamanese and South Asian tribal groups ~30 kya has important implications for the debate concerning migration routes and settlement patterns of humans leaving Africa during the late Pleistocene, and indicates the need for more detailed genotyping strategies. The methodology serves as a low-cost, high-throughput model for the production and authentication of data from modern or ancient DNA, and demonstrates the value of museum collections as important records of human genetic diversity.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Endicott P, Metspalu M, Stringer C, Macaulay V, Cooper A, et al.  (2006) Multiplexed SNP Typing of Ancient DNA Clarifies the Origin of Andaman mtDNA Haplogroups amongst South Asian Tribal Populations. PLoS ONE 1(1):
          e81.
            doi:10.1371/journal.pone.0000081</p><p><strong>Academic Editor: </strong>Henry Harpending, University of Utah, United States of America</p><p><strong>Received:</strong> September 29, 2006; <strong>Accepted:</strong> October 30, 2006; <strong>Published:</strong> December 20, 2006</p><p><strong>Copyright:</strong> © 2006 Endicott et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no support or funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The rapidly increasing number of characterised human single nucleotide polymorphisms (SNPs) available on public databases represents a powerful resource with which to improve our understanding of population history, disease susceptibility and forensic studies<a href="#pone.0000081-Torroni1">[1]</a>. Yet the conventional approach used to produce this data, PCR amplification followed by sequencing or restriction fragment length polymorphism (RFLP) mapping, is known to generate error-strewn data sets<a href="#pone.0000081-Bandelt1">[2]</a>–<a href="#pone.0000081-Bandelt3">[4]</a>. The potential for contamination or mix-ups between samples increases rapidly with the number of steps or genotyping reactions required, and the approach also requires a large amount of resources (time, reagents, template DNA).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>With whole genome mtDNA data, the problem of multiple assays is exacerbated by the sheer number of nucleotides sequenced and interpreted. It is not uncommon for such data sets to contain mistakes arising through a combination of sub-optimal sequencing chemistry, automated analysis and human error<a href="#pone.0000081-Bandelt4">[5]</a>–<a href="#pone.0000081-Bandelt6">[10]</a>. Although some authors argue against whole genome sequencing as a routine approach<a href="#pone.0000081-Pakendorf1">[11]</a>, there is general agreement that mtDNA population genetics requires coding region information to both increase resolution and identify homoplasy common in control region data. With increasingly detailed phylogenetic interpretations being made from both modern<a href="#pone.0000081-Behar1">[12]</a> and ancient<a href="#pone.0000081-Haak1">[13]</a> DNA, the accuracy of both the novel and comparative genetic data is of paramount importance.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>These issues are exemplified in genetic analyses of the indigenous peoples of the Andaman Islands, in the Bay of Bengal (Indian Ocean), who remain central to discussions concerning the early population history of Asia, including the concept of a ‘southern route’ for the last successful migration of modern humans out of Africa to Australia<a href="#pone.0000081-Cordaux1">[14]</a>–<a href="#pone.0000081-Mellars1">[17]</a>. Due to their distinctive phenotype and languages the inhabitants of these islands are often considered to be physically and genetically isolated from the rest of South-East Asia. The continental shelf joining the Andaman archipelago with Myanmar is relatively shallow and there may have been the opportunity to cross by a land-bridge, or over short stretches of water during periods of depressed sea levels during the late Pleistocene.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The two main mtDNA lineages (M31 and M32) found in the Andamanese both contain control region motifs that are very common throughout haplogroup M<a href="#pone.0000081-Endicott1">[18]</a>–<a href="#pone.0000081-Thangaraj2">[20]</a>, which is considered to be the broader genetic marker for an early, southern migration route of humans from the Indian sub-continent eastwards<a href="#pone.0000081-Metspalu1">[21]</a>, <a href="#pone.0000081-Sun1">[22]</a>. As a consequence, the genetic evidence about the origins of the Andamanese<a href="#pone.0000081-Thangaraj2">[20]</a>, <a href="#pone.0000081-Thangaraj3">[23]</a> remains ambiguous, and increased resolution is important. Recently, a sister clade of the M31 lineage has been detected amongst a tribal population in East India<a href="#pone.0000081-Palanichamy1">[24]</a>, which might indicate that M31 reached the Andamans after the first migrations of humans carrying haplogroup M through the region. However, these interpretations are dependent upon whole-genome sequences that cannot be assumed to be flawless given the number of errors identified in other mtDNA sequencing studies.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Although a proven methodology exists for retrospectively identifying errors in published mtDNA data through careful phylogenetic analysis<a href="#pone.0000081-Salas1">[25]</a>–<a href="#pone.0000081-Bandelt8">[27]</a> this form of internal control is often neglected, particularly when many of the SNPs reported have no bearing on the research being conducted<a href="#pone.0000081-Bandelt9">[28]</a>, <a href="#pone.0000081-Yao2">[29]</a>. It is also heavily reliant upon comparative data that may not be available; for example, when sequencing novel mtDNA lineages such as those present in the Andaman Islands<a href="#pone.0000081-Thangaraj3">[23]</a>, or phylogenetically deep-rooting genomes. Therefore, in all situations, the primary concern of the laboratory should lie with the accuracy of data production. However, even when all reasonable safeguards are taken, the issue of how to discern genuine novel variation from sequence or human errors remains, especially when working with the degraded templates typical of ancient DNA (aDNA)<a href="#pone.0000081-Cooper1">[30]</a>–<a href="#pone.0000081-Pbo1">[32]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Multiplexed PCR reactions provide a potential solution to these problems, because multiple primer pairs amplify different parts of the genome together in a single reaction tube<a href="#pone.0000081-Sanchez1">[33]</a>. Sequence variants within the multiplex PCR products can be characterised by numerous approaches<a href="#pone.0000081-Syvnen1">[34]</a>, but the technically simplest and most reliable method of detecting specific nucleotide variation(s) appears to be single base extension (SBE) where one of four fluorescently labelled ddNTPs present is incorporated 5′ of the SBE primer and screened with a capillary sequencer<a href="#pone.0000081-Vallone1">[35]</a>. The SBE technique is less sensitive to the frailties of conventional sequencing, because only a single ddNTP needs to be incorporated, and the assay can be designed to work in either direction to overcome strand-specific amplification and sequencing problems.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Despite a relatively long history<a href="#pone.0000081-Syvnen2">[36]</a>, there have been few SNP typing studies using a multiplex methodology and the method has not yet been adopted for the authentication of data. Typically, the number of SNPs tested is small so assays often require multiple reactions to achieve acceptable levels of discrimination between samples<a href="#pone.0000081-Vallone1">[35]</a>, <a href="#pone.0000081-Coble1">[37]</a>, <a href="#pone.0000081-Umetsu1">[38]</a>. Although some assays target segments over 800 bp<a href="#pone.0000081-Brandsttter2">[39]</a>, it is desirable to limit designs to the smallest possible products in order to minimize the error rate due to allelic drop out. This approach has been used in forensics studies but assays still usually target segments too large for degraded templates typical of aDNA<a href="#pone.0000081-Quintans1">[40]</a>. With very well preserved ancient specimens it is possible to use a two-step multiplex PCR to generate large segments of mtDNA sequence data<a href="#pone.0000081-Krause1">[41]</a> but this involves a second round of exponential amplification for each individual product. In contrast, SBE reactions can use PCR products as small as 50–60 bp for templates, and can be multiplexed in a linear amplification, providing greater sensitivity.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>The enormous demographic loss during the British colonial era makes it impossible to adequately reconstruct the evolutionary history of the remaining Andaman populations, especially those of the Greater Andamans, who are critically important because they formed the absolute majority at the time of colonisation (1858), and spoke a different set of dialects<a href="#pone.0000081-Endicott1">[18]</a>. Therefore, this study utilizes the resources of human skeletal material from the colonial era preserved in various museum collections in an attempt to reconstruct the population diversity present in the latter part of the 19<sup>th</sup> century. Due to the predictably degraded nature of the DNA, the key to this strategy is a PCR design which targets products under 130 bp in length, without sacrificing any of the highly specific target SNPs, through careful primer design<a href="#pone.0000081-Sanchez2">[42]</a>.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>Twenty potentially informative coding region SNPs were identified using whole mtDNA genome data from two previously defined mtDNA lineages, M31 and M32<a href="#pone.0000081-Thangaraj2">[20]</a>, <a href="#pone.0000081-Thangaraj3">[23]</a>, <a href="#pone.0000081-Palanichamy1">[24]</a>. These SNPs were used to genotype 20 historical samples of Andaman Islanders from the Bay of Bengal (<a href="#pone-0000081-g001">Fig. 1</a>), assigned by their control region sequences to M31 and M32<a href="#pone.0000081-Thangaraj3">[23]</a>. The samples cover three different linguistic groups; Greater Andamanese (n = 16), Jarawa (n = 2), and Onge (n = 2). Because two of the M31 SNPs are present amongst the Rajbansi of West Bengal (forming a sister clade M31b<a href="#pone.0000081-Palanichamy1">[24]</a>) the multiplex method was also used to screen modern DNA extracts from three Indian tribal populations (Chenchu, Lambadi, and Lodha), who possess similar control region motifs. Late Pleistocene inter-regional mtDNA links are the exception in South and South-East Asia<a href="#pone.0000081-Metspalu1">[21]</a>, <a href="#pone.0000081-Macaulay1">[43]</a>, so any genetic connections would be of considerable importance to reconstructing human migration and settlement patterns during this formative period.</p>
<div class="figure" id="pone-0000081-g001"><div class="img"><a name="pone-0000081-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Map of the principal islands in the Andaman archipelago in the Bay of Bengal showing the major tribal divisions present in 1858.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>The locations of the three contemporary Indian tribal populations surveyed are given on the regional map.</p>
<span>doi:10.1371/journal.pone.0000081.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>The final cross-validated results of the SBE amplifications are presented in <a href="#pone-0000081-g006">Table 1</a>, and show that all target SNPs were determined for each sample, demonstrating the robustness of the multiplex methodology. <a href="#pone-0000081-g002">Fig. 2</a> shows the actual profile of two samples, one each from the M31a and M32 mtDNA lineages. The M31 and M32 genotypes are distinctive, aiding the identification of contaminating DNA<a href="#pone.0000081-Pbo1">[32]</a>, but the behaviour of multiplexed SBE reactions in the presence of multiple populations of templates is not well studied. So, to further investigate the specificity and sensitivity of the multiplex-SBE genotyping methodology one of the Andaman M32 samples was mixed with decreasing amounts of a modern European extract, which had been diluted to the same concentration, and the resulting aliquots were used to initiate the M31_M32 multiplexed SBE assay. The presence of both alleles at each of the M32 specific sites (<a href="#pone-0000081-g003">Fig. 3</a>) indicates the presence of the second set of (contaminating) templates, which are detectable at all concentrations. The lowest of these is just 4%, a level that is beyond the threshold of detection by routine cloning approaches<a href="#pone.0000081-Bower1">[44]</a>.</p>
<div class="figure" id="pone-0000081-g002"><div class="img"><a name="pone-0000081-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Genotypes of two Andaman Islanders using a panel of 20 informative SNPs.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Each line represents the four systems of G, A, C, and T, in descending order.</p>
<a id="article1.body1.sec2.fig1.caption1.p2" name="article1.body1.sec2.fig1.caption1.p2"></a><p>SNPs are named by their position relative to the Cambridge Reference Sequence and are separated by variation in electrophoretic mobility, which is determined by the length of the SBE primer.</p>
<a id="article1.body1.sec2.fig1.caption1.p3" name="article1.body1.sec2.fig1.caption1.p3"></a><p>The segregation of the two samples at all lineage defining sites, for M31a1 and M32 respectively, demonstrates the accuracy and reliability of genotyping using this methodology.</p>
<span>doi:10.1371/journal.pone.0000081.g002</span></div><div class="figure" id="pone-0000081-g003"><div class="img"><a name="pone-0000081-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>The results from a Snapshot assay of an M32 sample that has been deliberately contaminated with a modern European DNA extract.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>Alleles at np15754 and 3817 on the G line are from the contaminant DNA, whilst those on the A system represent the M32 sample.</p>
<a id="article1.body1.sec2.fig2.caption1.p2" name="article1.body1.sec2.fig2.caption1.p2"></a><p>The level of the contaminant increases, as a proportion of the total mix, with each of the descending lines shown within the oval.</p>
<a id="article1.body1.sec2.fig2.caption1.p3" name="article1.body1.sec2.fig2.caption1.p3"></a><p>The relative amounts of contaminant are 4%, 8% and 20% of the effective copy number (assessed by quantitative PCR) prior to mixing of the two extracts to seed the multiplex.</p>
<span>doi:10.1371/journal.pone.0000081.g003</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>The absence of additional alleles at any of the segregating sites for all twenty samples in <a href="#pone-0000081-g006">Table 1</a> indicates that contamination, either from endogenous sources or between samples, did not affect the results. Yet, the multiplex reveals that the A08108G mutation occurs in both M31 and M32. Although this SNP is also present in the East Asian mtDNA haplogroup M11<a href="#pone.0000081-Kong1">[45]</a>, it would normally be considered unusual to find it on two independent lineages in the same geographic location. If a standard singleplex PCR approach had been used, there would be a strong suspicion of artificial recombination via sample mix up. However, the simultaneous amplification of all the SNPs excludes this possibility because only single alleles have been observed and all other sites tested are consistent.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>The revised phylogenetic structure of M31 and M32 is depicted in <a href="#pone-0000081-g004">Fig. 4</a>, which includes data from the control region sequences provided in the supplementary information (<a href="#pone.0000081.s004">Table S4</a>). Both lineages now have substantial internal structure and revised groups of defining mutations. Of particular interest is the establishment of the major clade M31a, which is present in both South and South-East Asia, linking the two regions more recently than previously thought. The main conflicts in the coding-region data between this tree and the previous data sets, together with the resulting reticulations are depicted in the network presented in <a href="#pone-0000081-g005">Fig. 5</a>. The thicker line represents the most parsimonious phylogeny for the M31a1 and M32 mtDNA lineages.</p>
<div class="figure" id="pone-0000081-g004"><div class="img"><a name="pone-0000081-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Maximum parsimony tree of mtDNA lineages M31 and M32 obtained by multiplexed genotyping, demonstrating the fine structure apportioned to these lineages [Ra = Rajbanshi; Lo = Lodha; La = Lambadi; Ch = Chenchu; GA = Greater Andamanese; On = Onge; J = Jarawa].</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>The definitions of the Andaman-specific M31a1 and M32 lineages are revised to include additional control region SNPs resulting from the cloned sequences of the historic samples.</p>
<a id="article1.body1.sec2.fig3.caption1.p2" name="article1.body1.sec2.fig3.caption1.p2"></a><p>The Indian variant found amongst the Lodha, Chenchu and Lambadi develops the inter-regional structure of M31 and allows for the settlement of the Andaman archipelago substantially after the last successful migrations Out of Africa by modern humans.</p>
<a id="article1.body1.sec2.fig3.caption1.p3" name="article1.body1.sec2.fig3.caption1.p3"></a><p>These sequences contribute a deletion at np249 on the trunk of M31a1 and the motif of 143-195-207 on the stem of M32, contra to the tree of Thangaraj et al.<a href="#pone.0000081-Thangaraj3">[23]</a>, which did not report the deletion and the three M32 SNPs never occurred together in their five samples.</p>
<a id="article1.body1.sec2.fig3.caption1.p4" name="article1.body1.sec2.fig3.caption1.p4"></a><p>The original data also reported a transition at np200 on all four of their M31 samples, but in the historic data it is only found in a single Onge sample.</p>
<span>doi:10.1371/journal.pone.0000081.g004</span></div><div class="figure" id="pone-0000081-g005"><div class="img"><a name="pone-0000081-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Full median network depicting the conflicts between the data sets of this study and that of Thangaraj et al.<a href="#pone.0000081-Thangaraj3">[23]</a> in the coding region Nodes A-K represent distinct haplotypes. </span></strong></p><a id="article1.body1.sec2.fig4.caption1.p1" name="article1.body1.sec2.fig4.caption1.p1"></a><p>A-G correspond to our hapolotypes assigned to M31a1b1, M31a1b*, M31a1*, M31a1a, M32*, M31a* and M32a1, respectively, whilst H-K are from Thangaraj et al<a href="#pone.0000081-Thangaraj3">[23]</a>.</p>
<a id="article1.body1.sec2.fig4.caption1.p2" name="article1.body1.sec2.fig4.caption1.p2"></a><p>The node marked “Root” is the root of haplogroup M and ticks on the network indicate mutations, the most important of which are shown by their position; the long internal branches are shown compressed.</p>
<a id="article1.body1.sec2.fig4.caption1.p3" name="article1.body1.sec2.fig4.caption1.p3"></a><p>The diagram summarizes all most parsimonious reconstructions of mutations on a tree.</p>
<a id="article1.body1.sec2.fig4.caption1.p4" name="article1.body1.sec2.fig4.caption1.p4"></a><p>Note how our data can be explained (bold) by a single homoplasy at position 8108.</p>
<a id="article1.body1.sec2.fig4.caption1.p5" name="article1.body1.sec2.fig4.caption1.p5"></a><p>However to attach H, I and J to the bold tree would require a further four recurrent mutations to be introduced.</p>
<span>doi:10.1371/journal.pone.0000081.g005</span></div><div class="figure" id="pone-0000081-g006"><div class="img"><a name="pone-0000081-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000081" data-uri="info:doi/10.1371/journal.pone.0000081.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000081.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000081.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000081.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000081.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Results from the M31-M32 multiplex.</span></strong></p><span>doi:10.1371/journal.pone.0000081.g006</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>The genotyping results demonstrate that the errors occurring in published mtDNA data sets can be much more than mere ‘background noise’. The correction of errors and the discovery of novel mutations have significantly altered the M31 and M32 phylogenies, providing important additional fine structure. Of special significance are the two sub-clades of M31a1 defined by A90617G (M31a1a) and A13710G (M31a1b), which are separated both geographically and culturally, being broadly contiguous with speakers of the Greater Andamanese (GA) and Onge-Jarawa (On/Ja) dialects, respectively. A similar pattern is apparent in M32, where M32a1 is only found amongst the Onge-Jarawa populations. This intriguing result, suggests a high degree of genetic as well as cultural isolation within the Andaman Islands.</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>Until the modern data sets are corrected, the sample sizes for Onge and Jarawa limit what can be said about genetic input to these populations. However, it is difficult to reconcile the lack of overlapping haplotypes amongst the Greater Andamanese simply as a product of genetic drift, unless there was an extremely limited movement of women from the neighbouring Jarawa for a substantial period of time. This finding is somewhat surprising given that these two groups shared a common border. The clear structuring mirrors observations from linguistics, which place the Onge and Jarawa languages together<a href="#pone.0000081-Portman1">[46]</a>, and material culture where different bow types reflect the same divisions<a href="#pone.0000081-Man1">[47]</a>. The length of this isolation is difficult to determine exactly but assuming a mutation rate of one new coding region SNP per 5300 years<a href="#pone.0000081-Mishmar1">[48]</a> this would place it somewhere during the last 10,000 years, and could, therefore, be associated with the rising sea levels of this period.</p>
<a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>These important insights into prehistoric demography were made possible by the recovery of information from the Greater Andaman populations and demonstrate the potential of museum collections for reconstructing the genetic diversity of pre-colonial and early colonial populations. The sensitivity of the SBE reaction and its ability to detect multiple populations of templates indicate that these collections do not suffer from routine contamination by exogenous DNA. This demonstrates that repeated handling and washing is not necessarily the source of contamination in DNA extractions from human teeth<a href="#pone.0000081-Sampietro1">[49]</a>, despite the Andaman Islanders being one of the most studied sets of human remains held in European museums. It also shows that clean extractions from human skeletal material are possible provided adequate care and suitable precautions are taken in the laboratory.</p>
<a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>The rapid screening potential of the multiplex for specific mtDNA lineages is exemplified by the characterisation of M31a2 amongst the Chenchu, Lambadi and Lodha, and highlights the limitations of the control region for phylogenetic analysis. This is because M31a1 and M31a2 have completely different motifs, and the latter appears (erroneously) to be linked to M31b. If the analysis had been limited to control region sequences the connection would never have been made, and it is difficult to see how it could have been detected without the availability of whole-genome data<a href="#pone.0000081-Pakendorf1">[11]</a>.</p>
<a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p>This inter-regional connection is significant because it demonstrates a shared maternal gene-pool approximately half the age of haplogroup M31 itself. Assuming an age for haplogroup M of around 60 k years<a href="#pone.0000081-Mishmar1">[48]</a> the six coding region mutations preceding the splitting of M31a would centre this date around 30 kya. This is close to LGM low sea levels (20–25 kya), which may be significant in terms of access to the archipelago from the mainland. Without comprehensive data from Myanmar it is not possible to identify whether the Andaman M31a1 arrived from India or if the Indian M31a2 came from South-East Asia. But either scenario casts serious doubts on the concept that the Andaman Islands were settled at the time of the migrations out of Africa carrying the current Eurasian mtDNA diversity.</p>
<a id="article1.body1.sec2.p9" name="article1.body1.sec2.p9"></a><p>The additional insights into this seminal period of human pre-history afforded by this study suggest that more comprehensive sampling of contemporary populations is required and that tribal groups may be an important reservoir of genetic diversity vital to reconstructing human evolutionary history. In the context of the current study, Myanmar may prove vital to retracing population movements of the Late Pleistocene, including the settlement of the Andaman archipelago. It is also clear that there is an important role for museum collections in this process to recover crucial signals from populations that did not survive their encounter with modernity, or who have suffered considerable demographic reduction.</p>
<a id="article1.body1.sec2.p10" name="article1.body1.sec2.p10"></a><p>Whether using modern or ancient samples, the level of resolution required for this work necessitates the availability of whole-genome sequence data and a method for the rapid genotyping of individuals, but this is itself dependent on the sequence data being correct.</p>
<a id="article1.body1.sec2.p11" name="article1.body1.sec2.p11"></a><p>The levels of accuracy, built-in controls for internal consistency (particularly useful for aDNA), and a reduced need for reagents and templates make multiplexing with SBE an ideal tool for detailed population phylogenetics with both ancient and modern human DNA. The incorporation of this methodology into the protocols of laboratories would also lead to a significant reduction of errors prior to data publication. In a more general study, a lower resolution multiplex design could quickly assign individuals to the major mtDNA haplogroups, establishing the correct phylogenetic placement and identifying novel or related lineages for further investigation via whole genome sequencing.</p>
<a id="article1.body1.sec2.p12" name="article1.body1.sec2.p12"></a><p>Once a framework is established it is simple to design a multiplex to check sequence variations for the early identification of errors. This methodology would allow additional population samples to be rapidly checked, further reducing the chance of mistakes entering the public databases, and has the potential to uncover fine structure within the data. The initial cost of developing multiplex assays is quickly repaid in savings of both time and reagents due to the parallel testing of SNPs. The simple protocols described here mean that this powerful experimental procedure is within the reach of all laboratories currently producing direct sequence data.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>DNA was extracted from powder removed from the dental cavity of teeth held in the collections of the Natural History Museum London, the Oxford University Museum of Natural History, and the Royal College of Surgeons Edinburgh. Tooth external surfaces were cleaned by wiping with sodium hypochlorite solution (6% w/v) before being encased in dental alginate; this facilitates specimen handling while also preventing exposure of the outer contaminated surfaces to either the laboratory or personnel. A dental motor and 90 degree hand-piece (Bien Aire, Switzerland) were used to slice through the alginate-encased roots, and remove the surface layers of the exposed dental cavity. Thirty milligrams of powder was digested on a rotor for 20 hours at 55°C (50 mM EDTA; 10 mM ea Tris-HCL ph 8.0, NaCl, N-phenacylthiazone bromide; 20 mM proteinase K, 80 mM dithiothreitol, and 1% w/v sodium dodecylsulphate), followed by a standard Phenol-Chloroform extraction<a href="#pone.0000081-Sambrook1">[50]</a> (2 Phenol washes, 1 Chloroform), and washed by H<sub>2</sub>0 (twice) on a Microconcentrator (30,000KD, Millipore USA), to generate a final volume of 125 µl (for details of the modern samples see Kivisild et al.<a href="#pone.0000081-Kivisild1">[51]</a> and Metspalu et al.<a href="#pone.0000081-Metspalu1">[21]</a>).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The Henry Wellcome Ancient Biomolecules Centre (ABC) at Oxford was used to extract DNA, and set up PCRs. The ABC is physically isolated, subject to stringent anti-contamination procedures, equipped with positive air pressure and UV lighting, and has a DNA laboratory and equipment (glove box, instruments, full body suits, protective masks, etc.) dedicated solely to ancient human specimens. The thermal cycling reactions and subsequent downstream work took place in a separate laboratory located in the Department of Zoology. Quantitative PCR (qPCR) was used to measure amplifiable copy number and identify inhibition in a subset of the extracts, using SYBR Green® master-mix (AB), human-specific primers synthesized standards (Biomers, Germany) and an ABI 7000 real-time PCR machine. The results were used to calculate dilution and loading levels of template DNA (<a href="#pone.0000081.s004">Table S4</a>). For the samples analyzed, an effective copy number of 10 k–20 k molecules per µl produced optimal results, although no reduction in fidelity was observed with copy number as low as 2 k per µl.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>PCR primers were designed to give product lengths ranging from 67 to 127 nucleotides (<a href="#pone.0000081.s001">Table S1</a>). The aim was to obtain a theoretical melting temperature of 60°C±2°C and a purine:pyrimidine content close to 1:1 using Primer3 (<a href="http://www-genome.wi.mit.edu/cgi-bin/primer/primer3_www.cgi">http://www-genome.wi.mit.edu/cgi-bin/pri​mer/primer3_www.cgi</a>). Primer candidates were checked for primer-dimer formation, hairpin structures, homology and complementarity to other primers in the multiplex according to the guidelines outlined in<a href="#pone.0000081-Sanchez1">[33]</a>. All oligonucleotides were synthesized and reverse-phase HPLC purified by Biomers.net GmbH (Ulm, Germany). The initial multiplex PCR was performed in a 25 µl reaction volume containing 1× PCR buffer, 6.5 mM MgCl<sub>2</sub>, 600 µM of each dNTP, 0.01 µM of each primer, and 2 U of AmpliTaq Gold® DNA polymerase (AB) using the following cycling programme: denaturation at 94°C for 5 minutes followed by 35 cycles at 95°C for 30 sec., 60°C for 30 sec., and 65°C for 30 sec., followed by 6 minutes at 65°C. 2.5 µl of PCR products were purified by adding 0.75 units Shrimp Alkaline Phosphatase (SAP, Amersham Biosciences) and 0.225 units Exonuclease I (ExoI, USB Corporation). The mix was incubated at 37°C for 30 minutes, and inactivated at 75°C for 15 minutes.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The SBE primers were designed to range between 16 and 60 nucleotides, using size intervals of 4 nucleotides for primers longer than 32 nt nucleotides and 5–7 nucleotides for shorter primers to allow for differing electrophoretic mobility (<a href="#pone.0000081.s002">Table S2</a>). Loci with opposite allele combinations (e.g. A/G with C/T) were analysed with SBE oligos in the same size interval whenever possible, to maximize the number of SNPs typed in the minimum read window. SBE reactions were performed in a total volume of 5 µl with 0.5 µl purified PCR product, 2.5 µl SNaPshot™ reaction mix (AB) and 0.5 µl SBE primer mix (0.05 µM each primer and 300 mM Ammonium Sulphate), using the following program: 35 cycles of 96°C for 10 sec., 50°C for 5 sec., and 60°C for 30 sec. After the SBE reaction, 1U of SAP (Amersham Biosciences) was used as above. One µl of SAP purified SBE product was then mixed with 20 µl Hi-Di formamide (AB) and analyzed by capillary electrophoresis using an ABI 3100 genetic analyzer with 50-cm capillary arrays, POP-6 polymer (AB), injecting for 22 seconds at 3.0 kV and a run-time of 25 min. Negative controls were included in both stages of the assay to assess the potential for cross-contamination between samples and amplification of non-Andamanese DNA.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Automated allele calls were made using macros constructed in Genotyper 3.7 (AB). All peaks in the size standard were detected and the height of the largest peak in the electropherogram had to be a minimum of 800 Relative Fluorescent Units before analysis could proceed. Adjustments were made to the pre-defined windows allowing for variation in electrophoretic mobility, unequal emission energies and differential incorporation of the fluorescent ddNTPs<a href="#pone.0000081-Barnard1">[52]</a>, <a href="#pone.0000081-Haff1">[53]</a>. Primer concentrations were adjusted to give balanced amounts of products and peak heights, concentrating on the lineal SBE reaction<a href="#pone.0000081-Norton1">[54]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Once all peaks were reliably and reproducibly called, SNP calls that did not conform to those expected from the main phylogenetic tree of Thangaraj et al.<a href="#pone.0000081-Thangaraj2">[20]</a>, <a href="#pone.0000081-Thangaraj3">[23]</a> were re-assessed using singleplex SBE reactions to exclude the possibility of interference from other sites, and all found to be accurate. An additional cross-check was performed by re-amplifying the original multiplex products (1 µl of PCR product diluted 1:100 in a 25 µl volume reaction: 1× buffer, 2 mM MgCl<sub>2</sub>, 250 µM dNTPs, 0.25 U Taq, 250 nM each primer; at 94°C for 2 minutes followed by 30 cycles at 94°C for 45 secs., 58°C for 45 sec., and 72°C for 90 sec., followed by 10 minutes at 72°C), before gel purification and cloning with TOPO TA Cloning® kit (Invitrogen, UK) as per the supplier's instructions. Eight white colonies were amplified using T7 and M13R primers, and sequenced using ABI BigDye® Terminator v3.1 and an ABI Prism® capillary DNA 3700 automated sequencer. The same protocols were used for the production of control region sequences using additional pairs of primers designed to the same specification as those in the multiplex (<a href="#pone.0000081.s003">Table S3</a>). All PCR products and negative controls were checked by electrophoresis on 3% agarose gels, prior to purification with a Montage® PCRµ96 system (Millipore, U.S.A).</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Independent replication of the results was not undertaken for this study because it was considered that sufficient grounds are provided for accepting the results as they stand<a href="#pone.0000081-Gilbert1">[55]</a>. The absence of any allelic dropout, or detectable contamination, in the twenty-plex SBE reaction provides a novel method of judging the accuracy of the results, and demonstrates the extracts are free of non M31_M32 DNA. The multiplex assays were repeated, without variation, up to four times for each sample, and real-time quantitative PCR reported copy number consistent with the ability to repeat the multiplex results. The control region haplotypes (also multiplexed and re-amplified from the same DNA aliquot) are 100% consistent with the two Andaman specific haplgroups assigned by the multiplex SBE results. These sequences are invariant at all the main hg defining sites, and additional control region diversity (<a href="#pone.0000081.s004">Table S4</a>) is limited to two nucleotide positions and one of these (np 16129) is found in three different samples. All deviations from the published data<a href="#pone.0000081-Thangaraj3">[23]</a> were rigorously cloned and showed no ambiguity at the sites tested.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000081"></div><a name="pone.0000081.s001" id="pone.0000081.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000081.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Details of M31_M32 multiplex PCR primers and target SNPs</p>
<p class="siDoi">doi:10.1371/journal.pone.0000081.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.41 MB EPS)</p>
<a name="pone.0000081.s002" id="pone.0000081.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000081.s002">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Details of M31_M32 SBE primers</p>
<p class="siDoi">doi:10.1371/journal.pone.0000081.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(2.25 MB EPS)</p>
<a name="pone.0000081.s003" id="pone.0000081.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000081.s003">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Details of PCR primers for control region sequencing</p>
<p class="siDoi">doi:10.1371/journal.pone.0000081.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.33 MB EPS)</p>
<a name="pone.0000081.s004" id="pone.0000081.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000081.s004">Table S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Details of control regions polymorphisms detected in each sample.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000081.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(1.72 MB EPS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank the Natural History Museum, London, the Royal College of Surgeons, Edinburgh, and the Oxford University Museum of Natural History for permission to examine specimens from their collections.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PE JS. Performed the experiments: PE MM. Analyzed the data: VM PE MM. Contributed reagents/materials/analysis tools: AC MM JS CS. Wrote the paper: PE JS. Other: Edited the paper: AC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000081-Torroni1" id="pone.0000081-Torroni1"></a>Torroni A, Achilli A, Macaulay V, Richards M, Bandelt HJ (2006) Harvesting the fruit of the human mtDNA tree. Trends in Genetics.   in press.  <ul class="find" data-citedArticleID="919518" data-doi="10.1016/j.tig.2006.04.001"><li><a href="http://dx.doi.org/10.1016/j.tig.2006.04.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Harvesting+the+fruit+of+the+human+mtDNA+tree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Harvesting+the+fruit+of+the+human+mtDNA+tree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000081-Bandelt1" id="pone.0000081-Bandelt1"></a>Bandelt HJ, Salas A, Bravi C (2004) Problems in FBI mtDNA database. Science  305: 1402–1404.  <ul class="find" data-citedArticleID="919418" data-doi="10.1126/science.305.5689.1402b"><li><a href="http://dx.doi.org/10.1126/science.305.5689.1402b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Problems+in+FBI+mtDNA+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Problems+in+FBI+mtDNA+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000081-Bandelt2" id="pone.0000081-Bandelt2"></a>Bandelt HJ, Salas A, Lutz-Bonengel S (2004) Artificial recombination in forensic mtDNA population databases. Int J Legal Med  118: 267–273.  <ul class="find" data-citedArticleID="919422" data-doi="10.1007/s00414-004-0455-2"><li><a href="http://dx.doi.org/10.1007/s00414-004-0455-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Artificial+recombination+in+forensic+mtDNA+population+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Artificial+recombination+in+forensic+mtDNA+population+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000081-Bandelt3" id="pone.0000081-Bandelt3"></a>Bandelt HJ, Kong QP, Parson W, Salas A (2005) More evidence for non-maternal inheritance of mitochondrial DNA? J Med Genet  42: 957–960.  <ul class="find" data-citedArticleID="919424" data-doi="10.1136/jmg.2005.033589"><li><a href="http://dx.doi.org/10.1136/jmg.2005.033589" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=More+evidence+for+non-maternal+inheritance+of+mitochondrial+DNA%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22More+evidence+for+non-maternal+inheritance+of+mitochondrial+DNA%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000081-Bandelt4" id="pone.0000081-Bandelt4"></a>Bandelt HJ, Herrnstadt C, Yao YG, Kong QP, Kivisild T, et al.  (2003) Identification of Native American founder mtDNAs through the analysis of complete mtDNA sequences: some caveats. Ann Hum Genet  67: 512–524.  <ul class="find" data-citedArticleID="919426" data-doi="10.1046/j.1469-1809.2003.00049.x"><li><a href="http://dx.doi.org/10.1046/j.1469-1809.2003.00049.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Native+American+founder+mtDNAs+through+the+analysis+of+complete+mtDNA+sequences%3A+some+caveats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Native+American+founder+mtDNAs+through+the+analysis+of+complete+mtDNA+sequences%3A+some+caveats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000081-Bandelt5" id="pone.0000081-Bandelt5"></a>Bandelt HJ (2004) Etruscan artifacts. Am J Hum Genet  75: 919–920.  author reply 923–917.  <ul class="find" data-citedArticleID="919428" data-doi="10.1086/425180"><li><a href="http://dx.doi.org/10.1086/425180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Etruscan+artifacts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Etruscan+artifacts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000081-Brandsttter1" id="pone.0000081-Brandsttter1"></a>Brandsättter A, Parson W (2003) Mitochondrial DNA heteroplasmy or artefacts–a matter of the amplification strategy? Int J Legal Med  117: 180–184.  <ul class="find" data-citedArticleID="919444"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mitochondrial+DNA+heteroplasmy+or+artefacts%26%23x2013%3Ba+matter+of+the+amplification+strategy%3F&amp;auth=&amp;atitle=Mitochondrial+DNA+heteroplasmy+or+artefacts%26%23x2013%3Ba+matter+of+the+amplification+strategy%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+heteroplasmy+or+artefacts%26%23x2013%3Ba+matter+of+the+amplification+strategy%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+heteroplasmy+or+artefacts%26%23x2013%3Ba+matter+of+the+amplification+strategy%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000081-Yao1" id="pone.0000081-Yao1"></a>Yao YG, Macauley V, Kivisild T, Zhang YP, Bandelt HJ (2003) To trust or not to trust an idiosyncratic mitochondrial data set. Am J Hum Genet  72: 1341–1346.  author reply 1346–1349.  <ul class="find" data-citedArticleID="919524" data-doi="10.1086/375119"><li><a href="http://dx.doi.org/10.1086/375119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=To+trust+or+not+to+trust+an+idiosyncratic+mitochondrial+data+set." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22To+trust+or+not+to+trust+an+idiosyncratic+mitochondrial+data+set.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000081-Pompanon1" id="pone.0000081-Pompanon1"></a>Pompanon F, Bonin A, Bellemain E, Taberlet P (2005) Genotyping errors: causes, consequences and solutions. Nat Rev Genet  6: 847–859.  <ul class="find" data-citedArticleID="919490" data-doi="10.1038/nrg1707"><li><a href="http://dx.doi.org/10.1038/nrg1707" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotyping+errors%3A+causes%2C+consequences+and+solutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotyping+errors%3A+causes%2C+consequences+and+solutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000081-Bandelt6" id="pone.0000081-Bandelt6"></a>Bandelt HJ, Kivisild T (2006) Quality assessment of DNA sequence data: autopsy of a mis-sequenced mtDNA population sample. Ann Hum Genet  70: 314–326.  <ul class="find" data-citedArticleID="919430" data-doi="10.1111/j.1529-8817.2005.00234.x"><li><a href="http://dx.doi.org/10.1111/j.1529-8817.2005.00234.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quality+assessment+of+DNA+sequence+data%3A+autopsy+of+a+mis-sequenced+mtDNA+population+sample." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quality+assessment+of+DNA+sequence+data%3A+autopsy+of+a+mis-sequenced+mtDNA+population+sample.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000081-Pakendorf1" id="pone.0000081-Pakendorf1"></a>Pakendorf B, Stoneking M (2005) Mitochondrial DNA and human evolution. Annu Rev Genomics Hum Genet  6: 165–183.  <ul class="find" data-citedArticleID="919484" data-doi="10.1146/annurev.genom.6.080604.162249"><li><a href="http://dx.doi.org/10.1146/annurev.genom.6.080604.162249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+and+human+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+and+human+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000081-Behar1" id="pone.0000081-Behar1"></a>Behar DM, Metspalu E, Kivisild T, Achilli A, Hadid Y, et al.  (2006) The matrilineal ancestry of Ashkenazi Jewry: portrait of a recent founder event. Am J Hum Genet  78: 487–497.  <ul class="find" data-citedArticleID="919440" data-doi="10.1086/500307"><li><a href="http://dx.doi.org/10.1086/500307" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+matrilineal+ancestry+of+Ashkenazi+Jewry%3A+portrait+of+a+recent+founder+event." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+matrilineal+ancestry+of+Ashkenazi+Jewry%3A+portrait+of+a+recent+founder+event.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000081-Haak1" id="pone.0000081-Haak1"></a>Haak W, Forster P, Bramanti B, Matsumura S, Brandt G, et al.  (2005) Ancient DNA from the first European farmers in 7500-year-old Neolithic sites. Science  310: 1016–1018.  <ul class="find" data-citedArticleID="919462" data-doi="10.1126/science.1123936"><li><a href="http://dx.doi.org/10.1126/science.1123936" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+DNA+from+the+first+European+farmers+in+7500-year-old+Neolithic+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+DNA+from+the+first+European+farmers+in+7500-year-old+Neolithic+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000081-Cordaux1" id="pone.0000081-Cordaux1"></a>Cordaux R, Stoneking M (2003) South Asia, the Andamanese, and the genetic evidence for an “early” human dispersal out of Africa. Am J Hum Genet  72: 1586–1590.  author reply 1590–1583.  <ul class="find" data-citedArticleID="919452" data-doi="10.1086/375407"><li><a href="http://dx.doi.org/10.1086/375407" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=South+Asia%2C+the+Andamanese%2C+and+the+genetic+evidence+for+an+%26%23x201c%3Bearly%26%23x201d%3B+human+dispersal+out+of+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22South+Asia%2C+the+Andamanese%2C+and+the+genetic+evidence+for+an+%26%23x201c%3Bearly%26%23x201d%3B+human+dispersal+out+of+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000081-Field1" id="pone.0000081-Field1"></a>Field JS, Lahr MM (2005) Assessment of the southern dispersal: GIS-based analyses of potential routes at oxygen isotopic stage. Journal of World Prehistory  19: 1–45.  <ul class="find" data-citedArticleID="919456" data-doi="10.1007/s10963-005-9000-6"><li><a href="http://dx.doi.org/10.1007/s10963-005-9000-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessment+of+the+southern+dispersal%3A+GIS-based+analyses+of+potential+routes+at+oxygen+isotopic+stage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessment+of+the+southern+dispersal%3A+GIS-based+analyses+of+potential+routes+at+oxygen+isotopic+stage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000081-Forster1" id="pone.0000081-Forster1"></a>Forster P, Matsumura S (2005) Evolution. Did early humans go north or south? Science  308: 965–966.  <ul class="find" data-citedArticleID="919458" data-doi="10.1126/science.1113261"><li><a href="http://dx.doi.org/10.1126/science.1113261" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution.+Did+early+humans+go+north+or+south%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution.+Did+early+humans+go+north+or+south%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000081-Mellars1" id="pone.0000081-Mellars1"></a>Mellars P (2006) Why did modern human populations disperse from Africa ca. 60,000 years ago? A new model. Proc Natl Acad Sci U S A  103: 9381–9386.  <ul class="find" data-citedArticleID="919476" data-doi="10.1073/pnas.0510792103"><li><a href="http://dx.doi.org/10.1073/pnas.0510792103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+did+modern+human+populations+disperse+from+Africa+ca.+60%2C000+years+ago%3F+A+new+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+did+modern+human+populations+disperse+from+Africa+ca.+60%2C000+years+ago%3F+A+new+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000081-Endicott1" id="pone.0000081-Endicott1"></a>Endicott P, Gilbert MT, Stringer C, Lalueza-Fox C, Willerslev E, et al.  (2003) The genetic origins of the Andaman Islanders. Am J Hum Genet  72: 178–184.  <ul class="find" data-citedArticleID="919454" data-doi="10.1086/345487"><li><a href="http://dx.doi.org/10.1086/345487" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+origins+of+the+Andaman+Islanders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+origins+of+the+Andaman+Islanders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000081-Thangaraj1" id="pone.0000081-Thangaraj1"></a>Thangaraj K, Singh L, Reddy AG, Rao VR, Sehgal SC, et al.  (2003) Genetic affinities of the Andaman Islanders, a vanishing human population. Curr Biol  13: 86–93.  <ul class="find" data-citedArticleID="919512" data-doi="10.1016/s0960-9822(02)01336-2"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)01336-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+affinities+of+the+Andaman+Islanders%2C+a+vanishing+human+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+affinities+of+the+Andaman+Islanders%2C+a+vanishing+human+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000081-Thangaraj2" id="pone.0000081-Thangaraj2"></a>Thangaraj K, Chaubey G, Reddy AG, Singh VK, Singh L (2006) Autosomal STR data on the enigmatic Andaman Islanders. Forensic Sci Int.   <ul class="find" data-citedArticleID="919514" data-doi="10.1016/j.forsciint.2006.03.014"><li><a href="http://dx.doi.org/10.1016/j.forsciint.2006.03.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Autosomal+STR+data+on+the+enigmatic+Andaman+Islanders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Autosomal+STR+data+on+the+enigmatic+Andaman+Islanders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000081-Metspalu1" id="pone.0000081-Metspalu1"></a>Metspalu M, Kivisild T, Metspalu E, Parik J, Hudjashov G, et al.  (2004) Most of the extant mtDNA boundaries in south and southwest Asia were likely shaped during the initial settlement of Eurasia by anatomically modern humans. BMC Genet  5: 26.  <ul class="find" data-citedArticleID="919478"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Most+of+the+extant+mtDNA+boundaries+in+south+and+southwest+Asia+were+likely+shaped+during+the+initial+settlement+of+Eurasia+by+anatomically+modern+humans.&amp;auth=&amp;atitle=Most+of+the+extant+mtDNA+boundaries+in+south+and+southwest+Asia+were+likely+shaped+during+the+initial+settlement+of+Eurasia+by+anatomically+modern+humans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Most+of+the+extant+mtDNA+boundaries+in+south+and+southwest+Asia+were+likely+shaped+during+the+initial+settlement+of+Eurasia+by+anatomically+modern+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Most+of+the+extant+mtDNA+boundaries+in+south+and+southwest+Asia+were+likely+shaped+during+the+initial+settlement+of+Eurasia+by+anatomically+modern+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000081-Sun1" id="pone.0000081-Sun1"></a>Sun C, Kong QP, Palanichamy MG, Agrawal S, Bandelt HJ, et al.  (2006) The dazzling array of basal branches in the mtDNA macrohaplogroup M from India as inferred from complete genomes. Mol Biol Evol  23: 683–690.  <ul class="find" data-citedArticleID="919506" data-doi="10.1093/molbev/msj078"><li><a href="http://dx.doi.org/10.1093/molbev/msj078" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dazzling+array+of+basal+branches+in+the+mtDNA+macrohaplogroup+M+from+India+as+inferred+from+complete+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dazzling+array+of+basal+branches+in+the+mtDNA+macrohaplogroup+M+from+India+as+inferred+from+complete+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000081-Thangaraj3" id="pone.0000081-Thangaraj3"></a>Thangaraj K, Chaubey G, Kivisild T, Reddy AG, Singh VK, et al.  (2005) Reconstructing the origin of Andaman Islanders. Science  308: 996.  <ul class="find" data-citedArticleID="919516" data-doi="10.1126/science.1109987"><li><a href="http://dx.doi.org/10.1126/science.1109987" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstructing+the+origin+of+Andaman+Islanders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstructing+the+origin+of+Andaman+Islanders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000081-Palanichamy1" id="pone.0000081-Palanichamy1"></a>Palanichamy MG, Agrawal S, Yao YG, Kong QP, Sun C, et al.  (2006) Comment on “Reconstructing the origin of Andaman islanders”. Science  311: 470.  author reply 470.  <ul class="find" data-citedArticleID="919486" data-doi="10.1126/science.1120176"><li><a href="http://dx.doi.org/10.1126/science.1120176" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comment+on+%26%23x201c%3BReconstructing+the+origin+of+Andaman+islanders%26%23x201d%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comment+on+%26%23x201c%3BReconstructing+the+origin+of+Andaman+islanders%26%23x201d%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000081-Salas1" id="pone.0000081-Salas1"></a>Salas A, Carracedo A, Macaulay V, Richards M, Bandelt HJ (2005) A practical guide to mitochondrial DNA error prevention in clinical, forensic, and population genetics. Biochem Biophys Res Commun  335: 891–899.  <ul class="find" data-citedArticleID="919496" data-doi="10.1016/j.bbrc.2005.07.161"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2005.07.161" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+practical+guide+to+mitochondrial+DNA+error+prevention+in+clinical%2C+forensic%2C+and+population+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+practical+guide+to+mitochondrial+DNA+error+prevention+in+clinical%2C+forensic%2C+and+population+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000081-Bandelt7" id="pone.0000081-Bandelt7"></a>Bandelt HJ, Quintana-Murci L, Salas A, Macaulay V (2002) The fingerprint of phantom mutations in mitochondrial DNA data. Am J Hum Genet  71: 1150–1160.  <ul class="find" data-citedArticleID="919432" data-doi="10.1086/344397"><li><a href="http://dx.doi.org/10.1086/344397" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fingerprint+of+phantom+mutations+in+mitochondrial+DNA+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fingerprint+of+phantom+mutations+in+mitochondrial+DNA+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000081-Bandelt8" id="pone.0000081-Bandelt8"></a>Bandelt HJ, Lahermo P, Richards M, Macaulay V (2001) Detecting errors in mtDNA data by phylogenetic analysis. Int J Legal Med  115: 64–69.  <ul class="find" data-citedArticleID="919434" data-doi="10.1007/s004140100228"><li><a href="http://dx.doi.org/10.1007/s004140100228" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+errors+in+mtDNA+data+by+phylogenetic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+errors+in+mtDNA+data+by+phylogenetic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000081-Bandelt9" id="pone.0000081-Bandelt9"></a>Bandelt HJ, Achilli A, Kong QP, Salas A, Lutz-Bonengel S, et al.  (2005) Low “penetrance” of phylogenetic knowledge in mitochondrial disease studies. Biochem Biophys Res Commun  333: 122–130.  <ul class="find" data-citedArticleID="919436" data-doi="10.1016/j.bbrc.2005.04.055"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2005.04.055" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Low+%26%23x201c%3Bpenetrance%26%23x201d%3B+of+phylogenetic+knowledge+in+mitochondrial+disease+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Low+%26%23x201c%3Bpenetrance%26%23x201d%3B+of+phylogenetic+knowledge+in+mitochondrial+disease+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000081-Yao2" id="pone.0000081-Yao2"></a>Yao YG, Salas A, Bravi CM, Bandelt HJ (2006) A reappraisal of complete mtDNA variation in East Asian families with hearing impairment. Hum Genet  119: 505–515.  <ul class="find" data-citedArticleID="919526" data-doi="10.1007/s00439-006-0154-9"><li><a href="http://dx.doi.org/10.1007/s00439-006-0154-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+reappraisal+of+complete+mtDNA+variation+in+East+Asian+families+with+hearing+impairment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+reappraisal+of+complete+mtDNA+variation+in+East+Asian+families+with+hearing+impairment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000081-Cooper1" id="pone.0000081-Cooper1"></a>Cooper A, Poinar HN (2000) Ancient DNA: do it right or not at all. Science  289: 1139.  <ul class="find" data-citedArticleID="919450" data-doi="10.1126/science.289.5482.1139b"><li><a href="http://dx.doi.org/10.1126/science.289.5482.1139b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+DNA%3A+do+it+right+or+not+at+all." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+DNA%3A+do+it+right+or+not+at+all.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000081-Bandelt10" id="pone.0000081-Bandelt10"></a>Bandelt HJ (2005) Mosaics of ancient mitochondrial DNA: positive indicators of nonauthenticity. Eur J Hum Genet  13: 1106–1112.  <ul class="find" data-citedArticleID="919420" data-doi="10.1038/sj.ejhg.5201476"><li><a href="http://dx.doi.org/10.1038/sj.ejhg.5201476" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mosaics+of+ancient+mitochondrial+DNA%3A+positive+indicators+of+nonauthenticity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mosaics+of+ancient+mitochondrial+DNA%3A+positive+indicators+of+nonauthenticity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000081-Pbo1" id="pone.0000081-Pbo1"></a>Pääbo S, Poinar H, Serre D, Jaenicke-Despres V, Hebler J, et al.  (2004) Genetic analyses from ancient DNA. Annu Rev Genet  38: 645–679.  <ul class="find" data-citedArticleID="919488" data-doi="10.1146/annurev.genet.37.110801.143214"><li><a href="http://dx.doi.org/10.1146/annurev.genet.37.110801.143214" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+analyses+from+ancient+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+analyses+from+ancient+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000081-Sanchez1" id="pone.0000081-Sanchez1"></a>Sanchez JJ, Borsting C, Morling N (2005) Typing of Y chromosome SNPs with multiplex PCR methods. Methods Mol Biol  297: 209–228.  <ul class="find" data-citedArticleID="919502" data-doi="10.1385/1-59259-867-6:209"><li><a href="http://dx.doi.org/10.1385/1-59259-867-6:209" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Typing+of+Y+chromosome+SNPs+with+multiplex+PCR+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Typing+of+Y+chromosome+SNPs+with+multiplex+PCR+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000081-Syvnen1" id="pone.0000081-Syvnen1"></a>Syävnen AC (1999) From gels to chips: “minisequencing” primer extension for analysis of point mutations and single nucleotide polymorphisms. Hum Mutat  13: 1–10.  <ul class="find" data-citedArticleID="919508" data-doi="10.1002/(sici)1098-1004(1999)13:1<1::aid-humu1&gt;3.3.co;2-9"><li><a href="http://dx.doi.org/10.1002/(sici)1098-1004(1999)13:1<1::aid-humu1&gt;3.3.co;2-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+gels+to+chips%3A+%26%23x201c%3Bminisequencing%26%23x201d%3B+primer+extension+for+analysis+of+point+mutations+and+single+nucleotide+polymorphisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+gels+to+chips%3A+%26%23x201c%3Bminisequencing%26%23x201d%3B+primer+extension+for+analysis+of+point+mutations+and+single+nucleotide+polymorphisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000081-Vallone1" id="pone.0000081-Vallone1"></a>Vallone PM, Butler JM (2004) Y-SNP typing of U.S. African American and Caucasian samples using allele-specific hybridization and primer extension. J Forensic Sci  49: 723–732.  <ul class="find" data-citedArticleID="919522" data-doi="10.1520/jfs2003303"><li><a href="http://dx.doi.org/10.1520/jfs2003303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Y-SNP+typing+of+U.S.+African+American+and+Caucasian+samples+using+allele-specific+hybridization+and+primer+extension." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Y-SNP+typing+of+U.S.+African+American+and+Caucasian+samples+using+allele-specific+hybridization+and+primer+extension.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000081-Syvnen2" id="pone.0000081-Syvnen2"></a>Syvänen AC, Sajantila A, Lukka M (1993) Forensic DNA typing by the solid-phase minisequencing method. Exs  67: 275–282.  <ul class="find" data-citedArticleID="919510" data-doi="10.1007/978-3-0348-8583-6_25"><li><a href="http://dx.doi.org/10.1007/978-3-0348-8583-6_25" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Forensic+DNA+typing+by+the+solid-phase+minisequencing+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Forensic+DNA+typing+by+the+solid-phase+minisequencing+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000081-Coble1" id="pone.0000081-Coble1"></a>Coble MD, Just RS, O'Callaghan JE, Letmanyi IH, Peterson CT, et al.  (2004) Single nucleotide polymorphisms over the entire mtDNA genome that increase the power of forensic testing in Caucasians. Int J Legal Med  118: 137–146.  <ul class="find" data-citedArticleID="919448" data-doi="10.1007/s00414-004-0427-6"><li><a href="http://dx.doi.org/10.1007/s00414-004-0427-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single+nucleotide+polymorphisms+over+the+entire+mtDNA+genome+that+increase+the+power+of+forensic+testing+in+Caucasians." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single+nucleotide+polymorphisms+over+the+entire+mtDNA+genome+that+increase+the+power+of+forensic+testing+in+Caucasians.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000081-Umetsu1" id="pone.0000081-Umetsu1"></a>Umetsu K, Tanaka M, Yuasa I, Adachi N, Miyoshi A, et al.  (2005) Multiplex amplified product-length polymorphism analysis of 36 mitochondrial single-nucleotide polymorphisms for haplogrouping of East Asian populations. Electrophoresis  26: 91–98.  <ul class="find" data-citedArticleID="919520" data-doi="10.1002/elps.200406129"><li><a href="http://dx.doi.org/10.1002/elps.200406129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplex+amplified+product-length+polymorphism+analysis+of+36+mitochondrial+single-nucleotide+polymorphisms+for+haplogrouping+of+East+Asian+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplex+amplified+product-length+polymorphism+analysis+of+36+mitochondrial+single-nucleotide+polymorphisms+for+haplogrouping+of+East+Asian+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000081-Brandsttter2" id="pone.0000081-Brandsttter2"></a>Brandstätter A, Salas A, Niederstatter H, Gassner C, Carracedo A, et al.  (2006) Dissection of mitochondrial superhaplogroup H using coding region SNPs. Electrophoresis.   <ul class="find" data-citedArticleID="919446" data-doi="10.1002/elps.200500772"><li><a href="http://dx.doi.org/10.1002/elps.200500772" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissection+of+mitochondrial+superhaplogroup+H+using+coding+region+SNPs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissection+of+mitochondrial+superhaplogroup+H+using+coding+region+SNPs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000081-Quintans1" id="pone.0000081-Quintans1"></a>Quintans B, Alvarez-Iglesias V, Salas A, Phillips C, Lareu MV, et al.  (2004) Typing of mitochondrial DNA coding region SNPs of forensic and anthropological interest using SNaPshot minisequencing. Forensic Sci Int  140: 251–257.  <ul class="find" data-citedArticleID="919494" data-doi="10.1016/j.forsciint.2003.12.005"><li><a href="http://dx.doi.org/10.1016/j.forsciint.2003.12.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Typing+of+mitochondrial+DNA+coding+region+SNPs+of+forensic+and+anthropological+interest+using+SNaPshot+minisequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Typing+of+mitochondrial+DNA+coding+region+SNPs+of+forensic+and+anthropological+interest+using+SNaPshot+minisequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000081-Krause1" id="pone.0000081-Krause1"></a>Krause J, Dear PH, Pollack JL, Slatkin M, Spriggs H, et al.  (2006) Multiplex amplification of the mammoth mitochondrial genome and the evolution of Elephantidae. Nature  439: 724–727.  <ul class="find" data-citedArticleID="919470" data-doi="10.1038/nature04432"><li><a href="http://dx.doi.org/10.1038/nature04432" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplex+amplification+of+the+mammoth+mitochondrial+genome+and+the+evolution+of+Elephantidae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplex+amplification+of+the+mammoth+mitochondrial+genome+and+the+evolution+of+Elephantidae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000081-Sanchez2" id="pone.0000081-Sanchez2"></a>Sanchez JJ, Endicott P (2006) Developing high-definition multiplexed SNP assays wth special reference to degraded DNA templates. Nature Protocols.   in press.  <ul class="find" data-citedArticleID="919504" data-doi="10.1038/nprot.2006.247"><li><a href="http://dx.doi.org/10.1038/nprot.2006.247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developing+high-definition+multiplexed+SNP+assays+wth+special+reference+to+degraded+DNA+templates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developing+high-definition+multiplexed+SNP+assays+wth+special+reference+to+degraded+DNA+templates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000081-Macaulay1" id="pone.0000081-Macaulay1"></a>Macaulay V, Hill C, Achilli A, Rengo C, Clarke D, et al.  (2005) Single, rapid coastal settlement of Asia revealed by analysis of complete mitochondrial genomes. Science  308: 1034–1036.  <ul class="find" data-citedArticleID="919472" data-doi="10.1126/science.1109792"><li><a href="http://dx.doi.org/10.1126/science.1109792" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single%2C+rapid+coastal+settlement+of+Asia+revealed+by+analysis+of+complete+mitochondrial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single%2C+rapid+coastal+settlement+of+Asia+revealed+by+analysis+of+complete+mitochondrial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000081-Bower1" id="pone.0000081-Bower1"></a>Bower MA, Spencer M, Matsumura S, Nisbet RE, Howe CJ (2005) How many clones need to be sequenced from a single forensic or ancient DNA sample in order to determine a reliable consensus sequence? Nucleic Acids Res  33: 2549–2556.  <ul class="find" data-citedArticleID="919442" data-doi="10.1093/nar/gki550"><li><a href="http://dx.doi.org/10.1093/nar/gki550" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+many+clones+need+to+be+sequenced+from+a+single+forensic+or+ancient+DNA+sample+in+order+to+determine+a+reliable+consensus+sequence%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+many+clones+need+to+be+sequenced+from+a+single+forensic+or+ancient+DNA+sample+in+order+to+determine+a+reliable+consensus+sequence%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000081-Kong1" id="pone.0000081-Kong1"></a>Kong QP, Yao YG, Sun C, Bandelt HJ, Zhu CL, et al.  (2003) Phylogeny of east Asian mitochondrial DNA lineages inferred from complete sequences. Am J Hum Genet  73: 671–676.  <ul class="find" data-citedArticleID="919468" data-doi="10.1086/377718"><li><a href="http://dx.doi.org/10.1086/377718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+of+east+Asian+mitochondrial+DNA+lineages+inferred+from+complete+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+of+east+Asian+mitochondrial+DNA+lineages+inferred+from+complete+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000081-Portman1" id="pone.0000081-Portman1"></a>Portman MV (1990 [1899]) A history of our relations with the Andamanese. New Delhi: Asian Educational Services.   <ul class="find-nolinks"></ul></li><li><span class="label">47.
              </span><a name="pone.0000081-Man1" id="pone.0000081-Man1"></a>Man EH (1932 [1884]) On the aboriginal inhabitants of the Andaman Islands. London: Royal Anthropological Insitutue of Great Britain and Ireland.   <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0000081-Mishmar1" id="pone.0000081-Mishmar1"></a>Mishmar D, Ruiz-Pesini E, Golik P, Macaulay V, Clark AG, et al.  (2003) Natural selection shaped regional mtDNA variation in humans. Proc Natl Acad Sci U S A  100: 171–176.  <ul class="find" data-citedArticleID="919480" data-doi="10.1073/pnas.0136972100"><li><a href="http://dx.doi.org/10.1073/pnas.0136972100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+shaped+regional+mtDNA+variation+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+shaped+regional+mtDNA+variation+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000081-Sampietro1" id="pone.0000081-Sampietro1"></a>Sampietro ML, Gilbert MT, Lao O, Caramelli D, Lari M, et al.  (2006) Tracking down human contamination in ancient human teeth. Mol Biol Evol  23: 1801–1807.  <ul class="find" data-citedArticleID="919500" data-doi="10.1093/molbev/msl047"><li><a href="http://dx.doi.org/10.1093/molbev/msl047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tracking+down+human+contamination+in+ancient+human+teeth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tracking+down+human+contamination+in+ancient+human+teeth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000081-Sambrook1" id="pone.0000081-Sambrook1"></a>Sambrook J, Fritsch EF, Maniatis T (1989) Molecular Cloning: A Laboratory Manual. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.   <ul class="find-nolinks"></ul></li><li><span class="label">51.
              </span><a name="pone.0000081-Kivisild1" id="pone.0000081-Kivisild1"></a>Kivisild T, Rootsi S, Metspalu M, Mastana S, Kaldma K, et al.  (2003) The genetic heritage of the earliest settlers persists both in Indian tribal and caste populations. Am J Hum Genet  72: 313–332.  <ul class="find" data-citedArticleID="919466" data-doi="10.1086/346068"><li><a href="http://dx.doi.org/10.1086/346068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+heritage+of+the+earliest+settlers+persists+both+in+Indian+tribal+and+caste+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+heritage+of+the+earliest+settlers+persists+both+in+Indian+tribal+and+caste+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000081-Barnard1" id="pone.0000081-Barnard1"></a>Barnard R, Futo V, Pecheniuk N, Slattery M, Walsh T (1998) PCR bias toward the wild-type k-ras and p53 sequences: implications for PCR detection of mutations and cancer diagnosis. Biotechniques  25: 684–691.  <ul class="find" data-citedArticleID="919438"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=PCR+bias+toward+the+wild-type+k-ras+and+p53+sequences%3A+implications+for+PCR+detection+of+mutations+and+cancer+diagnosis.&amp;auth=&amp;atitle=PCR+bias+toward+the+wild-type+k-ras+and+p53+sequences%3A+implications+for+PCR+detection+of+mutations+and+cancer+diagnosis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PCR+bias+toward+the+wild-type+k-ras+and+p53+sequences%3A+implications+for+PCR+detection+of+mutations+and+cancer+diagnosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PCR+bias+toward+the+wild-type+k-ras+and+p53+sequences%3A+implications+for+PCR+detection+of+mutations+and+cancer+diagnosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000081-Haff1" id="pone.0000081-Haff1"></a>Haff LA, Smirnov IP (1997) Multiplex genotyping of PCR products with MassTag-labeled primers. Nucleic Acids Res  25: 3749–3750.  <ul class="find" data-citedArticleID="919464" data-doi="10.1093/nar/25.18.3749"><li><a href="http://dx.doi.org/10.1093/nar/25.18.3749" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplex+genotyping+of+PCR+products+with+MassTag-labeled+primers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplex+genotyping+of+PCR+products+with+MassTag-labeled+primers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000081-Norton1" id="pone.0000081-Norton1"></a>Norton N, Williams NM, Williams HJ, Spurlock G, Kirov G, et al.  (2002) Universal, robust, highly quantitative SNP allele frequency measurement in DNA pools. Hum Genet  110: 471–478.  <ul class="find" data-citedArticleID="919482" data-doi="10.1007/s00439-002-0706-6"><li><a href="http://dx.doi.org/10.1007/s00439-002-0706-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Universal%2C+robust%2C+highly+quantitative+SNP+allele+frequency+measurement+in+DNA+pools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Universal%2C+robust%2C+highly+quantitative+SNP+allele+frequency+measurement+in+DNA+pools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000081-Gilbert1" id="pone.0000081-Gilbert1"></a>Gilbert MT, Bandelt HJ, Hofreiter M, Barnes I (2005) Assessing ancient DNA studies. Trends Ecol Evol  20: 541–544.  <ul class="find" data-citedArticleID="919460" data-doi="10.1016/j.tree.2005.07.005"><li><a href="http://dx.doi.org/10.1016/j.tree.2005.07.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+ancient+DNA+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+ancient+DNA+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.XML" value="115386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.PDF" value="365598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g001.PNG_L" value="1607963"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g001.PNG_M" value="65285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g001.PNG_S" value="2434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g001.TIF" value="1784226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g001.PNG_I" value="21274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g002.PNG_L" value="1086087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g002.PNG_M" value="58671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g002.PNG_S" value="3883"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g002.TIF" value="1284088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g002.PNG_I" value="41284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g003.PNG_L" value="556197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g003.PNG_M" value="54598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g003.PNG_S" value="2877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g003.TIF" value="983800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g003.PNG_I" value="31379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g004.PNG_L" value="27189"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g004.PNG_M" value="19008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g004.PNG_S" value="2628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g004.TIF" value="58704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g004.PNG_I" value="15419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g005.PNG_L" value="14943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g005.PNG_M" value="14577"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g005.PNG_S" value="2201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g005.TIF" value="31726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g005.PNG_I" value="8139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g006.PNG_L" value="737792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g006.PNG_M" value="54803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g006.PNG_S" value="2246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g006.TIF" value="1000544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.g006.PNG_I" value="24269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.s001.EPS" value="407000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.s002.EPS" value="2253875"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.s003.EPS" value="330202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000081.s004.EPS" value="1719358"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000081&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000081" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000081&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000081&volume=&issue=&title=Multiplexed SNP Typing of Ancient DNA Clarifies the Origin of Andaman mtDNA Haplogroups amongst South Asian Tribal Populations&author_name=Phillip%20Endicott%2C%20Mait%20Metspalu%2C%20Chris%20Stringer%2C%20Vincent%20Macaulay%2C%20Alan%20Cooper%2C%20Juan%20J.%20Sanchez&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000081" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000081" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000081" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000081&amp;t=Multiplexed%20SNP%20Typing%20of%20Ancient%20DNA%20Clarifies%20the%20Origin%20of%20Andaman%20mtDNA%20Haplogroups%20amongst%20South%20Asian%20Tribal%20Populations" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000081&title=Multiplexed%20SNP%20Typing%20of%20Ancient%20DNA%20Clarifies%20the%20Origin%20of%20Andaman%20mtDNA%20Haplogroups%20amongst%20South%20Asian%20Tribal%20Populations&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000081&amp;title=Multiplexed%20SNP%20Typing%20of%20Ancient%20DNA%20Clarifies%20the%20Origin%20of%20Andaman%20mtDNA%20Haplogroups%20amongst%20South%20Asian%20Tribal%20Populations" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000081" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000081&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Multiplexed SNP Typing of Ancient DNA Clarifies the Origin of Andaman mtDNA Haplogroups amongst South Asian Tribal Populations';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000081';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Multiplexed%20SNP%20Typing%20of%20Ancient%20DNA%20Clarifies%20the%20Origin%20of%20Andaman%20mtDNA%20Haplogroups%20amongst%20South%20Asian%20Tribal%20Populations http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000081" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000081" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">






















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23002"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplogroups%22" title="Search for articles in the subject area:'Haplogroups'"><div class="flagText">Haplogroups</div></a>
              <div data-categoryid="43141" data-articleid="23002"
                   data-categoryname="Haplogroups"
                   class="flagImage" title="Flag 'Haplogroups' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondrial+DNA%22" title="Search for articles in the subject area:'Mitochondrial DNA'"><div class="flagText">Mitochondrial DNA</div></a>
              <div data-categoryid="32451" data-articleid="23002"
                   data-categoryname="Mitochondrial DNA"
                   class="flagImage" title="Flag 'Mitochondrial DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Paleogenetics%22" title="Search for articles in the subject area:'Paleogenetics'"><div class="flagText">Paleogenetics</div></a>
              <div data-categoryid="33815" data-articleid="23002"
                   data-categoryname="Paleogenetics"
                   class="flagImage" title="Flag 'Paleogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogeography%22" title="Search for articles in the subject area:'Phylogeography'"><div class="flagText">Phylogeography</div></a>
              <div data-categoryid="32107" data-articleid="23002"
                   data-categoryname="Phylogeography"
                   class="flagImage" title="Flag 'Phylogeography' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="23002"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="23002"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="23002"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7700'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1800'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3209&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=609">A back migration from Southeas Asia accounts for M31a2 in South India</a><br>Posted by Clyde98</p>
      <p><a href="/annotation/listThread.action?root=2869">see Thangaraj et al.2006_BMC Genomics!</a><br>Posted by mutant</p>
      <p><a href="/annotation/listThread.action?root=2859">Original reference is not cited!</a><br>Posted by mutant</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
