

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The +4G Site in Kozak Consensus Is Not Related to the Efficiency of Translation Initiation</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000188"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000188" />

    <meta name="citation_title" content="The +4G Site in Kozak Consensus Is Not Related to the Efficiency of Translation Initiation"/>
    <meta itemprop="name" content="The +4G Site in Kozak Consensus Is Not Related to the Efficiency of Translation Initiation"/>

      <meta name="citation_author" content="Xuhua Xia"/>
            <meta name="citation_author_institution" content="Department of Biology, University of Ottawa, Ottawa, Canada"/>

    <meta name="citation_date" content="2007/2/7"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000188.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e188"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Terminal-sequence analysis of bacterial ribosomal RNA. Correlation between the 3′-terminal-polypyrimidine sequence of 16-S RNA and translational specificity of the ribosome.; citation_author=J Shine; citation_author=L Dalgarno; citation_journal_title=Eur J Biochem; citation_volume=57; citation_number=1; citation_pages=221-230; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Determinant of cistron specificity in bacterial ribosomes.; citation_author=J Shine; citation_author=L Dalgarno; citation_journal_title=Nature; citation_volume=254; citation_number=2; citation_pages=34-38; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Initiation of translation in prokaryotes and eukaryotes.; citation_author=M Kozak; citation_journal_title=Gene; citation_volume=234; citation_number=3; citation_pages=187-208; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Effects of long 5′ leader sequences on initiation by eukaryotic ribosomes in vitro.; citation_author=M Kozak; citation_journal_title=Gene Expr; citation_volume=1; citation_number=4; citation_pages=117-125; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Possible role of flanking nucleotides in recognition of the AUG initiator codon by eukaryotic ribosomes.; citation_author=M Kozak; citation_journal_title=Nucleic Acids Res; citation_volume=9; citation_number=5; citation_pages=5233-5252; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Recognition of AUG and alternative initiator codons is augmented by G in position +4 but is not generally affected by the nucleotides in positions +5 and +6.; citation_author=M Kozak; citation_journal_title=Embo J; citation_volume=16; citation_number=6; citation_pages=2482-2492; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Not every polymorphism close to the AUG codon can be explained by invoking context effects on initiation of translation.; citation_author=M Kozak; citation_journal_title=Blood; citation_volume=101; citation_number=7; citation_pages=1202-1203; author reply 1203; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A progress report on translational control in eukaryotes.; citation_author=M Kozak; citation_journal_title=Sci STKE; citation_volume=2001; citation_number=8; citation_pages=PE1; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sequence and structural features associated with translational initiator regions in yeast–a review.; citation_author=AM Cigan; citation_author=TF Donahue; citation_journal_title=Gene; citation_volume=59; citation_number=9; citation_pages=1-18; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=The immediate downstream codon strongly influences the efficiency of utilization of eukaryotic translation initiation codons.; citation_author=S Grunert; citation_author=RJ Jackson; citation_journal_title=Embo J; citation_volume=13; citation_number=10; citation_pages=3618-3630; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The specificities of yeast methionine aminopeptidase and acetylation of amino-terminal methionine in vivo. Processing of altered iso-1-cytochromes c created by oligonucleotide transformation.; citation_author=RP Moerschell; citation_author=Y Hosokawa; citation_author=S Tsunasawa; citation_author=F Sherman; citation_journal_title=J Biol Chem; citation_volume=265; citation_number=11; citation_pages=19638-19643; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Analysis of translational initiation in coxsackievirus B3 suggests an alternative explanation for the high frequency of R+4 in the eukaryotic consensus motif.; citation_author=S Harkins; citation_author=CT Cornell; citation_author=JL Whitton; citation_journal_title=J Virol; citation_volume=79; citation_number=12; citation_pages=987-996; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Biology and Enzymology of Protein N-Myristoylation.; citation_author=TA Farazi; citation_author=G Waksman; citation_author=JI Gordon; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=13; citation_pages=39501-39504; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Posttranslational myristoylation of caspase-activated p21-activated protein kinase 2 (PAK2) potentiates late apoptotic events.; citation_author=GL Vilas; citation_author=MM Corvi; citation_author=GJ Plummer; citation_author=AM Seime; citation_author=GR Lambkin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=14; citation_pages=6542-6547; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Posttranslational N-myristoylation is required for the anti-apoptotic activity of human tGelsolin, the C-terminal caspase cleavage product of human gelsolin.; citation_author=N Sakurai; citation_author=T Utsumi; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=15; citation_pages=14288-14295; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The myristoylation of TRIF-related adaptor molecule is essential for Toll-like receptor 4 signal transduction.; citation_author=DC Rowe; citation_author=AF McGettrick; citation_author=E Latz; citation_author=BG Monks; citation_author=NJ Gay; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=16; citation_pages=6299-6304; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tomato Pto encodes a functional N-myristoylation motif that is required for signal transduction in Nicotiana benthamiana.; citation_author=JS de Vries; citation_author=VM Andriotis; citation_author=AJ Wu; citation_author=JP Rathjen; citation_journal_title=Plant J; citation_volume=45; citation_number=17; citation_pages=31-45; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The pseudomonas syringae pv. tomato DC3000 Type III effector HopF2 has a putative myristoylation site required for its avirulence and virulence functions.; citation_author=A Robert-Seilaniantz; citation_author=L Shan; citation_author=JM Zhou; citation_author=X Tang; citation_journal_title=Mol Plant Microbe Interact; citation_volume=19; citation_number=18; citation_pages=130-138; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Role of myristoylation and N-terminal basic residues in membrane association of the human immunodeficiency virus type 1 Nef protein.; citation_author=M Bentham; citation_author=S Mazaleyrat; citation_author=M Harris; citation_journal_title=J Gen Virol; citation_volume=87; citation_number=19; citation_pages=563-571; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biochemical indication for myristoylation-dependent conformational changes in HIV-1 Nef.; citation_author=S Breuer; citation_author=H Gerlach; citation_author=B Kolaric; citation_author=C Urbanke; citation_author=N Opitz; citation_journal_title=Biochemistry; citation_volume=45; citation_number=20; citation_pages=2339-2349; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The effect of HIV-1 Gag myristoylation on membrane binding.; citation_author=P Provitera; citation_author=R El-Maghrabi; citation_author=S Scarlata; citation_journal_title=Biophys Chem; citation_volume=119; citation_number=21; citation_pages=23-32; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A sampling of the yeast proteome.; citation_author=B Futcher; citation_author=GI Latter; citation_author=P Monardo; citation_author=CS McLaughlin; citation_author=JI Garrels; citation_journal_title=Mol Cell Biol; citation_volume=19; citation_number=22; citation_pages=7357-7368; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Correlation between protein and mRNA abundance in yeast.; citation_author=SP Gygi; citation_author=Y Rochon; citation_author=BR Franza; citation_author=R Aebersold; citation_journal_title=Mol Cell Biol; citation_volume=19; citation_number=23; citation_pages=1720-1730; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Integrated genomic and proteomic analyses of a systematically perturbed metabolic network.; citation_author=T Ideker; citation_author=V Thorsson; citation_author=JA Ranish; citation_author=R Christmas; citation_author=J Buhler; citation_journal_title=Science; citation_volume=292; citation_number=24; citation_pages=929-934; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Complementary profiling of gene expression at the transcriptome and proteome levels in Saccharomyces cerevisiae.; citation_author=TJ Griffin; citation_author=SP Gygi; citation_author=T Ideker; citation_author=B Rist; citation_author=J Eng; citation_journal_title=Mol Cell Proteomics; citation_volume=1; citation_number=25; citation_pages=323-333; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Integrated Genomic and Proteomic Analyses of Gene Expression in Mammalian Cells.; citation_author=Q Tian; citation_author=SB Stepaniants; citation_author=M Mao; citation_author=L Weng; citation_author=MC Feetham; citation_journal_title=Mol Cell Proteomics; citation_volume=3; citation_number=26; citation_pages=960-969; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Reproducibility, bioinformatic analysis and power of the SAGE method to evaluate changes in transcriptome.; citation_author=S Dinel; citation_author=C Bolduc; citation_author=P Belleau; citation_author=A Boivin; citation_author=M Yoshioka; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=27; citation_pages=e26; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of human transcriptomes.; citation_author=VE Velculescu; citation_author=SL Madden; citation_author=L Zhang; citation_author=AE Lash; citation_author=J Yu; citation_journal_title=Nat Genet; citation_volume=23; citation_number=28; citation_pages=387-388; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Discordant protein and mRNA expression in lung adenocarcinomas.; citation_author=G Chen; citation_author=TG Gharib; citation_author=CC Huang; citation_author=JM Taylor; citation_author=DE Misek; citation_journal_title=Mol Cell Proteomics; citation_volume=1; citation_number=29; citation_pages=304-313; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=JH Zar; citation_number=30; citation_date=1999; citation_publisher=Prentice Hall; " />
      <meta name="citation_reference" content="citation_title=The codon Adaptation Index–a measure of directional synonymous codon usage bias, and its potential applications.; citation_author=PM Sharp; citation_author=WH Li; citation_journal_title=Nucleic Acids Res; citation_volume=15; citation_number=31; citation_pages=1281-1295; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Relationship of codon bias to mRNA concentration and protein length in Saccharomyces cerevisiae.; citation_author=A Coghlan; citation_author=KH Wolfe; citation_journal_title=Yeast; citation_volume=16; citation_number=32; citation_pages=1131-1145; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=EMBOSS: the European Molecular Biology Open Software Suite.; citation_author=P Rice; citation_author=I Longden; citation_author=A Bleasby; citation_journal_title=Trends Genet; citation_volume=16; citation_number=33; citation_pages=276-277; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=DAMBE: Software package for data analysis in molecular biology and evolution.; citation_author=X Xia; citation_author=Z Xie; citation_journal_title=Journal of Heredity; citation_volume=92; citation_number=34; citation_pages=371-373; citation_date=2001; " />
      <meta name="citation_reference" content="citation_author=X Xia; citation_number=35; citation_pages=277 p; citation_date=2001; citation_publisher=Kluwer Academic Publishers; " />
      <meta name="citation_reference" content="citation_title=Evolution of synonymous codon usage in metazoans.; citation_author=L Duret; citation_journal_title=Current Opinion in Genetics & Development; citation_volume=12; citation_number=36; citation_pages=640-649; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=No evidence for tissue-specific adaptation of synonymous codon usage in humans.; citation_author=M Semon; citation_author=JR Lobry; citation_author=L Duret; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=37; citation_pages=523-529; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Codon usage in bacteria: correlation with gene expressivity.; citation_author=M Gouy; citation_author=C Gautier; citation_journal_title=Nucleic Acids Res; citation_volume=10; citation_number=38; citation_pages=7055-7064; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E coli translational system.; citation_author=T Ikemura; citation_journal_title=J Mol Biol; citation_volume=151; citation_number=39; citation_pages=389-409; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Mutation and selection on the anticodon of tRNA genes in vertebrate mitochondrial genomes.; citation_author=X Xia; citation_journal_title=Gene; citation_volume=345; citation_number=40; citation_pages=13-20; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=How optimized is the translational machinery in Escherichia coli, Salmonella typhimurium and Saccharomyces cerevisiae?; citation_author=X Xia; citation_journal_title=Genetics; citation_volume=149; citation_number=41; citation_pages=37-44; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Tissue-specific codon usage and the expression of human genes.; citation_author=JB Plotkin; citation_author=H Robins; citation_author=AJ Levine; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=42; citation_pages=12588-12591; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.; citation_author=TM Lowe; citation_author=SR Eddy; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=43; citation_pages=955-964; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Point mutations define a sequence flanking the AUG initiator codon that modulates translation by eukaryotic ribosomes.; citation_author=M Kozak; citation_journal_title=Cell; citation_volume=44; citation_number=44; citation_pages=283-292; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Highly expressed proteins have an increased frequency of alanine in the second amino acid position.; citation_author=A Tats; citation_author=M Remm; citation_author=T Tenson; citation_journal_title=BMC Genomics; citation_volume=7; citation_number=45; citation_pages=28; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sequence determinants of cytosolic N-terminal protein processing.; citation_author=C Flinta; citation_author=B Persson; citation_author=H Jornvall; citation_author=G von Heijne; citation_journal_title=Eur J Biochem; citation_volume=154; citation_number=46; citation_pages=193-196; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=The Proteomics of N-terminal Methionine Cleavage.; citation_author=F Frottin; citation_author=A Martinez; citation_author=P Peynot; citation_author=S Mitra; citation_author=RC Holz; citation_journal_title=Mol Cell Proteomics; citation_volume=5; citation_number=47; citation_pages=2336-2349; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000188" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The +4G Site in Kozak Consensus Is Not Related to the Efficiency of Translation Initiation"/>
    <meta name="twitter:description" content="The optimal context for translation initiation in mammalian species is GCCRCCaugG (where R&#8202;=&#8202;purine and &ldquo;aug&rdquo; is the initiation codon), with the -3R and +4G being particularly important. The presence of +4G has been interpreted as necessary for efficient translation initiation. Accumulated experimental and bioinformatic evidence has suggested an alternative explanation based on amino acid constraint on the second codon, i.e., amino acid Ala or Gly are needed as the second amino acid in the nascent peptide for the cleavage of the initiator Met, and the consequent overuse of Ala and Gly codons (GCN and GGN) leads to the +4G consensus. I performed a critical test of these alternative hypotheses on +4G based on 34169 human protein-coding genes and published gene expression data. The result shows that the prevalence of +4G is not related to translation initiation. Among the five G-starting codons, only alanine codons (GCN), and glycine codons (GGN) to a much smaller extent, are overrepresented at the second codon, whereas the other three codons are not overrepresented. While highly expressed genes have more +4G than lowly expressed genes, the difference is caused by GCN and GGN codons at the second codon. These results are inconsistent with +4G being needed for efficient translation initiation, but consistent with the proposal of amino acid constraint hypothesis."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000188.g001"/>

  <meta property="og:title" content="The +4G Site in Kozak Consensus Is Not Related to the Efficiency of Translation Initiation" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000188" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=764'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6847'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6187&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000188">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000188" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000188&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The +4G Site in Kozak Consensus Is Not Related to the Efficiency of Translation Initiation
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xuhua Xia
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:xxia@uottawa.ca">xxia@uottawa.ca</a>
            </p>

                <p>Affiliation:
                  Department of Biology, University of Ottawa, Ottawa, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 07, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000188</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000188" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000188" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000188" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000188" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000188-g001" data-doi="info:doi/10.1371/journal.pone.0000188" data-uri="info:doi/10.1371/journal.pone.0000188.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000188.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000188-t001" data-doi="info:doi/10.1371/journal.pone.0000188" data-uri="info:doi/10.1371/journal.pone.0000188.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000188.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000188">Reader Comments (5)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000188" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The optimal context for translation initiation in mammalian species is GCCRCCaugG (where R = purine and “aug” is the initiation codon), with the -3R and +4G being particularly important. The presence of +4G has been interpreted as necessary for efficient translation initiation. Accumulated experimental and bioinformatic evidence has suggested an alternative explanation based on amino acid constraint on the second codon, i.e., amino acid Ala or Gly are needed as the second amino acid in the nascent peptide for the cleavage of the initiator Met, and the consequent overuse of Ala and Gly codons (GCN and GGN) leads to the +4G consensus. I performed a critical test of these alternative hypotheses on +4G based on 34169 human protein-coding genes and published gene expression data. The result shows that the prevalence of +4G is not related to translation initiation. Among the five G-starting codons, only alanine codons (GCN), and glycine codons (GGN) to a much smaller extent, are overrepresented at the second codon, whereas the other three codons are not overrepresented. While highly expressed genes have more +4G than lowly expressed genes, the difference is caused by GCN and GGN codons at the second codon. These results are inconsistent with +4G being needed for efficient translation initiation, but consistent with the proposal of amino acid constraint hypothesis.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Xia X (2007) The +4G Site in Kozak Consensus Is Not Related to the Efficiency of Translation Initiation. PLoS ONE 2(2):
          e188.
            doi:10.1371/journal.pone.0000188</p><p><strong>Academic Editor: </strong>Emmanouil Dermitzakis, The Wellcome Trust Sanger Institute, United Kingdom</p><p><strong>Received:</strong> September 7, 2006; <strong>Accepted:</strong> January 5, 2007; <strong>Published:</strong> February 7, 2007</p><p><strong>Copyright:</strong> © 2007 Xuhua Xia. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This project is supported by the Discovery, Strategic Research, and Research Tools and Instruments Grants from the Natural Science and Engineering Research Council of Canada.</p><p><strong>Competing interests:</strong> The author has declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>While translation initiation in prokaryotes is mediated by base-paring between the Shine-Dalgarno sequence at the 5-UTR on the mRNA and the anti-Shine-Dalgarno sequence at the 3′-end of the 16S rRNA <a href="#pone.0000188-Shine1">[1]</a>, <a href="#pone.0000188-Shine2">[2]</a>, translation initiation in eukaryotes is mediated by the Kozak consensus <a href="#pone.0000188-Kozak1">[3]</a>–<a href="#pone.0000188-Kozak4">[6]</a>. The optimal context for translation initiation in mammalian species is GCCRCCaugG (where R = purine), with the −3R and +4G being particularly important <a href="#pone.0000188-Kozak1">[3]</a>, <a href="#pone.0000188-Kozak4">[6]</a>–<a href="#pone.0000188-Kozak6">[8]</a>. Molecular biology textbooks abound with the implication that the −3R and +4G should be salient features of mRNA for highly expressed proteins.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The interpretation of +4G has been controversial. It has been suggested that +4G may have little to do with initiation site recognition, but is constrained by the requirement for particular type of amino acid residue at the N-terminus of the protein <a href="#pone.0000188-Cigan1">[9]</a>. One piece of supporting evidence came from a detailed study of an influenza virus NS cDNA derivative <a href="#pone.0000188-Grunert1">[10]</a> which showed that both +4 and +5 sites were important and changes at these sites reduced protein production. In contrast, the +6 site (the third codon position of the second codon) is less important. A simple explanation of this result is that changes at the +4 and +5 sites alter the amino acid, whereas those at the +6 site may not.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Recent studies, especially those involving the removal of the initiator methionine (Met) and myristoylation, revived the alternative explanation of amino acid constraint for the presence of +4G in protein-coding genes. First, amino-terminal modifications of nascent peptides occur in nearly all proteins in both prokaryotes and eukaryotes, and the removal of the initiator Met, which occurs soon after the amino terminus of the growing polypeptide chain emerges from the ribosome, is not only an important amino-terminal modification in itself, but also required for further amino-terminal modifications. The efficiency of removing the initiator Met depends heavily on the penultimate (the second) amino acid, with the cleavage occurring most efficiently when the penultimate amino acid is small <a href="#pone.0000188-Moerschell1">[11]</a>. Alanine (Ala) and glycine (Gly) happen to be the two smallest amino acids and both are coded by G-starting codons, i.e., Ala by the GCN (where N stands for any nucleotide) and Gly by the GGN codons. The need for removing the initiator Met in proteins implies the presence of many Ala and Gly at the penultimate amino acid position and consequently many +4G due to the GCN and GGN codons coding for Ala and Gly, respectively.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Another factor contributing to the prevalence of +4G, but independent of the efficiency of translation initiation, is the myristoylation process. For example, in Coxsackievirus B3, the initiation codon is flanked by both −3R and +4G, and viral mutants with a mutation from +4G to +4C is not viable <a href="#pone.0000188-Harkins1">[12]</a>. This may seem to confirm what one would expect based on the necessity of the Kozak consensus for efficient translation initiation in highly expressed genes. However, it turns out that the +4G is required in Coxsackievirus B3 not because it is essential for translation initiation, but because it is needed for coding Gly (coded by GGN). The Gly at the amino terminus, after the removal of the initiator methionine, is needed to attach to a myristoyl (C<sub>14</sub>H<sub>28</sub>O<sub>2</sub>) fatty acid side chain, and myristoylation occurs only on a Gly residue <a href="#pone.0000188-Farazi1">[13]</a>. Myristoylation may involve many proteins, and are implicated in protein subcellular relocalization <a href="#pone.0000188-Farazi1">[13]</a>, apoptosis <a href="#pone.0000188-Vilas1">[14]</a>, <a href="#pone.0000188-Sakurai1">[15]</a>, signal transduction <a href="#pone.0000188-Rowe1">[16]</a>, <a href="#pone.0000188-deVries1">[17]</a>, and the virulence and colonization of pathogens <a href="#pone.0000188-Harkins1">[12]</a>, <a href="#pone.0000188-RobertSeilaniantz1">[18]</a>–<a href="#pone.0000188-Provitera1">[21]</a>. The need for myristoylation in proteins would contribute to the presence of +4G in CDSs.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>We thus have two alternative hypotheses for the presence of +4G in protein-coding genes. The conventional translation initiation hypothesis argues that the presence of +4G is necessary for highly expressed proteins, with two predictions. First, the selection favoring +4G should drive the increased usage of amino acids coded by GNN codons (e.g., Ala coded by GCN, Asp by GAY, Glu by GAR, Gly by GGN, and Val by GUN) at the penultimate amino acid site. Second, the +4G should be more prevalent in highly expressed than in lowly expressed genes. In contrast, the amino acid constraint hypothesis, based on the amino-terminal modification involving the removal of the initiator Met and myristoylation, has two different predictions. First, not all GNN codons should have increased usage, but only GCN coding Ala and GGN coding Gly should have increased usage. Second, highly expressed genes may need more efficient N-terminal processing and may consequently need more GCN and GGN codons. This may increase the frequency of +4G in highly expressed genes relative to lowly expressed genes.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Differential use of GNN codons at penultimate site</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Results from 34169 human coding sequences (CDSs) do not support the translation initiation hypothesis for the presence of +4G. While the five amino acids coded by GNN codons (Ala, Asp, Gly, Glu, Val) account for a majority (64.24%) of the amino acids at the penultimate site (which implies that nucleotide G is the consensus nucleotide at the +4 site), there is no consistent overuse of amino acids coded by GNN codons at the second amino acid site relative to other sites (<a href="#pone-0000188-g001">Fig. 1</a>). This pattern also holds for mouse genes (data not shown).</p>
<div class="figure" id="pone-0000188-g001"><div class="img"><a name="pone-0000188-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000188.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000188" data-uri="info:doi/10.1371/journal.pone.0000188.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000188.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000188.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000188.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000188.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000188.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000188.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000188.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000188.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000188.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Frequencies of the penultimate codons (N<sub>2</sub>) relative to the frequencies of the rest of the codons (N<sub>3+</sub>) excluding initiation and termination codons. The line indicates the expected frequencies when the penultimate codons have the same codon usage as the rest of the codons.</p>
<span>doi:10.1371/journal.pone.0000188.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The expected number of codons (<a href="#pone-0000188-g001">Fig. 1</a>) at the penultimate site is calculated as follows. The total number of codons at non-penultimate sites is 16347992 (excluding the initiation and termination codon). Designate the number of codon XYZ at non-penultimate site as N<sub>XYZ</sub>. If codon usage at penultimate sites is the same as the rest of the genes, then the expected frequency of codon XYZ at the penultimate site is simply N<sub>XYZ</sub>*34169/16347992. Only alanine (GCN) codons deviate dramatically from the expected value (<a href="#pone-0000188-g001">Fig. 1</a>).GUN codons (coding for valine) is in fact underused at the penultimate site than at other sites (<a href="#pone-0000188-g001">Fig. 1</a>). Thus, there is no general increase in GNN codon usage at the penultimate site.</p>


<h4>Differences in +4G frequencies in highly and lowly expressed genes</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The translation initiation hypothesis also predicts that highly expressed genes should be more likely to have +4G than lowly expressed genes. Ideally we should have genes with different protein expression for testing the prediction. However, there is now substantial evidence suggesting a strong correlation between mRNA level and protein production, not only in <em>Saccharomyces cerevisiae</em> 
               <a href="#pone.0000188-Futcher1">[22]</a>–<a href="#pone.0000188-Griffin1">[25]</a>, but also in mammalian species <a href="#pone.0000188-Tian1">[26]</a>. We used published SAGE (serial analysis of gene expression) data to characterize gene expression because comparative studies have also demonstrated a much higher reproducibility of SAGE (serial analysis of gene expression) experiments than microarray experiments in characterizing mRNA levels <a href="#pone.0000188-Dinel1">[27]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To check any possible differences in amino acid usage at the penultimate site and at other sites between highly and lowly expressed genes, I used the 987 unique SAGE tags that were found ubiquitously in human tissues <a href="#pone.0000188-Velculescu1">[28]</a>. The main reason for using ubiquitously expressed genes is that the relationship between mRNA level and protein abundance is generally weak in cell-specific genes <a href="#pone.0000188-Chen1">[29]</a>.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>These 987 unique tags were matched against the 34169 human CDSs. One gene (ASNA1) matched 3 tags, 16 genes matched 2 tags and 987 genes matched exactly 1 tag (The number happens to be the same as the number of unique tags, but it is accidental). For those 17 multiple-match genes (MMGs), it is difficult to assign gene expression values. For example, if a gene matches two tags, one with 10 copies/cell and another with 100 copies/cell, there is no unequivocal way of assigning an expression value to the gene. For this reason, only 987 single-match genes (SMG) are used and the data set will be referred to as SMG data set.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The SMG data set has the problem involving multiple-match tags. For example, if a tag has n copies per cell and matches two genes, say SMG1 and SMG2, it is impossible to know if the n copies/cell of the tag is contributed by SMG 1 only, or SMG 2 only, or by both. In order to assign expression values unequivocally to genes, we also compiled a more limited data set, with on 168 genes that match only single-match tags (SMTs), i.e., each gene matches only one tag which matches only one gene. These genes are designated as SSGs (to reflect the fact that they are from SMG-SMT gene-tag pairs) and their expression values range from 11 copies/cell (gene LRFN4 matching GGGGGGCUGC, excluding the leading 4-bp <em>Nla</em>III anchoring enzyme site) to 4374 copies/cell (gene GRIN2C matching GGUGACCACG). This small data set will be referred to as the SSG data set.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>The 168 genes in the SSG data set were divided into a high-expression (HE) group, including 83 SSGs with expression level of at least 50 copies/cell, and a low expression (LE) group, including 85 SSGs with expression level less than 50 copies/cell. The proportion of +4G is 43.37% in the HE group and 49.40% in the LE group. We also contrasted 30 most highly expressed SSGs (expression level at least 114 copies/cell) with 30 least expressed SSGs (expression level equal to 27 copies/cell or less). The proportion of +4G is 43.33% for the former and 46.67% for the latter. Thus, there is no indication that highly expressed genes are more likely to have +4G than lowly expressed genes. The difference is in fact in the opposite direction. This result does not support the prediction of the translation initiation hypothesis.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>We then categorized the 168 SSGs into five groups according to the codon at the penultimate site (GCN for alanine, GGN for glycine, GAN for aspartate and glutamate, GUN for valine, and genes without +4G designated as NonG), and compared their expression values by one-way analysis of variance (ANOVA). Gene expression differs significantly among the five groups (F = 3.07, DF1 = 4, DF2 = 163, p = 0.0180), with the average gene expression being 92.5 for GCN genes, 554.89 for GGN genes, 116.74 for GAN genes, 89.40 for GUN genes and 123.91 for NonG genes. Multiple comparisons using the LSD (least significance difference) test [30, pp. 208–209] showed that only genes with GGN (glycine) codons at their penultimate site have significantly higher expression than other groups (p&lt;0.05). One of the genes with a glycine codon (GGU) at its penultimate site (GRIN2C matching GGUGACCACG) has a very high expression value (4374 copies/cell). Excluding this gene results in no significant difference among the five groups.</p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>We have performed similar ANOVA for the SMG data set and found the same result, i.e., genes with GGN codons at their penultimate sites have higher expression value than the other four groups, with the average gene expression being 178.51 for GCN genes, 263.21 for GGN genes, 163.79 for GAN genes, 175.52 for GUN genes and 145.14 for NonG genes. There is no other significance difference among the five groups. Thus, we may conclude from analyzing the two SAGE data sets that (1) there is no consistent pattern that GNN codons are overused in highly expressed genes, and (2) genes with GGN (glycine) codons at their penultimate site tend to be more highly expressed than other genes. The result is inconsistent with the translation initiation hypothesis but not incompatible with the amino acid constraint hypothesis.</p>
<a id="article1.body1.sec2.sec2.p8" name="article1.body1.sec2.sec2.p8"></a><p>An alternative index of gene expression is codon adaptation index, or CAI <a href="#pone.0000188-Sharp1">[31]</a>, which has been shown to correlate well with published gene expression in terms of mRNA level and protein abundance <a href="#pone.0000188-Futcher1">[22]</a>, <a href="#pone.0000188-Gygi1">[23]</a>, <a href="#pone.0000188-Coghlan1">[32]</a>. Note that CAI is computed with a codon usage table from a set of reference genes known to be highly expressed. I used the reference set in the Ehum.cut file that is distributed with EMBOSS <a href="#pone.0000188-Rice1">[33]</a>. However, the cai program in EMBOSS is biased because it does not exclude codon families with a single codon, e.g., AUG coding methionine and UGG coding tryptophan in the standard genetic code (see <a href="#s4">Materials and Methods</a> for details). I used DAMBE [34,35, version 4.5.10] to calculate CAI values.</p>
<a id="article1.body1.sec2.sec2.p9" name="article1.body1.sec2.sec2.p9"></a><p>We focus on two groups of genes, the high-CAI group with CAI&gt;0.8 and the low-CAI group with CAI&lt;0.7. Overall, high-CAI genes tend to have more genes with +4G than low-CAI genes (chi-square test, X<sup>2</sup> = 25.36, DF = 1, p&lt;0.0001, based on data in <a href="#pone-0000188-t001">Table 1</a>). This might seem to support the translation initiation hypothesis. However, genes with no +4G may include more false CDSs that would tend to have smaller CAI values. This would result in an association between low-CAI genes with no +4G. It is important to note that, among genes with +4G, there is little difference in their frequencies between the high-CAI and low-CAI group (<a href="#pone-0000188-t001">Table 1</a>, last column). The largest difference between high-CAI and low-CAI group are genes with GCN and GGN codons (coding for alanine and glycine, respectively) at their penultimate site (<a href="#pone-0000188-t001">Table 1</a>, last column), but the difference is not significant (p&gt;0.05). Thus, we may conclude that only GCN and GGN codons exhibit minor differences in their frequencies at the penultimate site between high-CAI and low-CAI genes. This is again inconsistent with the translation initiation hypothesis, but somewhat compatible with the amino acid constraint hypothesis.</p>
<div class="figure" id="pone-0000188-t001"><div class="img"><a name="pone-0000188-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000188.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000188" data-uri="info:doi/10.1371/journal.pone.0000188.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000188.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000188.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000188.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000188.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000188.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000188.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000188.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000188.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000188.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Number of genes with GCN, GGN, GAN, GUN and non-GNN codons at their penultimate site in the high-CAI and low-CAI group (N<sub>High-CAI</sub> and N<sub>Low-CAI</sub>), together with their proportions (P<sub>High-CAI</sub> and P<sub>Low-CAI</sub>).</span></strong></p><span>doi:10.1371/journal.pone.0000188.t001</span></div><a id="article1.body1.sec2.sec2.p10" name="article1.body1.sec2.sec2.p10"></a><p>The use of CAI value <a href="#pone.0000188-Sharp1">[31]</a> as an index of gene expression in human has been controversial <a href="#pone.0000188-Duret1">[36]</a>, <a href="#pone.0000188-Semon1">[37]</a>. While it is well established that codon usage in bacterial species and vertebrate mitochondria is strongly constrained by the relative tRNA abundance and that there is selection pressure favoring codon-anticodon adaptation <a href="#pone.0000188-Gouy1">[38]</a>–<a href="#pone.0000188-Xia4">[41]</a>, there is only limited evidence for eukaryotes <a href="#pone.0000188-Plotkin1">[42]</a>. One additional piece of evidence supporting codon-anticodon adaptation is that the codon frequencies of the 34169 annotated human CDSs are positively correlated with the copy number of their cognate tRNA genes found at The Genomic tRNA Database (<a href="http://lowelab.ucsc.edu/GtRNAdb/">http://lowelab.ucsc.edu/GtRNAdb/</a>), compiled with the tRNAscan-SE program <a href="#pone.0000188-Lowe1">[43]</a>. For example, from the 505 human tRNA genes decoding the regular set of 20 amino acids, one can obtain their cognate codon frequencies from their anticodons. These tRNA-derived cognate codon frequencies correlate positively with the codon frequencies of the 168 genes in the SSG data set (Pearson r = 0.5731, p&lt;0.0001, after grouping all C-ending and U-ending codons into Y-ending codons because these codons are typically translated by tRNA with nucleotide G at its wobble site). It is simpler to explain this significant positive correlation by invoking codon-anticodon adaptation than by random mutation, and suggests the utility of CAI as a measure of gene expression in human genes.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The original study documenting the importance of +4G <a href="#pone.0000188-Kozak7">[44]</a> does not constitute a sufficient proof that +4G is important in translation initiation. The study was based on the production of proinsulin from preproinsulin. The latter has a signal peptide at its amino terminus. The signal peptide is removed during translation, generating proinsulin. When +4G is mutated to other nucleotide, the production of proinsulin is reduced. This reduced proinsulin production was assumed to be caused by reduced efficiency in translation initiation due to the mutation of +4G to other nucleotides. However, one should note that altering +4G also alter the amino acid sequence in the signal peptide and may consequently affect the removal of the signal peptide, leading to reduced production of proinsulin. Thus, the result is compatible with the amino acid constraint hypothesis for the presence of +4G.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>One may think that the translation initiation hypothesis is partially correct because, after all, two of the five amino acids with G-starting codons (especially alanine) show increased usage. This is wrong. The increased usage of alanine exists not only in eukaryotes, but also in prokaryotes <a href="#pone.0000188-Tats1">[45]</a> that do not use the scanning mechanism for translation initiation and consequently do not need the +4G. The overuse of small amino acids at the penultimate amino acid site in both prokaryotes and eukaryotes is better explained by the necessity for removing the initiator Met.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>While the results above are inconsistent with the predictions of the translation initiation hypothesis, they generally appear to support the amino acid constraint hypothesis. First, the latter predicted the overuse of Ala and Gly at the second codon position (to facilitate the removal of the initiator Met and myristoylation), and Ala and Gly (especially Ala) are indeed overused (<a href="#pone-0000188-t001">Table 1</a>). Second, all differences between highly expressed and lowly expressed genes involve GCN and GGN (coding for alanine and glycine, respectively) codons.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Both amino acid constraint hypothesis and the translation initiation hypothesis have difficulties in explain certain observations. For example, Kozak <a href="#pone.0000188-Kozak4">[6]</a> found that +4G generally enhances translation initiation, but does not when it occurs in a GUN codon (coding for valine). An associated finding in this paper is that GUN is underused at the penultimate site (<a href="#pone-0000188-g001">Fig. 1</a>). While such findings are difficult to explain by the translation initiation hypothesis, they are also difficult for the amino acid constraint hypothesis unless valine at the penultimate site reduces the efficiency of initiator Met cleavage. Previous studies on prokaryotes and eukaryotes <a href="#pone.0000188-Flinta1">[46]</a> and on the yeast, <em>Saccharomyces cerevisiae</em> 
            <a href="#pone.0000188-Moerschell1">[11]</a>, suggest that cleavage of initiator Met occurs with valine at the penultimate site, but a recent study on <em>Escherichia coli</em> 
            <a href="#pone.0000188-Frottin1">[47]</a> demonstrates that peptides with Val at the penultimate site dramatically reduces the efficiency of initiator Met cleavage relative to other amino acids such as Ala, Cys, Gly, Pro, or Ser in this position. Further studies on initiator Met cleavage on mammalian species are needed before one can reach a solid conclusion.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In summary, we conclude that the presence of +4G is poorly explained by the translation initiation hypothesis that claims the necessity of +4G for efficient translation initiation, but well explained by the alternative amino acid constraint hypothesis that claims the necessity of Ala and Gly at the second amino acid position in many proteins (for the removal of the initiator Met or myristoylation) as the cause of the prevalence of +4G because Ala and Gly happen to be coded by GCN and GGN codons. The necessity of +4G for efficient translation initiation appears to be a misconception that has existed in the molecular biology textbooks for too long.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>I retrieved the rna.gbk.gz file at <a href="ftp://ftp.ncbi.nih.gov/genomes/H_sapiens/RNA/">ftp://ftp.ncbi.nih.gov/genomes/H_sapiens​/RNA/</a>, dated Sept. 3, 2006, and extracted all 34169 annotated coding sequences (CDSs) for evaluating the translation initiation hypothesis and the amino acid constraint hypothesis. CDS extraction and computation of codon adaptation index <a href="#pone.0000188-Sharp1">[31]</a>, as well as the analysis of codon usage at the second codon were carried out by using DAMBE (Xia 2001; Xia and Xie 2001). CDSs that are not multiples of three or not terminated with a stop codon are excluded.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Because the translation initiation hypothesis predicts that highly expressed genes should be more likely to have +4G than lowly expressed genes, we have used genes of different expression levels to check this prediction. Gene expression level is measured in two ways in this study, one by SAGE (serial analysis of gene expression) data and one by using codon adaptation index. The use of SAGE data instead of available microarray data is mainly because of the much higher reproducibility of SAGE experiments than microarray experiments <a href="#pone.0000188-Dinel1">[27]</a>.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>SAGE data were retrieved from <a href="http://www.nature.com/ng/journal/v23/n4/extref/ng1299-387b-S1.pdf">http://www.nature.com/ng/journal/v23/n4/​extref/ng1299-387b-S1.pdf</a> which listed, among others, 987 unique tags that are ubiquitously expressed in different human tissues, together with their abundance in copies/cell.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>We searched these tags against the 34169 human CDSs for exact matches. To facilitate presentation, we define multiple-match genes (MMGs) as those CDSs each matching multiple tags, single-match genes (SMGs) as those each matching a single tag, multiple-match tags (MMTs) as those tags each matching multiple CDSs, and single-match tags (SMTs) as those each matching a single CDS. It is difficult to assess the expression level of an MMG because different tags it matches have different copies/cell values. The presence of MMTs causes an even more serious problem. For example, when a tag with 100 copies/cell matches two genes, it is impossible to know if the 100 copies are contributed by only one gene or by both. It would be methodologically wrong to assign both genes an expression value of 100 copies/cell. For this reason, we have compiled two data sets, one including all SMGs and the other including only SMGs that match SMTs (i.e., a SMG that matches a MMT is not included).</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>CAI for a gene is computed from (1) the codon frequencies of the gene and (2) a codon usage table from a set of reference genes known to be highly expressed, according to the following equation <a href="#pone.0000188-Sharp1">[31]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000188.e001&amp;representation=PNG" class="inline-graphic"></span><br>where w<sub>i</sub> is computed from the Ehum.cut file distributed with EMBOSS <a href="#pone.0000188-Rice1">[33]</a> and n is the number of sense codons (excluding codon families with a single codon, e.g., AUG for methionine and UGG for tryptophan in the standard genetic code). Note that the exponent in equation (1) is simply a weighted average of ln(w). Because the maximum of w is 1, ln(w) will never be greater than 0. Consequently, the exponent will never be greater than 0. Thus, the maximum CAI value is 1.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>It is important to exclude codon families with a single codon. Note that for such codons (e.g., AUG and UGG in the standard genetic code), their corresponding w<sub>i</sub> value will always be 1 regardless of codon usage bias of the gene. If a gene happens to use a high proportion of methionine and tryptophan, then it will have a high CAI value even if the codon usage is not at all biased. The cai program in EMBOSS <a href="#pone.0000188-Rice1">[33]</a> does not exclude codon families with a single codon because the CAI values from that program are the same as those I computed without excluding the AUG and UGG codons. I used DAMBE [34,35, version 4.5.10] which excludes AUG and UGG in computing CAI.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>The reason for using CAI as an index of gene expression instead of taking advantage of the availability of gene expression data is that, in higher eukaryotes such as human, many genes are highly expressed only at specific time and in specific tissues. For this reason, a gene with no detectable expression in a specific study, which typically involves few time points and few tissues, should not be taken as a low expressed gene. However, the availability of the gene expression data does vindicate the use of CAI as a general measure of gene expression <a href="#pone.0000188-Coghlan1">[32]</a>.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>Because CAI is based on codon frequencies of a gene with respect to the codon usage of a reference set of genes known to be highly expressed, short sequences with few codons may produce unreliable CAI values. For this reason, two separate analyses were performed, one with all CDSs and the other excluding CDSs shorter than 300 bp. The two sets of results are almost identical because short CDSs constitute only a small fraction.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>I thank S. Aris-Brosou, M. Carullo, S. Khalouei, P. Ma, J. Mennigen, A. Meyer, Z. Song, H. Xiong, and X. Yao for comments and discussion. An anonymous reviewer and M. Dermitzakis provided many helpful suggestions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: XX. Performed the experiments: XX. Analyzed the data: XX. Contributed reagents/materials/analysis tools: XX. Wrote the paper: XX.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000188-Shine1" id="pone.0000188-Shine1"></a>Shine J, Dalgarno L (1975) Terminal-sequence analysis of bacterial ribosomal RNA. Correlation between the 3′-terminal-polypyrimidine sequence of 16-S RNA and translational specificity of the ribosome. Eur J Biochem  57: 221–230.  <ul class="find" data-citedArticleID="928622" data-doi="10.1111/j.1432-1033.1975.tb02294.x"><li><a href="http://dx.doi.org/10.1111/j.1432-1033.1975.tb02294.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Terminal-sequence+analysis+of+bacterial+ribosomal+RNA.+Correlation+between+the+3%26%23x2032%3B-terminal-polypyrimidine+sequence+of+16-S+RNA+and+translational+specificity+of+the+ribosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Terminal-sequence+analysis+of+bacterial+ribosomal+RNA.+Correlation+between+the+3%26%23x2032%3B-terminal-polypyrimidine+sequence+of+16-S+RNA+and+translational+specificity+of+the+ribosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000188-Shine2" id="pone.0000188-Shine2"></a>Shine J, Dalgarno L (1975) Determinant of cistron specificity in bacterial ribosomes. Nature  254: 34–38.  <ul class="find" data-citedArticleID="928624" data-doi="10.1038/254034a0"><li><a href="http://dx.doi.org/10.1038/254034a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determinant+of+cistron+specificity+in+bacterial+ribosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determinant+of+cistron+specificity+in+bacterial+ribosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000188-Kozak1" id="pone.0000188-Kozak1"></a>Kozak M (1999) Initiation of translation in prokaryotes and eukaryotes. Gene  234: 187–208.  <ul class="find" data-citedArticleID="928588" data-doi="10.1016/s0378-1119(99)00210-3"><li><a href="http://dx.doi.org/10.1016/s0378-1119(99)00210-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initiation+of+translation+in+prokaryotes+and+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initiation+of+translation+in+prokaryotes+and+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000188-Kozak2" id="pone.0000188-Kozak2"></a>Kozak M (1991) Effects of long 5′ leader sequences on initiation by eukaryotic ribosomes in vitro. Gene Expr  1: 117–125.  <ul class="find" data-citedArticleID="928590"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effects+of+long+5%26%23x2032%3B+leader+sequences+on+initiation+by+eukaryotic+ribosomes+in+vitro.&amp;auth=&amp;atitle=Effects+of+long+5%26%23x2032%3B+leader+sequences+on+initiation+by+eukaryotic+ribosomes+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+long+5%26%23x2032%3B+leader+sequences+on+initiation+by+eukaryotic+ribosomes+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+long+5%26%23x2032%3B+leader+sequences+on+initiation+by+eukaryotic+ribosomes+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000188-Kozak3" id="pone.0000188-Kozak3"></a>Kozak M (1981) Possible role of flanking nucleotides in recognition of the AUG initiator codon by eukaryotic ribosomes. Nucleic Acids Res  9: 5233–5252.  <ul class="find" data-citedArticleID="928592" data-doi="10.1093/nar/9.20.5233"><li><a href="http://dx.doi.org/10.1093/nar/9.20.5233" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Possible+role+of+flanking+nucleotides+in+recognition+of+the+AUG+initiator+codon+by+eukaryotic+ribosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Possible+role+of+flanking+nucleotides+in+recognition+of+the+AUG+initiator+codon+by+eukaryotic+ribosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000188-Kozak4" id="pone.0000188-Kozak4"></a>Kozak M (1997) Recognition of AUG and alternative initiator codons is augmented by G in position +4 but is not generally affected by the nucleotides in positions +5 and +6. Embo J  16: 2482–2492.  <ul class="find" data-citedArticleID="928594" data-doi="10.1093/emboj/16.9.2482"><li><a href="http://dx.doi.org/10.1093/emboj/16.9.2482" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+AUG+and+alternative+initiator+codons+is+augmented+by+G+in+position+%2B4+but+is+not+generally+affected+by+the+nucleotides+in+positions+%2B5+and+%2B6." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+AUG+and+alternative+initiator+codons+is+augmented+by+G+in+position+%2B4+but+is+not+generally+affected+by+the+nucleotides+in+positions+%2B5+and+%2B6.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000188-Kozak5" id="pone.0000188-Kozak5"></a>Kozak M (2003) Not every polymorphism close to the AUG codon can be explained by invoking context effects on initiation of translation. Blood  101: 1202–1203; author reply 1203.  <ul class="find" data-citedArticleID="928596" data-doi="10.1182/blood-2002-10-3194"><li><a href="http://dx.doi.org/10.1182/blood-2002-10-3194" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Not+every+polymorphism+close+to+the+AUG+codon+can+be+explained+by+invoking+context+effects+on+initiation+of+translation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Not+every+polymorphism+close+to+the+AUG+codon+can+be+explained+by+invoking+context+effects+on+initiation+of+translation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000188-Kozak6" id="pone.0000188-Kozak6"></a>Kozak M (2001) A progress report on translational control in eukaryotes. Sci STKE  2001: PE1.  <ul class="find" data-citedArticleID="928598" data-doi="10.1126/stke.2001.71.pe1"><li><a href="http://dx.doi.org/10.1126/stke.2001.71.pe1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+progress+report+on+translational+control+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+progress+report+on+translational+control+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000188-Cigan1" id="pone.0000188-Cigan1"></a>Cigan AM, Donahue TF (1987) Sequence and structural features associated with translational initiator regions in yeast–a review. Gene  59: 1–18.  <ul class="find" data-citedArticleID="928558" data-doi="10.1016/0378-1119(87)90261-7"><li><a href="http://dx.doi.org/10.1016/0378-1119(87)90261-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+and+structural+features+associated+with+translational+initiator+regions+in+yeast%26%23x2013%3Ba+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+and+structural+features+associated+with+translational+initiator+regions+in+yeast%26%23x2013%3Ba+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000188-Grunert1" id="pone.0000188-Grunert1"></a>Grunert S, Jackson RJ (1994) The immediate downstream codon strongly influences the efficiency of utilization of eukaryotic translation initiation codons. Embo J  13: 3618–3630.  <ul class="find" data-citedArticleID="928578"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+immediate+downstream+codon+strongly+influences+the+efficiency+of+utilization+of+eukaryotic+translation+initiation+codons.&amp;auth=&amp;atitle=The+immediate+downstream+codon+strongly+influences+the+efficiency+of+utilization+of+eukaryotic+translation+initiation+codons." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+immediate+downstream+codon+strongly+influences+the+efficiency+of+utilization+of+eukaryotic+translation+initiation+codons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+immediate+downstream+codon+strongly+influences+the+efficiency+of+utilization+of+eukaryotic+translation+initiation+codons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000188-Moerschell1" id="pone.0000188-Moerschell1"></a>Moerschell RP, Hosokawa Y, Tsunasawa S, Sherman F (1990) The specificities of yeast methionine aminopeptidase and acetylation of amino-terminal methionine in vivo. Processing of altered iso-1-cytochromes c created by oligonucleotide transformation. J Biol Chem  265: 19638–19643.  <ul class="find" data-citedArticleID="928604"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+specificities+of+yeast+methionine+aminopeptidase+and+acetylation+of+amino-terminal+methionine+in+vivo.+Processing+of+altered+iso-1-cytochromes+c+created+by+oligonucleotide+transformation.&amp;auth=&amp;atitle=The+specificities+of+yeast+methionine+aminopeptidase+and+acetylation+of+amino-terminal+methionine+in+vivo.+Processing+of+altered+iso-1-cytochromes+c+created+by+oligonucleotide+transformation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+specificities+of+yeast+methionine+aminopeptidase+and+acetylation+of+amino-terminal+methionine+in+vivo.+Processing+of+altered+iso-1-cytochromes+c+created+by+oligonucleotide+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+specificities+of+yeast+methionine+aminopeptidase+and+acetylation+of+amino-terminal+methionine+in+vivo.+Processing+of+altered+iso-1-cytochromes+c+created+by+oligonucleotide+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000188-Harkins1" id="pone.0000188-Harkins1"></a>Harkins S, Cornell CT, Whitton JL (2005) Analysis of translational initiation in coxsackievirus B3 suggests an alternative explanation for the high frequency of R+4 in the eukaryotic consensus motif. J Virol  79: 987–996.  <ul class="find" data-citedArticleID="928582" data-doi="10.1128/jvi.79.2.987-996.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.2.987-996.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+translational+initiation+in+coxsackievirus+B3+suggests+an+alternative+explanation+for+the+high+frequency+of+R%2B4+in+the+eukaryotic+consensus+motif." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+translational+initiation+in+coxsackievirus+B3+suggests+an+alternative+explanation+for+the+high+frequency+of+R%2B4+in+the+eukaryotic+consensus+motif.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000188-Farazi1" id="pone.0000188-Farazi1"></a>Farazi TA, Waksman G, Gordon JI (2001) The Biology and Enzymology of Protein N-Myristoylation. J Biol Chem  276: 39501–39504.  <ul class="find" data-citedArticleID="928566" data-doi="10.1074/jbc.r100042200"><li><a href="http://dx.doi.org/10.1074/jbc.r100042200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Biology+and+Enzymology+of+Protein+N-Myristoylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Biology+and+Enzymology+of+Protein+N-Myristoylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000188-Vilas1" id="pone.0000188-Vilas1"></a>Vilas GL, Corvi MM, Plummer GJ, Seime AM, Lambkin GR, et al.  (2006) Posttranslational myristoylation of caspase-activated p21-activated protein kinase 2 (PAK2) potentiates late apoptotic events. Proc Natl Acad Sci U S A  103: 6542–6547.  <ul class="find" data-citedArticleID="928632" data-doi="10.1073/pnas.0600824103"><li><a href="http://dx.doi.org/10.1073/pnas.0600824103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Posttranslational+myristoylation+of+caspase-activated+p21-activated+protein+kinase+2+%28PAK2%29+potentiates+late+apoptotic+events." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Posttranslational+myristoylation+of+caspase-activated+p21-activated+protein+kinase+2+%28PAK2%29+potentiates+late+apoptotic+events.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000188-Sakurai1" id="pone.0000188-Sakurai1"></a>Sakurai N, Utsumi T (2006) Posttranslational N-myristoylation is required for the anti-apoptotic activity of human tGelsolin, the C-terminal caspase cleavage product of human gelsolin. J Biol Chem  281: 14288–14295.  <ul class="find" data-citedArticleID="928616" data-doi="10.1074/jbc.m510338200"><li><a href="http://dx.doi.org/10.1074/jbc.m510338200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Posttranslational+N-myristoylation+is+required+for+the+anti-apoptotic+activity+of+human+tGelsolin%2C+the+C-terminal+caspase+cleavage+product+of+human+gelsolin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Posttranslational+N-myristoylation+is+required+for+the+anti-apoptotic+activity+of+human+tGelsolin%2C+the+C-terminal+caspase+cleavage+product+of+human+gelsolin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000188-Rowe1" id="pone.0000188-Rowe1"></a>Rowe DC, McGettrick AF, Latz E, Monks BG, Gay NJ, et al.  (2006) The myristoylation of TRIF-related adaptor molecule is essential for Toll-like receptor 4 signal transduction. Proc Natl Acad Sci U S A  103: 6299–6304.  <ul class="find" data-citedArticleID="928614" data-doi="10.1073/pnas.0510041103"><li><a href="http://dx.doi.org/10.1073/pnas.0510041103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+myristoylation+of+TRIF-related+adaptor+molecule+is+essential+for+Toll-like+receptor+4+signal+transduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+myristoylation+of+TRIF-related+adaptor+molecule+is+essential+for+Toll-like+receptor+4+signal+transduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000188-deVries1" id="pone.0000188-deVries1"></a>de Vries JS, Andriotis VM, Wu AJ, Rathjen JP (2006) Tomato Pto encodes a functional N-myristoylation motif that is required for signal transduction in Nicotiana benthamiana. Plant J  45: 31–45.  <ul class="find" data-citedArticleID="928644" data-doi="10.1111/j.1365-313x.2005.02590.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2005.02590.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tomato+Pto+encodes+a+functional+N-myristoylation+motif+that+is+required+for+signal+transduction+in+Nicotiana+benthamiana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tomato+Pto+encodes+a+functional+N-myristoylation+motif+that+is+required+for+signal+transduction+in+Nicotiana+benthamiana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000188-RobertSeilaniantz1" id="pone.0000188-RobertSeilaniantz1"></a>Robert-Seilaniantz A, Shan L, Zhou JM, Tang X (2006) The pseudomonas syringae pv. tomato DC3000 Type III effector HopF2 has a putative myristoylation site required for its avirulence and virulence functions. Mol Plant Microbe Interact  19: 130–138.  <ul class="find" data-citedArticleID="928612" data-doi="10.1094/mpmi-19-0130"><li><a href="http://dx.doi.org/10.1094/mpmi-19-0130" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+pseudomonas+syringae+pv.+tomato+DC3000+Type+III+effector+HopF2+has+a+putative+myristoylation+site+required+for+its+avirulence+and+virulence+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+pseudomonas+syringae+pv.+tomato+DC3000+Type+III+effector+HopF2+has+a+putative+myristoylation+site+required+for+its+avirulence+and+virulence+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000188-Bentham1" id="pone.0000188-Bentham1"></a>Bentham M, Mazaleyrat S, Harris M (2006) Role of myristoylation and N-terminal basic residues in membrane association of the human immunodeficiency virus type 1 Nef protein. J Gen Virol  87: 563–571.  <ul class="find" data-citedArticleID="928552" data-doi="10.1099/vir.0.81200-0"><li><a href="http://dx.doi.org/10.1099/vir.0.81200-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+myristoylation+and+N-terminal+basic+residues+in+membrane+association+of+the+human+immunodeficiency+virus+type+1+Nef+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+myristoylation+and+N-terminal+basic+residues+in+membrane+association+of+the+human+immunodeficiency+virus+type+1+Nef+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000188-Breuer1" id="pone.0000188-Breuer1"></a>Breuer S, Gerlach H, Kolaric B, Urbanke C, Opitz N, et al.  (2006) Biochemical indication for myristoylation-dependent conformational changes in HIV-1 Nef. Biochemistry  45: 2339–2349.  <ul class="find" data-citedArticleID="928554" data-doi="10.1021/bi052052c"><li><a href="http://dx.doi.org/10.1021/bi052052c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+indication+for+myristoylation-dependent+conformational+changes+in+HIV-1+Nef." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+indication+for+myristoylation-dependent+conformational+changes+in+HIV-1+Nef.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000188-Provitera1" id="pone.0000188-Provitera1"></a>Provitera P, El-Maghrabi R, Scarlata S (2006) The effect of HIV-1 Gag myristoylation on membrane binding. Biophys Chem  119: 23–32.  <ul class="find" data-citedArticleID="928608" data-doi="10.1016/j.bpc.2005.08.008"><li><a href="http://dx.doi.org/10.1016/j.bpc.2005.08.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+HIV-1+Gag+myristoylation+on+membrane+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+HIV-1+Gag+myristoylation+on+membrane+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000188-Futcher1" id="pone.0000188-Futcher1"></a>Futcher B, Latter GI, Monardo P, McLaughlin CS, Garrels JI (1999) A sampling of the yeast proteome. Mol Cell Biol  19: 7357–7368.  <ul class="find" data-citedArticleID="928572"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+sampling+of+the+yeast+proteome.&amp;auth=&amp;atitle=A+sampling+of+the+yeast+proteome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+sampling+of+the+yeast+proteome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+sampling+of+the+yeast+proteome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000188-Gygi1" id="pone.0000188-Gygi1"></a>Gygi SP, Rochon Y, Franza BR, Aebersold R (1999) Correlation between protein and mRNA abundance in yeast. Mol Cell Biol  19: 1720–1730.  <ul class="find" data-citedArticleID="928580"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Correlation+between+protein+and+mRNA+abundance+in+yeast.&amp;auth=&amp;atitle=Correlation+between+protein+and+mRNA+abundance+in+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+between+protein+and+mRNA+abundance+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+between+protein+and+mRNA+abundance+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000188-Ideker1" id="pone.0000188-Ideker1"></a>Ideker T, Thorsson V, Ranish JA, Christmas R, Buhler J, et al.  (2001) Integrated genomic and proteomic analyses of a systematically perturbed metabolic network. Science  292: 929–934.  <ul class="find" data-citedArticleID="928584" data-doi="10.1126/science.292.5518.929"><li><a href="http://dx.doi.org/10.1126/science.292.5518.929" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrated+genomic+and+proteomic+analyses+of+a+systematically+perturbed+metabolic+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrated+genomic+and+proteomic+analyses+of+a+systematically+perturbed+metabolic+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000188-Griffin1" id="pone.0000188-Griffin1"></a>Griffin TJ, Gygi SP, Ideker T, Rist B, Eng J, et al.  (2002) Complementary profiling of gene expression at the transcriptome and proteome levels in Saccharomyces cerevisiae. Mol Cell Proteomics  1: 323–333.  <ul class="find" data-citedArticleID="928576" data-doi="10.1074/mcp.m200001-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m200001-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complementary+profiling+of+gene+expression+at+the+transcriptome+and+proteome+levels+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complementary+profiling+of+gene+expression+at+the+transcriptome+and+proteome+levels+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000188-Tian1" id="pone.0000188-Tian1"></a>Tian Q, Stepaniants SB, Mao M, Weng L, Feetham MC, et al.  (2004) Integrated Genomic and Proteomic Analyses of Gene Expression in Mammalian Cells. Mol Cell Proteomics  3: 960–969.  <ul class="find" data-citedArticleID="928628" data-doi="10.1074/mcp.m400055-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m400055-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrated+Genomic+and+Proteomic+Analyses+of+Gene+Expression+in+Mammalian+Cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrated+Genomic+and+Proteomic+Analyses+of+Gene+Expression+in+Mammalian+Cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000188-Dinel1" id="pone.0000188-Dinel1"></a>Dinel S, Bolduc C, Belleau P, Boivin A, Yoshioka M, et al.  (2005) Reproducibility, bioinformatic analysis and power of the SAGE method to evaluate changes in transcriptome. Nucleic Acids Res  33: e26.  <ul class="find" data-citedArticleID="928562" data-doi="10.1093/nar/gni025"><li><a href="http://dx.doi.org/10.1093/nar/gni025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reproducibility%2C+bioinformatic+analysis+and+power+of+the+SAGE+method+to+evaluate+changes+in+transcriptome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reproducibility%2C+bioinformatic+analysis+and+power+of+the+SAGE+method+to+evaluate+changes+in+transcriptome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000188-Velculescu1" id="pone.0000188-Velculescu1"></a>Velculescu VE, Madden SL, Zhang L, Lash AE, Yu J, et al.  (1999) Analysis of human transcriptomes. Nat Genet  23: 387–388.  <ul class="find" data-citedArticleID="928630"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+human+transcriptomes.&amp;auth=&amp;atitle=Analysis+of+human+transcriptomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+human+transcriptomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+human+transcriptomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000188-Chen1" id="pone.0000188-Chen1"></a>Chen G, Gharib TG, Huang CC, Taylor JM, Misek DE, et al.  (2002) Discordant protein and mRNA expression in lung adenocarcinomas. Mol Cell Proteomics  1: 304–313.  <ul class="find" data-citedArticleID="928556" data-doi="10.1074/mcp.m200008-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m200008-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discordant+protein+and+mRNA+expression+in+lung+adenocarcinomas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discordant+protein+and+mRNA+expression+in+lung+adenocarcinomas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000188-Zar1" id="pone.0000188-Zar1"></a>Zar JH (1999) Biostatistical Aanalysis. Upper Saddle River, New Jersey: Prentice Hall.   <ul class="find-nolinks"></ul></li><li><span class="label">31.
              </span><a name="pone.0000188-Sharp1" id="pone.0000188-Sharp1"></a>Sharp PM, Li WH (1987) The codon Adaptation Index–a measure of directional synonymous codon usage bias, and its potential applications. Nucleic Acids Res  15: 1281–1295.  <ul class="find" data-citedArticleID="928620" data-doi="10.1093/nar/15.3.1281"><li><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+codon+Adaptation+Index%26%23x2013%3Ba+measure+of+directional+synonymous+codon+usage+bias%2C+and+its+potential+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+codon+Adaptation+Index%26%23x2013%3Ba+measure+of+directional+synonymous+codon+usage+bias%2C+and+its+potential+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000188-Coghlan1" id="pone.0000188-Coghlan1"></a>Coghlan A, Wolfe KH (2000) Relationship of codon bias to mRNA concentration and protein length in Saccharomyces cerevisiae. Yeast  16: 1131–1145.  <ul class="find" data-citedArticleID="928560"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Relationship+of+codon+bias+to+mRNA+concentration+and+protein+length+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Relationship+of+codon+bias+to+mRNA+concentration+and+protein+length+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+of+codon+bias+to+mRNA+concentration+and+protein+length+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+of+codon+bias+to+mRNA+concentration+and+protein+length+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000188-Rice1" id="pone.0000188-Rice1"></a>Rice P, Longden I, Bleasby A (2000) EMBOSS: the European Molecular Biology Open Software Suite. Trends Genet  16: 276–277.  <ul class="find" data-citedArticleID="928610" data-doi="10.1016/s0168-9525(00)02024-2"><li><a href="http://dx.doi.org/10.1016/s0168-9525(00)02024-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EMBOSS%3A+the+European+Molecular+Biology+Open+Software+Suite." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EMBOSS%3A+the+European+Molecular+Biology+Open+Software+Suite.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000188-Xia1" id="pone.0000188-Xia1"></a>Xia X, Xie Z (2001) DAMBE: Software package for data analysis in molecular biology and evolution. Journal of Heredity  92: 371–373.  <ul class="find" data-citedArticleID="928634" data-doi="10.1093/jhered/92.4.371"><li><a href="http://dx.doi.org/10.1093/jhered/92.4.371" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DAMBE%3A+Software+package+for+data+analysis+in+molecular+biology+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DAMBE%3A+Software+package+for+data+analysis+in+molecular+biology+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000188-Xia2" id="pone.0000188-Xia2"></a>Xia X (2001) Data analysis in molecular biology and evolution. Boston: Kluwer Academic Publishers.   <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0000188-Duret1" id="pone.0000188-Duret1"></a>Duret L (2002) Evolution of synonymous codon usage in metazoans. Current Opinion in Genetics &amp; Development  12: 640–649.  <ul class="find" data-citedArticleID="928564" data-doi="10.1016/s0959-437x(02)00353-2"><li><a href="http://dx.doi.org/10.1016/s0959-437x(02)00353-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+synonymous+codon+usage+in+metazoans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+synonymous+codon+usage+in+metazoans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000188-Semon1" id="pone.0000188-Semon1"></a>Semon M, Lobry JR, Duret L (2006) No evidence for tissue-specific adaptation of synonymous codon usage in humans. Mol Biol Evol  23: 523–529.  <ul class="find" data-citedArticleID="928618" data-doi="10.1093/molbev/msj053"><li><a href="http://dx.doi.org/10.1093/molbev/msj053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=No+evidence+for+tissue-specific+adaptation+of+synonymous+codon+usage+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22No+evidence+for+tissue-specific+adaptation+of+synonymous+codon+usage+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000188-Gouy1" id="pone.0000188-Gouy1"></a>Gouy M, Gautier C (1982) Codon usage in bacteria: correlation with gene expressivity. Nucleic Acids Res  10: 7055–7064.  <ul class="find" data-citedArticleID="928574" data-doi="10.1093/nar/10.22.7055"><li><a href="http://dx.doi.org/10.1093/nar/10.22.7055" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+usage+in+bacteria%3A+correlation+with+gene+expressivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+usage+in+bacteria%3A+correlation+with+gene+expressivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000188-Ikemura1" id="pone.0000188-Ikemura1"></a>Ikemura T (1981) Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E coli translational system. J Mol Biol  151: 389–409.  <ul class="find" data-citedArticleID="928586" data-doi="10.1016/0022-2836(81)90003-6"><li><a href="http://dx.doi.org/10.1016/0022-2836(81)90003-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+between+the+abundance+of+Escherichia+coli+transfer+RNAs+and+the+occurrence+of+the+respective+codons+in+its+protein+genes%3A+a+proposal+for+a+synonymous+codon+choice+that+is+optimal+for+the+E+coli+translational+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+between+the+abundance+of+Escherichia+coli+transfer+RNAs+and+the+occurrence+of+the+respective+codons+in+its+protein+genes%3A+a+proposal+for+a+synonymous+codon+choice+that+is+optimal+for+the+E+coli+translational+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000188-Xia3" id="pone.0000188-Xia3"></a>Xia X (2005) Mutation and selection on the anticodon of tRNA genes in vertebrate mitochondrial genomes. Gene  345: 13–20.  <ul class="find" data-citedArticleID="928638" data-doi="10.1016/j.gene.2004.11.019"><li><a href="http://dx.doi.org/10.1016/j.gene.2004.11.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+and+selection+on+the+anticodon+of+tRNA+genes+in+vertebrate+mitochondrial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+and+selection+on+the+anticodon+of+tRNA+genes+in+vertebrate+mitochondrial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000188-Xia4" id="pone.0000188-Xia4"></a>Xia X (1998) How optimized is the translational machinery in Escherichia coli, Salmonella typhimurium and Saccharomyces cerevisiae? Genetics  149: 37–44.  <ul class="find" data-citedArticleID="928640"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=How+optimized+is+the+translational+machinery+in+Escherichia+coli%2C+Salmonella+typhimurium+and+Saccharomyces+cerevisiae%3F&amp;auth=&amp;atitle=How+optimized+is+the+translational+machinery+in+Escherichia+coli%2C+Salmonella+typhimurium+and+Saccharomyces+cerevisiae%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+optimized+is+the+translational+machinery+in+Escherichia+coli%2C+Salmonella+typhimurium+and+Saccharomyces+cerevisiae%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+optimized+is+the+translational+machinery+in+Escherichia+coli%2C+Salmonella+typhimurium+and+Saccharomyces+cerevisiae%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000188-Plotkin1" id="pone.0000188-Plotkin1"></a>Plotkin JB, Robins H, Levine AJ (2004) Tissue-specific codon usage and the expression of human genes. Proc Natl Acad Sci U S A  101: 12588–12591.  <ul class="find" data-citedArticleID="928606" data-doi="10.1073/pnas.0404957101"><li><a href="http://dx.doi.org/10.1073/pnas.0404957101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tissue-specific+codon+usage+and+the+expression+of+human+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tissue-specific+codon+usage+and+the+expression+of+human+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000188-Lowe1" id="pone.0000188-Lowe1"></a>Lowe TM, Eddy SR (1997) tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence. Nucleic Acids Res  25: 955–964.  <ul class="find" data-citedArticleID="928602" data-doi="10.1093/nar/25.5.955"><li><a href="http://dx.doi.org/10.1093/nar/25.5.955" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNAscan-SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNAscan-SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000188-Kozak7" id="pone.0000188-Kozak7"></a>Kozak M (1986) Point mutations define a sequence flanking the AUG initiator codon that modulates translation by eukaryotic ribosomes. Cell  44: 283–292.  <ul class="find" data-citedArticleID="928600" data-doi="10.1016/0092-8674(86)90762-2"><li><a href="http://dx.doi.org/10.1016/0092-8674(86)90762-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Point+mutations+define+a+sequence+flanking+the+AUG+initiator+codon+that+modulates+translation+by+eukaryotic+ribosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Point+mutations+define+a+sequence+flanking+the+AUG+initiator+codon+that+modulates+translation+by+eukaryotic+ribosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000188-Tats1" id="pone.0000188-Tats1"></a>Tats A, Remm M, Tenson T (2006) Highly expressed proteins have an increased frequency of alanine in the second amino acid position. BMC Genomics  7: 28.  <ul class="find" data-citedArticleID="928626"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Highly+expressed+proteins+have+an+increased+frequency+of+alanine+in+the+second+amino+acid+position.&amp;auth=&amp;atitle=Highly+expressed+proteins+have+an+increased+frequency+of+alanine+in+the+second+amino+acid+position." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+expressed+proteins+have+an+increased+frequency+of+alanine+in+the+second+amino+acid+position." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+expressed+proteins+have+an+increased+frequency+of+alanine+in+the+second+amino+acid+position.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000188-Flinta1" id="pone.0000188-Flinta1"></a>Flinta C, Persson B, Jornvall H, von Heijne G (1986) Sequence determinants of cytosolic N-terminal protein processing. Eur J Biochem  154: 193–196.  <ul class="find" data-citedArticleID="928568" data-doi="10.1111/j.1432-1033.1986.tb09378.x"><li><a href="http://dx.doi.org/10.1111/j.1432-1033.1986.tb09378.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+determinants+of+cytosolic+N-terminal+protein+processing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+determinants+of+cytosolic+N-terminal+protein+processing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000188-Frottin1" id="pone.0000188-Frottin1"></a>Frottin F, Martinez A, Peynot P, Mitra S, Holz RC, et al.  (2006) The Proteomics of N-terminal Methionine Cleavage. Mol Cell Proteomics  5: 2336–2349.  <ul class="find" data-citedArticleID="928570" data-doi="10.1074/mcp.m600225-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m600225-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Proteomics+of+N-terminal+Methionine+Cleavage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Proteomics+of+N-terminal+Methionine+Cleavage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.XML" value="96662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.PDF" value="108368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.g001.PNG_L" value="99224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.g001.PNG_M" value="47787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.g001.PNG_S" value="2125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.g001.TIF" value="263300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.g001.PNG_I" value="20200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.t001.PNG_L" value="46489"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.t001.PNG_M" value="49243"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.t001.PNG_S" value="2810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.t001.TIF" value="233524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.t001.PNG_I" value="21338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.e001.PNG" value="10703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000188.e001.TIF" value="20496"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000188&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000188" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000188&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000188&volume=&issue=&title=The +4G Site in Kozak Consensus Is Not Related to the Efficiency of Translation Initiation&author_name=Xuhua%20Xia&start_page=1&end_page=5" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000188" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000188" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000188" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000188&amp;t=The%20%2B4G%20Site%20in%20Kozak%20Consensus%20Is%20Not%20Related%20to%20the%20Efficiency%20of%20Translation%20Initiation" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000188&title=The%20%2B4G%20Site%20in%20Kozak%20Consensus%20Is%20Not%20Related%20to%20the%20Efficiency%20of%20Translation%20Initiation&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000188&amp;title=The%20%2B4G%20Site%20in%20Kozak%20Consensus%20Is%20Not%20Related%20to%20the%20Efficiency%20of%20Translation%20Initiation" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000188" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000188&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The +4G Site in Kozak Consensus Is Not Related to the Efficiency of Translation Initiation';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000188';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20%2B4G%20Site%20in%20Kozak%20Consensus%20Is%20Not%20Related%20to%20the%20Efficiency%20of%20Translation%20Initiation http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000188" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000188" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alanine%22" title="Search for articles in the subject area:'Alanine'"><div class="flagText">Alanine</div></a>
              <div data-categoryid="40577" data-articleid="23214"
                   data-categoryname="Alanine"
                   class="flagImage" title="Flag 'Alanine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="23214"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23214"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glycine%22" title="Search for articles in the subject area:'Glycine'"><div class="flagText">Glycine</div></a>
              <div data-categoryid="40841" data-articleid="23214"
                   data-categoryname="Glycine"
                   class="flagImage" title="Flag 'Glycine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Myristoylation%22" title="Search for articles in the subject area:'Myristoylation'"><div class="flagText">Myristoylation</div></a>
              <div data-categoryid="28827" data-articleid="23214"
                   data-categoryname="Myristoylation"
                   class="flagImage" title="Flag 'Myristoylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Prokaryotic+cells%22" title="Search for articles in the subject area:'Prokaryotic cells'"><div class="flagText">Prokaryotic cells</div></a>
              <div data-categoryid="17761" data-articleid="23214"
                   data-categoryname="Prokaryotic cells"
                   class="flagImage" title="Flag 'Prokaryotic cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Serial+analysis+of+gene+expression%22" title="Search for articles in the subject area:'Serial analysis of gene expression'"><div class="flagText">Serial analysis of gene expression</div></a>
              <div data-categoryid="48515" data-articleid="23214"
                   data-categoryname="Serial analysis of gene expression"
                   class="flagImage" title="Flag 'Serial analysis of gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Translation+initiation%22" title="Search for articles in the subject area:'Translation initiation'"><div class="flagText">Translation initiation</div></a>
              <div data-categoryid="46325" data-articleid="23214"
                   data-categoryname="Translation initiation"
                   class="flagImage" title="Flag 'Translation initiation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4435'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1478'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3419&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=20923">N-end rule?</a><br>Posted by ralmeida</p>
      <p><a href="/annotation/listThread.action?root=5171">Missing work by JW Stewart et al. 1971</a><br>Posted by xxia</p>
      <p><a href="/annotation/listThread.action?root=5053">Error in GAN entry</a><br>Posted by xxia</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
