

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: An Assessment of the Role of DNA Adenine Methyltransferase on Gene Expression Regulation in E coli</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000273"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000273" />

    <meta name="citation_title" content="An Assessment of the Role of DNA Adenine Methyltransferase on Gene Expression Regulation in E coli"/>
    <meta itemprop="name" content="An Assessment of the Role of DNA Adenine Methyltransferase on Gene Expression Regulation in E coli"/>

      <meta name="citation_author" content="Aswin Sai Narain Seshasayee"/>
            <meta name="citation_author_institution" content="Genomics and Regulatory Systems Group, EMBL-European Bioinformatics Institute, Wellcome Trust Genome Campus, Cambridge, United Kingdom"/>

    <meta name="citation_date" content="2007/3/7"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000273.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e273"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=N6-methyl-adenine: an epigenetic signal for DNA-protein interactions.; citation_author=D Wion; citation_author=J Casadesus; citation_journal_title=Nat Rev Microbiol; citation_volume=4; citation_number=1; citation_pages=183-192; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Uneven distribution of GATC motifs in the Escherichia coli chromosome, its plasmids and its phages.; citation_author=A Henaut; citation_author=T Rouxel; citation_author=A Gleizes; citation_author=I Moszer; citation_author=A Danchin; citation_journal_title=J Mol Biol; citation_volume=257; citation_number=2; citation_pages=574-585; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Analysis of global gene expression and double-strand-break formation in DNA adenine methyltransferase-and mismatch repair-deficient Escherichia coli.; citation_author=JL Robbins-Manke; citation_author=ZZ Zdraveski; citation_author=M Marinus; citation_author=JM Essigmann; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=3; citation_pages=7027-7037; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transcriptome analysis of Escherichia coli using high-density oligonucleotide probe arrays.; citation_author=B Tjaden; citation_author=RM Saxena; citation_author=S Stolyar; citation_author=DR Haynor; citation_author=E Kolker; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=4; citation_pages=3732-3738; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=ASAP, a systematic annotation package for community analysis of genomes.; citation_author=JD Glasner; citation_author=P Liss; citation_author=G Plunkett; citation_author=A Darling; citation_author=T Prasad; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=5; citation_pages=147-151; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Integrating high-throughput and computational data elucidates bacterial networks.; citation_author=MW Covert; citation_author=EM Knight; citation_author=JL Reed; citation_author=MJ Herrgard; citation_author=BO Palsson; citation_journal_title=Nature; citation_volume=429; citation_number=6; citation_pages=92-96; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function.; citation_author=Y Kang; citation_author=KD Weber; citation_author=Y Qiu; citation_author=PJ Kiley; citation_author=FR Blattner; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=7; citation_pages=1135-1160; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RegulonDB (version 5.0): Escherichia coli K-12 transcriptional regulatory network, operon organization, and growth conditions.; citation_author=H Salgado; citation_author=S Gama-Castro; citation_author=M Peralta-Gil; citation_author=E Diaz-Peredo; citation_author=F Sanchez-Solano; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=8; citation_pages=D394-397; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A comparison of normalization methods for high density oligonucleotide array data based on variance and bias.; citation_author=BM Bolstad; citation_author=RA Irizarry; citation_author=M Astrand; citation_author=TP Speed; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=9; citation_pages=185-193; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Linear models and empirical bayes methods for assessing differential expression in microarray experiments.; citation_author=GK Smyth; citation_journal_title=Stat Appl Genet Mol Biol; citation_volume=3; citation_number=10; citation_pages=Article3; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bioconductor: open software development for computational biology and bioinformatics.; citation_author=RC Gentleman; citation_author=VJ Carey; citation_author=DM Bates; citation_author=B Bolstad; citation_author=M Dettling; citation_journal_title=Genome Biol; citation_volume=5; citation_number=11; citation_pages=R80; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=FatiGO: a web tool for finding significant associations of Gene Ontology terms with groups of genes.; citation_author=F Al-Shahrour; citation_author=R Diaz-Uriarte; citation_author=J Dopazo; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=12; citation_pages=578-580; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=EMBOSS: the European Molecular Biology Open Software Suite.; citation_author=P Rice; citation_author=I Longden; citation_author=A Bleasby; citation_journal_title=Trends Genet; citation_volume=16; citation_number=13; citation_pages=276-277; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of deoxyadenosine methyltransferase-mediated control of gene expression in Escherichia coli.; citation_author=T Oshima; citation_author=C Wada; citation_author=Y Kawagoe; citation_author=T Ara; citation_author=M Maeda; citation_journal_title=Mol Microbiol; citation_volume=45; citation_number=14; citation_pages=673-695; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Role of SeqA and Dam in Escherichia coli gene expression: a global/microarray analysis.; citation_author=A Lobner-Olesen; citation_author=MG Marinus; citation_author=FG Hansen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=15; citation_pages=4672-4677; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identifying global regulators in transcriptional regulatory networks in bacteria.; citation_author=A Martinez-Antonio; citation_author=J Collado-Vides; citation_journal_title=Curr Opin Microbiol; citation_volume=6; citation_number=16; citation_pages=482-489; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Differential Expression with the Bioconductor Package.; citation_author=A von Heydebreck; citation_author=W Huber; citation_author=R Gentleman; citation_number=17; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=RecA-dependent mutants in Escherichia coli reveal strategies to avoid chromosomal fragmentation.; citation_author=EA Kouzminova; citation_author=E Rotman; citation_author=L Macomber; citation_author=J Zhang; citation_author=A Kuzminov; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=18; citation_pages=16262-16267; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Characterization of the smtA gene encoding an S-adenosylmethionine-dependent methyltransferase of Escherichia coli.; citation_author=K Yamanaka; citation_author=T Ogura; citation_author=H Niki; citation_author=S Hiraga; citation_journal_title=FEMS Microbiol Lett; citation_volume=133; citation_number=19; citation_pages=59-63; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Promoter addresses: revelations from oligonucleotide profiling applied to the Escherichia coli genome.; citation_author=K Sivaraman; citation_author=AS Seshasayee; citation_author=K Swaminathan; citation_author=G Muthukumaran; citation_author=G Pennathur; citation_journal_title=Theor Biol Med Model; citation_volume=2; citation_number=20; citation_pages=20; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The solution structures of the trp repressor-operator DNA complex.; citation_author=H Zhang; citation_author=D Zhao; citation_author=M Revington; citation_author=W Lee; citation_author=X Jia; citation_journal_title=J Mol Biol; citation_volume=238; citation_number=21; citation_pages=592-614; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the met repressor-operator complex at 2.8 A resolution reveals DNA recognition by beta-strands.; citation_author=WS Somers; citation_author=SE Phillips; citation_journal_title=Nature; citation_volume=359; citation_number=22; citation_pages=387-393; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The complete genome sequence of Escherichia coli K-12.; citation_author=FR Blattner; citation_author=G Plunkett; citation_author=CA Bloch; citation_author=NT Perna; citation_author=V Burland; citation_journal_title=Science; citation_volume=277; citation_number=23; citation_pages=1453-1474; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Novel growth rate control of dam gene expression in Escherichia coli.; citation_author=LJ Rasmussen; citation_author=MG Marinus; citation_author=A Lobner-Olesen; citation_journal_title=Mol Microbiol; citation_volume=12; citation_number=24; citation_pages=631-638; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A switch from high-fidelity to error-prone DNA double-strand break repair underlies stress-induced mutation.; citation_author=RG Ponder; citation_author=NC Fonville; citation_author=SM Rosenberg; citation_journal_title=Mol Cell; citation_volume=19; citation_number=25; citation_pages=791-804; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Diiferential expression with the Bioconductor project.; citation_author=A von Heydebreck; citation_author=W Huber; citation_author=RC Gentleman; citation_number=26; citation_pages=7; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000273" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="An Assessment of the Role of DNA Adenine Methyltransferase on Gene Expression Regulation in E coli"/>
    <meta name="twitter:description" content="N6-Adenine methylation is an important epigenetic signal, which regulates various processes, such as DNA replication and repair and transcription. In &gamma;-proteobacteria, Dam is a stand-alone enzyme that methylates GATC sites, which are non-randomly distributed in the genome. Some of these overlap with transcription factor binding sites. This work describes a global computational analysis of a published Dam knockout microarray alongside other publicly available data to throw insights into the extent to which Dam regulates transcription by interfering with protein binding. The results indicate that DNA methylation by DAM may not globally affect gene transcription by physically blocking access of transcription factors to binding sites. Down-regulation of Dam during stationary phase correlates with the activity of TFs whose binding sites are enriched for GATC sites."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000273.g002"/>

  <meta property="og:title" content="An Assessment of the Role of DNA Adenine Methyltransferase on Gene Expression Regulation in E coli" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000273" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4585'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=950'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=186&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000273">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000273" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000273&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    An Assessment of the Role of DNA Adenine Methyltransferase on Gene Expression Regulation in <i>E coli</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Aswin Sai Narain Seshasayee
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:aswin@ebi.ac.uk">aswin@ebi.ac.uk</a>, <a href="mailto:asns3@cam.ac.uk">asns3@cam.ac.uk</a></p>

                <p>Affiliation:
                  Genomics and Regulatory Systems Group, EMBL-European Bioinformatics Institute, Wellcome Trust Genome Campus, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 07, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000273</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000273" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000273" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000273" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000273" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000273-g001" data-doi="info:doi/10.1371/journal.pone.0000273" data-uri="info:doi/10.1371/journal.pone.0000273.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000273.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000273-g002" data-doi="info:doi/10.1371/journal.pone.0000273" data-uri="info:doi/10.1371/journal.pone.0000273.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000273.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000273">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000273" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>N6-Adenine methylation is an important epigenetic signal, which regulates various processes, such as DNA replication and repair and transcription. In γ-proteobacteria, Dam is a stand-alone enzyme that methylates GATC sites, which are non-randomly distributed in the genome. Some of these overlap with transcription factor binding sites. This work describes a global computational analysis of a published Dam knockout microarray alongside other publicly available data to throw insights into the extent to which Dam regulates transcription by interfering with protein binding. The results indicate that DNA methylation by DAM may not globally affect gene transcription by physically blocking access of transcription factors to binding sites. Down-regulation of Dam during stationary phase correlates with the activity of TFs whose binding sites are enriched for GATC sites.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Seshasayee ASN (2007) An Assessment of the Role of DNA Adenine Methyltransferase on Gene Expression Regulation in <em>E coli</em>. PLoS ONE 2(3):
          e273.
            doi:10.1371/journal.pone.0000273</p><p><strong>Academic Editor: </strong>Winston Hide, University of the Western Cape, South Africa</p><p><strong>Received:</strong> November 21, 2006; <strong>Accepted:</strong> February 14, 2007; <strong>Published:</strong> March 7, 2007</p><p><strong>Copyright:</strong> © 2007 Aswin Sai Narain Seshasayee. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This is an entirely independent project carried out by me without the aid of any funding agency. My studentship at the EMBL-EBI is partly covered by the Cambridge Commonwealth Trust.</p><p><strong>Competing interests:</strong> The author has declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Dam is an N6-Adenine methyltransferase, which methylates GATC sites soon after replication. Methylation is a bacterial version of an immune response to phages. It has been described as a signal that influences DNA-protein interactions <a href="#pone.0000273-Wion1">[1]</a>. GATC sites have been shown to overlap with the binding sites of global transcriptional regulators, CRP and FNR, and thus influence their activity <a href="#pone.0000273-Henaut1">[2]</a>. This would imply that a knockout of Dam should have drastic effects on gene expression.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>This work analyses a recently published microarray data of a Dam mutant in order to assess the effect this has on transcription regulation. Further, GATC-containing TF binding sites are analysed in order to correlate any gene expression changes to Dam binding. Finally, a hypothesis concerning the balance between Dam binding and transcription regulation by the global factor CRP is presented.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Datasets</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Microarray dataset for Dam mutant was obtained from Robbins-Manke et al <a href="#pone.0000273-RobbinsManke1">[3]</a>. One set of stationary phase microarray data was obtained from Tjaden et al. <a href="#pone.0000273-Tjaden1">[4]</a> and the other was downloaded from the ASAP database <a href="#pone.0000273-Glasner1">[5]</a> in November 2005. FNR knockout microarray data on Affymetrix platform, which is used here as an example to assess the effect of the knockout of a global transcriptional regulator on gene expression, was obtained from Covert et al. and Kang et al. as raw data and processed as below <a href="#pone.0000273-Covert1">[6]</a>, <a href="#pone.0000273-Kang1">[7]</a>. Literature derived datasets for (1) transcription factor binding sites (TFBS) (2) sigma factor binding sites/promoters (SFBS) and (3) transcription units were obtained from RegulonDB 5.0 <a href="#pone.0000273-Salgado1">[8]</a>. COG functional category assignments for E coli were obtained from GenBank.</p>


<h4>Microarray data analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The raw CEL files were processed using the RMA procedure and differentially expressed genes were identified using LIMMA. RMA does not require a baseline array for normalization and is based on achieving quantile-quantile plots that are along the unit vector of the diagonal <a href="#pone.0000273-Bolstad1">[9]</a>. LIMMA uses a moderated t-test approach to identify differentially expressed genes <a href="#pone.0000273-Smyth1">[10]</a>. For all microarray data except the <em>dam</em> mutant, differential expression was defined by a q-value of 0.05 following FDR multiple testing. For the <em>dam</em> mutant, the cutoff was 0.01 without multiple testing The reasoning is explained in context in the results section. All these calculations were carried out using Bioconductor <a href="#pone.0000273-Gentleman1">[11]</a>.</p>


<h4>Functional category enrichment</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Enrichment of specific functional categories among differentially expressed genes was carried out using the an F-test followed by FDR as used in FatiGO to identified enriched functional categories among differentially expressed genes <a href="#pone.0000273-AlShahrour1">[12]</a>. This was done in R.</p>


<h4>Permutation tests</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Permutation tests were used for certain analyses as described in the results. For this the pairings of TFs/Sigma factors to binding sequences were randomly shuffled around.</p>


<h4>Tetranucleotide profiling</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Tetranucleotides in coding sequences and TFBS were counted using the compseq program in the EMBOSS package <a href="#pone.0000273-Rice1">[13]</a>. For the F-test, the FatiGO script implemented in R was used. An alternative scoring scheme was also used to test for enrichment. The formula for enrichment in this approach is:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000273.e001&amp;representation=PNG" class="inline-graphic"></span><br>where N<sub>T,TFBS</sub> is the number of occurrences of tetranucleotide T within TFBS, N<sub>∑T,TFBS</sub> is the sum of the counts of all tetranucleotides within TFBS, ∑N<sub>T,CODING</sub> is the number of occurrences of tetranucleotide T within coding regions and N<sub>∑T,CODING</sub> is the sum of the counts of all tetranucleotides within coding regions. A two-fold enrichment would correspond to a score of~0.7.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Dam mutant does not result in global changes in transcription</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>In recent years, three different microarray studies have analysed gene expression changes in <em>dam</em> E coli <a href="#pone.0000273-RobbinsManke1">[3]</a>, <a href="#pone.0000273-Oshima1">[14]</a>, <a href="#pone.0000273-LobnerOlesen1">[15]</a>. The most recent of these <a href="#pone.0000273-RobbinsManke1">[3]</a>, for which the raw data is available in GEO, describes an overall increase in expression of about 200 genes in <em>Dam</em> using Affymetrix GeneChip arrays. For the current study, this raw data was reanalyzed. The data was normalised using RMA as in the above study. However, instead of the ANOVA analysis used in the above study, the moderated t-test approach of LIMMA implemented in Bioconductor <a href="#pone.0000273-Gentleman1">[11]</a> was used to identify differentially expressed genes. Following p-value adjustment with multiple testing, it was found that none of the genes showed a statistically significant change in gene expression at an FDR of 0.05. This is very unlike an FNR–one of seven global regulators defined by Martinez-Antonio and Collado-Vides <a href="#pone.0000273-MartinezAntonio1">[16]</a>-knockout under anaerobic conditions <a href="#pone.0000273-Covert1">[6]</a>, <a href="#pone.0000273-Kang1">[7]</a>, which on normalisation with RMA followed by detection of differential expression with LIMMA and multiple testing with FDR results in 340–360 differentially expressed genes. While such a multiple correction approach is effective in normalizing for dependencies across genes, it can lead to a loss of sensitivity <a href="#pone.0000273-vonHeydebreck1">[17]</a>. Hence, a more conservative approach of a raw p-value cut-off of 0.01 and a log (base 2) change of 0.7 (2-fold change) was used on the <em>Dam</em> dataset. 109 genes were differentially expressed (<a href="#pone.0000273.s001">Table S1</a>). This included RecA and LexA confirming the activation of DNA repair mechanisms. An over-representation of genes involved in translation was also observed (F-test as used in FatiGO <a href="#pone.0000273-AlShahrour1">[12]</a>, FDR q-value: 10<sup>−15</sup>). This is as observed in the original study <a href="#pone.0000273-RobbinsManke1">[3]</a>. However, the present analysis shows that no other functional category is enriched. The lack of significance of differential expression after multiple-correction might imply that these changes are subtle. The gene expression changes may be restrained due to the activity of a relatively less characterized methyltransferase SmtA <a href="#pone.0000273-Kouzminova1">[18]</a>, <a href="#pone.0000273-Yamanaka1">[19]</a>. It may also arise because of variation in the extent of double strand breaks in the population as reported <a href="#pone.0000273-RobbinsManke1">[3]</a>.</p>


<h4>Dam binding sites and gene expression changes are not correlated</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Yet another dataset that was used in this study is experimentally verified transcription factor binding site (TFBS) and promoter sequence (PS) data downloaded from RegulonDB 5.0 <a href="#pone.0000273-Salgado1">[8]</a>. A list of experimentally verified transcription units was used in conjunction with the above data in order to identify promoters of genes that are differentially expressed, which also contain the GATC motif. The 109 differentially expressed genes fell in 65 different transcription units. Of these 38 had an experimentally verified sigma factor binding sequence (SFBS) and 25 had at least one known TFBS.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>If the change in expression levels were due to altered TF binding to DNA in the <em>dam</em> strain, then we would expect to see an over-representation of GATC containing SFBS and TFBS in the list of differentially expressed transcription units. However, this was not the case (<a href="#pone.0000273.s002">Tables S2</a> and <a href="#pone.0000273.s003">S3</a>). Only 3 differentially expressed TFBS and SFBS contained GATC sites. This was just random (Z-scores of-1.5 and 0.005 for SFBS and TFBS respectively; control: 1000 randomly shuffled gene-SFBS/TFBS pairs). In fact a lack of such correlation was reported by Robbins-Manke et al. as well <a href="#pone.0000273-RobbinsManke1">[3]</a>. This implies that any change in expression levels, despite being subtle, observed in the microarray data cannot be attributed to the direct influence of methylation on transcription factor or sigma factor binding to the DNA. These transcriptional changes might be in response to damage to the DNA that is caused by reduced methylation or due to hitherto unexplained indirect effects.</p>


<h4>Tetranucleotide profiling for GATC</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>In an earlier work, over-representation of specific words in gene-upstream regions was described as a measure of the regulatory potential of such motifs and validated using comparison with known structures of DNA-protein complexes <a href="#pone.0000273-Sivaraman1">[20]</a>. Hence, as further test for the role of GATC as a regulatory motif at the transcriptional level, tetranucleotide profiling analysis as described in the above publication was carried out. Instead of an arbitrary cut-off, as was used in the former work, the F-test-FDR test was used to test for enrichment. Further, for this test, enrichment of tetranucleotides within the experimentally validated TFBS in comparison to the coding sequences was tested. It was found that out of 256 possible tetranucleotides, 91 were statistically enriched (q-value&lt;0.05) within TFBS. This includes CTAG, which is the core binding sequence for trp <a href="#pone.0000273-Zhang1">[21]</a> and the met <a href="#pone.0000273-Somers1">[22]</a> repressors and is known to be highly restrained in the genome <a href="#pone.0000273-Blattner1">[23]</a>. However, despite the fact that over 35%of all tetranucleotides are enriched in TFBS, GATC is not, thus adding further to doubts on its role as a transcriptional regulatory motif. This is true even when an alternative approach to identifying enriched tetranucleotides is used. In this approach a score was computed for each tetranucleotide so that it would define whether the word is enriched within TFBS or not. At a two-fold enrichment cut-off, 53 tetranucleotides were obtained. This list was a subset of the list obtained using the F-test and thus GATC is found to be absent here as well. These lists are provided as tables <a href="#pone.0000273.s004">S4A</a> and <a href="#pone.0000273.s005">S4B</a>.</p>


<h4>Dam, CRP, Sigma38 and stationary phase</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>In the next step of the study, TFs and Sigma factors whose binding sites were significantly enriched for GATC were identified (<a href="#pone-0000273-g001">Fig. 1</a>). Among TFs for which more than 10 sites contained a GATC sequence, only CRP and FNR showed significant enrichments (tested using 1000 shuffling of TF to site mapping). CRP (Z-score: 8) showed a much greater significance than FNR (Z-score: 2.5). In addition, a survey of Sigma factor binding sequences for GATC using the same permutation tests as used for TFBS shows that only the stationary phase/stress response sigma factor, Sigma 38 or rpoS, is enriched for GATC, though only slightly (Z-score: 2.5).</p>
<div class="figure" id="pone-0000273-g001"><div class="img"><a name="pone-0000273-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000273.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000273" data-uri="info:doi/10.1371/journal.pone.0000273.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000273.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000273.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000273.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000273.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000273.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000273.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000273.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000273.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000273.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>Plots showing the propensities and Z-score for various (A) transcription factor binding sites and (B) sigma factor binding sites to contain GATC.</p>
<span>doi:10.1371/journal.pone.0000273.g001</span></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>CRP, which is activated by cAMP signalling in response to glucose starvation, can be expected to be active during the stationary phase of growth in minimal medium and would therefore not be active under the conditions in which the above microarray data was obtained. Hence it is reasonable that CRP targets with GATC sites do not significantly change in expression levels in a Dam mutant grown in rich media. The same is applicable to SigmaS as well. However, the question is: how does CRP access its binding sites even during stationary phase when the sites are methylated? Two different publicly available Affymetrix microarray datasets for stationary phase E coli cells <a href="#pone.0000273-Tjaden1">[4]</a>, <a href="#pone.0000273-Glasner1">[5]</a> were mined (using comparisons with log phase arrays from the same experiment) for genes that were differentially expressed (moderated t-test from LIMMA and FDR&lt;0.05) during the stationary phase (<a href="#pone.0000273.s006">Table S5</a>). It could be seen that Dam is consistently down regulated in the stationary phase (FDR of 10<sup>−8</sup> and 10<sup>−5</sup> in the two contrasts). This is consistent with results from a ten year old small-scale experiment showing that Dam levels are dependent on growth rate and that a ten-fold decrease in growth rate results in a four-fold fall in Dam levels <a href="#pone.0000273-Rasmussen1">[24]</a>. This would result in reduced methylation during stationary phase, allowing CRP to bind its targets in newly divided cells. This could also be one reason why double strand breaks are induced during stationary phase <a href="#pone.0000273-Ponder1">[25]</a>. This implies that Dam methylation does not really interfere with TF binding under conditions in which the TF might be expected to be active. Further the observation that only Sigma38 binding sites among those for all sigma factors show enrichment for GATC is further evidence to the above. This model is illustrated in <a href="#pone-0000273-g002">Fig. 2</a>.</p>
<div class="figure" id="pone-0000273-g002"><div class="img"><a name="pone-0000273-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000273.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000273" data-uri="info:doi/10.1371/journal.pone.0000273.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000273.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000273.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000273.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000273.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000273.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000273.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000273.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000273.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000273.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec3.sec4.fig2.caption1.p1" name="article1.body1.sec3.sec4.fig2.caption1.p1"></a><p>A schematic representation of the interplay between the growth phase, Dam-mediated methylation and transcriptional effects of CRP and Sigma38. This model is a hypothesis shows that Dam does not directly inhibit TF/Sigma binding and its downregulation in stationary phase correlates with the activation of CRP and Sigma38 whose binding sites which are enriched for GATC.</p>
<span>doi:10.1371/journal.pone.0000273.g002</span></div>

<h4>Conclusion</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Despite the description of isolated cases where DNA methylation plays an important role in transcriptional regulation <a href="#pone.0000273-Wion1">[1]</a>, it may not be a global player. Dam is down regulated in the stationary phase, which correlates with the enrichment of GATC in binding sites for CRP and Sigma 38, though the functional significance of the enrichment seen with FNR is not clear. <a href="#pone.0000273-vonHeydebreck2">[26]</a></p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000273"></div><a name="pone.0000273.s001" id="pone.0000273.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000273.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Genes differentially expressed in delta-dam mutant in comparison to wt. This is a reanalysis of data published by Robbins-Manke et al.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000273.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.01 MB TXT)</p>
<a name="pone.0000273.s002" id="pone.0000273.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000273.s002">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Experimentally verified transcription units containing the genes that are differentially expressed in a dam mutant. Sequences shown are sigma factor binding sites</p>
<p class="siDoi">doi:10.1371/journal.pone.0000273.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.00 MB TXT)</p>
<a name="pone.0000273.s003" id="pone.0000273.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000273.s003">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Experimentally verified transcription units containing the genes that are differentially expressed in a dam mutant. Sequences shown are transcription factor binding sites</p>
<p class="siDoi">doi:10.1371/journal.pone.0000273.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.00 MB TXT)</p>
<a name="pone.0000273.s004" id="pone.0000273.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000273.s004">Table S4A. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Tetranucleotides enriched in TFBS against coding regions as seen from F-test FDR</p>
<p class="siDoi">doi:10.1371/journal.pone.0000273.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.00 MB TXT)</p>
<a name="pone.0000273.s005" id="pone.0000273.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000273.s005">Table S4B. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Tetranucleotides enriched in TFBS against coding regions as seen from propensity score</p>
<p class="siDoi">doi:10.1371/journal.pone.0000273.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.00 MB TXT)</p>
<a name="pone.0000273.s006" id="pone.0000273.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000273.s006">Table S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Genes differentially expressed in two independent stationary phase vs. log phase contrasts</p>
<p class="siDoi">doi:10.1371/journal.pone.0000273.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.05 MB TXT)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>I thank Inlaks Foundation, India and the Cambridge Commonwealth Trust for scholarships.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AS. Performed the experiments: AS. Analyzed the data: AS. Wrote the paper: AS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000273-Wion1" id="pone.0000273-Wion1"></a>Wion D, Casadesus J (2006) N6-methyl-adenine: an epigenetic signal for DNA-protein interactions. Nat Rev Microbiol  4: 183–192.  <ul class="find" data-citedArticleID="935678" data-doi="10.1038/nrmicro1350"><li><a href="http://dx.doi.org/10.1038/nrmicro1350" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=N6-methyl-adenine%3A+an+epigenetic+signal+for+DNA-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22N6-methyl-adenine%3A+an+epigenetic+signal+for+DNA-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000273-Henaut1" id="pone.0000273-Henaut1"></a>Henaut A, Rouxel T, Gleizes A, Moszer I, Danchin A (1996) Uneven distribution of GATC motifs in the Escherichia coli chromosome, its plasmids and its phages. J Mol Biol  257: 574–585.  <ul class="find" data-citedArticleID="935648" data-doi="10.1006/jmbi.1996.0186"><li><a href="http://dx.doi.org/10.1006/jmbi.1996.0186" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uneven+distribution+of+GATC+motifs+in+the+Escherichia+coli+chromosome%2C+its+plasmids+and+its+phages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uneven+distribution+of+GATC+motifs+in+the+Escherichia+coli+chromosome%2C+its+plasmids+and+its+phages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000273-RobbinsManke1" id="pone.0000273-RobbinsManke1"></a>Robbins-Manke JL, Zdraveski ZZ, Marinus M, Essigmann JM (2005) Analysis of global gene expression and double-strand-break formation in DNA adenine methyltransferase-and mismatch repair-deficient Escherichia coli. J Bacteriol  187: 7027–7037.  <ul class="find" data-citedArticleID="935666" data-doi="10.1128/jb.187.20.7027-7037.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.20.7027-7037.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+global+gene+expression+and+double-strand-break+formation+in+DNA+adenine+methyltransferase-and+mismatch+repair-deficient+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+global+gene+expression+and+double-strand-break+formation+in+DNA+adenine+methyltransferase-and+mismatch+repair-deficient+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000273-Tjaden1" id="pone.0000273-Tjaden1"></a>Tjaden B, Saxena RM, Stolyar S, Haynor DR, Kolker E, et al.  (2002) Transcriptome analysis of Escherichia coli using high-density oligonucleotide probe arrays. Nucleic Acids Res  30: 3732–3738.  <ul class="find" data-citedArticleID="935676" data-doi="10.1093/nar/gkf505"><li><a href="http://dx.doi.org/10.1093/nar/gkf505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+analysis+of+Escherichia+coli+using+high-density+oligonucleotide+probe+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+analysis+of+Escherichia+coli+using+high-density+oligonucleotide+probe+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000273-Glasner1" id="pone.0000273-Glasner1"></a>Glasner JD, Liss P, Plunkett G 3rd, Darling A, Prasad T, et al.  (2003) ASAP, a systematic annotation package for community analysis of genomes. Nucleic Acids Res  31: 147–151.  <ul class="find" data-citedArticleID="935646" data-doi="10.1093/nar/gkg125"><li><a href="http://dx.doi.org/10.1093/nar/gkg125" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ASAP%2C+a+systematic+annotation+package+for+community+analysis+of+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ASAP%2C+a+systematic+annotation+package+for+community+analysis+of+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000273-Covert1" id="pone.0000273-Covert1"></a>Covert MW, Knight EM, Reed JL, Herrgard MJ, Palsson BO (2004) Integrating high-throughput and computational data elucidates bacterial networks. Nature  429: 92–96.  <ul class="find" data-citedArticleID="935642" data-doi="10.1038/nature02456"><li><a href="http://dx.doi.org/10.1038/nature02456" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrating+high-throughput+and+computational+data+elucidates+bacterial+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrating+high-throughput+and+computational+data+elucidates+bacterial+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000273-Kang1" id="pone.0000273-Kang1"></a>Kang Y, Weber KD, Qiu Y, Kiley PJ, Blattner FR (2005) Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function. J Bacteriol  187: 1135–1160.  <ul class="find" data-citedArticleID="935650" data-doi="10.1128/jb.187.3.1135-1160.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.3.1135-1160.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+expression+analysis+indicates+that+FNR+of+Escherichia+coli+K-12+regulates+a+large+number+of+genes+of+unknown+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+expression+analysis+indicates+that+FNR+of+Escherichia+coli+K-12+regulates+a+large+number+of+genes+of+unknown+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000273-Salgado1" id="pone.0000273-Salgado1"></a>Salgado H, Gama-Castro S, Peralta-Gil M, Diaz-Peredo E, Sanchez-Solano F, et al.  (2006) RegulonDB (version 5.0): Escherichia coli K-12 transcriptional regulatory network, operon organization, and growth conditions. Nucleic Acids Res  34: D394–397.  <ul class="find" data-citedArticleID="935668" data-doi="10.1093/nar/gkj156"><li><a href="http://dx.doi.org/10.1093/nar/gkj156" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RegulonDB+%28version+5.0%29%3A+Escherichia+coli+K-12+transcriptional+regulatory+network%2C+operon+organization%2C+and+growth+conditions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RegulonDB+%28version+5.0%29%3A+Escherichia+coli+K-12+transcriptional+regulatory+network%2C+operon+organization%2C+and+growth+conditions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000273-Bolstad1" id="pone.0000273-Bolstad1"></a>Bolstad BM, Irizarry RA, Astrand M, Speed TP (2003) A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. Bioinformatics  19: 185–193.  <ul class="find" data-citedArticleID="935640" data-doi="10.1093/bioinformatics/19.2.185"><li><a href="http://dx.doi.org/10.1093/bioinformatics/19.2.185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+variance+and+bias." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+variance+and+bias.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000273-Smyth1" id="pone.0000273-Smyth1"></a>Smyth GK (2004) Linear models and empirical bayes methods for assessing differential expression in microarray experiments. Stat Appl Genet Mol Biol  3: Article3.  <ul class="find" data-citedArticleID="935672" data-doi="10.2202/1544-6115.1027"><li><a href="http://dx.doi.org/10.2202/1544-6115.1027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linear+models+and+empirical+bayes+methods+for+assessing+differential+expression+in+microarray+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linear+models+and+empirical+bayes+methods+for+assessing+differential+expression+in+microarray+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000273-Gentleman1" id="pone.0000273-Gentleman1"></a>Gentleman RC, Carey VJ, Bates DM, Bolstad B, Dettling M, et al.  (2004) Bioconductor: open software development for computational biology and bioinformatics. Genome Biol  5: R80.  <ul class="find" data-citedArticleID="935644" data-doi="10.1186/gb-2004-5-10-r80"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-10-r80" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bioconductor%3A+open+software+development+for+computational+biology+and+bioinformatics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bioconductor%3A+open+software+development+for+computational+biology+and+bioinformatics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000273-AlShahrour1" id="pone.0000273-AlShahrour1"></a>Al-Shahrour F, Diaz-Uriarte R, Dopazo J (2004) FatiGO: a web tool for finding significant associations of Gene Ontology terms with groups of genes. Bioinformatics  20: 578–580.  <ul class="find" data-citedArticleID="935636" data-doi="10.1093/bioinformatics/btg455"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg455" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FatiGO%3A+a+web+tool+for+finding+significant+associations+of+Gene+Ontology+terms+with+groups+of+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FatiGO%3A+a+web+tool+for+finding+significant+associations+of+Gene+Ontology+terms+with+groups+of+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000273-Rice1" id="pone.0000273-Rice1"></a>Rice P, Longden I, Bleasby A (2000) EMBOSS: the European Molecular Biology Open Software Suite. Trends Genet  16: 276–277.  <ul class="find" data-citedArticleID="935664" data-doi="10.1016/s0168-9525(00)02024-2"><li><a href="http://dx.doi.org/10.1016/s0168-9525(00)02024-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EMBOSS%3A+the+European+Molecular+Biology+Open+Software+Suite." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EMBOSS%3A+the+European+Molecular+Biology+Open+Software+Suite.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000273-Oshima1" id="pone.0000273-Oshima1"></a>Oshima T, Wada C, Kawagoe Y, Ara T, Maeda M, et al.  (2002) Genome-wide analysis of deoxyadenosine methyltransferase-mediated control of gene expression in Escherichia coli. Mol Microbiol  45: 673–695.  <ul class="find" data-citedArticleID="935658" data-doi="10.1046/j.1365-2958.2002.03037.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.03037.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+deoxyadenosine+methyltransferase-mediated+control+of+gene+expression+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+deoxyadenosine+methyltransferase-mediated+control+of+gene+expression+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000273-LobnerOlesen1" id="pone.0000273-LobnerOlesen1"></a>Lobner-Olesen A, Marinus MG, Hansen FG (2003) Role of SeqA and Dam in Escherichia coli gene expression: a global/microarray analysis. Proc Natl Acad Sci U S A  100: 4672–4677.  <ul class="find" data-citedArticleID="935654" data-doi="10.1073/pnas.0538053100"><li><a href="http://dx.doi.org/10.1073/pnas.0538053100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+SeqA+and+Dam+in+Escherichia+coli+gene+expression%3A+a+global%2Fmicroarray+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+SeqA+and+Dam+in+Escherichia+coli+gene+expression%3A+a+global%2Fmicroarray+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000273-MartinezAntonio1" id="pone.0000273-MartinezAntonio1"></a>Martinez-Antonio A, Collado-Vides J (2003) Identifying global regulators in transcriptional regulatory networks in bacteria. Curr Opin Microbiol  6: 482–489.  <ul class="find" data-citedArticleID="935656" data-doi="10.1016/j.mib.2003.09.002"><li><a href="http://dx.doi.org/10.1016/j.mib.2003.09.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+global+regulators+in+transcriptional+regulatory+networks+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+global+regulators+in+transcriptional+regulatory+networks+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000273-vonHeydebreck1" id="pone.0000273-vonHeydebreck1"></a>von Heydebreck A, Huber W, Gentleman R (2004) Differential Expression with the Bioconductor Package. Bioconductor Working Papers.   <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0000273-Kouzminova1" id="pone.0000273-Kouzminova1"></a>Kouzminova EA, Rotman E, Macomber L, Zhang J, Kuzminov A (2004) RecA-dependent mutants in Escherichia coli reveal strategies to avoid chromosomal fragmentation. Proc Natl Acad Sci U S A  101: 16262–16267.  <ul class="find" data-citedArticleID="935652" data-doi="10.1073/pnas.0405943101"><li><a href="http://dx.doi.org/10.1073/pnas.0405943101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RecA-dependent+mutants+in+Escherichia+coli+reveal+strategies+to+avoid+chromosomal+fragmentation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RecA-dependent+mutants+in+Escherichia+coli+reveal+strategies+to+avoid+chromosomal+fragmentation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000273-Yamanaka1" id="pone.0000273-Yamanaka1"></a>Yamanaka K, Ogura T, Niki H, Hiraga S (1995) Characterization of the smtA gene encoding an S-adenosylmethionine-dependent methyltransferase of Escherichia coli. FEMS Microbiol Lett  133: 59–63.  <ul class="find" data-citedArticleID="935680" data-doi="10.1111/j.1574-6968.1995.tb07861.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.1995.tb07861.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+smtA+gene+encoding+an+S-adenosylmethionine-dependent+methyltransferase+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+smtA+gene+encoding+an+S-adenosylmethionine-dependent+methyltransferase+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000273-Sivaraman1" id="pone.0000273-Sivaraman1"></a>Sivaraman K, Seshasayee AS, Swaminathan K, Muthukumaran G, Pennathur G (2005) Promoter addresses: revelations from oligonucleotide profiling applied to the Escherichia coli genome. Theor Biol Med Model  2: 20.  <ul class="find" data-citedArticleID="935670"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Promoter+addresses%3A+revelations+from+oligonucleotide+profiling+applied+to+the+Escherichia+coli+genome.&amp;auth=&amp;atitle=Promoter+addresses%3A+revelations+from+oligonucleotide+profiling+applied+to+the+Escherichia+coli+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Promoter+addresses%3A+revelations+from+oligonucleotide+profiling+applied+to+the+Escherichia+coli+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Promoter+addresses%3A+revelations+from+oligonucleotide+profiling+applied+to+the+Escherichia+coli+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000273-Zhang1" id="pone.0000273-Zhang1"></a>Zhang H, Zhao D, Revington M, Lee W, Jia X, et al.  (1994) The solution structures of the trp repressor-operator DNA complex. J Mol Biol  238: 592–614.  <ul class="find" data-citedArticleID="935682" data-doi="10.1006/jmbi.1994.1317"><li><a href="http://dx.doi.org/10.1006/jmbi.1994.1317" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+solution+structures+of+the+trp+repressor-operator+DNA+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+solution+structures+of+the+trp+repressor-operator+DNA+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000273-Somers1" id="pone.0000273-Somers1"></a>Somers WS, Phillips SE (1992) Crystal structure of the met repressor-operator complex at 2.8 A resolution reveals DNA recognition by beta-strands. Nature  359: 387–393.  <ul class="find" data-citedArticleID="935674" data-doi="10.1038/359387a0"><li><a href="http://dx.doi.org/10.1038/359387a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+met+repressor-operator+complex+at+2.8+A+resolution+reveals+DNA+recognition+by+beta-strands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+met+repressor-operator+complex+at+2.8+A+resolution+reveals+DNA+recognition+by+beta-strands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000273-Blattner1" id="pone.0000273-Blattner1"></a>Blattner FR, Plunkett G 3rd, Bloch CA, Perna NT, Burland V, et al.  (1997) The complete genome sequence of Escherichia coli K-12. Science  277: 1453–1474.  <ul class="find" data-citedArticleID="935638" data-doi="10.1126/science.277.5331.1453"><li><a href="http://dx.doi.org/10.1126/science.277.5331.1453" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+genome+sequence+of+Escherichia+coli+K-12." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+genome+sequence+of+Escherichia+coli+K-12.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000273-Rasmussen1" id="pone.0000273-Rasmussen1"></a>Rasmussen LJ, Marinus MG, Lobner-Olesen A (1994) Novel growth rate control of dam gene expression in Escherichia coli. Mol Microbiol  12: 631–638.  <ul class="find" data-citedArticleID="935662" data-doi="10.1111/j.1365-2958.1994.tb01050.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1994.tb01050.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+growth+rate+control+of+dam+gene+expression+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+growth+rate+control+of+dam+gene+expression+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000273-Ponder1" id="pone.0000273-Ponder1"></a>Ponder RG, Fonville NC, Rosenberg SM (2005) A switch from high-fidelity to error-prone DNA double-strand break repair underlies stress-induced mutation. Mol Cell  19: 791–804.  <ul class="find" data-citedArticleID="935660" data-doi="10.1016/j.molcel.2005.07.025"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.07.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+switch+from+high-fidelity+to+error-prone+DNA+double-strand+break+repair+underlies+stress-induced+mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+switch+from+high-fidelity+to+error-prone+DNA+double-strand+break+repair+underlies+stress-induced+mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000273-vonHeydebreck2" id="pone.0000273-vonHeydebreck2"></a>von Heydebreck A, Huber W, Gentleman RC (2004) Diiferential expression with the Bioconductor project. Bioconductor Project Working Papers.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.XML" value="46444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.PDF" value="160233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.e001.PNG" value="6103"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.e001.TIF" value="27700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.g001.PNG_L" value="373714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.g001.PNG_M" value="35508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.g001.PNG_S" value="1917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.g001.TIF" value="699502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.g001.PNG_I" value="14481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.g002.PNG_L" value="313244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.g002.PNG_M" value="55086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.g002.PNG_S" value="4648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.g002.TIF" value="541306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.g002.PNG_I" value="56779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.s001.TXT" value="5415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.s002.TXT" value="4012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.s003.TXT" value="1473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.s004.TXT" value="2427"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.s005.TXT" value="1272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000273.s006.TXT" value="51173"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000273&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000273" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000273&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000273&volume=&issue=&title=An Assessment of the Role of DNA Adenine Methyltransferase on Gene Expression Regulation in E coli&author_name=Aswin%20Sai%20Narain%20Seshasayee&start_page=1&end_page=4" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000273" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000273" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000273" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000273&amp;t=An%20Assessment%20of%20the%20Role%20of%20DNA%20Adenine%20Methyltransferase%20on%20Gene%20Expression%20Regulation%20in%20E%20coli" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000273&title=An%20Assessment%20of%20the%20Role%20of%20DNA%20Adenine%20Methyltransferase%20on%20Gene%20Expression%20Regulation%20in%20E%20coli&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000273&amp;title=An%20Assessment%20of%20the%20Role%20of%20DNA%20Adenine%20Methyltransferase%20on%20Gene%20Expression%20Regulation%20in%20E%20coli" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000273" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000273&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'An Assessment of the Role of DNA Adenine Methyltransferase on Gene Expression Regulation in E coli';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000273';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20An%20Assessment%20of%20the%20Role%20of%20DNA%20Adenine%20Methyltransferase%20on%20Gene%20Expression%20Regulation%20in%20E%20coli http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000273" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000273" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="23384"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="23384"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzyme+regulation%22" title="Search for articles in the subject area:'Enzyme regulation'"><div class="flagText">Enzyme regulation</div></a>
              <div data-categoryid="19973" data-articleid="23384"
                   data-categoryname="Enzyme regulation"
                   class="flagImage" title="Flag 'Enzyme regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23384"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="23384"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="23384"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="23384"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="23384"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4672'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2626'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2167&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
