

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genome-Wide Prediction of Functional Gene-Gene Interactions Inferred from Patterns of Genetic Differentiation in Mice and Men</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001593"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001593" />

    <meta name="citation_title" content="Genome-Wide Prediction of Functional Gene-Gene Interactions Inferred from Patterns of Genetic Differentiation in Mice and Men"/>
    <meta itemprop="name" content="Genome-Wide Prediction of Functional Gene-Gene Interactions Inferred from Patterns of Genetic Differentiation in Mice and Men"/>

      <meta name="citation_author" content="Zoltán Bochdanovits"/>
            <meta name="citation_author_institution" content="Section Medical Genomics, Department of Clinical Genetics, Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands"/>
            <meta name="citation_author_institution" content="Center for Neurogenomics and Cognitive Research, Vrije Universiteit (VU), Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="David Sondervan"/>
            <meta name="citation_author_institution" content="Section Medical Genomics, Department of Clinical Genetics, Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands"/>
            <meta name="citation_author_institution" content="Center for Neurogenomics and Cognitive Research, Vrije Universiteit (VU), Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Sophie Perillous"/>
            <meta name="citation_author_institution" content="Section Medical Genomics, Department of Clinical Genetics, Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Toos van Beijsterveldt"/>
            <meta name="citation_author_institution" content="Department of Biological Psychology, Vrije Universiteit (VU), Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Dorret Boomsma"/>
            <meta name="citation_author_institution" content="Section Medical Genomics, Department of Clinical Genetics, Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands"/>
            <meta name="citation_author_institution" content="Center for Neurogenomics and Cognitive Research, Vrije Universiteit (VU), Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands"/>
            <meta name="citation_author_institution" content="Department of Biological Psychology, Vrije Universiteit (VU), Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Peter Heutink"/>
            <meta name="citation_author_institution" content="Section Medical Genomics, Department of Clinical Genetics, Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands"/>
            <meta name="citation_author_institution" content="Center for Neurogenomics and Cognitive Research, Vrije Universiteit (VU), Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands"/>
            <meta name="citation_author_institution" content="Department of Biological Psychology, Vrije Universiteit (VU), Amsterdam, The Netherlands"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001593.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1593"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Evolution in Mendelian Populations.; citation_author=S Wright; citation_journal_title=Genetics; citation_volume=16; citation_number=1; citation_pages=97-159; citation_date=1931; " />
      <meta name="citation_reference" content="citation_title=The role of mutation, inbreeding, crossbreeding and selection in evolution.; citation_author=S Wright; citation_journal_title=Proceedings of The Sixth International Congress of Genetics; citation_volume=Vol. I; citation_number=2; citation_pages=356-366; citation_date=1932; " />
      <meta name="citation_reference" content="citation_title=Perspective: The theories of Fisher and Wright in the context of metapopulations: When nature does many small experiments.; citation_author=MJ Wade; citation_author=CJ Goodnight; citation_journal_title=Evolution; citation_volume=52; citation_number=3; citation_pages=1537-1553; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Wright Shifting Balance Theory - an Experimental-Study.; citation_author=MJ Wade; citation_author=CJ Goodnight; citation_journal_title=Science; citation_volume=253; citation_number=4; citation_pages=1015-1018; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Some Models of Genetic Selection.; citation_author=E Pollak; citation_journal_title=Biometrics; citation_volume=35; citation_number=5; citation_pages=119-137; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Interrogating a high-density SNP map for signatures of natural selection.; citation_author=JM Akey; citation_author=G Zhang; citation_author=K Zhang; citation_author=L Jin; citation_author=MD Shriver; citation_journal_title=Genome Res; citation_volume=12; citation_number=6; citation_pages=1805-1814; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Whole-Genome Patterns of Common DNA Variation in Three Human Populations.; citation_author=DA Hinds; citation_author=LL Stuve; citation_author=GB Nilsen; citation_author=E Halperin; citation_author=E Eskin; citation_journal_title=Science; citation_volume=307; citation_number=7; citation_pages=1072-1079; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genomic signatures of positive selection in humans and the limits of outlier approaches.; citation_author=JL Kelley; citation_author=J Madeoy; citation_author=JC Calhoun; citation_author=W Swanson; citation_author=JM Akey; citation_journal_title=Genome Res; citation_volume=16; citation_number=8; citation_pages=980-989; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=How reliable are empirical genomic scans for selective sweeps?; citation_author=KM Teshima; citation_author=G Coop; citation_author=M Przeworski; citation_journal_title=Genome Research; citation_volume=16; citation_number=9; citation_pages=702-712; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Signatures of Reproductive Isolation in Patterns of Single Nucleotide Diversity Across Inbred Strains of Mice.; citation_author=BA Payseur; citation_author=HE Hoekstra; citation_journal_title=Genetics; citation_volume=171; citation_number=10; citation_pages=1905-1916; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Searching the Genomes of Inbred Mouse Strains for Incompatibilities That Reproductively Isolate Their Wild Relatives.; citation_author=BA Payseur; citation_author=M Place; citation_journal_title=J Hered; citation_volume=98; citation_number=11; citation_pages=115-122; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Calibrating a coalescent simulation of human genome sequence variation.; citation_author=SF Schaffner; citation_author=C Foo; citation_author=S Gabriel; citation_author=D Reich; citation_author=MJ Daly; citation_journal_title=Genome Res; citation_volume=15; citation_number=12; citation_pages=1576-1583; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=NF-E2-related factor-2 mediates neuroprotection against mitochondrial complex I inhibitors and increased concentrations of intracellular calcium in primary cortical neurons.; citation_author=JM Lee; citation_author=AY Shih; citation_author=TH Murphy; citation_author=JA Johnson; citation_journal_title=Journal of Biological Chemistry; citation_volume=278; citation_number=13; citation_pages=37948-37956; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Association analyses between polymorphisms of the phase II detoxification enzymes (GSTM1, NQO1 NQO2) and alcohol withdrawal symptoms.; citation_author=T Okubo; citation_author=S Harada; citation_author=S Higuchi; citation_author=S Matsushita; citation_journal_title=Alcoholism-Clinical and Experimental Research; citation_volume=27; citation_number=14; citation_pages=68s-71s; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Nuclear factor Nrf2 and antioxidant response element regulate NRH:quinone oxidoreductase 2 (NQO2) gene expression and antioxidant induction.; citation_author=W Wang; citation_author=AK Jaiswal; citation_journal_title=Free Radical Biology and Medicine; citation_volume=40; citation_number=15; citation_pages=1119-1130; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ascertainment bias in studies of human genome-wide polymorphism 10.1101/gr.4107905.; citation_author=AG Clark; citation_author=MJ Hubisz; citation_author=CD Bustamante; citation_author=SH Williamson; citation_author=R Nielsen; citation_journal_title=Genome Res; citation_volume=15; citation_number=16; citation_pages=1496-1502; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Neuregulin induces GABAA receptor 2 subunit expression in cultured rat cerebellar granule neurons by activating multiple signaling pathways.; citation_author=F Xie; citation_author=LT Raetzman; citation_author=RE Siegel; citation_journal_title=Journal of Neurochemistry; citation_volume=90; citation_number=17; citation_pages=1521-1529; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Neuregulin1 downregulates postsynaptic GABA(A) receptors at the hippocampal inhibitory synapse.; citation_author=M Okada; citation_author=G Corfas; citation_journal_title=Hippocampus; citation_volume=14; citation_number=18; citation_pages=337-344; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gephyrin Interacts with Dynein Light Chains 1 and 2, Components of Motor Protein Complexes.; citation_author=JC Fuhrmann; citation_author=S Kins; citation_author=P Rostaing; citation_author=O El Far; citation_author=J Kirsch; citation_journal_title=J Neurosci; citation_volume=22; citation_number=19; citation_pages=5393-5402; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Regulation of Somatodendritic GABAA Receptor Channels in Rat Hippocampal Neurons: Evidence for a Role of the Small GTPase Rac1.; citation_author=DK Meyer; citation_author=C Olenik; citation_author=F Hofmann; citation_author=H Barth; citation_author=J Leemhuis; citation_journal_title=J Neurosci; citation_volume=20; citation_number=20; citation_pages=6743-6751; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Don't worry ‘B’ happy!: a role for GABAB receptors in anxiety and depression.; citation_author=JF Cryan; citation_author=K Kaupmann; citation_journal_title=Trends in Pharmacological Sciences; citation_volume=26; citation_number=21; citation_pages=36-43; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic and environmental influences on Anxious/Depression during childhood: a study from the Netherlands Twin Register.; citation_author=DI Boomsma; citation_author=CEM van Beijsterveldt; citation_author=JJ Hudziak; citation_journal_title=Genes Brain and Behavior; citation_volume=4; citation_number=22; citation_pages=466-481; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The relative importance of heredity and environment in determining the piebald pattern of guinea pigs.; citation_author=S Wright; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=6; citation_number=23; citation_pages=320-332; citation_date=1920; " />
      <meta name="citation_reference" content="citation_title=Functional epistasis on a common MHC haplotype associated with multiple sclerosis.; citation_author=JW Gregersen; citation_author=KR Kranc; citation_author=XY Ke; citation_author=P Svendsen; citation_author=LS Madsen; citation_journal_title=Nature; citation_volume=443; citation_number=24; citation_pages=574-577; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dissection of epistasis in oligogenic Bardet-Biedl syndrome.; citation_author=JL Badano; citation_author=CC Leitch; citation_author=SJ Ansley; citation_author=H May-Simera; citation_author=S Lawson; citation_volume=439; citation_number=25; citation_pages=326-330; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Epistatic interaction between KIR3DS1 and HLA-B delays the progression to AIDS.; citation_author=MP Martin; citation_author=XJ Gao; citation_author=JH Lee; citation_author=GW Nelson; citation_author=R Detels; citation_journal_title=Nature Genetics; citation_volume=31; citation_number=26; citation_pages=429-434; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Epistasis and balanced polymorphism influencing complex trait variation.; citation_author=J Kroymann; citation_author=T Mitchell-Olds; citation_journal_title=Nature; citation_volume=435; citation_number=27; citation_pages=95-98; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Epistasis and the release of genetic variation during long-term selection.; citation_author=O Carlborg; citation_author=L Jacobsson; citation_author=P Ahgren; citation_author=P Siegel; citation_author=L Andersson; citation_volume=38; citation_number=28; citation_pages=418-420; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome-wide strategies for detecting multiple loci that influence complex diseases.; citation_author=J Marchini; citation_author=P Donnelly; citation_author=LR Cardon; citation_volume=37; citation_number=29; citation_pages=413-417; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Two-Stage Two-Locus Models in Genome-Wide Association.; citation_author=DM Evans; citation_author=J Marchini; citation_author=AP Morris; citation_author=LR Cardon; citation_journal_title=PLoS Genetics; citation_volume=2; citation_number=30; citation_pages=e157; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Maximum-likelihood estimation of molecular haplotype frequencies in a diploid population.; citation_author=L Excoffier; citation_author=M Slatkin; citation_journal_title=Mol Biol Evol; citation_volume=12; citation_number=31; citation_pages=921-927; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Estimating F-Statistics for the Analysis of Population-Structure.; citation_author=BS Weir; citation_author=CC Cockerham; citation_journal_title=Evolution; citation_volume=38; citation_number=32; citation_pages=1358-1370; citation_date=1984; " />
      <meta name="citation_reference" content="citation_author=BS Weir; citation_number=33; citation_pages=161-173; citation_date=1996; citation_publisher=Sinauer Associates; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001593" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genome-Wide Prediction of Functional Gene-Gene Interactions Inferred from Patterns of Genetic Differentiation in Mice and Men"/>
    <meta name="twitter:description" content="The human genome encodes a limited number of genes yet contributes to individual differences in a vast array of heritable traits. A possible explanation for the capacity our genome to generate this virtually unlimited range of phenotypic variation in complex traits is to assume functional interactions between genes. Therefore we searched two mammalian genomes to identify potential epistatic interactions by looking for co-adapted genes marked by excess two-locus genetic differentiation between populations/lineages using publicly available SNP genotype data. The practical motivation for this effort is to reduce the number of pair-wise tests that need to be performed in genome-wide association studies aimed at detecting G&times;G interactions, by focusing on pairs predicted to be more likely to jointly affect variation in complex traits. Hence, this approach generates a list of candidate interactions that can be empirically tested. In both the mouse and human data we observed two-locus genetic differentiation in excess of what can be expected from chance alone based on simulations. In an attempt to validate our hypothesis that pairs of genes showing excess genetic divergence represent potential functional interactions, we selected a small set of gene combinations postulated to be interacting based on our analyses and looked for a combined effect of the selected genes on variation in complex traits in both mice and man. In both cases the individual effect of the genes were not significant, instead we observed marginally significant interaction effects. These results show that genome wide searches for gene-gene interactions based on population genetic data are feasible and can generate interesting candidate gene pairs to be further tested for their contribution to phenotypic variation in complex traits."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001593.g005"/>

  <meta property="og:title" content="Genome-Wide Prediction of Functional Gene-Gene Interactions Inferred from Patterns of Genetic Differentiation in Mice and Men" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001593" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5308'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5432'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9922&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001593">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001593" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001593&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genome-Wide Prediction of Functional Gene-Gene Interactions Inferred from Patterns of Genetic Differentiation in Mice and Men
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zoltán Bochdanovits
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:z.bochdanovits@vumc.nl">z.bochdanovits@vumc.nl</a></p>

                <p>Affiliations:
                  Section Medical Genomics, Department of Clinical Genetics, Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands, 
                  Center for Neurogenomics and Cognitive Research, Vrije Universiteit (VU), Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Sondervan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Section Medical Genomics, Department of Clinical Genetics, Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands, 
                  Center for Neurogenomics and Cognitive Research, Vrije Universiteit (VU), Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sophie Perillous, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Section Medical Genomics, Department of Clinical Genetics, Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Toos van Beijsterveldt, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Psychology, Vrije Universiteit (VU), Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dorret Boomsma, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Section Medical Genomics, Department of Clinical Genetics, Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands, 
                  Center for Neurogenomics and Cognitive Research, Vrije Universiteit (VU), Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands, 
                  Department of Biological Psychology, Vrije Universiteit (VU), Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter Heutink
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Section Medical Genomics, Department of Clinical Genetics, Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands, 
                  Center for Neurogenomics and Cognitive Research, Vrije Universiteit (VU), Vrije Universiteit Medisch Centrum (VUMC), Amsterdam, The Netherlands, 
                  Department of Biological Psychology, Vrije Universiteit (VU), Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001593</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001593" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001593" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001593" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001593" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001593-g001" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001593-t001" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001593-g002" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001593-g003" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001593-g004" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001593-g005" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001593">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001593" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The human genome encodes a limited number of genes yet contributes to individual differences in a vast array of heritable traits. A possible explanation for the capacity our genome to generate this virtually unlimited range of phenotypic variation in complex traits is to assume functional interactions between genes. Therefore we searched two mammalian genomes to identify potential epistatic interactions by looking for co-adapted genes marked by excess two-locus genetic differentiation between populations/lineages using publicly available SNP genotype data. The practical motivation for this effort is to reduce the number of pair-wise tests that need to be performed in genome-wide association studies aimed at detecting G×G interactions, by focusing on pairs predicted to be more likely to jointly affect variation in complex traits. Hence, this approach generates a list of candidate interactions that can be empirically tested. In both the mouse and human data we observed two-locus genetic differentiation in excess of what can be expected from chance alone based on simulations. In an attempt to validate our hypothesis that pairs of genes showing excess genetic divergence represent potential functional interactions, we selected a small set of gene combinations postulated to be interacting based on our analyses and looked for a combined effect of the selected genes on variation in complex traits in both mice and man. In both cases the individual effect of the genes were not significant, instead we observed marginally significant interaction effects. These results show that genome wide searches for gene-gene interactions based on population genetic data are feasible and can generate interesting candidate gene pairs to be further tested for their contribution to phenotypic variation in complex traits.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Bochdanovits Z, Sondervan D, Perillous S, van Beijsterveldt T, Boomsma D, et al.  (2008) Genome-Wide Prediction of Functional Gene-Gene Interactions Inferred from Patterns of Genetic Differentiation in Mice and Men. PLoS ONE 3(2):
          e1593.
            doi:10.1371/journal.pone.0001593</p><p><strong>Academic Editor: </strong>Justin Fay, Washington University in St. Louis School of Medicine, United States of America</p><p><strong>Received:</strong> November 13, 2007; <strong>Accepted:</strong> January 13, 2008; <strong>Published:</strong> February 13, 2008</p><p><strong>Copyright:</strong> © 2008 Bochdanovits et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by the Centre for Medical Systems Biology (CMSB), a centre of excellence approved by the Netherlands Genomics Initiative/Netherlands Organization for Scientific Research (NWO). D.B. was supported by NWO grants NWO 575-25-012; NWO 575-25-006; NWO 904-57-94; NWO 051.02.060, 668.772; NWO/SPI 56-464-14192; NWO 480-04-004; NWO 400-05-717 and by NIMH RO1 MH58799-03.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The presence of epistatic interactions between genes has fundamental consequences for the course and outcome of evolution by natural selection <a href="#pone.0001593-Wright1">[1]</a>–<a href="#pone.0001593-Wade2">[4]</a> and results in the emergence of co-adapted gene complexes, i.e. combinations of variants at different genes that give a selective advantage only when both are present in the same individual. Consequently, it should be possible to detect genes involved in epistatic interactions from a population genetic approach, without prior knowledge of the specific phenotypes that are affected. Following up on the original concept introduced by Sewall Wright <a href="#pone.0001593-Wright1">[1]</a>, <a href="#pone.0001593-Wright2">[2]</a> we searched for co-adapted gene complexes by looking for excess genetic divergence between populations/lineages at two-locus genotypes. The basic rationale is that if a specific combination of alleles at different genes outperforms the alternative combination(s), than not all two-locus genotypes will be observed in their expected frequencies. Within a population selection for a trait underlined by epistatic gene effects will maintain linkage disequilibrium between the loci involved <a href="#pone.0001593-Pollak1">[5]</a>, because specific allelic combinations will be systematically removed. Between (sub-) populations the same process will generate genetic differentiation because different combinations of alleles may perform equally well, resulting in phenotypically equivalent but genetically different populations <a href="#pone.0001593-Wright2">[2]</a>. This theoretical prediction of Wright's Shifting Balance Theory of evolution has empirical support <a href="#pone.0001593-Wade2">[4]</a>. In essence, by looking for deviations from equilibrium two-locus genotype frequencies, we aim to detect the signature of natural selection <a href="#pone.0001593-Akey1">[6]</a>–<a href="#pone.0001593-Teshima1">[9]</a> on pairs of genes from population genomic data.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>We explored two mammalian genomes for functional gene-gene interactions and followed up a selection of candidate gene pairs to detect a possible joint effect on phenotypic variation in a complex trait in both mice using a well characterized set of mouse lines (BxD) and in humans using a Dutch twin cohort.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Recombinant inbred lines (RILs) are generated by inbreeding the progeny of two parental lines that are themselves inbred. As a result an individual line can have only one of a possible four two-locus genotypes: aabb, aaBB, AAbb or AABB and all are expected to occur in equal frequencies in a large collection of RILs. However, if these four genotypes do not equally persist through the many generations of inbreeding, i.e. if there is a selective advantage of a specific combination of alleles, than not all two-locus genotypes will be observed in equal frequencies. In other words, in a large set of RILs we might observe significant Linkage Disequilibrium (LD) between physically unlinked genes if these genes jointly affect an adaptive phenotype <a href="#pone.0001593-Pollak1">[5]</a> and some combinations are more likely than others to survive the process of inbreeding. In fact, this idea has been pursued before in the context of signatures of reproductive isolation and shown to reveal patterns consistent with epistatic gene interactions that arise in the shape of Dobzhansky-Muller incompatibilities <a href="#pone.0001593-Payseur1">[10]</a>, <a href="#pone.0001593-Payseur2">[11]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In contrast to the mouse data, the available human genotypes were derived from outbred, ethnically distinct populations. In this case pairs of functionally interacting genes can be detected following a slightly different approach. Considering only bi-allelic loci such as SNPs, two-locus genotypes are transmitted from one generation to another in one out of four possible combinations: for the AaBb genotype the gamets can be Ab, AB, ab, aB. If selection favors one of these configurations (i.e. virtual “haplotypes” or “trans-haplotypes” when the loci are unlinked) above the others, than excess genetic differentiation between three major human populations at the level of trans-haplotype frequencies may indicate epistatic interaction between genes for the reasons outlined above. In contrast to the recombinant inbred lines, where phase is evident, in outbred human populations trans-haplotype frequencies had to be estimated. From these frequencies F<sub>st</sub> was calculated which is a measure of genetic distance between populations originally introduced by Sewall Wright. When considering natural populations a known complication of this approach is the confounding effects of selection and demography <a href="#pone.0001593-Schaffner1">[12]</a>, i.e. a large genetic distance alone need not be sufficient evidence for natural selection. To allow for straightforward interpretation of our results and in addition to conducting coalescent simulations we restricted ourselves to a comparison of non-synonymous coding SNPs (NSCS) vs. neutral SNPs (NS). NSCS change the amino acid sequence of proteins and are therefore natural candidates to be involved in functional interactions, in contrast to combinations of NS that will provide an estimate of the background levels of genetic differences between populations. Because neutral, demographic processes have the same effect on all SNPs, any difference we might observe between the two classes, after correcting for the difference in their allele frequency spectra, should reflect the functional properties of NSCS.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>LD between unlinked loci in the mouse genome</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Based on publicly available genotype data on 89 BxD mouse RILs (<a href="http://www.genenetwork.org/dbdoc/BXDGeno.html">http://www.genenetwork.org/dbdoc/BXDGeno​.html</a>) we calculated LD and corresponding p-values for 6,659,288 pairs of physically unlinked, autosomal SNPs (<a href="#pone.0001593.s001">Table S1</a> and <a href="#pone.0001593.s002">S2</a>). Unlinked polymorphisms are not expected to be in LD, yet we observe very strong correlations between SNPs on different chromosomes. The lowest p-value in this dataset was 1.5×10<sup>−12</sup>, in fact 707 combinations of unlinked SNPs were in significant LD even after Bonferroni correction for 6,659,288 tests. As RILs are derived from a known breeding scheme, any significant result should not be caused by demography. Nevertheless it is straightforward to simulate RIL genotypes and determine empirically the expected distribution of LD between unlinked loci. We generated 1000 replicates of 89 RILs with 3800 SNPs equally distributed across the genome. The simulations showed that very high levels of LD may be present in RILs just by chance, however, starting from r<sup>2</sup>&gt;0.04, the real data consistently exhibits a significant excess of non-independent combinations (<a href="#pone-0001593-g001">Figure 1</a>), when compared to the average and standard deviation observed in the 1000 replicate simulations. These results clearly show that a large number of unlinked polymorphisms are co-occurring non-independently in the mouse genome. We propose that the most likely explanation of the data is that the specific combinations of SNPs that appear to be in strong LD in fact identify functionally interacting genes that have conveyed a selectively advantage during the process of inbreeding. The genome-wide prevalence of such epistatic interactions can be roughly estimated by quantifying the excess frequency of “real” interactions at those (higher) levels of LD where observed data is consistently in the majority. Several percent of all pairs (<a href="#pone-0001593-g001">Figure 1</a>) are implicated suggesting that the total number of functional interactions is at least a couple of times larger than the number of genes in the mouse genome.</p>
<div class="figure" id="pone-0001593-g001"><div class="img"><a name="pone-0001593-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001593.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001593.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Frequency distribution of LD between unlinked SNPs in the BxD mouse recombinant inbred lines.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A Black bars represent the observed frequency of pairs of unlinked SNPs at increasing levels of LD (measured as r<sup>2</sup>). White bars show the distribution of LD in 1000 simulated sets of RILs. The errors bars are standard deviations. As expected the distribution of the observed data is shifted to the right. B In the right tail of the distribution, the excess of unlinked SNPs that exhibit very strong LD relative to the neutral expectation remains consistently significant. It should be noted that very high levels of LD were observed in the simulated dataset but these are clearly outnumbered by the real data.</p>
<span>doi:10.1371/journal.pone.0001593.g001</span></div>

<h4>Gene-gene interactions in phase II detoxification determine alcohol preference in mice</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Given our interpretation that pairs of unlinked polymorphisms in strong LD represent functional interactions, we aimed to establish the extent to which the interaction between genes rather than their individual effect influences phenotypic variation in complex traits. For this purpose we selected a small subset of SNP pairs in strong LD. To enhance our chances to observe a real and sizeable gene-gene interaction effect we focused on the set of 707 combinations of SNPs that were in significant LD even after Bonferroni correction. In addition, we considered only gene related SNPs, for our purposes defined as either in coding sequence, splice site or mRNA 5′ and 3′ UTRs, because these directly implicate a specific gene. From the 707 SNP pairs we selected those 12 combinations where both SNPs marked a specific gene (<a href="#pone-0001593-t001">Table 1</a>). The purpose of this selection was to facilitate the interpretation of any association we might find.</p>
<div class="figure" id="pone-0001593-t001"><div class="img"><a name="pone-0001593-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001593.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001593.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>15 unlinked gene based SNPs involved 12 pair-wise combinations that exhibit significant LD after Bonferroni correction.</span></strong></p><span>doi:10.1371/journal.pone.0001593.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We used these 12 combinations of 15 gene based SNPs as predicting variables on the 626 phenotypes publicly available for a subset of the BxD RILs. To avoid fitting an over-parameterized model, in the initial screen only the interaction terms were included. The significance of the full model was evaluated compared to a model including only the intercept. We found one phenotype to be significantly associated with the genotype data, corrected for the number of tests performed. Following up this one phenotype and after excluding those variables that were not contributing to the prediction we found that 2 combinations of 4 different gene-based SNPs explained 66 % (adjusted r<sup>2</sup>) of the phenotypic variation (p = 6.5×10<sup>−5</sup>) in the amount of 3% ethanol consumed per 24 hours when given ad libitum access (<a href="#pone-0001593-g002">Figure 2</a>). When introduced to the model none of the four main effects were significant, in agreement with our expectation that the interaction terms would be of relevance, although admittedly the significance level of the two interaction terms were reduced to p = 0.04 and a marginally non-significant p = 0.06.</p>
<div class="figure" id="pone-0001593-g002"><div class="img"><a name="pone-0001593-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001593.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001593.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Consumption of 3% ethanol explained by the interaction between genes involved in oxidative stress and detoxification.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A Correspondence between the observed amount of ethanol consumed by BxD mice and the predicted values based on the genotype at four genes. The correlation between the observed and predicted values is 81% and the pattern is significant after Bonferroni correction for testing 626 phenotypes at p = 6.5×10<sup>−5</sup>. B Ethanol consumption of the four two-locus genotype classes based on Paraoxonase 2 and Nrf2. The error bars are standard error of the mean. The interaction between the two genes is apparent as the effect size of the increaser genotype at either gene depends on the genotype at the other locus. This pattern is very likely to be biologically relevant as Nrf2 is a known regulator of the expression of antioxidant proteins and Paraoxonase 2 is an antioxidant protein.</p>
<span>doi:10.1371/journal.pone.0001593.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The four genes implicated in ethanol preference in mice are: Nrf2 (rs13459064) interacting with Paraoxonase 2 (rs13478608) and LOC665113 (similar to Traf2 and NCK-interacting kinase) (rs13459183) interacting with Ptpre (rs13479546). The biological functions of the latter two are not yet known. In contrast, paraoxonase 2 is an antioxidant protein and Nrf2 is a known regulator of gene expression of antioxidant proteins and phase II detoxification enzymes <a href="#pone.0001593-Lee1">[13]</a>. Although the interaction between paraoxonase 2 and Nrf2 is marginally non-significant, the possible involvement of Nrf2 alcohol preference in mice is remarkable, as it has been recently shown that a promoter polymorphism in the phase II detoxification enzyme NQO2 plays an important role in the pathogenesis of alcoholism and alcohol withdrawal symptoms in humans <a href="#pone.0001593-Okubo1">[14]</a>. Nrf2 is known to directly regulate the expression of NQO2 <a href="#pone.0001593-Wang1">[15]</a> meaning that our tentative finding is plausible based on the known biological function of the genes involved. Clearly, more experimental studies would be necessary to establish the link between paraoxonase 2 and Nrf2 and ethanol consumption, but the present results are consistent with the prediction that unlinked loci in strong LD might be functionally interacting.</p>


<h4>Genetic differentiation of two-locus genotypes between three human populations</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Based on publicly available genotype data from Perlegen/HapMap we estimated trans-haplotype frequencies for 12,875,275 combinations of 5075 autosomal NSCS and compared the distribution of the genetic distances (Wright's F<sub>st</sub>) (<a href="#pone.0001593.s003">Table S3</a> and <a href="#pone.0001593.s004">S4</a>) to the distribution derived from 5075 randomly selected NS. Ascertainment bias of the SNPs present in this dataset is a known limitation for using the data for population genetic studies<a href="#pone.0001593-Clark1">[16]</a>, however we aim to compare two subsets of SNPs based on a property (NSCS vs. NS) not primarily involved in the SNP discovery process. In other words, we expect that the bias is similar for both NSCS and NS, although this assumption need not be correct. However, the distribution of LD (measured as r<sup>2</sup>) within populations among the NSCS and NS were identical indicating that the two samples are well matched with respect to potential sources of bias due to genomic localization, allele frequency spectrum and also ascertainment (<a href="#pone.0001593.s005">Figure S1</a>). NSCS trans-haplotypes differ significantly from those obtained from NS (Kolmogorov –Smirnof Z test, p&lt;10<sup>−13</sup>) (<a href="#pone-0001593-g003">Figure 3</a>). The analysis was also performed with the minor haplotype frequency (MHF) as a covariate to effectively compare F<sub>st</sub> values at the same level of total genetic variance. The effect of the MHF was indeed significant (p&lt;10<sup>−13</sup>), but the difference between NS and NSCS based haplotypes remained highly significant as well (p&lt;10<sup>−13</sup>). This result is underlined by simple visual inspection of the distributions, which reveals a much “thicker” right tail of the NSCS distribution. Although high levels of genetic differentiation were also observed among the NS, starting from approximately F<sub>st</sub>&gt;0.35, two third of the data were NSCS. Hence, in this dataset the prediction that a SNP pair is NSCS rather than NS based on high levels of F<sub>st</sub> alone yields a False Discovery Rate of ~0.33. In other words, the potentially functional NSCS pairs are 2 fold overrepresented at high levels of genetic differentiation.</p>
<div class="figure" id="pone-0001593-g003"><div class="img"><a name="pone-0001593-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001593.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001593.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Frequency distribution of genetic distance between three major human populations at pairwise combinations of non-synonymous coding vs. neutral SNPs.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A Black bars represent the observed frequency of F<sub>st</sub> calculated for combinations of NSCS at different genes. A higher value represents a larger genetic distance between populations considering two genes at a time. White bars represent the observed frequency calculated from randomly selected NS. The distribution based on NSCS is shifted to the right. B Starting from relatively low levels of genetic differentiation the NSCS derived trans-haplotypes are significantly in excess compared to the random variability in F<sub>st</sub> observerd in the NS.</p>
<span>doi:10.1371/journal.pone.0001593.g003</span></div>

<h4>Coalescent simulation of three major human populations</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Because both the NSCS and NS have been included randomly with respect to chromosomal position and are scattered across the entire genome, any demographic event is expected to have had the same impact on both classes of polymorphisms. Therefore, it is not likely that the clear overrepresentation of NSCS trans-haplotypes among the most strongly diverged combinations is the result of historic events. However, to estimate the variability in the amount of genetic divergence between populations due to historic processes we conducted coalescent simulations implementing parameters from a recent calibration of the method shown to accurately recreate the patterns of genetic variability between the three major world populations used here <a href="#pone.0001593-Schaffner1">[12]</a>. 100 replicates have been simulated resulting in a grand total of more than 2.5 billion haplotype based F<sub>st</sub> values. The observed variability between replicates is very low and the combinations randomly declared NSCS vs. NS show no significant difference in the pattern of genetic differentiation (<a href="#pone-0001593-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0001593-g004"><div class="img"><a name="pone-0001593-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001593.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001593.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Distribution of two-locus F<sub>st</sub> from a coalescent simulation of three major human populations.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Simulated SNPs randomly called NSCS vs. NS exhibit no difference in pair-wise genetic distance between populations. Simulated SNPs were ascertained for being polymorphic in all three populations. Results from 100 replicates and a total of 2.5 billion combinations are summarized, with the error bars representing standard deviations.</p>
<span>doi:10.1371/journal.pone.0001593.g004</span></div>

<h4>Interaction between two GABA related genes explain phenotypic differences in depression/anxiety between children</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Given our interpretation that NSCS trans-haplotypes that show excess genetic differentiation between populations represent functional interactions between genes again we scrutinized our results to see whether genes known to have functional interactions from molecular studies had been identified. We prioritized the NSCS based on the number of pairs each individual SNP was involved in at Fst&gt;0.35. This rather arbitrary cutoff was chosen because at this level of genetic differentiation the NSCS were 2 fold overrepresented compared to the NS in this dataset. Indeed, one of the NSCS (rs902790) with the highest number of reasonably strong interactions (r<sup>2</sup>&gt;0.35), identified GPR156, a GABA(B) related G-protein coupled receptor. We examined all interactors of GPR156 detected here and found GABRR3 (represented by rs832032), neuregulin 3 (rs17101193), DNAI2 (rs1979370) and DEF6 (rs2395617) all of which are related to GABA receptor functioning. GABRR3 is a GABA receptor. Family members of neuregulin 3, are known to affect GABA receptor expression<a href="#pone.0001593-Xie1">[17]</a>, <a href="#pone.0001593-Okada1">[18]</a>. DNAI2 is a dynein polypeptide and dynein light chains have been found to interact with Gephyrin, which in turn is a mediator of the clustering of major subtypes of the GABA(A) receptor <a href="#pone.0001593-Fuhrmann1">[19]</a>. DEF6 is an upstream activator of the Rho GTPase Rac1, a regulator of GABA(A) receptor channels in rat hippocampal neurons <a href="#pone.0001593-Meyer1">[20]</a>.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Because the GABA receptor pathway is known to be involved in depression in humans <a href="#pone.0001593-Cryan1">[21]</a> we explored the possibility that individual differences in a mood related phenotype might be explained by the four combinations of the GABA related genes claimed here to be functional. The SNP rs90279 and the four NSCS ascertained for potentially functional pair-wise interactions with it were genotyped in a Dutch twin cohort that was previously phenotyped for a measure of depression/anxiety as extensively described elsewhere <a href="#pone.0001593-Boomsma1">[22]</a>. Following appropriate (quality) checks three pairs of protein variants were subjected to a population based association test that allows for detecting statistical interaction between the variants. No individual effect of the four NSCS analysed was significant, however the interaction between GPR156 and DNAI2 was significantly associated (p = 0.04) with childhood depression/anxiety in this cohort (<a href="#pone-0001593-g005">Figure 5</a>). As three tests have been performed the above finding would not be significant if Bonferroni correction would be applied, however these test are not independent as they all include rs90279. Although only marginally significant, this result is in line with the prediction that these genes are members of a co-adapted gene complex and illustrates that searching for main effects alone will could miss relevant determinants of complex phenotypes, although a replication of this association in an independent cohort would be needed before any conclusions can be drawn.</p>
<div class="figure" id="pone-0001593-g005"><div class="img"><a name="pone-0001593-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001593" data-uri="info:doi/10.1371/journal.pone.0001593.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001593.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001593.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001593.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001593.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001593.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The interaction between two genes related to GABA receptor signaling is associated with anxiety/depression in a Dutch, family based cohort.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A The individual effects of both GPR156 (rs902790) and DNAI2 (rs1979370) do not predict the levels of anxiety/depression in children. Instead the phenotype depends on the specific combination of protein variants present at the two genes. B Given the small number of individuals homozygous for the minor alleles at either gene (15 out of 766 individuals), only the four common genotype classes are depicted here. A significant interaction between the genes can be observed as both the double heterozygous individuals and the individuals homozygous for both major alleles exhibit increased levels of anxiety/depression. The effect size of any given protein variant is dependent on the variant present at the other locus in the same individual.</p>
<span>doi:10.1371/journal.pone.0001593.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>More than 75 years ago Sewall Wright proposed that functional interactions between genes would allow the genome to code for a virtually unlimited range of phenotypes and in fact he had produced one of the first empirical studies to show that epistastic interactions indeed are involved in determining a complex trait <a href="#pone.0001593-Wright3">[23]</a>. The existence of gene-gene interactions and their relevance for the genetics of complex traits has been generally accepted as shown by ample examples <a href="#pone.0001593-Gregersen1">[24]</a>–<a href="#pone.0001593-Carlborg1">[28]</a>, but an estimate of the genome wide prevalence of epistasis and a feasible approach to ascertain functionally interacting genes from genotype data has been lacking.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>A profound difficulty in a phenotype based assessment of epistasis on a genome wide scale is the need to correct for a huge number of tests with obvious consequences for statistical power. Recently strategies have been proposed to tackle this problem <a href="#pone.0001593-Marchini1">[29]</a>, <a href="#pone.0001593-Evans1">[30]</a>, but these are yet to be put in practice. In the present approach we refrain from trying to assess the probability that an individual combination is indeed functional, instead we demonstrate an excess two-locus genetic differentiation between populations in two mammalian species. In agreement with previous results on single markers <a href="#pone.0001593-Akey1">[6]</a>, <a href="#pone.0001593-Hinds1">[7]</a>, we observe that high levels of genetic differentiation alone is not sufficient to prove function, as both in the mouse and human analyses very high values of LD and F<sub>st</sub> were indeed observed in the simulations (mice) or among the NS (humans). However, the aim of the present approach is rather to first ascertain candidate “co-adapted” genes to be tested for association with complex traits, thus strongly reducing the number of tests in the analysis of specific phenotypes. Indeed, in both the mouse and the human analysis we selected a limited subset of combinations that were likely to be involved in epistatic interactions. In both cases no main effects of these candidate “co-adapted” genes were found on the complex traits analysed. However, we observed marginally significant interaction effects of these “co-adapted” genes on ethanol consumption in mice and on anxiety/depression in humans. Replication of these associations are necessary to ascertain that they are not false positives, but the results are consistent with the prediction that the interaction between these genes rather than their individual effects should be associated with phenotypic variation.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Estimating the genome wide preference of functional interactions from these results remains difficult because divergence taking two genes into account does not necessarily have to be caused by the interaction among the genes. A strong individual effect of either of them might explain the observed pattern. While we can not exclude that strong main effects contribute to the observed patters we would argue that this can not explain the majority of the cases. If that were the case we should expect <em>all</em> combinations involving the gene/SNP with the strong main effect to yield a high signal. Clearly this is not the case demonstrated by the fact that the 707 Bonferroni significant combinations in the mouse data involved 315 SNPs. The SNP involved in the highest number of interactions was only present in 15 pairs and 40% of the 315 SNPs was involved in not more than 2 combinations. Real interactions rather than strong individual effects must clearly be involved in the majority of the cases. An obvious inference from these results is that standard genetic analyses of complex traits are bound to miss the vast majority of relevant genes. In contrast, here we show that first ascertaining candidate co-adapted genes from population genomic data followed up by an association study is a feasible way to simultaneously detect multiple susceptibility genes for complex (disease) phenotypes even if these lack any individual effect on their own.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>LD in and simulation of mouse RILs</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The genotype data on the BXD mouse recombinant inbred lines were downloaded from The GeneNetwork (<a href="http://www.genenetwork.org/dbdoc/BXDGeno.html">http://www.genenetwork.org/dbdoc/BXDGeno​.html</a>). Linkage disequilibrium was calculated as the square of the correlation coefficient between two loci across the 89 lines and corresponding p-values were derived from a chi-squared test. Heterozygous RIL genotypes were treated as missing data. SNPs were considered unlinked if they were on different chromosomes or were more than 50 cM apart. Extreme values of allele frequencies may bias LD estimates but in RILs, allele frequencies at all loci should be by definition 0.5. In this dataset the average allele frequency was 0.4996+/−0.079 (SD). QTLCartographer was used to simulate genotyes for 89 recombinant inbred lines with 200 neutral loci per chromosome. Pairwise r<sup>2</sup> values were calculated for unlinked SNPs from 1000 replicates.</p>


<h4>Genetic association with interactions between SNPs in mouse RILs</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The phenotype data on the BXD mouse recombinant inbred lines were downloaded from Nervenet (<a href="http://www.nervenet.org/main/databases.html">http://www.nervenet.org/main/databases.h​tml</a>). For the association analysis a multiple linear regression model was implemented in R with the RIL phenotypes as dependent variables and the interaction terms of the SNP pairs as predictors. The number of BxD lines with phenotype data is much lower than the number of lines that have been genotyped, around the 20 depending on the phenotype. Because of this, in the initial screen of the 626 phenotypes only the interaction terms were considered to avoid fitting an over-parameterized model. However, the main effects were included in the analysis of the best fitting model to assess the significance of the interaction terms per se. None of the four main effects were significant in the full model which explained 66% of the phenotypic variation between the RILs. Bonferroni correction was applied to correct for the number of phenotypes tested for determining the significance of the best fitting model.</p>


<h4>Estimation of trans-haplotype frequencies, calculation of F<sub>st</sub> between three human populations and correction for trans-haplotype frequencies and LD between markers</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Genotype data for a large number of SNPs is available based on 24, 23 and 24 individuals from respectively an Asian, African-American and Caucasian population from Perlegen/HapMap. Out of the approximately 1.5 million SNPs with available genotypes, 5075 were found to be non-synonymous coding variants, polymorphic in all three populations. Unlike in mouse RILs, phase is not evident in outbred genotype data, thus trans-haplotype frequencies were estimated in each population separately with the Expectation-Maximization algorithm implemented in the LDmax software <a href="#pone.0001593-Excoffier1">[31]</a>. F<sub>st</sub> among three populations was calculated according to the method of Weir <a href="#pone.0001593-Weir1">[32]</a>, <a href="#pone.0001593-Weir2">[33]</a> that takes into account small sample estimation bias, using custom made shell scripts. Measures of genetic distance are known to be a function of the allele frequency of the loci involved. Therefore, the comparison of NS vs. NSCS distributions was repeated based on the residuals after modeling F<sub>st</sub> in a linear model with all 12 trans-haplotype frequencies and 3 measures of LD (r2, one for each population) as predicting variables. Although all included predictors had a significant effect, the total amount of variance explained was small and the difference between NS and NSCS remained highly significant (Kolmogorov–Smirnof Z test, p&lt;10<sup>−13</sup>).</p>


<h4>Coalescent simulation of three human populations</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Using the cosi software for coalescent simulations we generated genotype data for 48, 46 and 48 chromosomes respectively from the three human populations. We implemented parameters as described in a recent calibration of the method <a href="#pone.0001593-Schaffner1">[12]</a>, shown to very well approximate the real data available from the HapMap project. 22 independent autosomal chromosomes were simulated by joining the output of separate cosi runs. This way the different “chromosomes” will have different coalescent trajectories but will be subject of the same demographic history. A total of 10150 loci have been randomly generated, conditioning on all loci being polymorphic in all three populations. These 10150 loci were randomly assigned in two groups of 5075 NS and 5075 NSCS. Computational restrictions limit the size of the chromosomal fragments that can be simulated to 1.5 MB. Due to this limitation the density of the simulated polymorphisms is higher that in the real data, but the distribution of pairwise LD (r<sup>2</sup>) from the simulations was very similar to the real data. This could be expected as 1.5 Mb is a large enough distance for most polymorphisms to be in low LD in the general populations simulated here. Once the average distance between loci is large enough not to be in strong LD a further decrease in marker spacing is not necessary for the simulation results to be representative. The distribution of “NS” vs. “NSCS” F<sub>st</sub> values were compared from this data, using 100 replicates and a total of approximately 2.5 billion genetic distances.</p>


<h4>Genetic association with interactions between NSCS in human family based data</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Details of the Dutch twin cohort and the phenotypic measures used have been described by previously <a href="#pone.0001593-Boomsma1">[22]</a>. A total of 758 individuals contributed both genotype and phenotype information for this study. The cohort contained 238 monozygous and 250 dizygous twin families. For this analysis, measures of depression/anxiety obtained from children at three different ages (7, 10 and 12 years) were averaged to yield a robust estimate of the individual phenotype. Because the distribution of this measure is right skewed the data were log transformed before analysis. Genotypes for the five NSCS were determined using the ABI SNPlex Genotyping system following the manufacturer's recommendations (Applied Biosystems, Foster city, CA, USA). All pre-PCR steps were performed on a cooled block. Reactions were carried out in Gene Amp 9700 Thermocycler (Applied Biosystems, Foster city, CA, USA). PCR products were analyzed with ABI3730 Sequencer (Applied Biosystems, Foster city, CA, USA). Alleles were called using Genemapper v3.7 (Applied Biosystems, Foster city, CA, USA). Initial data analyses were performed using the pedstats and QTDT software including gender as a covariate. All NSCS were in Hardy &amp; Weinberg equilibrium and four showed no evidence of population stratification but rs17101193 (neuregulin 3) could not be tested for population stratification due to a lack of informative families and was excluded from the population based association test. Given that no evidence for population stratification was found for the remaining four NSCS we chose to use the population based association test implemented in QTDT as it is known to have more statistical power. This is a linear model that includes variance components to model the phenotypic similarities within a family, i.e. between the twins. Standard software for family based genetic association with quantitative traits (such as QTDT) do not readily allow for testing the significance of gene-gene interactions, but QTDT does allow for testing interactions between covariates. We created dummy variables coding the three genotypes per locus −1,0 and 1 and compared the likelihood of the model including the interaction term between two loci with the likelihood of the model with only the two main effects present. The significance of the interaction term can be determined by considering minus twice the difference between the loglikelihoods of the models. This quantity has a chi squared distribution with one degree of freedom.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001593"></div><a name="pone.0001593.s001" id="pone.0001593.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001593.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001593.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(46.60 MB ZIP)</p>
<a name="pone.0001593.s002" id="pone.0001593.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001593.s002">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001593.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.02 MB ZIP)</p>
<a name="pone.0001593.s003" id="pone.0001593.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001593.s003">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001593.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(78.31 MB ZIP)</p>
<a name="pone.0001593.s004" id="pone.0001593.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001593.s004">Table S4. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001593.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(0.04 MB ZIP)</p>
<a name="pone.0001593.s005" id="pone.0001593.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001593.s005">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001593.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="postSiDOI">(0.73 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank prof. A. Brussaard for useful discussion, Eske Derks, Rosa Hoekstra en Marieke van Leeuwen for data collection.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: ZB. Performed the experiments: DS SP. Analyzed the data: ZB. Contributed reagents/materials/analysis tools: DB Tv. Wrote the paper: ZB. Other: Supervised study: PH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001593-Wright1" id="pone.0001593-Wright1"></a>Wright S (1931) Evolution in Mendelian Populations. Genetics  16: 97–159.  <ul class="find" data-citedArticleID="1055346"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+in+Mendelian+Populations.&amp;auth=&amp;atitle=Evolution+in+Mendelian+Populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+in+Mendelian+Populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+in+Mendelian+Populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001593-Wright2" id="pone.0001593-Wright2"></a>Wright S (1932) The role of mutation, inbreeding, crossbreeding and selection in evolution. Proceedings of The Sixth International Congress of Genetics  Vol. I: 356–366.  <ul class="find" data-citedArticleID="1055348"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+role+of+mutation%2C+inbreeding%2C+crossbreeding+and+selection+in+evolution.&amp;auth=&amp;atitle=The+role+of+mutation%2C+inbreeding%2C+crossbreeding+and+selection+in+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+mutation%2C+inbreeding%2C+crossbreeding+and+selection+in+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+mutation%2C+inbreeding%2C+crossbreeding+and+selection+in+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001593-Wade1" id="pone.0001593-Wade1"></a>Wade MJ, Goodnight CJ (1998) Perspective: The theories of Fisher and Wright in the context of metapopulations: When nature does many small experiments. Evolution  52: 1537–1553.  <ul class="find" data-citedArticleID="1055336" data-doi="10.2307/2411328"><li><a href="http://dx.doi.org/10.2307/2411328" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Perspective%3A+The+theories+of+Fisher+and+Wright+in+the+context+of+metapopulations%3A+When+nature+does+many+small+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Perspective%3A+The+theories+of+Fisher+and+Wright+in+the+context+of+metapopulations%3A+When+nature+does+many+small+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001593-Wade2" id="pone.0001593-Wade2"></a>Wade MJ, Goodnight CJ (1991) Wright Shifting Balance Theory - an Experimental-Study. Science  253: 1015–1018.  <ul class="find" data-citedArticleID="1055338" data-doi="10.1126/science.1887214"><li><a href="http://dx.doi.org/10.1126/science.1887214" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wright+Shifting+Balance+Theory+-+an+Experimental-Study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Wright+Shifting+Balance+Theory+-+an+Experimental-Study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001593-Pollak1" id="pone.0001593-Pollak1"></a>Pollak E (1979) Some Models of Genetic Selection. Biometrics  35: 119–137.  <ul class="find" data-citedArticleID="1055330" data-doi="10.2307/2529940"><li><a href="http://dx.doi.org/10.2307/2529940" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Some+Models+of+Genetic+Selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Some+Models+of+Genetic+Selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001593-Akey1" id="pone.0001593-Akey1"></a>Akey JM, Zhang G, Zhang K, Jin L, Shriver MD (2002) Interrogating a high-density SNP map for signatures of natural selection. Genome Res  12: 1805–1814.  <ul class="find" data-citedArticleID="1055288" data-doi="10.1101/gr.631202"><li><a href="http://dx.doi.org/10.1101/gr.631202" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interrogating+a+high-density+SNP+map+for+signatures+of+natural+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interrogating+a+high-density+SNP+map+for+signatures+of+natural+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001593-Hinds1" id="pone.0001593-Hinds1"></a>Hinds DA, Stuve LL, Nilsen GB, Halperin E, Eskin E, et al.  (2005) Whole-Genome Patterns of Common DNA Variation in Three Human Populations. Science  307: 1072–1079.  <ul class="find" data-citedArticleID="1055308" data-doi="10.1126/science.1105436"><li><a href="http://dx.doi.org/10.1126/science.1105436" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-Genome+Patterns+of+Common+DNA+Variation+in+Three+Human+Populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-Genome+Patterns+of+Common+DNA+Variation+in+Three+Human+Populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001593-Kelley1" id="pone.0001593-Kelley1"></a>Kelley JL, Madeoy J, Calhoun JC, Swanson W, Akey JM (2006) Genomic signatures of positive selection in humans and the limits of outlier approaches. Genome Res  16: 980–989.  <ul class="find" data-citedArticleID="1055310" data-doi="10.1101/gr.5157306"><li><a href="http://dx.doi.org/10.1101/gr.5157306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+signatures+of+positive+selection+in+humans+and+the+limits+of+outlier+approaches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+signatures+of+positive+selection+in+humans+and+the+limits+of+outlier+approaches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001593-Teshima1" id="pone.0001593-Teshima1"></a>Teshima KM, Coop G, Przeworski M (2006) How reliable are empirical genomic scans for selective sweeps? Genome Research  16: 702–712.  <ul class="find" data-citedArticleID="1055334" data-doi="10.1101/gr.5105206"><li><a href="http://dx.doi.org/10.1101/gr.5105206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+reliable+are+empirical+genomic+scans+for+selective+sweeps%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+reliable+are+empirical+genomic+scans+for+selective+sweeps%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001593-Payseur1" id="pone.0001593-Payseur1"></a>Payseur BA, Hoekstra HE (2005) Signatures of Reproductive Isolation in Patterns of Single Nucleotide Diversity Across Inbred Strains of Mice. Genetics  171: 1905–1916.  <ul class="find" data-citedArticleID="1055326" data-doi="10.1534/genetics.105.046193"><li><a href="http://dx.doi.org/10.1534/genetics.105.046193" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signatures+of+Reproductive+Isolation+in+Patterns+of+Single+Nucleotide+Diversity+Across+Inbred+Strains+of+Mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signatures+of+Reproductive+Isolation+in+Patterns+of+Single+Nucleotide+Diversity+Across+Inbred+Strains+of+Mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001593-Payseur2" id="pone.0001593-Payseur2"></a>Payseur BA, Place M (2007) Searching the Genomes of Inbred Mouse Strains for Incompatibilities That Reproductively Isolate Their Wild Relatives. J Hered  98: 115–122.  <ul class="find" data-citedArticleID="1055328" data-doi="10.1093/jhered/esl064"><li><a href="http://dx.doi.org/10.1093/jhered/esl064" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Searching+the+Genomes+of+Inbred+Mouse+Strains+for+Incompatibilities+That+Reproductively+Isolate+Their+Wild+Relatives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Searching+the+Genomes+of+Inbred+Mouse+Strains+for+Incompatibilities+That+Reproductively+Isolate+Their+Wild+Relatives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001593-Schaffner1" id="pone.0001593-Schaffner1"></a>Schaffner SF, Foo C, Gabriel S, Reich D, Daly MJ, et al.  (2005) Calibrating a coalescent simulation of human genome sequence variation. Genome Res  15: 1576–1583.  <ul class="find" data-citedArticleID="1055332" data-doi="10.1101/gr.3709305"><li><a href="http://dx.doi.org/10.1101/gr.3709305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Calibrating+a+coalescent+simulation+of+human+genome+sequence+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Calibrating+a+coalescent+simulation+of+human+genome+sequence+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001593-Lee1" id="pone.0001593-Lee1"></a>Lee JM, Shih AY, Murphy TH, Johnson JA (2003) NF-E2-related factor-2 mediates neuroprotection against mitochondrial complex I inhibitors and increased concentrations of intracellular calcium in primary cortical neurons. Journal of Biological Chemistry  278: 37948–37956.  <ul class="find" data-citedArticleID="1055314" data-doi="10.1074/jbc.m305204200"><li><a href="http://dx.doi.org/10.1074/jbc.m305204200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NF-E2-related+factor-2+mediates+neuroprotection+against+mitochondrial+complex+I+inhibitors+and+increased+concentrations+of+intracellular+calcium+in+primary+cortical+neurons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NF-E2-related+factor-2+mediates+neuroprotection+against+mitochondrial+complex+I+inhibitors+and+increased+concentrations+of+intracellular+calcium+in+primary+cortical+neurons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001593-Okubo1" id="pone.0001593-Okubo1"></a>Okubo T, Harada S, Higuchi S, Matsushita S (2003) Association analyses between polymorphisms of the phase II detoxification enzymes (GSTM1, NQO1 NQO2) and alcohol withdrawal symptoms. Alcoholism-Clinical and Experimental Research  27: 68s–71s.  <ul class="find" data-citedArticleID="1055324" data-doi="10.1097/01.alc.0000078616.63296.41"><li><a href="http://dx.doi.org/10.1097/01.alc.0000078616.63296.41" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+analyses+between+polymorphisms+of+the+phase+II+detoxification+enzymes+%28GSTM1%2C+NQO1+NQO2%29+and+alcohol+withdrawal+symptoms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+analyses+between+polymorphisms+of+the+phase+II+detoxification+enzymes+%28GSTM1%2C+NQO1+NQO2%29+and+alcohol+withdrawal+symptoms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001593-Wang1" id="pone.0001593-Wang1"></a>Wang W, Jaiswal AK (2006) Nuclear factor Nrf2 and antioxidant response element regulate NRH:quinone oxidoreductase 2 (NQO2) gene expression and antioxidant induction. Free Radical Biology and Medicine  40: 1119–1130.  <ul class="find" data-citedArticleID="1055340" data-doi="10.1016/j.freeradbiomed.2005.10.063"><li><a href="http://dx.doi.org/10.1016/j.freeradbiomed.2005.10.063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+factor+Nrf2+and+antioxidant+response+element+regulate+NRH%3Aquinone+oxidoreductase+2+%28NQO2%29+gene+expression+and+antioxidant+induction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+factor+Nrf2+and+antioxidant+response+element+regulate+NRH%3Aquinone+oxidoreductase+2+%28NQO2%29+gene+expression+and+antioxidant+induction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001593-Clark1" id="pone.0001593-Clark1"></a>Clark AG, Hubisz MJ, Bustamante CD, Williamson SH, Nielsen R (2005) Ascertainment bias in studies of human genome-wide polymorphism 10.1101/gr.4107905. Genome Res  15: 1496–1502.  <ul class="find" data-citedArticleID="1055296" data-doi="10.1101/gr.4107905"><li><a href="http://dx.doi.org/10.1101/gr.4107905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ascertainment+bias+in+studies+of+human+genome-wide+polymorphism+10.1101%2Fgr.4107905." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ascertainment+bias+in+studies+of+human+genome-wide+polymorphism+10.1101%2Fgr.4107905.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001593-Xie1" id="pone.0001593-Xie1"></a>Xie F, Raetzman LT, Siegel RE (2004) Neuregulin induces GABAA receptor 2 subunit expression in cultured rat cerebellar granule neurons by activating multiple signaling pathways. Journal of Neurochemistry  90: 1521–1529.  <ul class="find" data-citedArticleID="1055352" data-doi="10.1111/j.1471-4159.2004.02685.x"><li><a href="http://dx.doi.org/10.1111/j.1471-4159.2004.02685.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neuregulin+induces+GABAA+receptor+2+subunit+expression+in+cultured+rat+cerebellar+granule+neurons+by+activating+multiple+signaling+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neuregulin+induces+GABAA+receptor+2+subunit+expression+in+cultured+rat+cerebellar+granule+neurons+by+activating+multiple+signaling+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001593-Okada1" id="pone.0001593-Okada1"></a>Okada M, Corfas G (2004) Neuregulin1 downregulates postsynaptic GABA(A) receptors at the hippocampal inhibitory synapse. Hippocampus  14: 337–344.  <ul class="find" data-citedArticleID="1055322" data-doi="10.1002/hipo.10185"><li><a href="http://dx.doi.org/10.1002/hipo.10185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neuregulin1+downregulates+postsynaptic+GABA%28A%29+receptors+at+the+hippocampal+inhibitory+synapse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neuregulin1+downregulates+postsynaptic+GABA%28A%29+receptors+at+the+hippocampal+inhibitory+synapse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001593-Fuhrmann1" id="pone.0001593-Fuhrmann1"></a>Fuhrmann JC, Kins S, Rostaing P, El Far O, Kirsch J, et al.  (2002) Gephyrin Interacts with Dynein Light Chains 1 and 2, Components of Motor Protein Complexes. J Neurosci  22: 5393–5402.  <ul class="find" data-citedArticleID="1055304"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gephyrin+Interacts+with+Dynein+Light+Chains+1+and+2%2C+Components+of+Motor+Protein+Complexes.&amp;auth=&amp;atitle=Gephyrin+Interacts+with+Dynein+Light+Chains+1+and+2%2C+Components+of+Motor+Protein+Complexes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gephyrin+Interacts+with+Dynein+Light+Chains+1+and+2%2C+Components+of+Motor+Protein+Complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gephyrin+Interacts+with+Dynein+Light+Chains+1+and+2%2C+Components+of+Motor+Protein+Complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001593-Meyer1" id="pone.0001593-Meyer1"></a>Meyer DK, Olenik C, Hofmann F, Barth H, Leemhuis J, et al.  (2000) Regulation of Somatodendritic GABAA Receptor Channels in Rat Hippocampal Neurons: Evidence for a Role of the Small GTPase Rac1. J Neurosci  20: 6743–6751.  <ul class="find" data-citedArticleID="1055320"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+Somatodendritic+GABAA+Receptor+Channels+in+Rat+Hippocampal+Neurons%3A+Evidence+for+a+Role+of+the+Small+GTPase+Rac1.&amp;auth=&amp;atitle=Regulation+of+Somatodendritic+GABAA+Receptor+Channels+in+Rat+Hippocampal+Neurons%3A+Evidence+for+a+Role+of+the+Small+GTPase+Rac1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+Somatodendritic+GABAA+Receptor+Channels+in+Rat+Hippocampal+Neurons%3A+Evidence+for+a+Role+of+the+Small+GTPase+Rac1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+Somatodendritic+GABAA+Receptor+Channels+in+Rat+Hippocampal+Neurons%3A+Evidence+for+a+Role+of+the+Small+GTPase+Rac1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001593-Cryan1" id="pone.0001593-Cryan1"></a>Cryan JF, Kaupmann K (2005) Don't worry ‘B’ happy!: a role for GABAB receptors in anxiety and depression. Trends in Pharmacological Sciences  26: 36–43.  <ul class="find" data-citedArticleID="1055298" data-doi="10.1016/j.tips.2004.11.004"><li><a href="http://dx.doi.org/10.1016/j.tips.2004.11.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Don%27t+worry+%E2%80%98B%E2%80%99+happy%21%3A+a+role+for+GABAB+receptors+in+anxiety+and+depression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Don%27t+worry+%E2%80%98B%E2%80%99+happy%21%3A+a+role+for+GABAB+receptors+in+anxiety+and+depression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001593-Boomsma1" id="pone.0001593-Boomsma1"></a>Boomsma DI, van Beijsterveldt CEM, Hudziak JJ (2005) Genetic and environmental influences on Anxious/Depression during childhood: a study from the Netherlands Twin Register. Genes Brain and Behavior  4: 466–481.  <ul class="find" data-citedArticleID="1055292" data-doi="10.1111/j.1601-183x.2005.00141.x"><li><a href="http://dx.doi.org/10.1111/j.1601-183x.2005.00141.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+and+environmental+influences+on+Anxious%2FDepression+during+childhood%3A+a+study+from+the+Netherlands+Twin+Register." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+and+environmental+influences+on+Anxious%2FDepression+during+childhood%3A+a+study+from+the+Netherlands+Twin+Register.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001593-Wright3" id="pone.0001593-Wright3"></a>Wright S (1920) The relative importance of heredity and environment in determining the piebald pattern of guinea pigs. Proc Natl Acad Sci USA  6: 320–332.  <ul class="find" data-citedArticleID="1055350" data-doi="10.1073/pnas.6.6.320"><li><a href="http://dx.doi.org/10.1073/pnas.6.6.320" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+relative+importance+of+heredity+and+environment+in+determining+the+piebald+pattern+of+guinea+pigs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+relative+importance+of+heredity+and+environment+in+determining+the+piebald+pattern+of+guinea+pigs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001593-Gregersen1" id="pone.0001593-Gregersen1"></a>Gregersen JW, Kranc KR, Ke XY, Svendsen P, Madsen LS, et al.  (2006) Functional epistasis on a common MHC haplotype associated with multiple sclerosis. Nature  443: 574–577.  <ul class="find" data-citedArticleID="1055306" data-doi="10.1038/nature05133"><li><a href="http://dx.doi.org/10.1038/nature05133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+epistasis+on+a+common+MHC+haplotype+associated+with+multiple+sclerosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+epistasis+on+a+common+MHC+haplotype+associated+with+multiple+sclerosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001593-Badano1" id="pone.0001593-Badano1"></a>Badano JL, Leitch CC, Ansley SJ, May-Simera H, Lawson S, et al.  (2006) Dissection of epistasis in oligogenic Bardet-Biedl syndrome. 439: 326–330.  <ul class="find" data-citedArticleID="1055290" data-doi="10.1038/nature04370"><li><a href="http://dx.doi.org/10.1038/nature04370" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissection+of+epistasis+in+oligogenic+Bardet-Biedl+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissection+of+epistasis+in+oligogenic+Bardet-Biedl+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001593-Martin1" id="pone.0001593-Martin1"></a>Martin MP, Gao XJ, Lee JH, Nelson GW, Detels R, et al.  (2002) Epistatic interaction between KIR3DS1 and HLA-B delays the progression to AIDS. Nature Genetics  31: 429–434.  <ul class="find" data-citedArticleID="1055318" data-doi="10.1038/ng934"><li><a href="http://dx.doi.org/10.1038/ng934" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epistatic+interaction+between+KIR3DS1+and+HLA-B+delays+the+progression+to+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epistatic+interaction+between+KIR3DS1+and+HLA-B+delays+the+progression+to+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001593-Kroymann1" id="pone.0001593-Kroymann1"></a>Kroymann J, Mitchell-Olds T (2005) Epistasis and balanced polymorphism influencing complex trait variation. Nature  435: 95–98.  <ul class="find" data-citedArticleID="1055312" data-doi="10.1038/nature03480"><li><a href="http://dx.doi.org/10.1038/nature03480" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epistasis+and+balanced+polymorphism+influencing+complex+trait+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epistasis+and+balanced+polymorphism+influencing+complex+trait+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001593-Carlborg1" id="pone.0001593-Carlborg1"></a>Carlborg O, Jacobsson L, Ahgren P, Siegel P, Andersson L (2006) Epistasis and the release of genetic variation during long-term selection. 38: 418–420.  <ul class="find" data-citedArticleID="1055294" data-doi="10.1038/ng1761"><li><a href="http://dx.doi.org/10.1038/ng1761" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epistasis+and+the+release+of+genetic+variation+during+long-term+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epistasis+and+the+release+of+genetic+variation+during+long-term+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001593-Marchini1" id="pone.0001593-Marchini1"></a>Marchini J, Donnelly P, Cardon LR (2005) Genome-wide strategies for detecting multiple loci that influence complex diseases. 37: 413–417.  <ul class="find" data-citedArticleID="1055316" data-doi="10.1038/ng1537"><li><a href="http://dx.doi.org/10.1038/ng1537" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+strategies+for+detecting+multiple+loci+that+influence+complex+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+strategies+for+detecting+multiple+loci+that+influence+complex+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001593-Evans1" id="pone.0001593-Evans1"></a>Evans DM, Marchini J, Morris AP, Cardon LR (2006) Two-Stage Two-Locus Models in Genome-Wide Association. PLoS Genetics  2: e157.  <ul class="find" data-citedArticleID="1055300" data-doi="10.1371/journal.pgen.0020157"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two-Stage+Two-Locus+Models+in+Genome-Wide+Association." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two-Stage+Two-Locus+Models+in+Genome-Wide+Association.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001593-Excoffier1" id="pone.0001593-Excoffier1"></a>Excoffier L, Slatkin M (1995) Maximum-likelihood estimation of molecular haplotype frequencies in a diploid population. Mol Biol Evol  12: 921–927.  <ul class="find" data-citedArticleID="1055302"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maximum-likelihood+estimation+of+molecular+haplotype+frequencies+in+a+diploid+population.&amp;auth=&amp;atitle=Maximum-likelihood+estimation+of+molecular+haplotype+frequencies+in+a+diploid+population." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum-likelihood+estimation+of+molecular+haplotype+frequencies+in+a+diploid+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum-likelihood+estimation+of+molecular+haplotype+frequencies+in+a+diploid+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001593-Weir1" id="pone.0001593-Weir1"></a>Weir BS, Cockerham CC (1984) Estimating F-Statistics for the Analysis of Population-Structure. Evolution  38: 1358–1370.  <ul class="find" data-citedArticleID="1055342" data-doi="10.2307/2408641"><li><a href="http://dx.doi.org/10.2307/2408641" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+F-Statistics+for+the+Analysis+of+Population-Structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+F-Statistics+for+the+Analysis+of+Population-Structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001593-Weir2" id="pone.0001593-Weir2"></a>Weir BS (1996) Population substructure. <em>Genetic data analysis II</em>. Sunderland, MA: Sinauer Associates. pp. 161–173.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.XML" value="79209"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.PDF" value="252760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g001.PNG_L" value="172501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g001.PNG_M" value="12793"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g001.PNG_S" value="4265"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g001.TIF" value="386900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g001.PNG_I" value="5156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.t001.PNG_L" value="129141"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.t001.PNG_M" value="76256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.t001.PNG_S" value="10742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.t001.TIF" value="628426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.t001.PNG_I" value="23747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g002.PNG_L" value="338620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g002.PNG_M" value="18223"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g002.PNG_S" value="5104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g002.TIF" value="594968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g002.PNG_I" value="6713"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g003.PNG_L" value="147019"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g003.PNG_M" value="14348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g003.PNG_S" value="3950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g003.TIF" value="572682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g003.PNG_I" value="5759"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g004.PNG_L" value="558860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g004.PNG_M" value="25168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g004.PNG_S" value="5488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g004.TIF" value="1224824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g004.PNG_I" value="9005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g005.PNG_L" value="367531"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g005.PNG_M" value="14624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g005.PNG_S" value="5157"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g005.TIF" value="663918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.g005.PNG_I" value="5500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.s001.ZIP" value="46604647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.s002.ZIP" value="23479"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.s003.ZIP" value="78309051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.s004.ZIP" value="43524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001593.s005.TIF" value="487288"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001593&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001593" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001593&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001593&volume=&issue=&title=Genome-Wide Prediction of Functional Gene-Gene Interactions Inferred from Patterns of Genetic Differentiation in Mice and Men&author_name=Zolt%C3%A1n%20Bochdanovits%2C%20David%20Sondervan%2C%20Sophie%20Perillous%2C%20Toos%20van%20Beijsterveldt%2C%20Dorret%20Boomsma%2C%20Peter%20Heutink&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001593" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001593" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001593" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001593&amp;t=Genome-Wide%20Prediction%20of%20Functional%20Gene-Gene%20Interactions%20Inferred%20from%20Patterns%20of%20Genetic%20Differentiation%20in%20Mice%20and%20Men" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001593&title=Genome-Wide%20Prediction%20of%20Functional%20Gene-Gene%20Interactions%20Inferred%20from%20Patterns%20of%20Genetic%20Differentiation%20in%20Mice%20and%20Men&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001593&amp;title=Genome-Wide%20Prediction%20of%20Functional%20Gene-Gene%20Interactions%20Inferred%20from%20Patterns%20of%20Genetic%20Differentiation%20in%20Mice%20and%20Men" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001593" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001593&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genome-Wide Prediction of Functional Gene-Gene Interactions Inferred from Patterns of Genetic Differentiation in Mice and Men';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001593';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genome-Wide%20Prediction%20of%20Functional%20Gene-Gene%20Interactions%20Inferred%20from%20Patterns%20of%20Genetic%20Differentiation%20in%20Mice%20and%20Men http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001593" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001593" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Complex+traits%22" title="Search for articles in the subject area:'Complex traits'"><div class="flagText">Complex traits</div></a>
              <div data-categoryid="20863" data-articleid="26012"
                   data-categoryname="Complex traits"
                   class="flagImage" title="Flag 'Complex traits' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="26012"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+complexity%22" title="Search for articles in the subject area:'Genome complexity'"><div class="flagText">Genome complexity</div></a>
              <div data-categoryid="34139" data-articleid="26012"
                   data-categoryname="Genome complexity"
                   class="flagImage" title="Flag 'Genome complexity' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+genetics%22" title="Search for articles in the subject area:'Human genetics'"><div class="flagText">Human genetics</div></a>
              <div data-categoryid="17347" data-articleid="26012"
                   data-categoryname="Human genetics"
                   class="flagImage" title="Flag 'Human genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="26012"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="26012"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="26012"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Variant+genotypes%22" title="Search for articles in the subject area:'Variant genotypes'"><div class="flagText">Variant genotypes</div></a>
              <div data-categoryid="40547" data-articleid="26012"
                   data-categoryname="Variant genotypes"
                   class="flagImage" title="Flag 'Variant genotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1831'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1332'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6528&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=7427">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
