

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Patterns of Selection in Anti-Malarial Immune Genes in Malaria Vectors: Evidence for Adaptive Evolution in LRIM1 in Anopheles arabiensis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000793"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000793" />

    <meta name="citation_title" content="Patterns of Selection in Anti-Malarial Immune Genes in Malaria Vectors: Evidence for Adaptive Evolution in LRIM1 in Anopheles arabiensis"/>
    <meta itemprop="name" content="Patterns of Selection in Anti-Malarial Immune Genes in Malaria Vectors: Evidence for Adaptive Evolution in LRIM1 in Anopheles arabiensis"/>

      <meta name="citation_author" content="Michel A. Slotman"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Aristeidis Parmakelis"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Jonathon C. Marshall"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Parfait H. Awono-Ambene"/>
            <meta name="citation_author_institution" content="Organisation de Coordination pour la Lutte Contre les Endémies en Afrique Centrale, Yaoundé, Cameroon"/>
      <meta name="citation_author" content="Christophe Antonio-Nkondjo"/>
            <meta name="citation_author_institution" content="Organisation de Coordination pour la Lutte Contre les Endémies en Afrique Centrale, Yaoundé, Cameroon"/>
      <meta name="citation_author" content="Frederic Simard"/>
            <meta name="citation_author_institution" content="Organisation de Coordination pour la Lutte Contre les Endémies en Afrique Centrale, Yaoundé, Cameroon"/>
            <meta name="citation_author_institution" content="Institute de Recherche pour le Développement, Yaoundé, Cameroon"/>
      <meta name="citation_author" content="Adalgisa Caccone"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Jeffrey R. Powell"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut, United States of America"/>

    <meta name="citation_date" content="2007/8/29"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000793.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e793"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The intolerable burden of malaria: A new look at the numbers.; citation_author=JC Breman; citation_author=A Egan; citation_author=GT Teusch; citation_journal_title=Am J Trop Med Hyg; citation_volume=64s; citation_number=1; citation_pages=iv-vii; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of the malaria mosquito Anopheles gambiae.; citation_author=R Holt; citation_author=GM Subramanian; citation_author=A Halpern; citation_author=G Sutton; citation_author=R Charlab; citation_journal_title=Science; citation_volume=298; citation_number=2; citation_pages=129-148; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Natural malaria infection in Anopheles gambiae is regulated by a single genomic control region.; citation_author=MM Riehle; citation_author=K Markianos; citation_author=O Niaré; citation_author=J Xu; citation_author=J Li; citation_journal_title=Science; citation_volume=312; citation_number=3; citation_pages=577-579; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ectopic expression of a cecropin transgene in the human malaria vector mosquito Anopheles gambiae (Diptera: Culicidae): Effects on susceptibility to Plasmodium.; citation_author=W Kim; citation_author=H Koo; citation_author=AM Richman; citation_author=D Seeley; citation_author=J Vizoli; citation_journal_title=J Med Entomol; citation_volume=41; citation_number=4; citation_pages=447-455; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Immune response of Anopheles gambiae to the early sporogonic stages of the human malaria parasite Plasmodium falciparum.; citation_author=R Tahar; citation_author=C Boudin; citation_author=I Thiery; citation_author=C Bourgouin; citation_journal_title=The EMBO journal; citation_volume=21; citation_number=5; citation_pages=6673-6680; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Effects of mosquito genes on Plasmodium development.; citation_author=MA Osta; citation_author=GK Christophides; citation_author=FC Kafatos; citation_journal_title=Science; citation_volume=303; citation_number=6; citation_pages=2030-2032; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Anopheles gambiae immune responses to human and rodent Plasmodium parasite species.; citation_author=Y Dong; citation_author=R Aguilar; citation_author=Z Xi; citation_author=E Warr; citation_author=E Mongin; citation_journal_title=PLoS Pathol; citation_volume=2; citation_number=7; citation_pages=513-525; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Functional genomic analysis of midgut epithelial responses in Anopheles during Plasmodium invasion.; citation_author=D Vlachou; citation_author=T Schlegelmilch; citation_author=GK Christophides; citation_author=FC Kafatos; citation_journal_title=Current Biol; citation_volume=15; citation_number=8; citation_pages=1185-1195; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The evolution of immune-related genes from disease carrying mosquitoes: diversity in a peptidoglycan- and a thioester-recognizing protein.; citation_author=TJ Little; citation_author=N Cobbe; citation_journal_title=Insect Mol Biol; citation_volume=14; citation_number=9; citation_pages=599-605; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Polymorphism at the defensin gene in the Anopheles gambiae complex: testing different selection hypotheses.; citation_author=F Simard; citation_author=M Licht; citation_author=NJ Besansky; citation_author=T Lehmann; citation_journal_title=Infect Genet Evol; citation_volume=7; citation_number=10; citation_pages=285-292; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=Z Yang; citation_number=11; citation_pages=357; citation_date=2006; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Natural selection drives Drosophila immune system evolution.; citation_author=TA Schlenke; citation_author=DJ Begun; citation_journal_title=Genetics; citation_volume=164; citation_number=12; citation_pages=1471-1480; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=First field trial of an immunoradiometric assay for the detection of malaria sporozoites in mosquitoes.; citation_author=FH Collins; citation_author=F Zavala; citation_author=PM Graves; citation_author=AH Cochrane; citation_author=RW Gwadz; citation_journal_title=Am J Trop Med Hyg; citation_volume=33; citation_number=13; citation_pages=538-543; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Why is the effect of malaria parasites on mosquito survival still unresolved?; citation_author=HM Ferguson; citation_author=AF Read; citation_journal_title=Trends Parasitol; citation_volume=18; citation_number=14; citation_pages=256-261; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Annual Plasmodium falciparum entomological inoculation rates (EIR) across Africa: literature survey, Internet access and review.; citation_author=SI Hay; citation_author=DJ Rogers; citation_author=JF Toomer; citation_author=RW Snow; citation_journal_title=Trans R Soc Trop Med Hyg; citation_volume=94; citation_number=15; citation_pages=113-127; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cloning and analysis of a cecropin gene from the malaria vector mosquito, Anopheles gambiae.; citation_author=J Vizoli; citation_author=P Bulet; citation_author=M Charlet; citation_author=C Lowenberger; citation_author=C Blass; citation_journal_title=Insect Mol Biol; citation_volume=9; citation_number=16; citation_pages=75-84; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular immune response of the mosquito Anopheles gambiae to bacteria and malaria parasites.; citation_author=G Dimopoulos; citation_author=A Richman; citation_author=H-M Müller; citation_author=FC Kafatos; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=17; citation_pages=11508-11513; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Anopheles gambiae immune response to Sephadex beads: involvement of anti-Plasmodium factors in regulating melanization.; citation_author=E Warr; citation_author=L Lambrechts; citation_author=JC Koella; citation_author=C Bourgouin; citation_author=G Dimopoulos; citation_journal_title=Insect Biochem Mol Biol; citation_volume=36; citation_number=18; citation_pages=769-778; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level.; citation_author=J Zhang; citation_author=R Nielsen; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=19; citation_pages=2472-2479; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bayes empirical Bayes inference of amino acid sites under positive selection.; citation_author=Z Yang; citation_author=WSW Wong; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=20; citation_pages=1107-1118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Adaptive protein evolution at the Adh locus in Drosophila.; citation_author=JH MacDonald; citation_author=M Kreitman; citation_journal_title=Nature; citation_volume=351; citation_number=21; citation_pages=652-654; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Evidence for recent population expansion in the evolutionary history of the malaria vectors Anopheles arabiensis and Anopheles gambiae.; citation_author=MJ Donelly; citation_author=MC Licht; citation_author=T Lehmann; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=22; citation_pages=1353-1364; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Population structure of the malaria mosquito Anopheles arabiensis across Nigeria suggests range expansion.; citation_author=DY Onyabe; citation_author=JE Conn; citation_journal_title=Mol Ecol; citation_volume=10; citation_number=23; citation_pages=2577-2591; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Semipermeable species boundaries between Anopheles gambiae and Anopheles arabiensis: Evidence from multilocus DNA sequence variation.; citation_author=NJ Besansky; citation_author=J Krzywinski; citation_author=T Lehman; citation_author=F Simard; citation_author=M Kern; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=19; citation_number=24; citation_pages=10818-10823; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of the Anopheles gambiae complex suggests genetic introgression between principle malaria vectors.; citation_author=NJ Besansky; citation_author=JR Powell; citation_author=A Caccone; citation_author=DM Hamm; citation_author=JA Scott; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=25; citation_pages=6885-6888; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Differential introgression of chromosomal regions between Anopheles gambiae and An. arabiensis.; citation_author=MA Slotman; citation_author=A della Torre; citation_author=M Calzetta; citation_author=JR Powell; citation_journal_title=Am J Trop Med Hyg; citation_volume=73; citation_number=26; citation_pages=326-335; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structural principles of Leucine-Rich Repeat (LRR) Proteins.; citation_author=P Enkhbayar; citation_author=M Kamiya; citation_author=M Osaki; citation_author=T Matsumoto; citation_author=N Matsushima; citation_journal_title=Proteins; citation_volume=54; citation_number=27; citation_pages=394-403; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Anopheles and Plasmodium: from laboratory models to natural systems in the field.; citation_author=A Cohuet; citation_author=M Osta; citation_author=I Morlais; citation_author=PH Awono-Ambene; citation_author=K Michel; citation_journal_title=EMBO Reports; citation_volume=7; citation_number=28; citation_pages=1285-1289; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Host genotype by parasite genotype interactions underlying the resistance of anopheline mosquitoes to Plasmodium falciparum.; citation_author=L Lambrechts; citation_author=J Halbert; citation_author=P Durand; citation_author=LC Gouagna; citation_author=JC Koella; citation_journal_title=Malar J; citation_volume=4; citation_number=29; citation_pages=3; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Arms races between and within species.; citation_author=R Dawkins; citation_author=JR Krebs; citation_journal_title=Proc R Soc Lond B Biol Sci; citation_volume=205; citation_number=30; citation_pages=489-511; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Simultaneous identification of species and molecular forms of the Anopheles gambiae comples by PCR-RFLP.; citation_author=C Fanello; citation_author=F Santolamazza; citation_author=A della Torre; citation_journal_title=Med Vet Entomol; citation_volume=16; citation_number=31; citation_pages=461-464; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A species-specific PCR for the identification of the malaria vector Anopheles bambae.; citation_author=NJ Besansky; citation_author=FH Collins; citation_author=H Towson; citation_journal_title=Ann Trop Med Parasitol; citation_volume=100; citation_number=32; citation_pages=277-280; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular karyotyping of the 2La inversion in Anopheles gambiae.; citation_author=BJ White; citation_author=F Santolamazza; citation_author=L Kamau; citation_author=M Pombi; citation_author=O Grushko; citation_journal_title=Am J Trop Med Hyg; citation_volume=76; citation_number=33; citation_pages=334-339; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers.; citation_author=S Rozen; citation_author=HJ Skaletsky; citation_number=34; citation_pages=365-38; citation_date=2000; citation_publisher=Humana Press; " />
      <meta name="citation_reference" content="citation_title=MEGA3: integrated software for molecular evolutionary genetics analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Briefings in Bioinformatics; citation_volume=5; citation_number=35; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=DnaSP, DNA polymorphism analyses by the coalescent and other methods.; citation_author=J Rozas; citation_author=JC Sánchez-DelBarrio; citation_author=X Messeguer; citation_author=R Rozas; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=36; citation_pages=2496-2497; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=37; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Modeltest: testing the model of DNA substitution.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=38; citation_pages=817-818; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000793" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Patterns of Selection in Anti-Malarial Immune Genes in Malaria Vectors: Evidence for Adaptive Evolution in LRIM1 in Anopheles arabiensis"/>
    <meta name="twitter:description" content="BackgroundCo-evolution between Plasmodium species and its vectors may result in adaptive changes in genes that are crucial components of the vector's defense against the pathogen. By analyzing which genes show evidence of positive selection in malaria vectors, but not in closely related non-vectors, we can identify genes that are crucial for the mosquito's resistance against Plasmodium.Methodology/Principle FindingsWe investigated genetic variation of three anti-malarial genes; CEC1, GNBP-B1 and LRIM1, in both vector and non-vector species of the Anopheles gambiae complex. Whereas little protein differentiation was observed between species in CEC1 and GNBP-B1, McDonald-Kreitman and maximum likelihood tests of positive selection show that LRIM1 underwent adaptive evolution in a primary malaria vector; An. arabiensis. In particular, two adjacent codons show clear signs of adaptation by having accumulated three out of four replacement substitutions. Furthermore, our data indicate that this LRIM1 allele has introgressed from An. arabiensis into the other main malaria vector An. gambiae.Conclusions/SignificanceAlthough no evidence exists to link the adaptation of LRIM1 to P. falciparum infection, an adaptive response of a known anti-malarial gene in a primary malaria vector is intriguing, and may suggest that this gene could play a role in Plasmodium resistance in An. arabiensis. If so, our data also predicts that LRIM1 alleles in An. gambiae vary in their level of resistance against P. falciparum."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000793.g002"/>

  <meta property="og:title" content="Patterns of Selection in Anti-Malarial Immune Genes in Malaria Vectors: Evidence for Adaptive Evolution in LRIM1 in Anopheles arabiensis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000793" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9501'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2165'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4024&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000793">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000793" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000793&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Patterns of Selection in Anti-Malarial Immune Genes in Malaria Vectors: Evidence for Adaptive Evolution in <i>LRIM1</i> in <i>Anopheles arabiensis</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michel A. Slotman
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:michel.slotman@yale.edu">michel.slotman@yale.edu</a></p>

                <p>Affiliation:
                  Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Aristeidis Parmakelis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jonathon C. Marshall, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut, United States of America
                </p>

                <p>
                    Current address: Department of Biology, Southern Utah University, Cedar City, Utah, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Parfait H. Awono-Ambene, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Organisation de Coordination pour la Lutte Contre les Endémies en Afrique Centrale, Yaoundé, Cameroon
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christophe Antonio-Nkondjo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Organisation de Coordination pour la Lutte Contre les Endémies en Afrique Centrale, Yaoundé, Cameroon
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frederic Simard, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Organisation de Coordination pour la Lutte Contre les Endémies en Afrique Centrale, Yaoundé, Cameroon, 
                  Institute de Recherche pour le Développement, Yaoundé, Cameroon
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Adalgisa Caccone, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jeffrey R. Powell
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Ecology and Evolutionary Biology, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 29, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000793</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000793" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000793" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000793" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000793" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000793-t001" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000793-t002" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000793-t003" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000793-t004" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000793-g001" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000793-t005" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000793-g002" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000793">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000793" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Co-evolution between <em>Plasmodium</em> species and its vectors may result in adaptive changes in genes that are crucial components of the vector's defense against the pathogen. By analyzing which genes show evidence of positive selection in malaria vectors, but not in closely related non-vectors, we can identify genes that are crucial for the mosquito's resistance against <em>Plasmodium.</em></p>


<h3>Methodology/Principle Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We investigated genetic variation of three anti-malarial genes; <em>CEC1, GNBP-B1</em> and <em>LRIM1,</em> in both vector and non-vector species of the <em>Anopheles gambiae</em> complex. Whereas little protein differentiation was observed between species in <em>CEC1</em> and <em>GNBP-B1,</em> McDonald-Kreitman and maximum likelihood tests of positive selection show that <em>LRIM1</em> underwent adaptive evolution in a primary malaria vector; <em>An. arabiensis.</em> In particular, two adjacent codons show clear signs of adaptation by having accumulated three out of four replacement substitutions. Furthermore, our data indicate that this <em>LRIM1</em> allele has introgressed from <em>An. arabiensis</em> into the other main malaria vector <em>An. gambiae.</em></p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Although no evidence exists to link the adaptation of <em>LRIM1</em> to <em>P. falciparum</em> infection, an adaptive response of a known anti-malarial gene in a primary malaria vector is intriguing, and may suggest that this gene could play a role in <em>Plasmodium</em> resistance in <em>An. arabiensis.</em> If so, our data also predicts that <em>LRIM1</em> alleles in <em>An. gambiae</em> vary in their level of resistance against <em>P. falciparum</em>.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Slotman MA, Parmakelis A, Marshall JC, Awono-Ambene PH, Antonio-Nkondjo C, et al.  (2007) Patterns of Selection in Anti-Malarial Immune Genes in Malaria Vectors: Evidence for Adaptive Evolution in <em>LRIM1</em> in <em>Anopheles arabiensis</em>. PLoS ONE 2(8):
          e793.
            doi:10.1371/journal.pone.0000793</p><p><strong>Academic Editor: </strong>Leah Cowen, University of Toronto, Canada</p><p><strong>Received:</strong> June 29, 2007; <strong>Accepted:</strong> July 18, 2007; <strong>Published:</strong> August 29, 2007</p><p><strong>Copyright:</strong> © 2007 Slotman et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported by National Institutes of Health grant RO1 A1 046018 to JRP. Additionally, AP was supported by a Marie Curie Outgoing International Fellowship (Contract No. MOIF-CT-2006-021357).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Despite ongoing control efforts during the last decades, malaria remains one of the most deadly infectious diseases. The vast majority of its burden is carried by people on the African continent, where 1 to 2 million people die annually from this disease <a href="#pone.0000793-Breman1">[1]</a>. Current malaria control efforts are hampered by the spread of insecticide and drug resistance, which has inspired research programs aimed at the development and eventual release of genetically altered mosquitoes that would be resistant to <em>Plasmodium falciparum</em> transmission. The need to identify refractory genes for this effort has focused much attention on the immune system of malaria's main vector in Africa, <em>An. gambiae</em>. The completion of the <em>An. gambiae</em> genome <a href="#pone.0000793-Holt1">[2]</a> has greatly facilitated research in this direction, and various anti-malarial immunity genes have now been identified <a href="#pone.0000793-Riehle1">[<em>e.g.</em> 3]</a>–<a href="#pone.0000793-Osta1">[6]</a>. Additionally, two recent studies provided many candidate anti-malarial immune genes that are up-regulated in response to <em>Plasmodium</em> infection <a href="#pone.0000793-Dong1">[7]</a>, <a href="#pone.0000793-Vlachou1">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>So far little attention has been devoted to examining polymorphism of immunity genes in natural malaria vector populations <a href="#pone.0000793-Little1">[9]</a>, <a href="#pone.0000793-Simard1">[10]</a>. It is known however that molecules that are involved in interactions with pathogens, such as immune genes, are one of the major types of proteins on which positive selection has been demonstrated <a href="#pone.0000793-Yang1">[11]</a>, <a href="#pone.0000793-Schlenke1">[12]</a>. Presumably, this is because such genes are involved in co-evolution between hosts and pathogens. In the case of malaria, if <em>Plasmodium</em> infection affects the mosquito's fitness, we may expect the accumulation of adaptive amino acid substitutions in those anti-malarial genes that are crucial in specifically limiting <em>Plasmodium</em> infection in vector species, whereas such changes should not be found in closely related species that do not transmit malaria.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>That <em>An. gambiae</em> has in fact undergone an adaptive response to <em>P. falciparum</em> infection is suggested by several lines of evidence. First of all, <em>P. falciparum</em> goes through severe bottlenecks during its life cycle in this mosquito <a href="#pone.0000793-Collins1">[13]</a>, demonstrating that the mosquito immune system is limiting the <em>Plasmodium</em> infection. Furthermore, <em>P. berghei</em>, which is not transmitted naturally by <em>An. gambiae,</em> produces a much higher oocyst number in <em>An. gambiae</em> than its natural pathogen <em>P. falciparum.</em> In fact, in a review of studies estimating the fitness effect of <em>Plasmodium</em> infection on <em>Anopheles</em> species, reduced fitness was observed in 10 combinations of <em>Plasmodium</em> and <em>Anopheles</em> species that do not occur naturally, whereas in 10 natural combinations, including <em>An. gambiae</em> and <em>P. falciparum</em>, no fitness effects were observed <a href="#pone.0000793-Ferguson1">[14]</a>. This is an indication that <em>Anopheles</em> species have evolved to limit infections of the <em>Plasmodium</em> species they come into contact with. This is corroborated by the fact that the immune response to <em>P. falciparum</em> is specific, <em>i.e. An. gambiae</em> up-regulates different genes in response to infection with <em>P. falciparum</em> vs. <em>P. berghei</em> <a href="#pone.0000793-Tahar1">[5]</a>, <a href="#pone.0000793-Dong1">[7]</a>. Salivary gland infection rates of <em>P. falciparum</em> in <em>An. gambiae</em> and <em>An. arabiensis</em> are typically low, ranging between 3–9% <a href="#pone.0000793-Hay1">[15]</a>. This raises the question whether selection pressures on the mosquito immune system are strong enough to result in an adaptive response to <em>P. falciparum</em> infection. However, it should be kept in mind that the data summarized above indicate that the rate and intensity of <em>P. falciparum</em> infection is likely to have been much higher when <em>Anopheles</em> mosquitoes first came into contact with this pathogen.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p><em>An. gambiae</em> belongs to a complex of closely related species that includes another primary African malaria vector, <em>An. arabiensis</em>. Additionally it contains several species, <em>i.e. An. melas, An. merus</em> and <em>An. bwambae</em>, that occasionally transmit malaria locally, but do not have wide enough distributions to be considered important vectors. More importantly for the purpose of the present study, the <em>An. gambiae</em> complex also contains the highly zoophilic <em>An. quadriannulatus A</em> and <em>An. quadriannulatus B</em>, which are never or rarely exposed to the human-limited <em>P. falciparum.</em></p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In this study, we investigated patterns of polymorphism in three anti-malarial genes, <em>i.e. CEC1</em>, <em>GNBP-B1</em> and <em>LRIM1,</em> in six species of the <em>An. gambiae</em> complex. <em>CEC1</em> (ENSANGG00000009468, <a href="http://www.ensembl.org/Anopheles_gambiae">www.ensembl.org/Anopheles_gambiae</a>) is a cecropin gene whose expression in <em>An. gambiae</em> is induced by infection with bacteria and <em>Plasmodium berghei</em> <a href="#pone.0000793-Vizoli1">[16]</a>. Additionally, genetically modified <em>An. gambiae</em> that express <em>CEC1</em> 24 hours after a blood meal, showed a 60% reduction in the number of <em>P. berghei</em> oocysts <a href="#pone.0000793-Kim1">[4]</a>. <em>GNBP-B1</em> (ENSANGG00000015205) is a pattern recognition receptor whose expression is strongly upregulated in response to infection with both <em>P. berghei</em> <a href="#pone.0000793-Dimopoulos1">[17]</a>, and <em>P. falciparum</em> <a href="#pone.0000793-Tahar1">[5]</a>. <em>LRIM1</em> (ENSANGG00000010552) is a leucine-rich repeat immune protein that is an important plasmodium antagonist. This protein is up-regulated in response to infection with <em>P. berghei,</em> and silencing of this gene increases oocyst load 3.6-fold <a href="#pone.0000793-Osta1">[6]</a>. Furthermore, this gene has been implicated in the melanization reaction of parasites <a href="#pone.0000793-Warr1">[18]</a>. We performed various tests for positive selection on these anti-malarial genes in the two main vectors, <em>An. gambiae</em> and <em>An. arabiensis,</em> to examine if these genes show signs of an adaptive response that may implicate them in the co-evolution of the mosquito vector and <em>Plasmodium</em> pathogen. Whereas no evidence for positive selection was found in <em>CEC1</em> and <em>GNBP-B1</em>, our results clearly indicate that <em>LRIM1</em> underwent an adaptive response in the <em>An. arabiensis</em> lineage. Additionally our data also indicate that <em>LRIM1</em> has introgressed from <em>An. arabiensis</em> into <em>An. gambiae.</em></p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>The complete <em>CEC1</em> gene, consisting of 177 bp of coding sequence and two introns comprising a combined 90 bp, was amplified. A total of 186 alleles of this gene were obtained from six species of the <em>An. gambiae</em> complex, 66 of which were unique (<a href="#pone-0000793-t001">Table 1</a>, genbank accession nos EU073463–EU073527). Although several polymorphisms were shared between species, none of the alleles were. For the coding region, <em>Dxy,</em> the average number of nucleotide substitutions between alleles in different species, ranged from 0.829 to 2.54 (per 100 bp). Very few fixed differences were present between species, and in most comparisons no fixed non-synonymous differences were found (<a href="#pone-0000793-t002">Table 2</a>). Not surprisingly therefore, none of the McDonald-Kreitman tests indicated an excess of non-synonymous fixed differences between species. In particular, no fixed amino-acid changes were observed between the non-vector species <em>An. quadriannulatus A</em> and the two major malaria vectors <em>An. gambiae</em> and <em>An. arabiensis</em>.</p>
<div class="figure" id="pone-0000793-t001"><div class="img"><a name="pone-0000793-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000793.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000793.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Number of sampled alleles.</span></strong></p><span>doi:10.1371/journal.pone.0000793.t001</span></div><div class="figure" id="pone-0000793-t002"><div class="img"><a name="pone-0000793-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000793.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000793.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>MacDonald-Kreitman test on <em>CEC1.</em></span></strong></p><span>doi:10.1371/journal.pone.0000793.t002</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>For <em>GNBP-B1</em> a total of 38 alleles from six species, consisting of the complete 1188 bp coding sequence and 232 bp of intron sequence, were obtained (<a href="#pone-0000793-t001">Table 1</a>, genbank accession nos EU073426–EU073462). All of these alleles were unique, but some polymorphisms were shared between species. <em>Dxy</em> ranged from 0.723 to 2.5 (per 100 bp) for the coding region. Very few fixed replacement substitutions were observed between species (<a href="#pone-0000793-t003">Table 3</a>). Two of the McDonald-Kreitman tests were significant. However, both indicated an excess of non-synonymous polymorphisms, and in all comparisons the ratio of non-synonymous to synonymous substitutions was higher for polymorphisms than for fixed differences. Between the non-vector species <em>An. quadriannulatus A</em> and the malaria vectors <em>An. arabiensis</em> and <em>An. gambiae,</em> only a single replacement substitution was observed.</p>
<div class="figure" id="pone-0000793-t003"><div class="img"><a name="pone-0000793-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000793.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000793.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>MacDonald-Kreitman test on <em>GNBP-B1.</em></span></strong></p><span>doi:10.1371/journal.pone.0000793.t003</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>For <em>LRIM1</em> we sequenced 858 bp that were thought to represent a single exon constituting the entire gene. However, in the most recent release of the Ensembl <em>An. gambiae</em> genome (release 45) the annotation of this gene was altered, and it is now thought that these 858 bp represent about half of the coding sequence of <em>LRIM1</em>. We obtained a total of 138 alleles from six species, of which 108 were unique (<a href="#pone-0000793-t001">Table 1</a>, genbank accession nos EU073528–EU073597). As in the other two genes investigated here, polymorphisms were shared between species, but alleles were not. <em>Dxy</em> ranged from 1.03 to 3.06 (per 100 bp) between species. In contrast to <em>CEC1</em> and <em>GNBP-B1</em> however, McDonald-Kreitman tests of positive selection indicated a significant excess of fixed non-synonymous differences between <em>An. arabiensis</em> and <em>An. quadriannulatus A, An. merus</em> as well as <em>An. bwambae</em> (<a href="#pone-0000793-t004">Table 4</a>). The fact that all three comparisons involve <em>An. arabiensis,</em> suggests this lineage underwent more non-synonymous substitutions than expected under a neutral model.</p>
<div class="figure" id="pone-0000793-t004"><div class="img"><a name="pone-0000793-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000793.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000793.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>MacDonald-Kreitman test on <em>LRIM1.</em></span></strong></p><span>doi:10.1371/journal.pone.0000793.t004</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>Surprisingly no fixed differences were present between <em>An. gambiae</em> and three other species; <em>An. arabiensis, An. bwambae</em> and <em>An</em>. <em>quadriannulatus A.</em> Our phylogenetic analysis showed that alleles from <em>An. gambiae</em> did not form a single cluster but were interspersed across the tree (<a href="#pone.0000793.s001">Figure S1</a>, supporting information). In particular, several “<em>arabiensis</em>-like” <em>An. gambiae</em> alleles clustered with <em>An. arabiensis</em>, far removed from the majority of <em>An. gambiae</em> sequences. When <em>An. gambiae</em> was removed from the analysis (<a href="#pone-0000793-g001">Figure 1</a>), <em>An. arabiensis, An. melas</em> and <em>An. merus</em> formed monophyletic groups with posterior probabilities of 0.99 and higher. <em>An. bwambae</em> formed a paraphyletic group containing the monophyletic <em>An. quadriannulatus A</em>. Since the <em>An. gambiae</em> alleles did not form a single cluster, it was not possible to test for positive selection along a single branch leading to this species. However, we did test for positive selection along the branch leading to the other major malaria vector, <em>An. arabiensis.</em> To increase the power of this analysis, the length of this branch, henceforth referred to as the foreground branch, was maximized by excluding all <em>An. gambiae</em> alleles from the analysis.</p>
<div class="figure" id="pone-0000793-g001"><div class="img"><a name="pone-0000793-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000793.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000793.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Bayesian tree (unrooted) of <em>LRIM1</em> from five species of the <em>An. gambiae</em> complex.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Posterior probabilities ≥0.99 are indicated by *. Number of non-synonymous/synonymous substitutions are indicated above or on the left side of the branches. Estimated ω values are placed below or on the right side of the branches and are underlined. The foreground branch for the maximum likelihood tests of positive selection is indicated by a double line. For a more detailed phylogeny, including all posterior probabilities above 50% and sample names, see <a href="#pone.0000793.s002">Figure S2</a> (supporting materials).</p>
<span>doi:10.1371/journal.pone.0000793.g001</span></div><a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>Due to the absence of synonymous substitutions the ω value, <em>i.e.</em> the ratio of non-synonymous to synonymous substitutions or dN/dS, along the foreground branch could not be estimated in our PAML analyses and is indicated as ∞ (<a href="#pone-0000793-t005">Table 5</a>). The branch test did not indicate that ω along the foreground branch is significantly larger than 1 (model 2 <sub>free ω</sub> vs. model 2 <sub>ω = 1</sub>). However, a branch test did show that it is significantly larger than ω along the background branches (p = 0.011, model 2 <sub>free ω</sub> vs. model 0). Values of ω along the branches leading to <em>An. melas, An. merus</em> and <em>An. bwambae</em>/<em>An. quadriannulatus A</em> were estimated to be 0.299, 0.298 and 0.147 respectively (<a href="#pone-0000793-g001">Figure 1</a>), indicating purifying selection along these lineages. This confirms that the excess of non-synonymous substitutions detected by the McDonald-Kreitman tests mainly occurred along the lineage leading to <em>An. arabiensis.</em> Interestingly, the branch separating <em>An. arabiensis</em> and <em>An. melas</em> from the other species also lacked synonymous, but not non-synonymous substitutions, possibly indicating some positive selection along this branch.</p>
<div class="figure" id="pone-0000793-t005"><div class="img"><a name="pone-0000793-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000793.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000793.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Likelihood Ratio Test for positive selection on <em>LRIM1</em> in <em>An. arabiensis.</em></span></strong></p><span>doi:10.1371/journal.pone.0000793.t005</span></div><a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>The branch tests applied above use an ω value averaged across the entire gene. This severely diminishes the power of the analysis because at least some parts of each gene are expected to be under purifying selection. A more powerful test for positive selection is provided by branch-site models, which test if certain codons are under selection in the foreground branch <a href="#pone.0000793-Zhang1">[19]</a>. When implemented on our data set (model A vs. model A1), the branch-site test provided strong support for positive selection in the <em>An. arabiensis</em> lineage with p = 0.005 (<a href="#pone-0000793-t005">Table 5</a>). According to the Bayes Empirical Bayes (BEB) analysis <a href="#pone.0000793-Yang2">[20]</a>, amino-acid positions 108 and 236 are under positive selection (i.e. ω&gt;1), with probabilities of 0.983 and 0.985, respectively. Position 235 has a high probability (0.933) of being under positive selection as well.</p>
<a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>A total of four non-synonymous substitutions occurred along the foreground branch (<a href="#pone-0000793-g001">Figure 1</a>). Three of these non-synonymous substitutions are within two adjacent codons (235 and 236), with codon 236 having two replacement substitutions. That is, regardless of the order in which the two nucleotide substitutions in codon 236 occurred, two subsequent amino-acid changes were the result. Additionally, a replacement substitution at position 234 is fixed in <em>An. melas,</em> is at high frequency in <em>An. arabiensis</em> (0.83) and is also found in one of the “<em>arabiensis</em>-like” <em>An. gambiae</em> alleles.</p>
<a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p><em>An. arabiensis</em> is fixed for nucleotide A at sites 704, 706 and 707 (codons 235 and 236). With the exception of <em>An. gambiae,</em> all other species are fixed for nucleotides T, G and G at these respective positions. The few “<em>arabiensis</em>-like” <em>An. gambiae</em> alleles also have the (AAA) arrangement. Positions 416 through 718, a 302 bp stretch, cluster these <em>An. gambiae</em> (AAA) alleles with <em>An. arabiensis</em> (<a href="#pone-0000793-g002">Figure 2</a>). Only positions below 328 and above 767 contain polymorphisms that group some of the <em>An. gambiae</em> (AAA) alleles with the rest of <em>An. gambiae.</em> Additionally, no fixed differences were found between <em>An. gambiae</em> and <em>An. arabiensis</em> anywhere in this gene.</p>
<div class="figure" id="pone-0000793-g002"><div class="img"><a name="pone-0000793-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000793" data-uri="info:doi/10.1371/journal.pone.0000793.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000793.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000793.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000793.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000793.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000793.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Shared polymorphism of <em>LRIM1</em> in <em>An. gambiae</em> (TGG), <em>An. gambiae</em> (AAA), and <em>An. arabiensis.</em></span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>Only sites beyond position 324 that favor the clustering of <em>An. gambiae</em> (AAA) alleles with either <em>An. gambiae</em> (TGG) or <em>An. arabiensis</em> are included.</p>
<span>doi:10.1371/journal.pone.0000793.g002</span></div><a id="article1.body1.sec2.p9" name="article1.body1.sec2.p9"></a><p><em>LRIM1</em> is located inside the <em>2La</em> inversion. Therefore we determined the karyotype of our <em>An. gambiae</em> samples with respect to the <em>2La</em> arrangement. Out of 32 specimens from Cameroon, three were <em>2La/+</em> heterozygotes, all of which were also heterozygous for the (AAA)/(TGG) alleles. The karyotype of one (AAA)/(TGG) heterozygote was not clear, as it produced a second band of unexpected size. The remaining four (AAA)/(TGG) heterozygotes, as well as all (TGG) homozygotes, carried the <em>2L+/+</em> karyotype. These findings confirm that the (AAA) allele is at very high frequency or even fixed in the <em>2La</em> inversion in Cameroon, and is present at very low frequency in the <em>2L+</em> arrangement (≈7%).</p>
<a id="article1.body1.sec2.p10" name="article1.body1.sec2.p10"></a><p>To examine if <em>LRIM1</em> in <em>An. arabiensis</em> showed signs of a recent selective sweep, a HKA test was performed by comparing the polymorphism/fixed differences ratio of <em>LRIM1</em> to CEC1 and GNBP-B1. A selective sweep reduces the amount of standing genetic variation within a species, as indicated by a relatively low ratio. However, no significant differences were found between the genes, and in fact, this ratio was considerably higher for <em>LRIM1</em> (20/20.2) as compared to <em>CEC1</em> and <em>GNBP-B1</em> (17/28.1).</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p><em>CEC1</em> and <em>GNBP-B1</em> did not show any signs of positive selection, and in particular, showed little or no differentiation between malaria vectors and the non-vector species, indicating that these genes are largely subject to purifying selection. In two of the species comparisons <em>GNBP-B1</em> showed a significant excess of non-synonymous polymorphisms. Some cloning error is expected to be present in the <em>GNBP-B1</em> data set. Since a majority of possible mutations are non-synonymous, random errors will bias the observed number of non-synonymous polymorphisms upward. However, the number of PCR errors in the data is not nearly high enough to explain the difference. Therefore, most likely purifying selection is responsible, with numerous slightly deleterious substitutions present at low frequency in populations, but which are prevented from going to fixation. In contrast, <em>LRIM1</em>, a gene with no known homologue in other organisms, shows clear signs of positive selection in <em>An. arabiensis.</em></p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>As pointed out by MacDonald and Kreitman, an excess of non-synonymous fixed differences between species may result from a much smaller population size in the past <a href="#pone.0000793-MacDonald1">[21]</a>. This would have allowed slightly deleterious mutations to go to fixation by drift, whereas in the current larger population most of these are removed by purifying selection. Evidence for population expansion has been reported for both <em>An. gambiae</em> and <em>An. arabiensis</em> <a href="#pone.0000793-Donelly1">[22]</a>, <a href="#pone.0000793-Onyabe1">[23]</a>. It is unlikely however that this could explain the excess of non-synonymous substitutions observed in <em>LRIM1</em> in <em>An. arabiensis</em>. First of all, a demographic explanation should affect all genes. As noted before, <em>GNBP-B1</em> has a higher ratio of non-synonymous/synonymous polymorphisms than fixed differences in most or all populations, including <em>An. arabiensis</em>. This indicates the presence of a relatively large number of slightly deleterious alleles in this gene, few or none of which became fixed in ancestral populations. This is contrary to the demographic explanation. More importantly however, it is extremely unlikely that three out of four fixed amino acid changes would occur in two adjacent positions in a 285 amino acid protein, if the random process of genetic drift were responsible.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Polymorphisms are shared between species in all three genes we examined, and there is no doubt some of this is due to the retention of ancestral polymorphism. However, the pattern of polymorphism observed in <em>LRIM1</em> provides strong evidence that the presence of “<em>arabiensis</em>-like” alleles in <em>An. gambiae</em> is caused by introgression and not by the retention of ancestral polymorphism. It is unlikely that recombination would not have broken down a linkage group of at least 352 bp, if it were maintained in the population for long time. Furthermore, the introgression hypothesis is supported by the complete absence of fixed differences between these species anywhere in the <em>LRIM1</em> gene.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Based on the shared polymorphisms between <em>An. gambiae</em> (AAA) and <em>An. arabiensis</em> (positions 416, 624 and 701), introgression has occurred multiple times, after which, according to the shared polymorphism between the <em>An. gambiae</em> (AAA) and (TGG) alleles (positions 324 and 327), these alleles recombined between position 327 and 416. The introgression of <em>LRIM1</em> from <em>An. arabiensis</em> into <em>An. gambiae</em> is also consistent with previous studies that have shown that introgression between these two species has occurred in the past <a href="#pone.0000793-Besansky1">[24]</a>, <a href="#pone.0000793-Besansky2">[25]</a>. Additionally, it has been shown through sequence analyses <a href="#pone.0000793-Besansky2">[25]</a>, as well as crossing experiments <a href="#pone.0000793-Slotman1">[26]</a>, that different chromosomes vary in their capacity for horizontal transfer between these two species. The <em>2<sup>nd</sup></em> chromosome, on which <em>LRIM1</em> is located, has been shown to transfer most readily between <em>An. gambiae</em> and <em>An. arabiensis</em>. <a href="#pone.0000793-Slotman1">[26]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Mosquitoes, like other organisms, encounter numerous pathogens during their life cycle, all of which could potentially exert selection pressure on the immune system. In fact, molecules that play a role in host-parasite interactions are one of the main groups of proteins on which positive selection has been demonstrated <a href="#pone.0000793-Yang1">[11]</a>. No direct evidence is available to show that <em>LRIM1</em> in <em>An. arabiensis</em> has evolved in direct response to malaria infection, but our observation that this anti-malarial gene shows distinct signs of positive selection in a primary malaria vector, but not in a species that does not transmit <em>Plasmodium</em>, is intriguing. This suggests that the observed adaptive evolution of <em>LRIM1</em> may have been the result of the infection of <em>An. arabiensis</em> with <em>Plasmodium</em>, with at least two adjacent amino acids playing a crucial role in this adaptation. If this is true, variation for <em>Plasmodium</em> resistance should be present at <em>LRIM1</em> in <em>An. gambiae</em>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p><em>LRIM1</em> contains a leucine-rich repeat (LRR), which in other LRR genes is crucial for the three-dimensional structure by folding it into an arc <a href="#pone.0000793-Enkhbayar1">[27]</a>. Little is known about the structure of <em>LRIM1</em>, but the two neighboring adaptive amino acids (<em>i.e.</em> 235 and 236) are located well outside the leucine-rich repeat region (positions 30–160), suggesting that they could have a more specific function.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Although <em>LRIM1</em> is known to play a role in suppressing <em>P. berghei</em> infection in <em>An. gambiae</em> <a href="#pone.0000793-Osta1">[6]</a>, a recent RNAi study failed to show an effect of <em>LRIM1</em> on <em>P. falciparum</em> infection in field-collected <em>An. gambiae</em> <a href="#pone.0000793-Cohuet1">[28]</a>, while an effect on <em>P. berghei</em> infection was confirmed. This could be because the action of <em>LRIM1</em> in <em>An. gambiae</em> is specific against <em>P. berghei.</em> However, it is also possible that only some <em>LRIM1</em> alleles suppress infection with <em>P. falciparum,</em> and these may even be specific for certain <em>P</em>. <em>falciparum</em> strains. Another study demonstrated the existence of such genotype by genotype interactions between <em>P. falciparum</em> and <em>An. gambiae</em>, by showing that no single strain of <em>P. falciparum</em> was best at infecting all of a set of iso-female <em>An.gambiae</em> lines <a href="#pone.0000793-Lambrechts1">[29]</a>. Additionally, <em>LRIM1</em> is located inside the <em>2La</em> inversion. While <em>An. arabiensis</em> is fixed for this <em>2La</em> arrangement, <em>An. gambiae</em> is <em>2La/+</em> polymorphic. Since <em>LRIM1</em> alleles introgressed from <em>An. arabiensis</em> into <em>An. gambiae,</em> we may expect that these <em>An. gambiae</em> (AAA) alleles are mostly found in the <em>2La</em> arrangement. Interestingly, the mosquitoes that failed to show an effect of <em>LRIM1</em> knockdown on <em>P. falciparum</em> infection <a href="#pone.0000793-Cohuet1">[28]</a> all carried the standard chromosome arrangement (<em>2L+</em>). Our molecular karyotyping of the <em>2La</em> inversion in our <em>An. gambiae</em> specimens shows that the (AAA) allele is indeed found at very high frequency in <em>2La</em> inversions, whereas it is present in very low frequency in <em>2L+</em> (≈7%).</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Pathogen-host co-evolution has mainly been considered in terms of an evolutionary arms race <a href="#pone.0000793-Dawkins1">[30]</a>. Under this model, the host continuously evolves to limit infection with the pathogen, which in turns evolves to evade host defenses. This is expected to lead to repeated selective sweeps, which leave a signature in the selected genes in the form of a low level of standing genetic variation. A comparison of the polymorphism to divergence ratio in <em>LRIM1</em> vs <em>GNBP-B1</em>/<em>CEC1</em> did not show a relatively low level of genetic variation in <em>LRIM1</em> in <em>An. arabiensis.</em> In fact, the relative level of polymorphism was higher in this gene than in <em>CEC1</em> and <em>GNBP-B1</em>. Therefore we have no indication that <em>LRIM1</em> in <em>An. arabiensis</em> is currently involved in an evolutionary arms race. This also implies that possible selective sweeps occurred long enough ago to allow mutation to regenerate polymorphism.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>The data presented here indicate that the anti-malarial gene <em>LRIM1</em> has undergone adaptive evolution in a primary malaria vector. This could be because this gene has evolved in response to <em>P. falciparum</em> infection in this species. If so, <em>LRIM1</em> is expected to play a role in the resistance of <em>An. arabiensis</em> against <em>P. falciparum.</em> So far the immune system of this mosquito species has not yet been investigated, and our data suggest the possibility that a knockdown of <em>LRIM1</em> will enhance infections of <em>P. falciparum</em> in <em>An. arabiensis.</em> If <em>LRIM1</em> did indeed evolve in response to <em>P. falciparum</em> infection in <em>An. arabiensis</em>, this gene also deserves further study in <em>An. gambiae</em>, in particular with respect to potential variation in resistance between the two major alleles found in this species.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Mosquito sampling</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Adult females of <em>An. gambiae</em> were collected from the villages of Mbebé and Nyabessan, Cameroon in Dec. 2005. <em>An. gambiae</em> from Mali were collected from Banambani in 2000. Adult <em>An. arabiensis</em> females from Cameroon were collected from Kousseri in Dec 2005. Adult <em>An. melas</em> were collected in Ipono, Cameroon, Dec. 2005. Larvae of <em>An. gambiae, An. arabiensis</em> and <em>An. bwambae</em> from Bwamba county, Uganda (2004) were kindly provided by Ralph Harbach. DNA extractions of <em>An. merus</em> from Furvela, Mozambique (2001 and 2003) were kindly provided by David O'Brochta. <em>An. quadriannulatus A</em> from Kruger National Park, South Africa, were kindly provided by Anton Cornel. Sample sizes for each gene and species are represented in <a href="#pone-0000793-t001">Table 1</a>.</p>


<h4>DNA methods</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>DNA was extracted using the DNeasy tissue kit (Qiagen). Species and molecular form diagnostics were performed following Fanello <em>et al.</em> <a href="#pone.0000793-Fanello1">[31]</a> and Besansky <em>et al.</em> <a href="#pone.0000793-Besansky3">[32]</a>. All <em>An. gambiae</em> specimens belonged to the S molecular form. Molecular identification of <em>2La</em> karyotypes was performed following White <em>et al.</em> <a href="#pone.0000793-White1">[33]</a>. Primers to amplify <em>CEC1, GNBP-B1</em> and <em>LRIM1</em> were designed using Primer3 <a href="#pone.0000793-Rozen1">[34]</a> based on the <em>An. gambiae</em> genome and anneal to the flanking or non-transcribed regions of the genes <a href="#pone.0000793-Riehle1">[3]</a>. PCR of <em>CEC-A</em> and <em>LRIM1</em> was performed using Amplitaq Gold polymerase (Perkin Elmer) using respectively the following primer pairs CECin1 (<span class="gene">GTTAGCAGAGCCGTCGTCTT</span>)/CECin12 (<span class="gene">ACAGTCGGTTCAAAGCGTTC</span>) and LRIM1in6 (<span class="gene">AGGTAACGGACAGCAGCCTA</span>)/LRIM1in9 (<span class="gene">GTCCGGTACTGCTCCTTGAG</span>). The following program was used for PCR amplification of <em>CEC1</em> and <em>LRIM1</em>; 2 min at 94°, 35 cycles of 30 sec at 94°, 30 sec at 52° and 45 sec at 72°, followed by 20 min at 72°. PCR products were excised from an agarose gel and purified using the Gel Purification Kit (Qiagen) and submitted for direct sequencing. A subset of the sequences from individuals heterozygous for two or more positions were amplified again and cloned using the TOPO-TA cloning kit (Invitrogen). Individuals were selected for cloning such that all observed polymorphic sites were represented in the final data set. From each individual a single colony was sequenced. PCR of <em>GNBP-B1</em> was performed using Platinum High Fidelity Taq (Invitrogen) with the primer pair GNBPin1 (<span class="gene">GTTTGGTAGGGGACGAATGA</span>) /GNBPIN20 (<span class="gene">GCGCTTTCAGTGGTTTGTTT</span>) using the following program: 2 min at 94°, 35 cycles of 30 sec at 94°, 30 sec at 52° and 90 sec at 72°, followed by 20 min at 72°. Direct sequencing of the PCR product of <em>GNBP-B1</em> was not possible in many cases because of the presence of indels. Therefore, PCR products of this gene were cloned and sequenced as outlined above. However, nine sequences were produced through direct sequencing, allowing for an estimation of the PCR/cloning error. Sequencing was performed on an ABI 3730 Genetic Analyzer using Big Dye v 3.1 (Applied Biosystems )</p>


<h4>PCR error</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Based on a comparison between direct sequencing and plasmid sequencing, the PCR/cloning error using Amplitaq Gold was estimated to be approximately 1.5 per 1000 bp. However, because all <em>LRIM1</em> and <em>CEC1</em> samples that were cloned were also sequenced directly, we were able to derive both alleles from each individual while removing PCR/cloning errors. The PCR error in the <em>GNBP-B1</em> sequences amplified using the proof-reading polymerase was estimated to be 0.625 per 1000 bp. Therefore, each 1188 bp <em>GNBP-B1</em> allele for which no direct sequence was available is expected to have an average of 0.74 errors.</p>


<h4>Data analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>All sequences were aligned using MEGA3.1 <a href="#pone.0000793-Kumar1">[35]</a> and alignments were improved manually. Introns were included in the phylogeny reconstructions. For all other analyses the coding region was used. <em>Dxy</em> values were calculated using DnaSP 4.0 <a href="#pone.0000793-Rozas1">[36]</a>. This software was also used to perform McDonald-Kreitman tests <a href="#pone.0000793-MacDonald1">[21]</a>, using Fisher's exact test. The McDonald-Kreitman test compares the dN/dS ratio between species to within species and is based on the idea that substitutions under positive selection will go to fixation rapidly, and are therefore rarely observed as polymorphisms. However, they are present as fixed differences between species and an excess of replacement fixed differences is therefore an indication of positive selection.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Since few or no fixed differences were observed in <em>CEC1</em> and <em>GNBP-B1</em>, subsequent analyses were limited to <em>LRIM1</em>. Aimed at reducing the computational effort, a reduced <em>LRIM1</em> data set, containing 70 sequences, was used for phylogenetic analyses and maximum likelihood tests of positive selection. This data set was compiled in such a way that at every observed polymorphism and fixed difference, <em>i.e.</em> the relevant information for tests for maximum likelihood tests of positive selection, was retained. This reduced data set was used to construct 50% majority-rule consensus trees with MrBayes 3.1.2 <a href="#pone.0000793-Ronquist1">[37]</a>. Modeltest 3.7 <a href="#pone.0000793-Posada1">[38]</a> was used to determine the most appropriate nucleotide substitution model for our data set.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>Several <em>LRIM1</em> alleles from <em>An. gambiae</em> clustered within <em>An. arabiensis</em> (<a href="#pone.0000793.s001">Figure S1</a> supp. mat.). Therefore, phylogeny reconstruction was also performed excluding <em>An. gambiae</em> sequences. This inferred tree was used for maximum likelihood tests of positive selection along the branch leading to <em>An. arabiensis</em> in PAML3.15 and to estimate ω (<em>i.e.</em> dN/dS) along the major branches of the tree. Under the neutral model the relative number of synonymous and non-synonymous substitutions is expected to be 1. Under positive selection, amino acid substitutions are favored and ω&gt;1, whereas under purifying selection amino acid substitutions are prevented and ω&lt;1. The <em>An. arabiensis</em> lineage was designated as the foreground branch, <em>i.e.</em> the branch of interest, and model 2 <sub>free ω</sub> was compared to model 0 to test if ω along the foreground branch was significantly larger compared to the ω along the background branches, <em>i.e.</em> all other branches. Model 2 <sub>ω = 1</sub>, with the ω value fixed at 1 along foreground branch, was compared to model 2 <sub>free ω</sub> to test if ω along the foreground branch was significantly larger than 1. Model 1 was used to estimate ω along the central branches of the tree (<a href="#pone-0000793-g001">Figure 1</a>) and to infer the number of substitutions along these branches of the phylogeny. As an additional test for positive selection along the foreground branch, we used the more powerful branch-site test 2 by comparing model A and Model A1 <a href="#pone.0000793-Zhang1">[19]</a>. Bayes Empirical Bayes (BEB) analysis was used to identify positively selected codons in the foreground branch <a href="#pone.0000793-Yang2">[20]</a>.</p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>To test for a reduction in the polymorphism of <em>LRIM1</em> in <em>An. arabiensis,</em> an HKA test was performed in DnaSP 4.0, using eight <em>An. arabiensis</em> alleles for <em>CEC1</em> and <em>LRIM1</em>, as well as all eight <em>GNBP-B1</em> alleles from this species. The <em>CEC1</em> and <em>LRIM1</em> alleles were from the same individuals as the <em>GNBP-B1</em> sequences if possible, otherwise were randomly chosen from the same population. Seven <em>An. quadriannulatus A</em> alleles from each gene were used to calculate inter-specific divergence.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000793"></div><a name="pone.0000793.s001" id="pone.0000793.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000793.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Bayesian tree (unrooted) of <em>LRIM1</em> from six species of the <em>An.gambiae</em> complex. Posterior probabilities are indicated along branches. <em>An. gambiae</em> samples and <em>An. arabiensis</em> samples from Uganda, Madagascar and Mali are indicated by UG, MAD, and MAL respectively, remaining samples of these two species are from Cameroon.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000793.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.19 MB TIF)</p>
<a name="pone.0000793.s002" id="pone.0000793.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000793.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Bayesian tree (unrooted) of <em>LRIM1</em> in five species of the <em>An.gambiae</em> complex. Posterior probabilities are indicated along branches. <em>An. arabiensis</em> samples from Uganda, Madagascar and Mali are indicated by UG, MAD, and MAL respectively, with all remaining <em>An. arabiensis</em> samples originating from Cameroon.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000793.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.72 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are very grateful to Anton Cornel, Ralph Harbach and David O'Brochta for providing <em>An. quadriannulatus A, An. bwambae</em> and <em>An. merus</em> specimens, respectively. Additionally, we are grateful to the anonymous reviewers who provided many useful comments for improving earlier versions of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JP GC MS. Performed the experiments: MS. Analyzed the data: MS. Contributed reagents/materials/analysis tools: MS AP JM PA CN FS. Wrote the paper: MS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000793-Breman1" id="pone.0000793-Breman1"></a>Breman JC, Egan A, Teusch GT (2001) The intolerable burden of malaria: A new look at the numbers. Am J Trop Med Hyg  64s: iv–vii.  <ul class="find" data-citedArticleID="982754"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+intolerable+burden+of+malaria%3A+A+new+look+at+the+numbers.&amp;auth=&amp;atitle=The+intolerable+burden+of+malaria%3A+A+new+look+at+the+numbers." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+intolerable+burden+of+malaria%3A+A+new+look+at+the+numbers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+intolerable+burden+of+malaria%3A+A+new+look+at+the+numbers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000793-Holt1" id="pone.0000793-Holt1"></a>Holt R, Subramanian GM, Halpern A, Sutton G, Charlab R, et al.  (2002) The genome sequence of the malaria mosquito <em>Anopheles gambiae</em>. Science  298: 129–148.  <ul class="find" data-citedArticleID="982776" data-doi="10.1126/science.1076181"><li><a href="http://dx.doi.org/10.1126/science.1076181" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+the+malaria+mosquito+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+the+malaria+mosquito+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000793-Riehle1" id="pone.0000793-Riehle1"></a>Riehle MM, Markianos K, Niaré O, Xu J, Li J, et al.  (2006) Natural malaria infection in <em>Anopheles gambiae</em> is regulated by a single genomic control region. Science  312: 577–579.  <ul class="find" data-citedArticleID="982794" data-doi="10.1126/science.1124153"><li><a href="http://dx.doi.org/10.1126/science.1124153" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+malaria+infection+in+Anopheles+gambiae+is+regulated+by+a+single+genomic+control+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+malaria+infection+in+Anopheles+gambiae+is+regulated+by+a+single+genomic+control+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000793-Kim1" id="pone.0000793-Kim1"></a>Kim W, Koo H, Richman AM, Seeley D, Vizoli J, et al.  (2004) Ectopic expression of a cecropin transgene in the human malaria vector mosquito <em>Anopheles gambiae</em> (Diptera: Culicidae): Effects on susceptibility to <em>Plasmodium</em>. J Med Entomol  41: 447–455.  <ul class="find" data-citedArticleID="982778" data-doi="10.1603/0022-2585-41.3.447"><li><a href="http://dx.doi.org/10.1603/0022-2585-41.3.447" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ectopic+expression+of+a+cecropin+transgene+in+the+human+malaria+vector+mosquito+Anopheles+gambiae+%28Diptera%3A+Culicidae%29%3A+Effects+on+susceptibility+to+Plasmodium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ectopic+expression+of+a+cecropin+transgene+in+the+human+malaria+vector+mosquito+Anopheles+gambiae+%28Diptera%3A+Culicidae%29%3A+Effects+on+susceptibility+to+Plasmodium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000793-Tahar1" id="pone.0000793-Tahar1"></a>Tahar R, Boudin C, Thiery I, Bourgouin C (2002) Immune response of <em>Anopheles gambiae</em> to the early sporogonic stages of the human malaria parasite <em>Plasmodium falciparum</em>. The EMBO journal  21: 6673–6680.  <ul class="find" data-citedArticleID="982808" data-doi="10.1093/emboj/cdf664"><li><a href="http://dx.doi.org/10.1093/emboj/cdf664" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immune+response+of+Anopheles+gambiae+to+the+early+sporogonic+stages+of+the+human+malaria+parasite+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immune+response+of+Anopheles+gambiae+to+the+early+sporogonic+stages+of+the+human+malaria+parasite+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000793-Osta1" id="pone.0000793-Osta1"></a>Osta MA, Christophides GK, Kafatos FC (2004) Effects of mosquito genes on <em>Plasmodium</em> development. Science  303: 2030–2032.  <ul class="find" data-citedArticleID="982790" data-doi="10.1126/science.1091789"><li><a href="http://dx.doi.org/10.1126/science.1091789" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+mosquito+genes+on+Plasmodium+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+mosquito+genes+on+Plasmodium+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000793-Dong1" id="pone.0000793-Dong1"></a>Dong Y, Aguilar R, Xi Z, Warr E, Mongin E, et al.  (2006) <em>Anopheles gambiae</em> immune responses to human and rodent <em>Plasmodium</em> parasite species. PLoS Pathol  2: 513–525.  <ul class="find" data-citedArticleID="982766" data-doi="10.1371/journal.ppat.0020052"><li><a href="http://dx.doi.org/10.1371/journal.ppat.0020052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anopheles+gambiae+immune+responses+to+human+and+rodent+Plasmodium+parasite+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anopheles+gambiae+immune+responses+to+human+and+rodent+Plasmodium+parasite+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000793-Vlachou1" id="pone.0000793-Vlachou1"></a>Vlachou D, Schlegelmilch T, Christophides GK, Kafatos FC (2005) Functional genomic analysis of midgut epithelial responses in <em>Anopheles</em> during <em>Plasmodium</em> invasion. Current Biol  15: 1185–1195.  <ul class="find" data-citedArticleID="982812" data-doi="10.1016/j.cub.2005.06.044"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.06.044" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+genomic+analysis+of+midgut+epithelial+responses+in+Anopheles+during+Plasmodium+invasion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+genomic+analysis+of+midgut+epithelial+responses+in+Anopheles+during+Plasmodium+invasion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000793-Little1" id="pone.0000793-Little1"></a>Little TJ, Cobbe N (2005) The evolution of immune-related genes from disease carrying mosquitoes: diversity in a peptidoglycan- and a thioester-recognizing protein. Insect Mol Biol  14: 599–605.  <ul class="find" data-citedArticleID="982784" data-doi="10.1111/j.1365-2583.2005.00588.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2583.2005.00588.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+immune-related+genes+from+disease+carrying+mosquitoes%3A+diversity+in+a+peptidoglycan-+and+a+thioester-recognizing+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+immune-related+genes+from+disease+carrying+mosquitoes%3A+diversity+in+a+peptidoglycan-+and+a+thioester-recognizing+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000793-Simard1" id="pone.0000793-Simard1"></a>Simard F, Licht M, Besansky NJ, Lehmann T (2006) Polymorphism at the defensin gene in the <em>Anopheles gambiae</em> complex: testing different selection hypotheses. Infect Genet Evol  7: 285–292.  <ul class="find" data-citedArticleID="982804" data-doi="10.1016/j.meegid.2006.11.004"><li><a href="http://dx.doi.org/10.1016/j.meegid.2006.11.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphism+at+the+defensin+gene+in+the+Anopheles+gambiae+complex%3A+testing+different+selection+hypotheses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphism+at+the+defensin+gene+in+the+Anopheles+gambiae+complex%3A+testing+different+selection+hypotheses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000793-Yang1" id="pone.0000793-Yang1"></a>Yang Z (2006) Computational Molecular Evolution. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0000793-Schlenke1" id="pone.0000793-Schlenke1"></a>Schlenke TA, Begun DJ (2003) Natural selection drives Drosophila immune system evolution. Genetics  164: 1471–1480.  <ul class="find" data-citedArticleID="982802"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Natural+selection+drives+Drosophila+immune+system+evolution.&amp;auth=&amp;atitle=Natural+selection+drives+Drosophila+immune+system+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+drives+Drosophila+immune+system+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+drives+Drosophila+immune+system+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000793-Collins1" id="pone.0000793-Collins1"></a>Collins FH, Zavala F, Graves PM, Cochrane AH, Gwadz RW, et al.  (1984) First field trial of an immunoradiometric assay for the detection of malaria sporozoites in mosquitoes. Am J Trop Med Hyg  33: 538–543.  <ul class="find" data-citedArticleID="982758"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=First+field+trial+of+an+immunoradiometric+assay+for+the+detection+of+malaria+sporozoites+in+mosquitoes.&amp;auth=&amp;atitle=First+field+trial+of+an+immunoradiometric+assay+for+the+detection+of+malaria+sporozoites+in+mosquitoes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=First+field+trial+of+an+immunoradiometric+assay+for+the+detection+of+malaria+sporozoites+in+mosquitoes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22First+field+trial+of+an+immunoradiometric+assay+for+the+detection+of+malaria+sporozoites+in+mosquitoes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000793-Ferguson1" id="pone.0000793-Ferguson1"></a>Ferguson HM, Read AF (2002) Why is the effect of malaria parasites on mosquito survival still unresolved? Trends Parasitol  18: 256–261.  <ul class="find" data-citedArticleID="982772" data-doi="10.1016/s1471-4922(02)02281-x"><li><a href="http://dx.doi.org/10.1016/s1471-4922(02)02281-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+is+the+effect+of+malaria+parasites+on+mosquito+survival+still+unresolved%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+is+the+effect+of+malaria+parasites+on+mosquito+survival+still+unresolved%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000793-Hay1" id="pone.0000793-Hay1"></a>Hay SI, Rogers DJ, Toomer JF, Snow RW (2000) Annual Plasmodium falciparum entomological inoculation rates (EIR) across Africa: literature survey, Internet access and review. Trans R Soc Trop Med Hyg  94: 113–127.  <ul class="find" data-citedArticleID="982774" data-doi="10.1016/s0035-9203(00)90246-3"><li><a href="http://dx.doi.org/10.1016/s0035-9203(00)90246-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Annual+Plasmodium+falciparum+entomological+inoculation+rates+%28EIR%29+across+Africa%3A+literature+survey%2C+Internet+access+and+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Annual+Plasmodium+falciparum+entomological+inoculation+rates+%28EIR%29+across+Africa%3A+literature+survey%2C+Internet+access+and+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000793-Vizoli1" id="pone.0000793-Vizoli1"></a>Vizoli J, Bulet P, Charlet M, Lowenberger C, Blass C, et al.  (2000) Cloning and analysis of a cecropin gene from the malaria vector mosquito, <em>Anopheles gambiae.</em> Insect Mol Biol  9: 75–84.  <ul class="find" data-citedArticleID="982810" data-doi="10.1046/j.1365-2583.2000.00164.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2583.2000.00164.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+analysis+of+a+cecropin+gene+from+the+malaria+vector+mosquito%2C+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+analysis+of+a+cecropin+gene+from+the+malaria+vector+mosquito%2C+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000793-Dimopoulos1" id="pone.0000793-Dimopoulos1"></a>Dimopoulos G, Richman A, Müller H-M, Kafatos FC (1997) Molecular immune response of the mosquito <em>Anopheles gambiae</em> to bacteria and malaria parasites. Proc Natl Acad Sci U S A  94: 11508–11513.  <ul class="find" data-citedArticleID="982762" data-doi="10.1073/pnas.94.21.11508"><li><a href="http://dx.doi.org/10.1073/pnas.94.21.11508" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+immune+response+of+the+mosquito+Anopheles+gambiae+to+bacteria+and+malaria+parasites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+immune+response+of+the+mosquito+Anopheles+gambiae+to+bacteria+and+malaria+parasites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000793-Warr1" id="pone.0000793-Warr1"></a>Warr E, Lambrechts L, Koella JC, Bourgouin C, Dimopoulos G (2006) <em>Anopheles gambiae</em> immune response to Sephadex beads: involvement of anti-<em>Plasmodium</em> factors in regulating melanization. Insect Biochem Mol Biol  36: 769–778.  <ul class="find" data-citedArticleID="982814" data-doi="10.1016/j.ibmb.2006.07.006"><li><a href="http://dx.doi.org/10.1016/j.ibmb.2006.07.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anopheles+gambiae+immune+response+to+Sephadex+beads%3A+involvement+of+anti-Plasmodium+factors+in+regulating+melanization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anopheles+gambiae+immune+response+to+Sephadex+beads%3A+involvement+of+anti-Plasmodium+factors+in+regulating+melanization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000793-Zhang1" id="pone.0000793-Zhang1"></a>Zhang J, Nielsen R, Yang Z (2005) Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level. Mol Biol Evol  22: 2472–2479.  <ul class="find" data-citedArticleID="982822" data-doi="10.1093/molbev/msi237"><li><a href="http://dx.doi.org/10.1093/molbev/msi237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000793-Yang2" id="pone.0000793-Yang2"></a>Yang Z, Wong WSW, Nielsen R (2005) Bayes empirical Bayes inference of amino acid sites under positive selection. Mol Biol Evol  22: 1107–1118.  <ul class="find" data-citedArticleID="982820" data-doi="10.1093/molbev/msi097"><li><a href="http://dx.doi.org/10.1093/molbev/msi097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayes+empirical+Bayes+inference+of+amino+acid+sites+under+positive+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayes+empirical+Bayes+inference+of+amino+acid+sites+under+positive+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000793-MacDonald1" id="pone.0000793-MacDonald1"></a>MacDonald JH, Kreitman M (1991) Adaptive protein evolution at the <em>Adh</em> locus in <em>Drosophila</em>. Nature  351: 652–654.  <ul class="find" data-citedArticleID="982786" data-doi="10.1038/351652a0"><li><a href="http://dx.doi.org/10.1038/351652a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+protein+evolution+at+the+Adh+locus+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+protein+evolution+at+the+Adh+locus+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000793-Donelly1" id="pone.0000793-Donelly1"></a>Donelly MJ, Licht MC, Lehmann T (2001) Evidence for recent population expansion in the evolutionary history of the malaria vectors <em>Anopheles arabiensis</em> and <em>Anopheles gambiae.</em> Mol Biol Evol  18: 1353–1364.  <ul class="find" data-citedArticleID="982764" data-doi="10.1093/oxfordjournals.molbev.a003919"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003919" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+recent+population+expansion+in+the+evolutionary+history+of+the+malaria+vectors+Anopheles+arabiensis+and+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+recent+population+expansion+in+the+evolutionary+history+of+the+malaria+vectors+Anopheles+arabiensis+and+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000793-Onyabe1" id="pone.0000793-Onyabe1"></a>Onyabe DY, Conn JE (2001) Population structure of the malaria mosquito <em>Anopheles arabiensis</em> across Nigeria suggests range expansion. Mol Ecol  10: 2577–2591.  <ul class="find" data-citedArticleID="982788" data-doi="10.1046/j.0962-1083.2001.01387.x"><li><a href="http://dx.doi.org/10.1046/j.0962-1083.2001.01387.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+structure+of+the+malaria+mosquito+Anopheles+arabiensis+across+Nigeria+suggests+range+expansion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+structure+of+the+malaria+mosquito+Anopheles+arabiensis+across+Nigeria+suggests+range+expansion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000793-Besansky1" id="pone.0000793-Besansky1"></a>Besansky NJ, Krzywinski J, Lehman T, Simard F, Kern M, et al.  (2003) Semipermeable species boundaries between <em>Anopheles gambiae</em> and <em>Anopheles arabiensis</em>: Evidence from multilocus DNA sequence variation. Proc Natl Acad Sci U S A  19: 10818–10823.  <ul class="find" data-citedArticleID="982748" data-doi="10.1073/pnas.1434337100"><li><a href="http://dx.doi.org/10.1073/pnas.1434337100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Semipermeable+species+boundaries+between+Anopheles+gambiae+and+Anopheles+arabiensis%3A+Evidence+from+multilocus+DNA+sequence+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Semipermeable+species+boundaries+between+Anopheles+gambiae+and+Anopheles+arabiensis%3A+Evidence+from+multilocus+DNA+sequence+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000793-Besansky2" id="pone.0000793-Besansky2"></a>Besansky NJ, Powell JR, Caccone A, Hamm DM, Scott JA, et al.  (1994) Molecular phylogeny of the <em>Anopheles gambiae</em> complex suggests genetic introgression between principle malaria vectors. Proc Natl Acad Sci U S A  91: 6885–6888.  <ul class="find" data-citedArticleID="982750" data-doi="10.1073/pnas.91.15.6885"><li><a href="http://dx.doi.org/10.1073/pnas.91.15.6885" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+of+the+Anopheles+gambiae+complex+suggests+genetic+introgression+between+principle+malaria+vectors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+of+the+Anopheles+gambiae+complex+suggests+genetic+introgression+between+principle+malaria+vectors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000793-Slotman1" id="pone.0000793-Slotman1"></a>Slotman MA, della Torre A, Calzetta M, Powell JR (2005) Differential introgression of chromosomal regions between <em>Anopheles gambiae</em> and <em>An. arabiensis</em>. Am J Trop Med Hyg  73: 326–335.  <ul class="find" data-citedArticleID="982806"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Differential+introgression+of+chromosomal+regions+between+Anopheles+gambiae+and+An.+arabiensis.&amp;auth=&amp;atitle=Differential+introgression+of+chromosomal+regions+between+Anopheles+gambiae+and+An.+arabiensis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+introgression+of+chromosomal+regions+between+Anopheles+gambiae+and+An.+arabiensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+introgression+of+chromosomal+regions+between+Anopheles+gambiae+and+An.+arabiensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000793-Enkhbayar1" id="pone.0000793-Enkhbayar1"></a>Enkhbayar P, Kamiya M, Osaki M, Matsumoto T, Matsushima N (2004) Structural principles of Leucine-Rich Repeat (LRR) Proteins. Proteins  54: 394–403.  <ul class="find" data-citedArticleID="982768" data-doi="10.1002/prot.10605"><li><a href="http://dx.doi.org/10.1002/prot.10605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+principles+of+Leucine-Rich+Repeat+%28LRR%29+Proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+principles+of+Leucine-Rich+Repeat+%28LRR%29+Proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000793-Cohuet1" id="pone.0000793-Cohuet1"></a>Cohuet A, Osta M, Morlais I, Awono-Ambene PH, Michel K, et al.  (2006) <em>Anopheles</em> and <em>Plasmodium</em>: from laboratory models to natural systems in the field. EMBO Reports  7: 1285–1289.  <ul class="find" data-citedArticleID="982756" data-doi="10.1038/sj.embor.7400831"><li><a href="http://dx.doi.org/10.1038/sj.embor.7400831" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anopheles+and+Plasmodium%3A+from+laboratory+models+to+natural+systems+in+the+field." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anopheles+and+Plasmodium%3A+from+laboratory+models+to+natural+systems+in+the+field.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000793-Lambrechts1" id="pone.0000793-Lambrechts1"></a>Lambrechts L, Halbert J, Durand P, Gouagna LC, Koella JC (2005) Host genotype by parasite genotype interactions underlying the resistance of anopheline mosquitoes to <em>Plasmodium falciparum.</em> Malar J  4: 3.  <ul class="find" data-citedArticleID="982782"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Host+genotype+by+parasite+genotype+interactions+underlying+the+resistance+of+anopheline+mosquitoes+to+Plasmodium+falciparum.&amp;auth=&amp;atitle=Host+genotype+by+parasite+genotype+interactions+underlying+the+resistance+of+anopheline+mosquitoes+to+Plasmodium+falciparum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Host+genotype+by+parasite+genotype+interactions+underlying+the+resistance+of+anopheline+mosquitoes+to+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Host+genotype+by+parasite+genotype+interactions+underlying+the+resistance+of+anopheline+mosquitoes+to+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000793-Dawkins1" id="pone.0000793-Dawkins1"></a>Dawkins R, Krebs JR (1979) Arms races between and within species. Proc R Soc Lond B Biol Sci  205: 489–511.  <ul class="find" data-citedArticleID="982760" data-doi="10.1098/rspb.1979.0081"><li><a href="http://dx.doi.org/10.1098/rspb.1979.0081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arms+races+between+and+within+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arms+races+between+and+within+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000793-Fanello1" id="pone.0000793-Fanello1"></a>Fanello C, Santolamazza F, della Torre A (2002) Simultaneous identification of species and molecular forms of the <em>Anopheles gambiae</em> comples by PCR-RFLP. Med Vet Entomol  16: 461–464.  <ul class="find" data-citedArticleID="982770" data-doi="10.1046/j.1365-2915.2002.00393.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2915.2002.00393.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simultaneous+identification+of+species+and+molecular+forms+of+the+Anopheles+gambiae+comples+by+PCR-RFLP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simultaneous+identification+of+species+and+molecular+forms+of+the+Anopheles+gambiae+comples+by+PCR-RFLP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000793-Besansky3" id="pone.0000793-Besansky3"></a>Besansky NJ, Collins FH, Towson H (2006) A species-specific PCR for the identification of the malaria vector <em>Anopheles bambae</em>. Ann Trop Med Parasitol  100: 277–280.  <ul class="find" data-citedArticleID="982752" data-doi="10.1179/136485906x105624"><li><a href="http://dx.doi.org/10.1179/136485906x105624" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+species-specific+PCR+for+the+identification+of+the+malaria+vector+Anopheles+bambae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+species-specific+PCR+for+the+identification+of+the+malaria+vector+Anopheles+bambae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000793-White1" id="pone.0000793-White1"></a>White BJ, Santolamazza F, Kamau L, Pombi M, Grushko O, et al.  (2007) Molecular karyotyping of the 2La inversion in <em>Anopheles gambiae.</em> Am J Trop Med Hyg  76: 334–339.  <ul class="find" data-citedArticleID="982816"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+karyotyping+of+the+2La+inversion+in+Anopheles+gambiae.&amp;auth=&amp;atitle=Molecular+karyotyping+of+the+2La+inversion+in+Anopheles+gambiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+karyotyping+of+the+2La+inversion+in+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+karyotyping+of+the+2La+inversion+in+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000793-Rozen1" id="pone.0000793-Rozen1"></a>Rozen S, Skaletsky HJ (2000)  Primer3 on the WWW for general users and for biologist programmers. In: Krawetz S, Misener S, editors. Bioinformatics Methods and Protocols: Methods in Molecular Biology. Totowa: Humana Press. pp. 365–38.  <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0000793-Kumar1" id="pone.0000793-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: integrated software for molecular evolutionary genetics analysis and sequence alignment. Briefings in Bioinformatics  5: 150–163.  <ul class="find" data-citedArticleID="982780" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+integrated+software+for+molecular+evolutionary+genetics+analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+integrated+software+for+molecular+evolutionary+genetics+analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000793-Rozas1" id="pone.0000793-Rozas1"></a>Rozas J, Sánchez-DelBarrio JC, Messeguer X, Rozas R (2003) DnaSP, DNA polymorphism analyses by the coalescent and other methods. Bioinformatics  19: 2496–2497.  <ul class="find" data-citedArticleID="982798" data-doi="10.1093/bioinformatics/btg359"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000793-Ronquist1" id="pone.0000793-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="982796" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000793-Posada1" id="pone.0000793-Posada1"></a>Posada D, Crandall KA (1998) Modeltest: testing the model of DNA substitution. Bioinformatics  14: 817–818.  <ul class="find" data-citedArticleID="982792" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeltest%3A+testing+the+model+of+DNA+substitution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeltest%3A+testing+the+model+of+DNA+substitution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.XML" value="108515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.PDF" value="172345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t001.PNG_L" value="66036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t001.PNG_M" value="85339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t001.PNG_S" value="12811"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t001.TIF" value="294244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t001.PNG_I" value="28837"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t002.PNG_L" value="82721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t002.PNG_M" value="100962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t002.PNG_S" value="17085"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t002.TIF" value="443918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t002.PNG_I" value="35075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t003.PNG_L" value="91153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t003.PNG_M" value="108190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t003.PNG_S" value="17723"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t003.TIF" value="461060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t003.PNG_I" value="38525"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t004.PNG_L" value="91985"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t004.PNG_M" value="112461"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t004.PNG_S" value="18048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t004.TIF" value="464364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t004.PNG_I" value="39402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.g001.PNG_L" value="156561"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.g001.PNG_M" value="53727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.g001.PNG_S" value="7897"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.g001.TIF" value="228410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.g001.PNG_I" value="18971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t005.PNG_L" value="99017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t005.PNG_M" value="135535"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t005.PNG_S" value="17311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t005.TIF" value="437038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.t005.PNG_I" value="45427"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.g002.PNG_L" value="484952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.g002.PNG_M" value="25183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.g002.PNG_S" value="5454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.g002.TIF" value="722632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.g002.PNG_I" value="8572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.s001.TIF" value="1189183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000793.s002.TIF" value="1611050"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000793&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000793" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000793&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000793&volume=&issue=&title=Patterns of Selection in Anti-Malarial Immune Genes in Malaria Vectors: Evidence for Adaptive Evolution in LRIM1 in Anopheles arabiensis&author_name=Michel%20A.%20Slotman%2C%20Aristeidis%20Parmakelis%2C%20Jonathon%20C.%20Marshall%2C%20Parfait%20H.%20Awono-Ambene%2C%20Christophe%20Antonio-Nkondjo%2C%20Frederic%20Simard%2C%20Adalgisa%20Caccone%2C%20Jeffrey%20R.%20Powell&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000793" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000793" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000793" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000793&amp;t=Patterns%20of%20Selection%20in%20Anti-Malarial%20Immune%20Genes%20in%20Malaria%20Vectors%3A%20Evidence%20for%20Adaptive%20Evolution%20in%20LRIM1%20in%20Anopheles%20arabiensis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000793&title=Patterns%20of%20Selection%20in%20Anti-Malarial%20Immune%20Genes%20in%20Malaria%20Vectors%3A%20Evidence%20for%20Adaptive%20Evolution%20in%20LRIM1%20in%20Anopheles%20arabiensis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000793&amp;title=Patterns%20of%20Selection%20in%20Anti-Malarial%20Immune%20Genes%20in%20Malaria%20Vectors%3A%20Evidence%20for%20Adaptive%20Evolution%20in%20LRIM1%20in%20Anopheles%20arabiensis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000793" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000793&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Patterns of Selection in Anti-Malarial Immune Genes in Malaria Vectors: Evidence for Adaptive Evolution in LRIM1 in Anopheles arabiensis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000793';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Patterns%20of%20Selection%20in%20Anti-Malarial%20Immune%20Genes%20in%20Malaria%20Vectors%3A%20Evidence%20for%20Adaptive%20Evolution%20in%20LRIM1%20in%20Anopheles%20arabiensis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000793" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000793" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="24424"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Malaria%22" title="Search for articles in the subject area:'Malaria'"><div class="flagText">Malaria</div></a>
              <div data-categoryid="47353" data-articleid="24424"
                   data-categoryname="Malaria"
                   class="flagImage" title="Flag 'Malaria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mosquitoes%22" title="Search for articles in the subject area:'Mosquitoes'"><div class="flagText">Mosquitoes</div></a>
              <div data-categoryid="31953" data-articleid="24424"
                   data-categoryname="Mosquitoes"
                   class="flagImage" title="Flag 'Mosquitoes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="24424"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24424"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plasmodium%22" title="Search for articles in the subject area:'Plasmodium'"><div class="flagText">Plasmodium</div></a>
              <div data-categoryid="18205" data-articleid="24424"
                   data-categoryname="Plasmodium"
                   class="flagImage" title="Flag 'Plasmodium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plasmodium+falciparum%22" title="Search for articles in the subject area:'Plasmodium falciparum'"><div class="flagText">Plasmodium falciparum</div></a>
              <div data-categoryid="43481" data-articleid="24424"
                   data-categoryname="Plasmodium falciparum"
                   class="flagImage" title="Flag 'Plasmodium falciparum' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="24424"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5532'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1600'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=943&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=14587">Other relevant publication</a><br>Posted by DarrenObbard</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
