

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genome Sequence of Fusobacterium nucleatum Subspecies Polymorphum — a Genetically Tractable Fusobacterium</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000659"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000659" />

    <meta name="citation_title" content="Genome Sequence of Fusobacterium nucleatum Subspecies Polymorphum — a Genetically Tractable Fusobacterium"/>
    <meta itemprop="name" content="Genome Sequence of Fusobacterium nucleatum Subspecies Polymorphum — a Genetically Tractable Fusobacterium"/>

      <meta name="citation_author" content="Sandor E. Karpathy"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
            <meta name="citation_author_institution" content="Department of Microbiology and Molecular Genetics, University of Texas Medical School, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Xiang Qin"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Jason Gioia"/>
            <meta name="citation_author_institution" content="Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Huaiyang Jiang"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Yamei Liu"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Joseph F. Petrosino"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Shailaja Yerrapragada"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="George E. Fox"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Susan Kinder Haake"/>
            <meta name="citation_author_institution" content="Associated Clinical Specialties, University of California at Los Angeles School of Dentistry, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="George M. Weinstock"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
            <meta name="citation_author_institution" content="Department of Microbiology and Molecular Genetics, University of Texas Medical School, Houston, Texas, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Sarah K. Highlander"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America"/>

    <meta name="citation_date" content="2007/8/1"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000659.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e659"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Bacterial diversity in human subgingival plaque.; citation_author=BJ Paster; citation_author=SK Boches; citation_author=JL Galvin; citation_author=RE Ericson; citation_author=CN Lau; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=1; citation_pages=3770-3783; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Comparison of the microbiota of supra- and subgingival plaque in health and periodontitis.; citation_author=LA Ximénez-Fyvie; citation_author=AD Haffajee; citation_author=SS Socransky; citation_journal_title=J Clin Periodontol; citation_volume=27; citation_number=2; citation_pages=648-657; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Development of oral bacterial flora in young children.; citation_author=E Kononen; citation_journal_title=Ann Med; citation_volume=32; citation_number=3; citation_pages=107-112; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Adhere today, here tomorrow: Oral bacterial adherence.; citation_author=PE Kolenbrander; citation_author=J London; citation_journal_title=J Bacteriol; citation_volume=175; citation_number=4; citation_pages=3247-3252; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Fusobacterium nucleatum supports the growth of Porphyromonas gingivalis in oxygenated and carbon-dioxide-depleted environments.; citation_author=PI Diaz; citation_author=PS Zilm; citation_author=AH Rogers; citation_journal_title=Microbiol; citation_volume=148; citation_number=5; citation_pages=467-472; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The bacteria of periodontal diseases.; citation_author=WE Moore; citation_author=LV Moore; citation_journal_title=Periodontol 2000; citation_volume=5; citation_number=6; citation_pages=66-77; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Prevalence, extent, severity and progression of periodontal disease.; citation_author=LJ Brown; citation_author=H Loe; citation_journal_title=Periodontol 2000; citation_volume=2; citation_number=7; citation_pages=57-71; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Microbiology of liver and spleen abscesses.; citation_author=I Brook; citation_author=EH Frazier; citation_journal_title=J Med Microbiol; citation_volume=47; citation_number=8; citation_pages=1075-1080; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The microbial spectrum of brain abscess with special reference to anaerobic bacteria.; citation_author=R Chaudhry; citation_author=B Dhawan; citation_author=BVJ Laxmi; citation_author=VS Mehta; citation_journal_title=Br J Neurosurg; citation_volume=12; citation_number=9; citation_pages=127-130; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Septic arthritis of the hip due to Fusobacterium nucleatum.; citation_author=AM Chryssagi; citation_author=CB Brusselmans; citation_author=JJ Rombouts; citation_journal_title=Clin Rheumatol; citation_volume=20; citation_number=10; citation_pages=229-231; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A retrospective review of cases of anaerobic empyema and update of bacteriology.; citation_author=R Civen; citation_author=H Jousimies-Somer; citation_author=M Marina; citation_author=L Borenstein; citation_author=H Shah; citation_journal_title=Clin Infect Dis; citation_volume=20; citation_number=11; citation_pages=S224-229; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Fatal sepsis due to a beta-lactamase-producing strain of Fusobacterium nucleatum subspecies polymorphum.; citation_author=EJ Goldstein; citation_author=PH Summanen; citation_author=DM Citron; citation_author=MH Rosove; citation_author=SM Finegold; citation_journal_title=Clin Infect Dis; citation_volume=20; citation_number=12; citation_pages=797-800; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Bacterial vaginosis and vaginal microorganisms in idiopathic premature labor and association with pregnancy outcome.; citation_author=E Holst; citation_author=AR Goffeng; citation_author=B Andersch; citation_journal_title=J Clin Microbiol; citation_volume=32; citation_number=13; citation_pages=176-186; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Bacteriologic findings in peritonsillar abscesses in young adults.; citation_author=H Jousimies-Somer; citation_author=S Savolainen; citation_author=A Makitie; citation_author=J Ylikoski; citation_journal_title=Clin Infect Dis 16 Suppl; citation_volume=4; citation_number=14; citation_pages=S292-298; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Universal DNA primers amplify bacterial DNA from human fetal membranes and link Fusobacterium nucleatum with prolonged preterm membrane rupture.; citation_author=RJ Cahill; citation_author=S Tan; citation_author=G Dougan; citation_author=P O'Gaora; citation_author=D Pickard; citation_journal_title=Mol Hum Reprod; citation_volume=11; citation_number=15; citation_pages=761-766; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Possible association between amniotic fluid micro-organism infection and microflora in the mouth.; citation_author=C Bearfield; citation_author=ES Davenport; citation_author=V Sivapathasundaram; citation_author=RP Allaker; citation_journal_title=BJOG; citation_volume=109; citation_number=16; citation_pages=527-533; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Preterm birth: associations with genital and possibly oral microflora.; citation_author=GB Hill; citation_journal_title=Ann Periodontol; citation_volume=3; citation_number=17; citation_pages=222-232; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Preterm labor and bacterial intraamniotic infection: arachidonic acid liberation by phospholipase A2 of Fusobacterium nucleatum.; citation_author=H Mikamo; citation_author=K Kawazoe; citation_author=Y Sato; citation_author=A Imai; citation_author=T Tamaya; citation_journal_title=Am J Obstet Gynecol; citation_volume=179; citation_number=18; citation_pages=1579-1582; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Elastase activity of anaerobes isolated from amniotic fluid with preterm premature rupture of membranes.; citation_author=H Mikamo; citation_author=K Kawazoe; citation_author=Y Sato; citation_author=T Tamaya; citation_journal_title=Am J Obstet Gynecol; citation_volume=180; citation_number=19; citation_pages=378-380; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genome sequence and analysis of the oral bacterium Fusobacterium nucleatum strain ATCC 25586.; citation_author=V Kapatral; citation_author=I Anderson; citation_author=N Ivanova; citation_author=G Reznik; citation_author=T Los; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=20; citation_pages=2005-2018; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome analysis of F. nucleatum sub spp vincentii and its comparison with the genome of F. nucleatum ATCC 25586.; citation_author=V Kapatral; citation_author=N Ivanova; citation_author=I Anderson; citation_author=G Reznik; citation_author=A Bhattacharyya; citation_journal_title=Genome Res; citation_volume=13; citation_number=21; citation_pages=1180-1189; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The use of allozyme electrophoresis to assess genetic heterogeneity among previously subspeciated isolates of Fusobacterium nucleatum.; citation_author=ML Morris; citation_author=RH Andrews; citation_author=AH Rogers; citation_journal_title=Oral Microbiol Immunol; citation_volume=11; citation_number=22; citation_pages=15-21; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Taxonomy, biology, and periodontal aspects of Fusobacterium nucleatum.; citation_author=AI Bolstad; citation_author=HB Jensen; citation_author=V Bakken; citation_journal_title=Clin Microbiol Rev; citation_volume=9; citation_number=23; citation_pages=55-71; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=16S-23S rDNA internal transcribed spacer sequences for analysis of the phylogenetic relationships among species of the genus Fusobacterium.; citation_author=G Conrads; citation_author=MC Claros; citation_author=DM Citron; citation_author=KL Tyrrell; citation_author=V Merriam; citation_journal_title=Int J Syst Evol Microbiol; citation_volume=52; citation_number=24; citation_pages=493-499; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genotypic and phenotypic characterization of fusobacteria from Chinese and European patients with inflammatory periodontal disease.; citation_author=R Gmur; citation_author=RA Munson; citation_author=WG Wade; citation_journal_title=Syst App Microbiol; citation_volume=29; citation_number=25; citation_pages=120-130; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Amino acid-dependent transport of sugars by Fusobacterium nucleatum ATCC 10953.; citation_author=SA Robrish; citation_author=C Oliver; citation_author=J Thompson; citation_journal_title=J Bacteriol; citation_volume=169; citation_number=26; citation_pages=3891-3897; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Regulation of fructose metabolism and polymer synthesis by Fusobacterium nucleatum ATCC 10953.; citation_author=SA Robrish; citation_author=J Thompson; citation_journal_title=J Bacteriol; citation_volume=172; citation_number=27; citation_pages=5714-5723; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=The breakdown and utilization of peptides by strains of Fusobacterium nucleatum.; citation_author=AH Rogers; citation_author=NJ Gully; citation_author=AL Pfennig; citation_author=PS Zilm; citation_journal_title=Oral Microbiol Immunol; citation_volume=7; citation_number=28; citation_pages=299-303; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Aspects of the growth and metabolism of Fusobacterium nucleatum ATCC 10953 in continuous culture.; citation_author=AH Rogers; citation_author=PS Zilm; citation_author=NJ Gully; citation_author=AL Pfennig; citation_author=PD Marsh; citation_journal_title=Oral Microbiol Immunol; citation_volume=6; citation_number=29; citation_pages=250-255; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=A non-lectin-like mechanism by which Fusobacterium nucleatum 10953 adheres to and activates human lymphocytes.; citation_author=RS Tuttle; citation_author=NA Strubel; citation_author=J Mourad; citation_author=DF Mangan; citation_journal_title=Oral Microbiol Immunol; citation_volume=7; citation_number=30; citation_pages=78-83; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Binding specificity of Fusobacterium nucleatum to human erythrocytes, polymorphonuclear leukocytes, fibroblasts, and HeLa cells.; citation_author=M Ozaki; citation_author=Y Miyake; citation_author=M Shirakawa; citation_author=T Takemoto; citation_author=H Okamoto; citation_journal_title=J Periodontal Res; citation_volume=25; citation_number=31; citation_pages=129-134; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Interactions between periodontal bacteria and human oral epithelial cells: Fusobacterium nucleatum adheres to and invades epithelial cells.; citation_author=YW Han; citation_author=W Shi; citation_author=GT Huang; citation_author=S Kinder Haake; citation_author=NH Park; citation_journal_title=Infect Immun; citation_volume=68; citation_number=32; citation_pages=3140-3146; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Bacteria recovered from dental pulp induce apoptosis of lymph node cells.; citation_author=AP Ribeiro-Sobrinho; citation_author=FL Rabelo; citation_author=CB Figueiredo; citation_author=JI Alvarez-Leite; citation_author=JR Nicoli; citation_journal_title=J Med Microbiol; citation_volume=54; citation_number=33; citation_pages=413-416; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Induction of apoptotic cell death in peripheral blood mononuclear and polymorphonuclear cells by an oral bacterium, Fusobacterium nucleatum.; citation_author=A Jewett; citation_author=WR Hume; citation_author=H Le; citation_author=TN Huynh; citation_author=YW Han; citation_journal_title=Infect Immun; citation_volume=68; citation_number=34; citation_pages=1893-1898; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Immunobiological activities of a porin fraction isolated from Fusobacterium nucleatum ATCC 10953.; citation_author=H Takada; citation_author=T Ogawa; citation_author=F Yoshimura; citation_author=K Otsuka; citation_author=S Kokeguchi; citation_journal_title=Infect Immun; citation_volume=56; citation_number=35; citation_pages=855-863; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Role of Fusobacterium nucleatum and coaggregation in anaerobe survival in planktonic and biofilm oral microbial communities during aeration.; citation_author=DJ Bradshaw; citation_author=PD Marsh; citation_author=GK Watson; citation_author=C Allison; citation_journal_title=Infect Immun; citation_volume=66; citation_number=36; citation_pages=4729-4732; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The virulence of mixed infection with Streptococcus constellatus and Fusobacterium nucleatum in a murine orofacial infection model.; citation_author=T Kuriyama; citation_author=K Nakagawa; citation_author=S Kawashiri; citation_author=E Yamamoto; citation_author=S Nakamura; citation_journal_title=Microbes Infect; citation_volume=2; citation_number=37; citation_pages=1425-1430; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Characterization of Bacteroides forsythus isolates.; citation_author=T Takemoto; citation_author=H Kurihara; citation_author=G Dahlen; citation_journal_title=J Clin Microbiol; citation_volume=35; citation_number=38; citation_pages=1378-1381; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Native plasmids of Fusobacterium nucleatum: Characterization and use in development of genetic systems.; citation_author=S Kinder Haake; citation_author=SC Yoder; citation_author=G Attarian; citation_author=K Podkaminer; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=39; citation_pages=1176-1180; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Native plasmids of Fusobacterium nucleatum: characterization and use in development of genetic systems.; citation_author=SK Haake; citation_author=SC Yoder; citation_author=G Attarian; citation_author=K Podkaminer; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=40; citation_pages=1176-1180; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Ribosomal genes in Escherichia coli.; citation_author=L Lindahl; citation_author=JM Zengel; citation_journal_title=Ann Rev Genet; citation_volume=20; citation_number=41; citation_pages=297-326; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Conserved gene clusters in bacterial genomes provide further support for the primacy of RNA.; citation_author=JL Siefert; citation_author=KA Martin; citation_author=F Abdi; citation_author=WR Widger; citation_author=GE Fox; citation_journal_title=J Mol Evol; citation_volume=45; citation_number=42; citation_pages=467-472; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Identification and analysis of fipA, a Fusobacterium nucleatum immunosuppressive factor gene.; citation_author=DR Demuth; citation_author=R Savary; citation_author=E Golub; citation_author=BJ Shenker; citation_journal_title=Infect Immun; citation_volume=64; citation_number=43; citation_pages=1335-1341; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Sequence conservation and distribution of the fusobacterial immunosuppressive protein gene, fipA.; citation_author=S Hunt Gerardo; citation_author=SC Yoder; citation_author=DM Citron; citation_author=EJC Goldstein; citation_author=S Kinder Haake; citation_journal_title=Oral Microbiology and Immunology; citation_volume=17; citation_number=44; citation_pages=315-320; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Amelioration of bacterial genomes: rates of change and exchange.; citation_author=JG Lawrence; citation_author=H Ochman; citation_journal_title=J Mol Evol; citation_volume=44; citation_number=45; citation_pages=383-397; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Molecular archaeology of the Escherichia coli genome.; citation_author=JG Lawrence; citation_author=H Ochman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=46; citation_pages=9413-9417; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Global dinucleotide signatures and analysis of genomic heterogeneity.; citation_author=S Karlin; citation_journal_title=Curr Opin Microbiol; citation_volume=1; citation_number=47; citation_pages=598-610; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Detecting alien genes in bacterial genomes.; citation_author=J Mrazek; citation_author=S Karlin; citation_journal_title=Ann NY Acad Sci; citation_volume=870; citation_number=48; citation_pages=314-329; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Evidence for horizontal gene transfer in Escherichia coli speciation.; citation_author=C Medigue; citation_author=T Rouxel; citation_author=P Vigier; citation_author=A Henaut; citation_author=A Danchin; citation_journal_title=J Mol Biol; citation_volume=222; citation_number=49; citation_pages=851-856; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Codon usage and lateral gene transfer in Bacillus subtilis.; citation_author=I Moszer; citation_author=EP Rocha; citation_author=A Danchin; citation_journal_title=Curr Opin Microbiol; citation_volume=2; citation_number=50; citation_pages=524-528; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=How to interpret an anonymous bacterial genome: Machine learning approach to gene identification.; citation_author=WS Hayes; citation_author=M Borodovsky; citation_journal_title=Genome Res; citation_volume=8; citation_number=51; citation_pages=1154-1171; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=On surrogate methods for detecting lateral gene transfer.; citation_author=MA Ragan; citation_journal_title=FEMS Microbiol Lett; citation_volume=201; citation_number=52; citation_pages=187-191; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Inference of horizontal genetic transfer from molecular data: An approach using the bootstrap.; citation_author=JG Lawrence; citation_author=DL Hartl; citation_journal_title=Genetics; citation_volume=131; citation_number=53; citation_pages=753-760; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Horizontal gene transfer and phylogenetics.; citation_author=H Philippe; citation_author=CJ Douady; citation_journal_title=Curr Opin Microbiol; citation_volume=6; citation_number=54; citation_pages=498-505; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Propanediol utilization genes (pdu) of Salmonella typhimurium: three genes for the propanediol dehydratase.; citation_author=T Bobik; citation_author=Y Xu; citation_author=R Jeter; citation_author=K Otto; citation_author=J Roth; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=55; citation_pages=6633-6639; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The propanediol utilization (pdu) operon of Salmonella enterica serovar Typhimurium LT2 includes genes necessary for formation of polyhedral organelles involved in coenzyme B12-dependent 1,2-propanediol degradation.; citation_author=TA Bobik; citation_author=GD Havemann; citation_author=RJ Busch; citation_author=DS Williams; citation_author=HC Aldrich; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=56; citation_pages=5967-5975; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Bacterial degradation of glycoprotein sugars in human saliva.; citation_author=SA Leach; citation_author=P Critchley; citation_journal_title=Nature; citation_volume=209; citation_number=57; citation_pages=506; citation_date=1966; " />
      <meta name="citation_reference" content="citation_title=Dissimilatory pathways for sugars, polyols, and carboxylates.; citation_author=ECC Lin; citation_number=58; citation_pages=307-342; citation_date=1996; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of Clostridium tetani, the causative agent of tetanus disease.; citation_author=H Bruggemann; citation_author=S Baumer; citation_author=WF Fricke; citation_author=A Wiezer; citation_author=H Liesegang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=59; citation_pages=1316-1321; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genomic analysis of Clostridium perfringens bacteriophage phi3626, which integrates into guaA and possibly affects sporulation.; citation_author=M Zimmer; citation_author=S Scherer; citation_author=MJ Loessner; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=60; citation_pages=4359-4368; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Type IV secretion: the Agrobacterium VirB/D4 and related conjugation systems.; citation_author=PJ Christie; citation_journal_title=Biochim Biophys Acta; citation_volume=1694; citation_number=61; citation_pages=219-234; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Protein secretion systems in Fusobacterium nucleatum: Genomic identification of Type 4 piliation and complete Type V pathways brings new insight into mechanisms of pathogenesis.; citation_author=M Desvaux; citation_author=A Khan; citation_author=SA Beatson; citation_author=A Scott-Tucker; citation_author=IR Henderson; citation_journal_title=Biochim Biophys Acta; citation_volume=1713; citation_number=62; citation_pages=92-1125; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A chimeric ribozyme in Clostridium difficile combines features of group I introns and insertion elements.; citation_author=V Braun; citation_author=M Mehlig; citation_author=M Moos; citation_author=M Rupnik; citation_author=B Kalt; citation_journal_title=Mol Microbiol; citation_volume=36; citation_number=63; citation_pages=1447-1459; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Representation of the secondary and tertiary structure of group I introns.; citation_author=TR Cech; citation_author=SH Damberger; citation_author=RR Gutell; citation_journal_title=Nature Struct Biol; citation_volume=1; citation_number=64; citation_pages=273-280; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Novel sequence organization and insertion specificity of IS605 and IS606: chimaeric transposable elements of Helicobacter pylori.; citation_author=D Kersulyte; citation_author=NS Akopyants; citation_author=SW Clifton; citation_author=BA Roe; citation_author=DE Berg; citation_journal_title=Gene; citation_volume=223; citation_number=65; citation_pages=175-186; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Conserved sequence motifs in the initiator proteins for rolling circle DNA replication encoded by diverse replicons from eubacteria, eukaryotes and archaebacteria.; citation_author=TV Ilyina; citation_author=EV Koonin; citation_journal_title=Nucl Acids Res; citation_volume=20; citation_number=66; citation_pages=3279-3285; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Mobile genetic elements of Fusobacterium nucleatum.; citation_author=TL McKay; citation_author=J Ko; citation_author=Y Bilalis; citation_author=JM DiRienzo; citation_journal_title=Plasmid; citation_volume=33; citation_number=67; citation_pages=15-20; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of a chromosomal virulence gene, vacJ, required for intracellular spreading of Shigella flexneri.; citation_author=T Suzuki; citation_author=I Fukuda; citation_author=T Tobe; citation_author=M Yoshikawa; citation_author=C Sasakawa; citation_journal_title=Mol Microbiol; citation_volume=11; citation_number=68; citation_pages=31-41; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Fusobacterium nucleatum induces premature and term stillbirths in pregnant mice: Implication of oral bacteria in preterm birth.; citation_author=YW Han; citation_author=RW Redline; citation_author=M Li; citation_author=L Yin; citation_author=GB Hill; citation_journal_title=Infect Immun; citation_volume=72; citation_number=69; citation_pages=2272-2279; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Isolation of a corncob (coaggregation) receptor polypeptide from Fusobacterium nucleatum.; citation_author=J Kaufman; citation_author=JM DiRienzo; citation_journal_title=Infect Immun; citation_volume=57; citation_number=70; citation_pages=331-337; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Cloning and expression of FomA, the major outer-membrane protein gene from Fusobacterium nucleatum T18.; citation_author=S Kinder Haake; citation_author=X Wang; citation_journal_title=Arch Oral Biol; citation_volume=42; citation_number=71; citation_pages=19-24; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A Salmonella typhimurium virulence gene linked to flg.; citation_author=M Carsiotis; citation_author=B Stocker; citation_author=D Weinstein; citation_author=A O'Brien; citation_journal_title=Infect Immun; citation_volume=57; citation_number=72; citation_pages=3276-3280; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=TraT lipoprotein, a plasmid-specified mediator of interactions between Gram-negative bacteria and their environment.; citation_author=S Sukupolvi; citation_author=CD O'Connor; citation_journal_title=Microbiol Rev; citation_volume=54; citation_number=73; citation_pages=331-341; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=vacB, a novel chromosomal gene required for expression of virulence genes on the large plasmid of Shigella flexneri.; citation_author=T Tobe; citation_author=C Sasakawa; citation_author=N Okada; citation_author=Y Honma; citation_author=M Yoshikawa; citation_journal_title=J Bacteriol; citation_volume=174; citation_number=74; citation_pages=6359-6367; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The vacB gene required for virulence in Shigella flexneri and Escherichia coli encodes the exoribonuclease RNase R.; citation_author=Z-F Cheng; citation_author=Y Zuo; citation_author=Z Li; citation_author=KE Rudd; citation_author=MP Deutscher; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=75; citation_pages=14077-14080; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Short-chain carboxylic-acid-stimulated, PMN-mediated gingival inflammation.; citation_author=R Niederman; citation_author=J Zhang; citation_author=S Kashket; citation_journal_title=Critic Rev Oral Biol Med; citation_volume=8; citation_number=76; citation_pages=269-290; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Fusobacterium nucleatum inhibits human T-cell activation by arresting cells in the mid-G1 phase of the cell cycle.; citation_author=BJ Shenker; citation_author=S Datar; citation_journal_title=Infect Immun; citation_volume=63; citation_number=77; citation_pages=4830-4836; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Iron and microbial infection.; citation_author=UE Schaible; citation_author=SHE Kaufmann; citation_journal_title=Nat Rev Micro; citation_volume=2; citation_number=78; citation_pages=946-953; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chemical structure of the lipid A component of lipopolysaccharides from Fusobacterium nucleatum.; citation_author=S Hase; citation_author=T Hofstad; citation_author=ET Rietschel; citation_journal_title=J Bacteriol; citation_volume=129; citation_number=79; citation_pages=9-14; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Cross-species induction of luminescence in the quorum-sensing bacterium Vibrio harveyi.; citation_author=B Bassler; citation_author=E Greenberg; citation_author=A Stevens; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=80; citation_pages=4043-4045; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=LuxS-based signaling affects Streptococcus mutans biofilm formation.; citation_author=A Yoshida; citation_author=T Ansai; citation_author=T Takehara; citation_author=HK Kuramitsu; citation_journal_title=Appl Environ Microbiol; citation_volume=71; citation_number=81; citation_pages=2372-2380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Periodontal pathogens produce quorum sensing signal molecules.; citation_author=J Frias; citation_author=E Olle; citation_author=M Alsina; citation_journal_title=Infect Immun; citation_volume=69; citation_number=82; citation_pages=3431-3434; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The response to oxidative stress of Fusobacterium nucleatum grown in continuous culture.; citation_author=PI Diaz; citation_author=PS Zilm; citation_author=AH Rogers; citation_journal_title=FEMS Microbiol Lett; citation_volume=187; citation_number=83; citation_pages=31-34; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Host modulation of tissue destruction caused by periodontopathogens: effects on a mixed microbial infection composed of Porphyromonas gingivalis and Fusobacterium nucleatum.; citation_author=JL Ebersole; citation_author=F Feuille; citation_author=L Kesavalu; citation_author=SC Holt; citation_journal_title=Microb Pathog; citation_volume=23; citation_number=84; citation_pages=23-32; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Mixed infection with Porphyromonas gingivalis and Fusobacterium nucleatum in a murine lesion model: Potential synergistic effects on virulence.; citation_author=F Feuille; citation_author=JL Ebersole; citation_author=L Kesavalu; citation_author=MJ Steffen; citation_author=SC Holt; citation_journal_title=Infect Immun; citation_volume=64; citation_number=85; citation_pages=2095-2100; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Rubrerythrin from Clostridium perfringens: cloning of the gene, purification of the protein, and characterization of its superoxide dismutase function.; citation_author=Y Lehmann; citation_author=L Meile; citation_author=M Teuber; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=86; citation_pages=7152-7158; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Contribution of glutathione peroxidase to the virulence of Streptococcus pyogenes.; citation_author=A Brenot; citation_author=KY King; citation_author=B Janowiak; citation_author=O Griffith; citation_author=MG Caparon; citation_journal_title=Infect Immun; citation_volume=72; citation_number=87; citation_pages=408-413; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Aerotolerance and peroxide resistance in peroxidase and PerR mutants of Streptococcus pyogenes.; citation_author=KY King; citation_author=JA Horenstein; citation_author=MG Caparon; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=88; citation_pages=5290-5299; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Multiple thioredoxin-mediated routes to detoxify hydroperoxides in Mycobacterium tuberculosis.; citation_author=T Jaeger; citation_author=H Budde; citation_author=L Flohe; citation_author=U Menge; citation_author=M Singh; citation_journal_title=Arch Biochem Biophys; citation_volume=423; citation_number=89; citation_pages=182-191; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The glutathione-glutaredoxin system in Rhodobacter capsulatus: part of a complex regulatory network controlling defense against oxidative stress.; citation_author=K Li; citation_author=S Hein; citation_author=W Zou; citation_author=G Klug; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=90; citation_pages=6800-6808; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Oxidative stress response in Clostridium perfringens.; citation_author=D Jean; citation_author=V Briolat; citation_author=G Reysset; citation_journal_title=Microbiol; citation_volume=150; citation_number=91; citation_pages=1649-1659; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolutionary relationships of Fusobacterium nucleatum based on phylogenetic analysis and comparative genomics.; citation_author=A Mira; citation_author=R Pushker; citation_author=BA Legault; citation_author=D Moreira; citation_author= Rodriguez-Valera; citation_journal_title=BMC Evol Biol; citation_volume=4; citation_number=92; citation_pages=50-58; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Proposal of three subspecies of Fusobacterium nucleatum Knorr 1922: Fusobacterium nucleatum subsp. nucleatum subsp. nov., comb. nov.; Fusobacterium nucleatum subsp. polymorphum subsp. nov., nom. rev., comb. nov.; and Fusobacterium nucleatum subsp. vincentii subsp. nov., nom. rev., comb. nov.; citation_author=JL Dzink; citation_author=MT Sheenan; citation_author=SS Socransky; citation_journal_title=Int J Syst Bacteriol; citation_volume=40; citation_number=93; citation_pages=74-78; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Rickettsia typhi and comparison with sequences of other rickettsiae.; citation_author=MP McLeod; citation_author=X Qin; citation_author=SE Karpathy; citation_author=J Gioia; citation_author=SK Highlander; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=94; citation_pages=5842-5855; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using Phred. II. Error probabilities.; citation_author=B Ewing; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=95; citation_pages=186-194; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using Phred. I. Accuracy assessment.; citation_author=B Ewing; citation_author=L Hillier; citation_author=MC Wendl; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=96; citation_pages=175-185; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The Atlas genome assembly system.; citation_author=P Havlak; citation_author=R Chen; citation_author=KJ Durbin; citation_author=A Egan; citation_author=Y Ren; citation_journal_title=Genome Res; citation_volume=14; citation_number=97; citation_pages=721-732; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=GeneMark.hmm: new solutions for gene finding.; citation_author=A Lukashin; citation_author=M Borodovsky; citation_journal_title=Nucl Acids Res; citation_volume=26; citation_number=98; citation_pages=1107-1115; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.; citation_author=TM Lowe; citation_author=SR Eddy; citation_journal_title=Nucl Acids Res; citation_volume=25; citation_number=99; citation_pages=955-964; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Rfam: an RNA family database.; citation_author=S Griffiths-Jones; citation_author=A Bateman; citation_author=M Marshall; citation_author=A Khanna; citation_author=SR Eddy; citation_journal_title=Nucl Acids Res; citation_volume=31; citation_number=100; citation_pages=439-441; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A systematic method to identify genomic islands and its applications in analyzing the genomes of Corynebacterium glutamicum and Vibrio vulnificus CMCP6 chromosome I.; citation_author=R Zhang; citation_author=C-T Zhang; citation_journal_title=Bioinform; citation_volume=20; citation_number=101; citation_pages=612-622; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucl Acids Res; citation_volume=22; citation_number=102; citation_pages=4673-4680; citation_date=1994; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000659" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genome Sequence of Fusobacterium nucleatum Subspecies Polymorphum — a Genetically Tractable Fusobacterium"/>
    <meta name="twitter:description" content="Fusobacterium nucleatum is a prominent member of the oral microbiota and is a common cause of human infection. F. nucleatum includes five subspecies: polymorphum, nucleatum, vincentii, fusiforme, and animalis. F. nucleatum subsp. polymorphum ATCC 10953 has been well characterized phenotypically and, in contrast to previously sequenced strains, is amenable to gene transfer. We sequenced and annotated the 2,429,698 bp genome of F. nucleatum subsp. polymorphum ATCC 10953. Plasmid pFN3 from the strain was also sequenced and analyzed. When compared to the other two available fusobacterial genomes (F. nucleatum subsp. nucleatum, and F. nucleatum subsp. vincentii) 627 open reading frames unique to F. nucleatum subsp. polymorphum ATCC 10953 were identified. A large percentage of these mapped within one of 28 regions or islands containing five or more genes. Seventeen percent of the clustered proteins that demonstrated similarity were most similar to proteins from the clostridia, with others being most similar to proteins from other gram-positive organisms such as Bacillus and Streptococcus. A ten kilobase region homologous to the Salmonella typhimurium propanediol utilization locus was identified, as was a prophage and integrated conjugal plasmid. The genome contains five composite ribozyme/transposons, similar to the CdISt IStrons described in Clostridium difficile. IStrons are not present in the other fusobacterial genomes. These findings indicate that F. nucleatum subsp. polymorphum is proficient at horizontal gene transfer and that exchange with the Firmicutes, particularly the Clostridia, is common."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000659.g005"/>

  <meta property="og:title" content="Genome Sequence of Fusobacterium nucleatum Subspecies Polymorphum — a Genetically Tractable Fusobacterium" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000659" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4162'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6728'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9772&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000659">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000659" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000659&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genome Sequence of <i>Fusobacterium nucleatum</i> Subspecies <i>Polymorphum</i> — a Genetically Tractable Fusobacterium
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sandor E. Karpathy
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Sandor E. Karpathy, Xiang Qin</p>

              
              

                <p>Affiliations:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America, 
                  Department of Microbiology and Molecular Genetics, University of Texas Medical School, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiang Qin
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Sandor E. Karpathy, Xiang Qin</p>

              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jason Gioia, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Huaiyang Jiang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yamei Liu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joseph F. Petrosino, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America, 
                  Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shailaja Yerrapragada, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            George E. Fox, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Susan Kinder Haake, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Associated Clinical Specialties, University of California at Los Angeles School of Dentistry, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            George M. Weinstock, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America, 
                  Department of Microbiology and Molecular Genetics, University of Texas Medical School, Houston, Texas, United States of America, 
                  Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sarah K. Highlander
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:sarahh@bcm.tmc.edu">sarahh@bcm.tmc.edu</a></p>

                <p>Affiliations:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America, 
                  Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 01, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000659</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000659" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000659" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000659" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000659" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000659-g001" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000659-g002" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000659-t001" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000659-g003" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000659-t002" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000659-g004" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000659-g005" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000659-t003" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000659">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000659" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p><em>Fusobacterium nucleatum</em> is a prominent member of the oral microbiota and is a common cause of human infection. <em>F. nucleatum</em> includes five subspecies: <em>polymorphum</em>, <em>nucleatum</em>, <em>vincentii</em>, <em>fusiforme</em>, and <em>animalis</em>. <em>F. nucleatum</em> subsp. <em>polymorphum</em> ATCC 10953 has been well characterized phenotypically and, in contrast to previously sequenced strains, is amenable to gene transfer. We sequenced and annotated the 2,429,698 bp genome of <em>F. nucleatum</em> subsp. <em>polymorphum</em> ATCC 10953. Plasmid pFN3 from the strain was also sequenced and analyzed. When compared to the other two available fusobacterial genomes (<em>F. nucleatum</em> subsp. <em>nucleatum</em>, and <em>F. nucleatum</em> subsp. <em>vincentii</em>) 627 open reading frames unique to <em>F. nucleatum</em> subsp. <em>polymorphum</em> ATCC 10953 were identified. A large percentage of these mapped within one of 28 regions or islands containing five or more genes. Seventeen percent of the clustered proteins that demonstrated similarity were most similar to proteins from the clostridia, with others being most similar to proteins from other gram-positive organisms such as <em>Bacillus</em> and <em>Streptococcus</em>. A ten kilobase region homologous to the <em>Salmonella typhimurium</em> propanediol utilization locus was identified, as was a prophage and integrated conjugal plasmid. The genome contains five composite ribozyme/transposons, similar to the Cd<em>ISt</em> IStrons described in <em>Clostridium difficile</em>. IStrons are not present in the other fusobacterial genomes. These findings indicate that <em>F. nucleatum</em> subsp. <em>polymorphum</em> is proficient at horizontal gene transfer and that exchange with the <em>Firmicutes</em>, particularly the <em>Clostridia</em>, is common.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Karpathy SE, Qin X, Gioia J, Jiang H, Liu Y, et al.  (2007) Genome Sequence of <em>Fusobacterium nucleatum</em> Subspecies <em>Polymorphum</em> — a Genetically Tractable Fusobacterium. PLoS ONE 2(8):
          e659.
            doi:10.1371/journal.pone.0000659</p><p><strong>Academic Editor: </strong>Cecile Fairhead, Pasteur Institute, France</p><p><strong>Received:</strong> May 7, 2007; <strong>Accepted:</strong> June 26, 2007; <strong>Published:</strong> August 1, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>This project was supported by a grant from the National Institute of Dental and Craniofacial Research to George Weinstock. (R01 DE013759). Susan Kinder Haake was supported by R01 DEO15348.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Fusobacterium nucleatum</em> is a Gram-negative anaerobic species of the phylum <em>Fusobacteria</em>, numerically dominant in dental plaque biofilms, and important in biofilm ecology and human infectious diseases. Dental plaque is a complex and dynamic microbial community that forms as a biofilm on teeth, and harbors more that 400 distinct species <em>in vivo</em> <a href="#pone.0000659-Paster1">[1]</a>. <em>F. nucleatum</em> is a prominent component quantitatively and is one of the first Gram-negative species to become established in plaque biofilms <a href="#pone.0000659-XimnezFyvie1">[2]</a>, <a href="#pone.0000659-Kononen1">[3]</a>. It is a central species in physical interactions between Gram-positive and Gram-negative species <a href="#pone.0000659-Kolenbrander1">[4]</a> that are likely to be important in biofilm colonization, and contributes to the reducing conditions necessary for the emergence of oxygen-intolerant anaerobes <a href="#pone.0000659-Diaz1">[5]</a>. <em>F. nucleatum</em> is also one of a small number of oral species that is consistently associated with, and increased in number at, sites of periodontitis <a href="#pone.0000659-Paster1">[1]</a>, <a href="#pone.0000659-XimnezFyvie1">[2]</a>, <a href="#pone.0000659-Moore1">[6]</a>, one of the most common infections of humans <a href="#pone.0000659-Brown1">[7]</a>. It is one of the most common oral species isolated from extraoral infections, including blood, brain, chest, lung, liver, joint, abdominal and obstetrical and gynecological infections and abscesses <a href="#pone.0000659-Moore1">[6]</a>, <a href="#pone.0000659-Brook1">[8]</a>–<a href="#pone.0000659-JousimiesSomer1">[14]</a>. Further, <em>F. nucleatum</em> is a common anaerobic isolate from intrauterine infections and has been associated with pregnancy complications including the delivery of premature low birth weight infants <a href="#pone.0000659-Holst1">[13]</a>, <a href="#pone.0000659-Cahill1">[15]</a>–<a href="#pone.0000659-Mikamo2">[19]</a>. Thus, <em>F. nucleatum</em> is a significant pathogen in human infections, including several infections with significant societal impact.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>There are five recognized subspecies of <em>F. nucleatum</em>: <em>polymorphum</em>, <em>nucleatum</em>, <em>vincentii</em>, <em>fusiforme</em>, and <em>animalis</em>. DNA sequence analysis of the genome of <em>F. nucleatum</em> subspecies <em>nucleatum</em> (ATCC 25586) (FNN) and <em>vincentii</em> (ATCC 49256) (FNV) indicate clear differences in genetic content between the strains. Three hundred forty-six 346 (17%) and 441 (20%) of the ATCC 25586 and ATCC 49256 open reading frames (ORFs) are absent from the other strain, respectively, and rearrangements are evident in the ORFs present in both strains <a href="#pone.0000659-Kapatral1">[20]</a>, <a href="#pone.0000659-Kapatral2">[21]</a>. Phenotypic heterogeneity among <em>F. nucleatum</em> strains has led to the concept that it is a “species complex” <a href="#pone.0000659-Morris1">[22]</a>, <a href="#pone.0000659-Bolstad1">[23]</a>. Taxonomic studies confirm that <em>F. nucleatum</em> ssp. <em>polymorphum</em> ATCC 10953 (FNP) represents a phylogenetic branch separate from the previously sequenced <em>F. nucleatum</em> strains. This branch includes significant human pathogens <a href="#pone.0000659-Goldstein1">[12]</a>, <a href="#pone.0000659-Conrads1">[24]</a>, <a href="#pone.0000659-Gmur1">[25]</a>. Phenotypic investigations of ATCC 10953 have characterized its uptake and metabolism of amino acids, simple sugars and peptides <a href="#pone.0000659-Robrish1">[26]</a>–<a href="#pone.0000659-Rogers2">[29]</a>, its physical interaction with epithelial, connective tissue and host immune cells <a href="#pone.0000659-Tuttle1">[30]</a>–<a href="#pone.0000659-Han1">[32]</a>, its ability to modulate host immune cell function including the induction of apoptosis <a href="#pone.0000659-Tuttle1">[30]</a>, <a href="#pone.0000659-RibeiroSobrinho1">[33]</a>–<a href="#pone.0000659-Takada1">[35]</a>, its ability to enhance the survival of strict anaerobes in biofilm and planktonic multispecies cultures <a href="#pone.0000659-Diaz1">[5]</a>, <a href="#pone.0000659-Bradshaw1">[36]</a>, and its ability act synergistically with other oral pathogens to enhance virulence in animal model systems <a href="#pone.0000659-Kuriyama1">[37]</a>, <a href="#pone.0000659-Takemoto1">[38]</a>. In addition, ATCC 10953 harbors a single native plasmid, pFN3 <a href="#pone.0000659-KinderHaake1">[39]</a>, and has been shown to be amenable to gene transfer <a href="#pone.0000659-Haake1">[40]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The distinct taxonomic status, relatively extensive phenotypic analyses, and genetic transformability of <em>F. nucleatum</em> ssp <em>polymorphum</em> ATCC 10953 suggested that genomic analysis would prove valuable to subsequent studies of this species. Thus, we determined, analyzed, and present the genomic DNA sequence of <em>F. nucleatum</em> subspecies <em>polymorphum</em> strain ATCC 10953. Analysis of the FNP genome revealed that 25% of the genes identified are not represented in the previously sequenced fusobacterial genomes, and that horizontal gene transfer (HGT) has contributed to the evolution of this strain.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Genome anatomy</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The FNP genome consists of a single circular chromosome containing 2,429,698 bp (<a href="#pone-0000659-g001">Figure 1</a>, accession number CM000440) and a single circular plasmid of 11,934 bp (<a href="#pone-0000659-g002">Figure 2</a>, accession number CP111710). The FNP genome is larger than the FNN genome (2,174,499 bp), and the unfinished FNV genome (2,118,259 bp). The GC content of the chromosome is 26.84%, similar to FNN and FNV (<a href="#pone-0000659-t001">Table 1</a>) and the GC content for the plasmid is 24.53%. There are 2,433 predicted ORFs, 42 pseudogenes, 45 tRNA, 15 rRNA, and 11 ncRNA genes in the FNP genome.</p>
<div class="figure" id="pone-0000659-g001"><div class="img"><a name="pone-0000659-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000659.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000659.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Map of the FNP ATCC 10953 genome.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The inner circle (orange) shows the percent GC calculated using a sliding window of 5 kb. The triangles in the next circle show the location and directionality of tRNAs (red) and ncRNAs (blue). The next tract shows the coordinate scale; this is surrounded by the ORFs, on both strands. ORFs are colored by category, as follows: tan, cell processes; purple, cell structure; red; DNA replication and recombination; blue, general metabolism; green, regulation; yellow, transcription; orange, translation, cyan, transport; fuchsia, virulence; and black, unknown. The IStrons are indicated by the fuchsia arrowheads on the outside circle; the intact IStron is indicated with the star. Plasmid (green) and phage locations (cyan) also appear on the outside circle.</p>
<span>doi:10.1371/journal.pone.0000659.g001</span></div><div class="figure" id="pone-0000659-g002"><div class="img"><a name="pone-0000659-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000659.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000659.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Plasmid pFN3.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>a) Map of plasmid pFN3. Replication and recombination ORFs are shown in blue and hypotheticals are colored green. b) Alignments of fusobacterial relaxase protein domains to mobilization class consensus motifs <a href="#pone.0000659-Ilyina1">[66]</a>. Consensus sequence abbreviations: uppercase letters, conserved; lowercase letters, present in 50% of sites; U or u, bulky hydrophobic residues (I, L, V, M, F, Y and W); -, no consensus at this site; <span class="underline">Y</span>, putative active site tyrosine residue. Asterisks (*) above residues indicate identity with consensus sequence. Alignments were performed using Clustal W <a href="#pone.0000659-Thompson1">[102]</a> and then adjusted to best fit the consensus.</p>
<span>doi:10.1371/journal.pone.0000659.g002</span></div><div class="figure" id="pone-0000659-t001"><div class="img"><a name="pone-0000659-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000659.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000659.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>General genome statistics for FNP, FNN and FNV.</span></strong></p><span>doi:10.1371/journal.pone.0000659.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Forty-two of the tRNAS lie in one of seven clusters, each containing from two to fourteen tRNAs. Identical clusters, in both content and internal gene order, are found in FNN. However, the relative position of the clusters in the genome is different. In addition, FNN has two additional asparagine tRNAs, which are associated with rRNA gene regions that have not been fully sequenced in ATCC 1095300 so it is possible that these tRNAs are present in FNP. The tRNAs represent all twenty amino acids. In five cases (Ala, Cys, His, Trp, Tyr), there is only a single tRNA. Many of the tRNAs are duplicated (Asn, Gln, Gly, Leu, Pro, Lys, Ser, Thr, Val) or triplicated (Asp, Glu). The clusters include duplicates as well as singleton tRNAs with no clear pattern associated with the distribution of singletons, duplicates and triplicates. Some tRNAs have multiple copies only one of which is unique, e.g. Gly, Lys, Ser, Val. All four Arg tRNAs, the three Met tRNAs (one is identified as the likely initiator) and the two Phe tRNAs have unique sequences.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>A 131 bp repeat sequence occurs seventeen times within the genome (<a href="#pone-0000659-g003">Figure 3</a>). In several cases, the repeat regions have the potential to encode small hypothetical peptides, which we believe were incorrectly annotated as ORFs in FNN and FNV. All of the 131 nt. repeats from FNP were aligned and used to generate a consensus sequence for additional BLASTN searches. Five nearly complete repeats (≥90% of the element's length) and eight repeats with gaps were observed. Sequences corresponding to the 3′ half of the element were also found in six locations. In most cases, the repeat sequence was found in intergenic regions and not within coding sequence. One complete copy of the repeat and numerous subsequences were identified in FNN and twenty-eight complete copies plus numerous subsequences were identified in FNV. As in FNP, these fell within intergenic regions. Because the long repeats occur within intergenic regions, it is possible that the sequence is involved in gene regulation, though no particular regulatory motifs were found within the sequence.</p>
<div class="figure" id="pone-0000659-g003"><div class="img"><a name="pone-0000659-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000659.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000659.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Intergenic repeats.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>The repeats were aligned using ClustalW <a href="#pone.0000659-Thompson1">[102]</a> and conserved bases were shaded using BOXSHADE (<a href="http://www.ch.embnet.org/software/BOX_form.html">www.ch.embnet.org/software/BOX_form.html</a>). Coordinates are shown in the left column.</p>
<span>doi:10.1371/journal.pone.0000659.g003</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Many examples of conservation of gene order, operon structure and gene clustering were observed in the genome. Most of the ribosomal protein genes are organized into operons similar to the L11, L10, <em>str</em>, S10, <em>spc</em> and α operons in <em>Escherichia coli</em>; each encodes between two and eleven ribosomal proteins <a href="#pone.0000659-Lindahl1">[41]</a>. Several non-r-protein clusters, similar to conserved non-r-protein gene clusters described in <em>E.coli</em> and other bacteria <a href="#pone.0000659-Siefert1">[42]</a> were also identified in the FNP genome. <a href="#pone-0000659-t002">Table 2</a> shows the conservation of these gene regions between FNP, FN, <em>Clostridium difficile</em> (NC_009089), <em>Bacillus anthracis str. Ames</em> (NC_003997), <em>E.coli</em> MG1655 (NC_000913). The conserved gene clusters were categorized in five groups according to protein function. Clusters 1–11 in group I are primarily RNA and protein constituents of the ribosome. Group II encodes subunits of the F-type two-sector ATPase. Proteins encoded by the group III clusters are involved in RNA synthesis, modification, transcription, and translation. Cluster 17 in group IV, encodes a spermidine/putrescine ATP binding cassette (ABC) superfamily transporter, while group V contains cluster 18, which codes for the molecular chaperones GroEL and GroES.</p>
<div class="figure" id="pone-0000659-t002"><div class="img"><a name="pone-0000659-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000659.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000659.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Conserved gene clusters/operons in FNP.</span></strong></p><span>doi:10.1371/journal.pone.0000659.t002</span></div>

<h4>Genome Comparisons</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Slightly more than 62% of the FNP genes (1514) are found in both of the previously sequenced fusobacterial genomes. Thus, nearly 38% of the genome is either wholly unique to FNP or is shared by FNP and only one of the other two genomes. In terms of coding potential, these 919 genes account for the differences between FNP, FNN, and FNV and thus serve to distinguish FNP. Three comparative lists have been generated: ORFs unique to FNP with respect to FNN and FNV; ORFs common to FNP and FNV, but absent in FNN; and ORFs common to FNP and FNN, but absent in FNV (<a href="#pone.0000659.s001">Table S1</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>627 FNP ORFs have no ortholog in either FNN or FNV (<a href="#pone.0000659.s001">Table S1a</a>), including 106 conserved hypothetical proteins, 287 hypothetical proteins and 9 pseudogenes. Thirty-eight ORFs functioning in transport, including transporters of amino acids, oligopeptides, a siderophore, and divalent metals (Hg<sup>2+</sup>, Cu<sup>2+</sup>, Ni<sup>2+</sup>) are also unique. Seven additional membrane proteins, two phosphotransferases, and two beta-lactamases are also present only in FNP. Twenty-seven ORFs related to transcriptional regulation are unique to FNP, including a LuxS autoinducer ortholog and two sensor histidine kinases. Additionally, FNP encodes several unique proteins related to DNA modification. These include such functions as methylation, histone acetylation, recombination, integration, topoisomerase, and type I restriction and modification. FNP also contains numerous prophage and transpose genes not found in FNN or FNV. Four Tra conjugation genes (FNP_1868–1871) were found only in FNP. These are adjacent to a region encoding two proteins resembling Type IV secretion components, an outer membrane protein, and four hypothetical proteins. Thus, it is plausible that FNP may have obtained a Type IV secretion system via HGT. This region and the prophage sequences are discussed in more detail, below.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Ninety-six ORFs (5 are pseudogenes) are shared between FNP and FNV but are missing from FNN (<a href="#pone.0000659.s001">Table S1b</a>), including 20 conserved hypothetical proteins and 36 fusobacterial conserved hypothetical proteins. Citrate lyase, glutamate–ammonia lyase, serine-pyruvate aminotransferase, cholinephosphate cytidylyltransferase, sulfate reductase and malate dehydrogenase enzymes are shared by FNV and FNP. They also each contain two N-acetylmuramoyl-L-alanine amidases not found in FNN. Five regulatory proteins are common to FNP and FNV, including transcriptional regulators of the MarR, MerR, LysR, and Crp families, and the RNA polymerase sigma factor σ<sup>54</sup>.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Two hundred thirteen ORFs (5 are pseudogenes) are found in both FNP and FNN but not in FNV (<a href="#pone.0000659.s001">Table S1c</a>). The total may be inflated, however, because the FNV genome is incomplete and some ORFs may have been missed. This set includes 35 conserved hypothetical proteins and 49 fusobacterial conserved hypothetical proteins. There are 30 transport proteins in this group, including iron transporters, a siderophore transporter, amino acid symporters, ion symporters, and a formate/nitrate transporter. Twelve ORFs related to genetic regulation are in this group, including RpiR and TetR family regulators, Fur, an iron-dependent transcriptional regulator and a response regulator and sensor histidine kinase for the ethanolamine utilization pathway. Also common to FNP and FNN are the GroeSL chaperonins, a cold shock protein, a ribosome-related heat shock protein, and a translation inhibitor. Other proteins of interest in this group are a beta-lactamase, bacterioferrin, at least 12 genes encoding the ethanolamine utilization gene family, an autotransporter/adhesin, an O-antigen assembly gene, 5 glycosyltransferases and the gene for a possible immunosuppressive protein FipA <a href="#pone.0000659-Demuth1">[43]</a>, <a href="#pone.0000659-HuntGerardo1">[44]</a>.</p>


<h4>Horizontal Gene Transfer</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>HGT can be detected by several parametric methods based on deviant nucleotide composition <a href="#pone.0000659-Lawrence1">[45]</a>, <a href="#pone.0000659-Lawrence2">[46]</a>, dinucleotide frequencies <a href="#pone.0000659-Karlin1">[47]</a>, codon usage biases <a href="#pone.0000659-Mrazek1">[48]</a>–<a href="#pone.0000659-Moszer1">[50]</a> or patterns inferred by Markov chain analysis <a href="#pone.0000659-Hayes1">[51]</a>. Phylogenetic methods determine a gene's unusual similarity or distribution among organisms by comparing phylogenetic trees of different genes from the genome and assessing the significance of any resulting incongruities. Alternative phylogenetic methods exist that do not reconstruct phylogenetic trees like Clarke's phylogenetic discordance test <a href="#pone.0000659-Ragan1">[52]</a> and Lawrence's <a href="#pone.0000659-Lawrence3">[53]</a> rank correlation test. Another reliable inference of recent HGT events is the anomalous phylogenetic distribution method wherein a gene is present in one genome but not found in several closely related genomes <a href="#pone.0000659-Philippe1">[54]</a>. This is the approach used examine genes that had no top BLAST hit to either of the two sequenced fusobacterial genomes, FNN and FNV.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Based on BLASTP similarity searches, a total of 1235 ORFs, composed of the 621 FNP ORFs and 9 pseudogenes with no top hits to FNN or FNV (<a href="#pone.0000659.s001">Table S1a</a>) and 608 hypothetical or conserved hypothetical proteins, were graphically plotted to identify clusters that could represent regions of HGT (<a href="#pone-0000659-g004">Figure 4</a>). About 21% of these, or 255 ORFs, mapped within gene clusters. There were 28 specific regions or islands of interest with clusters of 5 or more genes. Top BLASTP hits for each cluster (<a href="#pone.0000659.s002">Table S2</a>) were examined to determine a consensus genus and species.</p>
<div class="figure" id="pone-0000659-g004"><div class="img"><a name="pone-0000659-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000659.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000659.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Whole genome display of FNP illustrating clustering of genes without hits in FNN or FNV.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Yellow boxes represent FNP genes with either FNN or FNV as top BLASTN hits (1835/2462 or 75%) and blue boxes represent genes whose top BLASTN hits are to genes from other organisms (627/2462 or 25%).</p>
<span>doi:10.1371/journal.pone.0000659.g004</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>One hundred forty of the ORFs (out of 255), or 55% were hypothetical proteins with no matches to other bacterial proteins. Of the remaining 115, 20 ORFs or 17% had top hits to the <em>Clostridia</em>. The most common top hits in this class were to <em>Clostridium tetani</em>, <em>Clostridium thermocellum, Clostridium perfringens,</em> and <em>Desulfitobacterium hafniense</em>. Other ORFs had top hits to other <em>Firmicutes</em> including <em>Bacillus</em>, <em>Streptococcus, Listeria</em> and <em>Enterococcus</em> species. Hits to the archaea <em>Methanosarcina mazei, Methanococcoides burtonii,</em> and <em>Methanothermobacter</em> species, as well as to cyanobacteria <em>Nostoc punctiforme</em>, <em>Trichodesmium erythraeum</em> and <em>Synechocystis</em> sp., were also observed.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>A 10 kilobase (kb) region of the FNP genome from nt. 27349 to 37954 (FNP_2111–FNP_2124) appears to have arisen via HGT since this region is not found in the other published fusobacterial genomes and since its GC content (30.4%) is higher than that of the remainder of the genome (26.8%). In the <em>Firmicutes</em>, the clostridial %GC ranges from 28.5 to 30.9 so the DNA may have been acquired from this genus. The region includes 14 predicted genes, 5 of which compose Cluster I (<a href="#pone.0000659.s002">Table S2</a>). Twelve of the genes have no orthologs in FNN or FNV. The genes in this region are homologous to the propanediol utilization locus (<em>pdu</em>) of <em>Salmonella enterica</em> serovar typhimurium, which also arose via HGT <a href="#pone.0000659-Bobik1">[55]</a>. Propanediol is a byproduct produced during the fermentation of fucose <a href="#pone.0000659-Bobik1">[55]</a>, <a href="#pone.0000659-Bobik2">[56]</a>, which has been shown to be present in saliva and metabolized by oral bacteria <a href="#pone.0000659-Leach1">[57]</a>. Although it appears that FNP is missing the fucose catabolism (<em>fuc</em>) operon, some bacteria such as <em>E. coli</em> secrete propanediol <a href="#pone.0000659-Lin1">[58]</a>, so it is possible that FNP can utilize this propanediol pool.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>According to Kapatral <em>et al.</em>, the genomes of FNN and FNV lack the necessary enzymes for valine, isoleucine, and leucine biosynthesis <a href="#pone.0000659-Kapatral1">[20]</a>, <a href="#pone.0000659-Kapatral2">[21]</a>. A region in the FNP genome carries the <em>ilv</em>/<em>leu</em> operon, which is responsible for the biosynthesis of these amino acids. The predicted products encoded by this locus include dihydroxy-acid dehydratase (IlvD, FNP_0059), threonine ammonia-lyase (IlvA, FNP_0060), acetolactate synthase (IlvB and IlvN, FNP_0061 and FNP_0062), 2-isopropyl malate synthase (LeuA, FNP_0063), 3-isopropyl malate synthase (LeuC and LeuD, FNP_0064 and FNP_0065), isopropyl malate dehydrogenase (LeuB, FNP_0067), and ketol-acid reductoisomerase (IlvC, FNP_0069). The cluster of unique genes (Cluster IV) also includes a small hypothetical protein gene between <em>leuD</em> and <em>leuB</em> (FNP_0066). With the exception of <em>ilvA</em>, all of these genes are missing from the genomes of the other two sequenced fusobacteria. Three additional <em>ilv</em> genes are located at non-adjacent loci in the genome including an additional copy of <em>ilvA</em> (FNP_1302), which is not in the other genomes, and two copies of <em>ilvE</em> (branched-chain-amino-acid transaminase), one that is unique to <em>F. nucleatum</em> ATCC 10953 (FNP_1952) and one that is also found in the other two genomes (FNP_1165).</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>A prophage genome was identified immediately downstream of an arginine tRNA gene between coordinates 2024189 and 2053649 (28.9% GC) (<a href="#pone-0000659-g001">Figures 1</a> and <a href="#pone-0000659-g005">5</a>). The ORFs are not found in FNN or FNV (<a href="#pone.0000659.s001">Table S1a</a>). Forty-two open reading frames (FNP_1662–1703) were predicted in this region spanning four clusters of unique genes (XXI–XXIV), including genes encoding integrase, DNA polymerase, antirepressor, helicase, and terminase proteins. Several genes encoding bacteriophage structural components, such as capsid and tail proteins, were also identified in the region, though 20 of the open reading frames encode hypothetical proteins. The predicted tail proteins were most similar to those in a potential prophage genome in <em>C. tetani</em> E88 <a href="#pone.0000659-Bruggemann1">[59]</a>, while the terminase and packaging proteins were most similar to those in the <em>C. perfringens</em> phage phi3626 <a href="#pone.0000659-Zimmer1">[60]</a>. High scoring matches to the non-structural proteins were found in other gram-positive genomes, such as <em>Bacillus halodurans, Streptococcus mitis</em>, and <em>C. thermocellum</em>. Homologs of 10 of the phage proteins were found in FNV, though only one (a helicase, FNP_1671) was found in FNN. An additional block of phage-like genes map between coordinates 1775962 and 1786106 (FNP_1415–1432) (<a href="#pone-0000659-g001">Figure 1</a> and <a href="#pone.0000659.s002">Table S2</a>, Cluster XVIII). Only one of the 19 proteins encoded in this region had orthologs in the other fusobacteria and only two of the proteins (a replication protein and integrase) matched to other bacteriophage sequences.</p>
<div class="figure" id="pone-0000659-g005"><div class="img"><a name="pone-0000659-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000659.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000659.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Linear map of prophage located between nts 2,024,189 and 2,053,649 in FNP.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Replication and regulatory ORFs are colored blue, ORFs encoding structural proteins are red, ORFs encoding proteins with homologs in the nr database but of unknown function are colored green, and hypotheticals are shaded gray.</p>
<span>doi:10.1371/journal.pone.0000659.g005</span></div><a id="article1.body1.sec2.sec3.p7" name="article1.body1.sec2.sec3.p7"></a><p>The region between nts. 2174023 and 2218775 in the FNP genome (FNP_1820–1879, <a href="#pone-0000659-g001">Figure 1</a>), containing 59 genes that include Clusters XXV–XXVII (25.4% GC), is predicted to contain a large conjugal plasmid (<a href="#pone-0000659-g001">Figure 1</a>, <a href="#pone.0000659.s001">Table S1a</a>, <a href="#pone.0000659.s002">Table S2</a>). Fifty-five unique genes, not found in FNN or FNP, encode a primase/helicase that could function as a replication initiation protein, topoisomerase, integrase, recombinase, a plasmid partitioning protein, and pseudogenes of a mobilization protein and a plasmid addiction system. This region also carries genes encoding homologs of seven Type IV secretion system (T4SS) proteins. T4SS can translocate DNA and proteins out of the bacterial cell to recipient cells; bacterial conjugation systems are a subset of this family <a href="#pone.0000659-Christie1">[61]</a>. Full-length copies genes encoding the T4SS proteins VirB4, VirB8, VirB10, VirB11 and VirD4 (FNP_1868–1871, 1873, 1875) are found within the conjugal plasmid region, as are truncated versions of VirB6 and VirB9. These proteins are most similar to orthologs in <em>Mesorhizobium</em>, <em>Ralstonia</em>, <em>Pseudomonas</em>, <em>Caulobacter</em> and <em>Rhodopseudomonas</em>. The T4SS proteins identified in FNP could constitute the inner membrane and periplasmic components of the transporter, but genes encoding components for biogenesis of the T-pilus (VirB1, VirB2, VirB3, VirB5 and VirB7) are missing. A different set of proposed Type IV pilus genes are present in FNP. A cluster of eleven genes (FNP_2389–2399) plus an unlinked <em>pilT</em> gene may encode the pilus, as suggested by Desvaux, et al. <a href="#pone.0000659-Desvaux1">[62]</a>.</p>
<a id="article1.body1.sec2.sec3.p8" name="article1.body1.sec2.sec3.p8"></a><p>Five composite ribozyme/transposons, similar to the Cd<em>ISt</em> IStrons described in <em>C. difficile</em> <a href="#pone.0000659-Braun1">[63]</a> were identified in the genome (<a href="#pone-0000659-g001">Figure 1</a>). The consensus IStron in FNP is 1811 nt. long and contains a 477 nt. intron followed by an open reading frame encoding the transposase-like protein, TlpB. The FNP IStron is 31% identical to Cd<em>ISt</em>-C34 from <em>C. difficile</em> and contains four conserved RNA sequences that form the catalytic core of group I introns <a href="#pone.0000659-Cech1">[64]</a>. All five IStrons in FNP are inserted directly downstream of the pentanucleotide TTGAT, which is the conserved site of insertion of the IS<em>8301</em> family of transposons <a href="#pone.0000659-Kersulyte1">[65]</a>. The IStrons have an average G+C content of 29%, consistent with that of both fusobacteria and clostridia. In <em>C. difficile,</em> Cd<em>ISt1</em> self-splices to remove itself from the mRNA into which it is inserted. As a result, the insertion does not disrupt expression of the gene. In FNP, we predict that only one copy of the IStron (1361160 to 1362978) is a fully functional element with self-splicing and transposition activities, since the other copies have mutations in either the ribozyme or <em>tlpB</em> regions of the element. Three additional sequences with homology to portions of the ribozyme were identified in the FNP genome and ten additional copies of <em>tlpB</em>-like genes occur in the genome. Homologs of this element were not found in any other organism, including the two strains of <em>Fusobacterium</em> that have been previously sequenced, though TlpB sequences are found in a variety of organisms, including cyanobacteria, <em>Bacillus cereus, Enterococcus, Deinococcus</em> and <em>Exiguobacterium</em>. Thus, it appears that a unique exchange between <em>C. difficile</em> and FNP has occurred.</p>
<a id="article1.body1.sec2.sec3.p9" name="article1.body1.sec2.sec3.p9"></a><p>ATCC 10953 harbors a single plasmid, pFN3 <a href="#pone.0000659-Haake1">[40]</a> (<a href="#pone-0000659-g002">Figure 2a</a>), which is 11,934 bp in length and has a GC content of 24.53%. Eleven pFN3 ORFs were identified: two possible replication protein genes, a possible resolvase/recombinase gene, a DNA relaxation protein gene and seven hypothetical protein genes. The two replication protein genes (FNP_pFN3g01 and FNP_pFNgo5) have predicted protein sequences with 20–22% identity and 27–32% similarity to the putative replication protein of the <em>F. nucleatum</em> native plasmid pFN1 <a href="#pone.0000659-Haake1">[40]</a>. The sequence upstream of the pFN3 replication protein gene at 1315 has a sequence (1007 to 1136) characterized by clusters of two overlapping 18 bp repeats (repeat 1: TAATAGTACAAATTTCCC; repeat 2: <span class="gene">TAGTACAAATTTCCCGAT</span>). Several of the repeats are spaced at 22 bp intervals, suggesting that they may represent replication protein binding sites that are characteristic of the replication origin of iteron-regulated plasmids. The resolvase (FNP_pFN3g09) was identified based on the presence of a N-terminal resolvase domain (pfam02796). The DNA relaxation protein (relaxase) (FNP_pFN3g07) has a relaxase domain and contains the conserved consensus motifs defined for relaxase proteins <a href="#pone.0000659-Ilyina1">[66]</a> (<a href="#pone-0000659-g002">Figure 2b</a>). The pFN3 resolvase and relaxase genes both have potential significance for HGT. Resolvases are important in DNA recombination events, including excision and integration of mobile DNA elements. Relaxase proteins mediate the initiation of conjugal transfer of plasmid DNA. Plasmids that encode relaxases, which are not conjugative themselves, may be mobilized with the additional conjugative functions provided <em>in trans</em>. Two other native <em>F. nucleatum</em> plasmids, pFN1 (AF159249) and pPA52 (AF022647), which are 98% identical, also carry relaxase genes <a href="#pone.0000659-Haake1">[40]</a>. The occurrence of the relaxase genes suggests the possibility that these plasmids were introduced into <em>F. nucleatum</em> by conjugative processes. Consistent with this mechanism of HGT is the finding that plasmids or DNA sequences related to pPA52 have been detected in 18% of <em>F. nucleatum</em> strains examined <a href="#pone.0000659-McKay1">[67]</a>.</p>


<h4>Virulence</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We identified 132 predicted proteins that may play a role in fusobacterial virulence (<a href="#pone-0000659-t003">Table 3</a>). Most of these are found in FNN and FNV, though there are a few notable exceptions. As in the two previously sequenced fusobacterial genomes <a href="#pone.0000659-Kapatral1">[20]</a>, <a href="#pone.0000659-Kapatral2">[21]</a>, we identified a VacJ homolog (FNP_0314). This protein has been shown to play a role in the intracellular spread of <em>Shigella flexneri</em> <a href="#pone.0000659-Suzuki1">[68]</a>. Although its mechanism of action has not been examined, VacJ may play a similar role in FNP since recent evidence suggests that <em>F. nucleatum</em> can invade epithelial cells <a href="#pone.0000659-Han1">[32]</a>, which may allow dissemination throughout the host to cause infections at non-oral sites <a href="#pone.0000659-Han2">[69]</a>. Other previously known virulence factors were identified in FNP including the porin FomA (FNP_0972; not in FNV) <a href="#pone.0000659-Kaufman1">[70]</a>, <a href="#pone.0000659-KinderHaake2">[71]</a>, MviN (FNP_1360), which plays a role in virulence in <em>Salmonella typhimurium</em> <a href="#pone.0000659-Carsiotis1">[72]</a>, TraT (FNP_1881) which provides resistance to complement <a href="#pone.0000659-Sukupolvi1">[73]</a>, and VacB (FNP_1921), a ribonuclease involved in virulence gene expression in <em>S. flexneri</em> <a href="#pone.0000659-Tobe1">[74]</a>, <a href="#pone.0000659-Cheng1">[75]</a>. The strain also carries genes for butyrate fermentation (FNP_0790, 0791, 0969, 0970, 0971, 1762 and either 1467 or 2146). The production of butyrate has been associated with mouth odor and gingival inflammation <a href="#pone.0000659-Niederman1">[76]</a>. We also include FipA (described above) as a virulence factor because of its immunosuppressive properties <a href="#pone.0000659-Demuth1">[43]</a>, <a href="#pone.0000659-Shenker1">[77]</a>, though it is most similar to an acetyl-CoA transferase of the butyrate fermentation system. The <em>fipA</em> gene is not present in FNV.</p>
<div class="figure" id="pone-0000659-t003"><div class="img"><a name="pone-0000659-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000659" data-uri="info:doi/10.1371/journal.pone.0000659.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000659.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000659.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000659.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000659.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000659.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Potential virulence factor genes.</span></strong></p><span>doi:10.1371/journal.pone.0000659.t003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The acquisition of iron from the host environment is an important function of most bacterial pathogens <a href="#pone.0000659-Schaible1">[78]</a>. We have identified 26 predicted proteins involved in iron uptake in FNP. Three proteins, HmuV (FNP_2267), HmuU (FNP_2266), and HmuT (FNP_2269), form a heme ABC superfamily ATP binding cassette transporter while two additional proteins (FNP_2270 and FNP_1765) are probable TonB-dependent heme receptors. There are also two additional iron ATP binding cassette transporters (FNP_428–430 and FNP_1451–1454) and a cobalamin/iron ATP binding cassette transporter (FNP_0398–0341). An Nramp family iron transporter (FNP_1660), and an OfeT family oxidase-dependent iron transporter (FNP_0531) were also annotated. Three hemolysin genes were identified (FNP_0006, FNP_0159, and FNP_0999); two of these (FNP_0159 and FNP_0999) have associated TPS family two-partner secretion proteins (FNP_0155/0156 and FNP_1246/1247, respectively). This is similar to what is seen in FNV, but different than FNN, which has three such pairs <a href="#pone.0000659-Kapatral2">[21]</a>. Several of the iron transporters (FNP_0339, FNP_0426, FNP_0428, FNP_0531, and FNP_0769) are present in FNN but are missing in FNV; thus FNV may have a diminished requirement for iron or may occupy a different niche. As mentioned previously, a homolog of the ferric uptake regulator, Fur (FNP_2353), was identified in the FNP genome, though it is not present in FNV.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Sixteen possible drug transporters were annotated. These included 7 MOP/MATE family multidrug efflux pumps (FNP_0174, FNP_0640, FNP_0890, FNP_1162, FNP_1207, FNP_1299, and FNP_1596), 2 DMT superfamily drug/metabolite transporters (FNP_0388 and FNP_0622) and 2 RND family antiporters (FNP_0507 and FNP_0508). Our annotation did not permit us to predict the substrates of these transporters but it is likely that many of them are antibiotic transporters. With respect to antibiotic resistance, we also annotated 4 genes predicted to encode beta-lactamases. One of these (FNP_0627) is unique to FNP.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>We identified all but one (FN0387) of the 14 outer membrane protein genes described by Kapatral, et. al <a href="#pone.0000659-Kapatral1">[20]</a> (two, FNP_1046 and FNP_2283, have been re-annotated as AT family autotransporters) and we discovered a gene encoding OmpW (FNP_1248), which is not found in FNN or FNV. Four potential adhesion proteins were identified including a fibronectin-binding protein homolog (FNP_1337), a possible autotransporter adhesion (FNP_1391), and two proteins (FNP_1880 and FNP_1888) containing von Willebrand (vWF) type A domains. In addition to the Type IV secretion system discussed previously, FNP also carries genes that belong to the Type V secretion system. These include ten autotransporter genes (8 class 1, Type Va; 2 class 2, Type Vc) and the Tps secretion genes <em>tpsA</em> and <em>tpsB</em> (Type Vb) (<a href="#pone-0000659-t003">Table 3</a>). These are a subset of the genes found in FNN and FNV <a href="#pone.0000659-Desvaux1">[62]</a>.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>Twenty-five ORFs predicted to be involved in the biosynthesis of LPS were identified. This is of interest because <em>F. nucleatum</em> has been shown to have endotoxin activity <a href="#pone.0000659-Bolstad1">[23]</a>, <a href="#pone.0000659-Hase1">[79]</a>. Unlike FNN, however, FNP does not possess the <em>lic</em> operon, which is predicted to attach choline residues to the LPS <a href="#pone.0000659-Kapatral1">[20]</a>; only the <em>licC</em> and <em>licD</em> genes, encoding phosphocholine cytidylyltransferase and a phosphotransferase, respectively, are present. In contrast, FNP does contain genes (FNP_1105–1107) that encode N-acylneuraminate cytidylyltransferase, N-acetyl neuraminate synthase, N-acetylneuraminate synthase and a possible lipooligosaccharide sialyltransferase (FNP_1109) that might incorporate sialic acid into LPS, like FNV (note, however, that FNP_1109 is not found in FNN or FNV). This may facilitate evasion from the host immune response.</p>


<h4>Signal Transduction</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Six potential two-component signal transduction systems were revealed in FNP and three are of particular interest. An OmpR-related response regulator maps immediately upstream of a possible sensor histidine kinase gene. While the response regulator (FNP_2108) has homologs in both previously sequenced <em>F. nucleatum</em> genomes <a href="#pone.0000659-Kapatral1">[20]</a>, <a href="#pone.0000659-Kapatral2">[21]</a> (FN1261, FNV1053), the sensor protein (FNP_2107) is not present in either of these two genomes. Also unique to this region is an ORF (FNP_2109) immediately downstream of the response regulator that is not found in the other sequenced fusobacterial genomes. This protein appears to contain both sensor histidine kinase and response regulator domains and may represent a fusion of two two-component domains. Another system is a possible ethanolamine two-component regulatory system (FNP_0128 and FNP_0129). This system is present in the FNN genome but is not found in the FNV genome, though it may be located in one of the unfinished regions of that genome. Both FNN and FNV have genes encoding the YesM/YesN two-component regulatory system. In FNP the response regulator, YesN (FNP_0212) has been interrupted by the insertion of an IStron, The IStron at this locus is predicted to be inactive so <em>yesN</em> should be non-functional. In addition to the six possible two-component systems identified in FNP, there is unmatched response regulator and one unmatched sensor histidine kinase.</p>


<h4>Communication with other bacteria</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p><em>F. nucleatum</em> plays an important role in the formation of oral biofilms, or plaque. <em>F. nucleatum</em> is believed to act as a bridging organism between the Gram-positive early colonizers and the Gram-negative late colonizers <a href="#pone.0000659-Bolstad1">[23]</a>, <a href="#pone.0000659-Bradshaw1">[36]</a>, <a href="#pone.0000659-Kaufman1">[70]</a>. It has been proposed <a href="#pone.0000659-Bassler1">[80]</a> that some bacteria use a compound known as AI-2 (autoinducer-2) for intra-species communication. AI-2 signaling is also involved with biofilm formation <a href="#pone.0000659-Yoshida1">[81]</a>. We have identified the protein responsible for AI-2 synthesis, LuxS (FNP_1558). LuxS activity has been previously reported in ATCC 10953 and in three additional <em>F. nucleatum</em> strains <a href="#pone.0000659-Frias1">[82]</a>. There is no homolog of <em>luxS</em> in either of the two previously annotated <em>F. nucleatum</em> genomes.</p>


<h4>Oxidative stress</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Oral anaerobes must cope with oxidative stress to survive and contribute to pathogenesis. Established cultures of FNP are aerotolerent, indicating the presence of mechanisms to detoxify oxygen or oxygen radicals <a href="#pone.0000659-Diaz2">[83]</a>. The ability of <em>F. nucleatum</em> to manage oxidative stress and to maintain reduced conditions is thought to facilitate the survival of other anaerobic pathogens. This facet of <em>F. nucleatum</em> ecology may explain why less aerotolerant organisms such as <em>Porphyromonas gingivalis</em> are increased in number in the presence of <em>F. nucleatum</em> <a href="#pone.0000659-Bradshaw1">[36]</a>. Aeroprotection of bystander organisms may also explain the synergistic increase in virulence of <em>F. nucleatum</em> when combined with <em>Porphyromonas gingivalis</em>, as compared to either species alone <a href="#pone.0000659-Ebersole1">[84]</a>, <a href="#pone.0000659-Feuille1">[85]</a>. Physiological studies demonstrate that in response to oxidative stress, FNP maintains a reduced environment <a href="#pone.0000659-Diaz1">[5]</a>, with increases in NADH oxidase and superoxide dismutase activities <a href="#pone.0000659-Diaz2">[83]</a>. Our analysis of the FNP genome revealed an NADH oxidase (FNP_1794), most closely related to treponemal and streptococcal homologs. An ORF encoding a superoxide dismutase was not identified, though, a rubrerythrin protein (FNP_1721), which confers superoxide dismutase-like activity <a href="#pone.0000659-Lehmann1">[86]</a> and has homology to a <em>C. perfringens</em> ruberythrin, is present in FNP. Other FNP proteins of potential importance in oxidative stress include a glutathione peroxidase (FNP_2310) <a href="#pone.0000659-Brenot1">[87]</a>, <a href="#pone.0000659-King1">[88]</a>, a thioredoxin (FNP_0146) <a href="#pone.0000659-Jaeger1">[89]</a>, a glutaredoxin (FNP_1273) <a href="#pone.0000659-Li1">[90]</a>, and an alkyl hydroperoxide (FNP_2288) <a href="#pone.0000659-Jaeger1">[89]</a>, <a href="#pone.0000659-Jean1">[91]</a>. Orthologs of these genes are present in both the FNN and FNV genomes.</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Analysis of the genome sequence of <em>F. nucleatum</em> subsp. <em>polymorphum</em> revealed that this microorganism has obtained numerous genes from the <em>Firmicute</em> phylum of bacteria. In particular, many of the regions of FNP that were unique to the <em>Fusobacteria</em> encoded proteins with top BLAST hits to the <em>Clostridia</em>. This is perhaps not unexpected as <em>Firmicutes</em> were the largest phylotype isolated in a gingival bacterial diversity study, and the <em>Clostridia</em> represented the largest class within the <em>Firmicutes</em> that were isolated <a href="#pone.0000659-Paster1">[1]</a>. It appears that FNP was the recipient of DNA from bacteria that are common to the subgingival niche in HGT events. A contrasting interpretation, presented by Mira, <em>et al</em>. <a href="#pone.0000659-Mira1">[92]</a>, is that <em>Fusobacterium</em> represents a genus with a <em>Clostridial</em> metabolic apparatus that has obtained a gram-negative envelope from the Proteobacteria. If this is the case, then it appears that FNN and FNV have lost a complement of genes that are present in FNP. As was revealed in the comparison of FNN to FNV, the fusobacterial genomes are mosaic in structure.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Bacterial strain, growth conditions, and DNA isolation</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p><em>F. nucleatum</em> subsp. <em>polymorphum</em> ATCC 10953 is a human strain originally isolated from the inflamed gingiva of an adult male <a href="#pone.0000659-Dzink1">[93]</a>. The strain was acquired as a lyophil from the American Type Culture Collection and recovered on Columbia agar with 5% sheep blood under anaerobic growth conditions. Genomic DNA was prepared from an anaerobically cultivated Columbia broth culture, as previously described <a href="#pone.0000659-KinderHaake2">[71]</a>.</p>


<h4>Library construction and sequencing</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Genomic DNA of <em>F. nucleatum</em> ATCC 10953 was sheared to 2–6 kb in size using a nebulizer (CIS-US, Inc., Bedford, Mass.), purified from an agarose gel, cloned into a pUC18 derivative sequencing vector, and sequenced as previous described <a href="#pone.0000659-McLeod1">[94]</a>.</p>


<h4>Sequence assembly</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>PHRED <a href="#pone.0000659-Ewing1">[95]</a>, <a href="#pone.0000659-Ewing2">[96]</a> was used to determine the sequence and quality of each base of sequencing reads. Atlas genome assembly tools <a href="#pone.0000659-Havlak1">[97]</a> were used to process reads, to remove repetitive reads, and to bin overlapping reads before assembling the contigs with the program PHRAP <a href="#pone.0000659-Ewing1">[95]</a>. An initial assembly of 54,948 reads gave 146 contigs. Determination of linkages among contigs and gap closing was carried out as previously described <a href="#pone.0000659-McLeod1">[94]</a> except that repetitive reads were put back into the contigs by separated bin assemblies. The rRNAs were not completely sequenced and there are 14 short gaps in the sequence.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>The coordinate system for the genome was selected to be similar to other sequenced fusobacteria. In this system, the origin of replication is tentatively localized to the region upstream of an ORF with low identity to DnaA. <em>recF</em> and <em>gyrB</em> are found downstream of this putative <em>dnaA</em> gene. <em>dnaE</em> and <em>repB</em>, which are often clustered with <em>dnaA</em> around the origin of replication, are found elsewhere in the FNP genome, as they are in the other two <em>Fusobacterium</em> genomes</p>


<h4>Gene identification and annotation</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Gene prediction and manual annotation were performed as previously described <a href="#pone.0000659-McLeod1">[94]</a>. For GeneMark <a href="#pone.0000659-Lukashin1">[98]</a> gene predictions, <em>Borrelia burgdorferi</em> was used as model. Predicted proteins with unknown functions were classified into three categories: “hypothetical protein” for proteins that do not have matches in the GenBank NR database, “conserved hypothetical protein” for proteins that have matches to proteins of unknown functions from organisms outside the genus <em>Fusobacterium</em>, and “fusobacterial conserved hypothetical protein” for proteins that have matches only to other fusobacterial proteins of unknown functions.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Transfer RNAs were identified using tRNAscan-SE <a href="#pone.0000659-Lowe1">[99]</a>. Non-coding RNA (ncRNA) elements where identified by BLASTing the known ncRNA sequences of FNN, available at the Rfam database (<a href="http://www.sanger.ac.uk/Software/Rfam/">http://www.sanger.ac.uk/Software/Rfam/</a>), against the FNP genome <a href="#pone.0000659-GriffithsJones1">[100]</a>. The FNP regions containing potential ncRNAs were then analyzed by the Rfam database to identify the exact coordinates of each RNA element.</p>


<h4>Genome analysis and identification of possible HGT regions</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>We performed automatic and manual comparison of best hits to the GenBank NR database and BLAST results to the FNN and FNV genomes to find possible genomic islands and recent horizontally transferred genes in the FNP genome. Genes and regions of possible HGT were examined by GC content, cumulative GC profile, and codon usage bias analysis <a href="#pone.0000659-Zhang1">[101]</a> but this did not add significantly to the conclusions. ClustalW was also used to align DNA sequences and to identify repeat sequences in the genome <a href="#pone.0000659-Thompson1">[102]</a>.</p>


<h4>Analysis of intergenic regions</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>After completing annotation of all ORFs called by either GeneMark or Glimmer we decided to confirm that the intergenic regions (IGR) contained no ORFs missed by the ORF calling software. Thus, the entire nucleotide sequence of each IGR was compared to Genbank using BLASTX and those results that had an e value of less than 1×10<sup>−5</sup> were marked for further analysis by annotators. The annotators defined the exact coordinates of each hit and then followed the same process used for the called ORFs to assign the annotation for each region, resulting in the addition of 50 annotation entries. While this process did result in the identification of a few missed ORFs (5–10), the vast majority of IGR annotations were pseudogenes. Of note concerning this analysis is that it allowed us to extend proteins called as small hypothetical proteins into full-length pseudogenes.</p>


<h4>Database submission</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>The FNP ATCC 10953 genome sequence was deposited in GenBank and assigned accession number CM000440 and the plasmid was assigned accession number CP000710.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000659"></div><a name="pone.0000659.s001" id="pone.0000659.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000659.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">FNP ORFS not in FNN or FNV; FNP ORFS not in FNN; and FNP ORFS not found in FNV.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000659.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.17 MB DOC)</p>
<a name="pone.0000659.s002" id="pone.0000659.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000659.s002">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Clusters of ORFs unique to FNP and not in FNN or FNV.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000659.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.74 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Richard Gibbs, Donna Muzny, Christie Kovar-Smith, Lynne Nazareth, Erica Sodergren, David Parker, Aleks Milosavljevic, and the rest of the staff at the Human Genome Sequencing Center for their support during this project. Part of the DNA sequencing was performed at Seqwright, Inc., Houston, TX.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: QX SK GW SHa SHi XQ. Performed the experiments: SK SHi. Analyzed the data: JP QX SK HJ SY YL JG GF SHa SHi XQ. Contributed reagents/materials/analysis tools: QX HJ SY GF GW SHa SHi XQ. Wrote the paper: SK GW SHa SHi.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000659-Paster1" id="pone.0000659-Paster1"></a>Paster BJ, Boches SK, Galvin JL, Ericson RE, Lau CN, et al.  (2001) Bacterial diversity in human subgingival plaque. J Bacteriol  183: 3770–3783.  <ul class="find" data-citedArticleID="970348" data-doi="10.1128/jb.183.12.3770-3783.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.12.3770-3783.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+diversity+in+human+subgingival+plaque." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+diversity+in+human+subgingival+plaque.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000659-XimnezFyvie1" id="pone.0000659-XimnezFyvie1"></a>Ximénez-Fyvie LA, Haffajee AD, Socransky SS (2000) Comparison of the microbiota of supra- and subgingival plaque in health and periodontitis. J Clin Periodontol  27: 648–657.  <ul class="find" data-citedArticleID="970384" data-doi="10.1034/j.1600-051x.2000.027009648.x"><li><a href="http://dx.doi.org/10.1034/j.1600-051x.2000.027009648.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+the+microbiota+of+supra-+and+subgingival+plaque+in+health+and+periodontitis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+the+microbiota+of+supra-+and+subgingival+plaque+in+health+and+periodontitis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000659-Kononen1" id="pone.0000659-Kononen1"></a>Kononen E (2000) Development of oral bacterial flora in young children. Ann Med  32: 107–112.  <ul class="find" data-citedArticleID="970300" data-doi="10.3109/07853890009011759"><li><a href="http://dx.doi.org/10.3109/07853890009011759" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+oral+bacterial+flora+in+young+children." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+oral+bacterial+flora+in+young+children.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000659-Kolenbrander1" id="pone.0000659-Kolenbrander1"></a>Kolenbrander PE, London J (1993) Adhere today, here tomorrow: Oral bacterial adherence. J Bacteriol  175: 3247–3252.  <ul class="find" data-citedArticleID="970298"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Adhere+today%2C+here+tomorrow%3A+Oral+bacterial+adherence.&amp;auth=&amp;atitle=Adhere+today%2C+here+tomorrow%3A+Oral+bacterial+adherence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adhere+today%2C+here+tomorrow%3A+Oral+bacterial+adherence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adhere+today%2C+here+tomorrow%3A+Oral+bacterial+adherence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000659-Diaz1" id="pone.0000659-Diaz1"></a>Diaz PI, Zilm PS, Rogers AH (2002) <em>Fusobacterium nucleatum</em> supports the growth of <em>Porphyromonas gingivalis</em> in oxygenated and carbon-dioxide-depleted environments. Microbiol  148: 467–472.  <ul class="find" data-citedArticleID="970232"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fusobacterium+nucleatum+supports+the+growth+of+Porphyromonas+gingivalis+in+oxygenated+and+carbon-dioxide-depleted+environments.&amp;auth=&amp;atitle=Fusobacterium+nucleatum+supports+the+growth+of+Porphyromonas+gingivalis+in+oxygenated+and+carbon-dioxide-depleted+environments." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fusobacterium+nucleatum+supports+the+growth+of+Porphyromonas+gingivalis+in+oxygenated+and+carbon-dioxide-depleted+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fusobacterium+nucleatum+supports+the+growth+of+Porphyromonas+gingivalis+in+oxygenated+and+carbon-dioxide-depleted+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000659-Moore1" id="pone.0000659-Moore1"></a>Moore WE, Moore LV (1994) The bacteria of periodontal diseases. Periodontol 2000  5: 66–77.  <ul class="find" data-citedArticleID="970336" data-doi="10.1111/j.1600-0757.1994.tb00019.x"><li><a href="http://dx.doi.org/10.1111/j.1600-0757.1994.tb00019.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+bacteria+of+periodontal+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+bacteria+of+periodontal+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000659-Brown1" id="pone.0000659-Brown1"></a>Brown LJ, Loe H (2000) Prevalence, extent, severity and progression of periodontal disease. Periodontol 2000  2: 57–71.  <ul class="find" data-citedArticleID="970206" data-doi="10.1111/j.1600-0757.1993.tb00220.x"><li><a href="http://dx.doi.org/10.1111/j.1600-0757.1993.tb00220.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence%2C+extent%2C+severity+and+progression+of+periodontal+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence%2C+extent%2C+severity+and+progression+of+periodontal+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000659-Brook1" id="pone.0000659-Brook1"></a>Brook I, Frazier EH (1998) Microbiology of liver and spleen abscesses. J Med Microbiol  47: 1075–1080.  <ul class="find" data-citedArticleID="970204" data-doi="10.1099/00222615-47-12-1075"><li><a href="http://dx.doi.org/10.1099/00222615-47-12-1075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbiology+of+liver+and+spleen+abscesses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbiology+of+liver+and+spleen+abscesses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000659-Chaudhry1" id="pone.0000659-Chaudhry1"></a>Chaudhry R, Dhawan B, Laxmi BVJ, Mehta VS (1998) The microbial spectrum of brain abscess with special reference to anaerobic bacteria. Br J Neurosurg  12: 127–130.  <ul class="find" data-citedArticleID="970216" data-doi="10.1080/02688699845258"><li><a href="http://dx.doi.org/10.1080/02688699845258" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+microbial+spectrum+of+brain+abscess+with+special+reference+to+anaerobic+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+microbial+spectrum+of+brain+abscess+with+special+reference+to+anaerobic+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000659-Chryssagi1" id="pone.0000659-Chryssagi1"></a>Chryssagi AM, Brusselmans CB, Rombouts JJ (2001) Septic arthritis of the hip due to <em>Fusobacterium nucleatum</em>. Clin Rheumatol  20: 229–231.  <ul class="find" data-citedArticleID="970222" data-doi="10.1007/s100670170072"><li><a href="http://dx.doi.org/10.1007/s100670170072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Septic+arthritis+of+the+hip+due+to+Fusobacterium+nucleatum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Septic+arthritis+of+the+hip+due+to+Fusobacterium+nucleatum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000659-Civen1" id="pone.0000659-Civen1"></a>Civen R, Jousimies-Somer H, Marina M, Borenstein L, Shah H, et al.  (1995) A retrospective review of cases of anaerobic empyema and update of bacteriology. Clin Infect Dis  20: Suppl 2S224–229.  <ul class="find" data-citedArticleID="970224" data-doi="10.1093/clinids/20.supplement_2.s224"><li><a href="http://dx.doi.org/10.1093/clinids/20.supplement_2.s224" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+retrospective+review+of+cases+of+anaerobic+empyema+and+update+of+bacteriology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+retrospective+review+of+cases+of+anaerobic+empyema+and+update+of+bacteriology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000659-Goldstein1" id="pone.0000659-Goldstein1"></a>Goldstein EJ, Summanen PH, Citron DM, Rosove MH, Finegold SM (1995) Fatal sepsis due to a beta-lactamase-producing strain of <em>Fusobacterium nucleatum</em> subspecies <em>polymorphum</em>. Clin Infect Dis  20: 797–800.  <ul class="find" data-citedArticleID="970250" data-doi="10.1093/clinids/20.4.797"><li><a href="http://dx.doi.org/10.1093/clinids/20.4.797" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fatal+sepsis+due+to+a+beta-lactamase-producing+strain+of+Fusobacterium+nucleatum+subspecies+polymorphum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fatal+sepsis+due+to+a+beta-lactamase-producing+strain+of+Fusobacterium+nucleatum+subspecies+polymorphum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000659-Holst1" id="pone.0000659-Holst1"></a>Holst E, Goffeng AR, Andersch B (1994) Bacterial vaginosis and vaginal microorganisms in idiopathic premature labor and association with pregnancy outcome. J Clin Microbiol  32: 176–186.  <ul class="find" data-citedArticleID="970268"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bacterial+vaginosis+and+vaginal+microorganisms+in+idiopathic+premature+labor+and+association+with+pregnancy+outcome.&amp;auth=&amp;atitle=Bacterial+vaginosis+and+vaginal+microorganisms+in+idiopathic+premature+labor+and+association+with+pregnancy+outcome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+vaginosis+and+vaginal+microorganisms+in+idiopathic+premature+labor+and+association+with+pregnancy+outcome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+vaginosis+and+vaginal+microorganisms+in+idiopathic+premature+labor+and+association+with+pregnancy+outcome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000659-JousimiesSomer1" id="pone.0000659-JousimiesSomer1"></a>Jousimies-Somer H, Savolainen S, Makitie A, Ylikoski J (1993) Bacteriologic findings in peritonsillar abscesses in young adults. Clin Infect Dis 16 Suppl  4: S292–298.  <ul class="find" data-citedArticleID="970280" data-doi="10.1093/clinids/16.supplement_4.s292"><li><a href="http://dx.doi.org/10.1093/clinids/16.supplement_4.s292" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteriologic+findings+in+peritonsillar+abscesses+in+young+adults." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteriologic+findings+in+peritonsillar+abscesses+in+young+adults.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000659-Cahill1" id="pone.0000659-Cahill1"></a>Cahill RJ, Tan S, Dougan G, O'Gaora P, Pickard D, et al.  (2005) Universal DNA primers amplify bacterial DNA from human fetal membranes and link Fusobacterium nucleatum with prolonged preterm membrane rupture. Mol Hum Reprod  11: 761–766.  <ul class="find" data-citedArticleID="970210" data-doi="10.1093/molehr/gah234"><li><a href="http://dx.doi.org/10.1093/molehr/gah234" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Universal+DNA+primers+amplify+bacterial+DNA+from+human+fetal+membranes+and+link+Fusobacterium+nucleatum+with+prolonged+preterm+membrane+rupture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Universal+DNA+primers+amplify+bacterial+DNA+from+human+fetal+membranes+and+link+Fusobacterium+nucleatum+with+prolonged+preterm+membrane+rupture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000659-Bearfield1" id="pone.0000659-Bearfield1"></a>Bearfield C, Davenport ES, Sivapathasundaram V, Allaker RP (2002) Possible association between amniotic fluid micro-organism infection and microflora in the mouth. BJOG  109: 527–533.  <ul class="find" data-citedArticleID="970190" data-doi="10.1111/j.1471-0528.2002.01349.x"><li><a href="http://dx.doi.org/10.1111/j.1471-0528.2002.01349.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Possible+association+between+amniotic+fluid+micro-organism+infection+and+microflora+in+the+mouth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Possible+association+between+amniotic+fluid+micro-organism+infection+and+microflora+in+the+mouth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000659-Hill1" id="pone.0000659-Hill1"></a>Hill GB (1998) Preterm birth: associations with genital and possibly oral microflora. Ann Periodontol  3: 222–232.  <ul class="find" data-citedArticleID="970266" data-doi="10.1902/annals.1998.3.1.222"><li><a href="http://dx.doi.org/10.1902/annals.1998.3.1.222" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preterm+birth%3A+associations+with+genital+and+possibly+oral+microflora." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preterm+birth%3A+associations+with+genital+and+possibly+oral+microflora.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000659-Mikamo1" id="pone.0000659-Mikamo1"></a>Mikamo H, Kawazoe K, Sato Y, Imai A, Tamaya T (1998) Preterm labor and bacterial intraamniotic infection: arachidonic acid liberation by phospholipase A2 of <em>Fusobacterium nucleatum</em>. Am J Obstet Gynecol  179: 1579–1582.  <ul class="find" data-citedArticleID="970330" data-doi="10.1016/s0002-9378(98)70028-6"><li><a href="http://dx.doi.org/10.1016/s0002-9378(98)70028-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preterm+labor+and+bacterial+intraamniotic+infection%3A+arachidonic+acid+liberation+by+phospholipase+A2+of+Fusobacterium+nucleatum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preterm+labor+and+bacterial+intraamniotic+infection%3A+arachidonic+acid+liberation+by+phospholipase+A2+of+Fusobacterium+nucleatum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000659-Mikamo2" id="pone.0000659-Mikamo2"></a>Mikamo H, Kawazoe K, Sato Y, Tamaya T (1999) Elastase activity of anaerobes isolated from amniotic fluid with preterm premature rupture of membranes. Am J Obstet Gynecol  180: 378–380.  <ul class="find" data-citedArticleID="970332" data-doi="10.1016/s0002-9378(99)70217-6"><li><a href="http://dx.doi.org/10.1016/s0002-9378(99)70217-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elastase+activity+of+anaerobes+isolated+from+amniotic+fluid+with+preterm+premature+rupture+of+membranes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elastase+activity+of+anaerobes+isolated+from+amniotic+fluid+with+preterm+premature+rupture+of+membranes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000659-Kapatral1" id="pone.0000659-Kapatral1"></a>Kapatral V, Anderson I, Ivanova N, Reznik G, Los T, et al.  (2002) Genome sequence and analysis of the oral bacterium <em>Fusobacterium nucleatum</em> strain ATCC 25586. J Bacteriol  184: 2005–2018.  <ul class="find" data-citedArticleID="970282" data-doi="10.1128/jb.184.7.2005-2018.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.7.2005-2018.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+and+analysis+of+the+oral+bacterium+Fusobacterium+nucleatum+strain+ATCC+25586." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+and+analysis+of+the+oral+bacterium+Fusobacterium+nucleatum+strain+ATCC+25586.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000659-Kapatral2" id="pone.0000659-Kapatral2"></a>Kapatral V, Ivanova N, Anderson I, Reznik G, Bhattacharyya A, et al.  (2003) Genome analysis of <em>F. nucleatum</em> sub spp <em>vincentii</em> and its comparison with the genome of <em>F. nucleatum</em> ATCC 25586. Genome Res  13: 1180–1189.  <ul class="find" data-citedArticleID="970284" data-doi="10.1101/gr.566003"><li><a href="http://dx.doi.org/10.1101/gr.566003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+analysis+of+F.+nucleatum+sub+spp+vincentii+and+its+comparison+with+the+genome+of+F.+nucleatum+ATCC+25586." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+analysis+of+F.+nucleatum+sub+spp+vincentii+and+its+comparison+with+the+genome+of+F.+nucleatum+ATCC+25586.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000659-Morris1" id="pone.0000659-Morris1"></a>Morris ML, Andrews RH, Rogers AH (1996) The use of allozyme electrophoresis to assess genetic heterogeneity among previously subspeciated isolates of <em>Fusobacterium nucleatum</em>. Oral Microbiol Immunol  11: 15–21.  <ul class="find" data-citedArticleID="970338" data-doi="10.1111/j.1399-302x.1996.tb00331.x"><li><a href="http://dx.doi.org/10.1111/j.1399-302x.1996.tb00331.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+allozyme+electrophoresis+to+assess+genetic+heterogeneity+among+previously+subspeciated+isolates+of+Fusobacterium+nucleatum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+allozyme+electrophoresis+to+assess+genetic+heterogeneity+among+previously+subspeciated+isolates+of+Fusobacterium+nucleatum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000659-Bolstad1" id="pone.0000659-Bolstad1"></a>Bolstad AI, Jensen HB, Bakken V (1996) Taxonomy, biology, and periodontal aspects of <em>Fusobacterium nucleatum</em>. Clin Microbiol Rev  9: 55–71.  <ul class="find" data-citedArticleID="970196"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Taxonomy%2C+biology%2C+and+periodontal+aspects+of+Fusobacterium+nucleatum.&amp;auth=&amp;atitle=Taxonomy%2C+biology%2C+and+periodontal+aspects+of+Fusobacterium+nucleatum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Taxonomy%2C+biology%2C+and+periodontal+aspects+of+Fusobacterium+nucleatum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Taxonomy%2C+biology%2C+and+periodontal+aspects+of+Fusobacterium+nucleatum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000659-Conrads1" id="pone.0000659-Conrads1"></a>Conrads G, Claros MC, Citron DM, Tyrrell KL, Merriam V, et al.  (2002) 16S-23S rDNA internal transcribed spacer sequences for analysis of the phylogenetic relationships among species of the genus <em>Fusobacterium</em>. Int J Syst Evol Microbiol  52: 493–499.  <ul class="find" data-citedArticleID="970226"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=16S-23S+rDNA+internal+transcribed+spacer+sequences+for+analysis+of+the+phylogenetic+relationships+among+species+of+the+genus+Fusobacterium.&amp;auth=&amp;atitle=16S-23S+rDNA+internal+transcribed+spacer+sequences+for+analysis+of+the+phylogenetic+relationships+among+species+of+the+genus+Fusobacterium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=16S-23S+rDNA+internal+transcribed+spacer+sequences+for+analysis+of+the+phylogenetic+relationships+among+species+of+the+genus+Fusobacterium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%2216S-23S+rDNA+internal+transcribed+spacer+sequences+for+analysis+of+the+phylogenetic+relationships+among+species+of+the+genus+Fusobacterium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000659-Gmur1" id="pone.0000659-Gmur1"></a>Gmur R, Munson RA, Wade WG (2006) Genotypic and phenotypic characterization of fusobacteria from Chinese and European patients with inflammatory periodontal disease. Syst App Microbiol  29: 120–130.  <ul class="find" data-citedArticleID="970248" data-doi="10.1016/j.syapm.2005.07.011"><li><a href="http://dx.doi.org/10.1016/j.syapm.2005.07.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotypic+and+phenotypic+characterization+of+fusobacteria+from+Chinese+and+European+patients+with+inflammatory+periodontal+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotypic+and+phenotypic+characterization+of+fusobacteria+from+Chinese+and+European+patients+with+inflammatory+periodontal+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000659-Robrish1" id="pone.0000659-Robrish1"></a>Robrish SA, Oliver C, Thompson J (1987) Amino acid-dependent transport of sugars by <em>Fusobacterium nucleatum</em> ATCC 10953. J Bacteriol  169: 3891–3897.  <ul class="find" data-citedArticleID="970356"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Amino+acid-dependent+transport+of+sugars+by+Fusobacterium+nucleatum+ATCC+10953.&amp;auth=&amp;atitle=Amino+acid-dependent+transport+of+sugars+by+Fusobacterium+nucleatum+ATCC+10953." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+acid-dependent+transport+of+sugars+by+Fusobacterium+nucleatum+ATCC+10953." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+acid-dependent+transport+of+sugars+by+Fusobacterium+nucleatum+ATCC+10953.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000659-Robrish2" id="pone.0000659-Robrish2"></a>Robrish SA, Thompson J (1990) Regulation of fructose metabolism and polymer synthesis by <em>Fusobacterium nucleatum</em> ATCC 10953. J Bacteriol  172: 5714–5723.  <ul class="find" data-citedArticleID="970358"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+fructose+metabolism+and+polymer+synthesis+by+Fusobacterium+nucleatum+ATCC+10953.&amp;auth=&amp;atitle=Regulation+of+fructose+metabolism+and+polymer+synthesis+by+Fusobacterium+nucleatum+ATCC+10953." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+fructose+metabolism+and+polymer+synthesis+by+Fusobacterium+nucleatum+ATCC+10953." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+fructose+metabolism+and+polymer+synthesis+by+Fusobacterium+nucleatum+ATCC+10953.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000659-Rogers1" id="pone.0000659-Rogers1"></a>Rogers AH, Gully NJ, Pfennig AL, Zilm PS (1992) The breakdown and utilization of peptides by strains of <em>Fusobacterium nucleatum</em>. Oral Microbiol Immunol  7: 299–303.  <ul class="find" data-citedArticleID="970360" data-doi="10.1111/j.1399-302x.1992.tb00592.x"><li><a href="http://dx.doi.org/10.1111/j.1399-302x.1992.tb00592.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+breakdown+and+utilization+of+peptides+by+strains+of+Fusobacterium+nucleatum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+breakdown+and+utilization+of+peptides+by+strains+of+Fusobacterium+nucleatum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000659-Rogers2" id="pone.0000659-Rogers2"></a>Rogers AH, Zilm PS, Gully NJ, Pfennig AL, Marsh PD (1991) Aspects of the growth and metabolism of <em>Fusobacterium nucleatum</em> ATCC 10953 in continuous culture. Oral Microbiol Immunol  6: 250–255.  <ul class="find" data-citedArticleID="970362" data-doi="10.1111/j.1399-302x.1991.tb00486.x"><li><a href="http://dx.doi.org/10.1111/j.1399-302x.1991.tb00486.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aspects+of+the+growth+and+metabolism+of+Fusobacterium+nucleatum+ATCC+10953+in+continuous+culture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aspects+of+the+growth+and+metabolism+of+Fusobacterium+nucleatum+ATCC+10953+in+continuous+culture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000659-Tuttle1" id="pone.0000659-Tuttle1"></a>Tuttle RS, Strubel NA, Mourad J, Mangan DF (1992) A non-lectin-like mechanism by which <em>Fusobacterium nucleatum</em> 10953 adheres to and activates human lymphocytes. Oral Microbiol Immunol  7: 78–83.  <ul class="find" data-citedArticleID="970382" data-doi="10.1111/j.1399-302x.1992.tb00513.x"><li><a href="http://dx.doi.org/10.1111/j.1399-302x.1992.tb00513.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+non-lectin-like+mechanism+by+which+Fusobacterium+nucleatum+10953+adheres+to+and+activates+human+lymphocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+non-lectin-like+mechanism+by+which+Fusobacterium+nucleatum+10953+adheres+to+and+activates+human+lymphocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000659-Ozaki1" id="pone.0000659-Ozaki1"></a>Ozaki M, Miyake Y, Shirakawa M, Takemoto T, Okamoto H, et al.  (1990) Binding specificity of <em>Fusobacterium nucleatum</em> to human erythrocytes, polymorphonuclear leukocytes, fibroblasts, and HeLa cells. J Periodontal Res  25: 129–134.  <ul class="find" data-citedArticleID="970346" data-doi="10.1111/j.1600-0765.1990.tb01034.x"><li><a href="http://dx.doi.org/10.1111/j.1600-0765.1990.tb01034.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Binding+specificity+of+Fusobacterium+nucleatum+to+human+erythrocytes%2C+polymorphonuclear+leukocytes%2C+fibroblasts%2C+and+HeLa+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Binding+specificity+of+Fusobacterium+nucleatum+to+human+erythrocytes%2C+polymorphonuclear+leukocytes%2C+fibroblasts%2C+and+HeLa+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000659-Han1" id="pone.0000659-Han1"></a>Han YW, Shi W, Huang GT, Kinder Haake S, Park NH, et al.  (2000) Interactions between periodontal bacteria and human oral epithelial cells: <em>Fusobacterium nucleatum</em> adheres to and invades epithelial cells. Infect Immun  68: 3140–3146.  <ul class="find" data-citedArticleID="970256" data-doi="10.1128/iai.68.6.3140-3146.2000"><li><a href="http://dx.doi.org/10.1128/iai.68.6.3140-3146.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interactions+between+periodontal+bacteria+and+human+oral+epithelial+cells%3A+Fusobacterium+nucleatum+adheres+to+and+invades+epithelial+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interactions+between+periodontal+bacteria+and+human+oral+epithelial+cells%3A+Fusobacterium+nucleatum+adheres+to+and+invades+epithelial+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000659-RibeiroSobrinho1" id="pone.0000659-RibeiroSobrinho1"></a>Ribeiro-Sobrinho AP, Rabelo FL, Figueiredo CB, Alvarez-Leite JI, Nicoli JR, et al.  (2005) Bacteria recovered from dental pulp induce apoptosis of lymph node cells. J Med Microbiol  54: 413–416.  <ul class="find" data-citedArticleID="970354" data-doi="10.1099/jmm.0.45728-0"><li><a href="http://dx.doi.org/10.1099/jmm.0.45728-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteria+recovered+from+dental+pulp+induce+apoptosis+of+lymph+node+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteria+recovered+from+dental+pulp+induce+apoptosis+of+lymph+node+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000659-Jewett1" id="pone.0000659-Jewett1"></a>Jewett A, Hume WR, Le H, Huynh TN, Han YW, et al.  (2000) Induction of apoptotic cell death in peripheral blood mononuclear and polymorphonuclear cells by an oral bacterium, <em>Fusobacterium nucleatum</em>. Infect Immun  68: 1893–1898.  <ul class="find" data-citedArticleID="970278" data-doi="10.1128/iai.68.4.1893-1898.2000"><li><a href="http://dx.doi.org/10.1128/iai.68.4.1893-1898.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+apoptotic+cell+death+in+peripheral+blood+mononuclear+and+polymorphonuclear+cells+by+an+oral+bacterium%2C+Fusobacterium+nucleatum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+apoptotic+cell+death+in+peripheral+blood+mononuclear+and+polymorphonuclear+cells+by+an+oral+bacterium%2C+Fusobacterium+nucleatum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000659-Takada1" id="pone.0000659-Takada1"></a>Takada H, Ogawa T, Yoshimura F, Otsuka K, Kokeguchi S, et al.  (1988) Immunobiological activities of a porin fraction isolated from <em>Fusobacterium nucleatum</em> ATCC 10953. Infect Immun  56: 855–863.  <ul class="find" data-citedArticleID="970374"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Immunobiological+activities+of+a+porin+fraction+isolated+from+Fusobacterium+nucleatum+ATCC+10953.&amp;auth=&amp;atitle=Immunobiological+activities+of+a+porin+fraction+isolated+from+Fusobacterium+nucleatum+ATCC+10953." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunobiological+activities+of+a+porin+fraction+isolated+from+Fusobacterium+nucleatum+ATCC+10953." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunobiological+activities+of+a+porin+fraction+isolated+from+Fusobacterium+nucleatum+ATCC+10953.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000659-Bradshaw1" id="pone.0000659-Bradshaw1"></a>Bradshaw DJ, Marsh PD, Watson GK, Allison C (1998) Role of <em>Fusobacterium nucleatum</em> and coaggregation in anaerobe survival in planktonic and biofilm oral microbial communities during aeration. Infect Immun  66: 4729–4732.  <ul class="find" data-citedArticleID="970198"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Role+of+Fusobacterium+nucleatum+and+coaggregation+in+anaerobe+survival+in+planktonic+and+biofilm+oral+microbial+communities+during+aeration.&amp;auth=&amp;atitle=Role+of+Fusobacterium+nucleatum+and+coaggregation+in+anaerobe+survival+in+planktonic+and+biofilm+oral+microbial+communities+during+aeration." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+Fusobacterium+nucleatum+and+coaggregation+in+anaerobe+survival+in+planktonic+and+biofilm+oral+microbial+communities+during+aeration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+Fusobacterium+nucleatum+and+coaggregation+in+anaerobe+survival+in+planktonic+and+biofilm+oral+microbial+communities+during+aeration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000659-Kuriyama1" id="pone.0000659-Kuriyama1"></a>Kuriyama T, Nakagawa K, Kawashiri S, Yamamoto E, Nakamura S, et al.  (2000) The virulence of mixed infection with <em>Streptococcus constellatus</em> and <em>Fusobacterium nucleatum</em> in a murine orofacial infection model. Microbes Infect  2: 1425–1430.  <ul class="find" data-citedArticleID="970302" data-doi="10.1016/s1286-4579(00)01296-x"><li><a href="http://dx.doi.org/10.1016/s1286-4579(00)01296-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+virulence+of+mixed+infection+with+Streptococcus+constellatus+and+Fusobacterium+nucleatum+in+a+murine+orofacial+infection+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+virulence+of+mixed+infection+with+Streptococcus+constellatus+and+Fusobacterium+nucleatum+in+a+murine+orofacial+infection+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000659-Takemoto1" id="pone.0000659-Takemoto1"></a>Takemoto T, Kurihara H, Dahlen G (1997) Characterization of <em>Bacteroides forsythus</em> isolates. J Clin Microbiol  35: 1378–1381.  <ul class="find" data-citedArticleID="970376"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+Bacteroides+forsythus+isolates.&amp;auth=&amp;atitle=Characterization+of+Bacteroides+forsythus+isolates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+Bacteroides+forsythus+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+Bacteroides+forsythus+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000659-KinderHaake1" id="pone.0000659-KinderHaake1"></a>Kinder Haake S, Yoder SC, Attarian G, Podkaminer K (2000) Native plasmids of <em>Fusobacterium nucleatum</em>: Characterization and use in development of genetic systems. J Bacteriol  182: 1176–1180.  <ul class="find" data-citedArticleID="970292" data-doi="10.1128/jb.182.4.1176-1180.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.4.1176-1180.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Native+plasmids+of+Fusobacterium+nucleatum%3A+Characterization+and+use+in+development+of+genetic+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Native+plasmids+of+Fusobacterium+nucleatum%3A+Characterization+and+use+in+development+of+genetic+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000659-Haake1" id="pone.0000659-Haake1"></a>Haake SK, Yoder SC, Attarian G, Podkaminer K (2000) Native plasmids of <em>Fusobacterium nucleatum</em>: characterization and use in development of genetic systems. J Bacteriol  182: 1176–1180.  <ul class="find" data-citedArticleID="970254" data-doi="10.1128/jb.182.4.1176-1180.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.4.1176-1180.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Native+plasmids+of+Fusobacterium+nucleatum%3A+characterization+and+use+in+development+of+genetic+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Native+plasmids+of+Fusobacterium+nucleatum%3A+characterization+and+use+in+development+of+genetic+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000659-Lindahl1" id="pone.0000659-Lindahl1"></a>Lindahl L, Zengel JM (1986) Ribosomal genes in <em>Escherichia coli</em>. Ann Rev Genet  20: 297–326.  <ul class="find" data-citedArticleID="970318" data-doi="10.1146/annurev.ge.20.120186.001501"><li><a href="http://dx.doi.org/10.1146/annurev.ge.20.120186.001501" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+genes+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+genes+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000659-Siefert1" id="pone.0000659-Siefert1"></a>Siefert JL, Martin KA, Abdi F, Widger WR, Fox GE (1997) Conserved gene clusters in bacterial genomes provide further support for the primacy of RNA. J Mol Evol  45: 467–472.  <ul class="find" data-citedArticleID="970368" data-doi="10.1007/pl00006251"><li><a href="http://dx.doi.org/10.1007/pl00006251" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+gene+clusters+in+bacterial+genomes+provide+further+support+for+the+primacy+of+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+gene+clusters+in+bacterial+genomes+provide+further+support+for+the+primacy+of+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000659-Demuth1" id="pone.0000659-Demuth1"></a>Demuth DR, Savary R, Golub E, Shenker BJ (1996) Identification and analysis of <em>fipA</em>, a <em>Fusobacterium nucleatum</em> immunosuppressive factor gene. Infect Immun  64: 1335–1341.  <ul class="find" data-citedArticleID="970228"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+and+analysis+of+fipA%2C+a+Fusobacterium+nucleatum+immunosuppressive+factor+gene.&amp;auth=&amp;atitle=Identification+and+analysis+of+fipA%2C+a+Fusobacterium+nucleatum+immunosuppressive+factor+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+analysis+of+fipA%2C+a+Fusobacterium+nucleatum+immunosuppressive+factor+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+analysis+of+fipA%2C+a+Fusobacterium+nucleatum+immunosuppressive+factor+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000659-HuntGerardo1" id="pone.0000659-HuntGerardo1"></a>Hunt Gerardo S, Yoder SC, Citron DM, Goldstein EJC, Kinder Haake S (2002) Sequence conservation and distribution of the fusobacterial immunosuppressive protein gene, <em>fipA</em>. Oral Microbiology and Immunology  17: 315–320.  <ul class="find" data-citedArticleID="970270" data-doi="10.1034/j.1399-302x.2002.170509.x"><li><a href="http://dx.doi.org/10.1034/j.1399-302x.2002.170509.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+conservation+and+distribution+of+the+fusobacterial+immunosuppressive+protein+gene%2C+fipA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+conservation+and+distribution+of+the+fusobacterial+immunosuppressive+protein+gene%2C+fipA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000659-Lawrence1" id="pone.0000659-Lawrence1"></a>Lawrence JG, Ochman H (1997) Amelioration of bacterial genomes: rates of change and exchange. J Mol Evol  44: 383–397.  <ul class="find" data-citedArticleID="970304" data-doi="10.1007/pl00006158"><li><a href="http://dx.doi.org/10.1007/pl00006158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amelioration+of+bacterial+genomes%3A+rates+of+change+and+exchange." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amelioration+of+bacterial+genomes%3A+rates+of+change+and+exchange.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000659-Lawrence2" id="pone.0000659-Lawrence2"></a>Lawrence JG, Ochman H (1998) Molecular archaeology of the <em>Escherichia coli</em> genome. Proc Natl Acad Sci U S A  95: 9413–9417.  <ul class="find" data-citedArticleID="970306" data-doi="10.1073/pnas.95.16.9413"><li><a href="http://dx.doi.org/10.1073/pnas.95.16.9413" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+archaeology+of+the+Escherichia+coli+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+archaeology+of+the+Escherichia+coli+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000659-Karlin1" id="pone.0000659-Karlin1"></a>Karlin S (1998) Global dinucleotide signatures and analysis of genomic heterogeneity. Curr Opin Microbiol  1: 598–610.  <ul class="find" data-citedArticleID="970286" data-doi="10.1016/s1369-5274(98)80095-7"><li><a href="http://dx.doi.org/10.1016/s1369-5274(98)80095-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+dinucleotide+signatures+and+analysis+of+genomic+heterogeneity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+dinucleotide+signatures+and+analysis+of+genomic+heterogeneity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000659-Mrazek1" id="pone.0000659-Mrazek1"></a>Mrazek J, Karlin S (1999) Detecting alien genes in bacterial genomes. Ann NY Acad Sci  870: 314–329.  <ul class="find" data-citedArticleID="970342" data-doi="10.1111/j.1749-6632.1999.tb08893.x"><li><a href="http://dx.doi.org/10.1111/j.1749-6632.1999.tb08893.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+alien+genes+in+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+alien+genes+in+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000659-Medigue1" id="pone.0000659-Medigue1"></a>Medigue C, Rouxel T, Vigier P, Henaut A, Danchin A (1991) Evidence for horizontal gene transfer in <em>Escherichia coli</em> speciation. J Mol Biol  222: 851–856.  <ul class="find" data-citedArticleID="970328" data-doi="10.1016/0022-2836(91)90575-q"><li><a href="http://dx.doi.org/10.1016/0022-2836(91)90575-q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+horizontal+gene+transfer+in+Escherichia+coli+speciation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+horizontal+gene+transfer+in+Escherichia+coli+speciation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000659-Moszer1" id="pone.0000659-Moszer1"></a>Moszer I, Rocha EP, Danchin A (1999) Codon usage and lateral gene transfer in <em>Bacillus subtilis</em>. Curr Opin Microbiol  2: 524–528.  <ul class="find" data-citedArticleID="970340" data-doi="10.1016/s1369-5274(99)00011-9"><li><a href="http://dx.doi.org/10.1016/s1369-5274(99)00011-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+usage+and+lateral+gene+transfer+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+usage+and+lateral+gene+transfer+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000659-Hayes1" id="pone.0000659-Hayes1"></a>Hayes WS, Borodovsky M (1998) How to interpret an anonymous bacterial genome: Machine learning approach to gene identification. Genome Res  8: 1154–1171.  <ul class="find" data-citedArticleID="970264"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=How+to+interpret+an+anonymous+bacterial+genome%3A+Machine+learning+approach+to+gene+identification.&amp;auth=&amp;atitle=How+to+interpret+an+anonymous+bacterial+genome%3A+Machine+learning+approach+to+gene+identification." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+to+interpret+an+anonymous+bacterial+genome%3A+Machine+learning+approach+to+gene+identification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+to+interpret+an+anonymous+bacterial+genome%3A+Machine+learning+approach+to+gene+identification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000659-Ragan1" id="pone.0000659-Ragan1"></a>Ragan MA (2001) On surrogate methods for detecting lateral gene transfer. FEMS Microbiol Lett  201: 187–191.  <ul class="find" data-citedArticleID="970352" data-doi="10.1111/j.1574-6968.2001.tb10755.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.2001.tb10755.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+surrogate+methods+for+detecting+lateral+gene+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+surrogate+methods+for+detecting+lateral+gene+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000659-Lawrence3" id="pone.0000659-Lawrence3"></a>Lawrence JG, Hartl DL (1992) Inference of horizontal genetic transfer from molecular data: An approach using the bootstrap. Genetics  131: 753–760.  <ul class="find" data-citedArticleID="970308"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inference+of+horizontal+genetic+transfer+from+molecular+data%3A+An+approach+using+the+bootstrap.&amp;auth=&amp;atitle=Inference+of+horizontal+genetic+transfer+from+molecular+data%3A+An+approach+using+the+bootstrap." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+of+horizontal+genetic+transfer+from+molecular+data%3A+An+approach+using+the+bootstrap." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+of+horizontal+genetic+transfer+from+molecular+data%3A+An+approach+using+the+bootstrap.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000659-Philippe1" id="pone.0000659-Philippe1"></a>Philippe H, Douady CJ (2003) Horizontal gene transfer and phylogenetics. Curr Opin Microbiol  6: 498–505.  <ul class="find" data-citedArticleID="970350" data-doi="10.1016/j.mib.2003.09.008"><li><a href="http://dx.doi.org/10.1016/j.mib.2003.09.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Horizontal+gene+transfer+and+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Horizontal+gene+transfer+and+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000659-Bobik1" id="pone.0000659-Bobik1"></a>Bobik T, Xu Y, Jeter R, Otto K, Roth J (1997) Propanediol utilization genes (<em>pdu</em>) of <em>Salmonella typhimurium</em>: three genes for the propanediol dehydratase. J Bacteriol  179: 6633–6639.  <ul class="find" data-citedArticleID="970192"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Propanediol+utilization+genes+%28pdu%29+of+Salmonella+typhimurium%3A+three+genes+for+the+propanediol+dehydratase.&amp;auth=&amp;atitle=Propanediol+utilization+genes+%28pdu%29+of+Salmonella+typhimurium%3A+three+genes+for+the+propanediol+dehydratase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Propanediol+utilization+genes+%28pdu%29+of+Salmonella+typhimurium%3A+three+genes+for+the+propanediol+dehydratase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Propanediol+utilization+genes+%28pdu%29+of+Salmonella+typhimurium%3A+three+genes+for+the+propanediol+dehydratase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000659-Bobik2" id="pone.0000659-Bobik2"></a>Bobik TA, Havemann GD, Busch RJ, Williams DS, Aldrich HC (1999) The propanediol utilization (<em>pdu</em>) operon of <em>Salmonella enterica</em> serovar Typhimurium LT2 includes genes necessary for formation of polyhedral organelles involved in coenzyme B12-dependent 1,2-propanediol degradation. J Bacteriol  181: 5967–5975.  <ul class="find" data-citedArticleID="970194" data-doi="10.1128/jb.185.17.5086-5095.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.17.5086-5095.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+propanediol+utilization+%28pdu%29+operon+of+Salmonella+enterica+serovar+Typhimurium+LT2+includes+genes+necessary+for+formation+of+polyhedral+organelles+involved+in+coenzyme+B12-dependent+1%2C2-propanediol+degradation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+propanediol+utilization+%28pdu%29+operon+of+Salmonella+enterica+serovar+Typhimurium+LT2+includes+genes+necessary+for+formation+of+polyhedral+organelles+involved+in+coenzyme+B12-dependent+1%2C2-propanediol+degradation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000659-Leach1" id="pone.0000659-Leach1"></a>Leach SA, Critchley P (1966) Bacterial degradation of glycoprotein sugars in human saliva. Nature  209: 506.  <ul class="find" data-citedArticleID="970310" data-doi="10.1038/209506a0"><li><a href="http://dx.doi.org/10.1038/209506a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+degradation+of+glycoprotein+sugars+in+human+saliva." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+degradation+of+glycoprotein+sugars+in+human+saliva.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000659-Lin1" id="pone.0000659-Lin1"></a>Lin ECC (1996)  Dissimilatory pathways for sugars, polyols, and carboxylates. In: Neidhardt FC, editor. <em>Escherichia coli</em> and <em>Salmonella</em>; Second edition. Washington, D.C.: ASM Press. pp. 307–342.  <ul class="find-nolinks"></ul></li><li><span class="label">59.
              </span><a name="pone.0000659-Bruggemann1" id="pone.0000659-Bruggemann1"></a>Bruggemann H, Baumer S, Fricke WF, Wiezer A, Liesegang H, et al.  (2003) The genome sequence of <em>Clostridium tetani</em>, the causative agent of tetanus disease. Proc Natl Acad Sci U S A  100: 1316–1321.  <ul class="find" data-citedArticleID="970208" data-doi="10.1073/pnas.0335853100"><li><a href="http://dx.doi.org/10.1073/pnas.0335853100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+Clostridium+tetani%2C+the+causative+agent+of+tetanus+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+Clostridium+tetani%2C+the+causative+agent+of+tetanus+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000659-Zimmer1" id="pone.0000659-Zimmer1"></a>Zimmer M, Scherer S, Loessner MJ (2002) Genomic analysis of <em>Clostridium perfringens</em> bacteriophage phi3626, which integrates into <em>guaA</em> and possibly affects sporulation. J Bacteriol  184: 4359–4368.  <ul class="find" data-citedArticleID="970390" data-doi="10.1128/jb.184.16.4359-4368.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.16.4359-4368.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+analysis+of+Clostridium+perfringens+bacteriophage+phi3626%2C+which+integrates+into+guaA+and+possibly+affects+sporulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+analysis+of+Clostridium+perfringens+bacteriophage+phi3626%2C+which+integrates+into+guaA+and+possibly+affects+sporulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000659-Christie1" id="pone.0000659-Christie1"></a>Christie PJ (2004) Type IV secretion: the <em>Agrobacterium</em> VirB/D4 and related conjugation systems. Biochim Biophys Acta  1694: 219–234.  <ul class="find" data-citedArticleID="970220" data-doi="10.1016/j.bbamcr.2004.02.013"><li><a href="http://dx.doi.org/10.1016/j.bbamcr.2004.02.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Type+IV+secretion%3A+the+Agrobacterium+VirB%2FD4+and+related+conjugation+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Type+IV+secretion%3A+the+Agrobacterium+VirB%2FD4+and+related+conjugation+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000659-Desvaux1" id="pone.0000659-Desvaux1"></a>Desvaux M, Khan A, Beatson SA, Scott-Tucker A, Henderson IR (2005) Protein secretion systems in <em>Fusobacterium nucleatum</em>: Genomic identification of Type 4 piliation and complete Type V pathways brings new insight into mechanisms of pathogenesis. Biochim Biophys Acta  1713: 92–1125.  <ul class="find" data-citedArticleID="970230" data-doi="10.1016/j.bbamem.2005.05.002"><li><a href="http://dx.doi.org/10.1016/j.bbamem.2005.05.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+secretion+systems+in+Fusobacterium+nucleatum%3A+Genomic+identification+of+Type+4+piliation+and+complete+Type+V+pathways+brings+new+insight+into+mechanisms+of+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+secretion+systems+in+Fusobacterium+nucleatum%3A+Genomic+identification+of+Type+4+piliation+and+complete+Type+V+pathways+brings+new+insight+into+mechanisms+of+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000659-Braun1" id="pone.0000659-Braun1"></a>Braun V, Mehlig M, Moos M, Rupnik M, Kalt B, et al.  (2000) A chimeric ribozyme in <em>Clostridium difficile</em> combines features of group I introns and insertion elements. Mol Microbiol  36: 1447–1459.  <ul class="find" data-citedArticleID="970200" data-doi="10.1046/j.1365-2958.2000.01965.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01965.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+chimeric+ribozyme+in+Clostridium+difficile+combines+features+of+group+I+introns+and+insertion+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+chimeric+ribozyme+in+Clostridium+difficile+combines+features+of+group+I+introns+and+insertion+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000659-Cech1" id="pone.0000659-Cech1"></a>Cech TR, Damberger SH, Gutell RR (1994) Representation of the secondary and tertiary structure of group I introns. Nature Struct Biol  1: 273–280.  <ul class="find" data-citedArticleID="970214" data-doi="10.1038/nsb0594-273"><li><a href="http://dx.doi.org/10.1038/nsb0594-273" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Representation+of+the+secondary+and+tertiary+structure+of+group+I+introns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Representation+of+the+secondary+and+tertiary+structure+of+group+I+introns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000659-Kersulyte1" id="pone.0000659-Kersulyte1"></a>Kersulyte D, Akopyants NS, Clifton SW, Roe BA, Berg DE (1998) Novel sequence organization and insertion specificity of IS<em>605</em> and IS<em>606</em>: chimaeric transposable elements of <em>Helicobacter pylori</em>. Gene  223: 175–186.  <ul class="find" data-citedArticleID="970290" data-doi="10.1016/s0378-1119(98)00164-4"><li><a href="http://dx.doi.org/10.1016/s0378-1119(98)00164-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+sequence+organization+and+insertion+specificity+of+IS605+and+IS606%3A+chimaeric+transposable+elements+of+Helicobacter+pylori." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+sequence+organization+and+insertion+specificity+of+IS605+and+IS606%3A+chimaeric+transposable+elements+of+Helicobacter+pylori.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000659-Ilyina1" id="pone.0000659-Ilyina1"></a>Ilyina TV, Koonin EV (1992) Conserved sequence motifs in the initiator proteins for rolling circle DNA replication encoded by diverse replicons from eubacteria, eukaryotes and archaebacteria. Nucl Acids Res  20: 3279–3285.  <ul class="find" data-citedArticleID="970272" data-doi="10.1093/nar/20.13.3279"><li><a href="http://dx.doi.org/10.1093/nar/20.13.3279" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+sequence+motifs+in+the+initiator+proteins+for+rolling+circle+DNA+replication+encoded+by+diverse+replicons+from+eubacteria%2C+eukaryotes+and+archaebacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+sequence+motifs+in+the+initiator+proteins+for+rolling+circle+DNA+replication+encoded+by+diverse+replicons+from+eubacteria%2C+eukaryotes+and+archaebacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000659-McKay1" id="pone.0000659-McKay1"></a>McKay TL, Ko J, Bilalis Y, DiRienzo JM (1995) Mobile genetic elements of <em>Fusobacterium nucleatum</em>. Plasmid  33: 15–20.  <ul class="find" data-citedArticleID="970324" data-doi="10.1006/plas.1995.1003"><li><a href="http://dx.doi.org/10.1006/plas.1995.1003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mobile+genetic+elements+of+Fusobacterium+nucleatum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mobile+genetic+elements+of+Fusobacterium+nucleatum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000659-Suzuki1" id="pone.0000659-Suzuki1"></a>Suzuki T, Fukuda I, Tobe T, Yoshikawa M, Sasakawa C (1994) Identification and characterization of a chromosomal virulence gene, <em>vacJ</em>, required for intracellular spreading of <em>Shigella flexneri</em>. Mol Microbiol  11: 31–41.  <ul class="find" data-citedArticleID="970372" data-doi="10.1111/j.1365-2958.1994.tb00287.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1994.tb00287.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+a+chromosomal+virulence+gene%2C+vacJ%2C+required+for+intracellular+spreading+of+Shigella+flexneri." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+a+chromosomal+virulence+gene%2C+vacJ%2C+required+for+intracellular+spreading+of+Shigella+flexneri.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000659-Han2" id="pone.0000659-Han2"></a>Han YW, Redline RW, Li M, Yin L, Hill GB, et al.  (2004) <em>Fusobacterium nucleatum</em> induces premature and term stillbirths in pregnant mice: Implication of oral bacteria in preterm birth. Infect Immun  72: 2272–2279.  <ul class="find" data-citedArticleID="970258" data-doi="10.1128/iai.72.4.2272-2279.2004"><li><a href="http://dx.doi.org/10.1128/iai.72.4.2272-2279.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fusobacterium+nucleatum+induces+premature+and+term+stillbirths+in+pregnant+mice%3A+Implication+of+oral+bacteria+in+preterm+birth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fusobacterium+nucleatum+induces+premature+and+term+stillbirths+in+pregnant+mice%3A+Implication+of+oral+bacteria+in+preterm+birth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000659-Kaufman1" id="pone.0000659-Kaufman1"></a>Kaufman J, DiRienzo JM (1989) Isolation of a corncob (coaggregation) receptor polypeptide from <em>Fusobacterium nucleatum</em>. Infect Immun  57: 331–337.  <ul class="find" data-citedArticleID="970288"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Isolation+of+a+corncob+%28coaggregation%29+receptor+polypeptide+from+Fusobacterium+nucleatum.&amp;auth=&amp;atitle=Isolation+of+a+corncob+%28coaggregation%29+receptor+polypeptide+from+Fusobacterium+nucleatum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+a+corncob+%28coaggregation%29+receptor+polypeptide+from+Fusobacterium+nucleatum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+a+corncob+%28coaggregation%29+receptor+polypeptide+from+Fusobacterium+nucleatum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000659-KinderHaake2" id="pone.0000659-KinderHaake2"></a>Kinder Haake S, Wang X (1997) Cloning and expression of FomA, the major outer-membrane protein gene from <em>Fusobacterium nucleatum</em> T18. Arch Oral Biol  42: 19–24.  <ul class="find" data-citedArticleID="970294" data-doi="10.1016/s0003-9969(96)00105-7"><li><a href="http://dx.doi.org/10.1016/s0003-9969(96)00105-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+expression+of+FomA%2C+the+major+outer-membrane+protein+gene+from+Fusobacterium+nucleatum+T18." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+expression+of+FomA%2C+the+major+outer-membrane+protein+gene+from+Fusobacterium+nucleatum+T18.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000659-Carsiotis1" id="pone.0000659-Carsiotis1"></a>Carsiotis M, Stocker B, Weinstein D, O'Brien A (1989) A <em>Salmonella typhimurium</em> virulence gene linked to <em>flg</em>. Infect Immun  57: 3276–3280.  <ul class="find" data-citedArticleID="970212"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+Salmonella+typhimurium+virulence+gene+linked+to+flg.&amp;auth=&amp;atitle=A+Salmonella+typhimurium+virulence+gene+linked+to+flg." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Salmonella+typhimurium+virulence+gene+linked+to+flg." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Salmonella+typhimurium+virulence+gene+linked+to+flg.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000659-Sukupolvi1" id="pone.0000659-Sukupolvi1"></a>Sukupolvi S, O'Connor CD (1990) TraT lipoprotein, a plasmid-specified mediator of interactions between Gram-negative bacteria and their environment. Microbiol Rev  54: 331–341.  <ul class="find" data-citedArticleID="970370"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=TraT+lipoprotein%2C+a+plasmid-specified+mediator+of+interactions+between+Gram-negative+bacteria+and+their+environment.&amp;auth=&amp;atitle=TraT+lipoprotein%2C+a+plasmid-specified+mediator+of+interactions+between+Gram-negative+bacteria+and+their+environment." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TraT+lipoprotein%2C+a+plasmid-specified+mediator+of+interactions+between+Gram-negative+bacteria+and+their+environment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TraT+lipoprotein%2C+a+plasmid-specified+mediator+of+interactions+between+Gram-negative+bacteria+and+their+environment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000659-Tobe1" id="pone.0000659-Tobe1"></a>Tobe T, Sasakawa C, Okada N, Honma Y, Yoshikawa M (1992) <em>vacB</em>, a novel chromosomal gene required for expression of virulence genes on the large plasmid of <em>Shigella flexneri</em>. J Bacteriol  174: 6359–6367.  <ul class="find" data-citedArticleID="970380"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=vacB%2C+a+novel+chromosomal+gene+required+for+expression+of+virulence+genes+on+the+large+plasmid+of+Shigella+flexneri.&amp;auth=&amp;atitle=vacB%2C+a+novel+chromosomal+gene+required+for+expression+of+virulence+genes+on+the+large+plasmid+of+Shigella+flexneri." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=vacB%2C+a+novel+chromosomal+gene+required+for+expression+of+virulence+genes+on+the+large+plasmid+of+Shigella+flexneri." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22vacB%2C+a+novel+chromosomal+gene+required+for+expression+of+virulence+genes+on+the+large+plasmid+of+Shigella+flexneri.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000659-Cheng1" id="pone.0000659-Cheng1"></a>Cheng Z-F, Zuo Y, Li Z, Rudd KE, Deutscher MP (1998) The <em>vacB</em> gene required for virulence in <em>Shigella flexneri</em> and <em>Escherichia coli</em> encodes the exoribonuclease RNase R. J Biol Chem  273: 14077–14080.  <ul class="find" data-citedArticleID="970218" data-doi="10.1074/jbc.273.23.14077"><li><a href="http://dx.doi.org/10.1074/jbc.273.23.14077" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+vacB+gene+required+for+virulence+in+Shigella+flexneri+and+Escherichia+coli+encodes+the+exoribonuclease+RNase+R." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+vacB+gene+required+for+virulence+in+Shigella+flexneri+and+Escherichia+coli+encodes+the+exoribonuclease+RNase+R.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000659-Niederman1" id="pone.0000659-Niederman1"></a>Niederman R, Zhang J, Kashket S (1997) Short-chain carboxylic-acid-stimulated, PMN-mediated gingival inflammation. Critic Rev Oral Biol Med  8: 269–290.  <ul class="find" data-citedArticleID="970344" data-doi="10.1177/10454411970080030301"><li><a href="http://dx.doi.org/10.1177/10454411970080030301" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Short-chain+carboxylic-acid-stimulated%2C+PMN-mediated+gingival+inflammation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Short-chain+carboxylic-acid-stimulated%2C+PMN-mediated+gingival+inflammation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000659-Shenker1" id="pone.0000659-Shenker1"></a>Shenker BJ, Datar S (1995) <em>Fusobacterium nucleatum</em> inhibits human T-cell activation by arresting cells in the mid-G<sub>1</sub> phase of the cell cycle. Infect Immun  63: 4830–4836.  <ul class="find" data-citedArticleID="970366"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fusobacterium+nucleatum+inhibits+human+T-cell+activation+by+arresting+cells+in+the+mid-G1+phase+of+the+cell+cycle.&amp;auth=&amp;atitle=Fusobacterium+nucleatum+inhibits+human+T-cell+activation+by+arresting+cells+in+the+mid-G1+phase+of+the+cell+cycle." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fusobacterium+nucleatum+inhibits+human+T-cell+activation+by+arresting+cells+in+the+mid-G1+phase+of+the+cell+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fusobacterium+nucleatum+inhibits+human+T-cell+activation+by+arresting+cells+in+the+mid-G1+phase+of+the+cell+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0000659-Schaible1" id="pone.0000659-Schaible1"></a>Schaible UE, Kaufmann SHE (2004) Iron and microbial infection. Nat Rev Micro  2: 946–953.  <ul class="find" data-citedArticleID="970364" data-doi="10.1038/nrmicro1046"><li><a href="http://dx.doi.org/10.1038/nrmicro1046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Iron+and+microbial+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Iron+and+microbial+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000659-Hase1" id="pone.0000659-Hase1"></a>Hase S, Hofstad T, Rietschel ET (1977) Chemical structure of the lipid A component of lipopolysaccharides from <em>Fusobacterium nucleatum</em>. J Bacteriol  129: 9–14.  <ul class="find" data-citedArticleID="970260"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chemical+structure+of+the+lipid+A+component+of+lipopolysaccharides+from+Fusobacterium+nucleatum.&amp;auth=&amp;atitle=Chemical+structure+of+the+lipid+A+component+of+lipopolysaccharides+from+Fusobacterium+nucleatum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemical+structure+of+the+lipid+A+component+of+lipopolysaccharides+from+Fusobacterium+nucleatum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemical+structure+of+the+lipid+A+component+of+lipopolysaccharides+from+Fusobacterium+nucleatum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0000659-Bassler1" id="pone.0000659-Bassler1"></a>Bassler B, Greenberg E, Stevens A (1997) Cross-species induction of luminescence in the quorum-sensing bacterium <em>Vibrio harveyi</em>. J Bacteriol  179: 4043–4045.  <ul class="find" data-citedArticleID="970188"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cross-species+induction+of+luminescence+in+the+quorum-sensing+bacterium+Vibrio+harveyi.&amp;auth=&amp;atitle=Cross-species+induction+of+luminescence+in+the+quorum-sensing+bacterium+Vibrio+harveyi." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cross-species+induction+of+luminescence+in+the+quorum-sensing+bacterium+Vibrio+harveyi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cross-species+induction+of+luminescence+in+the+quorum-sensing+bacterium+Vibrio+harveyi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0000659-Yoshida1" id="pone.0000659-Yoshida1"></a>Yoshida A, Ansai T, Takehara T, Kuramitsu HK (2005) LuxS-based signaling affects <em>Streptococcus mutans</em> biofilm formation. Appl Environ Microbiol  71: 2372–2380.  <ul class="find" data-citedArticleID="970386" data-doi="10.1128/aem.71.5.2372-2380.2005"><li><a href="http://dx.doi.org/10.1128/aem.71.5.2372-2380.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LuxS-based+signaling+affects+Streptococcus+mutans+biofilm+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LuxS-based+signaling+affects+Streptococcus+mutans+biofilm+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0000659-Frias1" id="pone.0000659-Frias1"></a>Frias J, Olle E, Alsina M (2001) Periodontal pathogens produce quorum sensing signal molecules. Infect Immun  69: 3431–3434.  <ul class="find" data-citedArticleID="970246" data-doi="10.1128/iai.69.5.3431-3434.2001"><li><a href="http://dx.doi.org/10.1128/iai.69.5.3431-3434.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Periodontal+pathogens+produce+quorum+sensing+signal+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Periodontal+pathogens+produce+quorum+sensing+signal+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0000659-Diaz2" id="pone.0000659-Diaz2"></a>Diaz PI, Zilm PS, Rogers AH (2000) The response to oxidative stress of <em>Fusobacterium nucleatum</em> grown in continuous culture. FEMS Microbiol Lett  187: 31–34.  <ul class="find" data-citedArticleID="970234" data-doi="10.1111/j.1574-6968.2000.tb09132.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.2000.tb09132.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+response+to+oxidative+stress+of+Fusobacterium+nucleatum+grown+in+continuous+culture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+response+to+oxidative+stress+of+Fusobacterium+nucleatum+grown+in+continuous+culture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0000659-Ebersole1" id="pone.0000659-Ebersole1"></a>Ebersole JL, Feuille F, Kesavalu L, Holt SC (1997) Host modulation of tissue destruction caused by periodontopathogens: effects on a mixed microbial infection composed of <em>Porphyromonas gingivalis</em> and <em>Fusobacterium nucleatum</em>. Microb Pathog  23: 23–32.  <ul class="find" data-citedArticleID="970238" data-doi="10.1006/mpat.1996.0129"><li><a href="http://dx.doi.org/10.1006/mpat.1996.0129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Host+modulation+of+tissue+destruction+caused+by+periodontopathogens%3A+effects+on+a+mixed+microbial+infection+composed+of+Porphyromonas+gingivalis+and+Fusobacterium+nucleatum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Host+modulation+of+tissue+destruction+caused+by+periodontopathogens%3A+effects+on+a+mixed+microbial+infection+composed+of+Porphyromonas+gingivalis+and+Fusobacterium+nucleatum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0000659-Feuille1" id="pone.0000659-Feuille1"></a>Feuille F, Ebersole JL, Kesavalu L, Steffen MJ, Holt SC (1996) Mixed infection with <em>Porphyromonas gingivalis</em> and <em>Fusobacterium nucleatum</em> in a murine lesion model: Potential synergistic effects on virulence. Infect Immun  64: 2095–2100.  <ul class="find" data-citedArticleID="970244"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mixed+infection+with+Porphyromonas+gingivalis+and+Fusobacterium+nucleatum+in+a+murine+lesion+model%3A+Potential+synergistic+effects+on+virulence.&amp;auth=&amp;atitle=Mixed+infection+with+Porphyromonas+gingivalis+and+Fusobacterium+nucleatum+in+a+murine+lesion+model%3A+Potential+synergistic+effects+on+virulence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mixed+infection+with+Porphyromonas+gingivalis+and+Fusobacterium+nucleatum+in+a+murine+lesion+model%3A+Potential+synergistic+effects+on+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mixed+infection+with+Porphyromonas+gingivalis+and+Fusobacterium+nucleatum+in+a+murine+lesion+model%3A+Potential+synergistic+effects+on+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0000659-Lehmann1" id="pone.0000659-Lehmann1"></a>Lehmann Y, Meile L, Teuber M (1996) Rubrerythrin from <em>Clostridium perfringens</em>: cloning of the gene, purification of the protein, and characterization of its superoxide dismutase function. J Bacteriol  178: 7152–7158.  <ul class="find" data-citedArticleID="970312"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rubrerythrin+from+Clostridium+perfringens%3A+cloning+of+the+gene%2C+purification+of+the+protein%2C+and+characterization+of+its+superoxide+dismutase+function.&amp;auth=&amp;atitle=Rubrerythrin+from+Clostridium+perfringens%3A+cloning+of+the+gene%2C+purification+of+the+protein%2C+and+characterization+of+its+superoxide+dismutase+function." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rubrerythrin+from+Clostridium+perfringens%3A+cloning+of+the+gene%2C+purification+of+the+protein%2C+and+characterization+of+its+superoxide+dismutase+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rubrerythrin+from+Clostridium+perfringens%3A+cloning+of+the+gene%2C+purification+of+the+protein%2C+and+characterization+of+its+superoxide+dismutase+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0000659-Brenot1" id="pone.0000659-Brenot1"></a>Brenot A, King KY, Janowiak B, Griffith O, Caparon MG (2004) Contribution of glutathione peroxidase to the virulence of <em>Streptococcus pyogenes</em>. Infect Immun  72: 408–413.  <ul class="find" data-citedArticleID="970202" data-doi="10.1128/iai.72.1.408-413.2004"><li><a href="http://dx.doi.org/10.1128/iai.72.1.408-413.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contribution+of+glutathione+peroxidase+to+the+virulence+of+Streptococcus+pyogenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contribution+of+glutathione+peroxidase+to+the+virulence+of+Streptococcus+pyogenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0000659-King1" id="pone.0000659-King1"></a>King KY, Horenstein JA, Caparon MG (2000) Aerotolerance and peroxide resistance in peroxidase and PerR mutants of <em>Streptococcus pyogenes</em>. J Bacteriol  182: 5290–5299.  <ul class="find" data-citedArticleID="970296" data-doi="10.1128/jb.182.19.5290-5299.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.19.5290-5299.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aerotolerance+and+peroxide+resistance+in+peroxidase+and+PerR+mutants+of+Streptococcus+pyogenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aerotolerance+and+peroxide+resistance+in+peroxidase+and+PerR+mutants+of+Streptococcus+pyogenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0000659-Jaeger1" id="pone.0000659-Jaeger1"></a>Jaeger T, Budde H, Flohe L, Menge U, Singh M, et al.  (2004) Multiple thioredoxin-mediated routes to detoxify hydroperoxides in <em>Mycobacterium tuberculosis</em>. Arch Biochem Biophys  423: 182–191.  <ul class="find" data-citedArticleID="970274" data-doi="10.1016/j.abb.2003.11.021"><li><a href="http://dx.doi.org/10.1016/j.abb.2003.11.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+thioredoxin-mediated+routes+to+detoxify+hydroperoxides+in+Mycobacterium+tuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+thioredoxin-mediated+routes+to+detoxify+hydroperoxides+in+Mycobacterium+tuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0000659-Li1" id="pone.0000659-Li1"></a>Li K, Hein S, Zou W, Klug G (2004) The glutathione-glutaredoxin system in <em>Rhodobacter capsulatus</em>: part of a complex regulatory network controlling defense against oxidative stress. J Bacteriol  186: 6800–6808.  <ul class="find" data-citedArticleID="970314" data-doi="10.1128/jb.186.20.6800-6808.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.20.6800-6808.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+glutathione-glutaredoxin+system+in+Rhodobacter+capsulatus%3A+part+of+a+complex+regulatory+network+controlling+defense+against+oxidative+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+glutathione-glutaredoxin+system+in+Rhodobacter+capsulatus%3A+part+of+a+complex+regulatory+network+controlling+defense+against+oxidative+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0000659-Jean1" id="pone.0000659-Jean1"></a>Jean D, Briolat V, Reysset G (2004) Oxidative stress response in <em>Clostridium perfringens</em>. Microbiol  150: 1649–1659.  <ul class="find" data-citedArticleID="970276" data-doi="10.1099/mic.0.27017-0"><li><a href="http://dx.doi.org/10.1099/mic.0.27017-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oxidative+stress+response+in+Clostridium+perfringens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oxidative+stress+response+in+Clostridium+perfringens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0000659-Mira1" id="pone.0000659-Mira1"></a>Mira A, Pushker R, Legault BA, Moreira D, Rodriguez-Valera  (2004) Evolutionary relationships of <em>Fusobacterium nucleatum</em> based on phylogenetic analysis and comparative genomics. BMC Evol Biol  4: 50–58.  <ul class="find" data-citedArticleID="970334" data-doi="10.1186/1471-2148-4-50"><li><a href="http://dx.doi.org/10.1186/1471-2148-4-50" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+relationships+of+Fusobacterium+nucleatum+based+on+phylogenetic+analysis+and+comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+relationships+of+Fusobacterium+nucleatum+based+on+phylogenetic+analysis+and+comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0000659-Dzink1" id="pone.0000659-Dzink1"></a>Dzink JL, Sheenan MT, Socransky SS (1990) Proposal of three subspecies of <em>Fusobacterium nucleatum</em> Knorr 1922: <em>Fusobacterium nucleatum</em> subsp. <em>nucleatum</em> subsp. nov., comb. nov.; <em>Fusobacterium nucleatum</em> subsp. <em>polymorphum</em> subsp. nov., nom. rev., comb. nov.; and <em>Fusobacterium nucleatum</em> subsp. <em>vincentii</em> subsp. nov., nom. rev., comb. nov. Int J Syst Bacteriol  40: 74–78.  <ul class="find" data-citedArticleID="970236" data-doi="10.1099/00207713-40-1-74"><li><a href="http://dx.doi.org/10.1099/00207713-40-1-74" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proposal+of+three+subspecies+of+Fusobacterium+nucleatum+Knorr+1922%3A+Fusobacterium+nucleatum+subsp.+nucleatum+subsp.+nov.%2C+comb.+nov.%3B+Fusobacterium+nucleatum+subsp.+polymorphum+subsp.+nov.%2C+nom.+rev.%2C+comb.+nov.%3B+and+Fusobacterium+nucleatum+subsp.+vincentii+subsp.+nov.%2C+nom.+rev.%2C+comb.+nov." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proposal+of+three+subspecies+of+Fusobacterium+nucleatum+Knorr+1922%3A+Fusobacterium+nucleatum+subsp.+nucleatum+subsp.+nov.%2C+comb.+nov.%3B+Fusobacterium+nucleatum+subsp.+polymorphum+subsp.+nov.%2C+nom.+rev.%2C+comb.+nov.%3B+and+Fusobacterium+nucleatum+subsp.+vincentii+subsp.+nov.%2C+nom.+rev.%2C+comb.+nov.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">94.
              </span><a name="pone.0000659-McLeod1" id="pone.0000659-McLeod1"></a>McLeod MP, Qin X, Karpathy SE, Gioia J, Highlander SK, et al.  (2004) Complete genome sequence of <em>Rickettsia typhi</em> and comparison with sequences of other rickettsiae. J Bacteriol  186: 5842–5855.  <ul class="find" data-citedArticleID="970326" data-doi="10.1128/jb.186.17.5842-5855.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.17.5842-5855.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Rickettsia+typhi+and+comparison+with+sequences+of+other+rickettsiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Rickettsia+typhi+and+comparison+with+sequences+of+other+rickettsiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">95.
              </span><a name="pone.0000659-Ewing1" id="pone.0000659-Ewing1"></a>Ewing B, Green P (1998) Base-calling of automated sequencer traces using Phred.&nbsp;II. Error&nbsp;probabilities. Genome Res  8: 186–194.  <ul class="find" data-citedArticleID="970240" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+Phred.%26%23xa0%3BII.+Error%26%23xa0%3Bprobabilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+Phred.%26%23xa0%3BII.+Error%26%23xa0%3Bprobabilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">96.
              </span><a name="pone.0000659-Ewing2" id="pone.0000659-Ewing2"></a>Ewing B, Hillier L, Wendl MC, Green P (1998) Base-calling of automated sequencer traces using Phred.&nbsp;I. Accuracy&nbsp;assessment. Genome Res  8: 175–185.  <ul class="find" data-citedArticleID="970242" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+Phred.%26%23xa0%3BI.+Accuracy%26%23xa0%3Bassessment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+Phred.%26%23xa0%3BI.+Accuracy%26%23xa0%3Bassessment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">97.
              </span><a name="pone.0000659-Havlak1" id="pone.0000659-Havlak1"></a>Havlak P, Chen R, Durbin KJ, Egan A, Ren Y, et al.  (2004) The Atlas genome assembly system. Genome Res  14: 721–732.  <ul class="find" data-citedArticleID="970262" data-doi="10.1101/gr.2264004"><li><a href="http://dx.doi.org/10.1101/gr.2264004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Atlas+genome+assembly+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Atlas+genome+assembly+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">98.
              </span><a name="pone.0000659-Lukashin1" id="pone.0000659-Lukashin1"></a>Lukashin A, Borodovsky M (1998) GeneMark.hmm: new solutions for gene finding. Nucl Acids Res  26: 1107–1115.  <ul class="find" data-citedArticleID="970322" data-doi="10.1093/nar/26.4.1107"><li><a href="http://dx.doi.org/10.1093/nar/26.4.1107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneMark.hmm%3A+new+solutions+for+gene+finding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneMark.hmm%3A+new+solutions+for+gene+finding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">99.
              </span><a name="pone.0000659-Lowe1" id="pone.0000659-Lowe1"></a>Lowe TM, Eddy SR (1997) tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence. Nucl Acids Res  25: 955–964.  <ul class="find" data-citedArticleID="970320" data-doi="10.1093/nar/25.5.955"><li><a href="http://dx.doi.org/10.1093/nar/25.5.955" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNAscan-SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNAscan-SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">100.
              </span><a name="pone.0000659-GriffithsJones1" id="pone.0000659-GriffithsJones1"></a>Griffiths-Jones S, Bateman A, Marshall M, Khanna A, Eddy SR (2003) Rfam: an RNA family database. Nucl Acids Res  31: 439–441.  <ul class="find" data-citedArticleID="970252" data-doi="10.1093/nar/gkg006"><li><a href="http://dx.doi.org/10.1093/nar/gkg006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rfam%3A+an+RNA+family+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rfam%3A+an+RNA+family+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">101.
              </span><a name="pone.0000659-Zhang1" id="pone.0000659-Zhang1"></a>Zhang R, Zhang C-T (2004) A systematic method to identify genomic islands and its applications in analyzing the genomes of <em>Corynebacterium glutamicum</em> and <em>Vibrio vulnificus</em> CMCP6 chromosome I. Bioinform  20: 612–622.  <ul class="find" data-citedArticleID="970388" data-doi="10.1093/bioinformatics/btg453"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg453" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+systematic+method+to+identify+genomic+islands+and+its+applications+in+analyzing+the+genomes+of+Corynebacterium+glutamicum+and+Vibrio+vulnificus+CMCP6+chromosome+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+systematic+method+to+identify+genomic+islands+and+its+applications+in+analyzing+the+genomes+of+Corynebacterium+glutamicum+and+Vibrio+vulnificus+CMCP6+chromosome+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">102.
              </span><a name="pone.0000659-Thompson1" id="pone.0000659-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucl Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="970378" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.XML" value="210095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.PDF" value="807360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g001.PNG_L" value="447695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g001.PNG_M" value="178424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g001.PNG_S" value="16168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g001.TIF" value="902808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g001.PNG_I" value="91988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g002.PNG_L" value="236124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g002.PNG_M" value="119303"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g002.PNG_S" value="12940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g002.TIF" value="422926"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g002.PNG_I" value="41961"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t001.PNG_L" value="87024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t001.PNG_M" value="47232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t001.PNG_S" value="8162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t001.TIF" value="352680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t001.PNG_I" value="14592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g003.PNG_L" value="1044088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g003.PNG_M" value="121830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g003.PNG_S" value="3558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g003.TIF" value="1459877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g003.PNG_I" value="42099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t002.PNG_L" value="189630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t002.PNG_M" value="102889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t002.PNG_S" value="12584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t002.TIF" value="899798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t002.PNG_I" value="31922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g004.PNG_L" value="334751"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g004.PNG_M" value="129398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g004.PNG_S" value="8900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g004.TIF" value="889880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g004.PNG_I" value="46169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g005.PNG_L" value="33580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g005.PNG_M" value="43995"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g005.PNG_S" value="5201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g005.TIF" value="130026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.g005.PNG_I" value="16672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t003.PNG_L" value="1562665"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t003.PNG_M" value="91825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t003.PNG_S" value="30295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t003.TIF" value="6780038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.t003.PNG_I" value="274140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.s001.DOC" value="1167872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000659.s002.DOC" value="739840"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000659&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000659" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000659&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000659&volume=&issue=&title=Genome Sequence of Fusobacterium nucleatum Subspecies Polymorphum — a Genetically Tractable Fusobacterium&author_name=Sandor%20E.%20Karpathy%2C%20Xiang%20Qin%2C%20Jason%20Gioia%2C%20Huaiyang%20Jiang%2C%20Yamei%20Liu%2C%20Joseph%20F.%20Petrosino%2C%20Shailaja%20Yerrapragada%2C%20George%20E.%20Fox%2C%20Susan%20Kinder%20Haake%2C%20George%20M.%20Weinstock%2C%20Sarah%20K.%20Highlander&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000659" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000659" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000659" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000659&amp;t=Genome%20Sequence%20of%20Fusobacterium%20nucleatum%20Subspecies%20Polymorphum%20%E2%80%94%20a%20Genetically%20Tractable%20Fusobacterium" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000659&title=Genome%20Sequence%20of%20Fusobacterium%20nucleatum%20Subspecies%20Polymorphum%20%E2%80%94%20a%20Genetically%20Tractable%20Fusobacterium&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000659&amp;title=Genome%20Sequence%20of%20Fusobacterium%20nucleatum%20Subspecies%20Polymorphum%20%E2%80%94%20a%20Genetically%20Tractable%20Fusobacterium" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000659" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000659&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genome Sequence of Fusobacterium nucleatum Subspecies Polymorphum — a Genetically Tractable Fusobacterium';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000659';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genome%20Sequence%20of%20Fusobacterium%20nucleatum%20Subspecies%20Polymorphum%20%E2%80%94%20a%20Genetically%20Tractable%20Fusobacterium http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000659" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000659" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="24156"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fusobacteria%22" title="Search for articles in the subject area:'Fusobacteria'"><div class="flagText">Fusobacteria</div></a>
              <div data-categoryid="44761" data-articleid="24156"
                   data-categoryname="Fusobacteria"
                   class="flagImage" title="Flag 'Fusobacteria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+prediction%22" title="Search for articles in the subject area:'Gene prediction'"><div class="flagText">Gene prediction</div></a>
              <div data-categoryid="40279" data-articleid="24156"
                   data-categoryname="Gene prediction"
                   class="flagImage" title="Flag 'Gene prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+metabolism%22" title="Search for articles in the subject area:'Membrane metabolism'"><div class="flagText">Membrane metabolism</div></a>
              <div data-categoryid="18165" data-articleid="24156"
                   data-categoryname="Membrane metabolism"
                   class="flagImage" title="Flag 'Membrane metabolism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+proteins%22" title="Search for articles in the subject area:'Membrane proteins'"><div class="flagText">Membrane proteins</div></a>
              <div data-categoryid="17569" data-articleid="24156"
                   data-categoryname="Membrane proteins"
                   class="flagImage" title="Flag 'Membrane proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Operons%22" title="Search for articles in the subject area:'Operons'"><div class="flagText">Operons</div></a>
              <div data-categoryid="21291" data-articleid="24156"
                   data-categoryname="Operons"
                   class="flagImage" title="Flag 'Operons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Outer+membrane+proteins%22" title="Search for articles in the subject area:'Outer membrane proteins'"><div class="flagText">Outer membrane proteins</div></a>
              <div data-categoryid="20669" data-articleid="24156"
                   data-categoryname="Outer membrane proteins"
                   class="flagImage" title="Flag 'Outer membrane proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24156"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2857'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4013'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2822&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
