

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Preference of Small Molecules for Local Minimum Conformations when Binding to Proteins</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000820"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000820" />

    <meta name="citation_title" content="Preference of Small Molecules for Local Minimum Conformations when Binding to Proteins"/>
    <meta itemprop="name" content="Preference of Small Molecules for Local Minimum Conformations when Binding to Proteins"/>

      <meta name="citation_author" content="Qi Wang"/>
            <meta name="citation_author_institution" content="Computer-Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America"/>
      <meta name="citation_author" content="Yuan-Ping Pang"/>
            <meta name="citation_author_institution" content="Computer-Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America"/>

    <meta name="citation_date" content="2007/9/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000820.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e820"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The key-lock theory and the induced fit theory.; citation_author=DE Koshland; citation_journal_title=Angew Chem Int Ed Engl; citation_volume=33; citation_number=1; citation_pages=2375-2378; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Conformational changes and drug action.; citation_author=AS Burgen; citation_journal_title=Fed Proc; citation_volume=40; citation_number=2; citation_pages=2723-2728; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Conformational induction versus conformational selection: evidence from allosteric enhancers.; citation_author=RF Bruns; citation_journal_title=Trends Pharmacol Sci; citation_volume=17; citation_number=3; citation_pages=189; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Receptor conformational induction versus selection: all part of the same energy landscape, agonists can differentially stabilize multiple active states of receptors.; citation_author=T Kenakin; citation_journal_title=Trends Pharmacol Sci; citation_volume=17; citation_number=4; citation_pages=190-191; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Docking studies on the complexed and uncomplexed FKBP12 structures with bound and unbound ligands: an implication of conformational selection mechanism for binding.; citation_author=Y-P Pang; citation_author=ND Silva; citation_author=C Hydock; citation_author=FG Prendergast; citation_journal_title=J Mol Model; citation_volume=3; citation_number=5; citation_pages=240-248; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=EUDOC: a computer program for identification of drug interaction sites in macromolecules and drug leads from chemical databases.; citation_author=Y-P Pang; citation_author=E Perola; citation_author=K Xu; citation_author=FG Prendergast; citation_journal_title=J Comput Chem; citation_volume=22; citation_number=6; citation_pages=1750-1771; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=In silico drug discovery: solving the “target-rich and lead-poor” imbalance using the genome-to-drug-lead paradigm.; citation_author=Y-P Pang; citation_journal_title=Clin Pharmacol Ther; citation_volume=81; citation_number=7; citation_pages=30-34; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=EUDOC on Blue Gene: accelerating the transfer of drug discoveries from laboratory to patient.; citation_author=Y-P Pang; citation_author=T Mullins; citation_author=B Swartz; citation_author=J McAllister; citation_author=B Smith; citation_journal_title=IBM J Res Dev in press; citation_number=8; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Rusting of the lock and key model for protein-ligand binding.; citation_author=WL Jorgensen; citation_journal_title=Science; citation_volume=254; citation_number=9; citation_pages=954-955; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Do active site conformations of small ligands correspond to low free-energy solution structures?; citation_author=M Vieth; citation_author=JD Hirst; citation_author=CL Brooks; citation_journal_title=J Comput-Aided Mol Design; citation_volume=12; citation_number=10; citation_pages=563-572; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Conformational energy penalties of protein-bound ligands.; citation_author=J Boström; citation_author=P-O Norrby; citation_author=T Liljefors; citation_journal_title=J Comput-Aided Mol Design; citation_volume=12; citation_number=11; citation_pages=383-396; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Conformational changes of small molecules binding to proteins.; citation_author=MC Nicklaus; citation_author=S Wang; citation_author=JS Driscoll; citation_author=GW Milne; citation_journal_title=Bioorg Med Chem; citation_volume=3; citation_number=12; citation_pages=411-428; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Conformational analysis of drug-like molecules bound to proteins: an extensive study of ligand reorganization upon binding.; citation_author=E Perola; citation_author=PS Charifson; citation_journal_title=J Med Chem; citation_volume=47; citation_number=13; citation_pages=2499-2510; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Normal mode theory and harmonic potential approximations;; citation_author=K Hinsen; citation_number=14; citation_pages=1-16; citation_date=2006; citation_publisher=Chapman & Hall/CRC; " />
      <meta name="citation_reference" content="citation_title=Rigidity theory and applications;; citation_author=D Case; citation_number=15; citation_pages=329-344; citation_date=1999; citation_publisher=Kluwer Academic/Plenum Publishers; " />
      <meta name="citation_reference" content="citation_title=Introduction to computational chemistry.; citation_author=F Jensen; citation_number=16; citation_pages=312-315; citation_date=1998; citation_publisher=John Wiley & Sons; " />
      <meta name="citation_reference" content="citation_title=Essentials of computational chemistry.; citation_author=CJ Cramer; citation_number=17; citation_pages=303; citation_date=2002; citation_publisher=John Wiley & Sons; " />
      <meta name="citation_reference" content="citation_title=A second generation force field for the simulation of proteins, nucleic acids, and organic molecules.; citation_author=WD Cornell; citation_author=P Cieplak; citation_author=CI Bayly; citation_author=IR Gould; citation_author=KM Merz; citation_journal_title=J Am Chem Soc; citation_volume=117; citation_number=18; citation_pages=5179-5197; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Development and testing of a general amber force field.; citation_author=J Wang; citation_author=RM Wolf; citation_author=JW Caldwell; citation_author=PA Kollman; citation_author=DA Case; citation_journal_title=J Comput Chem; citation_volume=25; citation_number=19; citation_pages=1157-1174; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=AMBER, a package of computer programs for applying molecular mechanics, normal mode analysis, molecular dynamics and free energy calculations to simulate the structural and energetic properties of molecules.; citation_author=DA Pearlman; citation_author=DA Case; citation_author=JW Caldwell; citation_author=WS Ross; citation_author=TE Cheatham; citation_journal_title=Comput Phys Commun; citation_volume=91; citation_number=20; citation_pages=1-41; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Structure of the Tsg101 UEV domain in complex with the PTAP motif of the HIV-1 p6 protein.; citation_author=O Pornillos; citation_author=SL Alam; citation_author=DR Davis; citation_author=WI Sundquist; citation_journal_title=Nat Struct Biol; citation_volume=9; citation_number=21; citation_pages=812-817; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=From genome to drug lead: identification of a small-molecule inhibitor of the SARS virus.; citation_author=AJ Dooley; citation_author=N Shindo; citation_author=B Taggart; citation_author=JG Park; citation_author=Y-P Pang; citation_journal_title=Bioorg Med Chem Lett; citation_volume=16; citation_number=22; citation_pages=830-833; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Three-dimensional model of a substrate-bound SARS chymotrypsin-like cysteine proteinase predicted by multiple molecular dynamics simulations: catalytic efficiency regulated by substrate binding.; citation_author=Y-P Pang; citation_journal_title=Proteins; citation_volume=57; citation_number=23; citation_pages=747-757; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Accurate reproduction of 161 small-molecule complex crystal structures using the EUDOC program: expanding the use of EUDOC to supramolecular chemistry.; citation_author=Q Wang; citation_author=Y-P Pang; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=24; citation_pages=e531; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Protein Data Bank.; citation_author=HM Berman; citation_author=J Westbrook; citation_author=Z Feng; citation_author=G Gilliland; citation_author=TN Bhat; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=25; citation_pages=235-242; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Application of the multimolecule and multiconformational RESP methodology to biopolymers: Charge derivation for DNA, RNA, and proteins.; citation_author=P Cieplak; citation_author=WD Cornell; citation_author=C Bayly; citation_author=PA Kollman; citation_journal_title=J Comput Chem; citation_volume=16; citation_number=26; citation_pages=1357-1377; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=GAUSSIAN 98, Revision A.7.; citation_author=MJ Frisch; citation_author=GW Trucks; citation_author=HB Schlegel; citation_author=PMW Gill; citation_author=BG Hohnson; citation_number=27; citation_date=1999; citation_publisher=Gaussian, Inc; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000820" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Preference of Small Molecules for Local Minimum Conformations when Binding to Proteins"/>
    <meta name="twitter:description" content="It is well known that small molecules (ligands) do not necessarily adopt their lowest potential energy conformations when binding to proteins. Analyses of protein-bound ligand crystal structures have reportedly shown that many of them do not even adopt the conformations at local minima of their potential energy surfaces (local minimum conformations). The results of these analyses raise a concern regarding the validity of virtual screening methods that use ligands in local minimum conformations. Here we report a normal-mode-analysis (NMA) study of 100 crystal structures of protein-bound ligands. Our data show that the energy minimization of a ligand alone does not automatically stop at a local minimum conformation if the minimum of the potential energy surface is shallow, thus leading to the folding of the ligand. Furthermore, our data show that all 100 ligand conformations in their protein-bound ligand crystal structures are nearly identical to their local minimum conformations obtained from NMA-monitored energy minimization, suggesting that ligands prefer to adopt local minimum conformations when binding to proteins. These results both support virtual screening methods that use ligands in local minimum conformations and caution about possible adverse effect of excessive energy minimization when generating a database of ligand conformations for virtual screening."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000820.g007"/>

  <meta property="og:title" content="Preference of Small Molecules for Local Minimum Conformations when Binding to Proteins" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000820" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4152'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4263'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8236&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000820">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000820" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000820&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Preference of Small Molecules for Local Minimum Conformations when Binding to Proteins
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Qi Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computer-Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yuan-Ping Pang
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pang@mayo.edu">pang@mayo.edu</a></p>

                <p>Affiliation:
                  Computer-Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 05, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000820</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000820" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000820" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000820" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000820" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000820-g001" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000820-g002" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000820-g003" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000820-t001" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000820-g004" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000820-g005" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000820-t002" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000820-g006" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000820-g007" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000820-t003" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000820">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000820" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>It is well known that small molecules (ligands) do not necessarily adopt their lowest potential energy conformations when binding to proteins. Analyses of protein-bound ligand crystal structures have reportedly shown that many of them do not even adopt the conformations at local minima of their potential energy surfaces (local minimum conformations). The results of these analyses raise a concern regarding the validity of virtual screening methods that use ligands in local minimum conformations. Here we report a normal-mode-analysis (NMA) study of 100 crystal structures of protein-bound ligands. Our data show that the energy minimization of a ligand alone does not automatically stop at a local minimum conformation if the minimum of the potential energy surface is shallow, thus leading to the folding of the ligand. Furthermore, our data show that all 100 ligand conformations in their protein-bound ligand crystal structures are nearly identical to their local minimum conformations obtained from NMA-monitored energy minimization, suggesting that ligands prefer to adopt local minimum conformations when binding to proteins. These results both support virtual screening methods that use ligands in local minimum conformations and caution about possible adverse effect of excessive energy minimization when generating a database of ligand conformations for virtual screening.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wang Q, Pang Y-P (2007) Preference of Small Molecules for Local Minimum Conformations when Binding to Proteins. PLoS ONE 2(9):
          e820.
            doi:10.1371/journal.pone.0000820</p><p><strong>Academic Editor: </strong>Floyd Romesberg, The Scripps Research Institute, United States of America</p><p><strong>Received:</strong> May 28, 2007; <strong>Accepted:</strong> August 6, 2007; <strong>Published:</strong> September 5, 2007</p><p><strong>Copyright:</strong> © 2007 Wang, Pang. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the U.S. Army Medical Research Acquisition Activity (W81XWH-04-2-0001), the National Institutes of Health (5R01AI054574-03 and 5R01GM061300-06), Mayo Graduate School, and the Mayo Foundation for Medical Education and Research.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Molecular complexation in biology is best described by the conformational induction theory <a href="#pone.0000820-Koshland1">[1]</a>—namely, a ligand (e.g., a small molecule) binds initially to a less compatible conformation of a receptor (e.g., a protein) and then adjusts its conformation to induce the most compatible conformation of the receptor. The conformation induction theory is, however, not ideal for computationally addressing the conformational flexibility of both ligand and receptor in docking studies, because computing the mutually dependent conformational changes of both partners on the fly is time-consuming and unsuitable for parallel computing. Alternatively, the conformation selection theory describes that both ligand and receptor select their <em>preformed</em> conformations that are most compatible with one another to effect binding by shifting two equilibriums progressively from less compatible to most compatible conformations for both partners, where the preformed and most compatible conformations are conformations at local minima of their potential energy surfaces (local minimum conformations) <a href="#pone.0000820-Burgen1">[2]</a>–<a href="#pone.0000820-Pang1">[5]</a>. When the most compatible conformations of both partners are most prevalent, the conformation selection theory becomes the lock–key theory <a href="#pone.0000820-Koshland1">[1]</a>. The conformation selection theory is ideal to computationally account for molecular flexibility in docking, because it can convert a ligand–receptor association best described by the conformational induction theory to a series of associations each of which can be described by the lock-key theory <a href="#pone.0000820-Pang2">[6]</a>. The conformation selection theory thereby affords parallel computing and enables a docking study to be performed on thousands of IBM Blue Gene processors with high processor utilization <a href="#pone.0000820-Pang2">[6]</a>–<a href="#pone.0000820-Pang4">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>It is well known that ligands do not necessarily adopt their lowest potential energy conformations when binding to their protein targets <a href="#pone.0000820-Jorgensen1">[9]</a>–<a href="#pone.0000820-Bostrm1">[11]</a>. Analyses of crystal structures of protein-bound ligands have reportedly shown, however, that many of them do not adopt their local minimum conformations <a href="#pone.0000820-Nicklaus1">[12]</a>, <a href="#pone.0000820-Perola1">[13]</a>. In particular, a study of 150 protein-bound ligand crystal structures showed that more than 60% of them do not adopt local minimum conformations <a href="#pone.0000820-Perola1">[13]</a>. The results of these analyses raise a concern regarding the validity of virtual screening methods that use ensembles of local minimum conformations of both ligand and receptor to address molecular flexibility according to the conformation selection theory <a href="#pone.0000820-Pang2">[6]</a>. To address this concern by investigating why many ligands reportedly do not adopt local minimum conformations when binding to proteins <a href="#pone.0000820-Perola1">[13]</a>, we carried out a normal-mode-analysis (NMA) study, that used analytic means to analyze harmonic potential wells and classify possible deformations of these ligands according to their energetic costs <a href="#pone.0000820-Hinsen1">[14]</a>–<a href="#pone.0000820-Cramer1">[17]</a>, using the second-generation AMBER force field <a href="#pone.0000820-Cornell1">[18]</a>, <a href="#pone.0000820-Wang1">[19]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here we report an NMA study of 100 available protein-bound ligand crystal structures that were studied in reference 13. Our data show that the energy minimization of a ligand alone does not automatically stop at a local minimum conformation. Furthermore, our data show that all 100 ligand conformations in their protein-bound ligand crystal structures are nearly identical to their local minimum conformations that were obtained from the crystal structures and refined by energy minimization whose progress was monitored with NMA. These results both support the virtual screening methods that use local minimum conformations and caution about possible adverse effect of excessive energy minimization on generating databases of ligand conformations for virtual screening.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Ligand folding caused by excessive energy minimization</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To understand why more than 60% of the 100 available protein-bound ligand crystal structures are reportedly not in their local minimum conformations <a href="#pone.0000820-Perola1">[13]</a>, we carried out an NMA study of these ligands in the absence of their protein partners. This study was pursued because the translational, rotational, and vibrational frequencies of a nonlinear ligand can be used diagnostically to determine how close the ligand conformation is to its local minimum conformation <a href="#pone.0000820-Cramer1">[17]</a>. Before starting NMA, it was necessary to perform a few steps of energy minimization on a ligand conformation, taken from the ligand-protein complex crystal structure, in the absence of its protein partner to “adapt” the ligand to the force field used by the NMA as well as to reduce the gradient of the ligand potential energy to zero <a href="#pone.0000820-Case1">[15]</a>. This preparation was necessary because ligand conformations in crystal structures are refined to best fit the electron density map. Such conformations can be energetically unstable or “strained” if their potential energies are evaluated in the absence of proteins using a force field that is different from the one used by the NMA.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>By performing 10<sup>6</sup> steps of energy minimization on one of the 100 ligands [Protein Data Bank (PDB) code: 1QL9] in the absence of its protein partner, we obtained a conformation that was a local minimum conformation (L1) according to NMA. This energy minimization procedure and the ones described hereafter were carried out using the SANDER module of the AMBER 5 program <a href="#pone.0000820-Pearlman1">[20]</a> with the AMBER force field <a href="#pone.0000820-Cornell1">[18]</a>, <a href="#pone.0000820-Wang1">[19]</a>. L1 has a mass-weighted root mean square deviation (mwRMSD) of 1.98 Å relative to the crystal structure conformation, indicating that the ligand conformation in the crystal structure was not in its local minimum conformation. This indication was based on a criterion that two conformations are different if their mwRMSD is more than 1.00 Å. Interestingly, using a new energy minimization procedure that uses NMA to monitor energy minimization progress as described below, we obtained a different local minimum conformation (L2) with an mwRMSD of 0.31 Å, indicating that the ligand conformation in the crystal structure was in its local minimum conformation.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>A further NMA study identified another different local minimum conformation (L3) and two transition state conformations (T1 and T2). These three conformations are related to the exchange between L1 and L2. As shown in <a href="#pone-0000820-g001">Figure 1</a>, L2 changes to T1, then to L3, then to T2, and then to L1; the transition from L2 to L1 markedly changes the ligand conformation found in the crystal structure yet the highest energy barrier for this transition is only 1.95 kcal/mol. Within the context of the AMBER force field (because local minimum conformations are specific to the force field used by NMA), these results show that the energy minimization of a ligand conformation alone does not automatically stop at a local minimum conformation if the minimum of the potential energy surface is shallow. The ligand can progress through multiple shallow, local minima if the progress of the energy minimization is not monitored with NMA.</p>
<div class="figure" id="pone-0000820-g001"><div class="img"><a name="pone-0000820-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Three local minimum conformations of the ligand taken from crystal structure 1QL9 and two related transition state conformations.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Upper panel: local minimum, transition state, and crystal structure conformations of the ligand. The C atom of the crystal structure conformation is green. The C atoms of the local minimum conformations of L1, L2, and L3 are yellow, magenta, and cyan, respectively. The C atoms of the two transition state conformations are blue. The O, N and S atoms are red, blue, and orange, respectively. Lower panel: the potential energy diagram for the exchange between L2 and L1. The unit of the potential energy is kcal/mol.</p>
<span>doi:10.1371/journal.pone.0000820.g001</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>After energy minimization of the 100 ligands in the absence of their protein partners using 10<sup>6</sup> steps of energy minimization, we found that all of the energy-minimized conformations were in their local minimum conformations according to NMA, but 42 and 72% of them had mwRMSDs greater than 1.00 and 0.50 Å, respectively, relative to their crystal structure conformations. These results suggest that, similar to the ligand in crystal structure 1QL9, other ligands may have multiple shallow minima on their potential energy surfaces. Within the context of the AMBER force field, these results also caution that excessive energy minimization can fold or partially fold a ligand in its free state. Such folding can mislead the determination of whether the crystal structure conformations are in their local minimal conformations, although the ability to progress through multiple shallow, local minima is desirable in searching for the global minimum conformation or lower local minimum conformations.</p>


<h4>Normal-mode-analysis-monitored energy minimization procedure</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To stop energy minimization of a ligand in its free state when it reaches at a local minimum conformation—thus avoiding ligand folding—we devised an NMA-monitored energy minimization (NEM) procedure to investigate whether the 100 ligands under study were in, near, or far from local minimum conformations. As shown in <a href="#pone-0000820-g002">Figure 2</a>, this procedure begins with 10 steps of energy minimization on a ligand (in this study the ligand conformation was taken from its protein–bound ligand crystal structure). The energy-minimized ligand conformation is then subject to NMA to check whether the ligand is in its local minimum conformation. The 10-step energy minimization uses a gradient cut-off of 10<sup>−7</sup> kcal/(mol•Å) and is repeated until the NMA shows that the ligand is in a local minimum conformation. After each 10-step energy minimization, the gradient of the ligand potential energy is checked first. If the gradient is &gt;0.06 kcal/(mol•Å), NMA is aborted, and the ligand is considered not to be in its local minimum conformation. If the gradient is ≤0.06 kcal/(mol•Å), NMA is performed, and the magnitudes of three translational and three rotational frequencies are checked. If the magnitudes of all translational frequencies are &lt;0.01 cm<sup>−1</sup> and the magnitudes of all rotational frequencies are &lt;10 cm<sup>−1</sup>, all vibrational frequencies are checked; otherwise, the analysis of vibrational frequencies is aborted and the ligand is considered not to be in its local minimum conformation. If all the vibrational frequencies are positive, the ligand is considered to be in its local minimum conformation <a href="#pone.0000820-Jensen1">[16]</a>, <a href="#pone.0000820-Cramer1">[17]</a>. The cut-offs for the gradient and for the translational and rotational frequencies are obtained from reference 16 and based on the fact that geometry cannot be optimized to a gradient of exact zero because of numeric truncations <a href="#pone.0000820-Jensen1">[16]</a>. The NEM procedure is automated by a Perl script shown in <a href="#pone.0000820.s006">Figure S1</a>.</p>
<div class="figure" id="pone-0000820-g002"><div class="img"><a name="pone-0000820-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Flowchart of the normal-mode-analysis–monitored energy minimization procedure.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>mwRMSD stands for mass-weighted root mean square deviation.</p>
<span>doi:10.1371/journal.pone.0000820.g002</span></div>

<h4>Preference for local minimum conformations</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>For each of the 100 ligands under study, we obtained a local minimum conformation with an mwRMSD of &lt;1.00 Å relative to the ligand conformation in the crystal structure (<a href="#pone-0000820-t001">Tables 1</a> and <a href="#pone.0000820.s001">S1</a>) when using the NEM procedure, the conjugate gradient minimization method, and a gradient cut-off of 0.06 kcal/(mol•Å). In contrast to the observation that 28 of the 100 ligands have local minimum conformations with mwRMSDs of ≤0.50 Å relative to their crystal structure conformations when using the conventional energy minimization method as described above, 69 of the 100 ligands have local minimum conformations with mwRMSDs of ≤0.50 Å when using the NEM procedure, the conjugate gradient minimization method, and a gradient cut-off of 0.06 kcal/(mol•Å) (<a href="#pone-0000820-t001">Tables 1</a> and <a href="#pone.0000820.s001">S1</a>). The same result was obtained when using the NEM procedure, the steepest descent minimization method, and a gradient cut-off of 0.06 kcal/(mol•Å) (<a href="#pone.0000820.s002">Table S2</a>). In addition, 78 of the 100 ligands have mwRMSDs of ≤1.00 Å between the ligand conformations in their crystal structures and their corresponding local minimum conformations when using the NEM procedure, the conjugate gradient minimization method, and a smaller gradient cut-off of 10<sup>−2</sup> kcal/(mol•Å) that adds extra steps of energy minimization (<a href="#pone.0000820.s003">Table S3</a>). <a href="#pone-0000820-g003">Figure 3</a> shows the closeness between the ligand conformations in the crystal structures and their corresponding local minimum conformations obtained using the NEM procedure (mwRMSDs of 0.11–0.97 Å). Within the context of the AMBER force field, these results indicate that all 100 ligand conformations in the crystal structures of their protein complexes are nearly identical to their local minimum conformations, demonstrating the preference of these ligands for local minimum conformations when binding to proteins.</p>
<div class="figure" id="pone-0000820-g003"><div class="img"><a name="pone-0000820-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Graphic representation of mass-weighted root mean square deviations (mwRMSDs) of 0.11–0.97 Å between crystal structure conformations and their corresponding local minimum conformations obtained by using normal mode analysis.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The C atoms of the crystal structure conformation and the local minimum conformation are green and yellow, respectively. The O, N, F and S atoms are red, blue, grey, and orange, respectively.</p>
<span>doi:10.1371/journal.pone.0000820.g003</span></div><div class="figure" id="pone-0000820-t001"><div class="img"><a name="pone-0000820-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000820.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000820.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Mass-Weighted Root Mean Square Deviations (mwRMSDs) of All Ligand Atoms between the Crystal Structure Conformation and the Local Minimum Conformation for the 100 Protein-Bound Ligands.</span></strong></p><span>doi:10.1371/journal.pone.0000820.t001</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Support for the conformation-selection-theory-based virtual screening</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Whereas other approaches such as performing a local conformational search to find multiple minima near the crystal structure conformation may find local minimum conformations with smaller mwRMSDs, this study already shows that all 100 and 69 of the 100 ligands have mwRMSDs less than 1.00 and 0.50 Å, respectively, between the ligand conformations in their complex crystal structures and the corresponding local minimum conformations. Although further studies with other force fields are desirable, given the similarity of the AMBER force field to other force fields and the results of this study, it is reasonable to suggest that ligands do not necessarily adopt their global minimum conformations when binding to proteins but they generally prefer to adopt local minimum conformations upon complexation. This preference seems to be independent of ligand properties such as the number of rotatable bonds because there is no correlation between the mwRMSD obtained with the NMA monitor (M1 column of <a href="#pone-0000820-t001">Table 1</a>) and the number of rotatable bonds for the 100 ligands under study (<a href="#pone-0000820-g004">Figure 4</a>). The results of this study lend support to the conformation-selection-theory–based virtual screening methods that use local minimum conformations generated by using the AMBER force field.</p>
<div class="figure" id="pone-0000820-g004"><div class="img"><a name="pone-0000820-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Distribution of the mass-weighted root mean square deviations (mwRMSDs) between the 100 ligand crystal structure conformations and their corresponding local minimum conformations obtained by using normal mode analysis among the number of rotatable bonds of these ligands.</span></strong></p><span>doi:10.1371/journal.pone.0000820.g004</span></div>

<h4>Adverse effect of excessive energy minimization on chemical database development</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The results of this study suggest that excessive energy minimization of a ligand alone may fold or partially fold the ligand, leading to a conformation that is more stable in its free state but not suitable for docking into a protein. The partial folding caused by the conventional energy minimization method is common to the 100 ligands under study, as evident from the observation that the potential energies of all 100 ligand conformations obtained using the energy minimization without the NMA monitor are by average 1.1 kcal/mol lower than those using the NMA monitor except for the ligand of crystal structure 3STD (<a href="#pone-0000820-g005">Figure 5</a>). It is worth noting that the potential energy of the ligand local minimum conformation of crystal structure 3STD obtained without the NMA monitor is only 0.08 kcal/mol higher than that obtained with the NMA monitor. Docking a folded or partially folded ligand into a protein binding site requires computational effort to unfold the ligand. This effort is necessary in view of the entropy contribution to binding. This is because the unfolded ligand conformation such as the NEM-generated local minimum conformation is often captured at a <em>shallow</em> minimum of the ligand potential energy surface. The unfolded conformation at a shallow “energy well” can gain more entropic energy than the partially folded conformation at a deep “energy well,” which can outweigh the penalty in potential energy (≈1.1 kcal/mol in average) for moving the conformation from the deep “energy well” to the shallow “energy well.” This cautions about possible adverse effect of excessive energy minimization on generating a database of ligand conformations for virtual screening.</p>
<div class="figure" id="pone-0000820-g005"><div class="img"><a name="pone-0000820-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Scatterplot of the potential energy differences between two local minimum conformations obtained without and with normal mode analysis for the 100 ligands.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>The ligand ID is the row order of the ligand in <a href="#pone.0000820.s001">Table S1</a>—namely, the IDs of the ligands in the first and last rows of <a href="#pone.0000820.s001">Table S1</a> are 1 and 100, respectively.</p>
<span>doi:10.1371/journal.pone.0000820.g005</span></div>

<h4>Generating protein-bound ligand conformations</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The NEM procedure described herein appears to be promising for generating unfolded or bound conformations <em>a priori</em>, given the fact that computers are nowadays fast enough to run NMA on drug-like ligands and even on proteins. For example, an Apple computer with two 2.0 GHz G5 processors can complete an NMA on a 153-residue protein TSG101 (PDB code: 1M4P <a href="#pone.0000820-Pornillos1">[21]</a>) and a 308-residue protein CCP (PDB code: 2AJ5 <a href="#pone.0000820-Dooley1">[22]</a>, <a href="#pone.0000820-Pang5">[23]</a>) in 21 minutes (using 320 MB memory) and 120 minutes (using 1.1 GB memory), respectively. The promise of the NEM procedure is further strengthened by the fact that the NEM procedure can be improved by enabling NMA to be performed on multiple processors and by guiding energy minimization with the information contained in the full Hessian matrix (Thompson and Pang's work in progress).</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>To test whether the NEM procedure using the AMBER force field can practically generate protein-bound ligand conformations from the two-dimensional (2D) ligand structures in the absence of their protein partners, six of the 100 protein-bound ligand crystal structures were selected as model systems. The selection criterion was that ligands do not have more than four conformation-governing rotatable bonds and this criterion was used in order to better test the necessity of the NEM procedure as described below. The PDB codes of these structures are 1BR6, 1EFY, 1L2S, 1QPE, 2CSN, and 4STD. The procedure for producing the six protein-bound ligand conformations from their 2D chemical structures <em>a priori</em> by using the NEM procedure and docking is detailed in <a href="#s4">METHODS</a>. Briefly, it began with generation of a set of rotamers from the 2D ligand structure by systematically changing all conformation-governing rotatable bonds of the ligand in increments of 60° of arc starting from 0°. All these rotamers were then optimized to local minimum conformations by the NEM procedure. The resulting local minimum conformations were then subject to a cluster analysis with consideration of molecular symmetry to remove duplicated or similar conformations. Different local minimum conformations of the ligand were then docked into the binding site of the protein structure taken from the complex crystal structure using the EUDOC program <a href="#pone.0000820-Pang2">[6]</a>, <a href="#pone.0000820-Pang4">[8]</a>, <a href="#pone.0000820-Wang2">[24]</a>. The EUDOC-generated protein-bound ligand complex with the strongest intermolecular interaction energy was compared to the corresponding complex crystal structure. Two types of ligand mwRMSDs were calculated. One was calculated in the absence of the protein partner–namely, the ligand mwRMSD was calculated by superimposing the ligand structure of the complex crystal structure over the ligand structure of the EUDOC-generated protein-bound ligand complex. The other was calculated in complex with the protein partner–namely, the ligand mwRMSD was calculated by superimposing the protein structure of the complex crystal structure over the protein structure of the EUDOC-generated protein-bound ligand complex. As apparent from <a href="#pone-0000820-t002">Table 2</a>, both mwRMSDs are &lt;1.00 Å for all six selected ligand crystal structures. These results demonstrate that the NEM procedure can indeed generate protein-bound conformations from their 2D ligand structures <em>a priori</em>.</p>
<div class="figure" id="pone-0000820-t002"><div class="img"><a name="pone-0000820-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000820.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000820.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Reproduction of the Six Selected Protein-Ligand Complex Crystal Structures by Using the Normal-Mode-Analysis-Monitored Energy Minimization Procedure and Docking.</span></strong></p><span>doi:10.1371/journal.pone.0000820.t002</span></div><a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>To evaluate the necessity of the NEM procedure for generating protein-bound ligand conformations to be used in docking, the above studies with the six selected ligands were repeated with the same procedure except that the rotamers were energy minimized using a conventional method that uses AMBER's default gradient cut-off of 10<sup>−4</sup> kcal/(mol•Å) and does not use NMA to monitor the energy minimization progress. Interestingly, of the six ligands, the conventional energy minimization method failed to generate two protein-bound ligand conformations from their 2D ligand structures (PDB codes: 1BR6 and 1L2S). This result is in contrast to the prediction that the conventional energy minimization method might be able to generate ligand bound conformations from their 2D ligand structures in these cases, because the selected ligands are relatively rigid and they can hardly fold. In the case of the ligand in crystal structure 1BR6, using the NEM-generated local minimum conformations of the ligand, EUDOC identified a ligand bound conformation with the strongest interaction energy of –103.6 kcal/mol and an mwRMSD of 0.49 Å relative to the protein-bound ligand crystal structure (<a href="#pone-0000820-g006">Figure 6</a>); however, using the local minimum conformations generated by the conventional energy minimization method, EUDOC identified a ligand bound conformation with the strongest interaction energy of –88.3 kcal/mol and an mwRMSD of 5.33 Å relative to the protein-bound ligand crystal structure. These results demonstrate that excessive energy minimization caused partial folding of the rotamers of the relatively rigid ligand 1BR6. The partial folding consequently made the energy minimized rotamers less complementary to its protein partner both energetically and structurally (<a href="#pone-0000820-t003">Table 3</a>). The study with ligand 1L2S offered the same conclusion (<a href="#pone-0000820-t003">Table 3</a> and <a href="#pone-0000820-g007">Figure 7</a>). These results suggest that the NEM procedure is not only practical but also desirable for improving the success rate of docking drug-like ligands through working with bound/unfolded local minimum conformations that have inherent entropic advantage as described above. A comprehensive study on the NEM procedure for generating small-molecule bound/unfolded local minimum conformations from their 2D structures via optimizing rotamers using the NEM procedure is in progress and will be reported in due course.</p>
<div class="figure" id="pone-0000820-g006"><div class="img"><a name="pone-0000820-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Overlay of crystal structure 1BR6 with the EUDOC-generated complex whose bound ligand conformation was generated by using the NEM procedure.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>The C atoms of the EUDOC-generated and crystal structures are green and yellow, respectively. The O and N atoms are red and blue, respectively. The mass-weighted root mean square deviation of the ligand conformation between EUDOC-generated and crystal structures is 0.49 Å.</p>
<span>doi:10.1371/journal.pone.0000820.g006</span></div><div class="figure" id="pone-0000820-g007"><div class="img"><a name="pone-0000820-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000820.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Overlay of crystal structure 1L2S with the EUDOC-generated complex whose bound ligand conformation was generated by using the NEM procedure.</span></strong></p><a id="article1.body1.sec3.sec3.fig2.caption1.p1" name="article1.body1.sec3.sec3.fig2.caption1.p1"></a><p>The C atoms of the EUDOC-generated and crystal structures are green and yellow, respectively. The O, N, and Cl atoms are red, blue, and magenta, respectively. The mass-weighted root mean square deviation of the ligand conformation between EUDOC-generated and crystal structures is 0.93 Å.</p>
<span>doi:10.1371/journal.pone.0000820.g007</span></div><div class="figure" id="pone-0000820-t003"><div class="img"><a name="pone-0000820-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000820" data-uri="info:doi/10.1371/journal.pone.0000820.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000820.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000820.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000820.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000820.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Comparison of the Normal-Mode-Analysis-Monitored Energy Minimization Procedure (NEM) with the Conventional Energy Minimization Procedure (Conventional) for Reproducing Ligand-Protein Complex Crystal Structures.</span></strong></p><span>doi:10.1371/journal.pone.0000820.t003</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Program modules and force field parameter files</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Due to varied performances obtained from code optimizations on the Apple G5 processors, the SANDER module of the AMBER 5 program <a href="#pone.0000820-Pearlman1">[20]</a> was used for energy minimization; the NMODE modules of the AMBER 5 and 8 programs <a href="#pone.0000820-Pearlman1">[20]</a> were used for transition-state structure search and NMA, respectively. The parm99.dat and gaff.dat files of the AMBER force field were used for proteins and small molecules, respectively.</p>


<h4>Preparation of the 100 protein-bound ligands</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The 100 ligands used in this study were obtained from corresponding crystal structures of protein-ligand complexes available at the Protein Data Bank <a href="#pone.0000820-Berman1">[25]</a>. The hydrogen atoms of the ligands were added using the QUANTA97 program (Accelrys Software, Inc, San Diego, California). The protonation state of each ligand shown in <a href="#pone.0000820.s007">Figures S2</a>, <a href="#pone.0000820.s008">S3</a>, <a href="#pone.0000820.s009">S4</a>, <a href="#pone.0000820.s010">S5</a> and <a href="#pone.0000820.s011">S6</a> was determined according to p<em>K</em>­a values of the functional groups of the ligand and its nearby residues of the protein partner at pH 7.4. Because the coordinates of the <em>p</em>-nitroanilide group of the ligand in crystal structure 5TLN were undetermined, this group was manually added to the truncated ligand by using the QUANTA97 program; it was included in the energy minimization and NMA but excluded in the mwRMSD calculation. The force field parameters for ligand atom types that are unavailable in the AMBER force field parameter file were generated with the ANTECHAMBER module of the AMBER 7 program using gaff.dat <a href="#pone.0000820-Wang1">[19]</a> (<a href="#pone.0000820.s004">Table S4</a> and Figures <a href="#pone.0000820.s012">S7</a>, <a href="#pone.0000820.s013">S8</a>, <a href="#pone.0000820.s014">S9</a>, <a href="#pone.0000820.s015">S10</a>, <a href="#pone.0000820.s016">S11</a>, <a href="#pone.0000820.s017">S12</a> and <a href="#pone.0000820.s018">S13</a>). The atomic charges of all the ligands (<a href="#pone.0000820.s005">Table S5</a>) were generated according to the RESP procedure <a href="#pone.0000820-Cieplak1">[26]</a> with ab initio calculations at the HF/6-31G* level using the Gaussian98 program <a href="#pone.0000820-Frisch1">[27]</a>. The topology and coordinate files of all the ligands used in energy minimization and NMA were generated with the PREP, LINK, EDIT, and PARM modules of the AMBER 5 program <a href="#pone.0000820-Pearlman1">[20]</a>.</p>


<h4>Energy minimization and normal mode analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Energy minimization used (1) 10<sup>6 </sup>steps of energy minimization, (2) a dielectric multiplicative constant of 80.0, (3) the steepest descent or conjugate gradient method, (4) a nonbonded cut-off of 12 Å, (9) a 10<sup>−7</sup>-kcal/(mole•Å) cut-off for the root-mean-square of the Cartesian elements of the gradient, and (10) defaults for other inputs of the SANDER module. NMA used (1) a dielectric multiplicative constant of 80.0, (2) a nonbonded cut-off of 12 Å, and (3) defaults for other inputs of the NMODE module. The transition state structures were obtained by using (1) 10<sup>5 </sup>steps of energy minimization, (2) a flag to save coordinates at every step of the energy minimization, (3) initial minimization step length of 0.01 Å, (4) 200 eigenvectors, (5) the ordering of the “true” vibrational normal modes of ≤200, and (6) defaults for other inputs of the NMODE module.</p>


<h4>Generation of protein-bound conformations using the NEM Procedure</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>For each ligand, a set of local minimum conformations as potential protein-bound ligand conformations was generated from the 2D ligand structure in the absence of a protein partner according to the following steps. (1) A 2D ligand chemical structure was converted to a 3D structure by using the QUANTA97 program (Accelrys Software, Inc, San Diego, California). The atomic charges of the 3D ligand structure were generated using the same method described above for the atomic charges of the 100 protein-bound ligands. (2) New conformations of the 3D ligand structure were generated by systematically changing all conformation-governing rotatable bonds using the INTERFACE module of the AMBER 5 program <a href="#pone.0000820-Pearlman1">[20]</a> at a torsion increment of 60° of arc starting from 0°. The INTERFACE module generated 6<sup>n</sup> conformations in total, where n is the number of conformation-governing rotatable bonds. The torsional restraints used by the INTERFACE module were set as parabolic to the designated angle ± 40° of arc and linear sides beyond that torsion range. The force constant used to restrain the conformation-governing rotatable bonds was set to 50 kcal/(mol•rad<sup>2</sup>). (3) Each conformation generated by the INTERFACE module was energy minimized to a local minimum conformation by using a Perl script for the NEM procedure (<a href="#pone.0000820.s006">Figure S1</a>). (4) The resulting local minimum conformations were subject to a cluster analysis with consideration of molecular symmetry. Each cluster contained all the conformations each of whose conformation-governing rotatable bonds is within ± 30° of arc of the average torsion of all conformations in the cluster. (5) One conformation was randomly chosen from each cluster as a representative conformation.</p>


<h4>Docking studies using the EUDOC program</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The algorithm of the EUDOC program has been reported elsewhere <a href="#pone.0000820-Pang2">[6]</a>. Briefly, it uses a systematic search protocol, translating and rotating a ligand in a putative binding pocket of a protein and repeating the translations and rotations with different conformations of both protein and ligand to search for energetically favorable conformations, orientations, and positions of the ligand relative to those of the protein. A docking box is defined within the binding pocket to confine the translation of the ligand. The intermolecular interaction energy is the potential energy of the protein-ligand complex relative to the potential energies of both partners in their free state. This energy is calculated according to Equations 1 and 2 using the second-generation AMBER force field <a href="#pone.0000820-Cornell1">[18]</a>. In calculating the intermolecular interaction energy, the multiplicative dielectric constant was set to 1.0, and the distance cut-offs for steric and electrostatic interactions were set to 10<sup>9</sup> Å.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(Eq.1)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000820.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(Eq.2)</span></span><br>In this study, a docking box was defined to enclose the ligand structure in the complex crystal structure; each dimension of the box is greater than 6 Å; the complex-prediction module of the EUDOC program was used to translate and rotate the ligand at translational and rotational increments of 1.0 Å and 10° of arc, respectively; the EUDOC program automatically repeated the translations and rotations with all different local minimum conformations of a ligand and one protein conformation taken from the complex crystal structure.</p>


<h4>Mass-weighted root mean square deviation calculations</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The mwRMSDs of the 100 ligand local minimum conformations obtained using the NEM or conventional procedure relative to the corresponding crystal structures were calculated by superimposing the ligand local minimum conformation over the ligand conformation in the corresponding crystal structure using the PTRAJ module of the AMBER 8 program <a href="#pone.0000820-Pearlman1">[20]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000820"></div><a name="pone.0000820.s001" id="pone.0000820.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Root mean square deviations, rotational and vibrational frequencies, and gradients of local minimum conformations obtained from normal-mode-analysis-monitored energy minimization of protein-bound ligand conformations taken from the 100 crystal structures using the conjugated gradient minimization method with a gradient cut-off of 0.06 kcal/(mol•Å). All three translational frequencies were zero and are not listed.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0000820.s002" id="pone.0000820.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Root mean square deviations, rotational and vibrational frequencies, and gradients of local minimum conformations obtained from normal-mode-analysis-monitored energy minimization of protein-bound ligand conformations taken from the 100 crystal structures using the steepest descent minimization method with a gradient cut-off of 0.06 kcal/(mol•Å). All three translational frequencies were zero and are not listed.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0000820.s003" id="pone.0000820.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Root mean square deviations, rotational and vibrational frequencies, and gradients of local minimum conformations obtained from normal-mode-analysis-monitored energy minimization of protein-bound ligand conformations taken from the 100 crystal structures using the steepest descent minimization method with a gradient cut-off of 0.01 kcal/(mol•Å). All three translational frequencies were zero and are not listed.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0000820.s004" id="pone.0000820.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">The AMBER force field parameters developed for the 100 protein-bound ligands.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0000820.s005" id="pone.0000820.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s005">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">The AMBER atom types and RESP charges of the 100 protein-bound ligands. The atom labels are shown in <a href="#pone.0000820.s012">Figures S7</a>, <a href="#pone.0000820.s013">S8</a>, <a href="#pone.0000820.s014">S9</a>, <a href="#pone.0000820.s015">S10</a>, <a href="#pone.0000820.s016">S11</a>, <a href="#pone.0000820.s017">S12</a> and <a href="#pone.0000820.s018">S13</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.21 MB PDF)</p>
<a name="pone.0000820.s006" id="pone.0000820.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s006">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Perl script for the normal-mode-analysis-monitored energy minimization procedure.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.05 MB PDF)</p>
<a name="pone.0000820.s007" id="pone.0000820.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s007">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Chemical structures and protonation states of the 100 protein-bound ligands (Part I).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(6.16 MB TIF)</p>
<a name="pone.0000820.s008" id="pone.0000820.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s008">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Chemical structures and protonation states of the 100 protein-bound ligands (Part II).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(6.25 MB TIF)</p>
<a name="pone.0000820.s009" id="pone.0000820.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s009">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Chemical structures and protonation states of the 100 protein-bound ligands (Part III).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(6.26 MB TIF)</p>
<a name="pone.0000820.s010" id="pone.0000820.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s010">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Chemical structures and protonation states of the 100 protein-bound ligands (Part IV).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(6.21 MB TIF)</p>
<a name="pone.0000820.s011" id="pone.0000820.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s011">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Chemical structures and protonation states of the 100 protein-bound ligands (Part V).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(4.23 MB TIF)</p>
<a name="pone.0000820.s012" id="pone.0000820.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s012">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Definitions of atom labels of the 100 protein-bound ligands (Part I).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(6.46 MB TIF)</p>
<a name="pone.0000820.s013" id="pone.0000820.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s013">Figure S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Definitions of atom labels of the 100 protein-bound ligands (Part II).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s013</p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(6.28 MB TIF)</p>
<a name="pone.0000820.s014" id="pone.0000820.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s014">Figure S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1"></a><p class="preSiDOI">Definitions of atom labels of the 100 protein-bound ligands (Part III).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s014</p><a id="article1.body1.sec5.supplementary-material14.caption1.p2" name="article1.body1.sec5.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(6.47 MB TIF)</p>
<a name="pone.0000820.s015" id="pone.0000820.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s015">Figure S10. </a></strong></p><a id="article1.body1.sec5.supplementary-material15.caption1.p1" name="article1.body1.sec5.supplementary-material15.caption1.p1"></a><p class="preSiDOI">Definitions of atom labels of the 100 protein-bound ligands (Part IV).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s015</p><a id="article1.body1.sec5.supplementary-material15.caption1.p2" name="article1.body1.sec5.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(6.44 MB TIF)</p>
<a name="pone.0000820.s016" id="pone.0000820.s016"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s016">Figure S11. </a></strong></p><a id="article1.body1.sec5.supplementary-material16.caption1.p1" name="article1.body1.sec5.supplementary-material16.caption1.p1"></a><p class="preSiDOI">Definitions of atom labels of the 100 protein-bound ligands (Part V).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s016</p><a id="article1.body1.sec5.supplementary-material16.caption1.p2" name="article1.body1.sec5.supplementary-material16.caption1.p2"></a><p class="postSiDOI">(6.61 MB TIF)</p>
<a name="pone.0000820.s017" id="pone.0000820.s017"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s017">Figure S12. </a></strong></p><a id="article1.body1.sec5.supplementary-material17.caption1.p1" name="article1.body1.sec5.supplementary-material17.caption1.p1"></a><p class="preSiDOI">Definitions of atom labels of the 100 protein-bound ligands (Part VI).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s017</p><a id="article1.body1.sec5.supplementary-material17.caption1.p2" name="article1.body1.sec5.supplementary-material17.caption1.p2"></a><p class="postSiDOI">(6.55 MB TIF)</p>
<a name="pone.0000820.s018" id="pone.0000820.s018"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000820.s018">Figure S13. </a></strong></p><a id="article1.body1.sec5.supplementary-material18.caption1.p1" name="article1.body1.sec5.supplementary-material18.caption1.p1"></a><p class="preSiDOI">Definitions of atom labels of the 100 protein-bound ligands (Part VII).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000820.s018</p><a id="article1.body1.sec5.supplementary-material18.caption1.p2" name="article1.body1.sec5.supplementary-material18.caption1.p2"></a><p class="postSiDOI">(2.80 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Michael K. Gilson, Jason D. Thompson, Alfonso T Garcia-Sosa, and two anonymous reviewers for their insightful comments and suggestions. The authors also acknowledge computing support from the University of Minnesota Supercomputing Institute.</p>
<a id="article1.back1.ack1.p2" name="article1.back1.ack1.p2"></a><p>Disclaimer:</p>
<a id="article1.back1.ack1.p3" name="article1.back1.ack1.p3"></a><p>The opinions or assertions contained herein belong to the authors and are not necessarily the official views of the U.S. Army, the U.S. Department of Defense, or the National Institutes of Health.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YP. Performed the experiments: QW. Analyzed the data: YP QW. Wrote the paper: YP QW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000820-Koshland1" id="pone.0000820-Koshland1"></a>Koshland DE (1995) The key-lock theory and the induced fit theory. Angew Chem Int Ed Engl  33: 2375–2378.  <ul class="find" data-citedArticleID="985190" data-doi="10.1002/anie.199423751"><li><a href="http://dx.doi.org/10.1002/anie.199423751" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+key-lock+theory+and+the+induced+fit+theory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+key-lock+theory+and+the+induced+fit+theory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000820-Burgen1" id="pone.0000820-Burgen1"></a>Burgen AS (1981) Conformational changes and drug action. Fed Proc  40: 2723–2728.  <ul class="find" data-citedArticleID="985168"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Conformational+changes+and+drug+action.&amp;auth=&amp;atitle=Conformational+changes+and+drug+action." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+changes+and+drug+action." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+changes+and+drug+action.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000820-Bruns1" id="pone.0000820-Bruns1"></a>Bruns RF (1996) Conformational induction versus conformational selection: evidence from allosteric enhancers. Trends Pharmacol Sci  17: 189.  <ul class="find" data-citedArticleID="985166" data-doi="10.1016/0165-6147(96)20027-6"><li><a href="http://dx.doi.org/10.1016/0165-6147(96)20027-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+induction+versus+conformational+selection%3A+evidence+from+allosteric+enhancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+induction+versus+conformational+selection%3A+evidence+from+allosteric+enhancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000820-Kenakin1" id="pone.0000820-Kenakin1"></a>Kenakin T (1996) Receptor conformational induction versus selection: all part of the same energy landscape, agonists can differentially stabilize multiple active states of receptors. Trends Pharmacol Sci  17: 190–191.  <ul class="find" data-citedArticleID="985188" data-doi="10.1016/0165-6147(96)20026-4"><li><a href="http://dx.doi.org/10.1016/0165-6147(96)20026-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Receptor+conformational+induction+versus+selection%3A+all+part+of+the+same+energy+landscape%2C+agonists+can+differentially+stabilize+multiple+active+states+of+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Receptor+conformational+induction+versus+selection%3A+all+part+of+the+same+energy+landscape%2C+agonists+can+differentially+stabilize+multiple+active+states+of+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000820-Pang1" id="pone.0000820-Pang1"></a>Pang Y-P, Silva ND, Hydock C, Prendergast FG (1997) Docking studies on the complexed and uncomplexed FKBP12 structures with bound and unbound ligands: an implication of conformational selection mechanism for binding. J Mol Model  3: 240–248.  <ul class="find" data-citedArticleID="985194" data-doi="10.1007/s008940050036"><li><a href="http://dx.doi.org/10.1007/s008940050036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Docking+studies+on+the+complexed+and+uncomplexed+FKBP12+structures+with+bound+and+unbound+ligands%3A+an+implication+of+conformational+selection+mechanism+for+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Docking+studies+on+the+complexed+and+uncomplexed+FKBP12+structures+with+bound+and+unbound+ligands%3A+an+implication+of+conformational+selection+mechanism+for+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000820-Pang2" id="pone.0000820-Pang2"></a>Pang Y-P, Perola E, Xu K, Prendergast FG (2001) EUDOC: a computer program for identification of drug interaction sites in macromolecules and drug leads from chemical databases. J Comput Chem  22: 1750–1771.  <ul class="find" data-citedArticleID="985196" data-doi="10.1002/jcc.1129"><li><a href="http://dx.doi.org/10.1002/jcc.1129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EUDOC%3A+a+computer+program+for+identification+of+drug+interaction+sites+in+macromolecules+and+drug+leads+from+chemical+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EUDOC%3A+a+computer+program+for+identification+of+drug+interaction+sites+in+macromolecules+and+drug+leads+from+chemical+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000820-Pang3" id="pone.0000820-Pang3"></a>Pang Y-P (2007) In silico drug discovery: solving the “target-rich and lead-poor” imbalance using the genome-to-drug-lead paradigm. Clin Pharmacol Ther  81: 30–34.  <ul class="find" data-citedArticleID="985198"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+silico+drug+discovery%3A+solving+the+%26%23x201c%3Btarget-rich+and+lead-poor%26%23x201d%3B+imbalance+using+the+genome-to-drug-lead+paradigm.&amp;auth=&amp;atitle=In+silico+drug+discovery%3A+solving+the+%26%23x201c%3Btarget-rich+and+lead-poor%26%23x201d%3B+imbalance+using+the+genome-to-drug-lead+paradigm." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+silico+drug+discovery%3A+solving+the+%26%23x201c%3Btarget-rich+and+lead-poor%26%23x201d%3B+imbalance+using+the+genome-to-drug-lead+paradigm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+silico+drug+discovery%3A+solving+the+%26%23x201c%3Btarget-rich+and+lead-poor%26%23x201d%3B+imbalance+using+the+genome-to-drug-lead+paradigm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000820-Pang4" id="pone.0000820-Pang4"></a>Pang Y-P, Mullins T, Swartz B, McAllister J, Smith B, et al.  (2007) EUDOC on Blue Gene: accelerating the transfer of drug discoveries from laboratory to patient. IBM J Res Dev in press.   <ul class="find" data-citedArticleID="985200" data-doi="10.1147/rd.521.0069"><li><a href="http://dx.doi.org/10.1147/rd.521.0069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EUDOC+on+Blue+Gene%3A+accelerating+the+transfer+of+drug+discoveries+from+laboratory+to+patient." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EUDOC+on+Blue+Gene%3A+accelerating+the+transfer+of+drug+discoveries+from+laboratory+to+patient.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000820-Jorgensen1" id="pone.0000820-Jorgensen1"></a>Jorgensen WL (1991) Rusting of the lock and key model for protein-ligand binding. Science  254: 954–955.  <ul class="find" data-citedArticleID="985186" data-doi="10.1126/science.1719636"><li><a href="http://dx.doi.org/10.1126/science.1719636" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rusting+of+the+lock+and+key+model+for+protein-ligand+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rusting+of+the+lock+and+key+model+for+protein-ligand+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000820-Vieth1" id="pone.0000820-Vieth1"></a>Vieth M, Hirst JD, Brooks CL III (1998) Do active site conformations of small ligands correspond to low free-energy solution structures? J Comput-Aided Mol Design  12: 563–572.  <ul class="find" data-citedArticleID="985210"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Do+active+site+conformations+of+small+ligands+correspond+to+low+free-energy+solution+structures%3F&amp;auth=&amp;atitle=Do+active+site+conformations+of+small+ligands+correspond+to+low+free-energy+solution+structures%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Do+active+site+conformations+of+small+ligands+correspond+to+low+free-energy+solution+structures%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Do+active+site+conformations+of+small+ligands+correspond+to+low+free-energy+solution+structures%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000820-Bostrm1" id="pone.0000820-Bostrm1"></a>Boström J, Norrby P-O, Liljefors T (1998) Conformational energy penalties of protein-bound ligands. J Comput-Aided Mol Design  12: 383–396.  <ul class="find" data-citedArticleID="985164"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Conformational+energy+penalties+of+protein-bound+ligands.&amp;auth=&amp;atitle=Conformational+energy+penalties+of+protein-bound+ligands." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+energy+penalties+of+protein-bound+ligands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+energy+penalties+of+protein-bound+ligands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000820-Nicklaus1" id="pone.0000820-Nicklaus1"></a>Nicklaus MC, Wang S, Driscoll JS, Milne GW (1995) Conformational changes of small molecules binding to proteins. Bioorg Med Chem  3: 411–428.  <ul class="find" data-citedArticleID="985192" data-doi="10.1016/0968-0896(95)00031-b"><li><a href="http://dx.doi.org/10.1016/0968-0896(95)00031-b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+changes+of+small+molecules+binding+to+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+changes+of+small+molecules+binding+to+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000820-Perola1" id="pone.0000820-Perola1"></a>Perola E, Charifson PS (2004) Conformational analysis of drug-like molecules bound to proteins: an extensive study of ligand reorganization upon binding. J Med Chem  47: 2499–2510.  <ul class="find" data-citedArticleID="985206" data-doi="10.1021/jm030563w"><li><a href="http://dx.doi.org/10.1021/jm030563w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+analysis+of+drug-like+molecules+bound+to+proteins%3A+an+extensive+study+of+ligand+reorganization+upon+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+analysis+of+drug-like+molecules+bound+to+proteins%3A+an+extensive+study+of+ligand+reorganization+upon+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000820-Hinsen1" id="pone.0000820-Hinsen1"></a>Hinsen K (2006)  Normal mode theory and harmonic potential approximations;. In: Cui Q, Bahar I, editors. Boca Raton: Chapman &amp; Hall/CRC. pp. 1–16.  <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0000820-Case1" id="pone.0000820-Case1"></a>Case D (1999)  Rigidity theory and applications;. In: Thorpe M, Duxbury P, editors.  New York: Kluwer Academic/Plenum Publishers. pp. 329–344.  <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0000820-Jensen1" id="pone.0000820-Jensen1"></a>Jensen F (1998) Introduction to computational chemistry. New York: John Wiley &amp; Sons. pp. 312–315.  <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0000820-Cramer1" id="pone.0000820-Cramer1"></a>Cramer CJ (2002) Essentials of computational chemistry. New York: John Wiley &amp; Sons.   <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0000820-Cornell1" id="pone.0000820-Cornell1"></a>Cornell WD, Cieplak P, Bayly CI, Gould IR, Merz KM Jr, et al.  (1995) A second generation force field for the simulation of proteins, nucleic acids, and organic molecules. J Am Chem Soc  117: 5179–5197.  <ul class="find" data-citedArticleID="985174" data-doi="10.1021/ja00124a002"><li><a href="http://dx.doi.org/10.1021/ja00124a002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+second+generation+force+field+for+the+simulation+of+proteins%2C+nucleic+acids%2C+and+organic+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+second+generation+force+field+for+the+simulation+of+proteins%2C+nucleic+acids%2C+and+organic+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000820-Wang1" id="pone.0000820-Wang1"></a>Wang J, Wolf RM, Caldwell JW, Kollman PA, Case DA (2004) Development and testing of a general amber force field. J Comput Chem  25: 1157–1174.  <ul class="find" data-citedArticleID="985212" data-doi="10.1002/jcc.20035"><li><a href="http://dx.doi.org/10.1002/jcc.20035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+and+testing+of+a+general+amber+force+field." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+and+testing+of+a+general+amber+force+field.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000820-Pearlman1" id="pone.0000820-Pearlman1"></a>Pearlman DA, Case DA, Caldwell JW, Ross WS, Cheatham TE III, et al.  (1995) AMBER, a package of computer programs for applying molecular mechanics, normal mode analysis, molecular dynamics and free energy calculations to simulate the structural and energetic properties of molecules. Comput Phys Commun  91: 1–41.  <ul class="find" data-citedArticleID="985204" data-doi="10.1016/0010-4655(95)00041-d"><li><a href="http://dx.doi.org/10.1016/0010-4655(95)00041-d" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AMBER%2C+a+package+of+computer+programs+for+applying+molecular+mechanics%2C+normal+mode+analysis%2C+molecular+dynamics+and+free+energy+calculations+to+simulate+the+structural+and+energetic+properties+of+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AMBER%2C+a+package+of+computer+programs+for+applying+molecular+mechanics%2C+normal+mode+analysis%2C+molecular+dynamics+and+free+energy+calculations+to+simulate+the+structural+and+energetic+properties+of+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000820-Pornillos1" id="pone.0000820-Pornillos1"></a>Pornillos O, Alam SL, Davis DR, Sundquist WI (2002) Structure of the Tsg101 UEV domain in complex with the PTAP motif of the HIV-1 p6 protein. Nat Struct Biol  9: 812–817.  <ul class="find" data-citedArticleID="985208" data-doi="10.1038/nsb856"><li><a href="http://dx.doi.org/10.1038/nsb856" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+the+Tsg101+UEV+domain+in+complex+with+the+PTAP+motif+of+the+HIV-1+p6+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+the+Tsg101+UEV+domain+in+complex+with+the+PTAP+motif+of+the+HIV-1+p6+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000820-Dooley1" id="pone.0000820-Dooley1"></a>Dooley AJ, Shindo N, Taggart B, Park JG, Pang Y-P (2006) From genome to drug lead: identification of a small-molecule inhibitor of the SARS virus. Bioorg Med Chem Lett  16: 830–833.  <ul class="find" data-citedArticleID="985178" data-doi="10.1016/j.bmcl.2005.11.018"><li><a href="http://dx.doi.org/10.1016/j.bmcl.2005.11.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genome+to+drug+lead%3A+identification+of+a+small-molecule+inhibitor+of+the+SARS+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genome+to+drug+lead%3A+identification+of+a+small-molecule+inhibitor+of+the+SARS+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000820-Pang5" id="pone.0000820-Pang5"></a>Pang Y-P (2004) Three-dimensional model of a substrate-bound SARS chymotrypsin-like cysteine proteinase predicted by multiple molecular dynamics simulations: catalytic efficiency regulated by substrate binding. Proteins  57: 747–757.  <ul class="find" data-citedArticleID="985202" data-doi="10.1002/prot.20249"><li><a href="http://dx.doi.org/10.1002/prot.20249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three-dimensional+model+of+a+substrate-bound+SARS+chymotrypsin-like+cysteine+proteinase+predicted+by+multiple+molecular+dynamics+simulations%3A+catalytic+efficiency+regulated+by+substrate+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three-dimensional+model+of+a+substrate-bound+SARS+chymotrypsin-like+cysteine+proteinase+predicted+by+multiple+molecular+dynamics+simulations%3A+catalytic+efficiency+regulated+by+substrate+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000820-Wang2" id="pone.0000820-Wang2"></a>Wang Q, Pang Y-P (2007) Accurate reproduction of 161 small-molecule complex crystal structures using the EUDOC program: expanding the use of EUDOC to supramolecular chemistry. PLoS ONE  2: e531.  <ul class="find" data-citedArticleID="985214" data-doi="10.1371/journal.pone.0000531"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000531" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+reproduction+of+161+small-molecule+complex+crystal+structures+using+the+EUDOC+program%3A+expanding+the+use+of+EUDOC+to+supramolecular+chemistry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+reproduction+of+161+small-molecule+complex+crystal+structures+using+the+EUDOC+program%3A+expanding+the+use+of+EUDOC+to+supramolecular+chemistry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000820-Berman1" id="pone.0000820-Berman1"></a>Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, et al.  (2000) The Protein Data Bank. Nucleic Acids Res  28: 235–242.  <ul class="find" data-citedArticleID="985162" data-doi="10.1093/nar/28.1.235"><li><a href="http://dx.doi.org/10.1093/nar/28.1.235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Protein+Data+Bank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Protein+Data+Bank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000820-Cieplak1" id="pone.0000820-Cieplak1"></a>Cieplak P, Cornell WD, Bayly C, Kollman PA (1995) Application of the multimolecule and multiconformational RESP methodology to biopolymers: Charge derivation for DNA, RNA, and proteins. J Comput Chem  16: 1357–1377.  <ul class="find" data-citedArticleID="985172" data-doi="10.1002/jcc.540161106"><li><a href="http://dx.doi.org/10.1002/jcc.540161106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+the+multimolecule+and+multiconformational+RESP+methodology+to+biopolymers%3A+Charge+derivation+for+DNA%2C+RNA%2C+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+the+multimolecule+and+multiconformational+RESP+methodology+to+biopolymers%3A+Charge+derivation+for+DNA%2C+RNA%2C+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000820-Frisch1" id="pone.0000820-Frisch1"></a>Frisch MJ, Trucks GW, Schlegel HB, Gill PMW, Hohnson BG, et al.  (1999) GAUSSIAN 98, Revision A.7. Pittsburgh, , PA: Gaussian, Inc.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.XML" value="98943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.PDF" value="310125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g001.PNG_L" value="525200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g001.PNG_M" value="99376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g001.PNG_S" value="9494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g001.TIF" value="936776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g001.PNG_I" value="39233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g002.PNG_L" value="32849"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g002.PNG_M" value="36499"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g002.PNG_S" value="6428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g002.TIF" value="46324"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g002.PNG_I" value="12211"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g003.PNG_L" value="530081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g003.PNG_M" value="99588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g003.PNG_S" value="9333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g003.TIF" value="895306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g003.PNG_I" value="35114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t001.PNG_L" value="272957"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t001.PNG_M" value="162100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t001.PNG_S" value="18318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t001.TIF" value="1250930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t001.PNG_I" value="50610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g004.PNG_L" value="68766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g004.PNG_M" value="18198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g004.PNG_S" value="5375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g004.TIF" value="137918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g004.PNG_I" value="6398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g005.PNG_L" value="66702"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g005.PNG_M" value="17373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g005.PNG_S" value="4970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g005.TIF" value="127264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g005.PNG_I" value="6468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t002.PNG_L" value="143124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t002.PNG_M" value="82587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t002.PNG_S" value="10555"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t002.TIF" value="640300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t002.PNG_I" value="25990"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g006.PNG_L" value="780987"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g006.PNG_M" value="137705"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g006.PNG_S" value="10717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g006.TIF" value="1164132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g006.PNG_I" value="46685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g007.PNG_L" value="639131"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g007.PNG_M" value="116474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g007.PNG_S" value="10174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g007.TIF" value="1013110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.g007.PNG_I" value="39712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t003.PNG_L" value="66730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t003.PNG_M" value="86172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t003.PNG_S" value="12917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t003.TIF" value="278294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.t003.PNG_I" value="30017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.e001.PNG" value="12967"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.e001.TIF" value="28344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.e002.PNG" value="9514"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.e002.TIF" value="25456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s001.PDF" value="40869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s002.PDF" value="41087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s003.PDF" value="38173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s004.PDF" value="41392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s005.PDF" value="211419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s006.PDF" value="45023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s007.TIF" value="774361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s008.TIF" value="785070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s009.TIF" value="786117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s010.TIF" value="781148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s011.TIF" value="531291"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s012.TIF" value="809307"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s013.TIF" value="788940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s014.TIF" value="812817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s015.TIF" value="809943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s016.TIF" value="829752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s017.TIF" value="822347"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000820.s018.TIF" value="352422"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000820&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000820" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000820&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000820&volume=&issue=&title=Preference of Small Molecules for Local Minimum Conformations when Binding to Proteins&author_name=Qi%20Wang%2C%20Yuan-Ping%20Pang&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000820" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000820" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000820" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000820&amp;t=Preference%20of%20Small%20Molecules%20for%20Local%20Minimum%20Conformations%20when%20Binding%20to%20Proteins" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000820&title=Preference%20of%20Small%20Molecules%20for%20Local%20Minimum%20Conformations%20when%20Binding%20to%20Proteins&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000820&amp;title=Preference%20of%20Small%20Molecules%20for%20Local%20Minimum%20Conformations%20when%20Binding%20to%20Proteins" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000820" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000820&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Preference of Small Molecules for Local Minimum Conformations when Binding to Proteins';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000820';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Preference%20of%20Small%20Molecules%20for%20Local%20Minimum%20Conformations%20when%20Binding%20to%20Proteins http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000820" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000820" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">









          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Atoms%22" title="Search for articles in the subject area:'Atoms'"><div class="flagText">Atoms</div></a>
              <div data-categoryid="24379" data-articleid="24478"
                   data-categoryname="Atoms"
                   class="flagImage" title="Flag 'Atoms' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Computer+software%22" title="Search for articles in the subject area:'Computer software'"><div class="flagText">Computer software</div></a>
              <div data-categoryid="2287" data-articleid="24478"
                   data-categoryname="Computer software"
                   class="flagImage" title="Flag 'Computer software' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="24478"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Potential+energy%22" title="Search for articles in the subject area:'Potential energy'"><div class="flagText">Potential energy</div></a>
              <div data-categoryid="23753" data-articleid="24478"
                   data-categoryname="Potential energy"
                   class="flagImage" title="Flag 'Potential energy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interactions%22" title="Search for articles in the subject area:'Protein interactions'"><div class="flagText">Protein interactions</div></a>
              <div data-categoryid="17661" data-articleid="24478"
                   data-categoryname="Protein interactions"
                   class="flagImage" title="Flag 'Protein interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure%22" title="Search for articles in the subject area:'Protein structure'"><div class="flagText">Protein structure</div></a>
              <div data-categoryid="31877" data-articleid="24478"
                   data-categoryname="Protein structure"
                   class="flagImage" title="Flag 'Protein structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transition+state%22" title="Search for articles in the subject area:'Transition state'"><div class="flagText">Transition state</div></a>
              <div data-categoryid="21819" data-articleid="24478"
                   data-categoryname="Transition state"
                   class="flagImage" title="Flag 'Transition state' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Vibration%22" title="Search for articles in the subject area:'Vibration'"><div class="flagText">Vibration</div></a>
              <div data-categoryid="20931" data-articleid="24478"
                   data-categoryname="Vibration"
                   class="flagImage" title="Flag 'Vibration' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6409'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4558'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5324&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
