

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Accurate Reproduction of 161 Small-Molecule Complex Crystal Structures using the EUDOC Program: Expanding the Use of EUDOC to Supramolecular Chemistry</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000531"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000531" />

    <meta name="citation_title" content="Accurate Reproduction of 161 Small-Molecule Complex Crystal Structures using the EUDOC Program: Expanding the Use of EUDOC to Supramolecular Chemistry"/>
    <meta itemprop="name" content="Accurate Reproduction of 161 Small-Molecule Complex Crystal Structures using the EUDOC Program: Expanding the Use of EUDOC to Supramolecular Chemistry"/>

      <meta name="citation_author" content="Qi Wang"/>
            <meta name="citation_author_institution" content="Computer-Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America"/>
      <meta name="citation_author" content="Yuan-Ping Pang"/>
            <meta name="citation_author_institution" content="Computer-Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America"/>

    <meta name="citation_date" content="2007/6/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000531.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e531"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The Cambridge Structural Database: a quarter of a million crystal structures and rising.; citation_author=FH Allen; citation_journal_title=Acta Crystallogr Sect B: Struct Sci; citation_volume=58; citation_number=1; citation_pages=380-388; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structure-based design of nonpeptide inhibitors specific for the human immunodeficiency virus 1 protease.; citation_author=RL DesJarlais; citation_author=GL Seibel; citation_author=ID Kuntz; citation_author=PS Furth; citation_author=JC Alvarez; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=87; citation_number=2; citation_pages=6644-6648; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=EUDOC: A computer program for identification of drug interaction sites in macromolecules and drug leads from chemical databases.; citation_author=Y-P Pang; citation_author=E Perola; citation_author=K Xu; citation_author=FG Prendergast; citation_journal_title=J Comput Chem; citation_volume=22; citation_number=3; citation_pages=1750-1771; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=In silico drug discovery: solving the “target-rich and lead-poor” imbalance using the genome-to-drug-lead paradigm.; citation_author=YP Pang; citation_journal_title=Clin Pharmacol Ther; citation_volume=81; citation_number=4; citation_pages=30-34; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=EUDOC on Blue Gene: accelerating the transfer of drug discoveries from laboratory to patient.; citation_author=Y-P Pang; citation_author=T Mullins; citation_author=B Swartz; citation_author=J McAllister; citation_author=B Smith; citation_journal_title=IBM J Res Dev; citation_volume=in press; citation_number=5; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Successful virtual screening of a chemical database for farnesyltransferase inhibitor leads.; citation_author=E Perola; citation_author=K Xu; citation_author=TM Kollmeyer; citation_author=SH Kaufmann; citation_author=FG Prendergast; citation_journal_title=J Med Chem; citation_volume=43; citation_number=6; citation_pages=401-408; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Chemical database techniques in drug discovery.; citation_author=MA Miller; citation_journal_title=Nat Rev Drug Discov; citation_volume=1; citation_number=7; citation_pages=220-227; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structure of acetylcholinesterase complexed with the nootropic alkaloid, (−)-huperzine A.; citation_author=ML Raves; citation_author=M Harel; citation_author=YP Pang; citation_author=I Silman; citation_author=AP Kozikowski; citation_journal_title=Nat Struct Biol; citation_volume=4; citation_number=8; citation_pages=57-63; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Discovery of a new inhibitor lead of adenovirus proteinase: steps toward selective, irreversible inhibitors of cysteine proteinases.; citation_author=Y-P Pang; citation_author=K Xu; citation_author=TM Kollmeyer; citation_author=E Perola; citation_author=WJ McGrath; citation_journal_title=FEBS Lett; citation_volume=502; citation_number=9; citation_pages=93-97; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Re-engineering butyrylcholinesterase as a cocaine hydrolase.; citation_author=H Sun; citation_author=YP Pang; citation_author=O Lockridge; citation_author=S Brimijoin; citation_journal_title=Mol Pharmacol; citation_volume=62; citation_number=10; citation_pages=220-224; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=From genome to drug lead: identification of a small-molecule inhibitor of the SARS virus.; citation_author=AJ Dooley; citation_author=N Shindo; citation_author=B Taggart; citation_author=JG Park; citation_author=Y-P Pang; citation_journal_title=Bioorg Med Chem Lett; citation_volume=16; citation_number=11; citation_pages=830-833; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Serotype-selective, small-molecule inhibitors of the zinc endopeptidase of botulinum neurotoxin serotype A.; citation_author=JG Park; citation_author=PC Sill; citation_author=EF Makiyi; citation_author=AT Garcia-Sosa; citation_author=CB Millard; citation_journal_title=Bioorg Med Chem; citation_volume=14; citation_number=12; citation_pages=395-408; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Supramolecular chemistry.; citation_author=JM Lehn; citation_journal_title=Science; citation_volume=260; citation_number=13; citation_pages=1762-1763; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The limit of accuracy of protein modeling: influence of crystal packing on protein structure.; citation_author=E Eyal; citation_author=S Gerzon; citation_author=V Potapov; citation_author=M Edelman; citation_author=V Sobolev; citation_journal_title=J Mol Biol; citation_volume=351; citation_number=14; citation_pages=431-442; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular-sructure of Rebek diacid quinoxaline - confirmation of 2-point binding.; citation_author=RA Pascal; citation_author=DM Ho; citation_journal_title=J Am Chem Soc; citation_volume=115; citation_number=15; citation_pages=8507-8508; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Structure and binding for complexes of Rebek's acridine diacid with pyrazine, quinoxaline, and pyridine from Monte Carlo simulations with an all-atom force field.; citation_author=EM Duffy; citation_author=WL Jorgensen; citation_journal_title=J Am Chem Soc; citation_volume=116; citation_number=16; citation_pages=6337-6343; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A second generation force field for the simulation of proteins, nucleic acids, and organic molecules.; citation_author=WD Cornell; citation_author=P Cieplak; citation_author=CI Bayly; citation_author=IR Gould; citation_author=KM Merz; citation_journal_title=J Am Chem Soc; citation_volume=117; citation_number=17; citation_pages=5179-5197; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Molecular recognition: Ionic and aromatic stacking interactions bind complementary functional groups in a molecular cleft.; citation_author=J Rebek; citation_author=D Nemeth; citation_journal_title=J Am Chem Soc; citation_volume=108; citation_number=18; citation_pages=5637-5638; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=AMBER, a package of computer programs for applying molecular mechanics, normal mode analysis, molecular dynamics and free energy calculations to simulate the structural and energetic properties of molecules.; citation_author=DA Pearlman; citation_author=DA Case; citation_author=JW Caldwell; citation_author=WS Ross; citation_author=TE Cheatham; citation_journal_title=Comput Phys Commun; citation_volume=91; citation_number=19; citation_pages=1-41; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Application of the multimolecule and multiconformational RESP methodology to biopolymers: Charge derivation for DNA, RNA, and proteins.; citation_author=P Cieplak; citation_author=WD Cornell; citation_author=C Bayly; citation_author=PA Kollman; citation_journal_title=J Comput Chem; citation_volume=16; citation_number=20; citation_pages=1357-1377; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Gaussian 03, Revision C.02. Gaussian, Inc, Wallingford CT.; citation_author=MJ Frisch; citation_author=GW Trucks; citation_author=HB Schlegel; citation_author=GE Scuseria; citation_author=MA Robb; citation_number=21; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000531" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Accurate Reproduction of 161 Small-Molecule Complex Crystal Structures using the EUDOC Program: Expanding the Use of EUDOC to Supramolecular Chemistry"/>
    <meta name="twitter:description" content="EUDOC is a docking program that has successfully predicted small-molecule-bound protein complexes and identified drug leads from chemical databases. To expand the application of the EUDOC program to supramolecular chemistry, we tested its ability to reproduce crystal structures of small-molecule complexes. Of 161 selected crystal structures of small-molecule guest-host complexes, EUDOC reproduced all these crystal structures with guest structure mass-weighted root mean square deviations (mwRMSDs) of &lt;1.0 &Aring; relative to the corresponding crystal structures. In addition, the average interaction energy of these 161 guest-host complexes (&minus;50.1 kcal/mol) was found to be nearly half of that of 153 previously tested small-molecule-bound protein complexes (&minus;108.5 kcal/mol), according to the interaction energies calculated by EUDOC. 31 of the 161 complexes could not be reproduced with mwRMSDs of &lt;1.0 &Aring; if neighboring hosts in the crystal structure of a guest-host complex were not included as part of the multimeric host system, whereas two of the 161 complexes could not be reproduced with mwRMSDs of &lt;1.0 &Aring; if water molecules were excluded from the host system. These results demonstrate the significant influence of crystal packing on small molecule complexation and suggest that EUDOC is able to predict small-molecule complexes and that it is useful for the design of new materials, molecular sensors, and multimeric inhibitors of protein-protein interactions."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000531.g002"/>

  <meta property="og:title" content="Accurate Reproduction of 161 Small-Molecule Complex Crystal Structures using the EUDOC Program: Expanding the Use of EUDOC to Supramolecular Chemistry" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000531" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8262'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3679'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6303&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000531">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000531" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000531&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Accurate Reproduction of 161 Small-Molecule Complex Crystal Structures using the EUDOC Program: Expanding the Use of EUDOC to Supramolecular Chemistry
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Qi Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computer-Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yuan-Ping Pang
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pang@mayo.edu">pang@mayo.edu</a></p>

                <p>Affiliation:
                  Computer-Aided Molecular Design Laboratory, Mayo Clinic, Rochester, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 13, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000531</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000531" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000531" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000531" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000531" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000531-g001" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000531-t001" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000531-t002" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000531-g002" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000531-t003" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000531-t004" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000531">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000531" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>EUDOC is a docking program that has successfully predicted small-molecule-bound protein complexes and identified drug leads from chemical databases. To expand the application of the EUDOC program to supramolecular chemistry, we tested its ability to reproduce crystal structures of small-molecule complexes. Of 161 selected crystal structures of small-molecule guest-host complexes, EUDOC reproduced all these crystal structures with guest structure mass-weighted root mean square deviations (mwRMSDs) of &lt;1.0 Å relative to the corresponding crystal structures. In addition, the average interaction energy of these 161 guest-host complexes (−50.1 kcal/mol) was found to be nearly half of that of 153 previously tested small-molecule-bound protein complexes (−108.5 kcal/mol), according to the interaction energies calculated by EUDOC. 31 of the 161 complexes could not be reproduced with mwRMSDs of &lt;1.0 Å if neighboring hosts in the crystal structure of a guest-host complex were not included as part of the multimeric host system, whereas two of the 161 complexes could not be reproduced with mwRMSDs of &lt;1.0 Å if water molecules were excluded from the host system. These results demonstrate the significant influence of crystal packing on small molecule complexation and suggest that EUDOC is able to predict small-molecule complexes and that it is useful for the design of new materials, molecular sensors, and multimeric inhibitors of protein-protein interactions.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wang Q, Pang Y-P (2007) Accurate Reproduction of 161 Small-Molecule Complex Crystal Structures using the EUDOC Program: Expanding the Use of EUDOC to Supramolecular Chemistry. PLoS ONE 2(6):
          e531.
            doi:10.1371/journal.pone.0000531</p><p><strong>Academic Editor: </strong>Mark Isalan, Center for Genomic Regulation, Spain</p><p><strong>Received:</strong> May 7, 2007; <strong>Accepted:</strong> May 21, 2007; <strong>Published:</strong> June 13, 2007</p><p><strong>Copyright:</strong> © 2007 Wang, Pang. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the U.S. Army Medical Research Acquisition Activity (W81XWH-04-2-0001), the National Institutes of Health (5R01AI054574-03), Mayo Graduate School, and the Mayo Foundation for Medical Education and Research.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>In 1990, a computer was used to screen 10,000 chemicals in the Cambridge Structural Database (CSD) <a href="#pone.0000531-Allen1">[1]</a>, leading to the identification of a haloperidol analog capable of inhibiting HIV-1 and HIV-2 proteases with a <em>K</em><sub>i</sub> of ≈100 µM <a href="#pone.0000531-DesJarlais1">[2]</a>. The screening was accomplished using a computer docking program, DOCK, that docked each chemical of the database into the active sites of the enzymes and evaluated the shape complementarity of the docked compound relative to the active sites. Inspired by this seminal work, the EUDOC program was devised to search for the specific conformations, positions, and orientations of two three-dimensional (3D) structures that permit the strongest nonbonded intermolecular interactions between the two. The EUDOC program uses docking algorithms that differ from those of DOCK <a href="#pone.0000531-Pang1">[3]</a>. It addresses molecular flexibility by using conformation selection and conformation substitution mechanisms that enable massively parallel computing <a href="#pone.0000531-Pang1">[3]</a>. EUDOC was devised to perform on a cluster of more than 300 loosely connected processors <a href="#pone.0000531-Pang1">[3]</a> and has recently been ported to the IBM Blue Gene/L supercomputer <a href="#pone.0000531-Pang2">[4]</a>, <a href="#pone.0000531-Pang3">[5]</a>. This program has successfully predicted small-molecule-bound protein complexes and identified drug leads from chemical databases <a href="#pone.0000531-Perola1">[6]</a>–<a href="#pone.0000531-Park1">[12]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The EUDOC program is also efficient. In a computational screen of 23,426 chemicals (at a resolution of 1.0 Å translation and 10° of arc rotation) for inhibitors of a chymotrypsin-like cysteine protease of the severe acute respiratory syndrome–associated coronavirus, the EUDOC program is able to reduce the wall-clock time of the screen from 242 minutes using 396 Xeon processors (2.2 GHz) on a Beowulf cluster to 13 and 7 minutes using 2048 and 4096 PowerPC-440 processors (700 MHz) on Blue Gene/L, respectively <a href="#pone.0000531-Pang2">[4]</a>, <a href="#pone.0000531-Pang3">[5]</a>. Because a large database can be divided into subsets, a sustained petaflops capability would be able to screen 23 million chemicals in about 10 minutes or to screen 200×5000 billion chemicals for one drug target in a year <a href="#pone.0000531-Pang2">[4]</a>. This capability offers the possibility of identifying inhibitors that are effective enough for in vivo testing, eliminating the need of medicinal chemistry to improve the efficiency of inhibitor leads identified by terascale computers <a href="#pone.0000531-Pang2">[4]</a>. In the context of this promise, we seek to extend the application of the EUDOC program to supramolecular chemistry.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Supramolecular chemistry deals with creation of a large molecule assembled with noncovalent bonding among small molecular units, in contrast to organic synthesis that involves breaking and making covalent bonds to create a new molecule <a href="#pone.0000531-Lehn1">[13]</a>. Such noncovalent bonding is reversible and comprises hydrogen bonding, metal coordination, hydrophobic force, van der Waals force, π-π interaction, cation-π interaction, and/or long-range electrostatic interaction to assemble small molecules into a multimolecular complex. Supramolecular chemistry principles have been used to develop new materials, molecular sensors, and multimolecular complexes designed to disrupt protein-protein interactions.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>To expand the application of the EUDOC program to supramolecular chemistry, we tested its ability to reproduce the crystal structures of small-molecule guest-host complexes. Previously we had tested the ability of the program to reproduce crystal structures of proteins in complex with small molecules and found that EUDOC reproduced 97% of 154 crystal structures using the bound conformations of both proteins and their small-molecule partners <a href="#pone.0000531-Pang1">[3]</a>. This success may not transfer to with small-molecule guest-host complexes such as a crown ether in complex with 4-nitrobenzene-1,2-diamine, however, because the binding pocket or cavity in a small-molecule host is not as well formed as that in a protein.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Herein we report the results of our docking studies with 161 selected crystal structures of small-molecule guest-host complexes using the EUDOC program. These results show that the program is able to reproduce all 161 crystal structures and that the average interaction energy of these small-molecule complexes (−50.1 kcal/mol) is nearly half of that of the 153 small molecule-bound protein complexes we studied in previous tests (−108.5 kcal/mol). The results also demonstrate the significant influence of crystal packing on small-molecule complex crystal structures and suggest that the EUDOC program is able to predict 3D structures of small-molecule guest-host complexes with reasonable reliability.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Docking without consideration of the influence of crystal packing or structural waters</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The ability of EUDOC to reproduce crystal structures of small-molecule guest-host complexes was evaluated with the following procedure. The guest and host molecules in the complex crystal structure were separated, and the guest structure was then docked back into the host structure by the EUDOC program. This docking process used translational and rotational increments of 1.0 Å and 10° of arc, respectively, and a docking box that was defined to enclose the guest structure in the guest-host complex crystal structure. Of many EUDOC-generated guest-host complexes, only the complex with the strongest interaction energy was compared to the corresponding crystal structure of the complex. In this comparison, the host portion of the EUDOC-generated complex was superimposed onto the host portion of the crystal structure, and the mass-weighted root mean square deviation (mwRMSD) of the guest portion between the two superimposed complexes was calculated. If the mwRMSD was &lt;2.0 or 1.0 Å, the crystal structure of the complex was reproduced or accurately reproduced, respectively, by the EUDOC program <a href="#pone.0000531-Pang1">[3]</a>. Because the uncertainty in calculating the interaction energy using the EUDOC program was estimated to be 0.7 kcal/mol <a href="#pone.0000531-Pang1">[3]</a>, occasionally, a few EUDOC-generated complexes were considered to have the strongest interaction energy and compared to the crystal structure thus resulting in multiple mwRMSDs, if their interaction energies differed from the strongest interaction energy by ≤0.7 kcal/mol. In that case, as long as one of the mwRMSDs was &lt;2.0 or 1.0 Å, the crystal structure of the binary complex was reproduced or accurately reproduced, respectively, by the EUDOC program.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>A total of 161 crystal structures of small-molecule guest-host complexes were obtained from CSD for this study <a href="#pone.0000531-Allen1">[1]</a>. The selection criteria included the followings: (1) no covalent bond between a host and a guest; (2) the R factor of &lt;15 to ensure good crystallographic quality; (3) a guest to host ratio of 1 in a unit cell; (4) no structures containing Ni<sup>+2</sup>, Ag<sup>+</sup>, Pd<sup>+2</sup>, Pt<sup>+2</sup>, Au<sup>+</sup> or Ru<sup>+2</sup> because force field parameters for these ions were unavailable in the EUDOC program. The results of the docking studies with the 161 guest-host complexes using the procedure described above are listed in <a href="#pone-0000531-t001">Table 1</a>. As apparent from the mwRMSD distribution listed in <a href="#pone-0000531-t002">Table 2</a>, the EUDOC program reproduced 93% and accurately reproduced 81% of the 161 complexes. The deviations between the EUDOC-generated and crystal complexes at different mwRMSD values are depicted in <a href="#pone-0000531-g001">Figure 1</a>.</p>
<div class="figure" id="pone-0000531-g001"><div class="img"><a name="pone-0000531-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000531.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000531.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Overlays of the EUDOC-generated and crystal structures of complexes ATUKEF, CICVUF, XUGPUK, and NEXLUX.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The C atoms of the EUDOC-generated and crystal structure are green and magenta, respectively. The O, N, and S atoms are red, blue and orange respectively.</p>
<span>doi:10.1371/journal.pone.0000531.g001</span></div><div class="figure" id="pone-0000531-t001"><div class="img"><a name="pone-0000531-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000531.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000531.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>The intermolecular interaction energies of the 161 small-molecule complexes generated by the EUDOC program and their mass-weighted root mean square deviations (mwRMSDs) relative to their corresponding crystal structures.</span></strong></p><span>doi:10.1371/journal.pone.0000531.t001</span></div><div class="figure" id="pone-0000531-t002"><div class="img"><a name="pone-0000531-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000531.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000531.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>The Number and percentage of the EUDOC-generated complexes distributed in four categories of the mass-weighted root mean square deviations (mwRMSDs).</span></strong></p><span>doi:10.1371/journal.pone.0000531.t002</span></div>

<h4>Docking with consideration of the influence of crystal packing</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p><a href="#pone-0000531-g002">Figure 2</a> shows the difference (mwRMSD of 3.52 Å) between the EUDOC-generated and crystal structures (CSD code: XAGMAT). Complex XAGMAT is one of the 12 complexes that the EUDOC program failed to reproduce. Despite a favorable π-π interaction between the guest and host structures predicted by the EUDOC program, in the corresponding crystal structure the guest structure surprisingly docks at a region at which it partly interacts with the host via a π-π interaction (see <a href="#pone-0000531-g002">Fig. 2</a>). This discrepancy suggests that the guest might partly interact with host(s) and/or guest(s) in neighboring unit cells of the crystal structure. To confirm this, the docking study with complex XAGMAT was repeated with consideration of the influence of crystal packing—namely, the guest was docked into a multimeric host system that included neighboring host(s) and/or guest(s). These neighboring structures were generated by applying the symmetry of the space group of the crystal structure. The host(s) and/or guest(s) in neighboring unit cells were excluded if these structures were &gt;4.0 Å away from the guest to be docked. Interestingly, when the influence of crystal packing was taken into account, the EUDOC program accurately reproduced complex XAGMAT with an mwRMSD of 0.07 Å, instead of the 3.52 Å obtained without consideration of crystal packing. This result prompted a new docking study that considered the influence of crystal packing.</p>
<div class="figure" id="pone-0000531-g002"><div class="img"><a name="pone-0000531-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000531.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000531.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Overlays of the EUDOC-generated and crystal structures of complex XAGMAT.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Left: Docking without consideration of the influence of crystal packing. Right: Docking with consideration of the influence of crystal packing. The C atoms of the EUDOC-generated and crystal structure are red and green, respectively. The N atom is blue.</p>
<span>doi:10.1371/journal.pone.0000531.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The results of the docking studies with consideration of the influence of crystal packing are listed in <a href="#pone-0000531-t003">Table 3</a>. The 12 complexes (CSD codes: AJUXUY, ATUKEF, BAXZAB, BIFKIK, CRAMCC10, FANJAG, GUGGUK, KOLMAZ, LAYMAZ, NEBQOA, QAJKAN, and RALQAW01) that were not reproduced previously by the EUDOC program were accurately reproduced after the influence of crystal packing was taken into account. With consideration of the influence of crystal packing, the EUDOC program reproduced all 161 complexes and accurately reproduced 99% of them (see <a href="#pone-0000531-t002">Table 2</a>).</p>
<div class="figure" id="pone-0000531-t003"><div class="img"><a name="pone-0000531-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000531.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000531.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>The intermolecular interaction energies of the 161 small-molecule complexes generated by the EUDOC program and their mass-weighted root mean square deviations (mwRMSDs) relative to their corresponding crystal structures.</span></strong></p><span>doi:10.1371/journal.pone.0000531.t003</span></div>

<h4>Docking with consideration of the influences of crystal packing and structural waters</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Two crystal structures (CSD codes: XAQJAA and XAQJEE) could not be accurately reproduced by the EUDOC program even after consideration of the influence of crystal packing (XAQJAA: mwRMSD = 1.65 Å; XAQJEE: mwRMSD = 1.89 Å). Visual inspection of these structures revealed that the binding between the guest and host structures was mediated by crystallographically determined water molecules. This mediation suggested that, similar to the crystal packing, water molecules might also play an important role in guest-host complexation, and it might be necessary to include them in the multimeric host system for docking. Accordingly, the docking studies with the 161 complexes were repeated with consideration of the influences of both crystal packing and structural waters. The results are listed in <a href="#pone-0000531-t004">Table 4</a>. Indeed, the EUDOC program accurately reproduced complexes XAQJAA and XAQJEE with mwRMSDs of 0.03 and 0.27 Å, respectively. Taking into account the influences of both crystal packing and structural waters, the EUDOC program accurately reproduced all 161 complexes (see <a href="#pone-0000531-t002">Table 2</a>).</p>
<div class="figure" id="pone-0000531-t004"><div class="img"><a name="pone-0000531-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000531" data-uri="info:doi/10.1371/journal.pone.0000531.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000531.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000531.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000531.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000531.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>The intermolecular interaction energies of the 161 small-molecule complexes generated by the EUDOC program and their mass-weighted root mean square deviations (mwRMSDs) relative to their corresponding crystal structures.</span></strong></p><span>doi:10.1371/journal.pone.0000531.t004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>The influences of crystal packing and structural water on docking</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>This study shows that 31 (19%) of the 161 complexes could not be accurately reproduced with mwRMSDs of &lt;1.0 Å by the EUDOC program if neighboring host(s) and/or guest(s) in the crystal structure were not included as part of the multimeric host system, whereas only 2 (1%) of these complexes could not be accurately reproduced with mwRMSDs of &lt;1.0 Å if neighboring structures were included but water molecules were excluded from the host system. These results show that the influence of crystal packing or crystal environment on crystal structures of guest-host complexes is significant, which is consistent with the reported influence of crystal packing on protein structures <a href="#pone.0000531-Eyal1">[14]</a>. These results also show that the influence of structural waters on guest–host complex crystal structures is insignificant, which is consistent with our reported finding that complexation between a small molecule and a protein is not commonly mediated by water molecules <a href="#pone.0000531-Pang1">[3]</a>. This study therefore suggests that crystal packing should be taken into account when reproducing crystal structures of small-molecule guest-host complexes through docking studies, whereas water molecules, counterions or other companying molecules such as ethanol can be excluded from the host system. This study also suggests that, <em>ideally</em>, to perform prospective and accurate docking of a small molecule into another small molecule, repetitive docking of a guest or host into a guest-host complex that is generated by the previous docking is preferred, because a host is sometimes too small to prevent a guest from interacting with nearby guest(s) and/or host(s). It is worth noting, however, that the success rate of docking a small molecule into another small molecule is about 93% if the influence of crystal packing is ignored.</p>


<h4>Demonstration of the accuracy of the nonbonded parameters of the AMBER force field</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>In the crystal structure of Rebek's acridine diacid in complex with quinoxaline (CSD code: YAWJIP) there are two crystallographically independent forms of the complex in the asymmetric unit <a href="#pone.0000531-Pascal1">[15]</a>. This structure was used to benchmark the all-atom AMBER/OPLS force field <a href="#pone.0000531-Duffy1">[16]</a>. In this study, complex YAWJIP was accurately reproduced by the EUDOC program using the nonbonded force field parameters of the second-generation AMBER force field <a href="#pone.0000531-Pang1">[3]</a>, <a href="#pone.0000531-Cornell1">[17]</a>; the mwRMSDs of the guest position between the EUDOC-generated and crystal complexes for forms A and B are both 0.16 Å. The accurate reproduction of the remaining crystal structures (see <a href="#pone-0000531-t002">Table 2</a>) further demonstrates the accuracy of the nonbonded force field parameters of the second-generation AMBER force field (parm99.dat) for reproducing crystal structures <a href="#pone.0000531-Pang1">[3]</a>, <a href="#pone.0000531-Cornell1">[17]</a>.</p>


<h4>Application to supramolecular chemistry</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The above results suggest that the EUDOC program can predict small-molecule guest-host complexes with a reasonable success rate (93%), without consideration to the mechanism of small-molecule complex aggregation—namely, without being given the multimeric host system. To demonstrate this ability herein, the crystal structure of complex YAWJIP is used as a model system. Based on the NMR spectroscopic data of complex YAWJIP, the guest structure quinoxaline was proposed to have face-to-face π-stacking with the acridine portion of Rebek's acridine diacids in an early report of the complex <a href="#pone.0000531-Rebek1">[18]</a>. However, the face-to-face π-stacking was found in neither the crystal structure of complex YAWJIP <a href="#pone.0000531-Pascal1">[15]</a> nor the Monte Carlo statistical mechanics calculations of the complex <a href="#pone.0000531-Duffy1">[16]</a>.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>To perform a prospective docking study, the two-dimensional structures of quinoxaline and Rebek's acridine diacid were converted to 3D structures using the PCModel program (Serena Software LLC, Bloomington, IN), respectively. Both 3D structures were refined with energy minimization monitored with a normal-mode (NMODE) analysis to ensure that the energy minimization stopped when the minimized conformation reached a local potential energy minimum. The energy minimization was performed by using the SANDER module of the AMBER5 program <a href="#pone.0000531-Pearlman1">[19]</a> with the second-generation AMBER force field <a href="#pone.0000531-Cornell1">[17]</a>, and the NMODE analysis was carried out using the NMODE module of the AMBER8 program <a href="#pone.0000531-Pearlman1">[19]</a>. Given the refined 3D structures of quinoxaline and Rebek's acridine diacid, the EUDOC program generated a complex nearly identical to the crystal structure of form A (mwRMSD: 0.7 Å) but not the proposed complex with nearly face-to-face π-stacking. This perspective docking result suggests that the EUDOC program is a useful tool for predicting 3D models of guest-host complexes to aid the design of new molecular entities according to the principles of supramolecular chemistry.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Visual inspection of 154 reported crystal structures of proteins in complex with small molecules <a href="#pone.0000531-Pang1">[3]</a> and the 161 crystal structures of guest-host complexes reported herein suggested that the noncovalent interactions of the guest-host complexes are in general weaker than those of the protein complexes. The average of the interaction energies of the guest-host complexes listed in <a href="#pone-0000531-t004">Table 4</a> (−50.1 kcal/mol) and the average of those of the protein complexes in Table VI (excluding 1PHA) of reference 3 (−108.5 kcal/mol) quantitatively confirm the relatively weak noncovalent interactions of the guest-host complex. This confirmation suggests that to design high-affinity guest-host complexes it is of advantage to incorporate the entropic energy into the binding, because the number of functional groups that can be introduced onto the guest and host structures to confer the nonbonded interactions is limited by the size of the two partners. This “saturation” problem is more apparent for small-molecule complexes than for protein complexes. It is therefore conceivable that the EUDOC program is also a useful tool for estimating the interaction energies of guest-host complexes to aid the design of new molecular entities according to the principles of supramolecular chemistry.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Preparation of the host and guest structures</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The guest and host structures were taken from the crystal structures of their corresponding complexes obtained from CSD <a href="#pone.0000531-Allen1">[1]</a>. Water molecules, counterions, and solvent molecules such as ethanol were removed from the guest or host structure. Hydrogen atoms were added by using the QUANTA97 program (Accelrys Software, Inc, San Diego, California) followed by energy minimization of the hydrogen atoms using the SANDER module of the AMBER5 program <a href="#pone.0000531-Pearlman1">[19]</a> with the second-generation AMBER force field (parm99.dat) <a href="#pone.0000531-Cornell1">[17]</a> and a positional constraint on all non-hydrogen atoms. The protonation states of the guest and host structures shown in Figures <a href="#pone.0000531.s003">S1</a>, <a href="#pone.0000531.s004">S2</a>, <a href="#pone.0000531.s005">S3</a>, <a href="#pone.0000531.s006">S4</a>, <a href="#pone.0000531.s007">S5</a>, <a href="#pone.0000531.s008">S6</a>, <a href="#pone.0000531.s009">S7</a>, <a href="#pone.0000531.s010">S8</a>, <a href="#pone.0000531.s011">S9</a>, <a href="#pone.0000531.s012">S10</a>, <a href="#pone.0000531.s013">S11</a>, <a href="#pone.0000531.s014">S12</a>, <a href="#pone.0000531.s015">S13</a>, <a href="#pone.0000531.s016">S14</a>, <a href="#pone.0000531.s017">S15</a>, <a href="#pone.0000531.s018">S16</a> of Supporting Information were determined according to p<em>K</em>a values of functional groups at pH of 7.0. The atomic charges of the guest and host structures listed in <a href="#pone.0000531.s001">Table S1</a> of Supporting Information were generated according to the RESP procedure <a href="#pone.0000531-Cieplak1">[20]</a> with ab initio calculations at the HF/6-31G* level using the Gaussian03 program <a href="#pone.0000531-Frisch1">[21]</a>. The AMBER atom types of the guest and host structures listed in <a href="#pone.0000531.s001">Table S1</a> of Supporting Information were assigned by the ANTECHAMBER module of AMBER7 <a href="#pone.0000531-Pearlman1">[19]</a>.</p>


<h4>Docking studies using the EUDOC program</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The algorithm of the EUDOC program has been reported elsewhere <a href="#pone.0000531-Pang1">[3]</a>. Briefly, it uses a systematic search protocol, translating and rotating a guest in a putative binding pocket of a host to search for energetically favorable orientations and positions of the guest relative to the host. A docking box is defined within the binding pocket to confine the translation of the ligand. The intermolecular interaction energy is the potential energy of the guest-host complex relative to the potential energies of the two partners in their free state. This energy was calculated according to equations 1 and 2 using the second-generation AMBER force field <a href="#pone.0000531-Cornell1">[17]</a>.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000531.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br></p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>In calculating the intermolecular interaction energy, the dielectric constant was set to 1.0, and the distance cutoffs for steric and electrostatic interactions were set to 10<sup>9</sup> Å. A docking box was defined to enclose the guest structure in the crystal structure of the guest-host complex. The size of the docking box and the cutoff for the interaction energy used by the EUDOC program are listed in <a href="#pone.0000531.s002">Table S2</a> of Supporting Information. The complex-prediction module of the EUDOC program (Version 41, executable available from YPP) was used to translate and rotate the guest around the host at increments of 1.0 Å and 10° of arc, respectively, unless noted otherwise in <a href="#pone-0000531-t001">Table 1</a>.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>To consider the influence of crystal packing, the PyMOL program (DeLano Scientific LLC, South San Francisco, California) was used to generate a multimeric host system by applying the symmetry of the space group of the crystal structure. The host or guest structure was excluded from the multimeric host system if the shortest distance between a heavy atom of the guest structure to be docked and the heavy atom of the host/guest structure in neighboring unit cells was &gt;4.0 Å.</p>


<h4>Energy minimizations monitored with normal-mode analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Energy minimization was performed with the SANDER module of AMBER5 <a href="#pone.0000531-Pearlman1">[19]</a> using (1) maxcyc = 10<sup>6</sup>, (2) dielc = 80, (3) scnb = 2.0, (4) scee = 1.2, (5) ntr = 0, (6) ntmin = 1 or 2, (7) ncyc = 0, (8) cut = 12, and (9) drms = 10<sup>−7</sup>. NMODE analysis was performed with the NMODE module of AMBER8 <a href="#pone.0000531-Pearlman1">[19]</a> using (1) ilevel = 0 or 1, (2) cut = 12, (3) scnb = 2.0, (4) scee = 1.2, (5) dielc = 80, and (6) idiel = 1.</p>


<h4>Mass-weighted root mean square deviations</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The mwRMSDs were calculated by superimposing the host portion of the EUDOC-generated complex over the corresponding host portion of the crystal structure followed by a calculation for the mwRMSD of all atoms of the guest portion in the two superimposed complexes using the PTRAJ module of AMBER8 <a href="#pone.0000531-Pearlman1">[19]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000531"></div><a name="pone.0000531.s001" id="pone.0000531.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">The RESP charges and the AMBER atom types of 161 host-guest complexes</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(2.63 MB DOC)</p>
<a name="pone.0000531.s002" id="pone.0000531.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">The docking box size and the interaction energy cutoff used by the EUDOC program for reproducing the 161 host-guest complexes</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.24 MB DOC)</p>
<a name="pone.0000531.s003" id="pone.0000531.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.10 MB TIF)</p>
<a name="pone.0000531.s004" id="pone.0000531.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.09 MB TIF)</p>
<a name="pone.0000531.s005" id="pone.0000531.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s005">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.11 MB TIF)</p>
<a name="pone.0000531.s006" id="pone.0000531.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s006">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.10 MB TIF)</p>
<a name="pone.0000531.s007" id="pone.0000531.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s007">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.09 MB TIF)</p>
<a name="pone.0000531.s008" id="pone.0000531.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s008">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.11 MB TIF)</p>
<a name="pone.0000531.s009" id="pone.0000531.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s009">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.12 MB TIF)</p>
<a name="pone.0000531.s010" id="pone.0000531.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s010">Figure S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.12 MB TIF)</p>
<a name="pone.0000531.s011" id="pone.0000531.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s011">Figure S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.12 MB TIF)</p>
<a name="pone.0000531.s012" id="pone.0000531.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s012">Figure S10. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.12 MB TIF)</p>
<a name="pone.0000531.s013" id="pone.0000531.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s013">Figure S11. </a></strong></p><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s013</p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(0.20 MB TIF)</p>
<a name="pone.0000531.s014" id="pone.0000531.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s014">Figure S12. </a></strong></p><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s014</p><a id="article1.body1.sec5.supplementary-material14.caption1.p2" name="article1.body1.sec5.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(0.12 MB TIF)</p>
<a name="pone.0000531.s015" id="pone.0000531.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s015">Figure S13. </a></strong></p><a id="article1.body1.sec5.supplementary-material15.caption1.p1" name="article1.body1.sec5.supplementary-material15.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s015</p><a id="article1.body1.sec5.supplementary-material15.caption1.p2" name="article1.body1.sec5.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(0.10 MB TIF)</p>
<a name="pone.0000531.s016" id="pone.0000531.s016"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s016">Figure S14. </a></strong></p><a id="article1.body1.sec5.supplementary-material16.caption1.p1" name="article1.body1.sec5.supplementary-material16.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s016</p><a id="article1.body1.sec5.supplementary-material16.caption1.p2" name="article1.body1.sec5.supplementary-material16.caption1.p2"></a><p class="postSiDOI">(0.21 MB TIF)</p>
<a name="pone.0000531.s017" id="pone.0000531.s017"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s017">Figure S15. </a></strong></p><a id="article1.body1.sec5.supplementary-material17.caption1.p1" name="article1.body1.sec5.supplementary-material17.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s017</p><a id="article1.body1.sec5.supplementary-material17.caption1.p2" name="article1.body1.sec5.supplementary-material17.caption1.p2"></a><p class="postSiDOI">(0.11 MB TIF)</p>
<a name="pone.0000531.s018" id="pone.0000531.s018"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000531.s018">Figure S16. </a></strong></p><a id="article1.body1.sec5.supplementary-material18.caption1.p1" name="article1.body1.sec5.supplementary-material18.caption1.p1"></a><p class="preSiDOI">Chemical structures of the 161 host-guest complexes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000531.s018</p><a id="article1.body1.sec5.supplementary-material18.caption1.p2" name="article1.body1.sec5.supplementary-material18.caption1.p2"></a><p class="postSiDOI">(0.12 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors acknowledge computing support from the University of Minnesota Supercomputing Institute.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YP. Performed the experiments: QW. Analyzed the data: YP QW. Wrote the paper: YP QW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000531-Allen1" id="pone.0000531-Allen1"></a>Allen FH (2002) The Cambridge Structural Database: a quarter of a million crystal structures and rising. Acta Crystallogr Sect B: Struct Sci  58: 380–388.  <ul class="find" data-citedArticleID="958966" data-doi="10.1107/s0108768102003890"><li><a href="http://dx.doi.org/10.1107/s0108768102003890" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Cambridge+Structural+Database%3A+a+quarter+of+a+million+crystal+structures+and+rising." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Cambridge+Structural+Database%3A+a+quarter+of+a+million+crystal+structures+and+rising.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000531-DesJarlais1" id="pone.0000531-DesJarlais1"></a>DesJarlais RL, Seibel GL, Kuntz ID, Furth PS, Alvarez JC, et al.  (1990) Structure-based design of nonpeptide inhibitors specific for the human immunodeficiency virus 1 protease. Proc Natl Acad Sci USA  87: 6644–6648.  <ul class="find" data-citedArticleID="958972" data-doi="10.1073/pnas.87.17.6644"><li><a href="http://dx.doi.org/10.1073/pnas.87.17.6644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure-based+design+of+nonpeptide+inhibitors+specific+for+the+human+immunodeficiency+virus+1+protease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure-based+design+of+nonpeptide+inhibitors+specific+for+the+human+immunodeficiency+virus+1+protease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000531-Pang1" id="pone.0000531-Pang1"></a>Pang Y-P, Perola E, Xu K, Prendergast FG (2001) EUDOC: A computer program for identification of drug interaction sites in macromolecules and drug leads from chemical databases. J Comput Chem  22: 1750–1771.  <ul class="find" data-citedArticleID="958986" data-doi="10.1002/jcc.1129"><li><a href="http://dx.doi.org/10.1002/jcc.1129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EUDOC%3A+A+computer+program+for+identification+of+drug+interaction+sites+in+macromolecules+and+drug+leads+from+chemical+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EUDOC%3A+A+computer+program+for+identification+of+drug+interaction+sites+in+macromolecules+and+drug+leads+from+chemical+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000531-Pang2" id="pone.0000531-Pang2"></a>Pang YP (2007) In silico drug discovery: solving the “target-rich and lead-poor” imbalance using the genome-to-drug-lead paradigm. Clin Pharmacol Ther  81: 30–34.  <ul class="find" data-citedArticleID="958988"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+silico+drug+discovery%3A+solving+the+%26%23x201c%3Btarget-rich+and+lead-poor%26%23x201d%3B+imbalance+using+the+genome-to-drug-lead+paradigm.&amp;auth=&amp;atitle=In+silico+drug+discovery%3A+solving+the+%26%23x201c%3Btarget-rich+and+lead-poor%26%23x201d%3B+imbalance+using+the+genome-to-drug-lead+paradigm." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+silico+drug+discovery%3A+solving+the+%26%23x201c%3Btarget-rich+and+lead-poor%26%23x201d%3B+imbalance+using+the+genome-to-drug-lead+paradigm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+silico+drug+discovery%3A+solving+the+%26%23x201c%3Btarget-rich+and+lead-poor%26%23x201d%3B+imbalance+using+the+genome-to-drug-lead+paradigm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000531-Pang3" id="pone.0000531-Pang3"></a>Pang Y-P, Mullins T, Swartz B, McAllister J, Smith B, et al.  (2007) EUDOC on Blue Gene: accelerating the transfer of drug discoveries from laboratory to patient. IBM J Res Dev  in press:   <ul class="find" data-citedArticleID="958990" data-doi="10.1147/rd.521.0069"><li><a href="http://dx.doi.org/10.1147/rd.521.0069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EUDOC+on+Blue+Gene%3A+accelerating+the+transfer+of+drug+discoveries+from+laboratory+to+patient." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EUDOC+on+Blue+Gene%3A+accelerating+the+transfer+of+drug+discoveries+from+laboratory+to+patient.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000531-Perola1" id="pone.0000531-Perola1"></a>Perola E, Xu K, Kollmeyer TM, Kaufmann SH, Prendergast FG, et al.  (2000) Successful virtual screening of a chemical database for farnesyltransferase inhibitor leads. J Med Chem  43: 401–408.  <ul class="find" data-citedArticleID="959000" data-doi="10.1021/jm990408a"><li><a href="http://dx.doi.org/10.1021/jm990408a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Successful+virtual+screening+of+a+chemical+database+for+farnesyltransferase+inhibitor+leads." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Successful+virtual+screening+of+a+chemical+database+for+farnesyltransferase+inhibitor+leads.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000531-Miller1" id="pone.0000531-Miller1"></a>Miller MA (2002) Chemical database techniques in drug discovery. Nat Rev Drug Discov  1: 220–227.  <ul class="find" data-citedArticleID="958984" data-doi="10.1038/nrd745"><li><a href="http://dx.doi.org/10.1038/nrd745" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemical+database+techniques+in+drug+discovery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemical+database+techniques+in+drug+discovery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000531-Raves1" id="pone.0000531-Raves1"></a>Raves ML, Harel M, Pang YP, Silman I, Kozikowski AP, et al.  (1997) Structure of acetylcholinesterase complexed with the nootropic alkaloid, (−)-huperzine A. Nat Struct Biol  4: 57–63.  <ul class="find" data-citedArticleID="959002" data-doi="10.1038/nsb0197-57"><li><a href="http://dx.doi.org/10.1038/nsb0197-57" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+acetylcholinesterase+complexed+with+the+nootropic+alkaloid%2C+%28%26%23x2212%3B%29-huperzine+A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+acetylcholinesterase+complexed+with+the+nootropic+alkaloid%2C+%28%26%23x2212%3B%29-huperzine+A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000531-Pang4" id="pone.0000531-Pang4"></a>Pang Y-P, Xu K, Kollmeyer TM, Perola E, McGrath WJ, et al.  (2001) Discovery of a new inhibitor lead of adenovirus proteinase: steps toward selective, irreversible inhibitors of cysteine proteinases. FEBS Lett  502: 93–97.  <ul class="find" data-citedArticleID="958992" data-doi="10.1016/s0014-5793(01)02672-2"><li><a href="http://dx.doi.org/10.1016/s0014-5793(01)02672-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovery+of+a+new+inhibitor+lead+of+adenovirus+proteinase%3A+steps+toward+selective%2C+irreversible+inhibitors+of+cysteine+proteinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovery+of+a+new+inhibitor+lead+of+adenovirus+proteinase%3A+steps+toward+selective%2C+irreversible+inhibitors+of+cysteine+proteinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000531-Sun1" id="pone.0000531-Sun1"></a>Sun H, Pang YP, Lockridge O, Brimijoin S (2002) Re-engineering butyrylcholinesterase as a cocaine hydrolase. Mol Pharmacol  62: 220–224.  <ul class="find" data-citedArticleID="959006" data-doi="10.1124/mol.62.2.220"><li><a href="http://dx.doi.org/10.1124/mol.62.2.220" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Re-engineering+butyrylcholinesterase+as+a+cocaine+hydrolase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Re-engineering+butyrylcholinesterase+as+a+cocaine+hydrolase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000531-Dooley1" id="pone.0000531-Dooley1"></a>Dooley AJ, Shindo N, Taggart B, Park JG, Pang Y-P (2006) From genome to drug lead: identification of a small-molecule inhibitor of the SARS virus. Bioorg Med Chem Lett  16: 830–833.  <ul class="find" data-citedArticleID="958974" data-doi="10.1016/j.bmcl.2005.11.018"><li><a href="http://dx.doi.org/10.1016/j.bmcl.2005.11.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genome+to+drug+lead%3A+identification+of+a+small-molecule+inhibitor+of+the+SARS+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genome+to+drug+lead%3A+identification+of+a+small-molecule+inhibitor+of+the+SARS+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000531-Park1" id="pone.0000531-Park1"></a>Park JG, Sill PC, Makiyi EF, Garcia-Sosa AT, Millard CB, et al.  (2006) Serotype-selective, small-molecule inhibitors of the zinc endopeptidase of botulinum neurotoxin serotype A. Bioorg Med Chem  14: 395–408.  <ul class="find" data-citedArticleID="958994" data-doi="10.1016/j.bmc.2005.08.018"><li><a href="http://dx.doi.org/10.1016/j.bmc.2005.08.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Serotype-selective%2C+small-molecule+inhibitors+of+the+zinc+endopeptidase+of+botulinum+neurotoxin+serotype+A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Serotype-selective%2C+small-molecule+inhibitors+of+the+zinc+endopeptidase+of+botulinum+neurotoxin+serotype+A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000531-Lehn1" id="pone.0000531-Lehn1"></a>Lehn JM (1993) Supramolecular chemistry. Science  260: 1762–1763.  <ul class="find" data-citedArticleID="958982" data-doi="10.1126/science.8511582"><li><a href="http://dx.doi.org/10.1126/science.8511582" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Supramolecular+chemistry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Supramolecular+chemistry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000531-Eyal1" id="pone.0000531-Eyal1"></a>Eyal E, Gerzon S, Potapov V, Edelman M, Sobolev V (2005) The limit of accuracy of protein modeling: influence of crystal packing on protein structure. J Mol Biol  351: 431–442.  <ul class="find" data-citedArticleID="958978" data-doi="10.1016/j.jmb.2005.05.066"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.05.066" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+limit+of+accuracy+of+protein+modeling%3A+influence+of+crystal+packing+on+protein+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+limit+of+accuracy+of+protein+modeling%3A+influence+of+crystal+packing+on+protein+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000531-Pascal1" id="pone.0000531-Pascal1"></a>Pascal RA, Ho DM (1993) Molecular-sructure of Rebek diacid quinoxaline - confirmation of 2-point binding. J Am Chem Soc  115: 8507–8508.  <ul class="find" data-citedArticleID="958996" data-doi="10.1021/ja00071a094"><li><a href="http://dx.doi.org/10.1021/ja00071a094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular-sructure+of+Rebek+diacid+quinoxaline+-+confirmation+of+2-point+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular-sructure+of+Rebek+diacid+quinoxaline+-+confirmation+of+2-point+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000531-Duffy1" id="pone.0000531-Duffy1"></a>Duffy EM, Jorgensen WL (1994) Structure and binding for complexes of Rebek's acridine diacid with pyrazine, quinoxaline, and pyridine from Monte Carlo simulations with an all-atom force field. J Am Chem Soc  116: 6337–6343.  <ul class="find" data-citedArticleID="958976" data-doi="10.1021/ja00093a038"><li><a href="http://dx.doi.org/10.1021/ja00093a038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+binding+for+complexes+of+Rebek%27s+acridine+diacid+with+pyrazine%2C+quinoxaline%2C+and+pyridine+from+Monte+Carlo+simulations+with+an+all-atom+force+field." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+binding+for+complexes+of+Rebek%27s+acridine+diacid+with+pyrazine%2C+quinoxaline%2C+and+pyridine+from+Monte+Carlo+simulations+with+an+all-atom+force+field.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000531-Cornell1" id="pone.0000531-Cornell1"></a>Cornell WD, Cieplak P, Bayly CI, Gould IR, Merz KM Jr, et al.  (1995) A second generation force field for the simulation of proteins, nucleic acids, and organic molecules. J Am Chem Soc  117: 5179–5197.  <ul class="find" data-citedArticleID="958970" data-doi="10.1021/ja00124a002"><li><a href="http://dx.doi.org/10.1021/ja00124a002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+second+generation+force+field+for+the+simulation+of+proteins%2C+nucleic+acids%2C+and+organic+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+second+generation+force+field+for+the+simulation+of+proteins%2C+nucleic+acids%2C+and+organic+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000531-Rebek1" id="pone.0000531-Rebek1"></a>Rebek J Jr, Nemeth D (1986) Molecular recognition: Ionic and aromatic stacking interactions bind complementary functional groups in a molecular cleft. J Am Chem Soc  108: 5637–5638.  <ul class="find" data-citedArticleID="959004" data-doi="10.1021/ja00278a052"><li><a href="http://dx.doi.org/10.1021/ja00278a052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+recognition%3A+Ionic+and+aromatic+stacking+interactions+bind+complementary+functional+groups+in+a+molecular+cleft." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+recognition%3A+Ionic+and+aromatic+stacking+interactions+bind+complementary+functional+groups+in+a+molecular+cleft.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000531-Pearlman1" id="pone.0000531-Pearlman1"></a>Pearlman DA, Case DA, Caldwell JW, Ross WS, Cheatham TE III, et al.  (1995) AMBER, a package of computer programs for applying molecular mechanics, normal mode analysis, molecular dynamics and free energy calculations to simulate the structural and energetic properties of molecules. Comput Phys Commun  91: 1–41.  <ul class="find" data-citedArticleID="958998" data-doi="10.1016/0010-4655(95)00041-d"><li><a href="http://dx.doi.org/10.1016/0010-4655(95)00041-d" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AMBER%2C+a+package+of+computer+programs+for+applying+molecular+mechanics%2C+normal+mode+analysis%2C+molecular+dynamics+and+free+energy+calculations+to+simulate+the+structural+and+energetic+properties+of+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AMBER%2C+a+package+of+computer+programs+for+applying+molecular+mechanics%2C+normal+mode+analysis%2C+molecular+dynamics+and+free+energy+calculations+to+simulate+the+structural+and+energetic+properties+of+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000531-Cieplak1" id="pone.0000531-Cieplak1"></a>Cieplak P, Cornell WD, Bayly C, Kollman PA (1995) Application of the multimolecule and multiconformational RESP methodology to biopolymers: Charge derivation for DNA, RNA, and proteins. J Comput Chem  16: 1357–1377.  <ul class="find" data-citedArticleID="958968" data-doi="10.1002/jcc.540161106"><li><a href="http://dx.doi.org/10.1002/jcc.540161106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+the+multimolecule+and+multiconformational+RESP+methodology+to+biopolymers%3A+Charge+derivation+for+DNA%2C+RNA%2C+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+the+multimolecule+and+multiconformational+RESP+methodology+to+biopolymers%3A+Charge+derivation+for+DNA%2C+RNA%2C+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000531-Frisch1" id="pone.0000531-Frisch1"></a>Frisch MJ, Trucks GW, Schlegel HB, Scuseria GE, Robb MA, et al.  (2004) Gaussian 03, Revision C.02. Gaussian, Inc, Wallingford CT.  <ul class="find" data-citedArticleID="958980"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gaussian+03%2C+Revision+C.02.+Gaussian%2C+Inc%2C+Wallingford+CT.&amp;auth=&amp;atitle=Gaussian+03%2C+Revision+C.02.+Gaussian%2C+Inc%2C+Wallingford+CT." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gaussian+03%2C+Revision+C.02.+Gaussian%2C+Inc%2C+Wallingford+CT." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gaussian+03%2C+Revision+C.02.+Gaussian%2C+Inc%2C+Wallingford+CT.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.XML" value="198128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.PDF" value="200835"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.g001.PNG_L" value="487520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.g001.PNG_M" value="79945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.g001.PNG_S" value="2922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.g001.TIF" value="808988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.g001.PNG_I" value="33292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t001.PNG_L" value="717321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t001.PNG_M" value="78403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t001.PNG_S" value="13859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t001.TIF" value="3396894"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t001.PNG_I" value="139588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t002.PNG_L" value="75468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t002.PNG_M" value="77791"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t002.PNG_S" value="3288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t002.TIF" value="344982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t002.PNG_I" value="33700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.g002.PNG_L" value="455394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.g002.PNG_M" value="37020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.g002.PNG_S" value="2050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.g002.TIF" value="759702"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.g002.PNG_I" value="15540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t003.PNG_L" value="752409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t003.PNG_M" value="81435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t003.PNG_S" value="14141"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t003.TIF" value="3514212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t003.PNG_I" value="146895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t004.PNG_L" value="752409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t004.PNG_M" value="81437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t004.PNG_S" value="14143"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t004.TIF" value="3514226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.t004.PNG_I" value="146897"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.e001.PNG" value="6882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.e001.TIF" value="28548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.e002.PNG" value="4049"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.e002.TIF" value="25480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s001.DOC" value="2626560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s002.DOC" value="240640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s003.TIF" value="107428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s004.TIF" value="94242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s005.TIF" value="116564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s006.TIF" value="102960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s007.TIF" value="94114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s008.TIF" value="116654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s009.TIF" value="123786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s010.TIF" value="118240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s011.TIF" value="129970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s012.TIF" value="129481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s013.TIF" value="110563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s014.TIF" value="127035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s015.TIF" value="109079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s016.TIF" value="191741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s017.TIF" value="117797"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000531.s018.TIF" value="65537"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000531&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000531" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000531&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000531&volume=&issue=&title=Accurate Reproduction of 161 Small-Molecule Complex Crystal Structures using the EUDOC Program: Expanding the Use of EUDOC to Supramolecular Chemistry&author_name=Qi%20Wang%2C%20Yuan-Ping%20Pang&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000531" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000531" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000531" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000531&amp;t=Accurate%20Reproduction%20of%20161%20Small-Molecule%20Complex%20Crystal%20Structures%20using%20the%20EUDOC%20Program%3A%20Expanding%20the%20Use%20of%20EUDOC%20to%20Supramolecular%20Chemistry" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000531&title=Accurate%20Reproduction%20of%20161%20Small-Molecule%20Complex%20Crystal%20Structures%20using%20the%20EUDOC%20Program%3A%20Expanding%20the%20Use%20of%20EUDOC%20to%20Supramolecular%20Chemistry&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000531&amp;title=Accurate%20Reproduction%20of%20161%20Small-Molecule%20Complex%20Crystal%20Structures%20using%20the%20EUDOC%20Program%3A%20Expanding%20the%20Use%20of%20EUDOC%20to%20Supramolecular%20Chemistry" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000531" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000531&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Accurate Reproduction of 161 Small-Molecule Complex Crystal Structures using the EUDOC Program: Expanding the Use of EUDOC to Supramolecular Chemistry';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000531';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Accurate%20Reproduction%20of%20161%20Small-Molecule%20Complex%20Crystal%20Structures%20using%20the%20EUDOC%20Program%3A%20Expanding%20the%20Use%20of%20EUDOC%20to%20Supramolecular%20Chemistry http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000531" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000531" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Computer+software%22" title="Search for articles in the subject area:'Computer software'"><div class="flagText">Computer software</div></a>
              <div data-categoryid="2287" data-articleid="23900"
                   data-categoryname="Computer software"
                   class="flagImage" title="Flag 'Computer software' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="23900"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystals%22" title="Search for articles in the subject area:'Crystals'"><div class="flagText">Crystals</div></a>
              <div data-categoryid="19465" data-articleid="23900"
                   data-categoryname="Crystals"
                   class="flagImage" title="Flag 'Crystals' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Electrostatics%22" title="Search for articles in the subject area:'Electrostatics'"><div class="flagText">Electrostatics</div></a>
              <div data-categoryid="17995" data-articleid="23900"
                   data-categoryname="Electrostatics"
                   class="flagImage" title="Flag 'Electrostatics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Molecular+structure%22" title="Search for articles in the subject area:'Molecular structure'"><div class="flagText">Molecular structure</div></a>
              <div data-categoryid="19867" data-articleid="23900"
                   data-categoryname="Molecular structure"
                   class="flagImage" title="Flag 'Molecular structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+complexes%22" title="Search for articles in the subject area:'Protein complexes'"><div class="flagText">Protein complexes</div></a>
              <div data-categoryid="17297" data-articleid="23900"
                   data-categoryname="Protein complexes"
                   class="flagImage" title="Flag 'Protein complexes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interactions%22" title="Search for articles in the subject area:'Protein interactions'"><div class="flagText">Protein interactions</div></a>
              <div data-categoryid="17661" data-articleid="23900"
                   data-categoryname="Protein interactions"
                   class="flagImage" title="Flag 'Protein interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Small+molecules%22" title="Search for articles in the subject area:'Small molecules'"><div class="flagText">Small molecules</div></a>
              <div data-categoryid="17289" data-articleid="23900"
                   data-categoryname="Small molecules"
                   class="flagImage" title="Flag 'Small molecules' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8717'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3852'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2049&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
