

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Kinetic Analysis of Yersinia pestis DNA Adenine Methyltransferase Activity Using a Hemimethylated Molecular Break Light Oligonucleotide</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000801"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000801" />

    <meta name="citation_title" content="Kinetic Analysis of Yersinia pestis DNA Adenine Methyltransferase Activity Using a Hemimethylated Molecular Break Light Oligonucleotide"/>
    <meta itemprop="name" content="Kinetic Analysis of Yersinia pestis DNA Adenine Methyltransferase Activity Using a Hemimethylated Molecular Break Light Oligonucleotide"/>

      <meta name="citation_author" content="Robert J. Wood"/>
            <meta name="citation_author_institution" content="School of Chemistry, University of Southampton, Southampton, United Kingdom"/>
      <meta name="citation_author" content="Michael D. Maynard-Smith"/>
            <meta name="citation_author_institution" content="School of Chemistry, University of Southampton, Southampton, United Kingdom"/>
      <meta name="citation_author" content="Victoria L. Robinson"/>
            <meta name="citation_author_institution" content="Chemical and Biological Sciences, Defence Science and Technology Laboratory, Salisbury, United Kingdom"/>
      <meta name="citation_author" content="Petra C.F. Oyston"/>
            <meta name="citation_author_institution" content="Chemical and Biological Sciences, Defence Science and Technology Laboratory, Salisbury, United Kingdom"/>
      <meta name="citation_author" content="Rick W. Titball"/>
            <meta name="citation_author_institution" content="Chemical and Biological Sciences, Defence Science and Technology Laboratory, Salisbury, United Kingdom"/>
      <meta name="citation_author" content="Peter L. Roach"/>
            <meta name="citation_author_institution" content="School of Chemistry, University of Southampton, Southampton, United Kingdom"/>

    <meta name="citation_date" content="2007/8/29"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000801.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e801"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Beyond Watson and Crick: DNA methylation and molecular enzymology of DNA methyltransferases.; citation_author=A Jeltsch; citation_journal_title=Chembiochem; citation_volume=3; citation_number=1; citation_pages=275-293; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=S-Adenosylmethionine Dependent Methyltransferases: Structures and Functions: World Scientific, London.; citation_author=DTF Dryden; citation_number=2; citation_pages=283-340; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Recognition Sequence of the Dam Methylase of E-Coli-K12.; citation_author=GE Geier; citation_author=P Modrich; citation_journal_title=Clin Res; citation_volume=27; citation_number=3; citation_pages=A604-A604; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Methyl-directed DNA mismatch correction.; citation_author=P Modrich; citation_journal_title=J Biol Chem; citation_volume=264; citation_number=4; citation_pages=6597-6600; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Dam methylation: coordinating cellular processes.; citation_author=A Lobner-Olesen; citation_author=O Skovgaard; citation_author=MG Marinus; citation_journal_title=Curr Opin Microbiol; citation_volume=8; citation_number=5; citation_pages=154-160; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The mechanism by which DNA adenine methylase and papl activate the pap epigenetic switch.; citation_author=AD Hernday; citation_author=BA Braaten; citation_author=DA Low; citation_journal_title=Mol Cell; citation_volume=12; citation_number=6; citation_pages=947-957; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Roles of DNA adenine methylation in regulating bacterial gene expression and virulence.; citation_author=DA Low; citation_author=NJ Weyand; citation_author=MJ Mahan; citation_journal_title=Infect Immun; citation_volume=69; citation_number=7; citation_pages=7197-7204; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DNA adenine methylase is essential for viability and plays a role in the pathogenesis of Yersinia pseudotuberculosis and Vibrio cholerae.; citation_author=SM Julio; citation_author=DM Heithoff; citation_author=D Provenzano; citation_author=KE Klose; citation_author=RL Sinsheimer; citation_journal_title=Infect Immun; citation_volume=69; citation_number=8; citation_pages=7610-7615; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=An essential role for DNA adenine methylation in bacterial virulence.; citation_author=DM Heithoff; citation_author=RL Sinsheimer; citation_author=DA Low; citation_author=MJ Mahan; citation_journal_title=Science; citation_volume=284; citation_number=9; citation_pages=967-970; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Oral immunization with a dam mutant of Yersinia pseudotuberculosis protects against plague.; citation_author=VL Taylor; citation_author=RW Titball; citation_author=PC Oyston; citation_journal_title=Microbiology; citation_volume=151; citation_number=10; citation_pages=1919-1926; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A dam mutant of Yersinia pestis is attenuated and induces protection against plague.; citation_author=VL Robinson; citation_author=PCF Oyston; citation_author=RW Titball; citation_journal_title=Fems Microbiol Lett; citation_volume=252; citation_number=11; citation_pages=251-256; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Selective inhibitors of bacterial DNA adenine methyltransferases.; citation_author=N Mashhoon; citation_author=C Pruss; citation_author=M Carroll; citation_author=PH Johnson; citation_author=NO Reich; citation_journal_title=J Biomol Screen; citation_volume=11; citation_number=12; citation_pages=497-510; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structure, function and mechanism of exocyclic DNA methyltransferases.; citation_author=S Bheemanaik; citation_author=YVR Reddy; citation_author=DN Rao; citation_journal_title=Biochem J; citation_volume=399; citation_number=13; citation_pages=177-190; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Recognition Sequence of the Dam Methylase of Escherichia Coli K12 and mode of cleavage of DpnI endonuclease.; citation_author=GE Geier; citation_author=P Modrich; citation_journal_title=J Biol Chem; citation_volume=254; citation_number=14; citation_pages=1408-1413; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Inhibition of EcoRI DNA methylase with cofactor analogs.; citation_author=N Reich; citation_author=N Mashhoon; citation_journal_title=J Biol Chem; citation_volume=265; citation_number=15; citation_pages=8966-8970; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Escherichia Coli Dam methylase - physical and catalytic properties of the homogeneous enzyme.; citation_author=GE Herman; citation_author=P Modrich; citation_journal_title=J Biol Chem; citation_volume=257; citation_number=16; citation_pages=2605-2612; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Functional characterization of Escherichia coli DNA adenine methyltransferase, a novel target for antibiotics.; citation_author=N Mashhoon; citation_author=M Carroll; citation_author=C Pruss; citation_author=J Eberhard; citation_author=S Ishikawa; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=17; citation_pages=52075-52081; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Comparative studies of the phage T2 and T4 DNA (N-6-adenine)methyltransferases: amino acid changes that affect catalytic activity.; citation_author=VG Kossykh; citation_author=SL Schlagman; citation_author=S Hattman; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=18; citation_pages=3239-3243; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Phage-T4 DNA [N-6-Adenine]methlyltransferase - overexpression, purification, and characterization.; citation_author=VG Kossykh; citation_author=SL Schlagman; citation_author=S Hattman; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=19; citation_pages=14389-14393; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A continuous assay for DNA cleavage: The application of “break lights” to enediynes, iron-dependent agents, and nucleases.; citation_author=JB Biggins; citation_author=JR Prudent; citation_author=DJ Marshall; citation_author=M Ruppen; citation_author=JS Thorson; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=97; citation_number=20; citation_pages=13537-13542; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Fluorescence based strategies for genetic analysis.; citation_author=RT Ranasinghe; citation_author=T Brown; citation_journal_title=Chem Commun; citation_number=21; citation_pages=5487-5502; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Using molecular beacons as a sensitive fluorescence assay for enzymatic cleavage of single-stranded DNA.; citation_author=JJ Li; citation_author=R Geyer; citation_author=W Tan; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=22; citation_pages=e52; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Chimeric RNA-DNA molecular beacon assay for ribonuclease H activity.; citation_author=J Rizzo; citation_author=LK Gifford; citation_author=X Zhang; citation_author=AM Gewirtz; citation_author=P Lu; citation_journal_title=Mol Cell Probe; citation_volume=16; citation_number=23; citation_pages=277-283; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Hairpin fluorescence DNA probe for real-time monitoring of DNA methylation.; citation_author=J Li; citation_author=H Yan; citation_author=K Wang; citation_author=W Tan; citation_author=X Zhou; citation_journal_title=Anal Chem; citation_volume=79; citation_number=24; citation_pages=1050-1056; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phenotypic reversal in Dam mutants of Escherichia-Coli K-12 by a recombinant plasmid containing the Dam+ gene.; citation_author=JA Arraj; citation_author=MG Marinus; citation_journal_title=J Bacteriol; citation_volume=153; citation_number=25; citation_pages=562-565; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Bacteriophage T4 Dam DNA-(N-6-adenine)-methyltransferase - processivity and orientation to the methylation target.; citation_author=VV Zinoviev; citation_author=AA Evdokimov; citation_author=EG Malygin; citation_author=SL Schlagman; citation_author=S Hattman; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=26; citation_pages=7829-7833; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Enzymatic cleavage of a bacterial chromosome at a transposon-inserted rare site.; citation_author=J Hanish; citation_author=M McClelland; citation_journal_title=Nucleic Acids Res; citation_volume=19; citation_number=27; citation_pages=829-832; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=In vitro replication of duplex circular DNA containing the Simian-Virus 40 DNA origin site.; citation_author=CR Wobbe; citation_author=F Dean; citation_author=L Weissbach; citation_author=J Hurwitz; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=82; citation_number=28; citation_pages=5710-5714; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=The efficiency and timing of plasmid DNA-replication in Xenopus eggs - correlations to the extent of prior chromatin assembly.; citation_author=JA Sanchez; citation_author=D Marek; citation_author=LJ Wangh; citation_journal_title=J Cell Sci; citation_volume=103; citation_number=29; citation_pages=907-918; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=A cell cycle-regulated adenine DNA methyltransferase from Caulobacter crescentus processively methylates GANTC sites on hemimethylated DNA.; citation_author=AJ Berdis; citation_author=I Lee; citation_author=JK Coward; citation_author=C Stephens; citation_author=R Wright; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=95; citation_number=30; citation_pages=2874-2879; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Functional analysis of amino acid residues at the dimerisation interface of Kpnl DNA methyltransferase.; citation_author=S Bheemanaik; citation_author=JM Bujnicki; citation_author=V Nagaraja; citation_author=DN Rao; citation_journal_title=Biol Chem; citation_volume=387; citation_number=31; citation_pages=515-523; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A simple statistical parameter for use in evaluation of high throughput screening.; citation_author=J-H Zhang; citation_author=TD Chung; citation_author=KR Oldenburg; citation_journal_title=J Biomol Screen; citation_volume=4; citation_number=32; citation_pages=67-73; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Molecular Cloning A Laboratory Manual.; citation_author=J Sambrook; citation_author=DW Russell; citation_author=N Irwin; citation_author=KA Jansen; citation_number=33; citation_date=2001; citation_publisher=Cold Spring Harbor Laboratory Press; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000801" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Kinetic Analysis of Yersinia pestis DNA Adenine Methyltransferase Activity Using a Hemimethylated Molecular Break Light Oligonucleotide"/>
    <meta name="twitter:description" content="BackgroundDNA adenine methylation plays an important role in several critical bacterial processes including mismatch repair, the timing of DNA replication and the transcriptional control of gene expression. The dependence of bacterial virulence on DNA adenine methyltransferase (Dam) has led to the proposal that selective Dam inhibitors might function as broad spectrum antibiotics.Methodology/Principal FindingsHerein we report the expression and purification of Yersinia pestis Dam and the development of a continuous fluorescence based assay for DNA adenine methyltransferase activity that is suitable for determining the kinetic parameters of the enzyme and for high throughput screening against potential Dam inhibitors. The assay utilised a hemimethylated break light oligonucleotide substrate containing a GATC methylation site. When this substrate was fully methylated by Dam, it became a substrate for the restriction enzyme DpnI, resulting in separation of fluorophore (fluorescein) and quencher (dabcyl) and therefore an increase in fluorescence. The assays were monitored in real time using a fluorescence microplate reader in 96 well format and were used for the kinetic characterisation of Yersinia pestis Dam, its substrates and the known Dam inhibitor, S-adenosylhomocysteine. The assay has been validated for high throughput screening, giving a Z-factor of 0.71&plusmn;0.07 indicating that it is a sensitive assay for the identification of inhibitors.Conclusions/SignificanceThe assay is therefore suitable for high throughput screening for inhibitors of DNA adenine methyltransferases and the kinetic characterisation of the inhibition."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000801.g006"/>

  <meta property="og:title" content="Kinetic Analysis of Yersinia pestis DNA Adenine Methyltransferase Activity Using a Hemimethylated Molecular Break Light Oligonucleotide" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000801" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8586'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5056'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2458&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000801">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000801" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000801&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Kinetic Analysis of <i>Yersinia pestis</i> DNA Adenine Methyltransferase Activity Using a Hemimethylated Molecular Break Light Oligonucleotide
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robert J. Wood
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rjw4@soton.ac.uk">rjw4@soton.ac.uk</a> (RW); <a href="mailto:plr2@soton.ac.uk">plr2@soton.ac.uk</a> (PR)</p>

                <p>Affiliation:
                  School of Chemistry, University of Southampton, Southampton, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael D. Maynard-Smith, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Chemistry, University of Southampton, Southampton, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Victoria L. Robinson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Chemical and Biological Sciences, Defence Science and Technology Laboratory, Salisbury, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Petra C.F. Oyston, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Chemical and Biological Sciences, Defence Science and Technology Laboratory, Salisbury, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rick W. Titball, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Chemical and Biological Sciences, Defence Science and Technology Laboratory, Salisbury, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter L. Roach
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rjw4@soton.ac.uk">rjw4@soton.ac.uk</a> (RW); <a href="mailto:plr2@soton.ac.uk">plr2@soton.ac.uk</a> (PR)</p>

                <p>Affiliation:
                  School of Chemistry, University of Southampton, Southampton, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 29, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000801</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000801" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000801" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000801" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000801" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000801-g001" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000801-g002" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000801-g003" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000801-g004" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000801-t001" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000801-g005" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000801-g006" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000801">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000801" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>DNA adenine methylation plays an important role in several critical bacterial processes including mismatch repair, the timing of DNA replication and the transcriptional control of gene expression. The dependence of bacterial virulence on DNA adenine methyltransferase (Dam) has led to the proposal that selective Dam inhibitors might function as broad spectrum antibiotics.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Herein we report the expression and purification of <em>Yersinia pestis</em> Dam and the development of a continuous fluorescence based assay for DNA adenine methyltransferase activity that is suitable for determining the kinetic parameters of the enzyme and for high throughput screening against potential Dam inhibitors. The assay utilised a hemimethylated break light oligonucleotide substrate containing a GATC methylation site. When this substrate was fully methylated by Dam, it became a substrate for the restriction enzyme DpnI, resulting in separation of fluorophore (fluorescein) and quencher (dabcyl) and therefore an increase in fluorescence. The assays were monitored in real time using a fluorescence microplate reader in 96 well format and were used for the kinetic characterisation of <em>Yersinia pestis</em> Dam, its substrates and the known Dam inhibitor, <em>S</em>-adenosylhomocysteine. The assay has been validated for high throughput screening, giving a Z-factor of 0.71±0.07 indicating that it is a sensitive assay for the identification of inhibitors.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The assay is therefore suitable for high throughput screening for inhibitors of DNA adenine methyltransferases and the kinetic characterisation of the inhibition.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wood RJ, Maynard-Smith MD, Robinson VL, Oyston PC, Titball RW, et al.  (2007) Kinetic Analysis of <em>Yersinia pestis</em> DNA Adenine Methyltransferase Activity Using a Hemimethylated Molecular Break Light Oligonucleotide. PLoS ONE 2(8):
          e801.
            doi:10.1371/journal.pone.0000801</p><p><strong>Academic Editor: </strong>Sebastian Fugmann, National Institute on Aging, United States of America</p><p><strong>Received:</strong> July 17, 2007; <strong>Accepted:</strong> August 5, 2007; <strong>Published:</strong> August 29, 2007</p><p><strong>Copyright:</strong> © 2007 Wood et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>RJW was supported by the Defence Science and Technology Laboratory, MDMS was supported by the Engineering and Physical Sciences Research Council and PLR was supported by the Royal Society via a University Research Fellowship. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>N<sup>6</sup>-methylation of adenine in DNA requires <em>S</em>-adenosylmethionine (AdoMet) as the methyl group donor and is catalysed by a family of DNA adenine methyltransferases (Dam) found in bacteria and viruses <a href="#pone.0000801-Jeltsch1">[1]</a>, <a href="#pone.0000801-Dryden1">[2]</a>. Dam is a member of this family that selectively modifies the adenine base in the sequence GATC and is responsible for most of the DNA methylation in <em>Escherichia coli</em> <a href="#pone.0000801-Geier1">[3]</a>. Unlike many other DNA adenine methyltransferases, Dam does not form part of a restriction modification system, but it has a pervasive influence in regulating many cellular processes including mismatch repair <a href="#pone.0000801-Modrich1">[4]</a> and the timing of DNA replication <a href="#pone.0000801-LobnerOlesen1">[5]</a>. Dam has also been implicated as a global regulator of gene expression: for example the control of the expression of several important genes in <em>E. coli</em> <a href="#pone.0000801-Hernday1">[6]</a>, <a href="#pone.0000801-Low1">[7]</a>, the <em>Yersinia</em> outer proteins <a href="#pone.0000801-Julio1">[8]</a> and at least 20 genes in <em>Salmonella typhimurium</em> <a href="#pone.0000801-Heithoff1">[9]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Manipulating the levels of Dam in bacteria has been shown to reduce bacterial virulence. Dam deficient mutants of <em>Salmonella typhimurium</em> <a href="#pone.0000801-Heithoff1">[9]</a>, <em>Yersinia pseudotuberculosis</em> <a href="#pone.0000801-Taylor1">[10]</a> and <em>Yersinia pestis</em> <a href="#pone.0000801-Robinson1">[11]</a> show reduction in virulence of 10,000-fold, 1 million-fold and &gt;2300-fold respectively. It has also been observed that inactivation of the <em>dam</em> gene is lethal to some bacteria, for example <em>Y. pseudotuberculosis</em> YPIII and <em>Vibrio cholerae</em> <a href="#pone.0000801-Julio1">[8]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The degree of attenuation afforded by manipulation of the <em>dam</em> gene has led to the proposal that Dam inhibitors could find application as broad spectrum antibiotics <a href="#pone.0000801-Heithoff1">[9]</a>, <a href="#pone.0000801-Mashhoon1">[12]</a>, <a href="#pone.0000801-Bheemanaik1">[13]</a>. To identify compounds with an inhibitory effect, a reliable assay for Dam activity is required, ideally one readily adapted to high throughput screening. Perhaps the most reliable reported method for measurement of Dam activity involved monitoring the incorporation of a <sup>3</sup>H labelled methyl group (transferred from <sup>3</sup>H labelled AdoMet) to an adenine residue within a DNA substrate, typically either calf thymus DNA <a href="#pone.0000801-Geier2">[14]</a> or a short double stranded oligonucleotide <a href="#pone.0000801-Reich1">[15]</a>. This method has allowed the kinetic analysis of a number of DNA adenine methyltransferases, including that from <em>E. coli</em> <a href="#pone.0000801-Herman1">[16]</a>, <a href="#pone.0000801-Mashhoon2">[17]</a>, bacteriophage T2 <a href="#pone.0000801-Kossykh1">[18]</a> and bacteriophage T4 <a href="#pone.0000801-Kossykh2">[19]</a>, but using this type of discontinuous assay for detailed kinetic studies is labour intensive.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Recently two assays have been reported that couple methylation sensitive restriction enzymes to a fluorescence signal. Both of these assays utilise ‘molecular break light’ oligonucleotides <a href="#pone.0000801-Biggins1">[20]</a>. Based on molecular beacon design <a href="#pone.0000801-Ranasinghe1">[21]</a>, they are single stranded, self complementary oligonucleotides which adopt a hairpin loop structure with a fluorophore at the 5′ terminus and a quenching dye at the 3′ terminus. Initially the stem of the oligonucleotide keeps the fluorophore and quencher in close proximity, promoting efficient quenching. Cleavage of the stem leads to separation of the fluorophore and quencher, resulting in an increase in fluorescence which is directly related to the quantity of cleaved DNA. The break light assay has been applied to several examples of DNA modification and cleavage, such as measuring the activity of restriction enzymes <a href="#pone.0000801-Mashhoon1">[12]</a>, nucleases <a href="#pone.0000801-Li1">[22]</a>, <a href="#pone.0000801-Rizzo1">[23]</a> and DNA cleavage by enediynes <a href="#pone.0000801-Biggins1">[20]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>By combining the use of methylation sensitive restriction enzymes with a molecular break light oligonucleotide, two alternative assay formats have been developed by Mashhoon <em>et al</em>. <a href="#pone.0000801-Mashhoon1">[12]</a> and Li <em>et al</em>. <a href="#pone.0000801-Li2">[24]</a>. Mashhoon <em>et al</em>. used a break light oligonucleotide in a protection assay. In the first stage, Dam was allowed to methylate the oligonucleotide, and potential Dam inhibitors could be added at this step. In a second step, the oligonucleotide was subjected to cleavage by the restriction endonuclease DpnII (which only restricted the unmethylated DNA). By measuring the resultant fluorescence, the proportion of DNA protected by Dam methylation could be estimated. The assay was suitable for high throughput screening, allowing the relative potency of a library of Dam inhibitors to be compared to a standard. However, the assay is discontinuous and did not permit the rate of methylation to the symmetrical product to be determined.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>The principal limitation of this assay is that it is discontinuous, making detailed kinetic analysis rather time consuming. The next logical development was a continuous assay for Dam activity in a format suitable for high throughput analysis. In a recent publication, Li <em>et al</em>. <a href="#pone.0000801-Li2">[24]</a> described a continuous fluorescence Dam activity assay. This monitored the methylation of an unmethylated break light oligonucleotide in a coupled assay. Two methylation steps are required to make the fully methylated oligonucleotide, which can then be restricted by DpnI, resulting in a proportional increase in fluorescence. Unfortunately, the kinetics of the first methylation to the intermediate hemimethylated oligonucleotide results in a significant lag phase in the observed fluorescence signal, so that the assay is not convenient for determining initial rates of reaction, and Li <em>et al</em>. do not report kinetic parameters such as <em>K</em><sub>M</sub> or <em>V</em><sub>max</sub> for Dam substrates, or the <em>K</em><sub>i </sub>for the potential Dam inhibitors.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>To kinetically characterise Dam and its substrates or inhibitors, it was essential to develop an assay that reported the initial rate of reaction. To do this, we have used a hemimethylated break light oligonucleotide which upon turnover yields the fully methylated product. This is cleaved <em>in situ</em> by the restriction enzyme DpnI, resulting in an increase in fluorescence. Having established a direct relationship between the observed fluorescence increase and the methylation activity of Dam with this hemimethylated substrate, we have used the assay to determine the <em>K</em><sub>M</sub> and <em>V</em><sub>max</sub> of <em>Y. pestis</em> Dam for <em>S</em>-adenosylmethionine and the oligonucleotide substrate. To demonstrate the convenience of this assay, we have used it to determine the <em>K</em><sub>i</sub> of the known Dam inhibitor, <em>S</em>-adenosylhomocysteine with <em>Y. pestis</em> Dam.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Expression and purification of His<sub>6</sub>-tagged <em>Y. pestis</em> Dam</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Initial attempts at expression of <em>Y. pestis</em> Dam in <em>E. coli</em> BL21(DE3) resulted in low protein yields and the attenuation of cell growth upon induction with arabinose. As <em>E. coli</em> was observed to be very sensitive to the intracellular Dam concentration, Dam was expressed in the <em>dam-3</em> and <em>end-1</em> strain GM215 <a href="#pone.0000801-Arraj1">[25]</a>, which did not show the attenuation of cell growth to the same extent observed in BL21(DE3). Large scale cell growth followed by induction at 37 °C and cell harvest two hours after induction was found to result in the highest yield of active protein. During the optimisation of purification of <em>Y. pestis</em> Dam, a rapid thermal inactivation was observed. It was therefore necessary to minimise the time taken to purify the protein, which was shortened to include a single nickel affinity purification step. The highest yield and purity of protein was achieved at pH 9.0. To ensure the purified Dam had the highest possible specific activity, the period of dialysis was kept as short as possible and the majority of the imidazole removed by two short dialysis steps (2×30 min) at 4°C, after which aliquots of protein were flash frozen on dry ice and stored at −80°C. The yield of Dam was consistently 1.5 mg purified protein/l cell culture.</p>


<h4>Continuous break light assay for <em>Y. pestis</em> Dam activity</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To allow continuous monitoring of Dam methylation, hemimethylated oligonucleotide <strong>1</strong> (<a href="#pone-0000801-g001">Fig. 1</a>) was used in the coupled enzyme assay. The kinetics of the reaction were greatly simplified by the use of this hemimethylated substrate oligonucleotide, which allowed a single methylation step (<a href="#pone-0000801-g001">Fig. 1, <strong>1</strong>→<strong>2</strong></a>) to be directly coupled to the DpnI cleavage. The hemimethylated substrate has the added advantage of being the natural substrate for the enzyme during DNA replication <a href="#pone.0000801-Zinoviev1">[26]</a>. Under the conditions of the assay, DpnI preferentially cleaves doubly methylated oligonucleotide <strong>1</strong> leading to a direct relationship between each single methylation event and fluorescence increase. Example assays are shown in <a href="#pone-0000801-g002">Fig. 2</a>, where the ability of Dam to methylate oligonucleotide <strong>1</strong> was followed by measuring the fluorescence increase in assays containing 0, 0.31 and 0.61 nM Dam and a large excess of DpnI. During the timecourse, the fluorescence intensity increased in the assays containing Dam. Rates of fluorescence increase were highest immediately after initiation, but then slowly decreased. The initial rates of fluorescence increase (up to 180 s) were proportional to the concentration of Dam: for example, at a Dam concentration of 0.31 nM, the rate of fluorescence change was 17.1±2.12 arbitrary units/s, but when the concentration of Dam was doubled to 0.61 nM, the rate of fluorescence also doubled to 34.7±1.82 arbitrary units/s. A background increase in fluorescence was observed in negative control assays (lacking Dam) with an initial rate of fluorescence change of 4.68±1.72 arbitrary units/s. This slow background cleavage is probably due to DpnI cleavage of hemimethylated oligonucleotide <strong>1</strong> <a href="#pone.0000801-Hanish1">[27]</a>. The addition of sodium chloride to the assays has been demonstrated to increase the specificity of DpnI for the doubly-methylated substrate <a href="#pone.0000801-Wobbe1">[28]</a>, <a href="#pone.0000801-Sanchez1">[29]</a>. The sodium chloride concentration was therefore optimised to 20 mM, minimising background cleavage by DpnI in the assay.</p>
<div class="figure" id="pone-0000801-g001"><div class="img"><a name="pone-0000801-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Break light Dam activity assay.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The fluorescence of the hemimethylated substrate oligonucleotide 1 is internally quenched by the dabcyl group. It is a substrate for Dam and yields the fully methylated product 2, which is rapidly cleaved by DpnI, thus forming fluorescent oligonucleotide 3.</p>
<span>doi:10.1371/journal.pone.0000801.g001</span></div><div class="figure" id="pone-0000801-g002"><div class="img"><a name="pone-0000801-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Activity of <em>Y. pestis</em> Dam with hemimethylated oligonucleotide substrate.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>The fluorescence increase was recorded with Dam at the following concentrations: 0 nM (dotted line), 0.31 nM (dashed line) and 0.61nM (solid line).</p>
<span>doi:10.1371/journal.pone.0000801.g002</span></div>

<h4>Calibration of oligonucleotide fluorescence</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To convert fluorescein emission into a concentration of cleaved oligonucleotide <strong>1</strong>, a calibration curve of DpnI cleaved oligonucleotide <strong>2</strong>, a doubly methylated analogue of oligonucleotide <strong>1</strong>, was plotted. Concentrations of oligonucleotide used were 0–3.5 nM and the reaction endpoint fluorescence was used for the calibration. The variation of fluorescence intensity with concentration of oligonucleotide was fitted to a linear function (<em>R</em><sup>2</sup> = 0.998), and the measured gradient of oligonucleotide <strong>2</strong> fluorescence increase was 9.3×10<sup>3</sup>±0.3×10<sup>3</sup> arbitrary units/nM.</p>


<h4>Effect of substrates on Dam activity</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p><em>Y. pestis</em> Dam was observed to be sensitive towards inactivation, and it was of interest to determine the kinetics of this process and to investigate the ability of substrates to stabilise the enzyme. To better understand this inactivation process, Dam was incubated for 0–500 s at 30°C, without substrates or in the presence of either 30 nM oligonucleotide <strong>1</strong> or 120 µM AdoMet. Aliquots of Dam were withdrawn throughout the time course and then assayed for activity. The first order rate constants (<em>k</em><sub>inact</sub>) for enzyme inactivation were obtained by fitting the results to an exponential function (<a href="#pone-0000801-g003">Fig. 3</a>) and half lives calculated from t<sub>½</sub> = 0.69/<em>k</em><sub>inact</sub>.</p>
<div class="figure" id="pone-0000801-g003"><div class="img"><a name="pone-0000801-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Inactivation of <em>Y. pestis</em> Dam.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Dam was incubated in a PCR machine at 30 °C with the hemimethylated substrate oligonucleotide (triangles), with AdoMet (open circles) or with no additions (filled circles). Aliquots were withdrawn at time intervals and the activity measured. Activity is expressed as a percentage of the initial activity.</p>
<span>doi:10.1371/journal.pone.0000801.g003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>When incubated alone, Dam was very unstable, with a half life of 42.5±7.5 seconds. Both substrates had a protective effect on the methyltransferase activity, increasing the half life of the enzyme by 3–50 fold. Incubation with oligonucleotide <strong>1</strong> had the greatest protective effect increasing the enzyme half life to &gt;2300 seconds. At least one other example of methyltransferase instability has been reported, for the <em>Caulobacter crescentus</em> cell cycle regulated adenine N-6 methyltransferase (Ccrm), for which a <em>k</em><sub>inact </sub>of 2.3×10<sup>−3</sup> s<sup>-1</sup> at 30°C was observed in the absence of substrates <a href="#pone.0000801-Berdis1">[30]</a>. CcrM was also stabilized by the addition of a saturating concentration of DNA.</p>


<h4>Kinetic analysis of <em>Y. pestis</em> Dam using a break light assay</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Our break light assay was used to determine <em>K</em><sub>M</sub><sup>AdoMet</sup> and <em>K</em><sub>M</sub><sup>DNA</sup> for oligonucleotide <strong>1</strong>. The dependence of Dam activity on AdoMet concentration was fitted to a hyperbola (<a href="#pone-0000801-g004">Fig. 4A</a>). The measured <em>K</em><sub>M</sub><sup>AdoMet</sup> was 11.3±0.63 µM, and is two times higher than that reported for <em>E. coli</em> Dam <a href="#pone.0000801-Mashhoon2">[17]</a> (<a href="#pone-0000801-t001">Table 1</a>).</p>
<div class="figure" id="pone-0000801-g004"><div class="img"><a name="pone-0000801-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Dependence of Dam activity on substrate concentration. (A) AdoMet and (B) oligonucleotide 1.</span></strong></p><span>doi:10.1371/journal.pone.0000801.g004</span></div><div class="figure" id="pone-0000801-t001"><div class="img"><a name="pone-0000801-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000801.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000801.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>A comparison of <em>Y. pestis</em> and <em>E. coli</em> Dam kinetic parameters.</span></strong></p><span>doi:10.1371/journal.pone.0000801.t001</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>For the DNA substrate, data were also fitted to a hyperbola (<a href="#pone-0000801-g004">Fig. 4B</a>) to give <em>K</em><sub>M</sub><sup>DNA</sup> = 3.43±1.68 nM. The <em>Y. pestis</em> enzyme shows a considerably lower <em>K</em><sub>M</sub><sup>DNA</sup> than that reported for the <em>E. coli</em> protein <a href="#pone.0000801-Mashhoon2">[17]</a>. The observed <em>k</em><sub>cat</sub> for <em>Y. pestis</em> Dam is 0.55±0.01 min<sup>−1</sup>, a similar rate to that reported for both the <em>E. coli</em> and T4 DNA adenine methyltransferases <a href="#pone.0000801-Bheemanaik2">[31]</a>. The error in <em>K</em><sub>M</sub><sup>AdoMet </sup>and <em>k</em><sub>cat</sub> were comparable to those obtained using the tritium labelled filter binding assay. The observed instability of <em>Y. pestis</em> Dam in the absence of high concentrations of stabilising DNA makes the determination of the <em>K</em><sub>M</sub><sup>DNA</sup> inherently more difficult and this is reflected in the relatively large observed error.</p>


<h4>Measurement of <em>K</em><sub>i</sub> of <em>S</em>-adenosylhomocysteine for <em>Y. pestis</em> Dam</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The effect of a known Dam inhibitor, <em>S</em>-adenosylhomocysteine on <em>Y. pestis</em> Dam was investigated using the break light assay. From the data shown in <a href="#pone-0000801-g005">Fig. 5</a>, the <em>K</em><sub>i</sub> was calculated to be 6.93±2.01 µM and is lower than the reported value for the <em>E. coli</em> enzyme (<em>K</em><sub>i</sub> = 41.6±10.4 µM). These experiments demonstrate the application of the assay to the detailed kinetic analysis of a single inhibitor.</p>
<div class="figure" id="pone-0000801-g005"><div class="img"><a name="pone-0000801-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Inhibition of Dam by <em>S</em>-adenosylhomocysteine.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) A double reciprocal plot of methylation rate against AdoMet concentration. <em>S</em>-adenosylhomocysteine concentrations were; 0 (•), 5 (○), 10 (▾), 15 (Δ), 20 (▪), 30 (□) and 40 (♦) µM. (B) Apparent <em>K</em><sub>M</sub> plotted against the concentration of <em>S</em>-adenosylhomocysteine.</p>
<span>doi:10.1371/journal.pone.0000801.g005</span></div>

<h4>High throughput screening assay validation</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>The use of the hemimethylated oligonucleotide substrate for high throughput screening was validated using the method of Zhang <a href="#pone.0000801-Zhang1">[32]</a>. Data were collected from three 96 well plates containing positive (normal assay conditions, no inhibitor) and negative controls (assays containing no Dam) to assess the reproducibility and distribution the rate of fluorescence increase at the two activity extremes. From the distribution of the rates of fluorescence increase (<a href="#pone-0000801-g006">Fig. 6</a>), the 99.7% confidence limit (3 standard deviations from the mean) for control assays show that the assay has a high separation band, indicating that it is a highly sensitive assay for inhibitor identification. The data gives an average screening window coefficient (Z-factor) of 0.71±0.07 over three 96 well plates, implying that it is an excellent assay for hit detection.</p>
<div class="figure" id="pone-0000801-g006"><div class="img"><a name="pone-0000801-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000801" data-uri="info:doi/10.1371/journal.pone.0000801.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000801.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000801.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000801.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000801.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>High throughput assay validation.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Rate of fluorescence increase was monitored over 96 wells (wells 1–48 contained positive control assays, wells 49–96 contained negative control assays with no Dam). The solid trend lines indicate the mean rate of fluorescence increase for the positive/negative control datasets; the broken lines indicate the ±3 standard deviations from the mean, which is the 99.7% confidence limit.</p>
<span>doi:10.1371/journal.pone.0000801.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our initial investigations into the use of a DpnI cleavage assay to monitor Dam methylation began with a non-methylated break light oligonucleotide using a similar substrate to that of Li <em>et al.</em> <a href="#pone.0000801-Li2">[24]</a>. We found that the methylation kinetics of this substrate were too complicated to be simply resolved to allow a detailed kinetic analysis of the Dam methylation reaction. To kinetically characterise Dam and its substrates or inhibitors, it was essential to develop an assay that reported the initial rate of reaction. To do this, we have used a hemimethylated break light oligonucleotide which upon turnover yields the fully methylated product. This is cleaved <em>in situ</em> by the restriction enzyme DpnI, resulting in an increase in fluorescence. Having established a direct relationship between the observed fluorescence increase and the methylation activity of Dam with this hemimethylated substrate, we have used the assay to determine the <em>K</em><sub>M</sub> and <em>V</em><sub>max</sub> of <em>Y. pestis</em> Dam for AdoMet and the oligonucleotide substrate. To demonstrate the convenience of this assay, we have used it to determine the <em>K</em><sub>i</sub> of the known Dam inhibitor, <em>S</em>-adenosylhomocysteine with <em>Y. pestis</em> Dam.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>This new approach has distinct practical advantages: the assay is simple to use, highly reproducible and gives real time continuous data which correlates directly to each methylation event giving the potential to allow more accurate determination of initial rates over shorter time scales. It permits the rapid determination of kinetic parameters and the <em>K</em><sub>i</sub> for potential inhibitors, providing a safer alternative to the radioactive filter binding assay.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The assay has been validated for high throughput screening, giving a Z-factor of 0.71±0.07, indicating that it is a highly sensitive and reproducible assay for hit detection and is sufficiently flexible to permit its application to screening large compound libraries for Dam inhibitors.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Materials</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Expression vector pBAD/HisA and <em>E. coli</em> TOP10 competent cells were obtained from Invitrogen (Groningen, NL). DpnI was obtained from New England Biolabs (Herts., UK). <em>Y. pestis</em> was manipulated using suitable biological safety precautions and containment facilities. <em>Y. pestis</em> genomic DNA was isolated from <em>Y. pestis</em> strain GB using a PureGene DNA isolation kit from Gentra Systems (Minneapolis, USA). Other DNA was isolated from cultures or agarose gels using commercially available kits following the manufacturers instructions. <em>E. coli</em> GM215 was obtained from the Yale <em>E. coli</em> genetic stock centre (CGSC# 6645). PCR primers were purchased from Sigma-Genosys (Haverhill, UK). For fluorescence measurements, oligonucleotides <strong>1</strong> and <strong>2</strong> were purchased from ATDBio (Southampton, UK), flat bottomed black 96 well polypropylene and half area flat bottomed 96 well polystyrene microplates were purchased from Greiner Bio-One Ltd (Stonehouse, UK) and measurements were taken using a Tecan Safire 2 microplate reader (Reading, UK). Assays for high throughput screening were prepared using a Beckman Coulter Biomek 3000 liquid handling system equipped with a 200 µl single channel and 20 µl 8 channel pipette head. Mineral oil (sterile filtered, mouse embryo tested, light oil) and <em>S</em>-adenosylmethionine chloride were obtained from Sigma Aldrich (Poole, UK). Bovine serum albumin (BSA) was purchased from Advanced Protein Products Ltd (Brierley Hill, UK).</p>


<h4>Construction of pRJW4213/07 for the expression of <em>Y. pestis</em> Dam</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>DNA manipulations were carried out using standard protocols <a href="#pone.0000801-Sambrook1">[33]</a>. The <em>dam</em> gene was amplified by PCR from <em>Y. pestis</em> GB genomic DNA using <em>Pfu</em> Turbo polymerase and the following oligonucleotide primers: <em>pfdam</em> 5′ GGCGGC<em>CCATGG</em>GC<strong>CACCACCACCACCACCAC</strong>AAGAAAAACCGCGCTTTTTTAAAATGG and <em>prdam</em> 5′ GCCGCC<em>AAGCTT</em>TCAGCTATAGAGCGCCAAAAG. The <em>dam</em> gene was amplified adding an NcoI site (italics) and DNA encoding a His<sub>6</sub>tag (bold) to the 5′ end of the amplified product, and a HindIII site (italics) at the 3′ end. The PCR amplification product was purified and the NcoI-HindIII fragment containing the modified <em>dam</em> gene inserted by ligation between unique NcoI and HindIII sites of pBAD/HisA to yield plasmid pRJW4213/07.</p>


<h4>Expression and purification of <em>Y. pestis</em> Dam</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>All purification steps were carried out at 4°C and samples centrifuged in a Beckman JA-14 rotor unless otherwise stated. All cell culture media contained 100 µg/ml ampicillin. Cell pastes and purified proteins were stored at −80°C. Protein purity was judged by SDS-PAGE with Coomassie staining.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>2YT medium (100 ml) was inoculated from stored strains (pRJW4213/07 in <em>E. coli</em> strain GM215) and grown overnight in a shaking incubator at 37°C and 180 rpm. This overnight culture was used as 1 % innocula into 4×1250 ml fresh medium and grown until the OD<sub>600</sub> reached 0.6. The cultures were induced by addition of 10 ml/l of a filter sterilised 20 % w/v arabinose solution and growth continued at 37°C for two hours. Cells were harvested by centrifugation at 8000 rpm, 4°C, for 12 minutes and the cell paste, typically 20 g was stored at −80°C until required.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>Dam was purified from 10 g of cell paste resuspended in 30 ml of buffer A [50 mM Tris/HCl (pH 9.0), 50 mM imidazole, 300 mM NaCl, 0.05 % v/v triton X-100, 10 % w/v glycerol, 10 mM 2-mercaptoethanol] and 0.3 ml of a 10 mg/ml lysozyme solution added. The suspension was stirred for 20 minutes at 4°C and then sonicated 25 times for 5 second bursts. The lysate was cleared by centrifugation at 12,000 rpm and 4°C for 30 min. The supernatant was applied to a nickel charged chelating sepharose FF column (5 ml bed volume) previously equilibrated in buffer A. The column was washed with 200 ml of buffer A and the proteins eluted with a 30 ml gradient to 100 % buffer B (buffer A plus 500 mM imidazole). The purest fractions (8 ml) were pooled and dialysed twice for 30 min at 4°C against 500 ml buffer C [50 mM Tris/HCl (pH 7.5), 200 mM NaCl, 0.2 mM EDTA, 20 % w/v glycerol, 2 mM dithiothreitol]. The purification yielded 1.5 mg Dam from 10 g cell paste and aliquots of protein solution (100 µl) were immediately frozen at −80°C.</p>


<h4>Dam activity assay</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Fluorescence changes were recorded in a Tecan Safire 2 microplate reader using 10 readings per well (each measurement), 0.5 s between each movement and reading, 1 second of shaking between data collections with 7 seconds of settle time. The following instrument settings were used: the excitation wavelength was 486 nm, the emission wavelength was 518 nm, the bandwidth was 5 nm, the gain was 200, the Z-position was 9300 µm and the integration time was 40 µs. Break light oligonucleotide sequences used in the assay were; oligonucleotide <strong>1</strong>: 5′ C(F)CG<strong>GA<sup>m</sup>TC</strong>CAGTTTTCTG<strong>GATC</strong>CGG(D) 3′; oligonucleotide <strong>2</strong>: 5′ C(F)CG<strong>GA<sup>m</sup>TC</strong>CAGTTTTCTG<strong>GA<sup>m</sup>TC</strong>CGG(D) 3′, where Dam recognition sequences are shown in bold, (F) represents fluorescein and (D) represents a dabcyl quencher.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>The activity of Dam was measured in triplicate in Greiner flat bottomed black 96 well polypropylene microplates, with a total assay volume of 200 µl, maintained at 37°C. Three buffers were required: buffer D [22.4 mM Tris-acetate (pH 7.9), 23.5 mM sodium chloride, 56 mM potassium acetate, 11.2 mM magnesium acetate, 1.12 mM dithiothreitol, 0.12 mg/ml BSA] which could be modified by the addition of between 0 and 235 µM AdoMet and between 0 and 35 nM oligonucleotide <strong>1</strong>; buffer E [20 mM Tris-acetate (pH 7.9), 50 mM potassium acetate, 10 mM magnesium acetate and 1 mM dithiothreitol]; buffer F [48 mM Tris/HCl (pH 7.4), 9.6 mM EDTA, 4.8 mM 2-mercaptoethanol, 0.4 mg/ml BSA and 30 nM oligonucleotide <strong>1</strong>]. Dam solution was prepared by defrosting an aliquot of Dam stock (as purified, 0.17 mg/ml) on ice for 10 minutes and diluting it 510 fold into buffer F. DpnI solution was prepared by diluting the stock solution to 1 U/µl in buffer E.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>170 µl of buffer D was added to each well of the plate and overlaid with 3 drops of mineral oil. The plate was then equilibrated at 37°C for 15 minutes and the reaction initiated by the addition of 10 µl of DpnI solution and 20 µl of Dam solution to each well. Fluorescein emission was then monitored over time. The rate of reaction was calculated by taking the initial rate of fluorescence change over 180 seconds unless otherwise stated. Background changes in fluorescence were accounted for by subtracting a negative control (lacking Dam) when appropriate. The rate of change in fluorescence was converted to a rate of reaction using a fluorescence calibration curve. Data were fitted with the program SigmaPlot.</p>


<h4>Oligonucleotide 1 fluorescence calibration curve</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Assays contained the fully methylated oligonucleotide <strong>2</strong> in place of <strong>1</strong> at concentrations 0, 0.5, 1, 2, 3, 3.5 nM, 25 µM AdoMet and no Dam. The endpoint fluorescence was measured.</p>


<h4>Kinetic analysis of <em>Y. pestis</em> Dam</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The dependence of enzyme activity on AdoMet concentration was measured in a series of assays containing 33 nM oligonucleotide <strong>1</strong>, 1.0 nM Dam and 2.5, 5, 10, 20, 40, 100, 200 µM AdoMet. The dependence of reaction rate on oligonucleotide <strong>1</strong> concentration was measured in a series of assays containing 0.31 nM Dam, 120 µM AdoMet and 0.5, 1.5, 2.5, 3.5, 4.5, 5.5, 6.5 nM oligonucleotide <strong>1</strong>.</p>


<h4>Inactivation of <em>Y. pestis</em> Dam</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The inactivation was monitored at 30°C under three conditions: in the absence of substrates, with 30 nM oligonucleotide <strong>1</strong> or with 120 µM AdoMet. The Dam solution was prepared by diluting Dam (as purified) 213 fold in buffer F lacking DNA. To 200 µl of Dam solution was added 100 µl of DpnI solution and the resultant mixture aliquotted into PCR tubes (35 µl in each). The PCR tubes were maintained at 30°C in a PCR machine and aliquots withdrawn at the required time points. Aliquots were then rapidly cooled in an ice bath and the activities assayed at the end of the timecourse.</p>


<h4>Inhibition of Dam by <em>S</em>-adenosylhomocysteine</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Reactions contained 1 nM Dam and 33 nM oligonucleotide <strong>1</strong>, with varying AdoMet and <em>S</em>-adenosylhomocysteine concentrations: <em>S</em>-adenosylhomocysteine; 0, 5, 10, 15, 20, 30, 40 µM, AdoMet; 20, 25, 30, 40, 70, 140, 200 µM. <em>K</em><sub>i</sub><sup>SAH</sup> was estimated as follows: a double reciprocal plot (<a href="#pone-0000801-g005">Fig. 5A</a>) of 1/rate of reaction against 1/concentration of AdoMet has a slope of <em>K</em><sub>M.app</sub>/<em>V</em><sub>max</sub>, yielding a series of values for the apparent <em>K</em><sub>M</sub> (<em>K</em><sub>M.app</sub>) at different inhibitor concentrations. A plot of <em>K</em><sub>M.app </sub>against concentration of <em>S</em>-adenosylhomocysteine (<a href="#pone-0000801-g005">Fig. 5B</a>) has intercepts of <em>K</em><sub>M</sub> on the <em>K</em><sub>M.app </sub>axis and –<em>K</em><sub>i</sub> on the concentration of <em>S</em>-adenosylhomocysteine axis.</p>


<h4>High throughput screening validation conditions</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>The activity of Dam was measured in half area Greiner flat bottomed black 96 well polystyrene microplates, with a total assay volume of 100 µl, maintained at 30°C. Four buffers were required: buffer G [buffer D containing 5.9 µM AdoMet and 35 nM oligonucleotide <strong>1</strong> (final concentrations of 5 µM and 33 nM respectively in the assay)]; buffer H [buffer E containing 1 U/µl DpnI (final concentration 5 U per assay)]; buffer I [buffer F containing 20 nM Dam (for a final concentration of 2 nM)]. Per 96 well plate, 1160 µl buffer I was mixed with 580 µl buffer H to make buffer J.</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>Using a Biomek 3000 liquid handling system, 85 µl of buffer G was added to each well. The plate was then equilibrated at 30°C for 20 minutes and the reaction initiated by the addition of 15 µl of buffer J per well. The plate was then immediately transferred to a Tecan Saffire 2 microplate reader and fluorescein emission monitored over time. The following instrument settings were used: 10 readings per well (each measurement), 0 s between each movement and reading and no shaking between data collections. The excitation wavelength was 486 nm with a bandwidth of 9 nm, the emission wavelength was 518 nm with a bandwidth of 20 nm, the gain was 130, the Z-position was 9300 µm and the integration time was 40 µs. The rate of reaction was calculated by taking the initial rate of fluorescence change over the first 675 seconds. The rate of fluorescence increase was estimated by fitting data to a linear trend line with the program Excel.</p>

</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RT PR RW MM VR PO. Performed the experiments: RW MM. Analyzed the data: RW MM. Wrote the paper: PR RW MM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000801-Jeltsch1" id="pone.0000801-Jeltsch1"></a>Jeltsch A (2002) Beyond Watson and Crick: DNA methylation and molecular enzymology of DNA methyltransferases. Chembiochem  3: 275–293.  <ul class="find" data-citedArticleID="983694" data-doi="10.1002/1439-7633(20020503)3:5<382::aid-cbic382&gt;3.0.co;2-p"><li><a href="http://dx.doi.org/10.1002/1439-7633(20020503)3:5<382::aid-cbic382&gt;3.0.co;2-p" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Beyond+Watson+and+Crick%3A+DNA+methylation+and+molecular+enzymology+of+DNA+methyltransferases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Beyond+Watson+and+Crick%3A+DNA+methylation+and+molecular+enzymology+of+DNA+methyltransferases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000801-Dryden1" id="pone.0000801-Dryden1"></a>Dryden DTF (1999) <em>S</em>-Adenosylmethionine Dependent Methyltransferases: Structures and Functions: World Scientific, London. pp. 283–340.  <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0000801-Geier1" id="pone.0000801-Geier1"></a>Geier GE, Modrich P (1979) Recognition Sequence of the Dam Methylase of <em>E-Coli</em>-K12. Clin Res  27: A604–A604.  <ul class="find" data-citedArticleID="983682"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Recognition+Sequence+of+the+Dam+Methylase+of+E-Coli-K12.&amp;auth=&amp;atitle=Recognition+Sequence+of+the+Dam+Methylase+of+E-Coli-K12." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+Sequence+of+the+Dam+Methylase+of+E-Coli-K12." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+Sequence+of+the+Dam+Methylase+of+E-Coli-K12.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000801-Modrich1" id="pone.0000801-Modrich1"></a>Modrich P (1989) Methyl-directed DNA mismatch correction. J Biol Chem  264: 6597–6600.  <ul class="find" data-citedArticleID="983714"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Methyl-directed+DNA+mismatch+correction.&amp;auth=&amp;atitle=Methyl-directed+DNA+mismatch+correction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methyl-directed+DNA+mismatch+correction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methyl-directed+DNA+mismatch+correction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000801-LobnerOlesen1" id="pone.0000801-LobnerOlesen1"></a>Lobner-Olesen A, Skovgaard O, Marinus MG (2005) Dam methylation: coordinating cellular processes. Curr Opin Microbiol  8: 154–160.  <ul class="find" data-citedArticleID="983706" data-doi="10.1016/j.mib.2005.02.009"><li><a href="http://dx.doi.org/10.1016/j.mib.2005.02.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dam+methylation%3A+coordinating+cellular+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dam+methylation%3A+coordinating+cellular+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000801-Hernday1" id="pone.0000801-Hernday1"></a>Hernday AD, Braaten BA, Low DA (2003) The mechanism by which DNA adenine methylase and papl activate the pap epigenetic switch. Mol Cell  12: 947–957.  <ul class="find" data-citedArticleID="983692" data-doi="10.1016/s1097-2765(03)00383-6"><li><a href="http://dx.doi.org/10.1016/s1097-2765(03)00383-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mechanism+by+which+DNA+adenine+methylase+and+papl+activate+the+pap+epigenetic+switch." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mechanism+by+which+DNA+adenine+methylase+and+papl+activate+the+pap+epigenetic+switch.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000801-Low1" id="pone.0000801-Low1"></a>Low DA, Weyand NJ, Mahan MJ (2001) Roles of DNA adenine methylation in regulating bacterial gene expression and virulence. Infect Immun  69: 7197–7204.  <ul class="find" data-citedArticleID="983708" data-doi="10.1128/iai.69.12.7197-7204.2001"><li><a href="http://dx.doi.org/10.1128/iai.69.12.7197-7204.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Roles+of+DNA+adenine+methylation+in+regulating+bacterial+gene+expression+and+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Roles+of+DNA+adenine+methylation+in+regulating+bacterial+gene+expression+and+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000801-Julio1" id="pone.0000801-Julio1"></a>Julio SM, Heithoff DM, Provenzano D, Klose KE, Sinsheimer RL, et al.  (2001) DNA adenine methylase is essential for viability and plays a role in the pathogenesis of <em>Yersinia pseudotuberculosis</em> and <em>Vibrio cholerae</em>. Infect Immun  69: 7610–7615.  <ul class="find" data-citedArticleID="983696" data-doi="10.1128/iai.69.12.7610-7615.2001"><li><a href="http://dx.doi.org/10.1128/iai.69.12.7610-7615.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+adenine+methylase+is+essential+for+viability+and+plays+a+role+in+the+pathogenesis+of+Yersinia+pseudotuberculosis+and+Vibrio+cholerae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+adenine+methylase+is+essential+for+viability+and+plays+a+role+in+the+pathogenesis+of+Yersinia+pseudotuberculosis+and+Vibrio+cholerae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000801-Heithoff1" id="pone.0000801-Heithoff1"></a>Heithoff DM, Sinsheimer RL, Low DA, Mahan MJ (1999) An essential role for DNA adenine methylation in bacterial virulence. Science  284: 967–970.  <ul class="find" data-citedArticleID="983688" data-doi="10.1126/science.284.5416.967"><li><a href="http://dx.doi.org/10.1126/science.284.5416.967" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+essential+role+for+DNA+adenine+methylation+in+bacterial+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+essential+role+for+DNA+adenine+methylation+in+bacterial+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000801-Taylor1" id="pone.0000801-Taylor1"></a>Taylor VL, Titball RW, Oyston PC (2005) Oral immunization with a dam mutant of <em>Yersinia pseudotuberculosis</em> protects against plague. Microbiology  151: 1919–1926.  <ul class="find" data-citedArticleID="983728" data-doi="10.1099/mic.0.27959-0"><li><a href="http://dx.doi.org/10.1099/mic.0.27959-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oral+immunization+with+a+dam+mutant+of+Yersinia+pseudotuberculosis+protects+against+plague." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oral+immunization+with+a+dam+mutant+of+Yersinia+pseudotuberculosis+protects+against+plague.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000801-Robinson1" id="pone.0000801-Robinson1"></a>Robinson VL, Oyston PCF, Titball RW (2005) A dam mutant of <em>Yersinia pestis</em> is attenuated and induces protection against plague. Fems Microbiol Lett  252: 251–256.  <ul class="find" data-citedArticleID="983722" data-doi="10.1016/j.femsle.2005.09.001"><li><a href="http://dx.doi.org/10.1016/j.femsle.2005.09.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+dam+mutant+of+Yersinia+pestis+is+attenuated+and+induces+protection+against+plague." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+dam+mutant+of+Yersinia+pestis+is+attenuated+and+induces+protection+against+plague.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000801-Mashhoon1" id="pone.0000801-Mashhoon1"></a>Mashhoon N, Pruss C, Carroll M, Johnson PH, Reich NO (2006) Selective inhibitors of bacterial DNA adenine methyltransferases. J Biomol Screen  11: 497–510.  <ul class="find" data-citedArticleID="983710" data-doi="10.1177/1087057106287933"><li><a href="http://dx.doi.org/10.1177/1087057106287933" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+inhibitors+of+bacterial+DNA+adenine+methyltransferases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+inhibitors+of+bacterial+DNA+adenine+methyltransferases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000801-Bheemanaik1" id="pone.0000801-Bheemanaik1"></a>Bheemanaik S, Reddy YVR, Rao DN (2006) Structure, function and mechanism of exocyclic DNA methyltransferases. Biochem J  399: 177–190.  <ul class="find" data-citedArticleID="983674" data-doi="10.1042/bj20060854"><li><a href="http://dx.doi.org/10.1042/bj20060854" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure%2C+function+and+mechanism+of+exocyclic+DNA+methyltransferases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure%2C+function+and+mechanism+of+exocyclic+DNA+methyltransferases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000801-Geier2" id="pone.0000801-Geier2"></a>Geier GE, Modrich P (1979) Recognition Sequence of the Dam Methylase of <em>Escherichia Coli</em> K12 and mode of cleavage of DpnI endonuclease. J Biol Chem  254: 1408–1413.  <ul class="find" data-citedArticleID="983684"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Recognition+Sequence+of+the+Dam+Methylase+of+Escherichia+Coli+K12+and+mode+of+cleavage+of+DpnI+endonuclease.&amp;auth=&amp;atitle=Recognition+Sequence+of+the+Dam+Methylase+of+Escherichia+Coli+K12+and+mode+of+cleavage+of+DpnI+endonuclease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+Sequence+of+the+Dam+Methylase+of+Escherichia+Coli+K12+and+mode+of+cleavage+of+DpnI+endonuclease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+Sequence+of+the+Dam+Methylase+of+Escherichia+Coli+K12+and+mode+of+cleavage+of+DpnI+endonuclease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000801-Reich1" id="pone.0000801-Reich1"></a>Reich N, Mashhoon N (1990) Inhibition of EcoRI DNA methylase with cofactor analogs. J Biol Chem  265: 8966–8970.  <ul class="find" data-citedArticleID="983718"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inhibition+of+EcoRI+DNA+methylase+with+cofactor+analogs.&amp;auth=&amp;atitle=Inhibition+of+EcoRI+DNA+methylase+with+cofactor+analogs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+EcoRI+DNA+methylase+with+cofactor+analogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+EcoRI+DNA+methylase+with+cofactor+analogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000801-Herman1" id="pone.0000801-Herman1"></a>Herman GE, Modrich P (1982) <em>Escherichia Coli</em> Dam methylase - physical and catalytic properties of the homogeneous enzyme. J Biol Chem  257: 2605–2612.  <ul class="find" data-citedArticleID="983690"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Escherichia+Coli+Dam+methylase+-+physical+and+catalytic+properties+of+the+homogeneous+enzyme.&amp;auth=&amp;atitle=Escherichia+Coli+Dam+methylase+-+physical+and+catalytic+properties+of+the+homogeneous+enzyme." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Escherichia+Coli+Dam+methylase+-+physical+and+catalytic+properties+of+the+homogeneous+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Escherichia+Coli+Dam+methylase+-+physical+and+catalytic+properties+of+the+homogeneous+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000801-Mashhoon2" id="pone.0000801-Mashhoon2"></a>Mashhoon N, Carroll M, Pruss C, Eberhard J, Ishikawa S, et al.  (2004) Functional characterization of <em>Escherichia coli</em> DNA adenine methyltransferase, a novel target for antibiotics. J Biol Chem  279: 52075–52081.  <ul class="find" data-citedArticleID="983712" data-doi="10.1074/jbc.m408182200"><li><a href="http://dx.doi.org/10.1074/jbc.m408182200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+characterization+of+Escherichia+coli+DNA+adenine+methyltransferase%2C+a+novel+target+for+antibiotics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+characterization+of+Escherichia+coli+DNA+adenine+methyltransferase%2C+a+novel+target+for+antibiotics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000801-Kossykh1" id="pone.0000801-Kossykh1"></a>Kossykh VG, Schlagman SL, Hattman S (1997) Comparative studies of the phage T2 and T4 DNA (N-6-adenine)methyltransferases: amino acid changes that affect catalytic activity. J Bacteriol  179: 3239–3243.  <ul class="find" data-citedArticleID="983698"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+studies+of+the+phage+T2+and+T4+DNA+%28N-6-adenine%29methyltransferases%3A+amino+acid+changes+that+affect+catalytic+activity.&amp;auth=&amp;atitle=Comparative+studies+of+the+phage+T2+and+T4+DNA+%28N-6-adenine%29methyltransferases%3A+amino+acid+changes+that+affect+catalytic+activity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+studies+of+the+phage+T2+and+T4+DNA+%28N-6-adenine%29methyltransferases%3A+amino+acid+changes+that+affect+catalytic+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+studies+of+the+phage+T2+and+T4+DNA+%28N-6-adenine%29methyltransferases%3A+amino+acid+changes+that+affect+catalytic+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000801-Kossykh2" id="pone.0000801-Kossykh2"></a>Kossykh VG, Schlagman SL, Hattman S (1995) Phage-T4 DNA [N-6-Adenine]methlyltransferase - overexpression, purification, and characterization. J Biol Chem  270: 14389–14393.  <ul class="find" data-citedArticleID="983700" data-doi="10.1074/jbc.270.24.14389"><li><a href="http://dx.doi.org/10.1074/jbc.270.24.14389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phage-T4+DNA+%5BN-6-Adenine%5Dmethlyltransferase+-+overexpression%2C+purification%2C+and+characterization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phage-T4+DNA+%5BN-6-Adenine%5Dmethlyltransferase+-+overexpression%2C+purification%2C+and+characterization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000801-Biggins1" id="pone.0000801-Biggins1"></a>Biggins JB, Prudent JR, Marshall DJ, Ruppen M, Thorson JS (2000) A continuous assay for DNA cleavage: The application of “break lights” to enediynes, iron-dependent agents, and nucleases. Proc Natl Acad Sci USA  97: 13537–13542.  <ul class="find" data-citedArticleID="983678" data-doi="10.1073/pnas.240460997"><li><a href="http://dx.doi.org/10.1073/pnas.240460997" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+continuous+assay+for+DNA+cleavage%3A+The+application+of+%26%23x201c%3Bbreak+lights%26%23x201d%3B+to+enediynes%2C+iron-dependent+agents%2C+and+nucleases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+continuous+assay+for+DNA+cleavage%3A+The+application+of+%26%23x201c%3Bbreak+lights%26%23x201d%3B+to+enediynes%2C+iron-dependent+agents%2C+and+nucleases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000801-Ranasinghe1" id="pone.0000801-Ranasinghe1"></a>Ranasinghe RT, Brown T (2005) Fluorescence based strategies for genetic analysis. Chem Commun 5487–5502.  <ul class="find" data-citedArticleID="983716" data-doi="10.1039/b509522k"><li><a href="http://dx.doi.org/10.1039/b509522k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fluorescence+based+strategies+for+genetic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fluorescence+based+strategies+for+genetic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000801-Li1" id="pone.0000801-Li1"></a>Li JJ, Geyer R, Tan W (2000) Using molecular beacons as a sensitive fluorescence assay for enzymatic cleavage of single-stranded DNA. Nucleic Acids Res  28: e52.  <ul class="find" data-citedArticleID="983702" data-doi="10.1093/nar/28.11.e52"><li><a href="http://dx.doi.org/10.1093/nar/28.11.e52" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+molecular+beacons+as+a+sensitive+fluorescence+assay+for+enzymatic+cleavage+of+single-stranded+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+molecular+beacons+as+a+sensitive+fluorescence+assay+for+enzymatic+cleavage+of+single-stranded+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000801-Rizzo1" id="pone.0000801-Rizzo1"></a>Rizzo J, Gifford LK, Zhang X, Gewirtz AM, Lu P (2002) Chimeric RNA-DNA molecular beacon assay for ribonuclease H activity. Mol Cell Probe  16: 277–283.  <ul class="find" data-citedArticleID="983720" data-doi="10.1006/mcpr.2002.0423"><li><a href="http://dx.doi.org/10.1006/mcpr.2002.0423" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chimeric+RNA-DNA+molecular+beacon+assay+for+ribonuclease+H+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chimeric+RNA-DNA+molecular+beacon+assay+for+ribonuclease+H+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000801-Li2" id="pone.0000801-Li2"></a>Li J, Yan H, Wang K, Tan W, Zhou X (2007) Hairpin fluorescence DNA probe for real-time monitoring of DNA methylation. Anal Chem  79: 1050–1056.  <ul class="find" data-citedArticleID="983704" data-doi="10.1021/ac061694i"><li><a href="http://dx.doi.org/10.1021/ac061694i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hairpin+fluorescence+DNA+probe+for+real-time+monitoring+of+DNA+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hairpin+fluorescence+DNA+probe+for+real-time+monitoring+of+DNA+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000801-Arraj1" id="pone.0000801-Arraj1"></a>Arraj JA, Marinus MG (1983) Phenotypic reversal in Dam mutants of <em>Escherichia-Coli</em> K-12 by a recombinant plasmid containing the Dam+ gene. J Bacteriol  153: 562–565.  <ul class="find" data-citedArticleID="983670"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phenotypic+reversal+in+Dam+mutants+of+Escherichia-Coli+K-12+by+a+recombinant+plasmid+containing+the+Dam%2B+gene.&amp;auth=&amp;atitle=Phenotypic+reversal+in+Dam+mutants+of+Escherichia-Coli+K-12+by+a+recombinant+plasmid+containing+the+Dam%2B+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+reversal+in+Dam+mutants+of+Escherichia-Coli+K-12+by+a+recombinant+plasmid+containing+the+Dam%2B+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+reversal+in+Dam+mutants+of+Escherichia-Coli+K-12+by+a+recombinant+plasmid+containing+the+Dam%2B+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000801-Zinoviev1" id="pone.0000801-Zinoviev1"></a>Zinoviev VV, Evdokimov AA, Malygin EG, Schlagman SL, Hattman S (2003) Bacteriophage T4 Dam DNA-(N-6-adenine)-methyltransferase - processivity and orientation to the methylation target. J Biol Chem  278: 7829–7833.  <ul class="find" data-citedArticleID="983734" data-doi="10.1074/jbc.m210769200"><li><a href="http://dx.doi.org/10.1074/jbc.m210769200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteriophage+T4+Dam+DNA-%28N-6-adenine%29-methyltransferase+-+processivity+and+orientation+to+the+methylation+target." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteriophage+T4+Dam+DNA-%28N-6-adenine%29-methyltransferase+-+processivity+and+orientation+to+the+methylation+target.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000801-Hanish1" id="pone.0000801-Hanish1"></a>Hanish J, McClelland M (1991) Enzymatic cleavage of a bacterial chromosome at a transposon-inserted rare site. Nucleic Acids Res  19: 829–832.  <ul class="find" data-citedArticleID="983686" data-doi="10.1093/nar/19.4.829"><li><a href="http://dx.doi.org/10.1093/nar/19.4.829" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzymatic+cleavage+of+a+bacterial+chromosome+at+a+transposon-inserted+rare+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzymatic+cleavage+of+a+bacterial+chromosome+at+a+transposon-inserted+rare+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000801-Wobbe1" id="pone.0000801-Wobbe1"></a>Wobbe CR, Dean F, Weissbach L, Hurwitz J (1985) <em>In vitro</em> replication of duplex circular DNA containing the Simian-Virus 40 DNA origin site. Proc Natl Acad Sci USA  82: 5710–5714.  <ul class="find" data-citedArticleID="983730" data-doi="10.1073/pnas.82.17.5710"><li><a href="http://dx.doi.org/10.1073/pnas.82.17.5710" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+replication+of+duplex+circular+DNA+containing+the+Simian-Virus+40+DNA+origin+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+replication+of+duplex+circular+DNA+containing+the+Simian-Virus+40+DNA+origin+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000801-Sanchez1" id="pone.0000801-Sanchez1"></a>Sanchez JA, Marek D, Wangh LJ (1992) The efficiency and timing of plasmid DNA-replication in <em>Xenopus</em> eggs - correlations to the extent of prior chromatin assembly. J Cell Sci  103: 907–918.  <ul class="find" data-citedArticleID="983726"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+efficiency+and+timing+of+plasmid+DNA-replication+in+Xenopus+eggs+-+correlations+to+the+extent+of+prior+chromatin+assembly.&amp;auth=&amp;atitle=The+efficiency+and+timing+of+plasmid+DNA-replication+in+Xenopus+eggs+-+correlations+to+the+extent+of+prior+chromatin+assembly." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+efficiency+and+timing+of+plasmid+DNA-replication+in+Xenopus+eggs+-+correlations+to+the+extent+of+prior+chromatin+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+efficiency+and+timing+of+plasmid+DNA-replication+in+Xenopus+eggs+-+correlations+to+the+extent+of+prior+chromatin+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000801-Berdis1" id="pone.0000801-Berdis1"></a>Berdis AJ, Lee I, Coward JK, Stephens C, Wright R, et al.  (1998) A cell cycle-regulated adenine DNA methyltransferase from <em>Caulobacter crescentus</em> processively methylates GANTC sites on hemimethylated DNA. Proc Natl Acad Sci USA  95: 2874–2879.  <ul class="find" data-citedArticleID="983672" data-doi="10.1073/pnas.95.6.2874"><li><a href="http://dx.doi.org/10.1073/pnas.95.6.2874" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cell+cycle-regulated+adenine+DNA+methyltransferase+from+Caulobacter+crescentus+processively+methylates+GANTC+sites+on+hemimethylated+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cell+cycle-regulated+adenine+DNA+methyltransferase+from+Caulobacter+crescentus+processively+methylates+GANTC+sites+on+hemimethylated+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000801-Bheemanaik2" id="pone.0000801-Bheemanaik2"></a>Bheemanaik S, Bujnicki JM, Nagaraja V, Rao DN (2006) Functional analysis of amino acid residues at the dimerisation interface of Kpnl DNA methyltransferase. Biol Chem  387: 515–523.  <ul class="find" data-citedArticleID="983676" data-doi="10.1515/bc.2006.067"><li><a href="http://dx.doi.org/10.1515/bc.2006.067" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+analysis+of+amino+acid+residues+at+the+dimerisation+interface+of+Kpnl+DNA+methyltransferase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+analysis+of+amino+acid+residues+at+the+dimerisation+interface+of+Kpnl+DNA+methyltransferase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000801-Zhang1" id="pone.0000801-Zhang1"></a>Zhang J-H, Chung TD, Oldenburg KR (1999) A simple statistical parameter for use in evaluation of high throughput screening. J Biomol Screen  4: 67–73.  <ul class="find" data-citedArticleID="983732" data-doi="10.1177/108705719900400206"><li><a href="http://dx.doi.org/10.1177/108705719900400206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple+statistical+parameter+for+use+in+evaluation+of+high+throughput+screening." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple+statistical+parameter+for+use+in+evaluation+of+high+throughput+screening.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000801-Sambrook1" id="pone.0000801-Sambrook1"></a>Sambrook J, Russell DW, Irwin N, Jansen KA (2001) Molecular Cloning A Laboratory Manual. New York: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.XML" value="72921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.PDF" value="149907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g001.PNG_L" value="22270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g001.PNG_M" value="28892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g001.PNG_S" value="2864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g001.TIF" value="52372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g001.PNG_I" value="11819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g002.PNG_L" value="16524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g002.PNG_M" value="25425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g002.PNG_S" value="2840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g002.TIF" value="32162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g002.PNG_I" value="10449"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g003.PNG_L" value="16299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g003.PNG_M" value="27015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g003.PNG_S" value="3087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g003.TIF" value="32966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g003.PNG_I" value="11057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g004.PNG_L" value="32067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g004.PNG_M" value="19821"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g004.PNG_S" value="2215"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g004.TIF" value="58428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g004.PNG_I" value="7927"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.t001.PNG_L" value="74809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.t001.PNG_M" value="101316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.t001.PNG_S" value="14477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.t001.TIF" value="304008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.t001.PNG_I" value="33838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g005.PNG_L" value="44393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g005.PNG_M" value="25802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g005.PNG_S" value="2579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g005.TIF" value="68738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g005.PNG_I" value="10296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g006.PNG_L" value="11905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g006.PNG_M" value="23007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g006.PNG_S" value="2878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g006.TIF" value="30526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000801.g006.PNG_I" value="9540"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000801&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000801" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000801&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000801&volume=&issue=&title=Kinetic Analysis of Yersinia pestis DNA Adenine Methyltransferase Activity Using a Hemimethylated Molecular Break Light Oligonucleotide&author_name=Robert%20J.%20Wood%2C%20Michael%20D.%20Maynard-Smith%2C%20Victoria%20L.%20Robinson%2C%20Petra%20C.F.%20Oyston%2C%20Rick%20W.%20Titball%2C%20Peter%20L.%20Roach&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000801" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000801" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000801" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000801&amp;t=Kinetic%20Analysis%20of%20Yersinia%20pestis%20DNA%20Adenine%20Methyltransferase%20Activity%20Using%20a%20Hemimethylated%20Molecular%20Break%20Light%20Oligonucleotide" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000801&title=Kinetic%20Analysis%20of%20Yersinia%20pestis%20DNA%20Adenine%20Methyltransferase%20Activity%20Using%20a%20Hemimethylated%20Molecular%20Break%20Light%20Oligonucleotide&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000801&amp;title=Kinetic%20Analysis%20of%20Yersinia%20pestis%20DNA%20Adenine%20Methyltransferase%20Activity%20Using%20a%20Hemimethylated%20Molecular%20Break%20Light%20Oligonucleotide" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000801" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000801&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Kinetic Analysis of Yersinia pestis DNA Adenine Methyltransferase Activity Using a Hemimethylated Molecular Break Light Oligonucleotide';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000801';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Kinetic%20Analysis%20of%20Yersinia%20pestis%20DNA%20Adenine%20Methyltransferase%20Activity%20Using%20a%20Hemimethylated%20Molecular%20Break%20Light%20Oligonucleotide http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000801" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000801" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Adenine%22" title="Search for articles in the subject area:'Adenine'"><div class="flagText">Adenine</div></a>
              <div data-categoryid="22567" data-articleid="24440"
                   data-categoryname="Adenine"
                   class="flagImage" title="Flag 'Adenine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="24440"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fluorescence%22" title="Search for articles in the subject area:'Fluorescence'"><div class="flagText">Fluorescence</div></a>
              <div data-categoryid="17783" data-articleid="24440"
                   data-categoryname="Fluorescence"
                   class="flagImage" title="Flag 'Fluorescence' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22High+throughput+screening%22" title="Search for articles in the subject area:'High throughput screening'"><div class="flagText">High throughput screening</div></a>
              <div data-categoryid="48565" data-articleid="24440"
                   data-categoryname="High throughput screening"
                   class="flagImage" title="Flag 'High throughput screening' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Methylation%22" title="Search for articles in the subject area:'Methylation'"><div class="flagText">Methylation</div></a>
              <div data-categoryid="18179" data-articleid="24440"
                   data-categoryname="Methylation"
                   class="flagImage" title="Flag 'Methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligonucleotides%22" title="Search for articles in the subject area:'Oligonucleotides'"><div class="flagText">Oligonucleotides</div></a>
              <div data-categoryid="19655" data-articleid="24440"
                   data-categoryname="Oligonucleotides"
                   class="flagImage" title="Flag 'Oligonucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="24440"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yersinia+pestis%22" title="Search for articles in the subject area:'Yersinia pestis'"><div class="flagText">Yersinia pestis</div></a>
              <div data-categoryid="44253" data-articleid="24440"
                   data-categoryname="Yersinia pestis"
                   class="flagImage" title="Flag 'Yersinia pestis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6916'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4300'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6739&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
