

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Nitric Oxide Destabilizes Pias3 and Regulates Sumoylation</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001085"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001085" />

    <meta name="citation_title" content="Nitric Oxide Destabilizes Pias3 and Regulates Sumoylation"/>
    <meta itemprop="name" content="Nitric Oxide Destabilizes Pias3 and Regulates Sumoylation"/>

      <meta name="citation_author" content="Jing Qu"/>
            <meta name="citation_author_institution" content="National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Guang-Hui Liu"/>
            <meta name="citation_author_institution" content="National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Kaiyuan Wu"/>
            <meta name="citation_author_institution" content="National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Peiwei Han"/>
            <meta name="citation_author_institution" content="National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Peng Wang"/>
            <meta name="citation_author_institution" content="National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Jiangmei Li"/>
            <meta name="citation_author_institution" content="National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Xu Zhang"/>
            <meta name="citation_author_institution" content="National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Chang Chen"/>
            <meta name="citation_author_institution" content="National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China"/>

    <meta name="citation_date" content="2007/10/31"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001085.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1085"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=SUMO: a history of modification.; citation_author=RT Hay; citation_journal_title=Mol Cell; citation_volume=18; citation_number=1; citation_pages=1-12; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=PIAS proteins modulate transcription factors by functioning as SUMO-1 ligases.; citation_author=N Kotaja; citation_author=U Karvonen; citation_author=OA Janne; citation_author=JJ Palvimo; citation_journal_title=Mol Cell Biol; citation_volume=22; citation_number=2; citation_pages=5222-5234; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The nucleoporin RanBP2 has SUMO1 E3 ligase activity.; citation_author=A Pichler; citation_author=A Gast; citation_author=JS Seeler; citation_author=A Dejean; citation_author=F Melchior; citation_journal_title=Cell; citation_volume=108; citation_number=3; citation_pages=109-120; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The polycomb protein Pc2 is a SUMO E3.; citation_author=MH Kagey; citation_author=TA Melhuish; citation_author=D Wotton; citation_journal_title=Cell; citation_volume=113; citation_number=4; citation_pages=127-137; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A mechanism for inhibiting the SUMO pathway.; citation_author=R Boggio; citation_author=R Colombo; citation_author=RT Hay; citation_author=GF Draetta; citation_author=S Chiocca; citation_journal_title=Mol Cell; citation_volume=16; citation_number=5; citation_pages=549-561; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Regulation of SUMOylation by reversible oxidation of SUMO conjugating enzymes.; citation_author=G Bossis; citation_author=F Melchior; citation_journal_title=Mol Cell; citation_volume=21; citation_number=6; citation_pages=349-357; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Protein S-nitrosylation: purview and parameters.; citation_author=DT Hess; citation_author=A Matsumoto; citation_author=SO Kim; citation_author=HE Marshall; citation_author=JS Stamler; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=6; citation_number=7; citation_pages=150-166; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Nitric oxide controls nuclear export of APE1/Ref-1 through S-nitrosation of Cysteines 93 and 310.; citation_author=J Qu; citation_author=GH Liu; citation_author=B Huang; citation_author=C Chen; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=8; citation_pages=2522-2532; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=S-nitrosylated GAPDH initiates apoptotic cell death by nuclear translocation following Siah1 binding.; citation_author=MR Hara; citation_author=N Agrawal; citation_author=SF Kim; citation_author=MB Cascio; citation_author=M Fujimuro; citation_journal_title=Nat Cell Biol; citation_volume=7; citation_number=9; citation_pages=665-674; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=S-nitrosylation of parkin regulates ubiquitination and compromises parkin's protective function.; citation_author=KK Chung; citation_author=B Thomas; citation_author=X Li; citation_author=O Pletnikova; citation_author=JC Troncoso; citation_journal_title=Science; citation_volume=304; citation_number=10; citation_pages=1328-1331; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The biotin switch method for the detection of S-nitrosylated proteins.; citation_author=SR Jaffrey; citation_author=SH Snyder; citation_journal_title=Sci STKE; citation_volume=2001; citation_number=11; citation_pages=L1; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Ubc9 fusion-directed SUMOylation (UFDS): a method to analyze function of protein SUMOylation.; citation_author=A Jakobs; citation_author=J Koehnke; citation_author=F Himstedt; citation_author=M Funk; citation_author=B Korn; citation_journal_title=Nat Methods; citation_volume=4; citation_number=12; citation_pages=245-250; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Association of the human SUMO-1 protease SENP2 with the nuclear pore.; citation_author=J Hang; citation_author=M Dasso; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=13; citation_pages=19961-19966; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Perturbation of SUMOlation enzyme Ubc9 by distinct domain within nucleoporin RanBP2/Nup358.; citation_author=H Saitoh; citation_author=MD Pizzi; citation_author=J Wang; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=14; citation_pages=4755-4763; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The interaction of Piasy with Trim32, an E3-ubiquitin ligase mutated in limb-girdle muscular dystrophy type 2H, promotes Piasy degradation and regulates UVB-induced keratinocyte apoptosis through NFkappaB.; citation_author=A Albor; citation_author=S El-Hizawi; citation_author=EJ Horn; citation_author=M Laederich; citation_author=P Frosk; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=15; citation_pages=25850-25866; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A crosstalk between hSiah2 and Pias E3-ligases modulates Pias-dependent activation.; citation_author=A Depaux; citation_author=F Regnier-Ricard; citation_author=A Germani; citation_author=N Varin-Blank; citation_journal_title=Oncogene; citation_number=16; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Trim32 is a ubiquitin ligase mutated in limb girdle muscular dystrophy type 2H that binds to skeletal muscle myosin and ubiquitinates actin.; citation_author=E Kudryashova; citation_author=D Kudryashov; citation_author=I Kramerova; citation_author=MJ Spencer; citation_journal_title=J Mol Biol; citation_volume=354; citation_number=17; citation_pages=413-424; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=S-nitrosylated protein-disulphide isomerase links protein misfolding to neurodegeneration.; citation_author=T Uehara; citation_author=T Nakamura; citation_author=D Yao; citation_author=ZQ Shi; citation_author=Z Gu; citation_journal_title=Nature; citation_volume=441; citation_number=18; citation_pages=513-517; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of S-nitrosylation motifs by site-specific mapping of the S-nitrosocysteine proteome in human vascular smooth muscle cells.; citation_author=TM Greco; citation_author=R Hodara; citation_author=I Parastatidis; citation_author=HF Heijnen; citation_author=MK Dennehy; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=19; citation_pages=7420-7425; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Regulation of the SUMO pathway sensitizes differentiating human endometrial stromal cells to progesterone.; citation_author=MC Jones; citation_author=L Fusi; citation_author=JH Higham; citation_author=H bdel-Hafiz; citation_author=KB Horwitz; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=20; citation_pages=16272-16277; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PIAS3 suppresses NF-kappaB-mediated transcription by interacting with the p65/RelA subunit.; citation_author=HD Jang; citation_author=K Yoon; citation_author=YJ Shin; citation_author=J Kim; citation_author=SY Lee; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=21; citation_pages=24873-24880; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Specific inhibition of Stat3 signal transduction by PIAS3.; citation_author=CD Chung; citation_author=J Liao; citation_author=B Liu; citation_author=X Rao; citation_author=P Jay; citation_journal_title=Science; citation_volume=278; citation_number=22; citation_pages=1803-1805; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Polymeric chains of SUMO-2 and SUMO-3 are conjugated to protein substrates by SAE1/SAE2 and Ubc9.; citation_author=MH Tatham; citation_author=E Jaffray; citation_author=OA Vaughan; citation_author=JM Desterro; citation_author=CH Botting; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=23; citation_pages=35368-35374; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Activation of p53 by conjugation to the ubiquitin-like protein SUMO-1.; citation_author=M Gostissa; citation_author=A Hengstermann; citation_author=V Fogal; citation_author=P Sandy; citation_author=SE Schwarz; citation_journal_title=EMBO J; citation_volume=18; citation_number=24; citation_pages=6462-6471; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Regulation of MBD1-mediated transcriptional repression by SUMO and PIAS proteins.; citation_author=MJ Lyst; citation_author=X Nan; citation_author=I Stancheva; citation_journal_title=EMBO J; citation_volume=25; citation_number=25; citation_pages=5317-5328; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PIAS proteins are involved in the SUMO-1 modification, intracellular translocation and transcriptional repressive activity of RET finger protein.; citation_author=T Matsuura; citation_author=Y Shimono; citation_author=K Kawai; citation_author=H Murakami; citation_author=T Urano; citation_journal_title=Exp Cell Res; citation_volume=308; citation_number=26; citation_pages=65-77; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Palmitoylation of inducible nitric-oxide synthase at Cys-3 is required for proper intracellular traffic and nitric oxide synthesis.; citation_author=I Navarro-Lerida; citation_author=MM Corvi; citation_author=AA Barrientos; citation_author=F Gavilanes; citation_author=LG Berthiaume; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=27; citation_pages=55682-55689; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PIAS3 induction of PRB sumoylation represses PRB transactivation by destabilizing its retention in the nucleus.; citation_author=JH Man; citation_author=HY Li; citation_author=PJ Zhang; citation_author=T Zhou; citation_author=K He; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=28; citation_pages=5552-5566; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Thioredoxin-mediated negative autoregulation of peroxisome proliferator-activated receptor alpha transcriptional activity.; citation_author=GH Liu; citation_author=J Qu; citation_author=X Shen; citation_journal_title=Mol Biol Cell; citation_volume=17; citation_number=29; citation_pages=1822-1833; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Protein inhibitor of activated STAT3 (PIAS3) interacts with activating transcription factor 1 (ATF1) and regulates the human ferritin H gene through an antioxidant responsive element.; citation_author=K Iwasaki; citation_author=K Hailemariam; citation_author=Y Tsuji; citation_journal_title=J Biol Chem; citation_volume=282; citation_number=30; citation_pages=22335-22343; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001085" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Nitric Oxide Destabilizes Pias3 and Regulates Sumoylation"/>
    <meta name="twitter:description" content="Small ubiquitin-related protein modifiers (SUMO) modification is an important mechanism for posttranslational regulation of protein function. However, it is largely unknown how the sumoylation pathway is regulated. Here, we report that nitric oxide (NO) causes global hyposumoylation in mammalian cells. Both SUMO E2 conjugating enzyme Ubc9 and E3 ligase protein inhibitor of activated STAT3 (Pias3) were targets for S-nitrosation. S-nitrosation did not interfere with the SUMO conjugating activity of Ubc9, but promoted Pias3 degradation by facilitating its interaction with tripartite motif-containing 32 (Trim32), a ubiquitin E3 ligase. On the one hand, NO promoted Trim32-mediated Pias3 ubiquitination. On the other hand, NO enhanced the stimulatory effect of Pias3 on Trim32 autoubiquitination. The residue Cys459 of Pias3 was identified as a target site for S-nitrosation. Mutation of Cys459 abolished the stimulatory effect of NO on the Pias3-Trim32 interaction, indicating a requirement of S-nitrosation at Cys459 for positive regulation of the Pias3-Trim32 interplay. This study reveals a novel crosstalk between S-nitrosation, ubiquitination, and sumoylation, which may be crucial for NO-related physiological and pathological processes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001085.g006"/>

  <meta property="og:title" content="Nitric Oxide Destabilizes Pias3 and Regulates Sumoylation" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001085" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4979'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6855'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3385&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001085">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001085" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001085&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Nitric Oxide Destabilizes Pias3 and Regulates Sumoylation
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jing Qu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Jing Qu, Guang-Hui Liu</p>

              
              

                <p>Affiliation:
                  National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guang-Hui Liu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Jing Qu, Guang-Hui Liu</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ghliu@scripps.edu">ghliu@scripps.edu</a> (GL); <a href="mailto:changchen@moon.ibp.ac.cn">changchen@moon.ibp.ac.cn</a> (CC)</p>

                <p>Affiliation:
                  National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kaiyuan Wu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peiwei Han, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peng Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jiangmei Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xu Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chang Chen
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ghliu@scripps.edu">ghliu@scripps.edu</a> (GL); <a href="mailto:changchen@moon.ibp.ac.cn">changchen@moon.ibp.ac.cn</a> (CC)</p>

                <p>Affiliation:
                  National Laboratory of Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences, and Graduate School of the Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 31, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001085</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001085" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001085" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001085" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001085" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001085-g001" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001085-g002" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001085-g003" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001085-g004" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001085-g005" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001085-g006" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001085">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001085" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Small ubiquitin-related protein modifiers (SUMO) modification is an important mechanism for posttranslational regulation of protein function. However, it is largely unknown how the sumoylation pathway is regulated. Here, we report that nitric oxide (NO) causes global hyposumoylation in mammalian cells. Both SUMO E2 conjugating enzyme Ubc9 and E3 ligase protein inhibitor of activated STAT3 (Pias3) were targets for S-nitrosation. S-nitrosation did not interfere with the SUMO conjugating activity of Ubc9, but promoted Pias3 degradation by facilitating its interaction with tripartite motif-containing 32 (Trim32), a ubiquitin E3 ligase. On the one hand, NO promoted Trim32-mediated Pias3 ubiquitination. On the other hand, NO enhanced the stimulatory effect of Pias3 on Trim32 autoubiquitination. The residue Cys459 of Pias3 was identified as a target site for S-nitrosation. Mutation of Cys459 abolished the stimulatory effect of NO on the Pias3-Trim32 interaction, indicating a requirement of S-nitrosation at Cys459 for positive regulation of the Pias3-Trim32 interplay. This study reveals a novel crosstalk between S-nitrosation, ubiquitination, and sumoylation, which may be crucial for NO-related physiological and pathological processes.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Qu J, Liu G-H, Wu K, Han P, Wang P, et al.  (2007) Nitric Oxide Destabilizes Pias3 and Regulates Sumoylation. PLoS ONE 2(10):
          e1085.
            doi:10.1371/journal.pone.0001085</p><p><strong>Academic Editor: </strong>Mark Cookson, National Institutes of Health, United States of America</p><p><strong>Received:</strong> August 21, 2007; <strong>Accepted:</strong> October 8, 2007; <strong>Published:</strong> October 31, 2007</p><p><strong>Copyright:</strong> © 2007 Qu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported by the National Basic Research Program of China (2006CB911000, 2005CB522804 and 2006CB503900), the National Natural Science Foundation of China (No. 30270352 and 30770512) and the Knowledge Innovation Program of the Chinese Academy of Sciences.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>SUMO modification is an important mechanism for posttranslational regulation of protein function including transcription, intracellular transport, DNA repair, replication, and cell signaling <a href="#pone.0001085-Hay1">[1]</a>. Three SUMO isoforms (SUMO1, 2, 3) exist. The enzymatic reactions of SUMO modification are similar to those in ubiquitin modification and involve the E1-activating enzyme consisting of an Aos1/Uba2 heterodimer, the E2-conjugating enzyme Ubc9, and the E3 ligase that promotes the transfer of SUMO from the E2 enzyme to substrate proteins. The protein inhibitors of activated STAT (Pias) <a href="#pone.0001085-Kotaja1">[2]</a>, nuclear pore component RanBP2 <a href="#pone.0001085-Pichler1">[3]</a>, and polycomb protein Pc2 <a href="#pone.0001085-Kagey1">[4]</a> have been identified as SUMO E3 ligases. Mammalian cells contain four members of the Pias family (Pias1, 3, x, and y) that share a similar domain structure. These ligases bring the reaction partners in close proximity and this leads to an increased rate of isopeptide bond formation. The conjugation of proteins with SUMO can be reversed by isopeptidases, allowing for dynamic regulation of SUMO-dependent processes <a href="#pone.0001085-Hay1">[1]</a>. Little is known about the regulation of the sumoylation pathway itself. One example of such regulation is that the viral protein Gam-1 can inhibit sumoylation by targeting E1 and E2 for degradation <a href="#pone.0001085-Boggio1">[5]</a>. In addition, reactive oxygen species (ROS) have been shown to cause repression of sumoylation by crosslinking E1 subunit Uba2 to the E2 subunit Ubc9 <a href="#pone.0001085-Bossis1">[6]</a>. To date, how SUMO E3 ligases are subjected to signal-directed regulation is largely unknown.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Nitric oxide (NO) plays important roles in cell signaling, acting mainly through S-nitrosation of protein cysteine residues <a href="#pone.0001085-Hess1">[7]</a>. S-nitrosation can link NO signaling to other cellular events through crosstalk with various other post-translational modifications, such as phosphorylation and ubiquitination <a href="#pone.0001085-Hess1">[7]</a>. However, the connection between S-nitrosation and sumoylation modifications has not been established. We noted that several SUMO-related enzymes contain conserved cysteines, raising the possibility that NO species might influence the biological function of these proteins through direct post-translational modification. In this report, we show that both SUMO E2 (Ubc9) and E3 (Pias) enzymes are targets for S-nitrosation. Importantly, S-nitrosation of Pias3 regulates its stability and thereby counteracts the sumoylation pathway. Our study reveals a novel molecular mechanism through which sumoylation is regulated, which provides new insight into the function of S-nitrosation in human health and disease.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Nitric oxide causes global hyposumoylation by a different mechanism than H<sub>2</sub>O<sub>2</sub></h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To investigate whether nitric oxide (NO) can regulate the formation of SUMO-conjugated species, HEK293 cells were transfected with the expression vector encoding the HA-tagged SUMO1, SUMO2, or SUMO3, and then treated with or without 500 µM GSNO (a widly-used NO donor and S-nitrosating agent) for 4 h <a href="#pone.0001085-Hess1">[7]</a>, <a href="#pone.0001085-Qu1">[8]</a>. The cells were lysed directly in Laemmli buffer and immunoblotted with HA antibody. We observed a marked reduction in the levels of both SUMO1 and SUMO2 conjugates, but only a slight reduction in SUMO3 conjugates (<a href="#pone-0001085-g001">Fig 1<em>A</em></a>). Simultaneous accumulation of free SUMO1 and SUMO2 was also observed in GSNO-challenged cells (<a href="#pone-0001085-g001">Fig 1<em>A</em></a>), suggesting that overall hyposumoylation was attributed to altered utilization rather than reduced expression of SUMO1/2 modifier. Most SUMO1 conjugates disappeared within 4 h, comparable to the effect of 1 h H<sub>2</sub>O<sub>2</sub> treatment (<a href="#pone-0001085-g001">Fig 1<em>B</em></a>). Notably, several sumoylated substrates, including SP3 and PML, were deconjugated upon GSNO or H<sub>2</sub>O<sub>2</sub> stimulation (<a href="#pone.0001085.s001">Fig <em>S1</em></a>). The analysis of endogenous SUMO1 conjugates in HeLa cells revealed a dose-dependent repression of overall sumoylation in response to less than 1 mM of GSNO (<a href="#pone-0001085-g001">Fig 1<em>C</em></a>). Correspondingly, global ubiquitination was also downregulated in a dose-dependent manner under GSNO treatment, in sharp contrast to its insensitivity to H<sub>2</sub>O<sub>2</sub> <a href="#pone.0001085-Bossis1">[6]</a>. It should be pointed out that much higher concentrations (2–8 mM) of GSNO induced a significant increase in overall sumoylation (<a href="#pone.0001085.s002">Fig <em>S2</em></a>), a phenomenon that has recently been reported for high concentrations of H<sub>2</sub>O<sub>2 </sub><a href="#pone.0001085-Bossis1">[6]</a>. To know whether the decrease in SUMO1 conjugates could occur under conditions of physiological NO production, we employed the RAW264.7 macrophage, which generates NO upon LPS stimulation <a href="#pone.0001085-Hara1">[9]</a>. <a href="#pone-0001085-g001">Fig 1<em>D</em></a> demonstrates that LPS treatment for 24 h discernibly decreased overall SUMO1 conjugates, accompanied by induction of inducible nitric oxide synthase (iNOS). This effect was prevented by cotreatment with the iNOS inhibitor SMT, indicating that the decrease in SUMO1 conjugates arises from the specific effect of NO.</p>
<div class="figure" id="pone-0001085-g001"><div class="img"><a name="pone-0001085-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Nitric oxide caused loss of sumo conjugates.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) HEK293 cells transfected with HA-SUMO1, 2 or 3 were treated with 500 µM GSNO for 4 h, lysed in Laemmli buffer, and immunoblotted with αHA or α actin. Asterisk: free SUMO1/2/3. (B) Time-dependent effect of GSNO on SUMO1 conjugates in transfected HEK293 cells, treated as in (A). In a control experiment, cells were treated with H<sub>2</sub>O<sub>2</sub> (1 mM) for 1h. (C) HeLa cells were treated with increasing concentrations of GSNO for 4 h, lysed in Laemmli buffer, and immunoblotted with αSUMO1, αubiquitin or αactin. (D) RAW264.7 cells pretreated with or without SMT were stimulated with LPS for 24 h, lysed in Laemmli buffer, and immunoblotted with αSUMO1, αiNOS or αactin.</p>
<span>doi:10.1371/journal.pone.0001085.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Since H<sub>2</sub>O<sub>2</sub> results in repression of sumoylation by formation of a disulfide bond between the E1 subunit Uba2 and the E2 Ubc9 <a href="#pone.0001085-Bossis1">[6]</a>, we firstly investigated whether NO might involve a similar mechanism. HeLa cells were transfected with HA-Ubc9 and then stimulated with H<sub>2</sub>O<sub>2</sub> or GSNO. Immunoblotting showed a DTT-sensitive crosslink between Ubc9 and Uba2 upon H<sub>2</sub>O<sub>2</sub> treatment, consistent with the previous report <a href="#pone.0001085-Bossis1">[6]</a>. GSNO, however, could not induce such a crosslink (<a href="#pone.0001085.s003">Fig <em>S3</em></a>). Considering that Ubc9 contains redox-sensitive cysteines <a href="#pone.0001085-Bossis1">[6]</a>, we next investigated whether NO could give rise to Ubc9 thiol modification. HEK293 cells transfected with HA-Ubc9 were treated with GSNO and the lysates were subjected to the biotin-switch assay, in which the S-nitrosation can be specifically identified as described previously <a href="#pone.0001085-Chung1">[10]</a>, <a href="#pone.0001085-Jaffrey1">[11]</a>. As shown in <a href="#pone-0001085-g002">Fig 2<em>A</em></a>, HEK293-expressed Ubc9 was readily S-nitrosated under GSNO treatment. Using site-directed mutagenesis, we identified Cys75 of Ubc9 as the target for S-nitrosation. Given that the SUMO transfer process involves a <em>trans</em>-esterification reaction between Uba2 and Ubc9, we examined whether S-nitrosation of Ubc9 could compromise its ability to receive SUMO. As shown in <a href="#pone-0001085-g002">Fig 2<em>B</em></a>, stimulation of cells with GSNO did not impair the formation of the Ubc9-SUMO thioester bond. In sharp contrast, the thioester intermediate rapidly disappeared upon H<sub>2</sub>O<sub>2</sub> treatment. Next, in order to know whether S-nitrosation of Ubc9 affects its ability to conjugate SUMO to substrate protein, the Ubc9 fusion-directed sumoylation system, in which Ubc9 can direct sumoylation to fused p53 in a manner independent of E3 ligase <a href="#pone.0001085-Jakobs1">[12]</a>, was employed. Immunoblotting revealed that the Ubc9-fused p53 was effectively mono- and di-sumoylated in HEK293 cells expressing p53-Ubc9 (<a href="#pone-0001085-g002">Fig 2<em>C</em></a>). The sumoylation of p53 was dependent on the catalytic activity of fused Ubc9, because mutation of Ubc9 at Cys93 diminished such modification and the endogenous p53 was not sumoylated even in the presence of overexpressed Ubc9 (<a href="#pone-0001085-g002">Fig 2<em>C</em></a>), consistent with the previous report <a href="#pone.0001085-Jakobs1">[12]</a>. Using this system, we observed that unlike H<sub>2</sub>O<sub>2</sub>, which deconjugated the mono- and di-sumoylated species efficiently, sumoylation of p53 was hardly influenced by GSNO (<a href="#pone-0001085-g002">Fig 2<em>C</em></a>). This experiment indicates that NO does not interfere with the SUMO conjugating activity of Ubc9, provided that Ubc9 and the protein substrate are “close” enough to each other in vivo. To further support the above conclusion that NO does not interfere with the SUMO conjugating function of Ubc9, either Ubc9 (WT) or Ubc9 (C75S) was overexpressed in HEK293 cells (<a href="#pone.0001085.s004">Fig. <em>S4</em></a>). One may predict that, if S-nitrosation of Ubc9 can result in repression of sumoylation, at least overexpression of Ubc9 (C75S), the S-nitrosation-deficient Ubc9 mutant, could prevent GSNO-caused hyposumoylation. However, the outcome was negative (<a href="#pone.0001085.s004">Fig <em>S4</em></a>). Together, these results indicate that NO results in overall hyposumoylation in a Ubc9-independent manner.</p>
<div class="figure" id="pone-0001085-g002"><div class="img"><a name="pone-0001085-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>S-nitrosation of Ubc9 did not compromise its SUMO E2 enzyme activity.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>(A) S-nitrosation of HA-Ubc9 (wild-type or cysteine mutants) in HEK293 cells stimulated with 500 µM GSNO determined by biotin-switch assay. (B) HeLa cells were incubated with 500 µM H<sub>2</sub>O<sub>2</sub> for 30 min or 500 µM GSNO for the indicated time, lysed in Laemmli buffer with (lower blot) or without (upper blot) DTT, and immunoblotted with αUbc9. (C) HEK293 cells were transfected with the indicated plasmids, treated with H<sub>2</sub>O<sub>2</sub> (for 1h) or GSNO (for 4 h), lysed in Laemmli buffer, and immunoblotted with αp53 or αactin.</p>
<span>doi:10.1371/journal.pone.0001085.g002</span></div>

<h4>Nitric oxide promotes the degradation of Pias3</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Since mislocalization of SUMO conjugating or deconjugating enzymes has been reported to result in disruption of the sumoylation pathway <a href="#pone.0001085-Hang1">[13]</a>, <a href="#pone.0001085-Saitoh1">[14]</a>, we examined the subcellular localizations of SUMO conjugating enzymes (Uba2, Ubc9, and Pias3) and deconjugating enzyme (Senp2), in the absence or presence of GSNO. As shown in <a href="#pone.0001085.s005">Fig <em>S5</em></a>, the intra-nuclear localization of all these enzymes remained unchanged irrespective of the presence of GSNO. Subsequently, we examined their protein expression in HeLa cells. Immunoblotting revealed a dose-dependent decline in the protein level of Pias3, but not Uba2, Ubc9 and Senp2 under GSNO stimulation (<a href="#pone-0001085-g003">Fig 3<em>A</em></a>). In a control experiment, the expression of RanBP2 slightly increased under the same conditions, thereby indicating a specific down-regulation of Pias3 among the SUMO E3 ligases. Preincubation of cells with proteasome inhibitor MG132 prevented the disappearance of Pias3 caused by GSNO (<a href="#pone-0001085-g003">Fig 3<em>A</em></a>), suggesting that NO might regulate Pias3 degradation in a proteasome-dependent manner. To evaluate whether the reduced level of Pias3 could account for global hyposumoylation, two different small interfering RNAs (siRNA) specific to Pias3 were transfected into HeLa cells. As shown in <a href="#pone-0001085-g003">Fig 3<em>B</em></a>, knockdown of endogenous Pias3 substantially reduced most SUMO1 conjugates, indicating that the degradation of endogenous Pias3 was responsible for NO-elicited repression of sumoylation, although other mechanisms may also be involved.</p>
<div class="figure" id="pone-0001085-g003"><div class="img"><a name="pone-0001085-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Effects of GSNO on the protein level of SUMO-related enzymes.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) HeLa cells were treated with increasing concentrations of GSNO for 4 h, lysed in Laemmli buffer and immunoblotted with the indicated antibodies. (B) Immunoblotting analysis of endogenous Pias3 and SUMO1 conjugates in HeLa cells transfected with non-specific, mismatched or Pias3-specific siRNAs.</p>
<span>doi:10.1371/journal.pone.0001085.g003</span></div>

<h4>Nitric oxide promotes Pias3-Trim32 interplay and enhances Pias3 ubiquitination</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Proteins are usually ubiquitinated before proteasomal degradation <a href="#pone.0001085-Albor1">[15]</a>, <a href="#pone.0001085-Depaux1">[16]</a>. To find out whether NO promotes ubiquitination of Pias3, HeLa cells were transfected with His-tagged ubiquitin, and the ubiquitin-conjugated proteins were recovered with Ni<sup>2+</sup> affinity agrose and subjected to immunoblotting for Pias3. As shown in <a href="#pone-0001085-g004">Fig 4<em>A</em></a>, the ubiquitinated forms of Pias3 (especially the mono-ubiquitinated one) obviously increased after cells were treated with GSNO. This increase was specific, because the overall amount of ubiquitin conjugates diminished under the same conditions.</p>
<div class="figure" id="pone-0001085-g004"><div class="img"><a name="pone-0001085-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>GSNO promotes Pias3-Trim32 interplay.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) HeLa cells expressing His-Ub were stimulated with or without GSNO for 4 h and then subjected to His-Ub pull down. Both longer and shorter exposures are shown. Bracket: poly-ubiquitinated Pias3; Arrow: mono-ubiquitinated Pias3. (B) Effect of GSNO on Pias3-Trim32 interaction determined by GST-pull down assay. GST-Pias3 fusion proteins, immobilized on beads, were mixed with cell lysates with GFP-Trim32 expression in the absence or presence of 500 µM GSNO. Pull-downed and input GFP-Trim32 was analyzed by immunoblotting. (C) Confocal microscopic analysis of subcellular localization of Myc-Pias3 and GFP-Trim32 in cotransfected HEK293 cells. 4 h after transfection, cells were incubated with or without LMB for 16 h. (D) HeLa cells transfected with or without GFP-Trim32 were stimulated with or without GSNO for 4 h and then the denatured lysates were immunoprecipitated with αPias3, followed by detection for Ub. Bracket: poly-ubiquitinated Pias3; Arrow: mono-ubiquitinated Pias3; Asterisk: unmodified Pias3. (E) HEK293 cells transfected with the indicated plasmids were stimulated with or without GSNO for 1 h and then subjected to His-Ub pull down. Bracket: ubiquitinated GFP-Trim32. In panels (A), (C), (D) and (E), cells were incubated with MG132 to prevent potential protein degradation.</p>
<span>doi:10.1371/journal.pone.0001085.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The observation that NO selectively promoted Pias3 ubiquitination suggested a possibility that NO might enhance the specific interaction between Pias3 and its ubiquitin E3 ligase. Trim32, a RING-containing protein, has recently been reported to promote ubiquitination of Piasy <a href="#pone.0001085-Albor1">[15]</a>. To assess the possibility that Pias3 employs Trim32 as its ubiquitin E3 ligase, we firstly utilized a GST pull-down assay to explore the potential association between Pias3 and Trim32. The results demonstrated that the recombinant GST-Pias3, but not GST alone, specifically associated with Trim32 (<a href="#pone-0001085-g004">Fig 4<em>B</em></a>). Such association was also verified in intact cells by co-immunoprecipitation (Co-IP) (<a href="#pone.0001085.s006">Fig <em>S6</em></a> and <a href="#pone.0001085.s007">Fig <em>S7</em></a>). Subsequently, it was investigated whether Pias3 and Trim32 colocalize in the cells. HEK293 cells were cotransfected with GFP-Trim32 and Myc-Pias3 and then their subcellular localizations were determined. As shown in <a href="#pone-0001085-g004">Fig 4<em>C</em></a>, Trim32 mainly resided in cytoplasmic small granules in Trim32-singly expressed cells. Surprisingly, coexpression of Pias3 induced the appearance of Trim32 in Pias3-localized nuclear dots. In addition, Trim32 was found in large cytosolic aggregates upon Pias3 cotransfection (<a href="#pone-0001085-g004">Fig 4<em>C</em></a> and <a href="#pone.0001085.s012">Movie <em>S1</em></a>), a phenomenon that may be explained by the finding that Pias3 stimulated autoubiquitination of Trim32 (<a href="#pone-0001085-g004">Fig 4<em>E</em></a>). To determine the mechanism by which Trim32 was recruited to the Pias3-localized nuclear compartment, LMB, a CRM1-targeting nuclear export inhibitor, was used. As shown in <a href="#pone-0001085-g004">Fig 4<em>C</em></a>, LMB prevented the colocalization of Trim32 and Pias3 in the nucleus as well as preventing the stimulatory effect of Pias3 on Trim32 cytosolic aggregation. Considering that Pias proteins bear both a nuclear localization sequence and a nuclear export sequence <a href="#pone.0001085-Albor1">[15]</a>, we speculated that the import of Trim32 into the nuclear dots might be mediated by the dynamic nucleocytosolic shuttling of Pias3.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>To investigate whether Trim32 controls ubiquitination of Pias3, HeLa cells were transiently transfected with GFP-Trim32, and then the endogenous Pias3 was immumoprecipitated from the denatured cell lysates, followed by immunoblotting analysis for ubiquitin. As shown in <a href="#pone-0001085-g004">Fig 4<em>D</em></a>, both poly- and mono-ubiquitinated Pias3 were elevated in the presence of overexpressed Trim32. It should be pointed out that although mono-ubiquitination may not necessarily be required for regulation of protein stability, degradation of either actin or Piasy by Trim32 has been shown to correlate with mono-ubiquitination <a href="#pone.0001085-Albor1">[15]</a>, <a href="#pone.0001085-Kudryashova1">[17]</a>. Next, we explored the possible effect of Pias3 on Trim32, including autoubiquitination and potential sumoylation of Trim32. It turned out that Trim32 could not be sumoylated, even in the presence of overexpressed Pias3 (data not shown). Interestingly, the autoubiquitination of Trim32, an indicator of ubiquitin E3 ligase activity <a href="#pone.0001085-Chung1">[10]</a>, <a href="#pone.0001085-Albor1">[15]</a>, was enhanced by cotransfected Pias3 (<a href="#pone-0001085-g004">Fig 4<em>E</em></a>), indicating that association with Pias3 elevates the ligase activity of Trim32. Finally, we examined the possible regulatory role of NO on the Trim32-Pias3 interplay. It was observed in a GST pulldown assay that addition of GSNO markedly strengthened the Pias3-Trim32 association (<a href="#pone-0001085-g004">Fig 4<em>B</em></a>). A Co-IP experiment demonstrated that the affinity between Pias3 and Trim32 was substantially increased in HEK293 cells ectopicly expressing iNOS (<a href="#pone.0001085.s007">Fig <em>S7</em></a>), indicating that the increased association occurs in cells where endogenous NO is generated. As for the functional interplay between Pias3 and Trim32, both Trim32-mediated Pias3 ubiquitination and Pias3-stimulated autoubiquitination of Trim32 were found to be substantially enhanced in the presence of GSNO (<a href="#pone-0001085-g004">Fig 4<em>D, E</em></a>). Together, these observations indicate that NO promotes the association and bidirectional influence between Trim32 and Pias3, which may facilitate Trim32-mediated Pias3 ubiquitination.</p>


<h4>S-nitrosation of Pias3 positively regulates Pias3-Trim32 association</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Since S-nitrosation has been reported to facilitate protein-protein interactions <a href="#pone.0001085-Hara1">[9]</a>, we speculated that NO-mediated specific molecular modification of Pias3 (or Trim32) might regulate its interaction with Trim32 (or Pias3). Using the biotin-switch method, we verified that Pias3, as well as other members of the Pias family, could be efficiently S-nitrosated (<a href="#pone-0001085-g005">Fig 5<em>A</em></a> and <a href="#pone.0001085.s008">Fig <em>S8</em></a>). The absence of S-nitrosated Trim32 under GSNO treatment revealed the specificity of the above modifications (<a href="#pone-0001085-g005">Fig 5<em>A</em></a>). Furthermore, Pias3 was found to be constitutively S-nitrosated in HEK293 cells ectopicly expressing iNOS (<a href="#pone.0001085.s009">Fig <em>S9</em></a>), indicating a role of endogenously produced NO in modifying Pias3. To locate the S-nitrosated cysteine residue(s) of Pias3, recombinant Pias3 protein was exposed to GSNO, blocked with MMTS, and then subjected to mass spectrometry analysis. As shown in <a href="#pone-0001085-g005">Fig 5<em>B</em></a> and <a href="#pone.0001085.s010">Fig <em>S10</em></a>, the Cys459 of Pias3 was oxidized to sulphonic acid (-SO<sub>3</sub>H) only after exposure to GSNO. Considering that S-nitrosation can facilitate further oxidation of the same cysteine thiol <a href="#pone.0001085-Hara1">[9]</a>, <a href="#pone.0001085-Uehara1">[18]</a>, and Cys459 is located within a hydrophobic pocket (<a href="#pone.0001085.s009">Fig <em>S9</em></a>), a factor known to govern S-nitrosation specificity <a href="#pone.0001085-Greco1">[19]</a>, this result implies that Cys459 is a S-nitrosation site. To further confirm this, HEK293-expressed Pias3 mutant (C459S), whose cysteine 459 was mutated to serine, was subjected to the biotin-switch assay. As expected, mutation of Pias3 at Cys459 substantially diminished its S-nitrosation (<a href="#pone.0001085.s011">Fig <em>S11</em></a>). Next, we investigated whether S-nitrosation of Pias3 regulated its association with Trim32. It was demonstrated by GST-pull down assay that although the Pias3 bearing a C459S mutation still retained the ability to associate with Trim32 in the absence of GSNO, the stimulatory effect of GSNO on Pias3(C459S)-Trim32 association disappeared (<a href="#pone-0001085-g005">Fig 5<em>C</em></a>). In addition, the GSNO-enhanced binding of wild-type Pias3 to Trim32 could be abolished by dithiothreitol (DTT), a nitrosothiol-reducing reagent (data not shown). Together, these results indicate that S-nitrosation of Pias3 at Cys459 is required for positive tuning of the Pias3-Trim32 association.</p>
<div class="figure" id="pone-0001085-g005"><div class="img"><a name="pone-0001085-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>S-nitrosation of Pias3 and its effects on Pias3-Trim32 association.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A) S-nitrosation of GFP-Pias3 in HEK293 cells stimulated with 500 µM GSNO determined by biotin-switch assay. (B) Recombinant Pias3 was exposed to 100 µM GSNO and blocked with MMTS. In-gel digested Pias3 was subjected to Nano-LC-MS/MS analysis. (C) Effect of GSNO on Pias3(C459S)-Trim32 interaction determined by GST-pull down assay.</p>
<span>doi:10.1371/journal.pone.0001085.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Post-translational protein modification is a crucial mechanism for regulation of protein function in eukaryotic cells. S-nitrosation has gained increasing recognition as a functionally important post-translational modification, which exerts pleiotropic effects on its target proteins ranging from altered protein stabilization to changed activity or function <a href="#pone.0001085-Hess1">[7]</a>. Here we describe the S-nitrosation of Pias3 and show that this modification enhances its association with Trim32, a ubiquitin E3 ligase, which results in Pias3 degradation and repression of sumoylation (<a href="#pone-0001085-g006">Fig 6</a>). This work may have implications in three different conceptual areas. First, it reveals a unique mechanism to counteract the sumoylation pathway. Second, it provides new evidence that S-nitrosation regulates protein-protein interactions. Third, it reveals an unexpected crosstalk between S-nitrosation, sumoylation and ubiquitination.</p>
<div class="figure" id="pone-0001085-g006"><div class="img"><a name="pone-0001085-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001085" data-uri="info:doi/10.1371/journal.pone.0001085.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001085.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001085.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001085.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001085.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Model demonstrating that oxidative and nitrosative stress block the sumoylation pathway through different mechanisms.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Top: In a normal redox environment, Ubc9 conjugates SUMO to the substrate with the help of E3 ligase. Middle: Oxidative stress leads to formation of a disulfide bond between the E1 subunit Uba2 and the E2 subunit Ubc9, resulting in inactivation of both E1 and E2 enzymes. Below: Under nitrosative stress, both Ubc9 and Pias3 are S-nitrosated. Whereas S-nitrosation of Ubc9 cannot interfere with its catalytic activity, S-nitrosation of Pias3 facilitates its degradation by promoting its interplay with Ub E3 ligase Trim32, thereby resulting in decrease of SUMO conjugating efficiency. Su, SUMO; S, substrate protein.</p>
<span>doi:10.1371/journal.pone.0001085.g006</span></div><a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Firstly, we demonstrated a novel biological phenomenon, namely that NO led to desumoylation of most SUMO targets, which was due to at least in part, destabilization of SUMO E3 ligase Pias3. Unlike the effects of low concentrations of H<sub>2</sub>O<sub>2 </sub>or the virus protein Gam1, both of which target SUMO E1 and E2 enzymes, NO failed to interfere with Ubc9-mediated transfer of SUMO from Uba2 to the target protein, although it could S-nitrosate Ubc9 at a non-catalytic cysteine. Despite targeting E3, the efficiency of NO in the repression of global sumoylation is high. The effect obtained with 500 µM GSNO was comparable to that of 1 mM H<sub>2</sub>O<sub>2</sub>, a much stronger oxidant. Moreover, the endogenous NO produced in LPS-stimulated macrophages demonstrated a slight inhibitory effect on overall sumoylation, an effect that is however more significant than that obtained with endogenous H<sub>2</sub>O<sub>2</sub> from activated NADPH oxidase <a href="#pone.0001085-Bossis1">[6]</a>. It should be pointed out that, although S-nitrosation-dependent degradation of Pias3 contributes to loss of SUMO conjugates (<a href="#pone-0001085-g003">Fig.3<em>B</em></a>), global sumoylation profile may be controlled by the net effect of NO on multiple protein targets. For example, Pias family members including Pias3, Piasxα, Piasxβ and Piasy were all found as S-nitrosated targets (<a href="#pone.0001085.s008">Fig <em>S8</em></a>). In addition, most of SUMO isopeptidases (such as Senp) contain redox-sensitive cysteine(s). If these isopeptidases can be inhibited by higher concentrations of NO, one may understand why high concentration of GSNO showed a stimulatory effect on overall sumoylation <a href="#pone.0001085-Bossis1">[6]</a>. In fact, it is not contradictory that various SUMO-related enzymes coordinately respond to certain signal and control sumoylation. This is indeed evidenced by a recent report that the cAMP pathway repressed global sumoylation in human endometrial stromal cells by differentially affecting the protein contents of various SUMO E3 ligases and isopeptidases <a href="#pone.0001085-Jones1">[20]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Concerning the molecular mechanism by which NO promotes Pias3 degradation, we observed the S-nitrosation-stimulated recruitment of ubiquitin E3 ligase to Pias3. This biological phenomenon reflects S-nitrosation-tuned protein-protein interactions. In our study, S-nitrosation of a single cysteine C459 was found to be sufficient to mediate regulation of association of Pias3 and Trim32. However, C459 is not located in the N-terminal region of Pias3 that harbors the Trim32-binding site (data not shown), suggesting that S-nitrosation probably regulates Pias3-Trim32 interaction by inducing a conformational change in Pias3. At this point, our finding seems similar to a report that S-nitrosation of GAPDH at a single cysteine stimulates its association with Siah1, a ubiquitin E3 ligase <a href="#pone.0001085-Hara1">[9]</a>. In the present study, NO enhanced Pias3-Trim32 affinity but did not affect the intranuclear localization of Pias3, suggesting that ubiquitination of Pias3 might occur in the nucleus. During preparation of this manuscript, Depaux et al reported an intranuclear degradation of Pias proteins by Siah2, another ubiquitin E3 ligase <a href="#pone.0001085-Depaux1">[16]</a>. In cells coexpressing Pias1 and Siah2, Siah2 was translocated from the cytosol to the nucleus where it colocalized with Pias1, and was distributed into cytosolic aggregates <a href="#pone.0001085-Depaux1">[16]</a>. This report is highly consistent with our finding, and suggests a possibility that NO may also stimulate the interplay of Pias3 with other ubiquitin E3 ligases. Unraveling the effect of NO in the regulation of the Siah2-Pias pathway would be an interesting topic for future study.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Although SUMO and ubiquitin pathways are highly similar, little is known about signal-directed coupling of the two systems. We show here that NO links ubiquitination to sumoylation by promoting the binding of ubiquitin and SUMO E3 ligases, a process intergrating three kinds of post-translational modifications: S-nitrosation, sumoylation and ubiquitination. Recently, a crosstalk between S-nitrosation and ubiquitination has been reported <a href="#pone.0001085-Chung1">[10]</a>. NO was shown to inhibit the ubiquitin E3 ligase activity of Parkin through S-nitrosation <a href="#pone.0001085-Chung1">[10]</a>. By such mechanism, NO might cause disappearance of the bulk of ubiquitin conjugates, as observed in <a href="#pone-0001085-g001">Fig 1<em>C</em></a>. In fact, based on a similar idea, we have investigated the possible influence of S-nitrosation on the SUMO E3 ligase activity of Pias3; however, the result was negative (data not shown). The insensitivity of Pias3 activity to NO is consistent with the fact that the C459 of Pias3 that is S-nitrosated does not reside in the catalytic RING domain. Therefore, we speculate that S-nitrosation probably serves mainly as a negative regulatory mechanism to control the stability of Pias3 and thereby tune the sumoylation pathway. However, it should not be ignored that Pias3 protein itself is a negative regulator of pro-inflammatory gene transcription <a href="#pone.0001085-Jang1">[21]</a>, <a href="#pone.0001085-Chung2">[22]</a>. Thus, Pias3 may also serve as an intranuclear NO sensor, whose degradation represents a step to turn on inflammatory gene expression under nitrosative stress.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In summary, we report here that NO regulates sumoylation through S-nitrosation-dependent degradation of Pias3. Since SUMO modification participates in the regulation of multiple protein functions, repression of sumoylation may mediate a number of NO-related cellular events and pathological processes. For instance, desumoylation of PML and SP3 may be involved in NO-elicited transcriptional regulation. In addition, NO has been shown to desumoylate DJ-1 in neural cells, whose sumoylation is required for the full protective functions of DJ-1 (unpublished observation). Therefore, identifying proteins whose SUMO modification is reduced under nitrosative stress and unraveling the related functional consequences presents an exciting and important challenge for future research.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Reagents, plasmids and antibodies</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Lipopolysaccharide (LPS), S-methyl thiocarbamate (SMT), N-ethylmaleimide (NEM), iodoacetamide, and MG132 were purchased from Sigma. Methyl methanethionsulphonate (MMTS) and <em>N</em>-[6-(biotinamido)hexyl]-3′-(2′-pyridyldi​thio)propionamide (Biotin-HPDP) were purchased from Pierce. Other reagents have previously been described <a href="#pone.0001085-Qu1">[8]</a>. The following expression plasmids were described in previous investigations: HA-SUMO1(GG), SUMO2(GG), and SUMO3(GG) <a href="#pone.0001085-Tatham1">[23]</a>, HA-Ubc9 <a href="#pone.0001085-Gostissa1">[24]</a>, p53-Ubc9 <a href="#pone.0001085-Jakobs1">[12]</a>, Flag-Uba2 <a href="#pone.0001085-Bossis1">[6]</a>, GFP-SENP2 <a href="#pone.0001085-Hang1">[13]</a>, GFP-Pias3 <a href="#pone.0001085-Lyst1">[25]</a>, YFP-Ubc9 <a href="#pone.0001085-Kagey1">[4]</a>, Myc-Pias3 <a href="#pone.0001085-Matsuura1">[26]</a>, GFP-Trim32 <a href="#pone.0001085-Kudryashova1">[17]</a>, iNOS <a href="#pone.0001085-NavarroLerida1">[27]</a>, HA-Pias3, and GST-Pias3 <a href="#pone.0001085-Man1">[28]</a>. Mutants of Ubc9 and Pias3 were engineered by site-directed mutagenesis (Stratagene). Myc, HA, Flag, His, Ub, p53, Pias3, SUMO1, GFP monoclonal antibodies and Ubc9, Pias3, RanBP2, Senp2, Trim32, GFP, actin polyclonal antibodies were purchased from Santa Cruz. SUMO1 and Uba2 monoclonal antibodies were purchased from Zymed and BD Pharmingin, respectively. iNOS polyclonal antibody was from BD Pharmingin.</p>


<h4>Cell transfection and immunoblotting</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>HeLa and HEK293 cells were respectively transfected with lipofectamine 2000 (Invitrogen) and JetPei (Polyplus) according to the manufacturer's protocol. For sumoylation analysis, cells were lysed directly in Laemmli buffer supplemented with 20 mM NEM and iodoacetamide. Extracts were boiled for 30 min prior to loading on SDS-polyacrylamide gels. Immunoblotting analysis was performed as previously described <a href="#pone.0001085-Liu1">[29]</a>.</p>


<h4>Ubiquitination analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Cells were transfected with His-Ub together with other expression plasmids. 36 h after transfection, cells were incubated with MG132 and then lysed in 1 ml of lysis buffer (6 M guanidinium HCl, 0.1 M Na<sub>2</sub>HPO<sub>4</sub>/NaH<sub>2</sub>PO<sub>4</sub>, and 10 mM Tris-HCl, pH 8.0). After sonication and centrifugation, 90% of the lysate was incubated with 25 µl of Ni<sup>2+</sup> Sepharose beads (Amersham) for 3 h. The beads were washed twice with lysis buffer, followed by washing three times with washing buffer (8 M urea, 0.1 M Na<sub>2</sub>HPO<sub>4</sub>/NaH<sub>2</sub>PO<sub>4, </sub>pH 6.4). After a final wash with phosphate-buffered saline, the beads were treated in SDS sample buffer for SDS-PAGE. 1/10th of the cells were lysed in RIPA and the lysates were used as input controls.</p>


<h4>RNA interference</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The non-targeting siRNA (siControl) we used was UUCUCCGAACGUGUCACGUTT. Two different siRNAs targeting human Pias3 were UUGGUCAUCUGAGUUCGGAdTdT (siPias3-1) <a href="#pone.0001085-Man1">[28]</a> and GGAGCCAAAUGUGAUUAUAdTdT (siPias3-2) <a href="#pone.0001085-Iwasaki1">[30]</a>. To evaluate the specificity of the above siRNA, the mismatched siRNAs (M-siRNA) with only a three-base difference were employed. M-siRNAs for siPIAS3(1) and siPIAS3(2) were UUGGUCAU<span class="underline">AGU</span>AGUUCGGAdTdT and GGAGCCAA<span class="underline">CGU</span>UGAUUAUAdTdT respectively. The siRNAs were transfected into HeLa cells using Lipofectamine 2000 according to the manufacturer's protocol. After 36 h, the cells were transfected again for another 36 h. The protein level of Pias3 was then monitored by immunoblotting.</p>


<h4>Immunofluorescence analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>HEK293 cells were transfected with Myc-Pias3 and GFP-Trim32 expression vectors. 4 h later, cells were incubated with or without LMB for 16 h. Immunofluorescence analysis was performed with αMyc primary antibody and Texas Red-conjugated secondary antibody, as previously described <a href="#pone.0001085-Qu1">[8]</a>. Pictures of cells mounted in ProLong Antifade (Molecular Probes) were taken with a laser-scanning confocal microscope (Olympus FV500, Tokyo, Japan), 60x PlanApo lens (aperture 1.4, oil), using FLUOVIEW software.</p>


<h4>Immunoprecipitations</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Cells were lysed in ice-cold lysis buffer (20 mM Tris-HCl (pH 7.5), 150 mM NaCl, 10 mM NaF, 1 mM EDTA, 10% glycerol, 0.1% NP-40, 10 µM MG132, and the protease inhibitor cocktail). For immunoprecipitation using harsh denaturing conditions (Ub assay), the final concentration of sodium dodecyl sulphate (SDS) in the lysis buffer was increased to 2% and after boiling for 5 min, lysates were diluted into lysis buffer lacking SDS to give a final concentration of 0.1% SDS. Corresponding agarose bead-conjugated antibodies were incubated with 500 µg cell lysate, and the immunoprecipitates were analyzed by immunoblotting with appropriate antibodies.</p>


<h4>GST pull-down assay</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>GFP-Trim32 protein, obtained from the whole cell lysate of transfected HEK293 cells, was mixed with GST or GST-Pias3 bound to Sepharose beads in 1ml of binding buffer (20 mM Tris-HCl (pH 7.5), 150 mM NaCl, 1 mM EDTA, 10% glycerol, 0.1% NP-40, 10 µM MG132, and the protease inhibitor cocktail). 500 µM GSNO was then added to the mixture and incubated with rotation for 2 h at room temperature in dark. The beads were washed three times with 1 ml binding buffer, separated by SDS–PAGE and analyzed by immunoblotting.</p>


<h4>Biotin-switch assay</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Cell lysates were prepared in lysis buffers (50 mM Tris (pH 7.5), 150 mM NaCl, 0.5% deoxycholic acid, 0.1% SDS, 1% NP-40, 0.1 mM neocuproine, and the protease inhibitor cocktail). The supernatant of cell lysis was analyzed for protein concentration by the BCA method and then adjusted to the appropriate concentration. Blocking buffer (2.5% SDS, 20 mM MMTS in HEN buffer) was mixed with the samples and incubated for 30 min at 50°C to block free thiol groups. After removing excess MMTS by acetone precipitation, nitrosothiols were then reduced to thiols and biotinylated by reducing buffer (1% SDS, 10 mM ascorbate, 4 mM Biotin-HPDP). The biotinylated proteins were then pulled down by streptavidin-agarose beads, eluted by SDS sample buffer and subjected to immunoblotting.</p>


<h4>Pias3 site identification by (nano) Liquid Chromatography-Mass Spectrometry</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Mass spectra were acquired after in-gel tryptic digestion of recombinant Pias3 on a Thermo LTQ linear trap instrument equipped with a Thermo nano electrospray source and a Thermo Surveyor pump and autosampler (Thermo Electron Corporation, San Jose, CA) according to a previously described method with minor modifications <a href="#pone.0001085-Hara1">[9]</a>, <a href="#pone.0001085-Uehara1">[18]</a>. Briefly, recombinant Pias3 was exposed to 100 µM of GSNO <em>in vitro</em> for 30 min at 37°C. Remaining free thiols in Pias3 were blocked with 20 mM methyl methanethiosulphonate (MMTS) in blocking buffer (25 mM Hepes, pH 7.7, 1 mM EDTA, 0.1 mM neocuproine, 2.5% SDS) at 50°C for 30 min with frequent vortexing followed by separation by non-reducing SDS-PAGE in the dark. The gel was stained with Commassie bright blue, cut and subjected to in-gel tryptic digestion. LC-MS/MS analyses were performed on a fused silica capillary (75 µm i.d.×12 cm) packed with Synergi Hydro-RP silica (reverse-phase, 4 μ, 80 Å, Phenomenex). Mobile phase A consisted of 5% acetonitrile/ 0.1% formic acid and mobile phase B was 80% acetonitrile/ 0.1% formic acid. Peptides were eluted initially with 100% A for 1 min, then 90% A for 5 min, then a linear gradient to 55% A by 60 min, then to 0% A at 70 min and held to 80 min, then to 100% A at 80.01 min and held until 90 min. MS/MS spectra were acquired by using a full scan followed by five MS/MS scans in a data-dependent mode. Precursors that were detected twice within 15 s were put on a dynamic exclusion list for a period of 60 s. MS/MS data were processed with in-house Bioworks (version 3.2, Thermo Electron Corporation, San Jose, CA) as described below: MS/MS spectra were extracted from the raw files by using SEQUEST with the following parameters; MW range: 250–4,000; threshold, 1,000; precursor mass tolerance: 2; group scan tolerance, 2; minimum group count, 1; precursor charge state, auto; MSn level, auto. Raw data were searched against NCBI human Refseq protein database (version: 2006.9.18) and filtered with the following criteria: Xcorr&gt;1.9 for singly charged, &gt;2.5 for doubly charged and &gt;3.75 for triply charged ions; Delta Cn&gt;0.1; RSp&lt;5; and preliminary score (Sp)&gt;500. Cysteines either modified by MMTS (+46), nitrosation (+29), sulphination (+32) or sulphonation (+48) and methionine oxidation (+16) were all specified as differential modifications. MS/MS spectra were manually validated by the following criteria: (i) a continuous b or y-ion series of at least five residues and (ii) the top three most intense fragment peaks assigned to either an a, b, y-ion, to an a, b, y-ion resulting from a neutral loss of water or ammonia, or to a multiply protonated fragment ion. All acquired data were further validated by Trans-Proteomics Pipeline (TPP, Institute of System Biology, Seattle) with PeptideProphet®(Keller et al., 2002) &gt;0.99 and ProteinProphet® = 1.0 as a threshold for peptides and protein validation, respectively (data not shown). The above results represent three independent experiments.</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>Materials used: MMTS (Pierce), sequencing-grade trypsin (Promega), formic acid and acetonitrile were HPLC-grade from J.T. Baker Chemicals. Synergi Hydro-RP silica (4 μ, 80 Å) was from Phenomenex.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001085"></div><a name="pone.0001085.s001" id="pone.0001085.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">HEK293 cells transfected with GFP-PML (left) or HA-SP3 (right) were treated with H2O2 (1 mM) for 1h or with GSNO (0.5 mM) for indicated time, lysed in Laemmli buffer and immunoblotted with anti-GFP or anti-HA.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.29 MB TIF)</p>
<a name="pone.0001085.s002" id="pone.0001085.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">HeLa cells were treated with increasing concentrations of GSNO for 4 h, lysed in Laemmli buffer, and immunoblotted with anti-SUMO1 or anti-actin.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.09 MB TIF)</p>
<a name="pone.0001085.s003" id="pone.0001085.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">HeLa cells transfected with HA-Ubc9 were incubated with 0.5 mM H2O2 for 30 min or with 0.5 mM GSNO for the indicated time, lysed in Laemmli Buffer with (lanes 3 and 10) or without (other lanes) DTT and immunoblotted with either anti-HA (left) or anti-Uba2 (right). Asterisk: unspecific crossreacting band.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.21 MB TIF)</p>
<a name="pone.0001085.s004" id="pone.0001085.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">HeLa cells were cotransfected with His-SUMO1 and increasing amounts (from 30 to 300ng) of HA-Ubc9(WT) or HA-Ubc9(C75S), treated with 0.5 mM GSNO for 4 h, lysed in Laemmli buffer, and immunoblotted with anti-His.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.39 MB TIF)</p>
<a name="pone.0001085.s005" id="pone.0001085.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">HEK293 cells were transfected with the indicated plasmids and subjected to immunofluorescent (for Flag-Uba2) or fluorescent protein imaging.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.26 MB TIF)</p>
<a name="pone.0001085.s006" id="pone.0001085.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Co-IP analysis of the Pias3-Trim32 interaction in MG132-treated HeLa cells using anti-Trim32 (or control IgG) as the IP antibody. No coimmunoprecipitation was found between Trim32 and RanBP2, indicating a specific interaction between Trim32 and Pias3.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.07 MB TIF)</p>
<a name="pone.0001085.s007" id="pone.0001085.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s007">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Co-IP analysis of Pias3-Trim32 interaction in transfected HEK293 cells. HEK293 cells were cotransfected with the indicated plasmids. 36h later, cells were treated with MG132 for 4h. The lysates were subjected to CO-IP assay using anti-Myc as the IP antibody. The expression of Myc-Pias3, GFP-Trim32 and/or iNOS in immunoprecipitates and cell lysates were monitored with the indicated antibodies.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.11 MB TIF)</p>
<a name="pone.0001085.s008" id="pone.0001085.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s008">Figure S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">S-nitrosation of various Flag-tagged Pias subtypes expressed in HEK293 cells in the presence of 0.5 mM GSNO was determined by biotin-switch assay.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.13 MB TIF)</p>
<a name="pone.0001085.s009" id="pone.0001085.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s009">Figure S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">S-Nitrosation of HA-Pias3 in iNOS-overexpressed HEK293 cells.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.05 MB TIF)</p>
<a name="pone.0001085.s010" id="pone.0001085.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s010">Figure S10. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Nano-LC-MS/MS analysis of Pias3 before and after 0.1 mM GSNO treatment indicating cysteine-459 (C459) on peptide 458HCSVTSAAIPALPGSK473 was the modified site of S-nitrosation. The m/z 795.22+ precursor ion corresponds to the Pias3 peptide 458HCSVTSAAIPALPGSK473 with cysteine sulphonation in the GSNO-treated sample. Cysteine sulphonation was only observed after exposure to the NO donor GSNO. Analysis of the informative fragment ions y14+, y15+ confirmed the presence of the sulphonated cysteine. Kyte-Doolittle hydropathic index plot from the region flanking the identified S-nitrosocysteine residue (arrow) also showed that Cys459 was located within a hydrophobic pocket. The hydropathy plot was constructed by using a window of 13 amino acids.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(1.73 MB TIF)</p>
<a name="pone.0001085.s011" id="pone.0001085.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s011">Figure S11. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">S-nitrosation of HA-Ubc9 or its C459S expressed in HEK293 cells in the presence of 0.5 mM GSNO was determined by biotin-switch assay.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.11 MB TIF)</p>
<a name="pone.0001085.s012" id="pone.0001085.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001085.s012">Movie S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">3D presentation of Trim32-Pias3 colocalization in the nucleus</p>
<p class="siDoi">doi:10.1371/journal.pone.0001085.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(10.30 MB AVI)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We particularly thank Drs. X.M. Zhang, R. Hay, G. Del Sal, R. Niedenthal, F. Melchior, M. Dasso, I. Stancheva, D. Wotton, M. Takahashi, M. Spencer, and I. Rodríguez-Crespo for help in providing plasmids, Prof. Sarah Perrett for editing the manuscript, Prof. Fuquan Yang for assistance with MS, Prof. Xun Shen for sharing materials, Ms. Yan Teng for technical assistance with microscopy.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: CC JQ GL. Performed the experiments: JQ GL KW PH PW JL XZ. Analyzed the data: CC JQ GL PH. Wrote the paper: CC GL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001085-Hay1" id="pone.0001085-Hay1"></a>Hay RT (2005) SUMO: a history of modification. Mol Cell  18: 1–12.  <ul class="find" data-citedArticleID="1009830" data-doi="10.1016/j.molcel.2005.03.012"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.03.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SUMO%3A+a+history+of+modification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SUMO%3A+a+history+of+modification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001085-Kotaja1" id="pone.0001085-Kotaja1"></a>Kotaja N, Karvonen U, Janne OA, Palvimo JJ (2002) PIAS proteins modulate transcription factors by functioning as SUMO-1 ligases. Mol Cell Biol  22: 5222–5234.  <ul class="find" data-citedArticleID="1009846" data-doi="10.1128/mcb.22.14.5222-5234.2002"><li><a href="http://dx.doi.org/10.1128/mcb.22.14.5222-5234.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PIAS+proteins+modulate+transcription+factors+by+functioning+as+SUMO-1+ligases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PIAS+proteins+modulate+transcription+factors+by+functioning+as+SUMO-1+ligases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001085-Pichler1" id="pone.0001085-Pichler1"></a>Pichler A, Gast A, Seeler JS, Dejean A, Melchior F (2002) The nucleoporin RanBP2 has SUMO1 E3 ligase activity. Cell  108: 109–120.  <ul class="find" data-citedArticleID="1009860" data-doi="10.1016/s0092-8674(01)00633-x"><li><a href="http://dx.doi.org/10.1016/s0092-8674(01)00633-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+nucleoporin+RanBP2+has+SUMO1+E3+ligase+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+nucleoporin+RanBP2+has+SUMO1+E3+ligase+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001085-Kagey1" id="pone.0001085-Kagey1"></a>Kagey MH, Melhuish TA, Wotton D (2003) The polycomb protein Pc2 is a SUMO E3. Cell  113: 127–137.  <ul class="find" data-citedArticleID="1009844" data-doi="10.1016/s0092-8674(03)00159-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)00159-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+polycomb+protein+Pc2+is+a+SUMO+E3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+polycomb+protein+Pc2+is+a+SUMO+E3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001085-Boggio1" id="pone.0001085-Boggio1"></a>Boggio R, Colombo R, Hay RT, Draetta GF, Chiocca S (2004) A mechanism for inhibiting the SUMO pathway. Mol Cell  16: 549–561.  <ul class="find" data-citedArticleID="1009812" data-doi="10.1016/j.molcel.2004.11.007"><li><a href="http://dx.doi.org/10.1016/j.molcel.2004.11.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mechanism+for+inhibiting+the+SUMO+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mechanism+for+inhibiting+the+SUMO+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001085-Bossis1" id="pone.0001085-Bossis1"></a>Bossis G, Melchior F (2006) Regulation of SUMOylation by reversible oxidation of SUMO conjugating enzymes. Mol Cell  21: 349–357.  <ul class="find" data-citedArticleID="1009814" data-doi="10.1016/j.molcel.2005.12.019"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.12.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+SUMOylation+by+reversible+oxidation+of+SUMO+conjugating+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+SUMOylation+by+reversible+oxidation+of+SUMO+conjugating+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001085-Hess1" id="pone.0001085-Hess1"></a>Hess DT, Matsumoto A, Kim SO, Marshall HE, Stamler JS (2005) Protein S-nitrosylation: purview and parameters. Nat Rev Mol Cell Biol  6: 150–166.  <ul class="find" data-citedArticleID="1009832" data-doi="10.1038/nrm1569"><li><a href="http://dx.doi.org/10.1038/nrm1569" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+S-nitrosylation%3A+purview+and+parameters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+S-nitrosylation%3A+purview+and+parameters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001085-Qu1" id="pone.0001085-Qu1"></a>Qu J, Liu GH, Huang B, Chen C (2007) Nitric oxide controls nuclear export of APE1/Ref-1 through S-nitrosation of Cysteines 93 and 310. Nucleic Acids Res  35: 2522–2532.  <ul class="find" data-citedArticleID="1009862" data-doi="10.1093/nar/gkl1163"><li><a href="http://dx.doi.org/10.1093/nar/gkl1163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nitric+oxide+controls+nuclear+export+of+APE1%2FRef-1+through+S-nitrosation+of+Cysteines+93+and+310." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nitric+oxide+controls+nuclear+export+of+APE1%2FRef-1+through+S-nitrosation+of+Cysteines+93+and+310.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001085-Hara1" id="pone.0001085-Hara1"></a>Hara MR, Agrawal N, Kim SF, Cascio MB, Fujimuro M, et al.  (2005) S-nitrosylated GAPDH initiates apoptotic cell death by nuclear translocation following Siah1 binding. Nat Cell Biol  7: 665–674.  <ul class="find" data-citedArticleID="1009828" data-doi="10.1038/ncb1268"><li><a href="http://dx.doi.org/10.1038/ncb1268" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=S-nitrosylated+GAPDH+initiates+apoptotic+cell+death+by+nuclear+translocation+following+Siah1+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22S-nitrosylated+GAPDH+initiates+apoptotic+cell+death+by+nuclear+translocation+following+Siah1+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001085-Chung1" id="pone.0001085-Chung1"></a>Chung KK, Thomas B, Li X, Pletnikova O, Troncoso JC, et al.  (2004) S-nitrosylation of parkin regulates ubiquitination and compromises parkin's protective function. Science  304: 1328–1331.  <ul class="find" data-citedArticleID="1009816" data-doi="10.1126/science.1093891"><li><a href="http://dx.doi.org/10.1126/science.1093891" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=S-nitrosylation+of+parkin+regulates+ubiquitination+and+compromises+parkin%27s+protective+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22S-nitrosylation+of+parkin+regulates+ubiquitination+and+compromises+parkin%27s+protective+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001085-Jaffrey1" id="pone.0001085-Jaffrey1"></a>Jaffrey SR, Snyder SH (2001) The biotin switch method for the detection of S-nitrosylated proteins. Sci STKE  2001: L1.  <ul class="find" data-citedArticleID="1009836" data-doi="10.1126/stke.2001.86.pl1"><li><a href="http://dx.doi.org/10.1126/stke.2001.86.pl1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+biotin+switch+method+for+the+detection+of+S-nitrosylated+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+biotin+switch+method+for+the+detection+of+S-nitrosylated+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001085-Jakobs1" id="pone.0001085-Jakobs1"></a>Jakobs A, Koehnke J, Himstedt F, Funk M, Korn B, et al.  (2007) Ubc9 fusion-directed SUMOylation (UFDS): a method to analyze function of protein SUMOylation. Nat Methods  4: 245–250.  <ul class="find" data-citedArticleID="1009838" data-doi="10.1038/nmeth1006"><li><a href="http://dx.doi.org/10.1038/nmeth1006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ubc9+fusion-directed+SUMOylation+%28UFDS%29%3A+a+method+to+analyze+function+of+protein+SUMOylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ubc9+fusion-directed+SUMOylation+%28UFDS%29%3A+a+method+to+analyze+function+of+protein+SUMOylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001085-Hang1" id="pone.0001085-Hang1"></a>Hang J, Dasso M (2002) Association of the human SUMO-1 protease SENP2 with the nuclear pore. J Biol Chem  277: 19961–19966.  <ul class="find" data-citedArticleID="1009826" data-doi="10.1074/jbc.m201799200"><li><a href="http://dx.doi.org/10.1074/jbc.m201799200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+of+the+human+SUMO-1+protease+SENP2+with+the+nuclear+pore." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+of+the+human+SUMO-1+protease+SENP2+with+the+nuclear+pore.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001085-Saitoh1" id="pone.0001085-Saitoh1"></a>Saitoh H, Pizzi MD, Wang J (2002) Perturbation of SUMOlation enzyme Ubc9 by distinct domain within nucleoporin RanBP2/Nup358. J Biol Chem  277: 4755–4763.  <ul class="find" data-citedArticleID="1009864" data-doi="10.1074/jbc.m104453200"><li><a href="http://dx.doi.org/10.1074/jbc.m104453200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Perturbation+of+SUMOlation+enzyme+Ubc9+by+distinct+domain+within+nucleoporin+RanBP2%2FNup358." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Perturbation+of+SUMOlation+enzyme+Ubc9+by+distinct+domain+within+nucleoporin+RanBP2%2FNup358.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001085-Albor1" id="pone.0001085-Albor1"></a>Albor A, El-Hizawi S, Horn EJ, Laederich M, Frosk P, et al.  (2006) The interaction of Piasy with Trim32, an E3-ubiquitin ligase mutated in limb-girdle muscular dystrophy type 2H, promotes Piasy degradation and regulates UVB-induced keratinocyte apoptosis through NFkappaB. J Biol Chem  281: 25850–25866.  <ul class="find" data-citedArticleID="1009810" data-doi="10.1074/jbc.m601655200"><li><a href="http://dx.doi.org/10.1074/jbc.m601655200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+interaction+of+Piasy+with+Trim32%2C+an+E3-ubiquitin+ligase+mutated+in+limb-girdle+muscular+dystrophy+type+2H%2C+promotes+Piasy+degradation+and+regulates+UVB-induced+keratinocyte+apoptosis+through+NFkappaB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+interaction+of+Piasy+with+Trim32%2C+an+E3-ubiquitin+ligase+mutated+in+limb-girdle+muscular+dystrophy+type+2H%2C+promotes+Piasy+degradation+and+regulates+UVB-induced+keratinocyte+apoptosis+through+NFkappaB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001085-Depaux1" id="pone.0001085-Depaux1"></a>Depaux A, Regnier-Ricard F, Germani A, Varin-Blank N (2007) A crosstalk between hSiah2 and Pias E3-ligases modulates Pias-dependent activation. Oncogene.   <ul class="find" data-citedArticleID="1009820" data-doi="10.1038/sj.onc.1210486"><li><a href="http://dx.doi.org/10.1038/sj.onc.1210486" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+crosstalk+between+hSiah2+and+Pias+E3-ligases+modulates+Pias-dependent+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+crosstalk+between+hSiah2+and+Pias+E3-ligases+modulates+Pias-dependent+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001085-Kudryashova1" id="pone.0001085-Kudryashova1"></a>Kudryashova E, Kudryashov D, Kramerova I, Spencer MJ (2005) Trim32 is a ubiquitin ligase mutated in limb girdle muscular dystrophy type 2H that binds to skeletal muscle myosin and ubiquitinates actin. J Mol Biol  354: 413–424.  <ul class="find" data-citedArticleID="1009848" data-doi="10.1016/j.jmb.2005.09.068"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.09.068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trim32+is+a+ubiquitin+ligase+mutated+in+limb+girdle+muscular+dystrophy+type+2H+that+binds+to+skeletal+muscle+myosin+and+ubiquitinates+actin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trim32+is+a+ubiquitin+ligase+mutated+in+limb+girdle+muscular+dystrophy+type+2H+that+binds+to+skeletal+muscle+myosin+and+ubiquitinates+actin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001085-Uehara1" id="pone.0001085-Uehara1"></a>Uehara T, Nakamura T, Yao D, Shi ZQ, Gu Z, et al.  (2006) S-nitrosylated protein-disulphide isomerase links protein misfolding to neurodegeneration. Nature  441: 513–517.  <ul class="find" data-citedArticleID="1009868" data-doi="10.1038/nature04782"><li><a href="http://dx.doi.org/10.1038/nature04782" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=S-nitrosylated+protein-disulphide+isomerase+links+protein+misfolding+to+neurodegeneration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22S-nitrosylated+protein-disulphide+isomerase+links+protein+misfolding+to+neurodegeneration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001085-Greco1" id="pone.0001085-Greco1"></a>Greco TM, Hodara R, Parastatidis I, Heijnen HF, Dennehy MK, et al.  (2006) Identification of S-nitrosylation motifs by site-specific mapping of the S-nitrosocysteine proteome in human vascular smooth muscle cells. Proc Natl Acad Sci U S A  103: 7420–7425.  <ul class="find" data-citedArticleID="1009824" data-doi="10.1073/pnas.0600729103"><li><a href="http://dx.doi.org/10.1073/pnas.0600729103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+S-nitrosylation+motifs+by+site-specific+mapping+of+the+S-nitrosocysteine+proteome+in+human+vascular+smooth+muscle+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+S-nitrosylation+motifs+by+site-specific+mapping+of+the+S-nitrosocysteine+proteome+in+human+vascular+smooth+muscle+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001085-Jones1" id="pone.0001085-Jones1"></a>Jones MC, Fusi L, Higham JH, bdel-Hafiz H, Horwitz KB, et al.  (2006) Regulation of the SUMO pathway sensitizes differentiating human endometrial stromal cells to progesterone. Proc Natl Acad Sci U S A  103: 16272–16277.  <ul class="find" data-citedArticleID="1009842" data-doi="10.1073/pnas.0603002103"><li><a href="http://dx.doi.org/10.1073/pnas.0603002103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+the+SUMO+pathway+sensitizes+differentiating+human+endometrial+stromal+cells+to+progesterone." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+the+SUMO+pathway+sensitizes+differentiating+human+endometrial+stromal+cells+to+progesterone.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001085-Jang1" id="pone.0001085-Jang1"></a>Jang HD, Yoon K, Shin YJ, Kim J, Lee SY (2004) PIAS3 suppresses NF-kappaB-mediated transcription by interacting with the p65/RelA subunit. J Biol Chem  279: 24873–24880.  <ul class="find" data-citedArticleID="1009840" data-doi="10.1074/jbc.m313018200"><li><a href="http://dx.doi.org/10.1074/jbc.m313018200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PIAS3+suppresses+NF-kappaB-mediated+transcription+by+interacting+with+the+p65%2FRelA+subunit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PIAS3+suppresses+NF-kappaB-mediated+transcription+by+interacting+with+the+p65%2FRelA+subunit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001085-Chung2" id="pone.0001085-Chung2"></a>Chung CD, Liao J, Liu B, Rao X, Jay P, et al.  (1997) Specific inhibition of Stat3 signal transduction by PIAS3. Science  278: 1803–1805.  <ul class="find" data-citedArticleID="1009818" data-doi="10.1126/science.278.5344.1803"><li><a href="http://dx.doi.org/10.1126/science.278.5344.1803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+inhibition+of+Stat3+signal+transduction+by+PIAS3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+inhibition+of+Stat3+signal+transduction+by+PIAS3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001085-Tatham1" id="pone.0001085-Tatham1"></a>Tatham MH, Jaffray E, Vaughan OA, Desterro JM, Botting CH, et al.  (2001) Polymeric chains of SUMO-2 and SUMO-3 are conjugated to protein substrates by SAE1/SAE2 and Ubc9. J Biol Chem  276: 35368–35374.  <ul class="find" data-citedArticleID="1009866" data-doi="10.1074/jbc.m104214200"><li><a href="http://dx.doi.org/10.1074/jbc.m104214200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymeric+chains+of+SUMO-2+and+SUMO-3+are+conjugated+to+protein+substrates+by+SAE1%2FSAE2+and+Ubc9." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymeric+chains+of+SUMO-2+and+SUMO-3+are+conjugated+to+protein+substrates+by+SAE1%2FSAE2+and+Ubc9.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001085-Gostissa1" id="pone.0001085-Gostissa1"></a>Gostissa M, Hengstermann A, Fogal V, Sandy P, Schwarz SE, et al.  (1999) Activation of p53 by conjugation to the ubiquitin-like protein SUMO-1. EMBO J  18: 6462–6471.  <ul class="find" data-citedArticleID="1009822" data-doi="10.1093/emboj/18.22.6462"><li><a href="http://dx.doi.org/10.1093/emboj/18.22.6462" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+of+p53+by+conjugation+to+the+ubiquitin-like+protein+SUMO-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+of+p53+by+conjugation+to+the+ubiquitin-like+protein+SUMO-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001085-Lyst1" id="pone.0001085-Lyst1"></a>Lyst MJ, Nan X, Stancheva I (2006) Regulation of MBD1-mediated transcriptional repression by SUMO and PIAS proteins. EMBO J  25: 5317–5328.  <ul class="find" data-citedArticleID="1009852" data-doi="10.1038/sj.emboj.7601404"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+MBD1-mediated+transcriptional+repression+by+SUMO+and+PIAS+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+MBD1-mediated+transcriptional+repression+by+SUMO+and+PIAS+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001085-Matsuura1" id="pone.0001085-Matsuura1"></a>Matsuura T, Shimono Y, Kawai K, Murakami H, Urano T, et al.  (2005) PIAS proteins are involved in the SUMO-1 modification, intracellular translocation and transcriptional repressive activity of RET finger protein. Exp Cell Res  308: 65–77.  <ul class="find" data-citedArticleID="1009856" data-doi="10.1016/j.yexcr.2005.04.022"><li><a href="http://dx.doi.org/10.1016/j.yexcr.2005.04.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PIAS+proteins+are+involved+in+the+SUMO-1+modification%2C+intracellular+translocation+and+transcriptional+repressive+activity+of+RET+finger+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PIAS+proteins+are+involved+in+the+SUMO-1+modification%2C+intracellular+translocation+and+transcriptional+repressive+activity+of+RET+finger+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001085-NavarroLerida1" id="pone.0001085-NavarroLerida1"></a>Navarro-Lerida I, Corvi MM, Barrientos AA, Gavilanes F, Berthiaume LG, et al.  (2004) Palmitoylation of inducible nitric-oxide synthase at Cys-3 is required for proper intracellular traffic and nitric oxide synthesis. J Biol Chem  279: 55682–55689.  <ul class="find" data-citedArticleID="1009858" data-doi="10.1074/jbc.m406621200"><li><a href="http://dx.doi.org/10.1074/jbc.m406621200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Palmitoylation+of+inducible+nitric-oxide+synthase+at+Cys-3+is+required+for+proper+intracellular+traffic+and+nitric+oxide+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Palmitoylation+of+inducible+nitric-oxide+synthase+at+Cys-3+is+required+for+proper+intracellular+traffic+and+nitric+oxide+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001085-Man1" id="pone.0001085-Man1"></a>Man JH, Li HY, Zhang PJ, Zhou T, He K, et al.  (2006) PIAS3 induction of PRB sumoylation represses PRB transactivation by destabilizing its retention in the nucleus. Nucleic Acids Res  34: 5552–5566.  <ul class="find" data-citedArticleID="1009854" data-doi="10.1093/nar/gkl691"><li><a href="http://dx.doi.org/10.1093/nar/gkl691" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PIAS3+induction+of+PRB+sumoylation+represses+PRB+transactivation+by+destabilizing+its+retention+in+the+nucleus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PIAS3+induction+of+PRB+sumoylation+represses+PRB+transactivation+by+destabilizing+its+retention+in+the+nucleus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001085-Liu1" id="pone.0001085-Liu1"></a>Liu GH, Qu J, Shen X (2006) Thioredoxin-mediated negative autoregulation of peroxisome proliferator-activated receptor alpha transcriptional activity. Mol Biol Cell  17: 1822–1833.  <ul class="find" data-citedArticleID="1009850" data-doi="10.1091/mbc.e05-10-0979"><li><a href="http://dx.doi.org/10.1091/mbc.e05-10-0979" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thioredoxin-mediated+negative+autoregulation+of+peroxisome+proliferator-activated+receptor+alpha+transcriptional+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thioredoxin-mediated+negative+autoregulation+of+peroxisome+proliferator-activated+receptor+alpha+transcriptional+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001085-Iwasaki1" id="pone.0001085-Iwasaki1"></a>Iwasaki K, Hailemariam K, Tsuji Y (2007) Protein inhibitor of activated STAT3 (PIAS3) interacts with activating transcription factor 1 (ATF1) and regulates the human ferritin H gene through an antioxidant responsive element. J Biol Chem  282: 22335–22343.  <ul class="find" data-citedArticleID="1009834" data-doi="10.1074/jbc.m701477200"><li><a href="http://dx.doi.org/10.1074/jbc.m701477200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+inhibitor+of+activated+STAT3+%28PIAS3%29+interacts+with+activating+transcription+factor+1+%28ATF1%29+and+regulates+the+human+ferritin+H+gene+through+an+antioxidant+responsive+element." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+inhibitor+of+activated+STAT3+%28PIAS3%29+interacts+with+activating+transcription+factor+1+%28ATF1%29+and+regulates+the+human+ferritin+H+gene+through+an+antioxidant+responsive+element.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.XML" value="85199"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.PDF" value="501068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g001.PNG_L" value="475405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g001.PNG_M" value="113624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g001.PNG_S" value="9201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g001.TIF" value="469078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g001.PNG_I" value="35000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g002.PNG_L" value="274618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g002.PNG_M" value="68845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g002.PNG_S" value="7249"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g002.TIF" value="289234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g002.PNG_I" value="22159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g003.PNG_L" value="195995"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g003.PNG_M" value="65481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g003.PNG_S" value="6832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g003.TIF" value="201342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g003.PNG_I" value="20868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g004.PNG_L" value="641792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g004.PNG_M" value="162880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g004.PNG_S" value="15914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g004.TIF" value="844310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g004.PNG_I" value="102768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g005.PNG_L" value="176326"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g005.PNG_M" value="61764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g005.PNG_S" value="9093"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g005.TIF" value="211476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g005.PNG_I" value="32254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g006.PNG_L" value="98823"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g006.PNG_M" value="57625"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g006.PNG_S" value="10088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g006.TIF" value="134484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.g006.PNG_I" value="19942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s001.TIF" value="294042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s002.TIF" value="93938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s003.TIF" value="215416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s004.TIF" value="393252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s005.TIF" value="256638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s006.TIF" value="71842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s007.TIF" value="108530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s008.TIF" value="131442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s009.TIF" value="51664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s010.TIF" value="1725828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s011.TIF" value="111134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001085.s012.AVI" value="10299904"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001085&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001085" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001085&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001085&volume=&issue=&title=Nitric Oxide Destabilizes Pias3 and Regulates Sumoylation&author_name=Jing%20Qu%2C%20Guang-Hui%20Liu%2C%20Kaiyuan%20Wu%2C%20Peiwei%20Han%2C%20Peng%20Wang%2C%20Jiangmei%20Li%2C%20Xu%20Zhang%2C%20Chang%20Chen&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001085" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001085" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001085" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001085&amp;t=Nitric%20Oxide%20Destabilizes%20Pias3%20and%20Regulates%20Sumoylation" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001085&title=Nitric%20Oxide%20Destabilizes%20Pias3%20and%20Regulates%20Sumoylation&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001085&amp;title=Nitric%20Oxide%20Destabilizes%20Pias3%20and%20Regulates%20Sumoylation" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001085" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001085&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Nitric Oxide Destabilizes Pias3 and Regulates Sumoylation';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001085';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Nitric%20Oxide%20Destabilizes%20Pias3%20and%20Regulates%20Sumoylation http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001085" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001085" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cysteine%22" title="Search for articles in the subject area:'Cysteine'"><div class="flagText">Cysteine</div></a>
              <div data-categoryid="40627" data-articleid="25004"
                   data-categoryname="Cysteine"
                   class="flagImage" title="Flag 'Cysteine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzymes%22" title="Search for articles in the subject area:'Enzymes'"><div class="flagText">Enzymes</div></a>
              <div data-categoryid="17863" data-articleid="25004"
                   data-categoryname="Enzymes"
                   class="flagImage" title="Flag 'Enzymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HeLa+cells%22" title="Search for articles in the subject area:'HeLa cells'"><div class="flagText">HeLa cells</div></a>
              <div data-categoryid="19567" data-articleid="25004"
                   data-categoryname="HeLa cells"
                   class="flagImage" title="Flag 'HeLa cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Immunoblotting%22" title="Search for articles in the subject area:'Immunoblotting'"><div class="flagText">Immunoblotting</div></a>
              <div data-categoryid="47655" data-articleid="25004"
                   data-categoryname="Immunoblotting"
                   class="flagImage" title="Flag 'Immunoblotting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ligases%22" title="Search for articles in the subject area:'Ligases'"><div class="flagText">Ligases</div></a>
              <div data-categoryid="18587" data-articleid="25004"
                   data-categoryname="Ligases"
                   class="flagImage" title="Flag 'Ligases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nitric+oxide%22" title="Search for articles in the subject area:'Nitric oxide'"><div class="flagText">Nitric oxide</div></a>
              <div data-categoryid="21699" data-articleid="25004"
                   data-categoryname="Nitric oxide"
                   class="flagImage" title="Flag 'Nitric oxide' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22SUMOylation%22" title="Search for articles in the subject area:'SUMOylation'"><div class="flagText">SUMOylation</div></a>
              <div data-categoryid="24427" data-articleid="25004"
                   data-categoryname="SUMOylation"
                   class="flagImage" title="Flag 'SUMOylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ubiquitination%22" title="Search for articles in the subject area:'Ubiquitination'"><div class="flagText">Ubiquitination</div></a>
              <div data-categoryid="21151" data-articleid="25004"
                   data-categoryname="Ubiquitination"
                   class="flagImage" title="Flag 'Ubiquitination' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=592'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=726'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7610&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
