

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Molecular Correlates of Host Specialization in Staphylococcus aureus</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001120"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001120" />

    <meta name="citation_title" content="Molecular Correlates of Host Specialization in Staphylococcus aureus"/>
    <meta itemprop="name" content="Molecular Correlates of Host Specialization in Staphylococcus aureus"/>

      <meta name="citation_author" content="Lisa Herron-Olson"/>
            <meta name="citation_author_institution" content="Department of Microbiology, Veterinary Pathobiology and Biomedical Genomics Center, University of Minnesota, Saint Paul, Minnesota, United States of America"/>
      <meta name="citation_author" content="J. Ross Fitzgerald"/>
            <meta name="citation_author_institution" content="Laboratory for Bacterial Evolution and Pathogenesis, University of Edinburgh, Edinburgh, Scotland, United Kingdom"/>
      <meta name="citation_author" content="James M. Musser"/>
            <meta name="citation_author_institution" content="Center for Molecular and Translational Human Infectious Disease Research, Methodist Hospital Research Institute, and Department of Pathology, Methodist Hospital, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Vivek Kapur"/>
            <meta name="citation_author_institution" content="Department of Microbiology, Veterinary Pathobiology and Biomedical Genomics Center, University of Minnesota, Saint Paul, Minnesota, United States of America"/>

    <meta name="citation_date" content="2007/10/31"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001120.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1120"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Key health issues for dairy cattle–new and old.; citation_author=SJ Wells; citation_author=SL Ott; citation_author=AH Seitzinger; citation_journal_title=J Dairy Sci; citation_volume=81; citation_number=1; citation_pages=3029-3035; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity and antimicrobial susceptibility profiles among mastitis-causing Staphylococcus aureus isolated from bovine milk samples.; citation_author=KL Anderson; citation_author=RL Lyman; citation_author=SM Bodeis-Jones; citation_author=DG White; citation_journal_title=Am J Vet Res; citation_volume=67; citation_number=2; citation_pages=1185-1191; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genotyping of Staphylococcus aureus isolated from humans, bovine subclinical mastitis and food samples in Argentina.; citation_author=EB Reinoso; citation_author=A El-Sayed; citation_author=C Lammler; citation_author=C Bogni; citation_author=M Zschock; citation_journal_title=Microbiol Res; citation_number=3; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Adherence of Staphylococcus aureus to cultured bovine mammary epithelial cells.; citation_author=E Cifrian; citation_author=AJ Guidry; citation_author=CN O'Brien; citation_author=SC Nickerson; citation_author=WW Marquardt; citation_journal_title=J Dairy Sci; citation_volume=77; citation_number=4; citation_pages=970-983; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Staphylococcus aureus strains isolated from bovine mastitis: virulence, antibody production and protection from challenge in a mouse model.; citation_author=G Leitner; citation_author=O Krifucks; citation_author=A Glickman; citation_author=A Younis; citation_author=A Saran; citation_journal_title=FEMS Immunol Med Microbiol; citation_volume=35; citation_number=5; citation_pages=99-106; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Leucotoxic activities of Staphylococcus aureus strains isolated from cows, ewes, and goats with mastitis: importance of LukM/LukF'-PV leukotoxin.; citation_author=P Rainard; citation_author=JC Corrales; citation_author=MB Barrio; citation_author=T Cochard; citation_author=B Poutrel; citation_journal_title=Clin Diagn Lab Immunol; citation_volume=10; citation_number=6; citation_pages=272-277; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phenotypic and genotypic characterization of bovine mastitis isolates of Staphylococcus aureus for biofilm formation.; citation_author=P Vasudevan; citation_author=MK Nair; citation_author=T Annamalai; citation_author=KS Venkitanarayanan; citation_journal_title=Vet Microbiol; citation_volume=92; citation_number=7; citation_pages=179-185; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Role of biofilm-associated protein bap in the pathogenesis of bovine Staphylococcus aureus.; citation_author=C Cucarella; citation_author=MA Tormo; citation_author=C Ubeda; citation_author=MP Trotonda; citation_author=M Monzon; citation_journal_title=Infect Immun; citation_volume=72; citation_number=8; citation_pages=2177-2185; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Factors involved in the early pathogenesis of bovine Staphylococcus aureus mastitis with emphasis on bacterial adhesion and invasion. A review.; citation_author=O Kerro Dego; citation_author=JE van Dijk; citation_author=H Nederbragt; citation_journal_title=Vet Q; citation_volume=24; citation_number=9; citation_pages=181-198; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A single clone of Staphylococcus aureus causes the majority of cases of toxic shock syndrome.; citation_author=JM Musser; citation_author=PM Schlievert; citation_author=AW Chow; citation_author=P Ewan; citation_author=BN Kreiswirth; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=87; citation_number=10; citation_pages=225-229; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Molecular population genetic analysis of Staphylococcus aureus recovered from cows.; citation_author=V Kapur; citation_author=WM Sischo; citation_author=RS Greer; citation_author=TS Whittam; citation_author=JM Musser; citation_journal_title=J Clin Microbiol; citation_volume=33; citation_number=11; citation_pages=376-380; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Fine-structure molecular epidemiological analysis of Staphylococcus aureus recovered from cows.; citation_author=JR Fitzgerald; citation_author=WJ Meaney; citation_author=PJ Hartigan; citation_author=CJ Smyth; citation_author=V Kapur; citation_journal_title=Epidemiol Infect; citation_volume=119; citation_number=12; citation_pages=261-269; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=RAPD-PCR analysis of Staphylococcus aureus strains isolated from bovine and human hosts.; citation_author=E Reinoso; citation_author=S Bettera; citation_author=C Frigerio; citation_author=M DiRenzo; citation_author=A Calzolari; citation_journal_title=Microbiol Res; citation_volume=159; citation_number=13; citation_pages=245-255; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Application of pulsed-field gel electrophoresis and binary typing as tools in veterinary clinical microbiology and molecular epidemiologic analysis of bovine and human Staphylococcus aureus isolates.; citation_author=R Zadoks; citation_author=W van Leeuwen; citation_author=H Barkema; citation_author=O Sampimon; citation_author=H Verbrugh; citation_journal_title=J Clin Microbiol; citation_volume=38; citation_number=14; citation_pages=1931-1939; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genetic analysis of natural populations of Staphylococcus aureus.; citation_number=15; citation_pages=59-67; citation_date=1990; citation_publisher=VCH Publishers; " />
      <meta name="citation_reference" content="citation_title=Acquisition of Mn(II) in addition to Fe(II) is required for full virulence of Salmonella enterica serovar Typhimurium.; citation_author=E Boyer; citation_author=I Bergevin; citation_author=D Malo; citation_author=P Gros; citation_author=MF Cellier; citation_journal_title=Infect Immun; citation_volume=70; citation_number=16; citation_pages=6032-6042; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Bacterial genotype affects the manifestation and persistence of bovine Staphylococcus aureus intramammary infection.; citation_author=M Haveri; citation_author=S Taponen; citation_author=J Vuopio-Varkila; citation_author=S Salmenlinna; citation_author=S Pyorala; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=17; citation_pages=959-961; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Whole genome sequencing of meticillin-resistant Staphylococcus aureus.; citation_author=M Kuroda; citation_author=T Ohta; citation_author=I Uchiyama; citation_author=T Baba; citation_author=H Yuzawa; citation_journal_title=Lancet; citation_volume=357; citation_number=18; citation_pages=1225-1240; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genome and virulence determinants of high virulence community-acquired MRSA.; citation_author=T Baba; citation_author=F Takeuchi; citation_author=M Kuroda; citation_author=H Yuzawa; citation_author=K Aoki; citation_journal_title=Lancet; citation_volume=359; citation_number=19; citation_pages=1819-1827; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Complete genomes of two clinical Staphylococcus aureus strains: evidence for the rapid evolution of virulence and drug resistance.; citation_author=MT Holden; citation_author=EJ Feil; citation_author=JA Lindsay; citation_author=SJ Peacock; citation_author=NP Day; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=20; citation_pages=9786-9791; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Insights on evolution of virulence and resistance from the complete genome analysis of an early methicillin-resistant Staphylococcus aureus strain and a biofilm-producing methicillin-resistant Staphylococcus epidermidis strain.; citation_author=SR Gill; citation_author=DE Fouts; citation_author=GL Archer; citation_author=EF Mongodin; citation_author=RT Deboy; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=21; citation_pages=2426-2438; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Design, validation, and application of a seven-strain Staphylococcus aureus PCR product microarray for comparative genomics.; citation_author=AA Witney; citation_author=GL Marsden; citation_author=MT Holden; citation_author=RA Stabler; citation_author=SE Husain; citation_journal_title=Appl Environ Microbiol; citation_volume=71; citation_number=22; citation_pages=7504-7514; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide molecular dissection of serotype M3 group A Streptococcus strains causing two epidemics of invasive infections.; citation_author=SB Beres; citation_author=GL Sylva; citation_author=DE Sturdevant; citation_author=CN Granville; citation_author=M Liu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=23; citation_pages=11833-11838; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolutionary genomics of Staphylococcus aureus: insights into the origin of methicillin-resistant strains and the toxic shock syndrome epidemic.; citation_author=JR Fitzgerald; citation_author=DE Sturdevant; citation_author=SM Mackie; citation_author=SR Gill; citation_author=JM Musser; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=24; citation_pages=8821-8826; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Characterization of a putative pathogenicity island from bovine Staphylococcus aureus encoding multiple superantigens.; citation_author=JR Fitzgerald; citation_author=SR Monday; citation_author=TJ Foster; citation_author=GA Bohach; citation_author=PJ Hartigan; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=25; citation_pages=63-70; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic locus for streptolysin S production by group A streptococcus.; citation_author=V Nizet; citation_author=B Beall; citation_author=DJ Bast; citation_author=V Datta; citation_author=L Kilburn; citation_journal_title=Infect Immun; citation_volume=68; citation_number=26; citation_pages=4245-4254; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of the genomes of the temperate bacteriophages phi 11, phi 12 and phi 13 of Staphylococcus aureus 8325.; citation_author=JJ Iandolo; citation_author=V Worrell; citation_author=KH Groicher; citation_author=Y Qian; citation_author=R Tian; citation_journal_title=Gene; citation_volume=289; citation_number=27; citation_pages=109-118; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions.; citation_author=M Nei; citation_author=T Gojobori; citation_journal_title=Mol Biol Evol; citation_volume=3; citation_number=28; citation_pages=418-426; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Three new members of the serine-aspartate repeat protein multigene family of Staphylococcus aureus.; citation_author=E Josefsson; citation_author=KW McCrea; citation_author=D Ni Eidhin; citation_author=D O'Connell; citation_author=J Cox; citation_journal_title=Microbiology 144 (Pt; citation_volume=12); citation_number=29; citation_pages=3387-3395; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Analysis of Ebh, a 1.1-megadalton cell wall-associated fibronectin-binding protein of Staphylococcus aureus.; citation_author=SR Clarke; citation_author=LG Harris; citation_author=RG Richards; citation_author=SJ Foster; citation_journal_title=Infect Immun; citation_volume=70; citation_number=30; citation_pages=6680-6687; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Characterization of leukocytotoxic and superantigen-like factors produced by Staphylococcus aureus isolates from milk of cows with mastitis.; citation_author=HJ Schuberth; citation_author=C Krueger; citation_author=H Zerbe; citation_author=E Bleckmann; citation_author=W Leibold; citation_journal_title=Vet Microbiol; citation_volume=82; citation_number=31; citation_pages=187-199; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cytotoxin and pyrogenic toxin superantigen gene profiles of Staphylococcus aureus associated with subclinical mastitis in dairy cows and relationships with macrorestriction genomic profiles.; citation_author=JM Fueyo; citation_author=MC Mendoza; citation_author=MR Rodicio; citation_author=J Muniz; citation_author=MA Alvarez; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=32; citation_pages=1278-1284; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Prophage, phiPV83-pro, carrying panton-valentine leukocidin genes, on the Staphylococcus aureus P83 chromosome: comparative analysis of the genome structures of phiPV83-pro, phiPVL, phi11, and other phages.; citation_author=D Zou; citation_author=J Kaneko; citation_author=S Narita; citation_author=Y Kamio; citation_journal_title=Biosci Biotechnol Biochem; citation_volume=64; citation_number=33; citation_pages=2631-2643; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Leukotoxin family genes in Staphylococcus aureus isolated from domestic animals and prevalence of lukM-lukF-PV genes by bacteriophages in bovine isolates.; citation_author=T Yamada; citation_author=N Tochimaru; citation_author=S Nakasuji; citation_author=E Hata; citation_author=H Kobayashi; citation_journal_title=Vet Microbiol; citation_volume=110; citation_number=34; citation_pages=97-103; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=LukM/LukF'-PV is the most active Staphylococcus aureus leukotoxin on bovine neutrophils.; citation_author=MB Barrio; citation_author=P Rainard; citation_author=G Prevost; citation_journal_title=Microbes Infect; citation_volume=8; citation_number=35; citation_pages=2068-2074; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Type 1 capsule genes of Staphylococcus aureus are carried in a staphylococcal cassette chromosome genetic element.; citation_author=TT Luong; citation_author=S Ouyang; citation_author=K Bush; citation_author=CY Lee; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=36; citation_pages=3623-3629; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comparison of seven antibiotic treatments with no treatment for bacteriological efficacy against bovine mastitis pathogens.; citation_author=DJ Wilson; citation_author=RN Gonzalez; citation_author=KL Case; citation_author=LL Garrison; citation_author=YT Grohn; citation_journal_title=J Dairy Sci; citation_volume=82; citation_number=37; citation_pages=1664-1670; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Trends in antibacterial susceptibility of mastitis pathogens during a seven-year period.; citation_author=RJ Erskine; citation_author=RD Walker; citation_author=CA Bolin; citation_author=PC Bartlett; citation_author=DG White; citation_journal_title=J Dairy Sci; citation_volume=85; citation_number=38; citation_pages=1111-1118; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A comparison of antimicrobial susceptibility patterns for Staphylococcus aureus in organic and conventional dairy herds.; citation_author=LL Tikofsky; citation_author=JW Barlow; citation_author=C Santisteban; citation_author=YH Schukken; citation_journal_title=Microb Drug Resist; citation_volume=9; citation_number=39; citation_pages=S39-45; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Staphylococcal iron requirements, siderophore production, and iron-regulated protein expression.; citation_author=JA Lindsay; citation_author=TV Riley; citation_journal_title=Infect Immun; citation_volume=62; citation_number=40; citation_pages=2309-2314; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Influence of iron depletion on growth kinetics, siderophore production, and protein expression of Staphylococcus aureus.; citation_author=D Trivier; citation_author=M Davril; citation_author=N Houdret; citation_author=RJ Courcol; citation_journal_title=FEMS Microbiol Lett; citation_volume=127; citation_number=41; citation_pages=195-199; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Iron depletion and virulence in Staphylococcus aureus.; citation_author=D Trivier; citation_author=RJ Courcol; citation_journal_title=FEMS Microbiol Lett; citation_volume=141; citation_number=42; citation_pages=117-127; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Passage of heme-iron across the envelope of Staphylococcus aureus.; citation_author=SK Mazmanian; citation_author=EP Skaar; citation_author=AH Gaspar; citation_author=M Humayun; citation_author=P Gornicki; citation_journal_title=Science; citation_volume=299; citation_number=43; citation_pages=906-909; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Iron-regulated surface determinants (Isd) of Staphylococcus aureus: stealing iron from heme.; citation_author=EP Skaar; citation_author=O Schneewind; citation_journal_title=Microbes Infect; citation_volume=6; citation_number=44; citation_pages=390-397; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Variation of the agr locus in Staphylococcus aureus isolates from cows with mastitis.; citation_author=S Takeuchi; citation_author=T Maeda; citation_author=N Hashimoto; citation_author=K Imaizumi; citation_author=T Kaidoh; citation_journal_title=Vet Microbiol; citation_volume=79; citation_number=45; citation_pages=267-274; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of agr locus diversification and overall genetic variability among bovine and human Staphylococcus aureus isolates.; citation_author=P Gilot; citation_author=W van Leeuwen; citation_journal_title=J Clin Microbiol; citation_volume=42; citation_number=46; citation_pages=1265-1269; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Regulation of exoprotein gene expression in Staphylococcus aureus by agar.; citation_author=P Recsei; citation_author=B Kreiswirth; citation_author=M O'Reilly; citation_author=P Schlievert; citation_author=A Gruss; citation_journal_title=Mol Gen Genet; citation_volume=202; citation_number=47; citation_pages=58-61; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Regulation of virulence determinants in Staphylococcus aureus.; citation_author=S Arvidson; citation_author=K Tegmark; citation_journal_title=Int J Med Microbiol; citation_volume=291; citation_number=48; citation_pages=159-170; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Synthesis of staphylococcal virulence factors is controlled by a regulatory RNA molecule.; citation_author=RP Novick; citation_author=HF Ross; citation_author=SJ Projan; citation_author=J Kornblum; citation_author=B Kreiswirth; citation_journal_title=Embo J; citation_volume=12; citation_number=49; citation_pages=3967-3975; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Bacterial interference caused by autoinducing peptide variants.; citation_author=G Ji; citation_author=R Beavis; citation_author=RP Novick; citation_journal_title=Science; citation_volume=276; citation_number=50; citation_pages=2027-2030; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Exfoliatin-producing strains define a fourth agr specificity group in Staphylococcus aureus.; citation_author=S Jarraud; citation_author=GJ Lyon; citation_author=AM Figueiredo; citation_author=L Gerard; citation_author=F Vandenesch; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=51; citation_pages=6517-6522; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The agr radiation: an early event in the evolution of staphylococci.; citation_author=JS Wright; citation_author=KE Traber; citation_author=R Corrigan; citation_author=SA Benson; citation_author=JM Musser; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=52; citation_pages=5585-5594; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microarrays reveal that each of the ten dominant lineages of Staphylococcus aureus has a unique combination of surface-associated and regulatory genes.; citation_author=JA Lindsay; citation_author=CE Moore; citation_author=NP Day; citation_author=SJ Peacock; citation_author=AA Witney; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=53; citation_pages=669-676; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Bacterial contingency loci: the role of simple sequence DNA repeats in bacterial adaptation.; citation_author=R Moxon; citation_author=C Bayliss; citation_author=D Hood; citation_journal_title=Annu Rev Genet; citation_volume=40; citation_number=54; citation_pages=307-333; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The simple sequence contingency loci of Haemophilus influenzae and Neisseria meningitidis.; citation_author=CD Bayliss; citation_author=D Field; citation_author=ER Moxon; citation_journal_title=J Clin Invest; citation_volume=107; citation_number=55; citation_pages=657-662; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The role of decay-accelerating factor as a receptor for Helicobacter pylori and a mediator of gastric inflammation.; citation_author=DP O'Brien; citation_author=DA Israel; citation_author=U Krishna; citation_author=J Romero-Gallo; citation_author=J Nedrud; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=56; citation_pages=13317-13323; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of Yersinia pestis, the causative agent of plague.; citation_author=J Parkhill; citation_author=BW Wren; citation_author=NR Thomson; citation_author=RW Titball; citation_author=MT Holden; citation_journal_title=Nature; citation_volume=413; citation_number=57; citation_pages=523-527; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Massive gene decay in the leprosy bacillus.; citation_author=ST Cole; citation_author=K Eiglmeier; citation_author=J Parkhill; citation_author=KD James; citation_author=NR Thomson; citation_journal_title=Nature; citation_volume=409; citation_number=58; citation_pages=1007-1011; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The decaying genome of Mycobacterium leprae.; citation_author=K Eiglmeier; citation_author=J Parkhill; citation_author=N Honore; citation_author=T Garnier; citation_author=F Tekaia; citation_journal_title=Lepr Rev; citation_volume=72; citation_number=59; citation_pages=387-398; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Intracellular Staphylococcus aureus escapes the endosome and induces apoptosis in epithelial cells.; citation_author=KW Bayles; citation_author=CA Wesson; citation_author=LE Liou; citation_author=LK Fox; citation_author=GA Bohach; citation_journal_title=Infect Immun; citation_volume=66; citation_number=60; citation_pages=336-342; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=In vivo and in vitro demonstration that Staphylococcus aureus is an intracellular pathogen in the presence or absence of fibronectin-binding proteins.; citation_author=E Brouillette; citation_author=G Grondin; citation_author=L Shkreta; citation_author=P Lacasse; citation_author=BG Talbot; citation_journal_title=Microb Pathog; citation_volume=35; citation_number=61; citation_pages=159-168; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Is Staphylococcus aureus an intracellular pathogen?; citation_author=FD Lowy; citation_journal_title=Trends Microbiol; citation_volume=8; citation_number=62; citation_pages=341-343; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Survival of Staphylococcus aureus inside neutrophils contributes to infection.; citation_author=HD Gresham; citation_author=JH Lowrance; citation_author=TE Caver; citation_author=BS Wilson; citation_author=AL Cheung; citation_journal_title=J Immunol; citation_volume=164; citation_number=63; citation_pages=3713-3722; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Production and characterization of chimeric transferrins for the determination of the binding domains for bacterial transferrin receptors.; citation_author=MD Retzer; citation_author=A Kabani; citation_author=LL Button; citation_author=RH Yu; citation_author=AB Schryvers; citation_journal_title=J Biol Chem; citation_volume=271; citation_number=64; citation_pages=1166-1173; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The role of transferrin-receptor variation in the host range of Trypanosoma brucei.; citation_author=W Bitter; citation_author=H Gerrits; citation_author=R Kieft; citation_author=P Borst; citation_journal_title=Nature; citation_volume=391; citation_number=65; citation_pages=499-502; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genome sequence survey identifies unique sequences and key virulence genes with unusual rates of amino Acid substitution in bovine Staphylococcus aureus.; citation_author=LL Herron; citation_author=R Chakravarty; citation_author=C Dwan; citation_author=JR Fitzgerald; citation_author=JM Musser; citation_journal_title=Infect Immun; citation_volume=70; citation_number=66; citation_pages=3978-3981; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities.; citation_author=B Ewing; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=67; citation_pages=186-194; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=MGView: an alignment and visualization tool to enhance gap closure of microbial genomes.; citation_author=L Herron-Olson; citation_author=J Freeman; citation_author=Q Zhang; citation_author=EF Retzel; citation_author=V Kapur; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=68; citation_pages=e106; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Consed: a graphical tool for sequence finishing.; citation_author=D Gordon; citation_author=C Abajian; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=69; citation_pages=195-202; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Viewing and annotating sequence data with Artemis.; citation_author=M Berriman; citation_author=K Rutherford; citation_journal_title=Brief Bioinform; citation_volume=4; citation_number=70; citation_pages=124-132; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Translational control of prokaryotic gene expression.; citation_author=JE McCarthy; citation_author=C Gualerzi; citation_journal_title=Trends Genet; citation_volume=6; citation_number=71; citation_pages=78-85; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Basic local alignment search tool.; citation_author=SF Altschul; citation_author=W Gish; citation_author=W Miller; citation_author=EW Myers; citation_author=DJ Lipman; citation_journal_title=J Mol Biol; citation_volume=215; citation_number=72; citation_pages=403-410; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=73; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Multilocus sequence typing for characterization of methicillin-resistant and methicillin-susceptible clones of Staphylococcus aureus.; citation_author=MC Enright; citation_author=NP Day; citation_author=CE Davies; citation_author=SJ Peacock; citation_author=BG Spratt; citation_journal_title=J Clin Microbiol; citation_volume=38; citation_number=74; citation_pages=1008-1015; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification of common molecular subsequences.; citation_author=TF Smith; citation_author=MS Waterman; citation_journal_title=J Mol Biol; citation_volume=147; citation_number=75; citation_pages=195-197; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Bias in phylogenetic estimation and its relevance to the choice between parsimony and likelihood methods.; citation_author=DL Swofford; citation_author=PJ Waddell; citation_author=JP Huelsenbeck; citation_author=PG Foster; citation_author=PO Lewis; citation_journal_title=Syst Biol; citation_volume=50; citation_number=76; citation_pages=525-539; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001120" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Molecular Correlates of Host Specialization in Staphylococcus aureus"/>
    <meta name="twitter:description" content="BackgroundThe majority of Staphylococcus aureus isolates that are recovered from either serious infections in humans or from mastitis in cattle represent genetically distinct sets of clonal groups. Moreover, population genetic analyses have provided strong evidence of host specialization among S. aureus clonal groups associated with human and ruminant infection. However, the molecular basis of host specialization in S. aureus is not understood.Methodology/Principal FindingsWe sequenced the genome of strain ET3-1, a representative isolate of a common bovine mastitis-causing S. aureus clone. Strain ET3-1 encodes several genomic elements that have not been previously identified in S. aureus, including homologs of virulence factors from other Gram-positive pathogens. Relative to the other sequenced S. aureus associated with human infection, allelic variation in ET3-1 was high among virulence and surface-associated genes involved in host colonization, toxin production, iron metabolism, antibiotic resistance, and gene regulation. Interestingly, a number of well-characterized S. aureus virulence factors, including protein A and clumping factor A, exist as pseudogenes in ET3-1. Whole-genome DNA microarray hybridization revealed considerable similarity in the gene content of highly successful S. aureus clones associated with bovine mastitis, but not among those clones that are only infrequently recovered from bovine hosts.Conclusions/SignificanceWhole genome sequencing and comparative genomic analyses revealed a set of molecular genetic features that distinguish clones of highly successful bovine-associated S. aureus optimized for mastitis pathogenesis in cattle from those that infect human hosts or are only infrequently recovered from bovine sources. Further, the results suggest that modern bovine specialist clones diverged from a common ancestor resembling human-associated S. aureus clones through a combination of foreign DNA acquisition and gene decay."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001120.g007"/>

  <meta property="og:title" content="Molecular Correlates of Host Specialization in Staphylococcus aureus" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001120" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3421'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8589'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5120&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001120">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001120" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001120&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Molecular Correlates of Host Specialization in <i>Staphylococcus aureus</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lisa Herron-Olson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, Veterinary Pathobiology and Biomedical Genomics Center, University of Minnesota, Saint Paul, Minnesota, United States of America
                </p>

                <p>
                    Current address: Syntiron, Saint Paul, Minnesota, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            J. Ross Fitzgerald, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory for Bacterial Evolution and Pathogenesis, University of Edinburgh, Edinburgh, Scotland, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James M. Musser, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Molecular and Translational Human Infectious Disease Research, Methodist Hospital Research Institute, and Department of Pathology, Methodist Hospital, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vivek Kapur
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:vkapur@umn.edu">vkapur@umn.edu</a></p>

                <p>Affiliation:
                  Department of Microbiology, Veterinary Pathobiology and Biomedical Genomics Center, University of Minnesota, Saint Paul, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 31, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001120</li>
      <li>Featured in <a href="http://www.ploscollections.org">PLOS Collections</a></li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001120" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001120" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001120" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001120" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001120-g001" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001120-g002" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001120-t001" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001120-g003" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001120-t002" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001120-g004" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001120-t003" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001120-g005" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001120-g006" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001120-g007" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001120">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001120" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The majority of <em>Staphylococcus aureus</em> isolates that are recovered from either serious infections in humans or from mastitis in cattle represent genetically distinct sets of clonal groups. Moreover, population genetic analyses have provided strong evidence of host specialization among <em>S. aureus</em> clonal groups associated with human and ruminant infection. However, the molecular basis of host specialization in <em>S. aureus</em> is not understood.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We sequenced the genome of strain ET3-1, a representative isolate of a common bovine mastitis-causing <em>S. aureus</em> clone. Strain ET3-1 encodes several genomic elements that have not been previously identified in <em>S. aureus</em>, including homologs of virulence factors from other Gram-positive pathogens. Relative to the other sequenced <em>S. aureus</em> associated with human infection, allelic variation in ET3-1 was high among virulence and surface-associated genes involved in host colonization, toxin production, iron metabolism, antibiotic resistance, and gene regulation. Interestingly, a number of well-characterized <em>S. aureus</em> virulence factors, including protein A and clumping factor A, exist as pseudogenes in ET3-1. Whole-genome DNA microarray hybridization revealed considerable similarity in the gene content of highly successful <em>S. aureus</em> clones associated with bovine mastitis, but not among those clones that are only infrequently recovered from bovine hosts.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Whole genome sequencing and comparative genomic analyses revealed a set of molecular genetic features that distinguish clones of highly successful bovine-associated <em>S. aureus</em> optimized for mastitis pathogenesis in cattle from those that infect human hosts or are only infrequently recovered from bovine sources. Further, the results suggest that modern bovine specialist clones diverged from a common ancestor resembling human-associated <em>S. aureus</em> clones through a combination of foreign DNA acquisition and gene decay.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Herron-Olson L, Fitzgerald JR, Musser JM, Kapur V (2007) Molecular Correlates of Host Specialization in <em>Staphylococcus aureus</em>. PLoS ONE 2(10):
          e1120.
            doi:10.1371/journal.pone.0001120</p><p><strong>Academic Editor: </strong>Niyaz Ahmed, Centre for DNA Fingerprinting and Diagnostics, India</p><p><strong>Received:</strong> September 10, 2007; <strong>Accepted:</strong> October 4, 2007; <strong>Published:</strong> October 31, 2007</p><p><strong>Copyright:</strong> © 2007 Herron-Olson et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was funded by a competitive award from the USDA-NRI Sustaining Animal Health and Well-Being Program (to VK). LHO was supported by NIH NIGMS Biological Process Technology Institute fellowship GM08347 and University of Minnesota Martha Kunze Graduate Fellowship. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Staphylococcus aureus</em> (SA) causes a variety of serious diseases in multiple host species, including humans and cattle. Widespread antimicrobial resistance among human isolates of SA is cause for considerable concern, and has resulted in substantial increases in the cost of treatment associated with SA infection. While the implementation of improved hygiene practices, teat dipping, vaccines and antimicrobials on dairy farms have limited the spread of antimicrobial resistance among bovine-associated SA, mastitis continues to cause billions of dollars in economic loss to the U.S. dairy industry on an annual basis <a href="#pone.0001120-Wells1">[1]</a>–<a href="#pone.0001120-Reinoso1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The molecular mechanisms by which SA causes bovine mastitis remain poorly understood. Importantly, studies of the molecular epidemiology of SA strongly suggest that a genetic subset of strains is particularly well adapted for causing infection in cattle <a href="#pone.0001120-Cifrian1">[4]</a>–<a href="#pone.0001120-1">[9]</a>. Population genetic studies of thousands of SA isolates from multiple hosts and geographic locales reveal a clonal population structure, with strong evidence of host-specialization among clones associated with human and bovine infection <a href="#pone.0001120-Reinoso1">[3]</a>, <a href="#pone.0001120-Musser1">[10]</a>–<a href="#pone.0001120-Zadoks1">[14]</a>. Among SA isolates, clones with distinctive genetic backgrounds are responsible for the majority of infections within a host type, although the distinct bovine clone groups are often interspersed among the human clone groups in a manner suggesting that human-associated SA clones were the evolutionary precursors to modern bovine specialist SA clones <a href="#pone.0001120-Kapur1">[11]</a>, <a href="#pone.0001120-Reinoso2">[13]</a>, <a href="#pone.0001120-Musser2">[15]</a>, <a href="#pone.0001120-Boyer1">[16]</a>. Furthermore, longitudinal studies show that a limited number of genotypes account for persistent mastitis in cattle herds, underscoring the notion that certain SA genotypes are particularly successful at causing bovine mastitis <a href="#pone.0001120-Kapur1">[11]</a>, <a href="#pone.0001120-Reinoso2">[13]</a>, <a href="#pone.0001120-Haveri1">[17]</a>. However, despite its clinical importance from a human and animal health perspective, its well-described clonal population structure, and the availability of complete genome sequences from several isolates for use in comparative analyses <a href="#pone.0001120-Kuroda1">[18]</a>–<a href="#pone.0001120-Gill1">[21]</a>, the molecular basis of host-specialization in SA remains unknown.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Thus, to begin to understand the molecular basis of virulence of SA clones associated with bovine mastitis and to identify the genetic features contributing to host specialization, we sequenced the complete genome of strain ET3-1, an isolate that represents the most common SA clone isolated from bovine mastitis worldwide <a href="#pone.0001120-Fitzgerald1">[12]</a>. Comparative whole genome analyses using DNA microarrays of the ET3-1 genome with that of other bovine-associated SA isolates and SA recovered from humans hosts shows that bovine specialist clones have accumulated substantial genetic divergence in genes previously associated with SA pathogenesis in the form of both genome decay and genome expansion. Further, our studies reveal a set of genomic features that distinguish clones of highly successful bovine-associated <em>S. aureus</em> optimized for pathogenesis in cattle from those that infect human hosts or those that are only infrequently recovered from bovine sources. Overall, these results provide a molecular population genetic framework for understanding the mechanism of host specialization in this important multi-species pathogen.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Genome organization and mobile genetic elements</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The 2,742,531 base pair genome of ET3-1 (GenBank accession no. AJ938182) is the smallest of the nine sequenced SA genomes included in the analysis (<a href="#pone-0001120-g001">Figure 1</a>, <a href="#pone-0001120-t001">Table 1</a>). A total of 2,589 open reading frames was identified and annotated (<a href="#pone.0001120.s001">Table S1</a>). Thirty-three genes encode products with homology solely to proteins from organisms other than SA or SA-associated phages and an additional 80 predicted proteins have no significant protein homologs in public databases. As observed in other SA genomes, the majority of the ET3-1-unique genes that are unique to GenBank or newly described in SA are encoded by mobile genetic elements (MGEs). At least two of the MGEs have not been described previously, and several MGEs resemble islands that are integrated at different locations in the sequenced human SA isolates than in ET3-1 (<a href="#pone-0001120-g002">Figure 2</a>) <a href="#pone.0001120-Witney1">[22]</a>. Excluding the MGEs, the remainder of the genome closely resembles that of the fully sequenced SA isolates from humans with respect to gene content and organization. These results are consistent with prior studies that show conserved gene order in the genome backbone and high density of genetic variation within MGEs of Gram-positive bacterial pathogens <a href="#pone.0001120-Holden1">[20]</a>, <a href="#pone.0001120-Beres1">[23]</a>, <a href="#pone.0001120-Fitzgerald2">[24]</a>. This has led to the hypothesis of a “plug and play” mechanism whereby specific combinations of virulence factors encoded within MGEs are exchanged among strains, resulting in clones that are particularly well-adapted for causing certain diseases or infecting specific hosts (<a href="#pone.0001120-Beres1">[23]</a>, Herron-Olson, Genomes 2004 Meeting, Hinxton, Cambridge, United Kingdom).</p>
<div class="figure" id="pone-0001120-g001"><div class="img"><a name="pone-0001120-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Comparative circular map of ET3-1 and other SA genomes</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>showing (from outside) 1) Scale in basepairs, 2) Mobile elements (red = conserved island, blue = pathogenicity island, pink = putative phage), 3) Ribosomal RNA operons (arrowheads), 4) Homology to other SA genomes, from outside and with color code, dark purple = MRSA252, light purple = MSA553, red = NCTC8325, orange = COL, yellow = Mu50, green = N315, teal = MW2, light blue = MSSA476, 5) ORF homology of ET3-1; navy blue = strong homolog to other SA, light blue = intermediate homolog to other SA, green = weak homolog to other SA, red = non-SA homolog, yellow = no GenBank matches (e-value&gt;10<sup>−5</sup>), 6) Location of tRNAs (small arrows), 7) ORF direction, light teal = forward strand, dark teal = reverse strand, 8) Microarray data from bovine isolates; light green inner portion = similar ET3 bovine isolates and dark green (outer) = non-ET3 bovine isolates. Yellow lines indicate insertion elements and transposases. 9) GC content = red graph line.</p>
<span>doi:10.1371/journal.pone.0001120.g001</span></div><div class="figure" id="pone-0001120-g002"><div class="img"><a name="pone-0001120-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>“Plug and play” genome organization in SA.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Fully sequenced SA genomes were locally aligned using nucleotide BLAST; gaps in the colored histogram represent missing sequence (e-value&gt;10<sup>−5</sup>). The alignments illustrate the conservation of the core genome and the importance of mobile genetic elements in SA genome variation. Similar genetic elements integrate in different locations; for example, phageSaBov in strain ET3-1 and bacteriophage Mu50β in strain Mu50 have homologous content but vary in their sites of integration. A linear path of mobile element acquisition is not evident among the sequenced strains, underscoring the role of lateral transfer in the evolution of SA.</p>
<span>doi:10.1371/journal.pone.0001120.g002</span></div><div class="figure" id="pone-0001120-t001"><div class="img"><a name="pone-0001120-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001120.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001120.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>SA strains evaluated in this study</span></strong></p><span>doi:10.1371/journal.pone.0001120.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Strain ET3-1 was the source isolate for the discovery and characterization of SaPIbov1, a pathogenicity island associated with the bovine SA isolates that encodes the genes for TSST-1 and a bovine-associated variant of staphylococcal enterotoxin C <a href="#pone.0001120-Fitzgerald3">[25]</a>. We discovered that SaPIbov1 is integrated adjacent to the conserved genomic islet νSaα (<a href="#pone-0001120-g001">Figure 1</a>). Previous studies of SaPIbov suggested that a portion of the island was duplicated <a href="#pone.0001120-Fitzgerald3">[25]</a>. Our investigation revealed that the duplicated portion encompasses 12 genes (SAB1894-SAB1906) encoded within a second island in strain ET3-1 that is referred to as SaPIbov3 (<a href="#pone-0001120-g003">Figure 3</a>). The duplicated genes share an average of 93.3% amino acid identity. To our knowledge, this is the first report of a partially duplicated pathogenicity island within a SA genome. In total, SaPIbov3 consists of 17,945 basepairs encoding 28 genes, including 4 unique genes and 6 genes with homologs in other Gram-positive organisms. Transposase and integrase gene homologs (SAB1883 and SAB1910) border SaPIbov3, located at an integration hotspot which encodes φSa3 or SaPI1 in other SA strains (<a href="#pone-0001120-g002">Figure 2</a>, <a href="#pone.0001120-Baba1">[19]</a>, <a href="#pone.0001120-Holden1">[20]</a>). No exact direct flanking repeats were identified, however, two 47-bp regions with 82% sequence identity lie between the third and fourth genes of SaPIbov3 and within the 3′ end of the terminal integrase.</p>
<div class="figure" id="pone-0001120-g003"><div class="img"><a name="pone-0001120-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Gene content of major mobile genetic elements in ET3-1.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>SaPIbov1- νSaα, is shown aligned with the partially duplicated version of the pathogenicity island SaPIbov3 where vertical black lines represent homologous genes. Three additional unique ET3-1 MGEs, including νSaBov carrying several streptolysin homologs, φ12Bov and the large element φSaBov-νSaβ, are illustrated (note individual scales). Arrowhead colors: red = previously characterized SaPI gene of unknown function, turquoise = toxin, blue = SA homolog, green = no homologs, pink = homolog from non-SA microbe or phage, yellow = mobile element enzyme, light blue = phage protein, gray = protease, purple = lipoprotein, hollow triangle = putative pseudogene.</p>
<span>doi:10.1371/journal.pone.0001120.g003</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Several other MGEs in the genome of strain ET3-1 encode genes that have not been previously identified in SA. The small genomic islet νSaBov encodes 14 genes novel to SA, including two homologs of streptolysins, the potent leukotoxins of the Sag locus originally identified in <em>Streptococcus pyogenes</em> GAS (<a href="#pone-0001120-g003">Figure 3</a>, SAB1372-SAB1383, <a href="#pone.0001120-Nizet1">[26]</a>). The streptolysin homologs (SAB1376 and SAB1373) have 31% and 27% amino acid identity, respectively, to the corresponding genes encoding SagB and SagD in GAS strain M1 <a href="#pone.0001120-Nizet1">[26]</a>. The 6,351-bp island is integrated at a position in the genome equivalent to that of φSa2 in MW2 and MRSA252 <a href="#pone.0001120-Baba1">[19]</a>, <a href="#pone.0001120-Holden1">[20]</a>, although no clear evidence of horizontal transfer was identified. Phages contribute to the unique gene content of ET3-1; φSaBov encodes 8 unique genes and is located adjacent to the genomic islet νSaβ in ET3-1. This phage resembles φ11/φETA and φMu50β carried by the MRSA isolate Mu50, although their integration sites differ (<a href="#pone-0001120-g002">Figure 2</a>). In the site where φMu50β is integrated in strain Mu50, strain ET3-1 encodes a transposase and several oligopeptide transporter genes that lie within an island previously identified as variable among diverse SA isolates <a href="#pone.0001120-Kuroda1">[18]</a>, <a href="#pone.0001120-Baba1">[19]</a>, <a href="#pone.0001120-Fitzgerald2">[24]</a>, <a href="#pone.0001120-Iandolo1">[27]</a>. Another phage, φ12Bov, resembles the well-characterized SA φ12 but like φSaBov, encodes several unique genes and genes with homologs in other Gram-positive organisms (<a href="#pone-0001120-g003">Figure 3</a>).</p>


<h4>Gene content differences between ET3-1 and previously sequenced SA isolates associated with human infections</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The majority of the more than 2000 non-MGE-associated genes shared among the SA genomes show a high degree of conservation in gene order and sequence. The ratio of the rate of nonsynonymous substitution to synonymous substitution (dn:ds) among homologous gene pairs between ET3-1 and other sequenced SA genomes was calculated in order to identify divergent genes (<a href="#pone-0001120-t002">Table 2</a>). A dn:ds ratio less than 1 is typical for most genes, and suggests that DNA mutations resulting in an amino acid replacement are selected against, presumably due to the deleterious effects of the change on protein function <a href="#pone.0001120-Nei1">[28]</a>. The majority of homologous gene pairs between ET3-1 and sequenced SA isolates associated with humans showed a low dn:ds ratio (average = 0.29±0.74) and a trend toward the conservation of protein sequence. In contrast, many genes involved in host colonization, toxin production, gene regulation, iron metabolism and antibiotic resistance had an almost 10-fold greater average dn:ds ratio in the ET3-1 genome relative to human associated sequenced SA isolates (<a href="#pone-0001120-t002">Table 2</a>).</p>
<div class="figure" id="pone-0001120-t002"><div class="img"><a name="pone-0001120-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001120.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001120.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Amino acid substitution rates for selected ET3-1 proteins</span></strong></p><span>doi:10.1371/journal.pone.0001120.t002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Interaction with host tissue plays a critical role in establishing mastitis infection <a href="#pone.0001120-1">[9]</a>. Several genes in strain ET3-1 that encode predicted surface proteins involved in colonization and host protein binding show higher-than-average rates of nonsynonymous substitution and gene decay relative to their homologs in the sequenced human SA isolates. The ET3-1 genes with the highest dn:ds ratios include fibronectin-binding protein (<em>fnbA</em>), staphylocoagulase precursor, secreted Von Willebrand factor-binding protein and several additional genes encoding host protein-binding motifs (<a href="#pone-0001120-t002">Table 2</a>).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We discovered that genes encoding several well-characterized virulence factors such as the antibody-binding protein <em>spa</em> and clumping factor A (<em>clfA</em>) contain premature stop codons, and thus are pseudogenes in strain ET3-1. Genes encoding at least six other predicted surface-expressed structural and adhesion proteins, including <em>sdrC</em>, <em>ebh,</em> and the peptidoglycan component <em>fmhA</em> also have premature stop codons, and seven ET3-1 pseudogenes with conserved non-truncated homologs among the sequenced human isolates encode hypothetical proteins with export or cell surface-localization signals. The <em>spa</em>, <em>ebh</em>, <em>fmhA</em>, and <em>sdrC</em> polymorphisms were verified by PCR and conventional capillary DNA sequencing in several bovine mastitis-associated SA strains. All of the premature truncation polymorphisms except for that of <em>sstC</em> were present in strain ET3-2, another successful bovine clone; however, premature truncations were generally not conserved in the less successful bovine mastitis-associated isolates (<a href="#pone-0001120-t003">Table 3</a>). In some genes, the decay is extensive; <em>sdrC</em> encodes two premature stop codons, resulting in three putative transcripts separated by sizeable intergenic regions <a href="#pone.0001120-Josefsson1">[29]</a>. The gene for the very large surface protein <em>ebh</em> is interrupted in multiple locations, resulting in five predicted transcripts separated by noncoding intergenic regions [SAB1289-SAB1293 and SAB1300; <a href="#pone-0001120-g004">Figure 4</a>; <a href="#pone.0001120-Clarke1">[30]</a>]. ET3-1 is the only SA genome sequenced thus far in which the majority of <em>ebh</em> is actually missing from the genome, with the remaining fragments constituting only about 30% of the full-length gene. The absence of the large internal portion of <em>ebh</em> was confirmed by PCR analysis in several other bovine mastitis-associated SA isolates. The truncations may have been caused by the insertion of a mobile genetic element, since sequencing revealed a transposase and 5 genes with homologs in <em>S. epidermidis</em> and on SA plasmid pN315 inserted into the <em>ebh</em> region. The inserted genes (SAB1294-1299) include homologs of 2 hypothetical proteins, an alcohol dehydrogenase, a transcription factor, a recombinase and the transposase for IS431. Taken together, these features indicate that gene decay in the successful bovine isolates has been ongoing for some time.</p>
<div class="figure" id="pone-0001120-g004"><div class="img"><a name="pone-0001120-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Variation in the fibronectin-binding protein Ebh.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Colored bars represent the amino acid alignment of the Ebh proteins from the sequenced SA isolates as labeled, including the sequenced bovine isolate ET3-1. Striped segments show areas of sequence variation; arrowheads indicate the insertion of 10 or more amino acids. The remnants of <em>ebh</em> in ET3-1 comprise 5 separate transcripts and are punctuated by a small mobile element encoding 6 genes with homologs on plasmid pN315 (from SA isolate N315) and in <em>S. epidermidis</em>.</p>
<span>doi:10.1371/journal.pone.0001120.g004</span></div><div class="figure" id="pone-0001120-t003"><div class="img"><a name="pone-0001120-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001120.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001120.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Pseudogene verification among diverse bovine-associated SA isolates</span></strong></p><span>doi:10.1371/journal.pone.0001120.t003</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Toxins play an important role in staphylococcal pathogenesis, and the ET3-1 genome encodes at least 30 toxin genes (<a href="#pone.0001120.s001">Table S1</a>), including the pyrogenic toxin superantigen TSST-1 and the bovine variant of staphylococcal enterotoxin C <a href="#pone.0001120-Fitzgerald3">[25]</a>. Other toxin homologs such as the staphylococcal enterotoxins <em>set9</em> and <em>set11</em> show high levels of allelic variation relative to other sequenced SA homologs (<a href="#pone-0001120-t002">Table 2</a>). Previous studies suggest that superantigens and leukotoxins are important during mastitis pathogenesis due to their immunomodulatory effects <a href="#pone.0001120-Schuberth1">[31]</a>, <a href="#pone.0001120-Fueyo1">[32]</a>. At least three sets of two-component leukotoxins are encoded in ET3-1, including <em>lukM</em>/<em>lukF'-PV</em>, which is associated with horizontal transmission by temperate bacteriophages of bovine origin (SAB0782 and SAB0783, <a href="#pone.0001120-Zou1">[33]</a>, <a href="#pone.0001120-Yamada1">[34]</a>). Recent investigations of LukM/LukF'-PV indicate that the assembled leukotoxin is several times more potent against bovine leukocytes than any other staphylococcal leukotoxins, including the Panton-Valentine leukotoxin that is highly active against human leukocytes <a href="#pone.0001120-Rainard1">[6]</a>, <a href="#pone.0001120-Barrio1">[35]</a>. The aforementioned putative streptolysin homologs encoded in ET3-1 genetic islet νSaBov are also newly described among SA genomes. Two additional toxin homologs, SAB0026 and SAB2421, are unique among sequenced SA isolates. SAB2421 shows some homology (30% amino acid identity) to pyrogenic exotoxin G (SpeG) of <em>Streptococcus pyogenes</em>, while the gene encoded by SAB0026 shares 97% amino acid identity to an enterotoxin identified in the capsule type 1-associated cassette chromosome element SCCcap1 that has not been identified in a fully sequenced SA isolate previously. In ET3-1, the enterotoxin is integrated at <em>orfX</em>, a location associated with the integration of staphylococcal cassette chromosome elements, and the remainder of the SCCcap1 cassette is missing <a href="#pone.0001120-Luong1">[36]</a>.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Antibiotic resistance among bovine SA is substantially less common than among human SA. Several recent studies show a decreasing incidence of antibiotic resistance among pathogens associated with bovine mastitis during the past 10 years, despite the use of antimicrobial therapy on dairy farms, and contrary to the substantial increase in antimicrobial resistance among human-associated SA <a href="#pone.0001120-Anderson1">[2]</a>, <a href="#pone.0001120-Wilson1">[37]</a>–<a href="#pone.0001120-Tikofsky1">[39]</a>. ET3-1 is susceptible to most antibiotics, excluding spectinomycin (64 µg/ml). A large majority (&gt;90%) of the ~800 proteins present in sequenced human-associated SA clones but absent from ET3-1 encode antimicrobial resistance determinants or hypothetical proteins associated with resistance-carrying MGEs. The ET3-1 genome encodes the lowest number of antibiotic resistance determinants of any of the sequenced SA genomes.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>Iron is a key limiting reagent during <em>in vivo</em> bacterial infection, and multiple systems have been identified in SA for binding and scavenging iron from mammalian host iron-binding proteins (IBPs) such as lactoferrin, transferrin and hemoglobin <a href="#pone.0001120-Lindsay1">[40]</a>–<a href="#pone.0001120-Trivier2">[42]</a>. While recent studies suggest that heme is a preferred iron source for SA during human infection, a predilection for particular iron sources has not been extensively studied in bovine SA <a href="#pone.0001120-Mazmanian1">[43]</a>, <a href="#pone.0001120-Skaar1">[44]</a>. Our analysis of the ET3-1 genome suggests that the genes involved in iron acquisition and metabolism vary considerably relative to their homologs encoded by SA recovered from humans. For example, despite high sequence conservation among most metabolic enzymes encoded within the ET3-1 genome, several genes encoding proteins involved in cation transport and metabolism (ferric hydroxamate receptor, multicopper oxidase protein, cadmium resistance proteins CadD and CadX) are missing. Furthermore, a premature stop codon in the siderophore transporter ATPase <em>sstC</em> (SAB0687) and a number of polymorphisms in other iron-regulated genes, including cell surface transferrin-binding protein <em>isdB</em> (SAB0994) and the iron-regulated surface protein encoded by SAB1590, suggest that iron metabolism in the bovine clone is considerably different from that previously described in SA.</p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>Interconnected SA regulatory systems control virulence factor expression during infection, however, very little is known regarding the function and interaction of SA regulatory genes during bovine mastitis <a href="#pone.0001120-Takeuchi1">[45]</a>, <a href="#pone.0001120-Gilot1">[46]</a>. Importantly, a single alteration in a regulatory system component could alter expression of hundreds of genes. The quorum-sensing <em>agr</em> system consists of four genes, <em>agrABCD</em> and the effector molecule, RNAIII <a href="#pone.0001120-Recsei1">[47]</a>–<a href="#pone.0001120-Novick1">[49]</a>. Four major <em>agr</em> groups have been described based on sequence divergence within the locus <a href="#pone.0001120-Ji1">[50]</a>, <a href="#pone.0001120-Jarraud1">[51]</a>. In ET3-1, <em>agrD</em> and <em>agrB</em> closely match that of allele group II, however, the 5′ end of the <em>agrC</em> locus differs from all 4 major <em>agr</em> groups. A recent study of several hundred isolates of human and bovine origin also suggests that the <em>agr</em> locus differs between the two host origin groups, setting up a plausible scenario of cross-inhibition by molecules produced by different strain types that may further explain the genetic divergence of bovine mastitis-associated SA clones <a href="#pone.0001120-Gilot1">[46]</a>, <a href="#pone.0001120-Wright1">[52]</a>. The ET3-1 genome also encodes at least 4 transcription factor homologs not previously identified in a SA chromosome. SAB1297, embedded in the fragmented <em>ebh</em> gene region, encodes a MarR family transcription factor homolog with high sequence identity to a gene on SA plasmid pN315. SAB1256 shares 54% amino acid identity with transcriptional regulator PaiB from several <em>Bacillus spp</em>., while SAB1757 shares 40% amino acid identity with the LexA repressor from <em>Bacillus cereus</em> and lies within φSaBov. A fourth transcription factor homolog, SAB2083, most closely resembles a <em>Listeria</em> and <em>Bacillus</em> regulatory factor (43% aa identity) and is not associated with a MGE.</p>


<h4>Comparative genomic analysis with other human and bovine-associated SA</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The molecular population genetic methods that elucidated the clonal population structure of SA use highly conserved housekeeping genes and are not optimal for differentiating strains with similar genomic backbones that encode substantially different sets of MGEs, particularly when the MGEs are recently acquired. To investigate the genome content of additional SA isolates associated with bovine mastitis, we used a comparative whole-genomic DNA hybridization microarray composed of oligonucleotide probes for each ORF in the ET3-1 genome and 8 other fully sequenced SA genomes to survey 10 additional SA isolates from cattle (<a href="#pone-0001120-g005">Figure 5</a>). The cattle isolates were selected based on their genomic backgrounds as determined by multilocus enzyme electrophoresis (MLEE) and MLST (<a href="#pone-0001120-t001">Table 1</a>). Isolates ET3-2 and ET3-3 belong to the same clone group as ET3-1, the most common clone group associated with bovine mastitis worldwide and accounting for 29% of the total isolates recovered from cattle in a large population genetic study of SA <a href="#pone.0001120-Kapur1">[11]</a>. Isolates PSA6, PSA10, PSA13, PSA17, PSA20 and PSA72 were chosen to represent 4 additional common bovine clones that are less prevalent than the ET3 clone type but still account for more than 78% of the remaining non-ET3 cattle isolates. PSA4 and PSA1001 were selected because their molecular typing profiles indicate that these isolates represent the relatively rare bovine clones (<a href="#pone-0001120-t001">Table 1</a>, <a href="#pone.0001120-Kapur1">[11]</a>). Multilocus sequence typing of these bovine isolates confirmed the MLEE results showing that some bovine SA clone types, including the successful clone represented by PSA20, are genetically more similar to the sequenced human SA isolates than to other SA isolates commonly recovered from cattle (<a href="#pone-0001120-g003">Figure 3</a>).</p>
<div class="figure" id="pone-0001120-g005"><div class="img"><a name="pone-0001120-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Linear comparative genomic analysis of SA associated with bovine and human infection illustrates hotspots of variation.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The ET3-1 genome (blue/multi) was aligned with the microarray probe data from the 10 bovine SA isolates (top section of green histograms) and the Smith-Waterman alignment <a href="#pone.0001120-Smith1">[75]</a> of the microarray probe with the 8 completely sequenced human SA isolates (bottom section of green histograms). The scale, in nucleotides, is shown in blue. Mobile genetic elements are annotated above each histogram block and ribosomal RNA operons are represented as large white gaps in the green histograms. The purple histogram above the ET3-1 genome representation shows the orientation of the ORFs. For the green microarray and Smith-Waterman alignment histograms, green = present, yellow = divergent, red = absent. For the blue/multi histogram showing ORF homology of ET3-1; navy blue = strong homolog to other SA, light blue = intermediate homolog to other SA, green = weak homolog to other SA, red = non-SA homolog, yellow = no GenBank matches (blastp e-value&gt;10<sup>−5</sup>).</p>
<span>doi:10.1371/journal.pone.0001120.g005</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Genomic DNA microarray hybridization data using the bovine SA isolates indicated that 44 of the 47 probes corresponding to unique ET3-1 gene sequences that were not previously identified in SA are also present in the closely related isolates ET3-2 and ET3-3 (<a href="#pone-0001120-g006">Figure 6</a>). Only two of the genes that are absent from the other sequenced isolates, only two of the genes are shared among all of the bovine mastitis-associated SA isolates based on microarray analysis. In total, four of the 3840 probes on the microarray were present in all of the non-ET3-1 isolates and were absent from all of the ET3-1 isolates. Hence, relatively few genes appear to be consistently present in bovine-associated SA and absent in human-associated SA, whereas a larger set of genes that are unique to bovine-associated SA appear to be conserved among the highly successful ET3 isolates.</p>
<div class="figure" id="pone-0001120-g006"><div class="img"><a name="pone-0001120-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Venn diagram summarizing the unique gene content among bovine mastitis-associated <em>S. aureus</em>.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>In total, 47 microarray probes corresponded to gene sequences that are unique to ET3-1 among the sequenced <em>S. aureus</em> genomes. Nearly 94% of these genes were conserved in all of the ET3 isolates, while only 2 of these ‘unique’ genes were conserved among all 11 isolates of bovine origin.</p>
<span>doi:10.1371/journal.pone.0001120.g006</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The microarray data (<a href="#pone-0001120-g007">Figure 7</a>) illustrate that the most diverse genetic regions, including SaPIbov, SaPIbov3, νSaBov, φSaBov, the insertion in the <em>ebh</em> gene and the SCCcap-associated toxin, are highly conserved in closely related and highly successful bovine SA clones. The strong correlation between the MGE content and molecular typing using housekeeping genes suggests that the MGEs in bovine SA may not be horizontally transferred at high frequency, in which case we would expect the MGE content of clones to be distributed independently of the typing profile. The correlation between MGE gene content and molecular typing also suggests that a specific combination of MGEs may enhance SA pathogenesis in cattle, since the MGE content among the most highly successful bovine SA clones is very similar. Interestingly, microarray analysis reveals that a set of 5 contiguous genes within SaPIbov3 (SAB1888-1892), 4 of which are duplicated in SaPIbov1, are consistently present in a majority of the isolates from cattle but are absent from previously sequenced human SA isolates. This includes the bovine SA isolates that are genetically more similar to human SA isolates by molecular typing (<a href="#pone-0001120-g007">Figure 7</a>). Overall, the microarray data show that much of the MGE content found among bovine SA is not shared with the sequenced human-associated SA genomes that were surveyed. The majority of these genes encode proteins of unknown function, so a precise role in bovine SA pathogenesis is undetermined.</p>
<div class="figure" id="pone-0001120-g007"><div class="img"><a name="pone-0001120-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001120" data-uri="info:doi/10.1371/journal.pone.0001120.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001120.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001120.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001120.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001120.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Comparative genomic DNA hybridization and <em>in silico</em> comparison of gene content within mobile elements of bovine and human SA isolates.</span></strong></p><a id="article1.body1.sec2.sec3.fig3.caption1.p1" name="article1.body1.sec2.sec3.fig3.caption1.p1"></a><p>In the MLST-based cladogram generated by PAUP <a href="#pone.0001120-Enright1">[74]</a>, <a href="#pone.0001120-Swofford1">[76]</a> at the left, “B” denotes a strain originally isolated from a bovine; “H” denotes isolation from a human. All data associated with “B” strains and MSA553<sup>A</sup> are microarray hybridization data; all remaining “H” strains show the results of <em>in silico</em> Smith-Waterman alignment of the 70mer microarray probe to the genome sequence. Red = absent, yellow = indeterminable/variable, green = present. OPtrans = oligopeptide transporter island and Ebh = region of probes representing the <em>ebh</em> gene or gene remnants+probes for 6 gene insertion in ET3-1. The white boxes within the SaPIbov1 and SaPIbov3 islands show gene clusters that were consistently absent from strains associated with human infection and largely conserved in strains recovered from bovine sources, including those that more closely resemble the isolates from humans.</p>
<span>doi:10.1371/journal.pone.0001120.g007</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The environment of the bovine teat canal and udder, including the presence of milk during lactation, differs markedly from common sites of infection in human hosts, and this may account for the high degree of genetic divergence identified among genes encoding surface-expressed proteins in strain ET3-1 relative to the human-associated SA isolates. Microarray studies of SA isolates collected from humans also identified lineage-specific surface protein features <a href="#pone.0001120-Lindsay2">[53]</a>. Furthermore, although unique or divergent MGEs have been identified in each SA genome sequenced thus far, gene homologs among the novel MGEs and ET3-1-unique genes identified in this study suggest that these elements may have been acquired from other mastitis-causing pathogens in the bovine environment. Additional studies of SA clones commonly associated with human and ruminant infection are recommended to investigate the role of MGEs in contributing to the tropism of some clones for infecting a certain host and causing a particular disease.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Our data suggest that the MGE content is similar among highly successful bovine SA clones that are closely related by MLST. These findings stand in contrast with prior studies indicating that human-associated SA isolates such as N315 and Mu50 are closely related by MLST, but differ considerably in MGE content, and they are consistent with the idea of relatively recent dissemination of a successful clonal bovine lineage. Alternatively, bovine SA may not undergo change as rapidly as human-associated SA isolates; the substantially lower number of genes associated with antibiotic resistance in the genome of strain ET3-1 provides additional evidence for a slower rate of genetic exchange. Overall, the conservation of gene content among the MGEs of highly successful bovine SA isolates illustrated by the microarray study is consistent with evidence from prior studies and suggests a plug-and-play mechanism whereby a particular combination of MGEs enhances pathogenicity in a specific infection model <a href="#pone.0001120-Holden1">[20]</a>. Additional studies of SA genomes that feature divergent backbone content and similar MGE combinations would be required to further explore this mechanism.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>A second very unexpected finding was the high level of gene decay observed among dozens of ET3-1 genes, including several well-characterized virulence factors that are conserved among the sequenced SA isolates associated with human infection. Gene decay has been observed among other SA genomes, but the quantity and the genes affected differ among strains. Strain MRSA252 is the most divergent of the sequenced human-associated SA isolates by MLST and encodes the largest number of annotated pseudogenes (n = 88). Strain ET3-1 encodes 70 annotated pseudogenes, whereas the other sequenced SA genomes encode substantially fewer pseudogenes (<a href="#pone-0001120-t001">Table 1</a>). Gene decay frequently results from random mutation, but may also be caused by insertion sequences or slip-strand mispairing resulting from short sequence repeats in the upstream or 5′ region of the gene <a href="#pone.0001120-Moxon1">[54]</a>, <a href="#pone.0001120-Bayliss1">[55]</a>. A search for simple sequence repeats (&lt;9 bases) within the ET3-1 genome found no correlation with pseudogenes, although many of the ET3-1 pseudogenes did contain longer repeat motifs of 20 to 60 bases that are likely related to their surface localization and host-protein binding function. Although insertion sequences are common in SA, they account for less than four percent of the pseudogenes in ET3-1; hence, the pseudogenes in the ET3-1 genome likely arose by random mutation.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Gene decay has been described as a mechanism of adaptation among obligate intracellular and enteric Gram-negative pathogens, including <em>Helicobacter pylori</em>, <em>Yersinia pestis</em>, <em>Mycobacterium leprae</em> and others <a href="#pone.0001120-Bayliss1">[55]</a>–<a href="#pone.0001120-Eiglmeier1">[59]</a>. In <em>Y. pestis</em>, genetic divergence appears to be driven by gene decay via insertional and point mutation, in combination with selective genome expansion by lateral gene transfer among plasmids. In contrast, ET3-1 contains no plasmids and the acquisition of MGEs has not yet led to substantial genome expansion on the scale of that observed among <em>Yersinia</em>; in comparison with other sequenced SA genomes, ET3-1 is substantially smaller (<a href="#pone-0001120-t001">Table 1</a>). Alternatively, gene decay has been associated with the transition to an intracellular lifestyle, whereby biosynthetic components are lost in favor of parasitizing nutrients from the host cell. Substantial evidence suggests that SA can exist as an intracellular pathogen, and that intracellular survival may be particularly important during bovine mastitis <a href="#pone.0001120-Bayles1">[60]</a>–<a href="#pone.0001120-Gresham1">[63]</a>. Our data support a scenario in which ET3-1 and similar SA clones are beginning the transition to an intracellular lifestyle, during which the role of adhesion and host protein-binding components is reduced and these components are being lost. A transition to a largely intracellular lifestyle would also mimic a reduced capacity for lateral gene transfer, because contact with other microbes with which genetic material could be exchanged would be reduced. An intracellular lifestyle may also account for the paucity of antibiotic resistance determinants, which would be unnecessary for bacteria living within host cells, and could also explain the observed allelic polymorphisms among genes involved in iron metabolism in ET3-1. Iron availability within cells is substantially different than in the extracellular milieu, where iron is scarce and often tightly bound to host iron-binding proteins (IBPs).</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Another explanation for the divergence in iron metabolism components observed in the ET3-1 genome relates to the structure of mammalian IBPs, which differ substantially among species. These differences provide a selection pressure for pathogens in which adaptation to enhance iron scavenging from host-specific IBPs contributes to virulence <a href="#pone.0001120-Retzer1">[64]</a>. For example, host-specific expression of transferrin-binding proteins in the protozoan pathogen <em>Trypanosoma brucei</em> in human, canine and bovine hosts has been described, suggesting that the relationship between host IBPs and pathogen scavenging systems is specific and important during infection <a href="#pone.0001120-Bitter1">[65]</a>. Furthermore, host IBPs are specific to certain tissues and anatomical compartments, such as the bovine udder, where most of the iron is bound to lactoferrin or ferric citrate and the balance of the two iron-containing molecules changes during different stages of lactation. By contrast, during most human SA infections, the pathogen is primarily exposed to IBPs such as hemoglobin and transferrin, and human-associated SA isolates appear to prefer heme as an iron source <a href="#pone.0001120-Mazmanian1">[43]</a>. The iron metabolism of common bovine SA clones therefore warrants further study of its role in exuding host-specificity and enhancing pathogenesis in bovine mastitis.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Overall, we identified more extensive variation between bovine and human-associated <em>S. aureus</em> genomes than that found amongst human-associated <em>S. aureus</em>, revealing a set of genomically encoded components that distinguish populations of highly successful human and bovine-associated <em>S. aureus</em> clones that are optimized for infection of different hosts. Furthermore, the results support prior population genetic studies suggesting that modern bovine specialist clones diverged from a common ancestor resembling human-associated <em>S. aureus</em> clones, and have since exchanged genetic material independently of human-associated <em>S. aureus</em>. These findings provide a strong foundation for investigations of the molecular basis of pathogenesis of SA-associated bovine mastitis, as well as a rational foundation for continuing the investigation of host specificity in this important human and animal pathogen.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Isolate selection</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The bovine SA clone selected for sequencing was isolated from a cow presenting with clinical mastitis in 1993 <a href="#pone.0001120-Fitzgerald1">[12]</a>. The clone is of the ET3 profile (and for simplicity is therefore referred to as ET3-1 in the body of the manuscript), the most common electrophoretic type associated with bovine mastitis, which is exclusive to cattle and distributed worldwide <a href="#pone.0001120-Kapur1">[11]</a>, <a href="#pone.0001120-Fitzgerald1">[12]</a>. The clone also belongs to the ST151 MLST type (<a href="#pone-0001120-t001">Table 1</a>). A novel 15,981-bp pathogenicity island, SaPIbov1, was identified in this isolate <a href="#pone.0001120-Fitzgerald3">[25]</a>.</p>


<h4>Library preparation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>A shotgun sequencing approach was utilized to initiate whole genome sequencing. Isolated whole genomic DNA from SA strain ET3-1 was randomly sheared to generate fragments of 1.7–2.4 kb using a HydroShear apparatus (GeneMachines, San Carlos, CA); fragment size was verified by agarose gel electrophoresis. The ends of 200 ul (approximately 25 µg) of the randomly sheared fragments were repaired using T4 polynucleotide kinase and Klenow enzyme (New England Biolabs, Beverly, MA) at 37°C. Enzymes were deactivated at 75°C for 10 minutes and the end-repaired fragments were run on a 0.8% ethidium bromide agarose gel. DNA of 1.8 to 2.2 kb was excised from the gel and purified using the QiaEXII Gel Extraction Kit according to the manufacturer's protocol (Qiagen, Valencia, CA). ET3-1 DNA fragments (approximately 850 ng) were ligated into <em>Sma</em>I-restricted pUC18 (Stratagene, LaJolla, CA) using T4 ligase (New England Biolabs). 4 µl of the ligation product were transformed into XL10-Gold Epicurian Coli ultracompetent cells according to the published protocols (Stratagene). Blue colonies were picked from LB-ampicillin/X-gal/IPTG plates after 24 hours at 37°C and incubated in 96-well culture blocks, each well containing 1.25 ml LB-ampicillin broth (50 µg/ml). Cells were pelleted after 21 hours by centrifugation at 1500×<em>g</em> for 5 minutes. Plasmids were purified using the 96 Turbo Miniprep preparation, according to the protocols of the manufacturer (Qiagen).</p>


<h4>Shotgun sequencing and assembly</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Plasmids (approximately 100 ng each) were transferred to new 96-well plates, dried overnight and resuspended with 6 picomoles of primer (T7), then sequenced on ABI 3700 automated DNA capillary sequencers (ABI, Foster City, CA). The initial shotgun sequencing approach yielded a total of 23,611 sequences (6.5-fold coverage) which were assembled into contigs using PhredPhrap <a href="#pone.0001120-Herron1">[66]</a>, <a href="#pone.0001120-Ewing1">[67]</a>. The original shotgun sequence assembly, using a minmatch value of 12 and force level of 8 resulted in a total of 127 contigs greater than 2 kb. Within the initial shotgun assembly, 65.6% of the cumulative sequence was represented in contigs greater than 10 kb. In order to expedite the genome assembly, a novel software tool called MGView was developed to graphically overlay the assembled contigs onto completed SA genomes that were available publicly <a href="#pone.0001120-HerronOlson1">[68]</a>. Contig joins suggested by MGView were used to design primers spanning assembly gaps for amplification by either regular PCR (up to 3 kb) or XL-PCR (up to 15 kb) followed by sequencing. For regular PCR, each 25 µl reaction contained 1 unit of Taq polymerase (Applied Biosystems) with appropriate manufacturer-supplied buffer at 1X, 1.5 mM MgCl, 0.1 µM primer (each), 200 µM dNTP (each) and 200 ng genomic ET3-1 DNA. Thermal cycling conditions were as follows: Denature at 95°C×30 seconds, anneal at 54°C×30 seconds and extend at 72°C×1 minute (repeat for 25 cycles), final extension at 72°C×8 minutes. For XL-PCR, TaqStart PCR polymerase-binding antibody (ClonTech, Mountain View, CA) was utilized according to the manufacturer's recommendations. Each 35 µl reaction also contained 5 units of r<em>Tth</em> XL DNA polymerase (GeneAmp XL PCR, Applied Biosystems) with supplied buffer at 1X containing glycerol and DMSO, 1.5 mM MgOAc<sub>2</sub>, 200 µM dNTP (each), 0.2 µM primer (each), and approximately 200 ng genomic ET3-1 DNA. PCR thermal cycling was conducted as follows: A) Pre-denature at 95°C×10 min.; B) Cycling: denature at 95°C×10 min, anneal/extend at 60°C×9 min+15 second increments added each round; repeat for 20 cycles; C) Final extension: 72°C×10 min. All PCR products were verified by gel electrophoresis, purified using Microcon 100 filter plates (Millipore, Billerica, MA) and sequenced according to conditions optimized for template length, template type, and template and primer GC content, which were determined from extensive testing in the Advanced Genetic Analysis Center at the University of Minnesota. Using these optimizations, single sequence reads of 750–1000 bp from single regular and XL-length double-stranded PCR-derived DNA templates were consistently obtained, without the need for subcloning or large-insert libraries. For most XL-PCR products, additional sequencing primers were generated in order to cover the entire length of the product, and these primers were determined based on homology with other SA genome sequences. Where this approach was unsuccessful due to sequence variation between ET3-1 and other SA genomes, new sequencing primers were generated along the length of the template following the acquisition of the new sequencing reads.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>As the PCR products were added, the assembly was continuously updated, proofread and edited using the software CONSED <a href="#pone.0001120-Gordon1">[69]</a>. False joints in the assembly were taken apart (e.g., using the “tear contig” function), then reattached in the correct orientation (“join contig” function). Once false assembly joints were identified, PCR products corresponding to the correct joint were generated and sequenced in order to verify the accuracy of the assembly. The addition of these sequences to the assembly acted as a ‘patch’ to reinforce the corrected assembly. Where the software repeatedly misassembled joints even after the addition of PCR patches due to ubiquitous repeat sequences, longer artificial reads covering the sequence flanking the repeat were used to guide the assembly. The final gaps that could not be closed using the MGView-guided directed PCR procedure (n = 14) corresponded to sequence assembly regions that were unique to strain ET3-1 relative to other available SA genome sequences, and were closed by random PCR. Closures and weakly covered regions were verified by PCR and sequence analysis, resulting in a final error rate of below 1 error per 10,000 bases [as estimated by CONSED <a href="#pone.0001120-Gordon1">[69]</a>]. In total, more than 1500 additional sequence reads from the PCR-based closure and proofreading strategy were added to the shotgun assembly.</p>


<h4>Annotation and computational comparative genomics</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Open reading frames (ORFs) were identified using ARTEMIS <a href="#pone.0001120-Berriman1">[70]</a> followed by manual verification of start sites based on GC skew and location of Shine-Dalgarno sequences <a href="#pone.0001120-McCarthy1">[71]</a>. Annotation of products was performed manually using evidence gathered from BLASTP searches against a microbial sequence database devoid of SA genome sequences in order to prevent bias <a href="#pone.0001120-Altschul1">[72]</a>. The final pass of sequence annotation included the completed SA genomes for comparison <a href="#pone.0001120-Kuroda1">[18]</a>–<a href="#pone.0001120-Holden1">[20]</a>. The sequence divergence between homologous gene pairs in ET3-1 and the fully sequenced human SA isolates was estimated by calculating the nonsynonymous and synonymous substitution rate ratio (dn:ds) for each gene pair shared between two sequenced genomes <a href="#pone.0001120-Nei1">[28]</a>. A dn:ds less than 1 suggests that DNA mutations resulting in an amino acid change are not conserved in subsequent generations, presumably due to the deleterious effects of the mutation on protein function, whereas a dn:ds greater than 1 suggests that mutations resulting in amino acid changes do not negatively impact the organism and that the gene product may be under pressure to change. For this study, dn:ds was calculated for each gene pair shared between ET3-1 and the sequenced human SA isolates MRSA252, N315 and MW2 <a href="#pone.0001120-Kuroda1">[18]</a>–<a href="#pone.0001120-Holden1">[20]</a>. Rates of synonymous and nonsynonymous substitution were calculated by first identifying orthologs between bovine ET3-1 and other sequenced isolates by BLASTP, then refining the sequence alignments using CLUSTALW and calculating the rates by Nei and Gojobori's algorithm <a href="#pone.0001120-Nei1">[28]</a>, <a href="#pone.0001120-Altschul1">[72]</a>, <a href="#pone.0001120-Thompson1">[73]</a>.</p>


<h4>Microarray analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Genome sequences from bovine strain ET3-1 and all available SA human isolates (<a href="#pone-0001120-t001">Table 1</a>) were used to construct a nonredundant 3840 probe 70mer DNA oligonucleotide array. Probes were synthesized by Illumina (San Diego, CA); dried oligos were resuspended in 3XSSC and spotted in triplicate onto Corning Gaps II aminosaline-coated microarray slides (Corning, Big Flats NY) using a BioRobotics Microgrid II Array Spotter (Genomic Solutions, Ann Arbor MI). Slides were rehydrated, UV cross-linked and stored under dessication until use in hybridization studies.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Bovine SA isolates for comparative genomic hybridization were selected based on previously identified multilocus enzyme elecrophoresis (MLEE) and multilocus sequence typing (MLST) profiles determined according to established protocols (<a href="#pone-0001120-t001">Table 1</a>, <a href="#pone.0001120-Enright1">[74]</a>). The SA isolates were taken from the original clone collection used for MLEE-based population genetic analysis of bovine cattle and were primarily of U.S. origin; specific information on the nature of the clinical manifestation (e.g., acute or subclnical mastitis) was unavailable <a href="#pone.0001120-Kapur1">[11]</a>. As controls, ET3-1 and human TSS-associated strain MSA553, for which the complete genome sequence has also been recently determined (unpublished data), were used to establish statistical thresholds for gene presence and absence. SA isolates were grown at 37°C in 100 ml tryptic soy broth to mid-log phase (OD<sub>600</sub> = 0.7−0.8). DNA was isolated by standard CTAB/organic extraction with the addition of lysostaphin (1 µg/ml) during cell lysis. Purified DNA was mechanically sheared into 350–700 bp fragments using a Hydroshear apparatus fitted with a custom short shearing assembly (GeneMachines). Sheared DNA (10 µg) was labeled overnight using 5 U Klenow enzyme and amino-allyl coupled dUTPs. Labeled DNA was washed and incubated with Cy3 and Cy5 fluorescent probes for 2 hours at room temperature (Amersham, Piscataway NJ). Meanwhile, array slides were incubated for 1 hr in preheated 42°C prehybridization buffer (25 ml formamide, 12.5 ml 20X SSC, 12 ml H<sub>2</sub>0, 300 µl 10% SDS and 500 µg BSA filtered with 2 µm syringe filter), rinsed thoroughly with 2 L H<sub>2</sub>0 and dried immediately by centrifugation. Labeled probes (12 µl each) were washed of excess dye, combined and added to the reaction mix: 9.8 µl formamide, 6.8 µl 20X SSC, 3.4 µl salmon sperm and 1 µl 10% SDS. Reactions were incubated at 99.9°C for 2 minutes in a thermal cycler, allowed to cool and applied to the prepared microarray. Arrays were incubated in a 42°C waterbath overnight, washed in successive dilutions of SSC wash buffer, dried and scanned using an Axon 4000B Scanner (Molecular Devices, Union City CA).</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>For each dual-colored probe hybridization experiment, ET3-1 DNA was used as the control sample. Each pair (ET3-1 vs. experimental strain) was repeated using a dye-swap format to obviate dye bias. With each probe on the array spotted in triplicate and each experiment repeated as a dye swap, 6 replicates per probe were generated. Human SA strain MSA553 was also hybridized against ET3-1 DNA to establish statistical thresholds for divergent and absent genes. Global and Lowess normalization procedures were not appropriate for analysis of these arrays because these methods both rely on the assumption that nearly all genes should have a balanced hybridization ratio. For each strain in these experiments, nearly one-quarter of the array probes were expected <em>not</em> to hybridize due to genomic variation. Thus, in order to establish an array normalization probe set, 1000 highly conserved genes were selected using <em>in silico</em> analysis of the nine completely sequenced SA genomes used to generate the array probes. A set of 750 conserved genes with consistently measurable and balanced intensity ratios in the control arrays were selected for standard global normalization using totaled mean channel intensity, and the resulting ratio was used to normalize the remaining array spots that has passed through filters (signal saturation and total intensity minus background, which were calculated by Axon GenePixPro software (Molecular Devices)). Genes were classified based on the ratio of experimental strain DNA signal intensity to ET3-1 DNA signal intensity (ratio≤0.35 = absent; 0.35&lt;ratio&lt;0.50 = variable; ratio≥0.50 = present). These thresholds were determined by comparing hybridization ratios with probe homology for the fully sequenced strains and by examining the ratios for three replicate hybridizations of MSA553 versus ET3-1 DNA.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>For analysis of the human sequenced SA genomes, the homology of the microarray probes to the sequenced human isolates was determined using Smith-Waterman alignment and assigning (74). A scoring system accounting for gaps and single mismatches within the 70mer probe was developed in order to classify genes analogously to the microarray hybridization as absent, variable, or present. The microarray hybridization and Smith-Waterman alignments for strains ET3-1 and MSA553 were used to verify the accuracy of the <em>in silico</em> scoring system. Data from the DNA microarray hybridizations and the <em>in silico</em> hybridizations were compiled to generate <a href="#pone-0001120-g006">Figure 6</a> using Spotfire (Somerville MA) for data visualization.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001120"></div><a name="pone.0001120.s001" id="pone.0001120.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001120.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">ET3-1 Feature Table</p>
<p class="siDoi">doi:10.1371/journal.pone.0001120.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.85 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We appreciate the support of the Advanced Genetic Analysis Center, Supercomputing Institute and Center for Computational Genomics and Bioinformatics at the University of Minnesota for assistance with automated DNA sequencing and computation. We also thank Barbara May, Lingling Li, Rajit Chakravarty and Daniel Wolf for their contributions during the construction of the genome library and sequencing and Michael Paustian for assistance with the microarray analysis.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JM LH VK. Performed the experiments: LH. Analyzed the data: LH VK. Contributed reagents/materials/analysis tools: JF. Wrote the paper: JM LH VK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001120-Wells1" id="pone.0001120-Wells1"></a>Wells SJ, Ott SL, Seitzinger AH (1998) Key health issues for dairy cattle–new and old. J Dairy Sci  81: 3029–3035.  <ul class="find" data-citedArticleID="1012926"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Key+health+issues+for+dairy+cattle%3Fnew+and+old.&amp;auth=&amp;atitle=Key+health+issues+for+dairy+cattle%3Fnew+and+old." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Key+health+issues+for+dairy+cattle%E2%80%93new+and+old." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Key+health+issues+for+dairy+cattle%E2%80%93new+and+old.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001120-Anderson1" id="pone.0001120-Anderson1"></a>Anderson KL, Lyman RL, Bodeis-Jones SM, White DG (2006) Genetic diversity and antimicrobial susceptibility profiles among mastitis-causing Staphylococcus aureus isolated from bovine milk samples. Am J Vet Res  67: 1185–1191.  <ul class="find" data-citedArticleID="1012792" data-doi="10.2460/ajvr.67.7.1185"><li><a href="http://dx.doi.org/10.2460/ajvr.67.7.1185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+diversity+and+antimicrobial+susceptibility+profiles+among+mastitis-causing+Staphylococcus+aureus+isolated+from+bovine+milk+samples." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+diversity+and+antimicrobial+susceptibility+profiles+among+mastitis-causing+Staphylococcus+aureus+isolated+from+bovine+milk+samples.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001120-Reinoso1" id="pone.0001120-Reinoso1"></a>Reinoso EB, El-Sayed A, Lammler C, Bogni C, Zschock M (2006) Genotyping of Staphylococcus aureus isolated from humans, bovine subclinical mastitis and food samples in Argentina. Microbiol Res.   <ul class="find" data-citedArticleID="1012900" data-doi="10.1016/j.micres.2006.05.013"><li><a href="http://dx.doi.org/10.1016/j.micres.2006.05.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotyping+of+Staphylococcus+aureus+isolated+from+humans%2C+bovine+subclinical+mastitis+and+food+samples+in+Argentina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotyping+of+Staphylococcus+aureus+isolated+from+humans%2C+bovine+subclinical+mastitis+and+food+samples+in+Argentina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001120-Cifrian1" id="pone.0001120-Cifrian1"></a>Cifrian E, Guidry AJ, O'Brien CN, Nickerson SC, Marquardt WW (1994) Adherence of Staphylococcus aureus to cultured bovine mammary epithelial cells. J Dairy Sci  77: 970–983.  <ul class="find" data-citedArticleID="1012814"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Adherence+of+Staphylococcus+aureus+to+cultured+bovine+mammary+epithelial+cells.&amp;auth=&amp;atitle=Adherence+of+Staphylococcus+aureus+to+cultured+bovine+mammary+epithelial+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adherence+of+Staphylococcus+aureus+to+cultured+bovine+mammary+epithelial+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adherence+of+Staphylococcus+aureus+to+cultured+bovine+mammary+epithelial+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001120-Leitner1" id="pone.0001120-Leitner1"></a>Leitner G, Krifucks O, Glickman A, Younis A, Saran A (2003) Staphylococcus aureus strains isolated from bovine mastitis: virulence, antibody production and protection from challenge in a mouse model. FEMS Immunol Med Microbiol  35: 99–106.  <ul class="find" data-citedArticleID="1012866" data-doi="10.1016/s0928-8244(02)00458-3"><li><a href="http://dx.doi.org/10.1016/s0928-8244(02)00458-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Staphylococcus+aureus+strains+isolated+from+bovine+mastitis%3A+virulence%2C+antibody+production+and+protection+from+challenge+in+a+mouse+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Staphylococcus+aureus+strains+isolated+from+bovine+mastitis%3A+virulence%2C+antibody+production+and+protection+from+challenge+in+a+mouse+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001120-Rainard1" id="pone.0001120-Rainard1"></a>Rainard P, Corrales JC, Barrio MB, Cochard T, Poutrel B (2003) Leucotoxic activities of Staphylococcus aureus strains isolated from cows, ewes, and goats with mastitis: importance of LukM/LukF'-PV leukotoxin. Clin Diagn Lab Immunol  10: 272–277.  <ul class="find" data-citedArticleID="1012896" data-doi="10.1128/cdli.10.2.272-277.2003"><li><a href="http://dx.doi.org/10.1128/cdli.10.2.272-277.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Leucotoxic+activities+of+Staphylococcus+aureus+strains+isolated+from+cows%2C+ewes%2C+and+goats+with+mastitis%3A+importance+of+LukM%2FLukF%27-PV+leukotoxin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Leucotoxic+activities+of+Staphylococcus+aureus+strains+isolated+from+cows%2C+ewes%2C+and+goats+with+mastitis%3A+importance+of+LukM%2FLukF%27-PV+leukotoxin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001120-Vasudevan1" id="pone.0001120-Vasudevan1"></a>Vasudevan P, Nair MK, Annamalai T, Venkitanarayanan KS (2003) Phenotypic and genotypic characterization of bovine mastitis isolates of Staphylococcus aureus for biofilm formation. Vet Microbiol  92: 179–185.  <ul class="find" data-citedArticleID="1012924" data-doi="10.1016/s0378-1135(02)00360-7"><li><a href="http://dx.doi.org/10.1016/s0378-1135(02)00360-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+and+genotypic+characterization+of+bovine+mastitis+isolates+of+Staphylococcus+aureus+for+biofilm+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+and+genotypic+characterization+of+bovine+mastitis+isolates+of+Staphylococcus+aureus+for+biofilm+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001120-Cucarella1" id="pone.0001120-Cucarella1"></a>Cucarella C, Tormo MA, Ubeda C, Trotonda MP, Monzon M, et al.  (2004) Role of biofilm-associated protein bap in the pathogenesis of bovine Staphylococcus aureus. Infect Immun  72: 2177–2185.  <ul class="find" data-citedArticleID="1012820" data-doi="10.1128/iai.72.4.2177-2185.2004"><li><a href="http://dx.doi.org/10.1128/iai.72.4.2177-2185.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+biofilm-associated+protein+bap+in+the+pathogenesis+of+bovine+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+biofilm-associated+protein+bap+in+the+pathogenesis+of+bovine+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001120-1" id="pone.0001120-1"></a>Kerro Dego O, van Dijk JE, Nederbragt H (2002) Factors involved in the early pathogenesis of bovine Staphylococcus aureus mastitis with emphasis on bacterial adhesion and invasion. A review. Vet Q  24: 181–198.  <ul class="find" data-citedArticleID="1012788" data-doi="10.1080/01652176.2002.9695135"><li><a href="http://dx.doi.org/10.1080/01652176.2002.9695135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Factors+involved+in+the+early+pathogenesis+of+bovine+Staphylococcus+aureus+mastitis+with+emphasis+on+bacterial+adhesion+and+invasion.+A+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Factors+involved+in+the+early+pathogenesis+of+bovine+Staphylococcus+aureus+mastitis+with+emphasis+on+bacterial+adhesion+and+invasion.+A+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001120-Musser1" id="pone.0001120-Musser1"></a>Musser JM, Schlievert PM, Chow AW, Ewan P, Kreiswirth BN, et al.  (1990) A single clone of Staphylococcus aureus causes the majority of cases of toxic shock syndrome. Proc Natl Acad Sci U S A  87: 225–229.  <ul class="find" data-citedArticleID="1012882" data-doi="10.1073/pnas.87.1.225"><li><a href="http://dx.doi.org/10.1073/pnas.87.1.225" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+clone+of+Staphylococcus+aureus+causes+the+majority+of+cases+of+toxic+shock+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+clone+of+Staphylococcus+aureus+causes+the+majority+of+cases+of+toxic+shock+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001120-Kapur1" id="pone.0001120-Kapur1"></a>Kapur V, Sischo WM, Greer RS, Whittam TS, Musser JM (1995) Molecular population genetic analysis of Staphylococcus aureus recovered from cows. J Clin Microbiol  33: 376–380.  <ul class="find" data-citedArticleID="1012862"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+population+genetic+analysis+of+Staphylococcus+aureus+recovered+from+cows.&amp;auth=&amp;atitle=Molecular+population+genetic+analysis+of+Staphylococcus+aureus+recovered+from+cows." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+population+genetic+analysis+of+Staphylococcus+aureus+recovered+from+cows." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+population+genetic+analysis+of+Staphylococcus+aureus+recovered+from+cows.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001120-Fitzgerald1" id="pone.0001120-Fitzgerald1"></a>Fitzgerald JR, Meaney WJ, Hartigan PJ, Smyth CJ, Kapur V (1997) Fine-structure molecular epidemiological analysis of Staphylococcus aureus recovered from cows. Epidemiol Infect  119: 261–269.  <ul class="find" data-citedArticleID="1012830" data-doi="10.1017/s0950268897007802"><li><a href="http://dx.doi.org/10.1017/s0950268897007802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fine-structure+molecular+epidemiological+analysis+of+Staphylococcus+aureus+recovered+from+cows." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fine-structure+molecular+epidemiological+analysis+of+Staphylococcus+aureus+recovered+from+cows.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001120-Reinoso2" id="pone.0001120-Reinoso2"></a>Reinoso E, Bettera S, Frigerio C, DiRenzo M, Calzolari A, et al.  (2004) RAPD-PCR analysis of Staphylococcus aureus strains isolated from bovine and human hosts. Microbiol Res  159: 245–255.  <ul class="find" data-citedArticleID="1012902" data-doi="10.1016/j.micres.2004.04.002"><li><a href="http://dx.doi.org/10.1016/j.micres.2004.04.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RAPD-PCR+analysis+of+Staphylococcus+aureus+strains+isolated+from+bovine+and+human+hosts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RAPD-PCR+analysis+of+Staphylococcus+aureus+strains+isolated+from+bovine+and+human+hosts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001120-Zadoks1" id="pone.0001120-Zadoks1"></a>Zadoks R, van Leeuwen W, Barkema H, Sampimon O, Verbrugh H, et al.  (2000) Application of pulsed-field gel electrophoresis and binary typing as tools in veterinary clinical microbiology and molecular epidemiologic analysis of bovine and human Staphylococcus aureus isolates. J Clin Microbiol  38: 1931–1939.  <ul class="find" data-citedArticleID="1012936"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Application+of+pulsed-field+gel+electrophoresis+and+binary+typing+as+tools+in+veterinary+clinical+microbiology+and+molecular+epidemiologic+analysis+of+bovine+and+human+Staphylococcus+aureus+isolates.&amp;auth=&amp;atitle=Application+of+pulsed-field+gel+electrophoresis+and+binary+typing+as+tools+in+veterinary+clinical+microbiology+and+molecular+epidemiologic+analysis+of+bovine+and+human+Staphylococcus+aureus+isolates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+pulsed-field+gel+electrophoresis+and+binary+typing+as+tools+in+veterinary+clinical+microbiology+and+molecular+epidemiologic+analysis+of+bovine+and+human+Staphylococcus+aureus+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+pulsed-field+gel+electrophoresis+and+binary+typing+as+tools+in+veterinary+clinical+microbiology+and+molecular+epidemiologic+analysis+of+bovine+and+human+Staphylococcus+aureus+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001120-Musser2" id="pone.0001120-Musser2"></a>Musser JMaRKS (1990)  Genetic analysis of natural populations of <em>Staphylococcus aureus</em>. In: Novick RP, editor. Molecular Biology of the Staphylococci. New York: VCH Publishers. pp. 59–67.  <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0001120-Boyer1" id="pone.0001120-Boyer1"></a>Boyer E, Bergevin I, Malo D, Gros P, Cellier MF (2002) Acquisition of Mn(II) in addition to Fe(II) is required for full virulence of Salmonella enterica serovar Typhimurium. Infect Immun  70: 6032–6042.  <ul class="find" data-citedArticleID="1012810" data-doi="10.1128/iai.70.11.6032-6042.2002"><li><a href="http://dx.doi.org/10.1128/iai.70.11.6032-6042.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acquisition+of+Mn%28II%29+in+addition+to+Fe%28II%29+is+required+for+full+virulence+of+Salmonella+enterica+serovar+Typhimurium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acquisition+of+Mn%28II%29+in+addition+to+Fe%28II%29+is+required+for+full+virulence+of+Salmonella+enterica+serovar+Typhimurium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001120-Haveri1" id="pone.0001120-Haveri1"></a>Haveri M, Taponen S, Vuopio-Varkila J, Salmenlinna S, Pyorala S (2005) Bacterial genotype affects the manifestation and persistence of bovine Staphylococcus aureus intramammary infection. J Clin Microbiol  43: 959–961.  <ul class="find" data-citedArticleID="1012846" data-doi="10.1128/jcm.43.2.959-961.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.2.959-961.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+genotype+affects+the+manifestation+and+persistence+of+bovine+Staphylococcus+aureus+intramammary+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+genotype+affects+the+manifestation+and+persistence+of+bovine+Staphylococcus+aureus+intramammary+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001120-Kuroda1" id="pone.0001120-Kuroda1"></a>Kuroda M, Ohta T, Uchiyama I, Baba T, Yuzawa H, et al.  (2001) Whole genome sequencing of meticillin-resistant Staphylococcus aureus. Lancet  357: 1225–1240.  <ul class="find" data-citedArticleID="1012864" data-doi="10.1016/s0140-6736(00)04403-2"><li><a href="http://dx.doi.org/10.1016/s0140-6736(00)04403-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole+genome+sequencing+of+meticillin-resistant+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole+genome+sequencing+of+meticillin-resistant+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001120-Baba1" id="pone.0001120-Baba1"></a>Baba T, Takeuchi F, Kuroda M, Yuzawa H, Aoki K, et al.  (2002) Genome and virulence determinants of high virulence community-acquired MRSA. Lancet  359: 1819–1827.  <ul class="find" data-citedArticleID="1012796" data-doi="10.1016/s0140-6736(02)08713-5"><li><a href="http://dx.doi.org/10.1016/s0140-6736(02)08713-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+and+virulence+determinants+of+high+virulence+community-acquired+MRSA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+and+virulence+determinants+of+high+virulence+community-acquired+MRSA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001120-Holden1" id="pone.0001120-Holden1"></a>Holden MT, Feil EJ, Lindsay JA, Peacock SJ, Day NP, et al.  (2004) Complete genomes of two clinical Staphylococcus aureus strains: evidence for the rapid evolution of virulence and drug resistance. Proc Natl Acad Sci U S A  101: 9786–9791.  <ul class="find" data-citedArticleID="1012852" data-doi="10.1073/pnas.0402521101"><li><a href="http://dx.doi.org/10.1073/pnas.0402521101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genomes+of+two+clinical+Staphylococcus+aureus+strains%3A+evidence+for+the+rapid+evolution+of+virulence+and+drug+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genomes+of+two+clinical+Staphylococcus+aureus+strains%3A+evidence+for+the+rapid+evolution+of+virulence+and+drug+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001120-Gill1" id="pone.0001120-Gill1"></a>Gill SR, Fouts DE, Archer GL, Mongodin EF, Deboy RT, et al.  (2005) Insights on evolution of virulence and resistance from the complete genome analysis of an early methicillin-resistant Staphylococcus aureus strain and a biofilm-producing methicillin-resistant Staphylococcus epidermidis strain. J Bacteriol  187: 2426–2438.  <ul class="find" data-citedArticleID="1012838" data-doi="10.1128/jb.187.7.2426-2438.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.7.2426-2438.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+on+evolution+of+virulence+and+resistance+from+the+complete+genome+analysis+of+an+early+methicillin-resistant+Staphylococcus+aureus+strain+and+a+biofilm-producing+methicillin-resistant+Staphylococcus+epidermidis+strain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+on+evolution+of+virulence+and+resistance+from+the+complete+genome+analysis+of+an+early+methicillin-resistant+Staphylococcus+aureus+strain+and+a+biofilm-producing+methicillin-resistant+Staphylococcus+epidermidis+strain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001120-Witney1" id="pone.0001120-Witney1"></a>Witney AA, Marsden GL, Holden MT, Stabler RA, Husain SE, et al.  (2005) Design, validation, and application of a seven-strain Staphylococcus aureus PCR product microarray for comparative genomics. Appl Environ Microbiol  71: 7504–7514.  <ul class="find" data-citedArticleID="1012930" data-doi="10.1128/aem.71.11.7504-7514.2005"><li><a href="http://dx.doi.org/10.1128/aem.71.11.7504-7514.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Design%2C+validation%2C+and+application+of+a+seven-strain+Staphylococcus+aureus+PCR+product+microarray+for+comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Design%2C+validation%2C+and+application+of+a+seven-strain+Staphylococcus+aureus+PCR+product+microarray+for+comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001120-Beres1" id="pone.0001120-Beres1"></a>Beres SB, Sylva GL, Sturdevant DE, Granville CN, Liu M, et al.  (2004) Genome-wide molecular dissection of serotype M3 group A Streptococcus strains causing two epidemics of invasive infections. Proc Natl Acad Sci U S A  101: 11833–11838.  <ul class="find" data-citedArticleID="1012804" data-doi="10.1073/pnas.0404163101"><li><a href="http://dx.doi.org/10.1073/pnas.0404163101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+molecular+dissection+of+serotype+M3+group+A+Streptococcus+strains+causing+two+epidemics+of+invasive+infections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+molecular+dissection+of+serotype+M3+group+A+Streptococcus+strains+causing+two+epidemics+of+invasive+infections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001120-Fitzgerald2" id="pone.0001120-Fitzgerald2"></a>Fitzgerald JR, Sturdevant DE, Mackie SM, Gill SR, Musser JM (2001) Evolutionary genomics of Staphylococcus aureus: insights into the origin of methicillin-resistant strains and the toxic shock syndrome epidemic. Proc Natl Acad Sci U S A  98: 8821–8826.  <ul class="find" data-citedArticleID="1012832" data-doi="10.1073/pnas.161098098"><li><a href="http://dx.doi.org/10.1073/pnas.161098098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+genomics+of+Staphylococcus+aureus%3A+insights+into+the+origin+of+methicillin-resistant+strains+and+the+toxic+shock+syndrome+epidemic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+genomics+of+Staphylococcus+aureus%3A+insights+into+the+origin+of+methicillin-resistant+strains+and+the+toxic+shock+syndrome+epidemic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001120-Fitzgerald3" id="pone.0001120-Fitzgerald3"></a>Fitzgerald JR, Monday SR, Foster TJ, Bohach GA, Hartigan PJ, et al.  (2001) Characterization of a putative pathogenicity island from bovine Staphylococcus aureus encoding multiple superantigens. J Bacteriol  183: 63–70.  <ul class="find" data-citedArticleID="1012834" data-doi="10.1128/jb.183.1.63-70.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.1.63-70.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+putative+pathogenicity+island+from+bovine+Staphylococcus+aureus+encoding+multiple+superantigens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+putative+pathogenicity+island+from+bovine+Staphylococcus+aureus+encoding+multiple+superantigens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001120-Nizet1" id="pone.0001120-Nizet1"></a>Nizet V, Beall B, Bast DJ, Datta V, Kilburn L, et al.  (2000) Genetic locus for streptolysin S production by group A streptococcus. Infect Immun  68: 4245–4254.  <ul class="find" data-citedArticleID="1012888" data-doi="10.1128/iai.68.7.4245-4254.2000"><li><a href="http://dx.doi.org/10.1128/iai.68.7.4245-4254.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+locus+for+streptolysin+S+production+by+group+A+streptococcus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+locus+for+streptolysin+S+production+by+group+A+streptococcus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001120-Iandolo1" id="pone.0001120-Iandolo1"></a>Iandolo JJ, Worrell V, Groicher KH, Qian Y, Tian R, et al.  (2002) Comparative analysis of the genomes of the temperate bacteriophages phi 11, phi 12 and phi 13 of Staphylococcus aureus 8325. Gene  289: 109–118.  <ul class="find" data-citedArticleID="1012854" data-doi="10.1016/s0378-1119(02)00481-x"><li><a href="http://dx.doi.org/10.1016/s0378-1119(02)00481-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+the+genomes+of+the+temperate+bacteriophages+phi+11%2C+phi+12+and+phi+13+of+Staphylococcus+aureus+8325." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+the+genomes+of+the+temperate+bacteriophages+phi+11%2C+phi+12+and+phi+13+of+Staphylococcus+aureus+8325.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001120-Nei1" id="pone.0001120-Nei1"></a>Nei M, Gojobori T (1986) Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions. Mol Biol Evol  3: 418–426.  <ul class="find" data-citedArticleID="1012886"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.&amp;auth=&amp;atitle=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001120-Josefsson1" id="pone.0001120-Josefsson1"></a>Josefsson E, McCrea KW, Ni Eidhin D, O'Connell D, Cox J, et al.  (1998) Three new members of the serine-aspartate repeat protein multigene family of Staphylococcus aureus. Microbiology 144 (Pt  12): 3387–3395.  <ul class="find" data-citedArticleID="1012860" data-doi="10.1099/00221287-144-12-3387"><li><a href="http://dx.doi.org/10.1099/00221287-144-12-3387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three+new+members+of+the+serine-aspartate+repeat+protein+multigene+family+of+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three+new+members+of+the+serine-aspartate+repeat+protein+multigene+family+of+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001120-Clarke1" id="pone.0001120-Clarke1"></a>Clarke SR, Harris LG, Richards RG, Foster SJ (2002) Analysis of Ebh, a 1.1-megadalton cell wall-associated fibronectin-binding protein of Staphylococcus aureus. Infect Immun  70: 6680–6687.  <ul class="find" data-citedArticleID="1012816" data-doi="10.1128/iai.70.12.6680-6687.2002"><li><a href="http://dx.doi.org/10.1128/iai.70.12.6680-6687.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+Ebh%2C+a+1.1-megadalton+cell+wall-associated+fibronectin-binding+protein+of+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+Ebh%2C+a+1.1-megadalton+cell+wall-associated+fibronectin-binding+protein+of+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001120-Schuberth1" id="pone.0001120-Schuberth1"></a>Schuberth HJ, Krueger C, Zerbe H, Bleckmann E, Leibold W (2001) Characterization of leukocytotoxic and superantigen-like factors produced by Staphylococcus aureus isolates from milk of cows with mastitis. Vet Microbiol  82: 187–199.  <ul class="find" data-citedArticleID="1012906" data-doi="10.1016/s0378-1135(01)00386-8"><li><a href="http://dx.doi.org/10.1016/s0378-1135(01)00386-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+leukocytotoxic+and+superantigen-like+factors+produced+by+Staphylococcus+aureus+isolates+from+milk+of+cows+with+mastitis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+leukocytotoxic+and+superantigen-like+factors+produced+by+Staphylococcus+aureus+isolates+from+milk+of+cows+with+mastitis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001120-Fueyo1" id="pone.0001120-Fueyo1"></a>Fueyo JM, Mendoza MC, Rodicio MR, Muniz J, Alvarez MA, et al.  (2005) Cytotoxin and pyrogenic toxin superantigen gene profiles of Staphylococcus aureus associated with subclinical mastitis in dairy cows and relationships with macrorestriction genomic profiles. J Clin Microbiol  43: 1278–1284.  <ul class="find" data-citedArticleID="1012836" data-doi="10.1128/jcm.43.3.1278-1284.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.3.1278-1284.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytotoxin+and+pyrogenic+toxin+superantigen+gene+profiles+of+Staphylococcus+aureus+associated+with+subclinical+mastitis+in+dairy+cows+and+relationships+with+macrorestriction+genomic+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytotoxin+and+pyrogenic+toxin+superantigen+gene+profiles+of+Staphylococcus+aureus+associated+with+subclinical+mastitis+in+dairy+cows+and+relationships+with+macrorestriction+genomic+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001120-Zou1" id="pone.0001120-Zou1"></a>Zou D, Kaneko J, Narita S, Kamio Y (2000) Prophage, phiPV83-pro, carrying panton-valentine leukocidin genes, on the Staphylococcus aureus P83 chromosome: comparative analysis of the genome structures of phiPV83-pro, phiPVL, phi11, and other phages. Biosci Biotechnol Biochem  64: 2631–2643.  <ul class="find" data-citedArticleID="1012938" data-doi="10.1271/bbb.64.2631"><li><a href="http://dx.doi.org/10.1271/bbb.64.2631" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prophage%2C+phiPV83-pro%2C+carrying+panton-valentine+leukocidin+genes%2C+on+the+Staphylococcus+aureus+P83+chromosome%3A+comparative+analysis+of+the+genome+structures+of+phiPV83-pro%2C+phiPVL%2C+phi11%2C+and+other+phages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prophage%2C+phiPV83-pro%2C+carrying+panton-valentine+leukocidin+genes%2C+on+the+Staphylococcus+aureus+P83+chromosome%3A+comparative+analysis+of+the+genome+structures+of+phiPV83-pro%2C+phiPVL%2C+phi11%2C+and+other+phages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001120-Yamada1" id="pone.0001120-Yamada1"></a>Yamada T, Tochimaru N, Nakasuji S, Hata E, Kobayashi H, et al.  (2005) Leukotoxin family genes in Staphylococcus aureus isolated from domestic animals and prevalence of lukM-lukF-PV genes by bacteriophages in bovine isolates. Vet Microbiol  110: 97–103.  <ul class="find" data-citedArticleID="1012934" data-doi="10.1016/j.vetmic.2005.07.006"><li><a href="http://dx.doi.org/10.1016/j.vetmic.2005.07.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Leukotoxin+family+genes+in+Staphylococcus+aureus+isolated+from+domestic+animals+and+prevalence+of+lukM-lukF-PV+genes+by+bacteriophages+in+bovine+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Leukotoxin+family+genes+in+Staphylococcus+aureus+isolated+from+domestic+animals+and+prevalence+of+lukM-lukF-PV+genes+by+bacteriophages+in+bovine+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001120-Barrio1" id="pone.0001120-Barrio1"></a>Barrio MB, Rainard P, Prevost G (2006) LukM/LukF'-PV is the most active Staphylococcus aureus leukotoxin on bovine neutrophils. Microbes Infect  8: 2068–2074.  <ul class="find" data-citedArticleID="1012798" data-doi="10.1016/j.micinf.2006.03.004"><li><a href="http://dx.doi.org/10.1016/j.micinf.2006.03.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LukM%2FLukF%27-PV+is+the+most+active+Staphylococcus+aureus+leukotoxin+on+bovine+neutrophils." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LukM%2FLukF%27-PV+is+the+most+active+Staphylococcus+aureus+leukotoxin+on+bovine+neutrophils.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001120-Luong1" id="pone.0001120-Luong1"></a>Luong TT, Ouyang S, Bush K, Lee CY (2002) Type 1 capsule genes of Staphylococcus aureus are carried in a staphylococcal cassette chromosome genetic element. J Bacteriol  184: 3623–3629.  <ul class="find" data-citedArticleID="1012874" data-doi="10.1128/jb.184.13.3623-3629.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.13.3623-3629.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Type+1+capsule+genes+of+Staphylococcus+aureus+are+carried+in+a+staphylococcal+cassette+chromosome+genetic+element." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Type+1+capsule+genes+of+Staphylococcus+aureus+are+carried+in+a+staphylococcal+cassette+chromosome+genetic+element.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001120-Wilson1" id="pone.0001120-Wilson1"></a>Wilson DJ, Gonzalez RN, Case KL, Garrison LL, Grohn YT (1999) Comparison of seven antibiotic treatments with no treatment for bacteriological efficacy against bovine mastitis pathogens. J Dairy Sci  82: 1664–1670.  <ul class="find" data-citedArticleID="1012928"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparison+of+seven+antibiotic+treatments+with+no+treatment+for+bacteriological+efficacy+against+bovine+mastitis+pathogens.&amp;auth=&amp;atitle=Comparison+of+seven+antibiotic+treatments+with+no+treatment+for+bacteriological+efficacy+against+bovine+mastitis+pathogens." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+seven+antibiotic+treatments+with+no+treatment+for+bacteriological+efficacy+against+bovine+mastitis+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+seven+antibiotic+treatments+with+no+treatment+for+bacteriological+efficacy+against+bovine+mastitis+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001120-Erskine1" id="pone.0001120-Erskine1"></a>Erskine RJ, Walker RD, Bolin CA, Bartlett PC, White DG (2002) Trends in antibacterial susceptibility of mastitis pathogens during a seven-year period. J Dairy Sci  85: 1111–1118.  <ul class="find" data-citedArticleID="1012826"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Trends+in+antibacterial+susceptibility+of+mastitis+pathogens+during+a+seven-year+period.&amp;auth=&amp;atitle=Trends+in+antibacterial+susceptibility+of+mastitis+pathogens+during+a+seven-year+period." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trends+in+antibacterial+susceptibility+of+mastitis+pathogens+during+a+seven-year+period." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trends+in+antibacterial+susceptibility+of+mastitis+pathogens+during+a+seven-year+period.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001120-Tikofsky1" id="pone.0001120-Tikofsky1"></a>Tikofsky LL, Barlow JW, Santisteban C, Schukken YH (2003) A comparison of antimicrobial susceptibility patterns for Staphylococcus aureus in organic and conventional dairy herds. Microb Drug Resist  9: Suppl 1S39–45.  <ul class="find" data-citedArticleID="1012918" data-doi="10.1089/107662903322541883"><li><a href="http://dx.doi.org/10.1089/107662903322541883" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+antimicrobial+susceptibility+patterns+for+Staphylococcus+aureus+in+organic+and+conventional+dairy+herds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+antimicrobial+susceptibility+patterns+for+Staphylococcus+aureus+in+organic+and+conventional+dairy+herds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001120-Lindsay1" id="pone.0001120-Lindsay1"></a>Lindsay JA, Riley TV (1994) Staphylococcal iron requirements, siderophore production, and iron-regulated protein expression. Infect Immun  62: 2309–2314.  <ul class="find" data-citedArticleID="1012868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Staphylococcal+iron+requirements%2C+siderophore+production%2C+and+iron-regulated+protein+expression.&amp;auth=&amp;atitle=Staphylococcal+iron+requirements%2C+siderophore+production%2C+and+iron-regulated+protein+expression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Staphylococcal+iron+requirements%2C+siderophore+production%2C+and+iron-regulated+protein+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Staphylococcal+iron+requirements%2C+siderophore+production%2C+and+iron-regulated+protein+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001120-Trivier1" id="pone.0001120-Trivier1"></a>Trivier D, Davril M, Houdret N, Courcol RJ (1995) Influence of iron depletion on growth kinetics, siderophore production, and protein expression of Staphylococcus aureus. FEMS Microbiol Lett  127: 195–199.  <ul class="find" data-citedArticleID="1012920" data-doi="10.1111/j.1574-6968.1995.tb07473.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.1995.tb07473.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+iron+depletion+on+growth+kinetics%2C+siderophore+production%2C+and+protein+expression+of+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+iron+depletion+on+growth+kinetics%2C+siderophore+production%2C+and+protein+expression+of+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001120-Trivier2" id="pone.0001120-Trivier2"></a>Trivier D, Courcol RJ (1996) Iron depletion and virulence in Staphylococcus aureus. FEMS Microbiol Lett  141: 117–127.  <ul class="find" data-citedArticleID="1012922" data-doi="10.1111/j.1574-6968.1996.tb08373.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.1996.tb08373.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Iron+depletion+and+virulence+in+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Iron+depletion+and+virulence+in+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001120-Mazmanian1" id="pone.0001120-Mazmanian1"></a>Mazmanian SK, Skaar EP, Gaspar AH, Humayun M, Gornicki P, et al.  (2003) Passage of heme-iron across the envelope of Staphylococcus aureus. Science  299: 906–909.  <ul class="find" data-citedArticleID="1012876" data-doi="10.1126/science.1081147"><li><a href="http://dx.doi.org/10.1126/science.1081147" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Passage+of+heme-iron+across+the+envelope+of+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Passage+of+heme-iron+across+the+envelope+of+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001120-Skaar1" id="pone.0001120-Skaar1"></a>Skaar EP, Schneewind O (2004) Iron-regulated surface determinants (Isd) of Staphylococcus aureus: stealing iron from heme. Microbes Infect  6: 390–397.  <ul class="find" data-citedArticleID="1012908" data-doi="10.1016/j.micinf.2003.12.008"><li><a href="http://dx.doi.org/10.1016/j.micinf.2003.12.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Iron-regulated+surface+determinants+%28Isd%29+of+Staphylococcus+aureus%3A+stealing+iron+from+heme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Iron-regulated+surface+determinants+%28Isd%29+of+Staphylococcus+aureus%3A+stealing+iron+from+heme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001120-Takeuchi1" id="pone.0001120-Takeuchi1"></a>Takeuchi S, Maeda T, Hashimoto N, Imaizumi K, Kaidoh T, et al.  (2001) Variation of the agr locus in Staphylococcus aureus isolates from cows with mastitis. Vet Microbiol  79: 267–274.  <ul class="find" data-citedArticleID="1012914" data-doi="10.1016/s0378-1135(00)00354-0"><li><a href="http://dx.doi.org/10.1016/s0378-1135(00)00354-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+of+the+agr+locus+in+Staphylococcus+aureus+isolates+from+cows+with+mastitis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+of+the+agr+locus+in+Staphylococcus+aureus+isolates+from+cows+with+mastitis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001120-Gilot1" id="pone.0001120-Gilot1"></a>Gilot P, van Leeuwen W (2004) Comparative analysis of agr locus diversification and overall genetic variability among bovine and human Staphylococcus aureus isolates. J Clin Microbiol  42: 1265–1269.  <ul class="find" data-citedArticleID="1012840" data-doi="10.1128/jcm.42.3.1265-1269.2004"><li><a href="http://dx.doi.org/10.1128/jcm.42.3.1265-1269.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+agr+locus+diversification+and+overall+genetic+variability+among+bovine+and+human+Staphylococcus+aureus+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+agr+locus+diversification+and+overall+genetic+variability+among+bovine+and+human+Staphylococcus+aureus+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001120-Recsei1" id="pone.0001120-Recsei1"></a>Recsei P, Kreiswirth B, O'Reilly M, Schlievert P, Gruss A, et al.  (1986) Regulation of exoprotein gene expression in Staphylococcus aureus by agar. Mol Gen Genet  202: 58–61.  <ul class="find" data-citedArticleID="1012898" data-doi="10.1007/bf00330517"><li><a href="http://dx.doi.org/10.1007/bf00330517" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+exoprotein+gene+expression+in+Staphylococcus+aureus+by+agar." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+exoprotein+gene+expression+in+Staphylococcus+aureus+by+agar.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001120-Arvidson1" id="pone.0001120-Arvidson1"></a>Arvidson S, Tegmark K (2001) Regulation of virulence determinants in Staphylococcus aureus. Int J Med Microbiol  291: 159–170.  <ul class="find" data-citedArticleID="1012794" data-doi="10.1078/1438-4221-00112"><li><a href="http://dx.doi.org/10.1078/1438-4221-00112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+virulence+determinants+in+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+virulence+determinants+in+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001120-Novick1" id="pone.0001120-Novick1"></a>Novick RP, Ross HF, Projan SJ, Kornblum J, Kreiswirth B, et al.  (1993) Synthesis of staphylococcal virulence factors is controlled by a regulatory RNA molecule. Embo J  12: 3967–3975.  <ul class="find" data-citedArticleID="1012890"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Synthesis+of+staphylococcal+virulence+factors+is+controlled+by+a+regulatory+RNA+molecule.&amp;auth=&amp;atitle=Synthesis+of+staphylococcal+virulence+factors+is+controlled+by+a+regulatory+RNA+molecule." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synthesis+of+staphylococcal+virulence+factors+is+controlled+by+a+regulatory+RNA+molecule." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synthesis+of+staphylococcal+virulence+factors+is+controlled+by+a+regulatory+RNA+molecule.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001120-Ji1" id="pone.0001120-Ji1"></a>Ji G, Beavis R, Novick RP (1997) Bacterial interference caused by autoinducing peptide variants. Science  276: 2027–2030.  <ul class="find" data-citedArticleID="1012858" data-doi="10.1126/science.276.5321.2027"><li><a href="http://dx.doi.org/10.1126/science.276.5321.2027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+interference+caused+by+autoinducing+peptide+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+interference+caused+by+autoinducing+peptide+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001120-Jarraud1" id="pone.0001120-Jarraud1"></a>Jarraud S, Lyon GJ, Figueiredo AM, Gerard L, Vandenesch F, et al.  (2000) Exfoliatin-producing strains define a fourth agr specificity group in Staphylococcus aureus. J Bacteriol  182: 6517–6522.  <ul class="find" data-citedArticleID="1012856" data-doi="10.1128/jb.182.22.6517-6522.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.22.6517-6522.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exfoliatin-producing+strains+define+a+fourth+agr+specificity+group+in+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exfoliatin-producing+strains+define+a+fourth+agr+specificity+group+in+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001120-Wright1" id="pone.0001120-Wright1"></a>Wright JS 3rd, Traber KE, Corrigan R, Benson SA, Musser JM, et al.  (2005) The agr radiation: an early event in the evolution of staphylococci. J Bacteriol  187: 5585–5594.  <ul class="find" data-citedArticleID="1012932" data-doi="10.1128/jb.187.16.5585-5594.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.16.5585-5594.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+agr+radiation%3A+an+early+event+in+the+evolution+of+staphylococci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+agr+radiation%3A+an+early+event+in+the+evolution+of+staphylococci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001120-Lindsay2" id="pone.0001120-Lindsay2"></a>Lindsay JA, Moore CE, Day NP, Peacock SJ, Witney AA, et al.  (2006) Microarrays reveal that each of the ten dominant lineages of Staphylococcus aureus has a unique combination of surface-associated and regulatory genes. J Bacteriol  188: 669–676.  <ul class="find" data-citedArticleID="1012870" data-doi="10.1128/jb.188.2.669-676.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.2.669-676.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarrays+reveal+that+each+of+the+ten+dominant+lineages+of+Staphylococcus+aureus+has+a+unique+combination+of+surface-associated+and+regulatory+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarrays+reveal+that+each+of+the+ten+dominant+lineages+of+Staphylococcus+aureus+has+a+unique+combination+of+surface-associated+and+regulatory+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001120-Moxon1" id="pone.0001120-Moxon1"></a>Moxon R, Bayliss C, Hood D (2006) Bacterial contingency loci: the role of simple sequence DNA repeats in bacterial adaptation. Annu Rev Genet  40: 307–333.  <ul class="find" data-citedArticleID="1012880" data-doi="10.1146/annurev.genet.40.110405.090442"><li><a href="http://dx.doi.org/10.1146/annurev.genet.40.110405.090442" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+contingency+loci%3A+the+role+of+simple+sequence+DNA+repeats+in+bacterial+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+contingency+loci%3A+the+role+of+simple+sequence+DNA+repeats+in+bacterial+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001120-Bayliss1" id="pone.0001120-Bayliss1"></a>Bayliss CD, Field D, Moxon ER (2001) The simple sequence contingency loci of Haemophilus influenzae and Neisseria meningitidis. J Clin Invest  107: 657–662.  <ul class="find" data-citedArticleID="1012802" data-doi="10.1172/jci12557"><li><a href="http://dx.doi.org/10.1172/jci12557" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+simple+sequence+contingency+loci+of+Haemophilus+influenzae+and+Neisseria+meningitidis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+simple+sequence+contingency+loci+of+Haemophilus+influenzae+and+Neisseria+meningitidis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001120-OBrien1" id="pone.0001120-OBrien1"></a>O'Brien DP, Israel DA, Krishna U, Romero-Gallo J, Nedrud J, et al.  (2006) The role of decay-accelerating factor as a receptor for Helicobacter pylori and a mediator of gastric inflammation. J Biol Chem  281: 13317–13323.  <ul class="find" data-citedArticleID="1012892" data-doi="10.1074/jbc.m601805200"><li><a href="http://dx.doi.org/10.1074/jbc.m601805200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+decay-accelerating+factor+as+a+receptor+for+Helicobacter+pylori+and+a+mediator+of+gastric+inflammation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+decay-accelerating+factor+as+a+receptor+for+Helicobacter+pylori+and+a+mediator+of+gastric+inflammation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001120-Parkhill1" id="pone.0001120-Parkhill1"></a>Parkhill J, Wren BW, Thomson NR, Titball RW, Holden MT, et al.  (2001) Genome sequence of Yersinia pestis, the causative agent of plague. Nature  413: 523–527.  <ul class="find" data-citedArticleID="1012894" data-doi="10.1038/35097083"><li><a href="http://dx.doi.org/10.1038/35097083" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+Yersinia+pestis%2C+the+causative+agent+of+plague." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+Yersinia+pestis%2C+the+causative+agent+of+plague.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001120-Cole1" id="pone.0001120-Cole1"></a>Cole ST, Eiglmeier K, Parkhill J, James KD, Thomson NR, et al.  (2001) Massive gene decay in the leprosy bacillus. Nature  409: 1007–1011.  <ul class="find" data-citedArticleID="1012818" data-doi="10.1038/35059006"><li><a href="http://dx.doi.org/10.1038/35059006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Massive+gene+decay+in+the+leprosy+bacillus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Massive+gene+decay+in+the+leprosy+bacillus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001120-Eiglmeier1" id="pone.0001120-Eiglmeier1"></a>Eiglmeier K, Parkhill J, Honore N, Garnier T, Tekaia F, et al.  (2001) The decaying genome of Mycobacterium leprae. Lepr Rev  72: 387–398.  <ul class="find" data-citedArticleID="1012822"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+decaying+genome+of+Mycobacterium+leprae.&amp;auth=&amp;atitle=The+decaying+genome+of+Mycobacterium+leprae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+decaying+genome+of+Mycobacterium+leprae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+decaying+genome+of+Mycobacterium+leprae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001120-Bayles1" id="pone.0001120-Bayles1"></a>Bayles KW, Wesson CA, Liou LE, Fox LK, Bohach GA, et al.  (1998) Intracellular Staphylococcus aureus escapes the endosome and induces apoptosis in epithelial cells. Infect Immun  66: 336–342.  <ul class="find" data-citedArticleID="1012800"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Intracellular+Staphylococcus+aureus+escapes+the+endosome+and+induces+apoptosis+in+epithelial+cells.&amp;auth=&amp;atitle=Intracellular+Staphylococcus+aureus+escapes+the+endosome+and+induces+apoptosis+in+epithelial+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intracellular+Staphylococcus+aureus+escapes+the+endosome+and+induces+apoptosis+in+epithelial+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intracellular+Staphylococcus+aureus+escapes+the+endosome+and+induces+apoptosis+in+epithelial+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001120-Brouillette1" id="pone.0001120-Brouillette1"></a>Brouillette E, Grondin G, Shkreta L, Lacasse P, Talbot BG (2003) In vivo and in vitro demonstration that Staphylococcus aureus is an intracellular pathogen in the presence or absence of fibronectin-binding proteins. Microb Pathog  35: 159–168.  <ul class="find" data-citedArticleID="1012812" data-doi="10.1016/s0882-4010(03)00112-8"><li><a href="http://dx.doi.org/10.1016/s0882-4010(03)00112-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+and+in+vitro+demonstration+that+Staphylococcus+aureus+is+an+intracellular+pathogen+in+the+presence+or+absence+of+fibronectin-binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+and+in+vitro+demonstration+that+Staphylococcus+aureus+is+an+intracellular+pathogen+in+the+presence+or+absence+of+fibronectin-binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001120-Lowy1" id="pone.0001120-Lowy1"></a>Lowy FD (2000) Is Staphylococcus aureus an intracellular pathogen? Trends Microbiol  8: 341–343.  <ul class="find" data-citedArticleID="1012872" data-doi="10.1016/s0966-842x(00)01803-5"><li><a href="http://dx.doi.org/10.1016/s0966-842x(00)01803-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+Staphylococcus+aureus+an+intracellular+pathogen%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+Staphylococcus+aureus+an+intracellular+pathogen%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001120-Gresham1" id="pone.0001120-Gresham1"></a>Gresham HD, Lowrance JH, Caver TE, Wilson BS, Cheung AL, et al.  (2000) Survival of Staphylococcus aureus inside neutrophils contributes to infection. J Immunol  164: 3713–3722.  <ul class="find" data-citedArticleID="1012844"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Survival+of+Staphylococcus+aureus+inside+neutrophils+contributes+to+infection.&amp;auth=&amp;atitle=Survival+of+Staphylococcus+aureus+inside+neutrophils+contributes+to+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Survival+of+Staphylococcus+aureus+inside+neutrophils+contributes+to+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Survival+of+Staphylococcus+aureus+inside+neutrophils+contributes+to+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001120-Retzer1" id="pone.0001120-Retzer1"></a>Retzer MD, Kabani A, Button LL, Yu RH, Schryvers AB (1996) Production and characterization of chimeric transferrins for the determination of the binding domains for bacterial transferrin receptors. J Biol Chem  271: 1166–1173.  <ul class="find" data-citedArticleID="1012904" data-doi="10.1074/jbc.271.2.1166"><li><a href="http://dx.doi.org/10.1074/jbc.271.2.1166" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Production+and+characterization+of+chimeric+transferrins+for+the+determination+of+the+binding+domains+for+bacterial+transferrin+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Production+and+characterization+of+chimeric+transferrins+for+the+determination+of+the+binding+domains+for+bacterial+transferrin+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001120-Bitter1" id="pone.0001120-Bitter1"></a>Bitter W, Gerrits H, Kieft R, Borst P (1998) The role of transferrin-receptor variation in the host range of Trypanosoma brucei. Nature  391: 499–502.  <ul class="find" data-citedArticleID="1012808" data-doi="10.1038/35166"><li><a href="http://dx.doi.org/10.1038/35166" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+transferrin-receptor+variation+in+the+host+range+of+Trypanosoma+brucei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+transferrin-receptor+variation+in+the+host+range+of+Trypanosoma+brucei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001120-Herron1" id="pone.0001120-Herron1"></a>Herron LL, Chakravarty R, Dwan C, Fitzgerald JR, Musser JM, et al.  (2002) Genome sequence survey identifies unique sequences and key virulence genes with unusual rates of amino Acid substitution in bovine Staphylococcus aureus. Infect Immun  70: 3978–3981.  <ul class="find" data-citedArticleID="1012848" data-doi="10.1128/iai.70.7.3978-3981.2002"><li><a href="http://dx.doi.org/10.1128/iai.70.7.3978-3981.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+survey+identifies+unique+sequences+and+key+virulence+genes+with+unusual+rates+of+amino+Acid+substitution+in+bovine+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+survey+identifies+unique+sequences+and+key+virulence+genes+with+unusual+rates+of+amino+Acid+substitution+in+bovine+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001120-Ewing1" id="pone.0001120-Ewing1"></a>Ewing B, Green P (1998) Base-calling of automated sequencer traces using phred. II. Error probabilities. Genome Res  8: 186–194.  <ul class="find" data-citedArticleID="1012828" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001120-HerronOlson1" id="pone.0001120-HerronOlson1"></a>Herron-Olson L, Freeman J, Zhang Q, Retzel EF, Kapur V (2003) MGView: an alignment and visualization tool to enhance gap closure of microbial genomes. Nucleic Acids Res  31: e106.  <ul class="find" data-citedArticleID="1012850" data-doi="10.1093/nar/gng107"><li><a href="http://dx.doi.org/10.1093/nar/gng107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MGView%3A+an+alignment+and+visualization+tool+to+enhance+gap+closure+of+microbial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MGView%3A+an+alignment+and+visualization+tool+to+enhance+gap+closure+of+microbial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001120-Gordon1" id="pone.0001120-Gordon1"></a>Gordon D, Abajian C, Green P (1998) Consed: a graphical tool for sequence finishing. Genome Res  8: 195–202.  <ul class="find" data-citedArticleID="1012842" data-doi="10.1101/gr.8.3.195"><li><a href="http://dx.doi.org/10.1101/gr.8.3.195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consed%3A+a+graphical+tool+for+sequence+finishing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consed%3A+a+graphical+tool+for+sequence+finishing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001120-Berriman1" id="pone.0001120-Berriman1"></a>Berriman M, Rutherford K (2003) Viewing and annotating sequence data with Artemis. Brief Bioinform  4: 124–132.  <ul class="find" data-citedArticleID="1012806" data-doi="10.1093/bib/4.2.124"><li><a href="http://dx.doi.org/10.1093/bib/4.2.124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viewing+and+annotating+sequence+data+with+Artemis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Viewing+and+annotating+sequence+data+with+Artemis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001120-McCarthy1" id="pone.0001120-McCarthy1"></a>McCarthy JE, Gualerzi C (1990) Translational control of prokaryotic gene expression. Trends Genet  6: 78–85.  <ul class="find" data-citedArticleID="1012878" data-doi="10.1016/0168-9525(90)90098-q"><li><a href="http://dx.doi.org/10.1016/0168-9525(90)90098-q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translational+control+of+prokaryotic+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translational+control+of+prokaryotic+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001120-Altschul1" id="pone.0001120-Altschul1"></a>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J Mol Biol  215: 403–410.  <ul class="find" data-citedArticleID="1012790" data-doi="10.1016/s0022-2836(05)80360-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80360-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Basic+local+alignment+search+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Basic+local+alignment+search+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001120-Thompson1" id="pone.0001120-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="1012916" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001120-Enright1" id="pone.0001120-Enright1"></a>Enright MC, Day NP, Davies CE, Peacock SJ, Spratt BG (2000) Multilocus sequence typing for characterization of methicillin-resistant and methicillin-susceptible clones of Staphylococcus aureus. J Clin Microbiol  38: 1008–1015.  <ul class="find" data-citedArticleID="1012824"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multilocus+sequence+typing+for+characterization+of+methicillin-resistant+and+methicillin-susceptible+clones+of+Staphylococcus+aureus.&amp;auth=&amp;atitle=Multilocus+sequence+typing+for+characterization+of+methicillin-resistant+and+methicillin-susceptible+clones+of+Staphylococcus+aureus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multilocus+sequence+typing+for+characterization+of+methicillin-resistant+and+methicillin-susceptible+clones+of+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multilocus+sequence+typing+for+characterization+of+methicillin-resistant+and+methicillin-susceptible+clones+of+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001120-Smith1" id="pone.0001120-Smith1"></a>Smith TF, Waterman MS (1981) Identification of common molecular subsequences. J Mol Biol  147: 195–197.  <ul class="find" data-citedArticleID="1012910" data-doi="10.1016/0022-2836(81)90087-5"><li><a href="http://dx.doi.org/10.1016/0022-2836(81)90087-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+common+molecular+subsequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+common+molecular+subsequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001120-Swofford1" id="pone.0001120-Swofford1"></a>Swofford DL, Waddell PJ, Huelsenbeck JP, Foster PG, Lewis PO, et al.  (2001) Bias in phylogenetic estimation and its relevance to the choice between parsimony and likelihood methods. Syst Biol  50: 525–539.  <ul class="find" data-citedArticleID="1012912"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bias+in+phylogenetic+estimation+and+its+relevance+to+the+choice+between+parsimony+and+likelihood+methods.&amp;auth=&amp;atitle=Bias+in+phylogenetic+estimation+and+its+relevance+to+the+choice+between+parsimony+and+likelihood+methods." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bias+in+phylogenetic+estimation+and+its+relevance+to+the+choice+between+parsimony+and+likelihood+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bias+in+phylogenetic+estimation+and+its+relevance+to+the+choice+between+parsimony+and+likelihood+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.XML" value="156644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.PDF" value="889567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g001.PNG_L" value="2803730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g001.PNG_M" value="525712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g001.PNG_S" value="19817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g001.TIF" value="6906422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g001.PNG_I" value="166309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g002.PNG_L" value="1403552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g002.PNG_M" value="157929"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g002.PNG_S" value="12092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g002.TIF" value="3454842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g002.PNG_I" value="49999"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t001.PNG_L" value="157687"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t001.PNG_M" value="85770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t001.PNG_S" value="12410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t001.TIF" value="857610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t001.PNG_I" value="26071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g003.PNG_L" value="904073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g003.PNG_M" value="94081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g003.PNG_S" value="9604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g003.TIF" value="1252736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g003.PNG_I" value="30453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t002.PNG_L" value="164691"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t002.PNG_M" value="204321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t002.PNG_S" value="22232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t002.TIF" value="679794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t002.PNG_I" value="83231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g004.PNG_L" value="204937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g004.PNG_M" value="67138"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g004.PNG_S" value="8367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g004.TIF" value="439470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g004.PNG_I" value="23737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t003.PNG_L" value="86910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t003.PNG_M" value="120461"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t003.PNG_S" value="16610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t003.TIF" value="396840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.t003.PNG_I" value="40501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g005.PNG_L" value="5575223"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g005.PNG_M" value="326692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g005.PNG_S" value="20512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g005.TIF" value="9265068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g005.PNG_I" value="171762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g006.PNG_L" value="27562"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g006.PNG_M" value="54711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g006.PNG_S" value="7869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g006.TIF" value="212976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g006.PNG_I" value="20269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g007.PNG_L" value="403180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g007.PNG_M" value="206294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g007.PNG_S" value="20394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g007.TIF" value="7907578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.g007.PNG_I" value="70537"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001120.s001.DOC" value="1854976"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001120&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001120" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001120&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001120&volume=&issue=&title=Molecular Correlates of Host Specialization in Staphylococcus aureus&author_name=Lisa%20Herron-Olson%2C%20J.%20Ross%20Fitzgerald%2C%20James%20M.%20Musser%2C%20Vivek%20Kapur&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001120" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001120" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001120" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001120&amp;t=Molecular%20Correlates%20of%20Host%20Specialization%20in%20Staphylococcus%20aureus" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001120&title=Molecular%20Correlates%20of%20Host%20Specialization%20in%20Staphylococcus%20aureus&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001120&amp;title=Molecular%20Correlates%20of%20Host%20Specialization%20in%20Staphylococcus%20aureus" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001120" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001120&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Molecular Correlates of Host Specialization in Staphylococcus aureus';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001120';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Molecular%20Correlates%20of%20Host%20Specialization%20in%20Staphylococcus%20aureus http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001120" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001120" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->

<div class="block">
    <div class="header">
        <h3>Included in the Following Collection</h3>
    </div>
    <p><dd><a href="http://www.ploscollections.org/article/browse/issue/info%3Adoi%2F10.1371%2Fissue.pcol.v02.i03" title="Browse the Open-Access Collection"><em>PLoS ONE</em>: Prokaryotic Genome Collection</a></dd></p>
</div>

<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="25074"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bovines%22" title="Search for articles in the subject area:'Bovines'"><div class="flagText">Bovines</div></a>
              <div data-categoryid="42759" data-articleid="25074"
                   data-categoryname="Bovines"
                   class="flagImage" title="Flag 'Bovines' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="25074"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25074"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="25074"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mastitis%22" title="Search for articles in the subject area:'Mastitis'"><div class="flagText">Mastitis</div></a>
              <div data-categoryid="46757" data-articleid="25074"
                   data-categoryname="Mastitis"
                   class="flagImage" title="Flag 'Mastitis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25074"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="25074"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5574'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8092'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2595&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=1863">Academic Editor's comments and Evaluation at F1000 Biology</a><br>Posted by Academic_Editor</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
