

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Positive Selection within the Schizophrenia-Associated GABAA Receptor β2 Gene</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000462"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000462" />

    <meta name="citation_title" content="Positive Selection within the Schizophrenia-Associated GABAA Receptor β2 Gene"/>
    <meta itemprop="name" content="Positive Selection within the Schizophrenia-Associated GABAA Receptor β2 Gene"/>

      <meta name="citation_author" content="Wing-Sze Lo"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China"/>
      <meta name="citation_author" content="Zhiwen Xu"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China"/>
      <meta name="citation_author" content="Zhiliang Yu"/>
            <meta name="citation_author_institution" content="Graduate program of Atmospheric, Marine, and Coastal Environment, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China"/>
      <meta name="citation_author" content="Frank W. Pun"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China"/>
      <meta name="citation_author" content="Siu-Kin Ng"/>
            <meta name="citation_author_institution" content="Graduate Program of Bioengineering, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China"/>
      <meta name="citation_author" content="Jianhuan Chen"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China"/>
      <meta name="citation_author" content="Ka-Lok Tong"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China"/>
      <meta name="citation_author" content="Cunyou Zhao"/>
            <meta name="citation_author_institution" content="Graduate Program of Bioengineering, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China"/>
      <meta name="citation_author" content="Xiaojing Xu"/>
            <meta name="citation_author_institution" content="Graduate Program of Bioengineering, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China"/>
      <meta name="citation_author" content="Shui-Ying Tsang"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China"/>
      <meta name="citation_author" content="Mutsuo Harano"/>
            <meta name="citation_author_institution" content="Department of Neuropsychiatry, Kurume University School of Medicine, Fukuka, Japan"/>
      <meta name="citation_author" content="Gerald Stöber"/>
            <meta name="citation_author_institution" content="Department of Psychiatry and Psychotherapy, University of Würzburg, Würzburg, Germany"/>
      <meta name="citation_author" content="Vishwajit L. Nimgaonkar"/>
            <meta name="citation_author_institution" content="Departments of Psychiatry and Human Genetics, University of Pittsburgh School of Medicine, and Graduate School of Public Health, Pittsburgh, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Hong Xue"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China"/>

    <meta name="citation_date" content="2007/5/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000462.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e462"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Vital statistics, demography, and schizophrenia: editor's introduction.; citation_author=II Gottesman; citation_journal_title=Schizophr Bull; citation_volume=15; citation_number=1; citation_pages=5-7; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Schizophrenia: an update and review.; citation_author=J Austin; citation_journal_title=J Genet Couns; citation_volume=14; citation_number=2; citation_pages=329-340; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Schizophrenia genes, gene expression, and neuropathology: on the matter of their convergence.; citation_author=PJ Harrison; citation_author=DR Weinberger; citation_journal_title=Mol Psychiatry; citation_volume=10; citation_number=3; citation_pages=40-68; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Association of SNPs and haplotypes in GABAA receptor β2 gene with schizophrenia.; citation_author=WS Lo; citation_author=CF Lau; citation_author=Z Xuan; citation_author=CF Chan; citation_author=GY Feng; citation_journal_title=Mol Psychiatry; citation_volume=9; citation_number=4; citation_pages=603-608; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=GABAA receptor subtypes as targets for neuropsychiatric drug development.; citation_author=ER Korpi; citation_author=ST Sinkkonen; citation_journal_title=Pharmacology & Therapeutics; citation_volume=109; citation_number=5; citation_pages=12-32; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Positive association of the human GABAA receptor β2 subunit gene haplotype with schizophrenia in the Chinese Han population.; citation_author=J Liu; citation_author=Y Shi; citation_author=W Tang; citation_author=T Guo; citation_author=D Li; citation_journal_title=Biochem Biophys Res Commun; citation_volume=334; citation_number=6; citation_pages=817-823; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic investigation of chromosome 5q GABAA receptor subunit genes in schizophrenia.; citation_author=TL Petryshen; citation_author=FA Middleton; citation_author=AR Tahl; citation_author=GN Rockwell; citation_author=S Purcell; citation_journal_title=Mol Psychiatry; citation_volume=10; citation_number=7; citation_pages=1074-88, 1057; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of GABRB2 association with schizophrenia in German population with DNA sequencing and one-label extension method for SNP genotyping.; citation_author=Z Yu; citation_author=J Chen; citation_author=H Shi; citation_author=G Stoëber; citation_author=SY Tsang; citation_journal_title=Clin Biochem; citation_volume=39; citation_number=8; citation_pages=210-218; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=GABRB2 association with schizophrenia: commonalities and differences between ethnic groups and clinical subtypes.; citation_author=WS Lo; citation_author=M Harano; citation_author=M Gawlik; citation_author=Z Yu; citation_author=J Chen; citation_journal_title=Biol Psychiatry; citation_volume=61; citation_number=9; citation_pages=653-660; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Association analysis of chromosome 5 GABAA receptor cluster in Japanese schizophrenia patients.; citation_author=M Ikeda; citation_author=N Iwata; citation_author=T Suzuki; citation_author=T Kitajima; citation_author=Y Yamanouchi; citation_journal_title=Biol Psychiatry; citation_volume=58; citation_number=10; citation_pages=440-445; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=No evidence for an association between variants at the GABAA receptor β2 locus and schizophrenia.; citation_author=RA Jamra; citation_author=T Becker; citation_author=N Klopp; citation_author=F Dahdouh; citation_author=TG Schulze; citation_journal_title=Psychiatr Genet; citation_volume=17; citation_number=11; citation_pages=43-45; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Two isoforms of GABAA receptor β2 subunit with different electrophysiological properties: differential expression and genotypical correlations in schizophrenia.; citation_author=C Zhao; citation_author=Z Xu; citation_author=J Chen; citation_author=Z Yu; citation_author=KL Tong; citation_journal_title=Mol Psychiatry; citation_volume=11; citation_number=12; citation_pages=1092-1105; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mammalian retroelements.; citation_author=PL Deininger; citation_author=MA Batzer; citation_journal_title=Genome Res; citation_volume=12; citation_number=13; citation_pages=1455-1465; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Recently integrated Alu elements and human genomic diversity.; citation_author=AH Salem; citation_author=GE Kilroy; citation_author=WS Watkins; citation_author=LB Jorde; citation_author=MA Batzer; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=14; citation_pages=1349-1361; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Alu repeats and human disease.; citation_author=PL Deininger; citation_author=MA Batzer; citation_journal_title=Mol Genet Metab; citation_volume=67; citation_number=15; citation_pages=183-193; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The sequence of the human genome.; citation_author=JC Venter; citation_author=MD Adams; citation_author=EW Myers; citation_author=PW Li; citation_author=RJ Mural; citation_journal_title=Science; citation_volume=291; citation_number=16; citation_pages=1304-1351; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Initial sequence of the chimpanzee genome and comparison with the human genome.; citation_journal_title=Nature; citation_volume=437; citation_number=17; citation_pages=69-87; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A test of neutral molecular evolution based on nucleotide data.; citation_author=RR Hudson; citation_author=M Kreitman; citation_author=M Aguade; citation_journal_title=Genetics; citation_volume=116; citation_number=18; citation_pages=153-159; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Functional variation and evolution of non-coding DNA.; citation_author=CP Bird; citation_author=BE Stranger; citation_author=ET Dermitzakis; citation_journal_title=Curr Opin Genet Dev; citation_volume=16; citation_number=19; citation_pages=559-564; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Conserved noncoding sequences are selectively constrained and not mutation cold spots.; citation_author=JA Drake; citation_author=C Bird; citation_author=J Nemesh; citation_author=DJ Thomas; citation_author=C Newton-Cheh; citation_journal_title=Nat Genet; citation_volume=38; citation_number=20; citation_pages=223-227; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microcephalin, a gene regulating brain size, continues to evolve adaptively in humans.; citation_author=PD Evans; citation_author=SL Gilbert; citation_author=N Mekel-Bobrov; citation_author=EJ Vallender; citation_author=JR Anderson; citation_journal_title=Science; citation_volume=309; citation_number=21; citation_pages=1717-1720; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Interrogating multiple aspects of variation in a full resequencing data set to infer human population size changes.; citation_author=BF Voight; citation_author=AM Adams; citation_author=LA Frisse; citation_author=Y Qian; citation_author=RR Hudson; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=22; citation_pages=18508-18513; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comment on “Ongoing adaptive evolution of ASPM, a brain size determinant in Homo sapiens” and “Microcephalin, a gene regulating brain size, continues to evolve adaptively in humans”.; citation_author=M Currat; citation_author=L Excoffier; citation_author=W Maddison; citation_author=SP Otto; citation_author=N Ray; citation_journal_title=Science; citation_volume=313; citation_number=23; citation_pages=172; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Reconstructing human origins in the genomic era.; citation_author=D Garrigan; citation_author=MF Hammer; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=24; citation_pages=669-680; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Recent common ancestry of human Y chromosomes: evidence from DNA sequence data.; citation_author=R Thomson; citation_author=JK Pritchard; citation_author=P Shen; citation_author=PJ Oefner; citation_author=MW Feldman; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=97; citation_number=25; citation_pages=7360-7365; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=mtDNA variation among Greenland Eskimos: the edge of the Beringian expansion.; citation_author=J Saillard; citation_author=P Forster; citation_author=N Lynnerup; citation_author=HJ Bandelt; citation_author=S Norby; citation_journal_title=Am J Hum Genet; citation_volume=67; citation_number=26; citation_pages=718-726; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Origin and evolution of Native American mtDNA variation: a reappraisal.; citation_author=P Forster; citation_author=R Harding; citation_author=A Torroni; citation_author=HJ Bandelt; citation_journal_title=Am J Hum Genet; citation_volume=59; citation_number=27; citation_pages=935-945; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Pleistocene Homo sapiens from Middle Awash, Ethiopia.; citation_author=TD White; citation_author=B Asfaw; citation_author=D DeGusta; citation_author=H Gilbert; citation_author=GD Richards; citation_journal_title=Nature; citation_volume=423; citation_number=28; citation_pages=742-747; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Why did modern human populations disperse from Africa ca. 60,000 years ago? A new model.; citation_author=P Mellars; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=29; citation_pages=9381-9386; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ice Ages and the mitochondrial DNA chronology of human dispersals: a review.; citation_author=P Forster; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=359; citation_number=30; citation_pages=255-264; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic structure of human populations.; citation_author=NA Rosenberg; citation_author=JK Pritchard; citation_author=JL Weber; citation_author=HM Cann; citation_author=KK Kidd; citation_journal_title=Science; citation_volume=298; citation_number=31; citation_pages=2381-2385; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Alu repeats and human genomic diversity.; citation_author=MA Batzer; citation_author=PL Deininger; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=32; citation_pages=370-379; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The evolution of recombination: removing the limits to natural selection.; citation_author=SP Otto; citation_author=NH Barton; citation_journal_title=Genetics; citation_volume=147; citation_number=33; citation_pages=879-906; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A pacific culture among wild baboons: its emergence and transmission.; citation_author=RM Sapolsky; citation_author=LJ Share; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=34; citation_pages=E106; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Incorporating psychosocial health into biocultural models: preliminary findings from Turkana women of Kenya.; citation_author=IL Pike; citation_author=SR Williams; citation_journal_title=Am J Hum Biol; citation_volume=18; citation_number=35; citation_pages=729-740; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The influence of social hierarchy on primate health.; citation_author=RM Sapolsky; citation_journal_title=Science; citation_volume=308; citation_number=36; citation_pages=648-652; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Paleopathology at the Origins of Agriculture: Editor's Summation.; citation_author=MN Cohen; citation_author=GJ Armelagos; citation_number=37; citation_pages=585-601; citation_date=1984; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=Assortative mating in schizophrenia: results from the Copenhagen High-Risk Study.; citation_author=J Parnas; citation_journal_title=Psychiatry; citation_volume=51; citation_number=38; citation_pages=58-64; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Natural selection and population history in the human angiotensinogen gene (AGT): 736 complete AGT sequences in chromosomes from around the world.; citation_author=T Nakajima; citation_author=S Wooding; citation_author=T Sakagami; citation_author=M Emi; citation_author=K Tokunaga; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=39; citation_pages=898-916; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CYP3A variation and the evolution of salt-sensitivity variants.; citation_author=EE Thompson; citation_author=H Kuttab-Boulos; citation_author=D Witonsky; citation_author=L Yang; citation_author=BA Roe; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=40; citation_pages=1059-1069; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Population genetics of CAPN10 and GPR35: implications for the evolution of type-II diabetes variants.; citation_author=MJ Vander; citation_author=LM Frisse; citation_author=SM Fullerton; citation_author=Y Qian; citation_author=L Del Bosque-Plata; citation_journal_title=Am J Hum Genet; citation_volume=76; citation_number=41; citation_pages=548-560; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The common PPARgamma Pro12Ala polymorphism is associated with decreased risk of type-II diabetes.; citation_author=D Altshuler; citation_author=JN Hirschhorn; citation_author=M Klannemark; citation_author=CM Lindgren; citation_author=MC Vohl; citation_journal_title=Nat Genet; citation_volume=26; citation_number=42; citation_pages=76-80; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A Pro12Ala substitution in PPARgamma2 associated with decreased receptor activity, lower body mass index and improved insulin sensitivity.; citation_author=SS Deeb; citation_author=L Fajas; citation_author=M Nemoto; citation_author=J Pihlajamaki; citation_author=L Mykkanen; citation_journal_title=Nat Genet; citation_volume=20; citation_number=43; citation_pages=284-287; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=An evolutionary framework for common diseases: the ancestral-susceptibility model.; citation_author=A Di Rienzo; citation_author=RR Hudson; citation_journal_title=Trends Genet; citation_volume=21; citation_number=44; citation_pages=596-601; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Blood pressure and dietary salt in human populations.; citation_author=L Gleibermann; citation_journal_title=Ecol Food Nutr; citation_volume=2; citation_number=45; citation_pages=143-156; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=Diabetes mellitus: a “thrifty” genotype rendered detrimental by “progress”?; citation_author=JV Neel; citation_journal_title=Am J Hum Genet; citation_volume=14; citation_number=46; citation_pages=353-362; citation_date=1962; " />
      <meta name="citation_reference" content="citation_number=47; citation_date=1994; citation_publisher=American Psychiatric Press; " />
      <meta name="citation_reference" content="citation_author=K Leonhard; citation_number=48; citation_date=1999; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Association and linkage analyses of RGS4 polymorphisms in schizophrenia.; citation_author=KV Chowdari; citation_author=K Mirnics; citation_author=P Semwal; citation_author=J Wood; citation_author=E Lawrence; citation_journal_title=Hum Mol Genet; citation_volume=11; citation_number=49; citation_pages=1373-1380; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Diagnostic interview for genetic studies. Rationale, unique features, and training. NIMH Genetics Initiative.; citation_author=JI Nurnberger; citation_author=MC Blehar; citation_author=CA Kaufmann; citation_author=C York-Cooler; citation_author=SG Simpson; citation_journal_title=Arch Gen Psychiatry; citation_volume=51; citation_number=50; citation_pages=849-859; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Bioinformatics Methods and Protocols: Methods in Molecular Biology.; citation_author=S Rozen; citation_author=H Skaletsky; citation_number=51; citation_pages=365-386; citation_date=2000; citation_publisher=Humana Press; " />
      <meta name="citation_reference" content="citation_title=PolyPhred: automating the detection and genotyping of single nucleotide substitutions using fluorescence-based resequencing.; citation_author=DA Nickerson; citation_author=VO Tobe; citation_author=SL Taylor; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=52; citation_pages=2745-2751; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A new statistical method for haplotype reconstruction from population data.; citation_author=M Stephens; citation_author=NJ Smith; citation_author=P Donnelly; citation_journal_title=Am J Hum Genet; citation_volume=68; citation_number=53; citation_pages=978-989; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Multiple sequence alignment with the Clustal series of programs.; citation_author=R Chenna; citation_author=H Sugawara; citation_author=T Koike; citation_author=R Lopez; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=54; citation_pages=3497-3500; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=VISTA: visualizing global DNA sequence alignments of arbitrary length.; citation_author=C Mayor; citation_author=M Brudno; citation_author=JR Schwartz; citation_author=A Poliakov; citation_author=EM Rubin; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=55; citation_pages=1046-1047; citation_date=2000; " />
      <meta name="citation_reference" content="citation_author=AFA Smit; citation_author=R Hubley; citation_author=P Green; citation_journal_title=RepeatMasker Open - 3.0.; citation_number=56; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=J Felsenstein; citation_journal_title=PHYLIP Phylogeny Inference Package, version 3.6. Department of Genome Sciences, University of Washington, Seattle.; citation_number=57; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=TREEVIEW: An application to display phylogenetic trees on personal computers.; citation_author=R Page; citation_journal_title=Computer Applications in the Biosciences; citation_volume=12; citation_number=58; citation_pages=357-358; citation_date=1996; " />
      <meta name="citation_reference" content="citation_author=M Kimura; citation_number=59; citation_date=1983; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood estimation of the number of nucleotide substitutions from restriction sites data.; citation_author=M Nei; citation_author=F Tajima; citation_journal_title=Genetics; citation_volume=105; citation_number=60; citation_pages=207-217; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Mathematical model for studying genetic variation in terms of restriction endonucleases.; citation_author=M Nei; citation_author=WH Li; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=76; citation_number=61; citation_pages=5269-5273; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=On the number of segregating sites in genetical models without recombination.; citation_author=GA Watterson; citation_journal_title=Theor Popul Biol; citation_volume=7; citation_number=62; citation_pages=256-276; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Hitchhiking under positive Darwinian selection.; citation_author=JC Fay; citation_author=CI Wu; citation_journal_title=Genetics; citation_volume=155; citation_number=63; citation_pages=1405-1413; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.; citation_author=F Tajima; citation_journal_title=Genetics; citation_volume=123; citation_number=64; citation_pages=585-595; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Statistical Tests of Neutrality of Mutations.; citation_author=YX Fu; citation_author=WH Li; citation_journal_title=Genetics; citation_volume=133; citation_number=65; citation_pages=693-709; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Generating samples under a Wright-Fisher neutral model of genetic variation.; citation_author=RR Hudson; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=66; citation_pages=337-338; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=DnaSP, DNA polymorphism analyses by the coalescent and other methods.; citation_author=J Rozas; citation_author=JC Sanchez-DelBarrio; citation_author=X Messeguer; citation_author=R Rozas; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=67; citation_pages=2496-2497; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Median-joining networks for inferring intraspecific phylogenies.; citation_author=HJ Bandelt; citation_author=P Forster; citation_author=A Rohl; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=68; citation_pages=37-48; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mutation of the GABAA receptor M1 transmembrane proline increases GABA affinity and reduces barbiturate enhancement.; citation_author=LJ Greenfield; citation_author=SH Zaman; citation_author=ML Sutherland; citation_author=SC Lummis; citation_author=MI Niemeyer; citation_journal_title=Neuropharmacology; citation_volume=42; citation_number=69; citation_pages=502-521; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Maintenance of recombinant type A γ-aminobutyric acid receptor function: role of protein tyrosine phosphorylation and calcineurin.; citation_author=RQ Huang; citation_author=GH Dillon; citation_journal_title=J Pharmacol Exp Ther; citation_volume=286; citation_number=70; citation_pages=243-255; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000462" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Positive Selection within the Schizophrenia-Associated GABAA Receptor β2 Gene"/>
    <meta name="twitter:description" content="The gamma-aminobutyric acid type-A (GABAA) receptor plays a major role in inhibitory neurotransmissions. Intronic SNPs and haplotypes in GABRB2, the gene for GABAA receptor &beta;2 subunit, are associated with schizophrenia and correlated with the expression of two alternatively spliced &beta;2 isoforms. In the present study, using chimpanzee as an ancestral reference, high frequencies were observed for the derived (D) alleles of the four SNPs rs6556547, rs187269, rs1816071 and rs1816072 in GABRB2, suggesting the occurrence of positive selection for these derived alleles. Coalescence-based simulation showed that the population frequency spectra and the frequencies of H56, the haplotype having all four D alleles, significantly deviated from neutral-evolution expectation in various demographic models. Haplotypes containing the derived allele of rs1816072 displayed significantly less diversity compared to haplotypes containing its ancestral allele, further supporting positive selection. The variations in DD-genotype frequencies in five human populations provided a snapshot of the evolutionary history, which suggested that the positive selections of the D alleles are recent and likely ongoing. The divergence between the DD-genotype profiles of schizophrenic and control samples pointed to the schizophrenia-relevance of positive selections, with the schizophrenic samples showing weakened selections compared to the controls. These DD-genotypes were previously found to increase the expression of &beta;2, especially its long isoform. Electrophysiological analysis showed that this long &beta;2 isoform favored by the positive selections is more sensitive than the short isoform to the inhibition of GABAA receptor function by energy depletion. These findings represent the first demonstration of positive selection in a schizophrenia-associated gene."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000462.g006"/>

  <meta property="og:title" content="Positive Selection within the Schizophrenia-Associated GABAA Receptor β2 Gene" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000462" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1857'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1459'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3613&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000462">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000462" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000462&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Positive Selection within the Schizophrenia-Associated GABA<sub>A</sub> Receptor β<sub>2</sub> Gene
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wing-Sze Lo
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Wing-Sze Lo, Zhiwen Xu, Zhiliang Yu, Frank W. Pun</p>

              
              

                <p>Affiliation:
                  Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhiwen Xu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Wing-Sze Lo, Zhiwen Xu, Zhiliang Yu, Frank W. Pun</p>

              
              

                <p>Affiliation:
                  Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhiliang Yu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Wing-Sze Lo, Zhiwen Xu, Zhiliang Yu, Frank W. Pun</p>

              
              

                <p>Affiliation:
                  Graduate program of Atmospheric, Marine, and Coastal Environment, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frank W. Pun
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Wing-Sze Lo, Zhiwen Xu, Zhiliang Yu, Frank W. Pun</p>

              
              

                <p>Affiliation:
                  Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Siu-Kin Ng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Graduate Program of Bioengineering, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jianhuan Chen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ka-Lok Tong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Cunyou Zhao, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Graduate Program of Bioengineering, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiaojing Xu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Graduate Program of Bioengineering, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shui-Ying Tsang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mutsuo Harano, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neuropsychiatry, Kurume University School of Medicine, Fukuka, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gerald Stöber, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Psychiatry and Psychotherapy, University of Würzburg, Würzburg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vishwajit L. Nimgaonkar, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departments of Psychiatry and Human Genetics, University of Pittsburgh School of Medicine, and Graduate School of Public Health, Pittsburgh, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hong Xue
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:hxue@ust.hk">hxue@ust.hk</a></p>

                <p>Affiliation:
                  Department of Biochemistry, Applied Genomics Laboratory and HKH Bioinformatics Center, Hong Kong University of Science and Technology, Clear Water Bay, Hong Kong, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 23, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000462</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000462" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000462" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000462" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000462" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000462-g001" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000462-g002" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000462-g003" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000462-g004" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000462-g005" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000462-g006" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000462">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000462" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The gamma-aminobutyric acid type-A (GABA<sub>A</sub>) receptor plays a major role in inhibitory neurotransmissions. Intronic SNPs and haplotypes in <em>GABRB2</em>, the gene for GABA<sub>A</sub> receptor β<sub>2</sub> subunit, are associated with schizophrenia and correlated with the expression of two alternatively spliced β<sub>2</sub> isoforms. In the present study, using chimpanzee as an ancestral reference, high frequencies were observed for the derived (D) alleles of the four SNPs rs6556547, rs187269, rs1816071 and rs1816072 in <em>GABRB2</em>, suggesting the occurrence of positive selection for these derived alleles. Coalescence-based simulation showed that the population frequency spectra and the frequencies of H56, the haplotype having all four D alleles, significantly deviated from neutral-evolution expectation in various demographic models. Haplotypes containing the derived allele of rs1816072 displayed significantly less diversity compared to haplotypes containing its ancestral allele, further supporting positive selection. The variations in DD-genotype frequencies in five human populations provided a snapshot of the evolutionary history, which suggested that the positive selections of the D alleles are recent and likely ongoing. The divergence between the DD-genotype profiles of schizophrenic and control samples pointed to the schizophrenia-relevance of positive selections, with the schizophrenic samples showing weakened selections compared to the controls. These DD-genotypes were previously found to increase the expression of β<sub>2</sub>, especially its long isoform. Electrophysiological analysis showed that this long β<sub>2</sub> isoform favored by the positive selections is more sensitive than the short isoform to the inhibition of GABA<sub>A</sub> receptor function by energy depletion. These findings represent the first demonstration of positive selection in a schizophrenia-associated gene.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Lo W-S, Xu Z, Yu Z, Pun FW, Ng S-K, et al.  (2007) Positive Selection within the Schizophrenia-Associated GABA<sub>A</sub> Receptor β<sub>2</sub> Gene. PLoS ONE 2(5):
          e462.
            doi:10.1371/journal.pone.0000462</p><p><strong>Academic Editor: </strong>Emmanouil Dermitzakis, The Wellcome Trust Sanger Institute, United Kingdom</p><p><strong>Received:</strong> January 15, 2007; <strong>Accepted:</strong> April 24, 2007; <strong>Published:</strong> May 23, 2007</p><p><strong>Copyright:</strong> © 2007 Lo et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>We thank the Research Grants Council and the Innovation and Technology Fund of Hong Kong for financial support.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Schizophrenia is one of the most debilitating mental disorders, afflicting peoples of all countries and cultures with about 1% lifetime risk <a href="#pone.0000462-Gottesman1">[1]</a>. Schizophrenics suffer perturbations in thought processes that manifest as hallucinations, delusions, disordered thinking, unusual speech or behavior, and social withdrawal. Schizophrenia tends to run in families with an estimated inheritability of 60–85% <a href="#pone.0000462-Austin1">[2]</a>. A feature of the disease is a prominent impairment in cognitive functions, especially language-related functions that are unique to mankind. Understanding the etiology of schizophrenia therefore not only paves the way to effective therapeutic treatment of the disease, but may also provide important insight into the nature of human cognition.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Over the past several years, advances in genomics have made possible an initial delineation of the genetic mechanisms of schizophrenia. DNA sequence polymorphisms in a number of genes are found to be associated with the disease <a href="#pone.0000462-Harrison1">[3]</a>. One of the strongest associations is that discovered by our laboratory in the single nucleotide polymorphisms (SNPs) and haplotypes in introns 8 and 9 of <em>GABRB2</em> <a href="#pone.0000462-Lo1">[4]</a>. <em>GABRB2</em> codes for the β<sub>2</sub> subunit of GABA<sub>A</sub> receptor, and is part of a cluster of genes on chromosome 5q34 for the GABA<sub>A</sub> receptor, the major inhibitory neurotransmitter-gated channel receptor family in the central nervous system (CNS) <a href="#pone.0000462-Korpi1">[5]</a>. Our initial findings from <em>Han</em> Chinese have since been validated by additional samples from the Chinese <a href="#pone.0000462-Liu1">[6]</a>, Portuguese <a href="#pone.0000462-Petryshen1">[7]</a>, German <a href="#pone.0000462-Petryshen1">[7]</a>–<a href="#pone.0000462-Lo2">[9]</a> and Japanese <a href="#pone.0000462-Lo2">[9]</a> populations, although conflicting results from Japanese <a href="#pone.0000462-Ikeda1">[10]</a> and German <a href="#pone.0000462-Jamra1">[11]</a> populations have also been reported.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>As in the case of most other genes underlying complex disorders, the schizophrenia-associated DNA sequence polymorphisms in <em>GABRB2</em> are located in the non-coding regions, and therefore not immediately evident in their functional implications. However, recent studies in our laboratory demonstrated that the schizophrenia-associated DNA sequence polymorphisms in <em>GABRB2</em> are correlated with under-expression in schizophrenic brains of two alternatively spliced forms of mRNA coding for two previously reported isoforms of β<sub>2</sub> subunit that differ in length by 38 amino acid residues including a potential Ser/Thr phosphorylation site <a href="#pone.0000462-Zhao1">[12]</a>. The longer isoform was also found to be more prone than the shorter isoform toward current run-down induced by repetitive activation. This finding of a genotype-dependent electrophysiological alteration, establishing an explicit relationship between genotype and potential neuronal behavior, represents the first instance of such relationships identified among the schizophrenia-associated genes.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>A human-specific Alu insertion</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Based on available genomic sequences of humans and a number of primates, an evolutionary analysis was conducted in the present study comparing human and non-human primate sequences in the schizophrenia-associated <em>GABRB2</em> region in order to detect any sequence signatures of human-lineage specific changes that could be indicative of natural selection and therefore functional significance. The non-human primate sequences homologous to the 3,551-bp genomic region of human <em>GABRB2</em> were either experimentally obtained or retrieved from the database (see <a href="#s4">Methods</a>) (<a href="#pone-0000462-g001">Figure 1A</a>). The comparison revealed a human specific 314-bp insertion within this otherwise highly conserved non-coding region of <em>GABRB2</em> (<a href="#pone.0000462.s008">Figure S1</a>).</p>
<div class="figure" id="pone-0000462-g001"><div class="img"><a name="pone-0000462-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>SNP positions and allele frequencies.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p><strong>(A)</strong> Positions of all twenty-nine SNPs in a 3551-bp region of schizophrenia-associated <em>GABRB2</em> from base 160,689,203 to base 160,692,753 of chromosome 5 contig NT_023133.12 are shown to scale. Solid horizontal lines represent introns flanking Exon 9 (green box) and the human specific Alu insert Yi6AH151 (yellow box), which spans bases 160,691,468 and 160,691,782. The ID numbers of SNPs with prefixes “rs” or “ss” are labeled according to the Single Nucleotide Polymorphism database (dbSNP; <a href="http://www.ncbi.nih.gov/SNP">www.ncbi.nih.gov/SNP</a>). The two alleles for each SNP are shown in parentheses following the SNP ID number in the format of (ancestral base)/(derived base). <strong>(B)</strong> Population allele frequencies of each SNP are represented by pie-charts for the AF, GE, US and JP populations, where the cyanine wedge indicates the frequency of the ancestral allele (N), and the red wedge the frequency of the derived allele (D). The four SNPs with high derived allele frequencies (high-D SNPs) are indicated by arrows and red ID Number.</p>
<span>doi:10.1371/journal.pone.0000462.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>By sequence homology this human-specific insert belongs to the Yi6 sub-group of Alu (<a href="#pone.0000462.s009">Figure S2</a>), a primate-specific family of short interspersed mobile elements with over one million members in the human genome <a href="#pone.0000462-Deininger1">[13]</a>. This insert, which is not identical to any known Alu sequence, is named Yi6AH151 in accordance to Batzer's Alu nomenclature, thereby adding to the 150 known Alu Yi6 elements <a href="#pone.0000462-Salem1">[14]</a>. It is absent from the non-human primates, but present in all the human subjects analyzed. Therefore it likely represents an insertion into the human lineage prior to human population subdivisions, rather than a deletion from the chimpanzee lineage. That the two SNPs rs34820615 and rs10060148 in Yi6AH151 were found to be associated with schizophrenia <a href="#pone.0000462-Lo2">[9]</a> suggests the involvement of this Alu in schizophrenia etiology. Possible Alu involvements also have been reported for other diseases <a href="#pone.0000462-Deininger2">[15]</a>.</p>


<h4>High sequence diversity in a conserved non-coding region</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Re-sequencing was performed on the 3,551-bp genomic segment in the neighborhood of Yi6AH151 for a total of 633 unrelated non-schizophrenic individuals of the four ethnically diverse human populations African (AF), German Caucasian (GE), USA Caucasian (US), and Japanese (JP). A total of 29 SNPs were identified, 15 in the 314-bp Alu Yi6AH151 and 14 on its flanking sequences. Using the chimpanzee and the Yi6 subfamily consensus sequences as references, the two alleles of each of the 29 SNPs could be determined as being the ancestral (N) or derived (D) allele (<a href="#pone-0000462-g001">Figure 1A</a> and <a href="#pone.0000462.s001">Table S1</a>). Based on the N or D status defined, four of the twenty-nine SNPs were found to be exceptionally high in D-allele frequency (&gt;50%), and designated as high-D SNPs (<a href="#pone-0000462-g001">Figure 1B</a> and <a href="#pone.0000462.s001">Table S1</a>). Their high D-allele frequencies signaled the occurrence of positive selection. The four high-D SNPs are in descending order of D-allele frequencies rs6556547, rs187269, rs1816071 and rs1816072. Since rs1816072 exhibits the lowest D-allele frequency among them, it could represent the youngest of the four.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The evolutionary divergences between species, and the diversities within the different human populations were scrutinized for any departure from neutrality. The 0.82% SNP density in the region was found to far exceed the genome-wide and locus-specific densities for the human genome (0.11–0.08%) <a href="#pone.0000462-Venter1">[16]</a>. Since this region falls within one of the recombination hotspots identified in the HapMap project (<a href="#pone.0000462.s010">Figure S3</a>), the high SNP density observed might have originated from high recombination activities.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>In contrast, the 0.74% sequence mismatch between human and chimpanzee in the region is lower than the human genome average of 1.23% nucleotide divergence <a href="#pone.0000462-The1">[17]</a>. This cross-species conservation was suggestive of purifying selection, which is not unexpected for an exon-proximal genic region. This combination of low human-chimpanzee divergence and high intra-human diversity departs from the expectations of neutral evolution (<em>P</em> = 0.0013 in the Hudson, Kreitman and Aguadé's test <a href="#pone.0000462-Hudson1">[18]</a>), and points to the occurrence of human-specific events in this otherwise evolutionarily conserved non-coding region <a href="#pone.0000462-Bird1">[19]</a>, <a href="#pone.0000462-Drake1">[20]</a>, resulting in accelerated evolution within the human lineage.</p>


<h4>Ancestral and derived haplotype groups</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Frequency spectrum summary statistics were employed to test for any departure from neutral evolution in the region. Based on either the datasets pooled from all four populations, or the datasets of individual populations, positive selection was indicated by some demographic models in the Fay and Wu's <em>H</em> test, which takes the ancestral status of polymorphism sites into consideration and is sensitive to an excess of derived alleles. However, neither Tajima's <em>D</em> nor Fu and Li's <em>D</em> and <em>F</em> tests detected significant positive selection in the region (<a href="#pone.0000462.s002">Table S2</a>).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Accordingly, further investigations of potential positive selection were conducted on separate haplotype groups in accordance to Evans <em>et al</em> <a href="#pone.0000462-Evans1">[21]</a>. Using the allelic state of rs1816072, the youngest of the four high-D SNPs, as defining criterion, all the haplotypes containing its N allele were assigned to haplotype group-N (HG-N; 42.9%), and all those containing its D allele were assigned to haplotype group-D (HG-D; 57.1%) (<a href="#pone-0000462-g002">Figure 2</a> and <a href="#pone.0000462.s003">Table S3</a>). These HG-N and HG-D haplotype groups were sufficiently similar in size to allow statistically acceptable comparisons. HG-N and HG-D were vastly dissimilar with respect to sequence diversity. The haplotype (<em>H</em><sub>d</sub>) and nucleotide (π) diversities of HG-D were much lower than those of HG-N (<a href="#pone.0000462.s002">Table S2</a>). HG-N displayed a highly diversified membership of fifty different haplotypes (H1 to H50), all of relatively low frequencies (<a href="#pone-0000462-g002">Figure 2</a> and <a href="#pone.0000462.s003">Table S3</a>). H6, the most abundant member of this group, amounted to a frequency of only 8.5%. In contrast, HG-D contained only twenty-five member haplotypes (H51–H75). H56, its highest-frequency member comprising the D alleles of the four high-D SNPs together with the ancestral alleles of the other 25 SNP sites (<a href="#pone.0000462.s003">Table S3</a>), was present in 51.7% of total N+D copies, or 90.5% of the D copies. This extraordinary predominance by a single haplotype confirmed to the occurrence of exceptionally strong positive selection for H56 and its four constituent D alleles. The differences between HG-N and HG-D in the Fay and Wu's <em>H</em> value populations were non-random for all three non-AF populations (<a href="#pone.0000462.s002">Table S2</a>: GE and JP <em>P</em>&lt;0.00001; US <em>P</em> = 0.07). The mutational spectra in the genealogies of these two classes of haplotypes are therefore sharply different (<a href="#pone-0000462-g003">Figure 3</a> and <a href="#pone.0000462.s011">Figure S4</a>).</p>
<div class="figure" id="pone-0000462-g002"><div class="img"><a name="pone-0000462-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Haplotype frequency distribution in the four populations (refer to <a href="#pone-0000462-g001">Figure 1</a> legend for abbreviations).</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Each haplotype is assigned into either the ancestral (HG-N; cyanine) or derived (HG-D; red) group according to its allelic status at rs1816072, likely the youngest high-D SNP. Details of the compositions and frequencies of different haplotypes are shown in <a href="#pone.0000462.s003">Table S3</a>.</p>
<span>doi:10.1371/journal.pone.0000462.g002</span></div><div class="figure" id="pone-0000462-g003"><div class="img"><a name="pone-0000462-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Fay and Wu's <em>H</em> values for both HG-N and HG-D haplotypes are plotted for AF, GE, US and JP in thermal scale.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Similar plots for Tajima's D and Fu and Li's D are presented in <a href="#pone.0000462.s011">Figure S4</a>.</p>
<span>doi:10.1371/journal.pone.0000462.g003</span></div>

<h4>Demographic evidence against neutral evolution</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To distinguish further whether the very high H56 frequency is the outcome of positive selection or demographic effects, coalescence-based simulations were performed. The null hypothesis that a haplotype carrying four or more D alleles could achieve a frequency equal to or higher than the observed H56 frequency in each of the AF, GE, US and JP populations under neutral evolution was tested (<a href="#pone.0000462.s004">Table S4</a>). For this purpose, thirty-three different demographic models embodying population growth <a href="#pone.0000462-Evans1">[21]</a>, population bottlenecks <a href="#pone.0000462-Evans1">[21]</a>, <a href="#pone.0000462-Voight1">[22]</a>, population substructure <a href="#pone.0000462-Evans1">[21]</a>, population growth plus substructure <a href="#pone.0000462-Currat1">[23]</a>, and structured ancestral population with subsequent bottlenecks and expansion <a href="#pone.0000462-Garrigan1">[24]</a>, were employed. For the AF samples, this null hypothesis was rejected with the Garrigan and Hammers' low-migration model (<em>P</em> = 0.00380–0.00420) and Voight's models with most severe bottleneck (<em>P</em> = 0.00118–0.00052). As well, it was marginally rejected with the ancient population expansion model (<em>P</em> = 0.06153), the extended growth core model with mild bottleneck (<em>P</em> = 0.06780) and Voight's bottleneck with modest bottleneck (<em>P</em> = 0.06143), although it could not be rejected with the other models. For all three non-AF populations the null hypothesis was rejected for all of the demographic models analyzed except for JP in the case of Model 10 incorporating the most severe bottleneck. These results provided compelling evidence for positive selection acting on H56.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>While any positive selection acting on the region surrounding Yi6AH151 would preclude a reliable dating of the SNPs in the region, the fact that the four high-D SNPs achieved high-D status in all four of the human populations examined (<a href="#pone-0000462-g001">Figure 1B</a>) suggests that these high-D SNPs appeared before the divergence of the human populations. Coalescence-based dating was carried out for the four high-D SNPs based on two different mutation rates (see <a href="#s4">Methods</a>). The mutation rate estimated from human-chimpanzee divergence based on a constant molecular clock <a href="#pone.0000462-Thomson1">[25]</a> yielded age estimates for these SNPs (<a href="#pone.0000462.s006">Table S6A</a>) far older than the generally accepted time of human speciation, further testifying to the process of positive selection acting on the region. On the other hand, an accelerated mutation rate equal to that of fast-evolving human mitochondrial DNA <a href="#pone.0000462-Saillard1">[26]</a> yielded age estimates of 139.91±10.09 Kya for rs6556547, 113.31±18.35 Kya for rs187269, 92.82±19.06 Kya for rs1816071, and 69.44±20.14 Kya for rs1816072 in coalescence-based calculations with the Rho index <a href="#pone.0000462-Forster1">[27]</a> (<a href="#pone.0000462.s006">Table S6</a>; <a href="#pone-0000462-g004">Figure 4</a>). These age estimates, within the time frame between the origin of the anatomically modern human around 150 Kya <a href="#pone.0000462-White1">[28]</a> and migrations out of Africa around 60 Kya <a href="#pone.0000462-Mellars1">[29]</a>, gave evidence to a fast rate of evolutionary change, driven by the positive selection, comparable to that of mitochondrial DNA.</p>
<div class="figure" id="pone-0000462-g004"><div class="img"><a name="pone-0000462-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Proposed molecular evolution events in relation to time frame of modern human origins and dispersal from Africa.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The age estimates of the four high-D SNPs are shown on the left of this figure. Relevant human evolution milestones, including divergence from chimpanzee and modern human evolution events delineated in Mellars' model <a href="#pone.0000462-Mellars1">[29]</a>, are represented on the right. Yi6AH151 is represented as Alu.</p>
<span>doi:10.1371/journal.pone.0000462.g004</span></div>

<h4>Population variations in allelic and haplotype frequencies</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Pronounced population variations were observed with respect to the frequencies of alleles (<a href="#pone-0000462-g001">Figure 1</a> and <a href="#pone.0000462.s001">Table S1</a>) and haplotypes (<a href="#pone-0000462-g002">Figure 2</a> and <a href="#pone.0000462.s003">Table S3</a>) among the Yi6AH151 SNPs. The D-allele frequency of the youngest high-D SNP rs1816072 was considerably lower in AF (31.2%) than in the three non-AF populations (57.1–63.9%). This was even more the case with H56, which displayed a frequency of 18.1% for AF compared to 53.4–63.6% for the three non-AF populations. These findings suggest that the accumulation of the D-alleles and HG-D began not long prior to the migrations of the non-AF populations from Africa approximately 60 Kya <a href="#pone.0000462-Forster2">[30]</a>, and continued after the separation of the different populations.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The evolutionary history of the region surrounding Yi6AH151 could be reconstructed from the male genotype frequencies from the AF and the three non-AF populations together with the Chinese cohort (CH) employed in the initial report on <em>GABRB2</em>-schizophrenia association <a href="#pone.0000462-Lo1">[4]</a> (<a href="#pone.0000462.s005">Table S5</a>). In <a href="#pone-0000462-g005">Figure 5</a>, the homozygous D-allele genotype (DD) frequencies of from left to right the oldest rs6556547, the second oldest rs187269, the third oldest rs1816071 and the youngest rs1816072 in the male control samples from these five populations are plotted in the order of AF-GE-US-JP-CH in accordance with the order of ascending DD frequencies for the three younger SNPs. This ascending order coincided largely with the order of genetic distances of these non-AF populations from AF, which placed the East Asia-Africa distances much greater than the Europe-Africa distances <a href="#pone.0000462-Rosenberg1">[31]</a>. The similarity of the monotonic ascending DD-frequency gradients for the three younger SNPs is consistent with the identified positive selection being a recent and possibly ongoing process. Comparable DD frequency gradients were not demonstrated for the female samples which were more limited in sample size. The rs6556547 SNP did not exhibit a monotonic ascending DD-frequency gradient, owing at least in part to the high DD frequency of 66.7% (<a href="#pone.0000462.s005">Table S5</a>) even in the AF population, which is keeping with the D-allele of this SNP being the oldest among the four high-D SNPs.</p>
<div class="figure" id="pone-0000462-g005"><div class="img"><a name="pone-0000462-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Homozygous D-allele genotype (DD) frequencies for the four high-D SNPs in controls (top) and schizophrenics (bottom) (<a href="#pone.0000462.s005">Table S5</a>).</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>These DD frequencies are plotted for AF, GE, US, JP and <em>Han</em> Chinese from Shanghai (CH) with the four high-D SNPs arranged from left to right in a descending order of their estimated ages. To avoid the potential influence from any gender effect, only male samples were employed in this controls-schizophrenics comparison. The absence of schizophrenic samples of AF origin is indicated by “#”.</p>
<span>doi:10.1371/journal.pone.0000462.g005</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>In contrast to the controls, the 436 male schizophrenic samples did not conform to a monotonic ascending DD-genotype frequency gradient for any of the four high-D SNPs (<a href="#pone-0000462-g005">Figure 5</a> and <a href="#pone.0000462.s005">Table S5</a>). In fact, a modest monotonic descending gradient was displayed by rs1816072, and a flat-top descending gradient by rs6556547. The divergent behavior of the control and schizophrenic DD-frequencies becomes particularly pronounced in the schizophrenia/control plot, which yielded a monotonic descending gradient for all four high-D SNPs. It indicates a relationship between DD-frequencies and schizophrenia that diverges from that between DD-frequencies and the controls, in accord with the association of the SNPs in this genomic region with schizophrenia observed at the genotype level or haplotype level or both <a href="#pone.0000462-Lo1">[4]</a>, <a href="#pone.0000462-Lo2">[9]</a>.</p>


<h4>ATP-dependences of receptors containing the two β<sub>2</sub>-isoforms</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Functionally, it has been demonstrated in a separate Caucasian cohort that the presence of the derived alleles of the three younger high-D SNPs rs187269, rs1816071 and rs1816072 in either homozygote or heterozygote forms significantly increased both the total β<sub>2</sub> mRNA expression of the combined alternatively-spliced long and short β<sub>2</sub> isoforms, and the long/short β<sub>2</sub> isoform ratio, relative to the homozygote forms of the ancestral alleles (the fourth high-D SNP rs6556547 was not included in the analysis) <a href="#pone.0000462-Zhao1">[12]</a>. An increase in total β<sub>2</sub> subunit expression would enhance GABA<sub>A</sub> receptor functions and therefore the level of inhibitory transmissions in the CNS, the majority of which are mediated by β<sub>2</sub>-containing GABA<sub>A</sub> receptors. On the other hand, an increase in the long/short β<sub>2</sub> isoform ratio would bring about a greater run-down of GABA<sub>A</sub>-mediated electrophysiological current upon repetitive GABA stimulation in the absence of externally added ATP <a href="#pone.0000462-Zhao1">[12]</a>.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>The pivotal role of energy status in the current run-down induced by repetitive GABA stimulation was confirmed in the present study by the results in <a href="#pone-0000462-g006">Figure 6</a>, where raising the intracellularly infused ATP from 1 mM to 4 mM was found to reduce the amplitude of the run-down. Since this reduction was much greater with the long β<sub>2</sub> isoform (β<sub>2L</sub>)-containing receptors compared to the short β<sub>2</sub> isoform (β<sub>2S</sub>)-containing receptors, the difference in current run-down between the two kinds of receptors became narrowed at 2 mM compared to 1 mM ATP, and practically eliminated at 4 mM ATP. Thus the long/short β<sub>2</sub> isoform ratio was confirmed as an important determinant in the energy regulation of GABA<sub>A</sub> receptor function. Such energy regulation of GABA<sub>A</sub> receptor function could contribute for example to the well-known heightening of alertness by hunger in contrast to the relative lethargy induced by satiety: hunger cuts down ATP supply, thereby inducing GABA-current rundown and decreasing inhibitory CNS neurotransmissions mediated by long β<sub>2</sub> isoform-containing GABA<sub>A</sub> receptors. It follows that enhancement of the long/short β<sub>2</sub> isoform ratio by the derived alleles of the high-D SNPs would reduce such inhibitory CNS neurotransmissions under normal conditions, but much less so under conditions of energy deprivation.</p>
<div class="figure" id="pone-0000462-g006"><div class="img"><a name="pone-0000462-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000462" data-uri="info:doi/10.1371/journal.pone.0000462.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000462.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000462.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000462.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000462.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Variations of GABA-potentiated current rundowns with infused ATP.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Currents were measured upon 30 successive additions of GABA at 1-minute intervals as described in <a href="#s4">Methods</a>. Electrophysiological currents, expressed as a percentage of the peak current (▵) measured upon initial response to stimulation by 100 µM (top row) or 300 µM (bottom row) GABA, were measured for α<sub>1</sub>β<sub>2L</sub>γ<sub>2L</sub> (▪) and α<sub>1</sub>β<sub>2S</sub>γ<sub>2L</sub> (○) receptors with intracellular infusion of 1 mM, 2 mM or 4 mM ATP. Each data point represents average±standard error recorded from 7–10 cells from at least six independent transfections. Responses that were significantly different between the α<sub>1</sub>β<sub>2L</sub>γ<sub>2L</sub> and α<sub>1</sub>β<sub>2S</sub>γ<sub>2L</sub> receptors based on the two-tailed Student's <em>t</em>-test are marked by asterisks (*, <em>P</em>&lt;0.05; **, <em>P</em>&lt;0.01).</p>
<span>doi:10.1371/journal.pone.0000462.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Within the schizophrenia-associated 3,551-bp region of human <em>GABRB2</em>, the human lineage displayed accelerated evolutionary adaptation. The high DD-frequencies of the oldest high-D SNP rs6556547 in the control samples, ranging between 70.5–87.1% for the five human populations (<a href="#pone.0000462.s001">Table S1</a> and Table 1 in <a href="#pone.0000462-Lo1">[4]</a>) signaled strong positive selection of the derived allele of this SNP over the ancestral allele found in the chimpanzee. For the three younger high-D SNPs rs187269, rs1816071 and rs1816072, the DD-frequencies were not as high, but the positive selections of derived over ancestral alleles were just as evident. These positive selections are striking in view of the low sequence divergence among primates in this region. They were shown to be statistically significant upon testing by stringent criteria (<a href="#pone.0000462.s004">Table S4</a>), and found confirmation in the lower diversity of the haplotypes containing the D-allele of rs1816072 (<a href="#pone-0000462-g002">Figure 2</a>, <a href="#pone-0000462-g003">Figure 3</a> and <a href="#pone.0000462.s002">Table S2</a>). They might be facilitated by the human-specific Alu Yi6AH151 insertion in the region; insertions of this type are known to increase homologous recombinations and mutations, on which selection could act <a href="#pone.0000462-Batzer1">[32]</a>, <a href="#pone.0000462-Otto1">[33]</a>. The dissimilar enrichments of the derived alleles displayed by the different populations suggest that the positive selections of these alleles, especially the younger ones, are recent and likely ongoing. Based on the age estimates of the four derived alleles obtained using the mitochondrial DNA mutation rate as a proximate, the time-span of these selection processes overlapped with the time frame of Mellars' proposed history of modern human evolution <a href="#pone.0000462-Mellars1">[29]</a>, with the emergence of the four high-D SNPs broadly coinciding with human population expansion in, and dispersal from, Africa (<a href="#pone-0000462-g004">Figure 4</a>).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Since the four high-D SNPs occur in the intronic regions of <em>GABRB2</em>, the impact of their allelic status may be expected to be regulatory in nature. In keeping with this expectation, their derived alleles were found to increase total β<sub>2</sub>-expression and to enhance inhibition of GABA<sub>A</sub> function upon ATP depletion [<a href="#pone.0000462-Zhao1">[12]</a>; <a href="#pone-0000462-g006">Figure 6</a>]. These two opposing effects of the derived alleles could be fundamental to the differences between populations. Because β<sub>2</sub>-containing GABA<sub>A</sub> receptors mediate the majority of CNS inhibitory neurotransmissions, the derived alleles would increase the overall activities of CNS inhibition by increasing total β<sub>2</sub>-expression. At the same time, however, they would also accentuate the inhibition of GABA<sub>A</sub> function under conditions of energy deprivation as occasioned by food shortages. Accordingly the strength of the positive derived-allele selections in any population would depend on the level of GABA<sub>A</sub>-mediated CNS inhibitions required by the population, as well as the historical pattern of famines and severe food shortages in the course of its evolution. Various environmental, social and other population-specific factors could affect the required level of GABA<sub>A</sub>-mediated CNS inhibitions. In baboons, negative modulation of GABA<sub>A</sub> receptor produced effects that varied with the social status of the subject <a href="#pone.0000462-Sapolsky1">[34]</a>. Among humans as well, evidence suggests that social organization could influence behavior pertinent to survival and evolution <a href="#pone.0000462-Pike1">[35]</a>, <a href="#pone.0000462-Sapolsky2">[36]</a>. The distance of the migrations out-of-Africa, the size of the populations participating in these migrations, and impact of agriculture <a href="#pone.0000462-Cohen1">[37]</a> are other determinants that could favor the evolution of dissimilar GABA<sub>A</sub> activity profiles in different populations. Consequently, the evolved frequencies of the derived alleles of the high-D SNPs can not be expected to be the same for different populations.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The schizophrenia-relevance of the positive selection of the derived genotypes is suggested by the drastic differences between schizophrenia and control in their DD-frequency profiles (<a href="#pone-0000462-g005">Figure 5</a>). The control samples showed an indefinite trend for the oldest high-D SNP, but an upward gradient for the three younger high-D SNPs, in the DD-frequency plots over the AF, GE, US, JP and CH populations, providing a useful evolutionary snapshot of the selection process. In contrast, an indefinite trend (rs187269 and rs1816071) or downward trend (rs6556547 and rs1816072) was observed over these five populations for the schizophrenic samples; the reasons for the two downward gradients yet remain to be understood, but it is not ruled out that assortative mating, which has been observed in schizophrenia <a href="#pone.0000462-Parnas1">[38]</a>, might be a contributing factor. The schizophrenia/control ratio in DD-frequency thus decreased monotonically for all four high-D SNPs over these five populations, clearly establishing that the positive selection manifested by the controls was substantially weakened among the schizophrenics. Although the ratio exceeded unity in the GE samples, was close to unity in the US samples, but less than unity in the JP and CH samples, the schizophrenic/control ratios for all four SNPs conformed to a similar descending gradient over the GE-US-JP-CH populations. This uniformity shown by the four high-D SNPs suggests that the nature of the difference between control and schizophrenic samples was basically comparable for the four individual high-D SNPs.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>An <em>inhibition demand</em> hypothesis may be proposed to address the biological basis of the overall positive selection of derived genotypes in <em>GABRB2</em>, and its potential significance to the etiology of schizophrenia and possibly also other psychiatric disorders involving GABA<sub>A</sub>. The proposal consists of: (a) a balance between CNS excitations and inhibitions is essential to mental health; (b) since there is only one inhibitory neurotransmitter, GABA, in contrast to several excitatory neurotransmitters in the brain, the GABAergic system is among the systems that are stringently subjected to evolutionary adjustment; (c) in general, human genetic evolution is slow relative to a rapidly changing environment that imposes increasing demand on CNS inhibitory functions; (d) the derived genotypes, which influence the regulation of the GABAergic system, tend to be reduced in schizophrenics; and (e) such and other alterations affecting the GABAergic system could contribute to schizophrenia etiology.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>While purifying selections are often encountered eliminating the rare, deleterious mutations responsible for mendelian diseases, positive selections of protective derived alleles are increasingly observed in common disorders. For example, there are indications of positive derived-allele selection in the <em>AGT</em> gene for angiotensinogen <a href="#pone.0000462-Nakajima1">[39]</a> and <em>CYP3A5</em> gene for cytochrome P450 <a href="#pone.0000462-Thompson1">[40]</a> protecting against hypertension, and similarly positive derived-allele selection in the <em>CAPN10</em> gene for calpain 10 <a href="#pone.0000462-Vander1">[41]</a> and <em>PPARgamma</em> gene <a href="#pone.0000462-Altshuler1">[42]</a>–<a href="#pone.0000462-DiRienzo1">[44]</a> protecting against type-II diabetes. These positive selections, consistent with the <em>sodium-retention</em> hypothesis for hypertension <a href="#pone.0000462-Gleibermann1">[45]</a> and <em>thrifty-gene</em> hypothesis for diabetes <a href="#pone.0000462-Neel1">[46]</a>, respectively, may be interpreted in terms of an ancestral-susceptibility model for common diseases <a href="#pone.0000462-DiRienzo1">[44]</a> which proposes that, on account of shifts of environment and lifestyle from ancient to modern times, originally harmless or even beneficial ancestral alleles could enhance disease susceptibility in the modern populations, such that protective derived alleles providing improved adaptation to the altered circumstances are positively selected. In this regard, it is noteworthy that because common diseases are usually multigenic in nature, the divergence between modern and ancestral circumstances that is beneficially addressed by the protective derived alleles of a disease-susceptibility gene could arise not only from shifts of environment and lifestyle. It could also arise within the body from alterations in the allelic status of other interacting, co-evolving genes, causing the ancestral alleles to interact suboptimally with other genes in the modern population. For common diseases not expressly related to diet or lifestyle, such internal alterations in the genomic context of a susceptibility gene could be as important as, or even more important than, external shifts of environment and lifestyle.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In conclusion, because of changes between ancient and modern populations with respect to the genomic context in which <em>GABRB2</em> is embedded, and with respect to environment and lifestyle, the ancestral alleles of the high-D SNPs rs6556547, rs187269, rs1816071 and rs1816072 in <em>GABRB2</em> have become functionally inadequate in the modern populations. This inadequacy has brought about recent and possibly ongoing positive selections for their cognate derived alleles. Because of the dissimilar evolutionary paths undergone by different human populations, each population has acquired through evolution its own optimized capacity of GABAergic receptors and response of these receptors to energy regulation, as determined by among other factors the derived alleles of the high-D SNPs. The occurrence of positive selection, observed for the first time for any schizophrenia susceptibility genes, in <em>GABRB2</em> points to a fundamental role of GABA<sub>A</sub> function in schizophrenia etiology, as well as the potential usefulness of searching for positive selection among various susceptibility genes for schizophrenia and other complex disorders.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Human subjects</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>A total of 1,179 individuals (Female 345; Male 834), including 743 unrelated non-schizophrenia subjects (Female 345; Male 398) and 436 schizophrenic male patients but not including relatives in trios, from four different ethnic populations, African (AF), German Caucasians (GE), Caucasians of USA (US) and Japanese (JP) were studied.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>For the JP and GE samples, details of sample source and diagnostic procedure were described in Lo <em>et al</em>. <a href="#pone.0000462-Lo2">[9]</a>. In brief, the JP samples consisted of 207 unrelated control subjects (Female 105; Male 102) and 210 unrelated schizophrenia male patients. The GE samples consisted of 190 unrelated control subjects (Female 76; Male 114) and 119 unrelated male schizophrenics of systematic subtype. All schizophrenia patients were in-patients and fulfilled the diagnostic criteria for schizophrenia according to the fourth edition of Diagnostic and Statistical Manual of Mental Disorders <a href="#pone.0000462-American1">[47]</a>. The GE patients belonged to the systematic schizophrenia subtype, the most severe form of schizophrenia, on the basis of Leonhard's classification of endogenous psychoses <a href="#pone.0000462-Leonhard1">[48]</a>. Written informed consent was obtained from each subject. Approval for the study was obtained from the ethnical committees of Kurume University for the JP samples and of University of Würzburg for the GE samples.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>A total of 370 US samples consisted of 263 unrelated control subjects (Female 127; Male 136) and 107 schizophrenia male patients. Recruitment of the samples was as described <a href="#pone.0000462-Chowdari1">[49]</a>. Briefly, the patients were evaluated using the semi-structured diagnostic interview scale called the Diagnostic Interview for Genetic Studies <a href="#pone.0000462-Nurnberger1">[50]</a>. This information was combined with medical records and available information from relatives. Consensus diagnoses were made using DMS-IV criteria. The controls were drawn unscreened from neonates from a local hospital without any demographic details apart from gender and reported maternal ethnicity. The study was approved by the University of Pittsburgh Institutional Review Board. Written informed consent was obtained from all participants except for the neonate controls in accordance with IRB guidelines.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>For the AF population, DNA samples of 25 unrelated individuals and 30 parent-offspring trios were obtained from Coriell Cell Repositories (Camden, NJ). The 25 unrelated individuals (Female 9; Male 16) included eight from North of Sahara (Female 2; Male 6; Panel ID HD12), seven from South of Sahara (Female 2; Male 5; Panel ID: HD11), five Biaka African-Pygmy from Bagandu in the Central African Republic (Female 3, Male 2; Repository Number NA10469–NA10473), and five Mbuti African-Pygmy from Ituri Forest in northeast Zaire (Female 2; Male 3; Repository Number NA10492–NA10496). All members of each of the 30 parent-offspring trios from Yoruba in Ibadan, Nigeria (Female offspring 7; Male offspring 23; Panel ID: HAPMAPPT03) were genotyped and yielded successful genotyping percentage greater than 90% except for two mother samples (Repository Numbers GM19209 and GM19099), which accordingly were not included in present study. The accuracy in phase estimation of haplotypes for parents was confirmed by genotype data from the offspring. Only data of unrelated parents (Mother 28; Father 30) were used in the statistical analysis.</p>


<h4>Genomic DNA of non-human primates</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Non-human primate genomic DNA samples obtained from Coriell Cell Repositories (Camden, NJ) included (with Coriell Repository Numbers): <em>Macaca arctoides</em> (stumptail macaque, NA03443), <em>Macaca nemestrina</em> (pigtailed macaque, NG07921), <em>Macaca nigra</em> (celebes crested macaque, NG07101), <em>Macaca fascicularis</em> (crab-eating macaque, NA03446) and <em>Erythrocebus patas</em> (patas monkey, NG06254).</p>


<h4>Polymerase chain reaction and sequencing</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>To fully sequence the 3,551 base pairs (bp) genomic region flanked by the two schizophrenia-associated SNPs rs6556547 and rs187269 in <em>GABRB2</em> <a href="#pone.0000462-Lo1">[4]</a>, <a href="#pone.0000462-Lo2">[9]</a>, a 7.4-Kb genomic region starting from 2,148 bp upstream of Exon 9 to 519 bp downstream of Exon 10 was generated by polymerase chain reaction (PCR) and served as first PCR template for amplification of two nested-PCR products. List of potential primers for PCR and resequencing were designed using the Primer 3 program <a href="#pone.0000462-Rozen1">[51]</a>. The sequences of primers, designed based on chromosome 5 contig NT_023133.12, are listed in <a href="#pone.0000462.s007">Table S7</a>. Specificity of each potential primer was checked with BLASTN, an alignment tool of the National Center for Biotechnology Information (<a href="http://www.ncbi.nlm.nih.gov/BLAST/">www.ncbi.nlm.nih.gov/BLAST/</a>). Only specific pairs of primers, with less than 5 hits to the human genome and none of them in GABA<sub>A</sub> receptor genes other than <em>GABRB2</em>, were accepted. Together with the relatively long length (7.4 Kb) of the first PCR product and the employment of nested-PCR, this level of primer specificity reduced the probability of amplifying paralogous sequences in the genome.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>PCR amplification of the 7.4-Kb region was performed in a 20 µl mixture containing 70 ng of genomic DNA, 2 µl of 10× PCR buffer for KOD Hot-Start DNA Polymerase, 20 µM of each dNTP, 0.4 mM of MgSO<sub>4</sub>, 0.12 µM of each primer and 0.5 U of KOD Hot-Start DNA Polymerase (Novagen, Madison WI). PCR conditions were optimized using gradient PCR to ensure homogeneity of the amplified products. PCR consisted of initial polymerase activation at 95°C for 3 min, followed by 36 cycles each of 30 sec at 95°C, 30 sec at 62°C and 5 min and 30 sec at 68°C, and a final extension step at 68°C for 5 min and 30 sec. Post-PCR product was immediately purified by ethanol precipitation and recovered in 1× Tris-Cl (USB, Cleveland, Ohio) and ethylene-diamine-tetra-acetic acid (EDTA) (Invitrogen Corporation, Grand Island, NY) buffer, as described in Lo <em>et al</em>. <a href="#pone.0000462-Lo1">[4]</a>.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>Two nested-PCR fragments, one from 2,148 bp upstream to 498 bp downstream (fragment A), and the other from 342 bp upstream to 2,039 bp downstream (fragment B), of exon 9 were amplified using the 7.4-Kb purified PCR product as template. Each nested PCR reaction contained 0.6 µl of the 7.4-Kb purified PCR product, 75 nM of each primer, 50 nM of each dNTP, 2.5 mM of MgCl<sub>2</sub> and 1 U of <em>Taq</em> DNA polymerase (Amersham Bioscience, Uppsala, Sweden) in a final volume of 20 µl. The nested-PCR reaction consisted of an initial denaturation at 94°C for 2 min, followed by 35 cycles each of 30 sec at 95°C, 30 sec at 58°C, 90 sec at 72°C, plus a final extension step at 72°C for 5 min. After nested PCR, the product in each instance was resolved on 1.2% agarose gel, stained with 0.5 µg/ml of ethidium bromide, and examined under UV to confirm the presence of PCR product of expected size and absence of non-specific products. The stringency of nested-PCR amplification exceeded that for normal PCR, further eliminating non-specific products. The nested-PCR products were purified and recovered as for the first PCR product.</p>


<h4>SNP discovery and genotyping</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Both SNP discovery and genotyping were carried out by resequencing the nested-PCR products. For each of the nested PCR Fragment A and B, four sequencing primers were employed (<a href="#pone.0000462.s007">Table S7</a>). Each sequencing reaction contained 3 µl of sequencing buffer, 0.5 µl of BigBye® Terminator version 3.1 (Applied Biosystems Inc., Foster City, California), ~100 ng purified nested PCR products and 1 mM sequencing primer. Each cycle of sequencing reaction consisted of initial denaturation at 96°C for 1 min, followed by 25 cycles each of 10 s at 96°C, 5 s at 50°C, and 4 min at 60°C. Ethanol precipitation was used to clean-up the post-sequencing products as for the PCR products. Each air-dried sequencing sample was dissolved in 10 µl Hi-Deionized Formamide (Applied Biosystems Inc., Foster City, California), denatured at 95°C for 1 min and immediately held at 4°C prior to sequencing with a Model 3100 Genetic Analyzer (Applied Biosystems Inc., Foster City, California).</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Sequence chromatogram alignment-based SNP discovery and genotype calling were carried out using the software package PolyPhred version 4.2 <a href="#pone.0000462-Nickerson1">[52]</a>. All genotyping results were manually confirmed by at least two independent researchers. All analyzed SNPs were located within the high-quality region (Quality Value ≥20), and occasional low-quality passes were re-sequenced.</p>


<h4>Inference of haplotype phase</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Inference of haplotypes from the genotype data was performed using PHASE version 2.1 as described <a href="#pone.0000462-Stephens1">[53]</a>. The haplotype phases of both parents in AF and US family-trios were estimated based on the genotype data of the corresponding offspring using the P1 option of the program. The inferred haplotypes of AF and US parents were used as phase-known samples in the haplotype phase estimation of population-based genotype data in order to increase the accuracy of haplotype reconstruction. Input of genotype data of unrelated individuals was then placed into the program to infer haplotype phase under the default settings for five independent runs, except that the final iteration was carried out with a 10-fold longer running time. Thus the inferred haplotypes were obtained from five independent runs giving the best average goodness of fit.</p>


<h4>Construction of hypothetical human ancestral sequence</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Alignment of human and non-human primate sequences for a ~1.8-Kb region upstream of Exon 9 of <em>GABRB2</em> was performed using ClustalW <a href="#pone.0000462-Chenna1">[54]</a>. This revealed a human-specific Alu insertion in Intron 8 of <em>GABRB2</em>. Similarity between sequences was visualized using VISTA <a href="#pone.0000462-Mayor1">[55]</a>.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>To characterize the sub-family identity of the human-specific Alu insertion in Intron 8 of <em>GABRB2</em>, its sequence was searched against the repetitive elements library RepBase using the program RepeatMasker <a href="#pone.0000462-Smit1">[56]</a>. Consensus sequences of Alu subfamilies and the sequence of the inserted Alu were aligned using ClustalW <a href="#pone.0000462-Chenna1">[54]</a>. To obtain the consensus phylogenetic tree, the SEQBOOT, DNAPARS and CONSENSE programs in the PHYLIP software package <a href="#pone.0000462-Felsenstein1">[57]</a> were employed. The consensus phylogenetic tree was displayed using TreeView version 1.6.6 <a href="#pone.0000462-Page1">[58]</a>.</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>The hypothetical human ancestral sequence of the 3,551-bp region of <em>GABRB2</em> was constructed in two stages. For SNPs on the sequences flanking the human-specific Yi6AH151, chimpanzee sequences were used as outgroup to infer their ancestral alleles. For SNPs within Yi6AH151, consensus sequence of the Alu Yi6 subfamily was used for this purpose.</p>


<h4>Summary statistics of mutation parameters</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The Hudson, Kreitman and Aguadé's test <a href="#pone.0000462-Hudson1">[18]</a> is based on the neutral molecular evolution expectation that DNA sequence polymorphism within a species and DNA sequence divergence between species, will be proportional to the neutral mutation rate <a href="#pone.0000462-Kimura1">[59]</a>. By comparing the number of nucleotide differences between and within species for the 3,551-bp region against the genome average estimates using DnaSP, the rate of evolution in this region could be tested for departure from neutrality.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>DNA diversity level among human sequences was measured using the program DnaSP to yield four mutation parameters: haplotype diversity <em>H<sub>d</sub></em> <a href="#pone.0000462-Nei1">[60]</a> equivalent to gene heterozygosity, nucleotide diversity π <a href="#pone.0000462-Nei2">[61]</a> measuring the average number of pairwise differences, Watterson's θ-<sub>W</sub> <a href="#pone.0000462-Watterson1">[62]</a> based on the number of segregating sites, and θ-<sub>H</sub> <a href="#pone.0000462-Fay1">[63]</a> weighting the presence of high-frequency derived variants. Under the standard neutral model of a random-mating population of constant size <a href="#pone.0000462-Kimura1">[59]</a>, these genetic mutation parameters would each approach θ  = 4<em>Ne</em>µ, where <em>Ne</em> is the diploid long-term inbreeding effective population size and µ is the mutation rate per generation.</p>
<a id="article1.body1.sec4.sec7.p3" name="article1.body1.sec4.sec7.p3"></a><p>The four summary statistics Tajima's <em>D</em> <a href="#pone.0000462-Tajima1">[64]</a>, Fu and Li's <em>D</em> and <em>F</em> <a href="#pone.0000462-Fu1">[65]</a> and Fay and Wu's <em>H</em> <a href="#pone.0000462-Fay1">[63]</a> were evaluated regarding the site-frequency spectrum for or against an assumption of neutrality. Departures from the neutral model are usually attributed to selective or demographic effects. The Tajima's <em>D</em> test compares θ-<sub>W</sub> to π. The Fu and Li's <em>D</em> and <em>F</em> tests compare θ-<sub>W</sub> and π respectively to the number of derived unique mutations, which represent the mutations on external branches of the tree. The Fay and Wu's <em>H</em> test compares π to θ-<sub>H</sub>, and is sensitive for detecting an excess of high-frequency-derived alleles, from a hitchhiking effect under positive directional selection. For the Fu and Li's <em>D</em> and <em>F</em> and Fay and Wu's <em>H</em> tests, an outgroup sequence is required to define the ancestral status of alleles. The hypothetical human ancestral sequence was used as the outgroup sequence.</p>
<a id="article1.body1.sec4.sec7.p4" name="article1.body1.sec4.sec7.p4"></a><p>The significance of any difference in Fay and Wu's <em>H</em> value between HG-N and HG-D was tested with 10<sup>5</sup> permutations. In each permutation, the N and D alleles of SNP rs1816072 were randomly exchanged between the two groups of haplotypes, and absolute differences in Fay and Wu's <em>H</em> value were calculated between the two groups. The number of permutations out of the total of 10<sup>5</sup> permutations yielding a greater difference in value between HG-N and HG-D were counted to give the probability value <em>P</em>.</p>


<h4>Coalescence-based neutrality test with demographic models</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>The simulation method based on the coalescence process was employed using the <em>mksample</em> program <a href="#pone.0000462-Hudson2">[66]</a>. For each human population, the observed number of chromosomes and segregating sites were specified to generate 10<sup>5</sup> simulation datasets to test the significance of Fay and Wu's <em>H</em> and Tajima's <em>D</em> values and the probability of positive selection of the major haplotype for each of the demographic models. Recombination rate and gene conversion were stringently set at the 10<sup>−8</sup> per generation close to the genome average, and an average tract length of 100 bp, respectively, to minimize type I error.</p>
<a id="article1.body1.sec4.sec8.p2" name="article1.body1.sec4.sec8.p2"></a><p>The significance of Fay and Wu's <em>H</em> and Tajima's <em>D</em> values in each population was calculated using simulations of various demographic models under the assumption of neutral evolution. For each population, the number of simulation datasets showing Fay and Wu's <em>H</em> or Tajima's <em>D</em> values smaller than the observed <em>D</em> or <em>H</em> values was counted. On the other hand, in order to access the statistical probability of a history of positive selection at the major haplotype H56 in human populations, a null hypothesis of observing a haplotype containing four derived alleles having the same or a higher frequency than the observed frequency of H56 in neutral evolution with various demographic histories was tested.</p>
<a id="article1.body1.sec4.sec8.p3" name="article1.body1.sec4.sec8.p3"></a><p>A total of thirty-three demographic models, namely M1 to M33 were examined. The models of M1 to M9 embodied the effects of population growth and substructure as described by Evans <em>et al</em>. <a href="#pone.0000462-Evans1">[21]</a>. M10 to M25 were models with population growth and substructure based on Currat <em>et al</em>. <a href="#pone.0000462-Currat1">[23]</a>. M25 to M29 were population bottleneck models based on the demographic analysis of multiloci data by Voight <em>et al.</em> <a href="#pone.0000462-Voight1">[22]</a>. M30 to M33 were the low-migration models as described by Garrigan and Hammers <a href="#pone.0000462-Garrigan1">[24]</a>.</p>
<a id="article1.body1.sec4.sec8.p4" name="article1.body1.sec4.sec8.p4"></a><p>The parameters for Evans' models <a href="#pone.0000462-Evans1">[21]</a> M1 to M9 were as follows: <strong>M1</strong>, a constant effective population size of 10<sup>4</sup>; <strong>M2</strong>, exponential population expansion since 5,000 generations ago from an initial population size of 10<sup>4</sup> to a present population size of 10<sup>7</sup>; <strong>M3</strong>, exponential population expansion since 1,000 generations ago from an initial population size of 10<sup>4</sup> to a present population size of 10<sup>7</sup>; <strong>M4</strong>, a severe bottleneck starting 5,000 generations ago reduced the population size instantly from 10<sup>4</sup> to 10<sup>3</sup>, which remained constant until 2,500 generations ago, when the population expanded exponentially to a present size of 10<sup>7</sup>; <strong>M5</strong>, five consecutive bottleneck events started at 7,000 generations ago, in each instance reducing the population size from 10<sup>4</sup> instantly to 10<sup>3</sup>, following by constant size for 500 generations and subsequent recovery by exponential growth back to 10<sup>4</sup> in the ensuing 500 generations; starting at the end of the fifth bottleneck at 2,500 generations ago, the population expanded exponentially to a present size of 10<sup>7</sup>; <strong>M6 to M9</strong>, having a population structure where the total number of chromosomes for each human population was split equally into two to five subpopulations, with 1 migration per generation between any two populations under constant population size condition.</p>
<a id="article1.body1.sec4.sec8.p5" name="article1.body1.sec4.sec8.p5"></a><p>In Currat's models <a href="#pone.0000462-Currat1">[23]</a>, a subpopulation split off from a core population at 1,000 generations before the present, and grew exponentially from an initial effective population size (<em>n</em><sub>s</sub>) to the present size of 10<sup>7</sup> individuals. For each model, a range of <em>n</em><sub>s</sub> from 10 to 10<sup>4</sup> was tested. For the stable core models <strong>M10 to M13</strong> among them, the effective population size of the core population remained constant at 10<sup>5</sup>; bottleneck severities ranged from 0.0001 for <em>n<sub>s</sub></em> = 10 to 0.1 for <em>n<sub>s</sub></em> = 10<sup>4</sup>. For the growing core models <strong>M14 to M17</strong>, the population core grew from an initial size of 10<sup>4</sup> individuals at 1,000 generations ago to 10<sup>7</sup> at present. For the extended growing core models <strong>M18 to M21</strong>, the core population grew from 10<sup>4</sup> individuals at 5,000 generations ago to the present 10<sup>7</sup>. For the early fission models <strong>M22 to M25</strong>, a subpopulation split from a core population of <em>n</em><sub>s</sub> at 5,000 generations ago and remained constant in size until 1,000 generations ago, and grew thereafter to the present 10<sup>7</sup>, while the core population grew from 10<sup>4</sup> individuals at 5,000 generations ago to the present 10<sup>7</sup>.</p>
<a id="article1.body1.sec4.sec8.p6" name="article1.body1.sec4.sec8.p6"></a><p><strong>M26</strong> to <strong>M29</strong> were population-bottleneck models based on Voight <em>et al</em>. <a href="#pone.0000462-Voight1">[22]</a>. For all of these four models, a bottleneck event occurred at 1,600 generations ago (t<sub>start</sub>), when an ancestral population (N<sub>A</sub>) of 10<sup>4</sup> was reduced to a size of 10<sup>3</sup> for M26 (bottleneck severity, <em>b</em> = 0.1), 4×10<sup>3 </sup>for M27 (<em>b</em> = 0.4), and 50 for both M28 and M29 (<em>b</em> = 0.005). After a period of t<sub>dur</sub>, the population grew exponentially back to its original size of 10<sup>4</sup>. The t<sub>dur</sub> was 400 generations for M26, zero for M27, 600 for M28, and 300 for M29.</p>
<a id="article1.body1.sec4.sec8.p7" name="article1.body1.sec4.sec8.p7"></a><p>The low-migration models <strong>M30 to M33</strong>, based on Garrigan and Hammers <a href="#pone.0000462-Garrigan1">[24]</a>, described a structured ancestral population of 2,000 chromosomes, which admixed with two other demes of the same size at a low-migration rate (4Nm = 0.5) 32,000 generations ago (800 Kya); a bottleneck event occurred at 1,600 generations ago as described in M26 to M29, except that the population size recovered to 20,000 chromosomes.</p>


<h4>Estimation of SNP coalescence age</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Two established mutation-based methods were employed to estimate the coalescence age of SNP: a) using the molecular clock derived from the divergence between human and chimpanzee to determine the time to the Most Recent Common Ancestor MRCA <a href="#pone.0000462-Thomson1">[25]</a>; and b) using the Rho (ρ) index to define the age in unit of mutation from MRCA to the descendent haplotypes of SNPs <a href="#pone.0000462-Forster1">[27]</a>, and the mutation rate of human mitochondria DNA (mtDNA) <a href="#pone.0000462-Saillard1">[26]</a> to obtain the estimated coalescence age of SNPs. Both approaches were suggested to be unbiased by demographic history <a href="#pone.0000462-Thomson1">[25]</a>, <a href="#pone.0000462-Forster1">[27]</a>.</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>In the first method, the SNP age was estimated by the mutation-based method of Thomson <em>et al.</em> <a href="#pone.0000462-Thomson1">[25]</a>. Average number of mutations per base in lineages from the MRCA to haplotypes carrying the D allele of SNP (<em>d</em>) for each human population was calculated using the DnaSP version 4.10 <a href="#pone.0000462-Rozas1">[67]</a>. By comparing human and chimpanzee sequences in this region, the human-chimpanzee nucleotide divergence (<em>D</em>) in the region was estimated for human populations in mutations per base. Assuming a human-chimpanzee divergence time (<em>T</em><sub>split</sub>) of 6 Mya, the estimated mutation rate per site per year (μ) was obtained by the formula μ = <em>D</em>/(<em>2*T</em><sub>split</sub>). The time to MRCA (TMRCA) in years was obtained by the formula TMRCA = <em>d</em>*μ.</p>
<a id="article1.body1.sec4.sec9.p3" name="article1.body1.sec4.sec9.p3"></a><p>In the second method, the age of haplotypes carrying derived alleles of SNPs in terms of the number of mutations was measured by the Rho (ρ) index <a href="#pone.0000462-Forster1">[27]</a> by means of the program NETWORK version 4.2 (NETWORK website). A phylogenetic tree of inferred haplotypes was constructed using the median-joining method <a href="#pone.0000462-Bandelt1">[68]</a>. By defining a root node, i.e. the hypothetical human ancestral haplotype, ρ between the root node and a descendent haplotype or a group of descendent haplotypes, i.e. a haplotype group, was calculated. The ρ statistic represents the average number of mutational changes between the root node and an individual haplotype within the phylogeny relating the intra-allelic diversity of the root node and the haplotypes of interest. Relationship between ρ and TMRCA (<em>t</em>) is given by the equation, ρ = μ<em>t</em> where μ is the mutation rate per site per year. To estimate the TMRCA, the mutation rate μ for human mtDNA, which is equal to 1 mutation per 20,180 years <a href="#pone.0000462-Saillard1">[26]</a>, was employed for converting ρ to <em>t</em>. The age of a descendent haplotype is the same as the minimum age estimate for the preceding mutation.</p>


<h4>Expression and electrophysiology of recombinant GABA<sub>A</sub> receptors</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>Human embryonic kidney (HEK293) cells were transiently co-transfected with pcDNA3.1-α<sub>1</sub>, pcDNA3.1-γ<sub>2L</sub> plus either pcDNA3.1-β<sub>2S</sub> or pcDNA3.1-β<sub>2L</sub>, using the Genejuice transfection reagent (Novagen) as described <a href="#pone.0000462-Zhao1">[12]</a>. Whole-cell patch clamp recordings were carried out on the cells using an EPC9 amplifier (HEKA, Germany) at 36 to 60 hours after transfection. Cells were voltage-clamped at −60 mV. Pipette-to-bath resistance was 3.0–6.0 MΩ when filled with the internal solution containing 140 mM of CsCl, 1 mM of MgCl<sub>2</sub>, 11 mM of EGTA, 10 mM of HEPES, and 1, 2 or 4 mM of Mg<sup>++</sup>-ATP, adjusted to pH 7.3 with CsOH. The external superfusion solution contained 150 mM of NaCl, 3 mM of KCl, 1 mM of MgCl<sub>2</sub>, 1 mM of CaCl<sub>2</sub> and 10 mM of HEPES, adjusted to pH 7.2 with NaOH <a href="#pone.0000462-Greenfield1">[69]</a>. The cells were constantly perfused with the external solution at 5 ml/min.</p>
<a id="article1.body1.sec4.sec10.p2" name="article1.body1.sec4.sec10.p2"></a><p>Repetitive applications of GABA (100 or 300 µM, 5-s duration, 60-s interval) to induce current rundown was initiated when the cells were found to display consistent responses to low concentration of GABA (10 µM, 3-min interval) <a href="#pone.0000462-Huang1">[70]</a>. Thirty consecutive whole-cell responses to the repetitive applications were evaluated in terms of the observed peak current as percentile initial peak current. The results obtained from 7–10 cells collected from 6 independent transfections were presented in mean±standard error. Two-tailed Student's <em>t</em>-test was employed for calculation of statistical differences between receptors containing β<sub>2L</sub> and those containing β<sub>2S</sub>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000462"></div><a name="pone.0000462.s001" id="pone.0000462.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000462.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">All twenty-nine SNPs identified from unrelated non-schizophrenic individuals of four different populations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000462.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0000462.s002" id="pone.0000462.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000462.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Summary statistics of mutation frequency spectrum for haplotype-groups of rs1816072 in different populations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000462.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0000462.s003" id="pone.0000462.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000462.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Compositions and population frequencies of 75 inferred haplotypes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000462.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.06 MB XLS)</p>
<a name="pone.0000462.s004" id="pone.0000462.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000462.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Coalescence-based neutrality test of various demographic models.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000462.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.04 MB XLS)</p>
<a name="pone.0000462.s005" id="pone.0000462.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000462.s005">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Population frequencies and frequency ratio of the four high-D SNPs in male controls and schizophrenics.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000462.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0000462.s006" id="pone.0000462.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000462.s006">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Estimated coalescence age of the four high-D SNPs by mutation-based methods.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000462.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0000462.s007" id="pone.0000462.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000462.s007">Table S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Primers for PCR and sequencing of the 3,551-bp <em>GABRB2</em> fragment.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000462.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0000462.s008" id="pone.0000462.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000462.s008">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Sequence similarity between human and 7 non-human primates displayed by VISTA <a href="#pone.0000462-Mayor1">[55]</a>. The sequences of chimpanzee and rhesus were obtained from the NCBI database. The DNA of the five other non-human primates were sequenced over this 1.8 Kb region in <em>GABRB2</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000462.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.82 MB TIF)</p>
<a name="pone.0000462.s009" id="pone.0000462.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000462.s009">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">An unrooted phylogenetic tree of Alu sequences. (A) The tree was generated from an alignment of the consensus sequences of Alu sub-families and the Yi6AH151 using DNAPARS in the PHYLIP software package <a href="#pone.0000462-Felsenstein1">[57]</a>. The consensus tree generated from 1,000 replications is labeled with bootstrap values at the nodes and displayed with TreeView <a href="#pone.0000462-Page1">[58]</a>. The clade containing human-specific Alu Yi6AH151 and its closest neighbor Alu Yi6 is highlighted in grey. (B) Sequence alignment of Alu Yi6 consensus sequence and Yi6AH151. Refer to Salem et al. <a href="#pone.0000462-Salem1">[14]</a> for the sequences of all 150 members of Yi6 subfamily.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000462.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.98 MB TIF)</p>
<a name="pone.0000462.s010" id="pone.0000462.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000462.s010">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Plot of estimated recombination rate and location of recombination hotspots in <em>GABRB2</em>. The plot is adapted from Genome Browser (<a href="http://genome.ucsc.edu/cgi-bin/hgGateway">http://genome.ucsc.edu/cgi-bin/hgGateway</a>) report, which employed the HapMap Phase II data. The red box indicates the region of <em>GABRB2</em> studied.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000462.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.75 MB TIF)</p>
<a name="pone.0000462.s011" id="pone.0000462.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000462.s011">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Summary statistics for all HG-N and HG-D haplotypes. Tajima's <em>D</em> and Fu and Li's <em>D</em> values are plotted for AF, GE, US and JP.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000462.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.97 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Professor J. Tze-Fei Wong for critical review of the manuscript and Miss Peggy Lee for technical support. We would like to acknowledge Hewlett-Packard for the DL580 server used in this project. The authors declare that there are no competing financial interests.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: WL HX SN ZX. Performed the experiments: WL ZY FP ZX. Analyzed the data: WL HX SN ZX. Contributed reagents/materials/analysis tools: HX MH JC KT CZ SN GS VN XX. Wrote the paper: WL HX ST ZX.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000462-Gottesman1" id="pone.0000462-Gottesman1"></a>Gottesman II (1989) Vital statistics, demography, and schizophrenia: editor's introduction. Schizophr Bull  15: 5–7.  <ul class="find" data-citedArticleID="953220" data-doi="10.1093/schbul/15.1.5"><li><a href="http://dx.doi.org/10.1093/schbul/15.1.5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vital+statistics%2C+demography%2C+and+schizophrenia%3A+editor%27s+introduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vital+statistics%2C+demography%2C+and+schizophrenia%3A+editor%27s+introduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000462-Austin1" id="pone.0000462-Austin1"></a>Austin J (2005) Schizophrenia: an update and review. J Genet Couns  14: 329–340.  <ul class="find" data-citedArticleID="953178" data-doi="10.1007/s10897-005-1622-4"><li><a href="http://dx.doi.org/10.1007/s10897-005-1622-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Schizophrenia%3A+an+update+and+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Schizophrenia%3A+an+update+and+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000462-Harrison1" id="pone.0000462-Harrison1"></a>Harrison PJ, Weinberger DR (2005) Schizophrenia genes, gene expression, and neuropathology: on the matter of their convergence. Mol Psychiatry  10: 40–68.  <ul class="find" data-citedArticleID="953224" data-doi="10.1038/sj.mp.4001558"><li><a href="http://dx.doi.org/10.1038/sj.mp.4001558" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Schizophrenia+genes%2C+gene+expression%2C+and+neuropathology%3A+on+the+matter+of+their+convergence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Schizophrenia+genes%2C+gene+expression%2C+and+neuropathology%3A+on+the+matter+of+their+convergence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000462-Lo1" id="pone.0000462-Lo1"></a>Lo WS, Lau CF, Xuan Z, Chan CF, Feng GY, et al.  (2004) Association of SNPs and haplotypes in GABA<sub>A</sub> receptor β<sub>2</sub> gene with schizophrenia. Mol Psychiatry  9: 603–608.  <ul class="find" data-citedArticleID="953244" data-doi="10.1038/sj.mp.4001461"><li><a href="http://dx.doi.org/10.1038/sj.mp.4001461" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+of+SNPs+and+haplotypes+in+GABAA+receptor+%26%23x3b2%3B2+gene+with+schizophrenia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+of+SNPs+and+haplotypes+in+GABAA+receptor+%26%23x3b2%3B2+gene+with+schizophrenia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000462-Korpi1" id="pone.0000462-Korpi1"></a>Korpi ER, Sinkkonen ST (2006) GABA<sub>A</sub> receptor subtypes as targets for neuropsychiatric drug development. Pharmacology &amp; Therapeutics  109: 12–32.  <ul class="find" data-citedArticleID="953238" data-doi="10.1016/j.pharmthera.2005.05.009"><li><a href="http://dx.doi.org/10.1016/j.pharmthera.2005.05.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GABAA+receptor+subtypes+as+targets+for+neuropsychiatric+drug+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GABAA+receptor+subtypes+as+targets+for+neuropsychiatric+drug+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000462-Liu1" id="pone.0000462-Liu1"></a>Liu J, Shi Y, Tang W, Guo T, Li D, et al.  (2005) Positive association of the human GABA<sub>A</sub> receptor β<sub>2</sub> subunit gene haplotype with schizophrenia in the Chinese <em>Han</em> population. Biochem Biophys Res Commun  334: 817–823.  <ul class="find" data-citedArticleID="953242" data-doi="10.1016/j.bbrc.2005.06.167"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2005.06.167" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+association+of+the+human+GABAA+receptor+%26%23x3b2%3B2+subunit+gene+haplotype+with+schizophrenia+in+the+Chinese+Han+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+association+of+the+human+GABAA+receptor+%26%23x3b2%3B2+subunit+gene+haplotype+with+schizophrenia+in+the+Chinese+Han+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000462-Petryshen1" id="pone.0000462-Petryshen1"></a>Petryshen TL, Middleton FA, Tahl AR, Rockwell GN, Purcell S, et al.  (2005) Genetic investigation of chromosome 5q GABA<sub>A</sub> receptor subunit genes in schizophrenia. Mol Psychiatry  10: 1074–88, 1057.  <ul class="find" data-citedArticleID="953270" data-doi="10.1038/sj.mp.4001739"><li><a href="http://dx.doi.org/10.1038/sj.mp.4001739" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+investigation+of+chromosome+5q+GABAA+receptor+subunit+genes+in+schizophrenia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+investigation+of+chromosome+5q+GABAA+receptor+subunit+genes+in+schizophrenia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000462-Yu1" id="pone.0000462-Yu1"></a>Yu Z, Chen J, Shi H, Stoëber G, Tsang SY, et al.  (2006) Analysis of <em>GABRB2</em> association with schizophrenia in German population with DNA sequencing and one-label extension method for SNP genotyping. Clin Biochem  39: 210–218.  <ul class="find" data-citedArticleID="953310" data-doi="10.1016/j.clinbiochem.2006.01.009"><li><a href="http://dx.doi.org/10.1016/j.clinbiochem.2006.01.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+GABRB2+association+with+schizophrenia+in+German+population+with+DNA+sequencing+and+one-label+extension+method+for+SNP+genotyping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+GABRB2+association+with+schizophrenia+in+German+population+with+DNA+sequencing+and+one-label+extension+method+for+SNP+genotyping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000462-Lo2" id="pone.0000462-Lo2"></a>Lo WS, Harano M, Gawlik M, Yu Z, Chen J, et al.  (2007) <em>GABRB2</em> association with schizophrenia: commonalities and differences between ethnic groups and clinical subtypes. Biol Psychiatry  61: 653–660.  <ul class="find" data-citedArticleID="953246" data-doi="10.1016/j.biopsych.2006.05.003"><li><a href="http://dx.doi.org/10.1016/j.biopsych.2006.05.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GABRB2+association+with+schizophrenia%3A+commonalities+and+differences+between+ethnic+groups+and+clinical+subtypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GABRB2+association+with+schizophrenia%3A+commonalities+and+differences+between+ethnic+groups+and+clinical+subtypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000462-Ikeda1" id="pone.0000462-Ikeda1"></a>Ikeda M, Iwata N, Suzuki T, Kitajima T, Yamanouchi Y, et al.  (2005) Association analysis of chromosome 5 GABA<sub>A</sub> receptor cluster in Japanese schizophrenia patients. Biol Psychiatry  58: 440–445.  <ul class="find" data-citedArticleID="953232" data-doi="10.1016/j.biopsych.2005.05.002"><li><a href="http://dx.doi.org/10.1016/j.biopsych.2005.05.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+analysis+of+chromosome+5+GABAA+receptor+cluster+in+Japanese+schizophrenia+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+analysis+of+chromosome+5+GABAA+receptor+cluster+in+Japanese+schizophrenia+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000462-Jamra1" id="pone.0000462-Jamra1"></a>Jamra RA, Becker T, Klopp N, Dahdouh F, Schulze TG, et al.  (2007) No evidence for an association between variants at the GABA<sub>A</sub> receptor β<sub>2</sub> locus and schizophrenia. Psychiatr Genet  17: 43–45.  <ul class="find" data-citedArticleID="953234" data-doi="10.1097/ypg.0b013e32801118cd"><li><a href="http://dx.doi.org/10.1097/ypg.0b013e32801118cd" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=No+evidence+for+an+association+between+variants+at+the+GABAA+receptor+%26%23x3b2%3B2+locus+and+schizophrenia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22No+evidence+for+an+association+between+variants+at+the+GABAA+receptor+%26%23x3b2%3B2+locus+and+schizophrenia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000462-Zhao1" id="pone.0000462-Zhao1"></a>Zhao C, Xu Z, Chen J, Yu Z, Tong KL, et al.  (2006) Two isoforms of GABA<sub>A</sub> receptor β<sub>2</sub> subunit with different electrophysiological properties: differential expression and genotypical correlations in schizophrenia. Mol Psychiatry  11: 1092–1105.  <ul class="find" data-citedArticleID="953312" data-doi="10.1038/sj.mp.4001899"><li><a href="http://dx.doi.org/10.1038/sj.mp.4001899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+isoforms+of+GABAA+receptor+%26%23x3b2%3B2+subunit+with+different+electrophysiological+properties%3A+differential+expression+and+genotypical+correlations+in+schizophrenia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+isoforms+of+GABAA+receptor+%26%23x3b2%3B2+subunit+with+different+electrophysiological+properties%3A+differential+expression+and+genotypical+correlations+in+schizophrenia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000462-Deininger1" id="pone.0000462-Deininger1"></a>Deininger PL, Batzer MA (2002) Mammalian retroelements. Genome Res  12: 1455–1465.  <ul class="find" data-citedArticleID="953196" data-doi="10.1101/gr.282402"><li><a href="http://dx.doi.org/10.1101/gr.282402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+retroelements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+retroelements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000462-Salem1" id="pone.0000462-Salem1"></a>Salem AH, Kilroy GE, Watkins WS, Jorde LB, Batzer MA (2003) Recently integrated Alu elements and human genomic diversity. Mol Biol Evol  20: 1349–1361.  <ul class="find" data-citedArticleID="953282" data-doi="10.1093/molbev/msg150"><li><a href="http://dx.doi.org/10.1093/molbev/msg150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recently+integrated+Alu+elements+and+human+genomic+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recently+integrated+Alu+elements+and+human+genomic+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000462-Deininger2" id="pone.0000462-Deininger2"></a>Deininger PL, Batzer MA (1999) Alu repeats and human disease. Mol Genet Metab  67: 183–193.  <ul class="find" data-citedArticleID="953198" data-doi="10.1006/mgme.1999.2864"><li><a href="http://dx.doi.org/10.1006/mgme.1999.2864" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alu+repeats+and+human+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alu+repeats+and+human+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000462-Venter1" id="pone.0000462-Venter1"></a>Venter JC, Adams MD, Myers EW, Li PW, Mural RJ, et al.  (2001) The sequence of the human genome. Science  291: 1304–1351.  <ul class="find" data-citedArticleID="953302" data-doi="10.1126/science.1058040"><li><a href="http://dx.doi.org/10.1126/science.1058040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sequence+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sequence+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000462-The1" id="pone.0000462-The1"></a>The Chimpanzee Sequencing and Analysis Consortium (2005) Initial sequence of the chimpanzee genome and comparison with the human genome. Nature  437: 69–87.  <ul class="find" data-citedArticleID="953294" data-doi="10.1038/nature04072"><li><a href="http://dx.doi.org/10.1038/nature04072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequence+of+the+chimpanzee+genome+and+comparison+with+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequence+of+the+chimpanzee+genome+and+comparison+with+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000462-Hudson1" id="pone.0000462-Hudson1"></a>Hudson RR, Kreitman M, Aguade M (1987) A test of neutral molecular evolution based on nucleotide data. Genetics  116: 153–159.  <ul class="find" data-citedArticleID="953228"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+test+of+neutral+molecular+evolution+based+on+nucleotide+data.&amp;auth=&amp;atitle=A+test+of+neutral+molecular+evolution+based+on+nucleotide+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+test+of+neutral+molecular+evolution+based+on+nucleotide+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+test+of+neutral+molecular+evolution+based+on+nucleotide+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000462-Bird1" id="pone.0000462-Bird1"></a>Bird CP, Stranger BE, Dermitzakis ET (2006) Functional variation and evolution of non-coding DNA. Curr Opin Genet Dev  16: 559–564.  <ul class="find" data-citedArticleID="953184" data-doi="10.1016/j.gde.2006.10.003"><li><a href="http://dx.doi.org/10.1016/j.gde.2006.10.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+variation+and+evolution+of+non-coding+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+variation+and+evolution+of+non-coding+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000462-Drake1" id="pone.0000462-Drake1"></a>Drake JA, Bird C, Nemesh J, Thomas DJ, Newton-Cheh C, et al.  (2006) Conserved noncoding sequences are selectively constrained and not mutation cold spots. Nat Genet  38: 223–227.  <ul class="find" data-citedArticleID="953202" data-doi="10.1038/ng1710"><li><a href="http://dx.doi.org/10.1038/ng1710" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+noncoding+sequences+are+selectively+constrained+and+not+mutation+cold+spots." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+noncoding+sequences+are+selectively+constrained+and+not+mutation+cold+spots.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000462-Evans1" id="pone.0000462-Evans1"></a>Evans PD, Gilbert SL, Mekel-Bobrov N, Vallender EJ, Anderson JR, et al.  (2005) Microcephalin, a gene regulating brain size, continues to evolve adaptively in humans. Science  309: 1717–1720.  <ul class="find" data-citedArticleID="953204" data-doi="10.1126/science.1113722"><li><a href="http://dx.doi.org/10.1126/science.1113722" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microcephalin%2C+a+gene+regulating+brain+size%2C+continues+to+evolve+adaptively+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microcephalin%2C+a+gene+regulating+brain+size%2C+continues+to+evolve+adaptively+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000462-Voight1" id="pone.0000462-Voight1"></a>Voight BF, Adams AM, Frisse LA, Qian Y, Hudson RR, et al.  (2005) Interrogating multiple aspects of variation in a full resequencing data set to infer human population size changes. Proc Natl Acad Sci USA  102: 18508–18513.  <ul class="find" data-citedArticleID="953304" data-doi="10.1073/pnas.0507325102"><li><a href="http://dx.doi.org/10.1073/pnas.0507325102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interrogating+multiple+aspects+of+variation+in+a+full+resequencing+data+set+to+infer+human+population+size+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interrogating+multiple+aspects+of+variation+in+a+full+resequencing+data+set+to+infer+human+population+size+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000462-Currat1" id="pone.0000462-Currat1"></a>Currat M, Excoffier L, Maddison W, Otto SP, Ray N, et al.  (2006) Comment on “Ongoing adaptive evolution of <em>ASPM</em>, a brain size determinant in Homo sapiens” and “Microcephalin, a gene regulating brain size, continues to evolve adaptively in humans”. Science  313: 172.  <ul class="find" data-citedArticleID="953192" data-doi="10.1126/science.1122712"><li><a href="http://dx.doi.org/10.1126/science.1122712" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comment+on+%26%23x201c%3BOngoing+adaptive+evolution+of+ASPM%2C+a+brain+size+determinant+in+Homo+sapiens%26%23x201d%3B+and+%26%23x201c%3BMicrocephalin%2C+a+gene+regulating+brain+size%2C+continues+to+evolve+adaptively+in+humans%26%23x201d%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comment+on+%26%23x201c%3BOngoing+adaptive+evolution+of+ASPM%2C+a+brain+size+determinant+in+Homo+sapiens%26%23x201d%3B+and+%26%23x201c%3BMicrocephalin%2C+a+gene+regulating+brain+size%2C+continues+to+evolve+adaptively+in+humans%26%23x201d%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000462-Garrigan1" id="pone.0000462-Garrigan1"></a>Garrigan D, Hammer MF (2006) Reconstructing human origins in the genomic era. Nat Rev Genet  7: 669–680.  <ul class="find" data-citedArticleID="953216" data-doi="10.1038/nrg1941"><li><a href="http://dx.doi.org/10.1038/nrg1941" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstructing+human+origins+in+the+genomic+era." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstructing+human+origins+in+the+genomic+era.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000462-Thomson1" id="pone.0000462-Thomson1"></a>Thomson R, Pritchard JK, Shen P, Oefner PJ, Feldman MW (2000) Recent common ancestry of human Y chromosomes: evidence from DNA sequence data. Proc Natl Acad Sci USA  97: 7360–7365.  <ul class="find" data-citedArticleID="953298" data-doi="10.1073/pnas.97.13.7360"><li><a href="http://dx.doi.org/10.1073/pnas.97.13.7360" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+common+ancestry+of+human+Y+chromosomes%3A+evidence+from+DNA+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+common+ancestry+of+human+Y+chromosomes%3A+evidence+from+DNA+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000462-Saillard1" id="pone.0000462-Saillard1"></a>Saillard J, Forster P, Lynnerup N, Bandelt HJ, Norby S (2000) mtDNA variation among Greenland Eskimos: the edge of the Beringian expansion. Am J Hum Genet  67: 718–726.  <ul class="find" data-citedArticleID="953280" data-doi="10.1086/303038"><li><a href="http://dx.doi.org/10.1086/303038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=mtDNA+variation+among+Greenland+Eskimos%3A+the+edge+of+the+Beringian+expansion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22mtDNA+variation+among+Greenland+Eskimos%3A+the+edge+of+the+Beringian+expansion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000462-Forster1" id="pone.0000462-Forster1"></a>Forster P, Harding R, Torroni A, Bandelt HJ (1996) Origin and evolution of Native American mtDNA variation: a reappraisal. Am J Hum Genet  59: 935–945.  <ul class="find" data-citedArticleID="953210"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Origin+and+evolution+of+Native+American+mtDNA+variation%3A+a+reappraisal.&amp;auth=&amp;atitle=Origin+and+evolution+of+Native+American+mtDNA+variation%3A+a+reappraisal." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+and+evolution+of+Native+American+mtDNA+variation%3A+a+reappraisal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+and+evolution+of+Native+American+mtDNA+variation%3A+a+reappraisal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000462-White1" id="pone.0000462-White1"></a>White TD, Asfaw B, DeGusta D, Gilbert H, Richards GD, et al.  (2003) Pleistocene Homo sapiens from Middle Awash, Ethiopia. Nature  423: 742–747.  <ul class="find" data-citedArticleID="953308" data-doi="10.1038/nature01669"><li><a href="http://dx.doi.org/10.1038/nature01669" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pleistocene+Homo+sapiens+from+Middle+Awash%2C+Ethiopia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pleistocene+Homo+sapiens+from+Middle+Awash%2C+Ethiopia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000462-Mellars1" id="pone.0000462-Mellars1"></a>Mellars P (2006) Why did modern human populations disperse from Africa ca. 60,000 years ago? A new model. Proc Natl Acad Sci USA  103: 9381–9386.  <ul class="find" data-citedArticleID="953250" data-doi="10.1073/pnas.0510792103"><li><a href="http://dx.doi.org/10.1073/pnas.0510792103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+did+modern+human+populations+disperse+from+Africa+ca.+60%2C000+years+ago%3F+A+new+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+did+modern+human+populations+disperse+from+Africa+ca.+60%2C000+years+ago%3F+A+new+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000462-Forster2" id="pone.0000462-Forster2"></a>Forster P (2004) Ice Ages and the mitochondrial DNA chronology of human dispersals: a review. Philos Trans R Soc Lond B Biol Sci  359: 255–264.  <ul class="find" data-citedArticleID="953212" data-doi="10.1098/rstb.2003.1394"><li><a href="http://dx.doi.org/10.1098/rstb.2003.1394" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ice+Ages+and+the+mitochondrial+DNA+chronology+of+human+dispersals%3A+a+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ice+Ages+and+the+mitochondrial+DNA+chronology+of+human+dispersals%3A+a+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000462-Rosenberg1" id="pone.0000462-Rosenberg1"></a>Rosenberg NA, Pritchard JK, Weber JL, Cann HM, Kidd KK, et al.  (2002) Genetic structure of human populations. Science  298: 2381–2385.  <ul class="find" data-citedArticleID="953274" data-doi="10.1126/science.1078311"><li><a href="http://dx.doi.org/10.1126/science.1078311" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+structure+of+human+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+structure+of+human+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000462-Batzer1" id="pone.0000462-Batzer1"></a>Batzer MA, Deininger PL (2002) Alu repeats and human genomic diversity. Nat Rev Genet  3: 370–379.  <ul class="find" data-citedArticleID="953182" data-doi="10.1038/nrg798"><li><a href="http://dx.doi.org/10.1038/nrg798" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alu+repeats+and+human+genomic+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alu+repeats+and+human+genomic+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000462-Otto1" id="pone.0000462-Otto1"></a>Otto SP, Barton NH (1997) The evolution of recombination: removing the limits to natural selection. Genetics  147: 879–906.  <ul class="find" data-citedArticleID="953264"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+evolution+of+recombination%3A+removing+the+limits+to+natural+selection.&amp;auth=&amp;atitle=The+evolution+of+recombination%3A+removing+the+limits+to+natural+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+recombination%3A+removing+the+limits+to+natural+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+recombination%3A+removing+the+limits+to+natural+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000462-Sapolsky1" id="pone.0000462-Sapolsky1"></a>Sapolsky RM, Share LJ (2004) A pacific culture among wild baboons: its emergence and transmission. PLoS Biol  2: E106.  <ul class="find" data-citedArticleID="953284" data-doi="10.1371/journal.pbio.0020106"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+pacific+culture+among+wild+baboons%3A+its+emergence+and+transmission." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+pacific+culture+among+wild+baboons%3A+its+emergence+and+transmission.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000462-Pike1" id="pone.0000462-Pike1"></a>Pike IL, Williams SR (2006) Incorporating psychosocial health into biocultural models: preliminary findings from Turkana women of Kenya. Am J Hum Biol  18: 729–740.  <ul class="find" data-citedArticleID="953272" data-doi="10.1002/ajhb.20548"><li><a href="http://dx.doi.org/10.1002/ajhb.20548" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Incorporating+psychosocial+health+into+biocultural+models%3A+preliminary+findings+from+Turkana+women+of+Kenya." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Incorporating+psychosocial+health+into+biocultural+models%3A+preliminary+findings+from+Turkana+women+of+Kenya.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000462-Sapolsky2" id="pone.0000462-Sapolsky2"></a>Sapolsky RM (2005) The influence of social hierarchy on primate health. Science  308: 648–652.  <ul class="find" data-citedArticleID="953286" data-doi="10.1126/science.1106477"><li><a href="http://dx.doi.org/10.1126/science.1106477" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+influence+of+social+hierarchy+on+primate+health." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+influence+of+social+hierarchy+on+primate+health.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000462-Cohen1" id="pone.0000462-Cohen1"></a>Cohen MN, Armelagos GJ (1984) Paleopathology at the Origins of Agriculture: Editor's Summation. Paleopathology at the Origins of Agriculture. Orlando, Florida: Academic Press. pp. 585–601.  <ul class="find-nolinks"></ul></li><li><span class="label">38.
              </span><a name="pone.0000462-Parnas1" id="pone.0000462-Parnas1"></a>Parnas J (1988) Assortative mating in schizophrenia: results from the Copenhagen High-Risk Study. Psychiatry  51: 58–64.  <ul class="find" data-citedArticleID="953268"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Assortative+mating+in+schizophrenia%3A+results+from+the+Copenhagen+High-Risk+Study.&amp;auth=&amp;atitle=Assortative+mating+in+schizophrenia%3A+results+from+the+Copenhagen+High-Risk+Study." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assortative+mating+in+schizophrenia%3A+results+from+the+Copenhagen+High-Risk+Study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assortative+mating+in+schizophrenia%3A+results+from+the+Copenhagen+High-Risk+Study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000462-Nakajima1" id="pone.0000462-Nakajima1"></a>Nakajima T, Wooding S, Sakagami T, Emi M, Tokunaga K, et al.  (2004) Natural selection and population history in the human angiotensinogen gene (<em>AGT</em>): 736 complete <em>AGT</em> sequences in chromosomes from around the world. Am J Hum Genet  74: 898–916.  <ul class="find" data-citedArticleID="953252" data-doi="10.1086/420793"><li><a href="http://dx.doi.org/10.1086/420793" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+and+population+history+in+the+human+angiotensinogen+gene+%28AGT%29%3A+736+complete+AGT+sequences+in+chromosomes+from+around+the+world." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+and+population+history+in+the+human+angiotensinogen+gene+%28AGT%29%3A+736+complete+AGT+sequences+in+chromosomes+from+around+the+world.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000462-Thompson1" id="pone.0000462-Thompson1"></a>Thompson EE, Kuttab-Boulos H, Witonsky D, Yang L, Roe BA, et al.  (2004) <em>CYP3A</em> variation and the evolution of salt-sensitivity variants. Am J Hum Genet  75: 1059–1069.  <ul class="find" data-citedArticleID="953296" data-doi="10.1086/426406"><li><a href="http://dx.doi.org/10.1086/426406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CYP3A+variation+and+the+evolution+of+salt-sensitivity+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CYP3A+variation+and+the+evolution+of+salt-sensitivity+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000462-Vander1" id="pone.0000462-Vander1"></a>Vander MJ, Frisse LM, Fullerton SM, Qian Y, Del Bosque-Plata L, et al.  (2005) Population genetics of <em>CAPN10</em> and <em>GPR35</em>: implications for the evolution of type-II diabetes variants. Am J Hum Genet  76: 548–560.  <ul class="find" data-citedArticleID="953300" data-doi="10.1086/428784"><li><a href="http://dx.doi.org/10.1086/428784" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+genetics+of+CAPN10+and+GPR35%3A+implications+for+the+evolution+of+type-II+diabetes+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+genetics+of+CAPN10+and+GPR35%3A+implications+for+the+evolution+of+type-II+diabetes+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000462-Altshuler1" id="pone.0000462-Altshuler1"></a>Altshuler D, Hirschhorn JN, Klannemark M, Lindgren CM, Vohl MC, et al.  (2000) The common <em>PPARgamma</em> Pro12Ala polymorphism is associated with decreased risk of type-II diabetes. Nat Genet  26: 76–80.  <ul class="find" data-citedArticleID="953174"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+common+PPARgamma+Pro12Ala+polymorphism+is+associated+with+decreased+risk+of+type-II+diabetes.&amp;auth=&amp;atitle=The+common+PPARgamma+Pro12Ala+polymorphism+is+associated+with+decreased+risk+of+type-II+diabetes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+common+PPARgamma+Pro12Ala+polymorphism+is+associated+with+decreased+risk+of+type-II+diabetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+common+PPARgamma+Pro12Ala+polymorphism+is+associated+with+decreased+risk+of+type-II+diabetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000462-Deeb1" id="pone.0000462-Deeb1"></a>Deeb SS, Fajas L, Nemoto M, Pihlajamaki J, Mykkanen L, et al.  (1998) A Pro12Ala substitution in <em>PPARgamma2</em> associated with decreased receptor activity, lower body mass index and improved insulin sensitivity. Nat Genet  20: 284–287.  <ul class="find" data-citedArticleID="953194"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+Pro12Ala+substitution+in+PPARgamma2+associated+with+decreased+receptor+activity%2C+lower+body+mass+index+and+improved+insulin+sensitivity.&amp;auth=&amp;atitle=A+Pro12Ala+substitution+in+PPARgamma2+associated+with+decreased+receptor+activity%2C+lower+body+mass+index+and+improved+insulin+sensitivity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Pro12Ala+substitution+in+PPARgamma2+associated+with+decreased+receptor+activity%2C+lower+body+mass+index+and+improved+insulin+sensitivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Pro12Ala+substitution+in+PPARgamma2+associated+with+decreased+receptor+activity%2C+lower+body+mass+index+and+improved+insulin+sensitivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000462-DiRienzo1" id="pone.0000462-DiRienzo1"></a>Di Rienzo A, Hudson RR (2005) An evolutionary framework for common diseases: the ancestral-susceptibility model. Trends Genet  21: 596–601.  <ul class="find" data-citedArticleID="953200" data-doi="10.1016/j.tig.2005.08.007"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.08.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+evolutionary+framework+for+common+diseases%3A+the+ancestral-susceptibility+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+evolutionary+framework+for+common+diseases%3A+the+ancestral-susceptibility+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000462-Gleibermann1" id="pone.0000462-Gleibermann1"></a>Gleibermann L (1973) Blood pressure and dietary salt in human populations. Ecol Food Nutr  2: 143–156.  <ul class="find" data-citedArticleID="953218" data-doi="10.1080/03670244.1973.9990329"><li><a href="http://dx.doi.org/10.1080/03670244.1973.9990329" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Blood+pressure+and+dietary+salt+in+human+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Blood+pressure+and+dietary+salt+in+human+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000462-Neel1" id="pone.0000462-Neel1"></a>Neel JV (1962) Diabetes mellitus: a “thrifty” genotype rendered detrimental by “progress”? Am J Hum Genet  14: 353–362.  <ul class="find" data-citedArticleID="953254"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Diabetes+mellitus%3A+a+%26%23x201c%3Bthrifty%26%23x201d%3B+genotype+rendered+detrimental+by+%26%23x201c%3Bprogress%26%23x201d%3B%3F&amp;auth=&amp;atitle=Diabetes+mellitus%3A+a+%26%23x201c%3Bthrifty%26%23x201d%3B+genotype+rendered+detrimental+by+%26%23x201c%3Bprogress%26%23x201d%3B%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diabetes+mellitus%3A+a+%26%23x201c%3Bthrifty%26%23x201d%3B+genotype+rendered+detrimental+by+%26%23x201c%3Bprogress%26%23x201d%3B%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diabetes+mellitus%3A+a+%26%23x201c%3Bthrifty%26%23x201d%3B+genotype+rendered+detrimental+by+%26%23x201c%3Bprogress%26%23x201d%3B%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000462-American1" id="pone.0000462-American1"></a>American Psychiatric Association (1994) Diagnostic and Statistical Manual of Mental Disorder, the fourth edition. Washington, DC: American Psychiatric Press.   <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0000462-Leonhard1" id="pone.0000462-Leonhard1"></a>Leonhard K (1999) Classification of Endogenous Psychoses and their Differentiated Etiology. New York: Springer.   <ul class="find-nolinks"></ul></li><li><span class="label">49.
              </span><a name="pone.0000462-Chowdari1" id="pone.0000462-Chowdari1"></a>Chowdari KV, Mirnics K, Semwal P, Wood J, Lawrence E, et al.  (2002) Association and linkage analyses of <em>RGS4</em> polymorphisms in schizophrenia. Hum Mol Genet  11: 1373–1380.  <ul class="find" data-citedArticleID="953188" data-doi="10.1093/hmg/11.12.1373"><li><a href="http://dx.doi.org/10.1093/hmg/11.12.1373" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+and+linkage+analyses+of+RGS4+polymorphisms+in+schizophrenia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+and+linkage+analyses+of+RGS4+polymorphisms+in+schizophrenia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000462-Nurnberger1" id="pone.0000462-Nurnberger1"></a>Nurnberger JI Jr, Blehar MC, Kaufmann CA, York-Cooler C, Simpson SG, et al.  (1994) Diagnostic interview for genetic studies. Rationale, unique features, and training. NIMH Genetics Initiative. Arch Gen Psychiatry  51: 849–859.  <ul class="find" data-citedArticleID="953262" data-doi="10.1001/archpsyc.1994.03950110009002"><li><a href="http://dx.doi.org/10.1001/archpsyc.1994.03950110009002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diagnostic+interview+for+genetic+studies.+Rationale%2C+unique+features%2C+and+training.+NIMH+Genetics+Initiative." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diagnostic+interview+for+genetic+studies.+Rationale%2C+unique+features%2C+and+training.+NIMH+Genetics+Initiative.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000462-Rozen1" id="pone.0000462-Rozen1"></a>Rozen S, Skaletsky H (2000)  Bioinformatics Methods and Protocols: Methods in Molecular Biology. In: Krawetz S, Misener S, editors. Primer3 on the WWW for general users and for biologist programmers. Totowa, NJ: Humana Press. pp. 365–386.  <ul class="find-nolinks"></ul></li><li><span class="label">52.
              </span><a name="pone.0000462-Nickerson1" id="pone.0000462-Nickerson1"></a>Nickerson DA, Tobe VO, Taylor SL (1997) PolyPhred: automating the detection and genotyping of single nucleotide substitutions using fluorescence-based resequencing. Nucleic Acids Res  25: 2745–2751.  <ul class="find" data-citedArticleID="953260" data-doi="10.1093/nar/25.14.2745"><li><a href="http://dx.doi.org/10.1093/nar/25.14.2745" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PolyPhred%3A+automating+the+detection+and+genotyping+of+single+nucleotide+substitutions+using+fluorescence-based+resequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PolyPhred%3A+automating+the+detection+and+genotyping+of+single+nucleotide+substitutions+using+fluorescence-based+resequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000462-Stephens1" id="pone.0000462-Stephens1"></a>Stephens M, Smith NJ, Donnelly P (2001) A new statistical method for haplotype reconstruction from population data. Am J Hum Genet  68: 978–989.  <ul class="find" data-citedArticleID="953290" data-doi="10.1086/319501"><li><a href="http://dx.doi.org/10.1086/319501" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+statistical+method+for+haplotype+reconstruction+from+population+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+statistical+method+for+haplotype+reconstruction+from+population+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000462-Chenna1" id="pone.0000462-Chenna1"></a>Chenna R, Sugawara H, Koike T, Lopez R, Gibson TJ, et al.  (2003) Multiple sequence alignment with the Clustal series of programs. Nucleic Acids Res  31: 3497–3500.  <ul class="find" data-citedArticleID="953186" data-doi="10.1093/nar/gkg500"><li><a href="http://dx.doi.org/10.1093/nar/gkg500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+sequence+alignment+with+the+Clustal+series+of+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+sequence+alignment+with+the+Clustal+series+of+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000462-Mayor1" id="pone.0000462-Mayor1"></a>Mayor C, Brudno M, Schwartz JR, Poliakov A, Rubin EM, et al.  (2000) VISTA: visualizing global DNA sequence alignments of arbitrary length. Bioinformatics  16: 1046–1047.  <ul class="find" data-citedArticleID="953248" data-doi="10.1093/bioinformatics/16.11.1046"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.11.1046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VISTA%3A+visualizing+global+DNA+sequence+alignments+of+arbitrary+length." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VISTA%3A+visualizing+global+DNA+sequence+alignments+of+arbitrary+length.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000462-Smit1" id="pone.0000462-Smit1"></a>Smit AFA, Hubley R, Green P (2004)  RepeatMasker Open - 3.0..   Available: <a href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</a>.  <ul class="find" data-citedArticleID="953288"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=RepeatMasker+Open+-+3.0.&amp;auth=&amp;atitle=RepeatMasker+Open+-+3.0." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RepeatMasker+Open+-+3.0." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RepeatMasker+Open+-+3.0.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000462-Felsenstein1" id="pone.0000462-Felsenstein1"></a>Felsenstein J (2005)  PHYLIP Phylogeny Inference Package, version 3.6. Department of Genome Sciences, University of Washington, Seattle..   Available: <a href="http://evolution.genetics.washington.edu/phylip.html">http://evolution.genetics.washington.edu​/phylip.html</a>.  <ul class="find" data-citedArticleID="953208"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=PHYLIP+Phylogeny+Inference+Package%2C+version+3.6.+Department+of+Genome+Sciences%2C+University+of+Washington%2C+Seattle.&amp;auth=&amp;atitle=PHYLIP+Phylogeny+Inference+Package%2C+version+3.6.+Department+of+Genome+Sciences%2C+University+of+Washington%2C+Seattle." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PHYLIP+Phylogeny+Inference+Package%2C+version+3.6.+Department+of+Genome+Sciences%2C+University+of+Washington%2C+Seattle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PHYLIP+Phylogeny+Inference+Package%2C+version+3.6.+Department+of+Genome+Sciences%2C+University+of+Washington%2C+Seattle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000462-Page1" id="pone.0000462-Page1"></a>Page R (1996) TREEVIEW: An application to display phylogenetic trees on personal computers. Computer Applications in the Biosciences  12: 357–358.  <ul class="find" data-citedArticleID="953266" data-doi="10.1093/bioinformatics/12.4.357"><li><a href="http://dx.doi.org/10.1093/bioinformatics/12.4.357" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TREEVIEW%3A+An+application+to+display+phylogenetic+trees+on+personal+computers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TREEVIEW%3A+An+application+to+display+phylogenetic+trees+on+personal+computers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000462-Kimura1" id="pone.0000462-Kimura1"></a>Kimura M (1983) The Neutral Theory of Molecular Evolution. Cambridge: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">60.
              </span><a name="pone.0000462-Nei1" id="pone.0000462-Nei1"></a>Nei M, Tajima F (1983) Maximum likelihood estimation of the number of nucleotide substitutions from restriction sites data. Genetics  105: 207–217.  <ul class="find" data-citedArticleID="953256"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maximum+likelihood+estimation+of+the+number+of+nucleotide+substitutions+from+restriction+sites+data.&amp;auth=&amp;atitle=Maximum+likelihood+estimation+of+the+number+of+nucleotide+substitutions+from+restriction+sites+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+likelihood+estimation+of+the+number+of+nucleotide+substitutions+from+restriction+sites+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+likelihood+estimation+of+the+number+of+nucleotide+substitutions+from+restriction+sites+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000462-Nei2" id="pone.0000462-Nei2"></a>Nei M, Li WH (1979) Mathematical model for studying genetic variation in terms of restriction endonucleases. Proc Natl Acad Sci USA  76: 5269–5273.  <ul class="find" data-citedArticleID="953258" data-doi="10.1073/pnas.76.10.5269"><li><a href="http://dx.doi.org/10.1073/pnas.76.10.5269" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mathematical+model+for+studying+genetic+variation+in+terms+of+restriction+endonucleases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mathematical+model+for+studying+genetic+variation+in+terms+of+restriction+endonucleases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000462-Watterson1" id="pone.0000462-Watterson1"></a>Watterson GA (1975) On the number of segregating sites in genetical models without recombination. Theor Popul Biol  7: 256–276.  <ul class="find" data-citedArticleID="953306" data-doi="10.1016/0040-5809(75)90020-9"><li><a href="http://dx.doi.org/10.1016/0040-5809(75)90020-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+number+of+segregating+sites+in+genetical+models+without+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+number+of+segregating+sites+in+genetical+models+without+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000462-Fay1" id="pone.0000462-Fay1"></a>Fay JC, Wu CI (2000) Hitchhiking under positive Darwinian selection. Genetics  155: 1405–1413.  <ul class="find" data-citedArticleID="953206"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hitchhiking+under+positive+Darwinian+selection.&amp;auth=&amp;atitle=Hitchhiking+under+positive+Darwinian+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hitchhiking+under+positive+Darwinian+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hitchhiking+under+positive+Darwinian+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000462-Tajima1" id="pone.0000462-Tajima1"></a>Tajima F (1989) Statistical method for testing the neutral mutation hypothesis by DNA polymorphism. Genetics  123: 585–595.  <ul class="find" data-citedArticleID="953292"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism.&amp;auth=&amp;atitle=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000462-Fu1" id="pone.0000462-Fu1"></a>Fu YX, Li WH (1993) Statistical Tests of Neutrality of Mutations. Genetics  133: 693–709.  <ul class="find" data-citedArticleID="953214"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Statistical+Tests+of+Neutrality+of+Mutations.&amp;auth=&amp;atitle=Statistical+Tests+of+Neutrality+of+Mutations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+Tests+of+Neutrality+of+Mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+Tests+of+Neutrality+of+Mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000462-Hudson2" id="pone.0000462-Hudson2"></a>Hudson RR (2002) Generating samples under a Wright-Fisher neutral model of genetic variation. Bioinformatics  18: 337–338.  <ul class="find" data-citedArticleID="953230" data-doi="10.1093/bioinformatics/18.2.337"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.2.337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generating+samples+under+a+Wright-Fisher+neutral+model+of+genetic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generating+samples+under+a+Wright-Fisher+neutral+model+of+genetic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000462-Rozas1" id="pone.0000462-Rozas1"></a>Rozas J, Sanchez-DelBarrio JC, Messeguer X, Rozas R (2003) DnaSP, DNA polymorphism analyses by the coalescent and other methods. Bioinformatics  19: 2496–2497.  <ul class="find" data-citedArticleID="953276" data-doi="10.1093/bioinformatics/btg359"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000462-Bandelt1" id="pone.0000462-Bandelt1"></a>Bandelt HJ, Forster P, Rohl A (1999) Median-joining networks for inferring intraspecific phylogenies. Mol Biol Evol  16: 37–48.  <ul class="find" data-citedArticleID="953180" data-doi="10.1093/oxfordjournals.molbev.a026036"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Median-joining+networks+for+inferring+intraspecific+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Median-joining+networks+for+inferring+intraspecific+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000462-Greenfield1" id="pone.0000462-Greenfield1"></a>Greenfield LJ Jr, Zaman SH, Sutherland ML, Lummis SC, Niemeyer MI, et al.  (2002) Mutation of the GABA<sub>A</sub> receptor M1 transmembrane proline increases GABA affinity and reduces barbiturate enhancement. Neuropharmacology  42: 502–521.  <ul class="find" data-citedArticleID="953222" data-doi="10.1016/s0028-3908(01)00196-4"><li><a href="http://dx.doi.org/10.1016/s0028-3908(01)00196-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+of+the+GABAA+receptor+M1+transmembrane+proline+increases+GABA+affinity+and+reduces+barbiturate+enhancement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+of+the+GABAA+receptor+M1+transmembrane+proline+increases+GABA+affinity+and+reduces+barbiturate+enhancement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000462-Huang1" id="pone.0000462-Huang1"></a>Huang RQ, Dillon GH (1998) Maintenance of recombinant type A γ-aminobutyric acid receptor function: role of protein tyrosine phosphorylation and calcineurin. J Pharmacol Exp Ther  286: 243–255.  <ul class="find" data-citedArticleID="953226"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maintenance+of+recombinant+type+A+%26%23x3b3%3B-aminobutyric+acid+receptor+function%3A+role+of+protein+tyrosine+phosphorylation+and+calcineurin.&amp;auth=&amp;atitle=Maintenance+of+recombinant+type+A+%26%23x3b3%3B-aminobutyric+acid+receptor+function%3A+role+of+protein+tyrosine+phosphorylation+and+calcineurin." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maintenance+of+recombinant+type+A+%26%23x3b3%3B-aminobutyric+acid+receptor+function%3A+role+of+protein+tyrosine+phosphorylation+and+calcineurin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maintenance+of+recombinant+type+A+%26%23x3b3%3B-aminobutyric+acid+receptor+function%3A+role+of+protein+tyrosine+phosphorylation+and+calcineurin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.XML" value="138451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.PDF" value="463755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g001.PNG_L" value="1278634"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g001.PNG_M" value="105321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g001.PNG_S" value="4119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g001.TIF" value="1665450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g001.PNG_I" value="57812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g002.PNG_L" value="158559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g002.PNG_M" value="32829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g002.PNG_S" value="2801"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g002.TIF" value="513542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g002.PNG_I" value="34680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g003.PNG_L" value="97604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g003.PNG_M" value="34713"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g003.PNG_S" value="2471"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g003.TIF" value="446240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g003.PNG_I" value="19287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g004.PNG_L" value="416425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g004.PNG_M" value="67768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g004.PNG_S" value="2663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g004.TIF" value="1000522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g004.PNG_I" value="37675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g005.PNG_L" value="166909"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g005.PNG_M" value="20298"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g005.PNG_S" value="2340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g005.TIF" value="502734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g005.PNG_I" value="13932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g006.PNG_L" value="222753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g006.PNG_M" value="23868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g006.PNG_S" value="2367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g006.TIF" value="447208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.g006.PNG_I" value="12701"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.s001.XLS" value="27648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.s002.XLS" value="24064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.s003.XLS" value="57856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.s004.XLS" value="39936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.s005.XLS" value="21504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.s006.XLS" value="30720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.s007.XLS" value="15360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.s008.TIF" value="584021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.s009.TIF" value="957991"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.s010.TIF" value="739680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000462.s011.TIF" value="745491"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000462&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000462" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000462&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000462&volume=&issue=&title=Positive Selection within the Schizophrenia-Associated GABAA Receptor β2 Gene&author_name=Wing-Sze%20Lo%2C%20Zhiwen%20Xu%2C%20Zhiliang%20Yu%2C%20Frank%20W.%20Pun%2C%20Siu-Kin%20Ng%2C%20Jianhuan%20Chen%2C%20Ka-Lok%20Tong%2C%20Cunyou%20Zhao%2C%20Xiaojing%20Xu%2C%20Shui-Ying%20Tsang%2C%20Mutsuo%20Harano%2C%20Gerald%20St%C3%B6ber%2C%20Vishwajit%20L.%20Nimgaonkar%2C%20Hong%20Xue&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000462" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000462" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000462" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000462&amp;t=Positive%20Selection%20within%20the%20Schizophrenia-Associated%20GABAA%20Receptor%20%CE%B22%20Gene" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000462&title=Positive%20Selection%20within%20the%20Schizophrenia-Associated%20GABAA%20Receptor%20%CE%B22%20Gene&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000462&amp;title=Positive%20Selection%20within%20the%20Schizophrenia-Associated%20GABAA%20Receptor%20%CE%B22%20Gene" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000462" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000462&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Positive Selection within the Schizophrenia-Associated GABAA Receptor β2 Gene';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000462';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Positive%20Selection%20within%20the%20Schizophrenia-Associated%20GABAA%20Receptor%20%CE%B22%20Gene http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000462" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000462" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="23762"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23762"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23762"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplotypes%22" title="Search for articles in the subject area:'Haplotypes'"><div class="flagText">Haplotypes</div></a>
              <div data-categoryid="19681" data-articleid="23762"
                   data-categoryname="Haplotypes"
                   class="flagImage" title="Flag 'Haplotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="23762"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="23762"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="23762"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Schizophrenia%22" title="Search for articles in the subject area:'Schizophrenia'"><div class="flagText">Schizophrenia</div></a>
              <div data-categoryid="18371" data-articleid="23762"
                   data-categoryname="Schizophrenia"
                   class="flagImage" title="Flag 'Schizophrenia' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9252'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5343'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1140&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
