

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Molecular Variation at a Candidate Gene Implicated in the Regulation of Fire Ant Social Behavior</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001088"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001088" />

    <meta name="citation_title" content="Molecular Variation at a Candidate Gene Implicated in the Regulation of Fire Ant Social Behavior"/>
    <meta itemprop="name" content="Molecular Variation at a Candidate Gene Implicated in the Regulation of Fire Ant Social Behavior"/>

      <meta name="citation_author" content="Dietrich Gotzek"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolution, University of Lausanne, Lausanne, Switzerland"/>
      <meta name="citation_author" content="D. DeWayne Shoemaker"/>
            <meta name="citation_author_institution" content="United States Department of Agriculture–Agricultural Research Service (USDA-ARS) Center for Medical, Agricultural, and Veterinary Entomology, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Kenneth G. Ross"/>
            <meta name="citation_author_institution" content="Department of Entomology, University of Georgia, Athens, Georgia, United States of America"/>

    <meta name="citation_date" content="2007/11/7"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001088.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1088"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The causes of molecular evolution.; citation_author=JH Gillespie; citation_number=1; citation_pages=352; citation_date=1991; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Genetics of populations, 3rd edition.; citation_author=PW Hedrick; citation_number=2; citation_pages=737; citation_date=2004; citation_publisher=Jones and Bartlett; " />
      <meta name="citation_reference" content="citation_title=The structural basis of molecular adaptation.; citation_author=GB Golding; citation_author=AM Dean; citation_journal_title=Mol Biol Evol; citation_volume=15; citation_number=3; citation_pages=355-369; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=From DNA to fitness differences: sequences and structures of adaptive variants of Colias phosphoglucose isomerase (PGI).; citation_author=CW Wheat; citation_author=WB Watt; citation_author=DD Pollock; citation_author=PM Schulte; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=4; citation_pages=499-512; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The genetic basis of adaptation: lessons from concealing coloration in pocket mice.; citation_author=MW Nachman; citation_journal_title=Genetica; citation_volume=123; citation_number=5; citation_pages=125-136; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Testing hypotheses regarding the genetics of adaptation.; citation_author=PC Phillips; citation_journal_title=Genetica; citation_volume=123; citation_number=6; citation_pages=15-24; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Challenges for identifying functionally important genetic variation: the promise of combining complementary research strategies.; citation_author=A Vasemägi; citation_author=CR Primmer; citation_journal_title=Mol Ecol; citation_volume=14; citation_number=7; citation_pages=3623-3642; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multilocus evolution in fire ants: effects of selection, gene flow, and recombination.; citation_author=KG Ross; citation_journal_title=Genetics; citation_volume=145; citation_number=8; citation_pages=961-974; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genetic control of social organization in an ant.; citation_author=KG Ross; citation_author=L Keller; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=9; citation_pages=14232-14237; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Identification of a major gene regulating complex social behavior.; citation_author=MJB Krieger; citation_author=KG Ross; citation_journal_title=Science; citation_volume=295; citation_number=10; citation_pages=328-332; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular evolutionary analyses of the odorant-binding protein gene Gp-9 in fire ants and other Solenopsis species.; citation_author=MJB Krieger; citation_author=KG Ross; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=11; citation_pages=2090-2103; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Experimental conversion of colony social organization by manipulation of worker genotype composition in fire ants (Solenopsis invicta).; citation_author=KG Ross; citation_author=L Keller; citation_journal_title=Behav Ecol Sociobiol; citation_volume=51; citation_number=12; citation_pages=287-295; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Experimental conversion of colony social organization in fire ants (Solenopsis invicta): worker genotype manipulation in the absence of queen effects.; citation_author=D Gotzek; citation_author=KG Ross; citation_number=13; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Social evolution in ants.; citation_author=AFG Bourke; citation_author=NR Franks; citation_number=14; citation_pages=550; citation_date=1995; citation_publisher=Princeton University Press; " />
      <meta name="citation_reference" content="citation_title=Ecology and evolution of social organization: insights from fire ants and other highly eusocial insects.; citation_author=KG Ross; citation_author=L Keller; citation_journal_title=Annu Rev Ecol Syst; citation_volume=26; citation_number=15; citation_pages=631-656; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The fire ants.; citation_author=WR Tschinkel; citation_number=16; citation_pages=752; citation_date=2006; citation_publisher=Harvard University Press; " />
      <meta name="citation_reference" content="citation_title=Potential cause of lethality of an allele implicated in social evolution in fire ants.; citation_author=BL Hallar; citation_author=MJB Krieger; citation_author=KG Ross; citation_journal_title=Genetica; citation_volume=131; citation_number=17; citation_pages=69-79; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Molecular basis of pheromone detection in insects.; citation_author=RG Vogt; citation_number=18; citation_pages=753-804; citation_date=2005; citation_publisher=Elsevier; " />
      <meta name="citation_reference" content="citation_title=Selfish genes: a green beard in the red fire ant.; citation_author=L Keller; citation_author=KG Ross; citation_journal_title=Nature; citation_volume=394; citation_number=19; citation_pages=573-575; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genetic regulation of colony social organization in fire ants: an integrative overview.; citation_author=D Gotzek; citation_author=KG Ross; citation_journal_title=Q Rev Biol; citation_volume=82; citation_number=20; citation_pages=201-226; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Alternative genetic foundations for a key social polymorphism in fire ants.; citation_author=KG Ross; citation_author=MJB Krieger; citation_author=DD Shoemaker; citation_journal_title=Genetics; citation_volume=165; citation_number=21; citation_pages=1853-1867; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The genomic rate of adaptive evolution.; citation_author=A Eyre-Walker; citation_journal_title=Trends Ecol Evol; citation_volume=21; citation_number=22; citation_pages=569-575; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3.1 manual.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_author=P van der Mark; citation_number=23; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cladistic analysis of the fire ants of the Solenopsis saevissima species-group (Hymenoptera: Formicidae).; citation_author=JP Pitts; citation_author=JV McHugh; citation_author=KG Ross; citation_journal_title=Zool Scripta; citation_volume=34; citation_number=24; citation_pages=493-505; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Variation in evolutionary processes at different codon positions.; citation_author=L Bofkin; citation_author=N Goldman; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=25; citation_pages=513-521; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Distribution of the two social forms of the fire ant Solenopsis invicta (Hymenoptera: Formicidae) in the native South American range.; citation_author=MC Mescher; citation_author=KG Ross; citation_author=DD Shoemaker; citation_author=L Keller; citation_author=MJB Krieger; citation_journal_title=Ann Ent Soc Amer; citation_volume=96; citation_number=26; citation_pages=810-817; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Not so different after all: a comparison of methods for detecting amino acid sites under selection.; citation_author=SL Kosakovsky Pond; citation_author=SWD Frost; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=27; citation_pages=1208-1222; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Small-sample tests of episodic adaptive evolution: a case study of primate lysozymes.; citation_author=J Zhang; citation_author=S Kumar; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=14; citation_number=28; citation_pages=1335-1338; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genetic variation and structure in native populations of the fire ant Solenopsis invicta: evolutionary and demographic implications.; citation_author=KG Ross; citation_author=MJB Krieger; citation_author=L Keller; citation_author=DD Shoemaker; citation_journal_title=Biol J Linn Soc.; citation_volume=volume 92; citation_number=29; citation_pages=541-560; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=To b or not to b: a pheromone-binding protein regulates colony social organization in fire ants.; citation_author=MJB Krieger; citation_journal_title=Bioessays; citation_volume=27; citation_number=30; citation_pages=91-99; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Biochemistry and diversity of insect odorant-binding proteins.; citation_author=P Nagnan-Le Meillour; citation_author=E Jacquin-Joly; citation_number=31; citation_pages=509-537; citation_date=2003; citation_publisher=Elsevier; " />
      <meta name="citation_reference" content="citation_title=Why the rate of silent codon substitutions is variable within a vertebrate's genome.; citation_author=J Filipski; citation_journal_title=J Theor Biol; citation_volume=134; citation_number=32; citation_pages=159-164; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Patterns of intron sequence evolution in Drosophila are dependent upon length and GC content.; citation_author=PR Haddrill; citation_author=B Charlesworth; citation_author=DL Halligan; citation_author=P Andolfatto; citation_journal_title=Genome Biol; citation_volume=6; citation_number=33; citation_pages=R67; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Strong regional heterogeneity in base composition evolution on the Drosophila X chromosome.; citation_author=W-Y Ko; citation_author=S Piao; citation_author=H Akashi; citation_journal_title=Genetics; citation_volume=174; citation_number=34; citation_pages=349-362; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Divergence of conserved non-coding sequences: rate estimates and relative rate tests.; citation_author=GP Wagner; citation_author=C Fried; citation_author=SJ Prohaska; citation_author=PF Stadler; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=35; citation_pages=2116-2121; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of non-coding DNA in Drosophila.; citation_author=P Andolfatto; citation_journal_title=Nature; citation_volume=437; citation_number=36; citation_pages=1149-1152; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Function and evolution of a gene family encoding odorant binding-like proteins in a social insect, the honey bee (Apis mellifera).; citation_author=S Forêt; citation_author=R Maleszka; citation_journal_title=Genome Res; citation_volume=16; citation_number=37; citation_pages=1404-1413; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Revision of the fire ants of the Solenopsis saevissima species-group (Hymenoptera: Formicidae).; citation_author=JP Pitts; citation_author=JV McHugh; citation_author=KG Ross; citation_number=38; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The shapes of neutral gene genealogies in two species: probabilities of monophyly, paraphyly, and polyphyly in a coalescent model.; citation_author=NA Rosenberg; citation_journal_title=Evolution; citation_volume=57; citation_number=39; citation_pages=1465-1477; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of fire ants of the Solenopsis saevissima species-group based on mtDNA sequences.; citation_author=DD Shoemaker; citation_author=ME Ahrens; citation_author=KG Ross; citation_journal_title=Mol Phyl Evol; citation_volume=38; citation_number=40; citation_pages=200-215; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolutionary history of Wolbachia infections in the fire ant Solenopsis invicta.; citation_author=M Ahrens; citation_author=DD Shoemaker; citation_journal_title=BMC Evol Biol; citation_volume=5; citation_number=41; citation_pages=35; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Behavioral genetics: a gene for supersociality.; citation_author=L Keller; citation_author=JD Parker; citation_journal_title=Curr Biol; citation_volume=12; citation_number=42; citation_pages=180-181; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Species delimitation in native South American fire ants.; citation_author=KG Ross; citation_author=DD Shoemaker; citation_journal_title=Mol Ecol; citation_volume=14; citation_number=43; citation_pages=3419-3438; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Hierarchical analysis of genetic structure in native fire ant populations: results from three classes of molecular markers.; citation_author=KG Ross; citation_author=MJB Krieger; citation_author=DD Shoemaker; citation_author=EL Vargo; citation_author=L Keller; citation_journal_title=Genetics; citation_volume=147; citation_number=44; citation_pages=643-655; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Improved reliability of allele-specific PCR.; citation_author=EN Imyanitov; citation_author=KG Buslov; citation_author=EN Suspitsin; citation_author=ES Kuligina; citation_author=EV Belogubova; citation_journal_title=Biotechniques; citation_volume=33; citation_number=45; citation_pages=484-490; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Polymorphism at dinucleotide microsatellite loci in fire ant Solenopsis invicta populations.; citation_author=MJB Krieger; citation_author=L Keller; citation_journal_title=Mol Ecol; citation_volume=6; citation_number=46; citation_pages=997-999; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Relatedness among co-existing queens within polygyne colonies of a Texas population of the fire ant, Solenopsis invicta.; citation_author=YP Chen; citation_author=LY Lu; citation_author=LC Skow; citation_author=SB Vinson; citation_journal_title=Southwest Ent; citation_volume=28; citation_number=47; citation_pages=27-36; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Population genetics of the invasive fire ant Solenopsis invicta (Hymenoptera: Formicidae) in the United States.; citation_author=DD Shoemaker; citation_author=CJ DeHeer; citation_author=MJB Krieger; citation_author=KG Ross; citation_journal_title=Ann Ent Soc Amer; citation_volume=99; citation_number=48; citation_pages=1213-1233; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Effect of a founder event on variation in the genetic sex-determining system of the fire ant Solenopsis invicta.; citation_author=KG Ross; citation_author=EL Vargo; citation_author=L Keller; citation_author=JC Trager; citation_journal_title=Genetics; citation_volume=135; citation_number=49; citation_pages=843-854; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=TOPALi: software for automatic identification of recombinant sequences within DNA multiple alignments.; citation_author=I Milne; citation_author=F Wright; citation_author=G Rowe; citation_author=DF Marshal; citation_author=D Husmeier; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=50; citation_pages=1806-1807; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PAUP*; Phylogenetic analysis using parsimony (*and other methods). Version 4.; citation_author=DL Swofford; citation_number=51; citation_date=2003; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=SeqVis: Visualization of compositional heterogeneity in large alignments of nucleotides.; citation_author=JWK Ho; citation_author=CE Adams; citation_author=JB Lew; citation_author=TJ Matthews; citation_author=CC Ng; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=52; citation_pages=2162-2163; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNASP, DNA polymorphism analyses by the coalescent and other methods.; citation_author=J Rozas; citation_author=JC Sanchez-DelBarrio; citation_author=X Messeguer; citation_author=R Rozas; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=53; citation_pages=2496-2497; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=DAMBE: software package for data analysis in molecular biology and evolution.; citation_author=X Xia; citation_author=Z Xie; citation_journal_title=J Hered; citation_volume=92; citation_number=54; citation_pages=371-373; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MEGA 3: Integrated software for molecular evolutionary genetics analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Brief Bioinf; citation_volume=5; citation_number=55; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution and phylogenetics.; citation_author=M Nei; citation_author=S Kumar; citation_number=56; citation_pages=333; citation_date=2000; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=MESQUITE: a modular system for evolutionary analysis.; citation_author=WP Maddison; citation_author=DR Maddison; citation_number=57; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sequence logos: a new way to display consensus sequences.; citation_author=TD Schneider; citation_author=RM Stephens; citation_journal_title=Nuc Acids Res; citation_volume=18; citation_number=58; citation_pages=6097-6100; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=WebLogo: A sequence logo generator.; citation_author=GE Crooks; citation_author=G Hon; citation_author=JM Chandonia; citation_author=SE Brenner; citation_journal_title=Genome Res; citation_volume=14; citation_number=59; citation_pages=1188-1190; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CINEMA–A novel colour interactive editor for multiple alignments.; citation_author=DJ Parry-Smith; citation_author=AWR Payne; citation_author=AD Michie; citation_author=TK Attwood; citation_journal_title=Gene; citation_volume=221; citation_number=60; citation_pages=GC57-GC63; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=MacClade 4: Analysis of phylogeny and character evolution, CD-ROM ed.; citation_author=DR Maddison; citation_author=WP Maddison; citation_number=61; citation_date=2000; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=The neighbor-joining method; a new method for reconstructing phylogenetic trees.; citation_author=N Saitou; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=4; citation_number=62; citation_pages=406-425; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Recovering evolutionary trees under a more realistic model of sequence evolution.; citation_author=PJ Lockhart; citation_author=MA Steel; citation_author=MD Hendy; citation_author=D Penny; citation_journal_title=Mol Biol Evol; citation_volume=11; citation_number=63; citation_pages=605-612; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Multiple comparisons of log-likelihoods with applications to phylogenetic inference.; citation_author=H Shimodaira; citation_author=M Hasegawa; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=64; citation_pages=1114-1116; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The Parsimony Ratchet, a new method for rapid parsimony analysis.; citation_author=KC Nixon; citation_journal_title=Cladistics; citation_volume=15; citation_number=65; citation_pages=407-414; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Software manual for PAUPRat: A tool to implement parsimony ratchet searches using PAUP*.; citation_author=DS Sikes; citation_author=PO Lewis; citation_number=66; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MODELTEST: testing the model of DNA substitution.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=67; citation_pages=817-818; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=New look at statistical model identification.; citation_author=H Akaike; citation_journal_title=IEEE Trans Auto Contr; citation_volume=19; citation_number=68; citation_pages=716-723; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Estimating the dimension of a model.; citation_author=G Schwarz; citation_journal_title=Ann Stat; citation_volume=6; citation_number=69; citation_pages=461-464; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Model selection and model averaging in phylogenetics: advantages of Aikake information criterion and Bayesian approaches over likelihood ratio tests.; citation_author=D Posada; citation_author=TR Buckley; citation_journal_title=Syst Biol; citation_volume=53; citation_number=70; citation_pages=793-808; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MRBAYES: Bayesian inference of phylogenetic trees.; citation_author=JP Huelsenbeck; citation_author=F Ronquist; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=71; citation_pages=754-755; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Introduction to Markov chain Monte Carlo methods in molecular evolution.; citation_author=B Larget; citation_number=72; citation_date=2005; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Likelihood models for detecting postively selected amino acid sites and applications to the HIV-1 envelope genes.; citation_author=R Nielsen; citation_author=Z Yang; citation_journal_title=Genetics; citation_volume=148; citation_number=73; citation_pages=929-936; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Bayesian estimation of positively selected sites.; citation_author=JP Huelsenbeck; citation_author=KA Dyer; citation_journal_title=J Mol Evol; citation_volume=58; citation_number=74; citation_pages=661-672; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Codon-substitution models for heterogeneous selection pressure at amino acid sites.; citation_author=Z Yang; citation_author=R Nielsen; citation_author=N Goldman; citation_author=A-M Krabbe Pedersen; citation_journal_title=Genetics; citation_volume=155; citation_number=75; citation_pages=431-449; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Codon-substitution models for detecting molecular adaptation at individual sites along specified lineages.; citation_author=Z Yang; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=76; citation_pages=908-917; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level.; citation_author=J Zhang; citation_author=R Nielsen; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=77; citation_pages=2472-2479; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Appl Biosci; citation_volume=13; citation_number=78; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Bayes empirical bayes inference of amino acid sites under positive selection.; citation_author=Z Yang; citation_author=WSW Wong; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=79; citation_pages=1107-1118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A Bayesian model comparison approach to inferring positive selection.; citation_author=K Scheffler; citation_author=C Seoighe; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=80; citation_pages=2531-2540; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A likelihood approach for comparing synonymous and nonsynonymous nucleotide substitution rates, with application to the chloroplast genome.; citation_author=SV Muse; citation_author=BS Gaut; citation_journal_title=Mol Biol Evol; citation_volume=11; citation_number=81; citation_pages=715-724; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A method for detecting positive selection at single amino acid sites.; citation_author=Y Suzuki; citation_author=T Gojobori; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=82; citation_pages=1315-1328; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=DATAMONKEY: rapid detection of selective pressure on individual sites of codon alignments.; citation_author=SL Kosakovsky Pond; citation_author=SWD Frost; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=83; citation_pages=2531-2533; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Episodic adaptive evolution of primate lysozymes.; citation_author=W Messier; citation_author=C-B Stewart; citation_journal_title=Nature; citation_volume=385; citation_number=84; citation_pages=151-154; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Rapid evolution of a primate sperm protein: relaxation of functional constraint or positive Darwinian selection?; citation_author=AP Rooney; citation_author=J Zhang; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=85; citation_pages=706-710; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Computational molecular evolution.; citation_author=Z Yang; citation_number=86; citation_pages=357; citation_date=2006; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data.; citation_author=L Excoffier; citation_author=PE Smouse; citation_author=JM Quattro; citation_journal_title=Genetics; citation_volume=131; citation_number=87; citation_pages=479-491; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Arlequin ver. 3.0: an integrated software package for population genetics data analysis.; citation_author=L Excoffier; citation_author=G Laval; citation_author=S Schneider; citation_journal_title=Evol Bioinf Online; citation_volume=1; citation_number=88; citation_pages=47-50; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=GENEPOP (version 1.2): population genetics software for exact tests and ecumenicism.; citation_author=M Raymond; citation_author=F Rousset; citation_journal_title=J Hered; citation_volume=86; citation_number=89; citation_pages=248-249; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mathematical model for studying genetic variation in terms of restriction endonucleases.; citation_author=M Nei; citation_author=WH Li; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=76; citation_number=90; citation_pages=5269-5273; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Isolation by distance in equilibrium and non-equilibrium populations.; citation_author=M Slatkin; citation_journal_title=Evolution; citation_volume=47; citation_number=91; citation_pages=264-279; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Genetic differentiation and estimation of gene flow from F-statistics under isolation by distance.; citation_author=F Rousset; citation_journal_title=Genetics; citation_volume=145; citation_number=92; citation_pages=1219-1228; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001088" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Molecular Variation at a Candidate Gene Implicated in the Regulation of Fire Ant Social Behavior"/>
    <meta name="twitter:description" content="The fire ant Solenopsis invicta and its close relatives display an important social polymorphism involving differences in colony queen number. Colonies are headed by either a single reproductive queen (monogyne form) or multiple queens (polygyne form). This variation in social organization is associated with variation at the gene Gp-9, with monogyne colonies harboring only B-like allelic variants and polygyne colonies always containing b-like variants as well. We describe naturally occurring variation at Gp-9 in fire ants based on 185 full-length sequences, 136 of which were obtained from S. invicta collected over much of its native range. While there is little overall differentiation between most of the numerous alleles observed, a surprising amount is found in the coding regions of the gene, with such substitutions usually causing amino acid replacements. This elevated coding-region variation may result from a lack of negative selection acting to constrain amino acid replacements over much of the protein, different mutation rates or biases in coding and non-coding sequences, negative selection acting with greater strength on non-coding than coding regions, and/or positive selection acting on the protein. Formal selection analyses provide evidence that the latter force played an important role in the basal b-like lineages coincident with the emergence of polygyny. While our data set reveals considerable paraphyly and polyphyly of S. invicta sequences with respect to those of other fire ant species, the b-like alleles of the socially polymorphic species are monophyletic. An expanded analysis of colonies containing alleles of this clade confirmed the invariant link between their presence and expression of polygyny. Finally, our discovery of several unique alleles bearing various combinations of b-like and B-like codons allows us to conclude that no single b-like residue is completely predictive of polygyne behavior and, thus, potentially causally involved in its expression. Rather, all three typical b-like residues appear to be necessary."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001088.g006"/>

  <meta property="og:title" content="Molecular Variation at a Candidate Gene Implicated in the Regulation of Fire Ant Social Behavior" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001088" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2185'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7121'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1329&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001088">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001088" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001088&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Molecular Variation at a Candidate Gene Implicated in the Regulation of Fire Ant Social Behavior
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dietrich Gotzek, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Ecology and Evolution, University of Lausanne, Lausanne, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            D. DeWayne Shoemaker, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  United States Department of Agriculture–Agricultural Research Service (USDA-ARS) Center for Medical, Agricultural, and Veterinary Entomology, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kenneth G. Ross
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kenross@uga.edu">kenross@uga.edu</a></p>

                <p>Affiliation:
                  Department of Entomology, University of Georgia, Athens, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 07, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001088</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001088" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001088" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001088" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001088" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001088-g001" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001088-g002" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001088-g003" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001088-g004" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001088-t001" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001088-t002" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001088-g005" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001088-t003" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001088-t004" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001088-g006" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001088">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001088" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The fire ant <em>Solenopsis invicta</em> and its close relatives display an important social polymorphism involving differences in colony queen number. Colonies are headed by either a single reproductive queen (monogyne form) or multiple queens (polygyne form). This variation in social organization is associated with variation at the gene <em>Gp-9</em>, with monogyne colonies harboring only <em>B</em>-like allelic variants and polygyne colonies always containing <em>b</em>-like variants as well. We describe naturally occurring variation at <em>Gp-9</em> in fire ants based on 185 full-length sequences, 136 of which were obtained from <em>S. invicta</em> collected over much of its native range. While there is little overall differentiation between most of the numerous alleles observed, a surprising amount is found in the coding regions of the gene, with such substitutions usually causing amino acid replacements. This elevated coding-region variation may result from a lack of negative selection acting to constrain amino acid replacements over much of the protein, different mutation rates or biases in coding and non-coding sequences, negative selection acting with greater strength on non-coding than coding regions, and/or positive selection acting on the protein. Formal selection analyses provide evidence that the latter force played an important role in the basal <em>b</em>-like lineages coincident with the emergence of polygyny. While our data set reveals considerable paraphyly and polyphyly of <em>S. invicta</em> sequences with respect to those of other fire ant species, the <em>b</em>-like alleles of the socially polymorphic species are monophyletic. An expanded analysis of colonies containing alleles of this clade confirmed the invariant link between their presence and expression of polygyny. Finally, our discovery of several unique alleles bearing various combinations of <em>b</em>-like and <em>B</em>-like codons allows us to conclude that no single <em>b</em>-like residue is completely predictive of polygyne behavior and, thus, potentially causally involved in its expression. Rather, all three typical <em>b</em>-like residues appear to be necessary.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Gotzek D, Shoemaker DD, Ross KG (2007) Molecular Variation at a Candidate Gene Implicated in the Regulation of Fire Ant Social Behavior. PLoS ONE 2(11):
          e1088.
            doi:10.1371/journal.pone.0001088</p><p><strong>Academic Editor: </strong>Stuart West, University of Edinburgh, United Kingdom</p><p><strong>Received:</strong> June 5, 2007; <strong>Accepted:</strong> October 5, 2007; <strong>Published:</strong> November 7, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>Partial funding for this research came from the Georgia Agricultural Experiment Stations (University of Georgia) and grants from the United States Department of Agriculture to DDS and KGR.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A main goal of evolutionary genetics is to document naturally occurring variation and to reconcile observed patterns with population history and demography, fitness consequences, and selection regimes at genes of interest <a href="#pone.0001088-Gillespie1">[1]</a>, <a href="#pone.0001088-Hedrick1">[2]</a>. Study of the adaptive maintenance of molecular variation historically has followed one of two approaches, elucidation of the functional components of molecular adaptations at the biochemical level, or description of the historical footprints of selection acting on sequence variants <a href="#pone.0001088-Golding1">[3]</a>. An important objective in modern studies of molecular adaptation is to bridge the two approaches by means of comprehensive research integrating functional data with information on patterns of variation that implicate past selection <a href="#pone.0001088-Wheat1">[4]</a> (see also <a href="#pone.0001088-Golding1">[3]</a>, <a href="#pone.0001088-Nachman1">[5]</a>–<a href="#pone.0001088-Vasemgi1">[7]</a>).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The fire ant <em>Solenopsis invicta</em> displays an important colony-level social polymorphism that is associated with variation at a single gene, <em>general protein-9</em> (<em>Gp-9</em>) <a href="#pone.0001088-Ross1">[8]</a>. Colonies with a single reproductive queen (monogyne colonies) always feature the exclusive presence of <em>B</em>-like alleles of <em>Gp-9</em> in all colony members. In contrast, colonies with multiple reproductive queens (polygyne colonies) always have an alternate class of alleles, designated <em>b</em>-like alleles, represented along with <em>B</em>-like alleles among colony members <a href="#pone.0001088-Ross1">[8]</a>–<a href="#pone.0001088-Krieger2">[11]</a>. This pattern, coupled with similar genetic compositions of monogyne and polygyne populations at numerous other nuclear genes, has led to the hypothesis that the presence of <em>b</em>-like alleles in a colony's workers is both necessary and sufficient to elicit polygyne social behavior <a href="#pone.0001088-Ross1">[8]</a>–<a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Ross3">[12]</a>, <a href="#pone.0001088-Gotzek1">[13]</a>. Because variation in queen number represents a fundamental social polymorphism that is associated with a suit of important reproductive, demographic, and life history differences <a href="#pone.0001088-Bourke1">[14]</a>–<a href="#pone.0001088-Tschinkel1">[16]</a>, variation at <em>Gp-9</em> is hypothesized to underlie the expression of major alternative adaptive syndromes in <em>S. invicta</em>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Our understanding of the association of variation at <em>Gp-9</em> with colony social organization has been advanced by the production of sequence data for <em>S. invicta</em> in its native (South American) and introduced (USA) ranges as well as for numerous other <em>Solenopsis</em> species <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a>. These studies revealed several important patterns. First, the monophyletic <em>b</em>-like alleles are restricted to a clade of six South American fire ant species that display the monogyne-polygyne polymorphism (this group of species, which includes <em>S. invicta</em>, is informally termed the socially polymorphic clade). Second, polygyne colonies in all of the socially polymorphic species always contain <em>b</em>-like alleles. Third, the <em>b</em>-like alleles of the socially polymorphic species bear diagnostic amino acid residues at positions 42, 95, and 139 that distinguish them from all other <em>Gp-9</em> alleles (collectively known as <em>B</em>-like alleles). This finding prompted speculation that the substitutions at one or more of these positions may alter the function of GP-9 protein with respect to its proposed role in modulating social behavior <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a>. Finally, the <em>b</em> alleles, which comprise a small clade of <em>b</em>-like alleles that apparently arose recently in <em>S. invicta</em>, feature a radical, charge-changing substitution (Glu151Lys) that may underlie their observed deleterious effects in homozygous condition <a href="#pone.0001088-Hallar1">[17]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>GP-9 protein is a member of the insect odorant-binding protein family <a href="#pone.0001088-Krieger1">[10]</a>. Several well studied proteins in this family have been implicated as important molecular components of chemoreception in insects, presumably effecting the transduction of pheromones or food chemostimulants to neuronal signals by transporting these ligands through the chemosensillar lymph to neuronal receptors <a href="#pone.0001088-Vogt1">[18]</a>. Regulation of colony queen number in fire ants involves reciprocal chemical signalling and perception between workers and queens, with workers ultimately making decisions about which queens, and how many, are tolerated as colony reproductives based on queen pheromonal signatures <a href="#pone.0001088-Ross2">[9]</a>, <a href="#pone.0001088-Keller1">[19]</a>. The role of some odorant-binding proteins in chemoreception, the invariant association of one class of <em>Gp-9</em> alleles with polygyny, and the restriction of this class of alleles to the socially polymorphic clade of South American fire ants have been viewed as evidence that <em>Gp-9</em> may directly influence social organization rather than merely being a marker for other genes of major effect on this trait (reviewed in <a href="#pone.0001088-Gotzek2">[20]</a>).</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Further complexities in our understanding of <em>Gp-9</em> and fire ant social evolution have arisen as additional sequence data have been generated. For instance, it is now apparent that variation at this gene is not invariably associated with colony social organization in the genus <em>Solenopsis</em>, given that the fire ant <em>S. geminata</em>, a distant North American relative of <em>S. invicta</em>, does not exhibit allelic variation associated with colony social form <a href="#pone.0001088-Ross5">[21]</a>. Also, discovery of a <em>Gp-9</em> allele with a <em>b</em>-like amino acid residue at position 95 but <em>B</em>-like residues at positions 42 and 139 in the undescribed <em>S.</em> species “X” indicates that the two classes of alleles in the socially polymorphic species are not as internally homogeneous as previously believed <a href="#pone.0001088-Krieger2">[11]</a>. Unfortunately, the social organization of the source colony for this sequence could not be determined, precluding a test of the importance of residue 95 in regulating queen number. These recent results suggest that progress in our understanding of <em>Gp-9</em> in fire ants has been hampered by limited knowledge of naturally occurring sequence variation; in even the best studied species, <em>S. invicta</em>, only a handful of individuals from a single site in the native range have been sequenced to date.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>To remedy this shortcoming, the present study documents sequence variation at <em>Gp-9</em> using extensive samples of <em>S. invicta</em> collected over a large portion of its vast native range as well as samples of many of its fire ant relatives. Few previous studies have employed such large collections of intraspecific sequence variants in order to analyze patterns of adaptive molecular variation at single genes <a href="#pone.0001088-EyreWalker1">[22]</a>. Our specific objectives were to characterize the molecular evolution of <em>Gp-9</em> in fire ants, to test for effects of selection on the gene, to confirm the association between polygyny and the presence of variants encoding <em>b</em>-like amino acid residues, and to test for phylogeographic patterns in the distribution of the observed variation in <em>S. invicta</em>. We were particularly interested in finding new variants encoding unique combinations of <em>B</em>-like and <em>b</em>-like residues at codons 42, 95, and 139, with the hope that their discovery might shed light on the role of each substitution in mediating the expression of social organization. In combination with the other analyses, information from such variant colonies is expected to aid progress in connecting the genetic and phenotypic variation underlying regulation of fire ant social behavior.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>General results</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The complete data set consisted of 185 full-length <em>Gp-9</em> sequences (149 newly generated), of which 136 were from the focal species, <em>S. invicta</em> (sampling sites for this species are shown in <a href="#pone-0001088-g001">Figure 1</a>; GenBank accession numbers for all sequences are listed in <a href="#pone.0001088.s001">Table S1</a>). A total of 164 unique sequences were identified, of which 121 were recovered from <em>S. invicta</em>. This is a large increase over the six alleles previously described from this species based on sampling at a single locality in Argentina and throughout the introduced USA range <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a>. The great majority of <em>Gp-9</em> alleles in the complete fire ant data set (91%) were represented as singletons. Among the 13 alleles recovered from more than one specimen, three were shared between <em>S. invicta</em> and ants identified as belonging to the closely related species <em>S. quinquecuspis</em> or <em>S. megergates</em> (see <a href="#pone-0001088-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0001088-g001"><div class="img"><a name="pone-0001088-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Locations of sampling sites for <em>S. invicta</em> in South America.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The native range of the species is indicated by gray shading on the map. The number of nests sampled at a site ( = number of sequences obtained) is indicated in parentheses in the key to the sites if greater than one. Sites at which <em>b</em>-like alleles were found are highlighted with black rectangles; those at which the <em>b</em> alleles of the <em>b</em>-like class were found are underlined.</p>
<span>doi:10.1371/journal.pone.0001088.g001</span></div><div class="figure" id="pone-0001088-g002"><div class="img"><a name="pone-0001088-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Summary hypothesis of phylogenetic relationships of fire ant <em>Gp-9</em> sequences based on Bayesian inference.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Sequences from species other than <em>S. invicta</em> are indicated by specific abbreviations in gray. Sequences from <em>S. invicta</em> are indicated by dots; the identity and social form (M, monogyne; P, polygyne; ?, unknown) of the colony of origin are also indicated for critical sequences. All colonies from which alleles of the <em>b</em>-like clade were recovered were confirmed to be polygyne. Inferred amino acid replacements at three codons considered to be diagnostic for <em>b</em>-like alleles are mapped onto the phylogeny (reversals are indicated by bars with cross-hatching). Posterior clade probabilities &gt;50% are indicated as percentages below branches.</p>
<span>doi:10.1371/journal.pone.0001088.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Plots of transition and transversion rates against pairwise sequence distances for various sets of alleles revealed little evidence of mutational saturation in the non-coding regions or at any of the three codon positions in exons (data not shown). For the entire data set, a consistent A+T bias in base composition was found, which is more substantial in the non-coding regions (0.816) than coding regions (0.574). No evidence of non-random codon usage was found over the entire set of sequences (scaled χ<sup>2</sup> = 0.104 using Yates correction; effective number of codons = 57.07 out of a maximum of 61; codon bias index = 0.348). Also, there is no evidence for intragenic recombination having occurred (all potential recombination events fall below 95<sup>th</sup> percentile significance level [DSS: 84%; LRT: 46%]). This latter result parallels the lack of evidence for recombination at <em>Gp-9</em> reported by Krieger and Ross <a href="#pone.0001088-Krieger2">[11]</a>.</p>


<h4>Phylogenetic relationships of <em>Gp-9</em> alleles</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>All phylogenetic hypotheses recovered by the differing methods are compatible with one another. However, the Shimodaira and Hasegawa (SH) test identified the relatively poorly resolved maximum parsimony (MP) tree as significantly worse than the Bayesian inference (BI) tree (Δ-lnL = 149.233, <em>P</em> = 0.015), whereas the minimum evolution (ME) and BI trees did not differ significantly (Δ-lnL = 36.944, <em>P</em> = 0.376). The poor overall node resolution obtained with the MP analysis is consistent with the relatively few parsimony-informative sites (134) among the 164 unique sequences. The relatively high standard deviations of split frequencies in the BI analyses (~0.085) also suggest limited information content of our data set, which results in an inability to resolve some parts of the phylogeny with confidence (e.g., <a href="#pone.0001088-Ronquist1">[23]</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The phylogenetic hypothesis produced by BI is shown in <a href="#pone-0001088-g002">Figure 2</a>. Three major <em>Gp-9</em> allele clades of unresolved relationship to one another are apparent among the ingroup sequences (those recovered from the South American fire ants). Clade I contains sequences from three <em>Solenopsis</em> species considered to be rather distant relatives of <em>S. invicta</em> <a href="#pone.0001088-Pitts1">[24]</a>. Clade II contains sequences from an assortment of species more or less closely related to <em>S. invicta</em>, as well as two <em>S. invicta</em> sequences. Clade III contains two relatively well supported lineages. One includes three <em>S. invicta</em> sequences along with sequences from two quite distant relatives (<em>S. altipunctata</em> and <em>S. saevissima</em>). The second, large lineage (Clade IIIa) contains only <em>Gp-9</em> alleles recovered from the socially polymorphic South American fire ants (<em>S. invicta</em> and its close relatives <em>S. richteri</em>, <em>S. megergates</em>, <em>S. quinquecuspis</em>, <em>S. macdonaghi</em>, and <em>S.</em> species “X”). The relationships of the major <em>Gp-9</em> lineages depicted in <a href="#pone-0001088-g002">Figure 2</a> mirror the relationships inferred by Krieger and Ross <a href="#pone.0001088-Krieger2">[11]</a> from a much smaller data set. Importantly, the previously detected paraphyly and polyphyly of <em>S. invicta Gp-9</em> alleles with respect to those of the other socially polymorphic species <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a> are extended here to even more distantly related fire ant species.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Within the lineage comprising exclusively alleles from the socially polymorphic species (Clade IIIa), a clade composed almost entirely of sequences from <em>S. richteri</em> forms a sister lineage to the remaining sequences. This evidence for a basal position of <em>S. richteri</em> alleles within the group of alleles of all socially polymorphic species again is consistent with earlier conclusions based on smaller data sets <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a>.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Significantly, the <em>b</em>-like allele clade is recovered once again in our study, confirming a monophyletic origin of the variants associated with polygyny in the South American fire ants. Support for such a clade varies depending on how it is defined. If defined only by possession of the Ile<sup>139</sup> residue (the first of the three substitutions typical of <em>b</em>-like alleles to appear in the lineage), the clade is supported by a posterior probability value &lt;0.5. However, if defined by possession of all three residues considered diagnostic for the <em>b</em>-like allele class, the posterior probability value increases to 0.5. (Note that the Val<sup>139</sup> encoded by a colony SC665 sequence apparently represents a reversal; <a href="#pone-0001088-g002">Figure 2</a>). Given the evidence that sequences encoding all three residues are required for the expression of polygyny (below), we choose to define the <em>b</em>-like clade by the synapomorphy of joint possession of residues Gly<sup>42</sup> and Ile<sup>95</sup>, thus excluding from the group the sequence from colony LP719 encoding Ile<sup>139</sup> but neither of these other residues (see <a href="#pone-0001088-g002">Figure 2</a>).</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Finally, our data verify that the <em>b</em> alleles of <em>S. invicta</em> form a relatively recently derived monophyletic group within the <em>b</em>-like clade. This confirms earlier evidence that the radical, charge-changing Glu151Lys replacement characterizing these alleles occurred only once, presumably in an ancestral <em>S. invicta</em> population from northeastern Argentina or southeastern Brazil, where the alleles currently predominate (see <a href="#pone-0001088-g001">Figure 1</a>).</p>


<h4>Nucleotide variation at <em>Gp-9</em></h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Most nucleotide sites (85%) are invariant across all <em>Gp-9</em> sequences from the different <em>Solenopsis</em> species, and the two most divergent sequences (from <em>S. aurea</em> and <em>S. invicta</em>) differ at just 47 (2.1%) of their nucleotides. Most of the variable sites in the total collection of sequences (66%) occur in the non-coding regions. However, because these regions encompass 80% of the total sequence length, proportionately more variable sites occur in the coding regions (22%; 21% for third codon positions) than in the non-coding regions (14%).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The magnitude of variation at various positions along <em>Gp-9</em> and its 3′ flanking region in all the study species is shown in <a href="#pone-0001088-g003">Figure 3</a>. This depiction of per-site nucleotide counts confirms that overall variation is not conspicuously lower in the exons than in the non-coding regions; indeed, the average number of different nucleotides per site is 1.23 in the exons (all sites as well as third codon positions) but only 1.14 in the non-coding regions. However, considerable heterogeneity exists within both types of regions. For instance, a notable spike in variation is evident in the 3′ portion of exon 5, with 34% of nucleotide positions 1646-1708 (33% of the third codon positions) comprising variable sites (these sites correspond to amino acid residues 134-153). On the other hand, conspicuous dips in variation are apparent at nucleotide positions 620-715 in intron 2 (5% variable sites) and in the 3′-UTR (3′-untranslated region; 6% variable sites). Very similar patterns in the distribution of variation along <em>Gp-9</em> were observed when just the sequences recovered from the socially polymorphic species or from <em>S. invicta</em> were considered (data not shown).</p>
<div class="figure" id="pone-0001088-g003"><div class="img"><a name="pone-0001088-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Nucleotide variation along <em>Gp-9</em> and its 3′ flanking region in all <em>Solenopsis</em> study species.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The top graph shows the number of different nucleotides at each position; the bottom graph shows the mean number in a 25-bp sliding window moved by 10-bp increments. Blue shading indicates approximate positions of exons (E1-E5) and pink shading indicates the 3′-UTR. Introns are designated by I1-I4.</p>
<span>doi:10.1371/journal.pone.0001088.g003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The great majority of unique alleles from <em>S. invicta</em> are highly similar to one another at the nucleotide sequence level; indeed, most alleles within the <em>B</em>-like and <em>b</em>-like classes differ by fewer than a dozen point substitutions (<a href="#pone-0001088-g004">Figure 4</a>). The two most divergent <em>B</em>-like alleles differ at only 36 of their nucleotides (1.5%), while the two most divergent <em>b</em>-like alleles differ at half that number. Considering both classes combined, an additional peak at 15–20 substitutions attributable to differences between <em>B</em>-like and <em>b</em>-like alleles becomes apparent (<a href="#pone-0001088-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0001088-g004"><div class="img"><a name="pone-0001088-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Distributions of numbers of nucleotide substitutions between pairs of unique <em>Gp-9</em> sequences from <em>S. invicta.</em></span></strong></p><span>doi:10.1371/journal.pone.0001088.g004</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Several diversity statistics for <em>Gp-9</em> from <em>S. invicta</em> are presented in <a href="#pone-0001088-t001">Table 1</a>. The mean number of substitutions between pairs of alleles (<em>d</em>) varies from less than one for third codon positions in <em>b</em>-like alleles to almost seven in non-coding regions when all alleles are combined. In parallel with the observed patterns of site variation across sequences from all the study species, the mean proportions of substitutions (<em>p</em>) between <em>S. invicta</em> coding-regions consistently exceed those between non-coding regions, regardless of whether all codon positions or just third positions are considered. While this difference between the coding and non-coding regions is marginally non-significant based on a Fisher's exact test (<em>P</em> = 0.071), a phylogeny-based resampling test showed that coding-region tree lengths always were considerably greater than any of the tree lengths derived from non-coding regions, under both MP and maximum likelihood (ML) criteria (thus, substitution rates in coding regions exceed those in non-coding regions at <em>P</em>&lt;0.001). Considering only the <em>b</em>-like alleles of <em>S. invicta</em>, a greater than 3-fold excess of coding-region substitutions exists, attributable mostly to augmented variation in the first and second codon positions. Remarkably, given the phylogenetic restriction of the <em>b</em>-like clade and the fact that only 1/3 as many <em>b</em>-like as <em>B</em>-like <em>S. invicta</em> sequences were studied, values of both <em>d</em> and <em>p</em> over all codon positions are greater for the <em>b</em>-like group. These patterns hint at the possibility that positive selection has driven the molecular evolution of <em>Gp-9</em> in the <em>b</em>-like clade.</p>
<div class="figure" id="pone-0001088-t001"><div class="img"><a name="pone-0001088-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001088.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001088.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Diversity statistics for <em>Gp-9</em> from <em>S. invicta</em>.</span></strong></p><span>doi:10.1371/journal.pone.0001088.t001</span></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Apart from point substitutions, a single previously unknown structural change in <em>Gp-9</em> also was detected. A sequence from an <em>S. invicta</em> colony in Santiago del Estero, Argentina, carries a unique point mutation in exon 5 that transforms the stop codon (TAA) at position 154 into a glutamine-encoding codon (CAA), thereby extending the C-terminal tail of the resulting protein by 22 amino acids.</p>


<h4>Transition bias at <em>Gp-9</em></h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Ratios of the rates of transitions to those of transversions in the coding regions are shown for several data subsets in <a href="#pone-0001088-t002">Table 2</a>. Transition bias is negligible and similar between the third codon position and positions 1+2 for the <em>B</em>-like alleles. In contrast, a huge disparity in transition bias exists between these codon positions in the <em>b</em>-like alleles, due to the combined effects of a modest bias towards transitions at positions 1+2 and a sharp bias towards transversions at the third position. Over all studied sequences, there is a slight transition bias at the first two codon positions and a negligible bias at the third. These results suggest different patterns of selection acting on <em>B</em>-like and <em>b</em>-like alleles; because the great majority of third position transversions are nonsynonymous, their elevated rates only in the <em>b</em>-like clade are consistent with positive selection having acted specifically on this lineage (see <a href="#pone.0001088-Bofkin1">[25]</a>).</p>
<div class="figure" id="pone-0001088-t002"><div class="img"><a name="pone-0001088-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001088.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001088.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Transition/transversion rate ratios (transition bias) for <em>Gp-9</em> coding regions in <em>Solenopsis</em>.</span></strong></p><span>doi:10.1371/journal.pone.0001088.t002</span></div>

<h4>Amino acid replacements at <em>Gp-9</em></h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The sequence logo depicting the variable amino acids encoded by <em>Gp-9</em> from all the study species (<a href="#pone-0001088-g005">Figure 5</a>) reveals that the great majority of polymorphic positions feature a single dominant residue along with a second minor residue represented in just one or two sequences. A very similar pattern is seen for sequences from just Clade IIIa or from just <em>S. invicta</em>, although several additional positions are monomorphic in each of these smaller data sets (data not shown). Major polymorphisms found in all the data sets at positions 39, 42, 117, 120, 151, and 152 feature substitutions between amino acids of different property groups. All of these positions except 151 also are implicated to be under positive selection by comparison of nonsynonymous and synonymous substitution rates (below). Position 42 is of special note because it features one of the three replacements defining the <em>b</em>-like allele clade.</p>
<div class="figure" id="pone-0001088-g005"><div class="img"><a name="pone-0001088-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Sequence logos for variable amino acids encoded by unique <em>Gp-9</em> sequences in all <em>Solenopsis</em> study species and in the <em>b</em>-like clade.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Logos represent each position by a stack of letters, with the height of each letter proportional to the frequency of the amino acid in the recovered sequences. Overall stack height is proportional to the sequence conservation at that position (measured in bits, maximum sequence conservation is 4.3 bits). A schematic of the exon/intron structure of fire ant <em>Gp-9</em> is shown above the logos. Codons in the highly variable 3′ portion of exon 5 are demarcated with light blue boxes.</p>
<span>doi:10.1371/journal.pone.0001088.g005</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>A surprisingly large number of amino acid replacements appear to have occurred during the evolution of the <em>b</em>-like clade, based on parsimony reconstruction. The three jointly diagnostic positions, 42, 95, and 139, feature replacements at or near the base of the clade (see <a href="#pone-0001088-g002">Figure 2</a>). Also, position 39 underwent one replacement at the base of and one within the clade, while position 117 underwent five replacements within the clade. Finally, positions 136, 151, and 152 underwent single replacements within the clade. Several of these replacements involve codons in the 3′ portion of exon 5 (<a href="#pone-0001088-g005">Figure 5</a>), thus paralleling the spike in nucleotide variation observed in this region (<a href="#pone-0001088-g003">Figure 3</a>).</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Within <em>S. invicta</em>, over 43% of codons and 35% of amino acid residues are variable across all recovered alleles (<a href="#pone-0001088-t001">Table 1</a>), values that drop minimally for the <em>B</em>-like alleles but substantially for the <em>b</em>-like alleles. The similar values for the two metrics within each allele class suggest that the great majority of coding-region nucleotide substitutions have yielded amino acid replacements, a conclusion reinforced by the similarities between the codon diversity and amino acid diversity estimates. The prevalence of nonsynonymous substitutions is reflected also in the similar or higher estimates of <em>p</em> from all codon positions relative to third codon positions (<a href="#pone-0001088-t001">Table 1</a>). A similar prevalence of nonsynonymous coding-region substitutions exists across all the species; about 90% of codons exhibit identical codon and amino acid diversities, indicating that every observed nucleotide substitution at these locations caused an amino acid replacement.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>An important finding of our survey is the existence of considerable amino acid variation across the three <em>Gp-9</em> codons regarded as jointly diagnostic for the <em>b</em>-like and <em>B</em>-like allele classes in the socially polymorphic species. Alleles of the <em>b</em>-like class were reported earlier to always encode Gly<sup>42</sup>, Ile<sup>95</sup>, and Ile<sup>139</sup> residues, whereas alleles of the <em>B</em>-like class generally were found to encode Ser<sup>42</sup>, Met<sup>95</sup>, and Val<sup>139</sup> residues <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a>. We recovered 17 sequences from native <em>S. invicta</em> that feature some combination of <em>b</em>-like residues at one or two of these crucial positions and <em>B</em>-like residues at the remaining position(s). Two <em>S. invicta</em> sequences in Clade II (from colonies O40 and Pi21) bear the <em>b</em>-like Gly<sup>42</sup> residue together with the <em>B</em>-like Met<sup>95</sup> and Val<sup>139</sup> residues (<a href="#pone-0001088-g002">Figure 2</a>); remarkably, the Ser42Gly replacement characterizing this clade apparently occurred independently of the analogous replacement at the base of the <em>b</em>-like clade. A well supported group within Clade IIIa includes 12 <em>S. invicta</em> sequences encoding the <em>b</em>-like Ile<sup>95</sup> residue together with the <em>B</em>-like Ser<sup>42</sup> and Val<sup>139</sup> residues; again, the Met95Ile replacement at the base of this group occurred independently of the analogous replacement in the stem lineage of the <em>b</em>-like clade (<a href="#pone-0001088-g002">Figure 2</a>). A sequence from colony LP719 encodes the <em>b</em>-like Ile<sup>139</sup> residue together with the <em>B</em>-like Ser<sup>42</sup> and Met<sup>95</sup> residues. Finally, two <em>S. invicta</em> sequences within the <em>b</em>-like clade experienced apparent reversals from a <em>b</em>-like to <em>B</em>-like residue, at position 42 (colony Pi72nmq27) or position 139 (colony SC665) (<a href="#pone-0001088-g002">Figure 2</a>). The significance of these novel <em>Gp-9</em> sequences with respect to the form of social organization expressed by the source colonies is explained next.</p>


<h4>Association of polygyny with b-like residues in S. <em>invicta</em></h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>All 28 <em>S. invicta</em> colonies from which alleles encoding all three characteristic <em>b</em>-like residues were recovered were shown by microsatellite analysis to be polygyne (i.e., they contained multiple offspring matrilines). On the other hand, ten exemplar colonies yielding sequences representing all the major <em>B</em>-like clades were shown to be monogyne (i.e., they contained only a single offspring matriline). Thus, we further confirm an important conclusion from previous limited surveys in the native range <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Mescher1">[26]</a>; possession of typical <em>b</em>-like alleles by a colony's workers invariably is linked to the expression of polygyne social organization.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>We also were able to determine the social form of 12 of the 17 <em>S. invicta</em> colonies that yielded novel <em>Gp-9</em> variants encoding some combination of <em>b</em>-like and <em>B</em>-like residues at positions 42, 95, and 139. Data from the first ten such colonies listed in <a href="#pone-0001088-t003">Table 3</a>, all of which were monogyne, reveal that no single <em>b</em>-like residue is associated with polygyny. Significantly, the two polygyne colonies found to contain such novel variants (Pu15c, SC665) were found upon further sequencing of additional colony members to also contain workers with typical <em>b</em>-like alleles. Based on these data, we conclude that all three characteristic <em>b</em>-like residues may be jointly required for the expression of polygyny in the socially polymorphic South American fire ants.</p>
<div class="figure" id="pone-0001088-t003"><div class="img"><a name="pone-0001088-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001088.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001088.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Association of colony social organization in <em>S. invicta</em> with amino acid residues at three <em>Gp-9</em> codons jointly diagnostic for <em>B</em>-like and <em>b</em>-like alleles.</span></strong></p><span>doi:10.1371/journal.pone.0001088.t003</span></div>

<h4>Selection on <em>Gp-9</em></h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>All of the selection analyses we undertook yielded evidence of selection of some form at various codons in <em>Gp-9</em> and on various branches of the allele phylogeny. Among the site-specific methods, the single likelihood ancestor counting (SLAC) method identified two positively selected codons (<a href="#pone-0001088-t004">Table 4</a>), but the Suzuki-Gojobori counting method identified none. The former method also detected negative selection on codon 32, while both methods detected such selection on codon 99 (the two positions are invariant in the complete data set). The Bayesian random-effects method detected ten positively selected positions with very high confidence (posterior probability &gt;95%) and another four with less confidence (posterior probability 90–95%) (<a href="#pone-0001088-t004">Table 4</a>). (A tendency of the counting methods to produce more conservative results than the random-effects method has been reported previously <a href="#pone.0001088-KosakovskyPond1">[27]</a>.) Three of the 14 positions implicated by the latter method as being positively selected (134, 145, and 152) are in the highly variable 3′ portion of exon 5 (<a href="#pone-0001088-g003">Figures 3</a> and <a href="#pone-0001088-g005">5</a>).</p>
<div class="figure" id="pone-0001088-t004"><div class="img"><a name="pone-0001088-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001088.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001088.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Site-specific positive selection on <em>Gp-9</em> in <em>Solenopsis</em> identified by different methods.</span></strong></p><span>doi:10.1371/journal.pone.0001088.t004</span></div><a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>Considering branch-specific selection, the counting method of Zhang et al. <a href="#pone.0001088-Zhang1">[28]</a> revealed evidence of positive selection along three branches of the <em>Gp-9</em> phylogeny. Two of these are the stem lineage and its succeeding descendant branch at the base of the <em>b</em>-like radiation (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 3/0 [<em>P</em> = 0.004] and <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 2/0 [<em>P</em> = 0.026], respectively); significantly, the canonical <em>b</em>-like Ser42Gly and Met95Ile substitutions occur along the stem (see <a href="#pone-0001088-g002">Figure 2</a>). The lineage leading to the clade that includes the colony LP719 allele as well as the <em>b</em>-like alleles was not identified as experiencing selection (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 1/0). A third branch under positive selection represents a relatively basal, well supported lineage within Clade IIIa consisting of one allele each from <em>S. invicta</em> and <em>S. macdonaghi</em> (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 4/1 [<em>P</em> = 0.003]).</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>When we applied the branch-site-specific (Bayes empirical Bayes random-effects) method to the ancestral branch subtending the LP719 allele and <em>b</em>-like clade, <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> again did not differ significantly from one (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 1.05, <em>P</em> = 0.75). On the other hand, the method did detect a ratio significantly greater than one over the stem and interior branches of the <em>b</em>-like clade (<em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> = 17.5, <em>P</em>&lt;0.00001). This approach identified with confidence two positions under positive selection on these <em>b</em>-like branches, 39 and 117, both of which also were identified by two of the site-specific methods applied over the entire tree (<a href="#pone-0001088-t004">Table 4</a>).</p>
<a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>When we applied the branch-site-specific analysis to just the clade of <em>b</em> alleles (those <em>b</em>-like alleles bearing a charge-changing amino acid substitution at position 151), no significant signature of selection was detected on its stem lineage or internal branches.</p>
<a id="article1.body1.sec2.sec7.p5" name="article1.body1.sec2.sec7.p5"></a><p>Examination of overall <em>d</em><sub>N</sub>-<em>d</em><sub>S</sub> estimates for each pair of <em>Gp-9</em> sequences from the socially polymorphic South American fire ants (Clade IIIa sequences) reveals a two-fold excess of positive values (<a href="#pone-0001088-g006">Figure 6</a>). Estimates obtained separately for the <em>B</em>-like and <em>b</em>-like classes show that the excess is especially marked for the latter class, consistent with evidence from several of the analyses above implicating positive selection in this clade of <em>Gp-9</em> alleles associated with polygyny.</p>
<div class="figure" id="pone-0001088-g006"><div class="img"><a name="pone-0001088-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001088" data-uri="info:doi/10.1371/journal.pone.0001088.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001088.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001088.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001088.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001088.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Distributions of overall pairwise estimates of <em>d</em><sub>N</sub>-<em>d</em><sub>S</sub> for <em>Gp-9.</em></span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Estimates are shown separately for the set of all <em>Gp-9</em> sequences from the socially polymorphic South American fire ant species (Clade IIIa sequences) as well as for the subsets of all <em>B</em>-like and <em>b</em>-like sequences. Proportions of positive and negative <em>d</em><sub>N</sub>-<em>d</em><sub>S</sub> values are indicated for each group of sequences.</p>
<span>doi:10.1371/journal.pone.0001088.g006</span></div>

<h4>Phylogeography of <em>Gp-9</em> variants in S. <em>invicta</em></h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Our sequence data confirm the occurrence of <em>b</em>-like alleles at all five sampling sites in the south-central portion of the native range previously reported to contain such alleles (based on allele-specific PCR analyses <a href="#pone.0001088-Mescher1">[26]</a>). In addition, we discovered alleles of this class at four other sites well outside of their previously known area of occurrence (Corumba, Coxim, La Paz, and Suncho Corral; see <a href="#pone-0001088-g001">Figure 1</a>). These results suggest that while polygyny seems to be concentrated in northern Argentina and southeastern Brazil, it may occur at some frequency through much of the native range of <em>S. invicta</em>.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>An initial analysis of molecular variance (AMOVA) of <em>B</em>-like alleles revealed that no detectable variation occurs among the ten arbitrarily clustered groups of sites once the 30% of total variation found among sites within groups is accounted for (significance of among-site differentiation; <em>P</em>&lt;0.001). Similar results were obtained in a second analysis of 14 sites clustered according to patterns of regional differentiation at 14 neutral nuclear genes; no variation occurs among the regional groups, but 21% of the total variation occurs among sites within groups (<em>P</em> = 0.001 for among-site differentiation). For comparison with the results of this second <em>Gp-9</em> analysis, 14% of the variation at the neutral genes occurs among the regional groups, while 10% resides among sites within groups (<em>P</em>&lt;0.001 for differentiation at both levels). Thus, <em>Gp-9</em> in the monogyne form of native <em>S. invicta</em> appears to exhibit somewhat stronger differentiation than neutral nuclear markers at very local scales but much weaker differentiation at broader geographic scales covering hundreds of kilometers.</p>
<a id="article1.body1.sec2.sec8.p3" name="article1.body1.sec2.sec8.p3"></a><p>No significant pattern of isolation-by-distance was detected using Nei's <em>D</em><sub>A</sub> values for the <em>B</em>-like alleles at 40 sites (one-tailed <em>P</em> = 0.087). Thus, differentiation in <em>Gp-9</em> composition does not increase in parallel with geographic separation of sites. This finding again contrasts with the strong isolation-by-distance patterns detected using neutral nuclear markers in native <em>S. invicta</em> populations <a href="#pone.0001088-Ross6">[29]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The objective of this study was to survey naturally occurring molecular variation at <em>Gp-9</em>, a candidate gene of major effect on the expression of fire ant colony social organization, with a special emphasis on uncovering the extent and distribution of this variation in native populations of the well studied pest species <em>Solenopsis invicta</em>. Patterns of observed sequence variation at <em>Gp-</em>9 were examined with the following goals: i) to reconstruct the evolutionary relationships of variant sequences from <em>S. invicta</em> and its close relatives, ii) to identify important mutational factors affecting variation at the gene, iii) to examine the historical role of selection in shaping the variation, iv) to learn whether any single candidate amino acid residue in GP-9 protein is completely predictive of social organization, and v) to examine the geographical distribution of the observed variation. The motivation of the work was to help bridge the gap between functional biochemical information and molecular population genetic data by constructing a cogent evolutionary narrative of the genetic underpinnings of a major social adaptation (see <a href="#pone.0001088-Gotzek2">[20]</a>).</p>

<h4>General features of the molecular evolution of <em>Gp-9</em></h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Our large survey of <em>Gp-9</em> sequences from native fire ant populations succeeded in uncovering a large number of unique variants, with over 120 alleles found in <em>S. invicta</em> alone. Intragenic recombination was inferred to be unimportant in generating the diversity of sequences observed, a conclusion reached also by Krieger and Ross <a href="#pone.0001088-Krieger2">[11]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The exon/intron structure of all 164 fire ant <em>Gp-9</em> variants is identical to that reported previously <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a>, with one remarkable exception. A single <em>S. invicta</em> sequence contained a nonsynonymous nucleotide substitution in codon 154 that transformed it from a stop codon to a glutamine-encoding codon, thereby extending the C-terminal tail of the GP-9 protein by 22 amino acids. Two of these supernumerary residues are basic (Lys<sup>159</sup>, His<sup>175</sup>) while none is acidic, so that the mutant protein is likely to have a charge change mirroring or exceeding that of the proteins encoded by the distantly related <em>b</em> alleles of <em>S. invicta</em> (which have a unique Lys<sup>151</sup> replacement). The charge change in the C-terminus of the <em>b</em>-encoded proteins is associated with recessive deleterious (lethal) effects not found in other <em>b</em>-like alleles <a href="#pone.0001088-Hallar1">[17]</a>; these may stem from changes in the ligand binding/unloading properties or in the ability of the protein to form biologically active dimers, judging from the fact that the C-termini of odorant-binding proteins seem to be involved in these functions <a href="#pone.0001088-Krieger3">[30]</a>. Demonstration of similar deleterious effects of the elongated mutant protein could pave the way for functional experiments intended to clarify some basic biochemical features of GP-9 protein.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Significant patterns in the nucleotide variation along <em>Gp-9</em> and its 3′ flanking region are evident when sequences from all the fire ant species are compared. First, the average amount of nucleotide variation in the exons exceeds that in the non-coding regions, based on the proportions of variable sites as well as the mean numbers of different nucleotides per site. Specifically, about one-fourth of coding-region nucleotide sites are variable but only half that proportion of non-coding sites are variable. The disparity persists regardless of whether all codon nucleotides or just third codon positions are considered, suggesting that most of the elevated exon nucleotide diversity consists of nonsynonymous substitutions. Superimposed on this general difference between coding and non-coding regions is considerable heterogeneity in the variation occurring within the two types of regions. Apparent elevation in variation above the background level for exons occurs in the 3′ portion of exon 5 (34% vs. 22% variable sites), much of which translates into elevated amino acid variation. Moreover, apparent depressions in variation below the background non-coding level occur in the middle of intron 2 and in the 3′-UTR (&lt;6% vs. 14% variable sites).</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>Very similar patterns of variation along <em>Gp-9</em> are evident for just the sequences from the socially polymorphic species (Clade IIIa sequences) or from <em>S. invicta</em>. For <em>S. invicta</em>, the mean proportion of nucleotide substitutions (<em>p</em>) in the coding regions is elevated as much as 3-fold over that in non-coding regions, a difference judged to be highly statistically significant by a phylogeny-based resampling test (<em>P</em>&lt;0.001). As is also true for the larger set of sequences, most coding-region nucleotide variation in <em>S. invicta</em> corresponds to amino acid replacements. With respect to absolute amounts of divergence over the entire gene region, only relatively modest nucleotide sequence differentiation occurs among the large number of <em>Gp-9</em> alleles recovered from nominal <em>S. invicta</em>. Most pairs within the <em>B</em>-like or <em>b</em>-like allele classes differ by fewer than a dozen point substitutions across the 2300 bp sequence alignment, and the most divergent alleles from this species differ at just 38 (1.6%) of their sites. The two most divergent sequences in the entire data set (from <em>S. aurea</em> and <em>S. invicta</em>) differ at just 47 (2.1%) of their sites.</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>The resulting picture of <em>Gp-9</em> sequence evolution in <em>S. invicta</em> and its fire ant relatives is that relatively few point mutations have accumulated across the gene, with only 15% of sites exhibiting variation, but a high proportion of these mutations occurred in the coding regions. Moreover, most of these coding-region substitutions led to amino acid replacements, so that in <em>S. invicta</em> alone over one-third of codons now encode variable residues. These findings raise several important points with respect to the molecular evolution of <em>Gp-9</em>. First, this pattern is consistent with a general lack of negative selection acting to constrain amino acid replacements over much of the encoded protein, as has been inferred also for other insect odorant-binding proteins based on their low amino acid sequence identities (the primary structure of these proteins evidently can be highly variable as long as the tertiary structure is conserved <a href="#pone.0001088-Vogt1">[18]</a>, <a href="#pone.0001088-NagnanLeMeillour1">[31]</a>). Second, the elevated diversity observed in coding relative to non-coding regions may reflect different mutation rates or biases in the two types of sequence, which may in turn be related to the dramatically different base compositions <a href="#pone.0001088-Filipski1">[32]</a>–<a href="#pone.0001088-Ko1">[34]</a>. Third, negative selection acting with greater overall force on the non-coding than coding regions may also play some role in the observed pattern of diversity; indeed, two non-coding tracts with extremely low variation across the surveyed sequences, including the entire 3′-UTR, potentially constitute evolutionarily constrained cis-regulatory elements <a href="#pone.0001088-Wagner1">[35]</a>, <a href="#pone.0001088-Andolfatto1">[36]</a>.</p>
<a id="article1.body1.sec3.sec1.p6" name="article1.body1.sec3.sec1.p6"></a><p>Finally, the elevated coding region nucleotide variation and high proportion of polymorphic amino acids might be construed as reflecting the historical action of positive selection, presumably an important general force in the evolution of insect odorant-binding proteins <a href="#pone.0001088-Fort1">[37]</a>. Direct evidence of some role for such selection on <em>Gp-9</em> in fire ants comes from our formal selection analyses. Several codons were identified by various site-specific methods as having significantly elevated rates of nonsynonymous over synonymous substitutions across the species, and two of these (at positions 39 and 117) were implicated as well by a branch-site-specific method applied to the <em>b</em>-like clade. Moreover, replacements at the latter two codons typically involved residues of different property groups. Position 117 also was identified by Krieger and Ross <a href="#pone.0001088-Krieger2">[11]</a> as subject to positive selection based on a more diverse set of <em>Solenopsis Gp-9</em> sequences. Neither of these two consistently identified positions appears to be in the binding cavity or C-terminus of GP-9 protein based on earlier structure prediction analyses <a href="#pone.0001088-Krieger2">[11]</a>. Thus, we cannot speculate about what physiological or other traits, if any, may be affected by the amino acid replacements at these locations.</p>
<a id="article1.body1.sec3.sec1.p7" name="article1.body1.sec3.sec1.p7"></a><p>More compelling evidence for positive selection having acted on <em>Gp-9</em> comes from the combined results of the branch-specific and branch-site-specific methods, as well as the overall <em>d</em><sub>N</sub>-<em>d</em><sub>S</sub> estimates. These analyses consistently revealed elevated rates of amino acid replacement at the base of and throughout the <em>b</em>-like allele clade associated with polygyny in the socially polymorphic species. Similar results were obtained from the earlier selection analyses of Krieger and Ross <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a>. Congruent with these findings is our discovery of a highly elevated bias towards transversions at the third codon position in just the <em>b</em>-like lineage. The end result of this apparent burst of adaptive molecular evolution is a clade of rather similar alleles distinguished by low levels of silent substitutions but relatively high levels of amino acid replacements. Our study thus adds to the evidence that selection has played some creative role in the molecular evolution of <em>Gp-9</em> in fire ants, primarily in the context of the origin and elaboration of an alternative form of social behavior.</p>
<a id="article1.body1.sec3.sec1.p8" name="article1.body1.sec3.sec1.p8"></a><p>It is worth emphasizing that adaptive divergence of <em>Gp-9</em> generally does not seem to be associated with speciation events in the South American fire ants, a conclusion evident also from earlier studies <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a>, <a href="#pone.0001088-Gotzek2">[20]</a>. This is inferred from the extensive paraphyly of <em>Gp-9</em> sequences with respect to the nominal species as well as the persistence of the <em>b</em>-like clade as a trans-species polymorphism. Together with the evidence that selection has promoted the divergence of the <em>b</em>-like clade, these patterns imply that intraspecific social evolution is a more important driver of <em>Gp-9</em> sequence diversification than is cladogenesis.</p>
<a id="article1.body1.sec3.sec1.p9" name="article1.body1.sec3.sec1.p9"></a><p>Two of the site-specific methods also yielded evidence of a single codon, at position 99, being under negative selection. This position is predicted to occur in the binding cavity of GP-9 <a href="#pone.0001088-Krieger2">[11]</a>, and so may represent a rare example where any variation in the encoded residue negatively impacts the binding capability and, hence, biological function of the protein.</p>


<h4>Association of <em>Gp-9</em> molecular variation with social organization</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Among the coding-region diversity uncovered in our survey was substantial variation across the three codons that typically are jointly diagnostic between <em>b</em>-like and <em>B</em>-like alleles (positions 42, 95, 139) and, thus, predictive of colony social organization <a href="#pone.0001088-Krieger1">[10]</a>. Rather than encoding alternate sets of amino acid residues at these three positions, the newly detected variants encode various combinations of <em>b</em>-like and <em>B</em>-like residues. Based on our determination of the social organization of the source colonies for these alleles, we conclude that no single residue is completely predictive of social behavior. Indeed, <em>b</em>-like residues invariably were present at all three positions in sequences from all polygyne colonies and, so, all of these residues may be jointly required for the expression of polygyny. Remarkably, placement of the novel variants in the <em>Gp-9</em> allele phylogeny indicates that both the Gly<sup>42</sup> and Ile<sup>95</sup> <em>b</em>-like residues arose independently on at least two occasions. Apparently, however, only when they appeared concurrently on the stem lineage of the <em>b</em>-like clade already bearing the Ile<sup>139</sup> <em>b</em>-like residue did expression of polygyny become possible.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Our proposal that all three characteristic <em>b</em>-like replacements are completely associated with polygyne behavior in the socially polymorphic species and, by implication, potentially involved in its expression, contradicts the conclusion of Krieger <a href="#pone.0001088-Krieger3">[30]</a> and Krieger and Ross <a href="#pone.0001088-Krieger2">[11]</a> that Val139Ile was the lone crucial replacement. This conclusion was based primarily on the inference from protein structure modeling that residue 139 forms part of the ligand-binding cavity. However, the same analysis also predicted that residue 95 lies in the binding cavity, and residue 42, although not expected to function in ligand binding, experienced a replacement between amino acids of different property groups. Although no study to date has identified codon 139 as being under positive selection, the Bayesian random-effects test implemented in this study implicated such selection on both codons 42 and 95. Moreover, our branch-specific analyses implicated positive selection on the stem lineage of the <em>b</em>-like clade, where the Ser42Gly and Met95Ile replacements occurred, but not on the preceding ancestral branch where the Val139Ile replacement occurred. We note that our data do not completely rule out the possibility that some combination of only two <em>b</em>-like residues may underlie polygyny, but testing this possibility must await discovery of colonies that contain such variants but lack typical <em>b</em>-like sequences.</p>


<h4>Geographic distribution of <em>Gp-9</em> molecular variation</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>A previous survey of <em>Gp-9</em> polymorphism in <em>S. invicta</em> using allele-specific PCR indicated that the <em>b</em>-like allele clade (and, by extension, polygyny) is restricted to the south-central portion of this species' range <a href="#pone.0001088-Mescher1">[26]</a>. Our far more extensive sampling has extended the known area over which these alleles occur considerably northward into central Brazil. Nonetheless, in view of the fact that the ranges of the other socially polymorphic species harboring <em>b</em>-like alleles are restricted to eastern Argentina, Uruguay, and southeastern Brazil <a href="#pone.0001088-Pitts2">[38]</a>, an origin of the <em>b</em>-like clade in this area seems likely.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>We found no indication of significant higher-level (regional) structure in the geographic distribution of <em>B</em>-like variants of <em>Gp-9</em> in <em>S. invicta</em>, although local populations are highly differentiated from one another. This pattern stands in contrast to the striking regional differentiation observed for numerous other, presumably neutral, nuclear loci. One possible explanation for the difference is that, by chance, the distribution of <em>Gp-9</em> variation does not closely track that of the remaining nuclear genome simply because of the probabilistic nature of allele lineage sorting (e.g., <a href="#pone.0001088-Rosenberg1">[39]</a>). Another possibility is that sporadic interspecific hybridization in different areas followed by introgression of heterospecific <em>Gp-9</em> alleles has broken down any geographic pattern that may have developed due to restricted inter-regional gene flow. Several lines of evidence support the plausibility of this scenario. First, the <em>B</em>-like alleles of <em>S. invicta</em> are extensively paraphyletic or polyphyletic with respect to the alleles of several other fire ants, including some species regarded as quite distant relatives <a href="#pone.0001088-Pitts1">[24]</a>. Second, the <em>S. invicta</em> sequences that are polyphyletic with respect to these distant relatives often were obtained from colonies located within or close to the range of the other species (e.g., the closely related <em>S. invicta</em> and <em>S. saevissima</em> alleles in Clade II and the closely related <em>S. invicta</em>, <em>S. saevissima</em>, and <em>S. altipunctata</em> alleles in the sister lineage of Clade IIIA; see <a href="#pone-0001088-g004">Figure 4</a>). Finally, parallel patterns of minimal regional differentiation coupled with interspecific sequence paraphyly and polyphyly have been observed for the mtDNA of <em>S. invicta</em> <a href="#pone.0001088-Ross6">[29]</a>, <a href="#pone.0001088-Shoemaker1">[40]</a>, with the latter features almost certainly the result of introgression.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>This scenario posits that <em>Gp-9</em> (and the mtDNA) flows more freely between fire ant species (and, perhaps, among regional <em>S. invicta</em> populations) than the bulk of the nuclear genome, perhaps because of a lack of selection against these introgressing elements (or, in the case of the mtDNA, because of selection favoring the spread of the cytoplasmic symbiont <em>Wolbachia</em> <a href="#pone.0001088-Ahrens1">[41]</a>). One important implication of this scenario, if true, is that polygyny may not have arisen in the common ancestor of the socially polymorphic fire ants, with the <em>b</em>-like lineage persisting through multiple speciation events <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a>. Rather, this allele class and the alternate form of social behavior with which it is associated conceivably arose more recently, then spread among species of the socially polymorphic clade through hybridization (see also <a href="#pone.0001088-Keller2">[42]</a>).</p>


<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>We recovered numerous <em>Gp-9</em> alleles from <em>S. invicta</em> and other South American fire ants in their native ranges. Relatively little overall variation distiguishes these alleles, but the distribution of the variation along the gene and within the gene phylogeny is noteworthy. A surprising amount is found in the coding regions of the gene, with substitutions there usually causing amino acid replacements. Indeed, the proportion of variable amino acid positions is more than twice the proportion of variable nucleotide sites over the entire gene region, both across species and within <em>S. invicta</em>. The elevated coding-region variation may result from a general lack of negative selection acting to constrain amino acid replacements, different mutation rates or biases in coding and non-coding regions, negative selection acting with greater force on non-coding than coding regions, or most likely, positive selection acting on the protein in the <em>b</em>-like allele clade associated with polygyny. Finally, our determination of the social organization of key colonies confirmed the invariant link between the presence of typical <em>b</em>-like alleles and expression of polygyny, while our discovery of several novel alleles bearing various combinations of <em>b</em>-like and <em>B</em>-like codons revealed that no single amino acid residue is completely predictive of polygyne behavior.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>This study thus yields information of use in bridging population genetic and functional approaches to understanding the genetic basis of polygyny in fire ants. With the inception of a broad, integrative approach to investigating the biochemical pathways in which the <em>Gp-9</em> product functions, the phenotypic effects of molecular variation at <em>Gp-9</em> and other pathway genes, and the potential involvement of other genes in linkage disequilibrium with <em>Gp-9</em>, substantial progress toward understanding the evolution of this key social adaptation can be expected.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sampling</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We obtained samples from several <em>Solenopsis</em> species of varying phylogenetic relationship to <em>S. invicta</em> <a href="#pone.0001088-Pitts1">[24]</a> to assess the nature and extent of <em>Gp-9</em> sequence variation in fire ants and to evaluate the monophyly of <em>S. invicta</em> sequences with respect to those of its closest relatives. These samples consisted of a single individual per nest collected in Argentina, Brazil, and the western USA between 1992 and 2001. Samples of the following species were obtained (numbers of individuals [nests] in parentheses): <em>S. altipunctata</em> (1), <em>S. amblychila</em> (1), <em>S. aurea</em> (1), <em>S. daguerrei</em> (2), <em>S. electra</em> (1), <em>S. interrupta</em> (4), <em>S. macdonaghi</em> (4), <em>S. megergates</em> (6), <em>S. pusillignis</em> (3), <em>S. quinquecuspis</em> (6), <em>S</em>. <em>richteri</em> (8), <em>S. saevissima</em> (9), <em>S. xyloni</em> (1), and the undescribed <em>S.</em> species “X” (2).</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Samples of nominal <em>S. invicta</em> were obtained from 132 nests from 44 sites distributed over much of the native South American range (<a href="#pone-0001088-g001">Figure 1</a>) as well as from two sites in the introduced range in the USA (Georgia and California) between 1988 and 2004. Sites in the native range were chosen not only to maximize geographic coverage but also to include all of the genetically differentiated populations distinguished in earlier studies of neutral nuclear and mtDNA variation <a href="#pone.0001088-Ross6">[29]</a>, <a href="#pone.0001088-Shoemaker1">[40]</a>, <a href="#pone.0001088-Ross7">[43]</a>. The purpose of this targeted sampling scheme was to uncover the maximal amount of diversity at <em>Gp-9</em> in the focal species in order to generate a complete sequence phylogeny, which in turn formed the framework for our formal selection analyses. Multiple nests were sampled at many of the sites (see <a href="#pone-0001088-g001">Figure 1</a>), but generally only a single specimen was used from any single nest. The social organization of many of the sampled <em>S. invicta</em> colonies was determined previously by a combination of methods including discovery of multiple reproductive queens, determination of the number of offspring matrilines using allozyme markers, and detection of <em>b</em>-like <em>Gp-9</em> alleles <a href="#pone.0001088-Mescher1">[26]</a>, <a href="#pone.0001088-Ross7">[43]</a>, <a href="#pone.0001088-Ross8">[44]</a>.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Live specimens of all the study species were collected directly from nests in the field then placed immediately on liquid nitrogen for transport back to the laboratory, where they were held in a −80°C freezer pending genetic analysis.</p>


<h4>Sequencing of <em>Gp-9</em></h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Sequencing methods followed the protocols of Krieger and Ross <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a>, with some modifications. DNA was extracted using the Puregene DNA Isolation Kit (Gentra Systems, Minneapolis, MN). Polymerase chain reaction (PCR) reactions were set up in 10 µl volumes using 1.1× high fidelity PCR-ready reaction mix (Bio-X-Act Short Mix, Bioline, Randolph, MA) and 0.2 µM primers, with a hotstart thermal cycling regime starting at 95°C and followed by 35 cycles at 94°C (20 s), 62°C (30 s), and 68°C (1 min 40 s), and with a final elongation step at 68°C (10 min). Primer sequences were those used by Krieger and Ross <a href="#pone.0001088-Krieger1">[10]</a> (Gp-9/-33 forward: 5′-CATTCAAAGTACAGTAGAATAACTGCC-3′, Gp-9_2218 reverse: 5′-CAGGAGTTTGAGTTTGTCACTGC-3′). The approximately 2200-bp amplification products included the full length 1700-bp <em>Gp-9</em> gene (containing five exons and four introns) as well as a 500-bp segment of the 3′ flanking region containing the 170-bp UTR. These products were gel purified (QIAquick Gel Extraction Kit, Qiagen, Valencia, CA) and cloned into pCR2.1 vectors (Invitrogen, Carlsbad, CA), which were then used to transfect competent TOP10F′ <em>E. coli</em> cells (Invitrogen). Blue-white screening was used to identify positive clones, which were picked and subjected directly to a hotstart PCR amplification using M13 or the <em>Gp-9</em> primers (same conditions as in the previous PCR but using Taq-Pro Complete, Denville Scientific Inc., Meutchen, NJ). The resulting PCR product was checked for correct length by running it out on an ethidium bromide-stained agarose gel, then it was purified using PEG 8000 (Promega, Madison, WI).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Methods for conducting DNA sequencing reactions using internal primers also followed the protocols of Krieger and Ross <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a>. Reactions were performed using the ABI PRISM BigDye Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems, Foster City, Calif.), with the products run out in an ABI PRISM 3740xl DNA Sequencer (Applied Biosystems). In light of the considerable overlap of the internal sequencing reads, sequences were not determined in the reverse directions. Critical base calls in phylogenetically important sequences were confirmed by resequencing.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>In order to ensure a sufficiently large sample of <em>b</em>-like alleles, clones derived from suspected polygyne <em>S. invicta</em> colonies were screened for such alleles using competitive allele-specific PCR <a href="#pone.0001088-Imyanitov1">[45]</a>. Reaction mixes contained 0.13 µM primers, 0.33 µM complementary primer, 1× Taq-Pro Complete, and 0.5 µL of the clone PCR product; PCR was conducted using a cycling regime of 94°C (2 min), 35 cycles at 94°C (45 s), 64°C (45 s), and 72°C (1 min), and with a final elongation step at 72°C (5 min). The primers used in this allele-specific PCR recognize the single nucleotide substitutions at codons 95 and 139 considered to be jointly diagnostic of all <em>b</em>-like alleles <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Ross5">[21]</a>.</p>


<h4>Determination of colony social organization</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Most colonies of unknown social organization yielding sequences that encoded one or more residues considered diagnostic for <em>b</em>-like alleles (Gly<sup>42</sup>, Ile<sup>95</sup>, Ile<sup>139</sup>) were subjected to microsatellite analyses to learn whether these colonies were monogyne or polygyne. Genotypes at eight loci (<em>Sol-6</em>, <em>Sol-11</em>, <em>Sol-18</em>, <em>Sol-42</em>, <em>Sol-49</em>, <em>Sol-55</em>, <em>SolM-III</em>, and <em>SolM-V</em>) were scored for 7–12 workers from each nest (PCR methods in <a href="#pone.0001088-Krieger4">[46]</a>–<a href="#pone.0001088-Shoemaker2">[48]</a>). PCR products were visualized using an ABI PRISM 3740xl DNA Sequencer. Queens of native <em>S. invicta</em> normally mate only once <a href="#pone.0001088-Ross8">[44]</a>, <a href="#pone.0001088-Ross9">[49]</a>, so the presence of more than three alleles at a locus among a colony's workers indicates the presence of multiple offspring matrilines (polygyny).</p>


<h4>Genetic analyses</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>All <em>Gp-9</em> sequences were readily aligned by hand. The aligned sequences were tested for evidence of recombination using the DSS and LRT methods implemented in the program TOPALI <a href="#pone.0001088-Milne1">[50]</a>.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Differentiation among <em>Gp-9</em> alleles in their nucleotide composition was tested using homogeneity χ<sup>2</sup> analysis (implemented in the program PAUP* <a href="#pone.0001088-Swofford1">[51]</a>) and visual inspection (implemented in the program SeqVis <a href="#pone.0001088-Ho1">[52]</a>). Non-random codon usage was tested using the program DNASP <a href="#pone.0001088-Rozas1">[53]</a>, with Yates' correction for the observed G+C content employed.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>Transition and transversion rates were plotted against Tamura-Nei (TN93) distances for all pairs of sequences in various subsets of the data using the program DAMBE <a href="#pone.0001088-Xia1">[54]</a> to look for evidence of mutational saturation. In addition, the extent of transition bias (transition/transversion rate ratio) was estimated for the combined first and second codon positions and for the third codon position of exons in various data subsets using the program MEGA <a href="#pone.0001088-Kumar1">[55]</a>.</p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>Several measures of nucleotide sequence variation were estimated for the <em>Gp-9</em> alleles of nominal <em>S. invicta</em>. The uncorrected mean number of nucleotide substitutions (<em>d</em>) and proportion of nucleotide substitutions (<em>p</em>) were calculated for the coding regions (all positions and third codon positions) and the non-coding regions of all pairs of unique alleles <a href="#pone.0001088-Nei1">[56]</a> using MEGA. To measure the levels of observed nucleotide and amino acid variation in the coding regions of <em>S. invicta</em> sequences, we estimated two additional diversity indices for all unique sequences, the codon diversity and amino acid diversity <a href="#pone.0001088-Krieger2">[11]</a>. Codon diversity denotes the variation at each in-frame coding-region triplet, while amino acid diversity denotes the variation at the corresponding amino acid residue. Values for the two measures range from zero to one for a given codon; a value of zero signifies a location with identical codons (or amino acids) in all sequences, whereas a value of one indicates that every unique sequence displays a unique codon (or amino acid) at this location. These indices were compared to determine the extent to which observed coding-region nucleotide variation translates into amino acid replacements. All of these analyses were performed separately for the <em>b</em>-like alleles, <em>B</em>-like alleles, and all alleles combined. Other measures of sequence variation that incorporate haplotype frequency estimates (such as π) were not estimated because our targeted sampling scheme would not yield unbiased frequency estimates.</p>
<a id="article1.body1.sec4.sec4.p5" name="article1.body1.sec4.sec4.p5"></a><p>We tested for differences in substitution rates between coding and non-coding regions of <em>S. invicta</em> sequences using two statistical approaches. First, a Fisher's exact test compared <em>p</em> between the two partitions. Second, a resampling test was used to compare tree lengths between coding and non-coding regions. A null distribution of tree lengths of non-coding nucleotides was generated by randomly resampling (without replacement) 462 such nucleotides from the original data matrix 1,000 times using MESQUITE <a href="#pone.0001088-Maddison1">[57]</a>, then calculating tree lengths for each resampled matrix on the Bayesian inference (BI) phylogeny under both maximum parsimony (MP) and maximum likelihood (ML) criteria using PAUP*. Tree lengths computed for the 462-bp coding region under these criteria on the same BI tree were compared to the tree lengths derived from the randomly resampled non-coding characters.</p>


<h4>Analyses of amino acid replacements</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Patterns of amino acid variation at GP-9 were depicted graphically by generating sequence logos <a href="#pone.0001088-Schneider1">[58]</a> using the program WEBLOGO <a href="#pone.0001088-Crooks1">[59]</a>. Symbols were color-coded based on the chemical properties of each amino acid according to the scheme of Parry-Smith et al. <a href="#pone.0001088-ParrySmith1">[60]</a>. Amino acid replacements were mapped onto the <em>Gp-9</em> phylogeny using parsimony reconstruction as implemented in the program MACCLADE <a href="#pone.0001088-Maddison2">[61]</a>.</p>


<h4>Phylogenetic analyses</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Sequences from the North American fire ant species <em>S. amblychila</em>, <em>S. aurea</em>, and <em>S. xyloni</em> were specified as outgroups for all phylogenetic analyses (see <a href="#pone.0001088-Pitts1">[24]</a>). We assessed the potential impact on the phylogenetic analyses of heterogeneity in sequence nucleotide composition by constructing preliminary phylogenies with the Neighbor-Joining method <a href="#pone.0001088-Saitou1">[62]</a>, using the minimum evolution (ME) criterion based on either LogDet <a href="#pone.0001088-Lockhart1">[63]</a> or the ML distances between alleles (with all parameters estimated from the data). An SH test <a href="#pone.0001088-Shimodaira1">[64]</a> conducted on the resulting two trees revealed no evidence of compositional heterogeneity (Δ-lnL = 7.543, <em>P</em> = 0.397).</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>Due to the prohibitive computational time required for even a single heuristic search under the MP criterion, we employed the parsimony ratchet method <a href="#pone.0001088-Nixon1">[65]</a>; this approach was implemented by means of the program PAUPRAT <a href="#pone.0001088-Sikes1">[66]</a> using 500 repetitions and randomly perturbing 25% of the characters for each re-weighting. The analysis was repeated ten times to ensure that tree space had been adequately searched, then repeated another ten times while considering gaps as character states.</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>Because the MP and ME phylogenies did not differ significantly according to an SH test (Δ-lnL = 112.3, <em>P</em> = 0. 124), we estimated the best fitting model of <em>Gp-9</em> nucleotide evolution from the better resolved ME tree using the program MODELTEST <a href="#pone.0001088-Posada1">[67]</a>. We selected the most appropriate models for the complete data set and various partitions of it (non-coding regions; coding regions; first, second, and third codon positions) using the Akaike information criterion <a href="#pone.0001088-Akaike1">[68]</a> and Bayesian information criterion <a href="#pone.0001088-Schwarz1">[69]</a> (see <a href="#pone.0001088-Posada2">[70]</a>).</p>
<a id="article1.body1.sec4.sec6.p4" name="article1.body1.sec4.sec6.p4"></a><p>Finally, we conducted four independent Markov chain Monte Carlo (MCMC) tree searches under the BI optimality criterion using the program MRBAYES <a href="#pone.0001088-Ronquist1">[23]</a>, <a href="#pone.0001088-Huelsenbeck1">[71]</a>. Multiple analyses were run to ensure adequate exploration of tree and parameter space <a href="#pone.0001088-Ronquist1">[23]</a>, <a href="#pone.0001088-Larget1">[72]</a>. Five parallel chains, four of which were heated incrementally (temperature = 0.1), were started from random trees for each analysis, with initial parameter values based on the evolutionary model selected by MODELTEST. The chains were run for two million generations, with sampling every 100 generations. Stationarity of the chains was ascertained visually by plotting sample log-likelihoods through the course of each run, as well as by examining the convergence diagnostics (the potential scale reduction factor for all parameters approached 1.0 at stationarity) <a href="#pone.0001088-Ronquist1">[23]</a>. Pre-stationarity MCMC samples were discarded as burn-in (usually, around the first 700 samples), and the model parameters and tree topology were estimated using the remaining samples. The log-likelihoods, substitution models, and tree topologies were compared among independent runs using SH tests. After ensuring that all runs had converged on the same area in tree/parameter space, the samples from the four different runs were combined for final analysis.</p>


<h4>Selection analyses</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Two general approaches for comparing nonsynonymous and synonymous substitution rates were employed to test for positive selection on <em>Gp-9</em> in our complete data set, random-effects and counting analyses <a href="#pone.0001088-KosakovskyPond1">[27]</a>. Fixed-effects analyses were not employed because they require a priori designation of sites evolving under different selective regimes <a href="#pone.0001088-KosakovskyPond1">[27]</a>; we intended to use our data set for a largely independent test of the findings of Krieger and Ross <a href="#pone.0001088-Krieger1">[10]</a>, <a href="#pone.0001088-Krieger2">[11]</a> and so wished to avoid biasing the results by focusing on specific sites previously identified as being under selection.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>We employed two different random-effects methods, which fit a distribution of substitution rates across sites and then infer the rate at which each site evolves <a href="#pone.0001088-Nielsen1">[73]</a>. Because of computational limitations, the Bayesian method <a href="#pone.0001088-Huelsenbeck2">[74]</a> was conducted on a reduced data set (25 exemplar sequences representing all major clades in the BI phylogeny) using the program MRBAYES. Coding and non-coding sites were separated into unlinked partitions for the analysis. Selection on the coding partition was estimated according to the M3 codon model <a href="#pone.0001088-Yang1">[75]</a>, which is less restrictive than the commonly used Nielsen and Yang model <a href="#pone.0001088-Nielsen1">[73]</a> (see <a href="#pone.0001088-Ronquist1">[23]</a>, <a href="#pone.0001088-Huelsenbeck1">[71]</a>). MCMC chains were run for one million generations, with sampling every 100 generations. Otherwise, model specifications followed those used in the BI phylogeny estimation.</p>
<a id="article1.body1.sec4.sec7.p3" name="article1.body1.sec4.sec7.p3"></a><p>The second random-effects method used was a maximum likelihood branch-site-specific model <a href="#pone.0001088-Yang2">[76]</a>, <a href="#pone.0001088-Zhang2">[77]</a>. We removed all non-coding sites from the data set, then used the BI phylogeny as a guide to prune sequences that were sisters to sequences with identical coding regions or occurred in unresolved clades of identical coding sequences (a single exemplar was retained). This resulted in a reduced data set of 92 sequences. The CODEML program in the PAML package <a href="#pone.0001088-Yang3">[78]</a> was used to run the branch-site-specific model A, test 2 <a href="#pone.0001088-Zhang2">[77]</a>, which infers selection on codons along specified branches. We used the pruned BI phylogeny (with branch lengths estimated under the M3 model) to test for selection on the stem lineages and internal branches of the <em>b</em>-like allele clade as well as the <em>b</em> allele clade (the smaller clade of <em>b</em>-like alleles featuring the Glu151Lys substitution). We adopted the Bayes empirical Bayes <a href="#pone.0001088-Yang4">[79]</a> approach in place of the naive empirical Bayes approach to identify sites under selection <a href="#pone.0001088-Scheffler1">[80]</a>. The analysis incorporated the F3×4MG model of codon substitution <a href="#pone.0001088-Muse1">[81]</a> because it yielded better likelihood scores than the F3×4NY model <a href="#pone.0001088-Nielsen1">[73]</a>. We ran each analysis three times using different initial values for the parameters ω and κ.</p>
<a id="article1.body1.sec4.sec7.p4" name="article1.body1.sec4.sec7.p4"></a><p>We employed three different counting methods based on the Suzuki-Gojobori approach <a href="#pone.0001088-Suzuki1">[82]</a>. These methods estimate the number of nonsynonymous and synonymous substitutions at each codon position, then test for significant differences between the number of nonsynonymous changes per nonsynonymous site (<em>d</em><sub>N</sub>) and number of synonymous changes per synonymous site (<em>d</em><sub>S</sub>). The reduced data set of 92 sequences created for the branch-site-specific random-effects method was used also for the counting methods. The first counting method was the single likelihood ancestor counting (SLAC) analysis implemented in the program DATAMONKEY <a href="#pone.0001088-KosakovskyPond2">[83]</a>. The global <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio was estimated along with 95% confidence intervals using the HKY model of nucleotide substitution, with ambiguous characters resolved according to the most likely solution given by the model.</p>
<a id="article1.body1.sec4.sec7.p5" name="article1.body1.sec4.sec7.p5"></a><p>The second counting method we used follows more closely the original Suzuki and Gojobori <a href="#pone.0001088-Suzuki1">[82]</a> approach. This analysis was conducted in association with the branch-site-specific random-effects analysis using CODEML.</p>
<a id="article1.body1.sec4.sec7.p6" name="article1.body1.sec4.sec7.p6"></a><p>As a third counting method for detecting selection, we used the Zhang et al. method <a href="#pone.0001088-Zhang1">[28]</a> adapted from the approach of Messier and Stewart <a href="#pone.0001088-Messier1">[84]</a>. This approach uses reconstructed ancestral sequences to test the null hypothesis of neutral evolution (<em>d</em><sub>N</sub> = <em>d</em><sub>S</sub>) along each branch of the inferred phylogeny by means of Fisher's exact tests. To maximize the statistical power of this test, we pooled non-coding sites with coding-region synonymous sites (e.g., <a href="#pone.0001088-Rooney1">[85]</a>) after determining that there was no significant difference in substitution rates between these partitions (Fisher's exact test on <em>p</em>, <em>P</em> = 0.115). Ancestral sequences were reconstructed using the BASEML program in PAML, with <em>d</em><sub>N</sub> and <em>d</em><sub>S</sub> for each branch estimated using the “free-ratio” model <a href="#pone.0001088-Nielsen1">[73]</a>. We did not test for negative selection using this method.</p>
<a id="article1.body1.sec4.sec7.p7" name="article1.body1.sec4.sec7.p7"></a><p>Finally, we calculated maximum likelihood estimates of <em>d</em><sub>N</sub> and <em>d</em><sub>S</sub> for each pair of <em>Gp-9</em> sequences from the socially polymorphic South American fire ants using CODEML to look for evidence of broad positive selection. These analyses also were conducted separately for the <em>B</em>-like and <em>b</em>-like alleles.</p>
<a id="article1.body1.sec4.sec7.p8" name="article1.body1.sec4.sec7.p8"></a><p>Tests such as the McDonald-Kreitman and HKA tests that compare intraspecific variation with interspecific divergence in order to detect signals of positive selection were not employed in this study because the major bouts of adaptive divergence in <em>Gp-9</em> appear to be associated with social evolution within species rather than with speciation events. Also, tests using site-frequency spectrum data to test for deviations from neutral theory predictions (e.g., Tajima <em>D</em> test, Fay and Wu <em>H</em> test) were not employed because our targeted samples cannot provide the unbiased estimates of haplotype frequencies required for their proper application <a href="#pone.0001088-Yang5">[86]</a>.</p>


<h4>Phylogeographic analyses</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Evidence for geographical partitioning of related <em>Gp-9</em> alleles in native <em>S. invicta</em> was examined by conducting a series of analysis of molecular variance (AMOVA) analyses <a href="#pone.0001088-Excoffier1">[87]</a> using the program ARLEQUIN <a href="#pone.0001088-Excoffier2">[88]</a>. This procedure partitions total genetic variation among the different sampling sites or clusters of sites in order to reveal hierarchical patterns of spatial differentiation. Only <em>B</em>-like alleles were included in order to avoid any effect of spatial restriction of polygyny (e.g., <a href="#pone.0001088-Mescher1">[26]</a>). Genetic distances between <em>Gp-9</em> alleles were estimated as the squares of the numbers of pairwise sequence differences. In an initial analysis, all 40 sites containing at least one <em>B</em>-like allele were clustered arbitrarily into ten regional groups (these groups occupied areas 100–300 km in diameter). In a second analysis, only the 14 sites for which three or more sequences were available were used. In this case, goups of sites were clustered on the basis of patterns of regional differentiation previously detected at 14 presumed neutral nuclear loci <a href="#pone.0001088-Ross6">[29]</a> (these groups occupied areas 100–600 km in diameter). Parallel AMOVA analyses using the neutral nuclear data of Ross et al. <a href="#pone.0001088-Ross6">[29]</a> from the same 14 sites were conducted to provide a direct comparison with the <em>Gp-9</em> results from the second analysis. All alleles at each neutral locus were assumed to be equally related to one another (i.e., an infinite alleles model of mutation was assumed). Statistical significance of genetic differentiation among sites or clusters of sites was determined by permuting alleles across individuals (20,000 replicates) for <em>Gp-9</em> or bootstrapping over loci (10,000 replicates) for the neutral markers.</p>
<a id="article1.body1.sec4.sec8.p2" name="article1.body1.sec4.sec8.p2"></a><p>Isolation-by-distance analyses were conducted for the <em>Gp-9</em> sequences of <em>S. invicta</em> to learn whether differentiation between sites in their allele composition increases in parallel with their geographic separation. Only the 40 sites from which one or more <em>B</em>-like alleles were sampled were considered. The program GENEPOP <a href="#pone.0001088-Raymond1">[89]</a> was used to examine the relationship of Nei's net number of nucleotide differences (<em>D</em><sub>A</sub> <a href="#pone.0001088-Nei2">[90]</a>) with the natural logarithm of geographic distances between sites (see <a href="#pone.0001088-Slatkin1">[91]</a>, <a href="#pone.0001088-Rousset1">[92]</a>). Significance of isolation-by-distance relationships was determined by means of Mantel tests based on 10,000 data permutations coupled with estimation of Spearman rank correlation coefficients.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001088"></div><a name="pone.0001088.s001" id="pone.0001088.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001088.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">GenBank Accession Numbers for <em>Gp-9</em> Sequences Used in this Study.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001088.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Laurent Keller, Mark Mescher, and James Pitts for help collecting samples and Michael Strand for comments on an earlier version of this manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KR DG. Performed the experiments: DG DS. Analyzed the data: KR DG. Contributed reagents/materials/analysis tools: KR DG DS. Wrote the paper: KR DG DS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001088-Gillespie1" id="pone.0001088-Gillespie1"></a>Gillespie JH (1991) The causes of molecular evolution. Oxford: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001088-Hedrick1" id="pone.0001088-Hedrick1"></a>Hedrick PW (2004) Genetics of populations, 3<sup>rd</sup> edition. Boston: Jones and Bartlett.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001088-Golding1" id="pone.0001088-Golding1"></a>Golding GB, Dean AM (1998) The structural basis of molecular adaptation. Mol Biol Evol  15: 355–369.  <ul class="find" data-citedArticleID="1010068" data-doi="10.1093/oxfordjournals.molbev.a025932"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025932" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structural+basis+of+molecular+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structural+basis+of+molecular+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001088-Wheat1" id="pone.0001088-Wheat1"></a>Wheat CW, Watt WB, Pollock DD, Schulte PM (2006) From DNA to fitness differences: sequences and structures of adaptive variants of <em>Colias</em> phosphoglucose isomerase (PGI). Mol Biol Evol  23: 499–512.  <ul class="find" data-citedArticleID="1010208" data-doi="10.1093/molbev/msj062"><li><a href="http://dx.doi.org/10.1093/molbev/msj062" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+DNA+to+fitness+differences%3A+sequences+and+structures+of+adaptive+variants+of+Colias+phosphoglucose+isomerase+%28PGI%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+DNA+to+fitness+differences%3A+sequences+and+structures+of+adaptive+variants+of+Colias+phosphoglucose+isomerase+%28PGI%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001088-Nachman1" id="pone.0001088-Nachman1"></a>Nachman MW (2005) The genetic basis of adaptation: lessons from concealing coloration in pocket mice. Genetica  123: 125–136.  <ul class="find" data-citedArticleID="1010124" data-doi="10.1007/s10709-004-2723-y"><li><a href="http://dx.doi.org/10.1007/s10709-004-2723-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+basis+of+adaptation%3A+lessons+from+concealing+coloration+in+pocket+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+basis+of+adaptation%3A+lessons+from+concealing+coloration+in+pocket+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001088-Phillips1" id="pone.0001088-Phillips1"></a>Phillips PC (2005) Testing hypotheses regarding the genetics of adaptation. Genetica  123: 15–24.  <ul class="find" data-citedArticleID="1010138" data-doi="10.1007/s10709-004-2704-1"><li><a href="http://dx.doi.org/10.1007/s10709-004-2704-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Testing+hypotheses+regarding+the+genetics+of+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Testing+hypotheses+regarding+the+genetics+of+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001088-Vasemgi1" id="pone.0001088-Vasemgi1"></a>Vasemägi A, Primmer CR (2005) Challenges for identifying functionally important genetic variation: the promise of combining complementary research strategies. Mol Ecol  14: 3623–3642.  <ul class="find" data-citedArticleID="1010202" data-doi="10.1111/j.1365-294x.2005.02690.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2005.02690.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Challenges+for+identifying+functionally+important+genetic+variation%3A+the+promise+of+combining+complementary+research+strategies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Challenges+for+identifying+functionally+important+genetic+variation%3A+the+promise+of+combining+complementary+research+strategies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001088-Ross1" id="pone.0001088-Ross1"></a>Ross KG (1997) Multilocus evolution in fire ants: effects of selection, gene flow, and recombination. Genetics  145: 961–974.  <ul class="find" data-citedArticleID="1010156"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multilocus+evolution+in+fire+ants%3A+effects+of+selection%2C+gene+flow%2C+and+recombination.&amp;auth=&amp;atitle=Multilocus+evolution+in+fire+ants%3A+effects+of+selection%2C+gene+flow%2C+and+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multilocus+evolution+in+fire+ants%3A+effects+of+selection%2C+gene+flow%2C+and+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multilocus+evolution+in+fire+ants%3A+effects+of+selection%2C+gene+flow%2C+and+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001088-Ross2" id="pone.0001088-Ross2"></a>Ross KG, Keller L (1998) Genetic control of social organization in an ant. Proc Natl Acad Sci U S A  95: 14232–14237.  <ul class="find" data-citedArticleID="1010158" data-doi="10.1073/pnas.95.24.14232"><li><a href="http://dx.doi.org/10.1073/pnas.95.24.14232" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+control+of+social+organization+in+an+ant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+control+of+social+organization+in+an+ant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001088-Krieger1" id="pone.0001088-Krieger1"></a>Krieger MJB, Ross KG (2002) Identification of a major gene regulating complex social behavior. Science  295: 328–332.  <ul class="find" data-citedArticleID="1010098" data-doi="10.1126/science.1065247"><li><a href="http://dx.doi.org/10.1126/science.1065247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+major+gene+regulating+complex+social+behavior." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+major+gene+regulating+complex+social+behavior.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001088-Krieger2" id="pone.0001088-Krieger2"></a>Krieger MJB, Ross KG (2005) Molecular evolutionary analyses of the odorant-binding protein gene <em>Gp-9</em> in fire ants and other <em>Solenopsis</em> species. Mol Biol Evol  22: 2090–2103.  <ul class="find" data-citedArticleID="1010100" data-doi="10.1093/molbev/msi203"><li><a href="http://dx.doi.org/10.1093/molbev/msi203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolutionary+analyses+of+the+odorant-binding+protein+gene+Gp-9+in+fire+ants+and+other+Solenopsis+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolutionary+analyses+of+the+odorant-binding+protein+gene+Gp-9+in+fire+ants+and+other+Solenopsis+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001088-Ross3" id="pone.0001088-Ross3"></a>Ross KG, Keller L (2002) Experimental conversion of colony social organization by manipulation of worker genotype composition in fire ants (<em>Solenopsis invicta</em>). Behav Ecol Sociobiol  51: 287–295.  <ul class="find" data-citedArticleID="1010160" data-doi="10.1007/s00265-001-0431-5"><li><a href="http://dx.doi.org/10.1007/s00265-001-0431-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+conversion+of+colony+social+organization+by+manipulation+of+worker+genotype+composition+in+fire+ants+%28Solenopsis+invicta%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+conversion+of+colony+social+organization+by+manipulation+of+worker+genotype+composition+in+fire+ants+%28Solenopsis+invicta%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001088-Gotzek1" id="pone.0001088-Gotzek1"></a>Gotzek D, Ross KG (2007) Experimental conversion of colony social organization in fire ants (<em>Solenopsis invicta</em>): worker genotype manipulation in the absence of queen effects. J Ins Behav. In press.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001088-Bourke1" id="pone.0001088-Bourke1"></a>Bourke AFG, Franks NR (1995) Social evolution in ants. Princeton: Princeton University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0001088-Ross4" id="pone.0001088-Ross4"></a>Ross KG, Keller L (1995) Ecology and evolution of social organization: insights from fire ants and other highly eusocial insects. Annu Rev Ecol Syst  26: 631–656.  <ul class="find" data-citedArticleID="1010162" data-doi="10.1146/annurev.es.26.110195.003215"><li><a href="http://dx.doi.org/10.1146/annurev.es.26.110195.003215" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecology+and+evolution+of+social+organization%3A+insights+from+fire+ants+and+other+highly+eusocial+insects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecology+and+evolution+of+social+organization%3A+insights+from+fire+ants+and+other+highly+eusocial+insects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001088-Tschinkel1" id="pone.0001088-Tschinkel1"></a>Tschinkel WR (2006) The fire ants. Cambridge, Massachusetts: Harvard University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0001088-Hallar1" id="pone.0001088-Hallar1"></a>Hallar BL, Krieger MJB, Ross KG (2007) Potential cause of lethality of an allele implicated in social evolution in fire ants. Genetica  131: 69–79.  <ul class="find" data-citedArticleID="1010076" data-doi="10.1007/s10709-006-9114-5"><li><a href="http://dx.doi.org/10.1007/s10709-006-9114-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Potential+cause+of+lethality+of+an+allele+implicated+in+social+evolution+in+fire+ants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Potential+cause+of+lethality+of+an+allele+implicated+in+social+evolution+in+fire+ants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001088-Vogt1" id="pone.0001088-Vogt1"></a>Vogt RG (2005)  Molecular basis of pheromone detection in insects. In: Gilbert L, Iatrou K, Gill S, editors. Endocrinology, vol. 3. Comprehensive molecular insect science. London: Elsevier. pp. 753–804.  <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0001088-Keller1" id="pone.0001088-Keller1"></a>Keller L, Ross KG (1998) Selfish genes: a green beard in the red fire ant. Nature  394: 573–575.  <ul class="find" data-citedArticleID="1010088"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Selfish+genes%3A+a+green+beard+in+the+red+fire+ant.&amp;auth=&amp;atitle=Selfish+genes%3A+a+green+beard+in+the+red+fire+ant." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selfish+genes%3A+a+green+beard+in+the+red+fire+ant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selfish+genes%3A+a+green+beard+in+the+red+fire+ant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001088-Gotzek2" id="pone.0001088-Gotzek2"></a>Gotzek D, Ross KG (2007) Genetic regulation of colony social organization in fire ants: an integrative overview. Q Rev Biol  82: 201–226.  <ul class="find" data-citedArticleID="1010072" data-doi="10.1086/519965"><li><a href="http://dx.doi.org/10.1086/519965" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+regulation+of+colony+social+organization+in+fire+ants%3A+an+integrative+overview." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+regulation+of+colony+social+organization+in+fire+ants%3A+an+integrative+overview.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001088-Ross5" id="pone.0001088-Ross5"></a>Ross KG, Krieger MJB, Shoemaker DD (2003) Alternative genetic foundations for a key social polymorphism in fire ants. Genetics  165: 1853–1867.  <ul class="find" data-citedArticleID="1010164"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Alternative+genetic+foundations+for+a+key+social+polymorphism+in+fire+ants.&amp;auth=&amp;atitle=Alternative+genetic+foundations+for+a+key+social+polymorphism+in+fire+ants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+genetic+foundations+for+a+key+social+polymorphism+in+fire+ants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+genetic+foundations+for+a+key+social+polymorphism+in+fire+ants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001088-EyreWalker1" id="pone.0001088-EyreWalker1"></a>Eyre-Walker A (2006) The genomic rate of adaptive evolution. Trends Ecol Evol  21: 569–575.  <ul class="find" data-citedArticleID="1010060" data-doi="10.1016/j.tree.2006.06.015"><li><a href="http://dx.doi.org/10.1016/j.tree.2006.06.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genomic+rate+of+adaptive+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genomic+rate+of+adaptive+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001088-Ronquist1" id="pone.0001088-Ronquist1"></a>Ronquist F, Huelsenbeck JP, van der Mark P (2005) MrBayes 3.1 manual.   Available: <a href="http://mrbayes.csit.fsu.edu/mb3.1_manual.pdf">http://mrbayes.csit.fsu.edu/mb3.1_manual​.pdf</a>. Accessed 11 November 2006.  <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0001088-Pitts1" id="pone.0001088-Pitts1"></a>Pitts JP, McHugh JV, Ross KG (2005) Cladistic analysis of the fire ants of the <em>Solenopsis saevissima</em> species-group (Hymenoptera: Formicidae). Zool Scripta  34: 493–505.  <ul class="find" data-citedArticleID="1010140" data-doi="10.1111/j.1463-6409.2005.00203.x"><li><a href="http://dx.doi.org/10.1111/j.1463-6409.2005.00203.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cladistic+analysis+of+the+fire+ants+of+the+Solenopsis+saevissima+species-group+%28Hymenoptera%3A+Formicidae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cladistic+analysis+of+the+fire+ants+of+the+Solenopsis+saevissima+species-group+%28Hymenoptera%3A+Formicidae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001088-Bofkin1" id="pone.0001088-Bofkin1"></a>Bofkin L, Goldman N (2007) Variation in evolutionary processes at different codon positions. Mol Biol Evol  24: 513–521.  <ul class="find" data-citedArticleID="1010048" data-doi="10.1093/molbev/msl178"><li><a href="http://dx.doi.org/10.1093/molbev/msl178" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+in+evolutionary+processes+at+different+codon+positions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+in+evolutionary+processes+at+different+codon+positions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001088-Mescher1" id="pone.0001088-Mescher1"></a>Mescher MC, Ross KG, Shoemaker DD, Keller L, Krieger MJB (2003) Distribution of the two social forms of the fire ant <em>Solenopsis invicta</em> (Hymenoptera: Formicidae) in the native South American range. Ann Ent Soc Amer  96: 810–817.  <ul class="find" data-citedArticleID="1010116" data-doi="10.1603/0013-8746(2003)096[0810:dottsf]2.0.co;2"><li><a href="http://dx.doi.org/10.1603/0013-8746(2003)096[0810:dottsf]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+the+two+social+forms+of+the+fire+ant+Solenopsis+invicta+%28Hymenoptera%3A+Formicidae%29+in+the+native+South+American+range." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+the+two+social+forms+of+the+fire+ant+Solenopsis+invicta+%28Hymenoptera%3A+Formicidae%29+in+the+native+South+American+range.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001088-KosakovskyPond1" id="pone.0001088-KosakovskyPond1"></a>Kosakovsky Pond SL, Frost SWD (2005) Not so different after all: a comparison of methods for detecting amino acid sites under selection. Mol Biol Evol  22: 1208–1222.  <ul class="find" data-citedArticleID="1010094" data-doi="10.1093/molbev/msi105"><li><a href="http://dx.doi.org/10.1093/molbev/msi105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Not+so+different+after+all%3A+a+comparison+of+methods+for+detecting+amino+acid+sites+under+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Not+so+different+after+all%3A+a+comparison+of+methods+for+detecting+amino+acid+sites+under+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001088-Zhang1" id="pone.0001088-Zhang1"></a>Zhang J, Kumar S, Nei M (1997) Small-sample tests of episodic adaptive evolution: a case study of primate lysozymes. Mol Biol Evol  14: 1335–1338.  <ul class="find" data-citedArticleID="1010222" data-doi="10.1093/oxfordjournals.molbev.a025743"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025743" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Small-sample+tests+of+episodic+adaptive+evolution%3A+a+case+study+of+primate+lysozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Small-sample+tests+of+episodic+adaptive+evolution%3A+a+case+study+of+primate+lysozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001088-Ross6" id="pone.0001088-Ross6"></a>Ross KG, Krieger MJB, Keller L, Shoemaker DD (2007) Genetic variation and structure in native populations of the fire ant <em>Solenopsis invicta</em>: evolutionary and demographic implications. Biol J Linn Soc.  volume 92: 541–560.  <ul class="find" data-citedArticleID="1010166" data-doi="10.1111/j.1095-8312.2007.00853.x"><li><a href="http://dx.doi.org/10.1111/j.1095-8312.2007.00853.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variation+and+structure+in+native+populations+of+the+fire+ant+Solenopsis+invicta%3A+evolutionary+and+demographic+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variation+and+structure+in+native+populations+of+the+fire+ant+Solenopsis+invicta%3A+evolutionary+and+demographic+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001088-Krieger3" id="pone.0001088-Krieger3"></a>Krieger MJB (2005) To <em>b</em> or not to <em>b</em>: a pheromone-binding protein regulates colony social organization in fire ants. Bioessays  27: 91–99.  <ul class="find" data-citedArticleID="1010102" data-doi="10.1002/bies.20129"><li><a href="http://dx.doi.org/10.1002/bies.20129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=To+b+or+not+to+b%3A+a+pheromone-binding+protein+regulates+colony+social+organization+in+fire+ants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22To+b+or+not+to+b%3A+a+pheromone-binding+protein+regulates+colony+social+organization+in+fire+ants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001088-NagnanLeMeillour1" id="pone.0001088-NagnanLeMeillour1"></a>Nagnan-Le Meillour P, Jacquin-Joly E (2003)  Biochemistry and diversity of insect odorant-binding proteins. In: Blomquist GJ, Vogt RG, editors. Insect pheromone biochemistry and molecular biology: The biosynthesis and detection of pheromones and plant volatiles. London: Elsevier. pp. 509–537.  <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0001088-Filipski1" id="pone.0001088-Filipski1"></a>Filipski J (1988) Why the rate of silent codon substitutions is variable within a vertebrate's genome. J Theor Biol  134: 159–164.  <ul class="find" data-citedArticleID="1010062" data-doi="10.1016/s0022-5193(88)80199-1"><li><a href="http://dx.doi.org/10.1016/s0022-5193(88)80199-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+the+rate+of+silent+codon+substitutions+is+variable+within+a+vertebrate%27s+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+the+rate+of+silent+codon+substitutions+is+variable+within+a+vertebrate%27s+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001088-Haddrill1" id="pone.0001088-Haddrill1"></a>Haddrill PR, Charlesworth B, Halligan DL, Andolfatto P (2005) Patterns of intron sequence evolution in <em>Drosophila</em> are dependent upon length and GC content. Genome Biol  6: R67.  <ul class="find" data-citedArticleID="1010074" data-doi="10.1186/gb-2005-6-8-r67"><li><a href="http://dx.doi.org/10.1186/gb-2005-6-8-r67" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+intron+sequence+evolution+in+Drosophila+are+dependent+upon+length+and+GC+content." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+intron+sequence+evolution+in+Drosophila+are+dependent+upon+length+and+GC+content.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001088-Ko1" id="pone.0001088-Ko1"></a>Ko W-Y, Piao S, Akashi H (2006) Strong regional heterogeneity in base composition evolution on the <em>Drosophila</em> X chromosome. Genetics  174: 349–362.  <ul class="find" data-citedArticleID="1010092" data-doi="10.1534/genetics.105.054346"><li><a href="http://dx.doi.org/10.1534/genetics.105.054346" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strong+regional+heterogeneity+in+base+composition+evolution+on+the+Drosophila+X+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strong+regional+heterogeneity+in+base+composition+evolution+on+the+Drosophila+X+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001088-Wagner1" id="pone.0001088-Wagner1"></a>Wagner GP, Fried C, Prohaska SJ, Stadler PF (2004) Divergence of conserved non-coding sequences: rate estimates and relative rate tests. Mol Biol Evol  21: 2116–2121.  <ul class="find" data-citedArticleID="1010206" data-doi="10.1093/molbev/msh221"><li><a href="http://dx.doi.org/10.1093/molbev/msh221" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Divergence+of+conserved+non-coding+sequences%3A+rate+estimates+and+relative+rate+tests." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Divergence+of+conserved+non-coding+sequences%3A+rate+estimates+and+relative+rate+tests.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001088-Andolfatto1" id="pone.0001088-Andolfatto1"></a>Andolfatto P (2005) Adaptive evolution of non-coding DNA in <em>Drosophila</em>. Nature  437: 1149–1152.  <ul class="find" data-citedArticleID="1010046" data-doi="10.1038/nature04107"><li><a href="http://dx.doi.org/10.1038/nature04107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+non-coding+DNA+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+non-coding+DNA+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001088-Fort1" id="pone.0001088-Fort1"></a>Forêt S, Maleszka R (2006) Function and evolution of a gene family encoding odorant binding-like proteins in a social insect, the honey bee (<em>Apis mellifera</em>). Genome Res  16: 1404–1413.  <ul class="find" data-citedArticleID="1010064" data-doi="10.1101/gr.5075706"><li><a href="http://dx.doi.org/10.1101/gr.5075706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Function+and+evolution+of+a+gene+family+encoding+odorant+binding-like+proteins+in+a+social+insect%2C+the+honey+bee+%28Apis+mellifera%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Function+and+evolution+of+a+gene+family+encoding+odorant+binding-like+proteins+in+a+social+insect%2C+the+honey+bee+%28Apis+mellifera%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001088-Pitts2" id="pone.0001088-Pitts2"></a>Pitts JP, McHugh JV, Ross KG (2007) Revision of the fire ants of the <em>Solenopsis saevissima</em> species-group (Hymenoptera: Formicidae). Zootaxa. In press.   <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0001088-Rosenberg1" id="pone.0001088-Rosenberg1"></a>Rosenberg NA (2003) The shapes of neutral gene genealogies in two species: probabilities of monophyly, paraphyly, and polyphyly in a coalescent model. Evolution  57: 1465–1477.  <ul class="find" data-citedArticleID="1010154" data-doi="10.1554/03-012"><li><a href="http://dx.doi.org/10.1554/03-012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+shapes+of+neutral+gene+genealogies+in+two+species%3A+probabilities+of+monophyly%2C+paraphyly%2C+and+polyphyly+in+a+coalescent+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+shapes+of+neutral+gene+genealogies+in+two+species%3A+probabilities+of+monophyly%2C+paraphyly%2C+and+polyphyly+in+a+coalescent+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001088-Shoemaker1" id="pone.0001088-Shoemaker1"></a>Shoemaker DD, Ahrens ME, Ross KG (2006) Molecular phylogeny of fire ants of the <em>Solenopsis saevissima</em> species-group based on mtDNA sequences. Mol Phyl Evol  38: 200–215.  <ul class="find" data-citedArticleID="1010188" data-doi="10.1016/j.ympev.2005.07.014"><li><a href="http://dx.doi.org/10.1016/j.ympev.2005.07.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+of+fire+ants+of+the+Solenopsis+saevissima+species-group+based+on+mtDNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+of+fire+ants+of+the+Solenopsis+saevissima+species-group+based+on+mtDNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001088-Ahrens1" id="pone.0001088-Ahrens1"></a>Ahrens M, Shoemaker DD (2005) Evolutionary history of <em>Wolbachia</em> infections in the fire ant <em>Solenopsis invicta</em>. BMC Evol Biol  5: 35.  <ul class="find" data-citedArticleID="1010042"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolutionary+history+of+Wolbachia+infections+in+the+fire+ant+Solenopsis+invicta.&amp;auth=&amp;atitle=Evolutionary+history+of+Wolbachia+infections+in+the+fire+ant+Solenopsis+invicta." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+history+of+Wolbachia+infections+in+the+fire+ant+Solenopsis+invicta." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+history+of+Wolbachia+infections+in+the+fire+ant+Solenopsis+invicta.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001088-Keller2" id="pone.0001088-Keller2"></a>Keller L, Parker JD (2002) Behavioral genetics: a gene for supersociality. Curr Biol  12: 180–181.  <ul class="find" data-citedArticleID="1010090" data-doi="10.1016/s0960-9822(02)00737-6"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)00737-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Behavioral+genetics%3A+a+gene+for+supersociality." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Behavioral+genetics%3A+a+gene+for+supersociality.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001088-Ross7" id="pone.0001088-Ross7"></a>Ross KG, Shoemaker DD (2005) Species delimitation in native South American fire ants. Mol Ecol  14: 3419–3438.  <ul class="find" data-citedArticleID="1010168" data-doi="10.1111/j.1365-294x.2005.02661.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2005.02661.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Species+delimitation+in+native+South+American+fire+ants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Species+delimitation+in+native+South+American+fire+ants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001088-Ross8" id="pone.0001088-Ross8"></a>Ross KG, Krieger MJB, Shoemaker DD, Vargo EL, Keller L (1997) Hierarchical analysis of genetic structure in native fire ant populations: results from three classes of molecular markers. Genetics  147: 643–655.  <ul class="find" data-citedArticleID="1010170"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hierarchical+analysis+of+genetic+structure+in+native+fire+ant+populations%3A+results+from+three+classes+of+molecular+markers.&amp;auth=&amp;atitle=Hierarchical+analysis+of+genetic+structure+in+native+fire+ant+populations%3A+results+from+three+classes+of+molecular+markers." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hierarchical+analysis+of+genetic+structure+in+native+fire+ant+populations%3A+results+from+three+classes+of+molecular+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hierarchical+analysis+of+genetic+structure+in+native+fire+ant+populations%3A+results+from+three+classes+of+molecular+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001088-Imyanitov1" id="pone.0001088-Imyanitov1"></a>Imyanitov EN, Buslov KG, Suspitsin EN, Kuligina ES, Belogubova EV, et al.  (2002) Improved reliability of allele-specific PCR. Biotechniques  33: 484–490.  <ul class="find" data-citedArticleID="1010086" data-doi="10.1016/s1359-6349(03)90637-9"><li><a href="http://dx.doi.org/10.1016/s1359-6349(03)90637-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+reliability+of+allele-specific+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+reliability+of+allele-specific+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001088-Krieger4" id="pone.0001088-Krieger4"></a>Krieger MJB, Keller L (1997) Polymorphism at dinucleotide microsatellite loci in fire ant <em>Solenopsis invicta</em> populations. Mol Ecol  6: 997–999.  <ul class="find" data-citedArticleID="1010104" data-doi="10.1046/j.1365-294x.1997.00264.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.1997.00264.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphism+at+dinucleotide+microsatellite+loci+in+fire+ant+Solenopsis+invicta+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphism+at+dinucleotide+microsatellite+loci+in+fire+ant+Solenopsis+invicta+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001088-Chen1" id="pone.0001088-Chen1"></a>Chen YP, Lu LY, Skow LC, Vinson SB (2003) Relatedness among co-existing queens within polygyne colonies of a Texas population of the fire ant, <em>Solenopsis invicta</em>. Southwest Ent  28: 27–36.  <ul class="find" data-citedArticleID="1010052"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Relatedness+among+co-existing+queens+within+polygyne+colonies+of+a+Texas+population+of+the+fire+ant%2C+Solenopsis+invicta.&amp;auth=&amp;atitle=Relatedness+among+co-existing+queens+within+polygyne+colonies+of+a+Texas+population+of+the+fire+ant%2C+Solenopsis+invicta." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relatedness+among+co-existing+queens+within+polygyne+colonies+of+a+Texas+population+of+the+fire+ant%2C+Solenopsis+invicta." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relatedness+among+co-existing+queens+within+polygyne+colonies+of+a+Texas+population+of+the+fire+ant%2C+Solenopsis+invicta.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001088-Shoemaker2" id="pone.0001088-Shoemaker2"></a>Shoemaker DD, DeHeer CJ, Krieger MJB, Ross KG (2006) Population genetics of the invasive fire ant <em>Solenopsis invicta</em> (Hymenoptera: Formicidae) in the United States. Ann Ent Soc Amer  99: 1213–1233.  <ul class="find" data-citedArticleID="1010190" data-doi="10.1603/0013-8746(2006)99[1213:pgotif]2.0.co;2"><li><a href="http://dx.doi.org/10.1603/0013-8746(2006)99[1213:pgotif]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+genetics+of+the+invasive+fire+ant+Solenopsis+invicta+%28Hymenoptera%3A+Formicidae%29+in+the+United+States." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+genetics+of+the+invasive+fire+ant+Solenopsis+invicta+%28Hymenoptera%3A+Formicidae%29+in+the+United+States.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001088-Ross9" id="pone.0001088-Ross9"></a>Ross KG, Vargo EL, Keller L, Trager JC (1993) Effect of a founder event on variation in the genetic sex-determining system of the fire ant <em>Solenopsis invicta</em>. Genetics  135: 843–854.  <ul class="find" data-citedArticleID="1010172"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effect+of+a+founder+event+on+variation+in+the+genetic+sex-determining+system+of+the+fire+ant+Solenopsis+invicta.&amp;auth=&amp;atitle=Effect+of+a+founder+event+on+variation+in+the+genetic+sex-determining+system+of+the+fire+ant+Solenopsis+invicta." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+a+founder+event+on+variation+in+the+genetic+sex-determining+system+of+the+fire+ant+Solenopsis+invicta." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+a+founder+event+on+variation+in+the+genetic+sex-determining+system+of+the+fire+ant+Solenopsis+invicta.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001088-Milne1" id="pone.0001088-Milne1"></a>Milne I, Wright F, Rowe G, Marshal DF, Husmeier D, et al.  (2004) TOPALi: software for automatic identification of recombinant sequences within DNA multiple alignments. Bioinformatics  20: 1806–1807.  <ul class="find" data-citedArticleID="1010120" data-doi="10.1093/bioinformatics/bth155"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TOPALi%3A+software+for+automatic+identification+of+recombinant+sequences+within+DNA+multiple+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TOPALi%3A+software+for+automatic+identification+of+recombinant+sequences+within+DNA+multiple+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001088-Swofford1" id="pone.0001088-Swofford1"></a>Swofford DL (2003) PAUP*; Phylogenetic analysis using parsimony (*and other methods). Version 4. Sunderland, Massachusetts: Sinauer.   <ul class="find-nolinks"></ul></li><li><span class="label">52.
              </span><a name="pone.0001088-Ho1" id="pone.0001088-Ho1"></a>Ho JWK, Adams CE, Lew JB, Matthews TJ, Ng CC, et al.  (2006) SeqVis: Visualization of compositional heterogeneity in large alignments of nucleotides. Bioinformatics  22: 2162–2163.  <ul class="find" data-citedArticleID="1010080" data-doi="10.1093/bioinformatics/btl283"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl283" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SeqVis%3A+Visualization+of+compositional+heterogeneity+in+large+alignments+of+nucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SeqVis%3A+Visualization+of+compositional+heterogeneity+in+large+alignments+of+nucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001088-Rozas1" id="pone.0001088-Rozas1"></a>Rozas J, Sanchez-DelBarrio JC, Messeguer X, Rozas R (2003) DNASP, DNA polymorphism analyses by the coalescent and other methods. Bioinformatics  19: 2496–2497.  <ul class="find" data-citedArticleID="1010176" data-doi="10.1093/bioinformatics/btg359"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNASP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNASP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001088-Xia1" id="pone.0001088-Xia1"></a>Xia X, Xie Z (2001) DAMBE: software package for data analysis in molecular biology and evolution. J Hered  92: 371–373.  <ul class="find" data-citedArticleID="1010210" data-doi="10.1093/jhered/92.4.371"><li><a href="http://dx.doi.org/10.1093/jhered/92.4.371" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DAMBE%3A+software+package+for+data+analysis+in+molecular+biology+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DAMBE%3A+software+package+for+data+analysis+in+molecular+biology+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001088-Kumar1" id="pone.0001088-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA 3: Integrated software for molecular evolutionary genetics analysis and sequence alignment. Brief Bioinf  5: 150–163.  <ul class="find" data-citedArticleID="1010106" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA+3%3A+Integrated+software+for+molecular+evolutionary+genetics+analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA+3%3A+Integrated+software+for+molecular+evolutionary+genetics+analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001088-Nei1" id="pone.0001088-Nei1"></a>Nei M, Kumar S (2000) Molecular evolution and phylogenetics. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">57.
              </span><a name="pone.0001088-Maddison1" id="pone.0001088-Maddison1"></a>Maddison WP, Maddison DR (2006) MESQUITE: a modular system for evolutionary analysis.   Available: <a href="http://mesquiteproject.org">http://mesquiteproject.org</a>. Accessed 4 September 2007.  <ul class="find-nolinks"></ul></li><li><span class="label">58.
              </span><a name="pone.0001088-Schneider1" id="pone.0001088-Schneider1"></a>Schneider TD, Stephens RM (1990) Sequence logos: a new way to display consensus sequences. Nuc Acids Res  18: 6097–6100.  <ul class="find" data-citedArticleID="1010182" data-doi="10.1093/nar/18.20.6097"><li><a href="http://dx.doi.org/10.1093/nar/18.20.6097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+logos%3A+a+new+way+to+display+consensus+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+logos%3A+a+new+way+to+display+consensus+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001088-Crooks1" id="pone.0001088-Crooks1"></a>Crooks GE, Hon G, Chandonia JM, Brenner SE (2004) WebLogo: A sequence logo generator. Genome Res  14: 1188–1190.  <ul class="find" data-citedArticleID="1010054" data-doi="10.1101/gr.849004"><li><a href="http://dx.doi.org/10.1101/gr.849004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WebLogo%3A+A+sequence+logo+generator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WebLogo%3A+A+sequence+logo+generator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001088-ParrySmith1" id="pone.0001088-ParrySmith1"></a>Parry-Smith DJ, Payne AWR, Michie AD, Attwood TK (1998) CINEMA–A novel colour interactive editor for multiple alignments. Gene  221: GC57–GC63.  <ul class="find" data-citedArticleID="1010136" data-doi="10.1016/s0378-1119(97)00650-1"><li><a href="http://dx.doi.org/10.1016/s0378-1119(97)00650-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CINEMA%E2%80%93A+novel+colour+interactive+editor+for+multiple+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CINEMA%E2%80%93A+novel+colour+interactive+editor+for+multiple+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001088-Maddison2" id="pone.0001088-Maddison2"></a>Maddison DR, Maddison WP (2000) MacClade 4: Analysis of phylogeny and character evolution, CD-ROM ed. Sunderland, Massachusetts: Sinauer.   <ul class="find-nolinks"></ul></li><li><span class="label">62.
              </span><a name="pone.0001088-Saitou1" id="pone.0001088-Saitou1"></a>Saitou N, Nei M (1987) The neighbor-joining method; a new method for reconstructing phylogenetic trees. Mol Biol Evol  4: 406–425.  <ul class="find" data-citedArticleID="1010178"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+neighbor-joining+method%3B+a+new+method+for+reconstructing+phylogenetic+trees.&amp;auth=&amp;atitle=The+neighbor-joining+method%3B+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neighbor-joining+method%3B+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neighbor-joining+method%3B+a+new+method+for+reconstructing+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001088-Lockhart1" id="pone.0001088-Lockhart1"></a>Lockhart PJ, Steel MA, Hendy MD, Penny D (1994) Recovering evolutionary trees under a more realistic model of sequence evolution. Mol Biol Evol  11: 605–612.  <ul class="find" data-citedArticleID="1010110"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Recovering+evolutionary+trees+under+a+more+realistic+model+of+sequence+evolution.&amp;auth=&amp;atitle=Recovering+evolutionary+trees+under+a+more+realistic+model+of+sequence+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recovering+evolutionary+trees+under+a+more+realistic+model+of+sequence+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recovering+evolutionary+trees+under+a+more+realistic+model+of+sequence+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001088-Shimodaira1" id="pone.0001088-Shimodaira1"></a>Shimodaira H, Hasegawa M (1999) Multiple comparisons of log-likelihoods with applications to phylogenetic inference. Mol Biol Evol  16: 1114–1116.  <ul class="find" data-citedArticleID="1010186" data-doi="10.1093/oxfordjournals.molbev.a026201"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+comparisons+of+log-likelihoods+with+applications+to+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+comparisons+of+log-likelihoods+with+applications+to+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001088-Nixon1" id="pone.0001088-Nixon1"></a>Nixon KC (1999) The Parsimony Ratchet, a new method for rapid parsimony analysis. Cladistics  15: 407–414.  <ul class="find" data-citedArticleID="1010134" data-doi="10.1006/clad.1999.0121"><li><a href="http://dx.doi.org/10.1006/clad.1999.0121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Parsimony+Ratchet%2C+a+new+method+for+rapid+parsimony+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Parsimony+Ratchet%2C+a+new+method+for+rapid+parsimony+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001088-Sikes1" id="pone.0001088-Sikes1"></a>Sikes DS, Lewis PO (2001) Software manual for PAUPRat: A tool to implement parsimony ratchet searches using PAUP*.   Available: <a href="http://www.ucalgary.ca/dsikes/software2.htm">http://www.ucalgary.ca/dsikes/software2.​htm</a>. Accessed 20 December 2006.  <ul class="find-nolinks"></ul></li><li><span class="label">67.
              </span><a name="pone.0001088-Posada1" id="pone.0001088-Posada1"></a>Posada D, Crandall KA (1998) MODELTEST: testing the model of DNA substitution. Bioinformatics  14: 817–818.  <ul class="find" data-citedArticleID="1010144" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MODELTEST%3A+testing+the+model+of+DNA+substitution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MODELTEST%3A+testing+the+model+of+DNA+substitution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001088-Akaike1" id="pone.0001088-Akaike1"></a>Akaike H (1974) New look at statistical model identification. IEEE Trans Auto Contr  19: 716–723.  <ul class="find" data-citedArticleID="1010044" data-doi="10.1109/tac.1974.1100705"><li><a href="http://dx.doi.org/10.1109/tac.1974.1100705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+look+at+statistical+model+identification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+look+at+statistical+model+identification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001088-Schwarz1" id="pone.0001088-Schwarz1"></a>Schwarz G (1978) Estimating the dimension of a model. Ann Stat  6: 461–464.  <ul class="find" data-citedArticleID="1010184" data-doi="10.1214/aos/1176344136"><li><a href="http://dx.doi.org/10.1214/aos/1176344136" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+the+dimension+of+a+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+the+dimension+of+a+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001088-Posada2" id="pone.0001088-Posada2"></a>Posada D, Buckley TR (2004) Model selection and model averaging in phylogenetics: advantages of Aikake information criterion and Bayesian approaches over likelihood ratio tests. Syst Biol  53: 793–808.  <ul class="find" data-citedArticleID="1010146"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Model+selection+and+model+averaging+in+phylogenetics%3A+advantages+of+Aikake+information+criterion+and+Bayesian+approaches+over+likelihood+ratio+tests.&amp;auth=&amp;atitle=Model+selection+and+model+averaging+in+phylogenetics%3A+advantages+of+Aikake+information+criterion+and+Bayesian+approaches+over+likelihood+ratio+tests." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Model+selection+and+model+averaging+in+phylogenetics%3A+advantages+of+Aikake+information+criterion+and+Bayesian+approaches+over+likelihood+ratio+tests." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Model+selection+and+model+averaging+in+phylogenetics%3A+advantages+of+Aikake+information+criterion+and+Bayesian+approaches+over+likelihood+ratio+tests.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001088-Huelsenbeck1" id="pone.0001088-Huelsenbeck1"></a>Huelsenbeck JP, Ronquist F (2001) MRBAYES: Bayesian inference of phylogenetic trees. Bioinformatics  17: 754–755.  <ul class="find" data-citedArticleID="1010082" data-doi="10.1093/bioinformatics/17.8.754"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.8.754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001088-Larget1" id="pone.0001088-Larget1"></a>Larget B (2005)  Introduction to Markov chain Monte Carlo methods in molecular evolution. In: Nielsen R, editor. Statistical methods in molecular evolution. New York: Springer.   <ul class="find-nolinks"></ul></li><li><span class="label">73.
              </span><a name="pone.0001088-Nielsen1" id="pone.0001088-Nielsen1"></a>Nielsen R, Yang Z (1998) Likelihood models for detecting postively selected amino acid sites and applications to the HIV-1 envelope genes. Genetics  148: 929–936.  <ul class="find" data-citedArticleID="1010132"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Likelihood+models+for+detecting+postively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+genes.&amp;auth=&amp;atitle=Likelihood+models+for+detecting+postively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Likelihood+models+for+detecting+postively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Likelihood+models+for+detecting+postively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001088-Huelsenbeck2" id="pone.0001088-Huelsenbeck2"></a>Huelsenbeck JP, Dyer KA (2004) Bayesian estimation of positively selected sites. J Mol Evol  58: 661–672.  <ul class="find" data-citedArticleID="1010084" data-doi="10.1007/s00239-004-2588-9"><li><a href="http://dx.doi.org/10.1007/s00239-004-2588-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+estimation+of+positively+selected+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+estimation+of+positively+selected+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001088-Yang1" id="pone.0001088-Yang1"></a>Yang Z, Nielsen R, Goldman N, Krabbe Pedersen A-M (2000) Codon-substitution models for heterogeneous selection pressure at amino acid sites. Genetics  155: 431–449.  <ul class="find" data-citedArticleID="1010212"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites.&amp;auth=&amp;atitle=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001088-Yang2" id="pone.0001088-Yang2"></a>Yang Z, Nielsen R (2002) Codon-substitution models for detecting molecular adaptation at individual sites along specified lineages. Mol Biol Evol  19: 908–917.  <ul class="find" data-citedArticleID="1010214" data-doi="10.1093/oxfordjournals.molbev.a004148"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-substitution+models+for+detecting+molecular+adaptation+at+individual+sites+along+specified+lineages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-substitution+models+for+detecting+molecular+adaptation+at+individual+sites+along+specified+lineages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001088-Zhang2" id="pone.0001088-Zhang2"></a>Zhang J, Nielsen R, Yang Z (2005) Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level. Mol Biol Evol  22: 2472–2479.  <ul class="find" data-citedArticleID="1010224" data-doi="10.1093/molbev/msi237"><li><a href="http://dx.doi.org/10.1093/molbev/msi237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0001088-Yang3" id="pone.0001088-Yang3"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Appl Biosci  13: 555–556.  <ul class="find" data-citedArticleID="1010216" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001088-Yang4" id="pone.0001088-Yang4"></a>Yang Z, Wong WSW, Nielsen R (2005) Bayes empirical bayes inference of amino acid sites under positive selection. Mol Biol Evol  22: 1107–1118.  <ul class="find" data-citedArticleID="1010218" data-doi="10.1093/molbev/msi097"><li><a href="http://dx.doi.org/10.1093/molbev/msi097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayes+empirical+bayes+inference+of+amino+acid+sites+under+positive+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayes+empirical+bayes+inference+of+amino+acid+sites+under+positive+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001088-Scheffler1" id="pone.0001088-Scheffler1"></a>Scheffler K, Seoighe C (2005) A Bayesian model comparison approach to inferring positive selection. Mol Biol Evol  22: 2531–2540.  <ul class="find" data-citedArticleID="1010180" data-doi="10.1093/molbev/msi250"><li><a href="http://dx.doi.org/10.1093/molbev/msi250" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Bayesian+model+comparison+approach+to+inferring+positive+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Bayesian+model+comparison+approach+to+inferring+positive+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0001088-Muse1" id="pone.0001088-Muse1"></a>Muse SV, Gaut BS (1994) A likelihood approach for comparing synonymous and nonsynonymous nucleotide substitution rates, with application to the chloroplast genome. Mol Biol Evol  11: 715–724.  <ul class="find" data-citedArticleID="1010122"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome.&amp;auth=&amp;atitle=A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0001088-Suzuki1" id="pone.0001088-Suzuki1"></a>Suzuki Y, Gojobori T (1999) A method for detecting positive selection at single amino acid sites. Mol Biol Evol  16: 1315–1328.  <ul class="find" data-citedArticleID="1010196" data-doi="10.1093/oxfordjournals.molbev.a026042"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+method+for+detecting+positive+selection+at+single+amino+acid+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+method+for+detecting+positive+selection+at+single+amino+acid+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0001088-KosakovskyPond2" id="pone.0001088-KosakovskyPond2"></a>Kosakovsky Pond SL, Frost SWD (2005) DATAMONKEY: rapid detection of selective pressure on individual sites of codon alignments. Bioinformatics  21: 2531–2533.  <ul class="find" data-citedArticleID="1010096" data-doi="10.1093/bioinformatics/bti320"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti320" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DATAMONKEY%3A+rapid+detection+of+selective+pressure+on+individual+sites+of+codon+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DATAMONKEY%3A+rapid+detection+of+selective+pressure+on+individual+sites+of+codon+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0001088-Messier1" id="pone.0001088-Messier1"></a>Messier W, Stewart C-B (1997) Episodic adaptive evolution of primate lysozymes. Nature  385: 151–154.  <ul class="find" data-citedArticleID="1010118" data-doi="10.1038/385151a0"><li><a href="http://dx.doi.org/10.1038/385151a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Episodic+adaptive+evolution+of+primate+lysozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Episodic+adaptive+evolution+of+primate+lysozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0001088-Rooney1" id="pone.0001088-Rooney1"></a>Rooney AP, Zhang J (1999) Rapid evolution of a primate sperm protein: relaxation of functional constraint or positive Darwinian selection? Mol Biol Evol  16: 706–710.  <ul class="find" data-citedArticleID="1010152" data-doi="10.1093/oxfordjournals.molbev.a026153"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026153" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+evolution+of+a+primate+sperm+protein%3A+relaxation+of+functional+constraint+or+positive+Darwinian+selection%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+evolution+of+a+primate+sperm+protein%3A+relaxation+of+functional+constraint+or+positive+Darwinian+selection%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0001088-Yang5" id="pone.0001088-Yang5"></a>Yang Z (2006) Computational molecular evolution. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">87.
              </span><a name="pone.0001088-Excoffier1" id="pone.0001088-Excoffier1"></a>Excoffier L, Smouse PE, Quattro JM (1992) Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data. Genetics  131: 479–491.  <ul class="find" data-citedArticleID="1010056"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data.&amp;auth=&amp;atitle=Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0001088-Excoffier2" id="pone.0001088-Excoffier2"></a>Excoffier L, Laval G, Schneider S (2005) Arlequin ver. 3.0: an integrated software package for population genetics data analysis. Evol Bioinf Online  1: 47–50.  <ul class="find" data-citedArticleID="1010058"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Arlequin+ver.+3.0%3A+an+integrated+software+package+for+population+genetics+data+analysis.&amp;auth=&amp;atitle=Arlequin+ver.+3.0%3A+an+integrated+software+package+for+population+genetics+data+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arlequin+ver.+3.0%3A+an+integrated+software+package+for+population+genetics+data+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arlequin+ver.+3.0%3A+an+integrated+software+package+for+population+genetics+data+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0001088-Raymond1" id="pone.0001088-Raymond1"></a>Raymond M, Rousset F (1995) GENEPOP (version 1.2): population genetics software for exact tests and ecumenicism. J Hered  86: 248–249.  <ul class="find" data-citedArticleID="1010148"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=GENEPOP+%28version+1.2%29%3A+population+genetics+software+for+exact+tests+and+ecumenicism.&amp;auth=&amp;atitle=GENEPOP+%28version+1.2%29%3A+population+genetics+software+for+exact+tests+and+ecumenicism." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GENEPOP+%28version+1.2%29%3A+population+genetics+software+for+exact+tests+and+ecumenicism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GENEPOP+%28version+1.2%29%3A+population+genetics+software+for+exact+tests+and+ecumenicism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0001088-Nei2" id="pone.0001088-Nei2"></a>Nei M, Li WH (1979) Mathematical model for studying genetic variation in terms of restriction endonucleases. Proc Natl Acad Sci U S A  76: 5269–5273.  <ul class="find" data-citedArticleID="1010130" data-doi="10.1073/pnas.76.10.5269"><li><a href="http://dx.doi.org/10.1073/pnas.76.10.5269" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mathematical+model+for+studying+genetic+variation+in+terms+of+restriction+endonucleases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mathematical+model+for+studying+genetic+variation+in+terms+of+restriction+endonucleases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0001088-Slatkin1" id="pone.0001088-Slatkin1"></a>Slatkin M (1993) Isolation by distance in equilibrium and non-equilibrium populations. Evolution  47: 264–279.  <ul class="find" data-citedArticleID="1010194" data-doi="10.2307/2410134"><li><a href="http://dx.doi.org/10.2307/2410134" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+by+distance+in+equilibrium+and+non-equilibrium+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+by+distance+in+equilibrium+and+non-equilibrium+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0001088-Rousset1" id="pone.0001088-Rousset1"></a>Rousset F (1997) Genetic differentiation and estimation of gene flow from <em>F</em>-statistics under isolation by distance. Genetics  145: 1219–1228.  <ul class="find" data-citedArticleID="1010174"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+differentiation+and+estimation+of+gene+flow+from+F-statistics+under+isolation+by+distance.&amp;auth=&amp;atitle=Genetic+differentiation+and+estimation+of+gene+flow+from+F-statistics+under+isolation+by+distance." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+differentiation+and+estimation+of+gene+flow+from+F-statistics+under+isolation+by+distance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+differentiation+and+estimation+of+gene+flow+from+F-statistics+under+isolation+by+distance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.XML" value="187384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.PDF" value="661384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g001.PNG_L" value="1317565"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g001.PNG_M" value="150861"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g001.PNG_S" value="9844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g001.TIF" value="1750960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g001.PNG_I" value="80653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g002.PNG_L" value="2874043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g002.PNG_M" value="131361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g002.PNG_S" value="14464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g002.TIF" value="4618982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g002.PNG_I" value="94318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g003.PNG_L" value="1098910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g003.PNG_M" value="74208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g003.PNG_S" value="10430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g003.TIF" value="2477094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g003.PNG_I" value="26535"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g004.PNG_L" value="215915"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g004.PNG_M" value="74832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g004.PNG_S" value="7125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g004.TIF" value="819650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g004.PNG_I" value="27295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t001.PNG_L" value="95009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t001.PNG_M" value="61095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t001.PNG_S" value="9312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t001.TIF" value="437412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t001.PNG_I" value="19109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t002.PNG_L" value="45260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t002.PNG_M" value="58924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t002.PNG_S" value="11111"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t002.TIF" value="217762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t002.PNG_I" value="18831"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g005.PNG_L" value="794728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g005.PNG_M" value="123946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g005.PNG_S" value="11472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g005.TIF" value="1922536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g005.PNG_I" value="45688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t003.PNG_L" value="137776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t003.PNG_M" value="157266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t003.PNG_S" value="20248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t003.TIF" value="650770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t003.PNG_I" value="73544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t004.PNG_L" value="131005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t004.PNG_M" value="124880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t004.PNG_S" value="17465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t004.TIF" value="645944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.t004.PNG_I" value="70214"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g006.PNG_L" value="100721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g006.PNG_M" value="44601"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g006.PNG_S" value="7284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g006.TIF" value="281534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.g006.PNG_I" value="17973"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001088.s001.DOC" value="57344"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001088&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001088" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001088&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001088&volume=&issue=&title=Molecular Variation at a Candidate Gene Implicated in the Regulation of Fire Ant Social Behavior&author_name=Dietrich%20Gotzek%2C%20D.%20DeWayne%20Shoemaker%2C%20Kenneth%20G.%20Ross&start_page=1&end_page=16" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001088" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001088" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001088" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001088&amp;t=Molecular%20Variation%20at%20a%20Candidate%20Gene%20Implicated%20in%20the%20Regulation%20of%20Fire%20Ant%20Social%20Behavior" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001088&title=Molecular%20Variation%20at%20a%20Candidate%20Gene%20Implicated%20in%20the%20Regulation%20of%20Fire%20Ant%20Social%20Behavior&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001088&amp;title=Molecular%20Variation%20at%20a%20Candidate%20Gene%20Implicated%20in%20the%20Regulation%20of%20Fire%20Ant%20Social%20Behavior" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001088" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001088&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Molecular Variation at a Candidate Gene Implicated in the Regulation of Fire Ant Social Behavior';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001088';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Molecular%20Variation%20at%20a%20Candidate%20Gene%20Implicated%20in%20the%20Regulation%20of%20Fire%20Ant%20Social%20Behavior http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001088" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001088" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="25010"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amino+acid+substitution%22" title="Search for articles in the subject area:'Amino acid substitution'"><div class="flagText">Amino acid substitution</div></a>
              <div data-categoryid="32897" data-articleid="25010"
                   data-categoryname="Amino acid substitution"
                   class="flagImage" title="Flag 'Amino acid substitution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="25010"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25010"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="25010"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotides%22" title="Search for articles in the subject area:'Nucleotides'"><div class="flagText">Nucleotides</div></a>
              <div data-categoryid="17443" data-articleid="25010"
                   data-categoryname="Nucleotides"
                   class="flagImage" title="Flag 'Nucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="25010"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="25010"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1966'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6652'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1031&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=757">Phylogeography question</a><br>Posted by mdunford</p>
      <p><a href="/annotation/listThread.action?root=4263">b-allele pattern question</a><br>Posted by mdunford</p>
      <p><a href="/annotation/listThread.action?root=15495">But why?</a><br>Posted by GregLaden</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
