

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Genomic Background Based Method for Association Analysis in Related Individuals</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001274"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001274" />

    <meta name="citation_title" content="A Genomic Background Based Method for Association Analysis in Related Individuals"/>
    <meta itemprop="name" content="A Genomic Background Based Method for Association Analysis in Related Individuals"/>

      <meta name="citation_author" content="Najaf Amin"/>
            <meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, Erasmus University Medical Center (MC) Rotterdam, Rotterdam, The Netherlands"/>
      <meta name="citation_author" content="Cornelia M. van Duijn"/>
            <meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, Erasmus University Medical Center (MC) Rotterdam, Rotterdam, The Netherlands"/>
      <meta name="citation_author" content="Yurii S. Aulchenko"/>
            <meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, Erasmus University Medical Center (MC) Rotterdam, Rotterdam, The Netherlands"/>

    <meta name="citation_date" content="2007/12/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001274.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1274"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Mapping quantitative trait loci in humans: achievements and limitations.; citation_author=PP Majumder; citation_author=S Ghosh; citation_journal_title=J Clin Invest; citation_volume=115; citation_number=1; citation_pages=1419-1424; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Role of the apolipoprotein E polymorphism in determining normal plasma lipid and lipoprotein variation.; citation_author=CF Sing; citation_author=J Davignon; citation_journal_title=Am J Hum Genet; citation_volume=37; citation_number=2; citation_pages=268-285; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Heritabilities, apolipoprotein E, and effects of inbreeding on plasma lipids in a genetically isolated population: the Erasmus Rucphen Family Study.; citation_author=A Isaacs; citation_author=FA Sayed-Tabatabaei; citation_author=YS Aulchenko; citation_author=MC Zillikens; citation_author=EJ Sijbrands; citation_journal_title=Eur J Epidemiol; citation_volume=22; citation_number=3; citation_pages=99-105; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The future of genetic studies of complex human diseases.; citation_author=N Risch; citation_author=K Merikangas; citation_journal_title=Science; citation_volume=273; citation_number=4; citation_pages=1516-1517; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A general test of association for quantitative traits in nuclear families.; citation_author=GR Abecasis; citation_author=LR Cardon; citation_author=WO Cookson; citation_journal_title=Am J Hum Genet; citation_volume=66; citation_number=5; citation_pages=279-292; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Family-based tests for associating haplotypes with general phenotype data: application to asthma genetics.; citation_author=S Horvath; citation_author=X Xu; citation_author=SL Lake; citation_author=EK Silverman; citation_author=ST Weiss; citation_journal_title=Genet Epidemiol; citation_volume=26; citation_number=6; citation_pages=61-69; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Power and design considerations for a general class of family-based association tests: quantitative traits.; citation_author=C Lange; citation_author=DL DeMeo; citation_author=NM Laird; citation_journal_title=Am J Hum Genet; citation_volume=71; citation_number=7; citation_pages=1330-1341; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The use of measured genotype information in the analysis of quantitative phenotypes in man. I. Models and analytical methods.; citation_author=E Boerwinkle; citation_author=R Chakraborty; citation_author=CF Sing; citation_journal_title=Ann Hum Genet; citation_volume=50; citation_number=8; citation_pages=181-194; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Association testing with Mendel.; citation_author=K Lange; citation_author=JS Sinsheimer; citation_author=E Sobel; citation_journal_title=Genet Epidemiol; citation_volume=29; citation_number=9; citation_pages=36-50; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The quantitative trait linkage disequilibrium test: a more powerful alternative to the quantitative transmission disequilibrium test for use in the absence of population stratification.; citation_author=LM Havill; citation_author=TD Dyer; citation_author=DK Richardson; citation_author=MC Mahaney; citation_author=J Blangero; citation_journal_title=BMC Genet; citation_volume=6; citation_number=10; citation_pages=S91; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genomewide rapid association using mixed model and regression: a fast and simple method for genomewide pedigree-based quantitative trait loci association analysis.; citation_author=YS Aulchenko; citation_author=DJ Koning; citation_author=C Haley; citation_journal_title=Genetics; citation_volume=177; citation_number=11; citation_pages=577-585; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A variant in CDKAL1 influences insulin response and risk of type 2 diabetes.; citation_author=V Steinthorsdottir; citation_author=G Thorleifsson; citation_author=I Reynisdottir; citation_author=R Benediktsson; citation_author=T Jonsdottir; citation_journal_title=Nat Genet; citation_volume=39; citation_number=12; citation_pages=770-775; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genomic control for association studies.; citation_author=B Devlin; citation_author=K Roeder; citation_journal_title=Biometrics; citation_volume=55; citation_number=13; citation_pages=997-1004; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Association studies for quantitative traits in structured populations.; citation_author=SA Bacanu; citation_author=B Devlin; citation_author=K Roeder; citation_journal_title=Genet Epidemiol; citation_volume=22; citation_number=14; citation_pages=78-93; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A genome-wide association study identifies novel risk loci for type 2 diabetes.; citation_author=R Sladek; citation_author=G Rocheleau; citation_author=J Rung; citation_author=C Dina; citation_author=L Shen; citation_journal_title=Nature; citation_volume=445; citation_number=15; citation_pages=881-885; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Population structure, differential bias and genomic control in a large-scale, case-control association study.; citation_author=DG Clayton; citation_author=NM Walker; citation_author=DJ Smyth; citation_author=R Pask; citation_author=JD Cooper; citation_journal_title=Nat Genet; citation_volume=37; citation_number=16; citation_pages=1243-1246; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Estimation of the inbreeding coefficient through use of genomic data.; citation_author=AL Leutenegger; citation_author=B Prum; citation_author=E Genin; citation_author=C Verny; citation_author=A Lemainque; citation_journal_title=Am J Hum Genet; citation_volume=73; citation_number=17; citation_pages=516-523; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=ASReml user guide release 1.0.; citation_author=AR Gilmour; citation_author=BJ Gogel; citation_author=BR Cullis; citation_author=SJ Welham; citation_author=R Thompson; citation_number=18; citation_pages=267; citation_date=2002; citation_publisher=NSW Agriculture; " />
      <meta name="citation_reference" content="citation_title=GenABEL: an R library for genome-wide association analysis.; citation_author=YS Aulchenko; citation_author=S Ripke; citation_author=A Isaacs; citation_author=CM van Duijn; citation_journal_title=Bioinformatics; citation_volume=23; citation_number=19; citation_pages=1294-1296; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The effect of genetic drift in a young genetically isolated population.; citation_author=LM Pardo; citation_author=I MacKay; citation_author=B Oostra; citation_author=CM van Duijn; citation_author=YS Aulchenko; citation_journal_title=Ann Hum Genet; citation_volume=69; citation_number=20; citation_pages=288-295; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Coalescent-based association mapping and fine mapping of complex trait loci.; citation_author=S Zollner; citation_author=JK Pritchard; citation_journal_title=Genetics; citation_volume=169; citation_number=21; citation_pages=1071-1092; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Linkage disequilibrium mapping via cladistic analysis of single-nucleotide polymorphism haplotypes.; citation_author=C Durrant; citation_author=KT Zondervan; citation_author=LR Cardon; citation_author=S Hunt; citation_author=P Deloukas; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=22; citation_pages=35-43; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001274" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Genomic Background Based Method for Association Analysis in Related Individuals"/>
    <meta name="twitter:description" content="BackgroundFeasibility of genotyping of hundreds and thousands of single nucleotide polymorphisms (SNPs) in thousands of study subjects have triggered the need for fast, powerful, and reliable methods for genome-wide association analysis. Here we consider a situation when study participants are genetically related (e.g. due to systematic sampling of families or because a study was performed in a genetically isolated population). Of the available methods that account for relatedness, the Measured Genotype (MG) approach is considered the &lsquo;gold standard&rsquo;. However, MG is not efficient with respect to time taken for the analysis of genome-wide data. In this context we proposed a fast two-step method called Genome-wide Association using Mixed Model and Regression (GRAMMAR) for the analysis of pedigree-based quantitative traits. This method certainly overcomes the drawback of time limitation of the measured genotype (MG) approach, but pays in power. One of the major drawbacks of both MG and GRAMMAR, is that they crucially depend on the availability of complete and correct pedigree data, which is rarely available.MethodologyIn this study we first explore type 1 error and relative power of MG, GRAMMAR, and Genomic Control (GC) approaches for genetic association analysis. Secondly, we propose an extension to GRAMMAR i.e. GRAMMAR-GC. Finally, we propose application of GRAMMAR-GC using the kinship matrix estimated through genomic marker data, instead of (possibly missing and/or incorrect) genealogy.ConclusionThrough simulations we show that MG approach maintains high power across a range of heritabilities and possible pedigree structures, and always outperforms other contemporary methods. We also show that the power of our proposed GRAMMAR-GC approaches to that of the &lsquo;gold standard&rsquo; MG for all models and pedigrees studied. We show that this method is both feasible and powerful and has correct type 1 error in the context of genome-wide association analysis in related individuals."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001274.g002"/>

  <meta property="og:title" content="A Genomic Background Based Method for Association Analysis in Related Individuals" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001274" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9680'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7701'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6798&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001274">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001274" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001274&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Genomic Background Based Method for Association Analysis in Related Individuals
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Najaf Amin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Epidemiology and Biostatistics, Erasmus University Medical Center (MC) Rotterdam, Rotterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Cornelia M. van Duijn, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Epidemiology and Biostatistics, Erasmus University Medical Center (MC) Rotterdam, Rotterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yurii S. Aulchenko
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:i.aoultchenko@erasmusmc.nl">i.aoultchenko@erasmusmc.nl</a></p>

                <p>Affiliation:
                  Department of Epidemiology and Biostatistics, Erasmus University Medical Center (MC) Rotterdam, Rotterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 05, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001274</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001274" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001274" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001274" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001274" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001274-t001" data-doi="info:doi/10.1371/journal.pone.0001274" data-uri="info:doi/10.1371/journal.pone.0001274.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001274-g001" data-doi="info:doi/10.1371/journal.pone.0001274" data-uri="info:doi/10.1371/journal.pone.0001274.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001274-g002" data-doi="info:doi/10.1371/journal.pone.0001274" data-uri="info:doi/10.1371/journal.pone.0001274.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001274">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001274" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Feasibility of genotyping of hundreds and thousands of single nucleotide polymorphisms (SNPs) in thousands of study subjects have triggered the need for fast, powerful, and reliable methods for genome-wide association analysis. Here we consider a situation when study participants are genetically related (e.g. due to systematic sampling of families or because a study was performed in a genetically isolated population). Of the available methods that account for relatedness, the Measured Genotype (MG) approach is considered the ‘gold standard’. However, MG is not efficient with respect to time taken for the analysis of genome-wide data. In this context we proposed a fast two-step method called Genome-wide Association using Mixed Model and Regression (GRAMMAR) for the analysis of pedigree-based quantitative traits. This method certainly overcomes the drawback of time limitation of the measured genotype (MG) approach, but pays in power. One of the major drawbacks of both MG and GRAMMAR, is that they crucially depend on the availability of complete and correct pedigree data, which is rarely available.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>In this study we first explore type 1 error and relative power of MG, GRAMMAR, and Genomic Control (GC) approaches for genetic association analysis. Secondly, we propose an extension to GRAMMAR i.e. GRAMMAR-GC. Finally, we propose application of GRAMMAR-GC using the kinship matrix estimated through genomic marker data, instead of (possibly missing and/or incorrect) genealogy.</p>


<h3>Conclusion</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Through simulations we show that MG approach maintains high power across a range of heritabilities and possible pedigree structures, and always outperforms other contemporary methods. We also show that the power of our proposed GRAMMAR-GC approaches to that of the ‘gold standard’ MG for all models and pedigrees studied. We show that this method is both feasible and powerful and has correct type 1 error in the context of genome-wide association analysis in related individuals.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Amin N, van Duijn CM, Aulchenko YS (2007) A Genomic Background Based Method for Association Analysis in Related Individuals. PLoS ONE 2(12):
          e1274.
            doi:10.1371/journal.pone.0001274</p><p><strong>Academic Editor: </strong>Peter Heutink, Vrije Universiteit Medical Centre, Netherlands</p><p><strong>Received:</strong> August 20, 2007; <strong>Accepted:</strong> November 7, 2007; <strong>Published:</strong> December 5, 2007</p><p><strong>Copyright:</strong> © 2007 Amin et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by European Special Population Network grant (FP6) and a joint grant from Netherlands organization for scientific research (NWO) and Russian Foundation for Basic Research (RFBR). YSA was supported by a grant from "Stichting MS". NA received support from Higher Education Commission of Pakistan.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Most of the complex genetic diseases have risk factors that are quantitative in nature. For instance, cholesterol level is a risk factor for myocardial infarction, Body mass index is a risk factor for type 2 diabetes. These quantitative traits (QTs) often have strong genetic determinants. It is therefore, of considerable interest to map the genes underlying QTs <a href="#pone.0001274-Majumder1">[1]</a>. For most QTs relevant for human health and disease a large proportion–ranging from 30 to 80%–of variation is explained by genetic factors. Multiple genes are expected to contribute to this variation. The proportion of variation explained by a single gene is expected, however, to be small (less than 5%). For example, one of most known quantitative trait loci (QTLs), APOE is strongly and consistently associated with increased total cholesterol levels. Yet it explains only about 2–5% of the variation of this trait <a href="#pone.0001274-Sing1">[2]</a>, <a href="#pone.0001274-Isaacs1">[3]</a>. When effects of specific common variants are expected to be small, association analysis provides a powerful approach to identify the gene compared to linkage analysis <a href="#pone.0001274-Risch1">[4]</a>. Genome-wide association analysis is a powerful tool to disentangle the complexity of quantitative traits, even in family based studies.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>For pedigree-based association analysis several methods and software packages have been developed that utilize information about transmission of alleles, such as the orthogonal test for within-family variation <a href="#pone.0001274-Abecasis1">[5]</a> and family-based association test <a href="#pone.0001274-Horvath1">[6]</a>, <a href="#pone.0001274-Lange1">[7]</a>. As these methods analyze within-family variation, they are robust to population stratification. However, these methods ignore a large proportion of information provided by the between-family variation leaving room for improvement.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A conventional polygenic model of inheritance, which is a statistical genetics' “gold standard”, is a mixed model<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <em>μ</em> is the overall mean, <em>G</em> is the vector of random polygenic effect, and <em>e</em> is the vector of random residuals. This model may be extended to study association by including a <em>kg</em> term, where <em>k</em> is the marker genotype effect, and <em>g</em> is the vector of genotypes. Such a model is known as the measured genotype (MG) model <a href="#pone.0001274-Boerwinkle1">[8]</a>. The MG approach, implemented using (restricted) maximum likelihood, is a powerful tool for the analysis of QTs when ethnic stratification can be ignored <a href="#pone.0001274-Lange2">[9]</a>, <a href="#pone.0001274-Havill1">[10]</a> and pedigrees are small or when there are few dozens or hundreds of candidate polymorphisms to be tested. This approach, however, is not efficient in terms of computation time. This hampers the application of MG in genome-wide association analysis.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>We proposed a fast two-step approximation to MG, a Genome Wide Rapid Association using Mixed Model and Regression (GRAMMAR) <a href="#pone.0001274-Aulchenko1">[11]</a> . In the first step the individual environmental residuals are estimated, using additive polygenic model. Then the test of association is performed using these familial correlation-free residuals with a rapid least squares or score method. Though the two-step method is indeed computationally efficient and outperforms family based approaches like FBAT and QTDT in terms of power and speed, it loses power compared to MG <a href="#pone.0001274-Aulchenko1">[11]</a>. The test becomes increasingly conservative and less powerful with the increase in the number of large full-sib families and increased heritability of the trait. Interestingly, empirical power of GRAMMAR is very close to that of MG.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Both classical MG and GRAMMAR approaches rely heavily on the availability, completeness, and correctness of genealogical information. When these assumptions are violated, the most likely outcome is inflation in type 1 error. Practically, genealogical information may often be available only for a limited number of generations, it may be inaccurate, and it may become increasingly inaccurate back in time. This may be taken as an argument for application of underpowered TDT-like methods to avoid false positive or negative results.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>We and others reason that genomic data can be used to correct for the (only partly observed) true genealogy. With the new array technologies, large numbers of markers can be typed over the genome. These provide information on ‘genomic background’, which can be used to infer population (sub)structure and relations between study participants, which is classically done using the genealogy. In a recent study of type 2 diabetes, genomic control (GC) was applied to control for relatedness among cases and controls from Icelandic population <a href="#pone.0001274-Steinthorsdottir1">[12]</a>, However, the type 1 and 2 errors of GC were not yet systematically investigated in the context of pedigree data analysis. In this study we aim to exploit the ideas of genomic background to extend our work on family based association <a href="#pone.0001274-Aulchenko1">[11]</a> and determine how powerful and efficient the method for genome-wide association analysis of QTs in samples of related individuals is.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Only minor proportion of markers in a genome-wide association study is expected to be truly associated with an analysis trait, and a vast majority of the markers may be thought of as realizations under the null hypothesis and can be used to characterize the null distribution of the test statistics. This idea follows that of Genomic Control (GC) method <a href="#pone.0001274-Devlin1">[13]</a>, which was introduced in the context of association analysis in population-based studies, where population stratification and cryptic relatedness may be present.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>Following Devlin and Roeder <a href="#pone.0001274-Devlin1">[13]</a>, <a href="#pone.0001274-Bacanu1">[14]</a> we suggest estimation of the test statistic inflation factor λ by regressing the trait on <em>N</em> loci. From each regression analysis, estimate <em>T<sub>i</sub><sup>2</sup> = β<sub>i</sub><sup>2</sup>/Var(β<sub>i</sub>)</em>, where <em>β<sub>i</sub></em> is the effect of the <em>i-</em>th SNP (<em>i</em> from 1 to <em>N</em>) and <em>Var(β<sub>i</sub>)</em> is the variance of the estimate. Inflation factor is estimated as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e002&amp;representation=PNG" class="inline-graphic"></span><br>where 0.456 is the median of the <em>χ<sub>1</sub><sup>2</sup></em> distribution with a non-central variance equal to <em>φ</em>. The number of loci used, <em>N</em>, in a genome-wide association study is typically reflecting all loci investigated or the ones generating lowest 95% of test statistics <a href="#pone.0001274-Sladek1">[15]</a>, <a href="#pone.0001274-Clayton1">[16]</a>. It should be noted that the value of λ in conventional GC is constrained to be greater than one as values less than one have no theoretical meaning.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>We propose use of GC to correct for conservativeness of the GRAMMAR approach outlined earlier. This method which we call GRAMMAR-GC involves three steps: (a) trait heritability is estimated by using trait and pedigree data using the following mixed model<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e003&amp;representation=PNG" class="inline-graphic"></span><br>where <em>β<sub>j</sub></em> is the effect of <em>j</em><sup>th</sup> covariate , c<em><sub>ij</sub></em> is the value of <em>j<sup>th</sup></em> covariate and <em>μ</em>, <em>G</em>, and <em>e</em> are defined earlier. And environmental residuals are estimated as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e004&amp;representation=PNG" class="inline-graphic"></span><br>(b) the markers are tested for association with these residuals using simple linear regression<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e005&amp;representation=PNG" class="inline-graphic"></span><br>where <em>k</em> and <em>g</em> are defined earlier</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>(c) GC procedure is applied to correct the test statistic. The deflation factor ζ is estimated by regressing residuals from step (a) on each of the <em>k</em> null loci and from each regression analysis <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e006&amp;representation=PNG" class="inline-graphic"></span> is estimated, where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e007&amp;representation=PNG" class="inline-graphic"></span> is the effect of the <em>k<sup>th</sup></em> SNP. The deflation factor ζ is estimated as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e008&amp;representation=PNG" class="inline-graphic"></span>. Then <em>T<sup>2</sup>/</em><span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e009&amp;representation=PNG" class="inline-graphic"></span> is compared with <em>χ</em><sup>2</sup><sub>(1)</sub> to determine whether the locus is significantly associated with the QT <a href="#pone.0001274-Bacanu1">[14]</a>.</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>Steps (a) and (b) comprise the original GRAMMAR approach, leading to a conservative test. In step (c), GC is used to estimate the deflation factor ζ. This deflation factor is estimated in exactly the same way Bacanu et al. <a href="#pone.0001274-Bacanu1">[14]</a> estimate inflation factor λ for quantitative traits. The only difference is that ζ&lt;1 in contrast to λ in conventional GC methods which is constrained to be &gt;1. This difference comes from the fact that residuals from step (a) are regressed on null loci to obtain the estimate of ζ instead of original trait data as in conventional GC.</p>
<a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>The original GRAMMAR relies on the availability of a precise and complete pedigree structure for heritablity estimation in step (a). This can, however, be done by using kinship coefficients estimated from genomic data where the genomic estimate of kinship for a pair of individuals <em>i</em> and <em>j</em> is obtained using the formula <a href="#pone.0001274-Leutenegger1">[17]</a><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e010&amp;representation=PNG" class="inline-graphic"></span><br>where <em>g<sub>ik</sub></em> is the genotype of the <em>i</em>-th person at the <em>k</em>-th SNP of the (coded as 0, 1/2 and 1, for rare allele homozygote, heterozygote and common homozygote, respectively), <em>p<sub>k</sub></em> is the frequency of the major allele, and <em>n</em> is the number of SNPs used for kinship estimation.</p>
<a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>Heritability is then estimated by maximizing the log-likelihood of the data<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e011&amp;representation=PNG" class="inline-graphic"></span><br>where <em>y</em> is the vector of trait values, <em>μ</em> is the mean and <em>Σ = Φσ<sub>G</sub><sup>2</sup>+Iσ<sub>e</sub><sup>2</sup></em> is the variance-covariance matrix. Here, <em>Φ</em> is the relationship matrix whose elements are defined as 2 <em>f<sub>ij</sub></em>, <em>σ<sub>G</sub><sup>2</sup></em> is the additive genetic variance due to polygenes, <em>I</em> is the identity matrix and <em>σ<sub>e</sub><sup>2</sup></em> is the residual variance. Trait environmental residuals were obtained as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.e012&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p>To avoid confusion we refer to the new method as Pedigree GRAMMAR-GC (PedGR-GC) when in step (a) heritability is estimated from the genealogy, and Genomic GRAMMAR-GC (GenGR-GC) when the heritability is estimated from the genomic data. For PedGR-GC, environmental residuals were estimated using ASReml <a href="#pone.0001274-Gilmour1">[18]</a>. All other computations were performed using freely available R software (<a href="http://www.r-project.org">http://www.r-project.org</a>); computations associated with GenGR-GC were facilitated by GenABEL package for R <a href="#pone.0001274-Aulchenko2">[19]</a>, implementing procedures to compute genomic kinship, maximize polygenic models and compute the residuals.</p>

<h4>Pedigrees used</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To investigate type 1 error and power of the proposed methods we used three different pedigree structures representing three different study scenarios. For example, Nuclear pedigrees (NP) simulated a study performed in the outbred population, the Erasmus Rucphen Family study (ERF) population is a study of a genetically isolated population and Idealized Pig Population (IPP) simulates a livestock population.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p><strong>NP:</strong> 337 sib-trios each having 3 phenotyped and genotyped siblings; in total, 1011 individuals were available for the analysis. The founders in each pedigree were assumed to be unrelated.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p><strong>ERF:</strong> 1010 phenotyped and genotyped individuals all related to each other in a single large complex pedigree of about 10,000 individuals. The pedigree extends up to 23 generations and contains thousands of loops. The phenotyped individuals are a part of Erasmus Rucphen Family (ERF) study, performed in a young genetically isolated Dutch population <a href="#pone.0001274-Pardo1">[20]</a>.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p><strong>IPP:</strong> idealized pig population, consisting of 10 sires, each mated with 10 dams, nine of which have 10 and one 11 measured offspring. Thus each sire has 101 half-sib offspring in families of 10 full-sibs. In total 1010 phenoytped individuals were available for the analysis.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>Genetic data was simulated using each of these pedigrees under several models. The SNP that was analyzed for association had a minor allele frequency of 10%. For studying type 1 error this SNP was not associated with the trait while for studying power this SNP explained 1, 2, or 3% of the total trait variation and acted in an additive manner. The total heritability of the trait was set to 30, 50, and 80% and this included the variation due to the QTL studied. To enable genomic control, we also simulated 200 unlinked SNPs for each realization.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>These pedigrees and models were used to evaluate original GRAMMAR and thus we can directly compare type 1 error and power of the suggested methods to these evaluated by Aulchenko et al <a href="#pone.0001274-Aulchenko1">[11]</a>, namely, MG, DFS (linear regression which does not take family structure into account), QTDT and FBAT.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p><a href="#pone-0001274-t001">Table 1</a> shows the 95<sup>th</sup> percentile of the test statistic and type 1 error (proportion of simulations that resulted in a χ<sup>2</sup>≥3.84 and χ<sup>2</sup>≥6.64, corresponding to tabulated α = 0.05 and 0.01 respectively) for GC and PedGR-GC and GRAMMAR. For GC and PedGR-GC type 1 error is close to the nominal α while GRAMMAR is conservative and this conservativeness increases with the increase in the number of sibships and the heritability of the trait.</p>
<div class="figure" id="pone-0001274-t001"><div class="img"><a name="pone-0001274-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001274" data-uri="info:doi/10.1371/journal.pone.0001274.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001274.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001274.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001274.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001274.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001274.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001274.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001274.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001274.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>95<sup>th</sup> percentiles of the distribution and type 1 error for PedGR-GC, GC and GRAMMAR.</span></strong></p><span>doi:10.1371/journal.pone.0001274.t001</span></div><a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Supplementary <a href="#pone.0001274.s001">Table S1</a> and <a href="#pone-0001274-g001">Figure 1</a> illustrate the power of the proposed (GC and PedGR-GC) and previous methods (MG and GRAMMAR). In the <a href="#pone-0001274-g001">Figure 1</a>, power to reach χ<sup>2</sup>≥6.64 (α = 0.01) is depicted with circles. From the available evaluation points we also estimated the slope of linear regression of non-centrality parameter on proportion of variance explained and used this slope to derive power curves.</p>
<div class="figure" id="pone-0001274-g001"><div class="img"><a name="pone-0001274-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001274" data-uri="info:doi/10.1371/journal.pone.0001274.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001274.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001274.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001274.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001274.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001274.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001274.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001274.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001274.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Power of MG (red line), GRAMMAR (green line), PedGR-GC (blue dashed line), and GC (pink dashed line) to detect association under different heritability models and pedigree structures.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>The three rows show the power under different heritability models (from 30% to 80%) and the three columns show power achieved in different pedigrees namely nuclear pedigrees (NP), Erasmus Rucphen Family (ERF), and idealized pig population (IPP). The y-axis of each panel shows power while the x-axis shows the proportion of variance explained by the QTL under study. The red (for MG), green (for GRAMMAR), blue (for PedGR-GC), and pink (for GC) circles show the empirical power estimates. The power estimates are based on α = 0.01. The empirical power estimates are based on 1000 simulations for NP, and IPP, and 100 simulations for ERF.</p>
<span>doi:10.1371/journal.pone.0001274.g001</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p><a href="#pone-0001274-g001">Figure 1</a> shows that the power of PedGR-GC (blue dashed line) is very close to the power of the ‘gold standard’ measured genotype approach (red line) for all scenarios. These two methods appear to be the most powerful of all methods for all pedigree structures and genetic models evaluated. The power of GC (pink dashed line) is close to that of MG and PedGR-GC when the heritability is low but its' power declines when the heritability of the trait increases or when pedigrees with large number of full-sib families (IPP) are investigated. GRAMMAR (green line) performs similar to GC in a sample of nuclear families and in the ERF sample, but is more powerful when IPP pedigree is investigated.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>To study the potential of described methods on genome-wide scale we used 695 people who are a part of the ERF pedigree and who were genotyped using Illumina 6K SNP linkage panel. Based on pedigree records, the 695 people formed 471 pairs of first-, 311 pairs of second-, 681 pairs of third- and 1,105 pairs of forth-degree relatives and 223,578 pairs of more distant relationship.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>We generated 500 replicas for each of the models assuming total trait heritability of 30, 50 and 80%. In each replica, we selected five hundred random SNPs each explaining equal proportion of variance amounting to the total heritability minus 4%, and one random SNP explaining 4% of the phenotypic variance. The analysis trait was obtained as a sum of the SNP effects and a random number from the Normal distribution with mean zero and variance 0.7, 0.5 or 0.2 for total trait heritabilities of 30, 50 and 80%, respectively.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Type one error was estimated as the proportion of non-associated SNPs (&gt;2.5 million tests in total) showing association P-value of 5% or less. The statistical power was estimated as the proportion of replicas in which the SNP explaining 4% of variance passed genome-wide significant threshold (Bonferroni-corrected P-value 0.05/5524 = 9×10<sup>−6</sup>).</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>For analysis, we used GC, PedGR-GC and GenGR-GC. For GenGR-GC, the kinship matrix used was estimated from genomic information on 5524 autosomal SNPs typed in 695 members of the ERF study.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>All methods showed a genome-wide type 1 error which was very close, but lower than the pre-specified threshold of 5% (<a href="#pone-0001274-g002">Figure 2A</a>). The methods tended to be more conservative at higher heritabilities. These results are consistent with the observations of others, that GC is conservative, and can be explained by the fact that all SNPs, some of which were associated with the trait, are used to estimate the null distribution of the test statistic.</p>
<div class="figure" id="pone-0001274-g002"><div class="img"><a name="pone-0001274-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001274" data-uri="info:doi/10.1371/journal.pone.0001274.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001274.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001274.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001274.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001274.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001274.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001274.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001274.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001274.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001274.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Type 1 error (A) and power (B) to achieve 5% genome-wide significance at the truly associated SNP in a study of 695 ERF people genotyped for 5524 autosomal SNPs.</span></strong></p><span>doi:10.1371/journal.pone.0001274.g002</span></div></div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>In this work we aimed to develop fast and powerful methods for genome-wide association analysis in samples of related individuals by exploiting the ideas of genomic background for correction of the distribution of the test statistics and for inferring the relation between study subjects. We show that methods, which exploit only genomic background, such as Genomic Control (GC) and GRAMMAR-GC using genomic kinship, are powerful and genome-wide feasible methods. Moreover, genomic GRAMMAR-GC, which infers genetic relations from genomic data, may be superior to the methods that use pedigree information in an exact manner.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Our simulations show that GC is a valid method to study data coming from samples of related individuals. GC can be a powerful tool for the analysis of pedigree based quantitative trait loci. It outperforms traditional family based approaches like QTDT and FBAT (cf. <a href="#pone-0001274-t001">Table 1</a> &amp; Table 2 of Aulchenko et. al.<a href="#pone.0001274-Aulchenko1">[11]</a>). The power of GC is close to that of the ‘gold standard’ measured genotype approach when trait heritability is low or moderate and human pedigrees are studied. However, the power of GC drops notably with high trait heritability and when pedigrees involve very large sibships, which are likely to be observed in animal pedigrees.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The results that GC was less powerful, than GRAMMAR-GC based methods, and tended to lose power at higher heritabilites (<a href="#pone-0001274-g002">Figure 2B</a>), are not surprising and are consistent with our previous findings <a href="#pone.0001274-Aulchenko1">[11]</a>. Interestingly enough in a study of real Genome-wide data in ERF pedigree, GenGR-GC was consistently more powerful than pedigree-based GRAMMAR-GC (PedGR-GC), and the power advantage became more pronounced at higher heritabilities.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>We proposed an extension to the GRAMMAR method <a href="#pone.0001274-Aulchenko1">[11]</a>, which increases its' power, maintains a nominal type 1 error and also does not require the precise knowledge of pedigree structure. Our method (GRAMMAR-GC) involves three steps which include removing the correlation from the data using relationship matrix estimated from either the pedigree or the genome (derivation of environmental residuals), using the correlation-free residuals from step 1 as the trait to perform association analysis, and then applying GC to correct the test statistic. We show through simulations that our proposed method performs very similar to the Measured Genotype (MG) approach with respect to type 1 error and power yet it is fast and feasible for genome-wide association analysis. By analysis of real genome-wide SNP data we showed that when the genomic data is used to estimate the relationship matrix (GenGR-GC) instead of the estimate obtained from genealogy, the power might be even improved.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>One of our conclusions is that in genome-wide association studies of related individuals genomic background based methods such as genomic GRAMMAR-GC should be preferred over the ones exploiting known pedigree structure, such as pedigree GRAMMAR-GC or MG approach. There are two reasons why we believe that GenGR-GC should be preferred over its' pedigree analog.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>First, errors in genealogy such as mis-specification of relations can lead to either false positives or negatives. Secondly, relationship coefficient computed from a pedigree is an expectation of the proportion of genome shared identical by descent (IBD) under the infinitesimal model, assuming infinite number of unlinked loci. The true proportion of genome shared, however, may deviate from this expectation <a href="#pone.0001274-Leutenegger1">[17]</a>. For example, for remote relatives there is a fair chance of not sharing any genomic loci IBD. We may speculate that kinship estimated based on marker data can reflect the true unobserved genomic sharing better then the expectation computed from even a correct pedigree. If this is true, under the polygenic model we should expect that methods based on kinship estimated from marker data will be more powerful than the methods estimating kinship from the pedigree. We however leave a more detailed investigation of effects of pedigree error and precision of genomic kinship estimates on type 1 error and power to future works.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>Another advantage of the GRAMMAR-GC is that the environmental residuals used for the analysis are free from familial correlations. Therefore the structure of the data becomes exchangeable. This means that permutation techniques may be applied to derive empirical measures of significance. In the analysis of data where adjacent SNPs are correlated due to linkage disequilibrium, thresholds set via permutation will account for these correlations and result in less stringent thresholds than those set by Bonferroni correction. This will become more and more important in the future, when denser marker maps with millions of SNPs will be applied to do association studies. Another attractive feature is that a range of new methods developed for classical “unrelated individuals” design can be applied to polygenic residuals obtained at the first stage of the analysis. In recent years, much progress was made in development of powerful methods and software which are robust to possible allelic heterogeneity through utilization of haplotype clustering and population genetic coalescence modeling <a href="#pone.0001274-Zollner1">[21]</a>, <a href="#pone.0001274-Durrant1">[22]</a>.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>Finally, GRAMMAR-GC is easily extendable: for example, it is easy to include covariates, interactions with sex and environment, gene-gene interactions and parent of origin effects.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p>To conclude, GRAMMAR-GC is a fast powerful approach for genome-wide association analysis of quantitative traits in samples of related individuals, which does not require precise knowledge of pedigree structure. This method is implemented as part of the GenABEL package, available at <a href="http://cran.r-project.org/">http://cran.r-project.org/</a>.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001274"></div><a name="pone.0001274.s001" id="pone.0001274.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001274.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Mean χ2 statistics and proportion of the test statistics ≥tabular critical value.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001274.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Prof. Tatiana Axenovich, Prof. Haley, Dr de Koning, Dr. Catherine Bourgain, Prof. Ober, and Dr. William Astle for valuable discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YA NA. Performed the experiments: YA NA. Analyzed the data: YA NA. Contributed reagents/materials/analysis tools: Cv YA NA. Wrote the paper: YA NA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001274-Majumder1" id="pone.0001274-Majumder1"></a>Majumder PP, Ghosh S (2005) Mapping quantitative trait loci in humans: achievements and limitations. J Clin Invest  115: 1419–1424.  <ul class="find" data-citedArticleID="1026862" data-doi="10.1172/jci24757"><li><a href="http://dx.doi.org/10.1172/jci24757" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+quantitative+trait+loci+in+humans%3A+achievements+and+limitations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+quantitative+trait+loci+in+humans%3A+achievements+and+limitations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001274-Sing1" id="pone.0001274-Sing1"></a>Sing CF, Davignon J (1985) Role of the apolipoprotein E polymorphism in determining normal plasma lipid and lipoprotein variation. Am J Hum Genet  37: 268–285.  <ul class="find" data-citedArticleID="1026868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Role+of+the+apolipoprotein+E+polymorphism+in+determining+normal+plasma+lipid+and+lipoprotein+variation.&amp;auth=&amp;atitle=Role+of+the+apolipoprotein+E+polymorphism+in+determining+normal+plasma+lipid+and+lipoprotein+variation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+apolipoprotein+E+polymorphism+in+determining+normal+plasma+lipid+and+lipoprotein+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+apolipoprotein+E+polymorphism+in+determining+normal+plasma+lipid+and+lipoprotein+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001274-Isaacs1" id="pone.0001274-Isaacs1"></a>Isaacs A, Sayed-Tabatabaei FA, Aulchenko YS, Zillikens MC, Sijbrands EJ, et al.  (2007) Heritabilities, apolipoprotein E, and effects of inbreeding on plasma lipids in a genetically isolated population: the Erasmus Rucphen Family Study. Eur J Epidemiol  22: 99–105.  <ul class="find" data-citedArticleID="1026854" data-doi="10.1007/s10654-006-9103-0"><li><a href="http://dx.doi.org/10.1007/s10654-006-9103-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heritabilities%2C+apolipoprotein+E%2C+and+effects+of+inbreeding+on+plasma+lipids+in+a+genetically+isolated+population%3A+the+Erasmus+Rucphen+Family+Study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heritabilities%2C+apolipoprotein+E%2C+and+effects+of+inbreeding+on+plasma+lipids+in+a+genetically+isolated+population%3A+the+Erasmus+Rucphen+Family+Study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001274-Risch1" id="pone.0001274-Risch1"></a>Risch N, Merikangas K (1996) The future of genetic studies of complex human diseases. Science  273: 1516–1517.  <ul class="find" data-citedArticleID="1026866" data-doi="10.1126/science.273.5281.1516"><li><a href="http://dx.doi.org/10.1126/science.273.5281.1516" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+future+of+genetic+studies+of+complex+human+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+future+of+genetic+studies+of+complex+human+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001274-Abecasis1" id="pone.0001274-Abecasis1"></a>Abecasis GR, Cardon LR, Cookson WO (2000) A general test of association for quantitative traits in nuclear families. Am J Hum Genet  66: 279–292.  <ul class="find" data-citedArticleID="1026832" data-doi="10.1086/302698"><li><a href="http://dx.doi.org/10.1086/302698" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+test+of+association+for+quantitative+traits+in+nuclear+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+test+of+association+for+quantitative+traits+in+nuclear+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001274-Horvath1" id="pone.0001274-Horvath1"></a>Horvath S, Xu X, Lake SL, Silverman EK, Weiss ST, et al.  (2004) Family-based tests for associating haplotypes with general phenotype data: application to asthma genetics. Genet Epidemiol  26: 61–69.  <ul class="find" data-citedArticleID="1026852" data-doi="10.1002/gepi.10295"><li><a href="http://dx.doi.org/10.1002/gepi.10295" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Family-based+tests+for+associating+haplotypes+with+general+phenotype+data%3A+application+to+asthma+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Family-based+tests+for+associating+haplotypes+with+general+phenotype+data%3A+application+to+asthma+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001274-Lange1" id="pone.0001274-Lange1"></a>Lange C, DeMeo DL, Laird NM (2002) Power and design considerations for a general class of family-based association tests: quantitative traits. Am J Hum Genet  71: 1330–1341.  <ul class="find" data-citedArticleID="1026856" data-doi="10.1086/344696"><li><a href="http://dx.doi.org/10.1086/344696" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Power+and+design+considerations+for+a+general+class+of+family-based+association+tests%3A+quantitative+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Power+and+design+considerations+for+a+general+class+of+family-based+association+tests%3A+quantitative+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001274-Boerwinkle1" id="pone.0001274-Boerwinkle1"></a>Boerwinkle E, Chakraborty R, Sing CF (1986) The use of measured genotype information in the analysis of quantitative phenotypes in man. I. Models and analytical methods. Ann Hum Genet  50: 181–194.  <ul class="find" data-citedArticleID="1026840" data-doi="10.1111/j.1469-1809.1986.tb01037.x"><li><a href="http://dx.doi.org/10.1111/j.1469-1809.1986.tb01037.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+measured+genotype+information+in+the+analysis+of+quantitative+phenotypes+in+man.+I.+Models+and+analytical+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+measured+genotype+information+in+the+analysis+of+quantitative+phenotypes+in+man.+I.+Models+and+analytical+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001274-Lange2" id="pone.0001274-Lange2"></a>Lange K, Sinsheimer JS, Sobel E (2005) Association testing with Mendel. Genet Epidemiol  29: 36–50.  <ul class="find" data-citedArticleID="1026858" data-doi="10.1002/gepi.20073"><li><a href="http://dx.doi.org/10.1002/gepi.20073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+testing+with+Mendel." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+testing+with+Mendel.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001274-Havill1" id="pone.0001274-Havill1"></a>Havill LM, Dyer TD, Richardson DK, Mahaney MC, Blangero J (2005) The quantitative trait linkage disequilibrium test: a more powerful alternative to the quantitative transmission disequilibrium test for use in the absence of population stratification. BMC Genet  6: Suppl 1S91.  <ul class="find" data-citedArticleID="1026850" data-doi="10.1186/1471-2156-6-s1-s91"><li><a href="http://dx.doi.org/10.1186/1471-2156-6-s1-s91" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+quantitative+trait+linkage+disequilibrium+test%3A+a+more+powerful+alternative+to+the+quantitative+transmission+disequilibrium+test+for+use+in+the+absence+of+population+stratification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+quantitative+trait+linkage+disequilibrium+test%3A+a+more+powerful+alternative+to+the+quantitative+transmission+disequilibrium+test+for+use+in+the+absence+of+population+stratification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001274-Aulchenko1" id="pone.0001274-Aulchenko1"></a>Aulchenko YS, Koning DJ, Haley C (2007) Genomewide rapid association using mixed model and regression: a fast and simple method for genomewide pedigree-based quantitative trait loci association analysis. Genetics  177: 577–585.  <ul class="find" data-citedArticleID="1026834" data-doi="10.1534/genetics.107.075614"><li><a href="http://dx.doi.org/10.1534/genetics.107.075614" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomewide+rapid+association+using+mixed+model+and+regression%3A+a+fast+and+simple+method+for+genomewide+pedigree-based+quantitative+trait+loci+association+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomewide+rapid+association+using+mixed+model+and+regression%3A+a+fast+and+simple+method+for+genomewide+pedigree-based+quantitative+trait+loci+association+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001274-Steinthorsdottir1" id="pone.0001274-Steinthorsdottir1"></a>Steinthorsdottir V, Thorleifsson G, Reynisdottir I, Benediktsson R, Jonsdottir T, et al.  (2007) A variant in CDKAL1 influences insulin response and risk of type 2 diabetes. Nat Genet  39: 770–775.  <ul class="find" data-citedArticleID="1026872" data-doi="10.1038/ng2043"><li><a href="http://dx.doi.org/10.1038/ng2043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+variant+in+CDKAL1+influences+insulin+response+and+risk+of+type+2+diabetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+variant+in+CDKAL1+influences+insulin+response+and+risk+of+type+2+diabetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001274-Devlin1" id="pone.0001274-Devlin1"></a>Devlin B, Roeder K (1999) Genomic control for association studies. Biometrics  55: 997–1004.  <ul class="find" data-citedArticleID="1026844" data-doi="10.1111/j.0006-341x.1999.00997.x"><li><a href="http://dx.doi.org/10.1111/j.0006-341x.1999.00997.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+control+for+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+control+for+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001274-Bacanu1" id="pone.0001274-Bacanu1"></a>Bacanu SA, Devlin B, Roeder K (2002) Association studies for quantitative traits in structured populations. Genet Epidemiol  22: 78–93.  <ul class="find" data-citedArticleID="1026838" data-doi="10.1002/gepi.1045"><li><a href="http://dx.doi.org/10.1002/gepi.1045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+studies+for+quantitative+traits+in+structured+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+studies+for+quantitative+traits+in+structured+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001274-Sladek1" id="pone.0001274-Sladek1"></a>Sladek R, Rocheleau G, Rung J, Dina C, Shen L, et al.  (2007) A genome-wide association study identifies novel risk loci for type 2 diabetes. Nature  445: 881–885.  <ul class="find" data-citedArticleID="1026870" data-doi="10.1038/nature05616"><li><a href="http://dx.doi.org/10.1038/nature05616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001274-Clayton1" id="pone.0001274-Clayton1"></a>Clayton DG, Walker NM, Smyth DJ, Pask R, Cooper JD, et al.  (2005) Population structure, differential bias and genomic control in a large-scale, case-control association study. Nat Genet  37: 1243–1246.  <ul class="find" data-citedArticleID="1026842" data-doi="10.1038/ng1653"><li><a href="http://dx.doi.org/10.1038/ng1653" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+structure%2C+differential+bias+and+genomic+control+in+a+large-scale%2C+case-control+association+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+structure%2C+differential+bias+and+genomic+control+in+a+large-scale%2C+case-control+association+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001274-Leutenegger1" id="pone.0001274-Leutenegger1"></a>Leutenegger AL, Prum B, Genin E, Verny C, Lemainque A, et al.  (2003) Estimation of the inbreeding coefficient through use of genomic data. Am J Hum Genet  73: 516–523.  <ul class="find" data-citedArticleID="1026860" data-doi="10.1086/378207"><li><a href="http://dx.doi.org/10.1086/378207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+the+inbreeding+coefficient+through+use+of+genomic+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+the+inbreeding+coefficient+through+use+of+genomic+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001274-Gilmour1" id="pone.0001274-Gilmour1"></a>Gilmour AR, Gogel BJ, Cullis BR, Welham SJ, Thompson R, et al.  (2002) ASReml user guide release 1.0. Orange, Australia: NSW Agriculture.   <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0001274-Aulchenko2" id="pone.0001274-Aulchenko2"></a>Aulchenko YS, Ripke S, Isaacs A, van Duijn CM (2007) GenABEL: an R library for genome-wide association analysis. Bioinformatics  23: 1294–1296.  <ul class="find" data-citedArticleID="1026836" data-doi="10.1093/bioinformatics/btm108"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btm108" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GenABEL%3A+an+R+library+for+genome-wide+association+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GenABEL%3A+an+R+library+for+genome-wide+association+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001274-Pardo1" id="pone.0001274-Pardo1"></a>Pardo LM, MacKay I, Oostra B, van Duijn CM, Aulchenko YS (2005) The effect of genetic drift in a young genetically isolated population. Ann Hum Genet  69: 288–295.  <ul class="find" data-citedArticleID="1026864" data-doi="10.1046/j.1469-1809.2005.00162.x"><li><a href="http://dx.doi.org/10.1046/j.1469-1809.2005.00162.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+genetic+drift+in+a+young+genetically+isolated+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+genetic+drift+in+a+young+genetically+isolated+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001274-Zollner1" id="pone.0001274-Zollner1"></a>Zollner S, Pritchard JK (2005) Coalescent-based association mapping and fine mapping of complex trait loci. Genetics  169: 1071–1092.  <ul class="find" data-citedArticleID="1026874" data-doi="10.1534/genetics.104.031799"><li><a href="http://dx.doi.org/10.1534/genetics.104.031799" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coalescent-based+association+mapping+and+fine+mapping+of+complex+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coalescent-based+association+mapping+and+fine+mapping+of+complex+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001274-Durrant1" id="pone.0001274-Durrant1"></a>Durrant C, Zondervan KT, Cardon LR, Hunt S, Deloukas P, et al.  (2004) Linkage disequilibrium mapping via cladistic analysis of single-nucleotide polymorphism haplotypes. Am J Hum Genet  75: 35–43.  <ul class="find" data-citedArticleID="1026846" data-doi="10.1086/422174"><li><a href="http://dx.doi.org/10.1086/422174" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+disequilibrium+mapping+via+cladistic+analysis+of+single-nucleotide+polymorphism+haplotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+disequilibrium+mapping+via+cladistic+analysis+of+single-nucleotide+polymorphism+haplotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.XML" value="69321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.PDF" value="149636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e001.PNG" value="6460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e001.TIF" value="22392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e002.PNG" value="9702"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e002.TIF" value="25724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e003.PNG" value="8871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e003.TIF" value="24764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e004.PNG" value="9155"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e004.TIF" value="24984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e005.PNG" value="7010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e005.TIF" value="22968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e006.PNG" value="8078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e006.TIF" value="23972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e007.PNG" value="5298"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e007.TIF" value="21248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e008.PNG" value="9850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e008.TIF" value="25836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e009.PNG" value="4970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e009.TIF" value="20896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e010.PNG" value="11605"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e010.TIF" value="27144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e011.PNG" value="9766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e011.TIF" value="25880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e012.PNG" value="7509"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.e012.TIF" value="23384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.t001.PNG_L" value="253611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.t001.PNG_M" value="107293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.t001.PNG_S" value="12517"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.t001.TIF" value="1295674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.t001.PNG_I" value="32378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.g001.PNG_L" value="142001"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.g001.PNG_M" value="101466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.g001.PNG_S" value="10790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.g001.TIF" value="405704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.g001.PNG_I" value="36920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.g002.PNG_L" value="36702"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.g002.PNG_M" value="54281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.g002.PNG_S" value="9996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.g002.TIF" value="332964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.g002.PNG_I" value="20626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001274.s001.DOC" value="54784"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001274&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001274" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001274&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001274&volume=&issue=&title=A Genomic Background Based Method for Association Analysis in Related Individuals&author_name=Najaf%20Amin%2C%20Cornelia%20M.%20van%20Duijn%2C%20Yurii%20S.%20Aulchenko&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001274" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001274" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001274" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001274&amp;t=A%20Genomic%20Background%20Based%20Method%20for%20Association%20Analysis%20in%20Related%20Individuals" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001274&title=A%20Genomic%20Background%20Based%20Method%20for%20Association%20Analysis%20in%20Related%20Individuals&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001274&amp;title=A%20Genomic%20Background%20Based%20Method%20for%20Association%20Analysis%20in%20Related%20Individuals" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001274" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001274&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Genomic Background Based Method for Association Analysis in Related Individuals';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001274';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Genomic%20Background%20Based%20Method%20for%20Association%20Analysis%20in%20Related%20Individuals http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001274" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001274" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+genomics%22" title="Search for articles in the subject area:'Animal genomics'"><div class="flagText">Animal genomics</div></a>
              <div data-categoryid="17455" data-articleid="25380"
                   data-categoryname="Animal genomics"
                   class="flagImage" title="Flag 'Animal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+analysis%22" title="Search for articles in the subject area:'Genome analysis'"><div class="flagText">Genome analysis</div></a>
              <div data-categoryid="40539" data-articleid="25380"
                   data-categoryname="Genome analysis"
                   class="flagImage" title="Flag 'Genome analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomics+statistics%22" title="Search for articles in the subject area:'Genomics statistics'"><div class="flagText">Genomics statistics</div></a>
              <div data-categoryid="35787" data-articleid="25380"
                   data-categoryname="Genomics statistics"
                   class="flagImage" title="Flag 'Genomics statistics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Grammar%22" title="Search for articles in the subject area:'Grammar'"><div class="flagText">Grammar</div></a>
              <div data-categoryid="24129" data-articleid="25380"
                   data-categoryname="Grammar"
                   class="flagImage" title="Flag 'Grammar' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="25380"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Quantitative+traits%22" title="Search for articles in the subject area:'Quantitative traits'"><div class="flagText">Quantitative traits</div></a>
              <div data-categoryid="24563" data-articleid="25380"
                   data-categoryname="Quantitative traits"
                   class="flagImage" title="Flag 'Quantitative traits' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Structural+genomics%22" title="Search for articles in the subject area:'Structural genomics'"><div class="flagText">Structural genomics</div></a>
              <div data-categoryid="18469" data-articleid="25380"
                   data-categoryname="Structural genomics"
                   class="flagImage" title="Flag 'Structural genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Test+statistics%22" title="Search for articles in the subject area:'Test statistics'"><div class="flagText">Test statistics</div></a>
              <div data-categoryid="35337" data-articleid="25380"
                   data-categoryname="Test statistics"
                   class="flagImage" title="Flag 'Test statistics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6032'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5590'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1148&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=44839">The relevant comparison is Merlin --fastAssoc</a><br>Posted by t0mpr1c3</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
