

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Non-Uniform Survival Rate of Heterodimerization Links in the Evolution of the Yeast Protein-Protein Interaction Network</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001667"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001667" />

    <meta name="citation_title" content="Non-Uniform Survival Rate of Heterodimerization Links in the Evolution of the Yeast Protein-Protein Interaction Network"/>
    <meta itemprop="name" content="Non-Uniform Survival Rate of Heterodimerization Links in the Evolution of the Yeast Protein-Protein Interaction Network"/>

      <meta name="citation_author" content="Takeshi Hase"/>
            <meta name="citation_author_institution" content="Department of Bioinformatics, Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan"/>
      <meta name="citation_author" content="Yoshihito Niimura"/>
            <meta name="citation_author_institution" content="Department of Bioinformatics, Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan"/>
      <meta name="citation_author" content="Tsuguchika Kaminuma"/>
            <meta name="citation_author_institution" content="Department of Bioinformatics, Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan"/>
      <meta name="citation_author" content="Hiroshi Tanaka"/>
            <meta name="citation_author_institution" content="Department of Bioinformatics, Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan"/>
            <meta name="citation_author_institution" content="Department of Bioinformatics, School of Biomedical Science, Tokyo Medical and Dental University, Tokyo, Japan"/>

    <meta name="citation_date" content="2008/2/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001667.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1667"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae.; citation_author=P Uetz; citation_author=L Giot; citation_author=G Cagney; citation_author=TA Mansfield; citation_author=RS Judson; citation_journal_title=Nature; citation_volume=403; citation_number=1; citation_pages=623-627; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A comprehensive two-hybrid analysis to explore the yeast protein interactome.; citation_author=T Ito; citation_author=T Chiba; citation_author=R Ozawa; citation_author=M Yoshida; citation_author=M Hattori; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=2; citation_pages=4569-4574; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mpact: The MIPS protein interaction resource on yeast.; citation_author=U Guldener; citation_author=M Munsterkotter; citation_author=M Oesterheld; citation_author=P Pagel; citation_author=A Ruepp; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=3; citation_pages=D436-441; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A map of the interactome network of the metazoan C. elegans.; citation_author=S Li; citation_author=CM Armstrong; citation_author=N Bertin; citation_author=H Ge; citation_author=S Milstein; citation_journal_title=Science; citation_volume=303; citation_number=4; citation_pages=540-543; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A protein interaction map of drosophila melanogaster.; citation_author=L Giot; citation_author=JS Bader; citation_author=C Brouwer; citation_author=A Chaudhuri; citation_author=B Kuang; citation_journal_title=Science; citation_volume=302; citation_number=5; citation_pages=1727-1736; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Network biology: understanding the cell's functional organization.; citation_author=AL Barabasi; citation_author=ZN Oltvai; citation_journal_title=Nat Rev Genet; citation_volume=5; citation_number=6; citation_pages=101-113; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Lethality and centrality in protein networks.; citation_author=H Jeong; citation_author=SP Mason; citation_author=AL Barabasi; citation_author=ZN Oltvai; citation_journal_title=Nature; citation_volume=411; citation_number=7; citation_pages=41-42; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The yeast protein interaction network evolves rapidly and contains few redundant genes.; citation_author=A Wagner; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=8; citation_pages=1283-1292; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Diameter of the World-Wide Web.; citation_author=R Albert; citation_author=H Jeong; citation_author=AL Barabasi; citation_journal_title=Nature; citation_volume=401; citation_number=9; citation_pages=130-131; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A model of large-scale proteome evolution.; citation_author=RV Sole; citation_author=R Pastor-Satorras; citation_author=ED Smith; citation_author=T Kepler; citation_journal_title=Adv Comp Syst; citation_volume=5; citation_number=10; citation_pages=43-54; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Error and attack tolerance of complex networks.; citation_author=R Albert; citation_author=H Jeong; citation_author=AL Barabasi; citation_journal_title=Nature; citation_volume=406; citation_number=11; citation_pages=378-382; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Collective dynamics of ‘small-world’ networks.; citation_author=DJ Watts; citation_author=SH Strogatz; citation_journal_title=Nature; citation_volume=393; citation_number=12; citation_pages=440-442; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Statistical mechanics of complex networks.; citation_author=R Albert; citation_author=AL Barabasi; citation_journal_title=Reviews of Modern Physics; citation_volume=74; citation_number=13; citation_pages=47-97; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The structure of scientific collaboration networks.; citation_author=MEJ Newman; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=14; citation_pages=404-409; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The large-scale organization of metabolic networks.; citation_author=H Jeong; citation_author=B Tombor; citation_author=R Albert; citation_author=AL Oltvai ZN Barabasi; citation_journal_title=Nature; citation_volume=407; citation_number=15; citation_pages=651-654; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Scale-free brain functional networks.; citation_author=VM Eguiluz; citation_author=DR Chialvo; citation_author=GA Cecchi; citation_author=M Baliki; citation_author=AV Apkarian; citation_journal_title=Phys Rev Lett; citation_volume=94; citation_number=16; citation_pages=018102; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Hierarchical organization of modularity in metabolic networks.; citation_author=RJ Williams; citation_author=ND Martinez; citation_author=EL Berlow; citation_author=JA Dunne; citation_author=AL Barabasi; citation_journal_title=Science; citation_volume=297; citation_number=17; citation_pages=1551-1555; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Functional and topological characterization of protein interaction networks.; citation_author=SH Yook; citation_author=ZN Oltvai; citation_author=AL Barabasi; citation_journal_title=Proteomics; citation_volume=4; citation_number=18; citation_pages=929-942; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Growing networks with local rules: preferential attachment, clustering hierarchy and degree correlations.; citation_author=A Vazquez; citation_journal_title=Phys Rev E; citation_volume=67; citation_number=19; citation_pages=056104; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Characterization of complex networks: A survey of measurements.; citation_author=LF Costa; citation_author=FA Rodrigues; citation_author=G Travieso; citation_author=V Boas; citation_journal_title=ADV PHYS; citation_volume=56; citation_number=20; citation_pages=167-242; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Specificity and stability in topology of protein networks.; citation_author=S Maslov; citation_author=K Sneppen; citation_journal_title=Science; citation_volume=296; citation_number=21; citation_pages=910-913; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Dynamical and correlation properties of the internet.; citation_author=R Pastor-Satorras; citation_author=A Vazquez; citation_author=A Vespignani; citation_journal_title=Phys Rev Lett; citation_volume=87; citation_number=22; citation_pages=258701; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Emergence of scaling in random networks.; citation_author=AL Barabasi; citation_author=R Albert; citation_journal_title=Science; citation_volume=286; citation_number=23; citation_pages=509-512; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Evolving protein interaction networks through gene duplication.; citation_author=R Pastor-Satorras; citation_author=E Smith; citation_author=RV Sole; citation_journal_title=J Theor Biol; citation_volume=222; citation_number=24; citation_pages=199-210; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Infinite-order percolation and giant fluctuations in a protein interaction network.; citation_author=J Kim; citation_author=PL Krapivsky; citation_author=B Kahng; citation_author=S Render; citation_journal_title=Phys Rev E; citation_volume=66; citation_number=25; citation_pages=055101; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Duplication models for biological networks.; citation_author=F Chung; citation_author=L Lu; citation_author=TG Dewey; citation_author=DJ Galas; citation_journal_title=J Comput Biol; citation_volume=10; citation_number=26; citation_pages=677-87; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Some asymptotic properties of duplication graph.; citation_author=A Raval; citation_journal_title=Phys Rev E; citation_volume=68; citation_number=27; citation_pages=066119; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Inferring network mechanisms: The Drosophila melanogaster protein interaction network.; citation_author=M Middendorf; citation_author=E Ziv; citation_author=CH Wiggins; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=28; citation_pages=3192-3197; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cliques and duplication-divergence network growth.; citation_author=I Ispolatov; citation_author=PL Krapivsky; citation_author=I Mazo; citation_author=A Yuryev; citation_journal_title=New J Phys; citation_volume=7; citation_number=29; citation_pages=145; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Modeling of protein interaction networks.; citation_author=A Vazquez; citation_author=A Flammini; citation_author=A Maritan; citation_author=A Vespignani; citation_journal_title=ComPlexUs; citation_volume=1; citation_number=30; citation_pages=38-44; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Filtering high-throughput protein-protein interaction data using a combination of genomic features.; citation_author=A Patil; citation_author=H Nakamura; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=31; citation_pages=100; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Binding properties and evolution of homodimers in protein-protein interaction networks.; citation_author=I Ispolatov; citation_author=A Yuryev; citation_author=I Mazo; citation_author=S Maslov; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=32; citation_pages=3629-3635; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Topology, tinkering and evolution of the human transcription factor networks.; citation_author=C Rodriguez-Caso; citation_author=MA Medina; citation_author=RV Sole; citation_journal_title=FEBS J; citation_volume=272; citation_number=33; citation_pages=6423-34; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Protein function, connectivity, and duplicability in yeast.; citation_author=A Prachumwat; citation_author=WH Li; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=34; citation_pages=30-39; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolutionary rate in the protein interaction network.; citation_author=HB Fraser; citation_author=AE Hirsh; citation_author=LM Steinmetz; citation_author=C Scharfe; citation_author=MW Feldman; citation_journal_title=Science; citation_volume=296; citation_number=35; citation_pages=750-752; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A simple dependence between protein evolution rate and the number of protein-protein interactions.; citation_author=HB Fraser; citation_journal_title=BMC Evol Biol; citation_volume=3; citation_number=36; citation_pages=11; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Modularity and evolutionary constraint on proteins.; citation_author=HB Fraser; citation_journal_title=Nature Genetics; citation_volume=37; citation_number=37; citation_pages=351-352; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=No simple dependence between protein evolution rate and the number of protein-protein interactions: only the most prolific interactors tend to evolve slowly.; citation_author=IK Jordan; citation_author=YI Wolf; citation_author=EV Koonin; citation_journal_title=BMC Evol Biol; citation_volume=3; citation_number=38; citation_pages=1; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The average distances in random graphs with given expected degrees.; citation_author=F Chung; citation_author=L Lu; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=39; citation_pages=15879-82; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Scale-free networks are ultrasmall.; citation_author=R Cohen; citation_author=S Havlin; citation_journal_title=Phys Rev Lett; citation_volume=90; citation_number=40; citation_pages=058701; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evidence for dynamically organized modularity in the yeast protein-protein interaction network.; citation_author=JD Han; citation_author=N Bertin; citation_author=T Hao; citation_author=DS Goldberg; citation_author=GF Berriz; citation_journal_title=Nature; citation_volume=430; citation_number=41; citation_pages=88-93; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Basic local alignment search tool.; citation_author=SF Altschul; citation_author=W Gish; citation_author=W Miller; citation_author=EW Myers; citation_author=DJ Lipman; citation_journal_title=J Mol Biol; citation_volume=215; citation_number=42; citation_pages=403-410; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Towards synthesis of a minimal cell.; citation_author=AC Forster; citation_author=GM Church; citation_journal_title=Mol Syst Biol; citation_volume=2; citation_number=43; citation_pages=45; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Duplication-divergence model of protein interaction network.; citation_author=I Ispolatov; citation_author=PL Krapivsky; citation_author=A Yuryev; citation_journal_title=Phys Rev E; citation_volume=71; citation_number=44; citation_pages=061911; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001667" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Non-Uniform Survival Rate of Heterodimerization Links in the Evolution of the Yeast Protein-Protein Interaction Network"/>
    <meta name="twitter:description" content="Protein-protein interaction networks (PINs) are scale-free networks with a small-world property. In a small-world network, the average cluster coefficient (&lt;C&gt;) is much higher than in a random network, but the average shortest path length (&lt;L&gt;) is similar between the two networks. To understand the evolutionary mechanisms shaping the structure of PINs, simulation studies using various network growth models have been performed. It has been reported that the heterodimerization (HD) model, in which a new link is added between duplicated nodes with a uniform probability, could reproduce scale-freeness and a high &lt;C&gt;. In this paper, however, we show that the HD model is unsatisfactory, because (i) to reproduce the high &lt;C&gt; in the yeast PIN, a much larger number (nHI) of HD links (links between duplicated nodes) are required than the estimated number of nHI in the yeast PIN and (ii) the spatial distribution of triangles in the yeast PIN is highly skewed but the HD model cannot reproduce the skewed distribution. To resolve these discrepancies, we here propose a new model named the non-uniform heterodimerization (NHD) model. In this model, an HD link is preferentially attached between duplicated nodes when they share many common neighbors. Simulation studies demonstrated that the NHD model can successfully reproduce the high &lt;C&gt;, the low nHI, and the skewed distribution of triangles in the yeast PIN. These results suggest that the survival rate of HD links is not uniform in the evolution of PINs, and that an HD link between high-degree nodes tends to be evolutionarily conservative. The non-uniform survival rate of HD links can be explained by assuming a low mutation rate for a high-degree node, and thus this model appears to be biologically plausible."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001667.g003"/>

  <meta property="og:title" content="Non-Uniform Survival Rate of Heterodimerization Links in the Evolution of the Yeast Protein-Protein Interaction Network" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001667" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2663'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5359'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7247&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001667">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001667" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001667&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Non-Uniform Survival Rate of Heterodimerization Links in the Evolution of the Yeast Protein-Protein Interaction Network
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takeshi Hase, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Bioinformatics, Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yoshihito Niimura
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:niimura@bioinfo.tmd.ac.jp">niimura@bioinfo.tmd.ac.jp</a></p>

                <p>Affiliation:
                  Department of Bioinformatics, Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tsuguchika Kaminuma, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Bioinformatics, Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hiroshi Tanaka
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Bioinformatics, Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan, 
                  Department of Bioinformatics, School of Biomedical Science, Tokyo Medical and Dental University, Tokyo, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 27, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001667</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001667" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001667" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001667" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001667" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001667-g001" data-doi="info:doi/10.1371/journal.pone.0001667" data-uri="info:doi/10.1371/journal.pone.0001667.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001667-t001" data-doi="info:doi/10.1371/journal.pone.0001667" data-uri="info:doi/10.1371/journal.pone.0001667.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001667-g002" data-doi="info:doi/10.1371/journal.pone.0001667" data-uri="info:doi/10.1371/journal.pone.0001667.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001667-g003" data-doi="info:doi/10.1371/journal.pone.0001667" data-uri="info:doi/10.1371/journal.pone.0001667.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001667">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001667" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Protein-protein interaction networks (PINs) are scale-free networks with a small-world property. In a small-world network, the average cluster coefficient (&lt;<em>C</em>&gt;) is much higher than in a random network, but the average shortest path length (&lt;<em>L</em>&gt;) is similar between the two networks. To understand the evolutionary mechanisms shaping the structure of PINs, simulation studies using various network growth models have been performed. It has been reported that the heterodimerization (HD) model, in which a new link is added between duplicated nodes with a uniform probability, could reproduce scale-freeness and a high &lt;<em>C</em>&gt;. In this paper, however, we show that the HD model is unsatisfactory, because (i) to reproduce the high &lt;<em>C</em>&gt; in the yeast PIN, a much larger number (<em>n</em><sub>HI</sub>) of HD links (links between duplicated nodes) are required than the estimated number of <em>n</em><sub>HI</sub> in the yeast PIN and (ii) the spatial distribution of triangles in the yeast PIN is highly skewed but the HD model cannot reproduce the skewed distribution. To resolve these discrepancies, we here propose a new model named the non-uniform heterodimerization (NHD) model. In this model, an HD link is preferentially attached between duplicated nodes when they share many common neighbors. Simulation studies demonstrated that the NHD model can successfully reproduce the high &lt;<em>C</em>&gt;, the low <em>n</em><sub>HI</sub>, and the skewed distribution of triangles in the yeast PIN. These results suggest that the survival rate of HD links is not uniform in the evolution of PINs, and that an HD link between high-degree nodes tends to be evolutionarily conservative. The non-uniform survival rate of HD links can be explained by assuming a low mutation rate for a high-degree node, and thus this model appears to be biologically plausible.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hase T, Niimura Y, Kaminuma T, Tanaka H (2008) Non-Uniform Survival Rate of Heterodimerization Links in the Evolution of the Yeast Protein-Protein Interaction Network. PLoS ONE 3(2):
          e1667.
            doi:10.1371/journal.pone.0001667</p><p><strong>Academic Editor: </strong>Matthew W. Hahn, Indiana University, United States of America</p><p><strong>Received:</strong> November 7, 2007; <strong>Accepted:</strong> January 22, 2008; <strong>Published:</strong> February 27, 2008</p><p><strong>Copyright:</strong> © 2008 Hase et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no support or funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The information of protein-protein interaction networks (PINs) at the whole-genome level is now available from several organisms, including <em>Saccharomyces cerevisiae</em> <a href="#pone.0001667-Uetz1">[1]</a>–<a href="#pone.0001667-Guldener1">[3]</a>, <em>Caenorhabditis elegans</em> <a href="#pone.0001667-Li1">[4]</a>, and <em>Drosophila melanogaster</em> <a href="#pone.0001667-Giot1">[5]</a>. These data were provided by using high-throughput experimental techniques such as yeast two-hybrid screens <a href="#pone.0001667-Uetz1">[1]</a>, <a href="#pone.0001667-Ito1">[2]</a>. The structure of PINs is represented as nodes (proteins) and links (interactions between proteins). Studies of PIN structures have revealed that PINs exhibit the following interesting properties <a href="#pone.0001667-Barabasi1">[6]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>First, PINs are scale-free networks <a href="#pone.0001667-Jeong1">[7]</a>, <a href="#pone.0001667-Wagner1">[8]</a>. The number of links connected to a node is called a degree. The degree distribution <em>P</em>(<em>k</em>) gives the probability that a node has <em>k</em> links (<em>i.e.</em>, degree <em>k</em>). In a scale-free network, <em>P</em>(<em>k</em>) decays as a power law, following <em>P</em>(<em>k</em>)~<em>k<sup>−γ</sup></em> <a href="#pone.0001667-Albert1">[9]</a>. (In the case of PINs, it is known that <em>P</em>(<em>k</em>) better fits a power law with an exponential cut-off, <em>i.e.</em>, <em>P</em>(<em>k</em>)~(<em>k</em><sub>0</sub>+<em>k</em>)<sup>−<em>γ</em> </sup>e<sup>−<em>k</em>/<em>k</em>c</sup> <a href="#pone.0001667-Jeong1">[7]</a>, <a href="#pone.0001667-Sole1">[10]</a>.) Therefore, a scale-free network is highly heterogeneous and is characterized by the presence of a large number of nodes having only a few links and a small number of nodes (hubs) that have numerous links. A scale-free network is known to be tolerant to random removal of nodes, but it is very fragile against selective removal of hubs <a href="#pone.0001667-Jeong1">[7]</a>, <a href="#pone.0001667-Albert2">[11]</a>. Second, PINs are small-world networks <a href="#pone.0001667-Li1">[4]</a>, <a href="#pone.0001667-Giot1">[5]</a>, <a href="#pone.0001667-Wagner1">[8]</a>, <a href="#pone.0001667-Sole1">[10]</a>. A small-world network is highly clustered like regular lattices, but it has small path lengths like a random network <a href="#pone.0001667-Watts1">[12]</a>. A small-world property is quantified by two statistics of a network, the average cluster coefficient &lt;<em>C</em>&gt; and the average shortest path length &lt;<em>L&gt;</em>. The cluster coefficient of node <em>i</em> is defined as <em>C<sub>i = </sub>2e<sub>i</sub></em>/<em>k<sub>i</sub>(k<sub>i</sub>−1)</em>, where <em>k<sub>i</sub></em> is the degree of node <em>i</em> and <em>e<sub>i</sub></em> is the number of links connecting <em>k<sub>i</sub></em> neighbors of node <em>i</em> to one another <a href="#pone.0001667-Watts1">[12]</a>. (When <em>k<sub>i</sub></em> is zero or one, <em>C<sub>i</sub></em> is defined to be zero.) In other words, <em>e<sub>i</sub></em> is the number of triangles that pass through node <em>i</em>. <em>C<sub>i</sub></em> is equal to one when all neighbors of node <em>i</em> are fully connected to one another, while <em>C<sub>i</sub></em> is zero when none of the neighbors are connected to one another. A small-world network is characterized by a &lt;<em>C</em>&gt; that is larger, and an &lt;<em>L</em>&gt; that is similar, to those of a random network <a href="#pone.0001667-Watts1">[12]</a>. (In a random network, &lt;<em>C</em>&gt; = &lt;<em>k</em>&gt;/<em>N</em> and &lt;<em>L</em>&gt;~log<em>N</em>/log&lt;<em>k</em>&gt; <a href="#pone.0001667-Albert3">[13]</a>, where &lt;<em>k</em>&gt; is the average degree and <em>N</em> is the number of nodes.) Scale-free and small-world properties are commonly observed in various complex networks such as the Internet <a href="#pone.0001667-Albert1">[9]</a>, coauthorship of scientific papers <a href="#pone.0001667-Newman1">[14]</a>, metabolic pathways <a href="#pone.0001667-Jeong2">[15]</a>, and functional connections in the human brain <a href="#pone.0001667-Eguiluz1">[16]</a>. Third, PINs show a hierarchical structure. In a network showing a hierarchical structure, &lt;<em>C</em>(<em>k</em>)&gt;, the average cluster coefficient of <em>k</em>-degree nodes, decays as a power law &lt;<em>C</em>(<em>k</em>)<em>&gt;</em>~<em>k<sup>−μ</sup></em> <a href="#pone.0001667-Williams1">[17]</a>, <a href="#pone.0001667-Yook1">[18]</a>. This indicates that a node with a small number of links has a high <em>C</em> and belongs to a small subnetwork in which all nodes are densely connected, while a hub has a low <em>C</em> and links different subnetworks. Fourth, PINs show a disassortative structure, in which &lt;<em>K</em><sub>nn</sub>(<em>k</em>)&gt; (“nn” represents “nearest neighbors”), the average degree among the neighbors of all <em>k</em>-degree nodes, follows &lt;<em>K</em><sub>nn</sub>(<em>k</em>)<em>&gt;</em>~<em>k<sup>−ν</sup></em> <a href="#pone.0001667-Vazquez1">[19]</a>–<a href="#pone.0001667-Maslov1">[21]</a>. Therefore, the connections between a hub and a low-degree node are favored, while those between hubs and those between low-degree nodes are suppressed <a href="#pone.0001667-Vazquez1">[19]</a>–<a href="#pone.0001667-PastorSatorras1">[22]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>It has been reported that the emergence of scale-free networks can be explained by the mechanisms of network growth and preferential attachment, in which a new node is preferentially attached to a node that already has many links <a href="#pone.0001667-Barabasi2">[23]</a>. In the PIN evolution, gene duplication is thought to be responsible for preferential attachment, because gene duplication creates a new node having the same interacting patterns as the original node, and a high-degree node is more likely to gain a new link by the duplication of a randomly selected node than a low-degree node <a href="#pone.0001667-Barabasi1">[6]</a>. To account for the properties of PINs mentioned above, several network growth models have been proposed. These models are generally based on gene duplication and divergence. In a divergence process, some of the links created by duplication are removed and some new links are added to a network. Sole et al. <a href="#pone.0001667-Sole1">[10]</a> proposed a model in which a divergence process includes two mechanisms, random removal of links from one of the duplicated nodes and random attachment of new links between a duplicated node and another node. Both simulation and analytical studies have shown that this model can generate scale-free and small-world properties <a href="#pone.0001667-Sole1">[10]</a>, <a href="#pone.0001667-PastorSatorras2">[24]</a>–<a href="#pone.0001667-Raval1">[27]</a>. However, studies have reported that a network generated by this model having the same number of nodes and links as those in the yeast and fly PINs showed a much smaller &lt;<em>C</em>&gt; than these PINs <a href="#pone.0001667-Sole1">[10]</a>, <a href="#pone.0001667-Middendorf1">[28]</a>, <a href="#pone.0001667-Ispolatov1">[29]</a>. To overcome this difficulty, Vazquez et al. <a href="#pone.0001667-Vazquez2">[30]</a> and Ispolatov et al. <a href="#pone.0001667-Ispolatov1">[29]</a> proposed the heterodimerization (HD) model. In this model, gene duplication is followed by divergence and HD; in the divergence process links are removed from duplicated nodes with a uniform probability <em>α</em>, and in the HD process a new link is established between two duplicated nodes with another probability <em>β</em>, forming a heterodimer <a href="#pone.0001667-Ispolatov1">[29]</a>, <a href="#pone.0001667-Vazquez2">[30]</a>. When a self-interacting protein is duplicated, the duplicated proteins will interact to each other. Therefore, <em>β</em> in the HD model represents the probability that a randomly selected protein is self-interacting and the link between two duplicated proteins survives after divergence. Simulation and analytical studies have shown that the HD model could reproduce a similar &lt;<em>C</em>&gt; to the yeast and fly PINs as well as a scale-free property <a href="#pone.0001667-Middendorf1">[28]</a>–<a href="#pone.0001667-Vazquez2">[30]</a>. The reason for the successful reproduction of a large &lt;<em>C</em>&gt; is that an HD process creates a triangle, and a network containing a large number of triangles shows a large &lt;<em>C</em>&gt;. Middendorf et al. <a href="#pone.0001667-Middendorf1">[28]</a> reported that the HD model could best reproduce the fly PIN among seven network growth models using a technique from machine learning.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In this paper, we examine the yeast PIN, since it constitutes the most reliable PIN data currently available at the whole genome level <a href="#pone.0001667-Patil1">[31]</a>. We first show that the HD model is unsatisfactory as an evolutionary model of the yeast PIN. We then propose a new model named the non-uniform heterodimerization (NHD) model, in which an HD link is preferentially attached between two duplicated nodes that share many common neighbors. The NHD model can successfully reproduce various features of the yeast PIN that cannot be explained by the HD model.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>In this study, we examined two models, the heterodimerization (HD) model and the non-uniform heterodimerization (NHD) model (see <a href="#s4">Materials and Methods</a> for details). In the HD model, there are two parameters, the probability that a link is removed from one of the duplicated nodes (<em>α</em>) and the probability that a new link is attached between two duplicated nodes (<em>β</em>) (<a href="#pone-0001667-g001">Figure 1A</a>), which represents the probability that a duplicated protein is self-interacting and the interaction between two duplicated proteins survives after the divergence process. These parameters were determined to let &lt;<em>k</em>&gt; and &lt;<em>C</em>&gt; in a generated network be the same as those in the yeast PIN. To compare the number of HD links in a generated network with that in the yeast PIN, we defined an evolutionary distance (<a href="#pone-0001667-g001">Figure 1B</a>). Two nodes are defined to be homologous when the evolutionary distance between these nodes is lower than or equal to a given threshold value <em>d</em><sub>T</sub>. The statistics of the networks generated by the HD model are shown in <a href="#pone-0001667-t001">Table 1</a>. The number of homologous pairs in the yeast PIN (<em>n</em><sub>H</sub> = 6,544) is between <em>n</em><sub>H</sub> for <em>d</em><sub>T = </sub>3 (5,309) and that for <em>d</em><sub>T = </sub>4 (8,337). However, the number of interactions between homologous nodes (<em>n</em><sub>HI</sub> = 395 and 514 for <em>d</em><sub>T = </sub>3 and 4, respectively) is much larger than that in the yeast PIN (175). This observation is consistent with the investigation of the fly PIN by Ispolatov et al. <a href="#pone.0001667-Ispolatov1">[29]</a>, in which it was reported that the HD model requires a much larger number of HD links (270) than the actual number in the fly PIN (142) <a href="#pone.0001667-Ispolatov2">[32]</a> to generate the 1,405 triangles present in the fly PIN.</p>
<div class="figure" id="pone-0001667-g001"><div class="img"><a name="pone-0001667-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001667" data-uri="info:doi/10.1371/journal.pone.0001667.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001667.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001667.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Simulation.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>(A) HD model. Node A is duplicated to generate node A′. Each of the links to node A′ is removed with a uniform probability <em>α</em> (left). Note that this method is based on completely asymmetric divergence <a href="#pone.0001667-Ispolatov3">[44]</a>, in which only one (A′) of the duplicated nodes is the target of removal of links. An HD link between node A and node A′ is attached with a uniform probability <em>β</em> (middle). (B) Evolutionary distance. When a node is duplicated, the evolutionary distance between each of the duplicated nodes and each of the other nodes in a network is assumed to increase by one due to mutations occurring in the duplicated nodes during the divergence process. Suppose that the evolutionary distance between node A and node B is <em>d</em> (left). After the duplication of node A to generate node A′ and the divergence of them, the evolutionary distance between nodes A and B, and that between nodes A′ and B become <em>d</em>+1 whether a link between nodes A and B and that between A′ and B are present or not (middle). (A dashed line indicates absence of a link.) The evolutionary distance between nodes A and A′ is defined to be 1 regardless of the presence of a link between them. After that, if node A′ is duplicated to create node A”, the evolutionary distance between nodes A and B continues to be <em>d</em>+1, while the evolutionary distances between nodes A and A′, A and A″, B and A′, and B and A″ become 2, 2, <em>d</em>+2, and <em>d</em>+2, respectively (right). (C) NHD model. In this model, the probability that a link is added between A and A′ is proportional to the number (<em>n</em><sub>N</sub>) of common neighbors shared by these nodes.</p>
<span>doi:10.1371/journal.pone.0001667.g001</span></div><div class="figure" id="pone-0001667-t001"><div class="img"><a name="pone-0001667-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001667" data-uri="info:doi/10.1371/journal.pone.0001667.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001667.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001667.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Statistics of the networks by the HD and NHD models and the yeast PIN</span></strong></p><span>doi:10.1371/journal.pone.0001667.t001</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>As was mentioned in the Introduction, the HD model can generate a network with a high &lt;<em>C</em>&gt;, because an HD link produces triangles. When two duplicated nodes share <em>n</em><sub>N</sub> common neighbors, <em>n</em><sub>N</sub> new triangles are created by an HD link between them (<a href="#pone-0001667-g001">Figure 1C</a>). Therefore, if a new link is attached between duplicated nodes more preferentially when a larger number of neighbors are shared between them, it is expected that HD links fewer than those required by the HD model can reproduce the high &lt;<em>C</em>&gt; in the PIN. For this reason, we examined the NHD model, in which the probability that a new link is added between duplicated nodes is proportional to the number of neighbors shared by these nodes. The probability of removing a link (<em>α</em>) and the proportionality constant to add a new link (<em>β</em>) were adjusted to let &lt;<em>k</em>&gt; and &lt;<em>C</em>&gt; in a generated network be the same as those in the yeast PIN. The results of simulations by the NHD model are shown in <a href="#pone-0001667-t001">Table 1</a>. Both <em>n</em><sub>H</sub> (6,544) and <em>n</em><sub>HI</sub> (175) in the yeast PIN are between the values for the NHD model with <em>d</em><sub>T</sub> = 3 and 4 (<em>n</em><sub>H</sub> = 5,315 and 8,351, respectively, and <em>n</em><sub>HI = </sub>157 and 208, respectively). Moreover, the values of <em>n</em><sub>HI</sub>/<em>n</em><sub>H</sub> for the NHD model with <em>d</em><sub>T</sub> = 3 and 4 are very close to that in the yeast PIN. Therefore, both a high &lt;<em>C</em>&gt; and a low <em>n</em><sub>HI</sub> were well reproduced by the NHD model. <a href="#pone-0001667-t001">Table 1</a> also shows that &lt;<em>L</em>&gt; in both HD and NHD networks are similar to that in a random network, indicating that they are small-world networks. However, interestingly, &lt;<em>L</em>&gt; in the yeast PIN is much lower than that in a random network (see <a href="#s3">Discussion</a>).</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p><a href="#pone-0001667-g002">Figure 2A</a> shows the degree distribution of the networks generated by the HD and NHD models and that of the yeast PIN. Although there is a discrepancy between the model networks and the yeast PIN for a large <em>k</em> (see <a href="#s3">Discussion</a>), the results showed that both models can reproduce the degree distribution of the yeast PIN that follows a power law with an exponential cut-off. <a href="#pone-0001667-g002">Figure 2B</a> shows that &lt;<em>C</em>(<em>k</em>)&gt; in the networks generated by the models follows a power law, indicating that these networks exhibit a hierarchical structure. In the case of the yeast PIN, however, &lt;<em>C</em>(<em>k</em>)&gt; decreases following a power law as <em>k</em> increases only for a non-small <em>k</em> (<em>k</em>&gt;10). This relationship was also observed in the previous studies <a href="#pone.0001667-Yook1">[18]</a>, <a href="#pone.0001667-Vazquez1">[19]</a>. As shown in <a href="#pone-0001667-g002">Figure 2C</a>, both the HD and NHD networks display a disassortative structure &lt;<em>K</em><sub>nn</sub>(<em>k</em>)&gt;~<em>k<sup>−ν</sup></em>, but the values of <em>ν</em> are smaller than that in the yeast PIN (see <a href="#s3">Discussion</a>).</p>
<div class="figure" id="pone-0001667-g002"><div class="img"><a name="pone-0001667-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001667" data-uri="info:doi/10.1371/journal.pone.0001667.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001667.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001667.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Properties in the networks by the HD and NHD models.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>Black squares, red diamonds, and green crosses show the values for the yeast PIN, the network generated by the NHD model, and the network by the HD model, respectively. The results for the HD and NHD models were obtained by taking the average among 100 networks generated by simulations. (A) Degree distribution <em>P</em>(<em>k</em>). The dashed line represents (<em>k</em><sub>0</sub>+<em>k</em>)<sup>−<em>γ</em></sup>e<sup>−<em>k</em>/<em>kc</em></sup> with <em>γ</em> = 2.7, <em>k</em><sub>0</sub><em> = </em>3.4, and <em>k</em><sub>c</sub> = 50. (B) Distribution of the average cluster coefficient &lt;<em>C</em>(<em>k</em>)&gt;. Dashed lines in red and green indicate <em>k</em><sup>−0.68</sup> and <em>k</em><sup>−0.90</sup>, respectively. (C) Distribution of &lt;<em>K</em><sub>nn</sub>(<em>k</em>)&gt; indicating a disassortative structure. Dashed lines in black, red, and green represent <em>k</em><sup>−0.47</sup>, <em>k</em><sup>−0.18</sup>, and <em>k</em><sup>−0.14</sup>, respectively. (D) Distribution of <em>P</em><sub>T</sub>(<em>n</em><sub>T</sub>), the probability that a given link is contained in <em>n</em><sub>T</sub> triangles.</p>
<span>doi:10.1371/journal.pone.0001667.g002</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p><a href="#pone-0001667-g002">Figure 2D</a> shows the probability <em>P</em><sub>T</sub>(<em>n</em><sub>T</sub>) that a given link is contained in <em>n</em><sub>T</sub> triangles in a network. For example, <em>n</em><sub>T</sub> = 2 for the link between nodes A and A′ (dashed line) in the middle of <a href="#pone-0001667-g001">Figure 1C</a>. The probability distribution <em>P</em><sub>T</sub>(<em>n</em><sub>T</sub>) is a statistic describing a spatial distribution of triangles in a network. In a network generated by the HD model, the spatial distribution of triangles can be regarded to be random, because addition of a new HD link occurs randomly. As shown in this figure, the distribution of <em>P</em><sub>T</sub>(<em>n</em><sub>T</sub>) in the yeast PIN is quite different from that in the network by the HD model, suggesting that the spatial distribution of triangles in the yeast PIN is highly skewed. In other words, in the yeast PIN, the extent of overlapping of triangles is larger than the expectation from a random distribution. On the other hand, the <em>P</em><sub>T</sub>(<em>n</em><sub>T</sub>) distribution for the NHD model is close to that in the yeast PIN. Therefore, the structure of a network generated by the NHD model is more similar to the PIN than that by the HD model.</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>In the HD and NHD models, self-interactions were not explicitly considered, though it was assumed that an HD link is created only when a self-interacting protein is duplicated. However, in the yeast PIN, the fraction of self-interacting proteins is only 0.049, and &lt;<em>k</em>&gt; increases slightly (3.84) when self-interactions are considered. The effect of self-interactions to other statistical properties of the yeast PIN is negligible (<a href="#pone.0001667.s002">Figure S1</a>). Therefore, it is expected that explicit consideration of self-interactions in a model does not essentially alter the results described above. We should also note that the fraction (0.049) of self-interactions in the yeast PIN is consistent with the value of <em>β</em> (0.028) in the NHD model. The fraction in the yeast PIN is much smaller than that (0.18) in the human transcription factor network, in which the statistical properties are considerably different between the networks with and without self-interactions <a href="#pone.0001667-RodriguezCaso1">[33]</a>.</p>
<a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>We also examined the effect of gene deletions that are caused by mutations. For this purpose, we modified the NHD model by adding the process of random elimination of nodes (NHD+E model). However, the elimination of nodes did not essentially change the results (<a href="#pone.0001667.s001">Table S1</a> and <a href="#pone.0001667.s003">Figure S2</a>).</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study, we showed that the NHD model can successfully reproduce both a high &lt;<em>C</em>&gt; and a low <em>n</em><sub>HI</sub> in the yeast PIN, whereas the HD model cannot regenerate the value of <em>n</em><sub>HI</sub>. We also demonstrated that the distribution of triangles in the yeast PIN is highly skewed and the skewed distribution can be reproduced by the NHD model but not by the HD model. These results suggest that the NHD model would reflect the actual evolutionary mechanism of PINs.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Is the NHD model biologically realistic? In the PIN evolution, when a self-interacting protein is duplicated, an HD link between duplicated proteins is added to the PIN. Some HD links survive in evolution, but other links disappear because of mutations occurring at interacting sites in one or both of the duplicated proteins. Therefore, in the HD model, an HD link is assumed to survive at a uniform rate. On the other hand, in the NHD model, it is assumed that the survival rate of an HD link is proportional to the number of common neighbors shared by the duplicated nodes. <a href="#pone-0001667-g003">Figure 3A</a> shows the probability <em>P</em><sub>HD</sub>(<em>n</em><sub>N</sub>) that two homologous nodes have an HD link when they share <em>n</em><sub>N</sub> common neighbors. This figure indicates that <em>P</em><sub>HD</sub>(<em>n</em><sub>N</sub>) is nearly constant regardless of <em>n</em><sub>N</sub> in the networks by the HD model. On the other hand, in the yeast PIN, <em>P</em><sub>HD</sub>(<em>n</em><sub>N</sub>) increases in proportion with <em>n</em><sub>N</sub>, which is consistent with the NHD model. These observations suggest that, in the evolution of PINs, the survival rate of HD links is not uniform in terms of <em>n</em><sub>N</sub>. Therefore, the NHD model appears to be realistic. The value of <em>P</em><sub>HD</sub>(<em>n</em><sub>N</sub>) in the NHD network is smaller than that in the yeast PIN for <em>n</em><sub>N</sub>&lt;15. This appears to happen because, in the NHD network, several protein pairs have very large values of <em>n</em><sub>N</sub>, which is not the case in the yeast PIN. That there are no protein pairs with large <em>n</em><sub>N</sub> in the yeast PIN may be due to the high duplicability of low-degree nodes <a href="#pone.0001667-Prachumwat1">[34]</a>, which was not considered in the NHD model (see below).</p>
<div class="figure" id="pone-0001667-g003"><div class="img"><a name="pone-0001667-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001667" data-uri="info:doi/10.1371/journal.pone.0001667.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001667.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001667.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001667.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001667.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001667.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>HD links in the yeast PIN and in the networks by simulations.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Black squares, red diamonds, and green crosses show the values for the yeast PIN, the network generated by the NHD model, and the network by the HD model, respectively. (A) Distribution of <em>P</em><sub>HD</sub>(<em>n</em><sub>N</sub>), the probability that an HD link exists between two homologous proteins when they share <em>n</em><sub>N</sub> common neighbors (for <em>d</em><sub>T</sub> = 3). The slopes of the dashed lines are 0.028 (red) and 0 (green). The result for <em>d</em><sub>T</sub> = 4 is nearly identical to this result (data not shown). (B) Distribution of &lt;<em>k</em><sub>HP</sub>(<em>n</em><sub>N</sub>)&gt;, the average degree of proteins that are connected by HD links and share <em>n</em><sub>N</sub> common neighbors with their homologous proteins. The dashed line is a regression line (<em>r</em> = 0.73).</p>
<span>doi:10.1371/journal.pone.0001667.g003</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Why, then, is the survival rate of HD links not uniform, but rather proportional to the number of common neighbors? One possible explanation is as follows. It has been reported that the degree of proteins in the yeast PIN is negatively correlated with their evolutionary rates <a href="#pone.0001667-Fraser1">[35]</a>–<a href="#pone.0001667-Fraser3">[37]</a>, though this assertion is controversial <a href="#pone.0001667-Jordan1">[38]</a>. Not surprisingly, proteins connected by an HD link that has a large <em>n</em><sub>N</sub> tend to have a high degree (<a href="#pone-0001667-g003">Figure 3B</a>). Therefore, the evolutionary rates of proteins in an HD link with a large <em>n</em><sub>N</sub> are expected to be low. If this is the case, the possibility of the occurrence of mutations at the binding sites would also be low, and thus the survival rates of HD links having a large <em>n</em><sub>N</sub> are thought to be higher than those of HD links having a small <em>n</em><sub>N</sub>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Although the degree distribution <em>P</em>(<em>k</em>) of the NHD network is generally in good agreement with that of the yeast PIN, the number of nodes with <em>k</em>&gt;50 in the former is much smaller than that in the latter (<a href="#pone-0001667-g002">Figure 2A</a>). The average of the maximum degrees among the NHD networks is 75.2, while the maximum degree in the yeast PIN is 286. Moreover, though the NHD network exhibits a disassortative structure &lt;<em>K</em><sub>nn</sub>(<em>k</em>)&gt;~<em>k</em><sup>−<em>ν</em></sup>, the value of <em>ν</em> is considerably smaller than that in the yeast PIN (<a href="#pone-0001667-g002">Figure 2C</a>). These discrepancies might be resolved by introducing a mechanism wherein low-degree nodes duplicate more frequently than high-degree nodes. Prachumwat and Li <a href="#pone.0001667-Prachumwat1">[34]</a> reported a negative correlation between the degree of proteins and their duplicability. Due to the disassortative structure (<a href="#pone-0001667-g002">Figure 2C</a>), low-degree nodes have more links to high-degree nodes than to low-degree nodes. Therefore, as a result of frequent duplication of low-degree nodes, links between a high-degree node and a low-degree node are preferentially generated, and a high-degree node tends to gain new links. For this reason, with the mechanism of high duplicability of low-degree nodes, the degrees of hubs and the value of <em>ν</em> in <a href="#pone-0001667-g002">Figure 2C</a> are expected to become larger than the current values. Moreover, the lack of HD links with high <em>n</em><sub>N</sub> in the yeast PIN (<a href="#pone-0001667-g003">Figure 3A</a>) would also be explained with this mechanism, because HD links with high <em>n</em><sub>N</sub> should be rare if the duplicability of high-degree nodes is low.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Although PINs are generally considered to be small-world networks <a href="#pone.0001667-Li1">[4]</a>, <a href="#pone.0001667-Wagner1">[8]</a>, <a href="#pone.0001667-Sole1">[10]</a>, <a href="#pone.0001667-PastorSatorras2">[24]</a>, the &lt;<em>L</em>&gt; in the yeast PIN is much lower than that in a random network (<a href="#pone-0001667-t001">Table 1</a>). In fact, this observation is consistent with the previous result by Pastor-Satorras et al. <a href="#pone.0001667-PastorSatorras2">[24]</a>, in which it was reported that &lt;<em>L</em>&gt; in the random network and that in their model network were 8.0 and 6.8, respectively. (However, they mentioned that these two values are “comparable”.) Therefore, the yeast PIN is an “ultra-small” network, in which &lt;<em>L</em>&gt; is lower than that in a random network. It is known that a scale-free random network is ultra-small <a href="#pone.0001667-Chung2">[39]</a>, <a href="#pone.0001667-Cohen1">[40]</a>. The yeast PIN can be randomized without changing the distribution of <em>P</em>(<em>k</em>) by using the random rewiring method <a href="#pone.0001667-Maslov1">[21]</a>. In this method, two links in a network were chosen randomly, and these links were rewired by exchanging their connecting partners. After randomization of the yeast PIN, &lt;<em>L</em>&gt; (4.49) is similar, but &lt;<em>C</em>&gt; (0.010) becomes much lower than that in the yeast PIN. Therefore, the yeast PIN is far from a scale-free random network. Nevertheless, interestingly, the yeast PIN is an ultra-small network.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The difference in the value of &lt;<em>L</em>&gt; between the yeast PIN and the NHD network may be explained in the following way. It was reported that the removal of hubs drastically increases the value of &lt;<em>L</em>&gt; in the yeast PIN <a href="#pone.0001667-Han1">[41]</a>. Therefore, the low &lt;<em>L</em>&gt; in the yeast PIN might be due to the fact that the number of hubs in the yeast PIN is larger than that in the NHD network (<a href="#pone-0001667-g002">Figure 2A</a>). (There are 17 nodes with <em>k</em>&gt;50 in the yeast PIN, while the average number of nodes with <em>k</em>&gt;50 among the NHD networks is 5.8.) In fact, if we eliminate all nodes with <em>k</em>&gt;50 and all links connected to them from the yeast PIN and the NHD network, both &lt;<em>L</em>&gt; and &lt;<em>C</em>&gt; become similar between the two networks (&lt;<em>L</em>&gt; = 6.13 and 6.51, and &lt;<em>C</em>&gt; = 0.063 and 0.060 for the yeast PIN and the NHD network, respectively). It therefore appears that the presence of a large number of hubs in the yeast PIN would be the reason for a very low &lt;<em>L</em>&gt;.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The above discussion would indicate that the NHD model is merely a rough approximation of the actual mechanism of the PIN evolution. However, we should note that although our new model contains only two free parameters, it could well capture various aspects of the structure of the yeast PIN. The availability of high-quality interaction data from other species will thus help to clarify the architecture and evolution of PINs in greater detail.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Human-curated interaction data of the yeast PIN were downloaded from the MIPS (Munich Information Center for Protein Sequences) database (<a href="http://mips.gsf.de">http://mips.gsf.de</a>) (18 May 2006) <a href="#pone.0001667-Guldener1">[3]</a>. The interaction data are separated into several components that are not connected to each other; we used the largest component containing 3,891 proteins and 7,270 non-redundant interactions. Among these proteins, 191 proteins are self-interacting. The amino acid sequences of 6,736 yeast proteins were also obtained from the MIPS database. In order to estimate the number of interactions between homologous proteins in the yeast PIN, we identified homologous gene pairs. Self-against-self homologous searches were conducted for the 6,736 sequences by using the BLASTP program <a href="#pone.0001667-Altschul1">[42]</a> with the cut-off E-value of 1e-5. We identified 6,544 homologous pairs (<em>n</em><sub>H</sub>) and 175 interactions between these pairs of proteins (<em>n</em><sub>HI</sub>) in the yeast PIN (see <a href="#pone-0001667-t001">Table 1</a>). The value of <em>n</em><sub>HI</sub>/<em>n</em><sub>H</sub> did not essentially change when a more stringent cut-off E-value was used (<em>n</em><sub>HI</sub>/<em>n</em><sub>H</sub> was 0.027 and 0.032 for the E-values of 1e-5 and 1e-10, respectively).</p>


<h4>Simulation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>In this study, we used the “minimal genome” containing 113 proteins as the initial network, because the first living organism is assumed to have had at least 113 proteins <a href="#pone.0001667-Forster1">[43]</a>. We generated a single component random network containing 113 nodes with &lt;<em>k</em>&gt; = 3.74, which is the average degree of the yeast PIN. The evolutionary distance between two nodes present in the initial network was assumed to be infinity. We obtained very similar results when we started a simulation from the initial network containing only two nodes.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>At each time step of simulation in the HD model, a new node is added to the network according to the following rules (<a href="#pone-0001667-g001">Figure 1A</a>). (1) A node is randomly selected (A) and is duplicated to generate a new node (A′), having the same interacting pattern as node A. (2) Each of the links to node A′ is removed with a probability <em>α</em> (completely asymmetric divergence <a href="#pone.0001667-Ispolatov3">[44]</a>). (3) A link between node A and node A′ is created with a probability <em>β</em>. If node A′ does not have any links after these processes (all links to node A′ were removed and no links were created), node A′ is not added to the network. These processes were repeated until the number of nodes became 3,891, which is the number of nodes contained in the yeast PIN. In the NHD model, the probability that a new link is added between two duplicated nodes (A′ and A) is defined to be <em>βn</em><sub>N</sub> (when <em>βn</em><sub>N</sub>≤1), where <em>n</em><sub>N</sub> is the number of common neighbors shared by these two nodes (<a href="#pone-0001667-g001">Figure 1C</a>). The probability is defined to be one when <em>βn</em><sub>N</sub>&gt;1. (However, there were no such cases in the simulations.) We performed simulations using various values of <em>α</em> and <em>β</em>. For a given <em>α</em> and <em>β</em>, we conducted simulations 100 times and computed the average of &lt;<em>k</em>&gt; and the average of &lt;<em>C</em>&gt; from the 100 networks. The values of <em>α</em> and <em>β</em> that could reproduce &lt;<em>k</em>&gt; (3.74) and &lt;<em>C</em>&gt; (0.066) in the yeast PIN were used (<a href="#pone-0001667-t001">Table 1</a>).</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>In the NHD+E model, the following process was added after the addition of new links at each step of the NHD model. A node in a network is randomly selected, and the selected node is eliminated from the network with a probability <em>δ</em> together with all interactions connecting to the selected node. If the selected node is connected to one-degree nodes, all of these one-degree nodes are also removed. We changed the value <em>δ</em> from 0.001 to 0.1 (see <a href="#pone.0001667.s001">Table S1</a>). The values of <em>α</em> and <em>β</em> were determined in the same way as in the NHD model.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001667"></div><a name="pone.0001667.s001" id="pone.0001667.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001667.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001667.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001667.s002" id="pone.0001667.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001667.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Properties in the yeast PIN with and without self-interactions. Red triangles and black squares show the values for the yeast PINs with and without self-interactions, respectively. (A) Degree distribution <em>P</em>(<em>k</em>). (B) Distribution of the average cluster coefficient &lt;<em>C</em>(<em>k</em>)&gt;. (C) Distribution of &lt;<em>K</em><sub>nn</sub>(<em>k</em>)&gt;. (D) Distribution of <em>P</em><sub>T</sub>(<em>n</em><sub>T</sub>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001667.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.78 MB TIF)</p>
<a name="pone.0001667.s003" id="pone.0001667.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001667.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Properties in the networks by the NHD and NHD+E models. Black squares, red diamonds, and blue crosses show the values for the yeast PIN, the network generated by the NHD model, and the network by the NHD+E model with <em>δ</em> = 0.1, respectively. The results for the NHD and NHD+E models were obtained by taking the average among 100 networks generated by simulations. (A) Degree distribution <em>P</em>(<em>k</em>). The dashed line represents (<em>k</em><sub>0</sub>+<em>k</em>)<sup>−<em>γ</em></sup>e<sup>−<em>k</em>/<em>k</em><sub>c</sub></sup> with <em>γ</em> = 2.7, <em>k</em><sub>0</sub> = 3.4, and <em>k</em><sub>c</sub> = 50. (B) Distribution of the average cluster coefficient &lt;<em>C</em>(<em>k</em>)&gt;. Dashed line in red indicates <em>k</em><sup>−0.68</sup>. (C) Distribution of &lt;<em>K</em><sub>nn</sub>(<em>k</em>)&gt;. Dashed lines in black and red represent <em>k</em><sup>−0.47</sup> and <em>k</em><sup>−0.18</sup>, respectively. (D) Distribution of <em>P</em><sub>T</sub>(<em>n</em><sub>T</sub>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001667.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.82 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank T. Masuda, Y. Fukuoka, T. Takai, F. Ren, M. Koeda, S. Ogishima, E. Campos, M. Morioka, and S. Nakagawa for their useful comments and discussion.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YN HT TH TK. Performed the experiments: TH. Analyzed the data: TH. Contributed reagents/materials/analysis tools: TH. Wrote the paper: YN TH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001667-Uetz1" id="pone.0001667-Uetz1"></a>Uetz P, Giot L, Cagney G, Mansfield TA, Judson RS, et al.  (2000) A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae. Nature  403: 623–627.  <ul class="find" data-citedArticleID="1062122" data-doi="10.1038/35001009"><li><a href="http://dx.doi.org/10.1038/35001009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+analysis+of+protein-protein+interactions+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+analysis+of+protein-protein+interactions+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001667-Ito1" id="pone.0001667-Ito1"></a>Ito T, Chiba T, Ozawa R, Yoshida M, Hattori M, et al.  (2001) A comprehensive two-hybrid analysis to explore the yeast protein interactome. Proc Natl Acad Sci USA  98: 4569–4574.  <ul class="find" data-citedArticleID="1062090" data-doi="10.1073/pnas.061034498"><li><a href="http://dx.doi.org/10.1073/pnas.061034498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+two-hybrid+analysis+to+explore+the+yeast+protein+interactome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+two-hybrid+analysis+to+explore+the+yeast+protein+interactome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001667-Guldener1" id="pone.0001667-Guldener1"></a>Guldener U, Munsterkotter M, Oesterheld M, Pagel P, Ruepp A, et al.  (2006) Mpact: The MIPS protein interaction resource on yeast. Nucleic Acids Res  34: D436–441.  <ul class="find" data-citedArticleID="1062080" data-doi="10.1093/nar/gkj003"><li><a href="http://dx.doi.org/10.1093/nar/gkj003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mpact%3A+The+MIPS+protein+interaction+resource+on+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mpact%3A+The+MIPS+protein+interaction+resource+on+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001667-Li1" id="pone.0001667-Li1"></a>Li S, Armstrong CM, Bertin N, Ge H, Milstein S, et al.  (2004) A map of the interactome network of the metazoan C. elegans. Science  303: 540–543.  <ul class="find" data-citedArticleID="1062100" data-doi="10.1126/science.1091403"><li><a href="http://dx.doi.org/10.1126/science.1091403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+map+of+the+interactome+network+of+the+metazoan+C.+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+map+of+the+interactome+network+of+the+metazoan+C.+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001667-Giot1" id="pone.0001667-Giot1"></a>Giot L, Bader JS, Brouwer C, Chaudhuri A, Kuang B, et al.  (2003) A protein interaction map of drosophila melanogaster. Science  302: 1727–1736.  <ul class="find" data-citedArticleID="1062078" data-doi="10.1126/science.1090289"><li><a href="http://dx.doi.org/10.1126/science.1090289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+protein+interaction+map+of+drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+protein+interaction+map+of+drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001667-Barabasi1" id="pone.0001667-Barabasi1"></a>Barabasi AL, Oltvai ZN (2004) Network biology: understanding the cell's functional organization. Nat Rev Genet  5: 101–113.  <ul class="find" data-citedArticleID="1062056" data-doi="10.1038/nrg1272"><li><a href="http://dx.doi.org/10.1038/nrg1272" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+biology%3A+understanding+the+cell%27s+functional+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+biology%3A+understanding+the+cell%27s+functional+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001667-Jeong1" id="pone.0001667-Jeong1"></a>Jeong H, Mason SP, Barabasi AL, Oltvai ZN (2001) Lethality and centrality in protein networks. Nature  411: 41–42.  <ul class="find" data-citedArticleID="1062092" data-doi="10.1038/35075138"><li><a href="http://dx.doi.org/10.1038/35075138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lethality+and+centrality+in+protein+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lethality+and+centrality+in+protein+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001667-Wagner1" id="pone.0001667-Wagner1"></a>Wagner A (2001) The yeast protein interaction network evolves rapidly and contains few redundant genes. Mol Biol Evol  18: 1283–1292.  <ul class="find" data-citedArticleID="1062128" data-doi="10.1093/oxfordjournals.molbev.a003913"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003913" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+yeast+protein+interaction+network+evolves+rapidly+and+contains+few+redundant+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+yeast+protein+interaction+network+evolves+rapidly+and+contains+few+redundant+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001667-Albert1" id="pone.0001667-Albert1"></a>Albert R, Jeong H, Barabasi AL (1999) Diameter of the World-Wide Web. Nature  401: 130–131.  <ul class="find" data-citedArticleID="1062048" data-doi="10.1038/43601"><li><a href="http://dx.doi.org/10.1038/43601" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diameter+of+the+World-Wide+Web." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diameter+of+the+World-Wide+Web.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001667-Sole1" id="pone.0001667-Sole1"></a>Sole RV, Pastor-Satorras R, Smith ED, Kepler T (2002) A model of large-scale proteome evolution. Adv Comp Syst  5: 43–54.  <ul class="find" data-citedArticleID="1062120" data-doi="10.1142/s021952590200047x"><li><a href="http://dx.doi.org/10.1142/s021952590200047x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+model+of+large-scale+proteome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+model+of+large-scale+proteome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001667-Albert2" id="pone.0001667-Albert2"></a>Albert R, Jeong H, Barabasi AL (2000) Error and attack tolerance of complex networks. Nature  406: 378–382.  <ul class="find" data-citedArticleID="1062050" data-doi="10.1038/35019019"><li><a href="http://dx.doi.org/10.1038/35019019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Error+and+attack+tolerance+of+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Error+and+attack+tolerance+of+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001667-Watts1" id="pone.0001667-Watts1"></a>Watts DJ, Strogatz SH (1998) Collective dynamics of ‘small-world’ networks. Nature  393: 440–442.  <ul class="find" data-citedArticleID="1062130" data-doi="10.1038/30918"><li><a href="http://dx.doi.org/10.1038/30918" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collective+dynamics+of+%E2%80%98small-world%E2%80%99+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Collective+dynamics+of+%E2%80%98small-world%E2%80%99+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001667-Albert3" id="pone.0001667-Albert3"></a>Albert R, Barabasi AL (2002) Statistical mechanics of complex networks. Reviews of Modern Physics  74: 47–97.  <ul class="find" data-citedArticleID="1062052" data-doi="10.1103/revmodphys.74.47"><li><a href="http://dx.doi.org/10.1103/revmodphys.74.47" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+mechanics+of+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+mechanics+of+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001667-Newman1" id="pone.0001667-Newman1"></a>Newman MEJ (2001) The structure of scientific collaboration networks. Proc Natl Acad Sci USA  98: 404–409.  <ul class="find" data-citedArticleID="1062106"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+structure+of+scientific+collaboration+networks.&amp;auth=&amp;atitle=The+structure+of+scientific+collaboration+networks." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+scientific+collaboration+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+scientific+collaboration+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001667-Jeong2" id="pone.0001667-Jeong2"></a>Jeong H, Tombor B, Albert R, Oltvai ZN Barabasi AL (2000) The large-scale organization of metabolic networks. Nature  407: 651–654.  <ul class="find" data-citedArticleID="1062094" data-doi="10.1038/35036627"><li><a href="http://dx.doi.org/10.1038/35036627" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+large-scale+organization+of+metabolic+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+large-scale+organization+of+metabolic+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001667-Eguiluz1" id="pone.0001667-Eguiluz1"></a>Eguiluz VM, Chialvo DR, Cecchi GA, Baliki M, Apkarian AV (2005) Scale-free brain functional networks. Phys Rev Lett  94: 018102.  <ul class="find" data-citedArticleID="1062068" data-doi="10.1103/physrevlett.94.018102"><li><a href="http://dx.doi.org/10.1103/physrevlett.94.018102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scale-free+brain+functional+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scale-free+brain+functional+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001667-Williams1" id="pone.0001667-Williams1"></a>Williams RJ, Martinez ND, Berlow EL, Dunne JA, Barabasi AL (2002) Hierarchical organization of modularity in metabolic networks. Science  297: 1551–1555.  <ul class="find" data-citedArticleID="1062132" data-doi="10.1126/science.1073374"><li><a href="http://dx.doi.org/10.1126/science.1073374" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hierarchical+organization+of+modularity+in+metabolic+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hierarchical+organization+of+modularity+in+metabolic+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001667-Yook1" id="pone.0001667-Yook1"></a>Yook SH, Oltvai ZN, Barabasi AL (2004) Functional and topological characterization of protein interaction networks. Proteomics  4: 929–942.  <ul class="find" data-citedArticleID="1062134" data-doi="10.1002/pmic.200300636"><li><a href="http://dx.doi.org/10.1002/pmic.200300636" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+and+topological+characterization+of+protein+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+and+topological+characterization+of+protein+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001667-Vazquez1" id="pone.0001667-Vazquez1"></a>Vazquez A (2003) Growing networks with local rules: preferential attachment, clustering hierarchy and degree correlations. Phys Rev E  67: 056104.  <ul class="find" data-citedArticleID="1062124" data-doi="10.1103/physreve.67.056104"><li><a href="http://dx.doi.org/10.1103/physreve.67.056104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Growing+networks+with+local+rules%3A+preferential+attachment%2C+clustering+hierarchy+and+degree+correlations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Growing+networks+with+local+rules%3A+preferential+attachment%2C+clustering+hierarchy+and+degree+correlations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001667-Costa1" id="pone.0001667-Costa1"></a>Costa LF, Rodrigues FA, Travieso G, Boas V (2007) Characterization of complex networks: A survey of measurements. ADV PHYS  56: 167–242.  <ul class="find" data-citedArticleID="1062066" data-doi="10.1080/00018730601170527"><li><a href="http://dx.doi.org/10.1080/00018730601170527" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+complex+networks%3A+A+survey+of+measurements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+complex+networks%3A+A+survey+of+measurements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001667-Maslov1" id="pone.0001667-Maslov1"></a>Maslov S, Sneppen K (2002) Specificity and stability in topology of protein networks. Science  296: 910–913.  <ul class="find" data-citedArticleID="1062102" data-doi="10.1126/science.1065103"><li><a href="http://dx.doi.org/10.1126/science.1065103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specificity+and+stability+in+topology+of+protein+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specificity+and+stability+in+topology+of+protein+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001667-PastorSatorras1" id="pone.0001667-PastorSatorras1"></a>Pastor-Satorras R, Vazquez A, Vespignani A (2001) Dynamical and correlation properties of the internet. Phys Rev Lett  87: 258701.  <ul class="find" data-citedArticleID="1062108" data-doi="10.1103/physrevlett.87.258701"><li><a href="http://dx.doi.org/10.1103/physrevlett.87.258701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamical+and+correlation+properties+of+the+internet." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamical+and+correlation+properties+of+the+internet.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001667-Barabasi2" id="pone.0001667-Barabasi2"></a>Barabasi AL, Albert R (1999) Emergence of scaling in random networks. Science  286: 509–512.  <ul class="find" data-citedArticleID="1062058" data-doi="10.1126/science.286.5439.509"><li><a href="http://dx.doi.org/10.1126/science.286.5439.509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emergence+of+scaling+in+random+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emergence+of+scaling+in+random+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001667-PastorSatorras2" id="pone.0001667-PastorSatorras2"></a>Pastor-Satorras R, Smith E, Sole RV (2003) Evolving protein interaction networks through gene duplication. J Theor Biol  222: 199–210.  <ul class="find" data-citedArticleID="1062110" data-doi="10.1016/s0022-5193(03)00028-6"><li><a href="http://dx.doi.org/10.1016/s0022-5193(03)00028-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolving+protein+interaction+networks+through+gene+duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolving+protein+interaction+networks+through+gene+duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001667-Kim1" id="pone.0001667-Kim1"></a>Kim J, Krapivsky PL, Kahng B, Render S (2002) Infinite-order percolation and giant fluctuations in a protein interaction network. Phys Rev E  66: 055101.  <ul class="find" data-citedArticleID="1062098" data-doi="10.1103/physreve.66.055101"><li><a href="http://dx.doi.org/10.1103/physreve.66.055101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Infinite-order+percolation+and+giant+fluctuations+in+a+protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Infinite-order+percolation+and+giant+fluctuations+in+a+protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001667-Chung1" id="pone.0001667-Chung1"></a>Chung F, Lu L, Dewey TG, Galas DJ (2003) Duplication models for biological networks. J Comput Biol  10: 677–87.  <ul class="find" data-citedArticleID="1062060" data-doi="10.1089/106652703322539024"><li><a href="http://dx.doi.org/10.1089/106652703322539024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duplication+models+for+biological+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duplication+models+for+biological+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001667-Raval1" id="pone.0001667-Raval1"></a>Raval A (2003) Some asymptotic properties of duplication graph. Phys Rev E  68: 066119.  <ul class="find" data-citedArticleID="1062116" data-doi="10.1103/physreve.68.066119"><li><a href="http://dx.doi.org/10.1103/physreve.68.066119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Some+asymptotic+properties+of+duplication+graph." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Some+asymptotic+properties+of+duplication+graph.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001667-Middendorf1" id="pone.0001667-Middendorf1"></a>Middendorf M, Ziv E, Wiggins CH (2005) Inferring network mechanisms: The Drosophila melanogaster protein interaction network. Proc Natl Acad Sci USA  102: 3192–3197.  <ul class="find" data-citedArticleID="1062104" data-doi="10.1073/pnas.0409515102"><li><a href="http://dx.doi.org/10.1073/pnas.0409515102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+network+mechanisms%3A+The+Drosophila+melanogaster+protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+network+mechanisms%3A+The+Drosophila+melanogaster+protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001667-Ispolatov1" id="pone.0001667-Ispolatov1"></a>Ispolatov I, Krapivsky PL, Mazo I, Yuryev A (2005) Cliques and duplication-divergence network growth. New J Phys  7: 145.  <ul class="find" data-citedArticleID="1062084" data-doi="10.1088/1367-2630/7/1/145"><li><a href="http://dx.doi.org/10.1088/1367-2630/7/1/145" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cliques+and+duplication-divergence+network+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cliques+and+duplication-divergence+network+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001667-Vazquez2" id="pone.0001667-Vazquez2"></a>Vazquez A, Flammini A, Maritan A, Vespignani A (2003) Modeling of protein interaction networks. ComPlexUs  1: 38–44.  <ul class="find" data-citedArticleID="1062126" data-doi="10.1159/000067642"><li><a href="http://dx.doi.org/10.1159/000067642" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+of+protein+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+of+protein+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001667-Patil1" id="pone.0001667-Patil1"></a>Patil A, Nakamura H (2005) Filtering high-throughput protein-protein interaction data using a combination of genomic features. BMC Bioinformatics  6: 100.  <ul class="find" data-citedArticleID="1062112"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Filtering+high-throughput+protein-protein+interaction+data+using+a+combination+of+genomic+features.&amp;auth=&amp;atitle=Filtering+high-throughput+protein-protein+interaction+data+using+a+combination+of+genomic+features." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Filtering+high-throughput+protein-protein+interaction+data+using+a+combination+of+genomic+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Filtering+high-throughput+protein-protein+interaction+data+using+a+combination+of+genomic+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001667-Ispolatov2" id="pone.0001667-Ispolatov2"></a>Ispolatov I, Yuryev A, Mazo I, Maslov S (2005) Binding properties and evolution of homodimers in protein-protein interaction networks. Nucleic Acids Res  33: 3629–3635.  <ul class="find" data-citedArticleID="1062086" data-doi="10.1093/nar/gki678"><li><a href="http://dx.doi.org/10.1093/nar/gki678" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Binding+properties+and+evolution+of+homodimers+in+protein-protein+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Binding+properties+and+evolution+of+homodimers+in+protein-protein+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001667-RodriguezCaso1" id="pone.0001667-RodriguezCaso1"></a>Rodriguez-Caso C, Medina MA, Sole RV (2005) Topology, tinkering and evolution of the human transcription factor networks. FEBS J  272: 6423–34.  <ul class="find" data-citedArticleID="1062118" data-doi="10.1111/j.1742-4658.2005.05041.x"><li><a href="http://dx.doi.org/10.1111/j.1742-4658.2005.05041.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Topology%2C+tinkering+and+evolution+of+the+human+transcription+factor+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Topology%2C+tinkering+and+evolution+of+the+human+transcription+factor+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001667-Prachumwat1" id="pone.0001667-Prachumwat1"></a>Prachumwat A, Li WH (2006) Protein function, connectivity, and duplicability in yeast. Mol Biol Evol  23: 30–39.  <ul class="find" data-citedArticleID="1062114" data-doi="10.1093/molbev/msi249"><li><a href="http://dx.doi.org/10.1093/molbev/msi249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+function%2C+connectivity%2C+and+duplicability+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+function%2C+connectivity%2C+and+duplicability+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001667-Fraser1" id="pone.0001667-Fraser1"></a>Fraser HB, Hirsh AE, Steinmetz LM, Scharfe C, Feldman MW (2002) Evolutionary rate in the protein interaction network. Science  296: 750–752.  <ul class="find" data-citedArticleID="1062072" data-doi="10.1126/science.1068696"><li><a href="http://dx.doi.org/10.1126/science.1068696" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+rate+in+the+protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+rate+in+the+protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001667-Fraser2" id="pone.0001667-Fraser2"></a>Fraser HB (2003) A simple dependence between protein evolution rate and the number of protein-protein interactions. BMC Evol Biol  3: 11.  <ul class="find" data-citedArticleID="1062074"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple+dependence+between+protein+evolution+rate+and+the+number+of+protein-protein+interactions.&amp;auth=&amp;atitle=A+simple+dependence+between+protein+evolution+rate+and+the+number+of+protein-protein+interactions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple+dependence+between+protein+evolution+rate+and+the+number+of+protein-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple+dependence+between+protein+evolution+rate+and+the+number+of+protein-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001667-Fraser3" id="pone.0001667-Fraser3"></a>Fraser HB (2005) Modularity and evolutionary constraint on proteins. Nature Genetics  37: 351–352.  <ul class="find" data-citedArticleID="1062076" data-doi="10.1038/ng1530"><li><a href="http://dx.doi.org/10.1038/ng1530" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modularity+and+evolutionary+constraint+on+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modularity+and+evolutionary+constraint+on+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001667-Jordan1" id="pone.0001667-Jordan1"></a>Jordan IK, Wolf YI, Koonin EV (2003) No simple dependence between protein evolution rate and the number of protein-protein interactions: only the most prolific interactors tend to evolve slowly. BMC Evol Biol  3: 1.  <ul class="find" data-citedArticleID="1062096"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=No+simple+dependence+between+protein+evolution+rate+and+the+number+of+protein-protein+interactions%3A+only+the+most+prolific+interactors+tend+to+evolve+slowly.&amp;auth=&amp;atitle=No+simple+dependence+between+protein+evolution+rate+and+the+number+of+protein-protein+interactions%3A+only+the+most+prolific+interactors+tend+to+evolve+slowly." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=No+simple+dependence+between+protein+evolution+rate+and+the+number+of+protein-protein+interactions%3A+only+the+most+prolific+interactors+tend+to+evolve+slowly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22No+simple+dependence+between+protein+evolution+rate+and+the+number+of+protein-protein+interactions%3A+only+the+most+prolific+interactors+tend+to+evolve+slowly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001667-Chung2" id="pone.0001667-Chung2"></a>Chung F, Lu L (2002) The average distances in random graphs with given expected degrees. Proc Natl Acad Sci USA  99: 15879–82.  <ul class="find" data-citedArticleID="1062062" data-doi="10.1073/pnas.252631999"><li><a href="http://dx.doi.org/10.1073/pnas.252631999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+average+distances+in+random+graphs+with+given+expected+degrees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+average+distances+in+random+graphs+with+given+expected+degrees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001667-Cohen1" id="pone.0001667-Cohen1"></a>Cohen R, Havlin S (2003) Scale-free networks are ultrasmall. Phys Rev Lett  90: 058701.  <ul class="find" data-citedArticleID="1062064" data-doi="10.1103/physrevlett.90.058701"><li><a href="http://dx.doi.org/10.1103/physrevlett.90.058701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scale-free+networks+are+ultrasmall." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scale-free+networks+are+ultrasmall.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001667-Han1" id="pone.0001667-Han1"></a>Han JD, Bertin N, Hao T, Goldberg DS, Berriz GF, et al.  (2004) Evidence for dynamically organized modularity in the yeast protein-protein interaction network. Nature  430: 88–93.  <ul class="find" data-citedArticleID="1062082" data-doi="10.1038/nature02555"><li><a href="http://dx.doi.org/10.1038/nature02555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+dynamically+organized+modularity+in+the+yeast+protein-protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+dynamically+organized+modularity+in+the+yeast+protein-protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001667-Altschul1" id="pone.0001667-Altschul1"></a>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J Mol Biol  215: 403–410.  <ul class="find" data-citedArticleID="1062054" data-doi="10.1016/s0022-2836(05)80360-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80360-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Basic+local+alignment+search+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Basic+local+alignment+search+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001667-Forster1" id="pone.0001667-Forster1"></a>Forster AC, Church GM (2006) Towards synthesis of a minimal cell. Mol Syst Biol  2: 45.  <ul class="find" data-citedArticleID="1062070" data-doi="10.1038/msb4100090"><li><a href="http://dx.doi.org/10.1038/msb4100090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+synthesis+of+a+minimal+cell." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+synthesis+of+a+minimal+cell.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001667-Ispolatov3" id="pone.0001667-Ispolatov3"></a>Ispolatov I, Krapivsky PL, Yuryev A (2005) Duplication-divergence model of protein interaction network. Phys Rev E  71: 061911.  <ul class="find" data-citedArticleID="1062088" data-doi="10.1103/physreve.71.061911"><li><a href="http://dx.doi.org/10.1103/physreve.71.061911" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duplication-divergence+model+of+protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duplication-divergence+model+of+protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.XML" value="92024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.PDF" value="242073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g001.PNG_L" value="138420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g001.PNG_M" value="62620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g001.PNG_S" value="13954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g001.TIF" value="286154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g001.PNG_I" value="23322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.t001.PNG_L" value="203096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.t001.PNG_M" value="118156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.t001.PNG_S" value="12859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.t001.TIF" value="945824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.t001.PNG_I" value="36360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g002.PNG_L" value="262655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g002.PNG_M" value="127747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g002.PNG_S" value="16067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g002.TIF" value="833348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g002.PNG_I" value="46662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g003.PNG_L" value="63777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g003.PNG_M" value="55199"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g003.PNG_S" value="10593"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g003.TIF" value="358900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.g003.PNG_I" value="19869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.s001.DOC" value="40960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.s002.TIF" value="782970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001667.s003.TIF" value="822010"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001667&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001667" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001667&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001667&volume=&issue=&title=Non-Uniform Survival Rate of Heterodimerization Links in the Evolution of the Yeast Protein-Protein Interaction Network&author_name=Takeshi%20Hase%2C%20Yoshihito%20Niimura%2C%20Tsuguchika%20Kaminuma%2C%20Hiroshi%20Tanaka&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001667" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001667" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001667" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001667&amp;t=Non-Uniform%20Survival%20Rate%20of%20Heterodimerization%20Links%20in%20the%20Evolution%20of%20the%20Yeast%20Protein-Protein%20Interaction%20Network" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001667&title=Non-Uniform%20Survival%20Rate%20of%20Heterodimerization%20Links%20in%20the%20Evolution%20of%20the%20Yeast%20Protein-Protein%20Interaction%20Network&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001667&amp;title=Non-Uniform%20Survival%20Rate%20of%20Heterodimerization%20Links%20in%20the%20Evolution%20of%20the%20Yeast%20Protein-Protein%20Interaction%20Network" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001667" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001667&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Non-Uniform Survival Rate of Heterodimerization Links in the Evolution of the Yeast Protein-Protein Interaction Network';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001667';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Non-Uniform%20Survival%20Rate%20of%20Heterodimerization%20Links%20in%20the%20Evolution%20of%20the%20Yeast%20Protein-Protein%20Interaction%20Network http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001667" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001667" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+evolution%22" title="Search for articles in the subject area:'Fungal evolution'"><div class="flagText">Fungal evolution</div></a>
              <div data-categoryid="17551" data-articleid="26160"
                   data-categoryname="Fungal evolution"
                   class="flagImage" title="Flag 'Fungal evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="26160"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interaction+networks%22" title="Search for articles in the subject area:'Protein interaction networks'"><div class="flagText">Protein interaction networks</div></a>
              <div data-categoryid="31803" data-articleid="26160"
                   data-categoryname="Protein interaction networks"
                   class="flagImage" title="Flag 'Protein interaction networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interactions%22" title="Search for articles in the subject area:'Protein interactions'"><div class="flagText">Protein interactions</div></a>
              <div data-categoryid="17661" data-articleid="26160"
                   data-categoryname="Protein interactions"
                   class="flagImage" title="Flag 'Protein interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Scale-free+networks%22" title="Search for articles in the subject area:'Scale-free networks'"><div class="flagText">Scale-free networks</div></a>
              <div data-categoryid="22077" data-articleid="26160"
                   data-categoryname="Scale-free networks"
                   class="flagImage" title="Flag 'Scale-free networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Simulation+and+modeling%22" title="Search for articles in the subject area:'Simulation and modeling'"><div class="flagText">Simulation and modeling</div></a>
              <div data-categoryid="8375" data-articleid="26160"
                   data-categoryname="Simulation and modeling"
                   class="flagImage" title="Flag 'Simulation and modeling' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Spatial+distribution%22" title="Search for articles in the subject area:'Spatial distribution'"><div class="flagText">Spatial distribution</div></a>
              <div data-categoryid="31855" data-articleid="26160"
                   data-categoryname="Spatial distribution"
                   class="flagImage" title="Flag 'Spatial distribution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast+and+fungal+models%22" title="Search for articles in the subject area:'Yeast and fungal models'"><div class="flagText">Yeast and fungal models</div></a>
              <div data-categoryid="35135" data-articleid="26160"
                   data-categoryname="Yeast and fungal models"
                   class="flagImage" title="Flag 'Yeast and fungal models' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7812'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1460'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7661&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=17009">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
