

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Three-Stage Colonization Model for the Peopling of the Americas</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001596"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001596" />

    <meta name="citation_title" content="A Three-Stage Colonization Model for the Peopling of the Americas"/>
    <meta itemprop="name" content="A Three-Stage Colonization Model for the Peopling of the Americas"/>

      <meta name="citation_author" content="Andrew Kitchen"/>
            <meta name="citation_author_institution" content="Department of Anthropology, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Michael M. Miyamoto"/>
            <meta name="citation_author_institution" content="Department of Zoology, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Connie J. Mulligan"/>
            <meta name="citation_author_institution" content="Department of Anthropology, University of Florida, Gainesville, Florida, United States of America"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001596.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1596"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The settlement of the Americas: a comparison of the linguistic, dental and genetic evidence.; citation_author=JH Greenberg; citation_author=CG Turner; citation_author=SL Zegura; citation_journal_title=Curr Anthropol; citation_volume=27; citation_number=1; citation_pages=477-497; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=The peopling of the New World: perspectives from molecular anthropology.; citation_author=TG Schurr; citation_journal_title=Annu Rev Anthropol; citation_volume=33; citation_number=2; citation_pages=551-583; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Population genetics, history, and health patterns in native Americans.; citation_author=CJ Mulligan; citation_author=K Hunley; citation_author=S Cole; citation_author=JC Long; citation_journal_title=Annu Rev Genom Hum Genet; citation_volume=5; citation_number=3; citation_pages=295-315; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic variation and population structure in Native Americans.; citation_author=S Wang; citation_author=CM Lewis; citation_author=M Jakobsson; citation_author=S Ramachandran; citation_author=N Ray; citation_journal_title=PLoS Genet; citation_volume=3; citation_number=4; citation_pages=e185; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A single and early migration for the peopling of the Americas supported by mitochondrial DNA sequence data.; citation_author=SL Bonatto; citation_author=FM Salzano; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=94; citation_number=5; citation_pages=1866-1871; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Diversity and age of the four major mtDNA haplogroups, and their implications for the peopling of the New World.; citation_author=SL Bonatto; citation_author=FM Salzano; citation_journal_title=Am J Hum Genet; citation_volume=61; citation_number=6; citation_pages=1413-1423; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Origin and evolution of Native American mtDNA variation: a reappraisal.; citation_author=P Forster; citation_author=R Harding; citation_author=A Torroni; citation_author=H-J Bandelt; citation_journal_title=Am J Hum Genet; citation_volume=59; citation_number=7; citation_pages=935-945; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The central Siberian origin for Native American Y chromosomes.; citation_author=FR Santos; citation_author=A Pandya; citation_author=C Tyler-Smith; citation_author=SDJ Pena; citation_author=M Schanfield; citation_journal_title=Am J Hum Genet; citation_volume=64; citation_number=8; citation_pages=619-628; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA and Y chromosome diversity and the peopling of the Americas: evolutionary and demographic evidence.; citation_author=TG Schurr; citation_author=SS Sherry; citation_journal_title=Am J Hum Biol; citation_volume=16; citation_number=9; citation_pages=420-439; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=mtDNA sequences suggest a recent evolutionary divergence for Beringian and northern North American populations.; citation_author=GF Shields; citation_author=AM Schmiechen; citation_author=BL Frazier; citation_author=A Redd; citation_author=MI Voevoda; citation_journal_title=Am J Hum Genet; citation_volume=53; citation_number=10; citation_pages=549-562; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial genome diversity of Native Americans supports a single early entry of founder populations into America.; citation_author=WA Silva; citation_author=SL Bonatto; citation_author=AJ Holanda; citation_author=AK Ribeiro-dos-Santos; citation_author=BM Paixao; citation_journal_title=Am J Hum Genet; citation_volume=71; citation_number=11; citation_pages=187-192; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genetics of aboriginal North Americans.; citation_author=EJE Szathmary; citation_journal_title=Evol Anthropol; citation_volume=1; citation_number=12; citation_pages=202-220; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Beringian standstill and spread of Native American founders.; citation_author=E Tamm; citation_author=T Kivisild; citation_author=M Reidla1; citation_author=M Metspalu1; citation_author=DG Smith; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=13; citation_pages=e829; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Asian affinities and continental radiation of the four founding Native American mtDNAs.; citation_author=A Torroni; citation_author=TG Schurr; citation_author=MF Cabell; citation_author=MD Brown; citation_author=JV Neel; citation_journal_title=Am J Hum Genet; citation_volume=53; citation_number=14; citation_pages=563-590; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=mtDNA variation of aboriginal Siberians reveals distinct genetic affinities with Native Americans.; citation_author=A Torroni; citation_author=RI Sukernik; citation_author=TG Schurr; citation_author=YB Starikovskaya; citation_author=MF Cabell; citation_journal_title=Am J Hum Genet; citation_volume=53; citation_number=15; citation_pages=591-608; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=On the number of New World founders: a population genetic portrait of the peopling of the Americas.; citation_author=J Hey; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=16; citation_pages=e193; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Spatial gradients in Clovis-age radiocarbon dates across North America suggest rapid colonization from the north.; citation_author=MJ Hamilton; citation_author=B Buchanan; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=104; citation_number=17; citation_pages=15625-15630; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Redefining the age of Clovis: implications for the peopling of the Americas.; citation_author=MR Waters; citation_author=TW Stafford; citation_journal_title=Science; citation_volume=315; citation_number=18; citation_pages=1122-1126; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The archaeological context and interpretation.; citation_number=19; citation_date=1997; citation_publisher=Smithsonian Institution Press; " />
      <meta name="citation_reference" content="citation_title=Chronology and new research on the Schaefer mammoth (Mammuthus primigenius) site, Kenosha County, Wisconsin, USA.; citation_author=DJ Joyce; citation_journal_title=Quat Intl; citation_volume=142; citation_number=20; citation_pages=44-57; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Late-glacial ice-marginal adaptation in southeastern Wisconsin.; citation_author=DF Overstreet; citation_number=21; citation_pages=183-195; citation_date=2005; citation_publisher=Center for the Study of the First Americans; " />
      <meta name="citation_reference" content="citation_title=The missing years for modern humans.; citation_author=T Goebel; citation_journal_title=Science; citation_volume=315; citation_number=22; citation_pages=194-196; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Yana RHS site: humans in the Arctic before the last glacial maximum.; citation_author=VV Pitulko; citation_author=PA Nikolsky; citation_author=EY Girya; citation_author=AE Basilyan; citation_author=VE Tumskoy; citation_journal_title=Science; citation_volume=303; citation_number=23; citation_pages=52-56; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Life and times of the Bering land bridge.; citation_author=SA Elias; citation_author=SK Short; citation_author=CH Nelson; citation_author=HH Birks; citation_journal_title=Nature; citation_volume=382; citation_number=24; citation_pages=60-63; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Aspects of the paleogeography of Beringia during the Late Pleistocene.; citation_author=DM Hopkins; citation_number=25; citation_pages=3-28; citation_date=1982; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=The colonization of Beringia and the peopling of the New World.; citation_author=JF Hoffecker; citation_author=WR Powers; citation_author=T Goebel; citation_journal_title=Science; citation_volume=259; citation_number=26; citation_pages=46-53; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Ice-age steppe vegetation in east Beringia.; citation_author=GD Zazula; citation_author=DG Froese; citation_author=CE Schweger; citation_author=RW Mathewes; citation_author=AB Beaudoin; citation_journal_title=Nature; citation_volume=426; citation_number=27; citation_pages=603; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Frozen fauna of the mammoth steppe.; citation_author=RD Guthrie; citation_number=28; citation_date=1990; citation_publisher=University of Chicago Press; " />
      <meta name="citation_reference" content="citation_title=Into new found lands. Human Evolutionary Genetics.; citation_author=MA Jobling; citation_author=ME Hurles; citation_author=C Tyler-Smith; citation_number=29; citation_pages=339-372; citation_date=2004; citation_publisher=Garland Science; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA and human evolution.; citation_author=B Pakendorf; citation_author=M Stoneking; citation_journal_title=Annu Rev Genom Hum Genet; citation_volume=6; citation_number=30; citation_pages=165-183; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA and two perspectives on evolutionary genetics.; citation_author=A Wilson; citation_author=R Cann; citation_author=S Carr; citation_author=M George; citation_author=U Gyllensten; citation_journal_title=Biol J Linn Soc; citation_volume=26; citation_number=31; citation_pages=375-400; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Bayesian coalescent inference of past population dynamics from molecular sequences.; citation_author=AJ Drummond; citation_author=A Rambaut; citation_author=B Shapiro; citation_author=OG Pybus; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=32; citation_pages=1185-1192; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of the most informative regions of the mitochondrial genome for phylogenetic and coalescent analyses.; citation_author=AL Non; citation_author=A Kitchen; citation_author=CJ Mulligan; citation_journal_title=Mol Phylogenet Evol; citation_volume=44; citation_number=33; citation_pages=1164-1171; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Dates are not just data: paleolithic settlement patterns in Siberia derived from radiocarbon records.; citation_author=YV Kuzmin; citation_author=SG Keates; citation_journal_title=Am Antiquity; citation_volume=70; citation_number=34; citation_pages=773-789; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogeographic analysis of mitochondrial DNA in northern Asian Populations.; citation_author=M Derenko; citation_author=B Malyarchuk; citation_author=T Grzybowski; citation_author=G Denisova; citation_author=I Dambueva; citation_journal_title=Am J Hum Genet; citation_volume=81; citation_number=35; citation_pages=1025-1041; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The structure of diversity within new world mitochondrial DNA haplogroups: Implications for the prehistory of North America.; citation_author=RS Malhi; citation_author=JA Eshleman; citation_author=JA Greenberg; citation_author=DA Weiss; citation_author=BAS Shook; citation_journal_title=Am J Hum Genet; citation_volume=70; citation_number=36; citation_pages=905-919; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Spatial and temporal stability of mtDNA haplogroup frequencies in native North America.; citation_author=DH O'Rourke; citation_author=MG Hayes; citation_author=SW Carlyle; citation_journal_title=Hum Biol; citation_volume=72; citation_number=37; citation_pages=15-34; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Environment and archaeology in Beringia.; citation_author=JF Hoffecker; citation_author=SA Elias; citation_journal_title=Evol Anthropol; citation_volume=12; citation_number=38; citation_pages=34-49; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Late Quaternary paleoenvironments of Northwestern North America: implications for inland versus coastal migration routes.; citation_author=CAS Mandryk; citation_author=H Josenhans; citation_author=DW Fedje; citation_author=RW Mathewes; citation_journal_title=Quat Sci Rev; citation_volume=20; citation_number=39; citation_pages=301-314; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Reduced mtDNA diversity in the Ngobe Amerinds of Panama.; citation_author=CJ Kolman; citation_author=E Bermingham; citation_author=R Cooke; citation_author=RH Ward; citation_author=TD Arias; citation_journal_title=Genetics; citation_volume=140; citation_number=40; citation_pages=273-283; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA analysis of Mongolian populations and implications for the origin of New World founders.; citation_author=CJ Kolman; citation_author=N Sambuughin; citation_author=E Bermingham; citation_journal_title=Genetics; citation_volume=142; citation_number=41; citation_pages=1321-1334; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Distribution of the 4 founding lineage haplotypes in Native Americans suggests a single wave of migration for the New World.; citation_author=DA Merriwether; citation_author=F Rothhamer; citation_author=RE Ferrell; citation_journal_title=Am J Phys Anthropol; citation_volume=98; citation_number=42; citation_pages=411-430; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Rapid deployment of the five founding Amerind mtDNA haplogroups via coastal and riverine colonization.; citation_author=AG Fix; citation_journal_title=Am J Phys Anthropol; citation_volume=128; citation_number=43; citation_pages=430-436; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Human races: a genetic and evolutionary perspective.; citation_author=AR Templeton; citation_journal_title=Am Anthropol; citation_volume=100; citation_number=44; citation_pages=632-650; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Reduced-median-network analysis of complete mitochondrial DNA coding-region sequences for the major African, Asian, and European haplogroups.; citation_author=C Herrnstadt; citation_author=JL Elson; citation_author=E Fahy; citation_author=G Preston; citation_author=DM Turnbull; citation_journal_title=Am J Hum Genet; citation_volume=70; citation_number=45; citation_pages=1152-1171; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial genome variation and the origin of modern humans.; citation_author=M Ingman; citation_author=H Kaessmann; citation_author=S Pääbo; citation_author=U Gyllensten; citation_journal_title=Nature; citation_volume=408; citation_number=46; citation_pages=708-713; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The role of selection in the evolution of human mitochondrial genomes.; citation_author=T Kivisild; citation_author=P Shen; citation_author=DP Wall; citation_author=B Do; citation_author=R Sung; citation_journal_title=Genetics; citation_volume=172; citation_number=47; citation_pages=373-387; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Natural selection shaped regional mtDNA variation in humans.; citation_author=D Mishmar; citation_author=E Ruiz-Pesini; citation_author=P Golik; citation_author=V Macaulay; citation_author=A Clark; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=100; citation_number=48; citation_pages=171-176; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=CLUSTALW: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucl Acids Res; citation_volume=22; citation_number=49; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Compilation of human mtDNA control region sequences.; citation_author=O Handt; citation_author=S Meyer; citation_author=A von Haeseler; citation_journal_title=Nucl Acids Res; citation_volume=26; citation_number=50; citation_pages=126-129; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Application of a time-dependent coalescence process for inferring the history of population size changes from DNA sequence data.; citation_author=A Polanski; citation_author=M Kimmel; citation_author=R Chakraborty; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=95; citation_number=51; citation_pages=5456-5461; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Utility of DNA viruses for studying human host history: Case study of JC virus.; citation_author=A Kitchen; citation_author=MM Miyamoto; citation_author=CJ Mulligan; citation_journal_title=Mol Phylogenet Evol: In Press; citation_number=52; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The pedigree rate of sequence divergence in the human mitochondrial genome: there is a difference between phylogenetic and pedigree rates.; citation_author=N Howell; citation_author=CB Smejkal; citation_author=DA Mackey; citation_author=PF Chinnery; citation_author=DM Turnbull; citation_journal_title=Am J Hum Genet; citation_volume=72; citation_number=53; citation_pages=659-670; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Y chromosome evidence for Anglo-Saxon mass migration.; citation_author=ME Weale; citation_author=DA Weiss; citation_author=RF Jager; citation_author=N Bradman; citation_author=MG Thomas; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=54; citation_pages=1008-1021; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cross-cultural estimation of the human generation interval for use in genetics-based population divergence studies.; citation_author=JN Fenner; citation_journal_title=Am J Phys Anthropol; citation_volume=128; citation_number=55; citation_pages=415-423; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sequence and organization of the human mitochondrial genome.; citation_author=S Anderson; citation_author=AT Bankier; citation_author=BG Barrell; citation_author=MH de Bruijn; citation_author=AR Coulson; citation_journal_title=Nature; citation_volume=290; citation_number=56; citation_pages=457-465; citation_date=1981; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001596" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Three-Stage Colonization Model for the Peopling of the Americas"/>
    <meta name="twitter:description" content="BackgroundWe evaluate the process by which the Americas were originally colonized and propose a three-stage model that integrates current genetic, archaeological, geological, and paleoecological data. Specifically, we analyze mitochondrial and nuclear genetic data by using complementary coalescent models of demographic history and incorporating non-genetic data to enhance the anthropological relevance of the analysis.Methodology/FindingsBayesian skyline plots, which provide dynamic representations of population size changes over time, indicate that Amerinds went through two stages of growth &asymp;40,000 and &asymp;15,000 years ago separated by a long period of population stability. Isolation-with-migration coalescent analyses, which utilize data from sister populations to estimate a divergence date and founder population sizes, suggest an Amerind population expansion starting &asymp;15,000 years ago.Conclusions/SignificanceThese results support a model for the peopling of the New World in which Amerind ancestors diverged from the Asian gene pool prior to 40,000 years ago and experienced a gradual population expansion as they moved into Beringia. After a long period of little change in population size in greater Beringia, Amerinds rapidly expanded into the Americas &asymp;15,000 years ago either through an interior ice-free corridor or along the coast. This rapid colonization of the New World was achieved by a founder group with an effective population size of &asymp;1,000&ndash;5,400 individuals. Our model presents a detailed scenario for the timing and scale of the initial migration to the Americas, substantially refines the estimate of New World founders, and provides a unified theory for testing with future datasets and analytic methods."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001596.g004"/>

  <meta property="og:title" content="A Three-Stage Colonization Model for the Peopling of the Americas" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001596" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4690'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9721'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7313&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001596">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001596" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001596&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Three-Stage Colonization Model for the Peopling of the Americas
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrew Kitchen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Anthropology, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael M. Miyamoto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Zoology, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Connie J. Mulligan
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mulligan@anthro.ufl.edu">mulligan@anthro.ufl.edu</a></p>

                <p>Affiliation:
                  Department of Anthropology, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001596</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001596" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001596" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001596" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001596" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001596-g001" data-doi="info:doi/10.1371/journal.pone.0001596" data-uri="info:doi/10.1371/journal.pone.0001596.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001596-g002" data-doi="info:doi/10.1371/journal.pone.0001596" data-uri="info:doi/10.1371/journal.pone.0001596.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001596-g003" data-doi="info:doi/10.1371/journal.pone.0001596" data-uri="info:doi/10.1371/journal.pone.0001596.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001596-g004" data-doi="info:doi/10.1371/journal.pone.0001596" data-uri="info:doi/10.1371/journal.pone.0001596.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001596">Reader Comments (7)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001596" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>We evaluate the process by which the Americas were originally colonized and propose a three-stage model that integrates current genetic, archaeological, geological, and paleoecological data. Specifically, we analyze mitochondrial and nuclear genetic data by using complementary coalescent models of demographic history and incorporating non-genetic data to enhance the anthropological relevance of the analysis.</p>


<h3>Methodology/Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Bayesian skyline plots, which provide dynamic representations of population size changes over time, indicate that Amerinds went through two stages of growth ≈40,000 and ≈15,000 years ago separated by a long period of population stability. Isolation-with-migration coalescent analyses, which utilize data from sister populations to estimate a divergence date and founder population sizes, suggest an Amerind population expansion starting ≈15,000 years ago.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>These results support a model for the peopling of the New World in which Amerind ancestors diverged from the Asian gene pool prior to 40,000 years ago and experienced a gradual population expansion as they moved into Beringia. After a long period of little change in population size in greater Beringia, Amerinds rapidly expanded into the Americas ≈15,000 years ago either through an interior ice-free corridor or along the coast. This rapid colonization of the New World was achieved by a founder group with an effective population size of ≈1,000–5,400 individuals. Our model presents a detailed scenario for the timing and scale of the initial migration to the Americas, substantially refines the estimate of New World founders, and provides a unified theory for testing with future datasets and analytic methods.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kitchen A, Miyamoto MM, Mulligan CJ (2008) A Three-Stage Colonization Model for the Peopling of the Americas. PLoS ONE 3(2):
          e1596.
            doi:10.1371/journal.pone.0001596</p><p><strong>Academic Editor: </strong>Henry Harpending, University of Utah, United States of America</p><p><strong>Received:</strong> January 8, 2008; <strong>Accepted:</strong> January 16, 2008; <strong>Published:</strong> February 13, 2008</p><p><strong>Copyright:</strong> © 2008 Kitchen et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by a grant from the National Science Foundation to CJM (BSR-0518530) and by funds from the Department of Zoology, University of Florida to MMM.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>For decades, intense and interdisciplinary attention has focused on the colonization of the last habitable landmass on the planet-the peopling of the Americas. The first comprehensive, interdisciplinary model for New World colonization incorporated linguistic, paleoanthropological, and genetic data and generated great controversy, which was due at least in part, to the uniquely broad scope of the research <a href="#pone.0001596-Greenberg1">[1]</a>. Since that time, more focused studies have resulted in agreement on the general parameters of the colonization process, such as a single migration in contrast to the original three-migration model that distinguished Amerinds, Na-Dene, and Eskimo-Aleuts <a href="#pone.0001596-Greenberg1">[1]</a>. However, a full understanding of the complex and dynamic nature of the timing and magnitude of the colonization process remains elusive.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The majority of the genetic literature supports a single migration of Paleoindians into the New World from an East Asian source population <a href="#pone.0001596-Schurr1">[2]</a>. Specifically, the reduced variation and ubiquitous distribution of mitochondrial and Y chromosome haplogroups and microsatellite diversity throughout the New World relative to Asia argue strongly for a single migration <a href="#pone.0001596-Mulligan1">[3]</a>, <a href="#pone.0001596-Wang1">[4]</a>. However, a great many models have been proposed that differ significantly in the timing and size of this migration event <a href="#pone.0001596-Schurr1">[2]</a>, <a href="#pone.0001596-Bonatto1">[5]</a>–<a href="#pone.0001596-Torroni2">[15]</a>. Different migration dates have been proposed ranging from ≈13 thousand years ago (kya) to ≈30–40 kya <a href="#pone.0001596-Schurr1">[2]</a>, <a href="#pone.0001596-Bonatto1">[5]</a>–<a href="#pone.0001596-Torroni2">[15]</a>. Numerical estimates of the founder effective population size (N<sub>e</sub>) are infrequent in the literature but vary substantially, from a high of ≈5000 <a href="#pone.0001596-Bonatto2">[6]</a> to a low of ≈70 Paleoindian founders <a href="#pone.0001596-Hey1">[16]</a>. These dates and population sizes have been proposed to accommodate a wealth of scenarios including ancient, recent, and/or additional migrations responsible for the peopling of the Americas.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Archaeological data provide clear support for a widespread human presence in the Americas by ≈13 kya (all calendar dates are recalibrated radiocarbon dates as reported in the cited literature), the time by which the Clovis complex was established across the interior of North America <a href="#pone.0001596-Hamilton1">[17]</a>, <a href="#pone.0001596-Waters1">[18]</a>. Older archaeological sites, e.g. the Nenana Complex in Alaska <a href="#pone.0001596-Waters1">[18]</a>, the Monte Verde site in Chile <a href="#pone.0001596-Dillehay1">[19]</a>, and the Schaefer, Hebior and Mud Lake sites in Wisconsin <a href="#pone.0001596-Joyce1">[20]</a>, <a href="#pone.0001596-Overstreet1">[21]</a>, document an earlier chronology possibly 2,400 years before Clovis <a href="#pone.0001596-Waters1">[18]</a>, <a href="#pone.0001596-Joyce1">[20]</a>, <a href="#pone.0001596-Overstreet1">[21]</a>. Additionally, very old radiocarbon dates have been obtained from sites in Asian Beringia suggesting that human populations had reached the north of western Beringia by ≈30 kya <a href="#pone.0001596-Goebel1">[22]</a>, <a href="#pone.0001596-Pitulko1">[23]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The geological and paleoecological records for Beringia and northwestern North America provide further constraints on the timing for the peopling of the Americas. Beringia was a continuous landmass that connected Asia and North America roughly 60 kya until ≈11–10 kya <a href="#pone.0001596-Pitulko1">[23]</a>–<a href="#pone.0001596-Hopkins1">[25]</a>. However, Beringia was isolated from continental North America until ≈14 kya when an intracontinental ice-free corridor opened up between the Laurentide and Cordilleran Ice Sheets <a href="#pone.0001596-Hoffecker1">[26]</a>. Paleoecological data indicate that Beringia was able to sustain at least small human populations. Fossil pollen and plant macrofossils from ancient eastern Beringia are indicative of a productive, dry grassland ecosystem <a href="#pone.0001596-Zazula1">[27]</a> and paleontological evidence from Alaska and Siberia demonstrates that large mammals roamed Beringia <a href="#pone.0001596-Guthrie1">[28]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>After 11–10 kya, Late Pleistocene sea levels rose sufficiently to re-inundate Beringia <a href="#pone.0001596-Elias1">[24]</a>, <a href="#pone.0001596-Hopkins1">[25]</a>, creating the Bering strait that now separates the New World from Siberia by at least 100 kilometers (km) of open frigid water. Studies of human settlement throughout the Pacific Islands indicate that open water distances of &gt;100 km constitute significant barriers to human migration, possibly because ancient people were unlikely to travel further than one day out of sight of land <a href="#pone.0001596-Jobling1">[29]</a>. Similar constraints (if not worse) would apply to early humans in Alaska and Siberia, thereby severely reducing the migration rate between the New and Old World once Beringia was re-inundated. Reduced migration due to the Bering Strait remains valid even as recent rates of short-range migration have increased between Siberia and Alaska <a href="#pone.0001596-Tamm1">[13]</a>. In effect, the two continents were essentially geographically isolated from 11–10 kya until modern times.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>No detailed, unified theory of New World colonization currently exists that can account for the breadth and complexity of these interdisciplinary data. We analyze Native American mitochondrial DNA (mtDNA) coding genomes plus non-coding control region sequences as well as a combined nuclear and mitochondrial coding DNA dataset from New World and Asian populations. Mitochondrial DNA data represent the ‘gold standard’ of genetic data types and provide the most extensive comparative database for human populations worldwide <a href="#pone.0001596-Pakendorf1">[30]</a>. Furthermore, it has been proposed that mtDNA may be more sensitive to demographic changes, such as population bottlenecks, due to its smaller effective population size <a href="#pone.0001596-Wilson1">[31]</a>. The combined nuclear and mtDNA dataset was recently used to propose an unusually small N<sub>e</sub> for the Amerind founders <a href="#pone.0001596-Hey1">[16]</a>, and thus investigation of this dataset is of much interest when attempting to reconcile the existing genetic evidence. We use two complementary coalescent methods to develop a comprehensive scenario of New World colonization, with a focus on the timing and scale of the migration process. Bayesian skyline plot analyses use data from a single population to provide an unbiased estimate of changes in N<sub>e</sub> through time, and thus are a powerful means for estimating past population growth patterns when the nature of the growth (e.g. exponential or constant) is unknown <a href="#pone.0001596-Drummond1">[32]</a>. The isolation-by-migration (IM) structured coalescent model uses data from sister populations to jointly estimate population divergence time, migration rates and a founder N<sub>e</sub>, with an assumption of exponential growth <a href="#pone.0001596-Hey1">[16]</a>. Importantly, we explicitly incorporate archaeological, geological, and paleoecological constraints into both analyses. Our goal is to provide a comprehensive model for the initial settlement of the Americas that generates new testable hypotheses and has high predictive power for the inclusion of new datasets. In light of our results, we propose a three-stage model in which a recent, rapid expansion into the Americas was preceded by a long period of population stability in greater Beringia by the Paleoindian population after divergence and expansion from their ancestral Asian population.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Skyline Plot Analyses</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Our alignment of 77 full mitochondrial coding genomes is one of the largest published alignments of Native American mtDNA coding genomes (<a href="#pone.0001596.s001">Figure S1</a>). It includes genomes from the four major mtDNA haplogroups in the Americas (haplogroups A, B, C, and D are each represented by 17–31% of the entire sample), as well as the minor haplogroup X (2%). Correspondingly, this set of 77 complete coding mtDNA genomes represents geographically and linguistically diverse populations distributed throughout the New World <a href="#pone.0001596-Mulligan1">[3]</a>. Bayesian skyline plots <a href="#pone.0001596-Drummond1">[32]</a> were used to visually illustrate changes in Amerind female effective population size (N<sub>ef</sub>) over time. Bayesian skyline plots assume a single migration event, which makes the approach ideal for questions concerning the peopling of the Americas since it is generally agreed that there was a single migration <a href="#pone.0001596-Mulligan1">[3]</a>. Our skyline plot of the coding genomes describes a three-stage process in which there are two distinct increases in N<sub>ef</sub> at ≈40 kya and ≈15 kya that are separated by a long period of little to no growth (<a href="#pone-0001596-g001">Figure 1</a>). Specifically, N<sub>ef</sub> increases from ≈640 [95% credible interval (CI) = 148–9,969] to ≈4,400 individuals (95% CI = 235–18,708) at the first inflection point, and from ≈4,000 (95% CI = 911–13,006) to ≈64,000 individuals (95% CI = 15,871–202,990) at the second inflection point. There is also an apparent decrease in N<sub>ef</sub> prior to the second inflection point in which median N<sub>ef</sub> drops to ≈2700 (95% CI = 404–36,628). We define a significant change in population size as the occurrence of non-overlapping 95% CIs at the beginning and end of an increase (see shading in <a href="#pone-0001596-g001">Figure 1</a>). Thus, we interpret the recent ≈16-fold increase in N<sub>ef</sub> over the interval ≈16–9 kya as significant. The earlier ≈7-fold increase at ≈43–36 kya is suggestive but not significant, although the increase is significant when compared over a much longer time period, e.g. from ≈25 kya to the coalescent. Overall, the recent increase is consistent with a rapid, large-scale expansion into the Americas while the older increase is suggestive of a gradual expansion within Asia or Beringia.</p>
<div class="figure" id="pone-0001596-g001"><div class="img"><a name="pone-0001596-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001596" data-uri="info:doi/10.1371/journal.pone.0001596.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001596.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001596.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Bayesian skyline plot for the mtDNA coding genome sequences.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The curve plots median N<sub>ef</sub> with its 95% CI indicated by the light gray lines. The calculated N<sub>ef</sub> assumes a generation time of 20 years following Hey <a href="#pone.0001596-Hey1">[16]</a>; alternatively, using a generation time of 25 years <a href="#pone.0001596-Fenner1">[55]</a> would uniformly decrease all estimates of N<sub>ef</sub> by 20%. “X” marks the median coalescent time with its 95% CI given in brackets. The shaded regions highlight two periods of substantial population growth. This skyline plot provides the principal evidence for our three-stage model of New World colonization, i.e. the three stages that are depicted and labeled here.</p>
<span>doi:10.1371/journal.pone.0001596.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The dataset of 812 concatenated mtDNA hypervariable region (HVR) I and II sequences is one of the largest published alignments of Native American HVRI+II sequences (<a href="#pone.0001596.s002">Figure S2</a>). It includes all major New World haplogroups, and represents geographically and linguistically diverse populations distributed throughout the Americas. The HVRI+II dataset was randomly divided into ten non-overlapping alignments of 81 HVRI+II sequences, which allowed for ten independent trials for parameter estimation with a sample size similar to the coding genome alignment. The HVRI+II skyline plot analyses (<a href="#pone-0001596-g002">Figure 2</a>) produce estimates for median time to coalescence (55.5 kya, 95% CI = 33.5–87.2 kya) and N<sub>ef</sub> at coalescence (820, 95% CI = 26–3,979) and the present (66,200, 95% CI = 9,839–346,289) that are similar to the coding genome analyses (<a href="#pone-0001596-g001">Figure 1</a>). However, in contrast to the coding genome skyline plot, the HVRI+II skyline plot traces a very gradual increase in N<sub>ef</sub> over ≈40,000 years with no clear inflection points. The HVRI+II plot does show a significant increase in N<sub>ef</sub> but only when measured over the past 35,000 years. The fine detail evidenced in the coding genome skyline plot likely reflects the greater phylogenetic signal in the mitochondrial coding genome relative to the HVR <a href="#pone.0001596-Non1">[33]</a>. In general, estimates of the time to the most recent common ancestor are less sensitive to reductions in the historical signal in mtDNA sequence data than phylogenetic estimation <a href="#pone.0001596-Non1">[33]</a>, a result consistent with our ability to recover similar coalescence times but not the changes in N<sub>ef</sub> seen when comparing the coding vs. HVRI+II skyline plots.</p>
<div class="figure" id="pone-0001596-g002"><div class="img"><a name="pone-0001596-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001596" data-uri="info:doi/10.1371/journal.pone.0001596.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001596.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001596.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Bayesian skyline plot for the mtDNA HVR I+II datasets.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>This plot follows the conventions of <a href="#pone-0001596-g001">Figure 1</a>. Its estimates of coalescent time and N<sub>ef</sub> at the coalescence and today are in agreement with the coding mtDNA skyline plot (<a href="#pone-0001596-g001">Figure 1</a>). In contrast, this HVRI+II plot provides little resolution for other population size changes, most likely because of mutational saturation in the non-coding control region (see text).</p>
<span>doi:10.1371/journal.pone.0001596.g002</span></div>

<h4>Isolation-with-Migration Coalescent Analyses</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Bayesian IM coalescent analyses were performed on a set of nine coding nuclear and mitochondrial loci that had been previously analyzed by Hey <a href="#pone.0001596-Hey1">[16]</a> in support of an extremely small New World founder N<sub>e</sub> of ≈70 individuals. Thus, we performed our analysis on his identical dataset and used the same coalescent and substitution models and model parameters with the exception of new priors on the divergence time and on migration rates between Asian and Amerind populations (<em>m</em><sub>Asia→NW</sub> and <em>m</em><sub>NW→Asia</sub>). The lower bound on divergence time was set to15 kya, which corresponds to the period immediately preceding the earliest archaeological evidence for human habitation in the Americas <a href="#pone.0001596-Waters1">[18]</a>–<a href="#pone.0001596-Overstreet1">[21]</a>. We also instituted serial constraints on <em>m</em> in order to gauge the effect of changing migration rates on founder N<sub>e</sub> estimates. We interpret the various <em>m</em> values in comparison to an empirical estimate of <em>m</em> for modern Europe (<em>m</em> = 4.3; see <a href="#s4">Materials and Methods</a>). In contrast to modern Europe, migration between the New World and Siberia from 15 kya to more recent times would have become increasingly limited as Late Pleistocene sea levels rose sufficiently to inundate the Bering land bridge <a href="#pone.0001596-Elias1">[24]</a>, <a href="#pone.0001596-Hopkins1">[25]</a>. Thus, we expect <em>m</em> for modern Europe to be much higher than ancient migration rates between Asia and the Americas, especially after the inundation of Beringia.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Constraining divergence time by applying a lower bound of 15 kya results in an estimate of ≈200 for the Amerind founding N<sub>e</sub>. Serially constraining <em>m</em><sub>Asia→NW</sub> and <em>m</em><sub>NW→Asia</sub>, in conjunction with the constrained divergence time, produces increasingly larger estimates of N<sub>e</sub> (<a href="#pone-0001596-g003">Figure 3</a>). Specifically, as both <em>m</em> parameters are simultaneously forced to lower and more biologically realistic values, estimates of N<sub>e</sub> steadily increase from ≈200 to ≈1,200, especially after their priors are constrained to be &lt;5. Regardless of the specific priors on the <em>m</em> parameters, estimates for the Amerind divergence/expansion event are consistently ≈15 kya (data not shown), which is very close to the lower bound of our prior established with known archaeological sites in the New World. Our results demonstrate that smaller estimates of N<sub>e</sub> depend upon a substantial level of migration from Asia to account for present-day levels of Amerind genetic diversity, e.g. Hey's <a href="#pone.0001596-Hey1">[16]</a> estimate of ≈70 founders is associated with a <em>m</em><sub>Asia→NW</sub>&gt;9.0, which is twice the migration rate for contemporary Europe (<em>m</em> = 4.3). Eliminating all migration between Asia and the New World (<em>m</em> = 0) results in the largest estimate of N<sub>e</sub> for the Amerind founding population of ≈1,200 individuals.</p>
<div class="figure" id="pone-0001596-g003"><div class="img"><a name="pone-0001596-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001596" data-uri="info:doi/10.1371/journal.pone.0001596.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001596.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001596.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Graph of IM results for the combined nuclear and mitochondrial coding DNA dataset.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The plot depicts mean N<sub>e</sub> for the Amerind founder population (y-axis) as a product of increasing the constraint on the upper bound of the priors for the migration rates (x-axis). In these analyses, the prior on the lower bound of the divergence time was uniformly set to 15 kya on the basis of known archaeological materials for human occupation in the New World (see text). Each point is based on the average of the estimated medians for ten independent replicate analyses, with the bars corresponding to ± 1 standard deviation. These standard deviations are often small (with coefficients of variation less than 0.01), since their Markov chains were run for 100 million generations each.</p>
<span>doi:10.1371/journal.pone.0001596.g003</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>When studying complex colonization scenarios, the interpretation of genetic data can benefit substantially from the incorporation of non-genetic material evidence. In our study, we do this in three ways. First, we interpret the skyline plot (see <a href="#pone-0001596-g001">Figure 1</a>) to reflect archaeological evidence that places Amerinds in the Americas by ≈15 kya and human populations in Beringia ≈30 kya, as well as geological and paleoecological evidence that Beringia was habitable yet isolated from the Americas from ≈30 kya to 17 kya. Second, we use archaeological radiocarbon dates to constrain the divergence time prior in our IM analyses to 15 kya as the latest possible date for both the divergence of the Amerind and Asian gene pools and the Amerind expansion into North America (<a href="#pone-0001596-g003">Figure 3</a>). Since the IM model assumes that divergence and expansion occur simultaneously, constraining the time of the expansion also requires identical constraint of the divergence date. Third, in our IM analyses we serially constrain the migration rate parameters to smaller values and deduce likely migration rates between Asia and the New World based on empirical estimates of current migration rates within Europe versus the greatly reduced migration rates of ancient people across the Bering Strait starting ≈11–10 kya.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Based on our results, we propose a three-stage colonization process for the peopling of the New World, with a specific focus on the dating and magnitude of the Amerind population expansions (<a href="#pone-0001596-g004">Figure 4</a>). We propose that the first stage was a period of gradual population growth as Amerind ancestors diverged from the central Asian gene pool and moved to the northeast. This was followed by an extended period of population stability in greater Beringia. The final stage was a single, rapid population expansion as Amerinds colonized the New World from Beringia.</p>
<div class="figure" id="pone-0001596-g004"><div class="img"><a name="pone-0001596-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001596" data-uri="info:doi/10.1371/journal.pone.0001596.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001596.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001596.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001596.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001596.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001596.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Maps depicting each phase of our three-step colonization model for the peopling of the Americas.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>(A) Divergence, then gradual population expansion of the Amerind ancestors from their East Central Asian gene pool (blue arrow). (B) Proto-Amerind occupation of Beringia with little to no population growth for ≈20,000 years. (C) Rapid colonization of the New World by a founder group migrating southward through the ice free, inland corridor between the eastern Laurentide and western Cordilleran Ice Sheets (green arrow) and/or along the Pacific coast (red arrow). In (B), the exposed seafloor is shown at its greatest extent during the last glacial maximum at ≈20–18 kya <a href="#pone.0001596-Hopkins1">[25]</a>. In (A) and (C), the exposed seafloor is depicted at ≈40 kya and ≈16 kya, when prehistoric sea levels were comparable <a href="#pone.0001596-Elias1">[24]</a>, <a href="#pone.0001596-Hopkins1">[25]</a>. Because of the earth's curvature, the km scale (which is based on the straight line distance at the equator) provides only an approximation of the same distance between two points on these maps. In addition, a scaled-down version of Beringia today (60% reduction of A–C) is presented in the lower left corner. This smaller map highlights the Bering Strait that has geographically separated the New World from Asia since ≈11–10 kya.</p>
<span>doi:10.1371/journal.pone.0001596.g004</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The initial stage of the colonization process involved the divergence of Amerind ancestors from the East Central Asian gene pool (<a href="#pone-0001596-g004">Figure 4A</a>). Based on previous studies that included Asian mtDNA sequences, this divergence likely occurred prior to ≈50 kya <a href="#pone.0001596-Bonatto1">[5]</a>, <a href="#pone.0001596-Bonatto2">[6]</a>. Our coding skyline plot (<a href="#pone-0001596-g001">Figure 1</a>) indicates that the divergence was followed by a period of gradual growth, during which the proto-Amerind population experienced a 7-fold increase from ≈640 to ≈4,400 females over ≈7,000 years, from ≈43–36 kya. The migrating founder population (N<sub>ef</sub> ≈640) was a small subset of the ancestral Asian population, as evidenced by the low levels of variation in New World populations relative to Asians (e.g. <a href="#pone.0001596-Mulligan1">[3]</a>) as well as the larger effective size of the ancestral Asian population <a href="#pone.0001596-Hey1">[16]</a>. Thus, divergence from the Asian gene pool was the time at which a severe population bottleneck occurred that reduced the genetic variation in Amerind populations. The lack of archaeological sites in Siberia and Beringia that date to ≈43–36 kya <a href="#pone.0001596-Kuzmin1">[34]</a> suggests that this first stage of slow population growth left a light “footprint” on the landscape because of relatively rapid and continuous movement. Consistent with this hypothesis are the younger coalescent dates for modern Siberian populations relative to modern New World populations <a href="#pone.0001596-Torroni2">[15]</a>, <a href="#pone.0001596-Derenko1">[35]</a>, which indicate that the New World migrants passed through Siberia before other East Central Asian population(s) settled permanently in this region at a later date. Such relatively rapid and continuous movement would leave few archaeological sites, which have not yet been discovered due to the vast expanse and harsh conditions of Siberia and the current inundation of Beringia. Thus, an important prediction of the first stage of our model is that older archaeological sites dating to ≈43–36 kya await discovery in these regions.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The proposed second stage (<a href="#pone-0001596-g004">Figure 4B</a>) consisted of an extended period of little change in population size from ≈36–16 kya (<a href="#pone-0001596-g001">Figure 1</a>). It is difficult to assign a precise geographic location to this population, but it may have occupied the large region from Siberia to Alaska, most of which is currently underwater. Our N<sub>ef</sub> estimates of ≈4,000–5,000 (equivalent to N<sub>e</sub> of ≈8,000–10,000, assuming an equal sex ratio) indicate that the proposed human presence would have been minor when compared to the size of greater Beringia. Nevertheless, the presence of this population in Beringia for ≈20,000 years would have afforded sufficient time for the generation of new mutations. Indeed, the existence of New World-specific variants that are distributed throughout the Americas indicate that substantial genetic diversification occurred during the Beringian occupation (e.g. <a href="#pone.0001596-Tamm1">[13]</a>–<a href="#pone.0001596-Torroni2">[15]</a>, <a href="#pone.0001596-Malhi1">[36]</a>, <a href="#pone.0001596-ORourke1">[37]</a>). The proposed period of Beringian occupation coincides with archaeological evidence for the first Arctic inhabitation of western Beringia (≈30 kya) <a href="#pone.0001596-Pitulko1">[23]</a> and pre-dates archaeological evidence for occupation of the New World <a href="#pone.0001596-Waters1">[18]</a>–<a href="#pone.0001596-Overstreet1">[21]</a>. This period also coincides with geological evidence for restricted access to North America because of the impenetrability of the Cordilleran and Laurentide ice sheets (≈17–30 kya) <a href="#pone.0001596-Hoffecker2">[38]</a>, <a href="#pone.0001596-Mandryk1">[39]</a>. Botanical remains, such as macrofossils and ancient pollen, indicate that Beringia was a productive grassland ecosystem rather than an exceedingly harsh Arctic desert environment <a href="#pone.0001596-Zazula1">[27]</a>. Paleontological evidence from Alaska and Siberia demonstrates that large mammals such as steppe bison, mammoth, horse, lion, musk-oxen, sheep, wholly rhinoceros, and caribou inhabited this area <a href="#pone.0001596-Guthrie1">[28]</a>.Thus, the paleoecological data are consistent with a human presence in Beringia although the carrying capacity of Beringia and technological limitations of the human population may have restricted growth until the population could expand into new and fertile lands in the Americas. The rapid expansion of the population only after an ice-free corridor into North America opened (see below) suggests that the population may have departed Beringia as soon as a viable alternative presented.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The final colonization stage (<a href="#pone-0001596-g004">Figure 4C</a>) was a rapid geographic expansion into the New World resulting in a significant population increase (≈16-fold; <a href="#pone-0001596-g001">Figure 1</a>). The rapid population increase occurred over the period ≈16–9 kya according to the coding skyline plot or over the past 15,000 years based on the IM analyses (the latter results supported only the most recent and largest expansion, most likely because IM analyses assume a single, simultaneous divergence/expansion event). The geological record indicates that North America became accessible from Beringia between ≈17–14 kya, when the ice sheets covering what is now Canada began to retreat <a href="#pone.0001596-Hoffecker1">[26]</a>, <a href="#pone.0001596-Mandryk1">[39]</a>. The coincident timing of an ice-free corridor into North America and the rapid expansion of the Amerind population suggests that a land route may have been the preferred entry into the New World. However, the northwest Pacific coast of North America also may have been deglaciated by ≈17 kya, thus presenting a viable coastal route to continental North America <a href="#pone.0001596-Wang1">[4]</a>, <a href="#pone.0001596-Mandryk1">[39]</a>. This period also coincides with the initial inundation of the Bering land bridge, after which migration with Asia would have been severely limited. The first unequivocal evidence for human occupation of the New World occurs in the form of Clovis sites dating to ≈13 kya <a href="#pone.0001596-Waters1">[18]</a> and pre-Clovis sites in both North and South America dating to ≈14–15 kya <a href="#pone.0001596-Dillehay1">[19]</a>–<a href="#pone.0001596-Overstreet1">[21]</a>. Our datasets do not include typings from the Na-Dene or Esk-Aleut, so we limit our scope to the largest, initial migration of Amerinds into the New World. However, Na-Dene and Esk-Aleut genetic diversity represents a subset of Amerind diversity (e.g. <a href="#pone.0001596-Kolman1">[40]</a>–<a href="#pone.0001596-Merriwether1">[42]</a>) suggesting that Na-Dene and Esk-Aleuts are derived from the same Beringian source population as Amerinds. As stated above, extensive archaeological evidence supports the presence of multiple distinct Native American material cultures by ≈13 kya (e.g. Clovis, Nenana and pre-Clovis lithic technologies <a href="#pone.0001596-Waters1">[18]</a>). Our results suggest that these distinct cultures derive from a single New World founder population and are most likely the product of an extensive and complex process of post-peopling migrations within the Americas, possibly through a combination of coastal and/or riverine routes <a href="#pone.0001596-Wang1">[4]</a>, <a href="#pone.0001596-Fix1">[43]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Determination of the size of the Amerind founding population has received considerable attention. Based on the coding Bayesian skyline plot (<a href="#pone-0001596-g001">Figure 1</a>), there is a slight decrease in population size preceding the increase seen at ≈15 kya. This decrease is consistent with a secondary founder effect in which a subset of the Beringian population seeded the proto-Amerind expansion into the Americas. Assuming the apparent decrease in N<sub>ef</sub> is the result of such a founder effect, the upper bound on the founder population size is ≈5,400 individuals (N<sub>ef</sub> ≈2,700). Our IM analyses suggest that the founder population size could be lower depending on prior assumptions about the over water migration rates between the Americas and Asia (see <a href="#pone-0001596-g003">Figure 3</a>). Migration rates (<em>m</em>) within Europe today based on census data have been determined to be 4.3, which can be taken as an extreme upper bound of possible ancient migration rates between the Americas and Asia, especially after the appearance of the Bering Strait ≈11–10 kya. Restricting migration rates to &lt;1 results in founder N<sub>e</sub> estimates between ≈1,000 and ≈1,200, with ≈1,200 serving as an asymptotic upper bound (see <a href="#pone-0001596-g003">Figure 3</a>). Taken together, our Bayesian skyline plot and IM analyses suggest that a founder population with N<sub>e</sub> = 1,000–5,400 colonized the New World in a process characterized by a rapid geographic and population expansion. The range of N<sub>e</sub> values can be translated into an approximate census population size by applying a scale factor estimated from large mammal populations (scale factor = 5) <a href="#pone.0001596-Templeton1">[44]</a>, which suggests that the founder population consisted of ≈5,000–27,000 people.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Our three-stage model now awaits further critical testing with new datasets of independent nuclear loci and more sophisticated methods of coalescent analysis. The extensive dataset of ≈700 autosomal microsatellites, compiled by Wang et al. <a href="#pone.0001596-Wang1">[4]</a> for both Native American and worldwide populations, offers the opportunity to evaluate critically the size, timing, and duration of each step in our model at essentially a population genomics level. Future versions of BEAST will incorporate a structured coalescent where migration as well as population growth will be allowed to occur among populations from both the New World and Asia (<a href="http://evolve.zoo.ox.ac.uk/beast/manual.html">http://evolve.zoo.ox.ac.uk/beast/manual.​html</a>). In these BEAST analyses, the microsatellites can be modeled under a stepwise “ladder process,” whereby alleles are inter-related according to their repeat lengths. One can then summarize over these microsatellite loci by assuming independence, which thereby allows for the multiplication of their separate posterior distributions and final estimations of their combined Bayesian skyline plot. In these ways, we fully anticipate that such critical testing will lead to many important refinements of our three-step model, including a further narrowing of our proposed range for the size of the founding population as well as new details about post-peopling expansions within the New World.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Datasets</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Three datasets were collected for analysis including: (<em>i</em>) 77 mtDNA coding genomes; (<em>ii</em>) 812 mtDNA HVRI+II sequences; and (<em>iii</em>) combined nuclear and mitochondrial coding DNA dataset. The 77 mtDNA coding genomes were collected from publicly available resources <a href="#pone.0001596-Herrnstadt1">[45]</a>–<a href="#pone.0001596-Mishmar1">[48]</a> and aligned using ClustalX <a href="#pone.0001596-Thompson1">[49]</a>. The resultant 15,500 base pair (bp) multiple alignment was edited by hand to minimize the number of unique gaps and to ensure the integrity of the reading frame (available online as <a href="#pone.0001596.s001">Figure S1</a>). A total of 812 combined HVRI+II sequences were collected from HVRbase (<a href="http://www.hvrbase.org">http://www.hvrbase.org</a>) <a href="#pone.0001596-Handt1">[50]</a>. These sequences were aligned following the coding mtDNAs, resulting in a multiple alignment of 771 bps (available online as Supplemental <a href="#pone.0001596.s002">Figure S2</a>). The complete dataset of 812 HVRI+II sequences was randomly divided into ten non-overlapping alignments of 81 sequences that approximate the sample size for the coding mtDNA dataset. Skyline plot analyses of larger datasets (up to 200 HVRI+II sequences) gave the same results as the 81 sequence datasets (data not shown). Thus, the smaller datasets of 81 sequences each were emphasized here since they avoided the likelihood rounding errors that can occur when using large, heterogeneous datasets in Bayesian skyline plot analyses. The coding nuclear and mtDNA dataset from Asian and Native American populations of Hey (available at <a href="http://lifesci.rutgers.edu/heylab/">http://lifesci.rutgers.edu/heylab/</a>) <a href="#pone.0001596-Hey1">[16]</a> consisted of two autosomal coding loci, five X-chromosome coding loci, one Y-chromosome coding locus, and the complete mtDNA coding genome (totaling 28,454 aligned bps). The sample sizes for these nuclear loci and mitochondrial genome varied from 12-108 sequences.</p>


<h4>Bayesian Skyline Plot Analyses</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Bayesian skyline plots <a href="#pone.0001596-Drummond1">[32]</a> were used to estimate changes in Amerind N<sub>ef</sub> over time by providing highly parametric, piecewise estimates of N<sub>ef</sub>. This approach produces serial estimates of effective population size from the time intervals between coalescent events in a genealogy of sampled individuals, and utilizes a Markov chain Monte Carlo simulation approach to integrate over all credible genealogies and other model parameters. It thereby differs from previous approaches (e.g., <a href="#pone.0001596-Polanski1">[51]</a>) in that Bayesian skyline plots fully parameterize both the mutation model (including relaxed clock models) and the genealogical process, whereas prior methods relied on generating estimates from summary statistics (e.g. the use of pairwise differences by <a href="#pone.0001596-Polanski1">[51]</a>).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>In these analyses, estimates of τ (N<sub>ef</sub>×generation time) were converted to N<sub>ef</sub> by dividing by a generation time of 20 years, following convention <a href="#pone.0001596-Hey1">[16]</a>. Skyline plots were generated for the 77 mtDNA coding genome sequences and the ten datasets of HVRI+II sequences using the program BEAST v1.4 (<a href="http://beast.bio.ed.ac.uk">http://beast.bio.ed.ac.uk</a>). These BEAST analyses relied on the same coalescent and substitution models and run conditions of Kitchen <em>et al.</em> <a href="#pone.0001596-Kitchen1">[52]</a> except as noted below. Plots were generated using the established mutation rates (μ) for coding mtDNA (μ = 1.7×10<sup>−8</sup> substitutions/site/year) <a href="#pone.0001596-Ingman1">[46]</a> and HVRI+II mtDNA (μ = 4.7×10<sup>−7</sup>) <a href="#pone.0001596-Howell1">[53]</a>. Markov chains were run for 100,000,000 generations and sampled every 2,500 generations with the first 10,000,000 generations discarded as burn-in. Three independent runs were performed for all coding and HVRI+II Bayesian skyline plot analyses. Markov chain samples from the three independent mtDNA coding replicates and from the 30 HVRI+II analyses were separately combined using the LogCompiler program (distributed with BEAST) and analyzed using Tracer v1.3 to produce the final Bayesian skyline plots.</p>


<h4>Isolation-with-Migration Coalescent Analyses</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Bayesian IM coalescent analyses were performed using the program IM <a href="#pone.0001596-Hey1">[16]</a> to estimate N<sub>e</sub> for the Amerind founder population (males+females) and the divergence time for Amerind and Asian populations. We used the same combined nuclear and mtDNA dataset, same coalescent and substitution models, and same model parameters as Hey <a href="#pone.0001596-Hey1">[16]</a> with the exception of new priors on the divergence time and on the migration rates between Asian and Amerind populations. All IM analyses were performed using a flat uniform prior for the divergence time of Amerind and Asian populations set to the interval 15–40 kya. The lower bound of this prior is based on accepted archaeological and climatological evidence for the first presence of humans in the Americas <a href="#pone.0001596-Waters1">[18]</a>–<a href="#pone.0001596-Overstreet1">[21]</a>. The upper bound of the flat uniform priors on the migration rates per mutation per generation between the Amerindian and Asian populations (<em>m</em><sub>Asia→NW</sub> and <em>m</em><sub>NW→Asia</sub>) was set to 12 different values (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, and 50). To help interpret these results, we relied on an estimate of the migration rate in modern Europe as obtained from census data <a href="#pone.0001596-Weale1">[54]</a>. Specifically, we converted their migration rate estimate of 0.0004 migrations per gene copy per generation (recalculated assuming a generation time of 20 years based on Hey <a href="#pone.0001596-Hey1">[16]</a>) to our units of migrations per mutation per generation (<em>m</em>) by dividing the former by the geometric mean of the mutation rates for the nine loci in this dataset (9.32×10<sup>−5</sup> mutations per locus per generation ). These calculations resulted in <em>m</em> = 4.3 for modern Europe. In contrast, the ancient migration rates between the New World and Asia would have been significantly less, especially after their geographic separation due to the re-inundation of Beringia starting at ≈11 kya (see <a href="#s1">Introduction</a>). Ten independent replicates were performed for each of the 12 upper bound values on the migration rates, for a total of 120 IM analyses. All Markov chains were run for 100,000,000 generations without heating.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001596"></div><a name="pone.0001596.s001" id="pone.0001596.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001596.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Multiple sequence alignment for the 77 Amerind mtDNA coding genomes used in this study. Here, “coding” refers to both protein and structural RNA genes following Pakendorf and Stoneking <a href="#pone.0001596-Pakendorf1">[30]</a>. Gaps are represented by “-.” Position 1 of this alignment corresponds to site 546 of the Anderson Reference Sequence (ARS; <a href="#pone.0001596-Anderson1">[56]</a>). The final position of this alignment (15,500) corresponds to site 16,042 of the ARS. Sequences starting with “Herrn,” “Ing,” “Kiv”, and “Mis” follow the naming conventions of Herrnstadt et al. <a href="#pone.0001596-Herrnstadt1">[45]</a>, Ingman et al. <a href="#pone.0001596-Ingman1">[46]</a>, Kivisild et al. <a href="#pone.0001596-Kivisild1">[47]</a>, and Mishmar et al. <a href="#pone.0001596-Mishmar1">[48]</a>, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001596.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.19 MB TXT)</p>
<a name="pone.0001596.s002" id="pone.0001596.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001596.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Multiple sequence alignments for the ten, randomly selected, non-overlapping sets of 81 HVRI+II sequences used in this study. In these alignments, positions 1-403 correspond to HVRI, whereas sites 404-781 refer to HVRII. In turn, these alignment positions correspond to sites 16003-16400 and 30-399 of the ARS, respectively. Gaps are represented by “-.” The HVRI+II sequences follow the naming conventions of HRVBase <a href="#pone.0001596-Handt1">[50]</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001596.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.64 MB TXT)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: CM MM AK. Performed the experiments: MM AK. Wrote the paper: CM MM AK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001596-Greenberg1" id="pone.0001596-Greenberg1"></a>Greenberg JH, Turner CG, Zegura SL (1986) The settlement of the Americas: a comparison of the linguistic, dental and genetic evidence. Curr Anthropol  27: 477–497.  <ul class="find" data-citedArticleID="1055548" data-doi="10.1086/203472"><li><a href="http://dx.doi.org/10.1086/203472" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+settlement+of+the+Americas%3A+a+comparison+of+the+linguistic%2C+dental+and+genetic+evidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+settlement+of+the+Americas%3A+a+comparison+of+the+linguistic%2C+dental+and+genetic+evidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001596-Schurr1" id="pone.0001596-Schurr1"></a>Schurr TG (2004) The peopling of the New World: perspectives from molecular anthropology. Annu Rev Anthropol  33: 551–583.  <ul class="find" data-citedArticleID="1055608" data-doi="10.1146/annurev.anthro.33.070203.143932"><li><a href="http://dx.doi.org/10.1146/annurev.anthro.33.070203.143932" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+peopling+of+the+New+World%3A+perspectives+from+molecular+anthropology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+peopling+of+the+New+World%3A+perspectives+from+molecular+anthropology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001596-Mulligan1" id="pone.0001596-Mulligan1"></a>Mulligan CJ, Hunley K, Cole S, Long JC (2004) Population genetics, history, and health patterns in native Americans. Annu Rev Genom Hum Genet  5: 295–315.  <ul class="find" data-citedArticleID="1055592" data-doi="10.1146/annurev.genom.5.061903.175920"><li><a href="http://dx.doi.org/10.1146/annurev.genom.5.061903.175920" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+genetics%2C+history%2C+and+health+patterns+in+native+Americans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+genetics%2C+history%2C+and+health+patterns+in+native+Americans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001596-Wang1" id="pone.0001596-Wang1"></a>Wang S, Lewis CM, Jakobsson M, Ramachandran S, Ray N, et al.  (2007) Genetic variation and population structure in Native Americans. PLoS Genet  3: e185.  <ul class="find" data-citedArticleID="1055628" data-doi="10.1371/journal.pgen.0030185"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variation+and+population+structure+in+Native+Americans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variation+and+population+structure+in+Native+Americans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001596-Bonatto1" id="pone.0001596-Bonatto1"></a>Bonatto SL, Salzano FM (1997) A single and early migration for the peopling of the Americas supported by mitochondrial DNA sequence data. Proc Natl Acad Sci USA  94: 1866–1871.  <ul class="find" data-citedArticleID="1055528" data-doi="10.1073/pnas.94.5.1866"><li><a href="http://dx.doi.org/10.1073/pnas.94.5.1866" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+and+early+migration+for+the+peopling+of+the+Americas+supported+by+mitochondrial+DNA+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+and+early+migration+for+the+peopling+of+the+Americas+supported+by+mitochondrial+DNA+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001596-Bonatto2" id="pone.0001596-Bonatto2"></a>Bonatto SL, Salzano FM (1997) Diversity and age of the four major mtDNA haplogroups, and their implications for the peopling of the New World. Am J Hum Genet  61: 1413–1423.  <ul class="find" data-citedArticleID="1055530" data-doi="10.1086/301629"><li><a href="http://dx.doi.org/10.1086/301629" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+and+age+of+the+four+major+mtDNA+haplogroups%2C+and+their+implications+for+the+peopling+of+the+New+World." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+and+age+of+the+four+major+mtDNA+haplogroups%2C+and+their+implications+for+the+peopling+of+the+New+World.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001596-Forster1" id="pone.0001596-Forster1"></a>Forster P, Harding R, Torroni A, Bandelt H-J (1996) Origin and evolution of Native American mtDNA variation: a reappraisal. Am J Hum Genet  59: 935–945.  <ul class="find" data-citedArticleID="1055544"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Origin+and+evolution+of+Native+American+mtDNA+variation%3A+a+reappraisal.&amp;auth=&amp;atitle=Origin+and+evolution+of+Native+American+mtDNA+variation%3A+a+reappraisal." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+and+evolution+of+Native+American+mtDNA+variation%3A+a+reappraisal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+and+evolution+of+Native+American+mtDNA+variation%3A+a+reappraisal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001596-Santos1" id="pone.0001596-Santos1"></a>Santos FR, Pandya A, Tyler-Smith C, Pena SDJ, Schanfield M, et al.  (1999) The central Siberian origin for Native American Y chromosomes. Am J Hum Genet  64: 619–628.  <ul class="find" data-citedArticleID="1055606" data-doi="10.1086/302242"><li><a href="http://dx.doi.org/10.1086/302242" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+central+Siberian+origin+for+Native+American+Y+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+central+Siberian+origin+for+Native+American+Y+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001596-Schurr2" id="pone.0001596-Schurr2"></a>Schurr TG, Sherry SS (2004) Mitochondrial DNA and Y chromosome diversity and the peopling of the Americas: evolutionary and demographic evidence. Am J Hum Biol  16: 420–439.  <ul class="find" data-citedArticleID="1055610" data-doi="10.1002/ajhb.20041"><li><a href="http://dx.doi.org/10.1002/ajhb.20041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+and+Y+chromosome+diversity+and+the+peopling+of+the+Americas%3A+evolutionary+and+demographic+evidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+and+Y+chromosome+diversity+and+the+peopling+of+the+Americas%3A+evolutionary+and+demographic+evidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001596-Shields1" id="pone.0001596-Shields1"></a>Shields GF, Schmiechen AM, Frazier BL, Redd A, Voevoda MI, et al.  (1993) mtDNA sequences suggest a recent evolutionary divergence for Beringian and northern North American populations. Am J Hum Genet  53: 549–562.  <ul class="find" data-citedArticleID="1055612"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=mtDNA+sequences+suggest+a+recent+evolutionary+divergence+for+Beringian+and+northern+North+American+populations.&amp;auth=&amp;atitle=mtDNA+sequences+suggest+a+recent+evolutionary+divergence+for+Beringian+and+northern+North+American+populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=mtDNA+sequences+suggest+a+recent+evolutionary+divergence+for+Beringian+and+northern+North+American+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22mtDNA+sequences+suggest+a+recent+evolutionary+divergence+for+Beringian+and+northern+North+American+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001596-Silva1" id="pone.0001596-Silva1"></a>Silva WA, Bonatto SL, Holanda AJ, Ribeiro-dos-Santos AK, Paixao BM, et al.  (2002) Mitochondrial genome diversity of Native Americans supports a single early entry of founder populations into America. Am J Hum Genet  71: 187–192.  <ul class="find" data-citedArticleID="1055614" data-doi="10.1086/341358"><li><a href="http://dx.doi.org/10.1086/341358" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+genome+diversity+of+Native+Americans+supports+a+single+early+entry+of+founder+populations+into+America." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+genome+diversity+of+Native+Americans+supports+a+single+early+entry+of+founder+populations+into+America.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001596-Szathmary1" id="pone.0001596-Szathmary1"></a>Szathmary EJE (1993) Genetics of aboriginal North Americans. Evol Anthropol  1: 202–220.  <ul class="find" data-citedArticleID="1055616" data-doi="10.1002/evan.1360010606"><li><a href="http://dx.doi.org/10.1002/evan.1360010606" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+aboriginal+North+Americans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+aboriginal+North+Americans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001596-Tamm1" id="pone.0001596-Tamm1"></a>Tamm E, Kivisild T, Reidla1 M, Metspalu1 M, Smith DG, et al.  (2007) Beringian standstill and spread of Native American founders. PLoS ONE  2: e829.  <ul class="find" data-citedArticleID="1055618" data-doi="10.1371/journal.pone.0000829"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000829" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Beringian+standstill+and+spread+of+Native+American+founders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Beringian+standstill+and+spread+of+Native+American+founders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001596-Torroni1" id="pone.0001596-Torroni1"></a>Torroni A, Schurr TG, Cabell MF, Brown MD, Neel JV, et al.  (1993) Asian affinities and continental radiation of the four founding Native American mtDNAs. Am J Hum Genet  53: 563–590.  <ul class="find" data-citedArticleID="1055624"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Asian+affinities+and+continental+radiation+of+the+four+founding+Native+American+mtDNAs.&amp;auth=&amp;atitle=Asian+affinities+and+continental+radiation+of+the+four+founding+Native+American+mtDNAs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asian+affinities+and+continental+radiation+of+the+four+founding+Native+American+mtDNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Asian+affinities+and+continental+radiation+of+the+four+founding+Native+American+mtDNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001596-Torroni2" id="pone.0001596-Torroni2"></a>Torroni A, Sukernik RI, Schurr TG, Starikovskaya YB, Cabell MF, et al.  (1993) mtDNA variation of aboriginal Siberians reveals distinct genetic affinities with Native Americans. Am J Hum Genet  53: 591–608.  <ul class="find" data-citedArticleID="1055626"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=mtDNA+variation+of+aboriginal+Siberians+reveals+distinct+genetic+affinities+with+Native+Americans.&amp;auth=&amp;atitle=mtDNA+variation+of+aboriginal+Siberians+reveals+distinct+genetic+affinities+with+Native+Americans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=mtDNA+variation+of+aboriginal+Siberians+reveals+distinct+genetic+affinities+with+Native+Americans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22mtDNA+variation+of+aboriginal+Siberians+reveals+distinct+genetic+affinities+with+Native+Americans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001596-Hey1" id="pone.0001596-Hey1"></a>Hey J (2005) On the number of New World founders: a population genetic portrait of the peopling of the Americas. PLoS Biol  3: e193.  <ul class="find" data-citedArticleID="1055558" data-doi="10.1371/journal.pbio.0030193"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030193" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+number+of+New+World+founders%3A+a+population+genetic+portrait+of+the+peopling+of+the+Americas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+number+of+New+World+founders%3A+a+population+genetic+portrait+of+the+peopling+of+the+Americas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001596-Hamilton1" id="pone.0001596-Hamilton1"></a>Hamilton MJ, Buchanan B (2007) Spatial gradients in Clovis-age radiocarbon dates across North America suggest rapid colonization from the north. Proc Natl Acad Sci USA  104: 15625–15630.  <ul class="find" data-citedArticleID="1055552" data-doi="10.1073/pnas.0704215104"><li><a href="http://dx.doi.org/10.1073/pnas.0704215104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spatial+gradients+in+Clovis-age+radiocarbon+dates+across+North+America+suggest+rapid+colonization+from+the+north." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spatial+gradients+in+Clovis-age+radiocarbon+dates+across+North+America+suggest+rapid+colonization+from+the+north.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001596-Waters1" id="pone.0001596-Waters1"></a>Waters MR, Stafford TW (2007) Redefining the age of Clovis: implications for the peopling of the Americas. Science  315: 1122–1126.  <ul class="find" data-citedArticleID="1055630" data-doi="10.1126/science.1137166"><li><a href="http://dx.doi.org/10.1126/science.1137166" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Redefining+the+age+of+Clovis%3A+implications+for+the+peopling+of+the+Americas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Redefining+the+age+of+Clovis%3A+implications+for+the+peopling+of+the+Americas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001596-Dillehay1" id="pone.0001596-Dillehay1"></a> Dillehay TD, editor.  (1997) The archaeological context and interpretation. Washington, DC: Smithsonian Institution Press.   <ul class="find-nolinks"></ul></li><li><span class="label">20.
              </span><a name="pone.0001596-Joyce1" id="pone.0001596-Joyce1"></a>Joyce DJ (2006) Chronology and new research on the Schaefer mammoth (<em>Mammuthus primigenius</em>) site, Kenosha County, Wisconsin, USA. Quat Intl  142: 44–57.  <ul class="find" data-citedArticleID="1055572" data-doi="10.1016/j.quaint.2005.03.004"><li><a href="http://dx.doi.org/10.1016/j.quaint.2005.03.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chronology+and+new+research+on+the+Schaefer+mammoth+%28Mammuthus+primigenius%29+site%2C+Kenosha+County%2C+Wisconsin%2C+USA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chronology+and+new+research+on+the+Schaefer+mammoth+%28Mammuthus+primigenius%29+site%2C+Kenosha+County%2C+Wisconsin%2C+USA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001596-Overstreet1" id="pone.0001596-Overstreet1"></a>Overstreet DF (2005)  Late-glacial ice-marginal adaptation in southeastern Wisconsin. In: Bonnichsen R, Lepper BT, Stanford D, Waters MR, editors. Paleoamerican origins: beyond Clovis. College Station, TX: Center for the Study of the First Americans. pp. 183–195.  <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0001596-Goebel1" id="pone.0001596-Goebel1"></a>Goebel T (2007) The missing years for modern humans. Science  315: 194–196.  <ul class="find" data-citedArticleID="1055546" data-doi="10.1126/science.1137564"><li><a href="http://dx.doi.org/10.1126/science.1137564" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+missing+years+for+modern+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+missing+years+for+modern+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001596-Pitulko1" id="pone.0001596-Pitulko1"></a>Pitulko VV, Nikolsky PA, Girya EY, Basilyan AE, Tumskoy VE, et al.  (2004) The Yana RHS site: humans in the Arctic before the last glacial maximum. Science  303: 52–56.  <ul class="find" data-citedArticleID="1055602" data-doi="10.1126/science.1085219"><li><a href="http://dx.doi.org/10.1126/science.1085219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Yana+RHS+site%3A+humans+in+the+Arctic+before+the+last+glacial+maximum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Yana+RHS+site%3A+humans+in+the+Arctic+before+the+last+glacial+maximum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001596-Elias1" id="pone.0001596-Elias1"></a>Elias SA, Short SK, Nelson CH, Birks HH (1996) Life and times of the Bering land bridge. Nature  382: 60–63.  <ul class="find" data-citedArticleID="1055538" data-doi="10.1038/382060a0"><li><a href="http://dx.doi.org/10.1038/382060a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Life+and+times+of+the+Bering+land+bridge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Life+and+times+of+the+Bering+land+bridge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001596-Hopkins1" id="pone.0001596-Hopkins1"></a>Hopkins DM (1982)  Aspects of the paleogeography of Beringia during the Late Pleistocene. In: Hopkins DM, Matthews JV, Schweger CE, Young SB, editors. Paleoecology of Beringia. New York: Academic Press. pp. 3–28.  <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0001596-Hoffecker1" id="pone.0001596-Hoffecker1"></a>Hoffecker JF, Powers WR, Goebel T (1993) The colonization of Beringia and the peopling of the New World. Science  259: 46–53.  <ul class="find" data-citedArticleID="1055560" data-doi="10.1126/science.259.5091.46"><li><a href="http://dx.doi.org/10.1126/science.259.5091.46" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+colonization+of+Beringia+and+the+peopling+of+the+New+World." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+colonization+of+Beringia+and+the+peopling+of+the+New+World.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001596-Zazula1" id="pone.0001596-Zazula1"></a>Zazula GD, Froese DG, Schweger CE, Mathewes RW, Beaudoin AB, et al.  (2003) Ice-age steppe vegetation in east Beringia. Nature  426: 603.  <ul class="find" data-citedArticleID="1055636" data-doi="10.1038/423603a"><li><a href="http://dx.doi.org/10.1038/423603a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ice-age+steppe+vegetation+in+east+Beringia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ice-age+steppe+vegetation+in+east+Beringia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001596-Guthrie1" id="pone.0001596-Guthrie1"></a>Guthrie RD (1990) Frozen fauna of the mammoth steppe. Chicago: University of Chicago Press.   <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0001596-Jobling1" id="pone.0001596-Jobling1"></a>Jobling MA, Hurles ME, Tyler-Smith C (2004) Into new found lands. Human Evolutionary Genetics. New York: Garland Science. pp. 339–372.  <ul class="find-nolinks"></ul></li><li><span class="label">30.
              </span><a name="pone.0001596-Pakendorf1" id="pone.0001596-Pakendorf1"></a>Pakendorf B, Stoneking M (2005) Mitochondrial DNA and human evolution. Annu Rev Genom Hum Genet  6: 165–183.  <ul class="find" data-citedArticleID="1055600" data-doi="10.1146/annurev.genom.6.080604.162249"><li><a href="http://dx.doi.org/10.1146/annurev.genom.6.080604.162249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+and+human+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+and+human+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001596-Wilson1" id="pone.0001596-Wilson1"></a>Wilson A, Cann R, Carr S, George M, Gyllensten U, et al.  (1985) Mitochondrial DNA and two perspectives on evolutionary genetics. Biol J Linn Soc  26: 375–400.  <ul class="find" data-citedArticleID="1055634" data-doi="10.1111/j.1095-8312.1985.tb02048.x"><li><a href="http://dx.doi.org/10.1111/j.1095-8312.1985.tb02048.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+and+two+perspectives+on+evolutionary+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+and+two+perspectives+on+evolutionary+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001596-Drummond1" id="pone.0001596-Drummond1"></a>Drummond AJ, Rambaut A, Shapiro B, Pybus OG (2005) Bayesian coalescent inference of past population dynamics from molecular sequences. Mol Biol Evol  22: 1185–1192.  <ul class="find" data-citedArticleID="1055536" data-doi="10.1093/molbev/msi103"><li><a href="http://dx.doi.org/10.1093/molbev/msi103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+coalescent+inference+of+past+population+dynamics+from+molecular+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+coalescent+inference+of+past+population+dynamics+from+molecular+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001596-Non1" id="pone.0001596-Non1"></a>Non AL, Kitchen A, Mulligan CJ (2007) Identification of the most informative regions of the mitochondrial genome for phylogenetic and coalescent analyses. Mol Phylogenet Evol  44: 1164–1171.  <ul class="find" data-citedArticleID="1055594" data-doi="10.1016/j.ympev.2006.12.020"><li><a href="http://dx.doi.org/10.1016/j.ympev.2006.12.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+the+most+informative+regions+of+the+mitochondrial+genome+for+phylogenetic+and+coalescent+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+the+most+informative+regions+of+the+mitochondrial+genome+for+phylogenetic+and+coalescent+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001596-Kuzmin1" id="pone.0001596-Kuzmin1"></a>Kuzmin YV, Keates SG (2005) Dates are not just data: paleolithic settlement patterns in Siberia derived from radiocarbon records. Am Antiquity  70: 773–789.  <ul class="find" data-citedArticleID="1055582" data-doi="10.2307/40035874"><li><a href="http://dx.doi.org/10.2307/40035874" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dates+are+not+just+data%3A+paleolithic+settlement+patterns+in+Siberia+derived+from+radiocarbon+records." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dates+are+not+just+data%3A+paleolithic+settlement+patterns+in+Siberia+derived+from+radiocarbon+records.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001596-Derenko1" id="pone.0001596-Derenko1"></a>Derenko M, Malyarchuk B, Grzybowski T, Denisova G, Dambueva I, et al.  (2007) Phylogeographic analysis of mitochondrial DNA in northern Asian Populations. Am J Hum Genet  81: 1025–1041.  <ul class="find" data-citedArticleID="1055532" data-doi="10.1086/522933"><li><a href="http://dx.doi.org/10.1086/522933" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeographic+analysis+of+mitochondrial+DNA+in+northern+Asian+Populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeographic+analysis+of+mitochondrial+DNA+in+northern+Asian+Populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001596-Malhi1" id="pone.0001596-Malhi1"></a>Malhi RS, Eshleman JA, Greenberg JA, Weiss DA, Shook BAS, et al.  (2002) The structure of diversity within new world mitochondrial DNA haplogroups: Implications for the prehistory of North America. Am J Hum Genet  70: 905–919.  <ul class="find" data-citedArticleID="1055584" data-doi="10.1086/339690"><li><a href="http://dx.doi.org/10.1086/339690" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+diversity+within+new+world+mitochondrial+DNA+haplogroups%3A+Implications+for+the+prehistory+of+North+America." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+diversity+within+new+world+mitochondrial+DNA+haplogroups%3A+Implications+for+the+prehistory+of+North+America.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001596-ORourke1" id="pone.0001596-ORourke1"></a>O'Rourke DH, Hayes MG, Carlyle SW (2000) Spatial and temporal stability of mtDNA haplogroup frequencies in native North America. Hum Biol  72: 15–34.  <ul class="find" data-citedArticleID="1055596"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Spatial+and+temporal+stability+of+mtDNA+haplogroup+frequencies+in+native+North+America.&amp;auth=&amp;atitle=Spatial+and+temporal+stability+of+mtDNA+haplogroup+frequencies+in+native+North+America." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spatial+and+temporal+stability+of+mtDNA+haplogroup+frequencies+in+native+North+America." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spatial+and+temporal+stability+of+mtDNA+haplogroup+frequencies+in+native+North+America.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001596-Hoffecker2" id="pone.0001596-Hoffecker2"></a>Hoffecker JF, Elias SA (2003) Environment and archaeology in Beringia. Evol Anthropol  12: 34–49.  <ul class="find" data-citedArticleID="1055562" data-doi="10.1002/evan.10103"><li><a href="http://dx.doi.org/10.1002/evan.10103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environment+and+archaeology+in+Beringia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environment+and+archaeology+in+Beringia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001596-Mandryk1" id="pone.0001596-Mandryk1"></a>Mandryk CAS, Josenhans H, Fedje DW, Mathewes RW (2001) Late Quaternary paleoenvironments of Northwestern North America: implications for inland versus coastal migration routes. Quat Sci Rev  20: 301–314.  <ul class="find" data-citedArticleID="1055586" data-doi="10.1016/s0277-3791(00)00115-3"><li><a href="http://dx.doi.org/10.1016/s0277-3791(00)00115-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Late+Quaternary+paleoenvironments+of+Northwestern+North+America%3A+implications+for+inland+versus+coastal+migration+routes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Late+Quaternary+paleoenvironments+of+Northwestern+North+America%3A+implications+for+inland+versus+coastal+migration+routes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001596-Kolman1" id="pone.0001596-Kolman1"></a>Kolman CJ, Bermingham E, Cooke R, Ward RH, Arias TD, et al.  (1995) Reduced mtDNA diversity in the Ngobe Amerinds of Panama. Genetics  140: 273–283.  <ul class="find" data-citedArticleID="1055578"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Reduced+mtDNA+diversity+in+the+Ngobe+Amerinds+of+Panama.&amp;auth=&amp;atitle=Reduced+mtDNA+diversity+in+the+Ngobe+Amerinds+of+Panama." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reduced+mtDNA+diversity+in+the+Ngobe+Amerinds+of+Panama." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reduced+mtDNA+diversity+in+the+Ngobe+Amerinds+of+Panama.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001596-Kolman2" id="pone.0001596-Kolman2"></a>Kolman CJ, Sambuughin N, Bermingham E (1996) Mitochondrial DNA analysis of Mongolian populations and implications for the origin of New World founders. Genetics  142: 1321–1334.  <ul class="find" data-citedArticleID="1055580"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mitochondrial+DNA+analysis+of+Mongolian+populations+and+implications+for+the+origin+of+New+World+founders.&amp;auth=&amp;atitle=Mitochondrial+DNA+analysis+of+Mongolian+populations+and+implications+for+the+origin+of+New+World+founders." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+analysis+of+Mongolian+populations+and+implications+for+the+origin+of+New+World+founders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+analysis+of+Mongolian+populations+and+implications+for+the+origin+of+New+World+founders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001596-Merriwether1" id="pone.0001596-Merriwether1"></a>Merriwether DA, Rothhamer F, Ferrell RE (1995) Distribution of the 4 founding lineage haplotypes in Native Americans suggests a single wave of migration for the New World. Am J Phys Anthropol  98: 411–430.  <ul class="find" data-citedArticleID="1055588" data-doi="10.1002/ajpa.1330980404"><li><a href="http://dx.doi.org/10.1002/ajpa.1330980404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+the+4+founding+lineage+haplotypes+in+Native+Americans+suggests+a+single+wave+of+migration+for+the+New+World." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+the+4+founding+lineage+haplotypes+in+Native+Americans+suggests+a+single+wave+of+migration+for+the+New+World.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001596-Fix1" id="pone.0001596-Fix1"></a>Fix AG (2005) Rapid deployment of the five founding Amerind mtDNA haplogroups via coastal and riverine colonization. Am J Phys Anthropol  128: 430–436.  <ul class="find" data-citedArticleID="1055542" data-doi="10.1002/ajpa.20230"><li><a href="http://dx.doi.org/10.1002/ajpa.20230" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+deployment+of+the+five+founding+Amerind+mtDNA+haplogroups+via+coastal+and+riverine+colonization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+deployment+of+the+five+founding+Amerind+mtDNA+haplogroups+via+coastal+and+riverine+colonization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001596-Templeton1" id="pone.0001596-Templeton1"></a>Templeton AR (1998) Human races: a genetic and evolutionary perspective. Am Anthropol  100: 632–650.  <ul class="find" data-citedArticleID="1055620" data-doi="10.1525/aa.1998.100.3.632"><li><a href="http://dx.doi.org/10.1525/aa.1998.100.3.632" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+races%3A+a+genetic+and+evolutionary+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+races%3A+a+genetic+and+evolutionary+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001596-Herrnstadt1" id="pone.0001596-Herrnstadt1"></a>Herrnstadt C, Elson JL, Fahy E, Preston G, Turnbull DM, et al.  (2002) Reduced-median-network analysis of complete mitochondrial DNA coding-region sequences for the major African, Asian, and European haplogroups. Am J Hum Genet  70: 1152–1171.  <ul class="find" data-citedArticleID="1055556" data-doi="10.1086/339933"><li><a href="http://dx.doi.org/10.1086/339933" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reduced-median-network+analysis+of+complete+mitochondrial+DNA+coding-region+sequences+for+the+major+African%2C+Asian%2C+and+European+haplogroups." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reduced-median-network+analysis+of+complete+mitochondrial+DNA+coding-region+sequences+for+the+major+African%2C+Asian%2C+and+European+haplogroups.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001596-Ingman1" id="pone.0001596-Ingman1"></a>Ingman M, Kaessmann H, Pääbo S, Gyllensten U (2000) Mitochondrial genome variation and the origin of modern humans. Nature  408: 708–713.  <ul class="find" data-citedArticleID="1055568" data-doi="10.1038/35047064"><li><a href="http://dx.doi.org/10.1038/35047064" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+genome+variation+and+the+origin+of+modern+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+genome+variation+and+the+origin+of+modern+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001596-Kivisild1" id="pone.0001596-Kivisild1"></a>Kivisild T, Shen P, Wall DP, Do B, Sung R, et al.  (2006) The role of selection in the evolution of human mitochondrial genomes. Genetics  172: 373–387.  <ul class="find" data-citedArticleID="1055576" data-doi="10.1534/genetics.105.043901"><li><a href="http://dx.doi.org/10.1534/genetics.105.043901" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+selection+in+the+evolution+of+human+mitochondrial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+selection+in+the+evolution+of+human+mitochondrial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001596-Mishmar1" id="pone.0001596-Mishmar1"></a>Mishmar D, Ruiz-Pesini E, Golik P, Macaulay V, Clark A, et al.  (2003) Natural selection shaped regional mtDNA variation in humans. Proc Natl Acad Sci USA  100: 171–176.  <ul class="find" data-citedArticleID="1055590" data-doi="10.1073/pnas.0136972100"><li><a href="http://dx.doi.org/10.1073/pnas.0136972100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+shaped+regional+mtDNA+variation+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+shaped+regional+mtDNA+variation+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001596-Thompson1" id="pone.0001596-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTALW: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucl Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="1055622" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTALW%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTALW%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001596-Handt1" id="pone.0001596-Handt1"></a>Handt O, Meyer S, von Haeseler A (1998) Compilation of human mtDNA control region sequences. Nucl Acids Res  26: 126–129.  <ul class="find" data-citedArticleID="1055554" data-doi="10.1093/nar/26.1.126"><li><a href="http://dx.doi.org/10.1093/nar/26.1.126" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compilation+of+human+mtDNA+control+region+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compilation+of+human+mtDNA+control+region+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001596-Polanski1" id="pone.0001596-Polanski1"></a>Polanski A, Kimmel M, Chakraborty R (1998) Application of a time-dependent coalescence process for inferring the history of population size changes from DNA sequence data. Proc Natl Acad Sci USA  95: 5456–5461.  <ul class="find" data-citedArticleID="1055604" data-doi="10.1073/pnas.95.10.5456"><li><a href="http://dx.doi.org/10.1073/pnas.95.10.5456" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+a+time-dependent+coalescence+process+for+inferring+the+history+of+population+size+changes+from+DNA+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+a+time-dependent+coalescence+process+for+inferring+the+history+of+population+size+changes+from+DNA+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001596-Kitchen1" id="pone.0001596-Kitchen1"></a>Kitchen A, Miyamoto MM, Mulligan CJ (2007) Utility of DNA viruses for studying human host history: Case study of JC virus. Mol Phylogenet Evol: In Press.   <ul class="find" data-citedArticleID="1055574" data-doi="10.1016/j.ympev.2007.09.005"><li><a href="http://dx.doi.org/10.1016/j.ympev.2007.09.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Utility+of+DNA+viruses+for+studying+human+host+history%3A+Case+study+of+JC+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Utility+of+DNA+viruses+for+studying+human+host+history%3A+Case+study+of+JC+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001596-Howell1" id="pone.0001596-Howell1"></a>Howell N, Smejkal CB, Mackey DA, Chinnery PF, Turnbull DM, et al.  (2003) The pedigree rate of sequence divergence in the human mitochondrial genome: there is a difference between phylogenetic and pedigree rates. Am J Hum Genet  72: 659–670.  <ul class="find" data-citedArticleID="1055566" data-doi="10.1086/368264"><li><a href="http://dx.doi.org/10.1086/368264" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+pedigree+rate+of+sequence+divergence+in+the+human+mitochondrial+genome%3A+there+is+a+difference+between+phylogenetic+and+pedigree+rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+pedigree+rate+of+sequence+divergence+in+the+human+mitochondrial+genome%3A+there+is+a+difference+between+phylogenetic+and+pedigree+rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001596-Weale1" id="pone.0001596-Weale1"></a>Weale ME, Weiss DA, Jager RF, Bradman N, Thomas MG (2002) Y chromosome evidence for Anglo-Saxon mass migration. Mol Biol Evol  19: 1008–1021.  <ul class="find" data-citedArticleID="1055632" data-doi="10.1093/oxfordjournals.molbev.a004160"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004160" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Y+chromosome+evidence+for+Anglo-Saxon+mass+migration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Y+chromosome+evidence+for+Anglo-Saxon+mass+migration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001596-Fenner1" id="pone.0001596-Fenner1"></a>Fenner JN (2004) Cross-cultural estimation of the human generation interval for use in genetics-based population divergence studies. Am J Phys Anthropol  128: 415–423.  <ul class="find" data-citedArticleID="1055540" data-doi="10.1002/ajpa.20188"><li><a href="http://dx.doi.org/10.1002/ajpa.20188" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cross-cultural+estimation+of+the+human+generation+interval+for+use+in+genetics-based+population+divergence+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cross-cultural+estimation+of+the+human+generation+interval+for+use+in+genetics-based+population+divergence+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001596-Anderson1" id="pone.0001596-Anderson1"></a>Anderson S, Bankier AT, Barrell BG, de Bruijn MH, Coulson AR, et al.  (1981) Sequence and organization of the human mitochondrial genome. Nature  290: 457–465.  <ul class="find" data-citedArticleID="1055526" data-doi="10.1038/290457a0"><li><a href="http://dx.doi.org/10.1038/290457a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+and+organization+of+the+human+mitochondrial+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+and+organization+of+the+human+mitochondrial+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.XML" value="97423"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.PDF" value="249720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g001.PNG_L" value="238018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g001.PNG_M" value="37895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g001.PNG_S" value="6581"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g001.TIF" value="328030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g001.PNG_I" value="13283"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g002.PNG_L" value="107266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g002.PNG_M" value="30570"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g002.PNG_S" value="5814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g002.TIF" value="170728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g002.PNG_I" value="11061"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g003.PNG_L" value="70733"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g003.PNG_M" value="22132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g003.PNG_S" value="5518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g003.TIF" value="124554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g003.PNG_I" value="8255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g004.PNG_L" value="1273446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g004.PNG_M" value="220736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g004.PNG_S" value="22275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g004.TIF" value="1473134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.g004.PNG_I" value="167637"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.s001.TXT" value="1194448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001596.s002.TXT" value="643378"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001596&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001596" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001596&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001596&volume=&issue=&title=A Three-Stage Colonization Model for the Peopling of the Americas&author_name=Andrew%20Kitchen%2C%20Michael%20M.%20Miyamoto%2C%20Connie%20J.%20Mulligan&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001596" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001596" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001596" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001596&amp;t=A%20Three-Stage%20Colonization%20Model%20for%20the%20Peopling%20of%20the%20Americas" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001596&title=A%20Three-Stage%20Colonization%20Model%20for%20the%20Peopling%20of%20the%20Americas&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001596&amp;title=A%20Three-Stage%20Colonization%20Model%20for%20the%20Peopling%20of%20the%20Americas" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001596" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001596&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Three-Stage Colonization Model for the Peopling of the Americas';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001596';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Three-Stage%20Colonization%20Model%20for%20the%20Peopling%20of%20the%20Americas http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001596" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001596" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Archaeological+dating%22" title="Search for articles in the subject area:'Archaeological dating'"><div class="flagText">Archaeological dating</div></a>
              <div data-categoryid="21569" data-articleid="26018"
                   data-categoryname="Archaeological dating"
                   class="flagImage" title="Flag 'Archaeological dating' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondrial+DNA%22" title="Search for articles in the subject area:'Mitochondrial DNA'"><div class="flagText">Mitochondrial DNA</div></a>
              <div data-categoryid="32451" data-articleid="26018"
                   data-categoryname="Mitochondrial DNA"
                   class="flagImage" title="Flag 'Mitochondrial DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Multiple+alignment+calculation%22" title="Search for articles in the subject area:'Multiple alignment calculation'"><div class="flagText">Multiple alignment calculation</div></a>
              <div data-categoryid="20427" data-articleid="26018"
                   data-categoryname="Multiple alignment calculation"
                   class="flagImage" title="Flag 'Multiple alignment calculation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Paleoecology%22" title="Search for articles in the subject area:'Paleoecology'"><div class="flagText">Paleoecology</div></a>
              <div data-categoryid="35111" data-articleid="26018"
                   data-categoryname="Paleoecology"
                   class="flagImage" title="Flag 'Paleoecology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Paleogenetics%22" title="Search for articles in the subject area:'Paleogenetics'"><div class="flagText">Paleogenetics</div></a>
              <div data-categoryid="33815" data-articleid="26018"
                   data-categoryname="Paleogenetics"
                   class="flagImage" title="Flag 'Paleogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+size%22" title="Search for articles in the subject area:'Population size'"><div class="flagText">Population size</div></a>
              <div data-categoryid="33245" data-articleid="26018"
                   data-categoryname="Population size"
                   class="flagImage" title="Flag 'Population size' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Radioactive+carbon+dating%22" title="Search for articles in the subject area:'Radioactive carbon dating'"><div class="flagText">Radioactive carbon dating</div></a>
              <div data-categoryid="21571" data-articleid="26018"
                   data-categoryname="Radioactive carbon dating"
                   class="flagImage" title="Flag 'Radioactive carbon dating' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="26018"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2013'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1657'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2577&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=53865">How far early migrating humans moved ...</a><br>Posted by hikinginapacheria</p>
      <p><a href="/annotation/listThread.action?root=21019">Media Coverage of This Article</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=17191">Will Open Access make interdisciplinary work more Post-Positivist?</a><br>Posted by alun</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
