

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Evolution of a Complex Locus: Exon Gain, Loss and Divergence at the Gr39a Locus in Drosophila</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001513"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001513" />

    <meta name="citation_title" content="Evolution of a Complex Locus: Exon Gain, Loss and Divergence at the Gr39a Locus in Drosophila"/>
    <meta itemprop="name" content="Evolution of a Complex Locus: Exon Gain, Loss and Divergence at the Gr39a Locus in Drosophila"/>

      <meta name="citation_author" content="Anastasia Gardiner"/>
            <meta name="citation_author_institution" content="School of Biology, University of St. Andrews, St. Andrews, Scotland, United Kingdom"/>
      <meta name="citation_author" content="Daniel Barker"/>
            <meta name="citation_author_institution" content="School of Biology, University of St. Andrews, St. Andrews, Scotland, United Kingdom"/>
      <meta name="citation_author" content="Roger K. Butlin"/>
            <meta name="citation_author_institution" content="Department of Animal and Plant Sciences, University of Sheffield, Sheffield, United Kingdom"/>
      <meta name="citation_author" content="William C. Jordan"/>
            <meta name="citation_author_institution" content="Institute of Zoology, Zoological Society of London, London, United Kingdom"/>
      <meta name="citation_author" content="Michael G. Ritchie"/>
            <meta name="citation_author_institution" content="School of Biology, University of St. Andrews, St. Andrews, Scotland, United Kingdom"/>

    <meta name="citation_date" content="2008/1/30"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001513.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1513"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Molecular evolution of the insect chemoreceptor gene superfamily in Drosophila melanogaster.; citation_author=HM Robertson; citation_author=CG Warr; citation_author=JR Carlson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=1; citation_pages=14537-14542; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolutionary dynamics of olfactory receptor genes in Drosophila species.; citation_author=M Nozawa; citation_author=M Nei; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=2; citation_pages=7122-7127; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution of Drosophila odorant receptor genes.; citation_author=S Guo; citation_author=J Kim; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=3; citation_pages=1198-1207; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=G protein-coupled receptors in Anopheles gambiae.; citation_author=CA Hill; citation_author=AN Fox; citation_author=RJ Pitts; citation_author=LB Kent; citation_author=PL Tan; citation_journal_title=Science; citation_volume=298; citation_number=4; citation_pages=176-178; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The chemoreceptor superfamily in the honey bee Apis mellifera: expansion of the odorant, but not gustatory, receptor family.; citation_author=HM Robertson; citation_author=KW Wanner; citation_journal_title=Genome Res; citation_volume=16; citation_number=5; citation_pages=1395-1403; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Candidate taste receptors in Drosophila.; citation_author=PJ Clyne; citation_author=CG Warr; citation_author=JR Carlson; citation_journal_title=Science; citation_volume=287; citation_number=6; citation_pages=1830-1834; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Drosophila Chemoreceptor Gene Evolution: Selection, Specialisation and Genome Size. Molecular Ecology, submitted.; citation_author=A Gardiner; citation_author=D Barker; citation_author=RK Butlin; citation_author=WC Jordan; citation_author=MG Ritchie; citation_number=7; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A putative Drosophila pheromone receptor expressed in male-specific taste neurons is required for efficient courtship.; citation_author=S Bray; citation_author=H Amrein; citation_journal_title=Neuron; citation_volume=39; citation_number=8; citation_pages=1019-1029; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Pheromone perception and behavior in Drosophila.; citation_author=H Amrein; citation_journal_title=Current Opinion in Neurobiology; citation_volume=14; citation_number=9; citation_pages=435-442; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolution of genes and genomes on the Drosophila phylogeny.; citation_journal_title=Nature; citation_volume=450; citation_number=10; citation_pages=203-218; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny and divergence times of drosophilid species.; citation_author=CA Russo; citation_author=N Takezaki; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=12; citation_number=11; citation_pages=391-404; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Temporal Patterns of Fruit Fly (Drosophila) Evolution Revealed by Mutation Clocks.; citation_author=K Tamura; citation_author=S Subramanian; citation_author=S Kumar; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=12; citation_pages=36-44; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Candidate odorant receptors from the malaria vector mosquito Anopheles gambiae and evidence of down-regulation in response to blood feeding.; citation_author=AN Fox; citation_author=RJ Pitts; citation_author=HM Robertson; citation_author=JR Carlson; citation_author=LJ Zwiebel; citation_journal_title=Proceedings of the National Academy of Sciences; citation_volume=98; citation_number=13; citation_pages=14693-14697; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A Hybrid Micro-Macroevolutionary Approach to Gene Tree Reconstruction.; citation_author=D Durand; citation_author=BH Halldorsson; citation_author=B Vernot; citation_journal_title=J Comput Biol; citation_volume=13; citation_number=14; citation_pages=320-335; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Statistical methods for testing functional divergence after gene duplication.; citation_author=X Gu; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=15; citation_pages=1664-1674; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mathematical Modeling for Functional Divergence after Gene Duplication.; citation_author=X Gu; citation_journal_title=Journal of Computational Biology; citation_volume=8; citation_number=16; citation_pages=221-234; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Functional divergence in protein (family) sequence evolution.; citation_author=X Gu; citation_journal_title=Genetica; citation_volume=118; citation_number=17; citation_pages=133-141; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Common exon duplication in animals and its role in alternative splicing.; citation_author=I Letunic; citation_author=RR Copley; citation_author=P Bork; citation_journal_title=Hum Mol Genet; citation_volume=11; citation_number=18; citation_pages=1561-1567; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Five Drosophila Genomes Reveal Non-Neutral Evolution and the Signature of Host Specialization in the Chemoreceptor Superfamily.; citation_author=CS McBride; citation_author=RJ Arguello; citation_journal_title=Genetics; citation_volume=177; citation_number=19; citation_pages=1395-1416; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The (In)dependence of Alternative Splicing and Gene Duplication.; citation_author=D Talavera; citation_author=C Vogel; citation_author=M Orozco; citation_author=SA Teichmann; citation_author=X de la Cruz; citation_journal_title=PLoS Comput Biol; citation_volume=3; citation_number=20; citation_pages=e33; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evolution by gene duplication.; citation_author=S Ohno; citation_number=21; citation_pages=160; citation_date=1970; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=The probability of duplicate gene preservation by subfunctionalization.; citation_author=M Lynch; citation_author=A Force; citation_journal_title=Genetics; citation_volume=154; citation_number=22; citation_pages=459-473; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The evolution of functionally novel proteins after gene duplication.; citation_author=AL Hughes; citation_journal_title=Proc R Soc Lond B; citation_volume=256; citation_number=23; citation_pages=119-124; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The odor coding system of Drosophila.; citation_author=EA Hallem; citation_author=JR Carlson; citation_journal_title=Trends in Genetics; citation_volume=20; citation_number=24; citation_pages=453-459; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Origin of alternative splicing by tandem exon duplication.; citation_author=FA Kondrashov; citation_author=EV Koonin; citation_journal_title=Hum Mol Genet; citation_volume=10; citation_number=25; citation_pages=2661-2669; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Allelic divergence precedes and promotes gene duplication.; citation_author=SR Proulx; citation_author=PC Phillips; citation_journal_title=Evolution; citation_volume=60; citation_number=26; citation_pages=881-892; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucl Acids Res; citation_volume=25; citation_number=27; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=GeneWise and Genomewise.; citation_author=E Birney; citation_author=M Clamp; citation_author=R Durbin; citation_journal_title=Genome Res; citation_volume=14; citation_number=28; citation_pages=988-995; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucl Acids Res; citation_volume=22; citation_number=29; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Predicting transmembrane protein topology with a hidden Markov model: Application to complete genomes.; citation_author=A Krogh; citation_author=B Larsson; citation_author=G von Heijne; citation_author=ELL Sonnhammer; citation_journal_title=J Mol Biol; citation_volume=305; citation_number=30; citation_pages=567-580; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=J Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=31; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Bayesian Estimation of Ancestral Character States on Phylogenies.; citation_author=M Pagel; citation_author=A Meade; citation_author=D Barker; citation_journal_title=System Biol; citation_volume=53; citation_number=32; citation_pages=673-684; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Further Results on Tests of Separate Families of Hypotheses.; citation_author=DR Cox; citation_journal_title=Journal of the Royal Statistical Society B; citation_volume=24; citation_number=33; citation_pages=406-424; citation_date=1962; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Computer Applications in the Biosciences; citation_volume=13; citation_number=34; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Statistical methods for detecting molecular adaptation.; citation_author=Z Yang; citation_author=JP Bielawski; citation_journal_title=Trends in Ecology and Evolution; citation_volume=15; citation_number=35; citation_pages=496-503; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001513" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Evolution of a Complex Locus: Exon Gain, Loss and Divergence at the Gr39a Locus in Drosophila"/>
    <meta name="twitter:description" content="BackgroundGene families typically evolve by gene duplication followed by the adoption of new or altered gene functions. A different way to evolve new but related functions is alternative splicing of existing exons of a complex gene. The chemosensory gene families of animals are characterised by numerous loci of related function. Alternative splicing has only rarely been reported in chemosensory loci, for example in 5 out of around 120 loci in Drosophila melanogaster. The gustatory receptor gene Gr39a has four large exons that are alternatively spliced with three small conserved exons. Recently the genome sequences of eleven additional species of Drosophila have become available allowing us to examine variation in the structure of the Gr39a locus across a wide phylogenetic range of fly species.Methodology/Principal FindingsWe describe a fifth exon and show that the locus has a complex evolutionary history with several duplications, pseudogenisations and losses of exons. PAML analyses suggested that the whole gene has a history of purifying selection, although this was less strong in exons which underwent duplication.Conclusions/SignificanceEstimates of functional divergence between exons were similar in magnitude to functional divergence between duplicated genes, suggesting that exon divergence is broadly equivalent to gene duplication."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001513.g004"/>

  <meta property="og:title" content="Evolution of a Complex Locus: Exon Gain, Loss and Divergence at the Gr39a Locus in Drosophila" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001513" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1730'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=433'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6207&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001513">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001513" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001513&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Evolution of a Complex Locus: Exon Gain, Loss and Divergence at the Gr39a Locus in Drosophila
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anastasia Gardiner
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ag91@st-andrews.ac.uk">ag91@st-andrews.ac.uk</a></p>

                <p>Affiliation:
                  School of Biology, University of St. Andrews, St. Andrews, Scotland, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel Barker, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biology, University of St. Andrews, St. Andrews, Scotland, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Roger K. Butlin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Animal and Plant Sciences, University of Sheffield, Sheffield, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            William C. Jordan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Zoology, Zoological Society of London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael G. Ritchie
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biology, University of St. Andrews, St. Andrews, Scotland, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 30, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001513</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001513" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001513" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001513" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001513" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001513-g001" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001513-g002" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001513-g003" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001513-t001" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001513-t002" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001513-g004" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001513">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001513" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Gene families typically evolve by gene duplication followed by the adoption of new or altered gene functions. A different way to evolve new but related functions is alternative splicing of existing exons of a complex gene. The chemosensory gene families of animals are characterised by numerous loci of related function. Alternative splicing has only rarely been reported in chemosensory loci, for example in 5 out of around 120 loci in <em>Drosophila melanogaster</em>. The gustatory receptor gene <em>Gr39a</em> has four large exons that are alternatively spliced with three small conserved exons. Recently the genome sequences of eleven additional species of Drosophila have become available allowing us to examine variation in the structure of the <em>Gr39a</em> locus across a wide phylogenetic range of fly species.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We describe a fifth exon and show that the locus has a complex evolutionary history with several duplications, pseudogenisations and losses of exons. PAML analyses suggested that the whole gene has a history of purifying selection, although this was less strong in exons which underwent duplication.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Estimates of functional divergence between exons were similar in magnitude to functional divergence between duplicated genes, suggesting that exon divergence is broadly equivalent to gene duplication.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Gardiner A, Barker D, Butlin RK, Jordan WC, Ritchie MG (2008) Evolution of a Complex Locus: Exon Gain, Loss and Divergence at the Gr39a Locus in Drosophila. PLoS ONE 3(1):
          e1513.
            doi:10.1371/journal.pone.0001513</p><p><strong>Academic Editor: </strong>Richard Copley, Wellcome Trust Centre for Human Genetics, United Kingdom</p><p><strong>Received:</strong> October 5, 2007; <strong>Accepted:</strong> January 2, 2008; <strong>Published:</strong> January 30, 2008</p><p><strong>Copyright:</strong> © 2008 Gardiner et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Natural Environment Research Council UK, grant NE/C003187/1. DB was supported by a Research Councils UK Academic Fellowship.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Gustatory receptor (<em>Gr</em>) genes comprise a large fraction (~50%) of the Drosophila chemosensory receptor gene superfamily <a href="#pone.0001513-Robertson1">[1]</a>, encoding 7-transmembrane (7TM) proteins involved in taste and smell. Most Drosophila <em>Grs</em> are very divergent, sometimes showing as little as 8% amino acid identity to each other <a href="#pone.0001513-Robertson1">[1]</a>. Much of the diversity of the chemoreceptor family has evolved through widespread and repeated whole-gene duplications, followed by functional divergence of those duplicates that do not degrade to pseudogenes <a href="#pone.0001513-Nozawa1">[2]</a>, <a href="#pone.0001513-Guo1">[3]</a>. Another mechanism that enlarges the eukaryotic protein repertoire in general is alternative splicing. Although this is currently thought to be rare among chemosensory receptor loci <a href="#pone.0001513-Hill1">[4]</a>, <a href="#pone.0001513-Robertson2">[5]</a>, three <em>D. melanogaster Gr</em> genes (<em>Gr23a, Gr28b</em>, <em>Gr39a</em>) are notable in that they have been shown to undergo alternative splicing, together coding for 11 proteins, or 16% of all gustatory receptors in the species <a href="#pone.0001513-Robertson1">[1]</a>, <a href="#pone.0001513-Clyne1">[6]</a>. Analysis of the <em>Gr</em> repertoire in 12 Drosophila species allowed us to identify the orthologues of the <em>Gr39a</em> genes in these species <a href="#pone.0001513-Gardiner1">[7]</a>. This locus showed an unusual pattern of structural changes compared with other <em>Grs</em>. Here we investigate in detail the evolution of the <em>Gr39a</em> gene using a comparative, bioinformatic approach.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Located on the left arm of the second chromosome of <em>D. melanogaster</em>, the <em>Gr39a</em> gene has four large exons (<em>A</em>, <em>B</em>, <em>C</em> and <em>D</em>), each including coding sequences for six transmembrane domains, followed by three small exons that together encode the seventh transmembrane domain and COOH-terminus <a href="#pone.0001513-Clyne1">[6]</a>. Any one of the large exons may be spliced to the smaller exons, generating four different 7TM protein products. These are expressed in the main taste organs of <em>D. melanogaster</em>, the labellum (Gr39aA, Gr39aB, Gr39aC, Gr39aD), though some are also expressed in the thorax (Gr39aC, Gr39aD), abdomen (Gr39aC) and wings (Gr39aD) <a href="#pone.0001513-Clyne1">[6]</a>. The function of <em>Gr39a</em> is unknown, but its close phylogenetic affinity with the <em>D. melanogaster</em> male specific pheromone receptor <em>Gr68a</em> <a href="#pone.0001513-Bray1">[8]</a> suggests a possible involvement in pheromone recognition <a href="#pone.0001513-Amrein1">[9]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We have annotated the orthologs of <em>D. melanogaster Gr39a</em> in eleven other recently sequenced Drosophila species <a href="#pone.0001513-Drosophila1">[10]</a>, representing a wide range of phylogenetic divergence from <em>D. melanogaster</em> (<a href="#pone-0001513-g001">Figure 1A</a>, <a href="#pone.0001513-Drosophila1">[10]</a>). Of these species, nine (<em>D. melanogaster, D. simulans, D. sechellia, D. yakuba</em>, <em>D. erecta</em>, <em>D. ananassae</em>, <em>D. pseudoobscura</em>, <em>D. persimilis</em> and <em>D. willistoni</em>) are in the subgenus <em>Sophophora</em>, and the remaining three (<em>D. mojavensis</em>, <em>D. virilis</em> and <em>D. grimshawi</em>) are within the subgenus <em>Drosophila</em>. The two subgenera are estimated to have diverged from each other 40–60 million years ago <a href="#pone.0001513-Russo1">[11]</a>, <a href="#pone.0001513-Tamura1">[12]</a>. Lower level groups and subgroups have been identified within the subgenera (<a href="#pone-0001513-g001">Figure 1A</a>). We examined the structural and potentially functional differences between the <em>Gr39a</em> genes across these twelve species. We identified a new large exon, exon <em>E</em>, found in most species but lost in the melanogaster lineage. After an analysis in which we employ phylogenomic approaches more usually used to examine gene evolution, we propose a model of the evolution of <em>Gr39a</em>. We conclude that, despite strong purifying constraints on the <em>Gr39a</em> locus overall, the exons that are prone to duplication or pseudogenisation show evidence of relaxed selection which probably facilitated “subfunctionalization” of the duplicated exon copies. Evidence of positive selection also suggests “specialization” and/or neofunctionalization of tandemly duplicated exons has occurred, though potential new functions are currently unknown.</p>
<div class="figure" id="pone-0001513-g001"><div class="img"><a name="pone-0001513-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001513.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001513.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Structure of the <em>Gr39a</em> locus and splicing pattern with species' phylogeny.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>A. Schematic presentation of the <em>Gr39a</em> structure (in order) in <em>D. melanogaster, D. simulans, D. sechellia, D. yakuba, D. erecta, D. ananassae, D. pseudoobscura, D. persimilis, D. willistoni, D. mojavensis, D. virilis, D. grimshawi.</em> Degraded exons or exons accumulated nonsense mutations are crossed. B. Schematic of the pattern of alternative splicing in the <em>Gr39a</em>. Large exons have the 5′-donor site (GT), while the first 3′-acceptor site (AG) appears in the beginning of the block of three conserved small exons.</p>
<span>doi:10.1371/journal.pone.0001513.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Chemosensory receptor repertoires among different insects show great divergence, for instance only a few orthologous groups were identified when the complete olfactory and gustatory receptor repertoires of the fruit fly, honeybee and mosquito were compared <a href="#pone.0001513-Hill1">[4]</a>, <a href="#pone.0001513-Robertson2">[5]</a>, <a href="#pone.0001513-Fox1">[13]</a>. Robertson and Wanner <a href="#pone.0001513-Robertson2">[5]</a> suggest that the exons <em>Gr39aA-D</em> of <em>Drosophila melanogaster</em> form an orthologous group with seventeen <em>Gr</em> genes of <em>Anopheles gambiae</em> (<em>AgGr9a</em>-<em>n</em>, <em>AgGr10</em>, <em>AgGr11</em>, and <em>AgGr12</em>), but the evidence of the orthology is lacking due to the very weak bootstrap support for this clade. In <em>Apis mellifera</em>, the orthologs of <em>Gr39a</em> were not identified <a href="#pone.0001513-Robertson2">[5]</a>. The <em>Drosophila</em> exons are related among themselves by duplications, which seem to have occurred after the <em>Drosophila</em>-<em>Anopheles</em> split.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>We were able to uncover complex structural changes that have occurred to <em>Gr39a</em> since the subgenera <em>Drosophila</em> and <em>Sophophora</em> diverged. The gene structure of <em>Gr39a</em> described for <em>D. melanogaster,</em> with four large exons <em>A, B, C,</em> and <em>D</em> followed by three small constitutive exons <a href="#pone.0001513-Clyne1">[6]</a>, is peculiar to the species of the melanogaster subgroup only (<a href="#pone-0001513-g001">Figure 1A</a>). In this subgroup, the first exon <em>A</em> has either accumulated frame shift mutations or significantly degraded in two species: <em>D. sechellia</em> and <em>D. erecta</em>, respectively. Each large exon contains its own start codon and 5′ splice signal allowing the locus to encode several protein products independent of the mutational alteration of the ORF in one of the exons, so we would expect the exons <em>B, C</em> and <em>D</em> to be expressed in <em>D. sechellia</em> and <em>D. erecta</em>. Species of the melanogaster subgroup have also lost an ancestral exon <em>E,</em> first described here and identified as a degraded copy in <em>D. ananassae</em>, an intact exon in the species of the obscura group, <em>D. willistoni, D. mojavensis</em> and <em>D. grimshawi</em>, and an exon with two frame-shift mutations in <em>D. virilis</em> (<a href="#pone-0001513-g001">Figure 1A</a>). In support of this interpretation, a search for evidence of exon <em>E</em> in the melanogaster subgroup revealed the presence of short sequences that code for about 100 amino acids alignable with the truncated exon <em>E</em> of <em>D. ananassae</em> and intact exon <em>E</em> of <em>D. pseudoobscura</em>.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>The Bayesian-MCMC phylogeny of <em>GR39a</em> exons and their homologues is summarized in <a href="#pone-0001513-g002">Figure 2</a>. The posterior probabilities shown in <a href="#pone-0001513-g002">Figure 2</a> appear to lack significant bias as estimates of the probabilities of the clade, conditional on the data, model and priors: average standard deviation of split frequencies was 0.0060 at the start of the final MCMC sample (close to the ideal value of zero), and lag-1 ACF for lnL in the two MrBayes runs was 0.014 (P&gt;0.05) and 0.036 (P&gt;0.05).</p>
<div class="figure" id="pone-0001513-g002"><div class="img"><a name="pone-0001513-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001513.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001513.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span><em>Gr39a</em> “exons” tree for 12 <em>Drosophila</em> species.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Species: Dmel (<em>D. melanogaster</em>), Dsec (<em>D. sechellia</em>), Dsim (<em>D. simulans</em>), Dyak (<em>D. yakuba</em>), Dere (<em>D. erecta</em>), Dana (<em>D. ananassae</em>), Dpse (<em>D. pseudoobscura</em>), Dwil (<em>D. willistoni</em>), Dmoj (<em>D. mojavensis</em>), Dvir (<em>D. virilis</em>), Dgri (<em>D. grimshawi</em>). Exons with nonsense mutations are indicated by symbol “P”. Numbers show clade posterior probabilities.</p>
<span>doi:10.1371/journal.pone.0001513.g002</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>Exon <em>E</em> occupies a basal position within a clade including exons <em>C</em> and <em>D</em> (<a href="#pone-0001513-g002">Figure 2</a>). The reconciliation of the species tree and phylogenetic tree of exons <em>E, C</em> and <em>D</em> by the Notung program <a href="#pone.0001513-Durand1">[14]</a> showed two events of exon duplication (duplication of exon <em>E</em> and subsequent origin of exons <em>D</em> and <em>C</em> before the subgenera <em>Sophophora</em> and <em>Drosophila</em> split) and three independent events of exon loss (<a href="#pone-0001513-g003">Figure 3</a>). This prediction supports the loss of exon <em>E</em> in the melanogaster group, and suggests the loss of exon <em>C</em> in <em>D. willistoni</em> and the subgenus <em>Drosophila</em>, but we did not find any evidence of the presence of exon <em>C</em> in these species.</p>
<div class="figure" id="pone-0001513-g003"><div class="img"><a name="pone-0001513-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001513.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001513.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Predictions of duplication (in blue) and loss (in red) events by parsimony.</span></strong></p><span>doi:10.1371/journal.pone.0001513.g003</span></div><a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>Closely related to each other, exons <em>B</em> and <em>A</em> were found in all species, <em>B</em> as a single copy exon, while exon <em>A</em> underwent multiple duplications in several lineages (<a href="#pone-0001513-g001">Figure 1A</a>). The relationships among copies of exon <em>A</em> are complex. We found three copies of exon <em>A</em> in the obscura and willistoni groups, seven copies (two pseudo- and five intact exons) in <em>D. mojavensis</em>, six intact copies in <em>D. virilis</em> and four in <em>D. grimshawii</em> (<a href="#pone-0001513-g001">Figure 1A</a> and <a href="#pone-0001513-g002">Figure 2</a>). Some exon copies are very recent and/or experienced gene conversion, others are likely to be of more ancient origin, possibly suggesting losses as well as gains. Such extensive exon duplication in an alternatively spliced chemoreceptor gene is an apparently unique case-analyses of the other alternatively spliced gustatory (<em>Gr23a, Gr28b</em>) and olfactory (<em>Or46a, Or69a</em>) receptor genes revealed a more conservative structure. Thus, the structure of <em>Gr23a,</em> with two large alternatively spliced exons, is preserved in all species examined. We found only cases of species-specific degradation or loss of some exons in the <em>Gr28b, Or46a</em> and <em>Or69a</em> genes due to accumulation of frame shifts, premature stop codons and large deletions (for example, <em>Gr28bA</em> in <em>D. sechellia</em>, <em>Gr28bD</em> and <em>E</em> in <em>D. grimshawii, Or69aC</em> in <em>D. sechellia</em> and <em>D. melanogaster</em>).</p>
<a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>The presence of sequence motifs that specify six transmembrane domains was detected in all the large exons of <em>Gr39a</em>; the last three small exons are present in all species and encode for the seventh transmembrane domain (<a href="#pone-0001513-g001">Figure 1A</a>). We analysed the splicing structure of <em>Gr39a</em> in all species. The location of splice sites is conserved throughout the genes examined (<a href="#pone-0001513-g001">Figure 1B</a>). All large exons (except the pseudo-exons <em>A</em> in <em>D. sechellia</em> and <em>D. erecta</em>) and two conserved small exons contain the 5′ donor-splice motif [AG]↑<strong>GT</strong>(NNGT), while the first 3′ acceptor-splice motif (C<sub>n</sub>T<sub>n</sub>)NC<strong>AG</strong>↑[GC] appears at the beginning of the block of three conserved small exons (<a href="#pone-0001513-g001">Figure 1B</a>). This structure supports a model of mutually exclusive alternative splicing, when a single large exon is spliced with the small conserved exons and the other large exons are excluded as part of an intron.</p>
<a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>The estimated <em>ω</em> values for the whole gene and its large exons were all substantially lower than 1 (<a href="#pone-0001513-t001">Table 1</a>) suggesting that all parts of the gene are subject to strong purifying selection, however weaker selective constraints act on exon <em>E</em> and the duplicated exon <em>A</em> (<em>ωE</em> = 0.27 and <em>ωA</em> = 0.24; c.f. ~0.17).</p>
<div class="figure" id="pone-0001513-t001"><div class="img"><a name="pone-0001513-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001513.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001513.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>PAML analysis of selection on the <em>Gr39a</em> gene and its tandemly duplicated large exons (<em>A, B, C, D, E</em>).</span></strong></p><span>doi:10.1371/journal.pone.0001513.t001</span></div><a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p>Evidence of expression of <em>Gr39a</em> in <em>D. melanogaster</em> <a href="#pone.0001513-Clyne1">[6]</a>, strong selective constraints, and preservation of the transmembrane domain structure and splice signals all suggest that <em>Gr39a</em> is a functionally active gene. The estimates of the coefficient of functional divergence (<em>θ)</em> <a href="#pone.0001513-Gu1">[15]</a>–<a href="#pone.0001513-Gu3">[17]</a> between the protein isoforms of <em>Gr39a</em> are generally high (0.4936–0.6432) (<a href="#pone-0001513-t002">Table 2</a>) and comparable with estimates of <em>θ</em> for highly functionally diverged duplicated genes <a href="#pone.0001513-Gu1">[15]</a> and duplicated <em>Grs</em> in Drosophila species (<a href="#pone.0001513.s002">Table S2</a>). These results suggest functional divergence between exons <em>B, C, D,</em> and <em>E</em>. Interestingly, when we analysed functional divergence between isoforms <em>B</em> and <em>A</em>, including the single copies of <em>A</em> from each species that were considered to be conserved orthologs of <em>A</em> of <em>D. melanogaster</em>, we detected very low values of <em>θ</em> between <em>A</em> and <em>B</em> (<em>θ</em> = 0.1408, not significantly different from zero), providing no evidence that these exons have diverged in their function, but rather suggesting that replacement mutations are due to neutral evolution. However, the inclusion of the duplicate isoforms of <em>A</em> of <em>D. mojavensis</em>, <em>D. virilis</em> and <em>D. grimshawii</em> increased <em>θ</em> to 0.28–0.38 (P&lt;0.0001), when the <em>A</em> and <em>B</em> clusters were compared. The comparison of the <em>A</em> isoforms of <em>Sophophora</em> versus the <em>A</em> isoforms of <em>D. mojavensis, D. virilis</em> and <em>D. grimsshawii</em> also provided evidence of functional divergence between them (<em>θ</em> ranged from 0.27 to 0.49, P&lt;0.004–0.002). We also observed low divergence between exons <em>C</em> and <em>D</em>, possibly indicating similar functions.</p>
<div class="figure" id="pone-0001513-t002"><div class="img"><a name="pone-0001513-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001513.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001513.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Estimates of functional divergence between the large exons (<em>A</em>, <em>B, C, D, E</em>) of the <em>Gr39a</em> gene.</span></strong></p><span>doi:10.1371/journal.pone.0001513.t002</span></div><a id="article1.body1.sec2.p9" name="article1.body1.sec2.p9"></a><p>We tested the main regions of the <em>Gr39a</em> gene for signs of positive selection using several models in PAML. Application of the site and branch-site models detected positive selection on exon <em>B</em> (P&lt;0.011) in the subgenus <em>Drosophila</em> during the diversification of <em>D. grimshawi</em>, <em>D. mojavensis</em> and <em>D. virilis</em> (<a href="#pone-0001513-t001">Table 1</a>). Only 1.2% of sites of exon <em>B</em> underwent positive selection with <em>ω</em> = 2.4; sites <em>28T, 60T, 68S</em> and <em>193D</em>. We applied branch models to test variation in <em>ω</em> in exons <em>B, C</em> and <em>D</em> on phylogenetic lineages within the melanogaster group (we excluded exon <em>A</em>, because it degraded in two melanogaster group species). <em>ω</em> exceeded 1 (the neutral expectation) in exon <em>C</em> of <em>D. sechellia</em> (<em>ω</em> = 1.43). Analyzing exon <em>C</em>, we also found an increase in <em>ω</em> in <em>D. simulans</em> (<em>ω = </em>0.8138) and <em>D. melanogaster</em> (<em>ω = </em>0.7554). To examine this further, pairwise comparisons of the <em>d<sub>N</sub></em> (nonsynonymous substitution rate) and <em>d<sub>S</sub></em> (synonymous substitution rate) of the exons <em>C, B, D</em> (excluding exon <em>A</em>, because it degraded in <em>D. sechellia</em>) and small conserved exons were carried out in <em>D. sechellia</em> vs. <em>D. melanogaster,</em> and <em>D. simulans</em> vs. <em>D. melanogaster</em> (<a href="#pone-0001513-g004">Figure 4A</a>) (comparing closely related species excludes potential problems of accounting for multiple hits at the synonymous sites and saturation). At the conserved small exons the <em>d<sub>S</sub></em> rates exceeded the <em>d<sub>N</sub></em> rates approximately ten times, indicating very strong negative selection against nonsynonymous mutations (<a href="#pone-0001513-g004">Figure 4A</a>). We found signs of a slight increase in the <em>d<sub>S</sub></em> and <em>d<sub>N</sub></em> rates in exon <em>D</em> of <em>D. sechellia</em>, and also an increase in the <em>d<sub>N</sub></em> rates of exon <em>C</em> in <em>D. sechellia</em> and <em>D. simulans</em>, while the <em>d<sub>S</sub></em> rates of exon <em>C</em> were relatively stable compared with the <em>d<sub>S</sub></em> rates in exons <em>B</em> or <em>D</em>. Thus, the <em>d<sub>S</sub></em> rates of exons <em>B</em> and <em>D</em> were nearly three times higher than their <em>d<sub>N</sub></em> rates, while in the exon <em>C</em> the substitution rate at the synonymous sites was about twice higher than that at the nonsynonymous sites (<a href="#pone-0001513-g004">Figure 4A</a>). For comparison, we also analysed the <em>d<sub>S</sub>, d<sub>N</sub></em> rates and the <em>ω</em> ratio <em>(d<sub>N</sub>/d<sub>S</sub>)</em> amongst the exons <em>A, B, C, D</em> and small conserved exons between <em>D. melanogaster</em> and <em>D. yakuba</em> (<a href="#pone-0001513-g004">Figure 4B</a>). Along with the increase of the <em>ω</em> in the exon <em>C</em>, we found similar patterns in exon <em>A</em>, in both cases changes of <em>ω</em> happened due to an increased rate of nonsynonymous substitution.</p>
<div class="figure" id="pone-0001513-g004"><div class="img"><a name="pone-0001513-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001513" data-uri="info:doi/10.1371/journal.pone.0001513.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001513.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001513.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001513.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001513.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Comparisons of substitution rates and <em>ω</em>.</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>A. Pairwise comparison of the <em>d<sub>S</sub></em> and <em>d<sub>N</sub></em> rates of exons <em>B, C, D</em> and three conservative small exons (symbol 3 on the figure) in <em>D. simulans</em> vs. <em>D. melanogaster</em> and <em>D. sechellia</em> vs. <em>D. melanogaster</em>. B. Results of pairwise comparison of the <em>d<sub>S</sub>, d<sub>N</sub></em> rates and of the <em>ω</em> ratio of the exons <em>A, B, C, D</em> and three conservative small exons (symbol 3 on the figure) in the pair of species: <em>D. melanogaster</em> vs. <em>D. yakuba</em>.</p>
<span>doi:10.1371/journal.pone.0001513.g004</span></div></div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>About 10% of Drosophila genes contain tandemly duplicated exons, many of which are believed to be involved in mutually exclusive alternative splicing events <a href="#pone.0001513-Letunic1">[18]</a>. Of the chemosensory receptor gene family, which contains around 120 genes, only three gustatory (<em>Gr23a, Gr28b, Gr39a</em>) and two olfactory (<em>Or46a, Or69a</em>) receptor genes have tandemly duplicated exons, which have been shown to undergo alternative splicing <a href="#pone.0001513-Robertson1">[1]</a>. Among these genes, <em>Gr39a</em> is a unique case of rapid evolution through exon duplication and divergence, and in some cases exon loss.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Several structural changes have occurred to <em>Gr39a</em>, including the loss of exon <em>C</em> in <em>D. willistoni</em> and the species of <em>Drosophila</em> subgenus, and multiple duplications of exon <em>A</em> in <em>D. pseudoobscura, D. persimilis, D. willistoni, D. mojavensis, D. virilis</em> and <em>D. grimshawii</em>, around the time of the <em>Drosophila-Sophophora</em> division. The species of the melanogaster group have lost the ancestral exon <em>E</em>, also pseudogenisation of exon <em>A</em> has occurred in <em>D. sechellia</em> and <em>D. erecta</em>. McBride and Arguello <a href="#pone.0001513-McBride1">[19]</a> also reported pseudogenisation of parts of the <em>Gr</em> repertoire in these last two species and related this to the ecological specialisation shown by these species <a href="#pone.0001513-Gardiner1">[7]</a>. Evidence for strong purifying constraints on <em>Gr39a</em> supports its status as a functional gene. We found evidence of functional divergence between exons <em>B, E, C</em> and <em>D</em>. However, based on the low level of functional divergence between the exons <em>C</em> and <em>D</em>, it is likely that they have similar functions, even though both exons have persisted over a long evolutionary period. In <em>D. melanogaster</em>, both exons are expressed in the labellum and thorax, but also show spatial delimition with Gr39aC being expressed in the abdomen and Gr39aD in the wings <a href="#pone.0001513-Clyne1">[6]</a>. The increase of the nonsynonymous substitution rate we detected in <em>Gr39aC</em> of <em>D. simulans</em> and, especially, in the specialist <em>D. sechellia</em> must indicate exon diversification.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We found little evidence of functional divergence between exons <em>A</em> and <em>B</em>. Curiously, exon <em>A</em> is multiply duplicated in all lineages except the melanogaster group, where it was lost in the two specialist species. The duplicates of exon <em>A</em> present in <em>D. mojavensis, D. virilis</em> and <em>D. grimshawii</em> show evidence of functional divergence from exon <em>B</em>, as well as from exon <em>A</em> of the <em>Sophophora</em> subgenus. We also detected positive selection acting on <em>Gr39aB</em> in the <em>Drosophila</em> subgenus. The extensive creation of new copies of exon <em>A</em> and signs of positive selection acting on <em>Gr39aB</em> in the <em>Drosophila</em> subgenus could indicate functional diversification of the <em>Gr39aA</em> and <em>Gr39aB</em> in these species.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Increased functional diversity results from extensive whole gene duplication in some cases, creating gene families, whereas in other cases it evolves through alternative splicing. Curiously, although the results of these two processes are similar, they are inversely correlated at the genomic level and it is unclear what conditions lead to one rather than the other <a href="#pone.0001513-Talavera1">[20]</a>. There are several hypotheses proposed to explain the functional diversification of duplicated genes which presumably also apply to exon duplication. According to Ohno's hypothesis, most gene duplicates, being functionally redundant, are eliminated from the genome (nonfunctionalization), except for rare occasions when beneficial mutations could lead to a new gene function (neofunctionalization) <a href="#pone.0001513-Ohno1">[21]</a>. Another model predicts dividing the functions between the duplicate and the original copy (subfunctionalization) <a href="#pone.0001513-Lynch1">[22]</a>. Both models assume that relaxation or a lack of selection on the duplicate allows the acquisition of novel replacement mutations and/or changes in expression pattern. According to Hughes <a href="#pone.0001513-Hughes1">[23]</a>, “specialization” of duplicates to different functions of the bi-functional ancestral gene can occur through positive selection. This model assumes a bi-functional nature of the ancestral gene, though in the case of already extensive gene families, different loci will already have similar functions but a degree of specialization (such as detecting related ligands). It has been shown that many chemoreceptors can recognize more than one ligand <a href="#pone.0001513-Hallem1">[24]</a>, suggesting their bi- or multi-functional nature. Duplication of such genes can facilitate the specialization of daughter genes and changes in expression, for instance in more restricted sets of tissues, can further drive specialization <a href="#pone.0001513-Hughes1">[23]</a>. For tandemly duplicated exons, Kondrashov and Koonin <a href="#pone.0001513-Kondrashov1">[25]</a> favour the “specialization” model of the duplicated exons, assuming that if both duplicated exons are translated immediately after duplication, both will be subject to stabilizing selection, which excludes the possibility of short-term relaxation or lack of selection required to allow accumulation of replacement mutations. Alternatively, adaptation of alleles to different functions or sub-functions might start even before duplication <a href="#pone.0001513-Proulx1">[26]</a>, thereby facilitating neofunctionalisation after duplication.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Despite our expectations that large alternatively spliced exons of the <em>Gr39a</em> would experience similar selective constraints and stabilizing selection, we found evidence of relaxation in the selective constraints on exon <em>A</em> (which is prone to duplication) and exon <em>E</em>. Exon <em>E</em> is either present as a single copy or lost in some lineages, while exon <em>A</em> has multiply duplicated in most species. We found evidence of functional divergence between the copies of <em>A</em> amongst the species of two subgenera, and we suggest that the exon divergence probably occurred through relaxation of selective constrains on this exon, which implies subfunctionalization of the duplicates according to Lynch and Force <a href="#pone.0001513-Lynch1">[22]</a> or neofunctionalization according to Ohno's hypothesis <a href="#pone.0001513-Ohno1">[21]</a>. The selective constraints that act on other large exons (<em>B, C</em> and <em>D</em>) are relatively constant. We detected signals of positive selection on the <em>Gr39aB</em> in the <em>Drosophila</em> subgenus and possibly on <em>Gr39aC</em> in some species of the melanogaster group. Exon <em>B</em> is present as a single copy in all Drosophila species examined; the signature of positive selection which we detected on this exon indicates an adaptive mode of its evolution. The ancient duplicates <em>Gr39aC</em> and <em>Gr39aD</em> have functionally diverged, but experience similar selective constrains. We suggest the participation of positive selection in their divergence. We found some evidence of an increase of the nonsynonymous substitution rate in several species of the melanogaster group with the strongest signal in the specialist <em>D. sechellia</em>. This observation might support Hughes's model of “specialization” of duplicates <a href="#pone.0001513-Hughes1">[23]</a> or the idea that positive selection can lead to a completely new function (neofunctionalization).</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Annotation of the orthologs of <em>D. melanogaster Gr39a</em> genes was performed for the other 11 Drosophila species whose genome sequences are publicly available <a href="#pone.0001513-Drosophila1">[10]</a> using a combination of Blast <a href="#pone.0001513-Altschul1">[27]</a>, GeneWise <a href="#pone.0001513-Birney1">[28]</a> and manual curation <a href="#pone.0001513-Gardiner1">[7]</a>. Annotations (<a href="#pone.0001513.s001">Table S1</a>, <a href="#pone.0001513.s003">Figure S1</a>) and proteins alignment (<a href="#pone.0001513.s004">Figure S2</a>) are available as Supplemental data. <a href="#pone.0001513.s003">Figure S1</a> contains sequences of genes from the start codon to stop including introns, and also contains coding sequences (open reading frames, ORFs) in which introns are represented as gaps. The resulting sets of <em>Gr39a</em> sequences were multiply aligned at the codon level using ClustalW <a href="#pone.0001513-Thompson1">[29]</a> on translations, followed by Protal2dna (K. Schuerer, C. Letondal; <a href="http://bioweb.pasteur.fr">http://bioweb.pasteur.fr</a>). Coding sequences identified were tested for the presence of transmembrane domains in their product using the TMHMM2.0 program <a href="#pone.0001513-Krogh1">[30]</a>. For comparison of the <em>Gr39a</em> locus with other Drosophila gustatory (<em>Gr23a, Gr28b</em>, <em>Gr39a)</em> and olfactory (<em>Or46a, Or69a</em>) receptor genes that are known to undergo alternative splicing, we also identified their orthologs in the same set of species following the same procedure.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>The phylogeny of <em>Gr39a</em> nucleotide sequences, with <em>Gr68aDmel</em> and <em>Gr32aDmel</em> as outgroups, was reconstructed by Bayesian Markov chain Monte Carlo (MCMC) analysis using MrBayes-3.1.2 <a href="#pone.0001513-Ronquist1">[31]</a> with the HKY+4Γ model; default priors on branch lengths, rate parameters and tree topology; and two runs, each with one chain of 30,000,000 generations sampled every 50,000 generations. The first 250 trees sampled in each run were discarded as burn-in, leaving a final MCMC sample of 702 trees. Convergence was assessed using the average standard deviation of split frequencies, output by MrBayes. Independence within the sample was assessed using autocorrelation in tree log-likelihoods of the 351 trees from each run <a href="#pone.0001513-Pagel1">[32]</a>, obtained with the ACF function of Minitab v14.20 (Minitab, inc.). The majority-rule consensus of the final MCMC sample was taken to be the phylogeny of <em>Gr39a</em>.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>We used Notung 2.1 <a href="#pone.0001513-Durand1">[14]</a> to reconcile the “gene” tree (in our case, exon tree) and the species tree. Notung maps duplication and loss events onto branches of the species tree by reconstructing ancestral states according to parsimony rules. The topology of the species tree (<a href="#pone-0001513-g001">Figure 1A</a>) was obtained from <a href="#pone.0001513-Drosophila1">[10]</a>.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>The structural and potentially functional divergence of the large exons of <em>Gr39a</em> was explored using DIVERGE v1.04 <a href="#pone.0001513-Gu1">[15]</a>. The approach applied in DIVERGE was developed for the analysis of the functional divergence of duplicated genes which, from an evolutionary perspective, we assume also applies to duplicated exons. This approach was also suggested to be useful for studying functional divergence after speciation events, domain shuffling, lateral gene transfer etc. <a href="#pone.0001513-Gu1">[15]</a>. Briefly, after gene (or exon) duplication, the evolutionary rate (<em>λ</em>) at an amino acid site may increase, leading to a functional divergence in the early stage after duplication, followed later by purifying constraints acting to maintain the novel function(s) <a href="#pone.0001513-Gu1">[15]</a>. If the evolutionary rates between the original (<em>λ<sub>1</sub></em>) and duplicate (<em>λ<sub>2</sub></em>) stay the same or change proportionally over time, the coefficient of rate correlation (<em>r<sub>λ</sub></em>) between them will be 1. A decrease of the <em>r<sub>λ</sub></em> indicates differences in the evolutionary rates between the original and the duplicate copy, and a measure of such divergence is assigned as <em>θ = 1−r<sub>λ</sub></em>, where <em>θ</em> is a coefficient of functional divergence <a href="#pone.0001513-Gu1">[15]</a>. <em>θ</em> = 0 indicates no functional divergence, and an increase in <em>θ</em> from 0 to 1 shows increasing functional divergence from weak to extremely strong <a href="#pone.0001513-Gu1">[15]</a>. The significance of <em>θ</em> is assessed using the likelihood ratio statistic, <em>LR</em>, defined as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001513.e001&amp;representation=PNG" class="inline-graphic"></span> where <em>H</em><sub>0</sub> is the likelihood of the model representing the null hypothesis (here, a model in which <em>θ</em> is constrained to equal zero) and <em>H</em><sub>1</sub> is the likelihood of a more general model, representing the alternative hypothesis (<em>θ</em> is allowed to vary). <em>LR</em> was converted to a P-value on the assumption that the null distribution of <em>LR</em> is χ<sup>2</sup> with degrees of freedom (d.f.) equal to the difference in the number of free parameters between <em>H</em><sub>1</sub> and <em>H</em><sub>0</sub> (here, d.f. = 1) <a href="#pone.0001513-Gu1">[15]</a>, <a href="#pone.0001513-Cox1">[33]</a>. We estimated <em>θ</em> between the encoded protein isoforms Gr39aA, Gr39aB, Gr39aC, Gr39aD and Gr39aE, excluding the last transmembrane domain specified by three conserved exons. Because exon <em>A</em> duplicated in most lineages, we performed several comparisons of the Gr39aA isoforms. Initially, we compared the Gr39aA isoforms with Gr39aB, Gr39aC, Gr39aD and Gr39aE, by including only single copies of <em>A</em> from each species that were considered to be conserved orthologues to the <em>A</em> of <em>D. melanogaster</em> on the basis of the highest similarity score calculated by GeneWise, then repeated this test by excluding <em>D. grimshawi</em> whose copies of A scored similar values in GeneWise. We then repeated the analysis including duplicates of Gr39aA of one of the species where exon A duplicated (<em>D. pseudoobscura</em>, <em>D. persimilis</em>, <em>D. willistoni, D. mojavensis</em>, <em>D. virilis</em> and <em>D. grimshawi</em>) and compared these with Gr39aB. We also compared the Gr39aA of the <em>Drosophila</em> subgenus with the Gr39aA isoforms of the <em>Sophophora</em> subgenus.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The “M0” model of <em>codeml</em> in the PAML computer package <a href="#pone.0001513-Yang1">[34]</a> was used to determine the average selective constraint on the whole gene, and also separately for its exons (<em>B, C, D, E</em> and the set of exons <em>A</em>, which comprised from the copies of <em>A</em> that were considered to be conserved orthologues to the <em>A</em> of <em>D. melanogaster</em>), through estimation of the ratio of the normalized nonsynonymous substitution rate (<em>d<sub>N</sub></em>) to normalized synonymous substitution rate (<em>d<sub>S</sub></em>), or <em>ω = d<sub>N</sub>/d<sub>S</sub></em>. <em>ω</em>&gt;1 is considered strong evidence of positive selection for amino acid replacements whereas <em>ω</em>≈0 indicates purifying selection, while <em>ω</em> = 1 conforms to a neutral expectation <a href="#pone.0001513-Yang2">[35]</a>. To test for positive selection on orthologous exons, we used a range of tests in which two models are compared, again by means of <em>LR</em>. The models permit detecting episodes of positive selection acting on a fraction of sites (“site” models) or on particular phylogenetic lineages (“branch” models), as well as a combination of both (“branch-site” models). The strength of evidence for site, branch and branch-site models was assessed by comparing <em>codeml</em> models “M8” with “M7”, “Mfree” with “M0”, and “M2” with “M1”, respectively. Model M7 allows several site classes with <em>ω</em> drawn from a <em>β</em> distribution and fixed between 0 and 1, while M8 adds another class of sites with <em>ω</em>&gt;1 <a href="#pone.0001513-Yang1">[34]</a>. Branch models allow testing variation in <em>ω</em> among different branches of a phylogeny <a href="#pone.0001513-Yang1">[34]</a>. The simplest model, M0, assumes one <em>ω</em> for all branches, while the “free-ratio” or Mfree model allows different <em>ω</em> for all branches <a href="#pone.0001513-Yang1">[34]</a>. We used M0 and Mfree to estimate <em>ω</em> for the exons <em>B, C</em> and <em>D</em> in the melanogaster group (we excluded exon <em>A</em> because it had degraded in two species of the melanogaster group). Finally, branch-site models can detect episodic events of adaptive evolution on specific sites in different branches of a phylogeny <a href="#pone.0001513-Yang1">[34]</a>. In the M1 model, there are four classes of sites with ω fixed below 1, while M2 allows a fraction of sites to have <em>ω</em>&gt;1 on user-selected (“foreground”) branches <a href="#pone.0001513-Yang1">[34]</a>. For converting <em>LR</em> to a P-value, for M8 vs M7, d.f. = 2; for Mfree vs M0 in this study, d.f. = 8; for M1 vs M0, d.f. = 1. Finally, the pairwise comparison of the <em>d<sub>S</sub></em> and <em>d<sub>N</sub></em> rates was performed using <em>codeml</em> option runmode = −2 <a href="#pone.0001513-Yang1">[34]</a>.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001513"></div><a name="pone.0001513.s001" id="pone.0001513.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001513.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Annotation and description of the identified Gr39a genes</p>
<p class="siDoi">doi:10.1371/journal.pone.0001513.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001513.s002" id="pone.0001513.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001513.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">The estimates of the functional divergence amongst Drosophila gustatory receptors</p>
<p class="siDoi">doi:10.1371/journal.pone.0001513.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001513.s003" id="pone.0001513.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001513.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">The nucleotide sequences of the identified Gr39a genes</p>
<p class="siDoi">doi:10.1371/journal.pone.0001513.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.23 MB TXT)</p>
<a name="pone.0001513.s004" id="pone.0001513.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001513.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">The Gr39a multiple protein sequence alignment</p>
<p class="siDoi">doi:10.1371/journal.pone.0001513.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(10.06 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The genomic sequences of <em>D. erecta, D. ananassae, D. mojavensis, D. virilis</em> and D. <em>grimshawi</em> were provided by Agencourt Bioscience Corporation. The genomes of <em>D. simulans</em> and <em>D. yakuba</em> were sequenced by Washington University (St. Louis), while <em>D. sechellia</em> and <em>D. persimilis</em> by the Broad Institute. Baylor University provided the data for the <em>D. pseudoobscura</em> genome. <em>D. melanogaster</em> genome was provided by the Berkeley Drosophila Genome Project and Celera. The authors are indebted to Christoph Echtermeyer for assistance with graphics and an anonymous reviewer for useful comments.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RB WJ MR. Performed the experiments: AG. Analyzed the data: DB AG. Wrote the paper: RB DB AG WJ MR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001513-Robertson1" id="pone.0001513-Robertson1"></a>Robertson HM, Warr CG, Carlson JR (2003) Molecular evolution of the insect chemoreceptor gene superfamily in <em>Drosophila melanogaster</em>. Proc Natl Acad Sci U S A  100: 14537–14542.  <ul class="find" data-citedArticleID="1048306" data-doi="10.1073/pnas.2335847100"><li><a href="http://dx.doi.org/10.1073/pnas.2335847100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolution+of+the+insect+chemoreceptor+gene+superfamily+in+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolution+of+the+insect+chemoreceptor+gene+superfamily+in+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001513-Nozawa1" id="pone.0001513-Nozawa1"></a>Nozawa M, Nei M (2007) Evolutionary dynamics of olfactory receptor genes in Drosophila species. Proc Natl Acad Sci U S A  104: 7122–7127.  <ul class="find" data-citedArticleID="1048298" data-doi="10.1073/pnas.0702133104"><li><a href="http://dx.doi.org/10.1073/pnas.0702133104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+dynamics+of+olfactory+receptor+genes+in+Drosophila+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+dynamics+of+olfactory+receptor+genes+in+Drosophila+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001513-Guo1" id="pone.0001513-Guo1"></a>Guo S, Kim J (2007) Molecular evolution of Drosophila odorant receptor genes. Mol Biol Evol  24: 1198–1207.  <ul class="find" data-citedArticleID="1048280" data-doi="10.1093/molbev/msm038"><li><a href="http://dx.doi.org/10.1093/molbev/msm038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolution+of+Drosophila+odorant+receptor+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolution+of+Drosophila+odorant+receptor+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001513-Hill1" id="pone.0001513-Hill1"></a>Hill CA, Fox AN, Pitts RJ, Kent LB, Tan PL, et al.  (2002) G protein-coupled receptors in <em>Anopheles gambiae</em>. Science  298: 176–178.  <ul class="find" data-citedArticleID="1048284" data-doi="10.1126/science.1076196"><li><a href="http://dx.doi.org/10.1126/science.1076196" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=G+protein-coupled+receptors+in+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22G+protein-coupled+receptors+in+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001513-Robertson2" id="pone.0001513-Robertson2"></a>Robertson HM, Wanner KW (2006) The chemoreceptor superfamily in the honey bee <em>Apis mellifera</em>: expansion of the odorant, but not gustatory, receptor family. Genome Res  16: 1395–1403.  <ul class="find" data-citedArticleID="1048308" data-doi="10.1101/gr.5057506"><li><a href="http://dx.doi.org/10.1101/gr.5057506" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+chemoreceptor+superfamily+in+the+honey+bee+Apis+mellifera%3A+expansion+of+the+odorant%2C+but+not+gustatory%2C+receptor+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+chemoreceptor+superfamily+in+the+honey+bee+Apis+mellifera%3A+expansion+of+the+odorant%2C+but+not+gustatory%2C+receptor+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001513-Clyne1" id="pone.0001513-Clyne1"></a>Clyne PJ, Warr CG, Carlson JR (2000) Candidate taste receptors in Drosophila. Science  287: 1830–1834.  <ul class="find" data-citedArticleID="1048262" data-doi="10.1126/science.287.5459.1830"><li><a href="http://dx.doi.org/10.1126/science.287.5459.1830" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Candidate+taste+receptors+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Candidate+taste+receptors+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001513-Gardiner1" id="pone.0001513-Gardiner1"></a>Gardiner A, Barker D, Butlin RK, Jordan WC, Ritchie MG (2007) Drosophila Chemoreceptor Gene Evolution: Selection, Specialisation and Genome Size. Molecular Ecology, submitted.  <ul class="find" data-citedArticleID="1048272" data-doi="10.1111/j.1365-294x.2008.03713.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2008.03713.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Drosophila+Chemoreceptor+Gene+Evolution%3A+Selection%2C+Specialisation+and+Genome+Size.+Molecular+Ecology%2C+submitted." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Drosophila+Chemoreceptor+Gene+Evolution%3A+Selection%2C+Specialisation+and+Genome+Size.+Molecular+Ecology%2C+submitted.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001513-Bray1" id="pone.0001513-Bray1"></a>Bray S, Amrein H (2003) A putative Drosophila pheromone receptor expressed in male-specific taste neurons is required for efficient courtship. Neuron  39: 1019–1029.  <ul class="find" data-citedArticleID="1048260" data-doi="10.1016/s0896-6273(03)00542-7"><li><a href="http://dx.doi.org/10.1016/s0896-6273(03)00542-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+putative+Drosophila+pheromone+receptor+expressed+in+male-specific+taste+neurons+is+required+for+efficient+courtship." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+putative+Drosophila+pheromone+receptor+expressed+in+male-specific+taste+neurons+is+required+for+efficient+courtship.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001513-Amrein1" id="pone.0001513-Amrein1"></a>Amrein H (2004) Pheromone perception and behavior in Drosophila. Current Opinion in Neurobiology  14: 435–442.  <ul class="find" data-citedArticleID="1048256" data-doi="10.1016/j.conb.2004.07.008"><li><a href="http://dx.doi.org/10.1016/j.conb.2004.07.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pheromone+perception+and+behavior+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pheromone+perception+and+behavior+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001513-Drosophila1" id="pone.0001513-Drosophila1"></a><em>Drosophila</em> 12 Genomes Consortium (2007) Evolution of genes and genomes on the Drosophila phylogeny. Nature  450: 203–218.  <ul class="find" data-citedArticleID="1048266"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+of+genes+and+genomes+on+the+Drosophila+phylogeny.&amp;auth=&amp;atitle=Evolution+of+genes+and+genomes+on+the+Drosophila+phylogeny." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+genes+and+genomes+on+the+Drosophila+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+genes+and+genomes+on+the+Drosophila+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001513-Russo1" id="pone.0001513-Russo1"></a>Russo CA, Takezaki N, Nei M (1995) Molecular phylogeny and divergence times of drosophilid species. Mol Biol Evol  12: 391–404.  <ul class="find" data-citedArticleID="1048312"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+phylogeny+and+divergence+times+of+drosophilid+species.&amp;auth=&amp;atitle=Molecular+phylogeny+and+divergence+times+of+drosophilid+species." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+and+divergence+times+of+drosophilid+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+and+divergence+times+of+drosophilid+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001513-Tamura1" id="pone.0001513-Tamura1"></a>Tamura K, Subramanian S, Kumar S (2004) Temporal Patterns of Fruit Fly (Drosophila) Evolution Revealed by Mutation Clocks. Mol Biol Evol  21: 36–44.  <ul class="find" data-citedArticleID="1048316" data-doi="10.1093/molbev/msg236"><li><a href="http://dx.doi.org/10.1093/molbev/msg236" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Temporal+Patterns+of+Fruit+Fly+%28Drosophila%29+Evolution+Revealed+by+Mutation+Clocks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Temporal+Patterns+of+Fruit+Fly+%28Drosophila%29+Evolution+Revealed+by+Mutation+Clocks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001513-Fox1" id="pone.0001513-Fox1"></a>Fox AN, Pitts RJ, Robertson HM, Carlson JR, Zwiebel LJ (2001) Candidate odorant receptors from the malaria vector mosquito <em>Anopheles gambiae</em> and evidence of down-regulation in response to blood feeding. Proceedings of the National Academy of Sciences  98: 14693–14697.  <ul class="find" data-citedArticleID="1048270" data-doi="10.1073/pnas.261432998"><li><a href="http://dx.doi.org/10.1073/pnas.261432998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Candidate+odorant+receptors+from+the+malaria+vector+mosquito+Anopheles+gambiae+and+evidence+of+down-regulation+in+response+to+blood+feeding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Candidate+odorant+receptors+from+the+malaria+vector+mosquito+Anopheles+gambiae+and+evidence+of+down-regulation+in+response+to+blood+feeding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001513-Durand1" id="pone.0001513-Durand1"></a>Durand D, Halldorsson BH, Vernot B (2005) A Hybrid Micro-Macroevolutionary Approach to Gene Tree Reconstruction. J Comput Biol  13: 320–335.  <ul class="find" data-citedArticleID="1048268" data-doi="10.1089/cmb.2006.13.320"><li><a href="http://dx.doi.org/10.1089/cmb.2006.13.320" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Hybrid+Micro-Macroevolutionary+Approach+to+Gene+Tree+Reconstruction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Hybrid+Micro-Macroevolutionary+Approach+to+Gene+Tree+Reconstruction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001513-Gu1" id="pone.0001513-Gu1"></a>Gu X (1999) Statistical methods for testing functional divergence after gene duplication. Mol Biol Evol  16: 1664–1674.  <ul class="find" data-citedArticleID="1048274" data-doi="10.1093/oxfordjournals.molbev.a026080"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026080" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+methods+for+testing+functional+divergence+after+gene+duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+methods+for+testing+functional+divergence+after+gene+duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001513-Gu2" id="pone.0001513-Gu2"></a>Gu X (2001) Mathematical Modeling for Functional Divergence after Gene Duplication. Journal of Computational Biology  8: 221–234.  <ul class="find" data-citedArticleID="1048276" data-doi="10.1089/10665270152530827"><li><a href="http://dx.doi.org/10.1089/10665270152530827" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mathematical+Modeling+for+Functional+Divergence+after+Gene+Duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mathematical+Modeling+for+Functional+Divergence+after+Gene+Duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001513-Gu3" id="pone.0001513-Gu3"></a>Gu X (2003) Functional divergence in protein (family) sequence evolution. Genetica  118: 133–141.  <ul class="find" data-citedArticleID="1048278" data-doi="10.1007/978-94-010-0229-5_4"><li><a href="http://dx.doi.org/10.1007/978-94-010-0229-5_4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+divergence+in+protein+%28family%29+sequence+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+divergence+in+protein+%28family%29+sequence+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001513-Letunic1" id="pone.0001513-Letunic1"></a>Letunic I, Copley RR, Bork P (2002) Common exon duplication in animals and its role in alternative splicing. Hum Mol Genet  11: 1561–1567.  <ul class="find" data-citedArticleID="1048292" data-doi="10.1093/hmg/11.13.1561"><li><a href="http://dx.doi.org/10.1093/hmg/11.13.1561" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Common+exon+duplication+in+animals+and+its+role+in+alternative+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Common+exon+duplication+in+animals+and+its+role+in+alternative+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001513-McBride1" id="pone.0001513-McBride1"></a>McBride CS, Arguello RJ (2007) Five Drosophila Genomes Reveal Non-Neutral Evolution and the Signature of Host Specialization in the Chemoreceptor Superfamily. Genetics  177: 1395–1416.  <ul class="find" data-citedArticleID="1048296" data-doi="10.1534/genetics.107.078683"><li><a href="http://dx.doi.org/10.1534/genetics.107.078683" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Five+Drosophila+Genomes+Reveal+Non-Neutral+Evolution+and+the+Signature+of+Host+Specialization+in+the+Chemoreceptor+Superfamily." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Five+Drosophila+Genomes+Reveal+Non-Neutral+Evolution+and+the+Signature+of+Host+Specialization+in+the+Chemoreceptor+Superfamily.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001513-Talavera1" id="pone.0001513-Talavera1"></a>Talavera D, Vogel C, Orozco M, Teichmann SA, de la Cruz X (2007) The (In)dependence of Alternative Splicing and Gene Duplication. PLoS Comput Biol  3: e33.  <ul class="find" data-citedArticleID="1048314" data-doi="10.1371/journal.pcbi.0030033"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0030033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+%28In%29dependence+of+Alternative+Splicing+and+Gene+Duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+%28In%29dependence+of+Alternative+Splicing+and+Gene+Duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001513-Ohno1" id="pone.0001513-Ohno1"></a>Ohno S (1970) Evolution by gene duplication. Heidelberg: Springer-Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0001513-Lynch1" id="pone.0001513-Lynch1"></a>Lynch M, Force A (2000) The probability of duplicate gene preservation by subfunctionalization. Genetics  154: 459–473.  <ul class="find" data-citedArticleID="1048294"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+probability+of+duplicate+gene+preservation+by+subfunctionalization.&amp;auth=&amp;atitle=The+probability+of+duplicate+gene+preservation+by+subfunctionalization." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+probability+of+duplicate+gene+preservation+by+subfunctionalization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+probability+of+duplicate+gene+preservation+by+subfunctionalization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001513-Hughes1" id="pone.0001513-Hughes1"></a>Hughes AL (1994) The evolution of functionally novel proteins after gene duplication. Proc R Soc Lond B  256: 119–124.  <ul class="find" data-citedArticleID="1048286" data-doi="10.1098/rspb.1994.0058"><li><a href="http://dx.doi.org/10.1098/rspb.1994.0058" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+functionally+novel+proteins+after+gene+duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+functionally+novel+proteins+after+gene+duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001513-Hallem1" id="pone.0001513-Hallem1"></a>Hallem EA, Carlson JR (2004) The odor coding system of Drosophila. Trends in Genetics  20: 453–459.  <ul class="find" data-citedArticleID="1048282" data-doi="10.1016/j.tig.2004.06.015"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.06.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+odor+coding+system+of+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+odor+coding+system+of+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001513-Kondrashov1" id="pone.0001513-Kondrashov1"></a>Kondrashov FA, Koonin EV (2001) Origin of alternative splicing by tandem exon duplication. Hum Mol Genet  10: 2661–2669.  <ul class="find" data-citedArticleID="1048288" data-doi="10.1093/hmg/10.23.2661"><li><a href="http://dx.doi.org/10.1093/hmg/10.23.2661" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+of+alternative+splicing+by+tandem+exon+duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+of+alternative+splicing+by+tandem+exon+duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001513-Proulx1" id="pone.0001513-Proulx1"></a>Proulx SR, Phillips PC (2006) Allelic divergence precedes and promotes gene duplication. Evolution  60: 881–892.  <ul class="find" data-citedArticleID="1048304" data-doi="10.1111/j.0014-3820.2006.tb01168.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2006.tb01168.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Allelic+divergence+precedes+and+promotes+gene+duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Allelic+divergence+precedes+and+promotes+gene+duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001513-Altschul1" id="pone.0001513-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucl Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="1048254" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001513-Birney1" id="pone.0001513-Birney1"></a>Birney E, Clamp M, Durbin R (2004) GeneWise and Genomewise. Genome Res  14: 988–995.  <ul class="find" data-citedArticleID="1048258" data-doi="10.1101/gr.1865504"><li><a href="http://dx.doi.org/10.1101/gr.1865504" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneWise+and+Genomewise." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneWise+and+Genomewise.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001513-Thompson1" id="pone.0001513-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucl Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="1048318" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001513-Krogh1" id="pone.0001513-Krogh1"></a>Krogh A, Larsson B, von Heijne G, Sonnhammer ELL (2001) Predicting transmembrane protein topology with a hidden Markov model: Application to complete genomes. J Mol Biol  305: 567–580.  <ul class="find" data-citedArticleID="1048290" data-doi="10.1006/jmbi.2000.4315"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+transmembrane+protein+topology+with+a+hidden+Markov+model%3A+Application+to+complete+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+transmembrane+protein+topology+with+a+hidden+Markov+model%3A+Application+to+complete+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001513-Ronquist1" id="pone.0001513-Ronquist1"></a>Ronquist F, Huelsenbeck J (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="1048310" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001513-Pagel1" id="pone.0001513-Pagel1"></a>Pagel M, Meade A, Barker D (2004) Bayesian Estimation of Ancestral Character States on Phylogenies. System Biol  53: 673–684.  <ul class="find" data-citedArticleID="1048302"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bayesian+Estimation+of+Ancestral+Character+States+on+Phylogenies.&amp;auth=&amp;atitle=Bayesian+Estimation+of+Ancestral+Character+States+on+Phylogenies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+Estimation+of+Ancestral+Character+States+on+Phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+Estimation+of+Ancestral+Character+States+on+Phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001513-Cox1" id="pone.0001513-Cox1"></a>Cox DR (1962) Further Results on Tests of Separate Families of Hypotheses. Journal of the Royal Statistical Society B  24: 406–424.  <ul class="find" data-citedArticleID="1048264"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Further+Results+on+Tests+of+Separate+Families+of+Hypotheses.&amp;auth=&amp;atitle=Further+Results+on+Tests+of+Separate+Families+of+Hypotheses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Further+Results+on+Tests+of+Separate+Families+of+Hypotheses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Further+Results+on+Tests+of+Separate+Families+of+Hypotheses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001513-Yang1" id="pone.0001513-Yang1"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Computer Applications in the Biosciences  13: 555–556.  <ul class="find" data-citedArticleID="1048320" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001513-Yang2" id="pone.0001513-Yang2"></a>Yang Z, Bielawski JP (2000) Statistical methods for detecting molecular adaptation. Trends in Ecology and Evolution  15: 496–503.  <ul class="find" data-citedArticleID="1048322" data-doi="10.1016/s0169-5347(00)01994-7"><li><a href="http://dx.doi.org/10.1016/s0169-5347(00)01994-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+methods+for+detecting+molecular+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+methods+for+detecting+molecular+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.XML" value="86352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.PDF" value="318874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g001.PNG_L" value="821015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g001.PNG_M" value="94874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g001.PNG_S" value="10241"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g001.TIF" value="1341264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g001.PNG_I" value="34379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g002.PNG_L" value="1710693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g002.PNG_M" value="138385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g002.PNG_S" value="11020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g002.TIF" value="2735020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g002.PNG_I" value="77907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g003.PNG_L" value="261966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g003.PNG_M" value="61792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g003.PNG_S" value="6521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g003.TIF" value="444428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g003.PNG_I" value="21669"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.t001.PNG_L" value="108556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.t001.PNG_M" value="59634"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.t001.PNG_S" value="9782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.t001.TIF" value="506928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.t001.PNG_I" value="18019"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.t002.PNG_L" value="108234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.t002.PNG_M" value="67914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.t002.PNG_S" value="9411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.t002.TIF" value="487634"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.t002.PNG_I" value="20573"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g004.PNG_L" value="376311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g004.PNG_M" value="62739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g004.PNG_S" value="9015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g004.TIF" value="908460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.g004.PNG_I" value="29779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.e001.PNG" value="8304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.e001.TIF" value="24204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.s001.XLS" value="16896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.s002.XLS" value="19456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.s003.TXT" value="231069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001513.s004.TIF" value="11483600"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001513&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001513" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001513&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001513&volume=&issue=&title=Evolution of a Complex Locus: Exon Gain, Loss and Divergence at the Gr39a Locus in Drosophila&author_name=Anastasia%20Gardiner%2C%20Daniel%20Barker%2C%20Roger%20K.%20Butlin%2C%20William%20C.%20Jordan%2C%20Michael%20G.%20Ritchie&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001513" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001513" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001513" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001513&amp;t=Evolution%20of%20a%20Complex%20Locus%3A%20Exon%20Gain%2C%20Loss%20and%20Divergence%20at%20the%20Gr39a%20Locus%20in%20Drosophila" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001513&title=Evolution%20of%20a%20Complex%20Locus%3A%20Exon%20Gain%2C%20Loss%20and%20Divergence%20at%20the%20Gr39a%20Locus%20in%20Drosophila&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001513&amp;title=Evolution%20of%20a%20Complex%20Locus%3A%20Exon%20Gain%2C%20Loss%20and%20Divergence%20at%20the%20Gr39a%20Locus%20in%20Drosophila" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001513" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001513&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Evolution of a Complex Locus: Exon Gain, Loss and Divergence at the Gr39a Locus in Drosophila';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001513';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Evolution%20of%20a%20Complex%20Locus%3A%20Exon%20Gain%2C%20Loss%20and%20Divergence%20at%20the%20Gr39a%20Locus%20in%20Drosophila http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001513" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001513" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alternative+splicing%22" title="Search for articles in the subject area:'Alternative splicing'"><div class="flagText">Alternative splicing</div></a>
              <div data-categoryid="17931" data-articleid="25852"
                   data-categoryname="Alternative splicing"
                   class="flagImage" title="Flag 'Alternative splicing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Divergent+evolution%22" title="Search for articles in the subject area:'Divergent evolution'"><div class="flagText">Divergent evolution</div></a>
              <div data-categoryid="17199" data-articleid="25852"
                   data-categoryname="Divergent evolution"
                   class="flagImage" title="Flag 'Divergent evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drosophila%22" title="Search for articles in the subject area:'Drosophila'"><div class="flagText">Drosophila</div></a>
              <div data-categoryid="42801" data-articleid="25852"
                   data-categoryname="Drosophila"
                   class="flagImage" title="Flag 'Drosophila' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drosophila+melanogaster%22" title="Search for articles in the subject area:'Drosophila melanogaster'"><div class="flagText">Drosophila melanogaster</div></a>
              <div data-categoryid="32461" data-articleid="25852"
                   data-categoryname="Drosophila melanogaster"
                   class="flagImage" title="Flag 'Drosophila melanogaster' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="25852"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="25852"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="25852"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="25852"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5891'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8010'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6287&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
