

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Evolutionarily Conserved Protein Sequences of Influenza A Viruses, Avian and Human, as Vaccine Targets</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001190"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001190" />

    <meta name="citation_title" content="Evolutionarily Conserved Protein Sequences of Influenza A Viruses, Avian and Human, as Vaccine Targets"/>
    <meta itemprop="name" content="Evolutionarily Conserved Protein Sequences of Influenza A Viruses, Avian and Human, as Vaccine Targets"/>

      <meta name="citation_author" content="A. T. Heiny"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="Olivo Miotto"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore"/>
            <meta name="citation_author_institution" content="Institute of Systems Science, National University of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="Kellathur N. Srinivasan"/>
            <meta name="citation_author_institution" content="Department of Pharmacology and Molecular Sciences, The Johns Hopkins University School of Medicine, Maryland, United States of America"/>
            <meta name="citation_author_institution" content="Product Evaluation and Registration Division, Centre for Drug Administration, Health Sciences Authority, Singapore, Singapore"/>
      <meta name="citation_author" content="Asif M. Khan"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore"/>
            <meta name="citation_author_institution" content="Department of Microbiology, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="G. L. Zhang"/>
            <meta name="citation_author_institution" content="Institute for Infocomm Research, Singapore, Singapore"/>
      <meta name="citation_author" content="Vladimir Brusic"/>
            <meta name="citation_author_institution" content="Cancer Vaccine Center, Dana-Farber Cancer Institute, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Tin Wee Tan"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="J. Thomas August"/>
            <meta name="citation_author_institution" content="Department of Pharmacology and Molecular Sciences, The Johns Hopkins University School of Medicine, Maryland, United States of America"/>

    <meta name="citation_date" content="2007/11/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001190.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1190"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Influenza virus: a master of metamorphosis.; citation_author=JC De Jong; citation_author=GF Rimmelzwaan; citation_author=RA Fouchier; citation_author=AD Osterhaus; citation_journal_title=J Infect; citation_volume=40; citation_number=1; citation_pages=218-228; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Weathering the influenza vaccine crisis.; citation_author=J Treanor; citation_journal_title=N Engl J Med; citation_volume=351; citation_number=2; citation_pages=2037-2040; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Influenza pandemics of the 20th century.; citation_author=ED Kilbourne; citation_journal_title=Emerg Infect Dis; citation_volume=12; citation_number=3; citation_pages=9-14; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Large-scale sequencing of human influenza reveals the dynamic nature of viral genome evolution.; citation_author=E Ghedin; citation_author=NA Sengamalay; citation_author=M Shumway; citation_author=J Zaborsky; citation_author=T Feldblyum; citation_journal_title=Nature; citation_volume=437; citation_number=4; citation_pages=1162-1166; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Large-scale sequence analysis of avian influenza isolates.; citation_author=JC Obenauer; citation_author=J Denson; citation_author=PK Mehta; citation_author=X Su; citation_author=S Mukatira; citation_journal_title=Science; citation_volume=311; citation_number=5; citation_pages=1576-1580; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Human genetics and responses to influenza vaccination: clinical implications.; citation_author=R Lambkin; citation_author=P Novelli; citation_author=J Oxford; citation_author=C Gelder; citation_journal_title=Am J Pharmacogenomics; citation_volume=4; citation_number=6; citation_pages=293-298; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Development of effective vaccines against pandemic influenza.; citation_author=K Subbarao; citation_author=BR Murphy; citation_author=AS Fauci; citation_journal_title=Immunity; citation_volume=24; citation_number=7; citation_pages=5-9; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Priority setting for pandemic influenza: an analysis of national preparedness plans.; citation_author=L Uscher-Pines; citation_author=SB Omer; citation_author=DJ Barnett; citation_author=TA Burke; citation_author=RD Balicer; citation_journal_title=PLoS Med; citation_volume=3; citation_number=8; citation_pages=e436; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A hypothesis to relate the specificity of T lymphocytes and the activity of I region-specific Ir genes in macrophages and B lymphocytes.; citation_author=B Benacerraf; citation_journal_title=J Immunol; citation_volume=120; citation_number=9; citation_pages=1809-1812; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Cytotoxic T cells recognize fragments of the influenza nucleoprotein.; citation_author=AR Townsend; citation_author=FM Gotch; citation_author=J Davey; citation_journal_title=Cell; citation_volume=42; citation_number=10; citation_pages=457-467; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=The foreign antigen binding site and T cell recognition regions of class I histocompatibility antigens.; citation_author=PJ Bjorkman; citation_author=MA Saper; citation_author=B Samraoui; citation_author=WS Bennett; citation_author=JL Strominger; citation_journal_title=Nature; citation_volume=329; citation_number=11; citation_pages=512-518; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=MHC-dependent antigen processing and peptide presentation: providing ligands for T lymphocyte activation.; citation_author=RN Germain; citation_journal_title=Cell; citation_volume=76; citation_number=12; citation_pages=287-299; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of MHC class I-restricted antigen processing and cross-presentation.; citation_author=P Cresswell; citation_author=AL Ackerman; citation_author=A Giodini; citation_author=DR Peaper; citation_author=PA Wearsch; citation_journal_title=Immunol Rev; citation_volume=207; citation_number=13; citation_pages=145-157; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cell biology of antigen processing in vitro and in vivo.; citation_author=ES Trombetta; citation_author=I Mellman; citation_journal_title=Annu Rev Immunol; citation_volume=23; citation_number=14; citation_pages=975-1028; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Allele-specific motifs revealed by sequencing of self-peptides eluted from MHC molecules.; citation_author=K Falk; citation_author=O Rotzschke; citation_author=S Stevanovic; citation_author=G Jung; citation_author=HG Rammensee; citation_journal_title=Nature; citation_volume=351; citation_number=15; citation_pages=290-296; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The structure of the antigen-binding groove of major histocompatibility complex class I molecules determines specific selection of self-peptides.; citation_author=GM van Bleek; citation_author=SG Nathenson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=88; citation_number=16; citation_pages=11032-11036; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Structure of peptides associated with class I and class II MHC molecules.; citation_author=VH Engelhard; citation_journal_title=Annu Rev Immunol; citation_volume=12; citation_number=17; citation_pages=181-207; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=MHC ligands and peptide motifs: first listing.; citation_author=HG Rammensee; citation_author=T Friede; citation_author=S Stevanoviic; citation_journal_title=Immunogenetics; citation_volume=41; citation_number=18; citation_pages=178-228; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Promiscuous and allele-specific anchors in HLA-DR-binding peptides.; citation_author=J Hammer; citation_author=P Valsasnini; citation_author=K Tolba; citation_author=D Bolin; citation_author=J Higelin; citation_journal_title=Cell; citation_volume=74; citation_number=19; citation_pages=197-203; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Several HLA alleles share overlapping peptide specificities.; citation_author=J Sidney; citation_author=MF del Guercio; citation_author=S Southwood; citation_author=VH Engelhard; citation_author=E Appella; citation_journal_title=J Immunol; citation_volume=154; citation_number=20; citation_pages=247-259; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Nine major HLA class I supertypes account for the vast preponderance of HLA-A and -B polymorphism.; citation_author=A Sette; citation_author=J Sidney; citation_journal_title=Immunogenetics; citation_volume=50; citation_number=21; citation_pages=201-212; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Definition of supertypes for HLA molecules using clustering of specificity matrices.; citation_author=O Lund; citation_author=M Nielsen; citation_author=C Kesmir; citation_author=AG Petersen; citation_author=C Lundegaard; citation_journal_title=Immunogenetics; citation_volume=55; citation_number=22; citation_pages=797-810; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=In silico identification of supertypes for class II MHCs.; citation_author=IA Doytchinova; citation_author=DR Flower; citation_journal_title=J Immunol; citation_volume=174; citation_number=23; citation_pages=7085-7095; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Discovery of promiscuous HLA-II-restricted T cell epitopes with TEPITOPE.; citation_author=H Bian; citation_author=J Hammer; citation_journal_title=Methods; citation_volume=34; citation_number=24; citation_pages=468-475; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An integrative approach to CTL epitope prediction: a combined algorithm integrating MHC class I binding, TAP transport efficiency, and proteasomal cleavage predictions.; citation_author=MV Larsen; citation_author=C Lundegaard; citation_author=K Lamberth; citation_author=S Buus; citation_author=S Brunak; citation_journal_title=Eur J Immunol; citation_volume=35; citation_number=25; citation_pages=2295-303; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MULTIPRED: a computational system for prediction of promiscuous HLA binding peptides.; citation_author=GL Zhang; citation_author=AM Khan; citation_author=KN Srinivasan; citation_author=JT August; citation_author=V Brusic; citation_journal_title=Nucleic Acids Res 33(Web Server issue); citation_number=26; citation_pages=W172-179; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Automated generation and evaluation of specific MHC binding predictive tools: ARB matrix applications.; citation_author=HH Bui; citation_author=J Sidney; citation_author=B Peters; citation_author=M Sathiamurthy; citation_author=A Sinichi; citation_journal_title=Immunogenetics; citation_volume=57; citation_number=27; citation_pages=304-314; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Development of a DNA vaccine designed to induce cytotoxic T lymphocyte responses to multiple conserved epitopes in HIV-1.; citation_author=CC Wilson; citation_author=D McKinney; citation_author=M Anders; citation_author=S MaWhinney; citation_author=J Forster; citation_journal_title=J Immunol; citation_volume=171; citation_number=28; citation_pages=5611-5623; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Epitope-based vaccines: an update on epitope identification, vaccine design and delivery.; citation_author=A Sette; citation_author=J Fikes; citation_journal_title=Curr Opin Immunol; citation_volume=15; citation_number=29; citation_pages=461-470; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The changing field of vaccine development in the genomics era.; citation_author=V Brusic; citation_author=JT August; citation_journal_title=Pharmacogenomics; citation_volume=5; citation_number=30; citation_pages=597-600; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Polyvalent vaccines for optimal coverage of potential T-cell epitopes in global HIV-1 variants.; citation_author=W Fischer; citation_author=S Perkins; citation_author=J Theiler; citation_author=T Bhattacharya; citation_author=K Yusim; citation_journal_title=Nat Med; citation_volume=13; citation_number=31; citation_pages=100-106; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Molecularly engineered vaccine which expresses an immunodominant T-cell epitope induces cytotoxic T lymphocytes that confer protection from lethal virus infection.; citation_author=LS Klavinskis; citation_author=JL Whitton; citation_author=MB Oldstone; citation_journal_title=J Virol; citation_volume=63; citation_number=32; citation_pages=4311-4316; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Protection against lethal lymphocytic choriomeningitis virus (LCMV) infection by immunization of mice with an influenza virus containing an LCMV epitope recognized by cytotoxic T lymphocytes.; citation_author=MR Castrucci; citation_author=S Hou; citation_author=PC Doherty; citation_author=Y Kawaoka; citation_journal_title=J Virol; citation_volume=68; citation_number=33; citation_pages=3486-3490; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Protection against lymphocytic choriomeningitis virus infection induced by a reduced peptide bond analogue of the H-2Db-restricted CD8(+) T cell epitope GP33.; citation_author=C Stemmer; citation_author=A Quesnel; citation_author=A Prevost-Blondel; citation_author=C Zimmermann; citation_author=S Muller; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=34; citation_pages=5550-5556; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Recombinant Sindbis viruses expressing a cytotoxic T-lymphocyte epitope of a malaria parasite or of influenza virus elicit protection against the corresponding pathogen in mice.; citation_author=M Tsuji; citation_author=CC Bergmann; citation_author=Y Takita-Sonoda; citation_author=K Murata; citation_author=EG Rodrigues; citation_journal_title=J Virol; citation_volume=72; citation_number=35; citation_pages=6907-6910; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Induction of protective immunity against malaria by priming-boosting immunization with recombinant cold-adapted influenza and modified vaccinia Ankara viruses expressing a CD8 = -T-cell epitope derived from the circumsporozoite protein of Plasmodium yoelii.; citation_author=G Gonzalez-Aseguinolaza; citation_author=Y Nakaya; citation_author=A Molano; citation_author=E Dy; citation_author=M Esteban; citation_journal_title=J Virol; citation_volume=77; citation_number=36; citation_pages=11859-11866; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Protection against lethal cytomegalovirus infection by a recombinant vaccine containing a single nonameric T-cell epitope.; citation_author=M Del Val; citation_author=HJ Schlicht; citation_author=H Volkmer; citation_author=M Messerle; citation_author=MJ Reddehase; citation_journal_title=J Virol; citation_volume=65; citation_number=37; citation_pages=3641-3646; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Cross-protection against lymphocytic choriomeningitis virus mediated by a CD4+ T-cell clone specific for an envelope glycoprotein epitope of Lassa virus.; citation_author=VJ La Posta; citation_author=DD Auperin; citation_author=R Kamin-Lewis; citation_author=GA Cole; citation_journal_title=J Virol; citation_volume=67; citation_number=38; citation_pages=3497-3506; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Protection against lethal viral infection by vaccination with nonimmunodominant peptides.; citation_author=M Oukka; citation_author=JC Manuguerra; citation_author=N Livaditis; citation_author=S Tourdot; citation_author=N Riche; citation_journal_title=J Immunol; citation_volume=157; citation_number=39; citation_pages=3039-3045; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Immunization with a single major histocompatibility complex class I-restricted cytotoxic T-lymphocyte recognition epitope of herpes simplex virus type 2 confers protective immunity.; citation_author=JE Blaney; citation_author=E Nobusawa; citation_author=MA Brehm; citation_author=RH Bonneau; citation_author=LM Mylin; citation_journal_title=J Virol; citation_volume=72; citation_number=40; citation_pages=9567-9574; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Cytotoxic T lymphocytes raised against a subdominant epitope offered as a synthetic peptide eradicate human papillomavirus type 16-induced tumors.; citation_author=MC Feltkamp; citation_author=GR Vreugdenhil; citation_author=MP Vierboom; citation_author=E Ras; citation_author=SH van der Burg; citation_journal_title=Eur J Immunol; citation_volume=25; citation_number=41; citation_pages=2638-2642; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=CD8 =  T cells specific for a single nonamer epitope of Listeria monocytogenes are protective in vivo.; citation_author=JT Harty; citation_author=MJ Bevan; citation_journal_title=J Exp Med; citation_volume=175; citation_number=42; citation_pages=1531-1538; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The immunodominant influenza matrix T cell epitope recognized in human induces influenza protection in HLA-A2/K(b) transgenic mice.; citation_author=H Plotnicky; citation_author=D Cyblat-Chanal; citation_author=JP Aubry; citation_author=F Derouet; citation_author=C Klinguer-Hamour; citation_journal_title=Virology; citation_volume=309; citation_number=43; citation_pages=320-329; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Protection against lethal vaccinia virus challenge in HLA-A2 transgenic mice by immunization with a single CD8 =  T-cell peptide epitope of vaccinia and variola viruses.; citation_author=JT Snyder; citation_author=IM Belyakov; citation_author=A Dzutsev; citation_author=F Lemonnier; citation_author=JA Berzofsky; citation_journal_title=J Virol; citation_volume=78; citation_number=44; citation_pages=7052-7060; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=HLA-A2-restricted protection against lethal lymphocytic choriomeningitis.; citation_author=J Botten; citation_author=JL Whitton; citation_author=P Barrowman; citation_author=J Sidney; citation_author=JK Whitmire; citation_journal_title=J Virol; citation_volume=81; citation_number=45; citation_pages=2307-2317; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Clinical experience with plasmid DNA- and modified vaccinia virus Ankara-vectored human immunodeficiency virus type 1 clade A vaccine focusing on T-cell induction.; citation_author=T Hanke; citation_author=AJ McMichael; citation_author=L Dorrell; citation_journal_title=J Gen Virol; citation_volume=88; citation_number=46; citation_pages=1-12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phase 1 evaluation of 3 highly immunogenic prime-boost regimens, including a 12-month reboosting vaccination, for malaria vaccination in Gambian men.; citation_author=VS Moorthy; citation_author=EB Imoukhuede; citation_author=S Keating; citation_author=M Pinder; citation_author=D Webster; citation_journal_title=J Infect Dis; citation_volume=189; citation_number=47; citation_pages=2213-2219; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Recognition of influenza A virus by human cytotoxic T lymphocytes.; citation_author=AJ McMichael; citation_author=FM Gotch; citation_journal_title=Adv Exp Med Biol; citation_volume=257; citation_number=48; citation_pages=109-114; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Recognition of influenza virus proteins by cytotoxic T lymphocytes.; citation_author=AR Townsend; citation_journal_title=Immunol Res; citation_volume=6; citation_number=49; citation_pages=80-100; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Cytotoxic T cells in influenza infection.; citation_author=BA Askonas; citation_author=PM Taylor; citation_author=F Esquivel; citation_journal_title=Ann N Y Acad Sci; citation_volume=532; citation_number=50; citation_pages=230-237; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=T-cell recognition of influenza viral antigens.; citation_author=JR Lamb; citation_author=AJ McMichael; citation_author=JB Rothbard; citation_journal_title=Hum Immunol; citation_volume=19; citation_number=51; citation_pages=79-89; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=CD4+ T-cell memory: generation and multi-faceted roles for CD4+ T cells in protective immunity to influenza.; citation_author=SL Swain; citation_author=JN Agrewala; citation_author=DM Brown; citation_author=DM Jelley-Gibbs; citation_author=S Golech; citation_journal_title=Immunol Rev; citation_volume=211; citation_number=52; citation_pages=8-22; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cell-mediated protection in influenza infection.; citation_author=PG Thomas; citation_author=R Keating; citation_author=DJ Hulse-Post; citation_author=PC Doherty; citation_journal_title=Emerg Infect Dis; citation_volume=12; citation_number=53; citation_pages=48-54; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Heterologous protection against influenza by injection of DNA encoding a viral protein.; citation_author=JB Ulmer; citation_author=JJ Donnelly; citation_author=SE Parker; citation_author=GH Rhodes; citation_author=PL Felgner; citation_journal_title=Science; citation_volume=259; citation_number=54; citation_pages=1745-1749; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Protective CD4+ and CD8 =  T cells against influenza virus induced by vaccination with nucleoprotein DNA.; citation_author=JB Ulmer; citation_author=TM Fu; citation_author=RR Deck; citation_author=A Friedman; citation_author=L Guan; citation_journal_title=J Virol; citation_volume=72; citation_number=55; citation_pages=5648-5653; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Dose dependence of CTL precursor frequency induced by a DNA vaccine and correlation with protective immunity against influenza virus challenge.; citation_author=TM Fu; citation_author=L Guan; citation_author=A Friedman; citation_author=TL Schofield; citation_author=JB Ulmer; citation_journal_title=J Immunol; citation_volume=162; citation_number=56; citation_pages=4163-4170; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=DNA vaccine expressing conserved influenza virus proteins protective against H5N1 challenge infection in mice.; citation_author=SL Epstein; citation_author=TM Tumpey; citation_author=JA Misplon; citation_author=CY Lo; citation_author=LA Cooper; citation_journal_title=Emerg Infect Dis; citation_volume=8; citation_number=57; citation_pages=796-801; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Protection against multiple influenza A subtypes by vaccination with highly conserved nucleoprotein.; citation_author=SL Epstein; citation_author=WP Kong; citation_author=JA Misplon; citation_author=CY Lo; citation_author=TM Tumpey; citation_journal_title=Vaccine; citation_volume=23; citation_number=58; citation_pages=5404-5410; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Induction of cytotoxic T-cell responses by gene gun DNA vaccination with minigenes encoding influenza A virus HA and NP CTL-epitopes.; citation_author=A Fomsgaard; citation_author=HV Nielsen; citation_author=N Kirkby; citation_author=K Bryder; citation_author=S Corbet; citation_journal_title=Vaccine; citation_volume=18; citation_number=59; citation_pages=681-691; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Primary pulmonary cytotoxic T lymphocytes induced by immunization with a vaccinia virus recombinant expressing influenza A virus nucleoprotein peptide do not protect mice against challenge.; citation_author=CM Lawson; citation_author=JR Bennink; citation_author=NP Restifo; citation_author=JW Yewdell; citation_author=BR Murphy; citation_journal_title=J Virol; citation_volume=68; citation_number=60; citation_pages=3505-3511; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Contribution of virus-specific CD8 =  cytotoxic T cells to virus clearance or pathologic manifestations of influenza virus infection in a T cell receptor transgenic mouse model.; citation_author=D Moskophidis; citation_author=D Kioussis; citation_journal_title=J Exp Med; citation_volume=188; citation_number=61; citation_pages=223-232; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Identification of protective and non-protective T cell epitopes in influenza.; citation_author=SR Crowe; citation_author=SC Miller; citation_author=DL Woodland; citation_journal_title=Vaccine; citation_volume=24; citation_number=62; citation_pages=452-456; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A systematic bioinformatics approach for selection of epitope-based vaccine targets.; citation_author=AM Khan; citation_author=O Miotto; citation_author=AT Heiny; citation_author=J Salmon; citation_author=KN Srinivasan; citation_journal_title=Cell Immunol; citation_volume=244; citation_number=63; citation_pages=141-147; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Rule-based Knowledge Aggregation for Large-Scale Protein Sequence Analysis of Influenza A Viruses.; citation_author=O Miotto; citation_author=TW Tan; citation_author=V Brusic; citation_journal_title=BMC Bioinformatics; citation_volume=8; citation_number=64; citation_pages=S7; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Orthomyxoviruses: influenza; citation_author=NJ Cox; citation_author=G Neumann; citation_author=RO Donis; citation_author=Y Kawaoka; citation_number=65; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: a multiple sequence alignment method with reduced time and space complexity.; citation_author=RC Edgar; citation_journal_title=BMC Bioinformatics; citation_volume=5; citation_number=66; citation_pages=113; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Characterization of a novel influenza A virus hemagglutinin subtype (H16) obtained from black-headed gulls.; citation_author=RA Fouchier; citation_author=V Munster; citation_author=A Wallensten; citation_author=TM Bestebroer; citation_author=S Herfst; citation_journal_title=J Virol; citation_volume=79; citation_number=67; citation_pages=2814-2822; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A mathematical theory of communication.; citation_author=CE Shannon; citation_journal_title=Bell System Technical Journal; citation_volume=27; citation_number=68; citation_pages=379-423 and 623–656; citation_date=1948; " />
      <meta name="citation_reference" content="citation_title=Chemistry of peptides associated with MHC class I and class II molecules.; citation_author=HG Rammensee; citation_journal_title=Curr Opin Immunol; citation_volume=7; citation_number=69; citation_pages=85-96; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Estimation of entropy and mutual information.; citation_author=L Paninski; citation_journal_title=Neural Computation; citation_volume=15; citation_number=70; citation_pages=1191-1253; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The immune epitope database and analysis resource: from vision to blueprint.; citation_author=B Peters; citation_author=J Sidney; citation_author=P Bourne; citation_author=HH Bui; citation_author=S Buus; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=71; citation_pages=e91; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A community resource benchmarking predictions of peptide binding to MHC-I molecules.; citation_author=B Peters; citation_author=HH Bui; citation_author=S Frankild; citation_author=M Nielson; citation_author=C Lundegaard; citation_journal_title=PLoS Comput Biol; citation_volume=2; citation_number=72; citation_pages=e65; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A novel influenza A virus mitochondrial protein that induces cell death.; citation_author=W Chen; citation_author=PA Calvo; citation_author=D Malide; citation_author=J Gibbs; citation_author=U Schubert; citation_journal_title=Nat Med.; citation_volume=7(12); citation_number=73; citation_pages=1306-1312; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The PB1-F2 protein of Influenza A virus: increasing pathogenicity by disrupting alveolar macrophages.; citation_author=JR Coleman; citation_journal_title=Virol J.; citation_volume=4; citation_number=74; citation_pages=9; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Identification of human-to-human transmissibility factors in PB2 proteins of influenza A by large-scale mutual information analysis.; citation_author=O Miotto; citation_author=AT Heiny; citation_author=TW Tan; citation_author=JT August; citation_author=V Brusic; citation_journal_title=BMC Bioinformatics,; citation_volume=8; citation_number=75; citation_pages=S18; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Computational analysis and identification of amino acid sites in dengue E proteins relevant to development of diagnostics and vaccines.; citation_author=R Mazumder; citation_author=ZZ Hu; citation_author=CR Vinayaka; citation_author=JL Sagripanti; citation_author=SD Frost; citation_journal_title=Virus Genes.; citation_volume=35; citation_number=76; citation_pages=175-186; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Coping with viral diversity in HIV vaccine design.; citation_author=DC Nickle; citation_author=M Rolland; citation_author=MA Jensen; citation_author=SL Pond; citation_author=W Deng; citation_journal_title=PLoS Comput Biol.; citation_volume=3; citation_number=77; citation_pages=e751; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Polyvalent vaccines for optimal coverage of potential T-cell epitopes in global HIV-1 variants.; citation_author=W Fischer; citation_author=S Perkins; citation_author=J Theiler; citation_author=T Bhattacharya; citation_author=K Yusim; citation_journal_title=Nat. Med.; citation_volume=13; citation_number=78; citation_pages=100-106; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=HIV vaccine development by computer assisted design: the GAIA vaccine.; citation_author=AS De Groot; citation_author=L Marcon; citation_author=EA Bishop; citation_author=D Rivera; citation_author=M Kutzler; citation_journal_title=Vaccine.; citation_volume=23; citation_number=79; citation_pages=2136-2148; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Development of a DNA vaccine designed to induce cytotoxic T lymphocyte responses to multiple conserved epitopes in HIV-1.; citation_author=CC Wilson; citation_author=D McKinney; citation_author=M Anders; citation_author=S MaWhinney; citation_author=J Forster; citation_journal_title=J Immunol; citation_volume=171; citation_number=80; citation_pages=5611-5623; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Optimization and immune recognition of multiple novel conserved HLA-A2, human immunodeficiency virus type 1-specific CTL epitopes.; citation_author=S Corbet; citation_author=HV Nielsen; citation_author=L Vinner; citation_author=S Lauemoller; citation_author=D Therrien; citation_journal_title=The Journal of general virology; citation_volume=84; citation_number=81; citation_pages=2409-2421; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Development of a DNA vaccine designed to induce cytotoxic T lymphocyte responses to multiple conserved epitopes in HIV-1.; citation_author=CC Wilson; citation_author=D McKinney; citation_author=M Anders; citation_author=S MaWhinney; citation_author=J Forster; citation_journal_title=Journal of immunology (Baltimore, Md : 1950); citation_volume=171; citation_number=82; citation_pages=5611-5623; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cytolytic T lymphocytes (CTLs) from HIV-1 subtype C-infected Indian patients recognize CTL epitopes from a conserved immunodominant region of HIV-1 Gag and Nef.; citation_author=MR Thakar; citation_author=LS Bhonge; citation_author=SK Lakhashe; citation_author=U Shankarkumar; citation_author=SS Sane; citation_journal_title=The Journal of infectious diseases; citation_volume=192; citation_number=83; citation_pages=749-759; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Clustering patterns of cytotoxic T-lymphocyte epitopes in human immunodeficiency virus type 1 (HIV-1) proteins reveal imprints of immune evasion on HIV-1 global variation.; citation_author=K Yusim; citation_author=C Kesmir; citation_author=B Gaschen; citation_author=MM Addo; citation_author=M Altfeld; citation_journal_title=Journal of virology; citation_volume=76; citation_number=84; citation_pages=8757-8768; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Selection, transmission, and reversion of an antigen-processing cytotoxic T-lymphocyte escape mutation in human immunodeficiency virus type 1 infection.; citation_author=TM Allen; citation_author=M Altfeld; citation_author=XG Yu; citation_author=KM O'Sullivan; citation_author=M Lichterfeld; citation_journal_title=Journal of virology; citation_volume=78; citation_number=85; citation_pages=7069-7078; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Impaired processing and presentation of cytotoxic-T-lymphocyte (CTL) epitopes are major escape mechanisms from CTL immune pressure in human immunodeficiency virus type 1 infection.; citation_author=Y Yokomaku; citation_author=H Miura; citation_author=H Tomiyama; citation_author=A Kawana-Tachikawa; citation_author=M Takiguchi; citation_journal_title=Journal of virology; citation_volume=78; citation_number=86; citation_pages=1324-1332; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ecological and immunological determinants of influenza evolution.; citation_author=NM Ferguson; citation_author=AP Galvani; citation_author=RM Bush; citation_journal_title=Nature; citation_volume=422; citation_number=87; citation_pages=428-433; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Naive CTLs require a single brief period of antigenic stimulation for clonal expansion and differentiation.; citation_author=MJ van Stipdonk; citation_author=EE Lemmens; citation_author=SP Schoenberger; citation_journal_title=Nat Immunol; citation_volume=2; citation_number=88; citation_pages=423-429; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=CD4+ T cells are required for the maintenance, not programming, of memory CD8 =  T cells after acute infection.; citation_author=JC Sun; citation_author=MA Williams; citation_author=MJ Bevan; citation_journal_title=Nature immunology; citation_volume=5; citation_number=89; citation_pages=927-933; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Duration of the initial TCR stimulus controls the magnitude but not functionality of the CD8 =  T cell response.; citation_author=M Prlic; citation_author=G Hernandez-Hoyos; citation_author=MJ Bevan; citation_journal_title=The Journal of experimental medicine; citation_volume=203; citation_number=90; citation_pages=2135-2143; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Defective CD8 T cell memory following acute infection without CD4 T cell help.; citation_author=JC Sun; citation_author=MJ Bevan; citation_journal_title=Science (New York, NY); citation_volume=300; citation_number=91; citation_pages=339-342; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=CD4+ T cells are required for secondary expansion and memory in CD8 =  T lymphocytes.; citation_author=EM Janssen; citation_author=EE Lemmens; citation_author=T Wolfe; citation_author=U Christen; citation_author=MG von Herrath; citation_journal_title=Nature; citation_volume=421; citation_number=92; citation_pages=852-856; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Requirement for CD4 T cell help in generating functional CD8 T cell memory.; citation_author=DJ Shedlock; citation_author=H Shen; citation_journal_title=Science (New York, NY); citation_volume=300; citation_number=93; citation_pages=337-339; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001190" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Evolutionarily Conserved Protein Sequences of Influenza A Viruses, Avian and Human, as Vaccine Targets"/>
    <meta name="twitter:description" content="BackgroundInfluenza A viruses generate an extreme genetic diversity through point mutation and gene segment exchange, resulting in many new strains that emerge from the animal reservoirs, among which was the recent highly pathogenic H5N1 virus. This genetic diversity also endows these viruses with a dynamic adaptability to their habitats, one result being the rapid selection of genomic variants that resist the immune responses of infected hosts. With the possibility of an influenza A pandemic, a critical need is a vaccine that will recognize and protect against any influenza A pathogen. One feasible approach is a vaccine containing conserved immunogenic protein sequences that represent the genotypic diversity of all current and future avian and human influenza viruses as an alternative to current vaccines that address only the known circulating virus strains.Methodology/Principal FindingsMethodologies for large-scale analysis of the evolutionary variability of the influenza A virus proteins recorded in public databases were developed and used to elucidate the amino acid sequence diversity and conservation of 36,343 sequences of the 11 viral proteins of the recorded virus isolates of the past 30 years. Technologies were also applied to identify the conserved amino acid sequences from isolates of the past decade, and to evaluate the predicted human lymphocyte antigen (HLA) supertype-restricted class I and II T-cell epitopes of the conserved sequences. Fifty-five (55) sequences of 9 or more amino acids of the polymerases (PB2, PB1, and PA), nucleoprotein (NP), and matrix 1 (M1) proteins were completely conserved in at least 80%, many in 95 to 100%, of the avian and human influenza A virus isolates despite the marked evolutionary variability of the viruses. Almost all (50) of these conserved sequences contained putative supertype HLA class I or class II epitopes as predicted by 4 peptide-HLA binding algorithms. Additionally, data of the Immune Epitope Database (IEDB) include 29 experimentally identified HLA class I and II T-cell epitopes present in 14 of the conserved sequences.Conclusions/SignificanceThis study of all reported influenza A virus protein sequences, avian and human, has identified 55 highly conserved sequences, most of which are predicted to have immune relevance as T-cell epitopes. This is a necessary first step in the design and analysis of a polyepitope, pan-influenza A vaccine. In addition to the application described herein, these technologies can be applied to other pathogens and to other therapeutic modalities designed to attack DNA, RNA, or protein sequences critical to pathogen function."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001190.g008"/>

  <meta property="og:title" content="Evolutionarily Conserved Protein Sequences of Influenza A Viruses, Avian and Human, as Vaccine Targets" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001190" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8133'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1582'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5367&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001190">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001190" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001190&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Evolutionarily Conserved Protein Sequences of Influenza A Viruses, Avian and Human, as Vaccine Targets
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            A. T. Heiny, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Olivo Miotto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Biochemistry, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore, 
                  Institute of Systems Science, National University of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kellathur N. Srinivasan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Pharmacology and Molecular Sciences, The Johns Hopkins University School of Medicine, Maryland, United States of America, 
                  Product Evaluation and Registration Division, Centre for Drug Administration, Health Sciences Authority, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Asif M. Khan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Biochemistry, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore, 
                  Department of Microbiology, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            G. L. Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Infocomm Research, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vladimir Brusic, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Cancer Vaccine Center, Dana-Farber Cancer Institute, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tin Wee Tan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            J. Thomas August
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:taugust@bs.jhmi.edu">taugust@bs.jhmi.edu</a></p>

                <p>Affiliation:
                  Department of Pharmacology and Molecular Sciences, The Johns Hopkins University School of Medicine, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001190</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001190" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001190" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001190" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001190" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001190-g001" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001190-t001" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001190-t002" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001190-g002" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001190-g003" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001190-g004" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001190-g005" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001190-g006" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001190-t003" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001190-g007" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001190-g008" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001190">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001190" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Influenza A viruses generate an extreme genetic diversity through point mutation and gene segment exchange, resulting in many new strains that emerge from the animal reservoirs, among which was the recent highly pathogenic H5N1 virus. This genetic diversity also endows these viruses with a dynamic adaptability to their habitats, one result being the rapid selection of genomic variants that resist the immune responses of infected hosts. With the possibility of an influenza A pandemic, a critical need is a vaccine that will recognize and protect against any influenza A pathogen. One feasible approach is a vaccine containing conserved immunogenic protein sequences that represent the genotypic diversity of all current and future avian and human influenza viruses as an alternative to current vaccines that address only the known circulating virus strains.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Methodologies for large-scale analysis of the evolutionary variability of the influenza A virus proteins recorded in public databases were developed and used to elucidate the amino acid sequence diversity and conservation of 36,343 sequences of the 11 viral proteins of the recorded virus isolates of the past 30 years. Technologies were also applied to identify the conserved amino acid sequences from isolates of the past decade, and to evaluate the predicted human lymphocyte antigen (HLA) supertype-restricted class I and II T-cell epitopes of the conserved sequences. Fifty-five (55) sequences of 9 or more amino acids of the polymerases (PB2, PB1, and PA), nucleoprotein (NP), and matrix 1 (M1) proteins were completely conserved in at least 80%, many in 95 to 100%, of the avian and human influenza A virus isolates despite the marked evolutionary variability of the viruses. Almost all (50) of these conserved sequences contained putative supertype HLA class I or class II epitopes as predicted by 4 peptide-HLA binding algorithms. Additionally, data of the Immune Epitope Database (IEDB) include 29 experimentally identified HLA class I and II T-cell epitopes present in 14 of the conserved sequences.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This study of all reported influenza A virus protein sequences, avian and human, has identified 55 highly conserved sequences, most of which are predicted to have immune relevance as T-cell epitopes. This is a necessary first step in the design and analysis of a polyepitope, pan-influenza A vaccine. In addition to the application described herein, these technologies can be applied to other pathogens and to other therapeutic modalities designed to attack DNA, RNA, or protein sequences critical to pathogen function.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Heiny AT, Miotto O, Srinivasan KN, Khan AM, Zhang GL, et al.  (2007) Evolutionarily Conserved Protein Sequences of Influenza A Viruses, Avian and Human, as Vaccine Targets. PLoS ONE 2(11):
          e1190.
            doi:10.1371/journal.pone.0001190</p><p><strong>Academic Editor: </strong>Berend Snel, Utrecht University, Netherlands</p><p><strong>Received:</strong> September 5, 2007; <strong>Accepted:</strong> October 17, 2007; <strong>Published:</strong> November 21, 2007</p><p><strong>Copyright:</strong> © 2007 Heiny et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The development of the computational tools reported herein was supported by in part with Federal funds from the National Institute of Allergy and Infectious Diseases, National Institutes of Health, Department of Health and Human Services, USA, under Grant No. 5 U19 AI56541 and Contract No. HHSN2662-00400085C.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>One of the most important threats to human health is infection by avian influenza A viruses <a href="#pone.0001190-DeJong1">[1]</a>–<a href="#pone.0001190-Kilbourne1">[3]</a>. While global influenza pandemics have occurred only a few times in the past century, the H1N1 pandemic of 1918–1919 caused 20–50 million deaths and was one of the most serious disease outbreaks in recorded history. The recent evolution of the highly lethal avian H5N1 virus, while not transmissible in humans, has emphasized the continued threat of influenza viruses on a global scale. It is widely predicted, given the increased human population and density, that a new pandemic on the scale of the H1N1 infection would have a devastating effect world-wide.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The two currently approved vaccines against influenza viruses are designed specifically to mimic the most recently recognized circulating forms listed in the 2006–2007 influenza prevention and control recommendations (<a href="http://www.cdc.gov/mmwr/preview/mmwrhtml/rr5510a1.htm">http://www.cdc.gov/mmwr/preview/mmwrhtml​/rr5510a1.htm</a>). Both vaccines contain three recently isolated human strains and are subject to possible annual revision of their virus composition. The rapid mutation of the viral HA and NA proteins facilitates the selective replication of new virus strains not subject to immunity based on previous vaccination and is a serious obstacle to the effectiveness of these vaccines <a href="#pone.0001190-Ghedin1">[4]</a>–<a href="#pone.0001190-Obenauer1">[5]</a>. Alternative vaccine strategies that overcome the problem of rapid viral mutation, can be applied to global populations, and provide for easy production are suggested goals <a href="#pone.0001190-Lambkin1">[6]</a>–<a href="#pone.0001190-UscherPines1">[8]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The design of a vaccine that guarantees antibody-mediated immunity to new influenza viruses is not currently feasible because the structural determinants of B-cell immunity are highly complex and there is no effective means for predicting the antibody epitope structure of target pathogens. Cell-mediated immunity, in contrast, is based upon the binding of short sequences of antigen proteins, termed T-cell epitopes, to specialized cellular proteins, known as human leukocyte antigens (HLAs), class I (HLA I) and class II (HLA II), that facilitate the presentation of the epitopes to T-cells of the immune system <a href="#pone.0001190-Benacerraf1">[9]</a>–<a href="#pone.0001190-Trombetta1">[14]</a>. The chemical and structural determinants of HLA-peptide binding have been defined for a number of HLA alleles <a href="#pone.0001190-Falk1">[15]</a>–<a href="#pone.0001190-Hammer1">[19]</a>. Of particular relevance for vaccine design are supertype groupings of similar HLA alleles that display overlapping peptide-binding capacities. The supertypes cover a large fraction of the HLA diversity in the human population and antigen epitopes that bind to the supertypes are considered prime candidates for vaccine formulations <a href="#pone.0001190-Sidney1">[20]</a>–<a href="#pone.0001190-Bian1">[24]</a>. Supertype-binding motifs and quantitative matrices have been incorporated into several computational prediction algorithms and it is now possible to identify, <em>in silico,</em> candidate HLA-restricted T-cell epitopes of protein sequences, allowing large-scale analysis of potential vaccine targets <a href="#pone.0001190-Bian1">[24]</a>–<a href="#pone.0001190-Bui1">[27]</a>. Moreover, increasing attention is being given to T-cell-based vaccines because they can be designed as genetic formulations to include selected regions of the viral antigens <a href="#pone.0001190-Wilson1">[28]</a>–<a href="#pone.0001190-Fischer1">[31]</a>, and have the many other desirable properties associated with DNA vaccines in general. Studies have demonstrated that epitope-specific T cell responses elicited by immunization with DNA or peptide, and adoptive transfer of epitope-specific T cell clones, could mediate protective immunity, in some cases with single CTL epitopes, against various pathogens in murine experimental models <a href="#pone.0001190-Klavinskis1">[32]</a>–<a href="#pone.0001190-Harty1">[42]</a>. Additionally, recent studies have shown that immunization of HLA-A2 transgenic mice against single HLA-A2-restricted T-cell epitopes conferred protection against lethal infection with influenza A virus, vaccinia virus, or LCMV <a href="#pone.0001190-Plotnicky1">[43]</a>–<a href="#pone.0001190-Botten1">[45]</a>. Human clinical trials with epitope-based DNA vaccines against HIV <a href="#pone.0001190-Hanke1">[46]</a> and malaria <a href="#pone.0001190-Moorthy1">[47]</a> were found to be safe and immunogenic for effector T-cell immune responses but in these first generation studies, failed to achieve the desired clinical goals in the vaccination of healthy volunteers.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Cellular immune responses are recognized to play a role in influenza immunity (for reviews see <a href="#pone.0001190-McMichael1">[48]</a>–<a href="#pone.0001190-Lamb1">[51]</a> and the application of T-cell epitopes has been extensively studied as an alternative to vaccines designed for humoral immunity <a href="#pone.0001190-Swain1">[52]</a>–<a href="#pone.0001190-Crowe1">[62]</a>. Mouse immunization with DNA encoding NP elicited CTL, IFN-γ and IL-2 responses, with cross-strain protection against virus challenge, and evidence from adoptive transfer, indicated that both types of T cells act as effectors in protective immunity <a href="#pone.0001190-Ulmer1">[54]</a>–<a href="#pone.0001190-Ulmer2">[55]</a>. Similarly, DNA immunization with H1N1 NP or H5N1 NP or M proteins was found to protect mice against lethal challenge <a href="#pone.0001190-Fu1">[56]</a>–<a href="#pone.0001190-Epstein2">[58]</a>. However, some influenza vaccine formulations were not successful <a href="#pone.0001190-Fomsgaard1">[59]</a>–<a href="#pone.0001190-Crowe1">[62]</a> and there remain multiple issues for the development of a human T-cell epitope-based vaccines, including epitope selection, delivery systems, epitope processing and presentation, and undoubtedly others.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>This study was focused on the large-scale analysis of all influenza A virus protein sequence data of the past 30 years that is recorded in public databases. Information entropy and consensus sequence methodologies were combined to identify sequences of 9 amino acids or longer with a history of complete conservation in 80% or more of both avian and human virus strains. These conserved sequences were further analyzed to identify targets for candidate epitope-based T-cell vaccine formulations against all current and possibly future influenza A pathogens of avian or human origin.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Methodology overview</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A general overview of the methodology is depicted in <a href="#pone-0001190-g001">Figure 1</a>. The details and rationale for the systematic approach adopted by this study have been previously published <a href="#pone.0001190-Khan1">[63]</a>. The primary goal was to identify viral protein sequences that have been conserved over long periods of time, and to select those sequences that have the highest potential for HLA-restricted immunogenicity in a broad spectrum of the human population. The process includes three major steps: i) extraction of all influenza A protein sequence data, processing and definition of data sets comprising relevant human and avian virus groups; ii) entropy analysis of sequence variability and identification of conserved peptide sequences of 9 amino acids or longer; and iii) prediction of supertype-restricted, HLA-binding sequences. Two complementary methods for the identification of conserved sequences were applied: a statistical entropy-based method that takes into account the combinatorial diversity of peptide epitopes and was used to elucidate the variability for different influenza subtypes, and a consensus method, which is robust against sampling biases (such as the predominance of certain influenza subtypes in the dataset), to confirm the conserved sequences. The conserved sequences were then submitted to several epitope prediction programs, whose results are combined. Sequences predicted, and in some cases demonstrated, to contain epitopes to several HLA supertypes are proposed as vaccine epitope candidates because of their wide human population coverage.</p>
<div class="figure" id="pone-0001190-g001"><div class="img"><a name="pone-0001190-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>An overview of the methodology of this study.</span></strong></p><span>doi:10.1371/journal.pone.0001190.g001</span></div>

<h4>Influenza A virus sequence data collection and processing</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A dataset of influenza A protein sequences annotated with isolate name, country and year of isolation, host organism, subtype, and protein name, was derived from all available sequences (as of September 2006) from the NCBI GenBank and GenPept databases, including entries mirrored from the UniProt database. Collection and cleaning of a total of 85,873 records was performed the Aggregator of Biological Knowledge (ABK) <a href="#pone.0001190-Miotto1">[64]</a>, which applied structural and semantic rules to automate the aggregation and annotation task. The final set of 36,343 protein sequences was manually verified by two independent curators. Most human influenza subtypes were represented by more than 100 sequences of each viral protein, the count varying depending on the protein. The H1N2 subtype had a lower number of sequences (ranging from 22 to 66) because of its recent emergence <a href="#pone.0001190-Cox1">[65]</a>. Separate multiple sequence alignments of the 11 proteins were carried out with MUSCLE 3.6 <a href="#pone.0001190-Edgar1">[66]</a>. Because of the great variability exhibited by the HA and NA proteins, separate alignments were obtained for each subtype (16 subtypes for HA and 9 for NA). The subtype alignments were subsequently merged using the MUSCLE tool, to obtain the final HA and NA alignments. The introduction of gaps in the resulting alignments was minimized by merging sequences based on sequence similarity between subtypes, as reported in phylogenetic studies <a href="#pone.0001190-Obenauer1">[5]</a>, <a href="#pone.0001190-Fouchier1">[67]</a>. The in-house developed Antigenic Variability Analyser tool (AVANA) was subsequently used to extract alignments of several subsets of the collected sequences, based on annotation values, such as viral subtype, host, and year of isolation.</p>


<h4>Information entropy analysis</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The diversity of the influenza A virus proteome was studied by creating subsets of the influenza A protein sequence alignments, comprising (1) avian sequences, subdivided into 3 decades (1977–1986, 1987–1996, and 1997–2006); (2) H5N1 viruses, subdivided into avian and human isolates; (3) circulating human subtypes, namely H1N1, H3N2, and H1N2. Assuming that each sequence represents an independent isolate, the information entropy methodology <a href="#pone.0001190-Shannon1">[68]</a> was used to measure the variability of influenza A virus proteomes in the context of overlapping nine-amino acid peptides spanning the length of each influenza A protein. The rationale of this selection was the length of peptides that are bound by HLA molecules for presentation to T-cell receptors, typically from 8–20 amino acids, with nine amino acids being the predominant length of class I peptides and the core of class II peptides <a href="#pone.0001190-Rammensee2">[69]</a>. Applying Shannon's formula <a href="#pone.0001190-Shannon1">[68]</a>, the nonamer peptide entropy <em>H</em>(<em>x)</em> at any given position <em>x</em> in the alignment is computed by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <em>p</em>(<em>i</em>,<em>x</em>) is the probability of a particular nonamer variant <em>i</em> being centered at position <em>x</em>. The entropy value increases with <em>n</em>(<em>x</em>), the total number of variants observed at position <em>x</em>; it is also sensitive to the relative frequency of the variants, such that it decreases when one variant is clearly dominant (<em>i.e.</em> the position is conserved). Only sequences that contain a valid amino acid at position <em>x</em> were used for the entropy computation, and alignment gaps were ignored. Although gaps tend to occur in high-diversity regions, proteins that have a high fraction of gaps have reduced statistical support, yielding an artificially low entropy value; for this reason, positions where more than 50% of sequences contained a gap were discarded. Because of the statistical nature of the entropy measure, both complete protein sequences and shorter fragments were used in this computation.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In theory, nonamer entropy values can range from 0, a completely conserved nonamer sequence in all proteins analyzed, to 2<sup>9</sup>; in practice, however, the upper bound is very much lower for alignments of closely related sequences. For finite-size sets of sequences, entropy computations are affected by the sequence count in the alignment. The effects of alignment size bias are especially noticeable for alignments containing fewer than about 100 sequences, and must be accounted for when making direct comparisons between sequence alignments of different sizes. It has been shown that, for an alignment of <em>N</em> sequences, alignment size bias is proportional to 1/<em>N</em> <a href="#pone.0001190-Paninski1">[70]</a>. This relationship allows a correction for size bias by applying to each alignment a statistical adjustment that estimates entropy values for an infinitely-sized alignment with analogous variant distribution. To obtain such estimate, the alignment was repeatedly randomly sampled to create smaller alignments of varying size, whose entropy was measured. At each alignment position, the entropy of these subset alignments of size <em>N</em> was plotted against 1/<em>N,</em> using a linear regression to extrapolate the entropy estimate for <em>N</em>→∞. The regression's coefficient of determination (<em>r</em><sup>2</sup>) was used as a goodness-of-fit of the resulting estimate, confirming the validity of our method (<em>r</em><sup>2</sup>&gt;0.9 in most cases). In this study, size bias correction was applied to all entropy calculations, so that alignment sequence counts could be ignored in comparisons. All entropy values reported are therefore infinite-size set estimates, rather than the values directly computed from the alignments.</p>


<h4>Conserved influenza A virus sequences</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Collected and cleaned influenza A virus records were grouped based on (a) subtype: the circulating human subtypes (H1N1, H3N2, H1N2), H5N1, and other subtypes in avian reservoir; (b) host: human and avian; and (c) year of isolation. The method gave equal weight to all groups and obviated the problem of particular groups being over-represented (such as human H3N2). Six subgroups were derived: (1) human H1N1, (2) human H3N2, (3) human H1N2, (4) human H5N1, (5) avian H5N1, and (6) other avian subtypes. The eleven influenza A proteins of each subgroups were individually aligned using MUSCLE 3.6 <a href="#pone.0001190-Edgar1">[66]</a>. The AVANA tool was used to select nonamers with conservation of ≥80% in each alignment. The minimum length of a conserved sequence was nine amino acids and conserved contiguous nonamers were joined as a single sequence. A consensus sequence (the most frequent sequence) for each conserved sequence in the alignments was generated for each of the 6 subgroups. Corresponding consensus sequences of the subgroups were then aligned and those sequences that were identical in each of the six subgroups and present in at least 80% of all recorded viruses were selected as the highly conserved sequences.</p>


<h4>HLA supertype-restricted T-cell epitopes</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The <em>in silico</em> prediction of HLA supertype-restricted HLA class I and class II T-cell epitope sequences in the conserved region<em>s</em> was performed through four computational systems: NetCTL MULTIPRED, ARB, and TEPITOPE. The NetCTL 1.2 algorithm <a href="#pone.0001190-Larsen1">[25]</a> (<a href="http://www.cbs.dtu.dk/services/NetCTL/">http://www.cbs.dtu.dk/services/NetCTL/</a>) predicts peptides restricted to 12 HLA class I supertypes (A1, A2, A3, A24, A26, B7, B8, B27, B39, B44, B58 and B62), integrated with predictions of HLA binding, proteasomal C-terminal cleavage and transport efficiency by the transporter associated with antigen processing (TAP) molecules. HLA binding and proteasomal cleavage predictions are performed by an artificial neural networks (ANN) method and TAP transport efficiency is predicted using a weight matrix method. The parameters used for NetCTL prediction were: 0.15 weight on C terminal cleavage (default), 0.05 weight on TAP transport efficiency (default), and 0.5 threshold for HLA supertype binding.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The Average Relative Binding (ARB) matrix binding prediction method (<a href="http://epitope.liai.org:8080/matrix/matrix_prediction.jsp">http://epitope.liai.org:8080/matrix/matr​ix_prediction.jsp</a>) <a href="#pone.0001190-Bui1">[27]</a> is allele specific and estimates a matrix of coefficients based upon the association of each of the 20 amino acids at each possible position along the peptide sequence. In this study the data were selected for representative alleles within studied supertypes and predictions are shown for 8 HLA class I alleles of the supertypes A1 (A*0101), A2 (A*0201), A3 (A*0301), A24 (A*2402), A26 (A*2601), B7 (B*0702), and B44 (B*4402, B*4403).</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>MULTIPRED (<a href="http://research.i2r.a-star.edu.sg/multipred/">http://research.i2r.a-star.edu.sg/multip​red/</a>) <a href="#pone.0001190-Zhang1">[26]</a> predicts peptides that bind to HLA class I supertypes A2 (A*0201, *0202, *0203, *0204, *0205, *0206, *0207 and *0209) and A3 (A*0301, *0302, *1101, *1102, *3101, *3301 and *6801)and class II HLA-DR supertype (DRB1*0101, *0401, *1501, *0701, *0901, *1302 and DRB5*0101). Hidden Markov model (HMM) and ANN methods are the predictive engines with sum thresholds of: A2, 31.33 (ANN; SN = 0.80 and SP = 0.83) and 47.08 (HMM; SN = 0.80 and SP = 0.78); A3, 24.53 (ANN; SN = 0.90 and SP = 0.95) and 37.58 (HMM; SN = 0.80 and SP = 0.87); and DR, 23.42 (ANN; SN = 0.90 and SP = 0.92) and 51.08 (HMM; SN = 0.90 and SP = 1.00). TEPITOPE predicts 25 HLA class II (DR) alleles are HLA allele-specific; however, sequences predicted to bind to ≥5 alleles were considered supertypic.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>The TEPITOPE software <a href="#pone.0001190-Bian1">[24]</a> (2000 beta version; obtained by the courtesy of J. Hammer) utilizes quantitative matrix-based motifs, obtained from experimental scanning of the binding of P1-anchored designer peptides to soluble HLA-DR molecules in <em>in-vitro</em> competition assays, to predict peptides binding to 25 common HLA-DR alleles (DRB1*0101, *0102, *0301, *0401, *0402, *0404, *0405, *0410, *0421, *0701, *0801, *0802, *0804, *0806, *1101, *1104, *1106, *1107, *1305, *1307, *1311, *1321, *1501, *1502 and DRB5*0101). The parameters for TEPITOPE predictions were: 5% quantitative threshold and putative determinants with a 10-fold inhibitory residue excluded. Predictions were performed for all 25 HLA-DR alleles and nonamer core peptides predicted to bind &gt;5 HLA-DR alleles were selected as supertype-restricted.</p>


<h4>Experimentally identified influenza A T-cell epitopes</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>T-cell epitope sequences within the conserved sequences were identified by matching the highly conserved sequences and the curated influenza epitope sequences obtained from the Immune Epitope Database and Analysis Resource (<a href="http://www.immuneepitope.org/">www.immuneepitope.org/</a>) <a href="#pone.0001190-Peters1">[71]</a>, <a href="#pone.0001190-Peters2">[72]</a>. These epitope sequences data were derived from reported HLA binding assays (IC50≤500 nM) or T-cell assays that included <sup>51</sup>Cr release, HLA tetramer staining, and ELISPOT assays. Only epitope data from unique sequences and containing HLA restriction information were included.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Avian and human influenza A virus isolates</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The collected and cleaned influenza A virus protein sequences were catalogued in two groups. The recently circulating (1997–2006) influenza A viruses, both avian and human comprising 25,812 sequences of the 11 influenza proteins, both full- and partial-length, from human H1N1 (2,466 sequences), H3N2 (12,199), H1N2 (405), H5N1 (1,055), avian H5N1 (4,361), and all other avian subtypes except H5N1 (5,326) (<a href="#pone-0001190-t001">Table 1</a>). There were over 100 sequences of each protein of every virus with exceptions for the most recent human viruses, H1N2 and H5N1, and the PB1-F2 protein. The second group comprised an additional 10,531 sequences of human H1N1 and H3N2 isolated prior to 1997, and all avian viruses isolated from 1977 to 1986, and 1987 to 1996 (<a href="#pone-0001190-t002">Table 2</a>). The H1N2 sequences before 1997 were excluded because the number of sequences that were available for analysis was insufficient.</p>
<div class="figure" id="pone-0001190-t001"><div class="img"><a name="pone-0001190-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001190.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001190.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Influenza types A virus protein sequences from the past decade (1997–2006).</span></strong></p><span>doi:10.1371/journal.pone.0001190.t001</span></div><div class="figure" id="pone-0001190-t002"><div class="img"><a name="pone-0001190-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001190.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001190.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Influenza A virus protein sequences from virus isolates before 1997.</span></strong></p><span>doi:10.1371/journal.pone.0001190.t002</span></div>

<h4>Diversity of influenza A virus proteins</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The diversity in the protein sequences of influenza A viruses was examined by application of the information entropy methodology to each 9 amino acid sequence of the viral proteins. Data of the past 30 years comprised 9,640 avian influenza A subtype sequences (1977–1986, 2,543 sequences; 1987–1996, 1,711; 1997–2006, 5,326 (<a href="#pone-0001190-t001">Table 1</a> and <a href="#pone-0001190-t002">2</a>, <a href="#pone-0001190-g002">Figure 2</a>). The gross patterns of protein variability of the avian viruses from each of the past 3 decades were very similar in the context of the relative diversity of the proteins. The viral surface glycoproteins, HA and NA, showed extreme sequence diversity, illustrative of the reassortment of the genome segments among the many subtypes of the avian group A viruses as well as the rapid rate of point mutation, with multiple amino acids at virtually every position (entropy &gt;2.0) except at a single region in HA that has remained remarkably conserved despite the extreme sequence modification of every other nonamer of the protein. The PB1-F2, NS1 and NS2, and to a lesser extent M2, also showed a history of high variability. In contrast, the polymerase proteins (PB2, PB1, and PA), as well as the NP and M1, contained many historically highly conserved regions (entropy &lt;1.0). The overall gradual increase in entropy over the three decades in many of the protein sequences, most apparent in the highly conserved sequences, is an indication of the continuing genetic evolution of the viruses as well as improved screening of sequence variants. However, these changes do not distort the overall pattern of highly conserved and highly variable sequences.</p>
<div class="figure" id="pone-0001190-g002"><div class="img"><a name="pone-0001190-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Entropy plots of avian influenza A viruses, excluding H5N1 subtype, for each of three decades: 1977–1986, 1987–1996, 1997–2006 (data as of September 30, 2006).</span></strong></p><span>doi:10.1371/journal.pone.0001190.g002</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>H5N1 protein entropy patterns of the 1997 to 2006 isolates from humans (1,055 sequences) and birds (4,361) were grossly similar and reflect the high mutational variability in amino acid composition of proteins of even a single subtype influenza A virus (<a href="#pone-0001190-g003">Figure 3</a>). Of the eleven known influenza A proteins, only the short PB1-F2 protein, the product of an alternative ORF of the PB1 RNA segment <a href="#pone.0001190-Chen1">[73]</a> showed notable differences when comparing the diversity profiles of the two groups. There is evidence that PB1-F2 is involved in the apoptosis of host immune cells, increased viral virulence in a mouse model, and destruction of alveolar macrophages <a href="#pone.0001190-Coleman1">[74]</a>, and the limited diversity of this protein in human isolates may have relevance to H5N1 virulence and pathogenicity. In the context of the remaining proteins, the similarity of human and avian H5N1 entropy patterns is consistent with the observations that, to date, all human H5N1 isolates represent avian to human transmission from isolated clusters of avian infection. Moreover, detailed analyses of mutations associated with human-to-human transmission have shown that all human H5N1 virus isolates have a predominant avian footprint <a href="#pone.0001190-Miotto2">[75]</a></p>
<div class="figure" id="pone-0001190-g003"><div class="img"><a name="pone-0001190-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Entropy plots of the sequence alignments of recorded H5N1 viruses isolated from avian and human hosts (data as of September 30, 2006).</span></strong></p><span>doi:10.1371/journal.pone.0001190.g003</span></div><a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Entropy of the protein sequences of each of the three circulating human viruses isolated between 1918 and 2006 (H1N1, 3,850 sequences; H3N2, 17,092; and H1N2, 414 [including 9 sequences before 1997]) reflect different patterns of sequence evolution (<a href="#pone-0001190-g004">Figure 4</a>). The complex protein sequence diversity pattern of the human H1N1 reflects its mutational evolution from its avian characteristics at the time of human transmission in 1918 to a sequence characteristic of human H1N1, with the greatest evolutionary diversity in the HA, NA, PB1-F2, NS1 and NS2, and to a lesser extent M2, similar to the diversity of the viruses in the avian host. There subsequently was further evolution of the human subtypes by gene segment exchange, resulting in H2N2 in 1957, H3N2 in 1968, and H1N2 in 1988. The continuing mutational modification of H1N2 and H3N2 have resulted in entropy patterns distinctive of the human transmitted influenza A viruses with a large number of amino acid sequence patterns that differ from those of the avian to avian counterpart. In contrast, the most recent H1N2 human subtype that appeared in 1988 (<a href="http://www.cdc.gov/flu/about/h1n2.htm">www.cdc.gov/flu/about/h1n2.htm</a>) continues to exhibit limited evolutionary variability with many identical or highly conserved sequences regions in all of the few (22 to 66) recorded individual protein sequences (see <a href="#pone-0001190-t001">Table 1</a>). It is likely that the human H1N2 virus evolved from a very limited, perhaps single reassortment of the HA gene segment in the case of an individual infected with both of the human transmitted H1N1 and H3N2 viruses.</p>
<div class="figure" id="pone-0001190-g004"><div class="img"><a name="pone-0001190-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Entropy plots of recorded human influenza A subtypes H1N1, H3N2, and H1N2 from 1918–2006 (data as of September 30, 2006).</span></strong></p><span>doi:10.1371/journal.pone.0001190.g004</span></div><a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>The nature of the entropy distribution of the conserved sequences is not demonstrated in these data as entropy is not a linear function but is defined both by the number of sites and frequency of variability. A given entropy value can be related to a high fraction of different amino acids at one site and limited variability at other amino acid sites, or to limited variability at a large number of amino acid sites. This absence of a direct correlation of entropy to the degree of sequence conservation is seen in the markedly diverse nonamer entropy values (~0.7 to 1.5) of the collected sequences with 80% conservation (<a href="#pone-0001190-g005">Figure 5</a>). A more limited range of entropy values can be associated with sequence conservation of 90–100%.</p>
<div class="figure" id="pone-0001190-g005"><div class="img"><a name="pone-0001190-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Entropy-sequence conservation relationship, plotted from data in this study (see <a href="#pone-0001190-g002">Figure 2</a>–<a href="#pone-0001190-g003"></a><a href="#pone-0001190-g004">4</a>).</span></strong></p><a id="article1.body1.sec3.sec2.fig4.caption1.p1" name="article1.body1.sec3.sec2.fig4.caption1.p1"></a><p>The boxed region indicates area whereby conservation of ≥90% correlates to entropy of 0.8 or less.</p>
<span>doi:10.1371/journal.pone.0001190.g005</span></div><a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>We concluded that the PB2, PB1, PA, NP, and M1 proteins of all recorded influenza A viruses, both avian and human, contain sequences of low variability and high conservation despite differences in evolutionary pathway, subtypes, and host species. These sequences with a history and predicted future of low variability are prime targets for epitope-based T-cell vaccine formulations.</p>


<h4>Amino acid composition of the highly conserved sequences</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>A total of 55 peptide sequences, ranging from 9 to 58 amino acids in length, and containing a total of 965 amino acids, ~21% of the total proteome (<a href="#pone-0001190-t003">Table 3</a>), were completely conserved in 80% to100% of the human and avian type A viruses recorded in the past decade (<a href="#pone-0001190-g006">Figure 6</a>, <a href="#pone.0001190.s001">Table S1</a>). Twenty-six (26) were present in 90% to100% of the viruses. The majority of the conserved sequences were in the nonstructural (NS) proteins. PB2 was the most conserved with 23 sequences, comprising 50% of the protein, conserved in 80% to 100% of the documented viruses (<a href="#pone-0001190-t003">Table 3</a>). PB1 was also highly conserved (11 sequences, 36%) and the PA, NP, and M1 proteins contained significant fractions (16% to 27%) of conserved sequences. HA contained one sequence, FGAIAGFIE, that was conserved in all type A viruses despite the extreme variability of all other HA amino acids (see <a href="#pone-0001190-g002">Figure 2</a>). There were no sequences in the PB1-F2, NA, M2, NS1 or NS2 proteins that were completely conserved in at least 80% of the viruses.</p>
<div class="figure" id="pone-0001190-g006"><div class="img"><a name="pone-0001190-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Highly conserved sequences of influenza A viruses in human H1N1, H3N2, H1N2, H5N1, avian H5N1, and other avian subtypes circulating between 1997 and 2006.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>A region in the viral proteome is considered as highly conserved when it has identical sequence conservation of at least 9 contiguous amino acids in 80% or more of the protein sequences of the analyzed dataset. The index of virus colored symbol is as shown at the top of the figure.</p>
<span>doi:10.1371/journal.pone.0001190.g006</span></div><div class="figure" id="pone-0001190-t003"><div class="img"><a name="pone-0001190-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001190.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001190.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>The influenza A virus proteins, their length, the number of conserved sequences, and the combined length of the conserved sequences of each protein</span></strong></p><span>doi:10.1371/journal.pone.0001190.t003</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>The H1N1, H3N2, and H1N2 viruses circulating in humans had the highest representation of conserved sequences, with almost all of the 55 sequences present in 95% to 100% of the isolates of each virus. All but one (22 of 23) of the H1N2 PB2 conserved sequences were identical in each of the virus isolates. By comparison, only 62% to 76% of the conserved sequences of the avian and human H5N1 subgroups, respectively, and only 33% of the conserved sequences of all other avian subtypes were found in 95–100% of the isolates. The greater proportion of conserved sequences in the human isolates can be attributed to the more recent history and limited rate of evolution the influenza viruses transmitted by humans. This is especially true of the human H1N2 virus, the most recent human influenza A virus.</p>


<h4>HLA-restricted T-cell epitopes</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The association of conserved sequences and T-cell epitopes was examined by (a) <em>in silico</em> prediction of HLA-restricted binding sequences corresponding to supertype alleles by TEPITOPE <a href="#pone.0001190-Bian1">[24]</a>, NetCTL <a href="#pone.0001190-Larsen1">[25]</a>, MULTIPRED <a href="#pone.0001190-Zhang1">[26]</a> and ARB <a href="#pone.0001190-Bui1">[27]</a> algorithms; and (b) reported experimental HLA-binding and T-cell assay data. Most of the peptides representing the conserved sequences (50 of 55) were predicted to contain class I and/or class II binding sequences (<a href="#pone-0001190-g007">Figure 7</a>). There was no significant difference in the density of predicted epitopes in the conserved as compared to non-conserved sequences (data not shown). The detailed listing of nonamer sequences of the conserved regions and the predicted supertypes of these specific nonamers in shown as a supplement (<a href="#pone.0001190.s002">Table S2</a>). For example over 500 HLA class I and over 100 class II HLA binding sequences of supertype alleles were predicted, with many of the nonamer sequences predicted to bind to multiple (2 to 9) individual class I alleles. Similarly, all of the DR binding predictions were selected as supertypes on the basis of predicted binding to multiple DR-alleles (individual predictions not shown). The consistency of class I predictions by the different algorithms ranged from 31% to 66% in those supertypes (A1, A2, A3, A24, A26, B7, B44) where more than one computational system was available. The highest consistency of binding sequences cross-predicted by more than one system was observed with A2 (57%), A3 (66%), and DR (56%).</p>
<div class="figure" id="pone-0001190-g007"><div class="img"><a name="pone-0001190-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Highly conserved sequences of influenza A viruses and their predicted HLA class I and II supertype-restricted T-cell epitopes by NetCTL, ARB, TEPITOPE, and MULTIPRED systems.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>The color symbols corresponding to the prediction systems are as shown at the top of the figure. Only conserved sequences containing predicted alleles are shown. NetCTL predicts all of the listed class I supertypes; MULTIPRED predictions cover A2 and A3; and ARB predicts each of the class I except B8, B27, B39, B58, and B62. Predictions of HLA class II supertypes by MULTIPRED AND TEPITOPE is described in Materials and <a href="#s2">Methods</a>.</p>
<span>doi:10.1371/journal.pone.0001190.g007</span></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Fourteen (14) of the 55 conserved regions contained a total of 29 reported T-cell epitopes based on T-cell assay and/or HLA-binding data entered into the Immune Epitope Database and Analysis Resource (<a href="http://www.immuneepitope.org/">www.immuneepitope.org/</a>) (<a href="#pone-0001190-g008">Figure 8</a>). These 14 experimentally derived sequences included all of the predicted HLA supertypes of the M1 protein, and 5 of the 11 predicted PB1 supertypes. The majority, 22 of the 29 reported T-cell epitopes, were present as clusters (hotspots) of 2 or more overlapping or closely associated reported epitopes; for example, PB1 518-575 contains 5 epitope sequences (9–10 amino acids) between position 537 and 574. Some of the sequences were promiscuous in their association with multiple supertype alleles, for example, the PA 29-54 sequence containing the nonamer FMYSDFHFI that was experimentally shown to bind to at least 5 class I supertype alleles (A*0201, A*0203, A*0206, A*0202, and A*6802).</p>
<div class="figure" id="pone-0001190-g008"><div class="img"><a name="pone-0001190-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001190" data-uri="info:doi/10.1371/journal.pone.0001190.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001190.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001190.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001190.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001190.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Highly conserved sequences of influenza A viruses and their associated HLA-restricted T-cell epitope based on data obtained from IEDB (<a href="http://www.immuneepitope.org/">www.immuneepitope.org/</a>).</span></strong></p><a id="article1.body1.sec3.sec4.fig2.caption1.p1" name="article1.body1.sec3.sec4.fig2.caption1.p1"></a><p>Only sequences with identified sites are included. The first amino acid of each identified allele is shown in bold.</p>
<span>doi:10.1371/journal.pone.0001190.g008</span></div><a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>All but one of these 29 unique influenza A HLA epitopes reported in the IEDB and located in the conserved sequences are class I. This HLA distribution differs markedly from the corresponding total IEDB reported influenza A epitopes representing the complete viral proteome, which show a much greater representation, almost 50%, of class II epitopes: 225 class I and 95 class II. Because the conserved sequences represent ~21% of the total proteome, if there were a random distribution of T-cell epitopes in the viral proteins, one could expect about 45 class I and 20 class II epitopes in the conserved sequences, as compared to the observed 28:1. These data are consistent with the conventional model that T-cell epitopes derived from the PB2, PB1, PA, NP, and M1 nonstructural proteins that contain the conserved sequences would be processed primarily in the cytoplasmic proteosomal class I pathway.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The marked variability of influenza A virus surface proteins, the major targets of the neutralizing antibodies, have posed a serious obstacle in the development of effective and long-lasting influenza vaccines. As a possible solution, we have identified virus protein sequences that are completely conserved in the majority of all recorded genomic variants that have evolve from avian reservoirs, both avian and human. The information entropy methodology for analysis of protein variability was modified to examine sequences of 9 amino acids or longer, instead of the more common application to single residues, as a means to relate the conserved sequences to the immune function of HLA-restricted peptides. This use of entropy methodology for the identification of highly conserved protein sequences ushers a new experimental strategy in the development of vaccines for pathogens with high rates of mutation. The comprehensive analysis of conserved sequences may also have other applications to pathogen diagnosis or therapy. These sequences are known or can be presumed to have critical roles in viral survival and thus are choice targets for the development of antiviral agents.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Many reports, particularly with respect to the human immunodeficiency virus type 1 (HIV-1) have described a strategic advantage in the use of computational analysis and conserved sequences for vaccine design <a href="#pone.0001190-Mazumder1">[76]</a>–<a href="#pone.0001190-Thakar1">[83]</a>. Additionally, the analysis of sequence and immunology databases for the relationship between amino acid sequences and CTL epitope distributions indicate a localization of CTL epitopes in conserved regions of proteins <a href="#pone.0001190-Yusim1">[84]</a>. In contrast, the highly variable regions that lacked epitopes showed evidence of past immune escape with an enrichment of amino acids that do not serve as C-terminal anchor residues and a paucity of predicted proteasome processing sites <a href="#pone.0001190-Allen1">[85]</a>–<a href="#pone.0001190-Yokomaku1">[86]</a>. Likewise, the high genetic variability with continually evolving variants of influenza viruses favors sequence modifications at all sites that result in enhanced virus propagation or survival by adaptation to the host cell immune response. Therefore, a vaccine based upon sequences that are naturally highly conserved in all influenza A viruses may greatly restrict the range of possible mutants that could selectively overcome immune suppression. Such a vaccine would have significant strategic advantage provided the sequences have immune function capability, the design of the immunogen is compatible with the requirements for appropriate immune processing and presentation of the protein, and the epitopes have sufficient HLA-representation to cover the global distribution of HLA genotypes. It appears that these requirements can be satisfied given the large number of predicted supertype MHC binding sequences in the conserved regions of the influenza proteins, the experimental reports of T-cell epitopes of the conserved sequences, and our findings of T-cell responses by HLA transgenic mice to almost all conserved sequences of West Nile virus (unpublished data).</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>A question, however, is why influenza A differs from other pathogens that elicit immune responses to natural infection or vaccination that prevent repeated infection. It is evident that the mechanisms involved in the immune response to influenza A virus infection are in some manner more complex. A discerning report <a href="#pone.0001190-Ferguson1">[87]</a> addresses the ecological and immunological determinants of influenza evolution in relation to several of the characteristic features of influenza infection; i.e., the marked replacement of existing strains during a pandemic caused by antigenic shift, the short-lived viral sublineages that characterize influenza A infection and evolution, and the marked seasonality of influenza incidence. A proposed model <a href="#pone.0001190-Yokomaku1">[86]</a> to address these characteristic features of influenza infection and evolution was that the host immune system responds in a manner that inhibits immediate re-infection but is short-lived with a time scale of weeks to months and is nonspecific to intra- and inter-subtypes. This pattern of short-lived, cross-reactive immunity points to an initial cytotoxic T-lymphocyte (CTL) response that does not persist. We attribute this to the extreme variability of the structural proteins of influenza A viruses, especially that of the HA and NA proteins. Studies of mice and model pathogens suggest that the initial response of naive CD8<sup>+</sup> T-cells to antigen requires only a brief stimulation with antigen early in the immune response, in a matter of hours, for the cells to become activated, divide, and differentiate into short lived effector cells <a href="#pone.0001190-vanStipdonk1">[88]</a>–<a href="#pone.0001190-Prlic1">[90]</a>. This initial activation can occur in the absence of T-cell help, but without the CD4<sup>+</sup> response, the quality of the cytotoxic response to antigen challenge after priming gradually decreases and fails to respond effectively to secondary encounters with antigen. Data of several studies indicate that generation of long term CD8<sup>+</sup> T-cell immune memory requires the concurrent function of professional antigen presenting cells for class II antigen processing and presentation to CD4<sup>+</sup> helper T-cells during the initial antigen priming period <a href="#pone.0001190-Sun2">[91]</a>–<a href="#pone.0001190-Shedlock1">[93]</a>. It is likely that the major sources of T-cell epitopes, both class I and II, early after influenza infection are those proteins delivered to the immune system by the virus, including the highly variable structural proteins, HA and NA. Thus, this initial response, and the memory T-cells elicited by this response, may lack the highly conserved epitope sequences of the non-structural proteins that would be synthesized at a later stage of infection and, as cytoplasmic proteins, function primarily as endogenous class I epitopes. In this context, it is noteworthy that of the 29 reported influenza T-cell epitopes found in conserved sequences, there was only a single class II epitope, further suggesting that following natural infection, the conserved sequences elicit primarily cytotoxic T-cell responses.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>We suggest that a vaccine composed of conserved influenza A virus sequences may provide a memory immunity to non-structural proteins of all viral variants as a means for augmenting the natural response to the virus structural proteins and to provide an enhanced and augmented immune response to any newly emerging avian influenza A pathogen, as well as to the persistence of mutant forms of human transmitted influenza A. This study establishes the identity of all the highly conserved sequences of both human and avian influenza proteomes as the first step in the selection of these sequences for the synthesis of a supertype, epitope-based genetic vaccine.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001190"></div><a name="pone.0001190.s001" id="pone.0001190.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001190.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Highly conserved sequences of influenza A viruses and their occurrence in each subgroup. <sup>a</sup> Highly conserved sequences refer to sequences with ⩾80% conservation in each of the six groups that were analyzed. <sup>b</sup> The percentage conservation (rounded down as whole numbers) was calculated as the number of sequences that are identical to the highly conserved sequences divided by the total number of sequences in the same position. The numbers in square brackets indicate the total number of unique sequences at the considered position, inclusive of the highly conserved sequences. <sup>c</sup> The total number of human H1N1 sequences ranged from 187 to 242. <sup>d</sup> The total number of human H3N2 sequences ranged from 969 to 1141. <sup>e</sup> The total number of human H1N2 sequences ranged from 24 to 40. <sup>f</sup> The total number of human H5N1 sequences ranged from 82 to 106. <sup>g</sup> The total number of avian H5N1 sequences ranged from 217 to 648. <sup>h </sup>The total number of avian influenza A subtypes sequences ranged from 210 to 633.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001190.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.26 MB DOC)</p>
<a name="pone.0001190.s002" id="pone.0001190.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001190.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Potential HLA-restricted binding sequences in the highly conserved sequences of influenza A virus that are predicted by the NetCTL, ARB, TEPITOPE, and MULTIPRED systems. <sup>a</sup> Highly conserved sequences of influenza A viruses (<a href="#pone-0001190-g004">Figure 4</a>) and nonameric binding sequences predicted by NetCTL, ARB, TEPITOPE, and/or MULTIPRED algorithms. The numbers in parentheses indicate the number of nonameric binding sequences in a highly conserved sequence that was predicted by at least one algorithm. <sup>b</sup> Nonamers that bind to HLA class I were predicted using NetCTL, ARB, and MULTIPRED. NetCTL 1.2 Server predicts for T cell epitopes that bind to 12 MHC I supertypes, by integrating MHC binding, proteasomal C terminal cleavage, and TAP transport efficiency. MULTIPRED predicts for potential HLA supertype-restricted nonameric sequences that bind to two HLA class I (A2 and A3) supertypes. Only sequences that were predicted by both artificial neural network (ANN) and hidden markov model (HMM) are included. ARB predicts for T-cell epitopes that bind to 30 MHC class I alleles and 12 class II alleles. This study focused on class I alleles that are the most common in each supertype (according to Lund et al., 2004), namely class I A*0101 in A1 supertype, A*0201 in A2 supertype, A*0301 in A3 supertype, A*2402 in A24 supertype, A*2601 in A26 supertype, B*0702 in B7 supertype, B*4402 and B*4403 in B44 supertype. Only sequences, 9aa for class I that were predicted to bind to these common alleles are listed. Nonamers that were predicted to bind in any one of the three systems are listed. <sup>c</sup> Nonamers that bind to HLA class II were predicted using TEPITOPE and MULTIPRED. TEPITOPE predicts for T cell epitopes that bind to 25 MHC II alleles. Only promiscuous nonameric sequences that were predicted to bind to at least 5 alleles by TEPITOPE system were listed and indicated as “DR”. MULTIPRED predicts for potential HLA supertype-restricted nonameric sequences that bind to 8 HLA DRB1 alleles. Only sequences that were predicted by both artificial neural network (ANN) and hidden markov model (HMM) are included. Nonamers that were predicted to bind in any one of the two systems are listed.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001190.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.55 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Dr. P. Nordstrom August for help with the illustrations and valuable suggestions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: GZ JA AH OM KS AK VB TT. Performed the experiments: GZ AH. Analyzed the data: GZ JA AH OM AK VB TT. Contributed reagents/materials/analysis tools: JA. Wrote the paper: JA AH OM AK VB TT. Other: Designed the study: JA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001190-DeJong1" id="pone.0001190-DeJong1"></a>De Jong JC, Rimmelzwaan GF, Fouchier RA, Osterhaus AD (2000) Influenza virus: a master of metamorphosis. J Infect  40: 218–228.  <ul class="find" data-citedArticleID="1018952" data-doi="10.1053/jinf.2000.0652"><li><a href="http://dx.doi.org/10.1053/jinf.2000.0652" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influenza+virus%3A+a+master+of+metamorphosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influenza+virus%3A+a+master+of+metamorphosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001190-Treanor1" id="pone.0001190-Treanor1"></a>Treanor J (2004) Weathering the influenza vaccine crisis. N Engl J Med  351: 2037–2040.  <ul class="find" data-citedArticleID="1019076" data-doi="10.1056/nejmp048290"><li><a href="http://dx.doi.org/10.1056/nejmp048290" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Weathering+the+influenza+vaccine+crisis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Weathering+the+influenza+vaccine+crisis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001190-Kilbourne1" id="pone.0001190-Kilbourne1"></a>Kilbourne ED (2006) Influenza pandemics of the 20th century. Emerg Infect Dis  12: 9–14.  <ul class="find" data-citedArticleID="1018998" data-doi="10.3201/eid1201.051254"><li><a href="http://dx.doi.org/10.3201/eid1201.051254" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influenza+pandemics+of+the+20th+century." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influenza+pandemics+of+the+20th+century.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001190-Ghedin1" id="pone.0001190-Ghedin1"></a>Ghedin E, Sengamalay NA, Shumway M, Zaborsky J, Feldblyum T, et al.  (2005) Large-scale sequencing of human influenza reveals the dynamic nature of viral genome evolution. Nature  437: 1162–1166.  <ul class="find" data-citedArticleID="1018984" data-doi="10.1038/nature04239"><li><a href="http://dx.doi.org/10.1038/nature04239" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+sequencing+of+human+influenza+reveals+the+dynamic+nature+of+viral+genome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+sequencing+of+human+influenza+reveals+the+dynamic+nature+of+viral+genome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001190-Obenauer1" id="pone.0001190-Obenauer1"></a>Obenauer JC, Denson J, Mehta PK, Su X, Mukatira S, et al.  (2006) Large-scale sequence analysis of avian influenza isolates. Science  311: 1576–1580.  <ul class="find" data-citedArticleID="1019028" data-doi="10.1126/science.1121586"><li><a href="http://dx.doi.org/10.1126/science.1121586" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+sequence+analysis+of+avian+influenza+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+sequence+analysis+of+avian+influenza+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001190-Lambkin1" id="pone.0001190-Lambkin1"></a>Lambkin R, Novelli P, Oxford J, Gelder C (2004) Human genetics and responses to influenza vaccination: clinical implications. Am J Pharmacogenomics  4: 293–298.  <ul class="find" data-citedArticleID="1019006" data-doi="10.2165/00129785-200404050-00002"><li><a href="http://dx.doi.org/10.2165/00129785-200404050-00002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+genetics+and+responses+to+influenza+vaccination%3A+clinical+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+genetics+and+responses+to+influenza+vaccination%3A+clinical+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001190-Subbarao1" id="pone.0001190-Subbarao1"></a>Subbarao K, Murphy BR, Fauci AS (2006) Development of effective vaccines against pandemic influenza. Immunity  24: 5–9.  <ul class="find" data-citedArticleID="1019060" data-doi="10.1016/j.immuni.2005.12.005"><li><a href="http://dx.doi.org/10.1016/j.immuni.2005.12.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+effective+vaccines+against+pandemic+influenza." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+effective+vaccines+against+pandemic+influenza.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001190-UscherPines1" id="pone.0001190-UscherPines1"></a>Uscher-Pines L, Omer SB, Barnett DJ, Burke TA, Balicer RD (2006) Priority setting for pandemic influenza: an analysis of national preparedness plans. PLoS Med  3: e436.  <ul class="find" data-citedArticleID="1019086" data-doi="10.1371/journal.pmed.0030436"><li><a href="http://dx.doi.org/10.1371/journal.pmed.0030436" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Priority+setting+for+pandemic+influenza%3A+an+analysis+of+national+preparedness+plans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Priority+setting+for+pandemic+influenza%3A+an+analysis+of+national+preparedness+plans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001190-Benacerraf1" id="pone.0001190-Benacerraf1"></a>Benacerraf B (1978) A hypothesis to relate the specificity of T lymphocytes and the activity of I region-specific Ir genes in macrophages and B lymphocytes. J Immunol  120: 1809–1812.  <ul class="find" data-citedArticleID="1018922"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+hypothesis+to+relate+the+specificity+of+T+lymphocytes+and+the+activity+of+I+region-specific+Ir+genes+in+macrophages+and+B+lymphocytes.&amp;auth=&amp;atitle=A+hypothesis+to+relate+the+specificity+of+T+lymphocytes+and+the+activity+of+I+region-specific+Ir+genes+in+macrophages+and+B+lymphocytes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+hypothesis+to+relate+the+specificity+of+T+lymphocytes+and+the+activity+of+I+region-specific+Ir+genes+in+macrophages+and+B+lymphocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+hypothesis+to+relate+the+specificity+of+T+lymphocytes+and+the+activity+of+I+region-specific+Ir+genes+in+macrophages+and+B+lymphocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001190-Townsend1" id="pone.0001190-Townsend1"></a>Townsend AR, Gotch FM, Davey J (1985) Cytotoxic T cells recognize fragments of the influenza nucleoprotein. Cell  42: 457–467.  <ul class="find" data-citedArticleID="1019072" data-doi="10.1016/0092-8674(85)90103-5"><li><a href="http://dx.doi.org/10.1016/0092-8674(85)90103-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytotoxic+T+cells+recognize+fragments+of+the+influenza+nucleoprotein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytotoxic+T+cells+recognize+fragments+of+the+influenza+nucleoprotein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001190-Bjorkman1" id="pone.0001190-Bjorkman1"></a>Bjorkman PJ, Saper MA, Samraoui B, Bennett WS, Strominger JL, et al.  (1987) The foreign antigen binding site and T cell recognition regions of class I histocompatibility antigens. Nature  329: 512–518.  <ul class="find" data-citedArticleID="1018926" data-doi="10.1038/329512a0"><li><a href="http://dx.doi.org/10.1038/329512a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+foreign+antigen+binding+site+and+T+cell+recognition+regions+of+class+I+histocompatibility+antigens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+foreign+antigen+binding+site+and+T+cell+recognition+regions+of+class+I+histocompatibility+antigens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001190-Germain1" id="pone.0001190-Germain1"></a>Germain RN (1994) MHC-dependent antigen processing and peptide presentation: providing ligands for T lymphocyte activation. Cell  76: 287–299.  <ul class="find" data-citedArticleID="1018982" data-doi="10.1016/0092-8674(94)90336-0"><li><a href="http://dx.doi.org/10.1016/0092-8674(94)90336-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MHC-dependent+antigen+processing+and+peptide+presentation%3A+providing+ligands+for+T+lymphocyte+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MHC-dependent+antigen+processing+and+peptide+presentation%3A+providing+ligands+for+T+lymphocyte+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001190-Cresswell1" id="pone.0001190-Cresswell1"></a>Cresswell P, Ackerman AL, Giodini A, Peaper DR, Wearsch PA (2005) Mechanisms of MHC class I-restricted antigen processing and cross-presentation. Immunol Rev  207: 145–157.  <ul class="find" data-citedArticleID="1018946" data-doi="10.1111/j.0105-2896.2005.00316.x"><li><a href="http://dx.doi.org/10.1111/j.0105-2896.2005.00316.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+MHC+class+I-restricted+antigen+processing+and+cross-presentation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+MHC+class+I-restricted+antigen+processing+and+cross-presentation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001190-Trombetta1" id="pone.0001190-Trombetta1"></a>Trombetta ES, Mellman I (2005) Cell biology of antigen processing in vitro and in vivo. Annu Rev Immunol  23: 975–1028.  <ul class="find" data-citedArticleID="1019078" data-doi="10.1146/annurev.immunol.22.012703.104538"><li><a href="http://dx.doi.org/10.1146/annurev.immunol.22.012703.104538" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+biology+of+antigen+processing+in+vitro+and+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+biology+of+antigen+processing+in+vitro+and+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001190-Falk1" id="pone.0001190-Falk1"></a>Falk K, Rotzschke O, Stevanovic S, Jung G, Rammensee HG (1991) Allele-specific motifs revealed by sequencing of self-peptides eluted from MHC molecules. Nature  351: 290–296.  <ul class="find" data-citedArticleID="1018966" data-doi="10.1038/351290a0"><li><a href="http://dx.doi.org/10.1038/351290a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Allele-specific+motifs+revealed+by+sequencing+of+self-peptides+eluted+from+MHC+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Allele-specific+motifs+revealed+by+sequencing+of+self-peptides+eluted+from+MHC+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001190-vanBleek1" id="pone.0001190-vanBleek1"></a>van Bleek GM, Nathenson SG (1991) The structure of the antigen-binding groove of major histocompatibility complex class I molecules determines specific selection of self-peptides. Proc Natl Acad Sci U S A  88: 11032–11036.  <ul class="find" data-citedArticleID="1019100" data-doi="10.1073/pnas.88.24.11032"><li><a href="http://dx.doi.org/10.1073/pnas.88.24.11032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+the+antigen-binding+groove+of+major+histocompatibility+complex+class+I+molecules+determines+specific+selection+of+self-peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+the+antigen-binding+groove+of+major+histocompatibility+complex+class+I+molecules+determines+specific+selection+of+self-peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001190-Engelhard1" id="pone.0001190-Engelhard1"></a>Engelhard VH (1994) Structure of peptides associated with class I and class II MHC molecules. Annu Rev Immunol  12: 181–207.  <ul class="find" data-citedArticleID="1018960" data-doi="10.1146/annurev.iy.12.040194.001145"><li><a href="http://dx.doi.org/10.1146/annurev.iy.12.040194.001145" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+peptides+associated+with+class+I+and+class+II+MHC+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+peptides+associated+with+class+I+and+class+II+MHC+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001190-Rammensee1" id="pone.0001190-Rammensee1"></a>Rammensee HG, Friede T, Stevanoviic S (1995) MHC ligands and peptide motifs: first listing. Immunogenetics  41: 178–228.  <ul class="find" data-citedArticleID="1019042" data-doi="10.1007/bf00172063"><li><a href="http://dx.doi.org/10.1007/bf00172063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MHC+ligands+and+peptide+motifs%3A+first+listing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MHC+ligands+and+peptide+motifs%3A+first+listing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001190-Hammer1" id="pone.0001190-Hammer1"></a>Hammer J, Valsasnini P, Tolba K, Bolin D, Higelin J, et al.  (1993) Promiscuous and allele-specific anchors in HLA-DR-binding peptides. Cell  74: 197–203.  <ul class="find" data-citedArticleID="1018988" data-doi="10.1016/0092-8674(93)90306-b"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)90306-b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Promiscuous+and+allele-specific+anchors+in+HLA-DR-binding+peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Promiscuous+and+allele-specific+anchors+in+HLA-DR-binding+peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001190-Sidney1" id="pone.0001190-Sidney1"></a>Sidney J, del Guercio MF, Southwood S, Engelhard VH, Appella E, et al.  (1995) Several HLA alleles share overlapping peptide specificities. J Immunol  154: 247–259.  <ul class="find" data-citedArticleID="1019054"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Several+HLA+alleles+share+overlapping+peptide+specificities.&amp;auth=&amp;atitle=Several+HLA+alleles+share+overlapping+peptide+specificities." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Several+HLA+alleles+share+overlapping+peptide+specificities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Several+HLA+alleles+share+overlapping+peptide+specificities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001190-Sette1" id="pone.0001190-Sette1"></a>Sette A, Sidney J (1999) Nine major HLA class I supertypes account for the vast preponderance of HLA-A and -B polymorphism. Immunogenetics  50: 201–212.  <ul class="find" data-citedArticleID="1019046" data-doi="10.1007/s002510050594"><li><a href="http://dx.doi.org/10.1007/s002510050594" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nine+major+HLA+class+I+supertypes+account+for+the+vast+preponderance+of+HLA-A+and+-B+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nine+major+HLA+class+I+supertypes+account+for+the+vast+preponderance+of+HLA-A+and+-B+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001190-Lund1" id="pone.0001190-Lund1"></a>Lund O, Nielsen M, Kesmir C, Petersen AG, Lundegaard C, et al.  (2004) Definition of supertypes for HLA molecules using clustering of specificity matrices. Immunogenetics  55: 797–810.  <ul class="find" data-citedArticleID="1019012" data-doi="10.1007/s00251-004-0647-4"><li><a href="http://dx.doi.org/10.1007/s00251-004-0647-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Definition+of+supertypes+for+HLA+molecules+using+clustering+of+specificity+matrices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Definition+of+supertypes+for+HLA+molecules+using+clustering+of+specificity+matrices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001190-Doytchinova1" id="pone.0001190-Doytchinova1"></a>Doytchinova IA, Flower DR (2005) In silico identification of supertypes for class II MHCs. J Immunol  174: 7085–7095.  <ul class="find" data-citedArticleID="1018956"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+silico+identification+of+supertypes+for+class+II+MHCs.&amp;auth=&amp;atitle=In+silico+identification+of+supertypes+for+class+II+MHCs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+silico+identification+of+supertypes+for+class+II+MHCs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+silico+identification+of+supertypes+for+class+II+MHCs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001190-Bian1" id="pone.0001190-Bian1"></a>Bian H, Hammer J (2004) Discovery of promiscuous HLA-II-restricted T cell epitopes with TEPITOPE. Methods  34: 468–475.  <ul class="find" data-citedArticleID="1018924" data-doi="10.1016/j.ymeth.2004.06.002"><li><a href="http://dx.doi.org/10.1016/j.ymeth.2004.06.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovery+of+promiscuous+HLA-II-restricted+T+cell+epitopes+with+TEPITOPE." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovery+of+promiscuous+HLA-II-restricted+T+cell+epitopes+with+TEPITOPE.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001190-Larsen1" id="pone.0001190-Larsen1"></a>Larsen MV, Lundegaard C, Lamberth K, Buus S, Brunak S, et al.  (2005) An integrative approach to CTL epitope prediction: a combined algorithm integrating MHC class I binding, TAP transport efficiency, and proteasomal cleavage predictions. Eur J Immunol  35: 2295–303.  <ul class="find" data-citedArticleID="1019008" data-doi="10.1002/eji.200425811"><li><a href="http://dx.doi.org/10.1002/eji.200425811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+integrative+approach+to+CTL+epitope+prediction%3A+a+combined+algorithm+integrating+MHC+class+I+binding%2C+TAP+transport+efficiency%2C+and+proteasomal+cleavage+predictions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+integrative+approach+to+CTL+epitope+prediction%3A+a+combined+algorithm+integrating+MHC+class+I+binding%2C+TAP+transport+efficiency%2C+and+proteasomal+cleavage+predictions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001190-Zhang1" id="pone.0001190-Zhang1"></a>Zhang GL, Khan AM, Srinivasan KN, August JT, Brusic V (2005) MULTIPRED: a computational system for prediction of promiscuous HLA binding peptides. Nucleic Acids Res 33(Web Server issue) W172–179.  <ul class="find" data-citedArticleID="1019098" data-doi="10.1093/nar/gki452"><li><a href="http://dx.doi.org/10.1093/nar/gki452" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MULTIPRED%3A+a+computational+system+for+prediction+of+promiscuous+HLA+binding+peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MULTIPRED%3A+a+computational+system+for+prediction+of+promiscuous+HLA+binding+peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001190-Bui1" id="pone.0001190-Bui1"></a>Bui HH, Sidney J, Peters B, Sathiamurthy M, Sinichi A, et al.  (2005) Automated generation and evaluation of specific MHC binding predictive tools: ARB matrix applications. Immunogenetics  57: 304–314.  <ul class="find" data-citedArticleID="1018934" data-doi="10.1007/s00251-005-0798-y"><li><a href="http://dx.doi.org/10.1007/s00251-005-0798-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automated+generation+and+evaluation+of+specific+MHC+binding+predictive+tools%3A+ARB+matrix+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automated+generation+and+evaluation+of+specific+MHC+binding+predictive+tools%3A+ARB+matrix+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001190-Wilson1" id="pone.0001190-Wilson1"></a>Wilson CC, McKinney D, Anders M, MaWhinney S, Forster J, et al.  (2003) Development of a DNA vaccine designed to induce cytotoxic T lymphocyte responses to multiple conserved epitopes in HIV-1. J Immunol  171: 5611–5623.  <ul class="find" data-citedArticleID="1019088"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1.&amp;auth=&amp;atitle=Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001190-Sette2" id="pone.0001190-Sette2"></a>Sette A, Fikes J (2003) Epitope-based vaccines: an update on epitope identification, vaccine design and delivery. Curr Opin Immunol  15: 461–470.  <ul class="find" data-citedArticleID="1019048" data-doi="10.1016/s0952-7915(03)00083-9"><li><a href="http://dx.doi.org/10.1016/s0952-7915(03)00083-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epitope-based+vaccines%3A+an+update+on+epitope+identification%2C+vaccine+design+and+delivery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epitope-based+vaccines%3A+an+update+on+epitope+identification%2C+vaccine+design+and+delivery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001190-Brusic1" id="pone.0001190-Brusic1"></a>Brusic V, August JT (2004) The changing field of vaccine development in the genomics era. Pharmacogenomics  5: 597–600.  <ul class="find" data-citedArticleID="1018932" data-doi="10.1517/14622416.5.6.597"><li><a href="http://dx.doi.org/10.1517/14622416.5.6.597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+changing+field+of+vaccine+development+in+the+genomics+era." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+changing+field+of+vaccine+development+in+the+genomics+era.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001190-Fischer1" id="pone.0001190-Fischer1"></a>Fischer W, Perkins S, Theiler J, Bhattacharya T, Yusim K, et al.  (2007) Polyvalent vaccines for optimal coverage of potential T-cell epitopes in global HIV-1 variants. Nat Med  13: 100–106.  <ul class="find" data-citedArticleID="1018972" data-doi="10.1038/nm1461"><li><a href="http://dx.doi.org/10.1038/nm1461" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polyvalent+vaccines+for+optimal+coverage+of+potential+T-cell+epitopes+in+global+HIV-1+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polyvalent+vaccines+for+optimal+coverage+of+potential+T-cell+epitopes+in+global+HIV-1+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001190-Klavinskis1" id="pone.0001190-Klavinskis1"></a>Klavinskis LS, Whitton JL, Oldstone MB (1989) Molecularly engineered vaccine which expresses an immunodominant T-cell epitope induces cytotoxic T lymphocytes that confer protection from lethal virus infection. J Virol  63: 4311–4316.  <ul class="find" data-citedArticleID="1019000"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecularly+engineered+vaccine+which+expresses+an+immunodominant+T-cell+epitope+induces+cytotoxic+T+lymphocytes+that+confer+protection+from+lethal+virus+infection.&amp;auth=&amp;atitle=Molecularly+engineered+vaccine+which+expresses+an+immunodominant+T-cell+epitope+induces+cytotoxic+T+lymphocytes+that+confer+protection+from+lethal+virus+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecularly+engineered+vaccine+which+expresses+an+immunodominant+T-cell+epitope+induces+cytotoxic+T+lymphocytes+that+confer+protection+from+lethal+virus+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecularly+engineered+vaccine+which+expresses+an+immunodominant+T-cell+epitope+induces+cytotoxic+T+lymphocytes+that+confer+protection+from+lethal+virus+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001190-Castrucci1" id="pone.0001190-Castrucci1"></a>Castrucci MR, Hou S, Doherty PC, Kawaoka Y (1994) Protection against lethal lymphocytic choriomeningitis virus (LCMV) infection by immunization of mice with an influenza virus containing an LCMV epitope recognized by cytotoxic T lymphocytes. J Virol  68: 3486–3490.  <ul class="find" data-citedArticleID="1018936"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Protection+against+lethal+lymphocytic+choriomeningitis+virus+%28LCMV%29+infection+by+immunization+of+mice+with+an+influenza+virus+containing+an+LCMV+epitope+recognized+by+cytotoxic+T+lymphocytes.&amp;auth=&amp;atitle=Protection+against+lethal+lymphocytic+choriomeningitis+virus+%28LCMV%29+infection+by+immunization+of+mice+with+an+influenza+virus+containing+an+LCMV+epitope+recognized+by+cytotoxic+T+lymphocytes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protection+against+lethal+lymphocytic+choriomeningitis+virus+%28LCMV%29+infection+by+immunization+of+mice+with+an+influenza+virus+containing+an+LCMV+epitope+recognized+by+cytotoxic+T+lymphocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protection+against+lethal+lymphocytic+choriomeningitis+virus+%28LCMV%29+infection+by+immunization+of+mice+with+an+influenza+virus+containing+an+LCMV+epitope+recognized+by+cytotoxic+T+lymphocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001190-Stemmer1" id="pone.0001190-Stemmer1"></a>Stemmer C, Quesnel A, Prevost-Blondel A, Zimmermann C, Muller S, et al.  (1999) Protection against lymphocytic choriomeningitis virus infection induced by a reduced peptide bond analogue of the H-2Db-restricted CD8(+) T cell epitope GP33. J Biol Chem  274: 5550–5556.  <ul class="find" data-citedArticleID="1019058" data-doi="10.1074/jbc.274.9.5550"><li><a href="http://dx.doi.org/10.1074/jbc.274.9.5550" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protection+against+lymphocytic+choriomeningitis+virus+infection+induced+by+a+reduced+peptide+bond+analogue+of+the+H-2Db-restricted+CD8%28%2B%29+T+cell+epitope+GP33." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protection+against+lymphocytic+choriomeningitis+virus+infection+induced+by+a+reduced+peptide+bond+analogue+of+the+H-2Db-restricted+CD8%28%2B%29+T+cell+epitope+GP33.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001190-Tsuji1" id="pone.0001190-Tsuji1"></a>Tsuji M, Bergmann CC, Takita-Sonoda Y, Murata K, Rodrigues EG, et al.  (1998) Recombinant Sindbis viruses expressing a cytotoxic T-lymphocyte epitope of a malaria parasite or of influenza virus elicit protection against the corresponding pathogen in mice. J Virol  72: 6907–6910.  <ul class="find" data-citedArticleID="1019080"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Recombinant+Sindbis+viruses+expressing+a+cytotoxic+T-lymphocyte+epitope+of+a+malaria+parasite+or+of+influenza+virus+elicit+protection+against+the+corresponding+pathogen+in+mice.&amp;auth=&amp;atitle=Recombinant+Sindbis+viruses+expressing+a+cytotoxic+T-lymphocyte+epitope+of+a+malaria+parasite+or+of+influenza+virus+elicit+protection+against+the+corresponding+pathogen+in+mice." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombinant+Sindbis+viruses+expressing+a+cytotoxic+T-lymphocyte+epitope+of+a+malaria+parasite+or+of+influenza+virus+elicit+protection+against+the+corresponding+pathogen+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombinant+Sindbis+viruses+expressing+a+cytotoxic+T-lymphocyte+epitope+of+a+malaria+parasite+or+of+influenza+virus+elicit+protection+against+the+corresponding+pathogen+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001190-GonzalezAseguinolaza1" id="pone.0001190-GonzalezAseguinolaza1"></a>Gonzalez-Aseguinolaza G, Nakaya Y, Molano A, Dy E, Esteban M, et al.  (2003) Induction of protective immunity against malaria by priming-boosting immunization with recombinant cold-adapted influenza and modified vaccinia Ankara viruses expressing a <sup>CD8 = </sup>-T-cell epitope derived from the circumsporozoite protein of Plasmodium yoelii. J Virol  77: 11859–11866.  <ul class="find" data-citedArticleID="1018986" data-doi="10.1128/jvi.77.21.11859-11866.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.21.11859-11866.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+protective+immunity+against+malaria+by+priming-boosting+immunization+with+recombinant+cold-adapted+influenza+and+modified+vaccinia+Ankara+viruses+expressing+a+CD8%E2%80%8A%3D%E2%80%8A-T-cell+epitope+derived+from+the+circumsporozoite+protein+of+Plasmodium+yoelii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+protective+immunity+against+malaria+by+priming-boosting+immunization+with+recombinant+cold-adapted+influenza+and+modified+vaccinia+Ankara+viruses+expressing+a+CD8%E2%80%8A%3D%E2%80%8A-T-cell+epitope+derived+from+the+circumsporozoite+protein+of+Plasmodium+yoelii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001190-DelVal1" id="pone.0001190-DelVal1"></a>Del Val M, Schlicht HJ, Volkmer H, Messerle M, Reddehase MJ, et al.  (1991) Protection against lethal cytomegalovirus infection by a recombinant vaccine containing a single nonameric T-cell epitope. J Virol  65: 3641–3646.  <ul class="find" data-citedArticleID="1018954"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Protection+against+lethal+cytomegalovirus+infection+by+a+recombinant+vaccine+containing+a+single+nonameric+T-cell+epitope.&amp;auth=&amp;atitle=Protection+against+lethal+cytomegalovirus+infection+by+a+recombinant+vaccine+containing+a+single+nonameric+T-cell+epitope." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protection+against+lethal+cytomegalovirus+infection+by+a+recombinant+vaccine+containing+a+single+nonameric+T-cell+epitope." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protection+against+lethal+cytomegalovirus+infection+by+a+recombinant+vaccine+containing+a+single+nonameric+T-cell+epitope.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001190-LaPosta1" id="pone.0001190-LaPosta1"></a>La Posta VJ, Auperin DD, Kamin-Lewis R, Cole GA (1993) Cross-protection against lymphocytic choriomeningitis virus mediated by a CD4+ T-cell clone specific for an envelope glycoprotein epitope of Lassa virus. J Virol  67: 3497–3506.  <ul class="find" data-citedArticleID="1019002"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cross-protection+against+lymphocytic+choriomeningitis+virus+mediated+by+a+CD4%2B+T-cell+clone+specific+for+an+envelope+glycoprotein+epitope+of+Lassa+virus.&amp;auth=&amp;atitle=Cross-protection+against+lymphocytic+choriomeningitis+virus+mediated+by+a+CD4%2B+T-cell+clone+specific+for+an+envelope+glycoprotein+epitope+of+Lassa+virus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cross-protection+against+lymphocytic+choriomeningitis+virus+mediated+by+a+CD4%2B+T-cell+clone+specific+for+an+envelope+glycoprotein+epitope+of+Lassa+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cross-protection+against+lymphocytic+choriomeningitis+virus+mediated+by+a+CD4%2B+T-cell+clone+specific+for+an+envelope+glycoprotein+epitope+of+Lassa+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001190-Oukka1" id="pone.0001190-Oukka1"></a>Oukka M, Manuguerra JC, Livaditis N, Tourdot S, Riche N, et al.  (1996) Protection against lethal viral infection by vaccination with nonimmunodominant peptides. J Immunol  157: 3039–3045.  <ul class="find" data-citedArticleID="1019030"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Protection+against+lethal+viral+infection+by+vaccination+with+nonimmunodominant+peptides.&amp;auth=&amp;atitle=Protection+against+lethal+viral+infection+by+vaccination+with+nonimmunodominant+peptides." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protection+against+lethal+viral+infection+by+vaccination+with+nonimmunodominant+peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protection+against+lethal+viral+infection+by+vaccination+with+nonimmunodominant+peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001190-Blaney1" id="pone.0001190-Blaney1"></a>Blaney JE Jr, Nobusawa E, Brehm MA, Bonneau RH, Mylin LM, et al.  (1998) Immunization with a single major histocompatibility complex class I-restricted cytotoxic T-lymphocyte recognition epitope of herpes simplex virus type 2 confers protective immunity. J Virol  72: 9567–9574.  <ul class="find" data-citedArticleID="1018928"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Immunization+with+a+single+major+histocompatibility+complex+class+I-restricted+cytotoxic+T-lymphocyte+recognition+epitope+of+herpes+simplex+virus+type+2+confers+protective+immunity.&amp;auth=&amp;atitle=Immunization+with+a+single+major+histocompatibility+complex+class+I-restricted+cytotoxic+T-lymphocyte+recognition+epitope+of+herpes+simplex+virus+type+2+confers+protective+immunity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunization+with+a+single+major+histocompatibility+complex+class+I-restricted+cytotoxic+T-lymphocyte+recognition+epitope+of+herpes+simplex+virus+type+2+confers+protective+immunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunization+with+a+single+major+histocompatibility+complex+class+I-restricted+cytotoxic+T-lymphocyte+recognition+epitope+of+herpes+simplex+virus+type+2+confers+protective+immunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001190-Feltkamp1" id="pone.0001190-Feltkamp1"></a>Feltkamp MC, Vreugdenhil GR, Vierboom MP, Ras E, van der Burg SH, et al.  (1995) Cytotoxic T lymphocytes raised against a subdominant epitope offered as a synthetic peptide eradicate human papillomavirus type 16-induced tumors. Eur J Immunol  25: 2638–2642.  <ul class="find" data-citedArticleID="1018968" data-doi="10.1002/eji.1830250935"><li><a href="http://dx.doi.org/10.1002/eji.1830250935" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytotoxic+T+lymphocytes+raised+against+a+subdominant+epitope+offered+as+a+synthetic+peptide+eradicate+human+papillomavirus+type+16-induced+tumors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytotoxic+T+lymphocytes+raised+against+a+subdominant+epitope+offered+as+a+synthetic+peptide+eradicate+human+papillomavirus+type+16-induced+tumors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001190-Harty1" id="pone.0001190-Harty1"></a>Harty JT, Bevan MJ (1992) <sup>CD8 = </sup> T cells specific for a single nonamer epitope of Listeria monocytogenes are protective in vivo. J Exp Med  175: 1531–1538.  <ul class="find" data-citedArticleID="1018992" data-doi="10.1084/jem.175.6.1531"><li><a href="http://dx.doi.org/10.1084/jem.175.6.1531" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD8%E2%80%8A%3D%E2%80%8A+T+cells+specific+for+a+single+nonamer+epitope+of+Listeria+monocytogenes+are+protective+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD8%E2%80%8A%3D%E2%80%8A+T+cells+specific+for+a+single+nonamer+epitope+of+Listeria+monocytogenes+are+protective+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001190-Plotnicky1" id="pone.0001190-Plotnicky1"></a>Plotnicky H, Cyblat-Chanal D, Aubry JP, Derouet F, Klinguer-Hamour C, et al.  (2003) The immunodominant influenza matrix T cell epitope recognized in human induces influenza protection in HLA-A2/K(b) transgenic mice. Virology  309: 320–329.  <ul class="find" data-citedArticleID="1019038" data-doi="10.1016/s0042-6822(03)00072-2"><li><a href="http://dx.doi.org/10.1016/s0042-6822(03)00072-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+immunodominant+influenza+matrix+T+cell+epitope+recognized+in+human+induces+influenza+protection+in+HLA-A2%2FK%28b%29+transgenic+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+immunodominant+influenza+matrix+T+cell+epitope+recognized+in+human+induces+influenza+protection+in+HLA-A2%2FK%28b%29+transgenic+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001190-Snyder1" id="pone.0001190-Snyder1"></a>Snyder JT, Belyakov IM, Dzutsev A, Lemonnier F, Berzofsky JA (2004) Protection against lethal vaccinia virus challenge in HLA-A2 transgenic mice by immunization with a single <sup>CD8 = </sup> T-cell peptide epitope of vaccinia and variola viruses. J Virol  78: 7052–7060.  <ul class="find" data-citedArticleID="1019056" data-doi="10.1128/jvi.78.13.7052-7060.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.13.7052-7060.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protection+against+lethal+vaccinia+virus+challenge+in+HLA-A2+transgenic+mice+by+immunization+with+a+single+CD8%E2%80%8A%3D%E2%80%8A+T-cell+peptide+epitope+of+vaccinia+and+variola+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protection+against+lethal+vaccinia+virus+challenge+in+HLA-A2+transgenic+mice+by+immunization+with+a+single+CD8%E2%80%8A%3D%E2%80%8A+T-cell+peptide+epitope+of+vaccinia+and+variola+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001190-Botten1" id="pone.0001190-Botten1"></a>Botten J, Whitton JL, Barrowman P, Sidney J, Whitmire JK, et al.  (2007) HLA-A2-restricted protection against lethal lymphocytic choriomeningitis. J Virol  81: 2307–2317.  <ul class="find" data-citedArticleID="1018930" data-doi="10.1128/jvi.02063-06"><li><a href="http://dx.doi.org/10.1128/jvi.02063-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA-A2-restricted+protection+against+lethal+lymphocytic+choriomeningitis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA-A2-restricted+protection+against+lethal+lymphocytic+choriomeningitis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001190-Hanke1" id="pone.0001190-Hanke1"></a>Hanke T, McMichael AJ, Dorrell L (2007) Clinical experience with plasmid DNA- and modified vaccinia virus Ankara-vectored human immunodeficiency virus type 1 clade A vaccine focusing on T-cell induction. J Gen Virol  88: 1–12.  <ul class="find" data-citedArticleID="1018990" data-doi="10.1099/vir.0.82493-0"><li><a href="http://dx.doi.org/10.1099/vir.0.82493-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clinical+experience+with+plasmid+DNA-+and+modified+vaccinia+virus+Ankara-vectored+human+immunodeficiency+virus+type+1+clade+A+vaccine+focusing+on+T-cell+induction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clinical+experience+with+plasmid+DNA-+and+modified+vaccinia+virus+Ankara-vectored+human+immunodeficiency+virus+type+1+clade+A+vaccine+focusing+on+T-cell+induction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001190-Moorthy1" id="pone.0001190-Moorthy1"></a>Moorthy VS, Imoukhuede EB, Keating S, Pinder M, Webster D, et al.  (2004) Phase 1 evaluation of 3 highly immunogenic prime-boost regimens, including a 12-month reboosting vaccination, for malaria vaccination in Gambian men. J Infect Dis  189: 2213–2219.  <ul class="find" data-citedArticleID="1019022" data-doi="10.1086/421118"><li><a href="http://dx.doi.org/10.1086/421118" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phase+1+evaluation+of+3+highly+immunogenic+prime-boost+regimens%2C+including+a+12-month+reboosting+vaccination%2C+for+malaria+vaccination+in+Gambian+men." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phase+1+evaluation+of+3+highly+immunogenic+prime-boost+regimens%2C+including+a+12-month+reboosting+vaccination%2C+for+malaria+vaccination+in+Gambian+men.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001190-McMichael1" id="pone.0001190-McMichael1"></a>McMichael AJ, Gotch FM (1989) Recognition of influenza A virus by human cytotoxic T lymphocytes. Adv Exp Med Biol  257: 109–114.  <ul class="find" data-citedArticleID="1019016" data-doi="10.1007/978-1-4684-5712-4_11"><li><a href="http://dx.doi.org/10.1007/978-1-4684-5712-4_11" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+influenza+A+virus+by+human+cytotoxic+T+lymphocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+influenza+A+virus+by+human+cytotoxic+T+lymphocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001190-Townsend2" id="pone.0001190-Townsend2"></a>Townsend AR (1987) Recognition of influenza virus proteins by cytotoxic T lymphocytes. Immunol Res  6: 80–100.  <ul class="find" data-citedArticleID="1019074" data-doi="10.1007/bf02918106"><li><a href="http://dx.doi.org/10.1007/bf02918106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+influenza+virus+proteins+by+cytotoxic+T+lymphocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+influenza+virus+proteins+by+cytotoxic+T+lymphocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001190-Askonas1" id="pone.0001190-Askonas1"></a>Askonas BA, Taylor PM, Esquivel F (1988) Cytotoxic T cells in influenza infection. Ann N Y Acad Sci  532: 230–237.  <ul class="find" data-citedArticleID="1018920" data-doi="10.1111/j.1749-6632.1988.tb36342.x"><li><a href="http://dx.doi.org/10.1111/j.1749-6632.1988.tb36342.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytotoxic+T+cells+in+influenza+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytotoxic+T+cells+in+influenza+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001190-Lamb1" id="pone.0001190-Lamb1"></a>Lamb JR, McMichael AJ, Rothbard JB (1987) T-cell recognition of influenza viral antigens. Hum Immunol  19: 79–89.  <ul class="find" data-citedArticleID="1019004" data-doi="10.1016/0198-8859(87)90096-6"><li><a href="http://dx.doi.org/10.1016/0198-8859(87)90096-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T-cell+recognition+of+influenza+viral+antigens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T-cell+recognition+of+influenza+viral+antigens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001190-Swain1" id="pone.0001190-Swain1"></a>Swain SL, Agrewala JN, Brown DM, Jelley-Gibbs DM, Golech S, et al.  (2006) CD4+ T-cell memory: generation and multi-faceted roles for CD4+ T cells in protective immunity to influenza. Immunol Rev  211: 8–22.  <ul class="find" data-citedArticleID="1019066" data-doi="10.1111/j.0105-2896.2006.00388.x"><li><a href="http://dx.doi.org/10.1111/j.0105-2896.2006.00388.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD4%2B+T-cell+memory%3A+generation+and+multi-faceted+roles+for+CD4%2B+T+cells+in+protective+immunity+to+influenza." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD4%2B+T-cell+memory%3A+generation+and+multi-faceted+roles+for+CD4%2B+T+cells+in+protective+immunity+to+influenza.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001190-Thomas1" id="pone.0001190-Thomas1"></a>Thomas PG, Keating R, Hulse-Post DJ, Doherty PC (2006) Cell-mediated protection in influenza infection. Emerg Infect Dis  12: 48–54.  <ul class="find" data-citedArticleID="1019070" data-doi="10.3201/eid1201.051237"><li><a href="http://dx.doi.org/10.3201/eid1201.051237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell-mediated+protection+in+influenza+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell-mediated+protection+in+influenza+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001190-Ulmer1" id="pone.0001190-Ulmer1"></a>Ulmer JB, Donnelly JJ, Parker SE, Rhodes GH, Felgner PL, et al.  (1993) Heterologous protection against influenza by injection of DNA encoding a viral protein. Science  259: 1745–1749.  <ul class="find" data-citedArticleID="1019082" data-doi="10.1126/science.8456302"><li><a href="http://dx.doi.org/10.1126/science.8456302" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterologous+protection+against+influenza+by+injection+of+DNA+encoding+a+viral+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterologous+protection+against+influenza+by+injection+of+DNA+encoding+a+viral+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001190-Ulmer2" id="pone.0001190-Ulmer2"></a>Ulmer JB, Fu TM, Deck RR, Friedman A, Guan L, et al.  (1998) Protective CD4+ and <sup>CD8 = </sup> T cells against influenza virus induced by vaccination with nucleoprotein DNA. J Virol  72: 5648–5653.  <ul class="find" data-citedArticleID="1019084"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Protective+CD4%2B+and+CD8%3F%3D%3F+T+cells+against+influenza+virus+induced+by+vaccination+with+nucleoprotein+DNA.&amp;auth=&amp;atitle=Protective+CD4%2B+and+CD8%3F%3D%3F+T+cells+against+influenza+virus+induced+by+vaccination+with+nucleoprotein+DNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protective+CD4%2B+and+CD8%E2%80%8A%3D%E2%80%8A+T+cells+against+influenza+virus+induced+by+vaccination+with+nucleoprotein+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protective+CD4%2B+and+CD8%E2%80%8A%3D%E2%80%8A+T+cells+against+influenza+virus+induced+by+vaccination+with+nucleoprotein+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001190-Fu1" id="pone.0001190-Fu1"></a>Fu TM, Guan L, Friedman A, Schofield TL, Ulmer JB, et al.  (1999) Dose dependence of CTL precursor frequency induced by a DNA vaccine and correlation with protective immunity against influenza virus challenge. J Immunol  162: 4163–4170.  <ul class="find" data-citedArticleID="1018980"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Dose+dependence+of+CTL+precursor+frequency+induced+by+a+DNA+vaccine+and+correlation+with+protective+immunity+against+influenza+virus+challenge.&amp;auth=&amp;atitle=Dose+dependence+of+CTL+precursor+frequency+induced+by+a+DNA+vaccine+and+correlation+with+protective+immunity+against+influenza+virus+challenge." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dose+dependence+of+CTL+precursor+frequency+induced+by+a+DNA+vaccine+and+correlation+with+protective+immunity+against+influenza+virus+challenge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dose+dependence+of+CTL+precursor+frequency+induced+by+a+DNA+vaccine+and+correlation+with+protective+immunity+against+influenza+virus+challenge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001190-Epstein1" id="pone.0001190-Epstein1"></a>Epstein SL, Tumpey TM, Misplon JA, Lo CY, Cooper LA, et al.  (2002) DNA vaccine expressing conserved influenza virus proteins protective against H5N1 challenge infection in mice. Emerg Infect Dis  8: 796–801.  <ul class="find" data-citedArticleID="1018962" data-doi="10.3201/eid0808.010476"><li><a href="http://dx.doi.org/10.3201/eid0808.010476" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+vaccine+expressing+conserved+influenza+virus+proteins+protective+against+H5N1+challenge+infection+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+vaccine+expressing+conserved+influenza+virus+proteins+protective+against+H5N1+challenge+infection+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001190-Epstein2" id="pone.0001190-Epstein2"></a>Epstein SL, Kong WP, Misplon JA, Lo CY, Tumpey TM, et al.  (2005) Protection against multiple influenza A subtypes by vaccination with highly conserved nucleoprotein. Vaccine  23: 5404–5410.  <ul class="find" data-citedArticleID="1018964" data-doi="10.1016/j.vaccine.2005.04.047"><li><a href="http://dx.doi.org/10.1016/j.vaccine.2005.04.047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protection+against+multiple+influenza+A+subtypes+by+vaccination+with+highly+conserved+nucleoprotein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protection+against+multiple+influenza+A+subtypes+by+vaccination+with+highly+conserved+nucleoprotein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001190-Fomsgaard1" id="pone.0001190-Fomsgaard1"></a>Fomsgaard A, Nielsen HV, Kirkby N, Bryder K, Corbet S, et al.  (1999) Induction of cytotoxic T-cell responses by gene gun DNA vaccination with minigenes encoding influenza A virus HA and NP CTL-epitopes. Vaccine  18: 681–691.  <ul class="find" data-citedArticleID="1018976" data-doi="10.1016/s0264-410x(99)00279-0"><li><a href="http://dx.doi.org/10.1016/s0264-410x(99)00279-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+cytotoxic+T-cell+responses+by+gene+gun+DNA+vaccination+with+minigenes+encoding+influenza+A+virus+HA+and+NP+CTL-epitopes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+cytotoxic+T-cell+responses+by+gene+gun+DNA+vaccination+with+minigenes+encoding+influenza+A+virus+HA+and+NP+CTL-epitopes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001190-Lawson1" id="pone.0001190-Lawson1"></a>Lawson CM, Bennink JR, Restifo NP, Yewdell JW, Murphy BR (1994) Primary pulmonary cytotoxic T lymphocytes induced by immunization with a vaccinia virus recombinant expressing influenza A virus nucleoprotein peptide do not protect mice against challenge. J Virol  68: 3505–3511.  <ul class="find" data-citedArticleID="1019010"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Primary+pulmonary+cytotoxic+T+lymphocytes+induced+by+immunization+with+a+vaccinia+virus+recombinant+expressing+influenza+A+virus+nucleoprotein+peptide+do+not+protect+mice+against+challenge.&amp;auth=&amp;atitle=Primary+pulmonary+cytotoxic+T+lymphocytes+induced+by+immunization+with+a+vaccinia+virus+recombinant+expressing+influenza+A+virus+nucleoprotein+peptide+do+not+protect+mice+against+challenge." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Primary+pulmonary+cytotoxic+T+lymphocytes+induced+by+immunization+with+a+vaccinia+virus+recombinant+expressing+influenza+A+virus+nucleoprotein+peptide+do+not+protect+mice+against+challenge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Primary+pulmonary+cytotoxic+T+lymphocytes+induced+by+immunization+with+a+vaccinia+virus+recombinant+expressing+influenza+A+virus+nucleoprotein+peptide+do+not+protect+mice+against+challenge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001190-Moskophidis1" id="pone.0001190-Moskophidis1"></a>Moskophidis D, Kioussis D (1998) Contribution of virus-specific <sup>CD8 = </sup> cytotoxic T cells to virus clearance or pathologic manifestations of influenza virus infection in a T cell receptor transgenic mouse model. J Exp Med  188: 223–232.  <ul class="find" data-citedArticleID="1019024" data-doi="10.1084/jem.188.2.223"><li><a href="http://dx.doi.org/10.1084/jem.188.2.223" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contribution+of+virus-specific+CD8%E2%80%8A%3D%E2%80%8A+cytotoxic+T+cells+to+virus+clearance+or+pathologic+manifestations+of+influenza+virus+infection+in+a+T+cell+receptor+transgenic+mouse+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contribution+of+virus-specific+CD8%E2%80%8A%3D%E2%80%8A+cytotoxic+T+cells+to+virus+clearance+or+pathologic+manifestations+of+influenza+virus+infection+in+a+T+cell+receptor+transgenic+mouse+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001190-Crowe1" id="pone.0001190-Crowe1"></a>Crowe SR, Miller SC, Woodland DL (2006) Identification of protective and non-protective T cell epitopes in influenza. Vaccine  24: 452–456.  <ul class="find" data-citedArticleID="1018948" data-doi="10.1016/j.vaccine.2005.07.090"><li><a href="http://dx.doi.org/10.1016/j.vaccine.2005.07.090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+protective+and+non-protective+T+cell+epitopes+in+influenza." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+protective+and+non-protective+T+cell+epitopes+in+influenza.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001190-Khan1" id="pone.0001190-Khan1"></a>Khan AM, Miotto O, Heiny AT, Salmon J, Srinivasan KN, et al.  (2007) A systematic bioinformatics approach for selection of epitope-based vaccine targets. Cell Immunol  244: 141–147.  <ul class="find" data-citedArticleID="1018996" data-doi="10.1016/j.cellimm.2007.02.005"><li><a href="http://dx.doi.org/10.1016/j.cellimm.2007.02.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+systematic+bioinformatics+approach+for+selection+of+epitope-based+vaccine+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+systematic+bioinformatics+approach+for+selection+of+epitope-based+vaccine+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001190-Miotto1" id="pone.0001190-Miotto1"></a>Miotto O, Tan TW, Brusic V (2007) Rule-based Knowledge Aggregation for Large-Scale Protein Sequence Analysis of Influenza A Viruses. BMC Bioinformatics  8: Suppl 10S7.  <ul class="find" data-citedArticleID="1019018" data-doi="10.1186/1471-2105-9-s1-s7"><li><a href="http://dx.doi.org/10.1186/1471-2105-9-s1-s7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rule-based+Knowledge+Aggregation+for+Large-Scale+Protein+Sequence+Analysis+of+Influenza+A+Viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rule-based+Knowledge+Aggregation+for+Large-Scale+Protein+Sequence+Analysis+of+Influenza+A+Viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001190-Cox1" id="pone.0001190-Cox1"></a>Cox NJ, Neumann G, Donis RO, Kawaoka Y (2005)  Orthomyxoviruses: influenza. In: Mahy BH, ter Meulen V, editors. Topley and Wilson's Microbiology and Microbial Infections, 10<sup>th</sup> Edition, Virology Volume 1, Chapter 32.   <ul class="find-nolinks"></ul></li><li><span class="label">66.
              </span><a name="pone.0001190-Edgar1" id="pone.0001190-Edgar1"></a>Edgar RC (2004) MUSCLE: a multiple sequence alignment method with reduced time and space complexity. BMC Bioinformatics  5: 113.  <ul class="find" data-citedArticleID="1018958"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity.&amp;auth=&amp;atitle=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001190-Fouchier1" id="pone.0001190-Fouchier1"></a>Fouchier RA, Munster V, Wallensten A, Bestebroer TM, Herfst S, et al.  (2005) Characterization of a novel influenza A virus hemagglutinin subtype (H16) obtained from black-headed gulls. J Virol  79: 2814–2822.  <ul class="find" data-citedArticleID="1018978" data-doi="10.1128/jvi.79.5.2814-2822.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.5.2814-2822.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+novel+influenza+A+virus+hemagglutinin+subtype+%28H16%29+obtained+from+black-headed+gulls." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+novel+influenza+A+virus+hemagglutinin+subtype+%28H16%29+obtained+from+black-headed+gulls.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001190-Shannon1" id="pone.0001190-Shannon1"></a>Shannon CE (1948) A mathematical theory of communication. Bell System Technical Journal  27: 379–423 and 623–656.  <ul class="find" data-citedArticleID="1019050" data-doi="10.1002/j.1538-7305.1948.tb01338.x"><li><a href="http://dx.doi.org/10.1002/j.1538-7305.1948.tb01338.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mathematical+theory+of+communication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mathematical+theory+of+communication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001190-Rammensee2" id="pone.0001190-Rammensee2"></a>Rammensee HG (1995) Chemistry of peptides associated with MHC class I and class II molecules. Curr Opin Immunol  7: 85–96.  <ul class="find" data-citedArticleID="1019044" data-doi="10.1016/0952-7915(95)80033-6"><li><a href="http://dx.doi.org/10.1016/0952-7915(95)80033-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemistry+of+peptides+associated+with+MHC+class+I+and+class+II+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemistry+of+peptides+associated+with+MHC+class+I+and+class+II+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001190-Paninski1" id="pone.0001190-Paninski1"></a>Paninski L (2003) Estimation of entropy and mutual information. Neural Computation  15: 1191–1253.  <ul class="find" data-citedArticleID="1019032" data-doi="10.1162/089976603321780272"><li><a href="http://dx.doi.org/10.1162/089976603321780272" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+entropy+and+mutual+information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+entropy+and+mutual+information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001190-Peters1" id="pone.0001190-Peters1"></a>Peters B, Sidney J, Bourne P, Bui HH, Buus S, et al.  (2005) The immune epitope database and analysis resource: from vision to blueprint. PLoS Biol  3: e91.  <ul class="find" data-citedArticleID="1019034" data-doi="10.1371/journal.pbio.0030091"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030091" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+immune+epitope+database+and+analysis+resource%3A+from+vision+to+blueprint." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+immune+epitope+database+and+analysis+resource%3A+from+vision+to+blueprint.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001190-Peters2" id="pone.0001190-Peters2"></a>Peters B, Bui HH, Frankild S, Nielson M, Lundegaard C, et al.  (2006) A community resource benchmarking predictions of peptide binding to MHC-I molecules. PLoS Comput Biol  2: e65.  <ul class="find" data-citedArticleID="1019036" data-doi="10.1371/journal.pcbi.0020065.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020065.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+community+resource+benchmarking+predictions+of+peptide+binding+to+MHC-I+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+community+resource+benchmarking+predictions+of+peptide+binding+to+MHC-I+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001190-Chen1" id="pone.0001190-Chen1"></a>Chen W, Calvo PA, Malide D, Gibbs J, Schubert U, et al.  (2001) A novel influenza A virus mitochondrial protein that induces cell death. Nat Med.  7(12): 1306–1312.  <ul class="find" data-citedArticleID="1018938" data-doi="10.1038/nm1201-1306"><li><a href="http://dx.doi.org/10.1038/nm1201-1306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+influenza+A+virus+mitochondrial+protein+that+induces+cell+death." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+influenza+A+virus+mitochondrial+protein+that+induces+cell+death.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001190-Coleman1" id="pone.0001190-Coleman1"></a>Coleman JR (2007) The PB1-F2 protein of Influenza A virus: increasing pathogenicity by disrupting alveolar macrophages. Virol J.  4: 9.  <ul class="find" data-citedArticleID="1018940"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+PB1-F2+protein+of+Influenza+A+virus%3A+increasing+pathogenicity+by+disrupting+alveolar+macrophages.&amp;auth=&amp;atitle=The+PB1-F2+protein+of+Influenza+A+virus%3A+increasing+pathogenicity+by+disrupting+alveolar+macrophages." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PB1-F2+protein+of+Influenza+A+virus%3A+increasing+pathogenicity+by+disrupting+alveolar+macrophages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PB1-F2+protein+of+Influenza+A+virus%3A+increasing+pathogenicity+by+disrupting+alveolar+macrophages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001190-Miotto2" id="pone.0001190-Miotto2"></a>Miotto O, Heiny AT, Tan TW, August JT, Brusic V (2007) Identification of human-to-human transmissibility factors in PB2 proteins of influenza A by large-scale mutual information analysis. BMC Bioinformatics,  8: Suppl 10S18.  <ul class="find" data-citedArticleID="1019020" data-doi="10.1186/1471-2105-9-s1-s18"><li><a href="http://dx.doi.org/10.1186/1471-2105-9-s1-s18" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+human-to-human+transmissibility+factors+in+PB2+proteins+of+influenza+A+by+large-scale+mutual+information+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+human-to-human+transmissibility+factors+in+PB2+proteins+of+influenza+A+by+large-scale+mutual+information+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001190-Mazumder1" id="pone.0001190-Mazumder1"></a>Mazumder R, Hu ZZ, Vinayaka CR, Sagripanti JL, Frost SD, et al.  (2007) Computational analysis and identification of amino acid sites in dengue E proteins relevant to development of diagnostics and vaccines. Virus Genes.  35: 175–186.  <ul class="find" data-citedArticleID="1019014" data-doi="10.1007/s11262-007-0103-2"><li><a href="http://dx.doi.org/10.1007/s11262-007-0103-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+analysis+and+identification+of+amino+acid+sites+in+dengue+E+proteins+relevant+to+development+of+diagnostics+and+vaccines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+analysis+and+identification+of+amino+acid+sites+in+dengue+E+proteins+relevant+to+development+of+diagnostics+and+vaccines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001190-Nickle1" id="pone.0001190-Nickle1"></a>Nickle DC, Rolland M, Jensen MA, Pond SL, Deng W, et al.  (2007) Coping with viral diversity in HIV vaccine design. PLoS Comput Biol.  3: e751.  <ul class="find" data-citedArticleID="1019026" data-doi="10.1371/journal.pcbi.0030075"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0030075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coping+with+viral+diversity+in+HIV+vaccine+design." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coping+with+viral+diversity+in+HIV+vaccine+design.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0001190-Fischer2" id="pone.0001190-Fischer2"></a>Fischer W, Perkins S, Theiler J, Bhattacharya T, Yusim K, et al.  (2007) Polyvalent vaccines for optimal coverage of potential T-cell epitopes in global HIV-1 variants. Nat. Med.  13: 100–106.  <ul class="find" data-citedArticleID="1018974" data-doi="10.1038/nm1461"><li><a href="http://dx.doi.org/10.1038/nm1461" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polyvalent+vaccines+for+optimal+coverage+of+potential+T-cell+epitopes+in+global+HIV-1+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polyvalent+vaccines+for+optimal+coverage+of+potential+T-cell+epitopes+in+global+HIV-1+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001190-DeGroot1" id="pone.0001190-DeGroot1"></a>De Groot AS, Marcon L, Bishop EA, Rivera D, Kutzler M, et al.  (2005) HIV vaccine development by computer assisted design: the GAIA vaccine. Vaccine.  23: 2136–2148.  <ul class="find" data-citedArticleID="1018950" data-doi="10.1016/j.vaccine.2005.01.097"><li><a href="http://dx.doi.org/10.1016/j.vaccine.2005.01.097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+vaccine+development+by+computer+assisted+design%3A+the+GAIA+vaccine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+vaccine+development+by+computer+assisted+design%3A+the+GAIA+vaccine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001190-Wilson2" id="pone.0001190-Wilson2"></a>Wilson CC, McKinney D, Anders M, MaWhinney S, Forster J, et al.  (2003) Development of a DNA vaccine designed to induce cytotoxic T lymphocyte responses to multiple conserved epitopes in HIV-1. J Immunol  171: 5611–5623.  <ul class="find" data-citedArticleID="1019090"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1.&amp;auth=&amp;atitle=Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0001190-Corbet1" id="pone.0001190-Corbet1"></a>Corbet S, Nielsen HV, Vinner L, Lauemoller S, Therrien D, et al.  (2003) Optimization and immune recognition of multiple novel conserved HLA-A2, human immunodeficiency virus type 1-specific CTL epitopes. The Journal of general virology  84: 2409–2421.  <ul class="find" data-citedArticleID="1018942" data-doi="10.1099/vir.0.19152-0"><li><a href="http://dx.doi.org/10.1099/vir.0.19152-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimization+and+immune+recognition+of+multiple+novel+conserved+HLA-A2%2C+human+immunodeficiency+virus+type+1-specific+CTL+epitopes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimization+and+immune+recognition+of+multiple+novel+conserved+HLA-A2%2C+human+immunodeficiency+virus+type+1-specific+CTL+epitopes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0001190-Wilson3" id="pone.0001190-Wilson3"></a>Wilson CC, McKinney D, Anders M, MaWhinney S, Forster J, et al.  (2003) Development of a DNA vaccine designed to induce cytotoxic T lymphocyte responses to multiple conserved epitopes in HIV-1. Journal of immunology (Baltimore, Md : 1950)  171: 5611–5623.  <ul class="find" data-citedArticleID="1019092"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1.&amp;auth=&amp;atitle=Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+a+DNA+vaccine+designed+to+induce+cytotoxic+T+lymphocyte+responses+to+multiple+conserved+epitopes+in+HIV-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0001190-Thakar1" id="pone.0001190-Thakar1"></a>Thakar MR, Bhonge LS, Lakhashe SK, Shankarkumar U, Sane SS, et al.  (2005) Cytolytic T lymphocytes (CTLs) from HIV-1 subtype C-infected Indian patients recognize CTL epitopes from a conserved immunodominant region of HIV-1 Gag and Nef. The Journal of infectious diseases  192: 749–759.  <ul class="find" data-citedArticleID="1019068" data-doi="10.1086/432547"><li><a href="http://dx.doi.org/10.1086/432547" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytolytic+T+lymphocytes+%28CTLs%29+from+HIV-1+subtype+C-infected+Indian+patients+recognize+CTL+epitopes+from+a+conserved+immunodominant+region+of+HIV-1+Gag+and+Nef." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytolytic+T+lymphocytes+%28CTLs%29+from+HIV-1+subtype+C-infected+Indian+patients+recognize+CTL+epitopes+from+a+conserved+immunodominant+region+of+HIV-1+Gag+and+Nef.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0001190-Yusim1" id="pone.0001190-Yusim1"></a>Yusim K, Kesmir C, Gaschen B, Addo MM, Altfeld M, et al.  (2002) Clustering patterns of cytotoxic T-lymphocyte epitopes in human immunodeficiency virus type 1 (HIV-1) proteins reveal imprints of immune evasion on HIV-1 global variation. Journal of virology  76: 8757–8768.  <ul class="find" data-citedArticleID="1019096" data-doi="10.1128/jvi.76.17.8757-8768.2002"><li><a href="http://dx.doi.org/10.1128/jvi.76.17.8757-8768.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clustering+patterns+of+cytotoxic+T-lymphocyte+epitopes+in+human+immunodeficiency+virus+type+1+%28HIV-1%29+proteins+reveal+imprints+of+immune+evasion+on+HIV-1+global+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clustering+patterns+of+cytotoxic+T-lymphocyte+epitopes+in+human+immunodeficiency+virus+type+1+%28HIV-1%29+proteins+reveal+imprints+of+immune+evasion+on+HIV-1+global+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0001190-Allen1" id="pone.0001190-Allen1"></a>Allen TM, Altfeld M, Yu XG, O'Sullivan KM, Lichterfeld M, et al.  (2004) Selection, transmission, and reversion of an antigen-processing cytotoxic T-lymphocyte escape mutation in human immunodeficiency virus type 1 infection. Journal of virology  78: 7069–7078.  <ul class="find" data-citedArticleID="1018918" data-doi="10.1128/jvi.78.13.7069-7078.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.13.7069-7078.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection%2C+transmission%2C+and+reversion+of+an+antigen-processing+cytotoxic+T-lymphocyte+escape+mutation+in+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection%2C+transmission%2C+and+reversion+of+an+antigen-processing+cytotoxic+T-lymphocyte+escape+mutation+in+human+immunodeficiency+virus+type+1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0001190-Yokomaku1" id="pone.0001190-Yokomaku1"></a>Yokomaku Y, Miura H, Tomiyama H, Kawana-Tachikawa A, Takiguchi M, et al.  (2004) Impaired processing and presentation of cytotoxic-T-lymphocyte (CTL) epitopes are major escape mechanisms from CTL immune pressure in human immunodeficiency virus type 1 infection. Journal of virology  78: 1324–1332.  <ul class="find" data-citedArticleID="1019094" data-doi="10.1128/jvi.78.3.1324-1332.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.3.1324-1332.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impaired+processing+and+presentation+of+cytotoxic-T-lymphocyte+%28CTL%29+epitopes+are+major+escape+mechanisms+from+CTL+immune+pressure+in+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impaired+processing+and+presentation+of+cytotoxic-T-lymphocyte+%28CTL%29+epitopes+are+major+escape+mechanisms+from+CTL+immune+pressure+in+human+immunodeficiency+virus+type+1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0001190-Ferguson1" id="pone.0001190-Ferguson1"></a>Ferguson NM, Galvani AP, Bush RM (2003) Ecological and immunological determinants of influenza evolution. Nature  422: 428–433.  <ul class="find" data-citedArticleID="1018970" data-doi="10.1038/nature01509"><li><a href="http://dx.doi.org/10.1038/nature01509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecological+and+immunological+determinants+of+influenza+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecological+and+immunological+determinants+of+influenza+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0001190-vanStipdonk1" id="pone.0001190-vanStipdonk1"></a>van Stipdonk MJ, Lemmens EE, Schoenberger SP (2001) Naive CTLs require a single brief period of antigenic stimulation for clonal expansion and differentiation. Nat Immunol  2: 423–429.  <ul class="find" data-citedArticleID="1019102"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Naive+CTLs+require+a+single+brief+period+of+antigenic+stimulation+for+clonal+expansion+and+differentiation.&amp;auth=&amp;atitle=Naive+CTLs+require+a+single+brief+period+of+antigenic+stimulation+for+clonal+expansion+and+differentiation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Naive+CTLs+require+a+single+brief+period+of+antigenic+stimulation+for+clonal+expansion+and+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Naive+CTLs+require+a+single+brief+period+of+antigenic+stimulation+for+clonal+expansion+and+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0001190-Sun1" id="pone.0001190-Sun1"></a>Sun JC, Williams MA, Bevan MJ (2004) CD4+ T cells are required for the maintenance, not programming, of memory <sup>CD8 = </sup> T cells after acute infection. Nature immunology  5: 927–933.  <ul class="find" data-citedArticleID="1019062" data-doi="10.1038/ni1105"><li><a href="http://dx.doi.org/10.1038/ni1105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD4%2B+T+cells+are+required+for+the+maintenance%2C+not+programming%2C+of+memory+CD8%E2%80%8A%3D%E2%80%8A+T+cells+after+acute+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD4%2B+T+cells+are+required+for+the+maintenance%2C+not+programming%2C+of+memory+CD8%E2%80%8A%3D%E2%80%8A+T+cells+after+acute+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0001190-Prlic1" id="pone.0001190-Prlic1"></a>Prlic M, Hernandez-Hoyos G, Bevan MJ (2006) Duration of the initial TCR stimulus controls the magnitude but not functionality of the <sup>CD8 = </sup> T cell response. The Journal of experimental medicine  203: 2135–2143.  <ul class="find" data-citedArticleID="1019040" data-doi="10.1084/jem.20060928"><li><a href="http://dx.doi.org/10.1084/jem.20060928" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duration+of+the+initial+TCR+stimulus+controls+the+magnitude+but+not+functionality+of+the+CD8%E2%80%8A%3D%E2%80%8A+T+cell+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duration+of+the+initial+TCR+stimulus+controls+the+magnitude+but+not+functionality+of+the+CD8%E2%80%8A%3D%E2%80%8A+T+cell+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0001190-Sun2" id="pone.0001190-Sun2"></a>Sun JC, Bevan MJ (2003) Defective CD8 T cell memory following acute infection without CD4 T cell help. Science (New York, NY)  300: 339–342.  <ul class="find" data-citedArticleID="1019064" data-doi="10.1126/science.1083317"><li><a href="http://dx.doi.org/10.1126/science.1083317" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Defective+CD8+T+cell+memory+following+acute+infection+without+CD4+T+cell+help." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Defective+CD8+T+cell+memory+following+acute+infection+without+CD4+T+cell+help.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0001190-Janssen1" id="pone.0001190-Janssen1"></a>Janssen EM, Lemmens EE, Wolfe T, Christen U, von Herrath MG, et al.  (2003) CD4+ T cells are required for secondary expansion and memory in CD8 =  T lymphocytes. Nature  421: 852–856.  <ul class="find" data-citedArticleID="1018994" data-doi="10.1038/nature01441"><li><a href="http://dx.doi.org/10.1038/nature01441" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD4%2B+T+cells+are+required+for+secondary+expansion+and+memory+in+CD8%E2%80%8A%3D%E2%80%8A+T+lymphocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD4%2B+T+cells+are+required+for+secondary+expansion+and+memory+in+CD8%E2%80%8A%3D%E2%80%8A+T+lymphocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0001190-Shedlock1" id="pone.0001190-Shedlock1"></a>Shedlock DJ, Shen H (2003) Requirement for CD4 T cell help in generating functional CD8 T cell memory. Science (New York, NY)  300: 337–339.  <ul class="find" data-citedArticleID="1019052" data-doi="10.1126/science.1082305"><li><a href="http://dx.doi.org/10.1126/science.1082305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Requirement+for+CD4+T+cell+help+in+generating+functional+CD8+T+cell+memory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Requirement+for+CD4+T+cell+help+in+generating+functional+CD8+T+cell+memory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.XML" value="158894"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.PDF" value="1033171"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g001.PNG_L" value="162266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g001.PNG_M" value="28449"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g001.PNG_S" value="5280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g001.TIF" value="274774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g001.PNG_I" value="9492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.e001.PNG" value="10412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.e001.TIF" value="26216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t001.PNG_L" value="116095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t001.PNG_M" value="64892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t001.PNG_S" value="10185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t001.TIF" value="569942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t001.PNG_I" value="20071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t002.PNG_L" value="87822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t002.PNG_M" value="118066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t002.PNG_S" value="17706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t002.TIF" value="421294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t002.PNG_I" value="39619"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g002.PNG_L" value="680815"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g002.PNG_M" value="106816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g002.PNG_S" value="7974"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g002.TIF" value="1251388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g002.PNG_I" value="36078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g003.PNG_L" value="945002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g003.PNG_M" value="154068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g003.PNG_S" value="10824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g003.TIF" value="1612644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g003.PNG_I" value="50861"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g004.PNG_L" value="848243"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g004.PNG_M" value="138961"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g004.PNG_S" value="9979"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g004.TIF" value="1602554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g004.PNG_I" value="45883"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g005.PNG_L" value="869405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g005.PNG_M" value="124158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g005.PNG_S" value="7802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g005.TIF" value="1009318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g005.PNG_I" value="40534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g006.PNG_L" value="2520098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g006.PNG_M" value="264594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g006.PNG_S" value="15219"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g006.TIF" value="3665664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g006.PNG_I" value="139982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t003.PNG_L" value="147053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t003.PNG_M" value="156563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t003.PNG_S" value="19681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t003.TIF" value="668874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.t003.PNG_I" value="77716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g007.PNG_L" value="1762476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g007.PNG_M" value="196517"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g007.PNG_S" value="22803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g007.TIF" value="3272292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g007.PNG_I" value="149655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g008.PNG_L" value="441803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g008.PNG_M" value="43790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g008.PNG_S" value="7347"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g008.TIF" value="739008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.g008.PNG_I" value="25226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.s001.DOC" value="261632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001190.s002.DOC" value="554496"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001190&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001190" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001190&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001190&volume=&issue=&title=Evolutionarily Conserved Protein Sequences of Influenza A Viruses, Avian and Human, as Vaccine Targets&author_name=A.%20T.%20Heiny%2C%20Olivo%20Miotto%2C%20Kellathur%20N.%20Srinivasan%2C%20Asif%20M.%20Khan%2C%20G.%20L.%20Zhang%2C%20Vladimir%20Brusic%2C%20Tin%20Wee%20Tan%2C%20J.%20Thomas%20August&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001190" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001190" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001190" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001190&amp;t=Evolutionarily%20Conserved%20Protein%20Sequences%20of%20Influenza%20A%20Viruses%2C%20Avian%20and%20Human%2C%20as%20Vaccine%20Targets" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001190&title=Evolutionarily%20Conserved%20Protein%20Sequences%20of%20Influenza%20A%20Viruses%2C%20Avian%20and%20Human%2C%20as%20Vaccine%20Targets&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001190&amp;title=Evolutionarily%20Conserved%20Protein%20Sequences%20of%20Influenza%20A%20Viruses%2C%20Avian%20and%20Human%2C%20as%20Vaccine%20Targets" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001190" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001190&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Evolutionarily Conserved Protein Sequences of Influenza A Viruses, Avian and Human, as Vaccine Targets';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001190';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Evolutionarily%20Conserved%20Protein%20Sequences%20of%20Influenza%20A%20Viruses%2C%20Avian%20and%20Human%2C%20as%20Vaccine%20Targets http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001190" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001190" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">





















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amino+acid+sequence+analysis%22" title="Search for articles in the subject area:'Amino acid sequence analysis'"><div class="flagText">Amino acid sequence analysis</div></a>
              <div data-categoryid="20173" data-articleid="25212"
                   data-categoryname="Amino acid sequence analysis"
                   class="flagImage" title="Flag 'Amino acid sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Avian+influenza+A+viruses%22" title="Search for articles in the subject area:'Avian influenza A viruses'"><div class="flagText">Avian influenza A viruses</div></a>
              <div data-categoryid="43251" data-articleid="25212"
                   data-categoryname="Avian influenza A viruses"
                   class="flagImage" title="Flag 'Avian influenza A viruses' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22H5N1%22" title="Search for articles in the subject area:'H5N1'"><div class="flagText">H5N1</div></a>
              <div data-categoryid="47163" data-articleid="25212"
                   data-categoryname="H5N1"
                   class="flagImage" title="Flag 'H5N1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Influenza%22" title="Search for articles in the subject area:'Influenza'"><div class="flagText">Influenza</div></a>
              <div data-categoryid="48257" data-articleid="25212"
                   data-categoryname="Influenza"
                   class="flagImage" title="Flag 'Influenza' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Influenza+A+virus%22" title="Search for articles in the subject area:'Influenza A virus'"><div class="flagText">Influenza A virus</div></a>
              <div data-categoryid="43725" data-articleid="25212"
                   data-categoryname="Influenza A virus"
                   class="flagImage" title="Flag 'Influenza A virus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="25212"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="25212"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22T+cells%22" title="Search for articles in the subject area:'T cells'"><div class="flagText">T cells</div></a>
              <div data-categoryid="40147" data-articleid="25212"
                   data-categoryname="T cells"
                   class="flagImage" title="Flag 'T cells' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3214'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2035'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5511&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=19003">Typo</a><br>Posted by asifkhan</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
