

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genome Sequence of the Saprophyte Leptospira biflexa Provides Insights into the Evolution of Leptospira and the Pathogenesis of Leptospirosis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001607"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001607" />

    <meta name="citation_title" content="Genome Sequence of the Saprophyte Leptospira biflexa Provides Insights into the Evolution of Leptospira and the Pathogenesis of Leptospirosis"/>
    <meta itemprop="name" content="Genome Sequence of the Saprophyte Leptospira biflexa Provides Insights into the Evolution of Leptospira and the Pathogenesis of Leptospirosis"/>

      <meta name="citation_author" content="Mathieu Picardeau"/>
            <meta name="citation_author_institution" content="Unité de Biologie des Spirochètes, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Dieter M. Bulach"/>
            <meta name="citation_author_institution" content="Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia"/>
            <meta name="citation_author_institution" content="Australian Research Council Centre of Excellence in Structural and Functional Microbial Genomics, Department of Microbiology, Monash University, Clayton, Victoria, Australia"/>
            <meta name="citation_author_institution" content="Australian Bacterial Pathogenesis Program, Department of Microbiology, Monash University, Clayton, Victoria, Australia"/>
      <meta name="citation_author" content="Christiane Bouchier"/>
            <meta name="citation_author_institution" content="Plate-forme Génomique, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Richard L. Zuerner"/>
            <meta name="citation_author_institution" content="Bacterial Diseases of Livestock Research Unit, National Animal Disease Center (NADC), Agricultural Research Service (ARS), United States Department of Agriculture (USDA), Ames, Iowa, United States of America"/>
      <meta name="citation_author" content="Nora Zidane"/>
            <meta name="citation_author_institution" content="Plate-forme Génomique, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Peter J. Wilson"/>
            <meta name="citation_author_institution" content="Australian Genome Research Facility, Gehrmann Laboratories, University of Queensland, St. Lucia, Queensland, Australia"/>
      <meta name="citation_author" content="Sophie Creno"/>
            <meta name="citation_author_institution" content="Plate-forme Génomique, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Elizabeth S. Kuczek"/>
            <meta name="citation_author_institution" content="Australian Genome Research Facility, Gehrmann Laboratories, University of Queensland, St. Lucia, Queensland, Australia"/>
      <meta name="citation_author" content="Simona Bommezzadri"/>
            <meta name="citation_author_institution" content="Unité de Biologie des Spirochètes, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="John C. Davis"/>
            <meta name="citation_author_institution" content="Australian Genome Research Facility, Gehrmann Laboratories, University of Queensland, St. Lucia, Queensland, Australia"/>
      <meta name="citation_author" content="Annette McGrath"/>
            <meta name="citation_author_institution" content="Australian Genome Research Facility, Gehrmann Laboratories, University of Queensland, St. Lucia, Queensland, Australia"/>
      <meta name="citation_author" content="Matthew J. Johnson"/>
            <meta name="citation_author_institution" content="Australian Genome Research Facility, Gehrmann Laboratories, University of Queensland, St. Lucia, Queensland, Australia"/>
      <meta name="citation_author" content="Caroline Boursaux-Eude"/>
            <meta name="citation_author_institution" content="Plate-forme Intégration et Analyse génomique, Institut Pasteur, Paris, France"/>
      <meta name="citation_author" content="Torsten Seemann"/>
            <meta name="citation_author_institution" content="Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia"/>
      <meta name="citation_author" content="Zoé Rouy"/>
            <meta name="citation_author_institution" content="Commissariat à l'Energie Atomique (CEA), Direction des Sciences du Vivant, Laboratoire de Génomique Comparative, Institut de Génomique, Genoscope, Evry, France"/>
      <meta name="citation_author" content="Ross L. Coppel"/>
            <meta name="citation_author_institution" content="Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia"/>
            <meta name="citation_author_institution" content="Australian Research Council Centre of Excellence in Structural and Functional Microbial Genomics, Department of Microbiology, Monash University, Clayton, Victoria, Australia"/>
      <meta name="citation_author" content="Julian I. Rood"/>
            <meta name="citation_author_institution" content="Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia"/>
            <meta name="citation_author_institution" content="Australian Research Council Centre of Excellence in Structural and Functional Microbial Genomics, Department of Microbiology, Monash University, Clayton, Victoria, Australia"/>
            <meta name="citation_author_institution" content="Australian Bacterial Pathogenesis Program, Department of Microbiology, Monash University, Clayton, Victoria, Australia"/>
      <meta name="citation_author" content="Aurélie Lajus"/>
            <meta name="citation_author_institution" content="Commissariat à l'Energie Atomique (CEA), Direction des Sciences du Vivant, Laboratoire de Génomique Comparative, Institut de Génomique, Genoscope, Evry, France"/>
      <meta name="citation_author" content="John K. Davies"/>
            <meta name="citation_author_institution" content="Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia"/>
            <meta name="citation_author_institution" content="Australian Research Council Centre of Excellence in Structural and Functional Microbial Genomics, Department of Microbiology, Monash University, Clayton, Victoria, Australia"/>
            <meta name="citation_author_institution" content="Australian Bacterial Pathogenesis Program, Department of Microbiology, Monash University, Clayton, Victoria, Australia"/>
      <meta name="citation_author" content="Claudine Médigue"/>
            <meta name="citation_author_institution" content="Commissariat à l'Energie Atomique (CEA), Direction des Sciences du Vivant, Laboratoire de Génomique Comparative, Institut de Génomique, Genoscope, Evry, France"/>
            <meta name="citation_author_institution" content="Centre National de la Recherche Scientifique (CNRS) UMR8030, Génomique Métabolique, Evry, France"/>
      <meta name="citation_author" content="Ben Adler"/>
            <meta name="citation_author_institution" content="Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia"/>
            <meta name="citation_author_institution" content="Australian Research Council Centre of Excellence in Structural and Functional Microbial Genomics, Department of Microbiology, Monash University, Clayton, Victoria, Australia"/>
            <meta name="citation_author_institution" content="Australian Bacterial Pathogenesis Program, Department of Microbiology, Monash University, Clayton, Victoria, Australia"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001607.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1607"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Cell aggregation: a mechanism of pathogenic Leptospira to survive in fresh water.; citation_author=G Trueba; citation_author=S Zapata; citation_author=K Madrid; citation_author=P Cullen; citation_author=D Haake; citation_journal_title=International Microbiology; citation_volume=7; citation_number=1; citation_pages=35-40; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Leptospira and Leptospirosis.; citation_author=S Faine; citation_author=B Adler; citation_author=C Bolin; citation_author=P Perolat; citation_number=2; citation_date=1999; citation_publisher=MediSci; " />
      <meta name="citation_reference" content="citation_title=Further determination of DNA relatedness between serogroups and serovars in the family Leptospiraceae with a proposal for Leptospira alexanderi sp. nov. and four new Leptospira genomospecies.; citation_author=DJ Brenner; citation_author=AF Kaufmann; citation_author=KR Sulzer; citation_author=AG Steigerwalt; citation_author=FC Rogers; citation_journal_title=International Journal of Systematic Bacteriology; citation_volume=49; citation_number=3; citation_pages=839-858; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Leptospirosis.; citation_author=PN Levett; citation_journal_title=Clinical Microbiology Reviews; citation_volume=14; citation_number=4; citation_pages=296-326; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genome reduction in Leptospira borgpetersenii reflects limited transmission potential.; citation_author=DM Bulach; citation_author=RL Zuerner; citation_author=P Wilson; citation_author=T Seemann; citation_author=A McGrath; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=103; citation_number=5; citation_pages=14560-14565; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparative genomics of two Leptospira interrogans serovars reveals novel insights into Physiology and Pathogenesis.; citation_author=ALTO Nascimento; citation_author=AI Ko; citation_author=EAL Martins; citation_author=CB Monteiro-Vitorello; citation_author=PL Ho; citation_journal_title=Journal of Bacteriology; citation_volume=186; citation_number=6; citation_pages=2164-2172; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Unique physiological and pathogenic features of Leptospira interrogans revealed by whole-genome sequencing.; citation_author=SX Ren; citation_author=G Fu; citation_author=XG Jiang; citation_author=R Zeng; citation_author=YG Miao; citation_journal_title=Nature; citation_volume=422; citation_number=7; citation_pages=888-893; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparison of genetic maps for two Leptospira interrogans serovars provides evidence for two chromosomes and intraspecies heterogeneity.; citation_author=RL Zuerner; citation_author=JL Herrmann; citation_author=I Saint Girons; citation_journal_title=Journal of Bacteriology; citation_volume=175; citation_number=8; citation_pages=5445-5451; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Deletions of recBCD or recD influence genetic transformation differently and are lethal together with a recJ deletion in Acinetobacter baylyi.; citation_author=E Kickstein; citation_author=K Harms; citation_author=W Wackernagel; citation_journal_title=Microbiology; citation_volume=153; citation_number=9; citation_pages=2259-2270; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Complete nucleotide sequence of the LE1 prophage from the spirochete Leptospira biflexa and characterization of Its replication and partition functions.; citation_author=P Bourhy; citation_author=L Frangeul; citation_author=E Couve; citation_author=P Glaser; citation_author=I Saint Girons; citation_journal_title=Journal of Bacteriology; citation_volume=187; citation_number=10; citation_pages=3931-3940; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The LE1 bacteriophage replicates as a plasmid within Leptospira biflexa: construction of an L. biflexa-Escherichia coli shuttle vector.; citation_author=I Saint Girons; citation_author=P Bourhy; citation_author=C Ottone; citation_author=M Picardeau; citation_author=D Yelton; citation_journal_title=Journal of Bacteriology; citation_volume=182; citation_number=11; citation_pages=5700-5705; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A Genomic island of the pathogen Leptospira interrogans serovar Lai can excise from its chromosome.; citation_author=P Bourhy; citation_author=L Salaun; citation_author=A Lajus; citation_author=C Medigue; citation_author=C Boursaux-Eude; citation_journal_title=Infection and Immunity; citation_volume=75; citation_number=12; citation_pages=677-683; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analysis of the spirochetes.; citation_author=BJ Paster; citation_author=FE Dewhirst; citation_author=WG Weisburg; citation_author=LA Tordoff; citation_author=GJ Fraser; citation_journal_title=Journal of Bacteriology; citation_volume=173; citation_number=13; citation_pages=6101-6109; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution and mosaicism of leptospiral outer membrane proteins involves horizontal DNA transfer.; citation_author=DA Haake; citation_author=MA Suchard; citation_author=MM Kelley; citation_author=M Dundoo; citation_author=DP Alt; citation_journal_title=Journal of Bacteriology; citation_volume=186; citation_number=14; citation_pages=2818-2828; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic characterization of pathogenic Leptospira species by DNA hybridization.; citation_author=P Ramadass; citation_author=BD Jarvis; citation_author=RJ Corner; citation_author=D Penny; citation_author=RB Marshall; citation_journal_title=International Journal of Systematic Bacteriology; citation_volume=42; citation_number=15; citation_pages=215-219; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of the Borrelia garinii genome.; citation_author=G Glockner; citation_author=R Lehmann; citation_author=A Romualdi; citation_author=S Pradella; citation_author=U Schulte-Spechtel; citation_journal_title=Nucleic Acids Research; citation_volume=32; citation_number=16; citation_pages=6038-6046; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mycobacterium marinum: the generalization and specialization of a pathogenic mycobacterium.; citation_author=LM Stamm; citation_author=EJ Brown; citation_journal_title=Microbes and Infection; citation_volume=6; citation_number=17; citation_pages=1418-1428; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Reductive evolution and niche adaptation inferred from the genome of Mycobacterium ulcerans, the causative agent of Buruli ulcer.; citation_author=TP Stinear; citation_author=T Seemann; citation_author=S Pidot; citation_author=W Frigui; citation_author=G Reysset; citation_journal_title=Genome Research; citation_volume=17; citation_number=18; citation_pages=192-200; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Multiple functions of the leucine-rich repeat protein LrrA of Treponema denticola.; citation_author=A Ikegami; citation_author=K Honma; citation_author=A Sharma; citation_author=H Kuramitsu; citation_journal_title=Infection and Immunity; citation_volume=72; citation_number=19; citation_pages=4619-4627; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Outer membrane proteins of pathogenic spirochetes.; citation_author=PA Cullen; citation_author=DA Haake; citation_author=B Adler; citation_journal_title=FEMS Microbiology Reviews; citation_volume=28; citation_number=20; citation_pages=291-318; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=LipL21 Is a novel surface-exposed lipoprotein of pathogenic Leptospira species.; citation_author=PA Cullen; citation_author=DA Haake; citation_author=DM Bulach; citation_author=RL Zuerner; citation_author=B Adler; citation_journal_title=Infection and Immunity; citation_volume=71; citation_number=21; citation_pages=2414-2421; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Physiological osmotic induction of Leptospira interrogans adhesion: LigA and LigB bind extracellular matrix proteins and fibrinogen.; citation_author=HA Choy; citation_author=MM Kelley; citation_author=TL Chen; citation_author=AK Moller; citation_author=J Matsunaga; citation_journal_title=Infection and Immunity; citation_volume=75; citation_number=22; citation_pages=2441-2450; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=LfhA, a novel Factor H-binding protein of Leptospira interrogans.; citation_author=A Verma; citation_author=J Hellwage; citation_author=S Artiushin; citation_author=PF Zipfel; citation_author=P Kraiczy; citation_journal_title=Infection and Immunity; citation_volume=74; citation_number=23; citation_pages=2659-2666; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A newly identified leptospiral adhesin mediates attachment to laminin.; citation_author=AS Barbosa; citation_author=PAE Abreu; citation_author=FO Neves; citation_author=MV Atzingen; citation_author=MM Watanabe; citation_journal_title=Infection and Immunity; citation_volume=74; citation_number=24; citation_pages=6356-6364; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparative and functional genomic analyses of iron transport and regulation in Leptospira spp.; citation_author=H Louvel; citation_author=S Bommezzadri; citation_author=N Zidane; citation_author=C Boursaux-Eude; citation_author=S Creno; citation_journal_title=Journal of Bacteriology; citation_volume=188; citation_number=25; citation_pages=7893-7904; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Presence of putative sphingomyelinase genes among members of the family Leptospiraceae.; citation_author=RP Segers; citation_author=JA van Gestel; citation_author=GJ van Eys; citation_author=BA van der Zeijst; citation_author=W Gaastra; citation_journal_title=Infection and Immunity; citation_volume=60; citation_number=26; citation_pages=1707-1710; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Molecular analysis of a sphingomyelinase C gene from Leptospira interrogans serovar hardjo.; citation_author=RP Segers; citation_author=A van der Drift; citation_author=A de Nijs; citation_author=P Corcione; citation_author=BA van der Zeijst; citation_journal_title=Infection and Immunity; citation_volume=58; citation_number=27; citation_pages=2177-2185; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Cloning of a hemolysin gene from Leptospira interrogans serovar hardjo.; citation_author=G del Real; citation_author=RP Segers; citation_author=BA van der Zeijst; citation_author=W Gaastra; citation_journal_title=Infection and Immunity; citation_volume=57; citation_number=28; citation_pages=2588-2590; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The OmpA-like protein Loa22 Is essential for leptospiral virulence.; citation_author=P Ristow; citation_author=P Bourhy; citation_author=F McBride; citation_author=C via Weykamp da; citation_author=C Figueira; citation_journal_title=PLoS Pathogens; citation_volume=3; citation_number=29; citation_pages=e97; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genetic manipulation of Leptospira biflexa.; citation_author=H Louvel; citation_author=M Picardeau; citation_number=30; citation_date=2007; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_title=Studio serologico del gruppo Semaranga-Patoc di Leptospira biflexa.;; citation_author=B Babudieri; citation_number=31; citation_pages=408-414; citation_date=1961; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. I. Accuracy assessment.; citation_author=B Ewing; citation_author=L Hillier; citation_author=MC Wendl; citation_author=P Green; citation_journal_title=Genome Research; citation_volume=8; citation_number=32; citation_pages=175-185; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Consed: a graphical tool for sequence finishing.; citation_author=D Gordon; citation_author=C Abajian; citation_author=P Green; citation_journal_title=Genome Research; citation_volume=8; citation_number=33; citation_pages=195-202; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Automated finishing with Autofinish.; citation_author=D Gordon; citation_author=C Desmarais; citation_author=P Green; citation_journal_title=Genome Research; citation_volume=11; citation_number=34; citation_pages=614-625; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=AMIGene: Annotation of MIcrobial Genes.; citation_author=S Bocs; citation_author=S Cruveiller; citation_author=D Vallenet; citation_author=G Nuel; citation_author=C Medigue; citation_journal_title=Nucleic Acids Research; citation_volume=31; citation_number=35; citation_pages=3723-3726; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MaGe - a microbial genome annotation system supported by synteny result.; citation_author=D Vallenet; citation_author=L Labarre; citation_author=Z Rouy; citation_author=V Barbe; citation_author=S Bocs; citation_journal_title=Nucleic Acids Research; citation_volume=34; citation_number=36; citation_pages=53-65; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Whole-genome reciprocal BLAST analysis reveals that Planctomycetes do not share an unusually large number of genes with Eukarya and Archaea.; citation_author=CA Fuchsman; citation_author=G Rocap; citation_journal_title=Applied and Environmental Microbiology; citation_volume=72; citation_number=37; citation_pages=6841-6844; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=First isolation of bacteriophages for a spirochaete: potential genetic tools for Leptospira.; citation_author=I Saint Girons; citation_author=D Margarita; citation_author=P Amouriaux; citation_author=G Baranton; citation_journal_title=Research in Microbiology; citation_volume=141; citation_number=38; citation_pages=1131-1138; citation_date=1990; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001607" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genome Sequence of the Saprophyte Leptospira biflexa Provides Insights into the Evolution of Leptospira and the Pathogenesis of Leptospirosis"/>
    <meta name="twitter:description" content="Leptospira biflexa is a free-living saprophytic spirochete present in aquatic environments. We determined the genome sequence of L. biflexa, making it the first saprophytic Leptospira to be sequenced. The L. biflexa genome has 3,590 protein-coding genes distributed across three circular replicons: the major 3,604 chromosome, a smaller 278-kb replicon that also carries essential genes, and a third 74-kb replicon. Comparative sequence analysis provides evidence that L. biflexa is an excellent model for the study of Leptospira evolution; we conclude that 2052 genes (61%) represent a progenitor genome that existed before divergence of pathogenic and saprophytic Leptospira species. Comparisons of the L. biflexa genome with two pathogenic Leptospira species reveal several major findings. Nearly one-third of the L. biflexa genes are absent in pathogenic Leptospira. We suggest that once incorporated into the L. biflexa genome, laterally transferred DNA undergoes minimal rearrangement due to physical restrictions imposed by high gene density and limited presence of transposable elements. In contrast, the genomes of pathogenic Leptospira species undergo frequent rearrangements, often involving recombination between insertion sequences. Identification of genes common to the two pathogenic species, L. borgpetersenii and L. interrogans, but absent in L. biflexa, is consistent with a role for these genes in pathogenesis. Differences in environmental sensing capacities of L. biflexa, L. borgpetersenii, and L. interrogans suggest a model which postulates that loss of signal transduction functions in L. borgpetersenii has impaired its survival outside a mammalian host, whereas L. interrogans has retained environmental sensory functions that facilitate disease transmission through water."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001607.g003"/>

  <meta property="og:title" content="Genome Sequence of the Saprophyte Leptospira biflexa Provides Insights into the Evolution of Leptospira and the Pathogenesis of Leptospirosis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001607" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4807'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=476'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6875&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001607">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001607" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001607&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genome Sequence of the Saprophyte <i>Leptospira biflexa</i> Provides Insights into the Evolution of <i>Leptospira</i> and the Pathogenesis of Leptospirosis
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mathieu Picardeau
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Mathieu Picardeau, Dieter M. Bulach</p>

              
              

                <p>Affiliation:
                  Unité de Biologie des Spirochètes, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dieter M. Bulach
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Mathieu Picardeau, Dieter M. Bulach</p>

              
              

                <p>Affiliations:
                  Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia, 
                  Australian Research Council Centre of Excellence in Structural and Functional Microbial Genomics, Department of Microbiology, Monash University, Clayton, Victoria, Australia, 
                  Australian Bacterial Pathogenesis Program, Department of Microbiology, Monash University, Clayton, Victoria, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christiane Bouchier, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Plate-forme Génomique, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Richard L. Zuerner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bacterial Diseases of Livestock Research Unit, National Animal Disease Center (NADC), Agricultural Research Service (ARS), United States Department of Agriculture (USDA), Ames, Iowa, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nora Zidane, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Plate-forme Génomique, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter J. Wilson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Australian Genome Research Facility, Gehrmann Laboratories, University of Queensland, St. Lucia, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sophie Creno, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Plate-forme Génomique, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elizabeth S. Kuczek, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Australian Genome Research Facility, Gehrmann Laboratories, University of Queensland, St. Lucia, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Simona Bommezzadri, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Unité de Biologie des Spirochètes, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John C. Davis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Australian Genome Research Facility, Gehrmann Laboratories, University of Queensland, St. Lucia, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Annette McGrath, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Australian Genome Research Facility, Gehrmann Laboratories, University of Queensland, St. Lucia, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew J. Johnson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Australian Genome Research Facility, Gehrmann Laboratories, University of Queensland, St. Lucia, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Caroline Boursaux-Eude, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Plate-forme Intégration et Analyse génomique, Institut Pasteur, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Torsten Seemann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zoé Rouy, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Commissariat à l'Energie Atomique (CEA), Direction des Sciences du Vivant, Laboratoire de Génomique Comparative, Institut de Génomique, Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ross L. Coppel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia, 
                  Australian Research Council Centre of Excellence in Structural and Functional Microbial Genomics, Department of Microbiology, Monash University, Clayton, Victoria, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Julian I. Rood, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia, 
                  Australian Research Council Centre of Excellence in Structural and Functional Microbial Genomics, Department of Microbiology, Monash University, Clayton, Victoria, Australia, 
                  Australian Bacterial Pathogenesis Program, Department of Microbiology, Monash University, Clayton, Victoria, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Aurélie Lajus, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Commissariat à l'Energie Atomique (CEA), Direction des Sciences du Vivant, Laboratoire de Génomique Comparative, Institut de Génomique, Genoscope, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John K. Davies, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia, 
                  Australian Research Council Centre of Excellence in Structural and Functional Microbial Genomics, Department of Microbiology, Monash University, Clayton, Victoria, Australia, 
                  Australian Bacterial Pathogenesis Program, Department of Microbiology, Monash University, Clayton, Victoria, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claudine Médigue, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Commissariat à l'Energie Atomique (CEA), Direction des Sciences du Vivant, Laboratoire de Génomique Comparative, Institut de Génomique, Genoscope, Evry, France, 
                  Centre National de la Recherche Scientifique (CNRS) UMR8030, Génomique Métabolique, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ben Adler
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:Ben.Adler@med.monash.edu.au">Ben.Adler@med.monash.edu.au</a></p>

                <p>Affiliations:
                  Victorian Bioinformatics Consortium, Monash University, Clayton, Victoria, Australia, 
                  Australian Research Council Centre of Excellence in Structural and Functional Microbial Genomics, Department of Microbiology, Monash University, Clayton, Victoria, Australia, 
                  Australian Bacterial Pathogenesis Program, Department of Microbiology, Monash University, Clayton, Victoria, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001607</li>
      <li>Featured in <a href="http://www.ploscollections.org">PLOS Collections</a></li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001607" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001607" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001607" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001607" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001607-g001" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001607-t001" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001607-t002" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001607-g002" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001607-t003" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001607-g003" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001607-t004" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001607-t005" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001607">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001607" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p><em>Leptospira biflexa</em> is a free-living saprophytic spirochete present in aquatic environments. We determined the genome sequence of <em>L. biflexa</em>, making it the first saprophytic <em>Leptospira</em> to be sequenced. The <em>L. biflexa</em> genome has 3,590 protein-coding genes distributed across three circular replicons: the major 3,604 chromosome, a smaller 278-kb replicon that also carries essential genes, and a third 74-kb replicon. Comparative sequence analysis provides evidence that <em>L. biflexa</em> is an excellent model for the study of <em>Leptospira</em> evolution; we conclude that 2052 genes (61%) represent a progenitor genome that existed before divergence of pathogenic and saprophytic <em>Leptospira</em> species. Comparisons of the <em>L. biflexa</em> genome with two pathogenic <em>Leptospira</em> species reveal several major findings. Nearly one-third of the <em>L. biflexa</em> genes are absent in pathogenic <em>Leptospira</em>. We suggest that once incorporated into the <em>L. biflexa</em> genome, laterally transferred DNA undergoes minimal rearrangement due to physical restrictions imposed by high gene density and limited presence of transposable elements. In contrast, the genomes of pathogenic <em>Leptospira</em> species undergo frequent rearrangements, often involving recombination between insertion sequences. Identification of genes common to the two pathogenic species, <em>L. borgpetersenii</em> and <em>L. interrogans</em>, but absent in <em>L. biflexa</em>, is consistent with a role for these genes in pathogenesis. Differences in environmental sensing capacities of <em>L. biflexa, L. borgpetersenii,</em> and <em>L. interrogans</em> suggest a model which postulates that loss of signal transduction functions in <em>L. borgpetersenii</em> has impaired its survival outside a mammalian host, whereas <em>L. interrogans</em> has retained environmental sensory functions that facilitate disease transmission through water.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Picardeau M, Bulach DM, Bouchier C, Zuerner RL, Zidane N, et al.  (2008) Genome Sequence of the Saprophyte <em>Leptospira biflexa</em> Provides Insights into the Evolution of <em>Leptospira</em> and the Pathogenesis of Leptospirosis. PLoS ONE 3(2):
          e1607.
            doi:10.1371/journal.pone.0001607</p><p><strong>Academic Editor: </strong>Dana Davis, University of Minnesota, United States of America</p><p><strong>Received:</strong> November 16, 2007; <strong>Accepted:</strong> January 17, 2008; <strong>Published:</strong> February 13, 2008</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>This work was supported by the following: National Health and Medical Research Council, Australia; Australian Research Council; U.S. Department of Agriculture; Institut Pasteur, Paris, France; French Ministry of Research ANR Jeunes Chercheurs (no. 05-JCJC-0105-01); Consortium National de Recherche en Génomique (no. RNG-20040057); ACI IMPBio 2004 (MicroScope project). Funders had no role in the planning, performance or analysis of the work.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The genus <em>Leptospira</em> contains pathogenic and saprophytic species that differ in their capacity for survival in a vast array of environments that range from soil and water <a href="#pone.0001607-Trueba1">[1]</a> to the tissues of mammalian hosts during acute and chronic infection <a href="#pone.0001607-Faine1">[2]</a>. Typically, long term colonization by pathogenic <em>Leptospira</em> of the proximal renal tubules of mammalian maintenance host species provides a persistent source of infection; thus, pathogenic <em>Leptospira</em> is shed in the urine of chronically infected animals, facilitating transmission to naïve hosts <a href="#pone.0001607-Faine1">[2]</a>. Leptospirosis is of considerable importance to international public health, with more than half a million cases reported annually due largely to environmental exposure to pathogenic <em>Leptospira</em> species, with mortality rates of up to 25% in some outbreaks. In addition, leptospirosis in production animals results in a significant economic burden worldwide <a href="#pone.0001607-Faine1">[2]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Recent applications of molecular taxonomy techniques to this genus reveal extensive genetic diversity within <em>Leptospira</em>, with more than 16 pathogenic and saprophytic species recognized <a href="#pone.0001607-Brenner1">[3]</a>, <a href="#pone.0001607-Levett1">[4]</a>. A significant challenge in the future will be to more precisely correlate these genetic differences with biological differences. The relationship between leptospiral genome content, pathogenesis and the ability to survive in diverse environmental niches is a particularly important area of investigation, highlighted by our recent findings. A process of genome erosion and loss of gene function in <em>L. borgpetersenii</em> serovar Hardjo <a href="#pone.0001607-Bulach1">[5]</a> limits its viability outside a mammalian host and likely impairs disease transmission through water, a route commonly used by <em>L. interrogans</em> to infect new hosts.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To gain insight into the genetic potential of <em>Leptospira</em> and to help identify genes that contribute to long-term survival in surface water, we determined the genome sequences of two <em>L. biflexa</em> strains, the first saprophytic <em>Leptospira</em> species to be characterized by genome analysis. Comparison of these data to genomes of the pathogenic species <em>L. borgpetersenii</em> <a href="#pone.0001607-Bulach1">[5]</a>, and <em>L. interrogans</em> <a href="#pone.0001607-Nascimento1">[6]</a>, <a href="#pone.0001607-Ren1">[7]</a> provides an opportunity to identify features that are unique to pathogenic and saprophytic species, thereby providing new experimental directions and novel perspectives on leptospiral evolution, environmental persistence and the causation of disease.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4><em>Leptospira</em> genomes vary in replicon content and genetic organization</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The genome of <em>Leptospira biflexa</em> serovar Patoc strain Patoc1 (Ames strain) consists of three replicons with a total of 3,956,086 base pairs (bp) (<a href="#pone-0001607-g001">Figure 1</a>). The two larger replicons share extensive similarity to the two chromosomes that comprise the genomes of <em>L. borgpetersenii</em> and <em>L. interrogans</em> and are therefore referred to as chromosome I (CI; 3,603,977 bp; GC% 38.89) and chromosome II (CII; 277,995 bp; GC% 39.27). <em>L. biflexa</em> possesses a third circular replicon (74,114 bp; GC% 37.47), that we designate p74, not found in the previously sequenced pathogenic <em>Leptospira</em> species. (The complete genomic sequences of <em>L. biflexa</em> serovar Patoc strain Patoc1, strains Paris and Ames, have been deposited in GenBank under the Accession Numbers: CP000777, CP000778, CP000779, CP000786, CP000787 and CP000788).</p>
<div class="figure" id="pone-0001607-g001"><div class="img"><a name="pone-0001607-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001607.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001607.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Circular maps of the three <em>L. biflexa</em> replicons.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(1) the coordinates in bp beginning at 0 = oriC; (2) dark pink: genes unique to <em>L. biflexa</em>, not found in <em>L. interrogans</em> serovar Copenhageni and <em>L. borgpetersenii</em> serovar Hardjobovis (identity &gt;40% over 80% of the length of the smallest protein). (3) dark purple: genes found in <em>L. biflexa</em>, <em>L. interrogans</em> and <em>L. borgpetersenii</em> (identity &gt;40% over 80% of the length of the smallest protein). (4) red: genes found in <em>L. biflexa</em> and <em>L. borgpetersenii</em>, but not in <em>L. interrogans</em> (identity &gt;40% over 80% of the length of the smallest protein). (5) brown : genes found in <em>L. biflexa</em> and <em>L. interrogans</em>, but not in <em>L. borgpetersenii</em> (identity &gt;40% over 80% of the length of the smallest protein). (6) blue: genes found in <em>L. biflexa</em> and other sequenced spirochetes (<em>Borrelia afzelii</em> PKo, <em>Borrelia burgdorferi</em>, <em>Borrelia garinii</em>, <em>Treponema denticola</em> and <em>Treponema pallidum</em>) (identity &gt;40% over 80% of the length of the smallest protein ). (7) The innermost ring shows GC skew; positive skew is shown in grey, and negative skew is shown in black.</p>
<span>doi:10.1371/journal.pone.0001607.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>A total of 3,590 protein-coding genes (CDSs) was identified in <em>L. biflexa</em> (<a href="#pone-0001607-t001">Table 1</a>). Most of these genes are located on CI, including two <em>rrf</em> genes, two <em>rrl</em> genes and two <em>rrs</em> genes, coding for 5S, 23S and 16S rRNA molecules respectively. These rRNA genes are not linked to each other, a feature unusual among most bacteria but common among <em>Leptospira</em>. Similar to the slower-growing pathogenic species <em>L. interrogans</em> and <em>L. borgpetersenii,</em> each of which has a relatively low number (37) of transfer RNA (tRNA) genes, the faster-growing <em>L. biflexa</em> surprisingly has only 35 tRNA genes. This finding indicates that the growth rate of <em>Leptospira</em> is not restricted by the low number of tRNA and rRNA genes as previously suggested <a href="#pone.0001607-Ren1">[7]</a>, but rather may be due to other differing metabolic capacities amongst <em>Leptospira</em> spp. As reported for other <em>Leptospira</em> spp. <a href="#pone.0001607-Zuerner1">[8]</a>, essential genes such as <em>gltB</em> (glutamate synthase) and <em>asd</em> (aspartate semialdehyde dehydrogenase) are carried on CII in <em>L. biflexa</em>.</p>
<div class="figure" id="pone-0001607-t001"><div class="img"><a name="pone-0001607-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001607.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001607.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of genome features of pathogenic and saprophytic <em>Leptospira</em></span></strong></p><span>doi:10.1371/journal.pone.0001607.t001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>It is unclear if replicon III (p74) functions as a chromosome, i.e. carries genes essential for survival, or is an extrachromosomal element or plasmid. Thirteen genes on p74 have orthologs located on CI in pathogenic <em>Leptospira</em> (<a href="#pone-0001607-t002">Table 2</a>). For example, <em>recBCD</em> are located on CI in <em>L. interrogans</em> and <em>L. borgpetersenii,</em> but are located on p74 in <em>L. biflexa.</em> Mutation of these housekeeping genes in other bacterial species can affect transformation competence and viability <a href="#pone.0001607-Kickstein1">[9]</a>, suggesting that p74 is essential for the survival of <em>L. biflexa</em>.</p>
<div class="figure" id="pone-0001607-t002"><div class="img"><a name="pone-0001607-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001607.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001607.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>CDS from Replicon III (p74) that have an ortholog in Chromosome I in other <em>Leptospira</em></span></strong></p><span>doi:10.1371/journal.pone.0001607.t002</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>GC skew analysis (<a href="#pone-0001607-g001">Figure 1</a>) suggests that CI, CII and p74 are theta-type replicons that replicate bi-directionally from a unique origin. Each <em>L. biflexa</em> replicon encodes its own partition proteins from origin-proximal genes that may recognize and interact with a replication-specific binding site. The replication origin of CI resembles typical circular chromosomes from other bacteria, i.e. it is adjacent to <em>dnaA</em>, which encodes the initiator protein, and other genes such as <em>dnaN</em>, <em>recF</em> and <em>gyrAB</em>. In contrast, the replication origins of CII, replicon p74 and the previously reported leptophage LE1 <a href="#pone.0001607-Bourhy1">[10]</a> resemble phage and plasmid replicons; they contain both a partition operon and a downstream putative <em>rep</em> gene. For p74, the predicted replication origin based on GC skew analysis includes a partition operon (pLEPBI0001/LBF_5000 and pLEPBI0002/LBF_5001) and an adjacent gene, pLEPBI0003/LBF_5003, the product of which shares 63% similarity with the leptophage LE1 Rep protein. Regardless of the similarity between the LE1 Rep protein and the product of pLEPBI003, p74 and the LE1 prophage can co-exist (data not shown). Furthermore, we demonstrated that this region of p74 directs autonomous replication in <em>L. biflexa</em> (<a href="#pone.0001607.s001">Figure S1</a>). This will facilitate the construction of new shuttle vectors for the genetics of <em>Leptospira</em> and their use in co-transformation experiments with the <em>L. biflexa</em>-<em>E. coli</em> shuttle vector derived from LE1 <a href="#pone.0001607-SaintGirons1">[11]</a>.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>We suggest that the CII and p74 replicons evolved from a stabilized circular intermediate of a progenitor phage related to LE1 <a href="#pone.0001607-Bourhy1">[10]</a>. Intermediates in this process may be related to the LaiGI-1 Genomic Island <a href="#pone.0001607-Bourhy2">[12]</a>, which exists both as an element integrated into CI of <em>L. interrogans</em> serovar Lai or as an autonomously replicating plasmid. Stabilization of these intermediates could have occurred through illegitimate recombination resulting in incorporation of essential genes from CI into the smaller replicons. Evidence for this model is provided by the presence of genes located on the CI replicon in pathogenic <em>Leptospira</em> on either CII or p74 in <em>L. biflexa</em>, as noted above.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>The genus <em>Leptospira</em> is renowned for the stability of its agglutinating antigens during <em>in vitro</em> culture, with examples of strains maintaining serovar identity during more than 80 years of propagation <a href="#pone.0001607-Faine1">[2]</a>, implying considerable genomic stability in the absence of selective pressure for antigenic change. Furthermore, more than 2% of the <em>L. interrogans</em> serovar Copenhageni genome is dedicated to LPS biosynthesis. In <em>L. biflexa</em> about 1.4% of the genome encodes LPS biosynthesis functions with essentially all relevant CDS encoded on the same strand, and these genes are identical in the Ames and Paris strains. In contrast, non-LPS encoding regions in the two <em>L. biflexa</em> strains maintained separately for 17 years show some evidence of minor accumulation of changes. Comparison of the Paris and Ames strains of <em>L. biflexa</em> reveals four indels, three of which are insertions of IS<em>Lbi1</em> elements into the coding regions of genes in the Ames strain (LBF_0259, LBF_2295 and LBF_2512). The first two genes encode proteins of unknown function, and the third encodes a protein with a predicted role in lipid metabolism; the significance of these insertions is not known. In the Paris strain, the insertion element is restricted to one copy each on CII and p74; these copies are maintained in the Ames strain. The fourth difference occurs in CII in which an additional 250 bp are found in an intergenic region in the Ames strain. Aside from these differences, the genomes of the Paris and Ames strains are virtually identical; in this article the term <em>L. biflexa</em> will refer to the Ames strain unless stated otherwise.</p>
<a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>Approximately two thirds of the genes in <em>L. biflexa</em> have orthologous genes in the pathogens <em>L. interrogans</em> and <em>L. borgpetersenii</em> (<a href="#pone-0001607-g002">Figure 2</a>), consistent with a common origin for leptospiral saprophytes and pathogens. The genes conserved across the three species are distributed in the two chromosomes of all <em>Leptospira</em> species and strains (<a href="#pone-0001607-t003">Table 3</a>). The high sequence identity of the small chromosome <em>parA</em> gene product (&gt;87%) between all <em>Leptospira</em> spp. suggests a shared ancestry for this replicon. Moreover, there is no difference in inter-chromosomal DNA parameters such as GC%, codon preferences and gene density within the three <em>L. biflexa</em> replicons, despite the relatively wide range of these parameters in pathogenic <em>Leptospira</em>. By contrast, codon preferences of genes of the leptospiral bacteriophage LE1 and its host, <em>L. biflexa</em> <a href="#pone.0001607-Bourhy1">[10]</a> are different (data not shown). These data suggest a long-standing relationship between the three <em>L. biflexa</em> replicons and perhaps a more recent acquisition of LE1.</p>
<div class="figure" id="pone-0001607-g002"><div class="img"><a name="pone-0001607-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001607.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001607.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Venn diagram showing numbers of unique and shared genes amongst <em>L. interrogans</em>, <em>L. borgpetersenii</em> and <em>L. biflexa.</em></span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Orthologous CDS were identified in a pair-wise fashion using Whole-Genome Reciprocal Best-Hit BLAST Analysis <a href="#pone.0001607-Fuchsman1">[37]</a>. Manual curation ensured a one to one relationship for orthologous CDS, particularly in situations where sets of paralogous CDS existed and in addition evaluated the nature of the relationship between CDS with reciprocal best-hits but low expect values. This analysis was performed using the <em>L. interrogans</em> serovar Copenhageni strain Fiocruz, <em>L. borgpetersenii</em> serovar Hardjo strain L550 and <em>L. bifle</em>xa serovar Patoc strain Ames genome sequences.</p>
<span>doi:10.1371/journal.pone.0001607.g002</span></div><div class="figure" id="pone-0001607-t003"><div class="img"><a name="pone-0001607-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001607.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001607.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Distribution of the orthologs over the two chromosomes of <em>Leptospira</em> spp.</span></strong></p><span>doi:10.1371/journal.pone.0001607.t003</span></div>

<h4>The evolution of the <em>Leptospira</em> species</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Phylogenetic analysis based on comparison of 16S rRNA sequences indicates that members of the family <em>Leptospiraceae</em> form the deepest branch in spirochete evolution, with divergence of saprophytic and pathogenic <em>Leptospira</em> likely being the result of a single event <a href="#pone.0001607-Paster1">[13]</a>. Further diversification of species within each of these two evolutionary branches of <em>Leptospira</em> is supported by multilocus sequence analysis and DNA renaturation kinetics <a href="#pone.0001607-Brenner1">[3]</a>, <a href="#pone.0001607-Haake1">[14]</a>, <a href="#pone.0001607-Ramadass1">[15]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We used comparative genome analysis to help identify key features showing patterns of variation consistent with the action of selective evolutionary pressure within <em>Leptospira</em>. The <em>L. biflexa</em> and <em>L. borgpetersenii</em> genomes are similar in size (<a href="#pone-0001607-t001">Table 1</a>), but the gene density in <em>L. biflexa</em> is much higher, probably as a result of IS-mediated genome erosion in <em>L. borgpetersenii</em>. In contrast, the <em>L. interrogans</em> genome is larger, probably reflecting the added genetic information required for survival both within mammalian hosts and aquatic environments, whereas <em>L. biflexa</em> and <em>L. borgpetersenii</em> are restricted to aquatic and mammalian host environments, respectively. The dearth of IS-elements in the <em>L. biflexa</em> genome (five IS-elements) is in stark contrast to their abundance in the <em>L. interrogans</em> (36 and 69 IS-elements in the Fiocruz LI-130 and Lai 56601 strains, respectively) and <em>L. borgpetersenii</em> (167 IS elements) genomes. The presence of large numbers of IS elements is an indicator of genome plasticity in <em>Leptospira</em> species. Taken together, our results suggest that the <em>L. biflexa</em> gene order is more likely to have a closer relationship to the progenitor genome for the genus <em>Leptospira</em>. There is low synteny between the sequenced pathogenic <em>Leptospira</em> (<a href="#pone-0001607-g003">Figure 3</a>) <a href="#pone.0001607-Bulach1">[5]</a>, despite the short evolutionary distance separating them.</p>
<div class="figure" id="pone-0001607-g003"><div class="img"><a name="pone-0001607-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001607.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001607.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Synteny plot between the five <em>Leptospira</em> genomes.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The line plots were obtained using synteny results between the large CI(A) or small CII(B) chromosomes of <em>L. biflexa</em> serovar Patoc strain Patoc1, <em>L. interrogans</em> serovar Lai strain 56601, <em>L. interrogans</em> serovar Copenhageni strain Fiocruz L1-130, <em>L. borgpetersenii</em> serovar Hardjo strain L550, and <em>L. borgpetersenii</em> serovar Hardjo strain JB197. A line plot (C) compares synteny between <em>L. borgpetersenii</em> serovar Hardjo strain JB197 and <em>L. interrogans</em> serovar Copenhageni strain Fiocruz L1-130. Comparative analysis was performed using the MaGe interface <a href="#pone.0001607-Bocs1">[35]</a> in the SpiroScope database (<a href="https://www.genoscope.cns.fr/agc/mage">https://www.genoscope.cns.fr/agc/mage</a>). The minimum size of the synteny groups is set to five genes. In green: synteny groups are organized on the same strand; in red: synteny groups are organized on opposite strands.</p>
<span>doi:10.1371/journal.pone.0001607.g003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Gene and functional redundancy is also more common in the pathogens in comparison to the saprophytes (<a href="#pone-0001607-t004">Table 4</a>). The pathogens have more paralogs (excluding transposases), with 203 (5.01%) and 438 (10.52%) paralogs for <em>L. interrogans</em> serovar Copenhageni Fiocruz L1-130 and <em>L. borgpetersenii</em> serovar Hardjo L550, respectively, compared to 62 paralogs (1.65%) in the saprophyte <em>L. biflexa</em>. We identified 43, 67, and 53 loci with gene duplication in the genomes of <em>L. biflexa</em>, <em>L. interrogans</em>, and <em>L. borgpetersenii</em>, respectively. The genome of <em>L. biflexa</em> therefore appears to be more stable than those of <em>L. interrogans</em> and <em>L. borgpetersenii</em>. Moreover, the <em>L. biflexa</em> genome (92%) has a greater gene density than the <em>L. borgpetersenii</em> (80%) and <em>L. interrogans</em> genomes (75%) (<a href="#pone-0001607-t001">Table 1</a>). Greater gene density can contribute to a relatively stable gene order. For example, the spirochete <em>Borrelia</em> has a gene density of more than 92% and the <em>B. garinii</em> and <em>B. burgdorferi</em> genomes are essentially collinear <a href="#pone.0001607-Glockner1">[16]</a>.</p>
<div class="figure" id="pone-0001607-t004"><div class="img"><a name="pone-0001607-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001607.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001607.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Comparative genomics of <em>Leptospira</em> spp. Putative orthologous relations between two genomes are defined as gene couples satisfying the bi-directional best hit (BBH) criterion or a blastP alignment threshold, a minimum of 40% sequence identity on 80% of the length of the smallest protein. Putative paralogous relations between two genomes are defined as gene couples satisfying the bi-directional best hit (BBH) criterion or a blastP alignment threshold, a minimum of 60% sequence identity on 80% of the length of the smallest protein.</span></strong></p><span>doi:10.1371/journal.pone.0001607.t004</span></div>

<h4>Genetic determinants involved in survival in the environment</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The spectrum of ecological niches occupied by diverse <em>Leptospira</em> species raises questions as to how the capacity to survive in these diverse environments has evolved within different species in the genus <em>Leptospira</em>. We propose that their common progenitor had a genome more like <em>L. biflexa</em>. Subsequently, acquisition of genes that enabled <em>Leptospira</em> to infect mammals would have expanded the range of environments that it could successfully occupy. Loss of environmental survival genes then would lead to dependence on a mammalian host and eventually return the genome to a smaller size (i.e. <em>L. borgpetersenii</em>), consistent with our model for IS-mediated genome degradation <a href="#pone.0001607-Bulach1">[5]</a>.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The extensive repertoire of genes (<a href="#pone-0001607-t005">Table 5</a>) encoding proteins involved in signal transduction in <em>L. biflexa</em> (287 CDS) compared with <em>L. interrogans</em> (214 CDS) and <em>L. borgpetersenii</em> (167 CDS) is consistent with an enhanced metabolic capability in <em>L. biflexa</em> reflected by its environmental habitat and likely contributes to its enhanced growth rate relative to the pathogens. An analogous situation is seen in <em>Mycobacterium</em>, where <em>M. marinum</em> which occupies both animal and aquatic environments has many more genes encoding environmental sensing and metabolic proteins than the closely related, obligate human pathogen <em>M. tuberculosis</em> <a href="#pone.0001607-Stamm1">[17]</a>, <a href="#pone.0001607-Stinear1">[18]</a>.</p>
<div class="figure" id="pone-0001607-t005"><div class="img"><a name="pone-0001607-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001607" data-uri="info:doi/10.1371/journal.pone.0001607.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001607.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001607.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001607.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001607.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001607.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Distribution of general protein functions between leptospiral species based on the COG function classification scheme.</span></strong></p><span>doi:10.1371/journal.pone.0001607.t005</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The genomes of <em>Leptospira</em> spp. contain a number of genes involved in the production of exopolysaccharides. These genes, such as those encoding glycosyltransferases, alginate biosynthesis, and lipopolysaccharide transport systems, may contribute to colonization of both biotic and abiotic surfaces. When investigating the formation of biofilms on solid surfaces, we have observed the production of a strong biofilm by <em>L. biflexa</em> and <em>L. interrogans</em> (Unpublished results). The formation of such a biofilm is consistent with the life of saprophytic species in water; it may facilitate <em>L. biflexa</em> occupying particular environmental niches. The presence of biofilms may also play an important role in chronic carriage of the pathogen <em>L. interrogans</em> in animal reservoirs by facilitating colonization of the renal tubules. Interestingly, genes involved in alginate biosynthesis are present in both <em>L. biflexa</em> (11 genes) and <em>L. interrogans</em> (8 genes), but are absent in <em>L. borgpetersenii</em>, a finding consistent with the reduced environmental survival of <em>L. borgpetersenii</em> <a href="#pone.0001607-Bulach1">[5]</a>.</p>


<h4>Comparative genomics of pathogenic and saprophytic <em>Leptospira</em></h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The molecular mechanisms of pathogenesis in leptospirosis remain almost entirely unknown. The set of leptospiral, pathogen-specific genes, defined as those with no orthologous gene in <em>L. biflexa</em>, is likely to be enriched for genes that play a role in pathogenesis. Significantly, the majority (893) of these 1,431 genes has no known function (<a href="#pone.0001607.s002">Table S1abc</a>), suggesting the presence of pathogenic mechanisms unique to <em>Leptospira</em>. Among the remaining 538 genes, there are several genes encoding response regulators and environment sensing proteins; these genes are likely to represent adaptations that enable survival in environments not encountered by the saprophytic strain. There is an expansion of genes encoding leucine–rich repeat (LLR) proteins from one gene in <em>L. biflexa</em> to eight and 18 genes in <em>L. borgpetersenii</em> and <em>L. interrogans</em>, respectively. Although these LLR proteins have no obvious function, in <em>Treponema denticola</em> the LRR protein LrrA appears to have roles in attachment and to, and penetration of, host tissues <a href="#pone.0001607-Ikegami1">[19]</a>; the diversity of LRR proteins in pathogenic <em>Leptospira</em> may be important for succesful infection of a wide variety of mammalian host species.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The regulation of transcription differs significantly between the saprophytic and pathogenic strains as indicated by the presence of more than 50 saprophyte-specific, and more than 20-pathogen specific, transcription regulators, whereas there are only 27 transcription regulators that are common to all of the sequenced <em>Leptospira</em> species.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Because the bacterial surface is an interface between the pathogen and the host, any differences in cell-surface proteins might reflect variation in pathogenesis mechanisms among <em>Leptospira</em> spp., which contain a relatively low repertoire of trans-membrane proteins in their outer membranes <a href="#pone.0001607-Cullen1">[20]</a>. However, the outer membranes contain a predominance of lipoproteins, that may be either surface-exposed or located in the periplasm. Predicted lipoproteins are prominent in both saprophytic (164 predicted lipoproteins) and pathogenic <em>Leptospira</em> (184 and 130 predicted lipoproteins in <em>L. interrogans</em> and <em>L. borgpetersenii</em> respectively). Despite these similarities, there are significant differences, most notably the absence in <em>L. biflexa</em> of an ortholog of the major outer membrane lipoprotein, LipL32 <a href="#pone.0001607-Cullen1">[20]</a>. Moreover, 89 <em>L. biflexa</em> lipoprotein genes have no orthologs in other <em>Leptospira</em> species, and more than 90 lipoproteins from the pathogenic species have no orthologs in the <em>L. biflexa</em> genome. In addition to LipL32, other characterized lipoproteins that have no orthologs in <em>L. biflexa</em> include LipL41, LipL36 and several LipL45 related proteins <a href="#pone.0001607-Cullen1">[20]</a>. However, LipL21, which was reported as a pathogen-specific lipoprotein <a href="#pone.0001607-Cullen2">[21]</a> based on antibody reactivity, has an ortholog in <em>L. biflexa</em> with 50% similarity. This level of similarity is consistent with a different function for the LipL21 orthologs in the two <em>Leptospira</em> species.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>Several putative virulence factors previously identified in pathogenic <em>Leptospira</em> spp. are not present in <em>L. biflexa</em>, including the Lig surface proteins containing immunoglobulin-like repeats predicted to play a role in the adhesion to host tissues <a href="#pone.0001607-Choy1">[22]</a>. Similarly, LfhA, a putative factor H binding protein <a href="#pone.0001607-Verma1">[23]</a> that has also been shown to bind the extracellular matrix protein laminin <a href="#pone.0001607-Barbosa1">[24]</a>, is shared among pathogenic <em>Leptospira</em>, but is lacking in <em>L. biflexa</em>. Although the genome of <em>L. biflexa</em> contains putative hemolysins <a href="#pone.0001607-Louvel1">[25]</a>, its genome is devoid of genes encoding enzymes capable of degrading tissues, such as the range of sphingomyelinases found in pathogenic species <a href="#pone.0001607-Segers1">[26]</a>, <a href="#pone.0001607-Segers2">[27]</a>, <a href="#pone.0001607-delReal1">[28]</a>. The role of sphingomyelinases in the pathogenesis of leptospirosis has been controversial; are they key virulence factors or do they merely play a role in nutrient acquisition? Their absence in <em>L. biflexa</em> strongly supports their involvement in survival within mammalian hosts. Interestingly, the membrane protein, Loa22, the only protein to date that has been shown genetically to be required for virulence in <em>L. interrogans</em> <a href="#pone.0001607-Ristow1">[29]</a>, has a <em>L. biflexa</em> ortholog with 73% similarity. Its role in either pathogenic or saprophytic species is unknown, but its presence in the saprophytic species suggests that it is involved in survival rather than being a direct virulence factor and is consistent with the common progenitor hypothesis.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p><em>L. interrogans</em> is the most frequently reported agent of human leptospirosis. The disease is also generally more severe with <em>L. interrogans</em> than with <em>L. borgpetersenii</em> <a href="#pone.0001607-Faine1">[2]</a>. On this basis we propose that the subset of <em>L. interrogans</em> genes that have no orthologs in either <em>L. biflexa</em> or <em>L. borgpetersenii</em> may contain virulence factors that are responsible for the more severe form of leptospirosis. Other subsets that may be enriched for genes involved in particular biological functions include those genes that have orthologs in <em>L. biflexa</em> and <em>L. interrogans</em> and not <em>L. borgpetersenii</em> which may contain genes involved in survival outside the animal host. While the loss of many genes from the <em>L. borgpetersenii</em> genome has occurred through genome reduction <a href="#pone.0001607-Bulach1">[5]</a>, the presence of 265 unique genes in <em>L. borgpetersenii</em> (<a href="#pone-0001607-g002">Figure 2</a>; <a href="#pone.0001607.s002">Table S1abc</a>) indicates that these genomes have also gained some additional genes during the course of their evolution. The lateral acquisition of genetic material is often associated with IS-elements. While there is an association between IS-elements and genes that are isolate- or species-specific, in particular <em>in L. interrogans</em>, no mechanism has been determined. Other possible mechanisms for horizontal gene acquisition may include the involvement of bacteriophage, as integrated prophages or bacteriophage remnants are present.</p>


<h4>The Core Leptospiral Genes</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Saprophytic and pathogenic <em>Leptospira</em> species belong to two distinct phylogenetic groups, leading us to conclude that the 2,052 genes shared by both groups constitute the core genome of this genus (<a href="#pone-0001607-g002">Figure 2</a>). As expected, many of the functional categories that are involved in essential housekeeping functions, such as DNA and RNA metabolism, protein processing and secretion, cell structure, cellular processes, and energetic and intermediary metabolism, are represented in the core gene set.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The presence of orthologous genes in all the sequenced leptospiral species is an indicator that these genes were acquired prior to the radiation of the genus, and as such, is a strong indicator that these genes have not been laterally acquired. The substantial proportion of the total leptospiral genes that are in this category is perhaps an indicator that lateral transfer of genes into the genus <em>Leptospira</em> is a minor contributor to the overall genetic composition of the genus and an indicator that the genus has undergone an extended period of ‘genetic isolation’. This notion is supported by the fact that approximately 20% of the core leptospiral genes are unique to the genus. However, this does not preclude horizontal acquisition of some genes or gene clusters.</p>


<h4>Implications for genetic studies</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Despite the development of basic tools such as transposon mutagenesis for the pathogenic species of <em>Leptospira</em>, targeted gene inactivation is not yet possible. Therefore, there is likely to be continued interest in the use of <em>L. biflexa</em> as a model bacterium for genetic analysis <a href="#pone.0001607-Louvel2">[30]</a>. Knowledge of the distribution of orthologous genes in <em>L. biflexa</em> will be an important resource for the elucidation of function for genes common to pathogenic and saprophytic species.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sequencing and annotation of the genome of <em>L. biflexa</em></h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The strain, <em>L. biflexa</em> serovar Patoc strain Patoc1, was initially isolated from stream water <a href="#pone.0001607-Babudieri1">[31]</a>, maintained in the collection of the National Reference Center of <em>Leptospira</em> (Institut Pasteur, Paris, France) and designated the Paris strain. A second strain was derived from the same source but kept in the culture collection at the National Animal Disease Center (NADC), Ames, IA since 1990 and is designated the Ames strain. Each strain was colony purified before growth for genomic DNA isolation. For each isolate, genomic DNA was randomly sheared by nebulization (HydroShear, GeneMachines) and the ends repaired enzymatically. Small fragments (~1.5–4 kb) were ligated either to a derivative of plasmid pGEM7Zf+ (Promega) (Paris strain) or pSMART-HC and pSMART-LC (Lucigen Corp.) (Ames strain). Large (35–45 kb) DNA fragments were ligated to fosmid pCC1FOS (Epicentre, Madison, WI) (Paris strain). Intermediate sized DNA fragments (7–12 kb) were prepared from partially <em>Bam</em>HI-digested Ames strain DNA and ligated into pZERO-1 (Invitrogen). Plasmid DNA preparation was performed with the TempliPhi DNA sequencing template amplification kit (GE Healthcare -Bio-Sciences) or SprintPrep plasmid preparation kits (Agencourt Bioscience). Fosmid DNA purification was performed with the Montage BAC Miniprep96 Kit (Millipore) <a href="#pone.0001607-Louvel1">[25]</a>. Sequencing reactions were performed from both ends of DNA templates using ABI PRISM BigDye Terminator cycle sequencing ready reactions kits and run on a 3700 or a 3730 xl Genetic Analyzer (Applied Biosystems) at the Genomics platform (Pasteur Genopole Île-de-France), the Australian Genome Research Facility, Brisbane, Australia or at the Genomics Facility at the NADC. Base-calls from sequence data were made using Phred <a href="#pone.0001607-Ewing1">[32]</a>. Sequences not meeting our production quality criteria (at least 100 bases called with a quality over 20) were discarded. The traces were assembled using Phrap and Consed <a href="#pone.0001607-Gordon1">[33]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Whole genome shotgun sequencing was performed until approximately 6× genome coverage was achieved. Autofinish <a href="#pone.0001607-Gordon2">[34]</a> was used to design primers for improving regions of low quality sequence and for primer walking along templates that spanned the gaps between contigs. Several strategies were used to orientate contigs to enable directed PCR-based approaches to span gaps between contigs. These strategies included Blast-based approaches that identified contig ends with hits to the same gene or to genes within the same locus, provided that there was conservation of gene order in both the <em>L. interrogans</em> and the <em>L. borgpertersenii</em> genomes. Repeated sequences longer than 600 bases, such as IS-elements and the <em>rrl</em> and <em>rrn</em> genes, were identified and curated manually to ensure that at least two templates spanned these regions, thus confirming the assembly of these regions. At this stage, single, circularised contigs representing CII and and p74 were identified. Combinational PCR was used to close the gaps between the final contigs making up CI. Outward-directed primers were designed for each of the contig ends; the primer sequences were subsequently checked and confirmed to be unique on the genome. The combinational PCR process required approximately 600 PCR reactions pairing each of the primers. In each instance where a gap was spanned, the size of the PCR product was less than 2 kb. The sequence of these PCR products was determined and added to the assembly to enable the closing of CI. PCR with independent primers was used to confirm the joins determined by the combinational PCR. In addition, for the Paris strain five fosmid clones were completely sequenced by transposon-assisted sequencing (Finnzymes, TGS II kit). Seven fosmid clones were also selected by Southern hybridization and sequenced in order to verify the 23S RNA and 16S RNA regions and 4 repeat regions. Validation of the final assembly was achieved by comparison of <em>in silico</em> digestion patterns with macrorestriction patterns obtained by PFGE with <em>Not</em>I and <em>Asc</em>I. For each genome, the error rate was less than 1 error per 10,000 bp.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>The complete genome sequence was obtained from 58,663 and 40,260 sequences for the Paris and Ames strains respectively (giving &gt;8× and 7× coverage). For the Paris strain, coding sequences (CDSs) likely to encode proteins were predicted with the AMIGene software <a href="#pone.0001607-Bocs1">[35]</a>. Annotation was performed as described previously <a href="#pone.0001607-Louvel1">[25]</a> using the MaGe annotation platform <a href="#pone.0001607-Vallenet1">[36]</a>. All the data were stored in SpiroScope, a relational database which is publicly available (<a href="http://www.genoscope.cns.fr/agc/mage">http://www.genoscope.cns.fr/agc/mage</a>). The Ames strain was annotated as described previously for the <em>L. borgpetersenii</em> serovar Hardjo genome using the Wasabi interactive platform <a href="#pone.0001607-Bulach1">[5]</a>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001607"></div><a name="pone.0001607.s001" id="pone.0001607.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001607.s001">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001607.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0001607.s002" id="pone.0001607.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001607.s002">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Supplementary <a href="#pone-0001607-t001">Table 1</a> a, b and c</p>
<p class="siDoi">doi:10.1371/journal.pone.0001607.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.46 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank David Alt, Richard Hornsby, A. Magnier and A. Blanchard for technical support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BA CB MP RZ DB PW EK AM CB-E. Performed the experiments: CM ZR NZ SC SB JD MJ AL. Analyzed the data: RC BA JR JD MP RZ DB AM TS. Wrote the paper: BA JR MP RZ DB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001607-Trueba1" id="pone.0001607-Trueba1"></a>Trueba G, Zapata S, Madrid K, Cullen P, Haake D (2004) Cell aggregation: a mechanism of pathogenic <em>Leptospira</em> to survive in fresh water. International Microbiology  7: 35–40.  <ul class="find" data-citedArticleID="1056682"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cell+aggregation%3A+a+mechanism+of+pathogenic+Leptospira+to+survive+in+fresh+water.&amp;auth=&amp;atitle=Cell+aggregation%3A+a+mechanism+of+pathogenic+Leptospira+to+survive+in+fresh+water." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+aggregation%3A+a+mechanism+of+pathogenic+Leptospira+to+survive+in+fresh+water." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+aggregation%3A+a+mechanism+of+pathogenic+Leptospira+to+survive+in+fresh+water.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001607-Faine1" id="pone.0001607-Faine1"></a>Faine S, Adler B, Bolin C, Perolat P (1999) <em>Leptospira</em> and Leptospirosis. Melbourne, Australia: MediSci.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001607-Brenner1" id="pone.0001607-Brenner1"></a>Brenner DJ, Kaufmann AF, Sulzer KR, Steigerwalt AG, Rogers FC, et al.  (1999) Further determination of DNA relatedness between serogroups and serovars in the family <em>Leptospiraceae</em> with a proposal for <em>Leptospira alexanderi</em> sp. nov. and four new <em>Leptospira</em> genomospecies. International Journal of Systematic Bacteriology  49: 839–858.  <ul class="find" data-citedArticleID="1056626" data-doi="10.1099/00207713-49-2-839"><li><a href="http://dx.doi.org/10.1099/00207713-49-2-839" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Further+determination+of+DNA+relatedness+between+serogroups+and+serovars+in+the+family+Leptospiraceae+with+a+proposal+for+Leptospira+alexanderi+sp.+nov.+and+four+new+Leptospira+genomospecies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Further+determination+of+DNA+relatedness+between+serogroups+and+serovars+in+the+family+Leptospiraceae+with+a+proposal+for+Leptospira+alexanderi+sp.+nov.+and+four+new+Leptospira+genomospecies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001607-Levett1" id="pone.0001607-Levett1"></a>Levett PN (2001) Leptospirosis. Clinical Microbiology Reviews  14: 296–326.  <ul class="find" data-citedArticleID="1056654" data-doi="10.1128/cmr.14.2.296-326.2001"><li><a href="http://dx.doi.org/10.1128/cmr.14.2.296-326.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Leptospirosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Leptospirosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001607-Bulach1" id="pone.0001607-Bulach1"></a>Bulach DM, Zuerner RL, Wilson P, Seemann T, McGrath A, et al.  (2006) Genome reduction in <em>Leptospira borgpetersenii</em> reflects limited transmission potential. Proceedings of the National Academy of Sciences of the United States of America  103: 14560–14565.  <ul class="find" data-citedArticleID="1056628" data-doi="10.1073/pnas.0603979103"><li><a href="http://dx.doi.org/10.1073/pnas.0603979103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+reduction+in+Leptospira+borgpetersenii+reflects+limited+transmission+potential." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+reduction+in+Leptospira+borgpetersenii+reflects+limited+transmission+potential.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001607-Nascimento1" id="pone.0001607-Nascimento1"></a>Nascimento ALTO, Ko AI, Martins EAL, Monteiro-Vitorello CB, Ho PL, et al.  (2004) Comparative genomics of two <em>Leptospira interrogans</em> serovars reveals novel insights into Physiology and Pathogenesis. Journal of Bacteriology  186: 2164–2172.  <ul class="find" data-citedArticleID="1056660" data-doi="10.1128/jb.186.7.2164-2172.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.7.2164-2172.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomics+of+two+Leptospira+interrogans+serovars+reveals+novel+insights+into+Physiology+and+Pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomics+of+two+Leptospira+interrogans+serovars+reveals+novel+insights+into+Physiology+and+Pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001607-Ren1" id="pone.0001607-Ren1"></a>Ren SX, Fu G, Jiang XG, Zeng R, Miao YG, et al.  (2003) Unique physiological and pathogenic features of <em>Leptospira interrogans</em> revealed by whole-genome sequencing. Nature  422: 888–893.  <ul class="find" data-citedArticleID="1056666" data-doi="10.1038/nature01597"><li><a href="http://dx.doi.org/10.1038/nature01597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unique+physiological+and+pathogenic+features+of+Leptospira+interrogans+revealed+by+whole-genome+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unique+physiological+and+pathogenic+features+of+Leptospira+interrogans+revealed+by+whole-genome+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001607-Zuerner1" id="pone.0001607-Zuerner1"></a>Zuerner RL, Herrmann JL, Saint Girons I (1993) Comparison of genetic maps for two <em>Leptospira interrogans</em> serovars provides evidence for two chromosomes and intraspecies heterogeneity. Journal of Bacteriology  175: 5445–5451.  <ul class="find" data-citedArticleID="1056688"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparison+of+genetic+maps+for+two+Leptospira+interrogans+serovars+provides+evidence+for+two+chromosomes+and+intraspecies+heterogeneity.&amp;auth=&amp;atitle=Comparison+of+genetic+maps+for+two+Leptospira+interrogans+serovars+provides+evidence+for+two+chromosomes+and+intraspecies+heterogeneity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+genetic+maps+for+two+Leptospira+interrogans+serovars+provides+evidence+for+two+chromosomes+and+intraspecies+heterogeneity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+genetic+maps+for+two+Leptospira+interrogans+serovars+provides+evidence+for+two+chromosomes+and+intraspecies+heterogeneity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001607-Kickstein1" id="pone.0001607-Kickstein1"></a>Kickstein E, Harms K, Wackernagel W (2007) Deletions of <em>recBCD</em> or <em>recD</em> influence genetic transformation differently and are lethal together with a <em>recJ</em> deletion in <em>Acinetobacter bayly</em>i. Microbiology  153: 2259–2270.  <ul class="find" data-citedArticleID="1056652" data-doi="10.1099/mic.0.2007/005256-0"><li><a href="http://dx.doi.org/10.1099/mic.0.2007/005256-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deletions+of+recBCD+or+recD+influence+genetic+transformation+differently+and+are+lethal+together+with+a+recJ+deletion+in+Acinetobacter+baylyi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deletions+of+recBCD+or+recD+influence+genetic+transformation+differently+and+are+lethal+together+with+a+recJ+deletion+in+Acinetobacter+baylyi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001607-Bourhy1" id="pone.0001607-Bourhy1"></a>Bourhy P, Frangeul L, Couve E, Glaser P, Saint Girons I, et al.  (2005) Complete nucleotide sequence of the LE1 prophage from the spirochete <em>Leptospira biflexa</em> and characterization of Its replication and partition functions. Journal of Bacteriology  187: 3931–3940.  <ul class="find" data-citedArticleID="1056622" data-doi="10.1128/jb.187.12.3931-3940.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.12.3931-3940.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+nucleotide+sequence+of+the+LE1+prophage+from+the+spirochete+Leptospira+biflexa+and+characterization+of+Its+replication+and+partition+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+nucleotide+sequence+of+the+LE1+prophage+from+the+spirochete+Leptospira+biflexa+and+characterization+of+Its+replication+and+partition+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001607-SaintGirons1" id="pone.0001607-SaintGirons1"></a>Saint Girons I, Bourhy P, Ottone C, Picardeau M, Yelton D, et al.  (2000) The LE1 bacteriophage replicates as a plasmid within <em>Leptospira biflexa</em>: construction of an <em>L. biflexa</em>-<em>Escherichia coli</em> shuttle vector. Journal of Bacteriology  182: 5700–5705.  <ul class="find" data-citedArticleID="1056670" data-doi="10.1128/jb.182.20.5700-5705.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.20.5700-5705.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+LE1+bacteriophage+replicates+as+a+plasmid+within+Leptospira+biflexa%3A+construction+of+an+L.+biflexa-Escherichia+coli+shuttle+vector." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+LE1+bacteriophage+replicates+as+a+plasmid+within+Leptospira+biflexa%3A+construction+of+an+L.+biflexa-Escherichia+coli+shuttle+vector.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001607-Bourhy2" id="pone.0001607-Bourhy2"></a>Bourhy P, Salaun L, Lajus A, Medigue C, Boursaux-Eude C, et al.  (2007) A Genomic island of the pathogen <em>Leptospira interrogans s</em>erovar Lai can excise from its chromosome. Infection and Immunity  75: 677–683.  <ul class="find" data-citedArticleID="1056624" data-doi="10.1128/iai.01067-06"><li><a href="http://dx.doi.org/10.1128/iai.01067-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Genomic+island+of+the+pathogen+Leptospira+interrogans+serovar+Lai+can+excise+from+its+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Genomic+island+of+the+pathogen+Leptospira+interrogans+serovar+Lai+can+excise+from+its+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001607-Paster1" id="pone.0001607-Paster1"></a>Paster BJ, Dewhirst FE, Weisburg WG, Tordoff LA, Fraser GJ, et al.  (1991) Phylogenetic analysis of the spirochetes. Journal of Bacteriology  173: 6101–6109.  <ul class="find" data-citedArticleID="1056662"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phylogenetic+analysis+of+the+spirochetes.&amp;auth=&amp;atitle=Phylogenetic+analysis+of+the+spirochetes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+analysis+of+the+spirochetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+analysis+of+the+spirochetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001607-Haake1" id="pone.0001607-Haake1"></a>Haake DA, Suchard MA, Kelley MM, Dundoo M, Alt DP, et al.  (2004) Molecular evolution and mosaicism of leptospiral outer membrane proteins involves horizontal DNA transfer. Journal of Bacteriology  186: 2818–2828.  <ul class="find" data-citedArticleID="1056648" data-doi="10.1128/jb.186.9.2818-2828.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.9.2818-2828.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolution+and+mosaicism+of+leptospiral+outer+membrane+proteins+involves+horizontal+DNA+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolution+and+mosaicism+of+leptospiral+outer+membrane+proteins+involves+horizontal+DNA+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001607-Ramadass1" id="pone.0001607-Ramadass1"></a>Ramadass P, Jarvis BD, Corner RJ, Penny D, Marshall RB (1992) Genetic characterization of pathogenic <em>Leptospira</em> species by DNA hybridization. International Journal of Systematic Bacteriology  42: 215–219.  <ul class="find" data-citedArticleID="1056664" data-doi="10.1099/00207713-42-2-215"><li><a href="http://dx.doi.org/10.1099/00207713-42-2-215" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+characterization+of+pathogenic+Leptospira+species+by+DNA+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+characterization+of+pathogenic+Leptospira+species+by+DNA+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001607-Glockner1" id="pone.0001607-Glockner1"></a>Glockner G, Lehmann R, Romualdi A, Pradella S, Schulte-Spechtel U, et al.  (2004) Comparative analysis of the <em>Borrelia garinii</em> genome. Nucleic Acids Research  32: 6038–6046.  <ul class="find" data-citedArticleID="1056642" data-doi="10.1093/nar/gkh953"><li><a href="http://dx.doi.org/10.1093/nar/gkh953" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+the+Borrelia+garinii+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+the+Borrelia+garinii+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001607-Stamm1" id="pone.0001607-Stamm1"></a>Stamm LM, Brown EJ (2004) <em>Mycobacterium marinum</em>: the generalization and specialization of a pathogenic mycobacterium. Microbes and Infection  6: 1418–1428.  <ul class="find" data-citedArticleID="1056678" data-doi="10.1016/j.micinf.2004.10.003"><li><a href="http://dx.doi.org/10.1016/j.micinf.2004.10.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mycobacterium+marinum%3A+the+generalization+and+specialization+of+a+pathogenic+mycobacterium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mycobacterium+marinum%3A+the+generalization+and+specialization+of+a+pathogenic+mycobacterium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001607-Stinear1" id="pone.0001607-Stinear1"></a>Stinear TP, Seemann T, Pidot S, Frigui W, Reysset G, et al.  (2007) Reductive evolution and niche adaptation inferred from the genome of <em>Mycobacterium ulcerans</em>, the causative agent of Buruli ulcer. Genome Research  17: 192–200.  <ul class="find" data-citedArticleID="1056680" data-doi="10.1101/gr.5942807"><li><a href="http://dx.doi.org/10.1101/gr.5942807" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reductive+evolution+and+niche+adaptation+inferred+from+the+genome+of+Mycobacterium+ulcerans%2C+the+causative+agent+of+Buruli+ulcer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reductive+evolution+and+niche+adaptation+inferred+from+the+genome+of+Mycobacterium+ulcerans%2C+the+causative+agent+of+Buruli+ulcer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001607-Ikegami1" id="pone.0001607-Ikegami1"></a>Ikegami A, Honma K, Sharma A, Kuramitsu H (2004) Multiple functions of the leucine-rich repeat protein LrrA of <em>Treponema denticola</em>. Infection and Immunity  72: 4619–4627.  <ul class="find" data-citedArticleID="1056650" data-doi="10.1128/iai.72.8.4619-4627.2004"><li><a href="http://dx.doi.org/10.1128/iai.72.8.4619-4627.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+functions+of+the+leucine-rich+repeat+protein+LrrA+of+Treponema+denticola." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+functions+of+the+leucine-rich+repeat+protein+LrrA+of+Treponema+denticola.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001607-Cullen1" id="pone.0001607-Cullen1"></a>Cullen PA, Haake DA, Adler B (2004) Outer membrane proteins of pathogenic spirochetes. FEMS Microbiology Reviews  28: 291–318.  <ul class="find" data-citedArticleID="1056632" data-doi="10.1016/j.femsre.2003.10.004"><li><a href="http://dx.doi.org/10.1016/j.femsre.2003.10.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Outer+membrane+proteins+of+pathogenic+spirochetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Outer+membrane+proteins+of+pathogenic+spirochetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001607-Cullen2" id="pone.0001607-Cullen2"></a>Cullen PA, Haake DA, Bulach DM, Zuerner RL, Adler B (2003) LipL21 Is a novel surface-exposed lipoprotein of pathogenic <em>Leptospira</em> species. Infection and Immunity  71: 2414–2421.  <ul class="find" data-citedArticleID="1056634" data-doi="10.1128/iai.71.5.2414-2421.2003"><li><a href="http://dx.doi.org/10.1128/iai.71.5.2414-2421.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LipL21+Is+a+novel+surface-exposed+lipoprotein+of+pathogenic+Leptospira+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LipL21+Is+a+novel+surface-exposed+lipoprotein+of+pathogenic+Leptospira+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001607-Choy1" id="pone.0001607-Choy1"></a>Choy HA, Kelley MM, Chen TL, Moller AK, Matsunaga J, et al.  (2007) Physiological osmotic induction of <em>Leptospira interrogans</em> adhesion: LigA and LigB bind extracellular matrix proteins and fibrinogen. Infection and Immunity  75: 2441–2450.  <ul class="find" data-citedArticleID="1056630" data-doi="10.1128/iai.01635-06"><li><a href="http://dx.doi.org/10.1128/iai.01635-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physiological+osmotic+induction+of+Leptospira+interrogans+adhesion%3A+LigA+and+LigB+bind+extracellular+matrix+proteins+and+fibrinogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physiological+osmotic+induction+of+Leptospira+interrogans+adhesion%3A+LigA+and+LigB+bind+extracellular+matrix+proteins+and+fibrinogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001607-Verma1" id="pone.0001607-Verma1"></a>Verma A, Hellwage J, Artiushin S, Zipfel PF, Kraiczy P, et al.  (2006) LfhA, a novel Factor H-binding protein of <em>Leptospira interrogans</em>. Infection and Immunity  74: 2659–2666.  <ul class="find" data-citedArticleID="1056686" data-doi="10.1128/iai.74.5.2659-2666.2006"><li><a href="http://dx.doi.org/10.1128/iai.74.5.2659-2666.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LfhA%2C+a+novel+Factor+H-binding+protein+of+Leptospira+interrogans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LfhA%2C+a+novel+Factor+H-binding+protein+of+Leptospira+interrogans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001607-Barbosa1" id="pone.0001607-Barbosa1"></a>Barbosa AS, Abreu PAE, Neves FO, Atzingen MV, Watanabe MM, et al.  (2006) A newly identified leptospiral adhesin mediates attachment to laminin. Infection and Immunity  74: 6356–6364.  <ul class="find" data-citedArticleID="1056618" data-doi="10.1128/iai.00460-06"><li><a href="http://dx.doi.org/10.1128/iai.00460-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+newly+identified+leptospiral+adhesin+mediates+attachment+to+laminin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+newly+identified+leptospiral+adhesin+mediates+attachment+to+laminin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001607-Louvel1" id="pone.0001607-Louvel1"></a>Louvel H, Bommezzadri S, Zidane N, Boursaux-Eude C, Creno S, et al.  (2006) Comparative and functional genomic analyses of iron transport and regulation in <em>Leptospira</em> spp. Journal of Bacteriology  188: 7893–7904.  <ul class="find" data-citedArticleID="1056656" data-doi="10.1128/jb.00711-06"><li><a href="http://dx.doi.org/10.1128/jb.00711-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+and+functional+genomic+analyses+of+iron+transport+and+regulation+in+Leptospira+spp." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+and+functional+genomic+analyses+of+iron+transport+and+regulation+in+Leptospira+spp.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001607-Segers1" id="pone.0001607-Segers1"></a>Segers RP, van Gestel JA, van Eys GJ, van der Zeijst BA, Gaastra W (1992) Presence of putative sphingomyelinase genes among members of the family <em>Leptospiraceae</em>. Infection and Immunity  60: 1707–1710.  <ul class="find" data-citedArticleID="1056674"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Presence+of+putative+sphingomyelinase+genes+among+members+of+the+family+Leptospiraceae.&amp;auth=&amp;atitle=Presence+of+putative+sphingomyelinase+genes+among+members+of+the+family+Leptospiraceae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Presence+of+putative+sphingomyelinase+genes+among+members+of+the+family+Leptospiraceae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Presence+of+putative+sphingomyelinase+genes+among+members+of+the+family+Leptospiraceae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001607-Segers2" id="pone.0001607-Segers2"></a>Segers RP, van der Drift A, de Nijs A, Corcione P, van der Zeijst BA, et al.  (1990) Molecular analysis of a sphingomyelinase C gene from <em>Leptospira interrogans</em> serovar hardjo. Infection and Immunity  58: 2177–2185.  <ul class="find" data-citedArticleID="1056676"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+analysis+of+a+sphingomyelinase+C+gene+from+Leptospira+interrogans+serovar+hardjo.&amp;auth=&amp;atitle=Molecular+analysis+of+a+sphingomyelinase+C+gene+from+Leptospira+interrogans+serovar+hardjo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+analysis+of+a+sphingomyelinase+C+gene+from+Leptospira+interrogans+serovar+hardjo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+analysis+of+a+sphingomyelinase+C+gene+from+Leptospira+interrogans+serovar+hardjo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001607-delReal1" id="pone.0001607-delReal1"></a>del Real G, Segers RP, van der Zeijst BA, Gaastra W (1989) Cloning of a hemolysin gene from <em>Leptospira interrogans</em> serovar hardjo. Infection and Immunity  57: 2588–2590.  <ul class="find" data-citedArticleID="1056690"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cloning+of+a+hemolysin+gene+from+Leptospira+interrogans+serovar+hardjo.&amp;auth=&amp;atitle=Cloning+of+a+hemolysin+gene+from+Leptospira+interrogans+serovar+hardjo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+a+hemolysin+gene+from+Leptospira+interrogans+serovar+hardjo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+a+hemolysin+gene+from+Leptospira+interrogans+serovar+hardjo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001607-Ristow1" id="pone.0001607-Ristow1"></a>Ristow P, Bourhy P, McBride F, via Weykamp da C, Figueira C, et al.  (2007) The OmpA-like protein Loa22 Is essential for leptospiral virulence. PLoS Pathogens  3: e97.  <ul class="find" data-citedArticleID="1056668" data-doi="10.1371/journal.ppat.0030097"><li><a href="http://dx.doi.org/10.1371/journal.ppat.0030097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+OmpA-like+protein+Loa22+Is+essential+for+leptospiral+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+OmpA-like+protein+Loa22+Is+essential+for+leptospiral+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001607-Louvel2" id="pone.0001607-Louvel2"></a>Louvel H, Picardeau M (2007)  Genetic manipulation of <em>Leptospira biflexa</em>. In: Coico R, Kowalik TF, Quarles JM, Stevenson B, Taylor R, editors. Current Protocols in Microbiology. Hoboken: Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">31.
              </span><a name="pone.0001607-Babudieri1" id="pone.0001607-Babudieri1"></a>Babudieri B (1961) Studio serologico del gruppo Semaranga-Patoc di Leptospira biflexa.;. Cagliari-Sassari. pp. 408–414.  <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0001607-Ewing1" id="pone.0001607-Ewing1"></a>Ewing B, Hillier L, Wendl MC, Green P (1998) Base-calling of automated sequencer traces using phred. I. Accuracy assessment. Genome Research  8: 175–185.  <ul class="find" data-citedArticleID="1056636" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001607-Gordon1" id="pone.0001607-Gordon1"></a>Gordon D, Abajian C, Green P (1998) Consed: a graphical tool for sequence finishing. Genome Research  8: 195–202.  <ul class="find" data-citedArticleID="1056644" data-doi="10.1101/gr.8.3.195"><li><a href="http://dx.doi.org/10.1101/gr.8.3.195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consed%3A+a+graphical+tool+for+sequence+finishing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consed%3A+a+graphical+tool+for+sequence+finishing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001607-Gordon2" id="pone.0001607-Gordon2"></a>Gordon D, Desmarais C, Green P (2001) Automated finishing with Autofinish. Genome Research  11: 614–625.  <ul class="find" data-citedArticleID="1056646" data-doi="10.1101/gr.171401"><li><a href="http://dx.doi.org/10.1101/gr.171401" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automated+finishing+with+Autofinish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automated+finishing+with+Autofinish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001607-Bocs1" id="pone.0001607-Bocs1"></a>Bocs S, Cruveiller S, Vallenet D, Nuel G, Medigue C (2003) AMIGene: Annotation of MIcrobial Genes. Nucleic Acids Research  31: 3723–3726.  <ul class="find" data-citedArticleID="1056620" data-doi="10.1093/nar/gkg590"><li><a href="http://dx.doi.org/10.1093/nar/gkg590" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AMIGene%3A+Annotation+of+MIcrobial+Genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AMIGene%3A+Annotation+of+MIcrobial+Genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001607-Vallenet1" id="pone.0001607-Vallenet1"></a>Vallenet D, Labarre L, Rouy Z, Barbe V, Bocs S, et al.  (2006) MaGe - a microbial genome annotation system supported by synteny result. Nucleic Acids Research  34: 53–65.  <ul class="find" data-citedArticleID="1056684" data-doi="10.1093/nar/gkj406"><li><a href="http://dx.doi.org/10.1093/nar/gkj406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MaGe+-+a+microbial+genome+annotation+system+supported+by+synteny+result." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MaGe+-+a+microbial+genome+annotation+system+supported+by+synteny+result.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001607-Fuchsman1" id="pone.0001607-Fuchsman1"></a>Fuchsman CA, Rocap G (2006) Whole-genome reciprocal BLAST analysis reveals that <em>Planctomycetes</em> do not share an unusually large number of genes with <em>Eukarya</em> and <em>Archaea</em>. Applied and Environmental Microbiology  72: 6841–6844.  <ul class="find" data-citedArticleID="1056640" data-doi="10.1128/aem.00429-06"><li><a href="http://dx.doi.org/10.1128/aem.00429-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+reciprocal+BLAST+analysis+reveals+that+Planctomycetes+do+not+share+an+unusually+large+number+of+genes+with+Eukarya+and+Archaea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+reciprocal+BLAST+analysis+reveals+that+Planctomycetes+do+not+share+an+unusually+large+number+of+genes+with+Eukarya+and+Archaea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001607-SaintGirons2" id="pone.0001607-SaintGirons2"></a>Saint Girons I, Margarita D, Amouriaux P, Baranton G (1990) First isolation of bacteriophages for a spirochaete: potential genetic tools for <em>Leptospira</em>. Research in Microbiology  141: 1131–1138.  <ul class="find" data-citedArticleID="1056672" data-doi="10.1016/0923-2508(90)90086-6"><li><a href="http://dx.doi.org/10.1016/0923-2508(90)90086-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=First+isolation+of+bacteriophages+for+a+spirochaete%3A+potential+genetic+tools+for+Leptospira." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22First+isolation+of+bacteriophages+for+a+spirochaete%3A+potential+genetic+tools+for+Leptospira.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.XML" value="118233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.PDF" value="266528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g001.PNG_L" value="1537434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g001.PNG_M" value="110749"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g001.PNG_S" value="7942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g001.TIF" value="2443740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g001.PNG_I" value="38493"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t001.PNG_L" value="154741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t001.PNG_M" value="88547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t001.PNG_S" value="12642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t001.TIF" value="785856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t001.PNG_I" value="27319"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t002.PNG_L" value="173646"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t002.PNG_M" value="98149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t002.PNG_S" value="11176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t002.TIF" value="750062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t002.PNG_I" value="29637"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g002.PNG_L" value="105792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g002.PNG_M" value="47912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g002.PNG_S" value="9726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g002.TIF" value="142958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g002.PNG_I" value="19058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t003.PNG_L" value="129987"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t003.PNG_M" value="76648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t003.PNG_S" value="9148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t003.TIF" value="509146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t003.PNG_I" value="24238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g003.PNG_L" value="4179888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g003.PNG_M" value="171599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g003.PNG_S" value="17697"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g003.TIF" value="7170414"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.g003.PNG_I" value="172617"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t004.PNG_L" value="113868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t004.PNG_M" value="65866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t004.PNG_S" value="9420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t004.TIF" value="442814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t004.PNG_I" value="20881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t005.PNG_L" value="266674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t005.PNG_M" value="153168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t005.PNG_S" value="15713"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t005.TIF" value="1301932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.t005.PNG_I" value="46600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.s001.DOC" value="45056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001607.s002.XLS" value="460288"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001607&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001607" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001607&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001607&volume=&issue=&title=Genome Sequence of the Saprophyte Leptospira biflexa Provides Insights into the Evolution of Leptospira and the Pathogenesis of Leptospirosis&author_name=Mathieu%20Picardeau%2C%20Dieter%20M.%20Bulach%2C%20Christiane%20Bouchier%2C%20Richard%20L.%20Zuerner%2C%20Nora%20Zidane%2C%20Peter%20J.%20Wilson%2C%20Sophie%20Creno%2C%20Elizabeth%20S.%20Kuczek%2C%20Simona%20Bommezzadri%2C%20John%20C.%20Davis%2C%20Annette%20McGrath%2C%20Matthew%20J.%20Johnson%2C%20Caroline%20Boursaux-Eude%2C%20Torsten%20Seemann%2C%20Zo%C3%A9%20Rouy%2C%20Ross%20L.%20Coppel%2C%20Julian%20I.%20Rood%2C%20Aur%C3%A9lie%20Lajus%2C%20John%20K.%20Davies%2C%20Claudine%20M%C3%A9digue%2C%20Ben%20Adler&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001607" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001607" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001607" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001607&amp;t=Genome%20Sequence%20of%20the%20Saprophyte%20Leptospira%20biflexa%20Provides%20Insights%20into%20the%20Evolution%20of%20Leptospira%20and%20the%20Pathogenesis%20of%20Leptospirosis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001607&title=Genome%20Sequence%20of%20the%20Saprophyte%20Leptospira%20biflexa%20Provides%20Insights%20into%20the%20Evolution%20of%20Leptospira%20and%20the%20Pathogenesis%20of%20Leptospirosis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001607&amp;title=Genome%20Sequence%20of%20the%20Saprophyte%20Leptospira%20biflexa%20Provides%20Insights%20into%20the%20Evolution%20of%20Leptospira%20and%20the%20Pathogenesis%20of%20Leptospirosis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001607" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001607&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genome Sequence of the Saprophyte Leptospira biflexa Provides Insights into the Evolution of Leptospira and the Pathogenesis of Leptospirosis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001607';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genome%20Sequence%20of%20the%20Saprophyte%20Leptospira%20biflexa%20Provides%20Insights%20into%20the%20Evolution%20of%20Leptospira%20and%20the%20Pathogenesis%20of%20Leptospirosis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001607" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001607" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->

<div class="block">
    <div class="header">
        <h3>Included in the Following Collection</h3>
    </div>
    <p><dd><a href="http://www.ploscollections.org/article/browse/issue/info%3Adoi%2F10.1371%2Fissue.pcol.v02.i03" title="Browse the Open-Access Collection"><em>PLoS ONE</em>: Prokaryotic Genome Collection</a></dd></p>
</div>

<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+genomics%22" title="Search for articles in the subject area:'Animal genomics'"><div class="flagText">Animal genomics</div></a>
              <div data-categoryid="17455" data-articleid="26040"
                   data-categoryname="Animal genomics"
                   class="flagImage" title="Flag 'Animal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="26040"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+pathogens%22" title="Search for articles in the subject area:'Bacterial pathogens'"><div class="flagText">Bacterial pathogens</div></a>
              <div data-categoryid="42717" data-articleid="26040"
                   data-categoryname="Bacterial pathogens"
                   class="flagImage" title="Flag 'Bacterial pathogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="26040"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="26040"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="26040"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Leptospira%22" title="Search for articles in the subject area:'Leptospira'"><div class="flagText">Leptospira</div></a>
              <div data-categoryid="44125" data-articleid="26040"
                   data-categoryname="Leptospira"
                   class="flagImage" title="Flag 'Leptospira' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Leptospira+interrogans%22" title="Search for articles in the subject area:'Leptospira interrogans'"><div class="flagText">Leptospira interrogans</div></a>
              <div data-categoryid="44465" data-articleid="26040"
                   data-categoryname="Leptospira interrogans"
                   class="flagImage" title="Flag 'Leptospira interrogans' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3476'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4111'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4796&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=16209">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=16213">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
