

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Microarray Analysis in the Archaeon Halobacterium salinarum Strain R1</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001064"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001064" />

    <meta name="citation_title" content="Microarray Analysis in the Archaeon Halobacterium salinarum Strain R1"/>
    <meta itemprop="name" content="Microarray Analysis in the Archaeon Halobacterium salinarum Strain R1"/>

      <meta name="citation_author" content="Jens Twellmeyer"/>
            <meta name="citation_author_institution" content="Max-Planck-Institute of Biochemistry, Membrane Biochemistry, Martinsried, Germany"/>
      <meta name="citation_author" content="Andy Wende"/>
            <meta name="citation_author_institution" content="Max-Planck-Institute of Biochemistry, Membrane Biochemistry, Martinsried, Germany"/>
      <meta name="citation_author" content="Jan Wolfertz"/>
            <meta name="citation_author_institution" content="Max-Planck-Institute of Biochemistry, Membrane Biochemistry, Martinsried, Germany"/>
      <meta name="citation_author" content="Friedhelm Pfeiffer"/>
            <meta name="citation_author_institution" content="Max-Planck-Institute of Biochemistry, Membrane Biochemistry, Martinsried, Germany"/>
      <meta name="citation_author" content="Markus Panhuysen"/>
            <meta name="citation_author_institution" content="Max-Planck-Institute of Psychiatry, Molecular Neurogenetics, Munich, Germany"/>
      <meta name="citation_author" content="Alexander Zaigler"/>
            <meta name="citation_author_institution" content="Institute of Molecular Biosciences, University of Frankfurt, Frankfurt am Main, Germany"/>
      <meta name="citation_author" content="Jörg Soppa"/>
            <meta name="citation_author_institution" content="Institute of Molecular Biosciences, University of Frankfurt, Frankfurt am Main, Germany"/>
      <meta name="citation_author" content="Gerhard Welzl"/>
            <meta name="citation_author_institution" content="Institute of Biomathematics and Biometry, Forschungszentrum für Umwelt und Gesundheit (GSF)-National Research Centre for Environment and Health, Neuherberg, Germany"/>
      <meta name="citation_author" content="Dieter Oesterhelt"/>
            <meta name="citation_author_institution" content="Max-Planck-Institute of Biochemistry, Membrane Biochemistry, Martinsried, Germany"/>

    <meta name="citation_date" content="2007/10/24"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001064.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1064"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Biosynthesis of the purple membrane of halobacteria.; citation_author=M Sumper; citation_author=H Reitmeier; citation_author=D Oesterhelt; citation_journal_title=Angew Chem Int Ed Engl; citation_volume=15; citation_number=1; citation_pages=187-94; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=Biogenesis of purple membrane: regulation of bacterio-opsin synthesis.; citation_author=M Sumper; citation_author=G Herrmann; citation_journal_title=FEBS Lett; citation_volume=69; citation_number=2; citation_pages=149-52; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=Phototrophic growth of halobacteria and its use for isolation of photosynthetically-deficient mutants.; citation_author=D Oesterhelt; citation_author=G Krippahl; citation_journal_title=Ann Microbiol; citation_volume=B134; citation_number=3; citation_pages=137-150; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Expression of the bop Gene Cluster of Halobacterium halobium Is Induced by Low Oxygen Tension and by Light.; citation_author=RF Shand; citation_author=MC Betlach; citation_journal_title=J Bacteriol; citation_volume=173; citation_number=4; citation_pages=4692-4699; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=brp and blh are required for synthesis of the retinal cofactor of bacteriorhodopsin in Halobacterium salinarum.; citation_author=RF Peck; citation_author=C Echavarri-Erasun; citation_author=EA Johnson; citation_author=WV Ng; citation_author=SP Kennedy; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=5; citation_pages=5739-5744; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genomic and genetic dissection of an archaeal regulon.; citation_author=NS Baliga; citation_author=SP Kennedy; citation_author=WV Ng; citation_author=L Hood; citation_author=S DasSarma; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=6; citation_pages=2521-2525; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Coordinate regulation of energy transduction modules in Halobacterium sp. analyzed by a global systems approach.; citation_author=NS Baliga; citation_author=M Pan; citation_author=YA Goo; citation_author=EC Yi; citation_author=DR Goodlett; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=7; citation_pages=14913-14918; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Quantitative profiling of the membrane proteome in a halophilic archaeon.; citation_author=B Bisle; citation_author=A Schmidt; citation_author=B Scheibe; citation_author=C Klein; citation_author=A Tebbe; citation_journal_title=Mol Cell Proteomics; citation_volume=5; citation_number=8; citation_pages=1543-1558; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The membrane proteome of Halobacterium salinarum.; citation_author=C Klein; citation_author=C Garcia-Rizo; citation_author=B Bisle; citation_author=B Scheffer; citation_author=H Zischka; citation_journal_title=Proteomics; citation_volume=5; citation_number=9; citation_pages=180-197; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of the cytosolic proteome of Halobacterium salinarum and its implication for genome annotation.; citation_author=A Tebbe; citation_author=C Klein; citation_author=B Bisle; citation_author=F Siedler; citation_author=B Scheffer; citation_journal_title=Proteomics; citation_volume=5; citation_number=10; citation_pages=168-179; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Importance of replication in microarray gene expression studies: statistical methods and evidence from repetitive cDNA hybridizations.; citation_author=ML Lee; citation_author=FC Kuo; citation_author=GA Whitmore; citation_author=J Sklar; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=11; citation_pages=9834-9839; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Design issues for cDNA microarray experiments.; citation_author=YH Yang; citation_author=T Speed; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=12; citation_pages=579-588; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=MAANOVA: a software package for the analysis of spotted cDNA microarray experiments.; citation_author=H Wu; citation_author=MK Kerr; citation_author=X Cui; citation_author=GA Churchill; citation_number=13; citation_pages=313-341; citation_date=2003; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Robust Regression & Outlier Detection.; citation_author=PJ Rousseeuw; citation_author=AM Leroy; citation_number=14; citation_date=1987; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_title=Spotting effect in microarray experiments.; citation_author=T Mary-Huard; citation_author=JJ Daudin; citation_author=S Robin; citation_author=F Bitton; citation_author=E Cabannes; citation_journal_title=BMC Bioinformatics; citation_volume=5; citation_number=15; citation_pages=63; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Missing value estimation methods for DNA microarrays.; citation_author=O Troyanskaya; citation_author=M Cantor; citation_author=G Sherlock; citation_author=P Brown; citation_author=T Hastie; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=16; citation_pages=520-525; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Controlling the false discovery rate–A practical and powerful approach to multiple testing.; citation_author=Y Benjamini; citation_author=Y Hochberg; citation_journal_title=J R Stat Soc Ser B Stat Method; citation_volume=57; citation_number=17; citation_pages=289-300; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The bat gene of Halobacterium halobium encodes a trans-acting oxygen inducibility factor.; citation_author=F Gropp; citation_author=MC Betlach; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=18; citation_pages=5475-5479; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Genomic Analysis of Anaerobic Respiration in the Archaeon Halobacterium sp. Strain NRC-1: Dimethyl Sulfoxide and Trimethylamine N-Oxide as Terminal Electron Acceptors.; citation_author=JA Muller; citation_author=S DasSarma; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=19; citation_pages=1659-1667; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Chemotaxis and phototaxis require a CheA histidine kinase in the archaeon Halobacterium salinarium.; citation_author=J Rudolph; citation_author=D Oesterhelt; citation_journal_title=EMBO J; citation_volume=14; citation_number=20; citation_pages=667-673; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of Halobacterium species NRC-1.; citation_author=WV Ng; citation_author=SP Kennedy; citation_author=GG Mahairas; citation_author=B Berquist; citation_author=M Pan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=21; citation_pages=12176-12181; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Primary structure of an archaebacterial transducer, a methylaccepting protein associated with sensory rhodopsin I.; citation_author=VJ Yao; citation_author=JL Spudich; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=89; citation_number=22; citation_pages=11915-11919; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=An archaeal aerotaxis transducer combines subunit I core structures of eukaryotic cytochrome c oxidase and eubacterial methylaccepting chemotaxis proteins.; citation_author=A Brooun; citation_author=J Bell; citation_author=T Freitas; citation_author=RW Larsen; citation_author=M Alam; citation_journal_title=J Bacteriol; citation_volume=180; citation_number=23; citation_pages=1642-1646; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=BasT, a membrane-bound transducer protein for amino acid detection in Halobacterium salinarum.; citation_author=MV Kokoeva; citation_author=D Oesterhelt; citation_journal_title=Mol Microbiol; citation_volume=35; citation_number=24; citation_pages=647-656; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Further characterization of particulate fractions from lysed cell envelopes of Halobacterium halobium and isolation of gas vacuole membranes.; citation_author=W Stoeckenius; citation_author=WH Kunau; citation_journal_title=J Cell Biol; citation_volume=38; citation_number=25; citation_pages=337-357; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=High-frequency mutations in a plasmid-encoded gas vesicle gene in Halobacterium halobium.; citation_author=S DasSarma; citation_author=JT Halladay; citation_author=JG Jones; citation_author=JW Donovan; citation_author=PJ Giannasca; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=85; citation_number=26; citation_pages=6861-6865; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=DNA microarrays: experimental issues, data analysis, and application to bacterial systems.; citation_author=Y Dharmadi; citation_author=R Gonzalez; citation_journal_title=Biotechnol Prog; citation_volume=20; citation_number=27; citation_pages=1309-1324; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Fermentative arginine degradation in Halobacterium salinarium (formerly Halobacterium halobium): genes, gene products, and transcripts of the arcRACB gene cluster.; citation_author=A Ruepp; citation_author=J Soppa; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=28; citation_pages=4942-7; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=An integrated systems approach for understanding cellular responses to gamma radiation.; citation_author=K Whitehead; citation_author=A Kish; citation_author=M Pan; citation_author=A Kaur; citation_author=DJ Reiss; citation_journal_title=Mol Syst Biol; citation_volume=2; citation_number=29; citation_pages=47; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Anaerobic Growth of Halobacteria.; citation_author=D Oesterhelt; citation_journal_title=Methods Enzymol; citation_volume=88; citation_number=30; citation_pages=417-420; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=PrimeArray: genome-scale primer design for DNA-microarray construction.; citation_author=G Raddatz; citation_author=M Dehio; citation_author=TF Meyer; citation_author=C Dehio; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=31; citation_pages=98-99; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Manufacturing DNA microarrays of high spot homogeneity and reduced background signal.; citation_author=F Diehl; citation_author=S Grahlmann; citation_author=M Beier; citation_author=JD Hoheisel; citation_journal_title=Nucl Acids Res methods online; citation_volume=29; citation_number=32; citation_pages=38; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Construction and usage of a onefold-coverage shotgun DNA microarray to characterize the metabolism of the archaeon Haloferax volcanii.; citation_author=A Zaigler; citation_author=SC Schuster; citation_author=J Soppa; citation_journal_title=Mol Microbiol; citation_volume=48; citation_number=33; citation_pages=1089-1105; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Quantitative RT-PCR: pitfalls and potential.; citation_author=WM Freeman; citation_author=SJ Walker; citation_author=KE Vrana; citation_journal_title=Biotechniques; citation_volume=26; citation_number=34; citation_pages=112-125; citation_date=1999; " />
      <meta name="citation_reference" content="citation_author=M Aregger; citation_number=35; citation_date=2003; citation_publisher=Ludwig-Maximilians-University; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001064" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Microarray Analysis in the Archaeon Halobacterium salinarum Strain R1"/>
    <meta name="twitter:description" content="BackgroundPhototrophy of the extremely halophilic archaeon Halobacterium salinarum was explored for decades. The research was mainly focused on the expression of bacteriorhodopsin and its functional properties. In contrast, less is known about genome wide transcriptional changes and their impact on the physiological adaptation to phototrophy. The tool of choice to record transcriptional profiles is the DNA microarray technique. However, the technique is still rarely used for transcriptome analysis in archaea.Methodology/Principal FindingsWe developed a whole-genome DNA microarray based on our sequence data of the Hbt. salinarum strain R1 genome. The potential of our tool is exemplified by the comparison of cells growing under aerobic and phototrophic conditions, respectively. We processed the raw fluorescence data by several stringent filtering steps and a subsequent MAANOVA analysis. The study revealed a lot of transcriptional differences between the two cell states. We found that the transcriptional changes were relatively weak, though significant. Finally, the DNA microarray data were independently verified by a real-time PCR analysis.Conclusion/SignificanceThis is the first DNA microarray analysis of Hbt. salinarum cells that were actually grown under phototrophic conditions. By comparing the transcriptomics data with current knowledge we could show that our DNA microarray tool is well applicable for transcriptome analysis in the extremely halophilic archaeon Hbt. salinarum. The reliability of our tool is based on both the high-quality array of DNA probes and the stringent data handling including MAANOVA analysis. Among the regulated genes more than 50% had unknown functions. This underlines the fact that haloarchaeal phototrophy is still far away from being completely understood. Hence, the data recorded in this study will be subject to future systems biology analysis."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001064.g006"/>

  <meta property="og:title" content="Microarray Analysis in the Archaeon Halobacterium salinarum Strain R1" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001064" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4756'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5136'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5177&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001064">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001064" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001064&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Microarray Analysis in the Archaeon <i>Halobacterium salinarum</i> Strain R1
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jens Twellmeyer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max-Planck-Institute of Biochemistry, Membrane Biochemistry, Martinsried, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andy Wende, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max-Planck-Institute of Biochemistry, Membrane Biochemistry, Martinsried, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jan Wolfertz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max-Planck-Institute of Biochemistry, Membrane Biochemistry, Martinsried, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Friedhelm Pfeiffer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max-Planck-Institute of Biochemistry, Membrane Biochemistry, Martinsried, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Markus Panhuysen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max-Planck-Institute of Psychiatry, Molecular Neurogenetics, Munich, Germany
                </p>

                <p>
                    Current address: Affectis Pharmaceuticals AG, Munich, Germany
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexander Zaigler, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Molecular Biosciences, University of Frankfurt, Frankfurt am Main, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jörg Soppa, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Molecular Biosciences, University of Frankfurt, Frankfurt am Main, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gerhard Welzl, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Biomathematics and Biometry, Forschungszentrum für Umwelt und Gesundheit (GSF)-National Research Centre for Environment and Health, Neuherberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dieter Oesterhelt
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:oesterhe@biochem.mpg.de">oesterhe@biochem.mpg.de</a></p>

                <p>Affiliation:
                  Max-Planck-Institute of Biochemistry, Membrane Biochemistry, Martinsried, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 24, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001064</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001064" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001064" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001064" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001064" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001064-g001" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001064-g002" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001064-g003" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001064-g004" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001064-g005" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001064-g006" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001064-t001" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001064-t002" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001064">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001064" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Phototrophy of the extremely halophilic archaeon <em>Halobacterium salinarum</em> was explored for decades. The research was mainly focused on the expression of bacteriorhodopsin and its functional properties. In contrast, less is known about genome wide transcriptional changes and their impact on the physiological adaptation to phototrophy. The tool of choice to record transcriptional profiles is the DNA microarray technique. However, the technique is still rarely used for transcriptome analysis in archaea.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We developed a whole-genome DNA microarray based on our sequence data of the <em>Hbt. salinarum</em> strain R1 genome. The potential of our tool is exemplified by the comparison of cells growing under aerobic and phototrophic conditions, respectively. We processed the raw fluorescence data by several stringent filtering steps and a subsequent MAANOVA analysis. The study revealed a lot of transcriptional differences between the two cell states. We found that the transcriptional changes were relatively weak, though significant. Finally, the DNA microarray data were independently verified by a real-time PCR analysis.</p>


<h3>Conclusion/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This is the first DNA microarray analysis of <em>Hbt. salinarum</em> cells that were actually grown under phototrophic conditions. By comparing the transcriptomics data with current knowledge we could show that our DNA microarray tool is well applicable for transcriptome analysis in the extremely halophilic archaeon <em>Hbt. salinarum</em>. The reliability of our tool is based on both the high-quality array of DNA probes and the stringent data handling including MAANOVA analysis. Among the regulated genes more than 50% had unknown functions. This underlines the fact that haloarchaeal phototrophy is still far away from being completely understood. Hence, the data recorded in this study will be subject to future systems biology analysis.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Twellmeyer J, Wende A, Wolfertz J, Pfeiffer F, Panhuysen M, et al.  (2007) Microarray Analysis in the Archaeon <em>Halobacterium salinarum</em> Strain R1. PLoS ONE 2(10):
          e1064.
            doi:10.1371/journal.pone.0001064</p><p><strong>Academic Editor: </strong>Steve Sandler, University of Massachusetts, United States of America</p><p><strong>Received:</strong> June 4, 2007; <strong>Accepted:</strong> September 28, 2007; <strong>Published:</strong> October 24, 2007</p><p><strong>Copyright:</strong> © 2007 Twellmeyer et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The work was supported by the Deutsche Forschungsgesellschaft (DFG) by the funds OE 52/18-1. The DFG was not involved in the design or conduct of the study nor in any other part of the work.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Regulation of gene expression is crucial for the survival of virtually all organisms on earth. It enables cells to adapt to changing environmental conditions. By expressing certain genes only if necessary the cells save both energy and resources. Basically, in archaea the phenomenon of gene regulation is less understood. One tool to study differential gene expression is the DNA microarray technique. It was developed in the early nineties of the last century. Since then it was used in numerous studies. However, its application in the investigation of gene regulation in archaea is still very limited.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p><em>Hbt. salinarum</em> has found its niche in saturated brines. Beside this specialization it has to cope with numerous environmental changes. Among them is the availability of oxygen as terminal electron acceptor and of light as alternative energy source. If oxygen is available, the cells utilize a respiration chain to maintain a proton gradient. Lowering oxygen tension and increasing illumination lead to overexpression of bacteriorhodopsin (BR) <a href="#pone.0001064-Sumper1">[1]</a>, <a href="#pone.0001064-Sumper2">[2]</a>. BR is a light-driven proton pump which is able to maintain a proton gradient to drive ATP production via an ATP synthase. When oxygen is depleted, light can even serve as the only energy source and the cells grow phototrophically <a href="#pone.0001064-Oesterhelt1">[3]</a>. Hence, the conditions that induce the archaeal retinal based photosystem are the same as in the case of the bacterial chlorophyll based photosynthetic system. In the eighties of the last century the use of mutants was introduced to reveal a regulatory network underlying BR expression. Among the regulatory proteins Bat and Brp were identified as the oxygen sensor and the light sensor, respectively <a href="#pone.0001064-Shand1">[4]</a>. However, Peck <em>et al.</em> also provided experimental evidence that Brp may be involved in retinal synthesis, either on a regulatory or catalytic level <a href="#pone.0001064-Peck1">[5]</a>. In more recent times global systems analyses were performed using tools such as transcriptomics and proteomics <a href="#pone.0001064-Baliga1">[6]</a>, <a href="#pone.0001064-Baliga2">[7]</a>. In these studies different <em>Halobacterium</em> spp. were compared in order to elucidate gene expression events governed by phototrophy. The authors found that synthesis of CrtB1, a key enzyme in carotenoid biosynthesis, correlates with the expression of bacterioopsin. Furthermore, they showed that ATP-producing arginine fermentation is regulated inversely to the expression of BR.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Recently, these changes were also analyzed by quantitative proteomic approaches, but these studies are still hampered by the incomplete quantitative analysis of proteins. Although tools such as ICPL have expanded the number of proteins, which can be quantified, the changes of more than 50% of the theoretical proteome is still not detectable by these approaches [8, Tebbe <em>et al.</em>, submitted]. Hence, transcriptional analysis is still superior in regard to the completeness of the analysis of regulatory events. Second, RNA analyses are capable to observe differential expression of genes, which encode cytosolic as well as membrane proteins, with comparatively low technical effort.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here we describe a PCR-product based DNA microarray approach for examination of genome-wide transcriptional changes that also reliably detects faint changes in gene transcription. We applied our tool to analyze the transcriptional differences in a fundamental biological phenomenon that is driven by surprisingly faint changes in gene expression, i.e. phototrophy in the euryarchaeon <em>Halobacterium salinarum</em> strain R1. For that it was important to optimize the data analysis procedure in order to get reliable data of the weak expression differences. We present the first DNA microarray data set of cells actually grown under anaerobic, phototrophic conditions.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Experiment design</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To better understand the complex regulations underlying the adaptations of the energy metabolism we established a whole-genome DNA microarray for <em>Hbt. salinarum</em> strain R1. The manufacture of the microarray was facilitated by the sequencing and annotation of the genome in our group (see <a href="http://www.halolex.mpg.de">www.halolex.mpg.de</a>; <a href="#pone.0001064-Klein1">[9]</a>, <a href="#pone.0001064-Tebbe1">[10]</a>). Taking into account that aerobic and anaerobic conditions represent very different physiological growth conditions, we initially expected large changes in gene expression. However, previous proteomic results proved the differences in gene expression to be rather small [8, and Tebbe <em>et al.</em>, submitted]. In these studies, very few genes were found to be regulated more than threefold. Consequently, the challenge was to detect such small changes in transcript abundances by a high-throughput approach.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Technical as well as biological replications are of great importance for successful transcriptome experiments <a href="#pone.0001064-Lee1">[11]</a>. We analyzed five independent cultures for each growth condition. The cell cultures were carefully adapted to the respective growth condition prior to transcriptome analysis. Further, samples for RNA isolation were drawn from exponentially growing cells. Our DNA microarray contains five replicates of each gene probe. The cDNAs were hybridized applying the common reference design. For that a “common reference”-cDNA was generated from a RNA mixture that contained all RNA samples in equal amounts. This cDNA served as reference in all hybridizations <a href="#pone.0001064-Yang1">[12]</a>.</p>


<h4>Data processing</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Due to the fairly small regulation levels it was crucial that the microarray data were processed very carefully. There are numerous software packages for the analysis of spotted microarray experiments. But specific requirements related to experiments and technical conditions make it lucrative to be able to program an individual data preprocessing system using an object-oriented language and integrating existing modules. Hence, the following paragraphs describe shortly how the fluorescence data were preprocessed and statistically analyzed using the MAANOVA package <a href="#pone.0001064-Wu1">[13]</a> in the R environment (see <a href="http://www.r-project.org/">http://www.r-project.org/</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The fluorescence values of all DNA probes derived from the ten microarrays (5 from aerobic and 5 from phototrophic cells) were collected in a text file. The data were subjected to several preprocessing steps, which included a background correction and two normalization procedures. All these steps were conducted with algorithms written for use in the R environment (source codes available upon request).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>A first description of the expression values showed two problems: censored values and values similar to background values. Saturation of the fluorescent signal results in censored data. These data are a specific form of missing data and must be handled carefully.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Background subtraction in general intensifies the problem of generating negative or small values with extreme and unreliable ratios. In our data sets the background fluorescence varied significantly throughout each microarray, but there was a clear coincidence between the green and red fluorescence channel (<a href="#pone-0001064-g001">Figure 1</a>). Consequently, it was feasible to subtract the local background fluorescence from the fluorescence of each probe spot without introducing biases in expression values. Because we wanted to keep as much as possible low intensity signals, we did not simply exclude all fluorescence data below a certain threshold. We used the dependency of green and red fluorescence structure to construct a distance based on the contour lines of a two dimensional normal distribution (<a href="#pone-0001064-g002">Figure 2</a>). Here, it was necessary to use robust estimators for means and standard deviations (Minimum volume ellipsoid, <a href="#pone.0001064-Rousseeuw1">[14]</a>).</p>
<div class="figure" id="pone-0001064-g001"><div class="img"><a name="pone-0001064-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Box plots of background sample (upper diagram) and reference fluorescence (lower diagram) values around probe spots.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The values are grouped according to the spotting pin. The horizontal line within the boxes shows the median. The borders of the boxes show the 75th and 25th percentiles. The dotted lines outside the boxes indicate upper and lower limit values. Open circles represent individual outlier spots.</p>
<span>doi:10.1371/journal.pone.0001064.g001</span></div><div class="figure" id="pone-0001064-g002"><div class="img"><a name="pone-0001064-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Scatter plots of local background values of probe spots in reference and sample fluorescence channel.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>The contour lines (green) were calculated using the minimum volume ellipsoid method for all spots printed by an individual pin (red).</p>
<span>doi:10.1371/journal.pone.0001064.g002</span></div><a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>There are plenty of biases, which are introduced to microarray experiments for technical reasons, e.g. by printing probes with a multi-pin head, by different incorporation efficiencies of the fluorescent dyes, or by just different fluorescence intensities of individual probes <a href="#pone.0001064-MaryHuard1">[15]</a>. All these variations have to be carefully inspected and removed before extracting any gene regulation from the data set. Among the methods available a regional median normalization was applicable for our data sets. The indications for that were (i) no dependence of the fluorescence ratios of the two dyes from the fluorescence intensity (<a href="#pone-0001064-g003">Figure 3</a>), (ii) a clear spatial effect generated by spotting of the microarray with a 48-pin print head (where each pin printed one of 48 blocks of probe spots) (<a href="#pone-0001064-g004">Figure 4</a>). Consequently, we normalized the fluorescence ratios so that the median of all probes spotted by the same pin is 1.</p>
<div class="figure" id="pone-0001064-g003"><div class="img"><a name="pone-0001064-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>MA plot of raw fluorescence data within a single microarray.</span></strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>The data are colored pin-wise (fluorescence intensities obtained from probes spotted by a certain pin are colored individually).</p>
<span>doi:10.1371/journal.pone.0001064.g003</span></div><div class="figure" id="pone-0001064-g004"><div class="img"><a name="pone-0001064-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Box plots of sample (upper diagram) and reference fluorescence (lower diagram) values of probe spots.</span></strong></p><a id="article1.body1.sec2.sec2.fig4.caption1.p1" name="article1.body1.sec2.sec2.fig4.caption1.p1"></a><p>See legend of <a href="#pone-0001064-g001">Figure 1</a> for a detailed description.</p>
<span>doi:10.1371/journal.pone.0001064.g004</span></div><a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>The aim of the statistical analysis was to identify differentially expressed genes when comparing aerobically and phototrophically grown cells. The R/<em>maanova</em> package (version 0.97-4) provides a complete work flow for microarray data analysis. Especially “mixed effect ANOVA” models are implemented to estimate variance components and to perform F- and t-tests for differential expressions.</p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>One problem with R/<em>maanova</em> is that it does not tolerate missing, zero or negative intensity data. The R/<em>maanova</em> manual suggests to use non-background subtracted data as input and to ignore bad flags. In our opinion this is risky because on the other hand biased expression values with many unreliable quantities are used. The application of only complete data would cause a strong reduction of the data matrix. Therefore it is necessary to make use of methods which impute missing values. In our data we observed 72 genes (2.7%) with more than 33% missing values. These genes were not further analyzed. Missing values of remaining genes were imputed using an algorithm based on Principal Component Analysis <a href="#pone.0001064-Troyanskaya1">[16]</a>.</p>
<a id="article1.body1.sec2.sec2.p8" name="article1.body1.sec2.sec2.p8"></a><p>With these completed data we used the R environment MAANOVA to calculate test statistics for testing the null hypothesis <em>no difference between aerobic and phototrophic growth condition</em> for each gene. The MAANOVA package provides four test statistics, i.e. F1, F2, F3, and Fs. We applied the Fs statistic, which uses a shrinkage estimator for gene-specific variance components based on the James-Stein estimator.</p>
<a id="article1.body1.sec2.sec2.p9" name="article1.body1.sec2.sec2.p9"></a><p>The application of formal hypothesis testing provides p-values. These p-values from tests of differential expression summarize the statistical significance of the applied test statistic, based on the variation in gene expression and the error variance. To cope with the problem of multiple test false discovery rates (FDR) adjusted rates were calculated using the step down method <a href="#pone.0001064-Benjamini1">[17]</a>. The ranking of the genes is based on these adjusted p-values.</p>


<h4>Expression differences between cells adapted to aerobic and phototrophic conditions</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>A rank number was associated to all genes, which were analyzable. The rank sorts the genes from the most statistical significantly regulated to the least significantly regulated one. We applied a false discovery rate of 5%, which resulted in a list of 239 significantly regulated genes. Under phototrophic conditions 130 genes were induced and 109 were repressed compared to aerobic growth conditions (see Supporting Information <a href="#pone.0001064.s001">Table S1</a> for the complete list of regulated genes).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p><a href="#pone-0001064-g005">Figure 5</a> shows a global inspection of the transcription differences between aerobic and phototrophic cells. It reveals that those genes, which are located on the plasmids of <em>Hbt. salinarum</em> strain R1, are obviously underrepresented. Although 25% of the genome is encoded on the four plasmids only 18% of the significantly regulated genes are located there. This difference is mainly due to the small number of regulated genes on pHs2 and no regulated genes at all on pHs4.</p>
<div class="figure" id="pone-0001064-g005"><div class="img"><a name="pone-0001064-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Distribution of all genes versus distribution of significantly regulated genes on the genome.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Chr: chromosome; pHs1-4: plasmids.</p>
<span>doi:10.1371/journal.pone.0001064.g005</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p><a href="#pone-0001064-g006">Figure 6</a> illustrates that more than 50% of the regulated genes belong to the function classes <em>hypothetical genes</em> (HY), <em>conserved hypothetical genes</em> (CHY), or genes with <em>no assigned function</em> (NOF). This is nearly the proportion of these three categories on the genome of <em>Hbt. salinarum</em> (60%). The huge number of 129 regulated genes without functional data illustrates, that the change of <em>Hbt. salinarum</em> strain R1 cells between aerobic and phototrophic lifestyle is less understood than one would expect. However, it should be mentioned that even <em>E. coli</em>, the best studied prokaryote on earth, has a comparably high number of genes without an assigned function.</p>
<div class="figure" id="pone-0001064-g006"><div class="img"><a name="pone-0001064-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001064.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Graphical overview of the functional classification of the significantly regulated genes.</span></strong></p><span>doi:10.1371/journal.pone.0001064.g006</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Among the significantly upregulated genes we found <em>bop</em>, <em>bat</em>, <em>crtB1</em>, and <em>crtB2</em>. The induction of the expression of bacterioopsin, the gene product of <em>bop</em>, by phototrophic conditions was already described several decades ago <a href="#pone.0001064-Sumper1">[1]</a>. Bat is a central regulatory protein necessary to activate the transcription of <em>bop</em>, whereas <em>crtB1</em> and <em>crtB2</em> encode enzymes that catalyze key steps in the synthesis of retinal <a href="#pone.0001064-Baliga2">[7]</a>, <a href="#pone.0001064-Gropp1">[18]</a>. The transcription of all these genes enables <em>Hbt. salinarum</em> strain R1 to prosper under phototrophic conditions. Taken together, they are responsible for the maintenance of the membrane potential and proton gradient, which allows ATP synthesis without the need for a respiratory chain. Consequently, the fact that we found these genes regulated underlines that the regulation data retrieved by our DNA microarray platform are biologically relevant.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Fifteen regulated genes belong to the function class <em>amino acid metabolism</em> (AA). Seven showed increased and eight showed decreased transcript levels during phototrophic growth. Four of these genes are of particular interest: <em>carA</em>, <em>carB</em>, <em>arcB</em>, and <em>arcC</em>. The induced genes <em>carA</em> and <em>carB</em> encode a carbamoylphosphate (Carb-P) synthase, whereas the repressed genes <em>arcB</em> and <em>arcC</em> encode proteins of the arginine deiminase (ADI) pathway. CarA/B synthesize Carb-P by consumption of 2 ATP. The purpose of the ADI pathway is to hydrolyze Carb-P for generation of 1 ATP. Obviously, contrary regulation of the two gene clusters makes sense. Because ATP is plenty in light, additional ATP generating reactions can be shut down. Synthesis of Carb-P, however, facilitates nucleotide synthesis, which in turn may enhance cell growth. Interestingly, the gene <em>pyrI</em>, encoding a subunit of the aspartate carbamoyltransferase, was also found to be induced in our study. This enzyme catalyzes the conversion of Carb-P to carbamoylaspartate, the first step in pyrimidine synthesis.</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>Furthermore, we found several genes involved in the energy metabolism of <em>Hbt. salinarum</em> to be regulated. Among them <em>nuoC</em>, <em>nuoD</em>, <em>nuoK</em>, <em>petB</em>, and <em>dmsB</em> showed increased mRNA levels under phototrophic growth conditions. NuoC, D, and K are subunits of the NADH dehydrogenase. PetB is part of the cytochrome bc complex. DmsB is a chain of the dimethylsulfoxide (DMSO) reductase, which utilizes DMSO as alternative terminal electron acceptor <a href="#pone.0001064-Muller1">[19]</a>. In contrast, <em>cydA1</em>, a subunit of one of the three terminal oxidases, was found to be repressed under phototrophic conditions. The genes <em>korA, korB</em>, and <em>acn</em> encode enzymes that are involved in the TCA cycle. The transcription of these genes was found to be induced under anaerobic conditions. KorA and KorB constitute the 4Fe-4S protein α-ketoglutarate-ferredoxin oxidoreductase, which enables an organism to assimilate CO<sub>2</sub> by a reductive TCA cycle. Acn is the aconitate hydratase that catalyzes the conversion of citrate to isocitrate.</p>
<a id="article1.body1.sec2.sec3.p7" name="article1.body1.sec2.sec3.p7"></a><p>Its chemotactic apparatus enables <em>Hbt. salinarum</em> to move through its aquatic habitat in order to keep itself at favorable conditions. This very important feature is controlled by a set of 18 sensor proteins, the so-called halobacterial transducers (Htrs), and the bacterial-like CheAY-system <a href="#pone.0001064-Rudolph1">[20]</a>, <a href="#pone.0001064-Ng1">[21]</a>. Under phototrophic conditions we found a repression of 4 Htr encoding genes, i.e. <em>htrI</em>, <em>htrVIII</em>, <em>basT</em>, and <em>htr18</em>. HtrI is involved in phototaxis dependent on orange and UV light, HtrVIII is responsible for aerotaxis, and BasT enables <em>Hbt. salinarum</em> to sense the presence of the amino acids Leu, Ile, Val, Met, and Cys <a href="#pone.0001064-Yao1">[22]</a>, <a href="#pone.0001064-Brooun1">[23]</a>, <a href="#pone.0001064-Kokoeva1">[24]</a>. Finally, for HtrVIII no functional data exist. At least the repression of HtrI and HtrVIII can be easily explained. Typical for phototrophic conditions are strong light intensity and lack of oxygen. These circumstances obviously eliminate the need for high numbers of light sensors and for the presence of oxygen sensors. Additionally to the chemotaxis system, gas vesicles can provide a unidirectional movement of cells, i.e. floating to the surface of a water body. Interestingly, <em>Hbt. salinarum</em> induced as well as repressed the expression of gas vesicle genes. The genes <em>gvpL1</em>, <em>gvpK1</em>, and <em>gvpI1</em> on plasmid pHs1 were induced under phototrophic conditions, whereas <em>gvpI2</em> and <em>gvpC2</em> on the chromosome were repressed. However, the interpretation of that phenotype is not trivial, since gas vesicle formation in <em>Hbt. salinarum</em> R1 is generally impaired by a transposon element in front of the plasmid-borne <em>gvp</em> operon <a href="#pone.0001064-Stoeckenius1">[25]</a>, <a href="#pone.0001064-DasSarma1">[26]</a>.</p>
<a id="article1.body1.sec2.sec3.p8" name="article1.body1.sec2.sec3.p8"></a><p>Four genes of the function class <em>transport</em> (TP) were induced and 8 genes were repressed under phototrophic conditions. Most of them encode ABC transport proteins such as permeases, substrate-, and ATP-binding proteins. Basically, the action of ABC transporters should be aided by the abundance of ATP in light. The induction of 4 genes may be explained by the need for the substrates, which are imported by these ABC transporters. Accordingly, repression of 8 transport genes may point to the fact that these are specific to substrates, which are not necessary for phototrophic growth. The general problem with the interpretation of transport protein regulation is that the substrate specificity is not accessible via homology search.</p>
<a id="article1.body1.sec2.sec3.p9" name="article1.body1.sec2.sec3.p9"></a><p>Finally, 11 transposases (function class ISH) showed increased transcript levels under phototrophic conditions. The proteins belong to the COGs 0675, 1943, and 3385. We can just speculate about the reason for the activation of these mobile elements. It might reflect a certain flexibility of the genome. Generally, <em>Hbt. salinarum</em> seems to be very sensitive to environmental changes, because e.g. even a simple freeze-thaw cycle induces mutations that result in phenotypic differences (D Oesterhelt, unpublished observation).</p>


<h4>Validation of transcriptional regulations via quantitative RT-PCR</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Due to the small regulation levels it was essential to validate the transcriptional differences found in the DNA microarray experiment by an independent method. For that we chose a quantitative two-step reverse transcription-PCR (RT-PCR). In the first step total RNA was unspecifically transcribed into cDNA by using random hexamer primers. In a second step quantitative real-time PCR was applied using the gene-specific primers shown in <a href="#pone-0001064-t001">Table 1</a>. For that, 6 of the 10 RNA samples (3 of 5 from each growth condition) used for the DNA microarray experiment were applied to RT-PCR. In order to get an overview of the reliability of the microarray data we performed RT-PCRs of 17 genes. Ten of which having statistical ranks from 4 to 221 and, hence, belong to the group of significantly regulated genes. The other 7 genes are distributed over the range of statistically insignificant genes (having statistical ranks from 332 to 2411).</p>
<div class="figure" id="pone-0001064-t001"><div class="img"><a name="pone-0001064-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001064.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001064.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Sequences of primers used for quantitative RT-PCR analysis.</span></strong></p><span>doi:10.1371/journal.pone.0001064.t001</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The 10 significantly regulated genes showed a differential expression in the RT-PCR as well (<a href="#pone-0001064-t002">Table 2</a>). But only in the case of OE3107F and OE3136F the expression ratio was quite high (&gt;10). Comparatively small regulation levels were also observed in quantitative proteomic studies on aerobic/phototrophic cells [8, Tebbe <em>et al.</em>, submitted]. As expected the RT-PCR data of the 7 genes with ranks lower than 239 showed much less consistency with the DNA microarray data. In the case of <em>arcA</em> and <em>crtI1</em> the RT-PCR data even indicated a gene regulation although the DNA microarray approach did not. This phenomenon is discussed in detail below. In spite of that, the RT-PCR results basically suggest that our DNA microarray platform reliably detects gene regulations, even those with small amplitudes.</p>
<div class="figure" id="pone-0001064-t002"><div class="img"><a name="pone-0001064-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001064" data-uri="info:doi/10.1371/journal.pone.0001064.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001064.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001064.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001064.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001064.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001064.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Comparison of microarray data and quantitative RT-PCR analysis.</span></strong></p><span>doi:10.1371/journal.pone.0001064.t002</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The objective of this study was to provide a tool for the analysis of regulatory events in <em>Halobacterium salinarum</em> strain R1. For that, we constructed a microarray containing probes for 97% of the genome of this halophilic archaeon. Due to the fact that we spotted every probe in five replicates the DNA microarray comprised more than 13500 probe spots. This sophisticated hardware was combined with several stringent data preprocessing steps and a statistical analysis based on the MAANOVA package of Wu <em>et al.</em> <a href="#pone.0001064-Wu1">[13]</a>. Together, our setup allows a reliable detection of gene regulations even if these are rather small. Consequently, our DNA microarray is currently applied by several further researchers to answer biological questions in <em>Hbt. salinarum</em>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>An evidence for the reliability of our data is the fact that several genes necessary for BR production were found to be induced. This phenotype is known for decades and, thus, is a good indicator for true biological relevance. Another measure for reliability is the independent verification of DNA microarray data by quantitative RT-PCR. We found that all significant genes tested by this method showed the same regulation as found by the DNA microarray analysis. Interestingly, in the RT-PCR approach nearly all genes showed higher regulation levels. This behavior is already known and led to the conclusion that microarray experiments are superior in order to identify regulatory trends in gene expression rather than giving the researcher a clue of quantitative regulation values for single genes <a href="#pone.0001064-Dharmadi1">[27]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Seven other genes were tested by RT-PCR, which were rejected by the statistical analysis as being not significantly regulated. Indeed, the coincidence between DNA microarray and RT-PCR results was clearly lower. The RT-PCR further revealed a four- to fivefold regulation for 2 genes, i.e. <em>crtI1</em> and <em>arcA</em>. However, also these two were not selected as being regulated by the microarray approach. The contradiction between DNA microarray and RT-PCR results may have several reasons: (i) cross-hybridization on the DNA microarray, (ii) only 3 of 5 RNAs from each growth condition were used for the RT-PCR analysis, and (iii) the RT-PCR data showed higher standard deviations. In case of <em>crtI1</em> cross-hybridization may be indeed the problem. The gene is highly homologous to <em>crtI2</em>, which is also among the non-significant genes, and showed no regulation in the DNA microarray as well as in the RT-PCR experiment. Though, the contradicting results for <em>arcA</em> may indicate that our stringent data handling possibly generated a reasonable number of “false negatives” among the non-significant genes. This is because <em>arcA</em> is part of the same operon as <em>arcB</em> and <em>arcC</em>, which were found to be regulated in both experimental approaches. ArcR, the putative regulator of these genes, is also encoded in this operon. According to our data <em>arcR</em> is not regulated during phototrophic growth. This corresponds well with the observation that the <em>arcR</em> mRNA is not subject to regulation but remains at a constant level no matter if aerobic or anaerobic conditions are present <a href="#pone.0001064-Ruepp1">[28]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Transcriptome analyses enable the researcher to detect changes in expression of both cytosolic and membrane proteins. In the present study 181 of the regulated genes encode cytosolic proteins, and 53 genes encode putative integral membrane or membrane-attached proteins. Furthermore, 5 genes encode proteins, which are probably secreted into the medium. Such comprehensive description of gene expression is an advantage of microarray analyses. In contrast, proteomic analyses would need an extensive effort of time and material to give a similarly complete picture. So far, two studies reported gene expression data of <em>Hbt. salinarum</em> grown aerobically and phototrophically. Both applied quantitative proteomics tools [8, Tebbe <em>et al.</em>, submitted]. Bisle <em>et al.</em> investigated the membrane-located proteome. The investigation was still hampered by the low number of quantifiable proteins. Tebbe <em>et al.</em> applied the ICPL technique to quantify changes in the cytosolic proteome. In both studies the intensity of gene regulation turned out to be very faint. The same was found in the present analysis of gene transcript levels. As done in the present report, both proteomic studies analyzed cells, which were fully adapted to the growth conditions of interest. However, experiments conducted in our group indicate that the gene expression differences are much larger when observing the transition of cells from aerobic to phototrophic growth conditions in a time-resolved manner (J Twellmeyer, unpublished results).</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Both the transcriptomics as well as the proteomics approaches showed an induction of the structural gene for bacterioopsin (<em>bop</em>) and its regulator (<em>bat</em>). Furthermore, both showed an induction of the DMSO reductase complex and a cytochrom C reductase under anaerobic conditions. These regulations, although rather weak, make sense in a physiological view and were also reported in a previous transcriptomics study <a href="#pone.0001064-Muller1">[19]</a>. Beside the regulatory overlap we also found six cases, in which transcriptomics data contradicted the proteomics results, e.g. the thiamin biosynthesis protein ThiC, the transducer protein HtrI, and a secreted hydrolase. In these cases our study showed an inverse regulation compared to the proteomics results for functionally linked proteins or for members of a polycistronic operon. Up to date it was not shown for archaea that higher transcript levels lead to higher protein levels, which is a matter of fact in bacteria. Hence, the contradictory regulations point to a partial independence of gene translation from transcriptional regulation. In fact, a recent study in <em>Halobacterium</em> NRC-1 showed that changes in mRNA abundance very often correlate with changes in protein abundance <a href="#pone.0001064-Whitehead1">[29]</a>. Whitehead <em>et al.</em> could also show that decreases in mRNA correlate better with decreases in protein abundance if a time lag of 30 minutes is considered. These findings further underline that transcriptomics approaches described represent a useful way to investigate gene expression changes in <em>Hbt. salinarum</em>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>So far, one transcriptomics study exists, which aimed to explore the differences in gene expression concerning energy transduction under aerobic and phototrophic conditions, respectively <a href="#pone.0001064-Baliga2">[7]</a>. However, the authors applied a strategy completely different to the present study. They compared the transcriptomes of two closely related <em>Halobacterium</em> strains, i.e. NRC-1 and S9 plus two mutants descending from S9. All were grown aerobically in light. In strain S9 expression of BR is not physiologically controlled but rather it is constitutively activated. Hence, this strain was considered to mimic the anaerobic, illuminated state of <em>Halobacterium</em> cells. In contrast, we compared cells of one strain (R1) grown under aerobic (in darkness) and anaerobic (in light) conditions.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Despite of the completely different approaches a reasonable overlap of gene regulations found by each study is visible. Interestingly, the overlap is not restricted to genes relevant for BR production and assembly. Both studies revealed an unexpected high proportion of genes with unknown functions among the regulated genes. Further correlations are the repression of genes driving ATP generation by arginine degradation, and the induction of the α-ketoglutarate-ferredoxin oxidoreductase KorAB, what supports a potential reductive TCA cycle. Contradicting results concern the expression of carbamoylphosphate synthase and aspartate carbamoyltransferase. Both catalyze the synthesis of Carb-P and channel it into the nucleotide synthesis. The present study showed repression of 4 Htrs and induction of 11 transposon elements. Both of which were not observed by Baliga <em>et al</em>. On the other hand, we did not observe induction of the superoxide dismutase genes <em>sod1</em> and <em>sod2</em> by phototrophic conditions. In our view the mentioned differences illustrate the advantage of comparing cells actually growing under the conditions of interest instead of using different genotypes. Our strategy provided a broader and more complete view of the transcriptome governed by phototrophy. Especially the induction of <em>sod</em> genes is unlikely under anaerobic conditions.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Since most of the expression differences were quite faint, it is uncertain, what impact these may have on the adaptation of the organism to environmental changes. Future functional interpretation of the regulatory data will benefit from the reconstruction of the genome scale metabolic network of <em>Hbt. salinarum</em>. This <em>in silico</em> network successfully predicted already the flux of 15 amino acids during growth of <em>Hbt. salinarum</em> (Gonzalez <em>et al.</em>, submitted). Currently, the network is used to examine, which metabolic pathways are probably utilized differently under aerobic and phototrophic growth conditions, respectively (Gonzalez <em>et al.</em>, in preparation).</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Finally, more than 50% of the genes found to be differentially transcribed miss any functional data or even hints to their functional role. This shows that long standing research about biosynthesis of the purple membrane, which is essential for phototrophy of Haloarchaea, has not revealed all phenomena and metabolic effects driven by this growth condition. Basically, the reliable identification of regulated genes by a high-throughput method such as the DNA microarray technique provides a solid base for further experimental analysis. However, the high proportion of unknown genes is a drawback inherent to virtually any organism. It clearly impairs the interpretation of the collected data. Combining high-throughput data sets with systems biology approaches might provide an additional tool to understand the physiology of phototrophy.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Growth of <em>Hbt. salinarum</em></h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>For aerobic growth <em>Halobacterium salinarum</em> strain R1 (DSM671) was shaken with 105 rpm in the dark at 40°C in 35 ml of complex medium <a href="#pone.0001064-Oesterhelt2">[30]</a> in 100 ml flasks. For anaerobic phototrophic growth the flasks were sealed with a silicon septum and shaken at 40°C with 105 rpm in front of a white light source in the same volume of complex medium. Cell growth was assessed by measuring optical densities using a Klett-Summerson Photoelectric Colorimeter (Klett Manufacturing, New York, USA). Cells used for RNA isolation were subcultured three times to the mid-exponential phase (cell density 30-40 Klett) under aerobic or phototrophic conditions to allow a full adaptation to the corresponding growth condition. All experiments were replicated five times with independently grown cultures.</p>


<h4>Manufacture of DNA microarrays</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>DNA probes for 2774 open reading frames (see <a href="http://www.halolex.mpg.de">http://www.halolex.mpg.de</a>) of the <em>Hbt. salinarum</em> strain R1 genome were amplified by PCR. The genome-scale primer design was accomplished with the program PrimeArray <a href="#pone.0001064-Raddatz1">[31]</a>. The primers were purchased from Metabion (Martinsried, Germany). The sequences of the primers are available upon request. Gene amplification was carried out in 100 µl reactions with HotStarTaq (Qiagen, Hilden, Germany) according to the instructions of the manufacturer. The reaction mixture contained 20 ng genomic DNA and 0.5 µM of each specific primer. All PCR products were quality checked by agarose gel electrophoresis. Furthermore, a randomly selected subset of 20% was checked by sequencing. Sequences were verified by BLAST search against the HaloLex database (<a href="http://www.halolex.mpg.de">http://www.halolex.mpg.de</a>). Finally, the DNA probes cover 97% of the <em>Hbt. salinarum</em> strain R1 genes. The PCR products were purified using MultiScreen-PCR 96-well filter plates (Millipore, Eschborn, Germany), resuspended in a 3xSSC-buffer (450 mM NaCl, 45 mM sodium citrate) containing 1.5 M betaine (<em>N,N,N</em>-trimethylglycine), and transferred to 384-well plates for arraying.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Five replicates of each probe were spotted with a 200 µm center-to-center spacing with 48 quill-pins on a Virtek ChipWriter Pro (Bio-Rad Laboratories GmbH, Munich, Germany) onto γ-amino-silane coated CMT-GAPS-II glass slides (Corning, Schipol-Rijk, NE). Subsequently, the slides were processed as described by Diehl <em>et al.</em> <a href="#pone.0001064-Diehl1">[32]</a>.</p>


<h4>Isolation of total RNA</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Total RNA was extracted from cultures grown to 30-40 Klett using peqGOLD RNApure (peqLAB Biotechnology, Erlangen, Germany) according to the manufacturers instructions. After dissolving the RNA in RNase-free H<sub>2</sub>O the residual DNA was treated with the DNase kit DNA-free (Ambion, Huntington, UK) following the manufacturers instructions. The yield and purity of the RNA was determined with RNA 6000 Nano LabChip Kits using an Agilent 2100 Bioanalyzer (Agilent Technologies, Karlsruhe, Germany).</p>


<h4>Preparation and hybridization of cDNA</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Synthesis of Cy5/Cy3-labeled cDNA from 5 µg DNA-free total RNA per labeled cDNA was performed using CyScribe First-Strand cDNA Synthesis Kit (GE Healthcare, Freiburg, Germany) with enclosed random nonamer primers and Cy3- and Cy5-dUTP (GE Healthcare, Freiburg, Germany) following the manufacturers instructions. Subsequently, the reaction was stopped, the RNA template was chemically degraded and the cDNA was cleaned and concentrated as described elsewhere <a href="#pone.0001064-Zaigler1">[33]</a>. The microarray hybridizations were designed as common reference experiments. Therefore, a pool of all RNA samples of each experiment was reverse transcribed to Cy5-labeled cDNA and served as reference in all hybridizations. Consequently, the individual RNA samples ended up as Cy3-labeled cDNA. This experiment design eliminates the need for a “dye-swap” and is more open to other experimental objectives such as transcriptome analyses over a certain time-course.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>A detailed description of the hybridization procedure is given in <a href="#pone.0001064-Zaigler1">[33]</a>. Briefly, the microarrays were prehybridized, the Cy5- and Cy3-labeled cDNAs were mixed with hybridization buffer (30 µl final volume), applied to the microarray, and covered with a Hybri-slip HS (Sigma-Aldrich, Taufkirchen, Germany). Hybridization of the cDNA was carried out over night at 64°C in sealed hybridization chambers (Corning, Schipol-Rijk, NE). The microarray slides were washed four times and dried by centrifugation for 5 min at 600 g. The fluorescence intensities were detected using the GenePix 4000B microarray scanner and the corresponding image-analysis software GenePixPro 4.0 (Axon Instruments, Union City, USA). These raw data were further processed as described in detail in the results section. The source codes of the algorithms for preprocessing and statistical analysis of the microarray data are available upon request. The microarray data have been submitted to ArrayExpress (<a href="http://www.ebi.ac.uk/arrayexpress">www.ebi.ac.uk/arrayexpress</a>) under the accession number E-MEXP-1096.</p>


<h4>Reverse transcription of RNA and quantitative PCR</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>5 µg total RNA were reverse transcribed with 0.55 µg random hexamer primer (Promega, Mannheim, Germany) using Superscript III (Invitrogen, Karlsruhe, Germany). The quantitative PCR reactions were done in a GeneAmp 5700 Sequence Detection System (Applied Biosystems, Darmstadt, Germany) using the SYBR Green PCR Master Mix Kit (Applied Biosystems). The final reaction volume was 25 µl with 1 µl of the reverse transcription reaction as template. Primers were designed with Primer Express 2.0 (Applied Biosystems) and were applied in a final concentration of 0.2 µM (see <a href="#pone-0001064-t001">Table 1</a> for primer sequences). Transcript level differences were calculated by a relative quantification approach using an internal standard gene <a href="#pone.0001064-Freeman1">[34]</a>. The gene <em>fdx</em> (OE4217R) was used as internal standard, since it was shown to be constitutively expressed <a href="#pone.0001064-Aregger1">[35]</a>. For all calculations the mean-C<sub>t</sub> of 3 replicate reactions per primer pair was used.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001064"></div><a name="pone.0001064.s001" id="pone.0001064.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001064.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Complete list of significantly regulated genes. cytopl: cytoplasmic protein; transm: integral membrane protein; anchN_lip: membrane attached protein.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001064.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.17 MB PDF)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DO JT. Performed the experiments: JT. Analyzed the data: JW FP GW. Contributed reagents/materials/analysis tools: MP AZ. Wrote the paper: AW GW. Other: Construction of DNA microarray: AW JT. Establishment of DNA microarray analysis: AZ JT. Initiated the project: DO JS. Supervised AZ: JS. Wrote algorithms for data processing and statistical analysis: GW. Supervised the project: DO.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001064-Sumper1" id="pone.0001064-Sumper1"></a>Sumper M, Reitmeier H, Oesterhelt D (1976) Biosynthesis of the purple membrane of halobacteria. Angew Chem Int Ed Engl  15: 187–94.  <ul class="find" data-citedArticleID="1007848" data-doi="10.1002/anie.197601871"><li><a href="http://dx.doi.org/10.1002/anie.197601871" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biosynthesis+of+the+purple+membrane+of+halobacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biosynthesis+of+the+purple+membrane+of+halobacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001064-Sumper2" id="pone.0001064-Sumper2"></a>Sumper M, Herrmann G (1976) Biogenesis of purple membrane: regulation of bacterio-opsin synthesis. FEBS Lett  69: 149–52.  <ul class="find" data-citedArticleID="1007850" data-doi="10.1016/0014-5793(76)80673-4"><li><a href="http://dx.doi.org/10.1016/0014-5793(76)80673-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biogenesis+of+purple+membrane%3A+regulation+of+bacterio-opsin+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biogenesis+of+purple+membrane%3A+regulation+of+bacterio-opsin+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001064-Oesterhelt1" id="pone.0001064-Oesterhelt1"></a>Oesterhelt D, Krippahl G (1983) Phototrophic growth of halobacteria and its use for isolation of photosynthetically-deficient mutants. Ann Microbiol  B134: 137–150.  <ul class="find" data-citedArticleID="1007830" data-doi="10.1016/s0769-2609(83)80101-x"><li><a href="http://dx.doi.org/10.1016/s0769-2609(83)80101-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phototrophic+growth+of+halobacteria+and+its+use+for+isolation+of+photosynthetically-deficient+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phototrophic+growth+of+halobacteria+and+its+use+for+isolation+of+photosynthetically-deficient+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001064-Shand1" id="pone.0001064-Shand1"></a>Shand RF, Betlach MC (1991) Expression of the <em>bop</em> Gene Cluster of <em>Halobacterium halobium</em> Is Induced by Low Oxygen Tension and by Light. J Bacteriol  173: 4692–4699.  <ul class="find" data-citedArticleID="1007844"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression+of+the+bop+Gene+Cluster+of+Halobacterium+halobium+Is+Induced+by+Low+Oxygen+Tension+and+by+Light.&amp;auth=&amp;atitle=Expression+of+the+bop+Gene+Cluster+of+Halobacterium+halobium+Is+Induced+by+Low+Oxygen+Tension+and+by+Light." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+the+bop+Gene+Cluster+of+Halobacterium+halobium+Is+Induced+by+Low+Oxygen+Tension+and+by+Light." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+the+bop+Gene+Cluster+of+Halobacterium+halobium+Is+Induced+by+Low+Oxygen+Tension+and+by+Light.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001064-Peck1" id="pone.0001064-Peck1"></a>Peck RF, Echavarri-Erasun C, Johnson EA, Ng WV, Kennedy SP, et al.  (2001) <em>brp</em> and <em>blh</em> are required for synthesis of the retinal cofactor of bacteriorhodopsin in <em>Halobacterium salinarum</em>. J Biol Chem  276: 5739–5744.  <ul class="find" data-citedArticleID="1007834" data-doi="10.1074/jbc.m009492200"><li><a href="http://dx.doi.org/10.1074/jbc.m009492200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=brp+and+blh+are+required+for+synthesis+of+the+retinal+cofactor+of+bacteriorhodopsin+in+Halobacterium+salinarum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22brp+and+blh+are+required+for+synthesis+of+the+retinal+cofactor+of+bacteriorhodopsin+in+Halobacterium+salinarum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001064-Baliga1" id="pone.0001064-Baliga1"></a>Baliga NS, Kennedy SP, Ng WV, Hood L, DasSarma S (2001) Genomic and genetic dissection of an archaeal regulon. Proc Natl Acad Sci U S A  98: 2521–2525.  <ul class="find" data-citedArticleID="1007798" data-doi="10.1073/pnas.051632498"><li><a href="http://dx.doi.org/10.1073/pnas.051632498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+and+genetic+dissection+of+an+archaeal+regulon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+and+genetic+dissection+of+an+archaeal+regulon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001064-Baliga2" id="pone.0001064-Baliga2"></a>Baliga NS, Pan M, Goo YA, Yi EC, Goodlett DR, et al.  (2002) Coordinate regulation of energy transduction modules in <em>Halobacterium</em> sp. analyzed by a global systems approach. Proc Natl Acad Sci U S A  99: 14913–14918.  <ul class="find" data-citedArticleID="1007800" data-doi="10.1073/pnas.192558999"><li><a href="http://dx.doi.org/10.1073/pnas.192558999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coordinate+regulation+of+energy+transduction+modules+in+Halobacterium+sp.+analyzed+by+a+global+systems+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coordinate+regulation+of+energy+transduction+modules+in+Halobacterium+sp.+analyzed+by+a+global+systems+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001064-Bisle1" id="pone.0001064-Bisle1"></a>Bisle B, Schmidt A, Scheibe B, Klein C, Tebbe A, et al.  (2006) Quantitative profiling of the membrane proteome in a halophilic archaeon. Mol Cell Proteomics  5: 1543–1558.  <ul class="find" data-citedArticleID="1007804" data-doi="10.1074/mcp.m600106-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m600106-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+profiling+of+the+membrane+proteome+in+a+halophilic+archaeon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+profiling+of+the+membrane+proteome+in+a+halophilic+archaeon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001064-Klein1" id="pone.0001064-Klein1"></a>Klein C, Garcia-Rizo C, Bisle B, Scheffer B, Zischka H, et al.  (2005) The membrane proteome of <em>Halobacterium salinarum</em>. Proteomics  5: 180–197.  <ul class="find" data-citedArticleID="1007818" data-doi="10.1002/pmic.200400943"><li><a href="http://dx.doi.org/10.1002/pmic.200400943" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+membrane+proteome+of+Halobacterium+salinarum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+membrane+proteome+of+Halobacterium+salinarum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001064-Tebbe1" id="pone.0001064-Tebbe1"></a>Tebbe A, Klein C, Bisle B, Siedler F, Scheffer B, et al.  (2005) Analysis of the cytosolic proteome of <em>Halobacterium salinarum</em> and its implication for genome annotation. Proteomics  5: 168–179.  <ul class="find" data-citedArticleID="1007852" data-doi="10.1002/pmic.200400910"><li><a href="http://dx.doi.org/10.1002/pmic.200400910" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+cytosolic+proteome+of+Halobacterium+salinarum+and+its+implication+for+genome+annotation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+cytosolic+proteome+of+Halobacterium+salinarum+and+its+implication+for+genome+annotation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001064-Lee1" id="pone.0001064-Lee1"></a>Lee ML, Kuo FC, Whitmore GA, Sklar J (2000) Importance of replication in microarray gene expression studies: statistical methods and evidence from repetitive cDNA hybridizations. Proc Natl Acad Sci U S A  97: 9834–9839.  <ul class="find" data-citedArticleID="1007822" data-doi="10.1073/pnas.97.18.9834"><li><a href="http://dx.doi.org/10.1073/pnas.97.18.9834" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Importance+of+replication+in+microarray+gene+expression+studies%3A+statistical+methods+and+evidence+from+repetitive+cDNA+hybridizations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Importance+of+replication+in+microarray+gene+expression+studies%3A+statistical+methods+and+evidence+from+repetitive+cDNA+hybridizations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001064-Yang1" id="pone.0001064-Yang1"></a>Yang YH, Speed T (2002) Design issues for cDNA microarray experiments. Nat Rev Genet  3: 579–588.  <ul class="find" data-citedArticleID="1007860"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Design+issues+for+cDNA+microarray+experiments.&amp;auth=&amp;atitle=Design+issues+for+cDNA+microarray+experiments." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Design+issues+for+cDNA+microarray+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Design+issues+for+cDNA+microarray+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001064-Wu1" id="pone.0001064-Wu1"></a>Wu H, Kerr MK, Cui X, Churchill GA (2003)  MAANOVA: a software package for the analysis of spotted cDNA microarray experiments. In: Parmigiani G, Garrett E, Irizarry R, Zeger S, editors. The Analysis of Gene Expression Data: Methods and Software. New York: Springer-Verlag. pp. 313–341.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001064-Rousseeuw1" id="pone.0001064-Rousseeuw1"></a>Rousseeuw PJ, Leroy AM (1987) Robust Regression &amp; Outlier Detection. New York: Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0001064-MaryHuard1" id="pone.0001064-MaryHuard1"></a>Mary-Huard T, Daudin JJ, Robin S, Bitton F, Cabannes E, et al.  (2004) Spotting effect in microarray experiments. BMC Bioinformatics  5: 63.  <ul class="find" data-citedArticleID="1007824" data-doi="10.1186/1471-2105-5-63"><li><a href="http://dx.doi.org/10.1186/1471-2105-5-63" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spotting+effect+in+microarray+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spotting+effect+in+microarray+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001064-Troyanskaya1" id="pone.0001064-Troyanskaya1"></a>Troyanskaya O, Cantor M, Sherlock G, Brown P, Hastie T, et al.  (2001) Missing value estimation methods for DNA microarrays. Bioinformatics  17: 520–525.  <ul class="find" data-citedArticleID="1007854" data-doi="10.1093/bioinformatics/17.6.520"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.6.520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Missing+value+estimation+methods+for+DNA+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Missing+value+estimation+methods+for+DNA+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001064-Benjamini1" id="pone.0001064-Benjamini1"></a>Benjamini Y, Hochberg Y (1995) Controlling the false discovery rate–A practical and powerful approach to multiple testing. J R Stat Soc Ser B Stat Method  57: 289–300.  <ul class="find" data-citedArticleID="1007802"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Controlling+the+false+discovery+rate%3FA+practical+and+powerful+approach+to+multiple+testing.&amp;auth=&amp;atitle=Controlling+the+false+discovery+rate%3FA+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+the+false+discovery+rate%E2%80%93A+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+the+false+discovery+rate%E2%80%93A+practical+and+powerful+approach+to+multiple+testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001064-Gropp1" id="pone.0001064-Gropp1"></a>Gropp F, Betlach MC (1994) The <em>bat</em> gene of <em>Halobacterium halobium</em> encodes a trans-acting oxygen inducibility factor. Proc Natl Acad Sci U S A  91: 5475–5479.  <ul class="find" data-citedArticleID="1007816" data-doi="10.1073/pnas.91.12.5475"><li><a href="http://dx.doi.org/10.1073/pnas.91.12.5475" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+bat+gene+of+Halobacterium+halobium+encodes+a+trans-acting+oxygen+inducibility+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+bat+gene+of+Halobacterium+halobium+encodes+a+trans-acting+oxygen+inducibility+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001064-Muller1" id="pone.0001064-Muller1"></a>Muller JA, DasSarma S (2005) Genomic Analysis of Anaerobic Respiration in the Archaeon <em>Halobacterium</em> sp. Strain NRC-1: Dimethyl Sulfoxide and Trimethylamine N-Oxide as Terminal Electron Acceptors. J Bacteriol  187: 1659–1667.  <ul class="find" data-citedArticleID="1007826" data-doi="10.1128/jb.187.5.1659-1667.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.5.1659-1667.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+Analysis+of+Anaerobic+Respiration+in+the+Archaeon+Halobacterium+sp.+Strain+NRC-1%3A+Dimethyl+Sulfoxide+and+Trimethylamine+N-Oxide+as+Terminal+Electron+Acceptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+Analysis+of+Anaerobic+Respiration+in+the+Archaeon+Halobacterium+sp.+Strain+NRC-1%3A+Dimethyl+Sulfoxide+and+Trimethylamine+N-Oxide+as+Terminal+Electron+Acceptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001064-Rudolph1" id="pone.0001064-Rudolph1"></a>Rudolph J, Oesterhelt D (1995) Chemotaxis and phototaxis require a CheA histidine kinase in the archaeon Halobacterium salinarium. EMBO J  14: 667–673.  <ul class="find" data-citedArticleID="1007840"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chemotaxis+and+phototaxis+require+a+CheA+histidine+kinase+in+the+archaeon+Halobacterium+salinarium.&amp;auth=&amp;atitle=Chemotaxis+and+phototaxis+require+a+CheA+histidine+kinase+in+the+archaeon+Halobacterium+salinarium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemotaxis+and+phototaxis+require+a+CheA+histidine+kinase+in+the+archaeon+Halobacterium+salinarium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemotaxis+and+phototaxis+require+a+CheA+histidine+kinase+in+the+archaeon+Halobacterium+salinarium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001064-Ng1" id="pone.0001064-Ng1"></a>Ng WV, Kennedy SP, Mahairas GG, Berquist B, Pan M, et al.  (2000) Genome sequence of <em>Halobacterium</em> species NRC-1. Proc Natl Acad Sci U S A  97: 12176–12181.  <ul class="find" data-citedArticleID="1007828" data-doi="10.1073/pnas.190337797"><li><a href="http://dx.doi.org/10.1073/pnas.190337797" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+Halobacterium+species+NRC-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+Halobacterium+species+NRC-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001064-Yao1" id="pone.0001064-Yao1"></a>Yao VJ, Spudich JL (1992) Primary structure of an archaebacterial transducer, a methylaccepting protein associated with sensory rhodopsin I. Proc Natl Acad Sci U S A  89: 11915–11919.  <ul class="find" data-citedArticleID="1007862" data-doi="10.1073/pnas.89.24.11915"><li><a href="http://dx.doi.org/10.1073/pnas.89.24.11915" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Primary+structure+of+an+archaebacterial+transducer%2C+a+methylaccepting+protein+associated+with+sensory+rhodopsin+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Primary+structure+of+an+archaebacterial+transducer%2C+a+methylaccepting+protein+associated+with+sensory+rhodopsin+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001064-Brooun1" id="pone.0001064-Brooun1"></a>Brooun A, Bell J, Freitas T, Larsen RW, Alam M (1998) An archaeal aerotaxis transducer combines subunit I core structures of eukaryotic cytochrome c oxidase and eubacterial methylaccepting chemotaxis proteins. J Bacteriol  180: 1642–1646.  <ul class="find" data-citedArticleID="1007806"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+archaeal+aerotaxis+transducer+combines+subunit+I+core+structures+of+eukaryotic+cytochrome+c+oxidase+and+eubacterial+methylaccepting+chemotaxis+proteins.&amp;auth=&amp;atitle=An+archaeal+aerotaxis+transducer+combines+subunit+I+core+structures+of+eukaryotic+cytochrome+c+oxidase+and+eubacterial+methylaccepting+chemotaxis+proteins." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+archaeal+aerotaxis+transducer+combines+subunit+I+core+structures+of+eukaryotic+cytochrome+c+oxidase+and+eubacterial+methylaccepting+chemotaxis+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+archaeal+aerotaxis+transducer+combines+subunit+I+core+structures+of+eukaryotic+cytochrome+c+oxidase+and+eubacterial+methylaccepting+chemotaxis+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001064-Kokoeva1" id="pone.0001064-Kokoeva1"></a>Kokoeva MV, Oesterhelt D (2000) BasT, a membrane-bound transducer protein for amino acid detection in <em>Halobacterium salinarum</em>. Mol Microbiol  35: 647–656.  <ul class="find" data-citedArticleID="1007820" data-doi="10.1046/j.1365-2958.2000.01735.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01735.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BasT%2C+a+membrane-bound+transducer+protein+for+amino+acid+detection+in+Halobacterium+salinarum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BasT%2C+a+membrane-bound+transducer+protein+for+amino+acid+detection+in+Halobacterium+salinarum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001064-Stoeckenius1" id="pone.0001064-Stoeckenius1"></a>Stoeckenius W, Kunau WH (1968) Further characterization of particulate fractions from lysed cell envelopes of <em>Halobacterium halobium</em> and isolation of gas vacuole membranes. J Cell Biol  38: 337–357.  <ul class="find" data-citedArticleID="1007846" data-doi="10.1083/jcb.38.2.337"><li><a href="http://dx.doi.org/10.1083/jcb.38.2.337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Further+characterization+of+particulate+fractions+from+lysed+cell+envelopes+of+Halobacterium+halobium+and+isolation+of+gas+vacuole+membranes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Further+characterization+of+particulate+fractions+from+lysed+cell+envelopes+of+Halobacterium+halobium+and+isolation+of+gas+vacuole+membranes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001064-DasSarma1" id="pone.0001064-DasSarma1"></a>DasSarma S, Halladay JT, Jones JG, Donovan JW, Giannasca PJ, et al.  (1988) High-frequency mutations in a plasmid-encoded gas vesicle gene in <em>Halobacterium halobium</em>. Proc Natl Acad Sci U S A  85: 6861–6865.  <ul class="find" data-citedArticleID="1007808" data-doi="10.1073/pnas.85.18.6861"><li><a href="http://dx.doi.org/10.1073/pnas.85.18.6861" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-frequency+mutations+in+a+plasmid-encoded+gas+vesicle+gene+in+Halobacterium+halobium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-frequency+mutations+in+a+plasmid-encoded+gas+vesicle+gene+in+Halobacterium+halobium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001064-Dharmadi1" id="pone.0001064-Dharmadi1"></a>Dharmadi Y, Gonzalez R (2004) DNA microarrays: experimental issues, data analysis, and application to bacterial systems. Biotechnol Prog  20: 1309–1324.  <ul class="find" data-citedArticleID="1007810" data-doi="10.1021/bp0400240"><li><a href="http://dx.doi.org/10.1021/bp0400240" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+microarrays%3A+experimental+issues%2C+data+analysis%2C+and+application+to+bacterial+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+microarrays%3A+experimental+issues%2C+data+analysis%2C+and+application+to+bacterial+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001064-Ruepp1" id="pone.0001064-Ruepp1"></a>Ruepp A, Soppa J (1996) Fermentative arginine degradation in <em>Halobacterium salinarium</em> (formerly <em>Halobacterium halobium</em>): genes, gene products, and transcripts of the <em>arcRACB</em> gene cluster. J Bacteriol  178: 4942–7.  <ul class="find" data-citedArticleID="1007842"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fermentative+arginine+degradation+in+Halobacterium+salinarium+%28formerly+Halobacterium+halobium%29%3A+genes%2C+gene+products%2C+and+transcripts+of+the+arcRACB+gene+cluster.&amp;auth=&amp;atitle=Fermentative+arginine+degradation+in+Halobacterium+salinarium+%28formerly+Halobacterium+halobium%29%3A+genes%2C+gene+products%2C+and+transcripts+of+the+arcRACB+gene+cluster." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fermentative+arginine+degradation+in+Halobacterium+salinarium+%28formerly+Halobacterium+halobium%29%3A+genes%2C+gene+products%2C+and+transcripts+of+the+arcRACB+gene+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fermentative+arginine+degradation+in+Halobacterium+salinarium+%28formerly+Halobacterium+halobium%29%3A+genes%2C+gene+products%2C+and+transcripts+of+the+arcRACB+gene+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001064-Whitehead1" id="pone.0001064-Whitehead1"></a>Whitehead K, Kish A, Pan M, Kaur A, Reiss DJ, et al.  (2006) An integrated systems approach for understanding cellular responses to gamma radiation. Mol Syst Biol  2: 47.  <ul class="find" data-citedArticleID="1007856" data-doi="10.1038/msb4100091"><li><a href="http://dx.doi.org/10.1038/msb4100091" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+integrated+systems+approach+for+understanding+cellular+responses+to+gamma+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+integrated+systems+approach+for+understanding+cellular+responses+to+gamma+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001064-Oesterhelt2" id="pone.0001064-Oesterhelt2"></a>Oesterhelt D (1982) Anaerobic Growth of Halobacteria. Methods Enzymol  88: 417–420.  <ul class="find" data-citedArticleID="1007832" data-doi="10.1016/0076-6879(82)88058-0"><li><a href="http://dx.doi.org/10.1016/0076-6879(82)88058-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anaerobic+Growth+of+Halobacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anaerobic+Growth+of+Halobacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001064-Raddatz1" id="pone.0001064-Raddatz1"></a>Raddatz G, Dehio M, Meyer TF, Dehio C (2001) PrimeArray: genome-scale primer design for DNA-microarray construction. Bioinformatics  17: 98–99.  <ul class="find" data-citedArticleID="1007836" data-doi="10.1093/bioinformatics/17.1.98"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.1.98" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PrimeArray%3A+genome-scale+primer+design+for+DNA-microarray+construction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PrimeArray%3A+genome-scale+primer+design+for+DNA-microarray+construction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001064-Diehl1" id="pone.0001064-Diehl1"></a>Diehl F, Grahlmann S, Beier M, Hoheisel JD (2001) Manufacturing DNA microarrays of high spot homogeneity and reduced background signal. Nucl Acids Res methods online  29: 38.  doi:10.1093/nar/29.7.e38.  <ul class="find" data-citedArticleID="1007812" data-doi="10.1093/nar/29.7.e38"><li><a href="http://dx.doi.org/10.1093/nar/29.7.e38" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Manufacturing+DNA+microarrays+of+high+spot+homogeneity+and+reduced+background+signal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Manufacturing+DNA+microarrays+of+high+spot+homogeneity+and+reduced+background+signal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001064-Zaigler1" id="pone.0001064-Zaigler1"></a>Zaigler A, Schuster SC, Soppa J (2003) Construction and usage of a onefold-coverage shotgun DNA microarray to characterize the metabolism of the archaeon <em>Haloferax volcanii</em>. Mol Microbiol  48: 1089–1105.  <ul class="find" data-citedArticleID="1007864" data-doi="10.1046/j.1365-2958.2003.03497.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03497.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+and+usage+of+a+onefold-coverage+shotgun+DNA+microarray+to+characterize+the+metabolism+of+the+archaeon+Haloferax+volcanii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+and+usage+of+a+onefold-coverage+shotgun+DNA+microarray+to+characterize+the+metabolism+of+the+archaeon+Haloferax+volcanii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001064-Freeman1" id="pone.0001064-Freeman1"></a>Freeman WM, Walker SJ, Vrana KE (1999) Quantitative RT-PCR: pitfalls and potential. Biotechniques  26: 112–125.  <ul class="find" data-citedArticleID="1007814"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Quantitative+RT-PCR%3A+pitfalls+and+potential.&amp;auth=&amp;atitle=Quantitative+RT-PCR%3A+pitfalls+and+potential." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+RT-PCR%3A+pitfalls+and+potential." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+RT-PCR%3A+pitfalls+and+potential.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001064-Aregger1" id="pone.0001064-Aregger1"></a>Aregger M (2003) Munich: Ludwig-Maximilians-University.   Ph.D. thesis. <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.XML" value="95421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.PDF" value="322856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g001.PNG_L" value="185816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g001.PNG_M" value="38322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g001.PNG_S" value="5681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g001.TIF" value="305770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g001.PNG_I" value="13367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g002.PNG_L" value="890829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g002.PNG_M" value="111869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g002.PNG_S" value="7491"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g002.TIF" value="1286044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g002.PNG_I" value="37175"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g003.PNG_L" value="868950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g003.PNG_M" value="88607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g003.PNG_S" value="6638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g003.TIF" value="1294592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g003.PNG_I" value="28240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g004.PNG_L" value="225328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g004.PNG_M" value="51521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g004.PNG_S" value="5995"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g004.TIF" value="483992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g004.PNG_I" value="18757"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g005.PNG_L" value="125162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g005.PNG_M" value="37440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g005.PNG_S" value="6582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g005.TIF" value="353168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g005.PNG_I" value="13843"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g006.PNG_L" value="151196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g006.PNG_M" value="67482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g006.PNG_S" value="8389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g006.TIF" value="290478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.g006.PNG_I" value="24256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.t001.PNG_L" value="167171"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.t001.PNG_M" value="255583"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.t001.PNG_S" value="25671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.t001.TIF" value="752524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.t001.PNG_I" value="114220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.t002.PNG_L" value="111420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.t002.PNG_M" value="136986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.t002.PNG_S" value="18885"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.t002.TIF" value="526574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.t002.PNG_I" value="51920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001064.s001.PDF" value="167320"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001064&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001064" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001064&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001064&volume=&issue=&title=Microarray Analysis in the Archaeon Halobacterium salinarum Strain R1&author_name=Jens%20Twellmeyer%2C%20Andy%20Wende%2C%20Jan%20Wolfertz%2C%20Friedhelm%20Pfeiffer%2C%20Markus%20Panhuysen%2C%20Alexander%20Zaigler%2C%20J%C3%B6rg%20Soppa%2C%20Gerhard%20Welzl%2C%20Dieter%20Oesterhelt&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001064" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001064" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001064" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001064&amp;t=Microarray%20Analysis%20in%20the%20Archaeon%20Halobacterium%20salinarum%20Strain%20R1" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001064&title=Microarray%20Analysis%20in%20the%20Archaeon%20Halobacterium%20salinarum%20Strain%20R1&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001064&amp;title=Microarray%20Analysis%20in%20the%20Archaeon%20Halobacterium%20salinarum%20Strain%20R1" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001064" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001064&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Microarray Analysis in the Archaeon Halobacterium salinarum Strain R1';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001064';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Microarray%20Analysis%20in%20the%20Archaeon%20Halobacterium%20salinarum%20Strain%20R1 http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001064" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001064" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA%22" title="Search for articles in the subject area:'DNA'"><div class="flagText">DNA</div></a>
              <div data-categoryid="32449" data-articleid="24962"
                   data-categoryname="DNA"
                   class="flagImage" title="Flag 'DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24962"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="24962"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Halobacterium%22" title="Search for articles in the subject area:'Halobacterium'"><div class="flagText">Halobacterium</div></a>
              <div data-categoryid="44159" data-articleid="24962"
                   data-categoryname="Halobacterium"
                   class="flagImage" title="Flag 'Halobacterium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="24962"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phototropism%22" title="Search for articles in the subject area:'Phototropism'"><div class="flagText">Phototropism</div></a>
              <div data-categoryid="36245" data-articleid="24962"
                   data-categoryname="Phototropism"
                   class="flagImage" title="Flag 'Phototropism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reverse+transcriptase-polymerase+chain+reaction%22" title="Search for articles in the subject area:'Reverse transcriptase-polymerase chain reaction'"><div class="flagText">Reverse transcriptase-polymerase chain reaction</div></a>
              <div data-categoryid="47921" data-articleid="24962"
                   data-categoryname="Reverse transcriptase-polymerase chain reaction"
                   class="flagImage" title="Flag 'Reverse transcriptase-polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcriptome+analysis%22" title="Search for articles in the subject area:'Transcriptome analysis'"><div class="flagText">Transcriptome analysis</div></a>
              <div data-categoryid="40151" data-articleid="24962"
                   data-categoryname="Transcriptome analysis"
                   class="flagImage" title="Flag 'Transcriptome analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1454'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9211'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5583&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
