

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Patterns of Genome Evolution among the Microsporidian Parasites Encephalitozoon cuniculi, Antonospora locustae and Enterocytozoon bieneusi</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001277"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001277" />

    <meta name="citation_title" content="Patterns of Genome Evolution among the Microsporidian Parasites Encephalitozoon cuniculi, Antonospora locustae and Enterocytozoon bieneusi"/>
    <meta itemprop="name" content="Patterns of Genome Evolution among the Microsporidian Parasites Encephalitozoon cuniculi, Antonospora locustae and Enterocytozoon bieneusi"/>

      <meta name="citation_author" content="Nicolas Corradi"/>
            <meta name="citation_author_institution" content="Department of Botany, Canadian Institute for Advanced Research, University of British Columbia, Vancouver, British Columbia, Canada"/>
      <meta name="citation_author" content="Donna E. Akiyoshi"/>
            <meta name="citation_author_institution" content="Division of Infectious Diseases, Tufts Cummings School of Veterinary Medicine, North Grafton, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Hilary G. Morrison"/>
            <meta name="citation_author_institution" content="Marine Biological Laboratory, Josephine Bay Paul Center for Comparative Molecular Biology and Evolution, Woods Hole, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Xiaochuan Feng"/>
            <meta name="citation_author_institution" content="Division of Infectious Diseases, Tufts Cummings School of Veterinary Medicine, North Grafton, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Louis M. Weiss"/>
            <meta name="citation_author_institution" content="Department of Medicine, Albert Einstein College of Medicine, Bronx, New York, United States of America"/>
      <meta name="citation_author" content="Saul Tzipori"/>
            <meta name="citation_author_institution" content="Division of Infectious Diseases, Tufts Cummings School of Veterinary Medicine, North Grafton, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Patrick J. Keeling"/>
            <meta name="citation_author_institution" content="Department of Botany, Canadian Institute for Advanced Research, University of British Columbia, Vancouver, British Columbia, Canada"/>

    <meta name="citation_date" content="2007/12/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001277.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1277"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Identification of microsporidia.; citation_author=JIR Larsson; citation_journal_title=Acta Protozoologica; citation_volume=38; citation_number=1; citation_pages=161-197; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Microsporidiosis: human diseases and diagnosis.; citation_author=C Franzen; citation_author=A Muller; citation_journal_title=Microbes Infect; citation_volume=3; citation_number=2; citation_pages=389-400; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Congruent evidence from alpha-tubulin and beta-tubulin gene phylogenies for a zygomycete origin of microsporidia.; citation_author=PJ Keeling; citation_journal_title=Fungal Genet Biol; citation_volume=38; citation_number=3; citation_pages=298-309; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Alpha-tubulin from early-diverging eukaryotic lineages and the evolution of the tubulin family.; citation_author=PJ Keeling; citation_author=WF Doolittle; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=4; citation_pages=1297-1305; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Evidence from beta-tubulin phylogeny that microsporidia evolved from within the fungi.; citation_author=PJ Keeling; citation_author=MA Luker; citation_author=JD Palmer; citation_journal_title=Mol Biol Evol; citation_volume=17; citation_number=5; citation_pages=23-31; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Microsporidia: accumulating molecular evidence that a group of amitochondriate and suspectedly primitive eukaryotes are just curious fungi.; citation_author=Y Van de Peer; citation_author=A Ben Ali; citation_author=A Meyer; citation_journal_title=Gene; citation_volume=246; citation_number=6; citation_pages=1-8; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Microsporidian Encephalitozoon cuniculi, a unicellular eukaryote with an unusual chromosomal dispersion of ribosomal genes and a LSU rRNA reduced to the universal core.; citation_author=E Peyretaillade; citation_author=C Biderre; citation_author=P Peyret; citation_author=F Duffieux; citation_author=G Metenier; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=7; citation_pages=3513-3520; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genome sequence and gene compaction of the eukaryote parasite Encephalitozoon cuniculi.; citation_author=MD Katinka; citation_author=S Duprat; citation_author=E Cornillot; citation_author=G Metenier; citation_author=F Thomarat; citation_journal_title=Nature; citation_volume=414; citation_number=8; citation_pages=450-453; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Towards the minimal eukaryotic parasitic genome.; citation_author=CP Vivares; citation_author=G Metenier; citation_journal_title=Curr Opin Microbiol; citation_volume=3; citation_number=9; citation_pages=463-467; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genome compaction and stability in microsporidian intracellular parasites.; citation_author=CH Slamovits; citation_author=NM Fast; citation_author=JS Law; citation_author=PJ Keeling; citation_journal_title=Curr Biol; citation_volume=14; citation_number=10; citation_pages=891-896; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Prevalence of small inversions in yeast gene order evolution.; citation_author=C Seoighe; citation_author=N Federspiel; citation_author=T Jones; citation_author=N Hansen; citation_author=V Bivolarovic; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=11; citation_pages=14433-14437; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Inversions and the dynamics of eukaryotic gene order.; citation_author=MA Huynen; citation_author=B Snel; citation_author=P Bork; citation_journal_title=Trends Genet; citation_volume=17; citation_number=12; citation_pages=304-306; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolution of prokaryotic gene order: genome rearrangements in closely related species.; citation_author=M Suyama; citation_author=P Bork; citation_journal_title=Trends Genet; citation_volume=17; citation_number=13; citation_pages=10-13; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Natural selection promotes the conservation of linkage of co-expressed genes.; citation_author=LD Hurst; citation_author=EJ Williams; citation_author=C Pal; citation_journal_title=Trends Genet; citation_volume=18; citation_number=14; citation_pages=604-606; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evidence for co-evolution of gene order and recombination rate.; citation_author=C Pal; citation_author=LD Hurst; citation_journal_title=Nat Genet; citation_volume=33; citation_number=15; citation_pages=392-395; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A high frequency of overlapping gene expression in compacted eukaryotic genomes.; citation_author=BA Williams; citation_author=CH Slamovits; citation_author=NJ Patron; citation_author=NM Fast; citation_author=PJ Keeling; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=16; citation_pages=10936-10941; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Causes and effects of nuclear genome reduction.; citation_author=PJ Keeling; citation_author=CH Slamovits; citation_journal_title=Curr Opin Genet Dev; citation_volume=15; citation_number=17; citation_pages=601-608; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Occurrence of a new microsporidan: Enterocytozoon bieneusi n.g., n. sp., in the enterocytes of a human patient with AIDS.; citation_author=I Desportes; citation_author=Y Le Charpentier; citation_author=A Galian; citation_author=F Bernard; citation_author=B Cochand-Priollet; citation_journal_title=J Protozool; citation_volume=32; citation_number=18; citation_pages=250-254; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Detection and identification of Enterocytozoon bieneusi and Encephalitozoon species in stool and urine specimens by PCR and differential hybridization.; citation_author=DW Notermans; citation_author=R Peek; citation_author=MD de Jong; citation_author=EM Wentink-Bonnema; citation_author=R Boom; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=19; citation_pages=610-614; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transfer of Nosema locustae (Microsporidia) to Antonospora locustae n. comb. based on molecular and ultrastructural data.; citation_author=CH Slamovits; citation_author=BA Williams; citation_author=PJ Keeling; citation_journal_title=J Eukaryot Microbiol; citation_volume=51; citation_number=20; citation_pages=207-213; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analysis of the complete genome sequence of Encephalitozoon cuniculi supports the fungal origin of microsporidia and reveals a high frequency of fast-evolving genes.; citation_author=F Thomarat; citation_author=CP Vivares; citation_author=M Gouy; citation_journal_title=J Mol Evol; citation_volume=59; citation_number=21; citation_pages=780-791; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bacterial catalase in the microsporidian Nosema locustae: implications for microsporidian metabolism and genome evolution.; citation_author=NM Fast; citation_author=JS Law; citation_author=BA Williams; citation_author=PJ Keeling; citation_journal_title=Eukaryot Cell; citation_volume=2; citation_number=22; citation_pages=1069-1075; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Sequence survey of the genome of the opportunistic microsporidian pathogen, Vittaforma corneae.; citation_author=D Mittleider; citation_author=LC Green; citation_author=VH Mann; citation_author=SF Michael; citation_author=ES Didier; citation_journal_title=J Eukaryot Microbiol; citation_volume=49; citation_number=23; citation_pages=393-401; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genes coding for reverse transcriptase, DNA-directed RNA polymerase, and chitin synthase from the microsporidian Spraguea lophii.; citation_author=G Hinkle; citation_author=HG Morrison; citation_author=ML Sogin; citation_journal_title=Biol Bull; citation_volume=193; citation_number=24; citation_pages=250-251; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The varying microsporidian genome: existence of long-terminal repeat retrotransposon in domesticated silkworm parasite Nosema bombycis.; citation_author=J Xu; citation_author=G Pan; citation_author=L Fang; citation_author=J Li; citation_author=X Tian; citation_journal_title=Int J Parasitol; citation_volume=36; citation_number=25; citation_pages=1049-1056; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.; citation_author=M Kellis; citation_author=N Patterson; citation_author=M Endrizzi; citation_author=B Birren; citation_author=ES Lander; citation_journal_title=Nature; citation_volume=423; citation_number=26; citation_pages=241-254; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Numerous small rearrangements of gene content, order and orientation differentiate grass genomes.; citation_author=JL Bennetzen; citation_author=W Ramakrishna; citation_journal_title=Plant Mol Biol; citation_volume=48; citation_number=27; citation_pages=821-827; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Fourfold faster rate of genome rearrangement in nematodes than in Drosophila.; citation_author=A Coghlan; citation_author=KH Wolfe; citation_journal_title=Genome Res; citation_volume=12; citation_number=28; citation_pages=857-867; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evolution's cauldron: duplication, deletion, and rearrangement in the mouse and human genomes.; citation_author=WJ Kent; citation_author=R Baertsch; citation_author=A Hinrichs; citation_author=W Miller; citation_author=D Haussler; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=29; citation_pages=11484-11489; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of microsporidians with particular reference to species that infect the muscles of fish.; citation_author=F Nilsen; citation_author=C Endresen; citation_author=I Hordvik; citation_journal_title=J Eukaryot Microbiol; citation_volume=45; citation_number=30; citation_pages=535-543; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Evolution of gene order and chromosome number in Saccharomyces, Kluyveromyces and related fungi.; citation_author=RS Keogh; citation_author=C Seoighe; citation_author=KH Wolfe; citation_journal_title=Yeast; citation_volume=14; citation_number=31; citation_pages=443-457; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Comparative genome and proteome analysis of Anopheles gambiae and Drosophila melanogaster.; citation_author=EM Zdobnov; citation_author=C von Mering; citation_author=I Letunic; citation_author=D Torrents; citation_author=M Suyama; citation_journal_title=Science; citation_volume=298; citation_number=32; citation_pages=149-159; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The murine and Drosophila homeobox gene complexes have common features of organization and expression.; citation_author=A Graham; citation_author=N Papalopulu; citation_author=R Krumlauf; citation_journal_title=Cell; citation_volume=57; citation_number=33; citation_pages=367-378; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Evolution of gene order conservation in prokaryotes.; citation_author=J Tamames; citation_journal_title=Genome Biol; citation_volume=2; citation_number=34; citation_pages=RESEARCH0020; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Characterization of a divergent Sec61beta gene in microsporidia.; citation_author=CH Slamovits; citation_author=L Burri; citation_author=PJ Keeling; citation_journal_title=J Mol Biol; citation_volume=359; citation_number=35; citation_pages=1196-1202; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The dynamics of chromosome evolution in birds and mammals.; citation_author=DW Burt; citation_author=C Bruley; citation_author=IC Dunn; citation_author=CT Jones; citation_author=A Ramage; citation_journal_title=Nature; citation_volume=402; citation_number=36; citation_pages=411-413; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=How malleable is the eukaryotic genome? Extreme rate of chromosomal rearrangement in the genus Drosophila.; citation_author=JM Ranz; citation_author=F Casals; citation_author=A Ruiz; citation_journal_title=Genome Res; citation_volume=11; citation_number=37; citation_pages=230-239; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Production and characterization of monoclonal antibodies against Enterocytozoon bieneusi purified from rhesus macaques.; citation_author=Q Zhang; citation_author=I Singh; citation_author=A Sheoran; citation_author=X Feng; citation_author=J Nunnari; citation_journal_title=Infect Immun; citation_volume=73; citation_number=38; citation_pages=5166-5172; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning: a laboratory manual, 2nd ed.: Cold Spring Harbor Laboratory Press.; citation_author=J Sambrook; citation_author=EF Fritsch; citation_author=T Maniatis; citation_number=39; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Basic local alignment search tool.; citation_author=SF Altschul; citation_author=W Gish; citation_author=W Miller; citation_author=EW Myers; citation_author=DJ Lipman; citation_journal_title=J Mol Biol; citation_volume=215; citation_number=40; citation_pages=403-410; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=41; citation_pages=3389-3402; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001277" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Patterns of Genome Evolution among the Microsporidian Parasites Encephalitozoon cuniculi, Antonospora locustae and Enterocytozoon bieneusi"/>
    <meta name="twitter:description" content="BackgroundMicrosporidia are intracellular parasites that are highly-derived relatives of fungi. They have compacted genomes and, despite a high rate of sequence evolution, distantly related species can share high levels of gene order conservation. To date, only two species have been analysed in detail, and data from one of these largely consists of short genomic fragments. It is therefore difficult to determine how&nbsp;conservation has been maintained through microsporidian evolution, and impossible to identify whether certain regions are more prone to genomic stasis.Principal FindingsHere, we analyse three large fragments of the Enterocytozoon bieneusi genome (in total 429 kbp), a species of medical significance. A total of 296 ORFs were identified, annotated and&nbsp;their context compared with Encephalitozoon cuniculi and Antonospora locustae. Overall, a high&nbsp;degree of conservation was found between all three species, and interestingly the level of conservation was similar in all three pairwise comparisons, despite the fact that A. locustae is more distantly related to E. cuniculi and E. bieneusi than either are to each other.Conclusions/SignificanceAny two genes that are found together in any pair of genomes are more likely to be conserved in the third genome as well, suggesting that a core of genes tends to be conserved across the entire group. The mechanisms of rearrangments identified among microsporidian genomes were consistent with a very slow evolution of their architecture, as opposed to the very rapid sequence evolution reported for these parasites."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001277.g005"/>

  <meta property="og:title" content="Patterns of Genome Evolution among the Microsporidian Parasites Encephalitozoon cuniculi, Antonospora locustae and Enterocytozoon bieneusi" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001277" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6145'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7978'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4254&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001277">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001277" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001277&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Patterns of Genome Evolution among the Microsporidian Parasites <i>Encephalitozoon cuniculi</i>, <i>Antonospora locustae</i> and <i>Enterocytozoon bieneusi</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nicolas Corradi
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Nicolas Corradi, Donna E. Akiyoshi</p>

              
              

                <p>Affiliation:
                  Department of Botany, Canadian Institute for Advanced Research, University of British Columbia, Vancouver, British Columbia, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Donna E. Akiyoshi
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Nicolas Corradi, Donna E. Akiyoshi</p>

              
              

                <p>Affiliation:
                  Division of Infectious Diseases, Tufts Cummings School of Veterinary Medicine, North Grafton, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hilary G. Morrison, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Marine Biological Laboratory, Josephine Bay Paul Center for Comparative Molecular Biology and Evolution, Woods Hole, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiaochuan Feng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Infectious Diseases, Tufts Cummings School of Veterinary Medicine, North Grafton, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Louis M. Weiss, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Medicine, Albert Einstein College of Medicine, Bronx, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Saul Tzipori, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Infectious Diseases, Tufts Cummings School of Veterinary Medicine, North Grafton, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patrick J. Keeling
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pkeeling@interchange.ubc.ca">pkeeling@interchange.ubc.ca</a></p>

                <p>Affiliation:
                  Department of Botany, Canadian Institute for Advanced Research, University of British Columbia, Vancouver, British Columbia, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 05, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001277</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001277" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001277" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001277" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001277" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001277-g001" data-doi="info:doi/10.1371/journal.pone.0001277" data-uri="info:doi/10.1371/journal.pone.0001277.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001277-g002" data-doi="info:doi/10.1371/journal.pone.0001277" data-uri="info:doi/10.1371/journal.pone.0001277.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001277-g003" data-doi="info:doi/10.1371/journal.pone.0001277" data-uri="info:doi/10.1371/journal.pone.0001277.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001277-g004" data-doi="info:doi/10.1371/journal.pone.0001277" data-uri="info:doi/10.1371/journal.pone.0001277.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001277-g005" data-doi="info:doi/10.1371/journal.pone.0001277" data-uri="info:doi/10.1371/journal.pone.0001277.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001277">Reader Comments (4)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001277" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Microsporidia are intracellular parasites that are highly-derived relatives of fungi. They have compacted genomes and, despite a high rate of sequence evolution, distantly related species can share high levels of gene order conservation. To date, only two species have been analysed in detail, and data from one of these largely consists of short genomic fragments. It is therefore difficult to determine how&nbsp;conservation has been maintained through microsporidian evolution, and impossible to identify whether certain regions are more prone to genomic stasis.</p>


<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here, we analyse three large fragments of the <em>Enterocytozoon bieneusi</em> genome (in total 429 kbp), a species of medical significance. A total of 296 ORFs were identified, annotated and&nbsp;their context compared with <em>Encephalitozoon cuniculi</em> and <em>Antonospora locustae</em>. Overall, a high&nbsp;degree of conservation was found between all three species, and interestingly the level of conservation was similar in all three pairwise comparisons, despite the fact that <em>A. locustae</em> is more distantly related to <em>E. cuniculi</em> and <em>E. bieneusi</em> than either are to each other.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Any two genes that are found together in any pair of genomes are more likely to be conserved in the third genome as well, suggesting that a core of genes tends to be conserved across the entire group. The mechanisms of rearrangments identified among microsporidian genomes were consistent with a very slow evolution of their architecture, as opposed to the very rapid sequence evolution reported for these parasites.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Corradi N, Akiyoshi DE, Morrison HG, Feng X, Weiss LM, et al.  (2007) Patterns of Genome Evolution among the Microsporidian Parasites <em>Encephalitozoon cuniculi</em>, <em>Antonospora locustae</em> and <em>Enterocytozoon bieneusi</em>. PLoS ONE 2(12):
          e1277.
            doi:10.1371/journal.pone.0001277</p><p><strong>Academic Editor: </strong>Geraldine Butler, University College Dublin, Ireland</p><p><strong>Received:</strong> October 1, 2007; <strong>Accepted:</strong> November 9, 2007; <strong>Published:</strong> December 5, 2007</p><p><strong>Copyright:</strong> © 2007 Corradi et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The Enterocytozoon bieneusi genome sequence survey was supported by National Institutes of Health grants R21 AI52792 and R21 AI064118 and analysis was supported by a grant from the Canadian Institutes for Heath Research (MOP-84265). We acknowledge the Josephine Bay Paul Center for Comparative Molecular Biology and Evolution for the use of data included in the Antonospora locustae Genome Project funded by NSF award number 0135272. NC is supported by a fellowship from the Swiss National Science Foundation (PBLAA-114238). PJK is a fellow of the Canadian Institute for Advanced Research and the Michael Smith Foundation for Health Research. None of these institutions played a role in the design and conduct of the study, in the collection, analysis and interpretation of the data and in the preparation, review or approval of this manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Microsporidia are a diverse group of intracellular eukaryotic parasites including 1,300 described species <a href="#pone.0001277-Larsson1">[1]</a>, over a dozen of which are known to infect humans <a href="#pone.0001277-Franzen1">[2]</a>. These organisms were once regarded as primitively simple eukaryotes, however it is now widely acknowledged that they are extremely specialized, highly derived relatives of fungi <a href="#pone.0001277-Keeling1">[3]</a>–<a href="#pone.0001277-VandePeer1">[6]</a>. Microsporidia are known to harbour very small genomes, the most extreme being the 2.3 Mbp genome of <em>Encephalitozoon intestinalis</em> <a href="#pone.0001277-Peyretaillade1">[7]</a>. Microsporidian genomes have shrunk in two different ways. First, their obligate intracellular parasitic lifestyle has permitted the loss of many genes whose functions can be provided by the host cell, so that that the number of genes is severely reduced, in the case of the complete genome of <em>E. cuniculi</em> to about 2,000 genes <a href="#pone.0001277-Katinka1">[8]</a>, <a href="#pone.0001277-Vivares1">[9]</a>. Second, those genes that remain have been packed into a smaller space by a reduction in the overall size of genes (due to a paucity of introns and slightly smaller proteins) and by a significant reduction in intergenic spaces <a href="#pone.0001277-Katinka1">[8]</a>, <a href="#pone.0001277-Slamovits1">[10]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>A survey of genomic sequences from <em>Antonospora locustae</em>, a microsporidian that is distantly related to <em>E. cuniculi</em>, suggested that this compaction has further effects on the evolution of microsporidian genomes. Specifically, this survey showed that the order of genes in the <em>A. locustae</em> and <em>E. cuniculi</em> genomes share a high degree of conservation, and that this conservation was correlated with short intergenic spaces <a href="#pone.0001277-Slamovits1">[10]</a>. Conservation of gene order usually degrades rapidly in eukaryotic genomes <a href="#pone.0001277-Seoighe1">[11]</a>, <a href="#pone.0001277-Huynen1">[12]</a>, leading to the suggestion that genome compaction could slow the rate of genome rearrangements <a href="#pone.0001277-Slamovits1">[10]</a>. This comparison also challenged the hypothesis that the rate of genome evolution may be correlated with the rate of sequence evolution <a href="#pone.0001277-Suyama1">[13]</a>, since the rate of sequence evolution between the two species is relatively high <a href="#pone.0001277-Slamovits1">[10]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The previous study raised a number of questions about genome evolution in microsporidia and other compacted nuclear genomes that cannot be addressed using a two-way comparison. For example, if gene order conservation is conserved in other microsporidia, are the same gene clusters preserved among many genomes, or merely some level of conservation across the genome? When genes do move, do they move individually or in blocks? Similarly, does the conservation between two genomes correlate with their evolutionary relatedness, or is conservation of gene order driven by other evolutionary mechanisms such as gene co-expression <a href="#pone.0001277-Hurst1">[14]</a> and recombination rate <a href="#pone.0001277-Pal1">[15]</a>? Surveying a third microsporidian genome has the potential to address some of these questions and reveal important aspects of other processes that may be related to genome compaction in microsporidia <a href="#pone.0001277-Slamovits1">[10]</a>, <a href="#pone.0001277-Williams1">[16]</a>, <a href="#pone.0001277-Keeling4">[17]</a>, as well as revealing whether conserved gene clusters are special, or only part of a more random but global process.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p><em>Enterocytozoon bieneusi</em> is a frequent and problematic pathogen of HIV-infected patients <a href="#pone.0001277-Desportes1">[18]</a>, <a href="#pone.0001277-Notermans1">[19]</a> and its phylogenetic position is slightly closer to <em>E. cuniculi</em> than to <em>A. locustae</em> <a href="#pone.0001277-Slamovits2">[20]</a>, making this an ideal species from which to compare the conservation of gene order within the microsporidia. In this study, which is part of an ongoing genome sequence survey of <em>E. bieneusi</em>, three large supercontigs encompassing 429 kbp were annotated and compared to both <em>E. cuniculi</em> and <em>A. locustae</em>. <em>E. bieneusi</em> genes were mapped to homologues in the <em>E. cuniculi</em> genome to determine the overall level of conservation between these two species. These gene order maps were also used to identify conserved blocks of genes and how genes that are not conserved cluster elsewhere in the genome. Finally, a three-way comparison including <em>A. locustae</em> was used to assess how genome rearrangements evolved along the phylogenetic tree of microsporidia.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussions"></a><h3>Results and Discussions</h3>
<h4>Evolution of genome structure between <em>E. bieneusi</em> and <em>E. cuniculi</em></h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Open reading frames were identified on three contiguous fragments of the <em>E. bieneusi</em> genome, which encode 116, 127, and 186 Kbp. A total of 296 open reading frames were identified and, for each of these putative genes, the position of orthologous genes in the <em>E. cuniculi</em> genome was determined. In the <em>E. bieneusi</em> genome, 83% of all the genes found in these blocks have a clearly recognisable orthologue in <em>E. cuniculi</em> and these orthologues are scattered across the genome. The remaining 17% of the ORFs identified showed little similarity to <em>E. cuniculi</em> ORFs, a consequence of either high sequence divergence among microsporidian genes <a href="#pone.0001277-Thomarat1">[21]</a> or independent insertions/deletions of genes along their evolutionary history. A careful analysis of these latter ORFs showed that some were indeed true orthologues, that they shared synteny with other genes in <em>E. cuniculi</em> and that the low similarity identified between the orthologues was a consequence of very high sequence divergence.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Overall, the <em>E. bieneusi</em> genome is compact, with around 0.7 genes per kilobase, but not as compact as <em>E. cuniculi</em> (~0.97 genes per kilobase, <a href="#pone.0001277-Katinka1">[8]</a>. Genetic distance between these two species is comparable with what has been observed between <em>A. locustae</em> and <em>E. cuniculi</em> <a href="#pone.0001277-Slamovits1">[10]</a> (p-distance of 0.52 averaged over 14 protein-coding genes). We found no evidence for repetitive and mobile DNA in the genomic regions we analysed, although some examples are known in <em>A. locustae</em> <a href="#pone.0001277-Fast1">[22]</a>, <em>Vairimorpha corneae</em> <a href="#pone.0001277-Mittleider1">[23]</a>, <em>Spraguea lophii</em> <a href="#pone.0001277-Hinkle1">[24]</a> and <em>Nosema bombyicis</em> <a href="#pone.0001277-Xu1">[25]</a>. Based on the identified orthologues, we first measured the degree of conservation of gene order as the percentage of adjacent gene pairs in <em>E. bieneusi</em> that were also adjacent or close neighbors in <em>E. cuniculi</em>. From 293 comparable <em>E. bieneusi</em> adjacent gene pairs, 16% were also adjacent in <em>E. cuniculi</em>, an additional 14% were close neighbors in <em>E. cuniculi</em>. Overall, 40% of the <em>E. bieneusi</em> paired genes were located on the same chromosome in <em>E. cuniculi</em> (<a href="#pone-0001277-g001">Fig. 1</a>). This shows that the degree of conservation is comparable to that previously reported for the <em>E. cuniculi</em>–<em>A. locustae</em> comparison <a href="#pone.0001277-Slamovits1">[10]</a> change to numerical citation format and that a relatively high level of genome order conservation therefore extends to <em>E. bieneusi</em>.</p>
<div class="figure" id="pone-0001277-g001"><div class="img"><a name="pone-0001277-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001277" data-uri="info:doi/10.1371/journal.pone.0001277.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001277.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001277.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Percentage of gene order conservation between <em>E. bieneusi</em> and <em>E. cuniculi.</em></span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Each category displays the percentage of gene pairs in <em>E. bieneusi</em> that are adjacent, close neighbours, or nonsyntenic in <em>E. cuniculi</em>.</p>
<span>doi:10.1371/journal.pone.0001277.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The comparison between <em>E. cuniculi</em> and <em>A. locustae</em> was primarily based on small regions of the <em>A. locustae</em> genome. The large <em>E. bieneusi</em> fragments allow insights not possible with these small fragments, so we compared in detail the order and orientation of the 296 <em>E. bieneusi</em> ORFs with homologous regions of the <em>E. cuniculi</em> genome (<a href="#pone-0001277-g002">Figs. 2</a>–<a href="#pone-0001277-g003"></a><a href="#pone-0001277-g004">4</a>) to see what kinds of rearrangements are most common. As previously determined for other species <a href="#pone.0001277-Kellis1">[26]</a>, but at a much larger evolutionary scale, these comparisons reveal very little conservation at the highest level (i.e., for each <em>E. bieneusi</em> fragment the genes are found on many <em>E. cuniculi</em> chromosomes), but there were many blocks of genes where local gene order is highly conserved. Numerous small rearrangement events have also occurred within regions of conserved gene order, many including small inversions of one or more genes between <em>E. bieneusi</em> and <em>E. cuniculi</em>. This has also been observed in closely related eukaryotic genomes and is thought to be a widespread evolutionary mechanism in eukaryotic nuclear genomes <a href="#pone.0001277-Seoighe1">[11]</a>, <a href="#pone.0001277-Bennetzen1">[27]</a>–<a href="#pone.0001277-Kent1">[29]</a>. Larger inversions are also seen, including the complete inversions of syntenic regions of as many as seven ORFs (e.g., <a href="#pone-0001277-g003">Fig. 3</a>). Single-ORF transpositions are also evident in some cases (e.g., Ecu03_240 in <a href="#pone-0001277-g002">Figure 2</a> and Ecu07_0270 in <a href="#pone-0001277-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0001277-g002"><div class="img"><a name="pone-0001277-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001277" data-uri="info:doi/10.1371/journal.pone.0001277.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001277.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001277.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Analysis of contigs included in the scaffold SC_2384 and gene order comparisons between the <em>E. bieneusi</em> (A) and <em>E. cuniculi</em> (B) orthologues.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Coloured boxes represent orthologues in the same context in both genomes, and the chromosome on which each <em>E. cuniculi</em> segment is located is indicated by Roman numerals. Grey boxes represent genes that are not identifiably orthologous or are not conserved in context. Regions of both genomes transposed above or below the aligned regions represent blocks of genes in different contexts. The chromosomal location of the <em>E. cuniculi</em> orthologue of such genes from <em>E. bieneusi</em> is indicated by Arabic numerals above the boxes. The chromosomal location of syntenic ORFs in the genome of <em>E. cuniculi</em> is shown in the lower part of the figure and annotated with Roman numbers. Straight lines join the homologous ORFs. Red triangles represent boundaries between the contigs used to generate the scaffolds presented in these figures.</p>
<span>doi:10.1371/journal.pone.0001277.g002</span></div><div class="figure" id="pone-0001277-g003"><div class="img"><a name="pone-0001277-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001277" data-uri="info:doi/10.1371/journal.pone.0001277.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001277.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001277.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Analysis of contigs included in the scaffold SC_2496 and gene order comparisons between the <em>E. bieneusi</em> (A) and <em>E. cuniculi</em> (B) orthologues.</span></strong></p><span>doi:10.1371/journal.pone.0001277.g003</span></div><div class="figure" id="pone-0001277-g004"><div class="img"><a name="pone-0001277-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001277" data-uri="info:doi/10.1371/journal.pone.0001277.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001277.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001277.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Analysis of contigs included in the scaffold SC_1888 and gene order comparisons between the <em>E. bieneusi</em> (A) and <em>E. cuniculi</em> (B) orthologues.</span></strong></p><span>doi:10.1371/journal.pone.0001277.g004</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The blocks of conservation are separated by regions of variable sizes where genes show no conservation of order. Many such regions are composed of a single gene or small number of genes, but in several, blocks of multiple genes exist where no conservation of order is found (i.e., grey genes above the aligned regions of the genomes in <a href="#pone-0001277-g002">Figs. 2</a>–<a href="#pone-0001277-g003"></a><a href="#pone-0001277-g004">4</a>). The ORFs in which order is reciprocally conserved in <em>E. cuniculi,</em> but not in <em>E. bieneusi,</em> cannot be determined until the complete genome is annotated. However, both genomes appear to have been rearranged in blocks of genes where there are a few instances of reorganisation, and blocks of genes where there are few instances of conservation.</p>


<h4>Genome conservation across the tree of microsporidia</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A schematic representation of microsporidia phylogeny based on SSU phylogenies is shown in <a href="#pone-0001277-g005">Figure 5</a> <a href="#pone.0001277-Slamovits2">[20]</a>, <a href="#pone.0001277-Nilsen1">[30]</a>. Molecular phylogeny consistently places the <em>Encephalitozoon</em> clade as sister to the <em>Nosema</em> clade, and the <em>Enterocytozoon</em> clade as sister to both. <em>Antonospora</em> falls elsewhere in the tree as sister to <em>Paranosema</em>, and is therefore more distantly related to either <em>E. cuniculi</em> or <em>E. bieneusi</em> than they are to one another. Generally, conservation of gene order correlates with the evolutionary distance between the species <a href="#pone.0001277-Seoighe1">[11]</a>, <a href="#pone.0001277-Keogh1">[31]</a>, so one might predict a significantly higher level of conservation between <em>E. cuniculi</em> and <em>E. bieneusi</em> than between either of these and <em>A. locustae</em>. Indeed, <em>E. bieneusi</em> and <em>E. cuniculi</em> do share a slightly higher proportion of adjacent gene pairs (16%, <a href="#pone-0001277-g001">Fig. 1</a>) compared to that found between <em>E. cuniculi</em> and <em>A. locustae</em> (13%, <a href="#pone.0001277-Slamovits1">[10]</a>), but the difference is small and the level of conservation of more distant pairs is actually lower (40% and 44%, respectively, <a href="#pone-0001277-g001">Fig. 1</a>). Overall, there is no strong evidence for a correlation between the level of gene order conservation and the evolutionary distances between the organisms, suggesting as an alternative that the level of conservation might be static.</p>
<div class="figure" id="pone-0001277-g005"><div class="img"><a name="pone-0001277-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001277" data-uri="info:doi/10.1371/journal.pone.0001277.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001277.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001277.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001277.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001277.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001277.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Schematic representation of phylogenetic relationships among microsporidian genera based on SSU rDNA sequences using Maximum Likelihood and genetic distances (Neighbor-Joining) <a href="#pone.0001277-Slamovits2">[20]</a>, <a href="#pone.0001277-Nilsen1">[30.]</a></span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Colored boxes linking pairs of taxa indicate the percentage of conservation of adjacent pairs of genes. Blue boxes indicate the percentage of adjacent loci that are also adjacent in the third microsporidian species analysed in this study.</p>
<span>doi:10.1371/journal.pone.0001277.g005</span></div>

<h4>Evidence for a core of gene pairs with conserved order among microsporidia</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To more fully analyse the nature of genome conservation between all three species, we conducted a parallel analysis of genes in complete adjacency between <em>E. bieneusi</em> and <em>A. locustae</em> using a dataset including 94 gene pairs previously analysed in <em>A. locustae</em> <a href="#pone.0001277-Slamovits1">[10]</a>. The conservation of adjacent pairs was calculated for each of three possible pairwise species comparisons, resulting in values similar to those reported above and previously (<a href="#pone-0001277-g005">Fig. 5</a>). Specifically, the percent conservation of <em>E. cuniculi</em> and <em>E. bieneusi</em> to <em>A. locustae</em> were 13% and 12%, respectively, while the conservation between <em>E. cuniculi</em> and <em>E. bieneusi</em> to one another was 16%. The important question arising from this is whether the same genes tend to be conserved in order in all three genomes. To test this, we took all conserved adjacent gene pairs from each of the three pairwise species comparisons, and determined the frequency with which they were also conserved in the third species (e.g. for the 16% of gene pairs conserved between <em>E. bieneusi</em> and <em>E. cuniculi</em>, homologues for all genes were identified in <em>A. locustae</em> and the frequency with which the same pairs were conserved was determined). Interestingly, this frequency was significantly higher than the baseline frequency for all three of the pairwise species comparisons (i.e., 42% to 52%: <a href="#pone-0001277-g005">Fig. 5</a>). This result suggests that there is a core of gene pairs that were most likely linked in the ancestor of all three species, and are much more likely to remain linked than other genes in the genome (for examples, see <a href="#pone.0001277.s001">Supporting Information File S1</a>). This is also consistent with the lack of any pronounced correlation between the level of genome conservation and the evolutionary distance beween any two species. This correlation is likely more obvious in closely related species, but between such distant relatives, it could be overwhelmed by a relatively larger pool of genes that tend to be conserved throughout the group. Interestingly, when the regions conserved in all three species are compared, it is difficult to determine which species are most similar in genomic structure. In some cases, genomic architecture seems to be more similar between <em>E. cuniculi</em> and <em>E. bieneusi</em> (<a href="#pone.0001277.s001">Supporting Information File S1</a>, regions B, E, F), while in others, <em>E. cuniculi</em> and <em>A. locustae</em> share a more similar genomic context (<a href="#pone.0001277.s001">Supporting Information File S1</a>, regions C, I, J).</p>


<h4>Possible causes and mechanisms leading to genomic stasis in microsporidia</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Examples of gene order conservation at deep levels of divergence are known in a few cases where order is maintained either by chance <a href="#pone.0001277-Zdobnov1">[32]</a>, because the proteins are part of a complex regulatory pathway <a href="#pone.0001277-Graham1">[33]</a> or because of coregulation of functionally associated genes <a href="#pone.0001277-Huynen1">[12]</a>, <a href="#pone.0001277-Tamames1">[34]</a>. However, with the exception of two loci encoding ribosomal proteins, neither of the latter predictions seem to account for the conserved gene order across the three microsporidian species. Using bioinformatics tools such as InterproScan <a href="#pone.0001277-Zdobnov1">[32]</a> we failed to identify any conserved gene pairs that appear to be part of a same gene network, or which are known to be functionally associated. Conservation was shown to be correlated with short intergenic regions in <em>E. cuniculi</em> and <em>A. locustae</em> and it was proposed that the compaction of the genome made it difficult to separate genes without introducing deleterious breakpoints <a href="#pone.0001277-Slamovits1">[10]</a>, a factor noted to contribute to conservation of gene order in other genomes as well <a href="#pone.0001277-Hurst1">[14]</a>. The recently discovered phenomenon of multigene transcription in microsporidia <a href="#pone.0001277-Williams1">[16]</a>, <a href="#pone.0001277-Slamovits3">[35]</a> could also contribute to the preservation of gene order, since control elements of one gene may be situated within an adjacent gene, selecting against their separation. Indeed, three gene pairs that we have identified as being conserved across all three species (<a href="#pone.0001277.s001">Supporting Information File S1</a>, regions I, J, K), have also been found to show overlapping transcription in <em>A. locustae</em> <a href="#pone.0001277-Williams1">[16]</a> and in <em>E. cuniculi</em> (unpublished). This correlation would extend the idea that co-expression of genes in yeast leads to greater conservation in order <a href="#pone.0001277-Hurst1">[14]</a>, however, large scale analyses of transcriptional patterns among syntenic genes in the microsporidia are necessary to determine if a correlation between multigene transcription and conservation of gene order is a significant factor or a minor one.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>One correlation that clearly does not hold is the proposed relationship between the rates of nucleotide substitution and genomic rearrangements, which are thought to correlate in other organisms <a href="#pone.0001277-Coghlan1">[28]</a>, <a href="#pone.0001277-Burt1">[36]</a>, <a href="#pone.0001277-Ranz1">[37]</a>. From that point of view, microsporidia can be considered as exceptional for harbouring very fast evolving genes <a href="#pone.0001277-Thomarat1">[21]</a> within slowly evolving genomes. According to these results, it is tempting to speculate that two independent evolutionary forces are acting on microsporidia genes and their genome structure.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4><em>E. bieneusi</em> genomic DNA isolation, genomic library construction and sequencing</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The genomic data used in the study come from a genome survey project funded by NIH grants to the Tufts University group (manuscript in preparation). <em>E. bieneusi</em> spores were purified from fresh stools of infected adult humans using the method described by Zhang and colleagues <a href="#pone.0001277-Zhang1">[38]</a>. Two independent genomic DNA extractions were carried out with using a modified proteinase K-phenol extraction protocol <a href="#pone.0001277-Sambrook1">[39]</a>. Using the two genomic DNA preparations, two random 2–3 kb genomic libraries were constructed by Agencourt Biosciences, Inc. (Beverly, MA) in a proprietary high copy number vector. In the construction of the library, genomic DNA was hydrodynamically sheared in the Hydroshear (Genomic Solutions, Ann Arbor, MI) and then separated on agarose gel. A fraction corresponding to ~3,500 bp in length was excised from the gel and purified by the GeneClean procedure (Qbiogene, Morgan Irvine, CA). The purified DNA fragments were blunt-ended using T4 DNA polymerase. The 3.5 kb DNA was ligated to unique <em>Bst</em>XI-linker adapters and the linker-adapted inserts were ligated to <em>Bst</em>XI-cut vector to construct a “shotgun” library. Clones were sequenced using ABI 3.1 BigDye terminator chemistry (Applied Biosystems, Foster City, CA). In total 37,383 reads were assembled into 2,821 contiguously assembled segments. consisting of two or more reads, using the Paracel GenomeAssembler (PGA) (<a href="http://www.paracel.com/products/pga.html">http://www.paracel.com/products/pga.html</a>) with default program parameters and quality scores. 1,078 contigs were discarded as contaminating bacterial genomic sequence, leaving 1,743 <em>E. bieneusi</em> contigs. This Whole Genome Shotgun project has been deposited at DDBJ/EMBL/GenBank under the project accession ABGB00000000. The version described in this paper is the first version, ABGB01000000. Three large scaffolds (SC_2384, SC_2496 and SC_1888) were obtained based on contig mate pair information and used in this study. SC_2384 was assembled from contigs ctg01_153 (43326 bp) and ctg01_154 (83950 bp). SC_2496 was assembled from contigs ctg01_1952 (65945 bp), ctg01_2519 (1260 bp), ctg01_2570 (1227 bp), ctg01_2003 (1338 bp) and ctg01_1951 (118303 bp). SC_1888 was assembled from contigs ctg01_110 (2152 bp), 111 (3690 bp), 112 (6021 bp), 113 (3057 bp), 114 (2484 bp) and 115 (98412 bp). All the contigs used in this study are included in accession ABGB00000000.</p>


<h4>Annotation of <em>E. bieneusi</em> supercontigs and gene order surveys</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Putative open reading frames (ORFs) were characterized in this study from the three <em>E. bieneusi</em> scaffolds (<a href="#pone.0001277.s002">Supporting Information File S2</a>). The genomic DNA sequence of the contigs was compared to the complete genome of <em>E. cuniculi</em> <a href="#pone.0001277-Katinka1">[8]</a> using tBLASTX <a href="#pone.0001277-Altschul1">[40]</a>, <a href="#pone.0001277-Altschul2">[41]</a> to identify all potential protein-coding genes. The list of putative ORFs identified along the three contigs, their description, their position and the accession number of their relative ORF in the <em>E. cuniculi</em> genome are listed in the supporting information file, <a href="#pone.0001277.s003">Table S1</a>. In parallel, to compare the level of conservation between <em>E. bieneusi</em> and <em>A. locustae</em>, the gene pairs used in the study by Slamovits et al. (2004) change to numerical citation were compared to the <em>E. bieneusi</em> genome database using tBLASTX. As this comparison was performed before the <em>E. bieneusi</em> genome was completely assembled, we limited our analysis to whether gene pairs were adjacent (no intervening ORFs) in each species, to avoid any biases due to contigs with different lengths or with incomplete ORF annotations. Finally, the set of adjacent gene pairs found in common between <em>E. bieneusi</em> and <em>E. cuniculi</em> were compared to the <em>A. locustae</em> genome database (<a href="http://gmod.mbl.edu/perl/site/antonospora">http://gmod.mbl.edu/perl/site/antonospor​a</a>), to determine the number of gene pairs conserved in order among all the three microsporidian species we analysed in this study (<a href="#pone-0001277-g005">Fig. 5</a>, percentages in blue squares). Function predictions were generated for the sets of syntenic proteins using Interproscan <a href="#pone.0001277-Zdobnov1">[32]</a>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001277"></div><a name="pone.0001277.s001" id="pone.0001277.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001277.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Examples of gene order conservation between <em>E. cuniculi</em>, <em>E. bieneusi</em> and <em>A. locustae</em>. This figure represent genomic regions of <em>A. locustae</em> previously identified by Slamovits et al. (2004) and Williams et al. (2004) numerical citations. Loci in the same order are shown in coloured arrows and are linked with straight lines. Transcriptional direction of genes is indicated by arrow direction. The accession numbers of <em>A. locustae</em> fragments shown in this figure are as follows. A. AY548887, B. AY548895, C. AY548905, D. AY548901, F. AY548898, G. AY548891, H. AY548889, I. DQ057555, J. DQ057548, K. DQ057569</p>
<p class="siDoi">doi:10.1371/journal.pone.0001277.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.09 MB DOC)</p>
<a name="pone.0001277.s002" id="pone.0001277.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001277.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Genomic sequence of the of the scaffolds used in this study.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001277.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.46 MB DOC)</p>
<a name="pone.0001277.s003" id="pone.0001277.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001277.s003">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Description of the ORFs, their position along the scaffolds, and the accession number of their best match against the <em>E. cuniculi</em> genome.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001277.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.37 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Claudio Slamovits, James Harper and three reviewers for critical readings of this manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PK NC. Analyzed the data: NC. Contributed reagents/materials/analysis tools: LW ST HM DA XF. Wrote the paper: PK NC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001277-Larsson1" id="pone.0001277-Larsson1"></a>Larsson JIR (1999) Identification of microsporidia. Acta Protozoologica  38: 161–197.  <ul class="find" data-citedArticleID="1027070"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+microsporidia.&amp;auth=&amp;atitle=Identification+of+microsporidia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+microsporidia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+microsporidia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001277-Franzen1" id="pone.0001277-Franzen1"></a>Franzen C, Muller A (2001) Microsporidiosis: human diseases and diagnosis. Microbes Infect  3: 389–400.  <ul class="find" data-citedArticleID="1027044" data-doi="10.1016/s1286-4579(01)01395-8"><li><a href="http://dx.doi.org/10.1016/s1286-4579(01)01395-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microsporidiosis%3A+human+diseases+and+diagnosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microsporidiosis%3A+human+diseases+and+diagnosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001277-Keeling1" id="pone.0001277-Keeling1"></a>Keeling PJ (2003) Congruent evidence from alpha-tubulin and beta-tubulin gene phylogenies for a zygomycete origin of microsporidia. Fungal Genet Biol  38: 298–309.  <ul class="find" data-citedArticleID="1027056" data-doi="10.1016/s1087-1845(02)00537-6"><li><a href="http://dx.doi.org/10.1016/s1087-1845(02)00537-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Congruent+evidence+from+alpha-tubulin+and+beta-tubulin+gene+phylogenies+for+a+zygomycete+origin+of+microsporidia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Congruent+evidence+from+alpha-tubulin+and+beta-tubulin+gene+phylogenies+for+a+zygomycete+origin+of+microsporidia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001277-Keeling2" id="pone.0001277-Keeling2"></a>Keeling PJ, Doolittle WF (1996) Alpha-tubulin from early-diverging eukaryotic lineages and the evolution of the tubulin family. Mol Biol Evol  13: 1297–1305.  <ul class="find" data-citedArticleID="1027058" data-doi="10.1093/oxfordjournals.molbev.a025576"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025576" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alpha-tubulin+from+early-diverging+eukaryotic+lineages+and+the+evolution+of+the+tubulin+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alpha-tubulin+from+early-diverging+eukaryotic+lineages+and+the+evolution+of+the+tubulin+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001277-Keeling3" id="pone.0001277-Keeling3"></a>Keeling PJ, Luker MA, Palmer JD (2000) Evidence from beta-tubulin phylogeny that microsporidia evolved from within the fungi. Mol Biol Evol  17: 23–31.  <ul class="find" data-citedArticleID="1027060" data-doi="10.1093/oxfordjournals.molbev.a026235"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+from+beta-tubulin+phylogeny+that+microsporidia+evolved+from+within+the+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+from+beta-tubulin+phylogeny+that+microsporidia+evolved+from+within+the+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001277-VandePeer1" id="pone.0001277-VandePeer1"></a>Van de Peer Y, Ben Ali A, Meyer A (2000) Microsporidia: accumulating molecular evidence that a group of amitochondriate and suspectedly primitive eukaryotes are just curious fungi. Gene  246: 1–8.  <ul class="find" data-citedArticleID="1027100" data-doi="10.1016/s0378-1119(00)00063-9"><li><a href="http://dx.doi.org/10.1016/s0378-1119(00)00063-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microsporidia%3A+accumulating+molecular+evidence+that+a+group+of+amitochondriate+and+suspectedly+primitive+eukaryotes+are+just+curious+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microsporidia%3A+accumulating+molecular+evidence+that+a+group+of+amitochondriate+and+suspectedly+primitive+eukaryotes+are+just+curious+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001277-Peyretaillade1" id="pone.0001277-Peyretaillade1"></a>Peyretaillade E, Biderre C, Peyret P, Duffieux F, Metenier G, et al.  (1998) Microsporidian Encephalitozoon cuniculi, a unicellular eukaryote with an unusual chromosomal dispersion of ribosomal genes and a LSU rRNA reduced to the universal core. Nucleic Acids Res  26: 3513–3520.  <ul class="find" data-citedArticleID="1027080" data-doi="10.1093/nar/26.15.3513"><li><a href="http://dx.doi.org/10.1093/nar/26.15.3513" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microsporidian+Encephalitozoon+cuniculi%2C+a+unicellular+eukaryote+with+an+unusual+chromosomal+dispersion+of+ribosomal+genes+and+a+LSU+rRNA+reduced+to+the+universal+core." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microsporidian+Encephalitozoon+cuniculi%2C+a+unicellular+eukaryote+with+an+unusual+chromosomal+dispersion+of+ribosomal+genes+and+a+LSU+rRNA+reduced+to+the+universal+core.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001277-Katinka1" id="pone.0001277-Katinka1"></a>Katinka MD, Duprat S, Cornillot E, Metenier G, Thomarat F, et al.  (2001) Genome sequence and gene compaction of the eukaryote parasite Encephalitozoon cuniculi. Nature  414: 450–453.  <ul class="find" data-citedArticleID="1027054" data-doi="10.1038/35106579"><li><a href="http://dx.doi.org/10.1038/35106579" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+and+gene+compaction+of+the+eukaryote+parasite+Encephalitozoon+cuniculi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+and+gene+compaction+of+the+eukaryote+parasite+Encephalitozoon+cuniculi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001277-Vivares1" id="pone.0001277-Vivares1"></a>Vivares CP, Metenier G (2000) Towards the minimal eukaryotic parasitic genome. Curr Opin Microbiol  3: 463–467.  <ul class="find" data-citedArticleID="1027102" data-doi="10.1016/s1369-5274(00)00123-5"><li><a href="http://dx.doi.org/10.1016/s1369-5274(00)00123-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+the+minimal+eukaryotic+parasitic+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+the+minimal+eukaryotic+parasitic+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001277-Slamovits1" id="pone.0001277-Slamovits1"></a>Slamovits CH, Fast NM, Law JS, Keeling PJ (2004) Genome compaction and stability in microsporidian intracellular parasites. Curr Biol  14: 891–896.  <ul class="find" data-citedArticleID="1027088" data-doi="10.1016/j.cub.2004.04.041"><li><a href="http://dx.doi.org/10.1016/j.cub.2004.04.041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+compaction+and+stability+in+microsporidian+intracellular+parasites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+compaction+and+stability+in+microsporidian+intracellular+parasites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001277-Seoighe1" id="pone.0001277-Seoighe1"></a>Seoighe C, Federspiel N, Jones T, Hansen N, Bivolarovic V, et al.  (2000) Prevalence of small inversions in yeast gene order evolution. Proc Natl Acad Sci U S A  97: 14433–14437.  <ul class="find" data-citedArticleID="1027086" data-doi="10.1073/pnas.240462997"><li><a href="http://dx.doi.org/10.1073/pnas.240462997" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence+of+small+inversions+in+yeast+gene+order+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence+of+small+inversions+in+yeast+gene+order+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001277-Huynen1" id="pone.0001277-Huynen1"></a>Huynen MA, Snel B, Bork P (2001) Inversions and the dynamics of eukaryotic gene order. Trends Genet  17: 304–306.  <ul class="find" data-citedArticleID="1027052" data-doi="10.1016/s0168-9525(01)02302-2"><li><a href="http://dx.doi.org/10.1016/s0168-9525(01)02302-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inversions+and+the+dynamics+of+eukaryotic+gene+order." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inversions+and+the+dynamics+of+eukaryotic+gene+order.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001277-Suyama1" id="pone.0001277-Suyama1"></a>Suyama M, Bork P (2001) Evolution of prokaryotic gene order: genome rearrangements in closely related species. Trends Genet  17: 10–13.  <ul class="find" data-citedArticleID="1027094" data-doi="10.1016/s0168-9525(00)02159-4"><li><a href="http://dx.doi.org/10.1016/s0168-9525(00)02159-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+prokaryotic+gene+order%3A+genome+rearrangements+in+closely+related+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+prokaryotic+gene+order%3A+genome+rearrangements+in+closely+related+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001277-Hurst1" id="pone.0001277-Hurst1"></a>Hurst LD, Williams EJ, Pal C (2002) Natural selection promotes the conservation of linkage of co-expressed genes. Trends Genet  18: 604–606.  <ul class="find" data-citedArticleID="1027050" data-doi="10.1016/s0168-9525(02)02813-5"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02813-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+promotes+the+conservation+of+linkage+of+co-expressed+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+promotes+the+conservation+of+linkage+of+co-expressed+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001277-Pal1" id="pone.0001277-Pal1"></a>Pal C, Hurst LD (2003) Evidence for co-evolution of gene order and recombination rate. Nat Genet  33: 392–395.  <ul class="find" data-citedArticleID="1027078" data-doi="10.1038/ng1111"><li><a href="http://dx.doi.org/10.1038/ng1111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+co-evolution+of+gene+order+and+recombination+rate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+co-evolution+of+gene+order+and+recombination+rate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001277-Williams1" id="pone.0001277-Williams1"></a>Williams BA, Slamovits CH, Patron NJ, Fast NM, Keeling PJ (2005) A high frequency of overlapping gene expression in compacted eukaryotic genomes. Proc Natl Acad Sci U S A  102: 10936–10941.  <ul class="find" data-citedArticleID="1027104" data-doi="10.1073/pnas.0501321102"><li><a href="http://dx.doi.org/10.1073/pnas.0501321102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+high+frequency+of+overlapping+gene+expression+in+compacted+eukaryotic+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+high+frequency+of+overlapping+gene+expression+in+compacted+eukaryotic+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001277-Keeling4" id="pone.0001277-Keeling4"></a>Keeling PJ, Slamovits CH (2005) Causes and effects of nuclear genome reduction. Curr Opin Genet Dev  15: 601–608.  <ul class="find" data-citedArticleID="1027062" data-doi="10.1016/j.gde.2005.09.003"><li><a href="http://dx.doi.org/10.1016/j.gde.2005.09.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Causes+and+effects+of+nuclear+genome+reduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Causes+and+effects+of+nuclear+genome+reduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001277-Desportes1" id="pone.0001277-Desportes1"></a>Desportes I, Le Charpentier Y, Galian A, Bernard F, Cochand-Priollet B, et al.  (1985) Occurrence of a new microsporidan: Enterocytozoon bieneusi n.g., n. sp., in the enterocytes of a human patient with AIDS. J Protozool  32: 250–254.  <ul class="find" data-citedArticleID="1027040" data-doi="10.1111/j.1550-7408.1985.tb03046.x"><li><a href="http://dx.doi.org/10.1111/j.1550-7408.1985.tb03046.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Occurrence+of+a+new+microsporidan%3A+Enterocytozoon+bieneusi+n.g.%2C+n.+sp.%2C+in+the+enterocytes+of+a+human+patient+with+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Occurrence+of+a+new+microsporidan%3A+Enterocytozoon+bieneusi+n.g.%2C+n.+sp.%2C+in+the+enterocytes+of+a+human+patient+with+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001277-Notermans1" id="pone.0001277-Notermans1"></a>Notermans DW, Peek R, de Jong MD, Wentink-Bonnema EM, Boom R, et al.  (2005) Detection and identification of Enterocytozoon bieneusi and Encephalitozoon species in stool and urine specimens by PCR and differential hybridization. J Clin Microbiol  43: 610–614.  <ul class="find" data-citedArticleID="1027076" data-doi="10.1128/jcm.43.2.610-614.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.2.610-614.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+and+identification+of+Enterocytozoon+bieneusi+and+Encephalitozoon+species+in+stool+and+urine+specimens+by+PCR+and+differential+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+and+identification+of+Enterocytozoon+bieneusi+and+Encephalitozoon+species+in+stool+and+urine+specimens+by+PCR+and+differential+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001277-Slamovits2" id="pone.0001277-Slamovits2"></a>Slamovits CH, Williams BA, Keeling PJ (2004) Transfer of Nosema locustae (Microsporidia) to Antonospora locustae n. comb. based on molecular and ultrastructural data. J Eukaryot Microbiol  51: 207–213.  <ul class="find" data-citedArticleID="1027090" data-doi="10.1111/j.1550-7408.2004.tb00547.x"><li><a href="http://dx.doi.org/10.1111/j.1550-7408.2004.tb00547.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transfer+of+Nosema+locustae+%28Microsporidia%29+to+Antonospora+locustae+n.+comb.+based+on+molecular+and+ultrastructural+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transfer+of+Nosema+locustae+%28Microsporidia%29+to+Antonospora+locustae+n.+comb.+based+on+molecular+and+ultrastructural+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001277-Thomarat1" id="pone.0001277-Thomarat1"></a>Thomarat F, Vivares CP, Gouy M (2004) Phylogenetic analysis of the complete genome sequence of Encephalitozoon cuniculi supports the fungal origin of microsporidia and reveals a high frequency of fast-evolving genes. J Mol Evol  59: 780–791.  <ul class="find" data-citedArticleID="1027098" data-doi="10.1007/s00239-004-2673-0"><li><a href="http://dx.doi.org/10.1007/s00239-004-2673-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+analysis+of+the+complete+genome+sequence+of+Encephalitozoon+cuniculi+supports+the+fungal+origin+of+microsporidia+and+reveals+a+high+frequency+of+fast-evolving+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+analysis+of+the+complete+genome+sequence+of+Encephalitozoon+cuniculi+supports+the+fungal+origin+of+microsporidia+and+reveals+a+high+frequency+of+fast-evolving+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001277-Fast1" id="pone.0001277-Fast1"></a>Fast NM, Law JS, Williams BA, Keeling PJ (2003) Bacterial catalase in the microsporidian Nosema locustae: implications for microsporidian metabolism and genome evolution. Eukaryot Cell  2: 1069–1075.  <ul class="find" data-citedArticleID="1027042" data-doi="10.1128/ec.2.5.1069-1075.2003"><li><a href="http://dx.doi.org/10.1128/ec.2.5.1069-1075.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+catalase+in+the+microsporidian+Nosema+locustae%3A+implications+for+microsporidian+metabolism+and+genome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+catalase+in+the+microsporidian+Nosema+locustae%3A+implications+for+microsporidian+metabolism+and+genome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001277-Mittleider1" id="pone.0001277-Mittleider1"></a>Mittleider D, Green LC, Mann VH, Michael SF, Didier ES, et al.  (2002) Sequence survey of the genome of the opportunistic microsporidian pathogen, Vittaforma corneae. J Eukaryot Microbiol  49: 393–401.  <ul class="find" data-citedArticleID="1027072" data-doi="10.1111/j.1550-7408.2002.tb00218.x"><li><a href="http://dx.doi.org/10.1111/j.1550-7408.2002.tb00218.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+survey+of+the+genome+of+the+opportunistic+microsporidian+pathogen%2C+Vittaforma+corneae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+survey+of+the+genome+of+the+opportunistic+microsporidian+pathogen%2C+Vittaforma+corneae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001277-Hinkle1" id="pone.0001277-Hinkle1"></a>Hinkle G, Morrison HG, Sogin ML (1997) Genes coding for reverse transcriptase, DNA-directed RNA polymerase, and chitin synthase from the microsporidian Spraguea lophii. Biol Bull  193: 250–251.  <ul class="find" data-citedArticleID="1027048"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genes+coding+for+reverse+transcriptase%2C+DNA-directed+RNA+polymerase%2C+and+chitin+synthase+from+the+microsporidian+Spraguea+lophii.&amp;auth=&amp;atitle=Genes+coding+for+reverse+transcriptase%2C+DNA-directed+RNA+polymerase%2C+and+chitin+synthase+from+the+microsporidian+Spraguea+lophii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+coding+for+reverse+transcriptase%2C+DNA-directed+RNA+polymerase%2C+and+chitin+synthase+from+the+microsporidian+Spraguea+lophii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+coding+for+reverse+transcriptase%2C+DNA-directed+RNA+polymerase%2C+and+chitin+synthase+from+the+microsporidian+Spraguea+lophii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001277-Xu1" id="pone.0001277-Xu1"></a>Xu J, Pan G, Fang L, Li J, Tian X, et al.  (2006) The varying microsporidian genome: existence of long-terminal repeat retrotransposon in domesticated silkworm parasite Nosema bombycis. Int J Parasitol  36: 1049–1056.  <ul class="find" data-citedArticleID="1027106" data-doi="10.1016/j.ijpara.2006.09.010"><li><a href="http://dx.doi.org/10.1016/j.ijpara.2006.09.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+varying+microsporidian+genome%3A+existence+of+long-terminal+repeat+retrotransposon+in+domesticated+silkworm+parasite+Nosema+bombycis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+varying+microsporidian+genome%3A+existence+of+long-terminal+repeat+retrotransposon+in+domesticated+silkworm+parasite+Nosema+bombycis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001277-Kellis1" id="pone.0001277-Kellis1"></a>Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES (2003) Sequencing and comparison of yeast species to identify genes and regulatory elements. Nature  423: 241–254.  <ul class="find" data-citedArticleID="1027064" data-doi="10.1038/nature01644"><li><a href="http://dx.doi.org/10.1038/nature01644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001277-Bennetzen1" id="pone.0001277-Bennetzen1"></a>Bennetzen JL, Ramakrishna W (2002) Numerous small rearrangements of gene content, order and orientation differentiate grass genomes. Plant Mol Biol  48: 821–827.  <ul class="find" data-citedArticleID="1027034"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Numerous+small+rearrangements+of+gene+content%2C+order+and+orientation+differentiate+grass+genomes.&amp;auth=&amp;atitle=Numerous+small+rearrangements+of+gene+content%2C+order+and+orientation+differentiate+grass+genomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Numerous+small+rearrangements+of+gene+content%2C+order+and+orientation+differentiate+grass+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Numerous+small+rearrangements+of+gene+content%2C+order+and+orientation+differentiate+grass+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001277-Coghlan1" id="pone.0001277-Coghlan1"></a>Coghlan A, Wolfe KH (2002) Fourfold faster rate of genome rearrangement in nematodes than in Drosophila. Genome Res  12: 857–867.  <ul class="find" data-citedArticleID="1027038" data-doi="10.1101/gr.172702"><li><a href="http://dx.doi.org/10.1101/gr.172702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fourfold+faster+rate+of+genome+rearrangement+in+nematodes+than+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fourfold+faster+rate+of+genome+rearrangement+in+nematodes+than+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001277-Kent1" id="pone.0001277-Kent1"></a>Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D (2003) Evolution's cauldron: duplication, deletion, and rearrangement in the mouse and human genomes. Proc Natl Acad Sci U S A  100: 11484–11489.  <ul class="find" data-citedArticleID="1027066" data-doi="10.1073/pnas.1932072100"><li><a href="http://dx.doi.org/10.1073/pnas.1932072100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution%27s+cauldron%3A+duplication%2C+deletion%2C+and+rearrangement+in+the+mouse+and+human+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution%27s+cauldron%3A+duplication%2C+deletion%2C+and+rearrangement+in+the+mouse+and+human+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001277-Nilsen1" id="pone.0001277-Nilsen1"></a>Nilsen F, Endresen C, Hordvik I (1998) Molecular phylogeny of microsporidians with particular reference to species that infect the muscles of fish. J Eukaryot Microbiol  45: 535–543.  <ul class="find" data-citedArticleID="1027074" data-doi="10.1111/j.1550-7408.1998.tb05113.x"><li><a href="http://dx.doi.org/10.1111/j.1550-7408.1998.tb05113.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+of+microsporidians+with+particular+reference+to+species+that+infect+the+muscles+of+fish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+of+microsporidians+with+particular+reference+to+species+that+infect+the+muscles+of+fish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001277-Keogh1" id="pone.0001277-Keogh1"></a>Keogh RS, Seoighe C, Wolfe KH (1998) Evolution of gene order and chromosome number in Saccharomyces, Kluyveromyces and related fungi. Yeast  14: 443–457.  <ul class="find" data-citedArticleID="1027068"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+of+gene+order+and+chromosome+number+in+Saccharomyces%2C+Kluyveromyces+and+related+fungi.&amp;auth=&amp;atitle=Evolution+of+gene+order+and+chromosome+number+in+Saccharomyces%2C+Kluyveromyces+and+related+fungi." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+gene+order+and+chromosome+number+in+Saccharomyces%2C+Kluyveromyces+and+related+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+gene+order+and+chromosome+number+in+Saccharomyces%2C+Kluyveromyces+and+related+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001277-Zdobnov1" id="pone.0001277-Zdobnov1"></a>Zdobnov EM, von Mering C, Letunic I, Torrents D, Suyama M, et al.  (2002) Comparative genome and proteome analysis of Anopheles gambiae and Drosophila melanogaster. Science  298: 149–159.  <ul class="find" data-citedArticleID="1027108" data-doi="10.1126/science.1077061"><li><a href="http://dx.doi.org/10.1126/science.1077061" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genome+and+proteome+analysis+of+Anopheles+gambiae+and+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genome+and+proteome+analysis+of+Anopheles+gambiae+and+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001277-Graham1" id="pone.0001277-Graham1"></a>Graham A, Papalopulu N, Krumlauf R (1989) The murine and Drosophila homeobox gene complexes have common features of organization and expression. Cell  57: 367–378.  <ul class="find" data-citedArticleID="1027046" data-doi="10.1016/0092-8674(89)90912-4"><li><a href="http://dx.doi.org/10.1016/0092-8674(89)90912-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+murine+and+Drosophila+homeobox+gene+complexes+have+common+features+of+organization+and+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+murine+and+Drosophila+homeobox+gene+complexes+have+common+features+of+organization+and+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001277-Tamames1" id="pone.0001277-Tamames1"></a>Tamames J (2001) Evolution of gene order conservation in prokaryotes. Genome Biol  2: RESEARCH0020.  <ul class="find" data-citedArticleID="1027096" data-doi="10.1186/gb-2001-2-6-research0020"><li><a href="http://dx.doi.org/10.1186/gb-2001-2-6-research0020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+gene+order+conservation+in+prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+gene+order+conservation+in+prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001277-Slamovits3" id="pone.0001277-Slamovits3"></a>Slamovits CH, Burri L, Keeling PJ (2006) Characterization of a divergent Sec61beta gene in microsporidia. J Mol Biol  359: 1196–1202.  <ul class="find" data-citedArticleID="1027092" data-doi="10.1016/j.jmb.2006.04.028"><li><a href="http://dx.doi.org/10.1016/j.jmb.2006.04.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+divergent+Sec61beta+gene+in+microsporidia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+divergent+Sec61beta+gene+in+microsporidia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001277-Burt1" id="pone.0001277-Burt1"></a>Burt DW, Bruley C, Dunn IC, Jones CT, Ramage A, et al.  (1999) The dynamics of chromosome evolution in birds and mammals. Nature  402: 411–413.  <ul class="find" data-citedArticleID="1027036" data-doi="10.1038/46555"><li><a href="http://dx.doi.org/10.1038/46555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dynamics+of+chromosome+evolution+in+birds+and+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dynamics+of+chromosome+evolution+in+birds+and+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001277-Ranz1" id="pone.0001277-Ranz1"></a>Ranz JM, Casals F, Ruiz A (2001) How malleable is the eukaryotic genome? Extreme rate of chromosomal rearrangement in the genus Drosophila. Genome Res  11: 230–239.  <ul class="find" data-citedArticleID="1027082" data-doi="10.1101/gr.162901"><li><a href="http://dx.doi.org/10.1101/gr.162901" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+malleable+is+the+eukaryotic+genome%3F+Extreme+rate+of+chromosomal+rearrangement+in+the+genus+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+malleable+is+the+eukaryotic+genome%3F+Extreme+rate+of+chromosomal+rearrangement+in+the+genus+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001277-Zhang1" id="pone.0001277-Zhang1"></a>Zhang Q, Singh I, Sheoran A, Feng X, Nunnari J, et al.  (2005) Production and characterization of monoclonal antibodies against Enterocytozoon bieneusi purified from rhesus macaques. Infect Immun  73: 5166–5172.  <ul class="find" data-citedArticleID="1027110" data-doi="10.1128/iai.73.8.5166-5172.2005"><li><a href="http://dx.doi.org/10.1128/iai.73.8.5166-5172.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Production+and+characterization+of+monoclonal+antibodies+against+Enterocytozoon+bieneusi+purified+from+rhesus+macaques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Production+and+characterization+of+monoclonal+antibodies+against+Enterocytozoon+bieneusi+purified+from+rhesus+macaques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001277-Sambrook1" id="pone.0001277-Sambrook1"></a>Sambrook J, Fritsch EF, Maniatis T (1989) Molecular cloning: a laboratory manual, 2nd ed.: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0001277-Altschul1" id="pone.0001277-Altschul1"></a>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J Mol Biol  215: 403–410.  <ul class="find" data-citedArticleID="1027030" data-doi="10.1016/s0022-2836(05)80360-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80360-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Basic+local+alignment+search+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Basic+local+alignment+search+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001277-Altschul2" id="pone.0001277-Altschul2"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="1027032" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.XML" value="73395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.PDF" value="500388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g001.PNG_L" value="492735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g001.PNG_M" value="63155"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g001.PNG_S" value="8915"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g001.TIF" value="834508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g001.PNG_I" value="21941"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g002.PNG_L" value="1003577"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g002.PNG_M" value="131802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g002.PNG_S" value="13873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g002.TIF" value="2748986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g002.PNG_I" value="49280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g003.PNG_L" value="1110458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g003.PNG_M" value="143315"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g003.PNG_S" value="13473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g003.TIF" value="3052964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g003.PNG_I" value="53109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g004.PNG_L" value="946949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g004.PNG_M" value="122679"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g004.PNG_S" value="13730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g004.TIF" value="2587566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g004.PNG_I" value="45477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g005.PNG_L" value="532867"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g005.PNG_M" value="76575"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g005.PNG_S" value="11262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g005.TIF" value="1325186"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.g005.PNG_I" value="26253"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.s001.DOC" value="90112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.s002.DOC" value="458240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001277.s003.DOC" value="373248"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001277&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001277" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001277&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001277&volume=&issue=&title=Patterns of Genome Evolution among the Microsporidian Parasites Encephalitozoon cuniculi, Antonospora locustae and Enterocytozoon bieneusi&author_name=Nicolas%20Corradi%2C%20Donna%20E.%20Akiyoshi%2C%20Hilary%20G.%20Morrison%2C%20Xiaochuan%20Feng%2C%20Louis%20M.%20Weiss%2C%20Saul%20Tzipori%2C%20Patrick%20J.%20Keeling&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001277" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001277" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001277" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001277&amp;t=Patterns%20of%20Genome%20Evolution%20among%20the%20Microsporidian%20Parasites%20Encephalitozoon%20cuniculi%2C%20Antonospora%20locustae%20and%20Enterocytozoon%20bieneusi" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001277&title=Patterns%20of%20Genome%20Evolution%20among%20the%20Microsporidian%20Parasites%20Encephalitozoon%20cuniculi%2C%20Antonospora%20locustae%20and%20Enterocytozoon%20bieneusi&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001277&amp;title=Patterns%20of%20Genome%20Evolution%20among%20the%20Microsporidian%20Parasites%20Encephalitozoon%20cuniculi%2C%20Antonospora%20locustae%20and%20Enterocytozoon%20bieneusi" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001277" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001277&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Patterns of Genome Evolution among the Microsporidian Parasites Encephalitozoon cuniculi, Antonospora locustae and Enterocytozoon bieneusi';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001277';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Patterns%20of%20Genome%20Evolution%20among%20the%20Microsporidian%20Parasites%20Encephalitozoon%20cuniculi%2C%20Antonospora%20locustae%20and%20Enterocytozoon%20bieneusi http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001277" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001277" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="25386"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Comparative+genomics%22" title="Search for articles in the subject area:'Comparative genomics'"><div class="flagText">Comparative genomics</div></a>
              <div data-categoryid="32937" data-articleid="25386"
                   data-categoryname="Comparative genomics"
                   class="flagImage" title="Flag 'Comparative genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Conservation+science%22" title="Search for articles in the subject area:'Conservation science'"><div class="flagText">Conservation science</div></a>
              <div data-categoryid="40107" data-articleid="25386"
                   data-categoryname="Conservation science"
                   class="flagImage" title="Flag 'Conservation science' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="25386"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+genomics%22" title="Search for articles in the subject area:'Fungal genomics'"><div class="flagText">Fungal genomics</div></a>
              <div data-categoryid="33097" data-articleid="25386"
                   data-categoryname="Fungal genomics"
                   class="flagImage" title="Flag 'Fungal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+prediction%22" title="Search for articles in the subject area:'Gene prediction'"><div class="flagText">Gene prediction</div></a>
              <div data-categoryid="40279" data-articleid="25386"
                   data-categoryname="Gene prediction"
                   class="flagImage" title="Flag 'Gene prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25386"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+library+construction%22" title="Search for articles in the subject area:'Genomic library construction'"><div class="flagText">Genomic library construction</div></a>
              <div data-categoryid="20959" data-articleid="25386"
                   data-categoryname="Genomic library construction"
                   class="flagImage" title="Flag 'Genomic library construction' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4627'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7454'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4232&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=15389">Comments from Faculty of 1000 Biology</a><br>Posted by NiyazAhmed</p>
      <p><a href="/annotation/listThread.action?root=8725">This article has been evaluated for Faculty of 1000 Biology</a><br>Posted by ncorradi</p>
      <p><a href="/annotation/listThread.action?root=17417">Referee Comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
