

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Accelerated FoxP2 Evolution in Echolocating Bats</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000900"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000900" />

    <meta name="citation_title" content="Accelerated FoxP2 Evolution in Echolocating Bats"/>
    <meta itemprop="name" content="Accelerated FoxP2 Evolution in Echolocating Bats"/>

      <meta name="citation_author" content="Gang Li"/>
            <meta name="citation_author_institution" content="School of Life Science, East China Normal University, Shanghai, China"/>
            <meta name="citation_author_institution" content="Institute of Zoology and Graduate University, Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Jinhong Wang"/>
            <meta name="citation_author_institution" content="School of Life Science, East China Normal University, Shanghai, China"/>
      <meta name="citation_author" content="Stephen J. Rossiter"/>
            <meta name="citation_author_institution" content="School of Biological and Chemical Sciences, Queen Mary, University of London, London, United Kingdom"/>
      <meta name="citation_author" content="Gareth Jones"/>
            <meta name="citation_author_institution" content="School of Biological Sciences, University of Bristol, Bristol, United Kingdom"/>
      <meta name="citation_author" content="Shuyi Zhang"/>
            <meta name="citation_author_institution" content="School of Life Science, East China Normal University, Shanghai, China"/>

    <meta name="citation_date" content="2007/9/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000900.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e900"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A forkhead-domain gene is mutated in a severe speech and language disorder.; citation_author=CSL Lai; citation_author=SE Fisher; citation_author=JA Hurst; citation_author=F Vargha-Khadem; citation_author=AP Monaco; citation_journal_title=Nature; citation_volume=413; citation_number=1; citation_pages=519-523; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Language fMRI abnormalities associated with FOXP2 gene mutation.; citation_author=F Liegeois; citation_author=T Baldeweg; citation_author=A Connelly; citation_author=DG Gadian; citation_author=M Mishkin; citation_journal_title=Nature Neuroscience; citation_volume=6; citation_number=2; citation_pages=1230-1237; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Accelerated protein evolution and origins of human-specific features: FOXP2 as an example.; citation_author=JZ Zhang; citation_author=DM Webb; citation_author=O Podlaha; citation_journal_title=Genetics; citation_volume=162; citation_number=3; citation_pages=1825-1835; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=FoxP2 in song-learning birds and vocal-learning mammals.; citation_author=DM Webb; citation_author=J Zhang; citation_journal_title=Journal of Heredity; citation_volume=96; citation_number=4; citation_pages=212-216; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution of FOXP2, a gene involved in speech and language.; citation_author=W Enard; citation_author=M Przeworski; citation_author=SE Fisher; citation_author=CSL Lai; citation_author=V Wiebe; citation_journal_title=Nature; citation_volume=418; citation_number=5; citation_pages=869-872; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=FOXP2 in focus: what can genes tell us about speech and language?; citation_author=GF Marcus; citation_author=SE Fisher; citation_journal_title=Trends in Cognitive Sciences; citation_volume=7; citation_number=6; citation_pages=257-262; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Altered ultrasonic vocalization in mice with a disruption in the Foxp2 gene.; citation_author=WG Shu; citation_author=JY Cho; citation_author=YH Jiang; citation_author=MH Zhang; citation_author=D Weisz; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=102; citation_number=7; citation_pages=9643-9648; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=FOXP2 expression during brain development coincides with adult sites of pathology in a severe speech and language disorder.; citation_author=CSL Lai; citation_author=D Gerrelli; citation_author=AP Monaco; citation_author=SE Fisher; citation_author=AJ Copp; citation_journal_title=Brain; citation_volume=126; citation_number=8; citation_pages=2455-2462; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=FoxP2 expression in avian vocal learners and non-learners.; citation_author=S Haesler; citation_author=K Wada; citation_author=A Nshdejan; citation_author=EE Morrisey; citation_author=T Lints; citation_journal_title=Journal of Neuroscience; citation_volume=24; citation_number=9; citation_pages=3164-3175; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The eloquent ape: genes, brains and the evolution of language.; citation_author=SE Fisher; citation_author=GF Marcus; citation_journal_title=Nature Reviews Genetics; citation_volume=7; citation_number=10; citation_pages=9-20; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Parallel FoxP1 and FoxP2 expression in songbird and human brain predicts functional interaction.; citation_author=I Teramitsu; citation_author=LC Kudo; citation_author=SE London; citation_author=DH Geschwind; citation_author=SA White; citation_journal_title=Journal of Neuroscience; citation_volume=24; citation_number=11; citation_pages=3152-3163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The Neural Basis of Echolocation in Bats;; citation_author=GD Pollak; citation_author=JH Casseday; citation_number=12; citation_pages=143; citation_date=1989; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Listening in the Dark.; citation_author=D Griffin; citation_number=13; citation_date=1958; citation_publisher=Yale University Press; " />
      <meta name="citation_reference" content="citation_title=Neurobiology of echolocation in bats.; citation_author=CF Moss; citation_author=SR Sinha; citation_journal_title=Current Opinion in Neurobiology; citation_volume=13; citation_number=14; citation_pages=751-758; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Behavioral studies of auditory information processing.; citation_number=15; citation_pages=87-145; citation_date=1995; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=The lower brainstem auditory pathways;; citation_author=E Covey; citation_author=JH Caseday; citation_number=16; citation_pages=235-295; citation_date=1995; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Structure and function of the bat superior olivary complex.; citation_author=B Grothe; citation_author=TJ Park; citation_journal_title=Microscopy Research and Technique; citation_volume=51; citation_number=17; citation_pages=382-402; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Echo-detecting characteristics of neurons in inferior colliculus of anaesthetized bats.; citation_author=G Pollak; citation_author=DM Marsh; citation_author=R Bodenhamer; citation_author=A Souther; citation_journal_title=Science; citation_volume=196; citation_number=18; citation_pages=675-678; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Brain structures of echolocating and nonecholocating bats, derived in vivo from magnetic resonance images.; citation_author=KL Hu; citation_author=YX Li; citation_author=XM Gu; citation_author=H Lei; citation_author=SY Zhang; citation_journal_title=Neuroreport; citation_volume=17; citation_number=19; citation_pages=1743-1746; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=FM signals produce robust paradoxical latency shifts in the bat's inferior colliculus.; citation_author=XM Wang; citation_author=AV Galazyuk; citation_author=AS Feng; citation_journal_title=Journal of Comparative Physiology a-Neuroethology Sensory Neural and Behavioral Physiology; citation_volume=193; citation_number=20; citation_pages=13-20; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The role of the cerebellum in motor control and perception.; citation_author=MG Paulin; citation_journal_title=Brain Behavior and Evolution; citation_volume=41; citation_number=21; citation_pages=39-50; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Combination-sensitive neurons in the medial geniculate-body of the moustached bat - encoding of target range information.; citation_author=JF Olsen; citation_author=N Suga; citation_journal_title=Journal of Neurophysiology; citation_volume=65; citation_number=22; citation_pages=1275-1296; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Neural axis representing target range in the auditory-cortex of the mustache bat.; citation_author=N Suga; citation_author=WE Oneill; citation_journal_title=Science; citation_volume=206; citation_number=23; citation_pages=351-353; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Combination-sensitive neurons in the medial geniculate-body of the moustached bat - encoding of relative velocity information.; citation_author=JF Olsen; citation_author=N Suga; citation_journal_title=Journal of Neurophysiology; citation_volume=65; citation_number=24; citation_pages=1254-1274; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Characterization of Foxp2 and Foxp1 mRNA and protein in the developing and mature brain.; citation_author=RJ Ferland; citation_author=TJ Cherry; citation_author=PO Preware; citation_author=EE Morrisey; citation_author=CA Walsh; citation_journal_title=Journal of Comparative Neurology; citation_volume=460; citation_number=25; citation_pages=266-279; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=FOXP2 and the neuroanatomy of speech and language.; citation_author=F Vargha-Khadem; citation_author=DG Gadian; citation_author=A Copp; citation_author=M Mishkin; citation_journal_title=Nature Reviews Neuroscience; citation_volume=6; citation_number=26; citation_pages=131-138; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Vocal learning by greater spear-nosed bats.; citation_author=JW Boughman; citation_journal_title=Proceedings of the Royal Society of London Series B-Biological Sciences; citation_volume=265; citation_number=27; citation_pages=227-233; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Echolocation calls of bats are influenced by maternal effects and change over a lifetime.; citation_author=G Jones; citation_author=RD Ransome; citation_journal_title=Proceedings of the Royal Society of London Series B-Biological Sciences; citation_volume=252; citation_number=28; citation_pages=125-128; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Base-compositional biases and the bat problem. III. The question of microchiropteran monophyly.; citation_author=JM Hutcheon; citation_author=JAW Kirsch; citation_author=JD Pettigrew; citation_journal_title=Philosophical Transactions of the Royal Society of London Series B-Biological Sciences; citation_volume=353; citation_number=29; citation_pages=607-617; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic relationships among recent chiropteran families and the importance of choosing appropriate out-group taxa.; citation_author=RA Van den Bussche; citation_author=SR Hoofer; citation_journal_title=Journal of Mammalogy; citation_volume=85; citation_number=30; citation_pages=321-330; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Resolution of the early placental mammal radiation using Bayesian phylogenetics.; citation_author=WJ Murphy; citation_author=E Eizirik; citation_author=SJ O'Brien; citation_author=O Madsen; citation_author=M Scally; citation_journal_title=Science; citation_volume=294; citation_number=31; citation_pages=2348-2351; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A nuclear DNA phylogenetic perspective on the evolution of echolocation and historical biogeography of extant bats (Chiroptera).; citation_author=GN Eick; citation_author=DS Jacobs; citation_author=CA Matthee; citation_journal_title=Molecular Biology and Evolution; citation_volume=22; citation_number=32; citation_pages=1869-1886; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A molecular phylogeny for bats illuminates biogeography and the fossil record.; citation_author=EC Teeling; citation_author=MS Springer; citation_author=O Madsen; citation_author=P Bates; citation_author=SJ O'Brien; citation_journal_title=Science; citation_volume=307; citation_number=33; citation_pages=580-584; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular evidence regarding the origin of echolocation and flight in bats.; citation_author=EC Teeling; citation_author=M Scally; citation_author=DJ Kao; citation_author=ML Romagnoli; citation_author=MS Springer; citation_journal_title=Nature; citation_volume=403; citation_number=34; citation_pages=188-192; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Microbat paraphyly and the convergent evolution of a key innovation in Old World rhinolophoid microbats.; citation_author=EC Teeling; citation_author=O Madsen; citation_author=RA Van den Bussche; citation_author=WW de Jong; citation_author=MJ Stanhope; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=99; citation_number=35; citation_pages=1431-1436; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Integrated fossil and molecular data reconstruct bat echolocation.; citation_author=MS Springer; citation_author=EC Teeling; citation_author=O Madsen; citation_author=MJ Stanhope; citation_author=WW de Jong; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=98; citation_number=36; citation_pages=6241-6246; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Placental mammal diversification and the Cretaceous-Tertiary boundary.; citation_author=MS Springer; citation_author=WJ Murphy; citation_author=E Eizirik; citation_author=SJ O'Brien; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=100; citation_number=37; citation_pages=1056-1061; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The evolution of echolocation in bats.; citation_author=G Jones; citation_author=EC Teeling; citation_journal_title=Trends in Ecology & Evolution; citation_volume=21; citation_number=38; citation_pages=149-156; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A maximum likelihood method for detecting functional divergence at individual codon sites, with application to gene family evolution.; citation_author=JP Bielawski; citation_author=ZH Yang; citation_journal_title=Journal of Molecular Evolution; citation_volume=59; citation_number=39; citation_pages=121-132; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Physicochemical constraint violation by missense substitutions mediates impairment of protein function and disease severity.; citation_author=EA Stone; citation_author=A Sidow; citation_journal_title=Genome Research; citation_volume=15; citation_number=40; citation_pages=978-986; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of FOXP2 truncation as a novel cause of developmental speech and language deficits.; citation_author=KD MacDermot; citation_author=E Bonora; citation_author=N Sykes; citation_author=AM Coupe; citation_author=CSL Lai; citation_journal_title=American Journal of Human Genetics; citation_volume=76; citation_number=41; citation_pages=1074-1080; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Speech and language impairment and oromotor dyspraxia due to deletion of 7q31 that involves FOXP2.; citation_author=S Zeesman; citation_author=MJM Nowaczyk; citation_author=I Teshima; citation_author=W Roberts; citation_author=JO Cardy; citation_journal_title=American Journal of Medical Genetics Part A; citation_volume=140A; citation_number=42; citation_pages=509-514; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An evolutionary perspective on FoxP2: strictly for the birds?; citation_author=C Scharff; citation_author=S Haesler; citation_journal_title=Current Opinion in Neurobiology; citation_volume=15; citation_number=43; citation_pages=694-703; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Speech, prosody, and voice characteristics of a mother and daughter with a 7;13 translocation affecting FOXP2.; citation_author=LD Shriberg; citation_author=KJ Ballard; citation_author=JB Tomblin; citation_author=JR Duffy; citation_author=KH Odell; citation_journal_title=Journal of Speech Language and Hearing Research; citation_volume=49; citation_number=44; citation_pages=500-525; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Statistical methods for detecting molecular adaptation.; citation_author=ZH Yang; citation_author=JP Bielawski; citation_journal_title=Trends in Ecology & Evolution; citation_volume=15; citation_number=45; citation_pages=496-503; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Accuracy and power of Bayes prediction of amino acid sites under positive selection.; citation_author=M Anisimova; citation_author=JP Bielawski; citation_author=ZH Yang; citation_journal_title=Molecular Biology and Evolution; citation_volume=19; citation_number=46; citation_pages=950-958; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood methods for detecting adaptive evolution after gene duplication.; citation_author=JP Bielawski; citation_author=ZH Yang; citation_journal_title=Journal of Structural and Functional Genomics; citation_volume=3; citation_number=47; citation_pages=201-212; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Effects of Linkage on Rates of Molecular Evolution.; citation_author=CW Birky; citation_author=JB Walsh; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=85; citation_number=48; citation_pages=6414-6418; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Audio-vocal learning in a nonhuman mammal - the lesser spear-nosed bat Phyllostomus discolor.; citation_author=KH Esser; citation_journal_title=Neuroreport; citation_volume=5; citation_number=49; citation_pages=1718-1720; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Intra-individual variation in the vocalized frequency of the Taiwanese leaf-nosed bat, Hipposideros terasensis, influenced by conspecific colony members.; citation_author=S Hiryu; citation_author=K Katsura; citation_author=T Nagato; citation_author=H Yamazaki; citation_author=LK Lin; citation_journal_title=Journal of Comparative Physiology a-Neuroethology Sensory Neural and Behavioral Physiology; citation_volume=192; citation_number=50; citation_pages=807-815; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Song-type sharing in song sparrows: implications for repertoire function and song learning.; citation_author=M Hughes; citation_author=S Nowicki; citation_author=WA Searcy; citation_author=S Peters; citation_journal_title=Behavioral Ecology and Sociobiology; citation_volume=42; citation_number=51; citation_pages=437-446; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A preference for own-subspecies' song guides vocal learning in a song bird.; citation_author=DA Nelson; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=97; citation_number=52; citation_pages=13348-13353; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Topographic organization of a forebrain pathway involved with vocal learning in zebra finches.; citation_author=F Johnson; citation_author=MM Sablan; citation_author=SW Bottjer; citation_journal_title=Journal of Comparative Neurology; citation_volume=358; citation_number=53; citation_pages=260-278; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Comparative approaches to the avian song system.; citation_author=EA Brenowitz; citation_journal_title=Journal of Neurobiology; citation_volume=33; citation_number=54; citation_pages=517-531; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Learned birdsong and the neurobiology of human language. Behavioral Neurobiology of Birdsong.; citation_author=ED Jarvis; citation_number=55; citation_pages=749-777; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Killer whales are capable of vocal learning.; citation_author=AD Foote; citation_author=RM Griffin; citation_author=D Howitt; citation_author=L Larsson; citation_author=PJO Miller; citation_journal_title=Biology Letters; citation_volume=2; citation_number=56; citation_pages=509-512; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Song copying by humpback whales: themes and variations.; citation_author=E Mercado; citation_author=LM Herman; citation_author=AA Pack; citation_journal_title=Animal Cognition; citation_volume=8; citation_number=57; citation_pages=93-102; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Whistle matching in wild bottlenose dolphins (Tursiops truncatus).; citation_author=VM Janik; citation_journal_title=Science; citation_volume=289; citation_number=58; citation_pages=1355-1357; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Vocal learning in mammals.; citation_author=VM Janik; citation_author=PJB Slater; citation_journal_title=Advances in the Study of Behavior; citation_volume=26; citation_number=59; citation_pages=59-99; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Elephants are capable of vocal learning.; citation_author=JH Poole; citation_author=PL Tyack; citation_author=AS Stoeger-Horwath; citation_author=S Watwood; citation_journal_title=Nature; citation_volume=434; citation_number=60; citation_pages=455-456; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Functional aspects of echolocation in dolphins.; citation_author=DL Herzing; citation_author=EM Santos; citation_number=61; citation_date=2004; citation_publisher=University of Chicago; " />
      <meta name="citation_reference" content="citation_title=Eocene evolution of whale hearing.; citation_author=S Nummela; citation_author=JGM Thewissen; citation_author=S Bajpai; citation_author=ST Hussain; citation_author=K Kumar; citation_journal_title=Nature; citation_volume=430; citation_number=62; citation_pages=776-778; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A bizarre new toothed mysticete (Cetacea) from Australia and the early evolution of baleen whales.; citation_author=EMG Fitzgerald; citation_journal_title=Proceedings of the Royal Society B-Biological Sciences; citation_volume=273; citation_number=63; citation_pages=2955-2963; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Roosting, vocalizations, and foraging by the African bat, Nycteris thebaica.; citation_author=H Aldridge; citation_author=M Obrist; citation_author=HG Merriam; citation_author=MB Fenton; citation_journal_title=Journal of Mammalogy; citation_volume=71; citation_number=64; citation_pages=242-246; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Transcriptional and DNA binding activity of the Foxp1/2/4 family is modulated by heterotypic and homotypic protein interactions.; citation_author=SR Li; citation_author=J Weidenfeld; citation_author=EE Morrisey; citation_journal_title=Molecular and Cellular Biology; citation_volume=24; citation_number=65; citation_pages=809-822; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The whn transcription factor encoded by the nude locus contains an evolutionarily conserved and functionally indispensable activation domain.; citation_author=K Schuddekopf; citation_author=M Schorpp; citation_author=T Boehm; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=93; citation_number=66; citation_pages=9661-9664; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=F Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Research; citation_volume=25; citation_number=67; citation_pages=4876-4882; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=MEGA3: Integrated software for molecular evolutionary genetics analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Briefings in Bioinformatics; citation_volume=5; citation_number=68; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions.; citation_author=M Nei; citation_author=T Gojobori; citation_journal_title=Molecular Biology and Evolution; citation_volume=3; citation_number=69; citation_pages=418-426; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=DnaSP version 3: an integrated program for molecular population genetics and molecular evolution analysis.; citation_author=J Rozas; citation_author=R Rozas; citation_journal_title=Bioinformatics; citation_volume=15; citation_number=70; citation_pages=174-175; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood; citation_author=ZH Yang; citation_journal_title=Computer Applications in the Biosciences; citation_volume=13; citation_number=71; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Likelihood ratio tests for detecting positive selection and application to primate lysozyme evolution.; citation_author=ZH Yang; citation_journal_title=Molecular Biology and Evolution; citation_volume=15; citation_number=72; citation_pages=568-573; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=PhyloDraw: a phylogenetic tree drawing system.; citation_author=JH Choi; citation_author=HY Jung; citation_author=HS Kim; citation_author=HG Cho; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=73; citation_pages=1056-1058; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level.; citation_author=JZ Zhang; citation_author=R Nielsen; citation_author=ZH Yang; citation_journal_title=Molecular Biology and Evolution; citation_volume=22; citation_number=74; citation_pages=2472-2479; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000900" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Accelerated FoxP2 Evolution in Echolocating Bats"/>
    <meta name="twitter:description" content="FOXP2 is a transcription factor implicated in the development and neural control of orofacial coordination, particularly with respect to vocalisation. Observations that orthologues show almost no variation across vertebrates yet differ by two amino acids between humans and chimpanzees have led to speculation that recent evolutionary changes might relate to the emergence of language. Echolocating bats face especially challenging sensorimotor demands, using vocal signals for orientation and often for prey capture. To determine whether mutations in the FoxP2 gene could be associated with echolocation, we sequenced FoxP2 from echolocating and non-echolocating bats as well as a range of other mammal species. We found that contrary to previous reports, FoxP2 is not highly conserved across all nonhuman mammals but is extremely diverse in echolocating bats. We detected divergent selection (a change in selective pressure) at FoxP2 between bats with contrasting sonar systems, suggesting the intriguing possibility of a role for FoxP2 in the evolution and development of echolocation. We speculate that observed accelerated evolution of FoxP2 in bats supports a previously proposed function in sensorimotor coordination."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000900.g003"/>

  <meta property="og:title" content="Accelerated FoxP2 Evolution in Echolocating Bats" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000900" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6491'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1977'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9996&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000900">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000900" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000900&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Accelerated <i>FoxP2</i> Evolution in Echolocating Bats
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gang Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  School of Life Science, East China Normal University, Shanghai, China, 
                  Institute of Zoology and Graduate University, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jinhong Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Life Science, East China Normal University, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen J. Rossiter
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:s.j.rossiter@qmul.ac.uk">s.j.rossiter@qmul.ac.uk</a> (SR); <a href="mailto:syzhang@bio.ecnu.edu.cn">syzhang@bio.ecnu.edu.cn</a> (SZ)</p>

                <p>Affiliation:
                  School of Biological and Chemical Sciences, Queen Mary, University of London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gareth Jones, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biological Sciences, University of Bristol, Bristol, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shuyi Zhang
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:s.j.rossiter@qmul.ac.uk">s.j.rossiter@qmul.ac.uk</a> (SR); <a href="mailto:syzhang@bio.ecnu.edu.cn">syzhang@bio.ecnu.edu.cn</a> (SZ)</p>

                <p>Affiliation:
                  School of Life Science, East China Normal University, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000900</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000900" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000900" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000900" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000900" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000900-g001" data-doi="info:doi/10.1371/journal.pone.0000900" data-uri="info:doi/10.1371/journal.pone.0000900.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000900-g002" data-doi="info:doi/10.1371/journal.pone.0000900" data-uri="info:doi/10.1371/journal.pone.0000900.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000900-g003" data-doi="info:doi/10.1371/journal.pone.0000900" data-uri="info:doi/10.1371/journal.pone.0000900.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000900-t001" data-doi="info:doi/10.1371/journal.pone.0000900" data-uri="info:doi/10.1371/journal.pone.0000900.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000900-t002" data-doi="info:doi/10.1371/journal.pone.0000900" data-uri="info:doi/10.1371/journal.pone.0000900.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000900">Reader Comments (4)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000900" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p><em>FOXP2</em> is a transcription factor implicated in the development and neural control of orofacial coordination, particularly with respect to vocalisation. Observations that orthologues show almost no variation across vertebrates yet differ by two amino acids between humans and chimpanzees have led to speculation that recent evolutionary changes might relate to the emergence of language. Echolocating bats face especially challenging sensorimotor demands, using vocal signals for orientation and often for prey capture. To determine whether mutations in the <em>FoxP2</em> gene could be associated with echolocation, we sequenced <em>FoxP2</em> from echolocating and non-echolocating bats as well as a range of other mammal species. We found that contrary to previous reports, <em>FoxP2</em> is not highly conserved across all nonhuman mammals but is extremely diverse in echolocating bats. We detected divergent selection (a change in selective pressure) at <em>FoxP2</em> between bats with contrasting sonar systems, suggesting the intriguing possibility of a role for <em>FoxP2</em> in the evolution and development of echolocation. We speculate that observed accelerated evolution of <em>FoxP2</em> in bats supports a previously proposed function in sensorimotor coordination.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Li G, Wang J, Rossiter SJ, Jones G, Zhang S (2007) Accelerated <em>FoxP2</em> Evolution in Echolocating Bats. PLoS ONE 2(9):
          e900.
            doi:10.1371/journal.pone.0000900</p><p><strong>Academic Editor: </strong>Hans Ellegren, University of Uppsala, Sweden</p><p><strong>Received:</strong> March 18, 2007; <strong>Accepted:</strong> August 15, 2007; <strong>Published:</strong> September 19, 2007</p><p><strong>Copyright:</strong> © 2007 Li et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was funded by a special grant from East China Normal University to SZ and a Darwin Initiative grant (14-008) to GJ.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The role of the forkhead transcription factor <em>FOXP2</em> in communication was first suggested following the discovery that mutations at this locus cause impaired speech-related motor coordination (orofacial dyspraxia) and comprehension (dysphasia) <a href="#pone.0000900-Lai1">[1]</a>, <a href="#pone.0000900-Liegeois1">[2]</a>. Observations that <em>FOXP2</em> orthologues show almost no variation across distantly related species of reptile, bird and mammal <a href="#pone.0000900-Zhang1">[3]</a>–<a href="#pone.0000900-Enard1">[5]</a>, while the gene differs by two adaptive amino acid changes between humans and chimpanzees <a href="#pone.0000900-Zhang1">[3]</a>, <a href="#pone.0000900-Enard1">[5]</a>, has led to speculation that recent evolutionary changes in <em>FOXP2</em> might be related to the emergence of language <a href="#pone.0000900-Zhang1">[3]</a>, <a href="#pone.0000900-Enard1">[5]</a>, <a href="#pone.0000900-Marcus1">[6]</a>. More recently, the lack of isolation calls produced by <em>FoxP2</em> knockout mice <a href="#pone.0000900-Shu1">[7]</a>, and concordant patterns of expression in the brains of humans, mice <a href="#pone.0000900-Lai2">[8]</a> and songbirds 9], support a wider function in sensorimotor integration and motor learning <a href="#pone.0000900-Fisher1">[10]</a>, <a href="#pone.0000900-Teramitsu1">[11]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Echolocating bats face especially challenging sensorimotor demands, using vocal signals for orientation and prey capture. They can emit echolocation pulses at rates of up to 200 sounds per second, interpret the resulting echoes within time-windows as short as several milliseconds and make motor responses such as changes in flight manoeuvres during these short time intervals <a href="#pone.0000900-Pollak1">[12]</a>. The reception of ultrasonic pulses for orientation, obstacle avoidance and prey capture in flight require complex aural and either orofacial or, in some species, nasofacial coordination <a href="#pone.0000900-Griffin1">[13]</a>–<a href="#pone.0000900-Moss2">[15]</a>. Bat echolocation signals show great diversity, contain complex tonal information, and can be modified in response to echo feedback from targets <a href="#pone.0000900-Griffin1">[13]</a>, <a href="#pone.0000900-Moss1">[14]</a>. Auditory processing of echoes in the cochlea and brainstem involve several nuclei that show hypertrophy and differentiation in bats compared with other mammals, including the anteroventral cochlear nucleus, superior olivary complex and the inferior colliculus <a href="#pone.0000900-Covey1">[16]</a>, <a href="#pone.0000900-Grothe1">[17]</a>, which appears to show specialisations for echo detection <a href="#pone.0000900-Pollak2">[18]</a>–<a href="#pone.0000900-Wang1">[20]</a>. Regions of the cerebellum, including the parafloccular lobes and the medial lobe (homologous to vermal lobules VI-VIII in other mammals), also show expansion in echolocating bats compared to non-echolocating mammals <a href="#pone.0000900-Paulin1">[21]</a>. Pulse-echo delay-tuned neurons encoding target distance are present in the cortex and thalamus (medial geniculate body) <a href="#pone.0000900-Olsen1">[22]</a>–<a href="#pone.0000900-Olsen2">[24]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Several of the neural areas implicated in echolocation in bats have also been shown to be associated with <em>FOXP2</em>/<em>FoxP2</em> expression in the brains of other vertebrate species. Sensory nuclei, including the superior and inferior colliculi, show localised expression in the brains of the adult mouse and human foetus <a href="#pone.0000900-Lai2">[8]</a>, <a href="#pone.0000900-Teramitsu1">[11]</a>, <a href="#pone.0000900-Ferland1">[25]</a>, while the thalamus (including the lateral and medial geniculate bodies) and the cerebellum also show localised <em>FOXP2</em>/<em>FoxP2</em> mRNA expression in the brains of mammals and birds <a href="#pone.0000900-Lai2">[8]</a>, <a href="#pone.0000900-Haesler1">[9]</a>, <a href="#pone.0000900-Ferland1">[25]</a>, <a href="#pone.0000900-VarghaKhadem1">[26]</a>. This overlap, together with the general acceptance that <em>FOXP2</em>/<em>FoxP2</em> functions in sensorimotor coordination, raises the possibility that <em>FoxP2</em> might be involved in the neural circuits that underpin bat echolocation. Furthermore, bats are also one of only a few groups of vertebrates to exhibit vocal learning <a href="#pone.0000900-Boughman1">[27]</a>, <a href="#pone.0000900-Jones1">[28]</a>, a condition that might be a precursor to language, and the evolution of which has been linked to developmental and seasonal patterns of <em>FoxP2</em> expression in the brains (area X) of some songbirds <a href="#pone.0000900-Haesler1">[9]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Our understanding of the phylogenetic relationships among extant bat species has increased considerably over the past two decades <a href="#pone.0000900-Hutcheon1">[29]</a>–<a href="#pone.0000900-Springer1">[36]</a>. Bats diverged from other ordinal groups within the Laurasiatheria around 80 million years ago <a href="#pone.0000900-Springer2">[37]</a> and diversified in the early Eocene <a href="#pone.0000900-Teeling1">[33]</a>. Bats were traditionally split into the two suborders Microchiroptera (bats that echolocate by producing sounds in the larynx) and Megachiroptera (Old World fruit bats represented by the single family Pteropodidae), however, the former is now known not to represent a true clade. Instead, the emerging and highly resolved molecular tree places all megachiropterans (which do not possess laryngeal echolocation) with some members of the microchiropteran superfamily Rhinolophoidea (horseshoe bats and allies) in a proposed new clade called the Yinpterochiroptera <a href="#pone.0000900-Teeling1">[33]</a>, <a href="#pone.0000900-Teeling3">[35]</a>, <a href="#pone.0000900-Springer1">[36]</a>. Other microchiropterans group together in a second clade-the Yangochiroptera-and this new arrangement raises the question of whether laryngeal echolocation has either evolved twice independently or has been lost in the Old World fruit bats <a href="#pone.0000900-Teeling1">[33]</a>, <a href="#pone.0000900-Teeling3">[35]</a>, <a href="#pone.0000900-Springer1">[36]</a>, <a href="#pone.0000900-Jones2">[38]</a>. Although similarities between early fossil bats -which appear basal to all other bats-and extant echolocating species appear to support the latter scenario <a href="#pone.0000900-Jones2">[38]</a>, others have argued in favour of convergence in the two clades <a href="#pone.0000900-Eick1">[32]</a>. A possible loss of laryngeal echolocation appears especially interesting given that one genus of cave roosting fruitbat (<em>Rousettus</em>) has subsequently evolved a simple from echolocation based on tongue clicking <a href="#pone.0000900-Jones2">[38]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Microchiropteran echolocation calls can be broadly classified as either predominately low duty cycle (i.e. signal switched on for typically &lt;20% of the time), frequency modulated (FM) or high duty cycle (&gt;30%) constant frequency (CF) <a href="#pone.0000900-Moss1">[14]</a>. Most lineages within the suborder Yangochiroptera use orally emitted FM signals, whereas the echolocating Yinpterochiroptera includes CF species that are mostly nasal emitters and able to modify their calls to compensate for Doppler shifted echoes induced by their own flight speed <a href="#pone.0000900-Moss1">[14]</a>, <a href="#pone.0000900-Jones2">[38]</a>. However, many exceptions exist, and sonar features such as Doppler shift compensation, nasal emission and whispering echolocation, as well as passive listening used by some gleaning species to localize prey, all show independent origins across phylogentically distant groups <a href="#pone.0000900-Jones2">[38]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>To assess whether mutations in the <em>FoxP2</em> gene could be associated with echolocation in bats, we sequenced <em>FoxP2</em> from echolocating and non-echolocating bat species, as well as a range of other mammals. We isolated mRNA and used reverse transcriptase-mediated polymerase chain reaction (RT–PCR) to amplify the complete gene in bats from six families, as well as representatives from five other mammalian orders and one reptile. Alignments of new sequences with published <em>FoxP2</em> sequences from mouse, primates and birds, and those obtained from archived genomic bacterial artificial chromosome (BAC) libraries, revealed high amino acid conservation across most vertebrates <a href="#pone.0000900-Zhang1">[3]</a>–<a href="#pone.0000900-Enard1">[5]</a> but two highly variable exons in bats. We tested for and found evidence of divergent selection between the two main clades of bats, which have contrasting sonar signals. We also identified two <em>FoxP2</em> exons that showed particularly high levels of variability and therefore surveyed these exons in a much wider range of bat species, as well as 18 cetacean species comprising 15 echolocating toothed whales and dolphins (suborder Odontoceti) and three non-echolocating baleen whales (suborder Mysticeti). This extensive survey confirmed that non-synonymous variation among bats exceeds levels recorded across all other vertebrates, but did not suggest that equivalent accelerated evolution was also a feature of echolocating cetaceans.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Complete <em>FoxP2</em> sequences</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We sequenced the complete <em>FoxP2</em> gene in 13 bat species, 7 other eutherian mammal species and 1 reptile. After combining with archived sequences, including those obtained from genomic BAC libraries, our analyses of complete <em>FoxP2</em> gene sequences were based on 13 bats, 22 additional (non-bat) eutherian mammals, 1 non-eutherian mammal (platypus), two birds and one reptile (see <a href="#pone.0000900.s001">table S1</a>).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Alignments revealed that bats show unparalleled numbers of non-synonymous changes compared with other eutherian mammals, (<a href="#pone-0000900-g001">figure 1</a> and <a href="#pone.0000900.s002">table S2</a> and <a href="#pone.0000900.s003">S3</a>). An analysis of polymorphic sites in 22 sequences of non-bat eutherian mammals revealed a total number of 365 synonymous changes and 20 non-synonymous changes. By comparison, nearly half the number of bat sequences revealed 385 synonymous changes and more than double (44) the number of non-synonymous changes. We also found significantly greater levels of divergence among bats than among other eutherian mammals at both non-synonymous (mean number of pairwise non-synonymous differences = 13.3 versus 4.3, respectively; Kolmogorov-Smirnov test = 0.88, <em>P</em>&lt;0.001) and synonymous sites (mean number of pairwise synonymous differences = 99.4 versus = 78.7, respectively; Kolmogorov-Smirnov test = 0.58, <em>P</em>&lt;0.001) (<a href="#pone-0000900-g002">figure 2</a>). The distribution of non-synonymous changes among bats was not uniform along the gene but instead showed peaks around the coding exons 7 and 17 (<a href="#pone-0000900-g003">figure 3</a>).</p>
<div class="figure" id="pone-0000900-g001"><div class="img"><a name="pone-0000900-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000900" data-uri="info:doi/10.1371/journal.pone.0000900.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000900.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000900.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Radial phylogenetic tree showing relative rates of non-synonymous evolution among 35 eutherian mammals, including 13 bats.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Bats species are given as italicised binomial names. Branch lengths based on maximum-likelihood estimates of non-synonymous substitutions along 1995 bp of the <em>FoxP2</em> gene are superimposed onto a cladogram based on published trees <a href="#pone.0000900-Murphy1">[31]</a>, <a href="#pone.0000900-Teeling1">[33]</a>, <a href="#pone.0000900-Teeling2">[34]</a>. Bat lineages are coloured to show the echolocating Yinpterochiroptera (blue) that mostly possess high duty constant frequency (CF) calls with at least partial Doppler shift compensation, the Yangochiroptera (orange) that mostly possess low duty cycle calls, as well as the absence of laryngeal echolocation in Yinpterochiroptera fruit bats (violet). The taxa analysed are listed in the <a href="#s4">Methods</a> and in <a href="#pone.0000900.s002">table S2</a>.</p>
<span>doi:10.1371/journal.pone.0000900.g001</span></div><div class="figure" id="pone-0000900-g002"><div class="img"><a name="pone-0000900-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000900" data-uri="info:doi/10.1371/journal.pone.0000900.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000900.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000900.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Percentage frequency distributions of pairwise Nei-Gojobori synonymous and non-synonymous differences among 22 eutherian non-bats (a and c, respectively) and among 13 bats (b and d, respectively) based on 1995 bp of <em>FoxP2</em>.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>The taxa analysed are listed in the <a href="#s4">Methods</a> and in <a href="#pone.0000900.s002">table S2</a>.</p>
<span>doi:10.1371/journal.pone.0000900.g002</span></div><div class="figure" id="pone-0000900-g003"><div class="img"><a name="pone-0000900-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000900" data-uri="info:doi/10.1371/journal.pone.0000900.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000900.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000900.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Percentage synonymous (black) and non-synonymous (red) differences per site along the <em>FoxP2</em> gene in 13 bat species (listed in <a href="#pone.0000900.s002">table S2</a>).</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Exon numbers follow <a href="#pone.0000900-Lai1">[ref. 1]</a>.</p>
<span>doi:10.1371/journal.pone.0000900.g003</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Branch-Site models detected no sites under positive selection along branches ancestral to all bats, the two major clades of bats or the Yinpterochiroptera excluding fruit bats (results not shown). On the other hand, we found evidence of a change in the strength of selection <a href="#pone.0000900-Bielawski1">[39]</a> in <em>FoxP2</em> sequences between bats and taxa from related orders within the superorder Laurasiatheria. Significant variation in <em>ω</em> was detected with around 16% of sites identified as evolving under divergent selection (likelihood ratio test = 36.3, df = 3, <em>P</em>&lt;0.001) (<a href="#pone-0000900-t001">table 1</a>). The higher estimated <em>ω</em> for bats over other laurasiatherians (0.23 versus 0.14) reflects the greater number of non-synonymous changes in this group. We repeated the analysis comparing the two major recognised lineages of bats (Yinpterochiroptera and Yangochiroptera) and found evidence for divergent selection at 19% sites (likelihood ratio test = 13.86, df = 3, <em>P</em>&lt;0.01) with respective <em>ω</em> values of 0.23 and 0.14 (<a href="#pone-0000900-t001">table 1</a>). Thus most change in selective constraints operating on the <em>FoxP2</em> protein probably occurred since these lineages diverged. We reconstructed the ancestral amino acid sequences for all bats, for yangochiropterans and for yinpterochiropterans, and confirmed that these are identical to the mammalian consensus sequence as well as the ancestral sequences of the Laurasiatheria and the Euarchontaglires. Both of these comparisons were also significant after fruit bats (Pteropodidae) were excluded from the analyses (for results <a href="#pone-0000900-t001">table 1</a>).</p>
<div class="figure" id="pone-0000900-t001"><div class="img"><a name="pone-0000900-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000900" data-uri="info:doi/10.1371/journal.pone.0000900.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000900.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000900.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Parameter estimates and log likelihood values for tests of divergent selection in <em>FoxP2</em> sequences among selected groups of mammals.</span></strong></p><span>doi:10.1371/journal.pone.0000900.t001</span></div><div class="figure" id="pone-0000900-t002"><div class="img"><a name="pone-0000900-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000900" data-uri="info:doi/10.1371/journal.pone.0000900.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000900.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000900.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000900.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000900.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000900.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Likelihood ratio tests for tests of divergent selection in <em>FoxP2</em> sequences among selected groups of mammals.</span></strong></p><span>doi:10.1371/journal.pone.0000900.t002</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>We used a Multivariate Analysis of Protein Polymorphism <a href="#pone.0000900-Stone1">[40]</a> to predict the physiochemical impact of bat-specific amino acid residues on the <em>FoxP2</em> protein structure and found that, on average, the observed changes had less effect than other possible replacements (mean rank of MAPP score for actual replacements = 3.48 versus an expected value of 4.14±0.003 (SE) based on 10,000 randomisations; <em>Z</em> test; <em>P</em> = 0.02). This finding, together with the absence of stop codons, suggests that overall <em>FoxP2</em> remains constrained in bats despite the comparatively high level of variation. However, at the same time, the distribution of predicted impact scores of observed replacements (mean MAPP score = 20.75±1.32 , N = 51) was positively skewed (Kolmogorov-Smirnov test for normality = 0.13, <em>P</em> = 0.037) with some changes expected to have a considerable influence on physiochemical properties.</p>


<h4>Survey of <em>FoxP2</em> exons 7 and 17</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Based on our alignments of complete gene sequences in bats, we identified two highly variable regions (exons 7 and 17). We undertook targeted sequences of these coding regions in a wider range of echolocating and non-echolocating bat species, as well as echolocating and non-echolocating cetaceans and additional eutherian mammals. In total, we obtained exon data for 65 mammal species, including 42 bats (from 10 families) and 18 cetaceans (from two families), two birds and one reptile (<a href="#pone.0000900.s001">table S1</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Alignments of exons 7 and 17 revealed the extent of accelerated amino acid evolution among bat lineages compared with other vertebrates as well as high conservation among vertebrates in general (see <a href="#pone.0000900.s004">tables S4</a> and <a href="#pone.0000900.s005">S5</a>). At exon 7, although high conservation across non-bats was confirmed <a href="#pone.0000900-Webb1">[4]</a>, we also found previously unreported single amino acid changes in two eulipotyphlan insectivores (Ala283), pig (Asp280) and, the goat (Thr315), though the latter of these was also previously found in the cow <a href="#pone.0000900-Webb1">[4]</a>. All cetaceans shared three amino acid substitutions (Pro302, Ala304 and Met316) but no differences occurred between echolocating toothed whales and non-echolocating baleen whales. The site Met316 was also recorded in all members of the bat family Hipposideridae within the Yinpterochiroptera, while the Pro302 was shared by all members of the family Vespertilionidae in the Yangochiroptera. Of the two mutations in exon 7 previously linked to language development in humans (Asn303 and Ser325), the former was not identified in any other species, while the latter was found to be shared by the two carnivores sequenced as well as a major clade of bats comprising the echolocating members of the Yinpterochiroptera. Exon 17 was invariant across all non-bat eutherian mammals surveyed with the exception of a single non-synonymous substitution (I697M) in the pig. Within bats, however, amino acid variation was considerable, with between up to 8 nonsynonymous changes recorded for a species (<em>Nycteris</em>). Variation at both exons appeared to correspond well to echolocation types/phylogenetic boundaries, with almost complete conservation across groups of confamilial species but contrasting signatures between families.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The <em>FOXP2</em>/<em>FoxP2</em> gene has been of tremendous interest in recent years, with sequence variation and patterns of expression linked to human speech defects <a href="#pone.0000900-Lai1">[1]</a>, <a href="#pone.0000900-MacDermot1">[41]</a>, <a href="#pone.0000900-Zeesman1">[42]</a>, the evolution of language <a href="#pone.0000900-Enard1">[5]</a>, vocal learning in animals <a href="#pone.0000900-Webb1">[4]</a>, <a href="#pone.0000900-Haesler1">[9]</a>, <a href="#pone.0000900-Scharff1">[43]</a>, and sensorimotor performance more generally <a href="#pone.0000900-Shu1">[7]</a>, <a href="#pone.0000900-Teramitsu1">[11]</a>, <a href="#pone.0000900-Shriberg1">[44]</a>. Here we present the most extensive taxonomic survey of <em>FoxP2</em> sequences undertaken to date and report that when compared with other groups, this gene shows greater variation in bats.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Although <em>FoxP2</em> has clearly undergone accelerated evolution in bats compared to other vertebrates, the basis for this change is not clear. Significant divergent selection based on differences in the <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratios (<em>ω</em>) was detected between laurasiatherians and bats, and also between the two major clades of bats, indicative of a change in the strength of natural selection between these groups. In these tests, higher <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratios were found in all bats and the clade Yinpterochiroptera, respectively, and values were almost identical, indicating that the detected change in selective pressure has probably occurred since the Yangochiroptera and Yinpterochiroptera diverged. Furthermore, both analyses gave similar results when fruit bats were removed, and thus the divergent selection cannot be attributed to the inclusion of taxa that might have lost the ability (and need) to echolocate.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The higher <em>d</em><sub>N</sub>/<em>d</em><sub>S</sub> ratio in the Yinpterochiroptera could arise from either a relaxation in selection pressure or a short burst of Darwinian selection. Although explicit tests for positive selection proved negative, the power of such tests to detect positive selection where sequence variability is low is very limited and prone to false negatives (type 2 error) <a href="#pone.0000900-Yang1">[45]</a>–<a href="#pone.0000900-Bielawski2">[47]</a>. It is therefore not possible to rule out the possibility that a period of Darwinian selection has occurred. Indeed several aspects of the system suggest to us that a relaxation in <em>FoxP2</em> in echolocating bats is unlikely. First, the fact that <em>FoxP2</em> is implicated in sensorimotor coordination in mammals and echolocation bats face exceptional challenges in such coordination provide <em>a priori</em> reasons to expect full <em>FoxP2</em> functionality in all echolocating bats. Second, <em>FoxP2</em> is highly conserved across divergent lineages of vertebrates including birds, reptiles and mammals, and there is no reason why to suspect that <em>FoxP2</em> should be any less important in bats than other taxonomic groups. Third, an absence of stop codons, and the non-random distribution of nonsynonymous changes among exons in bat <em>FoxP2</em> sequences, which are also predicted to have less impact on the protein's physiocochemical properties than random changes, all suggest that the amino acid replacements are not due to relaxed selection. For these reasons we contend that the changes we report are more likely to have some adaptive significance. The observed correlation between substitution rates at neighbouring non-synonymous and synonymous sites could arise by hitchhiking effects, in which selection increases the fixation probability of linked weakly deleterious sites <a href="#pone.0000900-Birky1">[48]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>One possible explanation for higher <em>FoxP2</em> variation in bats might relate to their capacity for vocal learning, which has been described in a number of species <a href="#pone.0000900-Boughman1">[27]</a>, <a href="#pone.0000900-Jones1">[28]</a>, <a href="#pone.0000900-Esser1">[49]</a>, <a href="#pone.0000900-Hiryu1">[50]</a>. Following speculation that mutations in humans <em>FoxP2</em> might be associated with the evolution of language, a number of studies have screened <em>FoxP2</em> in vertebrate species that show evidence of vocal learning, which is widely considered to be a critical substrate for the evolution of human language. In addition to bats, vocal learning has also been reported in some lineages of birds <a href="#pone.0000900-Hughes1">[51]</a>–<a href="#pone.0000900-Jarvis1">[55]</a>, cetaceans <a href="#pone.0000900-Foote1">[56]</a>–<a href="#pone.0000900-Janik1">[58]</a> (see ref. <a href="#pone.0000900-Janik2">[59]</a> for a review) and was recently described in the African elephant <a href="#pone.0000900-Poole1">[60]</a>. Evidence from <em>FoxP2</em> expression studies in avian brains have proven inconclusive <a href="#pone.0000900-Haesler1">[9]</a>, and sequence comparisons between non-vocal learners and song-learning birds as well as a small number of vocal learning mammals, have not identified specific mutations related to this ability <a href="#pone.0000900-Webb1">[4]</a>, <a href="#pone.0000900-Haesler1">[9]</a>, <a href="#pone.0000900-Teramitsu1">[11]</a>. The current study, which represents a considerable increase in the number of vocal learning taxa considered, with, 42 bat species, 18 cetacean species and African elephant added, found no consistently shared non-synonymous mutations among these species. Our results thus appear to support earlier studies that found no evidence of specific mutations associated with vocal learning abilities <a href="#pone.0000900-Zhang1">[3]</a>, <a href="#pone.0000900-Webb1">[4]</a>, though we cannot rule out differential expression of this gene. Indeed, though some related bats in the Yinpterochiroptera were found to share a Ser325 residue with humans, the absence of this amino acid in other bat lineages, together with its presence in carnivores (also described previously <a href="#pone.0000900-Zhang1">[3]</a>), suggest that it is unlikely to be associated with vocal learning. Perhaps more interesting is the result that two out of three amino acid substitutions that have occurred in the cetaceans (Pro302 and Met316) have also arisen independently in some bat families. These changes could in theory be linked to the evolution of vocal learning, however, if this were the case, then their absence in other bat genera known to exhibit learning (e.g. <em>Rhinolophus</em>) and the fact that both changes do not occur together in any bat, mean that if variation in <em>FoxP2</em> has a role in vocal learning then it is not straightforward.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>We instead speculate that observed variation in bats might be associated with aspects of echolocation. As mentioned, amino acid variation at both exons 7 and 17 in bats corresponds well to echolocation types/phylogenetic boundaries, with almost complete conservation across groups of confamilial species but contrasting signatures between families. Such high sequence diversity at exons 7 and 17 in bats relative to other mammals, including echolocating cetaceans, indicates that <em>FoxP2</em> plays a role in the sensorimotor demands that are peculiar to bat echolocation rather than echolocation in general. Indeed while all cetaceans shared three amino acid substitutions (Pro302, Ala304 and Met316) no differences were observed between echolocating and non-echolocating baleen cetaceans, also supporting an earlier comparison of one example from each suborder <a href="#pone.0000900-Webb1">[4]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The absence of consistent shared amino acids across echolocating bats and cetaceans is not necessarily inconsistent with our hypothesis that <em>FoxP2</em> variation is linked to echolocation in bats. <em>FoxP2</em> has previously been implicated in sensorimotor disorders with respect to vocalisations and orofacial coordination <a href="#pone.0000900-Lai1">[1]</a>, <a href="#pone.0000900-Shu1">[7]</a>, <a href="#pone.0000900-Shriberg1">[44]</a> and, although whales do show complex vocal behaviour, their sonar signals are emitted through their forehead (melon), thus circumventing the need for such rapid orofacial coordination. Moreover, echolocating whales and dolphins use relatively stereotyped clicks, and show neither the signal variability of bats nor the dynamic modification of signal design in relation to echo feedback <a href="#pone.0000900-Herzing1">[61]</a>.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>A lack of clear differences in nonsynonymous variation at exon 7 between echolocating and non-echolocating lineages in both cetaceans and bats can also be explained if the common ancestor in both groups had echolocation and this ability was subsequently lost in some lineages. This scenario has been previously been suggested for bats <a href="#pone.0000900-Springer1">[36]</a>, though some disagree <a href="#pone.0000900-Eick1">[32]</a>. On the other hand, if <em>FoxP2</em> does function in echolocation, then the signature at exon 17 might be more suggestive that echolocation has evolved twice in bats. Here, fruit bats that do not have laryngeal echolocation (Yinpterochiroptera) (echolocation status 1 and 2 in <a href="#pone.0000900.s002">tables S2</a>, <a href="#pone.0000900.s003">S3</a> <a href="#pone.0000900.s004">S4</a> and <a href="#pone.0000900.s005">S5</a>) show no differences from the dominant eutherian mammal sequence, while two amino acid changes are present in both the related genera <em>Rhinolophus</em> (T641A and D663E) and <em>Hipposideros</em> (V637E and D663E), with further non-synonymous substitutions in the hipposiderid genera <em>Aselliscus</em> (I602M and E657D) and <em>Coelops</em> (I621V, N644D, I655M and E657D). In cetaceans, evidence regarding the evolution of echolocation is also equivocal but fossil evidence indicates that early species were toothed rather than possessing baleen and could hear underwater, though there is no evidence to date of anatomical specialisations associated with echolocation <a href="#pone.0000900-Nummela1">[62]</a>, <a href="#pone.0000900-Fitzgerald1">[63]</a>.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Trends in <em>FoxP2</em> sequence variation with respect to other cases of convergence in bat echolocation are also informative. The methionine to isoleucine substitution (M637I) in <em>Hipposideros</em> has also arisen independently in the yangochiropteran species <em>Pteronotus parnellii</em> but not in its congeners <em>P. quadridens</em> or <em>P. macleayii</em>. It is therefore interesting that <em>P. parnellii</em> is the only species of non-rhinolophoid bat to have evolved Doppler-shift compensation in echolocation, a feature that it shares with the genus <em>Hipposideros</em>. A striking example of rapid <em>FoxP2</em> evolution is that of the yangochiropteran genus <em>Nycteris</em>, which, unlike its closest surveyed relatives, shows independent evolution of nasal emission, multi-harmonic call structure and prey location by passive listening <a href="#pone.0000900-Jones2">[38]</a>, <a href="#pone.0000900-Aldridge1">[64]</a>. <em>Nycteris</em> also shows the highest rate of non-synonymous change at exon 17 of all species surveyed, differing from both the mammalian consensus and its closest relative by eight amino acids (V678A, A680T, T692A, E698A, L699F, D701E, I705E, L710S; <a href="#pone.0000900.s004">table S4</a>). Prior to this study, data on <em>FoxP2</em> in bats was limited to a partial sequence of exon 7 in a single individual (<em>Tadarida sp</em>., family Molossidae) <a href="#pone.0000900-Zhang1">[3]</a>, <a href="#pone.0000900-Webb1">[4]</a>. Two amino acid differences at exon 7 (A307V and L292S) between the published <em>Tadarida</em> sequence and its close relative <em>Chaerephon plicata</em> (family Molossidae) further highlight the high degree of non-synonymous diversity in this order.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Comparatively high levels of <em>FoxP2</em> sequence diversity in bats presented in this study, together with evidence of significant divergent selection and a lack of evidence of positive selection at this time, all point to a need for more work to elucidate the role of this protein in bats. Based on current findings, we suggest that echolocation in bats might have involved the recruitment of the <em>FoxP2</em>, and, more generally, that the protein might function in the mobilisation of downstream genes (or genetic cascades) involved in one or more aspects of the development or regulation of complex sensorimotor coordination. To investigate these speculations, data on the expression of <em>FoxP2</em> in the bat brain, with particular attention to the neural pathways implicated in echolocation, would allow informative comparisons with published results from human, mouse and bird brains. To date, the delineated <em>FoxP2</em> protein domains thought to be involved in DNA binding and transcription regulation <a href="#pone.0000900-Li1">[65]</a> do not include either exon 7 or exon 17, though acidic transcription activation domains have been identified in the C-terminal of other forkhead genes <a href="#pone.0000900-Schuddekopf1">[66]</a>. Therefore the characterisation of these comparably variable domains in bats and other species, the former of which also contains two amino acids implicated in the evolution of human language <a href="#pone.0000900-Enard1">[5]</a>, should also to help assess further the potential role and evolutionary significance of <em>FoxP2</em> in sensorimotor integration and rapid motor learning.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sample collection</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Bat wing membrane biopsies were collected from five localities in China. Some individuals were sacrificed as part of an ongoing surveillance programme for coronaviruses, and their liver and brain tissue stored in liquid nitrogen for RNA preservation. Additional genetic material from bats and other mammals was obtained from tissue banks held at the Institute of Zoology in Beijing and the Institute of Zoology in London.</p>


<h4>Isolation, amplification and sequencing of nucleic acids</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We used RT-PCR to amplify the complete <em>FoxP2</em> coding sequence from mRNA transcripts. Total RNA was isolated from brain and liver tissue using RNAiso kits (TaKaRa, Japan) and reverse-transcribed to first-strand cDNA using Invitrogen SuperScript™ II RT and oligo-dT primers. We designed two primers pairs based on conserved sections to amplify two overlapping regions, spanning exons 2 to 10 (5′-GGT ATT AAG TCA TGA TGC AGG A-3′ and 5′-TCG CAT GTG CAA GTG GGT C-3′) and exons 8 to 17 (5′-GGC TGT GAA AGC ATT TGT GAA G-3′ and 5′-ATG GTT GTG GAG TGG TTA TGA AG-3′). Due to technical problems with amplifying the second region in one species (<em>Megaderma spasma</em>), we also designed the degenerate primer 5′-GAG GTY KCA CAA GYC AGT TCT CAT TCC -3′ to work with primer 5′-GGC TGT GAA AGC ATT TGT GAA G-3′.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>PCR products from cDNA were ligated into a pMD19-T vector (TaKaRa) and cloned. Positive clones were sequenced using Big Dye Terminator on an ABI 3730 DNA sequencer (Applied Biosystems). Complete sequences were reconstructed by overlapping shorter sequences, allowing us to verify that nucleotide changes were not attributable to RT-PCR artefacts.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>From alignments of whole <em>FoxP2</em> gene sequences, we identified two regions of especially high variability that correspond to exon 7 (188 bp) and exon 17 (145 bp). To survey these exons in a wider range of bat species, DNA was isolated with DNeasy kits (Qiagen) for PCR-based targeted amplification. For exon 7, we designed the degenerate primer 5′-CTG GCT TAA GTC CTG CYG ARA TTC-3′ to use with the published intronic primer 5′-GAA TAA AGC TCA TGA GAT TTA CCT GTC-3′ <a href="#pone.0000900-Webb1">[4]</a>. For exon 17, we designed and used three primer pairs (5′-CCA CTT CCC CAT CAC TCT GTT G-3′ and 5′-ATG GTT GTG GAG TGG TTA TGA AG-3′, 5′-CTC TAA CCA GCT CAT GCA ATC-3′ and 5′-ATG GTT GTG GAG TGG TTA TG–3′, 5′-ACT GCT GGG CTG AAG TTG ATT A-3′ and 5′-ATG GTT GTG GAG TGG TTA TG-3′) because no single pair was able to amplify every species. Details of primer combinations are available on request.</p>


<h4>Taxonomic coverage for whole <em>FoxP2</em> gene comparsion</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>For whole gene alignments, we sequenced 2100–2200 bp of the <em>FoxP2</em> gene in one individual each of thirteen species of bat (Pteropodidae: <em>Rousettus leschenaulti</em>, <em>Cynopterus sphinx</em>; Megadermatidae: <em>Megaderma spasma</em>; Rhinolophidae: <em>Rhinolophus ferrumequinum</em>, <em>R. luctus</em>; Hipposideridae: <em>Hipposideros armiger</em>, <em>Aselliscus stoliczkanus</em>, <em>Coelops frithii</em>; Emballonuridae: <em>Taphozous melanopogon</em>; Vespertilionidae: <em>Miniopterus schreibersi</em>, <em>Myotis ricketti</em>, <em>Tylonycteris pachypus</em>; Molossidae: <em>Chaerephon plicata</em>) as well as pig (<em>Sus scrofa</em>), goat (<em>Capra hircus</em>), donkey (<em>Equus asinus</em>), cat (<em>Felis catus</em>), hog-badger (<em>Arctonyx collaris</em>), oriental water shrew (<em>Chimarrogale himalayica</em>), rabbit (<em>Oryctolagus cuniculus</em>) and one reptile species, the red-eared slider terrapin (<em>Trachemys scripta</em>).</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>We also obtained from GenBank the published whole <em>FoxP2</em> sequences of six primate species (human (<em>Homo sapiens</em>; NM_148898), chimpanzee (<em>Pan troglodytes</em>; NM_001009020), gorilla (Gorilla gorilla; AF512948), orang utan (<em>Pongo pygmaeus</em>; AH011319), white-handed gibbon (<em>Hylobates lar</em>; AH011317), rhesus macaque (<em>Macaca mulatta</em>; AF512950) two bird species (zebra finch (<em>Taeniopygia guttata</em>, AY395709) and budgerigar (<em>Melopsittacus undulatus</em>; AY466101)) and mouse (<em>Mus musculus</em>; NM053242). We obtained additional <em>FoxP2</em> sequences by searching archived genomic BAC libraries (National Center for Biotechnology Information) using the BLASTN tool (<a href="http://www.ncbi.nlm.nih.gov/BLAST/">http://www.ncbi.nlm.nih.gov/BLAST/</a>) for olive baboon (<em>Papio anubis</em>; AC155878, AC149459, AC149861, AC157859), common marmoset (<em>Callithrix jacchus</em>; AC151545, AC151033, AC151040), small-eared galago (<em>Otolemur garnettii</em>; AC148960, AC148947, AC151626), gray mouse lemur (<em>Microcebus murinus</em>; AC186909, AC185373, AC187418), African elephant (<em>Loxondonta africana</em>; AC163970, AC164945, AC164509, AC172736), nine-banded armadillo (<em>Dasypus novemcinctus</em>; AC162148, AC152481, AC152132, AC152372, AC152126), African hedgehog (<em>Atelerix albiventris</em>; AC187948, AC183856, AC175228, AC173449, AC186110), common shew (<em>Sorex araneus</em>; AC168041, AC169146, AC168969, AC168968), and one non-eutherian mammal species, the platypus (<em>Ornithorhynchus anatinus</em>; AC155098, AC158426, AC154065).</p>


<h4>Taxonomic coverage for exon 7 and exon 17 comparsion</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>In addition to sequencing the complete <em>FoxP2</em> gene in 13 bat species, we also sequenced exons 7 (188 bp) and 17 (145 bp) in a wider range of bats (Pteropodidae: <em>Eonycteris spelaea</em>, <em>Nyctimene cephalotes</em>, <em>Pteropus rodricensis</em>; Rhinolophidae: <em>Rhinolophus affinis</em>, <em>R. macrotis</em>, <em>R. marshalli</em>, <em>R. osgoodi</em>, <em>R. pearsonii</em>, <em>R. pusillus</em>, <em>R. paradoxolophus</em>, <em>R. sinicus</em>; Hipposideridae: <em>Aselliscus tricuspidatus</em>, <em>Hipposideros larvatus</em>, <em>H. pomona</em>, <em>H. pratti</em>; Megadermatidae: <em>Megaderma lyra</em>; Nycteridae: <em>Nycteris tragata</em>; Phyllostomidae: <em>Carollia perspicillata</em>; Mormoopidae: <em>Pteronotus macleayii</em>, <em>P. parnellii</em>, <em>P. quadridens</em>; Vespertilionidae: <em>Barbastella leucomelas</em>, <em>Ia io</em>, <em>Murina</em> sp., <em>Nyctalus velutinus</em>, <em>Pipistrellus abramus</em>, <em>Plecotus</em> sp., <em>Scotophilus kuhlii</em>, <em>Scotomanes ornatus</em>, <em>Vespertilio sinensis</em>). For 12 bat species (<em>Cynopterus sphinx</em>, <em>Rousettus leschenaulti</em>, <em>Rhinolophus pusillus</em>, <em>R. ferrumequinum</em>, <em>Hipposideros pomona</em>, <em>H. amiger</em>, <em>Coelops frithi</em>, <em>Megaderma lyra</em>, <em>M. spasma</em>, <em>Nycteris tragata</em>, <em>Myotis ricketti</em> and <em>Taphozous melanopogon</em>) DNA was sequenced from multiple individuals (2-5) to confirm the results and in all cases no variation was found between conspecifics.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>In addition, to determine whether rapid <em>FoxP2</em> evolution is a common feature of vertebrate echolocation, we also sequenced 15 echolocating toothed whales (Atlantic white-sided dolphin (<em>Lagenorhynchus acutus</em>), Blainville's beaked whale (<em>Mesoplodon densirostris</em>), bottlenose dolphin (<em>Tursiops truncatus</em>), common dolphin (<em>Delphinus delphis</em>), Cuvieŕs beaked whale (<em>Ziphius cavirostris</em>), harbour porpoise (<em>Phocoena phocoena</em>), killer whale (<em>Orcinus orca</em>), long-finned pilot whale (<em>Globicephala melas</em>), northern bottlenose whale (<em>Hyperoodon ampullatus</em>), pygmy sperm whale (<em>Megaptera novaeangliae</em>), Risso's dolphin (<em>Grampus griseus</em>), Sowerby's beaked whale (<em>Mesoplodon bidens</em>), sperm whale (<em>Physeter catodon</em>), striped dolphin (<em>Stenella coeruleoalba</em>) and white-beaked dolphin (<em>Lagenorhynchus albirostris</em>) as well as three species of non-echolocating baleen whale (fin whale (<em>Balaenoptera physalus</em>), humpback whale (<em>Megaptera novaeangliae</em>) and minke whale (<em>Balaenoptera acutorostrata</em>)). To improve taxonomic coverage further, we also included the red and white giant squirrel (<em>Petaurista alborufus</em>: Rodentia) and the short-eared elephant-shrew (<em>Macroscelides proboscideus</em>: Macroscelidea).</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>GenBank accession numbers of all new complete <em>FoxP2</em> sequences are EU076391-EU076411 inclusive. For exons 7 they EU076412 and EU087927-EU087965 inclusive, and for exon 17 they are EU087966-EU088011 inclusive. Information on the taxonomic groupings of all sequenced species is given in <a href="#pone.0000900.s002">tables S2</a>, <a href="#pone.0000900.s003">S3</a>, <a href="#pone.0000900.s004">S4</a> and <a href="#pone.0000900.s005">S5</a>.</p>


<h4>Statistical analyses</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Nucleotide sequences (1995 bp) were aligned using the software ClustalX <a href="#pone.0000900-Thompson1">[67]</a>, after removing a polyglutamine stretch <a href="#pone.0000900-Enard1">[5]</a>, which shows length variability in bats and carnivores versus other taxa, and were translated in MEGA 3.1 <a href="#pone.0000900-Kumar1">[68]</a>.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>For each pair of complete sequences, we derived the total number of Nei-Gojobori synonymous and nonsynonymous differences <a href="#pone.0000900-Nei1">[69]</a> using the software DnaSP 4.0 <a href="#pone.0000900-Rozas1">[70]</a>. We generated distributions of values of pairwise synonymous and non-synonymous differences for bats and other eutherian mammals and tested for differences between the distributions using a Two-Sample Kolmogorov-Smirnov test in the software S-PLUS 2000 (MathSoft Inc.).</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>We constructed a phylogeny based on published studies <a href="#pone.0000900-Murphy1">[31]</a>, <a href="#pone.0000900-Teeling1">[33]</a>, <a href="#pone.0000900-Teeling2">[34]</a> and used the software PAML <a href="#pone.0000900-Yang2">[71]</a> to derive maximum-likelihood estimates of the number of synonymous and non-synonymous substitutions per site (<em>d</em><sub>S</sub> and <em>d</em><sub>N</sub>) for each branch assuming independent <em>d</em><sub>S</sub>/<em>d</em><sub>N</sub> (<em>ω</em>) ratios per lineage <a href="#pone.0000900-Yang3">[72]</a> (see figure S1). We then constructed a radial tree based on <em>d</em><sub>N</sub> values using the software PHYLODRAW <a href="#pone.0000900-Choi1">[73]</a>.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>We tested for evidence of positive selection along selected lineages using Zhang <em>et al</em>'s modified Branch-Site Model A <a href="#pone.0000900-Zhang2">[74]</a>. In this model, the phylogeny is partitioned into foreground and background branches, with positive selection potentially occurring along the former. Four site classes of codon are assumed, and two of these, evolving under purifying selection (0&lt;<em>ω</em><sub>0</sub>&lt;1) and neutral selection (<em>ω</em><sub>1</sub><em> = </em>1), occur across the tree. Two additional sites evolve under purifying or neutral selection on the background branches but under positive selection (<em>ω</em><sub>2</sub>&gt;1) on the branch under consideration (foreground). A likelihood ratio test (LRT) was conducted by comparing the likelihood result of this model to a null model in which all parameters were the same except that <em>ω</em><sub>2 </sub>is fixed at 1 (neutral evolution) on the foreground. We repeated this analysis four times, to test for positive selection along branches ancestral to all bats, to all Yangochiroptera, to all Yinpterochiroptera and to all fruit bats.</p>
<a id="article1.body1.sec4.sec5.p5" name="article1.body1.sec4.sec5.p5"></a><p>If <em>FoxP2</em> has in some way become recruited into neural pathways involved in bat echolocation, it might be associated with a change in selective pressure between echolocating bats and other mammals, and also between bat lineages that exhibit contrasting forms of echolocation system. Such changes in selection pressure can be small and detectable even when specific tests of positive selection prove negative. To test for evidence of divergent selective pressures among selected pairs of clades, we applied the modified version of Bielawski and Yang's maximum likelihood-based Clade Model C <a href="#pone.0000900-Bielawski1">[39]</a>, <a href="#pone.0000900-Yang2">[71]</a>, which is suited to sequence data with low divergence. This model is based on a branch-site model and allows variation in the <em>d</em><sub>S</sub>/<em>d</em><sub>N</sub> (<em>ω</em>) ratio among sites with a proportion of sites evolving under different selective constraints between a pair of clades <a href="#pone.0000900-Bielawski1">[39]</a>. In the modified Clade Model C, the two clades were assumed to share sites under purifying selection (0&lt;<em>ω</em><sub>0</sub>&lt;1) and neutral evolution (<em>ω</em><sub>1</sub><em> = </em>1), but to differ at a third site class under divergent selection (<em>ω</em><sub>2</sub> <sub>clade 1</sub>≠<em>ω</em><sub>2 clade 2</sub>). We compared the model results with those obtained from the site model M1a (Nearly Neutral) to conduct a LRT with 3 degrees of freedom.</p>
<a id="article1.body1.sec4.sec5.p6" name="article1.body1.sec4.sec5.p6"></a><p>We first test for evidence of divergent selection (i.e. a change in the strength of natural selection) between bats and their nearest known relatives in the superorder Laurasiatheria, represented by a clade comprising donkey, pig, goat, shrew, Asian shrew, hedgehog, cat and badger. We then compared the two major recognised lineages of bats (Yinpterochiroptera and Yangochiroptera). Both tests were repeated but with fruit bat sequences excluded, to assess whether any changes in selection pressure are likely to be due to relaxation resulting from a loss of echolocation. For the same reason we directly compared fruit bats and non-fruit bats within the clade Yinpterochiroptera. We tested the robustness of our results and tested for suboptimal likelihood peaks by rerunning the model under a range of initial values of <em>ω</em><sub>0, </sub>and found that estimates of the proportion of sites under divergent selection were consistent<sub>.</sub></p>
<a id="article1.body1.sec4.sec5.p7" name="article1.body1.sec4.sec5.p7"></a><p>We also undertook a Multivariate Analysis of Protein Polymorphism (MAPP) <a href="#pone.0000900-Stone1">[40]</a> to test whether amino acid variants in bats are likely to have had a greater or lower physiocochemical impact on the <em>FoxP2</em> protein than random non-synonymous changes. Amino acid sequences of <em>FoxP2</em> orthologues of all mammals excluding bats, birds and reptiles were aligned and, to correct for phylogenetic affiliations, were weighted based on PAML-derived branch lengths and known phylogeny topology <a href="#pone.0000900-Stone1">[40]</a>. From the evolutionary variation present, the physiocochemical constraint of each position was calculated based on several properties (hydropathy, polarity, charge, volume and free energy in an alpha helix and beta sheet), and the predicted impact (MAPP score) of all possible variants assessed <a href="#pone.0000900-Stone1">[40]</a>. For a second alignment, comprising bat orthologues, the MAPP score of each amino acid variant was derived, together with the score of all other variants at the same site in the alignment that differed from the ancestral sequence by an equal or fewer number of nucleotide substitutions. All such MAPP scores were then sorted and ranked. We then compared the mean MAPP rank score of observed variants to a null distribution of 10,000 mean rank scores, generated using the software MATHEMATICA (Wolfram Research Inc.) from ranks selected at random from the variable sites. Statistical difference between the means was assessed using a Z test.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000900"></div><a name="pone.0000900.s001" id="pone.0000900.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000900.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Summary of sequences surveyed in the study. Accession numbers are given for sequences obtained from GenBank.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000900.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.17 MB DOC)</p>
<a name="pone.0000900.s002" id="pone.0000900.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000900.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI"><em>FoxP2</em> gene (first half) with variable sites shown. For all species represented in <a href="#pone.0000900.s002">tables S2</a>, <a href="#pone.0000900.s003">S3</a>, <a href="#pone.0000900.s004">S4</a> and <a href="#pone.0000900.s005">S5</a>, abbreviations are given to denote Superordinal (S) group (E (Euarchontaglires), A (Atlantogenata) and L (Laurasiatheria)) and ordinal (O) group (Pr (Primates), Eu (Eulipotyphla), R (Rodentia), L (Lagomorpha), X (Xenarthra), Pr (Proboscidea), M (Macroscelidea), Ar (Artiodatyla), Pe (Perissodactyla), Ca (Carnivora), Ce (Cetacea), Ch (Chiroptera/bats)). Within bats, abbreviations are given to denote family (P = Pteropodidae/fruitbats, R = Rhinolophidae, H = Hipposideridae, Me = Megadermatidae, E = Emballonuridae, M = Molossidae, V = Verspertilionidae, Mo = Mormoopidae, N = Nycteridae, Phyllostomidae) and each family code is followed by either Yi or Ya to denote membership of the newly recognised clade Yinpterochiroptera or Yangochiroptera, respectively. For bats, echolocation status <a href="#pone.0000900-Liegeois1">[2]</a> is indicated by superscripts, numbered as follows: 1 = brief broadband tongue clicks (no laryngeal echolocation), 2 = no echolocation, 3 = constant frequency, 4 = short, broadband, multiharmonic, 5 = narrowband, multiharmonic, 6 = narrowband, dominated by fundamental harmonic, 7 = short, broadband, dominated by fundamental harmonic. Residue numbers are based on the human <em>FOXP2</em> orthologue. Amino acids that were linked to the evolution of language are given in red and those only found in bats are given in blue.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000900.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.18 MB DOC)</p>
<a name="pone.0000900.s003" id="pone.0000900.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000900.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI"><em>FoxP2</em> gene (second half) with variable sites shown. For abbreviations, see <a href="#pone.0000900.s002">Table S2</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000900.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.17 MB DOC)</p>
<a name="pone.0000900.s004" id="pone.0000900.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000900.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Exon 7 of <em>FoxP2</em>. For abbreviations, see <a href="#pone.0000900.s002">Table S2</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000900.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.13 MB DOC)</p>
<a name="pone.0000900.s005" id="pone.0000900.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000900.s005">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Exon 17 of <em>FoxP2</em>. For abbreviations, see <a href="#pone.0000900.s002">Table S2</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000900.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.10 MB DOC)</p>
<a name="pone.0000900.s006" id="pone.0000900.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000900.s006">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Cladogram showing PAML-derived maximum likelihood estimates of rates of non-synonymous and synonymous substitutions. Bat lineages are coloured to show the two major divergent forms of laryngeal echolocation (blue = high duty constant frequency (CF) calls with at least partial Doppler shift compensation, orange = low duty cycle calls, green = short, broadband multi-harmonic, violet = the absence of laryngeal echolocation in fruit bats. Estimates of the number of non-synonymous substitutions (dN) and synonymous substitutions (dS) under a free ratio model are given on the branches as (dN/dS).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000900.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to R. Deaville, D. Gotelli and P. Jepson for samples, X. Tang, P. Hua, W. Metzner, S. Parsons, L. Wei and J. Zhang for help with fieldwork, and S. Le Comber, S. Kundu and M. Sun for technical help. J. Bielawski gave advice on analyses of divergent selection. We thank S. Fisher and two anonymous reviewers for very helpful and detailed comments on an earlier version of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SZ GL. Performed the experiments: GL JW. Analyzed the data: SR GL. Contributed reagents/materials/analysis tools: SZ. Wrote the paper: GJ SR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000900-Lai1" id="pone.0000900-Lai1"></a>Lai CSL, Fisher SE, Hurst JA, Vargha-Khadem F, Monaco AP (2001) A forkhead-domain gene is mutated in a severe speech and language disorder. Nature  413: 519–523.  <ul class="find" data-citedArticleID="992778" data-doi="10.1038/35097076"><li><a href="http://dx.doi.org/10.1038/35097076" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+forkhead-domain+gene+is+mutated+in+a+severe+speech+and+language+disorder." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+forkhead-domain+gene+is+mutated+in+a+severe+speech+and+language+disorder.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000900-Liegeois1" id="pone.0000900-Liegeois1"></a>Liegeois F, Baldeweg T, Connelly A, Gadian DG, Mishkin M, et al.  (2003) Language fMRI abnormalities associated with FOXP2 gene mutation. Nature Neuroscience  6: 1230–1237.  <ul class="find" data-citedArticleID="992784" data-doi="10.1038/nn1138"><li><a href="http://dx.doi.org/10.1038/nn1138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Language+fMRI+abnormalities+associated+with+FOXP2+gene+mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Language+fMRI+abnormalities+associated+with+FOXP2+gene+mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000900-Zhang1" id="pone.0000900-Zhang1"></a>Zhang JZ, Webb DM, Podlaha O (2002) Accelerated protein evolution and origins of human-specific features: FOXP2 as an example. Genetics  162: 1825–1835.  <ul class="find" data-citedArticleID="992860"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Accelerated+protein+evolution+and+origins+of+human-specific+features%3A+FOXP2+as+an+example.&amp;auth=&amp;atitle=Accelerated+protein+evolution+and+origins+of+human-specific+features%3A+FOXP2+as+an+example." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accelerated+protein+evolution+and+origins+of+human-specific+features%3A+FOXP2+as+an+example." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accelerated+protein+evolution+and+origins+of+human-specific+features%3A+FOXP2+as+an+example.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000900-Webb1" id="pone.0000900-Webb1"></a>Webb DM, Zhang J (2005) FoxP2 in song-learning birds and vocal-learning mammals. Journal of Heredity  96: 212–216.  <ul class="find" data-citedArticleID="992850" data-doi="10.1093/jhered/esi025"><li><a href="http://dx.doi.org/10.1093/jhered/esi025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FoxP2+in+song-learning+birds+and+vocal-learning+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FoxP2+in+song-learning+birds+and+vocal-learning+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000900-Enard1" id="pone.0000900-Enard1"></a>Enard W, Przeworski M, Fisher SE, Lai CSL, Wiebe V, et al.  (2002) Molecular evolution of FOXP2, a gene involved in speech and language. Nature  418: 869–872.  <ul class="find" data-citedArticleID="992736" data-doi="10.1038/nature01025"><li><a href="http://dx.doi.org/10.1038/nature01025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolution+of+FOXP2%2C+a+gene+involved+in+speech+and+language." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolution+of+FOXP2%2C+a+gene+involved+in+speech+and+language.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000900-Marcus1" id="pone.0000900-Marcus1"></a>Marcus GF, Fisher SE (2003) FOXP2 in focus: what can genes tell us about speech and language? Trends in Cognitive Sciences  7: 257–262.  <ul class="find" data-citedArticleID="992788" data-doi="10.1016/s1364-6613(03)00104-9"><li><a href="http://dx.doi.org/10.1016/s1364-6613(03)00104-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FOXP2+in+focus%3A+what+can+genes+tell+us+about+speech+and+language%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FOXP2+in+focus%3A+what+can+genes+tell+us+about+speech+and+language%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000900-Shu1" id="pone.0000900-Shu1"></a>Shu WG, Cho JY, Jiang YH, Zhang MH, Weisz D, et al.  (2005) Altered ultrasonic vocalization in mice with a disruption in the Foxp2 gene. Proceedings of the National Academy of Sciences of the United States of America  102: 9643–9648.  <ul class="find" data-citedArticleID="992824" data-doi="10.1073/pnas.0503739102"><li><a href="http://dx.doi.org/10.1073/pnas.0503739102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Altered+ultrasonic+vocalization+in+mice+with+a+disruption+in+the+Foxp2+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Altered+ultrasonic+vocalization+in+mice+with+a+disruption+in+the+Foxp2+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000900-Lai2" id="pone.0000900-Lai2"></a>Lai CSL, Gerrelli D, Monaco AP, Fisher SE, Copp AJ (2003) FOXP2 expression during brain development coincides with adult sites of pathology in a severe speech and language disorder. Brain  126: 2455–2462.  <ul class="find" data-citedArticleID="992780" data-doi="10.1093/brain/awg247"><li><a href="http://dx.doi.org/10.1093/brain/awg247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FOXP2+expression+during+brain+development+coincides+with+adult+sites+of+pathology+in+a+severe+speech+and+language+disorder." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FOXP2+expression+during+brain+development+coincides+with+adult+sites+of+pathology+in+a+severe+speech+and+language+disorder.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000900-Haesler1" id="pone.0000900-Haesler1"></a>Haesler S, Wada K, Nshdejan A, Morrisey EE, Lints T, et al.  (2004) FoxP2 expression in avian vocal learners and non-learners. Journal of Neuroscience  24: 3164–3175.  <ul class="find" data-citedArticleID="992752" data-doi="10.1523/jneurosci.4369-03.2004"><li><a href="http://dx.doi.org/10.1523/jneurosci.4369-03.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FoxP2+expression+in+avian+vocal+learners+and+non-learners." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FoxP2+expression+in+avian+vocal+learners+and+non-learners.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000900-Fisher1" id="pone.0000900-Fisher1"></a>Fisher SE, Marcus GF (2006) The eloquent ape: genes, brains and the evolution of language. Nature Reviews Genetics  7: 9–20.  <ul class="find" data-citedArticleID="992742" data-doi="10.1038/nrg1747"><li><a href="http://dx.doi.org/10.1038/nrg1747" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+eloquent+ape%3A+genes%2C+brains+and+the+evolution+of+language." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+eloquent+ape%3A+genes%2C+brains+and+the+evolution+of+language.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000900-Teramitsu1" id="pone.0000900-Teramitsu1"></a>Teramitsu I, Kudo LC, London SE, Geschwind DH, White SA (2004) Parallel FoxP1 and FoxP2 expression in songbird and human brain predicts functional interaction. Journal of Neuroscience  24: 3152–3163.  <ul class="find" data-citedArticleID="992840" data-doi="10.1523/jneurosci.5589-03.2004"><li><a href="http://dx.doi.org/10.1523/jneurosci.5589-03.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+FoxP1+and+FoxP2+expression+in+songbird+and+human+brain+predicts+functional+interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+FoxP1+and+FoxP2+expression+in+songbird+and+human+brain+predicts+functional+interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000900-Pollak1" id="pone.0000900-Pollak1"></a>Pollak GD, Casseday JH (1989)  The Neural Basis of Echolocation in Bats;. In: SDB, Burggren W, Heller HC, Ishii S, Langer H, et al., editors. Berlin: Springer-Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0000900-Griffin1" id="pone.0000900-Griffin1"></a>Griffin D (1958) Listening in the Dark. New Haven: Yale University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000900-Moss1" id="pone.0000900-Moss1"></a>Moss CF, Sinha SR (2003) Neurobiology of echolocation in bats. Current Opinion in Neurobiology  13: 751–758.  <ul class="find" data-citedArticleID="992792" data-doi="10.1016/j.conb.2003.10.016"><li><a href="http://dx.doi.org/10.1016/j.conb.2003.10.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neurobiology+of+echolocation+in+bats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neurobiology+of+echolocation+in+bats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000900-Moss2" id="pone.0000900-Moss2"></a> Moss CF, Schnitzler H-U, editors.  (1995) Behavioral studies of auditory information processing. Berlin: Springer-Verlag. pp. 87–145.  <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0000900-Covey1" id="pone.0000900-Covey1"></a>Covey E, Caseday JH (1995)  The lower brainstem auditory pathways;. In: Popper AN, Fay RR, editors. Heidelberg, New York: Springer. pp. 235–295.  <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0000900-Grothe1" id="pone.0000900-Grothe1"></a>Grothe B, Park TJ (2000) Structure and function of the bat superior olivary complex. Microscopy Research and Technique  51: 382–402.  <ul class="find" data-citedArticleID="992750" data-doi="10.1002/1097-0029(20001115)51:4<382::aid-jemt7&gt;3.0.co;2-7"><li><a href="http://dx.doi.org/10.1002/1097-0029(20001115)51:4<382::aid-jemt7&gt;3.0.co;2-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+of+the+bat+superior+olivary+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+of+the+bat+superior+olivary+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000900-Pollak2" id="pone.0000900-Pollak2"></a>Pollak G, Marsh DM, Bodenhamer R, Souther A (1977) Echo-detecting characteristics of neurons in inferior colliculus of anaesthetized bats. Science  196: 675–678.  <ul class="find" data-citedArticleID="992812" data-doi="10.1126/science.857318"><li><a href="http://dx.doi.org/10.1126/science.857318" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Echo-detecting+characteristics+of+neurons+in+inferior+colliculus+of+anaesthetized+bats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Echo-detecting+characteristics+of+neurons+in+inferior+colliculus+of+anaesthetized+bats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000900-Hu1" id="pone.0000900-Hu1"></a>Hu KL, Li YX, Gu XM, Lei H, Zhang SY (2006) Brain structures of echolocating and nonecholocating bats, derived in vivo from magnetic resonance images. Neuroreport  17: 1743–1746.  <ul class="find" data-citedArticleID="992758" data-doi="10.1097/01.wnr.0000239959.91190.c8"><li><a href="http://dx.doi.org/10.1097/01.wnr.0000239959.91190.c8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Brain+structures+of+echolocating+and+nonecholocating+bats%2C+derived+in+vivo+from+magnetic+resonance+images." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Brain+structures+of+echolocating+and+nonecholocating+bats%2C+derived+in+vivo+from+magnetic+resonance+images.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000900-Wang1" id="pone.0000900-Wang1"></a>Wang XM, Galazyuk AV, Feng AS (2007) FM signals produce robust paradoxical latency shifts in the bat's inferior colliculus. Journal of Comparative Physiology a-Neuroethology Sensory Neural and Behavioral Physiology  193: 13–20.  <ul class="find" data-citedArticleID="992848" data-doi="10.1007/s00359-006-0167-9"><li><a href="http://dx.doi.org/10.1007/s00359-006-0167-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FM+signals+produce+robust+paradoxical+latency+shifts+in+the+bat%27s+inferior+colliculus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FM+signals+produce+robust+paradoxical+latency+shifts+in+the+bat%27s+inferior+colliculus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000900-Paulin1" id="pone.0000900-Paulin1"></a>Paulin MG (1993) The role of the cerebellum in motor control and perception. Brain Behavior and Evolution  41: 39–50.  <ul class="find" data-citedArticleID="992808" data-doi="10.1159/000113822"><li><a href="http://dx.doi.org/10.1159/000113822" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+the+cerebellum+in+motor+control+and+perception." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+the+cerebellum+in+motor+control+and+perception.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000900-Olsen1" id="pone.0000900-Olsen1"></a>Olsen JF, Suga N (1991) Combination-sensitive neurons in the medial geniculate-body of the moustached bat - encoding of target range information. Journal of Neurophysiology  65: 1275–1296.  <ul class="find" data-citedArticleID="992804"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Combination-sensitive+neurons+in+the+medial+geniculate-body+of+the+moustached+bat+-+encoding+of+target+range+information.&amp;auth=&amp;atitle=Combination-sensitive+neurons+in+the+medial+geniculate-body+of+the+moustached+bat+-+encoding+of+target+range+information." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combination-sensitive+neurons+in+the+medial+geniculate-body+of+the+moustached+bat+-+encoding+of+target+range+information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combination-sensitive+neurons+in+the+medial+geniculate-body+of+the+moustached+bat+-+encoding+of+target+range+information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000900-Suga1" id="pone.0000900-Suga1"></a>Suga N, Oneill WE (1979) Neural axis representing target range in the auditory-cortex of the mustache bat. Science  206: 351–353.  <ul class="find" data-citedArticleID="992832" data-doi="10.1126/science.482944"><li><a href="http://dx.doi.org/10.1126/science.482944" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neural+axis+representing+target+range+in+the+auditory-cortex+of+the+mustache+bat." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neural+axis+representing+target+range+in+the+auditory-cortex+of+the+mustache+bat.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000900-Olsen2" id="pone.0000900-Olsen2"></a>Olsen JF, Suga N (1991) Combination-sensitive neurons in the medial geniculate-body of the moustached bat - encoding of relative velocity information. Journal of Neurophysiology  65: 1254–1274.  <ul class="find" data-citedArticleID="992806"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Combination-sensitive+neurons+in+the+medial+geniculate-body+of+the+moustached+bat+-+encoding+of+relative+velocity+information.&amp;auth=&amp;atitle=Combination-sensitive+neurons+in+the+medial+geniculate-body+of+the+moustached+bat+-+encoding+of+relative+velocity+information." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combination-sensitive+neurons+in+the+medial+geniculate-body+of+the+moustached+bat+-+encoding+of+relative+velocity+information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combination-sensitive+neurons+in+the+medial+geniculate-body+of+the+moustached+bat+-+encoding+of+relative+velocity+information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000900-Ferland1" id="pone.0000900-Ferland1"></a>Ferland RJ, Cherry TJ, Preware PO, Morrisey EE, Walsh CA (2003) Characterization of Foxp2 and Foxp1 mRNA and protein in the developing and mature brain. Journal of Comparative Neurology  460: 266–279.  <ul class="find" data-citedArticleID="992740" data-doi="10.1002/cne.10654"><li><a href="http://dx.doi.org/10.1002/cne.10654" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+Foxp2+and+Foxp1+mRNA+and+protein+in+the+developing+and+mature+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+Foxp2+and+Foxp1+mRNA+and+protein+in+the+developing+and+mature+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000900-VarghaKhadem1" id="pone.0000900-VarghaKhadem1"></a>Vargha-Khadem F, Gadian DG, Copp A, Mishkin M (2005) FOXP2 and the neuroanatomy of speech and language. Nature Reviews Neuroscience  6: 131–138.  <ul class="find" data-citedArticleID="992846" data-doi="10.1038/nrn1605"><li><a href="http://dx.doi.org/10.1038/nrn1605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FOXP2+and+the+neuroanatomy+of+speech+and+language." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FOXP2+and+the+neuroanatomy+of+speech+and+language.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000900-Boughman1" id="pone.0000900-Boughman1"></a>Boughman JW (1998) Vocal learning by greater spear-nosed bats. Proceedings of the Royal Society of London Series B-Biological Sciences  265: 227–233.  <ul class="find" data-citedArticleID="992726" data-doi="10.1098/rspb.1998.0286"><li><a href="http://dx.doi.org/10.1098/rspb.1998.0286" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vocal+learning+by+greater+spear-nosed+bats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vocal+learning+by+greater+spear-nosed+bats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000900-Jones1" id="pone.0000900-Jones1"></a>Jones G, Ransome RD (1993) Echolocation calls of bats are influenced by maternal effects and change over a lifetime. Proceedings of the Royal Society of London Series B-Biological Sciences  252: 125–128.  <ul class="find" data-citedArticleID="992772" data-doi="10.1098/rspb.1993.0055"><li><a href="http://dx.doi.org/10.1098/rspb.1993.0055" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Echolocation+calls+of+bats+are+influenced+by+maternal+effects+and+change+over+a+lifetime." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Echolocation+calls+of+bats+are+influenced+by+maternal+effects+and+change+over+a+lifetime.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000900-Hutcheon1" id="pone.0000900-Hutcheon1"></a>Hutcheon JM, Kirsch JAW, Pettigrew JD (1998) Base-compositional biases and the bat problem. III. The question of microchiropteran monophyly. Philosophical Transactions of the Royal Society of London Series B-Biological Sciences  353: 607–617.  <ul class="find" data-citedArticleID="992762" data-doi="10.1098/rstb.1998.0229"><li><a href="http://dx.doi.org/10.1098/rstb.1998.0229" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-compositional+biases+and+the+bat+problem.+III.+The+question+of+microchiropteran+monophyly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-compositional+biases+and+the+bat+problem.+III.+The+question+of+microchiropteran+monophyly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000900-VandenBussche1" id="pone.0000900-VandenBussche1"></a>Van den Bussche RA, Hoofer SR (2004) Phylogenetic relationships among recent chiropteran families and the importance of choosing appropriate out-group taxa. Journal of Mammalogy  85: 321–330.  <ul class="find" data-citedArticleID="992844" data-doi="10.1644/1545-1542(2004)085<0321:prarcf&gt;2.0.co;2"><li><a href="http://dx.doi.org/10.1644/1545-1542(2004)085<0321:prarcf&gt;2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+relationships+among+recent+chiropteran+families+and+the+importance+of+choosing+appropriate+out-group+taxa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+relationships+among+recent+chiropteran+families+and+the+importance+of+choosing+appropriate+out-group+taxa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000900-Murphy1" id="pone.0000900-Murphy1"></a>Murphy WJ, Eizirik E, O'Brien SJ, Madsen O, Scally M, et al.  (2001) Resolution of the early placental mammal radiation using Bayesian phylogenetics. Science  294: 2348–2351.  <ul class="find" data-citedArticleID="992796" data-doi="10.1126/science.1067179"><li><a href="http://dx.doi.org/10.1126/science.1067179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resolution+of+the+early+placental+mammal+radiation+using+Bayesian+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resolution+of+the+early+placental+mammal+radiation+using+Bayesian+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000900-Eick1" id="pone.0000900-Eick1"></a>Eick GN, Jacobs DS, Matthee CA (2005) A nuclear DNA phylogenetic perspective on the evolution of echolocation and historical biogeography of extant bats (Chiroptera). Molecular Biology and Evolution  22: 1869–1886.  <ul class="find" data-citedArticleID="992734" data-doi="10.1093/molbev/msi180"><li><a href="http://dx.doi.org/10.1093/molbev/msi180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+nuclear+DNA+phylogenetic+perspective+on+the+evolution+of+echolocation+and+historical+biogeography+of+extant+bats+%28Chiroptera%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+nuclear+DNA+phylogenetic+perspective+on+the+evolution+of+echolocation+and+historical+biogeography+of+extant+bats+%28Chiroptera%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000900-Teeling1" id="pone.0000900-Teeling1"></a>Teeling EC, Springer MS, Madsen O, Bates P, O'Brien SJ, et al.  (2005) A molecular phylogeny for bats illuminates biogeography and the fossil record. Science  307: 580–584.  <ul class="find" data-citedArticleID="992834" data-doi="10.1126/science.1105113"><li><a href="http://dx.doi.org/10.1126/science.1105113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+molecular+phylogeny+for+bats+illuminates+biogeography+and+the+fossil+record." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+molecular+phylogeny+for+bats+illuminates+biogeography+and+the+fossil+record.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000900-Teeling2" id="pone.0000900-Teeling2"></a>Teeling EC, Scally M, Kao DJ, Romagnoli ML, Springer MS, et al.  (2000) Molecular evidence regarding the origin of echolocation and flight in bats. Nature  403: 188–192.  <ul class="find" data-citedArticleID="992836" data-doi="10.1038/35003188"><li><a href="http://dx.doi.org/10.1038/35003188" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evidence+regarding+the+origin+of+echolocation+and+flight+in+bats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evidence+regarding+the+origin+of+echolocation+and+flight+in+bats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000900-Teeling3" id="pone.0000900-Teeling3"></a>Teeling EC, Madsen O, Van den Bussche RA, de Jong WW, Stanhope MJ, et al.  (2002) Microbat paraphyly and the convergent evolution of a key innovation in Old World rhinolophoid microbats. Proceedings of the National Academy of Sciences of the United States of America  99: 1431–1436.  <ul class="find" data-citedArticleID="992838" data-doi="10.1073/pnas.022477199"><li><a href="http://dx.doi.org/10.1073/pnas.022477199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbat+paraphyly+and+the+convergent+evolution+of+a+key+innovation+in+Old+World+rhinolophoid+microbats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbat+paraphyly+and+the+convergent+evolution+of+a+key+innovation+in+Old+World+rhinolophoid+microbats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000900-Springer1" id="pone.0000900-Springer1"></a>Springer MS, Teeling EC, Madsen O, Stanhope MJ, de Jong WW (2001) Integrated fossil and molecular data reconstruct bat echolocation. Proceedings of the National Academy of Sciences of the United States of America  98: 6241–6246.  <ul class="find" data-citedArticleID="992826" data-doi="10.1073/pnas.111551998"><li><a href="http://dx.doi.org/10.1073/pnas.111551998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrated+fossil+and+molecular+data+reconstruct+bat+echolocation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrated+fossil+and+molecular+data+reconstruct+bat+echolocation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000900-Springer2" id="pone.0000900-Springer2"></a>Springer MS, Murphy WJ, Eizirik E, O'Brien SJ (2003) Placental mammal diversification and the Cretaceous-Tertiary boundary. Proceedings of the National Academy of Sciences of the United States of America  100: 1056–1061.  <ul class="find" data-citedArticleID="992828" data-doi="10.1073/pnas.0334222100"><li><a href="http://dx.doi.org/10.1073/pnas.0334222100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Placental+mammal+diversification+and+the+Cretaceous-Tertiary+boundary." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Placental+mammal+diversification+and+the+Cretaceous-Tertiary+boundary.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000900-Jones2" id="pone.0000900-Jones2"></a>Jones G, Teeling EC (2006) The evolution of echolocation in bats. Trends in Ecology &amp; Evolution  21: 149–156.  <ul class="find" data-citedArticleID="992774" data-doi="10.1016/j.tree.2006.01.001"><li><a href="http://dx.doi.org/10.1016/j.tree.2006.01.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+echolocation+in+bats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+echolocation+in+bats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000900-Bielawski1" id="pone.0000900-Bielawski1"></a>Bielawski JP, Yang ZH (2004) A maximum likelihood method for detecting functional divergence at individual codon sites, with application to gene family evolution. Journal of Molecular Evolution  59: 121–132.  <ul class="find" data-citedArticleID="992720" data-doi="10.1007/s00239-004-2597-8"><li><a href="http://dx.doi.org/10.1007/s00239-004-2597-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+maximum+likelihood+method+for+detecting+functional+divergence+at+individual+codon+sites%2C+with+application+to+gene+family+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+maximum+likelihood+method+for+detecting+functional+divergence+at+individual+codon+sites%2C+with+application+to+gene+family+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000900-Stone1" id="pone.0000900-Stone1"></a>Stone EA, Sidow A (2005) Physicochemical constraint violation by missense substitutions mediates impairment of protein function and disease severity. Genome Research  15: 978–986.  <ul class="find" data-citedArticleID="992830" data-doi="10.1101/gr.3804205"><li><a href="http://dx.doi.org/10.1101/gr.3804205" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physicochemical+constraint+violation+by+missense+substitutions+mediates+impairment+of+protein+function+and+disease+severity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physicochemical+constraint+violation+by+missense+substitutions+mediates+impairment+of+protein+function+and+disease+severity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000900-MacDermot1" id="pone.0000900-MacDermot1"></a>MacDermot KD, Bonora E, Sykes N, Coupe AM, Lai CSL, et al.  (2005) Identification of FOXP2 truncation as a novel cause of developmental speech and language deficits. American Journal of Human Genetics  76: 1074–1080.  <ul class="find" data-citedArticleID="992786" data-doi="10.1086/430841"><li><a href="http://dx.doi.org/10.1086/430841" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+FOXP2+truncation+as+a+novel+cause+of+developmental+speech+and+language+deficits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+FOXP2+truncation+as+a+novel+cause+of+developmental+speech+and+language+deficits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000900-Zeesman1" id="pone.0000900-Zeesman1"></a>Zeesman S, Nowaczyk MJM, Teshima I, Roberts W, Cardy JO, et al.  (2006) Speech and language impairment and oromotor dyspraxia due to deletion of 7q31 that involves FOXP2. American Journal of Medical Genetics Part A  140A: 509–514.  <ul class="find" data-citedArticleID="992858" data-doi="10.1002/ajmg.a.31110"><li><a href="http://dx.doi.org/10.1002/ajmg.a.31110" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Speech+and+language+impairment+and+oromotor+dyspraxia+due+to+deletion+of+7q31+that+involves+FOXP2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Speech+and+language+impairment+and+oromotor+dyspraxia+due+to+deletion+of+7q31+that+involves+FOXP2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000900-Scharff1" id="pone.0000900-Scharff1"></a>Scharff C, Haesler S (2005) An evolutionary perspective on FoxP2: strictly for the birds? Current Opinion in Neurobiology  15: 694–703.  <ul class="find" data-citedArticleID="992818" data-doi="10.1016/j.conb.2005.10.004"><li><a href="http://dx.doi.org/10.1016/j.conb.2005.10.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+evolutionary+perspective+on+FoxP2%3A+strictly+for+the+birds%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+evolutionary+perspective+on+FoxP2%3A+strictly+for+the+birds%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000900-Shriberg1" id="pone.0000900-Shriberg1"></a>Shriberg LD, Ballard KJ, Tomblin JB, Duffy JR, Odell KH, et al.  (2006) Speech, prosody, and voice characteristics of a mother and daughter with a 7;13 translocation affecting FOXP2. Journal of Speech Language and Hearing Research  49: 500–525.  <ul class="find" data-citedArticleID="992822" data-doi="10.1044/1092-4388(2006/038)"><li><a href="http://dx.doi.org/10.1044/1092-4388(2006/038)" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Speech%2C+prosody%2C+and+voice+characteristics+of+a+mother+and+daughter+with+a+7%3B13+translocation+affecting+FOXP2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Speech%2C+prosody%2C+and+voice+characteristics+of+a+mother+and+daughter+with+a+7%3B13+translocation+affecting+FOXP2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000900-Yang1" id="pone.0000900-Yang1"></a>Yang ZH, Bielawski JP (2000) Statistical methods for detecting molecular adaptation. Trends in Ecology &amp; Evolution  15: 496–503.  <ul class="find" data-citedArticleID="992852" data-doi="10.1016/s0169-5347(00)01994-7"><li><a href="http://dx.doi.org/10.1016/s0169-5347(00)01994-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+methods+for+detecting+molecular+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+methods+for+detecting+molecular+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000900-Anisimova1" id="pone.0000900-Anisimova1"></a>Anisimova M, Bielawski JP, Yang ZH (2002) Accuracy and power of Bayes prediction of amino acid sites under positive selection. Molecular Biology and Evolution  19: 950–958.  <ul class="find" data-citedArticleID="992718" data-doi="10.1093/oxfordjournals.molbev.a004152"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004152" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accuracy+and+power+of+Bayes+prediction+of+amino+acid+sites+under+positive+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accuracy+and+power+of+Bayes+prediction+of+amino+acid+sites+under+positive+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000900-Bielawski2" id="pone.0000900-Bielawski2"></a>Bielawski JP, Yang ZH (2003) Maximum likelihood methods for detecting adaptive evolution after gene duplication. Journal of Structural and Functional Genomics  3: 201–212.  <ul class="find" data-citedArticleID="992722" data-doi="10.1007/978-94-010-0263-9_20"><li><a href="http://dx.doi.org/10.1007/978-94-010-0263-9_20" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+likelihood+methods+for+detecting+adaptive+evolution+after+gene+duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+likelihood+methods+for+detecting+adaptive+evolution+after+gene+duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000900-Birky1" id="pone.0000900-Birky1"></a>Birky CW, Walsh JB (1988) Effects of Linkage on Rates of Molecular Evolution. Proceedings of the National Academy of Sciences of the United States of America  85: 6414–6418.  <ul class="find" data-citedArticleID="992724" data-doi="10.1073/pnas.85.17.6414"><li><a href="http://dx.doi.org/10.1073/pnas.85.17.6414" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+Linkage+on+Rates+of+Molecular+Evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+Linkage+on+Rates+of+Molecular+Evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000900-Esser1" id="pone.0000900-Esser1"></a>Esser KH (1994) Audio-vocal learning in a nonhuman mammal - the lesser spear-nosed bat <em>Phyllostomus discolor</em>. Neuroreport  5: 1718–1720.  <ul class="find" data-citedArticleID="992738" data-doi="10.1097/00001756-199409080-00007"><li><a href="http://dx.doi.org/10.1097/00001756-199409080-00007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Audio-vocal+learning+in+a+nonhuman+mammal+-+the+lesser+spear-nosed+bat+Phyllostomus+discolor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Audio-vocal+learning+in+a+nonhuman+mammal+-+the+lesser+spear-nosed+bat+Phyllostomus+discolor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000900-Hiryu1" id="pone.0000900-Hiryu1"></a>Hiryu S, Katsura K, Nagato T, Yamazaki H, Lin LK, et al.  (2006) Intra-individual variation in the vocalized frequency of the Taiwanese leaf-nosed bat, <em>Hipposideros terasensis</em>, influenced by conspecific colony members. Journal of Comparative Physiology a-Neuroethology Sensory Neural and Behavioral Physiology  192: 807–815.  <ul class="find" data-citedArticleID="992756" data-doi="10.1007/s00359-006-0118-5"><li><a href="http://dx.doi.org/10.1007/s00359-006-0118-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intra-individual+variation+in+the+vocalized+frequency+of+the+Taiwanese+leaf-nosed+bat%2C+Hipposideros+terasensis%2C+influenced+by+conspecific+colony+members." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intra-individual+variation+in+the+vocalized+frequency+of+the+Taiwanese+leaf-nosed+bat%2C+Hipposideros+terasensis%2C+influenced+by+conspecific+colony+members.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000900-Hughes1" id="pone.0000900-Hughes1"></a>Hughes M, Nowicki S, Searcy WA, Peters S (1998) Song-type sharing in song sparrows: implications for repertoire function and song learning. Behavioral Ecology and Sociobiology  42: 437–446.  <ul class="find" data-citedArticleID="992760" data-doi="10.1007/s002650050458"><li><a href="http://dx.doi.org/10.1007/s002650050458" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Song-type+sharing+in+song+sparrows%3A+implications+for+repertoire+function+and+song+learning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Song-type+sharing+in+song+sparrows%3A+implications+for+repertoire+function+and+song+learning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000900-Nelson1" id="pone.0000900-Nelson1"></a>Nelson DA (2000) A preference for own-subspecies' song guides vocal learning in a song bird. Proceedings of the National Academy of Sciences of the United States of America  97: 13348–13353.  <ul class="find" data-citedArticleID="992800" data-doi="10.1073/pnas.240457797"><li><a href="http://dx.doi.org/10.1073/pnas.240457797" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+preference+for+own-subspecies%27+song+guides+vocal+learning+in+a+song+bird." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+preference+for+own-subspecies%27+song+guides+vocal+learning+in+a+song+bird.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000900-Johnson1" id="pone.0000900-Johnson1"></a>Johnson F, Sablan MM, Bottjer SW (1995) Topographic organization of a forebrain pathway involved with vocal learning in zebra finches. Journal of Comparative Neurology  358: 260–278.  <ul class="find" data-citedArticleID="992770" data-doi="10.1002/cne.903580208"><li><a href="http://dx.doi.org/10.1002/cne.903580208" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Topographic+organization+of+a+forebrain+pathway+involved+with+vocal+learning+in+zebra+finches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Topographic+organization+of+a+forebrain+pathway+involved+with+vocal+learning+in+zebra+finches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000900-Brenowitz1" id="pone.0000900-Brenowitz1"></a>Brenowitz EA (1997) Comparative approaches to the avian song system. Journal of Neurobiology  33: 517–531.  <ul class="find" data-citedArticleID="992728" data-doi="10.1002/(sici)1097-4695(19971105)33:5<517::aid-neu3&gt;3.0.co;2-7"><li><a href="http://dx.doi.org/10.1002/(sici)1097-4695(19971105)33:5<517::aid-neu3&gt;3.0.co;2-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+approaches+to+the+avian+song+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+approaches+to+the+avian+song+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000900-Jarvis1" id="pone.0000900-Jarvis1"></a>Jarvis ED (2004) Learned birdsong and the neurobiology of human language. Behavioral Neurobiology of Birdsong. pp. 749–777.  <ul class="find-nolinks"></ul></li><li><span class="label">56.
              </span><a name="pone.0000900-Foote1" id="pone.0000900-Foote1"></a>Foote AD, Griffin RM, Howitt D, Larsson L, Miller PJO, et al.  (2006) Killer whales are capable of vocal learning. Biology Letters  2: 509–512.  <ul class="find" data-citedArticleID="992746" data-doi="10.1098/rsbl.2006.0525"><li><a href="http://dx.doi.org/10.1098/rsbl.2006.0525" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Killer+whales+are+capable+of+vocal+learning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Killer+whales+are+capable+of+vocal+learning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000900-Mercado1" id="pone.0000900-Mercado1"></a>Mercado E, Herman LM, Pack AA (2005) Song copying by humpback whales: themes and variations. Animal Cognition  8: 93–102.  <ul class="find" data-citedArticleID="992790" data-doi="10.1007/s10071-004-0238-7"><li><a href="http://dx.doi.org/10.1007/s10071-004-0238-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Song+copying+by+humpback+whales%3A+themes+and+variations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Song+copying+by+humpback+whales%3A+themes+and+variations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000900-Janik1" id="pone.0000900-Janik1"></a>Janik VM (2000) Whistle matching in wild bottlenose dolphins (Tursiops truncatus). Science  289: 1355–1357.  <ul class="find" data-citedArticleID="992764" data-doi="10.1126/science.289.5483.1355"><li><a href="http://dx.doi.org/10.1126/science.289.5483.1355" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whistle+matching+in+wild+bottlenose+dolphins+%28Tursiops+truncatus%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whistle+matching+in+wild+bottlenose+dolphins+%28Tursiops+truncatus%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000900-Janik2" id="pone.0000900-Janik2"></a>Janik VM, Slater PJB (1997) Vocal learning in mammals. Advances in the Study of Behavior  26: 59–99.  <ul class="find" data-citedArticleID="992766" data-doi="10.1016/s0065-3454(08)60377-0"><li><a href="http://dx.doi.org/10.1016/s0065-3454(08)60377-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vocal+learning+in+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vocal+learning+in+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000900-Poole1" id="pone.0000900-Poole1"></a>Poole JH, Tyack PL, Stoeger-Horwath AS, Watwood S (2005) Elephants are capable of vocal learning. Nature  434: 455–456.  <ul class="find" data-citedArticleID="992814" data-doi="10.1038/434455a"><li><a href="http://dx.doi.org/10.1038/434455a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elephants+are+capable+of+vocal+learning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elephants+are+capable+of+vocal+learning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000900-Herzing1" id="pone.0000900-Herzing1"></a>Herzing DL, Santos EM (2004)  Functional aspects of echolocation in dolphins. In: Thomas JA, Moss CF, Vater M, editors. Echolocation in bats and dolphins. Chicago: University of Chicago.   <ul class="find-nolinks"></ul></li><li><span class="label">62.
              </span><a name="pone.0000900-Nummela1" id="pone.0000900-Nummela1"></a>Nummela S, Thewissen JGM, Bajpai S, Hussain ST, Kumar K (2004) Eocene evolution of whale hearing. Nature  430: 776–778.  <ul class="find" data-citedArticleID="992802" data-doi="10.1038/nature02720"><li><a href="http://dx.doi.org/10.1038/nature02720" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eocene+evolution+of+whale+hearing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eocene+evolution+of+whale+hearing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000900-Fitzgerald1" id="pone.0000900-Fitzgerald1"></a>Fitzgerald EMG (2006) A bizarre new toothed mysticete (Cetacea) from Australia and the early evolution of baleen whales. Proceedings of the Royal Society B-Biological Sciences  273: 2955–2963.  <ul class="find" data-citedArticleID="992744" data-doi="10.1098/rspb.2006.3664"><li><a href="http://dx.doi.org/10.1098/rspb.2006.3664" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+bizarre+new+toothed+mysticete+%28Cetacea%29+from+Australia+and+the+early+evolution+of+baleen+whales." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+bizarre+new+toothed+mysticete+%28Cetacea%29+from+Australia+and+the+early+evolution+of+baleen+whales.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000900-Aldridge1" id="pone.0000900-Aldridge1"></a>Aldridge H, Obrist M, Merriam HG, Fenton MB (1990) Roosting, vocalizations, and foraging by the African bat, <em>Nycteris thebaica</em>. Journal of Mammalogy  71: 242–246.  <ul class="find" data-citedArticleID="992716" data-doi="10.2307/1382175"><li><a href="http://dx.doi.org/10.2307/1382175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Roosting%2C+vocalizations%2C+and+foraging+by+the+African+bat%2C+Nycteris+thebaica." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Roosting%2C+vocalizations%2C+and+foraging+by+the+African+bat%2C+Nycteris+thebaica.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000900-Li1" id="pone.0000900-Li1"></a>Li SR, Weidenfeld J, Morrisey EE (2004) Transcriptional and DNA binding activity of the Foxp1/2/4 family is modulated by heterotypic and homotypic protein interactions. Molecular and Cellular Biology  24: 809–822.  <ul class="find" data-citedArticleID="992782" data-doi="10.1128/mcb.24.2.809-822.2004"><li><a href="http://dx.doi.org/10.1128/mcb.24.2.809-822.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+and+DNA+binding+activity+of+the+Foxp1%2F2%2F4+family+is+modulated+by+heterotypic+and+homotypic+protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+and+DNA+binding+activity+of+the+Foxp1%2F2%2F4+family+is+modulated+by+heterotypic+and+homotypic+protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000900-Schuddekopf1" id="pone.0000900-Schuddekopf1"></a>Schuddekopf K, Schorpp M, Boehm T (1996) The whn transcription factor encoded by the nude locus contains an evolutionarily conserved and functionally indispensable activation domain. Proceedings of the National Academy of Sciences of the United States of America  93: 9661–9664.  <ul class="find" data-citedArticleID="992820" data-doi="10.1073/pnas.93.18.9661"><li><a href="http://dx.doi.org/10.1073/pnas.93.18.9661" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+whn+transcription+factor+encoded+by+the+nude+locus+contains+an+evolutionarily+conserved+and+functionally+indispensable+activation+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+whn+transcription+factor+encoded+by+the+nude+locus+contains+an+evolutionarily+conserved+and+functionally+indispensable+activation+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000900-Thompson1" id="pone.0000900-Thompson1"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Research  25: 4876–4882.  <ul class="find" data-citedArticleID="992842" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000900-Kumar1" id="pone.0000900-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: Integrated software for molecular evolutionary genetics analysis and sequence alignment. Briefings in Bioinformatics  5: 150–163.  <ul class="find" data-citedArticleID="992776" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+Integrated+software+for+molecular+evolutionary+genetics+analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+Integrated+software+for+molecular+evolutionary+genetics+analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000900-Nei1" id="pone.0000900-Nei1"></a>Nei M, Gojobori T (1986) Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions. Molecular Biology and Evolution  3: 418–426.  <ul class="find" data-citedArticleID="992798"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.&amp;auth=&amp;atitle=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000900-Rozas1" id="pone.0000900-Rozas1"></a>Rozas J, Rozas R (1999) DnaSP version 3: an integrated program for molecular population genetics and molecular evolution analysis. Bioinformatics  15: 174–175.  <ul class="find" data-citedArticleID="992816" data-doi="10.1093/bioinformatics/15.2.174"><li><a href="http://dx.doi.org/10.1093/bioinformatics/15.2.174" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DnaSP+version+3%3A+an+integrated+program+for+molecular+population+genetics+and+molecular+evolution+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DnaSP+version+3%3A+an+integrated+program+for+molecular+population+genetics+and+molecular+evolution+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000900-Yang2" id="pone.0000900-Yang2"></a>Yang ZH (1997) PAML: a program package for phylogenetic analysis by maximum likelihood Computer Applications in the Biosciences  13: 555–556.  <ul class="find" data-citedArticleID="992854" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000900-Yang3" id="pone.0000900-Yang3"></a>Yang ZH (1998) Likelihood ratio tests for detecting positive selection and application to primate lysozyme evolution. Molecular Biology and Evolution  15: 568–573.  <ul class="find" data-citedArticleID="992856" data-doi="10.1093/oxfordjournals.molbev.a025957"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025957" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Likelihood+ratio+tests+for+detecting+positive+selection+and+application+to+primate+lysozyme+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Likelihood+ratio+tests+for+detecting+positive+selection+and+application+to+primate+lysozyme+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000900-Choi1" id="pone.0000900-Choi1"></a>Choi JH, Jung HY, Kim HS, Cho HG (2000) PhyloDraw: a phylogenetic tree drawing system. Bioinformatics  16: 1056–1058.  <ul class="find" data-citedArticleID="992730" data-doi="10.1093/bioinformatics/16.11.1056"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.11.1056" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PhyloDraw%3A+a+phylogenetic+tree+drawing+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PhyloDraw%3A+a+phylogenetic+tree+drawing+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000900-Zhang2" id="pone.0000900-Zhang2"></a>Zhang JZ, Nielsen R, Yang ZH (2005) Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level. Molecular Biology and Evolution  22: 2472–2479.  <ul class="find" data-citedArticleID="992862" data-doi="10.1093/molbev/msi237"><li><a href="http://dx.doi.org/10.1093/molbev/msi237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.XML" value="137669"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.PDF" value="242752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g001.PNG_L" value="43540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g001.PNG_M" value="109664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g001.PNG_S" value="9382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g001.TIF" value="90650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g001.PNG_I" value="38098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g002.PNG_L" value="14316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g002.PNG_M" value="37367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g002.PNG_S" value="8335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g002.TIF" value="24898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g002.PNG_I" value="13414"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g003.PNG_L" value="45866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g003.PNG_M" value="48547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g003.PNG_S" value="6709"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g003.TIF" value="107196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.g003.PNG_I" value="18924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.t001.PNG_L" value="186881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.t001.PNG_M" value="100517"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.t001.PNG_S" value="13346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.t001.TIF" value="961352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.t001.PNG_I" value="30300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.t002.PNG_L" value="64914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.t002.PNG_M" value="89616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.t002.PNG_S" value="12921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.t002.TIF" value="262680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.t002.PNG_I" value="29565"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.s001.DOC" value="174592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.s002.DOC" value="179200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.s003.DOC" value="172032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.s004.DOC" value="126464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.s005.DOC" value="103936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000900.s006.DOC" value="46592"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000900&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000900" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000900&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000900&volume=&issue=&title=Accelerated FoxP2 Evolution in Echolocating Bats&author_name=Gang%20Li%2C%20Jinhong%20Wang%2C%20Stephen%20J.%20Rossiter%2C%20Gareth%20Jones%2C%20Shuyi%20Zhang&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000900" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000900" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000900" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000900&amp;t=Accelerated%20FoxP2%20Evolution%20in%20Echolocating%20Bats" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000900&title=Accelerated%20FoxP2%20Evolution%20in%20Echolocating%20Bats&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000900&amp;title=Accelerated%20FoxP2%20Evolution%20in%20Echolocating%20Bats" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000900" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000900&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Accelerated FoxP2 Evolution in Echolocating Bats';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000900';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Accelerated%20FoxP2%20Evolution%20in%20Echolocating%20Bats http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000900" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000900" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">






















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bats%22" title="Search for articles in the subject area:'Bats'"><div class="flagText">Bats</div></a>
              <div data-categoryid="42831" data-articleid="24638"
                   data-categoryname="Bats"
                   class="flagImage" title="Flag 'Bats' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Echolocation%22" title="Search for articles in the subject area:'Echolocation'"><div class="flagText">Echolocation</div></a>
              <div data-categoryid="51915" data-articleid="24638"
                   data-categoryname="Echolocation"
                   class="flagImage" title="Flag 'Echolocation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fruit+bats%22" title="Search for articles in the subject area:'Fruit bats'"><div class="flagText">Fruit bats</div></a>
              <div data-categoryid="43119" data-articleid="24638"
                   data-categoryname="Fruit bats"
                   class="flagImage" title="Flag 'Fruit bats' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammals%22" title="Search for articles in the subject area:'Mammals'"><div class="flagText">Mammals</div></a>
              <div data-categoryid="43131" data-articleid="24638"
                   data-categoryname="Mammals"
                   class="flagImage" title="Flag 'Mammals' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Marine+mammals%22" title="Search for articles in the subject area:'Marine mammals'"><div class="flagText">Marine mammals</div></a>
              <div data-categoryid="43861" data-articleid="24638"
                   data-categoryname="Marine mammals"
                   class="flagImage" title="Flag 'Marine mammals' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="24638"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24638"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Whales%22" title="Search for articles in the subject area:'Whales'"><div class="flagText">Whales</div></a>
              <div data-categoryid="32253" data-articleid="24638"
                   data-categoryname="Whales"
                   class="flagImage" title="Flag 'Whales' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=868'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3272'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2008&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=1227">Reply to Steve Rossiter</a><br>Posted by stjpetti</p>
      <p><a href="/annotation/listThread.action?root=10715">Reply to Prof Pettigrew</a><br>Posted by Steve_Rossiter</p>
      <p><a href="/annotation/listThread.action?root=1751">From John D. Pettigrew, University of Queensland</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
