

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Conformational Proofreading: The Impact of Conformational Changes on the Specificity of Molecular Recognition</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000468"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000468" />

    <meta name="citation_title" content="Conformational Proofreading: The Impact of Conformational Changes on the Specificity of Molecular Recognition"/>
    <meta itemprop="name" content="Conformational Proofreading: The Impact of Conformational Changes on the Specificity of Molecular Recognition"/>

      <meta name="citation_author" content="Yonatan Savir"/>
            <meta name="citation_author_institution" content="Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Tsvi Tlusty"/>
            <meta name="citation_author_institution" content="Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel"/>

    <meta name="citation_date" content="2007/5/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000468.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e468"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_author=L Pauling; citation_journal_title=Science; citation_volume=92; citation_number=1; citation_pages=77-79; citation_date=1940; " />
      <meta name="citation_reference" content="citation_title=Some factors in the interpretation of protein denaturation.; citation_author=W Kauzmann; citation_journal_title=Adv Protein Chem; citation_volume=14; citation_number=2; citation_pages=1-63; citation_date=1959; " />
      <meta name="citation_reference" content="citation_title=Interfacial free energy and the hydrophobic effect.; citation_author=C Tanford; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=76; citation_number=3; citation_pages=4175-4176; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Multiple conformational changes in enzyme catalysis.; citation_author=GG Hammes; citation_journal_title=Biochemistry; citation_volume=41; citation_number=4; citation_pages=8221-8228; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular recognition in antibody-antigen complexes.; citation_author=EJ Sundberg; citation_author=RA Mariuzza; citation_journal_title=Adv Protein Chem; citation_volume=61; citation_number=5; citation_pages=119-160; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structural evidence for induced fit as a mechanism for antibody-antigen recognition.; citation_author=JM Rini; citation_author=U Schulze-Gahmen; citation_author=IA Wilson; citation_journal_title=Science; citation_volume=255; citation_number=6; citation_pages=959-965; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=T cell receptor-ligand interactions: a conformational preequilibrium or an induced fit.; citation_author=DM Gakamsky; citation_author=IF Luescher; citation_author=I Pecht; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=7; citation_pages=9063-9066; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Flexibility and molecular recognition in the immune system.; citation_author=R Jimenez; citation_author=G Salazar; citation_author=KK Baldridge; citation_author=FE Romesberg; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=8; citation_pages=92-97; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structure and kinetics of a transient antibody binding intermediate reveal a kinetic discrimination mechanism in antigen recognition.; citation_author=LC James; citation_author=DS Tawfik; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=9; citation_pages=12730-12735; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Principles of protein-protein interactions.; citation_author=S Jones; citation_author=JM Thornton; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=10; citation_pages=13-20; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Conformational changes associated with protein-protein interactions.; citation_author=CS Goh; citation_author=D Milburn; citation_author=M Gerstein; citation_journal_title=Curr Opin Struct Biol; citation_volume=14; citation_number=11; citation_pages=104-109; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Protein-DNA recognition.; citation_author=CO Pabo; citation_author=RT Sauer; citation_journal_title=Annu Rev Biochem; citation_volume=53; citation_number=12; citation_pages=293-321; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Crystal structures of human DNA polymerase beta complexed with gapped and nicked DNA: evidence for an induced fit mechanism.; citation_author=MR Sawaya; citation_author=R Prasad; citation_author=SH Wilson; citation_author=J Kraut; citation_author=H Pelletier; citation_journal_title=Biochemistry; citation_volume=36; citation_number=13; citation_pages=11205-11215; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Conformational coupling in DNA polymerase fidelity.; citation_author=KA Johnson; citation_journal_title=Annu Rev Biochem; citation_volume=62; citation_number=14; citation_pages=685-713; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=DNA replication fidelity.; citation_author=TA Kunkel; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=15; citation_pages=16895-16898; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Protein-RNA recognition.; citation_author=DE Draper; citation_journal_title=Annu Rev Biochem; citation_volume=64; citation_number=16; citation_pages=593-620; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Induced fit in RNA-protein recognition.; citation_author=JR Williamson; citation_journal_title=Nat Struct Biol; citation_volume=7; citation_number=17; citation_pages=834-837; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Fidelity of aminoacyl-tRNA selection on the ribosome: kinetic and structural mechanisms.; citation_author=MV Rodnina; citation_author=W Wintermeyer; citation_journal_title=Annu Rev Biochem; citation_volume=70; citation_number=18; citation_pages=415-435; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Transcription factors: structural families and principles of DNA recognition.; citation_author=CO Pabo; citation_author=RT Sauer; citation_journal_title=Annu Rev Biochem; citation_volume=61; citation_number=19; citation_pages=1053-1095; citation_date=1992; " />
      <meta name="citation_reference" content="citation_author=C Kleanthous; citation_number=20; citation_date=2000; citation_publisher=Oxford Universtity Press; " />
      <meta name="citation_reference" content="citation_title=Catalysis, binding and enzyme-substrate complementarity.; citation_author=AR Fersht; citation_journal_title=Proc R Soc Lond B Biol Sci; citation_volume=187; citation_number=21; citation_pages=397-407; citation_date=1974; " />
      <meta name="citation_reference" content="citation_author=AR Fersht; citation_number=22; citation_date=1985; citation_publisher=W.H.Freeman and Co; " />
      <meta name="citation_reference" content="citation_title=The Role of Induced Fit and Conformational Changes of Enzymes in Specificity and Catalysis.; citation_author=D Herschlag; citation_journal_title=Bioorg Chem; citation_volume=16; citation_number=23; citation_pages=62-96; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Enzyme catalysis: conflicting requirements of substrate access and transition state affinity.; citation_author=R Wolfenden; citation_journal_title=Mol Cell Biochem; citation_volume=3; citation_number=24; citation_pages=207-211; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Reexamination of induced fit as a determinant of substrate specificity in enzymatic reactions.; citation_author=CB Post; citation_author=WJ Ray; citation_journal_title=Biochemistry; citation_volume=34; citation_number=25; citation_pages=15881-15885; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Thermodynamics of the interactions of lac repressor with variants of the symmetric lac operator: effects of converting a consensus site to a non-specific site.; citation_author=DE Frank; citation_author=RM Saecker; citation_author=JP Bond; citation_author=MW Capp; citation_author=OV Tsodikov; citation_journal_title=J Mol Biol; citation_volume=267; citation_number=26; citation_pages=1186-1206; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Conformational change of proteins arising from normal mode calculations.; citation_author=F Tama; citation_author=YH Sanejouand; citation_journal_title=Protein Eng; citation_volume=14; citation_number=27; citation_pages=1-6; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Large Amplitude Elastic Motions in Proteins from a Single-Parameter, Atomic Analysis.; citation_author=MM Tirion; citation_journal_title=Physical Review Letters; citation_volume=77; citation_number=28; citation_pages=1905-1908; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Direct evaluation of thermal fluctuations in proteins using a single-parameter harmonic potential.; citation_author=I Bahar; citation_author=AR Atilgan; citation_author=B Erman; citation_journal_title=Fold Des; citation_volume=2; citation_number=29; citation_pages=173-181; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Anisotropy of fluctuation dynamics of proteins with an elastic network model.; citation_author=AR Atilgan; citation_author=SR Durell; citation_author=RL Jernigan; citation_author=MC Demirel; citation_author=O Keskin; citation_journal_title=Biophys J; citation_volume=80; citation_number=30; citation_pages=505-515; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Structural changes involved in protein binding correlate with intrinsic motions of proteins in the unbound state.; citation_author=D Tobi; citation_author=I Bahar; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=31; citation_pages=18908-18913; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Kinetic proofreading: a new mechanism for reducing errors in biosynthetic processes requiring high specificity.; citation_author=JJ Hopfield; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=71; citation_number=32; citation_pages=4135-4139; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Kinetic amplification of enzyme discrimination.; citation_author=J Ninio; citation_journal_title=Biochimie; citation_volume=57; citation_number=33; citation_pages=587-595; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Protein-DNA computation by stochastic assembly cascade.; citation_author=R Bar-Ziv; citation_author=T Tlusty; citation_author=A Libchaber; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=34; citation_pages=11589-11592; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=High-fidelity DNA sensing by protein binding fluctuations.; citation_author=T Tlusty; citation_author=R Bar-Ziv; citation_author=A Libchaber; citation_journal_title=Phys Rev Lett; citation_volume=93; citation_number=35; citation_pages=258103; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=High fidelity of RecA-catalyzed recombination: a watchdog of genetic diversity.; citation_author=D Sagi; citation_author=T Tlusty; citation_author=J Stavans; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=36; citation_pages=5021-5031; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Kinetic proofreading in T-cell receptor signal transduction.; citation_author=TW McKeithan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=37; citation_pages=5042-5046; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Resolution of the Problems of Replacement Free Energy, 1/S, and Internal Consistency in Nucleation Theory by Consideration of the Length Scale for Mixing Entropy.; citation_author=H Reiss; citation_author=WK Kegel; citation_author=JL Katz; citation_journal_title=Phys Rev Lett; citation_volume=78; citation_number=38; citation_pages=4506-4509; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000468" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Conformational Proofreading: The Impact of Conformational Changes on the Specificity of Molecular Recognition"/>
    <meta name="twitter:description" content="To perform recognition, molecules must locate and specifically bind their targets within a noisy biochemical environment with many look-alikes. Molecular recognition processes, especially the induced-fit mechanism, are known to involve conformational changes. This raises a basic question: Does molecular recognition gain any advantage by such conformational changes? By introducing a simple statistical-mechanics approach, we study the effect of conformation and flexibility on the quality of recognition processes. Our model relates specificity to the conformation of the participant molecules and thus suggests a possible answer: Optimal specificity is achieved when the ligand is slightly off target; that is, a conformational mismatch between the ligand and its main target improves the selectivity of the process. This indicates that deformations upon binding serve as a conformational proofreading mechanism, which may be selected for via evolution."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000468.g008"/>

  <meta property="og:title" content="Conformational Proofreading: The Impact of Conformational Changes on the Specificity of Molecular Recognition" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000468" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7483'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6232'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4636&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000468">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000468" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000468&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Conformational Proofreading: The Impact of Conformational Changes on the Specificity of Molecular Recognition
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yonatan Savir, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tsvi Tlusty
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:tsvi.tlusty@weizmann.ac.il">tsvi.tlusty@weizmann.ac.il</a></p>

                <p>Affiliation:
                  Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 23, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000468</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000468" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000468" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000468" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000468" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000468-g001" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000468-g002" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000468-g003" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000468-g004" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000468-g005" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000468-g006" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000468-g007" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000468-g008" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000468">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000468" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>To perform recognition, molecules must locate and specifically bind their targets within a noisy biochemical environment with many look-alikes. Molecular recognition processes, especially the induced-fit mechanism, are known to involve conformational changes. This raises a basic question: Does molecular recognition gain any advantage by such conformational changes? By introducing a simple statistical-mechanics approach, we study the effect of conformation and flexibility on the quality of recognition processes. Our model relates specificity to the conformation of the participant molecules and thus suggests a possible answer: Optimal specificity is achieved when the ligand is slightly off target; that is, a conformational mismatch between the ligand and its main target improves the selectivity of the process. This indicates that deformations upon binding serve as a conformational proofreading mechanism, which may be selected for via evolution.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Savir Y, Tlusty T (2007) Conformational Proofreading: The Impact of Conformational Changes on the Specificity of Molecular Recognition. PLoS ONE 2(5):
          e468.
            doi:10.1371/journal.pone.0000468</p><p><strong>Academic Editor: </strong>Enrico Scalas, University of East Piedmont, Italy</p><p><strong>Received:</strong> April 10, 2007; <strong>Accepted:</strong> May 1, 2007; <strong>Published:</strong> May 23, 2007</p><p><strong>Copyright:</strong> © 2007 Savir, Tlusty. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Minerva Foundation and the Center for Complexity Science.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Practically all biological systems rely on the ability of bio-molecules to specifically recognize each other. Examples are antibodies targeting antigens, regulatory proteins binding DNA and enzymes catalyzing their substrates. These and other molecular recognizers must locate and preferentially interact with their specific targets among a vast variety of molecules that are often structurally similar. This task is further complicated by the inherent noise in the biochemical environment, whose magnitude is comparable with that of the non-covalent binding interactions <a href="#pone.0000468-Pauling1">[1]</a>–<a href="#pone.0000468-Tanford1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>It was realized early that recognizing molecules should be complementary in shape, akin of matching lock and key (<a href="#pone-0000468-g001">figure 1A</a>). Later, however, it was found that the native forms of many recognizers do not match exactly the shape of their targets. There is a growing body of evidence for conformational changes upon binding between the native and the bound states of many biomolecules, for example in enzyme-substrate <a href="#pone.0000468-Hammes1">[4]</a>, antibody-antigen <a href="#pone.0000468-Sundberg1">[5]</a>–<a href="#pone.0000468-James1">[9]</a> and other protein-protein complexes <a href="#pone.0000468-Jones1">[10]</a>, <a href="#pone.0000468-Goh1">[11]</a>. Binding of protein to DNA is also associated with conformational changes, which may affect the fidelity of DNA polymerase <a href="#pone.0000468-Pabo1">[12]</a>–<a href="#pone.0000468-Kunkel1">[15]</a>, and similar effects were observed in the binding of RNA by proteins <a href="#pone.0000468-Draper1">[16]</a>–<a href="#pone.0000468-Rodnina1">[18]</a>. The induced deformation typically involves displacements of binding sites in the range of tens of angstroms <a href="#pone.0000468-Sundberg1">[5]</a>, <a href="#pone.0000468-Jones1">[10]</a>, <a href="#pone.0000468-Pabo1">[12]</a>, <a href="#pone.0000468-Draper1">[16]</a>, <a href="#pone.0000468-Pabo2">[19]</a>, <a href="#pone.0000468-Kleanthous1">[20]</a>. To account for these conformational changes upon binding, the induced fit scheme was suggested. In this scheme, the participating molecules deform to fit each other before they bind into a complex (<a href="#pone-0000468-g001">figure 1B</a>). Another model, the pre-equilibrium hypothesis, assumes that the target native state interconverts within an ensemble of conformations and the ligand selectively binds to one of them (<a href="#pone-0000468-g001">figure 1C</a>).</p>
<div class="figure" id="pone-0000468-g001"><div class="img"><a name="pone-0000468-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Models of molecular recognition.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>(A) Lock and key. No conformational changes occur upon binding. The ligand (white) and the target (green) have complementary structures. (B) Induced fit. The target changes its conformation due to the interaction with the ligand. (C) Pre-existing equilibrium model. The native state is actually an ensemble of conformations, that is deformations may occur even before binding. The ligand selectively binds the matching target within this ensemble of fluctuating conformations.</p>
<span>doi:10.1371/journal.pone.0000468.g001</span></div><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The abundance of conformational changes raises the question of whether they occur due to biochemical constraints or whether they are perhaps the outcome of an evolutionary optimization of recognition processes. In the present work, we discuss the latter possibility by evaluating the effects of conformation and flexibility on recognition. To estimate the quality of recognition we use the common measure of <em>specificity</em>, that is the ability to discriminate between competing targets. Whether conformational changes and especially the induced-fit mechanism can provide or enhance specificity has been a matter of debate <a href="#pone.0000468-Johnson1">[14]</a>, <a href="#pone.0000468-Fersht1">[21]</a>–<a href="#pone.0000468-Post1">[25]</a>. Various detailed kinetic schemes have been suggested and their potential effects on specificity have been discussed – however without direct relation to concrete conformational mechanisms. Here we examine these underlying effects of flexibility and conformational changes that may govern the rate constants and thus determine specificity. Our approach tries to elucidate some of these basic effects by introducing a simple statistical-mechanics model and applying it to a generalized kinetic scheme of recognition in the presence of noise. As an outcome, the flexibility of the ligand and its relative mismatch with respect to the target which optimize specificity can be evaluated.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In the binding schemes described above (<a href="#pone-0000468-g001">figure 1</a>), the ligand is a “switch” that interconverts between a native, inactive form and an active form that fits the target. However, in a noisy biochemical environment, one may expect both the ligand and the targets to interconvert within an ensemble of many possible conformations. Such an ensemble may be the outcome, for example, of thermally induced distortions. Consider for example a scenario in which an elastic ligand is interacting with two rigid competing targets (<a href="#pone-0000468-g002">figure 2</a>). All the conformations of the ligand may interact with the targets and as a result a variety of complexes, differing by the structures of the bound ligand, is formed (<a href="#pone-0000468-g002">figure 2</a>). Among the complexes formed, some are composed of perfectly matched ligand and target. In those complexes, specific binding energy due to the alignment of binding sites is gained. However, a complex may be formed even if the ligand does not perfectly match the target, due to non-specific binding energy. For example, the <em>lac</em> repressor can bind non-specifically to DNA regardless of its sequence <a href="#pone.0000468-Frank1">[26]</a>. All the complexes, the matched and the mismatched, may retain some functionality. The efficiency of the recognition process depends on the elasticity of the ligand and on the structural mismatch between the ligand native state and the main target.</p>
<div class="figure" id="pone-0000468-g002"><div class="img"><a name="pone-0000468-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Competition between two rigid targets.</span></strong></p><a id="article1.body1.sec1.fig2.caption1.p1" name="article1.body1.sec1.fig2.caption1.p1"></a><p>The ligand (white) is interconverting within an ensemble of conformations while interacting with two rigid competitors, <em>A</em> and <em>B</em> (green and orange), characterized by different structures. Non-specific binding energy may lead to the formation of functional complexes in which the target and the ligand are not exactly matched. The unmatched complexes may also be functional but their product formation rates, ν<em><sub>um</sub></em>, may differ from these of the matched complexes, ν<em><sub>m</sub></em>. The specificity of the ligand, that is its ability to discriminate between <em>A</em> and <em>B</em>, depends on the ligand flexibility, the structural mismatch between its native state and the correct target and on the structural difference between the competing targets.</p>
<span>doi:10.1371/journal.pone.0000468.g002</span></div><a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The quality of recognition is measured by its specificity, which is defined as follows. Consider a ligand <em>a</em> interacting with a correct target <em>A</em> and an incorrect competitor <em>B</em>,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>where <em>K<sub>A</sub></em> and <em>K<sub>B</sub></em> are the dissociation constants, and ν<em><sub>A</sub></em> and ν<em><sub>B</sub></em> are the turnover numbers. Specificity is naturally defined as the ratio of the correct production rate, <em>R<sub>A</sub></em> = [<em>aA</em>]·ν<em><sub>A</sub></em>, and the incorrect production rate, <em>R<sub>B</sub></em> = [<em>aB</em>]·ν<em><sub>B</sub></em>, where [ ] denotes concentration. Typically, the chemical step is the rate-limiting one and the complex formation reaction is therefore in quasi-equilibrium, [<em>aA</em>] = [<em>a</em>][<em>A</em>]/<em>K<sub>A</sub></em>, [<em>aB</em>] = [<em>a</em>][<em>B</em>]/<em>K<sub>B</sub></em>. Thus the specificity ξ takes the form:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>If however the ligand and the targets interconvert within ensembles of many possible conformations (<a href="#pone-0000468-g002">figures 2</a>–<a href="#pone-0000468-g003">3</a>), the specificity includes potential contributions from all possible complexes,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>where <em>i</em> and <em>j</em> denote the conformations of the ligand and the target, respectively. <em>K<sub>ij</sub></em> is the dissociation constant of the complex formed from the <em>i</em>-th ligand conformation and <em>j</em>-th target conformation and <em>v<sub>ij</sub></em> is the turnover number of this complex.</p>
<div class="figure" id="pone-0000468-g003"><div class="img"><a name="pone-0000468-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>General molecular recognition scheme.</span></strong></p><a id="article1.body1.sec1.fig3.caption1.p1" name="article1.body1.sec1.fig3.caption1.p1"></a><p>Both the ligand (white) and the target (green) are interconverting between an ensemble of conformations denoted by indices, <em>a<sub>i</sub></em> and <em>A<sub>i</sub></em>, respectively. All the different conformations may interact and as a result, a variety of complexes is formed. In some of them the target and the ligand are perfectly matched, for example <em>a<sub>i</sub>A<sub>i</sub></em> and <em>a<sub>j</sub>A<sub>j</sub></em>, and in some there is only partial fit, for example <em>a<sub>i</sub>A<sub>j</sub></em> and <em>a<sub>j</sub>A<sub>i</sub></em>. The rate of product formation depends on the concentrations of the complexes, which depend on <em>K<sub>ij</sub></em>, and on the functionality of each complex, which depends on the turnover numbers, υ<em><sub>ij</sub></em>. In a similar fashion, the different ligand conformations, <em>a<sub>i</sub></em>, may interact with competing target conformations <em>B<sub>i</sub></em> and thus catalyze incorrect product.</p>
<span>doi:10.1371/journal.pone.0000468.g003</span></div><div class="figure" id="pone-0000468-g004"><div class="img"><a name="pone-0000468-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Lowest elastic mode model.</span></strong></p><a id="article1.body1.sec1.fig4.caption1.p1" name="article1.body1.sec1.fig4.caption1.p1"></a><p>Conformational changes occur upon binding of a ligand (white) and a target (green). In the native state of the ligand, the binding sites are equally spaced and positioned at <em>x<sub>i</sub></em><sup>0</sup> (<em>i</em> = 0,1,2…<em>N−1</em>) and the total length of the binding domain is <em>l<sub>0</sub></em>. The ligand is interacting with a rigid target on which <em>N</em> complementary binding sites are equally spaced and positioned at <em>y<sub>i</sub></em>  = <em>y<sub>0</sub>+i·s/</em>(<em>N−1</em>) where <em>s</em> is the length of the target binding domain. The ligand may undergo a conformational change to fit the target. Since we consider only the lowest mode motion, the ligand may only stretch or expand uniformly. Thus, its binding sites are displaced to <em>x<sub>i</sub></em> = <em>x<sub>0</sub></em>+<em>i·l</em>/(<em>N</em>−1) and the total length changes from <em>l<sub>0</sub></em> to <em>l</em>.</p>
<span>doi:10.1371/journal.pone.0000468.g004</span></div><a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Equation 3 and its generalization, equation 4, reflect the dependence of the specificity on both the concentrations of complexes, determined by the dissociation constants <em>K</em>, and on their functionality, determined by the turnover numbers <em>v</em>. These parameters, <em>K</em> and <em>v</em>, depend on the flexibility and structure of the participant molecules. Evaluating this dependence allows us to estimate the optimal flexibility and structure similarity between the ligand and the main target.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Lowest elastic mode model</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In essence, molecular recognition is governed by the interplay between the interaction energy gained from the alignment of the binding sites and the elastic energy required to deform the molecules to align. Motivated by deformation spectra measurements <a href="#pone.0000468-Tama1">[27]</a>, we treat this interplay within a simple model that takes into account only the lowest elastic mode. This is a vast simplification of the many degrees of freedom that are required to describe the details of a conformational change. However, as we suggest below, this simplified model still captures the essence of the energy tradeoff. Modeling proteins as elastic networks was previously applied to study large amplitude <a href="#pone.0000468-Tirion1">[28]</a> and thermal fluctuations <a href="#pone.0000468-Bahar1">[29]</a>, <a href="#pone.0000468-Atilgan1">[30]</a> of proteins, and to predict deformations and domain motion upon binding <a href="#pone.0000468-Tama1">[27]</a>, <a href="#pone.0000468-Tobi1">[31]</a>. These models are fitted with typical spring constants of a few <em>k<sub>B</sub>T/Å<sup>2</sup></em>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We consider first an elastic ligand interacting with a rigid target. Later, we discuss the case of deformable targets. The binding domain of the ligand is regarded as an elastic string on which <em>N</em> binding sites are equally spaced (<a href="#pone-0000468-g004">figure 4</a>). The elastic deformation energy is described by harmonic springs that connect adjacent binding sites. In the native state of the ligand, the length of the binding domain is <em>l<sub>0</sub></em>. The ligand interacts with a rigid target on which <em>N</em> complementary binding sites are equally spaced along a binding domain of length <em>s</em>. Binding is specific, that is a binding site of the ligand can gain binding energy ε only by fastening to its complementary binding site on the target. The ligand-target interactions are relatively short-range and therefore binding energy is gained only if the complementary binding sites are at the same position.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The presence of the target may induce a deformation of the ligand that, in order to gain binding energy, shifts the binding sites to new positions. However, such deformation of the ligand costs elastic energy. The conformation of the ligand is determined by <em>N</em> degrees of freedom, the <em>N</em> positions of the ligand binding sites. We assume for simplicity that all the springs that connect adjacent binding sites on the ligand have the same spring constant. We consider here only the deformation mode of lowest elastic energy, in which the binding domain of the ligand is stretched or shrunk uniformly. Thus, we reduce the number of degrees of freedom from <em>N</em> to two, the length of the deformed binding domain <em>l,</em> and the position of its edge (<a href="#pone-0000468-g004">figure 4</a>).</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>To evaluate the effect of conformational changes and flexibility on specificity one needs to estimate the concentrations and reaction constants in (4). Since all the reactions besides the product formation are assumed to be in equilibrium, we can regard each conformation of the ligand, specified by its length <em>l<sub>i</sub></em>, as a separate <em>chemical species a</em><sub>i</sub>. Thus we may apply the law of mass-action to each of the binding reactions <em>a</em><sub>i</sub>+<em>A</em>↔<em>a</em><sub>i</sub><em>A</em>, and obtain the equilibrium constant <em>K<sub>iA</sub></em> = [<em>a</em><sub>i</sub>][<em>A</em>]/[<em>a</em><sub>i</sub><em>A</em>]~<em>Z<sub>i</sub> Z<sub>A</sub></em>/<em>Z<sub>iA</sub></em>, where <em>Z<sub>i</sub></em>, <em>Z<sub>A</sub></em> and <em>Z<sub>iA</sub></em> are the single-particle partition functions of the <em>i</em>-th ligand conformation, target and complex, respectively. The equilibrium constant is (see <a href="#s3">Methods</a>):<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>where <em>f</em> is the non-specific free energy. The binding energy ε and the non-specific free energy <em>f</em> are in units of <em>k<sub>B</sub>T</em>. The concentration of free ligand of length <em>l<sub>i</sub></em> is proportional to the Boltzmann exponent of the distortion energy, [<em>a<sub>i</sub></em>]~[<em>a</em>]·exp(<em>−k/2</em>(<em>l<sub>i</sub></em>−<em>l<sub>0</sub></em>)<sup>2</sup>), where the effective spring constant <em>k</em> is in units of <em>k<sub>B</sub>T</em>/length<sup>2</sup> and [<em>a</em>] is the total concentration of the free ligand. Although some preferred conformations may be catalyzed much faster than the others, the interconversion is assumed to be fast enough to still maintain this equilibrium distribution.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>With the knowledge of how the rate constants depend on the conformation and the flexibility of the ligand, we analyze below the specificity to suggest a simple answer to the question raised above: What are the optimal geometry and flexibility that yield maximal specificity? The quality of a recognition process depends on two main properties of the participant molecules, their chemical affinity and the conformational match between them. To discuss the conformational effect, we consider a main or “correct” target <em>A</em> and an “incorrect” competitor <em>B</em> that differ in structure; their binding domains are of different lengths, <em>s<sub>A</sub></em> and <em>s<sub>B</sub></em>. Chemical affinity is taken into account by assuming that the competing target <em>B</em> has only <em>N−m</em> interacting binding sites while the main one has <em>N</em>. We test the specificity of a ligand specified by a native state length <em>l<sub>0</sub></em> and a flexibility <em>k</em>. We define the <em>mismatch d</em> as the difference between the ligand's native state length and the correct target's length, <em>d = l<sub>0</sub>−s<sub>A</sub></em>. We first examine the competition between two rigid “noiseless” targets and then discuss the noisy case. The generalization to more than two competing targets is straightforward.</p>


<h4>Recognizing noiseless targets</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Consider a ligand interconverting within an ensemble of conformations, each one with a different binding domain length <em>l<sub>i</sub></em>. This ligand interacts with two competing rigid targets that differ by their length Δ = <em>s<sub>A</sub>−s<sub>B</sub></em>. The ratio of the production rates due to unmatched and matched complexes is denoted by <em>r</em> = ν<em><sub>um</sub></em>/ν<em><sub>m</sub></em> where the production rates of the correct and incorrect products are assumed to be equal, ν<em><sub>A</sub></em> = ν<em><sub>B</sub></em>. For the sake of simplicity, we assume that the target is in excess with respect to the ligand, [<em>A</em>]~[<em>A<sub>total</sub></em>] and [<em>B</em>]~[<em>B<sub>total</sub></em>], and that the concentrations of the competing targets are equal, [<em>A</em>] = [<em>B</em>]. Substitution of the equilibrium constant (5) into (4) yields the specificity (see <a href="#s3">Methods</a>)<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e006&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>where <em>f</em> is the non-specific free energy. The dimensionless parameter α~1/(<em>k</em><sup>1/2</sup><em>g</em>) is the ratio between the typical length scales, <em>k</em><sup>−1/2</sup> of the elasticity and <em>g</em> of the binding potential. Thus, we obtained in (6) the specificity ξ as a function of the structural and energetic parameters: the difference between the target and the competitor Δ, the mismatch between the ligand and the target <em>d = l<sub>0</sub>−s<sub>A</sub></em>, the effective spring constant <em>k</em> and the specific and non-specific binding energies. Below we examine this dependence to find the optimal ligand, specified by its mismatch <em>d</em> (or its native state length <em>l<sub>0</sub></em>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The specificity (6) is simply the ratio of the formation rates of correct and incorrect products, <em>R<sub>A</sub></em> and <em>R<sub>B</sub></em>, respectively (<a href="#pone-0000468-g005">figure 5</a>). The correct production rate <em>R<sub>A</sub></em>, as a function of the mismatch <em>d</em>, is the sum of a Gaussian centered on <em>d</em> = 0, which accounts for the specific binding, and a uniform non-specific contribution. <em>R<sub>A</sub></em> is therefore maximal at a zero mismatch. The incorrect production rate <em>R<sub>B</sub></em> has the same uniform non-specific contribution and its specific contribution is now a Gaussian centered around <em>d</em> = Δ, where it exhibits its maximum. The crossover where the specific and non-specific contributions become comparable defines a <em>“window of recognition”</em>. When the windows of recognition of the correct and incorrect targets overlap, the resulting specificity exhibits a maximum at a <em>finite nonzero mismatch</em> (<a href="#pone-0000468-g005">figure 5A</a>). This optimal mismatch <em>d</em><sub>0</sub> is approximately<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e007&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br></p>
<div class="figure" id="pone-0000468-g005"><div class="img"><a name="pone-0000468-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The dependence of specificity on mismatch.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Each column is for a specific difference between the competing targets, Δ = <em>s<sub>A</sub></em>−<em>s<sub>B</sub></em>, given in Å. The rate production of the correct product <em>R<sub>A</sub></em> is a sum of a Gaussian centered at <em>d</em> = 0, which arises from the specific binding, and a uniform contribution due to non-specific binding (top row, blue). Similarly, the incorrect rate production, <em>R<sub>B</sub></em>, is composed of a Gaussian centered at <em>d</em> = Δ and a uniform non-specific contribution (top row, red). The specificity, ξ, is the ratio between the correct and incorrect production rates and therefore depends on the location and width of the recognition windows (bottom row). (A) If the competing targets differ in structure, Δ = 3 Å, the windows of recognition partly overlap and the resulting specificity is optimal at a <em>nonzero mismatch</em>. (B) For Δ = 0, both <em>R<sub>A</sub></em> and <em>R<sub>B</sub></em> are centered around zero mismatch and the resulting specificity is approximately a rectangular window of width, <em>d<sub>1/2</sub></em>≈<em>k<sup>−1/2</sup></em>((<em>N−m</em>)ε−<em>f</em>−log(αr)). (C) If the competing targets differ much, Δ≫<em>d<sub>1/2</sub></em>, the recognition windows do not overlap and the specificity is again optimal for zero mismatch. The parameters of the plot are <em>N</em> = 15, <em>m</em> = 2, ε = 2 <em>k</em><sub>B</sub><em>T</em>, <em>r</em> = 0.1, <em>g</em> = 1 Å, <em>k</em> = 1 <em>k</em><sub>B</sub><em>T</em>/Å<em><sup>2</sup></em>, <em>f</em> = 15 <em>k</em><sub>B</sub><em>T</em>.</p>
<span>doi:10.1371/journal.pone.0000468.g005</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>As the ligand becomes more rigid, the specificity increases while the optimal mismatch <em>d<sub>0</sub></em> tends to zero (<a href="#pone-0000468-g006">figure 6A</a>). The optimal mismatch is bounded by <em>d<sub>max</sub></em> = (<em>N</em>ε/<em>k</em>)<sup>1/2</sup>, the length-scale that reflects the interplay between the elastic and specific binding energies.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Competing targets of similar structure Δ≈0, have both correct and incorrect recognition windows centered on zero mismatch and the resulting specificity is akin to a rectangular window (<a href="#pone-0000468-g005">figure 5B</a>). The width of this window is the mismatch where specificity is half of its maximum, <em>d<sub>1/2</sub></em>≈<em>k<sup>−1/2</sup></em>((<em>N</em>−<em>m</em>)ε−<em>f</em>−log(α<em>r</em>)). As the ligand becomes more flexible the width of this rectangular window increases (<a href="#pone-0000468-g006">figure 6B</a>). Targets that differ much are evidently not competitors. Indeed, if the difference Δ is much larger than the window of recognition, the optimal mismatch vanishes (<a href="#pone-0000468-g005">figure 5C</a>). Thus, (7) provides a criterion for relevant competitors: these must lie within the window of recognition of the correct target.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>An interesting special case is when only the perfectly matched complexes are functional. This situation may occur if the non-specific binding energy is small and only matched complexes are formed, or if mismatched complexes are not functional, <em>r</em> = 0. The specificity in this case increases exponentially with the mismatch, ξ~exp(<em>k</em>·Δ·<em>d</em>) (<a href="#pone-0000468-g006">figure 6C</a>). Generalization of these results to more then 1D and for multiple competitors is straight forward. In <a href="#pone-0000468-g007">figure 7</a>, the specificity of a ligand as a function of mismatch in the presence of a few competitors is shown. The optimal mismatch depends on the structure of the various competitors. When the competitors have a structure similar to that of the main target, the mismatch is non-zero.</p>
<div class="figure" id="pone-0000468-g006"><div class="img"><a name="pone-0000468-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Specificity ξ as a function of mismatch <em>d</em> and flexibility, <em>k.</em></span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Colors denote various values of rigidity <em>k</em> (in units of <em>k</em><sub>B</sub><em>T</em>/Å<sup>2</sup>, legend). (A) For targets that differ by Δ = 3 Å the specificity is optimal at a nonzero mismatch. As the ligand becomes more rigid the optimal mismatch tends to zero as <em>d<sub>0</sub></em>~<em>k<sup>−1/2</sup></em>. (B) For competing targets with similar structure, Δ = 0, the specificity resembles a rectangular window centered on zero mismatch. The width of this window also decreases as <em>k<sup>−1/2</sup></em>. (C) The specificity when only matched complexes are functional, <em>r = 0</em>, increases exponentially with the mismatch as ξ~exp(<em>k</em>·Δ·<em>d</em>). The parameters of the plot are <em>N</em> = 15, <em>m</em> = 2, ε = 2 <em>k</em><sub>B</sub><em>T</em>, <em>r</em> = 0.1, <em>g</em> = 1 Å, <em>f</em> = 15 <em>k</em><sub>B</sub><em>T</em>.</p>
<span>doi:10.1371/journal.pone.0000468.g006</span></div><div class="figure" id="pone-0000468-g007"><div class="img"><a name="pone-0000468-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Specificity as a function of a 2D mismatch in the presence of multiple competitors.</span></strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>Color bar shows log of specificity. In two dimensions, the ligand structure may stretch or shrink along <em>x</em> and <em>y</em>. The mismatches along these axes are <em>d<sub>x</sub></em> and <em>d<sub>y</sub></em>. The gray circle denotes zero mismatch. In the presence of multiple competitors (green crosses), the optimal mismatch (black X) is nonzero and depends on the structure of the various competitors. Competitors that slightly differ from the correct target have a “window of recognition” which overlaps with correct target recognition window. As a result, the specificity is maximal for a non-zero mismatch. The parameters of the plot are <em>N</em> = 15, <em>m</em> = 2, ε = 2 <em>k</em><sub>B</sub><em>T</em>, <em>r</em> = 0.1, <em>g</em> = 1 Å, k = 1 <em>k</em><sub>B</sub><em>T</em>/<em>Å<sup>2</sup></em>, <em>f</em> = 15 <em>k</em><sub>B</sub><em>T</em>.</p>
<span>doi:10.1371/journal.pone.0000468.g007</span></div><a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>These results are reminiscent of kinetic proofreading <a href="#pone.0000468-Hopfield1">[32]</a>, <a href="#pone.0000468-Ninio1">[33]</a> in which the specificity of a biochemical reaction increases exponentially with the <em>temporal delay</em> or the number of additional intermediate states <a href="#pone.0000468-BarZiv1">[34]</a>–<a href="#pone.0000468-McKeithan1">[37]</a>. In kinetic proofreading, the delay reduces the production rates of both correct and incorrect products, but the reduction of the incorrect product is larger and thus specificity improves. In the present case, the equivalent of the temporal delay is the <em>spatial mismatch</em>. It is evident from equation (6) that mismatch reduces both the correct and incorrect rates, but as the effect on the incorrect rate is more significant the overall specificity increases. Of course, a major difference is that kinetic proofreading is an energy-consuming non-equilibrium scheme whereas the <em>conformational proofreading</em> suggested here is at quasi-equilibrium.</p>


<h4>Recognizing noisy targets</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In this case, the ligand still interconverts between an ensemble of conformations, but now the target is prone to error. We describe this noise as Gaussian fluctuations of the target's length <em>s</em> with a variance σ. These fluctuations may originate from various sources such as thermal noise, where the variance σ is related to the target's flexibility as σ<em><sub>A,B</sub></em>~<em>k<sub>A,B</sub><sup>−1/2</sup></em>. The noise introduces additional matched complexes and thus widens the windows of recognition of both the correct and incorrect targets. Similar to (6) (see <a href="#s3">Methods</a>), the resulting specificity is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e008&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br>If σ<em><sub>A</sub></em> = σ<em><sub>B</sub></em> = σ, the results are the same as for two rigid targets competing for a ligand with an effective spring constant <em>k′ = k/</em>(1+kσ<sup>2</sup>). For any values of σ<em><sub>A</sub></em> and σ<em><sub>B</sub></em>, when the targets differ in structure Δ≠0, the specificity is optimal at a nonzero mismatch as in the noiseless case (<a href="#pone-0000468-g008">figure 8A–B</a>). But unlike the noiseless scenario, even the specificity of an infinitely rigid ligand may be optimal at a nonzero mismatch. For Δ = 0 the specificity has an extremum at a zero mismatch. If the incorrect target is noisier, σ<em><sub>A</sub></em>&lt;σ<em><sub>B</sub></em>, identical ligand and target achieve maximal specificity (<a href="#pone-0000468-g008">figure 8C–D</a>). However, if the correct target is noisier σ<em><sub>A</sub></em>&gt;σ<em><sub>B</sub>,</em> a mismatched ligand is optimal <em>even for structurally similar targets</em>.</p>
<div class="figure" id="pone-0000468-g008"><div class="img"><a name="pone-0000468-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000468" data-uri="info:doi/10.1371/journal.pone.0000468.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000468.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000468.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000468.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Specificity ξ in the presence of two noisy targets as a function of mismatch <em>d.</em></span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Colors denote various values of rigidity <em>k</em> (in units of <em>k</em><sub>B</sub><em>T</em>/Å<sup>2</sup>, legend). The lengths of the target binding domain, <em>s<sub>A</sub></em> and <em>s<sub>B</sub></em> are fluctuating according to a Gaussian noise with variances σ<em><sub>A</sub></em> and σ<em><sub>B</sub></em>. (A, B) For competing targets of different structure, Δ = 3 Å, similarly to noiseless targets, the specificity is optimal at a nonzero mismatch. (C, D) Competing targets of similar structure, Δ = 0. The specificity has an extremum at <em>d</em> = 0, but whether this point is maximum or minimum depends on the noise. For a noisier correct target, σ<em><sub>A</sub></em>&gt;σ<em><sub>B</sub></em>, an optimal ligand has a nonzero mismatch. The parameters of the plot are: <em>N</em> = 15, <em>m</em> = 2, ε = 2 <em>k</em><sub>B</sub><em>T</em>, <em>r</em> = 0.1, <em>g</em> = 1 Å, σ<em><sub>A,B</sub></em> = 0.5, 0.6 Å, <em>f</em> = 15 <em>k</em><sub>B</sub><em>T</em>.</p>
<span>doi:10.1371/journal.pone.0000468.g008</span></div>

<h4>Possible experimental tests</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The conformational proofreading model makes several predictions that may be put to an experimental test. To begin with, the structure of the target, the ligand and the competing molecule should fulfill a number of relations. First, we expect a mismatch to occur only if a competitor is within the ligand's window of recognition, since this is the situation where competition may threaten the quality of recognition. This can be verified by comparing the structure of the native ligand, its main target and the competitor. Second, we predict that a compromise must be struck between the need for the native ligand to be as far as possible from the competitor and as close as possible to the target, so that the mismatch will place ligand and competitor at opposite sides of some structural axis. For example, in <a href="#pone-0000468-g007">figure 7</a> the mismatch which maximizes specificity is determined by the location of the competitors recognition windows. Experimentally, we expect that there will be a need for resolved 3D structures of ligands both in their native state and bound to their target, as well as these of the competitors. The rapidly increasing structural information that is available from studies of molecular recognition systems suggests that data that can validate or falsify our conformational proofreading hypothesis may already be available, or readily obtained.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Besides observing competition and specificity in known biological system, an experiment that in principle allows control over the nature of competition and the functional results of this competition may be carried out. A particularly appealing system that can be experimentally accessed and manipulated is that of transcription factors. While a transcription factor has one or several specific binding sites, there may be many competing sites on the DNA that would bind it. One can therefore experimentally alter the specific binding site or its competing sites, as well as the transcription factor, by point mutations and then observe the effect on specificity, e.g. by measuring the expression of upstream genes. The next step in this direction would be, instead of artificially manipulating the structures, tracing the coevolution of the transcription factor and all of the binding sites, looking at the <em>in-vivo</em> evolutionary optimization of recognition.</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The ability to perform efficient information processing in the presence of noise is crucial for almost any biological system. Enhancing the specificity of recognition, in the sense of discrimination between competing targets, is therefore expected to increase the fitness. By introducing a model that captures the essence of the tradeoff between the specific binding energy and the structural deformation energy, it appears possible to estimate the optimal flexibility and geometry of the fittest molecules. Our model suggests that to optimally discriminate between competing targets of different structures, the ligand should have a <em>finite mismatch</em> relative to the main target. This spatial mismatch is similar to the temporal delay that underlies kinetic proofreading. Our analysis suggests that conformational changes upon binding may arise as the outcome of an evolutionary selection for enhancing recognition specificity in a noisy environment. This may also suggest that the structure and flexibility of binding molecules are governed by evolutionary pressure to optimize not only specificity but other cost functions such as robustness to noise.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Methods"></a><h3>Methods</h3>
<h4>Dissociation constant calculation</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Within the lowest mode model assumptions, when the ligand and the target are perfectly aligned, <em>x<sub>0</sub> = y<sub>0</sub></em> and <em>l = s</em>, all the binding sites interact and contribute a total binding energy <em>N</em>ε. Otherwise, the binding energy is only due to a single interacting site. If there are many binding sites, we can neglect the single site contribution and approximate the interaction energy by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e009&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br>where <em>k</em> is the effective spring constant of the ligand binding domain. The interaction energy (9) describes an idealized scenario in which only perfectly aligned ligand and target gain specific binding energy. Of course, in reality there could be other conformations with partial alignment, but they would require the excitation of higher elastic modes.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>In order to calculate the partition function of the complex <em>Z<sub>iA</sub></em> = Tr(exp[−<em>H</em>(<em>l<sub>i</sub></em>)]) all the possible binding configurations of this complex should be specified. As mentioned above, only in the perfectly aligned configuration the specific binding energy <em>N</em>ε is gained. However, there may be other configurations in which the ligand and the target are bound non-specifically. We roughly estimate the non-specific contribution to the partition function as the product of the volume in which the non-specific binding occurs and the exponent of the non-specific binding energy. This non-specific contribution is exp(<em>f</em>), where <em>f</em> is defined to be the non-specific free energy. The total complex partition function is the product of the elastic contribution and the contribution due to binding, specific and non-specific, <em>Z<sub>iA</sub></em> = exp(<em>−k/2</em>(<em>l<sub>i</sub></em>−<em>l<sub>0</sub></em>)<sup>2</sup>)·(δ(<em>l<sub>i</sub></em>−<em>s</em>)·exp(<em>N</em>ε)+exp <em>f</em>). The elastic contributions to (5) cancel out since they are equal for both the ligand and ligand-target partition functions. The irrelevant kinetic contributions were also omitted.</p>


<h4>Specificity of a ligand with continuous ensemble of conformations</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The ligand may interconvert within a continuous ensemble of conformations specified by their binding site length <em>l</em>. Since the complex formation reaction is in quasi-equilibrium, the concentration of free ligand of length <em>l</em> is proportional to the Boltzmann exponent of the distortion energy, [<em>a</em>(<em>l</em>)]~[<em>a</em>]·exp(<em>−k/2</em>(<em>l</em>−<em>l<sub>0</sub></em>)<sup>2</sup>), where <em>l<sub>0</sub></em> is the native state length. The effective spring constant <em>k</em> is in units of <em>k<sub>B</sub>T</em>/length<sup>2</sup> and [<em>a</em>] is the total concentration of the free ligand. The dissociation constant in its continuous form is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e010&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br>Only matched complexes in which <em>l</em> = <em>s<sub>A,B</sub></em> gain specific binding energy and the turnover of these complexes, ν<em><sub>m</sub></em>, may be different from the turnover number of the unmatched complexes, ν<em><sub>um</sub></em>. Therefore, the continuous form of the turnover number is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(11)</span></span><br>If the competing targets are rigid, the contribution to specificity from all possible complexes (4) becomes an integral over all ligand conformations <em>l</em>,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(12)</span></span><br>For the sake of simplicity we assume that (i) ν<sub>A,m</sub> = ν<em><sub>B,m</sub></em> and ν<em><sub>A,um</sub></em> = ν<em><sub>B,um</sub></em>, (ii) the target is in excess with respect to the ligand, [<em>A</em>]~[<em>A<sub>total</sub></em>] and [<em>B</em>]~[<em>B<sub>total</sub></em>], and (iii) the concentrations of the competing targets are equal, [<em>A</em>] = [<em>B</em>]. Performing the integration yields<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(13)</span></span><br>where <em>d = l<sub>0</sub></em>−<em>s<sub>A</sub></em> and Δ = <em>s<sub>A</sub></em>−<em>s<sub>B</sub></em>. The normalization factor <em>g</em> reflects the assumption of a continuous ensemble of ligand conformations <em>l</em>. <em>g</em> is the phase space cell volume (actually, the translational factor of this cell volume). This cell volume appears as a proportionality constant of the partition functions. <em>g</em> is proportional to the typical length scale of thermal fluctuations in the system <a href="#pone.0000468-Reiss1">[38]</a> which is affected by the elastic and binding forces. The <em>k</em>-dependence of α = (<em>kg<sup>2</sup></em>/<em>2</em>π)<em><sup>−</sup></em><sup>1/2</sup> is at most α~<em>k<sup>−</sup></em><sup>1/2</sup> and therefore contributes only logarithmic correction in equations (6–8). Under the reasonable assumptions that ν<em><sub>m</sub></em>≫ν<em><sub>um</sub></em> and <em>k<sup>1/2</sup>l<sub>0</sub></em>≫1 the specificity becomes (6). The above assumptions are made for simplicity and clarity, they do not change the qualitative nature of the results.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>If the targets are subject to noise in their structure, (12) should also be integrated over all possible target conformations. If the fluctuations of the target binding site are around native state lengths <em>s<sub>A</sub></em> and <em>s<sub>B</sub></em> with variances σ<em><sub>A</sub></em> and σ<em><sub>B</sub></em>, the specificity is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000468.e014&amp;representation=PNG" class="inline-graphic"><span class="note">(14)</span></span><br>If again, we assume that ν<em><sub>m</sub></em>≫ν<em><sub>um</sub></em>, <em>k<sup>1/2</sup>l<sub>0</sub></em>≫1 and <em>s<sub>A,B</sub>/</em>σ<em><sub>A,B</sub></em>≫1, performing the integral (14) yields (8).</p>

</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: TT YS. Analyzed the data: TT YS. Wrote the paper: TT YS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000468-Pauling1" id="pone.0000468-Pauling1"></a>Pauling L (1940)  Science  92: 77–79.  <ul class="find" data-citedArticleID="953700" data-doi="10.1126/science.92.2378.77"><li><a href="http://dx.doi.org/10.1126/science.92.2378.77" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Science" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Science%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000468-Kauzmann1" id="pone.0000468-Kauzmann1"></a>Kauzmann W (1959) Some factors in the interpretation of protein denaturation. Adv Protein Chem  14: 1–63.  <ul class="find" data-citedArticleID="953686" data-doi="10.1016/s0065-3233(08)60608-7"><li><a href="http://dx.doi.org/10.1016/s0065-3233(08)60608-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Some+factors+in+the+interpretation+of+protein+denaturation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Some+factors+in+the+interpretation+of+protein+denaturation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000468-Tanford1" id="pone.0000468-Tanford1"></a>Tanford C (1979) Interfacial free energy and the hydrophobic effect. Proc Natl Acad Sci U S A  76: 4175–4176.  <ul class="find" data-citedArticleID="953718" data-doi="10.1073/pnas.76.9.4175"><li><a href="http://dx.doi.org/10.1073/pnas.76.9.4175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interfacial+free+energy+and+the+hydrophobic+effect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interfacial+free+energy+and+the+hydrophobic+effect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000468-Hammes1" id="pone.0000468-Hammes1"></a>Hammes GG (2002) Multiple conformational changes in enzyme catalysis. Biochemistry  41: 8221–8228.  <ul class="find" data-citedArticleID="953672" data-doi="10.1021/bi0260839"><li><a href="http://dx.doi.org/10.1021/bi0260839" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+conformational+changes+in+enzyme+catalysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+conformational+changes+in+enzyme+catalysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000468-Sundberg1" id="pone.0000468-Sundberg1"></a>Sundberg EJ, Mariuzza RA (2002) Molecular recognition in antibody-antigen complexes. Adv Protein Chem  61: 119–160.  <ul class="find" data-citedArticleID="953714" data-doi="10.1016/s0065-3233(02)61004-6"><li><a href="http://dx.doi.org/10.1016/s0065-3233(02)61004-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+recognition+in+antibody-antigen+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+recognition+in+antibody-antigen+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000468-Rini1" id="pone.0000468-Rini1"></a>Rini JM, Schulze-Gahmen U, Wilson IA (1992) Structural evidence for induced fit as a mechanism for antibody-antigen recognition. Science  255: 959–965.  <ul class="find" data-citedArticleID="953706" data-doi="10.1126/science.1546293"><li><a href="http://dx.doi.org/10.1126/science.1546293" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+evidence+for+induced+fit+as+a+mechanism+for+antibody-antigen+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+evidence+for+induced+fit+as+a+mechanism+for+antibody-antigen+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000468-Gakamsky1" id="pone.0000468-Gakamsky1"></a>Gakamsky DM, Luescher IF, Pecht I (2004) T cell receptor-ligand interactions: a conformational preequilibrium or an induced fit. Proc Natl Acad Sci U S A  101: 9063–9066.  <ul class="find" data-citedArticleID="953668" data-doi="10.1073/pnas.0402840101"><li><a href="http://dx.doi.org/10.1073/pnas.0402840101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T+cell+receptor-ligand+interactions%3A+a+conformational+preequilibrium+or+an+induced+fit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T+cell+receptor-ligand+interactions%3A+a+conformational+preequilibrium+or+an+induced+fit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000468-Jimenez1" id="pone.0000468-Jimenez1"></a>Jimenez R, Salazar G, Baldridge KK, Romesberg FE (2003) Flexibility and molecular recognition in the immune system. Proc Natl Acad Sci U S A  100: 92–97.  <ul class="find" data-citedArticleID="953680" data-doi="10.1073/pnas.262411399"><li><a href="http://dx.doi.org/10.1073/pnas.262411399" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Flexibility+and+molecular+recognition+in+the+immune+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Flexibility+and+molecular+recognition+in+the+immune+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000468-James1" id="pone.0000468-James1"></a>James LC, Tawfik DS (2005) Structure and kinetics of a transient antibody binding intermediate reveal a kinetic discrimination mechanism in antigen recognition. Proc Natl Acad Sci U S A  102: 12730–12735.  <ul class="find" data-citedArticleID="953678" data-doi="10.1073/pnas.0500909102"><li><a href="http://dx.doi.org/10.1073/pnas.0500909102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+kinetics+of+a+transient+antibody+binding+intermediate+reveal+a+kinetic+discrimination+mechanism+in+antigen+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+kinetics+of+a+transient+antibody+binding+intermediate+reveal+a+kinetic+discrimination+mechanism+in+antigen+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000468-Jones1" id="pone.0000468-Jones1"></a>Jones S, Thornton JM (1996) Principles of protein-protein interactions. Proc Natl Acad Sci U S A  93: 13–20.  <ul class="find" data-citedArticleID="953684"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Principles+of+protein-protein+interactions.&amp;auth=&amp;atitle=Principles+of+protein-protein+interactions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Principles+of+protein-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Principles+of+protein-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000468-Goh1" id="pone.0000468-Goh1"></a>Goh CS, Milburn D, Gerstein M (2004) Conformational changes associated with protein-protein interactions. Curr Opin Struct Biol  14: 104–109.  <ul class="find" data-citedArticleID="953670" data-doi="10.1016/j.sbi.2004.01.005"><li><a href="http://dx.doi.org/10.1016/j.sbi.2004.01.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+changes+associated+with+protein-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+changes+associated+with+protein-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000468-Pabo1" id="pone.0000468-Pabo1"></a>Pabo CO, Sauer RT (1984) Protein-DNA recognition. Annu Rev Biochem  53: 293–321.  <ul class="find" data-citedArticleID="953696" data-doi="10.1146/annurev.bi.53.070184.001453"><li><a href="http://dx.doi.org/10.1146/annurev.bi.53.070184.001453" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein-DNA+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein-DNA+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000468-Sawaya1" id="pone.0000468-Sawaya1"></a>Sawaya MR, Prasad R, Wilson SH, Kraut J, Pelletier H (1997) Crystal structures of human DNA polymerase beta complexed with gapped and nicked DNA: evidence for an induced fit mechanism. Biochemistry  36: 11205–11215.  <ul class="find" data-citedArticleID="953712" data-doi="10.1021/bi9703812"><li><a href="http://dx.doi.org/10.1021/bi9703812" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structures+of+human+DNA+polymerase+beta+complexed+with+gapped+and+nicked+DNA%3A+evidence+for+an+induced+fit+mechanism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structures+of+human+DNA+polymerase+beta+complexed+with+gapped+and+nicked+DNA%3A+evidence+for+an+induced+fit+mechanism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000468-Johnson1" id="pone.0000468-Johnson1"></a>Johnson KA (1993) Conformational coupling in DNA polymerase fidelity. Annu Rev Biochem  62: 685–713.  <ul class="find" data-citedArticleID="953682" data-doi="10.1146/annurev.bi.62.070193.003345"><li><a href="http://dx.doi.org/10.1146/annurev.bi.62.070193.003345" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+coupling+in+DNA+polymerase+fidelity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+coupling+in+DNA+polymerase+fidelity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000468-Kunkel1" id="pone.0000468-Kunkel1"></a>Kunkel TA (2004) DNA replication fidelity. J Biol Chem  279: 16895–16898.  <ul class="find" data-citedArticleID="953690" data-doi="10.1074/jbc.r400006200"><li><a href="http://dx.doi.org/10.1074/jbc.r400006200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+replication+fidelity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+replication+fidelity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000468-Draper1" id="pone.0000468-Draper1"></a>Draper DE (1995) Protein-RNA recognition. Annu Rev Biochem  64: 593–620.  <ul class="find" data-citedArticleID="953660" data-doi="10.1146/annurev.bi.64.070195.003113"><li><a href="http://dx.doi.org/10.1146/annurev.bi.64.070195.003113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein-RNA+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein-RNA+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000468-Williamson1" id="pone.0000468-Williamson1"></a>Williamson JR (2000) Induced fit in RNA-protein recognition. Nat Struct Biol  7: 834–837.  <ul class="find" data-citedArticleID="953726" data-doi="10.1038/79575"><li><a href="http://dx.doi.org/10.1038/79575" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induced+fit+in+RNA-protein+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induced+fit+in+RNA-protein+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000468-Rodnina1" id="pone.0000468-Rodnina1"></a>Rodnina MV, Wintermeyer W (2001) Fidelity of aminoacyl-tRNA selection on the ribosome: kinetic and structural mechanisms. Annu Rev Biochem  70: 415–435.  <ul class="find" data-citedArticleID="953708" data-doi="10.1146/annurev.biochem.70.1.415"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.70.1.415" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fidelity+of+aminoacyl-tRNA+selection+on+the+ribosome%3A+kinetic+and+structural+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fidelity+of+aminoacyl-tRNA+selection+on+the+ribosome%3A+kinetic+and+structural+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000468-Pabo2" id="pone.0000468-Pabo2"></a>Pabo CO, Sauer RT (1992) Transcription factors: structural families and principles of DNA recognition. Annu Rev Biochem  61: 1053–1095.  <ul class="find" data-citedArticleID="953698" data-doi="10.1146/annurev.bi.61.070192.005201"><li><a href="http://dx.doi.org/10.1146/annurev.bi.61.070192.005201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+factors%3A+structural+families+and+principles+of+DNA+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+factors%3A+structural+families+and+principles+of+DNA+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000468-Kleanthous1" id="pone.0000468-Kleanthous1"></a>Kleanthous C (2000) Protein-Prtoein Recognition.  D HB , M GD , editors. Oxford: Oxford Universtity Press.   <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0000468-Fersht1" id="pone.0000468-Fersht1"></a>Fersht AR (1974) Catalysis, binding and enzyme-substrate complementarity. Proc R Soc Lond B Biol Sci  187: 397–407.  <ul class="find" data-citedArticleID="953662" data-doi="10.1098/rspb.1974.0084"><li><a href="http://dx.doi.org/10.1098/rspb.1974.0084" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Catalysis%2C+binding+and+enzyme-substrate+complementarity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Catalysis%2C+binding+and+enzyme-substrate+complementarity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000468-Fersht2" id="pone.0000468-Fersht2"></a>Fersht AR (1985) Enzyme structure and mechanism 2nd ed. New York: W.H.Freeman and Co.   <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0000468-Herschlag1" id="pone.0000468-Herschlag1"></a>Herschlag D (1988) The Role of Induced Fit and Conformational Changes of Enzymes in Specificity and Catalysis. Bioorg Chem  16: 62–96.  <ul class="find" data-citedArticleID="953674" data-doi="10.1016/0045-2068(88)90038-7"><li><a href="http://dx.doi.org/10.1016/0045-2068(88)90038-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Role+of+Induced+Fit+and+Conformational+Changes+of+Enzymes+in+Specificity+and+Catalysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Role+of+Induced+Fit+and+Conformational+Changes+of+Enzymes+in+Specificity+and+Catalysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000468-Wolfenden1" id="pone.0000468-Wolfenden1"></a>Wolfenden R (1974) Enzyme catalysis: conflicting requirements of substrate access and transition state affinity. Mol Cell Biochem  3: 207–211.  <ul class="find" data-citedArticleID="953728" data-doi="10.1007/bf01686645"><li><a href="http://dx.doi.org/10.1007/bf01686645" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzyme+catalysis%3A+conflicting+requirements+of+substrate+access+and+transition+state+affinity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzyme+catalysis%3A+conflicting+requirements+of+substrate+access+and+transition+state+affinity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000468-Post1" id="pone.0000468-Post1"></a>Post CB, Ray WJ Jr (1995) Reexamination of induced fit as a determinant of substrate specificity in enzymatic reactions. Biochemistry  34: 15881–15885.  <ul class="find" data-citedArticleID="953702" data-doi="10.1021/bi00049a001"><li><a href="http://dx.doi.org/10.1021/bi00049a001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reexamination+of+induced+fit+as+a+determinant+of+substrate+specificity+in+enzymatic+reactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reexamination+of+induced+fit+as+a+determinant+of+substrate+specificity+in+enzymatic+reactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000468-Frank1" id="pone.0000468-Frank1"></a>Frank DE, Saecker RM, Bond JP, Capp MW, Tsodikov OV, et al.  (1997) Thermodynamics of the interactions of lac repressor with variants of the symmetric lac operator: effects of converting a consensus site to a non-specific site. J Mol Biol  267: 1186–1206.  <ul class="find" data-citedArticleID="953666" data-doi="10.1006/jmbi.1997.0920"><li><a href="http://dx.doi.org/10.1006/jmbi.1997.0920" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thermodynamics+of+the+interactions+of+lac+repressor+with+variants+of+the+symmetric+lac+operator%3A+effects+of+converting+a+consensus+site+to+a+non-specific+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thermodynamics+of+the+interactions+of+lac+repressor+with+variants+of+the+symmetric+lac+operator%3A+effects+of+converting+a+consensus+site+to+a+non-specific+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000468-Tama1" id="pone.0000468-Tama1"></a>Tama F, Sanejouand YH (2001) Conformational change of proteins arising from normal mode calculations. Protein Eng  14: 1–6.  <ul class="find" data-citedArticleID="953716" data-doi="10.1093/protein/14.1.1"><li><a href="http://dx.doi.org/10.1093/protein/14.1.1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+change+of+proteins+arising+from+normal+mode+calculations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+change+of+proteins+arising+from+normal+mode+calculations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000468-Tirion1" id="pone.0000468-Tirion1"></a>Tirion MM (1996) Large Amplitude Elastic Motions in Proteins from a Single-Parameter, Atomic Analysis. Physical Review Letters  77: 1905–1908.  <ul class="find" data-citedArticleID="953720" data-doi="10.1103/physrevlett.77.1905"><li><a href="http://dx.doi.org/10.1103/physrevlett.77.1905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large+Amplitude+Elastic+Motions+in+Proteins+from+a+Single-Parameter%2C+Atomic+Analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large+Amplitude+Elastic+Motions+in+Proteins+from+a+Single-Parameter%2C+Atomic+Analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000468-Bahar1" id="pone.0000468-Bahar1"></a>Bahar I, Atilgan AR, Erman B (1997) Direct evaluation of thermal fluctuations in proteins using a single-parameter harmonic potential. Fold Des  2: 173–181.  <ul class="find" data-citedArticleID="953656" data-doi="10.1016/s1359-0278(97)00024-2"><li><a href="http://dx.doi.org/10.1016/s1359-0278(97)00024-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Direct+evaluation+of+thermal+fluctuations+in+proteins+using+a+single-parameter+harmonic+potential." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Direct+evaluation+of+thermal+fluctuations+in+proteins+using+a+single-parameter+harmonic+potential.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000468-Atilgan1" id="pone.0000468-Atilgan1"></a>Atilgan AR, Durell SR, Jernigan RL, Demirel MC, Keskin O, et al.  (2001) Anisotropy of fluctuation dynamics of proteins with an elastic network model. Biophys J  80: 505–515.  <ul class="find" data-citedArticleID="953654" data-doi="10.1016/s0006-3495(01)76033-x"><li><a href="http://dx.doi.org/10.1016/s0006-3495(01)76033-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anisotropy+of+fluctuation+dynamics+of+proteins+with+an+elastic+network+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anisotropy+of+fluctuation+dynamics+of+proteins+with+an+elastic+network+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000468-Tobi1" id="pone.0000468-Tobi1"></a>Tobi D, Bahar I (2005) Structural changes involved in protein binding correlate with intrinsic motions of proteins in the unbound state. Proc Natl Acad Sci U S A  102: 18908–18913.  <ul class="find" data-citedArticleID="953724" data-doi="10.1073/pnas.0507603102"><li><a href="http://dx.doi.org/10.1073/pnas.0507603102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+changes+involved+in+protein+binding+correlate+with+intrinsic+motions+of+proteins+in+the+unbound+state." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+changes+involved+in+protein+binding+correlate+with+intrinsic+motions+of+proteins+in+the+unbound+state.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000468-Hopfield1" id="pone.0000468-Hopfield1"></a>Hopfield JJ (1974) Kinetic proofreading: a new mechanism for reducing errors in biosynthetic processes requiring high specificity. Proc Natl Acad Sci U S A  71: 4135–4139.  <ul class="find" data-citedArticleID="953676" data-doi="10.1073/pnas.71.10.4135"><li><a href="http://dx.doi.org/10.1073/pnas.71.10.4135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinetic+proofreading%3A+a+new+mechanism+for+reducing+errors+in+biosynthetic+processes+requiring+high+specificity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinetic+proofreading%3A+a+new+mechanism+for+reducing+errors+in+biosynthetic+processes+requiring+high+specificity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000468-Ninio1" id="pone.0000468-Ninio1"></a>Ninio J (1975) Kinetic amplification of enzyme discrimination. Biochimie  57: 587–595.  <ul class="find" data-citedArticleID="953694" data-doi="10.1016/s0300-9084(75)80139-8"><li><a href="http://dx.doi.org/10.1016/s0300-9084(75)80139-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinetic+amplification+of+enzyme+discrimination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinetic+amplification+of+enzyme+discrimination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000468-BarZiv1" id="pone.0000468-BarZiv1"></a>Bar-Ziv R, Tlusty T, Libchaber A (2002) Protein-DNA computation by stochastic assembly cascade. Proc Natl Acad Sci U S A  99: 11589–11592.  <ul class="find" data-citedArticleID="953658" data-doi="10.1073/pnas.162369099"><li><a href="http://dx.doi.org/10.1073/pnas.162369099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein-DNA+computation+by+stochastic+assembly+cascade." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein-DNA+computation+by+stochastic+assembly+cascade.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000468-Tlusty1" id="pone.0000468-Tlusty1"></a>Tlusty T, Bar-Ziv R, Libchaber A (2004) High-fidelity DNA sensing by protein binding fluctuations. Phys Rev Lett  93: 258103.  <ul class="find" data-citedArticleID="953722" data-doi="10.1103/physrevlett.93.258103"><li><a href="http://dx.doi.org/10.1103/physrevlett.93.258103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-fidelity+DNA+sensing+by+protein+binding+fluctuations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-fidelity+DNA+sensing+by+protein+binding+fluctuations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000468-Sagi1" id="pone.0000468-Sagi1"></a>Sagi D, Tlusty T, Stavans J (2006) High fidelity of RecA-catalyzed recombination: a watchdog of genetic diversity. Nucleic Acids Res  34: 5021–5031.  <ul class="find" data-citedArticleID="953710" data-doi="10.1093/nar/gkl586"><li><a href="http://dx.doi.org/10.1093/nar/gkl586" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+fidelity+of+RecA-catalyzed+recombination%3A+a+watchdog+of+genetic+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+fidelity+of+RecA-catalyzed+recombination%3A+a+watchdog+of+genetic+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000468-McKeithan1" id="pone.0000468-McKeithan1"></a>McKeithan TW (1995) Kinetic proofreading in T-cell receptor signal transduction. Proc Natl Acad Sci U S A  92: 5042–5046.  <ul class="find" data-citedArticleID="953692" data-doi="10.1073/pnas.92.11.5042"><li><a href="http://dx.doi.org/10.1073/pnas.92.11.5042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinetic+proofreading+in+T-cell+receptor+signal+transduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinetic+proofreading+in+T-cell+receptor+signal+transduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000468-Reiss1" id="pone.0000468-Reiss1"></a>Reiss H, Kegel WK, Katz JL (1997) Resolution of the Problems of Replacement Free Energy, 1/S, and Internal Consistency in Nucleation Theory by Consideration of the Length Scale for Mixing Entropy. Phys Rev Lett  78: 4506–4509.  <ul class="find" data-citedArticleID="953704" data-doi="10.1103/physrevlett.78.4506"><li><a href="http://dx.doi.org/10.1103/physrevlett.78.4506" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resolution+of+the+Problems+of+Replacement+Free+Energy%2C+1%2FS%2C+and+Internal+Consistency+in+Nucleation+Theory+by+Consideration+of+the+Length+Scale+for+Mixing+Entropy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resolution+of+the+Problems+of+Replacement+Free+Energy%2C+1%2FS%2C+and+Internal+Consistency+in+Nucleation+Theory+by+Consideration+of+the+Length+Scale+for+Mixing+Entropy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.XML" value="78641"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.PDF" value="375713"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g001.PNG_L" value="158379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g001.PNG_M" value="51514"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g001.PNG_S" value="2877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g001.TIF" value="470258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g001.PNG_I" value="23299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g002.PNG_L" value="139578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g002.PNG_M" value="30608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g002.PNG_S" value="2312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g002.TIF" value="238500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g002.PNG_I" value="13900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e001.PNG" value="3376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e001.TIF" value="25168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e002.PNG" value="3671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e002.TIF" value="25196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e003.PNG" value="4387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e003.TIF" value="26720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e004.PNG" value="6565"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e004.TIF" value="30348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g003.PNG_L" value="237031"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g003.PNG_M" value="49314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g003.PNG_S" value="3021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g003.TIF" value="505614"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g003.PNG_I" value="22927"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g004.PNG_L" value="114269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g004.PNG_M" value="33093"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g004.PNG_S" value="4285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g004.TIF" value="562406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g004.PNG_I" value="34492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e005.PNG" value="4248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e005.TIF" value="25712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e006.PNG" value="6776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e006.TIF" value="28080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e007.PNG" value="4138"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e007.TIF" value="25452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g005.PNG_L" value="336346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g005.PNG_M" value="29596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g005.PNG_S" value="2705"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g005.TIF" value="819166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g005.PNG_I" value="17257"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g006.PNG_L" value="329770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g006.PNG_M" value="29352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g006.PNG_S" value="2394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g006.TIF" value="806648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g006.PNG_I" value="17125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g007.PNG_L" value="799095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g007.PNG_M" value="138057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g007.PNG_S" value="5921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g007.TIF" value="1494362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g007.PNG_I" value="50649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e008.PNG" value="9991"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e008.TIF" value="31256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g008.PNG_L" value="343310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g008.PNG_M" value="26419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g008.PNG_S" value="2444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g008.TIF" value="756254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.g008.PNG_I" value="14996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e009.PNG" value="4425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e009.TIF" value="25620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e010.PNG" value="3602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e010.TIF" value="25128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e011.PNG" value="4703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e011.TIF" value="25716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e012.PNG" value="19039"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e012.TIF" value="40744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e013.PNG" value="11019"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e013.TIF" value="31984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e014.PNG" value="11926"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000468.e014.TIF" value="33320"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000468&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000468" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000468&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000468&volume=&issue=&title=Conformational Proofreading: The Impact of Conformational Changes on the Specificity of Molecular Recognition&author_name=Yonatan%20Savir%2C%20Tsvi%20Tlusty&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000468" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000468" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000468" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000468&amp;t=Conformational%20Proofreading%3A%20The%20Impact%20of%20Conformational%20Changes%20on%20the%20Specificity%20of%20Molecular%20Recognition" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000468&title=Conformational%20Proofreading%3A%20The%20Impact%20of%20Conformational%20Changes%20on%20the%20Specificity%20of%20Molecular%20Recognition&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000468&amp;title=Conformational%20Proofreading%3A%20The%20Impact%20of%20Conformational%20Changes%20on%20the%20Specificity%20of%20Molecular%20Recognition" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000468" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000468&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Conformational Proofreading: The Impact of Conformational Changes on the Specificity of Molecular Recognition';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000468';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Conformational%20Proofreading%3A%20The%20Impact%20of%20Conformational%20Changes%20on%20the%20Specificity%20of%20Molecular%20Recognition http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000468" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000468" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chemical+equilibrium%22" title="Search for articles in the subject area:'Chemical equilibrium'"><div class="flagText">Chemical equilibrium</div></a>
              <div data-categoryid="21113" data-articleid="23774"
                   data-categoryname="Chemical equilibrium"
                   class="flagImage" title="Flag 'Chemical equilibrium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="23774"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Deformation%22" title="Search for articles in the subject area:'Deformation'"><div class="flagText">Deformation</div></a>
              <div data-categoryid="17439" data-articleid="23774"
                   data-categoryname="Deformation"
                   class="flagImage" title="Flag 'Deformation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+processes%22" title="Search for articles in the subject area:'Evolutionary processes'"><div class="flagText">Evolutionary processes</div></a>
              <div data-categoryid="17655" data-articleid="23774"
                   data-categoryname="Evolutionary processes"
                   class="flagImage" title="Flag 'Evolutionary processes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Free+energy%22" title="Search for articles in the subject area:'Free energy'"><div class="flagText">Free energy</div></a>
              <div data-categoryid="18159" data-articleid="23774"
                   data-categoryname="Free energy"
                   class="flagImage" title="Flag 'Free energy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Molecular+structure%22" title="Search for articles in the subject area:'Molecular structure'"><div class="flagText">Molecular structure</div></a>
              <div data-categoryid="19867" data-articleid="23774"
                   data-categoryname="Molecular structure"
                   class="flagImage" title="Flag 'Molecular structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA-binding+proteins%22" title="Search for articles in the subject area:'RNA-binding proteins'"><div class="flagText">RNA-binding proteins</div></a>
              <div data-categoryid="43927" data-articleid="23774"
                   data-categoryname="RNA-binding proteins"
                   class="flagImage" title="Flag 'RNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="23774"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4253'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2644'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4562&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
