

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Linking Gene Expression and Functional Network Data in Human Heart Failure</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001347"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001347" />

    <meta name="citation_title" content="Linking Gene Expression and Functional Network Data in Human Heart Failure"/>
    <meta itemprop="name" content="Linking Gene Expression and Functional Network Data in Human Heart Failure"/>

      <meta name="citation_author" content="Anyela Camargo"/>
            <meta name="citation_author_institution" content="School of Computing and Mathematics, University of Ulster at Jordanstown, Newtownabbey, Northern Ireland, United Kingdom"/>
      <meta name="citation_author" content="Francisco Azuaje"/>
            <meta name="citation_author_institution" content="School of Computing and Mathematics, University of Ulster at Jordanstown, Newtownabbey, Northern Ireland, United Kingdom"/>

    <meta name="citation_date" content="2007/12/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001347.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1347"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Heart diseases and Stroke Statistics–2007 Update.; citation_number=1; citation_date=2007; citation_publisher=American Heart Association; " />
      <meta name="citation_reference" content="citation_title=Identification of a common gene expression signature in dilated cardiomyopathy across independent microarray studies.; citation_author=AS Barth; citation_author=R Kuner; citation_author=A Buness; citation_author=M Ruschhaupt; citation_author=S Merk; citation_journal_title=J Am Coll Cardiol.; citation_volume=48(8); citation_number=2; citation_pages=1610-7; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Pathway analysis of coronary atherosclerosis.; citation_author=JY King; citation_author=R Ferrara; citation_author=R Tabibiazar; citation_author=JM Spin; citation_author=MM Chen; citation_journal_title=Physiol Genomics.; citation_volume=23(1); citation_number=3; citation_pages=103-108; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An evaluation of human protein-protein interaction data in the public domain.; citation_author=S Mathivanan; citation_author=B Periaswamy; citation_author=TK Gandhi; citation_author=K Kandasamy; citation_author=S Suresh; citation_journal_title=BMC Bioinformatics.; citation_volume=7; citation_number=4; citation_pages=S19; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The gene expression fingerprint of human heart failure.; citation_author=FL Tan; citation_author=CS Moravec; citation_author=J Li; citation_author=C Apperson-Hansen; citation_author=PM McCarthy; citation_journal_title=Proc Natl Acad Sci USA.; citation_volume=20;99(17); citation_number=5; citation_pages=11387-92; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Protein interactions and disease: computational approaches to uncover the etiology of diseases.; citation_author=MG Kann; citation_number=6; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genomic expression profiling of human inflammatory cardiomyopathy (DCMi) suggests novel therapeutic targets.; citation_author=F Wittchen; citation_author=L Suckau; citation_author=H Witt; citation_author=C Skurk; citation_author=D Lassner; citation_journal_title=J Mol Med.; citation_volume=85(3); citation_number=7; citation_pages=253-67; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Predicting disease genes using protein-protein interactions.; citation_author=M Oti; citation_author=B Snel; citation_author=MA Huynen; citation_author=HG Brunner; citation_journal_title=J Med Genet.; citation_volume=43; citation_number=8; citation_pages=691-698; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Discovering disease-genes by topological features in human protein-protein interaction network.; citation_author=J Xu; citation_author=Y Li; citation_journal_title=Bioinformatics.; citation_volume=22(22); citation_number=9; citation_pages=2800-5; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Hubs in biological interaction networks exhibit low changes in expression in experimental asthma.; citation_author=X Lu; citation_author=VV Jain; citation_author=PW Finn; citation_author=DL Perkins; citation_journal_title=Mol Syst Biol.; citation_volume=3; citation_number=10; citation_pages=98; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Integration of biological networks and gene expression data using Cytoscape.; citation_author=MS Cline; citation_author=M Smoot; citation_author=E Cerami; citation_author=A Kuchinsky; citation_author=N Landys; citation_journal_title=Nat Protoc.; citation_volume=2(10); citation_number=11; citation_pages=2366-82; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Development of human protein reference database as an initial platform for approaching systems biology in humans.; citation_author=S Peri; citation_author=JD Navarro; citation_author=R Amanchy; citation_author=TZ Kristiansen; citation_author=CK Jonnalagadda; citation_journal_title=Genome Res,; citation_volume=13(10); citation_number=12; citation_pages=2363-71; citation_date=2003; " />
      <meta name="citation_reference" content="citation_number=13; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Significance analysis of microarrays applied to the ionizing radiation response.; citation_author=VG Tusher; citation_author=R Tibshirani; citation_author=G Chu; citation_journal_title=Proc Natl Acad Sci.; citation_volume=98; citation_number=14; citation_pages=5116-5121; citation_date=2001; " />
      <meta name="citation_reference" content="citation_number=15; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Apoptosis and heart failure: A critical review of the literature.; citation_author=PM Kang; citation_author=S Izumo; citation_journal_title=Circ Res.; citation_volume=86(11); citation_number=16; citation_pages=1107-13; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Signal transduction in heart failure.; citation_author=T Watanabe; citation_author=K Otsu; citation_journal_title=Nippon Rinsho.; citation_volume=62(12); citation_number=17; citation_pages=2337-42; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Recognition of the rotavirus mRNA 3′ consensus by an asymmetric NSP3 homodimer.; citation_author=RC Deo; citation_author=CM Groft; citation_author=KR Rajashankar; citation_author=SK Burley; citation_journal_title=Cell.; citation_volume=11;108(1); citation_number=18; citation_pages=71-81; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Monocyte-specific Bcl-2 expression attenuates inflammation and heart failure in monocyte chemoattractant protein-1 (MCP-1)-induced cardiomyopathy.; citation_author=J Niu; citation_author=A Azfer; citation_author=PE Kolattukudy; citation_journal_title=Cardiovasc Res.; citation_volume=71(1); citation_number=19; citation_pages=139-48; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=pathways in hypertrophy and heart failure: the gp130-STAT axis.; citation_author=P Fischer; citation_author=D Hilfiker-Kleiner; citation_journal_title=Basic Res Cardiol.; citation_volume=102(5); citation_number=20; citation_pages=393-411; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Deficiency of TNFR1 protects myocardium through SOCS3 and IL-6 but not p38 MAPK or IL-1beta.; citation_author=M Wang; citation_author=T Markel; citation_author=P Crisostomo; citation_author=C Herring; citation_author=KK Meldrum; citation_journal_title=Am J Physiol Heart Circ Physiol.,; citation_volume=292(4); citation_number=21; citation_pages=H1694-9; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gene expression analysis of ischemic and nonischemic cardiomyopathy: shared and distinct genes in the development of heart failure.; citation_author=MM Kittleson; citation_author=KM Minhas; citation_author=RA Irizarry; citation_author=SQ Ye; citation_author=G Edness; citation_journal_title=Physiol Genomics.,; citation_volume=3; citation_number=22; citation_pages=99-307; citation_date=2005; " />
      <meta name="citation_reference" content="citation_number=23; " />
      <meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks.; citation_author=P Shannon; citation_author=A Markiel; citation_author=O Ozier; citation_author=NS Baliga; citation_author=JT Wang; citation_journal_title=Genome Res.; citation_volume=13(11); citation_number=24; citation_pages=2498-504; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=BiNGO: a Cytoscape plugin to assess overrepresentation of Gene Ontology categories in Biological Networks.; citation_author=S Maere; citation_author=K Heymans; citation_author=M Kuiper; citation_journal_title=Bioinformatics.; citation_volume=21(16); citation_number=25; citation_pages=3448-3449; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001347" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Linking Gene Expression and Functional Network Data in Human Heart Failure"/>
    <meta name="twitter:description" content="BackgroundGene expression profiling and the analysis of protein-protein interaction (PPI) networks may support the identification of disease bio-markers and potential drug targets. Thus, a step forward in the development of systems approaches to medicine is the integrative analysis of these data sources in specific pathological conditions. We report such an integrative bioinformatics analysis in human heart failure (HF). A global PPI network in HF was assembled, which by itself represents a useful compendium of the current status of human HF-relevant interactions. This provided the basis for the analysis of interaction connectivity patterns in relation to a HF gene expression data set.ResultsRelationships between the significance of the differentiation of gene expression and connectivity degrees in the PPI network were established. In addition, relationships between gene co-expression and PPI network connectivity were analysed. Highly-connected proteins are not necessarily encoded by genes significantly differentially expressed. Genes that are not significantly differentially expressed may encode proteins that exhibit diverse network connectivity patterns. Furthermore, genes that were not defined as significantly differentially expressed may encode proteins with many interacting partners. Genes encoding network hubs may exhibit weak co-expression with the genes encoding their interacting protein partners. We also found that hubs and superhubs display a significant diversity of co-expression patterns in comparison to peripheral nodes. Gene Ontology (GO) analysis established that highly-connected proteins are likely to be engaged in higher level GO biological process terms, while low-connectivity proteins tend to be engaged in more specific disease-related processes.ConclusionThis investigation supports the hypothesis that the integrative analysis of differential gene expression and PPI network analysis may facilitate a better understanding of functional roles and the identification of potential drug targets in human heart failure."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001347.g003"/>

  <meta property="og:title" content="Linking Gene Expression and Functional Network Data in Human Heart Failure" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001347" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7671'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1399'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2786&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001347">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001347" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001347&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Linking Gene Expression and Functional Network Data in Human Heart Failure
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anyela Camargo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Computing and Mathematics, University of Ulster at Jordanstown, Newtownabbey, Northern Ireland, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Francisco Azuaje
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:fj.azuaje@ieee.org">fj.azuaje@ieee.org</a></p>

                <p>Affiliation:
                  School of Computing and Mathematics, University of Ulster at Jordanstown, Newtownabbey, Northern Ireland, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001347</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001347" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001347" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001347" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001347" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001347-g001" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001347-t001" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001347-g002" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001347-g003" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001347-t002" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001347-t003" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001347-t004" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001347">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001347" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Gene expression profiling and the analysis of protein-protein interaction (PPI) networks may support the identification of disease bio-markers and potential drug targets. Thus, a step forward in the development of systems approaches to medicine is the integrative analysis of these data sources in specific pathological conditions. We report such an integrative bioinformatics analysis in human heart failure (HF). A global PPI network in HF was assembled, which by itself represents a useful compendium of the current status of human HF-relevant interactions. This provided the basis for the analysis of interaction connectivity patterns in relation to a HF gene expression data set.</p>


<h3>Results</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Relationships between the significance of the differentiation of gene expression and connectivity degrees in the PPI network were established. In addition, relationships between gene co-expression and PPI network connectivity were analysed. Highly-connected proteins are not necessarily encoded by genes significantly differentially expressed. Genes that are not significantly differentially expressed may encode proteins that exhibit diverse network connectivity patterns. Furthermore, genes that were not defined as significantly differentially expressed may encode proteins with many interacting partners. Genes encoding network hubs may exhibit weak co-expression with the genes encoding their interacting protein partners. We also found that hubs and superhubs display a significant diversity of co-expression patterns in comparison to peripheral nodes. Gene Ontology (GO) analysis established that highly-connected proteins are likely to be engaged in higher level GO biological process terms, while low-connectivity proteins tend to be engaged in more specific disease-related processes.</p>


<h3>Conclusion</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This investigation supports the hypothesis that the integrative analysis of differential gene expression and PPI network analysis may facilitate a better understanding of functional roles and the identification of potential drug targets in human heart failure.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Camargo A, Azuaje F (2007) Linking Gene Expression and Functional Network Data in Human Heart Failure. PLoS ONE 2(12):
          e1347.
            doi:10.1371/journal.pone.0001347</p><p><strong>Academic Editor: </strong>Mark Isalan, Center for Genomic Regulation, Spain</p><p><strong>Received:</strong> November 7, 2007; <strong>Accepted:</strong> November 26, 2007; <strong>Published:</strong> December 19, 2007</p><p><strong>Copyright:</strong> © 2007 Camargo, Azuaje. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported in part by a grant from the European Union (FP6), CARDIOWORKBENCH project, to FA. Public data and methodologies were used. Funders did not have any role in the design and conduct of this study.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Heart failure (HF) stems from complex genetic, environmental and life style factors and is one of the main causes of death in the world <a href="#pone.0001347-American1">[1]</a>. Myocardial infarction, ischemic cardiomyopathy and dilated cardiomyopathy (DCM) may contribute to the emergence of HF. The latter, is a leading cause of congestive heart failure <a href="#pone.0001347-American1">[1]</a>, <a href="#pone.0001347-Barth1">[2]</a>. In DCM, the heart becomes enlarged, which makes the pumping of blood less efficient to vital organs. Due to the high rate of morbidity and mortality attributed to HF, previous studies have aimed to unveil the genetic factors crucial to the emergence and development of the disease. As a result, HF signature genes <a href="#pone.0001347-King1">[3]</a>, <a href="#pone.0001347-Mathivanan1">[4]</a>, <a href="#pone.0001347-Tan1">[5]</a>, protein-protein interactions (PPI) and larger gene expression data sets have been made publicly available <a href="#pone.0001347-Kann1">[6]</a>. Thus, it has been suggested that the integration of these sources may improve the identification of clinically-relevant disease markers <a href="#pone.0001347-Kann1">[6]</a>, <a href="#pone.0001347-Wittchen1">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Recent examples of the predictive power of integrative bioinformatics approaches to investigating diseases have been reported by Oti et al. (2006) <a href="#pone.0001347-Oti1">[8]</a> and Xu and Li (2006) <a href="#pone.0001347-Xu1">[9]</a>. They investigated whether signature genes of genetically heterogeneous and hereditary diseases could be predicted from the analysis of PPI networks. Lu et al. (2007) <a href="#pone.0001347-Lu1">[10]</a> integrated gene expression analysis and a biological interaction network to investigate the allergic response in asthma. Cline et al. (2007) <a href="#pone.0001347-Cline1">[11]</a> proposed a generic protocol to integrate gene expression data and biological networks, which may help to explain the control mechanisms underlying the observed changes in activity of a biological process. In the context of HF, Barth et al. (2006) <a href="#pone.0001347-Barth1">[2]</a> analysed gene expression patterns related to DCM and identified specific gene regulatory relationships. Here, using a DCM-related microarray data set, we report an analysis of human HF gene expression responses in relation to a HF-specific PPI network.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To build the HF PPI network, known HF-relevant genes (KHFG) were first identified together with validated PPIs of their encoded proteins according to the Human Protein Reference Database (HPRD) <a href="#pone.0001347-Peri1">[12]</a>. This was followed by the identification of differentially expressed genes from microarray data encoding molecular profiles of healthy vs. HF subjects. The proteins encoded by these significantly differentially-expressed genes were mapped onto the global HF PPI network. We first assessed key statistical and topological relationships between significantly differentially-expressed genes and the interaction network. Results showed that in terms of gene expression, genes significantly differentially expressed are not always represented by highly-connected nodes. Other results showed that, although not significantly differentially-regulated, some of the proteins encoded by genes traditionally associated with HF may interact with proteins encoded by significantly differentially-expressed genes, and that the latter tend to be highly connected. Further analyses, which integrated expression data and the PPI network, evaluated levels of co-expression between genes encoding network nodes and corresponding genes encoding their interacting partners. A key question was whether there was any significant quantitative relation between co-expression levels and network connectivity degree. This analysis indicate that: a) genes represented by network hubs may exhibit weak co-expression with the genes encoding their interacting protein partners; and b) genes that were not defined as significantly differentially-expressed in the gene expression data analysis may encode proteins with many interacting partners. These significant findings were replicated using a second, independent microarray data set. To identify biological process overrepresentations associated with the PPI network's topology, the data were analysed in the context of the Gene Ontology. Results show that genes represented by highly-connected nodes are more likely to be engaged in higher-level biological process terms (as defined by the Gene Ontology), than genes represented by low-connectivity nodes. This study supports the idea that a PPI network integrated with expression data may further assist researchers in identifying potential disease markers or therapeutic targets, which might be overlooked when results rely on expression profiling analyses only.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>This study evaluated human HF gene expression responses, in relation to the topology of a HF-specific PPI network. The microarray data set analysed was obtained from the Gene Expression Omnibus (GEO) <a href="#pone.0001347-Gene1">[13]</a>, accession number GDS2206. This data set, which was derived from a study on DCM, consisted of 28 samples: 15 and 13 samples obtained from non-failing hearts and HF patients respectively. After pre-processing, significantly differentially-expressed genes were identified by performing significance analysis of microarray (SAM) <a href="#pone.0001347-Tusher1">[14]</a>. To validate significant findings, a second microarray data set was used. This second data set, also derived from a study on DCM, was obtained from the GEO, accession number GDS2206, and included 12 samples: 5 samples originated from non-failing hearts and 7 samples from HF patients (see <a href="#s4">Methods</a>).</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>The PPI network was assembled by including validated interactions, as reported in the HPRD <a href="#pone.0001347-Peri1">[12]</a>, for KHFGs and for proteins encoded by genes included in the expression data sets. For the network visualisation, a colour labelling scheme was used to distinguish between the types of proteins each node represented. In addition, nodes were classified according to the degree of connectivity. Superhubs were represented by nodes with connectivity degree greater than 100, hubs referred to nodes with connectivity degree greater than 20 and lower than 100, peripheral-A were nodes with connectivity greater than two and lower than 20; and peripheral-B nodes represented proteins with one interacting partner (see <a href="#s4">Methods</a>).</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>To evaluate relationships between connectivity and significantly differentially-expressed expression patterns, topological analysis of the network was carried out. Furthermore, network topology was integrated with the DCM expression data to evaluate gene connectivity versus co-expression levels. To calculate co-expression levels linked to every node in the network, the nodes and their interaction partners were mapped into the DCM expression data set. Next, Pearson's correlation coefficients between the expression profile of each gene coding for a node and each gene coding for its interaction partners were calculated. Correlation value pairs were regarded as significantly co-expressed if P-value&lt;0.01. Finally, the co-expression level of a gene was calculated by comparing the number of its significantly co-expressed interactions against the total number of its interacting proteins (see <a href="#s4">Methods</a>). The PPI network was analysed in the context of Gene Ontology (GO) to identify biological process overrepresentations. Over-represented biological processes were ranked according to their position in the GO hierarchical scheme <a href="#pone.0001347-Gene2">[15]</a> (see <a href="#s4">Methods</a>).</p>

<h4>Network analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The PPI network (<a href="#pone-0001347-g001">Figure 1A</a>) consisted of nodes representing proteins and their interaction partners. Some of the nodes represented proteins encoded by significantly differentially-expressed genes obtained from expression pattern analysis. Initially, 1161 genes were identified (974 up-regulated in DCM and 187 down-regulated in DCM). However, only 506 (457 up-regulated and 49 down-regulated genes) were represented in the network because their encoded proteins were reported to have at least one interacting protein partner. The network also contained 71 nodes representing proteins encoded by KHFGs only. The network contained 2835 nodes representing proteins encoded by not significantly differentially-expressed genes (<a href="#pone-0001347-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001347-g001"><div class="img"><a name="pone-0001347-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001347.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001347.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>PPIs networks. PPIs networks corresponding to (A) global human HF network, (B) BCAR1's PPI network, (C) AKT1's PPI network. All PPIs were retrieved from the HPRD. Up-regulated genes are represented by red nodes. Down-regulated genes are represented by green nodes. Known HF genes (KHFG) are represented by nodes in yellow. Other genes encoding interacting partner proteins are represented by nodes in purple, if they have a corresponding transcript, or in grey if they have no corresponding transcripts in the gene expression data set.</p>
<span>doi:10.1371/journal.pone.0001347.g001</span></div><div class="figure" id="pone-0001347-t001"><div class="img"><a name="pone-0001347-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001347.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001347.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of nodes population according to connectivity.</span></strong></p><span>doi:10.1371/journal.pone.0001347.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>According to the node hierarchy described in <a href="#s4">Methods</a>, 2.3% of the genes were represented by network hubs or superhubs corresponding to 47 significantly differentially-expressed genes (4 superhubs and 43 hubs). In contrast, 97.0% of the genes were represented by either network peripheral-A or peripheral–B nodes corresponding to 459 significantly differentially-expressed genes (279 peripheral-A and 180 peripheral-B). Details are shown in <a href="#pone-0001347-t001">Table 1</a>. Three statistical significance tests based on random sampling were implemented to allow us to reject the null hypothesis that the proportion of significantly differentially-expressed genes (which were also categorised as either hubs or superhubs) was obtained by chance. These tests are described in <a href="#s4">Methods</a>. All the statistical significance tests reported <em>P</em> = 0. Thus, this supports the conclusion that the observed proportions are statistically significant, i.e. larger than the proportions expected by chance.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Examples of potentially relevant associations are described as follows. <em>BCAR1</em> is a protein represented by a network superhub (<a href="#pone-0001347-g001">Figure 1B</a>) and <em>NAP1L1</em> is represented by a peripheral-B. According to the gene expression analysis, both <em>BCAR1</em> and <em>NAP1L1</em> encoded significantly differentially-expressed genes. <em>AKT1</em> also represents a network superhub (<a href="#pone-0001347-g001">Figure 1C</a>). The gene encoding <em>AKT1</em> is known to be associated with HF and it is involved in several functional processes relevant to the development of this disease, such as Apoptosis and the <em>MAPK</em> (Mitogen-activated protein kinase) signalling pathway <a href="#pone.0001347-Kang1">[16]</a>.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Only 40, out 71, KHFGs had a corresponding transcript in the DCM data set, and only one of these genes, <em>SOD1</em>, was significantly differentially-expressed. Moreover, 2051 genes that encoded other protein's interaction partners in the network had a corresponding transcript in the gene expression data set and were not significantly differentially-expressed. There were KHFGs with no corresponding transcripts in the DCM data set because these were either included in the array but with significant missing values across the experimental samples, or their probes were not included in the array.</p>


<h4>Network connectivity versus significant gene expression patterns</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>This section of the study integrated gene expression data with the PPI network to describe potential significant relationships between network connectivity and gene expression patterns (as described in <a href="#s4">Methods</a>). The first set of results, involving significantly differentially- expressed genes, found that genes represented by network superhubs and hubs tend to have lower range of ‘<em>di</em>’s values (the score of class differentiation). In <a href="#pone-0001347-g002">Figure 2</a> genes with those characteristics are shown on the farthest right side of the plot. On the contrary, genes represented by network peripherals-A and -B tend to have higher range of ‘<em>di</em>’s values. When proteins encoded by non-significantly differentially-expressed genes were assessed, we found that some of these protein's interacting partners were encoded by several significantly differentially-expressed genes. For instance, <em>GRB2</em> has 180 interacting partners and was not found to be significantly differentially-expressed in the gene expression data. However, 19 genes encoding its interacting partners, such as <em>ABL1</em> or <em>BCAR1</em>, were identified as significantly differentially-expressed in the expression data analysis. We analysed the biological role of <em>GRB2</em>, and its corresponding interacting partners, and found that processes such as ‘signal transduction’, “regulation T Cell activation” or “regulation of <em>MAPK</em> activity” were over-represented (P&lt;0.0001). According to KEGG and Reactome, <em>GRB2</em> is involved in more than 15 pathways. Other proteins whose interacting partners were encoded by more than 15 significantly differentially-expressed genes were <em>TP53</em>, <em>NR3C1</em>, <em>SMAD2</em>, <em>CASP3</em>, <em>ESR1</em>, <em>RB1</em> and <em>YWHAG</em> (<a href="#pone.0001347.s001">Annex S1</a> shows complete list), which are involved in functional processes such as apoptosis or cell cycle. These findings stress the importance of performing gene expression analysis in conjunction with interaction networks to help to identify otherwise overlooked potential clinically-relevant targets.</p>
<div class="figure" id="pone-0001347-g002"><div class="img"><a name="pone-0001347-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001347.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001347.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Plot of t-statistics (<em>cL</em>).</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Plot of t-statistic (<em>di</em>) representing the score for gene <em>i</em> vs. number of interacting partners (IP Log2) associated with protein encoded by gene <em>i</em>.</p>
<span>doi:10.1371/journal.pone.0001347.g002</span></div>

<h4>Gene co-expression analyses in the context of network connectivity</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In this section gene expression data were integrated with the topology of the PPI network to assess significant <em>co-expression levels</em> (as detailed in <a href="#s4">Methods</a>). We found that genes represented by network hubs and superhubs are not necessarily significantly co-expressed with their attributed protein-coding partners (IPs), than other types of nodes. For example, genes MAPK1 and FXR2, represented by network superhubs, were significantly co-expressed with 15.5% and 10.2% of the genes encoding their IPs respectively. On the other hand, genes represented by network peripheral-A and -B may be strongly correlated with their interacting partners. For example, ALDOB, represented by a network peripheral-A, was significantly co-expressed with all the genes encoding its IPs (i.e. 100% significant co-expression level). <a href="#pone-0001347-t002">Table 2</a> shows more details of the difference between these categories in terms of mean <em>cL</em> values. No statistical significance difference between category means were found at <em>P</em> = 0.05. However, note that only network peripherals-A or –B showed cases with <em>cL</em> = 100%. The global trend, as shown in <a href="#pone-0001347-g003">Figure 3</a>, is that the higher the number of node connections the greater the tendency to display low <em>cL</em> values. <a href="#pone-0001347-g003">Figure 3A</a> shows a scatter plot of the number interacting partners (<em>IP<sub>i</sub></em>) for a gene <em>i</em>, versus its significant co-expression level (<em>cL<sub>i</sub></em>) (as defined in <a href="#s4">Methods</a>). Similar trend was observed when non-significantly differentially expressed genes were analysed. For example, <em>HAP1</em> and <em>SIN3A</em>, represented by network peripherals-A, were significantly co-expressed with all their partners, <em>IPs</em>. <a href="#pone-0001347-g003">Figure 3B</a> plots (<em>IP<sub>i</sub></em>) versus (<em>cL<sub>i</sub></em>) of non-significantly differentially-expressed genes. When analysing nodes representing KHFGs, results showed that in general <em>cL<sub>i</sub></em> of these genes was low. In fact, none of the 40 KHFGs, which had a corresponding transcript in the DCM expression data, obtained a <em>cL<sub>i</sub></em>&gt;50% (<a href="#pone-0001347-g003">Figure 3C</a>). For example, the <em>cL<sub>i</sub></em> for <em>PRKCA</em>, represented by a network superhub, was equal to 16.2% (i.e. this gene's expression pattern was significantly co-expressed with only a few of the genes coding for its <em>IPs</em>).</p>
<div class="figure" id="pone-0001347-g003"><div class="img"><a name="pone-0001347-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001347.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001347.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Scatter plot of the number interacting partners (<em>IP<sub>i</sub></em>) for a gene <em>i</em>, versus its significant co-expression level (<em>cL<sub>i</sub></em>).</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) significantly differentially-expressed genes. (B) Non-significantly differentially-expressed genes. (C) KHFGs. (D) Independent testing data set.</p>
<span>doi:10.1371/journal.pone.0001347.g003</span></div><div class="figure" id="pone-0001347-t002"><div class="img"><a name="pone-0001347-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001347.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001347.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Summary of average co-expression levels (<em>cL</em>) for each network node category.</span></strong></p><span>doi:10.1371/journal.pone.0001347.t002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In order to estimate the potential biological relevance of the significant relationships found, as well as their reproducibility, these pattern association procedures were replicated on a second, independent human HF gene expression data set. <a href="#pone-0001347-g003">Figure 3D</a>, which reviews the overall results of this second analysis, corroborates the findings presented above. Genes encoding network hubs and superhubs tend to be more weakly co-expressed with the genes coding for their network interacting partners (<em>IP<sub>i</sub></em>) than those genes represented by network peripherals-A and –B. For example, <em>cL<sub>i</sub></em> of genes encoding network superhubs and hubs, such as <em>AR</em>, <em>MAPK1</em>, <em>PTPN11</em> or <em>RAF1</em>, were lower than 5%. On the contrary, the <em>cL<sub>i</sub></em> of <em>MRFAP1L1</em>, encoding a network peripheral-A, was equal to 100%.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The final analysis of this sequence evaluated the variation of co-expression levels (<em>cLs</em>) across the four categories of network nodes. Gene's <em>cL</em> values were grouped according to node category and the significance of the variance among these categories was estimated by <em>F</em>-statistics. Results indicated (<a href="#pone-0001347-t003">Table 3</a>) that there are significant differences between high connectivity categories (hubs and superhubs) and low connectivity categories (peripherals-A and –B) (<em>F</em> = 10.93, <em>P</em>&lt;0.000001). By contrast, <em>F</em>-statistics found no significant difference when comparing high connectivity categories (network hubs vs. superhubs) (<em>F</em> = 4.53, <em>P</em> = 0.14). Further, analysis of variance (ANOVA) among all four categories was performed, from which no significant differences were found (<em>F</em> = 0.93, <em>P</em> = 0.42). This analysis suggests that hubs and superhubs display a significant diversity of co-expression patterns in comparison to peripheral nodes.</p>
<div class="figure" id="pone-0001347-t003"><div class="img"><a name="pone-0001347-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001347.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001347.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Analysis of variance of co-expression levels (<em>cL</em>) across categories of network nodes.</span></strong></p><span>doi:10.1371/journal.pone.0001347.t003</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Genes were ranked according to their co-expression levels (<em>cL</em>), such that a gene, <em>i</em>, was defined as <em>low co-expressed</em> if its <em>cL<sub>i</sub></em> was below 0.80 and <em>highly co-expressed</em> if its <em>cL<sub>i</sub></em> was greater or equal than 0.80. For example, <em>MAPK1</em>'s <em>cL</em> is equal to 0.15, therefore the gene was labelled as low co-expressed. An example of a highly co-expressed gene is <em>ALDOB</em>, whose <em>cL</em> is equal to 1.0 (100% significant co-expression). Once the genes were grouped according to this criterion, F-statistics were applied to estimate variability between significant co-expression levels of these two groups. We found that the difference of significant co-expression levels between these two groups was statistically significant (<em>F</em> = 103.97, <em>P</em>&lt;0.00001). Through this analysis, we found that the <em>cLs</em> of <em>MAP3K5</em> and <em>PABPC1</em> (represented by network peripherals-B) were of 0.80 and 0.87 respectively. Watanabe and Otsu (2004) <a href="#pone.0001347-Watanabe1">[17]</a> reported that <em>MAP3K5</em> promotes heart dysfunction and dilation, as well as cardiac fibrosis. A previous study by Deo et al. (2002) <a href="#pone.0001347-Deo1">[18]</a> linked <em>PABPC1</em> to <em>SOD1</em>, the latter was the only KHFG significantly differentially-expressed in our expression analysis.</p>


<h4>GO analysis</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The PPI network topology was also assessed in the context of the GO to identify major biological roles within each node category. Genes represented by network peripherals-A tend to be involved in a greater number of lower-level GO biological processes (e.g. “CD4-positive, alpha-beta T cell differentiation during immune response”, “G1/S transition of mitotic cell cycle”, “insulin-like growth factor receptor signalling pathway”) than those represented by network hubs and peripherals-B (<a href="#pone-0001347-t004">Table 4</a>). The “G1/S transition of mitotic cell cycle” biological process has been reported to be fundamental in the development of HF <a href="#pone.0001347-King1">[3]</a>. <a href="#pone-0001347-t004">Table 4</a> also shows that genes represented by network superhubs tend to be involved in a greater number of higher-level biological processes (e.g. “cell communication” or “signal transduction”). These results are consistent with results obtained by Lu et al. (2007) <a href="#pone.0001347-Lu1">[10]</a> in an asthma model. However, we also demonstrated that genes represented by network peripherals-B tend to perform higher-level biological processes (<a href="#pone.0001347.s002">Annex S2</a> contains complete list of over-represented GO biological processes). But the latter may be explained by the relative lack of experimental studies of these genes.</p>
<div class="figure" id="pone-0001347-t004"><div class="img"><a name="pone-0001347-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001347" data-uri="info:doi/10.1371/journal.pone.0001347.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001347.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001347.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001347.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001347.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>GO analysis, significantly over-represented biological processes.</span></strong></p><span>doi:10.1371/journal.pone.0001347.t004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>This analysis also corroborates that some genes found to be not significantly differentially-expressed may influence the activation or repression of several other gene products. For example, the protein encoded by <em>NR3C1</em> gene is known to interact with 82 proteins. GO analysis of <em>NR3C1</em>, and corresponding interacting partners, identified more than 60 significantly over-represented biological processes. Examples of such relevant functional categories are “transcription” or “transcription from RNA polymerase II promoter” (P&lt;0.00001). This gene was not significantly differentially-expressed, but its encoded protein interacts with 17 proteins encoded by genes that were significantly differentially-expressed in the expression data analysis (e.g. <em>HNRPU</em>, <em>MAPK1</em>, <em>PRPF6</em>, <em>RAF1</em>, <em>NFKB1</em> and <em>STAT3</em>). These genes are known to be involved in crucial processes relating to cardiac remodelling, such as apoptosis, <em>MAPK</em> signalling pathway or immune system signalling as described in KEGG and Reactome databases and by other authors <a href="#pone.0001347-King1">[3]</a>, <a href="#pone.0001347-Kang1">[16]</a>, <a href="#pone.0001347-Niu1">[19]</a>. In addition, Kang and Izumo (2000) <a href="#pone.0001347-Kang1">[16]</a> suggested that DCM is a one of the most common forms of HF associated with apoptosis pathways dysfunction.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Undoubtedly gene expression pattern analyses have provided insights into the biological basis of several deadly diseases. Because of this, an important question is how to take advantage of available public data and information bases. Moreover, there are concerns about the reproducibility of functional predictions and pattern identification results obtained using different data sets and platforms.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In human heart failure, a fair amount of microarray expression data have been produced and uploaded into public databases. Moreover, a number of disease signature genes have been reported <a href="#pone.0001347-Barth1">[2]</a>, <a href="#pone.0001347-King1">[3]</a>, <a href="#pone.0001347-Wittchen1">[7]</a>. The challenge now is to find ways to integrate such information in order to facilitate the discovery of disease-specific knowledge and targets, as well as their reproducibility using different data sources. This study evaluated gene expression responses from the perspective of a PPI network and several external functional information sources. Quantitative analysis approaches were applied to elucidate significant global patterns and trends encoded in these data sources. In particular, we address the following questions: a) how network-based targets relate to significantly differentially (or not differentially) gene expression-based targets; and b) how network connectivity relates to significant levels of co-expression between interacting partners. To answer these questions a PPI network based on proteins encoding KHFGs and other expressed genes was built. Significant quantitative relationships between gene expression data and the PPI network were identified. Such relationships and relevant functional patterns may allow one to identify significant genes and specific biological processes, which may be overlooked when analyses rely on gene expression data only. Furthermore, significant quantitative relationships were reproduced using an independent data set.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The integration of expression data and the PPI network allowed us to identify functionally-important, influential genes, which were not found to be significantly differentially expressed in the expression data analysis. Some of the proteins encoded by these genes were reported to influence relatively large sets of proteins, which may be encoded by significantly differentially-expressed genes. For instance, proteins encoded by genes <em>SMAD2</em>, <em>CASP3</em>, <em>ESR1</em> and <em>RB1</em>, interacted with 17, 16, 16 and 15 proteins encoded by significantly differentially-expressed genes. In the case of KHFGs, nearly 50% of them had no corresponding transcripts in the DCM data sets. However, the proteins they encoded were reported to interact with several others proteins, which were encoded by significantly differentially-expressed genes. For example, the protein encoded by <em>STAT3</em> (represented by a network hub) was reported to interact with other 94 proteins. Among them, nine were encoded by significantly differentially-expressed genes, such as <em>MAPK1</em> or <em>PDIA3</em>, which are involved in functional pathways such as natural killer cell mediated cytotoxicity or MAPK signalling. These pathways are involved in the aetiology of HF <a href="#pone.0001347-King1">[3]</a>. In addition, evidence based on published papers has demonstrated the involvement of <em>STAT3</em> in the protection of the myocardium from HF <a href="#pone.0001347-Fischer1">[20]</a>, <a href="#pone.0001347-Wang1">[21]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The integrative analysis of gene expression data and the PPI network suggests that proteins represented by network peripherals-A and -B tend to be encoded by genes that are significantly co-expressed with most of the genes encoding their interacting proteins. This relationship was not observed in the case of network hubs and superhubs. In terms of biological significance, these analyses also showed that genes encoding network superhubs and hubs tend to be involved in higher-level biological processes as defined by the GO Biological Process hierarchy. Furthermore, genes represented by network peripherals-A tend to be associated with lower-level biological processes.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The integrative study reported here highlights three important aspects. First, the approaches implemented may help to identify potentially influential genes (e.g. <em>STAT3</em>, <em>TGFB1</em>, <em>AKT1</em>, <em>SIN3A</em>, <em>PABPC1</em>, <em>MAP3K5</em>), otherwise overlooked by single-source expression pattern analysis. Second, it also represents a powerful methodology to trace biological processes that may outline potential clinical biomarkers or therapeutic targets and their corresponding interacting partners. Third, in combination with expression data this approach can be used as a tool to evaluate stimulus/response studies. For example, two microarray data sets can be mapped onto an interaction network to analyse gene repression or activation when different experimental conditions are studied. The latter application will be reported as part of a forthcoming study.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Despite the fact that the global human HF network presented here is far from complete and that it may include false positive interactions, biologically-significant quantitative and qualitative relations were identified. Moreover, significant quantitative findings were reproduced when an independent gene expression data set was analysed. Quantifiable differences between potentially influential functional components (as predicted by independent single-source analyses and categories) were also found <a href="#pone.0001347-Barth1">[2]</a>, <a href="#pone.0001347-King1">[3]</a>, <a href="#pone.0001347-Wittchen1">[7]</a>.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>A key conclusion of this research is that only a minority of hubs (or superhubs) genes are also significantly differentially expressed in HF. Furthermore, we showed that the observed proportions of such genes are statistically significant in comparison to the values expected by chance. The biological relevance of these results can be summarised as follows. First of all, this study confirms the weakness of performing functional characterisations of genes based on gene expression data only. The integrative analysis of gene expression and functional network data may improve the predictive ability of future studies. The identification of potential drug targets in heart failure should include integrative approaches to estimate significant roles of genes in regulatory processes driving the emergence of heart failure. Second, this study suggests that processes relevant to cardiac remodelling and the progression toward heart failure may be controlled by greater gene expression modifications of low-connectivity network nodes, and relatively smaller differential responses in nodes with higher connectivity. Moreover, hubs and superhubs display a significant diversity of quantitative co-expression patterns in comparison to peripheral nodes. This may suggest that gene expression coordination between hubs (or superhubs) and their interacting partners may be more subtle that that observed between other genes.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Microarray data analysis</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The microarray data analysed in this study were obtained from the Gene Expression Omnibus GEO) <a href="#pone.0001347-Gene1">[13]</a>, accession number GDS2206. This data set, which was derived from a study on DCM, was composed of 28 samples: 15 and 13 samples from non-failing hearts and HF patients respectively <a href="#pone.0001347-Barth1">[2]</a>, <a href="#pone.0001347-Kittleson1">[22]</a>. This data set was available in Log scale <a href="#pone.0001347-Barth1">[2]</a>, and probes with missing values in more than 50% of samples, in either group, were excluded. To quantity differential gene expression, significance analysis of microarray (SAM) <a href="#pone.0001347-Tusher1">[14]</a> was performed. The algorithm computes a <em>t</em> statistic, <em>di</em>, representing the score of class differentiation for gene <em>i</em>. Gene expression differences were considered significant if False Discovery Rate (FDR) &lt;0.05 and Folding change &gt;1.2.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>In addition, a second microarray data set was used to validate significant findings (i.e. gene co-expression vs. network patterns) obtained from the first gene expression data set. This second data set, also derived from a study on DCM, was obtained from the GEO, accession number GDS2206, and was composed of 12 samples: 5 samples originated from non-failing hearts and 7 samples were obtained from HF patients <a href="#pone.0001347-Barth1">[2]</a>, <a href="#pone.0001347-Kittleson1">[22]</a>.</p>


<h4>A human HF interaction network</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>A PPI network was assembled by including validated interactions reported for KHFGs and for proteins encoded by genes included in the gene expression data sets. This network is offered as a public resource of the current status of human HF-relevant interactions (network is provided on request). The list of KHFG was obtained from the Entrez database <a href="#pone.0001347-Entrez1">[23]</a>. Entrez query was restricted by the same set of keywords used in King et al. (2003) <a href="#pone.0001347-King1">[3]</a> (i.e. smooth muscle, endothelial cell, apoptosis, cytokine and adhesion molecule) and within the context of human HF. PPIs were retrieved from the HPRD <a href="#pone.0001347-Mathivanan1">[4]</a>. The HUGO nomenclature standard was used to define unique id identifiers.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>The PPI network was assembled by using a routine written in JAVA, and its structure was encoded in the SIF format that can be used by well-known network visualisation tools (e.g. Cytoscape). The product of this assembly was a network composed of 3412 nodes and 13164 interactions. The number of interacting partners range from 1 to more than 100. A colour labelling scheme was used to distinguish between the types of proteins each node represented. Proteins encoded by up- and down-regulated genes (as predicted in the gene expression data) were represented by nodes coloured in green and red respectively. Proteins encoded by KHFGs were represented by yellow nodes. Proteins encoded by not significantly differentially expressed genes were represented by purple nodes. Proteins encoded by genes whose expression pattern was not present in the data set, but which encoded relevant interacting partners in the HF network, were represented by grey nodes.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Nodes in the network were also classified according to the degree of connectivity, based on a scheme similar to that used in Lu et al. <a href="#pone.0001347-Lu1">[10]</a>. <em>Superhubs</em> are represented by nodes with connectivity degree greater than 100, <em>hubs</em> refer to nodes with connectivity degree greater than 20 and lower than 100, <em>peripheral-A</em> are nodes with connectivity greater than two and lower than 20; and <em>peripheral-B</em> nodes represent proteins with one interacting partner only.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>Cytoscape v2.4 <a href="#pone.0001347-Shannon1">[24]</a> was used for network visualisation.</p>


<h4>Comparing observed vs. expected proportions of relevant nodes</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The statistical significance of the observed proportion (<em>R<sub>obs</sub></em>) of significantly differentially-expressed genes that were also either hubs or superhubs was estimated by three independent statistical significance tests. In these analyses the null hypothesis was that the observed proportion was obtained by chance, i.e. a random null distribution of genes could have reported proportions as extreme as the value observed. These statistical significance analyses were implemented as follows. Based on the results shown above, each gene was assigned to one of the following labels: “Significantly differentially-expressed” (SDE) or “not-SDE”. This vector of labels will be referred to as “expression labels”. Similarly, each gene was labelled according to the binary class: “Superhub-or-hub” or otherwise. This vector of labels will be referred to as “connectivity labels”. The first statistical significance assessment was based on a permutation test as follows. The expression labels were randomly shuffled to obtain a permutated dataset. Using this dataset, the proportion (<em>R<sub>per</sub></em>) of SDE that were also either superhubs or hubs was calculated. This permutation process was repeated <em>N</em> times, and the number of times, <em>numSigPer</em>, that <em>R<sub>per</sub>≥R<sub>obs</sub></em> was calculated, with <em>R<sub>obs</sub></em> = 47/3412. Thus, the statistical significance of the observed value (probability that the observed value was obtained by chance) was estimated by <em>P</em> = <em>numSigPer</em>/<em>N</em>, with <em>N</em> = 50000. A second permutation test was conducted in a similar fashion, but now with only the connectivity labels randomly shuffled to generate the permutated datasets. In the third significance assessment test, 506 proteins (i.e. the number of SDE genes observed) were randomly sampled from the total population of network nodes (3412 proteins). Based on this sample, <em>R<sub>rand</sub></em> represents the proportion of genes that were both SDE and that encoded either hubs or superhubs. This sampling process was repeated <em>M</em> times, and the number of times, <em>numSigRand</em>, that <em>R<sub>rand</sub></em>≥<em>R<sub>obs</sub></em> was calculated, with <em>R<sub>obs</sub></em> = 47/506. Thus, the probability of finding <em>R<sub>obs</sub></em> at random was estimated by <em>P</em> = <em>numSigRand</em>/<em>M</em>, with <em>M</em> = 50000. All the statistical significance tests reported <em>P</em> = 0, with maximum <em>R<sub>per</sub></em> and <em>R<sub>rand</sub></em> values equal to 20/3412 and 19/506 respectively.</p>


<h4>Network vs. gene co-expression analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Results from the topological analysis of the network were integrated with the results obtained from the gene expression data analysis in order to evaluate relationships between connectivity and significantly differentially expressed expression patterns. Furthermore, network topology was integrated with the DCM expression data to evaluate gene connectivity versus co-expression levels.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>To calculate co-expression levels linked to every node (<em>i</em>) in the network, node and their interaction partners were mapped into the gene expression data set. Next, Pearson's correlation coefficients between the expression profile of each gene coding for a node and each gene coding for its interaction partners were calculated. Correlation value pairs were regarded as significantly co-expressed if its P-value&lt;0.01. Finally, the <em>co-expression level</em> (<em>cL</em>) of a gene (<em>i</em>) was calculated by comparing the number of its significantly co-expressed interactions (<em>cIP<sub>i</sub></em>) against the total number of its interacting proteins (<em>IP<sub>i</sub></em>), i.e. significantly co-expressed and non-significantly co-expressed interaction partners (Eq. 1). According to Eq. 1, a co-expression level (<em>cL<sub>i</sub></em>) equal to 100% indicates that the gene (<em>i</em>) was significantly co-expressed with each of the genes encoding its interacting partners in the PPI network. A co-expression level (<em>cL<sub>i</sub></em>) equal to 0% indicates that the gene (<em>i</em>) was not significantly co-expressed with any of its interacting partners.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001347.e001&amp;representation=PNG" class="inline-graphic"></span><br>Assuming that a node, <em>i</em>, in the PPI network represents a protein encoded by a gene, then: <em>i</em> is a node in the network; <em>IP<sub>i</sub></em> represents the set of interacting partners linked to <em>i</em>; <em>cIP<sub>i</sub></em> represents the set of interacting partners significantly co-expressed with <em>i</em>.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>To verify that major findings were reproducible, the same procedures were carried out using a second, independent microarray data set as described above.</p>


<h4>Gene Ontology analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Interaction network topology was analysed in the context of the GO. Cytoscape-BiNGO <a href="#pone.0001347-Maere1">[25]</a> was applied to detect significantly over-represented GO biological processes. Benjamini and Hochberg multiple-test corrections adjusted raw <em>P</em>-values at a significant level&lt;0.05. To increase the level of stringency, GO-IEA terms were discarded. GO annotations with IEA evidence code refer to annotations inferred from sequence-based similarity searches, which have not been reviewed by curators. In addition, using the GO interaction network, a hierarchical classification scheme was used to rank over-represented GO biological processes according to their proximity to the root node of the GO Biological Process hierarchy. Note that the GO root node is at the top level of the hierarchy, followed by the Biological Process term and the rest of the hierarchy terms (e.g., “developmental process” (third level), “multicellular organismal development”, on the fourth level). In this study a <em>higher-level Biological Process</em> is defined as any term above the fourth level in the GO Biological Process hierarchy. A <em>lower-level Biological Process</em> is defined as any annotation subsumed by any term in the fourth level in the GO Biological Process hierarchy.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001347"></div><a name="pone.0001347.s001" id="pone.0001347.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001347.s001">Annex S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Network proteins and corresponding number of interacting partners encoded by significantly differentiated genes</p>
<p class="siDoi">doi:10.1371/journal.pone.0001347.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.17 MB XLS)</p>
<a name="pone.0001347.s002" id="pone.0001347.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001347.s002">Annex S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Significantly over-represented GO biological processes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001347.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.09 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank the reviewers for corrections and helpful suggestions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: FA AC. Performed the experiments: AC. Analyzed the data: AC. Wrote the paper: FA AC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001347-American1" id="pone.0001347-American1"></a>American Heart Association (AHA) (2007) Heart diseases and Stroke Statistics–2007 Update. American Heart Association.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001347-Barth1" id="pone.0001347-Barth1"></a>Barth AS, Kuner R, Buness A, Ruschhaupt M, Merk S, et al.  (2006) Identification of a common gene expression signature in dilated cardiomyopathy across independent microarray studies. J Am Coll Cardiol.  48(8): 1610–7.  <ul class="find" data-citedArticleID="1033108" data-doi="10.1016/j.jacc.2006.07.026"><li><a href="http://dx.doi.org/10.1016/j.jacc.2006.07.026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+common+gene+expression+signature+in+dilated+cardiomyopathy+across+independent+microarray+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+common+gene+expression+signature+in+dilated+cardiomyopathy+across+independent+microarray+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001347-King1" id="pone.0001347-King1"></a>King JY, Ferrara R, Tabibiazar R, Spin JM, Chen MM, et al.  (2005) Pathway analysis of coronary atherosclerosis. Physiol Genomics.  23(1): 103–108.  <ul class="find" data-citedArticleID="1033126"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Pathway+analysis+of+coronary+atherosclerosis.&amp;auth=&amp;atitle=Pathway+analysis+of+coronary+atherosclerosis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathway+analysis+of+coronary+atherosclerosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathway+analysis+of+coronary+atherosclerosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001347-Mathivanan1" id="pone.0001347-Mathivanan1"></a>Mathivanan S, Periaswamy B, Gandhi TK, Kandasamy K, Suresh S, et al.  (2006) An evaluation of human protein-protein interaction data in the public domain. BMC Bioinformatics.  7: Suppl 5S19.  <ul class="find" data-citedArticleID="1033134" data-doi="10.1186/1471-2105-7-s5-s19"><li><a href="http://dx.doi.org/10.1186/1471-2105-7-s5-s19" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+evaluation+of+human+protein-protein+interaction+data+in+the+public+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+evaluation+of+human+protein-protein+interaction+data+in+the+public+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001347-Tan1" id="pone.0001347-Tan1"></a>Tan FL, Moravec CS, Li J, Apperson-Hansen C, McCarthy PM, et al.  (2002) The gene expression fingerprint of human heart failure. Proc Natl Acad Sci USA.  20;99(17): 11387–92.  <ul class="find" data-citedArticleID="1033144" data-doi="10.1073/pnas.162370099"><li><a href="http://dx.doi.org/10.1073/pnas.162370099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+gene+expression+fingerprint+of+human+heart+failure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+gene+expression+fingerprint+of+human+heart+failure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001347-Kann1" id="pone.0001347-Kann1"></a>Kann MG (2007) Protein interactions and disease: computational approaches to uncover the etiology of diseases. Brief Bioinform.   <ul class="find-nolinks"></ul></li><li><span class="label">7.
              </span><a name="pone.0001347-Wittchen1" id="pone.0001347-Wittchen1"></a>Wittchen F, Suckau L, Witt H, Skurk C, Lassner D, et al.  (2007) Genomic expression profiling of human inflammatory cardiomyopathy (DCMi) suggests novel therapeutic targets. J Mol Med.  85(3): 253–67.  <ul class="find" data-citedArticleID="1033152" data-doi="10.1007/s00109-006-0122-9"><li><a href="http://dx.doi.org/10.1007/s00109-006-0122-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+expression+profiling+of+human+inflammatory+cardiomyopathy+%28DCMi%29+suggests+novel+therapeutic+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+expression+profiling+of+human+inflammatory+cardiomyopathy+%28DCMi%29+suggests+novel+therapeutic+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001347-Oti1" id="pone.0001347-Oti1"></a>Oti M, Snel B, Huynen MA, Brunner HG (2006) Predicting disease genes using protein-protein interactions. J Med Genet.  43: 691–698.  <ul class="find" data-citedArticleID="1033138" data-doi="10.1136/jmg.2006.041376"><li><a href="http://dx.doi.org/10.1136/jmg.2006.041376" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+disease+genes+using+protein-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+disease+genes+using+protein-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001347-Xu1" id="pone.0001347-Xu1"></a>Xu J, Li Y (2006) Discovering disease-genes by topological features in human protein-protein interaction network. Bioinformatics.  22(22): 2800–5.  <ul class="find" data-citedArticleID="1033154" data-doi="10.1093/bioinformatics/btl467"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl467" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovering+disease-genes+by+topological+features+in+human+protein-protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovering+disease-genes+by+topological+features+in+human+protein-protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001347-Lu1" id="pone.0001347-Lu1"></a>Lu X, Jain VV, Finn PW, Perkins DL (2007) Hubs in biological interaction networks exhibit low changes in expression in experimental asthma. Mol Syst Biol.  3: 98.  <ul class="find" data-citedArticleID="1033130" data-doi="10.1038/msb4100138"><li><a href="http://dx.doi.org/10.1038/msb4100138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hubs+in+biological+interaction+networks+exhibit+low+changes+in+expression+in+experimental+asthma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hubs+in+biological+interaction+networks+exhibit+low+changes+in+expression+in+experimental+asthma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001347-Cline1" id="pone.0001347-Cline1"></a>Cline MS, Smoot M, Cerami E, Kuchinsky A, Landys N, et al.  (2007) Integration of biological networks and gene expression data using Cytoscape. Nat Protoc.  2(10): 2366–82.  <ul class="find" data-citedArticleID="1033110" data-doi="10.1038/nprot.2007.324"><li><a href="http://dx.doi.org/10.1038/nprot.2007.324" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integration+of+biological+networks+and+gene+expression+data+using+Cytoscape." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integration+of+biological+networks+and+gene+expression+data+using+Cytoscape.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001347-Peri1" id="pone.0001347-Peri1"></a>Peri S, Navarro JD, Amanchy R, Kristiansen TZ, Jonnalagadda CK, et al.  (2003) Development of human protein reference database as an initial platform for approaching systems biology in humans. Genome Res,  13(10): 2363–71.  <ul class="find" data-citedArticleID="1033140" data-doi="10.1101/gr.1680803"><li><a href="http://dx.doi.org/10.1101/gr.1680803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+human+protein+reference+database+as+an+initial+platform+for+approaching+systems+biology+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+human+protein+reference+database+as+an+initial+platform+for+approaching+systems+biology+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001347-Gene1" id="pone.0001347-Gene1"></a>Gene Expression Omnibus (GEO) (2007)   [<a href="http://www.ncbi.nlm.nih.gov.geo/geo">http://www.ncbi.nlm.nih.gov.geo/geo</a>].  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001347-Tusher1" id="pone.0001347-Tusher1"></a>Tusher VG, Tibshirani R, Chu G (2001) Significance analysis of microarrays applied to the ionizing radiation response. Proc Natl Acad Sci.  98: 5116–5121.  <ul class="find" data-citedArticleID="1033146" data-doi="10.1073/pnas.091062498"><li><a href="http://dx.doi.org/10.1073/pnas.091062498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001347-Gene2" id="pone.0001347-Gene2"></a>Gene Ontology (GO) (2007)   [<a href="http://www.geneontology.org">http://www.geneontology.org</a>].  <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0001347-Kang1" id="pone.0001347-Kang1"></a>Kang PM, Izumo S (2000) Apoptosis and heart failure: A critical review of the literature. Circ Res.  86(11): 1107–13.  <ul class="find" data-citedArticleID="1033122" data-doi="10.1161/01.res.86.11.1107"><li><a href="http://dx.doi.org/10.1161/01.res.86.11.1107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Apoptosis+and+heart+failure%3A+A+critical+review+of+the+literature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Apoptosis+and+heart+failure%3A+A+critical+review+of+the+literature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001347-Watanabe1" id="pone.0001347-Watanabe1"></a>Watanabe T, Otsu K (2004) Signal transduction in heart failure. Nippon Rinsho.  62(12): 2337–42.  <ul class="find" data-citedArticleID="1033150"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Signal+transduction+in+heart+failure.&amp;auth=&amp;atitle=Signal+transduction+in+heart+failure." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signal+transduction+in+heart+failure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signal+transduction+in+heart+failure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001347-Deo1" id="pone.0001347-Deo1"></a>Deo RC, Groft CM, Rajashankar KR, Burley SK (2002) Recognition of the rotavirus mRNA 3′ consensus by an asymmetric NSP3 homodimer. Cell.  11;108(1): 71–81.  <ul class="find" data-citedArticleID="1033112" data-doi="10.1016/s0092-8674(01)00632-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(01)00632-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+the+rotavirus+mRNA+3%E2%80%B2+consensus+by+an+asymmetric+NSP3+homodimer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+the+rotavirus+mRNA+3%E2%80%B2+consensus+by+an+asymmetric+NSP3+homodimer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001347-Niu1" id="pone.0001347-Niu1"></a>Niu J, Azfer A, Kolattukudy PE (2006) Monocyte-specific Bcl-2 expression attenuates inflammation and heart failure in monocyte chemoattractant protein-1 (MCP-1)-induced cardiomyopathy. Cardiovasc Res.  71(1): 139–48.  <ul class="find" data-citedArticleID="1033136" data-doi="10.1016/j.cardiores.2006.03.008"><li><a href="http://dx.doi.org/10.1016/j.cardiores.2006.03.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monocyte-specific+Bcl-2+expression+attenuates+inflammation+and+heart+failure+in+monocyte+chemoattractant+protein-1+%28MCP-1%29-induced+cardiomyopathy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monocyte-specific+Bcl-2+expression+attenuates+inflammation+and+heart+failure+in+monocyte+chemoattractant+protein-1+%28MCP-1%29-induced+cardiomyopathy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001347-Fischer1" id="pone.0001347-Fischer1"></a>Fischer P, Hilfiker-Kleiner D (2007) pathways in hypertrophy and heart failure: the gp130-STAT axis. Basic Res Cardiol.  102(5): 393–411.  <ul class="find" data-citedArticleID="1033116" data-doi="10.1007/s00395-007-0674-z"><li><a href="http://dx.doi.org/10.1007/s00395-007-0674-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=pathways+in+hypertrophy+and+heart+failure%3A+the+gp130-STAT+axis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22pathways+in+hypertrophy+and+heart+failure%3A+the+gp130-STAT+axis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001347-Wang1" id="pone.0001347-Wang1"></a>Wang M, Markel T, Crisostomo P, Herring C, Meldrum KK, et al.  (2007) Deficiency of TNFR1 protects myocardium through SOCS3 and IL-6 but not p38 MAPK or IL-1beta. Am J Physiol Heart Circ Physiol.,  292(4): H1694–9.  <ul class="find" data-citedArticleID="1033148" data-doi="10.1152/ajpheart.01063.2006"><li><a href="http://dx.doi.org/10.1152/ajpheart.01063.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deficiency+of+TNFR1+protects+myocardium+through+SOCS3+and+IL-6+but+not+p38+MAPK+or+IL-1beta." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deficiency+of+TNFR1+protects+myocardium+through+SOCS3+and+IL-6+but+not+p38+MAPK+or+IL-1beta.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001347-Kittleson1" id="pone.0001347-Kittleson1"></a>Kittleson MM, Minhas KM, Irizarry RA, Ye SQ, Edness G, et al.  (2005) Gene expression analysis of ischemic and nonischemic cardiomyopathy: shared and distinct genes in the development of heart failure. Physiol Genomics.,  3: 99–307.  <ul class="find" data-citedArticleID="1033128" data-doi="10.1152/physiolgenomics.00255.2004"><li><a href="http://dx.doi.org/10.1152/physiolgenomics.00255.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+analysis+of+ischemic+and+nonischemic+cardiomyopathy%3A+shared+and+distinct+genes+in+the+development+of+heart+failure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+analysis+of+ischemic+and+nonischemic+cardiomyopathy%3A+shared+and+distinct+genes+in+the+development+of+heart+failure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001347-Entrez1" id="pone.0001347-Entrez1"></a>  Entrez [<a href="http://www.ncbi.nlm.nih.gov/Entrez">www.ncbi.nlm.nih.gov/Entrez</a>].  <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0001347-Shannon1" id="pone.0001347-Shannon1"></a>Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, et al.  (2003) Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res.  13(11): 2498–504.  <ul class="find" data-citedArticleID="1033142" data-doi="10.1101/gr.1239303"><li><a href="http://dx.doi.org/10.1101/gr.1239303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001347-Maere1" id="pone.0001347-Maere1"></a>Maere S, Heymans K, Kuiper M (2005) BiNGO: a Cytoscape plugin to assess overrepresentation of Gene Ontology categories in Biological Networks. Bioinformatics.  21(16): 3448–3449.  <ul class="find" data-citedArticleID="1033132" data-doi="10.1093/bioinformatics/bti551"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti551" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BiNGO%3A+a+Cytoscape+plugin+to+assess+overrepresentation+of+Gene+Ontology+categories+in+Biological+Networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BiNGO%3A+a+Cytoscape+plugin+to+assess+overrepresentation+of+Gene+Ontology+categories+in+Biological+Networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.XML" value="79939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.PDF" value="382919"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g001.PNG_L" value="780779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g001.PNG_M" value="474627"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g001.PNG_S" value="21023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g001.TIF" value="991308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g001.PNG_I" value="206889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t001.PNG_L" value="73627"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t001.PNG_M" value="96648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t001.PNG_S" value="14013"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t001.TIF" value="309384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t001.PNG_I" value="33685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g002.PNG_L" value="64463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g002.PNG_M" value="73035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g002.PNG_S" value="7340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g002.TIF" value="108302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g002.PNG_I" value="23993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g003.PNG_L" value="65569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g003.PNG_M" value="54018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g003.PNG_S" value="6152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g003.TIF" value="126834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.g003.PNG_I" value="19322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t002.PNG_L" value="59320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t002.PNG_M" value="77448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t002.PNG_S" value="11762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t002.TIF" value="248276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t002.PNG_I" value="26559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t003.PNG_L" value="83933"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t003.PNG_M" value="111916"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t003.PNG_S" value="14973"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t003.TIF" value="359042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t003.PNG_I" value="39180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t004.PNG_L" value="41211"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t004.PNG_M" value="50390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t004.PNG_S" value="10336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t004.TIF" value="187788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.t004.PNG_I" value="17231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.e001.PNG" value="7021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.e001.TIF" value="22956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.s001.XLS" value="167936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001347.s002.XLS" value="91136"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001347&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001347" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001347&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001347&volume=&issue=&title=Linking Gene Expression and Functional Network Data in Human Heart Failure&author_name=Anyela%20Camargo%2C%20Francisco%20Azuaje&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001347" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001347" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001347" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001347&amp;t=Linking%20Gene%20Expression%20and%20Functional%20Network%20Data%20in%20Human%20Heart%20Failure" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001347&title=Linking%20Gene%20Expression%20and%20Functional%20Network%20Data%20in%20Human%20Heart%20Failure&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001347&amp;title=Linking%20Gene%20Expression%20and%20Functional%20Network%20Data%20in%20Human%20Heart%20Failure" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001347" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001347&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Linking Gene Expression and Functional Network Data in Human Heart Failure';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001347';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Linking%20Gene%20Expression%20and%20Functional%20Network%20Data%20in%20Human%20Heart%20Failure http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001347" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001347" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="25524"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25524"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+networks%22" title="Search for articles in the subject area:'Genetic networks'"><div class="flagText">Genetic networks</div></a>
              <div data-categoryid="40201" data-articleid="25524"
                   data-categoryname="Genetic networks"
                   class="flagImage" title="Flag 'Genetic networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Heart+failure%22" title="Search for articles in the subject area:'Heart failure'"><div class="flagText">Heart failure</div></a>
              <div data-categoryid="21143" data-articleid="25524"
                   data-categoryname="Heart failure"
                   class="flagImage" title="Flag 'Heart failure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25524"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Network+analysis%22" title="Search for articles in the subject area:'Network analysis'"><div class="flagText">Network analysis</div></a>
              <div data-categoryid="3497" data-articleid="25524"
                   data-categoryname="Network analysis"
                   class="flagImage" title="Flag 'Network analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interaction+networks%22" title="Search for articles in the subject area:'Protein interaction networks'"><div class="flagText">Protein interaction networks</div></a>
              <div data-categoryid="31803" data-articleid="25524"
                   data-categoryname="Protein interaction networks"
                   class="flagImage" title="Flag 'Protein interaction networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Topology%22" title="Search for articles in the subject area:'Topology'"><div class="flagText">Topology</div></a>
              <div data-categoryid="32019" data-articleid="25524"
                   data-categoryname="Topology"
                   class="flagImage" title="Flag 'Topology' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4794'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8008'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2652&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=18663">Interaction connectivity patterns</a><br>Posted by rkopak</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
