

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Leveraging Hierarchical Population Structure in Discrete Association Studies</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000591"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000591" />

    <meta name="citation_title" content="Leveraging Hierarchical Population Structure in Discrete Association Studies"/>
    <meta itemprop="name" content="Leveraging Hierarchical Population Structure in Discrete Association Studies"/>

      <meta name="citation_author" content="Jonathan Carlson"/>
            <meta name="citation_author_institution" content="Machine Learning and Applied Statistics Group, Microsoft Research, Redmond, Washington, United States of America"/>
            <meta name="citation_author_institution" content="Department of Computer Science and Engineering, University of Washington, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="Carl Kadie"/>
            <meta name="citation_author_institution" content="Machine Learning and Applied Statistics Group, Microsoft Research, Redmond, Washington, United States of America"/>
      <meta name="citation_author" content="Simon Mallal"/>
            <meta name="citation_author_institution" content="Center for Clinical Immunology and Biomedical Statistics, Royal Perth Hospital, Perth, Australia"/>
      <meta name="citation_author" content="David Heckerman"/>
            <meta name="citation_author_institution" content="Department of Computer Science and Engineering, University of Washington, Seattle, Washington, United States of America"/>

    <meta name="citation_date" content="2007/7/4"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000591.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e591"/>
    <meta name="citation_issue" content="7"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Phylogenies and the comparative method.; citation_author= Felsenstein; citation_journal_title=American Naturalist; citation_volume=125; citation_number=1; citation_date=1985; " />
      <meta name="citation_reference" content="citation_author=M Ridley; citation_number=2; citation_date=1983; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic Inference of Historical Pathways and Models of Evolutionary Change. Ph.D. thesis,; citation_author=DR Maddison; citation_number=3; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Detecting correlated evolution on phylogenies: a general method for the comparative analysis of discrete characters.; citation_author=M Pagel; citation_journal_title=Proc R Soc Lond B Biol Sci; citation_volume=255; citation_number=4; citation_pages=37-45; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Coevolving protein residues: maximum likelihood identification and relationship to structure.; citation_author=DD Pollock; citation_author=WR Taylor; citation_author=N Goldman; citation_journal_title=J Mol Biol; citation_volume=287; citation_number=5; citation_pages=187-198; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Modeling residue usage in aligned protein sequences via maximum likelihood.; citation_author=WJ Bruno; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=6; citation_pages=1368-1374; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Evolutionarily conserved pathways of energetic connectivity in protein families.; citation_author=SW Lockless; citation_author=R Ranganathan; citation_journal_title=Science; citation_volume=286; citation_number=7; citation_pages=295-299; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Analysis of covariation in an SH3 domain sequence alignment: applications in tertiary contact prediction and the design of compensating hydrophobic core substitutions.; citation_author=SM Larson; citation_author=AA Di Nardo; citation_author=AR Davidson; citation_journal_title=J Mol Biol; citation_volume=303; citation_number=8; citation_pages=433-446; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evolutionarily conserved networks of residues mediate allosteric communication in proteins.; citation_author=GM Suel; citation_author=SW Lockless; citation_author=MA Wall; citation_author=R Ranganathan; citation_journal_title=Nat Struct Biol; citation_volume=10; citation_number=9; citation_pages=59-69; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Separation of phylogenetic and functional associations in biological sequences by using the parametric bootstrap.; citation_author=KR Wollenberg; citation_author=WR Atchley; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=97; citation_number=10; citation_pages=3288-3291; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Networks of coevolving sites in structural and functional domains of serpin proteins.; citation_author=MJ Buck; citation_author=WR Atchley; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=11; citation_pages=1627-1634; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Founder effects in the assessment of HIV polymorphisms and HLA allele associations.; citation_author=T Bhattacharya; citation_author=M Daniels; citation_author=D Heckerman; citation_author=B Foley; citation_author=N Frahm; citation_journal_title=Science; citation_number=12; citation_date=In press; " />
      <meta name="citation_reference" content="citation_title=Genetic dissection of complex traits.; citation_author=ES Lander; citation_author=NJ Schork; citation_journal_title=Science; citation_volume=265; citation_number=13; citation_pages=2037-2048; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Searching for genetic determinants in the new millennium.; citation_author=NJ Risch; citation_journal_title=Nature; citation_volume=405; citation_number=14; citation_pages=847-856; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genome-wide association studies for common diseases and complex traits.; citation_author=JN Hirschhorn; citation_author=MJ Daly; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=15; citation_pages=95-108; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide association: a promising start to a long race.; citation_author=DM Evans; citation_author=LR Cardon; citation_journal_title=Trends Genet; citation_volume=22; citation_number=16; citation_pages=350-354; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The effects of human population structure on large genetic association studies.; citation_author=J Marchini; citation_author=LR Cardon; citation_author=MS Phillips; citation_author=P Donnelly; citation_journal_title=Nat Genet; citation_volume=36; citation_number=17; citation_pages=512-517; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide strategies for detecting multiple loci that influence complex diseases.; citation_author=J Marchini; citation_author=P Donnelly; citation_author=LR Cardon; citation_journal_title=Nat Genet; citation_volume=37; citation_number=18; citation_pages=413-417; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evidence for allelic association on chromosome 3q25-27 in families with autism spectrum disorders originating from a subisolate of Finland.; citation_author=M Auranen; citation_author=T Varilo; citation_author=R Alen; citation_author=R Vanhala; citation_author=K Ayers; citation_journal_title=Mol Psychiatry; citation_volume=8; citation_number=19; citation_pages=879-884; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=An Icelandic example of the impact of population structure on association studies.; citation_author=A Helgason; citation_author=B Yngvadottir; citation_author=B Hrafnkelsson; citation_author=J Gulcher; citation_author=K Stefansson; citation_journal_title=Nat Genet; citation_volume=37; citation_number=20; citation_pages=90-95; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Confounding from cryptic relatedness in case-control association studies.; citation_author=BF Voight; citation_author=JK Pritchard; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=21; citation_pages=e32; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Demonstrating stratification in a European American population.; citation_author=CD Campbell; citation_author=EL Ogburn; citation_author=KL Lunetta; citation_author=HN Lyon; citation_author=ML Freedman; citation_journal_title=Nat Genet; citation_volume=37; citation_number=22; citation_pages=868-872; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide association mapping in Arabidopsis identifies previously known flowering time and pathogen resistance genes.; citation_author=MJ Aranzana; citation_author=S Kim; citation_author=K Zhao; citation_author=E Bakker; citation_author=M Horton; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=23; citation_pages=e60; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genomic control for association studies.; citation_author=B Devlin; citation_author=K Roeder; citation_journal_title=Biometrics; citation_volume=55; citation_number=24; citation_pages=997-1004; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genomic control, a new approach to genetic-based association studies.; citation_author=B Devlin; citation_author=K Roeder; citation_author=L Wasserman; citation_journal_title=Theor Popul Biol; citation_volume=60; citation_number=25; citation_pages=155-166; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Association mapping in structured populations.; citation_author=JK Pritchard; citation_author=M Stephens; citation_author=NA Rosenberg; citation_author=P Donnelly; citation_journal_title=Am J Hum Genet; citation_volume=67; citation_number=26; citation_pages=170-181; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Dwarf8 polymorphisms associate with variation in flowering time.; citation_author=JM Thornsberry; citation_author=MM Goodman; citation_author=J Doebley; citation_author=S Kresovich; citation_author=D Nielsen; citation_journal_title=Nat Genet; citation_volume=28; citation_number=27; citation_pages=286-289; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Accounting for unmeasured population substructure in case-control studies of genetic association using a novel latent-class model.; citation_author=GA Satten; citation_author=WD Flanders; citation_author=Q Yang; citation_journal_title=Am J Hum Genet; citation_volume=68; citation_number=28; citation_pages=466-477; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Logistic regression protects against population structure in genetic association studies.; citation_author=E Setakis; citation_author=H Stirnadel; citation_author=DJ Balding; citation_journal_title=Genome Res; citation_volume=16; citation_number=29; citation_pages=290-296; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Principal components analysis corrects for stratification in genome-wide association studies.; citation_author=AL Price; citation_author=NJ Patterson; citation_author=RM Plenge; citation_author=ME Weinblatt; citation_author=NA Shadick; citation_journal_title=Nat Genet; citation_volume=38; citation_number=30; citation_pages=904-909; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Estimation of effects of single genes on quantitative traits.; citation_author=BW Kennedy; citation_author=M Quinton; citation_author=JA van Arendonk; citation_journal_title=J Anim Sci; citation_volume=70; citation_number=31; citation_pages=2000-2012; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.; citation_author=J Yu; citation_author=G Pressoir; citation_author=WH Briggs; citation_author=I Vroh Bi; citation_author=M Yamasaki; citation_journal_title=Nat Genet; citation_volume=38; citation_number=32; citation_pages=203-208; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence of HIV-1 adaptation to HLA-restricted immune responses at a population level.; citation_author=CB Moore; citation_author=M John; citation_author=IR James; citation_author=FT Christiansen; citation_author=CS Witt; citation_journal_title=Science; citation_volume=296; citation_number=33; citation_pages=1439-1443; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A tutorial on learning with Bayesian networks.; citation_author=D Heckerman; citation_number=34; citation_pages=301-354; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Evolutionary trees from DNA sequences: a maximum likelihood approach.; citation_author=J Felsenstein; citation_journal_title=J Mol Evol; citation_volume=17; citation_number=35; citation_pages=368-376; citation_date=1981; " />
      <meta name="citation_reference" content="citation_author=W Press; citation_author=S Teukolsky; citation_author=W Vetterling; citation_author=B Flannery; citation_number=36; citation_date=1992; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=A random effects model for binary data.; citation_author=MR Conaway; citation_journal_title=Biometrics; citation_volume=46; citation_number=37; citation_pages=317-328; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Antigen processing by the proteasome.; citation_author=P Kloetzel; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=2; citation_number=38; citation_pages=179-187; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Influence of conservation on calculations of amino acid covariance in multiple sequence alignments.; citation_author=AA Fodor; citation_author=RW Aldrich; citation_journal_title=Proteins; citation_volume=56; citation_number=39; citation_pages=211-221; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Solution structure of the human immunodeficiency virus type 1 p6 protein.; citation_author=T Fossen; citation_author=V Wray; citation_author=K Bruns; citation_author=J Rachmat; citation_author=P Henklein; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=40; citation_pages=42515-42527; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Plant pathogens and integrated defence responses to infection.; citation_author=JL Dangl; citation_author=JD Jones; citation_journal_title=Nature; citation_volume=411; citation_number=41; citation_pages=826-833; citation_date=2001; " />
      <meta name="citation_reference" content="citation_author=PH Harvey; citation_author=MO Pagel; citation_number=42; citation_date=1991; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_author=J Felsenstein; citation_number=43; citation_date=2004; citation_publisher=Sinauer Associates, Inc; " />
      <meta name="citation_reference" content="citation_title=Covariation of mutations in the V3 loop of human immunodeficiency virus type 1 envelope protein: an information theoretic analysis.; citation_author=BT Korber; citation_author=RM Farber; citation_author=DH Wolpert; citation_author=AS Lapedes; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=90; citation_number=44; citation_pages=7176-7180; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Correlated mutations and residue contacts in proteins.; citation_author=U Gobel; citation_author=C Sander; citation_author=R Schneider; citation_author=A Valencia; citation_journal_title=Proteins; citation_volume=18; citation_number=45; citation_pages=309-317; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Compensating changes in protein multiple sequence alignments.; citation_author=WR Taylor; citation_author=K Hatrick; citation_journal_title=Protein Eng; citation_volume=7; citation_number=46; citation_pages=341-348; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=How frequent are correlated changes in families of protein sequences?; citation_author=E Neher; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=91; citation_number=47; citation_pages=98-102; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Effectiveness of correlation analysis in identifying protein residues undergoing correlated evolution.; citation_author=DD Pollock; citation_author=WR Taylor; citation_journal_title=Protein Eng; citation_volume=10; citation_number=48; citation_pages=647-657; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Detection and reduction of evolutionary noise in correlated mutation analysis.; citation_author=O Noivirt; citation_author=M Eisenstein; citation_author=A Horovitz; citation_journal_title=Protein Eng Des Sel; citation_volume=18; citation_number=49; citation_pages=247-253; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dominant influence of HLA-B in mediating the potential co-evolution of HIV and HLA.; citation_author=P Kiepiela; citation_author=AJ Leslie; citation_author=I Honeyborne; citation_author=D Ramduth; citation_author=C Thobakgale; citation_journal_title=Nature; citation_volume=432; citation_number=50; citation_pages=769-775; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Syst Biol; citation_volume=52; citation_number=51; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=PHYLIP (Phylogeny Inference Package) version 3.6.; citation_author=J Felsenstein; citation_number=52; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.; citation_author=Y Benjamini; citation_author=Y Hochberg; citation_journal_title=J R Stat Soc B; citation_volume=57; citation_number=53; citation_pages=289-300; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Statistical significance for genomewide studies.; citation_author=JD Storey; citation_author=R Tibshirani; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=100; citation_number=54; citation_pages=9440-9445; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The extent of linkage disequilibrium in Arabidopsis thaliana.; citation_author=M Nordborg; citation_author=JO Borevitz; citation_author=J Bergelson; citation_author=CC Berry; citation_author=J Chory; citation_journal_title=Nat Genet; citation_volume=30; citation_number=55; citation_pages=190-193; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000591" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Leveraging Hierarchical Population Structure in Discrete Association Studies"/>
    <meta name="twitter:description" content="
Population structure can confound the identification of correlations in biological data. Such confounding has been recognized in multiple biological disciplines, resulting in a disparate collection of proposed solutions. We examine several methods that correct for confounding on discrete data with hierarchical population structure and identify two distinct confounding processes, which we call coevolution and conditional influence. We describe these processes in terms of generative models and show that these generative models can be used to correct for the confounding effects. Finally, we apply the models to three applications: identification of escape mutations in HIV-1 in response to specific HLA-mediated immune pressure, prediction of coevolving residues in an HIV-1 peptide, and a search for genotypes that are associated with bacterial resistance traits in Arabidopsis thaliana. We show that coevolution is a better description of confounding in some applications and conditional influence is better in others. That is, we show that no single method is best for addressing all forms of confounding. Analysis tools based on these models are available on the internet as both web based applications and downloadable source code at http://atom.research.microsoft.com/bio/phylod.aspx.
"/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000591.g011"/>

  <meta property="og:title" content="Leveraging Hierarchical Population Structure in Discrete Association Studies" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000591" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2682'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2627'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4861&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000591">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000591" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000591&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Leveraging Hierarchical Population Structure in Discrete Association Studies
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jonathan Carlson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Machine Learning and Applied Statistics Group, Microsoft Research, Redmond, Washington, United States of America, 
                  Department of Computer Science and Engineering, University of Washington, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carl Kadie, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Machine Learning and Applied Statistics Group, Microsoft Research, Redmond, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Simon Mallal, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Clinical Immunology and Biomedical Statistics, Royal Perth Hospital, Perth, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Heckerman
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:heckerma@microsoft.com">heckerma@microsoft.com</a></p>

                <p>Affiliation:
                  Department of Computer Science and Engineering, University of Washington, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: July 04, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000591</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000591" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000591" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000591" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000591" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000591-g001" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000591-g002" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000591-g003" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000591-g004" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000591-g005" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000591-g006" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000591-g007" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000591-g008" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000591-g009" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000591-t001" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000591-g010" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g010" title="Figure 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000591-g011" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g011" title="Figure 11">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g011&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000591">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000591" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Population structure can confound the identification of correlations in biological data. Such confounding has been recognized in multiple biological disciplines, resulting in a disparate collection of proposed solutions. We examine several methods that correct for confounding on discrete data with hierarchical population structure and identify two distinct confounding processes, which we call coevolution and conditional influence. We describe these processes in terms of generative models and show that these generative models can be used to correct for the confounding effects. Finally, we apply the models to three applications: identification of escape mutations in HIV-1 in response to specific HLA-mediated immune pressure, prediction of coevolving residues in an HIV-1 peptide, and a search for genotypes that are associated with bacterial resistance traits in <em>Arabidopsis thaliana</em>. We show that coevolution is a better description of confounding in some applications and conditional influence is better in others. That is, we show that no single method is best for addressing all forms of confounding. Analysis tools based on these models are available on the internet as both web based applications and downloadable source code at <a href="http://atom.research.microsoft.com/bio/phylod.aspx">http://atom.research.microsoft.com/bio/p​hylod.aspx</a>.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Carlson J, Kadie C, Mallal S, Heckerman D (2007) Leveraging Hierarchical Population Structure in Discrete Association Studies. PLoS ONE 2(7):
          e591.
            doi:10.1371/journal.pone.0000591</p><p><strong>Academic Editor: </strong>Philip Awadalla, North Carolina State University, United States of America</p><p><strong>Received:</strong> January 5, 2007; <strong>Accepted:</strong> June 8, 2007; <strong>Published:</strong> July 4, 2007</p><p><strong>Copyright:</strong> © 2007 Carlson et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was funded by Microsoft Research. Microsoft Research had no role in any other aspect of this work.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>There is now clear recognition across several application areas that population structure can confound the statistical identification of associations. An area where this problem was recognized early is the identification of coevolving traits. Felsenstein described the problem and proposed a solution for quantitative traits <a href="#pone.0000591-Felsenstein1">[1]</a>, while Ridley <a href="#pone.0000591-Ridley1">[2]</a> and Maddison <a href="#pone.0000591-Maddison1">[3]</a> described early solutions for discrete traits. A specific application area involving discrete traits is the identification of coevolving amino acids. Numerous researchers recognized the problem and proposed solutions <a href="#pone.0000591-Pagel1">[4]</a>–<a href="#pone.0000591-Buck1">[11]</a>. Yet another area of application that recognized the problem is the identification of Human Leukocyte Antigen (HLA) alleles that mediate mutation of Human Immunodeficiency Virus type 1 (HIV-1) to escape T-cell epitopes. Recently, Bhattacharya et al. <a href="#pone.0000591-Bhattacharya1">[12]</a> demonstrated the importance of correcting for population structure in this domain. In all these application areas, the population structure arises due to the phylogenetic relationships among the species or individuals being studied.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Another important application area where population structure can confound the identification of associations is genome-wide association (GWA) studies <a href="#pone.0000591-Lander1">[13]</a>, <a href="#pone.0000591-Risch1">[14]</a>. Although a number of candidate disease genes have been identified in this manner, there has been a lack of reproducibility due in part to confounding by common descent <a href="#pone.0000591-Hirschhorn1">[15]</a>, <a href="#pone.0000591-Evans1">[16]</a>. In these studies, the organisms studied typically reproduce sexually, and hence population structure is not due to phylogenetic relationships. Nonetheless, there is structure due to non-random mating <a href="#pone.0000591-Marchini1">[17]</a>, <a href="#pone.0000591-Marchini2">[18]</a>, and such structure may be hierarchical <a href="#pone.0000591-Auranen1">[19]</a>–<a href="#pone.0000591-Campbell1">[22]</a> especially in self-mating organisms such as <em>Arabidopsis thaliana</em> <a href="#pone.0000591-Aranzana1">[23]</a>. Numerous solutions have been proposed including genomic control <a href="#pone.0000591-Devlin1">[24]</a>, <a href="#pone.0000591-Devlin2">[25]</a>, structured association <a href="#pone.0000591-Pritchard1">[26]</a>, and other methods <a href="#pone.0000591-Thornsberry1">[27]</a>–<a href="#pone.0000591-Price1">[30]</a>. Several solutions address confounding due to hierarchical structure <a href="#pone.0000591-Aranzana1">[23]</a>, <a href="#pone.0000591-Kennedy1">[31]</a>, <a href="#pone.0000591-Yu1">[32]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this paper, we examine in detail several methods that correct for confounding on discrete data with hierarchical population structure. To introduce these methods, let us consider how confounding may arise when the population structure is due to phylogenetic relationships. In particular, consider the problem of identifying HLA-mediated immune pressure on HIV-1 intrahost adaptation. Moore <em>et al.</em> <a href="#pone.0000591-Moore1">[33]</a> have suggested that the immune pressure from the cellular arm of the immune system—in particular, CD8+ T cells interacting with MHC-I epitopes presented on the surface of infected cells—can exert substantial selection pressure on HIV. If a particular HLA allele exerts pressure on a certain amino acid in the HIV-1 genome, then we would expect to see a correlation between whether an HIV sequence has that amino acid and whether the patient infected with that sequence has the HLA allele. To determine whether these two variables are associated, we could count the number of patients with and without the amino acid and with and without the allele and apply a simple statistical test such as Fisher's exact test. This procedure, however, ignores the phylogenetic structure among the sequences.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Suppose these sequences have the phylogeny shown in <a href="#pone-0000591-g001">Figure 1a</a>. In essence, there are two clusters of sequences where sequences within a cluster are similar to each other but quite different from those in the other cluster. Now suppose we observe that HLA allele <em>X</em> and amino acid <em>Y</em> are present in two sequences on the upper cluster and absent in the two sequences on the lower cluster, as shown in <a href="#pone-0000591-g001">Figure 1a</a>. The observations of the amino acid are well explained by the phylogeny alone and should not be treated as independent observations. Consequently, the application of Fisher's exact test or some other test that ignores the phylogenetic structure would overcount these observations when determining the correlation of <em>X</em> and <em>Y</em>. Such overcounting could produce false positives. In contrast, suppose our findings are as shown in <a href="#pone-0000591-g001">Figure 1b</a>. Here, observing the presence and absence of the amino acid in the same branch of the phylogeny is quite surprising, until the observations of <em>Y</em> are taken into account. In this case, the application of a simple test would undercount the observations when determining the correlation of <em>X</em> and <em>Y</em> and potentially produce false negatives—that is, phylogenetic structure helps us to leverage the data to determine more associations than we would have otherwise.</p>
<div class="figure" id="pone-0000591-g001"><div class="img"><a name="pone-0000591-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Examples illustrating the (a) overcounting and (b) undercounting of evidence for an association between <em>X</em> and <em>Y</em>.</span></strong></p><span>doi:10.1371/journal.pone.0000591.g001</span></div><a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Simple statistical methods such as Fisher's exact test assume the data to be infinitely exchangeable or independent and identically distributed (IID). Although sequence data and other biological data are IID a priori, they are not IID once we learn their hierarchical structure. Furthermore, as we have just seen, this structure can easily confound the statistical search for associations within such data.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>An important point that has not been emphasized in previous work is that different applications may involve different evolutionary processes leading to different kinds of confounding and requiring different solutions. For example, in one process, <em>X</em> and <em>Y</em> coevolve according to the phylogenetic tree—any change in <em>Y</em> during evolution influences the evolution of <em>X</em>, and vice versa. Here, the phylogenetic tree serves as a confounder of <em>X</em> and <em>Y</em> in the traditional sense—the tree is a hidden common cause of both <em>X</em> and <em>Y</em>, leading to spurious correlations between <em>X</em> and <em>Y</em> when the tree is ignored. In another process, only <em>Y</em> evolves according to the tree and the influence of <em>X</em> on <em>Y</em> occurs only at the tips of the tree. <em>X</em> need not evolve according to the tree or “follow the tree,” but instead can have any distribution, including one in which the observations of <em>X</em> are IID. We refer to these two processes as <em>coevolution</em> and <em>conditional influence</em>, respectively.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Both processes lead to confounding, as evidenced by the example in <a href="#pone-0000591-g001">Figure 1</a>, which is agnostic about whether <em>X</em> follows the tree. Each process, however, leads to a particular form of confounding that, as we will show, requires a specific solution. In this paper, we use a particular statistical model class known as a <em>generative</em> model both to describe these processes and to address each form of confounding that they entail. We describe the <em>undirected joint model</em> of Pollock, Taylor and Goldman <a href="#pone.0000591-Pollock1">[5]</a>, which represents the traditional confounding process where both <em>X</em> and <em>Y</em> follow the tree, and our own <em>conditional model</em> introduced in Bhattacharya <em>et al.</em> <a href="#pone.0000591-Bhattacharya1">[12]</a>, which represents the confounding process where only <em>Y</em> follows the tree. We show how these models can be used to identify correlations among data when either form of confounding is present. The basic idea behind the approach is to determine the degree to which a generative model that includes a correlation between <em>X</em> and <em>Y</em> explains the data better than a model that uses only the phylogeny to explain the data.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>Using synthetic data, we show that the coevolution model better addresses the confounding of the coevolution process, and the conditional model better addresses the confounding of the conditional process. In addition, we apply these two models to real examples, including the identification of escape mutations in HIV-1 in response to specific HLA-mediated immune pressure and the prediction of coevolving residues in an HIV-1 peptide, and find that no one model is best for all applications.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>So far, we have considered only phylogeny as a source of hierarchical population structure. Nonetheless, we also will explore the use of these generative models, which incorporate evolutionary processes, to address hierarchical population structure in its more general case. In particular, we apply these models to a genomic search for genotypes that are associated with bacterial resistance traits in <em>Arabidopsis</em> and show that they are effective.</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>We have implemented methods for fitting these models in a package called PhyloD, which is available on the internet as both a web based application and downloadable source at <a href="http://atom.research.microsoft.com/bio/phylod.aspx">http://atom.research.microsoft.com/bio/p​hylod.aspx</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Overview of models</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The models that we describe are generative models, also known as directed acyclic graphical models <a href="#pone.0000591-Heckerman1">[34]</a>. A generative model consists of (1) a <em>structure</em>, a directed acyclic graph where nodes correspond to variables and missing arcs specify probabilistic independencies among the variables, and (2) a set of conditional probability distributions, one distribution for each node. The probabilistic independences specified by the structure of the graph allow the joint distribution of the data to be written as the product over the nodes of their conditional distributions. Given the structure of a generative model, we can use observations for some (and not necessarily all) nodes to infer the distributions (or parameters of the distributions) of the model <a href="#pone.0000591-Heckerman1">[34]</a>. One common criterion for inference, which we use here, is to identify the distributions that maximize the likelihood of the data.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To represent the hierarchical population structure, our generative models use the same machinery as that found in the maximum-likelihood phylogenetic tree <a href="#pone.0000591-Felsenstein2">[35]</a>. As we have discussed, however, such hierarchical structure may not arise due to phylogenetic relationships. Also, we note that the trees are built from sequence data alone (see <a href="#s4">Methods</a> and Discussion).</p>
<h5>Single-variable model.</h5><a id="article1.body1.sec2.sec1.sec1.p1" name="article1.body1.sec2.sec1.sec1.p1"></a><p>Before we consider models of associations between variables, let us consider the situation where a <em>single</em> variable <em>Y</em> with possible states present (<em>y</em>) and absent (<em>y̅</em>) is generated by a given phylogenetic tree. The structure of the generative model, which is simply the series of branches in the phylogenetic tree, is shown in <a href="#pone-0000591-g002">Figure 2a</a>. Nodes at the tips of tree, labeled <em>Y</em><sub>1</sub>, … , <em>Y<sub>N</sub></em> correspond to the observations of <em>Y</em>. Unlabeled nodes in the interior of the tree correspond to events of divergence. (Although each <em>Y<sub>i</sub></em> and interior node corresponds to a variable in the ordinary statistical sense, we sometimes refer to the collection of variables—<em>Y</em> in this case—as a variable. The proper meaning of the word should be clear from context.) As in the probabilistic model of Felsenstein <a href="#pone.0000591-Felsenstein2">[35]</a> for a phylogenetic tree, we assume that the transition from state <em>i</em> to state <em>j</em> along a transition with branch length <em>t</em> will occur with probability <strong>P</strong><em><sub>ij</sub></em>(<em>t</em>) that is determined by a continuous time Markov process (CTMP). In our single binary variable case, the CTMP is parameterized by a single rate parameter λ, and the stationary probability π, the probability of <em>Y = y</em> reached by the Markov process after an infinite amount of time. The instantaneous rate of change from <em>y̅</em> to <em>y</em> is λ π, yielding a transition probability matrix whose values as a function of time <em>t</em> are given by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>Where π<em><sub>j</sub></em> = π when <em>j</em> corresponds to the presence of <em>Y</em>, and π<em><sub>j</sub></em> = 1−π when <em>j</em> corresponds to the absence of <em>Y</em>.</p>
<div class="figure" id="pone-0000591-g002"><div class="img"><a name="pone-0000591-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Two generative (graphical) models.</span></strong></p><a id="article1.body1.sec2.sec1.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.sec1.fig1.caption1.p1"></a><p>(a) The single-variable model for <em>Y</em>. (b) The conditional model for <em>Y</em> given <em>X</em>. The variable <em>Z<sub>i</sub></em> represents the variable <em>Y<sub>i</sub></em> had there been no influence from <em>X<sub>i</sub></em>. Observed variables are shaded. Conditional probability distributions are not shown.</p>
<span>doi:10.1371/journal.pone.0000591.g002</span></div><a id="article1.body1.sec2.sec1.sec1.p2" name="article1.body1.sec2.sec1.sec1.p2"></a><p>As mentioned, given a set of observations for <em>Y</em><sub>1</sub>, … , <em>Y<sub>N</sub></em> at the tips of a tree, we choose the parameters π and λ to be those that maximize the likelihood of the data. Given values for these parameters, the tree structure ψ (the series of branches and their lengths), and the observed data <em>D<sub>Y</sub></em>, the likelihood <em>L</em>(<em>D<sub>Y</sub></em>|π, λ, Ψ) can be easily computed <a href="#pone.0000591-Felsenstein2">[35]</a>. There are a number of methods for identifying the maximum-likelihood parameters, including gradient decent and the Expectation-Maximization (EM) algorithm. For this model and the others that we describe, we use coordinate descent, iteratively maximizing the likelihood for each parameter individually using Brent's method <a href="#pone.0000591-Press1">[36]</a>.</p>
<a id="article1.body1.sec2.sec1.sec1.p3" name="article1.body1.sec2.sec1.sec1.p3"></a><p>This generative model is reversible, since π<em><sub>i</sub></em><strong>P</strong><em><sub>ij</sub></em>(<em>t</em>) = π<em><sub>j</sub></em><strong>P</strong><em><sub>ji</sub></em>(<em>t</em>), allowing us to work with an unrooted (or arbitrarily rooted) tree. Also, this model includes the situation where the observations of <em>Y</em><sub>1</sub>, … , <em>Y<sub>N</sub></em> are IID as a special case (i.e., the limit as λ tends to infinity.)</p>

<h5>Undirected joint model.</h5><a id="article1.body1.sec2.sec1.sec2.p1" name="article1.body1.sec2.sec1.sec2.p1"></a><p>Now let us consider two binary variables <em>X</em> and <em>Y</em> that evolve according to a given phylogenetic tree. Our first association model for <em>X</em> and <em>Y</em>, called the <em>undirected joint model</em>, assumes that <em>X</em> and <em>Y</em> coevolve according to the tree such that, throughout the course of evolution, a change in either variable can influence the evolution of the other. This generative model can be thought of as a single-variable model for a variable <em>XY</em> representing the cross product of <em>X</em> and <em>Y</em> with the four states: <em>xy</em>, <em>xy̅</em>, <em>x̅</em><em>y</em>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.e002&amp;representation=PNG" class="inline-graphic"></span>. The undirected joint model has five parameters: two rate parameters λ<em><sub>X</sub></em> and λ<em><sub>Y</sub></em>, and three parameters π<em><sub>xy</sub></em>, π<em><sub>xy̅</sub></em>, π<em><sub>x̅</sub></em><sub><em>y</em></sub>, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.e003&amp;representation=PNG" class="inline-graphic"></span> representing the stationary distribution of <em>XY</em>. The instantaneous rate matrix <em>Q</em> is given by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>where the diagonal entries assure that the rows sum to 0. Given a particular branch with length <em>t</em>, we can compute the probability that one instance of <em>XY</em> transitions to another by determining <strong>P</strong> = exp[<strong>Q</strong><em>t</em>], which can be computed using standard numerical Eigen-decomposition techniques <a href="#pone.0000591-Press1">[36]</a>.</p>
<a id="article1.body1.sec2.sec1.sec2.p2" name="article1.body1.sec2.sec1.sec2.p2"></a><p>This model, first presented by Pollock and colleagues <a href="#pone.0000591-Pollock1">[5]</a>, treats <em>X</em> and <em>Y</em> symmetrically so that the influence of <em>X</em> on <em>Y</em> and <em>Y</em> on <em>X</em> is “undirected,” hence our name for the model. In <a href="#s4">Methods</a>, we describe an alternative model for coevolution in which the relationship between the two variables is asymmetric or directed.</p>
<a id="article1.body1.sec2.sec1.sec2.p3" name="article1.body1.sec2.sec1.sec2.p3"></a><p>To determine the significance or strength of the correlation between <em>X</em> and <em>Y</em>, we determine how much better this model explains the data than a null model in which each variable independently follows the single-variable model described in the previous section. We use both frequentist and Bayesian methods to determine the degree to which the non-null model better explains the data. In the frequentist case, we compute a <em>p</em>-value using a likelihood ratio test, which is justified as the null model is nested within the non-null model. We control for multiple tests using <em>q</em>-values, which are estimates of one minus positive predictive value (see <a href="#s4">Methods</a>). In the Bayesian case, we use the Bayesian Information Criterion (BIC) to select the alternative model over the null (see section on model evaluation and comparison).</p>

<h5>Conditional model.</h5><a id="article1.body1.sec2.sec1.sec3.p1" name="article1.body1.sec2.sec1.sec3.p1"></a><p>In some situations, it may be reasonable to assume that <em>Y</em> follows a tree, but that <em>X</em> does not follow the same tree (or any tree at all), and that the influence of <em>X</em> on <em>Y</em> occurs only at the tips of the tree. In this situation, we sometimes refer to <em>Y</em> as the target variable and <em>X</em> as the predictor variable. This situation may arise if, for example, <em>Y</em> represents the amino acid at a particular position in a sequence and <em>X</em> represents an environmental variable that we suspect may be a source of selective pressure on <em>X</em>, as in the case where we are looking for HLA alleles that exert pressure on specific HIV-1 molecules to select for escape mutations. Whereas the amino acid likely follows the phylogenetic tree of the protein it is in, it is unreasonable to expect the HLA allele to follow that same tree (except perhaps at the clade level, where HLA alleles may be distributed similarly to the clades of the HIV species tree due to similar geographic constraints). Furthermore, although we expect that HLA pressure acts continually throughout evolution, this pressure could be distributed randomly along the evolutionary paths, and consequently the influence of the known HLA of the patient would only appear at a tip of the tree.</p>
<a id="article1.body1.sec2.sec1.sec3.p2" name="article1.body1.sec2.sec1.sec3.p2"></a><p>We present a model that captures this notion in Bhattacharya <em>et al.</em> <a href="#pone.0000591-Bhattacharya1">[12]</a>. The model is a single-variable model for the target variable <em>Y</em>, modified to include the influence of <em>X</em> at the tips of the tree. Consider the single-variable model for <em>Y</em> as shown in <a href="#pone-0000591-g002">Figure 2a</a>. To construct the conditional model for <em>Y</em> given <em>X</em>, shown in <a href="#pone-0000591-g002">Figure 2b</a>, we first change each <em>Y<sub>i</sub></em> to <em>Z<sub>i</sub></em>, which represents what <em>Y<sub>i</sub></em> would have been had there been no influence from <em>X<sub>i</sub></em>. We then make each <em>Y<sub>i</sub></em> a probabilistic function of <em>X<sub>i</sub></em> and <em>Z<sub>i</sub></em>, encoding this function with a probability matrix. There are four possible single-parameter transition matrices:</p>


<ol class="order">

<li><strong>Escape</strong> <em>X</em> = <em>x</em> increases the probability of <em>y</em>→<em>y̅</em> transitions,</li>

<li><strong>Attraction</strong> <em>X</em> = <em>x</em> increases the probability of <em>y̅</em>→<em>y</em> transitions,</li>

<li><strong>Reversion</strong> <em>X</em> = <em>x̅</em> increases the probability of <em>y̅</em>→<em>y</em> transitions,</li>

<li><strong>Repulsion</strong> <em>X</em> = <em>x̅</em> increases the probability of <em>y</em>→<em>y̅</em> transitions.</li>

</ol><a id="article1.body1.sec2.sec1.sec3.p3" name="article1.body1.sec2.sec1.sec3.p3"></a><p>As in the undirected joint case, we can use both frequentist (likelihood ratio test) and Bayesian (BIC) methods to determine the degree to which each of these models better explains the data for <em>Y</em> than does the single-variable model for <em>Y</em>. Note that we do not need to consider the explanation of the data for <em>X</em>, as the model is a conditional one.</p>
<a id="article1.body1.sec2.sec1.sec3.p4" name="article1.body1.sec2.sec1.sec3.p4"></a><p>Although it is possible that, for example, both escape and reversion processes are acting at the same time, we have found that allowing for two or more processes at the same time dramatically reduces the power of these models. Consequently, in our experimental evaluations, for any given pair of variables, we choose one model from the set above that best explains the data for those variables. We can use both frequentist and Bayesian methods for making this choice. In the frequentist case, we choose the model with the lowest <em>q</em>-value. In the Bayesian case, we choose the model with the largest BIC score.</p>
<a id="article1.body1.sec2.sec1.sec3.p5" name="article1.body1.sec2.sec1.sec3.p5"></a><p>The conditional model is reversible in the sense that the choice of root node among non-tip nodes does not affect the likelihood of the data. We also note that this model is a (discrete) mixed-effects model, wherein the predictor variables <em>X<sub>i</sub></em> correspond to the fixed effects and the hidden variables <em>Z<sub>i</sub></em> correspond to the random effects <a href="#pone.0000591-Conaway1">[37]</a>.</p>



<h4>Model evaluation and comparison</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We evaluate and compare our models on both synthetic and real data sets. On synthetic data, we examine two questions. One, does taking hierarchical population structure into account improve the analysis? For example, when we generate data from an undirected joint or conditional model, do these models perform better than Fisher's exact test (FET), which assumes the data to be IID? Two, when we generate data from an undirected joint model, does that model better fit the data than the conditional model, and vice-versa?</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We use two criteria to measure the performance of a model. First, we measure the ability of the model to discriminate true from false correlations via plots of true positive rate versus one minus positive predictive value. In particular, we compute a <em>q</em>-value for every pair <em>X–Y</em> that we test, and deem every test with a <em>q</em>-value less than some threshold <em>q</em><sub>0</sub> to be a real association. We then use the known true and false associations in the synthetic data to compute true positive rate and one minus positive predictive value for that threshold. Finally, we allow <em>q</em><sub>0</sub> to vary, producing a <em>discrimination curve</em>. We compute the significance of the difference between two curves by comparing the difference in the areas under the curves against a null model estimated by a permutation test (see <a href="#s4">Methods</a>). Second, we measure the <em>calibration</em> of the model—that is, the degree to which the <em>q</em>-value is a good estimate of one minus the positive predictive value. Note that the discrimination curves are independent of calibration, as any monotonic transformation of the threshold used to compute a curve (<em>q</em>-values in our case) leaves the curve unchanged.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>On real data, we examine whether the undirected joint model or conditional model better represents the data. Because we don't know whether a discovered association is real or not, we cannot use the discrimination curve or calibration criterion to evaluate performance. Furthermore, because neither model is nested within the other, we turn to Bayesian methods—in particular, the Bayesian Information Criterion (BIC)—for comparison. The BIC score for a model with maximum-likelihood parameters θ̂ and <em>d</em> free parameters is given by <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.e005&amp;representation=PNG" class="inline-graphic"></span>. The BIC is an asymptotic approximation to the marginal log likelihood of a model, a Bayesian measure of how well the model represents the data.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>To compare the conditional model, a model for <em>Y</em> given <em>X</em>, with the joint model, a model for <em>X</em> and <em>Y</em>, we compare the sum of the overall BIC scores for the conditional model and the single-variable model for <em>X</em> with the overall BIC score for the joint model. The overall BIC score for the conditional model is the highest BIC score among the conditional models (escape, attraction, reversion, and repulsion) and the independence model. The overall BIC score for the undirected joint model is the higher of the BIC scores for the undirected joint model and the independence model. To test whether there is a significant difference between the overall scores for the conditional and undirected joint models, we apply a two-sided, paired Wilcoxon test to these scores across all <em>X–Y</em> pairs in the data.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>In our applications, we sometimes find it useful to test whether a variable <em>Y follows a given tree</em>. To do so, we calculate the difference in BIC scores for the ordinary single-variable model for <em>Y</em> and a model that assumes the observations of <em>Y</em><sub>1</sub>, … , <em>Y<sub>N</sub></em> are IID (a single-variable model for <em>Y</em> with λ→∞). We say that <em>Y</em> follows the tree if the difference in scores is greater than zero.</p>


<h4>Experiments with synthetic data</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We constructed two synthetic data sets, one generated by the process of conditional influence (i.e., by a conditional model) and the other generated by coevolution (i.e., the undirected joint model). The data sets representing conditional influence and coevolution were patterned after the real data sets of Application 1 (effects of immune pressure on HIV mutation) and Application 2 (pairwise amino-acid correlations), respectively. In particular, the sample size (<em>N</em>) and number of tests in the data sets were those of the corresponding applications. In addition, the number of variable pairs that were and were not correlated was also patterned after the real data, and the parameters of the generating model for conditional influence (coevolution) were taken from those of a conditional (undirected joint) model fit to the real data. The predictor (HLA) observations for the data set reflecting conditional influence were generated from an IID model.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>As expected, when the data were generated according to the undirected joint model, the joint model was most discriminating, followed by the conditional model (<em>p</em> = 0.042) and FET (<em>p</em>&lt;0.0001; <a href="#pone-0000591-g003">Figure 3</a>). The <em>q</em>-value calibration of the joint model was slightly better than the conditional model, whereas FET was poorly calibrated (<a href="#pone-0000591-g004">Figure 4</a>). Conversely, when the data were generated according to the conditional model, the conditional model was most discriminating, followed by FET (<em>p</em> = 0.12) and the undirected joint model (<em>p</em> = 0.0013; <a href="#pone-0000591-g005">Figure 5</a>). Both the conditional and joint models were well calibrated, whereas the <em>q</em>-values of FET tended to overestimate one minus positive predictive value (<a href="#pone-0000591-g006">Figure 6</a>). Thus, not only did taking hierarchical population structure into account improve the analysis, but the two models could distinguish between the processes of coevolution and conditional influence.</p>
<div class="figure" id="pone-0000591-g003"><div class="img"><a name="pone-0000591-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Discrimination curves for synthetic coevolution data.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The data closely resemble pairwise amino-acid association data (Application 2).</p>
<span>doi:10.1371/journal.pone.0000591.g003</span></div><div class="figure" id="pone-0000591-g004"><div class="img"><a name="pone-0000591-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Calibration of <em>q</em>-values on synthetic coevolution data.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Computing <em>q</em>-values for Fisher's exact test using parametric bootstrap results in poor calibration.</p>
<span>doi:10.1371/journal.pone.0000591.g004</span></div><div class="figure" id="pone-0000591-g005"><div class="img"><a name="pone-0000591-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Discrimination curves for synthetic conditional influence data.</span></strong></p><a id="article1.body1.sec2.sec3.fig3.caption1.p1" name="article1.body1.sec2.sec3.fig3.caption1.p1"></a><p>The data closely resemble the HLA-amino-acid association data (Application 1).</p>
<span>doi:10.1371/journal.pone.0000591.g005</span></div><div class="figure" id="pone-0000591-g006"><div class="img"><a name="pone-0000591-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Calibration of <em>q</em>-values on synthetic conditional influence data.</span></strong></p><span>doi:10.1371/journal.pone.0000591.g006</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In both examples, the <em>q</em>-values from FET were liberal—that is, the <em>q</em>-values underestimated the false positives rates. Although we always expect liberal estimates when the data are generated by coevolution because the tree is a hidden common cause of <em>X</em> and <em>Y</em>, when only one variable follows the tree, the variance of FET's <em>p</em>-values will increase due to over- and under-counting, leading to an unpredictable <em>q</em>-value calibration. Indeed, on other data sets, we have seen FET produce both liberal and conservative estimates of one minus positive predictive value.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>To validate our use of BIC to evaluate model performance on real data, we compared the BIC scores of the undirected joint and conditional models on the synthetic data sets representing conditional influence and coevolution. As expected, we found that the conditional model had a significantly higher score than the undirected joint model on the conditional-influence data set (<em>p</em> = 7.6×10<sup>−53</sup>, <em>N</em> = 157317), and the joint model performed better model on the synthetic coevolution data set (<em>p</em> = 0.06, <em>N</em> = 10025).</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Finally, we note that, over a wide variety of data sets, the conditional model runs an order of magnitude faster than the undirected joint model, which requires optimization over a larger number of parameters and more complex Eigen decompositions.</p>
<h5>Sensitivity to tree structure.</h5><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1"></a><p>Our approach raises the question of how sensitive the results are to the structure of the tree used by the models. To address this question, we ran the conditional model on the synthetic conditional influence data using four different trees: the tree used to generate the data (<em>T<sub>gen</sub></em>), a tree with the same structure as <em>T<sub>gen</sub></em> but with the leaf-to-patient assignments randomized (<em>T<sub>rand</sub></em>), and two trees reconstructed from the synthetic amino acids using either a generalized maximum likelihood method (<em>T<sub>ML</sub></em>, the method we use throughout this work ) or a naïve parsimony method (<em>T<sub>pars</sub></em>).</p>
<a id="article1.body1.sec2.sec3.sec1.p2" name="article1.body1.sec2.sec3.sec1.p2"></a><p>As expected, the conditional model performed best using <em>T<sub>gen</sub></em>, though the discrimination curves were not significantly different from those of <em>T<sub>M</sub></em><sub><strong><em>L</em></strong></sub> and <em>T<sub>pars</sub></em>, indicating that the conditional model is robust to variations of the tree on this data set (<a href="#pone-0000591-g007">Figure 7</a>). Importantly, although the discrimination curve was significantly worse using <em>T<sub>rand</sub></em> rather than <em>T<sub>gen</sub></em> (<em>p</em> = 0.016), the conditional model was calibrated on all four trees, indicating that, on this data set, poor trees resulted in a loss of power but not in an inflation of false discovery rates (<a href="#pone-0000591-g008">Figure 8</a>). This point is reinforced by the number of associations identified at <em>q</em>&lt;0.2 for the different methods: <em>T<sub>gen</sub></em> yielded eighty nine predictions whereas <em>T<sub>rand</sub></em> yielded only sixty five, <em>T<sub>ML</sub></em> yielded seventy eight, and <em>T<sub>pars</sub></em> yielded eighty two. The positive predictive value for these predictions ranged from 0.80 to 0.85.</p>
<div class="figure" id="pone-0000591-g007"><div class="img"><a name="pone-0000591-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Discrimination curves for conditional models based on different trees applied to synthetic conditional influence data.</span></strong></p><span>doi:10.1371/journal.pone.0000591.g007</span></div><div class="figure" id="pone-0000591-g008"><div class="img"><a name="pone-0000591-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Calibration of <em>q</em>-values for conditional models based on different trees applied to synthetic conditional influence data.</span></strong></p><span>doi:10.1371/journal.pone.0000591.g008</span></div><a id="article1.body1.sec2.sec3.sec1.p3" name="article1.body1.sec2.sec3.sec1.p3"></a><p>Although it may seem counter-intuitive that the randomized tree could find any associations, we note that the problem with the conditional model using <em>T<sub>rand</sub></em> is analogous to that of an IID model. Namely, whereas an IID model will over or under count observations by not accounting for hierarchical structure that exists in the data, a randomized tree will over or under count observations by assuming false hierarchical structure. In addition, the conditional model can compensate for a tree that fits the target data poorly by setting the mutation rate to infinity and thereby assuming the data to be IID. Indeed, the median λ value under <em>T<sub>rand</sub></em> was an order of magnitude higher than that under <em>T<sub>gen</sub></em>.</p>



<h4>Application 1: Effect of immune pressure on HIV evolution</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To investigate the effects of immune pressure on HIV evolution, Moore <em>et al.</em> <a href="#pone.0000591-Moore1">[33]</a> obtained HIV sequences from 234 individuals along with the HLA-A and HLA-B alleles of the infected individuals. Performing several analyses, none of which explicitly incorporated phylogenetic structure, they found strong correlations between the presence or absence of amino acids at particular positions and the presence or absence of particular HLA alleles in the infected patients, presumably reflecting the “escape” of amino acids under immune pressure. In Bhattacharya <em>et al.</em> <a href="#pone.0000591-Bhattacharya1">[12]</a>, we analyzed a similar data set (N = 96, MHC-I and MHC-II alleles) and showed that use of the conditional model substantially improved the accuracy of identification of such amino-acid-HLA associations. Here, we analyzed a superset (N = 205) of the data used in <a href="#pone.0000591-Bhattacharya1">[12]</a> (MHC-I alleles only) using both the conditional and undirected joint model.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>First, we constructed a phylogenetic tree from the full set of sequences (see <a href="#s4">Methods</a>). We then used the single-variable model to determine whether any HLA alleles followed the tree. We found that two pairs of HLA-1 alleles—B*4201, Cw*1701, and A*0207, B*4601, where each pair is in tight linkage disequilibrium—followed the tree and thus separated the HLA data into two sets: (1) “C1701” consisting of these four alleles and (2) “notC1701” consisting of the remaining alleles, and analyzed these two sets separately.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Our results using BIC show that the conditional model better explains the notC1701 data, (<em>p</em> = 0.0001, <em>N</em> = 256296 ) whereas the undirected joint model better explains the C1701 data (<em>p</em> = 1.9×10<sup>−24</sup>, <em>N</em> = 5664). In the case of the C1701 data, it seems that the phylogenetic tree is more a confounder of the data in the traditional sense, wherein the tree is associated with both the HLA data and the sequences and induces false correlations between HLA allele and sequence.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>In this application, we were fortunate that additional information was available to help confirm the HLA-sequence associations that we found. In particular, a known epitope in the vicinity of a found association supports the validity of that association, as immune pressure is focused on epitopes and the immediate surrounding regions that participate in the presentation of the those epitopes on the HLA molecules at the cell surface <a href="#pone.0000591-Kloetzel1">[38]</a>. Thus, we constructed discrimination curves where an HLA-sequence association was considered “true” if it is within three amino acids of a known epitope (as described in the database at <a href="http://www.hiv.lanl.gov">www.hiv.lanl.gov</a> as of July 2006) with the corresponding HLA allele and “false” otherwise. This “bronze” standard does not take into account undiscovered epitopes or linkage disequilibrium, but should nonetheless be unbiased with respect to a comparison of the alternative methods for identifying associations. The discrimination curve in <a href="#pone-0000591-g009">Figure 9</a> for the notC1701 data is consistent with the BIC and synthetic results, indicating that the conditional model best fits this data. We could not construct a discrimination curve for the C1701 data, as there are no known A*0207, B*4201, B*4601, or Cw*1701 epitopes in Gag.</p>
<div class="figure" id="pone-0000591-g009"><div class="img"><a name="pone-0000591-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Discrimination curves for the real HLA-amino-acid data.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Ground truth was estimated by identifying known epitopes within three residues of the predicted association.</p>
<span>doi:10.1371/journal.pone.0000591.g009</span></div><a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>The associations found by the conditional model with <em>q</em>&lt;0.2 on the real data are shown in <a href="#pone-0000591-t001">Table 1</a>.</p>
<div class="figure" id="pone-0000591-t001"><div class="img"><a name="pone-0000591-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Predicted HLA-amino acid associations in Gag.</span></strong></p><span>doi:10.1371/journal.pone.0000591.t001</span></div>

<h4>Application 2: Pairwise correlations between amino acids in HIV</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Identification of pairwise correlations between amino acids is important to many areas of biology, as correlations can indicate structural or functional interaction <a href="#pone.0000591-Buck1">[11]</a>, <a href="#pone.0000591-Fodor1">[39]</a>. Many methods, including the undirected joint model <a href="#pone.0000591-Pollock1">[5]</a>, have been developed to identify correlated residues.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Continuing our focus on HIV, we applied both the undirected joint and the conditional model to the sequence data from the Western Australia cohort <a href="#pone.0000591-Moore1">[33]</a>. We concentrated on the HIV-1 p6 protein, which is cleaved from the Gag<sup>55</sup> polyprotein. This fifty two amino-acid protein was chosen because it is the shortest HIV protein, making pair-wise amino acid tests feasible for all models. We fit the conditional model in both directions (making both <em>X</em> and <em>Y</em> target variables), and selected the best model according to BIC.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Remarkably, the BIC scores of the conditional model are significantly higher than those of the joint model (<em>p</em>&lt;10<sup>−100</sup>, <em>N</em> = 52767). We suspect that the conditional model may be better because many mutations could be compensating for other mutations driven by HLA immune pressure. The conditional model finds that 893 of the 52767 (1.7%) amino acid pairs, and 310 of 1300 (24%) of position pairs, are correlated at <em>q</em>&lt;0.2. This dense network of interactions is consistent with the idea that many of the mutations are compensatory in nature. For example, the conditional model identifies two HLA-mediated escape mutations in p6 (<a href="#pone-0000591-t001">Table 1</a>). Mutations at these two positions account for forty two (13.5%) of the position-pair correlations.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>We have developed a tool for visualizing the network of dependencies (<a href="#pone-0000591-g010">Figure 10</a>). The visualization highlights at least one potentially interesting set of interactions. In particular, R16 is strongly correlated with residues at positions 21 through 36, many of which are correlated with each other as well as with residues throughout the protein. This complex network of interactions connects the two α-helix domains of p6 <a href="#pone.0000591-Fossen1">[40]</a> and may be of structural or functional significance.</p>
<div class="figure" id="pone-0000591-g010"><div class="img"><a name="pone-0000591-g010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g010.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 10.  <span>Correlated amino-acid pairs in HIV-1 p6.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>The fifty two consensus amino acids of P6 are drawn as a circle, with the N-terminal end shown at the far right and the protein extending counter-clockwise. Each arc represents an association predicted by the conditional model that is significant at <em>q</em>&lt;0.2. Arc color reflects the <em>q</em>-value of the association. Dark gray residues denote positions where there were fewer than three sequences with a non-consensus residue. The associations used to construct the figure are available as <a href="#pone.0000591.s001">Dataset S1</a>. Annotations of individual residues are: P, phosphorylated residue; Ub, site of ubiquitinization; +/−, charged residue.</p>
<span>doi:10.1371/journal.pone.0000591.g010</span></div>

<h4>Application 3: Genomic search for genotype-phenotype associations in Arabidopsis thaliana</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Aranzana <em>et al.</em> recently demonstrated the potential utility of genome wide association (GWA) studies, as well as the importance of accounting for hierarchical population structure <a href="#pone.0000591-Aranzana1">[23]</a>. In this study, the authors genotyped 848 loci in ninety six <em>Arabidopsis thaliana</em> strains and looked for haplotypes that were correlated with hypersensitive response to <em>P. syringae</em> strains expressing one of three avirulence (avr) genes (avrRpm1, avrRpt2, or avrPph3). In plants, each avr bacterial protein is recognized by a corresponding resistance (R) gene. If both plant and pathogen have active copies of the respective avr-R genes, a biochemical cascade is triggered at the point of infection, leading to massive programmed cell death and containment of the infection (for review, see <a href="#pone.0000591-Dangl1">[41]</a>). Using both an IID-based model and a method that used the hierarchical population structure that was constructed from the sequenced loci, the authors showed that loci adjacent to the known R genes are highly correlated with the corresponding avr phenotypes. Unfortunately, the authors noted that their statistics were poorly calibrated, precluding confident predictions of the other pathogen-response proteins that are involved in the hypersensitive response cascade. Here, we apply our well-calibrated methods to the same data, using a genetic-similarity tree constructed from the sequence data.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Although <em>Arabidopsis</em> is a sexually reproducing species, it is highly selfing, meaning that organisms primarily mate with themselves. As a result, the population structure induced is hierarchical and bears striking resemblance to a phylogenetic tree. Aranzana <em>et al.</em> found that a tree built from pairwise similarity matrices on shared alleles provided a good qualitative description of both the geographic distribution of the organisms and the distribution of avr and flowering time alleles <a href="#pone.0000591-Aranzana1">[23]</a>. Quantitatively, we found that sixty one percent of the haplotypes and two of the three phenotypes followed the “phylogenetic” tree constructed from the sequence data.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>When applying our conditional model to this application, it is not clear whether the target variables should be haplotypes or phenotypes. In general, genetic variations directly influence phenotypes, but phenotypes also indirectly influence haplotypes through selection pressure. As two thirds of both variables followed the tree, we ran the conditional model in both directions, once using the phenotypes as the target and once using them as the predictor, using BIC to determine which direction was best for any given haplotype-phenotype pair.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>We found that the BIC scores for the conditional and undirected joint models were not significantly different (<em>p</em> = 0.70, <em>N</em> = 14043). Consequently, we arbitrarily choose to examine the results of the conditional model in detail. <a href="#pone-0000591-g011">Figure 11</a> shows the genome wide distribution of conditional influence <em>q</em>-values for each of the three phenotypes. For each phenotype, the most significant association is a locus near the corresponding R gene. We constructed this figure to be similar to the one in Aranzana <em>et al.</em> to facilitate comparison.</p>
<div class="figure" id="pone-0000591-g011"><div class="img"><a name="pone-0000591-g011" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g011&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000591" data-uri="info:doi/10.1371/journal.pone.0000591.g011"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.g011&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g011/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g011/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g011/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g011/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g011.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000591.g011/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000591.g011/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000591.g011.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 11.  <span>Genomic distribution of genotype-phenotype association scores for <em>Arabidopsis</em> bacterial response.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>4681 haplotypes were compared against each of the three bacterial response phenotypes, Rpm1 (top), Rpt2 (middle) and Pph3 (bottom). For each haplotype, the four conditional models were run and negative log<sub>10</sub> of the most significant <em>q</em>-value is plotted. For each phenotype, the most significant association is a locus within 10 kb of the corresponding R gene (yellow lines). The dotted line shows the <em>q</em> = 0.2 threshold.</p>
<span>doi:10.1371/journal.pone.0000591.g011</span></div><a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p>Our synthetic tests indicate that the conditional method is well calibrated, implying that roughly 80% of the associations we find with a <em>q</em>&lt;0.2 cutoff should be legitimate. To explore this implication, we took the fifty one genotypic associations (comprising forty unique loci) that correlate with these three hypersensitive phenotypes at this cutoff, and noted which of the associated loci were near known or putative bacterial response proteins according to <a href="http://www.arabidopsis.org">http://www.arabidopsis.org</a>. Our standard of “true positive” was defined to be proximity to a protein whose description included the phrase “disease response” (see <a href="#s4">Methods</a>). We found that twenty three (45%) of the predictions were within fifty kilobases (kb) of such proteins. This “bronze” standard undoubtedly contains false positives and false negatives, and therefore cannot be used to confirm that our methods our calibrated. Nonetheless, we easily can reject the null hypothesis that these twenty three associations are found near disease-response proteins by chance (<em>p</em>&lt;0.0001).</p>


<h4>Related work</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Evolutionary biologists have long been interested in studying correlated traits in the midst of population structure due to phylogenetic relationships (for reviews, see <a href="#pone.0000591-Harvey1">[42]</a>, <a href="#pone.0000591-Felsenstein3">[43]</a>). For association studies on continuous variables, the method of choice has been to use independent contrasts <a href="#pone.0000591-Felsenstein1">[1]</a>. In this method, the data are assumed to be derived from a Brownian motion (or other Gaussian) model of evolution and the <em>n</em> samples are converted to <em>n</em>−1 differences between adjacent nodes in the tree, with the variance of the differences computed according to the branch lengths of the tree. The resulting differences are independent, allowing for regression tests between two traits. This method is effective and well used, but is appropriate only for continuous data. For discrete data, early work focused on reconstructing ancestral states using parsimony and looking for correlations between transitions <a href="#pone.0000591-Ridley1">[2]</a>, <a href="#pone.0000591-Maddison1">[3]</a>. Pagel <a href="#pone.0000591-Pagel1">[4]</a> presented a maximum likelihood approach that averages over all possible configurations of the internal nodes. Pollock, Taylor and Goldman <a href="#pone.0000591-Pollock1">[5]</a>, which we study here, is a reversible version of Pagel's method.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>Population structure in biological data has also been addressed in the area of GWA studies. There are rather different approaches in this community that have been used to compensate for population structure. A more commonly used approach attempts to recalibrate standard statistics by normalizing results according to the distribution of the statistic across the entire genome <a href="#pone.0000591-Devlin1">[24]</a>, <a href="#pone.0000591-Devlin2">[25]</a>. As we have seen, solutions to calibration are insufficient, as population structure also affects discriminatory power. Another approach assumes population structure is flat and can be captured by a small number of (perhaps overlapping) clusters or continuous hidden variables <a href="#pone.0000591-Pritchard1">[26]</a>–<a href="#pone.0000591-Price1">[30]</a>. Although these methods increase discriminatory power relative to simple IID models, there is mounting evidence that population is hierarchically structured. For example, in addition to high level geographical/social constraints, structure exists within a number of subpopulations that have been studied <a href="#pone.0000591-Auranen1">[19]</a>–<a href="#pone.0000591-Campbell1">[22]</a>. It stands to reason that, if hierarchical models describe the data better than flat cluster models, then such models will have higher discriminatory power. Thus, several authors have suggested that a more accurate approach would be to model population structure hierarchically <a href="#pone.0000591-Aranzana1">[23]</a>, <a href="#pone.0000591-Kennedy1">[31]</a>, <a href="#pone.0000591-Yu1">[32]</a>. Aranzana <em>et al.</em> described one such model in their <em>Arabidopsis</em> study. As mentioned, however, they found that their model was not calibrated and their data precluded a discriminatory power analysis. Yu <em>et al.</em> extended the linear mixed-effects model of Kennedy <em>et al.</em> <a href="#pone.0000591-Kennedy1">[31]</a>, in which hierarchical population structure is modeled with random effects. The approach is well calibrated, but is typically inappropriate for discrete data.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>Similar to what happened in the GWA community, those studying amino acid coevolution initially ignored the phylogenetic structure of the protein sequences <a href="#pone.0000591-Korber1">[44]</a>–<a href="#pone.0000591-Neher1">[47]</a>, but came to recognize that phylogeny played a confounding roll <a href="#pone.0000591-Wollenberg1">[10]</a>, <a href="#pone.0000591-Pollock2">[48]</a>, <a href="#pone.0000591-Noivirt1">[49]</a>. A number of methods emerged that attempted to calibrate <em>p</em>-values based on global or local measures of average similarity <a href="#pone.0000591-Bruno1">[6]</a>–<a href="#pone.0000591-Suel1">[9]</a> or flat clusters based on early bifurcations of the phylogenetic tree <a href="#pone.0000591-Buck1">[11]</a>. One approach that explicitly incorporates the phylogeny is the parametric bootstrap, in which standard IID models are recalibrated by generating independently evolved amino acids according to the single variable model <a href="#pone.0000591-Wollenberg1">[10]</a>, <a href="#pone.0000591-Buck1">[11]</a>. On our synthetic coevolution data, we found this approach to be too severe, resulting in overly conservative <em>q</em>-value estimates (see <a href="#pone-0000591-g005">Figure 5</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have identified two evolutionary processes that can confound association analyses and have defined two corresponding generative models for discrete data that can correct for and even leverage the existence of these processes. We have found that explicitly modeling evolutionary processes increases discriminatory power and results in well-calibrated estimates of one minus positive predictive value. We have implemented methods for fitting these models to data and a tool for visualizing the results of the analysis. These tools are available on the internet.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The undirected joint model assumes that the two variables coevolve such that a mutation event in either variable can elicit a corresponding change in the other variable. In contrast, the conditional model assumes that a single variable is distributed according to the tree and is influenced by the predictor variable only at the tips of the tree. Of course, other evolutionary processes are possible. In <a href="#s4">Methods</a>, we describe a third process in which variables coevolve, but only one variable influences the evolution of the other.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Neither the undirected joint nor conditional model outperformed the other on all real data sets, suggesting that both models should be considered when analyzing new data. Nonetheless, the conditional model better fit most of the real data that we analyzed. The conditional model better described the effects of immune pressure on HIV evolution, and perhaps more surprising, better described the correlation between HIV-1 p6 amino-acid pairs. This observation may be due to the rapid evolution of HIV and positive selection pressure from the immune response in conjunction with compensatory mutations in the observed sequences.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Our study has focused on the correlation of discrete (specifically binary) variables. Generalization to multistate variables is straightforward, although may suffer a loss in power. Our methods can also be generalized to continuous and/or discrete predictor and target variables. When the target variable is continuous, the conditional model is a special case of a linear mixed-effects model (K.M. Kang, N. Zaitlen, C.M. Wade, A. Kirby, D.H., M.J. Daly and E. Eskin, submitted). The conditional model can also be generalized to situations with multiple predictor and target variables, thus producing a directed network (acyclic or otherwise) of relationships among multiple variables. Potential applications of multiple predictor variables include learning the combined effects of drug and immune pressure on HIV evolution, identifying chains of compensatory mutations, learning the influence of diploid genes on phenotype, and learning networks of interacting genes and proteins. Finally, one could also use the undirected joint or conditional models to learn the structure of phylogenentic or hierarchical relationships rather than learning the tree structure with standard methods that ignore correlations.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The problem of population structure confounding association studies is a ubiquitous problem across many biological disciplines. Existing solutions vary across these disciplines, but typically focus on correcting for shared population structure. As we have seen, however, population structure in either variable can lead to loss of discriminatory power and poor statistical calibration. The flexibility and intuitive nature of generative models makes them a natural and powerful choice for dealing with a variety of biological processes.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Data sets</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We obtained HIV aligned sequences and HLA data from the Western Australia cohort (HIV sequence accession numbers AY856956–AY857186 and EF116290–EF116445) <a href="#pone.0000591-Moore1">[33]</a>, <a href="#pone.0000591-Kiepiela1">[50]</a>. We noted some anomalies in the alignment of p6 and corrected them by hand. We constructed the phylogenetic tree for Applications 1 and 2 from the full Gag DNA alignment by applying PHYML <a href="#pone.0000591-Guindon1">[51]</a> with the general reversible model, optimized tree topology, maximum likelihood estimates for the base frequencies, estimated proportion of invariable sites, and four substitution rate categories with an estimated gamma distribution parameter. When identifying associations, we binarized amino acids, such that a single association test compared the presence or absence of the residue in question. Ambiguous codons were treated as uncertainties. For example, if a codon was <em>X</em> or <em>Y</em>, then it was treated as unknown for codon <em>X</em>, but known to be false for any codon not equal to <em>X</em> or <em>Y</em>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The Arabidopsis data set was taken from Aranzana <em>et al.</em> <a href="#pone.0000591-Aranzana1">[23]</a>. We built the genetic-similarity tree using PHYML from a set of sequences consisting of positions in the locus alignments for which at least two sequences differed from the consensus. Following <a href="#pone.0000591-Aranzana1">[23]</a>, for each genotyped locus, we defined haplotypes according to sequence identity after removing positions for which only one sequence varied from the consensus. All sequences that accounted for less than five percent of the data were clustered together.</p>
<h5>Synthetic data sets.</h5><a id="article1.body1.sec4.sec1.sec1.p1" name="article1.body1.sec4.sec1.sec1.p1"></a><p>We generated synthetic data to approximate real data as closely as possible. In the case of synthetic conditional influence, we first ran the conditional model on the real HLA data set to obtain reasonable parameter values. To generate predictor variables, we permuted the real HLA alleles to ensure the data were IID. For each association in the real data set, we generated synthetic target data in one of two ways: (1) if the association was significant (<em>q</em>&lt;0.3), we took the corresponding HLA allele and the parameters learned on the real allele-amino acid pair and generated data for the target variable; otherwise (2) we generated the target variable data using the parameters learned by the single-variable model. Because there were only twenty six significant associations, we generated five synthetic associations for each real association. For each replicate, we chose a different HLA allele, using the five HLA alleles whose frequencies most closely matched that of the real HLA allele on which the association was based.</p>
<a id="article1.body1.sec4.sec1.sec1.p2" name="article1.body1.sec4.sec1.sec1.p2"></a><p>In the case of coevolution, we first fit the undirected joint model to the HIV p6 data set, and then generated synthetic associations using the learned parameters. For these data, we generated one synthetic association for each real association (<em>q</em>&lt;0.3). For all non-significant real associations, we generated the predictor and target variables using the single-variable model with the parameters learned by that model on the real data.</p>

<h5>Inferring trees from synthetic data.</h5><a id="article1.body1.sec4.sec1.sec2.p1" name="article1.body1.sec4.sec1.sec2.p1"></a><p>When analyzing the sensitivity of results to tree structure, we needed to infer a phylogenetic tree from synthetic data. To do so, we constructed binary sequences from the synthetic target variables, such that each position in the binary sequence corresponded to a target variable. We then used the PHYML software as described above for real data to infer a maximum likelihood phylogeny. In addition, we used the dnapars program from the PHYLIP package <a href="#pone.0000591-Felsenstein4">[52]</a> to infer a phylogeny using parsimony. We ran dnapars using default settings, with the exception that only a single tree was optimized.</p>



<h4>Computing q-values</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The independence (null) models are nested inside the undirected joint and conditional models and contain one less parameter. Therefore, the asymptotic distribution of the log-likelihood ratio is χ<sup>2</sup>-distributed with one degree of freedom from which <em>p</em>-values can be computed. Simple <em>p</em>-values are, however, of limited use in the context of high levels of multiple hypothesis testing. Whereas a Bonferroni correction could be used to limit false positives, the high number of tests (typically in the tens or hundreds of thousands) forces an almost complete loss of power. Consequently, we used <em>q</em>-values to control for multiple tests. To compute a <em>q</em>-value, we first computed a False Discovery Rate (FDR), which measures the expected fraction of true positives for any given <em>p</em>-value threshold <em>p</em><sub>0</sub> <a href="#pone.0000591-Benjamini1">[53]</a>. For a given <em>p</em><sub>0</sub>, we estimated FDR to be the ratio of the expected number of associations with <em>p</em>&lt;<em>p</em><sub>0</sub> under the null distribution to the observed number of associations with <em>p</em>&lt;<em>p</em><sub>0</sub> in the real data. In our experiments, we generated ten associations under the null model for each examined <em>X–Y</em> pair to compute the expectation under the null. (As described by, e.g., Storey and Tibshirani <a href="#pone.0000591-Storey1">[54]</a>, the true FDR should be computed using null data that is generated only from observed values from which clearly non-null data have been removed. As only a small number of tested associations were real in our applications, the inclusion of null data generated from these associations introduced a small, conservative bias in our computation of FDR.) We then computed the <em>q</em>-value as the minimum FDR seen for <em>p<sub>i</sub></em>≥<em>p</em><sub>0</sub> <a href="#pone.0000591-Storey1">[54]</a>. This guaranteed <em>q</em> to be a monotonic increasing function of <em>p</em><sub>0</sub>. (In general, FDR is expected to be a monotonic function of <em>p</em><sub>0</sub>, but is rarely monotonic in practice due to, e.g., variance in the statistic.)</p>
<h5>Generating null data.</h5><a id="article1.body1.sec4.sec2.sec1.p1" name="article1.body1.sec4.sec2.sec1.p1"></a><p>When testing for an association between variables <em>X</em> and <em>Y</em>, null data can be generated in one of four ways: permute the observations of <em>X</em>, permute the observations of <em>Y</em>, parametrically bootstrap observations for <em>X</em>, or parametrically bootstrap observations for <em>Y</em>. The best method—the one that most accurately constructs the null data—will depend on the data set being analyzed. For example, suppose we have data where both the predictor and target variable follow a given tree. Here, we should parametrically bootstrap observations for either the target or predictor variable, as doing otherwise would produce data that no longer follows the tree, introducing a bias in the null data. In contrast, suppose observations of the predictor variable are IID but observations of the target variable are not well described by the tree. Here, we should permute the observations of the predictor variable because (1) they remain IID under permutation and (2) a parametric bootstrap of the target variable under the assumption that the data follows the tree would produce biased data. The best method for generating null data may also depend on the model being used to analyze that data as the undirected joint and conditional models can be sensitive to different biases in the data.</p>
<a id="article1.body1.sec4.sec2.sec1.p2" name="article1.body1.sec4.sec2.sec1.p2"></a><p>We have developed and used a systematic approach for determining which null-generation method to use for a given data set and given model for analysis based on two observations. First, as the computation of <em>q</em>-values depends on the distribution of <em>p</em>-values under the null hypothesis, it was important to select a null-generation method that produced an accurate distribution of <em>p</em>-values. Second, in all the data sets that we analyzed, the vast majority of variable pairs were not associated—that is, they satisfied the null distribution. Consequently, given a data set and a given model for analysis, we chose the data-generation method by identifying the one that yielded a distribution of <em>p</em>-values that most closely matched that produced by the given (real) data.</p>
<a id="article1.body1.sec4.sec2.sec1.p3" name="article1.body1.sec4.sec2.sec1.p3"></a><p>Our approach yielded the following choices: permutation bootstrap of the predictor variable for Applications 1 and 3, and parametric bootstrap of the predictor variable for Application 2. For the analysis of synthetic data, we used null-generation methods that would preserve the known distributions of the predictor and target variables: permutation bootstrap of the predictor variable for the conditional influence data set, and parametric bootstrap of the predictor variable for the coevolution data set.</p>

<h5>Computing <em>q</em>-values on subsets of data.</h5><a id="article1.body1.sec4.sec2.sec2.p1" name="article1.body1.sec4.sec2.sec2.p1"></a><p>When computing <em>q</em>-values, it is sometimes useful to partition the tests into two or more sets to obtain more informative values. For example, suppose we partition a set of tests into two sets <em>A</em> and <em>B</em> corresponding to positive and negative correlations, respectively. For simplicity, assume the null distributions underlying <em>A</em> and <em>B</em> are identical. If <em>A</em> and <em>B</em> have the same number of associations with <em>p</em>&lt;<em>p</em><sub>0</sub>, then we expect <em>q<sub>A</sub></em>(<em>p</em><sub>0</sub>) = <em>q<sub>B</sub></em>(<em>p<sub>0</sub></em>) = <em>q<sub>A</sub></em><sub>∪<em>B</em></sub>(<em>p</em><sub>0</sub>). If <em>A</em> has a higher number of associations with <em>p</em>&lt;<em>p</em><sub>0</sub> than does <em>B</em>, however, then we expect <em>q<sub>A</sub></em>(<em>p</em><sub>0</sub>)&lt;<em>q<sub>A</sub></em><sub>∪<em>B</em></sub>(<em>p</em><sub>0</sub>)&lt;<em>q<sub>B</sub></em>(<em>p</em><sub>0</sub>). That is, computing <em>q</em>-values with respect to the merged set of associations will inflate the <em>q</em>-values of the associations in <em>A</em> and deflate the <em>q</em>-values of the associations in <em>B</em>. Whereas <em>q</em> is still the expected number of false positives, computing <em>q<sub>A</sub></em><sub>∪<em>B</em></sub>(<em>p</em><sub>0</sub>) assures that the majority of those false positives are in <em>B</em> and some true positives in <em>A</em> are likely to be lost. The disadvantage of partitioning the tests is that more null data must be generated in order to preserve the variance of the estimate for <em>q</em>.</p>
<a id="article1.body1.sec4.sec2.sec2.p2" name="article1.body1.sec4.sec2.sec2.p2"></a><p>In this work, we split our tests along natural boundaries. In our experiments with the conditional model, we computed <em>q</em>-values separately for each possible transition matrix (escape, attraction, reversion, and repulsion). For the undirected joint model and when computing FET, we computed <em>q</em>-values separately for positive and negative correlations.</p>



<h4>Performance Evaluation</h4>
<h5>Comparing discrimination curves.</h5><a id="article1.body1.sec4.sec3.sec1.p1" name="article1.body1.sec4.sec3.sec1.p1"></a><p>To compute the significance of the difference between the discrimination curves of two methods <em>A</em> and <em>B</em> applied to synthetic data, we computed the areas under each curve and tested whether the area under <em>A</em>'s curve was significantly different from the area under <em>B</em>'s curve.</p>
<a id="article1.body1.sec4.sec3.sec1.p2" name="article1.body1.sec4.sec3.sec1.p2"></a><p>The permutation test was carried out as follows. Given a set <em>T</em> of <em>n</em> association tests, a method <em>A</em> induces an ordering <em>T<sup>(A)</sup></em> over <em>T</em> such that <em>t<sub>i</sub></em><sup>(<em>A</em>)</sup>&lt;<em>t<sub>j</sub></em><sup>(<em>A</em>)</sup>(<em>i</em>≠<em>j</em>) implies that <em>t<sub>i</sub></em> is more significant than <em>t<sub>j</sub></em> by method <em>A</em>. Let <strong>a</strong> = [<em>a</em><sub>1</sub>, <em>a</em><sub>2</sub>,…,<em>a<sub>n</sub></em>] be a binary vector such that <em>a<sub>i</sub></em> = 1 if and only if <em>t<sub>i</sub></em><sup>(<em>A</em>)</sup> is an association. We computed the area under the curve (AUC) of <em>A</em> as a function of <strong>a</strong>.</p>
<a id="article1.body1.sec4.sec3.sec1.p3" name="article1.body1.sec4.sec3.sec1.p3"></a><p>When comparing the discrimination curves of two methods <em>A</em> and <em>B</em>, the null hypothesis is that the curves come from equivalent methods, such that the <em>i</em>-th prediction of <em>A</em> is just as likely to be an association as is the <em>i</em>-th prediction of <em>B</em> (that is, <em>Pr</em>[<em>a<sub>i</sub></em> = 1] = <em>Pr</em>[<em>b<sub>i</sub></em> = 1]). We estimated this null distribution using a permutation bootstrap, in which we randomly swapped the assignments of <em>a<sub>i</sub></em> and <em>b<sub>i</sub></em> and recomputed <em>AUC</em>(<strong>a</strong>)–<em>AUC</em>(<strong>b</strong>). (For a two-tailed test, we used the absolute value of the difference.) With synthetic data, we have prior knowledge as to which method will perform best (the model used to generate the data). Thus, we report the one-tailed <em>p</em>-value.</p>

<h5>Evaluating performance on <em>Arabidopsis thaliana</em> GWA study.</h5><a id="article1.body1.sec4.sec3.sec2.p1" name="article1.body1.sec4.sec3.sec2.p1"></a><p>To determine whether a set of predictions was likely to be enriched for disease response proteins, we downloaded the genomic positions of all genes whose description contained the phrase “disease response” (data taken from <a href="http://www.arabidopsis.org">http://www.arabidopsis.org</a>). There were 226 such <em>target genes</em> matching this search criterion, including the known R genes, genes that are known to be involved in the hypersensitive response cascade and a number of putative proteins with high sequence similarity to the known R genes. For each predicted locus, we calculated the minimum distance to one of these genes. We used the fifty kb threshold because it is the most conservative estimate of linkage disequilibrium <a href="#pone.0000591-Nordborg1">[55]</a> and because, at higher distances, it becomes unsurprising that a locus is proximal to one of the 226 target genes.</p>
<a id="article1.body1.sec4.sec3.sec2.p2" name="article1.body1.sec4.sec3.sec2.p2"></a><p>We computed the probability that <em>m</em> of the <em>n</em> predictions would fall within fifty kb of some target genes using a permutation bootstrap. In each iteration of the bootstrap, we randomly selected (without replacement) <em>n</em> haplotypes from our study and counted how many were within fifty kb of a target gene.</p>



<h4>Directed joint model</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The Pollock model for coevolution assumes a symmetric or undirected relationship between two coevolving variables <em>X</em> or <em>Y.</em> Alternatively, we can imagine a <em>directed joint model</em>, wherein <em>X</em> or <em>Y</em> coevolve, but <em>X</em> influences the evolution of <em>Y</em> but not vice versa. In this process, <em>X</em> evolves as in the single-variable model, and <em>Y</em> evolves with a rate that depends on whether <em>X</em> is absent or present. In particular, the transition probability matrix for the model is given by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000591.e006&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where the diagonal entries assure that the rows sum to 0. This model, like the undirected joint model, has five parameters: π<em><sub>x</sub></em>, π<em><sub>y</sub></em>, λ<em><sub>X</sub></em>, λ<em><sub>Y</sub></em><sub>|<em>X</em></sub>, and λ<em><sub>Y</sub></em><sub>|<em>X̅</em></sub>. As in the undirected case, we compute the probability that one instance of <em>XY</em> transitions to another along a branch of length <em>t</em> by determining <strong>P</strong> = exp[<strong>Q</strong><em>t</em>], using standard numerical Eigen-decomposition techniques <a href="#pone.0000591-Press1">[36]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000591"></div><a name="pone.0000591.s001" id="pone.0000591.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000591.s001">Dataset S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">HIV-1 p6 amino acid pairs that are correlated at q&lt;0.2.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000591.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.36 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Bette Korber, Tanmoy Bhattacharya, Eleazar Eskin, Noah Zaitlen, Hyun Min Kang, and Walter Ruzzo for helpful discussions. We also thank participants in the WA HIV Cohort Study as well as past and present clinical and laboratory staff of the Department of Clinical Immunology and Biochemical Genetics, Royal Perth Hospital, Western Australia. Finally, we thank the anonymous reviewer for insightful comments.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JC DH. Performed the experiments: JC. Analyzed the data: JC DH. Wrote the paper: JC DH. Other: Provided HLA and HIV data: SM. Implemented the conditional model: CK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000591-Felsenstein1" id="pone.0000591-Felsenstein1"></a>Felsenstein  (1985) Phylogenies and the comparative method. American Naturalist  125:   <ul class="find" data-citedArticleID="964334" data-doi="10.1086/284325"><li><a href="http://dx.doi.org/10.1086/284325" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenies+and+the+comparative+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenies+and+the+comparative+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000591-Ridley1" id="pone.0000591-Ridley1"></a>Ridley M (1983) The Explanation of Organic Diversity: The Comparative Method and Adaptations for Mating. Oxford: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0000591-Maddison1" id="pone.0000591-Maddison1"></a>Maddison DR (1990) Phylogenetic Inference of Historical Pathways and Models of Evolutionary Change. Ph.D. thesis,.   Harvard University, Cambridge, MA.  <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0000591-Pagel1" id="pone.0000591-Pagel1"></a>Pagel M (1994) Detecting correlated evolution on phylogenies: a general method for the comparative analysis of discrete characters. Proc R Soc Lond B Biol Sci  255: 37–45.  <ul class="find" data-citedArticleID="964346" data-doi="10.1098/rspb.1994.0006"><li><a href="http://dx.doi.org/10.1098/rspb.1994.0006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+correlated+evolution+on+phylogenies%3A+a+general+method+for+the+comparative+analysis+of+discrete+characters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+correlated+evolution+on+phylogenies%3A+a+general+method+for+the+comparative+analysis+of+discrete+characters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000591-Pollock1" id="pone.0000591-Pollock1"></a>Pollock DD, Taylor WR, Goldman N (1999) Coevolving protein residues: maximum likelihood identification and relationship to structure. J Mol Biol  287: 187–198.  <ul class="find" data-citedArticleID="964394" data-doi="10.1006/jmbi.1998.2601"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.2601" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coevolving+protein+residues%3A+maximum+likelihood+identification+and+relationship+to+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coevolving+protein+residues%3A+maximum+likelihood+identification+and+relationship+to+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000591-Bruno1" id="pone.0000591-Bruno1"></a>Bruno WJ (1996) Modeling residue usage in aligned protein sequences via maximum likelihood. Mol Biol Evol  13: 1368–1374.  <ul class="find" data-citedArticleID="964318" data-doi="10.1093/oxfordjournals.molbev.a025583"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025583" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+residue+usage+in+aligned+protein+sequences+via+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+residue+usage+in+aligned+protein+sequences+via+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000591-Lockless1" id="pone.0000591-Lockless1"></a>Lockless SW, Ranganathan R (1999) Evolutionarily conserved pathways of energetic connectivity in protein families. Science  286: 295–299.  <ul class="find" data-citedArticleID="964344" data-doi="10.1126/science.286.5438.295"><li><a href="http://dx.doi.org/10.1126/science.286.5438.295" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionarily+conserved+pathways+of+energetic+connectivity+in+protein+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionarily+conserved+pathways+of+energetic+connectivity+in+protein+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000591-Larson1" id="pone.0000591-Larson1"></a>Larson SM, Di Nardo AA, Davidson AR (2000) Analysis of covariation in an SH3 domain sequence alignment: applications in tertiary contact prediction and the design of compensating hydrophobic core substitutions. J Mol Biol  303: 433–446.  <ul class="find" data-citedArticleID="964390" data-doi="10.1006/jmbi.2000.4146"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4146" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+covariation+in+an+SH3+domain+sequence+alignment%3A+applications+in+tertiary+contact+prediction+and+the+design+of+compensating+hydrophobic+core+substitutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+covariation+in+an+SH3+domain+sequence+alignment%3A+applications+in+tertiary+contact+prediction+and+the+design+of+compensating+hydrophobic+core+substitutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000591-Suel1" id="pone.0000591-Suel1"></a>Suel GM, Lockless SW, Wall MA, Ranganathan R (2003) Evolutionarily conserved networks of residues mediate allosteric communication in proteins. Nat Struct Biol  10: 59–69.  <ul class="find" data-citedArticleID="964354" data-doi="10.1038/nsb881"><li><a href="http://dx.doi.org/10.1038/nsb881" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionarily+conserved+networks+of+residues+mediate+allosteric+communication+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionarily+conserved+networks+of+residues+mediate+allosteric+communication+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000591-Wollenberg1" id="pone.0000591-Wollenberg1"></a>Wollenberg KR, Atchley WR (2000) Separation of phylogenetic and functional associations in biological sequences by using the parametric bootstrap. Proc Natl Acad Sci USA  97: 3288–3291.  <ul class="find" data-citedArticleID="964398" data-doi="10.1073/pnas.97.7.3288"><li><a href="http://dx.doi.org/10.1073/pnas.97.7.3288" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Separation+of+phylogenetic+and+functional+associations+in+biological+sequences+by+using+the+parametric+bootstrap." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Separation+of+phylogenetic+and+functional+associations+in+biological+sequences+by+using+the+parametric+bootstrap.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000591-Buck1" id="pone.0000591-Buck1"></a>Buck MJ, Atchley WR (2005) Networks of coevolving sites in structural and functional domains of serpin proteins. Mol Biol Evol  22: 1627–1634.  <ul class="find" data-citedArticleID="964356" data-doi="10.1093/molbev/msi157"><li><a href="http://dx.doi.org/10.1093/molbev/msi157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Networks+of+coevolving+sites+in+structural+and+functional+domains+of+serpin+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Networks+of+coevolving+sites+in+structural+and+functional+domains+of+serpin+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000591-Bhattacharya1" id="pone.0000591-Bhattacharya1"></a>Bhattacharya T, Daniels M, Heckerman D, Foley B, Frahm N, et al.  (In press) Founder effects in the assessment of HIV polymorphisms and HLA allele associations. Science.   <ul class="find" data-citedArticleID="964364" data-doi="10.1126/science.1131528"><li><a href="http://dx.doi.org/10.1126/science.1131528" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Founder+effects+in+the+assessment+of+HIV+polymorphisms+and+HLA+allele+associations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Founder+effects+in+the+assessment+of+HIV+polymorphisms+and+HLA+allele+associations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000591-Lander1" id="pone.0000591-Lander1"></a>Lander ES, Schork NJ (1994) Genetic dissection of complex traits. Science  265: 2037–2048.  <ul class="find" data-citedArticleID="964338" data-doi="10.1126/science.8091226"><li><a href="http://dx.doi.org/10.1126/science.8091226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+dissection+of+complex+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+dissection+of+complex+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000591-Risch1" id="pone.0000591-Risch1"></a>Risch NJ (2000) Searching for genetic determinants in the new millennium. Nature  405: 847–856.  <ul class="find" data-citedArticleID="964314" data-doi="10.1038/35015718"><li><a href="http://dx.doi.org/10.1038/35015718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Searching+for+genetic+determinants+in+the+new+millennium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Searching+for+genetic+determinants+in+the+new+millennium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000591-Hirschhorn1" id="pone.0000591-Hirschhorn1"></a>Hirschhorn JN, Daly MJ (2005) Genome-wide association studies for common diseases and complex traits. Nat Rev Genet  6: 95–108.  <ul class="find" data-citedArticleID="964292" data-doi="10.1038/nrg1521"><li><a href="http://dx.doi.org/10.1038/nrg1521" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+association+studies+for+common+diseases+and+complex+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+association+studies+for+common+diseases+and+complex+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000591-Evans1" id="pone.0000591-Evans1"></a>Evans DM, Cardon LR (2006) Genome-wide association: a promising start to a long race. Trends Genet  22: 350–354.  <ul class="find" data-citedArticleID="964380" data-doi="10.1016/j.tig.2006.05.001"><li><a href="http://dx.doi.org/10.1016/j.tig.2006.05.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+association%3A+a+promising+start+to+a+long+race." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+association%3A+a+promising+start+to+a+long+race.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000591-Marchini1" id="pone.0000591-Marchini1"></a>Marchini J, Cardon LR, Phillips MS, Donnelly P (2004) The effects of human population structure on large genetic association studies. Nat Genet  36: 512–517.  <ul class="find" data-citedArticleID="964310" data-doi="10.1038/ng1337"><li><a href="http://dx.doi.org/10.1038/ng1337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effects+of+human+population+structure+on+large+genetic+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effects+of+human+population+structure+on+large+genetic+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000591-Marchini2" id="pone.0000591-Marchini2"></a>Marchini J, Donnelly P, Cardon LR (2005) Genome-wide strategies for detecting multiple loci that influence complex diseases. Nat Genet  37: 413–417.  <ul class="find" data-citedArticleID="964306" data-doi="10.1038/ng1537"><li><a href="http://dx.doi.org/10.1038/ng1537" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+strategies+for+detecting+multiple+loci+that+influence+complex+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+strategies+for+detecting+multiple+loci+that+influence+complex+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000591-Auranen1" id="pone.0000591-Auranen1"></a>Auranen M, Varilo T, Alen R, Vanhala R, Ayers K, et al.  (2003) Evidence for allelic association on chromosome 3q25-27 in families with autism spectrum disorders originating from a subisolate of Finland. Mol Psychiatry  8: 879–884.  <ul class="find" data-citedArticleID="964294" data-doi="10.1038/sj.mp.4001299"><li><a href="http://dx.doi.org/10.1038/sj.mp.4001299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+allelic+association+on+chromosome+3q25-27+in+families+with+autism+spectrum+disorders+originating+from+a+subisolate+of+Finland." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+allelic+association+on+chromosome+3q25-27+in+families+with+autism+spectrum+disorders+originating+from+a+subisolate+of+Finland.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000591-Helgason1" id="pone.0000591-Helgason1"></a>Helgason A, Yngvadottir B, Hrafnkelsson B, Gulcher J, Stefansson K (2005) An Icelandic example of the impact of population structure on association studies. Nat Genet  37: 90–95.  <ul class="find" data-citedArticleID="964384" data-doi="10.1038/ng1492"><li><a href="http://dx.doi.org/10.1038/ng1492" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Icelandic+example+of+the+impact+of+population+structure+on+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Icelandic+example+of+the+impact+of+population+structure+on+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000591-Voight1" id="pone.0000591-Voight1"></a>Voight BF, Pritchard JK (2005) Confounding from cryptic relatedness in case-control association studies. PLoS Genet  1: e32.  <ul class="find" data-citedArticleID="964374" data-doi="10.1371/journal.pgen.0010032.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010032.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Confounding+from+cryptic+relatedness+in+case-control+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Confounding+from+cryptic+relatedness+in+case-control+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000591-Campbell1" id="pone.0000591-Campbell1"></a>Campbell CD, Ogburn EL, Lunetta KL, Lyon HN, Freedman ML, et al.  (2005) Demonstrating stratification in a European American population. Nat Genet  37: 868–872.  <ul class="find" data-citedArticleID="964366" data-doi="10.1038/ng1607"><li><a href="http://dx.doi.org/10.1038/ng1607" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Demonstrating+stratification+in+a+European+American+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Demonstrating+stratification+in+a+European+American+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000591-Aranzana1" id="pone.0000591-Aranzana1"></a>Aranzana MJ, Kim S, Zhao K, Bakker E, Horton M, et al.  (2005) Genome-wide association mapping in Arabidopsis identifies previously known flowering time and pathogen resistance genes. PLoS Genet  1: e60.  <ul class="find" data-citedArticleID="964358" data-doi="10.1371/journal.pgen.0010060.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010060.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+association+mapping+in+Arabidopsis+identifies+previously+known+flowering+time+and+pathogen+resistance+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+association+mapping+in+Arabidopsis+identifies+previously+known+flowering+time+and+pathogen+resistance+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000591-Devlin1" id="pone.0000591-Devlin1"></a>Devlin B, Roeder K (1999) Genomic control for association studies. Biometrics  55: 997–1004.  <ul class="find" data-citedArticleID="964360" data-doi="10.1111/j.0006-341x.1999.00997.x"><li><a href="http://dx.doi.org/10.1111/j.0006-341x.1999.00997.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+control+for+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+control+for+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000591-Devlin2" id="pone.0000591-Devlin2"></a>Devlin B, Roeder K, Wasserman L (2001) Genomic control, a new approach to genetic-based association studies. Theor Popul Biol  60: 155–166.  <ul class="find" data-citedArticleID="964304" data-doi="10.1006/tpbi.2001.1542"><li><a href="http://dx.doi.org/10.1006/tpbi.2001.1542" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+control%2C+a+new+approach+to+genetic-based+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+control%2C+a+new+approach+to+genetic-based+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000591-Pritchard1" id="pone.0000591-Pritchard1"></a>Pritchard JK, Stephens M, Rosenberg NA, Donnelly P (2000) Association mapping in structured populations. Am J Hum Genet  67: 170–181.  <ul class="find" data-citedArticleID="964322" data-doi="10.1086/302959"><li><a href="http://dx.doi.org/10.1086/302959" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+mapping+in+structured+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+mapping+in+structured+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000591-Thornsberry1" id="pone.0000591-Thornsberry1"></a>Thornsberry JM, Goodman MM, Doebley J, Kresovich S, Nielsen D, et al.  (2001) Dwarf8 polymorphisms associate with variation in flowering time. Nat Genet  28: 286–289.  <ul class="find" data-citedArticleID="964378" data-doi="10.1038/90135"><li><a href="http://dx.doi.org/10.1038/90135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dwarf8+polymorphisms+associate+with+variation+in+flowering+time." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dwarf8+polymorphisms+associate+with+variation+in+flowering+time.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000591-Satten1" id="pone.0000591-Satten1"></a>Satten GA, Flanders WD, Yang Q (2001) Accounting for unmeasured population substructure in case-control studies of genetic association using a novel latent-class model. Am J Hum Genet  68: 466–477.  <ul class="find" data-citedArticleID="964392" data-doi="10.1086/318195"><li><a href="http://dx.doi.org/10.1086/318195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accounting+for+unmeasured+population+substructure+in+case-control+studies+of+genetic+association+using+a+novel+latent-class+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accounting+for+unmeasured+population+substructure+in+case-control+studies+of+genetic+association+using+a+novel+latent-class+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000591-Setakis1" id="pone.0000591-Setakis1"></a>Setakis E, Stirnadel H, Balding DJ (2006) Logistic regression protects against population structure in genetic association studies. Genome Res  16: 290–296.  <ul class="find" data-citedArticleID="964320" data-doi="10.1101/gr.4346306"><li><a href="http://dx.doi.org/10.1101/gr.4346306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Logistic+regression+protects+against+population+structure+in+genetic+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Logistic+regression+protects+against+population+structure+in+genetic+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000591-Price1" id="pone.0000591-Price1"></a>Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, et al.  (2006) Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet  38: 904–909.  <ul class="find" data-citedArticleID="964332" data-doi="10.1038/ng1847"><li><a href="http://dx.doi.org/10.1038/ng1847" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000591-Kennedy1" id="pone.0000591-Kennedy1"></a>Kennedy BW, Quinton M, van Arendonk JA (1992) Estimation of effects of single genes on quantitative traits. J Anim Sci  70: 2000–2012.  <ul class="find" data-citedArticleID="964296"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Estimation+of+effects+of+single+genes+on+quantitative+traits.&amp;auth=&amp;atitle=Estimation+of+effects+of+single+genes+on+quantitative+traits." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+effects+of+single+genes+on+quantitative+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+effects+of+single+genes+on+quantitative+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000591-Yu1" id="pone.0000591-Yu1"></a>Yu J, Pressoir G, Briggs WH, Vroh Bi I, Yamasaki M, et al.  (2006) A unified mixed-model method for association mapping that accounts for multiple levels of relatedness. Nat Genet  38: 203–208.  <ul class="find" data-citedArticleID="964328" data-doi="10.1038/ng1702"><li><a href="http://dx.doi.org/10.1038/ng1702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+unified+mixed-model+method+for+association+mapping+that+accounts+for+multiple+levels+of+relatedness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+unified+mixed-model+method+for+association+mapping+that+accounts+for+multiple+levels+of+relatedness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000591-Moore1" id="pone.0000591-Moore1"></a>Moore CB, John M, James IR, Christiansen FT, Witt CS, et al.  (2002) Evidence of HIV-1 adaptation to HLA-restricted immune responses at a population level. Science  296: 1439–1443.  <ul class="find" data-citedArticleID="964312" data-doi="10.1126/science.1069660"><li><a href="http://dx.doi.org/10.1126/science.1069660" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+of+HIV-1+adaptation+to+HLA-restricted+immune+responses+at+a+population+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+of+HIV-1+adaptation+to+HLA-restricted+immune+responses+at+a+population+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000591-Heckerman1" id="pone.0000591-Heckerman1"></a>Heckerman D (1998) A tutorial on learning with Bayesian networks. Learning in graphical models, Kluwer. pp. 301–354.  <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0000591-Felsenstein2" id="pone.0000591-Felsenstein2"></a>Felsenstein J (1981) Evolutionary trees from DNA sequences: a maximum likelihood approach. J Mol Evol  17: 368–376.  <ul class="find" data-citedArticleID="964302" data-doi="10.1007/bf01734359"><li><a href="http://dx.doi.org/10.1007/bf01734359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+trees+from+DNA+sequences%3A+a+maximum+likelihood+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+trees+from+DNA+sequences%3A+a+maximum+likelihood+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000591-Press1" id="pone.0000591-Press1"></a>Press W, Teukolsky S, Vetterling W, Flannery B (1992) Numerical Recipes in C. New York: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">37.
              </span><a name="pone.0000591-Conaway1" id="pone.0000591-Conaway1"></a>Conaway MR (1990) A random effects model for binary data. Biometrics  46: 317–328.  <ul class="find" data-citedArticleID="964342" data-doi="10.2307/2531437"><li><a href="http://dx.doi.org/10.2307/2531437" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+random+effects+model+for+binary+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+random+effects+model+for+binary+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000591-Kloetzel1" id="pone.0000591-Kloetzel1"></a>Kloetzel P (2001) Antigen processing by the proteasome. Nat Rev Mol Cell Biol  2: 179–187.  <ul class="find" data-citedArticleID="964348" data-doi="10.1038/35056572"><li><a href="http://dx.doi.org/10.1038/35056572" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antigen+processing+by+the+proteasome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antigen+processing+by+the+proteasome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000591-Fodor1" id="pone.0000591-Fodor1"></a>Fodor AA, Aldrich RW (2004) Influence of conservation on calculations of amino acid covariance in multiple sequence alignments. Proteins  56: 211–221.  <ul class="find" data-citedArticleID="964362" data-doi="10.1002/prot.20098"><li><a href="http://dx.doi.org/10.1002/prot.20098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+conservation+on+calculations+of+amino+acid+covariance+in+multiple+sequence+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+conservation+on+calculations+of+amino+acid+covariance+in+multiple+sequence+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000591-Fossen1" id="pone.0000591-Fossen1"></a>Fossen T, Wray V, Bruns K, Rachmat J, Henklein P, et al.  (2005) Solution structure of the human immunodeficiency virus type 1 p6 protein. J Biol Chem  280: 42515–42527.  <ul class="find" data-citedArticleID="964326" data-doi="10.1074/jbc.m507375200"><li><a href="http://dx.doi.org/10.1074/jbc.m507375200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solution+structure+of+the+human+immunodeficiency+virus+type+1+p6+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solution+structure+of+the+human+immunodeficiency+virus+type+1+p6+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000591-Dangl1" id="pone.0000591-Dangl1"></a>Dangl JL, Jones JD (2001) Plant pathogens and integrated defence responses to infection. Nature  411: 826–833.  <ul class="find" data-citedArticleID="964352" data-doi="10.1038/35081161"><li><a href="http://dx.doi.org/10.1038/35081161" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plant+pathogens+and+integrated+defence+responses+to+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plant+pathogens+and+integrated+defence+responses+to+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000591-Harvey1" id="pone.0000591-Harvey1"></a>Harvey PH, Pagel MO (1991) The Comparative Method in Evolutionary Biology. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">43.
              </span><a name="pone.0000591-Felsenstein3" id="pone.0000591-Felsenstein3"></a>Felsenstein J (2004) Inferring Phylogenies. Sunderland, MA: Sinauer Associates, Inc.   <ul class="find-nolinks"></ul></li><li><span class="label">44.
              </span><a name="pone.0000591-Korber1" id="pone.0000591-Korber1"></a>Korber BT, Farber RM, Wolpert DH, Lapedes AS (1993) Covariation of mutations in the V3 loop of human immunodeficiency virus type 1 envelope protein: an information theoretic analysis. Proc Natl Acad Sci USA  90: 7176–7180.  <ul class="find" data-citedArticleID="964340" data-doi="10.1073/pnas.90.15.7176"><li><a href="http://dx.doi.org/10.1073/pnas.90.15.7176" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Covariation+of+mutations+in+the+V3+loop+of+human+immunodeficiency+virus+type+1+envelope+protein%3A+an+information+theoretic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Covariation+of+mutations+in+the+V3+loop+of+human+immunodeficiency+virus+type+1+envelope+protein%3A+an+information+theoretic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000591-Gobel1" id="pone.0000591-Gobel1"></a>Gobel U, Sander C, Schneider R, Valencia A (1994) Correlated mutations and residue contacts in proteins. Proteins  18: 309–317.  <ul class="find" data-citedArticleID="964308" data-doi="10.1002/prot.340180402"><li><a href="http://dx.doi.org/10.1002/prot.340180402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlated+mutations+and+residue+contacts+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlated+mutations+and+residue+contacts+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000591-Taylor1" id="pone.0000591-Taylor1"></a>Taylor WR, Hatrick K (1994) Compensating changes in protein multiple sequence alignments. Protein Eng  7: 341–348.  <ul class="find" data-citedArticleID="964316" data-doi="10.1093/protein/7.3.341"><li><a href="http://dx.doi.org/10.1093/protein/7.3.341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compensating+changes+in+protein+multiple+sequence+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compensating+changes+in+protein+multiple+sequence+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000591-Neher1" id="pone.0000591-Neher1"></a>Neher E (1994) How frequent are correlated changes in families of protein sequences? Proc Natl Acad Sci USA  91: 98–102.  <ul class="find" data-citedArticleID="964388" data-doi="10.1073/pnas.91.1.98"><li><a href="http://dx.doi.org/10.1073/pnas.91.1.98" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+frequent+are+correlated+changes+in+families+of+protein+sequences%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+frequent+are+correlated+changes+in+families+of+protein+sequences%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000591-Pollock2" id="pone.0000591-Pollock2"></a>Pollock DD, Taylor WR (1997) Effectiveness of correlation analysis in identifying protein residues undergoing correlated evolution. Protein Eng  10: 647–657.  <ul class="find" data-citedArticleID="964372" data-doi="10.1093/protein/10.6.647"><li><a href="http://dx.doi.org/10.1093/protein/10.6.647" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effectiveness+of+correlation+analysis+in+identifying+protein+residues+undergoing+correlated+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effectiveness+of+correlation+analysis+in+identifying+protein+residues+undergoing+correlated+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000591-Noivirt1" id="pone.0000591-Noivirt1"></a>Noivirt O, Eisenstein M, Horovitz A (2005) Detection and reduction of evolutionary noise in correlated mutation analysis. Protein Eng Des Sel  18: 247–253.  <ul class="find" data-citedArticleID="964300" data-doi="10.1093/protein/gzi029"><li><a href="http://dx.doi.org/10.1093/protein/gzi029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+and+reduction+of+evolutionary+noise+in+correlated+mutation+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+and+reduction+of+evolutionary+noise+in+correlated+mutation+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000591-Kiepiela1" id="pone.0000591-Kiepiela1"></a>Kiepiela P, Leslie AJ, Honeyborne I, Ramduth D, Thobakgale C, et al.  (2004) Dominant influence of HLA-B in mediating the potential co-evolution of HIV and HLA. Nature  432: 769–775.  <ul class="find" data-citedArticleID="964290" data-doi="10.1038/nature03113"><li><a href="http://dx.doi.org/10.1038/nature03113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dominant+influence+of+HLA-B+in+mediating+the+potential+co-evolution+of+HIV+and+HLA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dominant+influence+of+HLA-B+in+mediating+the+potential+co-evolution+of+HIV+and+HLA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000591-Guindon1" id="pone.0000591-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol  52: 696–704.  <ul class="find" data-citedArticleID="964336"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000591-Felsenstein4" id="pone.0000591-Felsenstein4"></a>Felsenstein J (2005) PHYLIP (Phylogeny Inference Package) version 3.6.   Technical report, Department of Genome Sciences, University of Washington, Seattle, WA.  <ul class="find-nolinks"></ul></li><li><span class="label">53.
              </span><a name="pone.0000591-Benjamini1" id="pone.0000591-Benjamini1"></a>Benjamini Y, Hochberg Y (1995) Controlling the false discovery rate: a practical and powerful approach to multiple testing. J R Stat Soc B  57: 289–300.  <ul class="find" data-citedArticleID="964376"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing.&amp;auth=&amp;atitle=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000591-Storey1" id="pone.0000591-Storey1"></a>Storey JD, Tibshirani R (2003) Statistical significance for genomewide studies. Proc Natl Acad Sci USA  100: 9440–9445.  <ul class="find" data-citedArticleID="964368" data-doi="10.1073/pnas.1530509100"><li><a href="http://dx.doi.org/10.1073/pnas.1530509100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+significance+for+genomewide+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+significance+for+genomewide+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000591-Nordborg1" id="pone.0000591-Nordborg1"></a>Nordborg M, Borevitz JO, Bergelson J, Berry CC, Chory J, et al.  (2002) The extent of linkage disequilibrium in Arabidopsis thaliana. Nat Genet  30: 190–193.  <ul class="find" data-citedArticleID="964382" data-doi="10.1038/ng813"><li><a href="http://dx.doi.org/10.1038/ng813" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+extent+of+linkage+disequilibrium+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+extent+of+linkage+disequilibrium+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.XML" value="131612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.PDF" value="697004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g001.PNG_L" value="164092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g001.PNG_M" value="418676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g001.PNG_S" value="13996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g001.TIF" value="1226648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g001.PNG_I" value="23181"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e001.PNG" value="14293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e001.TIF" value="29104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g002.PNG_L" value="461432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g002.PNG_M" value="291507"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g002.PNG_S" value="14752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g002.TIF" value="1457028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g002.PNG_I" value="27719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e002.PNG" value="6559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e002.TIF" value="21084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e003.PNG" value="6545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e003.TIF" value="21064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e004.PNG" value="22185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e004.TIF" value="36316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e005.PNG" value="11334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e005.TIF" value="25872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g003.PNG_L" value="193785"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g003.PNG_M" value="198387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g003.PNG_S" value="17009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g003.TIF" value="471566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g003.PNG_I" value="31083"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g004.PNG_L" value="290933"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g004.PNG_M" value="257124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g004.PNG_S" value="20207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g004.TIF" value="568676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g004.PNG_I" value="40335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g005.PNG_L" value="350926"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g005.PNG_M" value="455125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g005.PNG_S" value="21057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g005.TIF" value="1140816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g005.PNG_I" value="41349"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g006.PNG_L" value="420949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g006.PNG_M" value="274693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g006.PNG_S" value="19956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g006.TIF" value="784548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g006.PNG_I" value="35779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g007.PNG_L" value="432346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g007.PNG_M" value="535984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g007.PNG_S" value="22044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g007.TIF" value="1252996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g007.PNG_I" value="47970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g008.PNG_L" value="565893"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g008.PNG_M" value="338209"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g008.PNG_S" value="21933"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g008.TIF" value="979746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g008.PNG_I" value="45530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g009.PNG_L" value="332944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g009.PNG_M" value="655033"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g009.PNG_S" value="17485"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g009.TIF" value="1508198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g009.PNG_I" value="47210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.t001.PNG_L" value="52249"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.t001.PNG_M" value="156500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.t001.PNG_S" value="19148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.t001.TIF" value="254618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.t001.PNG_I" value="23749"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g010.PNG_L" value="10837603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g010.PNG_M" value="938452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g010.PNG_S" value="26202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g010.TIF" value="17482116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g010.PNG_I" value="142403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g011.PNG_L" value="1389079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g011.PNG_M" value="459660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g011.PNG_S" value="19519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g011.TIF" value="3773146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.g011.PNG_I" value="45710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e006.PNG" value="19658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.e006.TIF" value="33544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000591.s001.XLS" value="364544"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000591&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000591" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000591&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000591&volume=&issue=&title=Leveraging Hierarchical Population Structure in Discrete Association Studies&author_name=Jonathan%20Carlson%2C%20Carl%20Kadie%2C%20Simon%20Mallal%2C%20David%20Heckerman&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000591" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000591" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000591" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000591&amp;t=Leveraging%20Hierarchical%20Population%20Structure%20in%20Discrete%20Association%20Studies" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000591&title=Leveraging%20Hierarchical%20Population%20Structure%20in%20Discrete%20Association%20Studies&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000591&amp;title=Leveraging%20Hierarchical%20Population%20Structure%20in%20Discrete%20Association%20Studies" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000591" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000591&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Leveraging Hierarchical Population Structure in Discrete Association Studies';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000591';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Leveraging%20Hierarchical%20Population%20Structure%20in%20Discrete%20Association%20Studies http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000591" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000591" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">






















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arabidopsis+thaliana%22" title="Search for articles in the subject area:'Arabidopsis thaliana'"><div class="flagText">Arabidopsis thaliana</div></a>
              <div data-categoryid="32845" data-articleid="24020"
                   data-categoryname="Arabidopsis thaliana"
                   class="flagImage" title="Flag 'Arabidopsis thaliana' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Coevolution%22" title="Search for articles in the subject area:'Coevolution'"><div class="flagText">Coevolution</div></a>
              <div data-categoryid="18077" data-articleid="24020"
                   data-categoryname="Coevolution"
                   class="flagImage" title="Flag 'Coevolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Data+processing%22" title="Search for articles in the subject area:'Data processing'"><div class="flagText">Data processing</div></a>
              <div data-categoryid="18073" data-articleid="24020"
                   data-categoryname="Data processing"
                   class="flagImage" title="Flag 'Data processing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV%22" title="Search for articles in the subject area:'HIV'"><div class="flagText">HIV</div></a>
              <div data-categoryid="42663" data-articleid="24020"
                   data-categoryname="HIV"
                   class="flagImage" title="Flag 'HIV' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42865" data-articleid="24020"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="24020"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24020"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="24020"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=42'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=967'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5474&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
