

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Combinatorial Computational Approaches to Identify Tetracycline Derivatives as Flavivirus Inhibitors</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000428"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000428" />

    <meta name="citation_title" content="Combinatorial Computational Approaches to Identify Tetracycline Derivatives as Flavivirus Inhibitors"/>
    <meta itemprop="name" content="Combinatorial Computational Approaches to Identify Tetracycline Derivatives as Flavivirus Inhibitors"/>

      <meta name="citation_author" content="Jinn-Moon Yang"/>
            <meta name="citation_author_institution" content="Department of Biological Science and Technology, National Chiao Tung University, Hsinchu, Taiwan"/>
            <meta name="citation_author_institution" content="Institute of Bioinformatics, National Chiao Tung University, Hsinchu, Taiwan"/>
      <meta name="citation_author" content="Yan-Fu Chen"/>
            <meta name="citation_author_institution" content="Department of Biological Science and Technology, National Chiao Tung University, Hsinchu, Taiwan"/>
      <meta name="citation_author" content="Yu-Yin Tu"/>
            <meta name="citation_author_institution" content="Department of Biological Science and Technology, National Chiao Tung University, Hsinchu, Taiwan"/>
      <meta name="citation_author" content="Kuei-Rong Yen"/>
            <meta name="citation_author_institution" content="Department of Biological Science and Technology, National Chiao Tung University, Hsinchu, Taiwan"/>
      <meta name="citation_author" content="Yun-Liang Yang"/>
            <meta name="citation_author_institution" content="Department of Biological Science and Technology, National Chiao Tung University, Hsinchu, Taiwan"/>

    <meta name="citation_date" content="2007/5/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000428.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e428"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=West Nile virus: Epidemiology and clinical features of an emerging epidemic in the United States.; citation_author=EB Hayes; citation_author=DJ Gubler; citation_journal_title=Annual Review of Medicine; citation_volume=57; citation_number=1; citation_pages=181-194; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The molecular biology of West Nile virus: A new invader of the Western hemisphere.; citation_author=MA Brinton; citation_journal_title=Annual Review of Microbiology; citation_volume=56; citation_number=2; citation_pages=371-402; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Perspectives for the treatment of infections with Flaviviridae.; citation_author=P Leyssen; citation_author=ED Clercq; citation_author=J Neyts; citation_journal_title=Clinical Microbiology Reviews; citation_volume=13; citation_number=3; citation_pages=67-82; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Dengue and dengue hemorrhagic hever: its history and resurgence as a global public health problem.; citation_author=DJ Gubler; citation_author=DJ Gubler; citation_number=4; citation_pages=1-22: New York; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A ligand-binding pocket in the dengue virus envelope glycoprotein.; citation_author=Y Modis; citation_author=S Ogata; citation_author=D Clements; citation_author=SC Harrison; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=100; citation_number=5; citation_pages=6986-6991; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structure-based virtual screening: an overview.; citation_author=PD Lyne; citation_journal_title=Drug Discovery Today; citation_volume=7; citation_number=6; citation_pages=1047-1055; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Lead discovery using molecular docking.; citation_author=BK Shoichet; citation_author=SL McGovern; citation_author=B Wei; citation_author=J Irwin; citation_journal_title=Current Opinion in Chemical Biology; citation_volume=6; citation_number=7; citation_pages=439-446; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Virtual screening of chemical libraries.; citation_author=BK Shoichet; citation_journal_title=Nature; citation_volume=432; citation_number=8; citation_pages=862-865; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structure-based virtual screening of chemical libraries for drug discovery.; citation_author=S Ghosh; citation_author=AH Nie; citation_author=J An; citation_author=ZW Huang; citation_journal_title=Current Opinion in Chemical Biology; citation_volume=10; citation_number=9; citation_pages=194-202; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Flaviviruses.; citation_author=TP Monath; citation_author=FX Henize; citation_number=10; citation_pages=961-1034; citation_date=1996; citation_publisher=Lipponcott-Raven; " />
      <meta name="citation_reference" content="citation_title=Structure of the dengue virus envelope protein after membrane fusion.; citation_author=Y Modis; citation_author=S Ogata; citation_author=D Clements; citation_author=SC Harrison; citation_journal_title=Nature; citation_volume=427; citation_number=11; citation_pages=313-319; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Changes in the dengue virus major envelope protein on passaging and their localization on the three-dimensional structure of the protein.; citation_author=E Lee; citation_author=RC Weir; citation_author=L Dalgarno; citation_journal_title=Virology; citation_volume=232; citation_number=12; citation_pages=281-290; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The envelope glycoprotein from tick-borne encephalitis virus at 2 A resolution.; citation_author=FA Rey; citation_author=FX Heinz; citation_author=C Mandl; citation_author=C Kunz; citation_author=SC Harrison; citation_journal_title=Nature; citation_volume=375; citation_number=13; citation_pages=291-298; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Consensus scoring sriteria for Improving enrichment in virtual screening.; citation_author=J-M Yang; citation_author=Y-F Chen; citation_author=T-W Shen; citation_author=BS Kristal; citation_author=DF Hsu; citation_journal_title=Journal of Chemical Information and Modeling; citation_volume=45; citation_number=14; citation_pages=1134-1146; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=GEMDOCK: a generic evolutionary method for molecular docking.; citation_author=J-M Yang; citation_author=C-C Chen; citation_journal_title=Proteins: Structure, Function, and Bioinformatics; citation_volume=55; citation_number=15; citation_pages=288-304; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A pharmacophore-based evolutionary approach for screening selective estrogen receptor modulators.; citation_author=J-M Yang; citation_author=T-W Shen; citation_journal_title=Proteins: Structure, Function, and Bioinformatics; citation_volume=59; citation_number=16; citation_pages=205-220; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Development and evaluation of a generic evolutionary method for protein-ligand docking.; citation_author=J-M Yang; citation_journal_title=Journal of Computational Chemistry; citation_volume=25; citation_number=17; citation_pages=843-857; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The structural basis for the action of the antibiotics tetracycline, pactamycin, and hygromycin B on the 30S ribosomal subunit.; citation_author=DE Brodersen; citation_author=JWM Clemons; citation_author=AP Carter; citation_author=RJ Morgan-Warren; citation_author=BT Wimberly; citation_journal_title=Cell; citation_volume=103; citation_number=18; citation_pages=1143-1154; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Tetracycline antibiotics: mode of action, applications, molecular biology, and epidemiology of bacterial resistance.; citation_author=I Chopra; citation_author=M Roberts; citation_journal_title=Microbiology and Molecular Biology Reviews; citation_volume=65; citation_number=19; citation_pages=232-260; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Quantitative structure-activity relationship analysis of functionalized amino acid anticonvulsant agents using k nearest neighbor and simulated annealing PLS methods.; citation_author=M Shen; citation_author=A LeTiran; citation_author=Y Xiao; citation_author=A Golbraikh; citation_author=H Kohn; citation_journal_title=Journal of Medicinal Chemistry; citation_volume=45; citation_number=20; citation_pages=2811-2823; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Surflex: fully automatic flexible molecular docking using a molecular similarity-based search engine.; citation_author=AJ Jain; citation_journal_title=Journal of Medicinal Chemistry; citation_volume=46; citation_number=21; citation_pages=499-511; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Epitopes on the dengue 1 virus envelope protein recognized by neutralizing IgM monoclonal antibodies.; citation_author=DWC Beasley; citation_author=JG Aaskov; citation_journal_title=Virology; citation_volume=279; citation_number=22; citation_pages=447-458; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Attenuation of Murray Valley encephalitis virus by site-directed mutagenesis of the hinge and putative receptor-binding regions of the envelope protein.; citation_author=RJ Hurrelbrink; citation_author=PC McMinn; citation_journal_title=Journal of Virology; citation_volume=75; citation_number=23; citation_pages=7692-7702; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A single amino acid substitution in the envelope protein of chimeric yellow fever-dengue 1 vaccine virus reduces neurovirulence for suckling mice and viremia/viscerotropism for monkeys.; citation_author=F Guirakhoo; citation_author=Z Zhang; citation_author=G Myers; citation_author=BW Johnson; citation_author=K Pugachev; citation_journal_title=Journal of Virology; citation_volume=78; citation_number=24; citation_pages=9998-10008; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of epitopes on the envelope (E) protein of dengue 2 and dengue 3 viruses using monoclonal antibodies.; citation_author=IL Serafin; citation_author=JG Aaskov; citation_journal_title=Archives of Virology; citation_volume=146; citation_number=25; citation_pages=2469-2479; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Single mutation in the flavivirus envelope protein hinge region increases neurovirulence for mice and monkeys but decreases viscerotropism for monkeys: relevance to development and safety testing of live, attenuated vaccines.; citation_author=TP Monath; citation_author=J Arroyo; citation_author=I Levenbook; citation_author=ZX Zhang; citation_author=J Catalan; citation_journal_title=Journal of Virology; citation_volume=76; citation_number=26; citation_pages=1932-1943; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Ribosomal protection proteins and their mechanism of tetracycline resistance.; citation_author=SR Connell; citation_author=DM Tracz; citation_author=KH Nierhaus; citation_author=DE Taylor; citation_journal_title=Antimicrobial Agents and Chemotherapy; citation_volume=47; citation_number=27; citation_pages=3675-3681; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structural basis of gene regulation by the tetracycline inducible Tetrepressor–operator system.; citation_author=P Orth; citation_author=D Schnappinger; citation_author=W Hillen; citation_author=W Saenger; citation_author=W Hinrichs; citation_journal_title=Nature; citation_volume=7; citation_number=28; citation_pages=215-219; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Q-SiteFinder: an energy-based method for the prediction of protein-ligand binding sites.; citation_author=AT Laurie; citation_author=RM Jackson; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=29; citation_pages=1908-1916; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Development and validation of a genetic algorithm for flexible docking.; citation_author=G Jones; citation_author=P Willett; citation_author=RC Glen; citation_author=AR Leach; citation_author=R Taylor; citation_journal_title=Journal of Molecular Biology; citation_volume=267; citation_number=30; citation_pages=727-748.31; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Evaluation of the flexX incremental construction algorithm for protein-ligand docking.; citation_author=B Kramer; citation_author=M Rarey; citation_author=T Lengauer; citation_journal_title=Proteins: Structure, Function, and Bioinformatics; citation_volume=37; citation_number=31; citation_pages=228-241; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Protein-based virtual screening of chemical databases. 1. Evaluation of different docking/scoring combinations.; citation_author=C Bissantz; citation_author=G Folkers; citation_author=D Rognan; citation_journal_title=Journal of Medical Chemistry; citation_volume=14; citation_number=32; citation_pages=4759-4767; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=An evolutionary approach with pharmacophore-based scoring functions for virtual database screening.; citation_author=J-M Yang; citation_author=T-W Shen; citation_author=Y-F Chen; citation_author=Y-Y Chiu; citation_journal_title=Lecture Notes in Computer Science; citation_volume=3102; citation_number=33; citation_pages=481-492; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Blocking the dengue virus 2 infections on BHK-21 cells with purified recombinant dengue virus 2 E protein expressed in Escherichia coli.; citation_author=M-W Chiu; citation_author=Y-L Yang; citation_journal_title=Biochemical and Biophysical Research Communications; citation_volume=309; citation_number=34; citation_pages=672-678; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000428" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Combinatorial Computational Approaches to Identify Tetracycline Derivatives as Flavivirus Inhibitors"/>
    <meta name="twitter:description" content="Limited structural information of drug targets, cellular toxicity possessed by lead compounds, and large amounts of potential leads are the major issues facing the design-oriented approach of discovering new leads. In an attempt to tackle these issues, we have developed a process of virtual screening based on the observation that conformational rearrangements of the dengue virus envelope protein are essential for the mediation of viral entry into host cells via membrane fusion. Screening was based solely on the structural information of the Dengue virus envelope protein and was focused on a target site that is presumably important for the conformational rearrangements necessary for viral entry. To circumvent the issue of lead compound toxicity, we performed screening based on molecular docking using structural databases of medical compounds. To enhance the identification of hits, we further categorized and selected candidates according to their novel structural characteristics. Finally, the selected candidates were subjected to a biological validation assay to assess inhibition of Dengue virus propagation in mammalian host cells using a plaque formation assay. Among the 10 compounds examined, rolitetracycline and doxycycline significantly inhibited plaque formation, demonstrating their inhibitory effect on dengue virus propagation. Both compounds were tetracycline derivatives with IC50s estimated to be 67.1 &micro;M and 55.6 &micro;M, respectively. Their docked conformations displayed common hydrophobic interactions with critical residues that affected membrane fusion during viral entry. These interactions will therefore position the tetracyclic ring moieties of both inhibitors to bind firmly to the target and, subsequently, disrupt conformational rearrangement and block viral entry. This process can be applied to other drug targets in which conformational rearrangement is critical to function."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000428.g007"/>

  <meta property="og:title" content="Combinatorial Computational Approaches to Identify Tetracycline Derivatives as Flavivirus Inhibitors" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000428" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6636'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8178'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1091&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000428">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000428" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000428&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Combinatorial Computational Approaches to Identify Tetracycline Derivatives as Flavivirus Inhibitors
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jinn-Moon Yang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Biological Science and Technology, National Chiao Tung University, Hsinchu, Taiwan, 
                  Institute of Bioinformatics, National Chiao Tung University, Hsinchu, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yan-Fu Chen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Science and Technology, National Chiao Tung University, Hsinchu, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yu-Yin Tu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Science and Technology, National Chiao Tung University, Hsinchu, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kuei-Rong Yen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Science and Technology, National Chiao Tung University, Hsinchu, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yun-Liang Yang
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:yyang@mail.nctu.edu.tw">yyang@mail.nctu.edu.tw</a></p>

                <p>Affiliation:
                  Department of Biological Science and Technology, National Chiao Tung University, Hsinchu, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 09, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000428</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000428" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000428" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000428" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000428" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000428-g001" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000428-g002" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000428-g003" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000428-g004" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000428-g005" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000428-g006" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000428-g007" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000428">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000428" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Limited structural information of drug targets, cellular toxicity possessed by lead compounds, and large amounts of potential leads are the major issues facing the design-oriented approach of discovering new leads. In an attempt to tackle these issues, we have developed a process of virtual screening based on the observation that conformational rearrangements of the dengue virus envelope protein are essential for the mediation of viral entry into host cells via membrane fusion. Screening was based solely on the structural information of the Dengue virus envelope protein and was focused on a target site that is presumably important for the conformational rearrangements necessary for viral entry. To circumvent the issue of lead compound toxicity, we performed screening based on molecular docking using structural databases of medical compounds. To enhance the identification of hits, we further categorized and selected candidates according to their novel structural characteristics. Finally, the selected candidates were subjected to a biological validation assay to assess inhibition of Dengue virus propagation in mammalian host cells using a plaque formation assay. Among the 10 compounds examined, rolitetracycline and doxycycline significantly inhibited plaque formation, demonstrating their inhibitory effect on dengue virus propagation. Both compounds were tetracycline derivatives with IC<sub>50</sub>s estimated to be 67.1 µM and 55.6 µM, respectively. Their docked conformations displayed common hydrophobic interactions with critical residues that affected membrane fusion during viral entry. These interactions will therefore position the tetracyclic ring moieties of both inhibitors to bind firmly to the target and, subsequently, disrupt conformational rearrangement and block viral entry. This process can be applied to other drug targets in which conformational rearrangement is critical to function.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Yang J-M, Chen Y-F, Tu Y-Y, Yen K-R, Yang Y-L (2007) Combinatorial Computational Approaches to Identify Tetracycline Derivatives as Flavivirus Inhibitors. PLoS ONE 2(5):
          e428.
            doi:10.1371/journal.pone.0000428</p><p><strong>Academic Editor: </strong>Martin Egli, Vanderbilt University, United States of America</p><p><strong>Received:</strong> February 9, 2007; <strong>Accepted:</strong> April 13, 2007; <strong>Published:</strong> May 9, 2007</p><p><strong>Copyright:</strong> © 2007 Yang et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Y.-L. Yang, Y.-Y. Tu, and K.-R. Yen, are supported by National Chiao Tung University under the MoE ATU plan, grant 95W821, and Academia Sinica under grant AS91IMB7PP. J.-M. Yang and Y.-F. Chen are both supported by the National Science Council and a grant from the University System at Taiwan-Veteran General Hospital.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The Dengue virus (DV) belongs to the Flavivirus family and has become a major threat to public health globally, especially in tropical and subtropical areas, due to the increases in population density and environmental changes. There are approximately 2.5 billion people who live under the shadow of DV infection. Other well-known Flaviviruses include yellow fever virus, Japanese encephalitis virus, West Nile virus <a href="#pone.0000428-Hayes1">[1]</a>, <a href="#pone.0000428-Brinton1">[2]</a>, and Murray Valley encephalitis virus <a href="#pone.0000428-Leyssen1">[3]</a>. The Dengue virus has four serotypes and is transmitted by <em>Aedes</em> mosquitoes. Patients with DV infection show various clinical symptoms that range from no significant illness or mild fever to life-threatening Dengue hemorrhagic fever (DHF) and Dengue shock syndrome (DSS) <a href="#pone.0000428-Gubler1">[4]</a>. Currently, only supportive treatments are available. Although considerable research has been directed towards the development of a safe and effective DV vaccine since the mid-20th century, there are no approved commercial products available <a href="#pone.0000428-Modis1">[5]</a>. Therefore, to combat DV and other related viral diseases, it is advisable to develop novel strategies for discovering new antiviral agents. Recent progress in the biology has brought with it many protein structures for virtual screening (VS) as drug targets <a href="#pone.0000428-Lyne1">[6]</a>–<a href="#pone.0000428-Ghosh1">[9]</a>. However, without a previously validated target site on the targeted protein as a reference point, the number of lead candidates obtained from this type of screening is very large. Cellular toxicity further complicates biological activity assays as well. Therefore, the utilization of VS is somewhat hindered by the processes that follow, namely, the labor-intense, time-consuming verification process and the toxicity assays required for processing large amounts of lead candidates. Here, in an attempt to devise a less resource-demanding screening process, we have focused on computational approaches that are solely based on the structures of a designated region of the target protein. Then, we performed VS on a set of medical compounds because we recognized that using medical compounds could potentially minimize cellular toxicity. To reduce the number of lead candidates, we further refined the VS output by structural clustering for the identification of novel structural characteristics. Compounds with novel structures were then subjected to a biological assay to validate their activities. In summary, we sacrificed the diversity of leads in exchange for the efficiency of screening.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The DV envelope (E) protein is 495 amino acids in length, forms oligomers, and, along with the M protein, constitutes most of the accessible virion surface that is covered by the envelope membrane. The E protein is responsible for activating “membrane fusion”, the central molecular event during the entry of enveloped RNA viruses into host cells. The Dengue virus enters a host cell when the E protein binds to the virus receptor <a href="#pone.0000428-Monath1">[10]</a> on the host cell surface and activates its conformational rearrangement, causing the E protein in its dimeric pre-fusion form to transform into a trimeric post-fusion structure. This essentially irreversible conformational change induces the fusion between the viral envelope membrane and the host cell membrane <a href="#pone.0000428-Modis2">[11]</a>, allowing entry to be completed. In short, the DV E protein mediates host cell binding and is essential for infection via a conformation-induced membrane fusion event between the host cell and the virion. In addition, it is also the primary antigen that induces protective immunity and the major antigen for virus neutralization <a href="#pone.0000428-Monath1">[10]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The crystal structures of the E protein of DV type 2 in both the presence (pre-fusion) and absence (post-fusion) of a bound ligand were deposited in the Protein Data Bank {PDB codes 1oke <a href="#pone.0000428-Modis1">[5]</a> and 1ok8 <a href="#pone.0000428-Modis2">[11]</a>, respectively; <a href="#pone-0000428-g001">Figure 1</a>). The key difference between these two structures is a local rearrangement of the “<em>kl</em>” β-hairpin (residues 268–280) and the concomitant opening up of a hydrophobic pocket for ligand binding. For example, the detergent <em>n</em>-octyl-β-D-glucoside (BOG) can occupy this pocket <a href="#pone.0000428-Modis2">[11]</a>. Mutations that affect the pH threshold for membrane fusion have also been mapped to this hydrophobic pocket <a href="#pone.0000428-Lee1">[12]</a>, <a href="#pone.0000428-Rey1">[13]</a>. Therefore, Modis et al. proposed that this pocket was a hinge point in the fusion-activating conformational change and suggested that it could be a target site for the development of fusion inhibitors <a href="#pone.0000428-Modis1">[5]</a>, <a href="#pone.0000428-Modis2">[11]</a> that could disrupt or even block the correct conformational changes necessary for DV entry. This concept made the utilization of structure-based VS to identify inhibitors of DV infection plausible.</p>
<div class="figure" id="pone-0000428-g001"><div class="img"><a name="pone-0000428-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Pre-fusion (PDB code 1oke) and post-fusion (PDB code 1ok8) conformations of Dengue virus E protein and the ligand-binding pocket for virtual screening.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>(A) Dengue virus E protein structures in pre-fusion (gray) and post-fusion (blue) states and the position of the binding regions (black strand representing <em>D′</em>, <em>k</em>, <em>l</em> in pre-fusion state, colored strand representing the post-fusion state). (B) The conformation rearrangement of the binding areas. Higher-order structures and domains I, II, and III are defined according to Modis et al. <a href="#pone.0000428-Modis2">[11]</a>.</p>
<span>doi:10.1371/journal.pone.0000428.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Therefore, in this study, a well-developed docking tool, GEMDOCK <a href="#pone.0000428-Yang1">[14]</a>–<a href="#pone.0000428-Yang4">[17]</a>, was utilized to perform VS on the Comprehensive Medicinal Chemistry (CMC) database for substances that could dock in this hydrophobic pocket of E proteins <a href="#pone.0000428-Modis1">[5]</a>. These compounds were then selectively tested, based on distinct structural characteristics, for the inhibition of DV propagation. We have now successfully identified two tetracycline derivatives <a href="#pone.0000428-Brodersen1">[18]</a>, <a href="#pone.0000428-Chopra1">[19]</a> that displayed significant inhibitory effects on the propagation of the DV type 2 PL046 strain in cell cultures. According to the docked conformations of these two active, and of two inactive tetracycline-derived compounds, we have proposed a model for the inhibition of DV E protein conformational change, which may provide a future direction for lead compound optimization.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Virtual screening for inhibitors of the E protein</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To assess the VS program, we first evaluated the docking accuracy of GEMDOCK for the DV E protein by docking the detergent ligand (BOG) into the binding site. The docked conformation of BOG (<a href="#pone-0000428-g002">Figure 2A</a>) with the lowest scoring value was compared with the crystal structure of BOG based on the root mean square deviation (RMSD) of heavy atoms. The average RMSD of 10 independent runs was less than 1.20 Å. Molecular recognition of the E protein was also investigated to determine the constraints of the ligand and pharmacophore preferences during the VS. This detergent-binding pocket, located at the juxtaposition of domains I and II of the E protein, is hydrophobic in the pocket <a href="#pone.0000428-Modis1">[5]</a>, <a href="#pone.0000428-Modis2">[11]</a> and hydrophilic on both sides of the protein surface.</p>
<div class="figure" id="pone-0000428-g002"><div class="img"><a name="pone-0000428-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Docked conformations of the candidate compounds in the BOG binding site according to GEMDOCK.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The residues affecting the pH threshold of fusion are indicated. (A) The crystal conformation is shown in the CPK model (i.e., oxygen in red, nitrogen in blue, and carbon in gray), and the docked conformation of the BOG is shown in yellow. The formation of the hydrogen bonds is shown by a green dashed line. The RMSD of the conformations is 1.20 Å, and both pre-fusion and post-fusion conformations form hydrogen bonds with Glu49 and Gln271. (B) The docked conformations of the 10 selected compounds are shown. The four tetracycline derivatives are colored (doxycycline in green, rolitetracycline in blue, tetracycline in orange, and oxytetracycline in red). The inhibitory compounds (doxycycline and rolitetracycline) are docked in the vicinity of residues Thr48, Glu49, Ala50, Lys51, and Gln52.</p>
<span>doi:10.1371/journal.pone.0000428.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>GEMDOCK was then used to perform VS on the DV E protein using a screening set from the CMC database that contained 5,331 molecules between 200 and 800 Daltons. Since the binding site of the DV E protein is hydrophobic, we set the electrostatic constraint, based on the upper bound number of charged atoms, to 0 and the hydrophilic constraint, based on the upper bound fraction of polar atoms, to 0.3 (equation 4 in <a href="#s4">Materials and Methods</a>) to reduce the effects of GEMDOCK bias toward charged polar compounds. The ligand preference served as a hydrophilic filter and penalized compounds that had high hydrophilicity. Since our previous studies indicated that the ligand and pharmacophore preferences contributed to improvements in the enrichment of VS <a href="#pone.0000428-Yang1">[14]</a>, <a href="#pone.0000428-Yang3">[16]</a>, we used the scoring values of both the empirical scoring function and pharmacophore-based scoring function as ranking conditions to identify inhibitor candidates of the DV E protein.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We selected the top-ranking 3% of compounds (~173 compounds, see <a href="#pone.0000428.s005">Appendix S1</a> in supporting material) for further analyses to enrich the hit rate after GEMDOCK screening. These candidate compounds were then clustered using a hierarchical cluster method based on both their two-dimensional compound structures and protein-ligand interactions <a href="#pone.0000428-Shen1">[20]</a>, <a href="#pone.0000428-Jain1">[21]</a>, similar to Jain's work <a href="#pone.0000428-Beasley1">[22]</a>. Here, atomic environments <a href="#pone.0000428-Shen1">[20]</a>, <a href="#pone.0000428-Jain1">[21]</a> were used to represent the two-dimensional compound structure for measurements of compound similarities and the protein-ligand interactions were used for the identification of docked positions and hot spots. Based on structural similarities, docked positions, protein-ligand interactions, and the limitations of commercial availability, two groups of structures (<a href="#pone-0000428-g003">Figure 3</a>) distinguished themselves for use in the <em>in vivo</em> plaque formation assay for their potential inhibitory effects on DV propagation in cultured cells. One group consisted of two tetracycline derivatives (tetracycline and rolitetracycline) and the other group consisted of connected ring structures with additional flexibility. To enrich possible hits, two more tetracycline derivatives (doxycycline and oxytetracycline) under similar atomic conditions were also included for the biological activity assay. Docked conformations of these selected compounds are shown in <a href="#pone-0000428-g002">Figure 2B</a> and the four tetracycline derivatives are indicated as blue (rolitetracycline), green (doxycycline), orange (tetracycline), and red (oxytetracycline). As shown in <a href="#pone-0000428-g002">Figure 2A</a>, BOG is docked in the pocket and is situated centrally among Gly275, Lys128, Leu277, and Gln52. All ten selected candidate compounds were able to dock in the pocket at various locations (<a href="#pone-0000428-g002">Figure 2B</a>).</p>
<div class="figure" id="pone-0000428-g003"><div class="img"><a name="pone-0000428-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>The ten compounds selected for competitive blocking assay of DV propagation.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>1, rolitetracycline; 2, doxycycline; 3, tetracycline, 4, oxytetracycline; 5, kanamycin; 6, proscillaridin; 7, astemizole; 8, ergosterol; 9, glipizide; 10, terfenadine.</p>
<span>doi:10.1371/journal.pone.0000428.g003</span></div>

<h4><em>In vivo</em> plaque formation assay</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To assess whether those individual compounds obtained by screening could indeed affect the propagation of Dengue virus replication as predicted, different concentrations of the compounds were added separately to cultures of BHK-21 cells, followed immediately by the addition of the DV type 2 PL046 strain at a fixed number of plaque forming units (PFUs). If the compounds bind to the E proteins as predicted, they may interfere with the interactions between the E protein and the host surface receptor, particularly with the E protein conformational change that is necessary to activate viral entry. This inhibition would reduce the frequency of DV infection in BHK-21 cells. Since every successful infection leads to the formation of a plaque, the number of plaques on the assay plate indicates the number of infection events. As a fixed number of PFUs was originally added to the culture, the reduction in the number of plaques reflected the portion of the virion infection that was inhibited by the presence of the particular compound. Therefore, using the number of PFUs from the culture plates added only media (no compounds) as 100%, the relative percentage of the PFUs from the culture plates with compounds was calculated. Of the 10 compounds, rolitetracycline and doxytetracycline (<a href="#pone-0000428-g004">Figure 4</a>) showed dramatic inhibitory effects on DV propagation. In addition, another compound, oxethazaine, also showed mild inhibition. There was a 12% reduction (down to 88% from that of control) in the PFUs when the concentration of oxethazaine in the culture was increased from 200 µM to 500 µM. For 10 µM rolitetracycline, there was almost no effect on DV plaque formation. But as the concentration of rolitetracycline was increased, there were significant inhibitory effects on DV propagation. Compared with controls, there were only 20% of the PFUs remaining at 100 µM and approximately 5% at 300 µM, yielding an estimated IC<sub>50</sub> value of 67.1 µM (<a href="#pone-0000428-g004">Figure 4</a>). At 500 µM, there were less than 3% PFUs remaining. As for doxycycline, 87% of the PFUs were retained at 10 µM and 14% were retained at 100 µM. When the concentration of doxycycline reached 500 µM, there was only 1% of the PFUs remaining, yielding an IC<sub>50</sub> value of 55.6 µM (<a href="#pone-0000428-g004">Figure 4</a>). Interestingly, neither tetracycline nor oxytetracycline showed an effect on DV propagation at concentrations ranging from 10 µM to 10 mM (data not shown), even though they share the tetracyclic ring structure with both rolitetracycline and doxycycline. <a href="#pone-0000428-g005">Figure 5</a> shows the molecular structures and IC<sub>50</sub> values of the tetracycline derivatives. These compounds had no cellular toxicity effects within the range of concentrations tested as judged from both cellular morphology and growth with one possible exception. When the culture contained doxycycline at a concentration of 500 µm or greater, the cell density appeared to be reduced. We further tested the effects of those compounds by the addition of 500 µm of individual compounds together with or at intervals after the addition of DV to the cultured cells. The results revealed that when 500 µM of either one of the active compounds were added to the cell cultures together with a fixed number of PFUs, the number of plaques formed was approximately 3% or less compared with controls, whereas approximately 75% of the PFUs remained when the compound was added 2 hours after the presence of viruses in the cell culture. Therefore, the inhibitory effect is time-dependent. That is, if the compounds are added after sufficient time was allowed for the infection to proceed, the compounds lost potency.</p>
<div class="figure" id="pone-0000428-g004"><div class="img"><a name="pone-0000428-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Effects of (A) doxycycline and (B) rolitetracycline on Dengue virus type 2 plaque formation using BHK-21 mammalian cells.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The IC<sub>50</sub> values of rolitetracycline and doxytetracycline are 67.1 µM and 55.6 µM, respectively. The <em>x</em> axis shows the percentage of the amount of plaque formation compared with control. The <em>y</em> axis denotes the drug concentration.</p>
<span>doi:10.1371/journal.pone.0000428.g004</span></div>

<h4>Computational analysis of inhibitor-E protein interactions</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The docked conformations of the two tetracycline-derived inhibitors were consistently different from those of the eight non-inhibitory compounds according to the computation program used (<a href="#pone-0000428-g002">Figures 2B</a>, <a href="#pone-0000428-g006">6</a>, and <a href="#pone-0000428-g007">7</a>). The inhibitors, doxytetracycline (green) and rolitetracycline (blue) were docked on the outside of the binding pocket and extended into the pocket while the non-inhibitory compounds (CPK model) were docked inside the pocket (<a href="#pone-0000428-g002">Figure 2B</a>). The inhibitors were docked between the D and I segments of which the conformations significantly differed between the pre-fusion and post-fusion forms (<a href="#pone-0000428-g001">Figures 1B</a> and <a href="#pone-0000428-g007">7</a>). In fact, the inhibitors were docked very close to or at the <em>D′c</em> space and their tetracyclic structure was beneath <em>D′c</em> between <em>D′c</em> and <em>Ic</em> (<a href="#pone-0000428-g001">Figures 1B</a> and <a href="#pone-0000428-g007">7</a>). Those compounds could not only cause steric hindrance by their structures <em>per se</em> but could also interact with the polypeptide stretch of residues 48 to 52 via their functional groups (<a href="#pone-0000428-g006">Figure 6 A–B and 7 A–B</a>). The residues in the stretch, formed by Thr48, Glu49, Ala50, Lys51, and Gln52 and several others in the vicinity, were shown to affect the pH-dependent membrane fusion process <a href="#pone.0000428-Modis1">[5]</a>, <a href="#pone.0000428-Lee1">[12]</a>, <a href="#pone.0000428-Beasley1">[22]</a>–<a href="#pone.0000428-Monath2">[26]</a>. The locations of these residues in the crystal model are indicated in <a href="#pone-0000428-g002">Figure 2</a>. On the other hand, the two inactive tetracycline compounds were docked further away from the <em>D′c</em> space and their tetracyclic structures were localized above <em>D′o</em> and <em>D′c</em>. Hence, they would not create steric hindrance during the switch from <em>D′o</em> to <em>D′c</em> (<a href="#pone-0000428-g007">Figure 7, C–D</a>). <a href="#pone-0000428-g006">Figure 6</a> shows the hydrogen-bonding networks and orientations of the four tetracycline derivatives with regard to the E protein in the pre-fusion form. We also observed that the derivatives could be divided into two groups by their docked locations. Those with inhibitory effects, rolitetracycline (<a href="#pone-0000428-g006">Figure 6A</a>) and doxycycline (<a href="#pone-0000428-g006">Figure 6B</a>), were docked in positions near residues 48–52 and formed hydrogen-bonding networks with residues Thr48, Glu49, Ala50, Lys51, and Gln52, as well as Gln271 and Gln200. Conversely, the other two compounds, tetracycline (<a href="#pone-0000428-g006">Figure 6C</a>) and oxytetracycline (<a href="#pone-0000428-g006">Figure 6D</a>), formed hydrogen bonds primarily with residues Thr280, Phe279, Gln271, and Gln200. Tetracycline interacted with the 48–52 stretch only at Thr48 while oxytetracycline interacted with residues Thr48 and Ala50 and both appeared to prefer Phe279 and Thr280. In addition, the inhibitors bound to opposite sides of the surrounding wall (residues 48–52 vs. residues 200 and 271) of the binding pocket and extended their structures centrally into the pocket, while the non-inhibitors bound entirely to one side of the pocket (residues 200, 271, 279, 280, and 48) (<a href="#pone-0000428-g002">Figures 2B</a> and <a href="#pone-0000428-g006">6</a>). Furthermore, GEMDOCK yielded lower binding energies for the two inhibitors than for the inactive compounds. The energy minimization process performed by SYBYL 6.9 also indicated that the predicted inhibitor complexes had lower energies than the non-inhibitors. The energies of rolitetracycline, doxytetracycline, tetracycline, and oxytetracycline were −395.2, −398.7, −356.8, and −371.8 kcal/mol according to SYBYL 6.9.</p>
<div class="figure" id="pone-0000428-g005"><div class="img"><a name="pone-0000428-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Chemical Structures and IC<sub>50</sub>s for the tetracycline derivatives.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Name, chemical name; IC<sub>50, </sub>the half maximal inhibitory concentration; NA, not applicable.</p>
<span>doi:10.1371/journal.pone.0000428.g005</span></div><div class="figure" id="pone-0000428-g006"><div class="img"><a name="pone-0000428-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Docked conformations and hydrogen bonding of rolitetracycline (A), doxycycline (B), tetracycline (C), and oxytetracycline (D) to the BOG binding site of the DV E protein.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Atoms of the E protein are shown in yellow and compound ligands are shown in CPK model. The hydrogen bonds are represented as green dashed lines. Not all residues are displayed for the sake of clarity. Thr48, Glu49, Ala50, Lys51, and Gln52 are in the <em>D′<sub>0</sub></em> segment (<a href="#pone-0000428-g001">Figure 1B</a>), while Gln271 and Phe279 are in the <em>k<sub>o</sub></em> and <em>l<sub>o</sub></em> segments, respectively.</p>
<span>doi:10.1371/journal.pone.0000428.g006</span></div><div class="figure" id="pone-0000428-g007"><div class="img"><a name="pone-0000428-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000428" data-uri="info:doi/10.1371/journal.pone.0000428.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000428.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000428.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000428.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Docked conformations of the four tetracycline derivatives related to residues 48–52 in the BOG binding site in the pre-fusion (gray) and post-fusion (blue) states.</span></strong></p><a id="article1.body1.sec2.sec3.fig3.caption1.p1" name="article1.body1.sec2.sec3.fig3.caption1.p1"></a><p>(A) rolitetracycline, (B) doxycycline, (C) tetracycline, and (D) oxytetracycline. Atoms within the compounds are displayed using the CPK model (oxygen in red, nitrogen in blue, and carbon in gray). The side chains of certain residues that overlap with the compounds are displayed. The segments <em>D</em>′<em><sub>o</sub></em>, <em>k<sub>o</sub></em>, and <em>l<sub>o</sub></em> are present in the pre-fusion conformation, while <em>D</em>′<em><sub>c</sub></em>, <em>k<sub>c</sub></em>, and <em>l<sub>c</sub></em> are present in the post-fusion conformation.</p>
<span>doi:10.1371/journal.pone.0000428.g007</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>For the eradication of infections caused by enveloped viruses, the identification of compounds that can block the function of viral envelope proteins to prevent viral entry has been a long-standing idea in the field. However, mass screening is usually considered too costly and, for the more design-oriented approaches, VS with limited information tends to yield too many candidates for biological activity assays and is usually further complicated by the cellular toxicity possessed by many of the candidates. Here, we have devised a scheme in which VS focused on both the steric hindrance and atomic environment between the compounds and the targeted E protein to minimize the number of candidates. And, to further reduce the number of candidates, instead of using the whole E protein structure as the target for VS, we isolated the small region around a chosen target site to serve as the target. Although this approach may limit the diversity of the potential leads due to the diminished choices of possible target sites for VS, we believe that this methodology will, in fact, help to enhance the chance of a successful hit because the program can screen many more compounds with more thoroughness within the same time frame. In this study, we chose the hydrophobic detergent-binding pocket reported by Modis et al. as the target <a href="#pone.0000428-Modis1">[5]</a>, <a href="#pone.0000428-Modis2">[11]</a>. This putative detergent-binding site is located in the E protein between domains I and II, which are the key structural elements involved in the pH-induced conformational rearrangement that is essential for DV entry. Therefore, a suitable target for small-molecule inhibitors would be the blockade of the conformational change of the E protein and, subsequently, the inhibition of viral-host membrane fusion, which would interrupt viral entry and block infection <a href="#pone.0000428-Modis1">[5]</a>, <a href="#pone.0000428-Modis2">[11]</a>. Additionally, mutations in the DV E protein mapped to this pocket indeed affect the pH threshold of fusion <a href="#pone.0000428-Modis1">[5]</a>, <a href="#pone.0000428-Lee1">[12]</a>, <a href="#pone.0000428-Beasley1">[22]</a>–<a href="#pone.0000428-Monath2">[26]</a>. In short, based on the structural study of Modis et al. <a href="#pone.0000428-Modis1">[5]</a>, <a href="#pone.0000428-Modis2">[11]</a>, we developed a VS process and was successful in applying it to the identification of lead compounds that inhibit DV propagation. After computation, there were only ten non-toxic candidate compounds that required validation by biological activity assays.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>It is very interesting that in this study, even though tetracycline and oxytetracycline share similar tetracyclic ring structures with both rolitetracycline and doxycycline, they are not inhibitory (<a href="#pone-0000428-g005">Figure 5</a>). Tetracycline derivatives are a group of broad-spectrum antibiotics and were first discovered in the 1940s <a href="#pone.0000428-Chopra1">[19]</a>. The mechanism of action of tetracycline and its derivatives on bacteria is via the inhibition of cellular protein synthesis by preventing the attachment of aminoacyl tRNAs to the ribosomal acceptor (A) site <a href="#pone.0000428-Brodersen1">[18]</a>, <a href="#pone.0000428-Chopra1">[19]</a>, <a href="#pone.0000428-Connell1">[27]</a>. Those antibiotics consist of a linear, fused tetracyclic core to which a variety of functional groups are attached <a href="#pone.0000428-Chopra1">[19]</a>. Tetracycline, in fact, contains the minimal common structure of the tetracycline-related molecules in this study. Therefore, this common structure <em>per se</em> does not possess the inhibitory effect on DV propagation. Instead, the substituted functional groups appear to confer anti-Dengue virus activity.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>On the other hand, it is also possible that the active compounds affect the host cells instead of the virions. If this is the case, the compounds might inactivate a host cellular component that is essential for viral propagation. We believe that this scenario is unlikely since there are no significant differences in cellular morphology and growth, unless the function of such a cellular component, when compromised, affects only the viruses. Nonetheless, to test this hypothesis, we performed an assay in which the compounds were added to the cultures either together or 2 hours after the presence of viruses in the cell cultures. If the compounds were active against the viruses instead of the hosts, then adding them together to the culture should effectively block viral infection whereas the addition of the compounds 2 hours after the presence of viruses would not have the same effect since the viruses would have already proceeded through the entry event and infected the host cells. As expected, when the compounds were added together with the viruses, the PFUs were approximately 3% or less than controls, whereas those added two hours later were approximately 75% of control levels. Therefore, the compounds were less potent after the viruses had entered the host cells. Hence, these compounds most likely act upon a virus target site and only affect an event that occurs prior to the completion of viral entry.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Another possibility is that the compounds act on viral RNA at locations where the RNA structures are similar to the tetracycline-binding sites on the ribosomal RNAs or tRNAs of the bacteria. This possibility requires that these compounds can penetrate the virion structure to interact with the viral RNA to prevent viral entry while, simultaneously, they do not affect the viral RNAs inside the host cells when these same compounds are added two hours later. We believe this scenario is also highly unlikely. First, for the compounds to reach the viral RNAs in the virions, they would have to overcome the physical obstacle consisting of viral structural proteins. Second, if the viral RNAs are indeed the targets, the compounds should be effective regardless of the time of compound addition because they could still enter the host cells and bind the viral RNAs to disrupt replication.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>As for the possibility that the compounds affect viral proteins other than the E protein, we believe that this scenario is also unlikely since the E protein is the only protein required for viral entry. However, we cannot rule out that the compounds may bind at sites other than our predicted locations on the E protein. To reveal the exact location of the compound-E protein interaction, it may be necessary to devise an experiment, such as co-crystallization of the protein and those compounds, in which the compounds can be labeled and traced at an atomic level so their exact docking locations can be identified relative to the binding pocket.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Nonetheless, we have conducted computational modeling in an attempt to provide a direction for future investigation. First, to assess the results of the binding of tetracycline derivates to the DV E protein, we compared the BOG binding sites of the DV E protein to the tetracycline-binding site on the tetracycline repressor, TetR. TetR regulates resistance to tetracycline in gram-negative bacteria. The tetracycline-binding site of the TetR protein has been defined and the structure determined by crystallography <a href="#pone.0000428-Orth1">[28]</a>. We found that the TetR protein shares similar characteristics with the E protein in the binding sites for the tetracycline derivatives. First, there is an appropriate volume in the binding sites. The volumes of the binding sites of various TetR crystals range from 359 Å<sup>3</sup> to 495 Å<sup>3</sup> whereas the BOG binding site on the E protein is 481 Å<sup>3</sup>, according to the tool program, Q-SiteFinder <a href="#pone.0000428-Laurie1">[29]</a> (the first column of <a href="#pone.0000428.s001">Table S1</a>). Therefore, there is proper space for the tetracycline derivatives to fit into the BOG binding site. Second, there are hydrophobic surfaces in the pockets of both binding sites (<a href="#pone.0000428.s003">Figure S1</a>). Third, according to the results of a cross-docking test performed for TetR and the tetracycline derivatives (<a href="#pone.0000428.s001">Table S1</a>), the binding sites of the DV E protein and TetR permit the binding of the tetracycline derivatives. In addition, the hydrogen bonds formed between the tetracycline derivatives and the DV E protein are similar to those between TetR and the tetracycline-derived ligands (<a href="#pone.0000428.s002">Table S2</a>). Therefore, tetracycline derivatives should reasonably bind the BOG pocket of the DV E protein.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>On the other hand, only two of the derivatives are inhibitory; therefore, the atomic details of the functional groups and the tetracyclic core must confer the inhibitory activity. Hence, we have analyzed the docked conformations and hydrogen bonding of the derivatives to assess the interaction between those compounds and the E protein. There are distinct differences between the effective and ineffective compounds (<a href="#pone-0000428-g006">Figures 6</a>, <a href="#pone-0000428-g007">7</a>, and S2); the effective compounds have their tetracyclic cores positioned inside the pocket while their side chains form hydrogen bonds with the residues located on the opposite sides of the wall around the pocket and are capable of creating steric hindrance to the conformational alteration of the E protein. In contrast, the ineffective compounds form hydrogen bonds only with one side of the wall and their cores lean away from the pocket.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Next, on an atomic level, the predicted positions of the tetracycline derivatives with the E protein are shown in <a href="#pone-0000428-g006">Figures 6</a> and <a href="#pone-0000428-g007">7</a>. The fused tetracyclic rings of rolitetracycline and doxytetracycline bind along the <em>D′o</em> strand and occupy the <em>D′c</em> space of the E protein. The residues 48–52 are in the D segments. These compounds both interact mainly with Thr48, Glu49, Ala50, Gln200, and Gln271 through hydrogen bonds. Such a hydrogen-bonding network provides strong attraction forces to stabilize the binding of rolitetracycline and doxytetracycline to the <em>D′o</em> strand and the <em>kl</em> β-hairpin. In contrast, although these compounds have the same tetracyclic core structures, neither tetracycline nor oxytetracycline is inhibitory. Both compounds form hydrogen-bonding networks with Thr48, Gln200, Gln271, Phe279, and Thr280 (<a href="#pone-0000428-g006">Figure 6</a>); therefore, their tetracyclic rings are docked toward one side of the binding site and contact the surrounding hydrophobic residues via van der Waals interactions, which are very different from those of rolitetracycline and doxytetracycline.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>During the process of E protein-host membrane fusion, the E protein structure is dramatically re-configured to allow the fusion peptide to properly interact with the host membrane. This event is marked by the rearrangement of the <em>kl</em> β-hairpin and the <em>D′o</em> segment (Thr48, Glu49, Ala50, Lys51, and Gln52) in the BOG binding site (<a href="#pone-0000428-g001">Figures 1A</a> and <a href="#pone-0000428-g006">6</a>). The docked positions of the inhibitors suggest that they occupy the <em>D′c</em> and <em>kl</em> β-hairpin spaces in the post-fusion state and form a stable hydrogen-bonding network (<a href="#pone-0000428-g006">Figures 6A, 6B</a>, <a href="#pone-0000428-g007">7A and 7B</a>). Therefore, these compounds block the rearrangement of the β-hairpin and <em>D′o</em> strand, and thereby block the rearrangement of domains II and I of the E protein during membrane fusion. Residues 48-52 are not only important to inhibitor binding but may also directly affect flavivirus membrane fusion. This hypothesis is consistent with previous reports that Gln52 may affect the pH threshold of fusion in flaviviruses <a href="#pone.0000428-Modis1">[5]</a>.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>Our study has presented a cost-effective and time-saving screening process that is based on limited structural information. We have successfully identified two novel tetracycline-derived inhibitors of the propagation of flavivirus DV type 2 PL046, by the computer-aided screening of the E protein structure followed by the biological assay validation of the candidate compounds in a cell culture system. These compounds may serve as the basis for the development of new treatments against Dengue virus infection. This procedure may be applied to other viral pathogens or for any other mechanism that involves specific conformational alterations for biological function. Our study also highlights the additional characteristics of certain tetracycline derivatives as effective inhibitors of DV propagation, which will allow further refinement of our screening program and potential medical application.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The VS method generally encompasses four phases that are based on high-throughput molecular docking methods and the crystal structures of the target proteins. These phases include target protein preparation, compound database preparation, molecular docking, and post-docking analysis <a href="#pone.0000428-Lyne1">[6]</a>. The preparation phases involve formatting the structural data from the target protein and compounds into acceptable forms for the docking program. Then, the method of molecular docking is employed to screen the compound library for potential leads that can dock onto the target protein, whereas post-docking analysis serves to enrich the hit rate.</p>

<h4>Preparations of the target protein and screening set</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We prepared the compound set from the CMC database in May 2004 based on two criteria: molecular weights ranging between 200 and 800 Daltons and excluding compounds with multiple components. We eventually obtained a set of structures that consisted of 5,331 compounds. To reduce the complexity and running time of the computational program, we isolated the structure of the BOG binding pocket of the DV E protein (<a href="#pone-0000428-g001">Figure 1B</a>) in the BOG-bound conformation {PDB code 1oke <a href="#pone.0000428-Modis1">[5]</a>} and prepared it for the docking tools. The isolated area included amino acids enclosed within a 10-Å radius that centered on the bound ligand. The coordinates of the protein atoms were taken from the PDB for the screening process. GEMDOCK docked each compound in the screening set against this binding cavity and ranked each compound by the docked energy of the docked conformation. Then, those candidates were subjected to structural clustering <a href="#pone.0000428-Jain1">[21]</a>. According to the ranking, compound structures and the interactions between compounds and residues in the binding site were further selected for <em>in vivo</em> biological activity assays to assess their inhibitory effect on Dengue virus propagation in cell culture.</p>


<h4>Docking method and scoring function</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Our previous work <a href="#pone.0000428-Yang1">[14]</a>–<a href="#pone.0000428-Yang4">[17]</a> showed that the docking accuracy of GEMDOCK was better than some well-known docking tools, such as GOLD <a href="#pone.0000428-Jones1">[30]</a> and FlexX <a href="#pone.0000428-Kramer1">[31]</a>, on a diverse data set of 100 protein-ligand complexes suggested by Jones et al. <a href="#pone.0000428-Jones1">[30]</a>. The accuracy of GEMDOCK was also better than GOLD, FlexX, and DOCK on screening the ligand database from Bissantz et al. <a href="#pone.0000428-Bissantz1">[32]</a> for TK <a href="#pone.0000428-Yang5">[33]</a> and the ER-antagonist receptor <a href="#pone.0000428-Yang3">[16]</a>. In this study, GEMDOCK parameters for flexible docking included the initial step sizes (σ = 0.8 and Ψ = 0.2), family competition length (<em>L</em> = 2), population size (<em>N</em> = 300), and recombination probability (<em>p<sub>c</sub></em> = 0.3). For each ligand screened, GEMDOCK optimization was terminated either when the convergence was below a certain threshold value or when the iterations exceeded a maximal preset value of 60. For the latter case, GEMDOCK produced 800 solutions in one generation and was terminated after it exhausted 48,000 solutions for each compound in the screening set.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>The screening quality of the docking methods using energy-based scoring functions alone is often influenced by the structure of the ligand screened (e.g., the numbers of charged and polar atoms). These methods are often biased toward charged polar compounds due to the pair-atom potentials of the electrostatic and hydrogen-bonding energies. In order to reduce this effect, GEMDOCK can evolve the pharmacological preferences from either a number of known active ligands or domain knowledge to take advantage of the similarities of putative ligands to those that are known to bind a protein's active site, thereby guiding the docking of the putative ligands <a href="#pone.0000428-Yang3">[16]</a>. Therefore, GEMDOCK is capable of using either a purely empirical scoring function <a href="#pone.0000428-Yang2">[15]</a> or a pharmacophore-based scoring function <a href="#pone.0000428-Yang3">[16]</a>. When GEMDOCK uses a pharmacophore-based scoring function, either certain known active ligands (more than two) or domain knowledge are required for the evolution of the pharmacological consensus. The empirical binding energy (<em>E<sub>bind</sub></em>) is given as <a href="#pone.0000428-Yang2">[15]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>E<sub>inter</sub></em> and <em>E<sub>intra</sub></em> are the intermolecular and intramolecular energies, respectively <a href="#pone.0000428-Yang2">[15]</a>. The pharmacophore-based energy function can be expressed as <a href="#pone.0000428-Yang3">[16]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>where <em>E<sub>GEMDOCK-Bind</sub></em> is the empirical binding energy defined in Equation (1), <em>E<sub>pharma</sub></em> is the energy of the binding site pharmacophores (hot spots), and <em>E<sub>ligpre</sub></em> is the penalty value when a ligand does not satisfy the ligand preferences <a href="#pone.0000428-Yang3">[16]</a>. <em>E<sub>pharma</sub></em> and <em>E<sub>ligpre</sub></em> are especially useful in the selection of active compounds from hundreds of thousands of non-active compounds by the exclusion of ligands that violate the characteristics of known active ligands (or domain knowledge). The pharmacophore-based interaction energy (<em>E<sub>pharma</sub></em>) between the ligand and the protein is calculated with the assumption that the binding energies of all hot-spot atoms can be represented by the following equation <a href="#pone.0000428-Yang3">[16]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where <em>CW</em>(<em>B<sub>ij</sub></em>) is a pharmacological-weight function of a hot-spot atom <em>j</em> with interaction type <em>B<sub>ij</sub></em>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.e005&amp;representation=PNG" class="inline-graphic"></span> is as defined previously <a href="#pone.0000428-Yang2">[15]</a>, <em>lig</em> is the number of heavy atoms in a screened ligand, and <em>hs</em> is the number of hot-spot atoms in the protein. The ligand preference (<em>E<sub>ligpre</sub></em>) is the penalty value for the screened ligands that violate the electrostatic or hydrophilic constraints. <em>E<sub>ligpre</sub></em> is given as <a href="#pone.0000428-Yang3">[16]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000428.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>where <em>LP<sub>elec</sub></em> and <em>LP<sub>hb</sub></em> are the penalties for the electrostatic (i.e., the number of charged atoms in a screened ligand) and hydrophilic (i.e., the fraction of polar atoms in a screened ligand) constraints, respectively.</p>


<h4>Plaque formation assay for the inhibitory effects of compounds on DV2 propagation</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A local DV type 2 strain, PL046, was used to infect mosquito C6/36 cells for the production of DV type 2 virions. Mammalian BHK-21 host cells were cultured at 37°C with 5% CO<sub>2</sub> in MEM medium (Gibco) supplemented with 0.22% sodium bicarbonate and 10% fetal bovine serum (FBS) (Gibco). C6/36 cells were grown at 28°C in MEM medium (Gibco) supplemented with 0.11% sodium bicarbonate and 10% FBS <a href="#pone.0000428-Hurrelbrink1">[23]</a>. BHK-21 cells were plated at a density of 4×10<sup>5</sup> cells per well in 6-well plates and incubated at 37°C with 5% CO<sub>2</sub> for 48 hours. Different dilutions of the compounds were added to the 6-well plates followed by 0.5 mL of medium containing 200 PFUs of the DV type 2 PL046 strain per well. The mixtures were mixed gently and then incubated at 37°C with 5% CO<sub>2</sub> for 1 hour. The supernatant in the culture was aspirated prior to the addition of a 1:1 mixture of MEM medium:2% methylcellulose. The culture was then incubated at 37°C with 5% CO<sub>2</sub> for 7 days. The medium was aspirated prior to fixation of the cells with 3.7% formaldehyde for 30 minutes. Then, the fixing solution was removed and the cells were stained with 1% crystal violet in 3.7% formaldehyde. Finally, the plates were washed with 3.7% formaldehyde prior to scoring of plaques <a href="#pone.0000428-Chiu1">[34]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000428"></div><a name="pone.0000428.s001" id="pone.0000428.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000428.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">GEMDOCK cross-docking results of docking seven tetracycline-derivatives into five TetR protein structures and DV E protein</p>
<p class="siDoi">doi:10.1371/journal.pone.0000428.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000428.s002" id="pone.0000428.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000428.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Comparisons of the hydrogen bonds of five compounds between the dengue E protein and TetR protein</p>
<p class="siDoi">doi:10.1371/journal.pone.0000428.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000428.s003" id="pone.0000428.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000428.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Docked conformations of the four tetracycline-derivatives. The two active compounds are rolitetracycline (blue) and doxycycline (green). The two inactive compounds are tetracycline (orange) and oxytetracycline (red). The inhibitory compounds are docked in positions leaning on the residues of the 48–52 stretch, of which the conformations in prefusion and postfusion states are very different. Residues affecting the pH threshold of fusion are indicated by numbers.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000428.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
<a name="pone.0000428.s004" id="pone.0000428.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000428.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">The surfaces and the docked conformations of the four tetracycline-derivatives on TetR protein and DV E protein according to GEMDOCK. (A) TetR protein (PDB code 2TRT); (B) DV E protein (PDB code 1OKE). The surfaces and sizes of the binding sites of these two proteins are similar. In addition, docked conformations of the four tetracycline-derivatives in these two proteins are also similar.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000428.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.09 MB PDF)</p>
<a name="pone.0000428.s005" id="pone.0000428.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000428.s005">Appendix S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Appendix A: The top 173 compounds of GEMDOCK by screening the CMC database.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000428.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.18 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Drs. Yi-Ling Lin and Wen Chang for their assistance with the viruses and cell lines. The authors are grateful to both the hardware and software support staff of the Structural Bioinformatics Core Facility at National Chiao Tung University.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YY JY. Performed the experiments: YC YT KY. Analyzed the data: YY JY YC YT KY. Contributed reagents/materials/analysis tools: YY JY. Wrote the paper: YY JY YC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000428-Hayes1" id="pone.0000428-Hayes1"></a>Hayes EB, Gubler DJ (2006) West Nile virus: Epidemiology and clinical features of an emerging epidemic in the United States. Annual Review of Medicine  57: 181–194.  <ul class="find" data-citedArticleID="949746" data-doi="10.1146/annurev.med.57.121304.131418"><li><a href="http://dx.doi.org/10.1146/annurev.med.57.121304.131418" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=West+Nile+virus%3A+Epidemiology+and+clinical+features+of+an+emerging+epidemic+in+the+United+States." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22West+Nile+virus%3A+Epidemiology+and+clinical+features+of+an+emerging+epidemic+in+the+United+States.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000428-Brinton1" id="pone.0000428-Brinton1"></a>Brinton MA (2002) The molecular biology of West Nile virus: A new invader of the Western hemisphere. Annual Review of Microbiology  56: 371–402.  <ul class="find" data-citedArticleID="949730" data-doi="10.1146/annurev.micro.56.012302.160654"><li><a href="http://dx.doi.org/10.1146/annurev.micro.56.012302.160654" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+biology+of+West+Nile+virus%3A+A+new+invader+of+the+Western+hemisphere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+biology+of+West+Nile+virus%3A+A+new+invader+of+the+Western+hemisphere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000428-Leyssen1" id="pone.0000428-Leyssen1"></a>Leyssen P, Clercq ED, Neyts J (2000) Perspectives for the treatment of infections with Flaviviridae. Clinical Microbiology Reviews  13: 67–82.  <ul class="find" data-citedArticleID="949760" data-doi="10.1128/cmr.13.1.67-82.2000"><li><a href="http://dx.doi.org/10.1128/cmr.13.1.67-82.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Perspectives+for+the+treatment+of+infections+with+Flaviviridae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Perspectives+for+the+treatment+of+infections+with+Flaviviridae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000428-Gubler1" id="pone.0000428-Gubler1"></a>Gubler DJ, Gubler DJ (1997) Dengue and dengue hemorrhagic hever: its history and resurgence as a global public health problem. Dengue and dengue hemorrhagic fever. New York: CAB International. pp. 1–22: New York.  <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0000428-Modis1" id="pone.0000428-Modis1"></a>Modis Y, Ogata S, Clements D, Harrison SC (2003) A ligand-binding pocket in the dengue virus envelope glycoprotein. Proceedings of the National Academy of Sciences of the United States of America  100: 6986–6991.  <ul class="find" data-citedArticleID="949764" data-doi="10.1073/pnas.0832193100"><li><a href="http://dx.doi.org/10.1073/pnas.0832193100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+ligand-binding+pocket+in+the+dengue+virus+envelope+glycoprotein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+ligand-binding+pocket+in+the+dengue+virus+envelope+glycoprotein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000428-Lyne1" id="pone.0000428-Lyne1"></a>Lyne PD (2002) Structure-based virtual screening: an overview. Drug Discovery Today  7: 1047–1055.  <ul class="find" data-citedArticleID="949762" data-doi="10.1016/s1359-6446(02)02483-2"><li><a href="http://dx.doi.org/10.1016/s1359-6446(02)02483-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure-based+virtual+screening%3A+an+overview." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure-based+virtual+screening%3A+an+overview.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000428-Shoichet1" id="pone.0000428-Shoichet1"></a>Shoichet BK, McGovern SL, Wei B, Irwin J (2002) Lead discovery using molecular docking. Current Opinion in Chemical Biology  6: 439–446.  <ul class="find" data-citedArticleID="949780" data-doi="10.1016/s1367-5931(02)00339-3"><li><a href="http://dx.doi.org/10.1016/s1367-5931(02)00339-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lead+discovery+using+molecular+docking." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lead+discovery+using+molecular+docking.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000428-Shoichet2" id="pone.0000428-Shoichet2"></a>Shoichet BK (2004) Virtual screening of chemical libraries. Nature  432: 862–865.  <ul class="find" data-citedArticleID="949782" data-doi="10.1038/nature03197"><li><a href="http://dx.doi.org/10.1038/nature03197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Virtual+screening+of+chemical+libraries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Virtual+screening+of+chemical+libraries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000428-Ghosh1" id="pone.0000428-Ghosh1"></a>Ghosh S, Nie AH, An J, Huang ZW (2006) Structure-based virtual screening of chemical libraries for drug discovery. Current Opinion in Chemical Biology  10: 194–202.  <ul class="find" data-citedArticleID="949740" data-doi="10.1016/j.cbpa.2006.04.002"><li><a href="http://dx.doi.org/10.1016/j.cbpa.2006.04.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure-based+virtual+screening+of+chemical+libraries+for+drug+discovery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure-based+virtual+screening+of+chemical+libraries+for+drug+discovery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000428-Monath1" id="pone.0000428-Monath1"></a>Monath TP, Henize FX (1996)  Flaviviruses. In: Fields BN, Knipe DM, Howley PM, editors. Fields virology, 3rd ed. Philadelphia, Pa: Lipponcott-Raven. pp. 961–1034.  vol. 1. <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0000428-Modis2" id="pone.0000428-Modis2"></a>Modis Y, Ogata S, Clements D, Harrison SC (2004) Structure of the dengue virus envelope protein after membrane fusion. Nature  427: 313–319.  <ul class="find" data-citedArticleID="949766" data-doi="10.1038/nature02165"><li><a href="http://dx.doi.org/10.1038/nature02165" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+the+dengue+virus+envelope+protein+after+membrane+fusion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+the+dengue+virus+envelope+protein+after+membrane+fusion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000428-Lee1" id="pone.0000428-Lee1"></a>Lee E, Weir RC, Dalgarno L (1997) Changes in the dengue virus major envelope protein on passaging and their localization on the three-dimensional structure of the protein. Virology  232: 281–290.  <ul class="find" data-citedArticleID="949758" data-doi="10.1006/viro.1997.8570"><li><a href="http://dx.doi.org/10.1006/viro.1997.8570" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changes+in+the+dengue+virus+major+envelope+protein+on+passaging+and+their+localization+on+the+three-dimensional+structure+of+the+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changes+in+the+dengue+virus+major+envelope+protein+on+passaging+and+their+localization+on+the+three-dimensional+structure+of+the+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000428-Rey1" id="pone.0000428-Rey1"></a>Rey FA, Heinz FX, Mandl C, Kunz C, Harrison SC (1995) The envelope glycoprotein from tick-borne encephalitis virus at 2 A resolution. Nature  375: 291–298.  <ul class="find" data-citedArticleID="949774" data-doi="10.1038/375291a0"><li><a href="http://dx.doi.org/10.1038/375291a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+envelope+glycoprotein+from+tick-borne+encephalitis+virus+at+2+A+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+envelope+glycoprotein+from+tick-borne+encephalitis+virus+at+2+A+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000428-Yang1" id="pone.0000428-Yang1"></a>Yang J-M, Chen Y-F, Shen T-W, Kristal BS, Hsu DF (2005) Consensus scoring sriteria for Improving enrichment in virtual screening. Journal of Chemical Information and Modeling  45: 1134–1146.  <ul class="find" data-citedArticleID="949784" data-doi="10.1021/ci050034w"><li><a href="http://dx.doi.org/10.1021/ci050034w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consensus+scoring+sriteria+for+Improving+enrichment+in+virtual+screening." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consensus+scoring+sriteria+for+Improving+enrichment+in+virtual+screening.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000428-Yang2" id="pone.0000428-Yang2"></a>Yang J-M, Chen C-C (2004) GEMDOCK: a generic evolutionary method for molecular docking. Proteins: Structure, Function, and Bioinformatics  55: 288–304.  <ul class="find" data-citedArticleID="949786" data-doi="10.1002/prot.20035"><li><a href="http://dx.doi.org/10.1002/prot.20035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GEMDOCK%3A+a+generic+evolutionary+method+for+molecular+docking." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GEMDOCK%3A+a+generic+evolutionary+method+for+molecular+docking.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000428-Yang3" id="pone.0000428-Yang3"></a>Yang J-M, Shen T-W (2005) A pharmacophore-based evolutionary approach for screening selective estrogen receptor modulators. Proteins: Structure, Function, and Bioinformatics  59: 205–220.  <ul class="find" data-citedArticleID="949788" data-doi="10.1002/prot.20387"><li><a href="http://dx.doi.org/10.1002/prot.20387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+pharmacophore-based+evolutionary+approach+for+screening+selective+estrogen+receptor+modulators." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+pharmacophore-based+evolutionary+approach+for+screening+selective+estrogen+receptor+modulators.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000428-Yang4" id="pone.0000428-Yang4"></a>Yang J-M (2004) Development and evaluation of a generic evolutionary method for protein-ligand docking. Journal of Computational Chemistry  25: 843–857.  <ul class="find" data-citedArticleID="949790" data-doi="10.1002/jcc.20013"><li><a href="http://dx.doi.org/10.1002/jcc.20013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+and+evaluation+of+a+generic+evolutionary+method+for+protein-ligand+docking." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+and+evaluation+of+a+generic+evolutionary+method+for+protein-ligand+docking.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000428-Brodersen1" id="pone.0000428-Brodersen1"></a>Brodersen DE, Clemons JWM, Carter AP, Morgan-Warren RJ, Wimberly BT, et al.  (2000) The structural basis for the action of the antibiotics tetracycline, pactamycin, and hygromycin B on the 30S ribosomal subunit. Cell  103: 1143–1154.  <ul class="find" data-citedArticleID="949732" data-doi="10.1016/s0092-8674(00)00216-6"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)00216-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structural+basis+for+the+action+of+the+antibiotics+tetracycline%2C+pactamycin%2C+and+hygromycin+B+on+the+30S+ribosomal+subunit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structural+basis+for+the+action+of+the+antibiotics+tetracycline%2C+pactamycin%2C+and+hygromycin+B+on+the+30S+ribosomal+subunit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000428-Chopra1" id="pone.0000428-Chopra1"></a>Chopra I, Roberts M (2001) Tetracycline antibiotics: mode of action, applications, molecular biology, and epidemiology of bacterial resistance. Microbiology and Molecular Biology Reviews  65: 232–260.  <ul class="find" data-citedArticleID="949736" data-doi="10.1128/mmbr.65.2.232-260.2001"><li><a href="http://dx.doi.org/10.1128/mmbr.65.2.232-260.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tetracycline+antibiotics%3A+mode+of+action%2C+applications%2C+molecular+biology%2C+and+epidemiology+of+bacterial+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tetracycline+antibiotics%3A+mode+of+action%2C+applications%2C+molecular+biology%2C+and+epidemiology+of+bacterial+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000428-Shen1" id="pone.0000428-Shen1"></a>Shen M, LeTiran A, Xiao Y, Golbraikh A, Kohn H, et al.  (2002) Quantitative structure-activity relationship analysis of functionalized amino acid anticonvulsant agents using k nearest neighbor and simulated annealing PLS methods. Journal of Medicinal Chemistry  45: 2811–2823.  <ul class="find" data-citedArticleID="949778" data-doi="10.1021/jm010488u"><li><a href="http://dx.doi.org/10.1021/jm010488u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+structure-activity+relationship+analysis+of+functionalized+amino+acid+anticonvulsant+agents+using+k+nearest+neighbor+and+simulated+annealing+PLS+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+structure-activity+relationship+analysis+of+functionalized+amino+acid+anticonvulsant+agents+using+k+nearest+neighbor+and+simulated+annealing+PLS+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000428-Jain1" id="pone.0000428-Jain1"></a>Jain AJ (2003) Surflex: fully automatic flexible molecular docking using a molecular similarity-based search engine. Journal of Medicinal Chemistry  46: 499–511.  <ul class="find" data-citedArticleID="949750" data-doi="10.1021/jm020406h"><li><a href="http://dx.doi.org/10.1021/jm020406h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Surflex%3A+fully+automatic+flexible+molecular+docking+using+a+molecular+similarity-based+search+engine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Surflex%3A+fully+automatic+flexible+molecular+docking+using+a+molecular+similarity-based+search+engine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000428-Beasley1" id="pone.0000428-Beasley1"></a>Beasley DWC, Aaskov JG (2001) Epitopes on the dengue 1 virus envelope protein recognized by neutralizing IgM monoclonal antibodies. Virology  279: 447–458.  <ul class="find" data-citedArticleID="949726" data-doi="10.1006/viro.2000.0721"><li><a href="http://dx.doi.org/10.1006/viro.2000.0721" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epitopes+on+the+dengue+1+virus+envelope+protein+recognized+by+neutralizing+IgM+monoclonal+antibodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epitopes+on+the+dengue+1+virus+envelope+protein+recognized+by+neutralizing+IgM+monoclonal+antibodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000428-Hurrelbrink1" id="pone.0000428-Hurrelbrink1"></a>Hurrelbrink RJ, McMinn PC (2001) Attenuation of Murray Valley encephalitis virus by site-directed mutagenesis of the hinge and putative receptor-binding regions of the envelope protein. Journal of Virology  75: 7692–7702.  <ul class="find" data-citedArticleID="949748" data-doi="10.1128/jvi.75.16.7692-7702.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.16.7692-7702.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Attenuation+of+Murray+Valley+encephalitis+virus+by+site-directed+mutagenesis+of+the+hinge+and+putative+receptor-binding+regions+of+the+envelope+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Attenuation+of+Murray+Valley+encephalitis+virus+by+site-directed+mutagenesis+of+the+hinge+and+putative+receptor-binding+regions+of+the+envelope+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000428-Guirakhoo1" id="pone.0000428-Guirakhoo1"></a>Guirakhoo F, Zhang Z, Myers G, Johnson BW, Pugachev K, et al.  (2004) A single amino acid substitution in the envelope protein of chimeric yellow fever-dengue 1 vaccine virus reduces neurovirulence for suckling mice and viremia/viscerotropism for monkeys. Journal of Virology  78: 9998–10008.  <ul class="find" data-citedArticleID="949744" data-doi="10.1128/jvi.78.18.9998-10008.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.18.9998-10008.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+amino+acid+substitution+in+the+envelope+protein+of+chimeric+yellow+fever-dengue+1+vaccine+virus+reduces+neurovirulence+for+suckling+mice+and+viremia%2Fviscerotropism+for+monkeys." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+amino+acid+substitution+in+the+envelope+protein+of+chimeric+yellow+fever-dengue+1+vaccine+virus+reduces+neurovirulence+for+suckling+mice+and+viremia%2Fviscerotropism+for+monkeys.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000428-Serafin1" id="pone.0000428-Serafin1"></a>Serafin IL, Aaskov JG (2001) Identification of epitopes on the envelope (E) protein of dengue 2 and dengue 3 viruses using monoclonal antibodies. Archives of Virology  146: 2469–2479.  <ul class="find" data-citedArticleID="949776" data-doi="10.1007/s007050170017"><li><a href="http://dx.doi.org/10.1007/s007050170017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+epitopes+on+the+envelope+%28E%29+protein+of+dengue+2+and+dengue+3+viruses+using+monoclonal+antibodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+epitopes+on+the+envelope+%28E%29+protein+of+dengue+2+and+dengue+3+viruses+using+monoclonal+antibodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000428-Monath2" id="pone.0000428-Monath2"></a>Monath TP, Arroyo J, Levenbook I, Zhang ZX, Catalan J, et al.  (2002) Single mutation in the flavivirus envelope protein hinge region increases neurovirulence for mice and monkeys but decreases viscerotropism for monkeys: relevance to development and safety testing of live, attenuated vaccines. Journal of Virology  76: 1932–1943.  <ul class="find" data-citedArticleID="949770" data-doi="10.1128/jvi.76.4.1932-1943.2002"><li><a href="http://dx.doi.org/10.1128/jvi.76.4.1932-1943.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single+mutation+in+the+flavivirus+envelope+protein+hinge+region+increases+neurovirulence+for+mice+and+monkeys+but+decreases+viscerotropism+for+monkeys%3A+relevance+to+development+and+safety+testing+of+live%2C+attenuated+vaccines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single+mutation+in+the+flavivirus+envelope+protein+hinge+region+increases+neurovirulence+for+mice+and+monkeys+but+decreases+viscerotropism+for+monkeys%3A+relevance+to+development+and+safety+testing+of+live%2C+attenuated+vaccines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000428-Connell1" id="pone.0000428-Connell1"></a>Connell SR, Tracz DM, Nierhaus KH, Taylor DE (2003) Ribosomal protection proteins and their mechanism of tetracycline resistance. Antimicrobial Agents and Chemotherapy  47: 3675–3681.  <ul class="find" data-citedArticleID="949738" data-doi="10.1128/aac.47.12.3675-3681.2003"><li><a href="http://dx.doi.org/10.1128/aac.47.12.3675-3681.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+protection+proteins+and+their+mechanism+of+tetracycline+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+protection+proteins+and+their+mechanism+of+tetracycline+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000428-Orth1" id="pone.0000428-Orth1"></a>Orth P, Schnappinger D, Hillen W, Saenger W, Hinrichs W (2000) Structural basis of gene regulation by the tetracycline inducible Tetrepressor–operator system. Nature  7: 215–219.  <ul class="find" data-citedArticleID="949772" data-doi="10.1038/73324"><li><a href="http://dx.doi.org/10.1038/73324" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+of+gene+regulation+by+the+tetracycline+inducible+Tetrepressor%26%23x2013%3Boperator+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+of+gene+regulation+by+the+tetracycline+inducible+Tetrepressor%26%23x2013%3Boperator+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000428-Laurie1" id="pone.0000428-Laurie1"></a>Laurie AT, Jackson RM (2005) Q-SiteFinder: an energy-based method for the prediction of protein-ligand binding sites. Bioinformatics  21: 1908–1916.  <ul class="find" data-citedArticleID="949756" data-doi="10.1093/bioinformatics/bti315"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Q-SiteFinder%3A+an+energy-based+method+for+the+prediction+of+protein-ligand+binding+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Q-SiteFinder%3A+an+energy-based+method+for+the+prediction+of+protein-ligand+binding+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000428-Jones1" id="pone.0000428-Jones1"></a>Jones G, Willett P, Glen RC, Leach AR, Taylor R (1997) Development and validation of a genetic algorithm for flexible docking. Journal of Molecular Biology  267: 727–748.31.  <ul class="find" data-citedArticleID="949752" data-doi="10.1006/jmbi.1996.0897"><li><a href="http://dx.doi.org/10.1006/jmbi.1996.0897" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+and+validation+of+a+genetic+algorithm+for+flexible+docking.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000428-Kramer1" id="pone.0000428-Kramer1"></a>Kramer B, Rarey M, Lengauer T (1999) Evaluation of the flexX incremental construction algorithm for protein-ligand docking. Proteins: Structure, Function, and Bioinformatics  37: 228–241.  <ul class="find" data-citedArticleID="949754" data-doi="10.1002/(sici)1097-0134(19991101)37:2<228::aid-prot8&gt;3.0.co;2-8"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0134(19991101)37:2<228::aid-prot8&gt;3.0.co;2-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+the+flexX+incremental+construction+algorithm+for+protein-ligand+docking." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+the+flexX+incremental+construction+algorithm+for+protein-ligand+docking.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000428-Bissantz1" id="pone.0000428-Bissantz1"></a>Bissantz C, Folkers G, Rognan D (2000) Protein-based virtual screening of chemical databases. 1. Evaluation of different docking/scoring combinations. Journal of Medical Chemistry  14: 4759–4767.  <ul class="find" data-citedArticleID="949728" data-doi="10.1021/jm001044l"><li><a href="http://dx.doi.org/10.1021/jm001044l" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein-based+virtual+screening+of+chemical+databases.+1.+Evaluation+of+different+docking%2Fscoring+combinations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein-based+virtual+screening+of+chemical+databases.+1.+Evaluation+of+different+docking%2Fscoring+combinations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000428-Yang5" id="pone.0000428-Yang5"></a>Yang J-M, Shen T-W, Chen Y-F, Chiu Y-Y (2004) An evolutionary approach with pharmacophore-based scoring functions for virtual database screening. Lecture Notes in Computer Science  3102: 481–492.  <ul class="find" data-citedArticleID="949792" data-doi="10.1007/978-3-540-24854-5_48"><li><a href="http://dx.doi.org/10.1007/978-3-540-24854-5_48" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+evolutionary+approach+with+pharmacophore-based+scoring+functions+for+virtual+database+screening." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+evolutionary+approach+with+pharmacophore-based+scoring+functions+for+virtual+database+screening.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000428-Chiu1" id="pone.0000428-Chiu1"></a>Chiu M-W, Yang Y-L (2003) Blocking the dengue virus 2 infections on BHK-21 cells with purified recombinant dengue virus 2 E protein expressed in Escherichia coli. Biochemical and Biophysical Research Communications  309: 672–678.  <ul class="find" data-citedArticleID="949734" data-doi="10.1016/j.bbrc.2003.08.053"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2003.08.053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Blocking+the+dengue+virus+2+infections+on+BHK-21+cells+with+purified+recombinant+dengue+virus+2+E+protein+expressed+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Blocking+the+dengue+virus+2+infections+on+BHK-21+cells+with+purified+recombinant+dengue+virus+2+E+protein+expressed+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.XML" value="87092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.PDF" value="702352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g001.PNG_L" value="423879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g001.PNG_M" value="192186"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g001.PNG_S" value="5598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g001.TIF" value="541334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g001.PNG_I" value="70310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g002.PNG_L" value="221310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g002.PNG_M" value="163539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g002.PNG_S" value="5605"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g002.TIF" value="295552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g002.PNG_I" value="58666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g003.PNG_L" value="158472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g003.PNG_M" value="49352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g003.PNG_S" value="2212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g003.TIF" value="272502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g003.PNG_I" value="20351"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g004.PNG_L" value="154674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g004.PNG_M" value="41261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g004.PNG_S" value="2115"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g004.TIF" value="312086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g004.PNG_I" value="17590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g005.PNG_L" value="122197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g005.PNG_M" value="81173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g005.PNG_S" value="2965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g005.TIF" value="193384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g005.PNG_I" value="38117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g006.PNG_L" value="384023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g006.PNG_M" value="150047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g006.PNG_S" value="4361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g006.TIF" value="575650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g006.PNG_I" value="57749"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g007.PNG_L" value="188764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g007.PNG_M" value="87321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g007.PNG_S" value="3267"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g007.TIF" value="254202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.g007.PNG_I" value="33813"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.e001.PNG" value="3386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.e001.TIF" value="24940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.e002.PNG" value="5202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.e002.TIF" value="26104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.e003.PNG" value="5674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.e003.TIF" value="27400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.e005.PNG" value="1495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.e005.TIF" value="22544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.e004.PNG" value="2442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.e004.TIF" value="23940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.s001.PDF" value="23819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.s002.PDF" value="18649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.s003.PDF" value="70741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.s004.PDF" value="88182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000428.s005.PDF" value="183481"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000428&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000428" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000428&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000428&volume=&issue=&title=Combinatorial Computational Approaches to Identify Tetracycline Derivatives as Flavivirus Inhibitors&author_name=Jinn-Moon%20Yang%2C%20Yan-Fu%20Chen%2C%20Yu-Yin%20Tu%2C%20Kuei-Rong%20Yen%2C%20Yun-Liang%20Yang&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000428" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000428" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000428" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000428&amp;t=Combinatorial%20Computational%20Approaches%20to%20Identify%20Tetracycline%20Derivatives%20as%20Flavivirus%20Inhibitors" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000428&title=Combinatorial%20Computational%20Approaches%20to%20Identify%20Tetracycline%20Derivatives%20as%20Flavivirus%20Inhibitors&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000428&amp;title=Combinatorial%20Computational%20Approaches%20to%20Identify%20Tetracycline%20Derivatives%20as%20Flavivirus%20Inhibitors" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000428" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000428&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Combinatorial Computational Approaches to Identify Tetracycline Derivatives as Flavivirus Inhibitors';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000428';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Combinatorial%20Computational%20Approaches%20to%20Identify%20Tetracycline%20Derivatives%20as%20Flavivirus%20Inhibitors http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000428" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000428" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Dengue+virus%22" title="Search for articles in the subject area:'Dengue virus'"><div class="flagText">Dengue virus</div></a>
              <div data-categoryid="43281" data-articleid="23694"
                   data-categoryname="Dengue virus"
                   class="flagImage" title="Flag 'Dengue virus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Host+cells%22" title="Search for articles in the subject area:'Host cells'"><div class="flagText">Host cells</div></a>
              <div data-categoryid="19355" data-articleid="23694"
                   data-categoryname="Host cells"
                   class="flagImage" title="Flag 'Host cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hydrogen+bonding%22" title="Search for articles in the subject area:'Hydrogen bonding'"><div class="flagText">Hydrogen bonding</div></a>
              <div data-categoryid="23891" data-articleid="23694"
                   data-categoryname="Hydrogen bonding"
                   class="flagImage" title="Flag 'Hydrogen bonding' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+fusion%22" title="Search for articles in the subject area:'Membrane fusion'"><div class="flagText">Membrane fusion</div></a>
              <div data-categoryid="20415" data-articleid="23694"
                   data-categoryname="Membrane fusion"
                   class="flagImage" title="Flag 'Membrane fusion' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+proteins%22" title="Search for articles in the subject area:'Membrane proteins'"><div class="flagText">Membrane proteins</div></a>
              <div data-categoryid="17569" data-articleid="23694"
                   data-categoryname="Membrane proteins"
                   class="flagImage" title="Flag 'Membrane proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure%22" title="Search for articles in the subject area:'Protein structure'"><div class="flagText">Protein structure</div></a>
              <div data-categoryid="31877" data-articleid="23694"
                   data-categoryname="Protein structure"
                   class="flagImage" title="Flag 'Protein structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Tetracyclines%22" title="Search for articles in the subject area:'Tetracyclines'"><div class="flagText">Tetracyclines</div></a>
              <div data-categoryid="43685" data-articleid="23694"
                   data-categoryname="Tetracyclines"
                   class="flagImage" title="Flag 'Tetracyclines' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+entry%22" title="Search for articles in the subject area:'Viral entry'"><div class="flagText">Viral entry</div></a>
              <div data-categoryid="19411" data-articleid="23694"
                   data-categoryname="Viral entry"
                   class="flagImage" title="Flag 'Viral entry' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6424'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4471'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8422&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
