

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Functional Group and Substructure Searching as a Tool in Metabolomics</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001537"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001537" />

    <meta name="citation_title" content="Functional Group and Substructure Searching as a Tool in Metabolomics"/>
    <meta itemprop="name" content="Functional Group and Substructure Searching as a Tool in Metabolomics"/>

      <meta name="citation_author" content="Masaaki Kotera"/>
            <meta name="citation_author_institution" content="School of Biochemistry and Immunology, Trinity College, Dublin, Ireland"/>
      <meta name="citation_author" content="Andrew G. McDonald"/>
            <meta name="citation_author_institution" content="School of Biochemistry and Immunology, Trinity College, Dublin, Ireland"/>
      <meta name="citation_author" content="Sinéad Boyce"/>
            <meta name="citation_author_institution" content="School of Biochemistry and Immunology, Trinity College, Dublin, Ireland"/>
      <meta name="citation_author" content="Keith F. Tipton"/>
            <meta name="citation_author_institution" content="School of Biochemistry and Immunology, Trinity College, Dublin, Ireland"/>

    <meta name="citation_date" content="2008/2/6"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001537.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1537"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_number=1; citation_date=2001; citation_publisher=American Chemical Society, Washington, DC/Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=The Nomenclature of Organic Chemistry. Sections A, B, C, D, E, F and H, Pergamon, Oxford.; citation_number=2; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Principles of Chemical Nomenclature: A Guide to IUPAC Recommendations.; citation_author=GJ Leigh; citation_author=HA Favre; citation_author=WV Metanomski; citation_number=3; citation_date=1998; citation_publisher=Blackwell Science; " />
      <meta name="citation_reference" content="citation_title=Nomenclature: automatic generation and conversion.; citation_author=JL Wisniewski; citation_number=4; citation_pages=1881-1894; citation_date=1998; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_title=Challenges to be faced in the reconstruction of metabolic networks from public databases.; citation_author=MG Poolman; citation_author=BK Bonde; citation_author=A Gevorgyan; citation_author=HH Patel; citation_author=DA Fell; citation_journal_title=IEE Proc Syst Biol; citation_volume=153; citation_number=5; citation_pages=379-384; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Pharmacophore modeling in drug discovery and development: an overview.; citation_author=SA Khedkar; citation_author=AK Malde; citation_author=EC Coutinho; citation_author=S Srivastava; citation_journal_title=Med Chem; citation_volume=3; citation_number=6; citation_pages=187-197; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Pharmacophore-based discovery of ligands for drug transporters.; citation_author=C Chang; citation_author=S Ekins; citation_author=P Bahadduri; citation_author=PW Swaan; citation_journal_title=Adv Drug Deliv Rev; citation_volume=58; citation_number=7; citation_pages=1431-1450; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Computational models for predicting interactions with cytochrome p450 enzyme.; citation_author=R Arimoto; citation_journal_title=Curr Topics Med Chem; citation_volume=6; citation_number=8; citation_pages=1609-1618; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PASS biological activity spectrum predictions in the Enhanced Open NCI Database Browser.; citation_author=VV Poroikov; citation_author=DA Filimonov; citation_author=WD Ihlenfeldt; citation_author=TA Gloriozova; citation_author=AA Lagunin; citation_journal_title=J Chem Inf Comput Sci; citation_volume=43; citation_number=9; citation_pages=228-236; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=InfVis - platform-independent visual data mining of multidimensional chemical data sets.; citation_author=F Oellien; citation_author=WD Ihlenfeldt; citation_author=J Johann Gasteiger; citation_journal_title=J Chem Inf Model; citation_volume=45; citation_number=10; citation_pages=1456-1467; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular optimization using computational multiobjective methods.; citation_author=CA Nicolaou; citation_author=N Brown; citation_author=CS Pattichis; citation_journal_title=Curr Opin Drug Discov Devel; citation_volume=10; citation_number=11; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=GALAHAD: 1. Pharmacophore identification by hypermolecular alignment of ligands in 3D.; citation_author=NJ Richmond; citation_author=CA Abrams; citation_author=PRN Wolohan; citation_author=E Abrahamian; citation_author=P Willett; citation_journal_title=J Comput Aided Mol Des; citation_volume=20; citation_number=12; citation_pages=567-587; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evaluation of machine-learning methods for ligand-based virtual screening.; citation_author=B Chen; citation_author=RF Harrison; citation_author=G Papadatos; citation_author=P Willett; citation_author=DJ Wood; citation_journal_title=J Comput Aided Mol Des; citation_volume=21; citation_number=13; citation_pages=53-62; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Self-organizing neural networks for pharmacophore mapping.; citation_author=J Polanski; citation_journal_title=Adv Drug Deliv Rev; citation_volume=55; citation_number=14; citation_pages=1149-1162; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Pharmacophore features of potential drugs.; citation_author=I Muegge; citation_journal_title=Chem Eur J; citation_volume=8; citation_number=15; citation_pages=1976-1981; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Metabolome diversity: too few genes, too many metabolites?; citation_author=W Schwab; citation_journal_title=Phytochemistry; citation_volume=62; citation_number=16; citation_pages=837-849; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Enzyme promiscuity: mechanism and applications.; citation_author=K Hult; citation_author=P Berglund; citation_journal_title=Trends Biochem; citation_volume=25; citation_number=17; citation_pages=231-238; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=From genomics to chemical genomics: new developments in KEGG.; citation_author=M Kanehisa; citation_author=S Goto; citation_author=M Hattori; citation_author=KF Aoki-Kinoshita; citation_author=M Itoh; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=18; citation_pages=D354-357; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Hash codes for the identification and classification of molecular structure elements.; citation_author=WD Ihlenfeldt; citation_author=J Gasteiger; citation_journal_title=J Comput Chem; citation_volume=15; citation_number=19; citation_pages=793-813; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Enhanced CACTVS browser of the open NCI Database.; citation_author=WD Ihlenfeldt; citation_author=JH Voigt; citation_author=B Bienfait; citation_author=F Oellien; citation_author=MC Nicklaus; citation_journal_title=J Chem Inf Comput Sci; citation_volume=42; citation_number=20; citation_pages=46-57; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=ExplorEnz: a MySQL database of the IUBMB Enzyme Nomenclature.; citation_author=AG McDonald; citation_author=S Boyce; citation_author=GP Moss; citation_author=HBF Dixon; citation_author=KF Tipton; citation_journal_title=BMC Biochem; citation_volume=8; citation_number=21; citation_pages=14; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Description of several chemical structure file formats used by computer programs developed at Molecular Design Limited.; citation_author=A Dalby; citation_author=JG Nourse; citation_author=WD Hounshell; citation_author=AKI Gushrst; citation_author=DL Grier; citation_journal_title=J Chem Inf Comput Sci; citation_volume=32; citation_number=22; citation_pages=244-255; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=RPAIR: a reactant-pair database representing chemical changes in enzymatic reactions.; citation_author=M Kotera; citation_author=M Hattori; citation_author=M Oh; citation_author=R Yamamoto; citation_author=T Komeno; citation_journal_title=Genome Inform; citation_volume=15; citation_number=23; citation_pages=P062; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Computational assignment of the EC numbers for genomic-scale analysis of enzymatic reactions.; citation_author=M Kotera; citation_author=Y Okuno; citation_author=M Hattori; citation_author=S Goto; citation_author=M Kanehisa; citation_journal_title=J Am Chem Soc; citation_volume=126; citation_number=24; citation_pages=16487-16498; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Concept-based annotation of enzyme classes.; citation_author=O Hofmann; citation_author=D Schomburg; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=25; citation_pages=2059-2066; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Specificity assay of serine proteinases by reverse-phase high-performance liquid chromatography analysis of competing oligopeptide substrate library.; citation_author=J Antal; citation_author=G Pal; citation_author=B Asboth; citation_author=Z Buzas; citation_author=A Patthy; citation_author=L Graf; citation_journal_title=Anal Biochem; citation_volume=288; citation_number=26; citation_pages=156-167; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The University of Minnesota Biocatalysis/Biodegradation Database: the first decade.; citation_author=LB Ellis; citation_author=D Roe; citation_author=LP Wackett; citation_journal_title=Nucleic Acids Res; citation_volume=34 (Database issue); citation_number=27; citation_pages=D517-21; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A Line-Formula Chemical Notation.; citation_author=WJ Wiswesser; citation_number=28; citation_date=1954; citation_publisher=Thomas Crowell; " />
      <meta name="citation_reference" content="citation_title=How the WLN began in 1949 and how it might be in 1999.; citation_author=WJ Wiswesser; citation_journal_title=J Chem Inf Comput Sci; citation_volume=22; citation_number=29; citation_pages=88-93; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Historic development of chemical notations.; citation_author=WJ Wiswesser; citation_journal_title=J Chem Inf Comput Sci; citation_volume=25; citation_number=30; citation_pages=258-263; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=ROSDAL: A universal structure/substructure representation for PC-host communication.; citation_author=JM Barnard; citation_author=CJ Jochum; citation_author=SM Welford; citation_number=31; citation_pages=76-81; citation_date=1989; citation_publisher=American Chemical Society; " />
      <meta name="citation_reference" content="citation_title=SMILES 1. Introduction and encoding rules. J Chem Inf Comput Sci 28: 31-36.; citation_author=D Weininger; citation_number=32; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=SMILES. 3. DEPICT. Graphical depiction of chemical structures.; citation_author=D Weininger; citation_journal_title=J Chem Inf Comput Sci; citation_volume=30(3); citation_number=33; citation_pages=237-243; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=SYBYL Line Notation (SLN): A versatile language for chemical structure representation.; citation_author=S Ash; citation_author=MA Cline; citation_author=RW Homer; citation_author=T Hurst; citation_author=GB Smith; citation_journal_title=J Chem Inf Comput Sci; citation_volume=37; citation_number=34; citation_pages=71-79; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Enhancement of the chemical semantic web through the use of InChI identifiers.; citation_author=SJ Coles; citation_author=NE Day; citation_author=P Murray-Rust; citation_author=HS Rzepa; citation_author=Y Zhang; citation_journal_title=Org Biomol Chem; citation_volume=3; citation_number=35; citation_pages=1832-1834; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Maximum common subgraph isomorphism algorithms for the matching of chemical structures.; citation_author=JW Raymond; citation_author=P Willett; citation_journal_title=J. Comput-Aided Mol Des; citation_volume=16; citation_number=36; citation_pages=521-533; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=RASCAL: Calculation of graph similarity using maximum common edge subgraphs.; citation_author=JW Raymond; citation_author=EJ Gardiner; citation_author=P Willett; citation_journal_title=Comput J; citation_volume=45; citation_number=37; citation_pages=631-644; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Heuristics for similarity searching of chemical graphs using a maximum common edge subgraph algorithm.; citation_author=JW Raymond; citation_author=EJ Gardiner; citation_author=P Willett; citation_journal_title=J Chem Inf Comput Sci; citation_volume=42; citation_number=38; citation_pages=305-316; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Development of a chemical structure comparison method for integrated analysis of chemical and genomic information in the metabolic pathways.; citation_author=M Hattori; citation_author=Y Okuno; citation_author=S Goto; citation_author=M Kanehisa; citation_journal_title=J Am Chem Soc; citation_volume=125; citation_number=39; citation_pages=11853-11865; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=On the properties of bit string-based measures of chemical similarity.; citation_author=DR Flower; citation_journal_title=J Chem Inf Comput Sci; citation_volume=38; citation_number=40; citation_pages=379-386; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=3D search and research using the Cambridge structural database.; citation_author=FH Allen; citation_author=O Kennard; citation_journal_title=Chem Des Autom News; citation_volume=8; citation_number=41; citation_pages=1 and 31-37; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Use of structure-activity data to compare structure-based clustering methods and descriptors for use in compound selection.; citation_author=RD Brown; citation_author=YC Martin; citation_journal_title=J Chem Inf Comput Sci; citation_volume=36; citation_number=42; citation_pages=572-584; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The information content of 2D and 3D structural descriptors relevant to ligand-receptor binding.; citation_author=RD Brown; citation_author=YC Martin; citation_journal_title=J Chem Inf Comput Sci; citation_volume=37; citation_number=43; citation_pages=1-9; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Molecular diversity and representativity in chemical databases.; citation_author=DM Bayada; citation_author=H Hamersma; citation_author=VJ van Geerestein; citation_journal_title=J Chem Inf Comput Sci; citation_volume=39; citation_number=44; citation_pages=1-10; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Comparison of 2D fingerprint types and hierarchy level selection methods for structural grouping using Ward's clustering.; citation_author=D Wild; citation_author=CJ Blankley; citation_journal_title=J Chem Inf Comput Sci; citation_volume=40; citation_number=45; citation_pages=155-162; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Daylight Theory Manual 4.71;; citation_author=CA James; citation_author=D Weininger; citation_author=J Delany; citation_number=46; citation_date=2000; citation_publisher=Daylight Chemical Information Systems, Inc; " />
      <meta name="citation_reference" content="citation_title=Knowledge-based expert systems for toxicity and metabolism prediction: Drug Metabolism:; citation_number=47; citation_pages=289-296; citation_date=1999; citation_publisher=Blackwell Science Ltd; " />
      <meta name="citation_reference" content="citation_title=Structure-biodegradability study and computer-automated prediction of aerobic biodegradation of chemicals.; citation_author=G Klopman; citation_author=M Tu; citation_journal_title=Environ Toxicol Chem; citation_volume=16; citation_number=48; citation_pages=1829-1835; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=PATTY: A programmable atom typer and language for automatic classification of atoms in molecular databases.; citation_author=BL Bush; citation_author=RP Sheridan; citation_journal_title=J Chem Inf Comput Sci; citation_volume=33; citation_number=49; citation_pages=756-762; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings.; citation_author=CA Lipinski; citation_author=F Lombardo; citation_author=BW Dominy; citation_author=PJ Feeny; citation_journal_title=Adv Drug Deliv Rev; citation_volume=23; citation_number=50; citation_pages=3-25; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001537" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Functional Group and Substructure Searching as a Tool in Metabolomics"/>
    <meta name="twitter:description" content="BackgroundA direct link between the names and structures of compounds and the functional groups contained within them is important, not only because biochemists frequently rely on literature that uses a free-text format to describe functional groups, but also because metabolic models depend upon the connections between enzymes and substrates being known and appropriately stored in databases.MethodologyWe have developed a database named &ldquo;Biochemical Substructure Search Catalogue&rdquo; (BiSSCat), which contains 489 functional groups, &gt;200,000 compounds and &gt;1,000,000 different computationally constructed substructures, to allow identification of chemical compounds of biological interest.ConclusionsThis database and its associated web-based search program (http://bisscat.org/) can be used to find compounds containing selected combinations of substructures and functional groups. It can be used to determine possible additional substrates for known enzymes and for putative enzymes found in genome projects. Its applications to enzyme inhibitor design are also discussed."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001537.g002"/>

  <meta property="og:title" content="Functional Group and Substructure Searching as a Tool in Metabolomics" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001537" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1922'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=416'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4805&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001537">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001537" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001537&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Functional Group and Substructure Searching as a Tool in Metabolomics
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Masaaki Kotera
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:koteram@tcd.ie">koteram@tcd.ie</a></p>

                <p>Affiliation:
                  School of Biochemistry and Immunology, Trinity College, Dublin, Ireland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrew G. McDonald, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biochemistry and Immunology, Trinity College, Dublin, Ireland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sinéad Boyce, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biochemistry and Immunology, Trinity College, Dublin, Ireland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Keith F. Tipton
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biochemistry and Immunology, Trinity College, Dublin, Ireland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 06, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001537</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001537" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001537" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001537" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001537" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001537-t001" data-doi="info:doi/10.1371/journal.pone.0001537" data-uri="info:doi/10.1371/journal.pone.0001537.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001537-g001" data-doi="info:doi/10.1371/journal.pone.0001537" data-uri="info:doi/10.1371/journal.pone.0001537.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001537-t002" data-doi="info:doi/10.1371/journal.pone.0001537" data-uri="info:doi/10.1371/journal.pone.0001537.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001537-g002" data-doi="info:doi/10.1371/journal.pone.0001537" data-uri="info:doi/10.1371/journal.pone.0001537.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001537">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001537" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>A direct link between the names and structures of compounds and the functional groups contained within them is important, not only because biochemists frequently rely on literature that uses a free-text format to describe functional groups, but also because metabolic models depend upon the connections between enzymes and substrates being known and appropriately stored in databases.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We have developed a database named “Biochemical Substructure Search Catalogue” (BiSSCat), which contains 489 functional groups, &gt;200,000 compounds and &gt;1,000,000 different computationally constructed substructures, to allow identification of chemical compounds of biological interest.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This database and its associated web-based search program (<a href="http://bisscat.org/">http://bisscat.org/</a>) can be used to find compounds containing selected combinations of substructures and functional groups. It can be used to determine possible additional substrates for known enzymes and for putative enzymes found in genome projects. Its applications to enzyme inhibitor design are also discussed.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kotera M, McDonald AG, Boyce S, Tipton KF (2008) Functional Group and Substructure Searching as a Tool in Metabolomics. PLoS ONE 3(2):
          e1537.
            doi:10.1371/journal.pone.0001537</p><p><strong>Academic Editor: </strong>Ji Zhu, University of Michigan, United States of America</p><p><strong>Received:</strong> October 1, 2007; <strong>Accepted:</strong> January 6, 2008; <strong>Published:</strong> February 6, 2008</p><p><strong>Copyright:</strong> © 2008 Kotera et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>We are grateful to Science Foundation Ireland (grant No. SFI 02/IN.1/B043- Tipton) for support.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Nomenclature is of fundamental importance in science <a href="#pone.0001537-Fox1">[1]</a>–<a href="#pone.0001537-Wisniewski1">[4]</a>. Without reliable nomenclature systems, it would be difficult to know if each person in a discussion was talking about the same thing, and carrying out a literature survey would be almost impossible. Nomenclature not only gives a name to objects, but can also be used to characterize objects. Previously, nomenclature systems were the only way to look up chemical compounds or enzymes of interest. However, the rapid increase in the volume of literature and scientific data is making the use of computer algorithms unavoidable in the search for relevant data.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Missing connections between metabolites is a major problem of metabolic modelling. Just as gene-sequence studies have revealed many putative enzymes with unknown substrates (orphan enzymes), metabolomic studies are revealing a plethora of orphan substrates, which makes the need for rational approaches to identifying the enzymes involved in their formation and breakdown a pressing concern. In this context, orphan substrates may be defined in different ways. Poolman et al. <a href="#pone.0001537-Poolman1">[5]</a> defined “orphan metabolites” as “metabolites involved in only one reaction” and “dead-end metabolites” as “metabolites involved in more than one reaction, but having no producing or no consuming reaction”. Although some metabolites, such as lactate, may be metabolic end products, in other cases the situation simply reflects a lack of knowledge. Both kinds of metabolite may cause the network to be unbalanced. Here we define an orphan substrate as one that is known to occur physiologically but neither the reaction to synthesize it nor degrade it are yet known. This kind of metabolite is problematic in metabolic-modelling studies, making it important to determine the possible reaction(s) in which it is involved. The same may apply to xenobiotics, many of which are either metabolized in some organisms or interact specifically with enzymes or “receptors”. A systematic approach based on the chemical structure of the metabolite should be of value in this respect.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The relationship between a chemical structure and its reactivity has been well investigated in pharmacology, the first step of which is pharmacophore searching prior to more detailed molecular analysis [see, e.g., 6–15]. There are a variety of tools for substructure searching, but their main purpose is drug design rather than novel pathway discovery. It is also hoped that BiSSCat will be useful for preliminary screening prior to more detailed molecular modelling studies and QSAR analysis.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In the field of organic chemistry, functional groups have been defined as atoms or atom groups that show relatively constant characteristics even when connected to different structures <a href="#pone.0001537-Leigh1">[3]</a>. Researchers who are interested in chemical compounds in living organisms face several specific problems. They might want to find the common features of a group of substrates for certain enzymes, or how a group of substrates is converted into other types of compounds, even when the total structures are not specified. Interactions between proteins and small chemical compounds, including enzymatic reactions, follow the same rules that apply in organic chemistry but also have some specific characteristics. Recognition of small compounds and catalytic mechanisms is usually much more complex than that found with catalysts in organic chemistry, making it difficult to predict the fate of chemical compounds in living organisms.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The most reliable clue for guessing the function of putative genes is protein sequence similarity to well-investigated gene products, but such annotations have to be interpreted with caution. This is because they inevitably include uncertainty associated with each of the steps from enzyme studies to genome annotation. Most enzyme-specificity studies are not exhaustive, because experimentalists are generally interested in identifying the presumed physiological substrate(s) and inhibitor(s), or artificial substrates that make enzyme assays easier to perform. Substitution of even a single amino-acid residue can cause changes in terms of substrate specificity or reactivity. The label of being “similar to” well-investigated genes provides a suggestion about function, but does not necessarily describe functional identity, which further increases the uncertainty associated with annotations.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Although some enzymes have very narrow substrate specificity, others are known to display wider substrate specificity. Metabolome analyses have uncovered many secondary metabolites that appear to be species specific and it has been suggested that broad substrate specificity may contribute to metabolome diversity <a href="#pone.0001537-Schwab1">[16]</a>. It has also been suggested that relaxed substrate and reaction specificities can have an important role in enzyme evolution <a href="#pone.0001537-Hult1">[17]</a>. Ideally, each of these enzyme specificities should be confirmed experimentally, however, it is practically impossible to check all enzymes for all compounds at present, as such experiments would be both costly and time-consuming to perform.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>We propose that studies on enzymes or compounds that have been less thoroughly investigated should be made without making any assumptions about enzyme specificity. This provides a starting point for the consideration of possible combinations of recognized and putative enzymes (gene products) and their functions (enzyme reactions) in an expanding set of gene products and metabolites. Substrate specificity is generally described using a free-text description of the functional groups involved, the generic names of compounds, or one or more equations that describe the reaction(s) catalysed. These are subsequently used in genome annotations. Enzymes and their substrates are sometimes identified by class names. For example, the names alcohol dehydrogenase (EC 1.1.1.1) and amine oxidase (EC 1.4.3.4) give no indication of the breadth of the specificities of these enzymes. Indeed, it is likely that several possible substrates for such enzymes are not registered as substrates in reaction databases, because they have not been studied. Such a lack of precision highlights the need to make the relationship among compounds' names, class names, substructures and functional groups clear.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>In this paper, we have defined substructures that include known functional groups, and made it possible to obtain chemical compounds from biochemical databases. We have also provided a web-based tool (<a href="http://bisscat.org/">http://bisscat.org/</a>) for searching defined substructures and obtaining a list of compounds containing them. One can combine a number of defined substructures to produce more complicated substructures, and can search for enzymes based on functional groups. As an example of what can be achieved using BiSSCat, we have determined which substructures are commonly used by a particular group of enzymes, and then proposed some possible candidate compounds that could act as substrates of those enzymes. Since substructure and location are important for all ligand-binding processes, this approach should also be of wider value. Furthermore, it should help to connect nomenclature and machine-readable expressions of chemical compounds, and to fill in the gaps in our knowledge of genomic and metabolomic relationships.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>The two major original parts of the BiSSCat dataset are SUBSTRUCTURE and FGROUP. The SUBSTRUCTURE part is constructed computationally and stores a collection of biochemical substructures. These were calculated using several different concepts, including the distinction between elements based on their electrostatic and physicochemical properties (<a href="#pone-0001537-t001">Table 1</a>). The FGROUP part comprises an index of names for functional groups and other biochemical substructures, which enables one to look up the substructure easily.</p>
<div class="figure" id="pone-0001537-t001"><div class="img"><a name="pone-0001537-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001537" data-uri="info:doi/10.1371/journal.pone.0001537.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001537.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001537.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Physicochemical properties defined in SUBSTRUCTURE.</span></strong></p><span>doi:10.1371/journal.pone.0001537.t001</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>The names used in FGROUP were assigned manually with the aid of the web-based BiSSCat substructure-search tool (described below). The merit of having this sub-database is that one can search for any substructure using a number of names without bothering about the definition of SUBSTRUCTURE entries unless one has a very complicated query. Most functional groups referred to in the IUBMB Enzyme List are covered, so the selection of FGROUP entries is currently biased for use with enzymatic reactions. For instance, many organic functional groups such as alcohols are further divided into their subgroups (primary, secondary and tertiary alcohols), whereas inorganic functional groups are not so detailed. It is hoped that BiSSCat users will give us feedback on any omissions. The database is designed so that a group of substructures can share one FGROUP, and a single substructure can belong to two or more FGROUPs. This rule might seem complicated, but it reflects the situation found in nature. For example, aldehyde, carboxylate, and amide groups belong to the carbonyl functional group, whereas the N-formyl group belongs to both the aldehyde and amide functional groups. Enzymes and other proteins often recognize more of a substructure than just the functional group(s), and the threshold for distinguishing between these is not always obvious. Therefore, FGROUP assigns names not only for functional groups but also for some larger substructures, such as sugars, which are specifically recognized by glycosyltransferases, glycosidases, <em>etc.</em></p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>The database currently comprises 241,709 chemical compounds whose non-hydrogen atoms are classified into 2,736 different ATOM entries. Each ATOM entry is given an ID number (ATOM0001–ATOM2736) based solely on its order of inclusion in the BiSSCat database. There are also 1,857,839 SUBSTRUCTURE entries in the database. Serial ID numbers are also assigned to these SUBSTRUCTURE entries (S0000001–S1857839) and, as discussed below, the IDs bear no relation to substructure type.</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>489 FGROUP entries were assigned for the current release (as of January 1, 2007). These correspond to 660,946 recognized SUBSTRUCTURE entries and to 4,964,487 non-hydrogen-atom locations in the KEGG <a href="#pone.0001537-Kanehisa1">[18]</a> and NCI <a href="#pone.0001537-Ihlenfeldt1">[19]</a>, <a href="#pone.0001537-Ihlenfeldt2">[20]</a> databases, which have been constructed for different purposes (containing mostly endogeneous compounds and xenobiotics, respectively) and have minimum overlap between them. An overall view of the classification of FGROUP entries is summarized in <a href="#pone-0001537-g001">Figure 1</a> (the complete set defined to date is available at <a href="http://bisscat.org/fgroup.html">http://bisscat.org/fgroup.html</a>). ID numbers are given to FGROUP entries in such a way that they approximate to a hierarchical classification. The FGROUP list does not strictly reflect classification of physicochemical or biochemical characteristics. Since the classification of some functional groups can be based on a number of different aspects, it is impossible to describe the classification of functional groups in a simple tree structure. There are 2,357 instances in the database where all atoms in a functional group are part of those in another functional group, and 8,625 cases where two functional groups share some atoms. The FGROUP list can be expanded to accommodate newly defined functional groups or substructures in the future.</p>
<div class="figure" id="pone-0001537-g001"><div class="img"><a name="pone-0001537-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001537" data-uri="info:doi/10.1371/journal.pone.0001537.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001537.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001537.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Partial classification tree of FGROUP: stars indicate FGROUP entries on which no enzymes are known to act.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>A complete list of FGROUP entries can be seen at <a href="http://bisscat.org/fgroup.html">http://bisscat.org/fgroup.html</a>.</p>
<span>doi:10.1371/journal.pone.0001537.g001</span></div><a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p><a href="#pone-0001537-t002">Table 2</a> provides details of the numbers of SUBSTRUCTURE and FGROUP entries that occur in KEGG but not in the NCI databases and vice versa. This table also gives details of the number of substructures known to be involved in reaction centres and in enzyme reactions. Some SUBSTRUCTURE entries were found in KEGG that were not present in the NCI databases. These SUBSTRUCTURE entries contained “R” (representing omission of substructures such as alkyl groups) and “*” (representing repetition in polymers such as glycans, nucleotides and proteins). FGROUP entries found only in the NCI databases are functional groups for which no enzyme reactions have been recorded, such as ammonium ylide, thioaldehyde, phosphine, silane and stannane. The FGROUP entries in KEGG that are not in the NCI database include thiamine, fluorophosphate, chlorophyll, heme and cobalamin. These are listed on the BiSSCat website. Acyl halides and alkyl magnesium halides are important agents in organic chemistry, but were not found in either database because they are generally unstable under physiological conditions.</p>
<div class="figure" id="pone-0001537-t002"><div class="img"><a name="pone-0001537-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001537" data-uri="info:doi/10.1371/journal.pone.0001537.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001537.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001537.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Statistics on SUBSTRUCTURE and FGROUP entries in BiSSCat.</span></strong></p><span>doi:10.1371/journal.pone.0001537.t002</span></div>
<h4>The web-based substructure-search tool</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>BiSSCat provides a number of alternative ways of looking up chemical compounds or biochemical substructures. Here we give an outline of the web-based program (<a href="http://bisscat.org/">http://bisscat.org/</a>), and further details are provided on the website's help page. The user must install an Adobe SVG plug-in (<a href="http://www.adobe.com/svg/">http://www.adobe.com/svg/</a>) and enable cookies in order to use these tools. Screenshots of the webpage are shown in <a href="#pone-0001537-g002">Figure 2</a>. Each chemical compound entry has an automatically generated interactive SVG image, which can be used to find the substructure of interest. The text-search option (located at the top of the homepage) can be used to search for (1) any term for compounds, functional groups, substructures and enzymes, (2) molecular formulae of compounds, functional groups and substructures and (3) EC numbers and other IDs registered in BiSSCat. One can use the text-search option to search the whole of BiSSCat or one can limit the search to compound, FGROUP, enzyme, reaction or SUBSTRUCTURE by selecting the item of interest from the drop-down menu.</p>
<div class="figure" id="pone-0001537-g002"><div class="img"><a name="pone-0001537-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001537" data-uri="info:doi/10.1371/journal.pone.0001537.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001537.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001537.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001537.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001537.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001537.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Screenshots of BiSSCat website: homepage (top), an example entry of FGROUP (left) and of SUBSTRUCTURE (right).</span></strong></p><span>doi:10.1371/journal.pone.0001537.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Three types of objects, FGROUP (or SUBSTRUCTURE), compound and enzyme, can each be searched in three different ways (by name, tree and structure). The first way is using the alphabetically ordered list of these objects' names. The second way is to use the hierarchical classification tree. The difference between an FGROUP and a compound's classification can be explained using “hexopyranose” as an example. Hexopyranose is a word used to describe a class of compounds with the molecular formula C<sub>6</sub>H<sub>12</sub>O<sub>6</sub>, and containing four hydroxy groups and one cyclic hemiacetal within a six-membered ring. 22 hexopyranose compounds can be found in the current release of the database, with examples being glucose and mannose. The corresponding FGROUP entry shows the substructures involved in compounds such as glycans, of which, 1,661 compounds can be found.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The third way of searching the database, i.e. the structure search option, needs further explanation. Searches of FGROUP, SUBSTRUCTURE and compound entries can be based on elements, electrostatic and physicochemical properties, and graph topology. For example, aryl carboxylate contains C<sub>2</sub>O<sub>2</sub> with the central carbon atom being a carboxylate carbon (cx), the other being an aromatic carbon (ar) and two oxygen anions (ep2). There are 17,925 SUBSTRUCTURE entries containing C<sub>2</sub>O<sub>2</sub>, which includes many FGROUP entries that are not carboxylates (e.g., F33400 Carboxylate ester). Among them, 2,988 entries have one carboxylate carbon and two negative oxygen atoms and these belong to six FGROUP entries that have “carboxylate” in their name (F331000 Carboxylate, F331100 Alkyl carboxylate, F331200 Allyl carboxylate, F331400 Aryl carboxylate, F331300 Acetylene carboxylate and F331400 2-Oxo carboxylate). 305 SUBSTRUCTURE entries are obtained when “aromatic” is added to the search condition.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Another option is to search for compounds based on structural information. Using the “Multiple Substructure Search” option, one can find compounds based on the presence or absence of substructures or functional groups. This can greatly increase the specificity of the search, and reduce the number of compounds to consider. For example, there are 55 compounds in the database that have “carboxylate” in their name but there are 22,160 compounds that contain the “carboxylate” structure. There are 685 compounds containing adenine in the database but there are only 62 compounds that contain both carboxylate and adenine. Of these, 28 of the compounds do not contain a thioester group.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>FGROUP entries in reaction equations can also be searched to find enzymes. For example, reaction equations that include generic names such as “alcohol+NAD = aldehyde+NADH” can be searched. Partial equations, such as “alcohol = aldehyde” or “amine = aldehyde” can also be used.</p>


<h4>Enzyme reactions and substructures</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>There are currently more than 4030 enzymes with assigned EC numbers (see <a href="http://www.enzyme-database.org/">http://www.enzyme-database.org/</a>) <a href="#pone.0001537-McDonald1">[21]</a>. Enzyme classification is based on the reaction catalysed but the type of reaction given for different enzymes falls into different categories, as follows: (1) one specific reaction, (2) a reaction is given where one of the physiological substrates is not known, (3) a general reaction is assigned to represent large polymers such as a glycan, nucleic acid or protein, (4) a general reaction is given, because the enzyme has a wide substrate specificity, (5) two or more reactions are provided, where the enzyme catalyses the same type of reaction but with different substrates, and (6) a multi-step reaction is catalysed and the overall reaction is given a single EC number. In cases where an enzyme catalyses two or more distinctly different types of reactions, EC numbers are given to each of these reactions. The descriptors of reactant structures (Molfiles <a href="#pone.0001537-Dalby1">[22]</a>) are incomplete in cases (2), (3) and (4), so it is difficult to automatically obtain the corresponding compounds from existing databases.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>FGROUP entries can be used to obtain specific compounds in the case of (4) above, where a class name is used in the reaction equation. As an example, EC 1.1.1.1 (alcohol dehydrogenase) comprises enzymes that oxidize alcohols with the concomitant reduction of NAD<sup>+</sup>. The term “alcohol” does not refer to a specific chemical compound, but is a generic term used for any chemical compound containing one or more hydroxy groups. Suppose that we would like to identify a dehydrogenase that acts on a newly identified alcohol. EC 1.1.1.1 acts on a limited set of alcohols, although the substrate specificity of such an enzyme depends upon its origins, i.e., species and tissue. Different enzymes accept different sets of alcohols, but it is not known how substrate specificity could change in orthologous enzymes. The substrate specificity of this enzyme is clarified in the comments' section of the IUBMB enzyme entry, where it states that it “Acts on primary or secondary alcohols or hemi-acetals; the animal, but not the yeast, enzyme acts also on cyclic secondary alcohols”. The terms “primary alcohol”, “secondary alcohol”, “hemi-acetal” and “cyclic secondary alcohol” are registered in the BiSSCat database and there are 13,371, 20,230, 775 and 12,660 examples of each, respectively. In this way, FGROUPs can provide a number of possible substrates for enzymes described in generic expressions.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>In the case of (5), as more than one specific compound is named as a substrate/product, it is possible to deduce substructures that are common to each substrate and/or product. For example, EC 2.1.1.50 (loganate O-methyltransferase) acts on two compounds, loganate and secologanate. The structural difference between these two substrates is, therefore, not sufficient to prevent recognition by the enzyme. Substructures were divided into two groups: those containing reaction centre atoms and those containing other substructures that might be recognized by the enzyme. A compound that has both of these attributes may be considered to be a possible candidate substrate for that enzyme.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>In a preliminary analysis, candidate substrates were defined as those compounds having one substructure involving a reaction centre and at least three substructures found in a reported substrate for a given enzyme. Application of these criteria to the compounds in the BiSSCat database showed that 1,912 known substrates have more than 10 related structures that were, therefore, candidate substrates, 1,166 known substrates had between 1 and 10 other candidate substrates, and 934 had no alternative candidate substrates.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>In cases where only a single specific reaction is provided, it is not possible to determine commonly used substructures, as there is no means of making a comparison. Some of the enzymes in the IUBMB Enzyme List appear to have narrow substrate specificities, so there might seem to be little need to predict other possible substrates. However, this may be a reflection of lack of knowledge. Furthermore, such information would be valuable if one needs to find the function of the corresponding orthologous gene products. Reaction centres can be defined as in the RPAIR database <a href="#pone.0001537-Kotera1">[23]</a>, <a href="#pone.0001537-Kotera2">[24]</a>. The reaction equation itself is not enough to determine which substructures are recognized by an enzyme, although the BRENDA database (<a href="http://www.brenda.uni-koeln.de/">http://www.brenda.uni-koeln.de/</a>) provides additional data on the specificities of many enzymes. In a situation where no information other than the reaction equation is available, the best one can do is to find compounds with the same types of atoms or functional group(s). Substructure searches of the BiSSCat database can be used to find atoms in the same environment. Among the compounds that are not currently known to be associated with any enzyme reaction, 62,402 compounds have the same type of atoms as those involved in reported enzyme reactions, and 2,182 of these are from the KEGG database.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>One example is a group of compounds that include the 5-methylcytidine residue SUBSTRUCTURE entry (S0265987), i.e., deoxy-5-methylcytidine (1), DNA 5-methylcytosine (2), 5-methyldeoxycytidine diphosphate (3) and 5-methyl-2′-deoxycytidine (4). Deoxy-5-methylcytidine can be balanced in metabolic modelling as it is known to be involved in two enzyme reactions (EC 2.1.1.54 and EC 2.7.4.19). DNA 5-methylcytosine and 5-methyldeoxycytidine diphosphate are involved in only one reaction each (EC 2.1.1.37 and EC 2.7.4.19, respectively), and are examples of ‘orphan metabolites’, as defined by Poolman et al. <a href="#pone.0001537-Poolman1">[5]</a>. Such orphan metabolites cause problems in metabolic modelling. No enzymes have been reported that act on 5-methyl-2′-deoxycytidine (4) but this does not cause the same types of problems as for compounds (2) and (3). However, it is expected that some reactions would involve compound (4) if it is naturally occurring. Substructure comparisons indicate that candidate enzymes would include deoxycytidine deaminase [EC 3.5.4.14] and deoxycytidine kinase [EC 2.7.1.74]. Two of the four compounds are involved in reactions that are catalysed by methyltransferases [EC 2.1.1.54 and EC 2.1.1.37 for (1) and (2), respectively], making it likely that methyltransferases also act upon compounds (3) and (4) to produce deoxycytidine diphosphate and 2′-deoxycytidine, respectively. The fact that EC 2.7.4.29 acts on both (1) and (3) also lends support to the presumption that there could be other enzymes acting on both compounds.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Although the approach taken in this study cannot ensure that a compound is truly a substrate for a given enzyme, it should help to minimize the number of candidate enzymes and compounds for experimental investigation. Further analysis of substructure changes during a reaction using RPAIR revealed that there were sometimes no corresponding products for the proposed substrates. A solution to this problem might be the addition of potential products to compound databases, however, it would first be preferable to confirm the existence of the predicted substrates/products experimentally, to avoid the inclusion of misleading information.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The BiSSCat substructure searching method is applicable to finding possible substrates having binding groups as well as a reaction centre. The process could also be applicable to identifying compounds that are unlikely to be substrates or might be inhibitors of a given enzyme. For example, EC 1.4.3.4 (monoamine oxidase) acts on many compounds that contain a primary amine group. If these substrates also contain a carboxy group, this can prevent the compound from being bound to the enzyme. The presence of an alpha-methyl group will not prevent binding of the substrate to the enzyme, but it does block the conversion of the substrate into the product. If information about binding groups and blocking groups is already known, BiSSCat can be used as an aid to the design of inhibitors. Such data are, in many instances, not presented explicitly in extant databases.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>It is intended to further enrich BiSSCat with data about interactions between proteins and small compounds from the existing literature that are not in the present source databases and to incorporate results of future experiments. Several newer techniques, such as text mining of the enzyme-assay literature <a href="#pone.0001537-Hofmann1">[25]</a> and high-performance systematic assays to determine substrate specificity <a href="#pone.0001537-Antal1">[26]</a>, can be applied. Needless to say, it is important to have a large collection of positive data, but the same can be said about negative data, i.e., compounds that the enzyme does not act on. It is meaningful to take account of compounds acting on enzymes in vivo, but it is also valuable to collect data about synthetic compounds that have not been observed in vivo. Information about mutated enzymes would also be valuable for enzyme-engineering purposes. It is intended to incorporate relevant data from more sources, including ERGO-light (<a href="http://www.ergo-light.com/">http://www.ergo-light.com/</a>) and UMBBD <a href="#pone.0001537-Ellis1">[27]</a>, in future developments of BiSSCat.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Given that the objectives of searching complete chemical structures and substructures are usually different, the search methods used are closely related to how they are represented. The first step of our method is to divide a chemical compound into its inherent substructures, which is similar to the first step in obtaining a systematic nomenclature for chemical compounds, such as obtained using IUPAC rules, and a variety of linear representations of chemical compounds, such as WLN <a href="#pone.0001537-Wiswesser1">[28]</a>–<a href="#pone.0001537-Wiswesser3">[30]</a>, ROSDAL <a href="#pone.0001537-Barnard1">[31]</a>, SMILES <a href="#pone.0001537-Weininger1">[32]</a>, <a href="#pone.0001537-Weininger2">[33]</a>, SLN <a href="#pone.0001537-Ash1">[34]</a> and InChI <a href="#pone.0001537-Coles1">[35]</a>. The steps thereafter are different. When searching complete structures, the inherent substructures have to be arranged according to predefined rules, as it is essential that each chemical structure has only a single representation. This is not necessary for substructure searches, where users can freely modify the search criteria according to their needs. Graph-oriented algorithms applying maximum common subgraph isomorphism <a href="#pone.0001537-Raymond1">[36]</a>–<a href="#pone.0001537-Hattori1">[39]</a> are better than fragment code or fingerprint methods <a href="#pone.0001537-Flower1">[40]</a>–<a href="#pone.0001537-James1">[46]</a> in terms of precision when searching for compounds in databases that are similar to the query structure, although they present difficulties in terms of computational time (the graph isomorphism problem is NP-hard) and in the interpretation of the derived subgraphs.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Our method takes advantage of a pre-computed and assigned set of substructures, making the search speed faster and interpretation easier. The manual assignment of FGROUP was the most time-consuming process in the construction of the BiSSCat database, but it was an important step as it provides a direct correspondence between the generic names described in the IUBMB Enzyme List and the concrete substructures found in chemical-compound databases. This should make it easier for computer algorithms to distinguish between generic names and specific names. More importantly, it also makes it easier to understand the meanings of substructures found in computational analysis, which could help our understanding of the structure-function relationships of ligand-binding processes, including enzymes.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Both the database and search program have scope for further development, for instance by allowing the user to define distances between substructures, input substructures using SMILES or SMART format, or use a structure-drawing tool. These aspects will be addressed in future releases. We believe that our method should be of value in gene-product identification and in increasing our understanding of previously unknown metabolic pathways or drug-selection processes.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data sources</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The SUBSTRUCTURE database was constructed using data on the structures of 10,046 and 247,617 chemical compounds derived from the KEGG <a href="#pone.0001537-Arimoto1">[8]</a> and NCI <a href="#pone.0001537-Poroikov1">[9]</a>, <a href="#pone.0001537-Oellien1">[10]</a> databases, respectively, in MDL Molfile format. For convenience, the original database IDs assigned to compounds have not been changed, so that they can be used to link to the corresponding data in the source databases. Information on reported activities, such as enzyme substrates and products, is also provided so that one can search and analyse compounds using these data. Most compounds from KEGG are known to be involved in metabolism in living organisms. Most compounds from NCI include other valuable information, such as <em>logP</em>, the octanol/water partition coefficient <a href="#pone.0001537-Greene1">[47]</a>, <a href="#pone.0001537-Klopman1">[48]</a> as well as anti-cancer and anti-HIV screening results.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>In order for a reaction to be catalysed, a chemical compound has to contain the appropriate functional groups, also referred to as the reaction centre. The KEGG/RPAIR database describes which atom in a substrate corresponds to which atom in a product in each enzyme reaction. The RPAIR database also defines reaction centre atoms, which undergo significantly more changes than other atoms in the reactant-pair during a reaction. These reaction-centre atoms are utilized in this study.</p>


<h4>Calculation of SUBSTRUCTURE</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Biochemical substructures are computationally defined using seven attributes: atom (ATOM), vicinity (VICI), bond (BOND), skeleton (SKEL), ring (RING), fragment (FRAG) and conjugate (CONJ). Every substructure is represented as a graph object, with non-hydrogen atoms and bonds described as nodes and edges, respectively. Each substructure is distinguished in terms of its elements (C for carbon, N for nitrogen, etc.), electrostatic and physicochemical properties, and topology. Detailed definitions of the substructure types are provided below.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>ATOM entries are distinguished by their elements and by their electrostatic and physicochemical properties, which are calculated for each non-hydrogen atom of each compound. Hydrogen atoms are not assigned individual ATOM entries, but are included with their adjacent non-hydrogen atoms. <a href="#pone-0001537-t001">Table 1</a> shows the list of electrostatic and physicochemical properties defined in ATOM and other substructure entries. Most of these properties are based on the programmable atom typer program, PATTY <a href="#pone.0001537-Bush1">[49]</a>. Ring properties are an exception and they are explained later in this section. Physicochemical properties are provided for each non-hydrogen atom rather than for the total structure of the chemical compound. For example, while ethanol (CH3CH2OH) is a hydrophilic molecule, using the PATTY method, the two carbon atoms of the ethyl group (CH3CH2-) and the oxygen atom of the hydroxy group (-OH) are assigned as being “hydrophobic” (ep6) and “polar” (ep5), respectively.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>VICI entries are defined in terms of ATOM entries. Other substructures (BOND, SKEL, RING, FRAG and CONJ) are defined in terms of VICI entries. A VICI entry is defined as a central atom and the atoms attached to it. Many functional groups correspond to VICI entries, e.g., carbamate, <em>N</em>-acetyl, and phosphate. A BOND entry is defined as a central bond between a pair of atoms, such as an amide bond. A SKEL entry is defined as a carbon skeleton/backbone, and examples include alkyl and aryl groups.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>A RING entry is defined as a cyclic substructure, containing 3-, 4-, 5- and 6-membered, or larger, rings. Some common examples are the phenyl, imidazole and pyrrole rings. Ring properties are also added to each ATOM entry if the atom is part of a 3-, 4-, 5- or 6-membered ring. These additional properties were added as 3- and 4-membered rings have especially strong ring strain, which gives rise to their specific reactivities (such as EC 3.3.2.3, epoxide hydrolase, which acts on epoxide). 5- and 6-membered rings are ubiquitous substructures, as found in many sugars etc., and many reactions are known to produce 5- and 6-membered rings. Larger cyclic substructures are not described in ATOM entries but are included in RING entries.</p>
<a id="article1.body1.sec4.sec2.p5" name="article1.body1.sec4.sec2.p5"></a><p>A FRAG entry is defined as a fragment obtained when all rotatable bonds are cut. A rotatable bond is defined in the following way: only a single bond (saturated bond) that is not included in any ring substructure can be rotated. Amide bonds are not rotatable, as they are known to have an energy barrier that prevents rotation. Two cases that remain to be incorporated are where steric hindrance prevents rotation, and where an enzymic reaction helps rotation (such as occurs with <em>cis-trans</em>-isomerases). A bond consisting of one hydrogen atom and one non-hydrogen atom is also excluded. Using this definition, many biologically important polycyclic structures, such as purines, pyrimidines, hemes or sterols, are obtained. Considering rotatable bonds should also be helpful in understanding the conformational changes that occur when a chemical compound is accepted by an enzyme. In pharmacology, an important step of drug design is determining the number of rotatable bonds of possible medicinal compounds <a href="#pone.0001537-Lipinski1">[50]</a>.</p>
<a id="article1.body1.sec4.sec2.p6" name="article1.body1.sec4.sec2.p6"></a><p>Finally, a CONJ entry is defined as a conjugated double or triple bond, i.e., a substructure with delocalized electrons. Technically speaking, CONJ entries are defined as connected sub-graphs consisting only of bonds where each of the two atoms has at least one resonance (res), conjugated double or triple bond (conj) or aromatic ring (ar) property. It is known that the delocalization of electrons leads to unique physicochemical characteristics and reactivities. In fact, CONJ includes many important substructures, such as 2-oxo carboxylate and triphosphate, which are found widely in biochemistry, and carotenoids and pheophytins, which are also found in pigments.</p>
<a id="article1.body1.sec4.sec2.p7" name="article1.body1.sec4.sec2.p7"></a><p>Substructures may be derived from other substructure types, which is the reason that IDs bear no relation to the type of substructure. For example, a phenyl ring is derived not only from the definition of RING entries, but also of FRAG entries (and CONJ entries in most cases). When a phenyl ring is connected to a heteroatom, the ring will also have a SKEL entry.</p>

</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MK. Performed the experiments: MK. Analyzed the data: KT MK AM SB. Contributed reagents/materials/analysis tools: MK AM. Wrote the paper: KT MK SB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001537-Fox1" id="pone.0001537-Fox1"></a> Fox RB, Powell WH, editors.  (2001) Nomenclature of Organic Compounds: Principles and Practice, 2nd Ed. Oxford, New York: American Chemical Society, Washington, DC/Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001537-International1" id="pone.0001537-International1"></a>International Union of Pure and Applied Chemistry, Commission on the Nomenclature of Organic Chemistry (1979) The Nomenclature of Organic Chemistry. Sections A, B, C, D, E, F and H, Pergamon, Oxford.   <a href="http://www.iupac.org">http://www.iupac.org</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001537-Leigh1" id="pone.0001537-Leigh1"></a>Leigh GJ, Favre HA, Metanomski WV (1998) Principles of Chemical Nomenclature: A Guide to IUPAC Recommendations. Malden, MA: Blackwell Science.   <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0001537-Wisniewski1" id="pone.0001537-Wisniewski1"></a>Wisniewski JL (1998)  Nomenclature: automatic generation and conversion. In: Schleyer Pvon R, Allinger NL, Clark , Gasteiger J, Kollman PA, Schaefer HF, Schreiner PR, editors. Encyclopedia of Computational Chemistry. Chichester: Wiley. pp. 1881–1894.  <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0001537-Poolman1" id="pone.0001537-Poolman1"></a>Poolman MG, Bonde BK, Gevorgyan A, Patel HH, Fell DA (2006) Challenges to be faced in the reconstruction of metabolic networks from public databases. IEE Proc Syst Biol  153: 379–384.  <ul class="find" data-citedArticleID="1050214" data-doi="10.1049/ip-syb:20060012"><li><a href="http://dx.doi.org/10.1049/ip-syb:20060012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Challenges+to+be+faced+in+the+reconstruction+of+metabolic+networks+from+public+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Challenges+to+be+faced+in+the+reconstruction+of+metabolic+networks+from+public+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001537-Khedkar1" id="pone.0001537-Khedkar1"></a>Khedkar SA, Malde AK, Coutinho EC, Srivastava S (2007) Pharmacophore modeling in drug discovery and development: an overview. Med Chem  3: 187–197.  <ul class="find" data-citedArticleID="1050192" data-doi="10.2174/157340607780059521"><li><a href="http://dx.doi.org/10.2174/157340607780059521" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pharmacophore+modeling+in+drug+discovery+and+development%3A+an+overview." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pharmacophore+modeling+in+drug+discovery+and+development%3A+an+overview.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001537-Chang1" id="pone.0001537-Chang1"></a>Chang C, Ekins S, Bahadduri P, Swaan PW (2006) Pharmacophore-based discovery of ligands for drug transporters. Adv Drug Deliv Rev  58: 1431–1450.  <ul class="find" data-citedArticleID="1050160" data-doi="10.1016/j.addr.2006.09.006"><li><a href="http://dx.doi.org/10.1016/j.addr.2006.09.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pharmacophore-based+discovery+of+ligands+for+drug+transporters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pharmacophore-based+discovery+of+ligands+for+drug+transporters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001537-Arimoto1" id="pone.0001537-Arimoto1"></a>Arimoto R (2006) Computational models for predicting interactions with cytochrome p450 enzyme. Curr Topics Med Chem  6: 1609–1618.  <ul class="find" data-citedArticleID="1050146" data-doi="10.2174/156802606778108951"><li><a href="http://dx.doi.org/10.2174/156802606778108951" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+models+for+predicting+interactions+with+cytochrome+p450+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+models+for+predicting+interactions+with+cytochrome+p450+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001537-Poroikov1" id="pone.0001537-Poroikov1"></a>Poroikov VV, Filimonov DA, Ihlenfeldt WD, Gloriozova TA, Lagunin AA, et al.  (2003) PASS biological activity spectrum predictions in the Enhanced Open NCI Database Browser. J Chem Inf Comput Sci  43: 228–236.  <ul class="find" data-citedArticleID="1050216" data-doi="10.1021/ci020048r"><li><a href="http://dx.doi.org/10.1021/ci020048r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PASS+biological+activity+spectrum+predictions+in+the+Enhanced+Open+NCI+Database+Browser." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PASS+biological+activity+spectrum+predictions+in+the+Enhanced+Open+NCI+Database+Browser.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001537-Oellien1" id="pone.0001537-Oellien1"></a>Oellien F, Ihlenfeldt WD, Johann Gasteiger J (2005) InfVis - platform-independent visual data mining of multidimensional chemical data sets. J Chem Inf Model  45: 1456–1467.  <ul class="find" data-citedArticleID="1050210" data-doi="10.1021/ci050202k"><li><a href="http://dx.doi.org/10.1021/ci050202k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=InfVis+-+platform-independent+visual+data+mining+of+multidimensional+chemical+data+sets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22InfVis+-+platform-independent+visual+data+mining+of+multidimensional+chemical+data+sets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001537-Nicolaou1" id="pone.0001537-Nicolaou1"></a>Nicolaou CA, Brown N, Pattichis CS (2007) Molecular optimization using computational multiobjective methods. Curr Opin Drug Discov Devel  10:   <ul class="find" data-citedArticleID="1050208"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+optimization+using+computational+multiobjective+methods.&amp;auth=&amp;atitle=Molecular+optimization+using+computational+multiobjective+methods." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+optimization+using+computational+multiobjective+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+optimization+using+computational+multiobjective+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001537-Richmond1" id="pone.0001537-Richmond1"></a>Richmond NJ, Abrams CA, Wolohan PRN, Abrahamian E, Willett P, et al.  (2006) GALAHAD: 1. Pharmacophore identification by hypermolecular alignment of ligands in 3D. J Comput Aided Mol Des  20: 567–587.  <ul class="find" data-citedArticleID="1050224" data-doi="10.1007/s10822-006-9082-y"><li><a href="http://dx.doi.org/10.1007/s10822-006-9082-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GALAHAD%3A+1.+Pharmacophore+identification+by+hypermolecular+alignment+of+ligands+in+3D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GALAHAD%3A+1.+Pharmacophore+identification+by+hypermolecular+alignment+of+ligands+in+3D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001537-Chen1" id="pone.0001537-Chen1"></a>Chen B, Harrison RF, Papadatos G, Willett P, Wood DJ, et al.  (2007) Evaluation of machine-learning methods for ligand-based virtual screening. J Comput Aided Mol Des  21: 53–62.  <ul class="find" data-citedArticleID="1050162" data-doi="10.1007/s10822-006-9096-5"><li><a href="http://dx.doi.org/10.1007/s10822-006-9096-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+machine-learning+methods+for+ligand-based+virtual+screening." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+machine-learning+methods+for+ligand-based+virtual+screening.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001537-Polanski1" id="pone.0001537-Polanski1"></a>Polanski J (2003) Self-organizing neural networks for pharmacophore mapping. Adv Drug Deliv Rev  55: 1149–1162.  <ul class="find" data-citedArticleID="1050212" data-doi="10.1016/s0169-409x(03)00116-9"><li><a href="http://dx.doi.org/10.1016/s0169-409x(03)00116-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Self-organizing+neural+networks+for+pharmacophore+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Self-organizing+neural+networks+for+pharmacophore+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001537-Muegge1" id="pone.0001537-Muegge1"></a>Muegge I (2002) Pharmacophore features of potential drugs. Chem Eur J  8: 1976–1981.  <ul class="find" data-citedArticleID="1050206" data-doi="10.1002/1521-3765(20020503)8:9<1976::aid-chem1976&gt;3.0.co;2-k"><li><a href="http://dx.doi.org/10.1002/1521-3765(20020503)8:9<1976::aid-chem1976&gt;3.0.co;2-k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pharmacophore+features+of+potential+drugs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pharmacophore+features+of+potential+drugs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001537-Schwab1" id="pone.0001537-Schwab1"></a>Schwab W (2003) Metabolome diversity: too few genes, too many metabolites? Phytochemistry  62: 837–849.  <ul class="find" data-citedArticleID="1050226" data-doi="10.1016/s0031-9422(02)00723-9"><li><a href="http://dx.doi.org/10.1016/s0031-9422(02)00723-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolome+diversity%3A+too+few+genes%2C+too+many+metabolites%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolome+diversity%3A+too+few+genes%2C+too+many+metabolites%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001537-Hult1" id="pone.0001537-Hult1"></a>Hult K, Berglund P (2007) Enzyme promiscuity: mechanism and applications. Trends Biochem  25: 231–238.  <ul class="find" data-citedArticleID="1050180" data-doi="10.1016/j.tibtech.2007.03.002"><li><a href="http://dx.doi.org/10.1016/j.tibtech.2007.03.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzyme+promiscuity%3A+mechanism+and+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzyme+promiscuity%3A+mechanism+and+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001537-Kanehisa1" id="pone.0001537-Kanehisa1"></a>Kanehisa M, Goto S, Hattori M, Aoki-Kinoshita KF, Itoh M, et al.  (2006) From genomics to chemical genomics: new developments in KEGG. Nucleic Acids Res  34: D354–357.  <ul class="find" data-citedArticleID="1050190" data-doi="10.1093/nar/gkj102"><li><a href="http://dx.doi.org/10.1093/nar/gkj102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genomics+to+chemical+genomics%3A+new+developments+in+KEGG." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genomics+to+chemical+genomics%3A+new+developments+in+KEGG.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001537-Ihlenfeldt1" id="pone.0001537-Ihlenfeldt1"></a>Ihlenfeldt WD, Gasteiger J (1994) Hash codes for the identification and classification of molecular structure elements. J Comput Chem  15: 793–813.  <ul class="find" data-citedArticleID="1050182" data-doi="10.1002/jcc.540150802"><li><a href="http://dx.doi.org/10.1002/jcc.540150802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hash+codes+for+the+identification+and+classification+of+molecular+structure+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hash+codes+for+the+identification+and+classification+of+molecular+structure+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001537-Ihlenfeldt2" id="pone.0001537-Ihlenfeldt2"></a>Ihlenfeldt WD, Voigt JH, Bienfait B, Oellien F, Nicklaus MC (2002) Enhanced CACTVS browser of the open NCI Database. J Chem Inf Comput Sci  42: 46–57.  <ul class="find" data-citedArticleID="1050184" data-doi="10.1021/ci010056s"><li><a href="http://dx.doi.org/10.1021/ci010056s" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhanced+CACTVS+browser+of+the+open+NCI+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhanced+CACTVS+browser+of+the+open+NCI+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001537-McDonald1" id="pone.0001537-McDonald1"></a>McDonald AG, Boyce S, Moss GP, Dixon HBF, Tipton KF (2007) ExplorEnz: a MySQL database of the IUBMB Enzyme Nomenclature. BMC Biochem  8: 14.  <ul class="find" data-citedArticleID="1050204" data-doi="10.1186/1471-2091-8-14"><li><a href="http://dx.doi.org/10.1186/1471-2091-8-14" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ExplorEnz%3A+a+MySQL+database+of+the+IUBMB+Enzyme+Nomenclature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ExplorEnz%3A+a+MySQL+database+of+the+IUBMB+Enzyme+Nomenclature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001537-Dalby1" id="pone.0001537-Dalby1"></a>Dalby A, Nourse JG, Hounshell WD, Gushrst AKI, Grier DL, et al.  (1992) Description of several chemical structure file formats used by computer programs developed at Molecular Design Limited. J Chem Inf Comput Sci  32: 244–255.  <ul class="find" data-citedArticleID="1050166" data-doi="10.1021/ci00007a012"><li><a href="http://dx.doi.org/10.1021/ci00007a012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Description+of+several+chemical+structure+file+formats+used+by+computer+programs+developed+at+Molecular+Design+Limited." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Description+of+several+chemical+structure+file+formats+used+by+computer+programs+developed+at+Molecular+Design+Limited.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001537-Kotera1" id="pone.0001537-Kotera1"></a>Kotera M, Hattori M, Oh M, Yamamoto R, Komeno T, et al.  (2004) RPAIR: a reactant-pair database representing chemical changes in enzymatic reactions. Genome Inform  15: P062.  <ul class="find" data-citedArticleID="1050196"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=RPAIR%3A+a+reactant-pair+database+representing+chemical+changes+in+enzymatic+reactions.&amp;auth=&amp;atitle=RPAIR%3A+a+reactant-pair+database+representing+chemical+changes+in+enzymatic+reactions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RPAIR%3A+a+reactant-pair+database+representing+chemical+changes+in+enzymatic+reactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RPAIR%3A+a+reactant-pair+database+representing+chemical+changes+in+enzymatic+reactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001537-Kotera2" id="pone.0001537-Kotera2"></a>Kotera M, Okuno Y, Hattori M, Goto S, Kanehisa M (2004) Computational assignment of the EC numbers for genomic-scale analysis of enzymatic reactions. J Am Chem Soc  126: 16487–16498.  <ul class="find" data-citedArticleID="1050198" data-doi="10.1021/ja0466457"><li><a href="http://dx.doi.org/10.1021/ja0466457" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+assignment+of+the+EC+numbers+for+genomic-scale+analysis+of+enzymatic+reactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+assignment+of+the+EC+numbers+for+genomic-scale+analysis+of+enzymatic+reactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001537-Hofmann1" id="pone.0001537-Hofmann1"></a>Hofmann O, Schomburg D (2005) Concept-based annotation of enzyme classes. Bioinformatics  21: 2059–2066.  <ul class="find" data-citedArticleID="1050178" data-doi="10.1093/bioinformatics/bti284"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti284" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Concept-based+annotation+of+enzyme+classes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Concept-based+annotation+of+enzyme+classes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001537-Antal1" id="pone.0001537-Antal1"></a>Antal J, Pal G, Asboth B, Buzas Z, Patthy A, Graf L (2001) Specificity assay of serine proteinases by reverse-phase high-performance liquid chromatography analysis of competing oligopeptide substrate library. Anal Biochem  288: 156–167.  <ul class="find" data-citedArticleID="1050144" data-doi="10.1006/abio.2000.4886"><li><a href="http://dx.doi.org/10.1006/abio.2000.4886" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specificity+assay+of+serine+proteinases+by+reverse-phase+high-performance+liquid+chromatography+analysis+of+competing+oligopeptide+substrate+library." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specificity+assay+of+serine+proteinases+by+reverse-phase+high-performance+liquid+chromatography+analysis+of+competing+oligopeptide+substrate+library.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001537-Ellis1" id="pone.0001537-Ellis1"></a>Ellis LB, Roe D, Wackett LP (2006) The University of Minnesota Biocatalysis/Biodegradation Database: the first decade. Nucleic Acids Res  34 (Database issue): D517–21.  <ul class="find" data-citedArticleID="1050168" data-doi="10.1093/nar/gkj076"><li><a href="http://dx.doi.org/10.1093/nar/gkj076" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+University+of+Minnesota+Biocatalysis%2FBiodegradation+Database%3A+the+first+decade." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+University+of+Minnesota+Biocatalysis%2FBiodegradation+Database%3A+the+first+decade.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001537-Wiswesser1" id="pone.0001537-Wiswesser1"></a>Wiswesser WJ (1954) A Line-Formula Chemical Notation. New York: Thomas Crowell.   <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0001537-Wiswesser2" id="pone.0001537-Wiswesser2"></a>Wiswesser WJ (1982) How the WLN began in 1949 and how it might be in 1999. J Chem Inf Comput Sci  22: 88–93.  <ul class="find" data-citedArticleID="1050238" data-doi="10.1021/ci00034a005"><li><a href="http://dx.doi.org/10.1021/ci00034a005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+the+WLN+began+in+1949+and+how+it+might+be+in+1999." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+the+WLN+began+in+1949+and+how+it+might+be+in+1999.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001537-Wiswesser3" id="pone.0001537-Wiswesser3"></a>Wiswesser WJ (1985) Historic development of chemical notations. J Chem Inf Comput Sci  25: 258–263.  <ul class="find" data-citedArticleID="1050240" data-doi="10.1021/ci00047a023"><li><a href="http://dx.doi.org/10.1021/ci00047a023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Historic+development+of+chemical+notations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Historic+development+of+chemical+notations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001537-Barnard1" id="pone.0001537-Barnard1"></a>Barnard JM, Jochum CJ, Welford SM (1989)  ROSDAL: A universal structure/substructure representation for PC-host communication. In: Warr WA, editor. Chemical Structure Information Systems: Interfaces, Communication and Standards, ACS Symposium Series No. 400. Washington, DC: American Chemical Society. pp. 76–81.  <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0001537-Weininger1" id="pone.0001537-Weininger1"></a>Weininger D (1988) SMILES 1. Introduction and encoding rules. J Chem Inf Comput Sci 28: 31-36.   <a href="http://www.daylight.com">http://www.daylight.com</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0001537-Weininger2" id="pone.0001537-Weininger2"></a>Weininger D (1990) SMILES. 3. DEPICT. Graphical depiction of chemical structures. J Chem Inf Comput Sci  30(3): 237–243.  <ul class="find" data-citedArticleID="1050230" data-doi="10.1021/ci00067a005"><li><a href="http://dx.doi.org/10.1021/ci00067a005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SMILES.+3.+DEPICT.+Graphical+depiction+of+chemical+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SMILES.+3.+DEPICT.+Graphical+depiction+of+chemical+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001537-Ash1" id="pone.0001537-Ash1"></a>Ash S, Cline MA, Homer RW, Hurst T, Smith GB (1997) SYBYL Line Notation (SLN): A versatile language for chemical structure representation. J Chem Inf Comput Sci  37: 71–79.  <ul class="find" data-citedArticleID="1050148" data-doi="10.1021/ci960109j"><li><a href="http://dx.doi.org/10.1021/ci960109j" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SYBYL+Line+Notation+%28SLN%29%3A+A+versatile+language+for+chemical+structure+representation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SYBYL+Line+Notation+%28SLN%29%3A+A+versatile+language+for+chemical+structure+representation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001537-Coles1" id="pone.0001537-Coles1"></a>Coles SJ, Day NE, Murray-Rust P, Rzepa HS, Zhang Y (2005) Enhancement of the chemical semantic web through the use of InChI identifiers. Org Biomol Chem  3: 1832–1834.  <ul class="find" data-citedArticleID="1050164" data-doi="10.1039/b502828k"><li><a href="http://dx.doi.org/10.1039/b502828k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhancement+of+the+chemical+semantic+web+through+the+use+of+InChI+identifiers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhancement+of+the+chemical+semantic+web+through+the+use+of+InChI+identifiers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001537-Raymond1" id="pone.0001537-Raymond1"></a>Raymond JW, Willett P (2002) Maximum common subgraph isomorphism algorithms for the matching of chemical structures. J. Comput-Aided Mol Des  16: 521–533.  <ul class="find" data-citedArticleID="1050218"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maximum+common+subgraph+isomorphism+algorithms+for+the+matching+of+chemical+structures.&amp;auth=&amp;atitle=Maximum+common+subgraph+isomorphism+algorithms+for+the+matching+of+chemical+structures." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+common+subgraph+isomorphism+algorithms+for+the+matching+of+chemical+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+common+subgraph+isomorphism+algorithms+for+the+matching+of+chemical+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001537-Raymond2" id="pone.0001537-Raymond2"></a>Raymond JW, Gardiner EJ, Willett P (2002) RASCAL: Calculation of graph similarity using maximum common edge subgraphs. Comput J  45: 631–644.  <ul class="find" data-citedArticleID="1050220" data-doi="10.1093/comjnl/45.6.631"><li><a href="http://dx.doi.org/10.1093/comjnl/45.6.631" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RASCAL%3A+Calculation+of+graph+similarity+using+maximum+common+edge+subgraphs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RASCAL%3A+Calculation+of+graph+similarity+using+maximum+common+edge+subgraphs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001537-Raymond3" id="pone.0001537-Raymond3"></a>Raymond JW, Gardiner EJ, Willett P (2002) Heuristics for similarity searching of chemical graphs using a maximum common edge subgraph algorithm. J Chem Inf Comput Sci  42: 305–316.  <ul class="find" data-citedArticleID="1050222" data-doi="10.1021/ci010381f"><li><a href="http://dx.doi.org/10.1021/ci010381f" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heuristics+for+similarity+searching+of+chemical+graphs+using+a+maximum+common+edge+subgraph+algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heuristics+for+similarity+searching+of+chemical+graphs+using+a+maximum+common+edge+subgraph+algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001537-Hattori1" id="pone.0001537-Hattori1"></a>Hattori M, Okuno Y, Goto S, Kanehisa M (2003) Development of a chemical structure comparison method for integrated analysis of chemical and genomic information in the metabolic pathways. J Am Chem Soc  125: 11853–11865.  <ul class="find" data-citedArticleID="1050176" data-doi="10.1021/ja036030u"><li><a href="http://dx.doi.org/10.1021/ja036030u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+a+chemical+structure+comparison+method+for+integrated+analysis+of+chemical+and+genomic+information+in+the+metabolic+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+a+chemical+structure+comparison+method+for+integrated+analysis+of+chemical+and+genomic+information+in+the+metabolic+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001537-Flower1" id="pone.0001537-Flower1"></a>Flower DR (1998) On the properties of bit string-based measures of chemical similarity. J Chem Inf Comput Sci  38: 379–386.  <ul class="find" data-citedArticleID="1050170" data-doi="10.1021/ci970437z"><li><a href="http://dx.doi.org/10.1021/ci970437z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+properties+of+bit+string-based+measures+of+chemical+similarity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+properties+of+bit+string-based+measures+of+chemical+similarity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001537-Allen1" id="pone.0001537-Allen1"></a>Allen FH, Kennard O (1993) 3D search and research using the Cambridge structural database. Chem Des Autom News  8: 1 and 31–37.  <ul class="find" data-citedArticleID="1050142"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=3D+search+and+research+using+the+Cambridge+structural+database.&amp;auth=&amp;atitle=3D+search+and+research+using+the+Cambridge+structural+database." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=3D+search+and+research+using+the+Cambridge+structural+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%223D+search+and+research+using+the+Cambridge+structural+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001537-Brown1" id="pone.0001537-Brown1"></a>Brown RD, Martin YC (1996) Use of structure-activity data to compare structure-based clustering methods and descriptors for use in compound selection. J Chem Inf Comput Sci  36: 572–584.  <ul class="find" data-citedArticleID="1050154" data-doi="10.1021/ci9501047"><li><a href="http://dx.doi.org/10.1021/ci9501047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+structure-activity+data+to+compare+structure-based+clustering+methods+and+descriptors+for+use+in+compound+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+structure-activity+data+to+compare+structure-based+clustering+methods+and+descriptors+for+use+in+compound+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001537-Brown2" id="pone.0001537-Brown2"></a>Brown RD, Martin YC (1997) The information content of 2D and 3D structural descriptors relevant to ligand-receptor binding. J Chem Inf Comput Sci  37: 1–9.  <ul class="find" data-citedArticleID="1050156" data-doi="10.1021/ci960373c"><li><a href="http://dx.doi.org/10.1021/ci960373c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+information+content+of+2D+and+3D+structural+descriptors+relevant+to+ligand-receptor+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+information+content+of+2D+and+3D+structural+descriptors+relevant+to+ligand-receptor+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001537-Bayada1" id="pone.0001537-Bayada1"></a>Bayada DM, Hamersma H, van Geerestein VJ (1999) Molecular diversity and representativity in chemical databases. J Chem Inf Comput Sci  39: 1–10.  <ul class="find" data-citedArticleID="1050152" data-doi="10.1021/ci980109e"><li><a href="http://dx.doi.org/10.1021/ci980109e" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+diversity+and+representativity+in+chemical+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+diversity+and+representativity+in+chemical+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001537-Wild1" id="pone.0001537-Wild1"></a>Wild D, Blankley CJ (2000) Comparison of 2D fingerprint types and hierarchy level selection methods for structural grouping using Ward's clustering. J Chem Inf Comput Sci  40: 155–162.  <ul class="find" data-citedArticleID="1050232" data-doi="10.1021/ci990086j"><li><a href="http://dx.doi.org/10.1021/ci990086j" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+2D+fingerprint+types+and+hierarchy+level+selection+methods+for+structural+grouping+using+Ward%27s+clustering." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+2D+fingerprint+types+and+hierarchy+level+selection+methods+for+structural+grouping+using+Ward%27s+clustering.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001537-James1" id="pone.0001537-James1"></a>James CA, Weininger D, Delany J (2000) Daylight Theory Manual 4.71;. Irvine, CA: Daylight Chemical Information Systems, Inc.   <ul class="find-nolinks"></ul></li><li><span class="label">47.
              </span><a name="pone.0001537-Greene1" id="pone.0001537-Greene1"></a> Greene N, Erhardt PW, editors.  (1999) Knowledge-based expert systems for toxicity and metabolism prediction: Drug Metabolism:. London: Blackwell Science Ltd. pp. 289–296.  <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0001537-Klopman1" id="pone.0001537-Klopman1"></a>Klopman G, Tu M (1997) Structure-biodegradability study and computer-automated prediction of aerobic biodegradation of chemicals. Environ Toxicol Chem  16: 1829–1835.  <ul class="find" data-citedArticleID="1050194" data-doi="10.1002/etc.5620160910"><li><a href="http://dx.doi.org/10.1002/etc.5620160910" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure-biodegradability+study+and+computer-automated+prediction+of+aerobic+biodegradation+of+chemicals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure-biodegradability+study+and+computer-automated+prediction+of+aerobic+biodegradation+of+chemicals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001537-Bush1" id="pone.0001537-Bush1"></a>Bush BL, Sheridan RP (1993) PATTY: A programmable atom typer and language for automatic classification of atoms in molecular databases. J Chem Inf Comput Sci  33: 756–762.  <ul class="find" data-citedArticleID="1050158" data-doi="10.1021/ci00015a015"><li><a href="http://dx.doi.org/10.1021/ci00015a015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PATTY%3A+A+programmable+atom+typer+and+language+for+automatic+classification+of+atoms+in+molecular+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PATTY%3A+A+programmable+atom+typer+and+language+for+automatic+classification+of+atoms+in+molecular+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001537-Lipinski1" id="pone.0001537-Lipinski1"></a>Lipinski CA, Lombardo F, Dominy BW, Feeny PJ (1997) Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings. Adv Drug Deliv Rev  23: 3–25.  <ul class="find" data-citedArticleID="1050202" data-doi="10.1016/s0169-409x(96)00423-1"><li><a href="http://dx.doi.org/10.1016/s0169-409x(96)00423-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.XML" value="90083"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.PDF" value="214239"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.t001.PNG_L" value="130557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.t001.PNG_M" value="181805"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.t001.PNG_S" value="20331"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.t001.TIF" value="526928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.t001.PNG_I" value="63036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.g001.PNG_L" value="135455"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.g001.PNG_M" value="165613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.g001.PNG_S" value="11912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.g001.TIF" value="635158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.g001.PNG_I" value="56803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.t002.PNG_L" value="82603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.t002.PNG_M" value="46922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.t002.PNG_S" value="8579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.t002.TIF" value="387182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.t002.PNG_I" value="14077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.g002.PNG_L" value="335324"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.g002.PNG_M" value="76990"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.g002.PNG_S" value="10476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.g002.TIF" value="543002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001537.g002.PNG_I" value="25191"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001537&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001537" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001537&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001537&volume=&issue=&title=Functional Group and Substructure Searching as a Tool in Metabolomics&author_name=Masaaki%20Kotera%2C%20Andrew%20G.%20McDonald%2C%20Sin%C3%A9ad%20Boyce%2C%20Keith%20F.%20Tipton&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001537" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001537" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001537" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001537&amp;t=Functional%20Group%20and%20Substructure%20Searching%20as%20a%20Tool%20in%20Metabolomics" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001537&title=Functional%20Group%20and%20Substructure%20Searching%20as%20a%20Tool%20in%20Metabolomics&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001537&amp;title=Functional%20Group%20and%20Substructure%20Searching%20as%20a%20Tool%20in%20Metabolomics" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001537" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001537&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Functional Group and Substructure Searching as a Tool in Metabolomics';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001537';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Functional%20Group%20and%20Substructure%20Searching%20as%20a%20Tool%20in%20Metabolomics http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001537" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001537" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chemical+compounds%22" title="Search for articles in the subject area:'Chemical compounds'"><div class="flagText">Chemical compounds</div></a>
              <div data-categoryid="24191" data-articleid="25900"
                   data-categoryname="Chemical compounds"
                   class="flagImage" title="Flag 'Chemical compounds' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Database+searching%22" title="Search for articles in the subject area:'Database searching'"><div class="flagText">Database searching</div></a>
              <div data-categoryid="32021" data-articleid="25900"
                   data-categoryname="Database searching"
                   class="flagImage" title="Flag 'Database searching' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drug+metabolism%22" title="Search for articles in the subject area:'Drug metabolism'"><div class="flagText">Drug metabolism</div></a>
              <div data-categoryid="18167" data-articleid="25900"
                   data-categoryname="Drug metabolism"
                   class="flagImage" title="Flag 'Drug metabolism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzyme+metabolism%22" title="Search for articles in the subject area:'Enzyme metabolism'"><div class="flagText">Enzyme metabolism</div></a>
              <div data-categoryid="20267" data-articleid="25900"
                   data-categoryname="Enzyme metabolism"
                   class="flagImage" title="Flag 'Enzyme metabolism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzyme+structure%22" title="Search for articles in the subject area:'Enzyme structure'"><div class="flagText">Enzyme structure</div></a>
              <div data-categoryid="18161" data-articleid="25900"
                   data-categoryname="Enzyme structure"
                   class="flagImage" title="Flag 'Enzyme structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzymes%22" title="Search for articles in the subject area:'Enzymes'"><div class="flagText">Enzymes</div></a>
              <div data-categoryid="17863" data-articleid="25900"
                   data-categoryname="Enzymes"
                   class="flagImage" title="Flag 'Enzymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Functional+groups%22" title="Search for articles in the subject area:'Functional groups'"><div class="flagText">Functional groups</div></a>
              <div data-categoryid="35147" data-articleid="25900"
                   data-categoryname="Functional groups"
                   class="flagImage" title="Flag 'Functional groups' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Xenobiotic+metabolism%22" title="Search for articles in the subject area:'Xenobiotic metabolism'"><div class="flagText">Xenobiotic metabolism</div></a>
              <div data-categoryid="24269" data-articleid="25900"
                   data-categoryname="Xenobiotic metabolism"
                   class="flagImage" title="Flag 'Xenobiotic metabolism' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7306'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3259'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5394&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=6441">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
