

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Intron Dynamics in Ribosomal Protein Genes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000141"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000141" />

    <meta name="citation_title" content="Intron Dynamics in Ribosomal Protein Genes"/>
    <meta itemprop="name" content="Intron Dynamics in Ribosomal Protein Genes"/>

      <meta name="citation_author" content="Maki Yoshihama"/>
            <meta name="citation_author_institution" content="Frontier Science Research Center, University of Miyazaki, Kiyotake, Miyazaki, Japan"/>
      <meta name="citation_author" content="Hung D. Nguyen"/>
            <meta name="citation_author_institution" content="Frontier Science Research Center, University of Miyazaki, Kiyotake, Miyazaki, Japan"/>
      <meta name="citation_author" content="Naoya Kenmochi"/>
            <meta name="citation_author_institution" content="Frontier Science Research Center, University of Miyazaki, Kiyotake, Miyazaki, Japan"/>

    <meta name="citation_date" content="2007/1/3"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000141.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e141"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Remarkable interkingdom conservation of intron positions and massive, lineage-specific intron loss and gain in eukaryotic evolution.; citation_author=IB Rogozin; citation_author=YI Wolf; citation_author=AV Sorokin; citation_author=BG Mirkin; citation_author=EV Koonin; citation_journal_title=Curr Biol; citation_volume=13; citation_number=1; citation_pages=1512-1517; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Rates of intron loss and gain: implications for early eukaryotic evolution.; citation_author=SW Roy; citation_author=W Gilbert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=2; citation_pages=5773-5778; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Likely scenarios of intron evolution.; citation_author=M Csürös; citation_volume=3678; citation_number=3; citation_pages=47-60; citation_date=2005; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=New maximum likelihood estimators for eukaryotic intron evolution.; citation_author=DH Nguyen; citation_author=M Yoshihama; citation_author=N Kenmochi; citation_journal_title=PLoS Comput Biol; citation_volume=1; citation_number=4; citation_pages=e79; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RPG: the Ribosomal Protein Gene database.; citation_author=A Nakao; citation_author=M Yoshihama; citation_author=N Kenmochi; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=5; citation_pages=D168-170; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The human ribosomal protein genes: sequencing and comparative analysis of 73 genes.; citation_author=M Yoshihama; citation_author=T Uechi; citation_author=S Asakawa; citation_author=K Kawasaki; citation_author=S Kato; citation_journal_title=Genome Res; citation_volume=12; citation_number=6; citation_pages=379-390; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of ribosomal protein gene structures: implications for intron evolution.; citation_author=M Yoshihama; citation_author=DH Nguyen; citation_author=N Kenmochi; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=7; citation_pages=e25; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phase distribution of spliceosomal introns: implications for intron origin.; citation_author=HD Nguyen; citation_author=M Yoshihama; citation_author=N Kenmochi; citation_journal_title=BMC Evol Biol; citation_volume=6; citation_number=8; citation_pages=69; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The structure and function of eukaryotic ribosomes.; citation_author=IG Wool; citation_journal_title=Annu Rev Biochem; citation_volume=48; citation_number=9; citation_pages=719-754; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Ribosomes and ribosomal proteins.; citation_author=N Kenmochi; citation_number=10; citation_pages=pp. 77-82; citation_date=2003; citation_publisher=Nature Publishing Group; " />
      <meta name="citation_reference" content="citation_title=A spliceosomal intron in Giardia lamblia.; citation_author=JE Nixon; citation_author=A Wang; citation_author=HG Morrison; citation_author=AG McArthur; citation_author=ML Sogin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=11; citation_pages=3701-3705; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=An ancient spliceosomal intron in the ribosomal protein L7a gene (Rpl7a) of Giardia lamblia.; citation_author=AG Russell; citation_author=TE Shutt; citation_author=RF Watkins; citation_author=MW Gray; citation_journal_title=BMC Evol Biol; citation_volume=5; citation_number=12; citation_pages=45; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The origin and evolution of model organisms.; citation_author=SB Hedges; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=13; citation_pages=838-849; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A genomic timescale of prokaryote evolution: insights into the origin of methanogenesis, phototrophy, and the colonization of land.; citation_author=FU Battistuzzi; citation_author=A Feijao; citation_author=SB Hedges; citation_journal_title=BMC Evol Biol; citation_volume=4; citation_number=14; citation_pages=44; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Large-scale comparison of intron positions in mammalian genes shows intron loss but no gain.; citation_author=SW Roy; citation_author=A Fedorov; citation_author=W Gilbert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=15; citation_pages=7158-7162; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparative genome and proteome analysis of Anopheles gambiae and Drosophila melanogaster.; citation_author=EM Zdobnov; citation_author=C von Mering; citation_author=I Letunic; citation_author=D Torrents; citation_author=M Suyama; citation_journal_title=Science; citation_volume=298; citation_number=16; citation_pages=149-159; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The new phylogeny of eukaryotes.; citation_author=H Philippe; citation_author=A Ger mot; citation_journal_title=Curr Opin Genet Dev; citation_volume=10; citation_number=17; citation_pages=596-601; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Prevalence of intron gain over intron loss in the evolution of paralogous gene families.; citation_author=VN Babenko; citation_author=IB Rogozin; citation_author=SL Mekhedov; citation_author=EV Koonin; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=18; citation_pages=3724-3733; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Alternative splicing and genome complexity.; citation_author=D Brett; citation_author=H Pospisil; citation_author=J Valcarcel; citation_author=J Reich; citation_author=P Bork; citation_journal_title=Nat Genet; citation_volume=30; citation_number=19; citation_pages=29-30; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome.; citation_journal_title=Nature; citation_volume=409; citation_number=20; citation_pages=860-921; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The sequence of the human genome.; citation_author=CJ Venter; citation_author=MD Adams; citation_author=EW Myers; citation_author=PW Li; citation_author=RJ Mural; citation_journal_title=Science; citation_volume=291; citation_number=21; citation_pages=1304-1351; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The biology of intron gain and loss.; citation_author=DC Jeffares; citation_author=T Mourier; citation_author=D Penny; citation_journal_title=Trends Genet; citation_volume=22; citation_number=22; citation_pages=16-22; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The evolution of controlled multitasked gene networks: the role of introns and other noncoding RNAs in the development of complex organisms.; citation_author=JS Mattick; citation_author=MJ Gagen; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=23; citation_pages=1611-1630; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The small nucleolar RNAs.; citation_author=ES Maxwell; citation_author=MJ Fournier; citation_journal_title=Annu Rev Biochem; citation_volume=64; citation_number=24; citation_pages=897-943; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A mammalian gene with introns instead of exons generating stable RNA products.; citation_author=KT Tycowski; citation_author=MD Shu; citation_author=JA Steitz; citation_journal_title=Nature; citation_volume=379; citation_number=25; citation_pages=464-466; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of non-coding DNA in Drosophila.; citation_author=P Andolfatto; citation_journal_title=Nature; citation_volume=437; citation_number=26; citation_pages=1149-1152; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: Improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=27; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=PHYLIP (Phylogeny Inference Package) version 3.5c.; citation_author=J Felsenstein; citation_number=28; citation_date=1993; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000141" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Intron Dynamics in Ribosomal Protein Genes"/>
    <meta name="twitter:description" content="The role of spliceosomal introns in eukaryotic genomes remains obscure. A large scale analysis of intron presence/absence patterns in many gene families and species is a necessary step to clarify the role of these introns. In this analysis, we used a maximum likelihood method to reconstruct the evolution of 2,961 introns in a dataset of 76 ribosomal protein genes from 22 eukaryotes and validated the results by a maximum parsimony method. Our results show that the trends of intron gain and loss differed across species in a given kingdom but appeared to be consistent within subphyla. Most subphyla in the dataset diverged around 1 billion years ago, when the &ldquo;Big Bang&rdquo; radiation occurred. We speculate that spliceosomal introns may play a role in the explosion of many eukaryotes at the Big Bang radiation."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000141.g003"/>

  <meta property="og:title" content="Intron Dynamics in Ribosomal Protein Genes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000141" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8254'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7578'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7470&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000141">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000141" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000141&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Intron Dynamics in Ribosomal Protein Genes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maki Yoshihama, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Frontier Science Research Center, University of Miyazaki, Kiyotake, Miyazaki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hung D. Nguyen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Frontier Science Research Center, University of Miyazaki, Kiyotake, Miyazaki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Naoya Kenmochi
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kenmochi@med.miyazaki-u.ac.jp">kenmochi@med.miyazaki-u.ac.jp</a>
            </p>

                <p>Affiliation:
                  Frontier Science Research Center, University of Miyazaki, Kiyotake, Miyazaki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 03, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000141</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000141" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000141" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000141" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000141" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000141-g001" data-doi="info:doi/10.1371/journal.pone.0000141" data-uri="info:doi/10.1371/journal.pone.0000141.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000141.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000141-g002" data-doi="info:doi/10.1371/journal.pone.0000141" data-uri="info:doi/10.1371/journal.pone.0000141.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000141.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000141-g003" data-doi="info:doi/10.1371/journal.pone.0000141" data-uri="info:doi/10.1371/journal.pone.0000141.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000141.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000141">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000141" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The role of spliceosomal introns in eukaryotic genomes remains obscure. A large scale analysis of intron presence/absence patterns in many gene families and species is a necessary step to clarify the role of these introns. In this analysis, we used a maximum likelihood method to reconstruct the evolution of 2,961 introns in a dataset of 76 ribosomal protein genes from 22 eukaryotes and validated the results by a maximum parsimony method. Our results show that the trends of intron gain and loss differed across species in a given kingdom but appeared to be consistent within subphyla. Most subphyla in the dataset diverged around 1 billion years ago, when the “Big Bang” radiation occurred. We speculate that spliceosomal introns may play a role in the explosion of many eukaryotes at the Big Bang radiation.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Yoshihama M, Nguyen HD, Kenmochi N (2007) Intron Dynamics in Ribosomal Protein Genes. PLoS ONE 2(1):
          e141.
            doi:10.1371/journal.pone.0000141</p><p><strong>Academic Editor: </strong>Oliver Hofmann, South African National Bioinformatics Institute, South Africa</p><p><strong>Received:</strong> September 26, 2006; <strong>Accepted:</strong> November 9, 2006; <strong>Published:</strong> January 3, 2007</p><p><strong>Copyright:</strong> © 2007 Yoshihama et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by Grants-in-Aid for Scientific Research (14035103, 17770207, and 188093) from the Ministry of Education, Culture, Sports, Science, and Technology (MEXT) and the Japan Society for the Promotion of Science (JSPS). HDN is a research fellow of the JSPS (17- 05174).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Many spliceosomal introns, which are non-coding DNA sequences, exist in eukaryotic nuclear genes. Their role in the genome, however, remains poorly understood. From the view of eukaryotic evolution, it is very important to know why exon/intron structures of genes differ across species and what the effects of intron gain and loss are. In order to clarify these issues, we must first reconstruct the process of intron gain and loss during eukaryotic evolution. This task became possible recently with the availability of many completely sequenced genomes. In a representative study, Rogozin <em>et al.</em> 
            <a href="#pone.0000141-Rogozin1">[1]</a> compiled a dataset of 684 gene orthologs from eight eukaryotes and used a maximum parsimony method to infer the evolution of introns in this dataset. The results of applying maximum likelihood methods to the same dataset were reported later <a href="#pone.0000141-Roy1">[2]</a>–<a href="#pone.0000141-Nguyen1">[4]</a>. Although the number of species in the dataset is not very large and the different methods inferred different patterns of intron gain and loss, it became clear that: (i) from 15% to 25% of present-day introns were already present in the last common ancestor of plantae, metazoa, and fungi, and (ii) many introns were gained after this divergence <a href="#pone.0000141-Rogozin1">[1]</a>–<a href="#pone.0000141-Nguyen1">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>We have recently compiled a dataset of ribosomal protein (RP) genes <a href="#pone.0000141-Nakao1">[5]</a>. RP genes offer several advantages for studying intron evolution <a href="#pone.0000141-Yoshihama1">[6]</a>–<a href="#pone.0000141-Nguyen2">[8]</a>. First, they exist in all species and, as they are involved in the vital process of translation, they are well conserved throughout evolution <a href="#pone.0000141-Wool1">[9]</a>, <a href="#pone.0000141-Kenmochi1">[10]</a>. Thus, it is fairly easy to compare intron positions in RP genes across a wide range of distantly diverged species. Second, there are a large number of conserved RP gene families. For instance, 79 distinct RPs are found in humans and of these 79, 78 are also found in yeast. Third, introns also exist in RP genes of very deep-branching eukaryotes that harbor very few introns, such as <em>Giardia lamblia</em> 
            <a href="#pone.0000141-Nixon1">[11]</a>, <a href="#pone.0000141-Russell1">[12]</a>. With these advantages, we expect that RP genes will become a powerful tool for discovering the roles of spliceosomal introns.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Compilation of the dataset and phylogenetic analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We compiled a dataset of 76 RP gene orthologs from 22 eukaryotes. The phylogenetic tree of these 22 species is depicted in <a href="#pone-0000141-g001">Figure 1</a>. These 22 species belong to four kingdoms, metazoa, fungi, protozoa, and plantae, and cover 14 different subphyla. The conserved regions of this dataset included 2,961 introns located at 1,182 different positions. To the best of our knowledge, this is the first time a dataset with this many gene families and species has been used for studying intron evolution.</p>
<div class="figure" id="pone-0000141-g001"><div class="img"><a name="pone-0000141-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000141.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000141" data-uri="info:doi/10.1371/journal.pone.0000141.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000141.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000141.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000141.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000141.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000141.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000141.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000141.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000141.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000141.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The Phylogenetic Tree of the 22 Eukaryotes in the Dataset.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>All conserved amino acids regions in 76 RP genes were used to generate the tree. The tree with bootstrap values in percentages was built using the Seqboot (1000 replicates), Protdist, Neighbor, and Consense programs of the PHYLIP package <a href="#pone.0000141-Felsenstein1">[28]</a>. The taxonomy is based on that of NCBI (<a href="http://www.ncbi.nlm.nih.gov/Taxonomy/">http://www.ncbi.nlm.nih.gov/Taxonomy/</a>) and ITIS (<a href="http://www.itis.usda.gov/index.html">http://www.itis.usda.gov/index.html</a>). Hs, <em>Homo sapiens</em>; Fr, <em>Fugu rubripes</em>; Ci, <em>Ciona intestinalis</em>; Dm, <em>Drosophila melanogaster</em>; Ag, <em>Anopheles gambiae</em>; Am, <em>Apis mellifera</em>; Ce, <em>Caenorhabditi elegans</em>; Mg, <em>Magnaporthe grisea</em>; Fg, <em>Fusarium graminearum</em>; Sn, <em>Stagonospora nodorum</em>; Yl, <em>Yarrowia lipolytica</em>; Sc, <em>Saccharomyces cerevisiae</em>; Sp, <em>Schizosaccharomyces pombe</em>; Cc, <em>Coprinus cinerea</em>; Cn, <em>Cryptococcus neoformans</em>; Um, <em>Ustilago maydis</em>; Ro, <em>Rhizopus oryzae</em>; Dd, <em>Dictyostelium discoideum</em>; At, <em>Arabidopsis thaliana</em>; Os, <em>Oryza sativa</em>; Cr, <em>Chlamydomonas reinhardtii</em>; Pf, <em>Plasmodium falciparum;</em> 1, vertebrata; 2, urochordata; 3, insecta; 4, nematoda; 5, pezizomycotina; 6, saccharomycotina; 7, schizosaccharomycetes; 8, hymenomycetes; 9, ustilaginomycetes; 10, zycomycetes; 11, mycetozoa; 12, magnoliophyta; 13, chlorophyta; 14, apicomplexa.</p>
<span>doi:10.1371/journal.pone.0000141.g001</span></div>

<h4>Patterns of intron gain and loss in 22 species</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We first used our recently developed maximum likelihood (ML) method <a href="#pone.0000141-Nguyen1">[4]</a> to infer the process of intron gain and loss (<a href="#pone-0000141-g002">Figure 2A</a>). We also used a maximum parsimony method to validate the result of the ML method, because the ML method may produce unreliable results when the data sample is small (<a href="#pone-0000141-g002">Figure 2B</a>). Since the two results show similar patterns of intron gain and loss in most subphyla of the dataset (the largest differences are in the two plant subphyla), the results from the ML method were used for subsequent analyses.</p>
<div class="figure" id="pone-0000141-g002"><div class="img"><a name="pone-0000141-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000141.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000141" data-uri="info:doi/10.1371/journal.pone.0000141.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000141.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000141.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000141.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000141.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000141.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000141.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000141.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000141.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000141.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Patterns of Intron Gain and Loss.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) Likelihood results. (B) Parsimony results. Numbers of introns present in modern species (known) are in black. Numbers of introns present in ancestors (estimated) are in green. Numbers of gains and losses (estimated) are in red and blue, respectively. Branches that experienced &gt;1.5 gains per loss are shown in red and those that experienced &gt;1.5 losses per gain are in blue. Abbreviations are the same as in <a href="#pone-0000141-g001">Figure 1</a>.</p>
<span>doi:10.1371/journal.pone.0000141.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The most significant feature in <a href="#pone-0000141-g002">Figure 2</a> is that species belonging to a given subphylum show similar trends of intron gain and loss. For example, all three species in insecta (subphylum 3 in <a href="#pone-0000141-g002">Figure 2</a>) show trends toward decreasing introns, whereas all three species in pezizomycotina (subphylum 5) show trends toward increasing introns. There is, however, no consensus trend of intron gain and loss among species of a given kingdom. This fact is most notable in the fungus kingdom. The subphyla pezizomycotina (subphylum 5) and hymenomycetes (subphylum 8) trended toward increasing introns, whereas the three other subphyla [saccharomycotina (subphylum 6), schizosaccharomycetes (subphylum 7), and ustilaginomycetes (subphylum 9)] showed the opposite trend. There is also no consensus trend of intron gain and loss among species of a given cell complexity (i.e., unicellular or multicellular). For multicellular species, introns decreased in insecta (subphylum 3) and nematoda (subphylum 4) but increased in <em>Coprinus cinerea</em> and pezizomycotina (mycelium, subphylum 5). Likewise, introns increased in the unicellular species <em>Cryptococcus neoformans</em> but decreased in saccharomycotina (subphylum 6) and schizosaccharomycetes (subphylum 7). Similar results were also obtained in an analysis using 13 additional fungal species (data not shown).</p>


<h4>Intron gain and loss and the speciation of eukaryotes</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>
               <a href="#pone-0000141-g003">Figure 3</a> shows the numbers of intron gains and losses divided into three periods of time (the timescale is based on that of Hedges and colleagues <a href="#pone.0000141-Hedges1">[13]</a>, <a href="#pone.0000141-Battistuzzi1">[14]</a>). The numbers of intron gains and losses are similar for each species in a given subphylum containing multiple species, but they clearly differ across subphyla. For example, during the period from 1.5 to 0 billion years ago (Ga), vertebrata (subphylum 1) had more intron gains than losses (~155 gains versus ~58 losses on average). In contrast, insecta (subphylum 3) had more intron losses than gains (~165 losses versus ~132 gains on average). In particular, the excess of intron losses over gains in <em>Drosophila melanogaster</em> and<em> Anopheles gambiae</em> were very strong during the period from 1.0 to 0 Ga (~151 losses versus ~20 gains on average).</p>
<div class="figure" id="pone-0000141-g003"><div class="img"><a name="pone-0000141-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000141.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000141" data-uri="info:doi/10.1371/journal.pone.0000141.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000141.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000141.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000141.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000141.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000141.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000141.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000141.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000141.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000141.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Intron Gains and Losses Over Time.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The numbers of intron gains and losses are divided into three periods of time: 0.5–0 Ga (black); 1.0–0.5 Ga (gray), and 1.5–1.0 Ga (white). The timescale is based on that of Hedges and colleagues <a href="#pone.0000141-Hedges1">[13]</a>, <a href="#pone.0000141-Battistuzzi1">[14]</a>. Ga, billion years ago; other abbreviations are the same as in <a href="#pone-0000141-g001">Figure 1</a>.</p>
<span>doi:10.1371/journal.pone.0000141.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>During the period from 1.5 to 0 Ga, the species with the most intron gains was <em>Stagonospora nodorum</em> (275 introns), and the species with the fewest gains was <em>Dictyostelium discoideum</em> (26 introns). During this period, the species with the most intron losses was <em>Saccharomyces cerevisiae</em> (255 introns), and the species with the fewest losses was <em>Homo sapiens</em> (57 introns). The patterns of intron gain and loss were most complicated in fungi. The number of intron losses in <em>Magnaporthe grisea</em> since the crown ancestor was 220, whereas the number of gains was 230. During the period from 1.5 to 0 Ga, fungal species in average experienced 2.2 intron gains and 2.6 intron losses per gene (as 168 gains and 198 losses in 76 genes). In contrast, humans experienced 2.0 intron gains and 0.8 intron losses per gene (as 155 gains and 57 losses in 76 genes).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study, we have analyzed intron dynamics in 76 RP gene families from 22 species. So far, there has been no analysis that used such a large dataset with many gene families and species. On one hand, the results inferred from the RP genes are mostly in-line with previous studies using fewer species. For example, when the same maximum likelihood was used, the trends of intron gains and losses in seven species of the 684-ortholog dataset <a href="#pone.0000141-Rogozin1">[1]</a>, <a href="#pone.0000141-Nguyen1">[4]</a> are similar to those presented here. In addition, the finding that species with short generation time have a trend of losing their introns <a href="#pone.0000141-Roy2">[15]</a>, <a href="#pone.0000141-Zdobnov1">[16]</a> can also be observed in this study. On the other hand, RP genes are easier to extend to more species because they are not too large in number and are highly conserved. Thus, we believe that RP genes are a suitable dataset for studying the evolution of introns.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>When the intron dynamics in RP genes from 22 species were viewed together, it becomes clear that the trends of intron gains and losses differ across species in a given kingdom but are similar within subphyla. It has been proposed that most subphyla in our dataset diverged around 1 Ga <a href="#pone.0000141-Hedges1">[13]</a>, when an explosion of eukaryotes occurred (the Big Bang hypothesis) <a href="#pone.0000141-Philippe1">[17]</a>. Furthermore, Babenko et al. <a href="#pone.0000141-Babenko1">[18]</a> suggested that intensive intron gains and losses occurred during limited time spans, perhaps coinciding with major evolutionary transitions. Therefore, we speculate that there may be a relationship between the process of intron gain and loss and the divergence of species at the Big Bang radiation.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Several roles have been proposed for spliceosomal introns. One of these is to increase the variation of proteins through alternative splicing <a href="#pone.0000141-Brett1">[19]</a>–<a href="#pone.0000141-Venter1">[21]</a>. Because alternative splicing is found in both multicellular plants and animals, it was proposed to be a necessary tool for the evolution of eukaryotic complexity <a href="#pone.0000141-Jeffares1">[22]</a>. However, although 40% to 50% of human genes are currently estimated to have splicing variations <a href="#pone.0000141-Brett1">[19]</a>–<a href="#pone.0000141-Venter1">[21]</a>, genes without splicing variations also exist. In fact, almost no alternative splicing of human RP genes has been reported so far.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>If alternative splicing is not the only role of introns, what are other roles? We speculate that there is a relationship between introns and non-coding RNAs (ncRNAs) and, through the process of intron gain and loss, introns may affect the variation of expression of ncRNAs to become a force for the speciation of eukaryotes. There are several lines of supporting evidence for this speculation. First, some ncRNAs are transcribed from introns in protein coding genes <a href="#pone.0000141-Mattick1">[23]</a>. In fact, many small nucleolar RNAs (snoRNAs) are encoded by introns of RP genes in vertebrates <a href="#pone.0000141-Nakao1">[5]</a>, <a href="#pone.0000141-Yoshihama1">[6]</a>. These snoRNAs are expressed through the splicing of the pre-mRNAs of RP genes and act as RNA molecules after they are spliced out from the introns <a href="#pone.0000141-Maxwell1">[24]</a>, <a href="#pone.0000141-Tycowski1">[25]</a>. Second, the finding that adaptive changes to non-coding DNA might have been more common in the evolution of <em>D. melanogaster</em> 
            <a href="#pone.0000141-Andolfatto1">[26]</a> suggests that introns may be functional not only in vertebrates but also in other eukaryotes. Third, ncRNAs have been proposed to be involved in the development of multicellular species <a href="#pone.0000141-Jeffares1">[22]</a>, <a href="#pone.0000141-Mattick1">[23]</a>. Therefore, we speculate that eukaryotic introns may function as carriers for ncRNAs and, through the process of intron gain and loss, may affect the level and variation of expression of ncRNAs.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Compilation of the dataset</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The RP genes of 8 eukaryotes (<em>H. sapiens, Ciona intestinalis, D. melanogaster, Caenorhabditi elegans, Schizosaccharomyces pombe, S. cerevisiae, Arabidopsis thaliana, and Plasmodium falciparum</em>) were taken from the manually curated Ribosomal Protein Gene database (RPG, <a href="http://ribosome.med.miyazaki-u.ac.jp">http://ribosome.med.miyazaki-u.ac.jp</a>) <a href="#pone.0000141-Nakao1">[5]</a>. The RP gene sequences of other species were collected by performing a BLAST search using human RP genes as queries. Their gene structures were then manually constructed both by using annotation (if available) and by aligning their sequences with those of other species. Finally, the constructed genes were used as queries to a BLAST search against the human genes to ensure that they and the human RP genes are reciprocal best hits. Genome homepages for the different species we investigated are as follows: <em>H. sapiens, A. gambiae</em>, and <em>Apis mellifera</em> (ENSEMBL, <a href="http://www.ensembl.org/index.html">http://www.ensembl.org/index.html</a>); <em>D. melanogaster</em> (FlyBase, <a href="http://flybase.net/">http://flybase.net/</a>); <em>C. elegans</em> (WormBase, <a href="http://www.wormbase.org/">http://www.wormbase.org/</a>); <em>S. pombe</em> (<a href="http://www.sanger.ac.uk/Projects/S_pombe/">http://www.sanger.ac.uk/Projects/S_pombe​/</a>); <em>S. cerevisiae</em> (SGD, <a href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>); <em>M. grisea, Fusarium graminearum, S. nodorum, C. cinerea, C. neoformans, Ustilago maydis</em>, and <em>Rhizopus oryzae</em> (Fungal Genome Initiative, <a href="http://www.broad.mit.edu/annotation/fgi/">http://www.broad.mit.edu/annotation/fgi/</a>); <em>Yarrowia lipolytica</em> (NCBI, <a href="ftp://ftp.ncbi.nih.gov/genomes/Fungi/Yarrowia_lipolytica_CLIB99">ftp://ftp.ncbi.nih.gov/genomes/Fungi/Yar​rowia_lipolytica_CLIB99</a>); <em>D. discoideum</em> (dictyBase, <a href="http://www.dictybase.org/">http://www.dictybase.org/</a>); <em>A. thaliana</em> (TAIR, <a href="http://www.arabidopsis.org/info/agi.jsp">http://www.arabidopsis.org/info/agi.jsp</a>); <em>Oryza sativa</em> (TIGR, <a href="http://www.tigr.org/tdb/e2k1/osa1/">http://www.tigr.org/tdb/e2k1/osa1/</a>); <em>C. intestinalis, Chlamydomonas reinhardtii</em>, and <em>Fugu rubripes</em>, (JGI, <a href="http://genome.jgi-psf.org/euk_cur1.html">http://genome.jgi-psf.org/euk_cur1.html</a>); and <em>P. falciparum</em> (<a href="http://www.sanger.ac.uk/Projects/P_falciparum/">http://www.sanger.ac.uk/Projects/P_falci​parum/</a>). When a gene existed in multiple copies in a given species, the copy with the most introns was used. The sequences of the 76 orthologs of RP genes are available at <a href="http://ribosome.miyazaki-med.ac.jp/">http://ribosome.miyazaki-med.ac.jp/</a>.</p>


<h4>Construction of intron presence/absence matrix and phylogenetic tree</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Multiple sequence alignments for each of the gene orthologs were built using ClustalW <a href="#pone.0000141-Thompson1">[27]</a>, and an <em>ad hoc</em> program was written in the C programming language to extract the intron presence/absence matrix (<a href="#pone.0000141.s001">Dataset S1</a>) and conserved amino acid regions of these alignments. The conserved amino acid regions were then concatenated together and the Seqboot (1000 replicates), Protdist, Neighbor, and Consense programs of the PHYLIP package <a href="#pone.0000141-Felsenstein1">[28]</a> were used to build a neighbor-joining tree with bootstrap values (<a href="#pone-0000141-g001">Figure 1</a>).</p>


<h4>Inference of intron evolution</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Two methods, maximum likelihood <a href="#pone.0000141-Nguyen1">[4]</a> and maximum parsimony <a href="#pone.0000141-Yoshihama2">[7]</a>, were used for inferring the patterns of intron gains and losses. Both methods take as input the intron presence/absence matrix and the phylogenetic tree (with <em>P. falciparum</em> as the out-group). In a previous analysis of intron evolution <a href="#pone.0000141-Nguyen1">[4]</a>, we have shown that there are ~42,000 possible sites for intron insertion, but the number of introns at the root of the phylogenetic tree is rather small (we estimated it to be ~450, which equals the number of introns in <em>P. falciparum</em>). Therefore, the cost function in the maximum parsimony method was modified as follows:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000141.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <em>gains</em> and <em>losses</em> are the numbers of intron gains and losses; <em>K</em> is the ratio between rate of loss and rate of gain; <em>λ</em> shows the fraction of introns present at the root of the phylogenetic tree; and <em>s<sub>r</sub>
               </em> = 0 or 1 means intron absence or presence at the root. The values <em>K</em> = 500 and <em>λ</em> = 0.01 were used here.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000141"></div><a name="pone.0000141.s001" id="pone.0000141.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000141.s001">Dataset S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Matrix of the Entire Set of Intron Presences and Absences.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000141.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB TXT)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank A. Nakao and M. Nagatomo for their help in the preparation of the dataset.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: NK MY HN. Performed the experiments: MY. Analyzed the data: NK MY HN. Wrote the paper: NK MY HN.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000141-Rogozin1" id="pone.0000141-Rogozin1"></a>Rogozin IB, Wolf YI, Sorokin AV, Mirkin BG, Koonin EV (2003) Remarkable interkingdom conservation of intron positions and massive, lineage-specific intron loss and gain in eukaryotic evolution. Curr Biol  13: 1512–1517.  <ul class="find" data-citedArticleID="924596" data-doi="10.1016/s0960-9822(03)00558-x"><li><a href="http://dx.doi.org/10.1016/s0960-9822(03)00558-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Remarkable+interkingdom+conservation+of+intron+positions+and+massive%2C+lineage-specific+intron+loss+and+gain+in+eukaryotic+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Remarkable+interkingdom+conservation+of+intron+positions+and+massive%2C+lineage-specific+intron+loss+and+gain+in+eukaryotic+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000141-Roy1" id="pone.0000141-Roy1"></a>Roy SW, Gilbert W (2005) Rates of intron loss and gain: implications for early eukaryotic evolution. Proc Natl Acad Sci U S A  102: 5773–5778.  <ul class="find" data-citedArticleID="924598" data-doi="10.1073/pnas.0500383102"><li><a href="http://dx.doi.org/10.1073/pnas.0500383102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+of+intron+loss+and+gain%3A+implications+for+early+eukaryotic+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+of+intron+loss+and+gain%3A+implications+for+early+eukaryotic+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000141-Csrs1" id="pone.0000141-Csrs1"></a>Csürös M (2005)  Likely scenarios of intron evolution. In: McLysaght A, Huson D, editors. Proceedings of the Comparative Genomics: RECOMB 2005 International Workshop; 18–20 September 2005; Dublin, Ireland. 3678. Berlin: Springer-Verlag. pp. 47–60.  Lecture Notes in Bioinformatics.  <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0000141-Nguyen1" id="pone.0000141-Nguyen1"></a>Nguyen DH, Yoshihama M, Kenmochi N (2005) New maximum likelihood estimators for eukaryotic intron evolution. PLoS Comput Biol  1: e79.  <ul class="find" data-citedArticleID="924588" data-doi="10.1371/journal.pcbi.0010079"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0010079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+maximum+likelihood+estimators+for+eukaryotic+intron+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+maximum+likelihood+estimators+for+eukaryotic+intron+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000141-Nakao1" id="pone.0000141-Nakao1"></a>Nakao A, Yoshihama M, Kenmochi N (2004) RPG: the Ribosomal Protein Gene database. Nucleic Acids Res  32: D168–170.  <ul class="find" data-citedArticleID="924586" data-doi="10.1093/nar/gkh004"><li><a href="http://dx.doi.org/10.1093/nar/gkh004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RPG%3A+the+Ribosomal+Protein+Gene+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RPG%3A+the+Ribosomal+Protein+Gene+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000141-Yoshihama1" id="pone.0000141-Yoshihama1"></a>Yoshihama M, Uechi T, Asakawa S, Kawasaki K, Kato S, et al.  (2002) The human ribosomal protein genes: sequencing and comparative analysis of 73 genes. Genome Res  12: 379–390.  <ul class="find" data-citedArticleID="924614" data-doi="10.1101/gr.214202"><li><a href="http://dx.doi.org/10.1101/gr.214202" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+ribosomal+protein+genes%3A+sequencing+and+comparative+analysis+of+73+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+ribosomal+protein+genes%3A+sequencing+and+comparative+analysis+of+73+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000141-Yoshihama2" id="pone.0000141-Yoshihama2"></a>Yoshihama M, Nguyen DH, Kenmochi N (2006) Analysis of ribosomal protein gene structures: implications for intron evolution. PLoS Genet  2: e25.  <ul class="find" data-citedArticleID="924616" data-doi="10.1371/journal.pgen.0020025.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020025.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+ribosomal+protein+gene+structures%3A+implications+for+intron+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+ribosomal+protein+gene+structures%3A+implications+for+intron+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000141-Nguyen2" id="pone.0000141-Nguyen2"></a>Nguyen HD, Yoshihama M, Kenmochi N (2006) Phase distribution of spliceosomal introns: implications for intron origin. BMC Evol Biol  6: 69.  <ul class="find" data-citedArticleID="924590"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phase+distribution+of+spliceosomal+introns%3A+implications+for+intron+origin.&amp;auth=&amp;atitle=Phase+distribution+of+spliceosomal+introns%3A+implications+for+intron+origin." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phase+distribution+of+spliceosomal+introns%3A+implications+for+intron+origin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phase+distribution+of+spliceosomal+introns%3A+implications+for+intron+origin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000141-Wool1" id="pone.0000141-Wool1"></a>Wool IG (1979) The structure and function of eukaryotic ribosomes. Annu Rev Biochem  48: 719–754.  <ul class="find" data-citedArticleID="924612" data-doi="10.1146/annurev.bi.48.070179.003443"><li><a href="http://dx.doi.org/10.1146/annurev.bi.48.070179.003443" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+and+function+of+eukaryotic+ribosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+and+function+of+eukaryotic+ribosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000141-Kenmochi1" id="pone.0000141-Kenmochi1"></a>Kenmochi N (2003)  Ribosomes and ribosomal proteins. In: Cooper DN, editor. Nature Encyclopedia of the Human Genome. London: Nature Publishing Group. pp. pp. 77–82. <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0000141-Nixon1" id="pone.0000141-Nixon1"></a>Nixon JE, Wang A, Morrison HG, McArthur AG, Sogin ML, et al.  (2002) A spliceosomal intron in <em>Giardia lamblia</em>. Proc Natl Acad Sci U S A  99: 3701–3705.  <ul class="find" data-citedArticleID="924592" data-doi="10.1073/pnas.042700299"><li><a href="http://dx.doi.org/10.1073/pnas.042700299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+spliceosomal+intron+in+Giardia+lamblia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+spliceosomal+intron+in+Giardia+lamblia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000141-Russell1" id="pone.0000141-Russell1"></a>Russell AG, Shutt TE, Watkins RF, Gray MW (2005) An ancient spliceosomal intron in the ribosomal protein L7a gene (Rpl7a) of <em>Giardia lamblia</em>. BMC Evol Biol  5: 45.  <ul class="find" data-citedArticleID="924602"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+ancient+spliceosomal+intron+in+the+ribosomal+protein+L7a+gene+%28Rpl7a%29+of+Giardia+lamblia.&amp;auth=&amp;atitle=An+ancient+spliceosomal+intron+in+the+ribosomal+protein+L7a+gene+%28Rpl7a%29+of+Giardia+lamblia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+ancient+spliceosomal+intron+in+the+ribosomal+protein+L7a+gene+%28Rpl7a%29+of+Giardia+lamblia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+ancient+spliceosomal+intron+in+the+ribosomal+protein+L7a+gene+%28Rpl7a%29+of+Giardia+lamblia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000141-Hedges1" id="pone.0000141-Hedges1"></a>Hedges SB (2002) The origin and evolution of model organisms. Nat Rev Genet  3: 838–849.  <ul class="find" data-citedArticleID="924576" data-doi="10.1038/nrg929"><li><a href="http://dx.doi.org/10.1038/nrg929" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+origin+and+evolution+of+model+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+origin+and+evolution+of+model+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000141-Battistuzzi1" id="pone.0000141-Battistuzzi1"></a>Battistuzzi FU, Feijao A, Hedges SB (2004) A genomic timescale of prokaryote evolution: insights into the origin of methanogenesis, phototrophy, and the colonization of land. BMC Evol Biol  4: 44.  <ul class="find" data-citedArticleID="924568"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+genomic+timescale+of+prokaryote+evolution%3A+insights+into+the+origin+of+methanogenesis%2C+phototrophy%2C+and+the+colonization+of+land.&amp;auth=&amp;atitle=A+genomic+timescale+of+prokaryote+evolution%3A+insights+into+the+origin+of+methanogenesis%2C+phototrophy%2C+and+the+colonization+of+land." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomic+timescale+of+prokaryote+evolution%3A+insights+into+the+origin+of+methanogenesis%2C+phototrophy%2C+and+the+colonization+of+land." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomic+timescale+of+prokaryote+evolution%3A+insights+into+the+origin+of+methanogenesis%2C+phototrophy%2C+and+the+colonization+of+land.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000141-Roy2" id="pone.0000141-Roy2"></a>Roy SW, Fedorov A, Gilbert W (2003) Large-scale comparison of intron positions in mammalian genes shows intron loss but no gain. Proc Natl Acad Sci U S A  100: 7158–7162.  <ul class="find" data-citedArticleID="924600" data-doi="10.1073/pnas.1232297100"><li><a href="http://dx.doi.org/10.1073/pnas.1232297100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+comparison+of+intron+positions+in+mammalian+genes+shows+intron+loss+but+no+gain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+comparison+of+intron+positions+in+mammalian+genes+shows+intron+loss+but+no+gain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000141-Zdobnov1" id="pone.0000141-Zdobnov1"></a>Zdobnov EM, von Mering C, Letunic I, Torrents D, Suyama M, et al.  (2002) Comparative genome and proteome analysis of <em>Anopheles gambiae</em> and <em>Drosophila melanogaster</em>. Science  298: 149–159.  <ul class="find" data-citedArticleID="924618" data-doi="10.1126/science.1077061"><li><a href="http://dx.doi.org/10.1126/science.1077061" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genome+and+proteome+analysis+of+Anopheles+gambiae+and+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genome+and+proteome+analysis+of+Anopheles+gambiae+and+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000141-Philippe1" id="pone.0000141-Philippe1"></a>Philippe H, Ger mot A (2000) The new phylogeny of eukaryotes. Curr Opin Genet Dev  10: 596–601.  <ul class="find" data-citedArticleID="924594"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+new+phylogeny+of+eukaryotes.&amp;auth=&amp;atitle=The+new+phylogeny+of+eukaryotes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+new+phylogeny+of+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+new+phylogeny+of+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000141-Babenko1" id="pone.0000141-Babenko1"></a>Babenko VN, Rogozin IB, Mekhedov SL, Koonin EV (2004) Prevalence of intron gain over intron loss in the evolution of paralogous gene families. Nucleic Acids Res  32: 3724–3733.  <ul class="find" data-citedArticleID="924566" data-doi="10.1093/nar/gkh686"><li><a href="http://dx.doi.org/10.1093/nar/gkh686" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence+of+intron+gain+over+intron+loss+in+the+evolution+of+paralogous+gene+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence+of+intron+gain+over+intron+loss+in+the+evolution+of+paralogous+gene+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000141-Brett1" id="pone.0000141-Brett1"></a>Brett D, Pospisil H, Valcarcel J, Reich J, Bork P (2002) Alternative splicing and genome complexity. Nat Genet  30: 29–30.  <ul class="find" data-citedArticleID="924570" data-doi="10.1038/ng803"><li><a href="http://dx.doi.org/10.1038/ng803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+splicing+and+genome+complexity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+splicing+and+genome+complexity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000141-The1" id="pone.0000141-The1"></a>The International Genome Sequencing Consortium (2001) Initial sequencing and analysis of the human genome. Nature  409: 860–921.  <ul class="find" data-citedArticleID="924604" data-doi="10.1038/nature04072"><li><a href="http://dx.doi.org/10.1038/nature04072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequencing+and+analysis+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000141-Venter1" id="pone.0000141-Venter1"></a>Venter CJ, Adams MD, Myers EW, Li PW, Mural RJ, et al.  (2001) The sequence of the human genome. Science  291: 1304–1351.  <ul class="find" data-citedArticleID="924610" data-doi="10.1126/science.1058040"><li><a href="http://dx.doi.org/10.1126/science.1058040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sequence+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sequence+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000141-Jeffares1" id="pone.0000141-Jeffares1"></a>Jeffares DC, Mourier T, Penny D (2006) The biology of intron gain and loss. Trends Genet  22: 16–22.  <ul class="find" data-citedArticleID="924578" data-doi="10.1016/j.tig.2005.10.006"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.10.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+biology+of+intron+gain+and+loss." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+biology+of+intron+gain+and+loss.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000141-Mattick1" id="pone.0000141-Mattick1"></a>Mattick JS, Gagen MJ (2001) The evolution of controlled multitasked gene networks: the role of introns and other noncoding RNAs in the development of complex organisms. Mol Biol Evol  18: 1611–1630.  <ul class="find" data-citedArticleID="924582" data-doi="10.1093/oxfordjournals.molbev.a003951"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003951" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+controlled+multitasked+gene+networks%3A+the+role+of+introns+and+other+noncoding+RNAs+in+the+development+of+complex+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+controlled+multitasked+gene+networks%3A+the+role+of+introns+and+other+noncoding+RNAs+in+the+development+of+complex+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000141-Maxwell1" id="pone.0000141-Maxwell1"></a>Maxwell ES, Fournier MJ (1995) The small nucleolar RNAs. Annu Rev Biochem  64: 897–943.  <ul class="find" data-citedArticleID="924584" data-doi="10.1146/annurev.bi.64.070195.004341"><li><a href="http://dx.doi.org/10.1146/annurev.bi.64.070195.004341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+small+nucleolar+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+small+nucleolar+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000141-Tycowski1" id="pone.0000141-Tycowski1"></a>Tycowski KT, Shu MD, Steitz JA (1996) A mammalian gene with introns instead of exons generating stable RNA products. Nature  379: 464–466.  <ul class="find" data-citedArticleID="924608" data-doi="10.1038/379464a0"><li><a href="http://dx.doi.org/10.1038/379464a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mammalian+gene+with+introns+instead+of+exons+generating+stable+RNA+products." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mammalian+gene+with+introns+instead+of+exons+generating+stable+RNA+products.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000141-Andolfatto1" id="pone.0000141-Andolfatto1"></a>Andolfatto P (2005) Adaptive evolution of non-coding DNA in <em>Drosophila</em>. Nature  437: 1149–1152.  <ul class="find" data-citedArticleID="924564" data-doi="10.1038/nature04107"><li><a href="http://dx.doi.org/10.1038/nature04107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+non-coding+DNA+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+non-coding+DNA+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000141-Thompson1" id="pone.0000141-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: Improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="924606" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+Improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+Improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000141-Felsenstein1" id="pone.0000141-Felsenstein1"></a>Felsenstein J (1993) PHYLIP (<em>Phylogeny Inference Package) version</em> 3.5c.  Distributed by the author, Department of Genetics, University of Washington, Seattle, WA.  <ul class="find" data-citedArticleID="924574"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=PHYLIP+%28Phylogeny+Inference+Package%29+version+3.5c.&amp;auth=&amp;atitle=PHYLIP+%28Phylogeny+Inference+Package%29+version+3.5c." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PHYLIP+%28Phylogeny+Inference+Package%29+version+3.5c." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PHYLIP+%28Phylogeny+Inference+Package%29+version+3.5c.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.XML" value="62056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.PDF" value="240532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g001.PNG_L" value="426611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g001.PNG_M" value="31527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g001.PNG_S" value="3090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g001.TIF" value="517488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g001.PNG_I" value="12368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g002.PNG_L" value="613733"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g002.PNG_M" value="81431"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g002.PNG_S" value="5198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g002.TIF" value="1270632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g002.PNG_I" value="70197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g003.PNG_L" value="65102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g003.PNG_M" value="22159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g003.PNG_S" value="2126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g003.TIF" value="109790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.g003.PNG_I" value="9941"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.e001.PNG" value="2271"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.e001.TIF" value="15712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000141.s001.TXT" value="27516"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000141&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000141" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000141&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000141&volume=&issue=&title=Intron Dynamics in Ribosomal Protein Genes&author_name=Maki%20Yoshihama%2C%20Hung%20D.%20Nguyen%2C%20Naoya%20Kenmochi&start_page=1&end_page=5" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000141" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000141" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000141" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000141&amp;t=Intron%20Dynamics%20in%20Ribosomal%20Protein%20Genes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000141&title=Intron%20Dynamics%20in%20Ribosomal%20Protein%20Genes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000141&amp;title=Intron%20Dynamics%20in%20Ribosomal%20Protein%20Genes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000141" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000141&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Intron Dynamics in Ribosomal Protein Genes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000141';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Intron%20Dynamics%20in%20Ribosomal%20Protein%20Genes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000141" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000141" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Eukaryotic+evolution%22" title="Search for articles in the subject area:'Eukaryotic evolution'"><div class="flagText">Eukaryotic evolution</div></a>
              <div data-categoryid="18539" data-articleid="23120"
                   data-categoryname="Eukaryotic evolution"
                   class="flagImage" title="Flag 'Eukaryotic evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="23120"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+evolution%22" title="Search for articles in the subject area:'Fungal evolution'"><div class="flagText">Fungal evolution</div></a>
              <div data-categoryid="17551" data-articleid="23120"
                   data-categoryname="Fungal evolution"
                   class="flagImage" title="Flag 'Fungal evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+genomics%22" title="Search for articles in the subject area:'Fungal genomics'"><div class="flagText">Fungal genomics</div></a>
              <div data-categoryid="33097" data-articleid="23120"
                   data-categoryname="Fungal genomics"
                   class="flagImage" title="Flag 'Fungal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23120"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="20277" data-articleid="23120"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="23120"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+evolution%22" title="Search for articles in the subject area:'Plant evolution'"><div class="flagText">Plant evolution</div></a>
              <div data-categoryid="18531" data-articleid="23120"
                   data-categoryname="Plant evolution"
                   class="flagImage" title="Flag 'Plant evolution' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8485'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5937'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4004&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
