

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Distinctive Core Histone Post-Translational Modification Patterns in Arabidopsis thaliana</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001210"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001210" />

    <meta name="citation_title" content="Distinctive Core Histone Post-Translational Modification Patterns in Arabidopsis thaliana"/>
    <meta itemprop="name" content="Distinctive Core Histone Post-Translational Modification Patterns in Arabidopsis thaliana"/>

      <meta name="citation_author" content="Kangling Zhang"/>
            <meta name="citation_author_institution" content="Mass Spectrometry Facility, School of Medicine, Loma Linda University, Loma Linda, California, United States of America"/>
      <meta name="citation_author" content="Vaniyambadi V. Sridhar"/>
            <meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California at Riverside, Riverside, California, United States of America"/>
      <meta name="citation_author" content="Jianhua Zhu"/>
            <meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California at Riverside, Riverside, California, United States of America"/>
      <meta name="citation_author" content="Avnish Kapoor"/>
            <meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California at Riverside, Riverside, California, United States of America"/>
      <meta name="citation_author" content="Jian-Kang Zhu"/>
            <meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California at Riverside, Riverside, California, United States of America"/>

    <meta name="citation_date" content="2007/11/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001210.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1210"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Crystal structure of the nucleosome core particle at 2.8 Å resolution.; citation_author=K Luger; citation_author=AW Mäder; citation_author=RK Richmond; citation_author=DF Sargent; citation_author=TJ Richmond; citation_journal_title=Nature; citation_volume=389; citation_number=1; citation_pages=251-260; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Higher-order structures of chromatin: the elusive 30 nm fiber.; citation_author=DJ Tremethick; citation_journal_title=Cell; citation_volume=128; citation_number=2; citation_pages=651-654; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Chromatin architecture of the human genome: gene-rich domains are enriched in open chromatin fiber. Mol.; citation_author=N Gilbert; citation_author=S Boyle; citation_author=H Fiehler; citation_author=K Woodfine; citation_author=NP Carter; citation_journal_title=Cell; citation_volume=15; citation_number=3; citation_pages=844-846; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Histone modification in constitutive heterochromatin versus unexpressed euchromatin in human cells.; citation_author=G Jiang; citation_author=F Yang; citation_author=C Sanchez; citation_author=M Ehrlich; citation_journal_title=J. Cell Biochem; citation_volume=93; citation_number=4; citation_pages=286-300; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Translating the histone code.; citation_author=T Jenuwein; citation_author=CD Allis; citation_journal_title=Science; citation_volume=293; citation_number=5; citation_pages=1074-1080; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of acetylated histones H3 and H4 and their assembly into nucleosomes.; citation_author=KW Marvin; citation_author=P Yau; citation_author=EM Bradbury; citation_journal_title=J. Biol. Chem.; citation_volume=265; citation_number=6; citation_pages=19839-19847; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Identification of acetylation and methylation sites of histone H3 from chicken erythrocytes by high-accuracy matrix-assisted laser desorption ionization-time-of flight, matrix-assisted laser desorption ionization-postsource decay, and nanoelectrospray ionization tandem mass spectrometry.; citation_author=K Zhang; citation_author=H Tang; citation_author=L Huang; citation_author=JW Blankenship; citation_author=PR Jones; citation_journal_title=Anal. Biochem.; citation_volume=306; citation_number=7; citation_pages=259-269; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mapping global histone acetylation patterns to gene expression.; citation_author=SK Kurdistani; citation_author=S Tavazole; citation_author=M Grunstein; citation_journal_title=Cell; citation_volume=117; citation_number=8; citation_pages=721-733; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Lysine methylation within the globular domain of histone H3 by Dot1 is important for telomeric silencing and Sir protein association.; citation_author=HH Ng; citation_author=Q Feng; citation_author=H Wang; citation_author=H Erdjument-Bromage; citation_author=C Borchers; citation_journal_title=Genes & Dev.; citation_volume=16; citation_number=9; citation_pages=1518-1527; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Acetylation in histone globular domain regulates gene expression in yeast.; citation_author=F Xu; citation_author=K Zhang; citation_author=M Grunstein; citation_journal_title=Cell; citation_volume=121; citation_number=10; citation_pages=375-385; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A role for cell-cycle-regulated histone H3 lysine 56 acetylation in the DNA damage response.; citation_author=H Masumoto; citation_author=D Hawke; citation_author=R Kobayashi; citation_author=A Verreault; citation_journal_title=Nature; citation_volume=436; citation_number=11; citation_pages=294-298; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Differentiation of acetylated peptides from methylated peptides by mass spectrometry: An application for determining lysine 9 acetylation and methylation of histone H3.; citation_author=K Zhang; citation_author=PM Yau; citation_author=R New; citation_author=R Kondrat; citation_author=BS Imai; citation_journal_title=Proteomics; citation_volume=4; citation_number=12; citation_pages=1-10; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A mass spectrometric “western blot” to evaluate the correlations between histone acetylation and methylation.; citation_author=K Zhang; citation_author=JS Siino; citation_author=PR Jones; citation_author=PM Yau; citation_author=EM Bradbury; citation_journal_title=Proteomics; citation_volume=4; citation_number=13; citation_pages=3765-3775; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Immunological analysis of yeast chromatin.; citation_author=P Meluh; citation_author=JR Broach; citation_journal_title=Methods Enzymol.; citation_volume=304; citation_number=14; citation_pages=414-430; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Signaling to chromatin through histone modifications.; citation_author=P Cheung; citation_author=CD Allis; citation_author=PS Corsi; citation_journal_title=Cell; citation_volume=103; citation_number=15; citation_pages=263-271; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Vernalization requires epigenetic silencing of FLC by histone methylation.; citation_author=R Bastow; citation_author=JS Mylne; citation_author=C Lister; citation_author=Z Lippman; citation_author=RA Martienssen; citation_journal_title=Nature; citation_volume=427; citation_number=16; citation_pages=164-167; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Vernalization in Arabidopsis thaliana is mediated by the PHD finger protein VIN3.; citation_author=S Sung; citation_author=RM Amasin; citation_journal_title=Nature; citation_volume=427; citation_number=17; citation_pages=159-164; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis thaliana vernalization response requires a polycomb-like protein complex that also includes VERNALIZATION INSENSITIVE 3.; citation_author=CC Wood; citation_author=M Robertson; citation_author=G Tanner; citation_author=WJ Peacock; citation_author=ES Dennis; citation_journal_title=Proc. Natl. Acad. Sci. U S A.; citation_volume=103; citation_number=18; citation_pages=14631-14636; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=RNAi-mediated pathways in the nucleus.; citation_author=MA Matzke; citation_author=JA Birchler; citation_journal_title=Nat. Rev. Genet.; citation_volume=6; citation_number=19; citation_pages=24-35; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Epigenetic codes for heterochromatin formation and silencing: rounding up the usual suspects.; citation_author=EJ Richards; citation_author=SC Elgin; citation_journal_title=Cell; citation_volume=108; citation_number=20; citation_pages=489-500; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=DNA and histone methylation in plants.; citation_author=M Tariq; citation_author=J Paszkowski; citation_journal_title=Trends Genet.; citation_volume=20; citation_number=21; citation_pages=244-251; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chromatin-based silencing mechanisms.; citation_author=J Bender; citation_journal_title=Curr Opin Plant Biol; citation_volume=7; citation_number=22; citation_pages=521-526; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chromosomal histone modification patterns-from conservation to diversity.; citation_author=J Fuchs; citation_author=D demidov; citation_author=A Houben; citation_author=I Schubert; citation_journal_title=Trends Plant; citation_volume=Sci.11; citation_number=23; citation_pages=199-208; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Control of CpNpG DNA methylation by the KRYPTONITE histone H3 methyltransferase.; citation_author=JP Jackson; citation_author=AM Lindroth; citation_author=X Cao; citation_author=SE Jacobsen; citation_journal_title=Nature; citation_volume=416; citation_number=24; citation_pages=556-560; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Requirement of CHROMOMETHYLASE3 for maintenance of CpXpG methylation.; citation_author=AM Lindroth; citation_author=X Cao; citation_author=JP Jackson; citation_author=D Zilberman; citation_author=CM McCallum; citation_journal_title=Science; citation_volume=292; citation_number=25; citation_pages=2077-2080; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Interplay between two epigenetic marks. DNA methylation and histone H3 lysine 9 methylation.; citation_author=L Johnson; citation_author=X Cao; citation_author=S Jacobsen; citation_journal_title=Curr. Biol.; citation_volume=12; citation_number=26; citation_pages=1360-1367; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Distinct regulation of histone H3 methylation at lysines 27 and 9 by CpG methylation in Arabidopsis.; citation_author=O Mathieu; citation_author=AV Probst; citation_author=J Paszkowski; citation_journal_title=EMBO J.; citation_volume=24; citation_number=27; citation_pages=2783-2791; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mass spectrometry analysis of Arabidopsis histone H3 reveals distinct combinations of post-translational modifications.; citation_author=L Johnson; citation_author=S Mollah; citation_author=BA Garcia; citation_author=TL Muratore; citation_author=J Shabanowitz; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=28; citation_pages=6511-6518; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Histones and histone modifications.; citation_author=JR Peterson; citation_author=MA Laniel; citation_journal_title=Curr. Biol.; citation_volume=14; citation_number=29; citation_pages=R546-R551; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Role of histone H2A ubiquiination in polycomb silencing.; citation_author=HL Wang; citation_author=L Wang; citation_author=H Erdjument-Bromage; citation_author=M Vidal; citation_author=P Tempst; citation_journal_title=Nature; citation_volume=431; citation_number=30; citation_pages=873-878; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Nucleosomal histone kinase-1 phosphorylates H2A Thr119 during mitosis in the early Drosophila embryo.; citation_author=H Aihara; citation_author=T Nakagawa; citation_author=K Yasui; citation_author=T Ohta; citation_author=S Hirose; citation_journal_title=Genes & Dev.; citation_volume=18; citation_number=31; citation_pages=877-888; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A rapid method for determining protein kinase phosphorylation specificity.; citation_author=JE Hutti; citation_author=ET Jarrell; citation_author=JD Chang; citation_author=DW Abbott; citation_author=P Storz; citation_journal_title=Nat. Methods; citation_volume=1; citation_number=32; citation_pages=27-29; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Quantitative proteomic analysis of post-translational modifications of human histones.; citation_author=HC Beck; citation_author=RM Nielsen; citation_author=R Matthiesen; citation_author=LH Jensen; citation_author=M Sehested; citation_journal_title=Mol. Cell. Proteomics; citation_volume=5; citation_number=33; citation_pages=1314-1325; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Control of DNA methylation and heterochromatic silencing by histone H2B deubiquitination.; citation_author=VV Sridhar; citation_author=A Kapoor; citation_author=K Zhang; citation_author=J Zhu; citation_author=T Zhou; citation_journal_title=Nature; citation_volume=447; citation_number=34; citation_pages=735-738; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Identification of ubiquitination sites and determination of ubiquitin-chain architectures by mass spectrometry.; citation_author=P Kaiser; citation_author=J Wohlschlegel; citation_journal_title=Methods Enzymol; citation_volume=399; citation_number=35; citation_pages=266-276; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mapping of ubiquitation sites on target proteins.; citation_author=S Hatakeyama; citation_author=M Matsumoto; citation_author=KI Nakayama; citation_journal_title=Methods Enzymol; citation_volume=399; citation_number=36; citation_pages=277-286; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Patterns of histone acetylation.; citation_author=AW Thorne; citation_author=D Kmiciek; citation_author=K Mitchelson; citation_author=P Sautiere; citation_author=C Crane-Robinson; citation_journal_title=Eur. J. Biochem.; citation_volume=179; citation_number=37; citation_pages=701-714; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Increased ionizing radiation sensitivity and genomic instability in the absence of histone H2AX.; citation_author=CH Bassing; citation_author=KF Chua; citation_author=J Sekiguchi; citation_author=H Suh; citation_author=SR Whitlow; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=99; citation_number=38; citation_pages=8173-8178; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Proline isomerization of histone H3 regulates lysine methylation and gene expression.; citation_author=CJ Nelson; citation_author=H Santos-Rosa; citation_author=T Kouzarides; citation_journal_title=Cell; citation_volume=126; citation_number=39; citation_pages=905-916; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sterile 20 kinase phosphorylates histone H2B at serine 10 during hydrogen peroxide-induced apoptosis in S. cerevisiae.; citation_author=S Ahn; citation_author=WL Cheung; citation_author=J Hsu; citation_author=RL Diaz; citation_author=MM Smith; citation_journal_title=Cell; citation_volume=120; citation_number=40; citation_pages=25-36; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis thaliana homolog of yeast BRE1 has a function in cell cycle regulation during early leaf and root growth.; citation_author=D Fleury; citation_author=K Himanen; citation_author=G Cnops; citation_author=H Nelissen; citation_author=TM Boccardi; citation_journal_title=Plant Cell; citation_volume=19; citation_number=41; citation_pages=417-32; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The absence of histone H2B monoubiquitination in the Arabidopsis hub1 (rdo4) mutant reveals a role for chromatin remodeling in seed dormancy.; citation_author=Y Liu; citation_author=M Koornneef; citation_author=WJ Soppe; citation_journal_title=Plant Cell; citation_volume=19; citation_number=42; citation_pages=433-44; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Structure of the single-stranded-DNA-binding domain of replication protein A bound to DNA.; citation_author=A Bochkrev; citation_author=RA Pfuetzner; citation_author=AM Edwards; citation_author=L Frappier; citation_journal_title=Nature; citation_volume=385; citation_number=43; citation_pages=176-181; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Purification and functional characterization of SET8, a nucleosomal histone H4-lysine 20-specific methyltransferase.; citation_author=J Fang; citation_author=Q Feng; citation_author=CS Ketel; citation_author=H Wang; citation_author=R Cao; citation_journal_title=Curr. Biol.; citation_volume=12; citation_number=44; citation_pages=1086-1099; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mitotic-specific methylation of histone H4 Lys 20 follows increased PR-Set7 expression and its localization to mitotic chromosomes.; citation_author=JC Rice; citation_author=K Nishioka; citation_author=K Sarma; citation_author=R Steward; citation_author=D Reinberg; citation_journal_title=Genes & Dev.; citation_volume=16; citation_number=45; citation_pages=2225-2230; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Methylation of histone H4 lysine 20 controls recruitment of Crb2 to sites of DNA damage.; citation_author=SL Sanders; citation_author=P Manuela; citation_author=J Mata; citation_author= Bähler; citation_author=RC JüAllshire; citation_journal_title=Cell; citation_volume=119; citation_number=46; citation_pages=603-614; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A silencing pathway to induce H3-K9 and H4-K20 trimethylation at constitutive heterochromatin.; citation_author=G Schotta; citation_author=M Lachner; citation_author=K Sarma; citation_author=A Ebert; citation_author=R Sengupta; citation_journal_title=Genes & Dev.; citation_volume=18; citation_number=47; citation_pages=1251-1262; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of five sites of acetylation in Alfalfa histone H4.; citation_author=JH Waterborg; citation_journal_title=Biochemistry; citation_volume=31; citation_number=48; citation_pages=6211-6219; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Histone variants and acetylated species from the alfalfa plant Medicago sativa. Arch. Biochem.; citation_author=JH Waterborg; citation_author=I Winicov; citation_author=RE Harrington; citation_journal_title=Biophys.; citation_volume=256; citation_number=49; citation_pages=167-178; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Global, in vivo, and site-specific phosphorylation dynamics in signaling networks.; citation_author=JV Olsen; citation_author=B Bagoev; citation_author=F Gnad; citation_author=B Macek; citation_author=C Kumar; citation_journal_title=Cell; citation_volume=127; citation_number=50; citation_pages=635-648; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=From purification of large amount of phosphor-compounds (nucleotides) to the enrichment of phospho-peptides by anion-exchanging resin.; citation_author=K Zhang; citation_journal_title=Anal.Biochem; citation_volume=357; citation_number=51; citation_pages=225-231; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001210" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Distinctive Core Histone Post-Translational Modification Patterns in Arabidopsis thaliana"/>
    <meta name="twitter:description" content="Post-translational modifications of histones play crucial roles in the genetic and epigenetic regulation of gene expression from chromatin. Studies in mammals and yeast have found conserved modifications at some residues of histones as well as non-conserved modifications at some other sites. Although plants have been excellent systems to study epigenetic regulation, and histone modifications are known to play critical roles, the histone modification sites and patterns in plants are poorly defined. In the present study we have used mass spectrometry in combination with high performance liquid chromatography (HPLC) separation and phospho-peptide enrichment to identify histone modification sites in the reference plant, Arabidopsis thaliana. We found not only modifications at many sites that are conserved in mammalian and yeast cells, but also modifications at many sites that are unique to plants. These unique modifications include H4 K20 acetylation (in contrast to H4 K20 methylation in non-plant systems), H2B K6, K11, K27 and K32 acetylation, S15 phosphorylation and K143 ubiquitination, and H2A K144 acetylation and S129, S141 and S145 phosphorylation, and H2A.X S138 phosphorylation. In addition, we found that lysine 79 of H3 which is highly conserved and modified by methylation and plays important roles in telomeric silencing in non-plant systems, is not modified in Arabidopsis. These results suggest distinctive histone modification patterns in plants and provide an invaluable foundation for future studies on histone modifications in plants."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001210.g004"/>

  <meta property="og:title" content="Distinctive Core Histone Post-Translational Modification Patterns in Arabidopsis thaliana" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001210" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8500'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9259'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7773&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001210">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001210" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001210&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Distinctive Core Histone Post-Translational Modification Patterns in <i>Arabidopsis thaliana</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kangling Zhang
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Kangling Zhang, Vaniyambadi V. Sridhar</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kzhang@llu.edu">kzhang@llu.edu</a> (KZ); <a href="mailto:jian-kang.zhu@ucr.edu">jian-kang.zhu@ucr.edu</a> (JZ)</p>

                <p>Affiliation:
                  Mass Spectrometry Facility, School of Medicine, Loma Linda University, Loma Linda, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vaniyambadi V. Sridhar
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Kangling Zhang, Vaniyambadi V. Sridhar</p>

              
              

                <p>Affiliation:
                  Department of Botany and Plant Sciences, University of California at Riverside, Riverside, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jianhua Zhu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Botany and Plant Sciences, University of California at Riverside, Riverside, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Avnish Kapoor, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Botany and Plant Sciences, University of California at Riverside, Riverside, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jian-Kang Zhu
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kzhang@llu.edu">kzhang@llu.edu</a> (KZ); <a href="mailto:jian-kang.zhu@ucr.edu">jian-kang.zhu@ucr.edu</a> (JZ)</p>

                <p>Affiliation:
                  Department of Botany and Plant Sciences, University of California at Riverside, Riverside, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001210</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001210" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001210" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001210" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001210" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001210-g001" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001210-t001" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001210-g002" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001210-t002" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001210-g003" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001210-t003" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001210-g004" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001210-t004" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001210">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001210" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Post-translational modifications of histones play crucial roles in the genetic and epigenetic regulation of gene expression from chromatin. Studies in mammals and yeast have found conserved modifications at some residues of histones as well as non-conserved modifications at some other sites. Although plants have been excellent systems to study epigenetic regulation, and histone modifications are known to play critical roles, the histone modification sites and patterns in plants are poorly defined. In the present study we have used mass spectrometry in combination with high performance liquid chromatography (HPLC) separation and phospho-peptide enrichment to identify histone modification sites in the reference plant, <em>Arabidopsis thaliana</em>. We found not only modifications at many sites that are conserved in mammalian and yeast cells, but also modifications at many sites that are unique to plants. These unique modifications include H4 K20 acetylation (in contrast to H4 K20 methylation in non-plant systems), H2B K6, K11, K27 and K32 acetylation, S15 phosphorylation and K143 ubiquitination, and H2A K144 acetylation and S129, S141 and S145 phosphorylation, and H2A.X S138 phosphorylation. In addition, we found that lysine 79 of H3 which is highly conserved and modified by methylation and plays important roles in telomeric silencing in non-plant systems, is not modified in Arabidopsis. These results suggest distinctive histone modification patterns in plants and provide an invaluable foundation for future studies on histone modifications in plants.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zhang K, Sridhar VV, Zhu J, Kapoor A, Zhu J-K (2007) Distinctive Core Histone Post-Translational Modification Patterns in <em>Arabidopsis thaliana</em>. PLoS ONE 2(11):
          e1210.
            doi:10.1371/journal.pone.0001210</p><p><strong>Academic Editor: </strong>Frederic Berger, Temasek Life Sciences Laboratory, Singapore</p><p><strong>Received:</strong> July 30, 2007; <strong>Accepted:</strong> October 22, 2007; <strong>Published:</strong> November 21, 2007</p><p><strong>Copyright:</strong> © 2007 Zhang et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by NIH grant R01GM070795 to JKZ.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Core histones, in the form of an octamer consisting of two copies of H2A, H2B, H3 and H4, are wrapped by 147 bp DNA to form the nucleosome <a href="#pone.0001210-Luger1">[1]</a>. Multiple nucleosome units are lined up to form beads-on-a-string structure, which can be further compacted into a 30 nm fiber-the foundation of chromatin structure, through the assistance of linker histone H1 and the binding of transcription factors or cofactors <a href="#pone.0001210-Tremethick1">[2]</a>. Chromatin can be either extended to form an open, active euchromatin accessible to transcription factors/cofactors or condensed to form a closed, silent heterochromatin inaccessible to transcription factors/cofactors <a href="#pone.0001210-Gilbert1">[3]</a>. Histone modifications play a vital role in determining these states of chromatin during growth and development <a href="#pone.0001210-Jiang1">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Histone post-translational modifications take place mainly on the N-terminal tails of histones. Histone methylation together with other modifications (acetylation, phosphorylation, ubiquitination, sumoylation and ADP-ribosylation) orchestrates an important functional role in gene expression and a “histone code” was accordingly hypothesized <a href="#pone.0001210-Jenuwein1">[5]</a>. The modifications such as acetylation and methylation of lysine residues, are conserved at some lysine residues of histones while not at others. For example, lysine 5, 8, 12 and 16 of histone H4 are acetylated in many species including humans, fly and yeast. However, the modifications at some other lysines, for example, H3 lysine 9 and lysine 27 can be both acetylated and methylated in humans but only acetylated in budding yeast and only methylated in chicken erythrocytes <a href="#pone.0001210-Marvin1">[6]</a>–<a href="#pone.0001210-Kurdistani1">[8]</a>. Histones can also be modified outside their N-terminal regions, as evidenced by the recent discovery of methylation of lysine 79 and acetylation of lysine 56 of histone H3 via mass spectrometry analysis, which is supported by subsequent genetic studies <a href="#pone.0001210-Zhang1">[7]</a>, <a href="#pone.0001210-Ng1">[9]</a>–<a href="#pone.0001210-Masumoto1">[11]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Mass spectrometry is capable of not only providing direct information on the site and type of modification, and differentiating between two nominally isobaric modifications (i.e., acetylation versus tri-methylation), but also implementing quantitative analysis. For example mass spectrometry can be used to determine the acetylation level at certain lysine residues and in some cases, also the level of methylation at specific lysines (such as lysine 4 and lysine 79 of H3) <a href="#pone.0001210-Zhang1">[7]</a>, <a href="#pone.0001210-Zhang2">[12]</a>–<a href="#pone.0001210-Zhang3">[13]</a>. Mass spectrometry thus has been widely recognized as an irreplaceable tool for studying histone modifications and in combination with chromatin immunoprecipitation (ChIP) and/or immunofluorescence assays has the potential for identifying new histone modification sites, modification patterns, and for genome-wide chromatin-related functional studies <a href="#pone.0001210-Meluh1">[14]</a>–<a href="#pone.0001210-Cheung1">[15]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The reference plant, <em>Arabidopsis thaliana</em>, is an excellent model organism for studies on epigenetic regulation. Histone modification patterns have been shown to be critical for establishing and maintaining stable epigenetic states of genes in <em>Arabidopsis</em>. For example, prolonged cold treatment (i.e. vernalization) triggers increases in H3K9 and H3K27 dimethylation and decreases in H3K4 trimethylation and histone acetylation at the FLC locus, causing a stable repression of FLC that is maintained through mitosis even at warm temperatures <a href="#pone.0001210-Bastow1">[16]</a>–<a href="#pone.0001210-Sung1">[17]</a>. The Polycomb Repressive Complex 2-mediated repression of FLC is necessary for flowering of the <em>Arabidopsis</em> plants <a href="#pone.0001210-Wood1">[18]</a>. <em>Arabidopsis</em> is also an outstanding model system to dissect the interplay between small interfering RNAs (siRNAs), DNA methylation and histone modifications <a href="#pone.0001210-Matzke1">[19]</a>–<a href="#pone.0001210-Fuchs1">[23]</a>. In plants, 24 nt siRNAs can direct DNA methylation of complementary sequences, causing stable transcriptional gene silencing <a href="#pone.0001210-Matzke1">[19]</a>. The heterochromatic marker, H3K9 methylation, can direct DNA methylation by the plant specific CMT3 DNA methyltransferase <a href="#pone.0001210-Jackson1">[24]</a>–<a href="#pone.0001210-Lindroth1">[25]</a>. On the other hand, DNA methylation is also required for high level H3K9 methylation <a href="#pone.0001210-Johnson1">[26]</a>–<a href="#pone.0001210-Mathieu1">[27]</a>. Notwithstanding the importance of histone modifications in chromatin regulation in plants, only the N-terminal modifications on <em>Arabidopsis</em> histone H3 have been systematically analyzed by mass spectrometry combined with chromatographic separation <a href="#pone.0001210-Johnson2">[28]</a>. The analysis found several conserved modifications on histone H3 such as methylation at K4, K9, K27 and K36 and acetylation at K14 and K18 <a href="#pone.0001210-Johnson2">[28]</a>. We have used mass spectrometry to analyze modifications of all core histones in <em>Arabidopsis.</em> Our results not only confirm acetylation and methylation at some conserved lysine residues in the four core histones (H2A, H2B, H3 and H4), but also reveal many unexpected distinctive modifications at other sites <a href="#pone.0001210-Peterson1">[29]</a>. These unique modifications include acetylation at K20 of H4, acetylation at K6, K11, K27 and K32, phosphorylation at S15 and ubiquitination at K143 of H2B, acetylation at K144 and phosphorylation at S129, S141 and S145 of H2A, and phosphorylation at S138 of H2AX. In addition, we did not find any modification at lysine 79 of H3 which is highly conserved in non-plant systems. Collectively, our analysis of core histone modification sites here, albeit still incomplete, will be invaluable for future studies of histone modifications in plant genetic and epigenetic regulation.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Identification of modification sites in H2A</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The histone H2A fraction collected from HPLC eluant was digested by trypsin and analyzed by liquid chromatography-tandem mass spectrometry (LC/MS/MS) on a QTOF instrument. The acquired raw data were converted to peak-list (PKL) files, which were submitted for MASCOT searching of proteins and protein post-translational modification sites. The TIC trace of the signature immonium ion of acetylated lysines at <em>m/z</em> 126.1 was also displayed to monitor the ion intensity for manually analyzing the MS/MS spectra of acetylated peptides, which could be missed by software search.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In <em>Arabidopsis thaliana</em>, H2A has eight isoforms (H2A.1–H2A.8) (<a href="#pone-0001210-g001">Fig. 1H</a>). Among the isoforms, H2A.1, H2A.2, H2A.3 and H2A.4 share a similar N-terminal sequence AGRGKT(Q)TLGSGS(V, A , G)AK where serine at position 11 of H2A.1 is replaced by either valine (H2A.2) or glycine (H2A.4), or two residues threonine and serine at position 6 and 11 are replaced by glutamine and alanine (H2A.3). We found that the lysine 5 in all the four H2A isoforms was acetylated (<a href="#pone.0001210.s002">Fig S2</a>). For instance, from the fragmentation ions of the four H2A peptides with sequences shown in <a href="#pone.0001210.s002">Figure S2A–D</a>, an observation of the immonium ion at m/z 126 and a 42 mass unit added to lysine 5 indicated an acetylation rather than tri-methylation at lysine 5. The fragmentation ions, namely “b” and “y” ions labeled in individual figures (<a href="#pone.0001210.s002">Figure S2A–D</a>), matched well with the peptide sequences of four H2As given by the NCBI accession number gi 15223708 (H2A.1), 15232330 (H2A.2), 15237024 (H2A.3) and 15239697 (H2A.4). Histone H2A is known to be conservatively acetylated at lysine 5 and lysine 9 in other species including human, bovine, chicken erythrocyte and yeast (yeast equivalent sites K4 and K7) <a href="#pone.0001210-Zhang1">[7]</a>, <a href="#pone.0001210-Peterson1">[29]</a>. In the plant H2A and its isoforms, there is no lysine at position 9 or its vicinity for acetylation (<a href="#pone-0001210-t001">Table 1</a> and <a href="#pone-0001210-g001">Fig. 1H</a>).</p>
<div class="figure" id="pone-0001210-g001"><div class="img"><a name="pone-0001210-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001210.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001210.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Identification of H2A modification sites.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A. MS/MS spectrum of doubly-charged precursor ion at <em>m/z</em> 411.72 demonstrating phosphorylation at S145 in the peptide SPVK<sup>145</sup><sub>p</sub>SPK of H2A.7. B. MS/MS spectrum of doubly-charged precursor ion at <em>m/z</em> 432.69 demonstrating phosphorylation at S145 and acetylation at K144 in peptide SPV<sup>144</sup>K<sub>ac</sub><sup>145</sup><sub>p</sub>SPK of H2A.7. C. MS/MS spectrum of doubly-charged precursor ion at <em>m/z</em> 582.88 demonstrating phosphorylation at S145 and acetylation at K144 in peptide ATKSPV<sup>144</sup>K<sub>ac</sub><sup>145</sup><sub>p</sub>SPK of H2A.7. D. MS/MS spectrum of doubly-charged precursor ion at <em>m/z</em> 601.78 demonstrating phosphorylation at S141 and S145 in peptide ATK<sup>141</sup><sub>p</sub>SPVK<sup>145</sup><sub>p</sub>SPK of H2A.7. E. MS/MS spectrum of the doubly-charged precursor ion at <em>m/z</em> 584.80 demonstrating phosphorylation at S129 in peptide K<sup>129</sup><sub>p</sub>SATKPAEEK of H2A.7. F. MS/MS spectrum of the doubly-charged precursor ion at <em>m/z</em> 456.24 demonstrating phosphorylation at S145 and acetylation at K144 in the peptide ASAT<sup>144</sup>K<sub>ac</sub><sup>145</sup><sub>p</sub>SPK of H2A.5. G. MS/MS spectrum of the doubly-charged precursor ion at <em>m/z</em> 666.81 demonstrating phosphorylation at S138 in the peptide NKGDIGSA<sup>138</sup><sub>p</sub>SQEF of H2AX. H. Sequence alignment of H2A isoforms (H2A.1-H2A.8). K in green color: acetylation; S in red color: phosphorylation.</p>
<span>doi:10.1371/journal.pone.0001210.g001</span></div><div class="figure" id="pone-0001210-t001"><div class="img"><a name="pone-0001210-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001210.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001210.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Comparison between plant and non-plant H2A modifications</span></strong></p><span>doi:10.1371/journal.pone.0001210.t001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Next, we determined whether there were other modifications (mainly phosphorylation and ubiquitination) associated with histone 2A. Particular attention was given to the known modification sites observed in other species. S1 is a known H2A phosphorylation site observed in human and yeast. However, in <em>Arabidopsis</em> H2A, S1 is replaced by alanine in isoforms H2A.1-H2A.4 and by other amino acids (E, D or V) in isoforms H2A.5-H2A.8. Therefore, S1 phosphorylation cannot exist in <em>Arabidopsis</em>. Ubiquitination at lysine 120/119 of H2A was observed in human/<em>Drosophila</em> with the consensus peptide sequence (P)K<sup>119/118</sup>KT <a href="#pone.0001210-Wang1">[30]</a>–<a href="#pone.0001210-Aihara1">[31]</a>, while K128, similar to K120 in yeast, in the same consensus sequence in <em>Arabidopsis</em> (and yeast) was found to be free of ubiquitination (<a href="#pone-0001210-t001">Table 1</a>). Surprisingly, in the H2A isoform H2A.7 (gi 15238549) a marked amount of phosphorylation was detected at serine 145 (estimated by the ion intensity) in the C-terminal extension even without enrichment for phospho-peptides (<a href="#pone-0001210-g001">Fig. 1A</a>). Considering the sulphuric acid used for the precipitation of histones from nucleosomes and that the process might decompose most of the phospho-histones, we used cation exchange (using Bio-Rex 70) column chromatography to purify DNA binding proteins including histones followed by trypsin digestion and enrichment of phospho-peptides by a mixture of anion exchange resin and TiO<sub>2</sub>. LC/MS/MS analysis of this “treated” sample detected six additional phospho-peptides. They are H2A.7 peptide SPV<sup>144</sup>K<sub>ac</sub><sup>145</sup><sub>p</sub>SPK (MS/MS spectrum shown in <a href="#pone-0001210-g001">Fig. 1B</a>) where K144 is acetylated and S145 is phosphorylated, peptide ATKSPV<sup>144</sup>K<sub>ac</sub><sup>145</sup><sub>p</sub>SPK (MS/MS spectrum shown in <a href="#pone-0001210-g001">Fig. 1C</a>) where K144 is acetylated and S145 is phosphorylated, peptide ATK<sup>141</sup><sub>p</sub>SPVK<sup>145</sup><sub>p</sub>SPK (MS/MS spectrum shown in <a href="#pone-0001210-g001">Fig. 1D</a>) where S141 and S145 are both phosphorylated, peptide K<sup>129</sup><sub>p</sub>SATKPAEEK (MS/MS spectrum shown in <a href="#pone-0001210-g001">Fig. 1E</a>) where S129 is phosphorylated, and H2A.5 peptide ASAT<sup>144</sup>K<sub>ac</sub><sup>145</sup><sub>p</sub>SPK (gi 15241016) (MS/MS spectrum shown in <a href="#pone-0001210-g001">Fig. 1F</a>) where K144 is acetylated and S145 is phosphorylated. The acetylation of lysine (K144) next to the phosphorylated serine (S145) in H2A.5 and H2A.7 was unexpected, and the observation provides evidence of coexistence of acetylation and phosphorylation at two adjacent residues (K and S). Because of the similarity of C-terminal sequence of H2A.6 (gi 15241857) to those of H2A.5 and H2A.7, K144 and S145 (<a href="#pone-0001210-g001">Fig. 1F</a>) in H2A.6 are likely acetylated and phosphorylated, although they were not detected by mass spectrometry. Unlike K5 acetylation observed on the N-terminus of H2A.1-H2A.4, we did not detect acetylation on the N-terminus of H2A.5-H2A.7, possibly because acetylation enzymes (histone acetyltransferases) like phosphorylation enzymes (kinases) also recognize specific peptide consensus sequence which is absent in these three H2A isoforms (<a href="#pone-0001210-g001">Figure 1H</a>) <a href="#pone.0001210-Hutti1">[32]</a>. More interestingly, we were able to identify S138 phosphorylation of H2A variant H2A.X (gi 15221875) from the MS/MS fragmentation of the doubly-charged precursor ion at <em>m/z</em> 666.90 (<a href="#pone-0001210-g001">Fig. 1G</a>). It is also noteworthy that we did not find any modifications in H2A.8 (gi 15236314), possibly because either H2A.8 has non-conservative N- and C-terminal sequences or the protein/its modification level was too low to be detected by mass spectrometry.</p>


<h4>Identification of modification sites in H2B</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The HPLC fraction containing H2B was digested by trypsin and analyzed by LC/MS/MS. From the MS/MS spectrum of the precursor ion at <em>m/z</em> 471.8 (<a href="#pone-0001210-g002">Figure 2A</a>), a peptide with the sequence AE<sup>6</sup>K<sub>ac</sub>KPAEK from the N-terminus of H2B was obtained where lysine 6 was determined to be acetylated. This was based on the detection of y6 and y7 ions corresponding to the fragmentation of an acetylated peptide rather than the neutral loss of y6-59 and y7-59 ions corresponding to the fragmentation of a tri-methylated peptide. Similar analyses can be applied to the peptide AE<sup>27</sup>K<sub>ac</sub>APAEK with doubly-charged precursor ion at <em>m/z</em> 443.24 where K27 was determined to be acetylated by MS/MS (<a href="#pone-0001210-g002">Fig. 2B</a>); the peptide SKAE<sup>27</sup>K<sub>ac</sub>APAEK with doubly-charged precursor ion at <em>m/z</em> 550.88 where K27 was determined to be acetylated by MS/MS (<a href="#pone-0001210-g002">Fig. 2C</a>); the peptide APAE<sup>32</sup>K<sub>ac</sub>KPK with triple-charged precursor ion at <em>m/z</em> 304.17 where K32 was determined to be acetylated by MS/MS (<a href="#pone-0001210-g002">Fig. 2D</a>); the H2B.1-specific peptide KPAE<sup>11</sup>K<sub>ac</sub> KPAAE<sup>17</sup>KPVEEK with quadruple-charged precursor ion at <em>m/z</em> 456.01 where K11 was determined to be acetylated by MS/MS (<a href="#pone-0001210-g002">Fig. 2E</a>); the peptide KPAE<sup>11</sup>K<sub>ac</sub> KPASE<sup>17</sup>KPVEEK specific to H2B.2 with quadruple-charged precursor ion at <em>m/z</em> 460.01 where K11 was determined to be acetylated by MS/MS (<a href="#pone-0001210-g002">Fig. 2F</a>); and the peptide AE<sup>6</sup>K<sub>ac</sub>KPAE<sup>11</sup>K<sub>ac</sub> KPASEKPVEEK specific to H2B.2 with quadruple-charged precursor ion at <em>m/z</em> 552.70 where both K6 and K11 were determined to be acetylated by MS/MS (<a href="#pone-0001210-g002">Fig. 2G</a>). Therefore, our mass spectrometric analysis revealed four acetylation sites on the N-terminus of <em>Arabidopsis</em> H2B (<a href="#pone-0001210-g002">Fig. 2I</a>), compared with six acetylation sites on the N-terminus of human <a href="#pone.0001210-Beck1">[33]</a> and yeast H2B (<a href="#pone-0001210-t002">Table 2</a> and <a href="#pone-0001210-g002">Fig. 2I</a>).</p>
<div class="figure" id="pone-0001210-g002"><div class="img"><a name="pone-0001210-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001210.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001210.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Identification of H2B modification sites.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A. MS/MS spectrum of the doubly-charged precursor ion at <em>m/z</em> 471.69 showing H2B acetylation site at K6 in the peptide AE<sup>6</sup>K<sub>ac</sub>KPAEK. B. MS/MS spectrum of the doubly-charged precursor ion at <em>m/z</em> 443.24 showing H2B acetylation site at K27 in the peptide AE<sup>27</sup>K<sub>ac</sub>APAEK. C. MS/MS spectrum of the doubly-charged precursor ion at <em>m/z</em> 550.88 for determining H2B acetylation site at K27 in the peptide SKAE<sup>27</sup>K<sub>ac</sub>APAEK. D. MS/MS spectrum of the triply-charged precursor ion at <em>m/z</em> 304.15 for determining H2B acetylation site at K32 in the peptide APAE<sup>32</sup>K<sub>ac</sub>KPK. E. MS/MS spectrum of the quadruply-charged precursor ion at <em>m/z</em> 456.01 for determining H2B.1 acetylation site at K11 in the peptide KPAE<sup>11</sup>K<sub>ac</sub>KPAAEKPVEEK. F. MS/MS spectrum of the quadruply-charged precursor ion at <em>m/z</em> 460.01 for determining H2B.2 acetylation site at K11 in the peptide KPAE<sup>11</sup>K<sub>ac</sub>KPASEKPVEEK. G. MS/MS spectrum of the quadruply-charged precursor ion at <em>m/z</em> 552.70 for determining H2B.2 acetylation sites at K6 and K11 in the peptide AE<sup>6</sup>K<sub>ac</sub>KPAE<sup>11</sup>K<sub>ac</sub>KPASEKPVEEK. H. MS/MS spectrum of the doubly-charged precursor ion at <em>m/z</em> 661.37 for determining H2B.2 phosphorylation site in the peptide KPA<sup>14</sup><sub>p</sub>SEKPVEEK. I. Sequence alignment of <em>Arabidopsis</em> H2B (H2B.1 and H2B.2). K in green color: acetylation; S in red color: phosphorylation.</p>
<span>doi:10.1371/journal.pone.0001210.g002</span></div><div class="figure" id="pone-0001210-t002"><div class="img"><a name="pone-0001210-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001210.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001210.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Comparison between plant H2B modifications and non-plant H2B modifications</span></strong></p><span>doi:10.1371/journal.pone.0001210.t002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Using MASCOT search we were able to determine the ubiquitin protein (Score 138 and 4 peptide sequences matched) in the HPLC fraction containing H2B protein, indicating that H2B was likely ubiquitinated. To confirm this, we loaded the H2B fraction on a SDS-PAGE gel and found in addition to the H2B band the presence of a band (approximate 27 kDa) larger in size than the major H2B band (~19 kDa). We further confirmed that this larger band corresponded to ubiquitinated form of H2B by Western-blot analysis with ubiquitin antibody <a href="#pone.0001210-Sridhar1">[34]</a>. Based on previous results that histone H2B is mono- ubiquitinated at lysine 123 on the C-terminal peptide with sequence AVT<sup>120</sup>KYTSS in human or AVT<sup>123</sup>KYSSS in yeast, we speculated that lysine 143 of plant H2B in the C-terminal sequence AVT<sup>143</sup>KFTSS may be ubiquitinated. Since trypsin cuts after arginine in the C-terminus of ubiquitin, it leaves two glycine residues of the ubiquitin, resulting in an increment of 114 dalton to the peptide mass. This increase in mass can be used for the detection of ubiquitination site <a href="#pone.0001210-Kaiser1">[35]</a>–<a href="#pone.0001210-Hatakeyama1">[36]</a>. We manually examined the MS/MS spectrum of the precursor ion at <em>m/z</em> 477.7 (doubly-charged and 57 Da added to the mass of the peptide AVTKFTSS) whose fragmentation ions were analyzed by Prospector_Product (prospector.ucsf.edu) . Indeed, the MS/MS fragmentation ions matched well the peptide sequence AVT<sub>GG</sub>KFTSS with GG chain attached to the lysine after trypsin digestion, confirming that H2B lysine 143 is ubiquitinated in plants <a href="#pone.0001210-Sridhar1">[34]</a>. It is of interest to note that the ubiquitinated H2B peptides in human, yeast and plant all had an aromatic amino acid residue, either tyrosine (Y) or phenylalanine (F), next to the ubiquitinated lysine residue (<a href="#pone-0001210-g002">Fig. 2I</a>).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Next, we searched for H2B phosphorylation sites. As illustrated for H2A phosphorylation site identification, LC/MS/MS analysis of the ‘treated’ sample by cation exchange (using Bio-Rex 70 resin) column chromatographic purification of histones and the phospho-peptide enrichment protocol identified phosphorylation at S15 in H2B.2. As shown in <a href="#pone-0001210-g002">Fig. 2H</a>, MS/MS fragmentation pattern of the doubly-charged precursor ion at <em>m/z</em> 661.37 matched the peptide sequence KPA<sup>15</sup><sub>p</sub>SEKPVEEK where S15 is phosphorylated. This assignment of phosphorylation was supported by the observation of the ion at <em>m/z</em> 1223.67 (converted from the doubly-charged ion at <em>m/z</em> 612.38) corresponding to the neutral loss of H<sub>3</sub>PO<sub>4 </sub>from the peptide ion at <em>m/z</em> 1321.62 (converted from the doubly-charged ion at <em>m/z</em> 661.37) and the observation of ‘y’ and ‘b’ satellite ions that are 98 (loss of H<sub>3</sub>PO<sub>4</sub>) dalton smaller than their corresponding y and b ions.</p>


<h4>Identification of modification sites of H3</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The HPLC fraction containing H3 was analyzed in two parts; one portion was digested by trypsin and analyzed by LC/MS/MS directly, and the other was digested by Arg_C and separated further into sub-fractions by HPLC before analysis by matrix assisted laser desorption/ionization-time of flight (MALDI-TOF) mass spectrometry. From LC/MS/MS runs of the trypsin digests, we examined thoroughly all the modification sites already known in other species, by both MASCOT search and manual analyses of the MS/MS raw data. As shown in <a href="#pone.0001210.s003">Figure S3A</a>, from the MS/MS spectrum of the precursor ion at <em>m/z</em> 472.32, the fragmentation ions (the majority is y ions) established the peptide sequence KSTGG<sup>14</sup>K<sub>ac</sub>APR in which acetylation at lysine 14 was assigned. This assignment is based on the fact that a functional group with 42 Da nominal mass was added to lysine 14 and that the fragmentation pattern was consistent with the one previously observed for the same lysine 14-acetylated peptide in other species <a href="#pone.0001210-Zhang1">[7]</a>. Similarly, <a href="#pone.0001210.s003">Figure S3B</a> illustrates the fragmentation of the precursor ion at <em>m/z</em> 493.3, which was consistent with peptide <sup>9</sup>K<sub>ac</sub>STGG<sup>14</sup>K<sub>ac</sub>APR where both lysine 9 and 14 are acetylated. We also observed four precursor ions containing modification information for lysine 18 and 23. Precursor ion at <em>m/z</em> 365.72 (doubly charged) corresponded to the peptide <sup>18</sup>K<sub>ac</sub>QLATK where lysine 18 is acetylated (<a href="#pone.0001210.s003">Figure S3C</a>); precursor ion at <em>m/z</em> 450.75 (doubly charged) corresponded to peptide QLAT<sup>23</sup>K<sub>ac</sub>AA R where lysine 23 is acetylated (<a href="#pone.0001210.s003">Figure S3D</a>); precursor ion at <em>m/z</em> 514.8 (doubly-charged) corresponded to peptide KQLAT<sup>23</sup>K<sub>ac</sub>AAR where lysine 23 is acetylated (<a href="#pone.0001210.s003">Figure S3E</a>); and precursor ion at m/z 535.8 corresponded to peptide <sup>18</sup>K<sub>ac</sub>QLAT<sup>23</sup>K<sub>ac</sub>AAR where both lysine 18 and 23 are acetylated (<a href="#pone.0001210.s003">Figure S3F</a>). We note that lysine 56, known to be acetylated in <em>Drosophila</em> and yeast <a href="#pone.0001210-Xu1">[10]</a>–<a href="#pone.0001210-Masumoto1">[11]</a>, was not found to be acetylated in our <em>Arabidopsis</em> samples.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We next examined all of the potential methylation sites of histone H3. Three groups of ions, at <em>m/z</em> 465.25, 472.26, 479.26 (<a href="#pone-0001210-g003">Fig. 3A</a>), at <em>m/z</em> 480.25, 487.26, 494.27 (<a href="#pone-0001210-g003">Fig. 3B</a>), and at <em>m/z</em> 675.34, 682.36, 689.36 (<a href="#pone-0001210-g003">Fig. 3E</a>) were detected to be the precursor ions of potentially methylated peptides since a mass difference of 7 units (doubly-charged) between two nearby peaks was observed. Interestingly, the first group of ions matched with the nominal mass of the peptide <sup>9</sup>KSTGG<sup>14</sup>KAPR where lysine 9 could be mono- to tri-methylated (K9_Di-methylation at <em>m/z</em> 465.3; K9_Tri-methylation at <em>m/z</em> 472.3; K9_Mono-methylaiton &amp; K14_Acetylation at <em>m/z</em> 479.3). The MS/MS spectra of those three ions could not establish the sequence of KSTGGKAPR, instead the spectra corresponded to the sequence K<sub>me1-3</sub>SAPATGGVK of H3 isoform H3.1 where lysine 27 is mono-, di- and tri-methylated with mono-methylation being the dominant form (<a href="#pone-0001210-g003">Figure 3A</a>). The production ion spectrum of the precursor ion at <em>m/z</em> 479.26 was shown here to illustrate lysine 27 tri-methylation (<a href="#pone-0001210-g003">Figure 3C</a>). The observation of MH<sup>+</sup>-59, a<sub>3</sub>-59 and b<sub>6</sub>-59 ions produced from the neutral loss of tri-methyl amine confirmed the presence of tri-methylation rather than acetylation at lysine 27.</p>
<div class="figure" id="pone-0001210-g003"><div class="img"><a name="pone-0001210-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001210.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001210.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Identification of H3 methylation sites.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A. ESI mass spectrum showing mono- (<em>m/z</em> 465.25), di- (<em>m/z</em> 472.26) and tri-methylation (<em>m/z</em> 479.26) at K27 of H3.1. B. ESI mass spectrum showing mono- (<em>m/z</em> 480.25), di- (<em>m/z</em> 487.26) and tri-methylation (<em>m/z</em> 494.27) at K27 of H3.2. C. MS/MS spectrum of the doubly-charged precursor ion at <em>m/z</em> 479.26 for determining tri-methylation at K27 of H3.1. D. MS/MS spectrum of the doubly-charged precursor ion at <em>m/z</em> 494.27 for determining tri-methylation at K27 of H3.2. E. ESI-MS spectrum showing mono- (<em>m/z</em> 675.34), di- (<em>m/z</em> 682.36) and tri-methylation (<em>m/z</em> 689.36) at K36 in H3.2 and tri-methylation (<em>m/z</em> 674.33) at K36 of H3.1. F. MS/MS spectrum of the doubly-charged precursor ion at <em>m/z</em> 689.36 for determining tri-methylation at K36 of H3.2. G. MALDI-TOF mass spectrum showing non- (<em>m/z</em> 704.4) mono- (<em>m/z</em> 718.4), di- (<em>m/z</em> 732.4) and tri-methylation (746.4) at K4. H. MALDI-TOF mass spectrum showing di-methylation (<em>m/z</em> 929.6) at K9, acetylation at K14 or tri-methylation (<em>m/z</em> 943.6) at K9, acetylation at K14 and mono-methylation (<em>m/z</em> 957.6) at K9. I. MALDI-TOF mass spectrum showing no methylation at K79. J. Sequence alignment of H3 isoforms (H3.1 and H3.2). K in green color: acetylation; K in yellow color: methylation; K in olive color: both acetylation and methylation. Note: All peptides in A–F were released from trypsin digestion and peptides in G–I were released from Arg-C digestion.</p>
<span>doi:10.1371/journal.pone.0001210.g003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Similarly, the second group of ions (at <em>m/z</em> 480.25, 487.26 and 494.27) were unambiguously assigned to the peptide <sup>27</sup>K<sub>me1-3</sub>SAPTTGGVK of H3 isoform H3.2 which has one amino acid residue different from H3.1, that is at position 31 with threonine replacing alanine. MS/MS fragmentations again supported the assignment of mono-, di- and tri-methylation at lysine 27. The fragmentation pattern of the tri-methylated peptide with the precursor ion at <em>m/z</em> 494.27 was shown in <a href="#pone-0001210-g003">Figure 3D</a>. The clearly observed MH<sup>+</sup>-59 ion supported the presence of tri-methylation at lysine 27 in the peptide sequence <sup>27</sup>K<sub>me3</sub>SAPTTGGVK. From the ESI-MS spectra of the third group ions, we concluded that lysine 36 in H3.2 is mono-, di- and tri-methylated, with tri-methylation being the most dominant form. The peak to the left of the peak at <em>m/z</em> 674.33 corresponded to the above stated lysine 36 mono-methylated peptide of H3.2 and matched the mono-isotopic mass of peptide SAPATGGV<sup>36</sup>K<sub>me3</sub>KPHR of H3.1, suggesting that lysine 36 of H3.1 is also tri-methylated. There was no detection of mono- and di-methylated ions (the mass region is not shown), possibly because they were too weak to be detected. Thus, our mass spectrometric data show that lysine 27 and lysine 36 in plants are mono-, di- and tri-methylated, and lysine 27 is prone to being more mono-methylated while lysine 36 is more tri-methylated. Our results demonstrate that K27 methylation and K36 methylation could each exist exclusively without the other as we have observed here or co-exist as previously reported <a href="#pone.0001210-Johnson2">[28]</a>.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>From the LC/MS/MS runs, we did not detect methylation at lysine 79. We could not obtain clear information on lysine 4 modification because the tryptic peptide containing K4 was short, hydrophilic and was eluted together with the solvent front (using formic acid as the ion-pairing agent). We also could not determine K9 methylation during the LC/MS/MS runs possibly because the majority of K9 was acetylated. So, we ran an HPLC using 0.1% Trifluoroacetic acid (TFA) as the ion-paring agent to separate the peptides from the Arg_C digest of H3. Arg_C protease prefers to cut arginine, leaving modified and un-modified lysine immune to cleavage. Each fraction of LC was collected and analyzed by MALDI-TOF mass spectrometry. In our LC system, H3 Arg_C digested peptides containing K4, K9 and K79 had relative retention time of 10, 15 and 35 minutes, respectively. We found that lysine 4 is largely mono-methylated, with detectable but significantly lower amounts of di- and tri-methylation (<a href="#pone-0001210-g003">Figure 3G</a>). For lysine 9, we found di-methylation (<a href="#pone-0001210-g003">Figure 3H</a>) based on the fact that the ion at <em>m/z</em> 929.60 is eluted in an earlier fraction than those lysine 27-containing peptides <a href="#pone.0001210-Zhang1">[7]</a>. Again, for lysine 79 we did not detect any methylation by MALDI. A very strong single peak of un-methylated peptide with mono-isotopic mass at <em>m/z</em> 1335.7 was detected while the peaks corresponding to the mono-methylated form (at <em>m/z</em> 1349.7), di-methylated form (at <em>m/z</em> 1363.7) and tri-methylated form (at <em>m/z</em> 1377.7) were missing (<a href="#pone-0001210-g003">Figure 3I</a>). A peak observed at <em>m/z</em> 1378.8 corresponding to a different peptide was labeled to distinguish it from the tri-methylated K79 peak. <a href="#pone-0001210-t003">Table 3</a> and <a href="#pone-0001210-g003">Fig. 3J</a> summarize the H3 modification sites in human, yeast and <em>Arabidopsis</em>.</p>
<div class="figure" id="pone-0001210-t003"><div class="img"><a name="pone-0001210-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001210.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001210.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Comparison between plant H3 modifications and non-plant H3 modifications</span></strong></p><span>doi:10.1371/journal.pone.0001210.t003</span></div>

<h4>Identification of modification sites of H4</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>From a single LC/MS/MS run of the tryptic digests of histone H4, we obtained a total of seven acetylated peptides. As shown in <a href="#pone.0001210.s004">Fig. S4A–F</a>, the MS/MS spectra indicate that lysine 16 is acetylated in the peptide GGA<sup>16</sup>K<sub>ac</sub>R with isotopic mass at <em>m/z</em> 265.67 (doubly-charged) (<a href="#pone.0001210.s004">Fig. S4A</a>); lysine 12 is acetylated in the peptide GLG<sup>12</sup>K<sub>ac</sub>GGAK with isotopic mass at <em>m/z</em> 365.2 (deconvoluted singly-charged ion at <em>m/z</em> 729.44) (<a href="#pone.0001210.s004">Fig. S4B</a>); lysine 8 is acetylated in the peptide GG<sup>8</sup>K<sub>ac</sub>GLG with mono-isotopic mass at <em>m/z</em> 329.7 (deconvoluted singly-charged ion at <em>m/z</em> 658.40) (<a href="#pone.0001210.s004">Fig. S4C</a>); two lysines K5 and K8 are acetylated in the peptide G<sup>5</sup>K<sub>ac</sub>GG<sup>8</sup>K<sub>ac</sub>GLGK with mono-isotopic mass at <em>m/z</em> 443.2 (deconvoluted singly-charged ion at <em>m/z</em> 885.55) (<a href="#pone.0001210.s004">Fig. S4D</a>); two lysines K12 and K16 are acetylated in the peptide GLG<sup>12</sup>K<sub>ac</sub>GGA<sup>16</sup>K<sub>ac</sub>R with mono-isotopic mass at <em>m/z</em> 464.3 (deconvoluted singly-charged ion at <em>m/z</em> 927.56) (<a href="#pone.0001210.s004">Fig. S4E</a>); and three lysines K8, K12, K16 are acetylated in the peptide GG<sup>8</sup>K<sub>ac</sub>GLG<sup>12</sup>K<sub>ac</sub>GGA<sup>16</sup>K<sub>ac</sub>R with mono-isotopic mass at <em>m/z</em> 606.33 (<a href="#pone.0001210.s004">Fig. S4F</a>). The seventh potentially acetylated peptide with mono-isotopic mass at <em>m/z</em> 279.2 (doubly-charged) was established by the MS/MS fragmentation pattern as KILR where the lysine residue is modified with a group, either acetyl or tri-methyl, whose unit mass is 42 dalton (<a href="#pone.0001210.s005">Fig. S5</a>). For the ion at m/z 126.1, a signature for acetylated lysines, was observed in the MS/MS spectrum (<a href="#pone.0001210.s005">Fig. S5D</a>) indicating the presence of acetylation and not tri-methylation for the modified lysine <a href="#pone.0001210-Zhang1">[7]</a>, <a href="#pone.0001210-Zhang2">[12]</a>. HPLC was also run to isolate modified peptides from the trypsin digest of H4 and the peptides were subsequently analyzed by MALDI-TOF (<a href="#pone.0001210.s005">Fig. S5A–C</a>). We note that the modified peptide was eluted after the un-modified peptide (see peak 7 &amp; 8 in <a href="#pone.0001210.s005">Fig. S5C</a>), demonstrating that the former was more hydrophobic than the latter. Except for the two ions related to the un-modified and modified KILR peptides whose masses differed by 42 Da, we did not observe mono- (addition of 14 Da) or di- (addition of 28 Da) methylated peptides. Taken together, we conclude that the lysine residue in KILR was acetylated. Histone H4 contains a tryptic peptide <sup>20</sup>KILR and lysine 20 is therefore the potential acetylation site.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Because the peptide K<sub>ac</sub>VLR was short, we could not completely rule out the possibility that other proteins co-eluted with H4 from HPLC separation had the same tryptic peptide. We therefore performed an Arg_C digestion of H4 for the purpose of obtaining a peptide with an extended sequence of KILR by controlling the digestion efficacy. Indeed, we were able to isolate two peptides, the un-modified peptide <sup>20</sup>KVLRDNIQGITKPAIR and the modified counterpart which eluted at around 40 minute from the HPLC column using TFA as the ion-paring agent. Mono-isotopic masses of the two peaks differed by 42.067 using MALDI-TOF mass spectrometric measurement (<a href="#pone-0001210-g004">Fig. 4A</a>). Using the mass of the un-modified peptide (known peptide sequence and confirmed by ESI-MS/MS) to calibrate the MALDI spectrum, we found that the modified peptide gave a mono-isotopic mass 1864.1084 with only 2.0 ppm error for the assignment of acetylation as compared to 22.1 ppm for the assignment of tri-methylation (<a href="#pone-0001210-g004">Fig. 4A</a>). Furthermore, a MALDI-TOF collision induced dissociation (CID) experiment was done towards the modified peptide with mono-isotopic mass at m/z 1864.1. The fragmentation ions, namely “a”, “b” and “y” ions, established the peptide sequence <sup>20</sup>K<sub>ac</sub>VLRDNIQGITKPAIR where lysine 20 was acetylated while lysine 31 was not (<a href="#pone-0001210-g004">Fig. 4B</a>). The observation of signature immonium ions at m/z 126.1 and 143.1 for acetylated lysine gave additional support for acetylation. The pool of peptides was re-injected on the LC column and LC/MS/MS analysis was run using formic acid as the ion-pairing agent. Besides obtaining sequence information of the two peptides with un-acetylated and acetylated lysine 20 (data not shown), we also noticed that the modified peptide was eluted about 4 minutes after the un-modified one (<a href="#pone-0001210-g004">Fig. 4C &amp; 4D</a>), which is in agreement with the notion of acetylation increasing the hydrophobicity of a molecule <a href="#pone.0001210-Thorne1">[37]</a>. Collectively these evidences support that lysine 20 of plant histone H4 is acetylated. <a href="#pone-0001210-t004">Table 4</a> and <a href="#pone-0001210-g004">Fig. 4E</a> summarize the H4 modification sites in human, yeast and <em>Arabidopsis</em>.</p>
<div class="figure" id="pone-0001210-g004"><div class="img"><a name="pone-0001210-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001210.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001210.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Identification of H4 acetylation sites at K20.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>A. MALDI-TOF mass spectrum of HPLC fraction of Arg_C digest containing K20 peptide. B. MALD-MS/MS spectrum of the precursor ion at <em>m/z</em> 1864.14 for determining acetylation at K20. C. Chromatogram of the ion at <em>m/z</em> 1822.10 (doubly-charged ion at <em>m/z</em> 911.55). D. Chromatogram of the ion at <em>m/z</em> 1864.11 (doubly-charged ion at <em>m/z</em> 932.55). E. Sequence alignment of <em>Arabidopsis</em> H4. K in green color: acetylation.</p>
<span>doi:10.1371/journal.pone.0001210.g004</span></div><div class="figure" id="pone-0001210-t004"><div class="img"><a name="pone-0001210-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001210" data-uri="info:doi/10.1371/journal.pone.0001210.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001210.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001210.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001210.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001210.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001210.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Comparison between plant H4 modifications and non-plant H4 modifications</span></strong></p><span>doi:10.1371/journal.pone.0001210.t004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We used a combination of various mass spectrometric methods including LC/MS/MS, MALDI-TOF and MALDI-CID as well as HPLC purification to identify histone modification sites in <em>Arabidopsis thaliana.</em> We also successfully implemented a phospho-peptide enrichment method to identify histone phosphorylation sites. Our results show that plant histones are extensively acetylated and/or methylated at most lysine sites that are also acetylated and/or methylated in mammals and yeast. Histone arginine methylation was not detected, possibly because of its low abundance or because it blocks cleavage by trypsin. The following modifications were found: acetylation at K5 and K144 of histone H2A; acetylation at K6, K11, K27 and K32 of histone H2B; acetylation at K9, K14, K18, K23 of histone H3; acetylation at K5, K8, K12, K16 and K20 of histone H4; methylation at K4, K9, K27 and K36 of H3; phosphorylation at S145 of H2A.5, at S129, S141 and S145 of H2A.7, at S138 of H2AX, and at S15 of H2B; ubiquitination at K143 of H2B. However, several modifications conserved in non-plant systems were missing or different in plants. For example, we found no methylation at lysine 79 of H3 and no methylation but acetylation at lysine 20 of H4.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Four <em>Arabidopsis</em> H2A isoforms H2A1-H2A.4 are acetylated at K5, a conserved H2A acetylation site in human and other species, while other three <em>Arabidopsis</em> H2A isoforms H2A.5-H2A.7 are acetylated at the C-terminal K144 and phosphorylated at S145. To our knowledge, these two modifications (acetylation at K144 and phosphorylation at S145) have not been reported previously to co-exist. There were no indications of modifications on H2A.8 whose N-terminal and C-terminal sequences are dissimilar to other H2A isoforms. The diversity of modification pattern in <em>Arabidopsis</em> H2A and the exclusive acetylation/phosphorylation preference of isoforms either on the N-terminus or on the C-terminus add additional complexity to the unique modification patterns of <em>Arabidopsis</em>. Moreover, we report for the first time the identification by mass spectrometry the DNA damage marker-H2A.X phosphorylation site, S138 of <em>Arabidopsis</em> H2A.X. The phosphorylation of an equivalent residue in human H2A.X, S139, has been well studied <a href="#pone.0001210-Bassing1">[38]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>H2B was determined to be acetylated at four sites (K6, K11, K27 and K32) on the N-terminus. We observed that the yeast acetylated peptide consensus AEK<sub>ac</sub>K (A) also applies to plants, implicating the existence of plant homologs of the yeast H2B acetyltransferase(s). We also noticed that lysine before proline could hardly be acetylated (i. e. K7, K12, K17, K33), in contrast to the fact that lysine after proline can be acetylated as we and others observed acetylation at K11 (after P10) of human H2B. The preference of acetylation at lysine after proline rather than acetylation at lysine before proline most likely arises from the conformational preference or hindrance of proline to an acetyltransferase <a href="#pone.0001210-Nelson1">[39]</a>. S15 was identified as the phosphorylation site of <em>Arabidopsis</em> H2B. S15 of <em>Arabidopsis</em> H2B is equivalent to S10 of yeast H2B because they are within similar peptide sequence contexts AEKKPASE (E is replaced by K in yeast). While S10 phosphorylation of yeast H2B has been implicated in apoptosis regulation <a href="#pone.0001210-Ahn1">[40]</a>, the function of S15 phosphorylation of <em>Arabidopsis</em> H2B remains to be determined. We also found that plant H2B was ubiquitinated at lysine 143, equivalent to lysine 123 in yeast H2B and lysine 120 in human H2B. H2B ubiquitination in <em>Arabidopsis</em> has also been detected using immunoblot analysis in several recent studies <a href="#pone.0001210-Sridhar1">[34]</a>, <a href="#pone.0001210-Fleury1">[41]</a>, <a href="#pone.0001210-Liu1">[42]</a>. It is interesting to note that human, yeast and plant H2B share the same peptide sequence AVTKY (F) with the feature that either Y or F is located next to the ubiquitinated lysine residue. The aromatic amino acids Y and F were thought to form base stacking interactions with DNA <a href="#pone.0001210-Bochkrev1">[43]</a>. In this context, ubiquitination of the lysine next to the aromatic amino acids might disrupt this interaction, suggesting that H2B ubiquitination might be involved in disrupting DNA binding <a href="#pone.0001210-Sridhar1">[34]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>H3 K79 is mono-, di- and tri-methylated in many mammalian and non-mammalian cell lines and in yeast studied to date. The DOT1 enzyme, an H3 K79 methyltransferase, is responsible for methylation at this site, which has been suggested to play a role in telomeric silencing <a href="#pone.0001210-Ng1">[9]</a>. Interestingly, in our study, H3 lysine 79 in <em>Arabidopsis</em> was found free of modification/methylation. Furthermore, we could not find a DOT1 homolog in <em>Arabidopsis</em> by BLAST search using human or yeast DOT1 as the query sequence. It would be of great interest to explore the functional implication of the lack of methylation at lysine 79 in plants. In our study, we also found that K27 from both H3.1 and H3.2 were mono-, di, or tri-methylated, with the dominance of mono-methylation and in the absence of K36 methylation. Likewise, K36 from H3.2 were mono-, di-, tri-methylated with the dominance of tri-methylation and in the absence of K27 methylation, although methylation of K27 and methylation of K36 could coexist in plants <a href="#pone.0001210-Johnson2">[28]</a> and in chicken erythrocytes <a href="#pone.0001210-Zhang1">[7]</a>. Interestingly, the study of Johnson et al <a href="#pone.0001210-Johnson2">[28]</a> showed that dimethylated K36 is higher than mono- or trimethylated forms for H3.2. This discrepancy with our result could be caused by the different plant tissues and ecotypes used. No mono- or di-methylation at K36 was detected on H3.1, possibly because the signals were too low compared with tri-methylation which was the dominant form observed for H3.2.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Lysine 20 of histone H4 was reported to be modified in the form of mono-, di- and tri-methylation in almost all multicellular organisms <a href="#pone.0001210-Fang1">[44]</a>, whereas lysine 20 of budding yeast has not been reported to be modified (acetylation or methylation). To date, at least five methyl transferases have been identified that specifically methylate lysine 20: SET7, SET8, SET9 and Suv4-20h1 and Suv4-20h2, with potential roles ranging from cell cycle regulation, development, gene silencing associated with pericentrimeric heterochromatin to the coordination of transportation of Crb2 to the sites of DNA damage <a href="#pone.0001210-Fang1">[44]</a>–<a href="#pone.0001210-Schotta1">[47]</a>. However, our results show that lysine 20 of H4 in <em>Arabidopsis</em> is free of methylation. Interestingly, our mass spectrometry data revealed the presence of acetylation at lysine 20 of plant H4, supporting the prediction made by Waterborg <a href="#pone.0001210-Waterborg1">[48]</a> more than a decade ago. Waterborg predicted acetylation at this site in alfalfa using a radioactive chemical acetylation method to protect the five un-modified lysine residues of the N-terminal 23 amino acids of H4 and counted the radioactivity by steps of hydrolysis <a href="#pone.0001210-Waterborg2">[49]</a>. In contrast to the tri-methylation of H4 lysine 20, which is thought to be associated with repressive chromatin, we speculate that acetylation at lysine 20 alone or together with the nearby lysine 16 acetylation may play a role in activating transcription.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In summary, we have used mass spectrometry to identify plant histone modification sites. Although it is still incomplete, the identification of modification sites here sets a a valuable foundation for further studies aimed at a comprehensive identification of histone modifications and their role in plant growth and development.</p>

<h4>Experimental protocol</h4>
<h5>Isolation of core histones from <em>Arabidopsis thaliana</em>.</h5><a id="article1.body1.sec3.sec1.sec1.p1" name="article1.body1.sec3.sec1.sec1.p1"></a><p><em>Arabidopsis</em> plants were grown in soil in a growth chamber for 4 weeks at 22°C with 16 hours of light and 8 hours of dark each day. Around 30 grams of above ground plant tissues were used for histone purification according to the protocol of Waterborg et al <a href="#pone.0001210-Waterborg2">[49]</a>. Briefly, nuclei were extracted in isolation buffer [0.25M sucrose, 25 mM HEPES (pH 7.5), 3 mM CaCl<sub>2</sub>, 10 mM NaCl, 1 mM PMSF,1 mM DTT, 0.25% Nonidet 40, leupeptin, and pepstatin A], and deacetylase and phosphatase inhibitors (1 mM sodium butyrate, 1 mM NaF, 1 mM sodium orthovandate and 10 µM calyculin) were added. The nuclei were pelleted by centrifugation at 3000 rpm for 10 min and then washed twice with the isolation buffer. Nuclei were suspended in 5%GuHCl (guanidine hydrochloride in 100mM potassium phosphate buffer (pH 6.8) and passed through Bio-Rex 70 (BIORAD) resin. The bound histones were eluted with 20% GuHCl and dialyzed with water. (<a href="#pone.0001210.s001">Fig S1B</a>)</p>

<h5>HPLC separation of histones.</h5><a id="article1.body1.sec3.sec1.sec2.p1" name="article1.body1.sec3.sec1.sec2.p1"></a><p>Core histones were separated into sub-histones in the order of H2B, H4, H2A and H3 by reverse-phase HPLC as described previously <a href="#pone.0001210-Kurdistani1">[8]</a>. Briefly, a 90-minute of gradient from 38% mobile phase B (0.1% TFA in acetonitrile), 62% mobile phase A (0.1% TFA in water) to 90% B through the time of 55 minutes with 55% mobile phase B, was run on an HP 1100 capillary HPLC instrument (Hewlett-Packard, Palo Alto) using a 150×2.0 mm Phenomenex C4 column running in the normal pump mode at a flow rate of 50 µl/min. (<a href="#pone.0001210.s001">Fig S1A</a>)</p>

<h5>HPLC purification of peptide from H3 Arg_C digests.</h5><a id="article1.body1.sec3.sec1.sec3.p1" name="article1.body1.sec3.sec1.sec3.p1"></a><p>Isolated histone H3 was digested by Arg_C in 25 mM ammonium biocarbonate overnight. The digests were SpeedVac dried, re-dissolved in 0.1% TFA and then purified by reverse-phase HPLC using an Agilent 150×0.5 mm (5 µm) Zorbax C18 column performed on the same HP 1100 capillary HPLC running in the micro pump mode. A 100 minute gradient was run from 2% mobile phase B (0.1% TFA in acetonitrile) and 98% mobile phase A (0.1% TFA in water) to 90% mobile phase B through the time of 65 minutes with 65% mobile phase B at a flow-rate of 6 µl/min. A diode array detector (DAD) was used to record the chromatogram and each fraction was manually collected in a 0.5 mL silicanized Eppentorf tube and then dried. The same gradient and column was used for HPLC linked to the mass spectrometry except 0.1% formic acid rather than TFA was used in mobile phases A and B.</p>

<h5>Phospho-peptide enrichment.</h5><a id="article1.body1.sec3.sec1.sec4.p1" name="article1.body1.sec3.sec1.sec4.p1"></a><p>Phospho-peptide enrichment was processed as described previously <a href="#pone.0001210-Olsen1">[50]</a>–<a href="#pone.0001210-Zhang4">[51]</a> with minor modifications. Briefly, a mixture of equal amount of TiO<sub>2</sub> and anion-exchange resin (LC-NH2, Supelco) was placed on the top of C18 ZipTip pipette tips (Millipore) followed by several steps of washing and elution.</p>

<h5>Electrospray mass spectrometry.</h5><a id="article1.body1.sec3.sec1.sec5.p1" name="article1.body1.sec3.sec1.sec5.p1"></a><p>Electrospray mass spectrometry was performed on a Waters hybrid quadrupole-time of flight (Q-TOF) mass spectrometer (Waters, Manchester, UK). The Q-TOF was run in a survey mode as described previously <a href="#pone.0001210-Zhang3">[13]</a>. The raw data were converted into peak-list (PKL) files that were processed by MASCOT (<a href="http://www.matrixscience.com">www.matrixscience.com</a>) for searching proteins and protein posttranslational modifications sites. <em>De-novo</em> sequencing, through the aide of Prospector Product software (prospector.ucsf.edu), was also performed for either confirmation of modification sites reported by MASCOT or to find other modification sites which MASCOT might fail to determine.</p>

<h5>MALDI-TOF mass spectrometry.</h5><a id="article1.body1.sec3.sec1.sec6.p1" name="article1.body1.sec3.sec1.sec6.p1"></a><p>MALDI-TOF measurement of peptides from protein digests was performed on the Voyager DE-STR Biospectrometry Workstation (ABI, Foster City, CA) with delayed extraction operated in the reflectron mode using α-cyano-4-hydroxycinnamic acid as the matrix. A MALDI MS/MS experiment was carried out on the QSTAR instrument (ABI, Foster City, CA) equipped with a MALDI source.</p>


</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001210"></div><a name="pone.0001210.s001" id="pone.0001210.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001210.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Separation of Arabidopsis histones. A. HPLC chromatogram of Arabidopsis histones. B. SDS-PAGE of total histones and HPLC fractions.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001210.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.15 MB PDF)</p>
<a name="pone.0001210.s002" id="pone.0001210.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001210.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Identification of H2A (in isoforms H2A.1-H2A.4) acetylation site at K5. A. MS/MS spectrum of the doubly-charged precursor ion at m/z 479.8. B. MS/MS spectrum of the doubly-charged precursor ion at m/z 480.3. C. MS/MS spectrum of the doubly-charged precursor ion at m/z 474.3. D. MS/MS spectrum of the doubly-charged precursor ion at m/z 459.3.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001210.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.78 MB EPS)</p>
<a name="pone.0001210.s003" id="pone.0001210.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001210.s003">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Identification of H3 acetylation sites at K9, K14, K18 and K23. A. MS/MS spectrum of the doubly-charged precursor ion at m/z 472.27 for determining acetylation site at K14. B. MS/MS spectrum of the doubly-charged precursor ion at m/z 493.27 for determining acetylation sites at K9 and K14. C. MS/MS spectrum of the doubly-charged precursor ion at m/z 450.75 for determining acetylation site at K23. D. MS/MS spectrum of the doubly-charged precursor ion at m/z 530.8 for determining acetylation sites at K18 and K23. E. MS/MS spectrum of the doubly-charged precursor ion at m/z 465.7 for determining acetylation site at K18. F. MS/MS spectrum of the doubly-charged precursor ion at m/z 514.8 for determining acetylation site at K23.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001210.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.99 MB EPS)</p>
<a name="pone.0001210.s004" id="pone.0001210.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001210.s004">Figure S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Identification of H4 acetylation sites at K5, K8, K12 and K16. A. MS/MS spectrum of the doubly-charged precursor ion at m/z 265.67 for determining acetylation site at K16. B. MS/MS spectrum of the doubly-charged precursor ion at m/z 365.22 for determining acetylation site at K12. C. MS/MS spectrum of the doubly-charged precursor ion at m/z 329.70 for determining acetylation site at K8. D. MS/MS spectrum of the doubly-charged precursor ion at m/z 443.28 for determining acetylation sites at K5 and K8. E. MS/MS spectrum of the doubly-charged precursor ion at m/z 464.28 for determining acetylation sites at K12 and K16. F. MS/MS spectrum of the doubly-charged precursor ion at m/z 606.33 for determining acetylation sites at K8, K12 and K16.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001210.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(1.03 MB EPS)</p>
<a name="pone.0001210.s005" id="pone.0001210.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001210.s005">Figure S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Identification of acetylation at K20 of H4. A. MALDI-TOF mass spectrum of HPLC fraction containing un-modified K20 peptide 20KVLR. B. MALDI-TOF mass spectrum of HPLC fraction containing acetylated K20 peptide 20KacVLR. C. HPLC chromatogram of the Arg_C digested peptides from mixture of H3 and H4. D. MS/MS spectrum of the precursor ion at m/z 279.2 (doubly-charged).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001210.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(1.32 MB EPS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>K. Z thanks for the support from the Department of Biochemistry, School of Medicine, Loma Linda University.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JZ. Performed the experiments: KZ VS JZ AK. Analyzed the data: JZ KZ VS AK. Contributed reagents/materials/analysis tools: KZ. Wrote the paper: JZ KZ VS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001210-Luger1" id="pone.0001210-Luger1"></a>Luger K, Mäder AW, Richmond RK, Sargent DF, Richmond TJ (1997) Crystal structure of the nucleosome core particle at 2.8 Å resolution. Nature  389: 251–260.  <ul class="find" data-citedArticleID="1020682" data-doi="10.1038/38444"><li><a href="http://dx.doi.org/10.1038/38444" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+nucleosome+core+particle+at+2.8+%C3%85+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+nucleosome+core+particle+at+2.8+%C3%85+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001210-Tremethick1" id="pone.0001210-Tremethick1"></a>Tremethick DJ (2007) Higher-order structures of chromatin: the elusive 30 nm fiber. Cell  128: 651–654.  <ul class="find" data-citedArticleID="1020718" data-doi="10.1016/j.cell.2007.02.008"><li><a href="http://dx.doi.org/10.1016/j.cell.2007.02.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Higher-order+structures+of+chromatin%3A+the+elusive+30+nm+fiber." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Higher-order+structures+of+chromatin%3A+the+elusive+30+nm+fiber.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001210-Gilbert1" id="pone.0001210-Gilbert1"></a>Gilbert N, Boyle S, Fiehler H, Woodfine K, Carter NP, et al.  (2004) Chromatin architecture of the human genome: gene-rich domains are enriched in open chromatin fiber. Mol. Cell  15: 844–846.  <ul class="find" data-citedArticleID="1020658"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chromatin+architecture+of+the+human+genome%3A+gene-rich+domains+are+enriched+in+open+chromatin+fiber.+Mol.&amp;auth=&amp;atitle=Chromatin+architecture+of+the+human+genome%3A+gene-rich+domains+are+enriched+in+open+chromatin+fiber.+Mol." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+architecture+of+the+human+genome%3A+gene-rich+domains+are+enriched+in+open+chromatin+fiber.+Mol." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+architecture+of+the+human+genome%3A+gene-rich+domains+are+enriched+in+open+chromatin+fiber.+Mol.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001210-Jiang1" id="pone.0001210-Jiang1"></a>Jiang G, Yang F, Sanchez C, Ehrlich M (2004) Histone modification in constitutive heterochromatin versus unexpressed euchromatin in human cells. J. Cell Biochem  93: 286–300.  <ul class="find" data-citedArticleID="1020668" data-doi="10.1002/jcb.20146"><li><a href="http://dx.doi.org/10.1002/jcb.20146" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+modification+in+constitutive+heterochromatin+versus+unexpressed+euchromatin+in+human+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+modification+in+constitutive+heterochromatin+versus+unexpressed+euchromatin+in+human+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001210-Jenuwein1" id="pone.0001210-Jenuwein1"></a>Jenuwein T, Allis CD (2001) Translating the histone code. Science  293: 1074–1080.  <ul class="find" data-citedArticleID="1020666" data-doi="10.1126/science.1063127"><li><a href="http://dx.doi.org/10.1126/science.1063127" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translating+the+histone+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translating+the+histone+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001210-Marvin1" id="pone.0001210-Marvin1"></a>Marvin KW, Yau P, Bradbury EM (1990) Isolation and characterization of acetylated histones H3 and H4 and their assembly into nucleosomes. J. Biol. Chem.  265: 19839–19847.  <ul class="find" data-citedArticleID="1020684"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Isolation+and+characterization+of+acetylated+histones+H3+and+H4+and+their+assembly+into+nucleosomes.&amp;auth=&amp;atitle=Isolation+and+characterization+of+acetylated+histones+H3+and+H4+and+their+assembly+into+nucleosomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+characterization+of+acetylated+histones+H3+and+H4+and+their+assembly+into+nucleosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+characterization+of+acetylated+histones+H3+and+H4+and+their+assembly+into+nucleosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001210-Zhang1" id="pone.0001210-Zhang1"></a>Zhang K, Tang H, Huang L, Blankenship JW, Jones PR, et al.  (2002) Identification of acetylation and methylation sites of histone H3 from chicken erythrocytes by high-accuracy matrix-assisted laser desorption ionization-time-of flight, matrix-assisted laser desorption ionization-postsource decay, and nanoelectrospray ionization tandem mass spectrometry. Anal. Biochem.  306: 259–269.  <ul class="find" data-citedArticleID="1020730" data-doi="10.1006/abio.2002.5719"><li><a href="http://dx.doi.org/10.1006/abio.2002.5719" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+acetylation+and+methylation+sites+of+histone+H3+from+chicken+erythrocytes+by+high-accuracy+matrix-assisted+laser+desorption+ionization-time-of+flight%2C+matrix-assisted+laser+desorption+ionization-postsource+decay%2C+and+nanoelectrospray+ionization+tandem+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+acetylation+and+methylation+sites+of+histone+H3+from+chicken+erythrocytes+by+high-accuracy+matrix-assisted+laser+desorption+ionization-time-of+flight%2C+matrix-assisted+laser+desorption+ionization-postsource+decay%2C+and+nanoelectrospray+ionization+tandem+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001210-Kurdistani1" id="pone.0001210-Kurdistani1"></a>Kurdistani SK, Tavazole S, Grunstein M (2004) Mapping global histone acetylation patterns to gene expression. Cell  117: 721–733.  <ul class="find" data-citedArticleID="1020676" data-doi="10.1016/j.cell.2004.05.023"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.05.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+global+histone+acetylation+patterns+to+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+global+histone+acetylation+patterns+to+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001210-Ng1" id="pone.0001210-Ng1"></a>Ng HH, Feng Q, Wang H, Erdjument-Bromage H, Borchers C, et al.  (2001) Lysine methylation within the globular domain of histone H3 by Dot1 is important for telomeric silencing and Sir protein association. Genes &amp; Dev.  16: 1518–1527.  <ul class="find" data-citedArticleID="1020696" data-doi="10.1101/gad.1001502"><li><a href="http://dx.doi.org/10.1101/gad.1001502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lysine+methylation+within+the+globular+domain+of+histone+H3+by+Dot1+is+important+for+telomeric+silencing+and+Sir+protein+association." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lysine+methylation+within+the+globular+domain+of+histone+H3+by+Dot1+is+important+for+telomeric+silencing+and+Sir+protein+association.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001210-Xu1" id="pone.0001210-Xu1"></a>Xu F, Zhang K, Grunstein M (2005) Acetylation in histone globular domain regulates gene expression in yeast. Cell  121: 375–385.  <ul class="find" data-citedArticleID="1020728" data-doi="10.1016/j.cell.2005.03.011"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.03.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acetylation+in+histone+globular+domain+regulates+gene+expression+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acetylation+in+histone+globular+domain+regulates+gene+expression+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001210-Masumoto1" id="pone.0001210-Masumoto1"></a>Masumoto H, Hawke D, Kobayashi R, Verreault A (2005) A role for cell-cycle-regulated histone H3 lysine 56 acetylation in the DNA damage response. Nature  436: 294–298.  <ul class="find" data-citedArticleID="1020686" data-doi="10.1038/nature03714"><li><a href="http://dx.doi.org/10.1038/nature03714" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+role+for+cell-cycle-regulated+histone+H3+lysine+56+acetylation+in+the+DNA+damage+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+role+for+cell-cycle-regulated+histone+H3+lysine+56+acetylation+in+the+DNA+damage+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001210-Zhang2" id="pone.0001210-Zhang2"></a>Zhang K, Yau PM, New R, Kondrat R, Imai BS, et al.  (2004) Differentiation of acetylated peptides from methylated peptides by mass spectrometry: An application for determining lysine 9 acetylation and methylation of histone H3. Proteomics  4: 1–10.  <ul class="find" data-citedArticleID="1020732" data-doi="10.1002/pmic.200300503"><li><a href="http://dx.doi.org/10.1002/pmic.200300503" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differentiation+of+acetylated+peptides+from+methylated+peptides+by+mass+spectrometry%3A+An+application+for+determining+lysine+9+acetylation+and+methylation+of+histone+H3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differentiation+of+acetylated+peptides+from+methylated+peptides+by+mass+spectrometry%3A+An+application+for+determining+lysine+9+acetylation+and+methylation+of+histone+H3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001210-Zhang3" id="pone.0001210-Zhang3"></a>Zhang K, Siino JS, Jones PR, Yau PM, Bradbury EM (2004) A mass spectrometric “western blot” to evaluate the correlations between histone acetylation and methylation. Proteomics  4: 3765–3775.  <ul class="find" data-citedArticleID="1020734" data-doi="10.1002/pmic.200400819"><li><a href="http://dx.doi.org/10.1002/pmic.200400819" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mass+spectrometric+%E2%80%9Cwestern+blot%E2%80%9D+to+evaluate+the+correlations+between+histone+acetylation+and+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mass+spectrometric+%E2%80%9Cwestern+blot%E2%80%9D+to+evaluate+the+correlations+between+histone+acetylation+and+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001210-Meluh1" id="pone.0001210-Meluh1"></a>Meluh P, Broach JR (1999) Immunological analysis of yeast chromatin. Methods Enzymol.  304: 414–430.  <ul class="find" data-citedArticleID="1020692" data-doi="10.1016/s0076-6879(99)04025-2"><li><a href="http://dx.doi.org/10.1016/s0076-6879(99)04025-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunological+analysis+of+yeast+chromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunological+analysis+of+yeast+chromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001210-Cheung1" id="pone.0001210-Cheung1"></a>Cheung P, Allis CD, Corsi PS (2000) Signaling to chromatin through histone modifications. Cell  103: 263–271.  <ul class="find" data-citedArticleID="1020650" data-doi="10.1016/s0092-8674(00)00118-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)00118-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signaling+to+chromatin+through+histone+modifications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signaling+to+chromatin+through+histone+modifications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001210-Bastow1" id="pone.0001210-Bastow1"></a>Bastow R, Mylne JS, Lister C, Lippman Z, Martienssen RA, et al.  (2004) Vernalization requires epigenetic silencing of FLC by histone methylation. Nature  427: 164–167.  <ul class="find" data-citedArticleID="1020642" data-doi="10.1038/nature02269"><li><a href="http://dx.doi.org/10.1038/nature02269" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vernalization+requires+epigenetic+silencing+of+FLC+by+histone+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vernalization+requires+epigenetic+silencing+of+FLC+by+histone+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001210-Sung1" id="pone.0001210-Sung1"></a>Sung S, Amasin RM (2004) Vernalization in Arabidopsis thaliana is mediated by the PHD finger protein VIN3. Nature  427: 159–164.  <ul class="find" data-citedArticleID="1020712" data-doi="10.1038/nature02195"><li><a href="http://dx.doi.org/10.1038/nature02195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vernalization+in+Arabidopsis+thaliana+is+mediated+by+the+PHD+finger+protein+VIN3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vernalization+in+Arabidopsis+thaliana+is+mediated+by+the+PHD+finger+protein+VIN3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001210-Wood1" id="pone.0001210-Wood1"></a>Wood CC, Robertson M, Tanner G, Peacock WJ, Dennis ES, et al.  (2006) The <em>Arabidopsis thaliana</em> vernalization response requires a polycomb-like protein complex that also includes VERNALIZATION INSENSITIVE 3. Proc. Natl. Acad. Sci. U S A.  103: 14631–14636.  <ul class="find" data-citedArticleID="1020726" data-doi="10.1073/pnas.0606385103"><li><a href="http://dx.doi.org/10.1073/pnas.0606385103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+thaliana+vernalization+response+requires+a+polycomb-like+protein+complex+that+also+includes+VERNALIZATION+INSENSITIVE+3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+thaliana+vernalization+response+requires+a+polycomb-like+protein+complex+that+also+includes+VERNALIZATION+INSENSITIVE+3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001210-Matzke1" id="pone.0001210-Matzke1"></a>Matzke MA, Birchler JA (2005) RNAi-mediated pathways in the nucleus. Nat. Rev. Genet.  6: 24–35.  <ul class="find" data-citedArticleID="1020690" data-doi="10.1038/nrg1500"><li><a href="http://dx.doi.org/10.1038/nrg1500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNAi-mediated+pathways+in+the+nucleus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNAi-mediated+pathways+in+the+nucleus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001210-Richards1" id="pone.0001210-Richards1"></a>Richards EJ, Elgin SC (2002) Epigenetic codes for heterochromatin formation and silencing: rounding up the usual suspects. Cell  108: 489–500.  <ul class="find" data-citedArticleID="1020704" data-doi="10.1016/s0092-8674(02)00644-x"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00644-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epigenetic+codes+for+heterochromatin+formation+and+silencing%3A+rounding+up+the+usual+suspects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epigenetic+codes+for+heterochromatin+formation+and+silencing%3A+rounding+up+the+usual+suspects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001210-Tariq1" id="pone.0001210-Tariq1"></a>Tariq M, Paszkowski J (2004) DNA and histone methylation in plants. Trends Genet.  20: 244–251.  <ul class="find" data-citedArticleID="1020714" data-doi="10.1016/j.tig.2004.04.005"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.04.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+and+histone+methylation+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+and+histone+methylation+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001210-Bender1" id="pone.0001210-Bender1"></a>Bender J (2004) Chromatin-based silencing mechanisms. Curr Opin Plant Biol  7: 521–526.  <ul class="find" data-citedArticleID="1020646" data-doi="10.1016/j.pbi.2004.07.003"><li><a href="http://dx.doi.org/10.1016/j.pbi.2004.07.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin-based+silencing+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin-based+silencing+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001210-Fuchs1" id="pone.0001210-Fuchs1"></a>Fuchs J, demidov D, Houben A, Schubert I (2006) Chromosomal histone modification patterns-from conservation to diversity. Trends Plant  Sci.11: 199–208.  <ul class="find" data-citedArticleID="1020656" data-doi="10.1016/j.tplants.2006.02.008"><li><a href="http://dx.doi.org/10.1016/j.tplants.2006.02.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+histone+modification+patterns-from+conservation+to+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+histone+modification+patterns-from+conservation+to+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001210-Jackson1" id="pone.0001210-Jackson1"></a>Jackson JP, Lindroth AM, Cao X, Jacobsen SE (2002) Control of CpNpG DNA methylation by the KRYPTONITE histone H3 methyltransferase. Nature  416: 556–560.  <ul class="find" data-citedArticleID="1020664" data-doi="10.1038/nature731"><li><a href="http://dx.doi.org/10.1038/nature731" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+CpNpG+DNA+methylation+by+the+KRYPTONITE+histone+H3+methyltransferase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+CpNpG+DNA+methylation+by+the+KRYPTONITE+histone+H3+methyltransferase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001210-Lindroth1" id="pone.0001210-Lindroth1"></a>Lindroth AM, Cao X, Jackson JP, Zilberman D, McCallum CM, et al.  (2001) Requirement of CHROMOMETHYLASE3 for maintenance of CpXpG methylation. Science  292: 2077–2080.  <ul class="find" data-citedArticleID="1020678" data-doi="10.1126/science.1059745"><li><a href="http://dx.doi.org/10.1126/science.1059745" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Requirement+of+CHROMOMETHYLASE3+for+maintenance+of+CpXpG+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Requirement+of+CHROMOMETHYLASE3+for+maintenance+of+CpXpG+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001210-Johnson1" id="pone.0001210-Johnson1"></a>Johnson L, Cao X, Jacobsen S (2002) Interplay between two epigenetic marks. DNA methylation and histone H3 lysine 9 methylation. Curr. Biol.  12: 1360–1367.  <ul class="find" data-citedArticleID="1020670" data-doi="10.1016/s0960-9822(02)00976-4"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)00976-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interplay+between+two+epigenetic+marks.+DNA+methylation+and+histone+H3+lysine+9+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interplay+between+two+epigenetic+marks.+DNA+methylation+and+histone+H3+lysine+9+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001210-Mathieu1" id="pone.0001210-Mathieu1"></a>Mathieu O, Probst AV, Paszkowski J (2005) Distinct regulation of histone H3 methylation at lysines 27 and 9 by CpG methylation in <em>Arabidopsis</em>. EMBO J.  24: 2783–2791.  <ul class="find" data-citedArticleID="1020688" data-doi="10.1038/sj.emboj.7600743"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600743" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+regulation+of+histone+H3+methylation+at+lysines+27+and+9+by+CpG+methylation+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+regulation+of+histone+H3+methylation+at+lysines+27+and+9+by+CpG+methylation+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001210-Johnson2" id="pone.0001210-Johnson2"></a>Johnson L, Mollah S, Garcia BA, Muratore TL, Shabanowitz J, et al.  (2004) Mass spectrometry analysis of <em>Arabidopsis</em> histone H3 reveals distinct combinations of post-translational modifications. Nucleic Acids Res  32: 6511–6518.  <ul class="find" data-citedArticleID="1020672" data-doi="10.1093/nar/gkh992"><li><a href="http://dx.doi.org/10.1093/nar/gkh992" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mass+spectrometry+analysis+of+Arabidopsis+histone+H3+reveals+distinct+combinations+of+post-translational+modifications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mass+spectrometry+analysis+of+Arabidopsis+histone+H3+reveals+distinct+combinations+of+post-translational+modifications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001210-Peterson1" id="pone.0001210-Peterson1"></a>Peterson JR, Laniel MA (2004) Histones and histone modifications. Curr. Biol.  14: R546–R551.  <ul class="find" data-citedArticleID="1020700" data-doi="10.1016/j.cub.2004.07.007"><li><a href="http://dx.doi.org/10.1016/j.cub.2004.07.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histones+and+histone+modifications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histones+and+histone+modifications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001210-Wang1" id="pone.0001210-Wang1"></a>Wang HL, Wang L, Erdjument-Bromage H, Vidal M, Tempst P, et al.  (2004) Role of histone H2A ubiquiination in polycomb silencing. Nature  431: 873–878.  <ul class="find" data-citedArticleID="1020720" data-doi="10.1038/nature02985"><li><a href="http://dx.doi.org/10.1038/nature02985" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+histone+H2A+ubiquiination+in+polycomb+silencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+histone+H2A+ubiquiination+in+polycomb+silencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001210-Aihara1" id="pone.0001210-Aihara1"></a>Aihara H, Nakagawa T, Yasui K, Ohta T, Hirose S, et al.  (2004) Nucleosomal histone kinase-1 phosphorylates H2A Thr119 during mitosis in the early <em>Drosophila</em> embryo. Genes &amp; Dev.  18: 877–888.  <ul class="find" data-citedArticleID="1020638" data-doi="10.1101/gad.1184604"><li><a href="http://dx.doi.org/10.1101/gad.1184604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleosomal+histone+kinase-1+phosphorylates+H2A+Thr119+during+mitosis+in+the+early+Drosophila+embryo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleosomal+histone+kinase-1+phosphorylates+H2A+Thr119+during+mitosis+in+the+early+Drosophila+embryo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001210-Hutti1" id="pone.0001210-Hutti1"></a>Hutti JE, Jarrell ET, Chang JD, Abbott DW, Storz P, et al.  (2004) A rapid method for determining protein kinase phosphorylation specificity. Nat. Methods  1: 27–29.  <ul class="find" data-citedArticleID="1020662" data-doi="10.1038/nmeth708"><li><a href="http://dx.doi.org/10.1038/nmeth708" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+rapid+method+for+determining+protein+kinase+phosphorylation+specificity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+rapid+method+for+determining+protein+kinase+phosphorylation+specificity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001210-Beck1" id="pone.0001210-Beck1"></a>Beck HC, Nielsen RM, Matthiesen R, Jensen LH, Sehested M, et al.  (2006) Quantitative proteomic analysis of post-translational modifications of human histones. Mol. Cell. Proteomics  5: 1314–1325.  <ul class="find" data-citedArticleID="1020644" data-doi="10.1074/mcp.m600007-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m600007-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+proteomic+analysis+of+post-translational+modifications+of+human+histones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+proteomic+analysis+of+post-translational+modifications+of+human+histones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001210-Sridhar1" id="pone.0001210-Sridhar1"></a>Sridhar VV, Kapoor A, Zhang K, Zhu J, Zhou T, et al.  (2007) Control of DNA methylation and heterochromatic silencing by histone H2B deubiquitination. Nature  447: 735–738.  <ul class="find" data-citedArticleID="1020710" data-doi="10.1038/nature05864"><li><a href="http://dx.doi.org/10.1038/nature05864" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+DNA+methylation+and+heterochromatic+silencing+by+histone+H2B+deubiquitination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+DNA+methylation+and+heterochromatic+silencing+by+histone+H2B+deubiquitination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001210-Kaiser1" id="pone.0001210-Kaiser1"></a>Kaiser P, Wohlschlegel J (2005) Identification of ubiquitination sites and determination of ubiquitin-chain architectures by mass spectrometry. Methods Enzymol  399: 266–276.  <ul class="find" data-citedArticleID="1020674" data-doi="10.1016/s0076-6879(05)99018-6"><li><a href="http://dx.doi.org/10.1016/s0076-6879(05)99018-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+ubiquitination+sites+and+determination+of+ubiquitin-chain+architectures+by+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+ubiquitination+sites+and+determination+of+ubiquitin-chain+architectures+by+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001210-Hatakeyama1" id="pone.0001210-Hatakeyama1"></a>Hatakeyama S, Matsumoto M, Nakayama KI (2005) Mapping of ubiquitation sites on target proteins. Methods Enzymol  399: 277–286.  <ul class="find" data-citedArticleID="1020660" data-doi="10.1016/s0076-6879(05)99019-8"><li><a href="http://dx.doi.org/10.1016/s0076-6879(05)99019-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+of+ubiquitation+sites+on+target+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+of+ubiquitation+sites+on+target+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001210-Thorne1" id="pone.0001210-Thorne1"></a>Thorne AW, Kmiciek D, Mitchelson K, Sautiere P, Crane-Robinson C (1990) Patterns of histone acetylation. Eur. J. Biochem.  179: 701–714.  <ul class="find" data-citedArticleID="1020716" data-doi="10.1111/j.1432-1033.1990.tb19390.x"><li><a href="http://dx.doi.org/10.1111/j.1432-1033.1990.tb19390.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+histone+acetylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+histone+acetylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001210-Bassing1" id="pone.0001210-Bassing1"></a>Bassing CH, Chua KF, Sekiguchi J, Suh H, Whitlow SR, et al.  (2002) Increased ionizing radiation sensitivity and genomic instability in the absence of histone H2AX. Proc. Natl. Acad. Sci. USA  99: 8173–8178.  <ul class="find" data-citedArticleID="1020640" data-doi="10.1073/pnas.122228699"><li><a href="http://dx.doi.org/10.1073/pnas.122228699" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Increased+ionizing+radiation+sensitivity+and+genomic+instability+in+the+absence+of+histone+H2AX." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Increased+ionizing+radiation+sensitivity+and+genomic+instability+in+the+absence+of+histone+H2AX.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001210-Nelson1" id="pone.0001210-Nelson1"></a>Nelson CJ, Santos-Rosa H, Kouzarides T (2006) Proline isomerization of histone H3 regulates lysine methylation and gene expression. Cell  126: 905–916.  <ul class="find" data-citedArticleID="1020694" data-doi="10.1016/j.cell.2006.07.026"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.07.026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proline+isomerization+of+histone+H3+regulates+lysine+methylation+and+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proline+isomerization+of+histone+H3+regulates+lysine+methylation+and+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001210-Ahn1" id="pone.0001210-Ahn1"></a>Ahn S, Cheung WL, Hsu J, Diaz RL, Smith MM, et al.  (2005) Sterile 20 kinase phosphorylates histone H2B at serine 10 during hydrogen peroxide-induced apoptosis in <em>S. cerevisiae</em>. Cell  120: 25–36.  <ul class="find" data-citedArticleID="1020636" data-doi="10.1016/j.cell.2004.11.016"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.11.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sterile+20+kinase+phosphorylates+histone+H2B+at+serine+10+during+hydrogen+peroxide-induced+apoptosis+in+S.+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sterile+20+kinase+phosphorylates+histone+H2B+at+serine+10+during+hydrogen+peroxide-induced+apoptosis+in+S.+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001210-Fleury1" id="pone.0001210-Fleury1"></a>Fleury D, Himanen K, Cnops G, Nelissen H, Boccardi TM, et al.  (2007) The <em>Arabidopsis thaliana</em> homolog of yeast BRE1 has a function in cell cycle regulation during early leaf and root growth. Plant Cell  19: 417–32.  <ul class="find" data-citedArticleID="1020654" data-doi="10.1105/tpc.106.041319"><li><a href="http://dx.doi.org/10.1105/tpc.106.041319" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+thaliana+homolog+of+yeast+BRE1+has+a+function+in+cell+cycle+regulation+during+early+leaf+and+root+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+thaliana+homolog+of+yeast+BRE1+has+a+function+in+cell+cycle+regulation+during+early+leaf+and+root+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001210-Liu1" id="pone.0001210-Liu1"></a>Liu Y, Koornneef M, Soppe WJ (2007) The absence of histone H2B monoubiquitination in the <em>Arabidopsis hub1</em> (<em>rdo4</em>) mutant reveals a role for chromatin remodeling in seed dormancy. Plant Cell  19: 433–44.  <ul class="find" data-citedArticleID="1020680" data-doi="10.1105/tpc.106.049221"><li><a href="http://dx.doi.org/10.1105/tpc.106.049221" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+absence+of+histone+H2B+monoubiquitination+in+the+Arabidopsis+hub1+%28rdo4%29+mutant+reveals+a+role+for+chromatin+remodeling+in+seed+dormancy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+absence+of+histone+H2B+monoubiquitination+in+the+Arabidopsis+hub1+%28rdo4%29+mutant+reveals+a+role+for+chromatin+remodeling+in+seed+dormancy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001210-Bochkrev1" id="pone.0001210-Bochkrev1"></a>Bochkrev A, Pfuetzner RA, Edwards AM, Frappier L (1997) Structure of the single-stranded-DNA-binding domain of replication protein A bound to DNA. Nature  385: 176–181.  <ul class="find" data-citedArticleID="1020648" data-doi="10.1038/385176a0"><li><a href="http://dx.doi.org/10.1038/385176a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+the+single-stranded-DNA-binding+domain+of+replication+protein+A+bound+to+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+the+single-stranded-DNA-binding+domain+of+replication+protein+A+bound+to+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001210-Fang1" id="pone.0001210-Fang1"></a>Fang J, Feng Q, Ketel CS, Wang H, Cao R, et al.  (2002) Purification and functional characterization of SET8, a nucleosomal histone H4-lysine 20-specific methyltransferase. Curr. Biol.  12: 1086–1099.  <ul class="find" data-citedArticleID="1020652" data-doi="10.1016/s0960-9822(02)00924-7"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)00924-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification+and+functional+characterization+of+SET8%2C+a+nucleosomal+histone+H4-lysine+20-specific+methyltransferase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification+and+functional+characterization+of+SET8%2C+a+nucleosomal+histone+H4-lysine+20-specific+methyltransferase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001210-Rice1" id="pone.0001210-Rice1"></a>Rice JC, Nishioka K, Sarma K, Steward R, Reinberg D, et al.  (2002) Mitotic-specific methylation of histone H4 Lys 20 follows increased PR-Set7 expression and its localization to mitotic chromosomes. Genes &amp; Dev.  16: 2225–2230.  <ul class="find" data-citedArticleID="1020702" data-doi="10.1101/gad.1014902"><li><a href="http://dx.doi.org/10.1101/gad.1014902" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitotic-specific+methylation+of+histone+H4+Lys+20+follows+increased+PR-Set7+expression+and+its+localization+to+mitotic+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitotic-specific+methylation+of+histone+H4+Lys+20+follows+increased+PR-Set7+expression+and+its+localization+to+mitotic+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001210-Sanders1" id="pone.0001210-Sanders1"></a>Sanders SL, Manuela P, Mata J, Bähler , JüAllshire RC, et al.  (2004) Methylation of histone H4 lysine 20 controls recruitment of Crb2 to sites of DNA damage. Cell  119: 603–614.  <ul class="find" data-citedArticleID="1020706" data-doi="10.1016/j.cell.2004.11.009"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.11.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methylation+of+histone+H4+lysine+20+controls+recruitment+of+Crb2+to+sites+of+DNA+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methylation+of+histone+H4+lysine+20+controls+recruitment+of+Crb2+to+sites+of+DNA+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001210-Schotta1" id="pone.0001210-Schotta1"></a>Schotta G, Lachner M, Sarma K, Ebert A, Sengupta R, et al.  (2006) A silencing pathway to induce H3-K9 and H4-K20 trimethylation at constitutive heterochromatin. Genes &amp; Dev.  18: 1251–1262.  <ul class="find" data-citedArticleID="1020708" data-doi="10.1101/gad.300704"><li><a href="http://dx.doi.org/10.1101/gad.300704" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+silencing+pathway+to+induce+H3-K9+and+H4-K20+trimethylation+at+constitutive+heterochromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+silencing+pathway+to+induce+H3-K9+and+H4-K20+trimethylation+at+constitutive+heterochromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001210-Waterborg1" id="pone.0001210-Waterborg1"></a>Waterborg JH (1992) Identification of five sites of acetylation in Alfalfa histone H4. Biochemistry  31: 6211–6219.  <ul class="find" data-citedArticleID="1020722" data-doi="10.1021/bi00142a006"><li><a href="http://dx.doi.org/10.1021/bi00142a006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+five+sites+of+acetylation+in+Alfalfa+histone+H4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+five+sites+of+acetylation+in+Alfalfa+histone+H4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001210-Waterborg2" id="pone.0001210-Waterborg2"></a>Waterborg JH, Winicov I, Harrington RE (1987) Histone variants and acetylated species from the alfalfa plant <em>Medicago sativa</em>. Arch. Biochem. Biophys.  256: 167–178.  <ul class="find" data-citedArticleID="1020724" data-doi="10.1016/0003-9861(87)90435-8"><li><a href="http://dx.doi.org/10.1016/0003-9861(87)90435-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+variants+and+acetylated+species+from+the+alfalfa+plant+Medicago+sativa.+Arch.+Biochem." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+variants+and+acetylated+species+from+the+alfalfa+plant+Medicago+sativa.+Arch.+Biochem.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001210-Olsen1" id="pone.0001210-Olsen1"></a>Olsen JV, Bagoev B, Gnad F, Macek B, Kumar C, et al.  (2006) Global, <em>in vivo</em>, and site-specific phosphorylation dynamics in signaling networks. Cell  127: 635–648.  <ul class="find" data-citedArticleID="1020698" data-doi="10.1016/j.cell.2006.09.026"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.09.026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global%2C+in+vivo%2C+and+site-specific+phosphorylation+dynamics+in+signaling+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global%2C+in+vivo%2C+and+site-specific+phosphorylation+dynamics+in+signaling+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001210-Zhang4" id="pone.0001210-Zhang4"></a>Zhang K (2006) From purification of large amount of phosphor-compounds (nucleotides) to the enrichment of phospho-peptides by anion-exchanging resin. Anal.Biochem  357: 225–231.  <ul class="find" data-citedArticleID="1020736" data-doi="10.1016/j.ab.2006.07.001"><li><a href="http://dx.doi.org/10.1016/j.ab.2006.07.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+purification+of+large+amount+of+phosphor-compounds+%28nucleotides%29+to+the+enrichment+of+phospho-peptides+by+anion-exchanging+resin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+purification+of+large+amount+of+phosphor-compounds+%28nucleotides%29+to+the+enrichment+of+phospho-peptides+by+anion-exchanging+resin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.XML" value="130056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.PDF" value="1010370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g001.PNG_L" value="276734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g001.PNG_M" value="228253"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g001.PNG_S" value="13228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g001.TIF" value="1277356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g001.PNG_I" value="110620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t001.PNG_L" value="110626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t001.PNG_M" value="66060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t001.PNG_S" value="9908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t001.TIF" value="516680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t001.PNG_I" value="19876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g002.PNG_L" value="198664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g002.PNG_M" value="196417"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g002.PNG_S" value="9931"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g002.TIF" value="725596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g002.PNG_I" value="66836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t002.PNG_L" value="109604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t002.PNG_M" value="59498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t002.PNG_S" value="10121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t002.TIF" value="534168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t002.PNG_I" value="17738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g003.PNG_L" value="163604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g003.PNG_M" value="165165"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g003.PNG_S" value="8998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g003.TIF" value="668104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g003.PNG_I" value="56447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t003.PNG_L" value="145677"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t003.PNG_M" value="82903"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t003.PNG_S" value="11313"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t003.TIF" value="698156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t003.PNG_I" value="25661"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g004.PNG_L" value="108662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g004.PNG_M" value="114154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g004.PNG_S" value="8023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g004.TIF" value="511166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.g004.PNG_I" value="41173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t004.PNG_L" value="89403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t004.PNG_M" value="53671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t004.PNG_S" value="9231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t004.TIF" value="424304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.t004.PNG_I" value="16157"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.s001.PDF" value="150623"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.s002.EPS" value="777003"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.s003.EPS" value="993195"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.s004.EPS" value="1026967"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001210.s005.EPS" value="1318515"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001210&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001210" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001210&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001210&volume=&issue=&title=Distinctive Core Histone Post-Translational Modification Patterns in Arabidopsis thaliana&author_name=Kangling%20Zhang%2C%20Vaniyambadi%20V.%20Sridhar%2C%20Jianhua%20Zhu%2C%20Avnish%20Kapoor%2C%20Jian-Kang%20Zhu&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001210" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001210" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001210" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001210&amp;t=Distinctive%20Core%20Histone%20Post-Translational%20Modification%20Patterns%20in%20Arabidopsis%20thaliana" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001210&title=Distinctive%20Core%20Histone%20Post-Translational%20Modification%20Patterns%20in%20Arabidopsis%20thaliana&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001210&amp;title=Distinctive%20Core%20Histone%20Post-Translational%20Modification%20Patterns%20in%20Arabidopsis%20thaliana" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001210" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001210&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Distinctive Core Histone Post-Translational Modification Patterns in Arabidopsis thaliana';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001210';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Distinctive%20Core%20Histone%20Post-Translational%20Modification%20Patterns%20in%20Arabidopsis%20thaliana http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001210" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001210" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Acetylation%22" title="Search for articles in the subject area:'Acetylation'"><div class="flagText">Acetylation</div></a>
              <div data-categoryid="21455" data-articleid="25252"
                   data-categoryname="Acetylation"
                   class="flagImage" title="Flag 'Acetylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arabidopsis+thaliana%22" title="Search for articles in the subject area:'Arabidopsis thaliana'"><div class="flagText">Arabidopsis thaliana</div></a>
              <div data-categoryid="32845" data-articleid="25252"
                   data-categoryname="Arabidopsis thaliana"
                   class="flagImage" title="Flag 'Arabidopsis thaliana' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="25252"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22High+performance+liquid+chromatography%22" title="Search for articles in the subject area:'High performance liquid chromatography'"><div class="flagText">High performance liquid chromatography</div></a>
              <div data-categoryid="20319" data-articleid="25252"
                   data-categoryname="High performance liquid chromatography"
                   class="flagImage" title="Flag 'High performance liquid chromatography' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Histones%22" title="Search for articles in the subject area:'Histones'"><div class="flagText">Histones</div></a>
              <div data-categoryid="18573" data-articleid="25252"
                   data-categoryname="Histones"
                   class="flagImage" title="Flag 'Histones' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lysine%22" title="Search for articles in the subject area:'Lysine'"><div class="flagText">Lysine</div></a>
              <div data-categoryid="40335" data-articleid="25252"
                   data-categoryname="Lysine"
                   class="flagImage" title="Flag 'Lysine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Peptides%22" title="Search for articles in the subject area:'Peptides'"><div class="flagText">Peptides</div></a>
              <div data-categoryid="17847" data-articleid="25252"
                   data-categoryname="Peptides"
                   class="flagImage" title="Flag 'Peptides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phosphorylation%22" title="Search for articles in the subject area:'Phosphorylation'"><div class="flagText">Phosphorylation</div></a>
              <div data-categoryid="42723" data-articleid="25252"
                   data-categoryname="Phosphorylation"
                   class="flagImage" title="Flag 'Phosphorylation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9813'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3278'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6455&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
