

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Horizontal Gene Transfer Regulation in Bacteria as a “Spandrel” of DNA Repair Mechanisms</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001055"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001055" />

    <meta name="citation_title" content="Horizontal Gene Transfer Regulation in Bacteria as a “Spandrel” of DNA Repair Mechanisms"/>
    <meta itemprop="name" content="Horizontal Gene Transfer Regulation in Bacteria as a “Spandrel” of DNA Repair Mechanisms"/>

      <meta name="citation_author" content="Saliou Fall"/>
            <meta name="citation_author_institution" content="Environmental Microbial Genomics Group, Laboratoire AMPERE UMR CNRS 5005, Ecole Centrale de Lyon et Université de Lyon, Ecully, France"/>
      <meta name="citation_author" content="Anne Mercier"/>
            <meta name="citation_author_institution" content="Ecologie Microbienne, UMR CNRS 5557, Université Claude Bernard–Lyon 1, Villeurbanne, France"/>
      <meta name="citation_author" content="Franck Bertolla"/>
            <meta name="citation_author_institution" content="Ecologie Microbienne, UMR CNRS 5557, Université Claude Bernard–Lyon 1, Villeurbanne, France"/>
      <meta name="citation_author" content="Alexandra Calteau"/>
            <meta name="citation_author_institution" content="Laboratoire de Biométrie et Biologie Évolutive, UMR CNRS 5558, Université Claude Bernard–Lyon 1, Villeurbanne, France"/>
      <meta name="citation_author" content="Laurent Gueguen"/>
            <meta name="citation_author_institution" content="Laboratoire de Biométrie et Biologie Évolutive, UMR CNRS 5558, Université Claude Bernard–Lyon 1, Villeurbanne, France"/>
      <meta name="citation_author" content="Guy Perrière"/>
            <meta name="citation_author_institution" content="Laboratoire de Biométrie et Biologie Évolutive, UMR CNRS 5558, Université Claude Bernard–Lyon 1, Villeurbanne, France"/>
      <meta name="citation_author" content="Timothy M. Vogel"/>
            <meta name="citation_author_institution" content="Environmental Microbial Genomics Group, Laboratoire AMPERE UMR CNRS 5005, Ecole Centrale de Lyon et Université de Lyon, Ecully, France"/>
      <meta name="citation_author" content="Pascal Simonet"/>
            <meta name="citation_author_institution" content="Environmental Microbial Genomics Group, Laboratoire AMPERE UMR CNRS 5005, Ecole Centrale de Lyon et Université de Lyon, Ecully, France"/>

    <meta name="citation_date" content="2007/10/24"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001055.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1055"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Biased biological functions of horizontally transferred genes in prokaryotic genomes.; citation_author=Y Nakamura; citation_author=T Itoh; citation_author=H Matsuda; citation_author=T Gojobori; citation_journal_title=Nature Genet; citation_volume=36; citation_number=1; citation_pages=760-766; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Reconciling the many faces of lateral gene transfer.; citation_author=JG Lawrence; citation_author=H Ochman; citation_journal_title=Trends Microbiol; citation_volume=10; citation_number=2; citation_pages=1-4; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=On the track of natural transformation in soil.; citation_author=E Paget; citation_author=P Simonet; citation_journal_title=FEMS Microb Ecol; citation_volume=15; citation_number=3; citation_pages=109-117; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Behavior of restriction-modification systems as selfish mobile elements and their impact on genome evolution.; citation_author=I Kobayashi; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=4; citation_pages=3742-3756; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic barriers among bacteria.; citation_author=I Matic; citation_author=F Taddei; citation_author=M Radman; citation_journal_title=Trends Microbiol; citation_volume=4; citation_number=5; citation_pages=69-72; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Homeologous recombination and mismatch repair during transformation in Streptococcus pneumoniae: saturation of the Hex mismatch repair system.; citation_author=O Humbert; citation_author=M Prudhomme; citation_author=R Hakenbeck; citation_author=CG Dowson; citation_author=JP Claverys; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=6; citation_pages=9052-9056; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Prokaryotic diversity–magnitude, dynamics, and controlling factors.; citation_author=V Torsvik; citation_author=L Ovreas; citation_author=TF Thingstad; citation_journal_title=Science; citation_volume=296; citation_number=7; citation_pages=1064-1066; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detection of lateral gene transfer among microbial genomes.; citation_author=MA Ragan; citation_journal_title=Curr Opin Genet Dev; citation_volume=11; citation_number=8; citation_pages=620-626; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of the plant pathogen Ralstonia solanacearum.; citation_author=M Salanoubat; citation_author=S Genin; citation_author=F Artiguenave; citation_author=J Gouzy; citation_author=S Mangenot; citation_journal_title=Nature; citation_volume=415; citation_number=9; citation_pages=497-502; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=During infection of its host, the plant pathogen Ralstonia solanacearum naturally develops a state of competence and exchanges genetic material.; citation_author=F Bertolla; citation_author=A Frostegård; citation_author=B Brito; citation_author=X Nesme; citation_author=P Simonet; citation_journal_title=molecular plant-microbe interact; citation_volume=12; citation_number=10; citation_pages=467-672; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The spandrels of San Marco and the Panglossian paradigm: a critique of the adaptationist programme.; citation_author=SJ Gould; citation_author=RC Lewontin; citation_journal_title=Proc R Soc Lond B Biol Sci; citation_volume=205; citation_number=11; citation_pages=581-598; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Evidence for the adaptive evolution of mutation rates.; citation_author=D Metzgar; citation_author=C Wills; citation_journal_title=Cell; citation_volume=101; citation_number=12; citation_pages=581-584; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The source of laterally transferred genes in bacterial genomes.; citation_author=V Daubin; citation_author=E Lerat; citation_author=G Perriere; citation_journal_title=Genome Biol; citation_volume=4; citation_number=13; citation_pages=R57; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=G+C3 Structuring along the genome: A common feature in prokaryotes.; citation_author=V Daubin; citation_author=G Perriere; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=14; citation_pages=471-483; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular archaeology of the Escherichia coli genome.; citation_author=JG Lawrence; citation_author=H Ochman; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=95; citation_number=15; citation_pages=9413-9417; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Base composition bias might result from competition for metabolic resources.; citation_author=EP Rocha; citation_author=A Danchin; citation_journal_title=Trends Genet; citation_volume=18; citation_number=16; citation_pages=291-294; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The codon Adaptation Index: a measure of directional synonymous codon usage bias, and its potential applications.; citation_author=PM Sharp; citation_author=WH Li; citation_journal_title=Nucl Acids Res; citation_volume=15; citation_number=17; citation_pages=1281-1295; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Genomic structure and phylogeny of the plant pathogen Ralstonia solanacearum inferred from gene distribution analysis.; citation_author=A Guidot; citation_author=P Prior; citation_author=J Schoenfeld; citation_author=S Carrere; citation_author=S Genin; citation_journal_title=J Bacteriol; citation_volume=189; citation_number=18; citation_pages=377-387; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Goodness-of-fit tests on a circle.; citation_author=GS Watson; citation_journal_title=Biometrika; citation_volume=48; citation_number=19; citation_pages=109-114; citation_date=1961; " />
      <meta name="citation_reference" content="citation_title=Role of the Escherichia coli recombination hotspot, Chi, in RecABCD-dependent homologous pairing.; citation_author=DA Dixon; citation_author=SC Kowalczykowski; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=20; citation_pages=16360-16370; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Very short patch repair: reducing the cost of cytosine methylation.; citation_author=M Lieb; citation_author=AS Bhagwat; citation_journal_title=Mol Microbiol; citation_volume=20; citation_number=21; citation_pages=467-473; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The vsr gene product of E. coli K-12 is a strand- and sequence-specific DNA mismatch endonuclease.; citation_author=F Hennecke; citation_author=H Kolmar; citation_author=K Brundl; citation_author=HJ Fritz; citation_journal_title=Nature; citation_volume=353; citation_number=22; citation_pages=776-778; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Role of the Multidrug Efflux Systems of Pseudomonas aeruginosa in Organic Solvent Tolerance.; citation_author=X-Z Li; citation_author=L Zhang; citation_author=K Poole; citation_journal_title=J Bacteriol; citation_volume=180; citation_number=23; citation_pages=2987-2991; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The pilA regulatory gene modulates the pilus-mediated adhesion of Neisseria meningitidis by controlling the transcription of pilC1.; citation_author=MK Taha; citation_author=D Giorgini; citation_author=X Nassif; citation_journal_title=Mol Microbiol; citation_volume=19; citation_number=24; citation_pages=1073-1084; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The hosts of Pseudomonas solanacearum.; citation_author=AC Hayward; citation_number=25; citation_pages=9-24; citation_date=1994; citation_publisher=CAB INTERNATIONAL; " />
      <meta name="citation_reference" content="citation_title=Opportunistic colonization of Ralstonia solanacearum-infected plants by Acinetobacter sp. and its natural competence development.; citation_author=E Kay; citation_author=F Bertolla; citation_author=TM Vogel; citation_author=P Simonet; citation_journal_title=Microb Ecol; citation_volume=43; citation_number=26; citation_pages=291-297; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The complete genome sequence of Chlorobium tepidum TLS, a photosynthetic, anaerobic, green-sulfur bacterium.; citation_author=JA Eisen; citation_author=KE Nelson; citation_author=IT Paulsen; citation_author=JF Heidelberg; citation_author=M Wu; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=27; citation_pages=9509-9514; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Pseudomonas aeruginosa PA01, an opportunistic pathogen.; citation_author=CK Stover; citation_author=XQ Pham; citation_author=AL Erwin; citation_author=SD Mizoguchi; citation_author=P Warrener; citation_journal_title=Nature; citation_volume=406; citation_number=28; citation_pages=959-964; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Regulation of stalk elongation by phosphate in Caulobacter crescentus.; citation_author=M Gonin; citation_author=EM Quardokus; citation_author=D O'Donnol; citation_author=J Maddock; citation_author=YV Brun; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=29; citation_pages=337-347; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Caulobacter crescentus.; citation_author=WC Nierman; citation_author=TV Feldblyum; citation_author=MT Laub; citation_author=IT Paulsen; citation_author=KE Nelson; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=30; citation_pages=4136-4141; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Conditions for natural transformation of Ralstonia solanacearum.; citation_author=F Bertolla; citation_author=F Van Gijsegem; citation_author=X Nesme; citation_author=P Simonet; citation_journal_title=Appl Environ Microbiol; citation_volume=63; citation_number=31; citation_pages=4965-4968; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=DNA as a nutrient: novel role for bacterial competence gene homologs.; citation_author=SE Finkel; citation_author=R Kolter; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=32; citation_pages=6288-6293; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genome dynamics of short oligonucleotides: The example of bacterial DNA uptake enhancing sequences.; citation_author=M Bakkali; citation_journal_title=PLoS ONE; citation_volume=e741; citation_number=33; citation_pages=1-15; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Characteristics of Chi distribution on different bacterial genomes.; citation_author=M El Karoui; citation_author=V Biaudet; citation_author=S Schbath; citation_author=A Gruss; citation_journal_title=Res Microbiol; citation_volume=150; citation_number=34; citation_pages=579-587; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Biochemistry of homologous recombination in Escherichia coli.; citation_author=SC Kowalczykowski; citation_author=DA Dixon; citation_author=AK Eggleston; citation_author=SD Lauder; citation_author=WM Rehrauer; citation_journal_title=Microbiol Rev; citation_volume=58; citation_number=35; citation_pages=401-465; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Cellular response to horizontally transferred DNA in Escherichia coli is tuned by DNA repair systems.; citation_author=S Delmas; citation_author=I Matic; citation_journal_title=DNA Repair (Amst); citation_volume=4; citation_number=36; citation_pages=221-229; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Horizontal gene transfer, genome innovation and evolution.; citation_author=JP Gogarten; citation_author=JP Townsend; citation_journal_title=Nat Rev Microbiol; citation_volume=3; citation_number=37; citation_pages=679-687; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phenotypic characterization and deoxyribonucleic acid homologies of Pseudomonas solanacearum.; citation_author=NJ Palleroni; citation_author=M Doudoroff; citation_journal_title=J Bacteriol; citation_volume=107; citation_number=38; citation_pages=690-696; citation_date=1971; " />
      <meta name="citation_reference" content="citation_title=Genomics and the bacterial species problem.; citation_author=WF Doolittle; citation_author=RT Papke; citation_journal_title=Genome Biol; citation_volume=7; citation_number=39; citation_pages=116; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic evidence for horizontal transfer of mutS alleles among naturally occurring Escherichia coli strains.; citation_author=EW Brown; citation_author=JE LeClerc; citation_author=B Li; citation_author=WL Payne; citation_author=TA Cebula; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=40; citation_pages=1631-1644; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolutionary implications of the frequent horizontal transfer of mismatch repair genes.; citation_author=E Denamur; citation_author=G Lecointre; citation_author=P Darlu; citation_author=O Tenaillon; citation_author=C Acquaviva; citation_journal_title=Cell; citation_volume=103; citation_number=41; citation_pages=711-721; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Virulence genes are carried by a megaplasmid of the plant pathogen Pseudomonas solanacearum.; citation_author=C Boucher; citation_author=A Martinel; citation_author=P Barberis; citation_author=G Alloing; citation_author=C Zischek; citation_journal_title=Mol Gen Genet; citation_volume=205; citation_number=42; citation_pages=270-275; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput.; citation_author=RC Edgar; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=43; citation_pages=1792-1797; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis.; citation_author=J Castresana; citation_journal_title=Mol Microbiol; citation_volume=17; citation_number=44; citation_pages=540-552; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Syst Biol; citation_volume=52; citation_number=45; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach.; citation_author=S Whelan; citation_author=N Goldman; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=46; citation_pages=691-699; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Multiple comparisons of log-likelihoods with applications to phylogenetic inference.; citation_author=H Shimodaira; citation_author=M Hasegawa; citation_journal_title=Mo Biol Evol; citation_volume=16; citation_number=47; citation_pages=1114-1116; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genes for gentamicin-(3)-N-acetyltransferases III and IV: I. Nucleotide sequence of the AAC(3)-IV gene and possible involvement of an IS140 element in its expression.; citation_author=B Brau; citation_author=U Pilz; citation_author=W Piepersberg; citation_journal_title=Mol Gen Genet; citation_volume=193; citation_number=48; citation_pages=179-187; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=SeqinR 1.0-2: a contributed package to the R project for statistical computing devoted to biological sequences retrieval and analysis.; citation_author=D Charif; citation_author=JR Lobry; citation_number=49; citation_pages=207-232; citation_date=2007; citation_publisher=Springer Verlag: Biological and Medical Physics, Biomedical Engineering; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001055" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Horizontal Gene Transfer Regulation in Bacteria as a “Spandrel” of DNA Repair Mechanisms"/>
    <meta name="twitter:description" content="Horizontal gene transfer (HGT) is recognized as the major force for bacterial genome evolution. Yet, numerous questions remain about the transferred genes, their function, quantity and frequency. The extent to which genetic transformation by exogenous DNA has occurred over evolutionary time was initially addressed by an in silico approach using the complete genome sequence of the Ralstonia solanacearum GMI1000 strain. Methods based on phylogenetic reconstruction of prokaryote homologous genes families detected 151 genes (13.3%) of foreign origin in the R. solanacearum genome and tentatively identified their bacterial origin. These putative transfers were analyzed in comparison to experimental transformation tests involving 18 different genomic DNA positions in the genome as sites for homologous or homeologous recombination. Significant transformation frequency differences were observed among these positions tested regardless of the overall genomic divergence of the R. solanacearum strains tested as recipients. The genomic positions containing the putative exogenous DNA were not systematically transformed at the highest frequencies. The two genomic &ldquo;hot spots&rdquo;, which contain recA and mutS genes, exhibited transformation frequencies from 2 to more than 4 orders of magnitude higher than positions associated with other genes depending on the recipient strain. These results support the notion that the bacterial cell is equipped with active mechanisms to modulate acquisition of new DNA in different genomic positions. Bio-informatics study correlated recombination &ldquo;hot-spots&rdquo; to the presence of Chi-like signature sequences with which recombination might be preferentially initiated. The fundamental role of HGT is certainly not limited to the critical impact that the very rare foreign genes acquired mainly by chance can have on the bacterial adaptation potential. The frequency to which HGT with homologous and homeologous DNA happens in the environment might have led the bacteria to hijack DNA repair mechanisms in order to generate genetic diversity without losing too much genomic stability."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001055.g006"/>

  <meta property="og:title" content="Horizontal Gene Transfer Regulation in Bacteria as a “Spandrel” of DNA Repair Mechanisms" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001055" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9760'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5628'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5459&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001055">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001055" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001055&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Horizontal Gene Transfer Regulation in Bacteria as a “Spandrel” of DNA Repair Mechanisms
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Saliou Fall, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Environmental Microbial Genomics Group, Laboratoire AMPERE UMR CNRS 5005, Ecole Centrale de Lyon et Université de Lyon, Ecully, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anne Mercier, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Ecologie Microbienne, UMR CNRS 5557, Université Claude Bernard–Lyon 1, Villeurbanne, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Franck Bertolla, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Ecologie Microbienne, UMR CNRS 5557, Université Claude Bernard–Lyon 1, Villeurbanne, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexandra Calteau, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Biométrie et Biologie Évolutive, UMR CNRS 5558, Université Claude Bernard–Lyon 1, Villeurbanne, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laurent Gueguen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Biométrie et Biologie Évolutive, UMR CNRS 5558, Université Claude Bernard–Lyon 1, Villeurbanne, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guy Perrière, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Biométrie et Biologie Évolutive, UMR CNRS 5558, Université Claude Bernard–Lyon 1, Villeurbanne, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Timothy M. Vogel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Environmental Microbial Genomics Group, Laboratoire AMPERE UMR CNRS 5005, Ecole Centrale de Lyon et Université de Lyon, Ecully, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pascal Simonet
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pascal.simonet@ec-lyon.fr">pascal.simonet@ec-lyon.fr</a></p>

                <p>Affiliation:
                  Environmental Microbial Genomics Group, Laboratoire AMPERE UMR CNRS 5005, Ecole Centrale de Lyon et Université de Lyon, Ecully, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 24, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001055</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001055" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001055" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001055" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001055" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001055-g001" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001055-g002" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001055-t001" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001055-g003" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001055-g004" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001055-t002" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001055-g005" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001055-g006" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001055-t003" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001055">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001055" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Horizontal gene transfer (HGT) is recognized as the major force for bacterial genome evolution. Yet, numerous questions remain about the transferred genes, their function, quantity and frequency. The extent to which genetic transformation by exogenous DNA has occurred over evolutionary time was initially addressed by an <em>in silico</em> approach using the complete genome sequence of the <em>Ralstonia solanacearum</em> GMI1000 strain. Methods based on phylogenetic reconstruction of prokaryote homologous genes families detected 151 genes (13.3%) of foreign origin in the <em>R. solanacearum</em> genome and tentatively identified their bacterial origin. These putative transfers were analyzed in comparison to experimental transformation tests involving 18 different genomic DNA positions in the genome as sites for homologous or homeologous recombination. Significant transformation frequency differences were observed among these positions tested regardless of the overall genomic divergence of the <em>R. solanacearum</em> strains tested as recipients. The genomic positions containing the putative exogenous DNA were not systematically transformed at the highest frequencies. The two genomic “hot spots”, which contain <em>recA</em> and <em>mutS</em> genes, exhibited transformation frequencies from 2 to more than 4 orders of magnitude higher than positions associated with other genes depending on the recipient strain. These results support the notion that the bacterial cell is equipped with active mechanisms to modulate acquisition of new DNA in different genomic positions. Bio-informatics study correlated recombination “hot-spots” to the presence of Chi-like signature sequences with which recombination might be preferentially initiated. The fundamental role of HGT is certainly not limited to the critical impact that the very rare foreign genes acquired mainly by chance can have on the bacterial adaptation potential. The frequency to which HGT with homologous and homeologous DNA happens in the environment might have led the bacteria to hijack DNA repair mechanisms in order to generate genetic diversity without losing too much genomic stability.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Fall S, Mercier A, Bertolla F, Calteau A, Gueguen L, et al.  (2007) Horizontal Gene Transfer Regulation in Bacteria as a “Spandrel” of DNA Repair Mechanisms. PLoS ONE 2(10):
          e1055.
            doi:10.1371/journal.pone.0001055</p><p><strong>Academic Editor: </strong>Angus Buckling, Oxford University, United Kingdom</p><p><strong>Received:</strong> April 3, 2007; <strong>Accepted:</strong> October 2, 2007; <strong>Published:</strong> October 24, 2007</p><p><strong>Copyright:</strong> © 2007 Fall et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was funded in part by the EU project TRANSBAC (QLK3-CT-2001-02242, 5th RTD Program) and in part by the ANR project PLOBEN (ANR-05-POGM-004-01).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The fundamental impact of horizontal gene transfer (HGT) in shaping the structure of bacterial genomes was only recently demonstrated by the analysis of numerous complete bacterial genome sequences <a href="#pone.0001055-Nakamura1">[1]</a>. The detection of relatively recently acquired genes is possible because the laterally transferred genes have compositional features that distinguish them from vertically inherited genes <a href="#pone.0001055-Lawrence1">[2]</a>. However, finding a significant proportion of transferred genes in a bacterial genome does not mean that the entire gene transfer process that leads to the stable inheritance of new genes occurs frequently in the environment. Actually, the frequency of gene transfer between phylogenetically remote bacteria is expected to be low due to the requirement for several successive and rare events including colonization of the same environmental niche by donor and recipient bacteria, physical contact, compatibility for conjugation and transduction or DNA persistence and competence development when DNA is directly taken up by natural transformation <a href="#pone.0001055-Paget1">[3]</a>. Foreign DNA that has successfully penetrated a bacterial cell is integrated into the host genome by illegitimate recombination only if it escapes degradation by the restriction-modification systems (RM) <a href="#pone.0001055-Kobayashi1">[4]</a> and the methyl-mismatch repair (MMR) system <a href="#pone.0001055-Matic1">[5]</a>. Due to these successive requirements, HGT with foreign DNA is unpredictable yet a single event occurring even once during bacterial evolution could fix a new trait in a bacterial lineage if the overall fitness is increased.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>When donor DNA originates from an organism closely related to the recipient, recombination between similar or partially divergent (called homeologous <a href="#pone.0001055-Humbert1">[6]</a>) sequences is much more likely. These transfer events will ensure genetic coherence and slow diversification when occurring within a group of closely related bacteria and will also promote environmental adaptation by sharing point mutations or transposon- and IS- mediated genetic rearrangements among the bacterial population <a href="#pone.0001055-Torsvik1">[7]</a>. However, bioinformatics methods cannot easily detect this new genetic information (unless significant numbers of individuals from the same species were completely sequenced) due to a lack of compositional features differentiating the donated DNA and the recipient genome <a href="#pone.0001055-Ragan1">[8]</a>. In this case, the frequency of DNA transfer, the differences in transfer potential of different genes, and the potential impact on population fitness can be addressed by experimental approaches.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this paper, we combined <em>in silico</em> and experimental approaches to study differences between these two types of HGT in bacteria. We used <em>R. solanacearum</em> as a model because of the evidence indicating a fundamental role of HGT in this plant pathogen's evolution. More than 7% of the genome was found to be encompassed by regions in which codon usage differed significantly from codon usage in the rest of the genome. In addition, most of these regions exhibited a base composition differing significantly from the G+C content for the entire genome indicating a foreign origin for these sequences <a href="#pone.0001055-Salanoubat1">[9]</a>. Other authors using the Bayesian method estimated that about 16% of the genome was acquired by HGT <a href="#pone.0001055-Nakamura1">[1]</a>. Our first goal was to use alternate methods, including phylogenetic reconstruction of prokaryote homologous gene families and calculation of two codon usage indices to complete the list of genes acquired by HGT. These methods would be less biased than intrinsic codon usage-based approaches, which tend to overestimate the number of transferred genes, and in addition would identify putative donor microorganisms. These <em>in silico</em> studies were complemented by natural transformation experiments (<a href="#pone-0001055-g001">Figure 1</a>) in order to determine if and how this bacterium regulates acquisition of the DNA, including the genes detected as recently acquired by the bioinformatics analyses, that originates from the same or very closely related strains. <em>R. solanacearum</em> seems particularly appropriate for addressing these evolutionary questions because it is a naturally transformable bacterium whose cells use natural transformation to exchange genes at significant frequencies under <em>in planta</em> conditions <a href="#pone.0001055-Bertolla1">[10]</a>. Our hypothesis is that <em>R. solanacearum</em> bacterial cells are subjected to a constant flux of more or less homologous DNA in the open environment and that this flux might have led to the adaptive use of (“spandrel”) of DNA acquisition regulation mechanisms in order to generate genetic diversity without losing too much genomic stability. The term “spandrel” refers to the adaptive use of a function selected for another purpose <a href="#pone.0001055-Gould1">[11]</a>, <a href="#pone.0001055-Metzgar1">[12]</a>.</p>
<div class="figure" id="pone-0001055-g001"><div class="img"><a name="pone-0001055-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Experimental design to measure recombination rate at different positions of the <em>R. solanacearum</em> chromosome.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Target positions on the chromosome (TCP) and on the megaplasmid (TMP) were identified and amplified by PCR. PCR products from the different genomic positions were cloned in appropriate vectors and afterwards, labeled with the <em>aacC3-IV</em> gene (gentamycin cassette). The pTCP (versus pTMP) plasmid carrying homologous GMI1000 fragments were linearized and resulting plasmids were used as donor to transform naturally the wild type strain GM1000 and recombination rate of each position designed. Total genomic DNA from <em>R. solanacearum</em> transformants and carrying <em>aacC3-IV</em> cassette resulting from double crossing-over were used as exogenous DNA donor to “re-transform” the wild type strain GM1000 and the CFBP2968, NCPPB332 and CFBP2957 strains to determine the recombination rate of genomic DNA. (Amp, ampicillin and Kn, kanamycin).</p>
<span>doi:10.1371/journal.pone.0001055.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Detection and bacterial origin of foreign genes acquired by <em>R. solanacearum</em> strain GMI1000</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In order to identify HGT events in the available <em>R. solanacearum</em> genome sequence, we used a phylogenetic approach to identify the putative donors of the newly acquired genes. With this approach, when <em>R. solanacearum</em> genes were not clustered with the other β-proteobacteria genes found in the family, and when this was supported by high probabilities for the Shimodaira-Hasagawa test, then we considered the possibility of an HGT event (<a href="#pone-0001055-g002">Figure 2</a>). When only a single β-proteobacteria other than <em>R. solanacearum</em> was present in a family, and when this bacteria was not clustered with <em>R. solanacearum</em>, we choose to ignore this family for the HGT count, since it was not possible to determine in which organism the transfer took place. We preferred this approach because it is less biased than codon usage-based methods, which tend to overestimate the number of transferred genes <a href="#pone.0001055-Daubin1">[13]</a>.</p>
<div class="figure" id="pone-0001055-g002"><div class="img"><a name="pone-0001055-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Two examples of HGT detected on the megaplasmid (a) and on the chromosome (b) through a phylogenetic approach.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The two trees have been built respectively with the sequences from the HOGENOM families HBG007143, corresponding to the Proline imminopeptidase (a), and HBG225336, corresponding to the transcriptional regulator of Acetoin catabolism (b). The value for the Shimodaira-Hasagawa likelihood ratio test is given for the internal branches (only when <em>P</em>≥0.95). In both cases, the sequence from <em>R. solanacearum</em> is not clustered with the other β-proteobacterial sequences, this with a significant value for the clustering test.</p>
<span>doi:10.1371/journal.pone.0001055.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Using the phylogenetic trees, we found 151 inconsistencies in the 1139 phylogenetic trees examined, which represents 13.3% of the total families studied (the set of corresponding alignments and trees can be downloaded at <a href="http://ftp://pbil.univ-lyon1.fr/pub/datasets/PLoS07">ftp://pbil.univ-lyon1.fr/pub/datasets/PL​oS07</a>). However, because horizontally transferred genes tend to be orphans enriched in A+T nucleotides <a href="#pone.0001055-Daubin2">[14]</a>–<a href="#pone.0001055-Rocha1">[16]</a>, we added to our list of putatively transferred genes those for which: (i) no available homologs were detected (and therefore no phylogenetic tree could be computed) or (ii) an extremely biased codon usage was found.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To do this, we computed the G+C content of the codon third position (G+C3%) and Codon Adaptation Index (CAI) <a href="#pone.0001055-Sharp1">[17]</a>. The CAI reference table was built with all the genes coding for ribosomal proteins in <em>R. solanacearum</em>. After computing G+C3% and CAI values, we selected the 10% of the genes having the lowest values for these indices. This left us with 268 genes from the 3.7 Mb chromosome and 138 genes from the second replicon, the 2.1 Mb megaplasmid. Then, we performed a BLASTP search using these genes as templates in order to find those without homologs outside <em>R. solanacearum</em>. For that purpose, we set the E-value threshold at 0.1. We found a total of 42 genes from the chromosome and 37 genes from the megaplasmid matching these criteria. These orphan genes were also added to the list of putatively transferred genes. Together, phylogenetic reconstitution and codon usage selected about 15% of the genes in <em>R. solanacearum</em> genome that could be associated with HGT events and identified the potential bacterial donors.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>All alignments and trees used to detect the transfers, as well as the list of orphans with high codon usage bias can be downloaded from the PBIL web server at <a href="http://ftp://pbil.univ-lyon1.fr/pub/datasets/PloS07">ftp://pbil.univ-lyon1.fr/pub/datasets/Pl​oS07</a>.</p>


<h4>Selection of DNA positions for natural transformation experiments</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Eighteen DNA positions on the genome encompassing a wide range of properties were selected to be amplified and cloned (<a href="#pone-0001055-t001">Table 1</a>). Fifteen of these positions were located on the chromosome (prefix TCP) and three positions were located on the megaplasmid (prefix TMP). Five DNA fragments (TCP4, TCP8, TCP14, TCP15 and TMP1) encompassed the DNA positions that were identified as being recently acquired by horizontal gene transfer by the phylogenetic reconstruction method (<a href="#pone-0001055-t001">Table 1</a>) and three DNA positions (TCP5, TCP14 and TMP2) contain DNA segments in which codon usage differed significantly from codon usage in the rest of the genome <a href="#pone.0001055-Salanoubat1">[9]</a>. The TCP4, TCP8, TCP14, TCP15 and TMP1 positions carried RSc3437 (<em>vsr</em>), RSc0558 (<em>pilA</em>), RSc1815, RSc3252 and RSp0313 (<em>mexC</em>) genes respectively. The <em>vsr</em> gene (HOGENOM family HBG327419) was among the genes predicted by our phylogenetic approach as having been horizontally transferred and exhibits a strong codon usage bias toward A+T. Three others selected genes (RSp0313 (<em>mexC</em>), RSc0558 (<em>pilA</em>) and RSc1815 were apparently acquired from the γ-proteobacteria, <em>Acinetobacter baumannii</em>, <em>Pseudomonas aeruginosa</em> and <em>Xanthomonas campestris</em>, respectively. The RSc3252 gene was acquired from the least related organism, <em>Chlorobium tepidum</em>, which belongs to green sulfurous bacteria.</p>
<div class="figure" id="pone-0001055-t001"><div class="img"><a name="pone-0001055-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001055.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001055.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Origin, properties of <em>Ralstonia solanacearum</em> (GM1000) genes used in this study as plasmid or genomic donor DNA; plasmid transformation frequencies and number of Chi-like sequences detected within the 2 kb long DNA positions.</span></strong></p><span>doi:10.1371/journal.pone.0001055.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Four DNA positions (TCP2, TCP10, TCP11 and TCP13) were selected because they contain well conserved house keeping genes (<em>ubiE</em>, <em>purD</em>, <em>ftsK</em> and <em>rpsG</em>). In addition, the megaplasmid TMP2 position was part of a 31 kb tandem position and TMP3 position encompasses a gene (<em>popA</em>) involved in plant pathogenicity. Four positions (TCP5, TCP9 TCP12 and TCP1) harbored an IS, a temperate bacteriophage, a transposon (TN) and a putative gene. Finally, DNA fragments, TCP3, TCP6 and TCP7, encompass genes whose functions are more or less directly involved in controlling genomic integrity (<em>recA, mutS</em>) and RSc1120, defined as coding for a DNA translocation protein which might be necessary for competence development (<em>comA-</em>like). A 2 kb DNA fragment from each of the 18 selected positions had the <em>aacC3-IV</em> gene, which confers resistance to gentamycin, inserted in its middle. The resulting recombinant plasmids were used as donor DNA in <em>R. solanacearum</em> natural transformation experiments (<a href="#pone-0001055-g001">Figure 1</a>).</p>


<h4>Transformation of <em>R. solanacearum</em> strain GMI1000 with plasmid-borne DNA fragments</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The TOPO plasmid vector without an insert did not produce any Km resistant recombinant <em>R. solanacearum</em> clones. Therefore, this plasmid was unable to replicate autonomously in <em>R. Solanacearum</em> GMI1000 strain and the integration frequency by illegitimate recombination was below the detection level. When plasmids contained inserts, DNA transformation frequencies varied by more than two orders of magnitude depending on the target position (<a href="#pone-0001055-g003">Figure 3</a>). The growth rate of transformants was not significantly modified in comparison to the wild type strain indicating that the differences in calculated transformation frequencies were not affected by differences in cell growth or survival (results not shown). The lowest frequency, which was found for pTCP2 (4.99+/−2.47×10<sup>−8</sup>), was 300 times lower than that of the highest frequency measured (pTCP3 at 1.66+/−0.26×10<sup>−5</sup>). Plasmids pTCP13 and pTMP1 also exhibited relatively low transformation frequency (1.40+/−0.29×10<sup>−7</sup> and 1.38+/−0.44×10<sup>−7</sup>, respectively; <a href="#pone-0001055-g003">Figure 3</a>) although they were still 3 times higher than that for pTCP2. Several plasmids transformed <em>R. solanacearum</em> strain GMI1000 at frequencies between one and two orders of magnitude higher than pTCP2. These included pTCP1, pTCP4, pTCP6, pTCP8, pTCP9, pTCP10, pTCP12, pTCP14, pTCP15 and pTMP2. Finally, pTCP5, pTCP7, pTCP11 and pTCP3 yielded transformants at frequencies more than two orders of magnitude higher than pTCP2. All the gentamycin resistant <em>R. solanacearum</em> clones tested resulted from double crossing over events according to their sensitivity to kanamycin and the size of the PCR products, which were all about 4 kb long as expected for gene replacement (results not shown).</p>
<div class="figure" id="pone-0001055-g003"><div class="img"><a name="pone-0001055-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Recombination frequency variation between 17 <em>R. solanacearum</em> chromosome and megaplasmid borne DNA positions.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Fifteen target positions on the chromosome (TCP) and three on the megaplasmid (TMP) were identified based on gene acquisition and function as described in the text. Plasmid donors were built following the general experimental design of <a href="#pone-0001055-g001">figure 1</a>. Recombination rate of these different genomic positions (proportional to red line) were measured after natural transformation and based on insertion of the <em>aacC3-IV</em> cassette by double crossing-over.</p>
<span>doi:10.1371/journal.pone.0001055.g003</span></div>

<h4>Transformation of different <em>R. solanacearum</em> strains with linear chromosomal DNA fragments from <em>R. solanacearum</em> strain GMI1000</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The total genomic DNA (composed of fragments ranging in size between 40 and 100 kb) from the different plasmid-transformed <em>R. solanacearum</em> GMI1000 clones was used as donor DNA with GMI1000, CFBP2968, NCPPB332 and CFBP2957 as recipient strains. Conditions for homologous recombination were established when GMI1000 strain was the recipient strain as the only difference between donor DNA from the same strain and the recipient genome was the marker gene inserted in the targeted DNA position. Recombination conditions for other recipient strains varied from homeologous to heterologous as strains CFBP2968 (phylotype I), NCPPB332 (phylotype III) and CFBP2957 (phylotype II) have respectively ~98%, ~81% and ~69% of the GMI1000 genes conserved in their genomes <a href="#pone.0001055-Guidot1">[18]</a>. In addition, these three strains were the most efficiently transformed among all strains tested in each phylotype and PCR carried out with GMI1000 primers for six out of the 18 positions tested confirmed presence of the corresponding genes in these isolates (results not shown).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Again, significant differences in transformation frequencies were detected for different genomic positions. In general, the DNA positions for the strain GMI1000 that exhibited the lowest transformation frequencies when recombination was mediated by plasmid borne DNA fragments also yielded low transformation rates when genomic DNA was used (<a href="#pone-0001055-g004">Figure 4</a> and <a href="#pone.0001055.s001">Table S1</a>). This included gTCP1 and gTCP13 with recombination frequencies&lt;10<sup>−10</sup> and 3+/−0.06×10<sup>−7</sup>, respectively (<a href="#pone-0001055-g004">Figure 4</a> and <a href="#pone.0001055.s001">Table S1</a>). Similarly, the highest transformation frequencies were observed when recombination was mediated by DNA positions (e.g., gTCP3 and gTCP7), which also yielded the highest frequency when they were located on a plasmid. However, comparison of transfer frequency between plasmids and genomic DNA of the same strain (GMI1000) for the other targeted DNA positions showed some discrepancies in spite of a significant (p&lt;0.01) correlation between the two experiments (correlation coefficient = 0.69; <a href="#pone-0001055-t002">Table 2</a>).</p>
<div class="figure" id="pone-0001055-g004"><div class="img"><a name="pone-0001055-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Correlation between recombination frequencies obtained with plasmid DNA and genomic DNA after natural transformation of <em>R. solanacearum</em> GMI1000 as recipient.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Donor DNA belongs to the same strain and was either plasmid DNA containing 2 kb long <em>R. solanacearum</em> DNA or genomic DNA from recombinant strains (see the text). The numbers along the curves refer to the DNA position acronym with white and black symbols for chromosome and megaplasmid positions, respectively (TCP2&lt;detection limit, TMP3, not determined). TF, transformation frequency.</p>
<span>doi:10.1371/journal.pone.0001055.g004</span></div><div class="figure" id="pone-0001055-t002"><div class="img"><a name="pone-0001055-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001055.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001055.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Correlation coefficients of plasmid DNA recombination frequencies with the genomic DNA recombination frequencies and with the main physico-chemical parameters of the seventeen 2 kb long DNA fragments targeted in this study.</span></strong></p><span>doi:10.1371/journal.pone.0001055.t002</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>With the other <em>R. solanacearum</em> strains as recipients (<a href="#pone-0001055-g005">Figure 5</a>), the transformation frequencies were systematically lower than with GMI1000 whatever the DNA positions tested and higher genomic divergence between the recipient strain and GMI1000 correlated with greater decreases in transformation frequencies (<a href="#pone-0001055-g005">Figure 5</a>). Moreover, the range between the highest and the lowest transformation frequencies also increased with overall genome divergence reaching more than 4 orders of magnitude for strains NCPPB332 and CFBP2957 while it was less than two orders of magnitude for the two most closely related strains (GMI1000 and CFBP2968).</p>
<div class="figure" id="pone-0001055-g005"><div class="img"><a name="pone-0001055-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Ratio of recombination frequencies after natural transformation of different distantly <em>R. solanacearum</em> strains.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Total genomic DNA of GMI1000 strain was used as donor with GMI1000, CFBP2968, NCPPB332 and CFBP2957 as recipient strains. These strains CFBP2968, NCPPB332 and CFBP2957 have respectively 98%, 81% and 69% of the GMI1000 genes conserved in their genomes (TCP2&lt;detection limit). TF, transformation frequency.</p>
<span>doi:10.1371/journal.pone.0001055.g005</span></div><a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>In spite of the increasing range of transformation frequencies with increasing genomic divergence, the DNA positions that yielded the highest transformation frequencies with GMI1000 (TCP3, TCP6, and TCP7) systematically yielded the highest frequencies for each recipient (<a href="#pone-0001055-g005">Figure 5</a>). The lowest transformation values observed for these strains corresponded to DNA positions that also yielded the lowest frequencies when recombination was homologous. In addition, the high frequency regions were less susceptible to increasing genomic divergence.</p>


<h4>Role of Chi<em>-</em>like sequences</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Physico-chemical parameters of the seventeen 2 kb long DNA fragments targeted in this study were calculated. These parameters, including GC%, GC “skew”, genome (chromosomal or plasmid) localization, distance from the origin at replication and the denaturing free energy (melting point), did not significantly correlate to recombination frequencies (<a href="#pone-0001055-t002">Table 2</a>).</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The 2 kb DNA sequences were examined in order to detect the presence of the longest repeated motifs, which might significantly correlate to the recombination frequencies: for each length = L with L being 7 and 8, we counted the occurrences of all the words of length L, and for each word we measured the correlation of the ranks between these numbers of occurrences and the frequencies, using a Spearman test. For L = 7, the two best words were 5′ cGCCGAA 3′ (p-value = 10<sup>−3.73</sup>) and 5′ GCCGAAc 3′ (p-value = 10<sup>−3.08</sup>) and for L = 8 the best word was 5′ cGCCGAAc 3′ (p-value = 10<sup>−3.06</sup>). These motifs were detected in 15 of the 17 fragments with 9 of them containing at least 3 copies of these specific sequences (<a href="#pone-0001055-t001">Table 1</a>). The number of motifs occurring in a given region correlated (R<sup>2</sup> = 0.48) to recombination frequencies (<a href="#pone-0001055-g006">Figure 6</a>). The three DNA positions that exhibited the highest transformation frequencies (TCP3, TCP7 and TCP11) contained two copies each of the consensus sequence (<a href="#pone-0001055-t001">Table 1</a>). The genes that yielded the lowest transformation values including <em>ubiE</em> (TCP2), and <em>mexC</em> (TMP1), did not contain any sequence inside the position associated with the donor plasmids (<a href="#pone-0001055-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001055-g006"><div class="img"><a name="pone-0001055-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001055.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Correlation between plasmid recombination frequencies and the number of Chi-like motifs within the 2 kb long DNA fragment.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>TF, transformation frequency.</p>
<span>doi:10.1371/journal.pone.0001055.g006</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>A whole genome analysis detected that the consensus DNA sequence, 5′ GCCGAA 3′, with a “c” located at the beginning and/or the end of the word was present on the <em>R. solanacearum</em> genome at a high frequency (average 2080+/−67 fold for the two best words with 7 letters and 751 fold for the consensus word). To test the uniformity of the repartition of the two consensus sequences, we used the classical Watson test of goodness of fit <a href="#pone.0001055-Watson1">[19]</a>. We found that the distribution of the 5′ GCCGAAc 3′ motif is non-uniform on both strands of the chromosome while the distribution of the 5′ cGCCGAA 3′ motif is uniform on the leading strand and non-uniform on the lagging strand. For the megaplasmid, on the other hand, distribution of both motifs was found to be uniform (<em>i.e.</em>, random) on the two strands (<a href="#pone-0001055-t003">Table 3</a>).</p>
<div class="figure" id="pone-0001055-t003"><div class="img"><a name="pone-0001055-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001055" data-uri="info:doi/10.1371/journal.pone.0001055.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001055.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001055.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001055.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001055.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001055.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Results of the Watson uniformity test for the two Chi motifs on the <em>R. solanacearum</em> chromosome and megaplasmid.</span></strong></p><span>doi:10.1371/journal.pone.0001055.t003</span></div><a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>These motifs could be Chi-like sequences with a putative role similar to that found for Chi sequences in <em>E. coli</em> for the attenuation of RecBCD exonuclease activity and the promotion of RecABCD-mediated homologous recombination <a href="#pone.0001055-Dixon1">[20]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Detection of laterally transferred genes in the <em>R. solanacearum</em> GMI1000 genome sequence</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Analysis of the complete sequence of the <em>R. solanacearum</em> strain GMI1000 with the phylogenetic tools presented earlier confirms that this bacterium has acquired numerous genes from other micro-organisms. For example, the <em>vsr</em>-like gene was possibly acquired from a α<em>-proteobacteria</em>, <em>Caulobacter crescentus</em> providing <em>R. solanacearum</em> with a mechanism for repairing mismatches in addition to MMR even if, in the case of <em>vsr</em>, the repair is limited to very short patches of DNA <a href="#pone.0001055-Lieb1">[21]</a>. In <em>E. coli</em>, MMR and VSP have complementary effects with efficiencies varying during the bacterial life cycle <a href="#pone.0001055-Hennecke1">[22]</a>. If <em>R. solanacearum</em> behaves similarly, then MMR could be expressed mainly when the pathogen multiplies actively in plant tissues and VSP during its less active life in the soil. If VSP contributes to increased genome stability under stringent environmental conditions, the overall fitness of the bacterium would be improved, thus, justifying the fixation of the <em>vsr</em> gene in the population.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The bio-informatics analysis also detected the putative acquisition of <em>pilA</em> and <em>mexC</em> from <em>Acinetobacter baumannii</em> and <em>Pseudomonas aeruginosa,</em> respectively. Both belong to the γ-class of proteobacteria. These genes, <em>pilA</em> and <em>mexC</em>, encode for cell envelope components and are involved in the formation of type IV fimbrial pilin signal peptide proteins and efflux pump antibiotic resistance proteins, respectively <a href="#pone.0001055-Li1">[23]</a>, <a href="#pone.0001055-Taha1">[24]</a>. Apparently, these new genes were sufficiently beneficial to <em>R. solanacearum</em> that they were fixed in its genome. Examples of other potentially transferred genes are RSc3252 and RSc1815 that originated from <em>Chlorobium tepidum</em> (green sulfur bacteria) and <em>Xanthomonas campestris</em> (γ-proteobacteria), respectively, but these genes have only putative functions. However, all genes detected as recently acquired are not necessarily beneficial to their new hosts, because either their presence resulted from co-transfer events and/or the selection process has not yet deleted the unnecessary DNA fragments. The half-life of unnecessary DNA in bacterial genomes is unknown.</p>


<h4><em>R. solanacearum</em> as a realistic and useful model to study intra and inter-species crosses</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The first requirement for HGT to occur is the contact between donor cells or their DNA and the recipient bacteria. In addition to soil and rhizosphere environments, the plant pathogen <em>R. solanacearum</em> colonizes numerous plants <a href="#pone.0001055-Hayward1">[25]</a> leading to the development of opportunistic soil bacteria in the degraded plant tissues <a href="#pone.0001055-Kay1">[26]</a>. The probability of transformation by these foreign genes is increased further by the competence development of <em>R. solanacearum in planta</em> as demonstrated by greenhouse experiments <a href="#pone.0001055-Bertolla1">[10]</a>. The ecology of different potential donor bacteria including <em>Xanthomonas campestris</em>, which is also a pathogen for plants, is compatible with that of <em>R. solanacearum</em>. The only marked exception would be the strictly anaerobic and obligate autotrophic <em>C. tepidum</em>, which is typically found in anoxic and sulfide-rich waters, mud, sediments, and microbial mats <a href="#pone.0001055-Eisen1">[27]</a>. Other bacteria including human pathogenic bacteria such as <em>A. baumannii</em> and <em>P. aeruginosa</em>, can live transiently in soils <a href="#pone.0001055-Stover1">[28]</a> and the <em>alpha</em>-purple aquatic bacterium <em>C. crescentus</em> is found in all types of water, including lakes, streams, sea water and waste water <a href="#pone.0001055-Gonin1">[29]</a>, <a href="#pone.0001055-Nierman1">[30]</a>. The possibility that most of these bacteria colonize alternate habitats, such as plant tissues, cannot be excluded although this was never reported.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Under <em>in planta</em> conditions as evidenced in experiments under greenhouse <a href="#pone.0001055-Kay1">[26]</a>, competent <em>R. solanacearum</em> cells are subjected to a flux of exogenous DNA, including plant and opportunistic bacterial DNA and its own DNA released by dying cells. Foreign DNA is degraded by the MMR and other systems except in the rare cases when illegitimate recombination mediates their integration into the genome. The fate of <em>R. solanacearum</em> DNA is totally different, inoculation experiments involving two <em>R. solanacearum</em> strains demonstrate that natural transformation mediated gene transfer occurs at high frequency under <em>in planta</em> conditions <a href="#pone.0001055-Bertolla1">[10]</a>. Sequence similarity between the incoming DNA and the recipient genome leads to integration of <em>R. solanacearum</em> genes by homologous (or homeologous in the case of more than one infecting strain) recombination at frequencies several orders of magnitude higher than for heterologous DNA. This combination of ecological, physiological and genetic conditions including extensive clonal multiplication, natural release of DNA, competence development, DNA uptake and genome integration, demonstrates the interest in <em>R. solanacearum</em> as a model for studying HGT regulation.</p>


<h4>Homologous recombination as the critical step that regulates gene acquisition</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Our hypothesis was that bacteria might regulate acquisition of homologous genetic material so that some genes accumulate genetic diversity while other genes maintain a higher stability level. The strategy used here was to test the frequency of homologous and homeologous recombination-mediated integration of a marker gene cloned into different positions of the <em>R. solanacearum</em> GMI1000 genome by using as recipients the same strain, GMI1000 and three other strains, CFBP2968, NCPPB332 and CFBP2957, that exhibited an overall genomic divergence relative to GMI1000 of 2, 19 and 31%, respectively <a href="#pone.0001055-Guidot1">[18]</a>. These <em>in vitro</em> transformation tests simulated the <em>in planta</em> situation where donors and recipients resulted from the clonal multiplication of bacterial cells belonging to one or more <em>R. solanacearum</em> strains.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Our results demonstrated that DNA fragments from various genomic positions of the same strain transformed the recipient strains at frequencies markedly different even under totally homologous DNA conditions. The <em>recA-</em> and <em>mutS-</em> gene containing positions were identified as natural transformation and certainly homologous recombination “hot spots” (see below). Previous studies suggest that the uptake of DNA by <em>R. solanacearum</em> would not require the presence of specific sequences to bind the cell wall for the donor DNA to be processed into the cell <a href="#pone.0001055-Bertolla2">[31]</a>. Although regulation of this uptake stage cannot be totally excluded, transformation frequency differences would more likely result from differences of recombination efficiency between the different DNA fragments. In addition, DNA uptake control might be linked in part to nutrient requirements <a href="#pone.0001055-Finkel1">[32]</a>. Whether the mechanism associated with DNA uptake defines the fate of the DNA once introduced into the cell is less clear, but the possibility that DNA uptake began as a nutrient uptake mechanism cannot be discounted <a href="#pone.0001055-Finkel1">[32]</a>. A recent bioinformatics study suggest that at least for some classes of short DNA sequences, DNA uptake is biased by sequence definition, which is not necessarily consistent with nutrient driven uptake <a href="#pone.0001055-Bakkali1">[33]</a>.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Transformation-recombination frequency decreased with sequence divergence for the three divergent strains, but the decrease was far from being identical for the various DNA positions tested. Interestingly, the DNA positions that were classified as “hot spots” with GMI1000 as recipient were also those for which transformation-recombination frequencies remained the highest and changed the least. Even with the least related strain (CFBP2957), transformation frequencies at the “hot spots” were only one order of magnitude lower than those for the homologous GMI1000, while frequencies decreased by 4 orders of magnitude for the DNA positions that transformed all strains at the lowest frequency.</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>Frequency differences are not apparently related to variable sizes of donor DNA fragments since transformation tests carried out with the plasmids (2kb in size) provided results consistent with those obtained with chromosomal fragments. PCR primers were designed to amplify a 2 kb long DNA fragment that was subsequently cloned into the plasmids. The resulting 1 kb long DNA fragments that flanked both sides of the marker gene were significantly longer than the minimal length necessary for efficient homologous recombination. Moreover, there was no theoretical limitation in the length of DNA fragments when chromosomal DNA with the marker gene inserted in the targeted position was used to transform the wild strain. Frequency variation could not be related to DNA physical or chemical parameters either. These results would indicate that differences in transfer frequency are only related to the nucleotide sequence of the DNA positions on which homologous recombination occurs. This would suggest that the genes present in a bacterial genome do not exhibit the same <em>sensu stricto</em> potential to be transferred even into a new isogenic host.</p>


<h4>Involvement of Chi-like sequences</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The sequences in the targeted genomic positions were analyzed and Chi-like (‘5-cGCCGAAc-3’) sequences were detected that might explain differences in homologous and homeologous recombination frequencies. The experimental results obtained with the four recipient strains were in general agreement, and thus, strengthened the hypothesis for the involvement of Chi-like sequences during recombination initiation. The highest transfer frequencies were found for fragments that contained more than two Chi-like sequences, thus, indicating that accumulation of these sequences could create “hot spots” for homologous recombination. Our results indicate that the Chi-like sequences are not distributed randomly in the <em>R. solanacearum</em> chromosome confirming what was already reported in other bacteria (e. g. <em>E. coli</em>, <em>Bacillus subtilis</em>, <em>Haemophilus influenzae</em> and <em>Lactococcus lactis)</em> <a href="#pone.0001055-ElKaroui1">[34]</a>. However, distribution of these motifs was found to be uniform on the <em>R. solanacearum</em> megaplasmid, a surprising (and unexplained) result that could be related to the involvement of replication mechanisms that differ between the 2 replicons.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>The interest in using recombinant plasmid-borne fragments as donor DNA was that the sequence analysis was restricted to the 1 kb long DNA fragments flanking the marker gene eliminating the possible influence of the Chi-like sequences located further upstream of the targeted fragments even if, on the other hand, the use of the entire genome extracted from recombinant strains was ecologically more realistic. Therefore, the reduced differences found between the lowest and the highest transformation frequencies with genomic borne fragments could be explained by the involvement of Chi-like sequences at some unknown distance upstream.</p>
<a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>In <em>E. coli,</em> Chi sequences are recombinational hotspots at which enzymes bind preferentially to repair DNA damaged by ionizing radiations or by the collapse of a replication fork when passing single-strand nicks <a href="#pone.0001055-Kowalczykowski1">[35]</a>. The ends of the broken DNA on double strands are processed by the multi-functional enzyme complex RecBCD involving successively a helicase activity to split the duplex into its component strands and a nuclease activity to digest them. At a Chi site, the nuclease activity is attenuated and the RecBCD loads RecA onto the 3′ tail of the DNA to initiate recombination. The foreign DNA acquired by HGT could be perceived by recipient cell as damaged DNA and be processed by the same enzymes <a href="#pone.0001055-Delmas1">[36]</a> with necessarily a critical role for Chi sequences as preferential sites to initiate recombination. Our results indicate that, in addition to a putative implication in the repair of endogenous damaged DNA like in <em>E. coli</em>, the Chi-like sequences in <em>R. solanacearum</em> (which possesses <em>addAB</em> genes having the same functions as <em>recBCD</em> in <em>E. coli</em>) could be key components of the adaptation potential by permitting the cell to regulate the gene acquisition process as already proposed in other naturally transformable bacteria such as <em>B. subtilis</em> and <em>H. influenzae</em> <a href="#pone.0001055-ElKaroui1">[34]</a>.</p>
<a id="article1.body1.sec3.sec4.p4" name="article1.body1.sec3.sec4.p4"></a><p>Chi-like sequences strongly limit the influence of sequence divergence, which usually decreases recombination efficiency dramatically <a href="#pone.0001055-Delmas1">[36]</a>, <a href="#pone.0001055-Gogarten1">[37]</a>. For instance, our results demonstrate that DNA exchange frequency for some DNA positions remains very high in spite of a significant overall genomic divergence between strains GMI1000 and CFBP2957 (up to 30%), a level that led to classifying these strains as two separate genomic species <a href="#pone.0001055-Palleroni1">[38]</a>. These results feed the debate on species boundaries in bacteria, on the strength of biological barriers to regulate DNA exchange, and confirm the difficulty to adapt a bacterial species concept that would be based on genomic coherence between members of a same species sharing an exclusive common gene pool <a href="#pone.0001055-Doolittle1">[39]</a>.</p>


<h4>Transformation “hot spots” in <em>R. solanacearum</em> and recombination potential of mobile elements and recently acquired genes</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Two of the main transformation “hot spots” detected in this study were the genomic positions (TCP3 and TCP7) containing the <em>recA</em> and <em>mutS</em> genes, which are involved in DNA repair and recombination. This could be justified by the need to maintain stability and integrity in DNA positions containing important housekeeping genes <a href="#pone.0001055-Torsvik1">[7]</a>. By analogy to <em>E. coli</em>, damaged endogenous DNA reparation efficiency is certainly increased by presence of Chi-like sequences to initiate recombination. However, genomic stability of these positions could also benefit from their spread at high frequency among bacteria that reduces the risk of genetic drift by point mutations in separate lineages. In addition, a recombination “hot spot” in the <em>mutS</em> gene is in agreement with the hypothesis involving HGT as a mechanism for <em>mutS</em> negative mutators to re-acquire a functional <em>mutS</em> copy to return to a more stable wild type phenotype <a href="#pone.0001055-Brown1">[40]</a>, <a href="#pone.0001055-Denamur1">[41]</a>.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>Surprisingly, the genomic positions containing mobile elements, such as insertion sequences, prophages and transposons that have developed specific mechanisms to displace from place to place within and among genomes exhibited a transformation potential significantly lower than the hot spots that carry <em>mutS</em> and <em>recA</em> genes. Our study also included genomic positions with genes acquired from other phyletically remote bacteria by HGT. These positions did not exhibit any copy of the specific Chi-like sequences detected in the recombination “hot spots”. Moreover, transformation frequency of the fragment containing the <em>vsr</em> gene was respectively 13 and 8 times lower than for those containing <em>recA</em> or <em>mutS</em> genes indicating that the corresponding positions should not be considered as “hot spots” for transformation. These data would indicate that acquisition of foreign genes might not be regulated as could be <em>R. solanacearum</em> genes but would rather result from the combination of several events happening mainly by chance and at very low frequency including the uptake of exogenous DNA and its integration and/or by rearrangement in the genome post-HGT.</p>


<h4>The two sides of HGT in bacteria</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>According to bioinformatics analysis of genome sequences (this study) and inoculation experiments in plant tissues <a href="#pone.0001055-Bertolla1">[10]</a> <em>R. solanacearum</em> like other bacteria seems to use two complementary HGT-based strategies to optimize adaptation. The first one is based on acquisition at high frequency of DNA from more or less closely related cells. This type of HGT would permit on one hand to maintain stability and integrity in some important DNA positions, a constant DNA homogenization reducing the risk of genetic drift <a href="#pone.0001055-Torsvik1">[7]</a>, and on another hand to spread potentially beneficial mutations efficiently among the population. However, our data demonstrate that the various DNA positions are transferred at significantly different frequencies indicating a possible regulation mechanism. As for other functions in bacteria, the fundamental evolutionary question is whether this property might have been specifically selected to increase adaptation potential or if it might be the side effect of cellular mechanisms in charge of DNA repair. Whatever the response (about the evolutionary process), bacteria in the open environment are confronted with genes that do not transfer at the same frequency and thus with evolutionary implications that cannot yet be precisely evaluated. The high frequency with which the preferentially transferred DNA positions are transformed in other strains indicates that such HGT events must have a strong impact on genome evolution and must significantly contribute to the adaptation potential of the bacteria.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>The second adaptive strategy of bacteria is to acquire genes from unrelated bacteria. The <em>in silico</em> analyses of the <em>R. solanacearum</em> genome sequence detected genes that were laterally transferred from a wide range of remotely related bacteria. Their fixation suggests that they also contributed to increase the adaptation potential of their new host. However, our experimental transformation results do not indicate that these laterally acquired DNA positions belong to the transformation-recombination “hot spots” detected for other typical <em>R. solanacearum</em> genes. This might mean that their acquisition resulted from a hypothetical contact with a donor bacteria and a successful integration by an extremely rare illegitimate recombination event. The likelihood that this was directly related to the active Chi-like sequences based mechanism, which leads to some other genes to be exchanged at high frequency, seems to be low. However, successful transfer to genomic “hot spots” and subsequent mobility to more stable genomic positions through mobile elements such as insertion sequences cannot be disproved.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4><em>Ralstonia solanacearum</em> strains</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The four strains used in this study are classified in the <em>R. solanacearum</em> species complex and belong to phylotype I (GMI1000, CFBP2968), phylotype II (CFBP2957) and phylotype III (NCPPB332) <a href="#pone.0001055-Guidot1">[18]</a>. These strains that exhibit sensitivity to ampicillin, kanamycin and gentamycin were cultured at 28°C in the complete B medium <a href="#pone.0001055-Boucher1">[42]</a> and exhibit the same natural transformation frequency when transformed by their own DNA (results not shown).</p>


<h4>Phylogenetic bioinformatics analysis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For the phylogenetic inference, 2039 homologous gene families containing at least one sequence from <em>R. solanacearum</em> GMI1000 and from other β-proteobacteria were extracted from the HOGENOM database (<a href="http://pbil.univ-lyon1.fr/databases/hogenom.html">http://pbil.univ-lyon1.fr/databases/hoge​nom.html</a>). After excluding eukaryotic sequences, multiple alignments of the families were computed using MUSCLE <a href="#pone.0001055-Edgar1">[43]</a>, with all default parameters. These alignments were then filtered with GBLOCKS <a href="#pone.0001055-Castresana1">[44]</a> in order to keep only their reliable parts. Using these filtered alignments, we kept only families containing a number of sites equal at least to 1.5× the number of taxa in the families. Only 1139 families remained after this final selection. Phylogenetic trees were computed (with 100 bootstrap replicates) on the remaining alignments with the fast Maximum-Likelihood method implemented in PHYML <a href="#pone.0001055-Guindon1">[45]</a>.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Under PhyML, the WAG amino acid substitution model <a href="#pone.0001055-Whelan1">[46]</a> was used, and across-site rate variation was modelled by a Gamma distribution with four classes of substitution rates. Estimation of the Alpha parameter for Gamma distributions was carried out by PhyML. Trees were then manually checked to detect those in which <em>R. solanacearum</em> was not grouped with the other(s) β-proteobacteria, this with a Shimodaira-Hasegawa likelihood ratio test≥95% <a href="#pone.0001055-Shimodaira1">[47]</a>.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Phylogenetic trees computations were performed on the IN2P3 Linux cluster containing more than 1300 CPUs.</p>


<h4>Plasmid construction</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Eighteen oligonucleotide pairs (<a href="#pone.0001055.s002">Table S2</a>) were designed according to the complete nucleotide sequence of <em>R. solanacearum</em> in order to amplify a 2 kb long DNA fragment for each selected position by PCR. PCR primers were designed by using the OLIGO 5.1 software (National Biosciences, Inc. NBI) applied to the complete GMI1000 sequence. The resulting PCR products were ligated to one of the following plasmids, pUC19, pBluecript or pCR® 2.1-TOPO vector or pGEMT®-T/Easy vector (InVitrogen France, Promega, France) depending on the restriction sites available for further restriction cleavage and cloned in <em>Escherichia coli</em> strain (DH5α) according to the manufacturer instructions. Recombinant plasmids were extracted and purified with the QIAprep Mini-prep Kit (Qiagen SA, Germany) and re-suspended in sterile purified water. Each plasmid was digested with the appropriate restriction enzyme (<a href="#pone.0001055.s003">Table S3</a>) that cut the insert once, approximately in the middle in order to clone the <em>aac</em>C3-IV gene conferring resistance to gentamycin (Gm<sup>R</sup>) <a href="#pone.0001055-Brau1">[48]</a>. The resulting plasmids were extracted and the construction was verified by electrophoresis on agarose gel after digestion with the appropriate restriction enzymes. PCR products that were initially cloned in the pGEM-T, pUC19 and pBluescript vectors were amplified again with the same initial primers and recombinant plasmids as template before the chimeric construction was cloned in the pCR® 2.1-TOPO vector. The 18 pCR® 2.1-TOPO-derivative resulting plasmids contained all the same vector background with the <em>bla</em> and <em>nptII</em> genes conferring resistance to ampicillin (Amp<sup>R</sup>) and kanamycin (Kn<sup>R</sup>) respectively and each contained a specific <em>R. solanacearum</em> 2 kb genomic DNA fragment in which the <em>aacC3-IV</em> gene conferring resistance to gentamycin (Gm<sup>R</sup>) had been inserted, approximately in the middle.</p>


<h4>Preparation of transforming DNA</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The plasmids were extracted and purified by using the QIAprep Mini-prep Kit before they were digested with <em>ScaI</em>. The only exception was plasmid pTCP9 that was treated by <em>CpoI</em> because of presence of a <em>ScaI</em> site in the insert (<a href="#pone.0001055.s003">Table S3</a>). These 2 enzymes <em>ScaI</em> and <em>CpoI</em> cut only once in the <em>bla</em> gene and the <em>nptII</em> gene respectively, linearizing the plasmid from the vector background without affecting the <em>R. solanacearum</em> fragments and the marker gene. Restricted plasmid DNA was subsequently purified with the GFX™ PCR DNA and gel purification kits (Amersham Biosciences Germany) before used in transformation tests. <em>R. solanacearum</em> genomic DNA used in transformation was extracted from gentamycin resistant and kanamycin sensitive recombinant strains after they were transformed with the recombinant plasmid set presented above according to the available protocol <a href="#pone.0001055-Bertolla2">[31]</a>.</p>


<h4><em>R. solanacearum</em> natural transformation</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Natural transformation of the <em>R. solanacearum</em> strains was carried out according to the procedure described by Bertolla et al. <a href="#pone.0001055-Bertolla2">[31]</a>. Briefly, cells of <em>R. solanacearum</em> were grown in minimal medium (MM) to an O.D. <sub>600 nm</sub> = 0.8 (about 5×10<sup>8</sup> cells ml<sup>−1</sup>). Fifty micro-liters of this cell suspension were incubated with either 100 ng of plasmid DNA or 400 ng of genomic DNA on polycarbonate membranes (Millipore, Ireland) deposited on the surface of solid MM (MMG) medium and incubated for 48 h at 30°C. Bacterial cells were then harvested from the membrane surface and suspended again in 5 ml of sterile water. A 500 µl aliquot was used to inoculate rich BG agar medium plates containing respectively gentamycin (12.5 µg ml<sup>−1</sup>) (Euromedex, France) in transformation tests with genomic or plasmid DNA or both gentamycin (12.5 µg ml<sup>−1</sup>) and kanamycin (25 µg ml<sup>−1</sup>) (Euromedex, France) in transformation tests with plasmid DNA. The recipient population was enumerated by plating appropriate dilutions on the BG medium without any antibiotics. Recombinant <em>R. solanacearum</em> colonies that exhibited resistance to gentamycin but sensitivity to kanamycin were those in which a double cross over event replaced the wild type gene by the chimeric construction while resistance to kanamycin indicated that the plasmid was totally integrated following a single cross over event (<a href="#pone-0001055-g001">Figure 1</a>). Integration of the cassette by single or double cross over events was verified by PCR for 3 randomly selected clones for each construction. Controls included transformation tests carried out without DNA that allowed determining spontaneous mutation frequency. Other control experiments used the plasmid DNA from non recombinant TOPO vector or the DNA from the wild type strain GMI1000 as transforming DNA. Each filter experiment was done at least in triplicate and all calculated transformation frequencies are given as the mean value. A statistical <em>t</em> student test was performed to evaluate significance of differing DNA source used.</p>


<h4>Detection of Chi –like sequences</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The 2 kb DNA sequences corresponding to <em>R. solanacearum</em> DNA fragment cloned into plasmids were analyzed by using the R software (seqinR <a href="#pone.0001055-Charif1">[49]</a>) to detect the presence of the longest repeated motifs, which might significantly correlate to the recombination frequencies. The occurrence of the words in <em>R. solanacearum</em> whole genome was determined using the Fuzznuc online program (<a href="http://bioweb.pasteur.fr/docs/EMBOSS/fuzznuc.html">http://bioweb.pasteur.fr/docs/EMBOSS/fuz​znuc.html</a>) which belongs to the EMBOSS package.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001055"></div><a name="pone.0001055.s001" id="pone.0001055.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001055.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001055.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0001055.s002" id="pone.0001055.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001055.s002">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001055.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0001055.s003" id="pone.0001055.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001055.s003">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001055.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.08 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Dominique Bernillon for technical assistance and to Ivan Matic and Dominique Schneider for a critical reading of the ms.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: GP PS SF FB TV. Performed the experiments: SF AM FB. Analyzed the data: GP PS SF AM AC LG TV. Contributed reagents/materials/analysis tools: GP PS SF AM FB AC LG TV. Wrote the paper: GP PS SF AM FB AC LG TV.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001055-Nakamura1" id="pone.0001055-Nakamura1"></a>Nakamura Y, Itoh T, Matsuda H, Gojobori T (2004) Biased biological functions of horizontally transferred genes in prokaryotic genomes. Nature Genet  36: 760–766.  <ul class="find" data-citedArticleID="1006958" data-doi="10.1038/ng1381"><li><a href="http://dx.doi.org/10.1038/ng1381" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biased+biological+functions+of+horizontally+transferred+genes+in+prokaryotic+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biased+biological+functions+of+horizontally+transferred+genes+in+prokaryotic+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001055-Lawrence1" id="pone.0001055-Lawrence1"></a>Lawrence JG, Ochman H (2002) Reconciling the many faces of lateral gene transfer. Trends Microbiol  10: 1–4.  <ul class="find" data-citedArticleID="1006946" data-doi="10.1016/s0966-842x(01)02282-x"><li><a href="http://dx.doi.org/10.1016/s0966-842x(01)02282-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconciling+the+many+faces+of+lateral+gene+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconciling+the+many+faces+of+lateral+gene+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001055-Paget1" id="pone.0001055-Paget1"></a>Paget E, Simonet P (1994) On the track of natural transformation in soil. FEMS Microb Ecol  15: 109–117.  <ul class="find" data-citedArticleID="1006962" data-doi="10.1111/j.1574-6941.1994.tb00235.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6941.1994.tb00235.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+track+of+natural+transformation+in+soil." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+track+of+natural+transformation+in+soil.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001055-Kobayashi1" id="pone.0001055-Kobayashi1"></a>Kobayashi I (2001) Behavior of restriction-modification systems as selfish mobile elements and their impact on genome evolution. Nucleic Acids Res  29: 3742–3756.  <ul class="find" data-citedArticleID="1006942" data-doi="10.1093/nar/29.18.3742"><li><a href="http://dx.doi.org/10.1093/nar/29.18.3742" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Behavior+of+restriction-modification+systems+as+selfish+mobile+elements+and+their+impact+on+genome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Behavior+of+restriction-modification+systems+as+selfish+mobile+elements+and+their+impact+on+genome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001055-Matic1" id="pone.0001055-Matic1"></a>Matic I, Taddei F, Radman M (1996) Genetic barriers among bacteria. Trends Microbiol  4: 69–72.  <ul class="find" data-citedArticleID="1006954" data-doi="10.1016/0966-842x(96)81514-9"><li><a href="http://dx.doi.org/10.1016/0966-842x(96)81514-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+barriers+among+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+barriers+among+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001055-Humbert1" id="pone.0001055-Humbert1"></a>Humbert O, Prudhomme M, Hakenbeck R, Dowson CG, Claverys JP (1995) Homeologous recombination and mismatch repair during transformation in <em>Streptococcus pneumoniae</em>: saturation of the Hex mismatch repair system. Proc Natl Acad Sci U S A  92: 9052–9056.  <ul class="find" data-citedArticleID="1006938" data-doi="10.1073/pnas.92.20.9052"><li><a href="http://dx.doi.org/10.1073/pnas.92.20.9052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homeologous+recombination+and+mismatch+repair+during+transformation+in+Streptococcus+pneumoniae%3A+saturation+of+the+Hex+mismatch+repair+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homeologous+recombination+and+mismatch+repair+during+transformation+in+Streptococcus+pneumoniae%3A+saturation+of+the+Hex+mismatch+repair+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001055-Torsvik1" id="pone.0001055-Torsvik1"></a>Torsvik V, Ovreas L, Thingstad TF (2002) Prokaryotic diversity–magnitude, dynamics, and controlling factors. Science  296: 1064–1066.  <ul class="find" data-citedArticleID="1006980" data-doi="10.1126/science.1071698"><li><a href="http://dx.doi.org/10.1126/science.1071698" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prokaryotic+diversity%E2%80%93magnitude%2C+dynamics%2C+and+controlling+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prokaryotic+diversity%E2%80%93magnitude%2C+dynamics%2C+and+controlling+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001055-Ragan1" id="pone.0001055-Ragan1"></a>Ragan MA (2001) Detection of lateral gene transfer among microbial genomes. Curr Opin Genet Dev  11: 620–626.  <ul class="find" data-citedArticleID="1006966" data-doi="10.1016/s0959-437x(00)00244-6"><li><a href="http://dx.doi.org/10.1016/s0959-437x(00)00244-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+lateral+gene+transfer+among+microbial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+lateral+gene+transfer+among+microbial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001055-Salanoubat1" id="pone.0001055-Salanoubat1"></a>Salanoubat M, Genin S, Artiguenave F, Gouzy J, Mangenot S, et al.  (2002) Genome sequence of the plant pathogen <em>Ralstonia solanacearum</em>. Nature  415: 497–502.  <ul class="find" data-citedArticleID="1006970" data-doi="10.1038/415497a"><li><a href="http://dx.doi.org/10.1038/415497a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+the+plant+pathogen+Ralstonia+solanacearum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+the+plant+pathogen+Ralstonia+solanacearum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001055-Bertolla1" id="pone.0001055-Bertolla1"></a>Bertolla F, Frostegård A, Brito B, Nesme X, Simonet P (1999) During infection of its host, the plant pathogen <em>Ralstonia solanacearum</em> naturally develops a state of competence and exchanges genetic material. molecular plant-microbe interact  12: 467–672.  <ul class="find" data-citedArticleID="1006890" data-doi="10.1094/mpmi.1999.12.5.467"><li><a href="http://dx.doi.org/10.1094/mpmi.1999.12.5.467" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=During+infection+of+its+host%2C+the+plant+pathogen+Ralstonia+solanacearum+naturally+develops+a+state+of+competence+and+exchanges+genetic+material." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22During+infection+of+its+host%2C+the+plant+pathogen+Ralstonia+solanacearum+naturally+develops+a+state+of+competence+and+exchanges+genetic+material.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001055-Gould1" id="pone.0001055-Gould1"></a>Gould SJ, Lewontin RC (1979) The spandrels of San Marco and the Panglossian paradigm: a critique of the adaptationist programme. Proc R Soc Lond B Biol Sci  205: 581–598.  <ul class="find" data-citedArticleID="1006928" data-doi="10.1098/rspb.1979.0086"><li><a href="http://dx.doi.org/10.1098/rspb.1979.0086" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+spandrels+of+San+Marco+and+the+Panglossian+paradigm%3A+a+critique+of+the+adaptationist+programme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+spandrels+of+San+Marco+and+the+Panglossian+paradigm%3A+a+critique+of+the+adaptationist+programme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001055-Metzgar1" id="pone.0001055-Metzgar1"></a>Metzgar D, Wills C (2000) Evidence for the adaptive evolution of mutation rates. Cell  101: 581–584.  <ul class="find" data-citedArticleID="1006956" data-doi="10.1016/s0092-8674(00)80869-7"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80869-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+the+adaptive+evolution+of+mutation+rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+the+adaptive+evolution+of+mutation+rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001055-Daubin1" id="pone.0001055-Daubin1"></a>Daubin V, Lerat E, Perriere G (2003) The source of laterally transferred genes in bacterial genomes. Genome Biol  4: R57.  <ul class="find" data-citedArticleID="1006904" data-doi="10.1186/gb-2003-4-9-r57"><li><a href="http://dx.doi.org/10.1186/gb-2003-4-9-r57" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+source+of+laterally+transferred+genes+in+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+source+of+laterally+transferred+genes+in+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001055-Daubin2" id="pone.0001055-Daubin2"></a>Daubin V, Perriere G (2003) G+C3 Structuring along the genome: A common feature in prokaryotes. Mol Biol Evol  20: 471–483.  <ul class="find" data-citedArticleID="1006906" data-doi="10.1093/molbev/msg022"><li><a href="http://dx.doi.org/10.1093/molbev/msg022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=G%2BC3+Structuring+along+the+genome%3A+A+common+feature+in+prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22G%2BC3+Structuring+along+the+genome%3A+A+common+feature+in+prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001055-Lawrence2" id="pone.0001055-Lawrence2"></a>Lawrence JG, Ochman H (1998) Molecular archaeology of the <em>Escherichia coli</em> genome. Proc Natl Acad Sci USA  95: 9413–9417.  <ul class="find" data-citedArticleID="1006948" data-doi="10.1073/pnas.95.16.9413"><li><a href="http://dx.doi.org/10.1073/pnas.95.16.9413" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+archaeology+of+the+Escherichia+coli+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+archaeology+of+the+Escherichia+coli+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001055-Rocha1" id="pone.0001055-Rocha1"></a>Rocha EP, Danchin A (2002) Base composition bias might result from competition for metabolic resources. Trends Genet  18: 291–294.  <ul class="find" data-citedArticleID="1006968" data-doi="10.1016/s0168-9525(02)02690-2"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02690-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base+composition+bias+might+result+from+competition+for+metabolic+resources." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base+composition+bias+might+result+from+competition+for+metabolic+resources.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001055-Sharp1" id="pone.0001055-Sharp1"></a>Sharp PM, Li WH (1987) The codon Adaptation Index: a measure of directional synonymous codon usage bias, and its potential applications. Nucl Acids Res  15: 1281–1295.  <ul class="find" data-citedArticleID="1006972" data-doi="10.1093/nar/15.3.1281"><li><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+codon+Adaptation+Index%3A+a+measure+of+directional+synonymous+codon+usage+bias%2C+and+its+potential+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+codon+Adaptation+Index%3A+a+measure+of+directional+synonymous+codon+usage+bias%2C+and+its+potential+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001055-Guidot1" id="pone.0001055-Guidot1"></a>Guidot A, Prior P, Schoenfeld J, Carrere S, Genin S, et al.  (2007) Genomic structure and phylogeny of the plant pathogen <em>Ralstonia solanacearum</em> inferred from gene distribution analysis. J Bacteriol  189: 377–387.  <ul class="find" data-citedArticleID="1006930" data-doi="10.1128/jb.00999-06"><li><a href="http://dx.doi.org/10.1128/jb.00999-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+structure+and+phylogeny+of+the+plant+pathogen+Ralstonia+solanacearum+inferred+from+gene+distribution+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+structure+and+phylogeny+of+the+plant+pathogen+Ralstonia+solanacearum+inferred+from+gene+distribution+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001055-Watson1" id="pone.0001055-Watson1"></a>Watson GS (1961) Goodness-of-fit tests on a circle. Biometrika  48: 109–114.  <ul class="find" data-citedArticleID="1006982" data-doi="10.1093/biomet/48.1-2.109"><li><a href="http://dx.doi.org/10.1093/biomet/48.1-2.109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Goodness-of-fit+tests+on+a+circle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Goodness-of-fit+tests+on+a+circle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001055-Dixon1" id="pone.0001055-Dixon1"></a>Dixon DA, Kowalczykowski SC (1995) Role of the <em>Escherichia coli</em> recombination hotspot, Chi, in RecABCD-dependent homologous pairing. J Biol Chem  270: 16360–16370.  <ul class="find" data-citedArticleID="1006912" data-doi="10.1074/jbc.270.27.16360"><li><a href="http://dx.doi.org/10.1074/jbc.270.27.16360" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+Escherichia+coli+recombination+hotspot%2C+Chi%2C+in+RecABCD-dependent+homologous+pairing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+Escherichia+coli+recombination+hotspot%2C+Chi%2C+in+RecABCD-dependent+homologous+pairing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001055-Lieb1" id="pone.0001055-Lieb1"></a>Lieb M, Bhagwat AS (1996) Very short patch repair: reducing the cost of cytosine methylation. Mol Microbiol  20: 467–473.  <ul class="find" data-citedArticleID="1006952" data-doi="10.1046/j.1365-2958.1996.5291066.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1996.5291066.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Very+short+patch+repair%3A+reducing+the+cost+of+cytosine+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Very+short+patch+repair%3A+reducing+the+cost+of+cytosine+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001055-Hennecke1" id="pone.0001055-Hennecke1"></a>Hennecke F, Kolmar H, Brundl K, Fritz HJ (1991) The vsr gene product of E. coli K-12 is a strand- and sequence-specific DNA mismatch endonuclease. Nature  353: 776–778.  <ul class="find" data-citedArticleID="1006936" data-doi="10.1038/353776a0"><li><a href="http://dx.doi.org/10.1038/353776a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+vsr+gene+product+of+E.+coli+K-12+is+a+strand-+and+sequence-specific+DNA+mismatch+endonuclease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+vsr+gene+product+of+E.+coli+K-12+is+a+strand-+and+sequence-specific+DNA+mismatch+endonuclease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001055-Li1" id="pone.0001055-Li1"></a>Li X-Z, Zhang L, Poole K (1998) Role of the Multidrug Efflux Systems of <em>Pseudomonas aeruginosa</em> in Organic Solvent Tolerance. J Bacteriol  180: 2987–2991.  <ul class="find" data-citedArticleID="1006950"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Role+of+the+Multidrug+Efflux+Systems+of+Pseudomonas+aeruginosa+in+Organic+Solvent+Tolerance.&amp;auth=&amp;atitle=Role+of+the+Multidrug+Efflux+Systems+of+Pseudomonas+aeruginosa+in+Organic+Solvent+Tolerance." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+Multidrug+Efflux+Systems+of+Pseudomonas+aeruginosa+in+Organic+Solvent+Tolerance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+Multidrug+Efflux+Systems+of+Pseudomonas+aeruginosa+in+Organic+Solvent+Tolerance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001055-Taha1" id="pone.0001055-Taha1"></a>Taha MK, Giorgini D, Nassif X (1996) The <em>pilA</em> regulatory gene modulates the pilus-mediated adhesion of <em>Neisseria meningitidis</em> by controlling the transcription of pilC1. Mol Microbiol  19: 1073–1084.  <ul class="find" data-citedArticleID="1006978" data-doi="10.1046/j.1365-2958.1996.448979.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1996.448979.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+pilA+regulatory+gene+modulates+the+pilus-mediated+adhesion+of+Neisseria+meningitidis+by+controlling+the+transcription+of+pilC1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+pilA+regulatory+gene+modulates+the+pilus-mediated+adhesion+of+Neisseria+meningitidis+by+controlling+the+transcription+of+pilC1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001055-Hayward1" id="pone.0001055-Hayward1"></a>Hayward AC (1994)  The hosts of <em>Pseudomonas solanacearum</em>. In: Hayward AC, Hartman GL, editors. Bacterial Wilt: The Disease and its causative agent, <em>Pseudomonas solanacearum</em>. Wallingford, UK: CAB INTERNATIONAL. pp. 9–24.  <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0001055-Kay1" id="pone.0001055-Kay1"></a>Kay E, Bertolla F, Vogel TM, Simonet P (2002) Opportunistic colonization of <em>Ralstonia solanacearum</em>-infected plants by <em>Acinetobacter sp</em>. and its natural competence development. Microb Ecol  43: 291–297.  <ul class="find" data-citedArticleID="1006940" data-doi="10.1007/s00248-002-2007-y"><li><a href="http://dx.doi.org/10.1007/s00248-002-2007-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Opportunistic+colonization+of+Ralstonia+solanacearum-infected+plants+by+Acinetobacter+sp.+and+its+natural+competence+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Opportunistic+colonization+of+Ralstonia+solanacearum-infected+plants+by+Acinetobacter+sp.+and+its+natural+competence+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001055-Eisen1" id="pone.0001055-Eisen1"></a>Eisen JA, Nelson KE, Paulsen IT, Heidelberg JF, Wu M, et al.  (2002) The complete genome sequence of <em>Chlorobium tepidum</em> TLS, a photosynthetic, anaerobic, green-sulfur bacterium. Proc Natl Acad Sci USA  99: 9509–9514.  <ul class="find" data-citedArticleID="1006918" data-doi="10.1073/pnas.132181499"><li><a href="http://dx.doi.org/10.1073/pnas.132181499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+genome+sequence+of+Chlorobium+tepidum+TLS%2C+a+photosynthetic%2C+anaerobic%2C+green-sulfur+bacterium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+genome+sequence+of+Chlorobium+tepidum+TLS%2C+a+photosynthetic%2C+anaerobic%2C+green-sulfur+bacterium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001055-Stover1" id="pone.0001055-Stover1"></a>Stover CK, Pham XQ, Erwin AL, Mizoguchi SD, Warrener P, et al.  (2000) Complete genome sequence of <em>Pseudomonas aeruginosa</em> PA01, an opportunistic pathogen. Nature  406: 959–964.  <ul class="find" data-citedArticleID="1006976"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Complete+genome+sequence+of+Pseudomonas+aeruginosa+PA01%2C+an+opportunistic+pathogen.&amp;auth=&amp;atitle=Complete+genome+sequence+of+Pseudomonas+aeruginosa+PA01%2C+an+opportunistic+pathogen." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Pseudomonas+aeruginosa+PA01%2C+an+opportunistic+pathogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Pseudomonas+aeruginosa+PA01%2C+an+opportunistic+pathogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001055-Gonin1" id="pone.0001055-Gonin1"></a>Gonin M, Quardokus EM, O'Donnol D, Maddock J, Brun YV (2000) Regulation of stalk elongation by phosphate in <em>Caulobacter crescentus</em>. J Bacteriol  182: 337–347.  <ul class="find" data-citedArticleID="1006926" data-doi="10.1128/jb.182.2.337-347.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.2.337-347.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+stalk+elongation+by+phosphate+in+Caulobacter+crescentus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+stalk+elongation+by+phosphate+in+Caulobacter+crescentus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001055-Nierman1" id="pone.0001055-Nierman1"></a>Nierman WC, Feldblyum TV, Laub MT, Paulsen IT, Nelson KE, et al.  (2001) Complete genome sequence of <em>Caulobacter crescentus</em>. Proc Natl Acad Sci USA  98: 4136–4141.  <ul class="find" data-citedArticleID="1006960" data-doi="10.1073/pnas.061029298"><li><a href="http://dx.doi.org/10.1073/pnas.061029298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Caulobacter+crescentus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Caulobacter+crescentus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001055-Bertolla2" id="pone.0001055-Bertolla2"></a>Bertolla F, Van Gijsegem F, Nesme X, Simonet P (1997) Conditions for natural transformation of <em>Ralstonia solanacearum</em>. Appl Environ Microbiol  63: 4965–4968.  <ul class="find" data-citedArticleID="1006892"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Conditions+for+natural+transformation+of+Ralstonia+solanacearum.&amp;auth=&amp;atitle=Conditions+for+natural+transformation+of+Ralstonia+solanacearum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conditions+for+natural+transformation+of+Ralstonia+solanacearum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conditions+for+natural+transformation+of+Ralstonia+solanacearum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001055-Finkel1" id="pone.0001055-Finkel1"></a>Finkel SE, Kolter R (2001) DNA as a nutrient: novel role for bacterial competence gene homologs. J Bacteriol  183: 6288–6293.  <ul class="find" data-citedArticleID="1006922" data-doi="10.1128/jb.183.21.6288-6293.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.21.6288-6293.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+as+a+nutrient%3A+novel+role+for+bacterial+competence+gene+homologs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+as+a+nutrient%3A+novel+role+for+bacterial+competence+gene+homologs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001055-Bakkali1" id="pone.0001055-Bakkali1"></a>Bakkali M (2007) Genome dynamics of short oligonucleotides: The example of bacterial DNA uptake enhancing sequences. PLoS ONE  e741: 1–15.  <ul class="find" data-citedArticleID="1006888" data-doi="10.1371/journal.pone.0000741"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000741" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+dynamics+of+short+oligonucleotides%3A+The+example+of+bacterial+DNA+uptake+enhancing+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+dynamics+of+short+oligonucleotides%3A+The+example+of+bacterial+DNA+uptake+enhancing+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001055-ElKaroui1" id="pone.0001055-ElKaroui1"></a>El Karoui M, Biaudet V, Schbath S, Gruss A (1999) Characteristics of Chi distribution on different bacterial genomes. Res Microbiol  150: 579–587.  <ul class="find" data-citedArticleID="1006920" data-doi="10.1016/s0923-2508(99)00132-1"><li><a href="http://dx.doi.org/10.1016/s0923-2508(99)00132-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characteristics+of+Chi+distribution+on+different+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characteristics+of+Chi+distribution+on+different+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001055-Kowalczykowski1" id="pone.0001055-Kowalczykowski1"></a>Kowalczykowski SC, Dixon DA, Eggleston AK, Lauder SD, Rehrauer WM (1994) Biochemistry of homologous recombination in <em>Escherichia coli</em>. Microbiol Rev  58: 401–465.  <ul class="find" data-citedArticleID="1006944"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Biochemistry+of+homologous+recombination+in+Escherichia+coli.&amp;auth=&amp;atitle=Biochemistry+of+homologous+recombination+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemistry+of+homologous+recombination+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemistry+of+homologous+recombination+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001055-Delmas1" id="pone.0001055-Delmas1"></a>Delmas S, Matic I (2005) Cellular response to horizontally transferred DNA in <em>Escherichia coli</em> is tuned by DNA repair systems. DNA Repair (Amst)  4: 221–229.  <ul class="find" data-citedArticleID="1006908" data-doi="10.1016/j.dnarep.2004.09.008"><li><a href="http://dx.doi.org/10.1016/j.dnarep.2004.09.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cellular+response+to+horizontally+transferred+DNA+in+Escherichia+coli+is+tuned+by+DNA+repair+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cellular+response+to+horizontally+transferred+DNA+in+Escherichia+coli+is+tuned+by+DNA+repair+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001055-Gogarten1" id="pone.0001055-Gogarten1"></a>Gogarten JP, Townsend JP (2005) Horizontal gene transfer, genome innovation and evolution. Nat Rev Microbiol  3: 679–687.  <ul class="find" data-citedArticleID="1006924" data-doi="10.1038/nrmicro1204"><li><a href="http://dx.doi.org/10.1038/nrmicro1204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Horizontal+gene+transfer%2C+genome+innovation+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Horizontal+gene+transfer%2C+genome+innovation+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001055-Palleroni1" id="pone.0001055-Palleroni1"></a>Palleroni NJ, Doudoroff M (1971) Phenotypic characterization and deoxyribonucleic acid homologies of <em>Pseudomonas solanacearum</em>. J Bacteriol  107: 690–696.  <ul class="find" data-citedArticleID="1006964" data-doi="10.1099/00207713-26-4-421"><li><a href="http://dx.doi.org/10.1099/00207713-26-4-421" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+characterization+and+deoxyribonucleic+acid+homologies+of+Pseudomonas+solanacearum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+characterization+and+deoxyribonucleic+acid+homologies+of+Pseudomonas+solanacearum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001055-Doolittle1" id="pone.0001055-Doolittle1"></a>Doolittle WF, Papke RT (2006) Genomics and the bacterial species problem. Genome Biol  7: 116.  <ul class="find" data-citedArticleID="1006914" data-doi="10.1186/gb-2006-7-9-116"><li><a href="http://dx.doi.org/10.1186/gb-2006-7-9-116" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomics+and+the+bacterial+species+problem." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomics+and+the+bacterial+species+problem.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001055-Brown1" id="pone.0001055-Brown1"></a>Brown EW, LeClerc JE, Li B, Payne WL, Cebula TA (2001) Phylogenetic evidence for horizontal transfer of <em>mutS</em> alleles among naturally occurring <em>Escherichia coli</em> strains. J Bacteriol  183: 1631–1644.  <ul class="find" data-citedArticleID="1006898" data-doi="10.1128/jb.183.5.1631-1644.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.5.1631-1644.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+evidence+for+horizontal+transfer+of+mutS+alleles+among+naturally+occurring+Escherichia+coli+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+evidence+for+horizontal+transfer+of+mutS+alleles+among+naturally+occurring+Escherichia+coli+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001055-Denamur1" id="pone.0001055-Denamur1"></a>Denamur E, Lecointre G, Darlu P, Tenaillon O, Acquaviva C, et al.  (2000) Evolutionary implications of the frequent horizontal transfer of mismatch repair genes. Cell  103: 711–721.  <ul class="find" data-citedArticleID="1006910" data-doi="10.1016/s0092-8674(00)00175-6"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)00175-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+implications+of+the+frequent+horizontal+transfer+of+mismatch+repair+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+implications+of+the+frequent+horizontal+transfer+of+mismatch+repair+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001055-Boucher1" id="pone.0001055-Boucher1"></a>Boucher C, Martinel A, Barberis P, Alloing G, Zischek C (1986) Virulence genes are carried by a megaplasmid of the plant pathogen <em>Pseudomonas solanacearum</em>. Mol Gen Genet  205: 270–275.  <ul class="find" data-citedArticleID="1006894" data-doi="10.1007/bf00430438"><li><a href="http://dx.doi.org/10.1007/bf00430438" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Virulence+genes+are+carried+by+a+megaplasmid+of+the+plant+pathogen+Pseudomonas+solanacearum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Virulence+genes+are+carried+by+a+megaplasmid+of+the+plant+pathogen+Pseudomonas+solanacearum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001055-Edgar1" id="pone.0001055-Edgar1"></a>Edgar RC (2004) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res  32: 1792–1797.  <ul class="find" data-citedArticleID="1006916" data-doi="10.1093/nar/gkh340"><li><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001055-Castresana1" id="pone.0001055-Castresana1"></a>Castresana J (2000) Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Microbiol  17: 540–552.  <ul class="find" data-citedArticleID="1006900" data-doi="10.1093/oxfordjournals.molbev.a026334"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026334" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+of+conserved+blocks+from+multiple+alignments+for+their+use+in+phylogenetic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+of+conserved+blocks+from+multiple+alignments+for+their+use+in+phylogenetic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001055-Guindon1" id="pone.0001055-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol  52: 696–704.  <ul class="find" data-citedArticleID="1006932"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001055-Whelan1" id="pone.0001055-Whelan1"></a>Whelan S, Goldman N (2001) A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach. Mol Biol Evol  18: 691–699.  <ul class="find" data-citedArticleID="1006984" data-doi="10.1093/oxfordjournals.molbev.a003851"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003851" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+empirical+model+of+protein+evolution+derived+from+multiple+protein+families+using+a+maximum-likelihood+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+empirical+model+of+protein+evolution+derived+from+multiple+protein+families+using+a+maximum-likelihood+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001055-Shimodaira1" id="pone.0001055-Shimodaira1"></a>Shimodaira H, Hasegawa M (1999) Multiple comparisons of log-likelihoods with applications to phylogenetic inference. Mo Biol Evol  16: 1114–1116.  <ul class="find" data-citedArticleID="1006974" data-doi="10.1093/oxfordjournals.molbev.a026201"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+comparisons+of+log-likelihoods+with+applications+to+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+comparisons+of+log-likelihoods+with+applications+to+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001055-Brau1" id="pone.0001055-Brau1"></a>Brau B, Pilz U, Piepersberg W (1984) Genes for gentamicin-(3)-N-acetyltransferases III and IV: I. Nucleotide sequence of the AAC(3)-IV gene and possible involvement of an IS140 element in its expression. Mol Gen Genet  193: 179–187.  <ul class="find" data-citedArticleID="1006896" data-doi="10.1007/bf00327434"><li><a href="http://dx.doi.org/10.1007/bf00327434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+for+gentamicin-%283%29-N-acetyltransferases+III+and+IV%3A+I.+Nucleotide+sequence+of+the+AAC%283%29-IV+gene+and+possible+involvement+of+an+IS140+element+in+its+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+for+gentamicin-%283%29-N-acetyltransferases+III+and+IV%3A+I.+Nucleotide+sequence+of+the+AAC%283%29-IV+gene+and+possible+involvement+of+an+IS140+element+in+its+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001055-Charif1" id="pone.0001055-Charif1"></a>Charif D, Lobry JR (2007)  SeqinR 1.0-2: a contributed package to the R project for statistical computing devoted to biological sequences retrieval and analysis. In: Bastolla U, Porto M, Roman HE, Vendruscolo M, editors. Structural approaches to sequence evolution: Molecules, networks, populations. Springer Verlag: Biological and Medical Physics, Biomedical Engineering. pp. 207–232.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.XML" value="121460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.PDF" value="400775"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g001.PNG_L" value="94477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g001.PNG_M" value="79519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g001.PNG_S" value="8555"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g001.TIF" value="992604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g001.PNG_I" value="40733"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g002.PNG_L" value="222618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g002.PNG_M" value="123220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g002.PNG_S" value="8484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g002.TIF" value="1589030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g002.PNG_I" value="40434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t001.PNG_L" value="319554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t001.PNG_M" value="175007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t001.PNG_S" value="15547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t001.TIF" value="1207768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t001.PNG_I" value="53564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g003.PNG_L" value="200426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g003.PNG_M" value="100287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g003.PNG_S" value="9452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g003.TIF" value="1336998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g003.PNG_I" value="37099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g004.PNG_L" value="68346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g004.PNG_M" value="29268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g004.PNG_S" value="6473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g004.TIF" value="320606"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g004.PNG_I" value="11183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t002.PNG_L" value="51344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t002.PNG_M" value="65587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t002.PNG_S" value="11373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t002.TIF" value="214228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t002.PNG_I" value="22873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g005.PNG_L" value="125123"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g005.PNG_M" value="43229"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g005.PNG_S" value="7209"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g005.TIF" value="555112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g005.PNG_I" value="15115"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g006.PNG_L" value="51035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g006.PNG_M" value="25568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g006.PNG_S" value="5957"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g006.TIF" value="297632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.g006.PNG_I" value="9255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t003.PNG_L" value="71835"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t003.PNG_M" value="96708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t003.PNG_S" value="14174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t003.TIF" value="323196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.t003.PNG_I" value="32621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.s001.DOC" value="81920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.s002.DOC" value="72704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001055.s003.DOC" value="77312"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001055&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001055" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001055&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001055&volume=&issue=&title=Horizontal Gene Transfer Regulation in Bacteria as a “Spandrel” of DNA Repair Mechanisms&author_name=Saliou%20Fall%2C%20Anne%20Mercier%2C%20Franck%20Bertolla%2C%20Alexandra%20Calteau%2C%20Laurent%20Gueguen%2C%20Guy%20Perri%C3%A8re%2C%20Timothy%20M.%20Vogel%2C%20Pascal%20Simonet&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001055" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001055" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001055" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001055&amp;t=Horizontal%20Gene%20Transfer%20Regulation%20in%20Bacteria%20as%20a%20%E2%80%9CSpandrel%E2%80%9D%20of%20DNA%20Repair%20Mechanisms" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001055&title=Horizontal%20Gene%20Transfer%20Regulation%20in%20Bacteria%20as%20a%20%E2%80%9CSpandrel%E2%80%9D%20of%20DNA%20Repair%20Mechanisms&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001055&amp;title=Horizontal%20Gene%20Transfer%20Regulation%20in%20Bacteria%20as%20a%20%E2%80%9CSpandrel%E2%80%9D%20of%20DNA%20Repair%20Mechanisms" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001055" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001055&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Horizontal Gene Transfer Regulation in Bacteria as a “Spandrel” of DNA Repair Mechanisms';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001055';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Horizontal%20Gene%20Transfer%20Regulation%20in%20Bacteria%20as%20a%20%E2%80%9CSpandrel%E2%80%9D%20of%20DNA%20Repair%20Mechanisms http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001055" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001055" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">





















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="24944"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA%22" title="Search for articles in the subject area:'DNA'"><div class="flagText">DNA</div></a>
              <div data-categoryid="32449" data-articleid="24944"
                   data-categoryname="DNA"
                   class="flagImage" title="Flag 'DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+recombination%22" title="Search for articles in the subject area:'DNA recombination'"><div class="flagText">DNA recombination</div></a>
              <div data-categoryid="33483" data-articleid="24944"
                   data-categoryname="DNA recombination"
                   class="flagImage" title="Flag 'DNA recombination' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24944"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24944"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genomics%22" title="Search for articles in the subject area:'Plant genomics'"><div class="flagText">Plant genomics</div></a>
              <div data-categoryid="33365" data-articleid="24944"
                   data-categoryname="Plant genomics"
                   class="flagImage" title="Flag 'Plant genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24944"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transformation+associated+recombination%22" title="Search for articles in the subject area:'Transformation associated recombination'"><div class="flagText">Transformation associated recombination</div></a>
              <div data-categoryid="36955" data-articleid="24944"
                   data-categoryname="Transformation associated recombination"
                   class="flagImage" title="Flag 'Transformation associated recombination' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8414'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6480'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1569&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
