

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Haplotyping a Quantitative Trait with a High-Density Map in Experimental Crosses</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000732"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000732" />

    <meta name="citation_title" content="Haplotyping a Quantitative Trait with a High-Density Map in Experimental Crosses"/>
    <meta itemprop="name" content="Haplotyping a Quantitative Trait with a High-Density Map in Experimental Crosses"/>

      <meta name="citation_author" content="Wei Hou"/>
            <meta name="citation_author_institution" content="Department of Epidemiology and Health Policy Research, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="John Stephen F. Yap"/>
            <meta name="citation_author_institution" content="Department of Statistics, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Song Wu"/>
            <meta name="citation_author_institution" content="Department of Statistics, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Tian Liu"/>
            <meta name="citation_author_institution" content="Department of Statistics, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="James M. Cheverud"/>
            <meta name="citation_author_institution" content="Department of Anatomy and Neurobiology, Washington University Medical School, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Rongling Wu"/>
            <meta name="citation_author_institution" content="Department of Statistics, University of Florida, Gainesville, Florida, United States of America"/>

    <meta name="citation_date" content="2007/8/15"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000732.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e732"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps.; citation_author=ES Lander; citation_author=D Botstein; citation_journal_title=Genetics; citation_volume=121; citation_number=1; citation_pages=185-199; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits.; citation_author=M Lynch; citation_author=B Walsh; citation_number=2; citation_date=1998; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=High resolution mapping of quantitative traits into multiple loci via interval mapping.; citation_author=RC Jansen; citation_author=P Stam; citation_journal_title=Genetics; citation_volume=136; citation_number=3; citation_pages=1447-1455; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Precision mapping of quantitative trait loci.; citation_author=Z-B Zeng; citation_journal_title=Genetics; citation_volume=136; citation_number=4; citation_pages=1457-1468; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Multiple interval mapping for quantitative trait loci.; citation_author=C-H Kao; citation_author=Z-B Zeng; citation_author=RD Teasdale; citation_journal_title=Genetics; citation_volume=152; citation_number=5; citation_pages=1203-1216; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Quantitative trait loci in Drosophila.; citation_author=TFC Mackay; citation_journal_title=Nat Rev Genet; citation_volume=2; citation_number=6; citation_pages=11-20; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=fw2.2: A quantitative trait locus key to the evolution of tomato fruit size.; citation_author=A Frary; citation_author=TC Nesbitt; citation_author=A Frary; citation_author=S Grandillo; citation_author=E van der Knaap; citation_journal_title=Science; citation_volume=289; citation_number=7; citation_pages=85-88; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Rice domestication by reducing shattering.; citation_author=CB Li; citation_author=AL Zhou; citation_author=T Sang; citation_journal_title=Science; citation_volume=311; citation_number=8; citation_pages=1936-1939; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Strategies for mapping and cloning quantitative trait genes in rodents.; citation_author=J Flint; citation_author=W Valdar; citation_author=S Shifman; citation_author=R Mott; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=9; citation_pages=271-286; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Empirical threshold values for quantitative trait mapping.; citation_author=GA Churchill; citation_author=RW Doerge; citation_journal_title=Genetics; citation_volume=138; citation_number=10; citation_pages=963-971; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The predictive power of haplotypes in clinical response.; citation_author=R Judson; citation_author=JC Stephens; citation_author=A Windemuth; citation_journal_title=Pharmacogenomics; citation_volume=1; citation_number=11; citation_pages=15-26; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The relative power of SNPs and haplotype as genetic markers for association tests.; citation_author=JS Bader; citation_journal_title=Pharmacogenomics; citation_volume=2; citation_number=12; citation_pages=11-24; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sequencing complex diseases with HapMap.; citation_author=T Liu; citation_author=JA Johnson; citation_author=G Casella; citation_author=RL Wu; citation_journal_title=Genetics; citation_volume=168; citation_number=13; citation_pages=503-511; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A haplotype-based algorithm for multilocus linkage disequilibrium mapping of quantitative trait loci with epistasis in natural populations.; citation_author=X-Y Lou; citation_author=G Casella; citation_author=RC Littell; citation_author=MKC Yang; citation_author=RL Wu; citation_journal_title=Genetics; citation_volume=163; citation_number=14; citation_pages=1533-1548; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Blocks of limited haplotype diversity revealed by high-resolution scanning of human chromosome 21.; citation_author=N Patil; citation_author=AJ Berno; citation_author=DA Hinds; citation_author=WA Barrett; citation_author=JM Doshi; citation_journal_title=Science; citation_volume=294; citation_number=15; citation_pages=1719-1723; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A dynamic programming algorithm for haplotype block partitioning.; citation_author=K Zhang; citation_author=M Deng; citation_author=T Chen; citation_author=MS Waterman; citation_author=F Sun; citation_journal_title=Proc Natl Acad Sci; citation_volume=99; citation_number=16; citation_pages=7335-7339; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Minimal haplotype tagging.; citation_author=P Sebastiani; citation_author=SW Lazarus; citation_author=LM Kunkel; citation_author=IS Kohane; citation_author=M Ramoni; citation_journal_title=Proc Natl Acad Sci; citation_volume=100; citation_number=17; citation_pages=9900-9905; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A model-based approach to capture genetic variation for future association studies.; citation_author=S Eyheramendy; citation_author=J Marchini; citation_author=G McVean; citation_author=S Myers; citation_author=P Donnelly; citation_journal_title=Genome Res; citation_volume=17; citation_number=18; citation_pages=88-95; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Finding the observed information matrix when using the EM algorithm.; citation_author=TA Louis; citation_journal_title=J Roy Stat Soc Ser B; citation_volume=44; citation_number=19; citation_pages=226-233; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Model Selection and Inference. A Practical Information-Theoretic Approach.; citation_author=KP Burnham; citation_author=DR Andersson; citation_number=20; citation_date=1998; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Mapping quantitative trait loci for murine growth - A closer look at genetic architecture.; citation_author=TT Vaughn; citation_author=LS Pletscher; citation_author=A Peripato; citation_author=K King-Ellison; citation_author=E Adams; citation_journal_title=Genet Res; citation_volume=74; citation_number=21; citation_pages=313-322; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genetic dissection of complex traits.; citation_author=ES Lander; citation_author=NJ Schork; citation_journal_title=Science; citation_volume=265; citation_number=22; citation_pages=2037-2048; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Detecting sequence-sequence interactions for complex diseases.; citation_author=M Lin; citation_author=RL Wu; citation_journal_title=Current Genomics; citation_volume=7; citation_number=23; citation_pages=59-72; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000732" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Haplotyping a Quantitative Trait with a High-Density Map in Experimental Crosses"/>
    <meta name="twitter:description" content="
Background
The ultimate goal of genetic mapping of quantitative trait loci (QTL) is the positional cloning of genes involved in any agriculturally or medically important phenotype. However, only a small portion (&le; 1%) of the QTL detected have been characterized at the molecular level, despite the report of hundreds of thousands of QTL for different traits and populations.

Methods/Results
We develop a statistical model for detecting and characterizing the nucleotide structure and organization of haplotypes that underlie QTL responsible for a quantitative trait in an F2 pedigree. The discovery of such haplotypes by the new model will facilitate the molecular cloning of a QTL. Our model is founded on population genetic properties of genes that are segregating in a pedigree, constructed with the mixture-based maximum likelihood context and implemented with the EM algorithm. The closed forms have been derived to estimate the linkage and linkage disequilibria among different molecular markers, such as single nucleotide polymorphisms, and quantitative genetic effects of haplotypes constructed by non-alleles of these markers. Results from the analysis of a real example in mouse have validated the usefulness and utilization of the model proposed.

Conclusion
The model is flexible to be extended to model a complex network of genetic regulation that includes the interactions between different haplotypes and between haplotypes and environments.
"/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000732.g002"/>

  <meta property="og:title" content="Haplotyping a Quantitative Trait with a High-Density Map in Experimental Crosses" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000732" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5627'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8525'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4761&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000732">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000732" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000732&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Haplotyping a Quantitative Trait with a High-Density Map in Experimental Crosses
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wei Hou, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Epidemiology and Health Policy Research, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John Stephen F. Yap, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistics, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Song Wu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistics, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tian Liu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistics, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James M. Cheverud, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Anatomy and Neurobiology, Washington University Medical School, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rongling Wu
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rwu@mail.ifas.ufl.edu">rwu@mail.ifas.ufl.edu</a></p>

                <p>Affiliation:
                  Department of Statistics, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 15, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000732</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000732" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000732" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000732" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000732" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000732-g001" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000732-g002" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000732-t001" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000732-t002" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000732-t003" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000732-t004" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000732">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000732" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The ultimate goal of genetic mapping of quantitative trait loci (QTL) is the positional cloning of genes involved in any agriculturally or medically important phenotype. However, only a small portion (≤ 1%) of the QTL detected have been characterized at the molecular level, despite the report of hundreds of thousands of QTL for different traits and populations.</p>


<h3>Methods/Results</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We develop a statistical model for detecting and characterizing the nucleotide structure and organization of haplotypes that underlie QTL responsible for a quantitative trait in an F<sub>2</sub> pedigree. The discovery of such haplotypes by the new model will facilitate the molecular cloning of a QTL. Our model is founded on population genetic properties of genes that are segregating in a pedigree, constructed with the mixture-based maximum likelihood context and implemented with the EM algorithm. The closed forms have been derived to estimate the linkage and linkage disequilibria among different molecular markers, such as single nucleotide polymorphisms, and quantitative genetic effects of haplotypes constructed by non-alleles of these markers. Results from the analysis of a real example in mouse have validated the usefulness and utilization of the model proposed.</p>


<h3>Conclusion</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The model is flexible to be extended to model a complex network of genetic regulation that includes the interactions between different haplotypes and between haplotypes and environments.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hou W, Yap JSF, Wu S, Liu T, Cheverud JM, et al.  (2007) Haplotyping a Quantitative Trait with a High-Density Map in Experimental Crosses. PLoS ONE 2(8):
          e732.
            doi:10.1371/journal.pone.0000732</p><p><strong>Academic Editor: </strong>Peter Heutink, Vrije Universiteit Medical Centre, Netherlands</p><p><strong>Received:</strong> April 10, 2007; <strong>Accepted:</strong> July 13, 2007; <strong>Published:</strong> August 15, 2007</p><p><strong>Copyright:</strong> © 2007 Hou et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The preparation of this manuscript is supported by NSF grant (0540745) to RW.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The basic principle for quantitative trait locus (QTL) mapping is the cosegregation of the alleles at a QTL with those at one or a set of known polymorphic markers genotyped on a genome in an experimental cross <a href="#pone.0000732-Lander1">[1]</a>, <a href="#pone.0000732-Lynch1">[2]</a>. If a QTL is cosegregating with molecular markers, the genetic effects of the QTL on a quantitative trait and its genomic position can be estimated from the marker genotypes and phenotypic values of the trait. This estimation process particularly assumes the QTL to be located within an interval constructed by a pair of flanking markers in which a test statistics calculated under the reduced (there is no QTL) and full model (there is a QTL) is used to test the existence of the QTL and estimate its position. This so-called interval mapping approach and its extensions <a href="#pone.0000732-Jansen1">[3]</a>–<a href="#pone.0000732-Kao1">[5]</a> is robust and powerful for the detection of major QTL and presents the most efficient way to utilize marker information when marker maps are sparse <a href="#pone.0000732-Mackay1">[6]</a>. However, interval mapping is limited by its incapacity to infer any information about the sequence structure and organization of the QTL. Partly for this reason, only a few QTL mapped from markers have been successfully cloned <a href="#pone.0000732-Frary1">[7]</a>–<a href="#pone.0000732-Flint1">[9]</a>, despite a considerable number of QTL reported in the literature.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Interval QTL mapping also has an unsolved statistical difficulty when it is used with a high-density linkage map. With more markers genotyped, a genetic map for QTL identification has tended to be infinitely dense. For such an infinitely dense map in which markers are located everywhere over the genome, test statistics at nearby intervals are not independent any more. Thus, the critical threshold used to acclaim the existence of a QTL by interval mapping will be difficult to analytically determine. Although an empirical alternative based on permutation tests has been proposed for threshold determination <a href="#pone.0000732-Churchill1">[10]</a>, extensive computing may affect the use efficiency of interval mapping.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Despite its unsuitability for interval mapping of QTL, an infinitely dense map provides an important tool for characterizing genetic variants that contribute to quantitative variation via the analysis of haplotypes composed of non-alleles at a set of highly linked markers. Recent genetic studies suggest that a gene may determine a complex trait, such as body weight or drug response, through its haplotype rather than genotype <a href="#pone.0000732-Judson1">[11]</a>, <a href="#pone.0000732-Bader1">[12]</a>. The completion of the genome projects for several important organisms, Arabdopsis, chicken, human, mouse and poplar, has made massive amounts of DNA sequence data available. In particular, single nucleotide polymorphisms (SNPs), being the most common type of variant in the DNA sequence, provide a powerful means for genotyping the whole genome or any part of it. This facilitates the identification of specific SNP-constructed haplotypes which are responsible for quantitative traits. A set of SNPs that cause quantitative differences among individuals are called quantitative trait nucleotides (QTNs). Liu et al. <a href="#pone.0000732-Liu1">[13]</a> proposed a statistical model for estimating and testing haplotype effects at a QTN in a random sample drawn from a natural population. This model is based on the population genetic properties of gene segregation. Through the implementation of the EM algorithm, population genetic parameters of SNPs, such as haplotype frequencies, allele frequencies and linkage disequilibria, and quantitative genetic parameters, such as haplotype effects of a QTN, are estimated with closed forms.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The motivation of this work is to derive a statistical model for haplotype discovery responsible for quantitative variation in a mapping population derived an experimental cross. Unlike a natural population in which gene co-segregation analysis is based on linkage disequilibria <a href="#pone.0000732-Lou1">[14]</a>, experimental crosses, such as the backcross or F<sub>2</sub>, have usually been analyzed in terms of the linkage between different markers and QTL. In this article, we will frame a general statistical model for estimating the linkage between different SNPs and testing haplotype effects within the context of linkage disequilibrium analysis in an F<sub>2</sub> pedigree. We show that the new model can test for the dependence of SNPs when a multi-point analysis is performed. We have derived closed forms for the EM algorithm to estimate a variety of genetic parameters. A worked example is used to validate the usefulness and utilization of the model.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Haplotype and diplotype</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A haplotype represents a linear arrangement of nucleotides (alleles) at different SNPs on a single chromosome, or part of a chromosome. The pair of haplotypes is called a diplotype. The observed phenotype of a diplotype is called a genotype. A diplotype is always constructed by two haplotypes, one from the maternal parent and the other from the paternal parent. Suppose there are two different SNPs on the same genomic region, one with two alleles <em>A</em> and <em>a</em> and the other with two alleles <em>B</em> and <em>b</em>, respectively. Allele <em>A</em> from SNP 1 and allele <em>B</em> from SNP 2 are located on the first homologous chromosome, whereas allele <em>a</em> from SNP 1 and allele <em>b</em> from SNP 2 located on the second homologous chromosome. Thus, [<em>AB</em>] is one haplotype and [<em>ab</em>] is a second haplotype, and both constitute a diplotype [<em>AB</em>][<em>ab</em>] (<a href="#pone-0000732-g001">Fig. 1</a>).</p>
<div class="figure" id="pone-0000732-g001"><div class="img"><a name="pone-0000732-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000732.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000732.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Haplotype configuration of a diplotype for two hypothesized SNPs.</span></strong></p><span>doi:10.1371/journal.pone.0000732.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In a practical genetic analysis, we can only observe the genotype expressed as <em>Aa/Bb</em>. However, the double heterozygote may be one (and only one) of two possible diplotypes [<em>AB</em>][<em>ab</em>] and [<em>Ab</em>][<em>aB</em>]. But these two diplotypes cannot be directly observed and should be inferred from SNP genotype data (<a href="#pone-0000732-g002">Fig. 2</a>). In practice, it is important to estimate haplotype effects on a quantitative trait based on the diplotypes and therefore genotypes. For example, if an animal carries haplotype [<em>AB</em>], it will grow better than other animals that carries any other haplotypes, [<em>Ab</em>], [<em>aB</em>] and [<em>ab</em>]. For this reason, the same genotype <em>Aa/Bb</em> may perform differently, depending on what diplotype it carries. If this genotype is diplotype [<em>AB</em>][<em>ab</em>], then it will have a better growth. If the animal is diplotype [<em>Ab</em>][<em>aB</em>], its growth will be poorer. The statistical model being developed will be used to determine which diplotype is associated with better growth in experimental crosses.</p>
<div class="figure" id="pone-0000732-g002"><div class="img"><a name="pone-0000732-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000732.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000732.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Diplotype configuration of a genotype for two hypothesized SNPs.</span></strong></p><span>doi:10.1371/journal.pone.0000732.g002</span></div>

<h4>Linkage disequilibrium in the F<sub>2</sub> intercross</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p><strong>A general model:</strong> Haplotype analysis in the backcross is straightforward because the diplotype is determined for all the backcross genotype. Simple analysis of variance can be used to detect haplotype effects on a quantitative trait. In the F<sub>2</sub>, this is not a case in which the double heterozygote is a mixture of two possible diplotypes.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Suppose many SNPs are genotyped each of which is segregating in a 1:2:1 Mendelian ratio in the F<sub>2</sub> population. As seen in the human genome <a href="#pone.0000732-Patil1">[15]</a>, these SNPs are divided into different haplotype blocks. For a given block, there are a particular number of representative SNPs or htSNPs that uniquely identify the common haplotypes in this block or QTN. Several algorithms have been developed to identify a minimal subset of htSNPs that can characterize the most common haplotypes <a href="#pone.0000732-Zhang1">[16]</a>–<a href="#pone.0000732-Eyheramendy1">[18]</a>. Consider a QTN that contains <em>L</em> htSNPs among which there exist linkage disequilibria of different orders. The two alleles, 1 and 0, at each of these SNPs are symbolized by <em>r</em><sub>1</sub>,…,<em>r<sub>L</sub></em>, respectively. For a cross initiated with two inbred parents, the allele frequencies for each of these htSNPs should be 1/2. A haplotype frequency, denoted as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e001&amp;representation=PNG" class="inline-graphic"></span>, is decomposed into the following components:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>D</em>'s are the linkage disequilibria of different orders among particular SNPs.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Totally, <em>L</em> SNPs form 2<em><sup>L</sup></em> haplotypes expressed as [<em>r</em><sub>1</sub>…<em>r<sub>L</sub></em>], 2<em><sup>L</sup></em><sup>−1</sup>(2<em><sup>L</sup></em>+1) diplotypes, i.e., a pair of maternally- (m) and paternally-derived haplotypes (p), expressed as [<em>r</em><sub>1</sub><sup>m</sup>…<em>r<sub>L</sub></em><sup>m</sup>][<em>r</em><sub>1</sub><sup>p</sup>…<em>r<sub>L</sub></em><sup>p</sup>] (<em>r</em><sub>1</sub><sup>m</sup>, <em>r</em><sub>1</sub><sup>p</sup>,…;<em>r<sub>L</sub></em><sup>m</sup>,<em>r<sub>L</sub></em><sup>p</sup> = 1,0) and 3<em><sup>L</sup></em> genotypes expressed as <em>r</em><sub>1</sub><em>r</em>′<sub>1</sub>/…/<em>r<sub>L</sub>r</em>′<em><sub>L</sub></em> (<em>r</em><sub>1</sub>≥<em>r</em>′<sub>1,</sub>…,<em>r<sub>L</sub></em>≥<em>r</em>′<em><sub>L</sub> = </em>1,0). Only genotypes can be observed. The number of diplotypes is smaller than the number of genotypes because the genotypes that are heterozygous at two or more SNPs contain multiple different diplotypes. Diplotype (and therefore genotype) frequencies can be expressed in terms of haplotype frequencies. We use <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e003&amp;representation=PNG" class="inline-graphic"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e004&amp;representation=PNG" class="inline-graphic"></span> to denote the diplotype and genotype frequencies, respectively, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e005&amp;representation=PNG" class="inline-graphic"></span> to denote genotype observation.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p><strong>A special case: Two-point linkage disequilibrium:</strong> For two given SNPs (<strong>S</strong><sub>1</sub> and <strong>S</strong><sub>2</sub>), there are four different haplotypes in a cross population. According to the definition given above, these four haplotypes are denoted as <a href="#pone.0000732-Judson1">[11]</a>, <a href="#pone.0000732-Churchill1">[10]</a>, [01] and [00], whose frequencies in a cross population are, respectively, expressed as <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e006&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>Assume that the two SNPs are linked with a recombination fraction <em>r</em>. The haplotype frequencies can be expressed in terms of <em>r</em>, i.e., <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e007&amp;representation=PNG" class="inline-graphic"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e008&amp;representation=PNG" class="inline-graphic"></span>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e009&amp;representation=PNG" class="inline-graphic"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e010&amp;representation=PNG" class="inline-graphic"></span>. Combining equation (2), this establishes the relation between the linkage disequilibrium and recombination fraction as <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>or<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br></p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p><strong>A special case: Three-point linkage disequilibrium:</strong> For three given SNPs (<strong>S</strong><sub>1,</sub> <strong>S</strong><sub>2, </sub>and <strong>S</strong><sub>3</sub>), there are eight different haplotypes, i.e., [111], [110], [101], [100], [011], [010], [001], and [000]. The haplotype frequencies in a cross population are, respectively, expressed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>where <em>D</em><sub>12</sub>, <em>D</em><sub>23</sub> and <em>D</em><sub>13</sub> are the linkage disequilibria between SNP <strong>S</strong><sub>1</sub> and <strong>S</strong><sub>2</sub>, between <strong>S</strong><sub>2</sub> and <strong>S</strong><sub>3</sub> and between <strong>S</strong><sub>1</sub> and <strong>S</strong><sub>2</sub>, respectively, and <em>D</em><sub>123</sub> is the linkage disequilibrium among the three SNPs. The four disequilibrium coefficients can be estimated, by solving equation (5), as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e014&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>The first three first-order linkage disequilibria can be used to describe the linkage between different SNPs and crossover interference, whereas the last second-order linkage disequilibrium is thought to be associated with chromatid interference.</p>


<h4>Haplotyping a trait with two SNPs</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Our interest is to search for the haplotype diversity that can explain phenotypic variation in a complex trait. The association between haplotype diversity and phenotypic variation has been detected in several studies of drug responses <a href="#pone.0000732-Judson1">[11]</a>, <a href="#pone.0000732-Bader1">[12]</a>. This allows us to assume that a particular haplotype is different from other haplotypes for a given trait. Here, our focus will be on modelling haplotype effects in experimental crosses. Although haplotypes (comprising diplotypes) can be directly observed in the backcross, this is not possible for the F<sub>2</sub> because their heterozygous genotypes are not concordant with diplotypes or haplotypes. For the F<sub>2</sub> population, the effects of different haplotypes on the phenotype need be postulated from observed zygotic genotypes. The inference of diplotypes for a particular genotype is statistically a missing data problem that can be formulated by a finite mixture model.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p><strong>Mixture model:</strong> The statistical method for the genomewide scan of QTN is formulated on the basis of a finite mixture model. The mixture model assumes that each observation comes from one of an assumed set of distributions. The mixture model derived to detect haplotype effects on a quantitative trait based on SNP genotype data contains three major parts: (1) the mixture proportions of each distribution, denoted as the relative frequencies of different diplotypes for the same SNP genotype, (2) the mean for each diplotype in the density function, and (3) the residual variance common to all diplotypes.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>For simplicity, we consider a QTN that is composed of only two SNPs each with two alleles designated as 1 and 0. These two SNPs segregating in the F<sub>2</sub> population form four haplotypes whose frequencies are arrayed in vector Θp = (<em>p</em><sub>11</sub>, <em>p</em><sub>10</sub>, <em>p</em><sub>01</sub>, <em>p</em><sub>00</sub>). All the genotypes are consistent with diplotypes, except for the double heterozygote, 10/10, that contains two different diplotypes <a href="#pone.0000732-Judson1">[11]</a>[00] with a frequency of 2 <em>p</em><sub>11</sub><em>p</em><sub>00</sub> and <a href="#pone.0000732-Churchill1">[10]</a>[01] with a frequency of 2 <em>p</em><sub>10</sub><em>p</em><sub>01</sub> (<a href="#pone-0000732-t001">Table 1</a>). The relative frequencies of different diplotypes for the double heterozygote are a function of haplotype frequencies.</p>
<div class="figure" id="pone-0000732-t001"><div class="img"><a name="pone-0000732-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000732.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000732.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Diplotypes and their frequencies for each of nine genotypes at two SNPs within a QTN, haplotype composition frequencies for each genotype, and composite diplotypes for four possible risk haplotypes.</span></strong></p><span>doi:10.1371/journal.pone.0000732.t001</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>A total of <em>n</em> individuals in the F<sub>2</sub> are classified into 9 genotypes for the two SNPs, each genotype with observation generally expressed as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e022&amp;representation=PNG" class="inline-graphic"></span> (<em>r</em><sub>1</sub>≥<em>r</em>′<sub>1</sub>,<em>r</em><sub>2</sub>≥<em>r</em>′<sub>2</sub>,<em>r</em><sub>3</sub>≥<em>r</em>′<sub>3</sub> = 1,0). The frequency of each genotype can be expressed in terms of haplotype frequencies (<a href="#pone-0000732-t001">Table 1</a>). Considering a quantitative trait controlled by diplotype (rather than genotype) diversity, the phenotypic value of the trait (<em>y<sub>i</sub></em>) for individual <em>i</em> is expressed by a linear model, i.e.,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e023&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>where <em>ξ<sub>i</sub></em> is the indicator variable defined as 1 if a diplotype considered is compatible with subject <em>i</em> and as 0 otherwise, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e024&amp;representation=PNG" class="inline-graphic"></span> is the genotypic value for diplotype <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e025&amp;representation=PNG" class="inline-graphic"></span>, and <em>e<sub>i</sub></em> is the residual error distributed as <em>N</em>(0,<em>σ</em><sup>2</sup>).</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Assume that this QTN triggers an effect on the trait because at least one haplotype is different from the remaining seven. Without loss of generality, let <a href="#pone.0000732-Judson1">[11]</a> be such a distinct haplotype, called <em>risk haplotype</em>, designated as <em>A</em>. All the other non-risk haplotypes, <a href="#pone.0000732-Churchill1">[10]</a>, [01] and [00], are collectively expressed as <em>A̅</em>. The risk and non-risk haplotypes form three <em>composite diplotypes AA</em> (<strong>2</strong>), <em>AA̅</em> (<strong>1</strong>) and <em>A̅</em><em>A̅</em> (<strong>0</strong>). Let <em>μ</em><sub>2</sub>, <em>μ</em><sub>1 </sub>and <em>μ</em><sub>0</sub> be the genotypic value of the three composite diplotypes, respectively (<a href="#pone-0000732-t001">Table 1</a>). The means for different composite diplotypes and residual variance are arrayed by a quantitative genetic parameter vector Θ<em><sub>q</sub></em> = (<em>μ</em><sub>2, </sub><em>μ</em><sub>1, </sub><em>μ</em><sub>0, </sub><em>σ<sup>2</sup>)</em>.</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p><strong>Likelihoods:</strong> With the above notation, we construct two likelihoods, one for haplotype frequencies (Θ<em><sub>p</sub></em>) based on SNP data (<strong>S</strong>) and the other for quantitative genetic parameters (Θ<em><sub>q</sub></em>) based on haplotype frequencies (Θ<em><sub>p</sub></em>), phenotypic (<em>y</em>) and SNP data (<strong>S</strong>). They are, respectively, expressed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e026&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br>where <em>f <sub>j</sub></em> (<em>y<sub>i</sub></em>) is a normal distribution density function of composite diplotype <em>j</em> (<em>j</em> = 2,1,0), i.e.,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e027&amp;representation=PNG" class="inline-graphic"></span><br>It can be seen from the above likelihood functions that, although most zygote genotypes contain a single component (diplotype), the double heterozygote is the mixture of two possible diplotypes weighted by <em>φ</em> and 1-<em>φ</em>, expressed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e028&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br>which represents the relative frequency of diplotype <a href="#pone.0000732-Judson1">[11]</a>[00] for the double heterozygote.</p>
<a id="article1.body1.sec2.sec3.p7" name="article1.body1.sec2.sec3.p7"></a><p>It should be noted that <em>L</em>(Θ<em><sub>p</sub></em>, Θ<em><sub>q</sub></em><sub> </sub>| <em>y</em>, <strong>S</strong>) relies on the haplotype frequencies defined in <em>L</em>(Θ<em><sub>p</sub></em>|<strong>S</strong>) and, thus, the latter is thought to be nested within the former. The estimates of parameters that maximize <em>L</em>(Θ<em><sub>p</sub></em>|<strong>S</strong>) can also maximize the <em>L</em>(Θ<em><sub>p</sub></em>, Θ<em><sub>q</sub></em><sub> </sub>| <em>y</em>, <strong>S</strong>).</p>
<a id="article1.body1.sec2.sec3.p8" name="article1.body1.sec2.sec3.p8"></a><p><strong>The EM algorithm:</strong> A closed-form solution for the EM algorithm has been derived to estimate the unknown parameters that maximize the two likelihoods of (26) <a href="#pone.0000732-Liu1">[13]</a>. The estimates of haplotype frequencies are based on the log-likelihood function <em>L</em>(Θ<em><sub>p</sub></em>|<strong>M</strong>), whereas the estimates of diplotype genotypic means and residual variance are based on the log-likelihood function <em>L</em>(Θ<em><sub>p</sub></em>, Θ<em><sub>q</sub></em><sub> </sub>| <em>y</em>, <strong>M</strong>). These two different types of parameters can be estimated using a two-stage hierarchical EM algorithm.</p>
<a id="article1.body1.sec2.sec3.p9" name="article1.body1.sec2.sec3.p9"></a><p>At a higher hierarchy of the EM algorithm, the E step is aimed to calculate the relative frequency (<em>φ</em>) of diplotype <a href="#pone.0000732-Judson1">[11]</a>[00] in the double heterozygote is calculated by equation (9). The M step is aimed to estimate the haplotype frequencies based on the probabilities calculated in the previous iteration using<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e029&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br></p>
<a id="article1.body1.sec2.sec3.p10" name="article1.body1.sec2.sec3.p10"></a><p>At a lower hierarchy of the EM algorithm, the E step is derived to calculate the posterior probability (Ω<a href="#pone.0000732-Judson1">[<sub>11</sub>]</a><sub>[00]<em>i</em></sub>) of individual <em>i</em> with the double heterozygous genotype to be diplotype <a href="#pone.0000732-Judson1">[11]</a>[00] by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e030&amp;representation=PNG" class="inline-graphic"></span><br>Note that for all the other genotypes, such posterior probabilities do not exist.</p>
<a id="article1.body1.sec2.sec3.p11" name="article1.body1.sec2.sec3.p11"></a><p>By assuming that <a href="#pone.0000732-Judson1">[11]</a> is a risk haplotype, the M step is derived to estimate the genotypic values (<em>μ<sub>j</sub></em>) for each composite diplotype and the residual variance based on the calculated posterior probabilities by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e031&amp;representation=PNG" class="inline-graphic"><span class="note">(12)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e032&amp;representation=PNG" class="inline-graphic"><span class="note">(13)</span></span><br>where<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e033&amp;representation=PNG" class="inline-graphic"></span><br>Iterations including the E and M steps are repeated at the higher hierarchy between equations (9) and (10) and at the lower hierarchy among equations (12) and (13) until the estimates of the parameters converge to stable values. The sampling errors of these parameters can be estimated by calculating Louis' <a href="#pone.0000732-Louis1">[19]</a> observed information matrix.</p>
<a id="article1.body1.sec2.sec3.p12" name="article1.body1.sec2.sec3.p12"></a><p>Haplotype frequencies can be expressed as a function of allelic frequencies and linkage disequilibrium. Based on equation (2), we solve the linkage disequilibrium between two SNPs by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e034&amp;representation=PNG" class="inline-graphic"><span class="note">(14)</span></span><br>With the genotypic means of composite diplotypes, we can estimate the overall mean (<em>μ</em>) and additive (<em>a</em>) and dominant genetic effects (<em>d</em>) due to the QTN detected, respectively, by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e035&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec2.sec3.p13" name="article1.body1.sec2.sec3.p13"></a><p><strong>Model selection:</strong> The likelihood <em>L</em>(Θ<em><sub>p</sub></em>, Θ<em><sub>q</sub></em><sub> </sub>| <em>y</em>, <strong>S</strong>) is formulated by assuming that haplotype <a href="#pone.0000732-Judson1">[11]</a>,<a href="#pone.0000732-Judson1">[11]</a> is a risk haplotype. However, a real risk haplotype is unknown from raw data (<em>y</em>, <strong>S</strong>). An additional step for the choice of the most likely risk haplotype should be implemented. The simplest way to do so is to calculate the likelihood values by assuming that any one of the four haplotypes can be a risk haplotype (<a href="#pone-0000732-t001">Table 1</a>). Thus, we obtain four possible likelihood values under different risk haplotypes; that is, (1) <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e036&amp;representation=PNG" class="inline-graphic"></span> for <a href="#pone.0000732-Judson1">[11]</a>, (2) <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e037&amp;representation=PNG" class="inline-graphic"></span> for <a href="#pone.0000732-Churchill1">[10]</a>, (3) <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e038&amp;representation=PNG" class="inline-graphic"></span> for [01], and (4) <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e039&amp;representation=PNG" class="inline-graphic"></span> for [00]. Under each possible risk haplotype, we estimate the quantitative genetic parameters <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e040&amp;representation=PNG" class="inline-graphic"></span> (<em>k</em> = 1,…,4). The largest likelihood value calculated is thought to correspond to the most likely risk haplotype.</p>
<a id="article1.body1.sec2.sec3.p14" name="article1.body1.sec2.sec3.p14"></a><p>In practice, it is also possible that there exist more than one risk haplotypes for a QTN. Relative to the bi-“allelic” QTN with one risk haplotype, such a QTN is called a multi-“allelic” QTN. If there are two risk haplotypes, we will have six composite diplotypes. Assuming that <a href="#pone.0000732-Judson1">[11]</a> (denoted by <em>A</em><sub>1</sub>) and <a href="#pone.0000732-Churchill1">[10]</a> (denoted by <em>A</em><sub>2</sub>) are risk haplotypes and the remaining haplotypes <a href="#pone.0000732-Churchill1">[10]</a> and [01] are non-risk haplotypes (denoted by <em>A</em><sub>3</sub>), then six composite diplotypes, expressed as <em>A</em><sub>1</sub><em>A</em><sub>1</sub>, <em>A</em><sub>1</sub><em>A</em><sub>2</sub>, <em>A</em><sub>1</sub><em>A</em><sub>3</sub>, <em>A</em><sub>2</sub><em>A</em><sub>2</sub>, <em>A</em><sub>2</sub><em>A</em><sub>3</sub> and <em>A</em><sub>3</sub><em>A</em><sub>3</sub>, can be specified according to the diplotype distribution as shown in <a href="#pone-0000732-t001">Table 1</a>. Totally, there are six such haplotype combinations for a two-SNP QTL, each combination corresponding to a likelihood value. Based on the calculated likelihoods, we can determine a most likely risk and non-risk haplotype combination. If there are three risk haplotypes, we will have 10 different composite diplotypes. The optimal risk and non-risk haplotype combination will be selected from three combinations based on the likelihoods.</p>
<a id="article1.body1.sec2.sec3.p15" name="article1.body1.sec2.sec3.p15"></a><p>The likelihood can be used as a criterion to select the optimal risk and non-risk haplotype combination when the number of risk haplotype is the same. However, when the number of risk haplotype is different, an AIC- or BIC-based model selection strategy <a href="#pone.0000732-Burnham1">[20]</a> should be used because of different numbers of parameters being estimated in this case.</p>
<a id="article1.body1.sec2.sec3.p16" name="article1.body1.sec2.sec3.p16"></a><p><strong>Hypothesis tests:</strong> We can test two major hypotheses in the following sequence: (1) the association between two SNPs by testing their linkage disequilibrium, and (2) the difference of a given haplotype from the remaining haplotypes by testing the significance of haplotype additive and dominant effects on the trait. The linkage disequilibrium between two given SNPs can be tested using two alternative hypotheses:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e041&amp;representation=PNG" class="inline-graphic"><span class="note">(15)</span></span><br>The log-likelihood ratio test statistic for the significance of LD is calculated by comparing the likelihood values under the <em>H<sub>1</sub></em> (full model) and <em>H<sub>0</sub></em> (reduced model) using<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e042&amp;representation=PNG" class="inline-graphic"><span class="note">(16)</span></span><br>The <em>LR<sub>1</sub></em> is considered to asymptotically follow a χ<sup>2</sup> distribution with one degree of freedom.</p>
<a id="article1.body1.sec2.sec3.p17" name="article1.body1.sec2.sec3.p17"></a><p>Diplotype or haplotype effects on the trait, i.e., the existence of a QTN, can be tested using the following hypotheses expressed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e043&amp;representation=PNG" class="inline-graphic"><span class="note">(17)</span></span><br>The log-likelihood ratio test statistic (<em>LR</em><sub>2</sub>) under these two hypotheses can be similarly calculated,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e044&amp;representation=PNG" class="inline-graphic"><span class="note">(18)</span></span><br>where the tildes and hats denote the MLEs of parameters under the null and alternative hypotheses of (17), respectively. Although the critical threshold for determining the existence of a QTN can be based on empirical permutation tests, the <em>LR</em><sub>2</sub> may asymptotically follow a χ<sup>2</sup> distribution with two degrees of freedom, so that the threshold can be obtained from the χ<sup>2</sup> distribution table.</p>


<h4>Haplotyping a trait with multiple SNPs</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p><strong>Haplotype structure:</strong> The statistical method for QTN mapping is exemplified by a set of three SNPs, <strong>S</strong><sub>1</sub>–<strong>S</strong><sub>3</sub>, for a QTN. Two alleles 1 and 0 at each SNP are symbolized by <em>r</em><sub>1</sub>, <em>r</em><sub>2</sub> and <em>r</em><sub>3</sub>, respectively. Eight haplotypes, [111], [110], [101], [100], [011], [010], [001] and [000], formed by these three SNPs, have the frequencies arrayed in Θ<em><sub>p</sub></em> = (<em>p</em><sub>111, </sub><em>p</em><sub>110, </sub><em>p</em><sub>101, </sub><em>p</em><sub>100,</sub> <em>p</em><sub>011, </sub><em>p</em><sub>010, </sub><em>p</em><sub>001, </sub><em>p</em><sub>000</sub>). Some genotypes are consistent with diplotypes, whereas the others that are heterozygous at two or more SNPs are not. Each double heterozygote contains two different diplotypes. One triple heterozygote, i.e., 10/10/10, contains four different diplotypes, [111][000] (in a probability of 2<em>p</em><sub>111</sub><em>p</em><sub>000</sub>), [110][001] (in a probability of 2<em>p</em><sub>110</sub><em>p</em><sub>001</sub>), [101][010] (in a probability of 2<em>p</em><sub>101</sub><em>p</em><sub>010</sub>) and [100][011] (in a probability of 2<em>p</em><sub>100</sub><em>p</em><sub>011</sub>). The relative frequencies of different diplotypes for this double or triple heterozygote are a function of haplotype frequencies (<a href="#pone-0000732-t002">Table 2</a>).</p>
<div class="figure" id="pone-0000732-t002"><div class="img"><a name="pone-0000732-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000732.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000732.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Possible diplotypes and their frequencies for each of 27 genotypes at three SNPs within a QTN, and genotypic value vectors of composite diplotypes (assuming that [111] (<em>A</em>) is the risk haplotype and the others (<em>A̅</em>) are the non-risk haplotype).</span></strong></p><span>doi:10.1371/journal.pone.0000732.t002</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In the F<sub>2</sub> population, there are 27 genotypes for the three SNPs. Let <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e080&amp;representation=PNG" class="inline-graphic"></span> (<em>r</em><sub>1</sub>≥<em>r</em>′<sub>1</sub>,<em>r</em><sub>2</sub>≥<em>r</em>′<sub>2</sub>,<em>r</em><sub>3</sub>≥<em>r</em>′<sub>3</sub> = 1,0) be the number of offspring for a genotype. As seen in <a href="#pone-0000732-t002">Table 2</a>, the frequency of each genotype is expressed in terms of haplotype frequencies. Similar to equation (25), the phenotypic value of the trait for individual <em>i</em> is expressed, at the diplotype level, as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e081&amp;representation=PNG" class="inline-graphic"><span class="note">(19)</span></span><br>where <em>ξ<sub>i</sub></em> is the indicator variable defined as 1 if a diplotype considered is compatible with subject <em>i</em> and as 0 otherwise, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e082&amp;representation=PNG" class="inline-graphic"></span> is the genotypic value for diplotype [<em>r</em><sub>1</sub><sup>m</sup><em>r</em><sub>2</sub><sup>m</sup><em>r</em><sub>3</sub><sup>m</sup>][<em>r</em><sub>1</sub><sup>p</sup><em>r</em><sub>2</sub><sup>p</sup><em>r</em><sub>3</sub><sup>p</sup>], and <em>e<sub>i</sub></em> is the residual error distributed as <em>N</em>(0,<em>σ</em><sup>2</sup>). Note that <strong>m</strong> and <strong>p</strong> stand for the maternally and paternally derived alleles, respectively.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>By assuming [111] as a risk haplotype (labelled by <em>A</em>) and all the others as non-risk haplotypes (labelled by <em>A̅</em>), <a href="#pone-0000732-t002">Table 2</a> provides the formulation of genotypic values for three composite diplotypes, <em>μ</em><sub>2</sub> for <em>AA</em>, <em>μ</em><sub>1</sub> for <em>AA̅</em> and <em>μ</em><sub>0</sub> for <em>A̅</em><em>A̅</em>. The haplotype effect parameters and residual covariance matrix are arrayed by a quantitative genetic parameter vector Θ<em><sub>q</sub></em> = (<em>μ</em><sub>2</sub>, <em>μ</em><sub>1</sub>, <em>μ</em><sub>0</sub>,<em>σ</em><sup>2</sup>).</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p><strong>Likelihoods and algorithms:</strong>With the above notation, we construct two likelihoods, one for haplotype frequencies (Θ<em><sub>p</sub></em>) based on SNP data (<strong>S</strong>) and the other for quantitative genetic parameters (Θ<em><sub>q</sub></em>) based on haplotype frequencies (Θ<em><sub>p</sub></em>), phenotypic (<em>y</em>) and SNP data (<strong>S</strong>). They are, respectively, expressed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e083&amp;representation=PNG" class="inline-graphic"><span class="note">(20)</span></span><br>where <em>φ</em><sub>.</sub>'s (<em>φ̅</em>. = 1−<em>φ</em>) are defined below, and <em>f <sub>j </sub>( y<sub>j</sub>)</em> (<em>j</em> = 2, 1, 0) is a normal distribution density function of composite diplotype <em>j</em>.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>A two-stage hierarchical EM algorithm is derived to estimate haplotype frequencies and quantitative genetic parameters. At the higher hierarchy of the EM framework, we calculate the proportions of a particular diplotype within double or triple heterozygous genotypes (E step) by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e084&amp;representation=PNG" class="inline-graphic"><span class="note">(21)</span></span><br>The calculated relative proportions by equation (21) were used to estimate the haplotype frequencies with<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e085&amp;representation=PNG" class="inline-graphic"><span class="note">(22)</span></span><br></p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>At the lower hierarchy of the EM framework, we calculate the posterior probabilities of a double or triple heterozygous individual <em>i</em> to be a particular diplotype (<em>A̅</em>) (E step), for which where [111] is assumed as the risk haplotype, expressed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e086&amp;representation=PNG" class="inline-graphic"><span class="note">(23)</span></span><br>With the calculated posterior probabilities by the above equation (23), we then estimate the quantitative genetic parameters, Θ<em><sub>q</sub></em>, based on the log-likelihood equations. These equations have similar, but more complicated, forms like equations (12) and (13).</p>
<a id="article1.body1.sec2.sec4.p7" name="article1.body1.sec2.sec4.p7"></a><p>Hypothesis tests can be made for linkage disequilibria among three SNPs and haplotype effects. Four different linkage disequilibria, <em>D</em><sub>12</sub>, <em>D</em><sub>13</sub>, <em>D</em><sub>23</sub> and <em>D</em><sub>123</sub>, that describe the linkage among three SNPs can each be tested using the null hypotheses described by equation (21). The log-likelihood ratios for each hypothesis are thought to follow a χ<sup>2</sup> distribution.</p>
<a id="article1.body1.sec2.sec4.p8" name="article1.body1.sec2.sec4.p8"></a><p><strong><em>R</em></strong><strong>-SNP model:</strong> The idea for haplotyping a quantitative trait is described for two- and three-SNP models. It is possible that these models are too simple to characterize genetic variants for quantitative variation. With the analytical line for the two- and three-SNP sequencing model, a model can be developed to include an arbitrary number of SNPs whose sequences are associated with the phenotypic variation. A key issue for the multi-SNP sequencing model is how to distinguish among 2<em><sup>r</sup></em><sup>−1</sup> different diplotypes for the same genotype heterozygous at <em>r</em> loci. The relative frequencies of these diplotypes can be expressed in terms of haplotype frequencies. The integrative EM algorithm can be employed to estimate the MLEs of haplotype frequencies. A general formula for estimating haplotype frequencies can be derived.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The statistical model described above can be used to map and identify QTNs for a quantitative trait in an F<sub>2</sub> population. Because the marker data we have for mouse are microsatellites rather than SNPs, we use these microsatellite markers as a surrogate of SNPs for the purpose to demonstrate the utility of the model. Our marker data were from Vaughn et al.'s <a href="#pone.0000732-Vaughn1">[21]</a> study in which a linkage map composed of 19 chromosomes was constructed with 96 microsatellite markers for 502 F<sub>2</sub> mice (259 males and 243 females) derived from two strains, the Large (LG/J) and Small (SM/J). This map has a total map distance of ~1780 cM (in Haldane's units) and an average interval length of ~23 cM. The F<sub>2</sub> progeny was measured for their body mass at 10 weekly intervals starting at age 7 days. The raw weights were corrected for the effects of each covariate due to dam, litter size at birth, parity and sex <a href="#pone.0000732-Vaughn1">[21]</a>. Here, only adult body weights at week 10 are used for “QTN” analysis.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>For each F<sub>2</sub> mouse, the parental origin of alleles at each marker can be discerned in molecular studies. Let <em>L</em> and <em>S</em> be the alleles inherited from the Large (LG/J) and Small (SM/J) strains, respectively. For any pair of markers, there are four different haplotypes, <em>LL</em>, <em>LS</em>, <em>SL</em> and <em>SS</em>, whose frequencies are accordingly denoted as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e087&amp;representation=PNG" class="inline-graphic"></span><br>and <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.e088&amp;representation=PNG" class="inline-graphic"></span><br>By assuming all the four haplotypes as a risk haplotype, respectively, the above model allows for the estimates of haplotype frequencies by the EM iteration at the higher hierarchy and of composite genotypic values by the EM iteration at the lower hierarchy. The estimated haplotype frequencies are used to estimate linkage disequilibrium based on equation (14) and the recombination fraction (<em>r</em>) based on equation (4). This estimation process is moved from the first (<strong>M</strong><sub>1</sub>–<strong>M</strong><sub>2</sub>) to last pair of markers (<strong>M</strong><sub>6</sub>–<strong>M</strong><sub>7</sub>) on chromosome 1 and then from chromosome 1 to 19.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p><a href="#pone-0000732-t003">Table 3</a> tabulates the results of the MLEs of haplotype frequencies and log-likelihoods under the assumptions of different risk haplotypes. A total of 96 markers are sparsely located on 19 mouse chromosomes, with the estimated recombination fractions from the linkage disequilibrium model <a href="#pone.0000732-Li1">[8]</a> consistent with those obtained from the linkage model <a href="#pone.0000732-Vaughn1">[21]</a>. Significant likelihood ratios for testing haplotype effects were determined by critical values obtained from the <em>χ</em><sup>2</sup>-square distribution with two degrees of freedom with a Bonferroni adjustment to the type I error. The adjusted critical values for the two- and three-marker QTN models are 18.20 and 18.76, respectively, at the 5% significance level. Significant haplotype effects are detected for a total of eight marker pairs (<a href="#pone-0000732-t003">Table 3</a>), which include one pair on chromosome 4, two consecutive pairs on chromosome 6, four consecutive pairs on chromosome 7 and one pair on chromosome 14. For some pairs, multiple significant risk haplotypes were detected. Risk haplotypes purely composed of alleles inherited from the LG/J or SM/J parent exert a positive or negative additive effect on body weight, respectively. Based on the relative values of estimated additive and dominant effects, the significant marker pairs detected display partial dominant effects (<a href="#pone-0000732-t003">Table 3</a>).</p>
<div class="figure" id="pone-0000732-t003"><div class="img"><a name="pone-0000732-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000732.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000732.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>The MLEs of haplotype frequencies and significant log-likelihood ratios (LR) by assuming different risk haplotypes in the F<sub>2</sub> population of mice.</span></strong></p><span>doi:10.1371/journal.pone.0000732.t003</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The results from the three-marker model are basically consistent with those from the two-marker model (<a href="#pone-0000732-t004">Table 4</a>). The advantage of the three-marker model is that it incorporates the interferences between adjacent marker intervals into the estimation process and, thus, can potentially increase the estimation precision of haplotype effects.</p>
<div class="figure" id="pone-0000732-t004"><div class="img"><a name="pone-0000732-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000732" data-uri="info:doi/10.1371/journal.pone.0000732.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000732.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000732.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000732.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000732.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000732.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>The MLEs of haplotype frequencies and significant log-likelihood ratios (LR) by assuming different risk haplotypes in the F<sub>2</sub> population of mice.</span></strong></p><span>doi:10.1371/journal.pone.0000732.t004</span></div></div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Quantitative trait locus (QTL) mapping aims to identify narrow chromosomal segments for a quantitative trait by using a statistical method, and has proven its value to study the genetic architecture of the trait in a variety of species <a href="#pone.0000732-Mackay1">[6]</a>–<a href="#pone.0000732-Li1">[8]</a>. The limitations of this technique lie in its inability to characterize the structure and organization of DNA sequences and statistical difficulty in deriving the distribution of test statistics under the null hypothesis of no QTL <a href="#pone.0000732-Lander2">[22]</a>. At least partly for these reasons, despite thousands of QTL reported for different traits and populations, a very small portion of them have been cloned <a href="#pone.0000732-Flint1">[9]</a>. With the completion of the genome projects for several important organisms, a new line of thought in the post genomic era has begun to emerge for the identification of specific combinations of nucleotides or haplotypes that contribute to a complex quantitative trait <a href="#pone.0000732-Liu1">[13]</a>, <a href="#pone.0000732-Lin1">[23]</a>.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Theory and methods for haplotype discovery have well been established for natural populations <a href="#pone.0000732-Liu1">[13]</a> in which the non-random association among different single nucleotide polymorphims (SNP), specified by the coefficients of linkage disequilibria, lays a foundation for the mixture model of haplotyping a quantitative trait. In this article, we derived a statistical model for detecting haplotypes and estimating their effects on quantitative variation of a trait in experimental crosses. We used the principle of linkage disequilibrium analysis to characterize the linkage among different markers that is usually described by the recombination fractions in a commonly used F<sub>2</sub> population, initiated with two inbred lines. We established an interchangeable relationship between the linkage and linkage disequilibrium. The merit of this relationship in trait haplotyping includes the incorporation of interferences between adjacent marker intervals into the estimation and test of haplotype effects when multiple markers are modelled simultaneously.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The haplotyping model developed in this article was used to analyze a published F<sub>2</sub> population of mouse <a href="#pone.0000732-Vaughn1">[21]</a>, but we used microsatellite markers as the surrogate of SNPs so that we can detect the effects of haplotypes constructed by microsatellite alleles. The whole-genome of mouse was scanned for haplotype effects on body weight by a two- and multi-marker model, respectively. Consistent results were observed from the two models, which suggests that four regions in mouse chromosomes 4, 6, 7, and 14 contribute to variation in body weight. These findings are in a good agreement with those from traditional interval QTL mapping <a href="#pone.0000732-Vaughn1">[21]</a>. But our haplotype discovery is more informative in terms of the characterization of specific haplotype structure and organization responsible for trait variation.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>We have proposed a new model for haplotyping a quantitative trait in the F<sub>2</sub> progeny population. The tenet of the model can be extended to haplotype a complicated trans-generational pedigree, founded with multiple original parents and involving individuals with different relatedness. The model can also be modified to dissect the epistatic effects of different genes <a href="#pone.0000732-Lin1">[23]</a> and the interaction of genes with environment. For these extensions, haplotype selection aimed to detect the risk haplotypes that are expressed differently from the others present many challenges, but is crucial for the facilitation of the process of detecting the association between haplotype diversity and phenotypic variation.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>Our haplotyping model offers a powerful tool for positional cloning of QTL that are important for a complex trait. Flint et al. <a href="#pone.0000732-Flint1">[9]</a> reviewed the potential of currently available cloning strategies, such as probabilistic ancestral haplotype reconstruction, Yin-Yang crosses and in silico analysis of sequence variants, to identify genes that underlie QTL in rodents. Our model, in conjunction with these strategies, may open a new gateway for the illustration of a detailed picture of the genetic architecture for a complex trait.</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RW. Performed the experiments: JC. Analyzed the data: WH SW TL. Wrote the paper: RW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000732-Lander1" id="pone.0000732-Lander1"></a>Lander ES, Botstein D (1989) Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps. Genetics  121: 185–199.  <ul class="find" data-citedArticleID="976912"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mapping+Mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps.&amp;auth=&amp;atitle=Mapping+Mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+Mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+Mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000732-Lynch1" id="pone.0000732-Lynch1"></a>Lynch M, Walsh B (1998) Genetics and Analysis of Quantitative Traits. Sunderland, MA: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0000732-Jansen1" id="pone.0000732-Jansen1"></a>Jansen RC, Stam P (1994) High resolution mapping of quantitative traits into multiple loci via interval mapping. Genetics  136: 1447–1455.  <ul class="find" data-citedArticleID="976928"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=High+resolution+mapping+of+quantitative+traits+into+multiple+loci+via+interval+mapping.&amp;auth=&amp;atitle=High+resolution+mapping+of+quantitative+traits+into+multiple+loci+via+interval+mapping." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+resolution+mapping+of+quantitative+traits+into+multiple+loci+via+interval+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+resolution+mapping+of+quantitative+traits+into+multiple+loci+via+interval+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000732-Zeng1" id="pone.0000732-Zeng1"></a>Zeng Z-B (1994) Precision mapping of quantitative trait loci. Genetics  136: 1457–1468.  <ul class="find" data-citedArticleID="976896"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Precision+mapping+of+quantitative+trait+loci.&amp;auth=&amp;atitle=Precision+mapping+of+quantitative+trait+loci." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Precision+mapping+of+quantitative+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Precision+mapping+of+quantitative+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000732-Kao1" id="pone.0000732-Kao1"></a>Kao C-H, Zeng Z-B, Teasdale RD (1999) Multiple interval mapping for quantitative trait loci. Genetics  152: 1203–1216.  <ul class="find" data-citedArticleID="976922" data-doi="10.1534/genetics.103.021642"><li><a href="http://dx.doi.org/10.1534/genetics.103.021642" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+interval+mapping+for+quantitative+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+interval+mapping+for+quantitative+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000732-Mackay1" id="pone.0000732-Mackay1"></a>Mackay TFC (2001) Quantitative trait loci in <em>Drosophila</em>. Nat Rev Genet  2: 11–20.  <ul class="find" data-citedArticleID="976892" data-doi="10.1038/nrg1206"><li><a href="http://dx.doi.org/10.1038/nrg1206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+trait+loci+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+trait+loci+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000732-Frary1" id="pone.0000732-Frary1"></a>Frary A, Nesbitt TC, Frary A, Grandillo S, van der Knaap E, et al.  (2000) <em>fw2.2</em>: A quantitative trait locus key to the evolution of tomato fruit size. Science  289: 85–88.  <ul class="find" data-citedArticleID="976930" data-doi="10.1126/science.289.5476.85"><li><a href="http://dx.doi.org/10.1126/science.289.5476.85" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=fw2.2%3A+A+quantitative+trait+locus+key+to+the+evolution+of+tomato+fruit+size." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22fw2.2%3A+A+quantitative+trait+locus+key+to+the+evolution+of+tomato+fruit+size.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000732-Li1" id="pone.0000732-Li1"></a>Li CB, Zhou AL, Sang T (2006) Rice domestication by reducing shattering. Science  311: 1936–1939.  <ul class="find" data-citedArticleID="976916" data-doi="10.1126/science.1123604"><li><a href="http://dx.doi.org/10.1126/science.1123604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rice+domestication+by+reducing+shattering." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rice+domestication+by+reducing+shattering.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000732-Flint1" id="pone.0000732-Flint1"></a>Flint J, Valdar W, Shifman S, Mott R (2005) Strategies for mapping and cloning quantitative trait genes in rodents. Nat Rev Genet  6: 271–286.  <ul class="find" data-citedArticleID="976924" data-doi="10.1038/nrg1576"><li><a href="http://dx.doi.org/10.1038/nrg1576" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strategies+for+mapping+and+cloning+quantitative+trait+genes+in+rodents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strategies+for+mapping+and+cloning+quantitative+trait+genes+in+rodents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000732-Churchill1" id="pone.0000732-Churchill1"></a>Churchill GA, Doerge RW (1994) Empirical threshold values for quantitative trait mapping. Genetics  138: 963–971.  <ul class="find" data-citedArticleID="976926"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Empirical+threshold+values+for+quantitative+trait+mapping.&amp;auth=&amp;atitle=Empirical+threshold+values+for+quantitative+trait+mapping." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+threshold+values+for+quantitative+trait+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+threshold+values+for+quantitative+trait+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000732-Judson1" id="pone.0000732-Judson1"></a>Judson R, Stephens JC, Windemuth A (2000) The predictive power of haplotypes in clinical response. Pharmacogenomics  1: 15–26.  <ul class="find" data-citedArticleID="976894" data-doi="10.1517/14622416.1.1.15"><li><a href="http://dx.doi.org/10.1517/14622416.1.1.15" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+predictive+power+of+haplotypes+in+clinical+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+predictive+power+of+haplotypes+in+clinical+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000732-Bader1" id="pone.0000732-Bader1"></a>Bader JS (2001) The relative power of SNPs and haplotype as genetic markers for association tests. Pharmacogenomics  2: 11–24.  <ul class="find" data-citedArticleID="976900" data-doi="10.1517/14622416.2.1.11"><li><a href="http://dx.doi.org/10.1517/14622416.2.1.11" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+relative+power+of+SNPs+and+haplotype+as+genetic+markers+for+association+tests." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+relative+power+of+SNPs+and+haplotype+as+genetic+markers+for+association+tests.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000732-Liu1" id="pone.0000732-Liu1"></a>Liu T, Johnson JA, Casella G, Wu RL (2004) Sequencing complex diseases with HapMap. Genetics  168: 503–511.  <ul class="find" data-citedArticleID="976904" data-doi="10.1534/genetics.104.029603"><li><a href="http://dx.doi.org/10.1534/genetics.104.029603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+complex+diseases+with+HapMap." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+complex+diseases+with+HapMap.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000732-Lou1" id="pone.0000732-Lou1"></a>Lou X-Y, Casella G, Littell RC, Yang MKC, Wu RL (2003) A haplotype-based algorithm for multilocus linkage disequilibrium mapping of quantitative trait loci with epistasis in natural populations. Genetics  163: 1533–1548.  <ul class="find" data-citedArticleID="976920"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+haplotype-based+algorithm+for+multilocus+linkage+disequilibrium+mapping+of+quantitative+trait+loci+with+epistasis+in+natural+populations.&amp;auth=&amp;atitle=A+haplotype-based+algorithm+for+multilocus+linkage+disequilibrium+mapping+of+quantitative+trait+loci+with+epistasis+in+natural+populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+haplotype-based+algorithm+for+multilocus+linkage+disequilibrium+mapping+of+quantitative+trait+loci+with+epistasis+in+natural+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+haplotype-based+algorithm+for+multilocus+linkage+disequilibrium+mapping+of+quantitative+trait+loci+with+epistasis+in+natural+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000732-Patil1" id="pone.0000732-Patil1"></a>Patil N, Berno AJ, Hinds DA, Barrett WA, Doshi JM, et al.  (2001) Blocks of limited haplotype diversity revealed by high-resolution scanning of human chromosome 21. Science  294: 1719–1723.  <ul class="find" data-citedArticleID="976914" data-doi="10.1126/science.1065573"><li><a href="http://dx.doi.org/10.1126/science.1065573" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Blocks+of+limited+haplotype+diversity+revealed+by+high-resolution+scanning+of+human+chromosome+21." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Blocks+of+limited+haplotype+diversity+revealed+by+high-resolution+scanning+of+human+chromosome+21.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000732-Zhang1" id="pone.0000732-Zhang1"></a>Zhang K, Deng M, Chen T, Waterman MS, Sun F (2002) A dynamic programming algorithm for haplotype block partitioning. Proc Natl Acad Sci  99: 7335–7339.  <ul class="find" data-citedArticleID="976902" data-doi="10.1073/pnas.102186799"><li><a href="http://dx.doi.org/10.1073/pnas.102186799" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+dynamic+programming+algorithm+for+haplotype+block+partitioning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+dynamic+programming+algorithm+for+haplotype+block+partitioning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000732-Sebastiani1" id="pone.0000732-Sebastiani1"></a>Sebastiani P, Lazarus SW, Kunkel LM, Kohane IS, Ramoni M (2003) Minimal haplotype tagging. Proc Natl Acad Sci  100: 9900–9905.  <ul class="find" data-citedArticleID="976906" data-doi="10.1073/pnas.1633613100"><li><a href="http://dx.doi.org/10.1073/pnas.1633613100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Minimal+haplotype+tagging." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Minimal+haplotype+tagging.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000732-Eyheramendy1" id="pone.0000732-Eyheramendy1"></a>Eyheramendy S, Marchini J, McVean G, Myers S, Donnelly P (2007) A model-based approach to capture genetic variation for future association studies. Genome Res  17: 88–95.  <ul class="find" data-citedArticleID="976908" data-doi="10.1101/gr.5675406"><li><a href="http://dx.doi.org/10.1101/gr.5675406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+model-based+approach+to+capture+genetic+variation+for+future+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+model-based+approach+to+capture+genetic+variation+for+future+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000732-Louis1" id="pone.0000732-Louis1"></a>Louis TA (1982) Finding the observed information matrix when using the EM algorithm. J Roy Stat Soc Ser B  44: 226–233.  <ul class="find" data-citedArticleID="976932"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Finding+the+observed+information+matrix+when+using+the+EM+algorithm.&amp;auth=&amp;atitle=Finding+the+observed+information+matrix+when+using+the+EM+algorithm." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finding+the+observed+information+matrix+when+using+the+EM+algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finding+the+observed+information+matrix+when+using+the+EM+algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000732-Burnham1" id="pone.0000732-Burnham1"></a>Burnham KP, Andersson DR (1998) Model Selection and Inference. A Practical Information-Theoretic Approach. New York: Springer.   <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0000732-Vaughn1" id="pone.0000732-Vaughn1"></a>Vaughn TT, Pletscher LS, Peripato A, King-Ellison K, Adams E, et al.  (1999) Mapping quantitative trait loci for murine growth - A closer look at genetic architecture. Genet Res  74: 313–322.  <ul class="find" data-citedArticleID="976910" data-doi="10.1017/s0016672399004103"><li><a href="http://dx.doi.org/10.1017/s0016672399004103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+quantitative+trait+loci+for+murine+growth+-+A+closer+look+at+genetic+architecture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+quantitative+trait+loci+for+murine+growth+-+A+closer+look+at+genetic+architecture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000732-Lander2" id="pone.0000732-Lander2"></a>Lander ES, Schork NJ (1994) Genetic dissection of complex traits. Science  265: 2037–2048.  <ul class="find" data-citedArticleID="976934" data-doi="10.1126/science.8091226"><li><a href="http://dx.doi.org/10.1126/science.8091226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+dissection+of+complex+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+dissection+of+complex+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000732-Lin1" id="pone.0000732-Lin1"></a>Lin M, Wu RL (2006) Detecting sequence-sequence interactions for complex diseases. Current Genomics  7: 59–72.  <ul class="find" data-citedArticleID="976918" data-doi="10.2174/138920206776389775"><li><a href="http://dx.doi.org/10.2174/138920206776389775" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+sequence-sequence+interactions+for+complex+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+sequence-sequence+interactions+for+complex+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.XML" value="117563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.PDF" value="208040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.g001.PNG_L" value="173774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.g001.PNG_M" value="601420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.g001.PNG_S" value="25129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.g001.TIF" value="248286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.g001.PNG_I" value="75003"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.g002.PNG_L" value="186025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.g002.PNG_M" value="282102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.g002.PNG_S" value="13643"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.g002.TIF" value="290710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.g002.PNG_I" value="29888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e001.PNG" value="7066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e001.TIF" value="21596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e002.PNG" value="26444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e002.TIF" value="40496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e003.PNG" value="8132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e003.TIF" value="22756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e004.PNG" value="7798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e004.TIF" value="22404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e005.PNG" value="7672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e005.TIF" value="22272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e006.PNG" value="14065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e006.TIF" value="28132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e007.PNG" value="8184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e007.TIF" value="22788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e008.PNG" value="7493"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e008.TIF" value="22084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e009.PNG" value="7475"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e009.TIF" value="22068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e010.PNG" value="8264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e010.TIF" value="22860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e011.PNG" value="8327"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e011.TIF" value="22968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e012.PNG" value="8238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e012.TIF" value="22876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e013.PNG" value="43799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e013.TIF" value="59380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e014.PNG" value="28874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e014.TIF" value="45856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t001.PNG_L" value="130133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t001.PNG_M" value="187607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t001.PNG_S" value="18188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t001.TIF" value="629356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t001.PNG_I" value="26340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e015.PNG" value="8288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e015.TIF" value="23136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e016.PNG" value="8385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e016.TIF" value="22976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e017.PNG" value="7189"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e017.TIF" value="21776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e018.PNG" value="7364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e018.TIF" value="21892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e019.PNG" value="7364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e019.TIF" value="21892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e020.PNG" value="7364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e020.TIF" value="21892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e021.PNG" value="7364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e021.TIF" value="21892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e022.PNG" value="7396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e022.TIF" value="21936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e023.PNG" value="13775"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e023.TIF" value="27940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e024.PNG" value="9330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e024.TIF" value="24076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e025.PNG" value="8391"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e025.TIF" value="23104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e026.PNG" value="45969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e026.TIF" value="60812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e027.PNG" value="12064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e027.TIF" value="26500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e028.PNG" value="10599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e028.TIF" value="25032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e029.PNG" value="27606"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e029.TIF" value="43312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e030.PNG" value="13316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e030.TIF" value="28304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e031.PNG" value="37309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e031.TIF" value="51576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e032.PNG" value="19617"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e032.TIF" value="34492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e033.PNG" value="13009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e033.TIF" value="27752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e034.PNG" value="9593"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e034.TIF" value="24148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e035.PNG" value="15262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e035.TIF" value="29120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e036.PNG" value="9350"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e036.TIF" value="23860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e037.PNG" value="9470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e037.TIF" value="23936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e038.PNG" value="9481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e038.TIF" value="23948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e039.PNG" value="9413"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e039.TIF" value="23920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e040.PNG" value="6906"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e040.TIF" value="21428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e041.PNG" value="9688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e041.TIF" value="24192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e042.PNG" value="12693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e042.TIF" value="27632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e043.PNG" value="11699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e043.TIF" value="26460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e044.PNG" value="12060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e044.TIF" value="26712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t002.PNG_L" value="257613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t002.PNG_M" value="348613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t002.PNG_S" value="27584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t002.TIF" value="1388154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t002.PNG_I" value="45500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e045.PNG" value="8667"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e045.TIF" value="23632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e046.PNG" value="8687"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e046.TIF" value="23308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e047.PNG" value="7028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e047.TIF" value="21624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e048.PNG" value="7066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e048.TIF" value="21820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e049.PNG" value="6978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e049.TIF" value="21620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e050.PNG" value="8673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e050.TIF" value="23660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e051.PNG" value="8693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e051.TIF" value="23324"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e052.PNG" value="7154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e052.TIF" value="21692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e053.PNG" value="7066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e053.TIF" value="21820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e054.PNG" value="6978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e054.TIF" value="21620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e055.PNG" value="8639"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e055.TIF" value="23600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e056.PNG" value="8737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e056.TIF" value="23328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e057.PNG" value="7106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e057.TIF" value="21696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e058.PNG" value="7066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e058.TIF" value="21820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e059.PNG" value="6978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e059.TIF" value="21620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e060.PNG" value="12540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e060.TIF" value="28904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e061.PNG" value="12399"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e061.TIF" value="27548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e062.PNG" value="8628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e062.TIF" value="23556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e063.PNG" value="8858"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e063.TIF" value="23620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e064.PNG" value="8300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e064.TIF" value="23120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e065.PNG" value="8965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e065.TIF" value="23820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e066.PNG" value="8811"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e066.TIF" value="23392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e067.PNG" value="7144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e067.TIF" value="21724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e068.PNG" value="7066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e068.TIF" value="21820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e069.PNG" value="7071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e069.TIF" value="21660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e070.PNG" value="8966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e070.TIF" value="23812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e071.PNG" value="8857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e071.TIF" value="23508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e072.PNG" value="7135"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e072.TIF" value="21704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e073.PNG" value="7066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e073.TIF" value="21820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e074.PNG" value="7071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e074.TIF" value="21660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e075.PNG" value="8922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e075.TIF" value="23824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e076.PNG" value="8842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e076.TIF" value="23532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e077.PNG" value="7074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e077.TIF" value="21692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e078.PNG" value="7066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e078.TIF" value="21820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e079.PNG" value="7071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e079.TIF" value="21660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e080.PNG" value="8004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e080.TIF" value="22604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e081.PNG" value="15040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e081.TIF" value="29652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e082.PNG" value="10389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e082.TIF" value="25004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e083.PNG" value="242218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e083.TIF" value="269112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e084.PNG" value="57268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e084.TIF" value="73144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e085.PNG" value="80854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e085.TIF" value="102676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e086.PNG" value="56316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e086.TIF" value="73472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e087.PNG" value="7897"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e087.TIF" value="22404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e088.PNG" value="8807"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e088.TIF" value="23360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t003.PNG_L" value="136092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t003.PNG_M" value="187603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t003.PNG_S" value="20541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t003.TIF" value="676500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t003.PNG_I" value="24881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e089.PNG" value="6409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e089.TIF" value="20976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e090.PNG" value="6166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e090.TIF" value="20744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e091.PNG" value="6266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e091.TIF" value="20828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e092.PNG" value="6278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e092.TIF" value="20848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t004.PNG_L" value="132610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t004.PNG_M" value="180070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t004.PNG_S" value="19473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t004.TIF" value="689282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.t004.PNG_I" value="24011"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e093.PNG" value="6134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e093.TIF" value="20724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e094.PNG" value="6271"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000732.e094.TIF" value="20832"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000732&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000732" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000732&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000732&volume=&issue=&title=Haplotyping a Quantitative Trait with a High-Density Map in Experimental Crosses&author_name=Wei%20Hou%2C%20John%20Stephen%20F.%20Yap%2C%20Song%20Wu%2C%20Tian%20Liu%2C%20James%20M.%20Cheverud%2C%20Rongling%20Wu&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000732" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000732" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000732" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000732&amp;t=Haplotyping%20a%20Quantitative%20Trait%20with%20a%20High-Density%20Map%20in%20Experimental%20Crosses" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000732&title=Haplotyping%20a%20Quantitative%20Trait%20with%20a%20High-Density%20Map%20in%20Experimental%20Crosses&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000732&amp;title=Haplotyping%20a%20Quantitative%20Trait%20with%20a%20High-Density%20Map%20in%20Experimental%20Crosses" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000732" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000732&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Haplotyping a Quantitative Trait with a High-Density Map in Experimental Crosses';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000732';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Haplotyping%20a%20Quantitative%20Trait%20with%20a%20High-Density%20Map%20in%20Experimental%20Crosses http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000732" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000732" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplotypes%22" title="Search for articles in the subject area:'Haplotypes'"><div class="flagText">Haplotypes</div></a>
              <div data-categoryid="32131" data-articleid="24302"
                   data-categoryname="Haplotypes"
                   class="flagImage" title="Flag 'Haplotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Linkage+disequilibrium%22" title="Search for articles in the subject area:'Linkage disequilibrium'"><div class="flagText">Linkage disequilibrium</div></a>
              <div data-categoryid="42897" data-articleid="24302"
                   data-categoryname="Linkage disequilibrium"
                   class="flagImage" title="Flag 'Linkage disequilibrium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="24302"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotides%22" title="Search for articles in the subject area:'Nucleotides'"><div class="flagText">Nucleotides</div></a>
              <div data-categoryid="17443" data-articleid="24302"
                   data-categoryname="Nucleotides"
                   class="flagImage" title="Flag 'Nucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="24302"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Quantitative+traits%22" title="Search for articles in the subject area:'Quantitative traits'"><div class="flagText">Quantitative traits</div></a>
              <div data-categoryid="24563" data-articleid="24302"
                   data-categoryname="Quantitative traits"
                   class="flagImage" title="Flag 'Quantitative traits' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Statistical+models%22" title="Search for articles in the subject area:'Statistical models'"><div class="flagText">Statistical models</div></a>
              <div data-categoryid="33295" data-articleid="24302"
                   data-categoryname="Statistical models"
                   class="flagImage" title="Flag 'Statistical models' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Variant+genotypes%22" title="Search for articles in the subject area:'Variant genotypes'"><div class="flagText">Variant genotypes</div></a>
              <div data-categoryid="40547" data-articleid="24302"
                   data-categoryname="Variant genotypes"
                   class="flagImage" title="Flag 'Variant genotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5722'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8274'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9641&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
