

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Ribonuclease Activity of Dis3 Is Required for Mitotic Progression and Provides a Possible Link between Heterochromatin and Kinetochore Function</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000317"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000317" />

    <meta name="citation_title" content="Ribonuclease Activity of Dis3 Is Required for Mitotic Progression and Provides a Possible Link between Heterochromatin and Kinetochore Function"/>
    <meta itemprop="name" content="Ribonuclease Activity of Dis3 Is Required for Mitotic Progression and Provides a Possible Link between Heterochromatin and Kinetochore Function"/>

      <meta name="citation_author" content="Hiroaki Murakami"/>
            <meta name="citation_author_institution" content="CREST Research Program, Japan Science and Technology Corporation, Department of Gene Mechanisms, Graduate School of Biostudies, Kyoto University, Yoshida-Honmachi, Sakyo-ku, Kyoto, Japan"/>
      <meta name="citation_author" content="Derek B. Goto"/>
            <meta name="citation_author_institution" content="Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, United States of America"/>
      <meta name="citation_author" content="Takashi Toda"/>
            <meta name="citation_author_institution" content="Laboratory of Cell Regulation, Cancer Research UK, London Research Institute, Lincoln's Inn Fields Laboratories, London, United Kingdom"/>
      <meta name="citation_author" content="Ee Sin Chen"/>
            <meta name="citation_author_institution" content="Laboratory of Molecular Cell Biology, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Shiv I. Grewal"/>
            <meta name="citation_author_institution" content="Laboratory of Molecular Cell Biology, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Robert A. Martienssen"/>
            <meta name="citation_author_institution" content="Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, United States of America"/>
      <meta name="citation_author" content="Mitsuhiro Yanagida"/>
            <meta name="citation_author_institution" content="CREST Research Program, Japan Science and Technology Corporation, Department of Gene Mechanisms, Graduate School of Biostudies, Kyoto University, Yoshida-Honmachi, Sakyo-ku, Kyoto, Japan"/>

    <meta name="citation_date" content="2007/3/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000317.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e317"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Basic mechanism of eukaryotic chromosome segregation.; citation_author=M Yanagida; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=360; citation_number=1; citation_pages=609-621; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cold-sensitive and caffeine-supersensitive mutants of the Schizosaccharomyces pombe dis genes implicated in sister chromatid separation during mitosis.; citation_author=H Ohkura; citation_author=Y Adachi; citation_author=N Kinoshita; citation_author=O Niwa; citation_author=T Toda; citation_journal_title=EMBO J; citation_volume=7; citation_number=2; citation_pages=1465-1473; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=p93dis1, which is required for sister chromatid separation, is a novel microtubule and spindle pole body-associating protein phosphorylated at the Cdc2 target sites.; citation_author=K Nabeshima; citation_author=H Kurooka; citation_author=M Takeuchi; citation_author=K Kinoshita; citation_author=Y Nakaseko; citation_journal_title=Genes Dev; citation_volume=9; citation_number=3; citation_pages=1572-1585; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=M phase-specific kinetochore proteins in fission yeast: microtubule-associating Dis1 and Mtc1 display rapid separation and segregation during anaphase.; citation_author=Y Nakaseko; citation_author=G Goshima; citation_author=J Morishita; citation_author=M Yanagida; citation_journal_title=Curr Biol; citation_volume=11; citation_number=4; citation_pages=537-549; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The fission yeast dis2+ gene required for chromosome disjoining encodes one of two putative type 1 protein phosphatases.; citation_author=H Ohkura; citation_author=N Kinoshita; citation_author=S Miyatani; citation_author=T Toda; citation_author=M Yanagida; citation_journal_title=Cell; citation_volume=57; citation_number=5; citation_pages=997-1007; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Phosphorylation of dis2 protein phosphatase at the C-terminal cdc2 consensus and its potential role in cell cycle regulation.; citation_author=H Yamano; citation_author=K Ishii; citation_author=M Yanagida; citation_journal_title=EMBO J; citation_volume=13; citation_number=6; citation_pages=5310-5318; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=CDC2 phosphorylation of the fission yeast dis1 ensures accurate chromosome segregation.; citation_author=K Aoki; citation_author=Y Nakaseko; citation_author=K Kinoshita; citation_author=G Goshima; citation_author=M Yanagida; citation_journal_title=Curr Biol; citation_volume=16; citation_number=7; citation_pages=1627-1635; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The exosome: a conserved eukaryotic RNA processing complex containing multiple 3′→5′ exoribonucleases.; citation_author=P Mitchell; citation_author=E Petfalski; citation_author=A Shevchenko; citation_author=M Mann; citation_author=D Tollervey; citation_journal_title=Cell; citation_volume=91; citation_number=8; citation_pages=457-466; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Making ribosomes.; citation_author=A Fatica; citation_author=D Tollervey; citation_journal_title=Curr Opin Cell Biol; citation_volume=14; citation_number=9; citation_pages=313-318; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=RNA-quality control by the exosome.; citation_author=J Houseley; citation_author=J LaCava; citation_author=D Tollervey; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=7; citation_number=10; citation_pages=529-539; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Unravelling the dynamics of RNA degradation by ribonuclease II and its RNA-bound complex.; citation_author=C Frazao; citation_author=CE McVey; citation_author=M Amblar; citation_author=A Barbas; citation_author=C Vonrhein; citation_journal_title=Nature; citation_volume=443; citation_number=11; citation_pages=110-114; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structural basis for processivity and single-strand specificity of RNase II.; citation_author=Y Zuo; citation_author=HA Vincent; citation_author=J Zhang; citation_author=Y Wang; citation_author=MP Deutscher; citation_journal_title=Mol Cell; citation_volume=24; citation_number=12; citation_pages=149-156; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Isolation of murine and human homologues of the fission-yeast dis3+ gene encoding a mitotic-control protein and its overexpression in cancer cells with progressive phenotype.; citation_author=J Lim; citation_author=T Kuroki; citation_author=K Ozaki; citation_author=H Kohsaki; citation_author=T Yamori; citation_journal_title=Cancer Res; citation_volume=57; citation_number=13; citation_pages=921-925; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Human dis3p, which binds to either GTP- or GDP-Ran, complements Saccharomyces cerevisiae dis3.; citation_author=T Shiomi; citation_author=K Fukushima; citation_author=N Suzuki; citation_author=N Nakashima; citation_author=E Noguchi; citation_journal_title=J Biochem (Tokyo); citation_volume=123; citation_number=14; citation_pages=883-890; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=An evolutionarily conserved fission yeast protein, Ned1, implicated in normal nuclear morphology and chromosome stability, interacts with Dis3, Pim1/RCC1 and an essential nucleoporin.; citation_author=Y Tange; citation_author=A Hirata; citation_author=O Niwa; citation_journal_title=J Cell Sci; citation_volume=115; citation_number=15; citation_pages=4375-4385; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The fission yeast dis3+ gene encodes a 110-kDa essential protein implicated in mitotic control.; citation_author=N Kinoshita; citation_author=M Goebl; citation_author=M Yanagida; citation_journal_title=Mol Cell Biol; citation_volume=11; citation_number=16; citation_pages=5839-5847; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Oscillatory nuclear movement in fission yeast meiotic prophase is driven by astral microtubules, as revealed by continuous observation of chromosomes and microtubules in living cells.; citation_author=DQ Ding; citation_author=Y Chikashige; citation_author=T Haraguchi; citation_author=Y Hiraoka; citation_journal_title=J Cell Sci; citation_volume=111; citation_number=17; citation_pages=701-712; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Dynamics of centromeres during metaphase-anaphase transition in fission yeast: Dis1 is implicated in force balance in metaphase bipolar spindle.; citation_author=K Nabeshima; citation_author=T Nakagawa; citation_author=AF Straight; citation_author=A Murray; citation_author=Y Chikashige; citation_journal_title=Mol Biol Cell; citation_volume=9; citation_number=18; citation_pages=3211-3225; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Fission yeast Cut1 and Cut2 are essential for sister chromatid separation, concentrate along the metaphase spindle and form large complexes.; citation_author=H Funabiki; citation_author=K Kumada; citation_author=M Yanagida; citation_journal_title=EMBO J; citation_volume=15; citation_number=19; citation_pages=6617-6628; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Feedback control of mitosis in budding yeast.; citation_author=R Li; citation_author=AW Murray; citation_journal_title=Cell; citation_volume=66; citation_number=20; citation_pages=519-531; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Fission yeast Slp1: an effector of the Mad2-dependent spindle checkpoint.; citation_author=SH Kim; citation_author=DP Lin; citation_author=S Matsumoto; citation_author=A Kitazono; citation_author=T Matsumoto; citation_journal_title=Science; citation_volume=279; citation_number=21; citation_pages=1045-1047; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Requirement of chromatid cohesion proteins rad21/scc1 and mis4/scc2 for normal spindle-kinetochore interaction in fission yeast.; citation_author=Y Toyoda; citation_author=K Furuya; citation_author=G Goshima; citation_author=K Nagao; citation_author=K Takahashi; citation_journal_title=Curr Biol; citation_volume=12; citation_number=22; citation_pages=347-358; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Fission yeast bub1 is a mitotic centromere protein essential for the spindle checkpoint and the preservation of correct ploidy through mitosis.; citation_author=P Bernard; citation_author=K Hardwick; citation_author=JP Javerzat; citation_journal_title=J Cell Biol; citation_volume=143; citation_number=23; citation_pages=1775-1787; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Spindle-kinetochore attachment requires the combined action of Kin I-like Klp5/6 and Alp14/Dis1-MAPs in fission yeast.; citation_author=MA Garcia; citation_author=N Koonrugsa; citation_author=T Toda; citation_journal_title=EMBO J; citation_volume=21; citation_number=24; citation_pages=6015-6024; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cell division cycle genes nda2 and nda3 of the fission yeast Schizosaccharomyces pombe control microtubular organization and sensitivity to anti-mitotic benzimidazole compounds.; citation_author=K Umesono; citation_author=T Toda; citation_author=S Hayashi; citation_author=M Yanagida; citation_journal_title=J Mol Biol; citation_volume=168; citation_number=25; citation_pages=271-284; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Mal3, the fission yeast homologue of the human APC-interacting protein EB-1 is required for microtubule integrity and the maintenance of cell form.; citation_author=JD Beinhauer; citation_author=IM Hagan; citation_author=JH Hegemann; citation_author=U Fleig; citation_journal_title=J Cell Biol; citation_volume=139; citation_number=26; citation_pages=717-728; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Fission yeast ch-TOG/XMAP215 homologue Alp14 connects mitotic spindles with the kinetochore and is a component of the Mad2-dependent spindle checkpoint.; citation_author=MA Garcia; citation_author=L Vardy; citation_author=N Koonrugsa; citation_author=T Toda; citation_journal_title=EMBO J; citation_volume=20; citation_number=27; citation_pages=3389-3401; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The A78V mutation in the Mad3-like domain of Schizosaccharomyces pombe Bub1p perturbs nuclear accumulation and kinetochore targeting of Bub1p, Bub3p, and Mad3p and spindle assembly checkpoint function.; citation_author=S Kadura; citation_author=X He; citation_author=V Vanoosthuyse; citation_author=KG Hardwick; citation_author=S Sazer; citation_journal_title=Mol Biol Cell; citation_volume=16; citation_number=28; citation_pages=385-395; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cold-sensitive nuclear division arrest mutants of the fission yeast Schizosaccharomyces pombe.; citation_author=T Toda; citation_author=K Umesono; citation_author=A Hirata; citation_author=M Yanagida; citation_journal_title=J Mol Biol; citation_volume=168; citation_number=29; citation_pages=251-270; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Exoribonuclease superfamilies: structural analysis and phylogenetic distribution.; citation_author=Y Zuo; citation_author=MP Deutscher; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=30; citation_pages=1017-1026; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Overexpression, purification, and properties of Escherichia coli ribonuclease II.; citation_author=GA Coburn; citation_author=GA Mackie; citation_journal_title=J Biol Chem; citation_volume=271; citation_number=31; citation_pages=1048-1053; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Regulation of heterochromatic silencing and histone H3 lysine-9 methylation by RNAi.; citation_author=TA Volpe; citation_author=C Kidner; citation_author=IM Hall; citation_author=G Teng; citation_author=SI Grewal; citation_journal_title=Science; citation_volume=297; citation_number=32; citation_pages=1833-1837; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=RNAi-mediated targeting of heterochromatin by the RITS complex.; citation_author=A Verdel; citation_author=S Jia; citation_author=S Gerber; citation_author=T Sugiyama; citation_author=S Gygi; citation_journal_title=Science; citation_volume=303; citation_number=33; citation_pages=672-676; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Two RNAi complexes, RITS and RDRC, physically interact and localize to noncoding centromeric RNAs.; citation_author=MR Motamedi; citation_author=A Verdel; citation_author=SU Colmenares; citation_author=SA Gerber; citation_author=SP Gygi; citation_journal_title=Cell; citation_volume=119; citation_number=34; citation_pages=789-802; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=RNA interference and heterochromatin in the fission yeast Schizosaccharomyces pombe.; citation_author=RA Martienssen; citation_author=M Zaratiegui; citation_author=DB Goto; citation_journal_title=Trends Genet; citation_volume=21; citation_number=35; citation_pages=450-456; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A low copy number central sequence with strict symmetry and unusual chromatin structure in fission yeast centromere.; citation_author=K Takahashi; citation_author=S Murakami; citation_author=Y Chikashige; citation_author=H Funabiki; citation_author=O Niwa; citation_journal_title=Mol Biol Cell; citation_volume=3; citation_number=36; citation_pages=819-835; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Distinct protein interaction domains and protein spreading in a complex centromere.; citation_author=JF Patridge; citation_author=B Borgstrom; citation_author=RC Allshire; citation_journal_title=Genes Dev.; citation_volume=14; citation_number=37; citation_pages=783-791; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sim4: a novel fission yeast kinetochore protein required for centromeric silencing and chromosome segregation.; citation_author=AL Pidoux; citation_author=W Richardson; citation_author=RC Allshire; citation_journal_title=J Cell Biol; citation_volume=161; citation_number=38; citation_pages=295-307; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Mal2p protein is an essential component of the fission yeast centromere.; citation_author=Q-W Jin; citation_author=AL Pidoux; citation_author=C Decker; citation_author=RC Allshire; citation_author=U Fleig; citation_journal_title=Mol Cell Biol; citation_volume=22; citation_number=39; citation_pages=7168-7183; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Small RNAs correspond to centromere heterochromatic repeats.; citation_author=BJ Reinhart; citation_author=DP Bartel; citation_journal_title=Science; citation_volume=297; citation_number=40; citation_pages=1831; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The fission yeast chromo domain encoding gene chp1(+) is required for chromosome segregation and shows a genetic interaction with alpha-tubulin.; citation_author=CL Doe; citation_author=G Wang; citation_author=C Chow; citation_author=MD Fricker; citation_author=PB Singh; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=41; citation_pages=4222-4229; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=swi6, a gene required for mating-type switching, prohibits meiotic recombination in the mat2-mat3 “cold spot” of fission yeast.; citation_author=AJ Klar; citation_author=MJ Bonaduce; citation_journal_title=Genetics; citation_volume=129; citation_number=42; citation_pages=1033-1042; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Switching gene swi6, involved in repression of silent mating-type loci in fission yeast, encodes a homologue of chromatin-associated proteins from Drosophila and mammals.; citation_author=A Lorentz; citation_author=K Ostermann; citation_author=O Fleck; citation_author=H Schmidt; citation_journal_title=Gene; citation_volume=143; citation_number=43; citation_pages=139-143; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Mutations derepressing silent centromeric domains in fission yeast disrupt chromosome segregation.; citation_author=RC Allshire; citation_author=ER Nimmo; citation_author=K Ekwall; citation_author=JP Javerzat; citation_author=G Cranston; citation_journal_title=Genes Dev; citation_volume=9; citation_number=44; citation_pages=218-233; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Establishment and maintenance of a heterochromatin domain.; citation_author=IM Hall; citation_author=GD Shankaranarayana; citation_author=K Noma; citation_author=N Ayoub; citation_author=A Cohen; citation_journal_title=Science; citation_volume=297; citation_number=45; citation_pages=2232-2237; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The chromatin structure of centromeres from fission yeast: differentiation of the central core that correlates with function.; citation_author=C Polizzi; citation_author=L Clarke; citation_journal_title=J. Cell Biol.; citation_volume=112; citation_number=46; citation_pages=191-201; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Mis6, a fission yeast inner centromere protein, acts during G1/S and forms specialized chromatin required for equal segregation.; citation_author=S Saitoh; citation_author=K Takahashi; citation_author=M Yanagida; citation_journal_title=Cell; citation_volume=90; citation_number=47; citation_pages=131-143; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Proper metaphase spindle length is determined by centromere proteins Mis12 and Mis6 required for faithful chromosome segregation.; citation_author=G Goshima; citation_author=S Saitoh; citation_author=M Yanagida; citation_journal_title=Genes Dev; citation_volume=13; citation_number=48; citation_pages=1664-1677; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Requirement of Mis6 centromere connector for localizing a CENP-A-like protein in fission yeast.; citation_author=K Takahashi; citation_author=ES Chen; citation_author=M Yanagida; citation_journal_title=Science; citation_volume=288; citation_number=49; citation_pages=2215-2219; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Human centromere chromatin protein hMis12, essential for equal segregation, is independent of CENP-A loading pathway.; citation_author=G Goshima; citation_author=T Kiyomitsu; citation_author=K Yoda; citation_author=M Yanagida; citation_journal_title=J Cell Biol; citation_volume=160; citation_number=50; citation_pages=25-39; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Length control of the metaphase spindle.; citation_author=G Goshima; citation_author=R Wollman; citation_author=N Stuurman; citation_author=JM Scholey; citation_author=RD Vale; citation_journal_title=Curr Biol; citation_volume=15; citation_number=51; citation_pages=1979-1988; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The dhp1(+) gene, encoding a putative nuclear 5′→3′ exoribonuclease, is required for proper chromosome segregation in fission yeast.; citation_author=T Shobuike; citation_author=K Tatebayashi; citation_author=T Tani; citation_author=S Sugano; citation_author=H Ikeda; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=52; citation_pages=1326-1333; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Inactivation of the pre-mRNA cleavage and polyadenylation factor Pfs2 in fission yeast causes lethal cell cycle defects.; citation_author=SW Wang; citation_author=K Asakawa; citation_author=TZ Win; citation_author=T Toda; citation_author=CJ Norbury; citation_journal_title=Mol Cell Biol; citation_volume=25; citation_number=53; citation_pages=2288-2296; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Requirement of fission yeast Cid14 in polyadenylation of rRNAs.; citation_author=TZ Win; citation_author=S Draper; citation_author=RL Read; citation_author=J Pearce; citation_author=CJ Norbury; citation_journal_title=Mol Cell Biol; citation_volume=26; citation_number=54; citation_pages=1710-1721; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Characterization of Schizosaccharomyces pombe minichromosome deletion derivatives and a functional allocation of their centromere.; citation_author=O Niwa; citation_author=T Matsumoto; citation_author=Y Chikashige; citation_author=M Yanagida; citation_journal_title=EMBO J.; citation_volume=8; citation_number=55; citation_pages=3045-3052; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=BAF53/Arp4 homolog Alp5 in fission yeast is required for histone H4 acetylation, kinetochore-spindle attachment, and gene silencing at centromere.; citation_author=A Minoda; citation_author=S Saitoh; citation_author=K Takahashi; citation_author=T Toda; citation_journal_title=Mol Cell Biol; citation_volume=16; citation_number=56; citation_pages=316-327; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Argonaute slicing is required for heterochromatic silencing and spreading.; citation_author=DV Irvine; citation_author=M Zaratiegui; citation_author=NH Tolia; citation_author=DB Goto; citation_author=DH Chitwood; citation_journal_title=Science; citation_volume=313; citation_number=57; citation_pages=1134-1137; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A single subunit, Dis3, is essentially responsible for yeast exosome core activity.; citation_author=A Dziembowski; citation_author=E Lorentzen; citation_author=E Conti; citation_author=B Seraphin; citation_journal_title=Nat Struct Mol Biol; citation_volume=14; citation_number=58; citation_pages=15-22; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The chromodomain protein Swi6: a key component at fission yeast centromeres.; citation_author=K Ekwall; citation_author=JP Javerzat; citation_author=A Lorentz; citation_author=H Schmidt; citation_author=G Cranston; citation_journal_title=Science; citation_volume=269; citation_number=59; citation_pages=1429-1431; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Cut1 is loaded onto the spindle by binding to Cut2 and promotes anaphase spindle movement upon Cut2 proteolysis.; citation_author=K Kumada; citation_author=T Nakamura; citation_author=K Nagao; citation_author=H Funabiki; citation_author=T Nakagawa; citation_journal_title=Curr Biol; citation_volume=8; citation_number=60; citation_pages=633-641; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The use of cell division cycle mutants to investigate the control of microtubule distribution in the fission yeast Schizosaccharomyces pombe.; citation_author=IM Hagan; citation_author=JS Hyams; citation_journal_title=J Cell Sci; citation_volume=89; citation_number=61; citation_pages=343-357; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Definition of individual components within the cytoskeleton of Trypanosoma brucei by a library of monoclonal antibodies.; citation_author=A Woods; citation_author=T Sherwin; citation_author=R Sasse; citation_author=TH MacRae; citation_author=AJ Baines; citation_journal_title=J Cell Sci; citation_volume=93; citation_number=62; citation_pages=491-500; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Two novel proteins, dos1 and dos2, interact with rik1 to regulate heterochromatic RNA interference and histone modification.; citation_author=F Li; citation_author=DB Goto; citation_author=M Zaratiegui; citation_author=X Tang; citation_author=R Martienssen; citation_journal_title=Curr Biol; citation_volume=15; citation_number=63; citation_pages=1448-1457; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RNA-dependent RNA polymerase is an essential component of a self-enforcing loop coupling heterochromatin assembly to siRNA production.; citation_author=T Sugiyama; citation_author=H Cam; citation_author=A Verdel; citation_author=D Moazed; citation_author=SI Grewal; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=64; citation_pages=152-157; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The solution structure of the S1 RNA binding domain: a member of an ancient nucleic acid-binding fold.; citation_author=M Bycroft; citation_author=TJ Hubbard; citation_author=M Proctor; citation_author=SM Freund; citation_author=AG Murzin; citation_journal_title=Cell; citation_volume=88; citation_number=65; citation_pages=235-242; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000317" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Ribonuclease Activity of Dis3 Is Required for Mitotic Progression and Provides a Possible Link between Heterochromatin and Kinetochore Function"/>
    <meta name="twitter:description" content="BackgroundCellular RNA metabolism has a broad range of functional aspects in cell growth and division, but its role in chromosome segregation during mitosis is only poorly understood. The Dis3 ribonuclease is a key component of the RNA-processing exosome complex. Previous isolation of the dis3-54 cold-sensitive mutant of fission yeast Schizosaccharomyces pombe suggested that Dis3 is also required for correct chromosome segregation.Methodology/Principal FindingsWe show here that the progression of mitosis is arrested in dis3-54, and that segregation of the chromosomes is blocked by activation of the mitotic checkpoint control. This block is dependent on the Mad2 checkpoint protein. Double mutant and inhibitor analyses revealed that Dis3 is required for correct kinetochore formation and function, and that this activity is monitored by the Mad2 checkpoint. Dis3 is a member of the highly conserved RNase II family and is known to be an essential subunit of the exosome complex. The dis3-54 mutation was found to alter the RNaseII domain of Dis3, which caused a reduction in ribonuclease activity in vitro. This was associated with loss of silencing of an ura4+ reporter gene inserted into the outer repeats (otr) and central core (cnt and imr) regions of the centromere. On the other hand, centromeric siRNA maturation and formation of the RITS RNAi effector complex was normal in the dis3-54 mutant. Micrococcal nuclease assay also suggested the overall chromatin structure of the centromere was not affected in dis3-54 mutant.Conclusions/SignificanceRNase activity of Dis3, a core subunit of exosome, was found to be required for proper kinetochore formation and establishment of kinetochore-microtubule interactions. Moreover, Dis3 was suggested to contribute to kinetochore formation through an involvement in heterochromatic silencing at both outer centromeric repeats and within the central core region. This activity is likely monitored by the mitotic checkpoint, and distinct from that of RNAi-mediated heterochromatin formation directly targeting outer centromeric repeats."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000317.g008"/>

  <meta property="og:title" content="Ribonuclease Activity of Dis3 Is Required for Mitotic Progression and Provides a Possible Link between Heterochromatin and Kinetochore Function" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000317" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3897'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1926'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6463&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000317">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000317" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000317&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Ribonuclease Activity of Dis3 Is Required for Mitotic Progression and Provides a Possible Link between Heterochromatin and Kinetochore Function
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hiroaki Murakami, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  CREST Research Program, Japan Science and Technology Corporation, Department of Gene Mechanisms, Graduate School of Biostudies, Kyoto University, Yoshida-Honmachi, Sakyo-ku, Kyoto, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Derek B. Goto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, United States of America
                </p>

                <p>
                    Current address: Laboratory for Nanosystems Physiology, Research Institute for Electronic Science, Hokkaido University, Sapporo, Japan
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takashi Toda, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Cell Regulation, Cancer Research UK, London Research Institute, Lincoln's Inn Fields Laboratories, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ee Sin Chen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Molecular Cell Biology, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shiv I. Grewal, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Molecular Cell Biology, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robert A. Martienssen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mitsuhiro Yanagida
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:yanagida@kozo.lif.kyoto-u.ac.jp">yanagida@kozo.lif.kyoto-u.ac.jp</a></p>

                <p>Affiliation:
                  CREST Research Program, Japan Science and Technology Corporation, Department of Gene Mechanisms, Graduate School of Biostudies, Kyoto University, Yoshida-Honmachi, Sakyo-ku, Kyoto, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000317</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000317" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000317" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000317" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000317" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000317-g001" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000317-g002" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000317-g003" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000317-g004" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000317-g005" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000317-g006" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000317-g007" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000317-g008" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000317">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000317" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Cellular RNA metabolism has a broad range of functional aspects in cell growth and division, but its role in chromosome segregation during mitosis is only poorly understood. The Dis3 ribonuclease is a key component of the RNA-processing exosome complex. Previous isolation of the <em>dis3-54</em> cold-sensitive mutant of fission yeast <em>Schizosaccharomyces pombe</em> suggested that Dis3 is also required for correct chromosome segregation.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We show here that the progression of mitosis is arrested in <em>dis3-54</em>, and that segregation of the chromosomes is blocked by activation of the mitotic checkpoint control. This block is dependent on the Mad2 checkpoint protein. Double mutant and inhibitor analyses revealed that Dis3 is required for correct kinetochore formation and function, and that this activity is monitored by the Mad2 checkpoint. Dis3 is a member of the highly conserved RNase II family and is known to be an essential subunit of the exosome complex. The <em>dis3-54</em> mutation was found to alter the RNaseII domain of Dis3, which caused a reduction in ribonuclease activity <em>in vitro</em>. This was associated with loss of silencing of an <em>ura4<sup>+</sup></em> reporter gene inserted into the outer repeats (<em>otr</em>) and central core (<em>cnt</em> and <em>imr</em>) regions of the centromere. On the other hand, centromeric siRNA maturation and formation of the RITS RNAi effector complex was normal in the <em>dis3-54</em> mutant. Micrococcal nuclease assay also suggested the overall chromatin structure of the centromere was not affected in <em>dis3-54</em> mutant.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>RNase activity of Dis3, a core subunit of exosome, was found to be required for proper kinetochore formation and establishment of kinetochore-microtubule interactions. Moreover, Dis3 was suggested to contribute to kinetochore formation through an involvement in heterochromatic silencing at both outer centromeric repeats and within the central core region. This activity is likely monitored by the mitotic checkpoint, and distinct from that of RNAi-mediated heterochromatin formation directly targeting outer centromeric repeats.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Murakami H, Goto DB, Toda T, Chen ES, Grewal SI, et al.  (2007) Ribonuclease Activity of Dis3 Is Required for Mitotic Progression and Provides a Possible Link between Heterochromatin and Kinetochore Function. PLoS ONE 2(3):
          e317.
            doi:10.1371/journal.pone.0000317</p><p><strong>Academic Editor: </strong>Beth Sullivan, Duke University, United States of America</p><p><strong>Received:</strong> January 17, 2007; <strong>Accepted:</strong> March 5, 2007; <strong>Published:</strong> March 21, 2007</p><p><strong>Copyright:</strong> © 2007 Murakami et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a Specially Promoted Center of Excellence (COE) grant from the Ministry of Education, Culture, Sports, Science and Technology to M.Y., a Core Research for Evolutional Science and Technology (CREST) research grant from the Japan Science and Technology Corporation (JST) to M.Y., and a grant from the National Institutes of Health (R01-GM067014) to R.M. D.G. was a DOE-Energy Biosciences Research Fellow of the Life Sciences Research Foundation. T.T. is supported by Cancer Research UK.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Accurate chromosome segregation during mitosis is an essential event for proper cell multiplication. A large number of the genes required for different aspects of chromosome segregation have been identified. This includes genes encoding proteins implicated in centromere/kinetochore structure and function, sister chromatid cohesion, and chromosome condensation, together with proteins required for the spindle assembly checkpoint, activation of APC (anaphase promoting complex)/C (cyclosome) for ubiquitin-dependent proteolysis, and for anaphase promoting proteolysis that leads to the destruction of securin and the activation of separase <a href="#pone.0000317-Yanagida1">[1]</a>. In addition, various protein kinases such as Cdc2, polo and aurora and related protein phosphatases are implicated in mitotic control. These essential genes for chromosome segregation are conserved throughout eukaryotic organisms.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>A series of fission yeast <em>Schizosaccharomyces pombe</em> cold sensitive mutants have been previously isolated that showed severe defects in sister chromatid separation at 20°C <a href="#pone.0000317-Ohkura1">[2]</a>. These mutants, termed <em>dis1, dis2</em>, and <em>dis3 (<span class="underline">d</span>efective <span class="underline">i</span>n <span class="underline">s</span>ister chromatid disjoining)</em>, also contain hyper-condensed chromosomes at the restrictive temperature. Dis1 is a microtubule binding protein that is required for the proper attachment of kinetochore microtubules to the kinetochores, and is localized at the kinetochore specifically during mitosis <a href="#pone.0000317-Nabeshima1">[3]</a>, <a href="#pone.0000317-Nakaseko1">[4]</a>. Dis1 is also the founding member of the Dis1-XMAP215-chTOG family of microtubule-associating proteins that are universally present in eukaryotes. Dis2 is one of the two catalytic subunits (Dis2, Sds21) of Type 1 protein phosphatase PP1. The <em>dis2-11</em> mutation is semi-dominant and blocks sister chromatid separation at the restrictive temperature <a href="#pone.0000317-Ohkura2">[5]</a>. Both Dis1 and Dis2 proteins are phosphorylated and their activities are regulated by Cdc2 kinase <a href="#pone.0000317-Yamano1">[6]</a>, <a href="#pone.0000317-Aoki1">[7]</a>. Cdc2 down regulates the phosphatase activity of Dis2 and up regulates the kinetochore localization of Dis1 by direct phosphorylation during mitosis.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Unlike Dis1 and Dis2, the role of Dis3 in proper kinetochore function or mitotic progression is not clear. Dis3 is a relatively large protein (110 kD) that includes the characteristic RNase II (RNB) domain, which is highly conserved in bacteria, archaea and eukaryotes. RNase II contains a highly processive 3′→5′ exoribonuclease activity, and eukaryotic Dis3 homologs (Rrp44 in budding yeast) were identified as a component of the exosome that is required for RNA maturation and turnover in both the nucleus and cytoplasm <a href="#pone.0000317-Mitchell1">[8]</a>–<a href="#pone.0000317-Houseley1">[10]</a>. Rrp44/Dis3 is one of more than ten protein components in the exosome, which can act upon different RNA substrates including mRNA, snRNA, snoRNA, rRNA and tRNA. The structure of <em>E. coli</em> RNase II was recently determined <a href="#pone.0000317-Frazao1">[11]</a>, <a href="#pone.0000317-Zuo1">[12]</a> and revealed that the active site in the RNB domain is composed of four Asp residues and one Arg residue in two respective motifs referred to as Motif I and Motif IV. Mg<sup>2+</sup> ion is chelated by the first and fourth asp residues in Motif I.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The human homologue of Dis3 was identified as an oncogene <a href="#pone.0000317-Lim1">[13]</a>, consistent with a potential role in chromosome segregation and/or mitotic progression. Human Dis3 is tightly bound to Ran a small G protein <a href="#pone.0000317-Shiomi1">[14]</a>. Fission yeast Ned1, a homologue of human disease gene product lipin, directly interacts with Dis3, Pim1/Rcc1 (GEF for Ran) and nuclear pore proteins, and is implicated in complex cellular functions including chromosome organization and nucleo-cytoplasmic transport <a href="#pone.0000317-Tange1">[15]</a>. The Dis3 protein is located in both the nucleus and cytoplasm throughout the cell division cycle, and deletion of the <em>dis3<sup>+</sup></em> gene in fission yeast showed that it is essential for cell viability <a href="#pone.0000317-Kinoshita1">[16]</a>. The mitotically arrested phenotype is thus specific to the <em>dis3-54</em> mutant allele upon shifting to the restrictive temperature.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In order to determine how Dis3 contributes to mitotic progression, and more importantly, whether this represents a role for Dis3 ribonuclease in kinetochore formation/function, we have focused on the <em>dis3-54</em> mutant of fission yeast. Here, we report that the <em>dis3-54</em> mutation resides in the Dis3 RNB domain, and that the mutant protein contains a reduced RNase activity. We show that the spindle-kinetochore interaction is abnormal in <em>dis3-54</em> mutant cells and that this is associated with mitotic arrest at a pre-anaphase stage, which is dependent on the essential spindle assembly (mitotic) checkpoint protein, Mad2. We also show evidence that Dis3 is involved in a heterochromatic silencing process within the centromere.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Arrested <em>dis3-54</em> mutant cells show an elongated metaphase-like spindle</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Cold-sensitive <em>dis3-54</em> mutant cells grown at 33°C were shifted to the restrictive temperature of 20°C and incubated for 4 hr to induce the nuclear division arrest phenotype. Arrested cells were fixed with methanol and the chromosomal DNA and microtubules were stained using DAPI and anti-tubulin antibodies, respectively. As expected, mutant cells displayed the <em>dis</em>-like phenotype <a href="#pone.0000317-Ohkura1">[2]</a>, in that sister chromatids appeared not be separated and condensed chromosomes were scattered along the mitotic spindle (<a href="#pone-0000317-g001">Fig. 1<em>A</em></a>). The average spindle length in <em>dis3-54</em> mutant cells arrested at 20°C (4.5 µm, <a href="#pone-0000317-g001">Fig. 1<em>B</em></a>) was longer than that of the wild-type metaphase spindle (2.5 µm), but much shorter than the fully extended anaphase B spindle in wild-type (approximately 10–13 µm).</p>
<div class="figure" id="pone-0000317-g001"><div class="img"><a name="pone-0000317-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Elongated metaphase-like spindle in arrested <em>dis3-54</em> mutant cells. (A) Mitotic spindles in wild-type (WT) and <em>dis3-54</em> mutant cells at the restrictive temperature of 20°C. Cells grown in liquid culture were at 33°C were shifted to the restrictive temperature of 20°C for 4 hr and fixed with methanol. Anti-tubulin antibody TAT1 (green) and DAPI (red) were used for immunostaining and DNA staining, respectively. WT cells containing nuclei consistent with both metaphase and anaphase were observed as indicated, whereas <em>dis3-54</em> cells appeared to have arrested during mitosis with abnormally long metaphase-like spindles. Scale bar = 10 µm. (B) Mitotic spindle length distributions in WT and <em>dis3-54</em> mutant cells grown at restrictive temperature of 20°C for 4 hr. Cells were stained with anti-Sad1 (spindle pole body component) and anti-tubulin antibody after fixation with 3% paraformaldehyde and the pole-to-pole distance of the spindle microtubule in mitotic cells was measured. The inverted triangle represents the position of mode. (C) Time-lapse images of <em>dis3-54</em> mutant cells expressing chromosomally integrated GFP-tagged α2-tubulin (Atb2) under the inducible <em>nmt1</em> promoter. Images of cells growing at both 20°C (restrictive temperature, left panels) and 33°C (permissive temperature, right) were taken at 30 sec intervals for approx. 90 min with a confocal microscope. Numbers within each panel represent the time (min) after start of observation. The white arrow indicates the collapsed spindle microtubule, often observed after an extended period of mitotic arrest in <em>dis3-54</em>. (D) Time course plot of spindle length based on time-lapse images shown in Fig. 1<em>C</em>. Cells grown at 33°C (blue line) and 20°C (yellow line; “cell 1”) were measured and the 0 min time point was adjusted to reflect the start of mitosis. Normal changes in spindle length representing phases I, II and III were observed during the time-course at 33°C <a href="#pone.0000317-Nabeshima2">[18]</a>.</p>
<span>doi:10.1371/journal.pone.0000317.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In order to monitor spindle changes over a period of time in actual living cells, a chromosomally integrated GFP-tagged α2-tubulin gene <a href="#pone.0000317-Ding1">[17]</a> was introduced into <em>dis3-54</em>. Mutant cells expressing GFP-tubulin both at restrictive (20°C) and permissive (33°C) temperatures were examined using a confocal microscope system equipped with a temperature control unit. Time-lapse images of cells cultured at 20°C and 33°C were obtained (representative cells shown in <a href="#pone-0000317-g001">Fig. 1<em>C</em></a>; <a href="#pone.0000317.s002">Movies S1</a>, <a href="#pone.0000317.s003">S2</a> and <a href="#pone.0000317.s004">S3</a>) and used for quantitative measurements of spindle lengths following initiation of mitosis (<a href="#pone-0000317-g001">Fig. 1<em>D</em></a>). At the restrictive temperature of 20°C, phase 1 (period of spindle formation; <a href="#pone.0000317-Nabeshima2">[18]</a>) occurred normally in <em>dis3-54</em> cells with an initial spindle length of 2.5–3.0 µm, however, the cells then appeared to remain in phase 2 (constant spindle length period; metaphase) as the spindle slowly reached a length of 4–6 µm within 90 min. Phase 3 (spindle extension period; anaphase B) did not take place and the spindle often collapsed after prolonged incubation at 20°C (indicated by white arrows in <a href="#pone-0000317-g001">Fig. 1<em>C</em></a>). In contrast, <em>dis3-54</em> cells grown at the permissive temperature (33°C) displayed normal spindle elongation and completed anaphase with a long anaphase B spindle within 30 min after entry into phase 1.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To determine whether <em>dis3-54</em> cells cultured at 20°C were arrested at a pre-anaphase stage or following the switch to anaphase, the <em>dis3-54</em> mutation was combined with a chromosomally integrated copy of a GFP-tagged Cut2/securin gene under the control of its native promoter. Cut2/securin acts to prevent sister chromatid separation during metaphase and is degraded upon the triggering of anaphase <a href="#pone.0000317-Funabiki1">[19]</a>. In wild-type cells, Cut2-GFP signals were associated with condensed chromosomes during metaphase and disappeared in all anaphase and post-anaphase cells. In contrast, <em>dis3-54</em> cells cultured at 20°C constantly showed intense Cut2-GFP signals that remained with the condensed chromosomes and did not disappear (<a href="#pone-0000317-g002">Fig. 2</a>). This confirmed that anaphase (sister-chromatid separation) did not occur and <em>dis3-54</em> cells were indeed arrested at a pre-anaphase stage. Taken together, the above data indicates that a pre-anaphase spindle is produced in arrested <em>dis3-54</em> cells grown at the restrictive temperature, which remains present for 60–80 min.</p>
<div class="figure" id="pone-0000317-g002"><div class="img"><a name="pone-0000317-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Mitotic arrest in <em>dis3-54</em> cells occurs at a pre-anaphase stage. WT and <em>dis3-54</em> mutant strains that carried a chromosomally integrated Cut2/securin-GFP fusion gene under control of the native promoter were cultured at 20°C for 4 hr in synthetic EMM2 medium. Cells were stained with Hoechst H33342 for visualizing DNA together with GFP signals, and were not subject to fixation. WT cells in both pre-anaphase (p) and anaphase (a) were observed, whereas persistent Cut2 signals in <em>dis3-54</em> (three representative cells shown) confirmed they were in a pre-anaphase stage. Scale bar = 10 µm.</p>
<span>doi:10.1371/journal.pone.0000317.g002</span></div>

<h4>Mitotic arrest of <em>dis3-54</em> involves the checkpoint protein Mad2</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Correct progression through mitosis is essential for cell survival and it is therefore carefully monitored by several proteins that prevent completion of the mitotic cycle should a potentially lethal problem be encountered <a href="#pone.0000317-Li1">[20]</a>, <a href="#pone.0000317-Kim1">[21]</a>. Mad2 is known to restrain the mitotic progression by preventing the activation of ubiquitin-dependent anaphase promoting proteolysis. Although <em>dis3-54</em> cells were found to be arrested at a pre-anaphase stage, it was unclear whether this was due to the <em>dis3-54</em> mutation directly preventing sister chromatid separation, or due to <em>dis3-54</em> resulting in activation of the mitotic checkpoint to halt mitotic progression at the pre-anaphase stage.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To distinguish between these two possibilities, we first investigated the localization of two spindle assemble checkpoint proteins, Mad2 and Bub1, in wild-type and mutant cells using chromosomally integrated GFP-tagged versions of these genes driven by their native promoter. Intense kinetochore dot signals corresponding to Mad2-GFP were observed in most <em>dis3-54</em> cells arrested at 20°C, whereas such signals were only normally seen in a small number of wild-type mitotic cells (<a href="#pone-0000317-g003">Fig. 3<em>A</em></a>). Similar results were also obtained from the analysis of Bub1-GFP. In wild-type cells, Bub1-GFP was only observed in early mitotic stages, however. Intense kinetochore localization of Bub1-GFP was observed in most of the arrested <em>dis3-54</em> cells (<a href="#pone-0000317-g003">Fig. 3<em>B</em></a>).</p>
<div class="figure" id="pone-0000317-g003"><div class="img"><a name="pone-0000317-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Pre-anaphase arrest in <em>dis3-54</em> is dependent on the spindle checkpoint protein Mad2. (A) Cellular localization of Mad2-GFP in WT and <em>dis3-54</em> mutant cells at 20°C. WT and <em>dis3-54</em> strains carrying a chromosomally integrated copy of Mad2-GFP were shifted to 20°C for 8 hr prior to imaging. Cells were stained with Hoechst H33342 for DNA and were not subject to fixation. WT cells in both pre-anaphase (p) and anaphase (a) were observed. All mitotically arrested <em>dis3-54</em> mutant cells contained intense Mad2-GFP signals (n = 23). (B) Cellular localization of Bub1-GFP in WT and <em>dis3-54</em> at 20°C. WT and <em>dis3-54</em> strains carrying a chromosomally integrated copy of Bub1-GFP were cultured as in A, however, cells were first fixed in methanol prior to staining with Hoechst H33342. Bub1-GFP dot signals were observed in WT cells only during the pre-anaphase stage, whereas intense signals were observed in all mitotically arrested <em>dis3-54</em> mutant cells (n = 24). Scale bar = 10 µm. (C) Full mitotic spindle elongation and asymmetrical nuclei in a <em>dis3-54 Δmad2</em> double mutant as observed by immunostaining. Cells were grown and treated as in <a href="#pone-0000317-g001">Fig. 1<em>A</em></a>. (D) Cellular localization of Cut2-GFP in the <em>dis3-54 Δmad2</em> double mutant at 20°C. Loss of Cut2 signals demonstrates progression from pre-anaphase (p) to anaphase (a) in the absence of Mad2, similar to that observed in WT (compare with <a href="#pone-0000317-g002">Fig. 2</a>).</p>
<span>doi:10.1371/journal.pone.0000317.g003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>To confirm that the observed presence and kinetochore localization of Mad2 was functionally contributing to the mitotic arrest in <em>dis3-54</em>, the mutant cells were crossed with a Mad2 deletion mutant (<em>Δmad2</em>). In striking contrast to that of single <em>dis3-54</em> mutants, the spindle was fully extended in the double mutant cultured at 20°C (<a href="#pone-0000317-g003">Fig. 3<em>C</em></a>) and similar to that observed in wild-type (see <a href="#pone-0000317-g001">Fig. 1<em>A</em></a>). Analysis of GFP-tagged Cut2/securin localization in the double mutant also revealed loss of Cut2 associated with the appearance of daughter nuclei following pre-anaphase (<a href="#pone-0000317-g003">Fig. 3<em>D</em></a>). This was similar to that observed in wild-type (see <a href="#pone-0000317-g002">Fig. 2</a>) and demonstrated that in the absence of Mad2, <em>dis3-54</em> mutant cells are able to overcome the arrest at pre-anaphase and proceed to anaphase. Two daughter nuclei that were asymmetrical in size or lagging chromosomes were often observed in the <em>dis3-54 Δmad2</em> double mutant cells, suggesting that although the spindle was extending normally, chromosome segregation was unequal. In both fission yeast and animal cells, the spindle checkpoint proteins Mad2 and Bub1 have been shown to be recruited to kinetochores that are not attached by kinetochore microtubules or in which the stable bipolar attachment is not established <a href="#pone.0000317-Toyoda1">[22]</a>–<a href="#pone.0000317-Garcia1">[24]</a>. The above data strongly indicates that normal bipolar attachment between the kinetochore and spindle is impaired in <em>dis3-54</em> mutant cells, and the resulting spindle checkpoint inhibits the metaphase/anaphase transition.</p>


<h4>Sister chromatid separation and abnormal segregation in <em>dis3-54 Δmad2</em></h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To investigate further whether sister chromatid separation could occur when the spindle checkpoint was compromised in <em>dis3-54</em> cells, the nuclear phenotype of both single <em>dis3-54</em> and double <em>dis3-54 Δmad2</em> mutants was examined over an extended time period following shift to the restrictive 20°C temperature. As shown in <a href="#pone-0000317-g004">Fig. 4<em>A</em></a>, almost all single mutant cells contained the typical <em>dis</em>-like pre-anaphase nuclei (see <a href="#pone-0000317-g001">Fig. 1<em>A</em></a>). In contrast, the proportion of cells with the “asymmetrical daughter nuclei” phenotype (see <a href="#pone-0000317-g003">Fig. 3<em>C</em></a>) greatly increased in the double mutant cells and approximately 60% of the double mutant cells showed the asymmetrical nuclei phenotype by 4 hr after the shift to 20°C.</p>
<div class="figure" id="pone-0000317-g004"><div class="img"><a name="pone-0000317-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Removal of the mitotic checkpoint reveals unequal chromosome segregation in a <em>dis3-54 Δmad2</em> double mutant. (A) Time-dependent changes in frequency of different nuclei phenotypes observed in <em>dis3-54</em> single (left panel) and <em>dis3-54 Δmad2</em> double (right panel) mutants upon shift to the restrictive temperature of 20°C. Red line, <em>dis</em> nuclei phenotype (see <a href="#pone-0000317-g001">Fig. 1<em>A</em></a>); Blue line, asymmetrical nuclei and lagging chromosomes (see <a href="#pone-0000317-g003">Fig. 3<em>C</em></a>). (B) Localization of the CenI-GFP in the wild-type (WT), <em>dis3-54</em> single mutant and <em>dis3-54 Δmad2</em> double mutant cells. Cells were cultured and treated with Hoechst as in <a href="#pone-0000317-g001">Fig. 1<em>A</em></a>. Sister chromatids were able to separate in the double mutant, but chromosome segregation was abnormal. (C) <em>dis3-54</em> and <em>dis3-54 Δmad2</em> mutants are hypersensitive to TBZ. Serial dilutions (1:5) of each strain were spotted onto YPD solid media containing 0 and 10 µg/ml TBZ and incubated at 30°C. The cold- and TBZ-sensitive <em>nda2</em> mutant strain defective in α1-tubulin was used as control <a href="#pone.0000317-Umesono1">[25]</a>.</p>
<span>doi:10.1371/journal.pone.0000317.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To directly observe how the sister centromeres were segregating in individual <em>dis3-54</em> and double <em>dis3-54 Δmad2</em> mutant cells, we crossed these mutants with a strain that expresses LacI-GFP and contains lacO repeats integrated at the <em>lys1</em> locus that is close to the centromere of chromosome 1 (<em>cen1</em>-GFP; <a href="#pone.0000317-Nabeshima2">[18]</a>). All wild-type cells contained a single <em>cen1</em>-GFP spot at opposite ends of the cell corresponding to each copy of chromosome 1 (<a href="#pone-0000317-g004">Fig. 4<em>B</em></a>), confirming normal segregation during anaphase. On the other hand, one or two <em>cen1</em>-GFP spots on the single pair of condensed chromosomes could be observed in the single <em>dis3-54</em> mutant cells arrested at 20°C, consistent with the notion that they were arrested at a metaphase-like stage. In the double mutant <em>dis3-54 Δmad2</em> cells, however, approximately 55% of anaphase cells showed the two segregated spots as seen in wild-type. The remaining double mutant cells contained two closely situated spots within the same daughter nucleus, suggesting that the sister chromatids had separated but did not move to opposite nuclei. This data indicates that although sister chromatid separation occurred in the double mutant upon loss of the mitotic checkpoint, chromosome segregation towards the opposite poles became abnormal, possibly due to lack of bi-orientation in paired sister centromeres that then moved to the poles in an apparently random fashion and caused the high frequency of unequal segregation.</p>


<h4>Hypersensitivity to a microtubule poison</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Fission yeast mutants defective in kinetochore-microtubule attachment are usually hypersensitive to microtubule destabilizing drugs such as thiabendazole, TBZ <a href="#pone.0000317-Toyoda1">[22]</a>, <a href="#pone.0000317-Umesono1">[25]</a>–<a href="#pone.0000317-Kadura1">[28]</a>. We examined whether this was also the case for the single <em>dis3-54</em> and double <em>dis3-54 Δmad2</em> mutants. Serial dilution of mutant cells were spotted onto YPD plates containing 0 and 10 µg/ml concentrations of TBZ, together with the control hypersensitive α1-tubulin mutant, <em>nda2</em><a href="#pone.0000317-Toda1">[29]</a>. Both <em>dis3-54</em> and <em>dis3-54 Δmad2</em> were hypersensitive to TBZ even at the semi-permissive temperature of 30°C (<a href="#pone-0000317-g004">Fig. 4<em>C</em></a>). The double mutant <em>dis3-54 Δmad2</em> also showed a recognizable synthetic effect, as the cells grew slowly in comparison with single <em>dis3-54</em> or <em>Δmad2</em> mutant cells.</p>


<h4>
          <em>S. pombe</em> Dis3 contains an RNase activity that is reduced by the <em>dis3-54</em> mutation</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>In order to further understand the role for Dis3 in mitosis, the causative mutation in <em>dis3-54</em> was first determined by sequencing the <em>dis3</em> gene in <em>dis3-54</em> mutant cells. A single nucleotide substitution was identified (C1690→T1690) that resulted in an amino acid substitution from Pro509 to Leu509 (<a href="#pone-0000317-g005">Fig. 5<em>A</em></a>). Pro509 is located within the central region of Dis3 that contains a highly conserved RNB domain <a href="#pone.0000317-Zuo2">[30]</a>. The RNB domain has been shown to be a catalytic module in RNaseII family proteins <a href="#pone.0000317-Frazao1">[11]</a>, <a href="#pone.0000317-Coburn1">[31]</a> and Pro509 is conserved in all eukaryotic homologues, although the prokaryotic <em>E. coli</em> RNase II contains a serine residue at the equivalent position.</p>
<div class="figure" id="pone-0000317-g005"><div class="img"><a name="pone-0000317-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>The <em>dis3-54</em> mutation alters the Dis3 RNase II domain and causes reduced RNase activity. (A) The Dis3 protein contains the PINc, RNB and S1 domains. PINc belongs to a large family of protein domains associated with exonuclease or nucleotide binding activity. S1 is an RNA binding domain <a href="#pone.0000317-Bycroft1">[65]</a>. The RNaseII RNB domain contains four high consensus regions, Motif I, II, III and IV <a href="#pone.0000317-Zuo2">[30]</a>. Amino acid sequences from Motifs I and IV of <em>E. coli</em> (EcRNaseII), <em>S. pombe</em> (SpDis3), <em>Saccharomyces cerevisiae</em> (ScDis3) and <em>Human</em> (Hs) are compared. Identity and similarity are indicated as black and gray boxes, respectively. The <em>dis3-54</em> mutation alter the Pro509 residue to Leu (P509L) in Motif I, while another mutation affecting RNase activity (T808A) is located in Motif IV. (B) <em>In vitro</em> RNase activity of WT, P509L and T808A mutant Dis3-GST proteins at 37°C (top panel) and 20°C (bottom panel). Wild-type and mutant Dis3-GST proteins (1 pmol/lane) purified from <em>S. pombe</em> cells were incubated with total <em>S. pombe</em> RNA (1 µg/lane) as described in <a href="#s4">Materials and Methods</a>. Reactions were stopped at the indicated time points and subject to denatured agarose gel electrophoresis. RNA was detected by ethidium bromide staining. 25S and 18S rRNA bands are indicated. (C) Quantitation of three independent experiments represented in Fig. 5<em>B</em>. Total ethidium bromide signal intensities for the major 25S and 18S rRNA bands were measured using an Image Analyzer (Bio-Rad) and normalized against the intensities for the 0 min time points. (D) A multicopy plasmid carrying the wild-type <em>dis3</em> gene is able to complement the cold sensitivity of <em>dis3-54</em> mutant at 20°C on YPD media, whereas equivalent plasmids carrying either the <em>dis3-54</em> (<em>P509L</em>) or <em>dis3-T808A</em> RNase mutant genes are unable to complement the cold sensitivity.</p>
<span>doi:10.1371/journal.pone.0000317.g005</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We next attempted to confirm that Dis3 does indeed contain ribonuclease activity, and if so, whether this is affected by the <em>dis3-54</em> mutation. Plasmids were constructed that contained either wild-type Dis3 (WT) or <em>dis3-54</em> mutant Dis3 <em>(P509L)</em> fused to a GST tag under control of the <em>nmt1</em> inducible promoter. Another residue in the RNB domain, Thr808, has been shown to be located in the catalytic region of RNB by crystallographic structure analyses <a href="#pone.0000317-Frazao1">[11]</a>, <a href="#pone.0000317-Zuo1">[12]</a> and differs from Pro509 in that it is also highly conserved in prokaryotes, although its contribution to RNase activity was yet to be determined. We therefore targeted this residue as a control and constructed a third plasmid with RNB mutant Dis3 (T808A) fused to GST. After overproduction in fission yeast cells, GST fusion proteins were purified and <em>in vitro</em> RNase assays were performed at 37°C and 20°C using total RNA isolated from fission yeast as the substrate <a href="#pone.0000317-Mitchell1">[8]</a>.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>When the substrate RNA was incubated with control GST protein only, the RNA remained stable throughout the reaction periods. In contrast, the substrate RNA was rapidly degraded in a time-course dependent manner when incubated with wild-type Dis3-GST protein (<a href="#pone-0000317-g005">Fig. 5<em>B</em></a>). This demonstrates that fission yeast Dis3 is indeed a ribonuclease that exhibits activity towards at least rRNAs <em>in vitro</em>. The RNase activity of wild-type Dis3 was somewhat slower at 20°C compared to that at 37°C, as most RNA was degraded within 15 min at 37°C compared to substrate RNA still remaining by 60 min at 20°C.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>The <em>dis3-54</em> (P509L) and RNB (T808A) mutant Dis3 proteins were assayed in a similar manner to determine whether these mutations alter Dis3 RNase activity. Time-course analysis of substrate RNA degradation at 37°C showed that the P509L substitution result in reduced RNase activity of Dis3. In reactions containing wild-type Dis3, 50% of substrate RNA was degraded within 4 min at 37°C, whereas approximately 10 min was required for the same level of degradation with P509L and T808A mutant Dis3 (<a href="#pone-0000317-g005">Fig. 5<em>C</em></a>). At 20°C, the effect of P509L and T808A mutations on RNase activity was more severe. Approximately 110 min was required for 50% degradation of substrate RNA by P509L mutant Dis3, in contrast to only 26 min for wild-type Dis3. The T808A mutation severely disrupted Dis3 activity as more than 50% of the target RNA still remained after 240 minutes.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>The <em>dis3-54</em> mutation can be complemented with a plasmid containing a wild-type copy of the <em>dis3<sup>+</sup></em> gene. To confirm that the reduction in RNase activity of Dis3 is directly linked to the <em>dis3</em> phenotype, we constructed equivalent Dis3 expression plasmid containing either the P509L or T808A mutations. As expected, P509L mutant Dis3 was unable to complement the <em>dis3-54</em> cold-sensitive (<em>cs)</em> phenotype. Expression of RNB mutant (T808A) Dis3 also failed to complement the <em>cs</em> phenotype, confirming the importance of Dis3 RNase activity for its kinetochore function (<a href="#pone-0000317-g005">Fig. 5<em>D</em></a>).</p>


<h4>The <em>dis3-54</em> mutant contains a centromeric silencing defect</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Formation of <em>S. pombe</em> centromeric heterochromatin requires a pathway involving RNA metabolism <a href="#pone.0000317-Volpe1">[32]</a>–<a href="#pone.0000317-Martienssen1">[35]</a>. To examine whether Dis3 may play a role in this pathway, centromeric silencing in <em>dis3-54</em> mutant cells was first tested. The fission yeast centromere consists of a central core region (<em>cnt</em>) flanked by inverted repeat domains (<em>imr</em>) and outer heterochromatic repeats (<em>otr</em>) <a href="#pone.0000317-Takahashi1">[36]</a>. Silencing in the central core region involved factors distinct from that at the outer centromeric repeats <a href="#pone.0000317-Patridge1">[37]</a>, and also involves kinetochore proteins such as Mis6, Cnp1, Mal2, and Sim4 <a href="#pone.0000317-Pidoux1">[38]</a>, <a href="#pone.0000317-Jin1">[39]</a>. The <em>dis3-54</em> mutation was combined with <em>ura4<sup>+</sup></em> marker genes inserted into either the heterochromatic <em>otr</em> region, inner <em>imr1</em> region, or central core <em>cnt1</em> regions (<em>otr, imr1, and cnt1::ura4</em><sup>+</sup>, see <a href="#pone-0000317-g006">Fig. 6</a>). Silencing of the <em>ura4</em><sup>+</sup> reporter gene in the centromere context was examined. Control strains included a Dicer deletion RNAi mutant that is defective in heterochromatin silencing and formation (<em>Δdcr1 otr::ura4<sup>+</sup></em>; <a href="#pone.0000317-Volpe1">[32]</a>), a <em>mis6</em> kinetochore mutant that is defective in central core silencing at 33°C (<em>mis6 cnt1::ura4</em><sup>+</sup>; <a href="#pone.0000317-Jin1">[39]</a>), wild-type Dis3 containing the reporter construct (WT <em>otr::ura4<sup>+</sup></em>), and a <em>dis3-54</em> mutant lacking the <em>ura4</em> gene completely (<em>dis3-54 ura4<sup>−</sup></em>) .</p>
<div class="figure" id="pone-0000317-g006"><div class="img"><a name="pone-0000317-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6. </strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>The <em>dis3-54</em> mutant shows defects in centromeric silencing. Centromeric silencing of four <em>S. pombe</em> strains, wild-type (WT), Dicer deletion RNAi mutant (<em>Δdcr1</em>), Mis6 kinetochore mutant (<em>mis6</em>) and single <em>dis3-54</em> mutant, was examined in a genetic background containing the <em>ura4</em><sup>+</sup> reporter gene integrated into different regions of the centromere (<em>otr::ura4, imr1::ura4, cnt1::ura4</em>). Schematic diagram of <em>S. pombe</em> centromere I (upper panel) shows the <em>ura4<sup>+</sup></em> insertion sites. A strain without the <em>otr::ura4</em> reporter was included as a control (<em>ura4</em><sup>−</sup>). Silencing of a reporter gene integrated into the mating type locus (<em>mat1)</em> was also examined for <em>dis3-54</em>. Serial dilutions (1:5) of each strain were plated on YPD media without selection (N/S) and on YPD media containing 5-FOA (inhibitor of Ura+ strains), and incubated at permissive (33°C) and restrictive (26°C) temperatures as indicated. Note that <em>mis6</em> is a heat-sensitive mutant and that the permissive and restrictive temperatures for <em>mis6</em> are 26°C and 33°C, respectively.</p>
<span>doi:10.1371/journal.pone.0000317.g006</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Expression of the integrated <em>ura4<sup>+</sup></em> gene was assayed by spotting serial dilutions of each strain on either non-selective media (N/S) or counter-selective media containing FOA (inhibitor of growth of Ura4<sup>+</sup>-expressing cells). The <em>otr::ura4<sup>+</sup></em> construct was normally silenced in the wild-type strain such that cells could grow in the presence of FOA, whereas growth of the control non-silenced strain <em>Δdcr1 otr::ura4<sup>+</sup></em> was inhibited by FOA (<a href="#pone-0000317-g006">Fig. 6</a>). In <em>dis3-54</em> mutant cells at 26°C (the semi-permissive temperature), silencing at the outer <em>otr</em> repeat and inner <em>imr1</em> regions was clearly reduced as FOA inhibited colony formation of <em>dis3-54 otr::ura4</em><sup>+</sup> and <em>dis3-54 imr1::ura4</em><sup>+</sup>. The <em>ura4<sup>+</sup></em> gene inserted at <em>cnt1</em> central core region was also desilenced in the <em>dis3-54</em> mutant at the semi-permissive 26°C. This silencing defect was more severe than that at the <em>imr1</em> and <em>otr</em> regions, and could be observed even at the permissive 33°C (<a href="#pone-0000317-g006">Fig. 6</a>). The Dis3 protein therefore has a role in silencing in all three major centromeric regions. In addition to the centromeric silencing defect, we found that silencing at the heterochromatic mating type locus (mat) was also reduced in the <em>dis3-54</em> mutant (<a href="#pone-0000317-g006">Fig. 6</a>).</p>


<h4>Native transcripts from outer centromeric repeats accumulate to low levels in the <em>dis3-54</em> mutant</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>RT-PCR was employed to directly detect transcripts from the <em>otr</em> region in the <em>dis3-54</em> mutant. Specific primers were used to distinguish between both forward and reverse transcripts derived from the <em>dh</em> sequence of the <em>otr</em> repeats (see <a href="#pone-0000317-g006">Fig. 6</a>). In control RNAi mutant strains <em>Δdcr1, Δago1</em> and Δ<em>rdp1</em>, strong accumulation of transcripts from both the forward and reverse strands were observed, whereas only the reverse strand was just detectable in wild-type (<a href="#pone-0000317-g007">Fig. 7<em>A</em></a> and <a href="#pone.0000317-Volpe1">[32]</a>). In <em>dis3-54</em> cells grown at the restrictive temperature, accumulation of both the forward and reverse strands was observed, although this was weak and much less than that for RNAi mutants. Loss of centromeric silencing in <em>dis3-54</em> was nonetheless significant as no transcripts could be detected in wild-type cells grown under the same conditions.</p>
<div class="figure" id="pone-0000317-g007"><div class="img"><a name="pone-0000317-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7. </strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Centromeric siRNA production and and overall chromatin structure in <em>dis3-54</em> is similar to wild-type. (A) Accumulation of RNA transcripts derived from native centromeric repeats was determined by semi-quantitative RT-PCR using strand-specific primers. Total RNA isolated from WT and <em>S. pombe</em> RNAi deletion mutants, <em>Δrdp1, Δago1</em> and <em>Δdcr1</em>, grown at 33°C, and from WT and <em>dis3-54</em> cultured at the restrictive temperature (20°C) were assayed. Actin was also assayed as a loading control, and reactions without reverse transcriptase (-RTase) were included as a control against DNA contamination. (B) Total small RNAs were extracted from <em>S. pombe</em> wild-type (WT) and <em>dis3-54</em> mutant cells cultured at the restrictive temperature (20°C). WT and <em>Δdcr1</em> deletion mutant strains cultured at 33°C were included as a positive control. Small RNA northern blots were probed with a specific probe to detect siRNAs derived from the centromeric <em>dh</em> repeat. Position of RNA size marker is shown to the left. Blots were re-probed with U6 snRNA for loading control. (C) The RITS complex was isolated from wild-type (WT), <em>Δdcr1</em> and <em>dis3-54</em> mutant cells by immunoprecipitation of the FLAG-tagged Chp1 component. Small RNAs that co-purified with FLAG-Chp1 were extracted by phenol-chloroform and separated on a denaturing polyacrylamide gel after labeling with [5′-<sup>32</sup>P] pCp using RNA ligase. Position of the RNA size marker is shown to the right. (D) Localization of Swi6 protein at centromeric <em>otr</em> region in the <em>dis3-54</em> mutant. ChIP analysis was performed using antibody against Swi6 protein <a href="#pone.0000317-Dziembowski1">[58]</a>. Co-precipitated DNA was purified and PCR amplified using <em>otr</em> (<em>dg1</em> and <em>dg3</em>) region specific primers. (E) Chromatin structure at the three centromere regions, <em>cnt1, imr1</em> and <em>otr</em> (<em>dg1</em>) in WT, <em>mis12</em> and <em>dis3-54</em> mutants. Nuclear chromatin fractions were prepared from WT and <em>dis3-54</em> mutant cells grown at 20°C for 8 hr, and <em>mis12</em> at 36°C (restrictive) for 8 hr, and digested with MNase for 0, 1, 2, 4 and 8 min. Southern Hybridization was then performed using specific probes for <em>cnt1, imr1</em> and <em>dg1</em>.</p>
<span>doi:10.1371/journal.pone.0000317.g007</span></div>

<h4>Centromeric siRNA production and incorporation into RITS is not affected in the <em>dis3-54</em> mutant</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>RNAi-mediated centromeric heterochromatin formation in <em>S. pombe</em> involves the formation of 22 ~ 26 nucleotide (nt) small interfering RNAs (siRNAs) by Dcr1-dependent cleavage of centromeric dsRNAs derived from the <em>otr</em> repeats <a href="#pone.0000317-Volpe1">[32]</a>, <a href="#pone.0000317-Reinhart1">[40]</a>. These siRNAs are also incorporated into the effector complex called RITS, which is required for the downstream histone modification within the <em>otr</em> region <a href="#pone.0000317-Verdel1">[33]</a>. In order to examine the role, if any, of Dis3 protein in siRNA production or incorporation into RITS, accumulation of centromeric siRNAs was first determined in <em>dis3-54</em> mutant cells.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>Small RNAs were purified from <em>S. pombe</em> wild-type, <em>Δdcr1</em> deletion and <em>dis3-54</em> mutant cells grown at 33 or 20°C, and subject to Northern blot analysis using a probe specific to the <em>dh</em> region. As shown in <a href="#pone-0000317-g007">Fig. 7<em>B</em></a>, siRNA production in the <em>dis3-54</em> mutant grown at the restrictive temperature was similar to that in wild-type and was not impaired. As expected, no siRNA was detected in the negative control <em>Δdcr1</em> mutant strain.</p>
<a id="article1.body1.sec2.sec8.p3" name="article1.body1.sec2.sec8.p3"></a><p>We next examined whether the siRNAs were normally incorporated into the RITS complex in <em>dis3-54</em> mutant cells. The <em>dis3-54</em> mutant was crossed with an <em>S. pombe</em> strain that carried the chromosomally integrated Flag-tagged Chp1 protein, which is a chromo-domain protein and a component of the RITS complex <a href="#pone.0000317-Verdel1">[33]</a>, <a href="#pone.0000317-Doe1">[41]</a>. Chp1-Flag was purified from cells grown at either 33°C or 20°C by immunoprecipitation, and analyzed for co-precipitated siRNAs after labeling with [5′-<sup>32</sup>P] pCp and RNA ligase <a href="#pone.0000317-Verdel1">[33]</a>. In <em>dis3-54</em> mutant cells, small RNAs coprecipitated with Chp1-FLAG at both 20°C and 33°C similar to that in wild-type cells, although the association may be slightly weaker at the restrictive temperature (<a href="#pone-0000317-g007">Fig. 7C</a>). Consistent with previous published data, no small RNAs were associated with RITS in control <em>Δdcr1</em> cells <a href="#pone.0000317-Verdel1">[33]</a>. The above data demonstrates that centromeric siRNAs are produced normally and are bound to RITS in <em>dis3-54</em> cells grown at the restrictive temperature.</p>


<h4>Swi6 protein remains associated with the <em>otr</em> region at reduced levels in <em>dis3-54</em></h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>Heterochromatin structure at the <em>otr</em> region in <em>S. pombe</em> centromeres is characterized by specific histone modifications and the subsequent binding of Swi6, which is a homolog of the mammalian heterochromatin protein 1 (HP1) and involved in heterochromatin spreading <a href="#pone.0000317-Klar1">[42]</a>–<a href="#pone.0000317-Hall1">[45]</a>. The presence of Swi6 at the outer repeats in the <em>dis3-54</em> mutant at the restrictive temperature was therefore examined using the chromatin immunoprecipitation (ChIP) assay. As previously reported, strong binding of Swi6 to the <em>otr</em> region was detected in WT cells, whereas no binding was observed in control <em>Δdcr1</em> RNAi mutant cells and a <em>Δswi6</em> mutant. In contrast, Swi6 remained associated with the <em>otr</em> region in the <em>dis3-54</em> mutant, although at a reduced level compared to that in wild-type (<a href="#pone-0000317-g007">Fig. 7D</a>). This result is consistent with the slight accumulation of transcripts from <em>dh</em> repeats in <em>dis3-54</em>. The fact that Swi6 was still localized at the outer repeats also supports the above finding that the mechanism for RNAi-mediated heterochromatin formation is not affected in the <em>dis3-54</em> mutant.</p>


<h4>Overall centromeric chromatin structure in the <em>dis3-54</em> mutant is similar to that in wild-type</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>The central core region of the centromere is known to have a specific unique chromatin structure, which is observed as a smeared pattern following micrococcal nuclease (MNase) digestion as opposed to a regular nucleosomal ladder pattern observed for the outer repeat region <a href="#pone.0000317-Takahashi1">[36]</a>, <a href="#pone.0000317-Polizzi1">[46]</a>. Several mutants defective in silencing at core centromere, such as <em>mis6, mis12, cnp1, mal2</em> and <em>sim4</em>, have also been shown to be defective in maintenance of the specific central core chromatin structure <a href="#pone.0000317-Pidoux1">[38]</a>, <a href="#pone.0000317-Jin1">[39]</a>, <a href="#pone.0000317-Saitoh1">[47]</a>–<a href="#pone.0000317-Takahashi2">[49]</a>. We therefore used the MNase assay to investigate overall chromatin structure at the three major centromeric regions in the <em>dis3-54</em> mutant. As previously reported, the smeared pattern for <em>cnt1</em> and <em>imr1</em> regions in wild-type was lost in the <em>mis12</em> kinetochore mutant and changed to a regular pattern (<a href="#pone-0000317-g007">Fig. 7E</a>), consistent with the disruption of chromatin structure at core regions <a href="#pone.0000317-Goshima1">[48]</a>. On the other hand, smeared MNase digestion for <em>cnt1</em> and <em>imr1</em> regions was maintained in the <em>dis3-54</em> mutant. At the outer repeats, the regular pattern observed in wild-type was also maintained in <em>dis3-54</em>. This data suggests that although Dis3 has a role in centromeric silencing, it is not required for overall centromeric chromatin structure.</p>


<h4>Synthetic lethality of <em>dis3-54</em> mutation with <em>Δswi6</em> or <em>Δdcr1</em></h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>Due to the fact that silencing of centromeric heterochromatin was defective in the <em>dis3-54</em> mutant, particularly in the <em>otr:: ura4<sup>+</sup></em> region that is silenced by a spreading mechanism from neighbouring centromeric repeats, we examined whether the <em>dis3-54</em> mutation genetically interacts with heterochromatin mutations such as Δ<em>swi6</em> and Δ<em>dcr1</em>. Swi6 is a fission yeast homolog of mammalian heterochromatin protein HP1 that is involved in heterochromatin spreading <a href="#pone.0000317-Klar1">[42]</a>–<a href="#pone.0000317-Hall1">[45]</a> downstream of RNAi-mediated heterochromatin formation involving Dcr1. As shown in <a href="#pone-0000317-g008">Fig. 8<em>A</em></a>, both the double mutants <em>dis3-54 Δswi6</em> and <em>dis3-54 Δdcr1</em> were synthetic lethal at 30 and 33°C, respectively, whereas each of the individual single mutants could produce colonies at the same temperature. The result suggested that the Dis3 protein might share an essential function with Swi6 and Dcr1 in heterochromatin formation or heterochromatin function during mitosis.</p>
<div class="figure" id="pone-0000317-g008"><div class="img"><a name="pone-0000317-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000317" data-uri="info:doi/10.1371/journal.pone.0000317.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000317.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000317.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000317.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000317.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8. </strong></p><a id="article1.body1.sec2.sec11.fig1.caption1.p1" name="article1.body1.sec2.sec11.fig1.caption1.p1"></a><p>Dis3 may act downstream of RNAi in heterochromatic silencing and correct kinetochore formation. (A) Synthetic lethality of <em>dis3-54</em> mutant with the deletion strains of Swi6 (HP1 homolog; <em>Δswi6</em>) and Dicer (RNAi component; <em>Δdcr1</em>). Single and double mutants were plated on solid media and replica plates incubated at a range of temperatures from permissive to restrictive (36°C, 33°C, 30°C, 20°C). Synthetic defects were observed at semi-permissive temperatures for the double mutants as indicated. (B) One model for the role of Dis3 in correct kinetochore formation and function. Dis3 exosome activity is involved in heterochromatin silencing at regions neighbouring centromeric repeats, possibly processing transcripts cleaved by Ago1. This processing mechanism would also be important for establishment of a functional kinetochore and microtubule attachment. Dis3 activity (either directly or indirectly) is then monitored as a key step by the mitotic checkpoint, preventing serious errors in the absence of correct kinetochore formation.</p>
<span>doi:10.1371/journal.pone.0000317.g008</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study, we show that the <em>S. pombe</em> cold-sensitive <em>dis3-54</em> mutation results in a single amino acid change (Pro509→Leu; P509L) within the characteristic RNase II RNB domain of Dis3 and causes a Mad2-dependent mitotic arrest at the restrictive temperature. Arrested cells show hyper-condensed mitotic chromosomes that are scattered along a metaphase-like spindle, which is longer in length than that normally observed during metaphase in wild-type. Continuous localization of the GFP-tagged spindle checkpoint proteins Mad2 and Bub1 at the kinetochores of arrested <em>dis3-54</em> cells suggested activation of the mitotic checkpoint. Consistent with this, deletion of the <em>mad2<sup>+</sup></em> gene in <em>dis3-54</em> led to a release of the mitotic arrest and revealed unequal separation of sister chromatids and centromeres, strongly suggesting that the mitotic checkpoint was induced by a failure in proper kinetochore-spindle interaction. The segregation phenotype of the <em>dis3-54 Δmad2</em> double mutant resembles that of other centromere/kinetochore mutants in <em>S. pombe</em><a href="#pone.0000317-Pidoux1">[38]</a>, <a href="#pone.0000317-Saitoh1">[47]</a>–<a href="#pone.0000317-Takahashi2">[49]</a>, and the elongated metaphase spindle in arrested <em>dis3-54</em> cells is a common kinetochore mutant phenotype in fission yeast <a href="#pone.0000317-Goshima1">[48]</a>, <a href="#pone.0000317-Goshima2">[50]</a>, <a href="#pone.0000317-Goshima3">[51]</a>. Taken together with the hypersensitivity of <em>dis3-54</em> to the tubulin poison TBZ, the above data indicates that the kinetochore structure and function is impaired in <em>dis3-54</em> at the restrictive temperature.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In other eukaryotes, Dis3 has been identified as a component of the exosome that is involved in 3′→5′ RNA processing. However, to the best of our knowledge, no reports to date have identified a relationship between the RNase activity of Dis3 and proper mitotic progression. It is possible that Dis3 in fission yeast had evolved a kinetochore-specific function and that the RNase activity is either lost or distinct from this role. Our <em>in vitro</em> analyses confirmed that WT <em>S. pombe</em> Dis3 protein does indeed retain RNase activity. Moreover, the amino acid change caused by the <em>dis3-54</em> mutation, Pro509→Leu509, was found to cause reduced Dis3 RNase activity. The possibility that Pro509 is a specific residue required for a mitotic function independent of RNase activity was addressed by examining a second mutation in the RNB domain that is located almost 300 residues C-terminal of <em>dis3-54</em> and also reduced RNase activity (Thr808→Ala; T808A). Dis3 constructs containing mutations located outside the RNB domain are able to complement the cold-sensitive phenotype of <em>dis3-54</em> (H.M., M.Y., unpublished data), similar to the wild-type Dis3 gene. In contrast, Dis3 constructs containing the T808A RNB mutation and the wild-type Pro509 residue could not complement the cold-sensitive <em>dis3-54</em> phenotype. This strongly suggests that it is the correct RNase activity, rather than the actual Pro509 residue, that is required for the kinetochore function of Dis3.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Mitotic and chromosome segregation defects have been reported for three fission yeast mutants, <em>dhp1-1, pfs2-3169</em> and <em>Δcid14</em>, that are also defective in RNA metabolism. Dhp1 is a 5′→3′ exoribonuclease required for the processing and export of poly(A) RNA in the nucleus, whereas Pfs2 and Cid14 are involved in mRNA 3′-end processing and polyadenylation <a href="#pone.0000317-Shobuike1">[52]</a>–<a href="#pone.0000317-Win1">[54]</a>. However, unlike for Dis3, these proteins are not directly related to the exosome. Nonetheless, the mitotic phenotypes of <em>dis3-54</em> and the above mutants do indicate significant roles for RNA processing in correct kinetochore function and chromosome segregation.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>How might the exosome activity of Dis3 contribute to kinetochore formation, such that when this activity is compromised the mitotic checkpoint is activated? One possible explanation is that RNA metabolism of unknown target genes whose products are essential for kinetochore function is defective in <em>dis3-54</em> mutant. In this case, identification of accumulated RNAs in <em>dis3-54</em> mutant may be crucial for understanding the mechanism of mitotic arrest in <em>dis3-54</em> mutant. Another plausible explanation suggested by our data is through a possible link with heterochromatin silencing. In the <em>dis3-54</em> mutant, silencing was defective at reporter genes inserted into the outer repeats and central core of centromere. Both these regions of the centromere, especially the central core, are known to be essential for faithful segregation of chromosomes in mitosis <a href="#pone.0000317-Niwa1">[55]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>At the central core region of the centromere, the <em>dis3-54</em> mutant showed a more severe silencing defect without a disruption of the overall specific chromatin structure itself. The conserved kinetochore proteins Mis6 and Mis12 were also normally localized at core regions in the <em>dis3-54</em> mutant (data not shown). This phenotype is similar to that in <em>alp5-1134</em>, a histone acetyltransferase component mutant, which was reported to be required for silencing at the central core <a href="#pone.0000317-Minoda1">[56]</a>. The <em>alp5-1134</em> mutant also causes a chromosome segregation defect that results in activation of spindle checkpoint in mitosis <a href="#pone.0000317-Minoda1">[56]</a>. Like the Alp5 protein, Dis3 may have an important function in silencing at core centromere to ensure the faithful kinetochore function, but not in the maintenance of the structure itself.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Centromeric transcripts derived from outer centromeric repeats are known to be processed by the RNAi machinery to mediate heterochromatin formation <a href="#pone.0000317-Volpe1">[32]</a>–<a href="#pone.0000317-Motamedi1">[34]</a>. Recently, it was shown that heterochromatin spreading and silencing of reporter genes neighboring outer centromeric repeats requires an endonucleolytic cleavage (slicing) activity by the Ago1 RNAi component <a href="#pone.0000317-Irvine1">[57]</a>. Significantly, the cleavage products resulting from wild-type Ago1 slicing activity accumulated in mutants that lacked the Rrp6 nuclear-specific exosome subunit, indicating that the nuclear exosome is involved in processing these transcripts. In the absence of Ago1 slicing activity, heterochromatin structure was only slightly reduced at the outer centromeric repeats, whereas silencing of the <em>otr::ura4</em><sup>+</sup> reporter gene was lost. This is similar to the phenotype observed here for <em>dis3-54</em>. The Dis3 subunit was also recently shown to be responsible for core exosome activity and to have a nuclear function that partially overlaps with Rrp6 <a href="#pone.0000317-Dziembowski1">[58]</a>. It is thus highly likely that Dis3 also processes centromeric transcripts, possibly even those cleaved by the Ago1 protein. Although further study is required, the above data support a hypothesis that the Dis3 activity is involved in a heterochromatin silencing process that may be linked to its role in the formation of a function kinetochore (<a href="#pone-0000317-g008">Fig. 8B</a>)</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Yeast strains and culture media</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Culture media used was YPD (1% yeast extract, 2%polypeptone, and 2% glucose). For plating, media was solidified with 1.5% agarose. <em>S. pombe</em> mutants <em>dis3-54, Δmad2, nda2, mis6, mis12, Δswi6, Δdcr1, Δrdp1</em> and <em>Δago1</em> were described previously <a href="#pone.0000317-Kinoshita1">[16]</a>, <a href="#pone.0000317-Toyoda1">[22]</a>, <a href="#pone.0000317-Toda1">[29]</a>, <a href="#pone.0000317-Volpe1">[32]</a>, <a href="#pone.0000317-Goshima1">[48]</a>, <a href="#pone.0000317-Ekwall1">[59]</a>. Parental <em>S. pombe</em> strains used for visualization of Cut2-GFP, Mad2-GFP, Bub1-GFP and cenI-GFP were also described previously <a href="#pone.0000317-Nabeshima2">[18]</a>, <a href="#pone.0000317-Toyoda1">[22]</a>, <a href="#pone.0000317-Kumada1">[60]</a>.</p>


<h4>Immunofluorescence microscopy</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Cell expressing GFP-tagged genes integrated on the chromosome were fixed with methanol and imaged using immunofluorescence microscopy as previously described <a href="#pone.0000317-Hagan1">[61]</a>. The TAT1 antibody was used for tubulin immunostaining <a href="#pone.0000317-Woods1">[62]</a>.</p>


<h4>In vitro RNase assays</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The procedure for <em>in vitro</em> RNase assays was basically as according to <a href="#pone.0000317-Mitchell1">[8]</a>. Assays were performed using affinity-purified recombinant protein estimated by SDS-PAGE. Purity of the purified GST fusion proteins were confirmed by Coomassie blue staining (<a href="#pone.0000317.s001">Figure S1</a>). Assay were performed in 10 mM Tris-HCl at pH7.6, 50 mM KCl, 5 mM MgCl<sub>2</sub>, 10 mM DTT, 100 µg/ml BSA, 5.0 µg Yeast RNA and 0.8 U/µl RNasin; Promega. Reactions were stopped by addition of 1/10 vol of 8% SDS and 100 mM EDTA, and heated at 65°C for 15 min after adding 1 vol of 50% formamide and loading dye. Aliquots of 20 µl were resolved by agarose gel (1.2% agarose, 20 mM MOPS, 1 mM EDTA, 50 mM sodium acetate and 2.2 M formaldehyde) electrophoresis, and stained by ethidium bromide (EtBr). The quantity of EtBr stained RNA was assessed by Molecular Imager GS-700 (Bio-Rad).</p>


<h4>RNA analyses</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Small RNA was prepared from cells growing exponentially in appropriate medium, and RT-PCR and Northern blots were performed as described <a href="#pone.0000317-Li2">[63]</a>. Analysis of RITS complex containing siRNA was described previously <a href="#pone.0000317-Sugiyama1">[64]</a>.</p>


<h4>ChIP analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>ChIP analysis was performed as described <a href="#pone.0000317-Saitoh1">[47]</a>. Cells cultured at 20°C for 8hr were fixed by formaldehyde. Sequence of primers used are:</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>For dg1: 5′-CTATAAATGGTTGACACAGC-3′ and 5′-CAAGCCAAGTCAGAGCAG-3′</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>For dg3: 5′-TGGTACCGAAGCACTGAC-3′ and 5′-ACTATCACTACTCTGAAGAC-3′</p>


<h4>MNase nuclease digestion</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The MNase digestion assay was performed as described previously <a href="#pone.0000317-Saitoh1">[47]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000317"></div><a name="pone.0000317.s001" id="pone.0000317.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000317.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Supplemental figure.1</p>
<p class="siDoi">doi:10.1371/journal.pone.0000317.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(8.74 MB TIF)</p>
<a name="pone.0000317.s002" id="pone.0000317.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000317.s002">Movie S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Tubulin-GFP living in dis3-54 at 20C #cell-1</p>
<p class="siDoi">doi:10.1371/journal.pone.0000317.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(677 KB MOV)</p>
<a name="pone.0000317.s003" id="pone.0000317.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000317.s003">Movie S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Tubulin-GFP living in dis3-54 at 20C #cell-2</p>
<p class="siDoi">doi:10.1371/journal.pone.0000317.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(858 KB MOV)</p>
<a name="pone.0000317.s004" id="pone.0000317.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000317.s004">Movie S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Tubulin-GFP living in dis3-54 at 33C</p>
<p class="siDoi">doi:10.1371/journal.pone.0000317.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(296 KB MOV)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are greatly indebted to Dr Robin C. Allshire for the anti-Swi6 antibody. We thank Dr. Mikel Zaratiegui and Dr. Susheela Dhut for experimental assistance, and members of the Yanagida and Martienssen laboratories for helpful discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RM SG TT MY HM DG EC. Performed the experiments: TT HM DG EC. Analyzed the data: TT HM DG EC. Contributed reagents/materials/analysis tools: TT MY HM DG EC. Wrote the paper: MY HM DG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000317-Yanagida1" id="pone.0000317-Yanagida1"></a>Yanagida M (2005) Basic mechanism of eukaryotic chromosome segregation. Philos Trans R Soc Lond B Biol Sci  360: 609–621.  <ul class="find" data-citedArticleID="939676" data-doi="10.1098/rstb.2004.1615"><li><a href="http://dx.doi.org/10.1098/rstb.2004.1615" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Basic+mechanism+of+eukaryotic+chromosome+segregation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Basic+mechanism+of+eukaryotic+chromosome+segregation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000317-Ohkura1" id="pone.0000317-Ohkura1"></a>Ohkura H, Adachi Y, Kinoshita N, Niwa O, Toda T, et al.  (1988) Cold-sensitive and caffeine-supersensitive mutants of the Schizosaccharomyces pombe dis genes implicated in sister chromatid separation during mitosis. EMBO J  7: 1465–1473.  <ul class="find" data-citedArticleID="939632"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cold-sensitive+and+caffeine-supersensitive+mutants+of+the+Schizosaccharomyces+pombe+dis+genes+implicated+in+sister+chromatid+separation+during+mitosis.&amp;auth=&amp;atitle=Cold-sensitive+and+caffeine-supersensitive+mutants+of+the+Schizosaccharomyces+pombe+dis+genes+implicated+in+sister+chromatid+separation+during+mitosis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cold-sensitive+and+caffeine-supersensitive+mutants+of+the+Schizosaccharomyces+pombe+dis+genes+implicated+in+sister+chromatid+separation+during+mitosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cold-sensitive+and+caffeine-supersensitive+mutants+of+the+Schizosaccharomyces+pombe+dis+genes+implicated+in+sister+chromatid+separation+during+mitosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000317-Nabeshima1" id="pone.0000317-Nabeshima1"></a>Nabeshima K, Kurooka H, Takeuchi M, Kinoshita K, Nakaseko Y, et al.  (1995) p93dis1, which is required for sister chromatid separation, is a novel microtubule and spindle pole body-associating protein phosphorylated at the Cdc2 target sites. Genes Dev  9: 1572–1585.  <ul class="find" data-citedArticleID="939624" data-doi="10.1101/gad.9.13.1572"><li><a href="http://dx.doi.org/10.1101/gad.9.13.1572" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=p93dis1%2C+which+is+required+for+sister+chromatid+separation%2C+is+a+novel+microtubule+and+spindle+pole+body-associating+protein+phosphorylated+at+the+Cdc2+target+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22p93dis1%2C+which+is+required+for+sister+chromatid+separation%2C+is+a+novel+microtubule+and+spindle+pole+body-associating+protein+phosphorylated+at+the+Cdc2+target+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000317-Nakaseko1" id="pone.0000317-Nakaseko1"></a>Nakaseko Y, Goshima G, Morishita J, Yanagida M (2001) M phase-specific kinetochore proteins in fission yeast: microtubule-associating Dis1 and Mtc1 display rapid separation and segregation during anaphase. Curr Biol  11: 537–549.  <ul class="find" data-citedArticleID="939628" data-doi="10.1016/s0960-9822(01)00155-5"><li><a href="http://dx.doi.org/10.1016/s0960-9822(01)00155-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=M+phase-specific+kinetochore+proteins+in+fission+yeast%3A+microtubule-associating+Dis1+and+Mtc1+display+rapid+separation+and+segregation+during+anaphase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22M+phase-specific+kinetochore+proteins+in+fission+yeast%3A+microtubule-associating+Dis1+and+Mtc1+display+rapid+separation+and+segregation+during+anaphase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000317-Ohkura2" id="pone.0000317-Ohkura2"></a>Ohkura H, Kinoshita N, Miyatani S, Toda T, Yanagida M (1989) The fission yeast dis2<sup>+</sup> gene required for chromosome disjoining encodes one of two putative type 1 protein phosphatases. Cell  57: 997–1007.  <ul class="find" data-citedArticleID="939634" data-doi="10.1016/0092-8674(89)90338-3"><li><a href="http://dx.doi.org/10.1016/0092-8674(89)90338-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fission+yeast+dis2%2B+gene+required+for+chromosome+disjoining+encodes+one+of+two+putative+type+1+protein+phosphatases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fission+yeast+dis2%2B+gene+required+for+chromosome+disjoining+encodes+one+of+two+putative+type+1+protein+phosphatases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000317-Yamano1" id="pone.0000317-Yamano1"></a>Yamano H, Ishii K, Yanagida M (1994) Phosphorylation of dis2 protein phosphatase at the C-terminal cdc2 consensus and its potential role in cell cycle regulation. EMBO J  13: 5310–5318.  <ul class="find" data-citedArticleID="939674"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phosphorylation+of+dis2+protein+phosphatase+at+the+C-terminal+cdc2+consensus+and+its+potential+role+in+cell+cycle+regulation.&amp;auth=&amp;atitle=Phosphorylation+of+dis2+protein+phosphatase+at+the+C-terminal+cdc2+consensus+and+its+potential+role+in+cell+cycle+regulation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylation+of+dis2+protein+phosphatase+at+the+C-terminal+cdc2+consensus+and+its+potential+role+in+cell+cycle+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylation+of+dis2+protein+phosphatase+at+the+C-terminal+cdc2+consensus+and+its+potential+role+in+cell+cycle+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000317-Aoki1" id="pone.0000317-Aoki1"></a>Aoki K, Nakaseko Y, Kinoshita K, Goshima G, Yanagida M (2006) CDC2 phosphorylation of the fission yeast dis1 ensures accurate chromosome segregation. Curr Biol  16: 1627–1635.  <ul class="find" data-citedArticleID="939554" data-doi="10.1016/j.cub.2006.06.065"><li><a href="http://dx.doi.org/10.1016/j.cub.2006.06.065" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CDC2+phosphorylation+of+the+fission+yeast+dis1+ensures+accurate+chromosome+segregation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CDC2+phosphorylation+of+the+fission+yeast+dis1+ensures+accurate+chromosome+segregation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000317-Mitchell1" id="pone.0000317-Mitchell1"></a>Mitchell P, Petfalski E, Shevchenko A, Mann M, Tollervey D (1997) The exosome: a conserved eukaryotic RNA processing complex containing multiple 3′→5′ exoribonucleases. Cell  91: 457–466.  <ul class="find" data-citedArticleID="939620" data-doi="10.1016/s0092-8674(00)80432-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80432-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+exosome%3A+a+conserved+eukaryotic+RNA+processing+complex+containing+multiple+3%26%23x2032%3B%26%23x2192%3B5%26%23x2032%3B+exoribonucleases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+exosome%3A+a+conserved+eukaryotic+RNA+processing+complex+containing+multiple+3%26%23x2032%3B%26%23x2192%3B5%26%23x2032%3B+exoribonucleases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000317-Fatica1" id="pone.0000317-Fatica1"></a>Fatica A, Tollervey D (2002) Making ribosomes. Curr Opin Cell Biol  14: 313–318.  <ul class="find" data-citedArticleID="939572" data-doi="10.1016/s0955-0674(02)00336-8"><li><a href="http://dx.doi.org/10.1016/s0955-0674(02)00336-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Making+ribosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Making+ribosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000317-Houseley1" id="pone.0000317-Houseley1"></a>Houseley J, LaCava J, Tollervey D (2006) RNA-quality control by the exosome. Nat Rev Mol Cell Biol  7: 529–539.  <ul class="find" data-citedArticleID="939592" data-doi="10.1038/nrm1964"><li><a href="http://dx.doi.org/10.1038/nrm1964" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA-quality+control+by+the+exosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA-quality+control+by+the+exosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000317-Frazao1" id="pone.0000317-Frazao1"></a>Frazao C, McVey CE, Amblar M, Barbas A, Vonrhein C, et al.  (2006) Unravelling the dynamics of RNA degradation by ribonuclease II and its RNA-bound complex. Nature  443: 110–114.  <ul class="find" data-citedArticleID="939574" data-doi="10.1038/nature05080"><li><a href="http://dx.doi.org/10.1038/nature05080" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unravelling+the+dynamics+of+RNA+degradation+by+ribonuclease+II+and+its+RNA-bound+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unravelling+the+dynamics+of+RNA+degradation+by+ribonuclease+II+and+its+RNA-bound+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000317-Zuo1" id="pone.0000317-Zuo1"></a>Zuo Y, Vincent HA, Zhang J, Wang Y, Deutscher MP, et al.  (2006) Structural basis for processivity and single-strand specificity of RNase II. Mol Cell  24: 149–156.  <ul class="find" data-citedArticleID="939678" data-doi="10.1016/j.molcel.2006.09.004"><li><a href="http://dx.doi.org/10.1016/j.molcel.2006.09.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+processivity+and+single-strand+specificity+of+RNase+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+processivity+and+single-strand+specificity+of+RNase+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000317-Lim1" id="pone.0000317-Lim1"></a>Lim J, Kuroki T, Ozaki K, Kohsaki H, Yamori T, et al.  (1997) Isolation of murine and human homologues of the fission-yeast dis3<sup>+</sup> gene encoding a mitotic-control protein and its overexpression in cancer cells with progressive phenotype. Cancer Res  57: 921–925.  <ul class="find" data-citedArticleID="939612"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Isolation+of+murine+and+human+homologues+of+the+fission-yeast+dis3%2B+gene+encoding+a+mitotic-control+protein+and+its+overexpression+in+cancer+cells+with+progressive+phenotype.&amp;auth=&amp;atitle=Isolation+of+murine+and+human+homologues+of+the+fission-yeast+dis3%2B+gene+encoding+a+mitotic-control+protein+and+its+overexpression+in+cancer+cells+with+progressive+phenotype." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+murine+and+human+homologues+of+the+fission-yeast+dis3%2B+gene+encoding+a+mitotic-control+protein+and+its+overexpression+in+cancer+cells+with+progressive+phenotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+murine+and+human+homologues+of+the+fission-yeast+dis3%2B+gene+encoding+a+mitotic-control+protein+and+its+overexpression+in+cancer+cells+with+progressive+phenotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000317-Shiomi1" id="pone.0000317-Shiomi1"></a>Shiomi T, Fukushima K, Suzuki N, Nakashima N, Noguchi E, et al.  (1998) Human dis3p, which binds to either GTP- or GDP-Ran, complements Saccharomyces cerevisiae dis3. J Biochem (Tokyo)  123: 883–890.  <ul class="find" data-citedArticleID="939646" data-doi="10.1093/oxfordjournals.jbchem.a022020"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.jbchem.a022020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+dis3p%2C+which+binds+to+either+GTP-+or+GDP-Ran%2C+complements+Saccharomyces+cerevisiae+dis3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+dis3p%2C+which+binds+to+either+GTP-+or+GDP-Ran%2C+complements+Saccharomyces+cerevisiae+dis3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000317-Tange1" id="pone.0000317-Tange1"></a>Tange Y, Hirata A, Niwa O (2002) An evolutionarily conserved fission yeast protein, Ned1, implicated in normal nuclear morphology and chromosome stability, interacts with Dis3, Pim1/RCC1 and an essential nucleoporin. J Cell Sci  115: 4375–4385.  <ul class="find" data-citedArticleID="939656" data-doi="10.1242/jcs.00135"><li><a href="http://dx.doi.org/10.1242/jcs.00135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+evolutionarily+conserved+fission+yeast+protein%2C+Ned1%2C+implicated+in+normal+nuclear+morphology+and+chromosome+stability%2C+interacts+with+Dis3%2C+Pim1%2FRCC1+and+an+essential+nucleoporin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+evolutionarily+conserved+fission+yeast+protein%2C+Ned1%2C+implicated+in+normal+nuclear+morphology+and+chromosome+stability%2C+interacts+with+Dis3%2C+Pim1%2FRCC1+and+an+essential+nucleoporin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000317-Kinoshita1" id="pone.0000317-Kinoshita1"></a>Kinoshita N, Goebl M, Yanagida M (1991) The fission yeast dis3<sup>+</sup> gene encodes a 110-kDa essential protein implicated in mitotic control. Mol Cell Biol  11: 5839–5847.  <ul class="find" data-citedArticleID="939602"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+fission+yeast+dis3%2B+gene+encodes+a+110-kDa+essential+protein+implicated+in+mitotic+control.&amp;auth=&amp;atitle=The+fission+yeast+dis3%2B+gene+encodes+a+110-kDa+essential+protein+implicated+in+mitotic+control." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fission+yeast+dis3%2B+gene+encodes+a+110-kDa+essential+protein+implicated+in+mitotic+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fission+yeast+dis3%2B+gene+encodes+a+110-kDa+essential+protein+implicated+in+mitotic+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000317-Ding1" id="pone.0000317-Ding1"></a>Ding DQ, Chikashige Y, Haraguchi T, Hiraoka Y (1998) Oscillatory nuclear movement in fission yeast meiotic prophase is driven by astral microtubules, as revealed by continuous observation of chromosomes and microtubules in living cells. J Cell Sci  111( Pt 6): 701–712.  <ul class="find" data-citedArticleID="939564"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Oscillatory+nuclear+movement+in+fission+yeast+meiotic+prophase+is+driven+by+astral+microtubules%2C+as+revealed+by+continuous+observation+of+chromosomes+and+microtubules+in+living+cells.&amp;auth=&amp;atitle=Oscillatory+nuclear+movement+in+fission+yeast+meiotic+prophase+is+driven+by+astral+microtubules%2C+as+revealed+by+continuous+observation+of+chromosomes+and+microtubules+in+living+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oscillatory+nuclear+movement+in+fission+yeast+meiotic+prophase+is+driven+by+astral+microtubules%2C+as+revealed+by+continuous+observation+of+chromosomes+and+microtubules+in+living+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oscillatory+nuclear+movement+in+fission+yeast+meiotic+prophase+is+driven+by+astral+microtubules%2C+as+revealed+by+continuous+observation+of+chromosomes+and+microtubules+in+living+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000317-Nabeshima2" id="pone.0000317-Nabeshima2"></a>Nabeshima K, Nakagawa T, Straight AF, Murray A, Chikashige Y, et al.  (1998) Dynamics of centromeres during metaphase-anaphase transition in fission yeast: Dis1 is implicated in force balance in metaphase bipolar spindle. Mol Biol Cell  9: 3211–3225.  <ul class="find" data-citedArticleID="939626" data-doi="10.1091/mbc.9.11.3211"><li><a href="http://dx.doi.org/10.1091/mbc.9.11.3211" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamics+of+centromeres+during+metaphase-anaphase+transition+in+fission+yeast%3A+Dis1+is+implicated+in+force+balance+in+metaphase+bipolar+spindle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamics+of+centromeres+during+metaphase-anaphase+transition+in+fission+yeast%3A+Dis1+is+implicated+in+force+balance+in+metaphase+bipolar+spindle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000317-Funabiki1" id="pone.0000317-Funabiki1"></a>Funabiki H, Kumada K, Yanagida M (1996) Fission yeast Cut1 and Cut2 are essential for sister chromatid separation, concentrate along the metaphase spindle and form large complexes. EMBO J  15: 6617–6628.  <ul class="find" data-citedArticleID="939576"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fission+yeast+Cut1+and+Cut2+are+essential+for+sister+chromatid+separation%2C+concentrate+along+the+metaphase+spindle+and+form+large+complexes.&amp;auth=&amp;atitle=Fission+yeast+Cut1+and+Cut2+are+essential+for+sister+chromatid+separation%2C+concentrate+along+the+metaphase+spindle+and+form+large+complexes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fission+yeast+Cut1+and+Cut2+are+essential+for+sister+chromatid+separation%2C+concentrate+along+the+metaphase+spindle+and+form+large+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fission+yeast+Cut1+and+Cut2+are+essential+for+sister+chromatid+separation%2C+concentrate+along+the+metaphase+spindle+and+form+large+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000317-Li1" id="pone.0000317-Li1"></a>Li R, Murray AW (1991) Feedback control of mitosis in budding yeast. Cell  66: 519–531.  <ul class="find" data-citedArticleID="939608" data-doi="10.1016/0092-8674(81)90015-5"><li><a href="http://dx.doi.org/10.1016/0092-8674(81)90015-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feedback+control+of+mitosis+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Feedback+control+of+mitosis+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000317-Kim1" id="pone.0000317-Kim1"></a>Kim SH, Lin DP, Matsumoto S, Kitazono A, Matsumoto T (1998) Fission yeast Slp1: an effector of the Mad2-dependent spindle checkpoint. Science  279: 1045–1047.  <ul class="find" data-citedArticleID="939600" data-doi="10.1126/science.279.5353.1045"><li><a href="http://dx.doi.org/10.1126/science.279.5353.1045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fission+yeast+Slp1%3A+an+effector+of+the+Mad2-dependent+spindle+checkpoint." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fission+yeast+Slp1%3A+an+effector+of+the+Mad2-dependent+spindle+checkpoint.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000317-Toyoda1" id="pone.0000317-Toyoda1"></a>Toyoda Y, Furuya K, Goshima G, Nagao K, Takahashi K, et al.  (2002) Requirement of chromatid cohesion proteins rad21/scc1 and mis4/scc2 for normal spindle-kinetochore interaction in fission yeast. Curr Biol  12: 347–358.  <ul class="find" data-citedArticleID="939660" data-doi="10.1016/s0960-9822(02)00692-9"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)00692-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Requirement+of+chromatid+cohesion+proteins+rad21%2Fscc1+and+mis4%2Fscc2+for+normal+spindle-kinetochore+interaction+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Requirement+of+chromatid+cohesion+proteins+rad21%2Fscc1+and+mis4%2Fscc2+for+normal+spindle-kinetochore+interaction+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000317-Bernard1" id="pone.0000317-Bernard1"></a>Bernard P, Hardwick K, Javerzat JP (1998) Fission yeast bub1 is a mitotic centromere protein essential for the spindle checkpoint and the preservation of correct ploidy through mitosis. J Cell Biol  143: 1775–1787.  <ul class="find" data-citedArticleID="939558" data-doi="10.1083/jcb.143.7.1775"><li><a href="http://dx.doi.org/10.1083/jcb.143.7.1775" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fission+yeast+bub1+is+a+mitotic+centromere+protein+essential+for+the+spindle+checkpoint+and+the+preservation+of+correct+ploidy+through+mitosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fission+yeast+bub1+is+a+mitotic+centromere+protein+essential+for+the+spindle+checkpoint+and+the+preservation+of+correct+ploidy+through+mitosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000317-Garcia1" id="pone.0000317-Garcia1"></a>Garcia MA, Koonrugsa N, Toda T (2002) Spindle-kinetochore attachment requires the combined action of Kin I-like Klp5/6 and Alp14/Dis1-MAPs in fission yeast. EMBO J  21: 6015–6024.  <ul class="find" data-citedArticleID="939578" data-doi="10.1093/emboj/cdf611"><li><a href="http://dx.doi.org/10.1093/emboj/cdf611" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spindle-kinetochore+attachment+requires+the+combined+action+of+Kin+I-like+Klp5%2F6+and+Alp14%2FDis1-MAPs+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spindle-kinetochore+attachment+requires+the+combined+action+of+Kin+I-like+Klp5%2F6+and+Alp14%2FDis1-MAPs+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000317-Umesono1" id="pone.0000317-Umesono1"></a>Umesono K, Toda T, Hayashi S, Yanagida M (1983) Cell division cycle genes nda2 and nda3 of the fission yeast Schizosaccharomyces pombe control microtubular organization and sensitivity to anti-mitotic benzimidazole compounds. J Mol Biol  168: 271–284.  <ul class="find" data-citedArticleID="939662" data-doi="10.1016/s0022-2836(83)80018-7"><li><a href="http://dx.doi.org/10.1016/s0022-2836(83)80018-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+division+cycle+genes+nda2+and+nda3+of+the+fission+yeast+Schizosaccharomyces+pombe+control+microtubular+organization+and+sensitivity+to+anti-mitotic+benzimidazole+compounds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+division+cycle+genes+nda2+and+nda3+of+the+fission+yeast+Schizosaccharomyces+pombe+control+microtubular+organization+and+sensitivity+to+anti-mitotic+benzimidazole+compounds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000317-Beinhauer1" id="pone.0000317-Beinhauer1"></a>Beinhauer JD, Hagan IM, Hegemann JH, Fleig U (1997) Mal3, the fission yeast homologue of the human APC-interacting protein EB-1 is required for microtubule integrity and the maintenance of cell form. J Cell Biol  139: 717–728.  <ul class="find" data-citedArticleID="939556" data-doi="10.1083/jcb.139.3.717"><li><a href="http://dx.doi.org/10.1083/jcb.139.3.717" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mal3%2C+the+fission+yeast+homologue+of+the+human+APC-interacting+protein+EB-1+is+required+for+microtubule+integrity+and+the+maintenance+of+cell+form." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mal3%2C+the+fission+yeast+homologue+of+the+human+APC-interacting+protein+EB-1+is+required+for+microtubule+integrity+and+the+maintenance+of+cell+form.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000317-Garcia2" id="pone.0000317-Garcia2"></a>Garcia MA, Vardy L, Koonrugsa N, Toda T (2001) Fission yeast ch-TOG/XMAP215 homologue Alp14 connects mitotic spindles with the kinetochore and is a component of the Mad2-dependent spindle checkpoint. EMBO J  20: 3389–3401.  <ul class="find" data-citedArticleID="939580" data-doi="10.1093/emboj/20.13.3389"><li><a href="http://dx.doi.org/10.1093/emboj/20.13.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fission+yeast+ch-TOG%2FXMAP215+homologue+Alp14+connects+mitotic+spindles+with+the+kinetochore+and+is+a+component+of+the+Mad2-dependent+spindle+checkpoint." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fission+yeast+ch-TOG%2FXMAP215+homologue+Alp14+connects+mitotic+spindles+with+the+kinetochore+and+is+a+component+of+the+Mad2-dependent+spindle+checkpoint.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000317-Kadura1" id="pone.0000317-Kadura1"></a>Kadura S, He X, Vanoosthuyse V, Hardwick KG, Sazer S (2005) The A78V mutation in the Mad3-like domain of Schizosaccharomyces pombe Bub1p perturbs nuclear accumulation and kinetochore targeting of Bub1p, Bub3p, and Mad3p and spindle assembly checkpoint function. Mol Biol Cell  16: 385–395.  <ul class="find" data-citedArticleID="939598" data-doi="10.1091/mbc.e04-07-0558"><li><a href="http://dx.doi.org/10.1091/mbc.e04-07-0558" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+A78V+mutation+in+the+Mad3-like+domain+of+Schizosaccharomyces+pombe+Bub1p+perturbs+nuclear+accumulation+and+kinetochore+targeting+of+Bub1p%2C+Bub3p%2C+and+Mad3p+and+spindle+assembly+checkpoint+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+A78V+mutation+in+the+Mad3-like+domain+of+Schizosaccharomyces+pombe+Bub1p+perturbs+nuclear+accumulation+and+kinetochore+targeting+of+Bub1p%2C+Bub3p%2C+and+Mad3p+and+spindle+assembly+checkpoint+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000317-Toda1" id="pone.0000317-Toda1"></a>Toda T, Umesono K, Hirata A, Yanagida M (1983) Cold-sensitive nuclear division arrest mutants of the fission yeast Schizosaccharomyces pombe. J Mol Biol  168: 251–270.  <ul class="find" data-citedArticleID="939658" data-doi="10.1016/s0022-2836(83)80017-5"><li><a href="http://dx.doi.org/10.1016/s0022-2836(83)80017-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cold-sensitive+nuclear+division+arrest+mutants+of+the+fission+yeast+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cold-sensitive+nuclear+division+arrest+mutants+of+the+fission+yeast+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000317-Zuo2" id="pone.0000317-Zuo2"></a>Zuo Y, Deutscher MP (2001) Exoribonuclease superfamilies: structural analysis and phylogenetic distribution. Nucleic Acids Res  29: 1017–1026.  <ul class="find" data-citedArticleID="939680" data-doi="10.1093/nar/29.5.1017"><li><a href="http://dx.doi.org/10.1093/nar/29.5.1017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exoribonuclease+superfamilies%3A+structural+analysis+and+phylogenetic+distribution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exoribonuclease+superfamilies%3A+structural+analysis+and+phylogenetic+distribution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000317-Coburn1" id="pone.0000317-Coburn1"></a>Coburn GA, Mackie GA (1996) Overexpression, purification, and properties of Escherichia coli ribonuclease II. J Biol Chem  271: 1048–1053.  <ul class="find" data-citedArticleID="939562" data-doi="10.1074/jbc.271.2.1048"><li><a href="http://dx.doi.org/10.1074/jbc.271.2.1048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overexpression%2C+purification%2C+and+properties+of+Escherichia+coli+ribonuclease+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overexpression%2C+purification%2C+and+properties+of+Escherichia+coli+ribonuclease+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000317-Volpe1" id="pone.0000317-Volpe1"></a>Volpe TA, Kidner C, Hall IM, Teng G, Grewal SI, et al.  (2002) Regulation of heterochromatic silencing and histone H3 lysine-9 methylation by RNAi. Science  297: 1833–1837.  <ul class="find" data-citedArticleID="939666" data-doi="10.1126/science.1074973"><li><a href="http://dx.doi.org/10.1126/science.1074973" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+heterochromatic+silencing+and+histone+H3+lysine-9+methylation+by+RNAi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+heterochromatic+silencing+and+histone+H3+lysine-9+methylation+by+RNAi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000317-Verdel1" id="pone.0000317-Verdel1"></a>Verdel A, Jia S, Gerber S, Sugiyama T, Gygi S, et al.  (2004) RNAi-mediated targeting of heterochromatin by the RITS complex. Science  303: 672–676.  <ul class="find" data-citedArticleID="939664" data-doi="10.1126/science.1093686"><li><a href="http://dx.doi.org/10.1126/science.1093686" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNAi-mediated+targeting+of+heterochromatin+by+the+RITS+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNAi-mediated+targeting+of+heterochromatin+by+the+RITS+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000317-Motamedi1" id="pone.0000317-Motamedi1"></a>Motamedi MR, Verdel A, Colmenares SU, Gerber SA, Gygi SP, et al.  (2004) Two RNAi complexes, RITS and RDRC, physically interact and localize to noncoding centromeric RNAs. Cell  119: 789–802.  <ul class="find" data-citedArticleID="939622" data-doi="10.1016/j.cell.2004.11.034"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.11.034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+RNAi+complexes%2C+RITS+and+RDRC%2C+physically+interact+and+localize+to+noncoding+centromeric+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+RNAi+complexes%2C+RITS+and+RDRC%2C+physically+interact+and+localize+to+noncoding+centromeric+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000317-Martienssen1" id="pone.0000317-Martienssen1"></a>Martienssen RA, Zaratiegui M, Goto DB (2005) RNA interference and heterochromatin in the fission yeast Schizosaccharomyces pombe. Trends Genet  21: 450–456.  <ul class="find" data-citedArticleID="939616" data-doi="10.1016/j.tig.2005.06.005"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.06.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+interference+and+heterochromatin+in+the+fission+yeast+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+interference+and+heterochromatin+in+the+fission+yeast+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000317-Takahashi1" id="pone.0000317-Takahashi1"></a>Takahashi K, Murakami S, Chikashige Y, Funabiki H, Niwa O, et al.  (1992) A low copy number central sequence with strict symmetry and unusual chromatin structure in fission yeast centromere. Mol Biol Cell  3: 819–835.  <ul class="find" data-citedArticleID="939652" data-doi="10.1091/mbc.3.7.819"><li><a href="http://dx.doi.org/10.1091/mbc.3.7.819" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+low+copy+number+central+sequence+with+strict+symmetry+and+unusual+chromatin+structure+in+fission+yeast+centromere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+low+copy+number+central+sequence+with+strict+symmetry+and+unusual+chromatin+structure+in+fission+yeast+centromere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000317-Patridge1" id="pone.0000317-Patridge1"></a>Patridge JF, Borgstrom B, Allshire RC (2000) Distinct protein interaction domains and protein spreading in a complex centromere. Genes Dev.  14: 783–791.  <ul class="find" data-citedArticleID="939636"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Distinct+protein+interaction+domains+and+protein+spreading+in+a+complex+centromere.&amp;auth=&amp;atitle=Distinct+protein+interaction+domains+and+protein+spreading+in+a+complex+centromere." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+protein+interaction+domains+and+protein+spreading+in+a+complex+centromere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+protein+interaction+domains+and+protein+spreading+in+a+complex+centromere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000317-Pidoux1" id="pone.0000317-Pidoux1"></a>Pidoux AL, Richardson W, Allshire RC (2003) Sim4: a novel fission yeast kinetochore protein required for centromeric silencing and chromosome segregation. J Cell Biol  161: 295–307.  <ul class="find" data-citedArticleID="939638" data-doi="10.1083/jcb.200212110"><li><a href="http://dx.doi.org/10.1083/jcb.200212110" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sim4%3A+a+novel+fission+yeast+kinetochore+protein+required+for+centromeric+silencing+and+chromosome+segregation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sim4%3A+a+novel+fission+yeast+kinetochore+protein+required+for+centromeric+silencing+and+chromosome+segregation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000317-Jin1" id="pone.0000317-Jin1"></a>Jin Q-W, Pidoux AL, Decker C, Allshire RC, Fleig U (2002) The Mal2p protein is an essential component of the fission yeast centromere. Mol Cell Biol  22: 7168–7183.  <ul class="find" data-citedArticleID="939596" data-doi="10.1128/mcb.22.20.7168-7183.2002"><li><a href="http://dx.doi.org/10.1128/mcb.22.20.7168-7183.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Mal2p+protein+is+an+essential+component+of+the+fission+yeast+centromere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Mal2p+protein+is+an+essential+component+of+the+fission+yeast+centromere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000317-Reinhart1" id="pone.0000317-Reinhart1"></a>Reinhart BJ, Bartel DP (2002) Small RNAs correspond to centromere heterochromatic repeats. Science  297: 1831.  <ul class="find" data-citedArticleID="939642" data-doi="10.1126/science.1077183"><li><a href="http://dx.doi.org/10.1126/science.1077183" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Small+RNAs+correspond+to+centromere+heterochromatic+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Small+RNAs+correspond+to+centromere+heterochromatic+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000317-Doe1" id="pone.0000317-Doe1"></a>Doe CL, Wang G, Chow C, Fricker MD, Singh PB, et al.  (1998) The fission yeast chromo domain encoding gene chp1(+) is required for chromosome segregation and shows a genetic interaction with alpha-tubulin. Nucleic Acids Res  26: 4222–4229.  <ul class="find" data-citedArticleID="939566" data-doi="10.1093/nar/26.18.4222"><li><a href="http://dx.doi.org/10.1093/nar/26.18.4222" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fission+yeast+chromo+domain+encoding+gene+chp1%28%2B%29+is+required+for+chromosome+segregation+and+shows+a+genetic+interaction+with+alpha-tubulin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fission+yeast+chromo+domain+encoding+gene+chp1%28%2B%29+is+required+for+chromosome+segregation+and+shows+a+genetic+interaction+with+alpha-tubulin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000317-Klar1" id="pone.0000317-Klar1"></a>Klar AJ, Bonaduce MJ (1991) swi6, a gene required for mating-type switching, prohibits meiotic recombination in the mat2-mat3 “cold spot” of fission yeast. Genetics  129: 1033–1042.  <ul class="find" data-citedArticleID="939604"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=swi6%2C+a+gene+required+for+mating-type+switching%2C+prohibits+meiotic+recombination+in+the+mat2-mat3+%26%23x201c%3Bcold+spot%26%23x201d%3B+of+fission+yeast.&amp;auth=&amp;atitle=swi6%2C+a+gene+required+for+mating-type+switching%2C+prohibits+meiotic+recombination+in+the+mat2-mat3+%26%23x201c%3Bcold+spot%26%23x201d%3B+of+fission+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=swi6%2C+a+gene+required+for+mating-type+switching%2C+prohibits+meiotic+recombination+in+the+mat2-mat3+%26%23x201c%3Bcold+spot%26%23x201d%3B+of+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22swi6%2C+a+gene+required+for+mating-type+switching%2C+prohibits+meiotic+recombination+in+the+mat2-mat3+%26%23x201c%3Bcold+spot%26%23x201d%3B+of+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000317-Lorentz1" id="pone.0000317-Lorentz1"></a>Lorentz A, Ostermann K, Fleck O, Schmidt H (1994) Switching gene swi6, involved in repression of silent mating-type loci in fission yeast, encodes a homologue of chromatin-associated proteins from Drosophila and mammals. Gene  143: 139–143.  <ul class="find" data-citedArticleID="939614" data-doi="10.1016/0378-1119(94)90619-x"><li><a href="http://dx.doi.org/10.1016/0378-1119(94)90619-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Switching+gene+swi6%2C+involved+in+repression+of+silent+mating-type+loci+in+fission+yeast%2C+encodes+a+homologue+of+chromatin-associated+proteins+from+Drosophila+and+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Switching+gene+swi6%2C+involved+in+repression+of+silent+mating-type+loci+in+fission+yeast%2C+encodes+a+homologue+of+chromatin-associated+proteins+from+Drosophila+and+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000317-Allshire1" id="pone.0000317-Allshire1"></a>Allshire RC, Nimmo ER, Ekwall K, Javerzat JP, Cranston G (1995) Mutations derepressing silent centromeric domains in fission yeast disrupt chromosome segregation. Genes Dev  9: 218–233.  <ul class="find" data-citedArticleID="939552" data-doi="10.1101/gad.9.2.218"><li><a href="http://dx.doi.org/10.1101/gad.9.2.218" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+derepressing+silent+centromeric+domains+in+fission+yeast+disrupt+chromosome+segregation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+derepressing+silent+centromeric+domains+in+fission+yeast+disrupt+chromosome+segregation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000317-Hall1" id="pone.0000317-Hall1"></a>Hall IM, Shankaranarayana GD, Noma K, Ayoub N, Cohen A, et al.  (2002) Establishment and maintenance of a heterochromatin domain. Science  297: 2232–2237.  <ul class="find" data-citedArticleID="939590" data-doi="10.1126/science.1076466"><li><a href="http://dx.doi.org/10.1126/science.1076466" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Establishment+and+maintenance+of+a+heterochromatin+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Establishment+and+maintenance+of+a+heterochromatin+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000317-Polizzi1" id="pone.0000317-Polizzi1"></a>Polizzi C, Clarke L (1991) The chromatin structure of centromeres from fission yeast: differentiation of the central core that correlates with function. J. Cell Biol.  112: 191–201.  <ul class="find" data-citedArticleID="939640" data-doi="10.1083/jcb.112.2.191"><li><a href="http://dx.doi.org/10.1083/jcb.112.2.191" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+chromatin+structure+of+centromeres+from+fission+yeast%3A+differentiation+of+the+central+core+that+correlates+with+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+chromatin+structure+of+centromeres+from+fission+yeast%3A+differentiation+of+the+central+core+that+correlates+with+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000317-Saitoh1" id="pone.0000317-Saitoh1"></a>Saitoh S, Takahashi K, Yanagida M (1997) Mis6, a fission yeast inner centromere protein, acts during G1/S and forms specialized chromatin required for equal segregation. Cell  90: 131–143.  <ul class="find" data-citedArticleID="939644" data-doi="10.1016/s0092-8674(00)80320-7"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80320-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mis6%2C+a+fission+yeast+inner+centromere+protein%2C+acts+during+G1%2FS+and+forms+specialized+chromatin+required+for+equal+segregation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mis6%2C+a+fission+yeast+inner+centromere+protein%2C+acts+during+G1%2FS+and+forms+specialized+chromatin+required+for+equal+segregation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000317-Goshima1" id="pone.0000317-Goshima1"></a>Goshima G, Saitoh S, Yanagida M (1999) Proper metaphase spindle length is determined by centromere proteins Mis12 and Mis6 required for faithful chromosome segregation. Genes Dev  13: 1664–1677.  <ul class="find" data-citedArticleID="939582" data-doi="10.1101/gad.13.13.1664"><li><a href="http://dx.doi.org/10.1101/gad.13.13.1664" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proper+metaphase+spindle+length+is+determined+by+centromere+proteins+Mis12+and+Mis6+required+for+faithful+chromosome+segregation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proper+metaphase+spindle+length+is+determined+by+centromere+proteins+Mis12+and+Mis6+required+for+faithful+chromosome+segregation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000317-Takahashi2" id="pone.0000317-Takahashi2"></a>Takahashi K, Chen ES, Yanagida M (2000) Requirement of Mis6 centromere connector for localizing a CENP-A-like protein in fission yeast. Science  288: 2215–2219.  <ul class="find" data-citedArticleID="939654" data-doi="10.1126/science.288.5474.2215"><li><a href="http://dx.doi.org/10.1126/science.288.5474.2215" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Requirement+of+Mis6+centromere+connector+for+localizing+a+CENP-A-like+protein+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Requirement+of+Mis6+centromere+connector+for+localizing+a+CENP-A-like+protein+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000317-Goshima2" id="pone.0000317-Goshima2"></a>Goshima G, Kiyomitsu T, Yoda K, Yanagida M (2003) Human centromere chromatin protein hMis12, essential for equal segregation, is independent of CENP-A loading pathway. J Cell Biol  160: 25–39.  <ul class="find" data-citedArticleID="939584" data-doi="10.1083/jcb.200210005"><li><a href="http://dx.doi.org/10.1083/jcb.200210005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+centromere+chromatin+protein+hMis12%2C+essential+for+equal+segregation%2C+is+independent+of+CENP-A+loading+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+centromere+chromatin+protein+hMis12%2C+essential+for+equal+segregation%2C+is+independent+of+CENP-A+loading+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000317-Goshima3" id="pone.0000317-Goshima3"></a>Goshima G, Wollman R, Stuurman N, Scholey JM, Vale RD (2005) Length control of the metaphase spindle. Curr Biol  15: 1979–1988.  <ul class="find" data-citedArticleID="939586" data-doi="10.1016/j.cub.2005.09.054"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.09.054" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Length+control+of+the+metaphase+spindle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Length+control+of+the+metaphase+spindle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000317-Shobuike1" id="pone.0000317-Shobuike1"></a>Shobuike T, Tatebayashi K, Tani T, Sugano S, Ikeda H (2001) The dhp1(+) gene, encoding a putative nuclear 5′→3′ exoribonuclease, is required for proper chromosome segregation in fission yeast. Nucleic Acids Res  29: 1326–1333.  <ul class="find" data-citedArticleID="939648" data-doi="10.1093/nar/29.6.1326"><li><a href="http://dx.doi.org/10.1093/nar/29.6.1326" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dhp1%28%2B%29+gene%2C+encoding+a+putative+nuclear+5%26%23x2032%3B%26%23x2192%3B3%26%23x2032%3B+exoribonuclease%2C+is+required+for+proper+chromosome+segregation+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dhp1%28%2B%29+gene%2C+encoding+a+putative+nuclear+5%26%23x2032%3B%26%23x2192%3B3%26%23x2032%3B+exoribonuclease%2C+is+required+for+proper+chromosome+segregation+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000317-Wang1" id="pone.0000317-Wang1"></a>Wang SW, Asakawa K, Win TZ, Toda T, Norbury CJ (2005) Inactivation of the pre-mRNA cleavage and polyadenylation factor Pfs2 in fission yeast causes lethal cell cycle defects. Mol Cell Biol  25: 2288–2296.  <ul class="find" data-citedArticleID="939668" data-doi="10.1128/mcb.25.6.2288-2296.2005"><li><a href="http://dx.doi.org/10.1128/mcb.25.6.2288-2296.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inactivation+of+the+pre-mRNA+cleavage+and+polyadenylation+factor+Pfs2+in+fission+yeast+causes+lethal+cell+cycle+defects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inactivation+of+the+pre-mRNA+cleavage+and+polyadenylation+factor+Pfs2+in+fission+yeast+causes+lethal+cell+cycle+defects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000317-Win1" id="pone.0000317-Win1"></a>Win TZ, Draper S, Read RL, Pearce J, Norbury CJ, et al.  (2006) Requirement of fission yeast Cid14 in polyadenylation of rRNAs. Mol Cell Biol  26: 1710–1721.  <ul class="find" data-citedArticleID="939670" data-doi="10.1128/mcb.26.5.1710-1721.2006"><li><a href="http://dx.doi.org/10.1128/mcb.26.5.1710-1721.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Requirement+of+fission+yeast+Cid14+in+polyadenylation+of+rRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Requirement+of+fission+yeast+Cid14+in+polyadenylation+of+rRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000317-Niwa1" id="pone.0000317-Niwa1"></a>Niwa O, Matsumoto T, Chikashige Y, Yanagida M (1989) Characterization of <em>Schizosaccharomyces pombe</em> minichromosome deletion derivatives and a functional allocation of their centromere. EMBO J.  8: 3045–3052.  <ul class="find" data-citedArticleID="939630"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+Schizosaccharomyces+pombe+minichromosome+deletion+derivatives+and+a+functional+allocation+of+their+centromere.&amp;auth=&amp;atitle=Characterization+of+Schizosaccharomyces+pombe+minichromosome+deletion+derivatives+and+a+functional+allocation+of+their+centromere." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+Schizosaccharomyces+pombe+minichromosome+deletion+derivatives+and+a+functional+allocation+of+their+centromere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+Schizosaccharomyces+pombe+minichromosome+deletion+derivatives+and+a+functional+allocation+of+their+centromere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000317-Minoda1" id="pone.0000317-Minoda1"></a>Minoda A, Saitoh S, Takahashi K, Toda T (2005) BAF53/Arp4 homolog Alp5 in fission yeast is required for histone H4 acetylation, kinetochore-spindle attachment, and gene silencing at centromere. Mol Cell Biol  16: 316–327.  <ul class="find" data-citedArticleID="939618" data-doi="10.1091/mbc.e04-06-0519"><li><a href="http://dx.doi.org/10.1091/mbc.e04-06-0519" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BAF53%2FArp4+homolog+Alp5+in+fission+yeast+is+required+for+histone+H4+acetylation%2C+kinetochore-spindle+attachment%2C+and+gene+silencing+at+centromere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BAF53%2FArp4+homolog+Alp5+in+fission+yeast+is+required+for+histone+H4+acetylation%2C+kinetochore-spindle+attachment%2C+and+gene+silencing+at+centromere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000317-Irvine1" id="pone.0000317-Irvine1"></a>Irvine DV, Zaratiegui M, Tolia NH, Goto DB, Chitwood DH, et al.  (2006) Argonaute slicing is required for heterochromatic silencing and spreading. Science  313: 1134–1137.  <ul class="find" data-citedArticleID="939594" data-doi="10.1126/science.1128813"><li><a href="http://dx.doi.org/10.1126/science.1128813" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Argonaute+slicing+is+required+for+heterochromatic+silencing+and+spreading." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Argonaute+slicing+is+required+for+heterochromatic+silencing+and+spreading.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000317-Dziembowski1" id="pone.0000317-Dziembowski1"></a>Dziembowski A, Lorentzen E, Conti E, Seraphin B (2007) A single subunit, Dis3, is essentially responsible for yeast exosome core activity. Nat Struct Mol Biol  14: 15–22.  <ul class="find" data-citedArticleID="939568" data-doi="10.1038/nsmb1184"><li><a href="http://dx.doi.org/10.1038/nsmb1184" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+subunit%2C+Dis3%2C+is+essentially+responsible+for+yeast+exosome+core+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+subunit%2C+Dis3%2C+is+essentially+responsible+for+yeast+exosome+core+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000317-Ekwall1" id="pone.0000317-Ekwall1"></a>Ekwall K, Javerzat JP, Lorentz A, Schmidt H, Cranston G, et al.  (1995) The chromodomain protein Swi6: a key component at fission yeast centromeres. Science  269: 1429–1431.  <ul class="find" data-citedArticleID="939570" data-doi="10.1126/science.7660126"><li><a href="http://dx.doi.org/10.1126/science.7660126" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+chromodomain+protein+Swi6%3A+a+key+component+at+fission+yeast+centromeres." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+chromodomain+protein+Swi6%3A+a+key+component+at+fission+yeast+centromeres.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000317-Kumada1" id="pone.0000317-Kumada1"></a>Kumada K, Nakamura T, Nagao K, Funabiki H, Nakagawa T, et al.  (1998) Cut1 is loaded onto the spindle by binding to Cut2 and promotes anaphase spindle movement upon Cut2 proteolysis. Curr Biol  8: 633–641.  <ul class="find" data-citedArticleID="939606" data-doi="10.1016/s0960-9822(98)70250-7"><li><a href="http://dx.doi.org/10.1016/s0960-9822(98)70250-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cut1+is+loaded+onto+the+spindle+by+binding+to+Cut2+and+promotes+anaphase+spindle+movement+upon+Cut2+proteolysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cut1+is+loaded+onto+the+spindle+by+binding+to+Cut2+and+promotes+anaphase+spindle+movement+upon+Cut2+proteolysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000317-Hagan1" id="pone.0000317-Hagan1"></a>Hagan IM, Hyams JS (1988) The use of cell division cycle mutants to investigate the control of microtubule distribution in the fission yeast Schizosaccharomyces pombe. J Cell Sci  89( Pt 3): 343–357.  <ul class="find" data-citedArticleID="939588"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+use+of+cell+division+cycle+mutants+to+investigate+the+control+of+microtubule+distribution+in+the+fission+yeast+Schizosaccharomyces+pombe.&amp;auth=&amp;atitle=The+use+of+cell+division+cycle+mutants+to+investigate+the+control+of+microtubule+distribution+in+the+fission+yeast+Schizosaccharomyces+pombe." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+cell+division+cycle+mutants+to+investigate+the+control+of+microtubule+distribution+in+the+fission+yeast+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+cell+division+cycle+mutants+to+investigate+the+control+of+microtubule+distribution+in+the+fission+yeast+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000317-Woods1" id="pone.0000317-Woods1"></a>Woods A, Sherwin T, Sasse R, MacRae TH, Baines AJ, et al.  (1989) Definition of individual components within the cytoskeleton of Trypanosoma brucei by a library of monoclonal antibodies. J Cell Sci  93( Pt 3): 491–500.  <ul class="find" data-citedArticleID="939672"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Definition+of+individual+components+within+the+cytoskeleton+of+Trypanosoma+brucei+by+a+library+of+monoclonal+antibodies.&amp;auth=&amp;atitle=Definition+of+individual+components+within+the+cytoskeleton+of+Trypanosoma+brucei+by+a+library+of+monoclonal+antibodies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Definition+of+individual+components+within+the+cytoskeleton+of+Trypanosoma+brucei+by+a+library+of+monoclonal+antibodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Definition+of+individual+components+within+the+cytoskeleton+of+Trypanosoma+brucei+by+a+library+of+monoclonal+antibodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000317-Li2" id="pone.0000317-Li2"></a>Li F, Goto DB, Zaratiegui M, Tang X, Martienssen R, et al.  (2005) Two novel proteins, dos1 and dos2, interact with rik1 to regulate heterochromatic RNA interference and histone modification. Curr Biol  15: 1448–1457.  <ul class="find" data-citedArticleID="939610" data-doi="10.1016/j.cub.2005.07.021"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.07.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+novel+proteins%2C+dos1+and+dos2%2C+interact+with+rik1+to+regulate+heterochromatic+RNA+interference+and+histone+modification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+novel+proteins%2C+dos1+and+dos2%2C+interact+with+rik1+to+regulate+heterochromatic+RNA+interference+and+histone+modification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000317-Sugiyama1" id="pone.0000317-Sugiyama1"></a>Sugiyama T, Cam H, Verdel A, Moazed D, Grewal SI (2005) RNA-dependent RNA polymerase is an essential component of a self-enforcing loop coupling heterochromatin assembly to siRNA production. Proc Natl Acad Sci U S A  102: 152–157.  <ul class="find" data-citedArticleID="939650" data-doi="10.1073/pnas.0407641102"><li><a href="http://dx.doi.org/10.1073/pnas.0407641102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA-dependent+RNA+polymerase+is+an+essential+component+of+a+self-enforcing+loop+coupling+heterochromatin+assembly+to+siRNA+production." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA-dependent+RNA+polymerase+is+an+essential+component+of+a+self-enforcing+loop+coupling+heterochromatin+assembly+to+siRNA+production.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000317-Bycroft1" id="pone.0000317-Bycroft1"></a>Bycroft M, Hubbard TJ, Proctor M, Freund SM, Murzin AG (1997) The solution structure of the S1 RNA binding domain: a member of an ancient nucleic acid-binding fold. Cell  88: 235–242.  <ul class="find" data-citedArticleID="939560" data-doi="10.1016/s0092-8674(00)81844-9"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81844-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+solution+structure+of+the+S1+RNA+binding+domain%3A+a+member+of+an+ancient+nucleic+acid-binding+fold." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+solution+structure+of+the+S1+RNA+binding+domain%3A+a+member+of+an+ancient+nucleic+acid-binding+fold.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.XML" value="130989"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.PDF" value="1602058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g001.PNG_L" value="644597"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g001.PNG_M" value="122194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g001.PNG_S" value="5487"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g001.TIF" value="1015908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g001.PNG_I" value="46044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g002.PNG_L" value="148094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g002.PNG_M" value="94444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g002.PNG_S" value="4078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g002.TIF" value="190130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g002.PNG_I" value="35804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g003.PNG_L" value="436015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g003.PNG_M" value="80838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g003.PNG_S" value="4380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g003.TIF" value="791056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g003.PNG_I" value="31801"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g004.PNG_L" value="475466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g004.PNG_M" value="83088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g004.PNG_S" value="3668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g004.TIF" value="820074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g004.PNG_I" value="33047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g005.PNG_L" value="1215188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g005.PNG_M" value="167431"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g005.PNG_S" value="6419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g005.TIF" value="1644384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g005.PNG_I" value="62610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g006.PNG_L" value="659112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g006.PNG_M" value="152802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g006.PNG_S" value="5920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g006.TIF" value="805340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g006.PNG_I" value="105490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g007.PNG_L" value="1126091"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g007.PNG_M" value="176212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g007.PNG_S" value="6828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g007.TIF" value="1813054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g007.PNG_I" value="65124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g008.PNG_L" value="538902"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g008.PNG_M" value="133879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g008.PNG_S" value="7600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g008.TIF" value="622194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.g008.PNG_I" value="89691"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.s001.TIF" value="8732886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.s002.MOV" value="693873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.s003.MOV" value="878975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000317.s004.MOV" value="303727"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000317&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000317" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000317&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000317&volume=&issue=&title=Ribonuclease Activity of Dis3 Is Required for Mitotic Progression and Provides a Possible Link between Heterochromatin and Kinetochore Function&author_name=Hiroaki%20Murakami%2C%20Derek%20B.%20Goto%2C%20Takashi%20Toda%2C%20Ee%20Sin%20Chen%2C%20Shiv%20I.%20Grewal%2C%20Robert%20A.%20Martienssen%2C%20Mitsuhiro%20Yanagida&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000317" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000317" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000317" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000317&amp;t=Ribonuclease%20Activity%20of%20Dis3%20Is%20Required%20for%20Mitotic%20Progression%20and%20Provides%20a%20Possible%20Link%20between%20Heterochromatin%20and%20Kinetochore%20Function" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000317&title=Ribonuclease%20Activity%20of%20Dis3%20Is%20Required%20for%20Mitotic%20Progression%20and%20Provides%20a%20Possible%20Link%20between%20Heterochromatin%20and%20Kinetochore%20Function&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000317&amp;title=Ribonuclease%20Activity%20of%20Dis3%20Is%20Required%20for%20Mitotic%20Progression%20and%20Provides%20a%20Possible%20Link%20between%20Heterochromatin%20and%20Kinetochore%20Function" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000317" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000317&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Ribonuclease Activity of Dis3 Is Required for Mitotic Progression and Provides a Possible Link between Heterochromatin and Kinetochore Function';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000317';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Ribonuclease%20Activity%20of%20Dis3%20Is%20Required%20for%20Mitotic%20Progression%20and%20Provides%20a%20Possible%20Link%20between%20Heterochromatin%20and%20Kinetochore%20Function http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000317" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000317" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Anaphase%22" title="Search for articles in the subject area:'Anaphase'"><div class="flagText">Anaphase</div></a>
              <div data-categoryid="18585" data-articleid="23472"
                   data-categoryname="Anaphase"
                   class="flagImage" title="Flag 'Anaphase' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Centromeres%22" title="Search for articles in the subject area:'Centromeres'"><div class="flagText">Centromeres</div></a>
              <div data-categoryid="43147" data-articleid="23472"
                   data-categoryname="Centromeres"
                   class="flagImage" title="Flag 'Centromeres' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatids%22" title="Search for articles in the subject area:'Chromatids'"><div class="flagText">Chromatids</div></a>
              <div data-categoryid="42825" data-articleid="23472"
                   data-categoryname="Chromatids"
                   class="flagImage" title="Flag 'Chromatids' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Heterochromatin%22" title="Search for articles in the subject area:'Heterochromatin'"><div class="flagText">Heterochromatin</div></a>
              <div data-categoryid="35021" data-articleid="23472"
                   data-categoryname="Heterochromatin"
                   class="flagImage" title="Flag 'Heterochromatin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="23472"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribonucleases%22" title="Search for articles in the subject area:'Ribonucleases'"><div class="flagText">Ribonucleases</div></a>
              <div data-categoryid="46199" data-articleid="23472"
                   data-categoryname="Ribonucleases"
                   class="flagImage" title="Flag 'Ribonucleases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Schizosaccharomyces+pombe%22" title="Search for articles in the subject area:'Schizosaccharomyces pombe'"><div class="flagText">Schizosaccharomyces pombe</div></a>
              <div data-categoryid="33499" data-articleid="23472"
                   data-categoryname="Schizosaccharomyces pombe"
                   class="flagImage" title="Flag 'Schizosaccharomyces pombe' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Small+interfering+RNA%22" title="Search for articles in the subject area:'Small interfering RNA'"><div class="flagText">Small interfering RNA</div></a>
              <div data-categoryid="19855" data-articleid="23472"
                   data-categoryname="Small interfering RNA"
                   class="flagImage" title="Flag 'Small interfering RNA' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4636'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4476'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8085&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
