

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Mutants in the Mouse NuRD/Mi2 Component P66α Are Embryonic Lethal</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000519"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000519" />

    <meta name="citation_title" content="Mutants in the Mouse NuRD/Mi2 Component P66α Are Embryonic Lethal"/>
    <meta itemprop="name" content="Mutants in the Mouse NuRD/Mi2 Component P66α Are Embryonic Lethal"/>

      <meta name="citation_author" content="Susan Marino"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Developmental Biology, Beckman Center, Stanford University School of Medicine, Stanford, California, United States of America"/>
      <meta name="citation_author" content="Roel Nusse"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Developmental Biology, Beckman Center, Stanford University School of Medicine, Stanford, California, United States of America"/>

    <meta name="citation_date" content="2007/6/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000519.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e519"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Chromatin modification and epigenetic reprogramming in mammalian development.; citation_author=E Li; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=1; citation_pages=662-673; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The diverse functions of histone acetyltransferase complexes.; citation_author=MJ Carrozza; citation_author=RT Utley; citation_author=JL Workman; citation_author=J Cote; citation_journal_title=Trends Genet; citation_volume=19; citation_number=2; citation_pages=321-329; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Histone lysine methylation: a signature for chromatin function.; citation_author=RJ Sims; citation_author=K Nishioka; citation_author=D Reinberg; citation_journal_title=Trends Genet; citation_volume=19; citation_number=3; citation_pages=629-639; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Repressed by a NuRD.; citation_author=JM Crook; citation_author=NR Dunn; citation_author=A Colman; citation_journal_title=Nat Cell Biol; citation_volume=8; citation_number=4; citation_pages=212-214; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Targeted mutation of the DNA methyltransferase gene results in embryonic lethality.; citation_author=E Li; citation_author=TH Bestor; citation_author=R Jaenisch; citation_journal_title=Cell; citation_volume=69; citation_number=5; citation_pages=915-926; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=DNA methyltransferases Dnmt3a and Dnmt3b are essential for de novo methylation and mammalian development.; citation_author=M Okano; citation_author=DW Bell; citation_author=DA Haber; citation_author=E Li; citation_journal_title=Cell; citation_volume=99; citation_number=6; citation_pages=247-257; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Identification of a mammalian protein that binds specifically to DNA containing methylated CpGs.; citation_author=RR Meehan; citation_author=JD Lewis; citation_author=S McKay; citation_author=EL Kleiner; citation_author=AP Bird; citation_journal_title=Cell; citation_volume=58; citation_number=7; citation_pages=499-507; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Purification, sequence, and cellular localization of a novel chromosomal protein that binds to methylated DNA.; citation_author=JD Lewis; citation_author=RR Meehan; citation_author=WJ Henzel; citation_author=I Maurer-Fogy; citation_author=P Jeppesen; citation_journal_title=Cell; citation_volume=69; citation_number=8; citation_pages=905-914; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of a family of mammalian methyl-CpG binding proteins.; citation_author=B Hendrich; citation_author=A Bird; citation_journal_title=Mol Cell Biol; citation_volume=18; citation_number=9; citation_pages=6538-6547; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Analysis of the NuRD subunits reveals a histone deacetylase core complex and a connection with DNA methylation.; citation_author=Y Zhang; citation_author=HH Ng; citation_author=H Erdjument-Bromage; citation_author=P Tempst; citation_author=A Bird; citation_journal_title=Genes Dev; citation_volume=13; citation_number=10; citation_pages=1924-1935; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Methylation-dependent silencing at the H19 imprinting control region by MeCP2.; citation_author=RA Drewell; citation_author=CJ Goddard; citation_author=JO Thomas; citation_author=MA Surani; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=11; citation_pages=1139-1144; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=MeCP2 is a transcriptional repressor with abundant binding sites in genomic chromatin.; citation_author=X Nan; citation_author=FJ Campoy; citation_author=A Bird; citation_journal_title=Cell; citation_volume=88; citation_number=12; citation_pages=471-481; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Roles of histone acetyltransferases and deacetylases in gene regulation.; citation_author=MH Kuo; citation_author=CD Allis; citation_journal_title=Bioessays; citation_volume=20; citation_number=13; citation_pages=615-626; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Developmental roles of the Mi-2/NURD-associated protein p66 in Drosophila.; citation_author=C Kon; citation_author=KM Cadigan; citation_author=SL da Silva; citation_author=R Nusse; citation_journal_title=Genetics; citation_volume=169; citation_number=14; citation_pages=2087-2100; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MBD2/NuRD and MBD3/NuRD, Two Distinct Complexes with Different Biochemical and Functional Properties.; citation_author=X Le Guezennec; citation_author=M Vermeulen; citation_author=AB Brinkman; citation_author=WAM Hoeijmakers; citation_author=A Cohen; citation_journal_title=Mol Cell Biol; citation_volume=26; citation_number=15; citation_pages=843-851; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The MeCP1 complex represses transcription through preferential binding, remodeling, and deacetylating methylated nucleosomes.; citation_author=Q Feng; citation_author=Y Zhang; citation_journal_title=Genes Dev; citation_volume=15; citation_number=16; citation_pages=827-832; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Two highly related p66 proteins comprise a new family of potent transcriptional repressors interacting with MBD2 and MBD3.; citation_author=M Brackertz; citation_author=J Boeke; citation_author=R Zhang; citation_author=R Renkawitz; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=17; citation_pages=40958-40966; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification and functional characterization of the p66/p68 components of the MeCP1 complex.; citation_author=Q Feng; citation_author=R Cao; citation_author=L Xia; citation_author=H Erdjument-Bromage; citation_author=P Tempst; citation_journal_title=Mol Cell Biol; citation_volume=22; citation_number=18; citation_pages=536-546; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=p66{alpha} and p66{beta} of the Mi-2/NuRD complex mediate MBD2 and histone interaction.; citation_author=M Brackertz; citation_author=Z Gong; citation_author=J Leers; citation_author=R Renkawitz; citation_journal_title=Nucl Acids Res; citation_volume=34; citation_number=19; citation_pages=397-406; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=B Hogan; citation_author=R Beddington; citation_author=F Costantini; citation_author=E Lacy; citation_number=20; citation_date=1994; citation_publisher=Laboratory Presss; " />
      <meta name="citation_reference" content="citation_title=G9a histone methyltransferase plays a dominant role in euchromatic histone H3 lysine 9 methylation and is essential for early embryogenesis.; citation_author=M Tachibana; citation_author=K Sugimoto; citation_author=M Nozaki; citation_author=J Ueda; citation_author=T Ohta; citation_journal_title=Genes Dev; citation_volume=16; citation_number=21; citation_pages=1779-1791; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Loss of the Suv39h histone methyltransferases impairs mammalian heterochromatin and genome stability.; citation_author=AH Peters; citation_author=D O'Carroll; citation_author=H Scherthan; citation_author=K Mechtler; citation_author=S Sauer; citation_journal_title=Cell; citation_volume=107; citation_number=22; citation_pages=323-337; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=De novo DNA cytosine methyltransferase activities in mouse embryonic stem cells.; citation_author=H Lei; citation_author=SP Oh; citation_author=M Okano; citation_author=R Juttermann; citation_author=KA Goss; citation_journal_title=Development; citation_volume=122; citation_number=23; citation_pages=3195-3205; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Expression and localization of components of the histone deacetylases multiprotein repressory complexes in the mouse preimplantation embryo.; citation_author=B Kantor; citation_author=K Makedonski; citation_author=R Shemer; citation_author=A Razin; citation_journal_title=Gene Expr Patterns; citation_volume=3; citation_number=24; citation_pages=697-702; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mi2, an auto-antigen for dermatomyositis, is an ATP-dependent nucleosome remodeling factor.; citation_author=HB Wang; citation_author=Y Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=25; citation_pages=2517-2521; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Closely related proteins MBD2 and MBD3 play distinctive but interacting roles in mouse development.; citation_author=B Hendrich; citation_author=J Guy; citation_author=B Ramsahoye; citation_author=VA Wilson; citation_author=A Bird; citation_journal_title=Genes Dev; citation_volume=15; citation_number=26; citation_pages=710-723; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Essential function of histone deacetylase 1 in proliferation control and CDK inhibitor repression.; citation_author=G Lagger; citation_author=D O'Carroll; citation_author=M Rembold; citation_author=H Khier; citation_author=J Tischler; citation_journal_title=Embo J; citation_volume=21; citation_number=27; citation_pages=2672-2681; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The NuRD component Mbd3 is required for pluripotency of embryonic stem cells.; citation_author=K Kaji; citation_author=IM Caballero; citation_author=R MacLeod; citation_author=J Nichols; citation_author=VA Wilson; citation_journal_title=Nat Cell Biol; citation_volume=8; citation_number=28; citation_pages=285-292; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptional repression by the methyl-CpG-binding protein MeCP2 involves a histone deacetylase complex.; citation_author=X Nan; citation_author=HH Ng; citation_author=CA Johnson; citation_author=CD Laherty; citation_author=BM Turner; citation_journal_title=Nature; citation_volume=393; citation_number=29; citation_pages=386-389; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A mouse Mecp2-null mutation causes neurological symptoms that mimic Rett syndrome.; citation_author=J Guy; citation_author=B Hendrich; citation_author=M Holmes; citation_author=JE Martin; citation_author=A Bird; citation_journal_title=Nat Genet; citation_volume=27; citation_number=30; citation_pages=322-326; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The methyl-CpG binding domain and the evolving role of DNA methylation in animals.; citation_author=B Hendrich; citation_author=S Tweedie; citation_journal_title=Trends Genet; citation_volume=19; citation_number=31; citation_pages=269-277; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mi-2 complex couples DNA methylation to chromatin remodelling and histone deacetylation.; citation_author=PA Wade; citation_author=A Gegonne; citation_author=PL Jones; citation_author=E Ballestar; citation_author=F Aubry; citation_journal_title=Nat Genet; citation_volume=23; citation_number=32; citation_pages=62-66; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Failure of blood-island formation and vasculogenesis in Flk-1-deficient mice.; citation_author=F Shalaby; citation_author=J Rossant; citation_author=TP Yamaguchi; citation_author=M Gertsenstein; citation_author=XF Wu; citation_journal_title=Nature; citation_volume=376; citation_number=33; citation_pages=62-66; citation_date=1995; " />
      <meta name="citation_reference" content="citation_number=34; citation_date=2000; citation_publisher=Oxford University Press; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000519" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Mutants in the Mouse NuRD/Mi2 Component P66α Are Embryonic Lethal"/>
    <meta name="twitter:description" content="BackgroundThe NuRD/Mi2 chromatin complex is involved in histone modifications and contains a large number of subunits, including the p66 protein. There are two mouse and human p66 paralogs, p66&alpha; and p66&beta;. The functions of these genes are not clear, in part because there are no mutants available, except in invertebrate model systems.MethodologyWe made loss of function mutants in the mouse p66&alpha; gene (mp66&alpha;, official name Gatad2a, MGI:2384585). We found that mp66&alpha; is essential for development, as mutant embryos die around day 10 of embryogenesis. The gene is not required for normal blastocyst development or for implantation. The phenotype of mutant embryos and the pattern of gene expression in mutants are consistent with a role of mp66&alpha; in gene silencing.Conclusionmp66&alpha; is an essential gene, required for early mouse development. The lethal phenotype supports a role in execution of methylated DNA silencing."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000519.g005"/>

  <meta property="og:title" content="Mutants in the Mouse NuRD/Mi2 Component P66α Are Embryonic Lethal" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000519" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4987'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=558'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7207&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000519">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000519" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000519&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Mutants in the Mouse NuRD/Mi2 Component P66α Are Embryonic Lethal
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Susan Marino, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Howard Hughes Medical Institute, Department of Developmental Biology, Beckman Center, Stanford University School of Medicine, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Roel Nusse
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rnusse@stanford.edu">rnusse@stanford.edu</a></p>

                <p>Affiliation:
                  Howard Hughes Medical Institute, Department of Developmental Biology, Beckman Center, Stanford University School of Medicine, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 13, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000519</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000519" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000519" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000519" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000519" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000519-g001" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000519-g002" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000519-g003" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000519-g004" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000519-t001" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000519-g005" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000519-t002" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000519-t003" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000519">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000519" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The NuRD/Mi2 chromatin complex is involved in histone modifications and contains a large number of subunits, including the p66 protein. There are two mouse and human p66 paralogs, p66α and p66β. The functions of these genes are not clear, in part because there are no mutants available, except in invertebrate model systems.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We made loss of function mutants in the mouse p66α gene (mp66α, official name Gatad2a, MGI:2384585). We found that mp66α is essential for development, as mutant embryos die around day 10 of embryogenesis. The gene is not required for normal blastocyst development or for implantation. The phenotype of mutant embryos and the pattern of gene expression in mutants are consistent with a role of mp66α in gene silencing.</p>


<h3>Conclusion</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>mp66α is an essential gene, required for early mouse development. The lethal phenotype supports a role in execution of methylated DNA silencing.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Marino S, Nusse R (2007) Mutants in the Mouse NuRD/Mi2 Component P66α Are Embryonic Lethal. PLoS ONE 2(6):
          e519.
            doi:10.1371/journal.pone.0000519</p><p><strong>Academic Editor: </strong>Suzannah Rutherford, Fred Hutchinson Cancer Research Center, United States of America</p><p><strong>Received:</strong> April 23, 2007; <strong>Accepted:</strong> May 17, 2007; <strong>Published:</strong> June 13, 2007</p><p><strong>Copyright:</strong> © 2007 Marino, Nusse. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The work was supported by the Howard Hughes Medical Institute and a grant from the US ARMY Medical Research and Material Command (DAMD17-99-1-9386).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Epigenetic changes in gene expression, such as occurring during imprinting and X chromosome inactivation, can result from DNA methylation and from chromatin remodeling. The latter includes histone modifications such as deacetylation <a href="#pone.0000519-Li1">[1]</a>, acetylation <a href="#pone.0000519-Carrozza1">[2]</a>, and methylation <a href="#pone.0000519-Sims1">[3]</a>. Histone modifications and DNA methylation are functionally linked to each other. Histone modifications are catalyzed by a number of enzyme complexes, including the NuRD/Mi2 complex <a href="#pone.0000519-Crook1">[4]</a>. DNA methyltransferases maintain previously existing methylation (Dnmt1) and apply methyl groups to previously non-methylated sites (Dnmt3a and Dnmt3b) <a href="#pone.0000519-Li2">[5]</a>, <a href="#pone.0000519-Okano1">[6]</a>. Proteins that bind preferentially to methylated DNA have been isolated. These include MeCP2, and MBDs 1, 2, and 4 <a href="#pone.0000519-Meehan1">[7]</a>–<a href="#pone.0000519-Hendrich1">[9]</a>. Both MBD2 <a href="#pone.0000519-Zhang1">[10]</a> and MeCP2 <a href="#pone.0000519-Drewell1">[11]</a> can recruit histone deacetylase activity to methylated DNA, and this activity has been shown to repress reporter genes in cell culture studies. MBD2 interacts with the NuRD/Mi2 complex (<a href="#pone-0000519-g001">Figure 1</a>) and acts in the nucleus where it is localized to both euchromatic and heterochromatic foci, suggesting a role in facultative gene activity <a href="#pone.0000519-Zhang1">[10]</a>. MeCP2 interacts with the Sin3A complex and localizes primarily to pericentric heterochromatin <a href="#pone.0000519-Nan1">[12]</a>, <a href="#pone.0000519-Kuo1">[13]</a>, indicating that it may contribute to chromosome stability.</p>
<div class="figure" id="pone-0000519-g001"><div class="img"><a name="pone-0000519-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000519.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000519.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Composition of the NuRD/Mi2 chromatin complex, including p66.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Together with NuRD/Mi2 and MBD2 , p66 comprises the MeCP1 complex. MBD2 and MeCP2 can recruit histone deacetylase activity (HDAC) to methylated DNA See the text for references.</p>
<span>doi:10.1371/journal.pone.0000519.g001</span></div><a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Another protein in the NuRD/Mi2 chromatin complex is termed p66, cloned in <em>Drosophila</em> as a genetic modifier of Wingless signaling <a href="#pone.0000519-Kon1">[14]</a>. Together with NuRD/Mi2 and MBD2, p66 comprises the MeCP1 complex <a href="#pone.0000519-LeGuezennec1">[15]</a>, <a href="#pone.0000519-Feng1">[16]</a> (<a href="#pone-0000519-g001">Figure 1</a>). There are two human p66 paralogs, p66α and p66β <a href="#pone.0000519-Brackertz1">[17]</a>, encoding proteins with two conserved regions known as CR1 and CR2. The CR1 domain has been shown to bind strongly to MBD3, a scaffolding protein of the NuRD/Mi2 complex <a href="#pone.0000519-Brackertz1">[17]</a>. The 3′ conserved region of p66α, the Zn finger-containing CR2 domain, also interacts with MBD3, while the CR2 region of p66β targets the complex to multiple individual chromosomal loci in cell culture experiments <a href="#pone.0000519-Brackertz1">[17]</a>–<a href="#pone.0000519-Brackertz2">[19]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To characterize the activity of p66 and assess its importance in mouse development, we have made loss of function mutants in the mouse p66α gene (mp66α, official name Gatad2a, MGI:2384585). We report here that the gene is required for normal embryogenesis.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Cloning of mp66α, Genomic map, restriction mapping, and knock-out strategy</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>By a combination of EST and cDNA mapping, we assembled a 6.4 kb cDNA contig for mp66α. The transcript theoretically encodes a protein of 630 amino acids with 87% identity and 93% identity or similarity with human p66α (<a href="#pone-0000519-g002">Figure 2</a>). The protein consists of two known conserved regions (CR): CR1 contains a coiled-coil, a known protein-protein interaction domain, and CR2 possesses a GATA type zinc finger region (<a href="#pone-0000519-g002">Figure 2</a>). To design a construct for a targeted deletion, we isolated several BAC clones from mouse strain 129/Sv and mapped the location of the introns. The gene consists of 14 exons distributed over 15 kilobases (<a href="#pone-0000519-g003">Figure 3</a>). We developed a strategy of homologous recombination replacing the exons encoding CR1 and the two flanking exons (exons 2–4) by a floxed pgk-neo cassette (<a href="#pone-0000519-g003">Figure 3</a>). The resulting protein would consist of 86 amino acids from the non-conserved region, followed by 34 amino acids not encoded by the gene (due to frame-shifting) and a stop codon well before the CR1 region (see <a href="#s4">Materials and Methods</a>).</p>
<div class="figure" id="pone-0000519-g002"><div class="img"><a name="pone-0000519-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000519.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000519.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Sequence alignment of mouse and human p66α proteins.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Identical residues are indicated by a *. The coiled-coil (CR1) and GATA type zinc finger (CR2) regions are underlined. The arrows indicate the positions of the boundaries of Exons 2 and 4, which are eliminated after gene targeting. The truncated protein consists of the region upstream from the first arrow, followed by 34 frameshifted amino acids.</p>
<span>doi:10.1371/journal.pone.0000519.g002</span></div><div class="figure" id="pone-0000519-g003"><div class="img"><a name="pone-0000519-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000519.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000519.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>mp66α Knock-out and Genotyping Strategy.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>A: ploxPNT mp66α targeting vector. B: wild type mp66α genomic region. C: mutant mp66α genomic region. D: mutant mp66α with pgkneo cassette removed. Gray rectangles are exons. Heavy black lines at top indicate location of homologous arms. 5 and 6 are primers N-6 and 2-36 respectively. They were used to screen targeted clones by long range PCR for homologous recombination. The expected band size was 2069 nt. Labeled restriction sites and external probes 9 and 11 were used to confirm correct targeting and for some genotyping. 1–4 are primers used for PCR genotyping after recombination was shown to be correct by Southern blot genotyping. Short Arm: Probe 11 Long Arm: Probe 9 NcoI WT = 2335 bp Spe I, Sac I WT = 8724 bp NcoI mutant = 2799 bp Spe I, Sac I mutant = 7633 bp</p>
<span>doi:10.1371/journal.pone.0000519.g003</span></div>

<h4>Homologous recombination and generation of mp66α +/− mice</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We confirmed homologous recombination in 11% of the neomycin-resistant R1 embryonic stem cells (<a href="#pone-0000519-g004">Figure 4A, B</a>). Northern analysis of mutant ES cells showed that the mp66α RNA was aberrant compared to wild type (<a href="#pone-0000519-g004">Figure 4C</a>), confirming a successful deletion. Four independent ES cell clones were injected into a total of 90 blastocysts, producing 33 pups altogether. Of these, 14 were chimeric according to coat color. Two males, one each from clones 26 and 110, produced heterozygous offspring (<a href="#pone-0000519-g004">Figure 4D, E</a>). Heterozygotes were viable, normal looking, and fertile, producing offspring in normal Mendelian ratios when crossed with wild type animals. Genotyping of 375 3-week-old pups from heterozygote intercrosses revealed the presence of wild-type and heterozygous animals at the expected 1:2 ratio (<a href="#pone-0000519-t001">Table 1</a>), but no mp66α homozygous mutant pups were found among these animals, suggesting that the gene is essential for development.</p>
<div class="figure" id="pone-0000519-g004"><div class="img"><a name="pone-0000519-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000519.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000519.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Confirmation of correct targeting by molecular hybridization.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A: Southern blot of SpeI, SacI digested clones hybridized with probe 9. The long homologous arm is correctly integrated in clones numbered 26, 88,104 as indicated by the 7.6 kb fragment. B: Southern blot of Nco I digested clones hybridized with probe 11. The short homologous arm is integrated correctly in clones numbered 329 and 88, as indicated by the 2.8 kb fragment. C. Northern blot of RNA in +/− and −/− ES cells. mp66α RNA is normal in the +/− clone but aberrant in the −/− clone. GAPDH is the loading control. D, E. Pups 1, 2, 3, 4, and 6 are heterozygous for the correctly integrated mutant alleles as shown for both the long arm (D) and the short arm (E). These pups were the founders for two independently targeted lines, line 26 and line 110.</p>
<span>doi:10.1371/journal.pone.0000519.g004</span></div><div class="figure" id="pone-0000519-t001"><div class="img"><a name="pone-0000519-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000519.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000519.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Intercross Genotypic Ratios.</span></strong></p><span>doi:10.1371/journal.pone.0000519.t001</span></div>

<h4>Phenotypes of intercross embryos at day e8.5–e10.5</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We examined intercross embryos at days e8.5 (embryogenesis 8.5) through e12.5. Overall, we found a wide variety of defects. Virtually every organ of candidate mutant embryos could be affected, resulting in dramatically smaller embryos and advanced stages of necrosis. We could not identify particular defects or abnormalities that would suggest a specific step in embryogenesis being dependent on mp66α. We present this analysis in <a href="#pone-0000519-t001">tables 1</a> and <a href="#pone-0000519-t002">2</a>, related to the timing of developmental arrest; and in <a href="#pone-0000519-g005">figure 5</a> illustrating the degree of failure of development.</p>
<div class="figure" id="pone-0000519-g005"><div class="img"><a name="pone-0000519-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000519.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000519.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Mutant embryos.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Embryonic Day 9.5 and 10.5 mp66α −/− Embryos. Mutant embryos, significantly smaller, are shown in comparison with wild-type littermates. Abnormalities in mp66α null embryos around day 9.5 (A, B) include malformed and non-fused neural folds, failure of closure of anterior neuropore, missing or excessively large blood vessels in the yolk sac, Day 10.5 embryos (C,D) are disintegrating due to multiple defects.</p>
<span>doi:10.1371/journal.pone.0000519.g005</span></div><div class="figure" id="pone-0000519-t002"><div class="img"><a name="pone-0000519-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000519.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000519.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Intercross phenotypic ratios.</span></strong></p><span>doi:10.1371/journal.pone.0000519.t002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We found that by day e12.5, 61% of embryos were either wild-type or heterozygous, while 39% were in such an advanced stage of necrosis that they could not be examined or genotyped. For embryonic days 8.5–10.5, we initially genotyped 82 whole animals using a Southern blot protocol to account for all animals in a litter. We obtained ratios close to normal Mendelian ratios: +/+, 0.96; +/−, 1.92; −/−,0.84 (<a href="#pone-0000519-t001">Table 1</a>). However, of the 6 animals not genotyped in this group, 4 were less than half the size of siblings and were probably null. This would change the ratios to .96:1.92:1.04, indicating that all null embryos had implanted successfully. When 114 e9.5 and e10.5 animals were visually examined (<a href="#pone-0000519-t002">Table 2</a>), 24% displayed the arrested phenotype, supporting this finding. Therefore, mp66α is essential for development but not required for normal blastocyst development and implantation.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>An examination of 83 homozygous null e7.5–e10.5 embryos from additional litters revealed a range of defects, delayed development, and much smaller size than litter mates (<a href="#pone-0000519-g005">Figure 5</a>), but with healthy placentas and often healthy tissue despite severe delays.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>A normal day 8.5 embryo has 8–12 somites, is in the process of turning, has a paired heart primordial fusing anteriorly, and has closed neural folds at somites 4–5 <a href="#pone.0000519-Hogan1">[20]</a>. Of the 27 e8.5 null embryos examined, 7 were in the egg cylinder stage as expected at day 6 to 6.5. Two of these were lumpy and abnormally shaped and two were in the early neural fold stage but were misshapen (not shown). One of these had developed a single anterior neural fold, but no somites. The remaining 18 embryos had a normal appearance with somite numbers ranging from 3 to 9 (<a href="#pone-0000519-g005">Figure 5</a>).</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Abnormalities in mp66α null embryos around day 9.5 included malformed neural folds that were ruffled and not fused, failure of closure of anterior neuropore, missing or excessively large blood vessels in the yolk sac, turned but small underdeveloped embryos, and embryos that failed to turn despite having 20 or more somites (<a href="#pone-0000519-g005">Figure 5</a>).</p>


<h4>Differential gene expression in mp66α mutant embryos</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Given the evidence that p66 is part of a repressor complex, we were interested in asking whether specific genes are affected by the absence of mp66α in embryos. We carried out an Affymetrix microarray analysis of 4 pairs of null vs. wild type day 8.5 siblings that had achieved an equal level of development as judged during dissection.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Over 45 genes were identified as being increased in the null embryos of at least three but usually all four matched sibling pairs (<a href="#pone-0000519-t003">Table 3</a>). Four of these genes, Psx2, Psx1(Placenta specific homeobox 2 and 1), Cx31 (Connexin 31, specific to trophoblast stem cells), and an unknown bHLH protein similar to Mash 2 are active in the trophoblast lineage in the extraembryonic tissues of the mouse. The remaining genes are either unknown, or not confined to the extraembryonic lineages. Of particular interest are two genes normally expressed in males only, but not found on the Y chromosome, Amhr2 (anti-Mullerian hormone type 2 receptor) and Ldh3 (lactate dehydrogenase 3, C chain, expressed normally in the germinal epithelium of the testis). Despite the fact that three of the null embryos were females, these genes were expressed at heightened levels.</p>
<div class="figure" id="pone-0000519-t003"><div class="img"><a name="pone-0000519-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000519" data-uri="info:doi/10.1371/journal.pone.0000519.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000519.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000519.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000519.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000519.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000519.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Genes differentially expressed in mp66α mutant embryos.</span></strong></p><span>doi:10.1371/journal.pone.0000519.t003</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Because of the presumed role of p66 proteins in epigenetic gene repression, we compared the mp66α null embryonic phenotype with published mutants in other genes implicated in this process: the DNA methyltransferases, Dnmt1, the maintenance methyltransferase, and Dnmt3a and Dnmt 3b, the de novo methyltransferases, and the histone methyltransferases G9a and SUV39H <a href="#pone.0000519-Sims1">[3]</a>. G9a <a href="#pone.0000519-Tachibana1">[21]</a> is the histone methyltransferase thought to be responsible for H3-K9 methylation in euchromatin and therefore may be involved in developmental gene regulation. SUV39H1 <a href="#pone.0000519-Peters1">[22]</a> is the name for both SUV39h homologs of this histone methyltransferase known to be active in the pericentric heterochromatin and required for genome stability.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>The mp66α null embryos were very similar to the Dnmt1 null, Dnmt3a/3b double null, and the G9a null embryos. The Dnmt1 null embryo <a href="#pone.0000519-Lei1">[23]</a> is distinguishable from wild type embryos at day e8.5 and by day e9.5 often has distorted neural tubes and no somites. The most advanced null embryos had no more than 8 somites. The Dnmt3a/3b double null embryos were of smaller size, had abnormal morphology at eE8.5 and e9.5, and died before e11.5. They had no somites and did not turn <a href="#pone.0000519-Okano1">[6]</a>. The G9a null embryos do not develop beyond the 5–6 somite stage, representative of approximately day 8.25, though they survive until day 9.5 when they begin to die. Like mp66α null embryos, some G9a null embryos arrest as early as the egg cylinder and neural plate stage <a href="#pone.0000519-Tachibana1">[21]</a>. The mp66α null phenotype was slightly less severe than each of these – while the embryos died at about the same time, the mp66α null embryos often had 20–30 somites. Nevertheless, the consistency of these phenotypes implicates mp66α in the epigenetic events that become important as differentiation begins.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The NuRD/Mi2 complex, which includes p66 is found in all fetal and adult tissues and its components are expressed throughout early embryonic development from at least the 8–16 cell stage onward <a href="#pone.0000519-Kantor1">[24]</a>. Its nucleosomal remodeling ability is the highest of the SWI2/SNF2 type ATPases <a href="#pone.0000519-Wang1">[25]</a> and it is the most active HDAC activity found in <em>Xenopus</em> and mammalian cells. By making mice mutant for mp66α, we show that the gene is essential for normal development.</p>

<h4>The lethal phenotype is consistent with a role in execution of methylated DNA silencing</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The mp66α−/− mutation results in an embryonic lethal phenotype at around day e9.5. Because methylated DNA binding proteins or activities are capable of repressing reporter genes by recruiting histone deacetylase activity, the loss of HDAC due to loss of mp66α should result in a DNA methyltransferase knockout phenotype. The maintenance methyltransferase (Dnmt1)−/− embryo dies by the eight somite stage, around day e8.5), while the de novo Dnmt3a/3b methyltransferase double null embryo dies before e11.5, with a variety of developmental abnormalities <a href="#pone.0000519-Li2">[5]</a>, <a href="#pone.0000519-Okano1">[6]</a>. This is consistent with the fact that methylation and developmental gene silencing begins in earnest when the embryo implants and gastrulation begins. The death of the mp66α−/− embryo at e9.5 with variable developmental defects supports therefore a role for mp66α in gene silencing.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The mp66α null phenotype appears to be less severe than the knock-out phenotypes of two individual NuRD/Mi2 components. The MBD3 null embryo is less developed than Dnmt−/− embryo and the HDAC1null embryo dies prior to e9.5 with a cell proliferation defect <a href="#pone.0000519-Hendrich2">[26]</a>, <a href="#pone.0000519-Lagger1">[27]</a>. Moreover, MBD3-deficient embryonic stem cells, while able to initiate differentiation in embryoid bodies, fail to commit to developmental lineages <a href="#pone.0000519-Kaji1">[28]</a>, whereas we found that mp66α homozygous mutant ES cells differentiate normally (data not shown). These differences may be the result of non-MeCP1 functions of the NuRD/Mi2 complex. This may not be unexpected, as mammalian p66α has only been associated with NuRD/Mi2 in the MeCP1 context, yet MeCP1 contains only about 10% of the NuRD in the cell <a href="#pone.0000519-Feng1">[16]</a>.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>On the other hand, we found that the mp66α phenotype is also not as severe as that due to loss of all methylation. mp66α−/− embryonic stem cells were capable of differentiating into embryoid bodies at normal rates (data not shown), while Dnmt1 homozygous mutant ES cells are reported to undergo more limited differentiation at slower rates <a href="#pone.0000519-Lei1">[23]</a>. This suggests that if the phenotype is a result of the elimination of MeCP1activity, then MeCP1 probably mediates only a subset of the methylation-dependent repression. This might be expected because it is known that the MeCP2 and the Sin3A HDAC complex are active in the pericentromeric heterochromatin <a href="#pone.0000519-Hendrich1">[9]</a>, <a href="#pone.0000519-Nan2">[29]</a>. Alternatively, this result could mean that loss of mp66α only partially disables the MeCP1 function.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>If DNA methylation were dependent upon H3-K9 methylation, and if mp66α were required for H3-K9 deacetylation, one would expect to see loss of DNA methylation in mp66α−/− ES cells, even in a hypomethylated state. We used probe for minor satellite methylation and an IAP probe to assess methylation of dispersed repetitive elements. DNA methylation at all these loci was normal in the ES cell cultures (data not shown).</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>Given the fact that methylated DNA binding activities and proteins have been shown to repress reporter genes through histone deacetylase activity, it has been puzzling that the MeCP2 and MBD2 knockouts have given viable, mild phenotypes <a href="#pone.0000519-Hendrich2">[26]</a>, <a href="#pone.0000519-Guy1">[30]</a>. Adding to the mystery is the fact that until now, only one endogenous gene has been shown to be affected by a methylated DNA binding protein or activity, and in this case the gene was silenced by the absence of the binding protein, MBD2 <a href="#pone.0000519-Hendrich3">[31]</a>, the reverse effect.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cloning and genomic structure</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We used the 3′ UTR sequence of an existing mp66α clone to identify additional mp66α clones in the NCBI mouse EST database, of which only two extended beyond the UTR (AI507147; AI429910). We sequenced these and assembled a sequence (<a href="#pone-0000519-g002">Figure 2</a>) that extended 61 amino acids N terminal of the published <em>Xenopus</em> p66α start site <a href="#pone.0000519-Wade1">[32]</a>. Incomplete genomic sequence in the NCBI database indicated that there was another exon 37 kb upstream that included the probable start site with an imperfect Kozak sequence, but there were gaps in the intervening cDNA sequence. This position, 141 amino acids N terminal of the xp66α start site has now been established as the hp66α start site <a href="#pone.0000519-Brackertz1">[17]</a>. Recent mouse ESTs (BY282871; CA317502) place the orthologous start site at 139 amino acids N terminal of the mp66α start site. We will assume this is the mouse start site as the human and mouse genes are almost identical in all other respects.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>We probed a BAC library (CITB Mouse BAC Clones, Research Genetics, Cat. No. 96022) derived from mouse strain 129/Sv and obtained 4 positive clones for mp66α (95E12; 412G8; 504L10; 604M1). Except for the 37 kb intron, we mapped the genomic structure using long-range PCR with primers made to the cDNA at regular intervals, and sequenced the amplimers into exon/intron boundaries when introns were detected. We identified critical sites for cloning and verified their presence in the 129/Sv genomic DNA we was using for the construct. For the long-range PCR we used the level 2 protocol of the Expand<sup>TM</sup> Long Template PCR System (Boehringer Mannheim) but scaled down to a 25 µL reaction volume.</p>


<h4>Gene targeting, ES cell culture, and genotyping</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We made the targeting construct by cutting a 1.4 kb short homologous arm and a 6.3 kb long homologous arm from the 129/Sv BAC clone 95E12, then used a shuttle vector to place these arms into the targeting vector, ploxPNT. The targeting vector was designed to replace exons 2–4 of mp66α with a floxed pgkneo cassette. The homologous arms for the targeting construct were cut from BAC 95E12, placed into Bluescript SK-, and finally into the targeting vector ploxPNT (<a href="#pone-0000519-g003">Figure 3</a>) <a href="#pone.0000519-Shalaby1">[33]</a>. A portion of each BAC digest was analyzed by Southern blot for identification of the correct band. The HpaII/PstI 1.4 kb short arm fragment was placed into PstI/ClaI site of BS SK- modified by the removal of the Xho1 site by SalI/XhoI digestion and religation. The BamHI/KpnI fragment was then removed and placed into the BamHI/KpnI site of ploxPNT. The EcoRI/DraI 6.3 kb long arm fragment was placed into the EcoRI/SmaI site of BS SK-. The XhoI/NotI fragment was then removed and placed into the XhoI/NotI site of ploxPNT. Internal probes and PCR assays for the short and long arms utilized the following primers: short arm: 2-24 (5′-gcccaccatgccactcttctt-3′), 2-25 (5′-gtcagatgcacaggtccacaac-3′); long arm: 2-14 (5′-cgaggtgggcagcaggtgtct-3′), 2-M2 (5′-ggggacattggcgacacggataag-3′). All junctions of the completed targeting construct were sequenced to confirm correct construction.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Gene targeting was performed on passage #15 R1 ES cells purchased from the Stanford Transgenic Research Facility. This line was developed by the Andras Nagy lab (<a href="http://www.mshri.on.ca/nagy/">http://www.mshri.on.ca/nagy/</a>) in 1991 from 129/Svx129/SV-CP stock and has agouti coat color in the F1 generation. ES cells were grown in gelatinized dishes on neomycin resistant γ irradiated mouse embryonic fibroblasts in ES complete medium consisting of DME-H21 (UCSF#AA400) supplemented with 20% fetal bovine serum (Hyclone, ES tested lot AKH12368), 0.1 mM non-essential amino acids (Gibco#11140-0500), 1× penicillin-streptomycin (Gibco #15140-122), 1 mM sodium pyruvate (Gibco#11360-070), 100 µM β-Mercaptoethanol (Sigma # M-7522), and 2000 U/mL murine LIF (ESGRO<sup>TM</sup>, Chemicon ESG1106) (Stanford Transgenic Facility protocol) <a href="#pone.0000519-Joyner1">[34]</a>. The targeting vector was linearized at the Not 1 site and the ES cells were transfected by electroporation. After 24 hours 0.5 mg/mL G418 (Cellgro MT61-234-RF) and 2 µM gancyclovir (Syntex) were added for positive and negative selection. After 7–10 days positive colonies were picked and grown for stocks and analysis. Homologous recombinants were identified first by detection of a 2069 bp band by long range PCR (above) over the short (5′) arm with primers 2-36 (5′-acctggaattgaagctgtctgat-3′) and N-6 (5′-cgccttcttgacgagttcttctg-3′), and then they were confirmed by Southern blot analysis of both arms. For the short arm, ES cell DNA was digested with NcoI and the blots probed with probe 11, a 550 bp fragment amplified from BAC 95E12 with primers 2-39 (5′-tggaaggcacagttaaagggtcac-3′) and 2-40 (5′-ccccgaggtcaggtttcaaggtg-3′). The wild type allele gave a 2.34 kb band while homologously recombined alleles gave a 2.80 kb band. For the long arm, DNA was digested with SpeI and SacI, then probed with probe 9, a 535 bp fragment amplified from BAC 95E12 with primers 2-41 (5′-ggtagcatcccatagtcccaggtt-3′) and 2-42 (5′-atgtggctgattgggggatgg-3′). The wild type gene gave an 8.72 kb band, while homologous recombinants gave a 7.63 kb band (<a href="#pone-0000519-g004">Figure 4</a>).</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>After southern blot confirmation of homologous recombination in cell and animal lines, a PCR genotyping protocol was developed utilizing primers 2-30 (5′-agacttgagtgtgcagggttgagg-3′), 2-32 (5′-gccgggtattcgagtggaggaggt-3′), 2-Q (5′-aatcacatcaggcgagggaggtct-3′), and N-5 (5′-ttggctacccgtgatattgctgaa-3′). N-5 is found only in the pgk-neo cassette while 2-Q is only in the targeted region; primers 2-30 and 2-32 flank the targeted region. A 25 µL PCR reaction consisting of 0.4 µM primer 2-30, 0.2 µM each remaining primer, 0.8 mM dNTPs, 1.5 U Taq DNA polymerase (Promega, Cat.M2861), 1× Mg-free supplied PCR buffer, 4.5 mM MgCl<sub>2</sub>, 1 µL template, and H<sub>2</sub>O to volume, was cycled as follows: 1 cycle: 94°C, 2 min; 30 cycles: 94°C, 1 min, 61°C, 1 min, 72°C, 1 min; 1 cycle: 72°C, 5 min. Primers 2-30 and N-5 gave an 835 mutant band; primers 2-30 and 2-Q gave a 632 bp wild type band; primers 2-30 and 2-32 gave a 322 bp Δ mutant band and a 2000 bp wild type band.</p>


<h4>Generation and breeding of mice</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Correctly targeted ES cells were injected into C57BL/6 blastyocysts <a href="#pone.0000519-Hogan1">[20]</a> at the Stanford Transgenic Research Facility. Chimeric pups were identified by mixed black and agouti coat color and were crossed with C57BL/6 females at 6 weeks of age. The chimeras with the highest percentage of 129/Sv agouti color all developed teratocarcinomas (3) or were sterile (3). Three more were non-transmitters, and three were female. Agouti pup tail DNA was genotyped initially by Southern blot (above) but once the mutation was confirmed in both lines 110 and 26, a PCR protocol was used. The phenotype was analyzed for the most part on a mixed C57BL/6X129/Sv background in which there was no more than 75% of one of the strains. However, we did obtain a few 100% 129/Sv pups for both lines by crossing the proven chimeras with 129/Sv females.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>We wanted to eliminate possible confounding effects of the pgk promoter on neighboring genes. To do this, we crossed the heterozygous mp66α females to a CMVCre recombinase male to produce offspring mosaic for the pgkneo deletion. These were then crossed to wild type animals to obtain pure Δpgk mp66α lines for both lines 26 and 110 (<a href="#pone-0000519-g004">Figure 4D</a>). These results were on animals with a mixed 129/Sv X C57Bl/6 background. It was possible that variability would be reduced on an isogenic background. We crossed the chimeras with 129/Sv females and obtained 100% 129/Sv heterozygotes. Although fertility was poor in these lines, we were able to confirm the phenotype in a few (6) embryos on the 100% background. In this case the variability ranged from abnormal looking neural folds at e8.5 to embryos that did not turn at e9.5.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>To remove the floxed pgk-neo cassette, we crossed heterozygous mp66α females of both lines, 110 and 26, to a CMV-Cre male on a C57BL/129/SV background. Offspring receiving the CMV-Cre transgene were mosaic for the deletion so these were crossed again to wild type to produce pure Δmp66α lines on a mixed background. The Δmp66α110 and Δmp66α26 lines were then maintained in the heterozygous condition on a mixed background as stock.</p>


<h4>Microarray analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Because aberrant RNA is produced in the null animals, we could genotype the embryonic RNA using RT-PCR. Individual RNA from three litters was examined and genotyped for quality and mp66α genotype. Extraembryonic membranes were included with each embryo. Four sibling pairs were chosen from the three litters for microarray analysis. An Affymetrix protocol was used so that each embryo produced an individual set of expression data that could be compared as desired, and all sample pairs were prepared simultaneously. Sex was determined by examining the expression data. Two pairs were entirely female while two other pairs were mixed. The data for each pair was examined in parallel for each gene and genes were not listed unless they were above a for at least three of the four pairs. There was one male each of null and wild type, so these were compared to each other also. The wild type male and wild type females were compared to confirm known male/female differences and to include them in interpretation. In all cases examined for known genes, the results were sex appropriate.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>PolyA-RNA was then isolated using the spin column protocol of the Oligotex mRNA midi kit (Qiagen). For microarray analysis, the RNA was quantitated in a Beckman DU530 spectrophotometer and labeled cDNA was made using the Cyscribe labeling kit. The lab reference RNA, (1/4 four-week whole brain : 3/4 e 12.5 complete embryo), was labeled with Cy3 and the ES cell RNA was labeled with Cy5. 0.25 M NaOH was added to degrade RNA during a 15 minute 37°C incubation, then 0.7 M HEPES free acid was added to neutralize. Final purification was carried out with the CyScribe GFX Purification Kit. Absorbance at 550 nm(Cy3) and 650 nm (Cy5) was determined to quantitate incorporation of label, so that equimolar amounts of Cy3 and Cy5 could be used for hybridization. Mouse microarrays were purchased from the SFGF facility, and were stabilized, immobilized, and pre-hybridized as follows. Slides were etched in one corner to identify side of slide printed with the array, and then rehydrated, array side down, over 45°C 1× SSC, then snap-dried on a 100°C surface until the vapor disappeared, then UV cross-linked at 65 mJoules. To prehybridize, arrays were incubated with 50% formamide, 3× SSC, 0.1%SDS, 0.1 mg/mL BSA for 45 minutes at 42°C, immersed in milliQ water to remove SDS, and dried by centrifugation at 600 rpm for 2 minutes.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>For hybridization, 40 pmol of reference and ES cell DNA were mixed, 20 ug each of mouse Cot1 DNA, polyA RNA, and tRNA were added, and the sample was concentrated to 28 µL in a foil covered speed-vac. 6 µL 20× SSC and 0.5 µL 20% SDS were added, samples were denatured at 100°C for 2 minutes, and then spun at 14,000 rpm in a microfuge for 15 minutes. The mixture was then pipetted onto the microarray slide and covered. Hybridization was carried out in a Gene Machines HybChamber<sup>TM</sup> humidified with 3× SSC and submersed in a 65°C water bath for 14–18 hours. The slides were then washed in foil covered slide dishes as follows: 2× SSC, 0.2% SDS at 55°C, 10 minutes, removing cover slips when immersed; 1× SSC. 0.2% SDS at 55°C, 10 minutes; 0.1× SSC, 0.2% SDS at RT, 10 minutes; 0.1× SSC at RT, 10 minutes; distilled water for 10 seconds followed immediately by centrifugation drying as above. The slides were then immediately scanned with a GenePix scanner with photomultiplier tube voltages in each channel adjusted to a ratio of between 0.8–1.2. Images were then gridded and flagged using the GenePix Pro 4.0 Software, then sent to the Stanford Microarray Database.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Charlene Kon, Derk ten Berge and other labmembers for advice during these studies.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RN SM. Performed the experiments: SM. Analyzed the data: RN SM. Wrote the paper: RN SM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000519-Li1" id="pone.0000519-Li1"></a>Li E (2002) Chromatin modification and epigenetic reprogramming in mammalian development. Nat Rev Genet  3: 662–673.  <ul class="find" data-citedArticleID="958020" data-doi="10.1038/nrg887"><li><a href="http://dx.doi.org/10.1038/nrg887" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+modification+and+epigenetic+reprogramming+in+mammalian+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+modification+and+epigenetic+reprogramming+in+mammalian+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000519-Carrozza1" id="pone.0000519-Carrozza1"></a>Carrozza MJ, Utley RT, Workman JL, Cote J (2003) The diverse functions of histone acetyltransferase complexes. Trends Genet  19: 321–329.  <ul class="find" data-citedArticleID="957982" data-doi="10.1016/s0168-9525(03)00115-x"><li><a href="http://dx.doi.org/10.1016/s0168-9525(03)00115-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+diverse+functions+of+histone+acetyltransferase+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+diverse+functions+of+histone+acetyltransferase+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000519-Sims1" id="pone.0000519-Sims1"></a>Sims RJ 3rd, Nishioka K, Reinberg D (2003) Histone lysine methylation: a signature for chromatin function. Trends Genet  19: 629–639.  <ul class="find" data-citedArticleID="958036" data-doi="10.1016/j.tig.2003.09.007"><li><a href="http://dx.doi.org/10.1016/j.tig.2003.09.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+lysine+methylation%3A+a+signature+for+chromatin+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+lysine+methylation%3A+a+signature+for+chromatin+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000519-Crook1" id="pone.0000519-Crook1"></a>Crook JM, Dunn NR, Colman A (2006) Repressed by a NuRD. Nat Cell Biol  8: 212–214.  <ul class="find" data-citedArticleID="957984" data-doi="10.1038/ncb0306-212"><li><a href="http://dx.doi.org/10.1038/ncb0306-212" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repressed+by+a+NuRD." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repressed+by+a+NuRD.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000519-Li2" id="pone.0000519-Li2"></a>Li E, Bestor TH, Jaenisch R (1992) Targeted mutation of the DNA methyltransferase gene results in embryonic lethality. Cell  69: 915–926.  <ul class="find" data-citedArticleID="958022" data-doi="10.1016/0092-8674(92)90611-f"><li><a href="http://dx.doi.org/10.1016/0092-8674(92)90611-f" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeted+mutation+of+the+DNA+methyltransferase+gene+results+in+embryonic+lethality." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeted+mutation+of+the+DNA+methyltransferase+gene+results+in+embryonic+lethality.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000519-Okano1" id="pone.0000519-Okano1"></a>Okano M, Bell DW, Haber DA, Li E (1999) DNA methyltransferases Dnmt3a and Dnmt3b are essential for de novo methylation and mammalian development. Cell  99: 247–257.  <ul class="find" data-citedArticleID="958030" data-doi="10.1016/s0092-8674(00)81656-6"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81656-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methyltransferases+Dnmt3a+and+Dnmt3b+are+essential+for+de+novo+methylation+and+mammalian+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methyltransferases+Dnmt3a+and+Dnmt3b+are+essential+for+de+novo+methylation+and+mammalian+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000519-Meehan1" id="pone.0000519-Meehan1"></a>Meehan RR, Lewis JD, McKay S, Kleiner EL, Bird AP (1989) Identification of a mammalian protein that binds specifically to DNA containing methylated CpGs. Cell  58: 499–507.  <ul class="find" data-citedArticleID="958024" data-doi="10.1016/0092-8674(89)90430-3"><li><a href="http://dx.doi.org/10.1016/0092-8674(89)90430-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+mammalian+protein+that+binds+specifically+to+DNA+containing+methylated+CpGs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+mammalian+protein+that+binds+specifically+to+DNA+containing+methylated+CpGs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000519-Lewis1" id="pone.0000519-Lewis1"></a>Lewis JD, Meehan RR, Henzel WJ, Maurer-Fogy I, Jeppesen P, et al.  (1992) Purification, sequence, and cellular localization of a novel chromosomal protein that binds to methylated DNA. Cell  69: 905–914.  <ul class="find" data-citedArticleID="958018" data-doi="10.1016/0092-8674(92)90610-o"><li><a href="http://dx.doi.org/10.1016/0092-8674(92)90610-o" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification%2C+sequence%2C+and+cellular+localization+of+a+novel+chromosomal+protein+that+binds+to+methylated+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification%2C+sequence%2C+and+cellular+localization+of+a+novel+chromosomal+protein+that+binds+to+methylated+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000519-Hendrich1" id="pone.0000519-Hendrich1"></a>Hendrich B, Bird A (1998) Identification and characterization of a family of mammalian methyl-CpG binding proteins. Mol Cell Biol  18: 6538–6547.  <ul class="find" data-citedArticleID="957994" data-doi="10.1017/s0016672398533307"><li><a href="http://dx.doi.org/10.1017/s0016672398533307" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+a+family+of+mammalian+methyl-CpG+binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+a+family+of+mammalian+methyl-CpG+binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000519-Zhang1" id="pone.0000519-Zhang1"></a>Zhang Y, Ng HH, Erdjument-Bromage H, Tempst P, Bird A, et al.  (1999) Analysis of the NuRD subunits reveals a histone deacetylase core complex and a connection with DNA methylation. Genes Dev  13: 1924–1935.  <ul class="find" data-citedArticleID="958044" data-doi="10.1101/gad.13.15.1924"><li><a href="http://dx.doi.org/10.1101/gad.13.15.1924" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+NuRD+subunits+reveals+a+histone+deacetylase+core+complex+and+a+connection+with+DNA+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+NuRD+subunits+reveals+a+histone+deacetylase+core+complex+and+a+connection+with+DNA+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000519-Drewell1" id="pone.0000519-Drewell1"></a>Drewell RA, Goddard CJ, Thomas JO, Surani MA (2002) Methylation-dependent silencing at the H19 imprinting control region by MeCP2. Nucleic Acids Res  30: 1139–1144.  <ul class="find" data-citedArticleID="957986" data-doi="10.1093/nar/30.5.1139"><li><a href="http://dx.doi.org/10.1093/nar/30.5.1139" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methylation-dependent+silencing+at+the+H19+imprinting+control+region+by+MeCP2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methylation-dependent+silencing+at+the+H19+imprinting+control+region+by+MeCP2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000519-Nan1" id="pone.0000519-Nan1"></a>Nan X, Campoy FJ, Bird A (1997) MeCP2 is a transcriptional repressor with abundant binding sites in genomic chromatin. Cell  88: 471–481.  <ul class="find" data-citedArticleID="958026" data-doi="10.1016/s0092-8674(00)81887-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81887-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MeCP2+is+a+transcriptional+repressor+with+abundant+binding+sites+in+genomic+chromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MeCP2+is+a+transcriptional+repressor+with+abundant+binding+sites+in+genomic+chromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000519-Kuo1" id="pone.0000519-Kuo1"></a>Kuo MH, Allis CD (1998) Roles of histone acetyltransferases and deacetylases in gene regulation. Bioessays  20: 615–626.  <ul class="find" data-citedArticleID="958010" data-doi="10.1002/(sici)1521-1878(199808)20:8<615::aid-bies4&gt;3.0.co;2-h"><li><a href="http://dx.doi.org/10.1002/(sici)1521-1878(199808)20:8<615::aid-bies4&gt;3.0.co;2-h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Roles+of+histone+acetyltransferases+and+deacetylases+in+gene+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Roles+of+histone+acetyltransferases+and+deacetylases+in+gene+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000519-Kon1" id="pone.0000519-Kon1"></a>Kon C, Cadigan KM, da Silva SL, Nusse R (2005) Developmental roles of the Mi-2/NURD-associated protein p66 in Drosophila. Genetics  169: 2087–2100.  <ul class="find" data-citedArticleID="958008" data-doi="10.1534/genetics.104.034595"><li><a href="http://dx.doi.org/10.1534/genetics.104.034595" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developmental+roles+of+the+Mi-2%2FNURD-associated+protein+p66+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developmental+roles+of+the+Mi-2%2FNURD-associated+protein+p66+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000519-LeGuezennec1" id="pone.0000519-LeGuezennec1"></a>Le Guezennec X, Vermeulen M, Brinkman AB, Hoeijmakers WAM, Cohen A, et al.  (2006) MBD2/NuRD and MBD3/NuRD, Two Distinct Complexes with Different Biochemical and Functional Properties. Mol Cell Biol  26: 843–851.  <ul class="find" data-citedArticleID="958014" data-doi="10.1128/mcb.26.3.843-851.2006"><li><a href="http://dx.doi.org/10.1128/mcb.26.3.843-851.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MBD2%2FNuRD+and+MBD3%2FNuRD%2C+Two+Distinct+Complexes+with+Different+Biochemical+and+Functional+Properties." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MBD2%2FNuRD+and+MBD3%2FNuRD%2C+Two+Distinct+Complexes+with+Different+Biochemical+and+Functional+Properties.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000519-Feng1" id="pone.0000519-Feng1"></a>Feng Q, Zhang Y (2001) The MeCP1 complex represses transcription through preferential binding, remodeling, and deacetylating methylated nucleosomes. Genes Dev  15: 827–832.  <ul class="find" data-citedArticleID="957988"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+MeCP1+complex+represses+transcription+through+preferential+binding%2C+remodeling%2C+and+deacetylating+methylated+nucleosomes.&amp;auth=&amp;atitle=The+MeCP1+complex+represses+transcription+through+preferential+binding%2C+remodeling%2C+and+deacetylating+methylated+nucleosomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+MeCP1+complex+represses+transcription+through+preferential+binding%2C+remodeling%2C+and+deacetylating+methylated+nucleosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+MeCP1+complex+represses+transcription+through+preferential+binding%2C+remodeling%2C+and+deacetylating+methylated+nucleosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000519-Brackertz1" id="pone.0000519-Brackertz1"></a>Brackertz M, Boeke J, Zhang R, Renkawitz R (2002) Two highly related p66 proteins comprise a new family of potent transcriptional repressors interacting with MBD2 and MBD3. J Biol Chem  277: 40958–40966.  <ul class="find" data-citedArticleID="957978" data-doi="10.1074/jbc.m207467200"><li><a href="http://dx.doi.org/10.1074/jbc.m207467200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+highly+related+p66+proteins+comprise+a+new+family+of+potent+transcriptional+repressors+interacting+with+MBD2+and+MBD3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+highly+related+p66+proteins+comprise+a+new+family+of+potent+transcriptional+repressors+interacting+with+MBD2+and+MBD3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000519-Feng2" id="pone.0000519-Feng2"></a>Feng Q, Cao R, Xia L, Erdjument-Bromage H, Tempst P, et al.  (2002) Identification and functional characterization of the p66/p68 components of the MeCP1 complex. Mol Cell Biol  22: 536–546.  <ul class="find" data-citedArticleID="957990" data-doi="10.1128/mcb.22.2.536-546.2002"><li><a href="http://dx.doi.org/10.1128/mcb.22.2.536-546.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+functional+characterization+of+the+p66%2Fp68+components+of+the+MeCP1+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+functional+characterization+of+the+p66%2Fp68+components+of+the+MeCP1+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000519-Brackertz2" id="pone.0000519-Brackertz2"></a>Brackertz M, Gong Z, Leers J, Renkawitz R (2006) p66{alpha} and p66{beta} of the Mi-2/NuRD complex mediate MBD2 and histone interaction. Nucl Acids Res  34: 397–406.  <ul class="find" data-citedArticleID="957980" data-doi="10.1093/nar/gkj437"><li><a href="http://dx.doi.org/10.1093/nar/gkj437" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=p66%7Balpha%7D+and+p66%7Bbeta%7D+of+the+Mi-2%2FNuRD+complex+mediate+MBD2+and+histone+interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22p66%7Balpha%7D+and+p66%7Bbeta%7D+of+the+Mi-2%2FNuRD+complex+mediate+MBD2+and+histone+interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000519-Hogan1" id="pone.0000519-Hogan1"></a>Hogan B, Beddington R, Costantini F, Lacy E (1994) Manipulating the Mouse Embryo. A Laboratory Manual. Second Edition. Cold Spring Harbor: Laboratory Presss.   <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0000519-Tachibana1" id="pone.0000519-Tachibana1"></a>Tachibana M, Sugimoto K, Nozaki M, Ueda J, Ohta T, et al.  (2002) G9a histone methyltransferase plays a dominant role in euchromatic histone H3 lysine 9 methylation and is essential for early embryogenesis. Genes Dev  16: 1779–1791.  <ul class="find" data-citedArticleID="958038" data-doi="10.1101/gad.989402"><li><a href="http://dx.doi.org/10.1101/gad.989402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=G9a+histone+methyltransferase+plays+a+dominant+role+in+euchromatic+histone+H3+lysine+9+methylation+and+is+essential+for+early+embryogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22G9a+histone+methyltransferase+plays+a+dominant+role+in+euchromatic+histone+H3+lysine+9+methylation+and+is+essential+for+early+embryogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000519-Peters1" id="pone.0000519-Peters1"></a>Peters AH, O'Carroll D, Scherthan H, Mechtler K, Sauer S, et al.  (2001) Loss of the Suv39h histone methyltransferases impairs mammalian heterochromatin and genome stability. Cell  107: 323–337.  <ul class="find" data-citedArticleID="958032" data-doi="10.1016/s0092-8674(01)00542-6"><li><a href="http://dx.doi.org/10.1016/s0092-8674(01)00542-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Loss+of+the+Suv39h+histone+methyltransferases+impairs+mammalian+heterochromatin+and+genome+stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Loss+of+the+Suv39h+histone+methyltransferases+impairs+mammalian+heterochromatin+and+genome+stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000519-Lei1" id="pone.0000519-Lei1"></a>Lei H, Oh SP, Okano M, Juttermann R, Goss KA, et al.  (1996) De novo DNA cytosine methyltransferase activities in mouse embryonic stem cells. Development  122: 3195–3205.  <ul class="find" data-citedArticleID="958016"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=De+novo+DNA+cytosine+methyltransferase+activities+in+mouse+embryonic+stem+cells.&amp;auth=&amp;atitle=De+novo+DNA+cytosine+methyltransferase+activities+in+mouse+embryonic+stem+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=De+novo+DNA+cytosine+methyltransferase+activities+in+mouse+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22De+novo+DNA+cytosine+methyltransferase+activities+in+mouse+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000519-Kantor1" id="pone.0000519-Kantor1"></a>Kantor B, Makedonski K, Shemer R, Razin A (2003) Expression and localization of components of the histone deacetylases multiprotein repressory complexes in the mouse preimplantation embryo. Gene Expr Patterns  3: 697–702.  <ul class="find" data-citedArticleID="958006" data-doi="10.1016/j.modgep.2003.07.003"><li><a href="http://dx.doi.org/10.1016/j.modgep.2003.07.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+and+localization+of+components+of+the+histone+deacetylases+multiprotein+repressory+complexes+in+the+mouse+preimplantation+embryo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+and+localization+of+components+of+the+histone+deacetylases+multiprotein+repressory+complexes+in+the+mouse+preimplantation+embryo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000519-Wang1" id="pone.0000519-Wang1"></a>Wang HB, Zhang Y (2001) Mi2, an auto-antigen for dermatomyositis, is an ATP-dependent nucleosome remodeling factor. Nucleic Acids Res  29: 2517–2521.  <ul class="find" data-citedArticleID="958042" data-doi="10.1093/nar/29.12.2517"><li><a href="http://dx.doi.org/10.1093/nar/29.12.2517" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mi2%2C+an+auto-antigen+for+dermatomyositis%2C+is+an+ATP-dependent+nucleosome+remodeling+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mi2%2C+an+auto-antigen+for+dermatomyositis%2C+is+an+ATP-dependent+nucleosome+remodeling+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000519-Hendrich2" id="pone.0000519-Hendrich2"></a>Hendrich B, Guy J, Ramsahoye B, Wilson VA, Bird A (2001) Closely related proteins MBD2 and MBD3 play distinctive but interacting roles in mouse development. Genes Dev  15: 710–723.  <ul class="find" data-citedArticleID="957996" data-doi="10.1101/gad.194101"><li><a href="http://dx.doi.org/10.1101/gad.194101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Closely+related+proteins+MBD2+and+MBD3+play+distinctive+but+interacting+roles+in+mouse+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Closely+related+proteins+MBD2+and+MBD3+play+distinctive+but+interacting+roles+in+mouse+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000519-Lagger1" id="pone.0000519-Lagger1"></a>Lagger G, O'Carroll D, Rembold M, Khier H, Tischler J, et al.  (2002) Essential function of histone deacetylase 1 in proliferation control and CDK inhibitor repression. Embo J  21: 2672–2681.  <ul class="find" data-citedArticleID="958012" data-doi="10.1093/emboj/21.11.2672"><li><a href="http://dx.doi.org/10.1093/emboj/21.11.2672" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Essential+function+of+histone+deacetylase+1+in+proliferation+control+and+CDK+inhibitor+repression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Essential+function+of+histone+deacetylase+1+in+proliferation+control+and+CDK+inhibitor+repression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000519-Kaji1" id="pone.0000519-Kaji1"></a>Kaji K, Caballero IM, MacLeod R, Nichols J, Wilson VA, et al.  (2006) The NuRD component Mbd3 is required for pluripotency of embryonic stem cells. Nat Cell Biol  8: 285–292.  <ul class="find" data-citedArticleID="958004" data-doi="10.1038/ncb1372"><li><a href="http://dx.doi.org/10.1038/ncb1372" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+NuRD+component+Mbd3+is+required+for+pluripotency+of+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+NuRD+component+Mbd3+is+required+for+pluripotency+of+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000519-Nan2" id="pone.0000519-Nan2"></a>Nan X, Ng HH, Johnson CA, Laherty CD, Turner BM, et al.  (1998) Transcriptional repression by the methyl-CpG-binding protein MeCP2 involves a histone deacetylase complex. Nature  393: 386–389.  <ul class="find" data-citedArticleID="958028"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transcriptional+repression+by+the+methyl-CpG-binding+protein+MeCP2+involves+a+histone+deacetylase+complex.&amp;auth=&amp;atitle=Transcriptional+repression+by+the+methyl-CpG-binding+protein+MeCP2+involves+a+histone+deacetylase+complex." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+repression+by+the+methyl-CpG-binding+protein+MeCP2+involves+a+histone+deacetylase+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+repression+by+the+methyl-CpG-binding+protein+MeCP2+involves+a+histone+deacetylase+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000519-Guy1" id="pone.0000519-Guy1"></a>Guy J, Hendrich B, Holmes M, Martin JE, Bird A (2001) A mouse Mecp2-null mutation causes neurological symptoms that mimic Rett syndrome. Nat Genet  27: 322–326.  <ul class="find" data-citedArticleID="957992"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+mouse+Mecp2-null+mutation+causes+neurological+symptoms+that+mimic+Rett+syndrome.&amp;auth=&amp;atitle=A+mouse+Mecp2-null+mutation+causes+neurological+symptoms+that+mimic+Rett+syndrome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mouse+Mecp2-null+mutation+causes+neurological+symptoms+that+mimic+Rett+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mouse+Mecp2-null+mutation+causes+neurological+symptoms+that+mimic+Rett+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000519-Hendrich3" id="pone.0000519-Hendrich3"></a>Hendrich B, Tweedie S (2003) The methyl-CpG binding domain and the evolving role of DNA methylation in animals. Trends Genet  19: 269–277.  <ul class="find" data-citedArticleID="957998" data-doi="10.1016/s0168-9525(03)00080-5"><li><a href="http://dx.doi.org/10.1016/s0168-9525(03)00080-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+methyl-CpG+binding+domain+and+the+evolving+role+of+DNA+methylation+in+animals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+methyl-CpG+binding+domain+and+the+evolving+role+of+DNA+methylation+in+animals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000519-Wade1" id="pone.0000519-Wade1"></a>Wade PA, Gegonne A, Jones PL, Ballestar E, Aubry F, et al.  (1999) Mi-2 complex couples DNA methylation to chromatin remodelling and histone deacetylation. Nat Genet  23: 62–66.  <ul class="find" data-citedArticleID="958040" data-doi="10.1038/12664"><li><a href="http://dx.doi.org/10.1038/12664" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mi-2+complex+couples+DNA+methylation+to+chromatin+remodelling+and+histone+deacetylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mi-2+complex+couples+DNA+methylation+to+chromatin+remodelling+and+histone+deacetylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000519-Shalaby1" id="pone.0000519-Shalaby1"></a>Shalaby F, Rossant J, Yamaguchi TP, Gertsenstein M, Wu XF, et al.  (1995) Failure of blood-island formation and vasculogenesis in Flk-1-deficient mice. Nature  376: 62–66.  <ul class="find" data-citedArticleID="958034" data-doi="10.1038/376062a0"><li><a href="http://dx.doi.org/10.1038/376062a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Failure+of+blood-island+formation+and+vasculogenesis+in+Flk-1-deficient+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Failure+of+blood-island+formation+and+vasculogenesis+in+Flk-1-deficient+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000519-Joyner1" id="pone.0000519-Joyner1"></a> Joyner AL, editor.  (2000) Gene Targeting, A Practical Approach. Second Edition. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.XML" value="78633"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.PDF" value="407595"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g001.PNG_L" value="13289"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g001.PNG_M" value="22300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g001.PNG_S" value="1867"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g001.TIF" value="38756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g001.PNG_I" value="11275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g002.PNG_L" value="1013292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g002.PNG_M" value="239970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g002.PNG_S" value="8356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g002.TIF" value="1397372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g002.PNG_I" value="157776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g003.PNG_L" value="762834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g003.PNG_M" value="32581"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g003.PNG_S" value="2435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g003.TIF" value="1399462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g003.PNG_I" value="16028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g004.PNG_L" value="3631624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g004.PNG_M" value="194906"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g004.PNG_S" value="5885"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g004.TIF" value="4131374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g004.PNG_I" value="62907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t001.PNG_L" value="33132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t001.PNG_M" value="32799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t001.PNG_S" value="2255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t001.TIF" value="190122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t001.PNG_I" value="14622"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g005.PNG_L" value="5654341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g005.PNG_M" value="321338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g005.PNG_S" value="8780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g005.TIF" value="6946046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.g005.PNG_I" value="107187"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t002.PNG_L" value="33719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t002.PNG_M" value="32616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t002.PNG_S" value="2231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t002.TIF" value="177216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t002.PNG_I" value="14716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t003.PNG_L" value="145555"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t003.PNG_M" value="114944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t003.PNG_S" value="6830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t003.TIF" value="696394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000519.t003.PNG_I" value="72647"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000519&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000519" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000519&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000519&volume=&issue=&title=Mutants in the Mouse NuRD/Mi2 Component P66α Are Embryonic Lethal&author_name=Susan%20Marino%2C%20Roel%20Nusse&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000519" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000519" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000519" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000519&amp;t=Mutants%20in%20the%20Mouse%20NuRD%2FMi2%20Component%20P66%CE%B1%20Are%20Embryonic%20Lethal" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000519&title=Mutants%20in%20the%20Mouse%20NuRD%2FMi2%20Component%20P66%CE%B1%20Are%20Embryonic%20Lethal&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000519&amp;title=Mutants%20in%20the%20Mouse%20NuRD%2FMi2%20Component%20P66%CE%B1%20Are%20Embryonic%20Lethal" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000519" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000519&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Mutants in the Mouse NuRD/Mi2 Component P66α Are Embryonic Lethal';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000519';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Mutants%20in%20the%20Mouse%20NuRD%2FMi2%20Component%20P66%CE%B1%20Are%20Embryonic%20Lethal http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000519" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000519" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="23876"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="23876"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Embryos%22" title="Search for articles in the subject area:'Embryos'"><div class="flagText">Embryos</div></a>
              <div data-categoryid="17321" data-articleid="23876"
                   data-categoryname="Embryos"
                   class="flagImage" title="Flag 'Embryos' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Homologous+recombination%22" title="Search for articles in the subject area:'Homologous recombination'"><div class="flagText">Homologous recombination</div></a>
              <div data-categoryid="34037" data-articleid="23876"
                   data-categoryname="Homologous recombination"
                   class="flagImage" title="Flag 'Homologous recombination' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="23876"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="23876"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="47289" data-articleid="23876"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Somites%22" title="Search for articles in the subject area:'Somites'"><div class="flagText">Somites</div></a>
              <div data-categoryid="43051" data-articleid="23876"
                   data-categoryname="Somites"
                   class="flagImage" title="Flag 'Somites' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1475'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4553'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4426&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
