

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Identification of Novel Drosophila melanogaster MicroRNAs</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001265"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001265" />

    <meta name="citation_title" content="Identification of Novel Drosophila melanogaster MicroRNAs"/>
    <meta itemprop="name" content="Identification of Novel Drosophila melanogaster MicroRNAs"/>

      <meta name="citation_author" content="Thomas Sandmann"/>
            <meta name="citation_author_institution" content="Temasek Lifesciences Laboratory, Department of Biological Sciences, National University of Singapore, Singapore, Singapore"/>
            <meta name="citation_author_institution" content="European Molecular Biology Laboratory, Heidelberg, Germany"/>
      <meta name="citation_author" content="Stephen M. Cohen"/>
            <meta name="citation_author_institution" content="Temasek Lifesciences Laboratory, Department of Biological Sciences, National University of Singapore, Singapore, Singapore"/>
            <meta name="citation_author_institution" content="European Molecular Biology Laboratory, Heidelberg, Germany"/>

    <meta name="citation_date" content="2007/11/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001265.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1265"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14.; citation_author=R Lee; citation_author=R Feinbaum; citation_author=V Ambros; citation_journal_title=Cell; citation_volume=75; citation_number=1; citation_pages=843-854; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The 21-nucleotide let-7 RNA regulates developmental timing in Caenorhabditis elegans.; citation_author=B Reinhart; citation_author=F Slack; citation_author=M Basson; citation_author=A Pasquinelli; citation_author=J Bettinger; citation_journal_title=Nature; citation_volume=403; citation_number=2; citation_pages=901-906; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Posttranscriptional regulation of the heterochronic gene lin-14 by lin-4 mediates temporal pattern formation in C. elegans.; citation_author=B Wightman; citation_author=I Ha; citation_author=G Ruvkun; citation_journal_title=Cell; citation_volume=75; citation_number=3; citation_pages=855-862; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A complex system of small RNAs in the unicellular green alga Chlamydomonas reinhardtii.; citation_author=T Zhao; citation_author=G Li; citation_author=S Mi; citation_author=S Li; citation_author=GJ Hannon; citation_journal_title=Genes Dev; citation_volume=21; citation_number=4; citation_pages=1190-1203; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=miRBase: microRNA sequences, targets and gene nomenclature.; citation_author=S Griffiths-Jones; citation_author=R Grocock; citation_author=S van Dongen; citation_author=A Bateman; citation_author=A Enright; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=5; citation_pages=D140-144; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic shadowing and computational identification of human microRNA genes.; citation_author=E Berezikov; citation_author=V Guryev; citation_author=J van de Belt; citation_author=E Wienholds; citation_author=R Plasterk; citation_journal_title=Cell; citation_volume=120; citation_number=6; citation_pages=21-24; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Towards a complete description of the microRNA complement of animal genomes.; citation_author=J Brennecke; citation_author=S Cohen; citation_journal_title=Genome Biol; citation_volume=4; citation_number=7; citation_pages=228; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genomics of microRNA.; citation_author=V Kim; citation_author=J Nam; citation_journal_title=Trends Genet; citation_volume=22; citation_number=8; citation_pages=165-173; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A large imprinted microRNA gene cluster at the mouse Dlk1-Gtl2 domain.; citation_author=H Seitz; citation_author=H Royo; citation_author=ML Bortolin; citation_author=SP Lin; citation_author=AC Ferguson-Smith; citation_journal_title=Genome Res; citation_volume=14; citation_number=9; citation_pages=1741-1748; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=microRNA Functions.; citation_author=N Bushati; citation_author=S Cohen; citation_journal_title=Annu Rev Cell Dev Biol; citation_number=10; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Mirtron Pathway Generates microRNA-Class Regulatory RNAs in Drosophila.; citation_author=K Okamura; citation_author=J Hagen; citation_author=H Duan; citation_author=D Tyler; citation_author=E Lai; citation_journal_title=Cell; citation_number=11; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Intronic microRNA precursors that bypass Drosha processing.; citation_author=J Ruby; citation_author=C Jan; citation_author=D Bartel; citation_journal_title=Nature; citation_number=12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Functional siRNAs and miRNAs exhibit strand bias.; citation_author=A Khvorova; citation_author=A Reynolds; citation_author=S Jayasena; citation_journal_title=Cell; citation_volume=115; citation_number=13; citation_pages=209-216; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Asymmetry in the assembly of the RNAi enzyme complex.; citation_author=D Schwarz; citation_author=G Hutvágner; citation_author=T Du; citation_author=Z Xu; citation_author=N Aronin; citation_journal_title=Cell; citation_volume=115; citation_number=14; citation_pages=199-208; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Prediction of mammalian microRNA targets.; citation_author=BP Lewis; citation_author=IH Shih; citation_author=MW Jones-Rhoades; citation_author=DP Bartel; citation_author=CB Burge; citation_journal_title=Cell; citation_volume=115; citation_number=15; citation_pages=787-798; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of Drosophila MicroRNA targets.; citation_author=A Stark; citation_author=J Brennecke; citation_author=RB Russell; citation_author=SM Cohen; citation_journal_title=PLoS Biol; citation_volume=1; citation_number=16; citation_pages=E60; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Principles of microRNA-target recognition.; citation_author=J Brennecke; citation_author=A Stark; citation_author=RB Russell; citation_author=SM Cohen; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=17; citation_pages=e85; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MicroRNA biogenesis: coordinated cropping and dicing.; citation_author=V Kim; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=6; citation_number=18; citation_pages=376-385; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microarray analysis shows that some microRNAs downregulate large numbers of target mRNAs.; citation_author=L Lim; citation_author=N Lau; citation_author=P Garrett-Engele; citation_author=A Grimson; citation_author=J Schelter; citation_journal_title=Nature; citation_volume=433; citation_number=19; citation_pages=769-773; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Maternal microRNAs are essential for mouse zygotic development.; citation_author=F Tang; citation_author=M Kaneda; citation_author=D O'Carroll; citation_author=P Hajkova; citation_author=S Barton; citation_journal_title=Genes Dev; citation_volume=21; citation_number=20; citation_pages=644-648; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A functional study of miR-124 in the developing neural tube.; citation_author=X Cao; citation_author=S Pfaff; citation_author=F Gage; citation_journal_title=Genes Dev; citation_volume=21; citation_number=21; citation_pages=531-536; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Dynamic regulation of miRNA expression in ordered stages of cellular development.; citation_author=J Neilson; citation_author=G Zheng; citation_author=C Burge; citation_author=P Sharp; citation_journal_title=Genes Dev; citation_volume=21; citation_number=22; citation_pages=578-589; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Hippo pathway regulates the bantam microRNA to control cell proliferation and apoptosis in Drosophila.; citation_author=B Thompson; citation_author=S Cohen; citation_journal_title=Cell; citation_volume=126; citation_number=23; citation_pages=767-774; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Drosophila lacking microRNA miR-278 are defective in energy homeostasis.; citation_author=AA Teleman; citation_author=S Maitra; citation_author=SM Cohen; citation_journal_title=Genes Dev; citation_volume=20; citation_number=24; citation_pages=417-422; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A microRNA component of the p53 tumour suppressor network.; citation_author=L He; citation_author=X He; citation_author=L Lim; citation_author=E de Stanchina; citation_author=Z Xuan; citation_journal_title=Nature; citation_number=25; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Rfam: annotating non-coding RNAs in complete genomes.; citation_author=S Griffiths-Jones; citation_author=S Moxon; citation_author=M Marshall; citation_author=A Khanna; citation_author=SR Eddy; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=26; citation_pages=D121-124; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The expansion of the metazoan microRNA repertoire.; citation_author=J Hertel; citation_author=M Lindemeyer; citation_author=K Missal; citation_author=C Fried; citation_author=A Tanzer; citation_journal_title=BMC Genomics; citation_volume=7; citation_number=27; citation_pages=25; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Computational identification of Drosophila microRNA genes.; citation_author=E Lai; citation_author=P Tomancak; citation_author=R Williams; citation_author=G Rubin; citation_journal_title=Genome Biol; citation_volume=4; citation_number=28; citation_pages=R42; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MicroRNA identification based on sequence and structure alignment.; citation_author=X Wang; citation_author=J Zhang; citation_author=F Li; citation_author=J Gu; citation_author=T He; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=29; citation_pages=3610-3614; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mapping of conserved RNA secondary structures predicts thousands of functional noncoding RNAs in the human genome.; citation_author=S Washietl; citation_author=I Hofacker; citation_author=M Lukasser; citation_author=A Hüttenhofer; citation_author=P Stadler; citation_journal_title=Nat Biotechnol; citation_volume=23; citation_number=30; citation_pages=1383-1390; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mapping of conserved RNA secondary structures predicts thousands of functional noncoding RNAs in the human genome.; citation_author=S Washietl; citation_author=IL Hofacker; citation_author=M Lukasser; citation_author=A Huttenhofer; citation_author=PF Stadler; citation_journal_title=Nat Biotechnol; citation_volume=23; citation_number=31; citation_pages=1383-1390; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification and classification of conserved RNA secondary structures in the human genome.; citation_author=J Pedersen; citation_author=G Bejerano; citation_author=A Siepel; citation_author=K Rosenbloom; citation_author=K Lindblad-Toh; citation_journal_title=PLoS Comput Biol; citation_volume=2; citation_number=32; citation_pages=e33; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Drosophila melanogaster genome.; citation_author=SE Celniker; citation_author=GM Rubin; citation_journal_title=Annu Rev Genomics Hum Genet; citation_volume=4; citation_number=33; citation_pages=89-117; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The small RNA profile during Drosophila melanogaster development.; citation_author=AA Aravin; citation_author=M Lagos-Quintana; citation_author=A Yalcin; citation_author=M Zavolan; citation_author=D Marks; citation_journal_title=Dev Cell; citation_volume=5; citation_number=34; citation_pages=337-350; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Discrete small RNA-generating loci as master regulators of transposon activity in Drosophila.; citation_author=J Brennecke; citation_author=A Aravin; citation_author=A Stark; citation_author=M Dus; citation_author=M Kellis; citation_journal_title=Cell; citation_volume=128; citation_number=35; citation_pages=1089-1103; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The UCSC genome browser database: update 2007.; citation_author=RM Kuhn; citation_author=D Karolchik; citation_author=AS Zweig; citation_author=H Trumbower; citation_author=DJ Thomas; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=36; citation_pages=D668-673; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.; citation_author=A Siepel; citation_author=G Bejerano; citation_author=JS Pedersen; citation_author=AS Hinrichs; citation_author=M Hou; citation_journal_title=Genome Res; citation_volume=15; citation_number=37; citation_pages=1034-1050; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=miRNAMap: genomic maps of microRNA genes and their target genes in mammalian genomes.; citation_author=PW Hsu; citation_author=HD Huang; citation_author=SD Hsu; citation_author=LZ Lin; citation_author=AP Tsou; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=38; citation_pages=D135-139; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Secondary structure prediction for aligned RNA sequences.; citation_author=IL Hofacker; citation_author=M Fekete; citation_author=PF Stadler; citation_journal_title=J Mol Biol; citation_volume=319; citation_number=39; citation_pages=1059-1066; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Vienna RNA secondary structure server.; citation_author=IL Hofacker; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=40; citation_pages=3429-3431; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Carbodiimide-mediated cross-linking of RNA to nylon membranes improves the detection of siRNA, miRNA and piRNA by northern blot.; citation_author=G Pall; citation_author=C Codony-Servat; citation_author=J Byrne; citation_author=L Ritchie; citation_author=A Hamilton; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=41; citation_pages=e60; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A greedy algorithm for aligning DNA sequences.; citation_author=Z Zhang; citation_author=S Schwartz; citation_author=L Wagner; citation_author=W Miller; citation_journal_title=J Comput Biol; citation_volume=7; citation_number=42; citation_pages=203-214; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The Jalview Java alignment editor.; citation_author=M Clamp; citation_author=J Cuff; citation_author=SM Searle; citation_author=GJ Barton; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=43; citation_pages=426-427; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001265" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Identification of Novel Drosophila melanogaster MicroRNAs"/>
    <meta name="twitter:description" content="MicroRNAs (miRNAs) are small non-coding RNAs with important regulatory roles in post-transcriptional regulation of metazoan development, homeostasis and disease. The full set of miRNAs is not known for any species and it is believed that many await discovery. The recent assembly of 15 insect genomes has provided the opportunity to identify novel miRNAs in the fruit fly, Drosophila melanogaster. We have performed a computational screen for novel microRNAs in Drosophila melanogaster by searching for phylogenetically conserved putative pre-miRNA structures. The ability of predicted novel miRNA precursors to be processed to produce miRNAs was experimentally verified in S2 cells and in several cases their endogenous expression at was validated by Northern blots. After experimental validation, the predictions were cross-checked with reference to a newly released set of small RNA sequences. Combining both datasets allowed us to identify 53 novel miRNA loci in the fruit fly genome 22 of which we had predicted computationally. This significantly expands the set of known miRNAs in Drosophila melanogaster. Most novel miRNAs contain unique seed sequences not found in other Drosophila miRNAs and are therefore expected to regulate novel sets of target genes. This data provides the basis for future genetic analysis of miRNA function and will aid the discovery of orthologous sequences in other species."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001265.g006"/>

  <meta property="og:title" content="Identification of Novel Drosophila melanogaster MicroRNAs" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001265" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3315'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6513'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4348&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001265">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001265" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001265&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Identification of Novel <i>Drosophila melanogaster</i> MicroRNAs
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas Sandmann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Temasek Lifesciences Laboratory, Department of Biological Sciences, National University of Singapore, Singapore, Singapore, 
                  European Molecular Biology Laboratory, Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen M. Cohen
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:cohen@tll.org.sg">cohen@tll.org.sg</a></p>

                <p>Affiliations:
                  Temasek Lifesciences Laboratory, Department of Biological Sciences, National University of Singapore, Singapore, Singapore, 
                  European Molecular Biology Laboratory, Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001265</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001265" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001265" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001265" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001265" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001265-g001" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001265-g002" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001265-t001" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001265-g003" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001265-g004" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001265-g005" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001265-g006" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001265">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001265" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>MicroRNAs (miRNAs) are small non-coding RNAs with important regulatory roles in post-transcriptional regulation of metazoan development, homeostasis and disease. The full set of miRNAs is not known for any species and it is believed that many await discovery. The recent assembly of 15 insect genomes has provided the opportunity to identify novel miRNAs in the fruit fly, <em>Drosophila melanogaster.</em> We have performed a computational screen for novel microRNAs in <em>Drosophila melanogaster</em> by searching for phylogenetically conserved putative pre-miRNA structures. The ability of predicted novel miRNA precursors to be processed to produce miRNAs was experimentally verified in S2 cells and in several cases their endogenous expression at was validated by Northern blots. After experimental validation, the predictions were cross-checked with reference to a newly released set of small RNA sequences. Combining both datasets allowed us to identify 53 novel miRNA loci in the fruit fly genome 22 of which we had predicted computationally. This significantly expands the set of known miRNAs in <em>Drosophila melanogaster</em>. Most novel miRNAs contain unique seed sequences not found in other <em>Drosophila</em> miRNAs and are therefore expected to regulate novel sets of target genes. This data provides the basis for future genetic analysis of miRNA function and will aid the discovery of orthologous sequences in other species.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Sandmann T, Cohen SM (2007) Identification of Novel <em>Drosophila melanogaster</em> MicroRNAs. PLoS ONE 2(11):
          e1265.
            doi:10.1371/journal.pone.0001265</p><p><strong>Academic Editor: </strong>Cayetano Gonzalez, Institute for Research in Biomedicine, Spain</p><p><strong>Received:</strong> October 31, 2007; <strong>Accepted:</strong> November 10, 2007; <strong>Published:</strong> November 28, 2007</p><p><strong>Copyright:</strong> © 2007 Sandmann, Cohen. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the European Molecular Biology Laboratory, Heidelberg Germany and by the Temasek Life Sciences Laboratory. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>MicroRNAs (miRNAs) are small non-coding RNAs, typically 21–23 nt long, which serve as post-transcriptional regulators of gene expression. The founding members of this class, <em>lin-4</em> and <em>let-7</em> <a href="#pone.0001265-Lee1">[1]</a>–<a href="#pone.0001265-Wightman1">[3]</a>, were discovered using forward genetic screens in <em>C. elegans</em>. Subsequently large numbers of miRNAs have been identified by cloning and sequencing of short RNAs isolated from animals, plants and even the unicellular green <em>alga Chlamydomonas reinhardtii</em> <a href="#pone.0001265-Zhao1">[4]</a>. To date, evidence for 533 human, 442 mouse and 93 <em>Drosophila</em> miRNAs has been deposited in mirBase (version 10.0) <a href="#pone.0001265-GriffithsJones1">[5]</a>. Based on comparison of miRNA predictions and sequence data it has been suggested that these genomes might contain many more, yet undiscovered, miRNA loci, with estimates of ~1000 miRNAs in the human and ~120 in the <em>Drosophila melanogaster</em> genome <a href="#pone.0001265-Berezikov1">[6]</a>, <a href="#pone.0001265-Brennecke1">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>MicroRNA genes have been found as independent transcription units as well as within introns of protein coding genes. More than half of the known loci are part of tandem arrays within operon-like clusters <a href="#pone.0001265-Kim1">[8]</a>, containing up to 40 individual miRNAs <a href="#pone.0001265-Seitz1">[9]</a>. Each primary miRNA transcript (pri-miRNA) contains an extended stem-loop structure, from which a precursor pre-miRNA is released through the action of the RNAseIII enzyme Drosha and the double-stranded RNA-binding domain (dsRBD) protein DGCR8/Pasha (reviewed in <a href="#pone.0001265-Bushati1">[10]</a>). Additionally, a novel mechanism of pre-miRNA generation from very short introns through the splicing process itself has been reported recently <a href="#pone.0001265-Okamura1">[11]</a>, <a href="#pone.0001265-Ruby1">[12]</a>. Animal pre-miRNAs are exported from the nucleus to the cytoplasm and processed further by the Dicer enzyme to yield a characteristic ~22-nt miRNA duplex. The strand with the lower base-pairing energy at its 5′ end is then loaded onto the RNA-induced silencing complex (RISC) and guides it towards its regulatory targets <a href="#pone.0001265-Khvorova1">[13]</a>, <a href="#pone.0001265-Schwarz1">[14]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>miRNAs mediate post-transcriptional inhibition by base-pairing with their cognate target mRNAs. Extended complementarity, as is typically found in plants, leads to cleavage of the target. Limited pairing, including but not limited to the 5′ seed region of the microRNA causes translational inhibition, and in some cases increased RNA turnover <a href="#pone.0001265-Lewis1">[15]</a>–<a href="#pone.0001265-Kim2">[18]</a>. Evidence from microarray studies suggests that single miRNAs can target hundreds of messenger RNAs <a href="#pone.0001265-Lim1">[19]</a> and recent <em>in vivo</em> studies of loss-of-function phenotypes demonstrate the important regulatory role of miRNAs e.g. in development <a href="#pone.0001265-Tang1">[20]</a>–<a href="#pone.0001265-Thompson1">[23]</a>, homeostasis <a href="#pone.0001265-Teleman1">[24]</a> or disease <a href="#pone.0001265-He1">[25]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The majority of miRNAs known to date has been identified through cloning and sequencing of small RNA libraries. Additionally, complementary computational tools for predicting miRNAs have been developed. Two basic strategies have been described to predict novel microRNAs: Sequence similarity searches led to the identification of novel members of previously established miRNA families, many of which are conserved across the animal kingdom <a href="#pone.0001265-GriffithsJones2">[26]</a>, <a href="#pone.0001265-Hertel1">[27]</a>. Other approaches have successfully taken advantage of the well-defined secondary-structure constraints of pre-miRNA sequences as well as evolutionary conservation in related species to detect members of completely novel miRNA families <a href="#pone.0001265-Lai1">[28]</a>, <a href="#pone.0001265-Wang1">[29]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The publication of many newly sequenced and assembled genomes presents us with an unprecedented wealth of information about evolutionary conservation of coding and non-coding sequences. Based on the observation that the evolution of many non-coding RNAs (ncRNAs) is constrained by functionally important secondary structure elements, novel approaches have been developed to scan whole metazoan genomes for putative ncRNAs. The RNAz program employs a support vector machine (SVM) to evaluate both thermodynamic stability of candidate sequences as well as the conservation of the predicted secondary structure <a href="#pone.0001265-Washietl1">[30]</a>, <a href="#pone.0001265-Washietl2">[31]</a>. Evofold uses stochastic context free grammars (SCFG) to identify conservation patterns indicating folded RNA structures <a href="#pone.0001265-Pedersen1">[32]</a>. Both programs are not limited to any particular class of ncRNA and have been used to predict several tens of thousands of putative ncRNAs in the human genome. To identify novel miRNAs in <em>Drosophila melanogaster</em>, we have combined a computational screen for generic ncRNAs using RNAz with stringent filtering criteria specifically targeting miRNA precursors. We have validated 22 of the newly predicted loci experimentally. 31 additional loci were identified from recently published small RNA sequence libraries.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>The fruit fly <em>Drosophila melanogaster</em> is a powerful model system to study metazoan development. Its compact genome is well annotated, owing to many years of genetic experimentation <a href="#pone.0001265-Celniker1">[33]</a>, and has been mined extensively for coding as well as non-coding gene loci <a href="#pone.0001265-Lai1">[28]</a>, <a href="#pone.0001265-Aravin1">[34]</a>, <a href="#pone.0001265-Brennecke3">[35]</a>. With the assemblies of a range of additional insect genomes, information about twelve <em>Drosophila</em> species, the honeybee (<em>A. mellifera</em>), the flour beetle (<em>T. castaneum</em>) and the mosquito (<em>A. gambiae</em>) has recently become available. To identify novel microRNAs, we took advantage of the whole-genome alignments of 9 and 15 insect species provided by the UCSC genome browser <a href="#pone.0001265-Kuhn1">[36]</a>. Multiple sequence alignments for all “most conserved” regions, as identified by phastcons analysis <a href="#pone.0001265-Siepel1">[37]</a>, were extracted and scanned with RNAz for putative conserved non-coding sequences and several thousand candidate regions were recovered at a p-value cutoff of &gt;0.9 (4003 with multiz9 and 17727 with multiz15 input alignments). At this threshold, 51 of the 65 previously known Drosophila miRNAs covered by the multiz9 and 51 of the 76 covered by the multiz15 input alignments were recovered (37/51 miRNAs were detected in both inputs).</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>To identify novel miRNA precursor structures, we applied a stringent filtering strategy <a href="#pone.0001265-Hsu1">[38]</a> to the RNAz output, selecting only RNAz predictions with i) a stem-loop structure exceeding 20 nt, and ii) high predicted thermodynamic stability, as indicated by a z-score of less than −3.5. To enrich further for true miRNA precursors, we selected sequences displaying the bimodal conservation pattern typical of microRNAs <a href="#pone.0001265-Berezikov1">[6]</a> by visually inspecting the “conservation” track in the UCSC genome browser (<a href="#pone-0001265-g001">Figure 1</a>). As a number of the known miRNA loci in the <em>Drosophila melanogaster</em> genome show high conservation across the full sequence of the pre-miRNA, we also included regions of constantly high conservation (<a href="#pone.0001265.s001">Data S1</a>, e.g. locus19, manual3). This strategy yielded 47 novel miRNA candidates, an example prediction is shown in <a href="#pone-0001265-g001">Figure 1</a>.</p>
<div class="figure" id="pone-0001265-g001"><div class="img"><a name="pone-0001265-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Example of a pre-miRNA predicted by RNAz.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>A) Locus1 shows the bimodal conservation pattern typical of a conserved miRNA hairpin structure in the phastcons track of the UCSC genome browser. Sequences from Drosophilids as distant as <em>D. mojavensis</em> contribute to the multiz alignment at this genomic position. B) Sequences from six species were chosen as input for RNAz and lend different levels of support to stabilizing selection of the predicted secondary structure prediction (first row). The color code indicates the number of different base-pairs (green = 3 pairs to red = 1 pair) and the number of pair incompatible with the predicted structure (dark color = 0 to faint color = 2) at each position. C, D) Locus1 is predicted to fold into symmetrical hairpins in both possible directions of transcription (color code as in B).</p>
<span>doi:10.1371/journal.pone.0001265.g001</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>In addition to the RNAz predicted stem-loop structures, we noticed the presence of several additional bimodally-conserved sequences close to some of the identified candidates. As many known miRNAs are found in clusters in animal genomes, we tested these sequences for their potential to form stable hairpin structures using RNAalifold <a href="#pone.0001265-Hofacker1">[39]</a>. In this way, nine additional sequences were added as “manual” predictions to the list of candidates, giving rise to several predicted clusters of miRNAs (<a href="#pone-0001265-g002">Figure 2</a> and <a href="#pone.0001265.s001">Data S1</a>). In one of them, intriguingly miRNAs are predicted in two groups flanking an exon of the protein-coding gene CG31646 (<a href="#pone-0001265-g002">Figure 2C</a>, cluster 3).</p>
<div class="figure" id="pone-0001265-g002"><div class="img"><a name="pone-0001265-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Prediction of four novel miRNA clusters.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>A) locus1, manual8, manual9 form a tight cluster downstream of the <em>Grip84</em> locus (green). Small RNAs detected by sequencing map to each of the novel miRNA loci (matches 50–52, blue). In addition, sequences map to two loci in the vicinity, giving rise to identical mature miRNAs (matches 48,49). Finally, a sixth locus is detected immediately downstream of <em>Grip84</em> (match 53). B) <em>mir-318</em>, located downstream of the <em>Irp1B</em> locus, clusters together with novel locus41 (green and small RNA sequence match 28, blue) C) Five novel miRNAs were predicted (green) and validated (blue) on either side of an exon of the CG31646 locus. D) Novel locus4 and manual5 (green) give rise to mature miRNAs (blue) and form cluster4.</p>
<span>doi:10.1371/journal.pone.0001265.g002</span></div>
<h4>Experimental validation of predictions</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Like protein-coding genes, miRNAs are often expressed only at specific developmental stages or in a subset of tissues. To test if mature microRNAs can be processed from primary transcripts originating from the predicted genomic sequences in a single experimental system, we cloned the genomic regions covering a number of putative novel miRNA loci and expressed them under them control of the actin 5C promoter by transfection into <em>Drosophila</em> S2 cells. The majority of candidate sequences had significant RNAz scores (&gt;0.9) in both reading directions (53/56). The predicted hairpins were often symmetrical. These candidates were therefore expressed in both orientations to direct production of primary transcripts from both strands. Processing of these long primary transcripts was assayed by Northern blotting, and detected by hybridization with oligonucleotide probes directed against each arm of the predicted hairpin. Out of 33 candidates tested, 16 gave rise to a distinct small RNA (<a href="#pone-0001265-t001">Table 1</a>). Five of these proved to be endogenously expressed in S2 cells and could therefore be detected in untransfected cells.</p>
<div class="figure" id="pone-0001265-t001"><div class="img"><a name="pone-0001265-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001265.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001265.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Validation of predicted miRNAs</span></strong></p><span>doi:10.1371/journal.pone.0001265.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>While this experimental analysis was nearing completion, high-throughput sequencing data was published for small RNA libraries cloned from ten different samples, including the major developmental stages of <em>Drosophila melanogaster</em> development <a href="#pone.0001265-Ruby1">[12]</a> [GEO:GSE7448]. This allowed us to compare our experimental data with an independent unbiased global expression analysis. We mapped these small RNAs to the non-repetitive part of the <em>Drosophila</em> genome and used RNAfold <a href="#pone.0001265-Hofacker2">[40]</a> to screen for sequenced RNAs falling into regions predicted to fold into an extended hairpin structure. This way, 53 novel miRNA loci were identified. 19 of these corresponded to one of our 56 microRNA predictions (<a href="#pone-0001265-t001">Table 1</a>). 13 of these were miRNAs independently validated by our Northern blot analysis. In each case small RNA sequences mapped only to a single strand, corresponding to the previously identified direction of transcription. In agreement with the stepwise processing of pre-miRNA precursors, 21–25 long oligonucleotides often originated from both arms of the hairpin, albeit at different frequencies: the mature miRNA outnumbers its cognate miRNA* sequence with a ratio up to 1713:3 (locus12, <a href="#pone-0001265-g003">Figure 3A</a>).</p>
<div class="figure" id="pone-0001265-g003"><div class="img"><a name="pone-0001265-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Examples of small RNA sequences mapping to predicted miRNA loci.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Small RNA sequences recovered by sequencing <a href="#pone.0001265-Ruby1">[12]</a> were mapped to the predicted loci using Megablast. A) locus12 is transcribed from the ‘–’ strand and multiple overlapping sequences map to its coordinates. Numbers on the left indicate how often each sequence was identified in the small RNA libraries. The most abundant species (red box) most likely represents the mature miRNA. B) locus3 maps to the ‘+’ strand. Details for all predicted loci is available as supplemental data.</p>
<span>doi:10.1371/journal.pone.0001265.g003</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We have used available sequence data from ten different anatomies or developmental stages to validate and complement our predictions, pushing the total number of annotated miRNAs to 146 - beyond the previous expectation of ~120. Although there is considerable overlap between the prediction/validation and sequence data sets, comparing them showed that neither set is complete. 19/53 of the novel candidates (38.2%) were identified both computationally and by sequencing. In addition, the small RNA libraries provided evidence for 31 additional loci that we had not predicted. Three predicted miRNAs (locus26, locus7 and manual3) could be detected in untransfected S2 cells and in developing embryos by Northern blot analysis (<a href="#pone-0001265-g004">Figure 4</a>), but were not found in the sequencing data. Locus7 and manual3 are expressed strongly in early stages and decline in abundance toward the end of embryogenesis, whereas locus26 expression begins low and increases during mid- to late embryogenesis (<a href="#pone-0001265-g004">Figure 4E–G</a>). In aggregate, this illustrates that any single approach aimed at identifying miRNAs continues to underestimate the miRNA complement of the <em>Drosophila melanogaster</em> genome.</p>
<div class="figure" id="pone-0001265-g004"><div class="img"><a name="pone-0001265-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Northern blot validation of three miRNAs not found in small RNA libraries.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Three predicted miRNAs were not represented in the small RNA libraries, yet could be detected by Northern blot analysis. A–C) Endogenous small RNAs were detected with probes directed against locus7, locus25 and manual3 in total RNA extracted from untransfected S2 cells. D–F) All three transcripts were also detected in RNA from different embryonic stages (time after egg-laying). F) A probe against valine tRNA was used as a loading control.</p>
<span>doi:10.1371/journal.pone.0001265.g004</span></div>

<h4>Mature miRNA sequences</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>By identifying the small RNA sequences detected most frequently in the small RNA libraries, we could pinpoint the mature microRNAs. Interestingly, eight of the novel miRNAs share identical seed sequences with known miRNAs (<a href="#pone.0001265.s002">Table S1</a>). The first eight bases of the mature manual7 sequence correspond to that of another <em>Drosophila melanogaster</em> miRNA, <em>mir-12</em>, suggesting that both miRNAs could regulate a similar set of targets (<a href="#pone-0001265-g005">Figure 5A</a>). While <em>mir-12</em> was detected in samples from all developmental stages/tissues with the exception of the early embryo (<a href="#pone-0001265-g005">Figure 5B</a>), mature manual7 is highly enriched in imaginal disks (80.9% of all sequence hits, <a href="#pone-0001265-g005">Figure 5C</a>), where it might tighten the repression of the common set of targets specifically at this stage of development. Similarly, “seed paralogs” were found for <em>Drosophila mir-279, mir-285</em> and <em>mir-286</em>. Also, several of the newly identified miRNAs have unique seeds in this model organism, but similar seeds are found in miRNAs of other species, including vertebrates (<a href="#pone.0001265.s002">Table S1</a>).</p>
<div class="figure" id="pone-0001265-g005"><div class="img"><a name="pone-0001265-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>miRNAs sharing seed sequences.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A) The novel miRNA manual7 contains the same seed sequence as the D. melanogaster <em>mir-12</em> miRNA and its orthologs. B) Sequence similarity between the novel locus11 and vertebrate <em>miR-22</em> extends beyond the seed region, hinting at a possible common ancestry of these miRNAs. C) The mature sequence of <em>Drosophila mir-12</em> was detected more than 1400 times in the small RNA libraries in samples from all developmental stages/tissues with the exception of the early embryo. D) The most abundant sequence mapping to the novel manual7 locus is distributed differently over the 10 sequenced samples, with RNA from discs contributing &gt;80% of hits.</p>
<span>doi:10.1371/journal.pone.0001265.g005</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In some cases, the alignment between novel and previously described miRNAs extends well beyond the seed sequences, suggesting a common evolutionary ancestry. Locus11 shares e.g. 14 out of 22 bases with vertebrate <em>mir-22</em>, a microRNA highly conserved in vertebrates but until now lacking a <em>Drosophila</em> ortholog (<a href="#pone-0001265-g005">Figure 5D</a>, <a href="#pone.0001265.s002">Table S1</a>).</p>


<h4>Clusters</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Several of the predicted miRNAs were found in distinct experimentally validated genomic clusters (<a href="#pone-0001265-g002">Figure 2</a>). To investigate whether miRNAs from each genomic cluster are likely to be processed from the same primary transcript, we compared the relative frequencies of the mature miRNA from these validated clusters in different RNA libraries (<a href="#pone-0001265-g006">Figure 6B–E</a>). Although the absolute number of sequence hits varies between the different members of the same cluster (e.g. 19 hits for locus1 and 162 hits for manual9 in cluster1), their relative abundance in different sequenced tissues is remarkably similar. Comparable results were obtained with a known miRNA cluster (<em>mir-310, mir-311, mir-312 and mir-313</em>, <a href="#pone-0001265-g006">Figure 6A</a>), suggesting that the novel clusters may also encode polycistronic transcripts. While the miRNAs of cluster1, cluster2 and cluster3 seem to be unrelated in sequence, cluster4 gives rise to two very similar mature sequences (<a href="#pone-0001265-g006">Figure 6F</a>). Manual5 carries an additional U at its 5′ end compared to locus4, giving rise to a novel putative seed sequence. As both mature sequences clearly outnumber any other matching oligonucleotides mapped to the same locus (<a href="#pone-0001265-t001">Table 1</a>), locus4 and manual5 most likely represent distinct, recently diverged microRNAs.</p>
<div class="figure" id="pone-0001265-g006"><div class="img"><a name="pone-0001265-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001265" data-uri="info:doi/10.1371/journal.pone.0001265.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001265.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001265.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001265.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001265.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Four novel clusters encoding tightly co-expressed miRNAs.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Though the individual number of sequence hits recovered in the small RNA libraries differs greatly between different miRNAs, the relative abundance in the different sequenced samples is highly correlated for miRNAs originating from the same cluster. A) Similar fractions of the four members of the Drosophila <em>miR-310-313</em> cluster are recovered in embryonic, imaginal disc and adult body samples, but virtually absent from the other samples. B–E) Members of the four novel miRNA clusters are distributed equally tightly over the small RNA libraries, demonstrating tight co-regulation. F) The mature sequences of locus4 and manual5 are highly similar and map to neighboring genomic coordinates (E), suggesting a duplication event that gave rise to two independent loci.</p>
<span>doi:10.1371/journal.pone.0001265.g006</span></div>

<h4>Conclusions</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>By combining RNAz, a “generic” tool to predict non-coding RNAs, with specific filters to identify miRNA precursors, we have taken advantage of the recent sequencing of a large number of insect species. The fact that 22 out of our 56 predictions (39%) could be verified by Northern blot analysis or sequencing, but were missed in previous studies, demonstrates the power of this approach.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The degree of sequence conservation of miRNA precursors between related species varies widely (compare e.g. the phastcons tracks for manual5 and locus48, <a href="#pone.0001265.s001">Data S1</a>). The genome-wide alignments of 9 or 15 insects enabled us to select suitable sequences for each candidate region separately, allowing for sequence variation of rapidly as well as slowly diverging miRNAs. This approach is likely to prove useful for the analysis of other animal genomes, as more related genomes are sequenced.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Computational miRNA predictions</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Genome-wide alignments of 9 (multiz9) and 15 (multiz15) insect species were downloaded from the UCSC genome browser. Starting from the “most conserved” track for each alignment generated by the Phastcons program, we padded the genomic coordinates by 25 bps on both sides. Neighboring sequences closer than 50 bps to each other were combined into a single region. All sequences overlapping annotated genes according to the “Flybase genes” and “Refseq genes” tracks of the UCSC genome browser were removed.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Input windows were prepared with the rnazWindow.pl script of the RNAz package with default settings. This resulted in alignments with a maximum length of 200 bps (longer alignments were split into overlapping windows of 120 bps with a step size of 40) and a minimum number of aligned sequences (&gt;50% identity) from at least 4 different species. RNAz (version 1.0) was used to score the alignments and hits with a probability score p&gt;0.9 were stored. Overlapping predictions were joined using the rnazCluster.pl script of the RNAz package and filtered for hairpin length (&gt;20 bps, total length of the prediction &gt;65 bps), thermodynamic stability (z-score &lt;−3.5). The remaining predictions were uploaded to a local installation of the UCSC genome browser and their multiz15 phastcons conservation profile inspected manually.</p>


<h4>Cloning and expression of putative miRNAs in S2 cells</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Amplicons of ca. 0.5 kb length, centered on the predicted pre-miRNA were generated by PCR and cloned into vector pAc5.1B (Invitrogen). S2 cells were grown in the absence of serum, transfected using Cellfectin (Invitrogen) and incubated for 3 days at 25°C. Total RNA was extracted from transfected or control cells, as well as from staged collections of <em>Drosophila</em> embryos using Trizol (Invitrogen) and precipitated with isopropoanol overnight at −20°C. 15 µg of total RNA was separated on a 15% denaturing polyacrylamide gel and blotted using a chemical cross-linking procedure <a href="#pone.0001265-Pall1">[41]</a>. Oligonucleotide probes were designed separately against each full arms of the predicted hairpin and end-labeled with <sup>32</sup>P-ATP using T4 polynucleotide kinase (NEB).</p>


<h4>Mapping small RNA sequencing data to predicted loci</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Sequences detected in Drosophila small RNA libraries <a href="#pone.0001265-Ruby1">[12]</a> were obtained from GEO [GEO:GSE7448] and mapped to the putative pre-miRNA sequences using megablast <a href="#pone.0001265-Zhang1">[42]</a> with wordsize 14 and a score cutoff of 16. Only perfectly matched sequence matches were retained. The resulting alignments were displayed with Jalview <a href="#pone.0001265-Clamp1">[43]</a>.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>miRNA loci were identified from the small RNA libraries <em>de novo</em> by moving a 110 bp window (stepsize 5) over all bases recovered at least 10 times. RNAfold <a href="#pone.0001265-Hofacker2">[40]</a> was used to predict the secondary structure of each sequence window and hairpins with more than 30 base-paired nucleotides and bulges of less than 15 nucleotides were retained. Repetitive regions or sites overlapping with annotated loci were removed.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001265"></div><a name="pone.0001265.s001" id="pone.0001265.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001265.s001">Data S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Pdf document containing information about all validated novel miRNAs, including RNAz alignment and structure models.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001265.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(2.17 MB PDF)</p>
<a name="pone.0001265.s002" id="pone.0001265.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001265.s002">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Genome data on 53 new miRNAs. Information about all 53 validated novel miRNAs, including their genomic coordinates (genome version 5) and their mature sequences.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001265.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Natascha Bushati, Lars J. Jensen, Charles Girardot and Julian Gagneur (EMBL Heidelberg, Germany) for helpful discussion and valuable advice. Thomas Sandmann is a Fellow of the Singapore Millennium Foundation.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SC TS. Performed the experiments: TS. Analyzed the data: TS. Wrote the paper: SC TS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001265-Lee1" id="pone.0001265-Lee1"></a>Lee R, Feinbaum R, Ambros V (1993) The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14. Cell  75: 843–854.  <ul class="find" data-citedArticleID="1026038" data-doi="10.1016/0092-8674(93)90529-y"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)90529-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+C.+elegans+heterochronic+gene+lin-4+encodes+small+RNAs+with+antisense+complementarity+to+lin-14." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+C.+elegans+heterochronic+gene+lin-4+encodes+small+RNAs+with+antisense+complementarity+to+lin-14.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001265-Reinhart1" id="pone.0001265-Reinhart1"></a>Reinhart B, Slack F, Basson M, Pasquinelli A, Bettinger J, et al.  (2000) The 21-nucleotide let-7 RNA regulates developmental timing in Caenorhabditis elegans. Nature  403: 901–906.  <ul class="find" data-citedArticleID="1026052"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+21-nucleotide+let-7+RNA+regulates+developmental+timing+in+Caenorhabditis+elegans.&amp;auth=&amp;atitle=The+21-nucleotide+let-7+RNA+regulates+developmental+timing+in+Caenorhabditis+elegans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+21-nucleotide+let-7+RNA+regulates+developmental+timing+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+21-nucleotide+let-7+RNA+regulates+developmental+timing+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001265-Wightman1" id="pone.0001265-Wightman1"></a>Wightman B, Ha I, Ruvkun G (1993) Posttranscriptional regulation of the heterochronic gene lin-14 by lin-4 mediates temporal pattern formation in C. elegans. Cell  75: 855–862.  <ul class="find" data-citedArticleID="1026076" data-doi="10.1016/0092-8674(93)90530-4"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)90530-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Posttranscriptional+regulation+of+the+heterochronic+gene+lin-14+by+lin-4+mediates+temporal+pattern+formation+in+C.+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Posttranscriptional+regulation+of+the+heterochronic+gene+lin-14+by+lin-4+mediates+temporal+pattern+formation+in+C.+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001265-Zhao1" id="pone.0001265-Zhao1"></a>Zhao T, Li G, Mi S, Li S, Hannon GJ, et al.  (2007) A complex system of small RNAs in the unicellular green alga Chlamydomonas reinhardtii. Genes Dev  21: 1190–1203.  <ul class="find" data-citedArticleID="1026080" data-doi="10.1101/gad.1543507"><li><a href="http://dx.doi.org/10.1101/gad.1543507" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+complex+system+of+small+RNAs+in+the+unicellular+green+alga+Chlamydomonas+reinhardtii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+complex+system+of+small+RNAs+in+the+unicellular+green+alga+Chlamydomonas+reinhardtii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001265-GriffithsJones1" id="pone.0001265-GriffithsJones1"></a>Griffiths-Jones S, Grocock R, van Dongen S, Bateman A, Enright A (2006) miRBase: microRNA sequences, targets and gene nomenclature. Nucleic Acids Res  34: D140–144.  <ul class="find" data-citedArticleID="1026014" data-doi="10.1093/nar/gkj112"><li><a href="http://dx.doi.org/10.1093/nar/gkj112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=miRBase%3A+microRNA+sequences%2C+targets+and+gene+nomenclature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22miRBase%3A+microRNA+sequences%2C+targets+and+gene+nomenclature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001265-Berezikov1" id="pone.0001265-Berezikov1"></a>Berezikov E, Guryev V, van de Belt J, Wienholds E, Plasterk R, et al.  (2005) Phylogenetic shadowing and computational identification of human microRNA genes. Cell  120: 21–24.  <ul class="find" data-citedArticleID="1025998" data-doi="10.1016/j.cell.2004.12.031"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.12.031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+shadowing+and+computational+identification+of+human+microRNA+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+shadowing+and+computational+identification+of+human+microRNA+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001265-Brennecke1" id="pone.0001265-Brennecke1"></a>Brennecke J, Cohen S (2003) Towards a complete description of the microRNA complement of animal genomes. Genome Biol  4: 228.  <ul class="find" data-citedArticleID="1026000"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Towards+a+complete+description+of+the+microRNA+complement+of+animal+genomes.&amp;auth=&amp;atitle=Towards+a+complete+description+of+the+microRNA+complement+of+animal+genomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+a+complete+description+of+the+microRNA+complement+of+animal+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+a+complete+description+of+the+microRNA+complement+of+animal+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001265-Kim1" id="pone.0001265-Kim1"></a>Kim V, Nam J (2006) Genomics of microRNA. Trends Genet  22: 165–173.  <ul class="find" data-citedArticleID="1026030" data-doi="10.1016/j.tig.2006.01.003"><li><a href="http://dx.doi.org/10.1016/j.tig.2006.01.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomics+of+microRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomics+of+microRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001265-Seitz1" id="pone.0001265-Seitz1"></a>Seitz H, Royo H, Bortolin ML, Lin SP, Ferguson-Smith AC, et al.  (2004) A large imprinted microRNA gene cluster at the mouse Dlk1-Gtl2 domain. Genome Res  14: 1741–1748.  <ul class="find" data-citedArticleID="1026058" data-doi="10.1101/gr.2743304"><li><a href="http://dx.doi.org/10.1101/gr.2743304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+large+imprinted+microRNA+gene+cluster+at+the+mouse+Dlk1-Gtl2+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+large+imprinted+microRNA+gene+cluster+at+the+mouse+Dlk1-Gtl2+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001265-Bushati1" id="pone.0001265-Bushati1"></a>Bushati N, Cohen S (2007) microRNA Functions. Annu Rev Cell Dev Biol.   <ul class="find" data-citedArticleID="1026006" data-doi="10.1146/annurev.cellbio.23.090506.123406"><li><a href="http://dx.doi.org/10.1146/annurev.cellbio.23.090506.123406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=microRNA+Functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22microRNA+Functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001265-Okamura1" id="pone.0001265-Okamura1"></a>Okamura K, Hagen J, Duan H, Tyler D, Lai E (2007) The Mirtron Pathway Generates microRNA-Class Regulatory RNAs in Drosophila. Cell.   <ul class="find" data-citedArticleID="1026046" data-doi="10.1016/j.cell.2007.06.028"><li><a href="http://dx.doi.org/10.1016/j.cell.2007.06.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Mirtron+Pathway+Generates+microRNA-Class+Regulatory+RNAs+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Mirtron+Pathway+Generates+microRNA-Class+Regulatory+RNAs+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001265-Ruby1" id="pone.0001265-Ruby1"></a>Ruby J, Jan C, Bartel D (2007) Intronic microRNA precursors that bypass Drosha processing. Nature.   <ul class="find" data-citedArticleID="1026054" data-doi="10.1038/nature05983"><li><a href="http://dx.doi.org/10.1038/nature05983" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intronic+microRNA+precursors+that+bypass+Drosha+processing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intronic+microRNA+precursors+that+bypass+Drosha+processing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001265-Khvorova1" id="pone.0001265-Khvorova1"></a>Khvorova A, Reynolds A, Jayasena S (2003) Functional siRNAs and miRNAs exhibit strand bias. Cell  115: 209–216.  <ul class="find" data-citedArticleID="1026028" data-doi="10.1016/s0092-8674(03)00801-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)00801-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+siRNAs+and+miRNAs+exhibit+strand+bias." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+siRNAs+and+miRNAs+exhibit+strand+bias.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001265-Schwarz1" id="pone.0001265-Schwarz1"></a>Schwarz D, Hutvágner G, Du T, Xu Z, Aronin N, et al.  (2003) Asymmetry in the assembly of the RNAi enzyme complex. Cell  115: 199–208.  <ul class="find" data-citedArticleID="1026056" data-doi="10.1016/s0092-8674(03)00759-1"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)00759-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asymmetry+in+the+assembly+of+the+RNAi+enzyme+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Asymmetry+in+the+assembly+of+the+RNAi+enzyme+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001265-Lewis1" id="pone.0001265-Lewis1"></a>Lewis BP, Shih IH, Jones-Rhoades MW, Bartel DP, Burge CB (2003) Prediction of mammalian microRNA targets. Cell  115: 787–798.  <ul class="find" data-citedArticleID="1026040" data-doi="10.1016/s0092-8674(03)01018-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)01018-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+mammalian+microRNA+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+mammalian+microRNA+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001265-Stark1" id="pone.0001265-Stark1"></a>Stark A, Brennecke J, Russell RB, Cohen SM (2003) Identification of Drosophila MicroRNA targets. PLoS Biol  1: E60.  <ul class="find" data-citedArticleID="1026062" data-doi="10.1371/journal.pbio.0000060"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0000060" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Drosophila+MicroRNA+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Drosophila+MicroRNA+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001265-Brennecke2" id="pone.0001265-Brennecke2"></a>Brennecke J, Stark A, Russell RB, Cohen SM (2005) Principles of microRNA-target recognition. PLoS Biol  3: e85.  <ul class="find" data-citedArticleID="1026002" data-doi="10.1371/journal.pbio.0030085"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Principles+of+microRNA-target+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Principles+of+microRNA-target+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001265-Kim2" id="pone.0001265-Kim2"></a>Kim V (2005) MicroRNA biogenesis: coordinated cropping and dicing. Nat Rev Mol Cell Biol  6: 376–385.  <ul class="find" data-citedArticleID="1026032" data-doi="10.1038/nrm1644"><li><a href="http://dx.doi.org/10.1038/nrm1644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+biogenesis%3A+coordinated+cropping+and+dicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+biogenesis%3A+coordinated+cropping+and+dicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001265-Lim1" id="pone.0001265-Lim1"></a>Lim L, Lau N, Garrett-Engele P, Grimson A, Schelter J, et al.  (2005) Microarray analysis shows that some microRNAs downregulate large numbers of target mRNAs. Nature  433: 769–773.  <ul class="find" data-citedArticleID="1026042" data-doi="10.1038/nature03315"><li><a href="http://dx.doi.org/10.1038/nature03315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+analysis+shows+that+some+microRNAs+downregulate+large+numbers+of+target+mRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+analysis+shows+that+some+microRNAs+downregulate+large+numbers+of+target+mRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001265-Tang1" id="pone.0001265-Tang1"></a>Tang F, Kaneda M, O'Carroll D, Hajkova P, Barton S, et al.  (2007) Maternal microRNAs are essential for mouse zygotic development. Genes Dev  21: 644–648.  <ul class="find" data-citedArticleID="1026064" data-doi="10.1101/gad.418707"><li><a href="http://dx.doi.org/10.1101/gad.418707" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maternal+microRNAs+are+essential+for+mouse+zygotic+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maternal+microRNAs+are+essential+for+mouse+zygotic+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001265-Cao1" id="pone.0001265-Cao1"></a>Cao X, Pfaff S, Gage F (2007) A functional study of miR-124 in the developing neural tube. Genes Dev  21: 531–536.  <ul class="find" data-citedArticleID="1026008" data-doi="10.1101/gad.1519207"><li><a href="http://dx.doi.org/10.1101/gad.1519207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+functional+study+of+miR-124+in+the+developing+neural+tube." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+functional+study+of+miR-124+in+the+developing+neural+tube.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001265-Neilson1" id="pone.0001265-Neilson1"></a>Neilson J, Zheng G, Burge C, Sharp P (2007) Dynamic regulation of miRNA expression in ordered stages of cellular development. Genes Dev  21: 578–589.  <ul class="find" data-citedArticleID="1026044" data-doi="10.1101/gad.1522907"><li><a href="http://dx.doi.org/10.1101/gad.1522907" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+regulation+of+miRNA+expression+in+ordered+stages+of+cellular+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+regulation+of+miRNA+expression+in+ordered+stages+of+cellular+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001265-Thompson1" id="pone.0001265-Thompson1"></a>Thompson B, Cohen S (2006) The Hippo pathway regulates the bantam microRNA to control cell proliferation and apoptosis in Drosophila. Cell  126: 767–774.  <ul class="find" data-citedArticleID="1026068" data-doi="10.1016/j.cell.2006.07.013"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.07.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Hippo+pathway+regulates+the+bantam+microRNA+to+control+cell+proliferation+and+apoptosis+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Hippo+pathway+regulates+the+bantam+microRNA+to+control+cell+proliferation+and+apoptosis+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001265-Teleman1" id="pone.0001265-Teleman1"></a>Teleman AA, Maitra S, Cohen SM (2006) Drosophila lacking microRNA miR-278 are defective in energy homeostasis. Genes Dev  20: 417–422.  <ul class="find" data-citedArticleID="1026066" data-doi="10.1101/gad.374406"><li><a href="http://dx.doi.org/10.1101/gad.374406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Drosophila+lacking+microRNA+miR-278+are+defective+in+energy+homeostasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Drosophila+lacking+microRNA+miR-278+are+defective+in+energy+homeostasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001265-He1" id="pone.0001265-He1"></a>He L, He X, Lim L, de Stanchina E, Xuan Z, et al.  (2007) A microRNA component of the p53 tumour suppressor network. Nature.   <ul class="find" data-citedArticleID="1026018" data-doi="10.1038/nature05939"><li><a href="http://dx.doi.org/10.1038/nature05939" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+microRNA+component+of+the+p53+tumour+suppressor+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+microRNA+component+of+the+p53+tumour+suppressor+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001265-GriffithsJones2" id="pone.0001265-GriffithsJones2"></a>Griffiths-Jones S, Moxon S, Marshall M, Khanna A, Eddy SR, et al.  (2005) Rfam: annotating non-coding RNAs in complete genomes. Nucleic Acids Res  33: D121–124.  <ul class="find" data-citedArticleID="1026016" data-doi="10.1093/nar/gki081"><li><a href="http://dx.doi.org/10.1093/nar/gki081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rfam%3A+annotating+non-coding+RNAs+in+complete+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rfam%3A+annotating+non-coding+RNAs+in+complete+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001265-Hertel1" id="pone.0001265-Hertel1"></a>Hertel J, Lindemeyer M, Missal K, Fried C, Tanzer A, et al.  (2006) The expansion of the metazoan microRNA repertoire. BMC Genomics  7: 25.  <ul class="find" data-citedArticleID="1026020" data-doi="10.1186/1471-2164-7-25"><li><a href="http://dx.doi.org/10.1186/1471-2164-7-25" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+expansion+of+the+metazoan+microRNA+repertoire." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+expansion+of+the+metazoan+microRNA+repertoire.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001265-Lai1" id="pone.0001265-Lai1"></a>Lai E, Tomancak P, Williams R, Rubin G (2003) Computational identification of Drosophila microRNA genes. Genome Biol  4: R42.  <ul class="find" data-citedArticleID="1026036" data-doi="10.1186/gb-2003-4-7-r42"><li><a href="http://dx.doi.org/10.1186/gb-2003-4-7-r42" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+identification+of+Drosophila+microRNA+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+identification+of+Drosophila+microRNA+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001265-Wang1" id="pone.0001265-Wang1"></a>Wang X, Zhang J, Li F, Gu J, He T, et al.  (2005) MicroRNA identification based on sequence and structure alignment. Bioinformatics  21: 3610–3614.  <ul class="find" data-citedArticleID="1026070" data-doi="10.1093/bioinformatics/bti562"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti562" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+identification+based+on+sequence+and+structure+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+identification+based+on+sequence+and+structure+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001265-Washietl1" id="pone.0001265-Washietl1"></a>Washietl S, Hofacker I, Lukasser M, Hüttenhofer A, Stadler P (2005) Mapping of conserved RNA secondary structures predicts thousands of functional noncoding RNAs in the human genome. Nat Biotechnol  23: 1383–1390.  <ul class="find" data-citedArticleID="1026072" data-doi="10.1038/nbt1144"><li><a href="http://dx.doi.org/10.1038/nbt1144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+of+conserved+RNA+secondary+structures+predicts+thousands+of+functional+noncoding+RNAs+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+of+conserved+RNA+secondary+structures+predicts+thousands+of+functional+noncoding+RNAs+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001265-Washietl2" id="pone.0001265-Washietl2"></a>Washietl S, Hofacker IL, Lukasser M, Huttenhofer A, Stadler PF (2005) Mapping of conserved RNA secondary structures predicts thousands of functional noncoding RNAs in the human genome. Nat Biotechnol  23: 1383–1390.  <ul class="find" data-citedArticleID="1026074" data-doi="10.1038/nbt1144"><li><a href="http://dx.doi.org/10.1038/nbt1144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+of+conserved+RNA+secondary+structures+predicts+thousands+of+functional+noncoding+RNAs+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+of+conserved+RNA+secondary+structures+predicts+thousands+of+functional+noncoding+RNAs+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001265-Pedersen1" id="pone.0001265-Pedersen1"></a>Pedersen J, Bejerano G, Siepel A, Rosenbloom K, Lindblad-Toh K, et al.  (2006) Identification and classification of conserved RNA secondary structures in the human genome. PLoS Comput Biol  2: e33.  <ul class="find" data-citedArticleID="1026050" data-doi="10.1371/journal.pcbi.0020033.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020033.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+classification+of+conserved+RNA+secondary+structures+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+classification+of+conserved+RNA+secondary+structures+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001265-Celniker1" id="pone.0001265-Celniker1"></a>Celniker SE, Rubin GM (2003) The Drosophila melanogaster genome. Annu Rev Genomics Hum Genet  4: 89–117.  <ul class="find" data-citedArticleID="1026010" data-doi="10.1146/annurev.genom.4.070802.110323"><li><a href="http://dx.doi.org/10.1146/annurev.genom.4.070802.110323" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Drosophila+melanogaster+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Drosophila+melanogaster+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001265-Aravin1" id="pone.0001265-Aravin1"></a>Aravin AA, Lagos-Quintana M, Yalcin A, Zavolan M, Marks D, et al.  (2003) The small RNA profile during Drosophila melanogaster development. Dev Cell  5: 337–350.  <ul class="find" data-citedArticleID="1025996" data-doi="10.1016/s1534-5807(03)00228-4"><li><a href="http://dx.doi.org/10.1016/s1534-5807(03)00228-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+small+RNA+profile+during+Drosophila+melanogaster+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+small+RNA+profile+during+Drosophila+melanogaster+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001265-Brennecke3" id="pone.0001265-Brennecke3"></a>Brennecke J, Aravin A, Stark A, Dus M, Kellis M, et al.  (2007) Discrete small RNA-generating loci as master regulators of transposon activity in Drosophila. Cell  128: 1089–1103.  <ul class="find" data-citedArticleID="1026004" data-doi="10.1016/j.cell.2007.01.043"><li><a href="http://dx.doi.org/10.1016/j.cell.2007.01.043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discrete+small+RNA-generating+loci+as+master+regulators+of+transposon+activity+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discrete+small+RNA-generating+loci+as+master+regulators+of+transposon+activity+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001265-Kuhn1" id="pone.0001265-Kuhn1"></a>Kuhn RM, Karolchik D, Zweig AS, Trumbower H, Thomas DJ, et al.  (2007) The UCSC genome browser database: update 2007. Nucleic Acids Res  35: D668–673.  <ul class="find" data-citedArticleID="1026034" data-doi="10.1093/nar/gkl928"><li><a href="http://dx.doi.org/10.1093/nar/gkl928" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+UCSC+genome+browser+database%3A+update+2007." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+UCSC+genome+browser+database%3A+update+2007.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001265-Siepel1" id="pone.0001265-Siepel1"></a>Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, et al.  (2005) Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res  15: 1034–1050.  <ul class="find" data-citedArticleID="1026060" data-doi="10.1101/gr.3715005"><li><a href="http://dx.doi.org/10.1101/gr.3715005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001265-Hsu1" id="pone.0001265-Hsu1"></a>Hsu PW, Huang HD, Hsu SD, Lin LZ, Tsou AP, et al.  (2006) miRNAMap: genomic maps of microRNA genes and their target genes in mammalian genomes. Nucleic Acids Res  34: D135–139.  <ul class="find" data-citedArticleID="1026026" data-doi="10.1093/nar/gkj135"><li><a href="http://dx.doi.org/10.1093/nar/gkj135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=miRNAMap%3A+genomic+maps+of+microRNA+genes+and+their+target+genes+in+mammalian+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22miRNAMap%3A+genomic+maps+of+microRNA+genes+and+their+target+genes+in+mammalian+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001265-Hofacker1" id="pone.0001265-Hofacker1"></a>Hofacker IL, Fekete M, Stadler PF (2002) Secondary structure prediction for aligned RNA sequences. J Mol Biol  319: 1059–1066.  <ul class="find" data-citedArticleID="1026022" data-doi="10.1016/s0022-2836(02)00308-x"><li><a href="http://dx.doi.org/10.1016/s0022-2836(02)00308-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Secondary+structure+prediction+for+aligned+RNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Secondary+structure+prediction+for+aligned+RNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001265-Hofacker2" id="pone.0001265-Hofacker2"></a>Hofacker IL (2003) Vienna RNA secondary structure server. Nucleic Acids Res  31: 3429–3431.  <ul class="find" data-citedArticleID="1026024" data-doi="10.1093/nar/gkg599"><li><a href="http://dx.doi.org/10.1093/nar/gkg599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vienna+RNA+secondary+structure+server." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vienna+RNA+secondary+structure+server.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001265-Pall1" id="pone.0001265-Pall1"></a>Pall G, Codony-Servat C, Byrne J, Ritchie L, Hamilton A (2007) Carbodiimide-mediated cross-linking of RNA to nylon membranes improves the detection of siRNA, miRNA and piRNA by northern blot. Nucleic Acids Res  35: e60.  <ul class="find" data-citedArticleID="1026048" data-doi="10.1093/nar/gkm112"><li><a href="http://dx.doi.org/10.1093/nar/gkm112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carbodiimide-mediated+cross-linking+of+RNA+to+nylon+membranes+improves+the+detection+of+siRNA%2C+miRNA+and+piRNA+by+northern+blot." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carbodiimide-mediated+cross-linking+of+RNA+to+nylon+membranes+improves+the+detection+of+siRNA%2C+miRNA+and+piRNA+by+northern+blot.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001265-Zhang1" id="pone.0001265-Zhang1"></a>Zhang Z, Schwartz S, Wagner L, Miller W (2000) A greedy algorithm for aligning DNA sequences. J Comput Biol  7: 203–214.  <ul class="find" data-citedArticleID="1026078" data-doi="10.1089/10665270050081478"><li><a href="http://dx.doi.org/10.1089/10665270050081478" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+greedy+algorithm+for+aligning+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+greedy+algorithm+for+aligning+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001265-Clamp1" id="pone.0001265-Clamp1"></a>Clamp M, Cuff J, Searle SM, Barton GJ (2004) The Jalview Java alignment editor. Bioinformatics  20: 426–427.  <ul class="find" data-citedArticleID="1026012" data-doi="10.1093/bioinformatics/btg430"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg430" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Jalview+Java+alignment+editor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Jalview+Java+alignment+editor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.XML" value="85613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.PDF" value="759719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g001.PNG_L" value="430688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g001.PNG_M" value="200500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g001.PNG_S" value="14010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g001.TIF" value="1724754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g001.PNG_I" value="67675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g002.PNG_L" value="585027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g002.PNG_M" value="174499"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g002.PNG_S" value="16323"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g002.TIF" value="2580094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g002.PNG_I" value="73385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.t001.PNG_L" value="325099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.t001.PNG_M" value="170027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.t001.PNG_S" value="17487"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.t001.TIF" value="1564880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.t001.PNG_I" value="55775"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g003.PNG_L" value="142550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g003.PNG_M" value="110939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g003.PNG_S" value="6618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g003.TIF" value="523668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g003.PNG_I" value="39763"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g004.PNG_L" value="2997727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g004.PNG_M" value="449427"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g004.PNG_S" value="14183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g004.TIF" value="4183618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g004.PNG_I" value="134670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g005.PNG_L" value="149179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g005.PNG_M" value="97865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g005.PNG_S" value="8280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g005.TIF" value="489116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g005.PNG_I" value="34996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g006.PNG_L" value="270171"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g006.PNG_M" value="139519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g006.PNG_S" value="9774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g006.TIF" value="732490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.g006.PNG_I" value="50659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.s001.PDF" value="2167419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001265.s002.PDF" value="22413"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001265&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001265" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001265&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001265&volume=&issue=&title=Identification of Novel Drosophila melanogaster MicroRNAs&author_name=Thomas%20Sandmann%2C%20Stephen%20M.%20Cohen&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001265" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001265" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001265" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001265&amp;t=Identification%20of%20Novel%20Drosophila%20melanogaster%20MicroRNAs" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001265&title=Identification%20of%20Novel%20Drosophila%20melanogaster%20MicroRNAs&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001265&amp;title=Identification%20of%20Novel%20Drosophila%20melanogaster%20MicroRNAs" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001265" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001265&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Identification of Novel Drosophila melanogaster MicroRNAs';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001265';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Identification%20of%20Novel%20Drosophila%20melanogaster%20MicroRNAs http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001265" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001265" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+genomics%22" title="Search for articles in the subject area:'Animal genomics'"><div class="flagText">Animal genomics</div></a>
              <div data-categoryid="17455" data-articleid="25362"
                   data-categoryname="Animal genomics"
                   class="flagImage" title="Flag 'Animal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drosophila+melanogaster%22" title="Search for articles in the subject area:'Drosophila melanogaster'"><div class="flagText">Drosophila melanogaster</div></a>
              <div data-categoryid="32461" data-articleid="25362"
                   data-categoryname="Drosophila melanogaster"
                   class="flagImage" title="Flag 'Drosophila melanogaster' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+libraries%22" title="Search for articles in the subject area:'Genomic libraries'"><div class="flagText">Genomic libraries</div></a>
              <div data-categoryid="40315" data-articleid="25362"
                   data-categoryname="Genomic libraries"
                   class="flagImage" title="Flag 'Genomic libraries' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invertebrate+genomics%22" title="Search for articles in the subject area:'Invertebrate genomics'"><div class="flagText">Invertebrate genomics</div></a>
              <div data-categoryid="17221" data-articleid="25362"
                   data-categoryname="Invertebrate genomics"
                   class="flagImage" title="Flag 'Invertebrate genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22MicroRNA%22" title="Search for articles in the subject area:'MicroRNA'"><div class="flagText">MicroRNA</div></a>
              <div data-categoryid="19821" data-articleid="25362"
                   data-categoryname="MicroRNA"
                   class="flagImage" title="Flag 'MicroRNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+alignment%22" title="Search for articles in the subject area:'RNA alignment'"><div class="flagText">RNA alignment</div></a>
              <div data-categoryid="32671" data-articleid="25362"
                   data-categoryname="RNA alignment"
                   class="flagImage" title="Flag 'RNA alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+sequencing%22" title="Search for articles in the subject area:'RNA sequencing'"><div class="flagText">RNA sequencing</div></a>
              <div data-categoryid="19765" data-articleid="25362"
                   data-categoryname="RNA sequencing"
                   class="flagImage" title="Flag 'RNA sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="25362"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6345'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8923'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4584&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
