

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Extensive Gains and Losses of Olfactory Receptor Genes in Mammalian Evolution</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000708"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000708" />

    <meta name="citation_title" content="Extensive Gains and Losses of Olfactory Receptor Genes in Mammalian Evolution"/>
    <meta itemprop="name" content="Extensive Gains and Losses of Olfactory Receptor Genes in Mammalian Evolution"/>

      <meta name="citation_author" content="Yoshihito Niimura"/>
            <meta name="citation_author_institution" content="Department of Bioinformatics, Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan"/>
            <meta name="citation_author_institution" content="Institute of Molecular Evolutionary Genetics and Department of Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Masatoshi Nei"/>
            <meta name="citation_author_institution" content="Institute of Molecular Evolutionary Genetics and Department of Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America"/>

    <meta name="citation_date" content="2007/8/8"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000708.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e708"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A novel multigene family may encode odorant receptors: a molecular basis for odor recognition.; citation_author=L Buck; citation_author=R Axel; citation_journal_title=Cell; citation_volume=65; citation_number=1; citation_pages=175-187; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Genes and ligands for odorant, vomeronasal and taste receptors.; citation_author=P Mombaerts; citation_journal_title=Nat Rev Neurosci; citation_volume=5; citation_number=2; citation_pages=263-278; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolutionary dynamics of olfactory and other chemosensory receptor genes in vertebrates.; citation_author=Y Niimura; citation_author=M Nei; citation_journal_title=J Hum Genet; citation_volume=51; citation_number=3; citation_pages=505-517; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The complete human olfactory subgenome.; citation_author=G Glusman; citation_author=I Yanai; citation_author=I Rubin; citation_author=D Lancet; citation_journal_title=Genome Res; citation_volume=11; citation_number=4; citation_pages=685-702; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The human olfactory receptor repertoire.; citation_author=S Zozulya; citation_author=F Echeverri; citation_author=T Nguyen; citation_journal_title=Genome Biol; citation_volume=2; citation_number=5; citation_pages=RESEARCH0018; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolution of olfactory receptor genes in the human genome.; citation_author=Y Niimura; citation_author=M Nei; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=6; citation_pages=12235-12240; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The olfactory receptor gene superfamily of the mouse.; citation_author=X Zhang; citation_author=S Firestein; citation_journal_title=Nat Neurosci; citation_volume=5; citation_number=7; citation_pages=124-133; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Different evolutionary processes shaped the mouse and human olfactory receptor gene families.; citation_author=JM Young; citation_author=C Friedman; citation_author=EM Williams; citation_author=JA Ross; citation_author=L Tonnes-Priddy; citation_journal_title=Hum Mol Genet; citation_volume=11; citation_number=8; citation_pages=535-546; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comparative evolutionary analysis of olfactory receptor gene clusters between humans and mice.; citation_author=Y Niimura; citation_author=M Nei; citation_journal_title=Gene; citation_volume=346; citation_number=9; citation_pages=13-21; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolutionary dynamics of olfactory receptor genes in fishes and tetrapods.; citation_author=Y Niimura; citation_author=M Nei; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=10; citation_pages=6039-6044; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The family of genes encoding odorant receptors in the channel catfish.; citation_author=J Ngai; citation_author=MM Dowling; citation_author=L Buck; citation_author=R Axel; citation_author=A Chess; citation_journal_title=Cell; citation_volume=72; citation_number=11; citation_pages=657-666; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Evolution by the birth-and-death process in multigene families of the vertebrate immune system.; citation_author=M Nei; citation_author=X Gu; citation_author=T Sitnikova; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=94; citation_number=12; citation_pages=7799-7806; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Concerted and birth-and-death evolution of multigene families.; citation_author=M Nei; citation_author=AP Rooney; citation_journal_title=Annu Rev Genet; citation_volume=39; citation_number=13; citation_pages=121-152; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The dog and rat olfactory receptor repertoires.; citation_author=P Quignon; citation_author=M Giraud; citation_author=M Rimbault; citation_author=P Lavigne; citation_author=S Tacher; citation_journal_title=Genome Biol; citation_volume=6; citation_number=14; citation_pages=R83; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ancient genomic architecture for mammalian olfactory receptor clusters.; citation_author=R Aloni; citation_author=T Olender; citation_author=D Lancet; citation_journal_title=Genome Biol; citation_volume=7; citation_number=15; citation_pages=R88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The olfactory receptor gene superfamily: data mining, classification, and nomenclature.; citation_author=G Glusman; citation_author=A Bahar; citation_author=D Sharon; citation_author=Y Pilpel; citation_author=J White; citation_journal_title=Mamm Genome; citation_volume=11; citation_number=16; citation_pages=1016-1023; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Dramatic variation of the vomeronasal pheromone receptor gene repertoire among five orders of placental and marsupial mammals.; citation_author=WE Grus; citation_author=P Shi; citation_author=YP Zhang; citation_author=J Zhang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=17; citation_pages=5767-5772; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Lineage-specific expansions and contractions of the bitter taste receptor gene repertoire in vertebrates.; citation_author=Y Go; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=18; citation_pages=964-972; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The evolution of mammalian gene families.; citation_author=JP Demuth; citation_author=TD Bie; citation_author=JE Stajich; citation_author=N Cristianini; citation_author=MW Hahn; citation_journal_title=PLoS ONE; citation_volume=1; citation_number=19; citation_pages=e85; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolutionary changes of the number of olfactory receptor genes in the human and mouse lineages.; citation_author=Y Niimura; citation_author=M Nei; citation_journal_title=Gene; citation_volume=346; citation_number=20; citation_pages=23-28; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Resolution of the early placental mammal radiation using Bayesian phylogenetics.; citation_author=WJ Murphy; citation_author=E Eizirik; citation_author=SJ O'Brien; citation_author=O Madsen; citation_author=M Scally; citation_journal_title=Science; citation_volume=294; citation_number=21; citation_pages=2348-2351; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecules consolidate the placental mammal tree.; citation_author=MS Springer; citation_author=MJ Stanhope; citation_author=O Madsen; citation_author=WW de Jong; citation_journal_title=Trends Ecol Evol; citation_volume=19; citation_number=22; citation_pages=430-438; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Retroposed elements as archives for the evolutionary history of placental mammals.; citation_author=JO Kriegs; citation_author=G Churakov; citation_author=M Kiefmann; citation_author=U Jordan; citation_author=J Brosius; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=23; citation_pages=E91; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Housekeeping genes for phylogenetic analysis of eutherian relationships.; citation_author=M Kullberg; citation_author=MA Nilsson; citation_author=U Arnason; citation_author=EH Harley; citation_author=A Janke; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=24; citation_pages=1493-1503; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A phylogenomic study of human, dog, and mouse.; citation_author=G Cannarozzi; citation_author=A Schneider; citation_author=G Gonnet; citation_journal_title=PLoS Comput Biol; citation_volume=3; citation_number=25; citation_pages=e2; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Loss of olfactory receptor genes coincides with the acquisition of full trichromatic vision in primates.; citation_author=Y Gilad; citation_author=V Wiebe; citation_author=M Przeworski; citation_author=D Lancet; citation_author=S Pääbo; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=26; citation_pages=E5; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Electroreception in monotremes.; citation_author=JD Pettigrew; citation_journal_title=J Exp Biol; citation_volume=202; citation_number=27; citation_pages=1447-1454; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The human olfactory receptor gene family.; citation_author=B Malnic; citation_author=PA Godfrey; citation_author=LB Buck; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=28; citation_pages=2584-2589; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The human sense of smell: are we better than we think?; citation_author=GM Shepherd; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=29; citation_pages=E146; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The new mutation theory of phenotypic evolution.; citation_author=M Nei; citation_journal_title=Proc Natl Acad Sci U S A; citation_number=30; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evolutionary dynamics of olfactory receptor genes in Drosophila species.; citation_author=M Nozawa; citation_author=M Nei; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=31; citation_pages=7122-7127; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Comparative chemosensation from receptors to ecology.; citation_author=CI Bargmann; citation_journal_title=Nature; citation_volume=444; citation_number=32; citation_pages=295-301; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of odor receptor gene choice in Drosophila.; citation_author=A Ray; citation_author=WG van Naters; citation_author=T Shiraiwa; citation_author=JR Carlson; citation_journal_title=Neuron; citation_volume=53; citation_number=33; citation_pages=353-369; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Negative feedback regulation ensures the one receptor-one olfactory neuron rule in mouse.; citation_author=S Serizawa; citation_author=K Miyamichi; citation_author=H Nakatani; citation_author=M Suzuki; citation_author=M Saito; citation_journal_title=Science; citation_volume=302; citation_number=34; citation_pages=2088-2094; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=35; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=MAFFT version 5: improvement in accuracy of multiple sequence alignment.; citation_author=K Katoh; citation_author=K Kuma; citation_author=H Toh; citation_author=T Miyata; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=36; citation_pages=511-518; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Fitting the gene lineage into its species lineage, a parsimony strategy illustrated by cladograms constructed from globin sequences.; citation_author=M Goodman; citation_author=J Czelusniak; citation_author=GW Moore; citation_author=AE Romero-Herrera; citation_author=G Matsuda; citation_journal_title=Syst Zool; citation_volume=28; citation_number=37; citation_pages=132-168; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=From gene to organismal phylogeny: reconciled trees and the gene tree/species tree problem.; citation_author=RD Page; citation_author=MA Charleston; citation_journal_title=Mol Phylogenet Evol; citation_volume=7; citation_number=38; citation_pages=231-240; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Evolutionary change of the numbers of homeobox genes in bilateral animals.; citation_author=J Nam; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=39; citation_pages=2386-2394; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution and phylogenetics.; citation_author=M Nei; citation_author=S Kumar; citation_number=40; citation_pages=175; citation_date=2000; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees.; citation_author=N Saitou; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=4; citation_number=41; citation_pages=406-425; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic test of molecular clock and linearized trees.; citation_author=N Takezaki; citation_author=A Rzhetsky; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=12; citation_number=42; citation_pages=823-833; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mammalian phylogenomics comes of age.; citation_author=WJ Murphy; citation_author=PA Pevzner; citation_author=SJ O'Brien; citation_journal_title=Trends Genet; citation_volume=20; citation_number=43; citation_pages=631-639; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Estimation of divergence times for major lineages of primate species.; citation_author=GV Glazko; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=44; citation_pages=424-434; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000708" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Extensive Gains and Losses of Olfactory Receptor Genes in Mammalian Evolution"/>
    <meta name="twitter:description" content="Odor perception in mammals is mediated by a large multigene family of olfactory receptor (OR) genes. The number of OR genes varies extensively among different species of mammals, and most species have a substantial number of pseudogenes. To gain some insight into the evolutionary dynamics of mammalian OR genes, we identified the entire set of OR genes in platypuses, opossums, cows, dogs, rats, and macaques and studied the evolutionary change of the genes together with those of humans and mice. We found that platypuses and primates have &lt;400 functional OR genes while the other species have 800&ndash;1,200 functional OR genes. We then estimated the numbers of gains and losses of OR genes for each branch of the phylogenetic tree of mammals. This analysis showed that (i) gene expansion occurred in the placental lineage each time after it diverged from monotremes and from marsupials and (ii) hundreds of gains and losses of OR genes have occurred in an order-specific manner, making the gene repertoires highly variable among different orders. It appears that the number of OR genes is determined primarily by the functional requirement for each species, but once the number reaches the required level, it fluctuates by random duplication and deletion of genes. This fluctuation seems to have been aided by the stochastic nature of OR gene expression."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000708.g002"/>

  <meta property="og:title" content="Extensive Gains and Losses of Olfactory Receptor Genes in Mammalian Evolution" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000708" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2446'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3889'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7619&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000708">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000708" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000708&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Extensive Gains and Losses of Olfactory Receptor Genes in Mammalian Evolution
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yoshihito Niimura
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:niimura@bioinfo.tmd.ac.jp">niimura@bioinfo.tmd.ac.jp</a></p>

                <p>Affiliations:
                  Department of Bioinformatics, Medical Research Institute, Tokyo Medical and Dental University, Tokyo, Japan, 
                  Institute of Molecular Evolutionary Genetics and Department of Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Masatoshi Nei
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Molecular Evolutionary Genetics and Department of Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 08, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000708</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000708" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000708" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000708" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000708" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000708-t001" data-doi="info:doi/10.1371/journal.pone.0000708" data-uri="info:doi/10.1371/journal.pone.0000708.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000708-t002" data-doi="info:doi/10.1371/journal.pone.0000708" data-uri="info:doi/10.1371/journal.pone.0000708.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000708-g001" data-doi="info:doi/10.1371/journal.pone.0000708" data-uri="info:doi/10.1371/journal.pone.0000708.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000708-g002" data-doi="info:doi/10.1371/journal.pone.0000708" data-uri="info:doi/10.1371/journal.pone.0000708.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000708">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000708" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Odor perception in mammals is mediated by a large multigene family of olfactory receptor (OR) genes. The number of OR genes varies extensively among different species of mammals, and most species have a substantial number of pseudogenes. To gain some insight into the evolutionary dynamics of mammalian OR genes, we identified the entire set of OR genes in platypuses, opossums, cows, dogs, rats, and macaques and studied the evolutionary change of the genes together with those of humans and mice. We found that platypuses and primates have &lt;400 functional OR genes while the other species have 800–1,200 functional OR genes. We then estimated the numbers of gains and losses of OR genes for each branch of the phylogenetic tree of mammals. This analysis showed that (i) gene expansion occurred in the placental lineage each time after it diverged from monotremes and from marsupials and (ii) hundreds of gains and losses of OR genes have occurred in an order-specific manner, making the gene repertoires highly variable among different orders. It appears that the number of OR genes is determined primarily by the functional requirement for each species, but once the number reaches the required level, it fluctuates by random duplication and deletion of genes. This fluctuation seems to have been aided by the stochastic nature of OR gene expression.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Niimura Y, Nei M (2007) Extensive Gains and Losses of Olfactory Receptor Genes in Mammalian Evolution. PLoS ONE 2(8):
          e708.
            doi:10.1371/journal.pone.0000708</p><p><strong>Academic Editor: </strong>Matthew Hahn, Indiana University, United States of America</p><p><strong>Received:</strong> May 14, 2007; <strong>Accepted:</strong> July 4, 2007; <strong>Published:</strong> August 8, 2007</p><p><strong>Copyright:</strong> © 2007 Niimura, Nei. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by US NIH grant GM020293 to M.N. and by grant 17710162 from Ministry of Education, Culture, Sports, Science and Technology, Japan and by the Genome Information Integration Project of the Ministry of Economy, Trade and Industry of Japan and the Japan Biological Informatics Consortium to Y.N.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Vertebrate olfactory receptors (ORs) are G-protein-coupled receptors (GPCRs) containing seven transmembrane α-helical regions and function as the receptors for various odor molecules in the environment <a href="#pone.0000708-Buck1">[1]</a>–<a href="#pone.0000708-Niimura1">[3]</a>. It is known that OR genes form the largest multigene family in vertebrates. However, the numbers of OR genes are quite different among different species, and each species has a large number of pseudogenes in addition to functional genes. For example, humans have ~800 OR genes, but ~50% of them are pseudogenes <a href="#pone.0000708-Glusman1">[4]</a>–<a href="#pone.0000708-Niimura2">[6]</a>. By contrast mice have ~1,400 OR genes, and the fraction of pseudogenes is 20–25% <a href="#pone.0000708-Zhang1">[7]</a>–<a href="#pone.0000708-Niimura3">[9]</a>. Therefore, the number of functional genes is ~2.7 times larger in mice than in humans. OR genes are present as genomic clusters that are scattered on many different chromosomes. Despite the difference in the number of genes between humans and mice, the organization of OR genomic clusters is well conserved between the two species <a href="#pone.0000708-Niimura3">[9]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Previously we studied the evolutionary change of OR genes in vertebrates using zebrafish, pufferfish, frog, chicken, mouse, and human data <a href="#pone.0000708-Niimura4">[10]</a> and showed that the OR gene family is considerably smaller (~100) <a href="#pone.0000708-Ngai1">[11]</a> but is more diversified in fishes than in mammals. We also showed that particular groups of genes have expanded and others were completely lost in the tetrapod lineage. It therefore appears that the OR gene family is subject to an extreme form of birth-and-death evolution <a href="#pone.0000708-Nei1">[12]</a>, <a href="#pone.0000708-Nei2">[13]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To understand the evolutionary mechanism of this multigene family, it is important to study the variation of OR genes among mammalian species living in diverse environments, because they have much larger repertoires than non-mammalian species do. Now that the draft genome sequences are available for at least six different orders of mammals including two early-diverged lineages, monotremes (platypus) and marsupials (opossum), we conducted comparative and evolutionary analyses of OR genes from eight mammalian species.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p><a href="#pone-0000708-t001">Table 1</a> shows the numbers of OR genes identified from platypuses, opossums, cows, dogs, rats, and macaques as well as those from humans and mice. The number of functional OR genes having intact coding sequences is considerably smaller in primates and platypuses than in other species. The numbers in <a href="#pone-0000708-t001">Table 1</a> are minimum estimates of the numbers of functional OR genes because we used draft genome sequences that were incomplete. It is possible that some functional OR genes were misannotated as pseudogenes because of sequencing errors or nearly identical copies of genes were collapsed into one sequence because of assembly errors. Moreover, genome sequences containing short contigs tend to give an underestimate of the number of functional genes, because a functional OR gene located at the end of a contig is truncated. For this reason, we counted the numbers of truncated genes that could become functional when the genome sequence is completed. We identified large numbers of truncated genes from the cow and platypus genomes (<a href="#pone-0000708-t001">Table 1</a>), reflecting a relatively low quality of the genome sequences of these species. The fraction of pseudogenes in platypuses is estimated to be ~50% under the assumption that truncated genes are functional, and this fraction is similar to that in humans. By contrast, opossums showed the lowest fraction of pseudogenes (&lt;20%) in the species examined.</p>
<div class="figure" id="pone-0000708-t001"><div class="img"><a name="pone-0000708-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000708" data-uri="info:doi/10.1371/journal.pone.0000708.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000708.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000708.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>OR genes in eight mammalian species.</span></strong></p><span>doi:10.1371/journal.pone.0000708.t001</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>The nearly complete OR gene repertoires in dogs, rats, and opossums were also reported in Quignon et al. <a href="#pone.0000708-Quignon1">[14]</a> and Aloni et al. <a href="#pone.0000708-Aloni1">[15]</a>. Although the method of OR gene identification in this study is different from theirs, the numbers of genes identified are generally similar to each other. Quignon et al. <a href="#pone.0000708-Quignon1">[14]</a> identified 1,094 and 1,493 OR genes from dogs and rats, respectively, on the basis of the presence of five amino acid motifs that were extracted from already annotated OR genes. They estimated the fractions of pseudogenes to be 20.3% and 19.5% and for dogs and rats, respectively, which are considerably lower than our estimates. One reason for the discrepancy would be that we used more stringent criteria for the identification of putatively functional OR gene. In our criteria, a functional gene should have initiation and stop codons at proper positions and should not contain any nonsense or frameshift mutations or long deletions (see <a href="#pone.0000708.s001">Protocol S1</a>), while Quignon et al. <a href="#pone.0000708-Quignon1">[14]</a> regarded the sequences other than mutation-containing ones as functional. It is also possible that a considerable number of pseudogenes that are fragmented or do not retain the motifs were not contained in their datasets. We distinguished OR genes from non-OR GPCR genes by constructing phylogenetic trees and did not use the information of motif sequences. However, the motif sequences characteristic to OR genes such as the MA(Y/F)DRYVAIC (single-letter amino acid notation) motif <a href="#pone.0000708-Zozulya1">[5]</a>, <a href="#pone.0000708-Zhang1">[7]</a> were well conserved among the functional OR genes identified in this study. Aloni et al. <a href="#pone.0000708-Aloni1">[15]</a> identified 1,518 OR genes from opossums, which is similar to our result, but they did not mention the fraction of pseudogenes. We used more recent versions of the genome sequences than the previous studies, and therefore our results are expected to be more accurate.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>To investigate the evolutionary change of the number of OR genes in mammals, we estimated the numbers of genes in the ancestral species and the numbers of gene gains and losses for each branch of the evolutionary tree of the eight species using parsimony principle (see <a href="#s4">Materials and Methods</a>). To estimate these numbers, we classified OR genes into several groups, because the number of genes was very large (&gt;6,000). Mammalian OR genes can be divided into Class I and Class II genes by sequence similarities <a href="#pone.0000708-Niimura2">[6]</a>, <a href="#pone.0000708-Niimura3">[9]</a>, <a href="#pone.0000708-Glusman2">[16]</a>. A majority of the genes belong to Class II. (Class I genes are 10–20%; see <a href="#pone-0000708-t002">Table 2</a>.) We therefore divided Class II OR genes into subgroups by considering phylogenetic relationships. This generated 34 phylogenetic clades that were supported with high (&gt;90%) bootstrap values (<a href="#pone.0000708.s004">Figure S1</a>) <a href="#pone.0000708-Niimura2">[6]</a>, <a href="#pone.0000708-Niimura3">[9]</a>. Note that a considerable number of Class II genes remained unclassified, because the phylogenetic relationships were not completely resolved.</p>
<div class="figure" id="pone-0000708-t002"><div class="img"><a name="pone-0000708-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000708" data-uri="info:doi/10.1371/journal.pone.0000708.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000708.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000708.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Number of functional OR genes belonging to each clade.</span></strong></p><span>doi:10.1371/journal.pone.0000708.t002</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>The results of this classification are shown in <a href="#pone-0000708-t002">Table 2</a>. Note that the number of genes belonging to one clade is often highly variable among different species. For example, platypuses have 52 functional genes belonging to Clade L, which is the largest clade for this species, and opossums have 63 Clade L genes. By contrast, cows and dogs have only two Clade L genes. Similarly, opossums have 59 Clade AD genes, but humans have only one gene and macaques have no gene belonging to this clade. A phylogenetic tree in <a href="#pone-0000708-g001">Figure 1A</a> shows that all of the Clade AD genes in opossums form a monophyletic clade, suggesting that marsupial-specific gene expansion has occurred. This tree also indicates that several gene duplications have occurred in the rodent lineage before the divergence of mice and rats. In accordance with these observations, it was estimated that a large number of gene gains (+58) in the opossum lineage, a moderate number of gene gains (+9) in the rodent lineage before mouse-rat divergence, and some gene losses (-4) in the primate lineage before human-macaque divergence have occurred (<a href="#pone-0000708-g001">Figure 1B</a>).</p>
<div class="figure" id="pone-0000708-g001"><div class="img"><a name="pone-0000708-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000708" data-uri="info:doi/10.1371/journal.pone.0000708.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000708.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000708.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Gains and losses of OR genes during mammalian evolution.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>(A) NJ tree for 105 Clade AD genes and eight outgroup genes. The outgroup genes used are human Class II genes belonging to Clades A–H and are shown in black. The number of amino acids used is 288. Bootstrap values were obtained by 500 resamplings, and only the values that are &gt;70% are shown. A scale bar indicates the number of amino acid substitutions per site. (B) Evolutionary changes of the number of Clade AD genes inferred from data in (A). The Euarchontoglires tree topology and a 70% bootstrap condensed tree were used for the estimation. The numbers in rectangular boxes are those of Clade AD functional genes for the extant or ancestral species. The numbers with plus and minus signs for a branch indicate gene gains and losses, respectively. (C) NJ tree for 45 Clade P genes and eight outgroup genes. The number of amino acids used is 290. Bootstrap values &gt;70% are shown. Phylogenetic clades indicated by α and γ contain one gene from each of the seven species (opossums and placentals), and a clade shown by β contains one gene from each of the six placental mammals. (D) Evolutionary changes of the number of Clade P genes inferred from data in (C). Platypuses are not shown, because they lack Clade P genes. A 70% bootstrap condensed tree was used.</p>
<span>doi:10.1371/journal.pone.0000708.g001</span></div><a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>In contrast, Clade P shows a relatively stable number of genes in the evolutionary process (<a href="#pone-0000708-g001">Figure 1C, D</a>). The numbers of genes in Clade P are similar (5–8) for all the species except platypuses, which lack Clade P genes. Moreover, phylogenetic clades indicated by α and γ in <a href="#pone-0000708-g001">Figure 1C</a> contain one gene from each of the seven species, and clade β contains one gene from each of the six species of placental mammals, suggesting that the occurrence of gene gains and losses in this clade was not frequent (<a href="#pone-0000708-g001">Figure 1D</a>). However, this kind of one-to-one orthologous relationships among different mammalian species are rare for OR genes. We found only four, 14, and 19 phylogenetic clades that contained one gene from each of the eight, seven (opossums and placentals), and six (placentals) species, respectively, and were supported with &gt;90% bootstrap values. It therefore appears that the dynamic change of the number of OR genes was the general rule.</p>
<a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p><a href="#pone-0000708-g002">Figure 2A</a> shows estimates of the evolutionary changes of the number of OR genes when the currently popular mammalian phylogenetic tree is used. These estimates are obtained by considering all clades of genes and unclassified genes. The results suggest that the number of OR genes in the most recent common ancestor (MRCA) for all placental mammals is much larger than that in the MRCA between marsupials and placentals, and the latter is in turn much larger than that in the MRCA for all the species. It was estimated that &gt;300 gene gains have occurred in branches a and c of <a href="#pone-0000708-g002">Figure 2B</a>. Furthermore, hundreds of gene gains and losses occurred in an order-specific manner. Apparently &gt;750 gene gains occurred in the marsupial lineage (branch b in <a href="#pone-0000708-g002">Figure 2B</a>) and &gt;400 gene gains occurred in the cetartiodactyl (branch d) and rodent (branch f) lineages. Moreover, &gt;170 gene losses occurred in each of the branches leading to different placental orders (branches d–g). These findings indicate that although the current numbers of functional OR genes in several mammalian species are similar (~1,000), their OR gene repertoires have been highly variable. Interestingly, order-specific expansions or contractions of multigene families have been reported for other chemosensory receptors such as vomeronasal receptors <a href="#pone.0000708-Grus1">[17]</a> and bitter taste receptors <a href="#pone.0000708-Go1">[18]</a>. Demuth et al. <a href="#pone.0000708-Demuth1">[19]</a> reported that such lineage-specific expansions or contractions are frequently observed in mammalian gene families. Because our method is expected to give underestimates of the numbers of genes in the ancestral species, it is possible that the estimates will increase when the additional species are used for the analysis. Nevertheless, our estimates for the MRCA between humans and mice (~690) is fairly close to the number (~750) obtained by a different method using both functional genes and pseudogenes from the two species <a href="#pone.0000708-Niimura5">[20]</a>.</p>
<div class="figure" id="pone-0000708-g002"><div class="img"><a name="pone-0000708-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000708" data-uri="info:doi/10.1371/journal.pone.0000708.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000708.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000708.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000708.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000708.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000708.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Evolutionary changes of the number of OR genes in mammals.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>(A) The numbers in rectangular boxes are those of functional OR genes for the extant or ancestral species. The Euarchontoglires tree topology is used. We used a 70% bootstrap condensed tree of OR genes, but the results were essentially the same when 50%, 60%, 80%, and 90% bootstrap condensed trees were used (see <a href="#pone.0000708.s002">Table S1</a> and <a href="#pone.0000708.s004">Figure S4A</a>). (B) Schematic representation of the results of (A). A plus sign and a minus sign for a branch represent 150 gene gains and losses, respectively. The evolutionary timescale is shown at the bottom in million years (MY). The divergence times were obtained from Murphy et al. <a href="#pone.0000708-Murphy2">[43]</a> except for the time of the human-macaque divergence, which was taken from Glazko and Nei <a href="#pone.0000708-Glazko1">[44]</a>. (C) Results obtained by using the mouse-outside tree and a 70% bootstrap condensed tree. The results for 50%, 60%, 80%, and 90% bootstrap condensed trees were given in <a href="#pone.0000708.s003">Table S2</a> and <a href="#pone.0000708.s007">Figure S4B</a>.</p>
<span>doi:10.1371/journal.pone.0000708.g002</span></div><a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>The branching patterns of the four placental mammalian orders examined are not fully resolved. Murphy et al. <a href="#pone.0000708-Murphy1">[21]</a> proposed that primates and rodents are sister groups and they form a clade named Euarchontoglires together with several other orders. This topology shown in <a href="#pone-0000708-g002">Figure 2A</a> (Euarchontoglires tree) has been supported by some other studies as well <a href="#pone.0000708-Springer1">[22]</a>, <a href="#pone.0000708-Kriegs1">[23]</a>. However, several authors suggested the topology in <a href="#pone-0000708-g002">Figure 2C</a> (rodent-outside tree) <a href="#pone.0000708-Kullberg1">[24]</a>, <a href="#pone.0000708-Cannarozzi1">[25]</a>. We therefore conducted the same statistical analysis using the rodent-outside tree. Our data favored the rodent-outside tree, because the total number of gene gains and losses (4,968) is smaller in this tree than in the Euarchontoglires tree (5,134). However, our general conclusion about gene gains and losses was essentially the same for the two trees.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study, we showed that the numbers of OR genes have changed extensively in mammalian evolution. Why did the number change so frequently in mammalian evolution? One obvious factor would be the requirement for a species to adapt to a particular environmental condition. For most mammalian species, detection of millions of different odorants is crucial for their survival. Yet, animals living in different environments require different numbers of ORs. For example, olfaction seems to be less important for the primate species that are endowed with trichromatic vision than for other dichromatic mammalian species, because trichromatic color vision is very powerful for perceiving environment signals. This could be the reason why humans or macaques have a smaller number of OR genes than rodents <a href="#pone.0000708-Gilad1">[26]</a>. Platypuses also show a small number of functional genes and a large fraction of pseudogenes. The real reason is unclear, but it may have to do with their semi-aquatic lifestyle <a href="#pone.0000708-Niimura4">[10]</a>. Platypuses have the bill sense, which is a sophisticated combination of electroreception and mechanoreception, and they can find prey with their eyes, ears, and nostrils closed <a href="#pone.0000708-Pettigrew1">[27]</a>. This situation is similar to that of toothed whales, which have apparently lost the olfactory system and developed the echolocation system to adapt to the full aquatic life. In fact, there are data suggesting that the fraction of OR pseudogenes in toothed whales is extremely high (Go et al., unpublished).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Previously we proposed that the dynamic expansion of OR genes has occurred in the tetrapod lineage during the process of the terrestrial adaptation <a href="#pone.0000708-Niimura4">[10]</a>. This expansion has happened presumably because olfaction is more important in terrestrial life than in aquatic life. Our results (<a href="#pone-0000708-g002">Figure 2B</a>) suggest that the expansion of OR genes continued until the time of mammalian radiation approximately 100 million years ago. As mentioned above, a particular group of genes have often expanded in one lineage (<a href="#pone-0000708-t002">Table 2</a>; <a href="#pone-0000708-g001">Figure 1A</a>). This might have happened because this group of genes is useful specifically for the lineage. For example, Clade AD genes may be able to detect odors that are essential for opossums. At the present time, however, information about the ligands for mammalian OR genes is quite limited. One of the mouse genes belonging to Clade B, which has many genes in humans, is known to detect the smell of lemons (limonene), and one of the mouse genes in Clade G, which is abundant in rodents, perceives floral or woody smell (acetophenone) <a href="#pone.0000708-Malnic1">[28]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Nevertheless, the relationship between the number of OR genes and the environmental factor is not always clear. Dogs, which are supposed to have good sense of smell, do not have the largest number of functional OR genes. It is also difficult to explain why cows have nearly 1,000 functional genes and nearly the same number of pseudogenes. Furthermore, it is known that in rats up to 80 percent of the glomerular layer in the olfactory bulb can be removed without significant effect on olfactory detection and discrimination <a href="#pone.0000708-Shepherd1">[29]</a>. Shepherd <a href="#pone.0000708-Shepherd1">[29]</a> pointed out the importance of processing of odor distinction in the brain, stating that the expansion of higher brain mechanisms may offset the reduced repertoire of OR genes in humans.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>If we consider there factors, it appears that the number of OR genes in a species is not directly related to the environmental requirement or life style, and there are random elements that determine the number of OR genes <a href="#pone.0000708-Nei3">[30]</a>. These random elements are of course caused by random duplication and random inactivation of genes. In other words, the number of OR genes may fluctuate around the most appropriate number of the genes for a given species, and this fluctuation appears to be quite high if we consider the existence of a large number of pseudogenes in many species.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p><a href="#pone-0000708-g002">Figure 2</a> shows that the evolutionary change of the number of OR genes is exceptionally high. Many multigene families show some evolutionary change of the number of member genes, but the extent of the change is much smaller except for a few other sensory receptor genes <a href="#pone.0000708-Nei2">[13]</a>. Even with OR genes, the evolutionary change in insects is not as extensive as in mammals. In a group of 12 <em>Drosophila</em> species encompassing the divergence times up to about 60 million years the number of OR genes is known to have been quite stable during the evolution <a href="#pone.0000708-Nozawa1">[31]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Why then did the number of OR genes change so dramatically in mammals but not in <em>Drosophila</em>? One possible explanation is the difference in the mechanism of gene expression system between mammals and <em>Drosophila</em>. In <em>Drosophila</em>, a specific OR gene tends to be expressed deterministically in a given olfactory neuron <a href="#pone.0000708-Bargmann1">[32]</a>, <a href="#pone.0000708-Ray1">[33]</a>. Therefore, if an OR gene is duplicated or lost from the genome, the gene expression system may be disturbed. In mammals, however, one of the clustered OR genes in the genome is stochastically chosen to be expressed in each olfactory neuron <a href="#pone.0000708-Serizawa1">[34]</a>. Therefore, the expression pattern of OR genes appears to be considerably different among different individuals, and consequently the number of OR genes may change relatively easily in the evolutionary process <a href="#pone.0000708-Nozawa1">[31]</a>. Of course, this is a hypothesis at present, and it should be tested by experiments.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The draft genome sequences of rhesus macaques (<em>Macaca mulatta</em>; rheMac2, released in Jan. 2006; 5.1× coverage), rats (<em>Rattus norvegicus</em>; rn4, released in Nov. 2004; 7× coverage), dogs (<em>Canis familiaris</em>; canFam2, released in May 2005; 7.6× coverage), and cows (<em>Bos taurus</em>; bosTau2, released in Mar. 2005; 6.2× coverage) were downloaded from the UCSC Genome Bioinformatics Site (<a href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>). The opossum genome sequences (<em>Monodelphis domestica</em>; monDom4, released in Jan. 2006; 6.5× coverage) were downloaded from the Ensembl Genome Browser (<a href="http://www.ensembl.org">http://www.ensembl.org</a>). The platypus genome sequences (<em>Ornithorhynchus anatinus</em>, released in Dec. 2005; 6× coverage) were retrieved from the website of the Genome Sequencing Center at Washington University School of Medicine (<a href="http://genome.wustl.edu">http://genome.wustl.edu</a>). We did not use the sequences in the ‘bin0’ category for the cow genome, because they were not assembled.</p>


<h4>OR Gene Identification</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The method to identify functional OR genes from draft genome sequences of platypuses, opossums, dogs, cows, rats, and macaques is essentially the same as that used in our previous studies <a href="#pone.0000708-Niimura2">[6]</a>, <a href="#pone.0000708-Niimura3">[9]</a>, but we improved it to be applicable to any mammalian species. Details of the method are provided in <a href="#pone.0000708.s001">Protocol S1</a> and <a href="#pone.0000708.s005">Figure S2</a>. OR pseudogenes and truncated genes were identified in the following way. We first conducted TBLASTN <a href="#pone.0000708-Altschul1">[35]</a> searches against the genome sequences using all functional genes in each species identified in this study as queries with the E-value below 1e-20. We then extracted the non-overlapping blast-hits showing the lowest E-values among the hits to a given genomic region. After excluding functional OR genes identified, we regarded all remaining sequences as pseudogenes or truncated genes. The reason we used the cutoff E-value of 1e-20 is as follows. First, the lowest E-value for non-OR blast-hits was around 1e-17 or 1e-18. Second, we confirmed that all blast-hits showing the E-value below 1e-20 are more similar to OR genes than to known non-OR genes. Therefore, OR pseudogenes and non-OR genes are distinguishable by using the E-value of 1e-20. To identify truncated genes from these sequences, we extracted the sequences that did not have any nonsense or frameshift mutations and were located close (&lt;30 base pairs) to the contig end. We then constructed a multiple alignment of these sequences together with functional OR genes by the program E-INS-i in MAFFT version 5.8 <a href="#pone.0000708-Katoh1">[36]</a>. From the alignment, we extracted truncated sequences that meet the following condition. When the C-terminal portion of an OR gene is missing from the genome sequence, the N-terminal portion should contain an initiation codon at a proper position and should not contain any nonsense mutations, frameshifts, or long gaps. When the N-terminal portion is missing, the C-terminal portion should have a stop codon at a proper position and should not contain any nonsense mutations, frameshifts, or long gaps. Amino acid sequences of all OR genes identified in this study are available in <a href="#pone.0000708.s008">Dataset S1</a>.</p>


<h4>Estimation of the numbers of genes in the ancestral species and those of gene gains and losses</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>To estimate these numbers, we used the reconciled tree method <a href="#pone.0000708-Goodman1">[37]</a>–<a href="#pone.0000708-Nam1">[39]</a>, in which the topology of a gene tree is reconciled with that of a species tree. A simple example is shown in <a href="#pone.0000708.s001">Protocol S1</a> and <a href="#pone.0000708.s006">Figure S3</a>. Since phylogenetic relationships of genes are not completely resolved due to low bootstrap values, we considered a condensed tree with a given bootstrap value level as a gene tree <a href="#pone.0000708-Nam1">[39]</a>, <a href="#pone.0000708-Nei4">[40]</a>. To apply this method to OR genes, we developed a computer program, which is available on request to Y. N.</p>


<h4>Classification of OR genes</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>In the previous studies <a href="#pone.0000708-Niimura2">[6]</a>, <a href="#pone.0000708-Niimura3">[9]</a>, we classified human and mouse OR genes into phylogenetic clades that were supported with &gt;90% bootstrap values. We classified functional OR genes identified from six mammalian species into these clades. For this purpose, we constructed phylogenetic trees for all functional genes from each of the six species together with those from humans or mice. Using these trees, the assignment of clades could be conducted without any ambiguity, because all human or mouse genes belonging to one clade were always contained in a larger clade supported with a high bootstrap value. We then constructed phylogenetic trees using all genes in any pairs of species out of the six species. In every tree obtained, genes assigned to the same clade formed a monophyletic clade supported with a high bootstrap value, almost all of which was &gt;90% (<a href="#pone-0000708-t002">Table 2</a>; <a href="#pone.0000708.s002">Figure S1</a>), showing that the classification is robust. We identified four new clades (BA–BD) with &gt;90% bootstrap supports that contained ten or more member genes from at least one species. We did not use Clades AJ–AS in reference <a href="#pone.0000708-Niimura3">[9]</a>, because the numbers of genes belonging to these clades are small. We used Class I gene clade and 34 Class II gene clades (A–S, AA–AJ, AT, BA–BD) to apply the reconciled tree method. Several Class II genes remained unclassified and were examined separately. The names of functional OR genes belonging to each clade are provided in <a href="#pone.0000708.s009">Dataset S2</a>.</p>


<h4>Evolutionary changes of the number of OR genes</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>We first constructed a phylogenetic tree using all genes belonging to each of the 35 clades (Class I gene clade and 34 Class II gene clades) together with eight outgroup genes each of which was chosen from Clades A–H. The reconciled tree method was applied to the 35 phylogenetic trees. We used 50%, 60%, 70%, 80%, and 90% bootstrap condensed trees of OR genes. Unclassified Class II genes were examined in the following way. We constructed a phylogenetic tree using all unclassified genes together with 34 representative genes each of which was randomly chosen from the 34 Class II gene clades. Five Class I genes were also selected randomly and were used as outgroup genes. Because the tree topology slightly changed depending on the genes used, we repeated tree construction for 20 times. Out of the 20 trees, we selected one tree of which the phylogenetic relationships were best resolved in the following way. The total of the numbers of clades with &gt;50%, &gt;70%, &gt;80%, &gt;90%, and &gt;95% bootstrap supports was calculated for each of the 20 trees, and the tree showing the largest value was regarded to be the best tree. Numbers in <a href="#pone-0000708-g002">Figure 2A,C</a> were obtained by summing up the results for the 35 clades of genes and unclassified genes.</p>


<h4>Phylogenetic Tree Construction</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Translated amino acid sequences of OR genes were aligned by the program E-INS-i in MAFFT version 5.8 <a href="#pone.0000708-Katoh1">[36]</a>. Poisson correction distances were calculated after all alignment gaps were eliminated. A phylogenetic tree was constructed from these distances using the neighbor-joining (NJ) method <a href="#pone.0000708-Saitou1">[41]</a> by the program LINTREE <a href="#pone.0000708-Takezaki1">[42]</a> available at <a href="http://www.bio.psu.edu/People/Faculty/Nei/Lab">http://www.bio.psu.edu/People/Faculty/Ne​i/Lab</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000708"></div><a name="pone.0000708.s001" id="pone.0000708.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000708.s001">Protocol S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Supplementary materials and methods.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000708.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000708.s002" id="pone.0000708.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000708.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Estimated numbers of genes in the ancestral species and those of gene gains and losses for the Euarchontoglires tree and various bootstrap condensed trees.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000708.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0000708.s003" id="pone.0000708.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000708.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Estimated numbers of genes in the ancestral species and those of gene gains and losses for the mouse-outside tree and various bootstrap condensed trees.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000708.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0000708.s004" id="pone.0000708.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000708.s004">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">(A) A neighbor-joining (NJ) phylogenetic tree for 265 functional OR genes in platypuses and 1,188 genes in opossums. Purple and blue lines represent branches for platypuses and opossums, respectively. Bootstrap values obtained from 500 replications are shown for the branches determining Class I clade and 34 Class II clades. The scale bar indicates the estimated number of amino acid substitutions per site. (B) An NJ tree for 811 functional OR genes in dogs and 387 genes in humans. Green and orange lines represent branches for dogs and humans, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000708.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.47 MB PDF)</p>
<a name="pone.0000708.s005" id="pone.0000708.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000708.s005">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Flowchart for the identification of functional OR genes and OR pseudogenes. See <a href="#s4">Materials and Methods</a> and <a href="#pone.0000708.s001">Protocol S1</a> for details.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000708.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.30 MB PDF)</p>
<a name="pone.0000708.s006" id="pone.0000708.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000708.s006">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Estimation of the numbers of genes in the ancestral species and those of gene gains and losses by the reconciled tree method. See <a href="#pone.0000708.s001">Protocol S1</a>. (A) A species tree. (B) A gene tree. (C) A gene tree for estimating the number of genes α in (A). A diamond represents the divergence between marsupials and placentals. A dashed line indicates a gene loss. (D) A gene tree for estimating the number of genes β in (A). A diamond represents the divergence between rodents and primates. (E) Evolutionary changes of the number of genes inferred from (B). “-1” indicates a gene loss. There are no gene gains in this case.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000708.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.24 MB PDF)</p>
<a name="pone.0000708.s007" id="pone.0000708.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000708.s007">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Names of nodes and branches for (A) <a href="#pone.0000708.s002">Table S1</a> and (B) <a href="#pone.0000708.s003">Table S2</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000708.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.22 MB PDF)</p>
<a name="pone.0000708.s008" id="pone.0000708.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000708.s008">Dataset S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Amino acid sequences of OR genes from six mammalian species. “Oran”, “Modo”, “Bota”, “Cafa”, “Rano”, and “Mamu” represent platypus, opossum, cow, dog, rat, and macaque OR genes, respectively. A gene name with “P” and “T” indicate a pseudogene and a truncated gene, respectively. An asterisk and a slash in an amino acid sequence represent a stop codon and a frameshift mutation, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000708.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(6.30 MB DOC)</p>
<a name="pone.0000708.s009" id="pone.0000708.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000708.s009">Dataset S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Names of functional OR genes belonging to each clade.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000708.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.47 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Yasuhiro Go, Todd Johnson, Tadashi Masuda, So Nakagawa, and Masafumi Nozawa for helpful comments and discussion. We also thank Baylor College of Medicine Human Genome Sequencing Center, the Broad Institute, Agencourt Bioscience, and the Genome Sequencing Center at Washington University of Medicine in St. Louis for providing platypus, opossum, cow, dog, rat, and macaque genome sequences.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MN YN. Performed the experiments: YN. Analyzed the data: YN. Contributed reagents/materials/analysis tools: YN. Wrote the paper: MN YN.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000708-Buck1" id="pone.0000708-Buck1"></a>Buck L, Axel R (1991) A novel multigene family may encode odorant receptors: a molecular basis for odor recognition. Cell  65: 175–187.  <ul class="find" data-citedArticleID="974798" data-doi="10.1016/0092-8674(91)90418-x"><li><a href="http://dx.doi.org/10.1016/0092-8674(91)90418-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+multigene+family+may+encode+odorant+receptors%3A+a+molecular+basis+for+odor+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+multigene+family+may+encode+odorant+receptors%3A+a+molecular+basis+for+odor+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000708-Mombaerts1" id="pone.0000708-Mombaerts1"></a>Mombaerts P (2004) Genes and ligands for odorant, vomeronasal and taste receptors. Nat Rev Neurosci  5: 263–278.  <ul class="find" data-citedArticleID="974826" data-doi="10.1038/nrn1365"><li><a href="http://dx.doi.org/10.1038/nrn1365" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+and+ligands+for+odorant%2C+vomeronasal+and+taste+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+and+ligands+for+odorant%2C+vomeronasal+and+taste+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000708-Niimura1" id="pone.0000708-Niimura1"></a>Niimura Y, Nei M (2006) Evolutionary dynamics of olfactory and other chemosensory receptor genes in vertebrates. J Hum Genet  51: 505–517.  <ul class="find" data-citedArticleID="974844" data-doi="10.1007/s10038-006-0391-8"><li><a href="http://dx.doi.org/10.1007/s10038-006-0391-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+dynamics+of+olfactory+and+other+chemosensory+receptor+genes+in+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+dynamics+of+olfactory+and+other+chemosensory+receptor+genes+in+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000708-Glusman1" id="pone.0000708-Glusman1"></a>Glusman G, Yanai I, Rubin I, Lancet D (2001) The complete human olfactory subgenome. Genome Res  11: 685–702.  <ul class="find" data-citedArticleID="974808" data-doi="10.1101/gr.171001"><li><a href="http://dx.doi.org/10.1101/gr.171001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+human+olfactory+subgenome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+human+olfactory+subgenome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000708-Zozulya1" id="pone.0000708-Zozulya1"></a>Zozulya S, Echeverri F, Nguyen T (2001) The human olfactory receptor repertoire. Genome Biol  2: RESEARCH0018.  <ul class="find" data-citedArticleID="974878" data-doi="10.1186/gb-2001-2-6-research0018"><li><a href="http://dx.doi.org/10.1186/gb-2001-2-6-research0018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+olfactory+receptor+repertoire." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+olfactory+receptor+repertoire.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000708-Niimura2" id="pone.0000708-Niimura2"></a>Niimura Y, Nei M (2003) Evolution of olfactory receptor genes in the human genome. Proc Natl Acad Sci U S A  100: 12235–12240.  <ul class="find" data-citedArticleID="974846" data-doi="10.1073/pnas.1635157100"><li><a href="http://dx.doi.org/10.1073/pnas.1635157100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+olfactory+receptor+genes+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+olfactory+receptor+genes+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000708-Zhang1" id="pone.0000708-Zhang1"></a>Zhang X, Firestein S (2002) The olfactory receptor gene superfamily of the mouse. Nat Neurosci  5: 124–133.  <ul class="find" data-citedArticleID="974876"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+olfactory+receptor+gene+superfamily+of+the+mouse.&amp;auth=&amp;atitle=The+olfactory+receptor+gene+superfamily+of+the+mouse." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+olfactory+receptor+gene+superfamily+of+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+olfactory+receptor+gene+superfamily+of+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000708-Young1" id="pone.0000708-Young1"></a>Young JM, Friedman C, Williams EM, Ross JA, Tonnes-Priddy L, et al.  (2002) Different evolutionary processes shaped the mouse and human olfactory receptor gene families. Hum Mol Genet  11: 535–546.  <ul class="find" data-citedArticleID="974874" data-doi="10.1093/hmg/11.5.535"><li><a href="http://dx.doi.org/10.1093/hmg/11.5.535" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Different+evolutionary+processes+shaped+the+mouse+and+human+olfactory+receptor+gene+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Different+evolutionary+processes+shaped+the+mouse+and+human+olfactory+receptor+gene+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000708-Niimura3" id="pone.0000708-Niimura3"></a>Niimura Y, Nei M (2005) Comparative evolutionary analysis of olfactory receptor gene clusters between humans and mice. Gene  346: 13–21.  <ul class="find" data-citedArticleID="974848" data-doi="10.1016/j.gene.2004.09.025"><li><a href="http://dx.doi.org/10.1016/j.gene.2004.09.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+evolutionary+analysis+of+olfactory+receptor+gene+clusters+between+humans+and+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+evolutionary+analysis+of+olfactory+receptor+gene+clusters+between+humans+and+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000708-Niimura4" id="pone.0000708-Niimura4"></a>Niimura Y, Nei M (2005) Evolutionary dynamics of olfactory receptor genes in fishes and tetrapods. Proc Natl Acad Sci U S A  102: 6039–6044.  <ul class="find" data-citedArticleID="974850" data-doi="10.1073/pnas.0501922102"><li><a href="http://dx.doi.org/10.1073/pnas.0501922102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+dynamics+of+olfactory+receptor+genes+in+fishes+and+tetrapods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+dynamics+of+olfactory+receptor+genes+in+fishes+and+tetrapods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000708-Ngai1" id="pone.0000708-Ngai1"></a>Ngai J, Dowling MM, Buck L, Axel R, Chess A (1993) The family of genes encoding odorant receptors in the channel catfish. Cell  72: 657–666.  <ul class="find" data-citedArticleID="974842" data-doi="10.1016/0092-8674(93)90395-7"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)90395-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+family+of+genes+encoding+odorant+receptors+in+the+channel+catfish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+family+of+genes+encoding+odorant+receptors+in+the+channel+catfish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000708-Nei1" id="pone.0000708-Nei1"></a>Nei M, Gu X, Sitnikova T (1997) Evolution by the birth-and-death process in multigene families of the vertebrate immune system. Proc Natl Acad Sci USA  94: 7799–7806.  <ul class="find" data-citedArticleID="974834" data-doi="10.1073/pnas.94.15.7799"><li><a href="http://dx.doi.org/10.1073/pnas.94.15.7799" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+by+the+birth-and-death+process+in+multigene+families+of+the+vertebrate+immune+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+by+the+birth-and-death+process+in+multigene+families+of+the+vertebrate+immune+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000708-Nei2" id="pone.0000708-Nei2"></a>Nei M, Rooney AP (2005) Concerted and birth-and-death evolution of multigene families. Annu Rev Genet  39: 121–152.  <ul class="find" data-citedArticleID="974836" data-doi="10.1146/annurev.genet.39.073003.112240"><li><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.112240" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Concerted+and+birth-and-death+evolution+of+multigene+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Concerted+and+birth-and-death+evolution+of+multigene+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000708-Quignon1" id="pone.0000708-Quignon1"></a>Quignon P, Giraud M, Rimbault M, Lavigne P, Tacher S, et al.  (2005) The dog and rat olfactory receptor repertoires. Genome Biol  6: R83.  <ul class="find" data-citedArticleID="974860" data-doi="10.1186/gb-2005-6-10-r83"><li><a href="http://dx.doi.org/10.1186/gb-2005-6-10-r83" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dog+and+rat+olfactory+receptor+repertoires." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dog+and+rat+olfactory+receptor+repertoires.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000708-Aloni1" id="pone.0000708-Aloni1"></a>Aloni R, Olender T, Lancet D (2006) Ancient genomic architecture for mammalian olfactory receptor clusters. Genome Biol  7: R88.  <ul class="find" data-citedArticleID="974792" data-doi="10.1186/gb-2006-7-10-r88"><li><a href="http://dx.doi.org/10.1186/gb-2006-7-10-r88" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+genomic+architecture+for+mammalian+olfactory+receptor+clusters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+genomic+architecture+for+mammalian+olfactory+receptor+clusters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000708-Glusman2" id="pone.0000708-Glusman2"></a>Glusman G, Bahar A, Sharon D, Pilpel Y, White J, et al.  (2000) The olfactory receptor gene superfamily: data mining, classification, and nomenclature. Mamm Genome  11: 1016–1023.  <ul class="find" data-citedArticleID="974810" data-doi="10.1007/s003350010196"><li><a href="http://dx.doi.org/10.1007/s003350010196" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+olfactory+receptor+gene+superfamily%3A+data+mining%2C+classification%2C+and+nomenclature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+olfactory+receptor+gene+superfamily%3A+data+mining%2C+classification%2C+and+nomenclature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000708-Grus1" id="pone.0000708-Grus1"></a>Grus WE, Shi P, Zhang YP, Zhang J (2005) Dramatic variation of the vomeronasal pheromone receptor gene repertoire among five orders of placental and marsupial mammals. Proc Natl Acad Sci U S A  102: 5767–5772.  <ul class="find" data-citedArticleID="974816" data-doi="10.1073/pnas.0501589102"><li><a href="http://dx.doi.org/10.1073/pnas.0501589102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dramatic+variation+of+the+vomeronasal+pheromone+receptor+gene+repertoire+among+five+orders+of+placental+and+marsupial+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dramatic+variation+of+the+vomeronasal+pheromone+receptor+gene+repertoire+among+five+orders+of+placental+and+marsupial+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000708-Go1" id="pone.0000708-Go1"></a>Go Y (2006) Lineage-specific expansions and contractions of the bitter taste receptor gene repertoire in vertebrates. Mol Biol Evol  23: 964–972.  <ul class="find" data-citedArticleID="974812" data-doi="10.1093/molbev/msj106"><li><a href="http://dx.doi.org/10.1093/molbev/msj106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lineage-specific+expansions+and+contractions+of+the+bitter+taste+receptor+gene+repertoire+in+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lineage-specific+expansions+and+contractions+of+the+bitter+taste+receptor+gene+repertoire+in+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000708-Demuth1" id="pone.0000708-Demuth1"></a>Demuth JP, Bie TD, Stajich JE, Cristianini N, Hahn MW (2006) The evolution of mammalian gene families. PLoS ONE  1: e85.  <ul class="find" data-citedArticleID="974802" data-doi="10.1371/journal.pone.0000085"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+mammalian+gene+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+mammalian+gene+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000708-Niimura5" id="pone.0000708-Niimura5"></a>Niimura Y, Nei M (2005) Evolutionary changes of the number of olfactory receptor genes in the human and mouse lineages. Gene  346: 23–28.  <ul class="find" data-citedArticleID="974852" data-doi="10.1016/j.gene.2004.09.027"><li><a href="http://dx.doi.org/10.1016/j.gene.2004.09.027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+changes+of+the+number+of+olfactory+receptor+genes+in+the+human+and+mouse+lineages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+changes+of+the+number+of+olfactory+receptor+genes+in+the+human+and+mouse+lineages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000708-Murphy1" id="pone.0000708-Murphy1"></a>Murphy WJ, Eizirik E, O'Brien SJ, Madsen O, Scally M, et al.  (2001) Resolution of the early placental mammal radiation using Bayesian phylogenetics. Science  294: 2348–2351.  <ul class="find" data-citedArticleID="974828" data-doi="10.1126/science.1067179"><li><a href="http://dx.doi.org/10.1126/science.1067179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resolution+of+the+early+placental+mammal+radiation+using+Bayesian+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resolution+of+the+early+placental+mammal+radiation+using+Bayesian+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000708-Springer1" id="pone.0000708-Springer1"></a>Springer MS, Stanhope MJ, Madsen O, de Jong WW (2004) Molecules consolidate the placental mammal tree. Trends Ecol Evol  19: 430–438.  <ul class="find" data-citedArticleID="974870" data-doi="10.1016/j.tree.2004.05.006"><li><a href="http://dx.doi.org/10.1016/j.tree.2004.05.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecules+consolidate+the+placental+mammal+tree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecules+consolidate+the+placental+mammal+tree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000708-Kriegs1" id="pone.0000708-Kriegs1"></a>Kriegs JO, Churakov G, Kiefmann M, Jordan U, Brosius J, et al.  (2006) Retroposed elements as archives for the evolutionary history of placental mammals. PLoS Biol  4: E91.  <ul class="find" data-citedArticleID="974820" data-doi="10.1371/journal.pbio.0040091"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040091" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroposed+elements+as+archives+for+the+evolutionary+history+of+placental+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroposed+elements+as+archives+for+the+evolutionary+history+of+placental+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000708-Kullberg1" id="pone.0000708-Kullberg1"></a>Kullberg M, Nilsson MA, Arnason U, Harley EH, Janke A (2006) Housekeeping genes for phylogenetic analysis of eutherian relationships. Mol Biol Evol  23: 1493–1503.  <ul class="find" data-citedArticleID="974822" data-doi="10.1093/molbev/msl027"><li><a href="http://dx.doi.org/10.1093/molbev/msl027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Housekeeping+genes+for+phylogenetic+analysis+of+eutherian+relationships." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Housekeeping+genes+for+phylogenetic+analysis+of+eutherian+relationships.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000708-Cannarozzi1" id="pone.0000708-Cannarozzi1"></a>Cannarozzi G, Schneider A, Gonnet G (2007) A phylogenomic study of human, dog, and mouse. PLoS Comput Biol  3: e2.  <ul class="find" data-citedArticleID="974800" data-doi="10.1371/journal.pcbi.0030002"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0030002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+phylogenomic+study+of+human%2C+dog%2C+and+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+phylogenomic+study+of+human%2C+dog%2C+and+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000708-Gilad1" id="pone.0000708-Gilad1"></a>Gilad Y, Wiebe V, Przeworski M, Lancet D, Pääbo S (2004) Loss of olfactory receptor genes coincides with the acquisition of full trichromatic vision in primates. PLoS Biol  2: E5.  <ul class="find" data-citedArticleID="974804" data-doi="10.1371/journal.pbio.0020005"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Loss+of+olfactory+receptor+genes+coincides+with+the+acquisition+of+full+trichromatic+vision+in+primates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Loss+of+olfactory+receptor+genes+coincides+with+the+acquisition+of+full+trichromatic+vision+in+primates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000708-Pettigrew1" id="pone.0000708-Pettigrew1"></a>Pettigrew JD (1999) Electroreception in monotremes. J Exp Biol  202: 1447–1454.  <ul class="find" data-citedArticleID="974858"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Electroreception+in+monotremes.&amp;auth=&amp;atitle=Electroreception+in+monotremes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Electroreception+in+monotremes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Electroreception+in+monotremes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000708-Malnic1" id="pone.0000708-Malnic1"></a>Malnic B, Godfrey PA, Buck LB (2004) The human olfactory receptor gene family. Proc Natl Acad Sci U S A  101: 2584–2589.  <ul class="find" data-citedArticleID="974824" data-doi="10.1073/pnas.0307882100"><li><a href="http://dx.doi.org/10.1073/pnas.0307882100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+olfactory+receptor+gene+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+olfactory+receptor+gene+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000708-Shepherd1" id="pone.0000708-Shepherd1"></a>Shepherd GM (2004) The human sense of smell: are we better than we think? PLoS Biol  2: E146.  <ul class="find" data-citedArticleID="974868" data-doi="10.1371/journal.pbio.0020146"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020146" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+sense+of+smell%3A+are+we+better+than+we+think%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+sense+of+smell%3A+are+we+better+than+we+think%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000708-Nei3" id="pone.0000708-Nei3"></a>Nei M (2007) The new mutation theory of phenotypic evolution. Proc Natl Acad Sci U S A.   In press.  <ul class="find" data-citedArticleID="974838"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+new+mutation+theory+of+phenotypic+evolution.&amp;auth=&amp;atitle=The+new+mutation+theory+of+phenotypic+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+new+mutation+theory+of+phenotypic+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+new+mutation+theory+of+phenotypic+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000708-Nozawa1" id="pone.0000708-Nozawa1"></a>Nozawa M, Nei M (2007) Evolutionary dynamics of olfactory receptor genes in <em>Drosophila</em> species. Proc Natl Acad Sci U S A  104: 7122–7127.  <ul class="find" data-citedArticleID="974854" data-doi="10.1073/pnas.0702133104"><li><a href="http://dx.doi.org/10.1073/pnas.0702133104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+dynamics+of+olfactory+receptor+genes+in+Drosophila+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+dynamics+of+olfactory+receptor+genes+in+Drosophila+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000708-Bargmann1" id="pone.0000708-Bargmann1"></a>Bargmann CI (2006) Comparative chemosensation from receptors to ecology. Nature  444: 295–301.  <ul class="find" data-citedArticleID="974796" data-doi="10.1038/nature05402"><li><a href="http://dx.doi.org/10.1038/nature05402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+chemosensation+from+receptors+to+ecology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+chemosensation+from+receptors+to+ecology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000708-Ray1" id="pone.0000708-Ray1"></a>Ray A, van Naters WG, Shiraiwa T, Carlson JR (2007) Mechanisms of odor receptor gene choice in <em>Drosophila</em>. Neuron  53: 353–369.  <ul class="find" data-citedArticleID="974862" data-doi="10.1016/j.neuron.2006.12.010"><li><a href="http://dx.doi.org/10.1016/j.neuron.2006.12.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+odor+receptor+gene+choice+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+odor+receptor+gene+choice+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000708-Serizawa1" id="pone.0000708-Serizawa1"></a>Serizawa S, Miyamichi K, Nakatani H, Suzuki M, Saito M, et al.  (2003) Negative feedback regulation ensures the one receptor-one olfactory neuron rule in mouse. Science  302: 2088–2094.  <ul class="find" data-citedArticleID="974866" data-doi="10.1126/science.1089122"><li><a href="http://dx.doi.org/10.1126/science.1089122" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Negative+feedback+regulation+ensures+the+one+receptor-one+olfactory+neuron+rule+in+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Negative+feedback+regulation+ensures+the+one+receptor-one+olfactory+neuron+rule+in+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000708-Altschul1" id="pone.0000708-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="974794" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000708-Katoh1" id="pone.0000708-Katoh1"></a>Katoh K, Kuma K, Toh H, Miyata T (2005) MAFFT version 5: improvement in accuracy of multiple sequence alignment. Nucleic Acids Res  33: 511–518.  <ul class="find" data-citedArticleID="974818" data-doi="10.1093/nar/gki198"><li><a href="http://dx.doi.org/10.1093/nar/gki198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAFFT+version+5%3A+improvement+in+accuracy+of+multiple+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAFFT+version+5%3A+improvement+in+accuracy+of+multiple+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000708-Goodman1" id="pone.0000708-Goodman1"></a>Goodman M, Czelusniak J, Moore GW, Romero-Herrera AE, Matsuda G (1979) Fitting the gene lineage into its species lineage, a parsimony strategy illustrated by cladograms constructed from globin sequences. Syst Zool  28: 132–168.  <ul class="find" data-citedArticleID="974814" data-doi="10.2307/2412519"><li><a href="http://dx.doi.org/10.2307/2412519" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitting+the+gene+lineage+into+its+species+lineage%2C+a+parsimony+strategy+illustrated+by+cladograms+constructed+from+globin+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitting+the+gene+lineage+into+its+species+lineage%2C+a+parsimony+strategy+illustrated+by+cladograms+constructed+from+globin+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000708-Page1" id="pone.0000708-Page1"></a>Page RD, Charleston MA (1997) From gene to organismal phylogeny: reconciled trees and the gene tree/species tree problem. Mol Phylogenet Evol  7: 231–240.  <ul class="find" data-citedArticleID="974856" data-doi="10.1006/mpev.1996.0390"><li><a href="http://dx.doi.org/10.1006/mpev.1996.0390" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+gene+to+organismal+phylogeny%3A+reconciled+trees+and+the+gene+tree%2Fspecies+tree+problem." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+gene+to+organismal+phylogeny%3A+reconciled+trees+and+the+gene+tree%2Fspecies+tree+problem.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000708-Nam1" id="pone.0000708-Nam1"></a>Nam J, Nei M (2005) Evolutionary change of the numbers of homeobox genes in bilateral animals. Mol Biol Evol  22: 2386–2394.  <ul class="find" data-citedArticleID="974832" data-doi="10.1093/molbev/msi229"><li><a href="http://dx.doi.org/10.1093/molbev/msi229" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+change+of+the+numbers+of+homeobox+genes+in+bilateral+animals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+change+of+the+numbers+of+homeobox+genes+in+bilateral+animals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000708-Nei4" id="pone.0000708-Nei4"></a>Nei M, Kumar S (2000) Molecular evolution and phylogenetics. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0000708-Saitou1" id="pone.0000708-Saitou1"></a>Saitou N, Nei M (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol  4: 406–425.  <ul class="find" data-citedArticleID="974864"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.&amp;auth=&amp;atitle=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000708-Takezaki1" id="pone.0000708-Takezaki1"></a>Takezaki N, Rzhetsky A, Nei M (1995) Phylogenetic test of molecular clock and linearized trees. Mol Biol Evol  12: 823–833.  <ul class="find" data-citedArticleID="974872"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phylogenetic+test+of+molecular+clock+and+linearized+trees.&amp;auth=&amp;atitle=Phylogenetic+test+of+molecular+clock+and+linearized+trees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+test+of+molecular+clock+and+linearized+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+test+of+molecular+clock+and+linearized+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000708-Murphy2" id="pone.0000708-Murphy2"></a>Murphy WJ, Pevzner PA, O'Brien SJ (2004) Mammalian phylogenomics comes of age. Trends Genet  20: 631–639.  <ul class="find" data-citedArticleID="974830" data-doi="10.1016/j.tig.2004.09.005"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.09.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+phylogenomics+comes+of+age." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+phylogenomics+comes+of+age.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000708-Glazko1" id="pone.0000708-Glazko1"></a>Glazko GV, Nei M (2003) Estimation of divergence times for major lineages of primate species. Mol Biol Evol  20: 424–434.  <ul class="find" data-citedArticleID="974806" data-doi="10.1093/molbev/msg050"><li><a href="http://dx.doi.org/10.1093/molbev/msg050" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+divergence+times+for+major+lineages+of+primate+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+divergence+times+for+major+lineages+of+primate+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.XML" value="103311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.PDF" value="312513"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.t001.PNG_L" value="101284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.t001.PNG_M" value="56705"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.t001.PNG_S" value="9225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.t001.TIF" value="461426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.t001.PNG_I" value="17267"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.t002.PNG_L" value="249659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.t002.PNG_M" value="125388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.t002.PNG_S" value="18393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.t002.TIF" value="1393374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.t002.PNG_I" value="45348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.g001.PNG_L" value="972233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.g001.PNG_M" value="132544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.g001.PNG_S" value="14992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.g001.TIF" value="2423854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.g001.PNG_I" value="71411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.g002.PNG_L" value="777337"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.g002.PNG_M" value="115539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.g002.PNG_S" value="17277"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.g002.TIF" value="1623340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.g002.PNG_I" value="122239"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.s001.DOC" value="43008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.s002.PDF" value="30129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.s003.PDF" value="26326"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.s004.PDF" value="471172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.s005.PDF" value="296309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.s006.PDF" value="236413"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.s007.PDF" value="223763"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.s008.DOC" value="6303232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000708.s009.DOC" value="470528"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000708&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000708" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000708&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000708&volume=&issue=&title=Extensive Gains and Losses of Olfactory Receptor Genes in Mammalian Evolution&author_name=Yoshihito%20Niimura%2C%20Masatoshi%20Nei&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000708" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000708" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000708" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000708&amp;t=Extensive%20Gains%20and%20Losses%20of%20Olfactory%20Receptor%20Genes%20in%20Mammalian%20Evolution" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000708&title=Extensive%20Gains%20and%20Losses%20of%20Olfactory%20Receptor%20Genes%20in%20Mammalian%20Evolution&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000708&amp;title=Extensive%20Gains%20and%20Losses%20of%20Olfactory%20Receptor%20Genes%20in%20Mammalian%20Evolution" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000708" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000708&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Extensive Gains and Losses of Olfactory Receptor Genes in Mammalian Evolution';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000708';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Extensive%20Gains%20and%20Losses%20of%20Olfactory%20Receptor%20Genes%20in%20Mammalian%20Evolution http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000708" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000708" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+phylogenetics%22" title="Search for articles in the subject area:'Animal phylogenetics'"><div class="flagText">Animal phylogenetics</div></a>
              <div data-categoryid="20335" data-articleid="24254"
                   data-categoryname="Animal phylogenetics"
                   class="flagImage" title="Flag 'Animal phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="24254"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24254"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="24254"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammals%22" title="Search for articles in the subject area:'Mammals'"><div class="flagText">Mammals</div></a>
              <div data-categoryid="43131" data-articleid="24254"
                   data-categoryname="Mammals"
                   class="flagImage" title="Flag 'Mammals' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Opossums%22" title="Search for articles in the subject area:'Opossums'"><div class="flagText">Opossums</div></a>
              <div data-categoryid="43761" data-articleid="24254"
                   data-categoryname="Opossums"
                   class="flagImage" title="Flag 'Opossums' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24254"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Pseudogenes%22" title="Search for articles in the subject area:'Pseudogenes'"><div class="flagText">Pseudogenes</div></a>
              <div data-categoryid="33795" data-articleid="24254"
                   data-categoryname="Pseudogenes"
                   class="flagImage" title="Flag 'Pseudogenes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5783'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4730'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4750&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
