

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Post-Synaptic Scaffold at the Origin of the Animal Kingdom</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000506"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000506" />

    <meta name="citation_title" content="A Post-Synaptic Scaffold at the Origin of the Animal Kingdom"/>
    <meta itemprop="name" content="A Post-Synaptic Scaffold at the Origin of the Animal Kingdom"/>

      <meta name="citation_author" content="Onur Sakarya"/>
            <meta name="citation_author_institution" content="Neuroscience Research Institute, University of California, Santa Barbara, Santa Barbara, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of California, Santa Barbara, Santa Barbara, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Computer Science, University of California, Santa Barbara, Santa Barbara, California, United States of America"/>
      <meta name="citation_author" content="Kathryn A. Armstrong"/>
            <meta name="citation_author_institution" content="Biological Engineering Division, Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Maja Adamska"/>
            <meta name="citation_author_institution" content="School of Integrative Biology, University of Queensland, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="Marcin Adamski"/>
            <meta name="citation_author_institution" content="School of Integrative Biology, University of Queensland, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="I-Fan Wang"/>
            <meta name="citation_author_institution" content="Neuroscience Research Institute, University of California, Santa Barbara, Santa Barbara, California, United States of America"/>
      <meta name="citation_author" content="Bruce Tidor"/>
            <meta name="citation_author_institution" content="Biological Engineering Division, Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Bernard M. Degnan"/>
            <meta name="citation_author_institution" content="School of Integrative Biology, University of Queensland, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="Todd H. Oakley"/>
            <meta name="citation_author_institution" content="Department of Ecology, Evolution and Marine Biology, University of California, Santa Barbara, Santa Barbara, California, United States of America"/>
      <meta name="citation_author" content="Kenneth S. Kosik"/>
            <meta name="citation_author_institution" content="Neuroscience Research Institute, University of California, Santa Barbara, Santa Barbara, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of California, Santa Barbara, Santa Barbara, California, United States of America"/>

    <meta name="citation_date" content="2007/6/6"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000506.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e506"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Exaptation; a missing term in the science of form Paleobiology; citation_author=SJ Gould; citation_author=ES Vrba; citation_volume=8; citation_number=1; citation_pages=4-15; citation_date=1982; " />
      <meta name="citation_reference" content="citation_author=JNA Hooper; citation_author=RWM Van Soest; citation_number=2; citation_date=2002; citation_publisher=Kluwer Academic/Plenum Publishers; " />
      <meta name="citation_reference" content="citation_title=Phylogenomics and the reconstruction of the tree of life.; citation_author=F Delsuc; citation_author=H Brinkmann; citation_author=H Philippe; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=3; citation_pages=361-375; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial Genome of the Homoscleromorph Oscarella carmela (Porifera, Demospongiae) Reveals Unexpected Complexity in the Common Ancestor of Sponges and Other Animals.; citation_author=X Wang; citation_author=DV Lavrov; citation_journal_title=Molecular biology and evolution; citation_volume=24; citation_number=4; citation_pages=363-373; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial diversity of early-branching metazoa is revealed by the complete mt genome of a haplosclerid demosponge.; citation_author=D Erpenbeck; citation_author=O Voigt; citation_author=M Adamski; citation_author=M Adamska; citation_author=JN Hooper; citation_journal_title=Molecular biology and evolution; citation_volume=24; citation_number=5; citation_pages=19-22; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Sponge phylogeny, animal monophyly, and the origin of the nervous system: 18S rRNA evidence.; citation_author=T Cavalier-Smith; citation_author=E Chao; citation_author=N Boury-Esnault; citation_author=J Vacelet; citation_journal_title=Can J Zool; citation_volume=74; citation_number=6; citation_pages=2031-2045; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Sponge paraphyly and the origin of Metazoa.; citation_author=C Borchiellini; citation_author=M Manuel; citation_author=E Alivon; citation_author=N Boury-Esnault; citation_author=J Vacelet; citation_journal_title=J Evol Biol; citation_volume=(14); citation_number=7; citation_pages=171-179; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Origin of the Eumetazoa: testing ecological predictions of molecular clocks against the Proterozoic fossil record.; citation_author=KJ Peterson; citation_author=NJ Butterfield; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=102; citation_number=8; citation_pages=9547-9552; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Proteomic analysis of NMDA receptor-adhesion protein signaling complexes.; citation_author=H Husi; citation_author=MA Ward; citation_author=JS Choudhary; citation_author=WP Blackstock; citation_author=SG Grant; citation_journal_title=Nature neuroscience; citation_volume=3; citation_number=9; citation_pages=661-669; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Semiquantitative proteomic analysis of rat forebrain postsynaptic density fractions by mass spectrometry.; citation_author=J Peng; citation_author=MJ Kim; citation_author=D Cheng; citation_author=DM Duong; citation_author=SP Gygi; citation_journal_title=The Journal of biological chemistry; citation_volume=279; citation_number=10; citation_pages=21003-21011; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The synapse proteome and phosphoproteome: a new paradigm for synapse biology.; citation_author=SG Grant; citation_journal_title=Biochemical Society transactions; citation_volume=34; citation_number=11; citation_pages=59-63; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The nervous systems of cnidarians.; citation_author=CJ Grimmelikhuijzen; citation_author=JA Westfall; citation_journal_title=Exs; citation_volume=72; citation_number=12; citation_pages=7-24; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Ion channels and synaptic organization: analysis of the Drosophila genome.; citation_author=JT Littleton; citation_author=B Ganetzky; citation_journal_title=Neuron; citation_volume=26; citation_number=13; citation_pages=35-43; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Crystal structures of a complexed and peptide-free membrane protein-binding domain: molecular basis of peptide recognition by PDZ.; citation_author=DA Doyle; citation_author=A Lee; citation_author=J Lewis; citation_author=E Kim; citation_author=M Sheng; citation_journal_title=Cell; citation_volume=85; citation_number=14; citation_pages=1067-1076; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The Protein Data Bank.; citation_author=HM Berman; citation_author=J Westbrook; citation_author=Z Feng; citation_author=G Gilliland; citation_author=TN Bhat; citation_journal_title=Nucleic acids research; citation_volume=28; citation_number=15; citation_pages=235-242; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=(To be published (PDB id: 2I1N)) PDB id: 2I1N. Crystal structure of the 1st PDZ domain of Human DLG3; citation_author=AP Turnbull; citation_author=C Phillips; citation_author=G Bunkoczi; citation_author=J Debreczeni; citation_author=E Ugochukwu; citation_number=16; " />
      <meta name="citation_reference" content="citation_title=(To be published (PDB id: 2G2L)) PDB id: 2G2L. Crystal Structure of the Second PDZ Domain of SAP97 in Complex with a GluR-A C-terminal Peptide; citation_author=I Von Ossowski; citation_author=E Oksanen; citation_author=C Cai; citation_author=M Sundberg; citation_author=A Goldman; citation_number=17; " />
      <meta name="citation_reference" content="citation_title=Early evolution of animal cell signaling and adhesion genes.; citation_author=SA Nichols; citation_author=W Dirks; citation_author=JS Pearse; citation_author=N King; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=103; citation_number=18; citation_pages=12451-12456; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The binding of the PDZ tandem of syntenin to target proteins.; citation_author=J Grembecka; citation_author=T Cierpicki; citation_author=Y Devedjiev; citation_author=U Derewenda; citation_author=BS Kang; citation_journal_title=Biochemistry; citation_volume=45; citation_number=19; citation_pages=3674-3683; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Intramolecular interactions regulate SAP97 binding to GKAP.; citation_author=H Wu; citation_author=C Reissner; citation_author=S Kuhlendahl; citation_author=B Coblentz; citation_author=S Reuver; citation_journal_title=The EMBO journal; citation_volume=19; citation_number=20; citation_pages=5740-5751; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Developmental expression of transcription factor genes in a demosponge: insights into the origin of metazoan multicellularity.; citation_author=C Larroux; citation_author=B Fahey; citation_author=D Liubicich; citation_author=VF Hinman; citation_author=M Gauthier; citation_journal_title=Evolution & development; citation_volume=8; citation_number=21; citation_pages=150-173; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Conduction system in a sponge.; citation_author=ID Lawn; citation_author=GO Mackie; citation_author=G Silver; citation_journal_title=Science; citation_volume=211; citation_number=22; citation_pages=1169-1171; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Impulse conduction in a sponge.; citation_author=SP Leys; citation_author=GO Mackie; citation_author=RW Meech; citation_journal_title=J Exp Biol; citation_volume=202; citation_number=23; citation_pages=1139-1150; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Cytological basis of photoresponsive behavior in a sponge larva.; citation_author=SP Leys; citation_author=BM Degnan; citation_journal_title=The Biological bulletin; citation_volume=201; citation_number=24; citation_pages=323-338; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Neurexin-neuroligin signaling in synapse development.; citation_author=AM Craig; citation_author=Y Kang; citation_journal_title=Current opinion in neurobiology; citation_volume=17; citation_number=25; citation_pages=43-52; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Orthologs, paralogs, and evolutionary genomics.; citation_author=EV Koonin; citation_journal_title=Annual review of genetics; citation_volume=39; citation_number=26; citation_pages=309-338; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Orthology, paralogy and proposed classification for paralog subtypes.; citation_author=EL Sonnhammer; citation_author=EV Koonin; citation_journal_title=Trends Genet; citation_volume=18; citation_number=27; citation_pages=619-620; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=StellaBase: the Nematostella vectensis Genomics Database.; citation_author=JC Sullivan; citation_author=JF Ryan; citation_author=JA Watson; citation_author=J Webb; citation_author=JC Mullikin; citation_journal_title=Nucleic acids research; citation_volume=34(Database issue); citation_number=28; citation_pages=D495-499; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic acids research; citation_volume=25; citation_number=29; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A greedy algorithm for aligning DNA sequences.; citation_author=Z Zhang; citation_author=S Schwartz; citation_author=L Wagner; citation_author=W Miller; citation_journal_title=J Comput Biol; citation_volume=7; citation_number=30; citation_pages=203-214; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Consed: a graphical tool for sequence finishing.; citation_author=D Gordon; citation_author=C Abajian; citation_author=P Green; citation_journal_title=Genome research; citation_volume=8; citation_number=31; citation_pages=195-202; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=PCAP: a whole-genome assembly program.; citation_author=X Huang; citation_author=J Wang; citation_author=S Aluru; citation_author=SP Yang; citation_author=L Hillier; citation_journal_title=Genome research; citation_volume=13; citation_number=32; citation_pages=2164-2170; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Computational inference of homologous gene structures in the human genome.; citation_author=RF Yeh; citation_author=LP Lim; citation_author=CB Burge; citation_journal_title=Genome research; citation_volume=11; citation_number=33; citation_pages=803-816; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Prediction of complete gene structures in human genomic DNA.; citation_author=C Burge; citation_author=S Karlin; citation_journal_title=Journal of molecular biology; citation_volume=268; citation_number=34; citation_pages=78-94; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Conflicting phylogenetic signals at the base of the metazoan tree.; citation_author=A Rokas; citation_author=N King; citation_author=J Finnerty; citation_author=SB Carroll; citation_journal_title=Evolution & development; citation_volume=5; citation_number=35; citation_pages=346-359; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=SMART, a simple modular architecture research tool: identification of signaling domains.; citation_author=J Schultz; citation_author=F Milpetz; citation_author=P Bork; citation_author=CP Ponting; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=95; citation_number=36; citation_pages=5857-5864; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The Pfam protein families database.; citation_author=A Bateman; citation_author=E Birney; citation_author=L Cerruti; citation_author=R Durbin; citation_author=L Etwiller; citation_journal_title=Nucleic acids research; citation_volume=30; citation_number=37; citation_pages=276-280; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic acids research; citation_volume=22; citation_number=38; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=T-Coffee: A novel method for fast and accurate multiple sequence alignment.; citation_author=C Notredame; citation_author=DG Higgins; citation_author=J Heringa; citation_journal_title=Journal of molecular biology; citation_volume=302; citation_number=39; citation_pages=205-217; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=PAUP*: Phylogenetic Analylsis Using Parsimony (* and Other Methods): Sinauer Associates, Sunderland, MA.; citation_author=D Swofford; citation_number=40; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Systematic biology; citation_volume=52; citation_number=41; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics (Oxford, England); citation_volume=19; citation_number=42; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=ProtTest: selection of best-fit models of protein evolution.; citation_author=F Abascal; citation_author=R Zardoya; citation_author=D Posada; citation_journal_title=Bioinformatics (Oxford, England); citation_volume=21; citation_number=43; citation_pages=2104-2105; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A hybrid micro-macroevolutionary approach to gene tree reconstruction.; citation_author=D Durand; citation_author=BV Halldorsson; citation_author=B Vernot; citation_journal_title=J Comput Biol; citation_volume=13; citation_number=44; citation_pages=320-335; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform.; citation_author=K Katoh; citation_author=K Misawa; citation_author=K Kuma; citation_author=T Miyata; citation_journal_title=Nucleic acids research; citation_volume=30; citation_number=45; citation_pages=3059-3066; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Polar hydrogen positions in proteins: empirical energy placement and neutron diffraction comparison.; citation_author=AT Brunger; citation_author=M Karplus; citation_journal_title=Proteins; citation_volume=4; citation_number=46; citation_pages=148-156; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=All-Atom Empirical Potential for Molecular Modeling and Dynamics Studies of Proteins.; citation_author=AD MacKerell; citation_author=B D; citation_author=M Bellott; citation_author=RL Dunbrack; citation_author=JD Evanseck; citation_journal_title=J Phys Chem B; citation_volume=102; citation_number=47; citation_pages=3586-3616; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=CHARMM: A program for macromolecular energy, minimization, and dynamics calculations.; citation_author=BR Brooks; citation_author=RE Bruccoleri; citation_author=BD Olafson; citation_author=DJ States; citation_author=S Swaminathan; citation_journal_title=J Comput Chem; citation_volume=4; citation_number=48; citation_pages=187-217; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Comparative protein modeling by satisfaction of spatial restraints.; citation_author=A Sali; citation_journal_title=Molecular medicine today; citation_volume=1; citation_number=49; citation_pages=270-277; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=PROCHECK: a program to check the stereochemical quality of protein structures.; citation_author=RA Laskowski; citation_author=MW MacArthur; citation_author=DS Moss; citation_author=JM Thornton; citation_journal_title=J Appl Cryst; citation_volume=26; citation_number=50; citation_pages=283-291; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Accurate calculation of hydration free-energies using macroscopic solvent models.; citation_author=D Sitkoff; citation_author=KA Sharp; citation_author=B Honig; citation_journal_title=J Phys Chem; citation_volume=98; citation_number=51; citation_pages=1978-1988; citation_date=1994; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000506" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Post-Synaptic Scaffold at the Origin of the Animal Kingdom"/>
    <meta name="twitter:description" content="BackgroundThe evolution of complex sub-cellular structures such as the synapse requires the assembly of multiple proteins, each conferring added functionality to the integrated structure. Tracking the early evolution of synapses has not been possible without genomic information from the earliest branching animals. As the closest extant relatives to the Eumetazoa, Porifera (sponges) represent a pivotal group for understanding the evolution of nervous systems, because sponges lack neurons with clearly recognizable synapses, in contrast to eumetazoan animals.Methodology/Principal FindingsWe show that the genome of the demosponge Amphimedon queenslandica possesses a nearly complete set of post-synaptic protein homologs whose conserved interaction motifs suggest assembly into a complex structure. In the critical synaptic scaffold gene, dlg, residues that make hydrogen bonds and van der Waals interactions with the PDZ ligand are 100% conserved between sponge and human, as is the motif organization of the scaffolds. Expression in Amphimedon of multiple post-synaptic gene homologs in larval flask cells further supports the existence of an assembled structure. Among the few post-synaptic genes absent from Amphimedon, but present in Eumetazoa, are receptor genes including the entire ionotropic glutamate receptor family.Conclusions/SignificanceHighly conserved protein interaction motifs and co-expression in sponges of multiple proteins whose homologs interact in eumetazoan synapses indicate that a complex protein scaffold was present at the origin of animals, perhaps predating nervous systems. A relatively small number of crucial innovations to this pre-existing structure may represent the founding changes that led to a post-synaptic element."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000506.g003"/>

  <meta property="og:title" content="A Post-Synaptic Scaffold at the Origin of the Animal Kingdom" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000506" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2098'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1452'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8864&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000506">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000506" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000506&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Post-Synaptic Scaffold at the Origin of the Animal Kingdom
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Onur Sakarya
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Onur Sakarya, Kathryn A. Armstrong</p>

              
              

                <p>Affiliations:
                  Neuroscience Research Institute, University of California, Santa Barbara, Santa Barbara, California, United States of America, 
                  Department of Molecular, Cellular and Developmental Biology, University of California, Santa Barbara, Santa Barbara, California, United States of America, 
                  Department of Computer Science, University of California, Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kathryn A. Armstrong
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Onur Sakarya, Kathryn A. Armstrong</p>

              
              

                <p>Affiliation:
                  Biological Engineering Division, Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maja Adamska, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Integrative Biology, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marcin Adamski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Integrative Biology, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            I-Fan Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Neuroscience Research Institute, University of California, Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bruce Tidor, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biological Engineering Division, Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bernard M. Degnan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Integrative Biology, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Todd H. Oakley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Ecology, Evolution and Marine Biology, University of California, Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kenneth S. Kosik
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kosik@lifesci.ucsb.edu">kosik@lifesci.ucsb.edu</a></p>

                <p>Affiliations:
                  Neuroscience Research Institute, University of California, Santa Barbara, Santa Barbara, California, United States of America, 
                  Department of Molecular, Cellular and Developmental Biology, University of California, Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 06, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000506</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000506" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000506" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000506" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000506" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000506-g001" data-doi="info:doi/10.1371/journal.pone.0000506" data-uri="info:doi/10.1371/journal.pone.0000506.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000506-t001" data-doi="info:doi/10.1371/journal.pone.0000506" data-uri="info:doi/10.1371/journal.pone.0000506.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000506-g002" data-doi="info:doi/10.1371/journal.pone.0000506" data-uri="info:doi/10.1371/journal.pone.0000506.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000506-g003" data-doi="info:doi/10.1371/journal.pone.0000506" data-uri="info:doi/10.1371/journal.pone.0000506.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000506">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000506" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The evolution of complex sub-cellular structures such as the synapse requires the assembly of multiple proteins, each conferring added functionality to the integrated structure. Tracking the early evolution of synapses has not been possible without genomic information from the earliest branching animals. As the closest extant relatives to the Eumetazoa, Porifera (sponges) represent a pivotal group for understanding the evolution of nervous systems, because sponges lack neurons with clearly recognizable synapses, in contrast to eumetazoan animals.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We show that the genome of the demosponge <em>Amphimedon queenslandica</em> possesses a nearly complete set of post-synaptic protein homologs whose conserved interaction motifs suggest assembly into a complex structure. In the critical synaptic scaffold gene, <em>dlg</em>, residues that make hydrogen bonds and van der Waals interactions with the PDZ ligand are 100% conserved between sponge and human, as is the motif organization of the scaffolds. Expression in <em>Amphimedon</em> of multiple post-synaptic gene homologs in larval flask cells further supports the existence of an assembled structure. Among the few post-synaptic genes absent from <em>Amphimedon</em>, but present in Eumetazoa, are receptor genes including the entire ionotropic glutamate receptor family.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Highly conserved protein interaction motifs and co-expression in sponges of multiple proteins whose homologs interact in eumetazoan synapses indicate that a complex protein scaffold was present at the origin of animals, perhaps predating nervous systems. A relatively small number of crucial innovations to this pre-existing structure may represent the founding changes that led to a post-synaptic element.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Sakarya O, Armstrong KA, Adamska M, Adamski M, Wang I-F, et al.  (2007) A Post-Synaptic Scaffold at the Origin of the Animal Kingdom. PLoS ONE 2(6):
          e506.
            doi:10.1371/journal.pone.0000506</p><p><strong>Academic Editor: </strong>Leslie Vosshall, The Rockefeller University, United States of America</p><p><strong>Received:</strong> March 13, 2007; <strong>Accepted:</strong> May 3, 2007; <strong>Published:</strong> June 6, 2007</p><p><strong>Copyright:</strong> © 2007 Sakarya et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by Australian Research Council grants to B.M.D.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A fundamental question in biology is how complex, integrated traits like nervous systems originated. One approach to this question involves analyzing individually the numerous components of a defining structure, such as the post-synaptic proteins of nervous systems, to determine which – if any – predate an integrated nervous system. These exapted <a href="#pone.0000506-Gould1">[1]</a> components, defined as a biologic unit originating with a function other than that for which it was later selected, can be revealed by comparative genomics of animals with nervous systems and their closest relatives that lack them. Furthermore, integration of the exapted components as an assembled structure may be inferred by the conservation of protein-protein interaction motifs between proteins of modern nervous systems and those of early branching animals that lack nervous systems.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Porifera (sponges) lack organs and nervous systems and possess a limited number of discrete cell types <a href="#pone.0000506-Hooper1">[2]</a>. They lie in one of at least two highly informative phylogenetic positions with regard to the last common ancestor of all animals. One phylogenetic hypothesis, favored mostly by mitochondrial genome data, but known in other taxa to produce a misleading phylogenetic signal <a href="#pone.0000506-Delsuc1">[3]</a>, is that sponges and Cnidarians form part of a monophyletic group sister to Bilaterians <a href="#pone.0000506-Wang1">[4]</a>, <a href="#pone.0000506-Erpenbeck1">[5]</a>. Under a second phylogenetic hypothesis supported by a large amount of molecular and morphological evidence, the siliceous (demosponges + hexactinellids) and calcareous sponge lineages are the earliest branches off the main metazoan tree <a href="#pone.0000506-CavalierSmith1">[6]</a>–<a href="#pone.0000506-Peterson1">[8]</a>. Under either phylogenetic hypothesis, shared features between sponges and bilaterians likely represent features of ancestral animals. Under the second phylogenetic hypothesis, sponge genomes additionally can reveal shared features that predate nervous systems and cell-specific adaptations, such as the synaptic junction.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A core structure of the post-synaptic complex is the post-synaptic density, a membrane region specialized for signaling and plasticity. While a definitive proteomic analysis of post-synaptic components does not exist, the enumerated proteins range from 77 to ~1000 <a href="#pone.0000506-Husi1">[9]</a>–<a href="#pone.0000506-Grant1">[11]</a>, boundaries which probably represent under- and over-estimates. These reports describe multiple isoforms of receptors, channels, adaptors, scaffolds, and proteins involved in adhesion, signaling, translation, and the cytoskeleton. Orthologous family members of proteins found in the post-synaptic complex are present in many animals; thus one can track this gene set over evolutionary time. Critical to the assembly of these proteins into a functional structure are protein-protein interaction domains. Among synaptic post-synaptic proteins the PDZ domain (<strong>p</strong>ost synaptic density protein (PSD95), <strong>D</strong>rosophila disc large tumor suppressor (<em>dlg</em>), and <strong>z</strong>o-1 protein) is a highly versatile interaction motif that connects many of the proteins in the post-synaptic junction. PDZ domains consist of 80–90 amino-acids and proteins containing these domains are found in bacteria, yeast, plants, and animals.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Recently, genomes from two representative organisms at pivotal positions in nervous system evolution have become available. They are the demosponge <em>Amphimedon queenslandica</em> (formerly known as <em>Reniera</em> sp.), which lacks neurons and a nervous system, and the cnidarian <em>Nematostella vectensis</em>, which possesses a nerve net that has condensed at some locations into plexuses and nerve tracts <a href="#pone.0000506-Grimmelikhuijzen1">[12]</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Post-synaptic Genes in Early Branching Animals</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We assembled phylogenies for 36 gene families of the post-synaptic excitatory vertebrate complex (<a href="#pone.0000506.s001">Figure S1</a>). The genomes of two basal metazoans were surveyed, <em>Amphimedon queenslandica</em> and <em>Nematostella vectensis</em>, as well as genes from <em>Drosophila melanogaster</em>, as a representative protostome, and <em>Homo sapiens</em>, as a representative deuterostome. A surprisingly large number of vertebrate post-synaptic gene homologs are present in the sponge and nearly the entire gene set is present in <em>Nematostella</em> (<a href="#pone-0000506-g001">Figure 1</a>). Furthermore, the domain architecture of all those proteins is highly conserved in the investigated animal lineages (<a href="#pone.0000506.s002">Figure S2</a>). Based on conservation of established binding domains, this set of genes appears capable of assembling their products into a complete sub-synaptic scaffold layer. For example, the <em>dlg</em> gene is present as a single copy and includes an L27 domain that has been lost from mammalian <em>dlg</em> paralogs except SAP-97. Conservation among PDZ ligand sequences is displayed in <a href="#pone-0000506-t001">Table 1</a>. For comparisons of the intron-exon structure of sponge <em>dlg</em> gene with human SAP-97, see <a href="#pone.0000506.s008">Table S2</a>. As shown in <a href="#pone.0000506.s009">Table S3</a>, most of these post-synaptic families do not have orthologs in yeast (<em>Saccharomyces cerevisiae</em>), dicty (<em>Dictyostelium discoideum</em>) and plants (<em>Arabidopsis thaliana</em> and <em>Oryza sativa</em>).</p>
<div class="figure" id="pone-0000506-g001"><div class="img"><a name="pone-0000506-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000506" data-uri="info:doi/10.1371/journal.pone.0000506.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000506.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000506.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Origination periods of post-synaptic genes.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>One possible configuration of the post-synaptic genes based on the known organization of the post-synaptic junction is illustrated. Each color represents the origination period (figure inset) of the gene family inferred from phylogenetic analyses (<a href="#pone.0000506.s001">Figure S1</a>). As further evidence for orthology, domain architectures of selected gene family members were compared (<a href="#pone.0000506.s002">Figure S2</a>). NCBI accession numbers for each gene family member are provided in <a href="#pone.0000506.s007">Table S1</a>. Some gene families may have been lost from the investigated genomes and originated with an earlier ancestor than shown. Question mark indicates insufficient traces to confirm this PDZ domain.</p>
<span>doi:10.1371/journal.pone.0000506.g001</span></div><div class="figure" id="pone-0000506-t001"><div class="img"><a name="pone-0000506-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000506" data-uri="info:doi/10.1371/journal.pone.0000506.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000506.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000506.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>PDZ ligand conservation in animal kingdom.</span></strong></p><span>doi:10.1371/journal.pone.0000506.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The main distinction among the post-synaptic gene set in the poriferan and cnidarian species are genes that encode excitatory post-synaptic receptors. Among the post-synaptic receptor genes absent from the <em>Amphimedon</em> and non-animal eukaryotic genomes such as yeast, <em>Tetrahymena</em>, and <em>Dictyostelium</em> are the Shaker type voltage gated K+ channel, neuroligin, and iGluRs (ionotropic glutamate receptors consisting of NMDA/AMPA/Kainate and Delta receptors). Orthologs of these genes are found in <em>Nematostella</em>. In the case of the iGluRs, <em>Nematostella</em> has a diversity of receptor subtypes comparable to human. At least 11 iGluRs are present in <em>Nematostella</em> compared to 18 in human and 20 to 30 in fly <a href="#pone.0000506-Littleton1">[13]</a>. Of the 11 <em>Nematostella</em> iGluRs, eight clade with NMDA receptors and the remaining ones clade with AMPA/Kainate receptors (<a href="#pone.0000506.s001">Figure S1.19</a>). Multiple metabotropic glutamate receptors (mGluRs) are present in <em>Amphimedon,</em> but neither in <em>Amphimedon</em>, nor in <em>Nematostella</em> do they have sufficiently distinct sequence identities to be classified as members of bilaterian groups I, II, or III mGluRs (<a href="#pone.0000506.s001">Figure S1.12</a>). mGluR scaffolding proteins (Homer, Shank, PICK1) are also present in <em>Amphimedon</em>.</p>


<h4>Atomic Level Conservation of the Binding Domain</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To investigate conservation of the binding domain at the atomic level we compared the PDZ3–CRIPT interaction in <em>Rattus norvegius</em> (rat), <em>Danio renio</em> (fish), <em>Drosophila melanogaster</em> (fly), <em>Anopheles gambiae</em> (mosquito), <em>Strongylocentrotus purpuratus</em> (sea urchin), <em>Caenorhabditis elegans</em> (worm), <em>Nematostella vectensis</em> (cnidarian), and <em>Amphimedon queenslandica</em> (sponge) (<a href="#pone.0000506.s003">Figure S3</a>). We used the rat PDZ3–CRIPT co-crystal structure (PDB code 1BE9 <a href="#pone.0000506-Doyle1">[14]</a>, 1.82-Å resolution), as the starting point for our analysis of this protein–protein complex. The rat and <em>Amphimedon</em> sequences are only 50% identical (over the residues in the 1BE9 structure that are designated as the PDZ3 domain, which are residues 302–402 in 1BE9), while the average identity between the rat and each of the other organisms is 66%. Despite these differences over the entire domain, the core residues in direct contact with the peptide are identical between the rat and sponge (<a href="#pone.0000506.s004">Figure S4</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Five separate homology models for the sponge PDZ3—CRIPT co-complex structure all demonstrated a shape that was highly similar to the rat crystal structure (average C<sub>α</sub> RMSD 2.7 Å) (<a href="#pone-0000506-g002">Figure 2A</a>), with the largest deviations occurring in the N- and C-terminal regions and at the proline insertion in the sponge sequence between positions 318 and 319. To assess the effect of these rat-to-sponge sequence changes on binding, we computationally estimated the free energy of binding for the PDZ3—CRIPT interaction for the rat crystal structure and for each sponge model complex. The calculated total binding affinity varied by less than two-fold between these organisms, over all of the homology models, indicating that PDZ3 and CRIPT could have interacted in the animal ancestor.</p>
<div class="figure" id="pone-0000506-g002"><div class="img"><a name="pone-0000506-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000506" data-uri="info:doi/10.1371/journal.pone.0000506.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000506.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000506.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>(A) Sponge PDZ3—CRIPT Homology Model.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The last five residues of the CRIPT protein (yellow) interact with PDZ3 residues (blue and orange) by making van der Waals contacts, hydrogen bonds, or electrostatic interactions of greater than 0.1 kcal/mol in magnitude in any of the PDZ3 homology models. (<a href="#pone.0000506.s004">Figure S4</a>). The subset of residues painted blue represent the core union set that interact directly with the ligand in the PDZ1 co-crystal (2I1N), the PDZ2 co-crystal (2G2L), or the PDZ3 co-crystal (1BE9) by either van der Waals contacts of 3.9 Å or shorter or by hydrogen-bond lengths of 3.5 Å or shorter. (B) Ligand-binding residues are very highly conserved within a specific type of PDZ domain. Conservation of the 13 binding residues compared to the remaining 61 more distant residues for 16 types of PDZ domains from <em>Homo, Drosophila, Nematostella</em> and <em>Amphimedon</em>. These frequencies are also calculated across all those domains at once (column 1). Comparison of the conservation of binding residues versus non-binding residues; *, p&lt;0.05; **, p&lt;0.01; ***, p&lt;0.001 (Probability associated with a Student's two-sample unequal variance t-Test).</p>
<span>doi:10.1371/journal.pone.0000506.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Using three available crystal structures from <em>dlg</em> PDZ1-3 <a href="#pone.0000506-Doyle1">[14]</a>–<a href="#pone.0000506-VonOssowski1">[17]</a>, a core union of 13 amino acids in contact with the ligand was defined (blue residues in <a href="#pone-0000506-g002">Figure 2A</a>). Within a specific PDZ domain these amino acids have 88.5% conservation among <em>Homo, Drososphila, Nematostella,</em> and <em>Amphimedon</em>; whereas the same conservation metric across the rest of the PDZ domain drops to ~68% (<a href="#pone-0000506-g002">Figure 2B</a>). When the core union set of amino acids is compared between all PDZ domains its conservation is ~59% suggesting that this group of amino acids—those that are most highly conserved within specific PDZ domains—are not only strong drivers of PDZ domain diversity and specificity but very likely gained their distinguishing features before the Eumetazoan-Poriferan ancestor.</p>


<h4>Conservation of Domain Organization</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>A defined arrangement of motifs among many post-synaptic proteins is a well-recognized architectural feature of the complex. For example, the supra-motif organization of <em>dlg</em> beginning at the amino terminus consists of an L27 domain, three adjacent PDZ domains, a Src homology-3 (SH3) domain, and an inactive guanylate kinase-like (GUK) domain (<a href="#pone.0000506.s002">Figure S2.23.a</a>). This organization is invariant and predates the origins of the animal kingdom as indicated by its presence in the choanoflagellate, <em>Monosiga brevicollis</em>, a class of protists considered to be the closest living relatives of the animals (data not shown).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Prokaryotes and fungi have very few PDZ domain genes. Based on analysis of recently available genomes, the choanoflagellate <em>Monosiga brevicollis</em> has 58 PDZ containing genes and the cnidarian, <em>Nematostella vectensis</em> has 66 PDZ containing genes. Of these, 15 can be considered orthologous in terms of bidirectional sequence alignments, but only seven are conserved through the entirety of their domain architectures (data not shown). <em>dlg</em> and <em>Shank</em> are among these seven. At least fifty-four PDZ containing genes, many of which are among the synaptic gene repertoire, were present in the Eumetazoan ancestor and are conserved till present day lineages of Deuterostomes, Protostomes and Cnidaria. Other gene families such as the cadherins and tyrosine kinases also appeared before the metazoa/choanozoa split and laid the ground work for rapid expansion of these families within metazoan lineages <a href="#pone.0000506-Nichols1">[18]</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>A conserved supra-motif organization implies evolutionary constraints on the manner in which the scaffold presents its ligands, and therefore, one might expect to find conserved features of inter-domain regions. Although the spacing between domains is not conserved (<a href="#pone.0000506.s002">Figure S2.23.a</a>), several highly conserved proline codons and other residues occur in the <em>dlg</em> inter-domain sequences. The positioning of ligands on the <em>dlg</em> platform does not appear to depend precisely on the spacing of the <em>dlg</em> PDZ domains; more likely axis rotation of the domains to allow multiple binding events as has been reported with PDZ1 and PDZ2 domains of syntenin <a href="#pone.0000506-Grembecka1">[19]</a> is the conserved feature.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Among the post-synaptic genes present in <em>Amphimedon,</em> many other interaction domains are also conserved making it likely that the ancient signature motifs were competent to interact with ligands. For example, in mammals the Shank—Cortactin interaction occurs through a PPVP motif in Shank and an SH3 domain in Cortactin, both of which are present in the <em>Amphimedon</em> genome. The interaction between Shank and Homer occurs via a conserved PPXXF motif in Shank, and an EVH1 domain in Homer which is also present in <em>Amphimedon</em> (as is an EVH1 domain in Cortactin). The conservation of PDZ ligand sequences is shown in <a href="#pone-0000506-t001">Table 1</a>. The mammalian Shank-mGluR interaction occurs via a PDZ interaction. <em>Amphimedon</em> Shank has a PDZ domain and its mGluR has a PDZ-binding motif. The Homer binding domain in IP3R is nearly identical throughout the Metazoa (the PPKKFR motif has a small change to PPMKFR in <em>Amphimedon</em>). On the other hand, GKAP, while present in <em>Amphimedon</em>, did not acquire its PDZ ligand sequence until after the protostome-deuterostome split. This observation suggests that the GKAP interaction with PSD-95 via its GK and SH3 domains <a href="#pone.0000506-Wu1">[20]</a> is more ancient than its association with Shank even though a Shank ortholog is present in the sponge.</p>


<h4>Expression of Post-synaptic Gene Orthologs in Sponge</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To determine whether the ‘synaptic’ genes identified in <em>Amphimedon</em> are expressed in the same cell type we performed <em>in situ</em> hybridization for DLG (<em>dlg</em>), HOMER, GRIP, CRIPT and GKAP using sponge gene probe sequences. All five genes were expressed in the flask cells, which are large ciliated cells that protrude from the columnar epithelial-like outer layer of the sponge larva (<a href="#pone-0000506-g003">Figure 3</a>). They are found through out the epidermis, but at higher density towards the anterior.&nbsp;Sections of an <em>Amphimedon</em> larva revealed that these genes are not expressed together in any other larval cell types. This expression data is therefore consistent with the existence of a proto-postsynaptic-like scaffold in larval flask cells.</p>
<div class="figure" id="pone-0000506-g003"><div class="img"><a name="pone-0000506-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000506" data-uri="info:doi/10.1371/journal.pone.0000506.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000506.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000506.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000506.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000506.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000506.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Expression of Post-Synaptic Orthologs in <em>Amphimedon</em> Larvae. DLG, GKAP, GRIP, HOMER and CRIPT.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>All five genes (listed across the top) are expressed in the flask cells of <em>Amphimedon</em> larvae. (A–E) Sections of whole mount <em>in situ</em> hybridized larvae, with the posterior pole to the top. OL, outer epithelial- like layer; SL, subepithelial (middle) layer; ICM, inner cell mass. (F–J) Magnification of OL and SLs, with flask cells distributed in OL. (K–O) Scale bars: a–e, 100 µm; f–j, 50 µm; k–o, 10 µm.</p>
<span>doi:10.1371/journal.pone.0000506.g003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In addition to the labelling in flask cells all five genes show pleiotropic expression patterns. <em>GRIP, HOMER</em> and <em>CRIPT</em> appear to be expressed in a subset of the flask cells (<a href="#pone.0000506.s005">Figure S5</a>). <em>dlg</em> transcripts also are detected throughout the outer epithelial- like layer (OL). Analysis of embryonic expression of <em>dlg</em> reveals that this gene is expressed early in cells fated for the outer layer and later down regulated in all cells, except flask cells (<a href="#pone.0000506.s006">Figure S6</a>). <em>GKAP</em> and <em>HOMER</em> also display relatively high levels of expression in the subepithelial (middle) layer (SL) and large cells inside the pigment ring and a subset of ICM cells. In addition to the flask cells, <em>GRIP</em> transcripts are detected at low levels in the SL and a few ICM cells. <em>CRIPT</em> is expressed in a similar pattern to <em>GRIP</em>. The differential expression patterns of these five genes support their specificity. Furthermore, our previous studies have demonstrated other unique labelling patterns in the larva <a href="#pone.0000506-Larroux1">[21]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The data presented here support the presence of a proto-post-synaptic scaffold in the last common ancestor to all living animals. The presence of a large number of post-synaptic genes in the genome of demosponge <em>Amphimedon</em>, the nearly absolute conservation of binding domains and ligands between this sponge and animals with neurons, as well as the expression of a set of post-synaptic mRNAs in the same cell type, suggest the proto-post-synaptic scaffold existed as an assembled functional structure very early in animal evolution. More definitive evidence will require immunocytochemical protein localization. The set of genes found in <em>Amphimedon</em> are mainly components of a sub-membranous specialization in animals with nervous systems known as the post-synaptic density (PSD). Although no clear morphological correlate of a PSD or a synapse has been observed in sponges, osmophilic regions with septae have been observed in the larvae of homoscleromorph sponges as well as impulse conduction in the syncytial hexactinellid sponge Rhabdocalyptus dawsoni (<a href="#pone.0000506-Lawn1">[22]</a>, <a href="#pone.0000506-Leys1">[23]</a>; and personal communication, Sally Leys).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The core potential for evolving synapses in sponges may extend to other types of junctions. As metazoan cell types emerged, both orthologs and paralogs of the genes found here became components of junctions other than synapses as well. Therefore, the set of protosynaptic genes (as well as other genes) might be viewed not as prototypical with regard to synaptic junction evolution, but as a set of genes capable of giving rise to a diversity of junctions. For example, the first and second PDZ domains of <em>dlg</em> diversified greatly in terms of its surrounding domain organization such that PDZ domains which possibly duplicated from <em>dlg</em> PDZ 1/2 are found in true metazoan genes such as Erbin/densin-180, scribbled, and ZO. These early metazoan paralogs may have contributed to the evolution of other junctions such as the tight junction and to the establishment of polarity in epithelial cells.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The <em>Amphimedon</em> cell type that most prominently expresses the post-synaptic orthologs is the flask cell found in larva and characterized by a deep cilium and a large population of basal vesicles. The flask cell may have some environmental sensing properties that require membrane specializations <a href="#pone.0000506-Leys2">[24]</a> and perhaps reflect an evolutionary intermediate and a cell type that served as a starting point for the evolution of neurons. Alternatively, the functional neuron evolved prior to the divergence of sponges and eumetazoan lineages, and selective loss of genes encoding transmembrane receptors yielded the flask cell, which in this scenario is an evolutionary relic of the nerve cell.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The modular structure of the proto-post-synaptic complex, defined in terms of its motif organization, changed little over the entire course of animal evolution. Yet, assuming sponges are sister to eumetazoans, the few innovations that came with the origination of nervous systems were novel binding partners to an existing scaffold (<a href="#pone.0000506.s009">Table S3</a>). These genes tend to be trans-membrane proteins. For example, both NMDA receptors and neuroligins seem to have appeared during evolution well after the <em>dlg</em> scaffold to which they bind. These two genes represent a central organizing unit for the excitatory glutamatergic synapse <a href="#pone.0000506-Craig1">[25]</a>. A Shaker type K channel, not present in <em>Amphimedon</em>, also linked to the pre-existing <em>dlg</em>. Similarly, the AMPA receptors appeared after GRIP, which serves as their scaffold. In fact, a GRIP PDZ 4 and PDZ 5 domain, the domains to which the AMPA receptors bind, is distinguishable in the sponge GRIP ortholog. And GKAP, albeit present in sponge, did not acquire its PDZ ligand sequence until after protostome-deuterostome split.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Whether orthologs of these few critical post-synaptic genes were lost in the sponge or acquired by an evolutionary descendant, does not negate the finding that a proto-post-synaptic complex existed at the origin of the animal kingdom. Sponge genomes can reveal shared features that predate nervous systems and cell-specific adaptations, such as the synaptic junction. The evolution of the nervous system required network strategies by which novel binding partners can compete with pre-existing ligands in a manner that does not disrupt the assembly of the post-synaptic complex. Up-regulation of scaffold protein expression represents a possible solution to this issue, and indeed most of the scaffold proteins represent abundant components of the synapse.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Ancestral membrane-spanning receptors related to those genes which appeared to transform the proto-synaptic scaffold to a functional synaptic machine were present in neoproterozoic animals, possibly lost in the sponge, and may have critically contributed to the evolution of the post-synaptic complex in a Poriferan-Eumetazoan common ancestor. Relatively minimal modifications of the core modular structure could result in functional evolutionary leaps. The existence of a versatile proto-synaptic complex suggests that the post-synaptic junction has a modular past which is seamlessly inapparent except through its evolutionary history.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Isolation of <em>Amphimedon</em> genes</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Genomic traces and ESTs were generated as part a collaborative genome project with the Joint Genome Institute and are publicly available (<a href="http://www.ncbi.nlm.nih.gov/Traces">http://www.ncbi.nlm.nih.gov/Traces</a>). Genes were identified in EST and genome trace archives based on similarity to vertebrate sequences. The 5′ part of these genes was cloned by means of 5′ RACE using BD Smart Kit (ClonTech) and gene specific primers. The complete coding sequence was confirmed by RT-PCR of embryonic RNA (primer sequences available upon request).</p>


<h4>Data Collection and Phylogenetic Analyses</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For selected synaptic genes, protein sequences of <em>Homo sapiens, Drosophila melanogaster, Saccharomyces cerevisiae, Dictyostelium discoideum, Arabidopsis thaliana</em>, and <em>Oryza sativa</em> orthologs were collected by BLASTP searches on the NCBI blast server (<a href="http://www.ncbi.nlm.nih.gov/BLAST/">http://www.ncbi.nlm.nih.gov/BLAST/</a>) following a symmetrical best hit approach (SymBet) <a href="#pone.0000506-Koonin1">[26]</a> with a cut-off threshold of E-value e<sup>−5</sup> or less. For each lineage, the dataset was enlarged, when possible, by the inclusion of co-orthologs <a href="#pone.0000506-Sonnhammer1">[27]</a> which are two or more genes in one lineage that are, collectively, orthologous to one or more genes in another lineage due to lineage specific duplications. As an example, human SAP-97, PSD-95, SAP-102 and Chapsyn-110 are co-orthologs with fly <em>dlg</em>.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Those sequences were used to query the database of <em>Amphimedon queenslandica</em> genomic traces and <em>Nematostella vectensis</em> genomic contigs <a href="#pone.0000506-Sullivan1">[28]</a> by tBlastn <a href="#pone.0000506-Altschul1">[29]</a> for SymBets. For <em>Amphimedon</em> queries, the matching traces were collected, manually evaluated and assembled into genomic contigs using an in house assembly pipeline (sequential use of MegaBlast <a href="#pone.0000506-Zhang1">[30]</a> for selection of additional traces and PHRAP or PCAP assemblers <a href="#pone.0000506-Gordon1">[31]</a>, <a href="#pone.0000506-Huang1">[32]</a> for construction of the contigs). To reduce the risk of analyzing non-sponge contaminants, any genomic region was required to be present on multiple traces. Gene intron/exon structures were identified using GenomeScan <a href="#pone.0000506-Yeh1">[33]</a> and GENSCAN <a href="#pone.0000506-Burge1">[34]</a> and additional manual inspection. Conserved regions were subjected to further scrutiny to make sure the homology spans a considerable length of the original gene. <em>Nematostella</em> sequences were also predicted by GenomeScan and GENSCAN and added to the dataset in cases where an <em>Amphimedon</em> ortholog could not be found.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Although it is not easy to resolve the early metazoan phylogenetic history <a href="#pone.0000506-Rokas1">[35]</a>, we focused on the simpler task of whether a given gene family originated before or after the Poriferan-Eumetazoan (or Cnidarian-Bilaterian) split. For each <em>Amphimedon</em> or <em>Nematostella</em> candidate for orthology, we ran comprehensive phylogenetic analyses. If orthologs of the gene family are found in selected non-animal species, then the choice of outgroups for the phylogenetic tree is straight-forward; they are the gene family members from those non-animals. On the other hand, if the gene family members are found only in animals, phylogeny is best established by including other closely related gene families in the dataset. Most closely related gene families are often outparalogs <a href="#pone.0000506-Koonin1">[26]</a>; paralogous genes resulting from a lineage-specific duplication preceding a given speciation event. Accordingly, our dataset was enlarged, when possible, by the inclusion of the most closely related outparalogs from the above mentioned genomes with respect to the Poriferan-Eumetazoan or Cnidarian-Bilaterian splits. Inclusion of outparalogs in the dataset ensured that we were not incorrectly classifying a closely-related gene in a wrong family. In some cases, it was not possible to enlarge the dataset due to lack of any outparalogs, so we had no choice but to make phylogeny with <em>Amphimedon</em> or <em>Nematostella</em> sequences as the root of the tree. In those cases, orthology was further concluded by the requirement of a high-scoring SymBet and shared domain architecture. Protein accession numbers for all those proteins in each dataset are provided in <a href="#pone.0000506.s007">Table S1</a>.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>Protein domains were identified using PFAM and SMART <a href="#pone.0000506-Schultz1">[36]</a>, <a href="#pone.0000506-Bateman1">[37]</a> and those domain architectures for each protein family from selected species are illustrated in <a href="#pone.0000506.s002">Figure S2</a>. Sequences spanning one or more of those shared domains were used for multiple sequence alignment with ClustalW <a href="#pone.0000506-Thompson1">[38]</a>. For uncertain cases, alignments with T-Coffee <a href="#pone.0000506-Notredame1">[39]</a> and different ClustalW parameters were compared to combine them into a final alignment. Gene family phylogenies were determined using Maximum Parsimony (MP) and Neighbour Joining (NJ) of p-distances, implemented in PAUP* <a href="#pone.0000506-Swofford1">[40]</a>; Maximum Likelihood (ML) implemented in phyml <a href="#pone.0000506-Guindon1">[41]</a>; and Bayesian MCMC (BMCMC), implemented in MrBayes <a href="#pone.0000506-Ronquist1">[42]</a>. To assess confidence in individual nodes, MP, ML and NJ bootstrapping were implemented with 1000 pseudoreplicates and BMCMC posterior probabilities were reported. For ML analyses, the best-fit likelihood model was determined using ProtTest <a href="#pone.0000506-Abascal1">[43]</a>. The BMCMC analyses were conducted by running two independent analyses, each with four heated Markov chains, using a mixed amino-acid model, until a convergence diagnostic (standard deviation of split frequencies (SDSF)) fell below 0.01. Initial steps in the Markov chains were discarded as burn-in. Trees were rooted by selecting an outgroup gene according to the most parsimonious duplication and loss scenario using NOTUNG 2.0 <a href="#pone.0000506-Durand1">[44]</a> with respect to conventional species trees. Those phylogenetic trees are illustrated in <a href="#pone.0000506.s001">Figure S1</a>.</p>
<a id="article1.body1.sec4.sec2.p5" name="article1.body1.sec4.sec2.p5"></a><p>As a special case, phylogeny of 12 PDZ domain-containing protein families were determined with a single domain tree which includes selected PDZ domains of those proteins, numbered according to their spatial distribution on their corresponding scaffold. Those sequences were aligned using the global pair algorithm of MAFFT <a href="#pone.0000506-Katoh1">[45]</a> which aligns large sets of distantly related sequences of similar length with high accuracy. For phylogenetic analysis, only a single human ortholog for each of those PDZ domain-containing proteins were included to ensure the BMCMC trees converge (otherwise our computers were not able to obtain a SDSF below 0.01).</p>


<h4>Computation of Structures</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Hydrogen atoms were added to the 1BE9 rat crystal structure using the HBUILD facility <a href="#pone.0000506-Brunger1">[46]</a> and the PARAM22 parameter set <a href="#pone.0000506-MacKerell1">[47]</a> of CHARMM <a href="#pone.0000506-Brooks1">[48]</a>. All water molecules were removed from the structure, except for those making hydrogen bonds with the peptide. Missing side chains (F301, D332, and K5) were re-built, and the missing atoms of K5 were then minimized.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Homology models were generating using Modeller version 8v2 <a href="#pone.0000506-Sali1">[49]</a>, using the rat (1BE9) crystal structure and each other sequence as aligned in <a href="#pone.0000506.s003">Figure S3</a>. Ten models were requested for each sequence, and the structure was selected that had the lowest Modeller objective function without errors in PROCHECK <a href="#pone.0000506-Laskowski1">[50]</a>. The five homology models referenced for sponge are the first five homology models generated by Modeller. Binding free energies were calculated as the sum of van der Waals, solvent-accessible surface area <a href="#pone.0000506-Sitkoff1">[51]</a>, and continuum electrostatic interaction and solvation terms, after minimizing the CRIPT peptide in the bound-state.</p>


<h4>In Situ Hybridization</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Whole mount in situ hybridizations were performed as described <a href="#pone.0000506-Larroux1">[21]</a>. In short, larvae were fixed in 4% paraformaldehyde/0.05% glutaraldehyde in MOPS buffer and stepped into 70% ethanol. After rehydratation, proteinase K treatment and postfixation, larvae were hybridized over 20 h with digoxigenin-labeled antisense riboprobes transcribed from cDNAs fragments cloned into pGEMT plasmid. Probe lengths were as follows: DLG: 1kb, Homer: ~ 800 bp, GKAP: ~500 bp, GRIP: 400 bp, CRIPT: 350 bp. Following washes to remove excess probe larvae were incubated with alkaline phosphatase-conjugated anti-digoxigenin antibody, washed, and the color reaction was performed using NBT/BCIP as a substrate. Larvae were photographed whole mount and subsequently dehydrated in ethanol and infiltrated with Epon 812. Sections were cut at 5 um and mounted in Histomount. A detailed protocol is available upon request.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000506"></div><a name="pone.0000506.s001" id="pone.0000506.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000506.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Phylogenetic analyses of post-synaptic gene families. Statistical values of each essential clade is given in the order of, top left box, Bayesian Inference; top right box, Maximum likelihood; bottom left box, Maximum parsimony; bottom right box, Neighbor joining. Red and yellow colored clades represent gene families that originated before Poriferan-Eumetazoan and Cnidarian-Bilaterian splits, respectively. Green tagged sequences are from <em>Amphimedon queenslandica</em> and blue tagged sequences are from <em>Nematostella vectensis</em>. Abbreviations used in trees are: Sponge, <em>Amphimedon queenslandica</em>; CN, <em>Nematostella vectensis</em>; Human, <em>Homo sapiens</em>; Fly, <em>Drosophila melanogaster</em>; Yeast, <em>Saccharomyces cerevisiae</em>; Dicty, <em>Dictyostelium discoideum</em>; At, <em>Arabidopsis thaliana</em>; Os, <em>Oryza sativa</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000506.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(2.97 MB PDF)</p>
<a name="pone.0000506.s002" id="pone.0000506.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000506.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Domain architectures of post-synaptic gene families. Domain architectures of representative members for each gene family are displayed as SMART output. Output was manually edited for legibility for some PFAM domains, otherwise it is presented as SMART prediction. Abbreviations used in displays are: Sponge, <em>Amphimedon queenslandica</em>; CN, <em>Nematostella vectensis</em>; Human, <em>Homo sapiens</em>; Fly, <em>Drosophila melanogaster</em>; Beetle, <em>Tribolium castaneum</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000506.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(3.66 MB PDF)</p>
<a name="pone.0000506.s003" id="pone.0000506.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000506.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">DLG PDZ-3 sequence alignment through 8 species. Alignment of PDZ3 sequences used for homology modeling. Conserved residues are colored according to the scheme shown. Residues that can interact with CRIPT are noted by a gray line.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000506.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.10 MB PDF)</p>
<a name="pone.0000506.s004" id="pone.0000506.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000506.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Structural interactions between the C-terminus of the CRIPT peptide and the PDZ3 domain. (A), the C-terminus of the CRIPT peptide and the PDZ3 domain in rat, fish, fly, mosquito, cnidarian, sea urchin, and worm and (B), the C-terminus of the CRIPT peptide and the PDZ-3 domain in sponge. All the distances shown in part A and the “ray” marks indicating hydrophobic contacts are based on the rat (1BE9) crystal structure. Distances to Lys5 are shown to the beta-carbon because this is the last atom whose coordinates were reported in the crystal structure. Hydrophobic contacts from residues G322, A376, and L379 were found to interact in some organisms, but in the rat they were somewhat distant. For each residue, the main label corresponds to the rat sequence, and the smaller (or “residue name”) notes correspond to other organisms (not rat or sponge). Two interactions seen in the worm homology model were not included because they were unlike those seen in other organisms (G330 and P335). In the sponge diagram (part B), interactions that are possible given homology to the rat sequence are indicated, such as the hydrogen-bonds between PDZ3 and peptide. Residue numbering in the sponge structure corresponds to the aligned residue in the rat sequence. Some of the homology models suggested variant hydrogen-bonding patterns in the sponge, especially for residues Gln6 and Asn326. The figure was created using Ligplot and HBPLUS. The identical residues between rat and sponge are: 311, 312, 314, 318, 322*, 323*, 324*, 325*, 326*, 327*, 328*, 329, 330, 331, 335, 336, 337, 338, 339*, 341, 345, 347, 351, 353, 354, 356, 357, 359, 360, 362, 363, 364, 367, 371, 372*, 373, 375, 376, 378, 379, 380*, 382, 383, 385, 386, 387, 392, 393, 394, 396, and 400. Residues that make direct contact with the CRIPT ligand in the rat crystal structure are marked with an asterisk.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000506.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.13 MB PDF)</p>
<a name="pone.0000506.s005" id="pone.0000506.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000506.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">A surface view of an <em>Amphimedon</em> larva showing HOMER is expressed in a limited number of flask cells (arrows) by whole mount <em>in situ</em> hybridization. The large cells are the flask cells interspersed among the more numerous columnar epithelial cells. Transcripts are not detected in the remaining negatively stained flask cells in this field of view.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000506.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
<a name="pone.0000506.s006" id="pone.0000506.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000506.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Developmental expression of <em>dlg</em> during <em>Amphimedon</em> embryogenesis. All panels are sections of <em>in situ</em> hybridized embryos; posterior pole is to the top. In the blastula, a small number of small cells express <em>dlg</em>. During the gastrulation-like stage, <em>dlg</em>-expressing cells sort to the outer layer; no expression is detected in the inner cell mass. At the later spot and ring stages, prior to hatching, cells expressing <em>dlg</em> are restricted to the outer epithelial-like layer. After hatching (<a href="#pone-0000506-g003">Figure 3</a>), flask cells in this layer express <em>dlg</em> at a higher level than the surrounding columnar epithelium.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000506.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(1.01 MB PDF)</p>
<a name="pone.0000506.s007" id="pone.0000506.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000506.s007">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Protein accession numbers for sequences used in phylogenetic analyses. Each table contains the accession numbers for the sequences used in the corresponding tree from <a href="#pone.0000506.s001">Figure S1</a>. PDZ and iGluR genes/domains that are shown in <a href="#pone.0000506.s007">table S1.19 and S1.23</a> in grey letters are not included in phylogenetic analyses. In <a href="#pone.0000506.s007">table S1.23</a>, PDZ domain amino-acid locations on their corresponding proteins are shown in parenthesis after abbreviation and all gi numbers correspond to their proteins. Abbreviations used are: Sponge, <em>Amphimedon queenslandica</em>; CN, <em>Nematostella vectensis</em>; Human, <em>Homo sapiens</em>; Fly, <em>Drosophila melanogaster</em>; Yeast, <em>Saccharomyces cerevisiae</em>; Dicty, <em>Dictyostelium discoideum</em>; At, <em>Arabidopsis thaliana</em>; Os, <em>Oryza sativa</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000506.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0000506.s008" id="pone.0000506.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000506.s008">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Sponge <em>dlg</em> and Human SAP97 intron/exon structure comparison. Cloned cDNAs of Sponge (<em>Amphimedon</em>) <em>dlg</em> were fully sequenced and mapped to its genomic sequence. Intron/exon structure was compared with its human ortholog SAP97, and about half the exons were found to be almost same size (highlighted in red) and content (in bold). However, human SAP-97 introns were in average 100 times larger than <em>Amphimedon dlg</em> introns (data not shown).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000506.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0000506.s009" id="pone.0000506.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000506.s009">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Presence of post-synaptic gene orthologs in animals, yeast, dicty and plants. Species abbreviations used: Human, <em>Homo sapiens</em>; Fly, <em>Drosophila Melanogaster</em>; Nema, <em>Nematostella vectensis</em>; Sponge, <em>Amphimedon queenslandica</em>; Yeast, <em>Saccharomyces cerevisiae</em>; Dicty, <em>Dictyostelium discoideum</em>; Plants, <em>Arabidopsis thaliana</em> and <em>Oryza sativa</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000506.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.08 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We gratefully acknowledge the significant contribution and support of The US Department of Energy Joint Genome Institute in the production of <em>Amphimedon</em> (<em>Reniera</em>) and <em>Nematostella</em> genomic and EST sequences used in this study through the Community Sequencing Program. The following individuals contributed helpful comments: Boris Shraiman, Yoram Barak, Ilya Feit, James Weaver, Michi Izumi, Dan Morse, Ambuj Singh, Omer Egecioglu, Eduardo Orias, Sally Leys, Sira Sriswasdi, and Gus Gurley. We thank Daniel Rokhsar and Mark Martindale for making <em>Amphimedon</em> (<em>Reniera</em>) and <em>Nematostella</em> genomic and EST sequences available.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KK BT TO BD OS KA. Performed the experiments: OS KA MA IW. Analyzed the data: KK TO BD OS KA MA MA. Contributed reagents/materials/analysis tools: KK BT TO BD MA. Wrote the paper: KK TO OS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000506-Gould1" id="pone.0000506-Gould1"></a>Gould SJ, Vrba ES (1982) Exaptation; a missing term in the science of form Paleobiology 8(1): 4–15.  <ul class="find" data-citedArticleID="956748"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Exaptation%3B+a+missing+term+in+the+science+of+form+Paleobiology&amp;auth=&amp;atitle=Exaptation%3B+a+missing+term+in+the+science+of+form+Paleobiology" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exaptation%3B+a+missing+term+in+the+science+of+form+Paleobiology" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exaptation%3B+a+missing+term+in+the+science+of+form+Paleobiology%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000506-Hooper1" id="pone.0000506-Hooper1"></a>Hooper JNA, Van Soest RWM (2002) Systema Porifera: A Guide to the classification of sponges Vols 1&amp;2. New York: Kluwer Academic/Plenum Publishers.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0000506-Delsuc1" id="pone.0000506-Delsuc1"></a>Delsuc F, Brinkmann H, Philippe H (2005) Phylogenomics and the reconstruction of the tree of life. Nat Rev Genet  6(5): 361–375.  <ul class="find" data-citedArticleID="956738" data-doi="10.1038/nrg1603"><li><a href="http://dx.doi.org/10.1038/nrg1603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenomics+and+the+reconstruction+of+the+tree+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenomics+and+the+reconstruction+of+the+tree+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000506-Wang1" id="pone.0000506-Wang1"></a>Wang X, Lavrov DV (2007) Mitochondrial Genome of the Homoscleromorph Oscarella carmela (Porifera, Demospongiae) Reveals Unexpected Complexity in the Common Ancestor of Sponges and Other Animals. Molecular biology and evolution  24(2): 363–373.  <ul class="find" data-citedArticleID="956812" data-doi="10.1093/molbev/msl167"><li><a href="http://dx.doi.org/10.1093/molbev/msl167" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+Genome+of+the+Homoscleromorph+Oscarella+carmela+%28Porifera%2C+Demospongiae%29+Reveals+Unexpected+Complexity+in+the+Common+Ancestor+of+Sponges+and+Other+Animals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+Genome+of+the+Homoscleromorph+Oscarella+carmela+%28Porifera%2C+Demospongiae%29+Reveals+Unexpected+Complexity+in+the+Common+Ancestor+of+Sponges+and+Other+Animals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000506-Erpenbeck1" id="pone.0000506-Erpenbeck1"></a>Erpenbeck D, Voigt O, Adamski M, Adamska M, Hooper JN, et al.  (2007) Mitochondrial diversity of early-branching metazoa is revealed by the complete mt genome of a haplosclerid demosponge. Molecular biology and evolution  24(1): 19–22.  <ul class="find" data-citedArticleID="956744" data-doi="10.1093/molbev/msl154"><li><a href="http://dx.doi.org/10.1093/molbev/msl154" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+diversity+of+early-branching+metazoa+is+revealed+by+the+complete+mt+genome+of+a+haplosclerid+demosponge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+diversity+of+early-branching+metazoa+is+revealed+by+the+complete+mt+genome+of+a+haplosclerid+demosponge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000506-CavalierSmith1" id="pone.0000506-CavalierSmith1"></a>Cavalier-Smith T, Chao E, Boury-Esnault N, Vacelet J (1996) Sponge phylogeny, animal monophyly, and the origin of the nervous system: 18S rRNA evidence. Can J Zool  74: 2031–2045.  <ul class="find" data-citedArticleID="956734" data-doi="10.1139/z96-231"><li><a href="http://dx.doi.org/10.1139/z96-231" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sponge+phylogeny%2C+animal+monophyly%2C+and+the+origin+of+the+nervous+system%3A+18S+rRNA+evidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sponge+phylogeny%2C+animal+monophyly%2C+and+the+origin+of+the+nervous+system%3A+18S+rRNA+evidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000506-Borchiellini1" id="pone.0000506-Borchiellini1"></a>Borchiellini C, Manuel M, Alivon E, Boury-Esnault N, Vacelet J, et al.  (2001) Sponge paraphyly and the origin of Metazoa. J Evol Biol  (14): 171–179.  <ul class="find" data-citedArticleID="956726" data-doi="10.1046/j.1420-9101.2001.00244.x"><li><a href="http://dx.doi.org/10.1046/j.1420-9101.2001.00244.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sponge+paraphyly+and+the+origin+of+Metazoa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sponge+paraphyly+and+the+origin+of+Metazoa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000506-Peterson1" id="pone.0000506-Peterson1"></a>Peterson KJ, Butterfield NJ (2005) Origin of the Eumetazoa: testing ecological predictions of molecular clocks against the Proterozoic fossil record. Proceedings of the National Academy of Sciences of the United States of America  102(27): 9547–9552.  <ul class="find" data-citedArticleID="956788" data-doi="10.1073/pnas.0503660102"><li><a href="http://dx.doi.org/10.1073/pnas.0503660102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+of+the+Eumetazoa%3A+testing+ecological+predictions+of+molecular+clocks+against+the+Proterozoic+fossil+record." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+of+the+Eumetazoa%3A+testing+ecological+predictions+of+molecular+clocks+against+the+Proterozoic+fossil+record.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000506-Husi1" id="pone.0000506-Husi1"></a>Husi H, Ward MA, Choudhary JS, Blackstock WP, Grant SG (2000) Proteomic analysis of NMDA receptor-adhesion protein signaling complexes. Nature neuroscience  3(7): 661–669.  <ul class="find" data-citedArticleID="956762" data-doi="10.1038/76615"><li><a href="http://dx.doi.org/10.1038/76615" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteomic+analysis+of+NMDA+receptor-adhesion+protein+signaling+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteomic+analysis+of+NMDA+receptor-adhesion+protein+signaling+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000506-Peng1" id="pone.0000506-Peng1"></a>Peng J, Kim MJ, Cheng D, Duong DM, Gygi SP, et al.  (2004) Semiquantitative proteomic analysis of rat forebrain postsynaptic density fractions by mass spectrometry. The Journal of biological chemistry  279(20): 21003–21011.  <ul class="find" data-citedArticleID="956786" data-doi="10.1074/jbc.m400103200"><li><a href="http://dx.doi.org/10.1074/jbc.m400103200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Semiquantitative+proteomic+analysis+of+rat+forebrain+postsynaptic+density+fractions+by+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Semiquantitative+proteomic+analysis+of+rat+forebrain+postsynaptic+density+fractions+by+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000506-Grant1" id="pone.0000506-Grant1"></a>Grant SG (2006) The synapse proteome and phosphoproteome: a new paradigm for synapse biology. Biochemical Society transactions  34(Pt 1): 59–63.  <ul class="find" data-citedArticleID="956750" data-doi="10.1042/bst0340059"><li><a href="http://dx.doi.org/10.1042/bst0340059" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+synapse+proteome+and+phosphoproteome%3A+a+new+paradigm+for+synapse+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+synapse+proteome+and+phosphoproteome%3A+a+new+paradigm+for+synapse+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000506-Grimmelikhuijzen1" id="pone.0000506-Grimmelikhuijzen1"></a>Grimmelikhuijzen CJ, Westfall JA (1995) The nervous systems of cnidarians. Exs  72: 7–24.  <ul class="find" data-citedArticleID="956754" data-doi="10.1007/978-3-0348-9219-3_2"><li><a href="http://dx.doi.org/10.1007/978-3-0348-9219-3_2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+nervous+systems+of+cnidarians." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+nervous+systems+of+cnidarians.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000506-Littleton1" id="pone.0000506-Littleton1"></a>Littleton JT, Ganetzky B (2000) Ion channels and synaptic organization: analysis of the Drosophila genome. Neuron  26(1): 35–43.  <ul class="find" data-citedArticleID="956778" data-doi="10.1016/s0896-6273(00)81135-6"><li><a href="http://dx.doi.org/10.1016/s0896-6273(00)81135-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ion+channels+and+synaptic+organization%3A+analysis+of+the+Drosophila+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ion+channels+and+synaptic+organization%3A+analysis+of+the+Drosophila+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000506-Doyle1" id="pone.0000506-Doyle1"></a>Doyle DA, Lee A, Lewis J, Kim E, Sheng M, et al.  (1996) Crystal structures of a complexed and peptide-free membrane protein-binding domain: molecular basis of peptide recognition by PDZ. Cell  85(7): 1067–1076.  <ul class="find" data-citedArticleID="956740" data-doi="10.1016/s0092-8674(00)81307-0"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81307-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structures+of+a+complexed+and+peptide-free+membrane+protein-binding+domain%3A+molecular+basis+of+peptide+recognition+by+PDZ." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structures+of+a+complexed+and+peptide-free+membrane+protein-binding+domain%3A+molecular+basis+of+peptide+recognition+by+PDZ.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000506-Berman1" id="pone.0000506-Berman1"></a>Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, et al.  (2000) The Protein Data Bank. Nucleic acids research  28(1): 235–242.  <ul class="find" data-citedArticleID="956724" data-doi="10.1093/nar/28.1.235"><li><a href="http://dx.doi.org/10.1093/nar/28.1.235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Protein+Data+Bank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Protein+Data+Bank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000506-Turnbull1" id="pone.0000506-Turnbull1"></a>Turnbull AP, Phillips C, Bunkoczi G, Debreczeni J, Ugochukwu E, et al. (To be published (PDB id: 2I1N)) PDB id: 2I1N. Crystal structure of the 1st PDZ domain of Human DLG3  <ul class="find" data-citedArticleID="956808"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=%28To+be+published+%28PDB+id%3A+2I1N%29%29+PDB+id%3A+2I1N.+Crystal+structure+of+the+1st+PDZ+domain+of+Human+DLG3&amp;auth=&amp;atitle=%28To+be+published+%28PDB+id%3A+2I1N%29%29+PDB+id%3A+2I1N.+Crystal+structure+of+the+1st+PDZ+domain+of+Human+DLG3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%28To+be+published+%28PDB+id%3A+2I1N%29%29+PDB+id%3A+2I1N.+Crystal+structure+of+the+1st+PDZ+domain+of+Human+DLG3" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%28To+be+published+%28PDB+id%3A+2I1N%29%29+PDB+id%3A+2I1N.+Crystal+structure+of+the+1st+PDZ+domain+of+Human+DLG3%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000506-VonOssowski1" id="pone.0000506-VonOssowski1"></a>Von Ossowski I, Oksanen E, Cai C, Sundberg M, Goldman A, et al. (To be published (PDB id: 2G2L)) PDB id: 2G2L. Crystal Structure of the Second PDZ Domain of SAP97 in Complex with a GluR-A C-terminal Peptide  <ul class="find" data-citedArticleID="956810" data-doi="10.1111/j.1742-4658.2006.05521.x"><li><a href="http://dx.doi.org/10.1111/j.1742-4658.2006.05521.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%28To+be+published+%28PDB+id%3A+2G2L%29%29+PDB+id%3A+2G2L.+Crystal+Structure+of+the+Second+PDZ+Domain+of+SAP97+in+Complex+with+a+GluR-A+C-terminal+Peptide" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%28To+be+published+%28PDB+id%3A+2G2L%29%29+PDB+id%3A+2G2L.+Crystal+Structure+of+the+Second+PDZ+Domain+of+SAP97+in+Complex+with+a+GluR-A+C-terminal+Peptide%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000506-Nichols1" id="pone.0000506-Nichols1"></a>Nichols SA, Dirks W, Pearse JS, King N (2006) Early evolution of animal cell signaling and adhesion genes. Proceedings of the National Academy of Sciences of the United States of America  103(33): 12451–12456.  <ul class="find" data-citedArticleID="956782" data-doi="10.1073/pnas.0604065103"><li><a href="http://dx.doi.org/10.1073/pnas.0604065103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+evolution+of+animal+cell+signaling+and+adhesion+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+evolution+of+animal+cell+signaling+and+adhesion+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000506-Grembecka1" id="pone.0000506-Grembecka1"></a>Grembecka J, Cierpicki T, Devedjiev Y, Derewenda U, Kang BS, et al.  (2006) The binding of the PDZ tandem of syntenin to target proteins. Biochemistry  45(11): 3674–3683.  <ul class="find" data-citedArticleID="956752" data-doi="10.1021/bi052225y"><li><a href="http://dx.doi.org/10.1021/bi052225y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+binding+of+the+PDZ+tandem+of+syntenin+to+target+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+binding+of+the+PDZ+tandem+of+syntenin+to+target+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000506-Wu1" id="pone.0000506-Wu1"></a>Wu H, Reissner C, Kuhlendahl S, Coblentz B, Reuver S, et al.  (2000) Intramolecular interactions regulate SAP97 binding to GKAP. The EMBO journal  19(21): 5740–5751.  <ul class="find" data-citedArticleID="956814" data-doi="10.1093/emboj/19.21.5740"><li><a href="http://dx.doi.org/10.1093/emboj/19.21.5740" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intramolecular+interactions+regulate+SAP97+binding+to+GKAP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intramolecular+interactions+regulate+SAP97+binding+to+GKAP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000506-Larroux1" id="pone.0000506-Larroux1"></a>Larroux C, Fahey B, Liubicich D, Hinman VF, Gauthier M, et al.  (2006) Developmental expression of transcription factor genes in a demosponge: insights into the origin of metazoan multicellularity. Evolution &amp; development  8(2): 150–173.  <ul class="find" data-citedArticleID="956768" data-doi="10.1111/j.1525-142x.2006.00086.x"><li><a href="http://dx.doi.org/10.1111/j.1525-142x.2006.00086.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developmental+expression+of+transcription+factor+genes+in+a+demosponge%3A+insights+into+the+origin+of+metazoan+multicellularity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developmental+expression+of+transcription+factor+genes+in+a+demosponge%3A+insights+into+the+origin+of+metazoan+multicellularity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000506-Lawn1" id="pone.0000506-Lawn1"></a>Lawn ID, Mackie GO, Silver G (1981) Conduction system in a sponge. Science  211(4487): 1169–1171.  <ul class="find" data-citedArticleID="956772" data-doi="10.1126/science.7466387"><li><a href="http://dx.doi.org/10.1126/science.7466387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conduction+system+in+a+sponge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conduction+system+in+a+sponge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000506-Leys1" id="pone.0000506-Leys1"></a>Leys SP, Mackie GO, Meech RW (1999) Impulse conduction in a sponge. J Exp Biol  202(Pt 9): 1139–1150.  <ul class="find" data-citedArticleID="956774"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Impulse+conduction+in+a+sponge.&amp;auth=&amp;atitle=Impulse+conduction+in+a+sponge." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impulse+conduction+in+a+sponge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impulse+conduction+in+a+sponge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000506-Leys2" id="pone.0000506-Leys2"></a>Leys SP, Degnan BM (2001) Cytological basis of photoresponsive behavior in a sponge larva. The Biological bulletin  201(3): 323–338.  <ul class="find" data-citedArticleID="956776" data-doi="10.2307/1543611"><li><a href="http://dx.doi.org/10.2307/1543611" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytological+basis+of+photoresponsive+behavior+in+a+sponge+larva." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytological+basis+of+photoresponsive+behavior+in+a+sponge+larva.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000506-Craig1" id="pone.0000506-Craig1"></a>Craig AM, Kang Y (2007) Neurexin-neuroligin signaling in synapse development. Current opinion in neurobiology  17(1): 43–52.  <ul class="find" data-citedArticleID="956736" data-doi="10.1016/j.conb.2007.01.011"><li><a href="http://dx.doi.org/10.1016/j.conb.2007.01.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neurexin-neuroligin+signaling+in+synapse+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neurexin-neuroligin+signaling+in+synapse+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000506-Koonin1" id="pone.0000506-Koonin1"></a>Koonin EV (2005) Orthologs, paralogs, and evolutionary genomics. Annual review of genetics  39: 309–338.  <ul class="find" data-citedArticleID="956766" data-doi="10.1146/annurev.genet.39.073003.114725"><li><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.114725" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Orthologs%2C+paralogs%2C+and+evolutionary+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Orthologs%2C+paralogs%2C+and+evolutionary+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000506-Sonnhammer1" id="pone.0000506-Sonnhammer1"></a>Sonnhammer EL, Koonin EV (2002) Orthology, paralogy and proposed classification for paralog subtypes. Trends Genet  18(12): 619–620.  <ul class="find" data-citedArticleID="956800" data-doi="10.1016/s0168-9525(02)02793-2"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02793-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Orthology%2C+paralogy+and+proposed+classification+for+paralog+subtypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Orthology%2C+paralogy+and+proposed+classification+for+paralog+subtypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000506-Sullivan1" id="pone.0000506-Sullivan1"></a>Sullivan JC, Ryan JF, Watson JA, Webb J, Mullikin JC, et al.  (2006) StellaBase: the Nematostella vectensis Genomics Database. Nucleic acids research  34(Database issue): D495–499.  <ul class="find" data-citedArticleID="956802" data-doi="10.1093/nar/gkj020"><li><a href="http://dx.doi.org/10.1093/nar/gkj020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=StellaBase%3A+the+Nematostella+vectensis+Genomics+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22StellaBase%3A+the+Nematostella+vectensis+Genomics+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000506-Altschul1" id="pone.0000506-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic acids research  25(17): 3389–3402.  <ul class="find" data-citedArticleID="956720" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000506-Zhang1" id="pone.0000506-Zhang1"></a>Zhang Z, Schwartz S, Wagner L, Miller W (2000) A greedy algorithm for aligning DNA sequences. J Comput Biol  7(1–2): 203–214.  <ul class="find" data-citedArticleID="956818" data-doi="10.1089/10665270050081478"><li><a href="http://dx.doi.org/10.1089/10665270050081478" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+greedy+algorithm+for+aligning+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+greedy+algorithm+for+aligning+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000506-Gordon1" id="pone.0000506-Gordon1"></a>Gordon D, Abajian C, Green P (1998) Consed: a graphical tool for sequence finishing. Genome research  8(3): 195–202.  <ul class="find" data-citedArticleID="956746" data-doi="10.1101/gr.8.3.195"><li><a href="http://dx.doi.org/10.1101/gr.8.3.195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consed%3A+a+graphical+tool+for+sequence+finishing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consed%3A+a+graphical+tool+for+sequence+finishing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000506-Huang1" id="pone.0000506-Huang1"></a>Huang X, Wang J, Aluru S, Yang SP, Hillier L (2003) PCAP: a whole-genome assembly program. Genome research  13(9): 2164–2170.  <ul class="find" data-citedArticleID="956760" data-doi="10.1101/gr.1390403"><li><a href="http://dx.doi.org/10.1101/gr.1390403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PCAP%3A+a+whole-genome+assembly+program." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PCAP%3A+a+whole-genome+assembly+program.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000506-Yeh1" id="pone.0000506-Yeh1"></a>Yeh RF, Lim LP, Burge CB (2001) Computational inference of homologous gene structures in the human genome. Genome research  11(5): 803–816.  <ul class="find" data-citedArticleID="956816" data-doi="10.1101/gr.175701"><li><a href="http://dx.doi.org/10.1101/gr.175701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+inference+of+homologous+gene+structures+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+inference+of+homologous+gene+structures+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000506-Burge1" id="pone.0000506-Burge1"></a>Burge C, Karlin S (1997) Prediction of complete gene structures in human genomic DNA. Journal of molecular biology  268(1): 78–94.  <ul class="find" data-citedArticleID="956732" data-doi="10.1006/jmbi.1997.0951"><li><a href="http://dx.doi.org/10.1006/jmbi.1997.0951" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+complete+gene+structures+in+human+genomic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+complete+gene+structures+in+human+genomic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000506-Rokas1" id="pone.0000506-Rokas1"></a>Rokas A, King N, Finnerty J, Carroll SB (2003) Conflicting phylogenetic signals at the base of the metazoan tree. Evolution &amp; development  5(4): 346–359.  <ul class="find" data-citedArticleID="956790" data-doi="10.1046/j.1525-142x.2003.03042.x"><li><a href="http://dx.doi.org/10.1046/j.1525-142x.2003.03042.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conflicting+phylogenetic+signals+at+the+base+of+the+metazoan+tree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conflicting+phylogenetic+signals+at+the+base+of+the+metazoan+tree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000506-Schultz1" id="pone.0000506-Schultz1"></a>Schultz J, Milpetz F, Bork P, Ponting CP (1998) SMART, a simple modular architecture research tool: identification of signaling domains. Proceedings of the National Academy of Sciences of the United States of America  95(11): 5857–5864.  <ul class="find" data-citedArticleID="956796" data-doi="10.1073/pnas.95.11.5857"><li><a href="http://dx.doi.org/10.1073/pnas.95.11.5857" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SMART%2C+a+simple+modular+architecture+research+tool%3A+identification+of+signaling+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SMART%2C+a+simple+modular+architecture+research+tool%3A+identification+of+signaling+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000506-Bateman1" id="pone.0000506-Bateman1"></a>Bateman A, Birney E, Cerruti L, Durbin R, Etwiller L, et al.  (2002) The Pfam protein families database. Nucleic acids research  30(1): 276–280.  <ul class="find" data-citedArticleID="956722" data-doi="10.1093/nar/30.1.276"><li><a href="http://dx.doi.org/10.1093/nar/30.1.276" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Pfam+protein+families+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Pfam+protein+families+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000506-Thompson1" id="pone.0000506-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic acids research  22(22): 4673–4680.  <ul class="find" data-citedArticleID="956806" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000506-Notredame1" id="pone.0000506-Notredame1"></a>Notredame C, Higgins DG, Heringa J (2000) T-Coffee: A novel method for fast and accurate multiple sequence alignment. Journal of molecular biology  302(1): 205–217.  <ul class="find" data-citedArticleID="956784" data-doi="10.1006/jmbi.2000.4042"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T-Coffee%3A+A+novel+method+for+fast+and+accurate+multiple+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T-Coffee%3A+A+novel+method+for+fast+and+accurate+multiple+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000506-Swofford1" id="pone.0000506-Swofford1"></a>Swofford D (2003) PAUP*: Phylogenetic Analylsis Using Parsimony (* and Other Methods): Sinauer Associates, Sunderland, MA.   <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0000506-Guindon1" id="pone.0000506-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Systematic biology  52(5): 696–704.  <ul class="find" data-citedArticleID="956756"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000506-Ronquist1" id="pone.0000506-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics (Oxford, England)  19(12): 1572–1574.  <ul class="find" data-citedArticleID="956792" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000506-Abascal1" id="pone.0000506-Abascal1"></a>Abascal F, Zardoya R, Posada D (2005) ProtTest: selection of best-fit models of protein evolution. Bioinformatics (Oxford, England)  21(9): 2104–2105.  <ul class="find" data-citedArticleID="956718" data-doi="10.1093/bioinformatics/bti263"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ProtTest%3A+selection+of+best-fit+models+of+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ProtTest%3A+selection+of+best-fit+models+of+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000506-Durand1" id="pone.0000506-Durand1"></a>Durand D, Halldorsson BV, Vernot B (2006) A hybrid micro-macroevolutionary approach to gene tree reconstruction. J Comput Biol  13(2): 320–335.  <ul class="find" data-citedArticleID="956742" data-doi="10.1089/cmb.2006.13.320"><li><a href="http://dx.doi.org/10.1089/cmb.2006.13.320" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+hybrid+micro-macroevolutionary+approach+to+gene+tree+reconstruction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+hybrid+micro-macroevolutionary+approach+to+gene+tree+reconstruction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000506-Katoh1" id="pone.0000506-Katoh1"></a>Katoh K, Misawa K, Kuma K, Miyata T (2002) MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform. Nucleic acids research  30(14): 3059–3066.  <ul class="find" data-citedArticleID="956764" data-doi="10.1093/nar/gkf436"><li><a href="http://dx.doi.org/10.1093/nar/gkf436" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAFFT%3A+a+novel+method+for+rapid+multiple+sequence+alignment+based+on+fast+Fourier+transform." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAFFT%3A+a+novel+method+for+rapid+multiple+sequence+alignment+based+on+fast+Fourier+transform.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000506-Brunger1" id="pone.0000506-Brunger1"></a>Brunger AT, Karplus M (1988) Polar hydrogen positions in proteins: empirical energy placement and neutron diffraction comparison. Proteins  4(2): 148–156.  <ul class="find" data-citedArticleID="956730" data-doi="10.1002/prot.340040208"><li><a href="http://dx.doi.org/10.1002/prot.340040208" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polar+hydrogen+positions+in+proteins%3A+empirical+energy+placement+and+neutron+diffraction+comparison." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polar+hydrogen+positions+in+proteins%3A+empirical+energy+placement+and+neutron+diffraction+comparison.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000506-MacKerell1" id="pone.0000506-MacKerell1"></a>MacKerell AD, D B, Bellott M, Dunbrack RL, Evanseck JD, et al.  (1998) All-Atom Empirical Potential for Molecular Modeling and Dynamics Studies of Proteins. J Phys Chem B  102(18): 3586–3616.  <ul class="find" data-citedArticleID="956780" data-doi="10.1021/jp973084f"><li><a href="http://dx.doi.org/10.1021/jp973084f" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=All-Atom+Empirical+Potential+for+Molecular+Modeling+and+Dynamics+Studies+of+Proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22All-Atom+Empirical+Potential+for+Molecular+Modeling+and+Dynamics+Studies+of+Proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000506-Brooks1" id="pone.0000506-Brooks1"></a>Brooks BR, Bruccoleri RE, Olafson BD, States DJ, Swaminathan S, et al.  (1983) CHARMM: A program for macromolecular energy, minimization, and dynamics calculations. J Comput Chem  4: 187–217.  <ul class="find" data-citedArticleID="956728" data-doi="10.1002/jcc.540040211"><li><a href="http://dx.doi.org/10.1002/jcc.540040211" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CHARMM%3A+A+program+for+macromolecular+energy%2C+minimization%2C+and+dynamics+calculations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CHARMM%3A+A+program+for+macromolecular+energy%2C+minimization%2C+and+dynamics+calculations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000506-Sali1" id="pone.0000506-Sali1"></a>Sali A (1995) Comparative protein modeling by satisfaction of spatial restraints. Molecular medicine today  1(6): 270–277.  <ul class="find" data-citedArticleID="956794" data-doi="10.1016/s1357-4310(95)91170-7"><li><a href="http://dx.doi.org/10.1016/s1357-4310(95)91170-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+protein+modeling+by+satisfaction+of+spatial+restraints." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+protein+modeling+by+satisfaction+of+spatial+restraints.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000506-Laskowski1" id="pone.0000506-Laskowski1"></a>Laskowski RA, MacArthur MW, Moss DS, Thornton JM (1993) PROCHECK: a program to check the stereochemical quality of protein structures. J Appl Cryst  26: 283–291.  <ul class="find" data-citedArticleID="956770" data-doi="10.1107/s0021889892009944"><li><a href="http://dx.doi.org/10.1107/s0021889892009944" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000506-Sitkoff1" id="pone.0000506-Sitkoff1"></a>Sitkoff D, Sharp KA, Honig B (1994) Accurate calculation of hydration free-energies using macroscopic solvent models. J Phys Chem  98: 1978–1988.  <ul class="find" data-citedArticleID="956798" data-doi="10.1021/j100058a043"><li><a href="http://dx.doi.org/10.1021/j100058a043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+calculation+of+hydration+free-energies+using+macroscopic+solvent+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+calculation+of+hydration+free-energies+using+macroscopic+solvent+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.XML" value="103431"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.PDF" value="447859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g001.PNG_L" value="3914042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g001.PNG_M" value="155452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g001.PNG_S" value="5740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g001.TIF" value="6068874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g001.PNG_I" value="62358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.t001.PNG_L" value="114539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.t001.PNG_M" value="44675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.t001.PNG_S" value="2484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.t001.TIF" value="501118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.t001.PNG_I" value="18363"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g002.PNG_L" value="2428170"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g002.PNG_M" value="117751"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g002.PNG_S" value="6968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g002.TIF" value="7239626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g002.PNG_I" value="75862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g003.PNG_L" value="7284547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g003.PNG_M" value="410828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g003.PNG_S" value="8649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g003.TIF" value="10076138"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.g003.PNG_I" value="122409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.s001.PDF" value="2969835"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.s002.PDF" value="3663592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.s003.PDF" value="95295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.s004.PDF" value="127489"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.s005.PDF" value="65628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.s006.PDF" value="1010226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.s007.PDF" value="28123"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.s008.PDF" value="29492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000506.s009.PDF" value="84485"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000506&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000506" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000506&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000506&volume=&issue=&title=A Post-Synaptic Scaffold at the Origin of the Animal Kingdom&author_name=Onur%20Sakarya%2C%20Kathryn%20A.%20Armstrong%2C%20Maja%20Adamska%2C%20Marcin%20Adamski%2C%20I-Fan%20Wang%2C%20Bruce%20Tidor%2C%20Bernard%20M.%20Degnan%2C%20Todd%20H.%20Oakley%2C%20Kenneth%20S.%20Kosik&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000506" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000506" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000506" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000506&amp;t=A%20Post-Synaptic%20Scaffold%20at%20the%20Origin%20of%20the%20Animal%20Kingdom" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000506&title=A%20Post-Synaptic%20Scaffold%20at%20the%20Origin%20of%20the%20Animal%20Kingdom&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000506&amp;title=A%20Post-Synaptic%20Scaffold%20at%20the%20Origin%20of%20the%20Animal%20Kingdom" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000506" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000506&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Post-Synaptic Scaffold at the Origin of the Animal Kingdom';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000506';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Post-Synaptic%20Scaffold%20at%20the%20Origin%20of%20the%20Animal%20Kingdom http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000506" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000506" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+evolution%22" title="Search for articles in the subject area:'Animal evolution'"><div class="flagText">Animal evolution</div></a>
              <div data-categoryid="18779" data-articleid="23850"
                   data-categoryname="Animal evolution"
                   class="flagImage" title="Flag 'Animal evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+genomics%22" title="Search for articles in the subject area:'Animal genomics'"><div class="flagText">Animal genomics</div></a>
              <div data-categoryid="17455" data-articleid="23850"
                   data-categoryname="Animal genomics"
                   class="flagImage" title="Flag 'Animal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23850"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invertebrate+genomics%22" title="Search for articles in the subject area:'Invertebrate genomics'"><div class="flagText">Invertebrate genomics</div></a>
              <div data-categoryid="17221" data-articleid="23850"
                   data-categoryname="Invertebrate genomics"
                   class="flagImage" title="Flag 'Invertebrate genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nervous+system%22" title="Search for articles in the subject area:'Nervous system'"><div class="flagText">Nervous system</div></a>
              <div data-categoryid="48377" data-articleid="23850"
                   data-categoryname="Nervous system"
                   class="flagImage" title="Flag 'Nervous system' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="23850"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="23850"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sponges%22" title="Search for articles in the subject area:'Sponges'"><div class="flagText">Sponges</div></a>
              <div data-categoryid="43951" data-articleid="23850"
                   data-categoryname="Sponges"
                   class="flagImage" title="Flag 'Sponges' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5143'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7190'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3230&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=711">phylogeny</a><br>Posted by RichardWeinberg</p>
      <p><a href="/annotation/listThread.action?root=10307">Could the author(s) deposit novel sequences analyzed in the studies to the NCBI database?</a><br>Posted by PLoS_Cai</p>
      <p><a href="/annotation/listThread.action?root=43085">Great insight -- a really interesting analysis</a><br>Posted by jstajich</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
