

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Artificially Introduced Aneuploid Chromosomes Assume a Conserved Position in Colon Cancer Cells</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000199"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000199" />

    <meta name="citation_title" content="Artificially Introduced Aneuploid Chromosomes Assume a Conserved Position in Colon Cancer Cells"/>
    <meta itemprop="name" content="Artificially Introduced Aneuploid Chromosomes Assume a Conserved Position in Colon Cancer Cells"/>

      <meta name="citation_author" content="Kundan Sengupta"/>
            <meta name="citation_author_institution" content="Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Madhvi B. Upender"/>
            <meta name="citation_author_institution" content="Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Linda Barenboim-Stapleton"/>
            <meta name="citation_author_institution" content="Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Quang Tri Nguyen"/>
            <meta name="citation_author_institution" content="Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Stephen M. Wincovitch Sr."/>
            <meta name="citation_author_institution" content="Laboratory of Experimental Carcinogenesis, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Susan H. Garfield"/>
            <meta name="citation_author_institution" content="Laboratory of Experimental Carcinogenesis, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Michael J. Difilippantonio"/>
            <meta name="citation_author_institution" content="Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Thomas Ried"/>
            <meta name="citation_author_institution" content="Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>

    <meta name="citation_date" content="2007/2/7"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000199.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e199"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Differences in the localization and morphology of chromosomes in the human nucleus.; citation_author=JA Croft; citation_author=JM Bridger; citation_author=S Boyle; citation_author=P Perry; citation_author=P Teague; citation_journal_title=J Cell Biol; citation_volume=145; citation_number=1; citation_pages=1119-1131; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Tissue-specific spatial organization of genomes.; citation_author=LA Parada; citation_author=PG McQueen; citation_author=T Misteli; citation_journal_title=Genome Biol; citation_volume=5; citation_number=2; citation_pages=R44; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolutionary conservation of chromosome territory arrangements in cell nuclei from higher primates.; citation_author=H Tanabe; citation_author=S Muller; citation_author=M Neusser; citation_author=J von Hase; citation_author=E Calcagno; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=3; citation_pages=4424-4429; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Common themes and cell type specific variations of higher order chromatin arrangements in the mouse.; citation_author=R Mayer; citation_author=A Brero; citation_author=J von Hase; citation_author=T Schroeder; citation_author=T Cremer; citation_journal_title=BMC Cell Biol; citation_volume=6; citation_number=4; citation_pages=44; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparative genomic hybridization reveals a specific pattern of chromosomal gains and losses during the genesis of colorectal tumors.; citation_author=T Ried; citation_author=R Knutzen; citation_author=R Steinbeck; citation_author=H Blegen; citation_author=E Schrock; citation_journal_title=Genes Chromosomes Cancer; citation_volume=15; citation_number=5; citation_pages=234-245; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Comparative genomic in situ hybridization of colon carcinomas with replication error.; citation_author=J Schlegel; citation_author=G Stumm; citation_author=H Scherthan; citation_author=T Bocker; citation_author=H Zirngibl; citation_journal_title=Cancer Res; citation_volume=55; citation_number=6; citation_pages=6002-6005; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Recurrent gain of chromosome arm 7q in low-grade astrocytic tumors studied by comparative genomic hybridization.; citation_author=E Schröck; citation_author=C Blume; citation_author=MC Meffert; citation_author=S du Manoir; citation_author=W Bersch; citation_journal_title=Genes Chromosomes Cancer; citation_volume=15; citation_number=7; citation_pages=199-205; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Impact of DNA amplification on gene expression patterns in breast cancer.; citation_author=E Hyman; citation_author=P Kauraniemi; citation_author=S Hautaniemi; citation_author=M Wolf; citation_author=S Mousses; citation_journal_title=Cancer Res; citation_volume=62; citation_number=8; citation_pages=6240-6245; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Microarray analysis reveals a major direct role of DNA copy number alteration in the transcriptional program of human breast tumors.; citation_author=JR Pollack; citation_author=T Sorlie; citation_author=CM Perou; citation_author=CA Rees; citation_author=SS Jeffrey; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=9; citation_pages=12963-12968; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular cytogenetics: Genomic imbalances in colorectal cancer and their clinical impact.; citation_author=M Grade; citation_author=H Becker; citation_author=T Liersch; citation_author=T Ried; citation_author=BM Ghadimi; citation_journal_title=Cell Oncol; citation_volume=28; citation_number=10; citation_pages=71-84; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gene expression profiling reveals a massive, aneuploidy-dependent transcriptional deregulation and distinct differences between lymph node negative and positive colon carcinomas.; citation_author=M Grade; citation_author=P Hörmann; citation_author=S Becker; citation_author=AB Hummon; citation_author=D Wangsa; citation_journal_title=Cancer Research; citation_number=11; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Chromosome transfer induced aneuploidy results in complex dysregulation of the cellular transcriptome in immortalized and cancer cells.; citation_author=MB Upender; citation_author=JK Habermann; citation_author=LM McShane; citation_author=EL Korn; citation_author=JC Barrett; citation_journal_title=Cancer Res; citation_volume=64; citation_number=12; citation_pages=6941-6949; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Inheritance of gene density-related higher order chromatin arrangements in normal and tumor cell nuclei.; citation_author=M Cremer; citation_author=K Kupper; citation_author=B Wagler; citation_author=L Wizelman; citation_author=J von Hase; citation_journal_title=J Cell Biol; citation_volume=162; citation_number=13; citation_pages=809-820; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Relationship of gene expression and chromosomal abnormalities in colorectal cancer.; citation_author=D Tsafrir; citation_author=M Bacolod; citation_author=Z Selvanayagam; citation_author=I Tsafrir; citation_author=J Shia; citation_journal_title=Cancer Res; citation_volume=66; citation_number=14; citation_pages=2129-2137; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Stage-specific alterations of the genome, transcriptome, and proteome during colorectal carcinogenesis.; citation_author=JK Habermann; citation_author=U Paulsen; citation_author=UJ Roblick; citation_author=MB Upender; citation_author=LM McShane; citation_journal_title=Genes Chromosomes Cancer; citation_volume=46; citation_number=15; citation_pages=10-26; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Conservation of relative chromosome positioning in normal and cancer cells.; citation_author=LA Parada; citation_author=PG McQueen; citation_author=PJ Munson; citation_author=T Misteli; citation_journal_title=Curr Biol; citation_volume=12; citation_number=16; citation_pages=1692-1697; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Three-dimensional maps of all chromosomes in human male fibroblast nuclei and prometaphase rosettes.; citation_author=A Bolzer; citation_author=G Kreth; citation_author=I Solovei; citation_author=D Koehler; citation_author=K Saracoglu; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=17; citation_pages=e157; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A chromatin binding site in the tail domain of nuclear lamins that interacts with core histones.; citation_author=H Taniura; citation_author=C Glass; citation_author=L Gerace; citation_journal_title=J Cell Biol; citation_volume=131; citation_number=18; citation_pages=33-44; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The tail domain of lamin Dm0 binds histones H2A and H2B.; citation_author=M Goldberg; citation_author=A Harel; citation_author=M Brandeis; citation_author=T Rechsteiner; citation_author=TJ Richmond; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=19; citation_pages=2852-2857; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Histone lysine methylation patterns in human cell types are arranged in distinct three-dimensional nuclear zones.; citation_author=R Zinner; citation_author=H Albiez; citation_author=J Walter; citation_author=AH Peters; citation_author=T Cremer; citation_journal_title=Histochem Cell Biol; citation_volume=125; citation_number=20; citation_pages=3-19; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Predicting three-dimensional genome structure from transcriptional activity.; citation_author=PR Cook; citation_journal_title=Nat Genet; citation_volume=32; citation_number=21; citation_pages=347-352; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The human transcriptome map: clustering of highly expressed genes in chromosomal domains.; citation_author=H Caron; citation_author=B van Schaik; citation_author=M van der Mee; citation_author=F Baas; citation_author=G Riggins; citation_journal_title=Science; citation_volume=291; citation_number=22; citation_pages=1289-1292; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The human transcriptome map reveals extremes in gene density, intron length, GC content, and repeat pattern for domains of highly and weakly expressed genes.; citation_author=R Versteeg; citation_author=BD van Schaik; citation_author=MF van Batenburg; citation_author=M Roos; citation_author=R Monajemi; citation_journal_title=Genome Res; citation_volume=13; citation_number=23; citation_pages=1998-2004; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Subchromosomal positioning of the epidermal differentiation complex (EDC) in keratinocyte and lymphoblast interphase nuclei.; citation_author=RR Williams; citation_author=S Broad; citation_author=D Sheer; citation_author=J Ragoussis; citation_journal_title=Exp Cell Res; citation_volume=272; citation_number=24; citation_pages=163-175; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Spectral karyotyping, a 24-colour FISH technique for the identification of chromosomal rearrangements.; citation_author=M Macville; citation_author=T Veldman; citation_author=H Padilla-Nash; citation_author=D Wangsa; citation_author=P O'Brien; citation_journal_title=HistochemCell Biol; citation_volume=108; citation_number=25; citation_pages=299-305; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000199" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Artificially Introduced Aneuploid Chromosomes Assume a Conserved Position in Colon Cancer Cells"/>
    <meta name="twitter:description" content="BackgroundChromosomal aneuploidy is a defining feature of carcinomas. For instance, in colon cancer, an additional copy of Chromosome 7 is not only observed in early pre-malignant polyps, but is faithfully maintained throughout progression to metastasis. These copy number changes show a positive correlation with average transcript levels of resident genes. An independent line of research has also established that specific chromosomes occupy a well conserved 3D position within the interphase nucleus.Methodology/Principal FindingsWe investigated whether cancer-specific aneuploid chromosomes assume a 3D-position similar to that of its endogenous homologues, which would suggest a possible correlation with transcriptional activity. Using 3D-FISH and confocal laser scanning microscopy, we show that Chromosomes 7, 18, or 19 introduced via microcell-mediated chromosome transfer into the parental diploid colon cancer cell line DLD-1 maintain their conserved position in the interphase nucleus.ConclusionsOur data is therefore consistent with the model that each chromosome has an associated zip code (possibly gene density) that determines its nuclear localization. Whether the nuclear localization determines or is determined by the transcriptional activity of resident genes has yet to be ascertained."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000199.g005"/>

  <meta property="og:title" content="Artificially Introduced Aneuploid Chromosomes Assume a Conserved Position in Colon Cancer Cells" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000199" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7101'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2657'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=911&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000199">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000199" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000199&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Artificially Introduced Aneuploid Chromosomes Assume a Conserved Position in Colon Cancer Cells
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kundan Sengupta, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Madhvi B. Upender, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Linda Barenboim-Stapleton, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Quang Tri Nguyen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen M. Wincovitch Sr., 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Experimental Carcinogenesis, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>

                <p>
                    Current address: Genetics Disease Research Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Susan H. Garfield, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Experimental Carcinogenesis, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael J. Difilippantonio, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas Ried
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:riedt@mail.nih.gov">riedt@mail.nih.gov</a>
            </p>

                <p>Affiliation:
                  Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 07, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000199</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000199" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000199" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000199" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000199" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000199-g001" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000199-g002" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000199-g003" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000199-g004" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000199-g005" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000199-t001" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000199">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000199" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Chromosomal aneuploidy is a defining feature of carcinomas. For instance, in colon cancer, an additional copy of Chromosome 7 is not only observed in early pre-malignant polyps, but is faithfully maintained throughout progression to metastasis. These copy number changes show a positive correlation with average transcript levels of resident genes. An independent line of research has also established that specific chromosomes occupy a well conserved 3D position within the interphase nucleus.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We investigated whether cancer-specific aneuploid chromosomes assume a 3D-position similar to that of its endogenous homologues, which would suggest a possible correlation with transcriptional activity. Using 3D-FISH and confocal laser scanning microscopy, we show that Chromosomes 7, 18, or 19 introduced via microcell-mediated chromosome transfer into the parental diploid colon cancer cell line DLD-1 maintain their conserved position in the interphase nucleus.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our data is therefore consistent with the model that each chromosome has an associated zip code (possibly gene density) that determines its nuclear localization. Whether the nuclear localization determines or is determined by the transcriptional activity of resident genes has yet to be ascertained.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Sengupta K, Upender MB, Barenboim-Stapleton L, Nguyen QT, Wincovitch SM Sr, et al.  (2007) Artificially Introduced Aneuploid Chromosomes Assume a Conserved Position in Colon Cancer Cells. PLoS ONE 2(2):
          e199.
            doi:10.1371/journal.pone.0000199</p><p><strong>Academic Editor: </strong>Beth Sullivan, Duke University, United States of America</p><p><strong>Received:</strong> December 13, 2006; <strong>Accepted:</strong> January 12, 2007; <strong>Published:</strong> February 7, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>This research was supported by the intramural research program of the NIH, National Cancer Institute.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Chromosomes assume a non-random and conserved position in the interphase nucleus of higher eukaryotes. It is believed that this localization is correlated with their gene densities. For instance, the gene rich Chromosome 19 is predominantly central, while the gene poor Chromosome 18 is peripherally positioned <a href="#pone.0000199-Croft1">[1]</a>. Such a pattern is conserved during evolution, is tissue specific <a href="#pone.0000199-Parada1">[2]</a>, <a href="#pone.0000199-Tanabe1">[3]</a>, and is also maintained when these chromosomes are involved in translocations <a href="#pone.0000199-Croft1">[1]</a>. Extensive studies in mice also show cell type specific, non-random chromosome arrangements based on both gene density and chromosome size <a href="#pone.0000199-Mayer1">[4]</a>. Together, these data suggest a functional significance of chromosome positioning. However, neither the basis for such an arrangement, nor the nature of its structure/function relationship, has yet been revealed. Thus it remains to be determined how the nuclear distribution of chromosomes correlates with their transcriptional activity.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Non-hereditary forms of colon cancer are defined by a non-random and strictly conserved pattern of chromosomal imbalances. For instance, extra copies of Chromosome 7 can be observed as the sole genomic abnormality in colon polyps <a href="#pone.0000199-Ried1">[5]</a>. Additional aneuploidies that result in copy number gains of chromosomes and chromosome arms 8q, 13q and 20, and losses of 8p, 17p, and 18q are sequentially acquired at later stages of colon cancer progression, and are faithfully maintained in both metastatic lesions and cell lines derived from the primary tumors <a href="#pone.0000199-Ried1">[5]</a>–<a href="#pone.0000199-Schrck1">[7]</a>. Through the advent of global gene expression profiling methodologies such as microarrays, it has become possible to identify the consequences of these remarkably conserved chromosomal aneuploidies on the cancer transcriptome. Several recently published studies provide clear evidence that genomic imbalances in tumors directly impact transcript levels <a href="#pone.0000199-Hyman1">[8]</a>–<a href="#pone.0000199-Grade2">[11]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We have previously described the establishment of a unique model system for systematically studying the consequences of aneuploidy on the cellular transcriptome. This model is based on the introduction of specific chromosomes into karyotypically stable immortalized cells or cancer cells using microcell-mediated chromosome transfer. As in primary tumors, an increase in genomic copy number resulted in increased average transcript levels of genes residing on the aneuploid chromosomes. Additionally, the aneuploidy-induced transcriptional deregulation was found to be neither chromosome nor cell type specific <a href="#pone.0000199-Upender1">[12]</a>. Thus, aneuploidy does not appear to target only one or a few genes on the affected chromosome, but results in a massive deregulation of a large portion of the transcriptionally active genes.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In the interphase nuclei of normal and tumor cells, the two homologous chromosomes assume a conserved position, largely correlated with their gene densities <a href="#pone.0000199-Croft1">[1]</a>, <a href="#pone.0000199-Cremer1">[13]</a>. Portions of chromosomes involved in translocations were also observed to orient themselves in such a manner as to localize to their inherent positions <a href="#pone.0000199-Croft1">[1]</a>. We were therefore curious whether an artificially introduced aneuploid chromosome was also capable of finding a position in the nucleus that is similar to its endogeneous homologues. This question, while intriguing of its own accord, was particularly interesting considering the results of our previous study described above <a href="#pone.0000199-Upender1">[12]</a>, which implied that the introduced chromosomes were transcriptionally active. The ability of the introduced chromosome to occupy a specific 3D location would indicate nuclear positioning as a prerequisite for the aneuploidy-induced increase in gene expression. Alternatively, failure to localize to their inherent nuclear space would imply that nuclear positioning of aneuploid chromosomes in cancer cells plays no part in determining their transcriptional activity.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In order to identify the position of aneuploid chromosomes in interphase nuclei, we used 3D-FISH, confocal laser scanning microscopy, and 3D distance measurements on DLD-1 parental and derivative cell lines carrying extra copies of Chromosomes 7, 18 or 19. From a teleological perspective, these experiments further our understanding of the interplay between maintenance of nuclear architecture and genome function. This may impact the way we currently think about treating disease, particularly in aneuploid cancer cells, in which both genomic content and gene expression have been greatly perturbed.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Microcell-mediated chromosome transfer of Chromosomes 7, 18 and 19</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>As previously reported, a single copy of human Chromosome 7 was successfully introduced into the diploid cell line DLD-1, thereby generating the derivative cell line DLD-1+7 (<a href="#pone-0000199-g001">Figure 1A</a>) <a href="#pone.0000199-Upender1">[12]</a>. The additional copy of this chromosome directly and significantly increased the average transcript levels of genes residing on Chromosome 7 <a href="#pone.0000199-Upender1">[12]</a>. This increase was similar upon introduction of Chromosomes 3 and 13 into DLD-1, and was also observed when Chromosome 3 was introduced into normal mammary epithelial cells <a href="#pone.0000199-Upender1">[12]</a>. The increase in transcript levels is therefore independent of the introduced chromosome and independent of the cell type. For the purpose of this study, we generated two additional cell lines by introducing Chromosomes 18 or 19 into DLD-1 thereby creating the derivative cell lines DLD-1+18 and DLD-1+19, respectively (<a href="#pone-0000199-g001">Figure 1A</a>). We chose these chromosomes because they are of equivalent DNA content (<a href="#pone-0000199-g001">Figure 1B</a>) and because their nuclear positions are distinct and conserved: the gene rich Chromosome 19 is positioned towards the interior of the nuclear space, whereas the gene poor Chromosome 18 is located towards the nuclear periphery <a href="#pone.0000199-Croft1">[1]</a>.</p>
<div class="figure" id="pone-0000199-g001"><div class="img"><a name="pone-0000199-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000199.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000199.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A: Schematic representation of the experimental design. DLD-1 (parental cell line) was subjected to MMCT to generated derivative cell lines DLD-1+7, DLD-1+18 and DLD-1+19. 3D-FISH was performed on each of the derivative cell lines with the probe combinations indicated. B: <a href="#pone-0000199-t001">Table</a> showing comparisons of DNA content and gene density between Chromosome 7, 18 and 19.</p>
<span>doi:10.1371/journal.pone.0000199.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The percentage of cells in a given clone maintaining trisomy for the introduced chromosome, despite continued selection, varied depending on the chromosome transferred and the number of passages. Thus, chromosome positioning measurements were strictly confined to DLD-1 derived cells that were trisomic for the artificially introduced chromosome. While early passage clones of DLD-1+3, DLD-1+7 and DLD-1+13 had a high percentage of trisomic cells and were able to maintain this frequency for up to 12 passages, approximately 20% of the cells in the initial clones of DLD-1+18 and DLD-1+19 were trisomic and this was further reduced at very early passages.</p>


<h4>3D Distance Measurement of Chromosome Territories</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We first performed dual-color 3D-FISH on morphologically preserved parental DLD-1 nuclei as outlined in <a href="#pone-0000199-g001">Figure 1A</a>. Representative maximum intensity projections of confocal image stacks from each of the three probe combinations (18 &amp; 19, 7 &amp; 18 and 7 &amp; 19) are shown in <a href="#pone-0000199-g002">Figure 2, panels A–C</a>, respectively. In order to objectively evaluate chromosome territory (CT) positioning and to enable a statistical comparison between the parental cell line and its derivatives, 3D image reconstructions were generated using the software Image-Pro Plus (<a href="#pone-0000199-g003">Figure 3</a>). Since we wanted to take into consideration that not all nuclei are completely spherical, we adopted a 3D measurement scheme similar to that of Tanabe et al. (<a href="#pone.0000199-Tanabe1">[3]</a> see Methods). The ability to obtain these measurements required the addition of a point on the periphery of the nucleus collinear with the geometric center of the nucleus and the geometric center of the chromosome territory (<a href="#pone-0000199-g003">Figure 3C</a>).</p>
<div class="figure" id="pone-0000199-g002"><div class="img"><a name="pone-0000199-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000199.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000199.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Representative maximum intensity projection of confocal image stacks from DLD-1 parental and derived nuclei. A–C: Parental DLD-1 nuclei. D: DLD-1+7 nuclei. E: DLD-1+18 nuclei. F: DLD-1+19 nuclei. DAPI: DNA counterstain; CT-7: Chromosome 7; CT-18: Chromosome 18; CT-19: Chromosome 19; Merge: merged image of DAPI and chromosome territories.</p>
<span>doi:10.1371/journal.pone.0000199.g002</span></div><div class="figure" id="pone-0000199-g003"><div class="img"><a name="pone-0000199-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000199.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000199.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>A: Maximum intensity projection of a representative confocal image stack with Chromosome territories 7 (Red, Spectrum orange) and 19 (Green, Rhodamine Green) from DLD-1+19 B: A 3D reconstruction of the nucleus and chromosome territories from the image shown in A (X-Y orientation). C: A scheme adopted for 3D distance measurements of chromosome territories in Red (R<sub>1</sub> and R<sub>2</sub>) and Green (G<sub>1</sub>, G<sub>2</sub>, and G<sub>3</sub>) from the geometric center of the nucleus (N<sub>c</sub>), to the nuclear periphery (N<sub>P</sub>). Points on the nuclear periphery (eg. N<sub>p</sub>R<sub>1</sub>) are extensions from the nuclear center through the geometric center of the chromosome territory. D: 3D reconstruction in B shown in X-Z orientation.</p>
<span>doi:10.1371/journal.pone.0000199.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The resulting radial distance measurements were plotted for each chromosome territory. As such, the origin at 0% represents the geometric center of the nucleus, while the nuclear border is considered 100%. Measurements of CT-18 and CT-19 in DLD-1 nuclei show that they are positioned predominantly at a radial distance of 70–80% (peripheral) and 40–50% (central), respectively (<a href="#pone-0000199-g004">Figure 4A</a>). This confirmed previous observations on the positioning of Chromosomes 18 and 19 in DLD-1 <a href="#pone.0000199-Cremer1">[13]</a>, and thereby validated our experimental system and analytical procedures. We subsequently performed 3D distance measurements of the intermediately sized, gene poor Chromosome 7 territories. Our results show that CT-7 is radially located in a peripheral position approximately 70–80% from the center of the nucleus (<a href="#pone-0000199-g004">Figure 4A</a>). Similar results were independently obtained using either MIPAV or Imaris software (data not shown).</p>
<div class="figure" id="pone-0000199-g004"><div class="img"><a name="pone-0000199-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000199.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000199.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>Radial distance measurement profiles of chromosome territories in A: DLD-1 B: DLD-1+7, C: DLD-1+18 and D: DLD-1+19. X-axis: Radial Distance (%); Y-axis: Frequency (%); 0 or origin: center of the nucleus; 100%: nuclear periphery; Red: Chromosome 7; Green: Chromosome 19; Blue: Chromosome 18.</p>
<span>doi:10.1371/journal.pone.0000199.g004</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Having determined the positions of Chromosomes 7, 18 and 19 territories in DLD-1, we analyzed the position of these chromosome territories in nuclei of the three derived cell lines (<a href="#pone-0000199-g002">Figure 2D–F</a>). In all instances dual-color 3D-FISH was performed in various labeling combinations. Our analysis of 3D distance measurements for the three Chromosome 7 territories in DLD-1+7 showed that they assumed a peripheral position in the nucleus at a radial distance of 70–80%, much like in the parental cells (<a href="#pone-0000199-g004">Figure 4B</a>). A comparison of the median values of the radial distance profiles between DLD-1 and DLD-1+7 (73.9 and 73.35, respectively) shows that they are nearly identical with a deviation (Δ<sub>M</sub> = −0.55) that was not statistically significant as shown by the Mann-Whitney-Wilcoxon test (P = 0.6811) (<a href="#pone-0000199-t001">Table 1</a>; <a href="#pone-0000199-g005">Figure 5</a>).</p>
<div class="figure" id="pone-0000199-g005"><div class="img"><a name="pone-0000199-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000199.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000199.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec2.sec2.fig4.caption1.p1" name="article1.body1.sec2.sec2.fig4.caption1.p1"></a><p>Raw distributions of 3D-distance measurements. CT-7 in DLD-1 &amp; DLD-1+7, CT-18 in DLD-1 &amp; DLD-1+18, CT-19 in DLD-1 &amp; DLD-1+19. X-axis: cell line, Y-axis: Normalized radial distance (%) of chromosome territories from the geometric center of the nucleus.</p>
<span>doi:10.1371/journal.pone.0000199.g005</span></div><div class="figure" id="pone-0000199-t001"><div class="img"><a name="pone-0000199-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000199" data-uri="info:doi/10.1371/journal.pone.0000199.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000199.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000199.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000199.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000199.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Statistical analyses of radial distance measurements of CT-7,18 and 19</span></strong></p><span>doi:10.1371/journal.pone.0000199.t001</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>3D distance measurements were also performed for Chromosome 18 in DLD-1+18, revealing that the three Chromosome 18 territories are positioned at a radial distance of 80–90% (<a href="#pone-0000199-g002">Figures 2E</a> and <a href="#pone-0000199-g004">4C</a>). The median radial distance values were comparable in the parental and derived nuclei (72.69 and 74.07, respectively; Δ<sub>M</sub> = +1.38) and were determined not to be significantly different by the Mann-Whitney-Wilcoxon test (P = 0.7820) (<a href="#pone-0000199-t001">Table 1</a>; <a href="#pone-0000199-g005">Figure 5</a>).</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Lastly, we determined the nuclear position of Chromosome 19, which was centrally located in the parental DLD-1 cells. 3D reconstructions and distance measurements show that the three Chromosome 19 territories in DLD-1+19 were centrally positioned in the nucleus at a radial distance of 50–60%, equivalent to their position in the parental cell line (51.73 and 55.02, respectively) (<a href="#pone-0000199-g002">Figures 2F</a> and <a href="#pone-0000199-g004">4D</a>). Again, the difference in median values was not statistically significant (Δ<sub>M</sub> = +3.29, P = 0.2677) (<a href="#pone-0000199-t001">Table 1</a>; <a href="#pone-0000199-g005">Figure 5</a>). Our studies therefore show that aneuploid chromosomes introduced via microcell-mediated chromosome transfer assume a conserved 3D position in the nucleus indistinguishable from their endogenous homologues.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>As mentioned above, we performed dual-color hybridizations with two different chromosome painting probes in the combinations described in <a href="#pone-0000199-g001">Figure 1A</a>. We were therefore not only able to assess the position of the introduced, aneuploid chromosomes, but also to query whether this aneuploidy had any affect on the nuclear position of other chromosome pairs. Upon introduction of an extra copy of Chromosome 7, we observed a tendency for CT-18 and CT-19 to be shifted to a more interior position (Δ<sub>M</sub> = −5.59 and Δ<sub>M</sub> = −3.02, respectively). These shifts, however, despite being much greater than those seen in the other derived cells lines, did not reach a statistically significant level (P = 0.0523 and P = 0.0688, respectively) (<a href="#pone-0000199-t001">Table 1</a>). One possible explanation would be that the percent distance measurements for CT-18 and CT-19 in DLD-1+7 had a bimodal distribution and a relaxation of chromosome positioning. The degree of spread as calculated using the weighted-average-Inter Quartile Range (IQR) was 23.84 and 21.08 (for CT-18 and CT-19, respectively) compared to 11.90 for Chromosome 7 (<a href="#pone-0000199-g004">Figure 4B</a>). The introduction of Chromosome 18 had no effect on the position of the two Chromosome 7 territories as they remained at a peripheral position of 70–80% and as such the median radial distance values did not demonstrate a significant shift in position (P = 0.4216). However, the two Chromosome 19 territories were once again shifted more centrally with a radial distance of ~40% in comparison to ~55% in the DLD-1 nuclei (<a href="#pone-0000199-g004">Figure 4C</a>). The Mann-Whitney-Wilcoxon test demonstrated that the Δ<sub>M</sub> = −8.19 was statistically significant (P = 0.0307). The comparison of median radial distance values of CT-7 in DLD-1+19, however, suggested that the position of this chromosome was significantly shifted (P = 0.0299) towards the periphery (73.90 and 77.52; Δ<sub>M</sub> = +3.62). In addition, Chromosome 18 territories were significantly shifted to a more internal position (72.69 and 66.65; Δ<sub>M</sub> = −6.04, P = 0.0257).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The systematic exploration of the consequences of chromosomal aneuploidies on gene expression profiles has shown that a direct relationship exists between genomic copy number and transcript levels <a href="#pone.0000199-Hyman1">[8]</a>, <a href="#pone.0000199-Grade1">[10]</a>, <a href="#pone.0000199-Tsafrir1">[14]</a>, <a href="#pone.0000199-Habermann1">[15]</a>. In order to generate a model system of chromosomal aneuploidy, we used microcell-mediated chromosome transfer to introduce specific chromosomes into karyotypically stable cells <a href="#pone.0000199-Upender1">[12]</a>. The results confirmed earlier observations in primary tumors and cancer cell lines, showing a direct impact of chromosomal aneuploidy on resident gene expression levels. This allowed us to study the relationship between aneuploidy and gene expression independent of other cytogenetic abnormalities usually observed in cancer genomes. Having established that the generation of artificial trisomies resulted in a significant increase in average transcript levels of genes on these aneuploid chromosomes, we were now curious as to whether they assume a conserved position in the interphase nucleus. This is an important question because there is firm evidence that native, endogenous mammalian chromosomes occupy specific, conserved 3D positions <a href="#pone.0000199-Tanabe1">[3]</a>. For instance, the gene rich Chromosome 19 is localized more centrally, while the gene poor Chromosome 18 territories are positioned more towards the periphery of the nucleus <a href="#pone.0000199-Croft1">[1]</a>. It is therefore reasonable to surmise a functional relevance of this structural conservation and, as an extension of that, a relationship between 3D architecture and transcriptional activity. With the aim to determine if the increased gene expression correlates with the placement of the introduced chromosome into its conserved nuclear space (e.g., interior for Chromosome 19, and peripheral for Chromosome 18), we performed 3D-FISH on three derived cell lines trisomic for Chromosomes 7, 18 or 19. The DNA mismatch repair deficient colon cancer cell line DLD-1 was used as the recipient cell line. This cell line, as are others with microsatellite instability, is karyotypically stable and diploid. This is advantageous because the position of introduced chromosomes can be assessed without potential confounding effects from other chromosomal aberrations. Here we report that an artificially introduced aneuploid chromosome assumes a non-random and conserved 3D position in the interphase nucleus that is equivalent to the localization of its other two endogenous homologues.</p>

<h4>Positioning of Chromosome 7, 18 and 19 territories</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Our analysis of chromosome territories in DLD-1 showed that CT-18 and CT-19 were predominantly peripheral and central, respectively, thus corroborating earlier observations in this cell line <a href="#pone.0000199-Cremer1">[13]</a>. Of note, Cremer et al. reported a smaller difference in the average radial distance between CT-18 and CT-19 in DLD-1 nuclei (~7.9%) and other tumor nuclei, while our analysis showed ~18.4% difference between the means of the radial distance measurements of CT-18 and CT-19. However, both studies clearly establish that Chromosome 19 is positioned more towards the interior of the nucleus compared to Chromosome 18. We also show that the gene poor Chromosome 7 is predominantly peripheral in DLD-1, further supporting a gene density based chromosome positioning pattern in both normal and tumor nuclei (<a href="#pone-0000199-g004">Figure 4A</a>) <a href="#pone.0000199-Cremer1">[13]</a>, <a href="#pone.0000199-Parada2">[16]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The primary objective of this study was to assess the relative positioning of the artificially introduced trisomic chromosome compared to its endogenous homologues in all three of the derivative cell lines. We were, however, unable to produce a robust signal with a neomycin FISH probe that would unequivocally denote the introduced chromosome, which is tagged with this selectable marker (data not shown). This was, however, not a major impediment since our statistical analysis did not reveal any significant differences between the localization of any of the three chromosome copies. For instance, all of the Chromosome 7 territories in DLD-1+7 assume a relatively peripheral position in the nucleus (<a href="#pone-0000199-g004">Figure 4B</a>). A similar result was obtained for the 18 (peripheral) and 19 (central) chromosome territories in the nuclei of their respective trisomic cell lines (<a href="#pone-0000199-g004">Figure 4, panels C and D</a>). Thus, there appears to be some mechanism whereby the artificially introduced trisomic chromosomes localize to their innate conserved 3D nuclear position.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>A still unexplained finding was the statistically significant, but subtle shift in the median position of Chromosome 19 in the DLD-1+18 cells (<a href="#pone-0000199-t001">Table 1</a>). In addition the mean radial distance between CT-18 and CT-19 increased from 18.4% to 22.69% in these nuclei. One might imagine that extra chromosomes occupying peripheral positions such as Chromosomes 7 or 18 might cause Chromosome 19 to assume an even more central position. Arguing against this reasoning is the fact that the CT-19 shift in DLD-1+7 cells was not significant (<a href="#pone-0000199-t001">Table 1</a>). Additionally, in DLD-1+19, CT-7 is shifted to a more peripheral position while CT-18 is shifted to a significantly more internal position, resulting in a smaller difference in the mean distances between CT-18 and CT-19 (~11.37%). Thus, while it is relatively easy to understand that the addition of extra chromosome territories into a physically constrained space such as the nucleus would have the potential to induce shifts in the positioning of the other chromosomes, what determines the directionality of that shift is not self-evident. It will be interesting to ascertain how these effects are compounded in aneuploid cancer cells that frequently contain far more than just the one numerical aberration as in our model system. This may possibly be an additional factor in explaining the enormous complexity of gene deregulation in the cancer transcriptome.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>We also observed a bimodal distribution of chromosome territories, particularly in the derivative cell lines (<a href="#pone-0000199-g004">Fig. 4A</a>). For example, in a population of DLD-1 nuclei (~6–8%), CT-18 occupied a more internal position as reflected in a peak at a radial distance of ~50% from the center of the nucleus. Careful analysis of the raw data did not indicate that this bimodality was a reflection of one chromosome territory in each nucleus behaving differently (e.g., the introduced chromosome), but rather that in some cells all three territories were more central or peripheral relative to the mean. While the relative position of Chromosome 18 and 19 territories is conserved in a wide range of cell types, the degree of this conservation can vary. For instance, some tumor nuclei also showed a decline in the normal radial distribution pattern of CT-18 and CT-19 compared to normal cells. This is particularly apparent for nuclei of the aneuploid colon cancer cell line SW480, where CT-18 and 19 are rather closely positioned <a href="#pone.0000199-Cremer1">[13]</a>, suggesting that aneuploidy or additional chromosomal gains could influence the gene density based radial distribution of chromosomes.</p>


<h4>A speculative mechanism of chromosome positioning</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>It is now clearly established that the positioning of chromosome territories within the 3D space of the interphase nucleus is non-random. This distribution is conserved across different tissues, both normal and malignant, as well as evolutionarily across divergent species <a href="#pone.0000199-Parada1">[2]</a>, <a href="#pone.0000199-Tanabe1">[3]</a>. The experiments performed in this study now demonstrate that this non-random and conserved nuclear localization also extends to artificially introduced, aneuploid chromosomes. Thus, such a high degree of conservation lends itself to the idea that there must be some biological implication for the placement of chromosome territories. But how is the functional reorganization of the nucleus established upon reformation of the nucleus after mitosis? Can such a phenomenon be explained mechanistically?</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>To reiterate the facts: chromosomes with a relatively high gene density occupy a more central position while gene poor chromosomes tend to be localized closer to the nuclear periphery <a href="#pone.0000199-Croft1">[1]</a>. It is also true that gene rich chromosomes have a higher G-C content. This may partially reflect the presence of CpG islands in gene promoters as well as the preponderance of G-C rich repetitive elements such as Alu sequences that are coincident with coding regions of the genome. In fibroblasts, for instance, an enhanced staining of Alu sequences was found in the nuclear interior <a href="#pone.0000199-Bolzer1">[17]</a>. Conversely, the nuclear periphery is enriched for heterochromatin, which has a tendency to be more A-T rich. It is well known that nuclear lamins are critical for the reformation of the nucleus after mitosis. Lamins have also been shown to interact through specific sequences in their tail domain with chromatin and in particular with two of the core histones H2A and H2B <a href="#pone.0000199-Taniura1">[18]</a>, <a href="#pone.0000199-Goldberg1">[19]</a>. An increased presence of methylated histones, such as tri-H3K27, has been observed near the nuclear periphery <a href="#pone.0000199-Zinner1">[20]</a>. Thus, lamins and variations in nucleosome composition and/or modifications may play a role in the non-random positioning of certain chromosome territories near the nuclear membrane.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>What possible factors might be responsible for establishing the above noted features of the nuclear architecture, particularly with respect to the positioning of individual chromosome territories? Perhaps the most intuitive model is one in which each chromosome is identified by a unique “zip-code” that determines where it will reside in the nucleus. One such distinguishing mark could be the unique sequences found in the centromeric or pericentromeric region of each homologue. This hypothesis could be tested experimentally by moving these sequences from one chromosome to another. Fortunately, such events occur naturally via chromosomal translocations. For example, the cancer cell line SW620 contains a der(18)t(17;18) in which material from the gene-rich Chromosome 17 has been translocated to the centromere containing gene-poor Chromosome 18. Despite containing the Chromosome 18 centromere, this derivative chromosome occupies a radial localization similar to the normal Chromosome 17 <a href="#pone.0000199-Cremer1">[13]</a>. This study therefore suggests that chromosome specific centromeres are not the main determinant of chromosome positioning and points more towards the material contained in the chromosome arms.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>An alternative to a centromere-specific “zip-code” sequence would be one in which a rather general feature of each chromosome is responsible for placing it in, or excluding it from, certain nuclear regions. In such a model it becomes imperative to explain how features such as gene density, nucleotide composition (G-C versus A-T content), DNA and histone modifications or transcriptional activity are sensed by the re-forming nucleus and are used to establish positioning. Since each of these features is present to a different extent on every chromosome, positioning of territories becomes more probabilistic than definitive. This is consistent with our experimental observations (<a href="#pone-0000199-g004">Figure 4</a>).</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>As an example, we propose the following scenario as a possible mechanism for establishing the interphase nuclear architecture. Non-transcribed, gene-poor regions of the genome tend to be more heterochromatic, which is predominantly A-T rich. Heterochromatin is established through a combination of DNA and histone modifications that are known to correlate with transcriptional inactivity. Thus, a higher absolute number or concentration of modified nucleosomes, for example tri-H3K27, might make it more likely for a gene-poor chromosome to be snared by lamins attached to the inside of the reforming nuclear membrane. One could then postulate that by default, unsnared G-C rich, gene rich, transcriptionally active chromosomes would have a tendency to be excluded from the nuclear periphery and thus are resolved to occupy a more central nuclear position. In this self-organizing system, the localization of gene-rich sequences in the center of the nucleus is not so much the driving force, but rather the end result of nuclear reformation after mitosis. Others have put forth a self-organizing model wherein the collective transcriptional activity of the genome has been proposed to dictate nuclear architecture based on the physical properties of chromatin and interacting polymerases <a href="#pone.0000199-Cook1">[21]</a>. Since it is likely that there is very little ongoing transcription in mitotically condensed chromosomes, we would posit that it is not active transcription per se which determines the architecture upon nuclear reformation, but rather the markings of previous transcriptionally active or inactive regions such as DNA and histone modifications.</p>
<a id="article1.body1.sec3.sec2.p6" name="article1.body1.sec3.sec2.p6"></a><p>Gene rich chromosomes are also transcribed more actively. Genome-wide analysis of mRNA expression profiles of the human genome shows that gene dense regions strongly correlate with <span class="underline">R</span>egions of <span class="underline">I</span>ncrease<span class="underline">d G</span>ene <span class="underline">E</span>xpression (RIDGES) <a href="#pone.0000199-Caron1">[22]</a>, <a href="#pone.0000199-Versteeg1">[23]</a>. This would require enrichment or a gradient of increasing concentration of transcription factors or factories towards the nuclear center where there is more transcriptional activity. It would be interesting to determine if such gradients actually exist in the nucleus. If there is a gradient, is it the reason for the non-random distribution of chromosomes or is it established in response to such a nuclear architecture? If a gradient does not exist, is the uniform concentration of transcription factors limiting in gene dense areas of the nucleus with high transcriptional activity? Are the factors in the nuclear interior more transcriptionally engaged than those towards the periphery? Does the higher concentration of heterochromatin in the nuclear periphery restrict not only the accessibility of transcription factors to chromatin, but also impede their ability to traverse the interior of chromosome territories? Answers to these questions will provide significant insight into the role of chromosome positioning in regulating gene expression, a factor which might need to be considered in cells with rearranged genomes.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Microcell-Mediated Chromosome Transfer (MMCT)</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Microcell mediated chromosome transfer (MMCT) was performed as previously described <a href="#pone.0000199-Upender1">[12]</a>. Briefly, the nuclei of A9 mouse cells maintaining a single human chromosome under G418 selection were fragmented in the presence of cytochalasin-B, sequentially filtered through 8.0, 5.0 and 3.0 µm filters and the purified micronuclei were then fused with the recipient diploid DLD-1 colon cancer cell line using PEG 1500 (Roche, Indianapolis, IN). The derivative cell lines DLD-1+7, DLD-1+18 and DLD-1+19 were generated by isolating single colonies under continuous selection in G418 (200 µg/ml, Geneticin, Invitrogen, Carlsbad, CA). Individual lines were then assessed for the presence of the specific intact transferred human chromosome by 2D FISH with chromosome-specific painting probes on metaphase preparations and the absence of mouse chromosomes as identified morphologically on DAPI stained metaphase preparations.</p>


<h4>Cell Culture</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>DLD-1 adenocarcinoma cells were grown at 37°C in the presence of 5% CO<sub>2</sub> in RPMI-1640 media supplemented with 1% L-Glutamine, Penicillin (50 units/ml)/Streptomycin (50 µg/ml) and 10% heat inactivated FBS. The derivative cell lines were grown in the same media supplemented with G418 (200 µg/ml). The cells were plated onto glass chamber slides at an appropriate dilution and allowed to attach overnight at 37°C prior to 3D-FISH analysis.</p>


<h4>Cell fixation and permeabilization</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Morphologically preserved nuclei were prepared by a modification of the protocol for 3D-FISH <a href="#pone.0000199-Williams1">[24]</a>. Cells grown on chamber slides were washed 3 times in 1× PBS for 5 minutes each. The cells were incubated on ice for 5 minutes in CSK buffer (0.1 M NaCl, 0.3 M Sucrose, 3 mM MgCl<sub>2</sub>, 10 mM PIPES adjust pH to 7.4, 0.5% Triton-X-100) and immediately fixed in 4% Paraformaldehyde (PFA) (prepared in 1× PBS (pH = 7.4)) for 5 minutes at RT. The cells were washed in 1.0 M Tris-HCl (pH 7.4) followed by 1× PBS washes 2 times at RT for 5 minutes each. The cells were permeabilized in 0.5% Triton-X-100 (prepared in 1× PBS) for 10 minutes and incubated in 20% glycerol (prepared in 1× PBS) for 60 minutes followed by four freeze-thaw cycles in liquid nitrogen. The cells were washed three times in 1× PBS for 5 minutes each and incubated in 0.1 N HCl for 10 minutes followed by three washes in 1× PBS for 5 minutes each. The cells were stored in 50% formamide (FA)/2× SSC (pH 7.4) overnight at 4°C or until used for hybridization <a href="#pone.0000199-Williams1">[24]</a>.</p>


<h4>3D FISH</h4>
<h5>Chromosome painting probes.</h5><a id="article1.body1.sec4.sec4.sec1.p1" name="article1.body1.sec4.sec4.sec1.p1"></a><p>Flow sorted chromosomes 7, 18 and 19 (purchased from M.A Ferguson-Smith and Patricia O'Brien, Univ. of Cambridge, U.K.) were DOP-PCR labeled with either Rhodamine green (Invitrogen) or Spectrum orange (Vysis) as described <a href="#pone.0000199-Macville1">[25]</a> to generate whole chromosome painting probes. Two differentially labeled chromosome painting probes (1.25 µg) were combined and precipitated with Cot-1 DNA (12.5 µg) in 100% cold ethanol and sodium acetate for 2 hours at −80°C, and centrifuged at 14,000 rpm at 4°C for 30 minutes. The probe was dried under vacuum for 5 minutes and subsequently resuspended in 2.5 µl 100% formamide for 30 minutes shaking in a thermomixer at 37°C and for an additional 30 minutes with 2.5 µl mastermix (50% Dextran sulfate and 2× SSC) at 37°C. The probe was denatured for 5 minutes at 80°C and pre-annealed for 60 minutes at 37°C.</p>

<h5>Hybridization.</h5><a id="article1.body1.sec4.sec4.sec2.p1" name="article1.body1.sec4.sec4.sec2.p1"></a><p>120 µl of 70% deionized formamide/2× SSC was applied onto the slide, which was covered with a cover glass and denatured at 78°C for 5 minutes. Immediately following denaturation the excess formamide was shaken off the slide and 5 µl of the probe was spotted to the area of hybridization on which an 18×18 mm<sup>2</sup> cover glass was placed and sealed with rubber cement. The slides were hybridized for 48 hours in a humid box at 37°C.</p>

<h5>Detection.</h5><a id="article1.body1.sec4.sec4.sec3.p1" name="article1.body1.sec4.sec4.sec3.p1"></a><p>The slides were washed in 50% FA/2× SSC, 3 times, shaking for 5 minutes each at 45°C, followed by three washes for 5 minutes, shaking each in 0.1×SSC at 60°C, briefly rinsed in 0.1% Tween20/4× SSC, were counterstained with 4′,6′-diamidino-2-phenylindole (DAPI) for 5 minutes, washed in 2× SSC and mounted in antifade (1,4-phenyline diamine), (prepared in 86% glycerol and pH adjusted to 8.0 with carbonate buffer).</p>



<h4>Confocal Imaging</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Confocal images were acquired on a Zeiss LSM 510 NLO confocal system (Carl Zeiss, Thornwood, NJ, USA) with a 100× Plan-Apochromat 1.4 NA oil immersion objective using scan zoom of 2. Z-stacked images were acquired at 512×512 pixels per frame using 8-bit pixel depth for each channel at a voxel size of 0.87 µm×0.87 µm×0.3 µm and line averaging set to 4 collected sequentially in a multi-track, three channel mode.</p>


<h4>3D reconstructions</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Individual nuclei from a merged confocal stack were manually cropped from a given field and 3D reconstructions were performed using Image-Pro Plus (v 5.1) (MediaCybernetics, Silver Spring, MD, USA). Segmentation of the chromosome territories was done by setting a visual threshold of the images in each channel using the original RGB image as a template. In addition, threshold values were independently determined for each channel by using extended depth of field measurements, in which threshold values were typically found to vary between 10–15 units for each channel from the isosurface values. In the 3D-constructor module of Image-Pro Plus, a 3×3×3 lo-pass 3D-filter was applied to each cropped nuclei in all channels to reduce background noise. Surface rendering was performed independently on each channel to obtain the geometric centers of the nuclei (blue channel, DAPI) and the chromosome territories (red channel, Spectrum orange and green channel, Rhodamine Green), respectively (<a href="#pone-0000199-g003">Figure 3</a>).</p>


<h4>3D-distance measurements</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>3D distance measurements were performed using Image-Pro Plus (v 5.1). Each nucleus was segmented into 10 equal shells following the model of Tanabe and colleagues <a href="#pone.0000199-Tanabe1">[3]</a>. As shown in <a href="#pone-0000199-g003">Figure 3C</a>, the geometric center of the DAPI stained nucleus (N<sub>C</sub>) and the chromosome territories (e.g., Red (Spectrum orange) chromosome territories R<sub>1</sub>, R<sub>2</sub>, etc.) were determined. These geometric centers were connected and extended to a third collinear point on the nuclear periphery (e.g. N<sub>P</sub>R<sub>1</sub>). The relative distance of a chromosome territory from the center of the nucleus was calculated as a percent of the total distance from the center of the nucleus to the nuclear periphery. For example, for the chromosome territory in the Spectrum Orange channel (R<sub>1</sub>):<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000199.e001&amp;representation=PNG" class="inline-graphic"></span><br>
            </p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>The %CT distance was preferred over the raw distance measurement from the nuclear center in order to scale for variations in nuclear shapes that deviate from a perfect sphere. A merged confocal image stack was subjected to 3D reconstructions (<a href="#pone-0000199-g003">Figure 3A, B</a>) and the distance measurements for each territory were determined. At least 30 nuclei were analyzed for each chromosome combination from DLD-1 and each of the derived cell lines (<a href="#pone-0000199-g001">Figure 1A</a>). Segmentation and 3D-distance measurements were also performed independently using either MIPAV software (CIT, NIH, Bethesda, MD) or Imaris software (Bitplane, Zurich, Switzerland).</p>


<h4>Statistical analyses</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>The Mann-Whitney-Wilcoxon sum-rank test was used to compare the 3D-distance measurements of chromosome territories between the parental DLD-1 cell line and each of the derivative cell lines. P-values were calculated using Graph pad Prism 3.0 software and were considered statistically significant only when the P-value &lt;0.05 (two sided). Graphical plots of the distance measurements were generated using Sigma Plot 9.0.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We sincerely thank Dr. Marion Cremer, Ludwig-Maximilians-Universität München, Martinsried, Germany for extensive advice on 3D-FISH procedures. Paul Leo and Amalia Dutra, NHGRI/NIH for help with confocal laser scanning microscopy. We also thank William Gandler and Mathew McAuliffe, CIT/NIH for expanding the capacity of MIPAV to make it amenable for measuring chromosome territories and Chris Tully of Media Cybernetics for input on 3D-measurements and analyses using Image Pro Plus. We thank all the members of our laboratory for useful discussions and Buddy Chen, Joseph Cheng and Tom Ellerman for IT support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MD TR KS MU. Performed the experiments: KS MU. Analyzed the data: MD KS QN SW. Contributed reagents/materials/analysis tools: MD KS MU LB SW SG. Wrote the paper: MD TR KS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000199-Croft1" id="pone.0000199-Croft1"></a>Croft JA, Bridger JM, Boyle S, Perry P, Teague P, et al.  (1999) Differences in the localization and morphology of chromosomes in the human nucleus. J Cell Biol  145: 1119–1131.  <ul class="find" data-citedArticleID="929244" data-doi="10.1083/jcb.145.6.1119"><li><a href="http://dx.doi.org/10.1083/jcb.145.6.1119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differences+in+the+localization+and+morphology+of+chromosomes+in+the+human+nucleus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differences+in+the+localization+and+morphology+of+chromosomes+in+the+human+nucleus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000199-Parada1" id="pone.0000199-Parada1"></a>Parada LA, McQueen PG, Misteli T (2004) Tissue-specific spatial organization of genomes. Genome Biol  5: R44.  <ul class="find" data-citedArticleID="929260" data-doi="10.1186/gb-2004-5-7-r44"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-7-r44" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tissue-specific+spatial+organization+of+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tissue-specific+spatial+organization+of+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000199-Tanabe1" id="pone.0000199-Tanabe1"></a>Tanabe H, Muller S, Neusser M, von Hase J, Calcagno E, et al.  (2002) Evolutionary conservation of chromosome territory arrangements in cell nuclei from higher primates. Proc Natl Acad Sci U S A  99: 4424–4429.  <ul class="find" data-citedArticleID="929272" data-doi="10.1073/pnas.072618599"><li><a href="http://dx.doi.org/10.1073/pnas.072618599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+conservation+of+chromosome+territory+arrangements+in+cell+nuclei+from+higher+primates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+conservation+of+chromosome+territory+arrangements+in+cell+nuclei+from+higher+primates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000199-Mayer1" id="pone.0000199-Mayer1"></a>Mayer R, Brero A, von Hase J, Schroeder T, Cremer T, et al.  (2005) Common themes and cell type specific variations of higher order chromatin arrangements in the mouse. BMC Cell Biol  6: 44.  <ul class="find" data-citedArticleID="929258" data-doi="10.1186/1471-2121-6-44"><li><a href="http://dx.doi.org/10.1186/1471-2121-6-44" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Common+themes+and+cell+type+specific+variations+of+higher+order+chromatin+arrangements+in+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Common+themes+and+cell+type+specific+variations+of+higher+order+chromatin+arrangements+in+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000199-Ried1" id="pone.0000199-Ried1"></a>Ried T, Knutzen R, Steinbeck R, Blegen H, Schrock E, et al.  (1996) Comparative genomic hybridization reveals a specific pattern of chromosomal gains and losses during the genesis of colorectal tumors. Genes Chromosomes Cancer  15: 234–245.  <ul class="find" data-citedArticleID="929266" data-doi="10.1002/(sici)1098-2264(199604)15:4<234::aid-gcc5&gt;3.0.co;2-2"><li><a href="http://dx.doi.org/10.1002/(sici)1098-2264(199604)15:4<234::aid-gcc5&gt;3.0.co;2-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomic+hybridization+reveals+a+specific+pattern+of+chromosomal+gains+and+losses+during+the+genesis+of+colorectal+tumors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomic+hybridization+reveals+a+specific+pattern+of+chromosomal+gains+and+losses+during+the+genesis+of+colorectal+tumors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000199-Schlegel1" id="pone.0000199-Schlegel1"></a>Schlegel J, Stumm G, Scherthan H, Bocker T, Zirngibl H, et al.  (1995) Comparative genomic in situ hybridization of colon carcinomas with replication error. Cancer Res  55: 6002–6005.  <ul class="find" data-citedArticleID="929268"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+genomic+in+situ+hybridization+of+colon+carcinomas+with+replication+error.&amp;auth=&amp;atitle=Comparative+genomic+in+situ+hybridization+of+colon+carcinomas+with+replication+error." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomic+in+situ+hybridization+of+colon+carcinomas+with+replication+error." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomic+in+situ+hybridization+of+colon+carcinomas+with+replication+error.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000199-Schrck1" id="pone.0000199-Schrck1"></a>Schröck E, Blume C, Meffert MC, du Manoir S, Bersch W, et al.  (1996) Recurrent gain of chromosome arm 7q in low-grade astrocytic tumors studied by comparative genomic hybridization. Genes Chromosomes Cancer  15: 199–205.  <ul class="find" data-citedArticleID="929270" data-doi="10.1002/(sici)1098-2264(199604)15:4<199::aid-gcc1&gt;3.0.co;2-x"><li><a href="http://dx.doi.org/10.1002/(sici)1098-2264(199604)15:4<199::aid-gcc1&gt;3.0.co;2-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recurrent+gain+of+chromosome+arm+7q+in+low-grade+astrocytic+tumors+studied+by+comparative+genomic+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recurrent+gain+of+chromosome+arm+7q+in+low-grade+astrocytic+tumors+studied+by+comparative+genomic+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000199-Hyman1" id="pone.0000199-Hyman1"></a>Hyman E, Kauraniemi P, Hautaniemi S, Wolf M, Mousses S, et al.  (2002) Impact of DNA amplification on gene expression patterns in breast cancer. Cancer Res  62: 6240–6245.  <ul class="find" data-citedArticleID="929254"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Impact+of+DNA+amplification+on+gene+expression+patterns+in+breast+cancer.&amp;auth=&amp;atitle=Impact+of+DNA+amplification+on+gene+expression+patterns+in+breast+cancer." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impact+of+DNA+amplification+on+gene+expression+patterns+in+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impact+of+DNA+amplification+on+gene+expression+patterns+in+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000199-Pollack1" id="pone.0000199-Pollack1"></a>Pollack JR, Sorlie T, Perou CM, Rees CA, Jeffrey SS, et al.  (2002) Microarray analysis reveals a major direct role of DNA copy number alteration in the transcriptional program of human breast tumors. Proc Natl Acad Sci U S A  99: 12963–12968.  <ul class="find" data-citedArticleID="929264" data-doi="10.1073/pnas.162471999"><li><a href="http://dx.doi.org/10.1073/pnas.162471999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+analysis+reveals+a+major+direct+role+of+DNA+copy+number+alteration+in+the+transcriptional+program+of+human+breast+tumors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+analysis+reveals+a+major+direct+role+of+DNA+copy+number+alteration+in+the+transcriptional+program+of+human+breast+tumors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000199-Grade1" id="pone.0000199-Grade1"></a>Grade M, Becker H, Liersch T, Ried T, Ghadimi BM (2006) Molecular cytogenetics: Genomic imbalances in colorectal cancer and their clinical impact. Cell Oncol  28: 71–84.  <ul class="find" data-citedArticleID="929248"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+cytogenetics%3A+Genomic+imbalances+in+colorectal+cancer+and+their+clinical+impact.&amp;auth=&amp;atitle=Molecular+cytogenetics%3A+Genomic+imbalances+in+colorectal+cancer+and+their+clinical+impact." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cytogenetics%3A+Genomic+imbalances+in+colorectal+cancer+and+their+clinical+impact." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cytogenetics%3A+Genomic+imbalances+in+colorectal+cancer+and+their+clinical+impact.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000199-Grade2" id="pone.0000199-Grade2"></a>Grade M, Hörmann P, Becker S, Hummon AB, Wangsa D, et al.  (2006) Gene expression profiling reveals a massive, aneuploidy-dependent transcriptional deregulation and distinct differences between lymph node negative and positive colon carcinomas. Cancer Research.   In Press.  <ul class="find" data-citedArticleID="929250" data-doi="10.1158/0008-5472.can-06-1514"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-1514" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+profiling+reveals+a+massive%2C+aneuploidy-dependent+transcriptional+deregulation+and+distinct+differences+between+lymph+node+negative+and+positive+colon+carcinomas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+profiling+reveals+a+massive%2C+aneuploidy-dependent+transcriptional+deregulation+and+distinct+differences+between+lymph+node+negative+and+positive+colon+carcinomas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000199-Upender1" id="pone.0000199-Upender1"></a>Upender MB, Habermann JK, McShane LM, Korn EL, Barrett JC, et al.  (2004) Chromosome transfer induced aneuploidy results in complex dysregulation of the cellular transcriptome in immortalized and cancer cells. Cancer Res  64: 6941–6949.  <ul class="find" data-citedArticleID="929278" data-doi="10.1158/0008-5472.can-04-0474"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-04-0474" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+transfer+induced+aneuploidy+results+in+complex+dysregulation+of+the+cellular+transcriptome+in+immortalized+and+cancer+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+transfer+induced+aneuploidy+results+in+complex+dysregulation+of+the+cellular+transcriptome+in+immortalized+and+cancer+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000199-Cremer1" id="pone.0000199-Cremer1"></a>Cremer M, Kupper K, Wagler B, Wizelman L, von Hase J, et al.  (2003) Inheritance of gene density-related higher order chromatin arrangements in normal and tumor cell nuclei. J Cell Biol  162: 809–820.  <ul class="find" data-citedArticleID="929242" data-doi="10.1083/jcb.200304096"><li><a href="http://dx.doi.org/10.1083/jcb.200304096" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inheritance+of+gene+density-related+higher+order+chromatin+arrangements+in+normal+and+tumor+cell+nuclei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inheritance+of+gene+density-related+higher+order+chromatin+arrangements+in+normal+and+tumor+cell+nuclei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000199-Tsafrir1" id="pone.0000199-Tsafrir1"></a>Tsafrir D, Bacolod M, Selvanayagam Z, Tsafrir I, Shia J, et al.  (2006) Relationship of gene expression and chromosomal abnormalities in colorectal cancer. Cancer Res  66: 2129–2137.  <ul class="find" data-citedArticleID="929276" data-doi="10.1158/0008-5472.can-05-2569"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-05-2569" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+of+gene+expression+and+chromosomal+abnormalities+in+colorectal+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+of+gene+expression+and+chromosomal+abnormalities+in+colorectal+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000199-Habermann1" id="pone.0000199-Habermann1"></a>Habermann JK, Paulsen U, Roblick UJ, Upender MB, McShane LM, et al.  (2007) Stage-specific alterations of the genome, transcriptome, and proteome during colorectal carcinogenesis. Genes Chromosomes Cancer  46: 10–26.  <ul class="find" data-citedArticleID="929252" data-doi="10.1002/gcc.20382"><li><a href="http://dx.doi.org/10.1002/gcc.20382" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stage-specific+alterations+of+the+genome%2C+transcriptome%2C+and+proteome+during+colorectal+carcinogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stage-specific+alterations+of+the+genome%2C+transcriptome%2C+and+proteome+during+colorectal+carcinogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000199-Parada2" id="pone.0000199-Parada2"></a>Parada LA, McQueen PG, Munson PJ, Misteli T (2002) Conservation of relative chromosome positioning in normal and cancer cells. Curr Biol  12: 1692–1697.  <ul class="find" data-citedArticleID="929262" data-doi="10.1016/s0960-9822(02)01166-1"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)01166-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservation+of+relative+chromosome+positioning+in+normal+and+cancer+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservation+of+relative+chromosome+positioning+in+normal+and+cancer+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000199-Bolzer1" id="pone.0000199-Bolzer1"></a>Bolzer A, Kreth G, Solovei I, Koehler D, Saracoglu K, et al.  (2005) Three-dimensional maps of all chromosomes in human male fibroblast nuclei and prometaphase rosettes. PLoS Biol  3: e157.  <ul class="find" data-citedArticleID="929236" data-doi="10.1371/journal.pbio.0030157"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three-dimensional+maps+of+all+chromosomes+in+human+male+fibroblast+nuclei+and+prometaphase+rosettes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three-dimensional+maps+of+all+chromosomes+in+human+male+fibroblast+nuclei+and+prometaphase+rosettes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000199-Taniura1" id="pone.0000199-Taniura1"></a>Taniura H, Glass C, Gerace L (1995) A chromatin binding site in the tail domain of nuclear lamins that interacts with core histones. J Cell Biol  131: 33–44.  <ul class="find" data-citedArticleID="929274" data-doi="10.1083/jcb.131.1.33"><li><a href="http://dx.doi.org/10.1083/jcb.131.1.33" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+chromatin+binding+site+in+the+tail+domain+of+nuclear+lamins+that+interacts+with+core+histones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+chromatin+binding+site+in+the+tail+domain+of+nuclear+lamins+that+interacts+with+core+histones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000199-Goldberg1" id="pone.0000199-Goldberg1"></a>Goldberg M, Harel A, Brandeis M, Rechsteiner T, Richmond TJ, et al.  (1999) The tail domain of lamin Dm0 binds histones H2A and H2B. Proc Natl Acad Sci U S A  96: 2852–2857.  <ul class="find" data-citedArticleID="929246" data-doi="10.1073/pnas.96.6.2852"><li><a href="http://dx.doi.org/10.1073/pnas.96.6.2852" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tail+domain+of+lamin+Dm0+binds+histones+H2A+and+H2B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tail+domain+of+lamin+Dm0+binds+histones+H2A+and+H2B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000199-Zinner1" id="pone.0000199-Zinner1"></a>Zinner R, Albiez H, Walter J, Peters AH, Cremer T, et al.  (2006) Histone lysine methylation patterns in human cell types are arranged in distinct three-dimensional nuclear zones. Histochem Cell Biol  125: 3–19.  <ul class="find" data-citedArticleID="929284" data-doi="10.1007/s00418-005-0049-1"><li><a href="http://dx.doi.org/10.1007/s00418-005-0049-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+lysine+methylation+patterns+in+human+cell+types+are+arranged+in+distinct+three-dimensional+nuclear+zones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+lysine+methylation+patterns+in+human+cell+types+are+arranged+in+distinct+three-dimensional+nuclear+zones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000199-Cook1" id="pone.0000199-Cook1"></a>Cook PR (2002) Predicting three-dimensional genome structure from transcriptional activity. Nat Genet  32: 347–352.  <ul class="find" data-citedArticleID="929240" data-doi="10.1038/ng1102-347"><li><a href="http://dx.doi.org/10.1038/ng1102-347" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+three-dimensional+genome+structure+from+transcriptional+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+three-dimensional+genome+structure+from+transcriptional+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000199-Caron1" id="pone.0000199-Caron1"></a>Caron H, van Schaik B, van der Mee M, Baas F, Riggins G, et al.  (2001) The human transcriptome map: clustering of highly expressed genes in chromosomal domains. Science  291: 1289–1292.  <ul class="find" data-citedArticleID="929238" data-doi="10.1126/science.1056794"><li><a href="http://dx.doi.org/10.1126/science.1056794" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+transcriptome+map%3A+clustering+of+highly+expressed+genes+in+chromosomal+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+transcriptome+map%3A+clustering+of+highly+expressed+genes+in+chromosomal+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000199-Versteeg1" id="pone.0000199-Versteeg1"></a>Versteeg R, van Schaik BD, van Batenburg MF, Roos M, Monajemi R, et al.  (2003) The human transcriptome map reveals extremes in gene density, intron length, GC content, and repeat pattern for domains of highly and weakly expressed genes. Genome Res  13: 1998–2004.  <ul class="find" data-citedArticleID="929280" data-doi="10.1101/gr.1649303"><li><a href="http://dx.doi.org/10.1101/gr.1649303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+transcriptome+map+reveals+extremes+in+gene+density%2C+intron+length%2C+GC+content%2C+and+repeat+pattern+for+domains+of+highly+and+weakly+expressed+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+transcriptome+map+reveals+extremes+in+gene+density%2C+intron+length%2C+GC+content%2C+and+repeat+pattern+for+domains+of+highly+and+weakly+expressed+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000199-Williams1" id="pone.0000199-Williams1"></a>Williams RR, Broad S, Sheer D, Ragoussis J (2002) Subchromosomal positioning of the epidermal differentiation complex (EDC) in keratinocyte and lymphoblast interphase nuclei. Exp Cell Res  272: 163–175.  <ul class="find" data-citedArticleID="929282" data-doi="10.1006/excr.2001.5400"><li><a href="http://dx.doi.org/10.1006/excr.2001.5400" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subchromosomal+positioning+of+the+epidermal+differentiation+complex+%28EDC%29+in+keratinocyte+and+lymphoblast+interphase+nuclei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subchromosomal+positioning+of+the+epidermal+differentiation+complex+%28EDC%29+in+keratinocyte+and+lymphoblast+interphase+nuclei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000199-Macville1" id="pone.0000199-Macville1"></a>Macville M, Veldman T, Padilla-Nash H, Wangsa D, O'Brien P, et al.  (1997) Spectral karyotyping, a 24-colour FISH technique for the identification of chromosomal rearrangements. HistochemCell Biol  108: 299–305.  <ul class="find" data-citedArticleID="929256" data-doi="10.1007/s004180050169"><li><a href="http://dx.doi.org/10.1007/s004180050169" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spectral+karyotyping%2C+a+24-colour+FISH+technique+for+the+identification+of+chromosomal+rearrangements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spectral+karyotyping%2C+a+24-colour+FISH+technique+for+the+identification+of+chromosomal+rearrangements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.XML" value="101094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.PDF" value="309534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g001.PNG_L" value="41869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g001.PNG_M" value="12434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g001.PNG_S" value="1664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g001.TIF" value="91362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g001.PNG_I" value="7222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g002.PNG_L" value="2274769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g002.PNG_M" value="243196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g002.PNG_S" value="12848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g002.TIF" value="2673214"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g002.PNG_I" value="173326"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g003.PNG_L" value="1391253"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g003.PNG_M" value="279765"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g003.PNG_S" value="7531"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g003.TIF" value="1561192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g003.PNG_I" value="94767"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g004.PNG_L" value="3842447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g004.PNG_M" value="130722"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g004.PNG_S" value="3355"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g004.TIF" value="5173726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g004.PNG_I" value="48457"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g005.PNG_L" value="848631"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g005.PNG_M" value="38376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g005.PNG_S" value="1940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g005.TIF" value="1376040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.g005.PNG_I" value="16401"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.t001.PNG_L" value="87235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.t001.PNG_M" value="95551"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.t001.PNG_S" value="5165"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.t001.TIF" value="488976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.t001.PNG_I" value="43058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.e001.PNG" value="3391"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000199.e001.TIF" value="12880"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000199&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000199" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000199&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000199&volume=&issue=&title=Artificially Introduced Aneuploid Chromosomes Assume a Conserved Position in Colon Cancer Cells&author_name=Kundan%20Sengupta%2C%20Madhvi%20B.%20Upender%2C%20Linda%20Barenboim-Stapleton%2C%20Quang%20Tri%20Nguyen%2C%20Stephen%20M.%20Wincovitch%20Sr.%2C%20Susan%20H.%20Garfield%2C%20Michael%20J.%20Difilippantonio%2C%20Thomas%20Ried&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000199" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000199" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000199" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000199&amp;t=Artificially%20Introduced%20Aneuploid%20Chromosomes%20Assume%20a%20Conserved%20Position%20in%20Colon%20Cancer%20Cells" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000199&title=Artificially%20Introduced%20Aneuploid%20Chromosomes%20Assume%20a%20Conserved%20Position%20in%20Colon%20Cancer%20Cells&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000199&amp;title=Artificially%20Introduced%20Aneuploid%20Chromosomes%20Assume%20a%20Conserved%20Position%20in%20Colon%20Cancer%20Cells" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000199" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000199&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Artificially Introduced Aneuploid Chromosomes Assume a Conserved Position in Colon Cancer Cells';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000199';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Artificially%20Introduced%20Aneuploid%20Chromosomes%20Assume%20a%20Conserved%20Position%20in%20Colon%20Cancer%20Cells http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000199" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000199" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">








          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Aneuploidy%22" title="Search for articles in the subject area:'Aneuploidy'"><div class="flagText">Aneuploidy</div></a>
              <div data-categoryid="46375" data-articleid="23236"
                   data-categoryname="Aneuploidy"
                   class="flagImage" title="Flag 'Aneuploidy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+18%22" title="Search for articles in the subject area:'Chromosome 18'"><div class="flagText">Chromosome 18</div></a>
              <div data-categoryid="27251" data-articleid="23236"
                   data-categoryname="Chromosome 18"
                   class="flagImage" title="Flag 'Chromosome 18' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+19%22" title="Search for articles in the subject area:'Chromosome 19'"><div class="flagText">Chromosome 19</div></a>
              <div data-categoryid="24875" data-articleid="23236"
                   data-categoryname="Chromosome 19"
                   class="flagImage" title="Flag 'Chromosome 19' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+7%22" title="Search for articles in the subject area:'Chromosome 7'"><div class="flagText">Chromosome 7</div></a>
              <div data-categoryid="24997" data-articleid="23236"
                   data-categoryname="Chromosome 7"
                   class="flagImage" title="Flag 'Chromosome 7' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="23236"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Distance+measurement%22" title="Search for articles in the subject area:'Distance measurement'"><div class="flagText">Distance measurement</div></a>
              <div data-categoryid="21185" data-articleid="23236"
                   data-categoryname="Distance measurement"
                   class="flagImage" title="Flag 'Distance measurement' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23236"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Homologous+chromosomes%22" title="Search for articles in the subject area:'Homologous chromosomes'"><div class="flagText">Homologous chromosomes</div></a>
              <div data-categoryid="21187" data-articleid="23236"
                   data-categoryname="Homologous chromosomes"
                   class="flagImage" title="Flag 'Homologous chromosomes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6668'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3918'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3315&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
