

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Predicted Functional RNAs within Coding Regions Constrain Evolutionary Rates of Yeast Proteins</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001559"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001559" />

    <meta name="citation_title" content="Predicted Functional RNAs within Coding Regions Constrain Evolutionary Rates of Yeast Proteins"/>
    <meta itemprop="name" content="Predicted Functional RNAs within Coding Regions Constrain Evolutionary Rates of Yeast Proteins"/>

      <meta name="citation_author" content="Charles D. Warden"/>
            <meta name="citation_author_institution" content="School of Biology, Georgia Institute of Technology, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="Seong-Ho Kim"/>
            <meta name="citation_author_institution" content="Division of Biostatistics, Indiana University School of Medicine, Indianapolis, Indiana, United States of America"/>
      <meta name="citation_author" content="Soojin V. Yi"/>
            <meta name="citation_author_institution" content="School of Biology, Georgia Institute of Technology, Atlanta, Georgia, United States of America"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001559.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1559"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=An abundance of RNA regulators.; citation_author=G Storz; citation_author=S Altuvia; citation_author=KM Wassarman; citation_journal_title=Annual Review of Biochemistry; citation_volume=74; citation_number=1; citation_pages=199-217; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MicroRNAs in gene regulation: When the smallest governs it all.; citation_author=DL Ouellet; citation_author=MP Perron; citation_author=LA Gobeil; citation_author=P Plante; citation_author=P Provost; citation_journal_title=Journal of Biomedicine and Biotechnology; citation_volume=2006; citation_number=2; citation_pages=Article ID 69616, 69620 pages; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification and classification of conserved RNA secondary structures in the human genome.; citation_author=JS Pedersen; citation_author=G Bejerano; citation_author=A Siepel; citation_author=K Rosenbloom; citation_author=K Lindblad-Toh; citation_journal_title=PLoS Computational Biology; citation_volume=2; citation_number=3; citation_pages=251-262; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of structured RNAs in S. cerevisiae indicates a multitude of different functions.; citation_author=S Steigele; citation_author=W Huber; citation_author=C Stocsits; citation_author=PF Stadler; citation_author=K Nieselt; citation_journal_title=BMC Biology; citation_volume=5; citation_number=4; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Discovery of functional elements in 12 Drosophila genomes using evolutionary signatures.; citation_author=A Stark; citation_author=MF Lin; citation_author=P Kheradpour; citation_author=JS Pedersen; citation_author=L Parts; citation_journal_title=Nature; citation_volume=450; citation_number=5; citation_pages=219-232; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Autoregulatory Frameshifting In Decoding Mammalian Ornithine Decarboxylase Antizyme.; citation_author=S Matsufuji; citation_author=T Matsufuji; citation_author=Y Miyazaki; citation_author=Y Murakami; citation_author=JF Atkins; citation_journal_title=Cell; citation_volume=80; citation_number=6; citation_pages=51-60; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Point mutation in an AMPA receptor gene rescues lethality in mice deficient in the RNA-editing enzyme ADAR2.; citation_author=M Higuchi; citation_author=M Stefan; citation_author=FN Single; citation_author=J Hartner; citation_author=A Rozov; citation_journal_title=Nature; citation_volume=406; citation_number=7; citation_pages=78-81; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Reprogrammed genetic decoding in cellular gene expression.; citation_author=O Namy; citation_author=JP Rousset; citation_author=S Napthine; citation_author=I Brierley; citation_journal_title=Molecular Cell; citation_volume=13; citation_number=8; citation_pages=157-168; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A single determinant dominates the rate of yeast protein evolution.; citation_author=DA Drummond; citation_author=A Raval; citation_author=CO Wilke; citation_journal_title=Molecular Biology and Evolution; citation_volume=23; citation_number=9; citation_pages=327-337; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Functional genomic analysis of the rates of protein evolution.; citation_author=DP Wall; citation_author=AE Hirsh; citation_author=HB Fraser; citation_author=J Kumm; citation_author=G Giaever; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=102; citation_number=10; citation_pages=5483-5488; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Understanding relationship between sequence and functional evolution in yeast proteins.; citation_author=S-H Kim; citation_author=SV Yi; citation_journal_title=Genetica; citation_volume=131; citation_number=11; citation_pages=151-156; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Fast and reliable prediction of noncoding RNAs.; citation_author=S Washietl; citation_author=IL Hofacker; citation_author=PF Stadler; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=102; citation_number=12; citation_pages=2454-2459; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.; citation_author=A Siepel; citation_author=G Bejerano; citation_author=JS Pedersen; citation_author=AS Hinrichs; citation_author=MM Hou; citation_journal_title=Genome Research; citation_volume=15; citation_number=13; citation_pages=1034-1050; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Functional analysis of gene duplications in Saccharomyces cerevisiae.; citation_author=YF Guan; citation_author=MJ Dunham; citation_author=OG Troyanskaya; citation_journal_title=Genetics; citation_volume=175; citation_number=14; citation_pages=933-943; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Rapid divergence in expression between duplicate genes inferred from microarray data.; citation_author=ZL Gu; citation_author=D Nicolae; citation_author=HHS Lu; citation_author=WH Li; citation_journal_title=Trends in Genetics; citation_volume=18; citation_number=15; citation_pages=609-613; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Why highly expressed proteins evolve slowly.; citation_author=DA Drummond; citation_author=JD Bloom; citation_author=C Adami; citation_author=CO Wilke; citation_author=FH Arnold; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=102; citation_number=16; citation_pages=14338-14343; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Correlated asymmetry of sequence and functional divergence between duplicate proteins of Saccharomyces cerevisiae.; citation_author=SH Kim; citation_author=SV Yi; citation_journal_title=Molecular Biology and Evolution; citation_volume=23; citation_number=17; citation_pages=1068-1075; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Adjusting for selection on synonymous sites in estimates of evolutionary distance.; citation_author=AE Hirsh; citation_author=HB Fraser; citation_author=DP Wall; citation_journal_title=Molecular Biology and Evolution; citation_volume=22; citation_number=18; citation_pages=174-177; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Synonymous codon usage, accuracy of translation, and gene length in Caenorhabditis elegans.; citation_author=G Marais; citation_author=L Duret; citation_journal_title=Journal of Molecular Evolution; citation_volume=52; citation_number=19; citation_pages=275-280; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=tRNA ligase is required for regulated mRNA splicing in the unfolded protein response.; citation_author=C Sidrauski; citation_author=JS Cox; citation_author=P Walter; citation_journal_title=Cell; citation_volume=87; citation_number=20; citation_pages=405-413; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Unconventional splicing of HAC1/ERN4 mRNA required for the unfolded protein response - Sequence-specific and non-sequential cleavage of the splice sites.; citation_author=T Kawahara; citation_author=H Yanagi; citation_author=T Yura; citation_author=K Mori; citation_journal_title=Journal of Biological Chemistry; citation_volume=273; citation_number=21; citation_pages=1802-1807; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A First Look at ARFome: Dual-Coding Genes in Mammalian Genomes.; citation_author=W-Y Chung; citation_author=S Wadhawan; citation_author=R Szklarczyk; citation_author=SK Pond; citation_author=A Nekrutenko; citation_journal_title=PLoS Computational Biology; citation_volume=3; citation_number=22; citation_pages=e91; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Computational identification of non-coding RNAs in Saccharomyces cerevisiae by comparative genomics.; citation_author=JP McCutcheon; citation_author=SR Eddy; citation_journal_title=Nucleic Acids Research; citation_volume=31; citation_number=23; citation_pages=4119-4128; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Annotating noncoding RNA genes.; citation_author=S Griffiths-Jones; citation_journal_title=Annual Review of Genomics and Human Genetics; citation_volume=8; citation_number=24; citation_pages=279-298; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Forces shaping the fastest evolving regions in the human genome.; citation_author=KS Pollard; citation_author=SR Salama; citation_author=B King; citation_author=AD Kern; citation_author=T Dreszer; citation_journal_title=PLoS Genetics; citation_volume=2; citation_number=25; citation_pages=1599-1611; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An RNA gene expressed during cortical development evolved rapidly in humans.; citation_author=KS Pollard; citation_author=SR Salama; citation_author=N Lambert; citation_author=MA Lambot; citation_author=S Coppens; citation_journal_title=Nature; citation_volume=443; citation_number=26; citation_pages=167-172; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Origin and evolution of human microRNAs from transposable elements.; citation_author=J Piriyapongsa; citation_author=L Marino-Ramirez; citation_author=IK Jordan; citation_journal_title=Genetics; citation_volume=176; citation_number=27; citation_pages=1323-1337; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Structured RNAs in the ENCODE selected regions of the human genome.; citation_author=S Washietl; citation_author=JS Pedersen; citation_author=JO Korbel; citation_author=C Stocsits; citation_author=AR Gruber; citation_journal_title=Genome Research; citation_volume=17; citation_number=28; citation_pages=852-864; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of genome tiling array data reveals many novel primate-specific functional RNAs in human.; citation_author=Z Zhang; citation_author=AWC Pang; citation_author=M Gerstein; citation_journal_title=BMC Evolutionary Biology; citation_volume=7; citation_number=29; citation_pages=S14; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Hairpins in a Haystack: recognizing microRNA precursors in comparative genomics data.; citation_author=J Hertel; citation_author=PF Stadler; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=30; citation_pages=E197-E202; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Use of tiling array data and RNA secondary structure predictions to identify noncoding RNA genes.; citation_author=C Weile; citation_author=PP Gardner; citation_author=MM Hedegaard; citation_author=J Vinther; citation_journal_title=Bmc Genomics; citation_volume=8; citation_number=31; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mapping of conserved RNA secondary structures predicts thousands of functional noncoding RNAs in the human genome.; citation_author=S Washietl; citation_author=IL Hofacker; citation_author=M Lukasser; citation_author=A Huttenhofer; citation_author=PF Stadler; citation_journal_title=Nature Biotechnology; citation_volume=23; citation_number=32; citation_pages=1383-1390; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Non-coding RNAs in Ciona intestinalis.; citation_author=K Missal; citation_author=D Rose; citation_author=PF Stadler; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=33; citation_pages=77-78; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Aligning multiple genomic sequences with the threaded blockset aligner.; citation_author=M Blanchette; citation_author=WJ Kent; citation_author=C Riemer; citation_author=L Elnitski; citation_author=AFA Smit; citation_journal_title=Genome Research; citation_volume=14; citation_number=34; citation_pages=708-715; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic and physical maps of Saccharomyces cerevisiae.; citation_author=JM Cherry; citation_author=C Ball; citation_author=S Weng; citation_author=G Juvik; citation_author=R Schmidt; citation_journal_title=Nature; citation_volume=387; citation_number=35; citation_pages=67-73; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Fast Folding and Comparison of RNA Secondary Structures.; citation_author=IL Hofacker; citation_author=W Fontana; citation_author=PF Stadler; citation_author=LS Bonhoeffer; citation_author=M Tacker; citation_journal_title=Monatshefte Fur Chemie; citation_volume=125; citation_number=36; citation_pages=167-188; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Widespread selection for local RNA secondary structure in coding regions of bacterial genes.; citation_author=L Katz; citation_author=CB Burge; citation_journal_title=Genome Research; citation_volume=13; citation_number=37; citation_pages=2042-2051; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene Ontology: tool for the unification of biology.; citation_author=M Ashburner; citation_author=CA Ball; citation_author=JA Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nature Genetics; citation_volume=25; citation_number=38; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_number=39; citation_date=2006; citation_publisher=R Foundation for Statistical Computing; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=ZH Yang; citation_journal_title=Computer Applications in the Biosciences; citation_volume=13; citation_number=40; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Galaxy: A platform for interactive large-scale genome analysis.; citation_author=B Giardine; citation_author=C Riemer; citation_author=RC Hardison; citation_author=R Burhans; citation_author=L Elnitski; citation_journal_title=Genome Research; citation_volume=15; citation_number=41; citation_pages=1451-1455; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The UCSC Genome Browser Database.; citation_author=D Karolchik; citation_author=R Baertsch; citation_author=M Diekhans; citation_author=TS Furey; citation_author=A Hinrichs; citation_journal_title=Nucleic Acids Research; citation_volume=31; citation_number=42; citation_pages=51-54; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Dissecting the regulatory circuitry of a eukaryotic genome.; citation_author=FCP Holstege; citation_author=EG Jennings; citation_author=JJ Wyrick; citation_author=TI Lee; citation_author=CJ Hengartner; citation_journal_title=Cell; citation_volume=95; citation_number=43; citation_pages=717-728; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Global analysis of protein expression in yeast.; citation_author=S Ghaemmaghami; citation_author=W Huh; citation_author=K Bower; citation_author=RW Howson; citation_author=A Belle; citation_journal_title=Nature; citation_volume=425; citation_number=44; citation_pages=737-741; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast.; citation_author=AM Deutschbauer; citation_author=DF Jaramillo; citation_author=M Proctor; citation_author=J Kumm; citation_author=ME Hillenmeyer; citation_journal_title=Genetics; citation_volume=169; citation_number=45; citation_pages=1915-1925; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evidence for dynamically organized modularity in the yeast protein-protein interaction network.; citation_author=JDJ Han; citation_author=N Bertin; citation_author=T Hao; citation_author=DS Goldberg; citation_author=GF Berriz; citation_journal_title=Nature; citation_volume=430; citation_number=46; citation_pages=88-93; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Proportion of solvent-exposed amino acids in a protein and rate of protein evolution.; citation_author=YS Lin; citation_author=WL Hsu; citation_author=JK Hwang; citation_author=WH Li; citation_journal_title=Molecular Biology and Evolution; citation_volume=24; citation_number=47; citation_pages=1005-1011; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The pls package: Principal component and partial least squares regression in R.; citation_author=BH Mevik; citation_author=R Wehrens; citation_journal_title=Journal of Statistical Software; citation_volume=18; citation_number=48; citation_date=2007; " />
      <meta name="citation_reference" content="citation_author=J Whittaker; citation_number=49; citation_pages=466; citation_date=1996; citation_publisher=John Wiley and Sons; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001559" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Predicted Functional RNAs within Coding Regions Constrain Evolutionary Rates of Yeast Proteins"/>
    <meta name="twitter:description" content="Functional RNAs (fRNAs) are being recognized as an important regulatory component in biological processes. Interestingly, recent computational studies suggest that the number and biological significance of functional RNAs within coding regions (coding fRNAs) may have been underestimated. We hypothesized that such coding fRNAs will impose additional constraint on sequence evolution because the DNA primary sequence has to simultaneously code for functional RNA secondary structures on the messenger RNA in addition to the amino acid codons for the protein sequence. To test this prediction, we first utilized computational methods to predict conserved fRNA secondary structures within multiple species alignments of Saccharomyces sensu strico genomes. We predict that as much as 5% of the genes in the yeast genome contain at least one functional RNA secondary structure within their protein-coding region. We then analyzed the impact of coding fRNAs on the evolutionary rate of protein-coding genes because a decrease in evolutionary rate implies constraint due to biological functionality. We found that our predicted coding fRNAs have a significant influence on evolutionary rates (especially at synonymous sites), independent of other functional measures. Thus, coding fRNA may play a role on sequence evolution. Given that coding regions of humans and flies contain many more predicted coding fRNAs than yeast, the impact of coding fRNAs on sequence evolution may be substantial in genomes of higher eukaryotes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001559.g002"/>

  <meta property="og:title" content="Predicted Functional RNAs within Coding Regions Constrain Evolutionary Rates of Yeast Proteins" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001559" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9598'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7468'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=925&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001559">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001559" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001559&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Predicted Functional RNAs within Coding Regions Constrain Evolutionary Rates of Yeast Proteins
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Charles D. Warden, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biology, Georgia Institute of Technology, Atlanta, Georgia, United States of America
                </p>

                <p>
                    Current address: Department of Molecular Biology, Princeton University, Princeton, New Jersey, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Seong-Ho Kim, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Biostatistics, Indiana University School of Medicine, Indianapolis, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Soojin V. Yi
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:soojinyi@gatech.edu">soojinyi@gatech.edu</a></p>

                <p>Affiliation:
                  School of Biology, Georgia Institute of Technology, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001559</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001559" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001559" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001559" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001559" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001559-g001" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001559-g002" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001559-t001" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001559-t002" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001559-t003" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001559-t004" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001559">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001559" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Functional RNAs (fRNAs) are being recognized as an important regulatory component in biological processes. Interestingly, recent computational studies suggest that the number and biological significance of functional RNAs <em>within coding regions</em> (coding fRNAs) may have been underestimated. We hypothesized that such coding fRNAs will impose additional constraint on sequence evolution because the DNA primary sequence has to <em>simultaneously</em> code for functional RNA secondary structures on the messenger RNA in addition to the amino acid codons for the protein sequence. To test this prediction, we first utilized computational methods to predict conserved fRNA secondary structures within multiple species alignments of S<em>accharomyces sensu strico</em> genomes. We predict that as much as 5% of the genes in the yeast genome contain at least one functional RNA secondary structure within their protein-coding region. We then analyzed the impact of coding fRNAs on the evolutionary rate of protein-coding genes because a decrease in evolutionary rate implies constraint due to biological functionality. We found that our predicted coding fRNAs have a significant influence on evolutionary rates (especially at synonymous sites), independent of other functional measures. Thus, coding fRNA may play a role on sequence evolution. Given that coding regions of humans and flies contain many more predicted coding fRNAs than yeast, the impact of coding fRNAs on sequence evolution may be substantial in genomes of higher eukaryotes.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Warden CD, Kim S-H, Yi SV (2008) Predicted Functional RNAs within Coding Regions Constrain Evolutionary Rates of Yeast Proteins. PLoS ONE 3(2):
          e1559.
            doi:10.1371/journal.pone.0001559</p><p><strong>Academic Editor: </strong>Matthew Hahn, Indiana University, United States of America</p><p><strong>Received:</strong> August 1, 2007; <strong>Accepted:</strong> December 30, 2007; <strong>Published:</strong> February 13, 2008</p><p><strong>Copyright:</strong> © 2008 Warden et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no support or funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>There are numerous exceptions to the “standard” flow of genetic information depicted in the central dogma of molecular biology. For example, some genes code for “non-coding” RNA (ncRNA) that are never translated. Such non-coding RNAs play important roles in vital biological processes, especially in the regulation of gene expression <a href="#pone.0001559-Storz1">[1]</a>, <a href="#pone.0001559-Ouellet1">[2]</a>. Here, we use the term functional RNA (fRNA) to refer to both ncRNA and conserved fRNA secondary structures within coding regions.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Interestingly, recent computational studies have predicted a large number of functional RNA secondary structures <em>within</em> protein-coding regions (referred to as ‘coding fRNA’) in vertebrates <a href="#pone.0001559-Pedersen1">[3]</a>, yeast <a href="#pone.0001559-Steigele1">[4]</a>, and flies <a href="#pone.0001559-Stark1">[5]</a>. There are already some known examples of coding fRNAs playing significant biological roles in higher eukaryotes, such as in programmed frameshifting <a href="#pone.0001559-Matsufuji1">[6]</a>, A-to-I RNA editing <a href="#pone.0001559-Higuchi1">[7]</a>, and selenocysteine insertion at stop codon sites <a href="#pone.0001559-Namy1">[8]</a>. However, the predicted <em>abundance</em> of coding fRNAs is surprising. If the majority of these computational predictions are accurate, the presence of coding fRNAs will play an important role in molecular evolution of protein sequences.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Specifically, we hypothesize that the presence of coding fRNAs will impose additional evolutionary constraint on coding sequences because coding fRNAs require the DNA primary sequence to <em>simultaneously</em> code for conserved secondary structures in addition to the amino acid sequence. As far as we are aware, the influence of coding fRNAs on evolutionary rates has not yet been explored.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In this study, we investigate the distribution and evolutionary impact of predicted coding fRNAs in <em>Saccharomyces cerevisiae</em>. Although there are relatively few known fRNAs in the yeast genome, studying yeast has several advantages. First, there are many sequenced yeast genomes available, enabling comparative studies. The divergence among the <em>Saccharomyces sensu stricto</em> genomes is comparable to that among the eight vertebrate genomes used to search for fRNAs in an earlier study <a href="#pone.0001559-Pedersen1">[3]</a>. Second, advances in yeast functional genomics have provided a wealth of other functional genomics data to be used in evolutionary analysis <a href="#pone.0001559-Drummond1">[9]</a>–<a href="#pone.0001559-Kim1">[11]</a>. Third, yeasts are more conducive to experimental manipulation, so the biological function of a predicted fRNA can be learned more easily, compared to vertebrates. For example, an fRNA may play a tissue- and developmental stage- specific role in humans and cannot be easily validated by experimental methods. Thus, our search for coding fRNAs in yeast complements similar computational searches in higher eukaryotes and provides a valuable set of coding fRNA candidates for future experimental studies.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>We found that as much as 5% of the genes in the yeast genome may contain at least one coding fRNA. These predicted coding fRNAs tend to constrain evolutionary rates in protein coding regions. In particular, evolutionary rates at synonymous sites were strongly affected by the proportion of predicted coding fRNAs within a gene. This relationship was independent of other functional variables known to affect protein evolutionary rates in yeast. Thus, the yeast genome may contain a considerable number of <em>coding</em> functional RNAs that decrease protein evolutionary rates.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Yeast genome harbors a substantial number of coding fRNAs</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We used several filtering steps while combining two prediction methods to assess the distribution of functional RNA secondary structures within genomes of the <em>Saccharomyces sensu stricto</em> clade. The first method, implemented in the EvoFold program, uses a phylogenetic stochastic context free grammar (phylo-SCFG) model that identifies fRNA based upon substitutions that maintain a conserved secondary structure among nucleotide sequences in a multiple species alignment <a href="#pone.0001559-Pedersen1">[3]</a>. The second prediction method, implemented in the RNAz program, utilizes information on both conserved secondary structure and thermodynamic stability to identify RNA secondary structures in multiple sequence alignments <a href="#pone.0001559-Washietl1">[12]</a>. We believe our methodology (outlined below) has produced a stringently defined set of potential fRNAs that should be useful in determining targets of future investigation. For further details, please refer to the <a href="#s4">Methods</a> section and Supplementary <a href="#pone.0001559.s001">Text S1</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We first determined ‘optimal’ sets of comparative alignments by maximizing the number of known ncRNAs, serving as positive controls, recovered by different prediction conditions (<a href="#pone.0001559.s001">Text S1</a>). RNAz and EvoFold exhibit different sensitivity in this positive control test (<a href="#pone.0001559.s005">Table S1</a>), reflecting the fundamental differences in their algorithms. From this analysis, we determined that data set with the best predictive power was the set of EvoFold predictions produced by the 5-species alignment (with an FPS value greater than 0, see below) that were independently verified by the RNAz predictions made using the 6-species alignment (with P-value of 0.9, see below). The number of folds predicted by different methods is shown in <a href="#pone.0001559.s002">Figure S1</a>.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The significance of a predicted fRNA from the EvoFold program was determined by a folding potential score (FPS). FPS is a length normalized likelihood-ratio score and is defined as follows: FPS = log (P(x|φ<sub>fRNA</sub>)/P(x|φ<sub>bg</sub>))/<em>l</em>, where P(x|φ<sub>fRNA</sub>) refers to the probability that a sequence fits an fRNA structural model, P(x|φ<sub>bg</sub>) refers to the probability that the sequence fits the background model (i.e. no-fRNA structure model), and <em>l</em> refers to the length of the fold (defined by the outermost basepair of a fRNA structure) <a href="#pone.0001559-Pedersen1">[3]</a>. We required all folds in the final dataset to have an FPS greater than 0. Requiring a higher cutoff value for the FPS score does not substantially improve the accuracy of our dataset, since it did not increase the recovery of positive controls (results not shown). The error rate of the phylo-SCFG method in EvoFold is predicted to be substantial (around 60%), even though it is difficult to determine the precise false positive rate for these predictions <a href="#pone.0001559-Pedersen1">[3]</a>.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Next, we chose a subset of fRNAs that were independently verified by EvoFold and RNAz predictions. The RNAz program uses a machine learning technique to produce p-values based on estimated false positive rates <a href="#pone.0001559-Washietl1">[12]</a>. For the set of RNAz predictions, we chose a cutoff P-value of 0.9, which corresponds roughly to a 1% false positive rate according to the RNAz authors <a href="#pone.0001559-Washietl1">[12]</a>. In comparison, a cutoff P-value of 0.5 corresponds to 4% false positive rates.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>Furthermore, we removed EvoFold predictions that were shorter than 10 nucleotides because the vast majority of predictions that were less than 10 nucleotides were not likely to form a stable RNA secondary structure.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>Following these four steps, we identified 919 predicted fRNAs. When compared to the maximum number of folds that could be predicted for either of these methods (using the 5-species alignment for EvoFold and the 4-species alignment for RNAz), our pipeline for reducing false positives resulted in a 55.5% reduction of EvoFold predictions and an 85.4% reduction in RNAz predictions (<a href="#pone.0001559.s002">Figure S1</a>).</p>
<a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>The genomic distribution of these folds is shown in <a href="#pone-0001559-g001">Figure 1</a>. The majority of fRNAs were predicted in intergenic regions. Nevertheless, a considerable proportion (33%) of the total fRNAs was found within protein coding regions. Overall, 272 genes were found to contain at least one coding fRNA. Given that there are approximately 6000 genes in the yeast genome, our results predict that as much as 5% of the yeast proteome may encode at least one coding fRNA.</p>
<div class="figure" id="pone-0001559-g001"><div class="img"><a name="pone-0001559-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001559.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001559.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Substantial Proportion of Predicted fRNAs within Coding Regions.</span></strong></p><span>doi:10.1371/journal.pone.0001559.g001</span></div><a id="article1.body1.sec2.sec1.p8" name="article1.body1.sec2.sec1.p8"></a><p>For our functional analyses, we further restricted our data to only use well-curated genes across different yeast genomes (see <a href="#s4">Methods</a>). For example, we removed genes with introns, because exon/intron boundaries may not be conserved in different yeast genomes. This procedure left a set of 169 genes. We performed two additional analyses to detect potential false positives. First, we only chose coding fRNAs with negative free energy, which is the minimum thermodynamic requirement to expect coding fRNAs could fold <em>in vivo</em>, resulting in 143 coding fRNAs considered for functional analysis (see <a href="#s4">Methods</a>). Note that results obtained using data without undergoing the last step were qualitatively similar to those obtained using the most restricted data. Second, we used a codon shuffling approach, which also led to similar results (see <a href="#s4">Methods</a> and <a href="#pone.0001559.s001">Text S1</a>).</p>
<a id="article1.body1.sec2.sec1.p9" name="article1.body1.sec2.sec1.p9"></a><p>The average length for a coding fRNA considered for functional analysis (see <a href="#s4">Methods</a>) is 22.5±10.3 bps. Coding fRNAs tend to be evenly distributed within coding regions (the average relative position for a coding fRNA is 0.51±0.29 of the length of the coding region).</p>


<h4>Under-representation of coding fRNAs in yeast compared to vertebrates and <em>Drosophila</em></h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We found that EvoFold had a greater propensity to predict coding fRNAs in vertebrates than in yeast. A previous study for conserved fRNAs in the human genome using the EvoFold program found that 23% of the predicted fRNAs were found within coding regions <a href="#pone.0001559-Pedersen1">[3]</a>. For comparison, only 18% of the comparative data set used in this study was coding regions (as measured by the proportion of phastCons elements found within coding regions) <a href="#pone.0001559-Siepel1">[13]</a>. In contrast, we found only 33% of fRNAs in coding regions of yeast, which contain 86.1% of the phastCons elements <a href="#pone.0001559-Siepel1">[13]</a>. Another way to understand this comparison is to note that 303 coding fRNAs were found in 65,348 phastCons CDS blocks in yeast while 12736 coding fRNAs were found in 23,580 phastCons CDS blocks in vertebrates <a href="#pone.0001559-Pedersen1">[3]</a>, <a href="#pone.0001559-Siepel1">[13]</a>. Thus, in terms of the proportion of coding fRNAs to phastCons CDS blocks, coding fRNAs are about 10 times more likely to be found in vertebrates than yeast.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>It should be noted that Pedersen et al. <a href="#pone.0001559-Pedersen1">[3]</a> use a slightly different method of defining the fold location for a given fRNA secondary structure, and the phastCons elements was defined across a slightly more diverged multi-species alignment for yeast (longest unconserved divergence = 1.290, 7 species compared) than for vertebrates (longest unconserved divergence = 1.198, 5 species compared) <a href="#pone.0001559-Siepel1">[13]</a>. Still, the large difference in the abundance of coding fRNAs in vertebrates and yeast warrants future investigations of the role of coding fRNA in higher eukaryotes.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>A recent study also revealed that a significant number of coding fRNAs reside in <em>Drosophila</em> genomes, using the EvoFold program <a href="#pone.0001559-Stark1">[5]</a>. The distribution of phastCons elements in Drosophila is roughly similar to the phastCons distribution for vertebrates <a href="#pone.0001559-Siepel1">[13]</a>. Thus, it appears that coding fRNAs are less abundant in yeast compared to humans and <em>Drosophila</em>. It would be informative to conduct functional genomic analysis similar to that described in this paper on recently discovered coding fRNA predictions in higher eukaryotes to determine the role of fRNAs on coding sequence evolution.</p>


<h4>Genes with coding fRNAs enriched with specific ontology annotations</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We analyzed the distribution of GO annotations for genes containing at least one coding fRNA, to test for possible sources of bias in the dataset and to determine whether genes containing coding fRNAs otherwise tend to be enriched with any particular biological functions. We compared the distribution of GO annotations in our dataset with that in the whole yeast genome and tested for significant deviations (<a href="#s4">Methods</a>). We found that genes containing coding fRNAs tend to be enriched with the following GO categories: various metabolic processes (amino acid (GO ID: 6519), carbohydrate (GO ID: 5975), and vitamin (GO ID: 6766)), transcription (GO ID: 6350), translation (GO ID: 6412), and transport (GO ID: 6810) (<a href="#pone-0001559-g002">Figure 2</a>). Enrichment with ribosomal genes can be problematic because of some of the unique characteristics associated with these generally well-conserved proteins <a href="#pone.0001559-Guan1">[14]</a>–<a href="#pone.0001559-Kim2">[17]</a>.</p>
<div class="figure" id="pone-0001559-g002"><div class="img"><a name="pone-0001559-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001559.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001559.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Distribution of GO Annotations in Strictly Defined Dataset.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>This figure shows the distribution of GO annotations for the set of genes containing at least one predicted fRNA fold (‘strict’) compared to the background set of genes in the yeast genome as annotated in the SGD database (‘SGD’). Some GO annotations have been abbreviated for the interest of space limitation. p-values: * = 0.05, ** = 0.01, *** = 0.001.</p>
<span>doi:10.1371/journal.pone.0001559.g002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>More specifically, the genes associated with translation (GO ID: 6412) have significantly greater values of fRNA coverage and significantly smaller values of evolutionary divergence than the set of all genes in the strictly defined dataset (Wilcoxon rank sum test, p-value&lt;0.05, data not shown). Due to this concern, ribosomal genes (GO ID: 6412 translation) were removed for the statistical analysis described in the following section.</p>


<h4>Predicted coding fRNAs significantly constrain evolutionary rates</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Here, we investigate whether the predicted coding fRNAs from yeast genomes are likely to be biologically relevant. If coding fRNAs serve a biological function, then the presence of coding fRNAs should constrain evolutionary rates because of the added constraint to conserve a fRNA sequence on the messenger RNA. Thus, evolutionary rates of genes harboring fRNA should be reduced more than expected based upon other known functional factors. We specifically test this prediction.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Although we employed several filtering steps to identify likely fRNAs, we took additional cautions to remove false positives before evolutionary analysis (see <a href="#s4">Methods</a>). We note that all the results presented here were qualitatively similar when we performed analyses without this last step or when we used a different approach to eliminate potential false positives (<a href="#pone.0001559.s001">Text S1</a>).</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>For our functional genomic analysis, we define a new variable, percent ‘fRNA coverage,’ which is the length of the coding region for a gene overlapping a predicted fRNA divided by the length of that gene. First, we assessed how fRNA coverage impacts evolutionary rates. After removing false positives for folds that were not thermodynamically stable, we found that fRNA coverage is negatively correlated with divergence at both nonsynonymous (Pearson's <em>ρ</em> = −0.235, <em>P</em>&lt;0.05, sample size = 81) and synonymous sites (<em>ρ</em> = −0.497, <em>P</em> = 0.005). All variables are log transformed to approximate normality. Non-parametric correlation tests provided similar results (data not shown). In order to correct for the impact of codon usage bias on evolutionary rates, we also calculated an adjusted value for synonymous site divergence, dS′ <a href="#pone.0001559-Hirsh1">[18]</a>. We found that there are significant negative correlations with dS′. Thus, coding regions with a greater proportion of sequence overlapping coding fRNAs evolve more slowly.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>However, we need to ensure that the association between predicted fRNA coverage and evolutionary rate is not due to the influence of a third secondary factor (or set of secondary factors) because many functional variables are known to affect protein evolutionary rates of yeast. Several recent studies emphasize the importance of proper statistical methods to assess independent effects of specific variables of interest <a href="#pone.0001559-Drummond1">[9]</a>–<a href="#pone.0001559-Kim1">[11]</a>, <a href="#pone.0001559-Kim2">[17]</a>. For example, Wall et al. <a href="#pone.0001559-Wall1">[10]</a> used partial correlation analysis to show that gene expression and gene dispensability have significant, independent impacts on evolutionary rates. Drummond et al. <a href="#pone.0001559-Drummond1">[9]</a> used principal component regression analysis to conclude that indicators related to gene expression (gene expression, CAI, and protein abundance) are the dominant determinants of evolutionary rates in yeast. We analyzed the impact of fRNA on evolutionary sequence divergence, while controlling for other factors, using both the partial correlation and the principal component regression methods.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>We did not consider gene length as a variable in our multivariate analysis because gene length and fRNA coverage are not independent variables (due to the way the variable fRNA coverage is defined: see above). Because gene length and evolutionary rates are positively correlated <a href="#pone.0001559-Kim1">[11]</a>, <a href="#pone.0001559-Marais1">[19]</a> and fRNA coverage and gene length are negatively correlated, it is important to show that the impact of fRNA coverage on evolutionary rate is still significant when controlling for the influence of gene length. To address this concern, we conducted partial correlation between fRNA coverage and evolutionary rates while controlling for gene length. Significant correlation between fRNA coverage and evolutionary rates remained after this step (<a href="#pone.0001559.s006">Table S2</a>). In addition, we compared the amount of variance in our data that can be explained by fRNA coverage versus to that by 1/(gene length). We observe that fRNA coverage can explain greater amount of variance than 1/(gene length), for all measures of evolutionary rates (<a href="#pone.0001559.s006">Table S2</a>). Thus, fRNA coverage appears as a measure that is more robust than either coding fRNA length or gene length alone.</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>We considered the following seven functional variables that are known to be important determinants of yeast evolutionary rates: gene expression, CAI, gene dispensability, degree, centrality, mRNA half-life, and fRNA coverage <a href="#pone.0001559-Drummond1">[9]</a>, <a href="#pone.0001559-Kim1">[11]</a>. Among our final data set, there are only 25 genes with all seven variables defined, and only 22 genes after removing genes with negative minimum free energy (<em>mfe</em>).</p>
<a id="article1.body1.sec2.sec4.p7" name="article1.body1.sec2.sec4.p7"></a><p>Because analyses of such a small data set may be strongly influenced by stochastic effects, we removed degree, centrality and mRNA half-life from our analyses presented in the main text. It has been shown in earlier studies that network variables such as degree and centrality tend to have minor effects on yeast evolutionary rates <a href="#pone.0001559-Drummond1">[9]</a>, <a href="#pone.0001559-Kim1">[11]</a>, <a href="#pone.0001559-Kim2">[17]</a>, and mRNA half-life is not often considered as an important determinant of yeast evolutionary rates. This step allowed us to have a moderate sample size (73 genes).</p>
<a id="article1.body1.sec2.sec4.p8" name="article1.body1.sec2.sec4.p8"></a><p><a href="#pone-0001559-t001">Table 1</a> presents Pearson's correlations and partial correlations for genes with negative <em>mfe</em>. There is a negative correlation between fRNA coverage and all five measures of evolutionary rates. When all other functional measures are considered for partial correlation analysis, fRNA coverage appears to affect dS′ most significantly. We found similar results with additional functional variables (sample size = 22, <a href="#pone.0001559.s007">Table S3</a>), across a shorter evolutionary timescale (sample size = 128, <a href="#pone.0001559.s008">Table S4</a>). Thus, partial correlation analysis reveals that fRNA coverage imposes a significant constraint on sequence evolution, especially on synonymous sites after correcting for the effect of codon usage bias.</p>
<div class="figure" id="pone-0001559-t001"><div class="img"><a name="pone-0001559-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001559.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001559.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Correlation and partial correlations show coding fRNAs decrease evolutionary rates (genes with negative <em>mfe</em>).</span></strong></p><span>doi:10.1371/journal.pone.0001559.t001</span></div><a id="article1.body1.sec2.sec4.p9" name="article1.body1.sec2.sec4.p9"></a><p>The relative impact of fRNA coverage on evolutionary rate observed from partial correlation analysis is also corroborated by results from principal component regression analysis (<a href="#pone-0001559-t002">Table 2</a>). The results in <a href="#pone-0001559-t002">table 2</a> show that principal components 1 and 4 are related to gene expression while principal components 2 and 3 are influenced by fRNA coverage. As seen previously <a href="#pone.0001559-Drummond1">[9]</a>, <a href="#pone.0001559-Kim1">[11]</a>, components associated with gene expression explain a significant percent of the variance in the dataset. Interestingly, the principal component 2, which has a large contribution of fRNA coverage, has a strong influence on dS and dS′.</p>
<div class="figure" id="pone-0001559-t002"><div class="img"><a name="pone-0001559-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001559.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001559.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Principal component regression reveals coding fRNAs have significant influence on evolutionary divergence (genes with negative <em>mfe</em>).</span></strong></p><span>doi:10.1371/journal.pone.0001559.t002</span></div><a id="article1.body1.sec2.sec4.p10" name="article1.body1.sec2.sec4.p10"></a><p>We also present results obtained after identifying and removing potential false positives using a codon shuffling method (<a href="#s4">Methods</a>). The sample size is 55 genes. Partial correlation (<a href="#pone-0001559-t003">Table 3</a>) and principal component regression analysis (<a href="#pone-0001559-t004">Table 4</a>) show that the fRNA coverage is generally negatively correlated with evolutionary rates, and the effect is the most pronounced for dS′. In particular, in <a href="#pone-0001559-t004">Table 4</a> we can see that the effect of fRNA coverage and gene dispensability are separated into components 2 and 3 respectively, and that the component 2 (which mostly represents the effect of fRNA coverage) has a clear effect on dS′. Results are also similar when principal component regression analysis is applied to evolutionary rates when considering additional functional variables (<a href="#pone.0001559.s009">Table S5</a>) and across a shorter timescale (<a href="#pone.0001559.s010">Table S6</a>). Thus, fRNA coverage has a significant, independent impact on evolutionary rates, especially at synonymous sites.</p>
<div class="figure" id="pone-0001559-t003"><div class="img"><a name="pone-0001559-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001559.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001559.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Correlations and Partial Correlations using Pearson Correlations on Genes with EFP&gt;0.</span></strong></p><span>doi:10.1371/journal.pone.0001559.t003</span></div><div class="figure" id="pone-0001559-t004"><div class="img"><a name="pone-0001559-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001559" data-uri="info:doi/10.1371/journal.pone.0001559.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001559.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001559.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001559.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001559.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001559.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Results of Principal Component Regression Analyses for Genes with EFP&gt;0.</span></strong></p><span>doi:10.1371/journal.pone.0001559.t004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study, we demonstrated that there are a substantial number of predicted coding fRNAs in the yeast genome (as much as ~5% of the protein-coding genes) and that these predicted fRNAs seem to play a biologically significant role (based upon statistical analysis of evolutionary rates). More specifically, genes containing a larger proportion of fRNAs evolve significantly more slowly at synonymous sites, independent of codon usage bias and effects of other functional variables (see <a href="#pone-0001559-t001">Tables 1</a>–<a href="#pone-0001559-t002"></a><a href="#pone-0001559-t003"></a><a href="#pone-0001559-t004">4</a>).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Coding fRNAs may have a stronger effect on evolutionary rates at synonymous sites than at nonsynonymous sites, because there are many more sources of functional constraint for nonsynonymous sites, thus requiring a survey with greater statistical power to understand the more subtle influences of coding fRNA on nonsynonymous rates. It is also interesting that coding fRNAs have a relatively greater influence on synonymous site evolution because synonymous sites are traditionally considered to evolve at a neutral rate, and we show that predicted coding fRNAs may be a significant source of non-neutral evolution at synonymous sites. Below we discuss the limits of computational predictions, and factors that could have influenced our statistical analyses and the conclusions on evolutionary impacts of coding fRNAs.</p>

<h4>Determining false positive rates for predicted fRNAs</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Although it is difficult to gauge the statistical power of this search for coding fRNAs because there are not many known coding fRNAs in yeast, there is promising evidence that our final set of predicted fRNAs has successfully recovered biologically relevant fRNA secondary structures. For example, HAC1 is a well-studied gene in yeast that undergoes <em>non-spliceosomal splicing</em> for dual-coding regions, and the mRNA for this gene is known to require conserved mRNA secondary structures in order to undergo alternative splicing <a href="#pone.0001559-Sidrauski1">[20]</a>, <a href="#pone.0001559-Kawahara1">[21]</a>. A stringently defined coding fRNA was recovered within HAC1 (although it should be noted that this gene was excluded from statistical functional analysis because it contains an intron). It is interesting to note that many dual-coding genes in higher eukaryotes (<em>GNAS1, XBP1, INK4a</em>, and <em>ADCY8</em>) discussed in a recent study <a href="#pone.0001559-Chung1">[22]</a> also contain at least one EvoFold prediction of a coding fRNA <a href="#pone.0001559-Pedersen1">[3]</a>. Therefore, the results of this and similar studies may help explain the splicing mechanisms for dual-coding regions and other exciting biological functions associated with coding fRNAs.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Some previous studies have discovered novel ncRNAs in yeast. For example, one earlier study used the QRNA program to produce a list of ~100 ncRNA candidate genes <a href="#pone.0001559-McCutcheon1">[23]</a>. More recently, a study identified a number of novel candidate coding and non-coding fRNAs in yeast <a href="#pone.0001559-Steigele1">[4]</a>. To our surprise, none of our stringently defined coding fRNAs overlap with predictions from the most stringently defined set of coding fRNAs in Steigele et al. <a href="#pone.0001559-Steigele1">[4]</a>.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>This observation is a poignant reminder that the current fRNA prediction programs and false-positive tests suffer from a large and essentially unknown error rates, and different computational methods likely to respond to different signals and/or categories of fRNAs. Indeed, it has been noted that in vertebrates, the predictions by the RNAz and EvolFold have less than 10% overlap <a href="#pone.0001559-GriffithsJones1">[24]</a>.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>Another source of discrepancy between our results and those in Steigele et al. <a href="#pone.0001559-Steigele1">[4]</a> is that in the latter the authors used an RNAz scoring measure that placed greater emphasis on conserved covariance between sites, whereas average thermodynamic stability between species was the dominant factor determining which RNAz predictions were defined in our dataset. Nevertheless, given that we used commonly used algorithms (EvoFold <a href="#pone.0001559-Pedersen1">[3]</a>, <a href="#pone.0001559-Stark1">[5]</a>, <a href="#pone.0001559-Pollard1">[25]</a>–<a href="#pone.0001559-Zhang1">[29]</a>, and RNAz <a href="#pone.0001559-Steigele1">[4]</a>, <a href="#pone.0001559-Washietl1">[12]</a>, <a href="#pone.0001559-Washietl2">[28]</a>–<a href="#pone.0001559-Missal1">[33]</a>) and that we used several filtering steps, including two different methods to exclude potential false positives (<a href="#s4">Methods</a>), we consider our results to have strong computational support.</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>Ultimately, the only way to determine true false positive rates is experimental validation. Thus, our results should provide a valuable complement to this earlier study and provide experimental scientists with a new list of candidate coding fRNAs. Our results should be also helpful to better evaluate computational methods to predict fRNAs.</p>


<h4>Prediction methods are not biased by evolutionary constraint</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Although there is clearly a significant negative correlation between fRNA coverage and evolutionary rates, it is necessary to show that the correlation between percent fRNA coverage and evolutionary rates is not due to a bias in prediction methods. The RNAz program is not known to have any specific bias towards predicting false positives <a href="#pone.0001559-Washietl1">[12]</a>. In fact, even though the RNAz program was designed to search for <em>non-coding RNAs</em>, it predicted a <em>larger</em> proportion of coding fRNAs than EvoFold (<a href="#pone.0001559.s001">Text S1</a>). In comparison, EvoFold requires moderately well conserved multiple species alignment to successfully predict fRNA secondary structures. In particular, EvoFold's measure of significance for folds, FPS (see <a href="#s2">Results</a> section), has a bias towards ranking highly-conserved, short fRNAs with a high FPS <a href="#pone.0001559-Pedersen1">[3]</a>. Indeed, we observed a negative correlation between FPS and synonymous and nonsynonymous rates in our original predicted fRNAs. However, these correlations were mainly caused by ribosomal genes. When we removed ribosomal genes from our data set, FPS was no longer significantly correlated with evolutionary rates. Furthermore, there is no significant difference in the average value for evolutionary rates in genes with short versus long fRNAs (see <a href="#pone.0001559.s011">Table S7</a>). Thus, it is unlikely that our analysis is biased due to spurious predictions of multiple and/or short coding fRNAs within conserved genes.</p>


<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Our results indicate that the presence of coding fRNAs constrain evolutionary rates of yeast proteins. The list of coding fRNAs presented in this study should warrant future experimental validation. Since coding fRNAs are likely to be more prevalent in genomes of higher eukaryotes including human and <em>Drosophila</em>, the impact of coding fRNA on sequence evolution in those species is likely to be substantial. Overall, this study suggests that the evolutionary impact of coding fRNAs may have been underestimated.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Functional RNA predictions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We use the EvoFold and RNAz algorithms to screen the Multiz alignment for <em>Saccharomyces sensu stricto</em> species for functional RNA secondary structures <a href="#pone.0001559-Pedersen1">[3]</a>, <a href="#pone.0001559-Washietl1">[12]</a>, <a href="#pone.0001559-Blanchette1">[34]</a>. EvoFold is a program that uses comparative genomic analysis to identify conserved fRNAs based upon compensatory substitutions required to maintain a particular RNA secondary structure <a href="#pone.0001559-Pedersen1">[3]</a>. In contrast, the RNAz program uses comparative genomic analysis to compare independently predicted RNA secondary structures for a multiple species alignment based upon thermodynamic predictions form each species' primary sequence <a href="#pone.0001559-Washietl1">[12]</a>. We required our fRNAs to be independently verified by both of these very different methods (in addition to other strict requirements – see “Calculation of Nonsynonymous and Synonymous Divergence” section).</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Screening for functional RNAs was conducted using EvoFold and RNAz programs to provide independent predictions of fRNAs <a href="#pone.0001559-Pedersen1">[3]</a>, <a href="#pone.0001559-Washietl1">[12]</a>. These two programs should predict fRNAs independently because EvoFold utilizes a functional RNA model based on stochastic context-free grammars while RNAz primarily utilizes thermodynamic information to predict RNA secondary structures (while also considering covariance between secondary structures in a multi-species alignment). For more information about the multi-species alignments used for these fRNA prediction programs, see <a href="#pone.0001559.s001">Text S1</a>. The optimal multi-species alignment for each program was determined by iteratively comparing the proportion of recovered known ncRNA annotations from the SGD database <a href="#pone.0001559-Cherry1">[35]</a> to the proportion of recovered known ncRNAs at a more liberal threshold (see <a href="#pone.0001559.s001">Text S1</a>, <a href="#pone.0001559.s005">Tables S1</a>,<a href="#pone.0001559.s006">S2</a>,<a href="#pone.0001559.s007">S3</a>, <a href="#pone.0001559.s002">Figures S1</a>,<a href="#pone.0001559.s003">S2</a>,<a href="#pone.0001559.s004">S3</a>).</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>The location of each fRNA was determined by the position of the middle of each fRNA secondary structure (i.e. a fRNA was in a particular category if &gt;50% of the fold was in that type of region). All folds were categorized as coding, intronic, or intergenic.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>Finally, we performed two tests to estimate potential false positive rates. First, we used the RNAfold program to calculate the minimum free energy (<em>mfe</em>) of each of the EvoFold predictions in the set of 169 genes described above <a href="#pone.0001559-Hofacker1">[36]</a>. If we require our folds to have a negative <em>mfe</em> for the EvoFold prediction in <em>S. cerevisiae</em>, then 148 of these genes meet this requirement and we get an estimated false positive rate of 12.4% (and every gene with a negative <em>mfe</em> in <em>S. cerevisiae</em> also has a negative average <em>mfe</em> for all the species in the multi-species alignment).</p>
<a id="article1.body1.sec4.sec1.p5" name="article1.body1.sec4.sec1.p5"></a><p>Additionally, we used the method in Katz and Burge <a href="#pone.0001559-Katz1">[37]</a>. Briefly, we calculated the excess folding potential (EFP) for genes containing coding fRNAs, as described by Katz and Burge <a href="#pone.0001559-Katz1">[37]</a>. This method uses the DicodonShuffle algorithm <a href="#pone.0001559-Katz1">[37]</a>, and then uses the RNAfold program to determine if the native ORF has greater local mRNA stability than the shuffled ORF <a href="#pone.0001559-Hofacker1">[36]</a>. When considering the set of 169 genes considered for functional analysis (i.e. the stringently defined dataset with ribosomal genes removed), we found 101 genes containing coding fRNAs had an EFP greater than 0 (which would correspond to a 40% false positive rate). However this method may be inappropriate for our data, because we have defined in such a way that coding fRNAs must have at least a 50% overlap with coding regions, allowing folds to have some overlap with upstream and downstream regions. The codon shuffled method above cannot capture selection for stability in non-coding regions surrounding ORFs. Thus, the false positive rates estimated this method is likely an overestimate. Nevertheless, evolutionary analyses yielded similar results after excluding false positives detected by these two methods (<a href="#pone-0001559-t001">Tables 1</a> and <a href="#pone-0001559-t002">2</a> versus <a href="#pone-0001559-t003">Tables 3</a> and <a href="#pone-0001559-t004">4</a>). This renders strong support to our conclusions that coding fRNAs likely to constrain evolutionary rates.</p>


<h4>GO annotation analysis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Biological Process GO Slim annotations were downloaded using the SGD GO Term Mapper interface <a href="#pone.0001559-Cherry1">[35]</a>, <a href="#pone.0001559-Ashburner1">[38]</a>. Enrichment of GO annotations was calculated by using the proportion test in R <a href="#pone.0001559-R1">[39]</a>. Similar results were found when a hypergeometric distribution was used to determine enrichment of GO terms. The Wilcoxon rank sum test was used to compare average values for fRNA coverage and evolutionary divergence (small dN, small dS, small dS?, small dN/dS, small dN/dS?) between the entire strictly defined dataset and subsets of genes associated with Cell Cycle (GO ID: 7049), Organelle Organization and Biogenesis (GO ID: 6996), RNA Metabolic Process (GO ID: 16070), Transcription (GO ID: 6350), Translation (GO ID: 6412), and Transport (GO ID: 6810). We used values for evolutionary divergence across a shorter timescale because they provided a larger dataset in order to control for bias due to small sample size. We choose the above subsets of genes because these are the only categories of GO annotations associated with greater than 20 genes in the strictly defined dataset, and a sample size of less than 20 genes would be too small for a robust statistical analysis.</p>


<h4>Calculation of nonsynonymous and synonymous divergence</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>We used data from Wall et al. <a href="#pone.0001559-Namy1">[8]</a> (available from the supplementary material for Drummond et al. <a href="#pone.0001559-Higuchi1">[7]</a>), which are evolutionary rates at synonymous and nonsynonymous sites calculated over four yeast genomes, providing an evolutionary measure of protein divergence. Additionally, we estimated divergence on the shorter timescale (referred to as small dN, small dS in the main text and <a href="#pone.0001559.s001">Text S1</a>) between <em>S. cerevisiae</em> and <em>S. paradoxus</em> using PAML <a href="#pone.0001559-Yang1">[40]</a>. Adjustment for codon usage bias at synonymous sites was calculated as described by <a href="#pone.0001559-Hirsh1">[18]</a> (namely, dS′ = dS−m* c, where m = −2.02 for the all 4-species and m = −0.386 for <em>S. cerevisiae</em>−<em>S. paradoxus</em> divergence). Recalculation on a shorter timescale is useful because it provides an opportunity to see if coding fRNAs have a different evolutionary impact on a species that are more closely related.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>In order to recalculate nonsynonymous and synonymous divergence on a shorter timescale, the Multiz alignment was downloaded for all verified protein-coding genes containing at least one coding functional RNA secondary structure <a href="#pone.0001559-Blanchette1">[34]</a>. The Multiz alignment for these genes was obtained using the Galaxy server on the UCSC Genome Browser <a href="#pone.0001559-Giardine1">[41]</a>, <a href="#pone.0001559-Karolchik1">[42]</a>. All coding fRNAs were first defined using annotations for protein-coding genes from the SGD database <a href="#pone.0001559-Cherry1">[35]</a>, <a href="#pone.0001559-Karolchik1">[42]</a>. We only considered experimentally verified SGD annotations for protein-coding genes. In order to obtain reliable values for evolutionary divergence using PAML, the set of genes was further filtered based upon the quality of the Multiz alignment. More specifically, we removed genes with introns, premature stop codons and/or gaps in Multiz alignment, alignments without all 4 species, non-AUG start codon, and genes less than 300 bp.</p>


<h4>Functional variables considered in the analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>In order to assess the biological relevance of our predicted coding fRNAs, we used rigorous statistical analysis to study the impact of fRNA coverage on evolutionary rates, relative to other previously established functional variables. The functional variables analyzed include gene expression, CAI, protein abundance, gene dispensability, gene length, degree, centrality, and mRNA half-life. Gene expression and mRNA half-life values are from Holstege et al. <a href="#pone.0001559-Holstege1">[43]</a>. Codon Adaptation Index (CAI) and gene length are from Drummond et al. <a href="#pone.0001559-Drummond1">[9]</a>. Protein abundance data are from Ghaemmaghami et al. <a href="#pone.0001559-Ghaemmaghami1">[44]</a>. Dispensability data was downloaded from <a href="http://chemogenomics.stanford.edu/supplements/01yfh/files/orfgenedata.txt">http://chemogenomics.stanford.edu/supple​ments/01yfh/files/orfgenedata.txt</a> <a href="#pone.0001559-Deutschbauer1">[45]</a>. The number of interactions in the yeast protein-protein interaction network (degree) was from the filtered yeast interactome data set <a href="#pone.0001559-Han1">[46]</a>. This dataset was also used to calculate the centrality for genes in the protein-protein interaction network.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Gene length, protein abundance, degree, centrality, and mRNA half-life were excluded from certain comparisons. Gene length was excluded from analysis simply because fRNA coverage is strongly correlated with gene length, meaning that the two variables are clearly not independent. As described earlier, most other variables were excluded to remove bias from small sample size and/or overfitting.</p>


<h4>Multivariate statistical analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Partial correlation and principal component regression are two primary tools for functional genomic analysis in yeast. These statistical tools work in fundamentally different ways, and combined analysis can provide useful information about significant biological factors that govern evolutionary rates <a href="#pone.0001559-Kim1">[11]</a>, <a href="#pone.0001559-Lin1">[47]</a>. More specifically, partial correlation analysis factors out the influence of a third <em>known</em> variable (or vector of known variables), while principal component regression analyzes the variance for a set of independent variables in order to identify <em>unknown</em> variables. For a more detailed discussion on the comparative performance of these two tools, see Kim and Yi <a href="#pone.0001559-Kim1">[11]</a>.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Thus, principal component regression analysis requires two steps; first, a principal component analysis to define components and second, a regression analysis to determine which components have a statistically significant impact on evolutionary rates. Principal component regression was carried out using the R “pls” package <a href="#pone.0001559-R1">[39]</a>, <a href="#pone.0001559-Mevik1">[48]</a>.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>Partial correlation analysis can be carried out by applying the relatively straightforward equation r<sub>DK|X</sub> = (r<sub>DK</sub>−r<sub>DX</sub>*r<sub>KX</sub>)/√[(1−r<sup>2</sup><sub>DX</sub>)(1−r<sup>2</sup><sub>KX</sub>)] when testing for a correlation between D and K while factoring out the influence of the third variable (or vector of variables) X. In other words, partial correlation analysis can be also used to remove the effects of a set of variables. Here, we can define X as a vector of the other N variables X<sub>1</sub>, X<sub>2</sub>, …X<sub>N</sub>. Then the correlation between D and K independent of X can be calculated as the correlation between D-D(X<sub>1</sub>, X<sub>2</sub>,…, X<sub>N</sub>) and K-K(X<sub>1</sub>, X<sub>2</sub>, …, X<sub>N</sub>), where D(X<sub>1</sub>, X<sub>2</sub>, …, X<sub>N</sub>) and K(X<sub>1</sub>, X<sub>2</sub>, …, X<sub>N</sub>) are the multiple linear regression of D and K, respectively, on X<sub>1</sub>, X<sub>2</sub>,..X<sub>N</sub>. This method was used in Kim and Yi <a href="#pone.0001559-Kim1">[11]</a> to assess the independent effect of each functional variable. We can also use the variance-covariance matrix using the assumption of normality (p. 134, <a href="#pone.0001559-Whittaker1">[49]</a>).</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>We modified R scripts available from the supplemental material for Drummond et al. 2006 for partial correlation (factoring out only expression) and principal component regression analysis <a href="#pone.0001559-R1">[39]</a>. An R code for our method of partial correlation analysis that controls for the influence of multiple variables (which was used to produce the data in <a href="#pone-0001559-t001">Table 1</a>) is available at Yi lab website (<a href="http://www.yilab.gatech.edu">www.yilab.gatech.edu</a>).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001559"></div><a name="pone.0001559.s001" id="pone.0001559.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001559.s001">Text S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001559.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.14 MB PDF)</p>
<a name="pone.0001559.s002" id="pone.0001559.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001559.s002">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001559.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.07 MB TIF)</p>
<a name="pone.0001559.s003" id="pone.0001559.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001559.s003">Figure S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001559.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.15 MB TIF)</p>
<a name="pone.0001559.s004" id="pone.0001559.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001559.s004">Figure S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001559.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(0.14 MB TIF)</p>
<a name="pone.0001559.s005" id="pone.0001559.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001559.s005">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001559.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0001559.s006" id="pone.0001559.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001559.s006">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001559.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001559.s007" id="pone.0001559.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001559.s007">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001559.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0001559.s008" id="pone.0001559.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001559.s008">Table S4. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001559.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0001559.s009" id="pone.0001559.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001559.s009">Table S5. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001559.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0001559.s010" id="pone.0001559.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001559.s010">Table S6. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001559.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0001559.s011" id="pone.0001559.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001559.s011">Table S7. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001559.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="postSiDOI">(0.05 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Jakob Pedersen for explaining the technical details of the EvoFold program. The Office of Information Technology at Georgia Tech generously allowed us to conduct research on the PACE Linux cluster. Navin Elango and Marco Lin provided valuable computational assistance. We thank Navin Elango, Zuogang Peng, King Jordan, Todd Streelman, Anton Nekrutenko, and an anonymous reviewer for useful comments and/or critical reading of this manuscript. Part of this research has been published as CDW's Honors Thesis from Georgia Institute of Technology.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SY CW. Performed the experiments: SK CW. Analyzed the data: SK CW. Contributed reagents/materials/analysis tools: SY. Wrote the paper: SY CW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001559-Storz1" id="pone.0001559-Storz1"></a>Storz G, Altuvia S, Wassarman KM (2005) An abundance of RNA regulators. Annual Review of Biochemistry  74: 199–217.  <ul class="find" data-citedArticleID="1052168" data-doi="10.1146/annurev.biochem.74.082803.133136"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.74.082803.133136" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+abundance+of+RNA+regulators." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+abundance+of+RNA+regulators.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001559-Ouellet1" id="pone.0001559-Ouellet1"></a>Ouellet DL, Perron MP, Gobeil LA, Plante P, Provost P (2006) MicroRNAs in gene regulation: When the smallest governs it all. Journal of Biomedicine and Biotechnology  2006: Article ID 69616, 69620 pages.  <ul class="find" data-citedArticleID="1052148" data-doi="10.1155/jbb/2006/69616"><li><a href="http://dx.doi.org/10.1155/jbb/2006/69616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs+in+gene+regulation%3A+When+the+smallest+governs+it+all." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs+in+gene+regulation%3A+When+the+smallest+governs+it+all.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001559-Pedersen1" id="pone.0001559-Pedersen1"></a>Pedersen JS, Bejerano G, Siepel A, Rosenbloom K, Lindblad-Toh K, et al.  (2006) Identification and classification of conserved RNA secondary structures in the human genome. PLoS Computational Biology  2: 251–262.  <ul class="find" data-citedArticleID="1052150" data-doi="10.1371/journal.pcbi.0020033.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020033.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+classification+of+conserved+RNA+secondary+structures+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+classification+of+conserved+RNA+secondary+structures+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001559-Steigele1" id="pone.0001559-Steigele1"></a>Steigele S, Huber W, Stocsits C, Stadler PF, Nieselt K (2007) Comparative analysis of structured RNAs in S. cerevisiae indicates a multitude of different functions. BMC Biology  5:   <ul class="find" data-citedArticleID="1052166" data-doi="10.1186/1741-7007-5-25"><li><a href="http://dx.doi.org/10.1186/1741-7007-5-25" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+structured+RNAs+in+S.+cerevisiae+indicates+a+multitude+of+different+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+structured+RNAs+in+S.+cerevisiae+indicates+a+multitude+of+different+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001559-Stark1" id="pone.0001559-Stark1"></a>Stark A, Lin MF, Kheradpour P, Pedersen JS, Parts L, et al.  (2007) Discovery of functional elements in 12 Drosophila genomes using evolutionary signatures. Nature  450: 219–232.  <ul class="find" data-citedArticleID="1052164" data-doi="10.1038/nature06340"><li><a href="http://dx.doi.org/10.1038/nature06340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovery+of+functional+elements+in+12+Drosophila+genomes+using+evolutionary+signatures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovery+of+functional+elements+in+12+Drosophila+genomes+using+evolutionary+signatures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001559-Matsufuji1" id="pone.0001559-Matsufuji1"></a>Matsufuji S, Matsufuji T, Miyazaki Y, Murakami Y, Atkins JF, et al.  (1995) Autoregulatory Frameshifting In Decoding Mammalian Ornithine Decarboxylase Antizyme. Cell  80: 51–60.  <ul class="find" data-citedArticleID="1052138" data-doi="10.1016/0092-8674(95)90450-6"><li><a href="http://dx.doi.org/10.1016/0092-8674(95)90450-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Autoregulatory+Frameshifting+In+Decoding+Mammalian+Ornithine+Decarboxylase+Antizyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Autoregulatory+Frameshifting+In+Decoding+Mammalian+Ornithine+Decarboxylase+Antizyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001559-Higuchi1" id="pone.0001559-Higuchi1"></a>Higuchi M, Stefan M, Single FN, Hartner J, Rozov A, et al.  (2000) Point mutation in an AMPA receptor gene rescues lethality in mice deficient in the RNA-editing enzyme ADAR2. Nature  406: 78–81.  <ul class="find" data-citedArticleID="1052116"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Point+mutation+in+an+AMPA+receptor+gene+rescues+lethality+in+mice+deficient+in+the+RNA-editing+enzyme+ADAR2.&amp;auth=&amp;atitle=Point+mutation+in+an+AMPA+receptor+gene+rescues+lethality+in+mice+deficient+in+the+RNA-editing+enzyme+ADAR2." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Point+mutation+in+an+AMPA+receptor+gene+rescues+lethality+in+mice+deficient+in+the+RNA-editing+enzyme+ADAR2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Point+mutation+in+an+AMPA+receptor+gene+rescues+lethality+in+mice+deficient+in+the+RNA-editing+enzyme+ADAR2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001559-Namy1" id="pone.0001559-Namy1"></a>Namy O, Rousset JP, Napthine S, Brierley I (2004) Reprogrammed genetic decoding in cellular gene expression. Molecular Cell  13: 157–168.  <ul class="find" data-citedArticleID="1052146" data-doi="10.1016/s1097-2765(04)00031-0"><li><a href="http://dx.doi.org/10.1016/s1097-2765(04)00031-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reprogrammed+genetic+decoding+in+cellular+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reprogrammed+genetic+decoding+in+cellular+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001559-Drummond1" id="pone.0001559-Drummond1"></a>Drummond DA, Raval A, Wilke CO (2006) A single determinant dominates the rate of yeast protein evolution. Molecular Biology and Evolution  23: 327–337.  <ul class="find" data-citedArticleID="1052098" data-doi="10.1093/molbev/msj038"><li><a href="http://dx.doi.org/10.1093/molbev/msj038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+determinant+dominates+the+rate+of+yeast+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+determinant+dominates+the+rate+of+yeast+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001559-Wall1" id="pone.0001559-Wall1"></a>Wall DP, Hirsh AE, Fraser HB, Kumm J, Giaever G, et al.  (2005) Functional genomic analysis of the rates of protein evolution. Proceedings of the National Academy of Sciences of the United States of America  102: 5483–5488.  <ul class="find" data-citedArticleID="1052170" data-doi="10.1073/pnas.0501761102"><li><a href="http://dx.doi.org/10.1073/pnas.0501761102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+genomic+analysis+of+the+rates+of+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+genomic+analysis+of+the+rates+of+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001559-Kim1" id="pone.0001559-Kim1"></a>Kim S-H, Yi SV (2007) Understanding relationship between sequence and functional evolution in yeast proteins. Genetica  131: 151–156.  <ul class="find" data-citedArticleID="1052130" data-doi="10.1007/s10709-006-9125-2"><li><a href="http://dx.doi.org/10.1007/s10709-006-9125-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+relationship+between+sequence+and+functional+evolution+in+yeast+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+relationship+between+sequence+and+functional+evolution+in+yeast+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001559-Washietl1" id="pone.0001559-Washietl1"></a>Washietl S, Hofacker IL, Stadler PF (2005) Fast and reliable prediction of noncoding RNAs. Proceedings of the National Academy of Sciences of the United States of America  102: 2454–2459.  <ul class="find" data-citedArticleID="1052172" data-doi="10.1073/pnas.0409169102"><li><a href="http://dx.doi.org/10.1073/pnas.0409169102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fast+and+reliable+prediction+of+noncoding+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fast+and+reliable+prediction+of+noncoding+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001559-Siepel1" id="pone.0001559-Siepel1"></a>Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou MM, et al.  (2005) Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Research  15: 1034–1050.  <ul class="find" data-citedArticleID="1052162" data-doi="10.1101/gr.3715005"><li><a href="http://dx.doi.org/10.1101/gr.3715005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001559-Guan1" id="pone.0001559-Guan1"></a>Guan YF, Dunham MJ, Troyanskaya OG (2007) Functional analysis of gene duplications in Saccharomyces cerevisiae. Genetics  175: 933–943.  <ul class="find" data-citedArticleID="1052110" data-doi="10.1534/genetics.106.064329"><li><a href="http://dx.doi.org/10.1534/genetics.106.064329" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+analysis+of+gene+duplications+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+analysis+of+gene+duplications+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001559-Gu1" id="pone.0001559-Gu1"></a>Gu ZL, Nicolae D, Lu HHS, Li WH (2002) Rapid divergence in expression between duplicate genes inferred from microarray data. Trends in Genetics  18: 609–613.  <ul class="find" data-citedArticleID="1052108" data-doi="10.1016/s0168-9525(02)02837-8"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02837-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+divergence+in+expression+between+duplicate+genes+inferred+from+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+divergence+in+expression+between+duplicate+genes+inferred+from+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001559-Drummond2" id="pone.0001559-Drummond2"></a>Drummond DA, Bloom JD, Adami C, Wilke CO, Arnold FH (2005) Why highly expressed proteins evolve slowly. Proceedings of the National Academy of Sciences of the United States of America  102: 14338–14343.  <ul class="find" data-citedArticleID="1052100" data-doi="10.1073/pnas.0504070102"><li><a href="http://dx.doi.org/10.1073/pnas.0504070102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+highly+expressed+proteins+evolve+slowly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+highly+expressed+proteins+evolve+slowly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001559-Kim2" id="pone.0001559-Kim2"></a>Kim SH, Yi SV (2006) Correlated asymmetry of sequence and functional divergence between duplicate proteins of Saccharomyces cerevisiae. Molecular Biology and Evolution  23: 1068–1075.  <ul class="find" data-citedArticleID="1052132" data-doi="10.1093/molbev/msj115"><li><a href="http://dx.doi.org/10.1093/molbev/msj115" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlated+asymmetry+of+sequence+and+functional+divergence+between+duplicate+proteins+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlated+asymmetry+of+sequence+and+functional+divergence+between+duplicate+proteins+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001559-Hirsh1" id="pone.0001559-Hirsh1"></a>Hirsh AE, Fraser HB, Wall DP (2005) Adjusting for selection on synonymous sites in estimates of evolutionary distance. Molecular Biology and Evolution  22: 174–177.  <ul class="find" data-citedArticleID="1052118" data-doi="10.1093/molbev/msh265"><li><a href="http://dx.doi.org/10.1093/molbev/msh265" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adjusting+for+selection+on+synonymous+sites+in+estimates+of+evolutionary+distance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adjusting+for+selection+on+synonymous+sites+in+estimates+of+evolutionary+distance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001559-Marais1" id="pone.0001559-Marais1"></a>Marais G, Duret L (2001) Synonymous codon usage, accuracy of translation, and gene length in Caenorhabditis elegans. Journal of Molecular Evolution  52: 275–280.  <ul class="find" data-citedArticleID="1052136"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Synonymous+codon+usage%2C+accuracy+of+translation%2C+and+gene+length+in+Caenorhabditis+elegans.&amp;auth=&amp;atitle=Synonymous+codon+usage%2C+accuracy+of+translation%2C+and+gene+length+in+Caenorhabditis+elegans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synonymous+codon+usage%2C+accuracy+of+translation%2C+and+gene+length+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synonymous+codon+usage%2C+accuracy+of+translation%2C+and+gene+length+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001559-Sidrauski1" id="pone.0001559-Sidrauski1"></a>Sidrauski C, Cox JS, Walter P (1996) tRNA ligase is required for regulated mRNA splicing in the unfolded protein response. Cell  87: 405–413.  <ul class="find" data-citedArticleID="1052160" data-doi="10.1016/s0092-8674(00)81361-6"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81361-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNA+ligase+is+required+for+regulated+mRNA+splicing+in+the+unfolded+protein+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNA+ligase+is+required+for+regulated+mRNA+splicing+in+the+unfolded+protein+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001559-Kawahara1" id="pone.0001559-Kawahara1"></a>Kawahara T, Yanagi H, Yura T, Mori K (1998) Unconventional splicing of HAC1/ERN4 mRNA required for the unfolded protein response - Sequence-specific and non-sequential cleavage of the splice sites. Journal of Biological Chemistry  273: 1802–1807.  <ul class="find" data-citedArticleID="1052128" data-doi="10.1074/jbc.273.3.1802"><li><a href="http://dx.doi.org/10.1074/jbc.273.3.1802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unconventional+splicing+of+HAC1%2FERN4+mRNA+required+for+the+unfolded+protein+response+-+Sequence-specific+and+non-sequential+cleavage+of+the+splice+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unconventional+splicing+of+HAC1%2FERN4+mRNA+required+for+the+unfolded+protein+response+-+Sequence-specific+and+non-sequential+cleavage+of+the+splice+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001559-Chung1" id="pone.0001559-Chung1"></a>Chung W-Y, Wadhawan S, Szklarczyk R, Pond SK, Nekrutenko A (2007) A First Look at ARFome: Dual-Coding Genes in Mammalian Genomes. PLoS Computational Biology  3: e91.  <ul class="find" data-citedArticleID="1052094" data-doi="10.1371/journal.pcbi.0030091"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0030091" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+First+Look+at+ARFome%3A+Dual-Coding+Genes+in+Mammalian+Genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+First+Look+at+ARFome%3A+Dual-Coding+Genes+in+Mammalian+Genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001559-McCutcheon1" id="pone.0001559-McCutcheon1"></a>McCutcheon JP, Eddy SR (2003) Computational identification of non-coding RNAs in Saccharomyces cerevisiae by comparative genomics. Nucleic Acids Research  31: 4119–4128.  <ul class="find" data-citedArticleID="1052140" data-doi="10.1093/nar/gkg438"><li><a href="http://dx.doi.org/10.1093/nar/gkg438" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+identification+of+non-coding+RNAs+in+Saccharomyces+cerevisiae+by+comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+identification+of+non-coding+RNAs+in+Saccharomyces+cerevisiae+by+comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001559-GriffithsJones1" id="pone.0001559-GriffithsJones1"></a>Griffiths-Jones S (2007) Annotating noncoding RNA genes. Annual Review of Genomics and Human Genetics  8: 279–298.  <ul class="find" data-citedArticleID="1052106" data-doi="10.1146/annurev.genom.8.080706.092419"><li><a href="http://dx.doi.org/10.1146/annurev.genom.8.080706.092419" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Annotating+noncoding+RNA+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Annotating+noncoding+RNA+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001559-Pollard1" id="pone.0001559-Pollard1"></a>Pollard KS, Salama SR, King B, Kern AD, Dreszer T, et al.  (2006) Forces shaping the fastest evolving regions in the human genome. PLoS Genetics  2: 1599–1611.  <ul class="find" data-citedArticleID="1052154" data-doi="10.1371/journal.pgen.0020168.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020168.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Forces+shaping+the+fastest+evolving+regions+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Forces+shaping+the+fastest+evolving+regions+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001559-Pollard2" id="pone.0001559-Pollard2"></a>Pollard KS, Salama SR, Lambert N, Lambot MA, Coppens S, et al.  (2006) An RNA gene expressed during cortical development evolved rapidly in humans. Nature  443: 167–172.  <ul class="find" data-citedArticleID="1052156" data-doi="10.1038/nature05113"><li><a href="http://dx.doi.org/10.1038/nature05113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+RNA+gene+expressed+during+cortical+development+evolved+rapidly+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+RNA+gene+expressed+during+cortical+development+evolved+rapidly+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001559-Piriyapongsa1" id="pone.0001559-Piriyapongsa1"></a>Piriyapongsa J, Marino-Ramirez L, Jordan IK (2007) Origin and evolution of human microRNAs from transposable elements. Genetics  176: 1323–1337.  <ul class="find" data-citedArticleID="1052152" data-doi="10.1534/genetics.107.072553"><li><a href="http://dx.doi.org/10.1534/genetics.107.072553" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+and+evolution+of+human+microRNAs+from+transposable+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+and+evolution+of+human+microRNAs+from+transposable+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001559-Washietl2" id="pone.0001559-Washietl2"></a>Washietl S, Pedersen JS, Korbel JO, Stocsits C, Gruber AR, et al.  (2007) Structured RNAs in the ENCODE selected regions of the human genome. Genome Research  17: 852–864.  <ul class="find" data-citedArticleID="1052174" data-doi="10.1101/gr.5650707"><li><a href="http://dx.doi.org/10.1101/gr.5650707" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structured+RNAs+in+the+ENCODE+selected+regions+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structured+RNAs+in+the+ENCODE+selected+regions+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001559-Zhang1" id="pone.0001559-Zhang1"></a>Zhang Z, Pang AWC, Gerstein M (2007) Comparative analysis of genome tiling array data reveals many novel primate-specific functional RNAs in human. BMC Evolutionary Biology  7: Suppl 1S14.  <ul class="find" data-citedArticleID="1052184" data-doi="10.1186/1471-2148-7-s1-s14"><li><a href="http://dx.doi.org/10.1186/1471-2148-7-s1-s14" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+genome+tiling+array+data+reveals+many+novel+primate-specific+functional+RNAs+in+human." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+genome+tiling+array+data+reveals+many+novel+primate-specific+functional+RNAs+in+human.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001559-Hertel1" id="pone.0001559-Hertel1"></a>Hertel J, Stadler PF (2006) Hairpins in a Haystack: recognizing microRNA precursors in comparative genomics data. Bioinformatics  22: E197–E202.  <ul class="find" data-citedArticleID="1052114" data-doi="10.1093/bioinformatics/btl257"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl257" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hairpins+in+a+Haystack%3A+recognizing+microRNA+precursors+in+comparative+genomics+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hairpins+in+a+Haystack%3A+recognizing+microRNA+precursors+in+comparative+genomics+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001559-Weile1" id="pone.0001559-Weile1"></a>Weile C, Gardner PP, Hedegaard MM, Vinther J (2007) Use of tiling array data and RNA secondary structure predictions to identify noncoding RNA genes. Bmc Genomics  8:   <ul class="find" data-citedArticleID="1052178" data-doi="10.1186/1471-2164-8-244"><li><a href="http://dx.doi.org/10.1186/1471-2164-8-244" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+tiling+array+data+and+RNA+secondary+structure+predictions+to+identify+noncoding+RNA+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+tiling+array+data+and+RNA+secondary+structure+predictions+to+identify+noncoding+RNA+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001559-Washietl3" id="pone.0001559-Washietl3"></a>Washietl S, Hofacker IL, Lukasser M, Huttenhofer A, Stadler PF (2005) Mapping of conserved RNA secondary structures predicts thousands of functional noncoding RNAs in the human genome. Nature Biotechnology  23: 1383–1390.  <ul class="find" data-citedArticleID="1052176" data-doi="10.1038/nbt1144"><li><a href="http://dx.doi.org/10.1038/nbt1144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+of+conserved+RNA+secondary+structures+predicts+thousands+of+functional+noncoding+RNAs+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+of+conserved+RNA+secondary+structures+predicts+thousands+of+functional+noncoding+RNAs+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001559-Missal1" id="pone.0001559-Missal1"></a>Missal K, Rose D, Stadler PF (2005) Non-coding RNAs in Ciona intestinalis. Bioinformatics  21: 77–78.  <ul class="find" data-citedArticleID="1052144" data-doi="10.1093/bioinformatics/bti1113"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti1113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-coding+RNAs+in+Ciona+intestinalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-coding+RNAs+in+Ciona+intestinalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001559-Blanchette1" id="pone.0001559-Blanchette1"></a>Blanchette M, Kent WJ, Riemer C, Elnitski L, Smit AFA, et al.  (2004) Aligning multiple genomic sequences with the threaded blockset aligner. Genome Research  14: 708–715.  <ul class="find" data-citedArticleID="1052090" data-doi="10.1101/gr.1933104"><li><a href="http://dx.doi.org/10.1101/gr.1933104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aligning+multiple+genomic+sequences+with+the+threaded+blockset+aligner." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aligning+multiple+genomic+sequences+with+the+threaded+blockset+aligner.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001559-Cherry1" id="pone.0001559-Cherry1"></a>Cherry JM, Ball C, Weng S, Juvik G, Schmidt R, et al.  (1997) Genetic and physical maps of Saccharomyces cerevisiae. Nature  387: 67–73.  <ul class="find" data-citedArticleID="1052092"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+and+physical+maps+of+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Genetic+and+physical+maps+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+and+physical+maps+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+and+physical+maps+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001559-Hofacker1" id="pone.0001559-Hofacker1"></a>Hofacker IL, Fontana W, Stadler PF, Bonhoeffer LS, Tacker M, et al.  (1994) Fast Folding and Comparison of RNA Secondary Structures. Monatshefte Fur Chemie  125: 167–188.  <ul class="find" data-citedArticleID="1052120" data-doi="10.1007/bf00818163"><li><a href="http://dx.doi.org/10.1007/bf00818163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fast+Folding+and+Comparison+of+RNA+Secondary+Structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fast+Folding+and+Comparison+of+RNA+Secondary+Structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001559-Katz1" id="pone.0001559-Katz1"></a>Katz L, Burge CB (2003) Widespread selection for local RNA secondary structure in coding regions of bacterial genes. Genome Research  13: 2042–2051.  <ul class="find" data-citedArticleID="1052126" data-doi="10.1101/gr.1257503"><li><a href="http://dx.doi.org/10.1101/gr.1257503" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+selection+for+local+RNA+secondary+structure+in+coding+regions+of+bacterial+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+selection+for+local+RNA+secondary+structure+in+coding+regions+of+bacterial+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001559-Ashburner1" id="pone.0001559-Ashburner1"></a>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al.  (2000) Gene Ontology: tool for the unification of biology. Nature Genetics  25: 25–29.  <ul class="find" data-citedArticleID="1052088" data-doi="10.1038/75556"><li><a href="http://dx.doi.org/10.1038/75556" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+Ontology%3A+tool+for+the+unification+of+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+Ontology%3A+tool+for+the+unification+of+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001559-R1" id="pone.0001559-R1"></a>R Development Core Team (2006) R: A language and environment for statistical computing. Vienna, Austria: R Foundation for Statistical Computing.   <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0001559-Yang1" id="pone.0001559-Yang1"></a>Yang ZH (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Computer Applications in the Biosciences  13: 555–556.  <ul class="find" data-citedArticleID="1052182" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001559-Giardine1" id="pone.0001559-Giardine1"></a>Giardine B, Riemer C, Hardison RC, Burhans R, Elnitski L, et al.  (2005) Galaxy: A platform for interactive large-scale genome analysis. Genome Research  15: 1451–1455.  <ul class="find" data-citedArticleID="1052104" data-doi="10.1101/gr.4086505"><li><a href="http://dx.doi.org/10.1101/gr.4086505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Galaxy%3A+A+platform+for+interactive+large-scale+genome+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Galaxy%3A+A+platform+for+interactive+large-scale+genome+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001559-Karolchik1" id="pone.0001559-Karolchik1"></a>Karolchik D, Baertsch R, Diekhans M, Furey TS, Hinrichs A, et al.  (2003) The UCSC Genome Browser Database. Nucleic Acids Research  31: 51–54.  <ul class="find" data-citedArticleID="1052124" data-doi="10.1093/nar/gkg129"><li><a href="http://dx.doi.org/10.1093/nar/gkg129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+UCSC+Genome+Browser+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+UCSC+Genome+Browser+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001559-Holstege1" id="pone.0001559-Holstege1"></a>Holstege FCP, Jennings EG, Wyrick JJ, Lee TI, Hengartner CJ, et al.  (1998) Dissecting the regulatory circuitry of a eukaryotic genome. Cell  95: 717–728.  <ul class="find" data-citedArticleID="1052122" data-doi="10.1016/s0092-8674(00)81641-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81641-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissecting+the+regulatory+circuitry+of+a+eukaryotic+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissecting+the+regulatory+circuitry+of+a+eukaryotic+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001559-Ghaemmaghami1" id="pone.0001559-Ghaemmaghami1"></a>Ghaemmaghami S, Huh W, Bower K, Howson RW, Belle A, et al.  (2003) Global analysis of protein expression in yeast. Nature  425: 737–741.  <ul class="find" data-citedArticleID="1052102" data-doi="10.1038/nature02046"><li><a href="http://dx.doi.org/10.1038/nature02046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+analysis+of+protein+expression+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+analysis+of+protein+expression+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001559-Deutschbauer1" id="pone.0001559-Deutschbauer1"></a>Deutschbauer AM, Jaramillo DF, Proctor M, Kumm J, Hillenmeyer ME, et al.  (2005) Mechanisms of haploinsufficiency revealed by genome-wide profiling in yeast. Genetics  169: 1915–1925.  <ul class="find" data-citedArticleID="1052096" data-doi="10.1534/genetics.104.036871"><li><a href="http://dx.doi.org/10.1534/genetics.104.036871" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+haploinsufficiency+revealed+by+genome-wide+profiling+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+haploinsufficiency+revealed+by+genome-wide+profiling+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001559-Han1" id="pone.0001559-Han1"></a>Han JDJ, Bertin N, Hao T, Goldberg DS, Berriz GF, et al.  (2004) Evidence for dynamically organized modularity in the yeast protein-protein interaction network. Nature  430: 88–93.  <ul class="find" data-citedArticleID="1052112" data-doi="10.1038/nature02555"><li><a href="http://dx.doi.org/10.1038/nature02555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+dynamically+organized+modularity+in+the+yeast+protein-protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+dynamically+organized+modularity+in+the+yeast+protein-protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001559-Lin1" id="pone.0001559-Lin1"></a>Lin YS, Hsu WL, Hwang JK, Li WH (2007) Proportion of solvent-exposed amino acids in a protein and rate of protein evolution. Molecular Biology and Evolution  24: 1005–1011.  <ul class="find" data-citedArticleID="1052134" data-doi="10.1093/molbev/msm019"><li><a href="http://dx.doi.org/10.1093/molbev/msm019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proportion+of+solvent-exposed+amino+acids+in+a+protein+and+rate+of+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proportion+of+solvent-exposed+amino+acids+in+a+protein+and+rate+of+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001559-Mevik1" id="pone.0001559-Mevik1"></a>Mevik BH, Wehrens R (2007) The pls package: Principal component and partial least squares regression in R. Journal of Statistical Software  18:   <ul class="find" data-citedArticleID="1052142"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+pls+package%3A+Principal+component+and+partial+least+squares+regression+in+R.&amp;auth=&amp;atitle=The+pls+package%3A+Principal+component+and+partial+least+squares+regression+in+R." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+pls+package%3A+Principal+component+and+partial+least+squares+regression+in+R." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+pls+package%3A+Principal+component+and+partial+least+squares+regression+in+R.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001559-Whittaker1" id="pone.0001559-Whittaker1"></a>Whittaker J (1996) Graphical models in applied multivariate statistics. New York: John Wiley and Sons.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.XML" value="115873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.PDF" value="217653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.g001.PNG_L" value="113649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.g001.PNG_M" value="50083"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.g001.PNG_S" value="6376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.g001.TIF" value="188712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.g001.PNG_I" value="17326"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.g002.PNG_L" value="1423051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.g002.PNG_M" value="88557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.g002.PNG_S" value="8016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.g002.TIF" value="4139226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.g002.PNG_I" value="28930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t001.PNG_L" value="109118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t001.PNG_M" value="67688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t001.PNG_S" value="9493"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t001.TIF" value="462246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t001.PNG_I" value="21216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t002.PNG_L" value="131152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t002.PNG_M" value="166782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t002.PNG_S" value="21228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t002.TIF" value="588430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t002.PNG_I" value="62856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t003.PNG_L" value="108641"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t003.PNG_M" value="66282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t003.PNG_S" value="9378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t003.TIF" value="454958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t003.PNG_I" value="20709"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t004.PNG_L" value="113994"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t004.PNG_M" value="152913"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t004.PNG_S" value="19939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t004.TIF" value="540066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.t004.PNG_I" value="53152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.s001.PDF" value="137377"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.s002.TIF" value="41477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.s003.TIF" value="119285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.s004.TIF" value="108653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.s005.DOC" value="50688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.s006.DOC" value="35840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.s007.DOC" value="48640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.s008.DOC" value="43008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.s009.DOC" value="63488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.s010.DOC" value="46592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001559.s011.DOC" value="42496"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001559&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001559" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001559&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001559&volume=&issue=&title=Predicted Functional RNAs within Coding Regions Constrain Evolutionary Rates of Yeast Proteins&author_name=Charles%20D.%20Warden%2C%20Seong-Ho%20Kim%2C%20Soojin%20V.%20Yi&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001559" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001559" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001559" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001559&amp;t=Predicted%20Functional%20RNAs%20within%20Coding%20Regions%20Constrain%20Evolutionary%20Rates%20of%20Yeast%20Proteins" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001559&title=Predicted%20Functional%20RNAs%20within%20Coding%20Regions%20Constrain%20Evolutionary%20Rates%20of%20Yeast%20Proteins&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001559&amp;title=Predicted%20Functional%20RNAs%20within%20Coding%20Regions%20Constrain%20Evolutionary%20Rates%20of%20Yeast%20Proteins" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001559" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001559&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Predicted Functional RNAs within Coding Regions Constrain Evolutionary Rates of Yeast Proteins';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001559';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Predicted%20Functional%20RNAs%20within%20Coding%20Regions%20Constrain%20Evolutionary%20Rates%20of%20Yeast%20Proteins http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001559" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001559" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Divergent+evolution%22" title="Search for articles in the subject area:'Divergent evolution'"><div class="flagText">Divergent evolution</div></a>
              <div data-categoryid="17199" data-articleid="25944"
                   data-categoryname="Divergent evolution"
                   class="flagImage" title="Flag 'Divergent evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="25944"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+evolution%22" title="Search for articles in the subject area:'Fungal evolution'"><div class="flagText">Fungal evolution</div></a>
              <div data-categoryid="17551" data-articleid="25944"
                   data-categoryname="Fungal evolution"
                   class="flagImage" title="Flag 'Fungal evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25944"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Molecular+evolution%22" title="Search for articles in the subject area:'Molecular evolution'"><div class="flagText">Molecular evolution</div></a>
              <div data-categoryid="17533" data-articleid="25944"
                   data-categoryname="Molecular evolution"
                   class="flagImage" title="Flag 'Molecular evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+alignment%22" title="Search for articles in the subject area:'RNA alignment'"><div class="flagText">RNA alignment</div></a>
              <div data-categoryid="32671" data-articleid="25944"
                   data-categoryname="RNA alignment"
                   class="flagImage" title="Flag 'RNA alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+structure%22" title="Search for articles in the subject area:'RNA structure'"><div class="flagText">RNA structure</div></a>
              <div data-categoryid="20937" data-articleid="25944"
                   data-categoryname="RNA structure"
                   class="flagImage" title="Flag 'RNA structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="25944"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7183'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5687'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3114&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=13543">DNA-level constraints on sequence evolution</a><br>Posted by forsdyke</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
