

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A New In Vitro Strand Transfer Assay for Monitoring Bacterial Class 1 Integron Recombinase IntI1 Activity</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001315"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001315" />

    <meta name="citation_title" content="A New In Vitro Strand Transfer Assay for Monitoring Bacterial Class 1 Integron Recombinase IntI1 Activity"/>
    <meta itemprop="name" content="A New In Vitro Strand Transfer Assay for Monitoring Bacterial Class 1 Integron Recombinase IntI1 Activity"/>

      <meta name="citation_author" content="Véronique Dubois"/>
            <meta name="citation_author_institution" content="Laboratory of Cellular and Molecular Microbiology and Pathogenicity (MCMP), UMR 5097-CNRS, University Victor Segalen Bordeaux 2, Bordeaux, France"/>
      <meta name="citation_author" content="Carole Debreyer"/>
            <meta name="citation_author_institution" content="Laboratory of Cellular and Molecular Microbiology and Pathogenicity (MCMP), UMR 5097-CNRS, University Victor Segalen Bordeaux 2, Bordeaux, France"/>
      <meta name="citation_author" content="Simon Litvak"/>
            <meta name="citation_author_institution" content="Laboratory of Cellular and Molecular Microbiology and Pathogenicity (MCMP), UMR 5097-CNRS, University Victor Segalen Bordeaux 2, Bordeaux, France"/>
      <meta name="citation_author" content="Claudine Quentin"/>
            <meta name="citation_author_institution" content="Laboratory of Cellular and Molecular Microbiology and Pathogenicity (MCMP), UMR 5097-CNRS, University Victor Segalen Bordeaux 2, Bordeaux, France"/>
      <meta name="citation_author" content="Vincent Parissi"/>
            <meta name="citation_author_institution" content="Laboratory of Cellular and Molecular Microbiology and Pathogenicity (MCMP), UMR 5097-CNRS, University Victor Segalen Bordeaux 2, Bordeaux, France"/>

    <meta name="citation_date" content="2007/12/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001315.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1315"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Mobile gene cassettes and integrons: capture and spread of genes by site-specific recombination.; citation_author=RM Hall; citation_author=CM Collis; citation_journal_title=Mol Microbiol; citation_volume=15; citation_number=1; citation_pages=593-600; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mobile gene cassettes and integrons in evolution.; citation_author=RM Hall; citation_author=CM Collis; citation_author=MJ Kim; citation_author=SR Partridge; citation_author=GD Recchia; citation_journal_title=Ann N Y Acad Sci; citation_volume=870; citation_number=2; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Site-specific insertion of genes into integrons: role of the 59-base element and determination of the recombination cross-over point.; citation_author=RM Hall; citation_author=DE Brookes; citation_author=HW Stokes; citation_journal_title=Mol Microbiol; citation_volume=5; citation_number=3; citation_pages=1941-1959; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Characterisation of specific and secondary recombination sites recognised by the integron DNA integrase.; citation_author=GD Recchia; citation_author=HW Stokes; citation_author=RM Hall; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=4; citation_pages=2071-2078; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Expression of antibiotic resistance genes in the integrated cassettes of integrons.; citation_author=CM Collis; citation_author=RM Hall; citation_journal_title=Antimicrob Agents Chemother; citation_volume=39; citation_number=5; citation_pages=155-162; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Structure and function of 59-base element recombination sites associated with mobile gene cassettes.; citation_author=HW Stokes; citation_author=DB O'Gorman; citation_author=GD Recchia; citation_author=M Parsekhian; citation_author=RM Hall; citation_journal_title=Mol Microbiol; citation_volume=26; citation_number=6; citation_pages=731-745; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Secondary-sites for integration mediated by the Tn21 integrase.; citation_author=MV Francia; citation_author=F de la Cruz; citation_author=JM Garcia Lobo; citation_journal_title=Mol Microbiol; citation_volume=10; citation_number=7; citation_pages=823-828; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Non-palindromic attl sites of integrons are capable of site-specific recombination with one another and with secondary targets.; citation_author=K Hansson; citation_author=O Skold; citation_author=L Sundstrom; citation_journal_title=Mol Microbiol; citation_volume=26; citation_number=8; citation_pages=441-453; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Binding of the purified integron DNA integrase Intl1 to integron- and cassette-associated recombination sites.; citation_author=CM Collis; citation_author=MJ Kim; citation_author=HW Stokes; citation_author=RM Hall; citation_journal_title=Mol Microbiol; citation_volume=29; citation_number=9; citation_pages=477-490; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=DNA complexes obtained with the integron integrase IntI1 at the attI1 site.; citation_author=A Gravel; citation_author=B Fournier; citation_author=PH Roy; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=10; citation_pages=4347-4355; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Definition of the attI1 site of class 1 integrons.; citation_author=SR Partridge; citation_author=GD Recchia; citation_author=C Scaramuzzi; citation_author=CM Collis; citation_author=HW Stokes; citation_journal_title=Microbiology; citation_volume=146; citation_number=11; citation_pages=2855-2864; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The integrase family of recombinase: organization and function of the active site.; citation_author=I Grainge; citation_author=M Jayaram; citation_journal_title=Mol Microbiol; citation_volume=33; citation_number=12; citation_pages=449-456; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Efficiency of recombination reactions catalyzed by class 1 integron integrase IntI1.; citation_author=CM Collis; citation_author=GD Recchia; citation_author=MJ Kim; citation_author=HW Stokes; citation_author=RM Hall; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=13; citation_pages=2535-2542; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Integron cassette insertion: a recombination process involving a folded single strand substrate.; citation_author=M Bouvier; citation_author=G Demarre; citation_author=D Mazel; citation_journal_title=Embo J; citation_volume=24; citation_number=14; citation_pages=4356-4367; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structural basis for broad DNA specificity in integron recombination.; citation_author=D MacDonald; citation_author=G Demarre; citation_author=M Bouvier; citation_author=D Mazel; citation_author=DN Gopaul; citation_journal_title=Nature; citation_volume=27; citation_number=15; citation_pages=1157-1162; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The IntI1 integron integrase preferentially binds single-stranded DNA of the attC site.; citation_author=MV Francia; citation_author=JC Zabala; citation_author=F de la Cruz; citation_author=JM Garcia Lobo; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=16; citation_pages=6844-6849; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Integron integrase binds to bulged hairpin DNA.; citation_author=C Johansson; citation_author=M Kamali-Moghaddam; citation_author=L Sundstrom; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=17; citation_pages=4033-4043; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of a novel class 1 integron containing bla(GES-1) and a fused product of aac3-Ib/aac6'-Ib' gene cassettes in Pseudomonas aeruginosa.; citation_author=V Dubois; citation_author=L Poirel; citation_author=C Marie; citation_author=C Arpin; citation_author=P Nordmann; citation_journal_title=Antimicrob Agents Chemother; citation_volume=46; citation_number=18; citation_pages=638-645; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Integron-encoded IntI integrases preferentially recognize the adjacent cognate attI site in recombination with a 59-be site.; citation_author=CM Collis; citation_author=MJ Kim; citation_author=HW Stokes; citation_author=RM Hall; citation_journal_title=Mol Microbiol; citation_volume=46; citation_number=19; citation_pages=1415-1427; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of higher order intermediates and synapsis in the bent-L pathway of bacteriophage lambda site-specific recombination; citation_author=AM Segall; citation_journal_title=J Biol Chem,; citation_volume=273; citation_number=20; citation_pages=24258-24265; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Peptide inhibitors of DNA cleavage by tyrosine recombinase and topoisomerases; citation_author=M Klemm; citation_author=C Cheng; citation_author=G Cassell; citation_author=S Shuman; citation_author=AM Segall; citation_journal_title=Journal of Molecular Biology,; citation_volume=299; citation_number=21; citation_pages=1203-1216; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Point mutations in the integron integrase IntI1 that affect recombination and/or substrate recognition.; citation_author=A Gravel; citation_author=N Messier; citation_author=PH Roy; citation_journal_title=J Bacteriol; citation_volume=180; citation_number=22; citation_pages=5437-5442; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=An ultrasensitive site-specific DNA recombination assay based on dual-color fluorescence cross-correlation spectroscopy.; citation_author=M Jahnz; citation_author=P Schwille; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=23; citation_pages=e60; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Efficient excision of phage lambda from the Escherichia coli chromosome requires the Fis protein.; citation_author=CA Ball; citation_author=RC Johnson; citation_journal_title=J Bacteriol; citation_volume=173; citation_number=24; citation_pages=4027-4031; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The FLP protein of the 2-micron plasmid of yeast. Purification of the protein from Escherichia coli cells expressing the cloned FLP gene.; citation_author=D Babineau; citation_author=D Vetter; citation_author=BJ Andrews; citation_author=RM Gronostajski; citation_author=GA Proteau; citation_journal_title=J Biol Chem; citation_volume=260; citation_number=25; citation_pages=12313-12319; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=The FLP protein of the 2-micron plasmid of yeast. Inter- and intramolecular reactions.; citation_author=RM Gronostajski; citation_author=PD Sadowski; citation_journal_title=J Biol Chem; citation_volume=260; citation_number=26; citation_pages=12328-12335; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Comparative kinetic analysis of FLP and cre recombinases: mathematical models for DNA binding and recombination.; citation_author=L Ringrose; citation_author=V Lounnas; citation_author=L Ehrlich; citation_author=F Buchholz; citation_author=R Wade; citation_journal_title=J Mol Biol; citation_volume=284; citation_number=27; citation_pages=363-384; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A structural basis for allosteric control of DNA recombination by lambda integrase.; citation_author=T Biswas; citation_author=H Aihara; citation_author=M Radman-Livaja; citation_author=D Filman; citation_author=A Landy; citation_journal_title=Nature; citation_volume=435; citation_number=28; citation_pages=1059-1066; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001315" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A New In Vitro Strand Transfer Assay for Monitoring Bacterial Class 1 Integron Recombinase IntI1 Activity"/>
    <meta name="twitter:description" content="IntI1 integrase is a tyrosine recombinase involved in the mobility of antibiotic resistance gene cassettes within bacterial class 1 integrons. Recent data have shown that its recombination specifically involves the bottom strand of the attC site, but the exact mechanism of the reaction is still unclear. An efficient in vitro assay is still required to better characterize the biochemical properties of the enzyme. In this report we describe for the first time an in vitro system partially reproducing the activity of a recombinant pure IntI1. This new assay, which constitutes the only available in vitro model of recombination by IntI1, was used to determine whether this enzyme might be the sole bacterial protein required for the recombination process. Results show that IntI1 possesses all the features needed for performing recombination between attI and attC sites. However, differences in the in vitro intermolecular recombination efficiencies were found according to the target sites and were correlated with DNA affinities of the enzyme but not with in vivo data. The differential affinity of the enzyme for each site, its capacity to bind to a single-stranded structure at the attC site and the recombination observed with single-stranded substrates unambiguously confirm that it constitutes an important intermediary in the reaction. Our data strongly suggest that the enzyme possesses all the functions for generating and/or recognizing this structure even in the absence of other cellular factors. Furthermore, the in vitro assay reported here constitutes a powerful tool for the analysis of the recombination steps catalyzed by IntI1, its structure-function studies and the search for specific inhibitors."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001315.g009"/>

  <meta property="og:title" content="A New In Vitro Strand Transfer Assay for Monitoring Bacterial Class 1 Integron Recombinase IntI1 Activity" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001315" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4891'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6244'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4678&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001315">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001315" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001315&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A New <i>In Vitro</i> Strand Transfer Assay for Monitoring Bacterial Class 1 Integron Recombinase IntI1 Activity
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Véronique Dubois, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Cellular and Molecular Microbiology and Pathogenicity (MCMP), UMR 5097-CNRS, University Victor Segalen Bordeaux 2, Bordeaux, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carole Debreyer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Cellular and Molecular Microbiology and Pathogenicity (MCMP), UMR 5097-CNRS, University Victor Segalen Bordeaux 2, Bordeaux, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Simon Litvak, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Cellular and Molecular Microbiology and Pathogenicity (MCMP), UMR 5097-CNRS, University Victor Segalen Bordeaux 2, Bordeaux, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claudine Quentin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Cellular and Molecular Microbiology and Pathogenicity (MCMP), UMR 5097-CNRS, University Victor Segalen Bordeaux 2, Bordeaux, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vincent Parissi
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:vincent.parissi@reger.u-bordeaux2.fr">vincent.parissi@reger.u-bordeaux2.fr</a></p>

                <p>Affiliation:
                  Laboratory of Cellular and Molecular Microbiology and Pathogenicity (MCMP), UMR 5097-CNRS, University Victor Segalen Bordeaux 2, Bordeaux, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001315</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001315" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001315" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001315" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001315" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001315-g001" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001315-g002" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001315-g003" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001315-g004" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001315-g005" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001315-g006" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001315-g007" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001315-g008" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001315-g009" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001315">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001315" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>IntI1 integrase is a tyrosine recombinase involved in the mobility of antibiotic resistance gene cassettes within bacterial class 1 integrons. Recent data have shown that its recombination specifically involves the bottom strand of the <em>attC</em> site, but the exact mechanism of the reaction is still unclear. An efficient <em>in vitro</em> assay is still required to better characterize the biochemical properties of the enzyme. In this report we describe for the first time an <em>in vitro</em> system partially reproducing the activity of a recombinant pure IntI1. This new assay, which constitutes the only available <em>in vitro</em> model of recombination by IntI1, was used to determine whether this enzyme might be the sole bacterial protein required for the recombination process. Results show that IntI1 possesses all the features needed for performing recombination between <em>attI</em> and <em>attC</em> sites. However, differences in the <em>in vitro</em> intermolecular recombination efficiencies were found according to the target sites and were correlated with DNA affinities of the enzyme but not with <em>in vivo</em> data. The differential affinity of the enzyme for each site, its capacity to bind to a single-stranded structure at the <em>attC</em> site and the recombination observed with single-stranded substrates unambiguously confirm that it constitutes an important intermediary in the reaction. Our data strongly suggest that the enzyme possesses all the functions for generating and/or recognizing this structure even in the absence of other cellular factors. Furthermore, the <em>in vitro</em> assay reported here constitutes a powerful tool for the analysis of the recombination steps catalyzed by IntI1, its structure-function studies and the search for specific inhibitors.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Dubois V, Debreyer C, Litvak S, Quentin C, Parissi V (2007) A New <em>In Vitro</em> Strand Transfer Assay for Monitoring Bacterial Class 1 Integron Recombinase IntI1 Activity. PLoS ONE 2(12):
          e1315.
            doi:10.1371/journal.pone.0001315</p><p><strong>Academic Editor: </strong>Floyd Romesberg, The Scripps Research Institute, United States of America</p><p><strong>Received:</strong> July 24, 2007; <strong>Accepted:</strong> November 21, 2007; <strong>Published:</strong> December 19, 2007</p><p><strong>Copyright:</strong> © 2007 Dubois et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the CNRS and the University of Bordeaux 2.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Since the introduction of antibiotics in the treatment of human infectious diseases, bacterial resistance has become an ever-increasing problem that threatens the clinical usefulness of these drugs. The prevalence of antibiotic resistance is mainly due to the horizontal transfer of antibiotic-resistance genes, conveyed by mobile genetic elements such as plasmids and transposons. Integrons are a class of site-specific recombination elements which insert and excise mobile antibiotic resistance gene cassettes, and which are located on plasmids and/or transposons. All class 1 integrons consist of two conserved sequences (CS) flanking a variable central region encompassing antibiotic resistance gene cassettes <a href="#pone.0001315-Hall1">[1]</a>. The highly conserved 5′CS includes an <em>intI</em> gene encoding an integrase, an adjacent recombination site <em>attI</em> and a promoter region, while the 3′CS is more variable. While several classes of integrons have been identified according to the type of integrase, the most prevalent class 1 integrons are characterized by an <em>intI1</em> gene encoding an integrase of 337 amino acids.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Gene cassettes can exist in two forms: either as free covalently closed supercoiled circular molecules that are unable to replicate, or as linear molecules integrated at the <em>attI</em> site into integrons <a href="#pone.0001315-Hall2">[2]</a>. Gene cassettes consist of a single coding sequence carrying at its 3′ end an <em>attC</em> recombination site. The <em>attC</em> sites, also called 59-base elements, are essentially formed from two imperfect inverted repeats with a 7 bp core site GTTRRRY in the right end consensus region which is essential for <em>in vivo</em> recombination <a href="#pone.0001315-Hall3">[3]</a>–<a href="#pone.0001315-Stokes1">[6]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The integrase is a member of the tyrosine recombinase family, which catalyzes cassette integration and excision by a site-specific recombination, occurring naturally between the <em>attI</em> of the integron and the <em>attC</em> of a gene cassette, or between two <em>attC</em> sites. Insertion can also take place, albeit rarely, at non-specific or secondary DNA sites which display sequence analogies with the core site <a href="#pone.0001315-Recchia1">[4]</a>, <a href="#pone.0001315-Francia1">[7]</a>. The <em>attI</em> site is 70 bp long and contains four IntI1 binding sites at −50, −30, −7 and 0 including the 7 base core region GTTRRRY<a href="#pone.0001315-Hall2">[2]</a>, <a href="#pone.0001315-Recchia1">[4]</a>, <a href="#pone.0001315-Hansson1">[8]</a>–<a href="#pone.0001315-Partridge1">[11]</a>. The cross-over point occurs between the G base of a core site and the first T base of a second core site <a href="#pone.0001315-Francia1">[7]</a>–<a href="#pone.0001315-Grainge1">[12]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Until recently, the reaction catalyzed by the IntI1 integrase encoded by class 1 integrons has essentially been studied <em>in vivo.</em> In bacteria, IntI1 can catalyze recombination between either two <em>attC</em>, one <em>attI1</em> and one <em>attC</em>, or two <em>attI</em> sites <a href="#pone.0001315-Collis3">[13]</a>. Recent <em>in vivo</em> and structural data have provided important information on the mechanism by which recombination occurs in class 1 integrons. It has been clearly demonstrated that the <em>in vivo</em> recombination process involves the hairpin-folded bottom strand of <em>attC</em> <a href="#pone.0001315-Bouvier1">[14]</a>. The now available crystal structure of IntI from <em>Vibrio cholerae</em> bound to the bottom strand of <em>attC</em> site showed that DNA target site recognition and high-order synaptic assembly are not dependent on canonical DNA but on the position of two flipped-out bases that interact in <em>cis</em> and in <em>trans</em> with the integrase. These extrahelical bases originate from the folding of the bottom strand of <em>attC</em> due to its imperfect dyad symmetry <a href="#pone.0001315-MacDonald1">[15]</a>. All these new data confirm previous reports of the <em>in vitro</em> interaction between integrase and its DNA substrates <a href="#pone.0001315-Collis2">[9]</a>, <a href="#pone.0001315-Francia2">[16]</a>, <a href="#pone.0001315-Johansson1">[17]</a>. Taken together these results support a new paradigm for how sequence-degenerate single-stranded genetic material is recognized and exchanged between bacteria.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Despite these functional and structural breakthroughs, several points remain obscure. Is the single-stranded intermediate <em>attC</em> generated during bacterial DNA replication or by IntI1 itself? Is IntI1 sufficient as sole bacterial protein for performing all the recombination steps or does it require other factors? These questions might be answered by carrying out an <em>in vitro</em> assay using recombinant pure enzyme. To date, however, all attempts to set up such experimental systems have been unsuccessful and no <em>in vitro</em> assay has been available, making it difficult to perform further biochemical analysis of the recombination mechanism.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Thus, we sought to produce and purify an active recombinant integrase from a class 1 integron previously isolated from a clinical strain of <em>Pseudomonas aeruginosa</em> <a href="#pone.0001315-Dubois1">[18]</a> see <a href="#pone-0001315-g001">figure 1</a>. Then we set up an <em>in vitro</em> recombination assay to characterize its biochemical properties. Using this new assay, we show that IntI1 possesses an <em>in vitro</em> recombination activity on both <em>attI1</em> and <em>attC</em> but with different efficiencies, consistent with its differential affinity for each DNA element. This new <em>in vitro</em> assay of IntI1 recombination activity allows further functional analysis of the protein.</p>
<div class="figure" id="pone-0001315-g001"><div class="img"><a name="pone-0001315-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Schematic representation of recombinant plasmid pC23 part structure encoding the class 1 integron in <em>P. aeruginosa</em> Pa695 (adapted from Dubois et al., 2002, accession number AF355189).</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>The horizontal arrows indicate the translation orientation of the genes. The conserved core and inverse core sites are underlined and the cassette boundaries are represented by vertical arrows. The black arrowheads indicate the different primers described in <a href="#s4">materials and methods</a> used for cloning the IntI1 gene and the different recombination and excision substrates: A: IntI1-3′-stop, B: IntI1-5′-Topo, C: attI1-LBamH1, D: attI1-RHindIII, E: C12T3bis and F: 3′CS.</p>
<span>doi:10.1371/journal.pone.0001315.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Expression, <em>in vivo</em> activity and purification of the recombinant IntI1 enzyme</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The IntI1(his)<sub>6</sub> recombinant protein was expressed from pET101D-Topo vector containing the complete gene encoding the <em>P. aeruginosa</em> IntI1 class 1 integron cloned as described in <a href="#s4">materials and methods</a>. In the resulting pET101D-IntI1 vector, the IntI1 open reading frame was expressed from T7 promoter and fused to a poly(his)6 C-terminal tag. The activity of the fused enzyme was first checked by <em>in vivo</em> excision and recombination assays. In the presence of plasmid pSf2032 carrying an integrase-defective class 2 integron (whose <em>attI2</em> sequences were shown to be recognized by IntI1), the recombinant integrase was shown to be active for excision activity (40% of cassettes lost). Moreover, in the presence of pSf2032 and pACYC184 containing the specific <em>attI1</em> recombination sequence, recombination was observed at a rate of 4.4×10<sup>5</sup>. No excision or recombination events were detected in the absence of pET101D-IntI1 vectors. These results were consistent with previously described recombination rates <a href="#pone.0001315-Collis4">[19]</a> and demonstrated that the integrase IntI1 fused to the (his)6 tag was functional <em>in vivo</em> for all the activities expected of bacterial recombinase in cells. Importantly, the (his)<sub>6</sub> tag did not interfere significantly with the catalysis, easily allowing us to purify an active enzyme and further characterize its <em>in vitro</em> properties.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To obtain a sufficient quantity for enzyme purification, overexpression of the IntI1 protein was performed in the BL21 <em>E. coli</em> bacterial strain at 25°C for 4 hours after 1 mM IPTG induction. At higher temperature, most of the protein remained in the insoluble fraction, reflecting the high insolubility of the protein previously observed <a href="#pone.0001315-Collis2">[9]</a>, <a href="#pone.0001315-Francia2">[16]</a>. Extraction in the presence of 500 mM NaCl and 0.25% Triton X-100 allowed us to obtain a highly soluble enzyme. The soluble fraction was used for nickel-affinity chromatography purification. As shown in <a href="#pone-0001315-g002">figure 2A</a>, a protein displaying a good level of purity was obtained in the 250–350 mM imidazole fractions. The major protein band of 40 kDa apparent molecular weight reacted with anti-His monoclonal antibodies, thereby confirming its nature (<a href="#pone-0001315-g002">figure 2B</a>).</p>
<div class="figure" id="pone-0001315-g002"><div class="img"><a name="pone-0001315-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>SDS-PAGE (A) and western blot (B) analysis of protein factions containing IntI1(his)<sub>6</sub>.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Lane M: molecular weight markers in kDa; lane 1: soluble crude extract from <em>E. coli</em> DH5α expressing IntI1(his)<sub>6</sub>; lane 2: non-retained fraction; lanes 3, 4, 5 and 6: fractions obtained after elution with respectively 20, 100, 250 and 350 mM imidazole. Western blot was performed using anti-(his)<sub>6</sub>Ct antibodies (INVITROGEN).</p>
<span>doi:10.1371/journal.pone.0001315.g002</span></div>

<h4><em>In vitro</em> binding of the recombinant IntI1 enzyme to <em>attI</em> and <em>attC</em> sites</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To investigate the ability of IntI1 to interact with the target sites <em>attI1 and attC</em>, standard gel mobility shift assays were performed using two radiolabeled fragments containing either the double-stranded <em>attI1</em> or the <em>attC</em> site (respectively <em>attI1ds</em> and <em>attCds</em>). As shown in <a href="#pone-0001315-g003">figure 3A</a>, the mobility of the DNA fragment carrying the <em>attI1ds</em> site was lowered in the presence of IntI1.</p>
<div class="figure" id="pone-0001315-g003"><div class="img"><a name="pone-0001315-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span><em>In vitro</em> DNA binding of IntI1 with free double-stranded <em>attI1</em> (A) and <em>attC</em> (B) recombination sites.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Free 5′ <sup>32</sup>P radiolabeled dsDNA fragments containing recombination sites (0.1 pmoles) were incubated with purified IntI1 (1–10 pmoles) at 4°C for 20 min before electrophoresis on 1% agarose gel run at 50 V, for 2 hours at 4°C. Arrows indicate the protein-DNA complexes and F corresponds to free recombination sites.</p>
<span>doi:10.1371/journal.pone.0001315.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The proportion of the bound substrate was dependent on the IntI1concentration. The IntI1-DNA complexes observed were consistent with those previously described using other recombinant enzymes such as MBP-IntI1 and FLAG-IntI1 <a href="#pone.0001315-Collis2">[9]</a>, <a href="#pone.0001315-Gravel1">[10]</a>. Even if the same IntI1-DNA complexes were detected, the intensity of the corresponding bands was different from that previously described. We assumed that this difference of affinity could be due to the different enzyme constructions used as well as differences in the DNA binding assay conditions (concentration of substrate and enzyme). In all cases the most intense band was assumed to be constituted by the target DNA and a single molecule of IntI1 and the other complexes could involve the interaction of several molecules of IntI1.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>In contrast to this, the <em>attCds</em> fragment led to discrete complexes only at high enzyme concentrations (<a href="#pone-0001315-g003">figure 3B</a>). Again, the main species appeared to be the complex between one molecule of IntI1 and the DNA fragment. This suggests that the binding of IntI1 to the <em>attCds</em> element was weaker than to <em>attI1ds</em>, as previously reported <a href="#pone.0001315-Collis2">[9]</a>, <a href="#pone.0001315-Gravel1">[10]</a>, <a href="#pone.0001315-Francia2">[16]</a>. These DNA binding data also suggest, as previously proposed, that IntI1 could bind to DNA forms different from the standard double helix. This hypothesis has been confirmed by several studies describing the specific interaction of the enzyme with ssDNA derived from the <em>attC</em> fragment <a href="#pone.0001315-Gravel1">[10]</a>, <a href="#pone.0001315-MacDonald1">[15]</a>, <a href="#pone.0001315-Francia2">[16]</a>. To determine whether our recombinant IntI1 share the same property, we performed gel retardation assays using single-stranded (ss) <em>attI1</em> and <em>attC</em>. As shown in <a href="#pone-0001315-g004">figure 4A</a>, IntI1 specifically retarded the bottom strand of <em>attC</em> (<em>attCbs</em>) but not the corresponding top strand (<em>attCts</em>). In contrast, only a low retardation rate was observed with the single-stranded <em>attI1</em> bottom fragments (<a href="#pone-0001315-g004">figure 4B</a>).</p>
<div class="figure" id="pone-0001315-g004"><div class="img"><a name="pone-0001315-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span><em>In vitro</em> DNA binding of IntI1 with free single-stranded <em>attC</em> (A) and <em>attI</em> (B) recombination sites.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Free 5′ <sup>32</sup>P radiolabeled ssDNA fragments containing recombination sites (0.1 pmoles) were incubated with purified IntI1 (5–10 pmoles) at 4°C for 20 min before electrophoresis on 1% agarose gel run at 50 V, for 2 hours at 4°C. Arrows indicate the protein-DNA complexes and F corresponds to free recombination sites.</p>
<span>doi:10.1371/journal.pone.0001315.g004</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Further determination of the IntI1 affinity for the recombination substrates used in this study was performed by quantitative filter binding assay. The data in <a href="#pone-0001315-g005">figure 5</a> confirm the affinity of the recombinant enzyme for double-stranded <em>attI</em> and bottom single-stranded <em>attC</em> fragments. They also indicate that at high protein concentrations (above 500 nM, corresponding to 10 pmoles per assay), some unspecific binding to double-stranded <em>attC</em> and single-stranded <em>attI</em> fragments was observed (unspecific binding was also observed under these concentration conditions when an unspecific random 100 bp ODN was used, data not shown). This may indicate that at high concentration the enzyme is able to aggregate on DNA, leading to unspecific complexes.</p>
<div class="figure" id="pone-0001315-g005"><div class="img"><a name="pone-0001315-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Comparison of IntI1 1 affinity for double-stranded (A) and single-stranded (B) recombination sites.</span></strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>Filter binding assays were performed as described in <a href="#s4">materials and methods</a> section using either double-stranded <em>attI</em> (<em>attI ds</em>) and <em>attC</em> (<em>attC ds</em>) either top (top) or bottom (bot) strand of <em>attI</em> and <em>attC.</em> Percentages of substrate retained on filters are shown. Values are the mean±standard deviation (error bars) of three independent experiments.</p>
<span>doi:10.1371/journal.pone.0001315.g005</span></div><a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Taken together, these results indicate that IntI1 can bind to <em>attI1</em> in a double-stranded form and to <em>attC</em> in its single-stranded form, with a bottom strand specificity as previously observed using other recombinant integrases <a href="#pone.0001315-Francia2">[16]</a>. In addition, all these data indicate that our recombinant IntI1(his)<sub>6</sub> shares DNA binding properties similar to those previously described with other purified proteins. From these results we decided to study the biochemical properties of our enzyme further by setting up <em>in vitro</em> recombination assays.</p>


<h4><em>In vitro</em> intermolecular strand transfer between <em>attI1</em> and <em>attC</em> sites catalyzed by the recombinant IntI1 enzyme</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>It has previously been reported that IntI1 can catalyze <em>in vivo</em> recombination between two <em>attI1</em>, two <em>attC</em>, or one <em>attI1</em> and one <em>attC</em>, although with varying efficiencies. In the case of class 1 integron recombinase, few data on the reaction mechanism are available due to the difficulty of obtaining a pure IntI1 that is active enough for an <em>in vitro</em> recombination assay to be performed. The lack of an easy-to-use <em>in vitro</em> system reproducing the recombination reaction further limits characterization of the catalytic mechanism and also the search for potential inhibitors. For these reasons we focused our attention on setting up an efficient <em>in vitro</em> IntI1 activity test.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The recombinant IntI1 was incubated with the radiolabeled donor substrate (either <em>attIds</em> or <em>attCds</em>) and the acceptor vector mimicking a circularized cassette (pGEM-T-<em>attI1</em> or pGEM-T-<em>attC</em>).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Products were detected only after proteinase K treatment of the reaction fractions, consistent with the fact that tyrosine recombinases are known to bind very tightly with DNA which is released from complexes after proteinase K digestion. After treatment a band migrating to the position expected for the recombinant product (3200 bp) was observed with both substrates, as shown in <a href="#pone-0001315-g006">figure 6A</a>. In addition to the expected product, other wider bands were also observed and were assumed to be intermediary recombination structures previously detected in recombination reactions catalyzed by other recombinases <a href="#pone.0001315-Segall1">[20]</a>, <a href="#pone.0001315-Klemm1">[21]</a>.</p>
<div class="figure" id="pone-0001315-g006"><div class="img"><a name="pone-0001315-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span><em>In vitro</em> recombination catalyzed by IntI1 at <em>attI1</em> and <em>attC</em> sites.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Reactions were performed for 90 min in the presence of purified IntI1 (5 or 10 pmoles), 0.1 to 0.2 pmoles of either pGEM-T-<em>attI1</em> or pGEM-T-<em>attC</em> (p<em>attI1</em> and p<em>attC</em> in the figure) and 0.1 pmoles free 5′ <sup>32</sup>P radiolabeled recombination sites under standard conditions described in <a href="#s4">materials and methods</a>. Products were loaded on 1% agarose gel and autoradiographied (A). F: free recombination sites, RP: recombination products. The recombination products were quantified and the percentage of recombination versus the amount of IntI in pmoles was plotted (B).</p>
<span>doi:10.1371/journal.pone.0001315.g006</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Quantification of the reactions products showed that reaction efficiencies varied from one to another experiments, the best recombination being obtained between two <em>attI1</em> elements compared to recombination involving only one <em>attC</em> fragment (see <a href="#pone-0001315-g006">figure 6B</a>). This result displays some differences with the <em>in vivo</em> published data, where recombination involving two <em>attI1</em> was less efficient than the reaction involving <em>attC</em>, but is consistent with the <em>in vitro</em> DNA binding property of recombinant IntI1. Higher concentrations of protein (above 500 nM, corresponding to 10 pmoles of IntI1 per assay) led to the inhibition of recombination activity. This is also consistent with the results of the filter binding assay shown in <a href="#pone-0001315-g005">figure 5</a>, since under those concentrations IntI1 bound DNA unspecifically, probably leading to inactive complexes on the recombination sites. Recombination was observed between both double-stranded <em>attC</em> despite the low affinity of the enzyme for the oligonucleotide. However, in this case, a higher amount of enzyme was required (10 pmoles) compatible with the DNA binding results shown in <a href="#pone-0001315-g004">figure 4B</a> and <a href="#pone-0001315-g005">5A</a>.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>To better ascertain the specificity of the <em>in vitro</em> recombination reaction catalyzed by IntI1, two mutated enzymes containing amino acid substitutions R146K and R280E were assayed. Those two invariant residues were demonstrated to be involved in the <em>in vivo</em> recombination activity. Since the two mutants were previously shown to be inactive for <em>in vivo</em> recombination activity <a href="#pone.0001315-Collis4">[19]</a>, we analyzed their <em>in vitro</em> properties. DNA binding experiments (data not reported here) showed that only the R146K mutant presented an <em>in vitro</em> DNA binding property in the presence of <em>attI1</em> but not in the presence of the <em>attC</em> element. The second R280E mutant showed no affinity at all for <em>attI1</em> or <em>attC</em> as previously described <a href="#pone.0001315-Gravel2">[22]</a>. The <em>in vitro</em> activities of both mutants were then assayed for both <em>attI</em> x <em>attI</em> and <em>attI</em> x <em>attC</em> recombinations. As shown in <a href="#pone-0001315-g007">figure 7</a>, no activity was detected in either mutant. These results demonstrate firstly that the two amino acids R146 and R280 are required for recombination catalysis, and that the reaction observed with wild type IntI1 was due to the intrinsic catalytic capability of the recombinant integrase.</p>
<div class="figure" id="pone-0001315-g007"><div class="img"><a name="pone-0001315-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span><em>In vitro</em> recombination catalyzed by wild type, R146E and R286K mutated IntI1.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Reactions were performed for 90 min in the presence of purified enzyme (5 pmoles), 0.1 pmoles of linear radiolabeled recombination sites <em>attI1</em> or <em>attC</em> and 0.1 pmoles of pGEM-T-<em>attI1</em> under standard conditions described in <a href="#s4">materials and methods</a> section. Products were loaded on 1% agarose gel and autoradiographied. F: free recombination sites, RP: recombination products.</p>
<span>doi:10.1371/journal.pone.0001315.g007</span></div>

<h4><em>In vitro att</em>C recombination involves the bottom single strand of the recombination fragment</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>As reported above (<a href="#pone-0001315-g006">figure 6</a>) recombination activity was detected even in the presence of double-stranded <em>attC</em>. Since it has been proposed that IntI1 recombination could involve the bottom single strand of the <em>attC</em> site, we further analyzed the <em>in vitro</em> activity of IntI1 using single-stranded ODNs. <a href="#pone-0001315-g008">Figure 8</a> shows that the only recombination product detected with the single-stranded substrate was in the presence of the bottom strand of <em>attC.</em> Therefore, the reaction involving the single-stranded bottom strand of <em>attC</em> was the most effective under our conditions. No recombination products were detected in the presence of single-stranded <em>attI</em>, strongly suggesting that these events do not share the same mechanism as <em>attC</em> recombination. These results are in agreement with previous reports <a href="#pone.0001315-Bouvier1">[14]</a> and confirm that <em>attC</em> recombination requires the bottom <em>attC</em> strand.</p>
<div class="figure" id="pone-0001315-g008"><div class="img"><a name="pone-0001315-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span><em>In vitro</em> recombination catalyzed by wild type IntI1 in presence of double- or single-stranded substrates.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Reactions were performed for 90 min in the presence of purified enzyme (5 pmoles), 0.1 pmoles of linear radiolabeled double-stranded (ds) or single-stranded (bottom strand: ss bot, or top strand: ss top) recombination sites <em>attI1</em> or <em>attC</em> and 0.1 pmoles of pGEM-T-<em>attI1</em> or pGEM-T-<em>attC</em> under standard conditions described in <a href="#s4">materials and methods</a> section. Products were loaded on 1% agarose gel and autoradiographied. F: free recombination sites, RP: recombination products.</p>
<span>doi:10.1371/journal.pone.0001315.g008</span></div>

<h4>Biochemical parameters of the <em>in vitro</em> recombination reaction</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To optimize the reaction conditions, we tested the requirement of the enzyme for cations (Mg<sup>++</sup> and Mn<sup>++</sup>) and NaCl. As shown in <a href="#pone-0001315-g009">figure 9A</a>, IntI1 can use both cations for recombination with an optimum at 7.5 mM Mg<sup>++</sup>. Importantly, a basal recombination activity was detected under our conditions even in the absence of cations in the reaction medium. The NaCl concentration was found to affect dramatically the recombination activity of IntI1 (<a href="#pone-0001315-g009">figure 9B</a>). A 125 mM concentration was required for optimal activity and the recombination reaction was inhibited at higher salt concentrations. We also investigated whether the addition of small amounts of detergent could improve the activity. Three molecules were tested: Np40, Tween 20 and Triton X-100. <a href="#pone-0001315-g009">Figure 9C</a> shows that all of them inhibited the recombination reaction at different levels, as recently reported for the yeast FLP recombinase belonging to the same tyrosine recombinase family <a href="#pone.0001315-Jahnz1">[23]</a>.</p>
<div class="figure" id="pone-0001315-g009"><div class="img"><a name="pone-0001315-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001315" data-uri="info:doi/10.1371/journal.pone.0001315.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001315.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001315.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001315.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001315.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Effect of cations (A), salt (B) and detergent (C) on <em>in vitro</em> recombination catalyzed by IntI1.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Recombination reactions were performed in the presence of 2 pmoles purified IntI1, 0.1 pmoles free <em>attI1<sup>*</sup></em> and 0.1 pmoles pGEM-T-<em>attI1</em> and different concentrations of MgCl<sub>2</sub>, MnCl<sub>2</sub> NaCl and detergent as indicated. Recombination products were quantified with DNAJ software and are shown on the graphs as the percentage of recombinant product versus the total substrate.</p>
<span>doi:10.1371/journal.pone.0001315.g009</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Gene cassette mobility in integrons in bacteria is a principal factor in the spread of antibiotic resistance, thereby reducing the efficiency of long-term antibiotherapy. IntI1 has previously been put forward as the enzyme responsible for this mobility. The availability of a pure <em>in vitro</em> active integrase is the prerequisite for detailed examination of the recombination reactions catalyzed by this enzyme. The purpose of our work was to set up an <em>in vitro</em> assay allowing the detection of junction molecules from the recombination activity catalyzed by IntI1 on <em>att</em> fragments and thus the further characterization of its biochemical properties.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>IntI1 was expressed and purified as a protein fused to (his)<sub>6 </sub>tag in C-terminal. The methodology presented here, in particular the use of a high salt concentration, allowed us to overcome the main obstacle to integrase purification, i.e. its very high insolubility <a href="#pone.0001315-Collis2">[9]</a>, <a href="#pone.0001315-Francia2">[16]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p><em>In vitro</em> DNA binding analysis using free double-stranded <em>attI1</em> and <em>attC</em> fragments indicated that IntI1 shared the same property as previously described purified enzymes such as native integrase, MBP-IntI1, Flag-IntI1 <a href="#pone.0001315-Collis2">[9]</a>, <a href="#pone.0001315-Gravel1">[10]</a>. In particular, IntI1 bound <em>in vitro</em> to double-stranded <em>attI1</em> with a higher efficiency than to double-stranded <em>attC</em> as previously reported <a href="#pone.0001315-Collis2">[9]</a>, <a href="#pone.0001315-Gravel1">[10]</a>, <a href="#pone.0001315-Francia2">[16]</a>. In contrast, IntI1 showed a better affinity for the bottom strand of <em>attC</em> than its opposite top strand or single strands derived from <em>attI</em>. These results confirmed the previously reported data indicating that the enzyme recognizes a preferred ssDNA structure in the <em>attC</em> recombination site. Some unspecific binding was also observed at high protein concentration, suggesting the formation of aggregates on DNA. To better determine the relationship between DNA affinity of IntI1 and its recombination activity, we used the <em>in vitro</em> assays developed in this study.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Intermolecular recombination reactions using free <em>attI1</em> and <em>attC</em> target sites showed purified IntI1 could catalyze at least a strand transfer <em>in vitro</em>. This result demonstrates that IntI1 is necessary and sufficient for basal recombination between <em>attI</em> or <em>attC</em> sites in class 1 integron. Indeed, IntI1 has been previously put forward as the enzyme responsible for the movement of gene cassettes in and out of integrons, but no data suggested conclusively that it was the sole protein involved in cellular recombination events. Other proteins belonging to the same family need cellular factors to perform their activity efficiently. For example, excision of lambda prophage from the bacterial chromosome requires the host-encoded integration factor (IHF) and the lambda-encoded excisionase in addition to integrase and both integration and excision are stimulated by the host-encoded factor for inversion stimulation (FIS) protein <a href="#pone.0001315-Ball1">[24]</a>. Our data provide the first evidence that IntI1 possesses all the catalytic activities needed to perform the first strand transfer step between recombination sites.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>However, differences were observed in recombination efficiency depending on the target DNA sites. In general, <em>in vitro</em> recombination between two <em>attI1</em> was preferred over recombination between <em>attI1</em> and <em>attC</em> sites, itself more efficient than between two <em>attC</em>. This result is in agreement with the <em>in vitro</em> DNA binding property of the recombinant IntI1 and can be explained by a lower <em>in vitro</em> affinity of IntI1 with the <em>attC</em> element compared to <em>attI</em>. This suggests a preference for the specific structure found in <em>attI1</em>. Our results differ somewhat from those reported in the literature. Indeed, <em>in vivo</em> recombination events between two <em>attI1</em> and <em>attC</em> sites are more efficient than those involving two <em>attC, which</em> themselves are more efficient than <em>attI</em> x <em>attI</em> recombination <a href="#pone.0001315-Collis3">[13]</a>. Differences observed between <em>in vivo</em> and <em>in vitro</em> suggest that the integration mechanism in the cell might be regulated, favoring the recombination between <em>attI1</em> and <em>attC</em>. Under our conditions we did not observe <em>in vitro</em> integration at secondary sites when using the receptor plasmid lacking <em>att</em> sites in the recombination assay (data not shown), thus confirming the low frequency of those events.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Biochemical analysis of the recombination reaction catalyzed by IntI1 showed that the enzyme could perform its activity in the absence of bivalent cations added to the reaction mixture at a basal level but preferring a 7.5 mM Mg<sup>++</sup> concentration (see <a href="#pone-0001315-g007">figure 7</a>). Although there is disagreement regarding the effect of ions on recombination, there is a consensus that divalent ions are required for the intermolecular DNA exchange reaction <a href="#pone.0001315-Babineau1">[25]</a>–<a href="#pone.0001315-Ringrose1">[27]</a>. Our observation raises the question whether cations are coordinated in the integrase structure and play a role in the reaction. In addition, we showed that IntI1 activity was highly sensitive to detergents and, at a lower level, to salt. All the tested detergents (Np40, Tween 20 and Triton X-100) inhibited the recombination reaction, as recently reported for other tyrosine recombinase <a href="#pone.0001315-Jahnz1">[23]</a>. This indicates the possible involvement of oligomeric forms of the enzyme in the reaction, as previously demonstrated for enzymes belonging to the same family <a href="#pone.0001315-Biswas1">[28]</a>. Such active oligomers could be dissociated by detergent and high salt concentration inhibiting the recombination reaction. This assumption is reinforced by the DNA protein complexes observed between IntI1 and the free purified recombination sites by us and others <a href="#pone.0001315-Collis2">[9]</a>, <a href="#pone.0001315-Francia2">[16]</a>. Nevertheless, we cannot rule out a possible dissociation between IntI1 and recombination sites since salt can also weaken such interactions.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The <em>in vitro</em> recombination data indicate that <em>attI1</em> and <em>attC</em> sites do not interact equally with IntI1, thereby confirming that there are different mechanisms for recombination depending on the sites involved. The difference in affinity of the enzyme for <em>attI1</em> and <em>attC</em> and the preference of IntI1 for single-stranded <em>attC</em> previously reported <a href="#pone.0001315-Francia2">[16]</a> and also observed by us strongly support the involvement of a single-strand DNA intermediary in the reaction, as recently proposed <a href="#pone.0001315-Bouvier1">[14]</a>. To better ascertain this requirement, we performed assays with single-stranded substrates. Our results indicate that <em>in vitro attC</em> recombination requires the bottom strand of the site in contrast to <em>attI</em> recombination. Our data unambiguously demonstrate that <em>attC</em> and <em>attI</em> recombination mechanisms do not share the same process.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Nevertheless, in all cases only a maximum of about 15–18% of the donor substrate was integrated into the acceptor DNA, indicating that the <em>in vitro</em> recombination still had a low level of efficiency. Whether this efficiency reflects that of <em>in vivo</em> recombination remains to be established. This low level of recombinative integration could be due to the fact that two independent molecules are involved in this <em>in vitro</em> reaction. Furthermore, the DNA fragments used in our assay do not share the exact structure of the total integron where intermolecular recombination takes place. In addition, the differential affinity of IntI1 for <em>attI1</em> and <em>attC</em> free fragments previously observed could explain the variation in recombination efficiency.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>How can our <em>in vitro</em> recombination data be reconciled with the specificity of IntI1 for single-stranded <em>attC</em>? The recombination activity observed in presence of the bottom strand of <em>attC</em> confirms that this single-stranded structure is an important intermediary in the reaction, as previously reported <a href="#pone.0001315-Bouvier1">[14]</a>, <a href="#pone.0001315-MacDonald1">[15]</a>. Moreover, an <em>in vitro</em> recombination activity between double-stranded <em>attI</em> and <em>attC</em> and between two double-stranded <em>attC</em> could also be detected, implying that the enzyme was able to generate and/or recognize the single-stranded structure in the <em>attC</em> site, even in the absence of other bacterial factors usually involved in this kind of mechanism such as helicase. Recently it has been shown that integron integrase binds to bulged hairpin DNA found at the <em>attC</em> site <a href="#pone.0001315-Johansson1">[17]</a>. These cruciform structures could be generated <em>in vivo</em> by a cellular mechanism such as DNA replication and transcription and then stabilized by IntI1 for recombination. The capacity of the enzyme to generate the single strand by itself should allow the recombination to be effected independently of the replication processes. However, in standard helicase assays performed with our pure fractions of IntI1, no activity was shown (data not shown). This suggests that generation of the single-stranded <em>attC</em> by the enzyme acts via a different way, as suggested by the structural data <a href="#pone.0001315-MacDonald1">[15]</a>, probably by strong interaction between IntI1 and the extrahelical bases in the recombination site, leading to opening of the strands.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>By dissecting the reaction, the recombination assays described will allow further characterization of this activity. In particular, the recombination test could serve as a basis for the selection and the study of inhibitory molecules as well as for the analysis of reaction selectivity using chimeric proteins. These could be useful tools in gene therapy. The results obtained with the mutated enzyme also indicate that the <em>in vitro</em> assays described here make it possible to study the relationship between the structure and function of IntI1 through analysis of the biochemical properties of the mutation carrying integrases in amino acids, which is potentially involved in recombination. In addition to such fundamental studies the <em>in vitro</em> assay described here will allow the search and the selection of specific inhibitors of IntI1 that could be useful for the <em>in vivo</em> limitation of the antibiotic resistance spread.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>DNA, bacterial strains and culture media</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The DNA sequence encoding the entire class I integron was previously cloned from <em>P. aeruginosa</em> Pa695 <a href="#pone.0001315-Dubois1">[18]</a> into pC23 vector (see <a href="#pone-0001315-g001">figure 1</a>). The natural conjugative plasmid pSf2032 which contains a class 2 integron carrying trimethoprime (cassette 1), streptothricin (cassette 2) and streptomycin-spectinomycin (cassette 3, silent gene) resistance genes was obtained from the <em>Shigella flexneri</em> Sf2032 clinical strain. The pACYC184-<em>attI1</em> vector, carrying a chloramphenicol resistance determinant and an <em>attI1</em> site, was obtained by ligating the PCR amplification product, on one hand, obtained with pC23 as template and attI1L-BamH1 (5′-GGATCCCAAGCAGCAAGCGCGTTAT-3′) and attI1R-HindIII (5′-AAGCTTGCAATGATGCTCATTGAGCC-3′) as primers, cut with <em>Bam</em>HI and <em>Hind</em>III and, on the other, pACYC184 with the same enzymes.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The <em>E. coli</em> DH5α strain was used for propagation of plasmids and <em>E. coli</em> BL21 strain was used for expression of IntI1(his)<sub>6</sub> recombinant enzyme. <em>E. coli</em> TOP10 was the recipient strain in <em>in vivo</em> recombination assays. All bacterial strains were cultured at 37°C for propagation of the plasmid or 25°C for expression, on LB medium supplemented with antibiotic.</p>


<h4>DNA manipulation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>All DNA vectors and PCR products were purified using the DNA purification systems from PROMEGA (Wizard plus SV miniprep and Wizard SV Gel kits). PCR amplifications were done under standard conditions using <em>Taq</em> polymerase (PROMEGA). Sequencing was performed by polymerase chain reaction-based sequencing (ABI Prism big dye terminator cycle sequencing ready reaction kit, Applied Biosystems).</p>
<h5>Cloning, expression and purification of IntI1.</h5><a id="article1.body1.sec4.sec2.sec1.p1" name="article1.body1.sec4.sec2.sec1.p1"></a><p>Wild type <em>IntI1</em> gene was amplified by PCR from pC23 vector using IntI1-5′-Topo primer containing the Topo1 site (5′-CACCATGAAAACCGCCACTGCGCCG-3′) and IntI1-3′-stop primer (5′-CCTCTCACTAGTGAGGGGCGGCAG-3′). The amplification product was then cloned into pET101D-Topo vector according to the manufacturer's recommendations (INVITROGEN). The resulting pET101D-IntI1 expression plasmid was checked by sequencing and introduced into the BL21 bacterial strain for expression of recombinant IntI1 fused to (his)6 tag at the 3′-end. The genes encoding the mutated R146K and R280E enzyme were amplified as for wild type enzyme but from pMalC2 plasmid kindly provided by Drs. P. Roy and N. Messier <a href="#pone.0001315-Segall1">[20]</a>. Both mutants were further cloned into pET101D vector as done for wild type enzyme.</p>
<a id="article1.body1.sec4.sec2.sec1.p2" name="article1.body1.sec4.sec2.sec1.p2"></a><p>Bacteria containing the pET101-IntI1 vector were cultured in LB medium supplemented with ampicillin (50 µg/ml) for 12 hours at 37°C with shaking. Cultures were then diluted at 1/60 in fresh LB-ampicillin (50 µg/ml) and cultured for 4 more hours to reach an OD<sub>600 nm</sub> of 0.4. Expression of IntI1 was induced by addition of IPTG to 1 mM for 4 hours at 25°C, since at 37°C a major proportion of the enzyme remains insoluble. Cells were harvested by centrifugation at 4000 rpm for 15 minutes and pellets were resuspended in 5 ml of lysis buffer I (50 mM NaH<sub>2</sub>PO<sub>4</sub> pH 7.5) containing a cocktail of proteases inhibitors (Complete Mini EDTA free, ROCHE) and 50 µg/ml of lysozyme. After 20 min incubation at room temperature, the suspension was centrifuged for 15 minutes at 4000 rpm. Pellets were resuspended in 5 ml of lysis buffer II (50 mM NaH<sub>2</sub>PO<sub>4</sub> pH 7.5, 500 mM NaCl, 1 mM DTT, 0.025% Triton X-100). The suspension was incubated for 15 min at 4°C, and then sonicated. The cell lysate was centrifuged at 18000 rpm for 20 min at 4°C and the supernatant was used for SDS-PAGE analysis and purification. With the procedure described above, the majority of IntI1 could be recovered in a soluble and thus suitable form for purification.</p>
<a id="article1.body1.sec4.sec2.sec1.p3" name="article1.body1.sec4.sec2.sec1.p3"></a><p>A 600 µl aliquot of the soluble fraction obtained as described above was loaded on a Ni-NTA Spin Column according to the supplier's instructions (QIAGEN). The column was washed twice with the elution buffer (50 mM NaH<sub>2</sub>PO<sub>4</sub> pH 7.5, 500 mM NaCl, 0.025 Triton X-100) containing 20 mM imidazole. Several elution steps were then performed from 20 mM to 400 mM imidazole. The fractions were analyzed by SDS-PAGE after staining with blue Coomassie and western blot using anti-(His)<sub>6</sub>Ct antibodies (INVITROGEN). The fractions containing IntI1 were dialyzed against 50 mM Na<sub>2</sub>HPO<sub>4</sub>/NaH<sub>2</sub>PO<sub>4</sub> pH 7.5, 500 mM NaCl, 0.025 Triton X-100 solution and then stored at −20°C after addition of 10% glycerol. Protein concentrations were determined by the standard Bradford method. Stock concentrations varied from 2.5 to 5 µM.</p>

<h5><em>In vivo</em> recombination assay.</h5><a id="article1.body1.sec4.sec2.sec2.p1" name="article1.body1.sec4.sec2.sec2.p1"></a><p>An <em>in vivo</em> excision assay was performed using DH5α cells containing the pSF2032 and pET101D-IntI1 plasmids. The strain was cultured in 5 ml LB medium supplemented with ampicillin (100 µg/ml ) and trimethroprime (20 µg/ml) for 4 hours at 37°C under shaking. Then, 1 mM IPTG was added, and after 3 and 24 hours of culture, 10 µl aliquots were plated on LB solid medium containing ampicillin (100 µg/ml) and trimethoprime (20 mg/ml). After 18 hours of incubation at 37°C, 10 clones were recovered and analyzed by PCR using primers int2inv (5′-AACCTTTTTGTCGCATATCCGTG-3′), dfrA1R (5′-GTTAGAGGCGAAGTCTTGGG-3′), satinv (5′-TTAGGCGTCATCCTGTGCTC-3′), and aad2di (5′-CAGGAACCGGATCAAAGAG-3′) to determine the number of gene cassettes in the integron. A DH5α strain carrying only pSf2032 served as control.</p>
<a id="article1.body1.sec4.sec2.sec2.p2" name="article1.body1.sec4.sec2.sec2.p2"></a><p>An <em>in vivo</em> recombination assay was carried out using DH5α cells harboring the pSf2032, pACYC184-<em>attI1</em> and pET101D-IntI1 vectors as the donor strain, and TOP10 as the recipient strain in a filter matting assay. Both strains were cultured in LB medium supplemented or not with antibiotics (100 µg/ml ampicillin, 20 µg/ml trimethoprime and 30 µg/ml chloramphenicol) for 3 hours at 37°C under shaking, and then for 3 more hours after addition of 1 mM IPTG to induce IntI1 expression. Cultures of each strain were concentrated by centrifugation, and 1 ml aliquots were loaded on Millipore GS 0.22 µm filter, then plated on LB medium. After 16 hours incubation at 37°C, transconjugants were selected on LB medium supplemented with trimethoprime (20 µg/ml), streptomycin (25 µg/ml) and chloramphenicol (30 µg/ml), or with trimethoprime and streptomycin only. Donor cells were numbered to determine the transfer frequency and recombination rate was calculated as the ratio between the number of transconjugants containing the recombinant plasmid versus the total number of transconjugants.</p>

<h5><em>In vitro</em> strand transfer assays.</h5><a id="article1.body1.sec4.sec2.sec3.p1" name="article1.body1.sec4.sec2.sec3.p1"></a><p>The <em>attI1</em> containing donor substrate (100 bp) was generated by annealing the 5′<sup>32</sup>P radiolabeled oligonucleotide AttI (5′-ACGCCGTGGGTCGATGTTTGATGTTATGGAGCAGCA​ACGATGTTACGCAGCAGGGCAGTCGCCCTAAAACAAAGTT​AGGTGGCTCAATGAGCATCATTGC-3′)with the complementary one AttI'. The <em>attC</em> containing donor substrate (120 bp) was obtained by annealing the 5′<sup>32</sup>P radiolabeled oligonucleotide AttC2 (5′-CGCCCGTCTAACAATTCGTTCAAGCCGACGTTGCTT​CGTGGCGGCGCTTGCGTGCTACGCTAAGCTTCGCACGCCG​CTTGCCACTGCGCACCGCGGCTTAACTCAGGCGTTAGATG​CACT-3′)with the complementary one AttC2′.</p>
<a id="article1.body1.sec4.sec2.sec3.p2" name="article1.body1.sec4.sec2.sec3.p2"></a><p>To construct the receptor plasmids pGEM-T-<em>attI1</em> (called p<em>attI1</em>) and pGEM-T-<em>attC</em> (called p<em>attC</em>), both <em>attI</em> and <em>attC2</em> were generated by PCR using attI1-LBamH1 and attI1-RHindIII as primers and pC23 as template (see <a href="#pone-0001315-g001">figure 1</a>). PCR led to the amplification of the <em>attC</em> site of the second cassette [<em>attC(2)</em> in <a href="#pone-0001315-g001">figure 1</a>]. The receptor plasmids pGEM-T-<em>attI1</em> (called p<em>attI1</em>) and pGEM-T-<em>attC</em> (called p<em>attC</em>) were obtained by cloning the corresponding fragments into pGEM-T easy vector (PROMEGA).</p>
<a id="article1.body1.sec4.sec2.sec3.p3" name="article1.body1.sec4.sec2.sec3.p3"></a><p>To determine its DNA binding activity, purified IntI1(his)<sub>6</sub> (1 to 10 pmoles) was incubated either with the 5′ radiolabeled double-stranded or single-stranded <em>attI1</em> fragment or with the 5′ radiolabeled double-stranded or single-stranded <em>attC</em> fragment for 20 min at 4°C in a total volume of 20 µl. The IntI1-DNA complexes were then loaded on vertical 1% agarose gel and run at 50 V for 4 hours at 4°C. The gel was then dried and autoradiographied. Quantification was performed by filter binding assays: Nitrocellulose filters (0.45 µm, Whatman) were treated with a solution of KOH 0.4 M and washed twice with water and 2 ml of pre-washing buffer (HEPES 20 mM; pH 7.5; MnCl<sub>2 </sub>10 mM; NaCl 10 mM; calf thymus DNA 100 µg/ml). IntI1 was incubated under <em>in vitro</em> assay conditions for 20 min at 4°C with the different radiolabeled substrates. After addition of 1 ml washing buffer (HEPES 20 mM; pH 7.5; MnCl<sub>2</sub> 10 mM; NaCl 30 mM) to the reaction mix, the solution was filtered. Filters were washed twice with 4 ml of washing buffer. The radioactivity retained on filters was quantitated with a scintillation counter (Wallac 1409).</p>
<a id="article1.body1.sec4.sec2.sec3.p4" name="article1.body1.sec4.sec2.sec3.p4"></a><p>The recombination reaction was performed by incubation of the purified IntI1(his)<sub>6</sub> (1 to 10 pmoles) with both donor and receptor substrates (0.1 to 0.2 pmoles) for 20 min at 4°C in a total volume of 5 µl to promote IntI1-DNA complexes. Then, the incubation proceeded at 37°C for 90 min in the presence of 7.5 mM Mg<sup>++</sup>, 50 mM TrisHCl pH 7.5 and 1 mM DTT in a total volume of 20 µl. Reaction fractions were treated by protease K (50 µg/ml) for one hour at 55°C and were then submitted to phenol/chloroform/isoamylalcohol (25/25/1, v/v/v) extraction. The aqueous fraction was loaded on vertical 1% agarose gel and run at 200 V. The gel was dried and autoradiographied.</p>


</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors are deeply grateful to J. Pageze and R. Cooke (English Department, University Victor Segalen Bordeaux 2) for their editorial assistance and P. Roy and N. Messier for providing us the IntI1 mutated genes. We also thank D. Mazel for fruitful discussion.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: VP CD VD SL CQ. Performed the experiments: VP CD VD. Analyzed the data: VP CD VD SL CQ. Contributed reagents/materials/analysis tools: VP VD CQ. Wrote the paper: VP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001315-Hall1" id="pone.0001315-Hall1"></a>Hall RM, Collis CM (1995) Mobile gene cassettes and integrons: capture and spread of genes by site-specific recombination. Mol Microbiol  15: 593–600.  <ul class="find" data-citedArticleID="1030080" data-doi="10.1111/j.1365-2958.1995.tb02368.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1995.tb02368.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mobile+gene+cassettes+and+integrons%3A+capture+and+spread+of+genes+by+site-specific+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mobile+gene+cassettes+and+integrons%3A+capture+and+spread+of+genes+by+site-specific+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001315-Hall2" id="pone.0001315-Hall2"></a>Hall RM, Collis CM, Kim MJ, Partridge SR, Recchia GD, et al.  (1999) Mobile gene cassettes and integrons in evolution. Ann N Y Acad Sci  870:   <ul class="find" data-citedArticleID="1030082" data-doi="10.1111/j.1749-6632.1999.tb08866.x"><li><a href="http://dx.doi.org/10.1111/j.1749-6632.1999.tb08866.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mobile+gene+cassettes+and+integrons+in+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mobile+gene+cassettes+and+integrons+in+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001315-Hall3" id="pone.0001315-Hall3"></a>Hall RM, Brookes DE, Stokes HW (1991) Site-specific insertion of genes into integrons: role of the 59-base element and determination of the recombination cross-over point. Mol Microbiol  5: 1941–1959.  <ul class="find" data-citedArticleID="1030084" data-doi="10.1111/j.1365-2958.1991.tb00817.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1991.tb00817.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Site-specific+insertion+of+genes+into+integrons%3A+role+of+the+59-base+element+and+determination+of+the+recombination+cross-over+point." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Site-specific+insertion+of+genes+into+integrons%3A+role+of+the+59-base+element+and+determination+of+the+recombination+cross-over+point.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001315-Recchia1" id="pone.0001315-Recchia1"></a>Recchia GD, Stokes HW, Hall RM (1994) Characterisation of specific and secondary recombination sites recognised by the integron DNA integrase. Nucleic Acids Res  22: 2071–2078.  <ul class="find" data-citedArticleID="1030098" data-doi="10.1093/nar/22.11.2071"><li><a href="http://dx.doi.org/10.1093/nar/22.11.2071" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterisation+of+specific+and+secondary+recombination+sites+recognised+by+the+integron+DNA+integrase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterisation+of+specific+and+secondary+recombination+sites+recognised+by+the+integron+DNA+integrase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001315-Collis1" id="pone.0001315-Collis1"></a>Collis CM, Hall RM (1995) Expression of antibiotic resistance genes in the integrated cassettes of integrons. Antimicrob Agents Chemother  39: 155–162.  <ul class="find" data-citedArticleID="1030058" data-doi="10.1128/aac.39.1.155"><li><a href="http://dx.doi.org/10.1128/aac.39.1.155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+antibiotic+resistance+genes+in+the+integrated+cassettes+of+integrons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+antibiotic+resistance+genes+in+the+integrated+cassettes+of+integrons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001315-Stokes1" id="pone.0001315-Stokes1"></a>Stokes HW, O'Gorman DB, Recchia GD, Parsekhian M, Hall RM (1997) Structure and function of 59-base element recombination sites associated with mobile gene cassettes. Mol Microbiol  26: 731–745.  <ul class="find" data-citedArticleID="1030104" data-doi="10.1046/j.1365-2958.1997.6091980.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1997.6091980.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+of+59-base+element+recombination+sites+associated+with+mobile+gene+cassettes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+of+59-base+element+recombination+sites+associated+with+mobile+gene+cassettes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001315-Francia1" id="pone.0001315-Francia1"></a>Francia MV, de la Cruz F, Garcia Lobo JM (1993) Secondary-sites for integration mediated by the Tn21 integrase. Mol Microbiol  10: 823–828.  <ul class="find" data-citedArticleID="1030068" data-doi="10.1111/j.1365-2958.1993.tb00952.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1993.tb00952.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Secondary-sites+for+integration+mediated+by+the+Tn21+integrase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Secondary-sites+for+integration+mediated+by+the+Tn21+integrase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001315-Hansson1" id="pone.0001315-Hansson1"></a>Hansson K, Skold O, Sundstrom L (1997) Non-palindromic attl sites of integrons are capable of site-specific recombination with one another and with secondary targets. Mol Microbiol  26: 441–453.  <ul class="find" data-citedArticleID="1030086" data-doi="10.1046/j.1365-2958.1997.5401964.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1997.5401964.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-palindromic+attl+sites+of+integrons+are+capable+of+site-specific+recombination+with+one+another+and+with+secondary+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-palindromic+attl+sites+of+integrons+are+capable+of+site-specific+recombination+with+one+another+and+with+secondary+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001315-Collis2" id="pone.0001315-Collis2"></a>Collis CM, Kim MJ, Stokes HW, Hall RM (1998) Binding of the purified integron DNA integrase Intl1 to integron- and cassette-associated recombination sites. Mol Microbiol  29: 477–490.  <ul class="find" data-citedArticleID="1030060" data-doi="10.1046/j.1365-2958.1998.00936.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.00936.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Binding+of+the+purified+integron+DNA+integrase+Intl1+to+integron-+and+cassette-associated+recombination+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Binding+of+the+purified+integron+DNA+integrase+Intl1+to+integron-+and+cassette-associated+recombination+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001315-Gravel1" id="pone.0001315-Gravel1"></a>Gravel A, Fournier B, Roy PH (1998) DNA complexes obtained with the integron integrase IntI1 at the attI1 site. Nucleic Acids Res  26: 4347–4355.  <ul class="find" data-citedArticleID="1030074" data-doi="10.1093/nar/26.19.4347"><li><a href="http://dx.doi.org/10.1093/nar/26.19.4347" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+complexes+obtained+with+the+integron+integrase+IntI1+at+the+attI1+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+complexes+obtained+with+the+integron+integrase+IntI1+at+the+attI1+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001315-Partridge1" id="pone.0001315-Partridge1"></a>Partridge SR, Recchia GD, Scaramuzzi C, Collis CM, Stokes HW, et al.  (2000) Definition of the attI1 site of class 1 integrons. Microbiology  146: 2855–2864.  <ul class="find" data-citedArticleID="1030096"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Definition+of+the+attI1+site+of+class+1+integrons.&amp;auth=&amp;atitle=Definition+of+the+attI1+site+of+class+1+integrons." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Definition+of+the+attI1+site+of+class+1+integrons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Definition+of+the+attI1+site+of+class+1+integrons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001315-Grainge1" id="pone.0001315-Grainge1"></a>Grainge I, Jayaram M (1999) The integrase family of recombinase: organization and function of the active site. Mol Microbiol  33: 449–456.  <ul class="find" data-citedArticleID="1030072" data-doi="10.1046/j.1365-2958.1999.01493.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01493.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+integrase+family+of+recombinase%3A+organization+and+function+of+the+active+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+integrase+family+of+recombinase%3A+organization+and+function+of+the+active+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001315-Collis3" id="pone.0001315-Collis3"></a>Collis CM, Recchia GD, Kim MJ, Stokes HW, Hall RM (2001) Efficiency of recombination reactions catalyzed by class 1 integron integrase IntI1. J Bacteriol  183: 2535–2542.  <ul class="find" data-citedArticleID="1030062" data-doi="10.1128/jb.183.8.2535-2542.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.8.2535-2542.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficiency+of+recombination+reactions+catalyzed+by+class+1+integron+integrase+IntI1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficiency+of+recombination+reactions+catalyzed+by+class+1+integron+integrase+IntI1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001315-Bouvier1" id="pone.0001315-Bouvier1"></a>Bouvier M, Demarre G, Mazel D (2005) Integron cassette insertion: a recombination process involving a folded single strand substrate. Embo J  24: 4356–4367.  <ul class="find" data-citedArticleID="1030056" data-doi="10.1038/sj.emboj.7600898"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600898" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integron+cassette+insertion%3A+a+recombination+process+involving+a+folded+single+strand+substrate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integron+cassette+insertion%3A+a+recombination+process+involving+a+folded+single+strand+substrate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001315-MacDonald1" id="pone.0001315-MacDonald1"></a>MacDonald D, Demarre G, Bouvier M, Mazel D, Gopaul DN (2006) Structural basis for broad DNA specificity in integron recombination. Nature  27: 1157–1162.  <ul class="find" data-citedArticleID="1030094" data-doi="10.1038/nature04643"><li><a href="http://dx.doi.org/10.1038/nature04643" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+broad+DNA+specificity+in+integron+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+broad+DNA+specificity+in+integron+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001315-Francia2" id="pone.0001315-Francia2"></a>Francia MV, Zabala JC, de la Cruz F, Garcia Lobo JM (1999) The IntI1 integron integrase preferentially binds single-stranded DNA of the attC site. J Bacteriol  181: 6844–6849.  <ul class="find" data-citedArticleID="1030070"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+IntI1+integron+integrase+preferentially+binds+single-stranded+DNA+of+the+attC+site.&amp;auth=&amp;atitle=The+IntI1+integron+integrase+preferentially+binds+single-stranded+DNA+of+the+attC+site." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+IntI1+integron+integrase+preferentially+binds+single-stranded+DNA+of+the+attC+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+IntI1+integron+integrase+preferentially+binds+single-stranded+DNA+of+the+attC+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001315-Johansson1" id="pone.0001315-Johansson1"></a>Johansson C, Kamali-Moghaddam M, Sundstrom L (2004) Integron integrase binds to bulged hairpin DNA. Nucleic Acids Res  32: 4033–4043.  <ul class="find" data-citedArticleID="1030090" data-doi="10.1093/nar/gkh730"><li><a href="http://dx.doi.org/10.1093/nar/gkh730" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integron+integrase+binds+to+bulged+hairpin+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integron+integrase+binds+to+bulged+hairpin+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001315-Dubois1" id="pone.0001315-Dubois1"></a>Dubois V, Poirel L, Marie C, Arpin C, Nordmann P, et al.  (2002) Molecular characterization of a novel class 1 integron containing bla(GES-1) and a fused product of aac3-Ib/aac6'-Ib' gene cassettes in Pseudomonas aeruginosa. Antimicrob Agents Chemother  46: 638–645.  <ul class="find" data-citedArticleID="1030066" data-doi="10.1128/aac.46.3.638-645.2002"><li><a href="http://dx.doi.org/10.1128/aac.46.3.638-645.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+a+novel+class+1+integron+containing+bla%28GES-1%29+and+a+fused+product+of+aac3-Ib%2Faac6%27-Ib%27+gene+cassettes+in+Pseudomonas+aeruginosa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+a+novel+class+1+integron+containing+bla%28GES-1%29+and+a+fused+product+of+aac3-Ib%2Faac6%27-Ib%27+gene+cassettes+in+Pseudomonas+aeruginosa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001315-Collis4" id="pone.0001315-Collis4"></a>Collis CM, Kim MJ, Stokes HW, Hall RM (2002) Integron-encoded IntI integrases preferentially recognize the adjacent cognate attI site in recombination with a 59-be site. Mol Microbiol  46: 1415–1427.  <ul class="find" data-citedArticleID="1030064" data-doi="10.1046/j.1365-2958.2002.03260.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.03260.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integron-encoded+IntI+integrases+preferentially+recognize+the+adjacent+cognate+attI+site+in+recombination+with+a+59-be+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integron-encoded+IntI+integrases+preferentially+recognize+the+adjacent+cognate+attI+site+in+recombination+with+a+59-be+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001315-Segall1" id="pone.0001315-Segall1"></a>Segall AM (1998) Analysis of higher order intermediates and synapsis in the bent-L pathway of bacteriophage lambda site-specific recombination J Biol Chem,  273: 24258–24265.  <ul class="find" data-citedArticleID="1030102" data-doi="10.1074/jbc.273.37.24258"><li><a href="http://dx.doi.org/10.1074/jbc.273.37.24258" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+higher+order+intermediates+and+synapsis+in+the+bent-L+pathway+of+bacteriophage+lambda+site-specific+recombination" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+higher+order+intermediates+and+synapsis+in+the+bent-L+pathway+of+bacteriophage+lambda+site-specific+recombination%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001315-Klemm1" id="pone.0001315-Klemm1"></a>Klemm M, Cheng C, Cassell G, Shuman S, Segall AM (2000) Peptide inhibitors of DNA cleavage by tyrosine recombinase and topoisomerases Journal of Molecular Biology,  299: 1203–1216.  <ul class="find" data-citedArticleID="1030092" data-doi="10.1006/jmbi.2000.3829"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3829" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peptide+inhibitors+of+DNA+cleavage+by+tyrosine+recombinase+and+topoisomerases" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peptide+inhibitors+of+DNA+cleavage+by+tyrosine+recombinase+and+topoisomerases%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001315-Gravel2" id="pone.0001315-Gravel2"></a>Gravel A, Messier N, Roy PH (1998) Point mutations in the integron integrase IntI1 that affect recombination and/or substrate recognition. J Bacteriol  180: 5437–5442.  <ul class="find" data-citedArticleID="1030076"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Point+mutations+in+the+integron+integrase+IntI1+that+affect+recombination+and%2For+substrate+recognition.&amp;auth=&amp;atitle=Point+mutations+in+the+integron+integrase+IntI1+that+affect+recombination+and%2For+substrate+recognition." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Point+mutations+in+the+integron+integrase+IntI1+that+affect+recombination+and%2For+substrate+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Point+mutations+in+the+integron+integrase+IntI1+that+affect+recombination+and%2For+substrate+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001315-Jahnz1" id="pone.0001315-Jahnz1"></a>Jahnz M, Schwille P (2005) An ultrasensitive site-specific DNA recombination assay based on dual-color fluorescence cross-correlation spectroscopy. Nucleic Acids Res  33: e60.  <ul class="find" data-citedArticleID="1030088" data-doi="10.1093/nar/gni060"><li><a href="http://dx.doi.org/10.1093/nar/gni060" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+ultrasensitive+site-specific+DNA+recombination+assay+based+on+dual-color+fluorescence+cross-correlation+spectroscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+ultrasensitive+site-specific+DNA+recombination+assay+based+on+dual-color+fluorescence+cross-correlation+spectroscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001315-Ball1" id="pone.0001315-Ball1"></a>Ball CA, Johnson RC (1991) Efficient excision of phage lambda from the Escherichia coli chromosome requires the Fis protein. J Bacteriol  173: 4027–4031.  <ul class="find" data-citedArticleID="1030052"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Efficient+excision+of+phage+lambda+from+the+Escherichia+coli+chromosome+requires+the+Fis+protein.&amp;auth=&amp;atitle=Efficient+excision+of+phage+lambda+from+the+Escherichia+coli+chromosome+requires+the+Fis+protein." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+excision+of+phage+lambda+from+the+Escherichia+coli+chromosome+requires+the+Fis+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+excision+of+phage+lambda+from+the+Escherichia+coli+chromosome+requires+the+Fis+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001315-Babineau1" id="pone.0001315-Babineau1"></a>Babineau D, Vetter D, Andrews BJ, Gronostajski RM, Proteau GA, et al.  (1985) The FLP protein of the 2-micron plasmid of yeast. Purification of the protein from Escherichia coli cells expressing the cloned FLP gene. J Biol Chem  260: 12313–12319.  <ul class="find" data-citedArticleID="1030050"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+FLP+protein+of+the+2-micron+plasmid+of+yeast.+Purification+of+the+protein+from+Escherichia+coli+cells+expressing+the+cloned+FLP+gene.&amp;auth=&amp;atitle=The+FLP+protein+of+the+2-micron+plasmid+of+yeast.+Purification+of+the+protein+from+Escherichia+coli+cells+expressing+the+cloned+FLP+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+FLP+protein+of+the+2-micron+plasmid+of+yeast.+Purification+of+the+protein+from+Escherichia+coli+cells+expressing+the+cloned+FLP+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+FLP+protein+of+the+2-micron+plasmid+of+yeast.+Purification+of+the+protein+from+Escherichia+coli+cells+expressing+the+cloned+FLP+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001315-Gronostajski1" id="pone.0001315-Gronostajski1"></a>Gronostajski RM, Sadowski PD (1985) The FLP protein of the 2-micron plasmid of yeast. Inter- and intramolecular reactions. J Biol Chem  260: 12328–12335.  <ul class="find" data-citedArticleID="1030078"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+FLP+protein+of+the+2-micron+plasmid+of+yeast.+Inter-+and+intramolecular+reactions.&amp;auth=&amp;atitle=The+FLP+protein+of+the+2-micron+plasmid+of+yeast.+Inter-+and+intramolecular+reactions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+FLP+protein+of+the+2-micron+plasmid+of+yeast.+Inter-+and+intramolecular+reactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+FLP+protein+of+the+2-micron+plasmid+of+yeast.+Inter-+and+intramolecular+reactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001315-Ringrose1" id="pone.0001315-Ringrose1"></a>Ringrose L, Lounnas V, Ehrlich L, Buchholz F, Wade R, et al.  (1998) Comparative kinetic analysis of FLP and cre recombinases: mathematical models for DNA binding and recombination. J Mol Biol  284: 363–384.  <ul class="find" data-citedArticleID="1030100" data-doi="10.1006/jmbi.1998.2149"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.2149" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+kinetic+analysis+of+FLP+and+cre+recombinases%3A+mathematical+models+for+DNA+binding+and+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+kinetic+analysis+of+FLP+and+cre+recombinases%3A+mathematical+models+for+DNA+binding+and+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001315-Biswas1" id="pone.0001315-Biswas1"></a>Biswas T, Aihara H, Radman-Livaja M, Filman D, Landy A, et al.  (2005) A structural basis for allosteric control of DNA recombination by lambda integrase. Nature  435: 1059–1066.  <ul class="find" data-citedArticleID="1030054" data-doi="10.1038/nature03657"><li><a href="http://dx.doi.org/10.1038/nature03657" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+structural+basis+for+allosteric+control+of+DNA+recombination+by+lambda+integrase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+structural+basis+for+allosteric+control+of+DNA+recombination+by+lambda+integrase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.XML" value="83189"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.PDF" value="974186"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g001.PNG_L" value="45043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g001.PNG_M" value="30039"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g001.PNG_S" value="5619"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g001.TIF" value="82958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g001.PNG_I" value="10820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g002.PNG_L" value="69870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g002.PNG_M" value="60633"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g002.PNG_S" value="5481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g002.TIF" value="94446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g002.PNG_I" value="23038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g003.PNG_L" value="150916"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g003.PNG_M" value="121526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g003.PNG_S" value="7965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g003.TIF" value="196500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g003.PNG_I" value="40834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g004.PNG_L" value="98605"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g004.PNG_M" value="34006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g004.PNG_S" value="6133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g004.TIF" value="209150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g004.PNG_I" value="12553"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g005.PNG_L" value="46818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g005.PNG_M" value="20921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g005.PNG_S" value="4719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g005.TIF" value="96632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g005.PNG_I" value="8510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g006.PNG_L" value="171134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g006.PNG_M" value="49992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g006.PNG_S" value="6103"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g006.TIF" value="279900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g006.PNG_I" value="18194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g007.PNG_L" value="77600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g007.PNG_M" value="77600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g007.PNG_S" value="8089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g007.TIF" value="102990"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g007.PNG_I" value="39355"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g008.PNG_L" value="118554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g008.PNG_M" value="103138"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g008.PNG_S" value="8274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g008.TIF" value="160680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g008.PNG_I" value="40480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g009.PNG_L" value="59915"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g009.PNG_M" value="25278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g009.PNG_S" value="5216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g009.TIF" value="122242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001315.g009.PNG_I" value="9958"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001315&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001315" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001315&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001315&volume=&issue=&title=A New In Vitro Strand Transfer Assay for Monitoring Bacterial Class 1 Integron Recombinase IntI1 Activity&author_name=V%C3%A9ronique%20Dubois%2C%20Carole%20Debreyer%2C%20Simon%20Litvak%2C%20Claudine%20Quentin%2C%20Vincent%20Parissi&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001315" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001315" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001315" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001315&amp;t=A%20New%20In%20Vitro%20Strand%20Transfer%20Assay%20for%20Monitoring%20Bacterial%20Class%201%20Integron%20Recombinase%20IntI1%20Activity" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001315&title=A%20New%20In%20Vitro%20Strand%20Transfer%20Assay%20for%20Monitoring%20Bacterial%20Class%201%20Integron%20Recombinase%20IntI1%20Activity&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001315&amp;title=A%20New%20In%20Vitro%20Strand%20Transfer%20Assay%20for%20Monitoring%20Bacterial%20Class%201%20Integron%20Recombinase%20IntI1%20Activity" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001315" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001315&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A New In Vitro Strand Transfer Assay for Monitoring Bacterial Class 1 Integron Recombinase IntI1 Activity';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001315';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20New%20In%20Vitro%20Strand%20Transfer%20Assay%20for%20Monitoring%20Bacterial%20Class%201%20Integron%20Recombinase%20IntI1%20Activity http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001315" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001315" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+binding+assay%22" title="Search for articles in the subject area:'DNA binding assay'"><div class="flagText">DNA binding assay</div></a>
              <div data-categoryid="22215" data-articleid="25460"
                   data-categoryname="DNA binding assay"
                   class="flagImage" title="Flag 'DNA binding assay' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+recombination%22" title="Search for articles in the subject area:'DNA recombination'"><div class="flagText">DNA recombination</div></a>
              <div data-categoryid="33483" data-articleid="25460"
                   data-categoryname="DNA recombination"
                   class="flagImage" title="Flag 'DNA recombination' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzymes%22" title="Search for articles in the subject area:'Enzymes'"><div class="flagText">Enzymes</div></a>
              <div data-categoryid="17863" data-articleid="25460"
                   data-categoryname="Enzymes"
                   class="flagImage" title="Flag 'Enzymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="47289" data-articleid="25460"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombinant+protein+purification%22" title="Search for articles in the subject area:'Recombinant protein purification'"><div class="flagText">Recombinant protein purification</div></a>
              <div data-categoryid="21881" data-articleid="25460"
                   data-categoryname="Recombinant protein purification"
                   class="flagImage" title="Flag 'Recombinant protein purification' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombinant+proteins%22" title="Search for articles in the subject area:'Recombinant proteins'"><div class="flagText">Recombinant proteins</div></a>
              <div data-categoryid="18471" data-articleid="25460"
                   data-categoryname="Recombinant proteins"
                   class="flagImage" title="Flag 'Recombinant proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombination+reactions%22" title="Search for articles in the subject area:'Recombination reactions'"><div class="flagText">Recombination reactions</div></a>
              <div data-categoryid="19659" data-articleid="25460"
                   data-categoryname="Recombination reactions"
                   class="flagImage" title="Flag 'Recombination reactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombination-based+assay%22" title="Search for articles in the subject area:'Recombination-based assay'"><div class="flagText">Recombination-based assay</div></a>
              <div data-categoryid="19661" data-articleid="25460"
                   data-categoryname="Recombination-based assay"
                   class="flagImage" title="Flag 'Recombination-based assay' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=660'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9993'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2918&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
