

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Physical Analyses of E. coli Heteroduplex Recombination Products In Vivo: On the Prevalence of 5′ and 3′ Patches</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001242"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001242" />

    <meta name="citation_title" content="Physical Analyses of E. coli Heteroduplex Recombination Products In Vivo: On the Prevalence of 5′ and 3′ Patches"/>
    <meta itemprop="name" content="Physical Analyses of E. coli Heteroduplex Recombination Products In Vivo: On the Prevalence of 5′ and 3′ Patches"/>

      <meta name="citation_author" content="Laura M. Gumbiner-Russo"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Susan M. Rosenberg"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Baylor College of Medicine, Houston, Texas, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America"/>
            <meta name="citation_author_institution" content="Dan L. Duncan Cancer Center, Baylor College of Medicine, Houston, Texas, United States of America"/>

    <meta name="citation_date" content="2007/11/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001242.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1242"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Biochemistry of homologous recombination in Escherichia coli.; citation_author=SC Kowalczykowski; citation_author=DA Dixon; citation_author=AK Eggleston; citation_author=SD Lauder; citation_author=WM Rehrauer; citation_journal_title=Microbiol Rev; citation_volume=58; citation_number=1; citation_pages=401-465; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=χ and RecBCD enzyme of Escherichia coli.; citation_author=RS Myers; citation_author=FW Stahl; citation_journal_title=Annu Rev Genet; citation_volume=28; citation_number=2; citation_pages=49-70; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Biochemical and physical characterization of exonuclease V from Escherichia coli. Comparison of the catalytic activities of the RecBC and RecBCD enzymes.; citation_author=KM Palas; citation_author=SR Kushner; citation_journal_title=J Biol Chem; citation_volume=265; citation_number=3; citation_pages=3447-3454; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Substrate specificity of the DNA unwinding activity of the RecBC enzyme of Escherichia coli.; citation_author=AF Taylor; citation_author=GR Smith; citation_journal_title=J Mol Biol; citation_volume=185; citation_number=4; citation_pages=431-443; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Strand-specific binding to duplex DNA ends by the subunits of the Escherichia coli RecBCD enzyme.; citation_author=S Ganesan; citation_author=GR Smith; citation_journal_title=J Mol Biol; citation_volume=229; citation_number=5; citation_pages=67-78; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Roles of RecBC enzyme and Chi sites in homologous recombination.; citation_author=GR Smith; citation_author=SK Amundsen; citation_author=AM Chaudhury; citation_author=KC Cheng; citation_author=AS Ponticelli; citation_journal_title=Cold Spring Harbor Symp Quant Biol; citation_volume=49; citation_number=6; citation_pages=485-495; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Chi sites, RecBC enzyme, and generalized recombination.; citation_author=GR Smith; citation_author=DW Schultz; citation_author=AF Taylor; citation_author=K Triman; citation_journal_title=Stadler Genet Symp; citation_volume=13; citation_number=7; citation_pages=25-37; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Initiation of genetic recombination and recombination-dependent replication.; citation_author=SC Kowalczykowski; citation_journal_title=Trends Biochem Sci; citation_volume=25; citation_number=8; citation_pages=156-165; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Chi stimulated patches are heteroduplex, with recombinant information on the phage λ r chain.; citation_author=SM Rosenberg; citation_journal_title=Cell; citation_volume=48; citation_number=9; citation_pages=855-865; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Chain-bias of Escherichia coli Rec-mediated λ patch recombinants is independent of the orientation of λ cos.; citation_author=SM Rosenberg; citation_journal_title=Genetics; citation_volume=120; citation_number=10; citation_pages=7-21; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Chain bias in Chi-stimulated heteroduplex patches in the λ ren gene is determined by the orientation of λ cos.; citation_author=AT Hagemann; citation_author=SM Rosenberg; citation_journal_title=Genetics; citation_volume=129; citation_number=11; citation_pages=611-621; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Evidence for both 3′ and 5′ single-strand DNA ends in intermediates in Chi-stimulated recombination in vivo.; citation_author=H Razavy; citation_author=SK Szigety; citation_author=SM Rosenberg; citation_journal_title=Genetics; citation_volume=142; citation_number=12; citation_pages=333-339; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Evidence that SbcB and RecF pathway functions contribute to RecBCD-dependent transductional recombination.; citation_author=L Miesel; citation_author=JR Roth; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=13; citation_pages=3146-3155; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The effects on strand exchange of 5′ versus 3′ ends of single-stranded DNA in RecA Nucleoprotein filaments.; citation_author=M Dutreix; citation_author=BJ Rao; citation_author=CM Radding; citation_journal_title=J Mol Biol; citation_volume=219; citation_number=14; citation_pages=645-654; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The efficiency of strand invasion by Escherichia coli RecA is dependent upon the length and polarity of ssDNA tails.; citation_author=MJ McIlwraith; citation_author=SC West; citation_journal_title=J Mol Biol; citation_volume=305; citation_number=15; citation_pages=23-31; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The direction of RecA protein assembly onto single strand DNA is the same as the direction of strand assimilation during strand exchange.; citation_author=JC Register; citation_author=J Griffith; citation_journal_title=J Biol Chem; citation_volume=260; citation_number=16; citation_pages=12308-12312; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=The preference for a 3′ homologous end is intrinsic to RecA-promoted strand exchange.; citation_author=BB Konforti; citation_author=RW Davis; citation_journal_title=J Biol Chem; citation_volume=265; citation_number=17; citation_pages=6916-6920; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=3′ homologous free ends are required for stable joint molecule formation by the RecA and single-stranded binding proteins of Escherichia coli.; citation_author=BB Konforti; citation_author=RW Davis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=84; citation_number=18; citation_pages=690-694; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=RecA protein filaments: end-dependent disassociation from ssDNA and stabilization by RecO and RecR proteins.; citation_author=Q Shan; citation_author=JM Bork; citation_author=BL Webb; citation_author=RB Inman; citation_author=MM Cox; citation_journal_title=J Mol Biol; citation_volume=265; citation_number=19; citation_pages=519-540; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The RecOR proteins modulate RecA protein function at 5′ ends of single-stranded DNA.; citation_author=JM Bork; citation_author=MM Cox; citation_author=RB Inman; citation_journal_title=EMBO Journal; citation_volume=20; citation_number=20; citation_pages=7313-7322; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The split-end model for homologous recombination at double-strand breaks and at Chi.; citation_author=SM Rosenberg; citation_author=PJ Hastings; citation_journal_title=Biochimie; citation_volume=73; citation_number=21; citation_pages=385-397; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Heteroduplex chain polarity in recombination of phage λ by the Red, RecBCD, RecBC(D−), and RecF pathways.; citation_author=I Siddiqi; citation_author=MM Stahl; citation_author=FW Stahl; citation_journal_title=Genetics; citation_volume=128; citation_number=22; citation_pages=7-22; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Chi enhances heteroduplex DNA levels during recombination.; citation_author=SL Holbeck; citation_author=GR Smith; citation_journal_title=Genetics; citation_volume=132; citation_number=23; citation_pages=879-891; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=On the molecular basis of high negative interference.; citation_author=RL White; citation_author=MS Fox; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=71; citation_number=24; citation_pages=1544-1548; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Genetic evidence for preferential strand transfer during meiotic recombination in yeast.; citation_author=DK Nag; citation_author=TD Petes; citation_journal_title=Genetics; citation_volume=125; citation_number=25; citation_pages=753-761; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Phage λ has an analog of the Escherichia coli recO, recR and recF genes.; citation_author=JA Sawitzke; citation_author=FW Stahl; citation_journal_title=Genetics; citation_volume=130; citation_number=26; citation_pages=7-16; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Holliday junction binding and resolution by the Rap structure-specific endonuclease of phage lambda.; citation_author=GJ Sharples; citation_author=FA Curtis; citation_author=P McGlynn; citation_author=EL Bolt; citation_journal_title=J Mol Biol; citation_volume=340; citation_number=27; citation_pages=739-751; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A PCR artifact: generation of heteroduplexes.; citation_author=CM Nagamine; citation_author=K Chan; citation_author=YF Lau; citation_journal_title=Am J Hum Genet; citation_volume=45; citation_number=28; citation_pages=337-339; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Heteroduplex strand-specificity in restriction-stimulated recombination by the RecE pathway of Escherichia coli.; citation_author=Z Silberstein; citation_author=M Shalit; citation_author=A Cohen; citation_journal_title=Genetics; citation_volume=133; citation_number=29; citation_pages=439-448; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Lambda Gam protein inhibits the helicase and Chi-stimulated recombination activities of Escherichia coli RecBCD enzyme.; citation_author=KC Murphy; citation_journal_title=J Bacteriol; citation_volume=173; citation_number=30; citation_pages=5808-5821; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Initiation of bacteriophage lambda DNA replication in vitro with purified lambda replication proteins.; citation_author=MS Wold; citation_author=JB Mallory; citation_author=JD Roberts; citation_author=JH LeBowitz; citation_author=R McMacken; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=79; citation_number=31; citation_pages=6176-6180; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Synthesis of linear plasmid multimers in Escherichia coli K-12.; citation_author=A Cohen; citation_author=AJ Clark; citation_journal_title=J Bacteriol; citation_volume=167; citation_number=32; citation_pages=327-335; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products.; citation_author=KA Datsenko; citation_author=BL Wanner; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=97; citation_number=33; citation_pages=6640-6645; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Chromosomal integration of heterologous DNA in Escherichia coli with precise removal of markers and replicons used during construction.; citation_author=F Martinez-Morales; citation_author=AC Borges; citation_author=A Martinez; citation_author=KT Shanmugam; citation_author=LO Ingram; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=34; citation_pages=7143-7148; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Experimental Methods for use with Lambda.; citation_author=W Arber; citation_author=L Enquist; citation_author=B Hohn; citation_author=NE Murray; citation_author=K Murray; citation_number=35; citation_pages=433-466; citation_date=1983; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=A Short Course in Bacterial Genetics.; citation_author=JH Miller; citation_number=36; citation_date=1992; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Improved in vitro packaging of coliphage lambda DNA: a one-strain system free from endogenous phage.; citation_author=SM Rosenberg; citation_author=MM Stahl; citation_author=I Kobayashi; citation_author=FW Stahl; citation_journal_title=Gene; citation_volume=38; citation_number=37; citation_pages=165-175; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Molecular Cloning: A Laboratory Manual.; citation_author=J Sambrook; citation_author=DW Russell; citation_number=38; citation_date=2001; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Gel Loading Buffer Affects Electrophoresis Results.; citation_author=H Jordan; citation_journal_title=Focus; citation_volume=19; citation_number=39; citation_pages=60-61; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genomic sequencing.; citation_author=GM Church; citation_author=W Gilbert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=81; citation_number=40; citation_pages=1991-1995; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Decay-accelerating factor induction on vascular endothelium by vascular endothelial growth factor (VEGF) is mediated via a VEGF receptor-2 (VEGF-R2)- and protein kinase C-alpha/epsilon (PKCalpha/epsilon)-dependent cytoprotective signaling pathway and is inhibited by cyclosporin A.; citation_author=JC Mason; citation_author=R Steinberg; citation_author=EA Lidington; citation_author=AR Kinderlerer; citation_author=M Ohba; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=41; citation_pages=41611-41618; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Defects in neuromuscular junction structure in dystrophic muscle are corrected by expression of a NOS transgene in dystrophin-deficient muscles, but not in muscles lacking alpha- and beta1-syntrophins.; citation_author=T Shiao; citation_author=A Fond; citation_author=B Deng; citation_author=M Wehling-Henricks; citation_author=ME Adams; citation_journal_title=Hum Mol Genet; citation_volume=13; citation_number=42; citation_pages=1873-1884; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An RBCC protein implicated in maintenance of steady-state neuregulin receptor levels.; citation_author=AJ Diamonti; citation_author=PM Guy; citation_author=C Ivanof; citation_author=K Wong; citation_author=C Sweeney; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=43; citation_pages=2866-2871; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Effects of DNA heterologies on bacteriophage lambda recombination.; citation_author=RK Pearson; citation_author=MS Fox; citation_journal_title=Genetics; citation_volume=118; citation_number=44; citation_pages=13-19; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Repair of DNA heteroduplexes containing small heterologous sequences in Escherichia coli.; citation_author=BO Parker; citation_author=MG Marinus; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=89; citation_number=45; citation_pages=1730-1734; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Repair of heteroduplex DNA molecules with multibase loops in Escherichia coli.; citation_author=M Carraway; citation_author=MG Marinus; citation_journal_title=J Bacteriol; citation_volume=175; citation_number=46; citation_pages=3972-3980; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A single nuclease active site of the Escherichia coli RecBCD enzyme catalyzes single-stranded DNA degradation in both directions.; citation_author=J Wang; citation_author=R Chen; citation_author=DA Julin; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=47; citation_pages=507-513; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The bacterial RecA protein and the recombinational DNA repair of stalled replication forks.; citation_author=SL Lusetti; citation_author=MM Cox; citation_journal_title=Annu Rev Biochem; citation_volume=71; citation_number=48; citation_pages=71-100; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=RecBCD enzyme of Escherichia coli.; citation_author=AF Taylor; citation_number=49; citation_pages=231-263; citation_date=1988; citation_publisher=Am. Soc. Microbiol; " />
      <meta name="citation_reference" content="citation_title=Functional similarities between phage lambda Orf and Escherichia coli RecFOR in initiation of genetic exchange.; citation_author=KL Maxwell; citation_author=P Reed; citation_author=RG Zhang; citation_author=S Beasley; citation_author=AR Walmsley; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=50; citation_pages=11260-11265; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The phage lambda orf gene encodes a trans-acting factor that suppresses Escherichia coli recO, recR, and recF mutations for recombination of lambda but not of E. coli.; citation_author=JA Sawitzke; citation_author=FW Stahl; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=51; citation_pages=6730-6737; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Roles for lambda Orf and Escherichia coli RecO, RecR and RecF in lambda recombination.; citation_author=JA Sawitzke; citation_author=FW Stahl; citation_journal_title=Genetics; citation_volume=147; citation_number=52; citation_pages=357-369; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=RecFOR proteins load RecA protein onto gapped DNA to accelerate DNA strand exchange: a universal step of recombinational repair.; citation_author=K Morimatsu; citation_author=SC Kowalczykowski; citation_journal_title=Mol Cell; citation_volume=11; citation_number=53; citation_pages=1337-1347; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Recombination-promoting activity of the bacteriophage lambda Rap protein in Escherichia coli K-12.; citation_author=AR Poteete; citation_author=AC Fenton; citation_author=HR Wang; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=54; citation_pages=4626-4629; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=λ Rap protein is a structure-specific endonuclease involved in phage recombination.; citation_author=GJ Sharples; citation_author=LM Corbett; citation_author=IR Graham; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=55; citation_pages=13507-13512; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Efficient RecABC-dependent, homologous recombination between coliphage lambda and plasmids requires a phage ninR region gene.; citation_author=WC Hollifield; citation_author=EN Kaplan; citation_author=H Huang; citation_journal_title=Mol Gen Genet; citation_volume=210; citation_number=56; citation_pages=248-255; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Substrate specificity of RusA resolvase reveals the DNA structures targeted by RuvAB and RecG in vivo.; citation_author=EL Bolt; citation_author=RG Lloyd; citation_journal_title=Mol Cell; citation_volume=10; citation_number=57; citation_pages=187-198; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Homologous recombination resolution defect in Werner syndrome.; citation_author=Y Saintigny; citation_author=K Makienko; citation_author=C Swanson; citation_author=MJ Emond; citation_author=RJ Monnat; citation_journal_title=Mol Cell Biol; citation_volume=22; citation_number=58; citation_pages=6971-6978; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The involvement of Srs2 in post-replication repair and homologous recombination in fission yeast.; citation_author=CL Doe; citation_author=MC Whitby; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=59; citation_pages=1480-1491; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=On the clustered exchanges of the RecBCD pathway operating on phage λ.; citation_author=FW Stahl; citation_author=CE Shurvinton; citation_author=LC Thomason; citation_author=S Hill; citation_author=MM Stahl; citation_journal_title=Genetics; citation_volume=139; citation_number=60; citation_pages=1107-1121; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The recombination hotspot, Chi, is a regulatory sequence that acts by attenuating the nuclease activity of the Escherichia coli recBCD enzyme.; citation_author=DA Dixon; citation_author=SC Kowalczykowski; citation_journal_title=Cell; citation_volume=73; citation_number=61; citation_pages=87-96; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The recombination hot spot chi is a regulatory element that switches the polarity of DNA degradation by the RecBCD enzyme.; citation_author=DG Anderson; citation_author=SC Kowalczykowski; citation_journal_title=Genes Dev; citation_volume=11; citation_number=62; citation_pages=571-581; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Chi-activated RecBCD enzyme possesses 5′–>3′ nucleolytic activity, but RecBC enzyme does not: evidence suggesting that the alteration induced by Chi is not simply ejection of the RecD subunit.; citation_author=DG Anderson; citation_author=JJ Churchill; citation_author=SC Kowalczykowski; citation_journal_title=Genes Cells; citation_volume=2; citation_number=63; citation_pages=117-128; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Chi sequence protects against RecBCD degradation of DNA in vivo.; citation_author=PS Dabert; citation_author=SD Ehrlich; citation_author=A Gruss; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=89; citation_number=64; citation_pages=12073-12077; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The recombination hot spot χ activates RecBCD recombination by converting Escherichia coli to a recD mutant phenocopy.; citation_author=RM Myers; citation_author=A Kuzminov; citation_author=FW Stahl; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=92; citation_number=65; citation_pages=6244-6248; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Kinetics and processivity of ATP hydrolysis and DNA unwinding by the RecBC enzyme from Escherichia coli.; citation_author=F Korangy; citation_author=DA Julin; citation_journal_title=Biochemistry; citation_volume=32; citation_number=66; citation_pages=4873-4880; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Efficiency of ATP hydrolysis and DNA unwinding by the RecBC enzyme from Escherichia coli.; citation_author=F Korangy; citation_author=DA Julin; citation_journal_title=Biochemistry; citation_volume=33; citation_number=67; citation_pages=9552-9560; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Evidence that recBC-dependent degradation of duplex DNA in Escherichia coli recD mutants involves DNA unwinding.; citation_author=R Rinken; citation_author=B Thoms; citation_author=W Wackernagel; citation_journal_title=J Bacteriol; citation_volume=174; citation_number=68; citation_pages=5424-5429; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of RecBCD enzyme reveals a machine for processing DNA breaks.; citation_author=MR Singleton; citation_author=MS Dillingham; citation_author=M Gaudier; citation_author=SC Kowalczykowski; citation_author=DB Wigley; citation_journal_title=Nature; citation_volume=432; citation_number=69; citation_pages=187-193; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Processing of recombination intermediates by the RecG and RuvAB proteins of Escherichia coli.; citation_author=RG Lloyd; citation_author=GJ Sharples; citation_journal_title=Nucleic Acids Res; citation_volume=21; citation_number=70; citation_pages=1719-1725; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=RecG helicase activity at three- and four-strand DNA structures.; citation_author=P McGlynn; citation_author=RG Lloyd; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=71; citation_pages=3049-3056; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Processing of recombination intermediates by the RuvABC proteins.; citation_author=SC West; citation_journal_title=Annu Rev Genet; citation_volume=31; citation_number=72; citation_pages=213-244; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Reverse branch migration of Holliday junctions by RecG protein: a new mechanism for resolution of intermediates in recombination and DNA repair.; citation_author=MC Whitby; citation_author=L Ryder; citation_author=RG Lloyd; citation_journal_title=Cell; citation_volume=75; citation_number=73; citation_pages=341-350; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Opposing roles of the Holliday junction processing systems of Escherichia coli in recombination-dependent adaptive mutation.; citation_author=RS Harris; citation_author=KJ Ross; citation_author=SM Rosenberg; citation_journal_title=Genetics; citation_volume=142; citation_number=74; citation_pages=681-691; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Tests of the double-strand-break repair model for red-mediated recombination of phage lambda and plasmid lambda dv.; citation_author=DS Thaler; citation_author=MM Stahl; citation_author=FW Stahl; citation_journal_title=Genetics; citation_volume=116; citation_number=75; citation_pages=501-511; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=A role for topoisomerase III in a recombination pathway alternative to RuvABC.; citation_author=CR Lopez; citation_author=S Yang; citation_author=RW Deibler; citation_author=SA Ray; citation_author=JM Pennington; citation_journal_title=Mol Microbiol; citation_volume=58; citation_number=76; citation_pages=80-101; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Double-strand break-repair in E. coli: direct physical evidence for a replicational mechanism in vivo.; citation_author=M Motamedi; citation_author=SK Szigety; citation_author=SM Rosenberg; citation_journal_title=Genes & Dev; citation_volume=13; citation_number=77; citation_pages=2889-2903; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=On Replicative and Break-Join Resolution, and RecQ-Promoted Accumulation of Recombination Intermediates in vivo. PhD Thesis; citation_author=DB Magner; citation_number=78; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Heteroduplex joint formation in Escherichia coli recombination is initiated by pairing of a 3′-ending strand.; citation_author=R Friedman-Ohana; citation_author=A Cohen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=79; citation_pages=6909-6914; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Detection of heteroduplex DNA molecules among the products of Saccharomyces cerevisiae meiosis.; citation_author=M Lichten; citation_author=C Goyon; citation_author=NP Schultes; citation_author=D Treco; citation_author=JW Szostak; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=87; citation_number=80; citation_pages=7653-7657; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=χ activity during transduction-associated recombination.; citation_author=NA Dower; citation_author=FW Stahl; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=78; citation_number=81; citation_pages=7033-7037; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Recombinational repair of DNA damage in Escherichia coli and bacteriophage lambda.; citation_author=A Kuzminov; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=63; citation_number=82; citation_pages=751-813; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Homologous recombination.; citation_author=RG Lloyd; citation_author=KB Low; citation_number=83; citation_pages=2236-2255; citation_date=1996; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=Homologous recombination promoted by Chi sites and RecBC enzyme of Escherichia coli.; citation_author=GR Smith; citation_author=FW Stahl; citation_journal_title=Bioessays; citation_volume=2; citation_number=84; citation_pages=244-249; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Recombination of bacteriophage λ in recD mutants of Escherichia coli.; citation_author=DS Thaler; citation_author=E Sampson; citation_author=I Siddiqi; citation_author=SM Rosenberg; citation_author=LC Thomason; citation_journal_title=Genome; citation_volume=31; citation_number=85; citation_pages=53-67; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Interaction with the recombination hot spot chi in vivo converts the RecBCD enzyme of Escherichia coli into a chi-independent recombinase by inactivation of the RecD subunit.; citation_author=A Koppen; citation_author=S Krobitsch; citation_author=B Thoms; citation_author=W Wackernagel; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=86; citation_pages=6249-6253; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of recD, a gene affecting plasmid maintenance and recombination in Escherichia coli.; citation_author=DP Biek; citation_author=SN Cohen; citation_journal_title=J Bacteriol; citation_volume=167; citation_number=87; citation_pages=594-603; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Conjugational recombination in resolvase-deficient ruvC mutants of Escherichia coli depends on recG.; citation_author=RG Lloyd; citation_journal_title=J Bacteriol; citation_volume=173; citation_number=88; citation_pages=5414-5418; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Multiple pathways of recombination induced by double-strand breaks in Saccharomyces cerevisiae.; citation_author=F Paques; citation_author=JE Haber; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=63; citation_number=89; citation_pages=349-404; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Targeted gene replacement in Drosophila via P element-induced gap repair.; citation_author=GB Gloor; citation_author=NA Nassif; citation_author=DM Johnson-Schlitz; citation_author=CR Preston; citation_author=WR Engels; citation_journal_title=Science; citation_volume=253; citation_number=90; citation_pages=1110-1117; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Efficient copying of nonhomologous sequences from ectopic sites via P-element-induced gap repair.; citation_author=N Nassif; citation_author=J Penney; citation_author=S Pal; citation_author=WR Engels; citation_author=GB Gloor; citation_journal_title=Mol Cell Biol; citation_volume=14; citation_number=91; citation_pages=1613-1625; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Differential timing and control of noncrossover and crossover recombination during meiosis.; citation_author=T Allers; citation_author=M Lichten; citation_journal_title=Cell; citation_volume=106; citation_number=92; citation_pages=47-57; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The double-strand break repair model for recombination.; citation_author=JW Szostak; citation_author=TL Orr-Weaver; citation_author=RJ Rothstein; citation_author=FW Stahl; citation_journal_title=Cell; citation_volume=33; citation_number=93; citation_pages=25-33; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Impaired expression of certain prereplicative bacteriophage T4 genes explains impaired T4 DNA synthesis in Escherichia coli rho (nusD) mutants.; citation_author=BL Stitt; citation_author=G Mosig; citation_journal_title=J Bacteriol; citation_volume=171; citation_number=94; citation_pages=3872-3880; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Reassembly of shattered chromosomes in Deinococcus radiodurans.; citation_author=K Zahradka; citation_author=D Slade; citation_author=A Bailone; citation_author=S Sommer; citation_author=D Averbeck; citation_journal_title=Nature; citation_volume=443; citation_number=95; citation_pages=569-573; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gene targeting by linear duplex DNA frequently occurs by assimilation of a single strand that is subject to preferential mismatch correction.; citation_author=W-Y Leung; citation_author=A Malkova; citation_author=JE Haber; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=96; citation_pages=6851-6856; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=High efficiency mutagenesis, repair, and engineering of chromosomal DNA using single-stranded oligonucleotides.; citation_author=HM Ellis; citation_author=D Yu; citation_author=T DiTizio; citation_author=DL Court; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=97; citation_pages=6742-6746; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic engineering using homologous recombination.; citation_author=DL Court; citation_author=JA Sawitzke; citation_author=LC Thomason; citation_journal_title=Annu Rev Genet; citation_volume=36; citation_number=98; citation_pages=361-388; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Chromosomal site-specific double-strand breaks are efficiently targeted for repair by oligonucleotides in yeast.; citation_author=F Storici; citation_author=CL Durham; citation_author=DA Gordenin; citation_author=MA Resnick; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=99; citation_pages=14994-14999; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Conservative repair of a chromosomal double-strand break by single-strand DNA through two steps of annealing.; citation_author=F Storici; citation_author=JR Snipe; citation_author=GK Chan; citation_author=DA Gordenin; citation_author=MA Resnick; citation_journal_title=Mol Cell Biol; citation_volume=26; citation_number=100; citation_pages=7645-7657; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA packaging and cutting.; citation_author=M Feiss; citation_author=A Becker; citation_number=101; citation_pages=305-350; citation_date=1983; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=DNA double-chain breaks in recombination of phage λ and yeast.; citation_author=DS Thaler; citation_author=FW Stahl; citation_journal_title=Annu Rev Genet; citation_volume=22; citation_number=102; citation_pages=169-197; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Derivatives and genotypes of some mutant derivatives of Escherichia coli K-12.; citation_author=BJ Bachmann; citation_number=103; citation_pages=2460-2488; citation_date=1996; citation_publisher=American Society for Microbiology; " />
      <meta name="citation_reference" content="citation_title=Quantitative evaluation of Escherichia coli host strains for tolerance to cytosine methylation in plasmid and phage recombinants.; citation_author=DM Woodcock; citation_author=PJ Crowther; citation_author=J Doherty; citation_author=S Jefferson; citation_author=E DeCruz; citation_journal_title=Nucleic Acids Res; citation_volume=17; citation_number=104; citation_pages=3469-3478; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection.; citation_author=T Baba; citation_author=T Ara; citation_author=M Hasegawa; citation_author=Y Takai; citation_author=Y Okumura; citation_journal_title=Mol Syst Biol; citation_volume=2; citation_number=105; citation_pages=2006 0008; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The TGV transgenic vectors for single-copy gene expression from the Escherichia coli chromosome.; citation_author=LM Gumbiner-Russo; citation_author=MJ Lombardo; citation_author=RG Ponder; citation_author=SM Rosenberg; citation_journal_title=Gene; citation_volume=273; citation_number=106; citation_pages=97-104; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Recombination in adaptive mutation.; citation_author=RS Harris; citation_author=S Longerich; citation_author=SM Rosenberg; citation_journal_title=Science; citation_volume=264; citation_number=107; citation_pages=258-260; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Recombination in bacteriophage lambda. I. Mutants deficient in general recombination.; citation_author=ER Signer; citation_author=J Weil; citation_journal_title=J Mol Biol; citation_volume=34; citation_number=108; citation_pages=261-271; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=The role of recombination in the growth of bacteriophage lambda. I. The gamma gene.; citation_author=J Zissler; citation_author=E Signer; citation_author=F Schaefer; citation_number=109; citation_pages=455-468; citation_date=1971; citation_publisher=Cold Spring Harbor Laboratory; " />
      <meta name="citation_reference" content="citation_title=Genetic structure.; citation_author=A Campbell; citation_number=110; citation_pages=13-14; citation_date=1971; citation_publisher=Cold Spring Harbor Laboratory; " />
      <meta name="citation_reference" content="citation_title=Manipulation of restriction targets in phage lambda to form receptor chromosomes for DNA fragments.; citation_author=NE Murray; citation_author=K Murray; citation_journal_title=Nature; citation_volume=251; citation_number=111; citation_pages=476-481; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Rec-mediated recombinational hotspot activity in bacteriophage lambda III. Chi mutations are site mutations stimulating Rec-mediated recombination.; citation_author=FW Stahl; citation_author=JM Crasemann; citation_author=MM Stahl; citation_journal_title=J Mol Biol; citation_volume=94; citation_number=112; citation_pages=203-212; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=New mutations in the S cistron of bacteriophage lambda affecting host cell lysis.; citation_author=AR Goldberg; citation_author=M Howe; citation_journal_title=Virology; citation_volume=38; citation_number=113; citation_pages=200-202; citation_date=1969; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001242" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Physical Analyses of E. coli Heteroduplex Recombination Products In Vivo: On the Prevalence of 5′ and 3′ Patches"/>
    <meta name="twitter:description" content="BackgroundHomologous recombination in Escherichia coli creates patches (non-crossovers) or splices (half crossovers), each of which may have associated heteroduplex DNA. Heteroduplex patches have recombinant DNA in one strand of the duplex, with parental flanking markers. Which DNA strand is exchanged in heteroduplex patches reflects the molecular mechanism of recombination. Several models for the mechanism of E. coli RecBCD-mediated recombinational double-strand-end (DSE) repair specify that only the 3&prime;-ending strand invades the homologous DNA, forming heteroduplex in that strand. There is, however, in vivo evidence that patches are found in both strands.Methodology/Principle FindingsThis paper re-examines heteroduplex-patch-strand polarity using phage &lambda; and the &lambda;dv plasmid as DNA substrates recombined via the E. coli RecBCD system in vivo. These DNAs are mutant for &lambda; recombination functions, including orf and rap, which were functional in previous studies. Heteroduplexes are isolated, separated on polyacrylamide gels, and quantified using Southern blots for heteroduplex analysis. This method reveals that heteroduplexes are still found in either 5&prime; or 3&prime; DNA strands in approximately equal amounts, even in the absence of orf and rap. Also observed is an independence of the RuvC Holliday-junction endonuclease on patch formation, and a slight but statistically significant alteration of patch polarity by recD mutation.Conclusions/SignificanceThese results indicate that orf and rap did not contribute to the presence of patches, and imply that patches occurring in both DNA strands reflects the molecular mechanism of recombination in E. coli. Most importantly, the lack of a requirement for RuvC implies that endonucleolytic resolution of Holliday junctions is not necessary for heteroduplex-patch formation, contrary to predictions of all of the major previous models. This implies that patches are not an alternative resolution of the same intermediate that produces splices, and do not bear on models for splice formation. We consider two mechanisms that use DNA replication instead of endonucleolytic resolution for formation of heteroduplex patches in either DNA strand: synthesis-dependent-strand annealing and a strand-assimilation mechanism."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001242.g009"/>

  <meta property="og:title" content="Physical Analyses of E. coli Heteroduplex Recombination Products In Vivo: On the Prevalence of 5′ and 3′ Patches" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001242" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4804'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3136'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4059&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001242">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001242" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001242&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Physical Analyses of <i>E. coli</i> Heteroduplex Recombination Products <i>In Vivo</i>: On the Prevalence of 5′ and 3′ Patches
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laura M. Gumbiner-Russo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Susan M. Rosenberg
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:smr@bcm.tmc.edu">smr@bcm.tmc.edu</a></p>

                <p>Affiliations:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America, 
                  Department of Biochemistry and Molecular Biology, Baylor College of Medicine, Houston, Texas, United States of America, 
                  Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America, 
                  Dan L. Duncan Cancer Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001242</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001242" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001242" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001242" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001242" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001242-g001" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001242-t001" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001242-t002" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001242-g002" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001242-g003" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001242-g004" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001242-g005" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001242-g006" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001242-g007" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001242-g008" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001242-g009" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001242-t003" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001242">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001242" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Homologous recombination in <em>Escherichia coli</em> creates patches (non-crossovers) or splices (half crossovers), each of which may have associated heteroduplex DNA. Heteroduplex patches have recombinant DNA in one strand of the duplex, with parental flanking markers. Which DNA strand is exchanged in heteroduplex patches reflects the molecular mechanism of recombination. Several models for the mechanism of <em>E</em>. <em>coli</em> RecBCD-mediated recombinational double-strand-end (DSE) repair specify that only the 3′-ending strand invades the homologous DNA, forming heteroduplex in that strand. There is, however, <em>in vivo</em> evidence that patches are found in both strands.</p>


<h3>Methodology/Principle Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>This paper re-examines heteroduplex-patch-strand polarity using phage λ and the λdv plasmid as DNA substrates recombined <em>via</em> the <em>E</em>. <em>coli</em> RecBCD system <em>in vivo</em>. These DNAs are mutant for λ recombination functions, including <em>orf</em> and <em>rap</em>, which were functional in previous studies. Heteroduplexes are isolated, separated on polyacrylamide gels, and quantified using Southern blots for heteroduplex analysis. This method reveals that heteroduplexes are still found in either 5′ or 3′ DNA strands in approximately equal amounts, even in the absence of <em>orf</em> and <em>rap</em>. Also observed is an independence of the RuvC Holliday-junction endonuclease on patch formation, and a slight but statistically significant alteration of patch polarity by <em>recD</em> mutation.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>These results indicate that <em>orf</em> and <em>rap</em> did not contribute to the presence of patches, and imply that patches occurring in both DNA strands reflects the molecular mechanism of recombination in <em>E</em>. <em>coli</em>. Most importantly, the lack of a requirement for RuvC implies that endonucleolytic resolution of Holliday junctions is not necessary for heteroduplex-patch formation, contrary to predictions of all of the major previous models. This implies that patches are not an alternative resolution of the same intermediate that produces splices, and do not bear on models for splice formation. We consider two mechanisms that use DNA replication instead of endonucleolytic resolution for formation of heteroduplex patches in either DNA strand: synthesis-dependent-strand annealing and a strand-assimilation mechanism.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Gumbiner-Russo LM, Rosenberg SM (2007) Physical Analyses of <em>E. coli</em> Heteroduplex Recombination Products <em>In Vivo</em>: On the Prevalence of 5′ and 3′ Patches. PLoS ONE 2(11):
          e1242.
            doi:10.1371/journal.pone.0001242</p><p><strong>Academic Editor: </strong>Steve Sandler, University of Massachusetts, United States of America</p><p><strong>Received:</strong> August 15, 2007; <strong>Accepted:</strong> October 15, 2007; <strong>Published:</strong> November 28, 2007</p><p><strong>Copyright:</strong> © 2007 Gumbiner-Russo, Rosenberg. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by National Cancer Institute grant R01-CA85777.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>RecBCD is a powerful exonuclease, and an important enzyme in homologous recombination in <em>Escherichia coli</em> <a href="#pone.0001242-Kowalczykowski1">[1]</a>–<a href="#pone.0001242-Palas1">[3]</a>. In many models for RecBCD-mediated recombination <em>in vivo</em>, the enzyme is proposed to bind a double-strand DNA end (DSE) at a double-strand break <a href="#pone.0001242-Taylor1">[4]</a>, <a href="#pone.0001242-Ganesan1">[5]</a>, and create single-stranded DNA with a 3′ end that is coated with RecA protein and invades a homologous duplex DNA molecule. The 3′ end has been hypothesized to be created in different ways. In one model, a single-strand nick is made in the strand ending 3′ (relative to the DSE where RecBCD loaded) when RecBCD encounters a Chi sequence in the proper orientation, from the 3′ side of 5′-GCTGGTGG <a href="#pone.0001242-Smith1">[6]</a>, <a href="#pone.0001242-Smith2">[7]</a> (<a href="#pone-0001242-g001">Figure 1A</a>). Another model creates a single-stranded 3′ end by preferential degradation of the complementary 5′-ending strand by RecBCD after an encounter with Chi <a href="#pone.0001242-Kowalczykowski2">[8]</a> (<a href="#pone-0001242-g001">Figure 1C</a>). Neither model accounted adequately for the strand polarity of heteroduplex DNA patches formed by RecBCD-mediated Chi-stimulated recombination <em>in vivo</em> <a href="#pone.0001242-Rosenberg1">[9]</a>–<a href="#pone.0001242-Hagemann1">[11]</a>. Heteroduplex patches are a stretch of duplex containing strands from different parental DNA molecules. Both of these models assume that patches result from an alternative resolution of the same intermediate(s) that produce splices, and both predict that patches will contain donated DNA only in the 3′-ending strand, with respect to the DSE where RecBCD loaded.</p>
<div class="figure" id="pone-0001242-g001"><div class="img"><a name="pone-0001242-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Previous models for RecBCD-mediated recombination in <em>E. coli.</em></span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>RecBCD (notched circle) loads onto a double-strand end (DSE), depicted here as the right end of the phage λ chromosome, the left end being occluded during packaging by terminase and the packaging proteins (depicted as an octagon) <a href="#pone.0001242-Feiss1">[101]</a>. (A) The nick-at Chi model <a href="#pone.0001242-Smith1">[6]</a>, <a href="#pone.0001242-Smith2">[7]</a> suggested that RecBCD unwinds and rewinds the DNA until it encounters Chi, at which point it nicks the 3′-ending strand which invades a homologous duplex DNA molecule (red), creating patches exclusively in the 3′-ending strand. (B) The split-end model <a href="#pone.0001242-Rosenberg3">[21]</a> proposed that RecBCD degrades both strands until an encounter with Chi, effectively translocating the DSE to the Chi site. At Chi, RecBCD was proposed to lose its nuclease activity, retain helicase activity, and unwind the DNA. This split-end intermediate might be acted on by single-strand-dependent exonucleases of one polarity or the other, creating single-strand ends of either polarity that could invade a homologous duplex DNA molecule <a href="#pone.0001242-Razavy1">[12]</a>. 5′-end invasions were proposed to lead to 5′ patches, and 3′-end invasions to lead to 3′ patches. (C) The asymmetric DNA degradation model <a href="#pone.0001242-Kowalczykowski2">[8]</a> incorporates the proposal of <a href="#pone.0001242-Rosenberg3">[21]</a> that RecBCD degrades one or both (depicted here) DNA strands until an encounter with Chi, at which point this model specifies that the nuclease activity is altered and only the 5′-ending strand is degraded. This creates a 3′ end that invades a homologous duplex DNA molecule leading to patches exclusively in the 3′-ending strand. All of these models include endonucleolytic resolution of the strand-exchange intermediate, such as a Holliday junction (HJ), as the final step, a prediction that has been upheld for “break-join” splices <a href="#pone.0001242-Motamedi1">[77]</a>, <a href="#pone.0001242-Magner1">[78]</a>, but that will be called into question for patch formation by data presented below. We will suggest that none of these models can explain patch formation and consider alternatives. DNA ends with a half arrowhead represent 3′ ends, and plain ends represent 5′ ends.</p>
<span>doi:10.1371/journal.pone.0001242.g001</span></div><a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>However, <em>in vivo</em> and biochemical evidence support the presence of heteroduplex DNA in either 5′- or 3′-ending strands. First, RecBCD-mediated Chi-stimulated genetic <a href="#pone.0001242-Rosenberg1">[9]</a>, <a href="#pone.0001242-Rosenberg2">[10]</a>, then physical <a href="#pone.0001242-Hagemann1">[11]</a> analyses of heteroduplex patches formed <em>in vivo</em> into phage λ or plasmid DNAs showed patches in either strand of DNA. In these studies, RecBCD could load on only one end of the λ molecule that initiated recombination. Thus, the data suggested that either the 5′- or 3′-ending strand released by RecBCD upon end processing could initiate recombination that forms a patch <em>in vivo</em>. Second, <em>in vivo</em> experiments testing the role of single-strand DNA exonucleases in recombination showed a significant decrease in splice formation only when exonucleases of both polarities were removed <a href="#pone.0001242-Razavy1">[12]</a>, <a href="#pone.0001242-Miesel1">[13]</a>. This suggested that single-strand exonucleases could create a recombination intermediate that contained an overhang of one polarity or the other, either of which could be used as a strand-exchange intermediate for recombination. Third, biochemical studies of DNA strand invasion with RecA protein suggest that 5′ or 3′ ssDNA ends are able to invade and form D-loops with a homologous duplex DNA <em>in vitro</em> <a href="#pone.0001242-Dutreix1">[14]</a>, <a href="#pone.0001242-McIlwraith1">[15]</a>. RecA polymerizes in a 5′ to 3′ direction on single-stranded DNA <a href="#pone.0001242-Register1">[16]</a>, making it obvious how 3′ ends can be coated, and perhaps accounting for preferential 3′-end invasion activity with purified RecA <a href="#pone.0001242-Konforti1">[17]</a>, <a href="#pone.0001242-Konforti2">[18]</a>. However, when RecOR accessory proteins are added, 5′ ends can also be used in strand-exchange reactions <a href="#pone.0001242-Shan1">[19]</a>, <a href="#pone.0001242-Bork1">[20]</a>, perhaps accounting for the <em>in vivo</em> results.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>These data appeared to be inconsistent with the recombination models with exclusively 3′-end invasions, and supported an alternative model <a href="#pone.0001242-Rosenberg3">[21]</a>. The split-end model (<a href="#pone-0001242-g001">Figure 1B</a>), was the first to suggest that RecBCD loads onto DNA ends in its exonuclease mode and degrades/removes both strands of DNA until it encounters a properly oriented Chi site, effectively translocating DSEs to Chi. Upon an encounter with Chi, RecBCD was proposed to lose its exonuclease activity, but retain helicase activity. The helicase activity was hypothesized to be the recombinagenic activity of the enzyme, unwinding the DNA and creating single-stranded DNA ends of both polarities that are coated with RecA and capable of invading a homologous duplex DNA molecule. These ends might be subject to digestion by 5′ or 3′ single-strand exonucleases to create 5′ or 3′ overhangs (in different molecules, <a href="#pone-0001242-g001">Figure 1B</a>) <a href="#pone.0001242-Razavy1">[12]</a>, <a href="#pone.0001242-Miesel1">[13]</a>, which might then lead to patches of either polarity. Supporting this possibility, <em>either</em> 5′ <em>or</em> 3′ single-strand exonucleases are required for efficient Chi-stimulated recombination (splicing) of phage λ DNAs <em>in vivo</em>, implying that digestion of a strand of <em>either</em> polarity is required at some stage of the reaction <em>in vivo</em>, for example, to create a recombinagenic DNA end <a href="#pone.0001242-Razavy1">[12]</a> (other interpretations are discussed below). Note that this, and the two other models described above (<a href="#pone-0001242-g001">Figure 1A–C</a>), all share the feature that patches are presumed to result from endonucleotylic cleavage of Holliday junctions, as an alternative product to splices—an idea that will be questioned below.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Analysis of the structures of heteroduplexes has been used to examine recombination products in order to probe the mechanism of recombination <em>in vivo</em>. Purely genetic methods used the <em>P</em> or <em>c</em>I genes of phage λ, and selection or screening for recombinant phage that had once contained heteroduplexes <a href="#pone.0001242-Rosenberg1">[9]</a>, <a href="#pone.0001242-Siddiqi1">[22]</a>–<a href="#pone.0001242-White1">[24]</a>, and also in yeast using the <em>his4</em> locus and tetrad analysis <a href="#pone.0001242-Nag1">[25]</a>. Hagemann and Rosenberg <a href="#pone.0001242-Hagemann1">[11]</a> examined the strand polarity of heteroduplex-patch recombinants physically using partially denaturing gel electrophoresis and found both 5′ and 3′ patches to be prevalent, with a small (two-fold) bias toward 5′ patches. Their heteroduplexes consisted of five mispairs within an 18 bp insert in the λ <em>ren</em> gene. Although reproducible, the method used was arduous and complicated. In addition, since that work, two open reading frames were identified in λ, <em>orf</em> and <em>rap</em>, that encode functions that might have influenced their results: Orf encodes an analog of RecO, RecR, and RecF strand-exchange accessory proteins <a href="#pone.0001242-Sawitzke1">[26]</a>. Rap encodes a Holliday-junction and D-loop endonuclease <a href="#pone.0001242-Sharples1">[27]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>This paper describes a different physical method for analysis of heteroduplex structure in patch recombinants formed <em>in vivo</em>. Heteroduplexes are formed <em>in vivo via</em> recombination of a λ phage that is wild-type at <em>ren</em>, and a λdv plasmid containing the 18 bp insert at <em>ren</em> <a href="#pone.0001242-Hagemann1">[11]</a> as recombination substrates. The 18 bp insert forms 18 nt looped heteroduplexes that can be separated from each other by native polyacrylamide gel electrophoresis (per the method of <a href="#pone.0001242-Nagamine1">[28]</a>). Separated DNA is then transferred from the gel to a membrane and hybridized with a general or specific non-radioactively labeled probe to identify and quantify heteroduplexes of each polarity (similar to <a href="#pone.0001242-Silberstein1">[29]</a>). The λ phage are mutant for their own recombination functions (<em>red</em> and <em>gam</em>), and allow RecBCD-mediated recombination (<a href="#pone.0001242-Murphy1">[30]</a>, and shown below).</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>λ chromosomes in cells are circular until packaging, at which point they acquire a DSB at <em>cos</em> with <em>one</em> DSE available for RecBCD loading: the phage λ chromosome right end only (as depicted in the standard λ map, and as shown at the top of <a href="#pone-0001242-g001">Figure 1</a>, reviewed <a href="#pone.0001242-Myers1">[2]</a>). The structures of heteroduplex-patch recombinants formed <em>in vivo</em> were re-examined, and compared with those from infections (crosses) using λ phage deleted for <em>orf</em> and <em>rap</em>. Patches formed in this system were found first to be dependent on RecA and RecB, as expected for <em>E</em>. <em>coli</em> RecBCD-mediated DSE-repair recombination. Second, although patch frequency is increased in the presence of functional λ <em>orf</em> and <em>rap</em> genes, approximately equal amounts of the two polarities of heteroduplex were obtained regardless of their presence, demonstrating no influence of λ Orf and Rap recombination proteins on patch-strand polarity, and implying that the patch polarity really does reflect <em>E</em>. <em>coli</em> recombination functions. Third, the loss of <em>recD</em> causes a small but statistically significant shift toward more 5′ patches. Fourth, surprisingly, RuvC, the Holliday-junction endonuclease, is not required for patching. This result suggests that none of the models shown in <a href="#pone-0001242-g001">Figure 1</a>, all of which require endonucleolytic cleavage of a Holliday-junction, bear on patch formation, though any of them might explain the formation of splices. We consider two different models in which patches form by a different route from that generating splices, and which uses DNA replication, rather than Holliday-junction cleavage, to resolve (non-Holliday-junction) intermediates.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Construction of strains, plasmids, and new alleles</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Bacterial strains used in this study are given in <a href="#pone-0001242-t001">Table 1</a>. Plasmids pLGR4 and pLGR5, containing the lambda <em>orf</em> deletion (Δ<em>orf432</em>), a net deletion of 432 bp, were constructed by PCR using outward-facing primers Δorf-146 forward 5′-GGGTTT<strong>GCTAGC</strong>CTGATGCATCAGTTC<span class="underline">GGGCTGCATGATAAATGTCG</span>-3′ and Δorf-146 reverse 5′-GGGTTT<strong>GCTAGC</strong><span class="underline">AAGGCCTGCGATTACCAGC</span>-3′ (underlined sequence has homology to the plasmid, and sequence in bold is the recognition site for <em>Nhe</em>I) on either side of the lambda <em>orf</em> gene present in the λdv derivatives pKC31 (made by R. N. Rao, Eli Lilly &amp; Co., Indianapolis, IN, and is similar to pRLM4 <a href="#pone.0001242-Wold1">[31]</a>, see also <a href="#pone.0001242-Cohen1">[32]</a>) and pAH15 <a href="#pone.0001242-Hagemann1">[11]</a> respectively. (pAH15 is derived from pKC31 and contains an 18 bp insert including a <em>Xho</em>I restriction site <a href="#pone.0001242-Hagemann1">[11]</a>.) This deletes λ base pairs 40,621–41,073, and inserts 21 bp including an <em>Nhe</em>I restriction site to create a net deletion of 432 bp. PCR products were digested with <em>Nhe</em>I (New England BioLabs), diluted, and ligated to form circular plasmids in which <em>orf</em> was deleted. Lambda strains (<a href="#pone-0001242-t002">Table 2</a>) were deleted first for <em>rap</em> (Δ<em>rap514</em>), a net deletion of 514 bp, by homologous linear replacement of the gene in λOH3 (<em>red3 gam210 imm<sup>21</sup>cts</em> SR4° χ<sup>+</sup>D <em>Sam7</em>) as a prophage. Primers Δrap-CAT forward 5′-TGATGAGCGATCCGAATAGCTCGATGCACGAGGAAGA​AGATGATGGCTAA<span class="underline">GTGTAGGCTGGAGCTGCTTC</span>-3′ and Δrap-CAT reverse 5′-GTATGCTTCAACGAGCATGTCTGGAATGGTTTTTACT​GAGAACGTCATGC<span class="underline">CATATGAATATCCTCCTTAGT</span>-3′ (underlined sequence has homology to the plasmid pKD3, and non-underlined sequence has homology to either side of the lambda <em>rap</em> gene) were used to amplify the chloramphenicol-resistance (<em>cat</em>) gene from pKD3 <em>per</em> <a href="#pone.0001242-Datsenko1">[33]</a>. The PCR product was transformed into strain AH9, which also contained pKD46 <a href="#pone.0001242-Datsenko1">[33]</a>, and chloramphenicol-resistant transformants were selected and verified by PCR using primers nin5L 40.410 5′-GTCTTCTGGTTATCGAAGG-3′ and nin5R 43.522 5′-CTTTGTCGTAATCGAGATT-3′ to create strain SMR6230. The <em>cat</em> gene was removed from the prophage in SMR6230 using pFT-A to express FLP <a href="#pone.0001242-MartinezMorales1">[34]</a> to create strain SMR10233, and the <em>rap</em> deletion was verified by PCR using the same primers as above. This deletes λ base pairs 42,440–43,037, and inserts 84 bp including the remaining FRT “scar” to create a net deletion of 514 bp. The <em>rap</em> deleted prophage was then heat induced by shaking vigorously at 44°C for 15 minutes, followed by 37°C for 2 hours. Phage were isolated by lysing the culture, and were then infected into cells carrying pSR1 (λdv <em>imm<sup>434</sup> Ots28</em> Kan<sup>R</sup>). Phage λ <em>Ots28</em> recombinants were screened for temperature sensitivity at 42°C, and then allowed to recombine with either pLGR5 or pLGR4 (which are <em>O</em><sup>+</sup>) and <em>O</em><sup>+</sup> recombinants were selected at 42°C to create λ strains deleted for both <em>orf</em> and <em>rap</em>, with or without the 18bp insert (strains λSR543 and λSR542, respectively). The double deletion was verified by PCR using primers nin5R 43.522 and nin5L 40.410 (above). Stocks of λ phage were made by standard methods <a href="#pone.0001242-Arber1">[35]</a>. <em>E</em>. <em>coli</em> strains were made by standard P1 transduction methods <a href="#pone.0001242-Miller1">[36]</a>.</p>
<div class="figure" id="pone-0001242-t001"><div class="img"><a name="pone-0001242-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span><em>E. coli</em> K-12 strains and plasmids</span></strong></p><span>doi:10.1371/journal.pone.0001242.t001</span></div><div class="figure" id="pone-0001242-t002"><div class="img"><a name="pone-0001242-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>λ phage strains</span></strong></p><span>doi:10.1371/journal.pone.0001242.t002</span></div>

<h4>λ by λdv crosses</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Bacterial strains carrying an <em>orf</em> deleted plasmid, pLGR4 or pLGR5, were inoculated into Mating Culture Broth (MCB = LBK <a href="#pone.0001242-Rosenberg4">[37]</a>, plus 5mM MgSO<sub>4</sub>, 10mM Thymine, and 10mM Vitamin B1) at a dilution of 1:100 from saturated overnight cultures grown in LBK with 50 µg/mL kanamycin shaking at 37°C. Mating cultures (enough for 5.0–10.0 mL) were grown for at least 2.5 hours at 37°C in the presence of 50 µg/mL kanamycin until a density of 1.5×10<sup>8</sup> cells/mL was reached, as determined using a Petroff-Hausser counter. Phage were introduced at a multiplicity of infection of 7, and allowed to infect cultures for 30 minutes at 37°C while shaking. 25mL of cold TM <a href="#pone.0001242-Sambrook1">[38]</a> buffer was added to the infected cells, which were then pelleted and resuspended in 50 or 100 mL of prewarmed MCB with kanamycin, and allowed to shake vigorously (300 rpm) for 3–4 hours at 37°C. Cultures were then centrifuged for 10 minutes at 7,000 rpm and pellets were stored at −20°C until recombinant plasmid was isolated.</p>


<h4>Isolation of DNA</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Plasmid DNA from crosses was isolated, and purified over CsCl gradients as described <a href="#pone.0001242-Sambrook1">[38]</a>. Ethidium bromide was extracted using isoamyl alcohol <a href="#pone.0001242-Sambrook1">[38]</a>, and CsCl was removed by dialysis against TE <a href="#pone.0001242-Sambrook1">[38]</a> using Slide-A-Lyzer Dialysis Cassettes, 10,000 MWCO (Pierce, Rockford, IL). Plasmid DNA preparations were then ethanol precipitated, and the pellets were dissolved in ddH<sub>2</sub>0. Any remaining lambda DNA was removed by linearization with <em>Ase</em>I (New England BioLabs), which does not linearize any of the plasmids used here, but cleaves λ; followed by digestion with Plasmid-Safe ATP-Dependent DNase (EPICENTRE, Madison, WI) overnight at 37°C. Pure plasmid DNA was again ethanol precipitated, and the pellets dissolved in 10 mM Tris pH 8.0.</p>


<h4>Analysis of recombinants</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The yield of plasmid DNA isolated from crosses was checked by agarose gel electrophoresis in 1X TAE buffer <a href="#pone.0001242-Sambrook1">[38]</a>. The amount of DNA to be run on an acrylamide gel was determined empirically because the plasmid DNA contained multimers and therefore could not be compared directly to a DNA size marker for estimation. (DNA concentration could not be determined using a spectrophotometer due to the low yields obtained from crosses.) DNA was digested with <em>Stu</em>I (New England BioLabs) to release a 604 bp fragment containing the loop heteroduplex. Artificial heteroduplexes were prepared by digesting pLGR4 (no insert) and pLGR5 (with the 18 bp insert) with <em>Stu</em>I, mixing equal amounts of each digest, boiling the mixture for 10 minutes, and allowing the duplexes to reanneal at 65°C for 20 minutes. Either one or two different amounts of DNA for each cross was loaded onto a 5% native polyacrylamide gel (BioRad), and run in 1X TBE buffer (BioRad) at room temperature for approximately 140 minutes at 110V. (Undiluted restriction digests were loaded using Ficoll loading dye, diluted digests were loaded using glycerol loading dye <a href="#pone.0001242-Jordan1">[39]</a>.) After electrophoresis, the DNA in the gel was denatured for 20 minutes in denaturation buffer (0.5 M NaOH, 1.5 M NaCl <a href="#pone.0001242-Sambrook1">[38]</a>), and neutralized in 1X TBE 3 times for 20 minutes each (George Church, personal communication). DNA was electroblotted to a positively charged nylon membrane (Roche) in very cold 1X TBE using an Electrophoretic Mini-Blotter (C.B.S. Scientific) at 75V for 2.5–3 hours (or 70V for 3–4 hours) with constant voltage in the cold room. After transfer, the DNA was fixed onto the blot by UV crosslinking with 1600 J/m<sup>2</sup> <a href="#pone.0001242-Church1">[40]</a> and the blot was stored at 4°C.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Blots were hybridized with either digoxygenin (DIG)-labeled oligonucleotide probes specific for the two complementary heteroduplex loops, or with a 700 bp DIG-labeled PCR product homologous to the restriction fragment containing the heteroduplex (similar to <a href="#pone.0001242-Silberstein1">[29]</a>). The two 18 nt oligonucleotide probes; 5′ probe–<em>Xho</em>I40.4 R strand probe 5′-AGGGCTCGAGCGGGGAGC-3′, 3′ probe–<em>Xho</em>I40.4 L strand probe 5′-GCTCCCCGCTCGAGCCCT-3′; were labeled by creating a tail of DIG molecules on the 3′ end of each using the DIG Oligonucleotide Tailing Kit (Roche). These were hybridized to blots in sequential hybridization reactions using an aqueous hybridization buffer similar to that suggested by Roche [5X SSC, 2% Blocking Reagent (Roche), 0.1% N-Lauroyl Sarcosine, 1% SDS, and 0.1 mg/mL polyA (Roche)] at 47.5°C overnight with 0.15 pmol/mL oligonucleotide probe. Blots were washed twice for 5 minutes each with 2X SSC, 0.1% SDS at room temperature; and then three times for 10 minutes each with 0.5X SSC, 0.1% SDS at 60°C. The PCR probe was labeled using the PCR DIG Probe Synthesis Kit (Roche), with primers <em>ren</em> Left 40.232 5′-GAAGATCGCAGAAATCAAA-3′ and <em>ren</em> Right 41.364 5′-ATAAATGGCTTCAGAACAG-3′ and hybridized to blots using DIG EasyHyb Buffer (Roche) with 0.1 mg/mL sheared salmon sperm DNA (Ambion) at 65°C overnight with 10 µL probe. Blots were washed twice for 5 minutes each at room temperature as above, and then three times for 10 minutes each with 0.1X SSC, 0.1% SDS at 72°C. All blots were then developed for quantification of heteroduplex bands using CDP-Star substrate (Roche) according to kit protocol. Developed blots were either exposed to XAR5 film (Kodak) for between 10 seconds and 2 hours, depending on which probes were used, or imaged with a ChemiImager 5500 (Alpha Innotech, San Leandro, CA). Images were acquired in 5 minutes or less, and were quantified using AlphaEase software (Alpha Innotech). This method has been shown to be quantitative <a href="#pone.0001242-Mason1">[41]</a>–<a href="#pone.0001242-Diamonti1">[43]</a>. Blots that were probed multiple times were stripped according to kit protocol. Oligonucleotide probes were stripped at 50°C, and PCR probes at 65°C.</p>


<h4>Analysis of λ patch recombinant frequencies</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Recombinant frequencies were determined by diluting and plating phage progeny from crosses on <em>E. coli</em> indicator strain SMR423 (<a href="#pone-0001242-t001">Table 1</a>) for 100–1000 plaques per plate/blot. Plaque blots were made on positively charged nylon membranes (Roche) and processed according to manufacturer's instructions. Blots were hybridized with DIG-labeled oligonucleotide probes as above, except that both probes were hybridized to blots together in the same hybridization solution instead of separately. Hybridized blots were developed as above, and exposed to XAR5 film (Kodak) for 5–20 minutes.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Complementary heteroduplexes can be separated and identified</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The heteroduplexes examined in this study consist of an 18 nt loop in one or the other strand of a 604 bp restriction fragment. These heteroduplexes are products of recombination between a λ phage (mutant for its recombination functions) and a λdv-derived plasmid. Heteroduplex recombinants containing at least 19 nt heteroduplex loops have been observed in λ recombination <a href="#pone.0001242-Pearson1">[44]</a>, and such heteroduplexes are not repaired in <em>E. coli</em> <a href="#pone.0001242-Pearson1">[44]</a>–<a href="#pone.0001242-Carraway1">[46]</a>. The λdv-derived plasmid pLGR5 contains approximately 4 kb of homology to the λ phage, and an 18 bp heterologous insert <a href="#pone.0001242-Hagemann1">[11]</a>. Because the two possible heteroduplex species have single-strand loops with unique complementary sequences (<a href="#pone-0001242-g002">Figure 2</a>), they are chemically distinct, and therefore should be capable of being separated from each other and from their respective homoduplexes <a href="#pone.0001242-Nagamine1">[28]</a>.</p>
<div class="figure" id="pone-0001242-g002"><div class="img"><a name="pone-0001242-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Separation of heteroduplexes by polyacrylamide gel electrophoresis.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Southern blot of artificial heteroduplexes, made by melting and reannealing the 604 bp long DNA fragment with and without an 18 bp insertion marker, run on a 5% polyacrylamide gel. (A) Hybridization of the blot with a PCR-labeled probe complementary to the entire 604 bp restriction fragment. (B) Hybridization with an oligo probe complementary to the loop sequence in the strand ending 5′ at the right. (C) Hybridization with an oligo probe complementary to the loop sequence in the 3′-ending strand. “+”, the homoduplex fragment containing the 18 bp insert; “-”, the homoduplex fragment with no insert; “mixed”, melted and reannealed “+” and “−” DNAs (artificially prepared heteroduplexes). Figures beneath the gels represent the structures of Homo- and Het-containing fragments, showing the sequences of the complementary loops. “Het”, heteroduplex; “Homo”, homoduplex.</p>
<span>doi:10.1371/journal.pone.0001242.g002</span></div><div class="figure" id="pone-0001242-g003"><div class="img"><a name="pone-0001242-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Strategy for heteroduplex analysis of patches formed <em>in vivo</em> in λ by λdv “crosses”.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>Red, 18 nt insertion marker; green box, region of the λdv plasmid DNA that is not homologous with λ; blue parallel lines, strands of DNA; hexagon, phage λ capsid and packaging proteins which bind the λ chromosome left end during cleavage of λ DNA for packaging. This leaves only the right end free for RecBCD-initiated recombination <a href="#pone.0001242-Thaler3">[102]</a>.</p>
<span>doi:10.1371/journal.pone.0001242.g003</span></div><div class="figure" id="pone-0001242-g004"><div class="img"><a name="pone-0001242-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Heteroduplex patches are formed <em>in vivo</em>, RecA- and RecB-dependently.</span></strong></p><a id="article1.body1.sec3.sec1.fig3.caption1.p1" name="article1.body1.sec3.sec1.fig3.caption1.p1"></a><p>(A) Heteroduplexes are formed <em>in vivo</em>. Lanes labeled “wild-type” are two different amounts of DNA from a <em>rec</em><sup>+</sup> cross (SMR6726×λSR542), lanes labeled “mock cross 1–3” contain two different amounts of DNA from each of three mock crosses (JC11450×λSR542 mixed with SMR6726 without λ infection, as described in text). The last lane also contains an artificial het control. (B) RecA dependence of het patches. Strains used are: <em>rec</em><sup>+</sup> (SMR6726–JC11450 [pLGR5]), <em>recA</em> (SMR10154-JC11450 Δ(<em>srlR-recA</em>)<em>306</em>::Tn<em>10</em> [pLGR5]), and <em>topB</em> (SMR10207-JC11450 Δ<em>topB</em>::FRT [pLGR5]) each crossed with λSR542. Numbers above the lanes indicate the relative amounts of DNA loaded (<em>i. e</em>. a lane marked “4” indicates that lane contains four times the amount of DNA than was loaded in the lane marked “1” for the same cross). No het bands are visible for <em>recA</em> on the exposed film, but the scan of the film gives the appearance of bands present. The last lane is an artificial het control. (C) RecB dependence of het patches. Strains used are: <em>rec</em><sup>+</sup> as in B., and <em>recB</em> (SMR9579–JC11450 <em>recB21</em> [pLGR5]). Plasmid controls are artificial het controls as in <a href="#pone-0001242-g002">Figure 2</a>. The homoduplex band was run off of the gel for the control lanes, but because the homoduplex fragment was 50–100x more prevalent in the cross DNAs, the homoduplex band was much broader and the upper portion of that band remained visible.</p>
<span>doi:10.1371/journal.pone.0001242.g004</span></div><div class="figure" id="pone-0001242-g005"><div class="img"><a name="pone-0001242-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Prevalence of 5′- and 3′-strand patches is independent of λ Orf and Rap.</span></strong></p><a id="article1.body1.sec3.sec1.fig4.caption1.p1" name="article1.body1.sec3.sec1.fig4.caption1.p1"></a><p>(A) For analyses of plasmid DNAs isolated from crosses with the marker in the plasmid, Het I corresponds to a 3′ patch, and Het II corresponds to a 5′ patch. When the marker is in the λ, Het I corresponds to a 5′ patch and Het II to a 3′ patch. Red lines represent the 18 bp marker insert. (B) Plasmid DNAs isolated from <em>orf</em><sup>+</sup> <em>rap</em><sup>+</sup> crosses. Two crosses are shown. For each set of crosses, the lane marked “pls” denotes that the insertion marker was present in the plasmid (SMR6721×λSR539), and the lane marked “λ” denotes that the marker was present in the λ (FS1607×λSR538). See text for quantification. (C) Plasmid DNAs from <em>orf</em><sup>−</sup> <em>rap</em><sup>−</sup> crosses. One cross of each type is shown (marker present in the plasmid [SMR6720×λSR542] or in the λ [SMR6205×λSR543]), and two different amounts of DNA were loaded for each cross. Numbers above the lanes are relative amounts based on the estimated DNA concentration. Plasmid controls, artificial heteroduplexes made by melting and reannealing 604 bp fragments with and without the 18 bp marker insert as for <a href="#pone-0001242-g002">Figure 2</a>. See text for quantification from multiple crosses.</p>
<span>doi:10.1371/journal.pone.0001242.g005</span></div><div class="figure" id="pone-0001242-g006"><div class="img"><a name="pone-0001242-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Crosses in a <em>recD</em> strain show a slight 5′ bias in patch polarity.</span></strong></p><a id="article1.body1.sec3.sec1.fig5.caption1.p1" name="article1.body1.sec3.sec1.fig5.caption1.p1"></a><p>Lanes marked “<em>rec</em><sup>+</sup>” are two different amounts of DNA from a <em>rec</em><sup>+</sup> cross (SMR6726×λSR542). Lanes marked “<em>recD</em> cross 1” are two amounts of DNA from a <em>recD</em> cross (SMR10215×λSR542), and lanes marked “<em>recD</em> cross 2” are from a second <em>recD</em> cross. Control het is as for <a href="#pone-0001242-g002">Fig. 2</a>. For all crosses the 18 bp marker was present on the λdv plasmid, (SMR6726-JC11450 [pLGR5], SMR10215-JC11450 <em>recD1903</em>::mini-<em>tet</em> [pLGR5]).</p>
<span>doi:10.1371/journal.pone.0001242.g006</span></div><div class="figure" id="pone-0001242-g007"><div class="img"><a name="pone-0001242-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Loss of RuvC does not affect heteroduplex-patch formation.</span></strong></p><a id="article1.body1.sec3.sec1.fig6.caption1.p1" name="article1.body1.sec3.sec1.fig6.caption1.p1"></a><p>Strains are from left to right, <em>rec</em><sup>+</sup> (SMR6726–JC11450 [pLGR5]), and both <em>ruvC</em> crosses were performed using (SMR10213–JC11450 Δ<em>ruvC</em>::FRT [pLGR5]), each infected with λSR542. See text for quantification.</p>
<span>doi:10.1371/journal.pone.0001242.g007</span></div><div class="figure" id="pone-0001242-g008"><div class="img"><a name="pone-0001242-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>An SDSA model for the formation of 3′ and 5′ patches.</span></strong></p><a id="article1.body1.sec3.sec1.fig7.caption1.p1" name="article1.body1.sec3.sec1.fig7.caption1.p1"></a><p>(A) A DSB occurs to the left of the marker. 5′ ends are resected, and one of the resulting 3′ ends invades a homologous duplex. DNA synthesis (dotted lines), strand displacement, and reannealing lead to heteroduplex-patch formation with new DNA in the 3′-ending strand. Only the 3′ end that leads to heteroduplex formation across from the marker is shown. (B) A DSB occurs to the right of the marker. 5′ end resection, strand invasion, DNA synthesis, and reannealing all occur as in A, but result in heteroduplex patch formation with new DNA in the 5′-ending strand. Again, only the 3′ end that leads to heteroduplex formation across from the marker is shown. Adapted from Allers and Lichten <a href="#pone.0001242-Allers1">[92]</a>.</p>
<span>doi:10.1371/journal.pone.0001242.g008</span></div><div class="figure" id="pone-0001242-g009"><div class="img"><a name="pone-0001242-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>A single-strand-assimilation model for the formation of 5′ and 3′ patches.</span></strong></p><a id="article1.body1.sec3.sec1.fig8.caption1.p1" name="article1.body1.sec3.sec1.fig8.caption1.p1"></a><p>RecBCD loads onto λ at a DSE, and degrades both strands of DNA into ssDNA fragments. When these fragments are present in a cell at the same time as a replicating λdv plasmid (with which some fragments have homology) some of the fragments may be assimilated into the plasmid during replication. Fragments may also form RecA-dependent paranemic joints with the plasmid, as depicted in Leung <em>et al</em> <a href="#pone.0001242-Leung1">[96]</a> before replication begins. Once a replication fork passes over the region of the joint, the ssDNA fragment will be incorporated into the newly synthesized DNA strand. Fragments that are assimilated across from the 18 bp insert marker (which is not present in the λ phage) will result in a heteroduplex patch at that site. This assimilation might occur with ssDNA from either DNA strand of λ being assimilated across from its complementary DNA in either the leading or lagging strand of a replication fork, creating heteroduplex patches of either polarity (drawn). Adapted from Leung <em>et al.</em> <a href="#pone.0001242-Leung1">[96]</a> and Ellis <em>et al.</em> <a href="#pone.0001242-Ellis1">[97]</a>; Court <em>et al.</em> <a href="#pone.0001242-Court1">[98]</a>. Alternatively, assimilation might occur preferentially into the lagging strand, as for Red-mediated recombination <a href="#pone.0001242-Leung1">[96]</a>–<a href="#pone.0001242-Court1">[98]</a> (not drawn), but because λdv has no replication terminus, replication might proceed in either direction <em>in vivo</em>, such that both strands are sometimes lagging strands. Thick blue lines represent DNA from the λ molecule, medium blue lines represent DNA from the λdv plasmid, thin blue lines represent newly synthesized λdv DNA, and pink lines represent the 18 bp insert marker. <em>ori</em> indicates the origin of replication for the λdv plasmid, but is not drawn to scale. RecBCD is depicted as in <a href="#pone-0001242-g001">Figure 1</a>, as a notched circle.</p>
<span>doi:10.1371/journal.pone.0001242.g009</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>In <a href="#pone-0001242-g002">Figure 2</a> we show that separation of these particular heteroduplexes can be achieved on polyacrylamide gels, and identified by Southern blotting, using artificial heteroduplexes made by melting and reannealing the DNAs with and without the insert. <a href="#pone-0001242-g002">Figure 2A</a> shows the migration of the two reannealed homoduplex restriction fragments, and the more slowly migrating heteroduplex species. The blot was then stripped and re-probed separately using oligonucleotide probes specific for the heteroduplex loop in either the 5′ or 3′ ending strand in each heteroduplex. <a href="#pone-0001242-g002">Figure 2B</a> shows that the most slowly migrating band contains the loop in the 5′ ending strand of the duplex (with respect to the right end of the fragment relative to the standard λ map), which will be referred to as heteroduplex (Het) I. Similarly, <a href="#pone-0001242-g002">Figure 2C</a> shows that the lower band of the two heteroduplex species (the middle band on the gel) contains the loop in the 3′ ending strand of the duplex, and will be referred to as Het II.</p>


<h4>Strategy for heteroduplex analysis <em>in vivo</em></h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Heteroduplex patches were generated <em>in vivo</em> by infecting <em>E. coli</em> strains carrying a λdv-derived plasmid with phage λ that lacks λ recombination functions <em>red</em>, <em>gam</em>, <em>orf</em>, and <em>rap</em> (<a href="#pone-0001242-g003">Figure 3</a>). The λ and λdv were allowed to recombine, after which plasmid DNA was isolated and analyzed for recombinant patches in the region carrying the insertion marker. (During the 3–4 hour incubation step during which recombination was allowed, multiple recombination events may happen to a λdv molecule. Replication of a λdv molecule across a heteroduplex patch would erase that patch, however new recombination events with another λ molecule would restore the heteroduplex region. Therefore, allowing replication would not lead exclusively to homoduplex patches.) Plasmid DNA was restriction digested to release a 604 bp fragment containing the heteroduplex. Digests were run on polyacrylamide gels to separate heteroduplexes of each polarity from one another and from homoduplexes, transferred to membranes, hybridized with a non-radioactively labeled probe complementary to the restriction fragment, and bands visualized with a chemiluminescent substrate and quantified.</p>


<h4>Formation of 5′ and 3′ patches <em>in vivo</em>, RecA and RecB dependently</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Several controls show that the heteroduplex patches observed here are formed <em>in vivo</em> and are products of RecBCD-mediated recombination. First, when λ is infected into a cell carrying λdv and the two molecules are allowed to recombine (referred to here as a cross), heteroduplexes of both polarities are obtained in an approximately 1:1 ratio for 5′ to 3′ patches (Het II:Het I, <a href="#pone-0001242-g004">Figure 4A</a>). This appears similar to the 2:1 ratio of 5′ to 3′ patches observed by Hagemann and Rosenberg <a href="#pone.0001242-Hagemann1">[11]</a>.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Second, mock crosses show that the heteroduplexes tested in the experiments reported here formed <em>in vivo</em>, and not as an artifact of DNA isolation. The <em>rec</em><sup>+</sup> strain carrying pLGR5 was treated as for a cross, but no λ phage were added to the culture. A similar <em>rec</em><sup>+</sup> strain lacking the plasmid was infected with the same λ phage used in the above crosses, and the two cultures were incubated as for a cross. These cultures were then mixed together and plasmid DNA prepared as for crosses. <a href="#pone-0001242-g004">Figure 4A</a> shows that no heteroduplex species are detected in three mock crosses. Thus, the heteroduplexes observed in λ by λdv crosses were formed <em>in vivo</em>.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Third, λ by λdv crosses were performed in strains lacking either RecB, which harbors the nuclease domain of RecBCD <a href="#pone.0001242-Wang1">[47]</a>, or RecA, the strand-exchange protein. Recombination of linear DNA is generally severely reduced in strains lacking either of these two proteins (reviewed by <a href="#pone.0001242-Kowalczykowski1">[1]</a>, <a href="#pone.0001242-Myers1">[2]</a>, <a href="#pone.0001242-Lusetti1">[48]</a>, <a href="#pone.0001242-Taylor2">[49]</a>). <a href="#pone-0001242-g004">Figures 4B and C</a> reveal that no recombinant heteroduplex patches are observed in crosses performed in strains lacking either RecA or RecB, respectively, but are observed in <em>rec</em><sup>+</sup> crosses. This reflects a &gt;99% decrease in the number of patches to λdv observed in crosses performed in a strain lacking <em>recA</em> as compared to <em>rec</em><sup>+</sup> on the blot, and a &gt;90% decrease in patches observed in a strain lacking <em>recB</em> (ratio of signal in patch area to DNA input). Additionally, frequencies of patch recombinants from crosses in these strains were quantified by plaque-blot hybridization in which λ phage isolated from these crosses were hybridized with a probe to the marker in the λdv plasmid. Because splices of the λdv into the λ create a λ molecule too large to be packaged, examination of λ DNA with plaque blots measures RecBCD-mediated patches only. <a href="#pone-0001242-t003">Table 3</a> shows that patches from λdv to λ are reduced as expected in strains lacking <em>recA</em> or <em>recB</em>. These data confirm that the heteroduplexes observed are the result of an <em>in vivo</em> RecABC-dependent recombination reaction.</p>
<div class="figure" id="pone-0001242-t003"><div class="img"><a name="pone-0001242-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001242" data-uri="info:doi/10.1371/journal.pone.0001242.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001242.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001242.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001242.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001242.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001242.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Patch frequencies measured by plaque hybridization</span></strong></p><span>doi:10.1371/journal.pone.0001242.t003</span></div>

<h4>λ Orf and Rap affect the frequency but not the polarity of heteroduplex patches</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Previous examination of heteroduplex patches formed during RecBCD-mediated recombination of phage λ DNA revealed patches in either the 5′- or 3′-ending strand, (with respect to the right end of phage λ onto which RecBCD loads <a href="#pone.0001242-Hagemann1">[11]</a>), and showed a roughly two-fold bias toward patches in the 5′-ending strand (the λ <em>r</em> strand) <a href="#pone.0001242-Hagemann1">[11]</a>. Since that study, two new λ recombination proteins were discovered. The λ <em>orf</em> gene product (open reading frame previously called <em>orf-146</em>) is an analog of and can substitute for <em>E. coli</em> RecFOR proteins <a href="#pone.0001242-Sawitzke1">[26]</a>, <a href="#pone.0001242-Maxwell1">[50]</a>–<a href="#pone.0001242-Sawitzke3">[52]</a>, which assist RecA loading <a href="#pone.0001242-Morimatsu1">[53]</a>. The λ <em>rap</em> gene product (open reading frame previously called <em>ninG</em>) encodes a Holliday-junction endonuclease that also cleaves D-loops <a href="#pone.0001242-Sharples1">[27]</a>, <a href="#pone.0001242-Poteete1">[54]</a>–<a href="#pone.0001242-Hollifield1">[56]</a>. Its homolog, <em>E. coli</em> Rus, substitutes for the Holliday-junction-cleavage activity of the <em>E. coli</em> RuvABC system <a href="#pone.0001242-Bolt1">[57]</a>, and can also substitute for human WRN <a href="#pone.0001242-Saintigny1">[58]</a> and fission yeast Rqh1 <a href="#pone.0001242-Doe1">[59]</a> recombination proteins. Either Orf or Rap or both of these proteins could have influenced the formation of patches examined previously, either by promoting formation of 5′-strand invasions (Orf), or by biasing Holliday-junction cleavage in favor of 5′ patches (Rap).</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>To address whether the prevalence of 5′ and 3′ patches seen previously was influenced by these λ recombination proteins, patch-strand polarity was re-examined in their absence (see previous section and results to follow). Precise deletions of <em>orf</em> and <em>rap</em> were made in the λ phage and the λ-derived plasmids used here, and crosses of λ by λdv were conducted as previously <a href="#pone.0001242-Hagemann1">[11]</a>. Frequencies of patches from λdv into λ were determined in the presence and absence of Orf and Rap, <em>via</em> plaque-blot hybridization (<a href="#pone-0001242-t003">Table 3</a>). The recombinant frequencies from <em>orf</em> <sup>+</sup> <em>rap</em><sup>+</sup> crosses were ~six-fold higher than those from <em>orf</em><sup>−</sup> <em>rap</em><sup>−</sup> crosses (<a href="#pone-0001242-t003">Table 3</a>). This is comparable to a two-fold decrease in patch frequency measured previously in λ by plasmid crosses when the λ was <em>rap</em><sup>−</sup> <a href="#pone.0001242-Stahl1">[60]</a>. When recombinant frequencies of splices were measured in similar crosses, decreases of 17-fold <a href="#pone.0001242-Stahl1">[60]</a> and 100-fold <a href="#pone.0001242-Hollifield1">[56]</a> were observed in <em>rap</em><sup>−</sup> λ, suggesting that <em>rap</em> has a greater influence on splices than on patches. To analyze heteroduplex-patch-strand polarity, plasmids were purified from these mixed infections and assayed on polyacrylamide gels (<a href="#pone-0001242-g005">Figure 5</a>). In the first of these crosses, the marker insert was in the λdv plasmid, and recombinant plasmid was isolated, such that Het I represents patching of wild-type DNA from the λ phage into the plasmid 3′ strand, creating a 3′ patch with a loop in the 5′-ending strand of the restriction fragment (<a href="#pone-0001242-g005">Figure 5A</a>, left). Conversely, Het II represents patching of DNA from the λ phage into the plasmid 5′ strand, creating a 5′ patch with a loop in the 3′-ending strand of the restriction fragment.</p>
<a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>The average of two crosses that were <em>orf</em> <sup>+</sup> <em>rap</em><sup>+</sup>, produced heteroduplex patches with a ratio of 5′ patches to 3′ patches of 1.0±0.2:1 (average±range; no bias toward one or the other) (<a href="#pone-0001242-g005">Figure 5B</a> lanes 3 and 5). This result is roughly similar to the previous report from similar crosses, in which the bias toward 5′ patches was roughly two-fold <a href="#pone.0001242-Hagemann1">[11]</a>. In both studies, both 5′ and 3′ patches are prevalent. Crosses in which <em>orf</em> and <em>rap</em> were deleted also show roughly equal numbers of 5′ and 3′ patches, and might show a very slight bias toward 5′ patches; the 5′ to 3′ patch ratio was 1.1±0.3:1 (average±S.D., n = 6) (<a href="#pone-0001242-g005">Figure 5C</a> lanes 4 and 5). These two ratios were not significantly different from each other (<em>P</em> = 0.874, by one way ANOVA).</p>
<a id="article1.body1.sec3.sec4.p4" name="article1.body1.sec3.sec4.p4"></a><p>Similar crosses were also performed with the marker in the λ instead of the λdv to ensure that any bias detected was not an artifact of marker placement. If some as yet undiscovered heteroduplex repair system in <em>E. coli</em> could repair 18 nt loops, and if it preferentially repaired either Het I or Het II, reversing the marker configuration would control for this by switching which patch type (<em>e.g.</em> 5′) is represented by which Het type (<em>e.g.</em> Het I), when the markers are reversed. In this configuration, Het I reflects 5′ patches of the marker insert from the λ into the plasmid, and Het II represents 3′ patches of the marker insert into the plasmid (<a href="#pone-0001242-g005">Figure 5A</a>, right). These <em>orf</em><sup>−</sup> <em>rap</em><sup>−</sup> crosses produced a 5′ to 3′ ratio of 0.9±0.1:1 (or 1:1.1, average±SD, n = 6) (<a href="#pone-0001242-g005">Figure 5C</a> lanes 6 and 7). Statistical analysis by one-way ANOVA indicated that the 1.1:1 ratio when the marker was in the plasmid was not significantly different from the 0.9:1 ratio when the marker was in the λ. That is, any possible bias was not strong enough to show a significant difference when the markers were swapped. For the <em>orf</em> <sup>+</sup> <em>rap</em><sup>+</sup> crosses, the ratio was 1.0±0.1:1 (average±range, n = 2) with the marker in the λ (<a href="#pone-0001242-g005">Figure 5B</a> lanes 4 and 6). A lack of significance is expected with an almost 1:1 ratio. These results indicate that the roughly 1:1 ratio is not caused by preferential repair of a particular Het type which would otherwise be much better represented due to a patch-strand bias. These results also indicate that although Orf and Rap increase patch frequency (<a href="#pone-0001242-t003">Table 3</a>), they do not contribute to the relative prevalence of both 5′ and 3′ patches.</p>


<h4>RecD influences patch-strand polarity</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>RecD is a subunit of the RecBCD enzyme that regulates recombination negatively, and is required for the strong exonuclease activity of RecBCD. Some models for recombination in <em>E. coli</em> depict RecBCD as preferentially degrading the 5′-ending strand after an encounter with Chi <a href="#pone.0001242-Kowalczykowski2">[8]</a>, <a href="#pone.0001242-Dixon1">[61]</a>, <a href="#pone.0001242-Anderson1">[62]</a>. RecBC(D<sup>−</sup>) is almost completely devoid of the exonuclease activity of wild-type RecBCD <em>in</em> vitro and <em>in vivo</em> <a href="#pone.0001242-Palas1">[3]</a>, <a href="#pone.0001242-Anderson2">[63]</a>–<a href="#pone.0001242-Myers2">[65]</a>, but possesses helicase activity <a href="#pone.0001242-Taylor1">[4]</a>, <a href="#pone.0001242-Korangy1">[66]</a>–<a href="#pone.0001242-Rinken1">[68]</a>. In addition, the crystal structure of RecBCD shows a 5′-ssDNA end passing through the RecD subunit and the 3′ strand through RecB <a href="#pone.0001242-Singleton1">[69]</a>. This suggested that a strain mutant for RecD might show a bias toward 5′ patches because 5′-ssDNA ends might not be degraded. Crosses performed in a <em>recD</em> mutant strain showed a slight yet significant bias toward 5′ patches (<a href="#pone-0001242-g006">Figure 6</a>), with a 5′:3′ patch ratio of 1.3±0.2:1 (mean±SD of four crosses). This ratio is significantly different from that for the <em>rec</em><sup>+</sup> controls run in parallel at 0.91±0.11:1 (<em>P</em> = 0.006, by one-way ANOVA). These results indicate that changes in the ratio of 5′:3′ patches could be detected using this assay, and also provide the first evidence of a cellular mutation that influences patch-strand polarity. The data further confirm that the patch-strand polarities reported are the result of recombination <em>in vivo via</em> the RecBCD-mediated DSBR pathway.</p>


<h4>Heteroduplex patches form independently of RuvC</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Most models for RecBCD-mediated recombination, including all shown in <a href="#pone-0001242-g001">Figure 1</a>, depict a Holliday-junction intermediate that requires endonucleolytic resolution to form recombinant products, either patches or splices. In <em>E. coli</em>, RuvAB and RecG catalyze branch migration of Holliday junctions, and endonucleolytic resolution of the junction is performed by RuvC <a href="#pone.0001242-Lloyd1">[70]</a>–<a href="#pone.0001242-Whitby1">[73]</a>. Models have been drawn that illustrate how the invasion of a 3′-ssDNA end could be resolved to form a heteroduplex on the 5′-ending strand, a 5′ patch, depending on the orientation in which RuvC resolves the junction <a href="#pone.0001242-Myers1">[2]</a>, <a href="#pone.0001242-Rosenberg1">[9]</a>, <a href="#pone.0001242-Hagemann1">[11]</a>, <a href="#pone.0001242-Siddiqi1">[22]</a>, <a href="#pone.0001242-Harris1">[74]</a>. There has also been speculation that <em>E. coli</em> topoisomerase III (TopB) might act to resolve Holliday junctions non-nucleolytically <a href="#pone.0001242-Thaler1">[75]</a>, and evidence that it acts in a pathway alternative to RuvC <a href="#pone.0001242-Lopez1">[76]</a>. Additionally, two pathways of RecBCD-mediated recombination have been demonstrated: a “break-join” route that requires the RuvABC Holliday-junction endonuclease system, and a replicative or “break-copy” resolution route that operates independently of RuvABC and RecG <a href="#pone.0001242-Motamedi1">[77]</a>, <a href="#pone.0001242-Magner1">[78]</a>. The latter pathway may operate independently of Holliday junctions <a href="#pone.0001242-Magner1">[78]</a>.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>To test whether patches, and their strand polarity, result from Holliday-junction resolution, λ by λdv crosses were performed in Δ<em>ruvC</em> or Δ<em>topB</em> deletion strains. Neither showed significantly different ratios of 5′ to 3′ patches (<a href="#pone-0001242-g007">Figure 7</a>, and <a href="#pone-0001242-g004">Figure 4B</a>, respectively) from the wild-type strain. The data from two experiments each yielded ratios as follows: <em>rec</em><sup>+</sup> showed a 0.7±0.5:1 ratio (mean±range, lower than in previous experiments), Δ<em>ruvC</em> showed a 0.9±0.2:1 ratio. These ratios are slightly lower than those seen in previous experiments, but are not significantly different from each other (<em>P</em> = 0.667, by one way ANOVA). In two separate experiments, Δ<em>topB</em> showed a ratio of 1.1±0.2:1 (mean±range), whereas the <em>rec</em><sup>+</sup> strain analyzed in parallel showed a ratio of 0.98±0.12:1. Moreover, surprisingly, patch frequencies show that patches are not dependent on the presence of either RuvC or TopB (<a href="#pone-0001242-t003">Table 3</a>). These data imply that processing of Holliday junctions by RuvC or Topo III is not required for the formation of patches, and suggest that <em>none</em> of the three models presented in <a href="#pone-0001242-g001">Figure 1</a> is likely to bear on patch formation. We shall suggest below that Holliday junctions might not be intermediates in patch formation, and that patches are not derived from the same intermediate(s) as splices. We will propose two different models for the formation of patches, models in which patches form by a fundamentally different mechanism from that leading to splices.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>This study employed physical analyses of heteroduplex DNAs using a loop-strand conformation to separate, on polyacrylamide gels, heteroduplex DNAs with an 18 nt loop in either the 5′ or 3′ strand. This method was used to analyze patches formed <em>in vivo</em> by RecBCD-mediated DSE-repair recombination of phage λ with the λ-homologous plasmid, λdv, and analyzed patches inserted into the plasmid. Like a previous study of RecBCD-mediated patches analyzed by a different physical method–partially denaturing gel electrophoresis <a href="#pone.0001242-Hagemann1">[11]</a>–a prevalence of 5′ and 3′ patches was observed (polarity relative to the DSE where RecBCD loaded, in λ, the λ right end <a href="#pone.0001242-Myers1">[2]</a>). The results presented here support their conclusion using an independent method, and extend them in the following ways. First, λ Orf and Rap recombination proteins, which were present in previous experiments <a href="#pone.0001242-Hagemann1">[11]</a>, do not contribute to the relative prevalence of 5′ and 3′ patches (<a href="#pone-0001242-g005">Figure 5</a>), even though they do stimulate recombination efficiency overall (<a href="#pone-0001242-t003">Table 3</a>, and <a href="#pone.0001242-Sawitzke1">[26]</a>, <a href="#pone.0001242-Hollifield1">[56]</a>, <a href="#pone.0001242-Stahl1">[60]</a>). Second, there is a small but statistically significant shift toward more 5′ patches in <em>recD</em> mutants (<a href="#pone-0001242-g006">Figure 6</a>). Third, RuvC Holliday-junction endonuclease, which is required for break-join resolution of RecBC-mediated DSE repair, which reflects Holliday-junction cleavage <em>in vivo</em> <a href="#pone.0001242-Motamedi1">[77]</a>, <a href="#pone.0001242-Magner1">[78]</a>, is not required for patch formation (<a href="#pone-0001242-g007">Figure 7</a>, and <a href="#pone-0001242-t003">Table 3</a>). This implies that patches do not result from endonucleolytic resolution of Holliday junctions, as about half of splices do. Similarly, this implies that none of the previous models in <a href="#pone-0001242-g001">Figure 1</a> will account for patch formation. The <em>E</em>. <em>coli</em> topoisomerase III (TopB) also has no effect on heteroduplex-patch polarity or patch formation (<a href="#pone-0001242-g004">Figure 4B</a>, and <a href="#pone-0001242-t003">Table 3</a>).</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Other genetic <a href="#pone.0001242-Rosenberg1">[9]</a>, <a href="#pone.0001242-Rosenberg2">[10]</a>, <a href="#pone.0001242-Siddiqi1">[22]</a>, <a href="#pone.0001242-White1">[24]</a>, <a href="#pone.0001242-Nag1">[25]</a> and physical <a href="#pone.0001242-Hagemann1">[11]</a>, <a href="#pone.0001242-Holbeck1">[23]</a>, <a href="#pone.0001242-Silberstein1">[29]</a>, <a href="#pone.0001242-FriedmanOhana1">[79]</a>, <a href="#pone.0001242-Lichten1">[80]</a> methods for heteroduplex analysis have been used to examine heteroduplex strand polarity of recombinants to elucidate the molecular mechanisms occurring at various steps of recombination. In one physical analysis of RecBCD-mediated splices, the linear DNA substrate was created by <em>in vivo</em> restriction digestion of a λ phage to release a linear molecule containing direct terminal repeats <a href="#pone.0001242-Silberstein1">[29]</a>, <a href="#pone.0001242-FriedmanOhana1">[79]</a>. This substrate was then inferred to undergo intramolecular recombination, leading to heteroduplex recombinant formation initiated by pairing and strand exchange of the 3′-ending strand as inferred from the polarity of the splice junctions. A concern in interpreting the results, however, is that the same recombinants would be formed by single-strand annealing (SSA) of the direct repeats after resection of the 5′ ends. RecBCD-mediated recombination is unlikely to proceed <em>via</em> an SSA mechanism in most biologically relevant contexts in which it has been studied (<em>e.g.</em>; conjugation, phage-mediated transduction, and DSE repair <a href="#pone.0001242-Dower1">[81]</a>–<a href="#pone.0001242-Lloyd2">[83]</a>) and in model recombination studies using λ <a href="#pone.0001242-Smith3">[84]</a>, because in all of these circumstances a single DSE recombines with an unbroken molecule; that is, these are one-ended reactions. The heteroduplex studies mentioned <a href="#pone.0001242-Silberstein1">[29]</a>, <a href="#pone.0001242-FriedmanOhana1">[79]</a> were unusual in employing a substrate with two complementary DNA ends that could produce recombinants by annealing. Thus, it is unclear whether those results apply to most other RecBCD-mediated recombination.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>One of the genetic methods <a href="#pone.0001242-Siddiqi1">[22]</a> used to examine patch polarity found that the recombinant heteroduplex was formed predominantly in the 3′-ending strand, heteroduplex material having “3′ overhangs” in the recombinant molecules. This work examined only splice recombinants, and is compatible with splices formed <em>via</em>, <em>e.g.</em>, the model shown in <a href="#pone-0001242-g001">Figure 1C</a> <a href="#pone.0001242-Kowalczykowski2">[8]</a>, though this model does not explain why 5′ <em>or</em> 3′ exonucleases are required for splice formation <a href="#pone.0001242-Razavy1">[12]</a>, as the split-end model does (<a href="#pone-0001242-g001">Figure 1B</a>).</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Another method examined heteroduplex strand polarity using partially denaturing gel electrophoresis of heteroduplexes containing five mispairs <a href="#pone.0001242-Hagemann1">[11]</a>. This system, though useful, has been difficult to work with, and required the use of special strains lacking the mismatch repair system to avoid unwanted repair of the heteroduplexes being examined.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The system for heteroduplex analysis described here utilizes several commonly performed methods, with a few modifications, to create, isolate, and detect heteroduplexes formed <em>in vivo</em>. This method can be used with many different strain backgrounds to test effects on recombination between λ and λdv.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Several models for recombination depict only 3′ ends invading a homologous duplex, which predicts the formation of 3′ patches only (<a href="#pone-0001242-g001">Figure 1A and C</a>, and <a href="#pone.0001242-Smith1">[6]</a>–<a href="#pone.0001242-Kowalczykowski2">[8]</a>, <a href="#pone.0001242-Dixon1">[61]</a>, <a href="#pone.0001242-Anderson1">[62]</a>). However, these models failed to account for existing data showing the presence of two different forms of heteroduplex, 5′ patches and 3′ patches <a href="#pone.0001242-Rosenberg1">[9]</a>–<a href="#pone.0001242-Hagemann1">[11]</a>. Two possible general hypotheses can account for the prevalence of both 5′ and 3′ patches. First, either 5′ or 3′ ends might be able to invade a homolog <em>in vivo</em> <a href="#pone.0001242-Rosenberg3">[21]</a>, or second, possibly, patch-strand polarity might reflect an entirely different aspect of recombination (suggested below). Evidence that 5′ ends are able to invade a homologous duplex to form D-loops <em>in vitro</em> has been well documented <a href="#pone.0001242-Dutreix1">[14]</a>, <a href="#pone.0001242-McIlwraith1">[15]</a>. The discovery of <em>orf</em> and <em>rap</em> in λ, and their potential roles in the recombination mechanism being studied (discussed above), may have led to the assumption that they contributed to the <em>in vivo</em> data demonstrating the presence of 5′ and 3′ patches. The results presented here provide evidence that those gene products did not contribute to the presence of 5′ patches seen in previous studies, and imply that 5′ patches, as well as 3′ patches, result from some aspect of RecBCD-mediated recombination.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>One possibility for the source of the two different patch polarities involves the exonuclease activity of RecBCD, and the lack of substantive activity after interaction with Chi, or when the strain is RecD<sup>−</sup> <a href="#pone.0001242-Myers1">[2]</a>, <a href="#pone.0001242-Palas1">[3]</a>, <a href="#pone.0001242-Dabert1">[64]</a>, <a href="#pone.0001242-Thaler2">[85]</a>, <a href="#pone.0001242-Koppen1">[86]</a>. The helicase activity of RecBC could generate single-stranded DNA ends of both polarities, either of which could invade a homologous duplex and create a heteroduplex patch of one polarity or the other <a href="#pone.0001242-Rosenberg3">[21]</a> (<a href="#pone-0001242-g001">Figure 1B</a>). One might predict that a strain mutant for RecD might show an increase in 5′ patches relative to 3′ patches because 5′ ends would not be degraded, and a slight yet significant increase in 5′ patches is seen in <a href="#pone-0001242-g006">Figure 6</a>. In addition, strains mutant for RecD have been shown to produce linear multimers of plasmids because the lack of dsDNA nuclease activity allows rolling-circle replication <a href="#pone.0001242-Biek1">[87]</a>. This might also affect the increase in 5′ patches observed here. This model appears to require endonucleolytic cleavage of Holliday junctions, and though it is still possible for generating splices, it seems unlikely to be relevant to patches, which form independently of RuvC (<a href="#pone-0001242-g007">Figure 7</a>, <a href="#pone-0001242-t003">Table 3</a>).</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>The lack of dependence on RuvC for patch formation is an important finding. RuvC is the sole demonstrated HJ-cleavage activity present in wild-type <em>E. coli</em>. RuvABC are also necessary for break-join RecBCD-mediated splice recombination <em>in vivo</em>, which is thought to reflect those recombination events resolved <em>via</em> endonucleolytic cleavage <a href="#pone.0001242-Motamedi1">[77]</a>, <a href="#pone.0001242-Magner1">[78]</a>. The inability of other proteins to substitute for RuvABC in break-join recombination suggests that Ruv is the only endonucleolytic resolution pathway available to RecBCD-mediated recombination <a href="#pone.0001242-Magner1">[78]</a>. Thus, the independence of patch formation on RuvC suggests to us that endonucleolytic resolution of Holliday junctions is not required for patch formation. Most recombination models, including all of those considered previously (<a href="#pone-0001242-g001">Figure 1A–C</a>), predict that RuvC should be required for patch formation. <em>ruvC</em> mutants are not completely deficient for splice recombination, but exhibit a recombinant frequency that is decreased by half <a href="#pone.0001242-Motamedi1">[77]</a>, <a href="#pone.0001242-Magner1">[78]</a>, <a href="#pone.0001242-Lloyd3">[88]</a>; <em>ruvC</em> removes “break-join” splice recombinants specifically and does not alter splice recombinants formed <em>via</em> the “break-copy” or BIR (break-induced replication) route <a href="#pone.0001242-Motamedi1">[77]</a>, <a href="#pone.0001242-Magner1">[78]</a>. The remaining break-copy recombinant products require the major replicative polymerase, Pol III, of <em>E. coli</em>, and were formed only when replication was allowed. The recombinant products in those assays were splices. Recombinant frequencies for the patches examined here for a Δ<em>ruvC</em> strain were found not to be different from <em>rec</em><sup>+</sup> (<a href="#pone-0001242-t003">Table 3</a>).</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p>Given the surprising lack of dependence of heteroduplex-patch formation on RuvC, and the expected dependence on RecB and RecA, what might be the origin of heteroduplex patches of both polarities? The presence of patches in the absence of RuvC suggests the possibility that a Holliday junction might not be involved in the formation of the heteroduplexes observed here. Additionally, a strictly replicative single-strand gap-filling type of mechanism would result in homoduplex patches, heteroduplex loops would not be observed. Two possible mechanisms that use replication rather than endonuclease to complete patch formation are suggested here.</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10"></a><p>One possible mechanism for the generation of the heteroduplex patches observed here is synthesis-dependent-strand annealing (SDSA) <a href="#pone.0001242-Paques1">[89]</a>. SDSA was first described in Drosophila <a href="#pone.0001242-Gloor1">[90]</a>, <a href="#pone.0001242-Nassif1">[91]</a>, and does not require Holliday-junction-endonucleolytic cleavage for the formation of recombinants. It does, however, require processing of a DNA break, and strand invasion, which could occur <em>via</em> RecBCD and RecA, respectively, in <em>E</em>. <em>coli</em>. In yeast, Allers and Lichten <a href="#pone.0001242-Allers1">[92]</a> showed that patch formation is differentially timed and patches appear in advance of splices (which require Holliday-junction resolution). This led them to suggest that SDSA produces patched (non-crossover) products primarily, whereas the double-strand break repair mechanism of Szostak <em>et al</em>. <a href="#pone.0001242-Szostak1">[93]</a>, which includes HJ cleavage, leads to patches and splices (crossovers).</p>
<a id="article1.body1.sec4.p11" name="article1.body1.sec4.p11"></a><p>How might both 5′ and 3′ patches be formed <em>via</em> SDSA? One possibility is that resection of the 5′ ends at a double-strand break (DSB) leaves 3′-ssDNA ends to invade a homologous duplex (<a href="#pone-0001242-g008">Figure 8</a>). One could imagine that the 3′ end on either side of the DSB could be the invading end, which would result in either a 3′ patch or a 5′ patch (with respect to the right end of the molecule) depending on which 3′ end invaded. When examining a particular locus, such as <em>ren</em>, if the break occurred to the left of the marker, a 3′ patch might result, and if the break occurred to the right of the marker, a 5′ patch might result. This model would produce both 5′ and 3′ patches using strand invasions of a single (3′) polarity.</p>
<a id="article1.body1.sec4.p12" name="article1.body1.sec4.p12"></a><p>SDSA could also explain previous data reporting the presence of 3′ and 5′ patches <a href="#pone.0001242-Rosenberg1">[9]</a>–<a href="#pone.0001242-Hagemann1">[11]</a>. The approximately 1:1 ratio observed here is not very different from the approximately 2:1 ratio reported previously <a href="#pone.0001242-Hagemann1">[11]</a>, and might be attributable to the difference between the two methods used for heteroduplex analysis. The more substantial 9:1 ratio reported for a marker in the <em>P</em> gene of lambda <a href="#pone.0001242-Rosenberg1">[9]</a> can also fit the SDSA model as described similarly for the split-end model <a href="#pone.0001242-Rosenberg3">[21]</a>. The <em>P</em> gene of lambda is very close to the origin of replication (<em>ori</em>) and perhaps as with phage T4 <a href="#pone.0001242-Stitt1">[94]</a>, replication is abandoned more frequently near <em>P</em>, causing breaks more often that would need to be repaired. A break would be processed by RecBCD to create 3′ ends which prime synthesis from a homologous DNA molecule. The newly synthesized DNA would create a heteroduplex patch. Perhaps there is a site just to the right of <em>P</em> that causes replication fork stalling and breakage such that the template strand of the leading strand, or the nascent strand of the lagging strand, often becomes the invading strand. This would bias patches toward the 5′-ending strand.</p>
<a id="article1.body1.sec4.p13" name="article1.body1.sec4.p13"></a><p>Although SDSA has not previously been considered as a recombination mechanism in <em>E. coli</em>, it has been studied for years in yeast and Drosophila, and also underlies genome reassembly following high-level double-strand breakage in <em>Deinococcus radiodurans</em> <a href="#pone.0001242-Zahradka1">[95]</a>. Further work should be able to define any role of SDSA in <em>E. coli</em> recombination more clearly.</p>
<a id="article1.body1.sec4.p14" name="article1.body1.sec4.p14"></a><p>A significant weakness of the SDSA model is the requirement that DSBs originate in the plasmid. We hypothesize that these might occur at some low frequency due to collapse of stalled replication forks, but we have not shown that DSBs are formed. Nevertheless, we cannot disprove that plasmid-bourne DSBs are formed leading to DSEs that are repaired <em>via</em> SDSA.</p>
<a id="article1.body1.sec4.p15" name="article1.body1.sec4.p15"></a><p>A more appealing alternative model that does not require DSBs to be formed in the plasmid is shown in <a href="#pone-0001242-g009">Figure 9</a>. This model is based on single-strand assimilation models of Leung <em>et al.</em> <a href="#pone.0001242-Leung1">[96]</a>, and Ellis <em>et al.</em> <a href="#pone.0001242-Ellis1">[97]</a>; Court <em>et al.</em> <a href="#pone.0001242-Court1">[98]</a>, for yeast and phage λ Red-mediated recombination, respectively. The model in <a href="#pone-0001242-g009">Figure 9</a> suggests that pieces of ssDNA that are released from the λ chromosome by the nuclease activity of RecBCD are assimilated into one or the other DNA strand of the plasmid during replication. ssDNA fragments that anneal across from the 18 bp insert will create heteroduplex DNA at that site. This model would require RecBCD and RecA, both of which are required for heteroduplex formation observed here. ssDNA oligonucleotides have been used to create mutations by gene targeting in yeast using Rad51, Rad52, and Rad59 annealing activities <a href="#pone.0001242-Storici1">[99]</a>, <a href="#pone.0001242-Storici2">[100]</a> and in <em>E. coli</em> using the phage λ Red beta protein <a href="#pone.0001242-Ellis1">[97]</a>, thus demonstrating the plausibility of this model.</p>
<a id="article1.body1.sec4.p16" name="article1.body1.sec4.p16"></a><p>One apparent difference between our results with RecABC and those of Ellis <em>et al.</em> for Red-mediated recombination, is that whereas we see <a href="#s3">results</a> consistent with equal incorporation of DNA fragments into either DNA strand, the Red system shows a bias toward incorporations of ss-DNA oligonucleotides into the lagging strand, though this preference is not absolute <a href="#pone.0001242-Ellis1">[97]</a>. This difference might reflect the fact that whereas Red uses a single-strand-annealing protein, beta, the RecABC-dependent recombination studied here uses RecA, which can catalyze both strand invasion and annealing. This might allow single-strand assimilations into the leading strand, where gaps are expected to be less frequent. These assimilations would still require replication and attachment to a new strand during synthesis for completion (per <a href="#pone-0001242-g009">Figure 9</a>). Alternatively, perhaps the RecBCD system also inserts patches preferentially into the lagging strand, but perhaps we see equal numbers of heteroduplex patches in both strands because of the difference between the <em>E. coli</em> and λdv replicons. Unlike the bacterial chromosome, λdv does not have a terminus of replication with replication pause sites that block replication forks that go past the “terminus”. Thus, it is possible that replication forks pass across the marker site in both directions, making both strands the lagging strand at some point in the reaction which results not in a bias, but in an equal number of patches in both strands.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank members of the Rosenberg laboratory, Phil Hastings, John Wilson, George Weinstock, and Jim Lupski for helpful discussions, and in particular Greg Ira for suggesting the single-strand assimilation model as an alternative; Phil Hastings, Greg Ira, and Jim Lupski for comments on the manuscript, and Jan Gibson for excellent technical advice.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SR LG. Performed the experiments: LG. Analyzed the data: SR LG. Wrote the paper: SR LG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001242-Kowalczykowski1" id="pone.0001242-Kowalczykowski1"></a>Kowalczykowski SC, Dixon DA, Eggleston AK, Lauder SD, Rehrauer WM (1994) Biochemistry of homologous recombination in <em>Escherichia coli</em>. Microbiol Rev  58: 401–465.  <ul class="find" data-citedArticleID="1023900"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Biochemistry+of+homologous+recombination+in+Escherichia+coli.&amp;auth=&amp;atitle=Biochemistry+of+homologous+recombination+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemistry+of+homologous+recombination+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemistry+of+homologous+recombination+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001242-Myers1" id="pone.0001242-Myers1"></a>Myers RS, Stahl FW (1994) χ and RecBCD enzyme of <em>Escherichia coli</em>. Annu Rev Genet  28: 49–70.  <ul class="find" data-citedArticleID="1023944" data-doi="10.1146/annurev.ge.28.120194.000405"><li><a href="http://dx.doi.org/10.1146/annurev.ge.28.120194.000405" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%CF%87+and+RecBCD+enzyme+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%CF%87+and+RecBCD+enzyme+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001242-Palas1" id="pone.0001242-Palas1"></a>Palas KM, Kushner SR (1990) Biochemical and physical characterization of exonuclease V from <em>Escherichia coli</em>. Comparison of the catalytic activities of the RecBC and RecBCD enzymes. J Biol Chem  265: 3447–3454.  <ul class="find" data-citedArticleID="1023954"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Biochemical+and+physical+characterization+of+exonuclease+V+from+Escherichia+coli.+Comparison+of+the+catalytic+activities+of+the+RecBC+and+RecBCD+enzymes.&amp;auth=&amp;atitle=Biochemical+and+physical+characterization+of+exonuclease+V+from+Escherichia+coli.+Comparison+of+the+catalytic+activities+of+the+RecBC+and+RecBCD+enzymes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+and+physical+characterization+of+exonuclease+V+from+Escherichia+coli.+Comparison+of+the+catalytic+activities+of+the+RecBC+and+RecBCD+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+and+physical+characterization+of+exonuclease+V+from+Escherichia+coli.+Comparison+of+the+catalytic+activities+of+the+RecBC+and+RecBCD+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001242-Taylor1" id="pone.0001242-Taylor1"></a>Taylor AF, Smith GR (1985) Substrate specificity of the DNA unwinding activity of the RecBC enzyme of <em>Escherichia coli</em>. J Mol Biol  185: 431–443.  <ul class="find" data-citedArticleID="1024022" data-doi="10.1016/0022-2836(85)90414-0"><li><a href="http://dx.doi.org/10.1016/0022-2836(85)90414-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Substrate+specificity+of+the+DNA+unwinding+activity+of+the+RecBC+enzyme+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Substrate+specificity+of+the+DNA+unwinding+activity+of+the+RecBC+enzyme+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001242-Ganesan1" id="pone.0001242-Ganesan1"></a>Ganesan S, Smith GR (1993) Strand-specific binding to duplex DNA ends by the subunits of the <em>Escherichia coli</em> RecBCD enzyme. J Mol Biol  229: 67–78.  <ul class="find" data-citedArticleID="1023870" data-doi="10.1006/jmbi.1993.1008"><li><a href="http://dx.doi.org/10.1006/jmbi.1993.1008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strand-specific+binding+to+duplex+DNA+ends+by+the+subunits+of+the+Escherichia+coli+RecBCD+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strand-specific+binding+to+duplex+DNA+ends+by+the+subunits+of+the+Escherichia+coli+RecBCD+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001242-Smith1" id="pone.0001242-Smith1"></a>Smith GR, Amundsen SK, Chaudhury AM, Cheng KC, Ponticelli AS, et al.  (1984) Roles of RecBC enzyme and Chi sites in homologous recombination. Cold Spring Harbor Symp Quant Biol  49: 485–495.  <ul class="find" data-citedArticleID="1024004" data-doi="10.1101/sqb.1984.049.01.055"><li><a href="http://dx.doi.org/10.1101/sqb.1984.049.01.055" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Roles+of+RecBC+enzyme+and+Chi+sites+in+homologous+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Roles+of+RecBC+enzyme+and+Chi+sites+in+homologous+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001242-Smith2" id="pone.0001242-Smith2"></a>Smith GR, Schultz DW, Taylor AF, Triman K (1981) Chi sites, RecBC enzyme, and generalized recombination. Stadler Genet Symp  13: 25–37.  <ul class="find" data-citedArticleID="1024006"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chi+sites%2C+RecBC+enzyme%2C+and+generalized+recombination.&amp;auth=&amp;atitle=Chi+sites%2C+RecBC+enzyme%2C+and+generalized+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chi+sites%2C+RecBC+enzyme%2C+and+generalized+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chi+sites%2C+RecBC+enzyme%2C+and+generalized+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001242-Kowalczykowski2" id="pone.0001242-Kowalczykowski2"></a>Kowalczykowski SC (2000) Initiation of genetic recombination and recombination-dependent replication. Trends Biochem Sci  25: 156–165.  <ul class="find" data-citedArticleID="1023902" data-doi="10.1016/s0968-0004(00)01569-3"><li><a href="http://dx.doi.org/10.1016/s0968-0004(00)01569-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initiation+of+genetic+recombination+and+recombination-dependent+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initiation+of+genetic+recombination+and+recombination-dependent+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001242-Rosenberg1" id="pone.0001242-Rosenberg1"></a>Rosenberg SM (1987) Chi stimulated patches are heteroduplex, with recombinant information on the phage λ <em>r</em> chain. Cell  48: 855–865.  <ul class="find" data-citedArticleID="1023970" data-doi="10.1016/0092-8674(87)90082-1"><li><a href="http://dx.doi.org/10.1016/0092-8674(87)90082-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chi+stimulated+patches+are+heteroduplex%2C+with+recombinant+information+on+the+phage+%CE%BB+r+chain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chi+stimulated+patches+are+heteroduplex%2C+with+recombinant+information+on+the+phage+%CE%BB+r+chain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001242-Rosenberg2" id="pone.0001242-Rosenberg2"></a>Rosenberg SM (1988) Chain-bias of <em>Escherichia coli</em> Rec-mediated λ patch recombinants is independent of the orientation of λ <em>cos</em>. Genetics  120: 7–21.  <ul class="find" data-citedArticleID="1023972"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chain-bias+of+Escherichia+coli+Rec-mediated+%3F+patch+recombinants+is+independent+of+the+orientation+of+%3F+cos.&amp;auth=&amp;atitle=Chain-bias+of+Escherichia+coli+Rec-mediated+%3F+patch+recombinants+is+independent+of+the+orientation+of+%3F+cos." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chain-bias+of+Escherichia+coli+Rec-mediated+%CE%BB+patch+recombinants+is+independent+of+the+orientation+of+%CE%BB+cos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chain-bias+of+Escherichia+coli+Rec-mediated+%CE%BB+patch+recombinants+is+independent+of+the+orientation+of+%CE%BB+cos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001242-Hagemann1" id="pone.0001242-Hagemann1"></a>Hagemann AT, Rosenberg SM (1991) Chain bias in Chi-stimulated heteroduplex patches in the λ <em>ren</em> gene is determined by the orientation of λ <em>cos</em>. Genetics  129: 611–621.  <ul class="find" data-citedArticleID="1023878"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chain+bias+in+Chi-stimulated+heteroduplex+patches+in+the+%3F+ren+gene+is+determined+by+the+orientation+of+%3F+cos.&amp;auth=&amp;atitle=Chain+bias+in+Chi-stimulated+heteroduplex+patches+in+the+%3F+ren+gene+is+determined+by+the+orientation+of+%3F+cos." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chain+bias+in+Chi-stimulated+heteroduplex+patches+in+the+%CE%BB+ren+gene+is+determined+by+the+orientation+of+%CE%BB+cos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chain+bias+in+Chi-stimulated+heteroduplex+patches+in+the+%CE%BB+ren+gene+is+determined+by+the+orientation+of+%CE%BB+cos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001242-Razavy1" id="pone.0001242-Razavy1"></a>Razavy H, Szigety SK, Rosenberg SM (1996) Evidence for both 3′ and 5′ single-strand DNA ends in intermediates in Chi-stimulated recombination <em>in vivo</em>. Genetics  142: 333–339.  <ul class="find" data-citedArticleID="1023964"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evidence+for+both+3%3F+and+5%3F+single-strand+DNA+ends+in+intermediates+in+Chi-stimulated+recombination+in+vivo.&amp;auth=&amp;atitle=Evidence+for+both+3%3F+and+5%3F+single-strand+DNA+ends+in+intermediates+in+Chi-stimulated+recombination+in+vivo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+both+3%E2%80%B2+and+5%E2%80%B2+single-strand+DNA+ends+in+intermediates+in+Chi-stimulated+recombination+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+both+3%E2%80%B2+and+5%E2%80%B2+single-strand+DNA+ends+in+intermediates+in+Chi-stimulated+recombination+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001242-Miesel1" id="pone.0001242-Miesel1"></a>Miesel L, Roth JR (1996) Evidence that SbcB and RecF pathway functions contribute to RecBCD-dependent transductional recombination. J Bacteriol  178: 3146–3155.  <ul class="find" data-citedArticleID="1023932"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evidence+that+SbcB+and+RecF+pathway+functions+contribute+to+RecBCD-dependent+transductional+recombination.&amp;auth=&amp;atitle=Evidence+that+SbcB+and+RecF+pathway+functions+contribute+to+RecBCD-dependent+transductional+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+that+SbcB+and+RecF+pathway+functions+contribute+to+RecBCD-dependent+transductional+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+that+SbcB+and+RecF+pathway+functions+contribute+to+RecBCD-dependent+transductional+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001242-Dutreix1" id="pone.0001242-Dutreix1"></a>Dutreix M, Rao BJ, Radding CM (1991) The effects on strand exchange of 5′ <em>versus</em> 3′ ends of single-stranded DNA in RecA Nucleoprotein filaments. J Mol Biol  219: 645–654.  <ul class="find" data-citedArticleID="1023862" data-doi="10.1016/0022-2836(91)90661-o"><li><a href="http://dx.doi.org/10.1016/0022-2836(91)90661-o" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effects+on+strand+exchange+of+5%E2%80%B2+versus+3%E2%80%B2+ends+of+single-stranded+DNA+in+RecA+Nucleoprotein+filaments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effects+on+strand+exchange+of+5%E2%80%B2+versus+3%E2%80%B2+ends+of+single-stranded+DNA+in+RecA+Nucleoprotein+filaments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001242-McIlwraith1" id="pone.0001242-McIlwraith1"></a>McIlwraith MJ, West SC (2001) The efficiency of strand invasion by <em>Escherichia coli</em> RecA is dependent upon the length and polarity of ssDNA tails. J Mol Biol  305: 23–31.  <ul class="find" data-citedArticleID="1023930" data-doi="10.1006/jmbi.2000.4268"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4268" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+efficiency+of+strand+invasion+by+Escherichia+coli+RecA+is+dependent+upon+the+length+and+polarity+of+ssDNA+tails." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+efficiency+of+strand+invasion+by+Escherichia+coli+RecA+is+dependent+upon+the+length+and+polarity+of+ssDNA+tails.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001242-Register1" id="pone.0001242-Register1"></a>Register JC 3rd, Griffith J (1985) The direction of RecA protein assembly onto single strand DNA is the same as the direction of strand assimilation during strand exchange. J Biol Chem  260: 12308–12312.  <ul class="find" data-citedArticleID="1023966"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+direction+of+RecA+protein+assembly+onto+single+strand+DNA+is+the+same+as+the+direction+of+strand+assimilation+during+strand+exchange.&amp;auth=&amp;atitle=The+direction+of+RecA+protein+assembly+onto+single+strand+DNA+is+the+same+as+the+direction+of+strand+assimilation+during+strand+exchange." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+direction+of+RecA+protein+assembly+onto+single+strand+DNA+is+the+same+as+the+direction+of+strand+assimilation+during+strand+exchange." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+direction+of+RecA+protein+assembly+onto+single+strand+DNA+is+the+same+as+the+direction+of+strand+assimilation+during+strand+exchange.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001242-Konforti1" id="pone.0001242-Konforti1"></a>Konforti BB, Davis RW (1990) The preference for a 3′ homologous end is intrinsic to RecA-promoted strand exchange. J Biol Chem  265: 6916–6920.  <ul class="find" data-citedArticleID="1023890"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+preference+for+a+3%3F+homologous+end+is+intrinsic+to+RecA-promoted+strand+exchange.&amp;auth=&amp;atitle=The+preference+for+a+3%3F+homologous+end+is+intrinsic+to+RecA-promoted+strand+exchange." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+preference+for+a+3%E2%80%B2+homologous+end+is+intrinsic+to+RecA-promoted+strand+exchange." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+preference+for+a+3%E2%80%B2+homologous+end+is+intrinsic+to+RecA-promoted+strand+exchange.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001242-Konforti2" id="pone.0001242-Konforti2"></a>Konforti BB, Davis RW (1987) 3′ homologous free ends are required for stable joint molecule formation by the RecA and single-stranded binding proteins of <em>Escherichia coli</em>. Proc Natl Acad Sci U S A  84: 690–694.  <ul class="find" data-citedArticleID="1023892" data-doi="10.1073/pnas.84.3.690"><li><a href="http://dx.doi.org/10.1073/pnas.84.3.690" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=3%E2%80%B2+homologous+free+ends+are+required+for+stable+joint+molecule+formation+by+the+RecA+and+single-stranded+binding+proteins+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%223%E2%80%B2+homologous+free+ends+are+required+for+stable+joint+molecule+formation+by+the+RecA+and+single-stranded+binding+proteins+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001242-Shan1" id="pone.0001242-Shan1"></a>Shan Q, Bork JM, Webb BL, Inman RB, Cox MM (1997) RecA protein filaments: end-dependent disassociation from ssDNA and stabilization by RecO and RecR proteins. J Mol Biol  265: 519–540.  <ul class="find" data-citedArticleID="1023988" data-doi="10.1006/jmbi.1996.0748"><li><a href="http://dx.doi.org/10.1006/jmbi.1996.0748" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RecA+protein+filaments%3A+end-dependent+disassociation+from+ssDNA+and+stabilization+by+RecO+and+RecR+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RecA+protein+filaments%3A+end-dependent+disassociation+from+ssDNA+and+stabilization+by+RecO+and+RecR+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001242-Bork1" id="pone.0001242-Bork1"></a>Bork JM, Cox MM, Inman RB (2001) The RecOR proteins modulate RecA protein function at 5′ ends of single-stranded DNA. EMBO Journal  20: 7313–7322.  <ul class="find" data-citedArticleID="1023838" data-doi="10.1093/emboj/20.24.7313"><li><a href="http://dx.doi.org/10.1093/emboj/20.24.7313" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+RecOR+proteins+modulate+RecA+protein+function+at+5%E2%80%B2+ends+of+single-stranded+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+RecOR+proteins+modulate+RecA+protein+function+at+5%E2%80%B2+ends+of+single-stranded+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001242-Rosenberg3" id="pone.0001242-Rosenberg3"></a>Rosenberg SM, Hastings PJ (1991) The split-end model for homologous recombination at double-strand breaks and at Chi. Biochimie  73: 385–397.  <ul class="find" data-citedArticleID="1023974" data-doi="10.1016/0300-9084(91)90105-a"><li><a href="http://dx.doi.org/10.1016/0300-9084(91)90105-a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+split-end+model+for+homologous+recombination+at+double-strand+breaks+and+at+Chi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+split-end+model+for+homologous+recombination+at+double-strand+breaks+and+at+Chi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001242-Siddiqi1" id="pone.0001242-Siddiqi1"></a>Siddiqi I, Stahl MM, Stahl FW (1991) Heteroduplex chain polarity in recombination of phage λ by the Red, RecBCD, RecBC(D<sup>−</sup>), and RecF pathways. Genetics  128: 7–22.  <ul class="find" data-citedArticleID="1023996"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Heteroduplex+chain+polarity+in+recombination+of+phage+%3F+by+the+Red%2C+RecBCD%2C+RecBC%28D%3F%29%2C+and+RecF+pathways.&amp;auth=&amp;atitle=Heteroduplex+chain+polarity+in+recombination+of+phage+%3F+by+the+Red%2C+RecBCD%2C+RecBC%28D%3F%29%2C+and+RecF+pathways." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heteroduplex+chain+polarity+in+recombination+of+phage+%CE%BB+by+the+Red%2C+RecBCD%2C+RecBC%28D%E2%88%92%29%2C+and+RecF+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heteroduplex+chain+polarity+in+recombination+of+phage+%CE%BB+by+the+Red%2C+RecBCD%2C+RecBC%28D%E2%88%92%29%2C+and+RecF+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001242-Holbeck1" id="pone.0001242-Holbeck1"></a>Holbeck SL, Smith GR (1992) Chi enhances heteroduplex DNA levels during recombination. Genetics  132: 879–891.  <ul class="find" data-citedArticleID="1023884"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chi+enhances+heteroduplex+DNA+levels+during+recombination.&amp;auth=&amp;atitle=Chi+enhances+heteroduplex+DNA+levels+during+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chi+enhances+heteroduplex+DNA+levels+during+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chi+enhances+heteroduplex+DNA+levels+during+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001242-White1" id="pone.0001242-White1"></a>White RL, Fox MS (1974) On the molecular basis of high negative interference. Proc Natl Acad Sci U S A  71: 1544–1548.  <ul class="find" data-citedArticleID="1024038" data-doi="10.1073/pnas.71.4.1544"><li><a href="http://dx.doi.org/10.1073/pnas.71.4.1544" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+molecular+basis+of+high+negative+interference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+molecular+basis+of+high+negative+interference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001242-Nag1" id="pone.0001242-Nag1"></a>Nag DK, Petes TD (1990) Genetic evidence for preferential strand transfer during meiotic recombination in yeast. Genetics  125: 753–761.  <ul class="find" data-citedArticleID="1023948"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+evidence+for+preferential+strand+transfer+during+meiotic+recombination+in+yeast.&amp;auth=&amp;atitle=Genetic+evidence+for+preferential+strand+transfer+during+meiotic+recombination+in+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+evidence+for+preferential+strand+transfer+during+meiotic+recombination+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+evidence+for+preferential+strand+transfer+during+meiotic+recombination+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001242-Sawitzke1" id="pone.0001242-Sawitzke1"></a>Sawitzke JA, Stahl FW (1992) Phage λ has an analog of the <em>Escherichia coli recO</em>, <em>recR</em> and <em>recF</em> genes. Genetics  130: 7–16.  <ul class="find" data-citedArticleID="1023982"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phage+%3F+has+an+analog+of+the+Escherichia+coli+recO%2C+recR+and+recF+genes.&amp;auth=&amp;atitle=Phage+%3F+has+an+analog+of+the+Escherichia+coli+recO%2C+recR+and+recF+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phage+%CE%BB+has+an+analog+of+the+Escherichia+coli+recO%2C+recR+and+recF+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phage+%CE%BB+has+an+analog+of+the+Escherichia+coli+recO%2C+recR+and+recF+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001242-Sharples1" id="pone.0001242-Sharples1"></a>Sharples GJ, Curtis FA, McGlynn P, Bolt EL (2004) Holliday junction binding and resolution by the Rap structure-specific endonuclease of phage lambda. J Mol Biol  340: 739–751.  <ul class="find" data-citedArticleID="1023990" data-doi="10.1016/j.jmb.2004.05.030"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.05.030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Holliday+junction+binding+and+resolution+by+the+Rap+structure-specific+endonuclease+of+phage+lambda." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Holliday+junction+binding+and+resolution+by+the+Rap+structure-specific+endonuclease+of+phage+lambda.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001242-Nagamine1" id="pone.0001242-Nagamine1"></a>Nagamine CM, Chan K, Lau YF (1989) A PCR artifact: generation of heteroduplexes. Am J Hum Genet  45: 337–339.  <ul class="find" data-citedArticleID="1023950"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+PCR+artifact%3A+generation+of+heteroduplexes.&amp;auth=&amp;atitle=A+PCR+artifact%3A+generation+of+heteroduplexes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+PCR+artifact%3A+generation+of+heteroduplexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+PCR+artifact%3A+generation+of+heteroduplexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001242-Silberstein1" id="pone.0001242-Silberstein1"></a>Silberstein Z, Shalit M, Cohen A (1993) Heteroduplex strand-specificity in restriction-stimulated recombination by the RecE pathway of <em>Escherichia coli</em>. Genetics  133: 439–448.  <ul class="find" data-citedArticleID="1024000"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Heteroduplex+strand-specificity+in+restriction-stimulated+recombination+by+the+RecE+pathway+of+Escherichia+coli.&amp;auth=&amp;atitle=Heteroduplex+strand-specificity+in+restriction-stimulated+recombination+by+the+RecE+pathway+of+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heteroduplex+strand-specificity+in+restriction-stimulated+recombination+by+the+RecE+pathway+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heteroduplex+strand-specificity+in+restriction-stimulated+recombination+by+the+RecE+pathway+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001242-Murphy1" id="pone.0001242-Murphy1"></a>Murphy KC (1991) Lambda Gam protein inhibits the helicase and Chi-stimulated recombination activities of <em>Escherichia coli</em> RecBCD enzyme. J Bacteriol  173: 5808–5821.  <ul class="find" data-citedArticleID="1023940"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Lambda+Gam+protein+inhibits+the+helicase+and+Chi-stimulated+recombination+activities+of+Escherichia+coli+RecBCD+enzyme.&amp;auth=&amp;atitle=Lambda+Gam+protein+inhibits+the+helicase+and+Chi-stimulated+recombination+activities+of+Escherichia+coli+RecBCD+enzyme." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lambda+Gam+protein+inhibits+the+helicase+and+Chi-stimulated+recombination+activities+of+Escherichia+coli+RecBCD+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lambda+Gam+protein+inhibits+the+helicase+and+Chi-stimulated+recombination+activities+of+Escherichia+coli+RecBCD+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001242-Wold1" id="pone.0001242-Wold1"></a>Wold MS, Mallory JB, Roberts JD, LeBowitz JH, McMacken R (1982) Initiation of bacteriophage lambda DNA replication <em>in vitro</em> with purified lambda replication proteins. Proc Natl Acad Sci U S A  79: 6176–6180.  <ul class="find" data-citedArticleID="1024040" data-doi="10.1073/pnas.79.20.6176"><li><a href="http://dx.doi.org/10.1073/pnas.79.20.6176" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initiation+of+bacteriophage+lambda+DNA+replication+in+vitro+with+purified+lambda+replication+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initiation+of+bacteriophage+lambda+DNA+replication+in+vitro+with+purified+lambda+replication+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001242-Cohen1" id="pone.0001242-Cohen1"></a>Cohen A, Clark AJ (1986) Synthesis of linear plasmid multimers in <em>Escherichia coli</em> K-12. J Bacteriol  167: 327–335.  <ul class="find" data-citedArticleID="1023846"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Synthesis+of+linear+plasmid+multimers+in+Escherichia+coli+K-12.&amp;auth=&amp;atitle=Synthesis+of+linear+plasmid+multimers+in+Escherichia+coli+K-12." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synthesis+of+linear+plasmid+multimers+in+Escherichia+coli+K-12." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synthesis+of+linear+plasmid+multimers+in+Escherichia+coli+K-12.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001242-Datsenko1" id="pone.0001242-Datsenko1"></a>Datsenko KA, Wanner BL (2000) One-step inactivation of chromosomal genes in <em>Escherichia coli</em> K-12 using PCR products. Proc Natl Acad Sci USA  97: 6640–6645.  <ul class="find" data-citedArticleID="1023852" data-doi="10.1073/pnas.120163297"><li><a href="http://dx.doi.org/10.1073/pnas.120163297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=One-step+inactivation+of+chromosomal+genes+in+Escherichia+coli+K-12+using+PCR+products." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22One-step+inactivation+of+chromosomal+genes+in+Escherichia+coli+K-12+using+PCR+products.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001242-MartinezMorales1" id="pone.0001242-MartinezMorales1"></a>Martinez-Morales F, Borges AC, Martinez A, Shanmugam KT, Ingram LO (1999) Chromosomal integration of heterologous DNA in <em>Escherichia coli</em> with precise removal of markers and replicons used during construction. J Bacteriol  181: 7143–7148.  <ul class="find" data-citedArticleID="1023922"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chromosomal+integration+of+heterologous+DNA+in+Escherichia+coli+with+precise+removal+of+markers+and+replicons+used+during+construction.&amp;auth=&amp;atitle=Chromosomal+integration+of+heterologous+DNA+in+Escherichia+coli+with+precise+removal+of+markers+and+replicons+used+during+construction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+integration+of+heterologous+DNA+in+Escherichia+coli+with+precise+removal+of+markers+and+replicons+used+during+construction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+integration+of+heterologous+DNA+in+Escherichia+coli+with+precise+removal+of+markers+and+replicons+used+during+construction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001242-Arber1" id="pone.0001242-Arber1"></a>Arber W, Enquist L, Hohn B, Murray NE, Murray K (1983)  Experimental Methods for use with Lambda. In: Hendrix RW, Roberts JW, Stahl FW, Weisberg RW, editors. Lambda II. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press. pp. 433–466.  <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0001242-Miller1" id="pone.0001242-Miller1"></a>Miller JH (1992) A Short Course in Bacterial Genetics. Cold Spring Harbor, N. Y.: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">37.
              </span><a name="pone.0001242-Rosenberg4" id="pone.0001242-Rosenberg4"></a>Rosenberg SM, Stahl MM, Kobayashi I, Stahl FW (1985) Improved in vitro packaging of coliphage lambda DNA: a one-strain system free from endogenous phage. Gene  38: 165–175.  <ul class="find" data-citedArticleID="1023976" data-doi="10.1016/0378-1119(85)90215-x"><li><a href="http://dx.doi.org/10.1016/0378-1119(85)90215-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+in+vitro+packaging+of+coliphage+lambda+DNA%3A+a+one-strain+system+free+from+endogenous+phage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+in+vitro+packaging+of+coliphage+lambda+DNA%3A+a+one-strain+system+free+from+endogenous+phage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001242-Sambrook1" id="pone.0001242-Sambrook1"></a>Sambrook J, Russell DW (2001) Molecular Cloning: A Laboratory Manual. Cold Spring Harbor, N. Y.: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0001242-Jordan1" id="pone.0001242-Jordan1"></a>Jordan H (1997) Gel Loading Buffer Affects Electrophoresis Results. Focus  19: 60–61.  <ul class="find" data-citedArticleID="1023888"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gel+Loading+Buffer+Affects+Electrophoresis+Results.&amp;auth=&amp;atitle=Gel+Loading+Buffer+Affects+Electrophoresis+Results." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gel+Loading+Buffer+Affects+Electrophoresis+Results." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gel+Loading+Buffer+Affects+Electrophoresis+Results.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001242-Church1" id="pone.0001242-Church1"></a>Church GM, Gilbert W (1984) Genomic sequencing. Proc Natl Acad Sci U S A  81: 1991–1995.  <ul class="find" data-citedArticleID="1023844"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genomic+sequencing.&amp;auth=&amp;atitle=Genomic+sequencing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001242-Mason1" id="pone.0001242-Mason1"></a>Mason JC, Steinberg R, Lidington EA, Kinderlerer AR, Ohba M, et al.  (2004) Decay-accelerating factor induction on vascular endothelium by vascular endothelial growth factor (VEGF) is mediated <em>via</em> a VEGF receptor-2 (VEGF-R2)- and protein kinase C-alpha/epsilon (PKCalpha/epsilon)-dependent cytoprotective signaling pathway and is inhibited by cyclosporin A. J Biol Chem  279: 41611–41618.  <ul class="find" data-citedArticleID="1023924" data-doi="10.1074/jbc.m407981200"><li><a href="http://dx.doi.org/10.1074/jbc.m407981200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Decay-accelerating+factor+induction+on+vascular+endothelium+by+vascular+endothelial+growth+factor+%28VEGF%29+is+mediated+via+a+VEGF+receptor-2+%28VEGF-R2%29-+and+protein+kinase+C-alpha%2Fepsilon+%28PKCalpha%2Fepsilon%29-dependent+cytoprotective+signaling+pathway+and+is+inhibited+by+cyclosporin+A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Decay-accelerating+factor+induction+on+vascular+endothelium+by+vascular+endothelial+growth+factor+%28VEGF%29+is+mediated+via+a+VEGF+receptor-2+%28VEGF-R2%29-+and+protein+kinase+C-alpha%2Fepsilon+%28PKCalpha%2Fepsilon%29-dependent+cytoprotective+signaling+pathway+and+is+inhibited+by+cyclosporin+A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001242-Shiao1" id="pone.0001242-Shiao1"></a>Shiao T, Fond A, Deng B, Wehling-Henricks M, Adams ME, et al.  (2004) Defects in neuromuscular junction structure in dystrophic muscle are corrected by expression of a NOS transgene in dystrophin-deficient muscles, but not in muscles lacking alpha- and beta1-syntrophins. Hum Mol Genet  13: 1873–1884.  <ul class="find" data-citedArticleID="1023994" data-doi="10.1093/hmg/ddh204"><li><a href="http://dx.doi.org/10.1093/hmg/ddh204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Defects+in+neuromuscular+junction+structure+in+dystrophic+muscle+are+corrected+by+expression+of+a+NOS+transgene+in+dystrophin-deficient+muscles%2C+but+not+in+muscles+lacking+alpha-+and+beta1-syntrophins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Defects+in+neuromuscular+junction+structure+in+dystrophic+muscle+are+corrected+by+expression+of+a+NOS+transgene+in+dystrophin-deficient+muscles%2C+but+not+in+muscles+lacking+alpha-+and+beta1-syntrophins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001242-Diamonti1" id="pone.0001242-Diamonti1"></a>Diamonti AJ, Guy PM, Ivanof C, Wong K, Sweeney C, et al.  (2002) An RBCC protein implicated in maintenance of steady-state neuregulin receptor levels. Proc Natl Acad Sci U S A  99: 2866–2871.  <ul class="find" data-citedArticleID="1023854" data-doi="10.1073/pnas.052709799"><li><a href="http://dx.doi.org/10.1073/pnas.052709799" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+RBCC+protein+implicated+in+maintenance+of+steady-state+neuregulin+receptor+levels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+RBCC+protein+implicated+in+maintenance+of+steady-state+neuregulin+receptor+levels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001242-Pearson1" id="pone.0001242-Pearson1"></a>Pearson RK, Fox MS (1988) Effects of DNA heterologies on bacteriophage lambda recombination. Genetics  118: 13–19.  <ul class="find" data-citedArticleID="1023960"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effects+of+DNA+heterologies+on+bacteriophage+lambda+recombination.&amp;auth=&amp;atitle=Effects+of+DNA+heterologies+on+bacteriophage+lambda+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+DNA+heterologies+on+bacteriophage+lambda+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+DNA+heterologies+on+bacteriophage+lambda+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001242-Parker1" id="pone.0001242-Parker1"></a>Parker BO, Marinus MG (1992) Repair of DNA heteroduplexes containing small heterologous sequences in <em>Escherichia coli</em>. Proc Natl Acad Sci U S A  89: 1730–1734.  <ul class="find" data-citedArticleID="1023958" data-doi="10.1073/pnas.89.5.1730"><li><a href="http://dx.doi.org/10.1073/pnas.89.5.1730" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repair+of+DNA+heteroduplexes+containing+small+heterologous+sequences+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repair+of+DNA+heteroduplexes+containing+small+heterologous+sequences+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001242-Carraway1" id="pone.0001242-Carraway1"></a>Carraway M, Marinus MG (1993) Repair of heteroduplex DNA molecules with multibase loops in <em>Escherichia coli</em>. J Bacteriol  175: 3972–3980.  <ul class="find" data-citedArticleID="1023842"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Repair+of+heteroduplex+DNA+molecules+with+multibase+loops+in+Escherichia+coli.&amp;auth=&amp;atitle=Repair+of+heteroduplex+DNA+molecules+with+multibase+loops+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repair+of+heteroduplex+DNA+molecules+with+multibase+loops+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repair+of+heteroduplex+DNA+molecules+with+multibase+loops+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001242-Wang1" id="pone.0001242-Wang1"></a>Wang J, Chen R, Julin DA (2000) A single nuclease active site of the <em>Escherichia coli</em> RecBCD enzyme catalyzes single-stranded DNA degradation in both directions. J Biol Chem  275: 507–513.  <ul class="find" data-citedArticleID="1024032" data-doi="10.1074/jbc.275.1.507"><li><a href="http://dx.doi.org/10.1074/jbc.275.1.507" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+nuclease+active+site+of+the+Escherichia+coli+RecBCD+enzyme+catalyzes+single-stranded+DNA+degradation+in+both+directions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+nuclease+active+site+of+the+Escherichia+coli+RecBCD+enzyme+catalyzes+single-stranded+DNA+degradation+in+both+directions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001242-Lusetti1" id="pone.0001242-Lusetti1"></a>Lusetti SL, Cox MM (2002) The bacterial RecA protein and the recombinational DNA repair of stalled replication forks. Annu Rev Biochem  71: 71–100.  <ul class="find" data-citedArticleID="1023918"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+bacterial+RecA+protein+and+the+recombinational+DNA+repair+of+stalled+replication+forks.&amp;auth=&amp;atitle=The+bacterial+RecA+protein+and+the+recombinational+DNA+repair+of+stalled+replication+forks." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+bacterial+RecA+protein+and+the+recombinational+DNA+repair+of+stalled+replication+forks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+bacterial+RecA+protein+and+the+recombinational+DNA+repair+of+stalled+replication+forks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001242-Taylor2" id="pone.0001242-Taylor2"></a>Taylor AF (1988)  RecBCD enzyme of <em>Escherichia coli</em>. In: Kucherlapati R, Smith GR, editors. Genetic Recombination. Washington D. C.: Am. Soc. Microbiol. pp. 231–263.  <ul class="find-nolinks"></ul></li><li><span class="label">50.
              </span><a name="pone.0001242-Maxwell1" id="pone.0001242-Maxwell1"></a>Maxwell KL, Reed P, Zhang RG, Beasley S, Walmsley AR, et al.  (2005) Functional similarities between phage lambda Orf and <em>Escherichia coli</em> RecFOR in initiation of genetic exchange. Proc Natl Acad Sci U S A  102: 11260–11265.  <ul class="find" data-citedArticleID="1023926" data-doi="10.1073/pnas.0503399102"><li><a href="http://dx.doi.org/10.1073/pnas.0503399102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+similarities+between+phage+lambda+Orf+and+Escherichia+coli+RecFOR+in+initiation+of+genetic+exchange." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+similarities+between+phage+lambda+Orf+and+Escherichia+coli+RecFOR+in+initiation+of+genetic+exchange.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001242-Sawitzke2" id="pone.0001242-Sawitzke2"></a>Sawitzke JA, Stahl FW (1994) The phage lambda <em>orf</em> gene encodes a <em>trans</em>-acting factor that suppresses <em>Escherichia coli recO</em>, <em>recR</em>, and <em>recF</em> mutations for recombination of lambda but not of <em>E. coli</em>. J Bacteriol  176: 6730–6737.  <ul class="find" data-citedArticleID="1023984"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+phage+lambda+orf+gene+encodes+a+trans-acting+factor+that+suppresses+Escherichia+coli+recO%2C+recR%2C+and+recF+mutations+for+recombination+of+lambda+but+not+of+E.+coli.&amp;auth=&amp;atitle=The+phage+lambda+orf+gene+encodes+a+trans-acting+factor+that+suppresses+Escherichia+coli+recO%2C+recR%2C+and+recF+mutations+for+recombination+of+lambda+but+not+of+E.+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+phage+lambda+orf+gene+encodes+a+trans-acting+factor+that+suppresses+Escherichia+coli+recO%2C+recR%2C+and+recF+mutations+for+recombination+of+lambda+but+not+of+E.+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+phage+lambda+orf+gene+encodes+a+trans-acting+factor+that+suppresses+Escherichia+coli+recO%2C+recR%2C+and+recF+mutations+for+recombination+of+lambda+but+not+of+E.+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001242-Sawitzke3" id="pone.0001242-Sawitzke3"></a>Sawitzke JA, Stahl FW (1997) Roles for lambda Orf and <em>Escherichia coli</em> RecO, RecR and RecF in lambda recombination. Genetics  147: 357–369.  <ul class="find" data-citedArticleID="1023986"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Roles+for+lambda+Orf+and+Escherichia+coli+RecO%2C+RecR+and+RecF+in+lambda+recombination.&amp;auth=&amp;atitle=Roles+for+lambda+Orf+and+Escherichia+coli+RecO%2C+RecR+and+RecF+in+lambda+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Roles+for+lambda+Orf+and+Escherichia+coli+RecO%2C+RecR+and+RecF+in+lambda+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Roles+for+lambda+Orf+and+Escherichia+coli+RecO%2C+RecR+and+RecF+in+lambda+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001242-Morimatsu1" id="pone.0001242-Morimatsu1"></a>Morimatsu K, Kowalczykowski SC (2003) RecFOR proteins load RecA protein onto gapped DNA to accelerate DNA strand exchange: a universal step of recombinational repair. Mol Cell  11: 1337–1347.  <ul class="find" data-citedArticleID="1023936" data-doi="10.1016/s1097-2765(03)00188-6"><li><a href="http://dx.doi.org/10.1016/s1097-2765(03)00188-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RecFOR+proteins+load+RecA+protein+onto+gapped+DNA+to+accelerate+DNA+strand+exchange%3A+a+universal+step+of+recombinational+repair." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RecFOR+proteins+load+RecA+protein+onto+gapped+DNA+to+accelerate+DNA+strand+exchange%3A+a+universal+step+of+recombinational+repair.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001242-Poteete1" id="pone.0001242-Poteete1"></a>Poteete AR, Fenton AC, Wang HR (2002) Recombination-promoting activity of the bacteriophage lambda Rap protein in <em>Escherichia coli</em> K-12. J Bacteriol  184: 4626–4629.  <ul class="find" data-citedArticleID="1023962" data-doi="10.1128/jb.184.16.4626-4629.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.16.4626-4629.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination-promoting+activity+of+the+bacteriophage+lambda+Rap+protein+in+Escherichia+coli+K-12." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination-promoting+activity+of+the+bacteriophage+lambda+Rap+protein+in+Escherichia+coli+K-12.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001242-Sharples2" id="pone.0001242-Sharples2"></a>Sharples GJ, Corbett LM, Graham IR (1998) λ Rap protein is a structure-specific endonuclease involved in phage recombination. Proc Natl Acad Sci U S A  95: 13507–13512.  <ul class="find" data-citedArticleID="1023992" data-doi="10.1073/pnas.95.23.13507"><li><a href="http://dx.doi.org/10.1073/pnas.95.23.13507" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%CE%BB+Rap+protein+is+a+structure-specific+endonuclease+involved+in+phage+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%CE%BB+Rap+protein+is+a+structure-specific+endonuclease+involved+in+phage+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001242-Hollifield1" id="pone.0001242-Hollifield1"></a>Hollifield WC, Kaplan EN, Huang H (1987) Efficient RecABC-dependent, homologous recombination between coliphage lambda and plasmids requires a phage <em>ninR</em> region gene. Mol Gen Genet  210: 248–255.  <ul class="find" data-citedArticleID="1023886" data-doi="10.1007/bf00325690"><li><a href="http://dx.doi.org/10.1007/bf00325690" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+RecABC-dependent%2C+homologous+recombination+between+coliphage+lambda+and+plasmids+requires+a+phage+ninR+region+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+RecABC-dependent%2C+homologous+recombination+between+coliphage+lambda+and+plasmids+requires+a+phage+ninR+region+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001242-Bolt1" id="pone.0001242-Bolt1"></a>Bolt EL, Lloyd RG (2002) Substrate specificity of RusA resolvase reveals the DNA structures targeted by RuvAB and RecG <em>in vivo</em>. Mol Cell  10: 187–198.  <ul class="find" data-citedArticleID="1023836" data-doi="10.1016/s1097-2765(02)00560-9"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00560-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Substrate+specificity+of+RusA+resolvase+reveals+the+DNA+structures+targeted+by+RuvAB+and+RecG+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Substrate+specificity+of+RusA+resolvase+reveals+the+DNA+structures+targeted+by+RuvAB+and+RecG+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001242-Saintigny1" id="pone.0001242-Saintigny1"></a>Saintigny Y, Makienko K, Swanson C, Emond MJ, Monnat RJ Jr (2002) Homologous recombination resolution defect in Werner syndrome. Mol Cell Biol  22: 6971–6978.  <ul class="find" data-citedArticleID="1023978" data-doi="10.1128/mcb.22.20.6971-6978.2002"><li><a href="http://dx.doi.org/10.1128/mcb.22.20.6971-6978.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homologous+recombination+resolution+defect+in+Werner+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homologous+recombination+resolution+defect+in+Werner+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001242-Doe1" id="pone.0001242-Doe1"></a>Doe CL, Whitby MC (2004) The involvement of Srs2 in post-replication repair and homologous recombination in fission yeast. Nucleic Acids Res  32: 1480–1491.  <ul class="find" data-citedArticleID="1023858" data-doi="10.1093/nar/gkh317"><li><a href="http://dx.doi.org/10.1093/nar/gkh317" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+involvement+of+Srs2+in+post-replication+repair+and+homologous+recombination+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+involvement+of+Srs2+in+post-replication+repair+and+homologous+recombination+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001242-Stahl1" id="pone.0001242-Stahl1"></a>Stahl FW, Shurvinton CE, Thomason LC, Hill S, Stahl MM (1995) On the clustered exchanges of the RecBCD pathway operating on phage λ. Genetics  139: 1107–1121.  <ul class="find" data-citedArticleID="1024010"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+the+clustered+exchanges+of+the+RecBCD+pathway+operating+on+phage+%3F.&amp;auth=&amp;atitle=On+the+clustered+exchanges+of+the+RecBCD+pathway+operating+on+phage+%3F." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+clustered+exchanges+of+the+RecBCD+pathway+operating+on+phage+%CE%BB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+clustered+exchanges+of+the+RecBCD+pathway+operating+on+phage+%CE%BB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001242-Dixon1" id="pone.0001242-Dixon1"></a>Dixon DA, Kowalczykowski SC (1993) The recombination hotspot, Chi, is a regulatory sequence that acts by attenuating the nuclease activity of the <em>Escherichia coli recBCD</em> enzyme. Cell  73: 87–96.  <ul class="find" data-citedArticleID="1023856" data-doi="10.1016/0092-8674(93)90162-j"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)90162-j" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+recombination+hotspot%2C+Chi%2C+is+a+regulatory+sequence+that+acts+by+attenuating+the+nuclease+activity+of+the+Escherichia+coli+recBCD+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+recombination+hotspot%2C+Chi%2C+is+a+regulatory+sequence+that+acts+by+attenuating+the+nuclease+activity+of+the+Escherichia+coli+recBCD+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001242-Anderson1" id="pone.0001242-Anderson1"></a>Anderson DG, Kowalczykowski SC (1997) The recombination hot spot chi is a regulatory element that switches the polarity of DNA degradation by the RecBCD enzyme. Genes Dev  11: 571–581.  <ul class="find" data-citedArticleID="1023824" data-doi="10.1101/gad.11.5.571"><li><a href="http://dx.doi.org/10.1101/gad.11.5.571" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+recombination+hot+spot+chi+is+a+regulatory+element+that+switches+the+polarity+of+DNA+degradation+by+the+RecBCD+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+recombination+hot+spot+chi+is+a+regulatory+element+that+switches+the+polarity+of+DNA+degradation+by+the+RecBCD+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001242-Anderson2" id="pone.0001242-Anderson2"></a>Anderson DG, Churchill JJ, Kowalczykowski SC (1997) Chi-activated RecBCD enzyme possesses 5′–&gt;3′ nucleolytic activity, but RecBC enzyme does not: evidence suggesting that the alteration induced by Chi is not simply ejection of the RecD subunit. Genes Cells  2: 117–128.  <ul class="find" data-citedArticleID="1023826" data-doi="10.1046/j.1365-2443.1997.1130311.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2443.1997.1130311.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chi-activated+RecBCD+enzyme+possesses+5%E2%80%B2%E2%80%93%26gt%3B3%E2%80%B2+nucleolytic+activity%2C+but+RecBC+enzyme+does+not%3A+evidence+suggesting+that+the+alteration+induced+by+Chi+is+not+simply+ejection+of+the+RecD+subunit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chi-activated+RecBCD+enzyme+possesses+5%E2%80%B2%E2%80%93%26gt%3B3%E2%80%B2+nucleolytic+activity%2C+but+RecBC+enzyme+does+not%3A+evidence+suggesting+that+the+alteration+induced+by+Chi+is+not+simply+ejection+of+the+RecD+subunit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001242-Dabert1" id="pone.0001242-Dabert1"></a>Dabert PS, Ehrlich SD, Gruss A (1992) Chi sequence protects against RecBCD degradation of DNA <em>in vivo</em>. Proc Natl Acad Sci U S A  89: 12073–12077.  <ul class="find" data-citedArticleID="1023850" data-doi="10.1073/pnas.89.24.12073"><li><a href="http://dx.doi.org/10.1073/pnas.89.24.12073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chi+sequence+protects+against+RecBCD+degradation+of+DNA+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chi+sequence+protects+against+RecBCD+degradation+of+DNA+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001242-Myers2" id="pone.0001242-Myers2"></a>Myers RM, Kuzminov A, Stahl FW (1995) The recombination hot spot χ activates RecBCD recombination by converting <em>Escherichia coli</em> to a <em>recD</em> mutant phenocopy. Proc Natl Acad Sci USA  92: 6244–6248.  <ul class="find" data-citedArticleID="1023946" data-doi="10.1073/pnas.92.14.6244"><li><a href="http://dx.doi.org/10.1073/pnas.92.14.6244" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+recombination+hot+spot+%CF%87+activates+RecBCD+recombination+by+converting+Escherichia+coli+to+a+recD+mutant+phenocopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+recombination+hot+spot+%CF%87+activates+RecBCD+recombination+by+converting+Escherichia+coli+to+a+recD+mutant+phenocopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001242-Korangy1" id="pone.0001242-Korangy1"></a>Korangy F, Julin DA (1993) Kinetics and processivity of ATP hydrolysis and DNA unwinding by the RecBC enzyme from Escherichia coli. Biochemistry  32: 4873–4880.  <ul class="find" data-citedArticleID="1023896" data-doi="10.1021/bi00069a024"><li><a href="http://dx.doi.org/10.1021/bi00069a024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinetics+and+processivity+of+ATP+hydrolysis+and+DNA+unwinding+by+the+RecBC+enzyme+from+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinetics+and+processivity+of+ATP+hydrolysis+and+DNA+unwinding+by+the+RecBC+enzyme+from+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001242-Korangy2" id="pone.0001242-Korangy2"></a>Korangy F, Julin DA (1994) Efficiency of ATP hydrolysis and DNA unwinding by the RecBC enzyme from Escherichia coli. Biochemistry  33: 9552–9560.  <ul class="find" data-citedArticleID="1023898" data-doi="10.1021/bi00198a022"><li><a href="http://dx.doi.org/10.1021/bi00198a022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficiency+of+ATP+hydrolysis+and+DNA+unwinding+by+the+RecBC+enzyme+from+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficiency+of+ATP+hydrolysis+and+DNA+unwinding+by+the+RecBC+enzyme+from+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001242-Rinken1" id="pone.0001242-Rinken1"></a>Rinken R, Thoms B, Wackernagel W (1992) Evidence that <em>recBC</em>-dependent degradation of duplex DNA in <em>Escherichia coli recD</em> mutants involves DNA unwinding. J Bacteriol  174: 5424–5429.  <ul class="find" data-citedArticleID="1023968"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evidence+that+recBC-dependent+degradation+of+duplex+DNA+in+Escherichia+coli+recD+mutants+involves+DNA+unwinding.&amp;auth=&amp;atitle=Evidence+that+recBC-dependent+degradation+of+duplex+DNA+in+Escherichia+coli+recD+mutants+involves+DNA+unwinding." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+that+recBC-dependent+degradation+of+duplex+DNA+in+Escherichia+coli+recD+mutants+involves+DNA+unwinding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+that+recBC-dependent+degradation+of+duplex+DNA+in+Escherichia+coli+recD+mutants+involves+DNA+unwinding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001242-Singleton1" id="pone.0001242-Singleton1"></a>Singleton MR, Dillingham MS, Gaudier M, Kowalczykowski SC, Wigley DB (2004) Crystal structure of RecBCD enzyme reveals a machine for processing DNA breaks. Nature  432: 187–193.  <ul class="find" data-citedArticleID="1024002" data-doi="10.1038/nature02988"><li><a href="http://dx.doi.org/10.1038/nature02988" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+RecBCD+enzyme+reveals+a+machine+for+processing+DNA+breaks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+RecBCD+enzyme+reveals+a+machine+for+processing+DNA+breaks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001242-Lloyd1" id="pone.0001242-Lloyd1"></a>Lloyd RG, Sharples GJ (1993) Processing of recombination intermediates by the RecG and RuvAB proteins of <em>Escherichia coli</em>. Nucleic Acids Res  21: 1719–1725.  <ul class="find" data-citedArticleID="1023910" data-doi="10.1093/nar/21.8.1719"><li><a href="http://dx.doi.org/10.1093/nar/21.8.1719" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Processing+of+recombination+intermediates+by+the+RecG+and+RuvAB+proteins+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Processing+of+recombination+intermediates+by+the+RecG+and+RuvAB+proteins+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001242-McGlynn1" id="pone.0001242-McGlynn1"></a>McGlynn P, Lloyd RG (1999) RecG helicase activity at three- and four-strand DNA structures. Nucleic Acids Res  27: 3049–3056.  <ul class="find" data-citedArticleID="1023928" data-doi="10.1093/nar/27.15.3049"><li><a href="http://dx.doi.org/10.1093/nar/27.15.3049" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RecG+helicase+activity+at+three-+and+four-strand+DNA+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RecG+helicase+activity+at+three-+and+four-strand+DNA+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001242-West1" id="pone.0001242-West1"></a>West SC (1997) Processing of recombination intermediates by the RuvABC proteins. Annu Rev Genet  31: 213–244.  <ul class="find" data-citedArticleID="1024034" data-doi="10.1146/annurev.genet.31.1.213"><li><a href="http://dx.doi.org/10.1146/annurev.genet.31.1.213" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Processing+of+recombination+intermediates+by+the+RuvABC+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Processing+of+recombination+intermediates+by+the+RuvABC+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001242-Whitby1" id="pone.0001242-Whitby1"></a>Whitby MC, Ryder L, Lloyd RG (1993) Reverse branch migration of Holliday junctions by RecG protein: a new mechanism for resolution of intermediates in recombination and DNA repair. Cell  75: 341–350.  <ul class="find" data-citedArticleID="1024036" data-doi="10.1016/0092-8674(93)80075-p"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)80075-p" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reverse+branch+migration+of+Holliday+junctions+by+RecG+protein%3A+a+new+mechanism+for+resolution+of+intermediates+in+recombination+and+DNA+repair." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reverse+branch+migration+of+Holliday+junctions+by+RecG+protein%3A+a+new+mechanism+for+resolution+of+intermediates+in+recombination+and+DNA+repair.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001242-Harris1" id="pone.0001242-Harris1"></a>Harris RS, Ross KJ, Rosenberg SM (1996) Opposing roles of the Holliday junction processing systems of <em>Escherichia coli</em> in recombination-dependent adaptive mutation. Genetics  142: 681–691.  <ul class="find" data-citedArticleID="1023880"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Opposing+roles+of+the+Holliday+junction+processing+systems+of+Escherichia+coli+in+recombination-dependent+adaptive+mutation.&amp;auth=&amp;atitle=Opposing+roles+of+the+Holliday+junction+processing+systems+of+Escherichia+coli+in+recombination-dependent+adaptive+mutation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Opposing+roles+of+the+Holliday+junction+processing+systems+of+Escherichia+coli+in+recombination-dependent+adaptive+mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Opposing+roles+of+the+Holliday+junction+processing+systems+of+Escherichia+coli+in+recombination-dependent+adaptive+mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001242-Thaler1" id="pone.0001242-Thaler1"></a>Thaler DS, Stahl MM, Stahl FW (1987) Tests of the double-strand-break repair model for red-mediated recombination of phage lambda and plasmid lambda dv. Genetics  116: 501–511.  <ul class="find" data-citedArticleID="1024026"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Tests+of+the+double-strand-break+repair+model+for+red-mediated+recombination+of+phage+lambda+and+plasmid+lambda+dv.&amp;auth=&amp;atitle=Tests+of+the+double-strand-break+repair+model+for+red-mediated+recombination+of+phage+lambda+and+plasmid+lambda+dv." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tests+of+the+double-strand-break+repair+model+for+red-mediated+recombination+of+phage+lambda+and+plasmid+lambda+dv." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tests+of+the+double-strand-break+repair+model+for+red-mediated+recombination+of+phage+lambda+and+plasmid+lambda+dv.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001242-Lopez1" id="pone.0001242-Lopez1"></a>Lopez CR, Yang S, Deibler RW, Ray SA, Pennington JM, et al.  (2005) A role for topoisomerase III in a recombination pathway alternative to RuvABC. Mol Microbiol  58: 80–101.  <ul class="find" data-citedArticleID="1023916" data-doi="10.1111/j.1365-2958.2005.04812.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2005.04812.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+role+for+topoisomerase+III+in+a+recombination+pathway+alternative+to+RuvABC." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+role+for+topoisomerase+III+in+a+recombination+pathway+alternative+to+RuvABC.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001242-Motamedi1" id="pone.0001242-Motamedi1"></a>Motamedi M, Szigety SK, Rosenberg SM (1999) Double-strand break-repair in <em>E. coli</em>: direct physical evidence for a replicational mechanism <em>in vivo</em>. Genes &amp; Dev  13: 2889–2903.  <ul class="find" data-citedArticleID="1023938" data-doi="10.1101/gad.13.21.2889"><li><a href="http://dx.doi.org/10.1101/gad.13.21.2889" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Double-strand+break-repair+in+E.+coli%3A+direct+physical+evidence+for+a+replicational+mechanism+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Double-strand+break-repair+in+E.+coli%3A+direct+physical+evidence+for+a+replicational+mechanism+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0001242-Magner1" id="pone.0001242-Magner1"></a>Magner DB (2006) On Replicative and Break-Join Resolution, and RecQ-Promoted Accumulation of Recombination Intermediates <em>in vivo</em>. PhD Thesis  <ul class="find" data-citedArticleID="1023920"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+Replicative+and+Break-Join+Resolution%2C+and+RecQ-Promoted+Accumulation+of+Recombination+Intermediates+in+vivo.+PhD+Thesis&amp;auth=&amp;atitle=On+Replicative+and+Break-Join+Resolution%2C+and+RecQ-Promoted+Accumulation+of+Recombination+Intermediates+in+vivo.+PhD+Thesis" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+Replicative+and+Break-Join+Resolution%2C+and+RecQ-Promoted+Accumulation+of+Recombination+Intermediates+in+vivo.+PhD+Thesis" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+Replicative+and+Break-Join+Resolution%2C+and+RecQ-Promoted+Accumulation+of+Recombination+Intermediates+in+vivo.+PhD+Thesis%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001242-FriedmanOhana1" id="pone.0001242-FriedmanOhana1"></a>Friedman-Ohana R, Cohen A (1998) Heteroduplex joint formation in <em>Escherichia coli</em> recombination is initiated by pairing of a 3′-ending strand. Proc Natl Acad Sci U S A  95: 6909–6914.  <ul class="find" data-citedArticleID="1023868" data-doi="10.1073/pnas.95.12.6909"><li><a href="http://dx.doi.org/10.1073/pnas.95.12.6909" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heteroduplex+joint+formation+in+Escherichia+coli+recombination+is+initiated+by+pairing+of+a+3%E2%80%B2-ending+strand." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heteroduplex+joint+formation+in+Escherichia+coli+recombination+is+initiated+by+pairing+of+a+3%E2%80%B2-ending+strand.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001242-Lichten1" id="pone.0001242-Lichten1"></a>Lichten M, Goyon C, Schultes NP, Treco D, Szostak JW, et al.  (1990) Detection of heteroduplex DNA molecules among the products of <em>Saccharomyces cerevisiae</em> meiosis. Proc Natl Acad Sci USA  87: 7653–7657.  <ul class="find" data-citedArticleID="1023908" data-doi="10.1073/pnas.87.19.7653"><li><a href="http://dx.doi.org/10.1073/pnas.87.19.7653" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+heteroduplex+DNA+molecules+among+the+products+of+Saccharomyces+cerevisiae+meiosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+heteroduplex+DNA+molecules+among+the+products+of+Saccharomyces+cerevisiae+meiosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0001242-Dower1" id="pone.0001242-Dower1"></a>Dower NA, Stahl FW (1981) χ activity during transduction-associated recombination. Proc Natl Acad Sci U S A  78: 7033–7037.  <ul class="find" data-citedArticleID="1023860" data-doi="10.1073/pnas.78.11.7033"><li><a href="http://dx.doi.org/10.1073/pnas.78.11.7033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%CF%87+activity+during+transduction-associated+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%CF%87+activity+during+transduction-associated+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0001242-Kuzminov1" id="pone.0001242-Kuzminov1"></a>Kuzminov A (1999) Recombinational repair of DNA damage in <em>Escherichia coli</em> and bacteriophage lambda. Microbiol Mol Biol Rev  63: 751–813.  <ul class="find" data-citedArticleID="1023904"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Recombinational+repair+of+DNA+damage+in+Escherichia+coli+and+bacteriophage+lambda.&amp;auth=&amp;atitle=Recombinational+repair+of+DNA+damage+in+Escherichia+coli+and+bacteriophage+lambda." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombinational+repair+of+DNA+damage+in+Escherichia+coli+and+bacteriophage+lambda." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombinational+repair+of+DNA+damage+in+Escherichia+coli+and+bacteriophage+lambda.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0001242-Lloyd2" id="pone.0001242-Lloyd2"></a>Lloyd RG, Low KB (1996)  Homologous recombination. In: Neidhardt FC, Curtiss R III, Ingraham JL, Lin ECC, Low KB, et al., editors. <em>Escherichia coli</em> and <em>Salmonella</em>: cellular and molecular biology. 2nd ed. Washington, D. C.: ASM Press. pp. 2236–2255.  <ul class="find-nolinks"></ul></li><li><span class="label">84.
              </span><a name="pone.0001242-Smith3" id="pone.0001242-Smith3"></a>Smith GR, Stahl FW (1985) Homologous recombination promoted by Chi sites and RecBC enzyme of <em>Escherichia coli</em>. Bioessays  2: 244–249.  <ul class="find" data-citedArticleID="1024008" data-doi="10.1002/bies.950020603"><li><a href="http://dx.doi.org/10.1002/bies.950020603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homologous+recombination+promoted+by+Chi+sites+and+RecBC+enzyme+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homologous+recombination+promoted+by+Chi+sites+and+RecBC+enzyme+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0001242-Thaler2" id="pone.0001242-Thaler2"></a>Thaler DS, Sampson E, Siddiqi I, Rosenberg SM, Thomason LC, et al.  (1989) Recombination of bacteriophage λ in <em>recD</em> mutants of <em>Escherichia coli</em>. Genome  31: 53–67.  <ul class="find" data-citedArticleID="1024028" data-doi="10.1139/g89-013"><li><a href="http://dx.doi.org/10.1139/g89-013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+of+bacteriophage+%CE%BB+in+recD+mutants+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+of+bacteriophage+%CE%BB+in+recD+mutants+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0001242-Koppen1" id="pone.0001242-Koppen1"></a>Koppen A, Krobitsch S, Thoms B, Wackernagel W (1995) Interaction with the recombination hot spot chi <em>in vivo</em> converts the RecBCD enzyme of <em>Escherichia coli</em> into a chi-independent recombinase by inactivation of the RecD subunit. Proc Natl Acad Sci U S A  92: 6249–6253.  <ul class="find" data-citedArticleID="1023894" data-doi="10.1073/pnas.92.14.6249"><li><a href="http://dx.doi.org/10.1073/pnas.92.14.6249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+with+the+recombination+hot+spot+chi+in+vivo+converts+the+RecBCD+enzyme+of+Escherichia+coli+into+a+chi-independent+recombinase+by+inactivation+of+the+RecD+subunit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+with+the+recombination+hot+spot+chi+in+vivo+converts+the+RecBCD+enzyme+of+Escherichia+coli+into+a+chi-independent+recombinase+by+inactivation+of+the+RecD+subunit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0001242-Biek1" id="pone.0001242-Biek1"></a>Biek DP, Cohen SN (1986) Identification and characterization of <em>recD</em>, a gene affecting plasmid maintenance and recombination in <em>Escherichia coli</em>. J Bacteriol  167: 594–603.  <ul class="find" data-citedArticleID="1023834"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+and+characterization+of+recD%2C+a+gene+affecting+plasmid+maintenance+and+recombination+in+Escherichia+coli.&amp;auth=&amp;atitle=Identification+and+characterization+of+recD%2C+a+gene+affecting+plasmid+maintenance+and+recombination+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+recD%2C+a+gene+affecting+plasmid+maintenance+and+recombination+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+recD%2C+a+gene+affecting+plasmid+maintenance+and+recombination+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0001242-Lloyd3" id="pone.0001242-Lloyd3"></a>Lloyd RG (1991) Conjugational recombination in resolvase-deficient <em>ruvC</em> mutants of <em>Escherichia coli</em> depends on <em>recG</em>. J Bacteriol  173: 5414–5418.  <ul class="find" data-citedArticleID="1023914"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Conjugational+recombination+in+resolvase-deficient+ruvC+mutants+of+Escherichia+coli+depends+on+recG.&amp;auth=&amp;atitle=Conjugational+recombination+in+resolvase-deficient+ruvC+mutants+of+Escherichia+coli+depends+on+recG." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conjugational+recombination+in+resolvase-deficient+ruvC+mutants+of+Escherichia+coli+depends+on+recG." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conjugational+recombination+in+resolvase-deficient+ruvC+mutants+of+Escherichia+coli+depends+on+recG.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0001242-Paques1" id="pone.0001242-Paques1"></a>Paques F, Haber JE (1999) Multiple pathways of recombination induced by double-strand breaks in <em>Saccharomyces cerevisiae</em>. Microbiol Mol Biol Rev  63: 349–404.  <ul class="find" data-citedArticleID="1023956"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multiple+pathways+of+recombination+induced+by+double-strand+breaks+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Multiple+pathways+of+recombination+induced+by+double-strand+breaks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+pathways+of+recombination+induced+by+double-strand+breaks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+pathways+of+recombination+induced+by+double-strand+breaks+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0001242-Gloor1" id="pone.0001242-Gloor1"></a>Gloor GB, Nassif NA, Johnson-Schlitz DM, Preston CR, Engels WR (1991) Targeted gene replacement in <em>Drosophila</em> via P element-induced gap repair. Science  253: 1110–1117.  <ul class="find" data-citedArticleID="1023872" data-doi="10.1126/science.1653452"><li><a href="http://dx.doi.org/10.1126/science.1653452" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeted+gene+replacement+in+Drosophila+via+P+element-induced+gap+repair." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeted+gene+replacement+in+Drosophila+via+P+element-induced+gap+repair.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0001242-Nassif1" id="pone.0001242-Nassif1"></a>Nassif N, Penney J, Pal S, Engels WR, Gloor GB (1994) Efficient copying of nonhomologous sequences from ectopic sites via P-element-induced gap repair. Mol Cell Biol  14: 1613–1625.  <ul class="find" data-citedArticleID="1023952"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Efficient+copying+of+nonhomologous+sequences+from+ectopic+sites+via+P-element-induced+gap+repair.&amp;auth=&amp;atitle=Efficient+copying+of+nonhomologous+sequences+from+ectopic+sites+via+P-element-induced+gap+repair." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+copying+of+nonhomologous+sequences+from+ectopic+sites+via+P-element-induced+gap+repair." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+copying+of+nonhomologous+sequences+from+ectopic+sites+via+P-element-induced+gap+repair.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0001242-Allers1" id="pone.0001242-Allers1"></a>Allers T, Lichten M (2001) Differential timing and control of noncrossover and crossover recombination during meiosis. Cell  106: 47–57.  <ul class="find" data-citedArticleID="1023822" data-doi="10.1016/s0092-8674(01)00416-0"><li><a href="http://dx.doi.org/10.1016/s0092-8674(01)00416-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+timing+and+control+of+noncrossover+and+crossover+recombination+during+meiosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+timing+and+control+of+noncrossover+and+crossover+recombination+during+meiosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0001242-Szostak1" id="pone.0001242-Szostak1"></a>Szostak JW, Orr-Weaver TL, Rothstein RJ, Stahl FW (1983) The double-strand break repair model for recombination. Cell  33: 25–33.  <ul class="find" data-citedArticleID="1024020" data-doi="10.1016/0092-8674(83)90331-8"><li><a href="http://dx.doi.org/10.1016/0092-8674(83)90331-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+double-strand+break+repair+model+for+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+double-strand+break+repair+model+for+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">94.
              </span><a name="pone.0001242-Stitt1" id="pone.0001242-Stitt1"></a>Stitt BL, Mosig G (1989) Impaired expression of certain prereplicative bacteriophage T4 genes explains impaired T4 DNA synthesis in <em>Escherichia coli rho</em> (<em>nusD</em>) mutants. J Bacteriol  171: 3872–3880.  <ul class="find" data-citedArticleID="1024014"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Impaired+expression+of+certain+prereplicative+bacteriophage+T4+genes+explains+impaired+T4+DNA+synthesis+in+Escherichia+coli+rho+%28nusD%29+mutants.&amp;auth=&amp;atitle=Impaired+expression+of+certain+prereplicative+bacteriophage+T4+genes+explains+impaired+T4+DNA+synthesis+in+Escherichia+coli+rho+%28nusD%29+mutants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impaired+expression+of+certain+prereplicative+bacteriophage+T4+genes+explains+impaired+T4+DNA+synthesis+in+Escherichia+coli+rho+%28nusD%29+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impaired+expression+of+certain+prereplicative+bacteriophage+T4+genes+explains+impaired+T4+DNA+synthesis+in+Escherichia+coli+rho+%28nusD%29+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">95.
              </span><a name="pone.0001242-Zahradka1" id="pone.0001242-Zahradka1"></a>Zahradka K, Slade D, Bailone A, Sommer S, Averbeck D, et al.  (2006) Reassembly of shattered chromosomes in <em>Deinococcus radiodurans</em>. Nature  443: 569–573.  <ul class="find" data-citedArticleID="1024044" data-doi="10.1038/nature05160"><li><a href="http://dx.doi.org/10.1038/nature05160" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reassembly+of+shattered+chromosomes+in+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reassembly+of+shattered+chromosomes+in+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">96.
              </span><a name="pone.0001242-Leung1" id="pone.0001242-Leung1"></a>Leung W-Y, Malkova A, Haber JE (1997) Gene targeting by linear duplex DNA frequently occurs by assimilation of a single strand that is subject to preferential mismatch correction. Proc Natl Acad Sci U S A  94: 6851–6856.  <ul class="find" data-citedArticleID="1023906" data-doi="10.1073/pnas.94.13.6851"><li><a href="http://dx.doi.org/10.1073/pnas.94.13.6851" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+targeting+by+linear+duplex+DNA+frequently+occurs+by+assimilation+of+a+single+strand+that+is+subject+to+preferential+mismatch+correction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+targeting+by+linear+duplex+DNA+frequently+occurs+by+assimilation+of+a+single+strand+that+is+subject+to+preferential+mismatch+correction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">97.
              </span><a name="pone.0001242-Ellis1" id="pone.0001242-Ellis1"></a>Ellis HM, Yu D, DiTizio T, Court DL (2001) High efficiency mutagenesis, repair, and engineering of chromosomal DNA using single-stranded oligonucleotides. Proc Natl Acad Sci U S A  98: 6742–6746.  <ul class="find" data-citedArticleID="1023864" data-doi="10.1073/pnas.121164898"><li><a href="http://dx.doi.org/10.1073/pnas.121164898" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+efficiency+mutagenesis%2C+repair%2C+and+engineering+of+chromosomal+DNA+using+single-stranded+oligonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+efficiency+mutagenesis%2C+repair%2C+and+engineering+of+chromosomal+DNA+using+single-stranded+oligonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">98.
              </span><a name="pone.0001242-Court1" id="pone.0001242-Court1"></a>Court DL, Sawitzke JA, Thomason LC (2002) Genetic engineering using homologous recombination. Annu Rev Genet  36: 361–388.  <ul class="find" data-citedArticleID="1023848" data-doi="10.1146/annurev.genet.36.061102.093104"><li><a href="http://dx.doi.org/10.1146/annurev.genet.36.061102.093104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+engineering+using+homologous+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+engineering+using+homologous+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">99.
              </span><a name="pone.0001242-Storici1" id="pone.0001242-Storici1"></a>Storici F, Durham CL, Gordenin DA, Resnick MA (2003) Chromosomal site-specific double-strand breaks are efficiently targeted for repair by oligonucleotides in yeast. Proc Natl Acad Sci U S A  100: 14994–14999.  <ul class="find" data-citedArticleID="1024016" data-doi="10.1073/pnas.2036296100"><li><a href="http://dx.doi.org/10.1073/pnas.2036296100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+site-specific+double-strand+breaks+are+efficiently+targeted+for+repair+by+oligonucleotides+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+site-specific+double-strand+breaks+are+efficiently+targeted+for+repair+by+oligonucleotides+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">100.
              </span><a name="pone.0001242-Storici2" id="pone.0001242-Storici2"></a>Storici F, Snipe JR, Chan GK, Gordenin DA, Resnick MA (2006) Conservative repair of a chromosomal double-strand break by single-strand DNA through two steps of annealing. Mol Cell Biol  26: 7645–7657.  <ul class="find" data-citedArticleID="1024018" data-doi="10.1128/mcb.00672-06"><li><a href="http://dx.doi.org/10.1128/mcb.00672-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservative+repair+of+a+chromosomal+double-strand+break+by+single-strand+DNA+through+two+steps+of+annealing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservative+repair+of+a+chromosomal+double-strand+break+by+single-strand+DNA+through+two+steps+of+annealing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">101.
              </span><a name="pone.0001242-Feiss1" id="pone.0001242-Feiss1"></a>Feiss M, Becker A (1983)  DNA packaging and cutting. In: Hendrix RW, Roberts JW, Stahl FW, Weisberg RW, editors. Lambda II. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press. pp. 305–350.  <ul class="find-nolinks"></ul></li><li><span class="label">102.
              </span><a name="pone.0001242-Thaler3" id="pone.0001242-Thaler3"></a>Thaler DS, Stahl FW (1988) DNA double-chain breaks in recombination of phage λ and yeast. Annu Rev Genet  22: 169–197.  <ul class="find" data-citedArticleID="1024030" data-doi="10.1146/annurev.ge.22.120188.001125"><li><a href="http://dx.doi.org/10.1146/annurev.ge.22.120188.001125" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+double-chain+breaks+in+recombination+of+phage+%CE%BB+and+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+double-chain+breaks+in+recombination+of+phage+%CE%BB+and+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">103.
              </span><a name="pone.0001242-Bachmann1" id="pone.0001242-Bachmann1"></a>Bachmann BJ (1996)  Derivatives and genotypes of some mutant derivatives of <em>Escherichia coli</em> K-12. In: Neidhardt FC, Curtiss R III, Ingraham JL, Lin ECC, Low KB, et al., editors. <em>Escherichia coli</em> and <em>Salmonella</em>: Cellular and Molecular Biology. 2nd ed. Washington, D. C.: American Society for Microbiology. pp. 2460–2488.  <ul class="find-nolinks"></ul></li><li><span class="label">104.
              </span><a name="pone.0001242-Woodcock1" id="pone.0001242-Woodcock1"></a>Woodcock DM, Crowther PJ, Doherty J, Jefferson S, DeCruz E, et al.  (1989) Quantitative evaluation of <em>Escherichia coli</em> host strains for tolerance to cytosine methylation in plasmid and phage recombinants. Nucleic Acids Res  17: 3469–3478.  <ul class="find" data-citedArticleID="1024042" data-doi="10.1093/nar/17.9.3469"><li><a href="http://dx.doi.org/10.1093/nar/17.9.3469" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+evaluation+of+Escherichia+coli+host+strains+for+tolerance+to+cytosine+methylation+in+plasmid+and+phage+recombinants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+evaluation+of+Escherichia+coli+host+strains+for+tolerance+to+cytosine+methylation+in+plasmid+and+phage+recombinants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">105.
              </span><a name="pone.0001242-Baba1" id="pone.0001242-Baba1"></a>Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, et al.  (2006) Construction of <em>Escherichia coli</em> K-12 in-frame, single-gene knockout mutants: the Keio collection. Mol Syst Biol  2: 2006 0008.  <ul class="find" data-citedArticleID="1023830" data-doi="10.1038/msb4100050"><li><a href="http://dx.doi.org/10.1038/msb4100050" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+Escherichia+coli+K-12+in-frame%2C+single-gene+knockout+mutants%3A+the+Keio+collection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+Escherichia+coli+K-12+in-frame%2C+single-gene+knockout+mutants%3A+the+Keio+collection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">106.
              </span><a name="pone.0001242-GumbinerRusso1" id="pone.0001242-GumbinerRusso1"></a>Gumbiner-Russo LM, Lombardo MJ, Ponder RG, Rosenberg SM (2001) The TGV <em>t</em>rans<em>g</em>enic <em>v</em>ectors for single-copy gene expression from the <em>Escherichia coli</em> chromosome. Gene  273: 97–104.  <ul class="find" data-citedArticleID="1023876" data-doi="10.1016/s0378-1119(01)00565-0"><li><a href="http://dx.doi.org/10.1016/s0378-1119(01)00565-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+TGV+transgenic+vectors+for+single-copy+gene+expression+from+the+Escherichia+coli+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+TGV+transgenic+vectors+for+single-copy+gene+expression+from+the+Escherichia+coli+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">107.
              </span><a name="pone.0001242-Harris2" id="pone.0001242-Harris2"></a>Harris RS, Longerich S, Rosenberg SM (1994) Recombination in adaptive mutation. Science  264: 258–260.  <ul class="find" data-citedArticleID="1023882" data-doi="10.1126/science.8146657"><li><a href="http://dx.doi.org/10.1126/science.8146657" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+in+adaptive+mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+in+adaptive+mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">108.
              </span><a name="pone.0001242-Signer1" id="pone.0001242-Signer1"></a>Signer ER, Weil J (1968) Recombination in bacteriophage lambda. I. Mutants deficient in general recombination. J Mol Biol  34: 261–271.  <ul class="find" data-citedArticleID="1023998" data-doi="10.1016/0022-2836(68)90251-9"><li><a href="http://dx.doi.org/10.1016/0022-2836(68)90251-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+in+bacteriophage+lambda.+I.+Mutants+deficient+in+general+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+in+bacteriophage+lambda.+I.+Mutants+deficient+in+general+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">109.
              </span><a name="pone.0001242-Zissler1" id="pone.0001242-Zissler1"></a>Zissler J, Signer E, Schaefer F (1971)  The role of recombination in the growth of bacteriophage lambda. I. The gamma gene. In: Hershey AD, editor. The Bacteriophage Lambda. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory. pp. 455–468.  <ul class="find-nolinks"></ul></li><li><span class="label">110.
              </span><a name="pone.0001242-Campbell1" id="pone.0001242-Campbell1"></a>Campbell A (1971)  Genetic structure. In: Hershey AD, editor. The Bacteriophage Lambda. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory. pp. 13–14.  <ul class="find-nolinks"></ul></li><li><span class="label">111.
              </span><a name="pone.0001242-Murray1" id="pone.0001242-Murray1"></a>Murray NE, Murray K (1974) Manipulation of restriction targets in phage lambda to form receptor chromosomes for DNA fragments. Nature  251: 476–481.  <ul class="find" data-citedArticleID="1023942" data-doi="10.1038/251476a0"><li><a href="http://dx.doi.org/10.1038/251476a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Manipulation+of+restriction+targets+in+phage+lambda+to+form+receptor+chromosomes+for+DNA+fragments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Manipulation+of+restriction+targets+in+phage+lambda+to+form+receptor+chromosomes+for+DNA+fragments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">112.
              </span><a name="pone.0001242-Stahl2" id="pone.0001242-Stahl2"></a>Stahl FW, Crasemann JM, Stahl MM (1975) Rec-mediated recombinational hotspot activity in bacteriophage lambda III. Chi mutations are site mutations stimulating Rec-mediated recombination. J Mol Biol  94: 203–212.  <ul class="find" data-citedArticleID="1024012" data-doi="10.1016/0022-2836(75)90078-9"><li><a href="http://dx.doi.org/10.1016/0022-2836(75)90078-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rec-mediated+recombinational+hotspot+activity+in+bacteriophage+lambda+III.+Chi+mutations+are+site+mutations+stimulating+Rec-mediated+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rec-mediated+recombinational+hotspot+activity+in+bacteriophage+lambda+III.+Chi+mutations+are+site+mutations+stimulating+Rec-mediated+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">113.
              </span><a name="pone.0001242-Goldberg1" id="pone.0001242-Goldberg1"></a>Goldberg AR, Howe M (1969) New mutations in the <em>S</em> cistron of bacteriophage lambda affecting host cell lysis. Virology  38: 200–202.  <ul class="find" data-citedArticleID="1023874" data-doi="10.1016/0042-6822(69)90148-2"><li><a href="http://dx.doi.org/10.1016/0042-6822(69)90148-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+mutations+in+the+S+cistron+of+bacteriophage+lambda+affecting+host+cell+lysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+mutations+in+the+S+cistron+of+bacteriophage+lambda+affecting+host+cell+lysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.XML" value="200766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.PDF" value="849971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g001.PNG_L" value="223162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g001.PNG_M" value="107802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g001.PNG_S" value="13116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g001.TIF" value="280154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g001.PNG_I" value="37656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t001.PNG_L" value="563802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t001.PNG_M" value="209823"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t001.PNG_S" value="18515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t001.TIF" value="2247938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t001.PNG_I" value="91635"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t002.PNG_L" value="187109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t002.PNG_M" value="101058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t002.PNG_S" value="10087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t002.TIF" value="686930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t002.PNG_I" value="31027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g002.PNG_L" value="170825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g002.PNG_M" value="111115"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g002.PNG_S" value="9358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g002.TIF" value="217882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g002.PNG_I" value="40077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g003.PNG_L" value="164535"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g003.PNG_M" value="72852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g003.PNG_S" value="11601"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g003.TIF" value="218796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g003.PNG_I" value="63119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g004.PNG_L" value="383162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g004.PNG_M" value="229001"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g004.PNG_S" value="12786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g004.TIF" value="506038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g004.PNG_I" value="88832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g005.PNG_L" value="262721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g005.PNG_M" value="128476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g005.PNG_S" value="12242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g005.TIF" value="346772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g005.PNG_I" value="63630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g006.PNG_L" value="158724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g006.PNG_M" value="158724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g006.PNG_S" value="12379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g006.TIF" value="224902"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g006.PNG_I" value="85368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g007.PNG_L" value="127782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g007.PNG_M" value="127782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g007.PNG_S" value="12993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g007.TIF" value="169824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g007.PNG_I" value="87742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g008.PNG_L" value="149193"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g008.PNG_M" value="70048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g008.PNG_S" value="11363"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g008.TIF" value="206160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g008.PNG_I" value="41375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g009.PNG_L" value="134320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g009.PNG_M" value="86626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g009.PNG_S" value="11613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g009.TIF" value="171652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.g009.PNG_I" value="51742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t003.PNG_L" value="120589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t003.PNG_M" value="65403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t003.PNG_S" value="8972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t003.TIF" value="531108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001242.t003.PNG_I" value="20300"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001242&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001242" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001242&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001242&volume=&issue=&title=Physical Analyses of E. coli Heteroduplex Recombination Products In Vivo: On the Prevalence of 5′ and 3′ Patches&author_name=Laura%20M.%20Gumbiner-Russo%2C%20Susan%20M.%20Rosenberg&start_page=1&end_page=15" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001242" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001242" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001242" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001242&amp;t=Physical%20Analyses%20of%20E.%20coli%20Heteroduplex%20Recombination%20Products%20In%20Vivo%3A%20On%20the%20Prevalence%20of%205%E2%80%B2%20and%203%E2%80%B2%20Patches" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001242&title=Physical%20Analyses%20of%20E.%20coli%20Heteroduplex%20Recombination%20Products%20In%20Vivo%3A%20On%20the%20Prevalence%20of%205%E2%80%B2%20and%203%E2%80%B2%20Patches&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001242&amp;title=Physical%20Analyses%20of%20E.%20coli%20Heteroduplex%20Recombination%20Products%20In%20Vivo%3A%20On%20the%20Prevalence%20of%205%E2%80%B2%20and%203%E2%80%B2%20Patches" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001242" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001242&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Physical Analyses of E. coli Heteroduplex Recombination Products In Vivo: On the Prevalence of 5′ and 3′ Patches';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001242';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Physical%20Analyses%20of%20E.%20coli%20Heteroduplex%20Recombination%20Products%20In%20Vivo%3A%20On%20the%20Prevalence%20of%205%E2%80%B2%20and%203%E2%80%B2%20Patches http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001242" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001242" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacteriophages%22" title="Search for articles in the subject area:'Bacteriophages'"><div class="flagText">Bacteriophages</div></a>
              <div data-categoryid="43083" data-articleid="25316"
                   data-categoryname="Bacteriophages"
                   class="flagImage" title="Flag 'Bacteriophages' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+recombination%22" title="Search for articles in the subject area:'DNA recombination'"><div class="flagText">DNA recombination</div></a>
              <div data-categoryid="33483" data-articleid="25316"
                   data-categoryname="DNA recombination"
                   class="flagImage" title="Flag 'DNA recombination' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+repair%22" title="Search for articles in the subject area:'DNA repair'"><div class="flagText">DNA repair</div></a>
              <div data-categoryid="17453" data-articleid="25316"
                   data-categoryname="DNA repair"
                   class="flagImage" title="Flag 'DNA repair' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+replication%22" title="Search for articles in the subject area:'DNA replication'"><div class="flagText">DNA replication</div></a>
              <div data-categoryid="19773" data-articleid="25316"
                   data-categoryname="DNA replication"
                   class="flagImage" title="Flag 'DNA replication' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+prediction%22" title="Search for articles in the subject area:'Gene prediction'"><div class="flagText">Gene prediction</div></a>
              <div data-categoryid="40279" data-articleid="25316"
                   data-categoryname="Gene prediction"
                   class="flagImage" title="Flag 'Gene prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Homologous+recombination%22" title="Search for articles in the subject area:'Homologous recombination'"><div class="flagText">Homologous recombination</div></a>
              <div data-categoryid="34037" data-articleid="25316"
                   data-categoryname="Homologous recombination"
                   class="flagImage" title="Flag 'Homologous recombination' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Probe+hybridization%22" title="Search for articles in the subject area:'Probe hybridization'"><div class="flagText">Probe hybridization</div></a>
              <div data-categoryid="19653" data-articleid="25316"
                   data-categoryname="Probe hybridization"
                   class="flagImage" title="Flag 'Probe hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombinant+proteins%22" title="Search for articles in the subject area:'Recombinant proteins'"><div class="flagText">Recombinant proteins</div></a>
              <div data-categoryid="18471" data-articleid="25316"
                   data-categoryname="Recombinant proteins"
                   class="flagImage" title="Flag 'Recombinant proteins' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9219'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3902'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2021&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
