

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The CD81 Partner EWI-2wint Inhibits Hepatitis C Virus Entry</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001866"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001866" />

    <meta name="citation_title" content="The CD81 Partner EWI-2wint Inhibits Hepatitis C Virus Entry"/>
    <meta itemprop="name" content="The CD81 Partner EWI-2wint Inhibits Hepatitis C Virus Entry"/>

      <meta name="citation_author" content="Vera Rocha-Perugini"/>
            <meta name="citation_author_institution" content="Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France"/>
      <meta name="citation_author" content="Claire Montpellier"/>
            <meta name="citation_author_institution" content="Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France"/>
      <meta name="citation_author" content="David Delgrange"/>
            <meta name="citation_author_institution" content="Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France"/>
      <meta name="citation_author" content="Czeslaw Wychowski"/>
            <meta name="citation_author_institution" content="Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France"/>
      <meta name="citation_author" content="François Helle"/>
            <meta name="citation_author_institution" content="Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France"/>
      <meta name="citation_author" content="André Pillez"/>
            <meta name="citation_author_institution" content="Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France"/>
      <meta name="citation_author" content="Hervé Drobecq"/>
            <meta name="citation_author_institution" content="Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France"/>
      <meta name="citation_author" content="François Le Naour"/>
            <meta name="citation_author_institution" content="INSERM-U602, Institut André-Lwoff, Université Paris XI, Hôpital Paul Brousse, Villejuif, France"/>
      <meta name="citation_author" content="Stéphanie Charrin"/>
            <meta name="citation_author_institution" content="INSERM-U602, Institut André-Lwoff, Université Paris XI, Hôpital Paul Brousse, Villejuif, France"/>
      <meta name="citation_author" content="Shoshana Levy"/>
            <meta name="citation_author_institution" content="Division of Oncology, Department of Medicine, Stanford University Medical Center, Stanford, California, United States of America"/>
      <meta name="citation_author" content="Eric Rubinstein"/>
            <meta name="citation_author_institution" content="INSERM-U602, Institut André-Lwoff, Université Paris XI, Hôpital Paul Brousse, Villejuif, France"/>
      <meta name="citation_author" content="Jean Dubuisson"/>
            <meta name="citation_author_institution" content="Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France"/>
      <meta name="citation_author" content="Laurence Cocquerel"/>
            <meta name="citation_author_institution" content="Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France"/>
            <meta name="citation_author_institution" content="Division of Oncology, Department of Medicine, Stanford University Medical Center, Stanford, California, United States of America"/>

    <meta name="citation_date" content="2008/4/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001866.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1866"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Hepatitis C Virus.; citation_author=SM Lemon; citation_author=C Walker; citation_author=MJ Alter; citation_author=M Yi; citation_number=1; citation_pages=1253-1304; citation_date=2007; citation_publisher=Lippincott Williams & Wilkins; " />
      <meta name="citation_reference" content="citation_title=Treating viral hepatitis C: efficacy, side effects, and complications.; citation_author=MP Manns; citation_author=H Wedemeyer; citation_author=M Cornberg; citation_journal_title=Gut; citation_volume=55; citation_number=2; citation_pages=1350-1359; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Highly infectious hepatitis C pseudo-viruses containing functional E1E2 envelope protein complexes.; citation_author=B Bartosch; citation_author=J Dubuisson; citation_author=F-L Cosset; citation_journal_title=J Exp Med; citation_volume=197; citation_number=3; citation_pages=633-642; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Hepatitis C virus glycoproteins mediate pH-dependent cell entry of pseudotyped retroviral particles.; citation_author=M Hsu; citation_author=J Zhang; citation_author=M Flint; citation_author=C Logvinoff; citation_author=C Cheng-Mayer; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=4; citation_pages=7271-7276; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Complete replication of hepatitis C virus in cell culture.; citation_author=BD Lindenbach; citation_author=MJ Evans; citation_author=AJ Syder; citation_author=B Wolk; citation_author=TL Tellinghuisen; citation_journal_title=Science; citation_volume=309; citation_number=5; citation_pages=623-626; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Production of infectious hepatitis C virus in tissue culture from a cloned viral genome.; citation_author=T Wakita; citation_author=T Pietschmann; citation_author=T Kato; citation_author=T Date; citation_author=M Miyamoto; citation_journal_title=Nat Med; citation_volume=11; citation_number=6; citation_pages=791-796; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Robust hepatitis C virus infection in vitro.; citation_author=J Zhong; citation_author=P Gastaminza; citation_author=G Cheng; citation_author=S Kapadia; citation_author=T Kato; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=7; citation_pages=9294-9299; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Biogenesis of hepatitis C virus envelope glycoproteins.; citation_author=A Op De Beeck; citation_author=L Cocquerel; citation_author=J Dubuisson; citation_journal_title=J Gen Virol; citation_volume=82; citation_number=8; citation_pages=2589-2595; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Early steps of the hepatitis C virus life cycle.; citation_author=J Dubuisson; citation_author=F Helle; citation_author=L Cocquerel; citation_journal_title=Cell Microbiol in press; citation_number=9; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cell entry of hepatitis C virus requires a set of co-receptors that include the CD81 tetraspanin and the SR-B1 scavenger receptor.; citation_author=B Bartosch; citation_author=A Vitelli; citation_author=C Granier; citation_author=C Goujon; citation_author=J Dubuisson; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=10; citation_pages=41624-41630; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Claudin-1 is a hepatitis C virus co-receptor required for a late step in entry.; citation_author=MJ Evans; citation_author=T von Hahn; citation_author=DM Tscherne; citation_author=AJ Syder; citation_author=M Panis; citation_journal_title=Nature; citation_volume=446; citation_number=11; citation_pages=801-805; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Claudin-6 and claudin-9 function as additional coreceptors for hepatitis C virus.; citation_author=A Zheng; citation_author=F Yuan; citation_author=Y Li; citation_author=F Zhu; citation_author=P Hou; citation_journal_title=J Virol; citation_volume=81; citation_number=12; citation_pages=12465-12471; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Correlation of the tight junction-like distribution of claudin-1 to the cellular tropism of HCV.; citation_author=W Yang; citation_author=C Qiu; citation_author=N Biswas; citation_author=J Jin; citation_author=SC Watkins; citation_journal_title=J Biol Chem; citation_number=13; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Tetraspanins.; citation_author=C Boucheix; citation_author=E Rubinstein; citation_journal_title=Cell Mol Life Sci; citation_volume=58; citation_number=14; citation_pages=1189-1205; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The tetraspanin web modulates immune-signalling complexes.; citation_author=S Levy; citation_author=T Shoham; citation_journal_title=Nat Rev Immunol; citation_volume=5; citation_number=15; citation_pages=136-148; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Protein-protein interactions in the tetraspanin web.; citation_author=S Levy; citation_author=T Shoham; citation_journal_title=Physiology (Bethesda); citation_volume=20; citation_number=16; citation_pages=218-224; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=CD9, CD63, CD81, and CD82 are components of a surface tetraspan network connected to HLA-DR and VLA integrins.; citation_author=E Rubinstein; citation_author=F Le Naour; citation_author=C Lagaudriere-Gesbert; citation_author=M Billard; citation_author=H Conjeaud; citation_journal_title=Eur J Immunol; citation_volume=26; citation_number=17; citation_pages=2657-2665; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=EWI-2 is a new component of the tetraspanin web in hepatocytes and lymphoid cells.; citation_author=S Charrin; citation_author=F Le Naour; citation_author=V Labas; citation_author=M Billard; citation_author=JP Le Caer; citation_journal_title=Biochem J; citation_volume=373; citation_number=18; citation_pages=409-421; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The major CD9 and CD81 molecular partner. Identification and characterization of the complexes.; citation_author=S Charrin; citation_author=F Le Naour; citation_author=M Oualid; citation_author=M Billard; citation_author=G Faure; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=19; citation_pages=14329-14337; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=PGRL is a major CD81-associated protein on lymphocytes and distinguishes a new family of cell surface proteins.; citation_author=KL Clark; citation_author=Z Zeng; citation_author=AL Langford; citation_author=SM Bowen; citation_author=SC Todd; citation_journal_title=J Immunol; citation_volume=167; citation_number=20; citation_pages=5115-5121; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=EWI-2 is a major CD9 and CD81 partner and member of a novel Ig protein subfamily.; citation_author=CS Stipp; citation_author=TV Kolesnikova; citation_author=ME Hemler; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=21; citation_pages=40545-40554; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=FPRP, a major, highly stoichiometric, highly specific CD81- and CD9- associated protein.; citation_author=CS Stipp; citation_author=D Orlicky; citation_author=ME Hemler; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=22; citation_pages=4853-4862; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=EWI-2 and EWI-F link the tetraspanin web to the actin cytoskeleton through their direct association with ezrin-radixin-moesin proteins.; citation_author=M Sala-Valdes; citation_author=A Ursa; citation_author=S Charrin; citation_author=E Rubinstein; citation_author=ME Hemler; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=23; citation_pages=19665-19675; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=EWI-2 modulates lymphocyte integrin alpha4beta1 functions.; citation_author=TV Kolesnikova; citation_author=CS Stipp; citation_author=RM Rao; citation_author=WS Lane; citation_author=FW Luscinskas; citation_journal_title=Blood; citation_volume=103; citation_number=24; citation_pages=3013-3019; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=EWI-2 regulates alpha3beta1 integrin-dependent cell functions on laminin-5.; citation_author=CS Stipp; citation_author=TV Kolesnikova; citation_author=ME Hemler; citation_journal_title=J Cell Biol; citation_volume=163; citation_number=25; citation_pages=1167-1177; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=EWI2/PGRL associates with the metastasis suppressor KAI1/CD82 and inhibits the migration of prostate cancer cells.; citation_author=XA Zhang; citation_author=WS Lane; citation_author=S Charrin; citation_author=E Rubinstein; citation_author=L Liu; citation_journal_title=Cancer Res; citation_volume=63; citation_number=26; citation_pages=2665-2674; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Recognition of native hepatitis C virus E1E2 heterodimers by a human monoclonal antibody.; citation_author=L Cocquerel; citation_author=ER Quinn; citation_author=M Flint; citation_author=KG Hadlock; citation_author=SK Foung; citation_journal_title=J Virol; citation_volume=77; citation_number=27; citation_pages=1604-1609; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of amino acid residues in CD81 critical for interaction with hepatitis C virus envelope glycoprotein E2.; citation_author=A Higginbottom; citation_author=ER Quinn; citation_author=CC Kuo; citation_author=M Flint; citation_author=LH Wilson; citation_journal_title=J Virol; citation_volume=74; citation_number=28; citation_pages=3642-3649; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=An extracellular site on tetraspanin CD151 determines alpha 3 and alpha 6 integrin-dependent cellular morphology.; citation_author=AR Kazarov; citation_author=X Yang; citation_author=CS Stipp; citation_author=B Sehgal; citation_author=ME Hemler; citation_journal_title=J Cell Biol; citation_volume=158; citation_number=29; citation_pages=1299-1309; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Contrasting effects of EWI proteins, integrins, and protein palmitoylation on cell surface CD9 organization.; citation_author=XH Yang; citation_author=OV Kovalenko; citation_author=TV Kolesnikova; citation_author=MM Andzelm; citation_author=E Rubinstein; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=30; citation_pages=12976-12985; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Initiation of Hepatitis C Virus Infection Is Dependent on Cholesterol and Cooperativity between CD81 and Scavenger Receptor B Type I.; citation_author=SB Kapadia; citation_author=H Barth; citation_author=T Baumert; citation_author=JA McKeating; citation_author=FV Chisari; citation_journal_title=J Virol; citation_volume=81; citation_number=31; citation_pages=374-383; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The level of CD81 cell surface expression is a key determinant for productive entry of hepatitis C virus into host cells.; citation_author=G Koutsoudakis; citation_author=E Herrmann; citation_author=S Kallis; citation_author=R Bartenschlager; citation_author=T Pietschmann; citation_journal_title=J Virol; citation_volume=81; citation_number=32; citation_pages=588-598; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Serum-derived hepatitis C virus infection of primary human hepatocytes is tetraspanin CD81 dependent.; citation_author=S Molina; citation_author=V Castet; citation_author=L Pichard-Garcia; citation_author=C Wychowski; citation_author=E Meurs; citation_journal_title=J Virol; citation_volume=82; citation_number=33; citation_pages=569-574; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=CD81 is an entry coreceptor for hepatitis C virus.; citation_author=EG Cormier; citation_author=F Tsamis; citation_author=F Kajumo; citation_author=RJ Durso; citation_author=JP Gardner; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=34; citation_pages=7270-7274; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Diverse CD81 proteins support hepatitis C virus infection.; citation_author=M Flint; citation_author=T von Hahn; citation_author=J Zhang; citation_author=M Farquhar; citation_author=CT Jones; citation_journal_title=J Virol; citation_volume=80; citation_number=35; citation_pages=11331-11342; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Characterization of the early steps of hepatitis C virus infection by using luciferase reporter viruses.; citation_author=G Koutsoudakis; citation_author=A Kaul; citation_author=E Steinmann; citation_author=S Kallis; citation_author=V Lohmann; citation_journal_title=J Virol; citation_volume=80; citation_number=36; citation_pages=5308-5320; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The tight junction proteins claudin-1, -6 and -9 are entry cofactors for the Hepatitis C virus.; citation_author=L Meertens; citation_author=C Bertaux; citation_author=L Cukierman; citation_author=E Cormier; citation_author=D Lavillette; citation_journal_title=J Virol; citation_number=37; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Cytoskeletal requirements for hepatitis C virus (HCV) RNA synthesis in the HCV replicon cell culture system.; citation_author=AG Bost; citation_author=D Venable; citation_author=L Liu; citation_author=BA Heinz; citation_journal_title=J Virol; citation_volume=77; citation_number=38; citation_pages=4401-4408; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Co-ligation of CD81 and the B-cell receptor by Hepatitis C Virus envelope protein E2 enhances B-cell signaling.; citation_author=G Coffey; citation_author=C-C Kuo; citation_author=S Levy; citation_number=39; " />
      <meta name="citation_reference" content="citation_title=Cytoskeleton rearrangement induced by tetraspanin engagement modulates the activation of T and NK cells.; citation_author=S Crotta; citation_author=V Ronconi; citation_author=C Ulivieri; citation_author=CT Baldari; citation_author=NM Valiante; citation_journal_title=Eur J Immunol; citation_volume=36; citation_number=40; citation_pages=919-929; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Hepatitis C virus entry: potential receptors and their biological functions.; citation_author=L Cocquerel; citation_author=C Voisset; citation_author=J Dubuisson; citation_journal_title=J Gen Virol; citation_volume=87; citation_number=41; citation_pages=1075-1084; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=CD81-dependent binding of hepatitis C virus E1E2 heterodimers.; citation_author=L Cocquerel; citation_author=CC Kuo; citation_author=J Dubuisson; citation_author=S Levy; citation_journal_title=J Virol; citation_volume=77; citation_number=42; citation_pages=10677-10683; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=TAPA-1, the target of an antiproliferative antibody, defines a new family of transmembrane proteins.; citation_author=R Oren; citation_author=S Takahashi; citation_author=C Doss; citation_author=R Levy; citation_author=S Levy; citation_journal_title=Mol Cell Biol; citation_volume=10; citation_number=43; citation_pages=4007-4015; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=A retention signal necessary and sufficient for endoplasmic reticulum localization maps to the transmembrane domain of hepatitis C virus glycoprotein E2.; citation_author=L Cocquerel; citation_author=JC Meunier; citation_author=A Pillez; citation_author=C Wychowski; citation_author=J Dubuisson; citation_journal_title=J Virol; citation_volume=72; citation_number=44; citation_pages=2183-2191; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Characterization of hepatitis C virus E2 glycoprotein interaction with a putative cellular receptor, CD81.; citation_author=M Flint; citation_author=C Maidens; citation_author=LD Loomis-Price; citation_author=C Shotton; citation_author=J Dubuisson; citation_journal_title=J Virol; citation_volume=73; citation_number=45; citation_pages=6235-6244; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Nonenveloped nucleocapsids of hepatitis C virus in the serum of infected patients.; citation_author=P Maillard; citation_author=K Krawczynski; citation_author=J Nitkiewicz; citation_author=C Bronnert; citation_author=M Sidorkiewicz; citation_journal_title=J Virol; citation_volume=75; citation_number=46; citation_pages=8240-8250; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Robust production of infectious viral particles in Huh-7 cells by introducing mutations in hepatitis C virus structural proteins.; citation_author=D Delgrange; citation_author=A Pillez; citation_author=S Castelain; citation_author=L Cocquerel; citation_author=Y Rouille; citation_journal_title=J Gen Virol; citation_volume=88; citation_number=47; citation_pages=2495-2503; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Time- and temperature-dependent activation of hepatitis C virus for low-pH-triggered entry.; citation_author=DM Tscherne; citation_author=CT Jones; citation_author=MJ Evans; citation_author=BD Lindenbach; citation_author=JA McKeating; citation_journal_title=J Virol; citation_volume=80; citation_number=48; citation_pages=1734-1741; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Efficient replication of the genotype 2a hepatitis C virus subgenomic replicon.; citation_author=T Kato; citation_author=T Date; citation_author=M Miyamoto; citation_author=A Furusaka; citation_author=K Tokushige; citation_journal_title=Gastroenterology; citation_volume=125; citation_number=49; citation_pages=1808-1817; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Characterization of functional hepatitis C virus envelope glycoproteins.; citation_author=A Op De Beeck; citation_author=C Voisset; citation_author=B Bartosch; citation_author=Y Ciczora; citation_author=L Cocquerel; citation_journal_title=J Virol; citation_volume=78; citation_number=50; citation_pages=2994-3002; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Lentiviral vectors pseudotyped with a modified RD114 envelope glycoprotein show increased stability in sera and augmented transduction of primary lymphocytes and CD34+ cells derived from human and nonhuman primates.; citation_author=V Sandrin; citation_author=B Boson; citation_author=P Salmon; citation_author=W Gay; citation_author=D Negre; citation_journal_title=Blood; citation_volume=100; citation_number=51; citation_pages=823-832; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001866" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The CD81 Partner EWI-2wint Inhibits Hepatitis C Virus Entry"/>
    <meta name="twitter:description" content="Two to three percent of the world's population is chronically infected with hepatitis C virus (HCV) and thus at risk of developing liver cancer. Although precise mechanisms regulating HCV entry into hepatic cells are still unknown, several cell surface proteins have been identified as entry factors for this virus. Among these molecules, the tetraspanin CD81 is essential for HCV entry. Here, we have identified a partner of CD81, EWI-2wint, which is expressed in several cell lines but not in hepatocytes. Ectopic expression of EWI-2wint in a hepatoma cell line susceptible to HCV infection blocked viral entry by inhibiting the interaction between the HCV envelope glycoproteins and CD81. This finding suggests that, in addition to the presence of specific entry factors in the hepatocytes, the lack of a specific inhibitor can contribute to the hepatotropism of HCV. This is the first example of a pathogen gaining entry into host cells that lack a specific inhibitory factor."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001866.g007"/>

  <meta property="og:title" content="The CD81 Partner EWI-2wint Inhibits Hepatitis C Virus Entry" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001866" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3356'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5428'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=929&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001866">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001866" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001866&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The CD81 Partner EWI-2wint Inhibits Hepatitis C Virus Entry
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vera Rocha-Perugini
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Vera Rocha-Perugini, Claire Montpellier</p>

              
              

                <p>Affiliation:
                  Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claire Montpellier
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Vera Rocha-Perugini, Claire Montpellier</p>

              
              

                <p>Affiliation:
                  Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Delgrange, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Czeslaw Wychowski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            François Helle, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            André Pillez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hervé Drobecq, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            François Le Naour, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  INSERM-U602, Institut André-Lwoff, Université Paris XI, Hôpital Paul Brousse, Villejuif, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stéphanie Charrin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  INSERM-U602, Institut André-Lwoff, Université Paris XI, Hôpital Paul Brousse, Villejuif, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shoshana Levy, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Oncology, Department of Medicine, Stanford University Medical Center, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eric Rubinstein, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  INSERM-U602, Institut André-Lwoff, Université Paris XI, Hôpital Paul Brousse, Villejuif, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean Dubuisson
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:jean.dubuisson@ibl.fr">jean.dubuisson@ibl.fr</a> (JD); <a href="mailto:laurence.cocquerel@ibl.fr">laurence.cocquerel@ibl.fr</a> (LC)</p>

                <p>Affiliation:
                  Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laurence Cocquerel
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:jean.dubuisson@ibl.fr">jean.dubuisson@ibl.fr</a> (JD); <a href="mailto:laurence.cocquerel@ibl.fr">laurence.cocquerel@ibl.fr</a> (LC)</p>

                <p>Affiliations:
                  Institut de Biologie de Lille (UMR8161), CNRS, Universités de Lille I et Lille II, Institut Pasteur de Lille, Lille, France, 
                  Division of Oncology, Department of Medicine, Stanford University Medical Center, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001866</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001866" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001866" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001866" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001866" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001866-g001" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001866-t001" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001866-g002" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001866-g003" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001866-t002" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001866-g004" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001866-g005" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001866-g006" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001866-g007" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001866">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001866" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Two to three percent of the world's population is chronically infected with hepatitis C virus (HCV) and thus at risk of developing liver cancer. Although precise mechanisms regulating HCV entry into hepatic cells are still unknown, several cell surface proteins have been identified as entry factors for this virus. Among these molecules, the tetraspanin CD81 is essential for HCV entry. Here, we have identified a partner of CD81, EWI-2wint, which is expressed in several cell lines but not in hepatocytes. Ectopic expression of EWI-2wint in a hepatoma cell line susceptible to HCV infection blocked viral entry by inhibiting the interaction between the HCV envelope glycoproteins and CD81. This finding suggests that, in addition to the presence of specific entry factors in the hepatocytes, the lack of a specific inhibitor can contribute to the hepatotropism of HCV. This is the first example of a pathogen gaining entry into host cells that lack a specific inhibitory factor.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Rocha-Perugini V, Montpellier C, Delgrange D, Wychowski C, Helle F, et al.  (2008) The CD81 Partner EWI-2wint Inhibits Hepatitis C Virus Entry. PLoS ONE 3(4):
          e1866.
            doi:10.1371/journal.pone.0001866</p><p><strong>Editor: </strong>Nina Papavasiliou, The Rockefeller University, United States of America</p><p><strong>Received:</strong> August 9, 2007; <strong>Accepted:</strong> February 18, 2008; <strong>Published:</strong> April 2, 2008</p><p><strong>Copyright:</strong> © 2008 Rocha-Perugini et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the “Institut Fédératif de Recherche-142” (IFR142) and by grants from the CNRS and the « Agence Nationale de Recherches sur le Sida et les hépatites virales » ANRS. V R-P was supported by a fellowship from the « Institut Pasteur de Lille/Région Nord Pas-de-Calais ». DD was supported by a fellowship from the ANRS. FH was supported by a fellowship from the French Ministry of Research. JD is an international scholar of the Howard Hughes Medical Institute.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Hepatitis C virus (HCV) infection is a global public health problem affecting over 130 million individuals worldwide; its symptoms including chronic hepatitis, liver cirrhosis, and hepatocellular carcinoma <a href="#pone.0001866-Lemon1">[1]</a>. Unfortunately, no vaccine is currently available to prevent new infections and the current treatments are not fully efficient <a href="#pone.0001866-Manns1">[2]</a>. Clearly, new therapeutic strategies are urgently required.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Over the past decade, due to the lack of a cell culture system supporting production of infectious virus particles, several surrogate models have been developed to facilitate analysis of the HCV life cycle. Among these models, pseudoparticles (HCVpp), consisting of native HCV envelope glycoproteins assembled onto retroviral core particles <a href="#pone.0001866-Bartosch1">[3]</a>, <a href="#pone.0001866-Hsu1">[4]</a> have been useful in investigating the HCV entry process. More recently, however, production of infectious HCV particles in cell culture (HCVcc) has finally become possible <a href="#pone.0001866-Lindenbach1">[5]</a>, <a href="#pone.0001866-Wakita1">[6]</a>, <a href="#pone.0001866-Zhong1">[7]</a>. This powerful system is based on the transfection of the human hepatoma cell line Huh-7 with the cloned JFH1 genome that replicates and produces infectious particles.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>HCV encodes two envelope glycoproteins, E1 and E2, that interact to form a noncovalent E1E2 heterodimer <a href="#pone.0001866-OpDeBeeck1">[8]</a> which is present at the surface of HCV particles <a href="#pone.0001866-Wakita1">[6]</a>, and is therefore the obvious candidate ligand for cellular receptor(s). Although the early steps of viral entry have yet to be elucidated, several cell-surface expressed molecules have been proposed as entry factors for HCV (reviewed in <a href="#pone.0001866-Dubuisson1">[9]</a>). Among these molecules, the tetraspanin CD81 and the scavenger receptor class B type I (SR-BI) have been shown to play major roles in HCV entry. However, co-expression of these two molecules in non-hepatic cell lines does not lead to HCV entry <a href="#pone.0001866-Bartosch2">[10]</a>, suggesting that additional molecule(s) are involved in control of HCV entry. Recently, the tight-junction components Claudins (CLDN-1, CLDN-6, CLDN-9) have been identified as additional key factors for HCV infection <a href="#pone.0001866-Evans1">[11]</a>, <a href="#pone.0001866-Zheng1">[12]</a>. Interestingly, CLDN-1 is the first entry factor shown to confer susceptibility to HCV when ectopically expressed in non-hepatic cells. Nevertheless, although CLDN-1 subcellular distribution seems to modulate HCV permissivity <a href="#pone.0001866-Yang1">[13]</a>, some human cell lines expressing CD81, SR-BI and CLDN-1 remain resistant to HCV entry suggesting that one or more human-specific HCV entry factor(s) remain to be discovered <a href="#pone.0001866-Evans1">[11]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>CD81 belongs to the tetraspanin family. Members of this family organize and regroup their associated transmembrane proteins and are involved in various functions such as cell morphology, motility, fusion and signalling <a href="#pone.0001866-Boucheix1">[14]</a>, <a href="#pone.0001866-Levy1">[15]</a>. A major characteristic of tetraspanins is their ability to interact with each other and with other transmembrane proteins, thus building membrane multi-molecular complexes, collectively referred to as the tetraspanin web <a href="#pone.0001866-Levy2">[16]</a>, <a href="#pone.0001866-Rubinstein1">[17]</a>. Within this network of interactions, tetraspanins form primary complexes with a limited number of proteins termed tetraspanin partners. These primary interactions are direct, highly specific and occur at high stoichiometry. Two major partners have been identified for CD81, EWI-F (also called CD9P-1, FPRP or CD315) and EWI-2 (also called PGRL, IgSF8 or CD316) <a href="#pone.0001866-Charrin1">[18]</a>, <a href="#pone.0001866-Charrin2">[19]</a>, <a href="#pone.0001866-Clark1">[20]</a>, <a href="#pone.0001866-Stipp1">[21]</a>, <a href="#pone.0001866-Stipp2">[22]</a>, which may provide a link between the tetraspanin web and the actin cytoskeleton by interacting with Ezrin, an Ezrin-Radixin-Moesin (ERM) protein <a href="#pone.0001866-SalaValdes1">[23]</a>. Although its function is still unclear, EWI-2 seems to participate in the regulation of cellular functions such as aggregation, spreading, motility and migration <a href="#pone.0001866-Kolesnikova1">[24]</a>, <a href="#pone.0001866-Stipp3">[25]</a>, <a href="#pone.0001866-Zhang1">[26]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In this work, we identified a cleavage product of EWI-2, which associates with CD81 and inhibits its interaction with the HCV envelope glycoproteins. Most importantly, this molecule, that we called EWI-2wint (EWI-2 <span class="underline">w</span>ithout <span class="underline">i</span>ts <span class="underline">N</span>-<span class="underline">t</span>erminus), has an inhibitory effect on HCV entry, highlighting a potential new mechanism for the regulation of cellular invasion by this pathogen.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>A CD81 partner blocks the interaction between CD81 and HCV envelope glycoproteins</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Tetraspanin microdomains are typically disrupted by Triton X-100 (TX), but are retained in less hydrophobic detergents such as Brij97 (Bj). In addition, the replacement of divalent cations (CaMg) by EDTA in the Brij97 lysis buffer causes a disruption of tetraspanin/tetraspanin interactions but conserves the tetraspanin/partner interactions <a href="#pone.0001866-Charrin1">[18]</a> (<a href="#pone-0001866-g001">Figure 1A</a>). These biochemical properties allowed us to investigate whether the interaction between CD81 and E1E2 heterodimers is similar when CD81 is embedded or not in a tetraspanin web or in a primary complex. For this purpose, we analysed the interaction between E1E2 heterodimers and CD81 in several cell lines lysed in different detergent conditions (<a href="#pone-0001866-g001">Figure 1A</a> and <a href="#pone-0001866-t001">Table 1</a>). The E1E2 complexes were immobilized on agarose beads bound to a conformation-sensitive monoclonal antibody (mAb H53). Analysis of these agarose-bound complexes under non-reducing conditions <a href="#pone.0001866-Cocquerel1">[27]</a> showed proper folding of the heterodimers in the different detergent conditions (data not shown). As positive and negative controls, we used anti-CD81 and irrelevant mouse (Cont) mAbs, respectively. A recombinant soluble form of the CD81 large extracellular loop (CD81-LEL) <a href="#pone.0001866-Higginbottom1">[28]</a> was also used as a control. CD81 from Molt-4 and 293T cells interacted with E1E2 heterodimers under all detergent conditions, as did the CD81-LEL (<a href="#pone-0001866-g001">Figure 1A</a>). In contrast, CD81 from Daudi and Ramos cells did not interact with E1E2 heterodimers when tetraspanin webs or primary complexes were conserved (<a href="#pone-0001866-g001">Figure 1A</a>, Bj/CaMg and Bj/EDTA, respectively). Daudi cells also did not react with E1E2 glycoproteins derived from HCVpp (data not shown). Together, our results suggest that a CD81 partner, present on Daudi and Ramos cells, blocks the interaction between CD81 and HCV glycoproteins. Testing of other cell lines indicated that such a CD81 partner is probably also present in other cell lines such as A431, a squamous carcinoma cell line (<a href="#pone-0001866-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001866-g001"><div class="img"><a name="pone-0001866-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The interaction between HCV glycoproteins and CD81 is blocked when CD81 interacts with partner protein(s).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A, HCV E1E2 heterodimers immobilized onto anti-E2 coated beads interacted with CD81 in all cells lysed in Triton X-100 (TX). However, the interaction of HCV-E1E2 with CD81 was blocked in Daudi and Ramos cells lysed in Brij (Bj). Maintenance of CD81 with other tetraspanins (T) and partners (P) is disrupted by the indicated lysis conditions, as diagrammed. Anti-CD81 and irrelevant (Cont) mAbs were used in immunoprecipitations as controls. Precipitation of CD81 was revealed by western blotting with the anti-CD81 5A6 mAb. CD81-LEL corresponds to the large extracellular loop of CD81 fused to the glutathione-S transferase. B, After cell surface biotinylation, the indicated cell lines were lysed with Bj/EDTA, immunoprecipitated with 5A6 mAb and the proteins revealed by Western blotting with HRP-conjugated streptavidin. The values on the right are molecular sizes in kilodaltons. C, EWI-2wint production is directly connected to EWI-2 expression. Daudi cells interfered with negative siRNA or EWI-2 siRNA were biotinylated, lysed in Bj/EDTA, immunoprecipitated with the anti-CD81 5A6 mAb or an anti-EWI-2 mAb (8A12) and blotted sequentially with HRP-conjugated streptavidin and 5A6 mAb.</p>
<span>doi:10.1371/journal.pone.0001866.g001</span></div><div class="figure" id="pone-0001866-t001"><div class="img"><a name="pone-0001866-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001866.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001866.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span><em>In vitro</em> interaction of HCV E1E2 heterodimers with CD81 and expression of cell markers.</span></strong></p><span>doi:10.1371/journal.pone.0001866.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To identify the CD81 partner blocking the interaction between CD81 and E1E2, we performed co-immunoprecipitation experiments using Bj/EDTA lysates from surface biotinylated cells. Under these conditions, only partners directly associated in primary complexes co-precipitate with CD81. In all cell lines, a 70-kDa surface protein, corresponding to EWI-2 <a href="#pone.0001866-Charrin1">[18]</a>, <a href="#pone.0001866-Stipp1">[21]</a> was co-precipitated with CD81 (<a href="#pone-0001866-g001">Figure 1B</a>). EWI-F, the second major CD81/CD9 partner <a href="#pone.0001866-Charrin2">[19]</a>, <a href="#pone.0001866-Stipp2">[22]</a> was also co-precipitated with CD81 and CD9 in 293T cells. To confirm the specificity of EWI-2 and EWI-F, these proteins were immunoprecipitated with specific mAbs 8A12 and 1F11, respectively (data not shown). Additional bands were also observed in these cells, but the identity of the corresponding proteins is unknown. Interestingly, an additional ~55 kDa surface protein co-precipitated with CD81 in Daudi and Ramos cells but not in Molt-4 and 293T cells. This protein, which we have named EWI-2wint, has been previously proposed as a cleavage product of EWI-2 that remains associated with CD81 and CD9 <a href="#pone.0001866-Charrin1">[18]</a>, <a href="#pone.0001866-Stipp1">[21]</a>. Furthermore, transfection of EWI-2 siRNA into Daudi cells reduced both the expression of EWI-2 and of EWI-2wint (<a href="#pone-0001866-g001">Figure 1C</a>) demonstrating that EWI-2wint production is directly connected to EWI-2 expression. We performed additional co-immunoprecipitation experiments using different surface biotinylated cell lysates (<a href="#pone-0001866-t001">Table 1</a>). Co-immunoprecipitation of EWI-2wint with CD81 correlated with the inhibition of CD81-E1E2 interaction in mild detergents. Conversely, cells for which CD81 bound to E1E2 did not express this protein. It is worth noting that primary hepatocytes and hepatoma cells do not have detectable levels of EWI-2wint. We also analysed SR-BI and CLDN-1 expression in the same cell lines, and the lack of interaction between CD81 and E1E2 glycoproteins cannot be correlated with the absence of these molecules (<a href="#pone-0001866-t001">Table 1</a>, see squamous carcinoma, A431). Altogether, our results suggest that EWI-2wint probably corresponds to the CD81 partner blocking the interaction between CD81 and HCV glycoproteins.</p>


<h4>EWI-2wint corresponds to EWI-2 with its amino-terminus deleted</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To confirm that EWI-2wint is a cleavage product of EWI-2, we constructed an EWI-2 cDNA with a C-terminal FLAG epitope tag (EWI-2FLAG), as described previously <a href="#pone.0001866-Stipp1">[21]</a>. CHO cells stably co-expressing human CD81 and EWI-2FLAG (CHO/CD81+EWI-2FLAG) were surface biotinylated, lysed in Bj/EDTA (<a href="#pone-0001866-g002">Figure 2A</a>) or TX/EDTA (<a href="#pone-0001866-g002">Figure 2D</a>) and analysed in immunoprecipitation experiments. Upon EWI-2FLAG coexpression with CD81, EWI-2wint was detected following anti-FLAG immunoprecipitation or co-immunoprecipitation with CD81 (wint-FLAG ; <a href="#pone-0001866-g002">Figure 2A</a>). These results and Western blotting analyses (data not shown) showed that EWI-2wint still contains the C-terminal FLAG epitope, making EWI-2wint an authentic cleavage product of EWI-2. It is interesting to note that EWI-2wint was not recognized by our anti-EWI-2 mAb and that only a small fraction of CD81 co-immunoprecipitated with EWI-2 products, as previously observed <a href="#pone.0001866-Charrin1">[18]</a>. Two smaller proteins of ~50, and 45 kDa, which remained associated with CD81 and probably correspond to additional cleavage products of EWI-2 were also detected (asterisks in <a href="#pone-0001866-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0001866-g002"><div class="img"><a name="pone-0001866-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>EWI-2wint is a cleavage product of EWI-2.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>CHO cells were (co)-transfected with pcDNA3.1/CD81 and pcDNA3.1/EWI-2FLAG (A and D), with pcDNA3.1/wint-FLAG (B and E), or with pcDNA3.1/CD81 and pcDNA3.1/EWI-2FLAG<sup>fur</sup> (C and F). EWI-2FLAG<sup>fur</sup> corresponds to EWI-2FLAG in which a furin cleavage site has been engineered. After cell surface biotinylation, cells were lysed with Bj/EDTA (A–C) or TX/EDTA (D–F) and proteins were immunoprecipitated with the indicated mAbs. Proteins were revealed by Western blotting with HRP-conjugated streptavidin. Asterisks indicate additional cleavage products of EWI-2.</p>
<span>doi:10.1371/journal.pone.0001866.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>EWI-2 belongs to a novel family of immunoglobulin (Ig) proteins, which also includes EWI-F/CD9P-1/PGRL, EWI-101/CD101/V7 and EWI-3/IgSF3 <a href="#pone.0001866-Stipp1">[21]</a>. The members of this family share a conserved EWI motif and contain an ectodomain composed of V-type Ig domains, a transmembrane domain and a short highly charged cytoplasmic tail <a href="#pone.0001866-Stipp1">[21]</a>. EWI-2 contains 4 Ig domains (<a href="#pone-0001866-g003">Figure 3</a>).</p>
<div class="figure" id="pone-0001866-g003"><div class="img"><a name="pone-0001866-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Alignment of EWI-2 sequence and the deduced EWI-2wint sequence.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Asterisk indicates the position at which an Arg (R) residue was introduced to make a furin cleavage site. Underlined amino acids correspond to the signal peptide of EWI-2. The four Ig domains are colored. Amino acid residues corresponding to the putative transmembrane domain are in italics and those corresponding to the cytoplasmic tail are bolded.</p>
<span>doi:10.1371/journal.pone.0001866.g003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>To identify the sequence of EWI-2wint, EWI-2FLAG and wint-FLAG proteins were purified by immunoaffinity from a Triton X-100 lysate of CHO/CD81+EWI-2FLAG and fractionated by SDS-PAGE. The bands corresponding to EWI-2FLAG and wint-FLAG were excised after colloidal blue staining and digested with trypsin or V8 endoproteinase Glu C and the resulting peptides analysed by mass spectrometry (<a href="#pone-0001866-t002">Table 2</a>). Interestingly, only peptides matching with the N-terminal membrane distal Ig domain (Ig 1 domain) of EWI-2 were not present in the wint-FLAG band, suggesting that this domain may be absent in the EWI-2wint protein. Indeed, sequencing of purified proteins by Edman degradation revealed that the N-terminal sequence of EWI-2wint corresponds to the QAPTS amino acids (<a href="#pone-0001866-t002">Table 2</a>) residues 166 to 170, which are located within the spacer separating the Ig1 and Ig2 domains of EWI-2 (<a href="#pone-0001866-g003">Figure 3</a>). Altogether, these results indicate that EWI-2wint corresponds to EWI-2 without its Ig 1 domain. Following these results, we chose to name this protein EWI-2wint for EWI-2 <span class="underline">w</span>ithout <span class="underline">i</span>ts <span class="underline">N</span>-<span class="underline">t</span>erminus.</p>
<div class="figure" id="pone-0001866-t002"><div class="img"><a name="pone-0001866-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001866.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001866.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Mass spectrometric analyses and N-terminal sequencing of EWI-2 and EWI-2wint proteins</span></strong></p><span>doi:10.1371/journal.pone.0001866.t002</span></div>

<h4>Expression of EWI-2wint in HCV target cells</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To characterize EWI-2wint, we cloned it as a FLAG-tagged protein in an expression vector (pcDNA3.1/wint-FLAG). CHO cells transfected with pcDNA3.1/wint-FLAG produced EWI-2wint and the two additional cleavage products (asterisks), as shown in <a href="#pone-0001866-g002">Figure 2B and 2E</a>. To further test the role of EWI-2wint in HCV infection, we tried to obtain hepatoma cells (Huh-7) stably expressing EWI-2wint. However, despite several attempts, EWI-2wint expression in cellular clones could not be detected by cell surface biotinylation and immunoprecipitation or by western blotting experiments (data not shown), suggesting that this protein was poorly expressed in these cells. To circumvent this problem, we added a single Arg residue into the EWI-2FLAG sequence directly upstream to Q<sup>166</sup> amino acid (asterisk, <a href="#pone-0001866-g003">Figure 3</a>), making an EWI-2FLAG protein with a RGRR cleavage motif for furin (pcDNA3.1/EWI-2FLAG<sup>fur</sup>), a trans-Golgi network associated endopeptidase. This strategy, when tested in CHO cells transfected with pcDNA3.1/EWI-2FLAG<sup>fur</sup>, yielded a high amount of the EWI-2wint protein, as shown in <a href="#pone-0001866-g002">Figure 2C and 2F</a>. Furthermore, EWI-2wint could be detected by direct immunoprecipitation or by co-immunoprecipitation with CD81 in Huh-7 cells stably expressing EWI-2FLAG<sup>fur</sup> (wint-FLAG, <a href="#pone-0001866-g004">Figure 4D</a>). Almost no whole EWI-2FLAG protein reached the surface of Huh-7/EWI-2FLAG<sup>fur</sup> cells, indicating that the cleavage of EWI-2FLAG<sup>fur</sup> by furin was highly efficient in these cells. It should be noted that at least one additional cleavage product of EWI-2FLAG (asterisk, <a href="#pone-0001866-g004">Figure 4D</a>) was detected as in CHO cells. The association of one or two additional cleavage products with the production of EWI-2wint suggests that the first cleavage of EWI-2 allows the unmasking of one or two additional sites that are not accessible in the whole protein. In contrast, Huh-7 cells (<a href="#pone-0001866-g004">Figure 4A</a>), Huh-7 stably transfected with the empty vector (pcDNA3.1, <a href="#pone-0001866-g004">Figure 4B</a>) or with EWI-2FLAG, expressing a N-terminal HA epitope (HAEWI-2FLAG, <a href="#pone-0001866-g004">Figure 4C</a>) were devoid of EWI-2wint. In Huh-7 cells stably transfected with HAEWI-2FLAG, the tagged molecule was detected following co-immunoprecipitation with CD81 or by direct immunoprecipitation with anti-FLAG or anti-HA mAbs, as compared to control cells (<a href="#pone-0001866-g004">Figure 4</a>). The anti-CD81 and anti-FLAG antibodies precipitated different amounts of wint-FLAG in Huh-7/EWI-2FLAG<sup>fur</sup> cells (<a href="#pone-0001866-g004">Figure 4D</a>). This might be due to differences in the affinity of the antibodies, as has been previously observed when EWI-2 and CD81 were coprecipitated <a href="#pone.0001866-Charrin1">[18]</a>. Alternative possibilities are that i) homodimerization and heterodimerization of EWI-2 and CD81 modulate recognition by antibodies, as described <a href="#pone.0001866-Kazarov1">[29]</a>, <a href="#pone.0001866-Yang2">[30]</a>, ii) binding of the anti-FLAG mAb at the C-terminus of EWI-2wint induces a dissociation of the CD81/EWI-2wint complexes.</p>
<div class="figure" id="pone-0001866-g004"><div class="img"><a name="pone-0001866-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>EWI-2wint expression in Huh-7 cells.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>After cell surface biotinylation, Huh-7 cells (A) stably expressing pcDNA3.1 (B), HAEWI-2FLAG (C) or EWI-2FLAG<sup>fur</sup> (D) were lysed with Bj/EDTA and analyzed by immunoprecipitation with indicated mAbs. Proteins were revealed by Western blotting with HRP-conjugated streptavidin. The molecular weights of the prestained molecular ladder are indicated in KDa. The asterisk indicates an additional cleavage product of EWI-2. The band indicated by a light arrow likely corresponds to a dimer of CD81, and the band indicated by a full arrow to an unidentified EWI-2 associated protein.</p>
<span>doi:10.1371/journal.pone.0001866.g004</span></div>

<h4>EWI-2wint inhibits HCV infection</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We then sought to determine whether EWI-2wint is able to modulate HCV infection of Huh-7 target cells. Originally we used cell culture produced HCV particles (HCVcc) <a href="#pone.0001866-Lindenbach1">[5]</a>, <a href="#pone.0001866-Wakita1">[6]</a>, <a href="#pone.0001866-Zhong1">[7]</a> to infect Huh-7 cells expressing EWI-2FLAG<sup>fur</sup>, HAEWI-2FLAG, pcDNA3.1 or naïve Huh-7 cells. In the absence of a reporter gene in HCVcc, infection levels were evaluated by immunofluorescence (<a href="#pone-0001866-g005">Figure 5A</a>) and western blotting (<a href="#pone-0001866-g005">Figure 5B</a>), 40 h post-infection. Interestingly, we observed a reduction in HCVcc infection level in cells expressing EWI-2FLAG<sup>fur</sup>, whereas it remained unmodified in control cells (<a href="#pone-0001866-g005">Figure 5C</a>). It is worth noting that in HAEWI-2FLAG cells, the HCVcc infection level was similar to that of pcDNA3.1 cells, indicating that ectopic expression of HAEWI-2FLAG does not affect HCV infection. In addition, flow cytometry analyses ruled out any potential bias due to different levels of CD81 cell surface expression on clones (<a href="#pone-0001866-g005">Figure 5D</a>). Altogether, these results indicate that EWI-2wint produced in Huh-7/EWI-2FLAG<sup>fur</sup> cells inhibits HCV infection.</p>
<div class="figure" id="pone-0001866-g005"><div class="img"><a name="pone-0001866-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>EWI-2wint inhibits HCVcc infection.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Cell clones were infected with JFH1 HCVcc and infectivity was analyzed by indirect immunofluorescence with an anti-C mAb (A) or Western blotting with an anti-E2 mAb (B). The fields displayed in (A) contained similar numbers of cells. C, Infection levels were measured by quantification with NIH Image 1.62 of the intensities of HCV E2 bands. The results are presented as percentages of HCVcc infection relative to the infection of Huh-7/pcDNA3.1 cells. Infection levels of three independent experiments are reported as the mean with standard deviation bars. D, CD81 expression on the surfaces of cells expressing or not EWI-2wint. Cells were stained by using an anti-CD81 mAb (5A6) and secondary antibody conjugated with PE. Cont corresponds to Huh-7 cells that were stained only with the secondary antibody.</p>
<span>doi:10.1371/journal.pone.0001866.g005</span></div>

<h4>EWI-2wint blocks HCV entry into target cells</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The life cycle of HCV can be divided into three major steps : entry of the virus into its target cells by receptor-mediated endocytosis, cytoplasmic and membrane-associated replication of the RNA genome, and assembly and release of the progeny virions.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>To analyze the effect of EWI-2wint expression on HCV genome replication, we compared the efficiency of RNA replication and virus production between transfected cell lines expressing or lacking EWI-2wint. We transfected equal amounts of RNA corresponding to the full-length JFH1 genome <a href="#pone.0001866-Wakita1">[6]</a> into the different cell lines (<a href="#pone-0001866-g006">Figures 6A and 6B</a>). Immunofluorescence analyses of transfected cells showed similar positivity for HCV anti-core antibody at 40 h post-transfection. Flow cytometry analyses using an anti-NS3 mAb showed that, although slightly stronger in Huh-7 cells, viral expression was similar in cellular clones that lacked or expressed EWI-2wint (<a href="#pone-0001866-g006">Figure 6B</a>), indicating that EWI-2wint does not affect HCV genome replication. Next, to assess the effect of EWI-2wint on the assembly and/or egress of particles, the culture medium of each transfected cell line was harvested and used to infect naïve Huh-7 cells. Through core-specific immunofluorescence staining, we found that Huh-7 cells were similarly infected by virus produced from all transfected cellular clones (data not shown), suggesting that EWI-2wint does not affect HCV assembly and release.</p>
<div class="figure" id="pone-0001866-g006"><div class="img"><a name="pone-0001866-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>EWI-2wint inhibits the entry stage of HCV life cycle by reducing the interaction between E1E2 and CD81.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A, EWI-2wint does not interfere with HCV replication. The indicated Huh-7 cell lines were transfected with the full-length JFH1 genome, and expression of the core antigen was evaluated at 40 h post-transfection. The displayed fields contained similar numbers of cells. B, NS3 expression in JFH-1 transfected cells. Cells were stained by using an anti-NS3 mAb (486-D39) and secondary antibody conjugated with PE. Cont corresponds to untransfected Huh-7 cells. C, Huh-7 cell lines were incubated for 3 h with virus pseudotyped with HCV envelope glycoprotein (HCVpp) or VSV G envelope protein (VSVpp). HCVpp were generated with envelope proteins from 1a (HCVpp 1a) or 2a (HCVpp 2a) genotype. The inoculum was then removed and the cells were further incubated. At 2 days post-inoculation, cells were lysed and processed to measure the luciferase activity. The luciferase activities were normalized for protein concentration in each cell lysate. The results are presented as relative percentages to HCVpp infectivity on Huh-7 cells. Results are reported as the mean±S.D. of three independent experiments. Pseudotyped particles produced in the absence of envelope proteins were used as controls. The mean fluorescence activity of such particles represented less than 2% of the activity measured for HCVpp. D, <em>In vitro</em> interaction of E1E2 heterodimers with CD81 from Huh-7/EWI-2FLAG<sup>fur</sup> and Huh-7/pcDNA3.1. This assay was performed as described in <a href="#pone-0001866-g001">Figure 1</a>. The ratio [CD81 band intensity in E1E2 IP in Bj/EDTA buffer]/[CD81 band intensity in E1E2 IP in TX/EDTA buffer] was 0,27 and 0,84 in Huh-7/EWI-2FLAG<sup>fur</sup> and Huh-7/pcDNA3.1, respectively.</p>
<span>doi:10.1371/journal.pone.0001866.g006</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Our observations indicate that EWI-2wint likely affects an early step in the HCV life cycle. To test this hypothesis, we analyzed the effect of EWI-2wint on the infectivity of retroviral particles pseudotyped with HCV E1E2 proteins (<a href="#pone-0001866-g006">Figure 6C</a>). Use of HCVpp enables the analysis of the HCV entry step. We generated particles pseudotyped with HCV envelope glycoproteins from 1a and 2a genotypes (HCVpp 1a and HCVpp 2a, respectively) and control particles pseudotyped with the envelope glycoprotein G of vesicular stomatitis virus (VSVpp). As shown in <a href="#pone-0001866-g006">Figure 6C</a>, HCVpp infectivity was reduced in cells expressing EWI-2wint whereas it remained unmodified in control cells. Expression of EWI-2wint in target cells had no effect on VSVpp infectivity. It is interesting to note that HAEWI-2FLAG expression slightly increased HCVpp and VSVpp infectivity. Altogether, our results indicate that EWI-2wint inhibits the entry stage of HCV lifecycle, likely by interacting with CD81.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>To determine whether the lower level of HCV infection observed in EWI-2wint producing cells resulted from a reduced interaction between HCV glycoproteins and CD81, we compared the interaction between E1E2 and CD81 in Huh-7/pcDNA3.1 and Huh-7/EWI-2FLAG<sup>fur</sup> cells, as detailed in <a href="#pone-0001866-g001">Figure 1</a>. Interestingly, our results demonstrated reduced interaction between E1E2 and CD81 in Huh-7/EWI-2FLAG<sup>fur</sup> cells lysed in Bj/EDTA which conserves tetraspanins/partners interactions, as compared to cells lysed in TX/EDTA (<a href="#pone-0001866-g006">Figure 6D</a>). In contrast, the binding of E1E2 to CD81 was similar in Huh-7/pcDNA3.1 cells in both detergent conditions. These data indicate that the inhibition of HCV infection by EWI-2wint correlates with its capacity to inhibit the interaction between E2 glycoprotein and CD81.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>In order to carry out statistical analyses of EWI-2wint effect on HCV infection, we next generated JFH1-based <em>Renilla</em> luciferase (R-Luc) reporter HCVcc and infected Huh-7 cells that lacked or expressed EWI-2wint (<a href="#pone-0001866-g007">Figure 7A</a>). Cellular populations (pcDNA3.1, HAEWI-2FLAG and EWI-2FLAG<sup>fur</sup> POP) and individual cellular clones (EWI-2FLAG<sup>fur</sup> A1, A6, B14, C1) have been used in these experiments. In parallel, we infected these cells with HCVpp 1a, HCVpp 2a or VSVpp (<a href="#pone-0001866-g007">Figure 7B</a>), as described in <a href="#pone-0001866-g006">Figure 6</a>. Our results showed that EWI-2wint expression inhibits HCVcc infectivity up to 80% (<a href="#pone-0001866-g007">Figure 7A</a>; EWI-2FLAG<sup>fur</sup> A6) confirming our previous findings (<a href="#pone-0001866-g005">Figure 5</a>). It is noteworthy that EWI-2wint was less effective, reducing HCVpp infectivity only by 40–50% (<a href="#pone-0001866-g007">Figure 7B</a>; EWI-2FLAG<sup>fur</sup> A1), which might be due to slight differences between HCVpp and HCVcc in the entry stages, as recently reported by others <a href="#pone.0001866-Kapadia1">[31]</a>. Once again, this study shows that EWI-2wint expression in Huh-7 cells leads to a significant reduction of HCV infection.</p>
<div class="figure" id="pone-0001866-g007"><div class="img"><a name="pone-0001866-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001866" data-uri="info:doi/10.1371/journal.pone.0001866.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001866.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001866.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001866.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001866.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>EWI-2wint inhibits the infection of both HCVcc and HCVpp.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>A, Cellular populations (pcDNA3.1, HAEWI-2FLAG and EWI-2FLAG<sup>fur</sup> POP) and individual cellular clones (EWI-2FLAG<sup>fur</sup> A1, A6, B14, C1) were infected with HCVcc expressing <em>Renilla</em> luciferase (R-Luc). In parallel, these cells were infected with HCVpp 1a, HCVpp 2a or VSVpp expressing firefly luciferase (F-Luc) (B), as described in <a href="#pone-0001866-g006">Figure 6</a>. At 2 days post-infection, cells were lysed and processed to measure the luciferase activity. The luciferase activities were normalized for protein concentration in each cell lysate. The results are presented as relative percentages to HCVcc (A) and HCVpp (B) infectivity on Huh-7 cells. HCVpp infections (B) were also normalized to VSVpp infections on Huh-7 cells. Results are reported as the mean±S.D. of three independent experiments.</p>
<span>doi:10.1371/journal.pone.0001866.g007</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Here, we identified a novel CD81 associated protein, EWI-2wint, which is able to inhibit HCV entry into target cells by blocking the interactions between HCV glycoproteins and CD81. This finding indicates that the early steps of HCV entry into its target cells involve tight control of CD81 accessibility to the viral particle.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>EWI-2wint may inhibit HCV entry by reducing E1E2-CD81 interactions for a number of possible reasons. EWI-2wint may reduce CD81 accessibility to envelope glycoproteins by steric hindrance. Alternatively, the association of EWI-2wint with CD81 may induce conformational modifications in CD81, blocking the binding of HCV heterodimers. Although the exact role of CD81 in the course of virus infection is not well defined, it is a key molecule required for productive infection by HCVpp and HCVcc of Huh-7 cells <a href="#pone.0001866-Bartosch1">[3]</a>, <a href="#pone.0001866-Hsu1">[4]</a>, <a href="#pone.0001866-Lindenbach1">[5]</a>, <a href="#pone.0001866-Wakita1">[6]</a>, <a href="#pone.0001866-Zhong1">[7]</a>, <a href="#pone.0001866-Bartosch2">[10]</a>, <a href="#pone.0001866-Koutsoudakis1">[32]</a>. Moreover, most recently CD81 was shown to be required for the infection of primary hepatocytes by serum-derived HCV <a href="#pone.0001866-Molina1">[33]</a>. Previous studies have suggested that CD81 may not be required for the first attachment step, but rather function as a post-attachment entry co-factor <a href="#pone.0001866-Cormier1">[34]</a>, <a href="#pone.0001866-Flint1">[35]</a>, <a href="#pone.0001866-Koutsoudakis2">[36]</a>. Very recently, it has been shown that CLDN-1, 6 and 9 membrane proteins localized at tight junctions, are additional entry factors for HCV <a href="#pone.0001866-Evans1">[11]</a>, <a href="#pone.0001866-Zheng1">[12]</a>, <a href="#pone.0001866-Meertens1">[37]</a>, indicating that HCV entry might involve a complex multistep process. CD81 might potentially be required for a post-binding step such as escorting the particle into the endocytic pathway or priming it for the pH triggered fusion mechanism. The association of EWI-2wint with CD81 could block such entry stages. Since our preliminary results suggest that CD81 and EWI-2wint likely colocalize in an endosomal compartment (unpublished data), EWI-2wint might exert its inhibitory effect on CD81 functionality in the endosomes. In addition, EWI-2wint may interfere with actin polymerization potentially required for HCV entry. Indeed, HCV replication requires microtubule and actin polymerization <a href="#pone.0001866-Bost1">[38]</a> and CD81 engagement leads to actin rearrangement <a href="#pone.0001866-Coffey1">[39]</a>, <a href="#pone.0001866-Crotta1">[40]</a>. Furthermore, it has also recently been shown that CD81 and EWI-2 interact with Ezrin, an actin-linking ERM protein <a href="#pone.0001866-SalaValdes1">[23]</a>. Lastly, we cannot exclude the possibility that EWI-2wint blocks other signaling pathways necessary for viral entry.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>EWI-2wint is a cleavage product of the EWI-2 protein. The proteolytic cleavage of EWI-2 Ig1 domain occurs downstream of the RGR amino acids. A site directed mutagenesis analysis showed that RXR (where X is any amino acid) correspond to the consensus site recognized by the protease involved in this cleavage (unpublished data). Further analyses using protease inhibitors and small interference RNAs should enable us to identify this protease. Pulse-chase experiments in CHO cells showed that the cleavage occurs after N-glycan maturation (unpublished data), suggesting that it likely involves a Golgi resident protease. We showed that a RGRR furin cleavage motif made by insertion of an Arg residue into the EWI-2 sequence allows efficient production of EWI-2wint and additional cleavage product(s) in Huh-7 cells. As shown here, EWI-2 can be cleaved at RGR site to produce EWI-2wint, and it is likely that a furin-like protease may be responsible for this cleavage. Such a protease may be expressed differently among cell types, such that liver cells lacking the enzyme are consequently more sensitive to HCV infection. On the other hand, the accessibility of EWI-2 to protease(s) may be cell-type specific and might be modulated by other components of the tetraspanin webs that vary in different cell types <a href="#pone.0001866-Boucheix1">[14]</a>, <a href="#pone.0001866-Levy2">[16]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Although several molecules have already been implicated in the entry process of HCV, it is likely that additional specific factors regulate its entry into hepatic cells <a href="#pone.0001866-Cocquerel2">[41]</a>. Silencing of endogenous EWI-2 in Huh-7 cells did not significantly affect HCVcc infection (data not shown) indicating that while being a major partner of CD81, EWI-2 is likely not involved in the HCV entry process. Here, we showed that EWI-2wint, present in some cell lines and absent from hepatic cells, is able to inhibit HCV entry. The production of a mAb directed against EWI-2wint will be essential to further analyse the expression profile of EWI-2wint in various tissues and cell lines. Nevertheless, our results suggest that, in addition to the presence of specific entry factors in the hepatocytes such as CD81, SR-BI, CLDN-1 and additional unidentified molecule(s), the absence of a specific inhibitor can also contribute to the hepatotropism of HCV. Interestingly, 293T cells, that are permissive for HCV entry when they express CLDN-1 <a href="#pone.0001866-Evans1">[11]</a>, do not express EWI-2wint (<a href="#pone-0001866-t001">Table 1</a>). Further experiments with HCV non permissive cells expressing all known positive entry factors will be necessary to determine if the silencing of endogenous EWI-2wint is sufficient to induce HCV permissivity.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In conclusion, our observation constitutes a basis for the rational design of new antiviral molecules. Moreover, it represents a new mechanism that contributes to the regulation of the entry of a pathogen into a host cell.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cells and antibodies</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>U937-CD81 cells have been described previously <a href="#pone.0001866-Cocquerel3">[42]</a>. Primary hepatocytes were isolated from three human foetal livers (Advanced Bioscience Resources, Inc, Alameda, CA). HepG2-CD81 <a href="#pone.0001866-Bartosch2">[10]</a> have been described previously and were kindly provided by F-L. Cosset.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The mAbs used in this study included: 5A6 (anti-CD81 <a href="#pone.0001866-Oren1">[43]</a>), 8A12 (anti-EWI-2 <a href="#pone.0001866-Charrin1">[18]</a>), 1F11 (anti-EWI-F <a href="#pone.0001866-Charrin2">[19]</a>), H53 (anti-HCV E2 <a href="#pone.0001866-Cocquerel4">[44]</a>), 3/11 (anti-HCV E2, kindly provided by J. McKeating <a href="#pone.0001866-Flint2">[45]</a>), ACAP27 and 486-D39 (anti-HCV core and HCV NS3, respectively, kindly provided by JF Delagneau <a href="#pone.0001866-Maillard1">[46]</a>). The anti-beta actin (clone AC-15), the M2 anti-FLAG epitope mAb and the M2 anti-FLAG affinity gel were from Sigma. The HA11 (anti-HA, Clone 16B12) mAb was from Covance. The anti-claudin-1 (JAY.8) was from Zymed Laboratories. The anti-SR-BI (Cla-1) was from BD Transduction Laboratories.</p>


<h4>Plasmids</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>pcDNA3.1/CD81 plasmid has been described previously <a href="#pone.0001866-Charrin1">[18]</a>. EWI-2 was FLAG-tagged (DYKDDDDK) at its C-terminus by polymerase chain reaction (PCR) amplification from template pCMVSport6/EWI-2 <a href="#pone.0001866-Charrin1">[18]</a> and subcloned into a pcDNA3.1 vector (pcDNA3.1/EWI-2FLAG). Three Gly residues were introduced between EWI-2 and FLAG epitope sequences to increase epitope accessibility. Plasmid pcDNA3.1/wint-FLAG encoding EWI-2 signal peptide (amino acids 1–25) and EWI-2 amino acids 166–613 fused to FLAG tag was made by fusion-PCR from template pcDNA3.1/EWI-2FLAG. pcDNA3.1/HAEWI-2FLAG encodes EWI-2 signal peptide followed by HA9 epitope (YPYDVPDYA) tag fused to the entire sequence of EWI-2FLAG. Three Gly residues were also introduced between HA epitope and EWI-2FLAG sequence. In pcDNA3.1/EWI-2FLAG<sup>fur</sup> plasmid, an Arg residue was introduced between EWI-2 amino acids 165 and 166. Cloning details and oligonucleotide sequences are available upon request.</p>


<h4>Cell transfection</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>CHO cells were transfected using ExGen500 (Eurogentec). CHO/CD81+EWI-2FLAG cell line was obtained by selection for 2 weeks with 500 µg/ml hygromycin and 800 µg/ml neomycin. The resulting cells were maintained in selection media as a polyclonal population. For stably transfected Huh-7 cells, cells were electroporated using the Gene Pulser apparatus (Bio-Rad) and neomycin was added 48 hours post-transfection at 600 µg/ml. After 2 weeks, isolated clones were split using selection cylinders and maintained in neomycin medium.</p>


<h4>CD81-E1E2 interaction assay</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>COS-7 cells were infected with a recombinant adenovirus (Ad) expressing the HCV proteins E1, E2, p7 and NS2 (Ad/E1E2p7NS2, amino acids 171–1026) <a href="#pone.0001866-Cocquerel1">[27]</a> at a multiplicity of 25PFU per cell. At 48 h post-infection, cells were washed twice with ice-cold Dulbecco's phosphate-buffered saline (D-PBS), and lysed in one of the following lysis buffers : 1% Brij97 in D-PBS with calcium and magnesium (Bj/CaMg) or with 2 mM EDTA (Bj/EDTA), 1% Triton X-100 in D-PBS with calcium and magnesium (TX/CaMg), or with 2 mM EDTA (TX/EDTA). Clarified lysates were then incubated with mAb H53 pre-adsorbed onto rabbit anti-mouse protein A-sepharose overnight at 4°C (beads-E1E2). In parallel, rabbit anti-mouse-protein A beads were incubated with mAb 5A6 (beads-CD81) or an irrelevant control mouse mAb (beads-Cont). Cell lines were lysed in Bj/CaMg, Bj/EDTA, TX/CaMg or TX/EDTA lysis buffer at 10×10<sup>6</sup> cells/ml, clarified and incubated with beads-E1E2, beads-CD81 and beads-Cont for 6 h at 4°C. After rinsing five times with the lysis buffer, complexes were eluted in non-reducing sample buffer, resolved by SDS-PAGE, and immunoblotted with mAb 5A6.</p>


<h4>Detection of cell surface biotinylated proteins</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Cells were biotinylated as previously described <a href="#pone.0001866-Charrin2">[19]</a> with 0.2 mg/ml EZ-link-Sulpho-NHS-LC-biotin (Pierce), and lysed into Bj/EDTA or TX/EDTA containing protease inhibitors (Complete, Roche). Lysates were precleared for 2 h at 4°C with protein A-sepharose (Amersham Biosciences) then incubated with mAbs pre-adsorbed onto rabbit anti-mouse-protein A beads for 2 h at 4°C. After rinsing, complexes were eluted, resolved by SDS-PAGE, and immunoblotted with peroxidase-conjugated streptavidin (Vector).</p>


<h4>EWI-2 silencing</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>To silence the expression of endogenous EWI-2 protein, Daudi cells were electroporated with negative siRNA directed against the FLAG epitope (<span class="gene">5′ AUUACAAGGACGACGAUGA 3′</span>, Dharmacon) and EWI-2 siRNA directed against the sequence <span class="gene">5′ GUUCUCCUAUGCUGUCUU 3′</span> (Dharmacon) corresponding to nucleotide sequence 253–270 of the ORF of EWI-2 <a href="#pone.0001866-SalaValdes1">[23]</a>. Daudi cells were electroporated at 4°C using the Gene Pulser apparatus (400V, 950uF). Seventy two hours post-transfection, cells were biotinylated, lysed in Bj/EDTA and immunoprecipitated with 5A6 and 8A12 mAbs. Immunoprecipitated proteins were revealed by Western blotting with peroxidase-conjugated streptavidin or 5A6 mAb.</p>


<h4>Flow cytometry analysis</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>After trypsinization, transfected cells were fixed with Formalin Solution (formaldehyde 4%, Sigma) and permeabilized with PBS 0.2% BSA 0.05% Saponin. After rinsing with PBS 2% BSA 2 mM EDTA, cells were incubated 1 h at 4°C with 486-D39 anti-NS3 mAb. After rinsing with washing solution, cells were incubated with PE labeled goat anti-mouse (BD Pharmingen) for 45 min at 4°C, then washed and fixed with Formalin Solution. For CD81 staining, cells were detached with PBS 2 mM EDTA and incubated 1 h at 4°C with 5A6 anti-CD81 mAb. After rinsing with washing solution, cell lines were incubated with PE labeled goat anti-mouse for 45 min at 4°C, then washed and fixed with Formalin Solution (PFA 10%, Sigma). Cells stained only with the secondary antibodies were used as negative control. Labeled cells were analyzed using a FACS Beckman EPICS-XL MCL.</p>


<h4>Protein purification, mass spectrometry and N-terminal sequencing</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>For purification of EWI-2FLAG and wint-FLAG, ~2×10<sup>8</sup> CHO/CD81+EWI-2FLAG cells were lysed in 20 mM Tris-HCl, 150 mM NaCl, pH 7.6 (TBS), 1 mM EDTA and 1% Triton X-100 in the presence of protease inhibitors. Clarified cell lysates were precleared for 2 h at 4°C with protein A-sepharose and then incubated with an anti-FLAG M2 affinity gel for 2 h at 4°C. After five washes with TBS containing 0.2% Triton X-100 (TBS-T), FLAG-tagged proteins were eluted with 100 µg/ml FLAG peptide (Sigma) in TBS-T. Purified proteins were then precipitated with 5 volumes of acetone, incubated overnight at −20°C, and centrifugated. After rinsing with 80% acetone, proteins were resuspended in sample buffer, and resolved by SDS-PAGE. Colloïdal blue stained bands corresponding to EWI-2FLAG and wint-FLAG were then excised from the gel, reduced, alkylated with iodoacetamide (10 mg/ml in NH4HCO3, 20 mM) and digested overnight with 50 ng trypsin (Promega) in 20 mM NH4HCO3, or 100 ng V8 endoproteinase Glu C (Roche) in 0.1 M phosphate buffer pH7.7. The resulting peptide mixture was eluted from the gel, desalted, and spotted on a Maldi plate with freshly dissolved α-cyano-4-hydroxycinnaminic acid (5 mg/ml in 50% CH3CN, 20 mM citric acid). The dried spot was then washed with 20 mM di-ammonium citrate pH 4.5. Mass spectrometry was performed with a MALDI-TOF Voyager-DE-STR (Applied Biosystems). Spots were analysed by setting the following parameters: positive and reflector modes, acceleration voltage of 20KV, grid voltage of 61%, 90 ns of delayed extraction, low mass gate 500 amu. The laser energy required to desorb/ionise the samples was kept at low value, compatible with a good signal/noise ratio. Spectra were calibrated externally by using the [M+H+] monoisotopics ions from trypsinized lyzozyme. The theoretical list of digested peptides was obtained using MS-Digest <a href="http://128.40.158.151/ucsfhtml3.4/msdigest.htm">http://128.40.158.151/ucsfhtml3.4/msdige​st.htm</a>. For N-terminal sequencing, purified proteins were transferred to PVDF and stained with amido black. Blotted EWI-2FLAG and wint-FLAG were sequenced in a Procise 492 sequanator (Applied Biosystems) using pulsed-liquid method.</p>


<h4>JFH-1 infection</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>The plasmid pJFH1, containing the full-length cDNA of JFH1 isolate (genotype 2a) and kindly provided by T. Wakita (National Institute of Infectious Diseases, Japan), was used to generate HCVcc as described <a href="#pone.0001866-Wakita1">[6]</a>, <a href="#pone.0001866-Delgrange1">[47]</a>. Infectious titers of viral stocks were estimated between 10<sup>5</sup> and 10<sup>6</sup> focus-forming units per ml, based on immunofluorescent detection of infected foci following infection of Huh-7 cells with serial dilutions of viral stocks. Cell lines were incubated with HCVcc (m.o.i. = 1) for 2 h at 37°C, washed and, incubated for additional 40 h at 37°C. Infections were scored by indirect immunofluorescence with anti-C mAb ACAP27 followed by Alexa<sup>488</sup>-conjugated goat anti-mouse (Jackson Immunoresearch). Fifteen micrograms of total protein were also analyzed by immunoblotting with 3/11 mAb followed by peroxidase-conjugated goat anti-rat immunoglobulins (Jackson Immunoresearch). The blots were then stripped and reprobed with an anti-beta actin followed by peroxidase-conjugated goat anti-mouse immunoglobulins (Sigma). Quantification of protein expression was performed using NIH Image 1.62 from the band mean densities.</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>To generate HCVcc expressing <em>Renilla</em> luciferase, we used the FL-J6/JFH-5′C19Rluc2AUbi genome <a href="#pone.0001866-Tscherne1">[48]</a> kindly provided by C.M. Rice. We replaced the region encoding the J6/JFH-1 HCV polyprotein with the CS-N6 JFH-1 sequence <a href="#pone.0001866-Delgrange1">[47]</a>. HCVcc were produced as described <a href="#pone.0001866-Wakita1">[6]</a>, <a href="#pone.0001866-Delgrange1">[47]</a>, <a href="#pone.0001866-Tscherne1">[48]</a>. HCVcc were added to Huh-7 cells (m.o.i. = 1) seeded the day before in 24-well plates and incubated for 2 h at 37°C. The supernatants were then removed and the cells were incubated in DMEM 10% FBS at 37°C. At 40–48 h post-infection, <em>Renilla</em> luciferase assays were performed as indicated by the manufacturer (Promega).</p>


<h4>HCVcc replication analyses</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p><em>In vitro</em> transcribed full-length JFH1 RNA was transfected by electroporation as described <a href="#pone.0001866-Kato1">[49]</a>. At 40 h post-transfection, cells were analyzed by indirect immunofluorescence and flow cytometry.</p>


<h4>Indirect immunofluorescence microscopy</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>Infected or transfected cells grown on coverslips were fixed with 3% paraformaldehyde and permeabilized with 0.05% Triton X-100 in PBS. Cells were then stained with anti-C mAb ACAP27 followed by Alexa<sup>488</sup>-conjugated goat anti-mouse (Jackson Immunoresearch). Coverslips were mounted on glass slides using Mowiol, and observed with a Zeiss Axioplan 2 Axiophot 2 equipped with a 20×/1.3 numerical aperture lens. Fluorescent signals were collected with a Princeton cooled charged device using specific fluorescence excitation and emission filters. Images were processed with Adobe Photoshop software.</p>


<h4>Production of HCVpp and infection assays</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>HCVpp were produced as described previously <a href="#pone.0001866-Bartosch1">[3]</a>, <a href="#pone.0001866-OpDeBeeck2">[50]</a> with plasmids kindly provided by B. Bartosch and F.L. Cosset (INSERM U412, Lyon, France). For the production of VSVpp, a plasmid encoding the vesicular stomatitis virus glycoprotein G <a href="#pone.0001866-Sandrin1">[51]</a> was used. Supernatants containing the pseudotyped particles were harvested 48 h after transfection, filtered through 0.45-µm pore-sized membranes and conserved at 4°C. HCVpp were added to Huh-7 cells seeded the day before in 24-well plates and incubated for 2–3 h at 37°C. The supernatants were then removed and the cells were incubated in DMEM 10% FBS at 37°C. At 48 h post-infection, luciferase assays were performed as indicated by the manufacturer (Promega).</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Sophana Ung for his excellent technical assistance. We thank Cécile Voisset, Karin Séron and Hann-Shuin Yew for critical reading of the manuscript. We are grateful to J. McKeating, T. Wakita, J-F. Delagneau, F-L. Cosset and C.M. Rice for providing us with reagents.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JD LC. Performed the experiments: FH CW LC VR CM DD AP HD FL. Analyzed the data: LC ER. Contributed reagents/materials/analysis tools: SL JD SC ER. Wrote the paper: SL LC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001866-Lemon1" id="pone.0001866-Lemon1"></a>Lemon SM, Walker C, Alter MJ, Yi M (2007)  Hepatitis C Virus. In: Knipe DM, Howley PM, editors. Fields Virology. Fifth ed. Philadelphia: Lippincott Williams &amp; Wilkins. pp. 1253–1304.  <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001866-Manns1" id="pone.0001866-Manns1"></a>Manns MP, Wedemeyer H, Cornberg M (2006) Treating viral hepatitis C: efficacy, side effects, and complications. Gut  55: 1350–1359.  <ul class="find" data-citedArticleID="1080692" data-doi="10.1136/gut.2005.076646"><li><a href="http://dx.doi.org/10.1136/gut.2005.076646" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Treating+viral+hepatitis+C%3A+efficacy%2C+side+effects%2C+and+complications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Treating+viral+hepatitis+C%3A+efficacy%2C+side+effects%2C+and+complications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001866-Bartosch1" id="pone.0001866-Bartosch1"></a>Bartosch B, Dubuisson J, Cosset F-L (2003) Highly infectious hepatitis C pseudo-viruses containing functional E1E2 envelope protein complexes. J Exp Med  197: 633–642.  <ul class="find" data-citedArticleID="1080628" data-doi="10.1084/jem.20021756"><li><a href="http://dx.doi.org/10.1084/jem.20021756" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+infectious+hepatitis+C+pseudo-viruses+containing+functional+E1E2+envelope+protein+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+infectious+hepatitis+C+pseudo-viruses+containing+functional+E1E2+envelope+protein+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001866-Hsu1" id="pone.0001866-Hsu1"></a>Hsu M, Zhang J, Flint M, Logvinoff C, Cheng-Mayer C, et al.  (2003) Hepatitis C virus glycoproteins mediate pH-dependent cell entry of pseudotyped retroviral particles. Proc Natl Acad Sci U S A  100: 7271–7276.  <ul class="find" data-citedArticleID="1080668" data-doi="10.1073/pnas.0832180100"><li><a href="http://dx.doi.org/10.1073/pnas.0832180100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hepatitis+C+virus+glycoproteins+mediate+pH-dependent+cell+entry+of+pseudotyped+retroviral+particles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hepatitis+C+virus+glycoproteins+mediate+pH-dependent+cell+entry+of+pseudotyped+retroviral+particles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001866-Lindenbach1" id="pone.0001866-Lindenbach1"></a>Lindenbach BD, Evans MJ, Syder AJ, Wolk B, Tellinghuisen TL, et al.  (2005) Complete replication of hepatitis C virus in cell culture. Science  309: 623–626.  <ul class="find" data-citedArticleID="1080688" data-doi="10.1126/science.1114016"><li><a href="http://dx.doi.org/10.1126/science.1114016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+replication+of+hepatitis+C+virus+in+cell+culture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+replication+of+hepatitis+C+virus+in+cell+culture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001866-Wakita1" id="pone.0001866-Wakita1"></a>Wakita T, Pietschmann T, Kato T, Date T, Miyamoto M, et al.  (2005) Production of infectious hepatitis C virus in tissue culture from a cloned viral genome. Nat Med  11: 791–796.  <ul class="find" data-citedArticleID="1080718" data-doi="10.1038/nm1268"><li><a href="http://dx.doi.org/10.1038/nm1268" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Production+of+infectious+hepatitis+C+virus+in+tissue+culture+from+a+cloned+viral+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Production+of+infectious+hepatitis+C+virus+in+tissue+culture+from+a+cloned+viral+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001866-Zhong1" id="pone.0001866-Zhong1"></a>Zhong J, Gastaminza P, Cheng G, Kapadia S, Kato T, et al.  (2005) Robust hepatitis C virus infection in vitro. Proc Natl Acad Sci U S A  102: 9294–9299.  <ul class="find" data-citedArticleID="1080728" data-doi="10.1073/pnas.0503596102"><li><a href="http://dx.doi.org/10.1073/pnas.0503596102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robust+hepatitis+C+virus+infection+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robust+hepatitis+C+virus+infection+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001866-OpDeBeeck1" id="pone.0001866-OpDeBeeck1"></a>Op De Beeck A, Cocquerel L, Dubuisson J (2001) Biogenesis of hepatitis C virus envelope glycoproteins. J Gen Virol  82: 2589–2595.  <ul class="find" data-citedArticleID="1080698" data-doi="10.1128/jvi.78.6.2994-3002.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.6.2994-3002.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biogenesis+of+hepatitis+C+virus+envelope+glycoproteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biogenesis+of+hepatitis+C+virus+envelope+glycoproteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001866-Dubuisson1" id="pone.0001866-Dubuisson1"></a>Dubuisson J, Helle F, Cocquerel L (2007) Early steps of the hepatitis C virus life cycle. Cell Microbiol in press.   <ul class="find" data-citedArticleID="1080658" data-doi="10.1111/j.1462-5822.2007.01107.x"><li><a href="http://dx.doi.org/10.1111/j.1462-5822.2007.01107.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+steps+of+the+hepatitis+C+virus+life+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+steps+of+the+hepatitis+C+virus+life+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001866-Bartosch2" id="pone.0001866-Bartosch2"></a>Bartosch B, Vitelli A, Granier C, Goujon C, Dubuisson J, et al.  (2003) Cell entry of hepatitis C virus requires a set of co-receptors that include the CD81 tetraspanin and the SR-B1 scavenger receptor. J Biol Chem  278: 41624–41630.  <ul class="find" data-citedArticleID="1080630" data-doi="10.1074/jbc.m305289200"><li><a href="http://dx.doi.org/10.1074/jbc.m305289200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+entry+of+hepatitis+C+virus+requires+a+set+of+co-receptors+that+include+the+CD81+tetraspanin+and+the+SR-B1+scavenger+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+entry+of+hepatitis+C+virus+requires+a+set+of+co-receptors+that+include+the+CD81+tetraspanin+and+the+SR-B1+scavenger+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001866-Evans1" id="pone.0001866-Evans1"></a>Evans MJ, von Hahn T, Tscherne DM, Syder AJ, Panis M, et al.  (2007) Claudin-1 is a hepatitis C virus co-receptor required for a late step in entry. Nature  446: 801–805.  <ul class="find" data-citedArticleID="1080660" data-doi="10.1038/nature05654"><li><a href="http://dx.doi.org/10.1038/nature05654" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Claudin-1+is+a+hepatitis+C+virus+co-receptor+required+for+a+late+step+in+entry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Claudin-1+is+a+hepatitis+C+virus+co-receptor+required+for+a+late+step+in+entry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001866-Zheng1" id="pone.0001866-Zheng1"></a>Zheng A, Yuan F, Li Y, Zhu F, Hou P, et al.  (2007) Claudin-6 and claudin-9 function as additional coreceptors for hepatitis C virus. J Virol  81: 12465–12471.  <ul class="find" data-citedArticleID="1080726" data-doi="10.1128/jvi.01457-07"><li><a href="http://dx.doi.org/10.1128/jvi.01457-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Claudin-6+and+claudin-9+function+as+additional+coreceptors+for+hepatitis+C+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Claudin-6+and+claudin-9+function+as+additional+coreceptors+for+hepatitis+C+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001866-Yang1" id="pone.0001866-Yang1"></a>Yang W, Qiu C, Biswas N, Jin J, Watkins SC, et al.  (2008) Correlation of the tight junction-like distribution of claudin-1 to the cellular tropism of HCV. J Biol Chem.   <ul class="find" data-citedArticleID="1080720" data-doi="10.1074/jbc.m709824200"><li><a href="http://dx.doi.org/10.1074/jbc.m709824200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+of+the+tight+junction-like+distribution+of+claudin-1+to+the+cellular+tropism+of+HCV." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+of+the+tight+junction-like+distribution+of+claudin-1+to+the+cellular+tropism+of+HCV.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001866-Boucheix1" id="pone.0001866-Boucheix1"></a>Boucheix C, Rubinstein E (2001) Tetraspanins. Cell Mol Life Sci  58: 1189–1205.  <ul class="find" data-citedArticleID="1080634" data-doi="10.1007/pl00000933"><li><a href="http://dx.doi.org/10.1007/pl00000933" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tetraspanins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tetraspanins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001866-Levy1" id="pone.0001866-Levy1"></a>Levy S, Shoham T (2005) The tetraspanin web modulates immune-signalling complexes. Nat Rev Immunol  5: 136–148.  <ul class="find" data-citedArticleID="1080684" data-doi="10.1038/nri1548"><li><a href="http://dx.doi.org/10.1038/nri1548" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tetraspanin+web+modulates+immune-signalling+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tetraspanin+web+modulates+immune-signalling+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001866-Levy2" id="pone.0001866-Levy2"></a>Levy S, Shoham T (2005) Protein-protein interactions in the tetraspanin web. Physiology (Bethesda)  20: 218–224.  <ul class="find" data-citedArticleID="1080686" data-doi="10.1152/physiol.00015.2005"><li><a href="http://dx.doi.org/10.1152/physiol.00015.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein-protein+interactions+in+the+tetraspanin+web." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein-protein+interactions+in+the+tetraspanin+web.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001866-Rubinstein1" id="pone.0001866-Rubinstein1"></a>Rubinstein E, Le Naour F, Lagaudriere-Gesbert C, Billard M, Conjeaud H, et al.  (1996) CD9, CD63, CD81, and CD82 are components of a surface tetraspan network connected to HLA-DR and VLA integrins. Eur J Immunol  26: 2657–2665.  <ul class="find" data-citedArticleID="1080704" data-doi="10.1002/eji.1830261117"><li><a href="http://dx.doi.org/10.1002/eji.1830261117" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD9%2C+CD63%2C+CD81%2C+and+CD82+are+components+of+a+surface+tetraspan+network+connected+to+HLA-DR+and+VLA+integrins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD9%2C+CD63%2C+CD81%2C+and+CD82+are+components+of+a+surface+tetraspan+network+connected+to+HLA-DR+and+VLA+integrins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001866-Charrin1" id="pone.0001866-Charrin1"></a>Charrin S, Le Naour F, Labas V, Billard M, Le Caer JP, et al.  (2003) EWI-2 is a new component of the tetraspanin web in hepatocytes and lymphoid cells. Biochem J  373: 409–421.  <ul class="find" data-citedArticleID="1080636" data-doi="10.1042/bj20030343"><li><a href="http://dx.doi.org/10.1042/bj20030343" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EWI-2+is+a+new+component+of+the+tetraspanin+web+in+hepatocytes+and+lymphoid+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EWI-2+is+a+new+component+of+the+tetraspanin+web+in+hepatocytes+and+lymphoid+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001866-Charrin2" id="pone.0001866-Charrin2"></a>Charrin S, Le Naour F, Oualid M, Billard M, Faure G, et al.  (2001) The major CD9 and CD81 molecular partner. Identification and characterization of the complexes. J Biol Chem  276: 14329–14337.  <ul class="find" data-citedArticleID="1080638"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+major+CD9+and+CD81+molecular+partner.+Identification+and+characterization+of+the+complexes.&amp;auth=&amp;atitle=The+major+CD9+and+CD81+molecular+partner.+Identification+and+characterization+of+the+complexes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+major+CD9+and+CD81+molecular+partner.+Identification+and+characterization+of+the+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+major+CD9+and+CD81+molecular+partner.+Identification+and+characterization+of+the+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001866-Clark1" id="pone.0001866-Clark1"></a>Clark KL, Zeng Z, Langford AL, Bowen SM, Todd SC (2001) PGRL is a major CD81-associated protein on lymphocytes and distinguishes a new family of cell surface proteins. J Immunol  167: 5115–5121.  <ul class="find" data-citedArticleID="1080640"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=PGRL+is+a+major+CD81-associated+protein+on+lymphocytes+and+distinguishes+a+new+family+of+cell+surface+proteins.&amp;auth=&amp;atitle=PGRL+is+a+major+CD81-associated+protein+on+lymphocytes+and+distinguishes+a+new+family+of+cell+surface+proteins." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PGRL+is+a+major+CD81-associated+protein+on+lymphocytes+and+distinguishes+a+new+family+of+cell+surface+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PGRL+is+a+major+CD81-associated+protein+on+lymphocytes+and+distinguishes+a+new+family+of+cell+surface+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001866-Stipp1" id="pone.0001866-Stipp1"></a>Stipp CS, Kolesnikova TV, Hemler ME (2001) EWI-2 is a major CD9 and CD81 partner and member of a novel Ig protein subfamily. J Biol Chem  276: 40545–40554.  <ul class="find" data-citedArticleID="1080710" data-doi="10.1074/jbc.m107338200"><li><a href="http://dx.doi.org/10.1074/jbc.m107338200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EWI-2+is+a+major+CD9+and+CD81+partner+and+member+of+a+novel+Ig+protein+subfamily." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EWI-2+is+a+major+CD9+and+CD81+partner+and+member+of+a+novel+Ig+protein+subfamily.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001866-Stipp2" id="pone.0001866-Stipp2"></a>Stipp CS, Orlicky D, Hemler ME (2001) FPRP, a major, highly stoichiometric, highly specific CD81- and CD9- associated protein. J Biol Chem  276: 4853–4862.  <ul class="find" data-citedArticleID="1080712" data-doi="10.1074/jbc.m009859200"><li><a href="http://dx.doi.org/10.1074/jbc.m009859200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FPRP%2C+a+major%2C+highly+stoichiometric%2C+highly+specific+CD81-+and+CD9-+associated+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FPRP%2C+a+major%2C+highly+stoichiometric%2C+highly+specific+CD81-+and+CD9-+associated+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001866-SalaValdes1" id="pone.0001866-SalaValdes1"></a>Sala-Valdes M, Ursa A, Charrin S, Rubinstein E, Hemler ME, et al.  (2006) EWI-2 and EWI-F link the tetraspanin web to the actin cytoskeleton through their direct association with ezrin-radixin-moesin proteins. J Biol Chem  281: 19665–19675.  <ul class="find" data-citedArticleID="1080706" data-doi="10.1074/jbc.m602116200"><li><a href="http://dx.doi.org/10.1074/jbc.m602116200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EWI-2+and+EWI-F+link+the+tetraspanin+web+to+the+actin+cytoskeleton+through+their+direct+association+with+ezrin-radixin-moesin+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EWI-2+and+EWI-F+link+the+tetraspanin+web+to+the+actin+cytoskeleton+through+their+direct+association+with+ezrin-radixin-moesin+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001866-Kolesnikova1" id="pone.0001866-Kolesnikova1"></a>Kolesnikova TV, Stipp CS, Rao RM, Lane WS, Luscinskas FW, et al.  (2004) EWI-2 modulates lymphocyte integrin alpha4beta1 functions. Blood  103: 3013–3019.  <ul class="find" data-citedArticleID="1080676" data-doi="10.1182/blood-2003-07-2201"><li><a href="http://dx.doi.org/10.1182/blood-2003-07-2201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EWI-2+modulates+lymphocyte+integrin+alpha4beta1+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EWI-2+modulates+lymphocyte+integrin+alpha4beta1+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001866-Stipp3" id="pone.0001866-Stipp3"></a>Stipp CS, Kolesnikova TV, Hemler ME (2003) EWI-2 regulates alpha3beta1 integrin-dependent cell functions on laminin-5. J Cell Biol  163: 1167–1177.  <ul class="find" data-citedArticleID="1080714" data-doi="10.1083/jcb.200309113"><li><a href="http://dx.doi.org/10.1083/jcb.200309113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EWI-2+regulates+alpha3beta1+integrin-dependent+cell+functions+on+laminin-5." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EWI-2+regulates+alpha3beta1+integrin-dependent+cell+functions+on+laminin-5.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001866-Zhang1" id="pone.0001866-Zhang1"></a>Zhang XA, Lane WS, Charrin S, Rubinstein E, Liu L (2003) EWI2/PGRL associates with the metastasis suppressor KAI1/CD82 and inhibits the migration of prostate cancer cells. Cancer Res  63: 2665–2674.  <ul class="find" data-citedArticleID="1080724"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=EWI2%2FPGRL+associates+with+the+metastasis+suppressor+KAI1%2FCD82+and+inhibits+the+migration+of+prostate+cancer+cells.&amp;auth=&amp;atitle=EWI2%2FPGRL+associates+with+the+metastasis+suppressor+KAI1%2FCD82+and+inhibits+the+migration+of+prostate+cancer+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EWI2%2FPGRL+associates+with+the+metastasis+suppressor+KAI1%2FCD82+and+inhibits+the+migration+of+prostate+cancer+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EWI2%2FPGRL+associates+with+the+metastasis+suppressor+KAI1%2FCD82+and+inhibits+the+migration+of+prostate+cancer+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001866-Cocquerel1" id="pone.0001866-Cocquerel1"></a>Cocquerel L, Quinn ER, Flint M, Hadlock KG, Foung SK, et al.  (2003) Recognition of native hepatitis C virus E1E2 heterodimers by a human monoclonal antibody. J Virol  77: 1604–1609.  <ul class="find" data-citedArticleID="1080642" data-doi="10.1128/jvi.77.2.1604-1609.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.2.1604-1609.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+native+hepatitis+C+virus+E1E2+heterodimers+by+a+human+monoclonal+antibody." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+native+hepatitis+C+virus+E1E2+heterodimers+by+a+human+monoclonal+antibody.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001866-Higginbottom1" id="pone.0001866-Higginbottom1"></a>Higginbottom A, Quinn ER, Kuo CC, Flint M, Wilson LH, et al.  (2000) Identification of amino acid residues in CD81 critical for interaction with hepatitis C virus envelope glycoprotein E2. J Virol  74: 3642–3649.  <ul class="find" data-citedArticleID="1080666" data-doi="10.1128/jvi.74.8.3642-3649.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.8.3642-3649.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+amino+acid+residues+in+CD81+critical+for+interaction+with+hepatitis+C+virus+envelope+glycoprotein+E2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+amino+acid+residues+in+CD81+critical+for+interaction+with+hepatitis+C+virus+envelope+glycoprotein+E2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001866-Kazarov1" id="pone.0001866-Kazarov1"></a>Kazarov AR, Yang X, Stipp CS, Sehgal B, Hemler ME (2002) An extracellular site on tetraspanin CD151 determines alpha 3 and alpha 6 integrin-dependent cellular morphology. J Cell Biol  158: 1299–1309.  <ul class="find" data-citedArticleID="1080674" data-doi="10.1083/jcb.200204056"><li><a href="http://dx.doi.org/10.1083/jcb.200204056" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+extracellular+site+on+tetraspanin+CD151+determines+alpha+3+and+alpha+6+integrin-dependent+cellular+morphology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+extracellular+site+on+tetraspanin+CD151+determines+alpha+3+and+alpha+6+integrin-dependent+cellular+morphology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001866-Yang2" id="pone.0001866-Yang2"></a>Yang XH, Kovalenko OV, Kolesnikova TV, Andzelm MM, Rubinstein E, et al.  (2006) Contrasting effects of EWI proteins, integrins, and protein palmitoylation on cell surface CD9 organization. J Biol Chem  281: 12976–12985.  <ul class="find" data-citedArticleID="1080722" data-doi="10.1074/jbc.m510617200"><li><a href="http://dx.doi.org/10.1074/jbc.m510617200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contrasting+effects+of+EWI+proteins%2C+integrins%2C+and+protein+palmitoylation+on+cell+surface+CD9+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contrasting+effects+of+EWI+proteins%2C+integrins%2C+and+protein+palmitoylation+on+cell+surface+CD9+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001866-Kapadia1" id="pone.0001866-Kapadia1"></a>Kapadia SB, Barth H, Baumert T, McKeating JA, Chisari FV (2007) Initiation of Hepatitis C Virus Infection Is Dependent on Cholesterol and Cooperativity between CD81 and Scavenger Receptor B Type I. J Virol  81: 374–383.  <ul class="find" data-citedArticleID="1080670" data-doi="10.1128/jvi.01134-06"><li><a href="http://dx.doi.org/10.1128/jvi.01134-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initiation+of+Hepatitis+C+Virus+Infection+Is+Dependent+on+Cholesterol+and+Cooperativity+between+CD81+and+Scavenger+Receptor+B+Type+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initiation+of+Hepatitis+C+Virus+Infection+Is+Dependent+on+Cholesterol+and+Cooperativity+between+CD81+and+Scavenger+Receptor+B+Type+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001866-Koutsoudakis1" id="pone.0001866-Koutsoudakis1"></a>Koutsoudakis G, Herrmann E, Kallis S, Bartenschlager R, Pietschmann T (2007) The level of CD81 cell surface expression is a key determinant for productive entry of hepatitis C virus into host cells. J Virol  81: 588–598.  <ul class="find" data-citedArticleID="1080678" data-doi="10.1128/jvi.01534-06"><li><a href="http://dx.doi.org/10.1128/jvi.01534-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+level+of+CD81+cell+surface+expression+is+a+key+determinant+for+productive+entry+of+hepatitis+C+virus+into+host+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+level+of+CD81+cell+surface+expression+is+a+key+determinant+for+productive+entry+of+hepatitis+C+virus+into+host+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001866-Molina1" id="pone.0001866-Molina1"></a>Molina S, Castet V, Pichard-Garcia L, Wychowski C, Meurs E, et al.  (2008) Serum-derived hepatitis C virus infection of primary human hepatocytes is tetraspanin CD81 dependent. J Virol  82: 569–574.  <ul class="find" data-citedArticleID="1080696" data-doi="10.1128/jvi.01443-07"><li><a href="http://dx.doi.org/10.1128/jvi.01443-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Serum-derived+hepatitis+C+virus+infection+of+primary+human+hepatocytes+is+tetraspanin+CD81+dependent." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Serum-derived+hepatitis+C+virus+infection+of+primary+human+hepatocytes+is+tetraspanin+CD81+dependent.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001866-Cormier1" id="pone.0001866-Cormier1"></a>Cormier EG, Tsamis F, Kajumo F, Durso RJ, Gardner JP, et al.  (2004) CD81 is an entry coreceptor for hepatitis C virus. Proc Natl Acad Sci U S A  101: 7270–7274.  <ul class="find" data-citedArticleID="1080652" data-doi="10.1073/pnas.0402253101"><li><a href="http://dx.doi.org/10.1073/pnas.0402253101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD81+is+an+entry+coreceptor+for+hepatitis+C+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD81+is+an+entry+coreceptor+for+hepatitis+C+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001866-Flint1" id="pone.0001866-Flint1"></a>Flint M, von Hahn T, Zhang J, Farquhar M, Jones CT, et al.  (2006) Diverse CD81 proteins support hepatitis C virus infection. J Virol  80: 11331–11342.  <ul class="find" data-citedArticleID="1080662" data-doi="10.1128/jvi.00104-06"><li><a href="http://dx.doi.org/10.1128/jvi.00104-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diverse+CD81+proteins+support+hepatitis+C+virus+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diverse+CD81+proteins+support+hepatitis+C+virus+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001866-Koutsoudakis2" id="pone.0001866-Koutsoudakis2"></a>Koutsoudakis G, Kaul A, Steinmann E, Kallis S, Lohmann V, et al.  (2006) Characterization of the early steps of hepatitis C virus infection by using luciferase reporter viruses. J Virol  80: 5308–5320.  <ul class="find" data-citedArticleID="1080680" data-doi="10.1128/jvi.02460-05"><li><a href="http://dx.doi.org/10.1128/jvi.02460-05" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+early+steps+of+hepatitis+C+virus+infection+by+using+luciferase+reporter+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+early+steps+of+hepatitis+C+virus+infection+by+using+luciferase+reporter+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001866-Meertens1" id="pone.0001866-Meertens1"></a>Meertens L, Bertaux C, Cukierman L, Cormier E, Lavillette D, et al.  (2008) The tight junction proteins claudin-1, -6 and -9 are entry cofactors for the Hepatitis C virus. J Virol.   <ul class="find" data-citedArticleID="1080694" data-doi="10.1128/jvi.01977-07"><li><a href="http://dx.doi.org/10.1128/jvi.01977-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tight+junction+proteins+claudin-1%2C+-6+and+-9+are+entry+cofactors+for+the+Hepatitis+C+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tight+junction+proteins+claudin-1%2C+-6+and+-9+are+entry+cofactors+for+the+Hepatitis+C+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001866-Bost1" id="pone.0001866-Bost1"></a>Bost AG, Venable D, Liu L, Heinz BA (2003) Cytoskeletal requirements for hepatitis C virus (HCV) RNA synthesis in the HCV replicon cell culture system. J Virol  77: 4401–4408.  <ul class="find" data-citedArticleID="1080632" data-doi="10.1128/jvi.77.7.4401-4408.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.7.4401-4408.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytoskeletal+requirements+for+hepatitis+C+virus+%28HCV%29+RNA+synthesis+in+the+HCV+replicon+cell+culture+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytoskeletal+requirements+for+hepatitis+C+virus+%28HCV%29+RNA+synthesis+in+the+HCV+replicon+cell+culture+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001866-Coffey1" id="pone.0001866-Coffey1"></a>Coffey G, Kuo C-C, Levy S Co-ligation of CD81 and the B-cell receptor by Hepatitis C Virus envelope protein E2 enhances B-cell signaling. In: Conferences FSR, editor. Tucson, AZ, USA:   2006;.  <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0001866-Crotta1" id="pone.0001866-Crotta1"></a>Crotta S, Ronconi V, Ulivieri C, Baldari CT, Valiante NM, et al.  (2006) Cytoskeleton rearrangement induced by tetraspanin engagement modulates the activation of T and NK cells. Eur J Immunol  36: 919–929.  <ul class="find" data-citedArticleID="1080654" data-doi="10.1002/eji.200535527"><li><a href="http://dx.doi.org/10.1002/eji.200535527" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytoskeleton+rearrangement+induced+by+tetraspanin+engagement+modulates+the+activation+of+T+and+NK+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytoskeleton+rearrangement+induced+by+tetraspanin+engagement+modulates+the+activation+of+T+and+NK+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001866-Cocquerel2" id="pone.0001866-Cocquerel2"></a>Cocquerel L, Voisset C, Dubuisson J (2006) Hepatitis C virus entry: potential receptors and their biological functions. J Gen Virol  87: 1075–1084.  <ul class="find" data-citedArticleID="1080644" data-doi="10.1099/vir.0.81646-0"><li><a href="http://dx.doi.org/10.1099/vir.0.81646-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hepatitis+C+virus+entry%3A+potential+receptors+and+their+biological+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hepatitis+C+virus+entry%3A+potential+receptors+and+their+biological+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001866-Cocquerel3" id="pone.0001866-Cocquerel3"></a>Cocquerel L, Kuo CC, Dubuisson J, Levy S (2003) CD81-dependent binding of hepatitis C virus E1E2 heterodimers. J Virol  77: 10677–10683.  <ul class="find" data-citedArticleID="1080646" data-doi="10.1128/jvi.77.19.10677-10683.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.19.10677-10683.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD81-dependent+binding+of+hepatitis+C+virus+E1E2+heterodimers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD81-dependent+binding+of+hepatitis+C+virus+E1E2+heterodimers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001866-Oren1" id="pone.0001866-Oren1"></a>Oren R, Takahashi S, Doss C, Levy R, Levy S (1990) TAPA-1, the target of an antiproliferative antibody, defines a new family of transmembrane proteins. Mol Cell Biol  10: 4007–4015.  <ul class="find" data-citedArticleID="1080702"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=TAPA-1%2C+the+target+of+an+antiproliferative+antibody%2C+defines+a+new+family+of+transmembrane+proteins.&amp;auth=&amp;atitle=TAPA-1%2C+the+target+of+an+antiproliferative+antibody%2C+defines+a+new+family+of+transmembrane+proteins." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TAPA-1%2C+the+target+of+an+antiproliferative+antibody%2C+defines+a+new+family+of+transmembrane+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TAPA-1%2C+the+target+of+an+antiproliferative+antibody%2C+defines+a+new+family+of+transmembrane+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001866-Cocquerel4" id="pone.0001866-Cocquerel4"></a>Cocquerel L, Meunier JC, Pillez A, Wychowski C, Dubuisson J (1998) A retention signal necessary and sufficient for endoplasmic reticulum localization maps to the transmembrane domain of hepatitis C virus glycoprotein E2. J Virol  72: 2183–2191.  <ul class="find" data-citedArticleID="1080648" data-doi="10.1016/s0248-4900(98)80287-6"><li><a href="http://dx.doi.org/10.1016/s0248-4900(98)80287-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+retention+signal+necessary+and+sufficient+for+endoplasmic+reticulum+localization+maps+to+the+transmembrane+domain+of+hepatitis+C+virus+glycoprotein+E2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+retention+signal+necessary+and+sufficient+for+endoplasmic+reticulum+localization+maps+to+the+transmembrane+domain+of+hepatitis+C+virus+glycoprotein+E2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001866-Flint2" id="pone.0001866-Flint2"></a>Flint M, Maidens C, Loomis-Price LD, Shotton C, Dubuisson J, et al.  (1999) Characterization of hepatitis C virus E2 glycoprotein interaction with a putative cellular receptor, CD81. J Virol  73: 6235–6244.  <ul class="find" data-citedArticleID="1080664"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+hepatitis+C+virus+E2+glycoprotein+interaction+with+a+putative+cellular+receptor%2C+CD81.&amp;auth=&amp;atitle=Characterization+of+hepatitis+C+virus+E2+glycoprotein+interaction+with+a+putative+cellular+receptor%2C+CD81." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+hepatitis+C+virus+E2+glycoprotein+interaction+with+a+putative+cellular+receptor%2C+CD81." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+hepatitis+C+virus+E2+glycoprotein+interaction+with+a+putative+cellular+receptor%2C+CD81.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001866-Maillard1" id="pone.0001866-Maillard1"></a>Maillard P, Krawczynski K, Nitkiewicz J, Bronnert C, Sidorkiewicz M, et al.  (2001) Nonenveloped nucleocapsids of hepatitis C virus in the serum of infected patients. J Virol  75: 8240–8250.  <ul class="find" data-citedArticleID="1080690" data-doi="10.1128/jvi.75.17.8240-8250.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.17.8240-8250.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonenveloped+nucleocapsids+of+hepatitis+C+virus+in+the+serum+of+infected+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonenveloped+nucleocapsids+of+hepatitis+C+virus+in+the+serum+of+infected+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001866-Delgrange1" id="pone.0001866-Delgrange1"></a>Delgrange D, Pillez A, Castelain S, Cocquerel L, Rouille Y, et al.  (2007) Robust production of infectious viral particles in Huh-7 cells by introducing mutations in hepatitis C virus structural proteins. J Gen Virol  88: 2495–2503.  <ul class="find" data-citedArticleID="1080656" data-doi="10.1099/vir.0.82872-0"><li><a href="http://dx.doi.org/10.1099/vir.0.82872-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robust+production+of+infectious+viral+particles+in+Huh-7+cells+by+introducing+mutations+in+hepatitis+C+virus+structural+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robust+production+of+infectious+viral+particles+in+Huh-7+cells+by+introducing+mutations+in+hepatitis+C+virus+structural+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001866-Tscherne1" id="pone.0001866-Tscherne1"></a>Tscherne DM, Jones CT, Evans MJ, Lindenbach BD, McKeating JA, et al.  (2006) Time- and temperature-dependent activation of hepatitis C virus for low-pH-triggered entry. J Virol  80: 1734–1741.  <ul class="find" data-citedArticleID="1080716" data-doi="10.1128/jvi.80.4.1734-1741.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.4.1734-1741.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Time-+and+temperature-dependent+activation+of+hepatitis+C+virus+for+low-pH-triggered+entry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Time-+and+temperature-dependent+activation+of+hepatitis+C+virus+for+low-pH-triggered+entry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001866-Kato1" id="pone.0001866-Kato1"></a>Kato T, Date T, Miyamoto M, Furusaka A, Tokushige K, et al.  (2003) Efficient replication of the genotype 2a hepatitis C virus subgenomic replicon. Gastroenterology  125: 1808–1817.  <ul class="find" data-citedArticleID="1080672" data-doi="10.1053/j.gastro.2003.09.023"><li><a href="http://dx.doi.org/10.1053/j.gastro.2003.09.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+replication+of+the+genotype+2a+hepatitis+C+virus+subgenomic+replicon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+replication+of+the+genotype+2a+hepatitis+C+virus+subgenomic+replicon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001866-OpDeBeeck2" id="pone.0001866-OpDeBeeck2"></a>Op De Beeck A, Voisset C, Bartosch B, Ciczora Y, Cocquerel L, et al.  (2004) Characterization of functional hepatitis C virus envelope glycoproteins. J Virol  78: 2994–3002.  <ul class="find" data-citedArticleID="1080700" data-doi="10.1128/jvi.78.6.2994-3002.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.6.2994-3002.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+functional+hepatitis+C+virus+envelope+glycoproteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+functional+hepatitis+C+virus+envelope+glycoproteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001866-Sandrin1" id="pone.0001866-Sandrin1"></a>Sandrin V, Boson B, Salmon P, Gay W, Negre D, et al.  (2002) Lentiviral vectors pseudotyped with a modified RD114 envelope glycoprotein show increased stability in sera and augmented transduction of primary lymphocytes and CD34+ cells derived from human and nonhuman primates. Blood  100: 823–832.  <ul class="find" data-citedArticleID="1080708" data-doi="10.1182/blood-2001-11-0042"><li><a href="http://dx.doi.org/10.1182/blood-2001-11-0042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lentiviral+vectors+pseudotyped+with+a+modified+RD114+envelope+glycoprotein+show+increased+stability+in+sera+and+augmented+transduction+of+primary+lymphocytes+and+CD34%2B+cells+derived+from+human+and+nonhuman+primates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lentiviral+vectors+pseudotyped+with+a+modified+RD114+envelope+glycoprotein+show+increased+stability+in+sera+and+augmented+transduction+of+primary+lymphocytes+and+CD34%2B+cells+derived+from+human+and+nonhuman+primates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.XML" value="128400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.PDF" value="1163048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g001.PNG_L" value="1518964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g001.PNG_M" value="85860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g001.PNG_S" value="7153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g001.TIF" value="1471890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.t001.PNG_L" value="208394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.t001.PNG_M" value="123667"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.t001.PNG_S" value="13486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.t001.TIF" value="1103082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.t001.PNG_I" value="39068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g002.PNG_L" value="359655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g002.PNG_M" value="78184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g002.PNG_S" value="3901"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g002.TIF" value="388588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g002.PNG_I" value="26741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g003.PNG_L" value="813327"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g003.PNG_M" value="304339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g003.PNG_S" value="7956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g003.TIF" value="1165786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g003.PNG_I" value="111817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.t002.PNG_L" value="249236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.t002.PNG_M" value="144472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.t002.PNG_S" value="15033"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.t002.TIF" value="1293828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.t002.PNG_I" value="43623"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g004.PNG_L" value="442533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g004.PNG_M" value="87379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g004.PNG_S" value="3645"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g004.TIF" value="434754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g004.PNG_I" value="29406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g005.PNG_L" value="683068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g005.PNG_M" value="170518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g005.PNG_S" value="7108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g005.TIF" value="1036630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g005.PNG_I" value="59068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g006.PNG_L" value="573708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g006.PNG_M" value="132987"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g006.PNG_S" value="7167"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g006.TIF" value="984094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g006.PNG_I" value="48235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g007.PNG_L" value="220774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g007.PNG_M" value="27390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g007.PNG_S" value="4936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g007.TIF" value="471814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001866.g007.PNG_I" value="23497"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001866&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001866" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001866&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001866&volume=&issue=&title=The CD81 Partner EWI-2wint Inhibits Hepatitis C Virus Entry&author_name=Vera%20Rocha-Perugini%2C%20Claire%20Montpellier%2C%20David%20Delgrange%2C%20Czeslaw%20Wychowski%2C%20Fran%C3%A7ois%20Helle%2C%20Andr%C3%A9%20Pillez%2C%20Herv%C3%A9%20Drobecq%2C%20Fran%C3%A7ois%20Le%20Naour%2C%20St%C3%A9phanie%20Charrin%2C%20Shoshana%20Levy%2C%20Eric%20Rubinstein%2C%20Jean%20Dubuisson%2C%20Laurence%20Cocquerel&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001866" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001866" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001866" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001866&amp;t=The%20CD81%20Partner%20EWI-2wint%20Inhibits%20Hepatitis%20C%20Virus%20Entry" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001866&title=The%20CD81%20Partner%20EWI-2wint%20Inhibits%20Hepatitis%20C%20Virus%20Entry&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001866&amp;title=The%20CD81%20Partner%20EWI-2wint%20Inhibits%20Hepatitis%20C%20Virus%20Entry" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001866" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001866&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The CD81 Partner EWI-2wint Inhibits Hepatitis C Virus Entry';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001866';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20CD81%20Partner%20EWI-2wint%20Inhibits%20Hepatitis%20C%20Virus%20Entry http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001866" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001866" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Antibodies%22" title="Search for articles in the subject area:'Antibodies'"><div class="flagText">Antibodies</div></a>
              <div data-categoryid="47389" data-articleid="26556"
                   data-categoryname="Antibodies"
                   class="flagImage" title="Flag 'Antibodies' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Co-immunoprecipitation%22" title="Search for articles in the subject area:'Co-immunoprecipitation'"><div class="flagText">Co-immunoprecipitation</div></a>
              <div data-categoryid="27791" data-articleid="26556"
                   data-categoryname="Co-immunoprecipitation"
                   class="flagImage" title="Flag 'Co-immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Daudi+cells%22" title="Search for articles in the subject area:'Daudi cells'"><div class="flagText">Daudi cells</div></a>
              <div data-categoryid="27793" data-articleid="26556"
                   data-categoryname="Daudi cells"
                   class="flagImage" title="Flag 'Daudi cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glycoproteins%22" title="Search for articles in the subject area:'Glycoproteins'"><div class="flagText">Glycoproteins</div></a>
              <div data-categoryid="19265" data-articleid="26556"
                   data-categoryname="Glycoproteins"
                   class="flagImage" title="Flag 'Glycoproteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hepatocytes%22" title="Search for articles in the subject area:'Hepatocytes'"><div class="flagText">Hepatocytes</div></a>
              <div data-categoryid="47449" data-articleid="26556"
                   data-categoryname="Hepatocytes"
                   class="flagImage" title="Flag 'Hepatocytes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Luciferase%22" title="Search for articles in the subject area:'Luciferase'"><div class="flagText">Luciferase</div></a>
              <div data-categoryid="31895" data-articleid="26556"
                   data-categoryname="Luciferase"
                   class="flagImage" title="Flag 'Luciferase' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26556"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+replication%22" title="Search for articles in the subject area:'Viral replication'"><div class="flagText">Viral replication</div></a>
              <div data-categoryid="20015" data-articleid="26556"
                   data-categoryname="Viral replication"
                   class="flagImage" title="Flag 'Viral replication' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3531'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=462'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9498&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
