

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Fine Mapping of Posttranslational Modifications of the Linker Histone H1 from Drosophila melanogaster</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001553"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001553" />

    <meta name="citation_title" content="Fine Mapping of Posttranslational Modifications of the Linker Histone H1 from Drosophila melanogaster"/>
    <meta itemprop="name" content="Fine Mapping of Posttranslational Modifications of the Linker Histone H1 from Drosophila melanogaster"/>

      <meta name="citation_author" content="Ana Villar-Garea"/>
            <meta name="citation_author_institution" content="Munich Center for Integrated Protein Science CIPSM, Histone Modifications Group, Adolf-Butenandt Institute, Ludwig-Maximilians University of Munich, Munich, Germany"/>
      <meta name="citation_author" content="Axel Imhof"/>
            <meta name="citation_author_institution" content="Munich Center for Integrated Protein Science CIPSM, Histone Modifications Group, Adolf-Butenandt Institute, Ludwig-Maximilians University of Munich, Munich, Germany"/>

    <meta name="citation_date" content="2008/2/6"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001553.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1553"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Origin of H1 linker histones.; citation_author=HE Kasinsky; citation_author=JD Lewis; citation_author=JB Dacks; citation_author=J Ausio; citation_journal_title=Faseb J; citation_volume=15; citation_number=1; citation_pages=34-42; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Histone H1.; citation_author=AP Wolffe; citation_journal_title=Int J Biochem Cell Biol; citation_volume=29; citation_number=2; citation_pages=1463-1466; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Role of linker histone in chromatin structure and function: H1 stoichiometry and nucleosome repeat length.; citation_author=CL Woodcock; citation_author=AI Skoultchi; citation_author=Y Fan; citation_journal_title=Chromosome Res; citation_volume=14; citation_number=3; citation_pages=17-25; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The dynamics of histone H1 function in chromatin.; citation_author=M Bustin; citation_author=F Catez; citation_author=JH Lim; citation_journal_title=Mol Cell; citation_volume=17; citation_number=4; citation_pages=617-620; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=H1 linker histones are essential for mouse development and affect nucleosome spacing in vivo.; citation_author=Y Fan; citation_author=T Nikitina; citation_author=EM Morin-Kensicki; citation_author=J Zhao; citation_author=TR Magnuson; citation_journal_title=Mol Cell Biol; citation_volume=23; citation_number=5; citation_pages=4559-4572; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Homogeneous reconstituted oligonucleosomes, evidence for salt-dependent folding in the absence of histone H1.; citation_author=JC Hansen; citation_author=J Ausio; citation_author=VH Stanik; citation_author=KE van Holde; citation_journal_title=Biochemistry; citation_volume=28; citation_number=6; citation_pages=9129-9136; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=An asymmetric model for the nucleosome: a binding site for linker histones inside the DNA gyres.; citation_author=D Pruss; citation_author=B Bartholomew; citation_author=J Persinger; citation_author=J Hayes; citation_author=G Arents; citation_journal_title=Science; citation_volume=274; citation_number=7; citation_pages=614-617; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Differential association of HMG1 and linker histones B4 and H1 with dinucleosomal DNA: structural transitions and transcriptional repression.; citation_author=K Ura; citation_author=K Nightingale; citation_author=AP Wolffe; citation_journal_title=Embo J; citation_volume=15; citation_number=8; citation_pages=4959-4969; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Histone hyperacetylation does not alter the positioning or stability of phased nucleosomes on the mouse mammary tumor virus long terminal repeat.; citation_author=EH Bresnick; citation_author=S John; citation_author=GL Hager; citation_journal_title=Biochemistry; citation_volume=30; citation_number=9; citation_pages=3490-3497; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Histone H1 enhances synergistic activation of the MMTV promoter in chromatin.; citation_author=R Koop; citation_author=L Di Croce; citation_author=M Beato; citation_journal_title=Embo J; citation_volume=22; citation_number=10; citation_pages=588-599; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Somatic linker histones cause loss of mesodermal competence in Xenopus.; citation_author=OC Steinbach; citation_author=AP Wolffe; citation_author=RA Rupp; citation_journal_title=Nature; citation_volume=389; citation_number=11; citation_pages=395-399; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Co-operative binding of the globular domain of histone H5 to DNA.; citation_author=PH Draves; citation_author=PT Lowary; citation_author=J Widom; citation_journal_title=J Mol Biol; citation_volume=225; citation_number=12; citation_pages=1105-1121; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Cooperative binding of the globular domains of histones H1 and H5 to DNA.; citation_author=JO Thomas; citation_author=C Rees; citation_author=JT Finch; citation_journal_title=Nucleic Acids Res; citation_volume=20; citation_number=13; citation_pages=187-194; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The globular domain of histone H1 is sufficient to direct specific gene repression in early Xenopus embryos.; citation_author=D Vermaak; citation_author=OC Steinbach; citation_author=S Dimitrov; citation_author=RA Rupp; citation_author=AP Wolffe; citation_journal_title=Curr Biol; citation_volume=8; citation_number=14; citation_pages=533-536; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Sequence complexity of histone H1 subtypes.; citation_author=I Ponte; citation_author=R Vila; citation_author=P Suau; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=15; citation_pages=371-380; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of a 34 amino acid stretch within the C-terminus of histone H1 as the DNA-condensing domain by site-directed mutagenesis.; citation_author=MM Bharath; citation_author=S Ramesh; citation_author=NR Chandra; citation_author=MR Rao; citation_journal_title=Biochemistry; citation_volume=41; citation_number=16; citation_pages=7617-7627; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Highly preferential nucleation of histone H1 assembly on scaffold-associated regions.; citation_author=E Izaurralde; citation_author=E Kas; citation_author=UK Laemmli; citation_journal_title=J Mol Biol; citation_volume=210; citation_number=17; citation_pages=573-585; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The H1 phosphorylation state regulates expression of CDC2 and other genes in response to starvation in Tetrahymena thermophila.; citation_author=Y Dou; citation_author=X Song; citation_author=Y Liu; citation_author=MA Gorovsky; citation_journal_title=Mol Cell Biol; citation_volume=25; citation_number=18; citation_pages=3914-3922; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Characterization of phosphorylation sites on histone H1 isoforms by tandem mass spectrometry.; citation_author=BA Garcia; citation_author=SA Busby; citation_author=CM Barber; citation_author=J Shabanowitz; citation_author=CD Allis; citation_journal_title=J Proteome Res; citation_volume=3; citation_number=19; citation_pages=1219-1227; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Characterization of sequence variations in human histone H1.2 and H1.4 subtypes.; citation_author=B Sarg; citation_author=A Green; citation_author=P Soderkvist; citation_author=W Helliger; citation_author=I Rundquist; citation_journal_title=Febs J; citation_volume=272; citation_number=20; citation_pages=3673-3683; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=In vivo phosphorylation of histone H1 variants during the cell cycle.; citation_author=H Talasz; citation_author=W Helliger; citation_author=B Puschendorf; citation_author=H Lindner; citation_journal_title=Biochemistry; citation_volume=35; citation_number=21; citation_pages=1761-1767; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=H1 family histones in the nucleus. Control of binding and localization by the C-terminal domain.; citation_author=JP Th'ng; citation_author=R Sung; citation_author=M Ye; citation_author=MJ Hendzel; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=22; citation_pages=27809-27814; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Linker histone binding and displacement: versatile mechanism for transcriptional regulation.; citation_author=J Zlatanova; citation_author=P Caiafa; citation_author=K Van Holde; citation_journal_title=Faseb J; citation_volume=14; citation_number=23; citation_pages=1697-1704; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Different EZH2-containing complexes target methylation of histone H1 or nucleosomal histone H3.; citation_author=A Kuzmichev; citation_author=T Jenuwein; citation_author=P Tempst; citation_author=D Reinberg; citation_journal_title=Mol Cell; citation_volume=14; citation_number=24; citation_pages=183-193; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=HP1 binds specifically to Lys26-methylated histone H1.4, whereas simultaneous Ser27 phosphorylation blocks HP1 binding.; citation_author=S Daujat; citation_author=U Zeissler; citation_author=T Waldmann; citation_author=N Happel; citation_author=R Schneider; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=25; citation_pages=38090-38095; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mass spectrometric mapping of linker histone H1 variants reveals multiple acetylations, methylations, and phosphorylation as well as differences between cell culture and tissue.; citation_author=JR Wisniewski; citation_author=A Zougman; citation_author=S Kruger; citation_author=M Mann; citation_journal_title=Mol Cell Proteomics; citation_volume=6; citation_number=26; citation_pages=72-87; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=N-formylation of lysine in histone proteins as a secondary modification arising from oxidative DNA damage.; citation_author=T Jiang; citation_author=X Zhou; citation_author=K Taghizadeh; citation_author=M Dong; citation_author=PC Dedon; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=27; citation_pages=60-65; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phosphorylation of linker histones regulates ATP-dependent chromatin remodeling enzymes.; citation_author=PJ Horn; citation_author=LM Carruthers; citation_author=C Logie; citation_author=DA Hill; citation_author=MJ Solomon; citation_journal_title=Nat Struct Biol; citation_volume=9; citation_number=28; citation_pages=263-267; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Phosphorylation of linker histone H1 regulates gene expression in vivo by mimicking H1 removal.; citation_author=Y Dou; citation_author=CA Mizzen; citation_author=M Abrams; citation_author=CD Allis; citation_author=MA Gorovsky; citation_journal_title=Mol Cell; citation_volume=4; citation_number=29; citation_pages=641-647; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Phosphorylation of the linker histone H1 by CDK regulates its binding to HP1alpha.; citation_author=TK Hale; citation_author=A Contreras; citation_author=AJ Morrison; citation_author=RE Herrera; citation_journal_title=Mol Cell; citation_volume=22; citation_number=30; citation_pages=693-699; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Histone H1 depletion in mammals alters global chromatin structure but causes specific changes in gene regulation.; citation_author=Y Fan; citation_author=T Nikitina; citation_author=J Zhao; citation_author=TJ Fleury; citation_author=R Bhattacharyya; citation_journal_title=Cell; citation_volume=123; citation_number=31; citation_pages=1199-1212; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Changing rates of histone mRNA synthesis and turnover in Drosophila embryos.; citation_author=KV Anderson; citation_author=JA Lengyel; citation_journal_title=Cell; citation_volume=21; citation_number=32; citation_pages=717-727; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=HMG-D and histone H1 interplay during chromatin assembly and early embryogenesis.; citation_author=SS Ner; citation_author=T Blank; citation_author=ML Perez-Paralle; citation_author=TA Grigliatti; citation_author=PB Becker; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=33; citation_pages=37569-37576; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=HMG-D, the Drosophila melanogaster homologue of HMG 1 protein, is associated with early embryonic chromatin in the absence of histone H1.; citation_author=SS Ner; citation_author=AA Travers; citation_journal_title=EMBO J; citation_volume=13; citation_number=34; citation_pages=1817-1822; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Phosphorylated H1 histone in Drosophila melanogaster.; citation_author=M Blumenfeld; citation_journal_title=Biochem Genet; citation_volume=17; citation_number=35; citation_pages=163-166; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Drosophila melanogaster H1 histone is phosphorylated stably.; citation_author=DA Talmage; citation_author=M Blumenfeld; citation_journal_title=Mol Cell Biol; citation_volume=7; citation_number=36; citation_pages=4118-4121; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Characterization of the mitotic specific phosphorylation site of histone H1. Absence of a consensus sequence for the p34cdc2/cyclin B kinase.; citation_author=LR Gurley; citation_author=JG Valdez; citation_author=JS Buchanan; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=37; citation_pages=27653-27660; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Histone modification in early and late Drosophila embryos.; citation_author=V Giancotti; citation_author=E Russo; citation_author=F de Cristini; citation_author=G Graziosi; citation_author=F Micali; citation_journal_title=Biochem J; citation_volume=218; citation_number=38; citation_pages=321-329; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Changes in the pattern of histone H1 phosphorylation during Drosophila hydei embryogenesis.; citation_author=NT Harisanova; citation_author=KH Ralchev; citation_journal_title=Mol Biol Rep; citation_volume=11; citation_number=39; citation_pages=199-203; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Human SirT1 interacts with histone H1 and promotes formation of facultative heterochromatin.; citation_author=A Vaquero; citation_author=M Scher; citation_author=D Lee; citation_author=H Erdjument-Bromage; citation_author=P Tempst; citation_journal_title=Mol Cell; citation_volume=16; citation_number=40; citation_pages=93-105; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Histone H4-lysine 20 monomethylation is increased in promoter and coding regions of active genes and correlates with hyperacetylation.; citation_author=H Talasz; citation_author=HH Lindner; citation_author=B Sarg; citation_author=W Helliger; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=41; citation_pages=38814-38822; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Hyperphosphorylation of histone H2A.X and dephosphorylation of histone H1 subtypes in the course of apoptosis.; citation_author=H Talasz; citation_author=W Helliger; citation_author=B Sarg; citation_author=PL Debbage; citation_author=B Puschendorf; citation_journal_title=Cell Death Differ; citation_volume=9; citation_number=42; citation_pages=27-39; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Chemical derivatization of phosphoserine and phosphothreonine containing peptides to increase sensitivity for MALDI-based analysis and for selectivity of MS/MS analysis.; citation_author=G Arrigoni; citation_author=S Resjo; citation_author=F Levander; citation_author=R Nilsson; citation_author=E Degerman; citation_journal_title=Proteomics; citation_volume=6; citation_number=43; citation_pages=757-766; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001553" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Fine Mapping of Posttranslational Modifications of the Linker Histone H1 from Drosophila melanogaster"/>
    <meta name="twitter:description" content="The linker histone H1 binds to the DNA in between adjacent nucleosomes and contributes to chromatin organization and transcriptional control. It is known that H1 carries diverse posttranslational modifications (PTMs), including phosphorylation, lysine methylation and ADP-ribosylation. Their biological functions, however, remain largely unclear. This is in part due to the fact that most of the studies have been performed in organisms that have several H1 variants, which complicates the analyses. We have chosen Drosophila melanogaster, a model organism, which has a single H1 variant, to approach the study of the role of H1 PTMs during embryonic development. Mass spectrometry mapping of the entire sequence of the protein showed phosphorylation only in the ten N-terminal amino acids, mostly at S10. For the first time, changes in the PTMs of a linker H1 during the development of a multicellular organism are reported. The abundance of H1 monophosphorylated at S10 decreases as the embryos age, which suggests that this PTM is related to cell cycle progression and/or cell differentiation. Additionally, we have found a polymorphism in the protein sequence that can be mistaken with lysine methylation if the analysis is not rigorous."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001553.g005"/>

  <meta property="og:title" content="Fine Mapping of Posttranslational Modifications of the Linker Histone H1 from Drosophila melanogaster" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001553" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1175'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9707'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1985&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001553">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001553" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001553&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Fine Mapping of Posttranslational Modifications of the Linker Histone H1 from <i>Drosophila melanogaster</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ana Villar-Garea, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Munich Center for Integrated Protein Science CIPSM, Histone Modifications Group, Adolf-Butenandt Institute, Ludwig-Maximilians University of Munich, Munich, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Axel Imhof
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:Imhof@lmu.de">Imhof@lmu.de</a></p>

                <p>Affiliation:
                  Munich Center for Integrated Protein Science CIPSM, Histone Modifications Group, Adolf-Butenandt Institute, Ludwig-Maximilians University of Munich, Munich, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 06, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001553</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001553" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001553" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001553" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001553" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001553-g001" data-doi="info:doi/10.1371/journal.pone.0001553" data-uri="info:doi/10.1371/journal.pone.0001553.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001553-g002" data-doi="info:doi/10.1371/journal.pone.0001553" data-uri="info:doi/10.1371/journal.pone.0001553.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001553-g003" data-doi="info:doi/10.1371/journal.pone.0001553" data-uri="info:doi/10.1371/journal.pone.0001553.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001553-g004" data-doi="info:doi/10.1371/journal.pone.0001553" data-uri="info:doi/10.1371/journal.pone.0001553.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001553-g005" data-doi="info:doi/10.1371/journal.pone.0001553" data-uri="info:doi/10.1371/journal.pone.0001553.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001553">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001553" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The linker histone H1 binds to the DNA in between adjacent nucleosomes and contributes to chromatin organization and transcriptional control. It is known that H1 carries diverse posttranslational modifications (PTMs), including phosphorylation, lysine methylation and ADP-ribosylation. Their biological functions, however, remain largely unclear. This is in part due to the fact that most of the studies have been performed in organisms that have several H1 variants, which complicates the analyses. We have chosen <em>Drosophila melanogaster</em>, a model organism, which has a single H1 variant, to approach the study of the role of H1 PTMs during embryonic development. Mass spectrometry mapping of the entire sequence of the protein showed phosphorylation only in the ten N-terminal amino acids, mostly at S10. For the first time, changes in the PTMs of a linker H1 during the development of a multicellular organism are reported. The abundance of H1 monophosphorylated at S10 decreases as the embryos age, which suggests that this PTM is related to cell cycle progression and/or cell differentiation. Additionally, we have found a polymorphism in the protein sequence that can be mistaken with lysine methylation if the analysis is not rigorous.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Villar-Garea A, Imhof A (2008) Fine Mapping of Posttranslational Modifications of the Linker Histone H1 from <em>Drosophila melanogaster</em>. PLoS ONE 3(2):
          e1553.
            doi:10.1371/journal.pone.0001553</p><p><strong>Academic Editor: </strong>Simon Williams, Texas Tech University Health Sciences Center, United States of America</p><p><strong>Received:</strong> November 18, 2007; <strong>Accepted:</strong> January 10, 2008; <strong>Published:</strong> February 6, 2008</p><p><strong>Copyright:</strong> © 2008 Villar-Garea, Imhof. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The work was funded by grants from a European Union (LSHG-CT-2006-037415) and the German research council (IM23/4-3).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Histone H1 is a linker histone, as it binds to the DNA that resides between two adjacent nucleosome particles (for review see <a href="#pone.0001553-Kasinsky1">[1]</a>–<a href="#pone.0001553-Bustin1">[4]</a>). The binding of H1 to nucleosomes is involved in the stabilization of the 30 nm chromatin fiber <a href="#pone.0001553-Fan1">[5]</a>, <a href="#pone.0001553-Hansen1">[6]</a>, affects the structure and positioning of nucleosomes <a href="#pone.0001553-Pruss1">[7]</a>, <a href="#pone.0001553-Ura1">[8]</a> and influences the accessibility of nucleosomes to other proteins <a href="#pone.0001553-Bresnick1">[9]</a>, <a href="#pone.0001553-Koop1">[10]</a>. As result, H1 contributes to chromatin organization, transcriptional control and is a key factor of cellular differentiation during early embryonic development <a href="#pone.0001553-Steinbach1">[11]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Histone H1 consists of a conserved globular domain, which is sufficient for the binding of H1 to nucleosomes <a href="#pone.0001553-Draves1">[12]</a>–<a href="#pone.0001553-Vermaak1">[14]</a>, and highly divergent N- and C-terminal extensions (tails), which are rich in lysine, alanine and proline <a href="#pone.0001553-Ponte1">[15]</a>. The tails play important roles in the H1 induced chromatin condensation <a href="#pone.0001553-Bharath1">[16]</a> and the binding to specific <em>loci</em> <a href="#pone.0001553-Izaurralde1">[17]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>It is known that H1 from various organisms is posttranslationally modified at several positions (PTMs), including phosphorylation <a href="#pone.0001553-Dou1">[18]</a>–<a href="#pone.0001553-Thng1">[22]</a> and poly(ADP-ribosylation) <a href="#pone.0001553-Zlatanova1">[23]</a>. More recently, some mammalian isoforms have been shown to be acetylated, methylated and formylated at lysine residues <a href="#pone.0001553-Garcia1">[19]</a>, <a href="#pone.0001553-Kuzmichev1">[24]</a>–<a href="#pone.0001553-Jiang1">[27]</a>. The only modification that has so far been associated with particular cellular functions is the phosphorylation of serines <a href="#pone.0001553-Horn1">[28]</a>–<a href="#pone.0001553-Hale1">[30]</a> and to a lesser degree the methylation of lysines <a href="#pone.0001553-Kuzmichev1">[24]</a>, <a href="#pone.0001553-Daujat1">[25]</a>. However, many studies addressing the biological relevance of the PTMs have been performed in organisms that have more than one H1 variant, which complicates the design and the interpretation of the experiments. In order to study the linker histone function during early murine embryonic development, three isoforms had to be knocked out due to the high level of redundancy <a href="#pone.0001553-Fan1">[5]</a>, <a href="#pone.0001553-Fan2">[31]</a>. Moreover, the amino acid composition of H1, in particular the abundance of lysine residues, challenges the analysis of the protein with techniques using specific proteases like mass spectrometry.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In order to better study the importance as well as the evolutionary conservation of H1 PTMs, we decided to analyze H1 from <em>Drosophila melanogaster</em> embryos by mass spectrometry. This model organism has several advantages with regards to the analysis of H1 modifications. It has a single variant of H1, the embryonic development is well characterized and it is possible to collect a sufficient amount of embryos in a relatively easy way. In <em>Drosophila</em>, H1 is first detected at the cycle 10, after zygotic transcription starts <a href="#pone.0001553-Anderson1">[32]</a>, when it replaces the HMG-D protein that compensates for the lack of H1 during early cleavage states <a href="#pone.0001553-Ner1">[33]</a>, <a href="#pone.0001553-Ner2">[34]</a>. Previous studies have shown that H1 in <em>D. melanogaster</em> is phosphorylated <a href="#pone.0001553-Blumenfeld1">[35]</a> in its N-terminal region <a href="#pone.0001553-Talmage1">[36]</a>, however, the exact position of the phosphorylated sites has not yet been described. Moreover, it has not been determined if H1 in this organism carries other PTMs besides phosphorylation. In contrast to other techniques, mass spectrometry neither requires the assumption of the existence or absence of a certain modification, like the antibody-based techniques, nor the presence of a certain number of electric charges on it, like the electrophoresis.</p>

<h4>Analysis of Histone H1 in 0–12 h embryos</h4>
<a id="article1.body1.sec1.sec1.p1" name="article1.body1.sec1.sec1.p1"></a><p>To develop a robust protocol that allows a parallel analysis of several samples of H1 from Drosophila embryos from different stages, we initially used embryos collected between 0 to 12 h after egg laying (a.e.l.). One of the major inconveniences when working with early embryos is the enormous amount of yolk present in the protein extracts. To circumvent this problem, nuclei are separated from the bulk of yolk proteins and subsequently extracted with perchloric acid. The resulting extract is highly enriched in histone H1 and HMG-D. After dialysis and freeze-drying, histone H1 can be easily purified by RP-HPLC. The use of HPLC for protein separation has a clear advantage over the use of SDS-PAGE: the isolated proteins remain in solution, and many proteases employed for MS analysis do not cleave efficiently when the substrate is embedded in a gel piece.</p>
<a id="article1.body1.sec1.sec1.p2" name="article1.body1.sec1.sec1.p2"></a><p>Purified H1 was digested with the endoprotease AspN, which hydrolyzes the peptide bond N-terminal of aspartic acid, and MALDI-TOF spectra were acquired in the linear, positive mode. As shown in <a href="#pone-0001553-g001">Fig. 1</a>, the resulting peptide mixture covers the entire sequence of the protein. Only the peaks corresponding to the N- and the C-terminus of H1 in the spectrum show additional signals that could be explained by PTMs in H1.</p>
<div class="figure" id="pone-0001553-g001"><div class="img"><a name="pone-0001553-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001553" data-uri="info:doi/10.1371/journal.pone.0001553.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001553.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001553.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>MALDI-TOF analysis of H1 from 0–12 h embryos after Asp-N digestion.</span></strong></p><a id="article1.body1.sec1.sec1.fig1.caption1.p1" name="article1.body1.sec1.sec1.fig1.caption1.p1"></a><p>H1 from 0–12 h embryos was purified and digested with Asp-N. Digestion mixtures were desalted and analyzed by MALDI-TOF mass spectrometry in positive, linear mode. A) A typical spectrum has signals corresponding to all the expected peptides. * labels the signals corresponding to [M+2H]<sup>2+</sup>. B and C) Zooms of the spectrum shown in A encompassing the two regions where signals corresponding to modified peptides are found. B) Peaks of the N-terminus of the protein (5003.5 and 5132.8) and its phosphorylated forms (5083.8, 5213.1, 5294.8). C) C-terminus of the protein (6743.7) and its presumptive methylated form (6756.6). Red: MALDI-TOF, linear positive mode; black: MALDI-TOF, reflector positive mode. D) Assignment of the peaks in A-C. <em>m/z</em>: experimental <em>m/z</em> values, [M+H]<sup>+</sup>: expected <em>m/z</em> values (accession number P02255), Amino acids: amino acids contained in the peptide, Sequence: aminoacid sequence of the corresponding peptide and indication of the presence of PTMs. Note that the aminoacids position are referred to the mature protein, without the first methionine.</p>
<span>doi:10.1371/journal.pone.0001553.g001</span></div><a id="article1.body1.sec1.sec1.p3" name="article1.body1.sec1.sec1.p3"></a><p>In the N-terminus (<a href="#pone-0001553-g001">Fig. 1B</a>), we detect the expected peak for the peptide 2–52 and an additional signal matching with the monophosphorylated form of the same peptide. Due to a missed cleavage of the bond between S1 and D2, peaks corresponding to the acetylated peptide 1–52 and it's mono- and diphosphorylated forms are also detected. Given the low resolution of the spectra in the linear mode and the proximity of the expected signal for diphosphorylated 2–52 (5163.8) to other signals (sodium salt of monoacetylated 1–52 at 5154.8 and a neutral loss of a methylsulfoxide from the the oxidized 1–52p at 5272.7), the existence of diphosphorylated 2–52 cannot be determined with AspN digestion. We tried to acquire the spectra of these digests on the reflector mode, which has higher resolution, however the phosphate groups were unstable in the conditions of the measurement and we could only detect signals corresponding to unmodified peptides or to diverse neutral losses (data not shown).</p>
<a id="article1.body1.sec1.sec1.p4" name="article1.body1.sec1.sec1.p4"></a><p>Because the acetylation can only be seen in the peptide 1–52 and not in 2–52, we conclude that H1 is acetylated at the amino terminus. This conclusion is also supported by digestion with other proteases (trypsine, Glu-C) and by the MS/MS analysis of peptides carrying the N-terminus.</p>
<a id="article1.body1.sec1.sec1.p5" name="article1.body1.sec1.sec1.p5"></a><p>The peak corresponding to a C-terminal peptide (aa 189–255) shows only one additional mass at a m/z value of 6758.3 (<a href="#pone-0001553-g001">Fig. 1C</a>). The resolution of the spectrometer in the linear mode (red trace in <a href="#pone-0001553-g001">Fig. 1C</a>) did not allow us to determine if the second peak differs from 189–255 by 16 (oxidation) or 14 units (one methyl group). However, when a spectrum of the same sample was recorded in the reflector mode (<a href="#pone-0001553-g001">Fig. 1C</a>, black trace), the higher resolution allowed us to determine the difference as being 14.1 amu and could potentially be the result of a monomethylation within the peptide 189–255.</p>


<h4>A polymorphism in the C-terminus of H1 can be mistaken for a monomethylation</h4>
<a id="article1.body1.sec1.sec2.p1" name="article1.body1.sec1.sec2.p1"></a><p>In order to further study the potential methylation, we decided to map the amino acid position(s) in the C-terminus of H1 at which the mass shift is observed. However, the composition of the peptide (22 lysine, 18 alanine, 7 threonine, 6 valine, 5 serine, 5 proline, 2 isoleucine, 1 glycine and 1 aspartic acid residue) and its size (6.7 kDa), made it very difficult to map the site of modification using conventional proteases or tandem mass spectrometry.</p>
<a id="article1.body1.sec1.sec2.p2" name="article1.body1.sec1.sec2.p2"></a><p>Therefore we decided to purify the peptide by RP-HPLC and subsequently perform a limited carboxypeptidase Y digestion on the purified peptides. The reaction products were then analyzed using MALDI-TOF MS. Carboxypeptidase Y removes the C-terminal amino acid of a peptide in a processive manner thereby resulting in a series of peaks that differ by the mass of one aminoacid. However, as the reaction rate is highly dependent on the nature of the removed amino acid, the resulting ladder is not continuous and only the peptides that contain an amino acid at the C-terminus, which is slowly cleaved by carboxypeptidase Y, can be seen.</p>
<a id="article1.body1.sec1.sec2.p3" name="article1.body1.sec1.sec2.p3"></a><p>In our conditions, the peptides corresponding to the peptide 189–255 could be separated from its potentially methylated form by RP-HPLC (<a href="#pone-0001553-g002">Fig. 2A</a>). The carboxypeptidase Y digestions performed in parallel indicate that the modified amino acid resides between the residues 215 and 224, as the signals corresponding to 189–214 and smaller have identical m/z values, whereas the peaks assigned to 189–224 and longer show a mass shift of 14 amu.</p>
<div class="figure" id="pone-0001553-g002"><div class="img"><a name="pone-0001553-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001553" data-uri="info:doi/10.1371/journal.pone.0001553.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001553.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001553.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>A polymorphism in the protein can be misinterpreted for a methylation in the C-terminus.</span></strong></p><a id="article1.body1.sec1.sec2.fig1.caption1.p1" name="article1.body1.sec1.sec2.fig1.caption1.p1"></a><p>A) Separation of the peptides produced by AspN digestion of H1 from 0–12 h embryos over a C18 column. Continous line: absorbance at 214 nm; dashed line: eluent composition expressed as percentage of eluent B. B and C) The purified peptides 189–255 and its methylated form were partially digested with carboxypeptidase Y. The digestion mixtures were desalted and analysed by MALDI-TOF mass spectrometry in the linear, reflector mode. Zooms are shown. In both panels, the upper spectrum corresponds to the unmodified form and the lower, to the modified one. The results indicate that the methylated residue is in the stretch 215–224. B) Region displaying the largest peptides where identical mass in both samples is detected. C) Region displaying the smallest peptides where a mass difference between both samples is detected. For the region 215–224 no signals are present in the spectra. D) H1 was isolated from different sources, digested with Asp-N, desalted and analysed by MALDI-TOF mass spectrometry in the linear, positive mode. Zoom of the region containing 189–255 is shown. Magenta: 2–3 h embryos, green: 12–15 h embryos, grey: 0–12 h embryos, black SL2 cells. The proportion unmethylated (6744.4)/methylated (6758.3) is identical in all the embryonic samples, whereas no methylated species can be detected in the SL2 cells. The peak with a m/z value of 6767.3 corresponds to a sodium adduct of the unmethylated form. E) The existence of the polymorphism V217I could explain our observations. Upper panel: a stretch of the DNA sequence of a H1 gene coding for the Val allele is shown (accession number: NM_165380). The numbers indicate the base position with respect to the transcription start. A SfcI restriction site is underlined. The corresponding translated sequence is shown underneath. The numbers indicate the aminoacid position respect to the mature protein. The long lines represent the fragment of DNA sequence amplified by PCR, the arrows, the SfcI restriction sites and the short lines at the bottom, the expected fragments after SfcI digestion. The lower panel represents the same but for a H1 gene coding for the Ile allele (for instance, accession number: NM_1032208). The shift 688 G-&gt;A results in the lost of a SfcI restriction site. F) The polymorphism Val217Ile is present in the fly population. A stretch of the coding sequence of H1 (606–806) was amplified from fly and SL2 cells genomic DNA. The products were digested with SfcI and the resulting fragments were analysed by polyacrylamide gel electrophoresis. As control, parallel incubations with no enzyme were performed. In the sample from SL2 cells, only the expected bands for the allele Val are observed (60 and 123 bp, the 18 bp bands ran out of the gel), whereas in the sample from the flies, the pattern fits with the existence of both the Val (60 and 123 bp) and the Ile (60 and 141 bp) alleles.</p>
<span>doi:10.1371/journal.pone.0001553.g002</span></div><a id="article1.body1.sec1.sec2.p4" name="article1.body1.sec1.sec2.p4"></a><p>We next wondered whether the ratio of the wild-type peptide and the shifted one change during development. In order to do this, we isolated H1 from embryos at different developmental stages (2–3 h, 3–6 h, 6–9 h, 9–12 h and 12–15h a.e.l.). A quantitative comparison of the relative intensities of the two peaks did not change during embryonic development (<a href="#pone-0001553-g002">Fig. 2D</a>). Therefore we concluded that the potential methylation is not regulated during development. In order to have an independent source of H1 molecules, we also extracted H1 from SL2 cells (cell line of embryonic origin) and analyzed the C-terminal peptides. In contrast to our expectations, H1 did not seem to be methylated at the C-terminus (<a href="#pone-0001553-g002">Fig. 2D</a>). As we also failed to observe a C-terminal methylation in H1 isolated from embryos derived from a different fly strain we wondered if there are alternative explanations for the observed mass shift.</p>
<a id="article1.body1.sec1.sec2.p5" name="article1.body1.sec1.sec2.p5"></a><p>One possible explanation could be the existence of different isoforms that differ by the mass of one methyl group. Indeed, several polymorphisms in the sequence of <em>D. melanogaster</em> H1 are recorded in the DNA and protein databases. One of them, V217I, could explain our observations. This polymorphism corresponds in the DNA sequence with G688A and results in the loss of a SfcI restriction site (<a href="#pone-0001553-g002">Fig. 2E</a>). To determine if this polymorphism is present in our fly population, a 201 bp fragment from H1 coding sequence encompassing the position 688 was amplified and digested with SfcI. DNA from SL2 cells was employed as control. The analysis of the resulting fragments by polyacrylamide gel electrophoresis is shown in <a href="#pone-0001553-g002">Fig. 2F</a>. In the sample from SL2 cells only the products of the digestion of the Val allele are detected (fragments of 60 bp and 123 bp). In contrast, in the sample from the embryos, digestion products of both alleles (fragments of 60 bp and 123 bp for the Val allele, fragments of 60 bp and 141 bp for the Ile allele) are present. Therefore, the signal with a m/z value of 6758 does not correspond to 189–255 with a methylated lysine residue, but to 189–255 with Val 217 replaced by Ile.</p>


<h4>Analysis of H1 phosphorylation in 0–12 h embryos</h4>
<a id="article1.body1.sec1.sec3.p1" name="article1.body1.sec1.sec3.p1"></a><p>As mentioned above, histone H1 in 0–12 h embryos is phosphorylated at the N-terminus. In order to enhance the detection of putative phosphorylated species, MALDI-TOF spectra in the negative, linear mode were acquired from H1 after digestion with AspN. However, even under conditions where phosphopeptides are preferentially ionized not more than 2 phosphate groups can be detected in the peptide 1–52. This is also the only region of the protein appears to be stably phosphorylated (<a href="#pone-0001553-g001">Fig. 1A</a> and <a href="#pone-0001553-g003">Fig. 3A</a>).</p>
<div class="figure" id="pone-0001553-g003"><div class="img"><a name="pone-0001553-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001553" data-uri="info:doi/10.1371/journal.pone.0001553.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001553.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001553.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Phosphorylation in 0–12 h embryos is located in the N-terminal 20 aminoacids.</span></strong></p><a id="article1.body1.sec1.sec3.fig1.caption1.p1" name="article1.body1.sec1.sec3.fig1.caption1.p1"></a><p>H1 from 0–12 h embryos was isolated and digested with AspN (A), trypsin (B) or Glu-C (C, D). Digests were desalted and MALDI-TOF spectra were acquired in the linear (A, C, D) or reflector (B) modes. For each sample, spectra in the positive (upper spectrum on each panel) and negative (lower spectrum on each panel) were recorded. Zooms of the interesting regions are shown. A) In the negative mode, no more phosphorylation states for 1–52 mode are detected, respect to the positive mode. B) After digestion with trypsin, mono- and diphosphorylation are detected in 1–21 and no modification (apart from oxidation) is present in 38–57. C, D) After digestion with Glu-C, no PTM is detected for 21–61 (only oxidation, C), however 1–20 appears clearly un-, mono and diphosphorylated (D).</p>
<span>doi:10.1371/journal.pone.0001553.g003</span></div><a id="article1.body1.sec1.sec3.p2" name="article1.body1.sec1.sec3.p2"></a><p>To narrow down the number of possible modified sites, H1 was digested with other proteases and the resulting digestion products were analyzed by MALDI-TOF in positive and negative mode. Trypsin digests show that a significant proportion of the peptide 1–21 is mono- and diphosphorylated, whereas no modification apart from oxidation is observed in 38–57 (<a href="#pone-0001553-g003">Fig. 3B</a>). Glu-C digestion confirms that no phosphorylation is detected outside 1–20 (<a href="#pone-0001553-g003">Fig. 3C, D</a>). The enrichment of phosphopeptides using TiO<sub>2</sub> columns only led to the isolation of mono- and diphosphorylated peptide 1–21 (tryptic digest) and 1–20 (Glu-C digest, data not shown).</p>
<a id="article1.body1.sec1.sec3.p3" name="article1.body1.sec1.sec3.p3"></a><p>Because the peptide 1–21 has 6 residues (4 Ser and 2 Thr, <a href="#pone-0001553-g004">Fig. 4A</a>) that could be phosphorylated, we analyzed the mono- and diphosphorylated forms by ESI-MS/MS. H1 was extracted from 0–12 h embryos and digested with trypsin. The attempts to sequence 1–21 directly from this sample were unsuccessful due to the instability of the phosphate groups in the measurement conditions. To overcome this problem, the product of the trypsin digest was treated with barium hydroxide to eliminate the phosphate groups and subsequently submitted to a 1,4-addition with 2-mercaptoethanol (<a href="#pone-0001553-g004">Fig 4</a>). The increased stability of the modified peptides allowed mapping of the phosphorylation site by tandem mass spectrometry. In the TOF-MS spectrum only unmodified, mono- and diphosphorylated 1–21 can be observed, together with their sodium adducts (<a href="#pone-0001553-g004">Fig. 4B</a>). As we have discussed earlier, the analysis of all these species proves that the acetylation observed in 1–52 (see <a href="#pone-0001553-g001">Fig. 1D</a>) resides in the amino acids 1–3 and, given the nature of the first 3 aminoacids (SDS) and the fact that the acetylation is not seen in a peptide covering aminoacids 2–52 (<a href="#pone-0001553-g001">Fig. 1B</a>), one can firmly conclude that the N-terminus of the protein is acetylated.</p>
<div class="figure" id="pone-0001553-g004"><div class="img"><a name="pone-0001553-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001553" data-uri="info:doi/10.1371/journal.pone.0001553.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001553.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001553.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>S10 is the major phosphorylation site in H1 isolated from 0–12 h embryos.</span></strong></p><a id="article1.body1.sec1.sec3.fig2.caption1.p1" name="article1.body1.sec1.sec3.fig2.caption1.p1"></a><p>H1 from 0–12 h embryos was purified, digested with trypsin, and submited to β-elimination followed by 1,4-addition of 2-mercaptoethanol. The resulting crude was desalted and the different forms of the peptide 1–21 were sequenced by ESI-MS/MS. A) Scheme showing the peptide 1–21 and the numbering of the b and y ions. The residues susceptible of phosphorylation are highlighted in red. B) TOF-MS spectrum of the sample. The double charged ions corresponding to 1–21, its modified forms and sodium adducts are indicated. Note that, due to the chemical treatment of the sample, the mass difference between the unmodified and the monomodified form is 60 instead of 80. C, D) Zooms of the product ion spectra for the unmodified (lower), monomodified (middle) and dimodified (upper) forms. * labels the ions with one modification; ** labels the ions with two modifications. C) For y11 only no modified ions are detected in both the mono- and dimodified peptide 1–21, which indicates that there is no detectable phosphorylation at T18. Therefore, the presence of monomodified y12 can only be due to phosphorylation at S10. The pattern of the b13 and b14 ions in the spectra of the mono- and dimodified species corroborate this assumption. In the spectrum of the diphosphorylated population, the presence of only monomodified y12 and y13 indicates that one of the two phosphate groups on each molecule is at S10. Additionally, the signal of dimodified y14 reveals the existence of a subpopulation of the peptide 1–21 containing phosphate groups at S10 and S8 simultaneously. Moreover, the higher relative intensity of dimodified vs. monomodified ions y15 and y16 respect to the same ratio for y14 suggests the presence of a species with simultaneous S10 and T7 phosphorylation. D) There is a small proportion of H1monophosphorylated at S1 and/or S3, as revealed by the intensities of b6, b6*, b7 andb7*. In the diphosphorylated population there is also modification at S1 and/or S3, however, simultaneous phosphorylation at S1 and S3 in the same molecule cannot be detected.</p>
<span>doi:10.1371/journal.pone.0001553.g004</span></div><a id="article1.body1.sec1.sec3.p4" name="article1.body1.sec1.sec3.p4"></a><p>Judging from the MS/MS spectra of the differentially modified peptides, we conclude that there is no detectable phosphorylation outside amino acids 1–10 and that most of the phosphorylation is found at position S10 in the singly- as well as the doubly phosphorylated peptide (see <a href="#pone-0001553-g004">Fig. 4C</a>). However, there also seems to be some minor subpopulations in the monophosphorylated H1 that carry the phosphate group at S1 or S3 (see y ions in <a href="#pone-0001553-g004">Fig. 4C</a> and b ions in <a href="#pone-0001553-g004">Fig. 4D</a>). Unfortunately, the absence of b1 and b2 ions makes it impossible to discriminate between S1 and S3 phosphorylation. We can therefore not distinguish if the monomodified b6 ion comes from a subpopulation of H1 monophosphorylated at S1 or from H1 monophosphorylated at S3 or if both subpopulations are present in the sample. The use of different collision energy values during acquisition did not improve the results. The same is true in case of the phosphorylation at T7 and/or S8 in the diphosphorylated peptide (<a href="#pone-0001553-g004">Fig. 4C</a>). The y ions indicate that there is some extent of phosphorylation at S8 and perhaps as well at T7, but the intensity of these ions and the corresponding b ions is too low to make any strong conclusion about the status of T7.</p>
<a id="article1.body1.sec1.sec3.p5" name="article1.body1.sec1.sec3.p5"></a><p>In summary at least two subpopulations of diphosphorylated H1 can be distinguished:</p>


<ol class="order">

<li>i) one of the phosphate groups is at S10 and the other at S1/S3; However, as mentioned above, we could not distinguish between S1 and S3 modification, but from the b ions (b6, b7 in <a href="#pone-0001553-g004">Fig. 4D</a>) it is clear that at least one of these two combinations (i.e., diphosphorylation at S10 and S1 or diphosphorylation at S10 and S3) is present in our sample.</li>

<li>ii) one of the phosphate groups is at S10 and the other at T7/S8. The presence of a dimodified y14 together with the absence of dimodified y12 and y13 indicates the existence of a phosphate group at S8. The apparent higher dimodified vs. monomodified ratio of y15 and y16 when compared to y14 suggests that there is also part of the population that is diphosphorylated at S10 and T7.</li>

</ol><a id="article1.body1.sec1.sec3.p6" name="article1.body1.sec1.sec3.p6"></a><p>The analysis of the sodium adducts of the mono- and diphosphorylated forms yield the same result.</p>


<h4>Analysis of H1 phosphorylation in staged embryos</h4>
<a id="article1.body1.sec1.sec4.p1" name="article1.body1.sec1.sec4.p1"></a><p>Once known that H1 in <em>D. melanogaster</em> embryos is phosphorylated at different positions in the first 10 amino acids, we addressed whether the proportion of phosphorylated protein and/or the positions that are modified change during embryonic development. We extracted H1 from embryos 2–3 h, 3–6 h, 6–9 h, 9–12 h and 12–15 h a.e.l., and analyzed the samples the same way as H1 from 0–12 h embryos. We have not analysed 0–2 h embryos because the synthesis of H1 begins with the cellularization i.e., approximately 2 h after egg laying. Indeed, in all our preparations, we got notably less H1 from the 2–3 h embryos than from the rest of the stages, even though the nuclear pellets were similar in size.</p>
<a id="article1.body1.sec1.sec4.p2" name="article1.body1.sec1.sec4.p2"></a><p>The relative intensities of the MALDI-TOF spectra indicate a decrease in the proportion of monophosphorylated protein in the oldest embryos, independently of the protease (AspN or trypsin) used and the mode of spectra acquisition (linear or reflector, positive or negative). An example is shown in <a href="#pone-0001553-g005">Fig. 5A</a>. Integration of the area of the signals for unmodified (proton and sodium adducts) and monophosphorylated 1–21 clearly points out the loss of modified H1 (<a href="#pone-0001553-g005">Fig. 5B</a>). Due to the low intensity of the signals, it was not possible to conclude if there were also changes in the proportion of diphosphorylated peptide.</p>
<div class="figure" id="pone-0001553-g005"><div class="img"><a name="pone-0001553-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001553" data-uri="info:doi/10.1371/journal.pone.0001553.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001553.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001553.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001553.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001553.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001553.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The monophosphorylation at S10 decreases as the embryos maturate.</span></strong></p><a id="article1.body1.sec1.sec4.fig1.caption1.p1" name="article1.body1.sec1.sec4.fig1.caption1.p1"></a><p>Histone H1 from 2–3 h, 3–6 h, 6–9 h, 9–12 h and 12–15 h was isolated and digested with trypsin. A) Zoom of two MALDI-TOF spectra acquired in the reflector positive mode are shown as examples. The upper spectrum corresponds to the 12–15 h old embryos, the lower to the 3–6 h ones. Note the different intensity of the signal at 2078. B) To compare the proportion of monophosphorylated H1 between the different stages, the area of the clusters corresponding to unmodified 1–21 (both proton and sodium adducts) and the monophosphorylated form (only proton adduct, sodium adduct not detected) was calculated for H1 from staged embryos. For each sample, the relative area of the monophosphorylated species (A<sub>rel</sub>) was calculated by division of the area of the monophosphorylated species through the total area (unmodified+monophosphorylated). The results of two independent biological replicates are shown. The error bars indicate the maximum and the minimum obtained values. C) Scheme showing the peptide 1–21 and the numbering of the b and y ions. The residues susceptible of phosphorylation are highlighted in red. D and E) H1 from 3–6 h and 12–15 h old embryos was purified, digested with trypsin, submitted to chemical modification (β-elimination and Michael addition) and finally, the forms of 1–21 were sequenced by ESI-MS/MS. Zooms of the product ions of the monomodified species are shown. * labels the modified species. D) The relative intensities of y12 and y12* indicate that the 12–15 h old embryos have less proportion of S10 phosphorylation than the younger embryos. E) The relative intensities of the b and b* ions suggest that monophosphorylated H1 from 12–15 h old embryos contains more proportion of S1 and/or S3 phosphorylation than the sample from the younger embryos.</p>
<span>doi:10.1371/journal.pone.0001553.g005</span></div><a id="article1.body1.sec1.sec4.p3" name="article1.body1.sec1.sec4.p3"></a><p>To determine if the loss of phosphorylation occurs selectively at a certain position, we determined the site of modification by ESI-MS/MS 3–6 h and 12–15 h embryos. The product ion spectra of the ion containing the monophosphorylated H1 peptide in both samples show noticeable changes in the relative intensities of unmodified vs. modified y12 (<a href="#pone-0001553-g005">Fig. 5D</a>) demonstrating that the major site of monophosphorylation in the younger embryos is S10, whereas in the older embryos the proportion of monophosphorylation occurring at S10 is smaller. Besides the main phosphorylation site at S10 we also find a smaller fraction of the monophosphorylated peptide to be modified at position S1 or S3 as indicated by the existence of an unmodified y12 ion (y12 <a href="#pone-0001553-g005">Fig 5D</a>) and a modified b6 ion ( b6*, <a href="#pone-0001553-g005">Fig 5E</a>). Interestingly the relative intensity of the modified versus unmodified b6 ion is higher in older embryos (b6*/b6 = 0.21±0.02) compare to younger ones (b6*/b6 = 0.13±0.03) suggesting that the decrease in monophosphorylation is due to a decrease in S10 phosphorylation whereas the phosphorylation at the N-terminus remains unchanged.</p>
<a id="article1.body1.sec1.sec4.p4" name="article1.body1.sec1.sec4.p4"></a><p>In summary, during the first hours of development, the proportion of H1 phosphorylated at S10 is decreasing.</p>

</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We present here a detailed analysis of PTMs in H1 from <em>D. melanogaster</em> embryos using mass spectrometry. The results show that the single most prominent PTM in these samples is a phosphorylation at S10. Other groups had already reported that phosphorylation in <em>D. melanogaster</em> H1 is found in the N-terminal region of the protein, but the exact amino acids, which carry the modification remained unidentified <a href="#pone.0001553-Talmage1">[36]</a>. There are several interesting features that distinguish H1 phosphorylation in <em>Drosophila</em> from H1 phosphorylation in mammals. First, in mammalian cells, H1 variants with 4, 5 or even 6 phosphate groups are detected <a href="#pone.0001553-Sarg1">[20]</a>, <a href="#pone.0001553-Gurley1">[37]</a>, whereas in <em>Drosophila</em> species we and others only detect mono- <a href="#pone.0001553-Talmage1">[36]</a>, <a href="#pone.0001553-Giancotti1">[38]</a> and diphosphorylated H1 <a href="#pone.0001553-Harisanova1">[39]</a>. Second, many mammalian H1 proteins have phosphorylation sites are in the C-terminus, which are often found to be consensus CDK-phosphorylation sites <a href="#pone.0001553-Sarg1">[20]</a>, <a href="#pone.0001553-Wisniewski1">[26]</a>. In contrast, <em>D. melanogaster</em> H1 lacks these (S/T)PXZ (where X is any amino acid and Z a basic amino acid) sequences and no phosphorylation at the C-terminal region has been detected so far. Because in mammalian cells the levels of H1 phosphorylation increase as the cell cycle progresses from G1 to mitosis, it has been suggested that CDK kinases are responsible for the phosphorylation of H1. However, research performed in different laboratories has shown that, at least in mammalian H1, the mitosis specific phosphorylation occurs in the N-tail of the protein, in sites that do not fit with the (S/T)PXZ sequence <a href="#pone.0001553-Sarg1">[20]</a>, <a href="#pone.0001553-Gurley1">[37]</a>. On the one hand, in chinese hamster ovary cells, the mitosis specific phosphorylation is found in S1 and T3 <a href="#pone.0001553-Gurley1">[37]</a>, on the other hand, in human H1.5, it is found in T10. Despite a general lack of interspecies homology within the tail regions of H1 molecules, the extreme N-terminus of most H1 molecules is reasonably well conserved. Based on this similarity and on the observations discussed above we suggest that the phosphorylation we observe is mitosis dependent. This would also explain the reduction of the proportion of monophosphorylated H1 as the embryos get older and the number of mitotically active cells decreases.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>Together with phosphorylation, lysine acetylation, methylation and formylation have been found in mammalian H1, but very little about their biological function is known <a href="#pone.0001553-Garcia1">[19]</a>, <a href="#pone.0001553-Kuzmichev1">[24]</a>–<a href="#pone.0001553-Jiang1">[27]</a>, <a href="#pone.0001553-Vaquero1">[40]</a>. In our study we have not detected any PTM besides phosphorylation. Two reasons may explain this fact: First, we have employed a different model organism, in which H1 is present as a single variant and which already differs with the other organisms in the position of the major phosphorylation sites. Second, we cannot rule out the existence of a minor proportion of H1 carrying any of these PTMs in <em>Drosophila</em>, whose detection would require the use of different analytical techniques or additional enrichment steps.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>Finally, we would like to point out that the existence of polymorphisms in the protein sequence can lead to erroneous interpretations. Diverse couples of amino acids have a similar chemical structure that differs only in the presence of a methyl group. This is the case for glycine and alanine, alanine and valine, valine and leucine or isoleucine, aspartic and glutamic acid, asparagine and glutamine, serine and threonine. Thus, if the peptides are not fully sequenced with all fragment ions visible, it is not possible to assess if the observed 14 units mass difference is due to a methyl group modifying a lysine or arginine residue or to a polymorphism in the protein sequence. Additionally, the mass difference between lysine and arginine is very similar to the mass difference between an unmodified and dimethylated lysine and can only be distinguished with mass spectrometers of very high resolution. Existence of these kind of substitutions in human H1.2 and H1.4 have already been reported <a href="#pone.0001553-Sarg1">[20]</a>.</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>In summary, we have analyzed the entire sequence of <em>D. melanogaster</em> H1 by mass spectrometry. Apart from the N-terminal acetylation, we have only found mono- and diphosphorylation in the first 10 aminoacids. The major site of phosphorylation is S10, however minor phosphorylation sites can also be found at S1 or 3 and at T7 or S8. The proportion of monophosphorylated H1 decreases as the embryos mature, mostly due to a decrease in the phosphorylation at S10. This fact together with previous findings about mammalian H1, suggest that this modification is regulated during mitosis.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Histone H1 isolation from embryos</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Embryos were washed from the agar plates with tap water and dechorionated by stirring for 1 min in diluted bleach solution (23 ml bleach 6–14% active chlorine in 100 ml final volume), rinsed with 1 l 0.12 M NaCl containing 0.04% Triton X-100 (v/v) and washed thoroughly with tap water. 4–5 g dechorionated embryos were dehydrated by immersion in 70% ethanol during 5 min at room temperature, decanted and stored at −80°C.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Nuclei from the frozen embryos were prepared in a similar way as described before <a href="#pone.0001553-Giancotti1">[38]</a>. Briefly, 4–5 g frozen embryos were thawed at 4°C in 8–10 ml buffer 1 (0.35 M sucrose, 15 mM HEPES pH = 7.6, 10 mM KCl, 5 mM MgCl<sub>2</sub>, 0.5 mM EGTA pH = 8.0, 0.1 mM EDTA pH = 8.0, 10 mM glycerolphosphate, 10 mM sodium butyrate, 10 mM NaF, 1 mM DTT, 0.2 mM PMSF, 1 mM Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>). The suspension was passed twice through a Yamato LH-31 homogenizer at 1000 rpm and the homogenate was filtered through a layer of Miracloth. Filtrates were transferred to 50 ml tubes, diluted with buffer 1 up to 5 ml/g embryo and centrifuged (3000 g, 4°C, 20 min). Supernatants were decanted and the nuclear pellet was washed twice with buffer 1 supplemented with 0.1% NP-40 (wash buffer), first employing 5 ml wash buffer/g embryo and next with 2 ml wash buffer/g embryo.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Nuclei were transferred to 1.5 ml tubes and extracted with diluted HClO<sub>4</sub> as described previously <a href="#pone.0001553-Talasz1">[21]</a>, <a href="#pone.0001553-Talasz2">[41]</a>, <a href="#pone.0001553-Talasz3">[42]</a>. Briefly, nuclei were resuspended in 3 vols 6.7% HClO<sub>4</sub> and rotated at 4°C 1 h. After centrifugation (16000 g, 5 min, 4°C), supernatants were kept in clean tubes whereas pellets were reextracted with 1 ml 5% HClO<sub>4</sub>. The HClO<sub>4</sub> extracts were dialysed against 0.1 M acetic acid containing 1 mM DTT (membrane MWCO 6000–8000, 2×0.5 l, 1×1l), lyophilyzed, reconstituted in 20 mM Tris pH = 6.8, 1mM DTT and stored at −20°C. The quality of the preparation was checked by SDS-PAGE. In none of the second HClO<sub>4 </sub>extracts H1 could be detected.</p>


<h4>Histone H1 isolation from SL2 cells</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>SL2 cells were grown at 26°C in Schneideŕs <em>Drosophila</em> medium with L-glutamine supplemented with 10% FBS, 100 U/ml penicilin and 100 µg/ml streptomycin. Cells were harvested by centrifugation (500 g, 10 min, room temperature) and directly resuspended in 3 vols 6.6 % HClO<sub>4</sub>. The rest of the procedure is identical to that one employed for the embryos.</p>


<h4>Histone H1 purification</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>H1 was purified by HPLC over a Jupiter C4 (nucleosil 5 µm, 300 Å 150×1 mm) column (Phenomenex) with 0.05% TFA as eluent A and 0.065% TFA/84% acetonitrile as eluent B. Elution was performed with a 3-steps gradient (31%B during 2 min, 31%B to 37% B in 10 min, 37% B to 45% B in 30 min) at 50 µl/min flow rate and absorbance at 214 nm was monitored. 50 µl fractions were collected. Before proceeding to the next sample, column was washed with 100% B during 10 min and reequilibrated with 31% B. For each run, fractions containing H1 were pooled, concentrated to dryness (speed-vac), reconstituted in 10–30 µl 0.1% TFA and stored at −20°C.</p>


<h4>Histone H1 protease digestion</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>HPLC-purified H1 (amount corresponding to 30 mAU/min in the chromatogram) was digested in a reaction volume of 20 µl overnight at 25°C with 80 ng AspN (Roche) in 0.1 M Tris pH = 8.5; or at 25°C with 50 ng GluC (Roche) in 25 mM NH<sub>4</sub>HCO<sub>3</sub> or at 37°C with 0.4 µg trypsin (Promega) in 50 mM NH<sub>4</sub>HCO<sub>3</sub>. Except otherwise stated, reaction crudes were quenched with 2 µl TFA.</p>


<h4>MALDI-TOF MS analysis</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>3 µl each quenched digestion crude were diluted with 7 µl 0.1% TFA and desalted with ZipTip µ-C18 (Millipore) according manufactureŕs instructions. Samples were eluted from the resin with 2 µl 50% acetonitrile/0.1% TFA, mixed with an identical volume of matrix (50% acetonitrile/0.3% TFA, saturated of α-cyano-hydroxycinnamic acid) and spotted immediately onto a stainless steel MALDI-TOF target plate. Spectra were acquired on a Voyager DE STR workstation (Applied Biosystems).</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>Integration of the signals corresponding to each peptide (isotopic cluster area) was automatically performed by the Data Explorer software (Applied Biosystems) excluding all the peaks whose signal-to-noise ratio was smaller than 4.</p>


<h4>Carboxypeptidase Y analysis of the peptide 189–255</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Quenched AspN digestions were pooled to get approximately 300 µl final volume and concentrated to ca. 80 µl. Formic acid was added (final concentration: 0.1%) and 40 µl were fractionated on a Gemini C18 column run at 0.2 ml/min. Eluent A was 0.05% TFA and eluent B 0.065% TFA/84 % acetonitrile; gradient from 4% B to 25% B in 60 min. 0.1 ml fractions were collected. Absorbance at 214 nm was monitored and 2 µl each fraction were used to analyze its composition by MALDI-TOF MS in positive, linear mode.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>Fractions corresponding to the unmethylated and monomethylated forms of 189–255 were collected, concentrated to dryness and reconstituted in 1 µl reaction mix (50 mM Tris pH = 6.8 and 40 ng/µl carboxypeptidase Y). 5 µl each reaction were quenched with 5 µl 2.5 % TFA after 0.5 h incubation and the rest, after 2.5 h. Quenched digestion mixtures were directly desalted and analyzed by MALDI-TOF as indicated above.</p>


<h4>Beta-elimination and Michael addition</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>For each sample, 1-2 tryptic digests (not quenched) were pooled and divided in 5 µl aliquots. Chemical modification was performed as described by Arrigoni et al. <a href="#pone.0001553-Arrigoni1">[43]</a> with some modifications. Each aliquot was mixed with 20 µl 0.1 M Ba(OH)<sub>2</sub> and incubated at 50°C for 30 min. Immediately 2.6 µl 0.1 M beta-mercaptoethanol were added to each crude and the mixtures were kept at 50°C for further 30 min. Each aliquot was quenched with 5 µl 10% TFA, aliquots were pooled and desalted with Perfect Pure C18 Tips (Eppendorf) according manufactureŕs instructions, replacing TFA for formic acid in all the buffers. Samples were eluted in 3–10 µl 50% MeOH/0.1% formic acid and analysed by ESI-MS/MS.</p>


<h4>ESI-MS/MS analysis</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>Tandem mass spectrometry analyses were performed on a Qstar XL hybrid quadrupole TOF spectrometer (Applied Biosystems). For sample ionization, either a nanospray ion source (Protana) or a Nanomate (Advion) were employed. For each spectrum, measurements during 5 min were accumulated. Acquisition was performed in the positive mode and the collision energy was manually adjusted during the measurements, using the same values during the same time for all the samples.</p>
<a id="article1.body1.sec3.sec8.p2" name="article1.body1.sec3.sec8.p2"></a><p>Spectra were manually interpreted.</p>


<h4>DNA extraction</h4>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1"></a><p>Nuclei from 0.6 g <em>D. melanogaster</em> dechorionated embryos were prepared as described above. Nuclei from SL2 cells were prepared as follows: about 50×10<sup>6</sup> cells were harvested (centrifugation at 500 g during 10 min at RT) and resuspended in 1.0 ml RSB (10 mM Tris pH = 7.5, 10 mM NaCl, 3 mM MgCl<sub>2</sub>). 0.1 ml 10% NP-40 was added and the suspension was incubated on ice during 10 min. After centrifugation (2000 g, 4°C, 5 min) the nuclear pellet was washed once with 1 ml RSB.</p>
<a id="article1.body1.sec3.sec9.p2" name="article1.body1.sec3.sec9.p2"></a><p>Nuclei from SL2 cells or from embryos were lysated in 0.5 ml proteinase K buffer (50 mM Tris pH = 8.5, 10 mM EDTA pH = 8.0, 0.1 M NaCl and 1% SDS). 20 µg RNAse A were added to each sample, followed by 30 min incubation at 30°C. Then, 40 µg proteinase K were added and digestion was allowed to proceed at 37°C overnight. After a wash with 1 vol phenol (saturated with TE) and another wash with phenol/chloroform/isoamylalcohol (50/49/1), DNA was precipitated with 1.5 ml absolute ethanol in the presence of 50 µl 3 M sodium acetate. DNA was pelleted (centrifugation at 16000 g, 2 min, r.t.), washed twice with 70% ethanol, reconstituted in water and stored at −20°C.</p>


<h4>PCR</h4>
<a id="article1.body1.sec3.sec10.p1" name="article1.body1.sec3.sec10.p1"></a><p>0.1 µg genomic DNA was amplified with T4 Taq Polymerase (NEB) according manufactureŕs instructions in a 50 µl reaction containing 1.0 µM each primer (forward: <span class="gene">GGATGCCAAGAAAACTGGA</span>, reverse: <span class="gene">TTTTTGGCAGCCGTAGTCTT</span>). 35 cycles were performed with 60°C annealing temperature. The Quality of the product was checked by agarose gel electrophoresis with ethidium bromide staining.</p>


<h4>Restriction enzyme digestion</h4>
<a id="article1.body1.sec3.sec11.p1" name="article1.body1.sec3.sec11.p1"></a><p>0.5 µg PCR product (purified with the Illumina GFX PCR DNA and gel band purification kit, Amersham, according manufactureŕs instructions) were digested with 5 U SfcI (NEB) in 50 µl reaction, according manufactureŕs instructions. To reach the complete digestion, after 1.5 h incubation at 37°C, 5 U SfcI more were added to each reaction and the incubation was prolonged overnight at 30°C. Reaction products were analyzed on an 8% acrylamide gel with ethidium bromide staining.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank all members of the Imhof lab for critical comments on the manuscript and H1 containing nuclear extracts. We would furthermore like to thank Lars Israel, Andreas Hochheimer and Tilman Schlunck for expert technical assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AI AV. Performed the experiments: AV. Analyzed the data: AI AV. Wrote the paper: AI.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001553-Kasinsky1" id="pone.0001553-Kasinsky1"></a>Kasinsky HE, Lewis JD, Dacks JB, Ausio J (2001) Origin of H1 linker histones. Faseb J  15: 34–42.  <ul class="find" data-citedArticleID="1051530" data-doi="10.1096/fj.00-0237rev"><li><a href="http://dx.doi.org/10.1096/fj.00-0237rev" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+of+H1+linker+histones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+of+H1+linker+histones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001553-Wolffe1" id="pone.0001553-Wolffe1"></a>Wolffe AP (1997) Histone H1. Int J Biochem Cell Biol  29: 1463–1466.  <ul class="find" data-citedArticleID="1051568" data-doi="10.1016/s1357-2725(97)00026-5"><li><a href="http://dx.doi.org/10.1016/s1357-2725(97)00026-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+H1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+H1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001553-Woodcock1" id="pone.0001553-Woodcock1"></a>Woodcock CL, Skoultchi AI, Fan Y (2006) Role of linker histone in chromatin structure and function: H1 stoichiometry and nucleosome repeat length. Chromosome Res  14: 17–25.  <ul class="find" data-citedArticleID="1051570" data-doi="10.1007/s10577-005-1024-3"><li><a href="http://dx.doi.org/10.1007/s10577-005-1024-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+linker+histone+in+chromatin+structure+and+function%3A+H1+stoichiometry+and+nucleosome+repeat+length." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+linker+histone+in+chromatin+structure+and+function%3A+H1+stoichiometry+and+nucleosome+repeat+length.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001553-Bustin1" id="pone.0001553-Bustin1"></a>Bustin M, Catez F, Lim JH (2005) The dynamics of histone H1 function in chromatin. Mol Cell  17: 617–620.  <ul class="find" data-citedArticleID="1051498" data-doi="10.1016/j.molcel.2005.02.019"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.02.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dynamics+of+histone+H1+function+in+chromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dynamics+of+histone+H1+function+in+chromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001553-Fan1" id="pone.0001553-Fan1"></a>Fan Y, Nikitina T, Morin-Kensicki EM, Zhao J, Magnuson TR, et al.  (2003) H1 linker histones are essential for mouse development and affect nucleosome spacing in vivo. Mol Cell Biol  23: 4559–4572.  <ul class="find" data-citedArticleID="1051508" data-doi="10.1128/mcb.23.13.4559-4572.2003"><li><a href="http://dx.doi.org/10.1128/mcb.23.13.4559-4572.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=H1+linker+histones+are+essential+for+mouse+development+and+affect+nucleosome+spacing+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22H1+linker+histones+are+essential+for+mouse+development+and+affect+nucleosome+spacing+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001553-Hansen1" id="pone.0001553-Hansen1"></a>Hansen JC, Ausio J, Stanik VH, van Holde KE (1989) Homogeneous reconstituted oligonucleosomes, evidence for salt-dependent folding in the absence of histone H1. Biochemistry  28: 9129–9136.  <ul class="find" data-citedArticleID="1051520" data-doi="10.1021/bi00449a026"><li><a href="http://dx.doi.org/10.1021/bi00449a026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homogeneous+reconstituted+oligonucleosomes%2C+evidence+for+salt-dependent+folding+in+the+absence+of+histone+H1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homogeneous+reconstituted+oligonucleosomes%2C+evidence+for+salt-dependent+folding+in+the+absence+of+histone+H1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001553-Pruss1" id="pone.0001553-Pruss1"></a>Pruss D, Bartholomew B, Persinger J, Hayes J, Arents G, et al.  (1996) An asymmetric model for the nucleosome: a binding site for linker histones inside the DNA gyres. Science  274: 614–617.  <ul class="find" data-citedArticleID="1051542" data-doi="10.1126/science.274.5287.614"><li><a href="http://dx.doi.org/10.1126/science.274.5287.614" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+asymmetric+model+for+the+nucleosome%3A+a+binding+site+for+linker+histones+inside+the+DNA+gyres." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+asymmetric+model+for+the+nucleosome%3A+a+binding+site+for+linker+histones+inside+the+DNA+gyres.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001553-Ura1" id="pone.0001553-Ura1"></a>Ura K, Nightingale K, Wolffe AP (1996) Differential association of HMG1 and linker histones B4 and H1 with dinucleosomal DNA: structural transitions and transcriptional repression. Embo J  15: 4959–4969.  <ul class="find" data-citedArticleID="1051560"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Differential+association+of+HMG1+and+linker+histones+B4+and+H1+with+dinucleosomal+DNA%3A+structural+transitions+and+transcriptional+repression.&amp;auth=&amp;atitle=Differential+association+of+HMG1+and+linker+histones+B4+and+H1+with+dinucleosomal+DNA%3A+structural+transitions+and+transcriptional+repression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+association+of+HMG1+and+linker+histones+B4+and+H1+with+dinucleosomal+DNA%3A+structural+transitions+and+transcriptional+repression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+association+of+HMG1+and+linker+histones+B4+and+H1+with+dinucleosomal+DNA%3A+structural+transitions+and+transcriptional+repression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001553-Bresnick1" id="pone.0001553-Bresnick1"></a>Bresnick EH, John S, Hager GL (1991) Histone hyperacetylation does not alter the positioning or stability of phased nucleosomes on the mouse mammary tumor virus long terminal repeat. Biochemistry  30: 3490–3497.  <ul class="find" data-citedArticleID="1051496" data-doi="10.1021/bi00228a020"><li><a href="http://dx.doi.org/10.1021/bi00228a020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+hyperacetylation+does+not+alter+the+positioning+or+stability+of+phased+nucleosomes+on+the+mouse+mammary+tumor+virus+long+terminal+repeat." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+hyperacetylation+does+not+alter+the+positioning+or+stability+of+phased+nucleosomes+on+the+mouse+mammary+tumor+virus+long+terminal+repeat.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001553-Koop1" id="pone.0001553-Koop1"></a>Koop R, Di Croce L, Beato M (2003) Histone H1 enhances synergistic activation of the MMTV promoter in chromatin. Embo J  22: 588–599.  <ul class="find" data-citedArticleID="1051532" data-doi="10.1093/emboj/cdg052"><li><a href="http://dx.doi.org/10.1093/emboj/cdg052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+H1+enhances+synergistic+activation+of+the+MMTV+promoter+in+chromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+H1+enhances+synergistic+activation+of+the+MMTV+promoter+in+chromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001553-Steinbach1" id="pone.0001553-Steinbach1"></a>Steinbach OC, Wolffe AP, Rupp RA (1997) Somatic linker histones cause loss of mesodermal competence in Xenopus. Nature  389: 395–399.  <ul class="find" data-citedArticleID="1051546"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Somatic+linker+histones+cause+loss+of+mesodermal+competence+in+Xenopus.&amp;auth=&amp;atitle=Somatic+linker+histones+cause+loss+of+mesodermal+competence+in+Xenopus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Somatic+linker+histones+cause+loss+of+mesodermal+competence+in+Xenopus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Somatic+linker+histones+cause+loss+of+mesodermal+competence+in+Xenopus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001553-Draves1" id="pone.0001553-Draves1"></a>Draves PH, Lowary PT, Widom J (1992) Co-operative binding of the globular domain of histone H5 to DNA. J Mol Biol  225: 1105–1121.  <ul class="find" data-citedArticleID="1051506" data-doi="10.1016/0022-2836(92)90108-v"><li><a href="http://dx.doi.org/10.1016/0022-2836(92)90108-v" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Co-operative+binding+of+the+globular+domain+of+histone+H5+to+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Co-operative+binding+of+the+globular+domain+of+histone+H5+to+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001553-Thomas1" id="pone.0001553-Thomas1"></a>Thomas JO, Rees C, Finch JT (1992) Cooperative binding of the globular domains of histones H1 and H5 to DNA. Nucleic Acids Res  20: 187–194.  <ul class="find" data-citedArticleID="1051558" data-doi="10.1093/nar/20.2.187"><li><a href="http://dx.doi.org/10.1093/nar/20.2.187" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cooperative+binding+of+the+globular+domains+of+histones+H1+and+H5+to+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cooperative+binding+of+the+globular+domains+of+histones+H1+and+H5+to+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001553-Vermaak1" id="pone.0001553-Vermaak1"></a>Vermaak D, Steinbach OC, Dimitrov S, Rupp RA, Wolffe AP (1998) The globular domain of histone H1 is sufficient to direct specific gene repression in early Xenopus embryos. Curr Biol  8: 533–536.  <ul class="find" data-citedArticleID="1051564" data-doi="10.1016/s0960-9822(98)70206-4"><li><a href="http://dx.doi.org/10.1016/s0960-9822(98)70206-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+globular+domain+of+histone+H1+is+sufficient+to+direct+specific+gene+repression+in+early+Xenopus+embryos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+globular+domain+of+histone+H1+is+sufficient+to+direct+specific+gene+repression+in+early+Xenopus+embryos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001553-Ponte1" id="pone.0001553-Ponte1"></a>Ponte I, Vila R, Suau P (2003) Sequence complexity of histone H1 subtypes. Mol Biol Evol  20: 371–380.  <ul class="find" data-citedArticleID="1051540" data-doi="10.1093/molbev/msg041"><li><a href="http://dx.doi.org/10.1093/molbev/msg041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+complexity+of+histone+H1+subtypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+complexity+of+histone+H1+subtypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001553-Bharath1" id="pone.0001553-Bharath1"></a>Bharath MM, Ramesh S, Chandra NR, Rao MR (2002) Identification of a 34 amino acid stretch within the C-terminus of histone H1 as the DNA-condensing domain by site-directed mutagenesis. Biochemistry  41: 7617–7627.  <ul class="find" data-citedArticleID="1051492" data-doi="10.1021/bi025773+"><li><a href="http://dx.doi.org/10.1021/bi025773+" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+34+amino+acid+stretch+within+the+C-terminus+of+histone+H1+as+the+DNA-condensing+domain+by+site-directed+mutagenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+34+amino+acid+stretch+within+the+C-terminus+of+histone+H1+as+the+DNA-condensing+domain+by+site-directed+mutagenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001553-Izaurralde1" id="pone.0001553-Izaurralde1"></a>Izaurralde E, Kas E, Laemmli UK (1989) Highly preferential nucleation of histone H1 assembly on scaffold-associated regions. J Mol Biol  210: 573–585.  <ul class="find" data-citedArticleID="1051526" data-doi="10.1016/0022-2836(89)90133-2"><li><a href="http://dx.doi.org/10.1016/0022-2836(89)90133-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+preferential+nucleation+of+histone+H1+assembly+on+scaffold-associated+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+preferential+nucleation+of+histone+H1+assembly+on+scaffold-associated+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001553-Dou1" id="pone.0001553-Dou1"></a>Dou Y, Song X, Liu Y, Gorovsky MA (2005) The H1 phosphorylation state regulates expression of CDC2 and other genes in response to starvation in Tetrahymena thermophila. Mol Cell Biol  25: 3914–3922.  <ul class="find" data-citedArticleID="1051502" data-doi="10.1128/mcb.25.10.3914-3922.2005"><li><a href="http://dx.doi.org/10.1128/mcb.25.10.3914-3922.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+H1+phosphorylation+state+regulates+expression+of+CDC2+and+other+genes+in+response+to+starvation+in+Tetrahymena+thermophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+H1+phosphorylation+state+regulates+expression+of+CDC2+and+other+genes+in+response+to+starvation+in+Tetrahymena+thermophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001553-Garcia1" id="pone.0001553-Garcia1"></a>Garcia BA, Busby SA, Barber CM, Shabanowitz J, Allis CD, et al.  (2004) Characterization of phosphorylation sites on histone H1 isoforms by tandem mass spectrometry. J Proteome Res  3: 1219–1227.  <ul class="find" data-citedArticleID="1051512" data-doi="10.1021/pr0498887"><li><a href="http://dx.doi.org/10.1021/pr0498887" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+phosphorylation+sites+on+histone+H1+isoforms+by+tandem+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+phosphorylation+sites+on+histone+H1+isoforms+by+tandem+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001553-Sarg1" id="pone.0001553-Sarg1"></a>Sarg B, Green A, Soderkvist P, Helliger W, Rundquist I, et al.  (2005) Characterization of sequence variations in human histone H1.2 and H1.4 subtypes. Febs J  272: 3673–3683.  <ul class="find" data-citedArticleID="1051544" data-doi="10.1111/j.1742-4658.2005.04793.x"><li><a href="http://dx.doi.org/10.1111/j.1742-4658.2005.04793.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+sequence+variations+in+human+histone+H1.2+and+H1.4+subtypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+sequence+variations+in+human+histone+H1.2+and+H1.4+subtypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001553-Talasz1" id="pone.0001553-Talasz1"></a>Talasz H, Helliger W, Puschendorf B, Lindner H (1996) In vivo phosphorylation of histone H1 variants during the cell cycle. Biochemistry  35: 1761–1767.  <ul class="find" data-citedArticleID="1051548" data-doi="10.1021/bi951914e"><li><a href="http://dx.doi.org/10.1021/bi951914e" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+phosphorylation+of+histone+H1+variants+during+the+cell+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+phosphorylation+of+histone+H1+variants+during+the+cell+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001553-Thng1" id="pone.0001553-Thng1"></a>Th'ng JP, Sung R, Ye M, Hendzel MJ (2005) H1 family histones in the nucleus. Control of binding and localization by the C-terminal domain. J Biol Chem  280: 27809–27814.  <ul class="find" data-citedArticleID="1051556" data-doi="10.1074/jbc.m501627200"><li><a href="http://dx.doi.org/10.1074/jbc.m501627200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=H1+family+histones+in+the+nucleus.+Control+of+binding+and+localization+by+the+C-terminal+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22H1+family+histones+in+the+nucleus.+Control+of+binding+and+localization+by+the+C-terminal+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001553-Zlatanova1" id="pone.0001553-Zlatanova1"></a>Zlatanova J, Caiafa P, Van Holde K (2000) Linker histone binding and displacement: versatile mechanism for transcriptional regulation. Faseb J  14: 1697–1704.  <ul class="find" data-citedArticleID="1051572" data-doi="10.1096/fj.99-0869rev"><li><a href="http://dx.doi.org/10.1096/fj.99-0869rev" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linker+histone+binding+and+displacement%3A+versatile+mechanism+for+transcriptional+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linker+histone+binding+and+displacement%3A+versatile+mechanism+for+transcriptional+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001553-Kuzmichev1" id="pone.0001553-Kuzmichev1"></a>Kuzmichev A, Jenuwein T, Tempst P, Reinberg D (2004) Different EZH2-containing complexes target methylation of histone H1 or nucleosomal histone H3. Mol Cell  14: 183–193.  <ul class="find" data-citedArticleID="1051534" data-doi="10.1016/s1097-2765(04)00185-6"><li><a href="http://dx.doi.org/10.1016/s1097-2765(04)00185-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Different+EZH2-containing+complexes+target+methylation+of+histone+H1+or+nucleosomal+histone+H3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Different+EZH2-containing+complexes+target+methylation+of+histone+H1+or+nucleosomal+histone+H3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001553-Daujat1" id="pone.0001553-Daujat1"></a>Daujat S, Zeissler U, Waldmann T, Happel N, Schneider R (2005) HP1 binds specifically to Lys26-methylated histone H1.4, whereas simultaneous Ser27 phosphorylation blocks HP1 binding. J Biol Chem  280: 38090–38095.  <ul class="find" data-citedArticleID="1051500" data-doi="10.1074/jbc.c500229200"><li><a href="http://dx.doi.org/10.1074/jbc.c500229200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HP1+binds+specifically+to+Lys26-methylated+histone+H1.4%2C+whereas+simultaneous+Ser27+phosphorylation+blocks+HP1+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HP1+binds+specifically+to+Lys26-methylated+histone+H1.4%2C+whereas+simultaneous+Ser27+phosphorylation+blocks+HP1+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001553-Wisniewski1" id="pone.0001553-Wisniewski1"></a>Wisniewski JR, Zougman A, Kruger S, Mann M (2007) Mass spectrometric mapping of linker histone H1 variants reveals multiple acetylations, methylations, and phosphorylation as well as differences between cell culture and tissue. Mol Cell Proteomics  6: 72–87.  <ul class="find" data-citedArticleID="1051566" data-doi="10.1074/mcp.m600255-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m600255-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mass+spectrometric+mapping+of+linker+histone+H1+variants+reveals+multiple+acetylations%2C+methylations%2C+and+phosphorylation+as+well+as+differences+between+cell+culture+and+tissue." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mass+spectrometric+mapping+of+linker+histone+H1+variants+reveals+multiple+acetylations%2C+methylations%2C+and+phosphorylation+as+well+as+differences+between+cell+culture+and+tissue.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001553-Jiang1" id="pone.0001553-Jiang1"></a>Jiang T, Zhou X, Taghizadeh K, Dong M, Dedon PC (2007) N-formylation of lysine in histone proteins as a secondary modification arising from oxidative DNA damage. Proc Natl Acad Sci U S A  104: 60–65.  <ul class="find" data-citedArticleID="1051528" data-doi="10.1073/pnas.0606775103"><li><a href="http://dx.doi.org/10.1073/pnas.0606775103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=N-formylation+of+lysine+in+histone+proteins+as+a+secondary+modification+arising+from+oxidative+DNA+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22N-formylation+of+lysine+in+histone+proteins+as+a+secondary+modification+arising+from+oxidative+DNA+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001553-Horn1" id="pone.0001553-Horn1"></a>Horn PJ, Carruthers LM, Logie C, Hill DA, Solomon MJ, et al.  (2002) Phosphorylation of linker histones regulates ATP-dependent chromatin remodeling enzymes. Nat Struct Biol  9: 263–267.  <ul class="find" data-citedArticleID="1051524" data-doi="10.1038/nsb776"><li><a href="http://dx.doi.org/10.1038/nsb776" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylation+of+linker+histones+regulates+ATP-dependent+chromatin+remodeling+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylation+of+linker+histones+regulates+ATP-dependent+chromatin+remodeling+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001553-Dou2" id="pone.0001553-Dou2"></a>Dou Y, Mizzen CA, Abrams M, Allis CD, Gorovsky MA (1999) Phosphorylation of linker histone H1 regulates gene expression in vivo by mimicking H1 removal. Mol Cell  4: 641–647.  <ul class="find" data-citedArticleID="1051504" data-doi="10.1016/s1097-2765(00)80215-4"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)80215-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylation+of+linker+histone+H1+regulates+gene+expression+in+vivo+by+mimicking+H1+removal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylation+of+linker+histone+H1+regulates+gene+expression+in+vivo+by+mimicking+H1+removal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001553-Hale1" id="pone.0001553-Hale1"></a>Hale TK, Contreras A, Morrison AJ, Herrera RE (2006) Phosphorylation of the linker histone H1 by CDK regulates its binding to HP1alpha. Mol Cell  22: 693–699.  <ul class="find" data-citedArticleID="1051518" data-doi="10.1016/j.molcel.2006.04.016"><li><a href="http://dx.doi.org/10.1016/j.molcel.2006.04.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylation+of+the+linker+histone+H1+by+CDK+regulates+its+binding+to+HP1alpha." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylation+of+the+linker+histone+H1+by+CDK+regulates+its+binding+to+HP1alpha.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001553-Fan2" id="pone.0001553-Fan2"></a>Fan Y, Nikitina T, Zhao J, Fleury TJ, Bhattacharyya R, et al.  (2005) Histone H1 depletion in mammals alters global chromatin structure but causes specific changes in gene regulation. Cell  123: 1199–1212.  <ul class="find" data-citedArticleID="1051510" data-doi="10.1016/j.cell.2005.10.028"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.10.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+H1+depletion+in+mammals+alters+global+chromatin+structure+but+causes+specific+changes+in+gene+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+H1+depletion+in+mammals+alters+global+chromatin+structure+but+causes+specific+changes+in+gene+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001553-Anderson1" id="pone.0001553-Anderson1"></a>Anderson KV, Lengyel JA (1980) Changing rates of histone mRNA synthesis and turnover in Drosophila embryos. Cell  21: 717–727.  <ul class="find" data-citedArticleID="1051488" data-doi="10.1016/0092-8674(80)90435-3"><li><a href="http://dx.doi.org/10.1016/0092-8674(80)90435-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changing+rates+of+histone+mRNA+synthesis+and+turnover+in+Drosophila+embryos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changing+rates+of+histone+mRNA+synthesis+and+turnover+in+Drosophila+embryos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001553-Ner1" id="pone.0001553-Ner1"></a>Ner SS, Blank T, Perez-Paralle ML, Grigliatti TA, Becker PB, et al.  (2001) HMG-D and histone H1 interplay during chromatin assembly and early embryogenesis. J Biol Chem  276: 37569–37576.  <ul class="find" data-citedArticleID="1051536" data-doi="10.1074/jbc.m105635200"><li><a href="http://dx.doi.org/10.1074/jbc.m105635200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HMG-D+and+histone+H1+interplay+during+chromatin+assembly+and+early+embryogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HMG-D+and+histone+H1+interplay+during+chromatin+assembly+and+early+embryogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001553-Ner2" id="pone.0001553-Ner2"></a>Ner SS, Travers AA (1994) HMG-D, the Drosophila melanogaster homologue of HMG 1 protein, is associated with early embryonic chromatin in the absence of histone H1. EMBO J  13: 1817–1822.  <ul class="find" data-citedArticleID="1051538"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=HMG-D%2C+the+Drosophila+melanogaster+homologue+of+HMG+1+protein%2C+is+associated+with+early+embryonic+chromatin+in+the+absence+of+histone+H1.&amp;auth=&amp;atitle=HMG-D%2C+the+Drosophila+melanogaster+homologue+of+HMG+1+protein%2C+is+associated+with+early+embryonic+chromatin+in+the+absence+of+histone+H1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HMG-D%2C+the+Drosophila+melanogaster+homologue+of+HMG+1+protein%2C+is+associated+with+early+embryonic+chromatin+in+the+absence+of+histone+H1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HMG-D%2C+the+Drosophila+melanogaster+homologue+of+HMG+1+protein%2C+is+associated+with+early+embryonic+chromatin+in+the+absence+of+histone+H1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001553-Blumenfeld1" id="pone.0001553-Blumenfeld1"></a>Blumenfeld M (1979) Phosphorylated H1 histone in Drosophila melanogaster. Biochem Genet  17: 163–166.  <ul class="find" data-citedArticleID="1051494" data-doi="10.1007/bf00484482"><li><a href="http://dx.doi.org/10.1007/bf00484482" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylated+H1+histone+in+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylated+H1+histone+in+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001553-Talmage1" id="pone.0001553-Talmage1"></a>Talmage DA, Blumenfeld M (1987) Drosophila melanogaster H1 histone is phosphorylated stably. Mol Cell Biol  7: 4118–4121.  <ul class="find" data-citedArticleID="1051554"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Drosophila+melanogaster+H1+histone+is+phosphorylated+stably.&amp;auth=&amp;atitle=Drosophila+melanogaster+H1+histone+is+phosphorylated+stably." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Drosophila+melanogaster+H1+histone+is+phosphorylated+stably." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Drosophila+melanogaster+H1+histone+is+phosphorylated+stably.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001553-Gurley1" id="pone.0001553-Gurley1"></a>Gurley LR, Valdez JG, Buchanan JS (1995) Characterization of the mitotic specific phosphorylation site of histone H1. Absence of a consensus sequence for the p34cdc2/cyclin B kinase. J Biol Chem  270: 27653–27660.  <ul class="find" data-citedArticleID="1051516" data-doi="10.1074/jbc.270.46.27653"><li><a href="http://dx.doi.org/10.1074/jbc.270.46.27653" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+mitotic+specific+phosphorylation+site+of+histone+H1.+Absence+of+a+consensus+sequence+for+the+p34cdc2%2Fcyclin+B+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+mitotic+specific+phosphorylation+site+of+histone+H1.+Absence+of+a+consensus+sequence+for+the+p34cdc2%2Fcyclin+B+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001553-Giancotti1" id="pone.0001553-Giancotti1"></a>Giancotti V, Russo E, de Cristini F, Graziosi G, Micali F, et al.  (1984) Histone modification in early and late Drosophila embryos. Biochem J  218: 321–329.  <ul class="find" data-citedArticleID="1051514"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Histone+modification+in+early+and+late+Drosophila+embryos.&amp;auth=&amp;atitle=Histone+modification+in+early+and+late+Drosophila+embryos." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+modification+in+early+and+late+Drosophila+embryos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+modification+in+early+and+late+Drosophila+embryos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001553-Harisanova1" id="pone.0001553-Harisanova1"></a>Harisanova NT, Ralchev KH (1986) Changes in the pattern of histone H1 phosphorylation during Drosophila hydei embryogenesis. Mol Biol Rep  11: 199–203.  <ul class="find" data-citedArticleID="1051522" data-doi="10.1007/bf00419597"><li><a href="http://dx.doi.org/10.1007/bf00419597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changes+in+the+pattern+of+histone+H1+phosphorylation+during+Drosophila+hydei+embryogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changes+in+the+pattern+of+histone+H1+phosphorylation+during+Drosophila+hydei+embryogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001553-Vaquero1" id="pone.0001553-Vaquero1"></a>Vaquero A, Scher M, Lee D, Erdjument-Bromage H, Tempst P, et al.  (2004) Human SirT1 interacts with histone H1 and promotes formation of facultative heterochromatin. Mol Cell  16: 93–105.  <ul class="find" data-citedArticleID="1051562" data-doi="10.1016/j.molcel.2004.08.031"><li><a href="http://dx.doi.org/10.1016/j.molcel.2004.08.031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+SirT1+interacts+with+histone+H1+and+promotes+formation+of+facultative+heterochromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+SirT1+interacts+with+histone+H1+and+promotes+formation+of+facultative+heterochromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001553-Talasz2" id="pone.0001553-Talasz2"></a>Talasz H, Lindner HH, Sarg B, Helliger W (2005) Histone H4-lysine 20 monomethylation is increased in promoter and coding regions of active genes and correlates with hyperacetylation. J Biol Chem  280: 38814–38822.  <ul class="find" data-citedArticleID="1051550" data-doi="10.1074/jbc.m505563200"><li><a href="http://dx.doi.org/10.1074/jbc.m505563200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+H4-lysine+20+monomethylation+is+increased+in+promoter+and+coding+regions+of+active+genes+and+correlates+with+hyperacetylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+H4-lysine+20+monomethylation+is+increased+in+promoter+and+coding+regions+of+active+genes+and+correlates+with+hyperacetylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001553-Talasz3" id="pone.0001553-Talasz3"></a>Talasz H, Helliger W, Sarg B, Debbage PL, Puschendorf B, et al.  (2002) Hyperphosphorylation of histone H2A.X and dephosphorylation of histone H1 subtypes in the course of apoptosis. Cell Death Differ  9: 27–39.  <ul class="find" data-citedArticleID="1051552" data-doi="10.1038/sj.cdd.4400925"><li><a href="http://dx.doi.org/10.1038/sj.cdd.4400925" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hyperphosphorylation+of+histone+H2A.X+and+dephosphorylation+of+histone+H1+subtypes+in+the+course+of+apoptosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hyperphosphorylation+of+histone+H2A.X+and+dephosphorylation+of+histone+H1+subtypes+in+the+course+of+apoptosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001553-Arrigoni1" id="pone.0001553-Arrigoni1"></a>Arrigoni G, Resjo S, Levander F, Nilsson R, Degerman E, et al.  (2006) Chemical derivatization of phosphoserine and phosphothreonine containing peptides to increase sensitivity for MALDI-based analysis and for selectivity of MS/MS analysis. Proteomics  6: 757–766.  <ul class="find" data-citedArticleID="1051490" data-doi="10.1002/pmic.200500073"><li><a href="http://dx.doi.org/10.1002/pmic.200500073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemical+derivatization+of+phosphoserine+and+phosphothreonine+containing+peptides+to+increase+sensitivity+for+MALDI-based+analysis+and+for+selectivity+of+MS%2FMS+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemical+derivatization+of+phosphoserine+and+phosphothreonine+containing+peptides+to+increase+sensitivity+for+MALDI-based+analysis+and+for+selectivity+of+MS%2FMS+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.XML" value="89440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.PDF" value="906301"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g001.PNG_L" value="1759345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g001.PNG_M" value="149804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g001.PNG_S" value="12185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g001.TIF" value="2765662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g001.PNG_I" value="81411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g002.PNG_L" value="2582839"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g002.PNG_M" value="139491"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g002.PNG_S" value="14954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g002.TIF" value="4778918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g002.PNG_I" value="78458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g003.PNG_L" value="862288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g003.PNG_M" value="88466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g003.PNG_S" value="13829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g003.TIF" value="1756904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g003.PNG_I" value="40315"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g004.PNG_L" value="2717372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g004.PNG_M" value="200312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g004.PNG_S" value="18924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g004.TIF" value="3991786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g004.PNG_I" value="110727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g005.PNG_L" value="949594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g005.PNG_M" value="75382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g005.PNG_S" value="11156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g005.TIF" value="1802272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001553.g005.PNG_I" value="37265"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001553&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001553" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001553&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001553&volume=&issue=&title=Fine Mapping of Posttranslational Modifications of the Linker Histone H1 from Drosophila melanogaster&author_name=Ana%20Villar-Garea%2C%20Axel%20Imhof&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001553" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001553" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001553" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001553&amp;t=Fine%20Mapping%20of%20Posttranslational%20Modifications%20of%20the%20Linker%20Histone%20H1%20from%20Drosophila%20melanogaster" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001553&title=Fine%20Mapping%20of%20Posttranslational%20Modifications%20of%20the%20Linker%20Histone%20H1%20from%20Drosophila%20melanogaster&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001553&amp;title=Fine%20Mapping%20of%20Posttranslational%20Modifications%20of%20the%20Linker%20Histone%20H1%20from%20Drosophila%20melanogaster" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001553" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001553&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Fine Mapping of Posttranslational Modifications of the Linker Histone H1 from Drosophila melanogaster';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001553';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Fine%20Mapping%20of%20Posttranslational%20Modifications%20of%20the%20Linker%20Histone%20H1%20from%20Drosophila%20melanogaster http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001553" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001553" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="25932"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drosophila+melanogaster%22" title="Search for articles in the subject area:'Drosophila melanogaster'"><div class="flagText">Drosophila melanogaster</div></a>
              <div data-categoryid="32461" data-articleid="25932"
                   data-categoryname="Drosophila melanogaster"
                   class="flagImage" title="Flag 'Drosophila melanogaster' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Embryos%22" title="Search for articles in the subject area:'Embryos'"><div class="flagText">Embryos</div></a>
              <div data-categoryid="17321" data-articleid="25932"
                   data-categoryname="Embryos"
                   class="flagImage" title="Flag 'Embryos' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lysine%22" title="Search for articles in the subject area:'Lysine'"><div class="flagText">Lysine</div></a>
              <div data-categoryid="40335" data-articleid="25932"
                   data-categoryname="Lysine"
                   class="flagImage" title="Flag 'Lysine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Matrix-assisted+laser+desorption+ionization+time-of-flight+mass+spectrometry%22" title="Search for articles in the subject area:'Matrix-assisted laser desorption ionization time-of-flight mass spectrometry'"><div class="flagText">Matrix-assisted laser desorption ionization time-of-flight mass spectrometry</div></a>
              <div data-categoryid="42347" data-articleid="25932"
                   data-categoryname="Matrix-assisted laser desorption ionization time-of-flight mass spectrometry"
                   class="flagImage" title="Flag 'Matrix-assisted laser desorption ionization time-of-flight mass spectrometry' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Peptides%22" title="Search for articles in the subject area:'Peptides'"><div class="flagText">Peptides</div></a>
              <div data-categoryid="17847" data-articleid="25932"
                   data-categoryname="Peptides"
                   class="flagImage" title="Flag 'Peptides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phosphorylation%22" title="Search for articles in the subject area:'Phosphorylation'"><div class="flagText">Phosphorylation</div></a>
              <div data-categoryid="42723" data-articleid="25932"
                   data-categoryname="Phosphorylation"
                   class="flagImage" title="Flag 'Phosphorylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Signal+peptides%22" title="Search for articles in the subject area:'Signal peptides'"><div class="flagText">Signal peptides</div></a>
              <div data-categoryid="19701" data-articleid="25932"
                   data-categoryname="Signal peptides"
                   class="flagImage" title="Flag 'Signal peptides' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4696'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2691'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4775&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
