

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Wide-Scale Analysis of Human Functional Transcription Factor Binding Reveals a Strong Bias towards the Transcription Start Site</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000807"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000807" />

    <meta name="citation_title" content="Wide-Scale Analysis of Human Functional Transcription Factor Binding Reveals a Strong Bias towards the Transcription Start Site"/>
    <meta itemprop="name" content="Wide-Scale Analysis of Human Functional Transcription Factor Binding Reveals a Strong Bias towards the Transcription Start Site"/>

      <meta name="citation_author" content="Yuval Tabach"/>
            <meta name="citation_author_institution" content="Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel"/>
            <meta name="citation_author_institution" content="Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Ran Brosh"/>
            <meta name="citation_author_institution" content="Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Yossi Buganim"/>
            <meta name="citation_author_institution" content="Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Anat Reiner"/>
            <meta name="citation_author_institution" content="Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Or Zuk"/>
            <meta name="citation_author_institution" content="Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Assif Yitzhaky"/>
            <meta name="citation_author_institution" content="Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Mark Koudritsky"/>
            <meta name="citation_author_institution" content="Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Varda Rotter"/>
            <meta name="citation_author_institution" content="Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Eytan Domany"/>
            <meta name="citation_author_institution" content="Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel"/>

    <meta name="citation_date" content="2007/8/29"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000807.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e807"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Locating mammalian transcription factor binding sites: a survey of computational and experimental techniques.; citation_author=L Elnitski; citation_author=VX Jin; citation_author=PJ Farnham; citation_author=SJ Jones; citation_journal_title=Genome Res; citation_volume=16; citation_number=1; citation_pages=1455-1464; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Toucan: deciphering the cis-regulatory logic of coregulated genes.; citation_author=S Aerts; citation_author=G Thijs; citation_author=B Coessens; citation_author=M Staes; citation_author=Y Moreau; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=2; citation_pages=1753-1764; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Physical constraints and functional characteristics of transcription factor-DNA interaction.; citation_author=U Gerland; citation_author=JD Moroz; citation_author=T Hwa; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=3; citation_pages=12015-12020; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Microarray data analysis: from disarray to consolidation and consensus.; citation_author=DB Allison; citation_author=X Cui; citation_author=GP Page; citation_author=M Sabripour; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=4; citation_pages=55-65; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Computational framework for the prediction of transcription factor binding sites by multiple data integration.; citation_author=A Ambesi-Impiombato; citation_author=M Bansal; citation_author=P Lio; citation_author=D di Bernardo; citation_journal_title=BMC Neurosci; citation_volume=7; citation_number=5; citation_pages=S8; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Constructing transcriptional regulatory networks.; citation_author=A Blais; citation_author=BD Dynlacht; citation_journal_title=Genes Dev; citation_volume=19; citation_number=6; citation_pages=1499-1511; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide location and function of DNA binding proteins.; citation_author=B Ren; citation_author=F Robert; citation_author=JJ Wyrick; citation_author=O Aparicio; citation_author=EG Jennings; citation_journal_title=Science; citation_volume=290; citation_number=7; citation_pages=2306-2309; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Control of gene expression and assembly of chromosomal subdomains by chromatin regulators with antagonistic functions.; citation_author=AL Lam; citation_author=DE Pazin; citation_author=BA Sullivan; citation_journal_title=Chromosoma; citation_volume=114; citation_number=8; citation_pages=242-251; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide in silico identification of transcriptional regulators controlling the cell cycle in human cells.; citation_author=R Elkon; citation_author=C Linhart; citation_author=R Sharan; citation_author=R Shamir; citation_author=Y Shiloh; citation_journal_title=Genome Res; citation_volume=13; citation_number=9; citation_pages=773-780; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=oPOSSUM: identification of over-represented transcription factor binding sites in co-expressed genes.; citation_author=SJ Ho Sui; citation_author=JR Mortimer; citation_author=DJ Arenillas; citation_author=J Brumm; citation_author=CJ Walsh; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=10; citation_pages=3154-3164; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Computationally identifying novel NF-kappa B-regulated immune genes in the human genome.; citation_author=R Liu; citation_author=RC McEachin; citation_author=DJ States; citation_journal_title=Genome Res; citation_volume=13; citation_number=11; citation_pages=654-661; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Finding protein-binding sites in DNA sequences: the next generation.; citation_author=K Frech; citation_author=K Quandt; citation_author=T Werner; citation_journal_title=Trends Biochem Sci; citation_volume=22; citation_number=12; citation_pages=103-104; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic footprinting of transcription factor binding sites in proteobacterial genomes.; citation_author=L McCue; citation_author=W Thompson; citation_author=C Carmack; citation_author=MP Ryan; citation_author=JS Liu; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=13; citation_pages=774-782; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The evolution of DNA regulatory regions for proteo-gamma bacteria by interspecies comparisons.; citation_author=N Rajewsky; citation_author=ND Socci; citation_author=M Zapotocky; citation_author=ED Siggia; citation_journal_title=Genome Res; citation_volume=12; citation_number=14; citation_pages=298-308; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Conservation of DNA regulatory motifs and discovery of new motifs in microbial genomes.; citation_author=AM McGuire; citation_author=JD Hughes; citation_author=GM Church; citation_journal_title=Genome Res; citation_volume=10; citation_number=15; citation_pages=744-757; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Systematic discovery of regulatory motifs in human promoters and 3′ UTRs by comparison of several mammals.; citation_author=X Xie; citation_author=J Lu; citation_author=EJ Kulbokas; citation_author=TR Golub; citation_author=V Mootha; citation_journal_title=Nature; citation_volume=434; citation_number=16; citation_pages=338-345; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.; citation_author=M Kellis; citation_author=N Patterson; citation_author=M Endrizzi; citation_author=B Birren; citation_author=ES Lander; citation_journal_title=Nature; citation_volume=423; citation_number=17; citation_pages=241-254; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Finding functional features in Saccharomyces genomes by phylogenetic footprinting.; citation_author=P Cliften; citation_author=P Sudarsanam; citation_author=A Desikan; citation_author=L Fulton; citation_author=B Fulton; citation_journal_title=Science; citation_volume=301; citation_number=18; citation_pages=71-76; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Enhancer sequence conservation between vertebrates is favoured in developmental regulator genes.; citation_author=C Plessy; citation_author=T Dickmeis; citation_author=F Chalmel; citation_author=U Strahle; citation_journal_title=Trends Genet; citation_volume=21; citation_number=19; citation_pages=207-210; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Conservation of regulatory elements between two species of Drosophila.; citation_author=E Emberly; citation_author=N Rajewsky; citation_author=ED Siggia; citation_journal_title=BMC Bioinformatics; citation_volume=4; citation_number=20; citation_pages=57; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Megabase deletions of gene deserts result in viable mice.; citation_author=MA Nobrega; citation_author=Y Zhu; citation_author=I Plajzer-Frick; citation_author=V Afzal; citation_author=EM Rubin; citation_journal_title=Nature; citation_volume=431; citation_number=21; citation_pages=988-993; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Inferring nonneutral evolution from human-chimp-mouse orthologous gene trios.; citation_author=AG Clark; citation_author=S Glanowski; citation_author=R Nielsen; citation_author=PD Thomas; citation_author=A Kejariwal; citation_journal_title=Science; citation_volume=302; citation_number=22; citation_pages=1960-1963; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Exploring the metabolic and genetic control of gene expression on a genomic scale.; citation_author=JL DeRisi; citation_author=VR Iyer; citation_author=PO Brown; citation_journal_title=Science; citation_volume=278; citation_number=23; citation_pages=680-686; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Computational identification of cis-regulatory elements associated with groups of functionally related genes in Saccharomyces cerevisiae.; citation_author=JD Hughes; citation_author=PW Estep; citation_author=S Tavazoie; citation_author=GM Church; citation_journal_title=J Mol Biol; citation_volume=296; citation_number=24; citation_pages=1205-1214; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Finding DNA regulatory motifs within unaligned noncoding sequences clustered by whole-genome mRNA quantitation.; citation_author=FP Roth; citation_author=JD Hughes; citation_author=PW Estep; citation_author=GM Church; citation_journal_title=Nat Biotechnol; citation_volume=16; citation_number=25; citation_pages=939-945; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Computational inference of transcriptional regulatory networks from expression profiling and transcription factor binding site identification.; citation_author=PM Haverty; citation_author=U Hansen; citation_author=Z Weng; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=26; citation_pages=179-188; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An approach to identify over-represented cis-elements in related sequences.; citation_author=J Zheng; citation_author=J Wu; citation_author=Z Sun; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=27; citation_pages=1995-2005; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The promoters of human cell cycle genes integrate signals from two tumor suppressive pathways during cellular transformation.; citation_author=Y Tabach; citation_author=M Milyavsky; citation_author=I Shats; citation_author=R Brosh; citation_author=O Zuk; citation_journal_title=Mol Syst Biol; citation_volume=1; citation_number=28; citation_pages=2005-0022; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.; citation_author=A Subramanian; citation_author=P Tamayo; citation_author=VK Mootha; citation_author=S Mukherjee; citation_author=BL Ebert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=29; citation_pages=15545-15550; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.; citation_author=M Ashburner; citation_author=CA Ball; citation_author=JA Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nat Genet; citation_volume=25; citation_number=30; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification of the binding sites of regulatory proteins in bacterial genomes.; citation_author=H Li; citation_author=V Rhodius; citation_author=C Gross; citation_author=ED Siggia; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=31; citation_pages=11772-11777; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identifying DNA and protein patterns with statistically significant alignments of multiple sequences.; citation_author=GZ Hertz; citation_author=GD Stormo; citation_journal_title=Bioinformatics; citation_volume=15; citation_number=32; citation_pages=563-577; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=MATCH: A tool for searching transcription factor binding sites in DNA sequences.; citation_author=AE Kel; citation_author=E Gossling; citation_author=I Reuter; citation_author=E Cheremushkin; citation_author=OV Kel-Margoulis; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=33; citation_pages=3576-3579; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Detection of functional DNA motifs via statistical over-representation.; citation_author=MC Frith; citation_author=Y Fu; citation_author=L Yu; citation_author=JF Chen; citation_author=U Hansen; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=34; citation_pages=1372-1381; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MatInd and MatInspector: new fast and versatile tools for detection of consensus matches in nucleotide sequence data.; citation_author=K Quandt; citation_author=K Frech; citation_author=H Karas; citation_author=E Wingender; citation_author=T Werner; citation_journal_title=Nucleic Acids Res; citation_volume=23; citation_number=35; citation_pages=4878-4884; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Identification of genes periodically expressed in the human cell cycle and their expression in tumors.; citation_author=ML Whitfield; citation_author=G Sherlock; citation_author=AJ Saldanha; citation_author=JI Murray; citation_author=CA Ball; citation_journal_title=Mol Biol Cell; citation_volume=13; citation_number=36; citation_pages=1977-2000; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Parallel induction of ATM-dependent pro- and antiapoptotic signals in response to ionizing radiation in murine lymphoid tissue.; citation_author=S Rashi-Elkeles; citation_author=R Elkon; citation_author=N Weizman; citation_author=C Linhart; citation_author=N Amariglio; citation_journal_title=Oncogene; citation_volume=25; citation_number=37; citation_pages=1584-1592; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular Cell Biology W. H. Freeman.; citation_author=H Lodish; citation_author=A Berk; citation_author=SL Zipursky; citation_author=P Matsudaira; citation_author=D Baltimore; citation_number=38; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory code of a eukaryotic genome.; citation_author=CT Harbison; citation_author=DB Gordon; citation_author=TI Lee; citation_author=NJ Rinaldi; citation_author=KD Macisaac; citation_journal_title=Nature; citation_volume=431; citation_number=39; citation_pages=99-104; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Campomelic dysplasia translocation breakpoints are scattered over 1 Mb proximal to SOX9: evidence for an extended control region.; citation_author=D Pfeifer; citation_author=R Kist; citation_author=K Dewar; citation_author=K Devon; citation_author=ES Lander; citation_journal_title=Am J Hum Genet; citation_volume=65; citation_number=40; citation_pages=111-124; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Characterization of the pufferfish Otx2 cis-regulators reveals evolutionarily conserved genetic mechanisms for vertebrate head specification.; citation_author=C Kimura-Yoshida; citation_author=K Kitajima; citation_author=I Oda-Ishii; citation_author=E Tian; citation_author=M Suzuki; citation_journal_title=Development; citation_volume=131; citation_number=41; citation_pages=57-71; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=cis-Regulatory control circuits in development.; citation_author=ML Howard; citation_author=EH Davidson; citation_journal_title=Dev Biol; citation_volume=271; citation_number=42; citation_pages=109-118; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CTCF mediates interchromosomal colocalization between Igf2/H19 and Wsb1/Nf1.; citation_author=JQ Ling; citation_author=T Li; citation_author=JF Hu; citation_author=TH Vu; citation_author=HL Chen; citation_journal_title=Science; citation_volume=312; citation_number=43; citation_pages=269-272; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of mammalian promoter architecture and evolution.; citation_author=P Carninci; citation_author=A Sandelin; citation_author=B Lenhard; citation_author=S Katayama; citation_author=K Shimokawa; citation_journal_title=Nat Genet; citation_volume=38; citation_number=44; citation_pages=626-635; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=CREME: Cis-Regulatory Module Explorer for the human genome.; citation_author=R Sharan; citation_author=A Ben-Hur; citation_author=GG Loots; citation_author=I Ovcharenko; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=45; citation_pages=W253-256; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Differential Regulation of NF-{kappa}B by Elongation Factors Is Determined by Core Promoter Type.; citation_author=L Amir-Zilberstein; citation_author=E Ainbinder; citation_author=L Toube; citation_author=Y Yamaguchi; citation_author=H Handa; citation_journal_title=Mol Cell Biol; citation_volume=27; citation_number=46; citation_pages=5246-5259; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Prevalence of the initiator over the TATA box in human and yeast genes and identification of DNA motifs enriched in human TATA-less core promoters.; citation_author=C Yang; citation_author=E Bolotin; citation_author=T Jiang; citation_author=FM Sladek; citation_author=E Martinez; citation_journal_title=Gene; citation_volume=389; citation_number=47; citation_pages=52-65; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A Simple Hyper-Geometric Approach for Discovering Putative Transcription Factor Binding Sites Algorithms in Bioinformatics: Springer Berlin/Heidelberg.; citation_author=Y Barash; citation_author=G Bejerano; citation_author=N Friedman; citation_number=48; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Controlling the False Discovery Rate: A practical and powerful approach to multiple testing.; citation_author=Y Benjamini; citation_author=Y Hochberg; citation_journal_title=J Roy Stat Soc B,; citation_volume=57; citation_number=49; citation_pages=289-300; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Human-mouse genome comparisons to locate regulatory sites.; citation_author=WW Wasserman; citation_author=M Palumbo; citation_author=W Thompson; citation_author=JW Fickett; citation_author=CE Lawrence; citation_journal_title=Nat Genet; citation_volume=26; citation_number=50; citation_pages=225-228; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Systematic determination of genetic network architecture.; citation_author=S Tavazoie; citation_author=JD Hughes; citation_author=MJ Campbell; citation_author=RJ Cho; citation_author=GM Church; citation_journal_title=Nat Genet; citation_volume=22; citation_number=51; citation_pages=281-285; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Myocardin and ternary complex factors compete for SRF to control smooth muscle gene expression.; citation_author=Z Wang; citation_author=DZ Wang; citation_author=D Hockemeyer; citation_author=J McAnally; citation_author=A Nordheim; citation_journal_title=Nature; citation_volume=428; citation_number=52; citation_pages=185-189; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Activation of cardiac gene expression by myocardin, a transcriptional cofactor for serum response factor.; citation_author=D Wang; citation_author=PS Chang; citation_author=Z Wang; citation_author=L Sutherland; citation_author=JA Richardson; citation_journal_title=Cell; citation_volume=105; citation_number=53; citation_pages=851-862; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Stem cells and their derivatives can bypass the requirement of myocardin for smooth muscle gene expression.; citation_author=GC Pipes; citation_author=S Sinha; citation_author=X Qi; citation_author=CH Zhu; citation_author=TD Gallardo; citation_journal_title=Dev Biol; citation_volume=288; citation_number=54; citation_pages=502-513; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Nuclear factor-kappaB p50 is required for tumor necrosis factor-alpha-induced colony-stimulating factor-1 gene expression in osteoblasts.; citation_author=GQ Yao; citation_author=BH Sun; citation_author=KL Insogna; citation_author=EC Weir; citation_journal_title=Endocrinology; citation_volume=141; citation_number=55; citation_pages=2914-2922; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Essential roles for NF-kappa B and a Toll/IL-1 receptor domain-specific signal(s) in the induction of I kappa B-zeta.; citation_author=A Eto; citation_author=T Muta; citation_author=S Yamazaki; citation_author=K Takeshige; citation_journal_title=Biochem Biophys Res Commun; citation_volume=301; citation_number=56; citation_pages=495-501; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The transcriptional landscape of the mammalian genome.; citation_author=P Carninci; citation_author=T Kasukawa; citation_author=S Katayama; citation_author=J Gough; citation_author=MC Frith; citation_journal_title=Science; citation_volume=309; citation_number=57; citation_pages=1559-1563; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Strong bias in the location of functional promoter polymorphisms.; citation_author=PR Buckland; citation_author=B Hoogendoorn; citation_author=SL Coleman; citation_author=CA Guy; citation_author=SK Smith; citation_journal_title=Hum Mutat; citation_volume=26; citation_number=58; citation_pages=214-223; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comprehensive analysis of transcriptional promoter structure and function in 1% of the human genome.; citation_author=SJ Cooper; citation_author=ND Trinklein; citation_author=ED Anton; citation_author=L Nguyen; citation_author=RM Myers; citation_journal_title=Genome Res; citation_volume=16; citation_number=59; citation_pages=1-10; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transposable elements donate lineage-specific regulatory sequences to host genomes.; citation_author=L Marino-Ramirez; citation_author=KC Lewis; citation_author=D Landsman; citation_author=IK Jordan; citation_journal_title=Cytogenet Genome Res; citation_volume=110; citation_number=60; citation_pages=333-341; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-scale identification of nucleosome positions in S. cerevisiae.; citation_author=GC Yuan; citation_author=YJ Liu; citation_author=MF Dion; citation_author=MD Slack; citation_author=LF Wu; citation_journal_title=Science; citation_volume=309; citation_number=61; citation_pages=626-630; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A genomic code for nucleosome positioning.; citation_author=E Segal; citation_author=Y Fondufe-Mittendorf; citation_author=L Chen; citation_author=A Thastrom; citation_author=Y Field; citation_journal_title=Nature; citation_volume=442; citation_number=62; citation_pages=772-778; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Practical strategies for discovering regulatory DNA sequence motifs.; citation_author=KD MacIsaac; citation_author=E Fraenkel; citation_journal_title=PLoS Comput Biol; citation_volume=2; citation_number=63; citation_pages=e36; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The control of the false discovery rate in multiple testing under dependency.; citation_author=Y Benjamini; citation_author=D Yekutieli; citation_journal_title=The Annals of Statistics; citation_volume=29; citation_number=64; citation_pages=1165-1188; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Inactivation of myocardin and p16 during malignant transformation contributes to a differentiation defect.; citation_author=M Milyavsky; citation_author=I Shats; citation_author=A Cholostoy; citation_author=R Brosh; citation_author=Y Buganim; citation_journal_title=Cancer Cell; citation_volume=11; citation_number=65; citation_pages=133-146; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000807" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Wide-Scale Analysis of Human Functional Transcription Factor Binding Reveals a Strong Bias towards the Transcription Start Site"/>
    <meta name="twitter:description" content="BackgroundTranscription factors (TF) regulate expression by binding to specific DNA sequences. A binding event is functional when it affects gene expression. Functionality of a binding site is reflected in conservation of the binding sequence during evolution and in over represented binding in gene groups with coherent biological functions. Functionality is governed by several parameters such as the TF-DNA binding strength, distance of the binding site from the transcription start site (TSS), DNA packing, and more. Understanding how these parameters control functionality of different TFs in different biological contexts is a must for identifying functional TF binding sites and for understanding regulation of transcription.Methodology/Principal FindingsWe introduce a novel method to screen the promoters of a set of genes with shared biological function (obtained from the functional Gene Ontology (GO) classification) against a precompiled library of motifs, and find those motifs which are statistically over-represented in the gene set. More than 8000 human (and 23,000 mouse) genes, were assigned to one of 134 GO sets. Their promoters were searched (from 200 bp downstream to 1000 bp upstream the TSS) for 414 known DNA motifs. We optimized the sequence similarity score threshold, independently for every location window, taking into account nucleotide heterogeneity along the promoters of the target genes. The method, combined with binding sequence and location conservation between human and mouse, identifies with high probability functional binding sites for groups of functionally-related genes. We found many location-sensitive functional binding events and showed that they clustered close to the TSS. Our method and findings were tested experimentally.Conclusions/SignificanceWe identified reliably functional TF binding sites. This is an essential step towards constructing regulatory networks. The promoter region proximal to the TSS is of central importance for regulation of transcription in human and mouse, just as it is in bacteria and yeast."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000807.g006"/>

  <meta property="og:title" content="Wide-Scale Analysis of Human Functional Transcription Factor Binding Reveals a Strong Bias towards the Transcription Start Site" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000807" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=894'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6318'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9578&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000807">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000807" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000807&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Wide-Scale Analysis of Human Functional Transcription Factor Binding Reveals a Strong Bias towards the Transcription Start Site
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yuval Tabach, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel, 
                  Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ran Brosh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yossi Buganim, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anat Reiner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Or Zuk, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Assif Yitzhaky, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark Koudritsky, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Varda Rotter, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Cell Biology, The Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eytan Domany
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:eytan.domany@weizmann.ac.il">eytan.domany@weizmann.ac.il</a></p>

                <p>Affiliation:
                  Department of Physics of Complex Systems, The Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 29, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000807</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000807" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000807" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000807" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000807" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000807-g001" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000807-g002" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000807-g003" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000807-g004" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000807-g005" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000807-g006" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000807">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000807" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Transcription factors (TF) regulate expression by binding to specific DNA sequences. A binding event is functional when it affects gene expression. Functionality of a binding site is reflected in conservation of the binding sequence during evolution and in over represented binding in gene groups with coherent biological functions. Functionality is governed by several parameters such as the TF-DNA binding strength, distance of the binding site from the transcription start site (TSS), DNA packing, and more. Understanding how these parameters control functionality of different TFs in different biological contexts is a must for identifying functional TF binding sites and for understanding regulation of transcription.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We introduce a novel method to screen the promoters of a set of genes with shared biological function (obtained from the functional Gene Ontology (GO) classification) against a precompiled library of motifs, and find those motifs which are statistically over-represented in the gene set. More than 8000 human (and 23,000 mouse) genes, were assigned to one of 134 GO sets. Their promoters were searched (from 200 bp downstream to 1000 bp upstream the TSS) for 414 known DNA motifs. We optimized the sequence similarity score threshold, independently for every location window, taking into account nucleotide heterogeneity along the promoters of the target genes. The method, combined with binding sequence and location conservation between human and mouse, identifies with high probability functional binding sites for groups of functionally-related genes. We found many location-sensitive functional binding events and showed that they clustered close to the TSS. Our method and findings were tested experimentally.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We identified reliably functional TF binding sites. This is an essential step towards constructing regulatory networks. The promoter region proximal to the TSS is of central importance for regulation of transcription in human and mouse, just as it is in bacteria and yeast.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Tabach Y, Brosh R, Buganim Y, Reiner A, Zuk O, et al.  (2007) Wide-Scale Analysis of Human Functional Transcription Factor Binding Reveals a Strong Bias towards the Transcription Start Site. PLoS ONE 2(8):
          e807.
            doi:10.1371/journal.pone.0000807</p><p><strong>Academic Editor: </strong>Sui Huang, Children's Hospital Boston, United States of America</p><p><strong>Received:</strong> April 12, 2007; <strong>Accepted:</strong> July 24, 2007; <strong>Published:</strong> August 29, 2007</p><p><strong>Copyright:</strong> © 2007 Tabach et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was partially supported by grants from the Ridgefield Foundation, the NIH grant #5 P01 CA 65930-06 and by EC FP6 funding.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Understanding the manner in which transcription is regulated is one of the central challenges of the post-genomic era. Since the most basic regulatory mechanism acts via binding of TFs to the promoter regions of the genes, considerable efforts have been devoted to elucidating TF binding to DNA <a href="#pone.0000807-Elnitski1">[1]</a>, <a href="#pone.0000807-Aerts1">[2]</a>, <a href="#pone.0000807-Gerland1">[3]</a> In spite of very significant advances that were made during the past years, leading to development of novel experimental and theoretical methods to measure and analyze gene expression <a href="#pone.0000807-Allison1">[4]</a> as well as TF binding (see reviews in <a href="#pone.0000807-Elnitski1">[1]</a>, <a href="#pone.0000807-AmbesiImpiombato1">[5]</a>, <a href="#pone.0000807-Blais1">[6]</a>), several basic questions remain largely unanswered. One of these concerns the extent to which a TF's functionality depends on the location of it's binding site (BS), and another-the relative regulatory importance of different regions of the promoters of higher organisms.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>This work makes two distinct but closely related contributions to our understanding of regulation of expression and TF binding. <em>First</em>, we study the manner in which the location of a binding site (BS) affects its functionality. One extreme situation is that of a fully location–specific TF, one that affects transcription of its various target genes only if it is bound within a narrow range of distances from the respective TSSs. On the other extreme there may be TFs whose functionality is nearly independent of the BS location, which may vary widely from target to target (most enhancers belong to this category). Knowing whether a TF belongs to the first or second kind provides important information on the mechanism by which it affects transcription (assignment of a TF to one of these classes may depend on the biological context in which it acts). The first finding we report here is that many TFs are functional when bound within a well defined location window. We also studied the distribution of the BSs of all such location-sensitive TFs, and discovered that the location-specific functional binding sites are much more likely to be observed in the interval (100,-200), i.e. between 100 bp downstream to the TSS and about 200 bp upstream, than in any window of similar size further upstream (we searched up to 1000 bp). The <em>second</em> component of the paper is methodological: we present a novel way of searching for functional transcription factor BSs on promoter sequences, in a position-dependent manner. Our method is sensitive enough to reveal the location bias described above. We now proceed to define the concepts used, clarify the question, explain the method and describe how it yields the biological findings.</p>

<h4>Functional binding: biological definition</h4>
<a id="article1.body1.sec1.sec1.p1" name="article1.body1.sec1.sec1.p1"></a><p>A TF may bind to a site on the DNA but this binding event is not necessarily functional. The ideal, biologically sound definition of a functional binding event is that the TF has been shown to bind at the site on a gene's promoter, <em>and</em> this binding has been demonstrated experimentally to affect the level of transcription of the gene. Clearly, only functional binding is relevant for understanding regulation of transcription. Such experimental data are, however, scarce and difficult to obtain on a scale that covers all genes and all known transcription factors (and our work poses questions on this scale, as explained below). In principle, had we known all functional BSs, as defined above, for every TF and every gene, we could have provided a definitive answer to the question posed above, regarding the positional distribution of functional BSs. In human the number of known TFs is on the scale of a thousand and the number of genes runs in tens of thousands; hence there are tens of millions of possible TF-promoter pairs. Measuring reliably binding events of all possible TF-promoter pairs is a tall order, but may be forthcoming <a href="#pone.0000807-Ren1">[7]</a> in a few years. However, establishing <em>functionality</em> for each bound TF-promoter combination, or even obtaining a large enough unbiased sampling of such pairs, is clearly unrealistic. For this reason we work with a modified “operational” definition of functional binding, that can be used within a computation-based attempt to identify functional BSs.</p>


<h4>Computational approach</h4>
<a id="article1.body1.sec1.sec2.p1" name="article1.body1.sec1.sec2.p1"></a><p>Binding of a TF to DNA at a particular location (BS) is influenced by a variety of factors that affect the energetics of the bound TF-DNA complex. The first factor is the binding sequence–i.e. the sequence of bases that appear at a putative BS. Another factor is the structure (e.g. bending) of the DNA at the BS; obviously epigenetic changes (such as methylation of nucleotides in or near the BS) are very important as well. The proximity of nucleosomes and the methylation, phosphorylation or acethylation states of their constituent histones also affect the chemical environment “seen” by the TF (reviewed in:<a href="#pone.0000807-Lam1">[8]</a> ) The same holds for other proteins that may be bound near the BS, whose presence can either inhibit binding of the TF on which we focus, or enhance binding by modifying it's conformation, or that of the DNA.</p>
<a id="article1.body1.sec1.sec2.p2" name="article1.body1.sec1.sec2.p2"></a><p>Intense recent computational efforts were devoted to identification of BSs for various TFs <a href="#pone.0000807-Elnitski1">[1]</a>, <a href="#pone.0000807-Aerts1">[2]</a>, <a href="#pone.0000807-Elkon1">[9]</a>, <a href="#pone.0000807-HoSui1">[10]</a>, <a href="#pone.0000807-Liu1">[11]</a>. These works focused largely on few or a single one of the factors listed above, namely recognizing BSs on the basis of their sequences. These studies relied on information accumulated from various experiments, that reported binding of the TF of interest to DNA segments that have been sequenced <a href="#pone.0000807-Frech1">[12]</a>. On the basis of simple statistical analysis of the frequency of appearance of various sequences among observed binding events, a position specific score matrix (PSSM) is constructed for the TF, that allows an easy calculation (see <a href="#s4">Methods</a>) of a binding score for every DNA sequence. Since sometimes a set of similar sequences is identified as binding a yet unidentified TF, in general each PSSM is associated with a motif <em>M</em> (rather than with a TF). A sequence whose score exceeds a threshold is defined as a “hit”, e.g. as corresponding to a binding event. Since sequence is only one of the several relevant factors that govern binding, the accuracy of a PSSM-based method to predict TF binding is limited.</p>
<a id="article1.body1.sec1.sec2.p3" name="article1.body1.sec1.sec2.p3"></a><p>Predicting <em>functional</em> binding is an even more difficult problem. A TF may bind to a site on the DNA (one that has a high-scoring sequence), without this “stray” binding having any effect on transcription. Since the biological definition of functional binding can be tested only experimentally, for a bioinformatic analysis we need to define functional binding in a way that can be used <em>in silico</em>.</p>


<h4>Functional binding: operational definition</h4>
<a id="article1.body1.sec1.sec3.p1" name="article1.body1.sec1.sec3.p1"></a><p>To define functional binding in a computation-based manner we use one or both of two criteria that go beyond the sequence information contained in the PSSM; conservation of the binding motif between species and it's co-occurrence on the promoters of functionally related genes.</p>
<a id="article1.body1.sec1.sec3.p2" name="article1.body1.sec1.sec3.p2"></a><p><em>The conservation-based approach</em> is to compare the regulatory regions of orthologous genes in different species to identify functionally conserved motifs <a href="#pone.0000807-McCue1">[13]</a>, <a href="#pone.0000807-Rajewsky1">[14]</a>, <a href="#pone.0000807-McGuire1">[15]</a>, <a href="#pone.0000807-Xie1">[16]</a>, <a href="#pone.0000807-Kellis1">[17]</a>, <a href="#pone.0000807-Cliften1">[18]</a>. If a TF is bound at a certain location and the binding is not functional, the probability for conservation of the binding sites across several species is low. Using this method, Xie et al <a href="#pone.0000807-Xie1">[16]</a> showed that BS conserved in four mammalian species exhibited a strong location bias relative to the TSS. However, for a genomic region to be functional, evolutionary conservation is neither necessary <a href="#pone.0000807-Plessy1">[19]</a>, <a href="#pone.0000807-Emberly1">[20]</a> nor sufficient <a href="#pone.0000807-Nobrega1">[21]</a>, <a href="#pone.0000807-Clark1">[22]</a>. Furthermore, their method is restricted to genes that have orthologues over four species, and have TF binding sites with conserved consensus sequences over all these species. For these reasons conservation is a hard test and imposing it is believed to lead to many false negatives (missed functional BSs). For example, their failure to identify the TATA box (see <a href="#pone-0000807-g001">Fig 1D</a>) as a conserved motif at a specific distance from the TSS is indicative of the limitations of relying on conservation alone.</p>
<div class="figure" id="pone-0000807-g001"><div class="img"><a name="pone-0000807-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The TATA box score and GC content versus location on the promoters.</span></strong></p><a id="article1.body1.sec1.sec3.fig1.caption1.p1" name="article1.body1.sec1.sec3.fig1.caption1.p1"></a><p>A. Each 11bp-long sequence, located in the interval [+200, −1000] bp with respect to the TSS on the promoters of 22,000 human genes, was scored using the PSSM of the TATA box. Scores above −10.4 were identified as hits and marked by a point whose horizontal coordinate represents the location of the hit and the vertical coordinate represents the value of its score. The high density of hits at about 25 and 35bp upstream from the TSS identifies these locations as most likely to be functionally relevant. The 3 colored dotted lines represent 3 different thresholds (T) and the numbers next to each one of them are the % of TATA BS that were found in the location window [−100,TSS]. B. The GC content as a function of location in the interval [+200, −1000] bp with respect to the TSS. The vertical axis represents the percentage of promoters of 22,000 human genes that have a G or a C at each location. Pronounced dips are observed at the TATA box and at the TSS. C. Distribution of identified hits obtained for various values of the threshold, marked by red, green and blue horizontal lines on <a href="#pone-0000807-g001">Fig 1A</a>, with the % of hits found within the proximal peak also given there, for each threshold. As the threshold increases, the overall number of hits decreases, but the relative weight of the proximal peak increases from 8% (for threshold <em>T = </em>−10.6) to 27% (for <em>T</em> = −6.5). D. Our method identified BSs on 55 out of 81 promoters known to have proximal TATA BSs (see <a href="#pone.0000807.s007">text</a>), in the window [TSS, −100], and none in any of the other location windows (red dot). The threshold selected by our method was −10.27; if this threshold is used uniformly in the entire [200, −1000] interval, about 250 hits are found outside the first upstream interval. The black dots denote the number of hits in and outside the first window, with the numbers next to each point indicating the value of the uniform threshold used. The blue point shows the hits found by Xie et al, demonstrating that their conservation-based method is also effective in eliminating false positives, but has a much larger number of false negatives than our method.</p>
<span>doi:10.1371/journal.pone.0000807.g001</span></div><a id="article1.body1.sec1.sec3.p3" name="article1.body1.sec1.sec3.p3"></a><p><em>The other criterion</em> we use is based on the premise or hypothesis <a href="#pone.0000807-DeRisi1">[23]</a>, <a href="#pone.0000807-Hughes1">[24]</a> that functionally related genes tend to be co-regulated, and hence if BSs of a TF are highly represented on the promoters of such a group of genes, there is a good chance that this TF is involved in their regulation, and hence these BSs are functional. In order to implement this criterion in a genome-wide study, we have to produce lists of functionally related and co-regulated genes and to test, in a statistically reliable manner, for over-representation of BSs of a TF on their promoters. Such a group of genes may be obtained, for example, from cluster analysis of expression data, which yields groups of co-expressed genes. It is standard procedure <a href="#pone.0000807-Roth1">[25]</a>, <a href="#pone.0000807-Haverty1">[26]</a>, <a href="#pone.0000807-Zheng1">[27]</a>, <a href="#pone.0000807-Tabach1">[28]</a> to perform on such a group of genes a Gene Set Enrichment Analysis (GSEA) <a href="#pone.0000807-Subramanian1">[29]</a> to search for functional Gene Ontology (GO) <a href="#pone.0000807-Ashburner1">[30]</a> classes whose member genes are over-represented in the expression-based gene group. This way one produces a list of genes with the desired property–they are co-expressed and functionally related, and hence are likely to be co-regulated. If a high fraction of these genes' promoters contains BSs of a TF, it is a likely regulator of these genes. The problem with implementing this standard approach in our context is that expression data are available only for a limited number of special conditions, cell types and tissues, and are mostly biased towards several pathologies and diseases. As a result, there are no available experiments and corresponding co-expression lists for all genes and all functional groups. In order to overcome this problem and generate comprehensive and less biased gene lists, <em>we make a key assumption: the genes that belong to a functional GO class</em> as our putative co-regulated group <em>G</em>. We introduce and implement a novel algorithm that searches for binding sites on the promoters of the genes of such a GO-based group. If the genes that do have a particular binding site are over-represented in the group, this is viewed <a href="#pone.0000807-Li1">[31]</a> as indicative of functionality of the corresponding BSs.</p>


<h4>Outline of our method and main differences from existing ones</h4>
<a id="article1.body1.sec1.sec4.p1" name="article1.body1.sec1.sec4.p1"></a><p>Existing techniques <a href="#pone.0000807-Elkon1">[9]</a>, <a href="#pone.0000807-Haverty1">[26]</a>, <a href="#pone.0000807-Hertz1">[32]</a>, <a href="#pone.0000807-Kel1">[33]</a>, <a href="#pone.0000807-Frith1">[34]</a> search for BSs by scanning one by one the promoters of various genes and comparing the PSSM-based score of every sequence to a threshold, which is fixed for the entire genome. We follow a different strategy: we execute the search for BSs associated with a motif <em>M simultaneously</em> for a group <em>G</em> of (probably) co-regulated genes, but the analysis is carried out <em>separately</em> for each location window <em>W,</em> obtained by dividing the promoters to short (100, 200 etc bp long) windows. A threshold value <em>T</em> is set, and we count the number of genes in <em>G</em> that scored a hit in <em>W</em>–i.e. a score above <em>T</em> has been recorded, using the PSSM of the tested motif <em>M</em>. Next, a hypergeometric test is used to calculate an enrichment p-value, <em>p(T,G,W,M)</em>, for the number of hits in <em>G</em>, compared to the number of hits expected in <em>W</em>, using the same <em>T</em>, for a reference group of promoters. The reference group contains all known promoter sequences of the genome. It is very important to use the same window on the promoters of the reference group, since there is a significant systematic variation of the GC content within the region on which we focus (−1000 to +200 bp). For further details, including control of the false discovery rate (FDR), see <a href="#s4">Methods</a>. Using this method, we performed a comprehensive bioinformatic analysis that encompassed promoters of the 8110 most annotated human genes, 23,400 mouse genes and the known <a href="#pone.0000807-Quandt1">[35]</a> PSSMs of 414 motifs <em>M</em>, using all functional GO groups to define the gene sets <em>G</em> that are scanned for enrichment. To reduce false positives further, we repeated the same analysis for the orthologue genes of mouse. Requiring that the same motif is overrepresented in the same location window on the mouse orthologues of the same GO-based genes, is a demanding filter <a href="#pone.0000807-Xie1">[16]</a>; hence passing it increases significantly our confidence in the validity of the hit.</p>
<a id="article1.body1.sec1.sec4.p2" name="article1.body1.sec1.sec4.p2"></a><p>Nevertheless, as stated above, the only way to <em>prove</em> functionality of a BS is by a series of careful experiments. This, unfortunately, cannot be done in the context of a wide-scale search for binding sites. The best one can do is to present a large number of supporting pieces of evidence and consistency checks; our work is to be viewed in this spirit.</p>


<h4>The biological finding and its relevance</h4>
<a id="article1.body1.sec1.sec5.p1" name="article1.body1.sec1.sec5.p1"></a><p>We found that there are many position-specific TFs, with BSs that satisfy our operational criteria of functionality only within a particular window. The most significant pattern that arises from our analysis is that these location-specific functional BSs are concentrated in a distinct region of about 300 bp that includes the TSS; the abundance of such BSs further upstream is significantly lower. The bioinformatic analysis is supplemented and supported by substantive experimental evidence from the literature on a variety of biological systems <a href="#pone.0000807-Whitfield1">[36]</a>, <a href="#pone.0000807-RashiElkeles1">[37]</a>, as well as by a direct experimental test of the dependence of the efficacy of regulation on the distance of the BS from the TSS. There is consensus in prokaryotes on the central regulatory role played by the region close to the TSS, based on evidence from studies in E.Coli <a href="#pone.0000807-Lodish1">[38]</a>. As to eukaryotes, arguments supporting the special role of the proximal region have been presented for yeast <a href="#pone.0000807-Lodish1">[38]</a>, <a href="#pone.0000807-Harbison1">[39]</a>. For higher eukaryotes such as human, the difference in terms of functionality between BS found close to the TSS and those found far has still not been fully addressed <a href="#pone.0000807-Lodish1">[38]</a>. Several papers showed positional bias towards the TSS by applying the standard approaches; however, most of them dealt with small numbers of <em>particular</em> TFs or targets genes <a href="#pone.0000807-Aerts1">[2]</a>, <a href="#pone.0000807-Elkon1">[9]</a>, <a href="#pone.0000807-Liu1">[11]</a>, <a href="#pone.0000807-Tabach1">[28]</a>, <a href="#pone.0000807-Frith1">[34]</a>. As opposed to these studies, several others suggested that remote regions of the mammalian genomes play decisive regulatory roles <a href="#pone.0000807-Pfeifer1">[40]</a>, <a href="#pone.0000807-KimuraYoshida1">[41]</a>. Howard et al <a href="#pone.0000807-Howard1">[42]</a> suggested that cis-regulatory elements exert their influence over a distance of 100kb; Pfeifer et al <a href="#pone.0000807-Pfeifer1">[40]</a> and Kimura-Yoshida et al <a href="#pone.0000807-KimuraYoshida1">[41]</a> argued that the genomic region harboring functionally important regulatory elements can stretch as far as 1Mb in both directions from the TSS, and most recent evidence identified a regulator of transcription that binds to a different chromosome <a href="#pone.0000807-Ling1">[43]</a>.</p>
<a id="article1.body1.sec1.sec5.p2" name="article1.body1.sec1.sec5.p2"></a><p>The main novelty of our findings is in their <em>generality</em>, as opposed to other works that describe positional bias and specificity of TF binding sites for <em>particular</em> TFs or targets genes <a href="#pone.0000807-Aerts1">[2]</a>, <a href="#pone.0000807-Elkon1">[9]</a>, <a href="#pone.0000807-Liu1">[11]</a>, <a href="#pone.0000807-Tabach1">[28]</a>, <a href="#pone.0000807-Frith1">[34]</a>, <a href="#pone.0000807-Carninci1">[44]</a>. The study of Xie et al <a href="#pone.0000807-Xie1">[16]</a> does make a fairly general statement that complements nicely our findings (see above). Their claims are, however, restricted to genes that have orthologues over four species, and have TF binding sites with conserved consensus sequences over all these species. We claim that positional bias is not limited to strongly conserved sequences; rather, it is a more general and much stronger effect than reported in previous work.</p>


<h4>Overview of the Results</h4>
<a id="article1.body1.sec1.sec6.p1" name="article1.body1.sec1.sec6.p1"></a><p>First, by studying the example of the TATA-box, we demonstrate the difficulties encountered when standard BS search methods are used; difficulties which our approach was designed to overcome. Next we apply the method on more than 8000 human promoters, searching for BSs of all tabulated human TF motifs, in a location and functional GO-group dependent manner. This analysis reveals two distinct classes of GO group–TF motif combinations. The BSs of one of these groups exhibit a strong location bias; analysis of experimental data on cell cycle motifs and their target genes (that belong to this group), confirm our finding of pronounced abundance of the functional BSs in the (0,-200) bp interval. Next we present an experiment that was designed and carried out to test our prediction of location bias. The final result contains re-analysis of another experiment, on NF-κB-regulated genes; here we show how location dependence of functionality explains observed differential expression of genes with similar PSSM-based scores and human-mouse conservation. Furthermore, since new NF-κB targets are also identified, the predictive power of our findings can be put to further experimental test.</p>

</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Positional bias: the TATA-box as a case study</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The TATA-box provides a very simple example that demonstrates the advantage of two main components of our method: testing many promoters simultaneously and the need for a position-dependent background. <a href="#pone-0000807-g001">Fig 1</a> demonstrates several problematic aspects of searching for BSs using standard methods, and it also presents the manner in which our method deals with them, yielding improved robustness and reliability. The PSSM of the TATA motif (see <a href="#pone.0000807.s004">Table S1</a>), as provided by MatIspector <a href="#pone.0000807-Quandt1">[35]</a>, was used to score each 11bp long window in the interval [+200, −1000] bp with respect to the TSS of a gene. This was done (see <a href="#s4">Methods</a>) for the promoters of 22,000 human genes, which had been derived from the UCSC genome browser. Scores (see <a href="#s4">Methods</a>) above −10.4 were identified as hits (i.e. putative binding sites) and marked by a point whose horizontal coordinate represents the location of the hit and the vertical coordinate-the value of its score. The threshold of −10.4 was chosen, as it gave rise to hits in 60% of the promoters, but any higher value of the threshold can be studied. The most striking feature is a high density of hits at about 25 and 35 bp upstream from the TSS, which immediately identifies these locations as most likely to be functionally relevant. Clearly, this would not have been evident had we analyzed the promoter of a single gene-which is the strategy used by nearly all BS search methods <a href="#pone.0000807-Kel1">[33]</a>, <a href="#pone.0000807-Quandt1">[35]</a>, <a href="#pone.0000807-Sharan1">[45]</a>. <em>Simultaneous viewing of a group of genes</em> is essential to highlight in a robust way locational bias, which in this case has a very clear meaning: the TATA box is <em>functional</em> only when it appears near the TSS. The second feature is the non-uniform spatial distribution of hits; their density becomes significantly higher as the distance from the TSS increases. This is due to the increase of the GC content (in human promoters) from 47% (beyond 1000 bp upstream to the TSS) to 55% (at 300 bp), up to 64% GC in the region between100 bp upstream, to 200 downstream from the TSS (<a href="#pone-0000807-g001">Fig 1B</a>). Hence AT rich motifs tend to be concentrated in the GC poor area, distant from the TSS, and as a result, the number of motifs discovered in the functionally irrelevant ranges, of 100–1000 bp upstream and TSS to 200 bp downstream, may exceed by a factor of 3 to 20 (depending on the threshold used, see <a href="#pone-0000807-g001">Fig 1A and 1C</a>) the proximal functionally relevant motifs. Whereas existing methods use the same threshold for the entire promoter <a href="#pone.0000807-Elkon1">[9]</a>, <a href="#pone.0000807-Hertz1">[32]</a>, <a href="#pone.0000807-Kel1">[33]</a> and hence generate in this case many false positives, we control this bias by setting <em>position-dependent thresholds</em>, using as our background model genome sequences from windows of the same location (e.g. TSS to −100 bp). Finally, most methods set the threshold value by a uniform criterion for all genes and motifs tested (e.g. set to get a fixed % of hits in the genome <a href="#pone.0000807-Elkon1">[9]</a>, <a href="#pone.0000807-Hertz1">[32]</a>, <a href="#pone.0000807-Kel1">[33]</a>); obviously, this will generate many false positives for rare motifs and false negatives to very common ones (such as the TATA box). Our method overcomes this: we applied it on the union of two gene sets, which are known to contain proximal TATA motifs. The first set <a href="#pone.0000807-AmirZilberstein1">[46]</a> contains 18 genes that appear in <a href="#pone-0000807-g001">Fig 1A</a> and the second <a href="#pone.0000807-Yang1">[47]</a> 66 genes (3 genes appear in both sets). By <em>determining the threshold separately</em> for each location (see below), we show that only in the first [0,-100] window we get over-representation of the TATA motif (p-value&lt;10<sup>−14</sup>), with 55 out of the 81 genes passing the threshold. As shown in <a href="#pone-0000807-g001">Fig 1D</a>, if the same value of threshold (−10.27) is used, but in a fixed, location independent manner, in addition to these 55 functional (i.e. proximal) BSs one identifies about 250 stray ones that are not at the functional location. The results obtained by the conservation based method <a href="#pone.0000807-Xie1">[16]</a> are also shown–as expected, it identifies only 12 proximal BSs, and has a relatively large number of false negatives (7 BSs were found out of the window [−100 ,TSS] ).</p>


<h4>Global analysis of Motif, GO and Location Combinations (MGLC)</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Having demonstrated the advantages of our method in a case when the association between location and functionality is very clear, we turn now to present our central results, based on a large-scale analysis. Here we use our operational definition of functionality (see <a href="#s1">Introduction</a>), to investigate the location bias of the functional BSs associated with 414 vertebrate motifs with known PSSMs <a href="#pone.0000807-Quandt1">[35]</a>, in the promoters of 8110 human and 23,400 mouse genes. As described in the Introduction, we used the GO annotation database <a href="#pone.0000807-Ashburner1">[30]</a> to assemble groups of 30 or more unique functionally related genes (smaller groups will not yield good enrichment statistics); there were 134 such different functional GO classes. In total, 7062 human genes (promoters) were assigned to at least one GO group. For each motif <em>M</em> and GO group <em>G</em>, we performed enrichment analyses <a href="#pone.0000807-Subramanian1">[29]</a>, <a href="#pone.0000807-Barash1">[48]</a> in 12 consecutive <em>location-dependent windows W,</em> each of length 100bp, from 200bp downstream from the TSS to 1000bp upstream. In addition to the 100bp long windows analysed here, we examined also windows of different sizes (of 200 bp ,300 bp, 400 bp and 700 bp), with and without overlap (<a href="#pone.0000807.s005">Table S2</a>). We found that irrespective of the window size we use, in windows that did not contain the region [100,-200] (or part of it) almost no over represented motifs were found for most of the GO groups. It is important to note that as the window size increases, a larger part of the important functional location range [100, −200] is covered by one window. Because of this reason the number of identified significant BSs increased. On the other hand, using small window gives better location specificity and better control on the GC content of the background–which reduces the number of false positives and yields better location resolution. We decide to focus on the 100bp window but the results for all the windows are presented in <a href="#pone.0000807.s005">Table S2</a>. Hence 414×134×12 = 665,712 separate tests were carried out, one for every <em>(M,G,W)</em> combination, represented as a path through the top three layers of boxes in <a href="#pone-0000807-g002">Fig 2A</a>. The score <em>threshold was determined independently</em> for every (<em>M,G,W</em>) combination, as described in Methods and in <a href="#pone-0000807-g002">Fig. 2B</a>. Controlling the False Discovery Rate <a href="#pone.0000807-Benjamini1">[49]</a> (FDR, see <a href="#s4">Methods</a>) at the level of 0.10, we identified 6331 statistically significant (<em>M,G,W)</em> combinations (134 GOs and 414 <em>M</em>'s appear in 4130 <em>M,G</em> pairs, as listed in <a href="#pone.0000807.s005">Table S2</a>). Statistical significance of such an <em>(M,G,W)</em> combination means that motif <em>M</em> is over-represented in window <em>W</em> on the promoters of the genes that belong to the human functional GO group <em>G.</em> We refer to such a <em>significant (M,G,W)</em> as a Motif, GO-group and Location Combination (MGLC).</p>
<div class="figure" id="pone-0000807-g002"><div class="img"><a name="pone-0000807-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Our method of search for BSs.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A. Schematic method of search: we used groups of human genes that belong to 134 different functional GO classes <em>G</em>, to search for each one of 414 motifs <em>M</em> in twelve 100bp-long windows <em>W</em> in the interval [+200, −1000] bp with respect to the TSS. In total, 414×134×12 = 665,712 independent analyses were carried out, one for every (<em>M,G,W</em>) combination, represented as a path through the top three layers of boxes. Each analysis produced the number of genes of <em>G</em> for which the score of <em>M</em> in the window <em>W</em> exceeded a threshold <em>T</em>. The extent of over-representation of such hits was assessed by the hypergeometric test, which compared their number with similar hits in a random selection of |<em>G</em>| out of all 8,110 human genes used. A variation of the resulting p-values <em>p (M,G,W)</em> with the threshold <em>T</em> was studied. All resulting p-values were submitted to FDR analysis, and the statistically significant (<em>M,G,W</em>) combinations were intersected with those that were found significant also in mouse. B. The dependence of –log[ <em>p (M,G,W)</em>] on the score threshold <em>T</em> is shown for 12 windows, for <em>G</em> = Mitosis GO class and <em>M</em> = NFY1. For the window [TSS, −100] bp we get a very prominent peak, for the windows in [−100, −300] bp the peak is much smaller and broader, and for the other windows it is within the noise. We find significant enrichment for the [TSS,−100] window, with the optimal threshold derived from the location of the peak.</p>
<span>doi:10.1371/journal.pone.0000807.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Since some of our assumptions may not always hold (e.g. our operational definition of functionality, the assumption that functionally related genes are likely to be co-regulated, etc), a significant fraction of these 6331 MGLCs may be false positives. As explained in the Introduction, <em>conservation</em> constitutes an independent and rather strict filter. We submitted our MGLCs to a test of co-appearance for human and mouse <a href="#pone.0000807-Xie1">[16]</a>, <a href="#pone.0000807-Wasserman1">[50]</a>, risking overshoot and producing now a list of MGLCs with many false negatives. By repeating the same analysis for the mouse orthologues of the human genes that had been tested above, we found 785 MGLCs in mouse. The lower number of MGLCs in mouse, compared to human, is probably caused by translation to orthologues; as a result, the GO groups of mouse were of 15%–30% smaller size. 333 (<em>M,G</em>) pairs (see <a href="#pone.0000807.s005">Table S2</a>) had an MGLC in human <em>and</em> mouse, albeit not necessarily at the same location windows. Each row of <a href="#pone-0000807-g003">Fig 3A</a> corresponds to one of these 333 (<em>M,G</em>) pairs, with each bar representing an MGLC. The corresponding 159 motifs (38% of the total 414 motifs in the analysis–see <a href="#pone.0000807.s005">Table S2</a>) are associated with 56 GO gene groups (42% of the 134 GO gene groups that were examined). A black bar represents a <em>conserved</em> MGLC, one that appears in human <em>and</em> mouse, at the same location window; a grey bar–an MGLC in human only <em>or</em> mouse only. There are 337 conserved (black) MGLCs, out of the maximal possible number of 785 (p-value&lt;10<sup>−320</sup>); they correspond to 114 different motifs. On the basis of their affiliation with GO groups we assign the MGLCs to one of two classes: the majority belong to a “general GO group” (see below); the distribution of these conserved MGLCs according to location windows (see <a href="#pone-0000807-g003">Fig 3B</a>) reveals a strong bias towards the TSS. The second class of conserved MGLCs are spread somewhat further upstream from the TSS (see <a href="#pone-0000807-g003">Fig 3C</a>). The division of the MGLCs into these two classes is a result of our analysis and is seen clearly in <a href="#pone-0000807-g003">Fig 3A</a>: we noticed that the MGLCs that belong to the second class are based on four functional GO groups, and the <em>(M,G)</em> pairs were grouped in <a href="#pone-0000807-g003">Fig 3A</a> accordingly.</p>
<div class="figure" id="pone-0000807-g003"><div class="img"><a name="pone-0000807-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Summary of our main results.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>A. Using FDR of 0.1 we identify the over-representation of motif <em>M</em> on promoters of genes of functional GO class <em>G</em>, in location window <em>W.</em> Each row corresponds to one of 333 (<em>M,G)</em> pairs, composed of 56 GO classes and 159 motifs. Each bar represents an MGLC: grey–either human only <em>or</em> mouse only, and black-human <em>and</em> mouse. The 56 GO clusters divide into two groups, a “general GO group” of 52 GO classes and a “transcriptional GO group” of 4 GO classes. The motifs <em>M</em> of the latter are GC-rich–see first column, where orange denotes GC content above 60%, blue–below 30% and white–between both these values. B. Number of conserved MGLCs of the General GO group, in each window. 93% are in the [−200,0] bp range. C. Same for the transcriptional GO group. The distribution is broad and peaks between 300–500bp upstream from the TSS. D. Distributions of BSs according to their GC content, plotted separately for BSs associated with the two GO groups.</p>
<span>doi:10.1371/journal.pone.0000807.g003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The very strong location bias of the MGLCs that belong to the general GO group <em>is our central finding:</em> it has been validated (for those genes that have orthologues in human, mouse and rat) using a very different method and a different database of TF binding sites (see <a href="#pone.0000807.s007">Text S1</a>, <a href="#pone.0000807.s003">Figure S3</a> and <a href="#pone.0000807.s006">Table S3</a>).</p>


<h4>The “General GO Group”; locational bias and experimental verification for cell cycle motifs</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>This group contains 177 conserved MGLCs that belong to 93% (52/56) of the significant GO groups: these include cell cycle, immune response, protein biosynthesis as well as others (see <a href="#pone.0000807.s005">Table S2</a>). This group shows a strong location bias relative to the TSS: 86% (153/177) of the conserved MGLCs were found in the two promoter windows 0–200 upstream to the TSS, while only 3% (5/177) were found in windows [−200,−1000], and the remaining 19 MGLCs are downstream to the TSS (see <a href="#pone-0000807-g003">Fig 3B</a>). As a further demonstration of our general claim of association between proximal location-specificity of the BS and functionality, we show for the cell cycle related functional class and associated TFs (for which experimental expression data <em>are</em> available), that proximal BSs are much more likely to be <em>functional</em> than those located further upstream.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We focused on the five cell-cycle associated motifs: CHR, CAAT, and three NFY-family motifs, whose MGLCs were overrepresented close to the TSS in both human and mouse. To test functionality in cell cycle, we used 3,308 genes for which we had both expression data measured during three cell cycles <a href="#pone.0000807-Whitfield1">[36]</a> in HeLa cells, as well as promoter sequences. We calculated the cell cycle periodicity index CCP <a href="#pone.0000807-Tavazoie1">[51]</a> of each of these genes from the expression data, and searched their promoter sequences, from +200 to −1000 bp from the TSS, for every one of the 5 motifs. A high CCP indicates strong cell-cycle dependence. We assembled (see <a href="#s4">Methods</a>) 5×12 = 60 groups of genes (one for each of 5 motifs <em>M</em> studied over 12 non-over lapping windows <em>W</em> of 100 bp). Each such group had <em>N<sub>MW</sub></em> genes on whose promoters we registered a hit (see <a href="#s4">Methods</a> for details) for the motif <em>M</em> in window <em>W</em>. Next, for each <em>MW</em> combination, the distribution of the CCP scores of the <em>N<sub>MW</sub></em> target genes was calculated and compared to the background distribution of the CCP scores over all the 3,308 genes studied. A sequence of these distributions (see <a href="#pone.0000807.s001">Figure S1</a>), demonstrates that the distribution of the CCP scores associated with those genes, whose CHR motif is found in the [−100,0] and [0,100] windows, differs significantly from the background distribution (of all the genes), with an enhanced number of genes with CCP scores as high as 5–10. A quantitative measure of the difference between distributions associated with each <em>MW</em> versus the background was produced by the Kolmogorov-Smirnov test, yielding a p-value for each <em>MW</em>. As seen in <a href="#pone-0000807-g004">Fig 4A</a>, 12 CCP distributions (out of the 60) had p-values below 0.05 and 9 differed from the background at 10% FDR. These include all the 5 motifs that were tested, with windows located near the TSS.</p>
<div class="figure" id="pone-0000807-g004"><div class="img"><a name="pone-0000807-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Testing our results for human cell cycle expression data.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A. For 60 groups of genes (each group defined by BS found for one of the 5 cell cycle associated motifs in one of the 12 location windows, see <a href="#pone.0000807.s007">text</a>) the distribution of the expression-based CCP scores <a href="#pone.0000807-Tavazoie1">[51]</a> was compared to a reference background distribution. The resulting 60 p-values are plotted for the different motifs versus their location window. All 5 motifs get p-values less than 0.05 and also pass at FDR of 0.10. Two windows, [0, −100] and [−100, −200] bp, are most significantly over-represented. B. Significant sequence-based hyper-geometric over-representation scores for each of the motif-window combinations. A grey bar represents a significant score (MGLC) for human <em>or</em> mouse, for at least one of the cell-cycle related GO classes. A black bar stands for an MGLC in <em>both</em> human and mouse.</p>
<span>doi:10.1371/journal.pone.0000807.g004</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>It is gratifying to note the agreement between the statistically significant MW combinations, obtained here from the expression-based CCP distributions, and the results obtained from the sequence-based hyper-geometric over-representation scores (<a href="#pone-0000807-g004">Fig 4B</a>). This agreement demonstrates (a) the reliability of our BS detection method to identify functionally important MGLCs and (b) the functional role of windows that are no more than 300bp upstream of the TSS. These findings concur with our previous observation, that the influence of cell cycle motifs on a cluster of genes obtained from cancer progression is limited to a very specific region, close to the TSS <a href="#pone.0000807-Tabach1">[28]</a>.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p><strong>The “transcriptional GO group”</strong> contains 160 MGLCs that belong to the following four GO classes: transcription, regulation of transcription, DNA-dependent regulation of transcription and development (<a href="#pone.0000807.s005">Table S2</a>). The promoters of the <em>“transcriptional GO group”</em> are enriched with motifs that belong to zinc finger proteins (like ZF5, ZF9, ZBP89, ZNF202); with EGR/nerve growth factor induced protein C (such as CKROX, EGR1, EGR3, NGFIC and WT1) and also with some other motifs, like the GC-box motifs NRF, MAZ and MYCMAX. The transcriptional GO group differs from the general GO group in several aspects. <em>Firstly</em>, its motifs are distributed over a much wider region (see <a href="#pone-0000807-g003">Fig 3C</a>), peaking at −300 to −500 bp from the TSS. <em>Secondly</em>, the GC content of its motifs (see <a href="#s4">Methods</a>) is 68% on average, with a minimum of 47%, while the motifs in the general GO group have lower GC content (see the first column of <a href="#pone-0000807-g003">Fig. 3A</a>), with an average of 48% and a minimum of 22% (<a href="#pone-0000807-g003">Fig 3D</a>). Motifs with high GC content are more likely to be found on GC-rich promoters; indeed, the GC content of promoters of the genes that comprise each of the GO classes in the transcriptional GO group is significantly higher (p&lt;0.0001) than expected within a random group of genes (see <a href="#s4">Methods</a>).We generated two “artificial” PSSMs that contain G/C or C with high probability (<a href="#pone.0000807.s004">Table S1</a>) and found that both were over-represented in human and mouse, generating 35 MGLCs in the 4 GO classes that comprise the transcriptional GO group (see <a href="#pone-0000807-g003">Fig. 3A</a> and <a href="#pone.0000807.s002">Figure S2</a>). These results suggest that the high GC content in their promoters plays an important regulatory role for genes of the transcriptional GO group. <em>Thirdly</em>, for each of the four transcriptional GO classes we found a relatively high number of significantly over-represented TF binding sites. The average of the four numbers was 18, compared to 5 motifs per class in the general GO group. Nearly all these motifs have GC content above 60%.</p>


<h4>Single gene binding sites</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Although our method identifies overrepresentation of a binding site in a GO-based functional group, we can use the information also for identification of BSs on particular promoters. We prepared a database where for every human gene in our analysis we provide the putative BSs found on it's promoter (including score, location and sequence). In order to get a conservative data base, the putative BSs for each gene were filtered, based on the parameters obtained from the GO groups to which the gene belongs. We register a hit for motif <em>M</em> of a TF on the promoter of a gene if three conditions are satisfied: first, motif <em>M</em> should be significantly over represented in at least one of the GO groups <em>G</em> to which our gene belongs (i.e. there is a corresponding MGLC). Second, the putative BS for motif <em>M</em> that was found on the promoter of our gene should be in the location window of this MGLC, and third; the score of <em>M</em> must pass the corresponding threshold value (that was determined based on the comprehensive GO group analysis). We used 2 different thresholds in this database: 1. ‘FDR’-we used the most permissive score threshold that passed FDR&lt;0.1 for the over representation analysis. 2. ‘PEAK’–the score threshold that got the best enrichment p-value was used (this score is more conservative). In addition, BSs that belong to MGLCs that are conserved also in mouse where marked as ‘motif and location enriched in murine MGLC’. The database can be downloaded from: <a href="http://www.weizmann.ac.il/complex/compphys/downloads/tabach/BS_database.zip">http://www.weizmann.ac.il/complex/compph​ys/downloads/tabach/BS_database.zip</a></p>


<h4>Direct Experimental verification of our main finding</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>We believe that if functionality of a particular TF's binding is position dependent in a given biological context, its functional BSs will be close to the TSS. Hence we predict that when a functional proximal binding site is moved away from the TSS, it will become less effective. The experiment described below was performed in order to test this prediction directly.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We designed an experiment to measure transcriptional activity of a TF as a function of the distance between its BS and the TSS. We used the motif <em>M</em> = SRF.01 which binds Myocardin, a transcription co-factor with highly potent ability to activate a set of differentiation-related genes <a href="#pone.0000807-Wang1">[52]</a>, <a href="#pone.0000807-Wang2">[53]</a>. We focused on the target gene SM22, whose expression increased significantly in response to Myocardin <a href="#pone.0000807-Pipes1">[54]</a>, and whose transcriptional activation by Myocardin depends on the integrity of specific regulatory motifs called CArG boxes, two of which are present within the SM22 promoter, at distances of ~105 bp and ~240 bp upstream of the TSS. The proximal motif was shown <a href="#pone.0000807-Wang2">[53]</a> to be significantly more important for the expression of SM22. Here we performed a controlled experiment to eliminate factors such as the possible impact of the large differences between the neighboring regulatory sequences of the two motifs and the (small albeit present) differences between the motif sequences themselves. We generated a set of four plasmids encoding a luciferase reporter gene (<em>luc)</em> under the regulation of three tandem repeats of a segment from the SM22 promoter, each of which contained either an intact (<em>wt</em>) CArG box, approximately at its center, or a mutated one (<em>m</em>). The resulting four constructs thus had identical architectures; while the intact motif is placed at different distances from the TSS (see <a href="#pone-0000807-g005">Fig 5 A</a>).</p>
<div class="figure" id="pone-0000807-g005"><div class="img"><a name="pone-0000807-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Direct experimental test of the effect of BS distance from the TSS.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A. A graphic representation of four constructs encoding a luciferase reporter (<em>luc</em>) gene under the regulation of three tandem repeats of a region from the SM22 promoter, each of which contained either an intact (<em>wt</em> in yellow ) or a mutated (<em>m</em> in red) BS of the Myocardin transcription factor. The intact motif is placed at different distances from the TSS in each construct. B. Fold activation of the luciferase construct, calculated as the ratio of promoter activity in the presence of Myocardin to the promoter activity in the absence of Myocardin. The <em>“m-m-wt-luc”</em> reporter was strongly activated by Myocardin. In contrast, the Myocardin-dependent activation of <em>“m-wt-m-luc”</em> and <em>“wt-m-m-luc”</em> was significantly weaker.</p>
<span>doi:10.1371/journal.pone.0000807.g005</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>We transfected HT1080 fibrosarcoma cells that express very low levels of Myocardin, with each of these constructs and measured their transcriptional activity (see <a href="#s4">Methods</a>). As evident in <a href="#pone-0000807-g005">Fig 5B</a>, co-transfection of Myocardin with the <em>“m-m-wt-luc”</em> reporter resulted in dramatic promoter activity increase. In contrast, the Myocardin-dependent activation of <em>“m-wt-m-luc”</em> and <em>“wt-m-m-luc”</em> constructs that also contain a single CArG box, but at more distal locations, was significantly weaker. Notably, some CArG box-independent activation of transcription exists since even the <em>“m-m-m-luc”</em> reporter was activated ~5 fold by Myocardin even though it harbors no functional CArG box. Our results indicate clearly that the transcriptional activity mediated by a regulatory DNA motif is strongly dependent on the distance of this motif from the TSS. The results of the experiment, of decreasing transcriptional effectiveness of the TF with increasing distance of the BS from the TSS, are consistent with our expectations, providing yet one more piece of consistent evidence of their validity.</p>


<h4>Transcriptional efficiency of NF-kB depends on the BS-TSS distance</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Recently Rashi-Elkeles et al <a href="#pone.0000807-RashiElkeles1">[37]</a> identified NF-kB as the most significantly overrepresented motif in the promoters of a cluster of 138 ATM-dependent genes. They focused on 11 out of these 138 genes; on those that have putative NF-kB motifs (associated with the general GO group) in both mouse and in their 11 human orthologues. Ionizing radiation was used to activate ATM, and the measured ATM-induced expression levels of these 11 genes were validated using RT–PCR. All 11 genes belong to the ATM dependent gene cluster reported by Rashi-Elkeles et al <a href="#pone.0000807-RashiElkeles1">[37]</a>, and NF-kB binding sites were identified, using PRIMA <a href="#pone.0000807-Elkon1">[9]</a>, as a significant <a href="#pone.0000807-RashiElkeles1">[37]</a> on all 11 promoters. Nevertheless, the experimental results show a wide variation of the corresponding 11 expression fold changes, ranging from 1.6 to 153 fold induction by ATM activity.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>We suggest that BS location and score information distinguishes between genes that are highly induced versus those that have been poorly induced by ATM. Using our algorithm we searched the promoters of all 138 genes for NF-kB binding sites. We found significant over representation of the NF-kB motif in both human and mouse only between the TSS to −200 bp (peaking between −50 and −150, for score threshold between −3 and −5.2; see <a href="#pone-0000807-g006">Fig 6A</a> for the results for mouse). These location bias results fully agree with a specific analysis on NF-κκB-regulated immune genes, that examine the positional distribution of NF-κκB binding sites <a href="#pone.0000807-Liu1">[11]</a></p>
<div class="figure" id="pone-0000807-g006"><div class="img"><a name="pone-0000807-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000807" data-uri="info:doi/10.1371/journal.pone.0000807.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000807.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000807.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000807.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Testing our method on ATM-induced expression data.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>A. Analysis of <em>G</em> = 138 ATM-dependent genes <a href="#pone.0000807-RashiElkeles1">[37]</a> for <em>M</em> = NF-kB.01 over representation. The dependence of –log[ <em>p (M,G,W)</em>] on the score threshold <em>T</em> is shown for 23 overlapping windows. Three upstream windows, (TSS- 100), (50–150) and (100–200) passed the hypergeometric test at an FDR of 10%. B. putative NF-kB.01 BSs of the 138 ATM-dependent genes; score (vertical axis) versus location (horizontal axis). Our algorithm identifies the location and scores that are indicative of functionality of the BS (rectangle enclosed by the red dashed line). Solid colored symbols mark genes whose expression was validated by PCR. The other genes in the box are marked with blank colored symbols, those that are likely targets of NF-kB outside the box–by X. We list next to the gene names the fold change as measured by the microarray and by PCR (in parentheses). The black dots are belong to the rest of the 138 genes,with putative BS outside the area indicative of functionality.</p>
<span>doi:10.1371/journal.pone.0000807.g006</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>In order to examine the influence of both the location and score threshold on the expression of the 138 ATM-dependent genes <a href="#pone.0000807-RashiElkeles1">[37]</a>, we examined all their putative NF-kB BSs (see <a href="#pone-0000807-g006">Fig 6B</a>). A group of BSs is clustered at high scores (−3 to −5.2) and in the location window [−200,0] (see box defined by dashed line in <a href="#pone-0000807-g006">Fig 6B</a>). The scores and location windows are almost identical to those found in our over-representation analysis. Inspection of the BSs in this region shows that this group includes those seven genes (out of the 11 marked by solid filled symbols validated by PCR) that had the highest fold induction <a href="#pone.0000807-RashiElkeles1">[37]</a>, ranging between 153 and 3.4 , whereas the 4 genes that are outside the box had fold changes between 2.6 and 1.6. With the exception of one gene, ranking the 7 genes in the box by their scores gives the same ordering as the fold induction. The p-value for finding the 7 genes with the highest fold change out of the 11 that were examined in this “location score area” is 0.003. In addition to these 7 genes, we found in the marked location-score box high scoring NF-kB BSs on the promoters of 7 more genes, marked by blank symbols. We predict that these genes are also NF-kB targets and will produce a high fold change. Csf1<a href="#pone.0000807-Yao1">[55]</a> and Nfkbiz <a href="#pone.0000807-Eto1">[56]</a> are two of these 7 genes and, indeed, both are known targets of NF-kB. Three others, Irf1, Ifngr2 and Naf1 are involved in the immune system and hence are likely NF-kB targets. These results demonstrate clearly the importance of the location of the BS, and that our method is able to differentiate between genes that are genuine targets of NF-kB on the one hand and the apparent false positives that were identified by another method.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We introduced a novel method to identify TF binding sites in a location and biological function–dependent manner. Using an operational definition of functional binding sites, we showed that for a large number of TFs, functional binding is location sensitive. These functional binding sites tend to concentrate close to the TSS–within an interval of a few hundred bp. Clearly we cannot rule out that there are functional binding sites also far from this region, but these are most probably not location dependent. These results were demonstrated for both human and mouse.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We found that binding sites that pass the simple test of scoring above a threshold, using the PSSM alone, are much more likely to be conserved across species if they are close to the TSS. They also are more likely to pass both our tests of functionality: over-representation in the promoters of a functional GO class and conservation across species.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>A possible point of criticism concerns “discovery bias”: the known transcription factor motifs used here represent probably only a small fraction of all transcription factors that exist in the human genome. Because of the ways these factors are discovered, these motifs are almost certainly biased towards proteins that bind to proximal promoter regions of genes. However, our results cannot be attributed to this bias. Simply stated, since the traditional experimental analysis of promoters typically covers regions of about 1000–2000 bp, there is no discovery bias that prefers the first 200 nucleotides upstream to the TSS over say the next window of 200–400 bp, or even 400–600, whereas our main results indicate a very strong bias to the first 200 bp, not the first 2000. Had we claimed overrepresentation of binding sites in the first 2000 bp, discovery bias would have been a valid concern.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Our main finding was obtained by a new motif-discovery method introduced here. The method is characterized by several novel features which make it both flexible and sensitive, identifying hits in a way that takes into account the location, genomic background and a group of potentially co-regulated genes. We performed a high throughput wide scale analysis, for all commonly used motif PSSMs in a large variety of biological functional classes. We found that the GO classes are divided into two groups: the “transcriptional GO group” of 1476 genes, whose promoters have relatively high GC concentrations upstream of 200bp from the TSS (giving rise to a broad distribution of BSs, peaking at ~ −400 bp), and a “general GO group”, of 6646 genes (with BSs concentrated in the first 200bp upstream of the TSS). The information about the different GO groups and the motifs that were found to be over represented in each are listed in <a href="#pone.0000807.s005">Table S2</a></p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>These results were substantiated by showing agreement between our sequence-based analysis with an expression based identification of BSs that are functional in cell cycle regulation. Furthermore, we were able to show, using only NF-κB BS locations and scores, that NF-κB BSs tend to lose their regulatory effect when found at a distance beyond several hundred bp from the TSS, and demonstrated that BS location and score are predictive of the relative expression level of the activated gene. The principal role played by the proximal region to the TSS emerged also from analysis of BSs that were classified as conserved over human, mouse and rat. Finally, we carried out an experiment whose aim was to test our prediction in the most direct way. By measuring the transcriptional activity of the TF Myocardin when its binding site is placed at several distances from the TSS on the promoter of the SM22 gene, we demonstrated that the effect of the TF on transcription of the target decreases significantly as the distance of its BS from the TSS increases. Whilst the result of this experiment cannot be viewed as a proof of our very general claims, it is consistent with a necessary condition for our main finding, and hence provides a robust opportunity to falsify our conclusions. Had we found that Myocardin's transcriptional activity is position dependent, but peaks far (say 5–600 bp downstream) from the TSS, this would have contradicted our prediction. Several recent diverse studies strongly support our results that the first 200 bp upstream to the TSS are different, especially in terms of functionality; e.g. the conservation of promoters and BS between species <a href="#pone.0000807-Xie1">[16]</a>, <a href="#pone.0000807-Carninci2">[57]</a>; the high percentage of functional promoter polymorphisms found within the first 100 bp <a href="#pone.0000807-Buckland1">[58]</a>; the central importance of the first 300 bp upstream to the TSS for core promoter activity <a href="#pone.0000807-Cooper1">[59]</a>; pronounced depletion of repeatable element concentration near the TSS <a href="#pone.0000807-Carninci2">[57]</a>, <a href="#pone.0000807-MarinoRamirez1">[60]</a>; identification of about 200 bp long nucleosome-free regions on either side of the TSS in yeast <a href="#pone.0000807-Yuan1">[61]</a>, <a href="#pone.0000807-Segal1">[62]</a> and in higher Eukaryotic genomes <a href="#pone.0000807-Segal1">[62]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>TFs are proteins that in order to be functional should be in a physical, contact-based interaction with the transcriptional machinery they regulate, which must have components near the TSS. Therefore, the simplest way to produce such physical interactions is to bind at a location near the TSS, and our findings are consistent with this “Occam's razor”. This does not imply, however, that <em>all</em> functionally important BSs appear near the TSS. Regulatory regions, such as enhancers, that extend many thousands of bp from the TSS may create complicated stable 3D structures that influence expression by physical interaction with the proximal promoter. Our analyses do indicate, however, that the most prevalent transcriptionally functional mechanisms involve binding in the vicinity of the TSS.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Identifying promoter sequences</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>DNA sequences upstream of human ORFs were downloaded from the GoldenPath server at UCSC <a href="http://genome.ucsc.edu/goldenPath/hg16/bigZips/">http://genome.ucsc.edu/goldenPath/hg16/b​igZips/</a>. Putative regulatory regions for the different genes were obtained (+200 to −1000 bp with respect to the TSS). We used as our working set of genes the GeneChip Human Genome Focus Array (Affymetrix, Santa Clara, CA, USA) that represents over 8,500 verified human sequences from the NCBI RefSeq database. Focus chip genes are very suitable for our analysis since they are highly annotated, have low redundancy and their putative promoters are assigned (We identified promoters for 8,110 genes out of the 8,500 of the Human genome Focus chip).</p>


<h4>Scoring a Binding Site</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The score <em>S (M; {n})</em> of a putative binding sequence of <em>L</em> nucleotides, (n<em><sub>1</sub>,n<sub>2</sub></em>,…,<em>n<sub>L</sub></em>), for a particular motif <em>M</em> is the log-likelihood score calculated from the motif's PSSM. The entries <em>m<sub>i</sub> (n) of</em> the PSSM are the probabilities of finding nucleotide <em>n<sub>i</sub> = C,G,A,T</em> at site <em>i</em> :<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000807.e001&amp;representation=PNG" class="inline-graphic"></span><br></p>


<h4>Determining the threshold <em>T(M,G,W)</em></h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>set some low initial value for <em>T</em> for the threshold, and count <em>N(M,G,W)</em>, the number of genes (of the GO group <em>G</em>) for which motif <em>M</em> registered a hit (score&gt;<em>T</em>) in the window <em>W</em>. Now repeat the process for the same motif, window and <em>T,</em> but use the promoters of all the 8,110 human genes to count the number of hits in the same window, <em>N<sub>0</sub>(M,W).</em> Now we determine (by the hypergeometric test <a href="#pone.0000807-Barash1">[48]</a>, <a href="#pone.0000807-MacIsaac1">[63]</a> <em>p(M,G,W)</em>, the p-value for over-representation, i.e. the probability of finding more than <em>N(M,G,W)</em> hits for motif <em>M</em> within window <em>W</em> for |<em>G</em>| genes selected at random from the 8,110. Finally, increase <em>T (</em>steps of 0.1) and repeat the procedure; plots of <em>p(T)</em> are shown in <a href="#pone-0000807-g002">Fig 2B</a> for <em>G = </em>mitosis GO group and the motif <em>M = </em>NFY1, for all 12 windows. For each of the α = 1,2, …665,712 (<em>M,G,W)</em> combinations we choose as the corresponding threshold <em>T<sub>α</sub></em> the value of <em>T</em> which gives the largest enrichment, i.e. maximal -log <em>p</em><sub>α</sub><em>(T)</em>. Evidently, for the window (TSS-100bp upstream) we get a clear prominent peak, for the next two windows (−100bp–−300bp) the peak is much smaller and broader, and for all the more distal windows it is within the noise. This way we get the optimal thresholds and corresponding p-values, <em>p</em><sub>α</sub><em>(T<sub>α</sub>),</em> as well as the full set of <em>p</em><sub>α</sub><em>(T<sub>α</sub>),</em> for each <em>M,G,W</em>. On the average, 500 different values of <em>T</em> were used for each <em>M.</em></p>


<h4>Controlling the False Discovery Rate</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We assembled the full set of values <em>p</em><sub>α</sub><em>(T)</em> and applied on it the FDR controlling procedure <a href="#pone.0000807-Benjamini1">[49]</a>. The test-statistics corresponding to all the tests cannot be assumed independent of each other, due to several aspects: First, there is an overlap in the genes assembling the different GOs such that two different GO could contain almost the same genes. Second, each score S comprises all the binding sites equal to S or better. As a result, the score S contains all binding sites obtained using more conserved scores. Third, there is spatial correlation among neighboring motifs that are tested with regard to the distance from the TSS. Forth, different PSSM could be very similar and by that to assign the same promoter sequence as a “hit”. Since all these imply common biological factor affecting the test, these types of dependencies are potential causes of positive correlation between the test-statistics. The procedure we applied has been shown to control the FDR also for positively correlated test-statistics <a href="#pone.0000807-Benjamini2">[64]</a>.</p>


<h4>Assigning genes to an (<em>M,W</em>) group to test the CCP score distribution</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>A hit was recorded for gene <em>g</em> when the score of <em>M</em> exceeded a threshold. We described above how the threshold <em>T (M,G,W)</em> was derived. For each gene we have to identify the proper <em>G</em> to be used. For genes that do not belong to any cell cycle related GO class we used the <em>T</em> that was calculated for the cell cycle GO class. If <em>g</em> belongs to a single cell-cycle related GO class, we use this as our <em>G.</em> For genes that belong to more than one cell-cycle related GO class we use the one that was most enriched in <em>M</em>.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p><strong>The GC content</strong> <em>of a motif</em> was obtained for a PSSM by calculating for each position the probability to get G or C, and averaging it over the L positions of the PSSM. <em>For a GO class</em> of <em>n</em> genes we calculate the percent (P) of G and C along the <em>n</em> promoters. The p-value for being GC rich was obtained by selecting randomly, 10000 times, groups of <em>n</em> genes and calculating the fraction of such groups that had GC percentage higher than P.</p>


<h4>Construction of reporters for Myocardin from SM22 promoter</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>SM22-luciferase reporter constructs were generated by cloning a genomic fragment of the SM22 gene spanning from 166 to −57 bp relative to the TSS into a pGL3 luciferase reporter vector (Promega). Briefly, the genomic region was amplified from human genomic DNA using the primers 5′ agcatgcagagaatgtctcg 3′ and 5′ acagacaggatggggcgctg 3′ and ligated into pGEM-T easy vector (Promega). Recognition sequences for either BglII, SmaI or NheI restriction enzymes were added to each primer at the 5′ end for the following steps. The Myocardin BS, known as CArG box within the cloned SM22 region, was mutated using QuikChange Site–Directed Mutagenesis kit (Stratagene) and the primers 5′ ggtgtCctttcccGaaTtCtggagcc 3′ and 5′ ggctccaGaAttCgggaaagacacc 3′ (Capital letters designate mutated base pairs). The SM22 fragments that contained either an intact Myocardin motif or one which was mutated were then cloned in three consecutive steps into pGL3 vector using the BglII, SmaI and NheI restriction enzymes. Four final luciferase reporters were constructed, all of which contained three tandem repeats of the ~220bp of SM22 regulatory region upstream of the luciferase coding sequence, but each harboring an intact or mutated Myocardin motif in a different order or distances from the luciferase TSS.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>The transcriptional activity of these constructs was measured in the HT1080 fibrosarcoma cell line, in which Myocardin is not expressed <a href="#pone.0000807-Milyavsky1">[65]</a>. The reporters were transiently transfected into these cells either in the absence or the presence of a small amount of a Myocardin expression vector. To control for the transfection efficiency in each sample, the activity of a cotransfected construct coding for the Renilla-luciferase under the regulation of a CMV promoter was measured.</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p><strong>Transfections and Reporter Assays</strong> HT1080 fibrosarcoma cells were plated at 2×10<sup>4</sup> cells/well in a 24-well plate, 24 hours prior to transfection. Cells were transfected using FuGene transfection reagent (Roche), with 500ng/well of luciferase reporter construct, 5 ng/well of CMV-Renilla expression vector for normalization of transfection efficiency, and either with or without 2.5ng/well pcDNA3 expression plasmid coding for the mouse Myocardin gene (E.Olson). Cell extracts were prepared 24 hours following transfection Luciferase and Renilla activities were determined using commercial reagents and procedures (Promega). All experiments were conducted in triplicate.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000807"></div><a name="pone.0000807.s001" id="pone.0000807.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000807.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Distributions of the CCP scores of 12 groups of genes that contain the CHR binding motif in their promoters, at 12 different location windows (of 100bp) with respect to the TSS. The black curve is our background: the CCP score distribution of all the genes in the experiment(Whitfield et al. 2002).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000807.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(2.60 MB TIF)</p>
<a name="pone.0000807.s002" id="pone.0000807.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000807.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Over representation of GC and C rich “artificial” motifs The hits found in various windows for two “artificial” PSSMs that contain G/C or C with high probability (<a href="#pone.0000807.s004">Table S1</a>) were over represented in human and mouse. MGLC analysis was done for the different windows and for all GO classes (<a href="#pone.0000807.s005">Table S2</a>) as explained in the <a href="#pone.0000807.s007">text</a>. Thirty five MGLCs were found, all of which are associated with one of the 4 GO classes that comprise the transcriptional GO group.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000807.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(4.23 MB TIF)</p>
<a name="pone.0000807.s003" id="pone.0000807.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000807.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Location distribution of conserved BSs, with high similarity in human, mouse and rat. Location bias analysis of conserved BSs, with high similarity in human, mouse and rat, that were downloaded from the UCSC browser<a href="#pone.0000807-Milyavsky1">[65]</a> and are based on PSSMs obtained from the Transfac Matrix Database v8.3 created by Biobase [66] . The motifs were aligned according to their distance from the closest TSS and the histogram of these distances was plotted. The total number of conserved binding sites of these motifs is 39829; 11829 (30%) of these were found in the first 200bp, compared to about 4% found in each of the other 200bp long windows between 400bp to 3000bp upstream.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000807.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.51 MB TIF)</p>
<a name="pone.0000807.s004" id="pone.0000807.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000807.s004">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">PSSMs used in our work: TATA PSSM provided by MatIspector (Quandt et al. 1995), and two artificial motifs that were generated by us: poly C motif and GC rich motif. The rows are the different nucleotides; the columns represent the nucleotides' position j in the motif. Each entry mj (n) is the probability to find nucleotide n at position j.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000807.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000807.s005" id="pone.0000807.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000807.s005">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Full results for over-representation of binding sites. Comprehensive study of the appearances of 414 vertebrate BSs with known PSSMs <a href="#pone.0000807-Frith1">[34]</a> in the promoters of 8110 human and 23,400 mouse genes. We used groups of human genes that belong to 134 different functional GO classes G, to search for each one of 414 motifs M in twelve 100bp-long windows W in the interval [+200, −1000] bp with respect to the TSS. We examined windows of different sizes (200bp ,300bp, 400bp and 700bp), with and without overlap between the windows. In this paper we focous mainly on the analysis of the 100bp window since its gave us better location specificity and better control on the background GC content. In total, for the 100bp windows, 414×134×12 = 665,712 independent analyses were carried out, one for every (M,G,W) combination. The analysis was carried out for human and mouse, and the statistically significant (M,G,W) combinations are shown. Each row represents an (M,G) pair (first two columns) with a statistically significant over-representation score (passed FDR of 0.10-see <a href="#pone.0000807.s007">text</a>), in at least one location window in Human (grey with the letter H) or Mouse (grey with the letter M) or in both (black box). The BS symbol and its GC content are also given.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000807.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.87 MB PDF)</p>
<a name="pone.0000807.s006" id="pone.0000807.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000807.s006">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Analysis of conserved BS in human mouse and rat. 39829 conserved BS in human mouse and rat from 409 Transfac PSSMs (v8.3 created by Biobase) (Wingender et al. 2000) were downloaded from UCSC browser (Kent et al. 2002). For each motif (first column) the number of BS found in the first 3000bp from the closest downstream TSS is given in the second column, the number of BSs found in the first 200bp upstream of the TSS are presented in the third column and the corresponding percentage and P-value for this over-representation (derived using the binomial distribution) are given in the fourth and fifth columns</p>
<p class="siDoi">doi:10.1371/journal.pone.0000807.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0000807.s007" id="pone.0000807.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000807.s007">Text S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000807.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank R. Elkon and Y.Groner for most useful comments and a reviewer for most constructive and helpful criticism and suggestions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: VR ED YT. Performed the experiments: YT. Analyzed the data: YT. Contributed reagents/materials/analysis tools: AY OZ AR MK. Wrote the paper: ED YT. Other: Performed the biological experiment: RB. Conceived and designed the biological experiment: RB YB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000807-Elnitski1" id="pone.0000807-Elnitski1"></a>Elnitski L, Jin VX, Farnham PJ, Jones SJ (2006) Locating mammalian transcription factor binding sites: a survey of computational and experimental techniques. Genome Res  16: 1455–1464.  <ul class="find" data-citedArticleID="984144" data-doi="10.1101/gr.4140006"><li><a href="http://dx.doi.org/10.1101/gr.4140006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Locating+mammalian+transcription+factor+binding+sites%3A+a+survey+of+computational+and+experimental+techniques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Locating+mammalian+transcription+factor+binding+sites%3A+a+survey+of+computational+and+experimental+techniques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000807-Aerts1" id="pone.0000807-Aerts1"></a>Aerts S, Thijs G, Coessens B, Staes M, Moreau Y, et al.  (2003) Toucan: deciphering the cis-regulatory logic of coregulated genes. Nucleic Acids Res  31: 1753–1764.  <ul class="find" data-citedArticleID="984110" data-doi="10.1093/nar/gkg268"><li><a href="http://dx.doi.org/10.1093/nar/gkg268" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toucan%3A+deciphering+the+cis-regulatory+logic+of+coregulated+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toucan%3A+deciphering+the+cis-regulatory+logic+of+coregulated+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000807-Gerland1" id="pone.0000807-Gerland1"></a>Gerland U, Moroz JD, Hwa T (2002) Physical constraints and functional characteristics of transcription factor-DNA interaction. Proc Natl Acad Sci U S A  99: 12015–12020.  <ul class="find" data-citedArticleID="984154" data-doi="10.1073/pnas.192693599"><li><a href="http://dx.doi.org/10.1073/pnas.192693599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physical+constraints+and+functional+characteristics+of+transcription+factor-DNA+interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physical+constraints+and+functional+characteristics+of+transcription+factor-DNA+interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000807-Allison1" id="pone.0000807-Allison1"></a>Allison DB, Cui X, Page GP, Sabripour M (2006) Microarray data analysis: from disarray to consolidation and consensus. Nat Rev Genet  7: 55–65.  <ul class="find" data-citedArticleID="984112" data-doi="10.1038/nrg1749"><li><a href="http://dx.doi.org/10.1038/nrg1749" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+data+analysis%3A+from+disarray+to+consolidation+and+consensus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+data+analysis%3A+from+disarray+to+consolidation+and+consensus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000807-AmbesiImpiombato1" id="pone.0000807-AmbesiImpiombato1"></a>Ambesi-Impiombato A, Bansal M, Lio P, di Bernardo D (2006) Computational framework for the prediction of transcription factor binding sites by multiple data integration. BMC Neurosci  7: Suppl 1S8.  <ul class="find" data-citedArticleID="984114" data-doi="10.1186/1471-2202-7-s1-s8"><li><a href="http://dx.doi.org/10.1186/1471-2202-7-s1-s8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+framework+for+the+prediction+of+transcription+factor+binding+sites+by+multiple+data+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+framework+for+the+prediction+of+transcription+factor+binding+sites+by+multiple+data+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000807-Blais1" id="pone.0000807-Blais1"></a>Blais A, Dynlacht BD (2005) Constructing transcriptional regulatory networks. Genes Dev  19: 1499–1511.  <ul class="find" data-citedArticleID="984126" data-doi="10.1101/gad.1325605"><li><a href="http://dx.doi.org/10.1101/gad.1325605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Constructing+transcriptional+regulatory+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Constructing+transcriptional+regulatory+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000807-Ren1" id="pone.0000807-Ren1"></a>Ren B, Robert F, Wyrick JJ, Aparicio O, Jennings EG, et al.  (2000) Genome-wide location and function of DNA binding proteins. Science  290: 2306–2309.  <ul class="find" data-citedArticleID="984208" data-doi="10.1126/science.290.5500.2306"><li><a href="http://dx.doi.org/10.1126/science.290.5500.2306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+location+and+function+of+DNA+binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+location+and+function+of+DNA+binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000807-Lam1" id="pone.0000807-Lam1"></a>Lam AL, Pazin DE, Sullivan BA (2005) Control of gene expression and assembly of chromosomal subdomains by chromatin regulators with antagonistic functions. Chromosoma  114: 242–251.  <ul class="find" data-citedArticleID="984174" data-doi="10.1007/s00412-005-0001-0"><li><a href="http://dx.doi.org/10.1007/s00412-005-0001-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+gene+expression+and+assembly+of+chromosomal+subdomains+by+chromatin+regulators+with+antagonistic+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+gene+expression+and+assembly+of+chromosomal+subdomains+by+chromatin+regulators+with+antagonistic+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000807-Elkon1" id="pone.0000807-Elkon1"></a>Elkon R, Linhart C, Sharan R, Shamir R, Shiloh Y (2003) Genome-wide in silico identification of transcriptional regulators controlling the cell cycle in human cells. Genome Res  13: 773–780.  <ul class="find" data-citedArticleID="984142" data-doi="10.1101/gr.947203"><li><a href="http://dx.doi.org/10.1101/gr.947203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+in+silico+identification+of+transcriptional+regulators+controlling+the+cell+cycle+in+human+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+in+silico+identification+of+transcriptional+regulators+controlling+the+cell+cycle+in+human+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000807-HoSui1" id="pone.0000807-HoSui1"></a>Ho Sui SJ, Mortimer JR, Arenillas DJ, Brumm J, Walsh CJ, et al.  (2005) oPOSSUM: identification of over-represented transcription factor binding sites in co-expressed genes. Nucleic Acids Res  33: 3154–3164.  <ul class="find" data-citedArticleID="984162" data-doi="10.1093/nar/gki624"><li><a href="http://dx.doi.org/10.1093/nar/gki624" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=oPOSSUM%3A+identification+of+over-represented+transcription+factor+binding+sites+in+co-expressed+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22oPOSSUM%3A+identification+of+over-represented+transcription+factor+binding+sites+in+co-expressed+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000807-Liu1" id="pone.0000807-Liu1"></a>Liu R, McEachin RC, States DJ (2003) Computationally identifying novel NF-kappa B-regulated immune genes in the human genome. Genome Res  13: 654–661.  <ul class="find" data-citedArticleID="984180" data-doi="10.1101/gr.911803"><li><a href="http://dx.doi.org/10.1101/gr.911803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computationally+identifying+novel+NF-kappa+B-regulated+immune+genes+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computationally+identifying+novel+NF-kappa+B-regulated+immune+genes+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000807-Frech1" id="pone.0000807-Frech1"></a>Frech K, Quandt K, Werner T (1997) Finding protein-binding sites in DNA sequences: the next generation. Trends Biochem Sci  22: 103–104.  <ul class="find" data-citedArticleID="984150" data-doi="10.1016/s0968-0004(97)01006-2"><li><a href="http://dx.doi.org/10.1016/s0968-0004(97)01006-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finding+protein-binding+sites+in+DNA+sequences%3A+the+next+generation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finding+protein-binding+sites+in+DNA+sequences%3A+the+next+generation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000807-McCue1" id="pone.0000807-McCue1"></a>McCue L, Thompson W, Carmack C, Ryan MP, Liu JS, et al.  (2001) Phylogenetic footprinting of transcription factor binding sites in proteobacterial genomes. Nucleic Acids Res  29: 774–782.  <ul class="find" data-citedArticleID="984188" data-doi="10.1093/nar/29.3.774"><li><a href="http://dx.doi.org/10.1093/nar/29.3.774" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+footprinting+of+transcription+factor+binding+sites+in+proteobacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+footprinting+of+transcription+factor+binding+sites+in+proteobacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000807-Rajewsky1" id="pone.0000807-Rajewsky1"></a>Rajewsky N, Socci ND, Zapotocky M, Siggia ED (2002) The evolution of DNA regulatory regions for proteo-gamma bacteria by interspecies comparisons. Genome Res  12: 298–308.  <ul class="find" data-citedArticleID="984204" data-doi="10.1101/gr.207502"><li><a href="http://dx.doi.org/10.1101/gr.207502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+DNA+regulatory+regions+for+proteo-gamma+bacteria+by+interspecies+comparisons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+DNA+regulatory+regions+for+proteo-gamma+bacteria+by+interspecies+comparisons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000807-McGuire1" id="pone.0000807-McGuire1"></a>McGuire AM, Hughes JD, Church GM (2000) Conservation of DNA regulatory motifs and discovery of new motifs in microbial genomes. Genome Res  10: 744–757.  <ul class="find" data-citedArticleID="984190" data-doi="10.1101/gr.10.6.744"><li><a href="http://dx.doi.org/10.1101/gr.10.6.744" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservation+of+DNA+regulatory+motifs+and+discovery+of+new+motifs+in+microbial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservation+of+DNA+regulatory+motifs+and+discovery+of+new+motifs+in+microbial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000807-Xie1" id="pone.0000807-Xie1"></a>Xie X, Lu J, Kulbokas EJ, Golub TR, Mootha V, et al.  (2005) Systematic discovery of regulatory motifs in human promoters and 3′ UTRs by comparison of several mammals. Nature  434: 338–345.  <ul class="find" data-citedArticleID="984230" data-doi="10.1038/nature03441"><li><a href="http://dx.doi.org/10.1038/nature03441" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+discovery+of+regulatory+motifs+in+human+promoters+and+3%26%23x2032%3B+UTRs+by+comparison+of+several+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+discovery+of+regulatory+motifs+in+human+promoters+and+3%26%23x2032%3B+UTRs+by+comparison+of+several+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000807-Kellis1" id="pone.0000807-Kellis1"></a>Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES (2003) Sequencing and comparison of yeast species to identify genes and regulatory elements. Nature  423: 241–254.  <ul class="find" data-citedArticleID="984170" data-doi="10.1038/nature01644"><li><a href="http://dx.doi.org/10.1038/nature01644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000807-Cliften1" id="pone.0000807-Cliften1"></a>Cliften P, Sudarsanam P, Desikan A, Fulton L, Fulton B, et al.  (2003) Finding functional features in Saccharomyces genomes by phylogenetic footprinting. Science  301: 71–76.  <ul class="find" data-citedArticleID="984136" data-doi="10.1126/science.1084337"><li><a href="http://dx.doi.org/10.1126/science.1084337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finding+functional+features+in+Saccharomyces+genomes+by+phylogenetic+footprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finding+functional+features+in+Saccharomyces+genomes+by+phylogenetic+footprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000807-Plessy1" id="pone.0000807-Plessy1"></a>Plessy C, Dickmeis T, Chalmel F, Strahle U (2005) Enhancer sequence conservation between vertebrates is favoured in developmental regulator genes. Trends Genet  21: 207–210.  <ul class="find" data-citedArticleID="984200" data-doi="10.1016/j.tig.2005.02.006"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.02.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhancer+sequence+conservation+between+vertebrates+is+favoured+in+developmental+regulator+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhancer+sequence+conservation+between+vertebrates+is+favoured+in+developmental+regulator+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000807-Emberly1" id="pone.0000807-Emberly1"></a>Emberly E, Rajewsky N, Siggia ED (2003) Conservation of regulatory elements between two species of Drosophila. BMC Bioinformatics  4: 57.  <ul class="find" data-citedArticleID="984146" data-doi="10.1186/1471-2105-4-57"><li><a href="http://dx.doi.org/10.1186/1471-2105-4-57" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservation+of+regulatory+elements+between+two+species+of+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservation+of+regulatory+elements+between+two+species+of+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000807-Nobrega1" id="pone.0000807-Nobrega1"></a>Nobrega MA, Zhu Y, Plajzer-Frick I, Afzal V, Rubin EM (2004) Megabase deletions of gene deserts result in viable mice. Nature  431: 988–993.  <ul class="find" data-citedArticleID="984194" data-doi="10.1038/nature03022"><li><a href="http://dx.doi.org/10.1038/nature03022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Megabase+deletions+of+gene+deserts+result+in+viable+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Megabase+deletions+of+gene+deserts+result+in+viable+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000807-Clark1" id="pone.0000807-Clark1"></a>Clark AG, Glanowski S, Nielsen R, Thomas PD, Kejariwal A, et al.  (2003) Inferring nonneutral evolution from human-chimp-mouse orthologous gene trios. Science  302: 1960–1963.  <ul class="find" data-citedArticleID="984134" data-doi="10.1126/science.1088821"><li><a href="http://dx.doi.org/10.1126/science.1088821" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+nonneutral+evolution+from+human-chimp-mouse+orthologous+gene+trios." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+nonneutral+evolution+from+human-chimp-mouse+orthologous+gene+trios.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000807-DeRisi1" id="pone.0000807-DeRisi1"></a>DeRisi JL, Iyer VR, Brown PO (1997) Exploring the metabolic and genetic control of gene expression on a genomic scale. Science  278: 680–686.  <ul class="find" data-citedArticleID="984140" data-doi="10.1126/science.278.5338.680"><li><a href="http://dx.doi.org/10.1126/science.278.5338.680" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+the+metabolic+and+genetic+control+of+gene+expression+on+a+genomic+scale." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+the+metabolic+and+genetic+control+of+gene+expression+on+a+genomic+scale.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000807-Hughes1" id="pone.0000807-Hughes1"></a>Hughes JD, Estep PW, Tavazoie S, Church GM (2000) Computational identification of cis-regulatory elements associated with groups of functionally related genes in Saccharomyces cerevisiae. J Mol Biol  296: 1205–1214.  <ul class="find" data-citedArticleID="984166" data-doi="10.1006/jmbi.2000.3519"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3519" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+identification+of+cis-regulatory+elements+associated+with+groups+of+functionally+related+genes+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+identification+of+cis-regulatory+elements+associated+with+groups+of+functionally+related+genes+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000807-Roth1" id="pone.0000807-Roth1"></a>Roth FP, Hughes JD, Estep PW, Church GM (1998) Finding DNA regulatory motifs within unaligned noncoding sequences clustered by whole-genome mRNA quantitation. Nat Biotechnol  16: 939–945.  <ul class="find" data-citedArticleID="984210" data-doi="10.1038/nbt1098-939"><li><a href="http://dx.doi.org/10.1038/nbt1098-939" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finding+DNA+regulatory+motifs+within+unaligned+noncoding+sequences+clustered+by+whole-genome+mRNA+quantitation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finding+DNA+regulatory+motifs+within+unaligned+noncoding+sequences+clustered+by+whole-genome+mRNA+quantitation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000807-Haverty1" id="pone.0000807-Haverty1"></a>Haverty PM, Hansen U, Weng Z (2004) Computational inference of transcriptional regulatory networks from expression profiling and transcription factor binding site identification. Nucleic Acids Res  32: 179–188.  <ul class="find" data-citedArticleID="984158" data-doi="10.1093/nar/gkh183"><li><a href="http://dx.doi.org/10.1093/nar/gkh183" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+inference+of+transcriptional+regulatory+networks+from+expression+profiling+and+transcription+factor+binding+site+identification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+inference+of+transcriptional+regulatory+networks+from+expression+profiling+and+transcription+factor+binding+site+identification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000807-Zheng1" id="pone.0000807-Zheng1"></a>Zheng J, Wu J, Sun Z (2003) An approach to identify over-represented cis-elements in related sequences. Nucleic Acids Res  31: 1995–2005.  <ul class="find" data-citedArticleID="984238" data-doi="10.1093/nar/gkg287"><li><a href="http://dx.doi.org/10.1093/nar/gkg287" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+approach+to+identify+over-represented+cis-elements+in+related+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+approach+to+identify+over-represented+cis-elements+in+related+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000807-Tabach1" id="pone.0000807-Tabach1"></a>Tabach Y, Milyavsky M, Shats I, Brosh R, Zuk O, et al.  (2005) The promoters of human cell cycle genes integrate signals from two tumor suppressive pathways during cellular transformation. Mol Syst Biol  1: 2005–0022.  <ul class="find" data-citedArticleID="984218" data-doi="10.1038/msb4100030"><li><a href="http://dx.doi.org/10.1038/msb4100030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+promoters+of+human+cell+cycle+genes+integrate+signals+from+two+tumor+suppressive+pathways+during+cellular+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+promoters+of+human+cell+cycle+genes+integrate+signals+from+two+tumor+suppressive+pathways+during+cellular+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000807-Subramanian1" id="pone.0000807-Subramanian1"></a>Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, et al.  (2005) Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A  102: 15545–15550.  <ul class="find" data-citedArticleID="984216" data-doi="10.1073/pnas.0506580102"><li><a href="http://dx.doi.org/10.1073/pnas.0506580102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000807-Ashburner1" id="pone.0000807-Ashburner1"></a>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al.  (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet  25: 25–29.  <ul class="find" data-citedArticleID="984118"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.&amp;auth=&amp;atitle=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000807-Li1" id="pone.0000807-Li1"></a>Li H, Rhodius V, Gross C, Siggia ED (2002) Identification of the binding sites of regulatory proteins in bacterial genomes. Proc Natl Acad Sci U S A  99: 11772–11777.  <ul class="find" data-citedArticleID="984176" data-doi="10.1073/pnas.112341999"><li><a href="http://dx.doi.org/10.1073/pnas.112341999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+the+binding+sites+of+regulatory+proteins+in+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+the+binding+sites+of+regulatory+proteins+in+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000807-Hertz1" id="pone.0000807-Hertz1"></a>Hertz GZ, Stormo GD (1999) Identifying DNA and protein patterns with statistically significant alignments of multiple sequences. Bioinformatics  15: 563–577.  <ul class="find" data-citedArticleID="984160" data-doi="10.1093/bioinformatics/15.7.563"><li><a href="http://dx.doi.org/10.1093/bioinformatics/15.7.563" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+DNA+and+protein+patterns+with+statistically+significant+alignments+of+multiple+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+DNA+and+protein+patterns+with+statistically+significant+alignments+of+multiple+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000807-Kel1" id="pone.0000807-Kel1"></a>Kel AE, Gossling E, Reuter I, Cheremushkin E, Kel-Margoulis OV, et al.  (2003) MATCH: A tool for searching transcription factor binding sites in DNA sequences. Nucleic Acids Res  31: 3576–3579.  <ul class="find" data-citedArticleID="984168" data-doi="10.1093/nar/gkg585"><li><a href="http://dx.doi.org/10.1093/nar/gkg585" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MATCH%3A+A+tool+for+searching+transcription+factor+binding+sites+in+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MATCH%3A+A+tool+for+searching+transcription+factor+binding+sites+in+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000807-Frith1" id="pone.0000807-Frith1"></a>Frith MC, Fu Y, Yu L, Chen JF, Hansen U, et al.  (2004) Detection of functional DNA motifs via statistical over-representation. Nucleic Acids Res  32: 1372–1381.  <ul class="find" data-citedArticleID="984152" data-doi="10.1093/nar/gkh299"><li><a href="http://dx.doi.org/10.1093/nar/gkh299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+functional+DNA+motifs+via+statistical+over-representation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+functional+DNA+motifs+via+statistical+over-representation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000807-Quandt1" id="pone.0000807-Quandt1"></a>Quandt K, Frech K, Karas H, Wingender E, Werner T (1995) MatInd and MatInspector: new fast and versatile tools for detection of consensus matches in nucleotide sequence data. Nucleic Acids Res  23: 4878–4884.  <ul class="find" data-citedArticleID="984202" data-doi="10.1093/nar/23.23.4878"><li><a href="http://dx.doi.org/10.1093/nar/23.23.4878" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MatInd+and+MatInspector%3A+new+fast+and+versatile+tools+for+detection+of+consensus+matches+in+nucleotide+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MatInd+and+MatInspector%3A+new+fast+and+versatile+tools+for+detection+of+consensus+matches+in+nucleotide+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000807-Whitfield1" id="pone.0000807-Whitfield1"></a>Whitfield ML, Sherlock G, Saldanha AJ, Murray JI, Ball CA, et al.  (2002) Identification of genes periodically expressed in the human cell cycle and their expression in tumors. Mol Biol Cell  13: 1977–2000.  <ul class="find" data-citedArticleID="984228" data-doi="10.1091/mbc.02-02-0030."><li><a href="http://dx.doi.org/10.1091/mbc.02-02-0030." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+genes+periodically+expressed+in+the+human+cell+cycle+and+their+expression+in+tumors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+genes+periodically+expressed+in+the+human+cell+cycle+and+their+expression+in+tumors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000807-RashiElkeles1" id="pone.0000807-RashiElkeles1"></a>Rashi-Elkeles S, Elkon R, Weizman N, Linhart C, Amariglio N, et al.  (2006) Parallel induction of ATM-dependent pro- and antiapoptotic signals in response to ionizing radiation in murine lymphoid tissue. Oncogene  25: 1584–1592.  <ul class="find" data-citedArticleID="984206" data-doi="10.1038/sj.onc.1209189"><li><a href="http://dx.doi.org/10.1038/sj.onc.1209189" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+induction+of+ATM-dependent+pro-+and+antiapoptotic+signals+in+response+to+ionizing+radiation+in+murine+lymphoid+tissue." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+induction+of+ATM-dependent+pro-+and+antiapoptotic+signals+in+response+to+ionizing+radiation+in+murine+lymphoid+tissue.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000807-Lodish1" id="pone.0000807-Lodish1"></a>Lodish H, Berk A, Zipursky SL, Matsudaira P, Baltimore D, et al.  (2000) Molecular Cell Biology W. H. Freeman.   <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0000807-Harbison1" id="pone.0000807-Harbison1"></a>Harbison CT, Gordon DB, Lee TI, Rinaldi NJ, Macisaac KD, et al.  (2004) Transcriptional regulatory code of a eukaryotic genome. Nature  431: 99–104.  <ul class="find" data-citedArticleID="984156" data-doi="10.1038/nature02800"><li><a href="http://dx.doi.org/10.1038/nature02800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+code+of+a+eukaryotic+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+code+of+a+eukaryotic+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000807-Pfeifer1" id="pone.0000807-Pfeifer1"></a>Pfeifer D, Kist R, Dewar K, Devon K, Lander ES, et al.  (1999) Campomelic dysplasia translocation breakpoints are scattered over 1 Mb proximal to SOX9: evidence for an extended control region. Am J Hum Genet  65: 111–124.  <ul class="find" data-citedArticleID="984196" data-doi="10.1086/302455"><li><a href="http://dx.doi.org/10.1086/302455" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Campomelic+dysplasia+translocation+breakpoints+are+scattered+over+1+Mb+proximal+to+SOX9%3A+evidence+for+an+extended+control+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Campomelic+dysplasia+translocation+breakpoints+are+scattered+over+1+Mb+proximal+to+SOX9%3A+evidence+for+an+extended+control+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000807-KimuraYoshida1" id="pone.0000807-KimuraYoshida1"></a>Kimura-Yoshida C, Kitajima K, Oda-Ishii I, Tian E, Suzuki M, et al.  (2004) Characterization of the pufferfish Otx2 cis-regulators reveals evolutionarily conserved genetic mechanisms for vertebrate head specification. Development  131: 57–71.  <ul class="find" data-citedArticleID="984172" data-doi="10.1242/dev.00877"><li><a href="http://dx.doi.org/10.1242/dev.00877" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+pufferfish+Otx2+cis-regulators+reveals+evolutionarily+conserved+genetic+mechanisms+for+vertebrate+head+specification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+pufferfish+Otx2+cis-regulators+reveals+evolutionarily+conserved+genetic+mechanisms+for+vertebrate+head+specification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000807-Howard1" id="pone.0000807-Howard1"></a>Howard ML, Davidson EH (2004) cis-Regulatory control circuits in development. Dev Biol  271: 109–118.  <ul class="find" data-citedArticleID="984164" data-doi="10.1016/j.ydbio.2004.06.014"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2004.06.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=cis-Regulatory+control+circuits+in+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22cis-Regulatory+control+circuits+in+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000807-Ling1" id="pone.0000807-Ling1"></a>Ling JQ, Li T, Hu JF, Vu TH, Chen HL, et al.  (2006) CTCF mediates interchromosomal colocalization between Igf2/H19 and Wsb1/Nf1. Science  312: 269–272.  <ul class="find" data-citedArticleID="984178" data-doi="10.1126/science.1123191"><li><a href="http://dx.doi.org/10.1126/science.1123191" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CTCF+mediates+interchromosomal+colocalization+between+Igf2%2FH19+and+Wsb1%2FNf1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CTCF+mediates+interchromosomal+colocalization+between+Igf2%2FH19+and+Wsb1%2FNf1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000807-Carninci1" id="pone.0000807-Carninci1"></a>Carninci P, Sandelin A, Lenhard B, Katayama S, Shimokawa K, et al.  (2006) Genome-wide analysis of mammalian promoter architecture and evolution. Nat Genet  38: 626–635.  <ul class="find" data-citedArticleID="984130" data-doi="10.1038/ng1789"><li><a href="http://dx.doi.org/10.1038/ng1789" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+mammalian+promoter+architecture+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+mammalian+promoter+architecture+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000807-Sharan1" id="pone.0000807-Sharan1"></a>Sharan R, Ben-Hur A, Loots GG, Ovcharenko I (2004) CREME: Cis-Regulatory Module Explorer for the human genome. Nucleic Acids Res  32: W253–256.  <ul class="find" data-citedArticleID="984214" data-doi="10.1093/nar/gkh385"><li><a href="http://dx.doi.org/10.1093/nar/gkh385" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CREME%3A+Cis-Regulatory+Module+Explorer+for+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CREME%3A+Cis-Regulatory+Module+Explorer+for+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000807-AmirZilberstein1" id="pone.0000807-AmirZilberstein1"></a>Amir-Zilberstein L, Ainbinder E, Toube L, Yamaguchi Y, Handa H, et al.  (2007) Differential Regulation of NF-{kappa}B by Elongation Factors Is Determined by Core Promoter Type. Mol Cell Biol  27: 5246–5259.  <ul class="find" data-citedArticleID="984116" data-doi="10.1128/mcb.00586-07"><li><a href="http://dx.doi.org/10.1128/mcb.00586-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+Regulation+of+NF-%7Bkappa%7DB+by+Elongation+Factors+Is+Determined+by+Core+Promoter+Type." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+Regulation+of+NF-%7Bkappa%7DB+by+Elongation+Factors+Is+Determined+by+Core+Promoter+Type.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000807-Yang1" id="pone.0000807-Yang1"></a>Yang C, Bolotin E, Jiang T, Sladek FM, Martinez E (2007) Prevalence of the initiator over the TATA box in human and yeast genes and identification of DNA motifs enriched in human TATA-less core promoters. Gene  389: 52–65.  <ul class="find" data-citedArticleID="984232" data-doi="10.1016/j.gene.2006.09.029"><li><a href="http://dx.doi.org/10.1016/j.gene.2006.09.029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence+of+the+initiator+over+the+TATA+box+in+human+and+yeast+genes+and+identification+of+DNA+motifs+enriched+in+human+TATA-less+core+promoters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence+of+the+initiator+over+the+TATA+box+in+human+and+yeast+genes+and+identification+of+DNA+motifs+enriched+in+human+TATA-less+core+promoters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000807-Barash1" id="pone.0000807-Barash1"></a>Barash Y, Bejerano G, Friedman N (2001) A Simple Hyper-Geometric Approach for Discovering Putative Transcription Factor Binding Sites Algorithms in Bioinformatics: Springer Berlin/Heidelberg.   <ul class="find-nolinks"></ul></li><li><span class="label">49.
              </span><a name="pone.0000807-Benjamini1" id="pone.0000807-Benjamini1"></a>Benjamini Y, Hochberg Y (1995) Controlling the False Discovery Rate: A practical and powerful approach to multiple testing. J Roy Stat Soc B,  57: 289–300.  <ul class="find" data-citedArticleID="984122"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Controlling+the+False+Discovery+Rate%3A+A+practical+and+powerful+approach+to+multiple+testing.&amp;auth=&amp;atitle=Controlling+the+False+Discovery+Rate%3A+A+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+the+False+Discovery+Rate%3A+A+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+the+False+Discovery+Rate%3A+A+practical+and+powerful+approach+to+multiple+testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000807-Wasserman1" id="pone.0000807-Wasserman1"></a>Wasserman WW, Palumbo M, Thompson W, Fickett JW, Lawrence CE (2000) Human-mouse genome comparisons to locate regulatory sites. Nat Genet  26: 225–228.  <ul class="find" data-citedArticleID="984226" data-doi="10.1038/79965"><li><a href="http://dx.doi.org/10.1038/79965" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human-mouse+genome+comparisons+to+locate+regulatory+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human-mouse+genome+comparisons+to+locate+regulatory+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000807-Tavazoie1" id="pone.0000807-Tavazoie1"></a>Tavazoie S, Hughes JD, Campbell MJ, Cho RJ, Church GM (1999) Systematic determination of genetic network architecture. Nat Genet  22: 281–285.  <ul class="find" data-citedArticleID="984220" data-doi="10.1038/10343"><li><a href="http://dx.doi.org/10.1038/10343" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+determination+of+genetic+network+architecture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+determination+of+genetic+network+architecture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000807-Wang1" id="pone.0000807-Wang1"></a>Wang Z, Wang DZ, Hockemeyer D, McAnally J, Nordheim A, et al.  (2004) Myocardin and ternary complex factors compete for SRF to control smooth muscle gene expression. Nature  428: 185–189.  <ul class="find" data-citedArticleID="984222" data-doi="10.1038/nature02382"><li><a href="http://dx.doi.org/10.1038/nature02382" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Myocardin+and+ternary+complex+factors+compete+for+SRF+to+control+smooth+muscle+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Myocardin+and+ternary+complex+factors+compete+for+SRF+to+control+smooth+muscle+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000807-Wang2" id="pone.0000807-Wang2"></a>Wang D, Chang PS, Wang Z, Sutherland L, Richardson JA, et al.  (2001) Activation of cardiac gene expression by myocardin, a transcriptional cofactor for serum response factor. Cell  105: 851–862.  <ul class="find" data-citedArticleID="984224" data-doi="10.1016/s0092-8674(01)00404-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(01)00404-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+of+cardiac+gene+expression+by+myocardin%2C+a+transcriptional+cofactor+for+serum+response+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+of+cardiac+gene+expression+by+myocardin%2C+a+transcriptional+cofactor+for+serum+response+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000807-Pipes1" id="pone.0000807-Pipes1"></a>Pipes GC, Sinha S, Qi X, Zhu CH, Gallardo TD, et al.  (2005) Stem cells and their derivatives can bypass the requirement of myocardin for smooth muscle gene expression. Dev Biol  288: 502–513.  <ul class="find" data-citedArticleID="984198" data-doi="10.1016/j.ydbio.2005.10.014"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2005.10.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stem+cells+and+their+derivatives+can+bypass+the+requirement+of+myocardin+for+smooth+muscle+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stem+cells+and+their+derivatives+can+bypass+the+requirement+of+myocardin+for+smooth+muscle+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000807-Yao1" id="pone.0000807-Yao1"></a>Yao GQ, Sun BH, Insogna KL, Weir EC (2000) Nuclear factor-kappaB p50 is required for tumor necrosis factor-alpha-induced colony-stimulating factor-1 gene expression in osteoblasts. Endocrinology  141: 2914–2922.  <ul class="find" data-citedArticleID="984234" data-doi="10.1210/endo.141.8.7592"><li><a href="http://dx.doi.org/10.1210/endo.141.8.7592" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+factor-kappaB+p50+is+required+for+tumor+necrosis+factor-alpha-induced+colony-stimulating+factor-1+gene+expression+in+osteoblasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+factor-kappaB+p50+is+required+for+tumor+necrosis+factor-alpha-induced+colony-stimulating+factor-1+gene+expression+in+osteoblasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000807-Eto1" id="pone.0000807-Eto1"></a>Eto A, Muta T, Yamazaki S, Takeshige K (2003) Essential roles for NF-kappa B and a Toll/IL-1 receptor domain-specific signal(s) in the induction of I kappa B-zeta. Biochem Biophys Res Commun  301: 495–501.  <ul class="find" data-citedArticleID="984148" data-doi="10.1016/s0006-291x(02)03082-6"><li><a href="http://dx.doi.org/10.1016/s0006-291x(02)03082-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Essential+roles+for+NF-kappa+B+and+a+Toll%2FIL-1+receptor+domain-specific+signal%28s%29+in+the+induction+of+I+kappa+B-zeta." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Essential+roles+for+NF-kappa+B+and+a+Toll%2FIL-1+receptor+domain-specific+signal%28s%29+in+the+induction+of+I+kappa+B-zeta.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000807-Carninci2" id="pone.0000807-Carninci2"></a>Carninci P, Kasukawa T, Katayama S, Gough J, Frith MC, et al.  (2005) The transcriptional landscape of the mammalian genome. Science  309: 1559–1563.  <ul class="find" data-citedArticleID="984132" data-doi="10.1126/science.1112014"><li><a href="http://dx.doi.org/10.1126/science.1112014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptional+landscape+of+the+mammalian+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptional+landscape+of+the+mammalian+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000807-Buckland1" id="pone.0000807-Buckland1"></a>Buckland PR, Hoogendoorn B, Coleman SL, Guy CA, Smith SK, et al.  (2005) Strong bias in the location of functional promoter polymorphisms. Hum Mutat  26: 214–223.  <ul class="find" data-citedArticleID="984128" data-doi="10.1002/humu.20207"><li><a href="http://dx.doi.org/10.1002/humu.20207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strong+bias+in+the+location+of+functional+promoter+polymorphisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strong+bias+in+the+location+of+functional+promoter+polymorphisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000807-Cooper1" id="pone.0000807-Cooper1"></a>Cooper SJ, Trinklein ND, Anton ED, Nguyen L, Myers RM (2006) Comprehensive analysis of transcriptional promoter structure and function in 1% of the human genome. Genome Res  16: 1–10.  <ul class="find" data-citedArticleID="984138" data-doi="10.1101/gr.4222606"><li><a href="http://dx.doi.org/10.1101/gr.4222606" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+analysis+of+transcriptional+promoter+structure+and+function+in+1%25+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+analysis+of+transcriptional+promoter+structure+and+function+in+1%25+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000807-MarinoRamirez1" id="pone.0000807-MarinoRamirez1"></a>Marino-Ramirez L, Lewis KC, Landsman D, Jordan IK (2005) Transposable elements donate lineage-specific regulatory sequences to host genomes. Cytogenet Genome Res  110: 333–341.  <ul class="find" data-citedArticleID="984186" data-doi="10.1159/000084965"><li><a href="http://dx.doi.org/10.1159/000084965" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transposable+elements+donate+lineage-specific+regulatory+sequences+to+host+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transposable+elements+donate+lineage-specific+regulatory+sequences+to+host+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000807-Yuan1" id="pone.0000807-Yuan1"></a>Yuan GC, Liu YJ, Dion MF, Slack MD, Wu LF, et al.  (2005) Genome-scale identification of nucleosome positions in S. cerevisiae. Science  309: 626–630.  <ul class="find" data-citedArticleID="984236" data-doi="10.1126/science.1112178"><li><a href="http://dx.doi.org/10.1126/science.1112178" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-scale+identification+of+nucleosome+positions+in+S.+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-scale+identification+of+nucleosome+positions+in+S.+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000807-Segal1" id="pone.0000807-Segal1"></a>Segal E, Fondufe-Mittendorf Y, Chen L, Thastrom A, Field Y, et al.  (2006) A genomic code for nucleosome positioning. Nature  442: 772–778.  <ul class="find" data-citedArticleID="984212" data-doi="10.1038/nature04979"><li><a href="http://dx.doi.org/10.1038/nature04979" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomic+code+for+nucleosome+positioning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomic+code+for+nucleosome+positioning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000807-MacIsaac1" id="pone.0000807-MacIsaac1"></a>MacIsaac KD, Fraenkel E (2006) Practical strategies for discovering regulatory DNA sequence motifs. PLoS Comput Biol  2: e36.  <ul class="find" data-citedArticleID="984184" data-doi="10.1371/journal.pcbi.0020036"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Practical+strategies+for+discovering+regulatory+DNA+sequence+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Practical+strategies+for+discovering+regulatory+DNA+sequence+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000807-Benjamini2" id="pone.0000807-Benjamini2"></a>Benjamini Y, Yekutieli D (2001) The control of the false discovery rate in multiple testing under dependency. The Annals of Statistics  29: 1165–1188.  <ul class="find" data-citedArticleID="984124" data-doi="10.1214/aos/1013699998"><li><a href="http://dx.doi.org/10.1214/aos/1013699998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+control+of+the+false+discovery+rate+in+multiple+testing+under+dependency." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+control+of+the+false+discovery+rate+in+multiple+testing+under+dependency.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000807-Milyavsky1" id="pone.0000807-Milyavsky1"></a>Milyavsky M, Shats I, Cholostoy A, Brosh R, Buganim Y, et al.  (2007) Inactivation of myocardin and p16 during malignant transformation contributes to a differentiation defect. Cancer Cell  11: 133–146.  <ul class="find" data-citedArticleID="984192" data-doi="10.1016/j.ccr.2006.11.022"><li><a href="http://dx.doi.org/10.1016/j.ccr.2006.11.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inactivation+of+myocardin+and+p16+during+malignant+transformation+contributes+to+a+differentiation+defect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inactivation+of+myocardin+and+p16+during+malignant+transformation+contributes+to+a+differentiation+defect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.XML" value="147027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.PDF" value="1384802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g001.PNG_L" value="1718115"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g001.PNG_M" value="267786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g001.PNG_S" value="14896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g001.TIF" value="1986940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g001.PNG_I" value="84589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g002.PNG_L" value="712476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g002.PNG_M" value="139752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g002.PNG_S" value="16360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g002.TIF" value="1113918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g002.PNG_I" value="88771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g003.PNG_L" value="628140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g003.PNG_M" value="169373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g003.PNG_S" value="12518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g003.TIF" value="1236638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g003.PNG_I" value="70983"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g004.PNG_L" value="380885"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g004.PNG_M" value="179560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g004.PNG_S" value="10281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g004.TIF" value="519146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g004.PNG_I" value="74089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g005.PNG_L" value="168157"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g005.PNG_M" value="81725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g005.PNG_S" value="5790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g005.TIF" value="258070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g005.PNG_I" value="29853"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g006.PNG_L" value="884310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g006.PNG_M" value="204437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g006.PNG_S" value="17341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g006.TIF" value="1218830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.g006.PNG_I" value="98422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.e001.PNG" value="10311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.e001.TIF" value="26244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.s001.TIF" value="2577480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.s002.TIF" value="4231160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.s003.TIF" value="1491492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.s004.PDF" value="8618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.s005.PDF" value="867343"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.s006.PDF" value="26292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000807.s007.DOC" value="29696"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000807&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000807" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000807&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000807&volume=&issue=&title=Wide-Scale Analysis of Human Functional Transcription Factor Binding Reveals a Strong Bias towards the Transcription Start Site&author_name=Yuval%20Tabach%2C%20Ran%20Brosh%2C%20Yossi%20Buganim%2C%20Anat%20Reiner%2C%20Or%20Zuk%2C%20Assif%20Yitzhaky%2C%20Mark%20Koudritsky%2C%20Varda%20Rotter%2C%20Eytan%20Domany&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000807" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000807" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000807" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000807&amp;t=Wide-Scale%20Analysis%20of%20Human%20Functional%20Transcription%20Factor%20Binding%20Reveals%20a%20Strong%20Bias%20towards%20the%20Transcription%20Start%20Site" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000807&title=Wide-Scale%20Analysis%20of%20Human%20Functional%20Transcription%20Factor%20Binding%20Reveals%20a%20Strong%20Bias%20towards%20the%20Transcription%20Start%20Site&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000807&amp;title=Wide-Scale%20Analysis%20of%20Human%20Functional%20Transcription%20Factor%20Binding%20Reveals%20a%20Strong%20Bias%20towards%20the%20Transcription%20Start%20Site" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000807" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000807&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Wide-Scale Analysis of Human Functional Transcription Factor Binding Reveals a Strong Bias towards the Transcription Start Site';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000807';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Wide-Scale%20Analysis%20of%20Human%20Functional%20Transcription%20Factor%20Binding%20Reveals%20a%20Strong%20Bias%20towards%20the%20Transcription%20Start%20Site http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000807" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000807" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Binding+analysis%22" title="Search for articles in the subject area:'Binding analysis'"><div class="flagText">Binding analysis</div></a>
              <div data-categoryid="19597" data-articleid="24452"
                   data-categoryname="Binding analysis"
                   class="flagImage" title="Flag 'Binding analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+cycle+and+cell+division%22" title="Search for articles in the subject area:'Cell cycle and cell division'"><div class="flagText">Cell cycle and cell division</div></a>
              <div data-categoryid="17353" data-articleid="24452"
                   data-categoryname="Cell cycle and cell division"
                   class="flagImage" title="Flag 'Cell cycle and cell division' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Conservation+science%22" title="Search for articles in the subject area:'Conservation science'"><div class="flagText">Conservation science</div></a>
              <div data-categoryid="40107" data-articleid="24452"
                   data-categoryname="Conservation science"
                   class="flagImage" title="Flag 'Conservation science' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24452"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="24452"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="24452"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="17701" data-articleid="24452"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24452"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3578'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5501'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=220&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
