

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Direct Selection on Genetic Robustness Revealed in the Yeast Transcriptome</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000911"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000911" />

    <meta name="citation_title" content="Direct Selection on Genetic Robustness Revealed in the Yeast Transcriptome"/>
    <meta itemprop="name" content="Direct Selection on Genetic Robustness Revealed in the Yeast Transcriptome"/>

      <meta name="citation_author" content="Stephen R. Proulx"/>
            <meta name="citation_author_institution" content="Department of Ecology, Evolution and Organismal Biology, Iowa State University, Ames, Iowa, United States of America"/>
      <meta name="citation_author" content="Sergey Nuzhdin"/>
            <meta name="citation_author_institution" content="Section of Evolution and Ecology, University of California at Davis, Davis, California, United States of America"/>
            <meta name="citation_author_institution" content="Molecular and Computational Biology, University of Southern California, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Daniel E. L. Promislow"/>
            <meta name="citation_author_institution" content="Department of Genetics, University of Georgia, Athens, Georgia, United States of America"/>

    <meta name="citation_date" content="2007/9/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000911.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e911"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Maintenance of Genetic Polymorphism in a Spatially Heterogeneous Environment-Variations On a Theme by Howard Levene.; citation_author=R Levins; citation_author=R MacArthur; citation_journal_title=American Naturalist; citation_volume=100; citation_number=1; citation_pages=585-589; citation_date=1966; " />
      <meta name="citation_reference" content="citation_title=Maintenance of genetic heterogeneity.; citation_author=ER Dempster; citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology; citation_volume=20; citation_number=2; citation_pages=25-32; citation_date=1955; " />
      <meta name="citation_reference" content="citation_title=Comparative quantitative genetics: evolution of the G matrix.; citation_author=SJ Steppan; citation_author=PC Phillips; citation_author=D Houle; citation_journal_title=Trends in Ecology & Evolution; citation_volume=17; citation_number=3; citation_pages=320-327; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mapping phenotypes: canalization, plasticity and developmental stability.; citation_author=V Debat; citation_author=P David; citation_journal_title=Trends in Ecology & Evolution; citation_volume=16; citation_number=4; citation_pages=555-561; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A regulatory network analysis of phenotypic plasticity in yeast.; citation_author=D Promislow; citation_journal_title=American Naturalist; citation_volume=165; citation_number=5; citation_pages=515-523; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The canalization of development and genetic assimilation of acquired characters.; citation_author=CH Waddington; citation_journal_title=Nature; citation_volume=150; citation_number=6; citation_pages=563-565; citation_date=1942; " />
      <meta name="citation_reference" content="citation_title=Genetic Assimilation.; citation_author=CH Waddington; citation_journal_title=Advances in Genetics Incorporating Molecular Genetic Medicine; citation_volume=10; citation_number=7; citation_pages=257-293; citation_date=1961; " />
      <meta name="citation_reference" content="citation_title=Factors of evolution.; citation_author=II Schmalhausen; citation_number=8; citation_date=1949; citation_publisher=University of Chicago Press; " />
      <meta name="citation_reference" content="citation_title=Components of Fitness.; citation_author=JM Thoday; citation_journal_title=Symposia of the Society for Experimental Biology; citation_volume=7; citation_number=9; citation_pages=96-113; citation_date=1953; " />
      <meta name="citation_reference" content="citation_title=Perspective: Evolution and detection of genetic robustness.; citation_author=J de Visser; citation_author=J Hermisson; citation_author=GP Wagner; citation_author=LA Meyers; citation_author=HC Bagheri; citation_journal_title=Evolution; citation_volume=57; citation_number=10; citation_pages=1959-1972; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The evolutionary genetics of canalization.; citation_author=T Flatt; citation_journal_title=Quarterly Review of Biology; citation_volume=80; citation_number=11; citation_pages=287-316; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolution of genetic architecture under directional selection.; citation_author=TF Hansen; citation_author=JM Alvarez-Castro; citation_author=AJR Carter; citation_author=J Hermisson; citation_author=GP Wagner; citation_journal_title=Evolution; citation_volume=60; citation_number=12; citation_pages=1523-1536; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A population genetic theory of canalization.; citation_author=GP Wagner; citation_author=G Booth; citation_author=HC Bagheri; citation_journal_title=Evolution; citation_volume=51; citation_number=13; citation_pages=329-347; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The evolution of genetic canalization under fluctuating selection.; citation_author=TJ Kawecki; citation_journal_title=Evolution; citation_volume=54; citation_number=14; citation_pages=1-12; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The evolution of canalization and the breaking of von Baer's laws: Modeling the evolution of development with epistasis.; citation_author=SH Rice; citation_journal_title=Evolution; citation_volume=52; citation_number=15; citation_pages=647-656; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The opportunity for canalization and the evolution of genetic networks.; citation_author=SR Proulx; citation_author=PC Phillips; citation_journal_title=American Naturalist; citation_volume=165; citation_number=16; citation_pages=147-162; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A quantitative-genetic model for selection on developmental noise.; citation_author=S Gavrilets; citation_author=A Hastings; citation_journal_title=Evolution; citation_volume=48; citation_number=17; citation_pages=1478-1486; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=An introduction to population genetics theory.; citation_author=JF Crow; citation_author=M Kimura; citation_number=18; citation_date=1970; citation_publisher=Harper and Row; " />
      <meta name="citation_reference" content="citation_title=Fitness Sensitivity and the Canalization of Life-History Traits.; citation_author=SC Stearns; citation_author=TJ Kawecki; citation_journal_title=Evolution; citation_volume=48; citation_number=19; citation_pages=1438-1450; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The Differential Genetic and Environmental Canalization of Fitness Components in Drosophila melanogaster.; citation_author=SC Stearns; citation_author=M Kaiser; citation_author=TJ Kawecki; citation_journal_title=Journal of Evolutionary Biology; citation_volume=8; citation_number=20; citation_pages=539-557; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Viral RNA and evolved mutational robustness.; citation_author=A Wagner; citation_author=PF Stadler; citation_journal_title=J Exp Zool; citation_volume=285; citation_number=21; citation_pages=119-127; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Epistasis and Its Relationship to Canalization in the RNA Virus phi 6.; citation_author=CL Burch; citation_author=L Chao; citation_journal_title=Genetics; citation_volume=167; citation_number=22; citation_pages=559-567; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Robustness in simple biochemical networks.; citation_author=N Barkai; citation_author=S Leibler; citation_journal_title=Nature; citation_volume=387; citation_number=23; citation_pages=913-917; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Hsp90 as a capacitor for morphological evolution.; citation_author=SL Rutherford; citation_author=S Lindquist; citation_journal_title=Nature; citation_volume=396; citation_number=24; citation_pages=336-342; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Plasticity, evolvability, and modularity in RNA.; citation_author=LW Ancel; citation_author=W Fontana; citation_journal_title=J Exp Zool; citation_volume=288; citation_number=25; citation_pages=242-283; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The segment polarity network is a robust developmental module.; citation_author=G von Dassow; citation_author=E Meir; citation_author=EM Munro; citation_author=GM Odell; citation_journal_title=Nature; citation_volume=406; citation_number=26; citation_pages=188-192; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A single mode of canalization.; citation_author=CD Meiklejohn; citation_author=DL Hartl; citation_journal_title=Trends in Ecology & Evolution; citation_volume=17; citation_number=27; citation_pages=468-473; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Progress on canalization.; citation_author=SC Stearns; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=99; citation_number=28; citation_pages=10229-10230; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comparative Genomics of Centrality and Essentiality in Three Eukaryotic Protein-Interaction Networks.; citation_author=MW Hahn; citation_author=AD Kern; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=29; citation_pages=803-806; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genomic expression programs in the response of yeast cells to environmental changes.; citation_author=AP Gasch; citation_author=PT Spellman; citation_author=CM Kao; citation_author=O Carmel-Harel; citation_author=MB Eisen; citation_journal_title=Mol Biol Cell; citation_volume=11; citation_number=30; citation_pages=4241-4257; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Functional discovery via a compendium of expression profiles.; citation_author=TR Hughes; citation_author=MJ Marton; citation_author=AR Jones; citation_author=CJ Roberts; citation_author=R Stoughton; citation_journal_title=Cell; citation_volume=102; citation_number=31; citation_pages=109-126; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Functional profiling of the Saccharomyces cerevisiae genome.; citation_author=G Giaever; citation_author=AM Chu; citation_author=L Ni; citation_author=C Connelly; citation_author=L Riles; citation_journal_title=Nature; citation_volume=418; citation_number=32; citation_pages=387-391; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Single-cell proteomic analysis of S. cerevisiae reveals the architecture of biological noise.; citation_author=JRS Newman; citation_author=S Ghaemmaghami; citation_author=J Ihmels; citation_author=DK Breslow; citation_author=M Noble; citation_journal_title=Nature; citation_volume=441; citation_number=33; citation_pages=840-846; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Network thinking in ecology and evolution.; citation_author=SR Proulx; citation_author=DEL Promislow; citation_author=PC Phillips; citation_journal_title=Trends in Ecology & Evolution; citation_volume=20; citation_number=34; citation_pages=345-353; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Complexity and robustness.; citation_author=JM Carlson; citation_author=J Doyle; citation_journal_title=PNAS; citation_volume=99; citation_number=35; citation_pages=2538-2545; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=CYGD: the Comprehensive Yeast Genome Database.; citation_author=U Güldener; citation_author=M Münsterkötter; citation_author=G Kastenmüller; citation_author=vH NS, J; citation_journal_title=Nucleic Acids Research; citation_volume=33; citation_number=36; citation_pages=364-368; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of variance for gene expression microarray data.; citation_author=MK Kerr; citation_author=M Martin; citation_author=GA Churchill; citation_journal_title=J Comput Biol; citation_volume=7; citation_number=37; citation_pages=819-837; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.; citation_author=M Kellis; citation_author=N Patterson; citation_author=M Endrizzi; citation_author=B Birren; citation_author=ES Lander; citation_journal_title=Nature; citation_volume=423; citation_number=38; citation_pages=241-254; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory networks in Saccharomyces cerevisiae.; citation_author=TI Lee; citation_author=NJ Rinaldi; citation_author=F Robert; citation_author=DT Odom; citation_author=Z Bar-Joseph; citation_journal_title=Science; citation_volume=298; citation_number=39; citation_pages=799-804; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae.; citation_author=P Uetz; citation_author=L Giot; citation_author=G Cagney; citation_author=TA Mansfield; citation_author=RS Judson; citation_journal_title=Nature; citation_volume=403; citation_number=40; citation_pages=623-627; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The GRID: the General Repository for Interaction Datasets.; citation_author=B Breitkreutz; citation_author=C, M.T Stark; citation_journal_title=Genome Biology; citation_volume=4; citation_number=41; citation_pages=R23; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=High-definition macromolecular composition of yeast RNA-processing complexes.; citation_author=NJ Krogan; citation_author=WT Peng; citation_author=G Cagney; citation_author=MD Robinson; citation_author=R Haw; citation_journal_title=Mol Cell; citation_volume=13; citation_number=42; citation_pages=225-239; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Assigning function to yeast proteins by integration of technologies.; citation_author=TR Hazbun; citation_author=L Malmstrom; citation_author=S Anderson; citation_author=BJ Graczyk; citation_author=B Fox; citation_journal_title=Mol Cell; citation_volume=12; citation_number=43; citation_pages=1353-1365; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry.; citation_author=Y Ho; citation_author=A Gruhler; citation_author=A Heilbut; citation_author=GD Bader; citation_author=L Moore; citation_journal_title=Nature; citation_volume=415; citation_number=44; citation_pages=180-183; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Functional organization of the yeast proteome by systematic analysis of protein complexes.; citation_author=AC Gavin; citation_author=M Bosche; citation_author=R Krause; citation_author=P Grandi; citation_author=M Marzioch; citation_journal_title=Nature; citation_volume=415; citation_number=45; citation_pages=141-147; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A comprehensive two-hybrid analysis to explore the yeast protein interactome.; citation_author=T Ito; citation_author=T Chiba; citation_author=R Ozawa; citation_author=M Yoshida; citation_author=M Hattori; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=46; citation_pages=4569-4574; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Toward a protein-protein interaction map of the budding yeast: A comprehensive system to examine two-hybrid interactions in all possible combinations between the yeast proteins.; citation_author=T Ito; citation_author=K Tashiro; citation_author=S Muta; citation_author=R Ozawa; citation_author=T Chiba; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=47; citation_pages=1143-1147; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Toward a functional analysis of the yeast genome through exhaustive two-hybrid screens.; citation_author=M Fromont-Racine; citation_author=JC Rain; citation_author=P Legrain; citation_journal_title=Nat Genet; citation_volume=16; citation_number=48; citation_pages=277-282; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Exploratory social network analysis with Pajek.; citation_author=W de Nooy; citation_author=A Mrvar; citation_author=V Batagelj; citation_number=49; citation_date=2005; citation_publisher=Cambridge University Press; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000911" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Direct Selection on Genetic Robustness Revealed in the Yeast Transcriptome"/>
    <meta name="twitter:description" content="BackgroundEvolutionary theory predicts that organisms should evolve the ability to produce high fitness phenotypes in the face of environmental disturbances (environmental robustness) or genetic mutations (genetic robustness). While several studies have uncovered mechanisms that lead to both environmental and genetic robustness, we have yet to understand why some components of the genome are more robust than others. According to evolutionary theory, environmental and genetic robustness will have different responses to selective forces. Selection on environmental robustness for a trait is expected to be strong and related to the fitness costs of altering that trait. In contrast to environmental robustness, selection on genetic robustness for a trait is expected to be largely independent of the fitness cost of altering the trait and instead should correlate with the standing genetic variation for the trait that can potentially be buffered. Several mechanisms that provide both environmental and genetic robustness have been described, and this correlation could be explained by direct selection on both forms of robustness (direct selection hypothesis), or through selection on environmental robustness and a correlated response in genetic robustness (congruence hypothesis).Methodology/Principal FindingsUsing both published and novel data on gene expression in the yeast Saccharomyces cerevisiae, we find that genetic robustness is correlated with environmental robustness across the yeast genome as predicted by the congruence hypothesis. However, we also show that environmental robustness, but not genetic robustness, is related to per-gene fitness effects. In contrast, genetic robustness is significantly correlated with network position, suggesting that genetic robustness has been under direct selection.Conclusions/SignificanceWe observed a significant correlation between our measures of genetic and environmental robustness, in agreement with the congruence hypothesis. However, this correlation alone cannot explain the co-variance of genetic robustness with position in the protein interaction network. We therefore conclude that direct selection on robustness has played a role in the evolution of genetic robustness in the transcriptome."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000911.g003"/>

  <meta property="og:title" content="Direct Selection on Genetic Robustness Revealed in the Yeast Transcriptome" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000911" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=737'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7365'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3746&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000911">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000911" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000911&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Direct Selection on Genetic Robustness Revealed in the Yeast Transcriptome
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen R. Proulx
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:proulx@iastate.edu">proulx@iastate.edu</a></p>

                <p>Affiliation:
                  Department of Ecology, Evolution and Organismal Biology, Iowa State University, Ames, Iowa, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sergey Nuzhdin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Section of Evolution and Ecology, University of California at Davis, Davis, California, United States of America, 
                  Molecular and Computational Biology, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel E. L. Promislow
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, University of Georgia, Athens, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000911</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000911" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000911" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000911" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000911" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000911-g001" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000911-g002" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000911-g003" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000911-t001" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000911-t002" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000911-t003" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000911-t004" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000911">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000911" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Evolutionary theory predicts that organisms should evolve the ability to produce high fitness phenotypes in the face of environmental disturbances (environmental robustness) or genetic mutations (genetic robustness). While several studies have uncovered mechanisms that lead to both environmental and genetic robustness, we have yet to understand why some components of the genome are more robust than others. According to evolutionary theory, environmental and genetic robustness will have different responses to selective forces. Selection on environmental robustness for a trait is expected to be strong and related to the fitness costs of altering that trait. In contrast to environmental robustness, selection on genetic robustness for a trait is expected to be largely independent of the fitness cost of altering the trait and instead should correlate with the standing genetic variation for the trait that can potentially be buffered. Several mechanisms that provide both environmental and genetic robustness have been described, and this correlation could be explained by direct selection on both forms of robustness (direct selection hypothesis), or through selection on environmental robustness and a correlated response in genetic robustness (congruence hypothesis).</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Using both published and novel data on gene expression in the yeast <em>Saccharomyces cerevisiae</em>, we find that genetic robustness is correlated with environmental robustness across the yeast genome as predicted by the congruence hypothesis. However, we also show that environmental robustness, but not genetic robustness, is related to per-gene fitness effects. In contrast, genetic robustness is significantly correlated with network position, suggesting that genetic robustness has been under direct selection.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We observed a significant correlation between our measures of genetic and environmental robustness, in agreement with the congruence hypothesis. However, this correlation alone cannot explain the co-variance of genetic robustness with position in the protein interaction network. We therefore conclude that direct selection on robustness has played a role in the evolution of genetic robustness in the transcriptome.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Proulx SR, Nuzhdin S, Promislow DEL (2007) Direct Selection on Genetic Robustness Revealed in the Yeast Transcriptome. PLoS ONE 2(9):
          e911.
            doi:10.1371/journal.pone.0000911</p><p><strong>Academic Editor: </strong>Jason Stajich, University of California at Berkeley, United States of America</p><p><strong>Received:</strong> June 7, 2007; <strong>Accepted:</strong> August 28, 2007; <strong>Published:</strong> September 19, 2007</p><p><strong>Copyright:</strong> © 2007 Proulx et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>DP is funded by a Senior Scholar award from the Ellison Medical Foundation. SN was funded by NIH grant 5R01GM076643.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Organisms are faced with the challenge of functioning and reproducing in the midst of both internal and external perturbations. Internal genetic changes arise due to mutation and recombination, while externally, organisms might experience a range of environments over various spatial and temporal scales, leading to variable selection pressures <a href="#pone.0000911-Levins1">[1]</a>, <a href="#pone.0000911-Dempster1">[2]</a>. How a species responds to these variable selection pressures depends on the details of the selective environment and the genetic variances and covariances for the traits under selection <a href="#pone.0000911-Steppan1">[3]</a>. The genetic architecture of the traits of interest and the nature of intrinsic or extrinsic variability will, in turn, determine whether fitness is maximized by, on the one hand, plastically varying phenotype to match the external environment or internal genetic background, or on the other hand by producing a constant, robust phenotype <a href="#pone.0000911-Debat1">[4]</a>, <a href="#pone.0000911-Promislow1">[5]</a>. The idea that phenotypic insensitivity to the environment might be under selection was advanced by Waddington, Schmalhausen, and Thoday in the 1940s and 50s <a href="#pone.0000911-Waddington1">[6]</a>–<a href="#pone.0000911-Thoday1">[9]</a>. Of these early ideas, Waddington's concept of developmental canalization has perhaps incited both the most debate and the most theoretical modeling <a href="#pone.0000911-deVisser1">[10]</a>–<a href="#pone.0000911-Hansen1">[12]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Theoretical models for the evolution of robustness have made clear the importance of distinguishing between environmental and genetic robustness (ER and GR, respectively) <a href="#pone.0000911-deVisser1">[10]</a>, <a href="#pone.0000911-Wagner1">[13]</a>–<a href="#pone.0000911-Proulx1">[16]</a>. We define environmental robustness as the insensitivity of a phenotype to environmental perturbations, while genetic robustness refers to the constancy of a phenotype when some component of the genotype is altered. In general, the strength of selection on robustness to some form of perturbation is limited by the fitness load that the perturbation can create <a href="#pone.0000911-Proulx1">[16]</a>. However, theory suggests that the extent and causes of robustness evolution differ for ER versus GR. For environmental disturbances, selection on robustness depends on both the frequency and fitness cost associated with environmental changes <a href="#pone.0000911-Wagner1">[13]</a>, <a href="#pone.0000911-Proulx1">[16]</a>, <a href="#pone.0000911-Gavrilets1">[17]</a>. Unlike environmental perturbations, the strength of selection on GR is not strongly related to the fitness cost associated with each genetic perturbation, but is instead related to the fraction of the overall mutational load that can be buffered by the focal gene <a href="#pone.0000911-deVisser1">[10]</a>, <a href="#pone.0000911-Wagner1">[13]</a>, <a href="#pone.0000911-Proulx1">[16]</a>. This is because the mutation load is largely insensitive to the per-mutation fitness effect <a href="#pone.0000911-Crow1">[18]</a>. This leads us to predict that selection for genetic robustness will not be related to any measure of how important the trait of interest is to overall fitness. Rather, selection for genetic robustness is predicted to be related to the total frequency of deleterious effects that the focal gene can potentially buffer <a href="#pone.0000911-Proulx1">[16]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Given these theoretical results, we expect mutational robustness to be typically weakly selected while environmental robustness should be under strong selection <a href="#pone.0000911-deVisser1">[10]</a>, <a href="#pone.0000911-Wagner1">[13]</a>, <a href="#pone.0000911-Proulx1">[16]</a>, <a href="#pone.0000911-Stearns1">[19]</a>–<a href="#pone.0000911-Burch1">[22]</a>. Several well known systems exhibit genetic robustness, but it is often linked to the mechanism of environmental robustness <a href="#pone.0000911-Barkai1">[23]</a>–<a href="#pone.0000911-vonDassow1">[26]</a>. What, then, might account for the evolution of GR? Two main hypotheses have been proposed. The first, known as the congruence hypothesis, argues that genetic robustness has arisen as a byproduct of environmental robustness <a href="#pone.0000911-deVisser1">[10]</a>, <a href="#pone.0000911-Stearns2">[20]</a>, <a href="#pone.0000911-Ancel1">[25]</a>, <a href="#pone.0000911-Meiklejohn1">[27]</a>, <a href="#pone.0000911-Stearns3">[28]</a>. An alternative hypothesis, direct selection, argues that selection can act directly on both mutational and environmental robustness, and that selection has been strong enough to have an observable effect on mutational robustness in particular <a href="#pone.0000911-deVisser1">[10]</a>, <a href="#pone.0000911-Wagner1">[13]</a>, <a href="#pone.0000911-Proulx1">[16]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>These two hypotheses lead to different sets of predictions. Under the congruence hypothesis, we expect that the traits that are most robust to environmental perturbations will also be those that are robust to genetic perturbations. This could occur, for instance, if enzymes that are selected to have excess capacity in order to produce consistent output under variable environmental conditions also turn out to have excess capacity in the face of genetic perturbations that reduce enzyme efficiency <a href="#pone.0000911-Meiklejohn1">[27]</a>. Furthermore, traits with high GR and ER will be those that are under the strongest selection.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The direct selection hypothesis argues that natural selection can favor both ER and GR independently. However, the means by which this occurs differs for the two types of robustness. Traits that are strongly correlated with fitness are expected to evolve high ER. In contrast, traits with high GR will be those that have the greatest potential to buffer other traits. For example, if we measure robustness in levels of gene expression across genes in the genome, theory predicts that GR will be greatest for genes whose protein products have the potential to buffer the largest number of mutations. Proteins that interact with the largest number of other proteins in the protein-protein interaction network could potentially buffer mutations at each interacting locus, and are therefore predicted to be under stronger selection to create robustness (<a href="#pone-0000911-g001">Figure 1</a>). The direct selection hypothesis does not predict that ER will be strongly correlated with network structure, although we might expect to see some relationship due to the fact that highly connected proteins tend to be under stronger selection <a href="#pone.0000911-Hahn1">[29]</a>, though the correlation is relatively weak. The ability of a particular gene to buffer or amplify environmental variability is likely to depend on both direct interactions with the environment and indirect interactions, such as through signal transduction pathways. However, the fitness effects of altering expression at a focal gene could easily swamp out the effects of network position or, at a minimum, reduce the importance of network position. Thus, we have a distinct set of predictions that differentiate GR from ER, and congruence from direct selection.</p>
<div class="figure" id="pone-0000911-g001"><div class="img"><a name="pone-0000911-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000911.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000911.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>A schematic diagram illustrating the role that network position can play in propagation of noise through the network.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>The circles in the diagram represent genes while the lines represent bi-directional interactions (like protein-protein interactions). The diagram shows how a focal node, shown in red, can affect noise produced by a perturbed node, shown in blue. The noise produced by the blue gene is represented by the blue oscillating arrows, and is dampened after passing through the red gene. Because the red gene lies on pathways between many other genes it has a large potential to buffer genetic noise.</p>
<span>doi:10.1371/journal.pone.0000911.g001</span></div><a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>While we have a clear body of theory and predictions, until now it has been difficult to obtain measures of genetic robustness, environmental robustness, selection intensity and network connectivity for a large number of traits in a single species. For example, detailed work on the segment polarity network of <em>Drosophila</em> has shown how genetic and environmental robustness are related in that specific network <a href="#pone.0000911-vonDassow1">[26]</a>. Induced mutation in <em>Drosophila</em> suggests that, for life history traits, both genetic and environmental robustness are correlated with the traits importance to fitness <a href="#pone.0000911-Stearns2">[20]</a>. The structure of the chemotaxis network of bacteria has been shown to produce robustness to both genetic mutation and environmental perturbation <a href="#pone.0000911-Barkai1">[23]</a>. Likewise, heat shock proteins have been shown to buffer both temperature effects and individual mutations in proteins that the chaperone interacts with <a href="#pone.0000911-Rutherford1">[24]</a>. However, each of these cases concerns robustness of a single trait. If we could study robustness at multiple traits simultaneously, we would have much more power to test theories of robustness. Datasets obtained from large-scale genomic studies now make this possible.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Here we use gene transcription data from the yeast, <em>Saccharomyces cerevisiae</em>, to test hypotheses for the evolution of robustness. By defining our traits of interest as the level of gene expression, a single organism provides us with measures of robustness for over 5000 traits (i.e., genes) simultaneously. While RNA production alone does not constitute a classical trait, it is an important step in producing functional proteins that contribute to adaptation. This gives us substantial power to examine the statistical relationships between robustness and potential causal variables using a common framework. For each trait, the phenotype within a given environment or genetic background is simply the level of gene expression relative to that of a control strain. Robustness for a single gene is defined as the relative constancy (i.e., the inverse of the variance) of that gene, measured across a series of environments or genotypes.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>Our measures of ER and GR are derived from measures of variation in levels of gene expression across 15 different stressors in a total of 35 environments [ER, 30], and across a set of 167 non-lethal knockout mutations [GR, 31]. Gene knockouts are a particularly severe form of genetic perturbation, and are probably rare in nature. Cellular responses to such large perturbations could be different from responses to smaller-scale genetic changes. Accordingly, we also obtained measures of variation in gene expression across a set of 30 wild yeast strains collected from vineyards in California (genetic background robustness, or ‘BR’). These strains represent genomes that are capable of living and reproducing in the environment, and so represent a particularly interesting kind of genetic variation. These wild yeast strains were brought into the laboratory and then grown under identical conditions before RNA was extracted. Therefore, expression levels measured for each gene show their response to a change in the genetic background.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>We found that these three forms of robustness were significantly correlated at a genomic scale. However, environmental and genetic robustness differ in their relationships to putative causal variables that represent both the evolutionary history and network position of genes. These causal variables include essentiality (whether a gene knockout is lethal or not), evidence of purifying selection as measured by <em>K<sub>a</sub></em>/<em>K<sub>s</sub></em> ratios, and knockout growth effects, which we assume to be measures of ‘gene importance’. We also include several measures of the structure of protein networks and gene regulatory networks <a href="#pone.0000911-Promislow1">[5]</a>, <a href="#pone.0000911-Hahn1">[29]</a>, <a href="#pone.0000911-Giaever1">[32]</a>. Environmental robustness was significantly correlated with several measures of gene importance, while genetic and background genotype robustness were correlated with position in the protein network. While we cannot rule out the possibility that congruence explains a portion of the observed genetic robustness, our analysis provides evidence that direct selection has a measurable effect on genetic robustness.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Correlations between traits</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We found that environmental robustness was positively and significantly correlated with both measures of genetic robustness (GR and BR) (<a href="#pone-0000911-g002">figure 2</a>), and that all three measures of robustness were positively correlated with one another (Spearman's <em>ρ</em>, ER-GR: <em>ρ</em> = 0.26, ER-BR: <em>ρ</em> = 0.13, GR-BR: <em>ρ</em> = 0.18; <em>P</em>&lt;10<sup>−20</sup> in all cases). In addition to the non-parametric correlations, we used linear regression to determine the correlations between each of the three measures of robustness. Each of the three possible pairwise regressions were highly significant with p&lt;10<sup>−20</sup>.</p>
<div class="figure" id="pone-0000911-g002"><div class="img"><a name="pone-0000911-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000911.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000911.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The relationship between environmental expression robustness and two forms of genetic robustness.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The data were separated into 15 bins based on ranked environmental robustness. For each bin the mean and standard error of each form of robustness was calculated. Filled squares indicate robustness to knockouts while open squares indicate robustness to background genotype. All statistical analyses were carried out on the un-binned data.</p>
<span>doi:10.1371/journal.pone.0000911.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The congruence hypothesis posits that both GR and BR are byproducts of selection on ER. If GR and BR had evolved solely in response to ER, we would expect that GR and BR would no longer be correlated after removing the effects of ER through partial regression. Put another way, under the direct selection hypothesis, we expect a correlation between GR and BR even after controlling for the statistical effects of ER. The two measures of genetic robustness are, in fact, correlated with one another after removing the effects of ER. We also used both non-parametric and regression based approaches to assess the partial correlation between GR and BR. We performed a multiple regression with ER and BR as factors and GR as the response variable. The model was highly significant with p&lt;10<sup>−30</sup> and had partial regression coefficients that were significantly positive. In particular, the two measures of genetic robustness were positively related (β = 0.16, s.e. = 0.014). Further, the multiple regression explained 8.6% of the total variance. We also took another, partially non-parametric approach to calculate the correlation between GR and BR. We computed residual GR and BR from a linear regression against ER. Their residual values are highly correlated with a coefficient similar to that of their raw values (<a href="#pone-0000911-g003">figure 3</a>; Spearman's <em>ρ</em> = 0.15, p&lt;0.0001).</p>
<div class="figure" id="pone-0000911-g003"><div class="img"><a name="pone-0000911-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000911.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000911.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Correlation between residual GR and residual BR.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>We independently fit GR and BR to ER and performed a linear regression. Residual values of GR and BR were calculated and are shown here. The measures are significantly correlated with Spearman's <em>ρ</em> of 0.15 and a Pearson's r of 0.16.</p>
<span>doi:10.1371/journal.pone.0000911.g003</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The direct selection hypothesis predicts that only the environmental robustness of a trait should be correlated with the intensity of selection acting on that trait, while the congruence hypothesis predicts that both ER and GR should be correlated with the intensity of selection. We used three measures of gene importance as proxies for the intensity of selection; whether a gene was lethal when knocked out, the <em>K<sub>a</sub>/K<sub>s</sub></em> ratio and whether colony growth was affected by heterozygous gene knock-outs. We found that all three types of robustness were higher in genes that are lethal when knocked out (<a href="#pone-0000911-t001">Table 1</a>). However, the other measures of gene importance were correlated with ER but not with GR or BR. While the lethality data suggested that genes that are more important determinants of fitness have greater environmental robustness, our analysis of <em>K<sub>a</sub>/K<sub>s</sub></em> showed the opposite pattern. Genes that historically have experienced the strongest intensity of selection (i.e., low <em>K<sub>a</sub>/K<sub>s</sub></em> values) showed the <em>least</em> robustness. The relationship between the effect of a heterozygous gene knockout on growth rate and ER depended on the medium. Genes that reduce growth in complete media are associated with lower robustness while genes that reduce growth in minimal media are associated with higher robustness.</p>
<div class="figure" id="pone-0000911-t001"><div class="img"><a name="pone-0000911-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000911.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000911.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Predictors of robustness (correlation coefficient, ±s.e.).</span></strong></p><span>doi:10.1371/journal.pone.0000911.t001</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>We also tested whether genes that have environment specific growth responses show a relationship with expression robustness. We defined a gene as having a differential growth effect if a heterozygous knockout mutant caused lower growth in one media (YPD or minimal) but not the other <a href="#pone.0000911-Giaever1">[32]</a>. We found a significant relationship between ER and differential growth, with differential growth associated with low robustness (F = 12.76, DF = 1, p&lt;0.001). In contrast, GR and BR were not significantly associated with differential growth (GR F = 1.24, DF = 1, p = 0.26; BR F = 0.0024, DF = 1, p = 0.96).</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>Under the congruence hypothesis, GR and BR evolve as byproducts of ER and would therefore not be expected to have statistical relationships with gene importance once the effects of ER were statistically controlled for. We calculated the residual values of GR and BR from linear regressions against ER and used those residual robustness values to test for effects on each of our measures of gene importance. Lethal genes were associated with higher residual robustness. In contrast, high <em>K<sub>a</sub>/K<sub>s</sub></em> was associated with lower residual robustness. This indicates that genes under stronger purifying selection are more genetically robust than expected, given their level of ER (GR: F = 6.61, p = 0.01, slope = −0.018 (0.0076); BR: F = 17.89, p&lt;0.0001, slope = −0.031 (0.0074) ). Colony growth and differential colony growth were not significantly associated with either measure of residual genetic robustness.</p>


<h4>Network structure</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The direct selection hypothesis for the evolution of robustness predicts that genetic, but not environmental, robustness should be correlated with the ability of a gene to buffer changes in a network of interacting genes. Of the three measures of network centrality (degree, closeness and betweenness), ER was positively correlated with degree but not with closeness or betweenness (<a href="#pone-0000911-t002">table 2</a>). In contrast, both GR and BR were correlated with all three measures of network centrality (<em>P</em>&lt;10<sup>−7</sup> in all cases). More central and higher degree proteins had higher robustness than less central proteins. After adjusting for multiple comparisons using a Bonferroni adjusted critical α = 0.0056, however, only GR and BR were significantly correlated with any measure of network position.</p>
<div class="figure" id="pone-0000911-t002"><div class="img"><a name="pone-0000911-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000911.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000911.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Spearman's <em>ρ</em> correlation between measures of robustness and protein centrality.</span></strong></p><span>doi:10.1371/journal.pone.0000911.t002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The number of genes that regulate a focal gene is known to be positively correlated with ER <a href="#pone.0000911-Promislow1">[5]</a>. The number of regulators was also positively correlated with both GR and BR. Position in the protein network is also correlated with position in the regulatory network, and this could produce the observed relationship between robustness and protein centrality. Because the number of regulatory factors has been previously shown to effect expression robustness <a href="#pone.0000911-Promislow1">[5]</a>, we wanted to ensure that these observed patterns were not simply caused by correlations with the number of regulatory binding sites (and putative regulators) at a gene (K<sub>in</sub>). All measures of robustness were negatively correlated with Log-transformed values of K<sub>in</sub> (ER <em>β</em>  = −0.070±0.013, GR <em>β</em> = −0.11±0.014, BR <em>β</em> = −0.037±0.013). In order to statistically correct for the effect of K<sub>in</sub> we first performed linear regression with of each measure of robustness against Log K<sub>in</sub> and calculated the residual robustness. We then measured the correlation between residual robustness and network position and found that more central proteins had higher residual GR and BR (<em>P</em>&lt;0.00001 for degree, <em>P</em>&lt;0.05 for closeness and betweenness). No measure of centrality was correlated with residual ER.</p>


<h4>Relationship to Protein Variability</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We obtained data from <a href="#pone.0000911-Newman1">[33]</a> on cell-to-cell variation in protein abundance. These data were collected from a library of fluorescently tagged yeast strains in a constant environment. Data were available both for yeast raised in complete (YPD) and minimal media. We first calculated the residual of the log variance in protein abundance from a cubic spline fit (λ = 0.1) with log protein abundance (LRV). This allowed us to remove the effect of protein abundance <em>per se</em> and determine if proteins with noisier expression were associated with robustness. We calculated correlations between our measures of robustness and LRV using both Spearman's <em>ρ</em> and Pearson's r. <a href="#pone-0000911-t003">Table 3</a> shows that all measures of robustness are always significantly negatively associated with protein variability. We performed a multiple regression with LRV and K<sub>in</sub> as factors and found that, for each measure of robustness, LRV still had a significant effect and that this effect was in the same direction as the pairwise correlation (<a href="#pone-0000911-t004">Table 4</a>). Thus, genes that had relatively high levels of variation in expression between environments and genetic backgrounds also had relatively high levels of variation in protein abundance within a single environment.</p>
<div class="figure" id="pone-0000911-t003"><div class="img"><a name="pone-0000911-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000911.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000911.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Correlations between robustness and log residual protein abundance (LRV).</span></strong></p><span>doi:10.1371/journal.pone.0000911.t003</span></div><div class="figure" id="pone-0000911-t004"><div class="img"><a name="pone-0000911-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000911" data-uri="info:doi/10.1371/journal.pone.0000911.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000911.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000911.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000911.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000911.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Multiple regression results for robustness as predicted by log protein abundance (LRV) and K<sub>in</sub>.</span></strong></p><span>doi:10.1371/journal.pone.0000911.t004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Two competing hypotheses—direct selection and congruence—have been put forward to explain why some traits might be more robust than others when faced with environmental or genetic perturbations. Our study of variation in transcription levels in the yeast genome provides us with the first large-scale and simultaneous test of these two hypotheses.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Our results provide support, albeit mixed, for both hypotheses. In line with the congruence hypothesis, all three measures of robustness (ER, GR and BR) were correlated with one another. However, if congruence alone were responsible for the evolution of GR and BR, then we would not expect GR and BR to be independently correlated with other causal variables. Since we find that GR and BR are correlated with each other once the effects of ER are statistically removed, that GR and BR are not correlated with <em>K<sub>a</sub>/K<sub>s</sub></em> or growth effects, and that GR and BR are each correlated with network position, we conclude that there has been direct selection for genetic robustness.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We measured several traits associated with the relative importance to fitness of each gene. The direct selection hypothesis predicts that only ER should be correlated with measures of gene importance. In fact, we found that ER was significantly correlated with each measure, whereas GR and BR were only correlated with lethality. On the face of it, this would appear to support the direct selection hypothesis. However, one result was unexpected, and in direct contrast to our prediction. The direct selection hypothesis predicts that robustness should be greatest in genes that exhibit the highest measures of selection intensity. In fact, we found just the opposite. Genes with low <em>K<sub>a</sub></em>/<em>K<sub>s</sub></em> ratios and genes that lowered growth rate when knocked out in minimal media had low environmental robustness. We return to this result later in the discussion.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Further supporting the direct selection hypothesis, GR and BR were more strongly correlated than was ER with measures of network centrality. This is in line with the theoretical prediction that genes that are better able to buffer genetic mutations at other loci will evolve higher genetic robustness. Our results show that more central proteins have higher GR and BR, indicating that highly connected genes maintain relatively constant levels of mRNA expression under a range of genetic perturbations. This would lead to robustness in terms of biological function if the presence of these robustly expressed proteins could buffer or compensate for changes in abundance or sequence of other proteins with which they interact.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Perhaps the most surprising result was that genes with high environmental robustness appeared to be under less intense selection than those that varied more across environments. This may seem to indicate that natural selection is favoring a lack of robustness. However, an alternative interpretation of these results is possible if we think of high variability across environments not as low robustness, but rather as high, and potentially adaptive, phenotypic plasticity. Phenotypic plasticity can evolve as an adaptive response to variation in the environment when selection favors alternative phenotypes in different environments <a href="#pone.0000911-Debat1">[4]</a>, <a href="#pone.0000911-Promislow1">[5]</a>. Thus, our observation that genes under strong stabilizing selection also have low ER is consistent with the evolution of adaptive plasticity in expression levels.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The observation that low <em>K<sub>a</sub></em>/<em>K<sub>s</sub></em> was associated with low ER suggested that our analysis of heterozygous knock-outs may have been incomplete. If the optimal level of expression of a gene were environment specific, then we would expect that that gene would have environment specific effects on fitness when its expression was artificially altered. To this end, we defined a gene as having a differential growth effect if a heterozygous knockout mutant caused lower growth in one growth media but not the other. Genes that have differential growth effects are likely to have different optimal expression levels in different environments, because the experimental protocol manipulates expression level in an environment independent way. We found that low ER was associated with differential growth effects, suggesting that genes that are highly variable in expression with respect to environment are likely to have environment specific fitness consequences when perturbed.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Previous work has suggested that plasticity in expression is functionally related to the number of transcription factors that regulate a focal gene <a href="#pone.0000911-Promislow1">[5]</a>. Because transcription factor genes are themselves responsive to changes in environmental conditions, it is not surprising that genes with more regulatory inputs have increased environmental plasticity. We found that genetic plasticity also increased with the number of regulatory inputs, although to a lesser degree than for environmental plasticity. On the other hand, genes that are more connected and more central in the protein interaction network have increased expression robustness to genetic perturbations even when we controlled for the number of regulatory inputs.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Further evidence regarding the congruence hypothesis is provided by analyzing the residual levels of GR and BR from a regression with ER. If GR and BR had evolved solely as a by-product of ER, then we would also expect that their residual values would have no relationship with lethality, purifying selection, or differential growth. Surprisingly, high residual GR and BR was associated with low <em>K<sub>a</sub>/K<sub>s</sub></em>, opposite to the pattern that we saw with raw ER and <em>K<sub>a</sub>/K<sub>s</sub></em>. Because genes with lower <em>K<sub>a</sub>/K<sub>s</sub></em> values had higher residual genetic robustness, we can infer that there is weak canalizing selection on expression robustness to genetic perturbations, but that this is often overwhelmed by selection for environmental plasticity. This implies that observed robustness represents a balance between congruence acting to allow transcription responses to environmental change and direct selection of genetic robustness to remain insensitive to genetic perturbations.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>We were also interested in determining if within-environment variability in expression explained our robustness data. We examined the correlation between robustness and protein variability among cells as measured by <a href="#pone.0000911-Newman1">[33]</a>. We found that relative variance in protein expression, when the effects of protein abundance were removed, was negatively correlated with robustness, indicating that genes with greater robustness had relatively lower cell-to-cell variability in protein abundance. One possible explanation for this relationship is that genes that have larger numbers of regulators are more responsive to changes, both because of stochastic variation in a constant environment and in response to larger magnitude environmental and genetic changes <a href="#pone.0000911-Promislow1">[5]</a>. However, a multiple regression with the number of regulators and protein variability as factors predicting both forms of robustness showed an independent effect of protein variability on robustness. This surprising result reinforces our conclusion that variability in gene expression represents the outcome of evolutionary pressures to maintain robust expression under some situations while allowing plastic expression under others.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>While we found a highly significant correlation between network position and genetic robustness, the rank correlation coefficients were in the range of 0.05 to 0.11, leaving much variation in robustness unexplained. Our measures of network position were relatively crude and required separate analysis of information from the protein network and transcription network. An important future goal is to develop a better understanding of the topological features of gene networks that make them vulnerable to damage <a href="#pone.0000911-Proulx2">[34]</a>. This approach is likely to provide additional insight into the complex relationships between genetic and environmental robustness.</p>

<h4>Final Thoughts</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We used whole genome expression data to evaluate two competing hypotheses for the evolution of expression robustness. The congruence hypothesis posits that selection for environmental robustness leads to the evolution of genetic robustness as a correlated response. In contrast, the direct selection hypothesis posits that selection acts independently on environmental and genetic robustness. In their strictest interpretation, these hypotheses have different predictions. However, direct selection on both forms of robustness could potentially create unexpected auxiliary correlations. For instance, if the same sets of genes were under selection to become both more environmentally and more genetically robust, then we might expect correlations between our measures of environmental and genetic robustness even without correlated evolution. Likewise, since measures of gene importance are often correlated with network position, direct selection on ER might produce a correlation between ER and network position. These effects would only cloud our analysis if the correlation between gene importance and network position were tight, but regressions of protein degree against lethality, <em>K<sub>a</sub>/K<sub>s</sub></em>, and heterozygous knockout growth rate have R<sup>2</sup> values less than seven percent. In addition, we did find significant relationships between measures of genetic robustness and measures of network position that could not be explained by congruence alone.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>While we cannot entirely rule out the possibility that direct selection on both forms of robustness is responsible for all of our results, we can rule out the possibility that congruence alone explains the observed pattern of genetic robustness. First, we observed a strong correlation between GR and BR even when the effect of ER was statistically controlled for. For this to be explained by the congruence hypothesis it would need to be the case that the correlated responses of both GR and BR with ER had a common mechanistic basis. Second, GR and BR did not show the same pattern of correlation as ER with our measures of gene importance. This suggests that, to the extent that correlated evolution plays a role in genetic robustness evolution, it is not any more intense when direct selection is acting on ER. Third and most significantly, GR and BR showed tighter correlations with measures of protein network centrality than did ER, and this pattern is only predicted by the direct selection hypothesis. Further, when we statistically controlled for the effect of the number of regulatory inputs on expression robustness, only GR and BR were significantly correlated with network position.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Taken together then, our results show a clear signature of direct selection acting on genetic robustness and further suggest that similar forces act on robustness to knockouts and robustness to changes in the genetic background. While we observe a correlation between our measures of environmental robustness and genetic robustness, the pattern of correlation between the putative causal variables (gene importance and network position) and each form of robustness is intriguing. In particular, while all measures of gene importance were correlated with environmental robustness, only lethality was correlated with genetic robustness. Conversely, all measures of network centrality were strongly correlated with GR and BR, but only network degree was correlated with environmental robustness, and the correlation was weak. Finally, there was no significant correlation between <em>K<sub>a</sub></em>/<em>K<sub>s</sub></em> and absolute measures of GR, but for a given value of ER, genes that have higher GR are under stronger purifying selection.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>How can we understand this suite of results? One possible explanation is that environmental and genetic robustness share a common mechanistic basis, but the direction of selection acting on these two traits may be divergent. Thus, genes under weaker selection for robustness would contribute to a positive correlation between environmental and genetic robustness. Genes under strong and divergent selection for robustness would reveal distinct correlations between each form of robustness and variables that are related to selection on robustness. For expression robustness, variation in the number and type of transcription regulators could be a mechanism that affects both expression robustness and adaptive plasticity <a href="#pone.0000911-Promislow1">[5]</a>. For instance, when a gene acquires additional regulatory elements that allow it to respond adaptively to environmental change, it may necessarily make expression of that gene sensitive to genetic changes at other loci, a trade-off predicted by recent complex systems theory <a href="#pone.0000911-Carlson1">[35]</a>. If this trade-off is an unavoidable feature of gene expression networks, then it may well be that one cost of phenotypic plasticity is a reduction in genetic robustness.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Expression Data</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We analyzed expression for the yeast <em>Saccharomyces cerevisiae</em> that had been exposed to environmental perturbations (ER), gene knockouts (GR), and changes in genetic background (BR). We measured the expression variability by calculating the mean and variance of expression changes of each gene in the yeast genome in response to each form of perturbation. In all datasets analyzed here, we find a strong mean-variance correlation—genes with large changes in expression levels relative to controls also show increased variance. We controlled for the possible bias created by the mean-variance correlation by calculating the residual of variance in expression versus mean expression using a cubic spline fit (λ = 0.1). Thus, the corrected measure of variance for the <em>i</em>th gene is Vc<sub>i</sub>. We calculated robustness as the relative invariance of expression by linearly transforming each measure of residual variance so that values with the lowest variance had a robustness value of 1 while values with the highest variance had robustness values of 0. Thus, the robustness of the <em>i</em>th gene is given by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000911.e001&amp;representation=PNG" class="inline-graphic"></span><br>where the minimum and maximum functions are taken over all genes in our sample. We independently performed this transformation on each of our gene expression datasets to obtain ER, GR, and BR.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Data on gene expression were obtained from Gasch <em>et al</em>. <a href="#pone.0000911-Gasch1">[30]</a> and Hughes <em>et al</em>. <a href="#pone.0000911-Hughes1">[31]</a>. Additional experiments, described below, were performed by S. Nuzhdin using wild yeast strains from the UC Davis collection. Gasch <em>et al</em>. and Hughes <em>et al</em>. reported expression ratios of unique mRNA sequences in response to genetic and environmental perturbations as compared to a wild-type strain. The Gasch <em>et al</em>. dataset included 6,152 genes while the Hughes <em>et al</em>. dataset included 6,287 genes. We limited our analysis to genes that correspond to known proteins as listed in the Comprehensive Yeast Genome Database (<a href="http://mips.gsf.de/genre/proj/yeast/">http://mips.gsf.de/genre/proj/yeast/</a>)<a href="#pone.0000911-Gldener1">[36]</a>, reducing our dataset to 5266 genes.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>To calculate ER, we used the Gasch data on expression ratios for the set of environmental perturbations to calculate the mean expression ratio and the variance in the expression ratio for each gene. This included 167 experiments with 15 specific stressors, including heat shock, hyper- and hypo-osmolarity, and a number of chemical exposure treatments. We selected a subset of environmental perturbations from the dataset consisting of 35 experiments and 15 stressors. These experiments were selected to minimize pseudoreplication of similar environmental perturbations. For example, only one experiment that involved a temperature shift from 37°C. to 25°C. was retained out of five such experiments. The dataset that we used can be obtained from <a href="http://www-genome.stanford.edu/yeast_stress/data/rawdata/complete_dataset.txt">http://www-genome.stanford.edu/yeast_str​ess/data/rawdata/complete_dataset.txt</a>. The experiments that we used correspond to columns 11, 18, 20, 24, 35, 38, 48, 57, 65, 70, 80, 87, 92, 93, 96, 101, 108, 110, 116, 125, 126, 137, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 163. Descriptions of these experiments can be found at <a href="http://www-genome.stanford.edu/yeast_stress/materials.pdf">http://www-genome.stanford.edu/yeast_str​ess/materials.pdf</a> <a href="#pone.0000911-Gasch1">[30]</a>.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>To calculate GR we used the Hughes <em>et al</em>. dataset, which included 276 unique gene knockout experiments. We calculated the mean expression ratio and the variance in the expression ratio over all experiments. We calculated BR using 30 wild yeast strains of <em>S. cerevisiae</em>.</p>
<a id="article1.body1.sec4.sec1.p5" name="article1.body1.sec4.sec1.p5"></a><p>If we find that genes differ in how variable their expression levels are across genotypes or environments, this difference could be due to intrinsic differences in variability among genes, and not to their response to environmental or genetic perturbations. Accordingly, we also compared our measures of robustness with intrinsic measures of variability for protein levels within a constant environment, using data from <a href="#pone.0000911-Newman1">[33]</a>. We tested for rank correlations between each of our measures of expression robustness and the coefficient of variation for protein expression (CV) measured in both complete and minimal media.</p>


<h4>Expression analysis of wild <em>S. cerevisiae</em> stocks</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The genetic background data (GR) were collected from 30 stocks from the UC Davis collection of natural <em>S. cerevisiae</em> (kindly provided by Linda Bisson, UC Davis). Microarrays were performed between overlapping pairs of strains in a dye-swapped design. Data were normalized to correct for dye effects using Agilent software. The expression of each gene was inferred using an ANOVA technique to estimate the expression level of each gene in each strain <a href="#pone.0000911-Kerr1">[37]</a>. This allowed us to estimate the effect of each genetic background on the expression of each gene and calculate the variance in log expression over all strains (See supplemental <a href="#pone.0000911.s001">Data S1</a>). We also calculated the mean expression level of each gene. Because these analyses did not include a single control wild-type strain to generate log expression ratios, the mean expression across all strains was used as the baseline expression value for each gene.</p>


<h4>Gene Importance</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>We used three approaches to determine how important each gene is to fitness in a yeast cell. First, genes were classified as viable if a knockout mutant could grow and survive. Viability data were obtained from the Comprehensive Yeast Genome Database <a href="#pone.0000911-Gldener1">[36]</a>. Second, we obtained data on the historical strength of purifying selection as measured by <em>K<sub>a</sub>/K<sub>s</sub></em> ratios, estimated in reference <a href="#pone.0000911-Kellis1">[38]</a> (data available at <a href="http://ftp://ftp-genome.wi.mit.edu/pub/annotation/fungi/comp_yeasts/S4.MutationCounts/b.KaKs_details.xls">ftp://ftp-genome.wi.mit.edu/pub/annotati​on/fungi/comp_yeasts/S4.MutationCounts/b​.KaKs_details.xls</a>] . Because there is a significant relationship between <em>K<sub>a</sub>/K<sub>s</sub></em> ratio and viability (d.f. = 2746, <em>t</em> = 7.93, p&lt;0.0001) we performed ANCOVA with viability as a fixed effect and log<sub>10</sub> <em>K<sub>a</sub>/K<sub>s</sub></em> as the covariate. Third, we used data from <a href="#pone.0000911-Giaever1">[32]</a> to determine which genes had significant effects on colony growth when their expression levels were altered, assuming that heterozygous knockouts have reduced expression. We used measurements of the reduced growth rate of heterozygous knockout lines in both complete and minimal media. We used a linear model with the four measures of gene importance as causal variables to test for effects on each form of robustness.</p>


<h4>Network Position</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We measured network statistics of genes in both the yeast gene regulatory network <a href="#pone.0000911-Lee1">[39]</a> and the yeast protein-protein interaction network <a href="#pone.0000911-Uetz1">[40]</a>. The regulatory network is a directional network with a small number of regulators (94 genes used in this study) and a larger number of regulated genes (1482 genes used in this study, 31 of which were also regulators). We calculated both the out degree, K<sub>out</sub> (number of genes regulated by the focal gene) and the in degree, K<sub>in</sub>. We used the dataset from Lee et al. to find predicted regulatory interactions at the <em>P</em> = 0.001 level.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Data on physical protein interactions were obtained from Yeast Grid (<a href="http://biodata.mshri.on.ca/yeast_grid/servlet/SearchPage">http://biodata.mshri.on.ca/yeast_grid/se​rvlet/SearchPage</a>) <a href="#pone.0000911-Breitkreutz1">[41]</a>. We excluded data based on synthetic lethal and dosage lethal interactions because they are not necessarily based on physical interactions and have not been systematically determined. Our network contained 4,692 genes involved in 15,035 interactions <a href="#pone.0000911-Uetz1">[40]</a>, <a href="#pone.0000911-Krogan1">[42]</a>–<a href="#pone.0000911-FromontRacine1">[48]</a>. We used Pajek <a href="#pone.0000911-deNooy1">[49]</a> to calculate the degree, betweenness, and closeness of all genes in the network.</p>


<h4>Statistical Methods</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>All statistical calculations were performed using JMP 5.1.2 (SAS Institute). We calculated the Spearman's <em>ρ</em> statistic to determine non-parametric correlations of untransformed data. For regressions and all other parametric tests we used transformed data to minimize deviations from normality. Expression variance was log transformed before calculating the cubic spline residuals. Log<sub>10</sub> transformations were also performed on <em>K<sub>a</sub>/K<sub>s</sub></em> ratios, K<sub>in</sub>, and K<sub>out</sub>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000911"></div><a name="pone.0000911.s001" id="pone.0000911.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000911.s001">Data S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Wild yeast strain expression data.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000911.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(3.66 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Linda Bisson for providing yeast strains from the UC Davis collection of natural yeast isolates. The manuscript was greatly improved by comments from two anonymous reviewers.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Performed the experiments: SN. Analyzed the data: DP SP. Wrote the paper: DP SP. Other: Discussed analysis of data: SN.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000911-Levins1" id="pone.0000911-Levins1"></a>Levins R, MacArthur R (1966) Maintenance of Genetic Polymorphism in a Spatially Heterogeneous Environment-Variations On a Theme by Howard Levene. American Naturalist  100: 585–589.  <ul class="find" data-citedArticleID="993860" data-doi="10.1086/282452"><li><a href="http://dx.doi.org/10.1086/282452" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maintenance+of+Genetic+Polymorphism+in+a+Spatially+Heterogeneous+Environment-Variations+On+a+Theme+by+Howard+Levene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maintenance+of+Genetic+Polymorphism+in+a+Spatially+Heterogeneous+Environment-Variations+On+a+Theme+by+Howard+Levene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000911-Dempster1" id="pone.0000911-Dempster1"></a>Dempster ER (1955) Maintenance of genetic heterogeneity. Cold Spring Harbor Symposia on Quantitative Biology  20: 25–32.  <ul class="find" data-citedArticleID="993820" data-doi="10.1101/sqb.1955.020.01.005"><li><a href="http://dx.doi.org/10.1101/sqb.1955.020.01.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maintenance+of+genetic+heterogeneity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maintenance+of+genetic+heterogeneity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000911-Steppan1" id="pone.0000911-Steppan1"></a>Steppan SJ, Phillips PC, Houle D (2002) Comparative quantitative genetics: evolution of the G matrix. Trends in Ecology &amp; Evolution  17: 320–327.  <ul class="find" data-citedArticleID="993884" data-doi="10.1016/s0169-5347(02)02505-3"><li><a href="http://dx.doi.org/10.1016/s0169-5347(02)02505-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+quantitative+genetics%3A+evolution+of+the+G+matrix." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+quantitative+genetics%3A+evolution+of+the+G+matrix.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000911-Debat1" id="pone.0000911-Debat1"></a>Debat V, David P (2001) Mapping phenotypes: canalization, plasticity and developmental stability. Trends in Ecology &amp; Evolution  16: 555–561.  <ul class="find" data-citedArticleID="993818" data-doi="10.1016/s0169-5347(01)02266-2"><li><a href="http://dx.doi.org/10.1016/s0169-5347(01)02266-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+phenotypes%3A+canalization%2C+plasticity+and+developmental+stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+phenotypes%3A+canalization%2C+plasticity+and+developmental+stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000911-Promislow1" id="pone.0000911-Promislow1"></a>Promislow D (2005) A regulatory network analysis of phenotypic plasticity in yeast. American Naturalist  165: 515–523.  <ul class="find" data-citedArticleID="993866" data-doi="10.1086/429161"><li><a href="http://dx.doi.org/10.1086/429161" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+regulatory+network+analysis+of+phenotypic+plasticity+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+regulatory+network+analysis+of+phenotypic+plasticity+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000911-Waddington1" id="pone.0000911-Waddington1"></a>Waddington CH (1942) The canalization of development and genetic assimilation of acquired characters. Nature  150: 563–565.  <ul class="find" data-citedArticleID="993890" data-doi="10.1038/150563a0"><li><a href="http://dx.doi.org/10.1038/150563a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+canalization+of+development+and+genetic+assimilation+of+acquired+characters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+canalization+of+development+and+genetic+assimilation+of+acquired+characters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000911-Waddington2" id="pone.0000911-Waddington2"></a>Waddington CH (1961) Genetic Assimilation. Advances in Genetics Incorporating Molecular Genetic Medicine  10: 257–293.  <ul class="find" data-citedArticleID="993892" data-doi="10.1016/s0065-2660(08)60119-4"><li><a href="http://dx.doi.org/10.1016/s0065-2660(08)60119-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+Assimilation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+Assimilation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000911-Schmalhausen1" id="pone.0000911-Schmalhausen1"></a>Schmalhausen II (1949) Factors of evolution. Chicago: University of Chicago Press.   <ul class="find-nolinks"></ul></li><li><span class="label">9.
              </span><a name="pone.0000911-Thoday1" id="pone.0000911-Thoday1"></a>Thoday JM (1953) Components of Fitness. Symposia of the Society for Experimental Biology  7: 96–113.  <ul class="find" data-citedArticleID="993886"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Components+of+Fitness.&amp;auth=&amp;atitle=Components+of+Fitness." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Components+of+Fitness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Components+of+Fitness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000911-deVisser1" id="pone.0000911-deVisser1"></a>de Visser J, Hermisson J, Wagner GP, Meyers LA, Bagheri HC, et al.  (2003) Perspective: Evolution and detection of genetic robustness. Evolution  57: 1959–1972.  <ul class="find" data-citedArticleID="993900" data-doi="10.1111/j.0014-3820.2003.tb00377.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2003.tb00377.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Perspective%3A+Evolution+and+detection+of+genetic+robustness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Perspective%3A+Evolution+and+detection+of+genetic+robustness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000911-Flatt1" id="pone.0000911-Flatt1"></a>Flatt T (2005) The evolutionary genetics of canalization. Quarterly Review of Biology  80: 287–316.  <ul class="find" data-citedArticleID="993822" data-doi="10.1086/432265"><li><a href="http://dx.doi.org/10.1086/432265" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+genetics+of+canalization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+genetics+of+canalization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000911-Hansen1" id="pone.0000911-Hansen1"></a>Hansen TF, Alvarez-Castro JM, Carter AJR, Hermisson J, Wagner GP (2006) Evolution of genetic architecture under directional selection. Evolution  60: 1523–1536.  <ul class="find" data-citedArticleID="993838" data-doi="10.1111/j.0014-3820.2006.tb00498.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2006.tb00498.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+genetic+architecture+under+directional+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+genetic+architecture+under+directional+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000911-Wagner1" id="pone.0000911-Wagner1"></a>Wagner GP, Booth G, Bagheri HC (1997) A population genetic theory of canalization. Evolution  51: 329–347.  <ul class="find" data-citedArticleID="993894" data-doi="10.2307/2411105"><li><a href="http://dx.doi.org/10.2307/2411105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+population+genetic+theory+of+canalization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+population+genetic+theory+of+canalization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000911-Kawecki1" id="pone.0000911-Kawecki1"></a>Kawecki TJ (2000) The evolution of genetic canalization under fluctuating selection. Evolution  54: 1–12.  <ul class="find" data-citedArticleID="993850" data-doi="10.1111/j.0014-3820.2000.tb00001.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2000.tb00001.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+genetic+canalization+under+fluctuating+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+genetic+canalization+under+fluctuating+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000911-Rice1" id="pone.0000911-Rice1"></a>Rice SH (1998) The evolution of canalization and the breaking of von Baer's laws: Modeling the evolution of development with epistasis. Evolution  52: 647–656.  <ul class="find" data-citedArticleID="993872" data-doi="10.2307/2411260"><li><a href="http://dx.doi.org/10.2307/2411260" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+canalization+and+the+breaking+of+von+Baer%27s+laws%3A+Modeling+the+evolution+of+development+with+epistasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+canalization+and+the+breaking+of+von+Baer%27s+laws%3A+Modeling+the+evolution+of+development+with+epistasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000911-Proulx1" id="pone.0000911-Proulx1"></a>Proulx SR, Phillips PC (2005) The opportunity for canalization and the evolution of genetic networks. American Naturalist  165: 147–162.  <ul class="find" data-citedArticleID="993868" data-doi="10.1086/426873"><li><a href="http://dx.doi.org/10.1086/426873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+opportunity+for+canalization+and+the+evolution+of+genetic+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+opportunity+for+canalization+and+the+evolution+of+genetic+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000911-Gavrilets1" id="pone.0000911-Gavrilets1"></a>Gavrilets S, Hastings A (1994) A quantitative-genetic model for selection on developmental noise. Evolution  48: 1478–1486.  <ul class="find" data-citedArticleID="993830" data-doi="10.2307/2410242"><li><a href="http://dx.doi.org/10.2307/2410242" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+quantitative-genetic+model+for+selection+on+developmental+noise." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+quantitative-genetic+model+for+selection+on+developmental+noise.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000911-Crow1" id="pone.0000911-Crow1"></a>Crow JF, Kimura M (1970) An introduction to population genetics theory. New York: Harper and Row.   <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0000911-Stearns1" id="pone.0000911-Stearns1"></a>Stearns SC, Kawecki TJ (1994) Fitness Sensitivity and the Canalization of Life-History Traits. Evolution  48: 1438–1450.  <ul class="find" data-citedArticleID="993878" data-doi="10.2307/2410238"><li><a href="http://dx.doi.org/10.2307/2410238" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitness+Sensitivity+and+the+Canalization+of+Life-History+Traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitness+Sensitivity+and+the+Canalization+of+Life-History+Traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000911-Stearns2" id="pone.0000911-Stearns2"></a>Stearns SC, Kaiser M, Kawecki TJ (1995) The Differential Genetic and Environmental Canalization of Fitness Components in <em>Drosophila melanogaster</em>. Journal of Evolutionary Biology  8: 539–557.  <ul class="find" data-citedArticleID="993880" data-doi="10.1046/j.1420-9101.1995.8050539.x"><li><a href="http://dx.doi.org/10.1046/j.1420-9101.1995.8050539.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Differential+Genetic+and+Environmental+Canalization+of+Fitness+Components+in+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Differential+Genetic+and+Environmental+Canalization+of+Fitness+Components+in+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000911-Wagner2" id="pone.0000911-Wagner2"></a>Wagner A, Stadler PF (1999) Viral RNA and evolved mutational robustness. J Exp Zool  285: 119–127.  <ul class="find" data-citedArticleID="993896" data-doi="10.1002/(sici)1097-010x(19990815)285:2<119::aid-jez4&gt;3.3.co;2-4"><li><a href="http://dx.doi.org/10.1002/(sici)1097-010x(19990815)285:2<119::aid-jez4&gt;3.3.co;2-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viral+RNA+and+evolved+mutational+robustness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Viral+RNA+and+evolved+mutational+robustness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000911-Burch1" id="pone.0000911-Burch1"></a>Burch CL, Chao L (2004) Epistasis and Its Relationship to Canalization in the RNA Virus phi 6. Genetics  167: 559–567.  <ul class="find" data-citedArticleID="993812" data-doi="10.1534/genetics.103.021196"><li><a href="http://dx.doi.org/10.1534/genetics.103.021196" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epistasis+and+Its+Relationship+to+Canalization+in+the+RNA+Virus+phi+6." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epistasis+and+Its+Relationship+to+Canalization+in+the+RNA+Virus+phi+6.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000911-Barkai1" id="pone.0000911-Barkai1"></a>Barkai N, Leibler S (1997) Robustness in simple biochemical networks. Nature  387: 913–917.  <ul class="find" data-citedArticleID="993808" data-doi="10.1038/43199"><li><a href="http://dx.doi.org/10.1038/43199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robustness+in+simple+biochemical+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robustness+in+simple+biochemical+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000911-Rutherford1" id="pone.0000911-Rutherford1"></a>Rutherford SL, Lindquist S (1998) Hsp90 as a capacitor for morphological evolution. Nature  396: 336–342.  <ul class="find" data-citedArticleID="993874" data-doi="10.1038/24550"><li><a href="http://dx.doi.org/10.1038/24550" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hsp90+as+a+capacitor+for+morphological+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hsp90+as+a+capacitor+for+morphological+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000911-Ancel1" id="pone.0000911-Ancel1"></a>Ancel LW, Fontana W (2000) Plasticity, evolvability, and modularity in RNA. J Exp Zool  288: 242–283.  <ul class="find" data-citedArticleID="993806" data-doi="10.1002/1097-010x(20001015)288:3<242::aid-jez5&gt;3.0.co;2-o"><li><a href="http://dx.doi.org/10.1002/1097-010x(20001015)288:3<242::aid-jez5&gt;3.0.co;2-o" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasticity%2C+evolvability%2C+and+modularity+in+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasticity%2C+evolvability%2C+and+modularity+in+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000911-vonDassow1" id="pone.0000911-vonDassow1"></a>von Dassow G, Meir E, Munro EM, Odell GM (2000) The segment polarity network is a robust developmental module. Nature  406: 188–192.  <ul class="find" data-citedArticleID="993902" data-doi="10.1038/35018085"><li><a href="http://dx.doi.org/10.1038/35018085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+segment+polarity+network+is+a+robust+developmental+module." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+segment+polarity+network+is+a+robust+developmental+module.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000911-Meiklejohn1" id="pone.0000911-Meiklejohn1"></a>Meiklejohn CD, Hartl DL (2002) A single mode of canalization. Trends in Ecology &amp; Evolution  17: 468–473.  <ul class="find" data-citedArticleID="993862" data-doi="10.1016/s0169-5347(02)02596-x"><li><a href="http://dx.doi.org/10.1016/s0169-5347(02)02596-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+mode+of+canalization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+mode+of+canalization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000911-Stearns3" id="pone.0000911-Stearns3"></a>Stearns SC (2002) Progress on canalization. Proceedings of the National Academy of Sciences of the United States of America  99: 10229–10230.  <ul class="find" data-citedArticleID="993882" data-doi="10.1073/pnas.172388999"><li><a href="http://dx.doi.org/10.1073/pnas.172388999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Progress+on+canalization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Progress+on+canalization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000911-Hahn1" id="pone.0000911-Hahn1"></a>Hahn MW, Kern AD (2005) Comparative Genomics of Centrality and Essentiality in Three Eukaryotic Protein-Interaction Networks. Mol Biol Evol  22: 803–806.  <ul class="find" data-citedArticleID="993836" data-doi="10.1093/molbev/msi072"><li><a href="http://dx.doi.org/10.1093/molbev/msi072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+Genomics+of+Centrality+and+Essentiality+in+Three+Eukaryotic+Protein-Interaction+Networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+Genomics+of+Centrality+and+Essentiality+in+Three+Eukaryotic+Protein-Interaction+Networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000911-Gasch1" id="pone.0000911-Gasch1"></a>Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, et al.  (2000) Genomic expression programs in the response of yeast cells to environmental changes. Mol Biol Cell  11: 4241–4257.  <ul class="find" data-citedArticleID="993826" data-doi="10.1091/mbc.11.12.4241"><li><a href="http://dx.doi.org/10.1091/mbc.11.12.4241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+expression+programs+in+the+response+of+yeast+cells+to+environmental+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+expression+programs+in+the+response+of+yeast+cells+to+environmental+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000911-Hughes1" id="pone.0000911-Hughes1"></a>Hughes TR, Marton MJ, Jones AR, Roberts CJ, Stoughton R, et al.  (2000) Functional discovery via a compendium of expression profiles. Cell  102: 109–126.  <ul class="find" data-citedArticleID="993844" data-doi="10.1016/s0092-8674(00)00015-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)00015-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+discovery+via+a+compendium+of+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+discovery+via+a+compendium+of+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000911-Giaever1" id="pone.0000911-Giaever1"></a>Giaever G, Chu AM, Ni L, Connelly C, Riles L, et al.  (2002) Functional profiling of the <em>Saccharomyces cerevisiae</em> genome. Nature  418: 387–391.  <ul class="find" data-citedArticleID="993832" data-doi="10.1038/nature00935"><li><a href="http://dx.doi.org/10.1038/nature00935" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+profiling+of+the+Saccharomyces+cerevisiae+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+profiling+of+the+Saccharomyces+cerevisiae+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000911-Newman1" id="pone.0000911-Newman1"></a>Newman JRS, Ghaemmaghami S, Ihmels J, Breslow DK, Noble M, et al.  (2006) Single-cell proteomic analysis of <em>S. cerevisiae</em> reveals the architecture of biological noise. Nature  441: 840–846.  <ul class="find" data-citedArticleID="993864" data-doi="10.1038/nature04785"><li><a href="http://dx.doi.org/10.1038/nature04785" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single-cell+proteomic+analysis+of+S.+cerevisiae+reveals+the+architecture+of+biological+noise." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single-cell+proteomic+analysis+of+S.+cerevisiae+reveals+the+architecture+of+biological+noise.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000911-Proulx2" id="pone.0000911-Proulx2"></a>Proulx SR, Promislow DEL, Phillips PC (2005) Network thinking in ecology and evolution. Trends in Ecology &amp; Evolution  20: 345–353.  <ul class="find" data-citedArticleID="993870" data-doi="10.1016/j.tree.2005.04.004"><li><a href="http://dx.doi.org/10.1016/j.tree.2005.04.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+thinking+in+ecology+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+thinking+in+ecology+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000911-Carlson1" id="pone.0000911-Carlson1"></a>Carlson JM, Doyle J (2002) Complexity and robustness. PNAS  99: 2538–2545.  <ul class="find" data-citedArticleID="993814" data-doi="10.1073/pnas.012582499"><li><a href="http://dx.doi.org/10.1073/pnas.012582499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complexity+and+robustness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complexity+and+robustness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000911-Gldener1" id="pone.0000911-Gldener1"></a>Güldener U, Münsterkötter M, Kastenmüller G, NS, J vH, et al.  (2005) CYGD: the Comprehensive Yeast Genome Database. Nucleic Acids Research  33: 364–368.  <ul class="find" data-citedArticleID="993834" data-doi="10.1093/nar/gki053"><li><a href="http://dx.doi.org/10.1093/nar/gki053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CYGD%3A+the+Comprehensive+Yeast+Genome+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CYGD%3A+the+Comprehensive+Yeast+Genome+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000911-Kerr1" id="pone.0000911-Kerr1"></a>Kerr MK, Martin M, Churchill GA (2000) Analysis of variance for gene expression microarray data. J Comput Biol  7: 819–837.  <ul class="find" data-citedArticleID="993854" data-doi="10.1089/10665270050514954"><li><a href="http://dx.doi.org/10.1089/10665270050514954" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+variance+for+gene+expression+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+variance+for+gene+expression+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000911-Kellis1" id="pone.0000911-Kellis1"></a>Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES (2003) Sequencing and comparison of yeast species to identify genes and regulatory elements. Nature  423: 241–254.  <ul class="find" data-citedArticleID="993852" data-doi="10.1038/nature01644"><li><a href="http://dx.doi.org/10.1038/nature01644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000911-Lee1" id="pone.0000911-Lee1"></a>Lee TI, Rinaldi NJ, Robert F, Odom DT, Bar-Joseph Z, et al.  (2002) Transcriptional regulatory networks in <em>Saccharomyces cerevisiae</em>. Science  298: 799–804.  <ul class="find" data-citedArticleID="993858" data-doi="10.1126/science.1075090"><li><a href="http://dx.doi.org/10.1126/science.1075090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+networks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+networks+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000911-Uetz1" id="pone.0000911-Uetz1"></a>Uetz P, Giot L, Cagney G, Mansfield TA, Judson RS, et al.  (2000) A comprehensive analysis of protein-protein interactions in <em>Saccharomyces cerevisiae</em>. Nature  403: 623–627.  <ul class="find" data-citedArticleID="993888" data-doi="10.1038/35001009"><li><a href="http://dx.doi.org/10.1038/35001009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+analysis+of+protein-protein+interactions+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+analysis+of+protein-protein+interactions+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000911-Breitkreutz1" id="pone.0000911-Breitkreutz1"></a>Breitkreutz B, Stark C, M.T (2003) The GRID: the General Repository for Interaction Datasets. Genome Biology  4: R23.  <ul class="find" data-citedArticleID="993810" data-doi="10.1186/gb-2003-4-3-r23"><li><a href="http://dx.doi.org/10.1186/gb-2003-4-3-r23" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+GRID%3A+the+General+Repository+for+Interaction+Datasets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+GRID%3A+the+General+Repository+for+Interaction+Datasets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000911-Krogan1" id="pone.0000911-Krogan1"></a>Krogan NJ, Peng WT, Cagney G, Robinson MD, Haw R, et al.  (2004) High-definition macromolecular composition of yeast RNA-processing complexes. Mol Cell  13: 225–239.  <ul class="find" data-citedArticleID="993856" data-doi="10.1016/s1097-2765(04)00003-6"><li><a href="http://dx.doi.org/10.1016/s1097-2765(04)00003-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-definition+macromolecular+composition+of+yeast+RNA-processing+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-definition+macromolecular+composition+of+yeast+RNA-processing+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000911-Hazbun1" id="pone.0000911-Hazbun1"></a>Hazbun TR, Malmstrom L, Anderson S, Graczyk BJ, Fox B, et al.  (2003) Assigning function to yeast proteins by integration of technologies. Mol Cell  12: 1353–1365.  <ul class="find" data-citedArticleID="993840" data-doi="10.1016/s1097-2765(03)00476-3"><li><a href="http://dx.doi.org/10.1016/s1097-2765(03)00476-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assigning+function+to+yeast+proteins+by+integration+of+technologies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assigning+function+to+yeast+proteins+by+integration+of+technologies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000911-Ho1" id="pone.0000911-Ho1"></a>Ho Y, Gruhler A, Heilbut A, Bader GD, Moore L, et al.  (2002) Systematic identification of protein complexes in <em>Saccharomyces cerevisiae</em> by mass spectrometry. Nature  415: 180–183.  <ul class="find" data-citedArticleID="993842" data-doi="10.1038/415180a"><li><a href="http://dx.doi.org/10.1038/415180a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+identification+of+protein+complexes+in+Saccharomyces+cerevisiae+by+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+identification+of+protein+complexes+in+Saccharomyces+cerevisiae+by+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000911-Gavin1" id="pone.0000911-Gavin1"></a>Gavin AC, Bosche M, Krause R, Grandi P, Marzioch M, et al.  (2002) Functional organization of the yeast proteome by systematic analysis of protein complexes. Nature  415: 141–147.  <ul class="find" data-citedArticleID="993828" data-doi="10.1038/415141a"><li><a href="http://dx.doi.org/10.1038/415141a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+organization+of+the+yeast+proteome+by+systematic+analysis+of+protein+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+organization+of+the+yeast+proteome+by+systematic+analysis+of+protein+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000911-Ito1" id="pone.0000911-Ito1"></a>Ito T, Chiba T, Ozawa R, Yoshida M, Hattori M, et al.  (2001) A comprehensive two-hybrid analysis to explore the yeast protein interactome. Proc Natl Acad Sci U S A  98: 4569–4574.  <ul class="find" data-citedArticleID="993846" data-doi="10.1073/pnas.061034498"><li><a href="http://dx.doi.org/10.1073/pnas.061034498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+two-hybrid+analysis+to+explore+the+yeast+protein+interactome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+two-hybrid+analysis+to+explore+the+yeast+protein+interactome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000911-Ito2" id="pone.0000911-Ito2"></a>Ito T, Tashiro K, Muta S, Ozawa R, Chiba T, et al.  (2000) Toward a protein-protein interaction map of the budding yeast: A comprehensive system to examine two-hybrid interactions in all possible combinations between the yeast proteins. Proc Natl Acad Sci U S A  97: 1143–1147.  <ul class="find" data-citedArticleID="993848" data-doi="10.1073/pnas.97.3.1143"><li><a href="http://dx.doi.org/10.1073/pnas.97.3.1143" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+a+protein-protein+interaction+map+of+the+budding+yeast%3A+A+comprehensive+system+to+examine+two-hybrid+interactions+in+all+possible+combinations+between+the+yeast+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+a+protein-protein+interaction+map+of+the+budding+yeast%3A+A+comprehensive+system+to+examine+two-hybrid+interactions+in+all+possible+combinations+between+the+yeast+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000911-FromontRacine1" id="pone.0000911-FromontRacine1"></a>Fromont-Racine M, Rain JC, Legrain P (1997) Toward a functional analysis of the yeast genome through exhaustive two-hybrid screens. Nat Genet  16: 277–282.  <ul class="find" data-citedArticleID="993824" data-doi="10.1038/ng0797-277"><li><a href="http://dx.doi.org/10.1038/ng0797-277" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+a+functional+analysis+of+the+yeast+genome+through+exhaustive+two-hybrid+screens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+a+functional+analysis+of+the+yeast+genome+through+exhaustive+two-hybrid+screens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000911-deNooy1" id="pone.0000911-deNooy1"></a>de Nooy W, Mrvar A, Batagelj V (2005) Exploratory social network analysis with Pajek. Cambridge: Cambridge University Press.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.XML" value="103437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.PDF" value="142429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g001.PNG_L" value="168076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g001.PNG_M" value="49337"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g001.PNG_S" value="6826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g001.TIF" value="375800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g001.PNG_I" value="16726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g002.PNG_L" value="14808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g002.PNG_M" value="24201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g002.PNG_S" value="5763"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g002.TIF" value="28644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g002.PNG_I" value="9000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g003.PNG_L" value="39847"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g003.PNG_M" value="55249"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g003.PNG_S" value="8084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g003.TIF" value="51114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.g003.PNG_I" value="19495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t001.PNG_L" value="106454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t001.PNG_M" value="147616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t001.PNG_S" value="16727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t001.TIF" value="396384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t001.PNG_I" value="51411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t002.PNG_L" value="64569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t002.PNG_M" value="84441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t002.PNG_S" value="12411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t002.TIF" value="263700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t002.PNG_I" value="30305"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t003.PNG_L" value="53891"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t003.PNG_M" value="33802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t003.PNG_S" value="8101"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t003.TIF" value="263822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t003.PNG_I" value="10373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t004.PNG_L" value="43955"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t004.PNG_M" value="57550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t004.PNG_S" value="10737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t004.TIF" value="194072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.t004.PNG_I" value="19897"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.e001.PNG" value="11396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.e001.TIF" value="27044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000911.s001.XLS" value="3664384"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000911&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000911" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000911&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000911&volume=&issue=&title=Direct Selection on Genetic Robustness Revealed in the Yeast Transcriptome&author_name=Stephen%20R.%20Proulx%2C%20Sergey%20Nuzhdin%2C%20Daniel%20E.%20L.%20Promislow&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000911" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000911" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000911" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000911&amp;t=Direct%20Selection%20on%20Genetic%20Robustness%20Revealed%20in%20the%20Yeast%20Transcriptome" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000911&title=Direct%20Selection%20on%20Genetic%20Robustness%20Revealed%20in%20the%20Yeast%20Transcriptome&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000911&amp;title=Direct%20Selection%20on%20Genetic%20Robustness%20Revealed%20in%20the%20Yeast%20Transcriptome" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000911" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000911&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Direct Selection on Genetic Robustness Revealed in the Yeast Transcriptome';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000911';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Direct%20Selection%20on%20Genetic%20Robustness%20Revealed%20in%20the%20Yeast%20Transcriptome http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000911" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000911" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="24660"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24660"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+networks%22" title="Search for articles in the subject area:'Genetic networks'"><div class="flagText">Genetic networks</div></a>
              <div data-categoryid="31807" data-articleid="24660"
                   data-categoryname="Genetic networks"
                   class="flagImage" title="Flag 'Genetic networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24660"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="24660"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="24660"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interaction+networks%22" title="Search for articles in the subject area:'Protein interaction networks'"><div class="flagText">Protein interaction networks</div></a>
              <div data-categoryid="18069" data-articleid="24660"
                   data-categoryname="Protein interaction networks"
                   class="flagImage" title="Flag 'Protein interaction networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="24660"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6854'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8659'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2632&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
