

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Novel Approach for Determining Cancer Genomic Breakpoints in the Presence of Normal DNA</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000380"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000380" />

    <meta name="citation_title" content="A Novel Approach for Determining Cancer Genomic Breakpoints in the Presence of Normal DNA"/>
    <meta itemprop="name" content="A Novel Approach for Determining Cancer Genomic Breakpoints in the Presence of Normal DNA"/>

      <meta name="citation_author" content="Yu-Tsueng Liu"/>
            <meta name="citation_author_institution" content="Moores UCSD Cancer Center, University of California San Diego, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Dennis A. Carson"/>
            <meta name="citation_author_institution" content="Moores UCSD Cancer Center, University of California San Diego, La Jolla, California, United States of America"/>

    <meta name="citation_date" content="2007/4/18"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000380.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e380"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The detection of large deletions or duplications in genomic DNA.; citation_author=JA Armour; citation_author=DE Barton; citation_author=DJ Cockburn; citation_author=GR Taylor; citation_journal_title=Hum Mutat; citation_volume=20; citation_number=1; citation_pages=325-337; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Philadelphia-positive leukemia: a personal perspective.; citation_author=N Heisterkamp; citation_author=J Groffen; citation_journal_title=Oncogene; citation_volume=21; citation_number=2; citation_pages=8536-8540; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Deletions of the cyclin-dependent kinase-4 inhibitor gene in multiple human cancers.; citation_author=T Nobori; citation_author=K Miura; citation_author=DJ Wu; citation_author=A Lois; citation_author=K Takabayashi; citation_journal_title=Nature; citation_volume=368; citation_number=3; citation_pages=753-756; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A cell cycle regulator potentially involved in genesis of many tumor types.; citation_author=A Kamb; citation_author=NA Gruis; citation_author=J Weaver-Feldhaus; citation_author=Q Liu; citation_author=K Harshman; citation_journal_title=Science; citation_volume=264; citation_number=4; citation_pages=436-440; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Frequency of homozygous deletion at p16/CDKN2 in primary human tumours.; citation_author=P Cairns; citation_author=TJ Polascik; citation_author=Y Eby; citation_author=K Tokino; citation_author=J Califano; citation_journal_title=Nat Genet; citation_volume=11; citation_number=5; citation_pages=210-212; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=p16(MTS-1/CDKN2/INK4a) in cancer progression.; citation_author=JW Rocco; citation_author=D Sidransky; citation_journal_title=Exp Cell Res; citation_volume=264; citation_number=6; citation_pages=42-55; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The p16INK4a/CDKN2A tumor suppressor and its relatives.; citation_author=M Ruas; citation_author=G Peters; citation_journal_title=Biochim Biophys Acta; citation_volume=1378; citation_number=7; citation_pages=F115-177; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=INK4a/ARF: a multifunctional tumor suppressor locus.; citation_author=NE Sharpless; citation_journal_title=Mutat Res; citation_volume=576; citation_number=8; citation_pages=22-38; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pancreatic cancer biology and genetics.; citation_author=N Bardeesy; citation_author=RA DePinho; citation_journal_title=Nat Rev Cancer; citation_volume=2; citation_number=9; citation_pages=897-909; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Focus on head and neck cancer.; citation_author=L Mao; citation_author=WK Hong; citation_author=VA Papadimitrakopoulou; citation_journal_title=Cancer Cell; citation_volume=5; citation_number=10; citation_pages=311-316; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Natural selection in neoplastic progression of Barrett's esophagus.; citation_author=CC Maley; citation_author=BJ Reid; citation_journal_title=Semin Cancer Biol; citation_volume=15; citation_number=11; citation_pages=474-483; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A new regulatory motif in cell-cycle control causing specific inhibition of cyclin D/CDK4.; citation_author=M Serrano; citation_author=GJ Hannon; citation_author=D Beach; citation_journal_title=Nature; citation_volume=366; citation_number=12; citation_pages=704-707; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The regulation of INK4/ARF in cancer and aging.; citation_author=WY Kim; citation_author=NE Sharpless; citation_journal_title=Cell; citation_volume=127; citation_number=13; citation_pages=265-275; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Alternative reading frames of the INK4a tumor suppressor gene encode two unrelated proteins capable of inducing cell cycle arrest.; citation_author=DE Quelle; citation_author=F Zindy; citation_author=RA Ashmun; citation_author=CJ Sherr; citation_journal_title=Cell; citation_volume=83; citation_number=14; citation_pages=993-1000; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Principles of tumor suppression.; citation_author=CJ Sherr; citation_journal_title=Cell; citation_volume=116; citation_number=15; citation_pages=235-246; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Senescent cells, tumor suppression, and organismal aging: good citizens, bad neighbors.; citation_author=J Campisi; citation_journal_title=Cell; citation_volume=120; citation_number=16; citation_pages=513-522; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulation of the INK4b-ARF-INK4a tumour suppressor locus: all for one or one for all.; citation_author=J Gil; citation_author=G Peters; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=7; citation_number=17; citation_pages=667-677; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cancer-susceptibility genes. Gatekeepers and caretakers.; citation_author=KW Kinzler; citation_author=B Vogelstein; citation_journal_title=Nature; citation_volume=386; citation_number=18; citation_pages=761, 763; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The genetic basis of human cancer.; citation_author=B Vogelstein; citation_author=KW Kinzler; citation_number=19; citation_date=2002; citation_publisher=McGraw-Hill Professional; " />
      <meta name="citation_reference" content="citation_title=NIH Institutes launch joint venture to map cancer genome.; citation_author=B Vastag; citation_journal_title=J Natl Cancer Inst; citation_volume=98; citation_number=20; citation_pages=162; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification and validation of oncogenes in liver cancer using an integrative oncogenomic approach.; citation_author=L Zender; citation_author=MS Spector; citation_author=W Xue; citation_author=P Flemming; citation_author=C Cordon-Cardo; citation_journal_title=Cell; citation_volume=125; citation_number=21; citation_pages=1253-1267; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Novel patterns of genome rearrangement and their association with survival in breast cancer.; citation_author=J Hicks; citation_author=A Krasnitz; citation_author=B Lakshmi; citation_author=NE Navin; citation_author=M Riggs; citation_journal_title=Genome Res; citation_volume=16; citation_number=22; citation_pages=1465-1479; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High-resolution characterization of the pancreatic adenocarcinoma genome.; citation_author=AJ Aguirre; citation_author=C Brennan; citation_author=G Bailey; citation_author=R Sinha; citation_author=B Feng; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=23; citation_pages=9067-9072; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An X chromosome gene, WTX, is commonly inactivated in Wilms tumor.; citation_author=MN Rivera; citation_author=WJ Kim; citation_author=J Wells; citation_author=DR Driscoll; citation_author=BW Brannigan; citation_journal_title=Science; citation_volume=315; citation_number=24; citation_pages=642-645; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=BRCA1 genomic deletions are major founder mutations in Dutch breast cancer patients.; citation_author=A Petrij-Bosch; citation_author=T Peelen; citation_author=M van Vliet; citation_author=R van Eijk; citation_author=R Olmer; citation_journal_title=Nat Genet; citation_volume=17; citation_number=25; citation_pages=341-345; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Detection of p16 gene deletions in gliomas: a comparison of fluorescence in situ hybridization (FISH) versus quantitative PCR.; citation_author=A Perry; citation_author=T Nobori; citation_author=N Ru; citation_author=K Anderl; citation_author=TJ Borell; citation_journal_title=J Neuropathol Exp Neurol; citation_volume=56; citation_number=26; citation_pages=999-1008; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Detection of hemizygous deletions in genomic DNA from leukaemia specimens for the diagnosis of patients.; citation_author=UR Kees; citation_author=PA Terry; citation_author=J Ford; citation_author=J Everett; citation_author=A Murch; citation_journal_title=Leuk Res; citation_volume=29; citation_number=27; citation_pages=165-171; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Hemizygous p16(INK4A) deletion in pediatric acute lymphoblastic leukemia predicts independent risk of relapse.; citation_author=TL Carter; citation_author=PM Watt; citation_author=R Kumar; citation_author=PR Burton; citation_author=GH Reaman; citation_journal_title=Blood; citation_volume=97; citation_number=28; citation_pages=572-574; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Detection of methylthioadenosine phosphorylase (MTAP) and p16 gene deletion in T cell acute lymphoblastic leukemia by real-time quantitative PCR assay.; citation_author=TJ M'Soka; citation_author=J Nishioka; citation_author=A Taga; citation_author=K Kato; citation_author=H Kawasaki; citation_journal_title=Leukemia; citation_volume=14; citation_number=29; citation_pages=935-940; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Frequent deletion in the methylthioadenosine phosphorylase gene in T-cell acute lymphoblastic leukemia: strategies for enzyme-targeted therapy.; citation_author=A Batova; citation_author=MB Diccianni; citation_author=T Nobori; citation_author=T Vu; citation_author=J Yu; citation_journal_title=Blood; citation_volume=88; citation_number=30; citation_pages=3083-3090; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Array comparative genomic hybridization and its applications in cancer.; citation_author=D Pinkel; citation_author=DG Albertson; citation_journal_title=Nat Genet; citation_number=31; citation_pages=S11-17; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=End-sequence profiling: sequence-based analysis of aberrant genomes.; citation_author=S Volik; citation_author=S Zhao; citation_author=K Chin; citation_author=JH Brebner; citation_author=DR Herndon; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=32; citation_pages=7696-7701; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genomic cloning of methylthioadenosine phosphorylase: a purine metabolic enzyme deficient in multiple different cancers.; citation_author=T Nobori; citation_author=K Takabayashi; citation_author=P Tran; citation_author=L Orvis; citation_author=A Batova; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=33; citation_pages=6203-6208; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Construction of a 2.8-megabase yeast artificial chromosome contig and cloning of the human methylthioadenosine phosphorylase gene from the tumor suppressor region on 9p21.; citation_author=OI Olopade; citation_author=HM Pomykala; citation_author=F Hagos; citation_author=LW Sveen; citation_author=R Espinosa; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=34; citation_pages=6489-6493; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Array-based comparative genomic hybridization analysis reveals recurrent chromosomal alterations and prognostic parameters in primary cutaneous large B-cell lymphoma.; citation_author=R Dijkman; citation_author=CP Tensen; citation_author=ES Jordanova; citation_author=J Knijnenburg; citation_author=JJ Hoefnagel; citation_journal_title=J Clin Oncol; citation_volume=24; citation_number=35; citation_pages=296-305; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Reverse genetics by chemical mutagenesis in Caenorhabditis elegans.; citation_author=G Jansen; citation_author=E Hazendonk; citation_author=KL Thijssen; citation_author=RH Plasterk; citation_journal_title=Nat Genet; citation_volume=17; citation_number=36; citation_pages=119-121; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Molecular processes of chromosome 9p21 deletions in human cancers.; citation_author=S Sasaki; citation_author=Y Kitagawa; citation_author=Y Sekido; citation_author=JD Minna; citation_author=H Kuwano; citation_journal_title=Oncogene; citation_volume=22; citation_number=37; citation_pages=3792-3798; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Homozygous deletions of CDKN2A caused by alternative mechanisms in various human cancer cell lines.; citation_author=S Raschke; citation_author=V Balz; citation_author=T Efferth; citation_author=WA Schulz; citation_author=AR Florl; citation_journal_title=Genes Chromosomes Cancer; citation_volume=42; citation_number=38; citation_pages=58-67; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Frequent somatic mutations and homozygous deletions of the p16 (MTS1) gene in pancreatic adenocarcinoma.; citation_author=C Caldas; citation_author=SA Hahn; citation_author=LT da Costa; citation_author=MS Redston; citation_author=M Schutte; citation_journal_title=Nat Genet; citation_volume=8; citation_number=39; citation_pages=27-32; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Highly parallel genomic assays.; citation_author=JB Fan; citation_author=MS Chee; citation_author=KL Gunderson; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=40; citation_pages=632-644; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Accurate multiplex polony sequencing of an evolved bacterial genome.; citation_author=J Shendure; citation_author=GJ Porreca; citation_author=NB Reppas; citation_author=X Lin; citation_author=JP McCutcheon; citation_journal_title=Science; citation_volume=309; citation_number=41; citation_pages=1728-1732; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors.; citation_author=M Margulies; citation_author=M Egholm; citation_author=WE Altman; citation_author=S Attiya; citation_author=JS Bader; citation_journal_title=Nature; citation_volume=437; citation_number=42; citation_pages=376-380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene sequencing. The race for the $1000 genome.; citation_author=RF Service; citation_journal_title=Science; citation_volume=311; citation_number=43; citation_pages=1544-1546; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A genotyping system capable of simultaneously analyzing>1000 single nucleotide polymorphisms in a haploid genome.; citation_author=HY Wang; citation_author=M Luo; citation_author=IV Tereshchenko; citation_author=DM Frikker; citation_author=X Cui; citation_journal_title=Genome Res; citation_volume=15; citation_number=44; citation_pages=276-283; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Quantification of plasma Epstein-Barr virus DNA in patients with advanced nasopharyngeal carcinoma.; citation_author=JC Lin; citation_author=WY Wang; citation_author=KY Chen; citation_author=YH Wei; citation_author=WM Liang; citation_journal_title=N Engl J Med; citation_volume=350; citation_number=45; citation_pages=2461-2470; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rapid clearance of plasma Epstein-Barr virus DNA after surgical treatment of nasopharyngeal carcinoma.; citation_author=EW To; citation_author=KC Chan; citation_author=SF Leung; citation_author=LY Chan; citation_author=KF To; citation_journal_title=Clin Cancer Res; citation_volume=9; citation_number=46; citation_pages=3254-3259; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Disruption of the multiple tumor suppressor gene MTS1/p16(INK4a)/CDKN2 by illegitimate V(D)J recombinase activity in T-cell acute lymphoblastic leukemias.; citation_author=JM Cayuela; citation_author=B Gardie; citation_author=F Sigaux; citation_journal_title=Blood; citation_volume=90; citation_number=47; citation_pages=3720-3726; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Prevalent involvement of illegitimate V(D)J recombination in chromosome 9p21 deletions in lymphoid leukemia.; citation_author=Y Kitagawa; citation_author=K Inoue; citation_author=S Sasaki; citation_author=Y Hayashi; citation_author=Y Matsuo; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=48; citation_pages=46289-46297; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detection of minimal residual disease in hematologic malignancies by real-time quantitative PCR: principles, approaches, and laboratory aspects.; citation_author=VH van der Velden; citation_author=A Hochhaus; citation_author=G Cazzaniga; citation_author=T Szczepanski; citation_author=J Gabert; citation_journal_title=Leukemia; citation_volume=17; citation_number=49; citation_pages=1013-1034; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Minimal residual disease directed therapy for childhood acute myeloid leukaemia: the time is now.; citation_author=N Goulden; citation_author=P Virgo; citation_author=D Grimwade; citation_journal_title=Br J Haematol; citation_volume=134; citation_number=50; citation_pages=273-282; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Monitoring CML patients responding to treatment with tyrosine kinase inhibitors: review and recommendations for harmonizing current methodology for detecting BCR-ABL transcripts and kinase domain mutations and for expressing results.; citation_author=T Hughes; citation_author=M Deininger; citation_author=A Hochhaus; citation_author=S Branford; citation_author=J Radich; citation_journal_title=Blood; citation_volume=108; citation_number=51; citation_pages=28-37; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Why and how to quantify minimal residual disease in acute lymphoblastic leukemia?; citation_author=T Szczepanski; citation_journal_title=Leukemia.; citation_number=52; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Monitoring of minimal residual disease in leukemia, advantages and pitfalls.; citation_author=G Cazzaniga; citation_author=G Gaipa; citation_author=V Rossi; citation_author=A Biondi; citation_journal_title=Ann Med; citation_volume=38; citation_number=53; citation_pages=512-521; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Myc translocations in B cell and plasma cell neoplasms.; citation_author=S Janz; citation_journal_title=DNA Repair (Amst); citation_volume=5; citation_number=54; citation_pages=1213-1224; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Variable breakpoints in Burkitt lymphoma cells with chromosomal t(8;14) translocation separate c-myc and the IgH locus up to several hundred kb.; citation_author=S Joos; citation_author=MH Falk; citation_author=P Lichter; citation_author=FG Haluska; citation_author=B Henglein; citation_journal_title=Hum Mol Genet; citation_volume=1; citation_number=55; citation_pages=625-632; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Mapping chromosomal breakpoints of Burkitt's t(8;14) translocations far upstream of c-myc.; citation_author=S Joos; citation_author=FG Haluska; citation_author=MH Falk; citation_author=B Henglein; citation_author=H Hameister; citation_journal_title=Cancer Res; citation_volume=52; citation_number=56; citation_pages=6547-6552; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=DNA arrays for analysis of gene expression.; citation_author=MB Eisen; citation_author=PO Brown; citation_journal_title=Methods Enzymol; citation_volume=303; citation_number=57; citation_pages=179-205; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Viral discovery and sequence recovery using DNA microarrays.; citation_author=D Wang; citation_author=A Urisman; citation_author=YT Liu; citation_author=M Springer; citation_author=TG Ksiazek; citation_journal_title=PLoS Biol; citation_volume=1; citation_number=58; citation_pages=E2; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Ensembl 2005.; citation_author=T Hubbard; citation_author=D Andrews; citation_author=M Caccamo; citation_author=G Cameron; citation_author=Y Chen; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=59; citation_pages=D447-453; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000380" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Novel Approach for Determining Cancer Genomic Breakpoints in the Presence of Normal DNA"/>
    <meta name="twitter:description" content="CDKN2A (encodes p16INK4A and p14ARF) deletion, which results in both Rb and p53 inactivation, is the most common chromosomal anomaly in human cancers. To precisely map the deletion breakpoints is important to understanding the molecular mechanism of genomic rearrangement and may also be useful for clinical applications. However, current methods for determining the breakpoint are either of low resolution or require the isolation of relatively pure cancer cells, which can be difficult for clinical samples that are typically contaminated with various amounts of normal host cells. To overcome this hurdle, we have developed a novel approach, designated Primer Approximation Multiplex PCR (PAMP), for enriching breakpoint sequences followed by genomic tiling array hybridization to locate the breakpoints. In a series of proof-of-concept experiments, we were able to identify cancer-derived CDKN2A genomic breakpoints when more than 99.9% of wild type genome was present in a model system. This design can be scaled up with bioinformatics support and can be applied to validate other candidate cancer-associated loci that are revealed by other more systemic but lower throughput assays."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000380.g006"/>

  <meta property="og:title" content="A Novel Approach for Determining Cancer Genomic Breakpoints in the Presence of Normal DNA" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000380" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5756'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6535'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4624&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000380">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000380" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000380&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Novel Approach for Determining Cancer Genomic Breakpoints in the Presence of Normal DNA
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yu-Tsueng Liu
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ytliu@ucsd.edu">ytliu@ucsd.edu</a></p>

                <p>Affiliation:
                  Moores UCSD Cancer Center, University of California San Diego, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dennis A. Carson
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Moores UCSD Cancer Center, University of California San Diego, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 18, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000380</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000380" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000380" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000380" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000380" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000380-g001" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000380-g002" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000380-g003" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000380-g004" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000380-g005" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000380-t001" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000380-g006" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000380-t002" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000380">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000380" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p><em>CDKN2A</em> (encodes p16<sup>INK4A</sup> and p14<sup>ARF</sup>) deletion, which results in both Rb and p53 inactivation, is the most common chromosomal anomaly in human cancers. To precisely map the deletion breakpoints is important to understanding the molecular mechanism of genomic rearrangement and may also be useful for clinical applications. However, current methods for determining the breakpoint are either of low resolution or require the isolation of relatively pure cancer cells, which can be difficult for clinical samples that are typically contaminated with various amounts of normal host cells. To overcome this hurdle, we have developed a novel approach, designated Primer Approximation Multiplex PCR (PAMP), for enriching breakpoint sequences followed by genomic tiling array hybridization to locate the breakpoints. In a series of proof-of-concept experiments, we were able to identify cancer-derived <em>CDKN2A</em> genomic breakpoints when more than 99.9% of wild type genome was present in a model system. This design can be scaled up with bioinformatics support and can be applied to validate other candidate cancer-associated loci that are revealed by other more systemic but lower throughput assays.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Liu Y-T, Carson DA (2007) A Novel Approach for Determining Cancer Genomic Breakpoints in the Presence of Normal DNA. PLoS ONE 2(4):
          e380.
            doi:10.1371/journal.pone.0000380</p><p><strong>Academic Editor: </strong>David Levens, National Cancer Institute, United States of America</p><p><strong>Received:</strong> February 20, 2007; <strong>Accepted:</strong> March 27, 2007; <strong>Published:</strong> April 18, 2007</p><p><strong>Copyright:</strong> © 2007 Liu, Carson. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work is supported in part by grants for the UCSD NanoTumor Center of Excellence for Cancer Nanotechnology (CA119335), CA23100 (D.A.C.) and AI36214-12S1 (Y-T L.) from the National Institutes of Health.</p><p><strong>Competing interests:</strong> The authors (Y.-T.L.&amp;D.A.C.) have submitted a provisional patent application based on this study.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Tumors evolve through the continuous accumulation and selection of randomly mutated genes. While sets of advantageous mutations are selected in tumors, neutral or even slightly detrimental mutations may also occur due to genomic instability and genetic drift. Recently, much effort has been expended to identify in primary human cancers point mutations in the exons of cancer-related genes. However, systemic mapping of genomic DNA rearrangements has lagged behind, due to technical difficulties in detecting smaller deletions, tumor heterogeneity, and the necessity to purify malignant from normal cells <a href="#pone.0000380-Armour1">[1]</a>. Historically, such work was done by time consuming and labor intensive genetics and molecular cloning on established cancer cell lines <a href="#pone.0000380-Heisterkamp1">[2]</a>, <a href="#pone.0000380-Nobori1">[3]</a>, <a href="#pone.0000380-Kamb1">[4]</a>. One of the most striking examples is the homozygous deletion of the <em>CDKN2A</em> (<em>INK4A/ARF</em>) tumor suppressor locus, which was discovered in this and other laboratories <a href="#pone.0000380-Nobori1">[3]</a>, <a href="#pone.0000380-Kamb1">[4]</a>, <a href="#pone.0000380-Cairns1">[5]</a>, <a href="#pone.0000380-Rocco1">[6]</a>, <a href="#pone.0000380-Ruas1">[7]</a>, <a href="#pone.0000380-Sharpless1">[8]</a>. The <em>CDKN2A</em> deletions occur early during tumor development <a href="#pone.0000380-Bardeesy1">[9]</a>, <a href="#pone.0000380-Mao1">[10]</a>, <a href="#pone.0000380-Maley1">[11]</a>. The p16<sup>INK4a</sup> (one of the <em>CDKN2A</em> products <a href="#pone.0000380-Serrano1">[12]</a>) protein constrains cell cycle progression by the Rb pathway and may be responsible for the decline in the replicative potential of stem cells during aging <a href="#pone.0000380-Kim1">[13]</a>. The p14<sup>ARF</sup> (the other alternative reading frame of <em>CDKN2A</em> <a href="#pone.0000380-Quelle1">[14]</a>) gene product regulates the expression of MDM2, the turnover of p53, and thereby controls the cellular response to stress (reviewed in <a href="#pone.0000380-Rocco1">[6]</a>, <a href="#pone.0000380-Ruas1">[7]</a>, <a href="#pone.0000380-Sharpless1">[8]</a>, <a href="#pone.0000380-Sherr1">[15]</a>, <a href="#pone.0000380-Campisi1">[16]</a>, <a href="#pone.0000380-Gil1">[17]</a>). Because the Rb and p53 pathways are central to cancer gate-keeping and caretaking <a href="#pone.0000380-Kinzler1">[18]</a>, <a href="#pone.0000380-Vogelstein1">[19]</a>, strong selection pressures exist for the disruption of the entire <em>CDKN2A</em> gene segment on both chromosomes. Few other deletions are as well characterized, although it is expected that more will be found when more data from array based comparative genomic hybridization (array-CGH) are reported and also through The Cancer Genome Atlas (TCGA) project <a href="#pone.0000380-Vastag1">[20]</a>, <a href="#pone.0000380-Zender1">[21]</a>, <a href="#pone.0000380-Hicks1">[22]</a>, <a href="#pone.0000380-Aguirre1">[23]</a>, <a href="#pone.0000380-Rivera1">[24]</a>. It will be important to validate the relevance of those genomic rearrangements to cancer development since many of the genomic structural changes may be simply due to genome instability in cancer. Large scale studies with clinical samples will be the most reliable confirmation.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>While point mutations and very small insertions or deletions in genomic DNA can be detected by exon re-sequencing, it can be more difficult to detect gene dosage changes of larger genomic fragments, especially deletions <a href="#pone.0000380-Armour1">[1]</a>. Current established techniques for deletion mapping, including Southern blotting <a href="#pone.0000380-PetrijBosch1">[25]</a>, fluorescent <em>in situ</em> hybridization (FISH) <a href="#pone.0000380-Perry1">[26]</a>, quantitative PCR <a href="#pone.0000380-Perry1">[26]</a>, <a href="#pone.0000380-Kees1">[27]</a>, <a href="#pone.0000380-Carter1">[28]</a>, <a href="#pone.0000380-MSoka1">[29]</a>, <a href="#pone.0000380-Batova1">[30]</a>, and array-CGH <a href="#pone.0000380-Pinkel1">[31]</a> rely on the absence of a detectable wild type signal <a href="#pone.0000380-Armour1">[1]</a>. This is problematic when a significant number of normal cells are present in a tumor sample. Array-CGH has the potential to analyze alterations of DNA copy number on a genome-wide scale with relatively high resolution, depending on whether BACs, PCR products or oligonucleotides are used for the array elements. However, these techniques often fail where there is a heterogeneous cell population or samples of poor quality <a href="#pone.0000380-Pinkel1">[31]</a>. FISH is less vulnerable to the presence of heterogeneous cell populations, but has relatively low resolution and is difficult to scale up. Except for FISH, the other techniques mentioned are not practical for mapping genomic translocations and inversions. End-sequencing profiling was developed to address this issue but the approach was costly and hard to scale up <a href="#pone.0000380-Volik1">[32]</a>. Therefore, there is a need to develop a scalable approach for detecting such genomic structural changes in solid tumors where heterogeneous cell populations are present.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here we report a novel approach, designated as Primer Approximation Multiplex PCR (PAMP), to enrich small amounts of deleted genomic DNA sequences in the presence of wild type DNA. The genomic locations of the enriched sequences are subsequently decoded by a genomic tiling array and confirmed by sequencing.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4><em>CDKN2A</em> locus</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The <em>CDKN2A</em> is located on chromosome 9p21 (<a href="#pone-0000380-g001">Figure 1</a>). It encodes two proteins in different reading frames: p16<sup>INK4A</sup> and p14<sup>ARF</sup>, which both have 3 exons and share exons 2 and 3. <em>CDKN2B</em> (p15<sup>INK4B</sup>) and <em>MTAP</em> (methylthioadenosine phosphorylase) (not shown) are centromeric and telomeric neighboring genes respectively <a href="#pone.0000380-Nobori1">[3]</a>, <a href="#pone.0000380-Gil1">[17]</a>, <a href="#pone.0000380-Nobori2">[33]</a>, <a href="#pone.0000380-Olopade1">[34]</a>. BAC clone RP11-149I2 contains the whole <em>CDKN2A</em> genomic fragment and was used as template to generate probes (excluding repetitive regions) for printing on the minigenomic tiling array. The frequency of repetitive sequences predicted by RepeatMasker is shown at the bottom of the diagram.</p>
<div class="figure" id="pone-0000380-g001"><div class="img"><a name="pone-0000380-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The <em>CDKN2A</em> locus.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The genomic map covers about 55 kb around <em>CDKN2A</em> according to Ensemble <a href="#pone.0000380-Hubbard1">[59]</a>. <em>CDKN2A</em>/B is located at chromosome 9p21 and their RNA products are encoded by the reverse strand. <em>CDKN2A</em> encodes 2 proteins (p16<sup>INK4A</sup> and p14<sup>ARF</sup>) that share the same exons 2 and 3. The first exons of <em>INK4A</em> and <em>ARF</em> are about 20 kb apart. <em>CDKN2B</em> encodes p15<sup>INK4B</sup> that is homologous to p16<sup>INK4A</sup>. In addition to transcripts, the map also shows repetitive sequences (Repeat) and available BAC clone (Human tilepath clones), RP11-149I2.</p>
<span>doi:10.1371/journal.pone.0000380.g001</span></div>

<h4>Primer Approximation Multiplex PCR (PAMP)</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>It is difficult to detect a small fraction of deleted mutant genomic DNA in the presence of a vast excess of wild type DNA with array CGH or other popular molecular biology tools <a href="#pone.0000380-Perry1">[26]</a>, <a href="#pone.0000380-Kees1">[27]</a>, <a href="#pone.0000380-Dijkman1">[35]</a>. In typically contaminated tumor samples, genomic DNA is composed of various ratios of WT and <em>CDKN2A</em> deficient DNA. We aimed to take advantage of the fact that a shorter deleted genome sequence should be preferentially amplified compared to a much longer WT sequence using “approximated” flanking primers (<a href="#pone-0000380-g002">Figure 2A</a>) <a href="#pone.0000380-Jansen1">[36]</a>.</p>
<div class="figure" id="pone-0000380-g002"><div class="img"><a name="pone-0000380-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Primer Approximation Multiplex PCR (PAMP).</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The efficiency of PCR amplification is inversely related to the distance of upstream and downstream primers. In this example, primers to amplify genomic sequences around the locus of interest (LOI) are divided into 20 groups: 10 each for forward (F1–F10) and reverse (R1-R10) groups (A). While all of the possible forward and reverse primer pairs are too far to each other for PCR amplification in wild type genome, certain pair of primers is brought closer (“approximated”) due to deletion (F3 and R3) in mutated genome. Multiplex PCR reactions are set and represented as a matrix to include one forward and one reverse primer group. The expected PCR results are shown as gray scale shadows in the matrix (B). This example shows that only group pairs close to breakpoint give PCR products (F3-R3, F3-R4, F4-R3).</p>
<span>doi:10.1371/journal.pone.0000380.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The approach is illustrated in <a href="#pone-0000380-g002">Figure 2</a>. In this example, relatively even-spaced primers (average 1 kb apart) surrounding the locus of interest are divided into 20 groups for PCR. There are 10 groups each of forward primers, F1, F2 …, F10 and reverse primer R1, R2, …,R10, respectively (<a href="#pone-0000380-g002">Figure 2A</a>). Therefore, there are 100 pairs (F1-R1, F1-R2, …, F1-R10; F2-R1, F2-R2,…, F2-R10; …; F10-R1, F10-R2,…., F10-R10) of PCR reactions (<a href="#pone-0000380-g002">Figure 2B</a>). It is expected that only one or two pairs of PCR reactions will produce specific PCR products spanning the deletion boundary, since the other primer pairs should be too far from the breakpoint for efficient amplification. Then aliquots from each reaction can be mixed to hybridize on a single genomic tiling array. Unlike traditional array-CGH, it is expected that only spots representing genomic sequences near the breakpoints will be theoretically lit up, which was confirmed in the following experiments.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>In order to increase the throughput and reduce the cost of reagents, every forward (F1-10) and reverse (R1-10) primer group can have multiple primers. Therefore, every PCR group (for example F1-R1) pair becomes multiplex PCR. Therefore, we designated this procedure as Primer Approximation Multiplex PCR (PAMP).</p>


<h4>Deletion breakpoint cloning by PAMP and minigenomic tiling array</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We reported previously that the Detroit 562 cell line has an approximate 20 kb (including <em>INK4A</em> exons 1 and 2) deletion on chromosome 9p21 <a href="#pone.0000380-Nobori1">[3]</a>. We used this cell line to test our deletion scanning approach. Four groups (F<sub>A</sub>, F<sub>B</sub>, R<sub>Y</sub> and R<sub>Z</sub>) of primers were used for four PAMP reactions (<a href="#pone-0000380-g003">Figure 3A</a>) using genomic DNA template either from Detroit 562 (<em>CDKN2A</em> deficient) or HEK293 (<em>CDKN2A</em> wild type) cell lines. Aliquots of all 4 PAMP reaction products were pooled and labeled for hybridization on an <em>INK4A</em> minigenomic tiling array that covers about 25 kb, including all of the exons of <em>INK4A</em>. As predicted in <a href="#pone-0000380-g002">Figure 2</a>, only spots with probes close to the breakpoints hybridized to the amplicons when Detroit 562 genomic DNA was used as a template (<a href="#pone-0000380-g003">Figure 3B</a>). Almost no signal was detected when HEK293 genomic DNA was used as a template. The control HEK293 sample had a significantly higher signal on Cot-1 DNA spots despite its general absolute signal intensity is low.</p>
<div class="figure" id="pone-0000380-g003"><div class="img"><a name="pone-0000380-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Breakpoint identification by PAMP with an <em>INK4A</em> minigenomic tiling array.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Five groups of primers (F<sub>A</sub>, F<sub>B</sub>, R<sub>x</sub>, R<sub>Y</sub> and R<sub>Z</sub>, the small arrows and arrow heads) near the potential breakpoints were generated for PAMP based on our previous mapping <a href="#pone.0000380-Nobori1">[3]</a>. The mapped <em>CDKN2A</em> breakpoints of the Detroit 562 cell line (<a href="#pone-0000380-g005">Figure 5</a>) are indicated for clarification. The “E1”, “E2” and “E3” designations (blue fonts) are the relative positions of <em>INK4A</em> exons. The first exon of <em>ARF</em> is further to the right of this diagram and is not covered by this array. The tiling probes for the array are indicated with two alternating colors (short black and orange lines) for ease of identification. (B) The first row of the <em>INK4A</em> minigenomic array was spotted with the tiling probes shown in panel A. Cot-1 DNA (repetitive sequence of genomic DNA) spots are indicated on this array. The rest of the spots are herring sperm DNA. Both Cot-1 and herring sperm DNA are used as nonspecific controls. This array was hybridized with labeled samples derived from two cell lines. The same sets of primers (F<sub>A</sub>, F<sub>B</sub>, R<sub>Y</sub> and R<sub>Z</sub>) were used for PAMP reactions on Detroit 562 (mutant) and HEK293 (wild type) genomic DNA to map the potential <em>CDKN2A</em> breakpoints. The amplicons were labeled with different dyes, yielding a green signal (Cy-3) for the mutant sample and a red signal (Cy-5) for the wild type sample, to be simultaneously hybridized on the array (two-color array). The two green spots on the first row revealed the breakpoint location as been discussed in <a href="#pone-0000380-g002">Figure 2</a>.</p>
<span>doi:10.1371/journal.pone.0000380.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In addition, four separate arrays were used to hybridize the individual PAMP products described above. A simple plot of signal intensity ratio of mutant/WT PCR products on the tiling array revealed the genomic location of the breakpoint (<a href="#pone-0000380-g004">Figure 4</a>). This analysis shows a very straightforward readout—the location of the deletion is bordered by two peaks. Only F<sub>B</sub>-R<sub>Y</sub> (array 27) and all products pooling (array 29) produce the same result as shown in <a href="#pone-0000380-g003">Figure 3</a>. In contrast, the other three pairs yielded only faint background signals on the arrays. This result indicates that PAMP product pooling with a single array analysis gives the same breakpoint information as four individual arrays. The data support the original experimental predictions, and suggest that the procedure should be generally applicable for deletion and translocation scanning.</p>
<div class="figure" id="pone-0000380-g004"><div class="img"><a name="pone-0000380-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Pooling of individual PAMP reactions for single array hybridization.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Four groups (F<sub>A</sub>, F<sub>B</sub>, R<sub>Y</sub> and R<sub>Z</sub>) of primers were used for four PAMP reactions each by pairing all of the possible forward and reverse primer groups using Detroit 562 (mutant) and HEK293 (control) as templates. The procedure has been briefly described in <a href="#pone-0000380-g003">Figure 3</a>. The products were labeled and used for array hybridization: F<sub>A</sub>-R<sub>Y</sub> for array 25; F<sub>A</sub>-R<sub>Z</sub> for array 26; F<sub>B</sub>-R<sub>Y</sub> for array 27 and F<sub>A</sub>-R<sub>Z</sub> for array 28. Aliquots of the individual PAMP samples were also pooled together and labeled for array hybridization (array 29, its array image is shown in <a href="#pone-0000380-g003">Figure 3B</a>). The results are presented with ratio signal intensity (Y-axis) of samples from Detroit 562 (mutant) and HEK293 (control) against the probe location (X-axis). The breakpoints can be identified through this plot by finding the two peaks that are analogous to the bright green spots in <a href="#pone-0000380-g003">Figure 3B</a>.</p>
<span>doi:10.1371/journal.pone.0000380.g004</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In order to pinpoint more precisely the area of deletion, nested PCR with pairs of specific primers was designed according to the earlier PAMP results. The PCR product was labeled for array hybridization, yielding a result very similar to that shown in <a href="#pone-0000380-g004">Figure 4</a> and is shown in <a href="#pone-0000380-g005">Figure 5A</a>. Furthermore, the single major product of the PCR reactions was resolved by agarose gel electrophoresis, excised, extracted and sequenced (<a href="#pone-0000380-g005">Figure 5B</a>). The breakpoint cloned is in agreement with two other reports (<a href="#pone-0000380-g005">Figure 5B</a>) <a href="#pone.0000380-Sasaki1">[37]</a>, <a href="#pone.0000380-Raschke1">[38]</a>.</p>
<div class="figure" id="pone-0000380-g005"><div class="img"><a name="pone-0000380-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Breakpoint mapping of Detroit 562 cell line.</span></strong></p><a id="article1.body1.sec2.sec3.fig3.caption1.p1" name="article1.body1.sec2.sec3.fig3.caption1.p1"></a><p>To map the exact breakpoint, a nested set of PCR primers were designed for uniplex PCR based on the previous PAMP results (<a href="#pone-0000380-g003">Figures 3</a> and <a href="#pone-0000380-g004">4</a>). The PCR products were used for labeling and hybridized on the array and also for agarose gel electrophoresis. The array data is shown as the same plot in <a href="#pone-0000380-g004">Figure 4</a>. A single major band on the agarose gel was excised and purified for sequencing (B). The breakpoint is indicated (from #21975226 to #21960809 according to NCBI human genome sequence build 36).</p>
<span>doi:10.1371/journal.pone.0000380.g005</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>To mimic the heterogeneous population of cancer and host cells typically found in solid tumors, various amounts of genomic DNA derived from Detroit 562 (mutant) and HEK293 (wild type) were mixed for PAMP and array hybridization. In order to test the sensitivity of our approach, we performed a titration experiment. The total genomic DNA for each assay was kept constant (100 ng). This is equivalent to about 28,000 copies of haploid genome (based on the estimate of 2.8×10<sup>5</sup> molecules/µg of haploid genome). The <em>CDKN2A</em> deleted cell line Detroit 562 was serially diluted with <em>CDKN2A</em> wild type HEK293 as shown in the <a href="#pone-0000380-t001">Table 1</a>. The assay was able to detect approximately 1 breakpoint sequence in the presence of an approximately 2000 fold excess of wild-type genome with sensitivity of 5–16 such molecules (<a href="#pone-0000380-t001">Table 1</a>). Thus, the PAMP approach provides a method for detecting genomic DNA deletions in the presence of more than 99.9% wild type DNA.</p>
<div class="figure" id="pone-0000380-t001"><div class="img"><a name="pone-0000380-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000380.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000380.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>The sensitivity of PAMP assay</span></strong></p><span>doi:10.1371/journal.pone.0000380.t001</span></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>The PAMP breakpoint cloning strategy was confirmed in another cell line. Because our array only covers 25 kb of the genome, we scanned our previous mapping information on 100 cell lines to find one that might have breakpoints within this region <a href="#pone.0000380-Nobori1">[3]</a>, <a href="#pone.0000380-Nobori2">[33]</a>. The Hs578T breast cancer cell line has a deletion in p16<sup>INK4A</sup> exons 1-3. With primers within and telomeric (F<sub>A</sub> and R<sub>X</sub> groups, <a href="#pone-0000380-t002">Table 2</a>) to the genomic fragment, we performed PAMP and array hybridization, and identified a single spot on the array (shown as a single peak in <a href="#pone-0000380-g006">Figure 6A</a>). The sequence of this probe is located from 69971 to 71219 in the RP11-149I2 BAC sequence (GenBank accession AL449423). Therefore, the centromeric end of the breakpoint should be located near this region. Uniplex PCR with primers from the two groups for PAMP was performed for sequencing. A PCR product about 2 kb was generated with a pair of primers and was subjected to direct sequencing (<a href="#pone-0000380-g006">Figure 6B</a>). The centromeric end of the breakpoint identified by PAMP is consistent with a previous report <a href="#pone.0000380-Sasaki1">[37]</a>. The telomeric end of the breakpoint was inferred from the primers used for PAMP and also confirmed by direct sequencing, although there was no genomic probe near the breakpoint that was included on the array.</p>
<div class="figure" id="pone-0000380-g006"><div class="img"><a name="pone-0000380-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000380.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Breakpoint mapping of Hs578T cell line.</span></strong></p><a id="article1.body1.sec2.sec3.fig4.caption1.p1" name="article1.body1.sec2.sec3.fig4.caption1.p1"></a><p>Two groups of primers: F<sub>A</sub> (F<sub>A1</sub>-F<sub>A4</sub>) and R<sub>X</sub> (R<sub>X1</sub>-R<sub>X5</sub>) were used for PAMP based on our previous mapping. The product was labeled for array hybridization (A). Only single peak is evident from the plot. It indicates the location of the other breakpoint is not covered by this minigenomic array. Two primers (R<sub>X3</sub> and R<sub>X4</sub>) located near the genomic location of the probe (human chromosome 9, 21969229 to 21970477, NCBI build 36) that was hybridized and two primers (F<sub>A1</sub> and F<sub>A2</sub>) located outside the array coverage were chosen for uniplex PCR. The F<sub>A2</sub>-R<sub>X3</sub> pair is expected to have the shortest distance when a deletion occurs. A band of about 2 kb on agarose gel was excised from the gel, purified and sequenced (B). The breakpoint and location is indicated (from #21955827 to #21968338 according to NCBI human genome sequence build 36).</p>
<span>doi:10.1371/journal.pone.0000380.g006</span></div><div class="figure" id="pone-0000380-t002"><div class="img"><a name="pone-0000380-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000380" data-uri="info:doi/10.1371/journal.pone.0000380.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000380.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000380.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000380.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000380.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000380.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Primers for PAMP</span></strong></p><span>doi:10.1371/journal.pone.0000380.t002</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have developed a general strategy that can be applied for pinpointing the genomic breakpoints in unpurified primary cancers. The amplification and tiling protocol described here allows for simple and precise <em>CDKN2A</em> breakpoint cloning, using contaminated DNA as a template. In contrast to the current available techniques for deletion mapping (including Southern blotting, fluorescent <em>in situ</em> hybridization, real time PCR, and array CGH) that rely on the absence of a detectable wild type signal, PAMP directly measures the deleted DNA. Therefore, this approach is much less vulnerable to problems associated with normal cell contamination. The experimental procedure is robust enough to detect deletions in the presence of at least 99.9% wild type sequence contamination, which could not be achieved by other procedures <a href="#pone.0000380-Nobori1">[3]</a>, <a href="#pone.0000380-Cairns1">[5]</a>, <a href="#pone.0000380-Perry1">[26]</a>, <a href="#pone.0000380-Kees1">[27]</a>, <a href="#pone.0000380-Carter1">[28]</a>, <a href="#pone.0000380-Nobori2">[33]</a>, <a href="#pone.0000380-Dijkman1">[35]</a>, <a href="#pone.0000380-Caldas1">[39]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Primer approximation PCR screening has been a useful tool for isolating deletion mutants in <em>C. elegans</em> <a href="#pone.0000380-Jansen1">[36]</a>. The method relies on identifying a single band that is the product of a successful PCR reaction when a pair of specific primers is brought together by deletion, on an agarose gel. The procedure can only identify deletions that happen in a very small genomic fragment (3 kb) in a relatively low throughput fashion. It also suffers from relatively high false positive rate because the identity of the bands on the agarose gel is difficult to know. However, by applying multiplex PCR together with a genomic tiling array, one can simultaneously screen a wider range of genomic regions <a href="#pone.0000380-Fan1">[40]</a>. In addition, preferential amplification of the sequences near the breakpoints generates a relatively straightforward readout on the tiling array. The signal to noise ratio on the hybridized spots is obvious compared to the readout from array CGH (see <a href="#pone-0000380-g004">Figure 4</a>). The junction can be readily identified as long as one end of the nearby genomic location of the breakpoints is covered by the tiling array, as shown in the case of Hs578T breast cancer cell (<a href="#pone-0000380-g006">Figure 6</a>). Since high-density genomic tiling arrays are commercially available, this approach can be easily adopted. In addition, high-throughput genome sequencing technology may also pinpoint the exact breakpoint sequence after PAMP, bypassing the need for array hybridization <a href="#pone.0000380-Shendure1">[41]</a>, <a href="#pone.0000380-Margulies1">[42]</a>, <a href="#pone.0000380-Service1">[43]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We used multiplex PCR to reduce the workload and cost for PAMP. We were able to multiplex 28 primers easily in a single PCR reaction. Theoretically, one can cover over 90% of the 0.5 Mb of genomic fragment around <em>CDKN2A</em> locus with a total of 500 primers in one single PCR reaction through computational simulation, which will be described elsewhere (manuscript submitted). A recent paper reported a successful multiplex PCR with more than 1000 primer pairs through the aid of computational design <a href="#pone.0000380-Wang1">[44]</a>. The PAMP approach targets deletion sizes between 10 kb and 1 Mb. The smaller or larger deletions can be detected by resequencing and FISH respectively.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Like other PCR technologies, PAMP can be easily adopted to a robotic system for clinical and research purposes. One example of a potential clinical application is to use the unique breakpoint sequence as a personalized cancer-specific biomarker for disease monitoring after treatment, when the precise breakpoint has been mapped. For example, unlike many current tumor markers, such as CA19-9, CA125 and PSA, which are not truly cancer-specific, the <em>CDKN2A</em> breakpoints are specific and unique for each cancer with this locus deleted. A highly sensitive assay, such as real-time PCR, can be designed to monitor the status of cancer progression in the blood or other body fluids. The assay should be very specific because amplification is expected to occur only from deletion-containing DNA due to very long distance between the primers in the wild type genome (see <a href="#pone-0000380-g002">Figure 2</a>). This is analogous to the detection of a foreign virus sequence, which has been applied as a useful biomarker for Epstein-Barr virus associated nasopharyngeal carcinoma <a href="#pone.0000380-Lin1">[45]</a>, <a href="#pone.0000380-To1">[46]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Our approach can also ease traditional labor-intensive experiments that aim to understand how genomic breakpoints are generated during cancer development, particularly in primary tumors. Although illegitimate V(D)J recombination may be responsible for creating <em>CDKN2A</em> deletions in acute lymphoblastic leukemia, more breakpoint sequence data will be needed for other types of cancers to delineate the molecular mechanisms <a href="#pone.0000380-Sasaki1">[37]</a>, <a href="#pone.0000380-Raschke1">[38]</a>, <a href="#pone.0000380-Cayuela1">[47]</a>, <a href="#pone.0000380-Kitagawa1">[48]</a>. Furthermore, the technique described in this paper can be used not only for deletion mapping, but it can also be applied to map other types of genomic rearrangement, such as translocations and inversions. Similar to the case of genomic deletion (see <a href="#pone-0000380-g002">Figure 2</a>), only “approximated” primers can generate amplicons when those primers are near the genomic fragments that are repositioned in translocations and inversions.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Using breakpoint sequences as cancer-specific biomarkers to monitor minimal residual diseases has been explored <a href="#pone.0000380-vanderVelden1">[49]</a>, <a href="#pone.0000380-Goulden1">[50]</a>,<a href="#pone.0000380-Hughes1">[51]</a>,<a href="#pone.0000380-Szczepanski1">[52]</a>,<a href="#pone.0000380-Cazzaniga1">[53]</a>. Disease monitoring based on personalized genomic DNA breakpoint is considered to be highly attractive approach for several reasons<a href="#pone.0000380-vanderVelden1">[49]</a>. First, many genomic DNA rearrangements are directly related to oncogenic process, therefore, are truly cancer-specific and stable over time. This is in contrast to more convenient Ig/TCR rearrangement based assay. Indeed, we found exactly the same <em>CDKN2A</em> breakpoints of the two cell lines used in this study as reported by others. Second, the DNA is more stable than RNA although it is easier to map fusion transcript if it exists, such as <em>BCR-ABL</em>. Third, the genomic breakpoints are very likely to be different from each patient and become personalized biomarkers, thereby, reducing the risk of false positive results due to cross contamination. However, this is also the biggest hurdle to overcome. For example, many efforts to improve the PCR amplification range for detecting <em>C-MYC</em>/immunoglobulin translocations have had limited success because the breakpoints are scattered across a more than 300 kb region <a href="#pone.0000380-Janz1">[54]</a>, <a href="#pone.0000380-Joos1">[55]</a>, <a href="#pone.0000380-Joos2">[56]</a>. Our strategy may be useful for such application.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Our approach aims to identify breakpoints within a 1 Mb genomic fragment as FISH or other cytogenetic techniques are available for larger genomic rearrangements and the cost and labor significantly increase when the target region expands. We are able to inexpensively produce a tiling array covering a genomic fragment of 0.5 Mb around the <em>CDKN2A</em> with 1 kb resolution. We are currently working on methods to increase multiplexing and reduce the volume of each PAMP reaction for broader applications.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cell culture and sample preparation</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The cell lines described in the paper were obtained from the American Type Culture Collection (ATCC, Manassas, VA) and cultured as recommended. The genomic DNA was extracted with DNAzol (Molecular Research Center, Inc., Cincinnati, OH) following the instructions from the manufacturer.</p>


<h4>Minigenomic tiling array</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We created an <em>INK4A</em> minigenomic tiling array covering a 25 kb fragment in the <em>CDKN2A</em> locus for proving the concept of our approach. DNA probes were generated by PCR with BAC clone RP11-149I2 (obtained from BACPAC Resources Center at Children's Hospital Oakland Research Institute, Oakland, CA) as template and avoiding the repetitive genomic sequences that were predicted by RepeatMasker. The PCR products were purified with DNA Clean-up and Concentrator-5 (Zymo Research, Orange, CA), resuspended in 3×SSC and printed on poly-L-lysine slides at 0.1 mg/ml along with Human Cot-1 DNA (Invitrogen, Carlsbad, CA), which is enriched for repetitive sequences, and herring sperm DNA (Promega, Madison, WI), which was used as nonspecific control. The printing procedure has been described and essentially followed the manual of the DeRisi arrayer with silicon microcontact printing pins (Parallel Synthesis Technologies, Inc. Santa Clara, CA) <a href="#pone.0000380-Eisen1">[57]</a>, <a href="#pone.0000380-Wang2">[58]</a>. Arrays were post-processed with succinic anhydride-based method for blocking before hybridization as previously described <a href="#pone.0000380-Eisen1">[57]</a>. The protocols related to array printing and hybridization in this paper generally can be found in microarrays.org (<a href="http://derisilab.ucsf.edu/microarray/protocols.html">http://derisilab.ucsf.edu/microarray/pro​tocols.html</a>).</p>


<h4>Primer-Approximation Multiplex PCR (PAMP) and array hybridization</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A simplified PAMP scheme is shown in <a href="#pone-0000380-g002">Figure 2</a>. A series of primers (<a href="#pone-0000380-t002">Table 2</a>) toward <em>INK4A</em> exons 1-2 along the <em>CDKN2A</em> locus were synthesized by Integrated DNA Technologies (Coralville, IA). Groups of forward and reverse primers (250 nM each in the final reaction) were used to generate amplicons from 0.1 µg of genomic DNA templates in a total of 10 µl of solution mixing with 10 µl of Taq 2×Master Mix (New England Biolabs, Ipswich, MA). The reaction was assembled at 4°C in a PCR workstation and transferred to a thermocycler with the block preheated to 94°C. The cycling conditions were a 3-minute denaturation step at 94°C followed by 35 cycles at 92°C for 30 sec, 55°C for 30 sec and 68°C for 2.5 minutes with a final extension step at 68°C for 5 minutes. One µl of unpurified product was subsequently used as templates for another round of amplification to label the amplicons with the same PCR protocol except that dTTP was replaced by a 4:1 mixture of aminoallyl dUTP (Ambion, Austin, TX) and dTTP for probe labeling. The labeled amplicons were purified with DNA Clean-up and Concentrator-5 columns, eluted in 9 µl of sodium bicarbonate (pH 9.0) and coupled with 1 µl of DMSO dissolved Cy3 or Cy5 NHS esters (GE Healthcare, Piscataway, NJ) for 30 to 60 minutes. The Cy3 and Cy5 labeled amplicons were purified with DNA Clean-up and Concentrator-5 columns and eluted with 10 µl of 10 mM Tris-HCl (pH 8.0). Paired Cy3 and Cy5 labeled amplicons were combined with 3.6 µl of 20×SSC, 0.5 µl of Hepes (pH 7.0) and finally 0.5 µl of 10% SDS. The mixed solution was heated for 2 minutes at 95°C, cooled to room temperature and hybridized to the minigenomic tiling arrays at 63°C overnight essentially as previously described <a href="#pone.0000380-Eisen1">[57]</a>, <a href="#pone.0000380-Wang2">[58]</a>. The hybridized arrays were washed and scanned with GenePix 4000B scanner (Molecular Device, Sunnyvale, CA) and analyzed by GenePix Pro 6.0 software.</p>


<h4>Tiling array data analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The human Cot-1 DNA and herring sperm DNA were designed to be positive and negative controls respectively and spotted multiple times on the array (see <a href="#pone-0000380-g003">Figure 3</a>). To normalize for day-to-day and sample-to-sample variation, the median intensity of all features representing herring sperm DNA (<strong><em>I<sub>50%-HS</sub></em></strong>) were used to divide the intensity (<strong><em>I<sub>G</sub></em></strong>) of each feature representing genomic probes. Each (<strong><em>I<sub>G</sub></em></strong>):(<strong><em>I<sub>50%-HS</sub></em></strong>) ratio, the normalized genomic probe signal, was plotted at the Y-axis against the corresponding probe's genomic location at the X-axis to ease data interpretation (see <a href="#pone-0000380-g004">Figure 4</a>).</p>


<h4><em>CDKN2A</em> breakpoint cloning</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>To confirm <em>CDKN2A</em> breakpoint mapping by PAMP approach, the genomic fragments flanking the breakpoints were cloned by traditional PCR approaches guided by the results from PAMP. Two examples are given in this paper.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p><em>Detroit 562 cell line</em>: The nested approach was used to clone the <em>CDKN2A</em> breakpoint in Detroit 562 human epithelial carcinoma cells. The primers were designed with clues from the PAMP experiment (see <a href="#pone-0000380-g003">Figures 3</a> and <a href="#pone-0000380-g004">4</a>). External primers (<span class="gene">AGGTTTGGTTAAGAGTCGTTC</span> and <span class="gene">AAGATCTATATGGTGGCCTTTAG</span>) were used for 35 cycles of PCR (92°C, 30 seconds; 55°C, 30 seconds; 68°C, 2 minutes). The two sets of internal primers (<span class="gene">ATGCTAGCTGTAACTGGAGC</span> and CTTAAGGCTAAATTGACTTG; <span class="gene">GGCTTAGAGCTAACTCTTCACCC</span> and <span class="gene">TATGTGTGTGTGTGTCTGTGTGATG</span>) were used for the second PCR reaction under the same conditions. A single band about 1 kb in size was excised and extracted with Qiaquick gel extraction kit (Qiagen, Valencia, CA). The purified product was directly sequenced with internal primers.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p><em>Hs578T cell line</em>: Four sets of uniplex PCR reactions were performed by pairing 2 single forward (F<sub>A1</sub> and F<sub>A2</sub>) and 2 single reverse (R<sub>X3</sub> and R<sub>X4</sub>) primers. The PCR program was the same as that for PAMP. The products were analyzed by agarose gel electrophoresis. A single band from the shortest distance pair (F<sub>A2</sub>-R<sub>X3</sub>) was excised and sequenced with an internal primer.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Yandong Zhao and Qing Lu for technical assistance in array hybridization, Vineet Bafna and Ali Bashir for discussions and April Livengood and Richard Schwab for critical reading of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DC YL. Performed the experiments: YL. Analyzed the data: DC YL. Contributed reagents/materials/analysis tools: DC YL. Wrote the paper: DC YL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000380-Armour1" id="pone.0000380-Armour1"></a>Armour JA, Barton DE, Cockburn DJ, Taylor GR (2002) The detection of large deletions or duplications in genomic DNA. Hum Mutat  20: 325–337.  <ul class="find" data-citedArticleID="945304" data-doi="10.1002/humu.10133"><li><a href="http://dx.doi.org/10.1002/humu.10133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+detection+of+large+deletions+or+duplications+in+genomic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+detection+of+large+deletions+or+duplications+in+genomic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000380-Heisterkamp1" id="pone.0000380-Heisterkamp1"></a>Heisterkamp N, Groffen J (2002) Philadelphia-positive leukemia: a personal perspective. Oncogene  21: 8536–8540.  <ul class="find" data-citedArticleID="945332" data-doi="10.1038/sj.onc.1206080"><li><a href="http://dx.doi.org/10.1038/sj.onc.1206080" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Philadelphia-positive+leukemia%3A+a+personal+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Philadelphia-positive+leukemia%3A+a+personal+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000380-Nobori1" id="pone.0000380-Nobori1"></a>Nobori T, Miura K, Wu DJ, Lois A, Takabayashi K, et al.  (1994) Deletions of the cyclin-dependent kinase-4 inhibitor gene in multiple human cancers. Nature  368: 753–756.  <ul class="find" data-citedArticleID="945368" data-doi="10.1038/368753a0"><li><a href="http://dx.doi.org/10.1038/368753a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deletions+of+the+cyclin-dependent+kinase-4+inhibitor+gene+in+multiple+human+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deletions+of+the+cyclin-dependent+kinase-4+inhibitor+gene+in+multiple+human+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000380-Kamb1" id="pone.0000380-Kamb1"></a>Kamb A, Gruis NA, Weaver-Feldhaus J, Liu Q, Harshman K, et al.  (1994) A cell cycle regulator potentially involved in genesis of many tumor types. Science  264: 436–440.  <ul class="find" data-citedArticleID="945348" data-doi="10.1126/science.8153634"><li><a href="http://dx.doi.org/10.1126/science.8153634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cell+cycle+regulator+potentially+involved+in+genesis+of+many+tumor+types." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cell+cycle+regulator+potentially+involved+in+genesis+of+many+tumor+types.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000380-Cairns1" id="pone.0000380-Cairns1"></a>Cairns P, Polascik TJ, Eby Y, Tokino K, Califano J, et al.  (1995) Frequency of homozygous deletion at p16/CDKN2 in primary human tumours. Nat Genet  11: 210–212.  <ul class="find" data-citedArticleID="945310" data-doi="10.1038/ng1095-210"><li><a href="http://dx.doi.org/10.1038/ng1095-210" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frequency+of+homozygous+deletion+at+p16%2FCDKN2+in+primary+human+tumours." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frequency+of+homozygous+deletion+at+p16%2FCDKN2+in+primary+human+tumours.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000380-Rocco1" id="pone.0000380-Rocco1"></a>Rocco JW, Sidransky D (2001) p16(MTS-1/CDKN2/INK4a) in cancer progression. Exp Cell Res  264: 42–55.  <ul class="find" data-citedArticleID="945386" data-doi="10.1006/excr.2000.5149"><li><a href="http://dx.doi.org/10.1006/excr.2000.5149" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=p16%28MTS-1%2FCDKN2%2FINK4a%29+in+cancer+progression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22p16%28MTS-1%2FCDKN2%2FINK4a%29+in+cancer+progression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000380-Ruas1" id="pone.0000380-Ruas1"></a>Ruas M, Peters G (1998) The p16INK4a/CDKN2A tumor suppressor and its relatives. Biochim Biophys Acta  1378: F115–177.  <ul class="find" data-citedArticleID="945388" data-doi="10.1016/s0304-419x(98)00017-1"><li><a href="http://dx.doi.org/10.1016/s0304-419x(98)00017-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+p16INK4a%2FCDKN2A+tumor+suppressor+and+its+relatives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+p16INK4a%2FCDKN2A+tumor+suppressor+and+its+relatives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000380-Sharpless1" id="pone.0000380-Sharpless1"></a>Sharpless NE (2005) INK4a/ARF: a multifunctional tumor suppressor locus. Mutat Res  576: 22–38.  <ul class="find" data-citedArticleID="945396" data-doi="10.1016/j.mrfmmm.2004.08.021"><li><a href="http://dx.doi.org/10.1016/j.mrfmmm.2004.08.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=INK4a%2FARF%3A+a+multifunctional+tumor+suppressor+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22INK4a%2FARF%3A+a+multifunctional+tumor+suppressor+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000380-Bardeesy1" id="pone.0000380-Bardeesy1"></a>Bardeesy N, DePinho RA (2002) Pancreatic cancer biology and genetics. Nat Rev Cancer  2: 897–909.  <ul class="find" data-citedArticleID="945306" data-doi="10.1038/nrc949"><li><a href="http://dx.doi.org/10.1038/nrc949" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pancreatic+cancer+biology+and+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pancreatic+cancer+biology+and+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000380-Mao1" id="pone.0000380-Mao1"></a>Mao L, Hong WK, Papadimitrakopoulou VA (2004) Focus on head and neck cancer. Cancer Cell  5: 311–316.  <ul class="find" data-citedArticleID="945364" data-doi="10.1016/s1535-6108(04)00090-x"><li><a href="http://dx.doi.org/10.1016/s1535-6108(04)00090-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Focus+on+head+and+neck+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Focus+on+head+and+neck+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000380-Maley1" id="pone.0000380-Maley1"></a>Maley CC, Reid BJ (2005) Natural selection in neoplastic progression of Barrett's esophagus. Semin Cancer Biol  15: 474–483.  <ul class="find" data-citedArticleID="945362" data-doi="10.1016/j.semcancer.2005.06.004"><li><a href="http://dx.doi.org/10.1016/j.semcancer.2005.06.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+in+neoplastic+progression+of+Barrett%27s+esophagus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+in+neoplastic+progression+of+Barrett%27s+esophagus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000380-Serrano1" id="pone.0000380-Serrano1"></a>Serrano M, Hannon GJ, Beach D (1993) A new regulatory motif in cell-cycle control causing specific inhibition of cyclin D/CDK4. Nature  366: 704–707.  <ul class="find" data-citedArticleID="945392" data-doi="10.1038/366704a0"><li><a href="http://dx.doi.org/10.1038/366704a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+regulatory+motif+in+cell-cycle+control+causing+specific+inhibition+of+cyclin+D%2FCDK4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+regulatory+motif+in+cell-cycle+control+causing+specific+inhibition+of+cyclin+D%2FCDK4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000380-Kim1" id="pone.0000380-Kim1"></a>Kim WY, Sharpless NE (2006) The regulation of INK4/ARF in cancer and aging. Cell  127: 265–275.  <ul class="find" data-citedArticleID="945352" data-doi="10.1016/j.cell.2006.10.003"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.10.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+regulation+of+INK4%2FARF+in+cancer+and+aging." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+regulation+of+INK4%2FARF+in+cancer+and+aging.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000380-Quelle1" id="pone.0000380-Quelle1"></a>Quelle DE, Zindy F, Ashmun RA, Sherr CJ (1995) Alternative reading frames of the INK4a tumor suppressor gene encode two unrelated proteins capable of inducing cell cycle arrest. Cell  83: 993–1000.  <ul class="find" data-citedArticleID="945380" data-doi="10.1016/0092-8674(95)90214-7"><li><a href="http://dx.doi.org/10.1016/0092-8674(95)90214-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+reading+frames+of+the+INK4a+tumor+suppressor+gene+encode+two+unrelated+proteins+capable+of+inducing+cell+cycle+arrest." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+reading+frames+of+the+INK4a+tumor+suppressor+gene+encode+two+unrelated+proteins+capable+of+inducing+cell+cycle+arrest.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000380-Sherr1" id="pone.0000380-Sherr1"></a>Sherr CJ (2004) Principles of tumor suppression. Cell  116: 235–246.  <ul class="find" data-citedArticleID="945400" data-doi="10.1016/s0092-8674(03)01075-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)01075-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Principles+of+tumor+suppression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Principles+of+tumor+suppression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000380-Campisi1" id="pone.0000380-Campisi1"></a>Campisi J (2005) Senescent cells, tumor suppression, and organismal aging: good citizens, bad neighbors. Cell  120: 513–522.  <ul class="find" data-citedArticleID="945314" data-doi="10.1016/j.cell.2005.02.003"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.02.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Senescent+cells%2C+tumor+suppression%2C+and+organismal+aging%3A+good+citizens%2C+bad+neighbors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Senescent+cells%2C+tumor+suppression%2C+and+organismal+aging%3A+good+citizens%2C+bad+neighbors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000380-Gil1" id="pone.0000380-Gil1"></a>Gil J, Peters G (2006) Regulation of the INK4b-ARF-INK4a tumour suppressor locus: all for one or one for all. Nat Rev Mol Cell Biol  7: 667–677.  <ul class="find" data-citedArticleID="945328" data-doi="10.1038/nrm1987"><li><a href="http://dx.doi.org/10.1038/nrm1987" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+the+INK4b-ARF-INK4a+tumour+suppressor+locus%3A+all+for+one+or+one+for+all." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+the+INK4b-ARF-INK4a+tumour+suppressor+locus%3A+all+for+one+or+one+for+all.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000380-Kinzler1" id="pone.0000380-Kinzler1"></a>Kinzler KW, Vogelstein B (1997) Cancer-susceptibility genes. Gatekeepers and caretakers. Nature  386: 761, 763.  <ul class="find" data-citedArticleID="945354" data-doi="10.1038/386761a0"><li><a href="http://dx.doi.org/10.1038/386761a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cancer-susceptibility+genes.+Gatekeepers+and+caretakers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cancer-susceptibility+genes.+Gatekeepers+and+caretakers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000380-Vogelstein1" id="pone.0000380-Vogelstein1"></a>Vogelstein B, Kinzler KW (2002) The genetic basis of human cancer. New York: McGraw-Hill Professional.   <ul class="find-nolinks"></ul></li><li><span class="label">20.
              </span><a name="pone.0000380-Vastag1" id="pone.0000380-Vastag1"></a>Vastag B (2006) NIH Institutes launch joint venture to map cancer genome. J Natl Cancer Inst  98: 162.  <ul class="find" data-citedArticleID="945406" data-doi="10.1093/jnci/djj062"><li><a href="http://dx.doi.org/10.1093/jnci/djj062" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NIH+Institutes+launch+joint+venture+to+map+cancer+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NIH+Institutes+launch+joint+venture+to+map+cancer+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000380-Zender1" id="pone.0000380-Zender1"></a>Zender L, Spector MS, Xue W, Flemming P, Cordon-Cardo C, et al.  (2006) Identification and validation of oncogenes in liver cancer using an integrative oncogenomic approach. Cell  125: 1253–1267.  <ul class="find" data-citedArticleID="945416" data-doi="10.1016/j.cell.2006.05.030"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.05.030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+validation+of+oncogenes+in+liver+cancer+using+an+integrative+oncogenomic+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+validation+of+oncogenes+in+liver+cancer+using+an+integrative+oncogenomic+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000380-Hicks1" id="pone.0000380-Hicks1"></a>Hicks J, Krasnitz A, Lakshmi B, Navin NE, Riggs M, et al.  (2006) Novel patterns of genome rearrangement and their association with survival in breast cancer. Genome Res  16: 1465–1479.  <ul class="find" data-citedArticleID="945334" data-doi="10.1101/gr.5460106"><li><a href="http://dx.doi.org/10.1101/gr.5460106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+patterns+of+genome+rearrangement+and+their+association+with+survival+in+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+patterns+of+genome+rearrangement+and+their+association+with+survival+in+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000380-Aguirre1" id="pone.0000380-Aguirre1"></a>Aguirre AJ, Brennan C, Bailey G, Sinha R, Feng B, et al.  (2004) High-resolution characterization of the pancreatic adenocarcinoma genome. Proc Natl Acad Sci U S A  101: 9067–9072.  <ul class="find" data-citedArticleID="945302" data-doi="10.1073/pnas.0402932101"><li><a href="http://dx.doi.org/10.1073/pnas.0402932101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-resolution+characterization+of+the+pancreatic+adenocarcinoma+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-resolution+characterization+of+the+pancreatic+adenocarcinoma+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000380-Rivera1" id="pone.0000380-Rivera1"></a>Rivera MN, Kim WJ, Wells J, Driscoll DR, Brannigan BW, et al.  (2007) An X chromosome gene, WTX, is commonly inactivated in Wilms tumor. Science  315: 642–645.  <ul class="find" data-citedArticleID="945384" data-doi="10.1126/science.1137509"><li><a href="http://dx.doi.org/10.1126/science.1137509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+X+chromosome+gene%2C+WTX%2C+is+commonly+inactivated+in+Wilms+tumor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+X+chromosome+gene%2C+WTX%2C+is+commonly+inactivated+in+Wilms+tumor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000380-PetrijBosch1" id="pone.0000380-PetrijBosch1"></a>Petrij-Bosch A, Peelen T, van Vliet M, van Eijk R, Olmer R, et al.  (1997) BRCA1 genomic deletions are major founder mutations in Dutch breast cancer patients. Nat Genet  17: 341–345.  <ul class="find" data-citedArticleID="945376" data-doi="10.1038/ng1197-341"><li><a href="http://dx.doi.org/10.1038/ng1197-341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BRCA1+genomic+deletions+are+major+founder+mutations+in+Dutch+breast+cancer+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BRCA1+genomic+deletions+are+major+founder+mutations+in+Dutch+breast+cancer+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000380-Perry1" id="pone.0000380-Perry1"></a>Perry A, Nobori T, Ru N, Anderl K, Borell TJ, et al.  (1997) Detection of p16 gene deletions in gliomas: a comparison of fluorescence in situ hybridization (FISH) versus quantitative PCR. J Neuropathol Exp Neurol  56: 999–1008.  <ul class="find" data-citedArticleID="945374" data-doi="10.1097/00005072-199709000-00005"><li><a href="http://dx.doi.org/10.1097/00005072-199709000-00005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+p16+gene+deletions+in+gliomas%3A+a+comparison+of+fluorescence+in+situ+hybridization+%28FISH%29+versus+quantitative+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+p16+gene+deletions+in+gliomas%3A+a+comparison+of+fluorescence+in+situ+hybridization+%28FISH%29+versus+quantitative+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000380-Kees1" id="pone.0000380-Kees1"></a>Kees UR, Terry PA, Ford J, Everett J, Murch A, et al.  (2005) Detection of hemizygous deletions in genomic DNA from leukaemia specimens for the diagnosis of patients. Leuk Res  29: 165–171.  <ul class="find" data-citedArticleID="945350" data-doi="10.1016/j.leukres.2004.05.021"><li><a href="http://dx.doi.org/10.1016/j.leukres.2004.05.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+hemizygous+deletions+in+genomic+DNA+from+leukaemia+specimens+for+the+diagnosis+of+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+hemizygous+deletions+in+genomic+DNA+from+leukaemia+specimens+for+the+diagnosis+of+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000380-Carter1" id="pone.0000380-Carter1"></a>Carter TL, Watt PM, Kumar R, Burton PR, Reaman GH, et al.  (2001) Hemizygous p16(INK4A) deletion in pediatric acute lymphoblastic leukemia predicts independent risk of relapse. Blood  97: 572–574.  <ul class="find" data-citedArticleID="945316" data-doi="10.1182/blood.v97.2.572"><li><a href="http://dx.doi.org/10.1182/blood.v97.2.572" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hemizygous+p16%28INK4A%29+deletion+in+pediatric+acute+lymphoblastic+leukemia+predicts+independent+risk+of+relapse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hemizygous+p16%28INK4A%29+deletion+in+pediatric+acute+lymphoblastic+leukemia+predicts+independent+risk+of+relapse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000380-MSoka1" id="pone.0000380-MSoka1"></a>M'Soka TJ, Nishioka J, Taga A, Kato K, Kawasaki H, et al.  (2000) Detection of methylthioadenosine phosphorylase (MTAP) and p16 gene deletion in T cell acute lymphoblastic leukemia by real-time quantitative PCR assay. Leukemia  14: 935–940.  <ul class="find" data-citedArticleID="945360" data-doi="10.1038/sj.leu.2401771"><li><a href="http://dx.doi.org/10.1038/sj.leu.2401771" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+methylthioadenosine+phosphorylase+%28MTAP%29+and+p16+gene+deletion+in+T+cell+acute+lymphoblastic+leukemia+by+real-time+quantitative+PCR+assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+methylthioadenosine+phosphorylase+%28MTAP%29+and+p16+gene+deletion+in+T+cell+acute+lymphoblastic+leukemia+by+real-time+quantitative+PCR+assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000380-Batova1" id="pone.0000380-Batova1"></a>Batova A, Diccianni MB, Nobori T, Vu T, Yu J, et al.  (1996) Frequent deletion in the methylthioadenosine phosphorylase gene in T-cell acute lymphoblastic leukemia: strategies for enzyme-targeted therapy. Blood  88: 3083–3090.  <ul class="find" data-citedArticleID="945308"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Frequent+deletion+in+the+methylthioadenosine+phosphorylase+gene+in+T-cell+acute+lymphoblastic+leukemia%3A+strategies+for+enzyme-targeted+therapy.&amp;auth=&amp;atitle=Frequent+deletion+in+the+methylthioadenosine+phosphorylase+gene+in+T-cell+acute+lymphoblastic+leukemia%3A+strategies+for+enzyme-targeted+therapy." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frequent+deletion+in+the+methylthioadenosine+phosphorylase+gene+in+T-cell+acute+lymphoblastic+leukemia%3A+strategies+for+enzyme-targeted+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frequent+deletion+in+the+methylthioadenosine+phosphorylase+gene+in+T-cell+acute+lymphoblastic+leukemia%3A+strategies+for+enzyme-targeted+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000380-Pinkel1" id="pone.0000380-Pinkel1"></a>Pinkel D, Albertson DG (2005) Array comparative genomic hybridization and its applications in cancer. Nat Genet 37 SupplS11–17.  <ul class="find" data-citedArticleID="945378" data-doi="10.1038/ng1569"><li><a href="http://dx.doi.org/10.1038/ng1569" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Array+comparative+genomic+hybridization+and+its+applications+in+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Array+comparative+genomic+hybridization+and+its+applications+in+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000380-Volik1" id="pone.0000380-Volik1"></a>Volik S, Zhao S, Chin K, Brebner JH, Herndon DR, et al.  (2003) End-sequence profiling: sequence-based analysis of aberrant genomes. Proc Natl Acad Sci U S A  100: 7696–7701.  <ul class="find" data-citedArticleID="945410" data-doi="10.1073/pnas.1232418100"><li><a href="http://dx.doi.org/10.1073/pnas.1232418100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=End-sequence+profiling%3A+sequence-based+analysis+of+aberrant+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22End-sequence+profiling%3A+sequence-based+analysis+of+aberrant+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000380-Nobori2" id="pone.0000380-Nobori2"></a>Nobori T, Takabayashi K, Tran P, Orvis L, Batova A, et al.  (1996) Genomic cloning of methylthioadenosine phosphorylase: a purine metabolic enzyme deficient in multiple different cancers. Proc Natl Acad Sci U S A  93: 6203–6208.  <ul class="find" data-citedArticleID="945370" data-doi="10.1073/pnas.93.12.6203"><li><a href="http://dx.doi.org/10.1073/pnas.93.12.6203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+cloning+of+methylthioadenosine+phosphorylase%3A+a+purine+metabolic+enzyme+deficient+in+multiple+different+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+cloning+of+methylthioadenosine+phosphorylase%3A+a+purine+metabolic+enzyme+deficient+in+multiple+different+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000380-Olopade1" id="pone.0000380-Olopade1"></a>Olopade OI, Pomykala HM, Hagos F, Sveen LW, Espinosa R 3rd, et al.  (1995) Construction of a 2.8-megabase yeast artificial chromosome contig and cloning of the human methylthioadenosine phosphorylase gene from the tumor suppressor region on 9p21. Proc Natl Acad Sci U S A  92: 6489–6493.  <ul class="find" data-citedArticleID="945372" data-doi="10.1073/pnas.92.14.6489"><li><a href="http://dx.doi.org/10.1073/pnas.92.14.6489" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+a+2.8-megabase+yeast+artificial+chromosome+contig+and+cloning+of+the+human+methylthioadenosine+phosphorylase+gene+from+the+tumor+suppressor+region+on+9p21." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+a+2.8-megabase+yeast+artificial+chromosome+contig+and+cloning+of+the+human+methylthioadenosine+phosphorylase+gene+from+the+tumor+suppressor+region+on+9p21.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000380-Dijkman1" id="pone.0000380-Dijkman1"></a>Dijkman R, Tensen CP, Jordanova ES, Knijnenburg J, Hoefnagel JJ, et al.  (2006) Array-based comparative genomic hybridization analysis reveals recurrent chromosomal alterations and prognostic parameters in primary cutaneous large B-cell lymphoma. J Clin Oncol  24: 296–305.  <ul class="find" data-citedArticleID="945322" data-doi="10.1200/jco.2005.02.0842"><li><a href="http://dx.doi.org/10.1200/jco.2005.02.0842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Array-based+comparative+genomic+hybridization+analysis+reveals+recurrent+chromosomal+alterations+and+prognostic+parameters+in+primary+cutaneous+large+B-cell+lymphoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Array-based+comparative+genomic+hybridization+analysis+reveals+recurrent+chromosomal+alterations+and+prognostic+parameters+in+primary+cutaneous+large+B-cell+lymphoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000380-Jansen1" id="pone.0000380-Jansen1"></a>Jansen G, Hazendonk E, Thijssen KL, Plasterk RH (1997) Reverse genetics by chemical mutagenesis in Caenorhabditis elegans. Nat Genet  17: 119–121.  <ul class="find" data-citedArticleID="945340" data-doi="10.1038/ng0997-119"><li><a href="http://dx.doi.org/10.1038/ng0997-119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reverse+genetics+by+chemical+mutagenesis+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reverse+genetics+by+chemical+mutagenesis+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000380-Sasaki1" id="pone.0000380-Sasaki1"></a>Sasaki S, Kitagawa Y, Sekido Y, Minna JD, Kuwano H, et al.  (2003) Molecular processes of chromosome 9p21 deletions in human cancers. Oncogene  22: 3792–3798.  <ul class="find" data-citedArticleID="945390" data-doi="10.1038/sj.onc.1206589"><li><a href="http://dx.doi.org/10.1038/sj.onc.1206589" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+processes+of+chromosome+9p21+deletions+in+human+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+processes+of+chromosome+9p21+deletions+in+human+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000380-Raschke1" id="pone.0000380-Raschke1"></a>Raschke S, Balz V, Efferth T, Schulz WA, Florl AR (2005) Homozygous deletions of CDKN2A caused by alternative mechanisms in various human cancer cell lines. Genes Chromosomes Cancer  42: 58–67.  <ul class="find" data-citedArticleID="945382" data-doi="10.1002/gcc.20119"><li><a href="http://dx.doi.org/10.1002/gcc.20119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homozygous+deletions+of+CDKN2A+caused+by+alternative+mechanisms+in+various+human+cancer+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homozygous+deletions+of+CDKN2A+caused+by+alternative+mechanisms+in+various+human+cancer+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000380-Caldas1" id="pone.0000380-Caldas1"></a>Caldas C, Hahn SA, da Costa LT, Redston MS, Schutte M, et al.  (1994) Frequent somatic mutations and homozygous deletions of the p16 (MTS1) gene in pancreatic adenocarcinoma. Nat Genet  8: 27–32.  <ul class="find" data-citedArticleID="945312" data-doi="10.1038/ng0994-27"><li><a href="http://dx.doi.org/10.1038/ng0994-27" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frequent+somatic+mutations+and+homozygous+deletions+of+the+p16+%28MTS1%29+gene+in+pancreatic+adenocarcinoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frequent+somatic+mutations+and+homozygous+deletions+of+the+p16+%28MTS1%29+gene+in+pancreatic+adenocarcinoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000380-Fan1" id="pone.0000380-Fan1"></a>Fan JB, Chee MS, Gunderson KL (2006) Highly parallel genomic assays. Nat Rev Genet  7: 632–644.  <ul class="find" data-citedArticleID="945326" data-doi="10.1038/nrg1901"><li><a href="http://dx.doi.org/10.1038/nrg1901" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+parallel+genomic+assays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+parallel+genomic+assays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000380-Shendure1" id="pone.0000380-Shendure1"></a>Shendure J, Porreca GJ, Reppas NB, Lin X, McCutcheon JP, et al.  (2005) Accurate multiplex polony sequencing of an evolved bacterial genome. Science  309: 1728–1732.  <ul class="find" data-citedArticleID="945398" data-doi="10.1126/science.1117389"><li><a href="http://dx.doi.org/10.1126/science.1117389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+multiplex+polony+sequencing+of+an+evolved+bacterial+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+multiplex+polony+sequencing+of+an+evolved+bacterial+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000380-Margulies1" id="pone.0000380-Margulies1"></a>Margulies M, Egholm M, Altman WE, Attiya S, Bader JS, et al.  (2005) Genome sequencing in microfabricated high-density picolitre reactors. Nature  437: 376–380.  <ul class="find" data-citedArticleID="945366"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.&amp;auth=&amp;atitle=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000380-Service1" id="pone.0000380-Service1"></a>Service RF (2006) Gene sequencing. The race for the $1000 genome. Science  311: 1544–1546.  <ul class="find" data-citedArticleID="945394" data-doi="10.1126/science.311.5767.1544"><li><a href="http://dx.doi.org/10.1126/science.311.5767.1544" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+sequencing.+The+race+for+the+%241000+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+sequencing.+The+race+for+the+%241000+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000380-Wang1" id="pone.0000380-Wang1"></a>Wang HY, Luo M, Tereshchenko IV, Frikker DM, Cui X, et al.  (2005) A genotyping system capable of simultaneously analyzing&gt;1000 single nucleotide polymorphisms in a haploid genome. Genome Res  15: 276–283.  <ul class="find" data-citedArticleID="945412" data-doi="10.1101/gr.2885205"><li><a href="http://dx.doi.org/10.1101/gr.2885205" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genotyping+system+capable+of+simultaneously+analyzing%26gt%3B1000+single+nucleotide+polymorphisms+in+a+haploid+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genotyping+system+capable+of+simultaneously+analyzing%26gt%3B1000+single+nucleotide+polymorphisms+in+a+haploid+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000380-Lin1" id="pone.0000380-Lin1"></a>Lin JC, Wang WY, Chen KY, Wei YH, Liang WM, et al.  (2004) Quantification of plasma Epstein-Barr virus DNA in patients with advanced nasopharyngeal carcinoma. N Engl J Med  350: 2461–2470.  <ul class="find" data-citedArticleID="945358" data-doi="10.1056/nejmoa032260"><li><a href="http://dx.doi.org/10.1056/nejmoa032260" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantification+of+plasma+Epstein-Barr+virus+DNA+in+patients+with+advanced+nasopharyngeal+carcinoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantification+of+plasma+Epstein-Barr+virus+DNA+in+patients+with+advanced+nasopharyngeal+carcinoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000380-To1" id="pone.0000380-To1"></a>To EW, Chan KC, Leung SF, Chan LY, To KF, et al.  (2003) Rapid clearance of plasma Epstein-Barr virus DNA after surgical treatment of nasopharyngeal carcinoma. Clin Cancer Res  9: 3254–3259.  <ul class="find" data-citedArticleID="945404"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rapid+clearance+of+plasma+Epstein-Barr+virus+DNA+after+surgical+treatment+of+nasopharyngeal+carcinoma.&amp;auth=&amp;atitle=Rapid+clearance+of+plasma+Epstein-Barr+virus+DNA+after+surgical+treatment+of+nasopharyngeal+carcinoma." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+clearance+of+plasma+Epstein-Barr+virus+DNA+after+surgical+treatment+of+nasopharyngeal+carcinoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+clearance+of+plasma+Epstein-Barr+virus+DNA+after+surgical+treatment+of+nasopharyngeal+carcinoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000380-Cayuela1" id="pone.0000380-Cayuela1"></a>Cayuela JM, Gardie B, Sigaux F (1997) Disruption of the multiple tumor suppressor gene MTS1/p16(INK4a)/CDKN2 by illegitimate V(D)J recombinase activity in T-cell acute lymphoblastic leukemias. Blood  90: 3720–3726.  <ul class="find" data-citedArticleID="945318"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Disruption+of+the+multiple+tumor+suppressor+gene+MTS1%2Fp16%28INK4a%29%2FCDKN2+by+illegitimate+V%28D%29J+recombinase+activity+in+T-cell+acute+lymphoblastic+leukemias.&amp;auth=&amp;atitle=Disruption+of+the+multiple+tumor+suppressor+gene+MTS1%2Fp16%28INK4a%29%2FCDKN2+by+illegitimate+V%28D%29J+recombinase+activity+in+T-cell+acute+lymphoblastic+leukemias." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disruption+of+the+multiple+tumor+suppressor+gene+MTS1%2Fp16%28INK4a%29%2FCDKN2+by+illegitimate+V%28D%29J+recombinase+activity+in+T-cell+acute+lymphoblastic+leukemias." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disruption+of+the+multiple+tumor+suppressor+gene+MTS1%2Fp16%28INK4a%29%2FCDKN2+by+illegitimate+V%28D%29J+recombinase+activity+in+T-cell+acute+lymphoblastic+leukemias.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000380-Kitagawa1" id="pone.0000380-Kitagawa1"></a>Kitagawa Y, Inoue K, Sasaki S, Hayashi Y, Matsuo Y, et al.  (2002) Prevalent involvement of illegitimate V(D)J recombination in chromosome 9p21 deletions in lymphoid leukemia. J Biol Chem  277: 46289–46297.  <ul class="find" data-citedArticleID="945356" data-doi="10.1074/jbc.m208353200"><li><a href="http://dx.doi.org/10.1074/jbc.m208353200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalent+involvement+of+illegitimate+V%28D%29J+recombination+in+chromosome+9p21+deletions+in+lymphoid+leukemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalent+involvement+of+illegitimate+V%28D%29J+recombination+in+chromosome+9p21+deletions+in+lymphoid+leukemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000380-vanderVelden1" id="pone.0000380-vanderVelden1"></a>van der Velden VH, Hochhaus A, Cazzaniga G, Szczepanski T, Gabert J, et al.  (2003) Detection of minimal residual disease in hematologic malignancies by real-time quantitative PCR: principles, approaches, and laboratory aspects. Leukemia  17: 1013–1034.  <ul class="find" data-citedArticleID="945418" data-doi="10.1038/sj.leu.2402922"><li><a href="http://dx.doi.org/10.1038/sj.leu.2402922" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+minimal+residual+disease+in+hematologic+malignancies+by+real-time+quantitative+PCR%3A+principles%2C+approaches%2C+and+laboratory+aspects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+minimal+residual+disease+in+hematologic+malignancies+by+real-time+quantitative+PCR%3A+principles%2C+approaches%2C+and+laboratory+aspects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000380-Goulden1" id="pone.0000380-Goulden1"></a>Goulden N, Virgo P, Grimwade D (2006) Minimal residual disease directed therapy for childhood acute myeloid leukaemia: the time is now. Br J Haematol  134: 273–282.  <ul class="find" data-citedArticleID="945330" data-doi="10.1111/j.1365-2141.2006.06182.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2141.2006.06182.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Minimal+residual+disease+directed+therapy+for+childhood+acute+myeloid+leukaemia%3A+the+time+is+now." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Minimal+residual+disease+directed+therapy+for+childhood+acute+myeloid+leukaemia%3A+the+time+is+now.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000380-Hughes1" id="pone.0000380-Hughes1"></a>Hughes T, Deininger M, Hochhaus A, Branford S, Radich J, et al.  (2006) Monitoring CML patients responding to treatment with tyrosine kinase inhibitors: review and recommendations for harmonizing current methodology for detecting BCR-ABL transcripts and kinase domain mutations and for expressing results. Blood  108: 28–37.  <ul class="find" data-citedArticleID="945338" data-doi="10.1182/blood-2006-01-0092"><li><a href="http://dx.doi.org/10.1182/blood-2006-01-0092" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monitoring+CML+patients+responding+to+treatment+with+tyrosine+kinase+inhibitors%3A+review+and+recommendations+for+harmonizing+current+methodology+for+detecting+BCR-ABL+transcripts+and+kinase+domain+mutations+and+for+expressing+results." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monitoring+CML+patients+responding+to+treatment+with+tyrosine+kinase+inhibitors%3A+review+and+recommendations+for+harmonizing+current+methodology+for+detecting+BCR-ABL+transcripts+and+kinase+domain+mutations+and+for+expressing+results.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000380-Szczepanski1" id="pone.0000380-Szczepanski1"></a>Szczepanski T (2007) Why and how to quantify minimal residual disease in acute lymphoblastic leukemia? Leukemia..   <ul class="find" data-citedArticleID="945402" data-doi="10.1038/sj.leu.2404603"><li><a href="http://dx.doi.org/10.1038/sj.leu.2404603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+and+how+to+quantify+minimal+residual+disease+in+acute+lymphoblastic+leukemia%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+and+how+to+quantify+minimal+residual+disease+in+acute+lymphoblastic+leukemia%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000380-Cazzaniga1" id="pone.0000380-Cazzaniga1"></a>Cazzaniga G, Gaipa G, Rossi V, Biondi A (2006) Monitoring of minimal residual disease in leukemia, advantages and pitfalls. Ann Med  38: 512–521.  <ul class="find" data-citedArticleID="945320" data-doi="10.1080/07853890600978123"><li><a href="http://dx.doi.org/10.1080/07853890600978123" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monitoring+of+minimal+residual+disease+in+leukemia%2C+advantages+and+pitfalls." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monitoring+of+minimal+residual+disease+in+leukemia%2C+advantages+and+pitfalls.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000380-Janz1" id="pone.0000380-Janz1"></a>Janz S (2006) Myc translocations in B cell and plasma cell neoplasms. DNA Repair (Amst)  5: 1213–1224.  <ul class="find" data-citedArticleID="945342" data-doi="10.1016/j.dnarep.2006.05.017"><li><a href="http://dx.doi.org/10.1016/j.dnarep.2006.05.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Myc+translocations+in+B+cell+and+plasma+cell+neoplasms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Myc+translocations+in+B+cell+and+plasma+cell+neoplasms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000380-Joos1" id="pone.0000380-Joos1"></a>Joos S, Falk MH, Lichter P, Haluska FG, Henglein B, et al.  (1992) Variable breakpoints in Burkitt lymphoma cells with chromosomal t(8;14) translocation separate c-myc and the IgH locus up to several hundred kb. Hum Mol Genet  1: 625–632.  <ul class="find" data-citedArticleID="945344" data-doi="10.1093/hmg/1.8.625"><li><a href="http://dx.doi.org/10.1093/hmg/1.8.625" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variable+breakpoints+in+Burkitt+lymphoma+cells+with+chromosomal+t%288%3B14%29+translocation+separate+c-myc+and+the+IgH+locus+up+to+several+hundred+kb." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variable+breakpoints+in+Burkitt+lymphoma+cells+with+chromosomal+t%288%3B14%29+translocation+separate+c-myc+and+the+IgH+locus+up+to+several+hundred+kb.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000380-Joos2" id="pone.0000380-Joos2"></a>Joos S, Haluska FG, Falk MH, Henglein B, Hameister H, et al.  (1992) Mapping chromosomal breakpoints of Burkitt's t(8;14) translocations far upstream of c-myc. Cancer Res  52: 6547–6552.  <ul class="find" data-citedArticleID="945346"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mapping+chromosomal+breakpoints+of+Burkitt%27s+t%288%3B14%29+translocations+far+upstream+of+c-myc.&amp;auth=&amp;atitle=Mapping+chromosomal+breakpoints+of+Burkitt%27s+t%288%3B14%29+translocations+far+upstream+of+c-myc." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+chromosomal+breakpoints+of+Burkitt%27s+t%288%3B14%29+translocations+far+upstream+of+c-myc." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+chromosomal+breakpoints+of+Burkitt%27s+t%288%3B14%29+translocations+far+upstream+of+c-myc.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000380-Eisen1" id="pone.0000380-Eisen1"></a>Eisen MB, Brown PO (1999) DNA arrays for analysis of gene expression. Methods Enzymol  303: 179–205.  <ul class="find" data-citedArticleID="945324" data-doi="10.1016/s0076-6879(99)03014-1"><li><a href="http://dx.doi.org/10.1016/s0076-6879(99)03014-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+arrays+for+analysis+of+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+arrays+for+analysis+of+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000380-Wang2" id="pone.0000380-Wang2"></a>Wang D, Urisman A, Liu YT, Springer M, Ksiazek TG, et al.  (2003) Viral discovery and sequence recovery using DNA microarrays. PLoS Biol  1: E2.  <ul class="find" data-citedArticleID="945414" data-doi="10.1371/journal.pbio.0000002"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0000002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viral+discovery+and+sequence+recovery+using+DNA+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Viral+discovery+and+sequence+recovery+using+DNA+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000380-Hubbard1" id="pone.0000380-Hubbard1"></a>Hubbard T, Andrews D, Caccamo M, Cameron G, Chen Y, et al.  (2005) Ensembl 2005. Nucleic Acids Res  33: D447–453.  <ul class="find" data-citedArticleID="945336" data-doi="10.1093/nar/gki138"><li><a href="http://dx.doi.org/10.1093/nar/gki138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ensembl+2005." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ensembl+2005.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.XML" value="103876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.PDF" value="316657"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g001.PNG_L" value="228197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g001.PNG_M" value="61269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g001.PNG_S" value="3368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g001.TIF" value="470748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g001.PNG_I" value="23836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g002.PNG_L" value="84090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g002.PNG_M" value="48735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g002.PNG_S" value="2954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g002.TIF" value="115470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g002.PNG_I" value="17394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g003.PNG_L" value="672383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g003.PNG_M" value="169009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g003.PNG_S" value="5654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g003.TIF" value="942374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g003.PNG_I" value="59447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g004.PNG_L" value="66407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g004.PNG_M" value="64123"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g004.PNG_S" value="3051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g004.TIF" value="175076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g004.PNG_I" value="26995"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g005.PNG_L" value="75891"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g005.PNG_M" value="101179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g005.PNG_S" value="3740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g005.TIF" value="280264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g005.PNG_I" value="43712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.t001.PNG_L" value="51207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.t001.PNG_M" value="48476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.t001.PNG_S" value="2681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.t001.TIF" value="320038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.t001.PNG_I" value="21950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g006.PNG_L" value="69065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g006.PNG_M" value="83478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g006.PNG_S" value="3936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g006.TIF" value="289648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.g006.PNG_I" value="35975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.t002.PNG_L" value="131496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.t002.PNG_M" value="111602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.t002.PNG_S" value="6567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.t002.TIF" value="666936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000380.t002.PNG_I" value="71110"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000380&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000380" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000380&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000380&volume=&issue=&title=A Novel Approach for Determining Cancer Genomic Breakpoints in the Presence of Normal DNA&author_name=Yu-Tsueng%20Liu%2C%20Dennis%20A.%20Carson&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000380" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000380" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000380" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000380&amp;t=A%20Novel%20Approach%20for%20Determining%20Cancer%20Genomic%20Breakpoints%20in%20the%20Presence%20of%20Normal%20DNA" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000380&title=A%20Novel%20Approach%20for%20Determining%20Cancer%20Genomic%20Breakpoints%20in%20the%20Presence%20of%20Normal%20DNA&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000380&amp;title=A%20Novel%20Approach%20for%20Determining%20Cancer%20Genomic%20Breakpoints%20in%20the%20Presence%20of%20Normal%20DNA" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000380" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000380&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Novel Approach for Determining Cancer Genomic Breakpoints in the Presence of Normal DNA';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000380';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Novel%20Approach%20for%20Determining%20Cancer%20Genomic%20Breakpoints%20in%20the%20Presence%20of%20Normal%20DNA http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000380" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000380" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+hybridization%22" title="Search for articles in the subject area:'Cell hybridization'"><div class="flagText">Cell hybridization</div></a>
              <div data-categoryid="21301" data-articleid="23598"
                   data-categoryname="Cell hybridization"
                   class="flagImage" title="Flag 'Cell hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="23598"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23598"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Exon+mapping%22" title="Search for articles in the subject area:'Exon mapping'"><div class="flagText">Exon mapping</div></a>
              <div data-categoryid="22245" data-articleid="23598"
                   data-categoryname="Exon mapping"
                   class="flagImage" title="Flag 'Exon mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23598"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+signal+processing%22" title="Search for articles in the subject area:'Genomic signal processing'"><div class="flagText">Genomic signal processing</div></a>
              <div data-categoryid="18635" data-articleid="23598"
                   data-categoryname="Genomic signal processing"
                   class="flagImage" title="Flag 'Genomic signal processing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="23598"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Probe+hybridization%22" title="Search for articles in the subject area:'Probe hybridization'"><div class="flagText">Probe hybridization</div></a>
              <div data-categoryid="48355" data-articleid="23598"
                   data-categoryname="Probe hybridization"
                   class="flagImage" title="Flag 'Probe hybridization' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7627'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6264'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7656&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=21013">update: computational design </a><br>Posted by ytliu</p>
      <p><a href="/annotation/listThread.action?root=18063">Targeting the absence</a><br>Posted by ytliu</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
