

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Meiotic Transmission of Drosophila pseudoobscura Chromosomal Arrangements</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000530"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000530" />

    <meta name="citation_title" content="Meiotic Transmission of Drosophila pseudoobscura Chromosomal Arrangements"/>
    <meta itemprop="name" content="Meiotic Transmission of Drosophila pseudoobscura Chromosomal Arrangements"/>

      <meta name="citation_author" content="Richard P. Meisel"/>
            <meta name="citation_author_institution" content="Intercollege Graduate Program in Genetics, Institute of Molecular Evolutionary Genetics, and Department of Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Stephen W. Schaeffer"/>
            <meta name="citation_author_institution" content="Intercollege Graduate Program in Genetics, Institute of Molecular Evolutionary Genetics, and Department of Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America"/>

    <meta name="citation_date" content="2007/6/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000530.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e530"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Inversion polymorphisms in Drosophila pseudoobscura and Drosophila persimilis.; citation_author=JR Powell; citation_number=1; citation_pages=73-126; citation_date=1992; citation_publisher=CRC Press; " />
      <meta name="citation_reference" content="citation_title=Genetics of natural populations. XVI. Altitudinal and seasonal changes produce by natural selection in certain populations of Drosophila pseudoobscura and Drosophila persimilis.; citation_author=T Dobzhansky; citation_journal_title=Genetics; citation_volume=33; citation_number=2; citation_pages=158-176; citation_date=1948; " />
      <meta name="citation_reference" content="citation_title=Four decades of inversion polymorphism in Drosophila pseudoobscura.; citation_author=WW Anderson; citation_author=J Arnold; citation_author=D Baldwin; citation_author=A Beckenbach; citation_author=C Brown; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=88; citation_number=3; citation_pages=10367-10371; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Distinguishing the forces controlling genetic variation at the Xdh locus in Drosophila pseudoobscura.; citation_author=MA Riley; citation_author=ME Hallas; citation_author=RC Lewontin; citation_journal_title=Genetics; citation_volume=123; citation_number=4; citation_pages=359-369; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Estimates of gene flow in Drosophila pseudoobscura determined from nucleotide sequence analysis of the alcohol dehydrogenase region.; citation_author=SW Schaeffer; citation_author=EL Miller; citation_journal_title=Genetics; citation_volume=132; citation_number=5; citation_pages=471-480; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Molecular population genetics of X-linked genes in Drosophila pseudoobscura.; citation_author=M Kovacevic; citation_author=SW Schaeffer; citation_journal_title=Genetics; citation_volume=156; citation_number=6; citation_pages=155-172; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genetics of natural populations. XII. Experimental reproduction of some of the changes caused by natural selection in certain populations of Drosophila pseudoobscura.; citation_author=S Wright; citation_author=T Dobzhansky; citation_journal_title=Genetics; citation_volume=31; citation_number=7; citation_pages=125-156; citation_date=1946; " />
      <meta name="citation_reference" content="citation_title=Genetics of natural populations. XIX. Origin of heterosis through natural selection in populations of Drosophila pseudoobscura.; citation_author=T Dobzhansky; citation_journal_title=Genetics; citation_volume=35; citation_number=8; citation_pages=288-302; citation_date=1950; " />
      <meta name="citation_reference" content="citation_title=Selection in experimental populations of Drosophila pseudoobscura with different initial frequencies of chromosomal variants.; citation_author=T Watanabe; citation_author=WW Anderson; citation_author=T Dobzhansky; citation_author=O Pavlovsky; citation_journal_title=Genet Res; citation_volume=15; citation_number=9; citation_pages=123-129; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Genotypic background and heterosis in Drosophila pseudoobscura.; citation_author=L Levine; citation_journal_title=Genetics; citation_volume=40; citation_number=10; citation_pages=832-849; citation_date=1955; " />
      <meta name="citation_reference" content="citation_title=Evolution of fitness in experimental populations of Drosophila pseudoobscura.; citation_author=MW Strickberger; citation_journal_title=Evolution; citation_volume=17; citation_number=11; citation_pages=40-55; citation_date=1963; " />
      <meta name="citation_reference" content="citation_title=Further evidence for coadaptation in crosses between geographic populations of Drosophila pseudoobscura.; citation_author=WW Anderson; citation_journal_title=Genet Res; citation_volume=12; citation_number=12; citation_pages=317-330; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Evolutionary genomics of inversions in Drosophila pseudoobscura: evidence for epistasis.; citation_author=SW Schaeffer; citation_author=MP Goetting-Minesky; citation_author=M Kovacevic; citation_author=JR Peoples; citation_author=JL Graybill; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=100; citation_number=13; citation_pages=8319-8324; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of genetic exchange within the chromosomal inversions of Drosophila pseudoobscura.; citation_author=SW Schaeffer; citation_author=WW Anderson; citation_journal_title=Genetics; citation_volume=171; citation_number=14; citation_pages=1729-1739; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The suppression of crossing over in inversion heterozygotes of Drosophila pseudoobscura.; citation_author=T Dobzhansky; citation_author=C Epling; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=34; citation_number=15; citation_pages=137-141; citation_date=1948; " />
      <meta name="citation_reference" content="citation_title=Crossing over and inversions in coadapted systems.; citation_author=RP Levine; citation_journal_title=Am Nat; citation_volume=90; citation_number=16; citation_pages=41-45; citation_date=1956; " />
      <meta name="citation_reference" content="citation_title=Variable crossing over arising in different strains of Drosophila pseudoobscura.; citation_author=RP Levine; citation_author=EE Levine; citation_journal_title=Genetics; citation_volume=40; citation_number=17; citation_pages=399-405; citation_date=1955; " />
      <meta name="citation_reference" content="citation_title=Meiotic segregation in Drosophila melanogaster females: moleculer, mechanisms, and myths.; citation_author=RS Hawley; citation_author=KS McKim; citation_author=T Arbel; citation_journal_title=Annu Rev Genet; citation_volume=27; citation_number=18; citation_pages=281-319; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Meiosis in Drosophila: seeing is believing.; citation_author=TL Orr-Weaver; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=92; citation_number=19; citation_pages=10443-10449; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Preferential segregation in triplo-IV females of Drosophila melanogaster.; citation_author=AH Sturtevant; citation_journal_title=Genetics; citation_volume=21; citation_number=20; citation_pages=444-466; citation_date=1936; " />
      <meta name="citation_reference" content="citation_title=The comparative genetics of Drosophila pseudoobscura and Drosophila melanogaster.; citation_author=AH Sturtevant; citation_author=CC Tan; citation_journal_title=J Genet; citation_volume=34; citation_number=21; citation_pages=415-432; citation_date=1937; " />
      <meta name="citation_reference" content="citation_title=Comparative genome sequencing of Drosophila pseudoobscura: chromosomal, gene, and cis-element evolution.; citation_author=S Richards; citation_author=Y Liu; citation_author=BR Bettencourt; citation_author=P Hradecky; citation_author=S Letovsky; citation_journal_title=Genome Res; citation_volume=15; citation_number=22; citation_pages=1-18; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=RR Sokal; citation_author=FJ Rohlf; citation_number=23; citation_date=1995; citation_publisher=W.H. Freeman and Co; " />
      <meta name="citation_reference" content="citation_title=The evolutionary history of the amylase multigene family in Drosophila pseudoobscura.; citation_author=A Popadic; citation_author=R Norman; citation_author=W Doanet; citation_author=W Anderson; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=24; citation_pages=883-888; citation_date=1996; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000530" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Meiotic Transmission of Drosophila pseudoobscura Chromosomal Arrangements"/>
    <meta name="twitter:description" content="Drosophila pseudoobscura harbors a rich gene arrangement polymorphism on the third chromosome generated by a series of overlapping paracentric inversions. The arrangements suppress recombination in heterokaryotypic individuals, which allows for the selective maintenance of coadapted gene complexes. Previous mapping experiments used to determine the degree to which recombination is suppressed in gene arrangement heterozygotes produced non-recombinant progeny in non-Mendelian ratios. The deviations from Mendelian expectations could be the result of viability differences between wild and mutant chromosomes, meiotic drive because of achiasmate pairing of homologues in heterokaryotypic females during meiosis, or a combination of both mechanisms. The possibility that the frequencies of the chromosomal arrangements in natural populations are affected by mechanisms other than adaptive selection led us to consider these hypotheses. We performed reciprocal crosses involving both heterozygous males and females to determine if the frequency of the non-recombinant progeny deviates significantly from Mendelian expectations and if the frequencies deviate between reciprocal crosses. We failed to observe non-Mendelian ratios in multiple crosses, and the frequency of the non-recombinant classes differed in only one of five pairs of reciprocal crosses despite sufficient power to detect these differences in all crosses. Our results indicate that deviations from Mendelian expectations in recombination experiments involving the D. pseudoobscura inversion system are most likely due to fitness differences of gene arrangement karyotypes in different environments."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000530.g004"/>

  <meta property="og:title" content="Meiotic Transmission of Drosophila pseudoobscura Chromosomal Arrangements" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000530" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=800'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9728'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5533&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000530">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000530" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000530&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Meiotic Transmission of <i>Drosophila pseudoobscura</i> Chromosomal Arrangements
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Richard P. Meisel
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:meisel@psu.edu">meisel@psu.edu</a></p>

                <p>Affiliation:
                  Intercollege Graduate Program in Genetics, Institute of Molecular Evolutionary Genetics, and Department of Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen W. Schaeffer
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Intercollege Graduate Program in Genetics, Institute of Molecular Evolutionary Genetics, and Department of Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 13, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000530</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000530" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000530" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000530" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000530" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000530-g001" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000530-g002" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000530-t001" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000530-g003" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000530-g004" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000530-t002" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000530-t003" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000530">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000530" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p><em>Drosophila pseudoobscura</em> harbors a rich gene arrangement polymorphism on the third chromosome generated by a series of overlapping paracentric inversions. The arrangements suppress recombination in heterokaryotypic individuals, which allows for the selective maintenance of coadapted gene complexes. Previous mapping experiments used to determine the degree to which recombination is suppressed in gene arrangement heterozygotes produced non-recombinant progeny in non-Mendelian ratios. The deviations from Mendelian expectations could be the result of viability differences between wild and mutant chromosomes, meiotic drive because of achiasmate pairing of homologues in heterokaryotypic females during meiosis, or a combination of both mechanisms. The possibility that the frequencies of the chromosomal arrangements in natural populations are affected by mechanisms other than adaptive selection led us to consider these hypotheses. We performed reciprocal crosses involving both heterozygous males and females to determine if the frequency of the non-recombinant progeny deviates significantly from Mendelian expectations and if the frequencies deviate between reciprocal crosses. We failed to observe non-Mendelian ratios in multiple crosses, and the frequency of the non-recombinant classes differed in only one of five pairs of reciprocal crosses despite sufficient power to detect these differences in all crosses. Our results indicate that deviations from Mendelian expectations in recombination experiments involving the <em>D. pseudoobscura</em> inversion system are most likely due to fitness differences of gene arrangement karyotypes in different environments.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Meisel RP, Schaeffer SW (2007) Meiotic Transmission of <em>Drosophila pseudoobscura</em> Chromosomal Arrangements. PLoS ONE 2(6):
          e530.
            doi:10.1371/journal.pone.0000530</p><p><strong>Academic Editor: </strong>Beth Sullivan, Duke University, United States of America</p><p><strong>Received:</strong> April 24, 2007; <strong>Accepted:</strong> May 22, 2007; <strong>Published:</strong> June 13, 2007</p><p><strong>Copyright:</strong> © 2007 Meisel, Schaeffer. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>No specific funding was used in this research.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Drosophila pseudoobscura</em> harbors a rich gene arrangement polymorphism on the third chromosome generated by a series of overlapping inversions <a href="#pone.0000530-Powell1">[1]</a> (<a href="#pone-0000530-g001">Figure 1</a>); over thirty different arrangements segregate in natural populations. The polymorphism has been thought to be maintained by natural selection in wild populations because the arrangements cycle seasonally, form altitudinal clines <a href="#pone.0000530-Dobzhansky1">[2]</a>, and form stable geographical clines <a href="#pone.0000530-Anderson1">[3]</a> despite extensive gene flow <a href="#pone.0000530-Riley1">[4]</a>–<a href="#pone.0000530-Kovacevic1">[6]</a>. Population cage experiments indicate that there are fitness differences between arrangement genotypes <a href="#pone.0000530-Wright1">[7]</a>, <a href="#pone.0000530-Dobzhansky2">[8]</a>. These laboratory crosses suggest that heterokaryotypic individuals within a population have a fitness advantage, but this advantage is lost when arrangements from different populations are combined.</p>
<div class="figure" id="pone-0000530-g001"><div class="img"><a name="pone-0000530-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000530.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000530.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Subset of <em>Drosophila pseudoobscura</em> third chromosome inversion network.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>The sets of inversion events thought to be responsible for a subset of the arrangements polymorphic in populations of <em>D. pseudoobscura</em> are shown. Arrows indicate single paracentric inversions; ambiguous inversion events (those for which the ancestral arrangement is unclear) are shown with double-ended arrows <a href="#pone.0000530-Popadic1">[24]</a>. Hypothetical (HY) is a necessary intermediate that has never been collected. The Olympic (OL), Chiricahua (CH), and Vandeventer Flat (VF) arrangements were used in previous recombination rate experiments <a href="#pone.0000530-Dobzhansky3">[15]</a>–<a href="#pone.0000530-Levine3">[17]</a>, but not in this study.</p>
<span>doi:10.1371/journal.pone.0000530.g001</span></div><a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The coadaptation model was developed to explain the apparent fitness differences of karyotypes in natural populations and laboratory crosses. Under this model, natural selection maintains favorable combinations of alleles within and between arrangements within populations, and suppressed recombination in arrangement heterozygotes prevents the breakup of coadapted alleles within arrangements <a href="#pone.0000530-Dobzhansky2">[8]</a>–<a href="#pone.0000530-Anderson2">[12]</a>. Despite the strong experimental evidence, a molecular study failed to find differentiation within gene arrangements between populations <a href="#pone.0000530-Schaeffer2">[13]</a>. The molecular data do provide evidence for coadaptation of alleles within different arrangements, as strong linkage disequilibrium is observed between distant loci within arrangements despite no statistical association between more closely linked sites. This suggests that strong epistatic selection maintains favorable combinations of alleles within arrangements in the face of some genetic recombination <a href="#pone.0000530-Schaeffer3">[14]</a>. A model with spatially varying selection coefficients can explain the maintenance of the arrangement polymorphism without differentiation of arrangements between populations (S.W. Schaeffer, unpublished).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The coadaptation model relies on suppressed recombination in arrangement heterozygotes, but it also assumes equal transmission of homologous chromosomes during meiosis. Previous studies of recombination in the <em>D. pseudoobscura</em> third chromosome inversion system confirm suppressed recombination in heterokaryotypic individuals <a href="#pone.0000530-Dobzhansky3">[15]</a>–<a href="#pone.0000530-Levine3">[17]</a>, but all three studies reported non-recombinant progeny transmitted in non-Mendelian ratios (<a href="#pone.0000530.s001">Figure S1</a>). These deviations from Mendelian expectations can be attributed to fitness differences between the wild-type chromosomes and the mutant marker chromosomes, meiotic drive due to achiasmate pairing of homologues in heterokaryotypic females during meiosis, or some combination of both mechanisms.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Previous genetic mapping experiments <a href="#pone.0000530-Dobzhansky3">[15]</a>–<a href="#pone.0000530-Levine3">[17]</a> measured recombination rates between phenotypic mutations in females carrying mutant marked chromosomes on the Standard (ST) gene arrangement background and various wild arrangements (<a href="#pone-0000530-g001">Figure 1</a>). The heterozygous females were crossed to mutant males, and the progeny were scored based on phenotype. The viability hypothesis is supported by observations that there was a significant excess of wild type non-recombinant chromosomes relative to mutant non-recombinants in most of the crosses <a href="#pone.0000530-Dobzhansky3">[15]</a>, <a href="#pone.0000530-Levine3">[17]</a> (<a href="#pone.0000530.s001">Figure S1</a>). Dobzhansky and Epling <a href="#pone.0000530-Dobzhansky3">[15]</a> performed two sets of crosses using two different mutant marked chromosomes, and the heterozygous females carried one of four wild arrangements. The meiotic drive hypothesis was supported by these mapping data because there is a positive correlation between the number of inversions differentiating the marked and wild chromosomes and the amount of deviation from Mendelian expectations (<a href="#pone.0000530.s001">Figure S1</a>). The non-recombinant chromosomes probably segregated achiasmately – achiasmate segregation differs from segregation with meiotic exchange in <em>Drosophila</em> <a href="#pone.0000530-Hawley1">[18]</a>, and inversion differences between homologs increase the probability that the achiasmate pathway is used. This suggests that an achiasmate pathway may allow for non-random disjunction of homologs when they differ by multiple rearrangements, with one homolog preferentially segregating to the oocyte as opposed to the first polar body.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Using a different set of wild-type arrangements, Levine <a href="#pone.0000530-Levine2">[16]</a> observed a significant deficiency of wild-type non-recombinants relative to mutant non-recombinants in three of four crosses (<a href="#pone.0000530.s001">Figure S1</a>). This suggests that viability effects of the mutations may not be entirely responsible for the excess of wild-type non-recombinant chromosomes observed in the other experiments <a href="#pone.0000530-Dobzhansky3">[15]</a>, <a href="#pone.0000530-Levine3">[17]</a>. Analysis of the transmission of individual marker loci in the seven crosses involving homokaryotypic females (where recombination is high enough to allow for shuffling of alleles) reveals one instance in which a particular mutant allele is transmitted significantly more often than the other markers in that cross and one instance in which a mutant allele is transmitted less often (<a href="#pone.0000530.s002">Figure S2</a>); the markers with different frequencies of transmission differ between these two crosses. There is no evidence that a single mutation contributes excessively to the deviations from Mendelian expectations.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>The mechanisms of meiotic segregation differ between male and female <em>Drosophila</em> <a href="#pone.0000530-OrrWeaver1">[19]</a>. A single male meiosis ends with four haploid gametes, but female meiosis produces a single haploid oocyte and two polar bodies. In males, achiasmate segregation is the norm, whereas it only occurs in females when homologues differ by inversions. Furthermore, the achiasmate pathway has been shown to be associated with non-random disjunction in females <a href="#pone.0000530-Sturtevant1">[20]</a>. If achiasmate pairing between rearranged chromosomes in heterokaryotypic females leads to non-random disjunction, we should observe deviations from Mendelian expectations only in female meioses and not in males (E. Novitski, University of Oregon, pers. comm.). If, however, viability differences between wild and mutant chromosomes cause the deviations, they should be observed using both heterozygous males and females. The previous analyses of recombination in <em>D. pseudoobscura</em> <a href="#pone.0000530-Dobzhansky3">[15]</a>–<a href="#pone.0000530-Levine3">[17]</a> only examined meiosis in heterozygous females, so they were not able to distinguish between the alternative hypotheses of meiotic drive and fitness effects.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>We performed reciprocal crosses to examine the meiotic segregation of <em>D. pseudoobscura</em> third chromosomes carrying phenotypic markers and wild-type chromosomes each carrying one of five naturally occurring third chromosome gene arrangements (<a href="#pone-0000530-g002">Figure 2</a>). The five arrangements were chosen for both their appreciable frequencies in natural populations <a href="#pone.0000530-Anderson1">[3]</a> and their position along the backbone of the inversion network (<a href="#pone-0000530-g001">Figure 1</a>). Heterozygous females and males were both crossed to a strain carrying a multiply marked Arrowhead (AR) gene arrangement to test for deviations from Mendelian expectations in the non-recombinant progeny and differences in the frequency of wild-type non-recombinant progeny between the reciprocal crosses. A significant difference in the fraction of wild-type non-recombinants between reciprocal crosses would support the meiotic drive hypothesis. Our data provide no support for meiotic drive operating on the <em>D. pseudoobscura</em> third chromosome, meaning that the deviations from expected Mendelian ratios in the previous experiments were most likely due to fitness effects.</p>
<div class="figure" id="pone-0000530-g002"><div class="img"><a name="pone-0000530-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000530.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000530.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Crossing scheme to determine the frequency of wild-type non-recombinant individuals.</span></strong></p><a id="article1.body1.sec1.fig2.caption1.p1" name="article1.body1.sec1.fig2.caption1.p1"></a><p>Females carrying a marker chromosome on the AR gene arrangement were crossed to males carrying one of five wild-type chromosomes (AR, ST, PP, SC, or TL). Individual male and female mutant progeny were test crossed to a marker stock carrying the <em>or</em> and <em>pr</em> mutations on an AR background. Single pair matings involving heterozygotes of the same sex carrying the same wild-type gene arrangement are considered replicates.</p>
<span>doi:10.1371/journal.pone.0000530.g002</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We mated single females heterozygous for a mutant marked chromosome on an AR background and a wild-type chromosome carrying either the AR, ST, Pikes Peak (PP), Santa Cruz (SC), or Tree Line (TL) arrangement to single males homozygous for chromosomes carrying some of the same mutant markers (<a href="#pone-0000530-g002">Figure 2</a>). The reciprocal crosses were also performed. Each single pair mating was carried out in multiple replicates (<a href="#pone-0000530-g002">Figure 2</a>, <a href="#pone-0000530-t001">Table 1</a>), and the replicates were analyzed by both adding the results of all replicates together (sum across replicates) and by taking the average of replicates. If meiotic drive were responsible for non-Mendelian ratios, the reciprocal crosses should differ in the frequency of individuals in each of the two non-recombinant classes. If selection were responsible for deviations from Mendelian expectations, the reciprocal crosses should show similar departures (or lack thereof) from equal frequencies of each non-recombinant class.</p>
<div class="figure" id="pone-0000530-t001"><div class="img"><a name="pone-0000530-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000530.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000530.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Frequency of wild-type non-recombinants.</span></strong></p><span>doi:10.1371/journal.pone.0000530.t001</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>The frequency of wild-type non-recombinant offspring was significantly greater than 50% for the crosses involving females carrying the wild-type ST and SC arrangements and the crosses involving males carrying the wild-type AR and ST arrangements when summing across replicates (<a href="#pone-0000530-t001">Table 1</a>). The observed frequencies of the wild-type non-recombinant chromosomes are not significantly different from Mendelian expectations when averaging replicates (<a href="#pone-0000530-t001">Table 1</a>). We only observe a significant difference in the frequency of transmission of non-recombinant chromosomes in one of the reciprocal pairs, that involving the wild-type AR chromosome when analyzing the sum across replicates (<em>z</em> = 2.317, <em>p</em> = 0.02). There is no evidence to support the meiotic drive hypothesis because males and females show the same transmission frequency of wild-type non-recombinant chromosomes when heterozygotes carry arrangements that differ by at least one inversion.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>Failure to reject the null hypothesis could occur if the null hypothesis is true or if we lack sufficient power to reject. Significant deviations from Mendelian expectations should be detected using the sum across replicates for any of the arrangements if the actual frequency of transmission of the wild-type non-recombinant chromosome is at least 0.54 (based on our sample sizes). Assuming the wild-type chromosome is transmitted according to Mendelian expectations in heterozygous males, we should be able to detect significant differences in the frequency of wild-type non-recombinants between the sexes when summing across replicates if the wild-type chromosome is transmitted at a frequency of at least 0.54 in heterozygous females carrying the AR, ST, PP, and SC wild-type arrangements and at least 0.55 in TL females (based on our sample sizes). Simulations were carried out to model our cross scheme using different expected frequencies of transmission of the wild-type non-recombinants. For all sets of crosses, we have a power greater than 0.80 to detect significant deviations from Mendelian expectations using the average of replicates when the actual frequency of transmission is at least 0.75 (<a href="#pone-0000530-g003">Figure 3</a>)<a href="#pone-0000530-g004"></a>.</p>
<div class="figure" id="pone-0000530-g003"><div class="img"><a name="pone-0000530-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000530.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000530.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Power to detect significant excess of wild-type non-recombinants.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>The power to detect significant excess of wild-type non-recombinants (<em>p</em>&lt;0.05) using the average across replicates (N&gt;30) is graphed for different frequencies of transmission of wild-type chromosomes. Simulations were carried out as described in <a href="#s4">Methods</a>. Simulated data sets are represented as follows: diamonds, AR; squares, ST; triangles, PP; circles, SC; no marker, TL. Simulations carried out using parameters from crosses involving heterozygous females are indicated by solid lines and filled in makers, those from crosses using heterozygous males with dashed lines and hollow markers.</p>
<span>doi:10.1371/journal.pone.0000530.g003</span></div><div class="figure" id="pone-0000530-g004"><div class="img"><a name="pone-0000530-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000530.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000530.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Distribution of variances from simulations of crossing scheme.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>1,000 simulations were carried out using the observed frequency of transmission (sum across replicates) for crosses involving the following heterozygotes: (A) AR females, (B) ST females, (C) PP females, (D) SC females, (E) TL females, (F) AR males, (G) ST males, (H) PP males, (I), SC males, (J) TL males. The observed sample variance of the replicates, mean of the sample variance from 1,000 simulations, and the significance of a one tailed test for observed variance greater than mean simulated variance are shown.</p>
<span>doi:10.1371/journal.pone.0000530.g004</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>The aforementioned deviations from Mendelian expectations (<a href="#pone-0000530-t001">Table 1</a>) may be the result of viability effects of the mutant chromosomes or the wild-type chromosomes. There is a significant effect of chromosomal arrangement carried by the heterozygous parent on the frequency of wild-type non-recombinants (<a href="#pone-0000530-t002">Table 2</a>). There is also a significant effect of the interaction of chromosomal arrangement and sex of the heterozygous parent, but no effect of the sex of the heterozygous parent alone (<a href="#pone-0000530-t002">Table 2</a>). Therefore, the viability of the mutant chromosome depends on the gene arrangement of the wild-type chromosome. In simulated data sets where the transmission of wild-type chromosomes do not depart from Mendelian expectations, we rarely observe a significant effect of the sex of the heterozygous parent, chromosomal arrangement, or sex by arrangement interaction (<a href="#pone-0000530-t003">Table 3</a>). If, however, we simulate data using the observed frequency of wild-type non-recombinants from summing across replicates, we observe a significant effect of chromosomal arrangement in over 75% of the simulations and sex by arrangement interaction in over 60% of the simulations (<a href="#pone-0000530-t003">Table 3</a>). A significant effect of sex of the heterozygous parent alone is only observed in 5% of the simulations. We also observe a significant effect of both arrangement and sex by arrangement interaction in about half of the simulations; this is consistent with the expected result if these two effects were independent.</p>
<div class="figure" id="pone-0000530-t002"><div class="img"><a name="pone-0000530-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000530.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000530.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Analysis of variance of replicates (N&gt;30).</span></strong></p><span>doi:10.1371/journal.pone.0000530.t002</span></div><div class="figure" id="pone-0000530-t003"><div class="img"><a name="pone-0000530-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000530" data-uri="info:doi/10.1371/journal.pone.0000530.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000530.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000530.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000530.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000530.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000530.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Frequency of significant effects of sex, arrangement, and sex × arrangement interaction in simulated data sets.</span></strong></p><span>doi:10.1371/journal.pone.0000530.t003</span></div><a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>The variance of the frequency of wild-type non-recombinants in the average across replicates was calculated for the simulated data sets in which the observed frequency of transmission of the wild-type non-recombinants (sum across replicates) was used. The observed variance of the frequency of wild-type non-recombinants in the average across replicates is significantly greater than that expected based on the simulations in only two crosses – the crosses involving heterozygous ST males and heterozygous PP females (<a href="#pone-0000530-g003">Figure 3</a>). In all other crosses, the observed variance falls within the 95% confidence interval of the simulated variances.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The transmission of gene arrangements was previously observed to depart from the expected one to one Mendelian expectations in experiments examining recombination in the <em>D. pseudoobscura</em> third chromosome inversion system <a href="#pone.0000530-Dobzhansky3">[15]</a>–<a href="#pone.0000530-Levine3">[17]</a>. These observed departures could be attributed to fitness differences between homologous chromosomes in heterozygotes, non-random disjunction of the two arrangements during achiasmate segregation in female meioses (meiotic drive), or some combination of fitness effects and meiotic drive effects. Distinguishing between the alternative hypotheses is not possible using the previously published data because reciprocal crosses are necessary to differentiate between fitness effects and meiotic drive. If both crosses in a reciprocal pair produce equivalent deviations from Mendelian expectations then the deviations are most likely the result of fitness differences between wild- and mutant-type chromosomes. If, however, the deviations are only observed in the crosses involving heterozygous females then they most likely result from differences in the achiasmate meiotic pathways of males and females. In cases where there is a significant effect of the sex of the heterozygous parent and both reciprocal crosses produce significant deviations from Mendelian expectations then both fitness effects and meiotic drive effects must be invoked to explain the observations.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We performed reciprocal crosses to determine whether the deviations from Mendelian expectations are due to fitness effects or non-random disjunction. We observed a significant excess of wild-type non-recombinant chromosomes in only 4 of 10 crosses (<a href="#pone-0000530-t001">Table 1</a>, Sum Across Replicates). We observed significant differences between reciprocal crosses in crosses involving one of the five arrangements, but the wild-type chromosome in this cross (AR) carries the same arrangement as the mutant marker chromosome. This supports the hypothesis that deviations from Mendelian expectations are due to fitness effects rather than mechanistic effects such as meiotic drive due to chromosomal inversions.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Given our sample sizes, we should be able to detect significant differences between reciprocal crosses (summing across replicates) if the wild-type chromosome is transmitted at a frequency of 0.50 in males and at least 0.55 in females (see <a href="#s2">Results</a>). Some of the previously published crossing data had over 70% wild-type non-recombinant progeny in every cross <a href="#pone.0000530-Dobzhansky3">[15]</a>, while other experiments observed 53–62% wild-type non-recombinants <a href="#pone.0000530-Levine3">[17]</a>. This indicates that we had sufficient power to detect meiotic drive were it responsible for the previously observed deviations from Mendelian expectations.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The differences between our results and the previously published studies could be a result of differences in rearing temperatures between experiments – our flies were mated and developed at 18°C, while some of the previous experiments were performed at 25°C <a href="#pone.0000530-Levine2">[16]</a>, <a href="#pone.0000530-Levine3">[17]</a>. We also used a different mutant marker chromosome than all of the previous experiments; our chromosome carried the same mutations as previous studies, but we used an AR chromosome rather than a ST arrangement because the ST marker chromosome is no longer available.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>It is possible that the population density within the vials could have lead to increased deviations from Mendelian expectations in the previous experiments, however Levine and Levine <a href="#pone.0000530-Levine2">[16]</a>, <a href="#pone.0000530-Levine3">[17]</a> also performed single pair matings. Dobzhansky and Epling <a href="#pone.0000530-Dobzhansky3">[15]</a> left no record as to whether they performed single pair matings or mass matings. Mass matings would increase the density of larva in the medium, which could increase competition between individuals. Increased competition would enhance the viability of the wild-type individuals relative to mutants. By performing single pair matings, viability effects should be minimized.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The analysis of variance revealed a significant effect of the gene arrangement carried by the heterozygous parent and the interaction of sex of the heterozygous parent and gene arrangement on the frequency of wild-type non-recombinants (<a href="#pone-0000530-t002">Table 2</a>). The lack of a significant effect of sex of the heterozygous parent alone on the transmission of the wild-type chromosomes further supports the hypothesis that fitness effects (rather than meiotic drive because of rearrangements) were responsible for the deviations from Mendelian expectations observed in previous experiments. The effect of arrangement and sex by arrangement interaction indicates that different arrangements confer different fitness benefits relative to the mutant chromosome and that this effect depends on the sex of the mutant parent. This may be the result of different maternal effects of the wild-type chromosomes on the viability of the progeny. A significant effect of arrangement and sex by arrangement interaction are expected to occur rarely if all arrangements are transmitted according to Mendelian expectations, but they should be common if each arrangement is transmitted at the frequency observed when summing across replicates (<a href="#pone-0000530-t003">Table 3</a>).</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The amount of variance among replicates is fairly consistent with that expected if all replicates had the same expected frequency of transmission of the wild-type chromosome and if that frequency were determined by the sum across replicates (<a href="#pone-0000530-g003">Figure 3</a>); we only observe two instances out of ten where the observed variance is significantly larger than expected. Furthermore, the observed variance is less than expected in five out of ten crosses, although these deviations are not significant. This suggests that all replicates of a particular cross (same sex of heterozygous parent and wild-type arrangement) are indeed replicates of the same binomial sampling process.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>We have no evidence that segregation distortion because of chromosomal inversions is responsible for the maintenance of the <em>D. pseudoobscura</em> inversion polymorphism. We cannot reject the possibility that environmental effects (such as temperature) or demography (such as population density) may influence allele frequencies. Examining these effects requires further experimentation in which the rearing temperature, size of vial, and number of mated females per vial is varied. Our data support the hypothesis that the <em>D. pseudoobscura</em> third chromosome arrangements are not subject to non-random disjunction because of the inverted regions. This conclusion is consistent with selection maintaining the arrangement polymorphism due to fitness benefits conferred by coadapted gene complexes, which are maintained by suppressed recombination between arrangements.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cross scheme:</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>A heterozygous strain for the AR and Cuernavaca (CU) gene arrangements was provided by Wyatt Anderson (University of Georgia). The AR chromosome carries four phenotypic markers: <em>orange</em> (<em>or</em>), <em>Blade</em> (<em>Bl</em>), <em>purple</em> (<em>pr</em>), and <em>crossveinless</em> (<em>cv</em>). The <em>or</em> and <em>pr</em> mutations correspond to <em>cinnabar</em> and <em>brown</em> in <em>D. melanogaster</em>, respectively <a href="#pone.0000530-Sturtevant2">[21]</a>. This differs from previous experiments in which the marker chromosome carried the ST arrangement <a href="#pone.0000530-Dobzhansky3">[15]</a>–<a href="#pone.0000530-Levine3">[17]</a>; the ST marker chromosome no longer exists. Wild-type third chromosomes came from the following isochromosomal or inbred lines: MV2-25 (AR), JR117ST L (ST), DM1053PP B (PP), JR0032SC B (SC), MSH130TL L (TL). The MV2-25 line was sequenced in the <em>D. pseudoobscura</em> genome project <a href="#pone.0000530-Richards1">[22]</a>, and the other four lines were used in a survey of nucleotide polymorphism in the gene arrangements <a href="#pone.0000530-Schaeffer2">[13]</a>. Each wild-type chromosome differs from the mutant AR chromosome by a different number of inversions (ranging from zero to four), and each wild-type arrangement is found at a frequency of at least 10% in some natural populations <a href="#pone.0000530-Anderson1">[3]</a>. All crosses were carried out on a cornmeal, molasses, agar, and yeast medium in 25×95 mm shell vials at 18°C.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>We crossed single virgin females from the <em>or Bl pr cv</em> marker stock to single males homozygous for a wild-type chromosome carrying either the AR, ST, PP, SC, or TL arrangement (<a href="#pone-0000530-g002">Figure 2</a>). The heterozygous male and female progeny from these crosses carrying the mutant marked AR chromosome and a wild-type chromosome were selected. These male and virgin female progeny were test crossed in single pair matings to individuals from a marker stock homozygous for the <em>or</em> and <em>pr</em> mutations on an AR background (<a href="#pone-0000530-g002">Figure 2</a>). The parents were cleared after two weeks and the emerging progeny were scored for the <em>or, Bl</em>, and <em>pr</em> mutations two weeks later. Single pair matings involving heterozygous parents of the same sex and carrying the same arrangement for their wild-type chromosome are considered replicates (<a href="#pone-0000530-g002">Figure 2</a>; <a href="#pone-0000530-t001">Table 1</a>).</p>


<h4>Data analysis:</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The number of individuals in each of two non-recombinant classes (either wild-type or <em>or Bl pr</em>) was summed across replicates, and the frequency of wild-type non-recombinants out of all non-recombinant progeny was determined for the “sum across replicates”. The frequency of wild-type non-recombinants was also determined for each replicate with at least thirty non-recombinant progeny (N&gt;30). Both the sample mean of the replicates (“average of replicates”) and the sample variance were calculated for each of the crosses using replicates with N&gt;30. An analysis of variance (general linear model) was performed using the replicates with N&gt;30 on the frequency of wild-type non-recombinants after an arcsine transformation <a href="#pone.0000530-Sokal1">[23]</a> to determine the effects of the sex of the heterozygous parent, the wild-type arrangement carried by the heterozygous parent, and the interaction of sex and arrangement.</p>


<h4>Simulations:</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>We simulated 1,000 runs of our experiment using different expected frequencies of transmission of wild-type non-recombinants. The number of progeny in each replicate and the number of replicates for each cross from the experiment were used in each simulation. Simulations were performed using expected frequencies of wild-type non-recombinants of 0.50, 0.55, 0.60, 0.65, 0.70, 0.75, 0.80, 0.85, 0.90, 0.95, and 0.99. We determined whether the frequency of wild-type non-recombinants (averaged across replicates) was significantly greater than that expected under Mendelian inheritance for each simulated replication of the experiment for all 11 frequencies of transmission. We also calculated the effects of sex of the heterozygous parent, chromosomal arrangement, and the sex by arrangement interaction in our simulated data using an ANOVA as described above. Simulations were also performed using the observed frequency of wild-type non-recombinants from the sum across replicates as the expected frequency of wild-type non-recombinants for each replicate in the simulation. For each of these simulations, we performed an ANOVA (see above) and determined the variance among replicates.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000530"></div><a name="pone.0000530.s001" id="pone.0000530.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000530.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Results of previous crossing experiments. The frequency of wild-type non-recombinants is shown for each wild-type arrangement in experiments performed by Dobzhansky and Epling <a href="#pone.0000530-Dobzhansky3">[15]</a>, Levine and Levine <a href="#pone.0000530-Levine3">[17]</a>, and Levine <a href="#pone.0000530-Levine2">[16]</a>. The first set of crosses by Dobzhanksy and Epling used a marker chromosome carrying the <em>or</em>, <em>pr</em>, and <em>cv</em> mutations on a ST background, while the second set used a chromosome carrying <em>or</em>, <em>Bl</em>, <em>Sc</em>, and <em>pr</em> on the ST background. Error bars represent 95% confidence intervals, and the dashed line shows the expectation under Mendelian inheritance and no natural selection. The relationships of the different arrangements are shown in <a href="#pone-0000530-g001">Figure 1</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000530.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.73 MB EPS)</p>
<a name="pone.0000530.s002" id="pone.0000530.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000530.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Transmission frequencies of mutant alleles in previous crossing experiments. Frequency of transmission for individual mutant alleles in crossing experiments involving wild-type and marker chromosomes carrying the ST arrangement were calculated. Error bars indicate 95% confidence intervals, and the dashed line shows the expectation under Mendelian inheritance. Data taken from Dobzhanksy and Epling's crosses <a href="#pone.0000530-Dobzhansky3">[15]</a> involving (A) the marker chromosome carrying the <em>or</em>, <em>pr</em>, and <em>cv</em> mutations, (B) the marker chromosome carrying the <em>or</em>, <em>Bl</em>, <em>Sc</em>, and <em>pr</em> mutations and (C–G) Levine and Levine's crosses <a href="#pone.0000530-Levine3">[17]</a> involving the marker chromosome carrying the <em>or</em>, <em>Bl</em>, <em>Sc</em>, and <em>pr</em> mutations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000530.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.66 MB EPS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank E. Novitski for providing the inspiration for this work. W. Anderson provided the mutant marker stocks, and C. dePamphilis suggested the power analysis.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RM SS. Performed the experiments: RM. Analyzed the data: RM SS. Wrote the paper: RM SS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000530-Powell1" id="pone.0000530-Powell1"></a>Powell JR (1992)  Inversion polymorphisms in <em>Drosophila pseudoobscura</em> and <em>Drosophila persimilis</em>. In: Krimbas CB, Powell JR, editors. Drosophila Inversion Polymorphism. Boca Raton: CRC Press. pp. 73–126.  <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000530-Dobzhansky1" id="pone.0000530-Dobzhansky1"></a>Dobzhansky T (1948) Genetics of natural populations. XVI. Altitudinal and seasonal changes produce by natural selection in certain populations of <em>Drosophila pseudoobscura</em> and <em>Drosophila persimilis</em>. Genetics  33: 158–176.  <ul class="find" data-citedArticleID="958922" data-doi="10.2307/2406099"><li><a href="http://dx.doi.org/10.2307/2406099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+natural+populations.+XVI.+Altitudinal+and+seasonal+changes+produce+by+natural+selection+in+certain+populations+of+Drosophila+pseudoobscura+and+Drosophila+persimilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+natural+populations.+XVI.+Altitudinal+and+seasonal+changes+produce+by+natural+selection+in+certain+populations+of+Drosophila+pseudoobscura+and+Drosophila+persimilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000530-Anderson1" id="pone.0000530-Anderson1"></a>Anderson WW, Arnold J, Baldwin D, Beckenbach A, Brown C, et al.  (1991) Four decades of inversion polymorphism in <em>Drosophila pseudoobscura</em>. Proc Natl Acad Sci USA  88: 10367–10371.  <ul class="find" data-citedArticleID="958918" data-doi="10.1073/pnas.88.22.10367"><li><a href="http://dx.doi.org/10.1073/pnas.88.22.10367" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Four+decades+of+inversion+polymorphism+in+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Four+decades+of+inversion+polymorphism+in+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000530-Riley1" id="pone.0000530-Riley1"></a>Riley MA, Hallas ME, Lewontin RC (1989) Distinguishing the forces controlling genetic variation at the <em>Xdh</em> locus in <em>Drosophila pseudoobscura</em>. Genetics  123: 359–369.  <ul class="find" data-citedArticleID="958946"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Distinguishing+the+forces+controlling+genetic+variation+at+the+Xdh+locus+in+Drosophila+pseudoobscura.&amp;auth=&amp;atitle=Distinguishing+the+forces+controlling+genetic+variation+at+the+Xdh+locus+in+Drosophila+pseudoobscura." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinguishing+the+forces+controlling+genetic+variation+at+the+Xdh+locus+in+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinguishing+the+forces+controlling+genetic+variation+at+the+Xdh+locus+in+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000530-Schaeffer1" id="pone.0000530-Schaeffer1"></a>Schaeffer SW, Miller EL (1992) Estimates of gene flow in <em>Drosophila pseudoobscura</em> determined from nucleotide sequence analysis of the alcohol dehydrogenase region. Genetics  132: 471–480.  <ul class="find" data-citedArticleID="958948"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Estimates+of+gene+flow+in+Drosophila+pseudoobscura+determined+from+nucleotide+sequence+analysis+of+the+alcohol+dehydrogenase+region.&amp;auth=&amp;atitle=Estimates+of+gene+flow+in+Drosophila+pseudoobscura+determined+from+nucleotide+sequence+analysis+of+the+alcohol+dehydrogenase+region." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimates+of+gene+flow+in+Drosophila+pseudoobscura+determined+from+nucleotide+sequence+analysis+of+the+alcohol+dehydrogenase+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimates+of+gene+flow+in+Drosophila+pseudoobscura+determined+from+nucleotide+sequence+analysis+of+the+alcohol+dehydrogenase+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000530-Kovacevic1" id="pone.0000530-Kovacevic1"></a>Kovacevic M, Schaeffer SW (2000) Molecular population genetics of X-linked genes in <em>Drosophila pseudoobscura</em>. Genetics  156: 155–172.  <ul class="find" data-citedArticleID="958930"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+population+genetics+of+X-linked+genes+in+Drosophila+pseudoobscura.&amp;auth=&amp;atitle=Molecular+population+genetics+of+X-linked+genes+in+Drosophila+pseudoobscura." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+population+genetics+of+X-linked+genes+in+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+population+genetics+of+X-linked+genes+in+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000530-Wright1" id="pone.0000530-Wright1"></a>Wright S, Dobzhansky T (1946) Genetics of natural populations. XII. Experimental reproduction of some of the changes caused by natural selection in certain populations of <em>Drosophila pseudoobscura</em>. Genetics  31: 125–156.  <ul class="find" data-citedArticleID="958964" data-doi="10.2307/2406099"><li><a href="http://dx.doi.org/10.2307/2406099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+natural+populations.+XII.+Experimental+reproduction+of+some+of+the+changes+caused+by+natural+selection+in+certain+populations+of+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+natural+populations.+XII.+Experimental+reproduction+of+some+of+the+changes+caused+by+natural+selection+in+certain+populations+of+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000530-Dobzhansky2" id="pone.0000530-Dobzhansky2"></a>Dobzhansky T (1950) Genetics of natural populations. XIX. Origin of heterosis through natural selection in populations of <em>Drosophila pseudoobscura</em>. Genetics  35: 288–302.  <ul class="find" data-citedArticleID="958924" data-doi="10.1086/281674"><li><a href="http://dx.doi.org/10.1086/281674" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+natural+populations.+XIX.+Origin+of+heterosis+through+natural+selection+in+populations+of+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+natural+populations.+XIX.+Origin+of+heterosis+through+natural+selection+in+populations+of+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000530-Watanabe1" id="pone.0000530-Watanabe1"></a>Watanabe T, Anderson WW, Dobzhansky T, Pavlovsky O (1970) Selection in experimental populations of <em>Drosophila pseudoobscura</em> with different initial frequencies of chromosomal variants. Genet Res  15: 123–129.  <ul class="find" data-citedArticleID="958962" data-doi="10.1017/s0016672300001427"><li><a href="http://dx.doi.org/10.1017/s0016672300001427" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+in+experimental+populations+of+Drosophila+pseudoobscura+with+different+initial+frequencies+of+chromosomal+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+in+experimental+populations+of+Drosophila+pseudoobscura+with+different+initial+frequencies+of+chromosomal+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000530-Levine1" id="pone.0000530-Levine1"></a>Levine L (1955) Genotypic background and heterosis in <em>Drosophila pseudoobscura</em>. Genetics  40: 832–849.  <ul class="find" data-citedArticleID="958932"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genotypic+background+and+heterosis+in+Drosophila+pseudoobscura.&amp;auth=&amp;atitle=Genotypic+background+and+heterosis+in+Drosophila+pseudoobscura." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotypic+background+and+heterosis+in+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotypic+background+and+heterosis+in+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000530-Strickberger1" id="pone.0000530-Strickberger1"></a>Strickberger MW (1963) Evolution of fitness in experimental populations of <em>Drosophila pseudoobscura</em>. Evolution  17: 40–55.  <ul class="find" data-citedArticleID="958956" data-doi="10.2307/2406334"><li><a href="http://dx.doi.org/10.2307/2406334" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+fitness+in+experimental+populations+of+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+fitness+in+experimental+populations+of+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000530-Anderson2" id="pone.0000530-Anderson2"></a>Anderson WW (1968) Further evidence for coadaptation in crosses between geographic populations of <em>Drosophila pseudoobscura</em>. Genet Res  12: 317–330.  <ul class="find" data-citedArticleID="958920" data-doi="10.1017/s0016672300011903"><li><a href="http://dx.doi.org/10.1017/s0016672300011903" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Further+evidence+for+coadaptation+in+crosses+between+geographic+populations+of+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Further+evidence+for+coadaptation+in+crosses+between+geographic+populations+of+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000530-Schaeffer2" id="pone.0000530-Schaeffer2"></a>Schaeffer SW, Goetting-Minesky MP, Kovacevic M, Peoples JR, Graybill JL, et al.  (2003) Evolutionary genomics of inversions in <em>Drosophila pseudoobscura</em>: evidence for epistasis. Proc Natl Acad Sci USA  100: 8319–8324.  <ul class="find" data-citedArticleID="958950" data-doi="10.1073/pnas.1432900100"><li><a href="http://dx.doi.org/10.1073/pnas.1432900100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+genomics+of+inversions+in+Drosophila+pseudoobscura%3A+evidence+for+epistasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+genomics+of+inversions+in+Drosophila+pseudoobscura%3A+evidence+for+epistasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000530-Schaeffer3" id="pone.0000530-Schaeffer3"></a>Schaeffer SW, Anderson WW (2005) Mechanisms of genetic exchange within the chromosomal inversions of <em>Drosophila pseudoobscura</em>. Genetics  171: 1729–1739.  <ul class="find" data-citedArticleID="958952" data-doi="10.1534/genetics.105.041947"><li><a href="http://dx.doi.org/10.1534/genetics.105.041947" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+genetic+exchange+within+the+chromosomal+inversions+of+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+genetic+exchange+within+the+chromosomal+inversions+of+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000530-Dobzhansky3" id="pone.0000530-Dobzhansky3"></a>Dobzhansky T, Epling C (1948) The suppression of crossing over in inversion heterozygotes of <em>Drosophila pseudoobscura</em>. Proc Natl Acad Sci USA  34: 137–141.  <ul class="find" data-citedArticleID="958926" data-doi="10.1073/pnas.34.4.137"><li><a href="http://dx.doi.org/10.1073/pnas.34.4.137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+suppression+of+crossing+over+in+inversion+heterozygotes+of+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+suppression+of+crossing+over+in+inversion+heterozygotes+of+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000530-Levine2" id="pone.0000530-Levine2"></a>Levine RP (1956) Crossing over and inversions in coadapted systems. Am Nat  90: 41–45.  <ul class="find" data-citedArticleID="958934" data-doi="10.1086/281905"><li><a href="http://dx.doi.org/10.1086/281905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crossing+over+and+inversions+in+coadapted+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crossing+over+and+inversions+in+coadapted+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000530-Levine3" id="pone.0000530-Levine3"></a>Levine RP, Levine EE (1955) Variable crossing over arising in different strains of <em>Drosophila pseudoobscura</em>. Genetics  40: 399–405.  <ul class="find" data-citedArticleID="958936"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Variable+crossing+over+arising+in+different+strains+of+Drosophila+pseudoobscura.&amp;auth=&amp;atitle=Variable+crossing+over+arising+in+different+strains+of+Drosophila+pseudoobscura." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variable+crossing+over+arising+in+different+strains+of+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variable+crossing+over+arising+in+different+strains+of+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000530-Hawley1" id="pone.0000530-Hawley1"></a>Hawley RS, McKim KS, Arbel T (1993) Meiotic segregation in <em>Drosophila melanogaster</em> females: moleculer, mechanisms, and myths. Annu Rev Genet  27: 281–319.  <ul class="find" data-citedArticleID="958928" data-doi="10.1146/annurev.ge.27.120193.001433"><li><a href="http://dx.doi.org/10.1146/annurev.ge.27.120193.001433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Meiotic+segregation+in+Drosophila+melanogaster+females%3A+moleculer%2C+mechanisms%2C+and+myths." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Meiotic+segregation+in+Drosophila+melanogaster+females%3A+moleculer%2C+mechanisms%2C+and+myths.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000530-OrrWeaver1" id="pone.0000530-OrrWeaver1"></a>Orr-Weaver TL (1995) Meiosis in <em>Drosophila</em>: seeing is believing. Proc Natl Acad Sci USA  92: 10443–10449.  <ul class="find" data-citedArticleID="958938" data-doi="10.1073/pnas.92.23.10443"><li><a href="http://dx.doi.org/10.1073/pnas.92.23.10443" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Meiosis+in+Drosophila%3A+seeing+is+believing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Meiosis+in+Drosophila%3A+seeing+is+believing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000530-Sturtevant1" id="pone.0000530-Sturtevant1"></a>Sturtevant AH (1936) Preferential segregation in triplo-IV females of <em>Drosophila melanogaster</em>. Genetics  21: 444–466.  <ul class="find" data-citedArticleID="958958"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Preferential+segregation+in+triplo-IV+females+of+Drosophila+melanogaster.&amp;auth=&amp;atitle=Preferential+segregation+in+triplo-IV+females+of+Drosophila+melanogaster." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preferential+segregation+in+triplo-IV+females+of+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preferential+segregation+in+triplo-IV+females+of+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000530-Sturtevant2" id="pone.0000530-Sturtevant2"></a>Sturtevant AH, Tan CC (1937) The comparative genetics of <em>Drosophila pseudoobscura</em> and <em>Drosophila melanogaster</em>. J Genet  34: 415–432.  <ul class="find" data-citedArticleID="958960" data-doi="10.1007/bf02982303"><li><a href="http://dx.doi.org/10.1007/bf02982303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+comparative+genetics+of+Drosophila+pseudoobscura+and+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+comparative+genetics+of+Drosophila+pseudoobscura+and+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000530-Richards1" id="pone.0000530-Richards1"></a>Richards S, Liu Y, Bettencourt BR, Hradecky P, Letovsky S, et al.  (2005) Comparative genome sequencing of <em>Drosophila pseudoobscura</em>: chromosomal, gene, and <em>cis</em>-element evolution. Genome Res  15: 1–18.  <ul class="find" data-citedArticleID="958944" data-doi="10.1101/gr.3059305"><li><a href="http://dx.doi.org/10.1101/gr.3059305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genome+sequencing+of+Drosophila+pseudoobscura%3A+chromosomal%2C+gene%2C+and+cis-element+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genome+sequencing+of+Drosophila+pseudoobscura%3A+chromosomal%2C+gene%2C+and+cis-element+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000530-Sokal1" id="pone.0000530-Sokal1"></a>Sokal RR, Rohlf FJ (1995) Biometry. New York: W.H. Freeman and Co.   <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0000530-Popadic1" id="pone.0000530-Popadic1"></a>Popadic A, Norman R, Doanet W, Anderson W (1996) The evolutionary history of the amylase multigene family in <em>Drosophila pseudoobscura</em>. Mol Biol Evol  13: 883–888.  <ul class="find" data-citedArticleID="958940" data-doi="10.1093/oxfordjournals.molbev.a025648"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025648" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+history+of+the+amylase+multigene+family+in+Drosophila+pseudoobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+history+of+the+amylase+multigene+family+in+Drosophila+pseudoobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.XML" value="73300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.PDF" value="154774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g001.PNG_L" value="6217"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g001.PNG_M" value="6217"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g001.PNG_S" value="2195"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g001.TIF" value="13602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g001.PNG_I" value="5515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g002.PNG_L" value="49892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g002.PNG_M" value="31230"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g002.PNG_S" value="2289"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g002.TIF" value="92494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g002.PNG_I" value="13621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t001.PNG_L" value="169408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t001.PNG_M" value="93050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t001.PNG_S" value="7750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t001.TIF" value="845068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t001.PNG_I" value="86063"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g003.PNG_L" value="42292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g003.PNG_M" value="18133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g003.PNG_S" value="2051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g003.TIF" value="108388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g003.PNG_I" value="8499"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g004.PNG_L" value="61849"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g004.PNG_M" value="23030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g004.PNG_S" value="2700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g004.TIF" value="179612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.g004.PNG_I" value="22564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t002.PNG_L" value="71260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t002.PNG_M" value="73738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t002.PNG_S" value="2977"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t002.TIF" value="342612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t002.PNG_I" value="33303"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t003.PNG_L" value="73663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t003.PNG_M" value="82660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t003.PNG_S" value="2955"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t003.TIF" value="321092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.t003.PNG_I" value="37549"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.s001.EPS" value="728164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000530.s002.EPS" value="655741"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000530&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000530" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000530&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000530&volume=&issue=&title=Meiotic Transmission of Drosophila pseudoobscura Chromosomal Arrangements&author_name=Richard%20P.%20Meisel%2C%20Stephen%20W.%20Schaeffer&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000530" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000530" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000530" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000530&amp;t=Meiotic%20Transmission%20of%20Drosophila%20pseudoobscura%20Chromosomal%20Arrangements" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000530&title=Meiotic%20Transmission%20of%20Drosophila%20pseudoobscura%20Chromosomal%20Arrangements&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000530&amp;title=Meiotic%20Transmission%20of%20Drosophila%20pseudoobscura%20Chromosomal%20Arrangements" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000530" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000530&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Meiotic Transmission of Drosophila pseudoobscura Chromosomal Arrangements';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000530';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Meiotic%20Transmission%20of%20Drosophila%20pseudoobscura%20Chromosomal%20Arrangements http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000530" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000530" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">









          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="23898"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+pairs%22" title="Search for articles in the subject area:'Chromosome pairs'"><div class="flagText">Chromosome pairs</div></a>
              <div data-categoryid="17419" data-articleid="23898"
                   data-categoryname="Chromosome pairs"
                   class="flagImage" title="Flag 'Chromosome pairs' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="23898"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Homologous+chromosomes%22" title="Search for articles in the subject area:'Homologous chromosomes'"><div class="flagText">Homologous chromosomes</div></a>
              <div data-categoryid="21187" data-articleid="23898"
                   data-categoryname="Homologous chromosomes"
                   class="flagImage" title="Flag 'Homologous chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Homologous+recombination%22" title="Search for articles in the subject area:'Homologous recombination'"><div class="flagText">Homologous recombination</div></a>
              <div data-categoryid="34037" data-articleid="23898"
                   data-categoryname="Homologous recombination"
                   class="flagImage" title="Flag 'Homologous recombination' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Inversions%22" title="Search for articles in the subject area:'Inversions'"><div class="flagText">Inversions</div></a>
              <div data-categoryid="43105" data-articleid="23898"
                   data-categoryname="Inversions"
                   class="flagImage" title="Flag 'Inversions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="23898"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sex+chromosomes%22" title="Search for articles in the subject area:'Sex chromosomes'"><div class="flagText">Sex chromosomes</div></a>
              <div data-categoryid="18743" data-articleid="23898"
                   data-categoryname="Sex chromosomes"
                   class="flagImage" title="Flag 'Sex chromosomes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=838'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6992'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1697&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=9699">Figure 4</a><br>Posted by rmeisel</p>
      <p><a href="/annotation/listThread.action?root=9693">Mislabel</a><br>Posted by rmeisel</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
